
<html lang="en"     class="pb-page"  data-request-id="971ac36a-93a2-4953-bace-85ebeaa190b4"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;article:article:10.1021/acs.jmedchem.9b00113;wgroup:string:ACHS website Group;website:website:acspubs;issue:issue:10.1021/jmcmar.2019.62.issue-5;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Development of Highly Potent, Selective, and Cellular Active Triazolo[1,5-a]pyrimidine-Based Inhibitors Targeting the DCN1–UBC12 Protein–Protein Interaction" /></meta><meta name="dc.Creator" content="Shuai  Wang" /></meta><meta name="dc.Creator" content="Lijie  Zhao" /></meta><meta name="dc.Creator" content="Xiao-Jing  Shi" /></meta><meta name="dc.Creator" content="Lina  Ding" /></meta><meta name="dc.Creator" content="Linlin  Yang" /></meta><meta name="dc.Creator" content="Zhi-Zheng  Wang" /></meta><meta name="dc.Creator" content="Dandan  Shen" /></meta><meta name="dc.Creator" content="Kai  Tang" /></meta><meta name="dc.Creator" content="Xiao-Jing  Li" /></meta><meta name="dc.Creator" content="MAA  Mamun" /></meta><meta name="dc.Creator" content="Huiju  Li" /></meta><meta name="dc.Creator" content="Bin  Yu" /></meta><meta name="dc.Creator" content="Yi-Chao  Zheng" /></meta><meta name="dc.Creator" content="Shaomeng  Wang" /></meta><meta name="dc.Creator" content="Hong-Min  Liu" /></meta><meta name="dc.Description" content="The cullin-RING ubiquitin ligases (CRLs) are responsible for about 20% of cellular protein degradation and regulate diverse cellular processes, and the dysfunction of CRLs is implicated in human di..." /></meta><meta name="Description" content="The cullin-RING ubiquitin ligases (CRLs) are responsible for about 20% of cellular protein degradation and regulate diverse cellular processes, and the dysfunction of CRLs is implicated in human di..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="February 25, 2019" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.9b00113" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Copyright © 2019 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.9b00113" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.9b00113" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.9b00113" /></link>
        
    
    

<title>Development of Highly Potent, Selective, and Cellular Active Triazolo[1,5-a]pyrimidine-Based Inhibitors Targeting the DCN1–UBC12 Protein–Protein Interaction | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.9b00113" /></meta><meta property="og:title" content="Development of Highly Potent, Selective, and Cellular Active Triazolo[1,5-a]pyrimidine-Based Inhibitors Targeting the DCN1–UBC12 Protein–Protein Interaction" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.9b00113/20190308/images/large/jm-2019-00113s_0023.jpeg" /></meta><meta property="og:description" content="The cullin-RING ubiquitin ligases (CRLs) are responsible for about 20% of cellular protein degradation and regulate diverse cellular processes, and the dysfunction of CRLs is implicated in human diseases. Targeting the CRLs has become an emerging strategy for the treatment of human diseases. Herein, we describe the discovery of a hit compound from our in-house library and further structure-based optimizations, which have enabled the identification of new triazolo[1,5-a]pyrimidine-based inhibitors targeting the DCN1–UBC12 interaction. Compound WS-383 blocks the DCN1–UBC12 interaction (IC50 = 11 nM) reversibly and shows selectivity over selected kinases. WS-383 exhibits cellular target engagement to DCN1 in MGC-803 cells. WS-383 inhibits Cul3/1 neddylation selectively over other cullins and also induces accumulation of p21, p27, and NRF2. Collectively, targeting the DCN1–UBC12 interaction would be a viable strategy for selective neddylation inhibition of Cul3/1 and may be of therapeutic potential for disease treatment in which Cul3/1 is dysregulated." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.9b00113"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.9b00113">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-14/achre4.2021.54.issue-14/20210720/achre4.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-29/aamick.2021.13.issue-29/20210728/aamick.2021.13.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-14/acncdm.2021.12.issue-14/20210721/acncdm.2021.12.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-7/amlcef.2021.3.issue-7/20210705/amlcef.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-29/acsodf.2021.6.issue-29/20210727/acsodf.2021.6.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-29/ascecg.2021.9.issue-29/20210726/ascecg.2021.9.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-29/ancham.2021.93.issue-29/20210727/ancham.2021.93.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-29/bichaw.2021.60.issue-29/20210727/bichaw.2021.60.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-27/cgeabj.2021.99.issue-27/20210726-01/cgeabj.2021.99.issue-27.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-7/cgdefu.2021.21.issue-7/20210707/cgdefu.2021.21.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-14/esthag.2021.55.issue-14/20210720/esthag.2021.55.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-29/iecred.2021.60.issue-29/20210728/iecred.2021.60.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-14/inocaj.2021.60.issue-14/20210719/inocaj.2021.60.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-29/jafcau.2021.69.issue-29/20210728/jafcau.2021.69.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-29/jacsat.2021.143.issue-29/20210728/jacsat.2021.143.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-7/jamsef.2021.32.issue-7/20210707/jamsef.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-29/langd5.2021.37.issue-29/20210727/langd5.2021.37.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-7/mpohbp.2021.18.issue-7/20210705/mpohbp.2021.18.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.9b00113&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.9b00113&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.9b00113&amp;href=/doi/10.1021/acs.jmedchem.9b00113" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2019</span><span class="cit-fg-volume">, 62</span><span class="cit-fg-issue">, 5</span><span class="cit-fg-pageRange">, 2772-2797</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/62/5" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.9b00062" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.9b00131" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Development of Highly Potent, Selective, and Cellular Active Triazolo[1,5-<i>a</i>]pyrimidine-Based Inhibitors Targeting the DCN1–UBC12 Protein–Protein Interaction</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Shuai Wang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Shuai Wang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">School of Pharmaceutical Sciences and Institute of Drug Discovery & Development, Zhengzhou University, Zhengzhou 450001, China</div><div class="loa-info-affiliations-info">Co-Innovation Center of Henan Province for New Drug R & D and Preclinical Safety, Zhengzhou 450001, China</div><div class="loa-info-affiliations-info">Key Laboratory of Advanced Technology of Drug Preparation Technologies, Zhengzhou University, Ministry of Education of China, Zhengzhou 450001, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Shuai++Wang">Shuai Wang</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Lijie Zhao</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Lijie Zhao</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">School of Pharmaceutical Sciences and Institute of Drug Discovery & Development, Zhengzhou University, Zhengzhou 450001, China</div><div class="loa-info-affiliations-info">Co-Innovation Center of Henan Province for New Drug R & D and Preclinical Safety, Zhengzhou 450001, China</div><div class="loa-info-affiliations-info">Key Laboratory of Advanced Technology of Drug Preparation Technologies, Zhengzhou University, Ministry of Education of China, Zhengzhou 450001, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Lijie++Zhao">Lijie Zhao</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Xiao-Jing Shi</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Xiao-Jing Shi</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">School of Pharmaceutical Sciences and Institute of Drug Discovery & Development, Zhengzhou University, Zhengzhou 450001, China</div><div class="loa-info-affiliations-info">Co-Innovation Center of Henan Province for New Drug R & D and Preclinical Safety, Zhengzhou 450001, China</div><div class="loa-info-affiliations-info">Key Laboratory of Advanced Technology of Drug Preparation Technologies, Zhengzhou University, Ministry of Education of China, Zhengzhou 450001, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Xiao-Jing++Shi">Xiao-Jing Shi</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Lina Ding</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Lina Ding</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">School of Pharmaceutical Sciences and Institute of Drug Discovery & Development, Zhengzhou University, Zhengzhou 450001, China</div><div class="loa-info-affiliations-info">Co-Innovation Center of Henan Province for New Drug R & D and Preclinical Safety, Zhengzhou 450001, China</div><div class="loa-info-affiliations-info">Key Laboratory of Advanced Technology of Drug Preparation Technologies, Zhengzhou University, Ministry of Education of China, Zhengzhou 450001, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Lina++Ding">Lina Ding</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Linlin Yang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Linlin Yang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Pharmacology, School of Basic Medical Sciences, Zhengzhou University, Zhengzhou 450001, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Linlin++Yang">Linlin Yang</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Zhi-Zheng Wang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Zhi-Zheng Wang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">School of Pharmaceutical Sciences and Institute of Drug Discovery & Development, Zhengzhou University, Zhengzhou 450001, China</div><div class="loa-info-affiliations-info">Co-Innovation Center of Henan Province for New Drug R & D and Preclinical Safety, Zhengzhou 450001, China</div><div class="loa-info-affiliations-info">Key Laboratory of Advanced Technology of Drug Preparation Technologies, Zhengzhou University, Ministry of Education of China, Zhengzhou 450001, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Zhi-Zheng++Wang">Zhi-Zheng Wang</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Dandan Shen</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Dandan Shen</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">School of Pharmaceutical Sciences and Institute of Drug Discovery & Development, Zhengzhou University, Zhengzhou 450001, China</div><div class="loa-info-affiliations-info">Co-Innovation Center of Henan Province for New Drug R & D and Preclinical Safety, Zhengzhou 450001, China</div><div class="loa-info-affiliations-info">Key Laboratory of Advanced Technology of Drug Preparation Technologies, Zhengzhou University, Ministry of Education of China, Zhengzhou 450001, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Dandan++Shen">Dandan Shen</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Kai Tang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Kai Tang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">School of Pharmaceutical Sciences and Institute of Drug Discovery & Development, Zhengzhou University, Zhengzhou 450001, China</div><div class="loa-info-affiliations-info">Co-Innovation Center of Henan Province for New Drug R & D and Preclinical Safety, Zhengzhou 450001, China</div><div class="loa-info-affiliations-info">Key Laboratory of Advanced Technology of Drug Preparation Technologies, Zhengzhou University, Ministry of Education of China, Zhengzhou 450001, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Kai++Tang">Kai Tang</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Xiao-Jing Li</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Xiao-Jing Li</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">School of Pharmaceutical Sciences and Institute of Drug Discovery & Development, Zhengzhou University, Zhengzhou 450001, China</div><div class="loa-info-affiliations-info">Co-Innovation Center of Henan Province for New Drug R & D and Preclinical Safety, Zhengzhou 450001, China</div><div class="loa-info-affiliations-info">Key Laboratory of Advanced Technology of Drug Preparation Technologies, Zhengzhou University, Ministry of Education of China, Zhengzhou 450001, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Xiao-Jing++Li">Xiao-Jing Li</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">MAA Mamun</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">MAA Mamun</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">School of Pharmaceutical Sciences and Institute of Drug Discovery & Development, Zhengzhou University, Zhengzhou 450001, China</div><div class="loa-info-affiliations-info">Co-Innovation Center of Henan Province for New Drug R & D and Preclinical Safety, Zhengzhou 450001, China</div><div class="loa-info-affiliations-info">Key Laboratory of Advanced Technology of Drug Preparation Technologies, Zhengzhou University, Ministry of Education of China, Zhengzhou 450001, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=MAA++Mamun">MAA Mamun</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Huiju Li</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Huiju Li</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">School of Pharmaceutical Sciences and Institute of Drug Discovery & Development, Zhengzhou University, Zhengzhou 450001, China</div><div class="loa-info-affiliations-info">Co-Innovation Center of Henan Province for New Drug R & D and Preclinical Safety, Zhengzhou 450001, China</div><div class="loa-info-affiliations-info">Key Laboratory of Advanced Technology of Drug Preparation Technologies, Zhengzhou University, Ministry of Education of China, Zhengzhou 450001, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Huiju++Li">Huiju Li</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Bin Yu</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Bin Yu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">School of Pharmaceutical Sciences and Institute of Drug Discovery & Development, Zhengzhou University, Zhengzhou 450001, China</div><div class="loa-info-affiliations-info">Co-Innovation Center of Henan Province for New Drug R & D and Preclinical Safety, Zhengzhou 450001, China</div><div class="loa-info-affiliations-info">Key Laboratory of Advanced Technology of Drug Preparation Technologies, Zhengzhou University, Ministry of Education of China, Zhengzhou 450001, China</div><div class="loa-info-affiliations-info">State Key Laboratory of Pharmaceutical Biotechnology, Nanjing University, Nanjing 210023, Jiangsu, People’s Republic of China</div></div><span class="conrtib-corresp"><strong>*</strong>B.Y.: phone, +86-0371-67781908; fax, +86-0371-67781908; e-mail, <a href="/cdn-cgi/l/email-protection#a1d8d4c3c8cfe1dbdbd48fc4c5d48fc2cf"><span class="__cf_email__" data-cfemail="b8c1cddad1d6f8c2c2cd96dddccd96dbd6">[email protected]</span></a></span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Bin++Yu">Bin Yu</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-0531-2236" title="Orcid link">http://orcid.org/0000-0002-0531-2236</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Yi-Chao Zheng</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Yi-Chao Zheng</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">School of Pharmaceutical Sciences and Institute of Drug Discovery & Development, Zhengzhou University, Zhengzhou 450001, China</div><div class="loa-info-affiliations-info">Co-Innovation Center of Henan Province for New Drug R & D and Preclinical Safety, Zhengzhou 450001, China</div><div class="loa-info-affiliations-info">Key Laboratory of Advanced Technology of Drug Preparation Technologies, Zhengzhou University, Ministry of Education of China, Zhengzhou 450001, China</div></div><span class="conrtib-corresp"><strong>*</strong>Y.-C.Z.: phone, +86-0371-67781908; fax, +86-0371-67781908; e-mail, <a href="/cdn-cgi/l/email-protection#f78e9e949f96988d9f929990b78d8d82d9929382d99499"><span class="__cf_email__" data-cfemail="ccb5a5afa4ada3b6a4a9a2ab8cb6b6b9e2a9a8b9e2afa2">[email protected]</span></a></span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Yi-Chao++Zheng">Yi-Chao Zheng</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Shaomeng Wang</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Shaomeng Wang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">School of Pharmaceutical Sciences and Institute of Drug Discovery & Development, Zhengzhou University, Zhengzhou 450001, China</div><div class="loa-info-affiliations-info">Departments of Internal Medicine, Pharmacology, Medicinal Chemistry, University of Michigan, 1600 Huron Parkway, Ann Arbor, Michigan 48109, United States</div></div><span class="conrtib-corresp"><strong>*</strong>S.W.: phone, 734-615-0362; fax, 734-764-2532; e-mail, <a href="/cdn-cgi/l/email-protection#3b48535a54565e555c7b4e56525853155e5f4e"><span class="__cf_email__" data-cfemail="611209000e0c040f0621140c0802094f040514">[email protected]</span></a></span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Shaomeng++Wang">Shaomeng Wang</a></span></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Hong-Min Liu</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Hong-Min Liu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">School of Pharmaceutical Sciences and Institute of Drug Discovery & Development, Zhengzhou University, Zhengzhou 450001, China</div><div class="loa-info-affiliations-info">Co-Innovation Center of Henan Province for New Drug R & D and Preclinical Safety, Zhengzhou 450001, China</div><div class="loa-info-affiliations-info">Key Laboratory of Advanced Technology of Drug Preparation Technologies, Zhengzhou University, Ministry of Education of China, Zhengzhou 450001, China</div></div><span class="conrtib-corresp"><strong>*</strong>H.-M.L.: phone, +86-0371-67781908; fax, +86-0371-67781908; e-mail, <a href="/cdn-cgi/l/email-protection#127e7b677a7f526868673c7776673c717c"><span class="__cf_email__" data-cfemail="3c50554954517c46464912595849125f52">[email protected]</span></a></span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Hong-Min++Liu">Hong-Min Liu</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-6771-9421" title="Orcid link">http://orcid.org/0000-0001-6771-9421</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.9b00113&amp;href=/doi/10.1021%2Facs.jmedchem.9b00113" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2019</span></span><span class="cit-volume">, 62</span><span class="cit-issue">, 5</span><span class="cit-pageRange">, 2772–2797</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">February 25, 2019</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>18 January 2019</li><li><span class="item_label"><b>Published</b> online</span>25 February 2019</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 14 March 2019</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.9b00113" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b00113</a></div><div class="article_header-article-copyright"><strong>Copyright © 2019 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D2772%26pageCount%3D26%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DShuai%2BWang%252C%2BLijie%2BZhao%252C%2BXiao-Jing%2BShi%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D62%26issueNum%3D5%26contentID%3Dacs.jmedchem.9b00113%26title%3DDevelopment%2Bof%2BHighly%2BPotent%252C%2BSelective%252C%2Band%2BCellular%2BActive%2BTriazolo%255B1%252C5-a%255Dpyrimidine-Based%2BInhibitors%2BTargeting%2Bthe%2BDCN1%25E2%2580%2593UBC12%2BProtein%25E2%2580%2593Protein%2BInteraction%26numPages%3D26%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D2797%26publicationDate%3DMarch%2B2019">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.9b00113"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">2581</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">32</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.9b00113" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Development of Highly Potent, Selective, and Cellular Active Triazolo[1,5-a]pyrimidine-Based Inhibitors Targeting the DCN1–UBC12 Protein–Protein Interaction&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Shuai&quot;,&quot;last_name&quot;:&quot;Wang&quot;},{&quot;first_name&quot;:&quot;Lijie&quot;,&quot;last_name&quot;:&quot;Zhao&quot;},{&quot;first_name&quot;:&quot;Xiao-Jing&quot;,&quot;last_name&quot;:&quot;Shi&quot;},{&quot;first_name&quot;:&quot;Lina&quot;,&quot;last_name&quot;:&quot;Ding&quot;},{&quot;first_name&quot;:&quot;Linlin&quot;,&quot;last_name&quot;:&quot;Yang&quot;},{&quot;first_name&quot;:&quot;Zhi-Zheng&quot;,&quot;last_name&quot;:&quot;Wang&quot;},{&quot;first_name&quot;:&quot;Dandan&quot;,&quot;last_name&quot;:&quot;Shen&quot;},{&quot;first_name&quot;:&quot;Kai&quot;,&quot;last_name&quot;:&quot;Tang&quot;},{&quot;first_name&quot;:&quot;Xiao-Jing&quot;,&quot;last_name&quot;:&quot;Li&quot;},{&quot;first_name&quot;:&quot;MAA&quot;,&quot;last_name&quot;:&quot;Mamun&quot;},{&quot;first_name&quot;:&quot;Huiju&quot;,&quot;last_name&quot;:&quot;Li&quot;},{&quot;first_name&quot;:&quot;Bin&quot;,&quot;last_name&quot;:&quot;Yu&quot;},{&quot;first_name&quot;:&quot;Yi-Chao&quot;,&quot;last_name&quot;:&quot;Zheng&quot;},{&quot;first_name&quot;:&quot;Shaomeng&quot;,&quot;last_name&quot;:&quot;Wang&quot;},{&quot;first_name&quot;:&quot;Hong-Min&quot;,&quot;last_name&quot;:&quot;Liu&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2019&quot;,&quot;month&quot;:&quot;02&quot;,&quot;day&quot;:&quot;25&quot;,&quot;issue&quot;:&quot;5&quot;,&quot;volume&quot;:&quot;62&quot;,&quot;pages&quot;:&quot;2772-2797&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.9b00113&quot;},&quot;abstract&quot;:&quot;The cullin-RING ubiquitin ligases (CRLs) are responsible for about 20% of cellular protein degradation and regulate diverse cellular processes, and the dysfunction of CRLs is implicated in human diseases. Targeting the CRLs has become an emerging strategy for the treatment of human diseases. Herein, we describe the discovery of a hit compound from our in-house library and further structure-based optimizations, which have enabled the identification of new triazolo[1,5-a]pyrimidine-based inhibitors targeting the DCN1–UBC12 interaction. Compound WS-383 blocks the DCN1–UBC12 interaction (IC50 = 11 nM) reversibly and shows selectivity over selected kinases. WS-383 exhibits cellular target engagement to DCN1 in MGC-803 cells. WS-383 inhibits Cul3/1 neddylation selectively over other cullins and also induces accumulation of p21, p27, and NRF2. Collectively, targeting the DCN1–UBC12 interaction would be a viable strategy for selective neddylation inhibition of Cul3/1 and may be of therapeutic potential for disease tr&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b00113&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b00113" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b00113&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b00113" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b00113&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b00113" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.9b00113&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b00113&amp;href=/doi/10.1021/acs.jmedchem.9b00113" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.9b00113" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.9b00113" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (12 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b00113&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.9b00113%26sid%3Dliteratum%253Aachs%26pmid%3D30803229%26genre%3Darticle%26aulast%3DWang%26date%3D2019%26atitle%3DDevelopment%2Bof%2BHighly%2BPotent%252C%2BSelective%252C%2Band%2BCellular%2BActive%2BTriazolo%255B1%252C5-a%255Dpyrimidine-Based%2BInhibitors%2BTargeting%2Bthe%2BDCN1%25E2%2580%2593UBC12%2BProtein%25E2%2580%2593Protein%2BInteraction%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D62%26issue%3D5%26spage%3D2772%26epage%3D2797%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (3)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=291241" title="Phenyls">Phenyls</a>,</li><li><a href="/action/doSearch?ConceptID=291120" title="Hydrophobicity">Hydrophobicity</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=291793" title="Inhibition">Inhibition</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/62/5" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/jmcmar.2019.62.issue-5/20190314/jmcmar.2019.62.issue-5.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.9b00113/20190308/images/medium/jm-2019-00113s_0023.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.9b00113/20190308/images/large/jm-2019-00113s_0023.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00113&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">The cullin-RING ubiquitin ligases (CRLs) are responsible for about 20% of cellular protein degradation and regulate diverse cellular processes, and the dysfunction of CRLs is implicated in human diseases. Targeting the CRLs has become an emerging strategy for the treatment of human diseases. Herein, we describe the discovery of a hit compound from our in-house library and further structure-based optimizations, which have enabled the identification of new triazolo[1,5-<i>a</i>]pyrimidine-based inhibitors targeting the DCN1–UBC12 interaction. Compound <b>WS-383</b> blocks the DCN1–UBC12 interaction (IC<sub>50</sub> = 11 nM) reversibly and shows selectivity over selected kinases. <b>WS-383</b> exhibits cellular target engagement to DCN1 in MGC-803 cells. <b>WS-383</b> inhibits Cul3/1 neddylation selectively over other cullins and also induces accumulation of p21, p27, and NRF2. Collectively, targeting the DCN1–UBC12 interaction would be a viable strategy for selective neddylation inhibition of Cul3/1 and may be of therapeutic potential for disease treatment in which Cul3/1 is dysregulated.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_08131" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_08131" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">The most prevalent class of ubiquitin ligases are cullin-RING ligases (CRLs), which degrade approximately 20% of cellular proteins through the ubiquitin–proteasome system<a onclick="showRef(event, 'ref1 ref2'); return false;" href="javascript:void(0);" class="ref ref1 ref2">(1,2)</a> and regulate diverse cellular processes (e.g., protein homeostasis). Dysfunction of CRLs has been observed in several human diseases including cancers.<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> The covalent neural precursor cell-expressed developmentally down-regulated protein 8 (NEDD8) modification of CRLs through the ATP-dependent enzymatic cascade has played essential roles in the CRL-mediated downstream substrates’ ubiquitination modifications (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>).<a onclick="showRef(event, 'ref4 ref5 ref6'); return false;" href="javascript:void(0);" class="ref ref4 ref5 ref6">(4−6)</a> To date, there are eight cullin members (Cul1–3, Cul4A/B, and Cul5/7/9) in mammalian cells, each of which serves as the central component of each CRL and specifically modulates substrate degradation.<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> Therefore, pharmacological inactivation of CRLs caused by cullin neddylation inhibition has become novel therapeutics for disease treatment.<a onclick="showRef(event, 'ref8 ref9 ref10'); return false;" href="javascript:void(0);" class="ref ref8 ref9 ref10">(8−10)</a> Among the reported NEDD8-activating enzyme (NAE) inhibitors,<a onclick="showRef(event, 'ref11 ref12 ref13 ref14 ref15 ref16 ref17'); return false;" href="javascript:void(0);" class="ref ref11 ref12 ref13 ref14 ref15 ref16 ref17">(11−17)</a> pevonedistat (MLN4924) is the most well characterized NAE inhibitor and is presently undergoing 25 clinical trials for the treatment of human diseases including cancer and myelofibrosis.<a onclick="showRef(event, 'ref18 ref19'); return false;" href="javascript:void(0);" class="ref ref18 ref19">(18,19)</a> Mechanistically, pevonedistat causes NAE inhibition effectively by covalently binding to NEDD8 and complete activation inhibition of all CRLs, thereby leading to accumulation of CRL substrates.<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a></div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.9b00113/20190308/images/medium/jm-2019-00113s_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.9b00113/20190308/images/large/jm-2019-00113s_0001.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Enzymatic cascades for protein neddylation.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.9b00113/20190308/images/large/jm-2019-00113s_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00113&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">In the human genome, there are five defective in cullin neddylation (DCN) isoforms (DCN1–5).<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> DCN1 is the most commonly dysregulated in many squamous cell carcinomas (SCCs)<a onclick="showRef(event, 'ref21 ref22'); return false;" href="javascript:void(0);" class="ref ref21 ref22">(21,22)</a> and is involved in the regulation of the neddylation pathway and tumorigenesis.<a onclick="showRef(event, 'ref23 ref24'); return false;" href="javascript:void(0);" class="ref ref23 ref24">(23,24)</a> DCN1 overexpression is negatively correlated with cause-specific survival and associated with anchorage-independent growth in soft agar.<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> Structurally, DCN1 interacts with both cullin 1 and UBC12 through two separate domains, the WHB domain of cullin 1 and the acetylated N terminus of UBC12, thereby facilitating cullin neddylation.<a onclick="showRef(event, 'ref26 ref27 ref28 ref29'); return false;" href="javascript:void(0);" class="ref ref26 ref27 ref28 ref29">(26−29)</a> Intriguingly, the interaction between DCN1 and UBC12 regulates assembly of a multiprotein complex; the interruption of this interaction can cause defective cullin neddylation.<a onclick="showRef(event, 'ref25 ref30'); return false;" href="javascript:void(0);" class="ref ref25 ref30">(25,30)</a> Computational analysis of the X-ray cocrystal structure of DCN1 in complex with UBC12 indicates that five hydrophobic hotspots exist at the UBC12 peptide binding site and could be used for the development of DCN1 inhibitors.<a onclick="showRef(event, 'ref30 ref31 ref32 ref33'); return false;" href="javascript:void(0);" class="ref ref30 ref31 ref32 ref33">(30−33)</a> Guy et al. reported that the piperidinyl urea-based DCN1 inhibitors <b>NAcM-OPT</b> and <b>NAcM-COV</b> (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>) effectively blocked the DCN1–UBC12 interaction (IC<sub>50</sub> = 60 nM), selectively inhibited Cul1/3 neddylation in HCC95 cells overexpressing DCN1, and suppressed anchorage-independent growth of HCC95 cells.<a onclick="showRef(event, 'ref25 ref31 ref34'); return false;" href="javascript:void(0);" class="ref ref25 ref31 ref34">(25,31,34)</a> On the basis of the 12-residue UBC12 peptide, Wang et al. recently synthesized a library of peptidomimetic inhibitors that bound to DCN1 protein with high affinity (<i>K</i><sub>D</sub> < 10 nM), of which <b>DI-404</b> (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>) effectively blocked the DCN1–UBC12 interaction, inhibited neddylation of Cul3 selectively over other cullins in both H2170 and SK-MES-1 cells, and increased expression of NRF2 protein, a cullin 3 CRL (CRL3) substrate.<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> The Wang group also developed <b>DI-591</b> (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>) that bound to human DCN1 tightly (<i>K</i><sub>i</sub> = 12 nM), blocked cellular DCN1–UBC12 interaction in KYSE70 cells, and inhibited neddylation of Cul3 selectively over other cullins in different cells.<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> These findings have demonstrated that blocking the DCN1–UBC12 interaction may be a viable strategy for the treatment of Cul3 dysregulated diseases.<a onclick="showRef(event, 'ref36 ref37'); return false;" href="javascript:void(0);" class="ref ref36 ref37">(36,37)</a> In the present work, we report the discovery of the hit compound <b>E1</b> (also named as <b>WS-291</b>, IC<sub>50</sub> = 5.82 μM) and further structure-based optimizations, leading to the generation of triazolo[1,5-<i>a</i>]pyrimidine-based DCN1 inhibitors. The shortlisted lead compound <b>WS-383</b> effectively blocked interaction between DCN1 and UBC12 (IC<sub>50</sub> = 11 nM), caused selective Cul3/1 neddylation inhibition over other cullins, and induced accumulation of p21, p27, and NRF2. Our studies demonstrate that the interaction blockage between DCN1 and UBC12 could be a feasible strategy for the treatment of Cul3/1 dysregulated diseases.</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.9b00113/20190308/images/medium/jm-2019-00113s_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.9b00113/20190308/images/large/jm-2019-00113s_0002.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Representative small-molecule DCN1 inhibitors.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.9b00113/20190308/images/large/jm-2019-00113s_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00113&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_68223" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_68223" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> Structure-Based Inhibitor Design Based on the Identified Hit Compound <b>E1</b></h3><div class="NLM_p">In order to identify hit compounds targeting the DCN1–UBC12 protein–protein interaction, we developed the HTRF (homogeneous time resolved fluorescence) assay based on the interaction between DCN1 PONY domain and acetylated UBC12 N-terminal (<i>Z</i>′ factor na0.7, <a href="/doi/suppl/10.1021/acs.jmedchem.9b00113/suppl_file/jm9b00113_si_003.pdf" class="ext-link">Figure S1</a>). The Eu<sup>3+</sup> cryptate conjugated glutathione <i>S</i>-transferase antibody (donor beads) binds the GST-DCN1 and d2-conjugated streptavidin (acceptor beads) to recognize N-terminally acetylated UBC12. <b>DI-591</b>, an active control compound used in this work, strongly inhibited the DCN1–UBC12 interaction (IC<sub>50</sub> = 19 nM), which was comparable to that reported by Wang et al.,<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> indicating that the assay developed was reliable for biochemical screening against the DCN1–UBC12 protein–protein interaction.</div><div class="NLM_p">Our compound collection was composed of ∼15 000 compounds with around 5000 distinct scaffolds, of which 10 000 compounds were purchased from ChemDiv and 5000 compounds synthesized in our lab. After screening our small-molecule library based on the above established method, the compounds with the inhibition rates over 50% at 50 μM were selected as hit compounds for further optimization. To our delight, four promising hit compounds were identified (the SMILES strings for some compounds and their inhibition rates are provided in the <a href="/doi/suppl/10.1021/acs.jmedchem.9b00113/suppl_file/jm9b00113_si_002.csv" class="ext-link">Supporting Information</a>), of which compound <b>E1</b> (also named as <b>WS-291</b>) inhibited DCN1 with an IC<sub>50</sub> value of 5.82 μM. Given its acceptable potency, compound <b>E1</b> was used as the starting point to perform further structural optimizations. Initially, we predicted the binding model of <b>E1</b> within the UBC12 binding site using the recently reported <b>DI-591</b>/DCN1 complex (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5UFI">5UFI</a>) as the docking receptor.<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> As shown in <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>A, <b>E1</b> (colored in magenta) occupied the hydrophobic pocket, partially overlapped with <b>DI-591</b> (colored in cyan) within the UBC12 binding site of DCN1 (colored in green). Clearly, the phenyl ring in <b>E1</b> occupied the same region with <b>DI-591</b>. The methyl group attached to the tetrazole moiety was directed toward a new subpocket formed by I83, I86, and P97, while another hydrophobic pocket occupied by the cyclohexyl group in <b>DI-591</b> was adjacent to the triazole ring. Installation of substituents at this site may increase the binding affinity to DCN1 by occupying this hydrophobic pocket. As shown in <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>B and <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>C, hydrophobic effects of <b>E1</b> and <b>DI-591</b> drove association at the sites defined by F117, I86, F89, F109, I105, V102, and F164 on DCN1. An H-bond interaction was formed with Y181. Differently, <b>DI-591</b> formed additional H-bond interactions with C115 and Q114, while the structurally rigid <b>E1</b> had more hydrophobic interactions by forming π–π stacking with F117 and F164 residues (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>D). On the basis of the structural analysis, we speculate that introduction of substituents to the tetrazole and triazole moieties may increase the binding affinity to DCN1 by occupying additional hydrophobic pockets while keeping the S-tethered aromatic rings intact.</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.9b00113/20190308/images/medium/jm-2019-00113s_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.9b00113/20190308/images/large/jm-2019-00113s_0003.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Structural basis of <b>E1</b>-DCN1 binding. (A) <b>E1</b> (magenta) and <b>DI-591</b> (cyan) occupied the same binding site in DCN1 (green). Protein (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5UFI">5UFI</a>) was shown as cartoon, and small molecules were shown as sticks. The surface (white) of the binding pocket was shown in 20% transparence. (B, C) Interaction models of <b>E1</b> and <b>DI-591</b> in DCN1, respectively. For clarity, DCN1 was shown in 80% transparent cartoon. DCN1 residues interacting with <b>E1</b> were shown as sticks. Hydrogen bonds were shown as dashed lines. (D) 2D-view of the interactions between <b>E1</b> and DCN1.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.9b00113/20190308/images/large/jm-2019-00113s_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00113&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5"> Variations of Substituents Attached to the Tetrazole Ring</h3><div class="NLM_p">As shown in <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>A–C, the methyl group in <b>E1</b> was directed to the hydrophobic pocket occupied by the <i>N</i>-propionyl group of <b>DI-591</b>, and replacement of the methyl group with larger groups may better occupy this hydrophobic pocket, thereby increasing binding affinity. As shown in <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>, <b>E2</b> bearing a phenyl group inhibited the DCN1–UBC12 interaction with an IC<sub>50</sub> value of 15.81 μM, about 2.7-fold less potent than <b>E1</b>, while <b>E3</b> possessing a <i>p</i>-hydroxyphenyl group showed about 4.1-fold increase in potency (IC<sub>50</sub> = 3.88 μM) compared to <b>E2</b>. To our surprise, substitution of the methyl group in <b>E1</b> with the <i>N</i>,<i>N</i>-dimethylethyl group gave compound <b>E4</b>, which exhibited significantly increased biochemical potency against DCN1 with an IC<sub>50</sub> value of 130 nM, around 45-fold more potent than <b>E1</b>, indicating the importance of the <i>N</i>,<i>N</i>-dimethylaminoethyl group for the activity.</div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.9b00113/20190308/images/medium/jm-2019-00113s_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.9b00113/20190308/images/large/jm-2019-00113s_0004.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Preliminary SARs studies based on the starting point <b>E1</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.9b00113/20190308/images/large/jm-2019-00113s_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00113&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i6" class="anchor-spacer"></div><h3 class="article-section__title" id="_i6"> Scaffold Replacement of the Core Structure</h3><div class="NLM_p">With compound <b>E4</b> in hand, we next replaced the [1,2,4]triazolo[1,5-<i>a</i>]pyrimidine with several heterocyclic rings such as the pyrazolo[1,5-<i>a</i>]pyrimidine, thieno[3,2-<i>d</i>]pyrimidine, and quinazoline, yielding compounds <b>E5</b>, <b>E6</b>, and <b>E7</b>, which showed significantly reduced potency, compared to <b>E4</b> (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>). The results underscore the importance of the [1,2,4]triazolo[1,5-<i>a</i>]pyrimidine for the activity. However, it should be noted that the phenyl group in <b>E4</b> may also be crucial for the interaction. The lack of the phenyl group in compounds <b>E5</b>, <b>E6</b>, and <b>E7</b> may lead to changes of their binding models, thus causing the decreased activity.</div><figure id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.9b00113/20190308/images/medium/jm-2019-00113s_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.9b00113/20190308/images/large/jm-2019-00113s_0005.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Scaffold replacement.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.9b00113/20190308/images/large/jm-2019-00113s_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00113&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i7" class="anchor-spacer"></div><h3 class="article-section__title" id="_i7"> Variations of Substituents (R<sup>1</sup>) Attached to the Triazolopyridine Ring</h3><div class="NLM_p">Inspired by the favorable potency of <b>E4</b>, we next replaced the phenyl ring in <b>E4</b> with several aliphatic groups to further explore the structure–activity relationships (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). Interestingly, replacement of the carbon (C) atom in <b>E5</b> with the nitrogen (N) atom yielded <b>E8</b>, which effectively blocked the DCN1–UBC12 interaction with an IC<sub>50</sub> value of 51 nM, being around 83- and 2.5-fold much more potent than <b>E5</b> and <b>E4</b>. The data may suggest that the methyl group was preferred over the phenyl group for the activity. The replacement of the N atom in the bicyclic N-heterocycle with the C atom caused dramatic decrease of the activity possibly due to the loss of potential electrostatic interaction in <b>E5</b>. Compounds <b>E9</b> and <b>E10</b> showed reduced potency with the IC<sub>50</sub> values of 706 and 1360 nM, respectively. Compound <b>E11</b> bearing a cyclopropyl ring exhibited comparable activity (IC<sub>50</sub> = 127 nM) with <b>E4</b>. Interestingly, <b>E8</b> and <b>E12</b> showed favorable potency with the IC<sub>50</sub> value of 51 and 61 nM, respectively. To conclude, the activity of <b>E8</b>, <b>E9</b>, and <b>E10</b> decreased correspondingly with the increase of the steric hindrance of R<sup>1</sup>, while <b>E4</b> and <b>E11</b> bearing more steric R<sup>1</sup> groups (e.g., Ph, cyclopropyl) also exhibited acceptable potency, much more potent than <b>E9</b> and <b>E10</b>. The rationale for the activity discrepancy may be due to that <b>E4</b> and <b>E11</b> adopted distinct binding models in the UBC12 binding site of DCN1.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Structures and Biochemical Potency of Compounds <b>E4</b> and <b>E8</b>–<b>E12</b></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.9b00113/20190308/images/medium/jm-2019-00113s_0017.gif" alt="" id="GRAPHIC-d7e1002-autogenerated" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.9b00113/20190308/images/medium/jm-2019-00113s_0018.gif" alt="" id="gr6" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup><sup>a</sup></sup><p class="last">Data are the mean ± SD. All experiments were carried out at least three times.</p></div></div><div></div></div></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i9" class="anchor-spacer"></div><h3 class="article-section__title" id="_i9"> Variations of Substituents (R<sup>1</sup> and R<sup>2</sup>) Attached to the Triazolopyridine Ring</h3><div class="NLM_p">As shown in <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>A, the hydrophobic pocket occupied by the cyclohexyl group in <b>DI-591</b> was adjacent to the triazole ring. We therefore speculated that installation of substituents to the triazole ring may occupy this hydrophobic pocket, thus increasing binding affinity to DCN1. Therefore, further SARs studies were performed based on the compound <b>E8</b> by adding additional hydrophobic groups (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). Generally, in comparison with compounds <b>E8</b> and <b>E12</b>, most of the compounds in <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a> showed decreased but acceptable potency against DCN1. Only <b>E15</b>, <b>E17</b>, <b>E18</b>, <b>E20</b>, and <b>E27</b> inhibited the DCN1–UBC12 interaction with the IC<sub>50</sub> values less than 100 nM. Particularly, compound <b>E20</b> effectively inhibited DCN1 with an IC<sub>50</sub> value of 22 nM, while replacement of the methyl group in <b>E20</b> with the phenyl group afforded compound <b>E25</b>, which exerted significantly decreased activity toward DCN1. Unexpectedly, <b>E26</b> bearing an ethyl group (R<sup>1</sup>) showed 26-fold decrease of the activity compared to <b>E20</b>. One possible explanation for the activity discrepancy was that the R<sup>1</sup> group was directed toward a narrow subpocket in DCN1. A slight increase of the size of the R<sup>1</sup> group may cause steric hindrance and finally led to decrease of the activity. A similar trend was also observed in compounds <b>E8</b>–<b>E12</b> (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>).</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Structures and Biochemical Potency of Compounds <b>E13</b>–<b>E27</b></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.9b00113/20190308/images/medium/jm-2019-00113s_0019.gif" alt="" id="GRAPHIC-d7e1115-autogenerated" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.9b00113/20190308/images/medium/jm-2019-00113s_0020.gif" alt="" id="gr7" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup><sup>a</sup></sup><p class="last">Data are the mean ± SD. All experiments were carried out at least three times.</p></div></div><div></div></div></div><div id="sec2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i11" class="anchor-spacer"></div><h3 class="article-section__title" id="_i11"> Variations of Substituents (R<sup>2</sup> and R<sup>3</sup>) Attached to the Triazolopyridine Ring</h3><div class="NLM_p">Encouraged by the biochemical potency of compound <b>E20</b>, we next mainly focused on the molecular fine-tuning of substituents attached to the phenyl ring of <b>E20</b>, leading to the generation of compounds <b>E28</b>–<b>E44</b>, and the data were summarized in <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>. As shown in <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>, substituents attached to the phenyl ring had remarkable effect on the activity. Generally, compounds containing 2- and 3-substituted phenyl ring had significantly reduced activity compared to those with 4-substituted phenyl groups. Compounds <b>E33</b> (2-Br), <b>E34</b> (3-Br), <b>E37</b> (2-CF<sub>3</sub>), <b>E38</b> (3-CF<sub>3</sub>), and <b>E42</b> (2,6-diCl) possessed relatively weak inhibitory activity with the IC<sub>50</sub> values over 190 nM. One exception is <b>E29</b> bearing a nitro group at the 2-position of the phenyl ring, which inhibited DCN1 potently (IC<sub>50</sub> = 24 nM), comparable to those with <i>para</i>-substitutions on the phenyl ring. The rationale for the decreased potency of the compounds containing 2- and 3-substituted phenyl rings was that the phenyl group was inserted into a narrow subpocket, and installation of substitutions at the 2- or 3-position of the phenyl group may cause steric hindrance, thus affecting their binding to DCN1 (as shown in <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>A). In contrast, the substitutions attached to the 4-position of the phenyl group were well tolerated. Compared to <b>E20</b>, some of these compounds showed comparable or marginally increased potency against DCN1 with the IC<sub>50</sub> values less than 20 nM, of which <b>E31</b> (also named as <b>WS-383</b>) showed the best potency against DCN1 with an IC<sub>50</sub> value of 11 nM. Apart from the variations of the bicyclic N-heterocycle ring (as shown in <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>), further SARs studies presented in <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a> indicated that introduction of methyl or <i>n</i>-pentyl group (R<sup>3</sup>) to the [1,2,4]triazolo[1,5-<i>a</i>]pyrimidine ring was also detrimental to the activity. <b>E44</b> (also known as <b>WS-524</b>) was found to be inactive toward the DCN1–UBC12 interaction. <b>E43</b> also showed significantly decreased activity (IC<sub>50</sub> = 3363 nM), about 306-fold less potent than <b>WS-383</b>. The data presented in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Tables <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>–<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a> highlighted the importance of the [1,2,4]triazolo[1,5-<i>a</i>]pyrimidine scaffold for the activity and also revealed essential structural elements for blocking the DCN1–UBC12 interaction.</div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. Structures and Biochemical Potency of Compounds <b>E28</b>–<b>E44</b></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.9b00113/20190308/images/medium/jm-2019-00113s_0021.gif" alt="" id="GRAPHIC-d7e1264-autogenerated" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.9b00113/20190308/images/medium/jm-2019-00113s_0022.gif" alt="" id="gr8" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t3fn1"><div class="footnote" id="t3fn1"><sup><sup>a</sup></sup><p class="last">Data are the mean ± SD. All experiments were carried out at least three times.</p></div></div><div></div></div></div><div id="sec2_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13"> Molecular Modeling Studies</h3><div class="NLM_p">To further explore the possible binding mode of compounds <b>WS-383</b> and <b>WS-524</b> with DCN1 and explain their activity discrepancy, a molecular modeling study was performed using molecular docking and molecular dynamics (MD) simulations. The crystal structure of <b>DI-591</b> bound to DCN1 (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5UFI">5UFI</a>) was selected as the docking receptor.<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a><b>DI-591</b> was first redocked into the protein to investigate the reliability of our docking simulation using the Dock module of MOE 2015.10. The RMSD value between crystal structure and docked structure of <b>DI-591</b> was 1.3 Å, suggesting that our docking study had a convincing accuracy (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00113/suppl_file/jm9b00113_si_003.pdf" class="ext-link">Figure S2</a>). Compounds <b>WS-383</b> and <b>WS-524</b> were then docked into the hydrophobic binding pocket of DCN1. The internal reorganization of binding was considered in <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>. To consider internal reorganization of binding, we docked compound <b>WS-383</b> with the released crystal structures of 6B5Q, 5V83, 5V86, 6BG3, 6BG5, which were complexes of DCN1 bound with ligand. As shown in <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>, the binding modes of these systems were consistent with the previous reports from the Wang group,<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> which suggested that the reorganization of F117 and F109 was not necessary for the combination of our compounds and DCN1. And the other portion of DCN1 had a stronger interaction with compounds than the conformational changes at Phe109 and Phe117.</div><figure id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.9b00113/20190308/images/medium/jm-2019-00113s_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.9b00113/20190308/images/large/jm-2019-00113s_0006.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Docking results of <b>WS</b>-<b>383</b> with DCN1. (A) Predicted binding model of compound <b>WS</b>-<b>383</b> (magenta) in complex with 6B5Q. (B) Predicted binding model of compound <b>WS</b>-<b>383</b> (orange) in complex with 5V83. (C) Predicted binding model of compound <b>WS</b>-<b>383</b> (cyan) in complex with 5V86. (D) Predicted binding model of compound <b>WS</b>-<b>383</b> (dark pink) in complex with 6BG3. (E) Predicted binding model of compound <b>WS</b>-<b>383</b> (green) in complex with 6BG5. The associated residues were shown in white sticks.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.9b00113/20190308/images/large/jm-2019-00113s_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00113&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">In this study, the docking simulation was performed with rigid (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>) and induced fit (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>) strategies, respectively. The rigid model was chosen to analyze the binding mode. The lowest-energy docking conformations of two compounds were basically the same except that compound <b>WS-524</b> had an extra pentyl group exposed to the solvent region (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>). The 4-Cl phenyl group was deeply inserted into the binding pocket and was observed to interact at the hydrophobic site defined by hydrophobic interactions with Ile86, Phe89, Val102, Ile105, Ala106, Phe109, Ala111, Phe117, and Phe164. Among these residues, 4-Cl phenyl group also had a π–π stacking with Phe89, Phe109, Phe117, and Phe164. The 5-methylpyrimidine group was observed to interact at the hydrophobic site defined by hydrophobic interactions with Ala180, Tyr181, and Leu184. Meanwhile, Ala98 had an arene-H electrostatic interaction with the pyrimidine ring. The tetrazole ring had an arene-H electrostatic interaction with Pro97. Furthermore, the <i>N</i>,<i>N</i>-dimethyl group was observed to interact at the hydrophobic site defined by hydrophobic interactions with Ile83 and Ile86. These interactions may be responsible for the observed potency of <b>WS-383</b>.</div><figure id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.9b00113/20190308/images/medium/jm-2019-00113s_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.9b00113/20190308/images/large/jm-2019-00113s_0007.jpeg" id="gr10" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Docking results of <b>WS-383</b> and <b>WS-524</b>. (A) Superimposition of <b>WS-383</b> (cyan) and <b>WS-524</b> (orange) with DCN1. Protein (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5UFI">5UFI</a>) was shown in surface, and hydrophobic region was shown in green. (B) Predicted binding model of compound <b>WS-383</b> (cyan) in complex with DCN1. (C) Two dimensional (2D) representation of compound <b>WS-383</b> in complex with DCN1; (D) Detailed views of <b>WS-383</b> and interacting residues in the P1–P3 pockets.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.9b00113/20190308/images/large/jm-2019-00113s_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00113&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig8" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.9b00113/20190308/images/medium/jm-2019-00113s_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.9b00113/20190308/images/large/jm-2019-00113s_0008.jpeg" id="gr11" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Docking results of <b>WS</b>-<b>383</b> and <b>WS</b>-<b>524</b> with <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5UFI">5UFI</a>. (A) Superimposition of <b>WS</b>-<b>383</b> (cyan) and <b>WS</b>-<b>524</b> (orange) with DCN1. Protein was shown in surface, and hydrophobic region was shown in green. Compounds <b>WS</b>-<b>383</b> and <b>WS-524</b> were shown in dark cyan and dark pink sticks, respectively. (B) Predicted binding model of compound <b>WS</b>-<b>383</b> (dark cyan) in complex with DCN1, the associated residues were shown in white sticks.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.9b00113/20190308/images/large/jm-2019-00113s_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00113&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">To investigate the binding mode of compounds <b>WS</b>-<b>383</b> and <b>WS</b>-<b>524</b> with <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5UFI">5UFI</a>, we considered the induced fit strategy on docking in this study. Docking results were shown in <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>. As for the docking results, there was little difference between rigid model and induced fit model as the binding modes and the related interactions of two systems were consistent.</div><div class="NLM_p">As shown in <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>, the increase of the hydrophobic properties of R<sup>2</sup> group decreased the potency for <b>E13</b>–<b>E24</b> (except <b>E20</b>), suggesting the importance of the steric effect of the group R<sup>2</sup> in subpocket P1. Too bulky substituent at subpocket P1 could cause a steric incompatibility and lead to decreased activity. However, compounds <b>E15</b>, <b>E17</b>, <b>E18</b>, and <b>E20</b> showed comparable inhibitory activity with <b>E8</b> (IC<sub>50</sub> = 51 nM) due to the π–π stacking. <b>E20</b> effectively inhibited DCN1 with an IC<sub>50</sub> value of 22 nM possibly because of the π–π stacking with surrounding residues. To conclude, the π–π stacking was important for the activity, and there would be a balance between the steric effect and π–π stacking effect of group R<sup>2</sup> in subpocket P1. This phenomenon was further observed in compounds listed in <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a> (<b>E28</b>–<b>E42</b>). The potency decreased obviously with the increase of steric hindrance in <b>E37</b>, <b>E38</b>, <b>E36</b>. The activity loss of <b>WS-524</b> may be due to the hydrophobic <i>n</i>-pentyl group being exposed to the solvent region, which caused the complex to be unstable and then led to the loss of the activity.</div><div class="NLM_p">Molecular dynamic simulations were then implemented to predict the interactions of <b>WS-383</b>, <b>WS-524</b>, and <b>DI-591</b> in complex with DCN1 (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5UFI">5UFI</a>) in a constant process using Gromacs 5.1.4. The Amber99SB-ildn force field and the gaff force field was used for the protein and ligand, respectively. Each protein–ligand complex was subjected to a 20 ns MD simulation. As shown in <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a></a>A, the 5UFI-<b>DI-591</b> and 5UFI-<b>WS-383</b> complexes stabilized quickly, and the RMSD value was about 1 and 1.5 Å, respectively, while the RMSD value of 5UFI-<b>WS-524</b> complex was stable at around 3.8 Å after a longer time. This result indicated that 5UFI-<b>WS-383</b> was more stable during the MD progress, and <b>WS-383</b> bound to DCN1 more tightly than <b>WS-524.</b> The interactions between <b>WS-383</b> and DCN1 after MD simulations were not significantly different from that of the start structure with DCN1 (<a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a></a>B). However, <b>WS-524</b> could not bind well with DCN1 and would get out of the binding pocket after repeated 20 ns MD (5 times). The exposed methyl (E43) or pentyl (<b>WS-524</b>) group did not form any interactions with DCN1. One possible reason was that the steric incompatibility between <b>WS-524</b> and the subpocket P1 pushed the ligand out of the pocket. At the same time, the change in dihedral angle was also observed in <b>WS-383</b>. The tetrazole ring of <b>WS-383</b> had a flip to get a more stable conformation as shown in <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a></a>B and moved out from a hydrophobic region (see <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>A), while the conformation of <b>WS-524</b> did not have this change. Therefore, another possible reason was that the methyl (<b>E43</b>) and pentyl (<b>WS-524</b>) groups were constrained to this flip and made the hydrophilic tetrazole ring stay at an unfavorable region for binding. Also, the exposed methyl (E43) and pentyl (<b>WS-524</b>) groups in the solvent region made the combination unstable. The steric incompatibility between <b>WS-524</b> and the subpocket P1 may also push the ligand out of the pocket. No flip in <b>WS-524</b> was observed in this case. The steric incompatibility only or together with the flip made <b>WS-524</b> move out from the binding pocket after MD simulations (<a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a></a>C), thus causing a decreased potency. Interactions between <b>WS-524</b> and surrounding residues after the 20 ns MD simulations were shown in <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a></a>D.</div><figure id="fig9" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.9b00113/20190308/images/medium/jm-2019-00113s_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.9b00113/20190308/images/large/jm-2019-00113s_0009.jpeg" id="gr12" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. (A) RMSD plot of the studied complexes during MD simulations. (B) Superimposition of the start structure (cyan) and that (yellow) after MD of compound <b>WS-383</b> with DCN1. Protein (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5UFI">5UFI</a>) was shown in white surface. (C) Superimposition of the start structure (orange) and that (pink) after MD of compound <b>WS-524</b> with DCN1. Protein was shown in white surface. (D) Interactions between compound <b>WS-524</b> and surrounding residues after the 20 ns MD simulations.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.9b00113/20190308/images/large/jm-2019-00113s_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00113&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The MM-PBSA method was utilized to calculate the binding free energy using ref <a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a>. The Δ<i>E</i><sub>binding</sub> values of compounds <b>DI-591</b>, <b>WS-383</b>, and <b>WS-524</b> with DCN1 were −266.143 kJ/mol, −278.381 kJ/mol, and −64.658 kJ/mol, respectively, suggesting that <b>WS-383</b> had a similar binding affinity with <b>DI-591</b> and a better bioactivity than <b>WS-524</b> (<a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>). The Δ<i>E</i><sub>elec</sub> plays a crucial role in the binding affinity, and the Δ<i>E</i><sub>vdw</sub> also has an important influence on the activity. Because of the poor contact with protein, <b>WS-524</b> exhibited a low Δ<i>E</i><sub>binding</sub>. The decreased potency of <b>E43</b> and <b>WS-524</b> implied that the exposed nonpolar group was unfavorable for the bioactivity and the small change in dihedral angle might be important for the activity. This conclusion could help us in the further drug design and optimization.</div><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. Binding Free Energy of Two Systems<a class="ref internalNav" href="#tbl4-fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center" char=".">Δ<i>E</i><sub>vdw</sub></th><th class="colsep0 rowsep0" align="center" char=".">Δ<i>E</i><sub>elec</sub></th><th class="colsep0 rowsep0" align="center" char=".">Δ<i>E</i><sub>polar</sub></th><th class="colsep0 rowsep0" align="center" char=".">Δ<i>E</i><sub>nonpolar</sub></th><th class="colsep0 rowsep0" align="center" char=".">Δ<i>E</i><sub>binding</sub></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">5UFI-<b>DI-591</b></td><td class="colsep0 rowsep0" align="char" char=".">–251.756</td><td class="colsep0 rowsep0" align="char" char=".">–259.791</td><td class="colsep0 rowsep0" align="char" char=".">256.759</td><td class="colsep0 rowsep0" align="char" char=".">–11.355</td><td class="colsep0 rowsep0" align="char" char=".">–266.143</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">5UFI-<b>WS-383</b></td><td class="colsep0 rowsep0" align="char" char=".">–182.696</td><td class="colsep0 rowsep0" align="char" char=".">–206.778</td><td class="colsep0 rowsep0" align="char" char=".">129.767</td><td class="colsep0 rowsep0" align="char" char=".">–18.764</td><td class="colsep0 rowsep0" align="char" char=".">–278.381</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">5UFI-<b>WS-524</b></td><td class="colsep0 rowsep0" align="char" char=".">–50.001</td><td class="colsep0 rowsep0" align="char" char=".">–18.486</td><td class="colsep0 rowsep0" align="char" char=".">3.983</td><td class="colsep0 rowsep0" align="char" char=".">–0.083</td><td class="colsep0 rowsep0" align="char" char=".">–64.658</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-tbl4-fn1"><div class="footnote" id="tbl4-fn1"><sup><sup>a</sup></sup><p class="last">All energies are in kJ/mol.</p></div></div></div></div><div id="sec2_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i15" class="anchor-spacer"></div><h3 class="article-section__title" id="_i15"> <b>WS-383</b> Inhibits DCN1–UBC12 Interaction Selectively and Reversibly</h3><div class="NLM_p"><b>WS-383</b> (<a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">10</a></a>A), the most potent compound, inactivated the DCN1–UBC12 interaction with an IC<sub>50</sub> of 11 nM (<a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">10</a></a>C). Besides, the structurally similar compound <b>WS-524</b> (<a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">10</a></a>B) was found to be devoid of the inhibitory activity against the DCN1–UBC12 interaction (<a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">10</a></a>C) and therefore used as the negatively inactive control. As some of the small-molecule kinase inhibitors (e.g., osimertinib) approved by FDA structurally feature the heteroatom-tethered aromatic system,<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> our shortlisted compound <b>WS-383</b> also possessed the similar structural features, namely, the S-tethered tetrazole and [1,2,4]triazolo[1,5-<i>a</i>]pyrimidine ring. We therefore examined the inhibitory activity of <b>WS-383</b> against a panel of kinases such as BTK, CDKs, and EGFR [L858R] using staurosporine and afatinib as the positive controls. As shown in <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">10</a></a>D, <b>WS-383</b> showed weak inhibitory activity at 10.0 μM, highlighting its selectivity to the DCN1–UBC12 interaction over the selected kinases. The IC<sub>50</sub> values of staurosporine against BTK, CDK1, CDK2, CDK4, CDK6, CDK7, and CDK9 were 100, 2.4, 1.0, 27, 133, 72, and 17 nM, respectively. The IC<sub>50</sub> value of afatinib against EGFR [L858R] was 0.85 nM.</div><figure id="fig10" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 10</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.9b00113/20190308/images/medium/jm-2019-00113s_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.9b00113/20190308/images/large/jm-2019-00113s_0010.jpeg" id="gr13" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 10. <b>WS-383</b> inhibited the DCN1–UBC12 interaction selectively and reversibly. (A) Structure of <b>WS-383</b>. (B) Structure of <b>WS-524</b>. (C) Inactivation curve of <b>WS-383</b>, <b>WS-524</b>, and <b>DI-591</b> against the DCN1–UBC12 interaction performed by the HTRF assay. (D) Inhibitory activity of <b>WS-383</b> (10 μM) against a panel of kinases. (E, F) The reversibility of <b>WS-383</b> to DCN1 was determined by the HTRF based dilution assay with a starting concentration at 500 nM following 80-fold dilution (E) and ultrafiltration experiment starting concentration at 500 nM following six round centrifuge. (F) <b>DI-591</b> was used as a control with a starting concentration at 800 nM. Data are the mean ± SD. <i>P</i> < 0.01 was considered statistically significant. All experiments were carried out at least three times.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.9b00113/20190308/images/large/jm-2019-00113s_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00113&amp;id=fig10"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">It has been reported that the thiol group in glutathione can react with 3-benzyl-7-(2-benzoxazolyl)thio-1,2,3-triazolo[4,5-<i>d</i>]pyrimidine (<b>VAS2870</b>), thereby causing off-target effects through covalent modifications.<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> The first covalent DCN1 inhibitor <b>NAcM-COV</b> (TR-FRET IC<sub>50</sub> = 28 nM) has been rationally designed through covalent linkage to the targeted Cys115 in the binding site of UBC12.<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> In this work, a dilution assay was used to test the reversibility of <b>WS-383</b> for DCN1. <b>DI-591</b> was used as a control. The results revealed that 80-fold dilution of the DCN1/<b>WS-383</b> and DCN1/<b>DI-591</b> mixture resulted in the recovery of the activity, suggesting the reversible inhibition of <b>WS-383</b> and <b>DI-591</b> toward DCN1 (<a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">10</a></a>E). Ultrafiltration experiment was also performed against assay buffer after incubation of DCN1 with high concentration of <b>WS-383</b> (500 nM). After the ultracentrifuge, the activity can also be restored (<a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">10</a></a>F), which further confirmed the reversible binding of <b>WS-383</b> and <b>DI-591</b> to DCN1. Hence, all the data indicate that <b>WS-383</b> inhibited the DCN1–UBC12 interaction selectively and reversibly.</div></div><div id="sec2_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i16" class="anchor-spacer"></div><h3 class="article-section__title" id="_i16"> UBC12 and DCN1 Are Overexpressed in Several Gastric Cancer Cells</h3><div class="NLM_p">As an oncogene, DCN1 is highly expressed in several human tumors. It has been reported that DCN1 mRNA is overexpressed in 21 of 44 (48%) primary lung, 16 of 45 (36%) head and neck, and 4 of 9 (44%) cervical carcinomas,<a onclick="showRef(event, 'ref41 ref42 ref43'); return false;" href="javascript:void(0);" class="ref ref41 ref42 ref43">(41−43)</a> and the expression of DCN1 is significant higher in prostate cancer compared to adjacent normal tissue.<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> Nevertheless, the expression and neddylation of DCN1 have not been reported in gastric cancer cell yet. Hence, the protein expression and mRNA levels of UBC12 and DCN1 in four gastric cancer cell lines (MGC-803, SGC-7901, HGC-27, and BGC-823) and gastric epithelial cell line GES-1 were investigated. As shown in <a data-tab="pane-pcw-Figures" href="#fig11" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig11" id="" class=" internalNav">11</a></a>, relative to HGC-27 and GES-1, not only in protein level (<a data-tab="pane-pcw-Figures" href="#fig11" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig11" id="" class=" internalNav">11</a></a>A and <a data-tab="pane-pcw-Figures" href="#fig11" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig11" id="" class=" internalNav">11</a></a>B) but also in mRNA level (<a data-tab="pane-pcw-Figures" href="#fig11" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig11" id="" class=" internalNav">11</a></a>C), UBC12 was overexpressed in the poor differentiated gastric cancer cell lines MGC-803 and BGC-823 and moderate differentiated gastric cancer cell lines SGC-7901. Besides, there was no significant difference for the DCN1 expression in both protein (<a data-tab="pane-pcw-Figures" href="#fig11" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig11" id="" class=" internalNav">11</a></a>A) and mRNA levels (<a data-tab="pane-pcw-Figures" href="#fig11" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig11" id="" class=" internalNav">11</a></a>D) in these cell lines. However, relative to HGC-27, BGC-823, and GES-1 cell lines, DCN1 appeared to be overmodified in both MGC-803 and SGC-7901 cells (<a data-tab="pane-pcw-Figures" href="#fig11" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig11" id="" class=" internalNav">11</a></a>A, the upper band). However, it remains unclear how the DCN1 protein was modified in cells. The DCN1 protein could be possibly modified with NEDD8 or ubiquitin. To date, modification of DCN1 with ubiquitin has been reported,<a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a> and no NEDD8 modification on DCN1 has been reported. Hence, MGC-803 was chosen for our following cellular studies due to its overexpression of UBC12 and DCN1 neddylation.</div><figure id="fig11" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 11</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.9b00113/20190308/images/medium/jm-2019-00113s_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.9b00113/20190308/images/large/jm-2019-00113s_0011.jpeg" id="gr14" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 11. Protein and mRNA levels of UBC12 and DCN1 in the five gastric cell lines (MGC-803, SGC-7901, HGC-27, BGC-823, and GES-1). (A, B) Expression of UBC12, DCN1, and neddylated/ubiquitinated DCN1 (N8/Ub-DCN1) in the five indicated gastric cell lines. (C, D) Relative amounts of UBC12 (C) and DCN1 (D) mRNA in the five gastric cell lines were quantified with RT-qPCR. GAPDH was used as the loading control. Data are the mean ± SD. <i>P</i> < 0.01 was considered statistically significant. All experiments were carried out three times.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.9b00113/20190308/images/large/jm-2019-00113s_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00113&amp;id=fig11"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">We next employed the cellular thermal shift assay (CETSA) to assess the target engagement of <b>WS-383</b> in MGC-803 cells using the well characterized <b>DI-591</b> as the control. Clearly, the DCN1 protein in MGC-803 cells was denatured at 53 °C with DMSO treatment (<a data-tab="pane-pcw-Figures" href="#fig12" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig12" id="" class=" internalNav">12</a></a>), while both treatment with <b>WS-383</b> and <b>DI-591</b> for 1 h dose-dependently enhanced the cellular thermal stability of DCN1 (<a data-tab="pane-pcw-Figures" href="#fig12" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig12" id="" class=" internalNav">12</a></a>A and <a data-tab="pane-pcw-Figures" href="#fig12" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig12" id="" class=" internalNav">12</a></a>B). <b>WS-383</b> enhanced the thermal stability of cellular DCN1 protein even at 1.0 μM, indicating cellular target engagement of <b>WS-383</b> in MGC-803 cells.</div><figure id="fig12" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Figure 12</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.9b00113/20190308/images/medium/jm-2019-00113s_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.9b00113/20190308/images/large/jm-2019-00113s_0012.jpeg" id="gr15" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 12. Enhancement of thermal stability of DCN1 protein by <b>WS-383</b> and <b>DI-591</b> treatment in MGC-803 cells. (A, B) Protein levels of DCN1 in MGC-803 cells treated with <b>WS-383</b> (A) and <b>DI-591</b> (B) at the indicated concentrations for 1 h, followed by heating at 53 °C for 3 min were analyzed by Western blot. GAPDH was used as a loading control.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.9b00113/20190308/images/large/jm-2019-00113s_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00113&amp;id=fig12"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> <b>WS-383</b> Inactivates Cullin1 and Cullin3 Neddylation and Accumulates CRL Substrates p21, p27, NRF2 in Cells</h3><div class="NLM_p">MLN4924, the most well characterized NEDD8-activating enzyme (NAE) inhibitor, has been reported to inhibit neddylation of all cullins in many cancers.<a onclick="showRef(event, 'ref46 ref47 ref48'); return false;" href="javascript:void(0);" class="ref ref46 ref47 ref48">(46−48)</a> Before examination of the cellular activity of <b>WS-383</b>, the effect of MLN4924 on cullin neddylation was initially investigated in human gastric cell line MGC-803, of which UBC12 and DCN1 were overactivated. The results showed that treatment of MGC-803 cells for 24 h with MLN4924 at 1.0 μM caused broad neddylation inhibition of cullins (<a data-tab="pane-pcw-Figures" href="#fig13" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig13" id="" class=" internalNav">13</a></a>A). The cellular effect of <b>WS-383</b> on cullin neddylation in MGC-803 cells was also examined using <b>WS-524</b> as the negative control. After 24 h treatment, the neddylation status of cullin substrates was examined, showing that <b>WS-383</b> blocked Cul3 neddylation at 3 μM and also had certain inhibition of Cul1 neddylation at 10 μM but was not effective in inhibiting neddylation of other cullin members (<a data-tab="pane-pcw-Figures" href="#fig13" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig13" id="" class=" internalNav">13</a></a>A). On the other hand, <b>WS-524</b>, the negative control, failed to inhibit neddylation of all cullin substrates (<a data-tab="pane-pcw-Figures" href="#fig13" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig13" id="" class=" internalNav">13</a></a>A). To further confirm the neddylation inhibitory activity of <b>WS-383</b> in cellular levels, KYSE70, the same cell line used in previous publication,<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> was treated with <b>WS-383</b> for 24 h. As shown in <a data-tab="pane-pcw-Figures" href="#fig13" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig13" id="" class=" internalNav">13</a></a>B, in KYSE70 cells, <b>WS-383</b> blocked Cul3 neddylation dose dependently and also decreased Cul1 neddylation at 10 μM slightly, while no impact on neddylation inhibition of other cullin members was observed. To make a side-by-side comparison between <b>WS-383</b> and <b>DI-591</b>, the cellular deneddylation activity of <b>DI-591</b> on cullin neddylation in MGC-803 cells was also examined. As indicated in <a data-tab="pane-pcw-Figures" href="#fig13" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig13" id="" class=" internalNav">13</a></a>C, <b>DI-591</b> blocked Cul3 neddylation in a dose dependent manner but was not effective in inhibiting neddylation of other cullin members, which was consistent with the previous findings in KYSE70 and THLE2 cell lines.<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> Taken together, <b>WS-383</b> exerted selective neddylation inhibition of Cul1 and Cul3. The results were consistent with previous findings reported by Schulman et al.<a onclick="showRef(event, 'ref25 ref31 ref34'); return false;" href="javascript:void(0);" class="ref ref25 ref31 ref34">(25,31,34)</a> Although <b>WS-383</b> and <b>DI-591</b> bound to the same UBC12 binding site in DCN1, the different cellular effects of <b>WS-383</b> and <b>DI-591</b> on cullin neddylation may be due to their different binding models. <b>DI-591</b> occupied five subpockets in DCN1, while <b>WS-383</b> only occupied three of these subpockets (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>). The binding difference of both compounds may induce distinct conformational changes, thus affecting biological functions of DCN1.</div><figure id="fig13" data-index="13" class="article__inlineFigure"><h2 class="fig-label">Figure 13</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.9b00113/20190308/images/medium/jm-2019-00113s_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.9b00113/20190308/images/large/jm-2019-00113s_0013.jpeg" id="gr16" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 13. Cellular effects of <b>WS-383</b>, <b>DI-591</b>, and <b>MLN4924</b> on cullin neddylation and CRL substrates. (A) Deneddylation effects of <b>MLN4924</b>, <b>WS-383</b>, and <b>WS-524</b> on cullin1–5 at indicated concentrations for 24 h treatment in MGC-803 cells. (B) Deneddylation effects on cullin1–5 and CRL substrates accumulation in KYSE70 cells with 24 h treatment of <b>WS-383</b> at indicated concentrations. (C) Deneddylation effects on cullin1–5 and CRL substrates accumulation in MGC-803 cells with 24 h treatment of <b>DI-591</b> at indicated concentrations. (D, E) The expressions of p21, p27, and NRF2 in MGC-803 cells after 24 h treatment of <b>WS-383</b>(D) and <b>MLN4924</b> (E). GAPDH was used as the loading control. All experiments were carried out three times.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.9b00113/20190308/images/large/jm-2019-00113s_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00113&amp;id=fig13"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">As stated above, <b>WS-383</b> can block neddylation of Cul1 and Cul3. So the expression levels of Cul1 and Cul3 substrates were also evaluated after treatment of MGC-803 and KYSE70 cells with <b>WS-383</b> for 24 h. Cul1 together with Skp1 (adaptor protein), F-box protein, and RBX1/RBX2 RING protein form SCF E3 complex. Cyclin dependent kinase inhibitor 1A (p21) and cyclin dependent kinase inhibitor 1B (p27), two well characterized substrates for the ubiquitin ligase activity of SCF complex,<a onclick="showRef(event, 'ref49 ref50 ref51 ref52 ref53'); return false;" href="javascript:void(0);" class="ref ref49 ref50 ref51 ref52 ref53">(49−53)</a> were dose-dependently accumulated after treatment with <b>WS-383</b> for 24 h in MGC-803 (<a data-tab="pane-pcw-Figures" href="#fig13" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig13" id="" class=" internalNav">13</a></a>D) and KYSE70 (<a data-tab="pane-pcw-Figures" href="#fig13" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig13" id="" class=" internalNav">13</a></a>B) cells. And the expression of NRF2, the best characterized substrate of CRL3,<a onclick="showRef(event, 'ref54 ref55 ref56 ref57'); return false;" href="javascript:void(0);" class="ref ref54 ref55 ref56 ref57">(54−57)</a> also increased in a dose-dependent manner in both of these two cells (<a data-tab="pane-pcw-Figures" href="#fig13" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig13" id="" class=" internalNav">13</a></a>B and <a data-tab="pane-pcw-Figures" href="#fig13" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig13" id="" class=" internalNav">13</a></a>D). <b>MLN4924</b>, an NAE inhibitor, induced accumulation of p21, p27, and NRF2 in MGC-803 cells (<a data-tab="pane-pcw-Figures" href="#fig13" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig13" id="" class=" internalNav">13</a></a>E). On the other hand, <b>DI-591</b> only induced accumulation of NRF2 as it only decreased Cul3 neddylation (<a data-tab="pane-pcw-Figures" href="#fig13" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig13" id="" class=" internalNav">13</a></a>C). However, compared with <b>MLN4924</b>, <b>DI-591</b>, and <b>NAcM-OPT</b>, <b>WS-383</b> exerted cellular effects on cullin neddylation and downstream substrates p21, p27, and NRF2 at relatively high concentration (10 μM) in MGC-803 and KYSE70 cells. This is possibly attributed to the solubility or permeability issues of <b>WS-383</b>.</div><div class="NLM_p">NRF2 is an important transcription factor that could mediate antioxidant protection. In view of the observed cellular effect of <b>WS-383</b> on NRF2 activation, we also examined its effects on the ROS generation in MGC-803 cells using <b>WS-524</b> (inactive), <b>NAcM-OPT</b>, and <b>DI-591</b> as the controls, respectively. The cellular ROS levels were detected by the DCFH-DA staining. As shown in <a data-tab="pane-pcw-Figures" href="#fig14" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig14" id="" class=" internalNav">14</a></a>, like previously reported DCN1 inhibitors <b>NAcM-OPT</b> and <b>DI-591</b>, <b>WS-383</b> and <b>WS-524</b> had no remarkable effect on ROS generation in MGC-803 cells, showing the ROS-independent NRF2 activation by <b>WS-383</b>. To conclude, <b>WS-383</b> inhibited Cul3 neddylation and then activated NRF2 by blocking the DCN1–UBC12 interaction, not elevating cellular ROS levels.</div><figure id="fig14" data-index="14" class="article__inlineFigure"><h2 class="fig-label">Figure 14</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.9b00113/20190308/images/medium/jm-2019-00113s_0014.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.9b00113/20190308/images/large/jm-2019-00113s_0014.jpeg" id="gr17" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 14. Effect on ROS generation in MGC-803 cells. MGC-803 cells were treated with indicated concentrations of <b>WS-383</b>, <b>WS-524</b>, <b>NAcM-OPT</b>, or <b>DI-591</b> for 24 h. The levels of ROS were detected by flow cytometry.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.9b00113/20190308/images/large/jm-2019-00113s_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00113&amp;id=fig14"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">In view of the validated cellular effects of <b>WS-383</b>, we next examined the metabolic stability of <b>WS-383</b> in human, rat, and dog liver microsomes. Unfortunately, <b>WS-383</b> metabolized quickly (<i>T</i><sub>1/2</sub> < 5 min) and had high clearance (Cl<sub>int</sub> > 347.66 (human), 496.74 (rat), and 691.00 (dog) mL min<sup>–1</sup> kg<sup>–1</sup>, respectively). However, because of the interesting cellular effects of <b>WS-383</b> on cullin neddylation and downstream proteins p21, p27, and NRF2 in MGC-803 and KYSE70 cells, <b>WS-383</b> could be potentially used as a tool compound to examine the function of the DCN1–UBC12 interaction in modulating neddylation pathway in cells. The data also suggest that the aryl thioethers are metabolically instable in different species. The results may provide direction for further optimization so as to improve the metabolic stability.</div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i18">Chemistry</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_30011" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_30011" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec3_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i19" class="anchor-spacer"></div><h3 class="article-section__title" id="_i19"> Synthetic Routes for the Synthesis of Compounds <b>E1</b>–<b>E44</b></h3><div class="NLM_p">As shown in <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>A, 4<i>H</i>-1,2,4-triazol-3-amine reacted with diverse β-ketoesters to give compounds <b>C1</b>–<b>C5</b>, chlorination of <b>C1</b>–<b>C5</b> in POCl<sub>3</sub> yielded <b>D1</b>–<b>D5</b>, followed by reactions with substituted 5-mercaptotetrazole, affording <b>E1</b>–<b>E4</b>, and <b>E8</b>–<b>E11</b> (obtained in HCl form). Similarly, treatment of methyl 2-cyclopentanonecarboxylate with 4<i>H</i>-1,2,4-triazol-3-amine in AcOH gave <b>C6</b>, which was then subjected to chlorination and substitution reactions following above-described routes, yielding <b>E12</b> (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>B). Following the route for the synthesis of <b>E12</b>, <b>E5</b> was obtained efficiently from 1<i>H</i>-pyrazol-5-amine and ethyl acetoacetate (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>C). 5-Amino-4<i>H</i>-1,2,4-triazole-3-thiol reacted with diverse alkyl halides to form <b>B1</b>–<b>B27</b>, followed by reactions with β-ketoesters, generating <b>C8</b>–<b>C39</b>. Subsequent chlorination of <b>C8</b>–<b>C39</b> in POCl<sub>3</sub> afforded <b>D8</b>–<b>D39</b>, which then reacted with 1-[2-(dimethylamino)ethyl]-1<i>H</i>-tetrazole-5-thiol in EtOH to give the title compounds <b>E13</b>–<b>E44</b> (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>D). Besides, 1-[2-(dimethylamino)ethyl]-1<i>H</i>-tetrazole-5-thiol reacted with commercially available 4-chlorothieno[3,2-<i>d</i>]pyrimidine and 4-chloro-6,7-bis(2-methoxyethoxy)quinazoline, respectively, to form <b>E6</b> and <b>E7</b> (<a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>).</div><figure id="sch1" data-index="15" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.9b00113/20190308/images/medium/jm-2019-00113s_0015.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.9b00113/20190308/images/large/jm-2019-00113s_0015.jpeg" id="gr18" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of Compounds <b>E1</b>–<b>E5</b> and <b>E8</b>–<b>E44</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.9b00113/20190308/images/large/jm-2019-00113s_0015.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00113&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) acetic acid, 120 °C, 2–15 h, reflux; (b) POCl<sub>3</sub>, 90 °C, 3–5 h, reflux; (c) EtOH, rt, 5 h; (d) Na<sub>2</sub>CO<sub>3</sub>, acetone, 60 °C, 3–5 h, reflux.</p></p></figure><figure id="sch2" data-index="16" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.9b00113/20190308/images/medium/jm-2019-00113s_0016.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.9b00113/20190308/images/large/jm-2019-00113s_0016.jpeg" id="gr19" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of Compounds <b>E6</b> and <b>E7</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.9b00113/20190308/images/large/jm-2019-00113s_0016.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00113&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) EtOH, rt, 5 h.</p></p></figure></div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i22">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_70158" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_70158" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">Starting from the hit compound <b>E1</b> identified from our molecular library, we performed further structure-based optimizations, leading to the generation of a series of triazolo[1,5-<i>a</i>]pyrimidine-based DCN1 inhibitors. The SARs studies revealed that relative to other heterocycles (as shown in <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>), the triazolopyridine is a privileged scaffold for the development of small-molecule DCN1 inhibitors. Generally, the less steric group (R<sup>1</sup>) is preferred over other bulky groups. The R<sup>2</sup> group is presumably inserted into a deep subpocket of DCN1, bulky groups, or introduction of substitutions at the <i>ortho</i> and <i>meta</i>-position of phenyl ring could cause drastic activity decrease (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Tables <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a> and <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a>). The R<sup>3</sup> group is exposed to the solvent region, and introduction of hydrophobic group (e.g., <b>WS-524</b>) could cause the complex to be unstable and lead to the loss of the activity. Among these compounds, <b>WS-383</b> effectively blocked the DCN1–UBC12 interaction (IC<sub>50</sub> = 11 nM) in a reversible manner and showed certain selectivity over BTK, CDK, and EGFR kinases. Docking studies were performed to reveal the binding models of <b>WS-383</b> in the binding site of UBC12 and to explain the activity discrepancy of <b>WS-383</b> and <b>WS-524</b>. The CETSA assay suggested cellular target engagement of <b>WS-383</b> to DCN1 in MGC-803 cells overexpressing DCN1 and UBC12. In MGC-803 cells, <b>MLN4924</b> caused broad neddylation inhibition of cullin substrates, while <b>WS-383</b> showed selective neddylation inhibition of Cul3/1 over other cullin members and also induced accumulation of downstream proteins p21, p27, and NRF2 in both MGC-803 and KYSE70 cells. The NRF2 activation by <b>WS-383</b> was ROS-independent. However, <b>WS-383</b> metabolized quickly (<i>T</i><sub>1/2</sub> < 5 min) and had high clearance in human, rat, and dog liver microsomes. Compared with previously reported <b>MLN4924</b>, <b>DI-591</b>, and <b>NAcM-OPT</b>, <b>WS-383</b> represents a new potent and specific small-molecule DCN1 inhibitor and therefore could be used as a tool compound to examine the function of the DCN1–UBC12 interaction in modulating neddylation pathway in cells. The discovery of <b>WS-383</b> makes the triazolo[1,5-<i>a</i>]pyrimidine a promising scaffold for the development of new small-molecule DCN1 inhibitors. To conclude, targeting the interaction of DCN1 with UBC12 would be a feasible strategy for the treatment of diseases, in which Cul3/1 was dysregulated.</div></div><div id="sec5" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i23">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_10614" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_10614" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec5_1_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i24" class="anchor-spacer"></div><h3 class="article-section__title" id="_i24"> Chemistry. General Information</h3><div class="NLM_p last">Reagents and solvents were purchased for direct use without further purification. Thin-layer chromatography (TLC) was carried out on glass plates coated with silica gel (Qingdao Haiyang Chemical Co., G60F-254) and visualized by UV light (254 nm). The products were purified by column chromatography over silica gel (Qingdao Haiyang Chemical Co., 200–300 mesh). Melting points were determined on an X-5 micromelting apparatus and are uncorrected. All the NMR spectra were recorded with a Bruker DPX 400 MHz spectrometer with TMS as internal standard in CDCl<sub>3</sub> or DMSO-<i>d</i><sub>6</sub>. Chemical shifts are given as δ ppm values relative to TMS. High-resolution mass spectra (HRMS) were recorded on a Waters Micromass Q-T of Micromass spectrometer by electrospray ionization (ESI). The purity of <b>E1</b>–<b>E44</b> was determined by Waters e2695 HPLC. All the compounds were >95% pure (Phenomenex column, C-18, 5.0 μm, 4.6 mm × 250 mm). The signal was monitored at 296 nm with a UV detector. Compounds <b>E1</b>–<b>E7</b>, <b>E9</b>–<b>E26</b>, <b>E28</b>, and <b>E31</b>–<b>E44</b> were detected by the condition of a mobile phase of acetonitrile in H<sub>2</sub>O (50:50, v/v) with a flow rate of 1.0 mL/min. Compound <b>E8</b> was treated with a mobile phase of acetonitrile in H<sub>2</sub>O (15:85, v/v), while compound <b>E27</b> was subjected to the condition of acetonitrile in H<sub>2</sub>O (30:70, v/v). Besides, compounds <b>E29</b>–<b>E30</b> were also determined by the condition of acetonitrile in H<sub>2</sub>O (40:60, v/v).</div></div><div id="sec5_1_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i25" class="anchor-spacer"></div><h3 class="article-section__title" id="_i25"> General Method for the Synthesis of <b>B1</b>–<b>B27</b></h3><div class="NLM_p last">Compounds <b>B1</b>–<b>B27</b> were prepared following the previously reported method.<a onclick="showRef(event, 'ref56'); return false;" href="javascript:void(0);" class="ref ref56">(56)</a> To a solution of 3-amino-5-mercapto-1,2,4-triazole (1.0 g, 8.61 mmol) in acetone (30 mL) were added sodium carbonate (1.37 g, 12.92 mmol), sodium iodide (129.06 mg, 0.861 mmol), and alkyl halide (9.47 mmol). The reaction mixture was stirred at 60 °C for 3–6 h before cooling to room temperature. Na<sub>2</sub>CO<sub>3</sub> and NaI were removed via filtration. The residue was concentrated under vacuum and then dissolved in EtOAc. The organic layer was washed with water (2 × 10 mL) and saturated brine (2 × 20 mL) and then dried over anhydrous MgSO<sub>4</sub>. After removal of the solvent, the resulting residue was subjected to column chromatography, giving the corresponding products <b>B1</b>–<b>B27</b>.</div></div><div id="sec5_1_2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i26" class="anchor-spacer"></div><h3 class="article-section__title" id="_i26"> 5-(Propylthio)-4<i>H</i>-1,2,4-triazol-3-amine (<b>B1</b>)</h3><div class="NLM_p last">White solid; yield, 74%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.86 (s, 1H), 6.01 (s, 2H), 2.91 (t, <i>J</i> = 7.1 Hz, 2H), 1.69–1.55 (m, 2H), 0.93 (t, <i>J</i> = 7.3 Hz, 3H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 157.3 (-C-), 156.0 (-C-), 32.8 (-<u class="uu">C</u>H<sub>2</sub>-), 22.8 (-<u class="uu">C</u>H<sub>2</sub>-), 13.1 (-<u class="uu">C</u>H<sub>3</sub>). HRMS (ESI): <i>m</i>/<i>z</i> calcd for C<sub>5</sub>H<sub>9</sub>N<sub>4</sub>S (M – H)<sup>−</sup>, 157.0547; found, 157.0544.</div></div><div id="sec5_1_2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i27" class="anchor-spacer"></div><h3 class="article-section__title" id="_i27"> 5-(Allylthio)-4<i>H</i>-1,2,4-triazol-3-amine (<b>B2</b>)</h3><div class="NLM_p last">White solid; yield, 72%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.92 (s, 1H), 6.03 (s, 2H), 5.91 (m, 1H), 5.20 (m, 1H), 5.04 (m, 1H), 3.62 (d, <i>J</i> = 6.8 Hz, 2H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 157.4 (-C-), 155.4 (-C-), 134.4 (-C-), 117.2 (-<u class="uu">C</u>H-), 33.6 (-<u class="uu">C</u>H<sub>2</sub>-). HRMS (ESI): <i>m</i>/<i>z</i> calcd for C<sub>5</sub>H<sub>7</sub>N<sub>4</sub>S (M – H)<sup>−</sup>, 155.0391; found, 155.0387.</div></div><div id="sec5_1_2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i28" class="anchor-spacer"></div><h3 class="article-section__title" id="_i28"> 5-(Prop-2-yn-1-ylthio)-4<i>H</i>-1,2,4-triazol-3-amine (<b>B3</b>)</h3><div class="NLM_p last">White solid; yield, 81%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.99 (s, 1H), 6.09 (s, 2H), 3.78 (d, <i>J</i> = 2.5 Hz, 2H), 3.10 (t, <i>J</i> = 2.5 Hz, 1H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 157.4 (-C-), 154.5 (-C-), 80.5 (-C-), 73.3 (-<u class="uu">C</u>H-), 19.0 (-<u class="uu">C</u>H<sub>2</sub>-). HRMS (ESI): <i>m</i>/<i>z</i> calcd for C<sub>5</sub>H<sub>5</sub>N<sub>4</sub>S (M – H)<sup>−</sup>, 153.0234; found, 153.0231.</div></div><div id="sec5_1_2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i29" class="anchor-spacer"></div><h3 class="article-section__title" id="_i29"> 5-(Isopropylthio)-4<i>H</i>-1,2,4-triazol-3-amine (<b>B4</b>)</h3><div class="NLM_p last">White solid; yield, 70%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.91 (s, 1H), 6.01 (s, 2H), 3.51 (m, 1H), 1.28 (d, <i>J</i> = 6.8 Hz, 6H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 157.7 (-C-), 156.1 (-C-), 36.5 (-<u class="uu">C</u>H-), 23.9 (-<u class="uu">C</u>H<sub>3</sub>). HRMS (ESI): <i>m</i>/<i>z</i> calcd for C<sub>5</sub>H<sub>11</sub>N<sub>4</sub>S (M + H)<sup>−</sup>, 159.0704; found, 159.0695.</div></div><div id="sec5_1_2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i30" class="anchor-spacer"></div><h3 class="article-section__title" id="_i30"> 5-((Cyclopropylmethyl)thio)-4<i>H</i>-1,2,4-triazol-3-amine (<b>B5</b>)</h3><div class="NLM_p last">White solid; yield, 74%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.87 (s, 1H), 6.00 (s, 2H), 2.90 (d, <i>J</i> = 7.1 Hz, 2H), 1.17–0.95 (m, 1H), 0.58–0.42 (m, 2H), 0.23 (m, 2H). <sup>13</sup>C NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 157.7 (-C-), 156.7 (-C-), 36.7 (-<u class="uu">C</u>H<sub>2</sub>-), 11.7 (-<u class="uu">C</u>H-), 5.9 (-<u class="uu">C</u>H<sub>2</sub>-). HRMS (ESI): <i>m</i>/<i>z</i> calcd for C<sub>6</sub>H<sub>11</sub>N<sub>4</sub>S (M + H)<sup>−</sup>, 171.0704; found, 171.0694.</div></div><div id="sec5_1_2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i31" class="anchor-spacer"></div><h3 class="article-section__title" id="_i31"> 5-((Cyclohexylmethyl)thio)-4<i>H</i>-1,2,4-triazol-3-amine (<b>B6</b>)</h3><div class="NLM_p last">White solid; yield, 63%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.84 (s, 1H), 6.00 (s, 2H), 2.85 (d, <i>J</i> = 6.7 Hz, 2H), 1.78 (d, <i>J</i> = 12.7 Hz, 2H), 1.71–1.44 (m, 4H), 1.24–1.04 (m, 3H), 1.01–0.87 (m, 2H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 157.7 (-C-), 156.8 (-C-), 38.2 (-<u class="uu">C</u>H<sub>2</sub>-), 37.9 (-<u class="uu">C</u>H-), 32.4 (-<u class="uu">C</u>H<sub>2</sub>-), 26.4 (-<u class="uu">C</u>H<sub>2</sub>-), 26.0 (-<u class="uu">C</u>H<sub>2</sub>-). HRMS (ESI): <i>m</i>/<i>z</i> calcd for C<sub>9</sub>H<sub>17</sub>N<sub>4</sub>S (M + H)<sup>−</sup>, 213.1173; found, 213.1162.</div></div><div id="sec5_1_2_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i32" class="anchor-spacer"></div><h3 class="article-section__title" id="_i32"> 5-((Naphthalen-2-ylmethyl)thio)-4<i>H</i>-1,2,4-triazol-3-amine (<b>B7</b>)</h3><div class="NLM_p last">White solid; yield, 78%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.05 (s, 1H), 8.15 (d, <i>J</i> = 8.2 Hz, 1H), 7.95 (d, <i>J</i> = 7.7 Hz, 1H), 7.85 (d, <i>J</i> = 8.2 Hz, 1H), 7.64–7.49 (m, 3H), 7.42 (t, <i>J</i> = 7.6 Hz, 1H), 6.08 (s, 2H), 4.73 (s, 2H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 134.1 (-C-), 134.0 (-C-), 131.5 (-C-), 129.1 (-C-), 128.4 (-C-), 127.7 (-<u class="uu">C</u>H-), 126.7 (-<u class="uu">C</u>H-), 126.4 (-<u class="uu">C</u>H-), 125.9 (-<u class="uu">C</u>H−), 124.4 (-<u class="uu">C</u>H-), 33.4 (-<u class="uu">C</u>H<sub>2</sub>-). HRMS (ESI): <i>m</i>/<i>z</i> calcd for C<sub>13</sub>H<sub>13</sub>N<sub>4</sub>S (M + H)<sup>−</sup>, 257.0860; found, 257.0849.</div></div><div id="sec5_1_2_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i33" class="anchor-spacer"></div><h3 class="article-section__title" id="_i33"> 5-(Benzylthio)-4<i>H</i>-1,2,4-triazol-3-amine (<b>B8</b>)</h3><div class="NLM_p last">White solid; yield, 73%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 7.41–7.18 (m, 5H), 4.34 (s, 2H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 152.2 (-C-), 147.1 (-C-), 136.6 (-C-), 128.8 (-<u class="uu">C</u>H-), 128.4 (-<u class="uu">C</u>H-), 127.5 (-<u class="uu">C</u>H-), 35.4 (-<u class="uu">C</u>H<sub>2</sub>-). HRMS (ESI): <i>m</i>/<i>z</i> calcd for C<sub>9</sub>H<sub>9</sub>N<sub>4</sub>S (M – H)<sup>−</sup>, 205.0547; found, 205.0548.</div></div><div id="sec5_1_2_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i34" class="anchor-spacer"></div><h3 class="article-section__title" id="_i34"> 5-(Phenethylthio)-4<i>H</i>-1,2,4-triazol-3-amine (<b>B9</b>)</h3><div class="NLM_p last">White solid; yield, 78%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.92 (s, 1H), 7.33–7.26 (m, 2H), 7.22 (m, 3H), 6.05 (s, 2H), 3.18 (m, 2H), 2.96–2.87 (m, 2H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 157.8 (-C-), 156.3 (-C-), 152.7 (-C-), 140.8 (−CH-), 129.0 (-<u class="uu">C</u>H-), 128.8 (-<u class="uu">C</u>H-), 126.7 (-<u class="uu">C</u>H-), 36.0 (-<u class="uu">C</u>H<sub>2</sub>-), 32.5 (-<u class="uu">C</u>H<sub>2</sub>-). HRMS (ESI): <i>m</i>/<i>z</i> calcd for C<sub>10</sub>H<sub>13</sub>N<sub>4</sub>S (M + H)<sup>−</sup>, 221.0860; found, 221.0852.</div></div><div id="sec5_1_2_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i35" class="anchor-spacer"></div><h3 class="article-section__title" id="_i35"> 5-((3-Phenylpropyl)thio)-4<i>H</i>-1,2,4-triazol-3-amine (<b>B10</b>)</h3><div class="NLM_p last">White solid; yield, 75%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 7.27 (m, 2H), 7.22–7.13 (m, 3H), 6.02 (s, 2H), 2.93 (t, <i>J</i> = 7.2 Hz, 2H), 2.73–2.62 (m, 2H), 2.01–1.83 (m, 2H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d6</i>) δ 157.5 (-C-), 156.2 (-C-), 141.3 (-C-), 128.3 (-<u class="uu">C</u>H-), 128.3 (-<u class="uu">C</u>H-), 125.8 (-<u class="uu">C</u>H-), 34.0 (-<u class="uu">C</u>H<sub>2</sub>-), 31.1 (-<u class="uu">C</u>H<sub>2</sub>-), 30.3 (-<u class="uu">C</u>H<sub>2</sub>-). HRMS (ESI): <i>m</i>/<i>z</i> calcd for C<sub>11</sub>H<sub>13</sub>N<sub>4</sub>S (M – H)<sup>−</sup>, 233.0860; found, 233.0863.</div></div><div id="sec5_1_2_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i36" class="anchor-spacer"></div><h3 class="article-section__title" id="_i36"> 5-(((5-Chlorobenzo[<i>b</i>]thiophen-3-yl)methyl)thio)-4<i>H</i>-1,2,4-triazol-3-amine (<b>B11</b>)</h3><div class="NLM_p last">White solid; yield, 64%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.02 (d, <i>J</i> = 8.6 Hz, 1H), 7.98 (d, <i>J</i> = 1.9 Hz, 1H), 7.70 (s, 1H), 7.41 (m, 1H), 6.10 (s, 2H), 4.50 (s, 2H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 139.2 (-C-), 138.2 (-C-), 132.1 (-C-), 129.4 (-C-), 127.5 (-<u class="uu">C</u>H-), 124.7 (-<u class="uu">C</u>H-), 124.5 (-<u class="uu">C</u>H-), 121.6 (-<u class="uu">C</u>H-), 28.2 (-<u class="uu">C</u>H<sub>2</sub>-). HRMS (ESI): <i>m</i>/<i>z</i> calcd for C<sub>11</sub>H<sub>10</sub>ClN<sub>4</sub>S<sub>2</sub> (M + H)<sup>−</sup>, 297.0035; found, 297.0024.</div></div><div id="sec5_1_2_12" class="NLM_sec NLM_sec_level_2"><div id="ac_i37" class="anchor-spacer"></div><h3 class="article-section__title" id="_i37"> 4-(((5-Amino-4<i>H</i>-1,2,4-triazol-3-yl)thio)methyl)-2<i>H</i>-benzo[<i>h</i>]chromen-2-one (<b>B12</b>)</h3><div class="NLM_p last">White solid; yield, 56%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.40–8.32 (m, 1H), 8.10–8.01 (m, 1H), 7.90 (m, 2H), 7.78–7.67 (m, 2H), 6.53 (s, 1H), 6.15 (s, 2H), 4.52 (s, 2H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 159.5 (-C-), 150.2 (-C-), 134.3 (-C-), 128.8 (-C-), 128.0 (-C-), 127.5 (-C-), 124.0 (-<u class="uu">C</u>H-), 122.3 (-<u class="uu">C</u>H-), 121.6 (-C-), 121.1 (-<u class="uu">C</u>H-), 113.9 (-C-), 113.6 (-<u class="uu">C</u>H-), 31.6 (-<u class="uu">C</u>H<sub>2</sub>-). HRMS (ESI): <i>m</i>/<i>z</i> calcd for C<sub>16</sub>H<sub>13</sub>N<sub>4</sub>O<sub>2</sub>S (M + H)<sup>−</sup>, 325.0759; found, 325.0747.</div></div><div id="sec5_1_2_13" class="NLM_sec NLM_sec_level_2"><div id="ac_i38" class="anchor-spacer"></div><h3 class="article-section__title" id="_i38"> 5-((4-Nitrobenzyl)thio)-4<i>H</i>-1,2,4-triazol-3-amine (<b>B13</b>)</h3><div class="NLM_p last">White solid; yield, 65%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.99 (s, 1H), 8.16 (d, <i>J</i> = 8.7 Hz, 2H), 7.64 (d, <i>J</i> = 8.7 Hz, 2H), 6.10 (s, 2H), 4.35 (s, 2H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 157.5 (-C-), 155.5 (-C-), 138.4 (-C-), 131.2 (-C-), 131.0 (-<u class="uu">C</u>H-), 120.1 (-<u class="uu">C</u>H-), 34.1 (-<u class="uu">C</u>H<sub>2</sub>-). HRMS (ESI): <i>m</i>/<i>z</i> calcd for C<sub>9</sub>H<sub>10</sub>N<sub>5</sub>O<sub>2</sub>S (M + H)<sup>−</sup>, 252.0555; found, 205.0544.</div></div><div id="sec5_1_2_14" class="NLM_sec NLM_sec_level_2"><div id="ac_i39" class="anchor-spacer"></div><h3 class="article-section__title" id="_i39"> 5-((2-Nitrobenzyl)thio)-4<i>H</i>-1,2,4-triazol-3-amine (<b>B14</b>)</h3><div class="NLM_p last">White solid; yield, 68%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.97 (s, 1H), 8.01 (d, <i>J</i> = 8.1 Hz, 1H), 7.66 (m, 2H), 7.56–7.49 (m, 1H), 6.07 (s, 2H), 4.50 (s, 2H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 157.9 (-C-), 155.6 (-C-), 148.7 (-C-), 134.3 (-C-), 134.0 (-<u class="uu">C</u>H-), 132.6 (-<u class="uu">C</u>H-), 129.1 (-<u class="uu">C</u>H-), 125.3 (-<u class="uu">C</u>H-), 32.4 (-<u class="uu">C</u>H<sub>2</sub>-). HRMS (ESI): <i>m</i>/<i>z</i> calcd for C<sub>9</sub>H<sub>10</sub>N<sub>5</sub>O<sub>2</sub>S (M + H)<sup>−</sup>, 252.0555; found, 252.0543.</div></div><div id="sec5_1_2_15" class="NLM_sec NLM_sec_level_2"><div id="ac_i40" class="anchor-spacer"></div><h3 class="article-section__title" id="_i40"> 5-((4-Fluorobenzyl)thio)-4<i>H</i>-1,2,4-triazol-3-amine (<b>B15</b>)</h3><div class="NLM_p last">White solid; yield, 78%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.98 (s, 1H), 7.40 (m, 2H), 7.11 (t, <i>J</i> = 8.8 Hz, 2H), 6.06 (s, 2H), 4.21 (s, 2H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 162.9 (-C-), 160.5 (-C-), 157.9 (-C-), 135.3 (-C-), 131.14 (-<u class="uu">C</u>H-), 131.06 (-<u class="uu">C</u>H-), 115.6 (-<u class="uu">C</u>H-), 115.4 (-<u class="uu">C</u>H-), 34.5 (-<u class="uu">C</u>H<sub>2</sub>-). HRMS (ESI): <i>m</i>/<i>z</i> calcd for C<sub>9</sub>H<sub>10</sub>FN<sub>4</sub>S (M + H)<sup>−</sup>, 225.0610; found, 225.0598.</div></div><div id="sec5_1_2_16" class="NLM_sec NLM_sec_level_2"><div id="ac_i41" class="anchor-spacer"></div><h3 class="article-section__title" id="_i41"> 5-((4-Chlorobenzyl)thio)-4<i>H</i>-1,2,4-triazol-3-amine (<b>B16</b>)</h3><div class="NLM_p last">White solid; yield, 76%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.98 (s, 1H), 7.45–7.30 (m, 4H), 6.08 (s, 2H), 4.21 (s, 2H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 157.4 (-C-), 155.4 (-C-), 138.3 (-C-), 131.1 (-C-), 130.9 (-<u class="uu">C</u>H-), 120.0 (-<u class="uu">C</u>H-), 34.0 (-<u class="uu">C</u>H<sub>2</sub>-). HRMS (ESI): <i>m</i>/<i>z</i> calcd for C<sub>9</sub>H<sub>10</sub>ClN<sub>4</sub>S (M + H)<sup>−</sup>, 241.0314; found, 241.0305.</div></div><div id="sec5_1_2_17" class="NLM_sec NLM_sec_level_2"><div id="ac_i42" class="anchor-spacer"></div><h3 class="article-section__title" id="_i42"> 5-((4-Bromobenzyl)thio)-4<i>H</i>-1,2,4-triazol-3-amine (<b>B17</b>)</h3><div class="NLM_p last">White solid; yield, 72%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.95 (s, 1H), 7.48 (d, <i>J</i> = 8.3 Hz, 2H), 7.32 (d, <i>J</i> = 8.3 Hz, 2H), 6.07 (s, 2H), 4.18 (s, 2H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 157.4 (-C-), 155.4 (-C-), 138.2 (-C-), 131.1 (-C-), 130.9 (-<u class="uu">C</u>H-), 120.0 (-<u class="uu">C</u>H-), 34.0 (-<u class="uu">C</u>H<sub>2</sub>-). HRMS (ESI): <i>m</i>/<i>z</i> calcd for C<sub>9</sub>H<sub>10</sub>BrN<sub>4</sub>S (M + H)<sup>−</sup>, 284.9809; found, 284.9798.</div></div><div id="sec5_1_2_18" class="NLM_sec NLM_sec_level_2"><div id="ac_i43" class="anchor-spacer"></div><h3 class="article-section__title" id="_i43"> 5-((2-Bromobenzyl)thio)-4<i>H</i>-1,2,4-triazol-3-amine (<b>B18</b>)</h3><div class="NLM_p last">White solid; yield, 75%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.99 (s, 1H), 7.62 (d, <i>J</i> = 7.9 Hz, 1H), 7.50 (d, <i>J</i> = 7.4 Hz, 1H), 7.32 (t, <i>J</i> = 7.4 Hz, 1H), 7.21 (t, <i>J</i> = 7.6 Hz, 1H), 6.09 (s, 2H), 4.31 (s, 2H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 157.9 (-C-), 155.8 (-C-), 137.8 (-C-), 133.1 (-C-), 131.5 (-<u class="uu">C</u>H-), 129.8 (-<u class="uu">C</u>H-), 128.3 (-<u class="uu">C</u>H-), 124.3 (-<u class="uu">C</u>H-), 36.0 (-<u class="uu">C</u>H<sub>2</sub>-). HRMS (ESI): <i>m</i>/<i>z</i> calcd for C<sub>9</sub>H<sub>10</sub>BrN<sub>4</sub>S (M + H)<sup>−</sup>, 284.9809; found, 284.9799.</div></div><div id="sec5_1_2_19" class="NLM_sec NLM_sec_level_2"><div id="ac_i44" class="anchor-spacer"></div><h3 class="article-section__title" id="_i44"> 5-((3-Bromobenzyl)thio)-4<i>H</i>-1,2,4-triazol-3-amine (<b>B19</b>)</h3><div class="NLM_p last">White solid; yield, 74%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.97 (s, 1H), 7.57 (s, 1H), 7.42 (d, <i>J</i> = 7.9 Hz, 1H), 7.37 (d, <i>J</i> = 7.7 Hz, 1H), 7.25 (t, <i>J</i> = 7.8 Hz, 1H), 6.08 (s, 2H), 4.21 (s, 2H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 157.9 (-C-), 155.8 (-C-), 131.8 (-C-), 130.9 (-C-), 130.2 (-<u class="uu">C</u>H-), 128.2 (-<u class="uu">C</u>H-), 121.9 (-<u class="uu">C</u>H-), 100.0 (-<u class="uu">C</u>H-), 34.5 (-<u class="uu">C</u>H<sub>2</sub>-). HRMS (ESI): <i>m</i>/<i>z</i> calcd for C<sub>9</sub>H<sub>10</sub>BrN<sub>4</sub>S (M + H)<sup>−</sup>, 284.9809; found, 284.9796.</div></div><div id="sec5_1_2_20" class="NLM_sec NLM_sec_level_2"><div id="ac_i45" class="anchor-spacer"></div><h3 class="article-section__title" id="_i45"> 5-((4-Methylbenzyl)thio)-4<i>H</i>-1,2,4-triazol-3-amine (<b>B20</b>)</h3><div class="NLM_p last">White solid; yield, 82%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.93 (s, 1H), 7.24 (d, <i>J</i> = 7.9 Hz, 2H), 7.09 (d, <i>J</i> = 7.8 Hz, 2H), 6.05 (s, 2H), 4.17 (s, 2H), 2.26 (s, 3H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 157.8 (-C-), 156.3 (-C-), 136.6 (-C-), 135.8 (-C-), 129.3 (-<u class="uu">C</u>H-), 129.1 (-<u class="uu">C</u>H-), 35.1 (-<u class="uu">C</u>H<sub>2</sub>-), 21.1 (-<u class="uu">C</u>H<sub>3</sub>). HRMS (ESI): <i>m</i>/<i>z</i> calcd for C<sub>10</sub>H<sub>13</sub>N<sub>4</sub>S (M + H)<sup>−</sup>, 221.0860; found, 221.0848.</div></div><div id="sec5_1_2_21" class="NLM_sec NLM_sec_level_2"><div id="ac_i46" class="anchor-spacer"></div><h3 class="article-section__title" id="_i46"> 5-((4-(Trifluoromethyl)benzyl)thio)-4<i>H</i>-1,2,4-triazol-3-amine (<b>B21</b>)</h3><div class="NLM_p last">White solid; yield, 69%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.21 (s, 1H), 7.66 (d, <i>J</i> = 8.2 Hz, 2H), 7.59 (d, <i>J</i> = 8.2 Hz, 2H), 6.09 (s, 2H), 4.31 (s, 2H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 158.3 (-C-), 144.3 (-C-), 129.9 (-C-), 128.2 (-C-), 127.8 (-C-), 126.1 (-C-), 125.6 (-<u class="uu">C</u>H-), 125.6 (-<u class="uu">C</u>H-), 125.5 (-<u class="uu">C</u>H-), 123.4 (-<u class="uu">C</u>H-), 34.7 (-<u class="uu">C</u>H<sub>2</sub>-). HRMS (ESI): <i>m</i>/<i>z</i> calcd for C<sub>10</sub>H<sub>10</sub>F<sub>3</sub>N<sub>4</sub>S (M + H)<sup>−</sup>, 275.0578; found, 275.0564.</div></div><div id="sec5_1_2_22" class="NLM_sec NLM_sec_level_2"><div id="ac_i47" class="anchor-spacer"></div><h3 class="article-section__title" id="_i47"> 5-((2-(Trifluoromethyl)benzyl)thio)-4<i>H</i>-1,2,4-triazol-3-amine (<b>B22</b>)</h3><div class="NLM_p last">White solid; yield, 71%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.01 (s, 1H), 7.67 (m, 3H), 7.47 (m, 1H), 6.13 (s, 2H), 4.40 (s, 2H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 158.0 (-C-), 155.8 (-C-), 137.0 (-C-), 133.2 (-C-), 132.0 (-C-), 128.3 (-C-), 127.5 (-<u class="uu">C</u>H-), 127.2 (-<u class="uu">C</u>H-), 126.5 (-<u class="uu">C</u>H-), 126.4 (-<u class="uu">C</u>H-), 126.2 (-<u class="uu">C</u>H-), 32.1 (-<u class="uu">C</u>H<sub>2</sub>-). HRMS (ESI): <i>m</i>/<i>z</i> calcd for C<sub>10</sub>H<sub>9</sub>F<sub>3</sub>N<sub>4</sub>S (M + H)<sup>−</sup>, 275.0578; found, 275.0568.</div></div><div id="sec5_1_2_23" class="NLM_sec NLM_sec_level_2"><div id="ac_i48" class="anchor-spacer"></div><h3 class="article-section__title" id="_i48"> 5-((3-(Trifluoromethyl)benzyl)thio)-4<i>H</i>-1,2,4-triazol-3-amine (<b>B23</b>)</h3><div class="NLM_p last">White solid; yield, 70%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.97 (s, 1H), 7.72 (s, 1H), 7.68 (d, <i>J</i> = 7.6 Hz, 1H), 7.59 (d, <i>J</i> = 7.7 Hz, 1H), 7.53 (t, <i>J</i> = 7.7 Hz, 1H), 6.09 (s, 2H), 4.30 (s, 2H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 157.9 (-C-), 155.7 (-C-), 149.8 (-C-), 148.1 (-C-), 141.0 (-C-), 133.3 (-<u class="uu">C</u>H-), 129.8 (-<u class="uu">C</u>H-), 125.7 (-<u class="uu">C</u>H-), 124.1 (-<u class="uu">C</u>H-), 34.6 (-<u class="uu">C</u>H<sub>2</sub>-). HRMS (ESI): <i>m</i>/<i>z</i> calcd for C<sub>10</sub>H<sub>9</sub>F<sub>3</sub>N<sub>4</sub>S (M)<sup>−</sup>, 274.0500; found, 274.0487.</div></div><div id="sec5_1_2_24" class="NLM_sec NLM_sec_level_2"><div id="ac_i49" class="anchor-spacer"></div><h3 class="article-section__title" id="_i49"> 5-((4-Methoxybenzyl)thio)-4<i>H</i>-1,2,4-triazol-3-amine (<b>B24</b>)</h3><div class="NLM_p last">White solid; yield, 87%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.04 (s, 1H), 7.27 (d, <i>J</i> = 8.6 Hz, 2H), 6.90–6.80 (m, 2H), 6.12 (s, 2H), 4.17 (s, 2H), 3.72 (s, 3H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 158.8 (-C-), 130.5 (-C-), 130.4 (-<u class="uu">C</u>H-), 114.2 (-<u class="uu">C</u>H-), 55.5 (-<u class="uu">C</u>H<sub>3</sub>), 35.0 (-<u class="uu">C</u>H<sub>2</sub>-). HRMS (ESI): <i>m</i>/<i>z</i> calcd for C<sub>10</sub>H<sub>13</sub>N<sub>4</sub>OS (M + H)<sup>−</sup>, 237.0810; found, 237.0799.</div></div><div id="sec5_1_2_25" class="NLM_sec NLM_sec_level_2"><div id="ac_i50" class="anchor-spacer"></div><h3 class="article-section__title" id="_i50"> 5-((4-Chloro-3-fluorobenzyl)thio)-4<i>H</i>-1,2,4-triazol-3-amine (<b>B25</b>)</h3><div class="NLM_p last">White solid; yield, 80%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.97 (s, 1H), 7.50 (t, <i>J</i> = 8.1 Hz, 1H), 7.40 (m, 1H), 7.23 (d, <i>J</i> = 8.2 Hz, 1H), 6.09 (s, 2H), 4.22 (s, 2H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 158.5 (-C-), 157.9 (-C-), 156.1 (-C-), 155.7 (-C-), 141.4 (-C-), 130.8 (-C-), 126.5 (-C-), 126.5 (-<u class="uu">C</u>H-), 117.5 (-<u class="uu">C</u>H-), 117.3 (-<u class="uu">C</u>H-), 34.2 (-<u class="uu">C</u>H<sub>2</sub>-). HRMS (ESI): <i>m</i>/<i>z</i> calcd for C<sub>9</sub>H<sub>9</sub>ClFN<sub>4</sub>S (M + H)<sup>−</sup>, 259.0220; found, 259.0210.</div></div><div id="sec5_1_2_26" class="NLM_sec NLM_sec_level_2"><div id="ac_i51" class="anchor-spacer"></div><h3 class="article-section__title" id="_i51"> 5-((3-Chloro-4-fluorobenzyl)thio)-4<i>H</i>-1,2,4-triazol-3-amine (<b>B26</b>)</h3><div class="NLM_p last">White solid; yield, 82%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.02 (s, 1H), 7.58 (m, <i>J</i> = 7.2, 1.8 Hz, 1H), 7.43–7.26 (m, 2H), 6.11 (s, 2H), 4.21 (s, 2H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 157.9 (-C-), 155.5 (-C-), 137.4 (-C-), 131.1 (-C-), 129.8 (-C-), 129.8 (-C-), 119.5 (-C-), 119.4 (-<u class="uu">C</u>H-), 117.2 (-<u class="uu">C</u>H-), 117.0 (-<u class="uu">C</u>H-), 34.0 (-<u class="uu">C</u>H<sub>2</sub>-). HRMS (ESI): <i>m</i>/<i>z</i> calcd for C<sub>9</sub>H<sub>9</sub>ClFN<sub>4</sub>S (M + H)<sup>−</sup>, 259.0220; found, 259.0210.</div></div><div id="sec5_1_2_27" class="NLM_sec NLM_sec_level_2"><div id="ac_i52" class="anchor-spacer"></div><h3 class="article-section__title" id="_i52"> 5-((2,6-Dichlorobenzyl)thio)-4<i>H</i>-1,2,4-triazol-3-amine (<b>B27</b>)</h3><div class="NLM_p last">White solid; yield, 74%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.05 (s, 1H), 7.49 (d, <i>J</i> = 4.8 Hz, 1H), 7.48 (s, 1H), 7.38–7.30 (m, 1H), 6.12 (s, 2H), 4.49 (s, 2H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 158.0 (-C-), 155.6 (-C-), 135.4 (-C-), 133.6 (-C-), 130.5 (-C-), 129.2 (-<u class="uu">C</u>H-), 32.2 (-<u class="uu">C</u>H<sub>2</sub>-). HRMS (ESI): <i>m</i>/<i>z</i> calcd for C<sub>9</sub>H<sub>9</sub>Cl<sub>2</sub>N<sub>4</sub>S (M + H)<sup>−</sup>, 274.9925; found, 274.9913.</div></div><div id="sec5_1_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i53" class="anchor-spacer"></div><h3 class="article-section__title" id="_i53"> General Method for the Synthesis of <b>C1</b>–<b>C39</b></h3><div class="NLM_p last">Compounds <b>C1</b>–<b>C39</b> were prepared following the previously reported method<a onclick="showRef(event, 'ref56'); return false;" href="javascript:void(0);" class="ref ref56">(56)</a> as described for <b>C1</b>. <b>B1</b> (1g, 11.89 mmol) and ethyl 3-oxo-3-phenylpropanoate (2.06 mL, 11.89 mmol) were stirred in AcOH (10 mL) at 120 °C. After 3–6 h, the reaction mixture was cooled to room temperature and the white precipitate formed was filtered and washed using water to afford the desired compound <b>C1</b>.</div></div><div id="sec5_1_3_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i54" class="anchor-spacer"></div><h3 class="article-section__title" id="_i54"> 5-Phenyl[1,2,4]triazolo[1,5-<i>a</i>]pyrimidin-7(4<i>H</i>)-one (<b>C1</b>)</h3><div class="NLM_p last">White solid; yield, 73%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.41 (s, 1H), 8.43 (s, 1H), 7.91 (m, 2H), 7.63–7.49 (m, 3H), 6.39 (s, 1H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 156.0 (-<u class="uu">C</u>H-), 150.9 (-C-), 131.0 (-C-), 128.9 (-C-), 127.5 (-<u class="uu">C</u>H-), 97.5 (-<u class="uu">C</u>H-). HRMS (ESI): <i>m</i>/<i>z</i> calcd for C<sub>11</sub>H<sub>9</sub>N<sub>4</sub>O (M + H)<sup><b>+</b></sup>, 213.0776; found, 213.0768.</div></div><div id="sec5_1_3_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i55" class="anchor-spacer"></div><h3 class="article-section__title" id="_i55"> 5-Methyl[1,2,4]triazolo[1,5-<i>a</i>]pyrimidin-7(4<i>H</i>)-one (<b>C2</b>)</h3><div class="NLM_p last">White solid; yield, 83%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 13.20 (s, 1H), 8.20 (s, 1H), 5.84 (s, 1H), 2.33 (s, 3H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 155.8 (-<u class="uu">C</u>H-), 151.8 (-C-), 151.6 (-C-), 150.6 (-C-), 98.1 (-<u class="uu">C</u>H-), 18.6 (-<u class="uu">C</u>H<sub>3</sub>). HRMS (ESI): <i>m</i>/<i>z</i> calcd for C<sub>6</sub>H<sub>7</sub>N<sub>4</sub>O (M + H)<sup><b>+</b></sup>, 151.0619; found, 151.0612.</div></div><div id="sec5_1_3_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i56" class="anchor-spacer"></div><h3 class="article-section__title" id="_i56"> 5-Ethyl[1,2,4]triazolo[1,5-<i>a</i>]pyrimidin-7(4<i>H</i>)-one (<b>C3</b>)</h3><div class="NLM_p last">White solid; yield, 78%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 13.20 (s, 1H), 8.21 (s, 1H), 5.85 (s, 1H), 2.62 (m, 2H), 1.23 (t, <i>J</i> = 7.6 Hz, 3H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 157.2 (-<u class="uu">C</u>H-), 156.6 (-C-), 152.4 (-C-), 151.2 (-C-), 97.2 (-<u class="uu">C</u>H-), 26.2 (-<u class="uu">C</u>H<sub>2</sub>-), 12.9 (-<u class="uu">C</u>H<sub>3</sub>). HRMS (ESI): <i>m</i>/<i>z</i> calcd for C<sub>7</sub>H<sub>9</sub>N<sub>4</sub>O (M + H)<sup><b>+</b></sup>, 165.0776; found, 165.0768.</div></div><div id="sec5_1_3_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i57" class="anchor-spacer"></div><h3 class="article-section__title" id="_i57"> 5-Isopropyl[1,2,4]triazolo[1,5-<i>a</i>]pyrimidin-7(4<i>H</i>)-one (<b>C4</b>)</h3><div class="NLM_p last">White solid; yield, 74%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 13.18 (s, 1H), 8.23 (s, 1H), 5.86 (s, 1H), 2.90 (m, 1H), 1.27 (d, <i>J</i> = 6.9 Hz, 6H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 161.2 (-<u class="uu">C</u>H-), 156.7 (-C-), 152.3 (-C-), 151.2 (-C-), 95.7 (-<u class="uu">C</u>H-), 32.2 (-<u class="uu">C</u>H<sub>2</sub>-), 21.4 (-<u class="uu">C</u>H<sub>3</sub>). HRMS (ESI): <i>m</i>/<i>z</i> calcd for C<sub>8</sub>H<sub>11</sub>N<sub>4</sub>O (M + H)<sup><b>+</b></sup>, 179.0932; found, 179.0923.</div></div><div id="sec5_1_3_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i58" class="anchor-spacer"></div><h3 class="article-section__title" id="_i58"> 5-Cyclopropyl[1,2,4]triazolo[1,5-<i>a</i>]pyrimidin-7(4<i>H</i>)-one (<b>C5</b>)</h3><div class="NLM_p last">White solid; yield, 66%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 13.31 (s, 1H), 8.23 (s, 1H), 5.67 (s, 1H), 2.00–1.84 (m, 1H), 1.20–0.89 (m, 4H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 160.7 (-<u class="uu">C</u>H-), 156.3 (-C-), 151.9 (-C-), 150.8 (-C-), 95.2 (-<u class="uu">C</u>H-), 31.8 (-<u class="uu">C</u>H-), 20.9 (-<u class="uu">C</u>H<sub>2</sub>-). HRMS (ESI): <i>m</i>/<i>z</i> calcd for C<sub>8</sub>H<sub>9</sub>N<sub>4</sub>O (M + H)<sup><b>+</b></sup>, 177.0776; found, 177.0767.</div></div><div id="sec5_1_3_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i59" class="anchor-spacer"></div><h3 class="article-section__title" id="_i59"> 4,5,6,7-Tetrahydro-8<i>H</i>-cyclopenta[<i>d</i>][1,2,4]triazolo[1,5-<i>a</i>]pyrimidin-8-one (<b>C6</b>)</h3><div class="NLM_p last">White solid; yield, 82%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 13.39 (s, 1H), 8.16 (s, 1H), 2.91 (t, <i>J</i> = 7.7 Hz, 2H), 2.68 (t, <i>J</i> = 7.3 Hz, 2H), 2.17–1.99 (m, 2H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 155.8 (-<u class="uu">C</u>H-), 154.9 (-C-), 152.1 (-C-), 151.4 (-C-), 110.4 (-C-), 31.8 (-<u class="uu">C</u>H<sub>2</sub>-), 27.2 (-<u class="uu">C</u>H<sub>2</sub>-), 22.2 (-<u class="uu">C</u>H<sub>2</sub>-). HRMS (ESI): <i>m</i>/<i>z</i> calcd for C<sub>8</sub>H<sub>9</sub>N<sub>4</sub>O (M + H)<sup><b>+</b></sup>, 177.0776; found, 177.0767.</div></div><div id="sec5_1_3_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i60" class="anchor-spacer"></div><h3 class="article-section__title" id="_i60"> 5-Methylpyrazolo[1,5-<i>a</i>]pyrimidin-7(4<i>H</i>)-one (<b>C7</b>)</h3><div class="NLM_p last">White solid; yield, 63%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.29 (s, 1H), 7.83 (d, <i>J</i> = 1.9 Hz, 1H), 6.09 (d, <i>J</i> = 1.9 Hz, 1H), 5.57 (s, 1H), 2.29 (s, 3H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 156.3 (-C-), 150.1 (-C-), 142.6 (-C-), 141.6 (-<u class="uu">C</u>H-), 94.8 (-<u class="uu">C</u>H-), 88.2 (-<u class="uu">C</u>H-), 18.5 (-<u class="uu">C</u>H<sub>3</sub>). HRMS (ESI): <i>m</i>/<i>z</i> calcd for C<sub>7</sub>H<sub>8</sub>N<sub>3</sub>O (M + H)<sup><b>+</b></sup>, 150.0667; found, 150.0669.</div></div><div id="sec5_1_3_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i61" class="anchor-spacer"></div><h3 class="article-section__title" id="_i61"> 5-Methyl-2-(propylthio)[1,2,4]triazolo[1,5-<i>a</i>]pyrimidin-7(4<i>H</i>)-one (<b>C8</b>)</h3><div class="NLM_p last">White solid; yield, 64%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 13.14 (s, 1H), 5.79 (s, 1H), 3.12 (t, <i>J</i> = 7.2 Hz, 2H), 2.29 (s, 3H), 1.73 (m, 2H), 0.99 (t, <i>J</i> = 7.3 Hz, 3H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 162.3 (-C-), 154.8 (-C-), 151.1 (-C-), 150.5 (-C-), 98.4 (-<u class="uu">C</u>H-), 32.5 (-<u class="uu">C</u>H<sub>2</sub>-), 22.5 (-<u class="uu">C</u>H<sub>2</sub>-), 18.4 (-<u class="uu">C</u>H<sub>3</sub>), 13.0 (-<u class="uu">C</u>H<sub>3</sub>). HRMS (ESI): <i>m</i>/<i>z</i> calcd for C<sub>9</sub>H<sub>11</sub>N<sub>4</sub>OS (M – H)<sup>−</sup>, 223.0653; found, 223.0665.</div></div><div id="sec5_1_3_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i62" class="anchor-spacer"></div><h3 class="article-section__title" id="_i62"> 2-(Allylthio)-5-methyl[1,2,4]triazolo[1,5-<i>a</i>]pyrimidin-7(4<i>H</i>)-one (<b>C9</b>)</h3><div class="NLM_p last">White solid; yield, 69%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 13.17 (s, 1H), 6.00 (m, 1H), 5.81 (s, 1H), 5.33 (m, 1H), 5.14 (d, <i>J</i> = 10.0 Hz, 1H), 3.84 (d, <i>J</i> = 6.8 Hz, 2H), 2.30 (s, 3H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 161.7 (-C-), 154.8 (-C-), 151.1 (-C-), 150.7 (-C-), 133.5 (-<u class="uu">C</u>H-), 118.1 (-<u class="uu">C</u>H<sub>2</sub>-), 98.5 (-<u class="uu">C</u>H-), 33.2 (-<u class="uu">C</u>H<sub>2</sub>-), 18.5 (-<u class="uu">C</u>H<sub>3</sub>). HRMS (ESI): <i>m</i>/<i>z</i> calcd for C<sub>9</sub>H<sub>9</sub>N<sub>4</sub>OS (M – H)<sup>−</sup>, 221.0497; found, 221.0499.</div></div><div id="sec5_1_3_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i63" class="anchor-spacer"></div><h3 class="article-section__title" id="_i63"> 5-Methyl-2-(prop-2-yn-1-ylthio)[1,2,4]triazolo[1,5-<i>a</i>]pyrimidin-7(4<i>H</i>)-one (<b>C10</b>)</h3><div class="NLM_p last">White solid; yield, 65%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 13.22 (s, 1H), 5.81 (s, 1H), 4.00 (d, <i>J</i> = 2.6 Hz, 2H), 3.19 (t, <i>J</i> = 2.5 Hz, 1H), 2.29 (s, 3H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 160.9 (-C-), 154.7 (-C-), 151.2 (-C-), 150.7 (-C-), 98.5 (-<u class="uu">C</u>H-), 79.9 (-C-), 73.8 (-<u class="uu">C</u>H-), 19.0 (-<u class="uu">C</u>H<sub>2</sub>-), 18.4 (-<u class="uu">C</u>H<sub>3</sub>). HRMS (ESI): <i>m</i>/<i>z</i> calcd for C<sub>9</sub>H<sub>7</sub>N<sub>4</sub>OS (M – H)<sup>−</sup>, 219.0340; found, 219.0343.</div></div><div id="sec5_1_3_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i64" class="anchor-spacer"></div><h3 class="article-section__title" id="_i64"> 2-(Isopropylthio)-5-methyl[1,2,4]triazolo[1,5-<i>a</i>]pyrimidin-7(4<i>H</i>)-one (<b>C11</b>)</h3><div class="NLM_p last">White solid; yield, 81%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 13.17 (s, 1H), 5.80 (s, 1H), 3.82 (m, 1H), 2.30 (s, 3H), 1.41 (d, <i>J</i> = 6.8 Hz, 6H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 162.4 (-C-), 155.4 (-C-), 151.5 (-C-), 151.1 (-C-), 98.9 (-<u class="uu">C</u>H-), 36.8 (-<u class="uu">C</u>H-), 23.7 (-<u class="uu">C</u>H<sub>3</sub>), 19.0 (-<u class="uu">C</u>H<sub>3</sub>). HRMS (ESI): <i>m</i>/<i>z</i> calcd for C<sub>9</sub>H<sub>13</sub>N<sub>4</sub>OS (M + H)<sup><b>+</b></sup>, 225.0810; found, 225.0800.</div></div><div id="sec5_1_3_12" class="NLM_sec NLM_sec_level_2"><div id="ac_i65" class="anchor-spacer"></div><h3 class="article-section__title" id="_i65"> 2-((Cyclopropylmethyl)thio)-5-methyl[1,2,4]triazolo[1,5-<i>a</i>]pyrimidin-7(4<i>H</i>)-one (<b>C12</b>)</h3><div class="NLM_p last">White solid; yield, 76%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 13.15 (s, 1H), 5.80 (s, 1H), 3.11 (d, <i>J</i> = 7.2 Hz, 2H), 2.29 (s, 3H), 1.20 (m, 1H), 0.73–0.44 (m, 2H), 0.39–0.20 (m, 2H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 163.1 (-C-), 155.3 (-C-), 151.6 (-C-), 151.1 (-C-), 99.0 (-<u class="uu">C</u>H-), 36.7 (-<u class="uu">C</u>H-), 19.0 (-<u class="uu">C</u>H<sub>3</sub>), 11.4 (-<u class="uu">C</u>H<sub>2</sub>-), 6.1 (-<u class="uu">C</u>H<sub>2</sub>-). HRMS (ESI): <i>m</i>/<i>z</i> calcd for C<sub>10</sub>H<sub>13</sub>N<sub>4</sub>OS (M + H)+, 237.0810; found, 237.0800.</div></div><div id="sec5_1_3_13" class="NLM_sec NLM_sec_level_2"><div id="ac_i66" class="anchor-spacer"></div><h3 class="article-section__title" id="_i66"> 2-((Cyclohexylmethyl)thio)-5-methyl[1,2,4]triazolo[1,5-<i>a</i>]pyrimidin-7(4<i>H</i>)-one (<b>C13</b>)</h3><div class="NLM_p last">White solid; yield, 79%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 13.15 (s, 1H), 5.79 (s, 1H), 3.07 (d, <i>J</i> = 6.9 Hz, 2H), 2.29 (s, 3H), 1.82 (d, <i>J</i> = 12.7 Hz, 2H), 1.73–1.65 (m, 2H), 1.64–1.54 (m, 2H), 1.26–1.09 (m, 3H), 1.07–0.94 (m, 2H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 163.1 (-C-), 155.3 (-C-), 151.6 (-C-), 151.1 (-C-), 99.0 (-<u class="uu">C</u>H-), 37.9 (-<u class="uu">C</u>H-), 37.7 (-<u class="uu">C</u>H<sub>2</sub>-), 32.3 (-<u class="uu">C</u>H<sub>2</sub>-), 26.3 (-<u class="uu">C</u>H<sub>2</sub>-), 25.9 (-<u class="uu">C</u>H<sub>2</sub>-), 19.0 (-<u class="uu">C</u>H<sub>3</sub>). HRMS (ESI): <i>m</i>/<i>z</i> calcd for C<sub>13</sub>H<sub>19</sub>N<sub>4</sub>OS (M + H)<sup><b>+</b></sup>, 279.1279; found, 279.1269.</div></div><div id="sec5_1_3_14" class="NLM_sec NLM_sec_level_2"><div id="ac_i67" class="anchor-spacer"></div><h3 class="article-section__title" id="_i67"> 5-Methyl-2-((naphthalen-1-ylmethyl)thio)[1,2,4]triazolo[1,5-<i>a</i>]pyrimidin-7(4<i>H</i>)-one (<b>C14</b>)</h3><div class="NLM_p last">White solid; yield, 61%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 13.22 (s, 1H), 8.18 (d, <i>J</i> = 8.2 Hz, 1H), 7.97 (d, <i>J</i> = 7.8 Hz, 1H), 7.89 (d, <i>J</i> = 8.2 Hz, 1H), 7.68 (d, <i>J</i> = 6.9 Hz, 1H), 7.64–7.53 (m, 2H), 7.49–7.42 (m, 1H), 5.84 (s, 1H), 4.95 (s, 2H), 2.31 (s, 3H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ162.5 (-C-), 155.4 (-C-), 151.7 (-C-), 151.2 (-C-), 134.0 (-C-), 133.0 (-C-), 131.4 (-C-), 129.2 (-<u class="uu">C</u>H-), 128.8 (-<u class="uu">C</u>H-), 128.1 (-<u class="uu">C</u>H-), 127.0 (-<u class="uu">C</u>H-), 126.5 (-<u class="uu">C</u>H-), 125.9 (-<u class="uu">C</u>H-), 124.2 (-<u class="uu">C</u>H-), 99.1 (-<u class="uu">C</u>H-), 33.1 (-<u class="uu">C</u>H<sub>2</sub>-), 19.0 (-<u class="uu">C</u>H<sub>3</sub>). HRMS (ESI): <i>m</i>/<i>z</i> calcd for C<sub>17</sub>H<sub>15</sub>N<sub>4</sub>OS (M + H)<sup><b>+</b></sup>, 323.0966; found, 323.0951.</div></div><div id="sec5_1_3_15" class="NLM_sec NLM_sec_level_2"><div id="ac_i68" class="anchor-spacer"></div><h3 class="article-section__title" id="_i68"> 2-(Benzylthio)-5-methyl[1,2,4]triazolo[1,5-<i>a</i>]pyrimidin-7(4<i>H</i>)-one (<b>C15</b>)</h3><div class="NLM_p last">White solid; yield, 85%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 13.17 (s, 1H), 7.44 (d, <i>J</i> = 7.1 Hz, 2H), 7.32 (t, <i>J</i> = 7.3 Hz, 2H), 7.26 (d, <i>J</i> = 7.2 Hz, 1H), 5.80 (s, 1H), 4.43 (s, 2H), 2.29 (s, 3H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 161.9 (-C-), 154.8 (-C-), 151.1 (-C-), 150.6 (-C-), 137.4 (-C-), 128.8 (−CH-), 128.4 (-<u class="uu">C</u>H-), 127. Two (-<u class="uu">C</u>H-), 98.5 (-<u class="uu">C</u>H-), 34.5 (-<u class="uu">C</u>H<sub>2</sub>-), 18.4 (-<u class="uu">C</u>H<sub>3</sub>). HRMS (ESI): <i>m</i>/<i>z</i> calcd for C<sub>13</sub>H<sub>11</sub>N<sub>4</sub>OS (M – H)<sup>−</sup>, 271.0654; found, 271.0661.</div></div><div id="sec5_1_3_16" class="NLM_sec NLM_sec_level_2"><div id="ac_i69" class="anchor-spacer"></div><h3 class="article-section__title" id="_i69"> 5-Methyl-2-(phenethylthio)[1,2,4]triazolo[1,5-<i>a</i>]pyrimidin-7(4<i>H</i>)-one (<b>C16</b>)</h3><div class="NLM_p last">White solid; yield, 72%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 13.18 (s, 1H), 7.61–6.94 (m, 5H), 5.81 (s, 1H), 3.42 (d, <i>J</i> = 7.2 Hz, 2H), 3.03 (t, <i>J</i> = 7.5 Hz, 2H), 2.30 (s, 3H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 162.7 (-C-), 155.3 (-C-), 151.7 (-C-), 151.1 (-C-), 140.4 (-C-), 129.0 (-<u class="uu">C</u>H-), 128.8 (-<u class="uu">C</u>H-), 126.8 (-<u class="uu">C</u>H-), 99.0 (-<u class="uu">C</u>H-), 35.6 (-<u class="uu">C</u>H<sub>2</sub>-), 32.4 (-<u class="uu">C</u>H<sub>2</sub>-), 19.0 (-<u class="uu">C</u>H<sub>3</sub>). HRMS (ESI): <i>m</i>/<i>z</i> calcd for C<sub>14</sub>H<sub>15</sub>N<sub>4</sub>OS (M + H)<sup><b>+</b></sup>, 287.0966; found, 287.0955.</div></div><div id="sec5_1_3_17" class="NLM_sec NLM_sec_level_2"><div id="ac_i70" class="anchor-spacer"></div><h3 class="article-section__title" id="_i70"> 5-Methyl-2-((3-phenylpropyl)thio)[1,2,4]triazolo[1,5-<i>a</i>]pyrimidin-7(4<i>H</i>)-one (<b>C17</b>)</h3><div class="NLM_p last">White solid; yield, 62%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 13.32 (s, 1H), 7.47–7.06 (m, 5H), 5.80 (s, 1H), 3.15 (t, <i>J</i> = 7.2 Hz, 2H), 2.79–2.69 (m, 2H), 2.29 (s, 3H), 2.08–1.95 (m, 2H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 162.7 (-C-), 155.3 (-C-), 151.6 (-C-), 151.1 (-C-), 141.5 (-C-), 128.8 (-<u class="uu">C</u>H-), 128.8 (-<u class="uu">C</u>H-), 126.4 (-<u class="uu">C</u>H-), 99.0 (-<u class="uu">C</u>H-), 34.4 (-<u class="uu">C</u>H<sub>2</sub>-), 31.1 (-<u class="uu">C</u>H<sub>2</sub>-), 30.6 (-<u class="uu">C</u>H<sub>2</sub>-), 19.0 (-<u class="uu">C</u>H<sub>3</sub>). HRMS (ESI): <i>m</i>/<i>z</i> calcd for C<sub>15</sub>H<sub>15</sub>N<sub>4</sub>OS (M – H)<sup>−</sup>, 299.0966; found, 299.0974.</div></div><div id="sec5_1_3_18" class="NLM_sec NLM_sec_level_2"><div id="ac_i71" class="anchor-spacer"></div><h3 class="article-section__title" id="_i71"> 2-(((5-Chlorobenzo[<i>b</i>]thiophen-3-yl)methyl)thio)-5-methyl[1,2,4]triazolo[1,5-<i>a</i>]pyrimidin-7(4<i>H</i>)-one (<b>C18</b>)</h3><div class="NLM_p last">White solid; yield, 73%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 13.21 (s, 1H), 8.05 (t, <i>J</i> = 5.7 Hz, 2H), 7.83 (s, 1H), 7.43 (m, <i>J</i> = 8.6, 2.0 Hz, 1H), 5.83 (s, 1H), 4.72 (s, 2H), 2.30 (s, 3H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 161.7 (-C-), 154.8 (-C-), 151.2 (-C-), 150.8 (-C-), 139.1 (-C-), 138.2 (-C-), 131.0 (-C-), 129.6 (-C-), 128.3 (-<u class="uu">C</u>H-), 124.7 (-<u class="uu">C</u>H-), 124.6 (-<u class="uu">C</u>H-), 121.6 (-<u class="uu">C</u>H-), 98.5 (-<u class="uu">C</u>H-), 27.9 (-<u class="uu">C</u>H<sub>2</sub>-), 18.5 (-<u class="uu">C</u>H<sub>3</sub>). HRMS (ESI): <i>m</i>/<i>z</i> calcd for C<sub>15</sub>H<sub>12</sub>ClN<sub>4</sub>OS<sub>2</sub> (M + H)<sup><b>+</b></sup>, 363.0141; found, 363.0127.</div></div><div id="sec5_1_3_19" class="NLM_sec NLM_sec_level_2"><div id="ac_i72" class="anchor-spacer"></div><h3 class="article-section__title" id="_i72"> 5-Methyl-2-(((2-oxo-2<i>H</i>-benzo[<i>h</i>]chromen-4-yl)methyl)thio)[1,2,4]triazolo[1,5-<i>a</i>]pyrimidin-7(4<i>H</i>)-one (<b>C19</b>)</h3><div class="NLM_p last">black solid, yield: 66%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 13.22 (s, 1H), 11.97 (s, 1H), 8.42–8.33 (m, 1H), 8.07 (m, 1H), 7.99 (d, <i>J</i> = 8.8 Hz, 1H), 7.92 (t, <i>J</i> = 7.4 Hz, 1H), 7.79–7.69 (m, 2H), 6.70 (s, 1H), 5.83 (s, 1H), 4.77 (s, 2H), 2.29 (s, 3H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 172.0 (-C-), 160.8 (-C-), 159.5 (-C-), 154.8 (-C-), 152.1 (-C-), 151.3 (-C-), 150.9 (-C-), 150.2 (-C-), 134.3 (-<u class="uu">C</u>H-), 128.9 (-<u class="uu">C</u>H-), 128.0 (-<u class="uu">C</u>H-), 127.5 (-C-), 124.1 (-<u class="uu">C</u>H-), 122.3 (-<u class="uu">C</u>H-), 121.6 (-<u class="uu">C</u>H-), 121.0 (-C-), 114.4 (-<u class="uu">C</u>H-), 113.6 (-<u class="uu">C</u>H-), 98.6 (-<u class="uu">C</u>H-), 31.2 (-<u class="uu">C</u>H<sub>2</sub>-), 18.5 (-<u class="uu">C</u>H<sub>3</sub>). HRMS (ESI): <i>m</i>/<i>z</i> calcd for C<sub>20</sub>H<sub>15</sub>N<sub>4</sub>O<sub>3</sub>S (M + H)<sup><b>+</b></sup>, 391.0864; found, 391.0851.</div></div><div id="sec5_1_3_20" class="NLM_sec NLM_sec_level_2"><div id="ac_i73" class="anchor-spacer"></div><h3 class="article-section__title" id="_i73"> 2-(Benzylthio)-5-phenyl[1,2,4]triazolo[1,5-<i>a</i>]pyrimidin-7(4<i>H</i>)-one (<b>C20</b>)</h3><div class="NLM_p last">White solid; yield, 74%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 13.49 (s, 1H), 7.88–7.82 (m, 2H), 7.62–7.53 (m, 3H), 7.48 (d, <i>J</i> = 7.2 Hz, 2H), 7.34 (t, <i>J</i> = 7.3 Hz, 2H), 7.30–7.24 (m, 1H), 6.29 (s, 1H), 4.49 (s, 2H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 154.9 (-C-), 151.7 (-C-), 137.3 (-C-), 131.7 (-C-), 131.2 (-<u class="uu">C</u>H-), 129.0 (-<u class="uu">C</u>H-), 128.9 (-<u class="uu">C</u>H-), 128.5 (-<u class="uu">C</u>H-), 127.6 (-<u class="uu">C</u>H-), 127.4 (-<u class="uu">C</u>H-), 97.7 (-<u class="uu">C</u>H-), 34.6 (-<u class="uu">C</u>H<sub>2</sub>-). HRMS (ESI): <i>m</i>/<i>z</i> calcd for C<sub>18</sub>H<sub>15</sub>N<sub>4</sub>OS (M + H)<sup><b>+</b></sup>, 335.0966; found, 335.0950.</div></div><div id="sec5_1_3_21" class="NLM_sec NLM_sec_level_2"><div id="ac_i74" class="anchor-spacer"></div><h3 class="article-section__title" id="_i74"> 2-(Benzylthio)-5-ethyl[1,2,4]triazolo[1,5-<i>a</i>]pyrimidin-7(4<i>H</i>)-one (<b>C21</b>)</h3><div class="NLM_p last">White solid; yield, 78%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 13.17 (s, 1H), 7.46 (d, <i>J</i> = 7.3 Hz, 2H), 7.33 (t, <i>J</i> = 7.3 Hz, 2H), 7.27 (m, 1H), 5.82 (s, 1H), 4.45 (s, 2H), 2.58 (q, <i>J</i> = 7.5 Hz, 2H), 1.21 (t, <i>J</i> = 7.5 Hz, 3H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 162.6 (-C-), 156.2 (-C-), 155.5 (-C-), 151.7 (-C-), 137.9 (-C-), 129.3 (-<u class="uu">C</u>H-), 129.0 (-<u class="uu">C</u>H-), 127.8 (-<u class="uu">C</u>H-), 97.6 (-<u class="uu">C</u>H-), 35.1 (-<u class="uu">C</u>H<sub>2</sub>-), 26.0 (-<u class="uu">C</u>H<sub>2</sub>-), 12.8 (-<u class="uu">C</u>H<sub>3</sub>). HRMS (ESI): <i>m</i>/<i>z</i> calcd for C<sub>14</sub>H<sub>15</sub>N<sub>4</sub>OS (M + H)<sup><b>+</b></sup>, 287.0966; found, 287.0949.</div></div><div id="sec5_1_3_22" class="NLM_sec NLM_sec_level_2"><div id="ac_i75" class="anchor-spacer"></div><h3 class="article-section__title" id="_i75"> 5-Phenyl-2-(prop-2-yn-1-ylthio)[1,2,4]triazolo[1,5-<i>a</i>]pyrimidin-7(4<i>H</i>)-one (<b>C22</b>)</h3><div class="NLM_p last">White solid; yield, 69%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 13.54 (s, 1H), 7.84 (m, 2H), 7.66–7.46 (m, 3H), 6.30 (s, 1H), 4.06 (d, <i>J</i> = 2.6 Hz, 2H), 3.24 (s, 1H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 154.9 (-C-), 151.8 (-C-), 131.7 (-C-), 131.3 (-C-), 129.0 (-<u class="uu">C</u>H-), 127.6 (-<u class="uu">C</u>H-), 97.8 (-<u class="uu">C</u>H-), 80.0 (-C-), 74.0 (-<u class="uu">C</u>H-), 19.1 (-<u class="uu">C</u>H<sub>2</sub>-). HRMS (ESI): <i>m</i>/<i>z</i> calcd for C<sub>14</sub>H<sub>11</sub>N<sub>4</sub>OS (M + H)<sup><b>+</b></sup>, 283.0653; found, 283.0640.</div></div><div id="sec5_1_3_23" class="NLM_sec NLM_sec_level_2"><div id="ac_i76" class="anchor-spacer"></div><h3 class="article-section__title" id="_i76"> 5-Methyl-2-((4-nitrobenzyl)thio)[1,2,4]triazolo[1,5-<i>a</i>]pyrimidin-7(4<i>H</i>)-one (<b>C23</b>)</h3><div class="NLM_p last">yellow solid, yield: 78%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 13.21 (s, 1H), 8.18 (d, <i>J</i> = 8.7 Hz, 2H), 7.73 (d, <i>J</i> = 8.7 Hz, 2H), 5.82 (s, 1H), 4.57 (s, 2H), 2.29 (s, 3H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 161.9 (-C-), 155.3 (-C-), 151.7 (-C-), 151.3 (-C-), 147.1 (-C-), 146.5 (-C-), 130.5 (-<u class="uu">C</u>H-), 124.0 (-<u class="uu">C</u>H-), 99.1 (-<u class="uu">C</u>H-), 34.2 (-<u class="uu">C</u>H<sub>2</sub>-), 19.0 (-<u class="uu">C</u>H<sub>3</sub>). HRMS (ESI): <i>m</i>/<i>z</i> calcd for C<sub>13</sub>H<sub>12</sub>N<sub>5</sub>O<sub>3</sub>S (M + H)<sup><b>+</b></sup>, 318.0660; found, 318.0646.</div></div><div id="sec5_1_3_24" class="NLM_sec NLM_sec_level_2"><div id="ac_i77" class="anchor-spacer"></div><h3 class="article-section__title" id="_i77"> 5-Methyl-2-((2-nitrobenzyl)thio)[1,2,4]triazolo[1,5-<i>a</i>]pyrimidin-7(4<i>H</i>)-one (<b>C24</b>)</h3><div class="NLM_p last">yellow solid, yield: 76%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 13.20 (s, 1H), 8.07 (d, <i>J</i> = 8.0 Hz, 1H), 7.78 (d, <i>J</i> = 7.0 Hz, 1H), 7.71 (t, <i>J</i> = 7.3 Hz, 1H), 7.62–7.51 (m, 1H), 5.82 (s, 1H), 4.72 (s, 2H), 2.30 (s, 3H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 162.0 (-C-), 155.3 (-C-), 151.7 (-C-), 151.3 (-C-), 148.5 (-C-), 134.4 (-C-), 133.4 (-<u class="uu">C</u>H-), 132.7 (-<u class="uu">C</u>H-), 129.6 (-<u class="uu">C</u>H-), 125.6 (-<u class="uu">C</u>H-), 99.1 (-<u class="uu">C</u>H-), 32.5 (-<u class="uu">C</u>H<sub>2</sub>-), 19.0 (-<u class="uu">C</u>H<sub>3</sub>). HRMS (ESI): <i>m</i>/<i>z</i> calcd for C<sub>13</sub>H<sub>12</sub>N<sub>5</sub>O<sub>3</sub>S (M + H)<sup><b>+</b></sup>, 318.0660; found, 318.0644.</div></div><div id="sec5_1_3_25" class="NLM_sec NLM_sec_level_2"><div id="ac_i78" class="anchor-spacer"></div><h3 class="article-section__title" id="_i78"> 2-((4-Fluorobenzyl)thio)-5-methyl[1,2,4]triazolo[1,5-<i>a</i>]pyrimidin-7(4<i>H</i>)-one (<b>C25</b>)</h3><div class="NLM_p last">White solid; yield, 73%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ13.19 (s, 1H), 7.66–7.41 (m, 2H), 7.31–6.96 (m, 2H), 5.81 (s, 1H), 4.43 (s, 2H), 2.29 (s, 3H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 163.1 (-C-), 162.4 (-C-), 160.7 (-C-), 155.3 (-C-), 151.7 (-C-), 151.2 (-C-), 134.3 (-C-), 134.3 (-C-), 131.4 (-<u class="uu">C</u>H-), 131.3 (-<u class="uu">C</u>H-), 115.8 (-<u class="uu">C</u>H-), 115.6 (-<u class="uu">C</u>H-), 99.0 (-<u class="uu">C</u>H-), 34.2 (-<u class="uu">C</u>H<sub>2</sub>-), 19.0 (-<u class="uu">C</u>H<sub>3</sub>). HRMS (ESI): <i>m</i>/<i>z</i> calcd for C<sub>13</sub>H<sub>12</sub>FN<sub>4</sub>OS (M + H)<sup><b>+</b></sup>, 291.0715; found, 291.0702.</div></div><div id="sec5_1_3_26" class="NLM_sec NLM_sec_level_2"><div id="ac_i79" class="anchor-spacer"></div><h3 class="article-section__title" id="_i79"> 2-((4-Chlorobenzyl)thio)-5-methyl[1,2,4]triazolo[1,5-<i>a</i>]pyrimidin-7(4<i>H</i>)-one (<b>C26</b>)</h3><div class="NLM_p last">White solid; yield, 71%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 13.20 (s, 1H), 7.48 (d, <i>J</i> = 7.5 Hz, 2H), 7.38 (d, <i>J</i> = 7.3 Hz, 2H), 5.82 (s, 1H), 4.44 (s, 2H), 2.30 (s, 3H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 162.2 (-C-), 155.3 (-C-), 151.7 (-C-), 151.2 (-C-), 137.3 (-C-), 132.4 (-C-), 131.2 (-<u class="uu">C</u>H-), 128.9 (-<u class="uu">C</u>H-), 99.0 (-<u class="uu">C</u>H-), 34.2 (-<u class="uu">C</u>H<sub>2</sub>-), 19.0 (-<u class="uu">C</u>H<sub>3</sub>). HRMS (ESI): <i>m</i>/<i>z</i> calcd for C<sub>13</sub>H<sub>12</sub>ClN<sub>4</sub>OS (M + H)<sup><b>+</b></sup>, 307.0420; found, 307.0405.</div></div><div id="sec5_1_3_27" class="NLM_sec NLM_sec_level_2"><div id="ac_i80" class="anchor-spacer"></div><h3 class="article-section__title" id="_i80"> 2-((4-Bromobenzyl)thio)-5-methyl[1,2,4]triazolo[1,5-<i>a</i>]pyrimidin-7(4<i>H</i>)-one (<b>C27</b>)</h3><div class="NLM_p last">White solid; yield, 64%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 13.19 (s, 1H), 7.52 (d, <i>J</i> = 8.4 Hz, 2H), 7.42 (d, <i>J</i> = 8.4 Hz, 2H), 5.82 (s, 1H), 4.41 (s, 2H), 2.29 (s, 3H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 162.2 (-C-), 155.3 (-C-), 151.7 (-C-), 151.2 (-C-), 137.7 (-C-), 131.8 (-C-), 131.5 (-<u class="uu">C</u>H-), 120.9 (-<u class="uu">C</u>H-), 99.0 (-<u class="uu">C</u>H-), 34.3 (-<u class="uu">C</u>H<sub>2</sub>-), 19.0 (-<u class="uu">C</u>H<sub>3</sub>). HRMS (ESI): <i>m</i>/<i>z</i> calcd for C<sub>13</sub>H<sub>12</sub>BrN<sub>4</sub>OS (M + H)<sup><b>+</b></sup>, 350.9915; found, 350.9899.</div></div><div id="sec5_1_3_28" class="NLM_sec NLM_sec_level_2"><div id="ac_i81" class="anchor-spacer"></div><h3 class="article-section__title" id="_i81"> 2-((2-Bromobenzyl)thio)-5-methyl[1,2,4]triazolo[1,5-<i>a</i>]pyrimidin-7(4<i>H</i>)-one (<b>C28</b>)</h3><div class="NLM_p last">White solid; yield, 73%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 13.21 (s, 1H), 7.71–7.59 (m, 2H), 7.36 (t, <i>J</i> = 7.4 Hz, 1H), 7.25 (m, 1H), 5.83 (s, 1H), 4.52 (s, 2H), 2.30 (s, 3H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 162.0 (-C-), 155.3 (-C-), 151.7 (-C-), 151.3 (-C-), 136.8 (-C-), 133.3 (-C-), 131.7 (-<u class="uu">C</u>H-), 130.2 (-<u class="uu">C</u>H-), 128.4 (-<u class="uu">C</u>H-), 124.5 (-<u class="uu">C</u>H-), 99.1 (-<u class="uu">C</u>H-), 35.9 (-<u class="uu">C</u>H<sub>2</sub>-), 19.0 (-<u class="uu">C</u>H<sub>3</sub>). HRMS (ESI): <i>m</i>/<i>z</i> calcd for C<sub>13</sub>H<sub>12</sub>BrN<sub>4</sub>OS (M + H)<sup><b>+</b></sup>, 350.9915; found, 350.9900.</div></div><div id="sec5_1_3_29" class="NLM_sec NLM_sec_level_2"><div id="ac_i82" class="anchor-spacer"></div><h3 class="article-section__title" id="_i82"> 2-((3-Bromobenzyl)thio)-5-methyl[1,2,4]triazolo[1,5-<i>a</i>]pyrimidin-7(4<i>H</i>)-one (<b>C29</b>)</h3><div class="NLM_p last">White solid; yield, 76%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 13.19 (s, 1H), 7.68 (s, 1H), 7.55–7.40 (m, 2H), 7.28 (t, <i>J</i> = 7.8 Hz, 1H), 5.81 (s, 1H), 4.43 (s, 2H), 2.29 (s, 3H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 162.2 (-C-), 155.3 (-C-), 151.7 (-C-), 151.2 (-C-), 141.1 (-C-), 132.0 (-C-), 131.1 (-<u class="uu">C</u>H-), 130.6 (-<u class="uu">C</u>H-), 128.4 (-<u class="uu">C</u>H-), 122.0 (-<u class="uu">C</u>H-), 99.1 (-<u class="uu">C</u>H-), 34.2 (-<u class="uu">C</u>H<sub>2</sub>-), 19.0 (-<u class="uu">C</u>H<sub>3</sub>). HRMS (ESI): <i>m</i>/<i>z</i> calcd for C<sub>13</sub>H<sub>12</sub>BrN<sub>4</sub>OS (M + H)<sup><b>+</b></sup>, 350.9915; found, 350.9903.</div></div><div id="sec5_1_3_30" class="NLM_sec NLM_sec_level_2"><div id="ac_i83" class="anchor-spacer"></div><h3 class="article-section__title" id="_i83"> 5-Methyl-2-((4-methylbenzyl)thio)[1,2,4]triazolo[1,5-<i>a</i>]pyrimidin-7(4<i>H</i>)-one (<b>C30</b>)</h3><div class="NLM_p last">White solid; yield, 80%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 13.18 (s, 1H), 7.33 (d, <i>J</i> = 7.9 Hz, 2H), 7.12 (d, <i>J</i> = 7.9 Hz, 2H), 5.81 (s, 1H), 4.39 (s, 2H), 2.28 (d, <i>J</i> = 10.2 Hz, 6H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 162.6 (-C-), 155.3 (-C-), 151.6 (-C-), 151.2 (-C-), 137.0 (-C-), 134.7 (-C-), 129.5 (-<u class="uu">C</u>H-), 129.3 (-<u class="uu">C</u>H-), 99.0 (-<u class="uu">C</u>H-), 34.9 (-<u class="uu">C</u>H<sub>2</sub>-), 21.2 (-<u class="uu">C</u>H<sub>3</sub>), 19.0 (-<u class="uu">C</u>H<sub>3</sub>). HRMS (ESI): <i>m</i>/<i>z</i> calcd for C<sub>14</sub>H<sub>15</sub>N<sub>4</sub>OS (M + H)<sup><b>+</b></sup>, 287.0966; found, 287.0952.</div></div><div id="sec5_1_3_31" class="NLM_sec NLM_sec_level_2"><div id="ac_i84" class="anchor-spacer"></div><h3 class="article-section__title" id="_i84"> 5-Methyl-2-((4-(trifluoromethyl)benzyl)thio)[1,2,4]triazolo[1,5-<i>a</i>]pyrimidin-7(4<i>H</i>)-one (<b>C31</b>)</h3><div class="NLM_p last">White solid; yield, 67%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 13.21 (s, 1H), 7.69 (s, 4H), 5.82 (s, 1H), 4.53 (s, 2H), 2.30 (s, 3H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 162.1 (-C-), 155.3 (-C-), 151.7 (-C-), 151.23 (-C-), 143.3 (-C-), 130.1 (-C-), 125.8 (-<u class="uu">C</u>H-), 125.7 (-<u class="uu">C</u>H-), 99.1 (-<u class="uu">C</u>H-), 34.3 (-<u class="uu">C</u>H<sub>2</sub>-), 19.0 (-<u class="uu">C</u>H<sub>3</sub>). HRMS (ESI): <i>m</i>/<i>z</i> calcd for C<sub>14</sub>H<sub>12</sub>F<sub>3</sub>N<sub>4</sub>OS (M + H)<sup><b>+</b></sup>, 341.0683; found, 341.0668.</div></div><div id="sec5_1_3_32" class="NLM_sec NLM_sec_level_2"><div id="ac_i85" class="anchor-spacer"></div><h3 class="article-section__title" id="_i85"> 5-Methyl-2-((2-(trifluoromethyl)benzyl)thio)[1,2,4]triazolo[1,5-<i>a</i>]pyrimidin-7(4<i>H</i>)-one (<b>C32</b>)</h3><div class="NLM_p last">White solid; yield, 65%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 13.23 (s, 1H), 7.77 (t, <i>J</i> = 7.6 Hz, 2H), 7.67 (t, <i>J</i> = 7.6 Hz, 1H), 7.53 (t, <i>J</i> = 7.6 Hz, 1H), 5.84 (s, 1H), 4.62 (s, 2H), 2.31 (s, 3H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 172.5 (-C-), 162.0 (-C-), 155.3 (-C-), 151.8 (-C-), 151.3 (-C-), 135.8 (-C-), 133.4 (-C-), 132.3 (-C-), 128.8 (-C-), 127.7 (-C-), 127.4 (-<u class="uu">C</u>H-), 126.7 (-<u class="uu">C</u>H-), 126.6 (-<u class="uu">C</u>H-), 126.2 (-<u class="uu">C</u>H-), 123.4 (-<u class="uu">C</u>H-), 99.1 (-<u class="uu">C</u>H-), 31.9 (-<u class="uu">C</u>H<sub>2</sub>-), 19.0 (−CH<sub>3</sub>). HRMS (ESI): <i>m</i>/<i>z</i> calcd for C<sub>14</sub>H<sub>10</sub>F<sub>3</sub>N<sub>4</sub>OS (M – H)<sup>−</sup>, 339.0527; found, 339.0532.</div></div><div id="sec5_1_3_33" class="NLM_sec NLM_sec_level_2"><div id="ac_i86" class="anchor-spacer"></div><h3 class="article-section__title" id="_i86"> 5-Methyl-2-((3-(trifluoromethyl)benzyl)thio)[1,2,4]triazolo[1,5-<i>a</i>]pyrimidin-7(4<i>H</i>)-one (<b>C33</b>)</h3><div class="NLM_p last">White solid; yield, 63%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 13.20 (s, 1H), 7.85 (s, 1H), 7.79 (d, <i>J</i> = 7.6 Hz, 1H), 7.63 (d, <i>J</i> = 7.8 Hz, 1H), 7.57 (t, <i>J</i> = 7.7 Hz, 1H), 5.82 (s, 1H), 4.54 (s, 2H), 2.30 (s, 3H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 162.1 (-C-), 155.3 (-C-), 151.7 (-C-), 151.2 (-C-), 139.9 (-C-), 133.5 (-C-), 130.0 (-C-), 129.7 (-C-), 129.4 (-C-), 125.9 (-C-), 125.9 (-<u class="uu">C</u>H-), 124.5 (-<u class="uu">C</u>H-), 124.4 (-<u class="uu">C</u>H-), 123.2 (-<u class="uu">C</u>H-), 99.1 (-<u class="uu">C</u>H-), 34.3 (-<u class="uu">C</u>H<sub>2</sub>-), 19.0 (-<u class="uu">C</u>H<sub>3</sub>). HRMS (ESI): <i>m</i>/<i>z</i> calcd for C<sub>14</sub>H<sub>12</sub>F<sub>3</sub>N<sub>4</sub>OS (M + H)<sup><b>+</b></sup>, 341.0683; found, 341.0669.</div></div><div id="sec5_1_3_34" class="NLM_sec NLM_sec_level_2"><div id="ac_i87" class="anchor-spacer"></div><h3 class="article-section__title" id="_i87"> 2-((4-Methoxybenzyl)thio)-5-methyl[1,2,4]triazolo[1,5-<i>a</i>]pyrimidin-7(4<i>H</i>)-one (<b>C34</b>)</h3><div class="NLM_p last">White solid; yield, 78%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 13.18 (s, 1H), 7.37 (d, <i>J</i> = 8.6 Hz, 2H), 6.88 (d, <i>J</i> = 8.6 Hz, 2H), 5.81 (s, 1H), 4.38 (s, 2H), 3.73 (s, 3H), 2.30 (s, 3H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 162.6 (-C-), 159.0 (-C-), 155.3 (-C-), 151.6 (-C-), 151.2 (-C-), 130.6 (-C-), 129.6 (-<u class="uu">C</u>H-), 114.4 (-<u class="uu">C</u>H-), 99.0 (-<u class="uu">C</u>H-), 55.6 (-<u class="uu">C</u>H<sub>3</sub>), 34.6 (-<u class="uu">C</u>H<sub>2</sub>-), 19.0 (-<u class="uu">C</u>H<sub>3</sub>). HRMS (ESI): <i>m</i>/<i>z</i> calcd for C<sub>14</sub>H<sub>15</sub>N<sub>4</sub>O<sub>2</sub>S (M + H)<sup><b>+</b></sup>, 303.0915; found, 303.0902.</div></div><div id="sec5_1_3_35" class="NLM_sec NLM_sec_level_2"><div id="ac_i88" class="anchor-spacer"></div><h3 class="article-section__title" id="_i88"> 2-((4-Chloro-3-fluorobenzyl)thio)-5-methyl[1,2,4]triazolo[1,5-<i>a</i>]pyrimidin-7(4<i>H</i>)-one (<b>C35</b>)</h3><div class="NLM_p last">White solid; yield, 69%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 13.19 (s, 1H), 7.53 (m, 2H), 7.34 (m, 1H), 5.82 (s, 1H), 4.45 (s, 2H), 2.30 (s, 3H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 162.0 (-C-), 158.6 (-C-), 156.1 (-C-), 155.3 (-C-), 151.7 (-C-), 151.3 (-C-), 140.3 (-C-), 140.2 (-C-), 131.0 (-C-), 126.7 (-C-), 126.6 (-C-), 118.8 (-C-), 118.6 (-<u class="uu">C</u>H-), 117.8 (-<u class="uu">C</u>H-), 117.6 (-<u class="uu">C</u>H-), 99.1 (-<u class="uu">C</u>H-), 33.9 (-<u class="uu">C</u>H<sub>2</sub>-), 19.0 (-<u class="uu">C</u>H<sub>3</sub>). HRMS (ESI): <i>m</i>/<i>z</i> calcd for C<sub>13</sub>H<sub>11</sub>ClFN<sub>4</sub>OS (M + H)<sup><b>+</b></sup>, 325.0326; found, 325.0315.</div></div><div id="sec5_1_3_36" class="NLM_sec NLM_sec_level_2"><div id="ac_i89" class="anchor-spacer"></div><h3 class="article-section__title" id="_i89"> 2-((3-Chloro-4-fluorobenzyl)thio)-5-methyl[1,2,4]triazolo[1,5-<i>a</i>]pyrimidin-7(4<i>H</i>)-one (<b>C36</b>)</h3><div class="NLM_p last">White solid; yield, 71%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 13.19 (s, 1H), 7.69 (m, 1H), 7.53–7.43 (m, 1H), 7.36 (t, <i>J</i> = 9.0 Hz, 1H), 5.81 (s, 1H), 4.43 (s, 2H), 2.29 (s, 3H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 162.1 (-C-), 158.1 (-C-), 155.7 (-C-), 155.3 (-C-), 151.7 (-C-), 151.2 (-C-), 136.4 (-C-), 136.3 (-C-), 131.4 (-C-), 130.1 (-C-), 130.0 (-C-), 119.7 (-C-), 119.6 (-<u class="uu">C</u>H-), 117.4 (-<u class="uu">C</u>H-), 117.2 (-<u class="uu">C</u>H-), 99.1 (-<u class="uu">C</u>H-), 33.7 (-<u class="uu">C</u>H<sub>2</sub>-), 19.0 (-<u class="uu">C</u>H<sub>3</sub>). HRMS (ESI): <i>m</i>/<i>z</i> calcd for C<sub>13</sub>H<sub>11</sub>ClFN<sub>4</sub>OS (M + H)<sup><b>+</b></sup>, 325.0326; found, 325.0314.</div></div><div id="sec5_1_3_37" class="NLM_sec NLM_sec_level_2"><div id="ac_i90" class="anchor-spacer"></div><h3 class="article-section__title" id="_i90"> 2-((2,6-Dichlorobenzyl)thio)-5-methyl[1,2,4]triazolo[1,5-<i>a</i>]pyrimidin-7(4<i>H</i>)-one (<b>C37</b>)</h3><div class="NLM_p last">White solid; yield, 77%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 13.25 (s, 1H), 7.55 (d, <i>J</i> = 8.1 Hz, 2H), 7.46–7.36 (m, 1H), 5.85 (s, 1H), 4.73 (s, 2H), 2.32 (s, 3H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 161.8 (-C-), 155.3 (-C-), 151.8 (-C-), 151.3 (-C-), 135.6 (-C-), 132.4 (-<u class="uu">C</u>H-), 131.0 (-<u class="uu">C</u>H-), 129.3 (-<u class="uu">C</u>H-), 99.1 (-<u class="uu">C</u>H-), 31.9 (-<u class="uu">C</u>H<sub>2</sub>-), 19.0 (-<u class="uu">C</u>H<sub>3</sub>). HRMS (ESI): <i>m</i>/<i>z</i> calcd for C<sub>13</sub>H<sub>11</sub>Cl<sub>2</sub>N<sub>4</sub>OS (M + H)<sup><b>+</b></sup>, 341.0030; found, 341.0019.</div></div><div id="sec5_1_3_38" class="NLM_sec NLM_sec_level_2"><div id="ac_i91" class="anchor-spacer"></div><h3 class="article-section__title" id="_i91"> 2-((4-Chlorobenzyl)thio)-5,6-dimethyl[1,2,4]triazolo[1,5-<i>a</i>]pyrimidin-7(4<i>H</i>)-one (<b>C38</b>)</h3><div class="NLM_p last">White solid; yield, 72%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 13.02 (s, 1H), 7.48 (d, <i>J</i> = 8.4 Hz, 2H), 7.41–7.28 (m, 2H), 4.43 (s, 2H), 2.30 (s, 3H), 1.95 (s, 3H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 162.3 (-C-), 156.1 (-C-), 150.6 (-C-), 146.4 (-C-), 137.3 (-C-), 132.4 (-C-), 131.2 (-<u class="uu">C</u>H-), 128.8 (-<u class="uu">C</u>H-), 105.2 (-<u class="uu">C</u>H-), 34.2 (-<u class="uu">C</u>H<sub>2</sub>-), 17.6 (-<u class="uu">C</u>H<sub>3</sub>), 10.8 (-<u class="uu">C</u>H<sub>3</sub>). HRMS (ESI): <i>m</i>/<i>z</i> calcd for C<sub>14</sub>H<sub>14</sub>ClN<sub>4</sub>OS (M + H)<sup><b>+</b></sup>, 321.0576; found, 321.0565.</div></div><div id="sec5_1_3_39" class="NLM_sec NLM_sec_level_2"><div id="ac_i92" class="anchor-spacer"></div><h3 class="article-section__title" id="_i92"> 2-((4-Chlorobenzyl)thio)-5-methyl-6-pentyl[1,2,4]triazolo[1,5-<i>a</i>]pyrimidin-7(4<i>H</i>)-one (<b>C39</b>)</h3><div class="NLM_p last">White solid; yield, 74%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.99 (s, 1H), 7.48 (d, <i>J</i> = 8.4 Hz, 2H), 7.37 (d, <i>J</i> = 8.4 Hz, 2H), 4.43 (s, 2H), 2.48–2.37 (m, 2H), 2.32 (s, 3H), 1.40 (d, <i>J</i> = 6.8 Hz, 2H), 1.29 (d, <i>J</i> = 3.0 Hz, 4H), 0.87 (t, <i>J</i> = 6.7 Hz, 3H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 162.3 (-C-), 155.9 (-C-), 150.6 (-C-), 146.3 (-C-), 137.4 (-C-), 132.4 (-C-), 131.2 (-C-), 128.9 (-<u class="uu">C</u>H-), 110.1 (-<u class="uu">C</u>H-), 34.2 (-<u class="uu">C</u>H<sub>2</sub>-), 31.5 (-<u class="uu">C</u>H<sub>2</sub>-), 28.4 (-<u class="uu">C</u>H<sub>2</sub>-), 25.2 (-<u class="uu">C</u>H<sub>2</sub>-), 22.5 (-<u class="uu">C</u>H<sub>2</sub>-), 17.2 (-<u class="uu">C</u>H<sub>3</sub>), 14.4 (-<u class="uu">C</u>H<sub>3</sub>). HRMS (ESI): <i>m</i>/<i>z</i> calcd for C<sub>18</sub>H<sub>22</sub>ClN<sub>4</sub>OS (M + H)<sup><b>+</b></sup>, 377.1202; found, 377.1188.</div></div><div id="sec5_1_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i93" class="anchor-spacer"></div><h3 class="article-section__title" id="_i93"> General Method for the Synthesis of <b>D1</b>–<b>D39</b></h3><div class="NLM_p last">Compounds <b>D1</b>–<b>D39</b> were prepared following the previously reported method<a onclick="showRef(event, 'ref56'); return false;" href="javascript:void(0);" class="ref ref56">(56)</a> as described for the synthesis of <b>D15</b>. <b>C15</b> was stirred in POCl<sub>3</sub> (excess) at 90 °C for 3–5 h. The resultant oil was added dropwise to ice cold bath. The resulting aqueous mixture was extracted with CH<sub>2</sub>Cl<sub>2</sub> (4 × 30 mL), and the organic layer was washed by saturated aqueous NaHCO<sub>3</sub> (3 × 10 mL). The organic layers were dried over MgSO<sub>4</sub> and concentrated under reduced pressure to give the orange solid <b>D15</b>, which was used directly without further purification. <b>D15</b> was unstable enough and therefore only characterized by HRMS. HRMS (ESI): <i>m</i>/<i>z</i> calcd for C<sub>13</sub>H<sub>12</sub>ClN<sub>4</sub>S (M – H)<sup>−</sup>, 291.0471; found, 291.0463.</div></div><div id="sec5_1_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i94" class="anchor-spacer"></div><h3 class="article-section__title" id="_i94"> General Method for the Synthesis of <b>E1</b>–<b>E44</b></h3><div class="NLM_p last">The general method was described for the synthesis of <b>E1</b>. Compound <b>D1</b> (150 mg, 0.65 mmol) was dissolved in ethanol (2 mL), followed by addition of 1-methyl-1<i>H</i>-tetrazole-5-thiol (75.53 mg, 0.65 mmol). The reaction mixture was stirred at room temperature for 10 h. The resultant mixture was concentrated under reduced pressure to the residue, which was then purified by column chromatography to give <b>E1</b>.</div></div><div id="sec5_1_5_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i95" class="anchor-spacer"></div><h3 class="article-section__title" id="_i95"> 7-((1-Methyl-1<i>H</i>-tetrazol-5-yl)thio)-5-phenyl[1,2,4]triazolo[1,5-<i>a</i>]pyrimidine (<b>E1</b>)</h3><div class="NLM_p last">White solid; yield, 88%. HPLC purity: 95.06%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.75 (s, 1H), 8.19–8.09 (m, 2H), 7.64–7.49 (m, 4H), 4.23 (s, 3H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 160.1 (-C-), 156.2 (-C-), 154.6 (-C-), 146.3 (-<u class="uu">C</u>H-), 143.5 (-C-), 135.6 (-C-), 131.6 (-<u class="uu">C</u>H-), 129.1 (-<u class="uu">C</u>H-), 127.8 (-<u class="uu">C</u>H-), 107.2 (-<u class="uu">C</u>H-), 35.1 (-<u class="uu">C</u>H<sub>3</sub>). HRMS (ESI): <i>m</i>/<i>z</i> calcd for C<sub>13</sub>H<sub>11</sub>N<sub>8</sub>S (M + H)<sup>+</sup>, 311.0827; found, 311.0803.</div></div><div id="sec5_1_5_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i96" class="anchor-spacer"></div><h3 class="article-section__title" id="_i96"> 5-Phenyl-7-((1-phenyl-1<i>H</i>-tetrazol-5-yl)thio)[1,2,4]triazolo[1,5-<i>a</i>]pyrimidine (<b>E2</b>)</h3><div class="NLM_p last">White solid; yield, 63%. HPLC purity: 99.25%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.69 (s, 1H), 8.18 (m, 2H), 7.96 (s, 1H), 7.81 (m, 2H), 7.67–7.53 (m, 6H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 160.2 (-C-), 156.1 (-C-), 154.5 (-C-), 146.8 (-<u class="uu">C</u>H-), 141.9 (-C-), 135.5 (-C-), 133.0 (-C-), 131.6 (-<u class="uu">C</u>H-), 131.0 (-<u class="uu">C</u>H-), 129.7 (-<u class="uu">C</u>H-), 129.1 (-<u class="uu">C</u>H-), 127.9 (-<u class="uu">C</u>H-), 125.3 (-<u class="uu">C</u>H-), 108.6 (-<u class="uu">C</u>H-). HRMS (ESI): <i>m</i>/<i>z</i> calcd for C<sub>18</sub>H<sub>13</sub>N<sub>8</sub>S (M + H)<sup>+</sup>, 373.0983; found, 373.0968.</div></div><div id="sec5_1_5_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i97" class="anchor-spacer"></div><h3 class="article-section__title" id="_i97"> 4-(5-((5-Phenyl[1,2,4]triazolo[1,5-<i>a</i>]pyrimidin-7-yl)thio)-1<i>H</i>-tetrazol-1-yl)phenol (<b>E3</b>)</h3><div class="NLM_p last">White solid; yield, 92%. HPLC purity: 99.60%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.25 (s, 1H), 8.71 (s, 1H), 8.26–8.10 (m, 2H), 7.91 (s, 1H), 7.67–7.46 (m, 5H), 6.91 (d, <i>J</i> = 8.7 Hz, 2H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 160.2 (-C-), 159.5 (-C-), 156.1 (-C-), 154.5 (-C-), 146.8 (-<u class="uu">C</u>H-), 142.0 (-C-), 135.5 (-C-), 131.6 (-<u class="uu">C</u>H-), 129.1 (-<u class="uu">C</u>H-), 127.9 (-<u class="uu">C</u>H-), 126.9 (-C-), 124.0 (-<u class="uu">C</u>H-), 115.9 (-<u class="uu">C</u>H-), 108.4 (-<u class="uu">C</u>H-). HRMS (ESI): <i>m</i>/<i>z</i> calcd for C<sub>18</sub>H<sub>13</sub>N<sub>8</sub>OS (M + H)<sup>+</sup>, 389.0933; found, 389.0900.</div></div><div id="sec5_1_5_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i98" class="anchor-spacer"></div><h3 class="article-section__title" id="_i98"> <i>N</i>,<i>N</i>-Dimethyl-2-(5-((5-phenyl[1,2,4]triazolo[1,5-<i>a</i>]pyrimidin-7-yl)thio)-1<i>H</i>-tetrazol-1-yl)ethan-1-amine Hydrochloride (<b>E4</b>)</h3><div class="NLM_p last">White solid; yield, 91%. HPLC purity: 99.53%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.13 (s, 1H), 8.76 (s, 1H), 8.18 (m, 2H), 7.71 (s, 1H), 7.63–7.54 (m, 3H), 5.08 (t, <i>J</i> = 6.4 Hz, 2H), 3.79 (s, 2H), 2.85 (s, 6H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 166.9 (-C-), 156.2 (-C-), 153.7 (-C-), 151.3 (-<u class="uu">C</u>H-), 150.0 (-C-), 133.4 (-C-), 130.7 (-<u class="uu">C</u>H-), 128.8 (-<u class="uu">C</u>H-), 127.4 (-<u class="uu">C</u>H-), 97.2 (-<u class="uu">C</u>H-), 55.7 (-<u class="uu">C</u>H<sub>2</sub>-), 42.6 (-<u class="uu">C</u>H<sub>2</sub>-), 40.7 (-<u class="uu">C</u>H<sub>3</sub>). HRMS (ESI): <i>m</i>/<i>z</i> calcd for C<sub>16</sub>H<sub>18</sub>N<sub>9</sub>S (M + H)<sup>+</sup>, 368.1400; found, 368.1373.</div></div><div id="sec5_1_5_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i99" class="anchor-spacer"></div><h3 class="article-section__title" id="_i99"> <i>N</i>,<i>N</i>-Dimethyl-2-(5-((5-methylpyrazolo[1,5-<i>a</i>]pyrimidin-7-yl)thio)-1<i>H</i>-tetrazol-1-yl)ethan-1-amine Hydrochloride (<b>E5</b>)</h3><div class="NLM_p last">White solid; yield, 52%. HPLC purity: 100.00%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.12 (s, 1H), 8.25 (d, <i>J</i> = 2.3 Hz, 1H), 6.68 (d, <i>J</i> = 2.3 Hz, 1H), 6.65 (s, 1H), 5.02 (t, <i>J</i> = 6.4 Hz, 2H), 3.72 (t, <i>J</i> = 6.0 Hz, 2H), 2.82 (s, 6H), 2.47 (s, 3H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 158.4 (-C-), 147.7 (-C-), 146.8 (-C-), 144.8 (-C-), 141.2 (-<u class="uu">C</u>H-), 107.3 (-<u class="uu">C</u>H-), 95.9 (-<u class="uu">C</u>H-), 54.1 (-<u class="uu">C</u>H<sub>2</sub>-), 43.1 (-<u class="uu">C</u>H<sub>2</sub>-), 42.5 (-<u class="uu">C</u>H<sub>3</sub>), 24.3 (-<u class="uu">C</u>H<sub>3</sub>). HRMS (ESI): <i>m</i>/<i>z</i> calcd for C<sub>12</sub>H<sub>17</sub>N<sub>8</sub>S (M + H)<sup>+</sup>, 305.1291; found, 305.1284.</div></div><div id="sec5_1_5_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i100" class="anchor-spacer"></div><h3 class="article-section__title" id="_i100"> <i>N</i>,<i>N</i>-Dimethyl-2-(5-(thieno[3,2-<i>d</i>]pyrimidin-4-ylthio)-1<i>H</i>-tetrazol-1-yl)ethan-1-amine Hydrochloride (<b>E6</b>)</h3><div class="NLM_p last">White solid; yield, 94%. HPLC purity: 99.72%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.09 (s, 1H), 8.90 (s, 1H), 8.60 (d, <i>J</i> = 5.4 Hz, 1H), 7.74 (d, <i>J</i> = 5.4 Hz, 1H), 4.97 (t, <i>J</i> = 6.6 Hz, 2H), 3.72 (t, <i>J</i> = 6.5 Hz, 2H), 2.80 (s, 6H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 160.4 (-C-), 158.0 (-C-), 153.8 (-<u class="uu">C</u>H-), 146.7 (-C-), 138.6 (-<u class="uu">C</u>H-), 127.6 (-<u class="uu">C</u>H-), 124.3 (-<u class="uu">C</u>H-), 54.2 (-<u class="uu">C</u>H<sub>2</sub>-), 42.9 (-<u class="uu">C</u>H<sub>2</sub>-), 42.6 (-<u class="uu">C</u>H<sub>3</sub>). HRMS (ESI): <i>m</i>/<i>z</i> calcd for C<sub>11</sub>H<sub>14</sub>N<sub>7</sub>S<sub>2</sub> (M + H)<sup>+</sup>, 308.0746; found, 308.0726.</div></div><div id="sec5_1_5_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i101" class="anchor-spacer"></div><h3 class="article-section__title" id="_i101"> (<i>Z</i>)-2-(1-(((6,7-Bis(2-methoxyethoxy)quinazolin-4-yl)thio)methyl)tetraaz-2-en-1-yl)-<i>N</i>,<i>N</i>-dimethylethan-1-amine Hydrochloride (<b>E7</b>)</h3><div class="NLM_p last">Yellow solid; yield, 78%. HPLC purity: 97.60%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.67 (s, 1H), 7.42 (d, <i>J</i> = 10.9 Hz, 2H), 4.90 (t, <i>J</i> = 6.5 Hz, 2H), 4.52–4.31 (m, 4H), 3.84–3.68 (m, 6H), 3.36 (t, <i>J</i> = 9.5 Hz, 7H), 2.83 (d, <i>J</i> = 9.2 Hz, 6H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 161.7 (-C-), 155.8 (-C-), 152.0 (-<u class="uu">C</u>H-), 150.1 (-C-), 147.4 (-C-), 146.4 (-C-), 117.8 (-C-), 107.9 (-<u class="uu">C</u>H-), 102.3 (-<u class="uu">C</u>H-), 70.1 (-<u class="uu">C</u>H<sub>2</sub>-), 69.9 (-<u class="uu">C</u>H<sub>2</sub>-), 68.7 (-<u class="uu">C</u>H<sub>2</sub>-), 68.7 (-<u class="uu">C</u>H<sub>2</sub>-), 58.4 (-<u class="uu">C</u>H<sub>3</sub>), 58.3 (-<u class="uu">C</u>H<sub>3</sub>), 54.2 (-<u class="uu">C</u>H<sub>2</sub>), 42.8 (-<u class="uu">C</u>H<sub>2</sub>), 42.6 (-<u class="uu">C</u>H<sub>3</sub>). HRMS (ESI): <i>m</i>/<i>z</i> calcd for C<sub>19</sub>H<sub>28</sub>N<sub>7</sub>O<sub>4</sub>S (M + H)<sup>+</sup>, 450.1918; found, 450.1885.</div></div><div id="sec5_1_5_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i102" class="anchor-spacer"></div><h3 class="article-section__title" id="_i102"> <i>N</i>,<i>N</i>-Dimethyl-2-(5-((5-methyl[1,2,4]triazolo[1,5-<i>a</i>]pyrimidin-7-yl)thio)-1<i>H</i>-tetrazol-1-yl)ethan-1-amine Hydrochloride (<b>E8</b>)</h3><div class="NLM_p last">White solid; yield, 75%. HPLC purity: 99.59%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.74 (s, 1H), 8.69 (s, 1H), 6.94 (s, 1H), 5.01 (t, <i>J</i> = 5.9 Hz, 2H), 3.71 (s, 2H), 2.83 (s, 6H), 2.56 (s, 3H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 167.0 (-C-), 155.8 (-C-), 151.8 (-C-), 151.6 (-<u class="uu">C</u>H-), 150.6 (-C-), 98.1 (-<u class="uu">C</u>H-), 55.8 (-<u class="uu">C</u>H<sub>2</sub>-), 42.6 (-<u class="uu">C</u>H<sub>3</sub>), 40.7 (-<u class="uu">C</u>H<sub>2</sub>-), 18.6 (-<u class="uu">C</u>H<sub>3</sub>). HRMS (ESI): <i>m</i>/<i>z</i> calcd for C<sub>11</sub>H<sub>16</sub>N<sub>9</sub>S (M + H)<sup>+</sup>, 306.1243; found, 306.1227.</div></div><div id="sec5_1_5_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i103" class="anchor-spacer"></div><h3 class="article-section__title" id="_i103"> 2-(5-((5-Ethyl[1,2,4]triazolo[1,5-<i>a</i>]pyrimidin-7-yl)thio)-1<i>H</i>-tetrazol-1-yl)-<i>N</i>,<i>N</i>-dimethylethan-1-amine Hydrochloride (<b>E9</b>)</h3><div class="NLM_p last">White solid; yield, 76%. HPLC purity: 98.85%. <sup>1</sup>H NMR (400 MHz, DMSO–<i>d</i><sub><i>6</i></sub>) δ 11.27 (s, 1H), 8.68 (s, 1H), 7.01 (s, 1H), 5.03 (s, 2H), 3.74 (s, 2H), 2.86 (m, 8H), 1.22 (t, <i>J</i> = 7.5 Hz, 3H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 169.2 (-C-), 155.4 (-C-), 154.3 (-C-), 146.3 (-<u class="uu">C</u>H-), 142.8 (-C-), 109.2 (-<u class="uu">C</u>H-), 54.0 (-<u class="uu">C</u>H<sub>2</sub>-), 43.1 (-<u class="uu">C</u>H<sub>2</sub>-), 42.4 (-<u class="uu">C</u>H<sub>3</sub>), 30.8 (-<u class="uu">C</u>H<sub>2</sub>-), 12.4 (-<u class="uu">C</u>H<sub>3</sub>). HRMS (ESI): <i>m</i>/<i>z</i> calcd for C<sub>12</sub>H<sub>18</sub>N<sub>9</sub>S (M + H)<sup>+</sup>, 320.1400; found, 320.1379.</div></div><div id="sec5_1_5_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i104" class="anchor-spacer"></div><h3 class="article-section__title" id="_i104"> 2-(5-((5-Isopropyl[1,2,4]triazolo[1,5-<i>a</i>]pyrimidin-7-yl)thio)-1<i>H</i>-tetrazol-1-yl)-<i>N</i>,<i>N</i>-dimethylethan-1-amine Hydrochloride (<b>E10</b>)</h3><div class="NLM_p last">White solid; yield, 68%. HPLC purity: 98.80%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.05 (s, 1H), 8.68 (s, 1H), 7.07 (s, 1H), 5.02 (t, <i>J</i> = 6.1 Hz, 2H), 3.73 (s, 2H), 3.05–3.12 (m, 1H), 2.82 (s, 6H), 1.23 (d, <i>J</i> = 6.8 Hz, 6H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 172.7 (-C-), 155.5 (-C-), 154.4 (-C-), 146.5 (-<u class="uu">C</u>H-), 142.8 (-C-), 108.8 (-<u class="uu">C</u>H-), 54.1 (-<u class="uu">C</u>H<sub>2</sub>-), 43.2 (-<u class="uu">C</u>H<sub>2</sub>-), 42.5 (-<u class="uu">C</u>H<sub>3</sub>), 35.8 (-<u class="uu">C</u>H-), 21.5 (-<u class="uu">C</u>H<sub>3</sub>). HRMS (ESI): <i>m</i>/<i>z</i> calcd for C<sub>13</sub>H<sub>20</sub>N<sub>9</sub>S (M + H)<sup>+</sup>, 334.1556; found, 334.1533.</div></div><div id="sec5_1_5_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i105" class="anchor-spacer"></div><h3 class="article-section__title" id="_i105"> 2-(5-((5-Cyclopropyl[1,2,4]triazolo[1,5-<i>a</i>]pyrimidin-7-yl)thio)-1<i>H</i>-tetrazol-1-yl)-<i>N</i>,<i>N</i>-dimethylethan-1-amine Hydrochloride (<b>E11</b>)</h3><div class="NLM_p last">White solid; yield, 68%. HPLC purity: 99.39%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.01 (s, 1H), 8.62 (s, 1H), 7.12 (s, 1H), 5.03 (t, <i>J</i> = 6.3 Hz, 2H), 3.74 (t, <i>J</i> = 5.9 Hz, 2H), 2.84 (s, 6H), 2.40–2.20 (m, 1H), 1.11 (t, <i>J</i> = 16.5 Hz, 4H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 169.7 (-C-), 155.4 (-C-), 154.6 (-C-), 146.4 (-<u class="uu">C</u>H-), 142.3 (-C-), 108.9 (-<u class="uu">C</u>H-), 54.1 (-<u class="uu">C</u>H<sub>2</sub>-), 43.1 (-<u class="uu">C</u>H<sub>2</sub>-), 42.5 (-<u class="uu">C</u>H<sub>3</sub>), 17.2 (-<u class="uu">C</u>H), 12.5 (-<u class="uu">C</u>H<sub>2</sub>). HRMS (ESI): <i>m</i>/<i>z</i> calcd for C<sub>13</sub>H<sub>18</sub>N<sub>9</sub>S (M + H)<sup>+</sup>, 332.1400; found, 332.1380.</div></div><div id="sec5_1_5_12" class="NLM_sec NLM_sec_level_2"><div id="ac_i106" class="anchor-spacer"></div><h3 class="article-section__title" id="_i106"> 2-(5-((6,7-Dihydro-5<i>H</i>-cyclopenta[d][1,2,4]triazolo[1,5-<i>a</i>]pyrimidin-8-yl)thio)-1<i>H</i>-tetrazol-1-yl)-<i>N</i>,<i>N</i>-dimethylethan-1-amine Hydrochloride (<b>E12</b>)</h3><div class="NLM_p last">White solid; yield, 86%. HPLC purity: 99.74%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.47 (s, 1H), 4.76–4.65 (m, 2H), 3.15–3.04 (m, 2H), 2.84 (t, <i>J</i> = 7.4 Hz, 2H), 2.75–2.65 (m, 2H), 2.18 (s, 6H), 2.17–2.12 (m, 2H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 174.0 (-C-), 154.7 (-C-), 154.6 (-C-), 148.4 (-<u class="uu">C</u>H-), 134.8 (-C-), 128.9 (-C-), 57.6 (-<u class="uu">C</u>H<sub>2</sub>-), 46.5 (-<u class="uu">C</u>H<sub>2</sub>-), 45.1 (-<u class="uu">C</u>H<sub>3</sub>), 34.4 (-<u class="uu">C</u>H<sub>2</sub>-), 28.0 (-<u class="uu">C</u>H<sub>2</sub>-), 23.0 (-<u class="uu">C</u>H<sub>2</sub>-). HRMS (ESI): <i>m</i>/<i>z</i> calcd for C<sub>13</sub>H<sub>18</sub>N<sub>9</sub>S (M + H)<sup>+</sup>, 332.1400; found, 332.1379.</div></div><div id="sec5_1_5_13" class="NLM_sec NLM_sec_level_2"><div id="ac_i107" class="anchor-spacer"></div><h3 class="article-section__title" id="_i107"> <i>N</i>,<i>N</i>-Dimethyl-2-(5-((5-methyl-2-(propylthio)[1,2,4]triazolo[1,5-<i>a</i>]pyrimidin-7-yl)thio)-1<i>H</i>-tetrazol-1-yl)ethan-1-amine Hydrochloride (<b>E13</b>)</h3><div class="NLM_p last">White solid; yield, 90%. HPLC purity: 100%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.22 (s, 1H), 6.82 (s, 1H), 5.00 (t, <i>J</i> = 6.2 Hz, 2H), 3.71 (s, 2H), 3.21 (t, <i>J</i> = 7.1 Hz, 2H), 2.81 (s, 6H), 2.50 (s, 3H, overlapped with DMSO-<i>d</i><sub>6</sub> peak), 1.85–1.69 (m, 2H), 1.02 (t, <i>J</i> = 7.3 Hz, 3H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 166.7 (-C-), 164.4 (-C-), 154.6 (-C-), 146.1 (-C-), 141.8 (-C-), 108.4 (-<u class="uu">C</u>H-), 54.1 (-<u class="uu">C</u>H<sub>2</sub>-), 43.1 (-<u class="uu">C</u>H<sub>2</sub>-), 42.4 (-<u class="uu">C</u>H<sub>3</sub>), 32.5 (-<u class="uu">C</u>H<sub>2</sub>-), 24.5 (-<u class="uu">C</u>H<sub>2</sub>-), 22.5 (-<u class="uu">C</u>H<sub>3</sub>), 13.0 (-<u class="uu">C</u>H<sub>3</sub>). HRMS (ESI): <i>m</i>/<i>z</i> calcd for C<sub>14</sub>H<sub>22</sub>N<sub>9</sub>S<sub>2</sub> (M + H)<sup>+</sup>, 380.1434; found, 380.1407.</div></div><div id="sec5_1_5_14" class="NLM_sec NLM_sec_level_2"><div id="ac_i108" class="anchor-spacer"></div><h3 class="article-section__title" id="_i108"> 2-(5-((2-(Allylthio)-5-methyl[1,2,4]triazolo[1,5-<i>a</i>]pyrimidin-7-yl)thio)-1<i>H</i>-tetrazol-1-yl)-<i>N</i>,<i>N</i>-dimethylethan-1-amine Hydrochloride (<b>E14</b>)</h3><div class="NLM_p last">White solid; yield, 93%. HPLC purity: 99.79%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.15 (s, 1H), 6.82 (s, 1H), 6.02 (m, 1H), 5.38 (d, <i>J</i> = 17.0 Hz, 1H), 5.16 (d, <i>J</i> = 10.0 Hz, 1H), 5.00 (t, <i>J</i> = 6.2 Hz, 2H), 3.92 (d, <i>J</i> = 6.8 Hz, 2H), 3.70 (s, 2H), 2.81 (s, 6H), 2.51 (s, 3H, overlapped with DMSO-<i>d</i><sub>6</sub> peak). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 166.0 (-C-), 164.5 (-C-), 154.6 (-C-), 146.0 (-C-), 142.0 (-C-), 133.4 (-<u class="uu">C</u>H-), 118.5 (-<u class="uu">C</u>H<sub>2</sub>), 108.4 (-<u class="uu">C</u>H-), 54.1 (-<u class="uu">C</u>H<sub>2</sub>-), 43.1 (-<u class="uu">C</u>H<sub>2</sub>-), 42.4 (-<u class="uu">C</u>H<sub>3</sub>), 33.2 (-<u class="uu">C</u>H<sub>2</sub>-), 24.6 (-<u class="uu">C</u>H<sub>3</sub>). HRMS (ESI): <i>m</i>/<i>z</i> calcd for C<sub>14</sub>H<sub>20</sub>N<sub>9</sub>S<sub>2</sub>(M + H)<sup>+</sup>, 378.1277; found, 378.1247.</div></div><div id="sec5_1_5_15" class="NLM_sec NLM_sec_level_2"><div id="ac_i109" class="anchor-spacer"></div><h3 class="article-section__title" id="_i109"> <i>N</i>,<i>N</i>-Dimethyl-2-(5-((5-methyl-2-(prop-2-yn-1-ylthio)[1,2,4]triazolo[1,5-<i>a</i>]pyrimidin-7-yl)thio)-1<i>H</i>-tetrazol-1-yl)ethan-1-amine Hydrochloride (<b>E15</b>)</h3><div class="NLM_p last">White solid; yield, 85%. HPLC purity: 99.12%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.07 (s, 1H), 6.85 (s, 1H), 5.00 (t, <i>J</i> = 6.2 Hz, 2H), 4.11 (d, <i>J</i> = 2.5 Hz, 2H), 3.70 (s, 2H), 3.24 (t, <i>J</i> = 2.5 Hz, 1H), 2.81 (s, 6H), 2.51 (s, 3H, overlapped with DMSO-<i>d</i><sub>6</sub> peak). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 165.1 (-C-), 164.7 (-C-), 154.6 (-C-), 146.0 (-C-), 142.1 (-C-), 108.8 (-<u class="uu">C</u>H-), 79.8 (-C-), 74.0 (-<u class="uu">C</u>H), 54.1 (-<u class="uu">C</u>H<sub>2</sub>-), 43.1 (-<u class="uu">C</u>H<sub>2</sub>-), 42.5 (-<u class="uu">C</u>H<sub>3</sub>), 24.6 (-<u class="uu">C</u>H<sub>2</sub>-), 19.1 (-<u class="uu">C</u>H<sub>3</sub>). HRMS (ESI): <i>m</i>/<i>z</i> calcd for C<sub>14</sub>H<sub>18</sub>N<sub>9</sub>S<sub>2</sub> (M + H)<sup>+</sup>, 376.1121; found, 376.1091.</div></div><div id="sec5_1_5_16" class="NLM_sec NLM_sec_level_2"><div id="ac_i110" class="anchor-spacer"></div><h3 class="article-section__title" id="_i110"> 2-(5-((2-(Isopropylthio)-5-methyl[1,2,4]triazolo[1,5-<i>a</i>]pyrimidin-7-yl)thio)-1<i>H</i>-tetrazol-1-yl)-<i>N</i>,<i>N</i>-dimethylethan-1-amine Hydrochloride (<b>E16</b>)</h3><div class="NLM_p last">White solid; yield, 62%. HPLC purity: 99.75%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.77 (s, 1H), 6.82 (s, 1H), 4.99 (t, <i>J</i> = 6.2 Hz, 2H), 3.88–3.98 (m, 1H), 3.70 (s, 2H), 2.83 (s, 6H), 2.51(s, 3H, overlapped with DMSO-<i>d</i><sub>6</sub> peak), 1.45 (d, <i>J</i> = 6.8 Hz, 6H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 166.3 (-C-), 164.4 (-C-), 154.5 (-C-), 146.1 (-C-), 141.8 (-C-), 108.5 (-<u class="uu">C</u>H-), 54.1 (-<u class="uu">C</u>H<sub>2</sub>-), 42.5 (-<u class="uu">C</u>H<sub>2</sub>-), 36.6 (-<u class="uu">C</u>H<sub>3</sub>), 24.6 (-<u class="uu">C</u>H-), 23.2 (-<u class="uu">C</u>H<sub>3</sub>). HRMS (ESI): <i>m</i>/<i>z</i> calcd for C<sub>14</sub>H<sub>22</sub>N<sub>9</sub>S<sub>2</sub> (M + H)<sup>+</sup>, 380.1434; found, 380.1412.</div></div><div id="sec5_1_5_17" class="NLM_sec NLM_sec_level_2"><div id="ac_i111" class="anchor-spacer"></div><h3 class="article-section__title" id="_i111"> 2-(5-((2-((Cyclopropylmethyl)thio)-5-methyl[1,2,4]triazolo[1,5-<i>a</i>]pyrimidin-7-yl)thio)-1<i>H</i>-tetrazol-1-yl)-<i>N</i>,<i>N</i>-dimethylethan-1-amine Hydrochloride (<b>E17</b>)</h3><div class="NLM_p last">White solid; yield, 82%. HPLC purity: 98.27%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.02 (s, 1H), 6.81 (s, 1H), 4.99 (s, 2H), 3.70 (s, 2H), 3.19 (d, <i>J</i> = 6.9 Hz, 2H), 2.81 (s, 6H), 2.50 (s, 3H, overlapped with DMSO-<i>d</i><sub>6</sub> peak), 1.23 (s, 1H), 0.58 (d, <i>J</i> = 6.8 Hz, 2H), 0.37 (s, 2H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 166.9 (-C-), 164.4 (-C-), 154.6 (-C-), 146.1 (-C-), 141.8 (-C-), 108.4 (-<u class="uu">C</u>H-), 54.1 (-<u class="uu">C</u>H<sub>2</sub>-), 43.1 (-<u class="uu">C</u>H<sub>2</sub>-), 42.4 (-<u class="uu">C</u>H<sub>2</sub>-), 36.2 (-<u class="uu">C</u>H<sub>3</sub>), 24.5 (-<u class="uu">C</u>H<sub>3</sub>), 11.0 (-<u class="uu">C</u>H-), 5.7 (-<u class="uu">C</u>H<sub>2</sub>-). HRMS (ESI): <i>m</i>/<i>z</i> calcd for C<sub>15</sub>H<sub>22</sub>N<sub>9</sub>S<sub>2</sub> (M + H)<sup>+</sup>, 392.1434; found, 392.1409.</div></div><div id="sec5_1_5_18" class="NLM_sec NLM_sec_level_2"><div id="ac_i112" class="anchor-spacer"></div><h3 class="article-section__title" id="_i112"> 2-(5-((2-((Cyclohexylmethyl)thio)-5-methyl[1,2,4]triazolo[1,5-<i>a</i>]pyrimidin-7-yl)thio)-1<i>H</i>-tetrazol-1-yl)-<i>N</i>,<i>N</i>-dimethylethan-1-amine Hydrochloride (<b>E18</b>)</h3><div class="NLM_p last">White solid; yield, 87%. HPLC purity: 99.69%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.90 (s, 1H), 6.80 (s, 1H), 4.99 (s, 2H), 3.70 (s, 2H), 3.16 (s, 2H), 2.82 (s, 6H), 2.50(s, 3H, overlapped with DMSO-<i>d</i><sub>6</sub> peak), 1.76 (m, 6H), 1.12 (d, <i>J</i> = 49.7 Hz, 5H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 167.0 (-C-), 164.3 (-C-), 154.5 (-C-), 146.0 (-C-), 141.8 (-C-), 108.3 (-<u class="uu">C</u>H-), 54.1 (-<u class="uu">C</u>H<sub>2</sub>-), 43.1 (-<u class="uu">C</u>H<sub>2</sub>-), 42.4 (-<u class="uu">C</u>H<sub>3</sub>), 31.8 (-<u class="uu">C</u>H<sub>2</sub>-), 25.8 (-<u class="uu">C</u>H-), 25.4 (-<u class="uu">C</u>H<sub>2</sub>-), 24.5 (-<u class="uu">C</u>H<sub>3</sub>). HRMS (ESI): <i>m</i>/<i>z</i> calcd for C<sub>18</sub>H<sub>28</sub>N<sub>9</sub>S<sub>2</sub> (M + H)<sup>+</sup>, 434.1903; found, 434.1874.</div></div><div id="sec5_1_5_19" class="NLM_sec NLM_sec_level_2"><div id="ac_i113" class="anchor-spacer"></div><h3 class="article-section__title" id="_i113"> <i>N</i>,<i>N</i>-Dimethyl-2-(5-((5-methyl-2-((naphthalen-1-ylmethyl)thio)[1,2,4]triazolo[1,5-<i>a</i>]pyrimidin-7-yl)thio)-1<i>H</i>-tetrazol-1-yl)ethan-1-amine Hydrochloride (<b>E19</b>)</h3><div class="NLM_p last">White solid; yield, 62%. HPLC purity: 98.98%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.81 (d, <i>J</i> = 109.5 Hz, 1H), 8.21 (d, <i>J</i> = 8.3 Hz, 1H), 7.98 (d, <i>J</i> = 7.8 Hz, 1H), 7.90 (d, <i>J</i> = 8.2 Hz, 1H), 7.74 (d, <i>J</i> = 7.0 Hz, 1H), 7.60 (m, 2H), 7.47 (t, <i>J</i> = 7.6 Hz, 1H), 6.82 (d, <i>J</i> = 4.4 Hz, 1H), 4.99 (m, 4H), 3.70 (s, 2H), 2.81 (s, 6H), 2.52(s, 3H, overlapped with DMSO-<i>d</i><sub>6</sub> peak). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 166.3 (-C-), 164.6 (-C-), 154.6 (-C-), 146.0 (-C-), 142.0 (-C-), 133.5 (-C-), 132.5 (-C-), 130.9 (-C-), 128.7 (-<u class="uu">C</u>H-), 128.4 (-<u class="uu">C</u>H-), 127.9 (-<u class="uu">C</u>H-), 126.5 (-<u class="uu">C</u>H-), 126.0 (-<u class="uu">C</u>H-), 125.5 (-<u class="uu">C</u>H-), 123.7 (-<u class="uu">C</u>H-), 108.5 (-<u class="uu">C</u>H-), 54.2 (-<u class="uu">C</u>H<sub>2</sub>-), 43.1 (-<u class="uu">C</u>H<sub>2</sub>-), 42.5 (-<u class="uu">C</u>H<sub>3</sub>), 32.5 (-<u class="uu">C</u>H<sub>2</sub>-), 24.6 (-<u class="uu">C</u>H<sub>3</sub>). HRMS (ESI): <i>m</i>/<i>z</i> calcd for C<sub>22</sub>H<sub>24</sub>N<sub>9</sub>S<sub>2</sub> (M + H)<sup>+</sup>, 478.1590; found, 478.1580.</div></div><div id="sec5_1_5_20" class="NLM_sec NLM_sec_level_2"><div id="ac_i114" class="anchor-spacer"></div><h3 class="article-section__title" id="_i114"> 2-(5-((2-(Benzylthio)-5-methyl[1,2,4]triazolo[1,5-<i>a</i>]pyrimidin-7-yl)thio)-1<i>H</i>-tetrazol-1-yl)-<i>N</i>,<i>N</i>-dimethylethan-1-amine Hydrochloride (<b>E20</b>)</h3><div class="NLM_p last">White solid; yield, 88%. HPLC purity: 99.96%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.07 (s, 1H), 7.50 (d, <i>J</i> = 7.3 Hz, 2H), 7.34 (t, <i>J</i> = 7.4 Hz, 2H), 7.27 (t, <i>J</i> = 7.3 Hz, 1H), 6.80 (s, 1H), 4.98 (t, <i>J</i> = 6.1 Hz, 2H), 4.52 (s, 2H), 3.67 (s, 2H), 3.35 (s, 2H), 2.79 (s, 6H), 2.50(s, 3H, overlapped with DMSO-<i>d</i><sub>6</sub> peak). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 166.3 (-C-), 164.5 (-C-), 154.6 (-C-), 146.0 (-C-), 142.0 (-C-), 137.5 (-C-), 129.0 (-<u class="uu">C</u>H-), 128.5 (-<u class="uu">C</u>H-), 127.4 (-<u class="uu">C</u>H-), 108.4 (-<u class="uu">C</u>H-), 54.2 (-<u class="uu">C</u>H<sub>2</sub>-), 43.2 (-<u class="uu">C</u>H<sub>2</sub>-), 42.5 (-<u class="uu">C</u>H<sub>3</sub>), 34.5 (-<u class="uu">C</u>H<sub>2</sub>-), 24.7 (-<u class="uu">C</u>H<sub>3</sub>). HRMS (ESI): <i>m</i>/<i>z</i> calcd for C<sub>18</sub>H<sub>22</sub>N<sub>9</sub>S<sub>2</sub>(M + H)<sup>+</sup>,428.1439; found,428.1402.</div></div><div id="sec5_1_5_21" class="NLM_sec NLM_sec_level_2"><div id="ac_i115" class="anchor-spacer"></div><h3 class="article-section__title" id="_i115"> <i>N</i>,<i>N</i>-Dimethyl-2-(5-((5-methyl-2-(phenethylthio)[1,2,4]triazolo[1,5-<i>a</i>]pyrimidin-7-yl)thio)-1<i>H</i>-tetrazol-1-yl)ethan-1-amine Hydrochloride (<b>E21</b>)</h3><div class="NLM_p last">White solid; yield, 82%. HPLC purity: 99.16%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.09 (s, 1H), 7.41–7.30 (m, 4H), 7.30–7.19 (m, 1H), 6.81 (s, 1H), 5.00 (t, <i>J</i> = 6.2 Hz, 2H), 3.70 (s, 2H), 3.56–3.42 (m, 2H), 3.14–3.03 (m, 2H), 2.81 (s, 6H), 2.52 (s, 3H, overlapped with DMSO-<i>d</i><sub>6</sub> peak). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 166.5 (-C-), 164.4 (-C-), 154.7 (-C-), 146.0 (-C-), 142.0 (-C-), 139.9 (-C-), 128.6 (-<u class="uu">C</u>H-), 128.4 (-<u class="uu">C</u>H-), 126.4 (-<u class="uu">C</u>H-), 108.3 (-<u class="uu">C</u>H-), 54.1 (-<u class="uu">C</u>H<sub>2</sub>-), 43.1 (-<u class="uu">C</u>H<sub>2</sub>-), 42.4 (-<u class="uu">C</u>H<sub>3</sub>), 35.2 (-<u class="uu">C</u>H<sub>2</sub>-), 32.0 (-<u class="uu">C</u>H<sub>2</sub>-), 24.5 (-<u class="uu">C</u>H<sub>3</sub>). HRMS (ESI): <i>m</i>/<i>z</i> calcd for C<sub>19</sub>H<sub>24</sub>N<sub>9</sub>S<sub>2</sub> (M + H)<sup>+</sup>, 442.1590; found, 442.1558.</div></div><div id="sec5_1_5_22" class="NLM_sec NLM_sec_level_2"><div id="ac_i116" class="anchor-spacer"></div><h3 class="article-section__title" id="_i116"> <i>N</i>,<i>N</i>-Dimethyl-2-(5-((5-methyl-2-((3-phenylpropyl)thio)[1,2,4]triazolo[1,5-<i>a</i>]pyrimidin-7-yl)thio)-1<i>H</i>-tetrazol-1-yl)ethan-1-amine Hydrochloride (<b>E22</b>)</h3><div class="NLM_p last">White solid; yield, 80%. HPLC purity: 99.80%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.92 (s, 1H), 7.30 (t, <i>J</i> = 7.4 Hz, 2H), 7.24 (d, <i>J</i> = 7.0 Hz, 2H), 7.19 (t, <i>J</i> = 7.1 Hz, 1H), 6.79 (s, 1H), 4.98 (t, <i>J</i> = 6.0 Hz, 2H), 3.69 (s, 2H), 3.24 (t, <i>J</i> = 7.1 Hz, 2H), 2.77 (m, 8H), 2.50 (s, 3H, overlapped with DMSO-<i>d</i><sub>6</sub> peak), 2.16–2.01 (m, 2H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 166.6 (-C-), 164.4 (-C-), 154.6 (-C-), 146.0 (-C-), 141.9 (-C-), 141.0 (-C-), 128.4 (−CH-), 128.3 (-<u class="uu">C</u>H-), 125.9 (-<u class="uu">C</u>H-), 108.3 (-<u class="uu">C</u>H-), 54.2 (-<u class="uu">C</u>H<sub>2</sub>-), 42.5 (-<u class="uu">C</u>H<sub>2</sub>-), 40.1 (-<u class="uu">C</u>H<sub>3</sub>), 34.0 (-<u class="uu">C</u>H<sub>2</sub>-), 30.7 (-<u class="uu">C</u>H<sub>2</sub>-), 30.2 (-<u class="uu">C</u>H<sub>2</sub>-), 24.5 (-<u class="uu">C</u>H<sub>3</sub>). HRMS (ESI): <i>m</i>/<i>z</i> calcd for C<sub>20</sub>H<sub>26</sub>N<sub>9</sub>S<sub>2</sub> (M + H)<sup>+</sup>, 456.1747; found, 456.1723.</div></div><div id="sec5_1_5_23" class="NLM_sec NLM_sec_level_2"><div id="ac_i117" class="anchor-spacer"></div><h3 class="article-section__title" id="_i117"> 2-(5-((2-(((5-Chlorobenzo[<i>b</i>]thiophen-3-yl)methyl)thio)-5-methyl[1,2,4]triazolo[1,5-<i>a</i>]pyrimidin-7-yl)thio)-1<i>H</i>-tetrazol-1-yl)-<i>N</i>,<i>N</i>-dimethylethan-1-amine Hydrochloride (<b>E23</b>)</h3><div class="NLM_p last">White solid; yield, 76%. HPLC purity: 99.05%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.08 (s, 1H), 8.11 (d, <i>J</i> = 1.8 Hz, 1H), 8.04 (d, <i>J</i> = 8.6 Hz, 1H), 7.90 (s, 1H), 7.44 (m, 1H), 6.82 (s, 1H), 4.99 (t, <i>J</i> = 6.1 Hz, 2H), 4.81 (s, 2H), 4.61 (t, <i>J</i> = 6.0 Hz, 1H), 3.68 (s, 2H), 2.79 (s, 6H), 2.50 (s, 3H, overlapped with DMSO-<i>d</i><sub>6</sub> peak). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 166.0 (-C-), 164.6 (-C-), 154.6 (-C-), 146.0 (-C-), 142.1 (-C-), 139.0 (-C-), 138.2 (-C-), 130.9 (-C-), 129.6 (-C-), 128.7 (-<u class="uu">C</u>H-), 124.7 (-<u class="uu">C</u>H-), 124.6 (-<u class="uu">C</u>H-), 121.6 (-<u class="uu">C</u>H-), 108.5 (-<u class="uu">C</u>H-), 54.2 (-<u class="uu">C</u>H<sub>2</sub>-), 43.2 (-<u class="uu">C</u>H<sub>2</sub>-), 42.5 (-<u class="uu">C</u>H<sub>3</sub>), 27.9 (-<u class="uu">C</u>H<sub>2</sub>-), 24.6 (-<u class="uu">C</u>H<sub>3</sub>). HRMS (ESI): <i>m</i>/<i>z</i> calcd for C<sub>20</sub>H<sub>21</sub>ClN<sub>9</sub>S<sub>3</sub> (M + H)<sup>+</sup>, 518.0765; found, 518.0732.</div></div><div id="sec5_1_5_24" class="NLM_sec NLM_sec_level_2"><div id="ac_i118" class="anchor-spacer"></div><h3 class="article-section__title" id="_i118"> 4-(((7-((1-(2-(Dimethylamino)ethyl)-1<i>H</i>-tetrazol-5-yl)thio)-5-methyl[1,2,4]triazolo[1,5-<i>a</i>]pyrimidin-2-yl)thio)methyl)-2<i>H</i>-benzo[<i>h</i>]chromen-2-one Hydrochloride (<b>E24</b>)</h3><div class="NLM_p last">White solid; yield, 74%. HPLC purity: 97.10%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.66 (s, 1H), 8.47–8.30 (m, 1H), 8.07 (m, 2H), 7.99–7.89 (m, 1H), 7.81–7.67 (m, 2H), 6.86–6.73 (m, 2H), 4.95 (t, <i>J</i> = 5.9 Hz, 2H), 4.88 (s, 2H), 3.67 (s, 2H), 2.74 (d, <i>J</i> = 49.7 Hz, 6H), 2.50 (s, 3H, overlapped with DMSO-<i>d</i><sub>6</sub> peak). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 172.0 (-C-), 167.0 (-C-), 160.7 (-C-), 159.4 (-C-), 154.8 (-C-), 152.1 (-C-), 151.4 (-C-), 151.0 (-C-), 150.2 (-C-), 134.3 (-<u class="uu">C</u>H-), 128.9 (-<u class="uu">C</u>H-), 128.0 (-<u class="uu">C</u>H-), 127.5 (-<u class="uu">C</u>H-), 124.1 (-<u class="uu">C</u>H-), 122.3 (-<u class="uu">C</u>H-), 121.6 (-C-), 121.0 (-<u class="uu">C</u>H-), 114.4 (-<u class="uu">C</u>H-), 113.6 (-<u class="uu">C</u>H-), 98.6 (-<u class="uu">C</u>H-), 55.8 (-<u class="uu">C</u>H<sub>2</sub>-), 42.6 (-<u class="uu">C</u>H<sub>3</sub>), 40.7 (-<u class="uu">C</u>H<sub>2</sub>-), 31.2 (-<u class="uu">C</u>H<sub>2</sub>), 21.0 (-<u class="uu">C</u>H<sub>3</sub>). HRMS (ESI): <i>m</i>/<i>z</i> calcd for C<sub>25</sub>H<sub>24</sub>N<sub>9</sub>O<sub>2</sub>S<sub>2</sub> (M + H)<sup>+</sup>, 546.1488; found, 546.1452.</div></div><div id="sec5_1_5_25" class="NLM_sec NLM_sec_level_2"><div id="ac_i119" class="anchor-spacer"></div><h3 class="article-section__title" id="_i119"> 2-(5-((2-(Benzylthio)-5-phenyl[1,2,4]triazolo[1,5-<i>a</i>]pyrimidin-7-yl)thio)-1<i>H</i>-tetrazol-1-yl)-<i>N</i>,<i>N</i>-dimethylethan-1-amine Hydrochloride (<b>E25</b>)</h3><div class="NLM_p last">White solid; yield, 81%. HPLC purity: 97.33%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.16 (s, 1H), 8.13 (d, <i>J</i> = 6.4 Hz, 2H), 7.63–7.46 (m, 6H), 7.35 (t, <i>J</i> = 7.4 Hz, 2H), 7.28 (t, <i>J</i> = 7.2 Hz, 1H), 5.05 (t, <i>J</i> = 6.1 Hz, 2H), 4.55 (s, 2H), 3.75 (s, 2H), 2.83 (s, 6H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 167.2 (-C-), 159.8 (-C-), 154.9 (-C-), 146.6 (-C-), 142.4 (-C-), 137.4 (-C-), 135.6 (-C-), 131.5 (-<u class="uu">C</u>H-), 129.1 (-<u class="uu">C</u>H-), 129.0 (-<u class="uu">C</u>H-), 128.5 (-<u class="uu">C</u>H-), 127.8 (-<u class="uu">C</u>H-), 127.4 (-<u class="uu">C</u>H-), 106.2 (-<u class="uu">C</u>H-), 54.1 (-<u class="uu">C</u>H<sub>2</sub>-), 43.2 (-<u class="uu">C</u>H<sub>2</sub>-), 42.5 (-<u class="uu">C</u>H<sub>3</sub>), 34.5 (-<u class="uu">C</u>H<sub>2</sub>-). HRMS (ESI): <i>m</i>/<i>z</i> calcd for C<sub>23</sub>H<sub>24</sub>N<sub>9</sub>S<sub>2</sub> (M + H)<sup>+</sup>, 490.1590; found, 490.1559.</div></div><div id="sec5_1_5_26" class="NLM_sec NLM_sec_level_2"><div id="ac_i120" class="anchor-spacer"></div><h3 class="article-section__title" id="_i120"> 2-(5-((2-(Benzylthio)-5-ethyl[1,2,4]triazolo[1,5-<i>a</i>]pyrimidin-7-yl)thio)-1<i>H</i>-tetrazol-1-yl)-<i>N</i>,<i>N</i>-dimethylethan-1-amine Hydrochloride (<b>E26</b>)</h3><div class="NLM_p last">White solid; yield, 91%. HPLC purity: 98.56%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.10 (s, 1H), 7.50 (d, <i>J</i> = 7.3 Hz, 2H), 7.34 (t, <i>J</i> = 7.4 Hz, 2H), 7.27 (t, <i>J</i> = 7.2 Hz, 1H), 6.84 (s, 1H), 4.98 (t, <i>J</i> = 6.0 Hz, 2H), 4.52 (s, 2H), 3.65 (s, 2H), 2.80 (m, 8H), 1.20 (t, <i>J</i> = 7.5 Hz, 3H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 168.9 (-C-), 166.3 (-C-), 154.7 (-C-), 146.1 (-C-), 142.0 (-C-), 137.5 (-C-), 129.0 (-<u class="uu">C</u>H-), 128.5 (-<u class="uu">C</u>H-), 127.4 (-<u class="uu">C</u>H-), 108.0 (-<u class="uu">C</u>H-), 54.3 (-<u class="uu">C</u>H<sub>2</sub>-), 43.2 (-<u class="uu">C</u>H<sub>2</sub>-), 42.6 (-<u class="uu">C</u>H<sub>3</sub>), 34.5 (-<u class="uu">C</u>H<sub>2</sub>-), 30.7 (-<u class="uu">C</u>H<sub>2</sub>-), 12.4 (-<u class="uu">C</u>H<sub>3</sub>). HRMS (ESI): <i>m</i>/<i>z</i> calcd for C<sub>19</sub>H<sub>24</sub>N<sub>9</sub>S<sub>2</sub> (M + H)<sup>+</sup>, 442.1590; found, 442.1566.</div></div><div id="sec5_1_5_27" class="NLM_sec NLM_sec_level_2"><div id="ac_i121" class="anchor-spacer"></div><h3 class="article-section__title" id="_i121"> <i>N</i>,<i>N</i>-Dimethyl-2-(5-((5-phenyl-2-(prop-2-yn-1-ylthio)[1,2,4]triazolo[1,5-<i>a</i>]pyrimidin-7-yl)thio)-1<i>H</i>-tetrazol-1-yl)ethan-1-amine Hydrochloride (<b>E27</b>)</h3><div class="NLM_p last">White solid; yield, 87%. HPLC purity: 97.72%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.23 (s, 1H), 8.15 (m, 2H), 7.67–7.43 (m, 4H), 5.07 (t, <i>J</i> = 6.4 Hz, 2H), 4.14 (d, <i>J</i> = 2.5 Hz, 2H), 3.77 (s, 2H), 3.28 (t, <i>J</i> = 2.5 Hz, 1H), 2.84 (s, 6H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 166.1 (-C-), 159.9 (-C-), 155.0 (-C-), 146.6 (-C-), 142.6 (-C-), 135.5 (-C-), 131.6 (-<u class="uu">C</u>H-), 129.1 (-<u class="uu">C</u>H-), 127.8 (-<u class="uu">C</u>H-), 106.5 (-<u class="uu">C</u>H-), 79.8 (-C-), 74.1 (-<u class="uu">C</u>H), 54.0 (-<u class="uu">C</u>H<sub>2</sub>-), 43.2 (-<u class="uu">C</u>H<sub>2</sub>-), 42.4 (-<u class="uu">C</u>H<sub>3</sub>), 19.1 (-<u class="uu">C</u>H<sub>2</sub>-). HRMS (ESI): <i>m</i>/<i>z</i> calcd for C<sub>19</sub>H<sub>20</sub>N<sub>9</sub>S<sub>2</sub> (M + H)<sup>+</sup>, 438.1277; found, 438.1253.</div></div><div id="sec5_1_5_28" class="NLM_sec NLM_sec_level_2"><div id="ac_i122" class="anchor-spacer"></div><h3 class="article-section__title" id="_i122"> <i>N</i>,<i>N</i>-Dimethyl-2-(5-((5-methyl-2-((4-nitrobenzyl)thio)[1,2,4]triazolo[1,5-<i>a</i>]pyrimidin-7-yl)thio)-1<i>H</i>-tetrazol-1-yl)ethan-1-amine Hydrochloride (<b>E28</b>)</h3><div class="NLM_p last">White solid; yield, 78%. HPLC purity: 99.26%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.20 (d, <i>J</i> = 8.5 Hz, 2H), 7.80 (d, <i>J</i> = 8.5 Hz, 2H), 6.79 (s, 1H), 4.97 (s, 2H), 4.63 (d, <i>J</i> = 14.1 Hz, 2H), 3.65 (s, 2H), 2.81 (d, <i>J</i> = 22.6 Hz, 6H), 2.50 (s, 3H, overlapped with DMSO-<i>d</i><sub>6</sub> peak). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 165.6 (-C-), 164.6 (-C-), 154.6 (-C-), 146.6 (-C-), 146.1 (-C-), 146.0 (-C-), 142.2 (-C-), 130.3 (-<u class="uu">C</u>H-), 123.5 (-<u class="uu">C</u>H-), 108.5 (-<u class="uu">C</u>H-), 54.3 (-<u class="uu">C</u>H<sub>2</sub>-), 43.3 (-<u class="uu">C</u>H<sub>2</sub>-), 42.6 (-<u class="uu">C</u>H<sub>3</sub>), 33.7 (-<u class="uu">C</u>H<sub>2</sub>-), 24.6 (-<u class="uu">C</u>H<sub>3</sub>). HRMS (ESI): <i>m</i>/<i>z</i> calcd for C<sub>18</sub>H<sub>21</sub>N<sub>10</sub>O<sub>2</sub>S<sub>2</sub>(M + H)<sup>+</sup>, 473.1284; found, 473.1254.</div></div><div id="sec5_1_5_29" class="NLM_sec NLM_sec_level_2"><div id="ac_i123" class="anchor-spacer"></div><h3 class="article-section__title" id="_i123"> <i>N</i>,<i>N</i>-Dimethyl-2-(5-((5-methyl-2-((2-nitrobenzyl)thio)[1,2,4]triazolo[1,5-<i>a</i>]pyrimidin-7-yl)thio)-1<i>H</i>-tetrazol-1-yl)ethan-1-amine Hydrochloride (<b>E29</b>)</h3><div class="NLM_p last">White solid; yield, 89%. HPLC purity: 98.90%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.24 (s, 1H), 7.81 (m, 4H), 6.78 (s, 1H), 4.93 (s, 2H), 4.82 (s, 2H), 4.57 (s, 1H), 2.78 (d, <i>J</i> = 45.2 Hz, 9H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 165.8 (-C-), 164.7 (-C-), 154.5 (-C-), 148.0 (-C-), 145.9 (-C-), 142.2 (-C-), 133.9 (-C-), 133.0 (-<u class="uu">C</u>H-), 132.6 (-<u class="uu">C</u>H-), 129.2 (-<u class="uu">C</u>H-), 125.1 (-<u class="uu">C</u>H-), 108.5 (-<u class="uu">C</u>H-), 54.4 (-<u class="uu">C</u>H<sub>2</sub>-), 43.2 (-<u class="uu">C</u>H<sub>2</sub>-), 42.6 (-<u class="uu">C</u>H<sub>3</sub>), 32.0 (-<u class="uu">C</u>H<sub>2</sub>-), 24.6 (-<u class="uu">C</u>H<sub>3</sub>). HRMS (ESI): <i>m</i>/<i>z</i> calcd for C<sub>18</sub>H<sub>21</sub>N<sub>10</sub>O<sub>2</sub>S<sub>2</sub>(M + H)<sup>+</sup>, 473.1284; found, 473.1253.</div></div><div id="sec5_1_5_30" class="NLM_sec NLM_sec_level_2"><div id="ac_i124" class="anchor-spacer"></div><h3 class="article-section__title" id="_i124"> 2-(5-((2-((4-Fluorobenzyl)thio)-5-methyl[1,2,4]triazolo[1,5-<i>a</i>]pyrimidin-7-yl)thio)-1<i>H</i>-tetrazol-1-yl)-<i>N</i>,<i>N</i>-dimethylethan-1-amine Hydrochloride (<b>E30</b>)</h3><div class="NLM_p last">White solid; yield, 77%. HPLC purity: 98.53%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.01 (s, 1H), 7.55 (m, 2H), 7.16 (t, <i>J</i> = 8.9 Hz, 2H), 6.80 (s, 1H), 4.99 (t, <i>J</i> = 6.2 Hz, 2H), 4.52 (s, 2H), 3.70 (s, 2H), 2.81 (s, 6H), 2.50 (s, 3H, overlapped with DMSO-<i>d</i><sub>6</sub> peak). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 166.1 (-C-), 164.6 (-C-), 162.6 (-C-), 160.12 (-C-), 154.6 (-C-), 146.0 (-C-), 142.1 (-C-), 134.0 (-<u class="uu">C</u>H-), 133.9 (-<u class="uu">C</u>H-), 131.1 (-<u class="uu">C</u>H-), 131.0 (-<u class="uu">C</u>H-), 115.3 (-<u class="uu">C</u>H-), 115.1 (-<u class="uu">C</u>H-), 108.4 (-<u class="uu">C</u>H-), 54.1 (-<u class="uu">C</u>H<sub>2</sub>-), 43.1 (-<u class="uu">C</u>H<sub>2</sub>-), 42.5 (-<u class="uu">C</u>H<sub>3</sub>), 33.7 (-<u class="uu">C</u>H<sub>2</sub>-), 24.6 (-<u class="uu">C</u>H<sub>3</sub>). HRMS (ESI): <i>m</i>/<i>z</i> calcd for C<sub>18</sub>H<sub>21</sub>FN<sub>9</sub>S<sub>2</sub> (M + H)<sup>+</sup>, 446.1339; found, 446.1316.</div></div><div id="sec5_1_5_31" class="NLM_sec NLM_sec_level_2"><div id="ac_i125" class="anchor-spacer"></div><h3 class="article-section__title" id="_i125"> 2-(5-((2-((4-Chlorobenzyl)thio)-5-methyl[1,2,4]triazolo[1,5-<i>a</i>]pyrimidin-7-yl)thio)-1<i>H</i>-tetrazol-1-yl)-<i>N</i>,<i>N</i>-dimethylethan-1-amine Hydrochloride (<b>E31</b>)</h3><div class="NLM_p last">White solid; yield, 76%. HPLC purity: 99.10%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.18 (s, 1H), 7.54 (d, <i>J</i> = 8.4 Hz, 2H), 7.39 (d, <i>J</i> = 8.4 Hz, 2H), 6.80 (s, 1H), 4.99 (t, <i>J</i> = 6.1 Hz, 2H), 4.51 (s, 2H), 3.68 (s, 2H), 2.82 (d, <i>J</i> = 15.9 Hz, 6H), 2.50 (s, 3H, overlapped with DMSO-<i>d</i><sub>6</sub> peak). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 166.0 (-C-), 164.6 (-C-), 154.6 (-C-), 146.0 (-C-), 142.1 (-C-), 136.9 (-C-), 131.9 (-C-), 130.9 (-<u class="uu">C</u>H-), 128.4 (-<u class="uu">C</u>H-), 108.4 (-<u class="uu">C</u>H-), 54.2 (-<u class="uu">C</u>H<sub>2</sub>-), 43.2 (-<u class="uu">C</u>H<sub>2</sub>-), 42.5 (-<u class="uu">C</u>H<sub>3</sub>), 33.7 (-<u class="uu">C</u>H<sub>2</sub>-), 24.6 (-<u class="uu">C</u>H<sub>3</sub>). HRMS (ESI): <i>m</i>/<i>z</i> calcd for C<sub>18</sub>H<sub>21</sub>ClN<sub>9</sub>S<sub>2</sub> (M + H)<sup>+</sup>, 462.1044; found, 462.1015.</div></div><div id="sec5_1_5_32" class="NLM_sec NLM_sec_level_2"><div id="ac_i126" class="anchor-spacer"></div><h3 class="article-section__title" id="_i126"> 2-(5-((2-((4-Bromobenzyl)thio)-5-methyl[1,2,4]triazolo[1,5-<i>a</i>]pyrimidin-7-yl)thio)-1<i>H</i>-tetrazol-1-yl)-<i>N</i>,<i>N</i>-dimethylethan-1-amine Hydrochloride (<b>E32</b>)</h3><div class="NLM_p last">White solid; yield, 79%. HPLC purity: 99.83%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.95 (s, 1H), 7.61–7.42 (m, 4H), 6.79 (s, 1H), 4.99 (t, <i>J</i> = 6.2 Hz, 2H), 4.50 (s, 2H), 3.70 (s, 2H), 2.81 (s, 6H), 2.50 (s, 3H, overlapped with DMSO-<i>d</i><sub>6</sub> peak). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 166.0 (-C-), 164.6 (-C-), 154.5 (-C-), 146.0 (-C-), 142.1 (-C-), 137.3 (-C-), 131.3 (-C-), 131.3 (-<u class="uu">C</u>H-), 120.4 (-<u class="uu">C</u>H-), 108.3 (-<u class="uu">C</u>H-), 54.1 (-<u class="uu">C</u>H<sub>2</sub>-), 43.1 (-<u class="uu">C</u>H<sub>2</sub>-), 42.5 (-<u class="uu">C</u>H<sub>3</sub>), 33.7 (-<u class="uu">C</u>H<sub>2</sub>-), 24.6 (-<u class="uu">C</u>H<sub>3</sub>). HRMS (ESI): <i>m</i>/<i>z</i> calcd for C<sub>18</sub>H<sub>21</sub>BrN<sub>9</sub>S<sub>2</sub>(M + H)<sup>+</sup>, 506.0539; found, 506.0510.</div></div><div id="sec5_1_5_33" class="NLM_sec NLM_sec_level_2"><div id="ac_i127" class="anchor-spacer"></div><h3 class="article-section__title" id="_i127"> 2-(5-((2-((2-Bromobenzyl)thio)-5-methyl[1,2,4]triazolo[1,5-<i>a</i>]pyrimidin-7-yl)thio)-1<i>H</i>-tetrazol-1-yl)-<i>N</i>,<i>N</i>-dimethylethan-1-amine Hydrochloride (<b>E33</b>)</h3><div class="NLM_p last">White solid; yield, 65%. HPLC purity: 98.14%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.89 (s, 1H), 7.69 (t, <i>J</i> = 8.5 Hz, 2H), 7.42–7.32 (m, 1H), 7.26 (t, <i>J</i> = 7.0 Hz, 1H), 6.81 (s, 1H), 4.99 (t, <i>J</i> = 6.1 Hz, 2H), 4.61 (s, 2H), 3.69 (s, 2H), 2.81 (s, 6H), 2.50 (s, 3H, overlapped with DMSO-<i>d</i><sub>6</sub> peak). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 165.8 (-C-), 164.6 (-C-), 154.6 (-C-), 146.0 (-C-), 142.1 (-C-), 136.3 (-C-), 132.8 (-C-), 131.5 (-<u class="uu">C</u>H-), 129.7 (-<u class="uu">C</u>H-), 127.9 (-<u class="uu">C</u>H-), 124.1 (-<u class="uu">C</u>H-), 108.5 (-<u class="uu">C</u>H-), 54.1 (-<u class="uu">C</u>H<sub>2</sub>-), 43.1 (-<u class="uu">C</u>H<sub>2</sub>-), 42.4 (-<u class="uu">C</u>H<sub>3</sub>), 35.3 (-<u class="uu">C</u>H<sub>2</sub>-), 24.6 (-<u class="uu">C</u>H<sub>3</sub>). HRMS (ESI): <i>m</i>/<i>z</i> calcd for C<sub>18</sub>H<sub>21</sub>BrN<sub>9</sub>S<sub>2</sub> (M + H)<sup>+</sup>, 506.0539; found, 506.0507.</div></div><div id="sec5_1_5_34" class="NLM_sec NLM_sec_level_2"><div id="ac_i128" class="anchor-spacer"></div><h3 class="article-section__title" id="_i128"> 2-(5-((2-((3-Bromobenzyl)thio)-5-methyl[1,2,4]triazolo[1,5-<i>a</i>]pyrimidin-7-yl)thio)-1<i>H</i>-tetrazol-1-yl)-<i>N</i>,<i>N</i>-dimethylethan-1-amine Hydrochloride (<b>E34</b>)</h3><div class="NLM_p last">White solid; yield, 80%. HPLC purity: 97.86%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.26 (s, 1H), 7.74 (s, 1H), 7.53 (d, <i>J</i> = 7.6 Hz, 1H), 7.47 (d, <i>J</i> = 8.0 Hz, 1H), 7.31 (t, <i>J</i> = 7.8 Hz, 1H), 6.81 (s, 1H), 5.00 (t, <i>J</i> = 6.2 Hz, 2H), 4.52 (s, 2H), 3.71 (s, 2H), 2.81 (s, 6H), 2.50 (s, 3H, overlapped with DMSO-<i>d</i><sub>6</sub> peak). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 167.0 (-C-), 161.5 (-C-), 154.8 (-C-), 151.3 (-C-), 150.8 (-C-), 136.3 (-C-), 132.8 (-C-), 131.2 (-<u class="uu">C</u>H-), 129.7 (-<u class="uu">C</u>H-), 127.9 (-<u class="uu">C</u>H-), 124.0 (-<u class="uu">C</u>H-), 98.5 (-<u class="uu">C</u>H-), 55.9 (-<u class="uu">C</u>H<sub>2</sub>-), 42.6 (-<u class="uu">C</u>H<sub>3</sub>), 40.7 (-<u class="uu">C</u>H<sub>2</sub>-), 35.4 (-<u class="uu">C</u>H<sub>2</sub>-), 18.6 (-<u class="uu">C</u>H<sub>3</sub>). HRMS (ESI): <i>m</i>/<i>z</i> calcd for C<sub>18</sub>H<sub>21</sub>BrN<sub>9</sub>S<sub>2</sub> (M + H)<sup>+</sup>, 506.0539; found, 506.0500.</div></div><div id="sec5_1_5_35" class="NLM_sec NLM_sec_level_2"><div id="ac_i129" class="anchor-spacer"></div><h3 class="article-section__title" id="_i129"> <i>N</i>,<i>N</i>-Dimethyl-2-(5-((5-methyl-2-((4-methylbenzyl)thio)[1,2,4]triazolo[1,5-<i>a</i>]pyrimidin-7-yl)thio)-1<i>H</i>-tetrazol-1-yl)ethan-1-amine Hydrochloride (<b>E35</b>)</h3><div class="NLM_p last">White solid; yield, 74%. HPLC purity: 100.00%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.08 (s, 1H), 7.38 (d, <i>J</i> = 8.0 Hz, 2H), 7.14 (d, <i>J</i> = 7.9 Hz, 2H), 6.80 (s, 1H), 5.00 (t, <i>J</i> = 6.2 Hz, 2H), 4.47 (s, 2H), 3.70 (s, 2H), 2.81 (s, 6H), 2.50 (s, 3H, overlapped with DMSO-<i>d</i><sub>6</sub> peak), 2.28 (s, 3H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 166.3 (-C-), 164.5 (-C-), 154.6 (-C-), 146.0 (-C-), 142.0 (-C-), 136.6 (-C-), 134.3 (-<u class="uu">C</u>-), 129.0 (-<u class="uu">C</u>H-), 128.9 (-<u class="uu">C</u>H-), 108.4 (-<u class="uu">C</u>H-), 54.1 (-<u class="uu">C</u>H<sub>2</sub>-), 43.1 (-<u class="uu">C</u>H<sub>2</sub>-), 42.5 (-<u class="uu">C</u>H<sub>3</sub>), 34.3 (-<u class="uu">C</u>H<sub>2</sub>-), 24.6 (-<u class="uu">C</u>H<sub>3</sub>), 20.7 (-<u class="uu">C</u>H<sub>3</sub>). HRMS (ESI): <i>m</i>/<i>z</i> calcd for C<sub>19</sub>H<sub>24</sub>N<sub>9</sub>S<sub>2</sub> (M + H)<sup>+</sup>, 442.1590; found, 442.1561.</div></div><div id="sec5_1_5_36" class="NLM_sec NLM_sec_level_2"><div id="ac_i130" class="anchor-spacer"></div><h3 class="article-section__title" id="_i130"> <i>N</i>,<i>N</i>-Dimethyl-2-(5-((5-methyl-2-((4-(trifluoromethyl)benzyl)thio)[1,2,4]triazolo[1,5-<i>a</i>]pyrimidin-7-yl)thio)-1<i>H</i>-tetrazol-1-yl)ethan-1-amine Hydrochloride (<b>E36</b>)</h3><div class="NLM_p last">White solid; yield, 82%. HPLC purity: 99.59%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.22 (s, 1H), 7.73 (m, 4H), 6.80 (s, 1H), 5.00 (s, 2H), 4.61 (s, 2H), 3.71 (s, 2H), 2.81 (s, 6H), 2.50 (s, 3H, overlapped with DMSO-<i>d</i><sub>6</sub> peak). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 165.8 (-C-), 164.6 (-C-), 154.6 (-C-), 145.9 (-C-), 142.9 (-C-), 142.2 (-C-), 129.8 (-C-), 128.0 (-C-), 127.7 (-C-), 125.6 (-<u class="uu">C</u>H-), 125.3 (-<u class="uu">C</u>H-), 125.2 (-<u class="uu">C</u>H-), 122.8 (-<u class="uu">C</u>H-), 108.4 (-<u class="uu">C</u>H-), 54.1 (-<u class="uu">C</u>H<sub>2</sub>-), 43.1 (-<u class="uu">C</u>H<sub>2</sub>-), 42.4 (-<u class="uu">C</u>H<sub>3</sub>), 33.8 (-<u class="uu">C</u>H<sub>2</sub>-), 24.5 (-<u class="uu">C</u>H<sub>3</sub>). HRMS (ESI): <i>m</i>/<i>z</i> calcd for C<sub>19</sub>H<sub>21</sub>F<sub>3</sub>N<sub>9</sub>S<sub>2</sub>(M + H)<sup>+</sup>, 496.1307; found, 496.1271.</div></div><div id="sec5_1_5_37" class="NLM_sec NLM_sec_level_2"><div id="ac_i131" class="anchor-spacer"></div><h3 class="article-section__title" id="_i131"> <i>N</i>,<i>N</i>-Dimethyl-2-(5-((5-methyl-2-((2-(trifluoromethyl)benzyl)thio)[1,2,4]triazolo[1,5-<i>a</i>]pyrimidin-7-yl)thio)-1<i>H</i>-tetrazol-1-yl)ethan-1-amine Hydrochloride (<b>E37</b>)</h3><div class="NLM_p last">White solid; yield, 85%. HPLC purity: 98.97%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.78 (s, 1H), 7.81 (m, 2H), 7.68 (t, <i>J</i> = 7.3 Hz, 1H), 7.54 (t, <i>J</i> = 7.3 Hz, 1H), 6.82 (s, 1H), 4.98 (s, 2H), 4.71 (s, 2H), 3.69 (s, 2H), 2.81 (s, 6H), 2.50 (s, 3H, overlapped with DMSO-<i>d</i><sub>6</sub> peak). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 165.7 (-C-), 164.7 (-C-), 154.6 (-C-), 145.9 (-C-), 142.2 (-C-), 135.3 (-C-), 133.0 (-C-), 131.9 (-C-), 128.4 (-<u class="uu">C</u>H-), 126.3 (-<u class="uu">C</u>H-), 108.5 (-<u class="uu">C</u>H-), 54.1 (-<u class="uu">C</u>H<sub>2</sub>-), 43.2 (-<u class="uu">C</u>H<sub>2</sub>-), 42.5 (-<u class="uu">C</u>H<sub>3</sub>), 31.4 (-<u class="uu">C</u>H<sub>2</sub>-), 24.6 (-<u class="uu">C</u>H<sub>3</sub>). HRMS (ESI): <i>m</i>/<i>z</i> calcd for C<sub>19</sub>H<sub>21</sub>F<sub>3</sub>N<sub>9</sub>S<sub>2</sub> (M + H)<sup>+</sup>, 496.1307; found, 496.1295.</div></div><div id="sec5_1_5_38" class="NLM_sec NLM_sec_level_2"><div id="ac_i132" class="anchor-spacer"></div><h3 class="article-section__title" id="_i132"> <i>N</i>,<i>N</i>-Dimethyl-2-(5-((5-methyl-2-((3-(trifluoromethyl)benzyl)thio)[1,2,4]triazolo[1,5-<i>a</i>]pyrimidin-7-yl)thio)-1<i>H</i>-tetrazol-1-yl)ethan-1-amine Hydrochloride (<b>E38</b>)</h3><div class="NLM_p last">White solid; yield, 72%. HPLC purity: 99.36%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.00 (s, 1H), 7.90 (s, 1H), 7.84 (d, <i>J</i> = 7.5 Hz, 1H), 7.60 (m, 2H), 6.79 (s, 1H), 4.98 (t, <i>J</i> = 6.2 Hz, 2H), 4.62 (s, 2H), 3.69 (s, 2H), 2.81 (s, 6H), 2.50 (s, 3H, overlapped with DMSO-<i>d</i><sub>6</sub> peak). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 165.9 (-C-), 164.6 (-C-), 154.6 (-C-), 145.9 (-C-), 142.2 (-C-), 139.5 (-C-), 133.3 (-C-), 129.5 (-C-), 125.5 (-<u class="uu">C</u>H-), 124.1 (-<u class="uu">C</u>H-), 122.8 (-<u class="uu">C</u>H-), 108.3 (-<u class="uu">C</u>H-), 54.1 (-<u class="uu">C</u>H<sub>2</sub>-), 43.1 (-<u class="uu">C</u>H<sub>2</sub>-), 42.4 (-<u class="uu">C</u>H<sub>3</sub>), 33.8 (-<u class="uu">C</u>H<sub>2</sub>-), 24.6 (-<u class="uu">C</u>H<sub>3</sub>). HRMS (ESI): <i>m</i>/<i>z</i> calcd for C<sub>19</sub>H<sub>21</sub>F<sub>3</sub>N<sub>9</sub>S<sub>2</sub> (M + H)<sup>+</sup>, 496.1307; found, 496.1294.</div></div><div id="sec5_1_5_39" class="NLM_sec NLM_sec_level_2"><div id="ac_i133" class="anchor-spacer"></div><h3 class="article-section__title" id="_i133"> 2-(5-((2-((4-Methoxybenzyl)thio)-5-methyl[1,2,4]triazolo[1,5-<i>a</i>]pyrimidin-7-yl)thio)-1<i>H</i>-tetrazol-1-yl)-<i>N</i>,<i>N</i>-dimethylethan-1-amine Hydrochloride (<b>E39</b>)</h3><div class="NLM_p last">White solid; yield, 71%. HPLC purity: 96.85%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.94 (s, 1H), 7.42 (d, <i>J</i> = 8.6 Hz, 2H), 6.89 (d, <i>J</i> = 8.7 Hz, 2H), 6.80 (s, 1H), 4.99 (t, <i>J</i> = 6.2 Hz, 2H), 4.46 (s, 2H), 3.73 (s, 3H), 3.70 (s, 2H), 2.82 (s, 6H), 2.50 (s, 3H, overlapped with DMSO-<i>d</i><sub>6</sub> peak). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 166.4 (-C-), 164.5 (-C-), 158.6 (-C-), 154.6 (-C-), 146.0 (-C-), 142.0 (-C-), 130.3 (-C-), 129.2 (-<u class="uu">C</u>H-), 113.9 (-<u class="uu">C</u>H-), 108.3 (-<u class="uu">C</u>H-), 55.1 (-<u class="uu">C</u>H<sub>3</sub>), 54.1 (-<u class="uu">C</u>H<sub>2</sub>-), 43.1 (-<u class="uu">C</u>H<sub>2</sub>-), 42.5 (-<u class="uu">C</u>H<sub>3</sub>), 34.1 (-<u class="uu">C</u>H<sub>2</sub>-), 24.6 (-<u class="uu">C</u>H<sub>3</sub>). HRMS (ESI): <i>m</i>/<i>z</i> calcd for C<sub>19</sub>H<sub>24</sub>N<sub>9</sub>OS<sub>2</sub> (M + H)<sup>+</sup>, 458.1539; found, 458.1510.</div></div><div id="sec5_1_5_40" class="NLM_sec NLM_sec_level_2"><div id="ac_i134" class="anchor-spacer"></div><h3 class="article-section__title" id="_i134"> 2-(5-((2-((4-Chloro-3-fluorobenzyl)thio)-5-methyl[1,2,4]triazolo[1,5-<i>a</i>]pyrimidin-7-yl)thio)-1<i>H</i>-tetrazol-1-yl)-<i>N</i>,<i>N</i>-dimethylethan-1-amine Hydrochloride (<b>E40</b>)</h3><div class="NLM_p last">White solid; yield, 64%. HPLC purity: 99.85%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.12 (s, 1H), 7.65–7.51 (m, 2H), 7.40 (d, <i>J</i> = 8.3 Hz, 1H), 6.80 (s, 1H), 4.99 (t, <i>J</i> = 6.0 Hz, 2H), 4.58 (d, <i>J</i> = 37.5 Hz, 2H), 3.77–3.57 (m, 2H), 2.82 (d, <i>J</i> = 15.7 Hz, 6H), 2.50 (s, 3H, overlapped with DMSO-<i>d</i><sub>6</sub> peak). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 165.8 (-C-), 164.6 (-C-), 158.1 (-C-), 155.6 (-C-), 154.6 (-C-), 145.9 (-C-), 142.2 (-C-), 139.9 (-C-), 139.8 (-C-), 130.5 (-<u class="uu">C</u>H-), 126.4 (-<u class="uu">C</u>H-), 118.3 (-<u class="uu">C</u>H-), 117.5 (-<u class="uu">C</u>H-), 117.2 (-<u class="uu">C</u>H-), 108.4 (-<u class="uu">C</u>H-), 54.2 (-<u class="uu">C</u>H<sub>2</sub>-), 43.2 (-<u class="uu">C</u>H<sub>2</sub>-), 42.5 (-<u class="uu">C</u>H<sub>3</sub>), 33.4 (-<u class="uu">C</u>H<sub>2</sub>-), 24.6 (-<u class="uu">C</u>H<sub>3</sub>). HRMS (ESI): <i>m</i>/<i>z</i> calcd for C<sub>18</sub>H<sub>20</sub>ClFN<sub>9</sub>S<sub>2</sub> (M + H)<sup>+</sup>, 480.0950; found, 480.0923.</div></div><div id="sec5_1_5_41" class="NLM_sec NLM_sec_level_2"><div id="ac_i135" class="anchor-spacer"></div><h3 class="article-section__title" id="_i135"> 2-(5-((2-((3-Chloro-4-fluorobenzyl)thio)-5-methyl[1,2,4]triazolo[1,5-<i>a</i>]pyrimidin-7-yl)thio)-1<i>H</i>-tetrazol-1-yl)-<i>N</i>,<i>N</i>-dimethylethan-1-amine Hydrochloride (<b>E41</b>)</h3><div class="NLM_p last">White solid; yield, 78%. HPLC purity: 99.85%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.91 (s, 1H), 7.75 (m, 1H), 7.59–7.51 (m, 1H), 7.38 (t, <i>J</i> = 9.0 Hz, 1H), 6.79 (s, 1H), 4.98 (t, <i>J</i> = 6.2 Hz, 2H), 4.52 (s, 2H), 3.69 (s, 2H), 2.81 (s, 6H), 2.50 (s, 3H, overlapped with DMSO-<i>d</i><sub>6</sub> peak). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 165.9 (-C-), 165.6 (-C-), 164.6 (-C-), 162.0 (-C-), 157.6 (-C-), 155.2 (-C-), 155.1 (-C-), 154.6 (-C-), 152.4 (-<u class="uu">C</u>H-), 145.9 (-<u class="uu">C</u>H-), 142.1 (-<u class="uu">C</u>H-), 137.2 (-<u class="uu">C</u>H-), 136.0 (-<u class="uu">C</u>H-), 131.0 (-<u class="uu">C</u>H-), 129.8 (-<u class="uu">C</u>H-), 129.8 (-<u class="uu">C</u>H-), 119.2 (-<u class="uu">C</u>H-), 119.0 (-<u class="uu">C</u>H-), 116.9 (-<u class="uu">C</u>H-), 116.7 (-<u class="uu">C</u>H-), 116.0 (-<u class="uu">C</u>H-), 108.4 (-<u class="uu">C</u>H-), 54.2 (-<u class="uu">C</u>H<sub>2</sub>-), 43.1 (-<u class="uu">C</u>H<sub>2</sub>-), 42.5 (-<u class="uu">C</u>H<sub>3</sub>), 33.1 (-<u class="uu">C</u>H<sub>2</sub>-), 24.6 (-<u class="uu">C</u>H<sub>3</sub>). HRMS (ESI): <i>m</i>/<i>z</i> calcd for C<sub>18</sub>H<sub>20</sub>ClFN<sub>9</sub>S<sub>2</sub> (M + H)<sup>+</sup>, 480.0950; found, 480.0922.</div></div><div id="sec5_1_5_42" class="NLM_sec NLM_sec_level_2"><div id="ac_i136" class="anchor-spacer"></div><h3 class="article-section__title" id="_i136"> 2-(5-((2-((2,6-Dichlorobenzyl)thio)-5-methyl[1,2,4]triazolo[1,5-<i>a</i>]pyrimidin-7-yl)thio)-1<i>H</i>-tetrazol-1-yl)-<i>N</i>,<i>N</i>-dimethylethan-1-amine Hydrochloride (<b>E42</b>)</h3><div class="NLM_p last">White solid; yield, 64%. HPLC purity: 98.70%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.70 (s, 1H), 7.56 (s, 2H), 7.42 (s, 1H), 6.84 (s, 1H), 4.99 (s, 2H), 4.82 (s, 2H), 3.70 (s, 2H), 2.82 (s, 6H), 2.50 (s, 3H, overlapped with DMSO-<i>d</i><sub>6</sub> peak). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 165.5 (-C-), 164.7 (-C-), 154.6 (-C-), 146.0 (-C-), 142.2 (-C-), 135.1 (-C-), 131.9 (-C-), 130.6 (-C-), 128.8 (-<u class="uu">C</u>H-), 124.5 (-<u class="uu">C</u>H-), 108.6 (-<u class="uu">C</u>H-), 54.1 (-<u class="uu">C</u>H<sub>2</sub>-), 43.1 (-<u class="uu">C</u>H<sub>2</sub>-), 42.4 (-<u class="uu">C</u>H<sub>3</sub>), 31.2 (-<u class="uu">C</u>H<sub>2</sub>-), 24.6 (-<u class="uu">C</u>H<sub>3</sub>). HRMS (ESI): <i>m</i>/<i>z</i> calcd for C<sub>18</sub>H<sub>20</sub>C<sub>l2</sub>N<sub>9</sub>S<sub>2</sub> (M + H)<sup>+</sup>, 496.0654; found, 496.0618.</div></div><div id="sec5_1_5_43" class="NLM_sec NLM_sec_level_2"><div id="ac_i137" class="anchor-spacer"></div><h3 class="article-section__title" id="_i137"> 2-(5-((2-((4-Chlorobenzyl)thio)-5,6-dimethyl[1,2,4]triazolo[1,5-<i>a</i>]pyrimidin-7-yl)thio)-1<i>H</i>-tetrazol-1-yl)-<i>N</i>,<i>N</i>-dimethylethan-1-amine Hydrochloride (<b>E43</b>)</h3><div class="NLM_p last">White solid; yield, 88%. HPLC purity: 99.61%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.14 (s, 1H), 7.42 (d, <i>J</i> = 8.5 Hz, 2H), 7.35 (d, <i>J</i> = 8.5 Hz, 2H), 5.13–4.93 (m, 2H), 4.37 (s, 2H), 3.70–3.84 (m, 2H), 2.85 (s, 6H), 2.69 (s, 3H), 2.63 (s, 3H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 165.2 (-C-), 164.8 (-C-), 153.3 (-C-), 149.2 (-C-), 136.6 (-C-), 134.7 (-C-), 131.9 (-<u class="uu">C</u>H-), 130.7 (-<u class="uu">C</u>H-), 128.4 (-C-), 124.7 (-C-), 53.9 (-<u class="uu">C</u>H<sub>2</sub>-), 43.1 (-<u class="uu">C</u>H<sub>2</sub>-), 42.6 (-<u class="uu">C</u>H<sub>3</sub>), 33.7 (-<u class="uu">C</u>H<sub>2</sub>-), 24.4 (-<u class="uu">C</u>H<sub>3</sub>), 16.0 (-<u class="uu">C</u>H<sub>3</sub>). HRMS (ESI): <i>m</i>/<i>z</i> calcd for C<sub>19</sub>H<sub>23</sub>ClN<sub>9</sub>S<sub>2</sub>(M + H)<sup>+</sup>, 476.1200; found, 476.1172.</div></div><div id="sec5_1_5_44" class="NLM_sec NLM_sec_level_2"><div id="ac_i138" class="anchor-spacer"></div><h3 class="article-section__title" id="_i138"> 2-(5-((2-((4-Chlorobenzyl)thio)-5-methyl-6-pentyl[1,2,4]triazolo[1,5-<i>a</i>]pyrimidin-7-yl)thio)-1<i>H</i>-tetrazol-1-yl)-<i>N</i>,<i>N</i>-dimethylethan-1-amine Hydrochloride (<b>E44</b>)</h3><div class="NLM_p last">White solid; yield, 82%. HPLC purity: 99.05%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 7.46–7.29 (m, 4H), 5.06 (t, <i>J</i> = 6.6 Hz, 2H), 4.34 (s, 2H), 3.78 (s, 2H), 3.11–2.99 (m, 2H), 2.87 (s, 6H), 2.72 (s, 3H), 1.73–1.58 (m, 2H), 1.51–1.29 (m, 4H), 0.90 (t, <i>J</i> = 7.1 Hz, 3H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 165.1 (-C-), 164.7 (-C-), 153.2 (-C-), 149.1 (-C-), 136.5 (-C-), 135.2 (-C-), 132.0 (-C-), 130.6 (-<u class="uu">C</u>H-), 128.4 (-C-), 128.2 (-<u class="uu">C</u>H-), 53.9 (-<u class="uu">C</u>H<sub>2</sub>-), 43.1 (-<u class="uu">C</u>H<sub>2</sub>-), 42.7 (-<u class="uu">C</u>H<sub>3</sub>), 33.7 (-<u class="uu">C</u>H<sub>2</sub>-), 31.2 (-<u class="uu">C</u>H<sub>2</sub>-), 29.4 (-<u class="uu">C</u>H<sub>2</sub>-), 28.8 (-<u class="uu">C</u>H<sub>2</sub>-), 23.7 (-<u class="uu">C</u>H<sub>2</sub>-), 21.8 (-<u class="uu">C</u>H<sub>3</sub>), 13.8 (-<u class="uu">C</u>H<sub>3</sub>). HRMS (ESI): <i>m</i>/<i>z</i> calcd for C<sub>23</sub>H<sub>31</sub>ClN<sub>9</sub>S<sub>2</sub> (M + H)<sup>+</sup>, 532.1826; found, 532.1788.</div></div><div id="sec5_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i139" class="anchor-spacer"></div><h3 class="article-section__title" id="_i139"> Molecular Docking</h3><div class="NLM_p last">All the molecular docking study, including protein preparation, ligand preparation, and docking simulation, was performed using MOE 2015.10. The crystal structure of a small molecular inhibitor <b>DI-591</b> binding to DCN1 (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5UFI">5UFI</a>) was prepared by default parameters using QuickPrep module, and the protonation states of the ionizable residues were adjusted at p<i>K</i><sub>a</sub> = 7. <b>DI-591</b> was first redocked into the protein to investigate the reliability of our docking simulation using the Dock module. The three-dimensional structures of <b>E1</b>, <b>WS-383</b>, and <b>WS-524</b> were generated by taking energy minimization and conformational search. Then compounds <b>E1</b>, <b>WS-383</b>, and <b>WS-524</b> were docked into the binding pocket of DCN1. After the docking simulations, the 20 best-scored ligand–protein complexes of each ligands were kept for further analyses.</div></div><div id="sec5_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i140" class="anchor-spacer"></div><h3 class="article-section__title" id="_i140"> Molecular Dynamics Simulations</h3><div class="NLM_p last">Molecular dynamics simulations study was performed using Gromacs 5.1.4. Docking results of <b>WS-383</b> and <b>WS-524</b> and crystal structure of <b>DI-591</b> in complex with DCN1 were regarded as the starting structures. The Amber99SB-ildn force field and gaff force field were employed for the proteins and the three ligands, respectively. The electrostatic potential (ESP) of each compound was computed at the HF/6-31G(d) level in the Gaussian 09 program, and the partial atomic charges for these compounds were generated using the RESP protocol. During the preparation of the protein structure for simulation, the protonation state of the protein and ligands was determined using the PROPKA 3.1. The complex was placed in the center of a cubic periodic box and solvated by the addition of TIP3P water molecules. The net charge on the system was then neutralized by adding 0.15 mol/L of NaCl as counterions. The energy was minimized using the steepest descent algorithm. The system was then heated to 300 K during 500 ps in <i>NVT</i> ensemble. The pressure was then equilibrated to 1 atm during a 500 ps <i>NPT</i> simulation. In both simulations, all heavy atoms were position restrained with the force constant of 1000 kJ/(mol·nm<sup>2</sup>). Production simulations were performed for 20 ns for each complex, and the temperature and pressure were maintained at 300 K and 1 atm. Then the binding free energy was calculated using MM-PBSA method with g_mmpbsa.</div></div><div id="sec5_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i141" class="anchor-spacer"></div><h3 class="article-section__title" id="_i141"> Constructs, Protein Expression, and Purification</h3><div class="NLM_p last">Full length of DCN1 was cloned into the expression vector pGEX-4T-1. Then the vector was transformed into <i>E. coli</i>. BL21(DE3). Cells were cultured at 37 °C, and the protein was induced with isopropyl-β-<span class="smallcaps smallerCapital">d</span>-thiogalactoside (IPTG) at a final concentration of 0.5 mM at 20 °C for 16 h. Cells were collected by centrifugation and resuspended in the binding buffer (140 mM NaCl, 2.7 mM KCl, 10 mM Na<sub>2</sub>HPO<sub>4</sub>, 1.8 mM KH<sub>2</sub>PO<sub>4</sub>, pH 7.3). After sonication and centrifuge, the cleared lysate was subjected to purification with GSH-Sepharose beads (GE, USA). The beads were washed with binding buffer, and the recombinant was eluted and collected with elution buffer (50 mM Tris-HCl, 10 mM reduced glutathione, pH 8.0). Then the recombinant was concentrated with 10 kDa molecular cutoff filter. The concentrated recombinant was then quantified with BCA assay and stored at −80 °C. The Ac-UBC12<sub>1–12</sub> peptide corresponding to N-terminally acetylated residues 1–12 of human UBC12 with an additional C-terminal biotin was synthesized by Shenggong biological technology.</div></div><div id="sec5_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i142" class="anchor-spacer"></div><h3 class="article-section__title" id="_i142"> HTRF Assay</h3><div class="NLM_p last">HTRF assays were carried out in white 384-well microliter plates at a final volume of 20 μL per well. In the screening, the assay cocktail was prepared as a mixture of 30 nM GST-DCN1, 2.84 μM Ac-UBC12-biotin, 0.125 μg/mL anti-GST-cryptate (Cisbio, France) and 2.5 μg/mL streptavidin-d2 (Cisbio, France) in assay buffer (50 mM HEPES, 50 mM NaCl, 400 mM KF, 0.1% BSA, 0.1% Tween 20, pH 6.0). The assay cocktail was incubated for 1 h at 25 °C, and compound to be screened was added from DMSO stock solutions. The assay mixture was incubated for another 1 h at 25 °C, and then the plate was read with Envision microplate reader (PerkinElmer, USA) equipped with HTRF modules for excitation at 314 nm and emission at 615 and 665 nm. The 665/615 ratio was used as the HTRF signal in calculations. To avoid interference of false positive compounds, PAINS screening of the synthesized compounds was carried out by employing the online program (“PAINS-Remover”, <a href="http://www.cbligand.org/PAINS/" class="extLink">http://www.cbligand.org/PAINS/</a>), and all the tested compounds passed the filter.</div></div><div id="sec5_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i143" class="anchor-spacer"></div><h3 class="article-section__title" id="_i143"> Reversible Assay</h3><div class="NLM_p">For the dilution assay, around 40-fold IC<sub>50</sub> of the tested compounds or DMSO as control was incubated with DCN1 for 1 h in HTRF assay buffer. Then the mixer was diluted 80-fold with HTRF assay buffer in order to have the same component as HTRF assay. After that, 2.84 μM Ac-UBC12-biotin, 0.125 μg/mL anti-GST-cryptate (Cisbio, France), and 2.5 μg/mL streptavidin-d2 (Cisbio, France) were added. Assay buffer was added to the mixture to have the final volume as 20 μL for following HTRF assay. This represents an 80-fold dilution of the inhibitor concentration, which is expected to yield the same inhibition rate for an irreversible inhibitor or significant difference for a reversible inhibitor.</div><div class="NLM_p last">For the ultrafiltration experiment, around 40-fold IC<sub>50</sub> of the tested compounds or DMSO as control was incubated with DCN1 for 1 h in HTRF assay buffer. Then the mixer was subject to 5 kDa cutoff ultrafiltration tube (Millipore, USA). After six round centrifuge at 10K rpm for 20 min, 2.84 μM Ac-UBC12-biotin, 0.125 μg/mL anti-GST-cryptate (Cisbio, France), and 2.5 μg/mL streptavidin-d2 (Cisbio, France) were added. Assay buffer was added to the mixture to have the final volume as 20 μL for following HTRF assay. This represents a compound clean wash step for ultrafiltration, which is expected to yield the same inhibition rate for an irreversible inhibitor or significant difference for a reversible inhibitor.</div></div><div id="sec5_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i144" class="anchor-spacer"></div><h3 class="article-section__title" id="_i144"> BTK Enzyme Assay</h3><div class="NLM_p last">BTK was purchased from Carna Biosciences. The kinases (5 nM) were assayed with test compound in a final volume of 5 μL, 3 μM peptide 2 (5-FAM-EAIYAAPFAKKK), 90 μM ATP, and reaction buffer (50 mM HEPES, pH 7.5, 0.0015% Brij-35, 10 mM MgCl<sub>2</sub>, 2 mM DTT). After incubation at 28 °C for 60 min, the reactions were stopped by adding 25 μL of stop buffer (100 mM HEPES, pH 7.5, 0.015% Brij-35, 50 mM EDTA). The reaction mixture was analyzed on Caliper, and the conversion values were converted to inhibition values.</div></div><div id="sec5_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i145" class="anchor-spacer"></div><h3 class="article-section__title" id="_i145"> EGFR L858R Enzyme Assay</h3><div class="NLM_p last">EGFR L858R was purchased from Carna Biosciences. The kinase (2.5 nM) was assayed with the test compound in a final volume of 5 μL, 3 μM kinase peptide 22 (5-FAM-EEPLYWSFPAKKK-CONH<sub>2</sub>), 80.8 μM ATP, and reaction buffer (50 mM HEPES, 0.01% Triton X-100, 10 mM MgCl<sub>2</sub>, 2 mM DTT). After incubation at room temperature for 30 min, the reactions were stopped by adding 25 μL of stop buffer (100 mM HEPES, 0.015% Brij-35, 50 mM EDTA). The reaction mixture was analyzed with Caliper EZ reader, and the conversion values were converted to inhibition.</div></div><div id="sec5_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i146" class="anchor-spacer"></div><h3 class="article-section__title" id="_i146"> CDKs Enzyme Assay</h3><div class="NLM_p last">CDK1/cyclinB, CDK2/CycA2, CDK7/cyclinH/MAT1, and CDK9/cyclinT1 kinase were purchased from Millipore, and CDK4/CycD3 and CDK6/cycD3 were purchased from Carna. The kinases were assayed with 3 μM peptide 8 (5-FAM-IPTSPITTTYFFFKKK-COOH for CDK4 and CDK6), peptide 18 (5-FAM-QSPKKG-CONH<sub>2</sub> for CDK1 and CDK2), peptide CTD3 (5-FAM-ACSYSPTSPSYSPTSPSYSPTSPSKK for CDK7 and CDK9) in the 20/30/280/800/77/10 μM ATP (for CDK1/CDK2/CDK4/CDK6/CDK7CDK9), and the test compounds in a final volume of 5 μL and the reaction buffer (50 mM HEPES, pH 7.5, 0.0015% Brij-35, 10 mM MgCl<sub>2</sub>, 2 mM DTT). After incubation at 28 °C for 60 min, the reactions were stopped by adding 25 μL of stop buffer (100 mM HEPES, pH 7.5, 0.015% Brij-35, 50 mM EDTA). The reaction mixture was analyzed on Caliper, and the conversion values were converted to inhibition values.</div></div><div id="sec5_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i147" class="anchor-spacer"></div><h3 class="article-section__title" id="_i147"> Cellular Thermal Shift Assay</h3><div class="NLM_p last">Cells (5 × 10<sup>5</sup>) were treated with a compound or with DMSO for 1 h, washed with PBS three times, and dissolved in 50 μL of PBS supplemented with a protease inhibitor cocktail, followed by heating at the indicated temperatures in a PCR instrument (Senso, Germany). Treated cells were then subjected to snap-freezing in liquid nitrogen and thawed on ice for 3 cycles. The protein levels of DCN1 in equal amounts of the supernatant were examined by Western blotting assay. GAPDH was used as the control. Results are representative of three independent experiments.</div></div><div id="sec5_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i148" class="anchor-spacer"></div><h3 class="article-section__title" id="_i148"> Cell Lines and Chemicals</h3><div class="NLM_p last">Human gastric epithelial cell line GES-1 and gastric cancer cell line MGC-803 were cultured in DMEM medium (BI, Israel). SGC-7901, HGC-27, BGC-823, and KYSE70 cells were cultured in RPMI 1640 medium (BI, Israel), supplemented with 10% fetal bovine serum, and incubated at 5% CO<sub>2</sub> incubator (Thermo, USA) at 37 °C.</div></div><div id="sec5_12" class="NLM_sec NLM_sec_level_2"><div id="ac_i149" class="anchor-spacer"></div><h3 class="article-section__title" id="_i149"> RT-qPCR</h3><div class="NLM_p last">Total RNA from cells was purified using Ultrapure RNA extraction kit (CWBIO, China) with the aid of DNase I (NEB, USA) to remove the possible DNA contamination, and cDNA was obtained using HiFi-MMLV cDNA kit (CWBIO, China). qPCR analysis was conducted using a LightCycler 96 real-time PCR system (Roche, USA). Primers for UBC12: F, 5′-AGCCAGTCCTTACGATAAACTCC-3′; R, 5′-TGCACGTTCTGCTCAAACAGCC-3′. Primers for DCN1: F, 5′-TTGCGTGGAAGTTCAGAGCAGC-3′; R, 5′-GTTCCATCTTGGGTATCTGGGC-3′. Primers for GAPDH: F, 5′-GTCTCCTCTGACTTCAACAGCG-3′; R, 5′-ACCACCCTGTTGCTGTAGCCAA-3′.</div></div><div id="sec5_13" class="NLM_sec NLM_sec_level_2"><div id="ac_i150" class="anchor-spacer"></div><h3 class="article-section__title" id="_i150"> Western Blotting</h3><div class="NLM_p last">Cells were seeded into 6-well plates and incubated with different concentrations of candidate compounds for 24 h. Then cells were collected and the total protein was collected with RIPA in the presence of complete protease inhibitor cocktail (Roche, USA). Cell lysates were quantified with BCA kit (Beyotime Biotechnology, China) and denatured with protein loading buffer in 100 °C water for 10 min. An amount of 30 μg of the denatured samples was subjected to SDS–PAGE (acrylamide 5% for stacking gel and 8% for separating gel) and transferred to NC (nitrocellulose) membranes (Millipore, USA) and incubated with the primary antibodies overnight at 4 °C. The membranes were washed with PBST 3 times, 5 min per time. Then membranes were incubated with suitable HRP-conjugated second antibodies. For exposure, ECL Western blotting substrate from ThermoFisher was used. Imaging was done with an X-ray film (Koda, Japan). The sources of primary antibodies were as follows: Cullin1 (Abcam, ab75817), Cullin2 (Abcam, ab166917), Cullin3 (Abcam, ab75851), Cullin4a (Abcam, ab92554), Cullin4b (Proteintech, 12916-1-AP), Cullin5 (Abcam, ab184177), UBC12 (Abcam, ab109507), p21 (Cell Signaling Technology, no. 2947), p27 (Cell Signaling Technology, no. 3686), NRF2 (Cell Signaling Technology, no. 4399), DCN1 (GenWay Biotech, GWB-E3D700).</div></div><div id="sec5_14" class="NLM_sec NLM_sec_level_2"><div id="ac_i151" class="anchor-spacer"></div><h3 class="article-section__title" id="_i151"> Intracellular ROS Assay</h3><div class="NLM_p last">Intracellular reactive oxygen species (ROS) was measured by DCFH-DA (Beyotime) according to the manual operation. Briefly, cells were digested and washed twice by serum-free medium, loaded with 1 μM 2′,7′-dichlorofluorescein diacetate (DCFH-DA). The cells were then incubated at 37 °C for 30 min. The cells were then washed twice with serum-free medium and suspended in PBS. Ten-thousand cells were collected and analyzed at 488 nm excitation and 525 nm emission wavelength by a flow cytometer (FACSCalibur, BD).</div></div><div id="sec5_15" class="NLM_sec NLM_sec_level_2"><div id="ac_i152" class="anchor-spacer"></div><h3 class="article-section__title" id="_i152"> Statistical Analysis</h3><div class="NLM_p last">Data were expressed as the mean ± SD. Statistical differences in two groups were performed by Student’s <i>t</i>-test, and one-way analysis of variance was used for a multiple-group comparison. <i>P</i> values of <0.05 or less were considered statistically significant.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i153"><a href="/doi/suppl/10.1021/acs.jmedchem.9b00113" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_34484" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_34484" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a href="/" class="ext-link">ACS Publications website</a> at DOI: <a href="/doi/abs/10.1021/acs.jmedchem.9b00113" class="ext-link">10.1021/acs.jmedchem.9b00113</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Coordinates of modeled structures in PDB format: <b>E1</b> in complex with DCN1; <b>DI-591</b> in complex with DCN1 after molecular dynamics simulations (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5UFI">5UFI</a>); <b>WS-383</b> in complex with DCN1; <b>WS-383</b> in complex with DCN1 after molecular dynamics simulations; <b>WS-524</b> in complex with DCN1; <b>WS-524</b> in complex with DCN1 after molecular dynamics simulations (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00113/suppl_file/jm9b00113_si_001.zip" class="ext-link">ZIP</a>)</p></li><li><p class="inline">Molecular formula strings and some data (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00113/suppl_file/jm9b00113_si_002.csv" class="ext-link">CSV</a>)</p></li><li><p class="inline">Assay development results, binding mode, and NMR, HRMS, and HPLC spectra of representative compounds (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00113/suppl_file/jm9b00113_si_003.pdf" class="ext-link">PDF</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00113/suppl_file/jm9b00113_si_001.zip">jm9b00113_si_001.zip (464.83 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00113/suppl_file/jm9b00113_si_002.csv">jm9b00113_si_002.csv (53.56 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00113/suppl_file/jm9b00113_si_003.pdf">jm9b00113_si_003.pdf (30.38 MB)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.9b00113" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_44464" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_44464" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Bin Yu</span> - <span class="hlFld-Affiliation affiliation">School
of Pharmaceutical Sciences and Institute of Drug Discovery & Development, Zhengzhou University, Zhengzhou 450001, China</span>; 
    <span class="hlFld-Affiliation affiliation">Co-Innovation
Center of Henan Province for New Drug R & D and Preclinical Safety, Zhengzhou 450001, China</span>; 
    <span class="hlFld-Affiliation affiliation">Key Laboratory
of Advanced Technology of Drug Preparation Technologies, Zhengzhou
University, Ministry of Education of China, Zhengzhou 450001, China</span>; 
    <span class="hlFld-Affiliation affiliation">State Key
Laboratory of Pharmaceutical Biotechnology, Nanjing University, Nanjing 210023, Jiangsu, People’s Republic of China</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-0531-2236" title="Orcid link">http://orcid.org/0000-0002-0531-2236</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#b5ccc0d7dcdbf5cfcfc09bd0d1c09bd6db"><span class="__cf_email__" data-cfemail="c5bcb0a7acab85bfbfb0eba0a1b0eba6ab">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yi-Chao Zheng</span> - <span class="hlFld-Affiliation affiliation">School
of Pharmaceutical Sciences and Institute of Drug Discovery & Development, Zhengzhou University, Zhengzhou 450001, China</span>; 
    <span class="hlFld-Affiliation affiliation">Co-Innovation
Center of Henan Province for New Drug R & D and Preclinical Safety, Zhengzhou 450001, China</span>; 
    <span class="hlFld-Affiliation affiliation">Key Laboratory
of Advanced Technology of Drug Preparation Technologies, Zhengzhou
University, Ministry of Education of China, Zhengzhou 450001, China</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#f881919b90999782909d969fb882828dd69d9c8dd69b96"><span class="__cf_email__" data-cfemail="1e67777d767f7164767b70795e64646b307b7a6b307d70">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Shaomeng Wang</span> - <span class="hlFld-Affiliation affiliation">School
of Pharmaceutical Sciences and Institute of Drug Discovery & Development, Zhengzhou University, Zhengzhou 450001, China</span>; 
    <span class="hlFld-Affiliation affiliation">Departments
of Internal Medicine, Pharmacology, Medicinal Chemistry, University of Michigan, 1600 Huron Parkway, Ann
Arbor, Michigan 48109, United States</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#d2a1bab3bdbfb7bcb592a7bfbbb1bafcb7b6a7"><span class="__cf_email__" data-cfemail="5e2d363f31333b30391e2b33373d36703b3a2b">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Hong-Min Liu</span> - <span class="hlFld-Affiliation affiliation">School
of Pharmaceutical Sciences and Institute of Drug Discovery & Development, Zhengzhou University, Zhengzhou 450001, China</span>; 
    <span class="hlFld-Affiliation affiliation">Co-Innovation
Center of Henan Province for New Drug R & D and Preclinical Safety, Zhengzhou 450001, China</span>; 
    <span class="hlFld-Affiliation affiliation">Key Laboratory
of Advanced Technology of Drug Preparation Technologies, Zhengzhou
University, Ministry of Education of China, Zhengzhou 450001, China</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-6771-9421" title="Orcid link">http://orcid.org/0000-0001-6771-9421</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#472b2e322f2a073d3d3269222332692429"><span class="__cf_email__" data-cfemail="c0aca9b5a8ad80babab5eea5a4b5eea3ae">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Shuai Wang</span> - <span class="hlFld-Affiliation affiliation">School
of Pharmaceutical Sciences and Institute of Drug Discovery & Development, Zhengzhou University, Zhengzhou 450001, China</span>; 
    <span class="hlFld-Affiliation affiliation">Co-Innovation
Center of Henan Province for New Drug R & D and Preclinical Safety, Zhengzhou 450001, China</span>; 
    <span class="hlFld-Affiliation affiliation">Key Laboratory
of Advanced Technology of Drug Preparation Technologies, Zhengzhou
University, Ministry of Education of China, Zhengzhou 450001, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Lijie Zhao</span> - <span class="hlFld-Affiliation affiliation">School
of Pharmaceutical Sciences and Institute of Drug Discovery & Development, Zhengzhou University, Zhengzhou 450001, China</span>; 
    <span class="hlFld-Affiliation affiliation">Co-Innovation
Center of Henan Province for New Drug R & D and Preclinical Safety, Zhengzhou 450001, China</span>; 
    <span class="hlFld-Affiliation affiliation">Key Laboratory
of Advanced Technology of Drug Preparation Technologies, Zhengzhou
University, Ministry of Education of China, Zhengzhou 450001, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Xiao-Jing Shi</span> - <span class="hlFld-Affiliation affiliation">School
of Pharmaceutical Sciences and Institute of Drug Discovery & Development, Zhengzhou University, Zhengzhou 450001, China</span>; 
    <span class="hlFld-Affiliation affiliation">Co-Innovation
Center of Henan Province for New Drug R & D and Preclinical Safety, Zhengzhou 450001, China</span>; 
    <span class="hlFld-Affiliation affiliation">Key Laboratory
of Advanced Technology of Drug Preparation Technologies, Zhengzhou
University, Ministry of Education of China, Zhengzhou 450001, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Lina Ding</span> - <span class="hlFld-Affiliation affiliation">School
of Pharmaceutical Sciences and Institute of Drug Discovery & Development, Zhengzhou University, Zhengzhou 450001, China</span>; 
    <span class="hlFld-Affiliation affiliation">Co-Innovation
Center of Henan Province for New Drug R & D and Preclinical Safety, Zhengzhou 450001, China</span>; 
    <span class="hlFld-Affiliation affiliation">Key Laboratory
of Advanced Technology of Drug Preparation Technologies, Zhengzhou
University, Ministry of Education of China, Zhengzhou 450001, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Linlin Yang</span> - <span class="hlFld-Affiliation affiliation">Department
of Pharmacology, School of Basic Medical Sciences, Zhengzhou University, Zhengzhou 450001, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Zhi-Zheng Wang</span> - <span class="hlFld-Affiliation affiliation">School
of Pharmaceutical Sciences and Institute of Drug Discovery & Development, Zhengzhou University, Zhengzhou 450001, China</span>; 
    <span class="hlFld-Affiliation affiliation">Co-Innovation
Center of Henan Province for New Drug R & D and Preclinical Safety, Zhengzhou 450001, China</span>; 
    <span class="hlFld-Affiliation affiliation">Key Laboratory
of Advanced Technology of Drug Preparation Technologies, Zhengzhou
University, Ministry of Education of China, Zhengzhou 450001, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Dandan Shen</span> - <span class="hlFld-Affiliation affiliation">School
of Pharmaceutical Sciences and Institute of Drug Discovery & Development, Zhengzhou University, Zhengzhou 450001, China</span>; 
    <span class="hlFld-Affiliation affiliation">Co-Innovation
Center of Henan Province for New Drug R & D and Preclinical Safety, Zhengzhou 450001, China</span>; 
    <span class="hlFld-Affiliation affiliation">Key Laboratory
of Advanced Technology of Drug Preparation Technologies, Zhengzhou
University, Ministry of Education of China, Zhengzhou 450001, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Kai Tang</span> - <span class="hlFld-Affiliation affiliation">School
of Pharmaceutical Sciences and Institute of Drug Discovery & Development, Zhengzhou University, Zhengzhou 450001, China</span>; 
    <span class="hlFld-Affiliation affiliation">Co-Innovation
Center of Henan Province for New Drug R & D and Preclinical Safety, Zhengzhou 450001, China</span>; 
    <span class="hlFld-Affiliation affiliation">Key Laboratory
of Advanced Technology of Drug Preparation Technologies, Zhengzhou
University, Ministry of Education of China, Zhengzhou 450001, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Xiao-Jing Li</span> - <span class="hlFld-Affiliation affiliation">School
of Pharmaceutical Sciences and Institute of Drug Discovery & Development, Zhengzhou University, Zhengzhou 450001, China</span>; 
    <span class="hlFld-Affiliation affiliation">Co-Innovation
Center of Henan Province for New Drug R & D and Preclinical Safety, Zhengzhou 450001, China</span>; 
    <span class="hlFld-Affiliation affiliation">Key Laboratory
of Advanced Technology of Drug Preparation Technologies, Zhengzhou
University, Ministry of Education of China, Zhengzhou 450001, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">MAA Mamun</span> - <span class="hlFld-Affiliation affiliation">School
of Pharmaceutical Sciences and Institute of Drug Discovery & Development, Zhengzhou University, Zhengzhou 450001, China</span>; 
    <span class="hlFld-Affiliation affiliation">Co-Innovation
Center of Henan Province for New Drug R & D and Preclinical Safety, Zhengzhou 450001, China</span>; 
    <span class="hlFld-Affiliation affiliation">Key Laboratory
of Advanced Technology of Drug Preparation Technologies, Zhengzhou
University, Ministry of Education of China, Zhengzhou 450001, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Huiju Li</span> - <span class="hlFld-Affiliation affiliation">School
of Pharmaceutical Sciences and Institute of Drug Discovery & Development, Zhengzhou University, Zhengzhou 450001, China</span>; 
    <span class="hlFld-Affiliation affiliation">Co-Innovation
Center of Henan Province for New Drug R & D and Preclinical Safety, Zhengzhou 450001, China</span>; 
    <span class="hlFld-Affiliation affiliation">Key Laboratory
of Advanced Technology of Drug Preparation Technologies, Zhengzhou
University, Ministry of Education of China, Zhengzhou 450001, China</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>S.W., L.Z., X.-J.S., and L. D. contributed equally.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i155">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_28217" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_28217" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">This work was supported by the National Key R&D Program of China (Grant 2016YFA0501800); National Natural Science Foundation of China (Grants 81430085, 81773562, 81703326, 81602961); China Postdoctoral Science Foundation (Grant 2018M630840); the Open Fund of State Key Laboratory of Pharmaceutical Biotechnology, Nanjing University, China (Grant KF-GN-201902); Key Research Program of Henan Province (Grants 161100310100, 182102310123); Outstanding Young Talent Research Fund of Zhengzhou University (Grant 1521331002); Key Scientific Research Project for Higher Education by Department of Education of Henan Educational Committee (Grants 15A350018, 18B350009); the Starting Grant of Zhengzhou University (Grant 32210533), and International Training of High-Level Talents in Henan (2017) from Henan Administration of Foreign Experts Affairs.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations Used</td></tr><tr><td class="NLM_term">DCN1</td><td class="NLM_def"><p class="first last">defective in cullin neddylation protein 1</p></td></tr><tr><td class="NLM_term">UBC12</td><td class="NLM_def"><p class="first last">ubiquitin-conjugating enzyme E2M</p></td></tr><tr><td class="NLM_term">Cul</td><td class="NLM_def"><p class="first last">cullin</p></td></tr><tr><td class="NLM_term">NEDD8</td><td class="NLM_def"><p class="first last">neural precursor cell expressed developmentally down-regulated protein 8</p></td></tr><tr><td class="NLM_term">NAE</td><td class="NLM_def"><p class="first last">NEDD8 activating enzyme</p></td></tr><tr><td class="NLM_term">NRF2</td><td class="NLM_def"><p class="first last">nuclear factor erythroid 2-related factor 2</p></td></tr><tr><td class="NLM_term">BTK</td><td class="NLM_def"><p class="first last">Bruton’s tyrosine kinase</p></td></tr><tr><td class="NLM_term">CDK</td><td class="NLM_def"><p class="first last">cyclin-dependent kinase</p></td></tr><tr><td class="NLM_term">p21</td><td class="NLM_def"><p class="first last">cyclin-dependent kinase inhibitor 1</p></td></tr><tr><td class="NLM_term">p27</td><td class="NLM_def"><p class="first last">cyclin-dependent kinase inhibitor 1B</p></td></tr><tr><td class="NLM_term">HTRF</td><td class="NLM_def"><p class="first last">homogeneous time-resolved fluorescence</p></td></tr><tr><td class="NLM_term">SCC</td><td class="NLM_def"><p class="first last">squamous cell carcinoma</p></td></tr><tr><td class="NLM_term">NC</td><td class="NLM_def"><p class="first last">nitrocellulose</p></td></tr><tr><td class="NLM_term">CMC-Na</td><td class="NLM_def"><p class="first last">sodium carboxymethyl cellulose</p></td></tr><tr><td class="NLM_term">SD</td><td class="NLM_def"><p class="first last">standard deviation</p></td></tr><tr><td class="NLM_term">FBS</td><td class="NLM_def"><p class="first last">fetal bovine serum</p></td></tr><tr><td class="NLM_term"><i>E. coli</i></td><td class="NLM_def"><p class="first last"><i>Escherichia coli</i></p></td></tr><tr><td class="NLM_term">SKP1</td><td class="NLM_def"><p class="first last">S-phase kinase associated protein 1</p></td></tr><tr><td class="NLM_term">RBX1</td><td class="NLM_def"><p class="first last">RING-box protein 1</p></td></tr><tr><td class="NLM_term">RBX2</td><td class="NLM_def"><p class="first last">RING-box protein 2</p></td></tr><tr><td class="NLM_term">SCF complex</td><td class="NLM_def"><p class="first last">Skp, cullin, F-box containing complex</p></td></tr><tr><td class="NLM_term">CDK</td><td class="NLM_def"><p class="first last">cyclin-dependent kinase</p></td></tr><tr><td class="NLM_term">AP2</td><td class="NLM_def"><p class="first last">activator protein 2</p></td></tr><tr><td class="NLM_term">MMP2</td><td class="NLM_def"><p class="first last">matrix metalloproteinase 2</p></td></tr><tr><td class="NLM_term">SAR</td><td class="NLM_def"><p class="first last">structure–activity relationship</p></td></tr><tr><td class="NLM_term">CETSA</td><td class="NLM_def"><p class="first last">cellular thermal shift assay</p></td></tr><tr><td class="NLM_term">TLC</td><td class="NLM_def"><p class="first last">thin-layer chromatography</p></td></tr><tr><td class="NLM_term">TMS</td><td class="NLM_def"><p class="first last">tetramethylsilane</p></td></tr><tr><td class="NLM_term">HRMS</td><td class="NLM_def"><p class="first last">high-resolution mass spectra</p></td></tr><tr><td class="NLM_term">HPLC</td><td class="NLM_def"><p class="first last">high-performance liquid chromatography</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i157">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_71668" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_71668" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 57 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Petroski, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deshaies, R. J.</span></span> <span> </span><span class="NLM_article-title">Function and regulation of cullin–RING ubiquitin ligases</span>. <i>Nat. Rev. Mol. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">9</span>– <span class="NLM_lpage">20</span>, <span class="refDoi"> DOI: 10.1038/nrm1547</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=10.1038%2Fnrm1547" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=15688063" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=1%3ACAS%3A528%3ADC%252BD2MXlvVWg" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2005&pages=9-20&author=M.+D.+Petroskiauthor=R.+J.+Deshaies&title=Function+and+regulation+of+cullin%E2%80%93RING+ubiquitin+ligases&doi=10.1038%2Fnrm1547"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Function and regulation of cullin-RING ubiquitin ligases</span></div><div class="casAuthors">Petroski, Matthew D.; Deshaies, Raymond J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Molecular Cell Biology</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">9-20</span>CODEN:
                <span class="NLM_cas:coden">NRMCBP</span>;
        ISSN:<span class="NLM_cas:issn">1471-0072</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Cullin-RING complexes comprise the largest known class of ubiquitin ligases.  Owing to the great diversity of their substrate-receptor subunits, it is possible that there are hundreds of distinct cullin-RING ubiquitin ligases in eukaryotic cells, which establishes these enzymes as key mediators of post-translational protein regulation.  Here, the authors focus on the compn., regulation, and function of cullin-RING ligases, and describe how these enzymes can be characterized by a set of general principles.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqFw5LuMMHZoLVg90H21EOLACvtfcHk0lhh-NSdrYEGuw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXlvVWg&md5=297f142f44f971c6858306b1adb36254</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Fnrm1547&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrm1547%26sid%3Dliteratum%253Aachs%26aulast%3DPetroski%26aufirst%3DM.%2BD.%26aulast%3DDeshaies%26aufirst%3DR.%2BJ.%26atitle%3DFunction%2520and%2520regulation%2520of%2520cullin%25E2%2580%2593RING%2520ubiquitin%2520ligases%26jtitle%3DNat.%2520Rev.%2520Mol.%2520Cell%2520Biol.%26date%3D2005%26volume%3D6%26spage%3D9%26epage%3D20%26doi%3D10.1038%2Fnrm1547" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, Y.</span></span> <span> </span><span class="NLM_article-title">Cullin-RING ligases as attractive anti-cancer targets</span>. <i>Curr. Pharm. Des.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">3215</span>– <span class="NLM_lpage">3225</span>, <span class="refDoi"> DOI: 10.2174/13816128113199990300</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=10.2174%2F13816128113199990300" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=23151137" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=1%3ACAS%3A528%3ADC%252BC3sXotFagtL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2013&pages=3215-3225&author=Y.+Zhaoauthor=Y.+Sun&title=Cullin-RING+ligases+as+attractive+anti-cancer+targets&doi=10.2174%2F13816128113199990300"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Cullin-RING ligases as attractive anti-cancer targets</span></div><div class="casAuthors">Zhao, Yongchao; Sun, Yi</div><div class="citationInfo"><span class="NLM_cas:title">Current Pharmaceutical Design</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">3215-3225</span>CODEN:
                <span class="NLM_cas:coden">CPDEFP</span>;
        ISSN:<span class="NLM_cas:issn">1381-6128</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  The ubiquitin-proteasome system (UPS) promotes the timely degrdn. of short-lived proteins with key regulatory roles in a vast array of biol. processes, such as cell cycle progression, oncogenesis and genome integrity.  Thus, abnormal regulation of UPS disrupts the protein homeostasis and causes many human diseases, particularly cancer.  Indeed, the FDA approval of bortezomib, the first class of general proteasome inhibitor, for the treatment of multiple myeloma, demonstrated that the UPS can be an attractive anti-cancer target.  However, normal cell toxicity assocd. with bortezomib, resulting from global inhibition of protein degrdn., promotes the focus of drug discovery efforts on targeting enzymes upstream of the proteasome for better specificity.  E3 ubiquitin ligases, particularly those known to be activated in human cancer, become an attractive choice.  Cullin-RING Ligases (CRLs) with multiple components are the largest family of E3 ubiquitin ligases and are responsible for ubiquitination of ∼20% of cellular proteins degraded through UPS.  Activity of CRLs is dynamically regulated and requires the RING component and cullin neddylation.  In this review, we will introduce the UPS and CRL E3s and discuss the biol. processes regulated by each of eight CRLs through substrate degrdn.  We will further discuss how cullin neddylation controls CRL activity, and how CRLs are being validated as the attractive cancer targets by abrogating the RING component through genetic means and by inhibiting cullin neddylation via MLN4924, a small mol. indirect inhibitor of CRLs, currently in several Phase I clin. trials.  Finally, we will discuss current efforts and future perspectives on the development of addnl. inhibitors of CRLs by targeting E2 and/or E3 of cullin neddylation and CRL-mediated ubiquitination as potential anti-cancer agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp-K-w4MaFbdLVg90H21EOLACvtfcHk0lj75rEzsmr8Pg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXotFagtL8%253D&md5=4498de520df1c0a24796bdbfd5b3ce17</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.2174%2F13816128113199990300&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F13816128113199990300%26sid%3Dliteratum%253Aachs%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DSun%26aufirst%3DY.%26atitle%3DCullin-RING%2520ligases%2520as%2520attractive%2520anti-cancer%2520targets%26jtitle%3DCurr.%2520Pharm.%2520Des.%26date%3D2013%26volume%3D19%26spage%3D3215%26epage%3D3225%26doi%3D10.2174%2F13816128113199990300" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Andérica-Romero, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">González-Herrera, I. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Santamaría, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pedraza-Chaverri, J.</span></span> <span> </span><span class="NLM_article-title">Cullin 3 as a novel target in diverse pathologies</span>. <i>Redox Biol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">366</span>– <span class="NLM_lpage">372</span>, <span class="refDoi"> DOI: 10.1016/j.redox.2013.07.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=10.1016%2Fj.redox.2013.07.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=24024173" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=1%3ACAS%3A528%3ADC%252BC2cXptFKntA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2013&pages=366-372&author=A.+C.+And%C3%A9rica-Romeroauthor=I.+G.+Gonz%C3%A1lez-Herreraauthor=A.+Santamar%C3%ADaauthor=J.+Pedraza-Chaverri&title=Cullin+3+as+a+novel+target+in+diverse+pathologies&doi=10.1016%2Fj.redox.2013.07.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Cullin 3 as a novel target in diverse pathologies</span></div><div class="casAuthors">Anderica-Romero, Ana Cristina; Gonzalez-Herrera, Irma Gabriela; Santamaria, Abel; Pedraza-Chaverri, Jose</div><div class="citationInfo"><span class="NLM_cas:title">Redox Biology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">366-372</span>CODEN:
                <span class="NLM_cas:coden">RBEIB3</span>;
        ISSN:<span class="NLM_cas:issn">2213-2317</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Recent evidence suggests that the malfunctioning disposal system of cell protein called ubiquitin-proteasome system (UPS) plays an important role in the development of disorders, including cancer and neurodegenerative diseases.  Accumulating evidence suggests that the abnormal regulation of the E3 ubiquitin ligases, essential components of the UPS, contributes to uncontrolled proliferation, genomic instability and cancer, since these ligases and their substrates are involved in the regulation of cell cycle progression, gene transcription, signal transduction, DNA replication and others.  Through selective degrdn. of specific substrates, E3 ligases regulate different biol. processes.  Cullins are a family of proteins that confer substrate specificity to multimeric complex of E3 ligases acting as scaffold proteins.  So far, seven members of the cullin family of proteins have been identified.  Interestingly, the data generated by several groups indicate that cullin 3 (Cul3) has begun to emerge as a protein involved in the etiopathol. of multiple diseases.  In this paper the authors examine the latest advances in basic research on the biol. of Cul3 and how it could help to direct drug discovery efforts on this target.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqQlQLrfY9SwLVg90H21EOLACvtfcHk0lj75rEzsmr8Pg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXptFKntA%253D%253D&md5=7d0b387948eba17c4695183bf5fad3a3</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1016%2Fj.redox.2013.07.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.redox.2013.07.003%26sid%3Dliteratum%253Aachs%26aulast%3DAnd%25C3%25A9rica-Romero%26aufirst%3DA.%2BC.%26aulast%3DGonz%25C3%25A1lez-Herrera%26aufirst%3DI.%2BG.%26aulast%3DSantamar%25C3%25ADa%26aufirst%3DA.%26aulast%3DPedraza-Chaverri%26aufirst%3DJ.%26atitle%3DCullin%25203%2520as%2520a%2520novel%2520target%2520in%2520diverse%2520pathologies%26jtitle%3DRedox%2520Biol.%26date%3D2013%26volume%3D1%26spage%3D366%26epage%3D372%26doi%3D10.1016%2Fj.redox.2013.07.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hori, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Osaka, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiba, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miyamoto, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Okabayashi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shimbara, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kato, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanaka, K.</span></span> <span> </span><span class="NLM_article-title">Covalent modification of all members of human cullin family proteins by NEDD8</span>. <i>Oncogene</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">6829</span>– <span class="NLM_lpage">6834</span>, <span class="refDoi"> DOI: 10.1038/sj.onc.1203093</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=10.1038%2Fsj.onc.1203093" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=10597293" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=1%3ACAS%3A528%3ADyaK1MXotV2ls78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=1999&pages=6829-6834&author=T.+Horiauthor=F.+Osakaauthor=T.+Chibaauthor=C.+Miyamotoauthor=K.+Okabayashiauthor=N.+Shimbaraauthor=S.+Katoauthor=K.+Tanaka&title=Covalent+modification+of+all+members+of+human+cullin+family+proteins+by+NEDD8&doi=10.1038%2Fsj.onc.1203093"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Covalent modification of all members of human cullin family proteins by NEDD8</span></div><div class="casAuthors">Hori, Tomoko; Osaka, Fumio; Chiba, Tomoki; Miyamoto, Chikara; Okabayashi, Ken; Shimbara, Naoki; Kato, Seishi; Tanaka, Keiji</div><div class="citationInfo"><span class="NLM_cas:title">Oncogene</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">48</span>),
    <span class="NLM_cas:pages">6829-6834</span>CODEN:
                <span class="NLM_cas:coden">ONCNES</span>;
        ISSN:<span class="NLM_cas:issn">0950-9232</span>.
    
            (<span class="NLM_cas:orgname">Stockton Press</span>)
        </div><div class="casAbstract">Recently the authors found that NEDD8, a ubiquitin-like protein, was linked covalently to human cullin-4A (abbreviated Cul-4A) by a new ubiquitin-related pathway that is analogous to but distinct from the ligating system for SUMO1, another ubiquitin-like protein.  However, it remained unknown whether the other five members of the family of human cullin/Cdc53 proteins are modified by NEDD8.  Here the authors report that all Hs-Cul family proteins, such as Cul-1, Cul-2, Cul-3, Cul-4B, and Cul-5, in addn. to Cul-4A, were modified by covalent attachment of NEDD8 in rabbit reticulocyte lysates.  Moreover, by comprehensive Northern-blot analyses, the authors examd. multiple tissue distributions of the messages for all Cul-family proteins, NEDD8, and the NEDD8-ligating system consisting of APP-BP1/hUba3, and hUbc12, which function as E1- and E2-like enzymes, resp.  The expressions of Cul-1, Cul-2, and Cul-3 resembled each other and were apparently correlated to those of NEDD8 and the NEDD8-ligating system in various human cells and tissues.  However, the mRNA levels of Cul-4A, Cul-4B, and Cul-5 differed considerably from each other as well as from other Cul-family proteins.  The enhanced expression of all Cul-family proteins except Cul-5 was obsd. in a variety of tumor cell lines.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpJYdFEgvbtMbVg90H21EOLACvtfcHk0lj75rEzsmr8Pg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXotV2ls78%253D&md5=25f186ac16e8875121b06e3e9baf2d18</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1038%2Fsj.onc.1203093&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.onc.1203093%26sid%3Dliteratum%253Aachs%26aulast%3DHori%26aufirst%3DT.%26aulast%3DOsaka%26aufirst%3DF.%26aulast%3DChiba%26aufirst%3DT.%26aulast%3DMiyamoto%26aufirst%3DC.%26aulast%3DOkabayashi%26aufirst%3DK.%26aulast%3DShimbara%26aufirst%3DN.%26aulast%3DKato%26aufirst%3DS.%26aulast%3DTanaka%26aufirst%3DK.%26atitle%3DCovalent%2520modification%2520of%2520all%2520members%2520of%2520human%2520cullin%2520family%2520proteins%2520by%2520NEDD8%26jtitle%3DOncogene%26date%3D1999%26volume%3D18%26spage%3D6829%26epage%3D6834%26doi%3D10.1038%2Fsj.onc.1203093" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Enchev, R. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schulman, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peter, M.</span></span> <span> </span><span class="NLM_article-title">Protein neddylation: beyond cullin–RING ligases</span>. <i>Nat. Rev. Mol. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">30</span>– <span class="NLM_lpage">44</span>, <span class="refDoi"> DOI: 10.1038/nrm3919</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=10.1038%2Fnrm3919" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=25531226" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitFKlt7fL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2015&pages=30-44&author=R.+I.+Enchevauthor=B.+A.+Schulmanauthor=M.+Peter&title=Protein+neddylation%3A+beyond+cullin%E2%80%93RING+ligases&doi=10.1038%2Fnrm3919"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Protein neddylation: beyond cullin-RING ligases</span></div><div class="casAuthors">Enchev, Radoslav I.; Schulman, Brenda A.; Peter, Matthias</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Molecular Cell Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">30-44</span>CODEN:
                <span class="NLM_cas:coden">NRMCBP</span>;
        ISSN:<span class="NLM_cas:issn">1471-0072</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  NEDD8 (neural precursor cell expressed developmentally downregulated protein 8) is a ubiquitin-like protein that activates the largest ubiquitin E3 ligase family, the cullin-RING ligases.  Many non-cullin neddylation targets have been proposed in recent years.  However, overexpression of exogenous NEDD8 can trigger NEDD8 conjugation through the ubiquitylation machinery, which makes validating potential NEDD8 targets challenging.  Here, we re-evaluate studies of non-cullin targets of NEDD8 in light of the current understanding of the neddylation pathway, and suggest criteria for identifying genuine neddylation substrates under homeostatic conditions.  We describe the biol. processes that might be regulated by non-cullin neddylation, and the utility of neddylation inhibitors for research and as potential therapies.  Understanding the biol. significance of non-cullin neddylation is an exciting research prospect primed to reveal fundamental insights.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqTz4DUdIwhJbVg90H21EOLACvtfcHk0lia_jd0oyHX_A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitFKlt7fL&md5=a73346bd599f5d5c09d983ca8f13f2f4</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1038%2Fnrm3919&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrm3919%26sid%3Dliteratum%253Aachs%26aulast%3DEnchev%26aufirst%3DR.%2BI.%26aulast%3DSchulman%26aufirst%3DB.%2BA.%26aulast%3DPeter%26aufirst%3DM.%26atitle%3DProtein%2520neddylation%253A%2520beyond%2520cullin%25E2%2580%2593RING%2520ligases%26jtitle%3DNat.%2520Rev.%2520Mol.%2520Cell%2520Biol.%26date%3D2015%26volume%3D16%26spage%3D30%26epage%3D44%26doi%3D10.1038%2Fnrm3919" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Deshaies, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Emberley, E. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saha, A.</span></span> <span> </span><span class="NLM_article-title">Control of cullin-ring ubiquitin ligase activity by nedd8</span>. <i>Subcell. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">41</span>– <span class="NLM_lpage">56</span>, <span class="refDoi"> DOI: 10.1007/978-1-4419-6676-6_4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=10.1007%2F978-1-4419-6676-6_4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=21222272" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtV2hsb%252FJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2010&pages=41-56&author=R.+J.+Deshaiesauthor=E.+D.+Emberleyauthor=A.+Saha&title=Control+of+cullin-ring+ubiquitin+ligase+activity+by+nedd8&doi=10.1007%2F978-1-4419-6676-6_4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Control of cullin-RING ubiquitin ligase activity by Nedd8</span></div><div class="casAuthors">Deshaies, Raymond J.; Emberley, Ethan D.; Saha, Anjanabha</div><div class="citationInfo"><span class="NLM_cas:title">Subcellular Biochemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">Conjugation and Deconjugation of Ubiquitin Family Modifiers</span>),
    <span class="NLM_cas:pages">41-56</span>CODEN:
                <span class="NLM_cas:coden">SBCBAG</span>;
        ISSN:<span class="NLM_cas:issn">0306-0225</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">A review.  The Cullin-RING ubiquitin ligase (CRL) family, which may no. as many as 350 different enzymes, has an enormous impact on cellular regulation.  CRL enzymes regulate cell biol. by conjugating ubiquitin onto target proteins that are involved in a multitude of processes.  In most cases, this leads to degrdn. of the target, but in some cases CRL-dependent ubiquitination acts as a switch to activate or repress target function.  The ubiquitin ligase activity of CRLs is controlled by cycles of attachment and removal of the ubiquitin-like protein Nedd8.  Conjugation of Nedd8 onto the cullin subunit of CRLs promotes assembly of an intact CRL complex and switches on ubiquitin ligase activity.  Conversely, removal of Nedd8 switches off ubiquitin ligase activity and initiates CRL disassembly.  Continuous maintenance of CRL function in vivo requires the activities of both the Nedd8-conjugating and deconjugating enzymes, pointing to a crit. role of complex dynamics in CRL function.  Here, we review how the Nedd8 cycle controls CRL activity and how perturbations of this cycle can lead to disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrqPoJEejak6rVg90H21EOLACvtfcHk0lia_jd0oyHX_A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtV2hsb%252FJ&md5=e6b0afe5b8dfe4864978ecc0b47c17a2</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1007%2F978-1-4419-6676-6_4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252F978-1-4419-6676-6_4%26sid%3Dliteratum%253Aachs%26aulast%3DDeshaies%26aufirst%3DR.%2BJ.%26aulast%3DEmberley%26aufirst%3DE.%2BD.%26aulast%3DSaha%26aufirst%3DA.%26atitle%3DControl%2520of%2520cullin-ring%2520ubiquitin%2520ligase%2520activity%2520by%2520nedd8%26jtitle%3DSubcell.%2520Biochem.%26date%3D2010%26volume%3D54%26spage%3D41%26epage%3D56%26doi%3D10.1007%2F978-1-4419-6676-6_4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morgan, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, Y.</span></span> <span> </span><span class="NLM_article-title">Targeting neddylation pathways to inactivate cullin-RING ligases for anticancer therapy</span>. <i>Antioxid. Redox Signaling</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">2383</span>– <span class="NLM_lpage">2400</span>, <span class="refDoi"> DOI: 10.1089/ars.2013.5795</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=10.1089%2Fars.2013.5795" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=24410571" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvFKltrzK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2014&pages=2383-2400&author=Y.+Zhaoauthor=M.+A.+Morganauthor=Y.+Sun&title=Targeting+neddylation+pathways+to+inactivate+cullin-RING+ligases+for+anticancer+therapy&doi=10.1089%2Fars.2013.5795"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting Neddylation Pathways to Inactivate Cullin-RING Ligases for Anticancer Therapy</span></div><div class="casAuthors">Zhao, Yongchao; Morgan, Meredith A.; Sun, Yi</div><div class="citationInfo"><span class="NLM_cas:title">Antioxidants & Redox Signaling</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">2383-2400</span>CODEN:
                <span class="NLM_cas:coden">ARSIF2</span>;
        ISSN:<span class="NLM_cas:issn">1523-0864</span>.
    
            (<span class="NLM_cas:orgname">Mary Ann Liebert, Inc.</span>)
        </div><div class="casAbstract">A review.  Significance: Protein neddylation is catalyzed by an E1 NEDD8-activating enzyme (NAE), an E2 NEDD8-conjugating enzyme, and an E3 NEDD8 ligase.  Known physiol. substrates of neddylation are cullin family members.  Cullin neddylation leads to activation of cullin-RING ligases (CRLs), the largest family of E3 ubiquitin ligases responsible for ubiquitylation and degrdn. of many key signaling/regulatory proteins.  Thus, through modulating CRLs, neddylation regulates many biol. processes, including cell cycle progression, signal transduction, and tumorigenesis.  Given that NEDD8 is overexpressed and CRLs are abnormally activated in many human cancers, targeting protein neddylation, in general, and cullin neddylation, in particular, appears to be an attractive anticancer approach.  Recent Advances: MLN4924, a small mol. inhibitor of NAE, was discovered that inactivates CRLs and causes accumulation of CRL substrates to suppress tumor cell growth both in vitro and in vivo.  Promising preclin. results advanced MLN4924 to several clin. trials for anticancer therapy.  Crit. Issues: In preclin. settings, MLN4924 effectively suppresses tumor cell growth by inducing apoptosis, senescence, and autophagy, and causes sensitization to chemoradiation therapies in a cellular context-dependent manner.  Signal mols. that det. the cell fate upon MLN4924 treatment, however, remain elusive.  Cancer cells develop MLN4924 resistance by selecting target mutations.  Future Directions: In the clin. side, several Phase 1b trials are under way to det. the safety and efficacy of MLN4924, acting alone or in combination with conventional chemotherapy, against human solid tumors.  In the preclin. side, the efforts are being made to develop addnl. neddylation inhibitors by targeting NEDD8 E2s and E3s.  Antioxid. Redox Signal. 21, 2383-2400.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr8ivf2UQt_DrVg90H21EOLACvtfcHk0lia_jd0oyHX_A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvFKltrzK&md5=42ff82c17092b1d5d6fc3526c9f2241a</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1089%2Fars.2013.5795&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1089%252Fars.2013.5795%26sid%3Dliteratum%253Aachs%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DMorgan%26aufirst%3DM.%2BA.%26aulast%3DSun%26aufirst%3DY.%26atitle%3DTargeting%2520neddylation%2520pathways%2520to%2520inactivate%2520cullin-RING%2520ligases%2520for%2520anticancer%2520therapy%26jtitle%3DAntioxid.%2520Redox%2520Signaling%26date%3D2014%26volume%3D21%26spage%3D2383%26epage%3D2400%26doi%3D10.1089%2Fars.2013.5795" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bulatov, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span> <span> </span><span class="NLM_article-title">Targeting cullin-RING E3 ubiquitin ligases for drug discovery: structure, assembly and small-molecule modulation</span>. <i>Biochem. J.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>467</i></span>,  <span class="NLM_fpage">365</span>– <span class="NLM_lpage">386</span>, <span class="refDoi"> DOI: 10.1042/BJ20141450</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=10.1042%2FBJ20141450" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=25886174" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=1%3ACAS%3A528%3ADC%252BC2MXmslChu7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=467&publication_year=2015&pages=365-386&author=E.+Bulatovauthor=A.+Ciulli&title=Targeting+cullin-RING+E3+ubiquitin+ligases+for+drug+discovery%3A+structure%2C+assembly+and+small-molecule+modulation&doi=10.1042%2FBJ20141450"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting Cullin-RING E3 ubiquitin ligases for drug discovery: structure, assembly and small-molecule modulation</span></div><div class="casAuthors">Bulatov, Emil; Ciulli, Alessio</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical Journal</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">467</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">365-386</span>CODEN:
                <span class="NLM_cas:coden">BIJOAK</span>;
        ISSN:<span class="NLM_cas:issn">0264-6021</span>.
    
            (<span class="NLM_cas:orgname">Portland Press Ltd.</span>)
        </div><div class="casAbstract">A review.  In the last decade, the ubiquitin-proteasome system has emerged as a valid target for the development of novel therapeutics.  E3 ubiquitin ligases are particularly attractive targets because they confer substrate specificity on the ubiquitin system.  CRLs [Cullin-RING (really interesting new gene) E3 ubiquitin ligases] draw particular attention, being the largest family of E3s.  The CRLs assemble into functional multisubunit complexes using a repertoire of substrate receptors, adaptors, Cullin scaffolds and RING-box proteins.  Drug discovery targeting CRLs is growing in importance due to mounting evidence pointing to significant roles of these enzymes in diverse biol. processes and human diseases, including cancer, where CRLs and their substrates often function as tumor suppressors or oncogenes.  In the present review, we provide an account of the assembly and structure of CRL complexes, and outline the current state of the field in terms of available knowledge of small-mol. inhibitors and modulators of CRL activity.  A comprehensive overview of the reported crystal structures of CRL subunits, components and full-size complexes, alone or with bound small mols. and substrate peptides, is included.  This information is providing increasing opportunities to aid the rational structure-based design of chem. probes and potential small-mol. therapeutics targeting CRLs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrgO7lBVCWaQrVg90H21EOLACvtfcHk0lia_jd0oyHX_A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXmslChu7k%253D&md5=8f76267e923c5e8868fa127b828296aa</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1042%2FBJ20141450&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252FBJ20141450%26sid%3Dliteratum%253Aachs%26aulast%3DBulatov%26aufirst%3DE.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DTargeting%2520cullin-RING%2520E3%2520ubiquitin%2520ligases%2520for%2520drug%2520discovery%253A%2520structure%252C%2520assembly%2520and%2520small-molecule%2520modulation%26jtitle%3DBiochem.%2520J.%26date%3D2015%26volume%3D467%26spage%3D365%26epage%3D386%26doi%3D10.1042%2FBJ20141450" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tanaka, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakatani, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kamitani, T.</span></span> <span> </span><span class="NLM_article-title">Inhibition of nedd8-conjugation pathway by novel molecules: potential approaches to anticancer therapy</span>. <i>Mol. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">267</span>– <span class="NLM_lpage">275</span>, <span class="refDoi"> DOI: 10.1016/j.molonc.2012.01.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=10.1016%2Fj.molonc.2012.01.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=22306028" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=1%3ACAS%3A528%3ADC%252BC38Xnslags7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2012&pages=267-275&author=T.+Tanakaauthor=T.+Nakataniauthor=T.+Kamitani&title=Inhibition+of+nedd8-conjugation+pathway+by+novel+molecules%3A+potential+approaches+to+anticancer+therapy&doi=10.1016%2Fj.molonc.2012.01.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of NEDD8-conjugation pathway by novel molecules: Potential approaches to anticancer therapy</span></div><div class="casAuthors">Tanaka, Tomoaki; Nakatani, Tatsuya; Kamitani, Tetsu</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Oncology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">267-275</span>CODEN:
                <span class="NLM_cas:coden">MOONC3</span>;
        ISSN:<span class="NLM_cas:issn">1574-7891</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Cancer cells can survive through the upregulation of cell cycle and the escape from apoptosis induced by numerous cellular stresses.  In the normal cells, these biol. cascades depend on scheduled proteolytic degrdn. of regulatory proteins via the ubiquitin-proteasome pathway.  Therefore, interruption of regulated proteolytic pathways leads to abnormal cell-proliferation.  Ubiquitin ligases called SCF complex (consisting of Skp-1, cullin, and F-box protein) or CRL (cullin-RING ubiquitin ligase) are predominant in a family of E3 ubiquitin ligases that control a final step in ubiquitination of diverse substrates.  To a great extent, the ubiquitin ligase activity of the SCF complex requires the conjugation of NEDD8 to cullins, i.e. scaffold proteins.  This review is anticipated to review the downregulation system of NEDD8 conjugation by several factors including a chem. compd. such as MLN4924 and protein mols. (e.g. COP9 signalosome, inactive mutant of Ubc12, and NUB1/NUB1L).  Since the downregulation of NEDD8 conjugation affects cell-cycle progression by inhibiting the ligase activity of SCF complexes, such knowledge in the NEDD8-conjugation pathway will contribute to the more magnificent therapies that selectively suppress tumorigenesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrtddvnq1BRvrVg90H21EOLACvtfcHk0ljpxTslXCVmTA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xnslags7g%253D&md5=1648458a47d5f38fb9ecf2678d9eff6b</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1016%2Fj.molonc.2012.01.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molonc.2012.01.003%26sid%3Dliteratum%253Aachs%26aulast%3DTanaka%26aufirst%3DT.%26aulast%3DNakatani%26aufirst%3DT.%26aulast%3DKamitani%26aufirst%3DT.%26atitle%3DInhibition%2520of%2520nedd8-conjugation%2520pathway%2520by%2520novel%2520molecules%253A%2520potential%2520approaches%2520to%2520anticancer%2520therapy%26jtitle%3DMol.%2520Oncol.%26date%3D2012%26volume%3D6%26spage%3D267%26epage%3D275%26doi%3D10.1016%2Fj.molonc.2012.01.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Soucy, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, P. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rolfe, M.</span></span> <span> </span><span class="NLM_article-title">Targeting nedd8-activated cullin-RING ligases for the treatment of cancer</span>. <i>Clin. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">3912</span>– <span class="NLM_lpage">3916</span>, <span class="refDoi"> DOI: 10.1158/1078-0432.CCR-09-0343</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=10.1158%2F1078-0432.CCR-09-0343" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=19509147" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=1%3ACAS%3A528%3ADC%252BD1MXntlOisrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2009&pages=3912-3916&author=T.+A.+Soucyauthor=P.+G.+Smithauthor=M.+Rolfe&title=Targeting+nedd8-activated+cullin-RING+ligases+for+the+treatment+of+cancer&doi=10.1158%2F1078-0432.CCR-09-0343"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting NEDD8-Activated Cullin-RING Ligases for the Treatment of Cancer</span></div><div class="casAuthors">Soucy, Teresa A.; Smith, Peter G.; Rolfe, Mark</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">3912-3916</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">A review.  E3 ubiquitin ligases regulate many dynamic cellular processes important for cancer cell survival.  Together with ubiquitin-activating enzyme (E1) and ubiquitin-conjugating enzymes (E2s), E3s catalyze the ubiquitination of numerous protein substrates that are subsequently targeted to the 26S proteasome for degrdn.  The clin. success of the proteasome inhibitor bortezomib has encouraged the evaluation of other components of the ubiquitin proteasome system for pharmaceutical intervention.  Targeting specific E3s is particularly attractive because there is the potential to selectively block the degrdn. of certain cellular proteins and possibly avoid unwanted effects on other proteins.  The cullin-RING ubiquitin E3 ligases (CRLs) represent the largest subfamily of E3s.  The requirement that CRLs be activated by NEDD8 modification on the cullin protein offers an "achilles heel" for modulating this entire subfamily.  NEDD8-activating enzyme (NAE) catalyzes the first step in the NEDD8 pathway and as such controls the activity of CRLs.  In this article, we describe the role of the NEDD8 pathway in activating CRLs and discuss the preclin. findings with a first-in-class NAE inhibitor that is currently in phase I clin. trials for both solid tumor and hematol. malignancies.  In addn., we speculate where NAE inhibitors may find clin. utility.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpt7UX7VycsELVg90H21EOLACvtfcHk0ljpxTslXCVmTA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXntlOisrg%253D&md5=cfbe3fc8c8a630e09fc31fe6e8429eb4</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1158%2F1078-0432.CCR-09-0343&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1078-0432.CCR-09-0343%26sid%3Dliteratum%253Aachs%26aulast%3DSoucy%26aufirst%3DT.%2BA.%26aulast%3DSmith%26aufirst%3DP.%2BG.%26aulast%3DRolfe%26aufirst%3DM.%26atitle%3DTargeting%2520nedd8-activated%2520cullin-RING%2520ligases%2520for%2520the%2520treatment%2520of%2520cancer%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2009%26volume%3D15%26spage%3D3912%26epage%3D3916%26doi%3D10.1158%2F1078-0432.CCR-09-0343" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhong, H.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, D. S.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leung, C.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, D.-L.</span></span> <span> </span><span class="NLM_article-title">A metal-based inhibitor of nedd8-activating enzyme</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>7</i></span>, <span class="NLM_elocation-id">e49574</span> <span class="refDoi"> DOI: 10.1371/journal.pone.0049574</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=10.1371%2Fjournal.pone.0049574" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=23185368" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvVSrtr7N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2012&author=H.-J.+Zhongauthor=H.+Yangauthor=D.+S.-H.+Chanauthor=C.-H.+Leungauthor=H.-M.+Wangauthor=D.-L.+Ma&title=A+metal-based+inhibitor+of+nedd8-activating+enzyme&doi=10.1371%2Fjournal.pone.0049574"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">A metal-based inhibitor of NEDD8-activating enzyme</span></div><div class="casAuthors">Zhong, Hai-Jing; Yang, Hui; Chan, Daniel Shiu-Hin; Leung, Chung-Hang; Wang, Hui-Min; Ma, Dik-Lung</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">e49574</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">A cyclometallated rhodium(III) complex [Rh(ppy)2(dppz)]+ (1) (where ppy = 2-phenylpyridine and dppz = dipyrido[3,2-a:2',3'-c]phenazine dipyridophenazine) has been prepd. and identified as an inhibitor of NEDD8-activating enzyme (NAE).  The complex inhibited NAE activity in cell-free and cell-based assays and suppressed the CRL-regulated substrate degrdn. and NF-κB activation in human cancer cells with potency comparable to known NAE inhibitor MLN4924.  Mol. modeling anal. suggested that the overall binding mode of 1 within the binding pocket of the APPBP1/UBA3 heterodimer resembled that for MLN4924.  Complex 1 is the first metal complex reported to suppress the NEDDylation pathway via inhibition of the NEDD8-activating enzyme.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpDnxHVwEANE7Vg90H21EOLACvtfcHk0ljpxTslXCVmTA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvVSrtr7N&md5=e5e4ac7aec1f46138a15e75f121acf10</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0049574&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0049574%26sid%3Dliteratum%253Aachs%26aulast%3DZhong%26aufirst%3DH.-J.%26aulast%3DYang%26aufirst%3DH.%26aulast%3DChan%26aufirst%3DD.%2BS.-H.%26aulast%3DLeung%26aufirst%3DC.-H.%26aulast%3DWang%26aufirst%3DH.-M.%26aulast%3DMa%26aufirst%3DD.-L.%26atitle%3DA%2520metal-based%2520inhibitor%2520of%2520nedd8-activating%2520enzyme%26jtitle%3DPLoS%2520One%26date%3D2012%26volume%3D7%26doi%3D10.1371%2Fjournal.pone.0049574" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Leung, C.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, D. S.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abagyan, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, S. M.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, G.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fong, W.-F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, D.-L.</span></span> <span> </span><span class="NLM_article-title">A natural product-like inhibitor of nedd8-activating enzyme</span>. <i>Chem. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">2511</span>– <span class="NLM_lpage">2513</span>, <span class="refDoi"> DOI: 10.1039/c0cc04927a</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=10.1039%2Fc0cc04927a" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=21240405" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=1%3ACAS%3A528%3ADC%252BC3MXitVCmt7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2011&pages=2511-2513&author=C.-H.+Leungauthor=D.+S.-H.+Chanauthor=H.+Yangauthor=R.+Abagyanauthor=S.+M.-Y.+Leeauthor=G.-Y.+Zhuauthor=W.-F.+Fongauthor=D.-L.+Ma&title=A+natural+product-like+inhibitor+of+nedd8-activating+enzyme&doi=10.1039%2Fc0cc04927a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">A natural product-like inhibitor of NEDD8-activating enzyme</span></div><div class="casAuthors">Leung, Chung-Hang; Chan, Daniel Shiu-Hin; Yang, Hui; Abagyan, Ruben; Lee, Simon Ming-Yuen; Zhu, Guo-Yuan; Fong, Wang-Fun; Ma, Dik-Lung</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Communications (Cambridge, United Kingdom)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2511-2513</span>CODEN:
                <span class="NLM_cas:coden">CHCOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-7345</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">The natural product-like 6,6''-biapigenin has been identified as only the second inhibitor of NEDD8-activating enzyme using virtual screening.  This compd. was active in enzyme and cell-based assays, with potency in the micromolar range.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoIgh-oaWCxuLVg90H21EOLACvtfcHk0lhWBQnrlaC0JQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXitVCmt7s%253D&md5=8783a238deaf9306f1bdaca88ab0575e</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1039%2Fc0cc04927a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc0cc04927a%26sid%3Dliteratum%253Aachs%26aulast%3DLeung%26aufirst%3DC.-H.%26aulast%3DChan%26aufirst%3DD.%2BS.-H.%26aulast%3DYang%26aufirst%3DH.%26aulast%3DAbagyan%26aufirst%3DR.%26aulast%3DLee%26aufirst%3DS.%2BM.-Y.%26aulast%3DZhu%26aufirst%3DG.-Y.%26aulast%3DFong%26aufirst%3DW.-F.%26aulast%3DMa%26aufirst%3DD.-L.%26atitle%3DA%2520natural%2520product-like%2520inhibitor%2520of%2520nedd8-activating%2520enzyme%26jtitle%3DChem.%2520Commun.%26date%3D2011%26volume%3D47%26spage%3D2511%26epage%3D2513%26doi%3D10.1039%2Fc0cc04927a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lu, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ouyang, P.-k.</span></span> <span> </span><span class="NLM_article-title">Discovery of a novel Nedd8 activating enzyme inhibitor with piperidin-4-amine scaffold by structure-based virtual screening</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">1901</span>– <span class="NLM_lpage">1907</span>, <span class="refDoi"> DOI: 10.1021/acschembio.6b00159</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.6b00159" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=1%3ACAS%3A528%3ADC%252BC28XmvFKqurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2016&pages=1901-1907&author=P.+Luauthor=X.+Liuauthor=X.+Yuanauthor=M.+Heauthor=Y.+Wangauthor=Q.+Zhangauthor=P.-k.+Ouyang&title=Discovery+of+a+novel+Nedd8+activating+enzyme+inhibitor+with+piperidin-4-amine+scaffold+by+structure-based+virtual+screening&doi=10.1021%2Facschembio.6b00159"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a novel NEDD8 Activating Enzyme Inhibitor with Piperidin-4-amine Scaffold by Structure-Based Virtual Screening</span></div><div class="casAuthors">Lu, Peng; Liu, Xiaoxin; Yuan, Xinrui; He, Minfang; Wang, Yubin; Zhang, Qi; Ouyang, Ping-kai</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1901-1907</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">NEDD8 activating enzyme (NAE) plays an important role in regulating intracellular proteins with key parts in a broad array of cellular functions.  Here, we report a structure-based virtual screening of a compd. library contg. 50000 small mol. entities against the active site of NAE.  Computational docking and scoring followed by biochem. screening and target validation lead to the identification of 1-benzyl-N-(2,4-dichlorophenethyl) piperidin-4-amine (M22) as a selective NAE inhibitor.  M22 is reversible for NAE, inhibits multiple cancer cell lines with GI50 values in the low micromolar range, and induces apoptosis in A549 cells.  Furthermore, it produces tumor inhibition in AGS xenografts in nude mice and low acute toxicity in a zebrafish model.  M22, a novel NAE inhibitor, represents a promising lead structure for the development of new antitumor agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp3wkfzasykYrVg90H21EOLACvtfcHk0lhWBQnrlaC0JQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XmvFKqurg%253D&md5=75b21e45e1698ccca8f4ac4a92a0a3b8</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1021%2Facschembio.6b00159&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.6b00159%26sid%3Dliteratum%253Aachs%26aulast%3DLu%26aufirst%3DP.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DYuan%26aufirst%3DX.%26aulast%3DHe%26aufirst%3DM.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DQ.%26aulast%3DOuyang%26aufirst%3DP.-k.%26atitle%3DDiscovery%2520of%2520a%2520novel%2520Nedd8%2520activating%2520enzyme%2520inhibitor%2520with%2520piperidin-4-amine%2520scaffold%2520by%2520structure-based%2520virtual%2520screening%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2016%26volume%3D11%26spage%3D1901%26epage%3D1907%26doi%3D10.1021%2Facschembio.6b00159" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhong, H.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, T.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, D.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leung, C.-H.</span></span> <span> </span><span class="NLM_article-title">A rhodium(iii) complex as an inhibitor of neural precursor cell expressed, developmentally down-regulated 8-activating enzyme with in vivo activity against inflammatory bowel disease</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">497</span>– <span class="NLM_lpage">503</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b00250</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b00250" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=1%3ACAS%3A528%3ADC%252BC28XitV2nurnK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=497-503&author=H.-J.+Zhongauthor=W.+Wangauthor=T.-S.+Kangauthor=H.+Yanauthor=Y.+Yangauthor=L.+Xuauthor=Y.+Wangauthor=D.-L.+Maauthor=C.-H.+Leung&title=A+rhodium%28iii%29+complex+as+an+inhibitor+of+neural+precursor+cell+expressed%2C+developmentally+down-regulated+8-activating+enzyme+with+in+vivo+activity+against+inflammatory+bowel+disease&doi=10.1021%2Facs.jmedchem.6b00250"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">A Rhodium(III) Complex as an Inhibitor of Neural Precursor Cell Expressed, Developmentally Down-Regulated 8-Activating Enzyme with in Vivo Activity against Inflammatory Bowel Disease</span></div><div class="casAuthors">Zhong, Hai-Jing; Wang, Wanhe; Kang, Tian-Shu; Yan, Hui; Yang, Yali; Xu, Lipeng; Wang, Yuqiang; Ma, Dik-Lung; Leung, Chung-Hang</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">497-503</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The authors report herein the identification of the rhodium(III) complex [Rh(phq)2(MOPIP)]+ (1) as a potent and selective ATP-competitive neural precursor cell expressed, developmentally down-regulated 8 (NEDD8)-activating enzyme (NAE) inhibitor.  Structure-activity relationship anal. indicated that the overall organometallic design of complex (1) was important for anti-inflammatory activity.  Complex (1) showed promising anti-inflammatory activity in vivo for the potential treatment of inflammatory bowel disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqOMmNvY-FeErVg90H21EOLACvtfcHk0lhWBQnrlaC0JQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitV2nurnK&md5=23c28735fe2477c54a97e21c956f79f5</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b00250&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b00250%26sid%3Dliteratum%253Aachs%26aulast%3DZhong%26aufirst%3DH.-J.%26aulast%3DWang%26aufirst%3DW.%26aulast%3DKang%26aufirst%3DT.-S.%26aulast%3DYan%26aufirst%3DH.%26aulast%3DYang%26aufirst%3DY.%26aulast%3DXu%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DMa%26aufirst%3DD.-L.%26aulast%3DLeung%26aufirst%3DC.-H.%26atitle%3DA%2520rhodium%2528iii%2529%2520complex%2520as%2520an%2520inhibitor%2520of%2520neural%2520precursor%2520cell%2520expressed%252C%2520developmentally%2520down-regulated%25208-activating%2520enzyme%2520with%2520in%2520vivo%2520activity%2520against%2520inflammatory%2520bowel%2520disease%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D497%26epage%3D503%26doi%3D10.1021%2Facs.jmedchem.6b00250" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schlierf, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Altmann, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quancard, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jefferson, A. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Assenberg, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Renatus, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hassiepen, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schaefer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kiffe, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weiss, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wiesmann, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sedrani, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eder, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martoglio, B.</span></span> <span> </span><span class="NLM_article-title">Targeted inhibition of the COP9 signalosome for treatment of cancer</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">13166</span>, <span class="refDoi"> DOI: 10.1038/ncomms13166</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=10.1038%2Fncomms13166" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=27774986" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=1%3ACAS%3A528%3ADC%252BC28XhslGrt77O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=13166&author=A.+Schlierfauthor=E.+Altmannauthor=J.+Quancardauthor=A.+B.+Jeffersonauthor=R.+Assenbergauthor=M.+Renatusauthor=M.+Jonesauthor=U.+Hassiepenauthor=M.+Schaeferauthor=M.+Kiffeauthor=A.+Weissauthor=C.+Wiesmannauthor=R.+Sedraniauthor=J.+Ederauthor=B.+Martoglio&title=Targeted+inhibition+of+the+COP9+signalosome+for+treatment+of+cancer&doi=10.1038%2Fncomms13166"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Targeted inhibition of the COP9 signalosome for treatment of cancer</span></div><div class="casAuthors">Schlierf, Anita; Altmann, Eva; Quancard, Jean; Jefferson, Anne B.; Assenberg, Rene; Renatus, Martin; Jones, Matthew; Hassiepen, Ulrich; Schaefer, Michael; Kiffe, Michael; Weiss, Andreas; Wiesmann, Christian; Sedrani, Richard; Eder, Jorg; Martoglio, Bruno</div><div class="citationInfo"><span class="NLM_cas:title">Nature Communications</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">13166</span>CODEN:
                <span class="NLM_cas:coden">NCAOBW</span>;
        ISSN:<span class="NLM_cas:issn">2041-1723</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The COP9 signalosome (CSN) is a central component of the activation and remodelling cycle of cullin-RING E3 ubiquitin ligases (CRLs), the largest enzyme family of the ubiquitin-proteasome system in humans.  CRLs are implicated in the regulation of numerous cellular processes, including cell cycle progression and apoptosis, and aberrant CRL activity is frequently assocd. with cancer.  Remodelling of CRLs is initiated by CSN-catalyzed cleavage of the ubiquitin-like activator NEDD8 from CRLs.  Here we describe CSN5i-3, a potent, selective and orally available inhibitor of CSN5, the proteolytic subunit of CSN.  The compd. traps CRLs in the neddylated state, which leads to inactivation of a subset of CRLs by inducing degrdn. of their substrate recognition module.  CSN5i-3 differentially affects the viability of tumor cell lines and suppresses growth of a human xenograft in mice.  Our results provide insights into how CSN regulates CRLs and suggest that CSN5 inhibition has potential for anti-tumor therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGojC7MhqQuitLVg90H21EOLACvtfcHk0ljjTktJQ5vy5g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhslGrt77O&md5=320de4d96f3201949335c06d613d78ed</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1038%2Fncomms13166&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms13166%26sid%3Dliteratum%253Aachs%26aulast%3DSchlierf%26aufirst%3DA.%26aulast%3DAltmann%26aufirst%3DE.%26aulast%3DQuancard%26aufirst%3DJ.%26aulast%3DJefferson%26aufirst%3DA.%2BB.%26aulast%3DAssenberg%26aufirst%3DR.%26aulast%3DRenatus%26aufirst%3DM.%26aulast%3DJones%26aufirst%3DM.%26aulast%3DHassiepen%26aufirst%3DU.%26aulast%3DSchaefer%26aufirst%3DM.%26aulast%3DKiffe%26aufirst%3DM.%26aulast%3DWeiss%26aufirst%3DA.%26aulast%3DWiesmann%26aufirst%3DC.%26aulast%3DSedrani%26aufirst%3DR.%26aulast%3DEder%26aufirst%3DJ.%26aulast%3DMartoglio%26aufirst%3DB.%26atitle%3DTargeted%2520inhibition%2520of%2520the%2520COP9%2520signalosome%2520for%2520treatment%2520of%2520cancer%26jtitle%3DNat.%2520Commun.%26date%3D2016%26volume%3D7%26spage%3D13166%26doi%3D10.1038%2Fncomms13166" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhong, H.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pui-Yan Ma, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shiu-Hin Chan, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, H.-Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leung, K.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, D.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leung, C.-H.</span></span> <span> </span><span class="NLM_article-title">Discovery of a natural product inhibitor targeting protein neddylation by structure-based virtual screening</span>. <i>Biochimie</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>94</i></span>,  <span class="NLM_fpage">2457</span>– <span class="NLM_lpage">2460</span>, <span class="refDoi"> DOI: 10.1016/j.biochi.2012.06.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=10.1016%2Fj.biochi.2012.06.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=22709868" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtVOqt7fF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=94&publication_year=2012&pages=2457-2460&author=H.-J.+Zhongauthor=V.+Pui-Yan+Maauthor=Z.+Chengauthor=D.+Shiu-Hin+Chanauthor=H.-Z.+Heauthor=K.-H.+Leungauthor=D.-L.+Maauthor=C.-H.+Leung&title=Discovery+of+a+natural+product+inhibitor+targeting+protein+neddylation+by+structure-based+virtual+screening&doi=10.1016%2Fj.biochi.2012.06.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a natural product inhibitor targeting protein neddylation by structure-based virtual screening</span></div><div class="casAuthors">Zhong, Hai-Jing; Ma, Victor Pui-Yan; Cheng, Zhen; Chan, Daniel Shiu-Hin; He, Hong-Zhang; Leung, Ka-Ho; Ma, Dik-Lung; Leung, Chung-Hang</div><div class="citationInfo"><span class="NLM_cas:title">Biochimie</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">94</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2457-2460</span>CODEN:
                <span class="NLM_cas:coden">BICMBE</span>;
        ISSN:<span class="NLM_cas:issn">0300-9084</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">NEDD8-activating enzyme (NAE) controls the specific degrdn. of proteins regulated by cullin-RING ubiquitin E3 ligase, and has been considered as an attractive mol. target for the development of anti-cancer drugs.  We report herein the identification of the dipeptide-conjugated deoxyvasicinone deriv. (1) as an inhibitor of NAE by virtual screening of over 90,000 compds. from the ZINC database of natural products.  Mol. modeling results suggested that 1 may be a non-covalent competitive inhibitor of NAE by blocking the ATP-binding domain.  Compd. 1 was able to inhibit NAE activity in both cell-free and cell-based assay with potencies in the micromolar range and selectivity over analogous E1 enzymes UAE and SAE.  We envisage that the identification and mol. docking anal. of this bioactive scaffold as an NAE inhibitor would provide the scientific community with useful information in order to generate more potent analogs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpKv_G29iM-W7Vg90H21EOLACvtfcHk0ljjTktJQ5vy5g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtVOqt7fF&md5=4500cd49c9ef5eb39a969781eaffe312</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1016%2Fj.biochi.2012.06.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.biochi.2012.06.004%26sid%3Dliteratum%253Aachs%26aulast%3DZhong%26aufirst%3DH.-J.%26aulast%3DPui-Yan%2BMa%26aufirst%3DV.%26aulast%3DCheng%26aufirst%3DZ.%26aulast%3DShiu-Hin%2BChan%26aufirst%3DD.%26aulast%3DHe%26aufirst%3DH.-Z.%26aulast%3DLeung%26aufirst%3DK.-H.%26aulast%3DMa%26aufirst%3DD.-L.%26aulast%3DLeung%26aufirst%3DC.-H.%26atitle%3DDiscovery%2520of%2520a%2520natural%2520product%2520inhibitor%2520targeting%2520protein%2520neddylation%2520by%2520structure-based%2520virtual%2520screening%26jtitle%3DBiochimie%26date%3D2012%26volume%3D94%26spage%3D2457%26epage%3D2460%26doi%3D10.1016%2Fj.biochi.2012.06.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ji, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, Y.</span></span> <span> </span><span class="NLM_article-title">Effective virtual screening strategy toward covalent ligands: identification of novel nedd8-activating enzyme inhibitors</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">1785</span>– <span class="NLM_lpage">1797</span>, <span class="refDoi"> DOI: 10.1021/ci5002058</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci5002058" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=1%3ACAS%3A528%3ADC%252BC2cXosFaitLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2014&pages=1785-1797&author=S.+Zhangauthor=J.+Tanauthor=Z.+Laiauthor=Y.+Liauthor=J.+Pangauthor=J.+Xiaoauthor=Z.+Huangauthor=Y.+Zhangauthor=H.+Jiauthor=Y.+Lai&title=Effective+virtual+screening+strategy+toward+covalent+ligands%3A+identification+of+novel+nedd8-activating+enzyme+inhibitors&doi=10.1021%2Fci5002058"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Effective Virtual Screening Strategy toward Covalent Ligands: Identification of Novel NEDD8-Activating Enzyme Inhibitors</span></div><div class="casAuthors">Zhang, Shengping; Tan, Jiani; Lai, Zhonghui; Li, Ying; Pang, Junxia; Xiao, Jianhu; Huang, Zhangjian; Zhang, Yihua; Ji, Hui; Lai, Yisheng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1785-1797</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The NEDD8-activating enzyme (NAE) is an emerging target for cancer therapy, which regulates the degrdn. and turnover of a variety of cancer-related proteins by activating the cullin-RING E3 ubiquitin ligases.  Among a limited no. of known NAE inhibitors, the covalent inhibitors have demonstrated the most potent efficacy through their covalently linked adducts with NEDD8.  Inspired by this unique mechanism, in this study, a novel combined strategy of virtual screening (VS) was adopted with the aim to identify diverse covalent inhibitors of NAE.  To be specific, a docking-enabled pharmacophore model was first built from the possible active conformations of chosen covalent inhibitors.  Meanwhile, a dynamic structure-based phamacophore was also established based on the snapshots derived from mol. dynamic simulation.  Subsequent screening of a focused ZINC database using these pharmacophore models combined with covalent docking discovered three novel active compds.  Among them, compd. LZ3 exhibited the most potent NAE inhibitory activity with an IC50 value of 1.06 ± 0.18 μM.  Furthermore, a cell-based washout expt. proved the proposed covalent binding mechanism for compd. LZ3, which confirmed the successful application of our combined VS strategy, indicating it may provide a viable soln. to systematically discover novel covalent ligands.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqhRJSduwx3CrVg90H21EOLACvtfcHk0ljjTktJQ5vy5g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXosFaitLw%253D&md5=07a5cf406764b37d2f68b8aba55649eb</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1021%2Fci5002058&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci5002058%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DS.%26aulast%3DTan%26aufirst%3DJ.%26aulast%3DLai%26aufirst%3DZ.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DPang%26aufirst%3DJ.%26aulast%3DXiao%26aufirst%3DJ.%26aulast%3DHuang%26aufirst%3DZ.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DJi%26aufirst%3DH.%26aulast%3DLai%26aufirst%3DY.%26atitle%3DEffective%2520virtual%2520screening%2520strategy%2520toward%2520covalent%2520ligands%253A%2520identification%2520of%2520novel%2520nedd8-activating%2520enzyme%2520inhibitors%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2014%26volume%3D54%26spage%3D1785%26epage%3D1797%26doi%3D10.1021%2Fci5002058" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Soucy, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, P. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Milhollen, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berger, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gavin, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adhikari, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brownell, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burke, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cardin, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Critchley, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cullis, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doucette, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garnsey, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaulin, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gershman, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lublinsky, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McDonald, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mizutani, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Narayanan, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olhava, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peluso, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rezaei, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sintchak, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Talreja, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Traore, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vyskocil, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weatherhead, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dick, L. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Claiborne, C. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rolfe, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bolen, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Langston, S. P.</span></span> <span> </span><span class="NLM_article-title">An inhibitor of nedd8-activating enzyme as a new approach to treat cancer</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>458</i></span>,  <span class="NLM_fpage">732</span>– <span class="NLM_lpage">736</span>, <span class="refDoi"> DOI: 10.1038/nature07884</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=10.1038%2Fnature07884" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=19360080" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=1%3ACAS%3A528%3ADC%252BD1MXksVenu7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=458&publication_year=2009&pages=732-736&author=T.+A.+Soucyauthor=P.+G.+Smithauthor=M.+A.+Milhollenauthor=A.+J.+Bergerauthor=J.+M.+Gavinauthor=S.+Adhikariauthor=J.+E.+Brownellauthor=K.+E.+Burkeauthor=D.+P.+Cardinauthor=S.+Critchleyauthor=C.+A.+Cullisauthor=A.+Doucetteauthor=J.+J.+Garnseyauthor=J.+L.+Gaulinauthor=R.+E.+Gershmanauthor=A.+R.+Lublinskyauthor=A.+McDonaldauthor=H.+Mizutaniauthor=U.+Narayananauthor=E.+J.+Olhavaauthor=S.+Pelusoauthor=M.+Rezaeiauthor=M.+D.+Sintchakauthor=T.+Talrejaauthor=M.+P.+Thomasauthor=T.+Traoreauthor=S.+Vyskocilauthor=G.+S.+Weatherheadauthor=J.+Yuauthor=J.+Zhangauthor=L.+R.+Dickauthor=C.+F.+Claiborneauthor=M.+Rolfeauthor=J.+B.+Bolenauthor=S.+P.+Langston&title=An+inhibitor+of+nedd8-activating+enzyme+as+a+new+approach+to+treat+cancer&doi=10.1038%2Fnature07884"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">An inhibitor of NEDD8-activating enzyme as a new approach to treat cancer</span></div><div class="casAuthors">Soucy, Teresa A.; Smith, Peter G.; Milhollen, Michael A.; Berger, Allison J.; Gavin, James M.; Adhikari, Sharmila; Brownell, James E.; Burke, Kristine E.; Cardin, David P.; Critchley, Stephen; Cullis, Courtney A.; Doucette, Amanda; Garnsey, James J.; Gaulin, Jeffrey L.; Gershman, Rachel E.; Lublinsky, Anna R.; McDonald, Alice; Mizutani, Hirotake; Narayanan, Usha; Olhava, Edward J.; Peluso, Stephane; Rezaei, Mansoureh; Sintchak, Michael D.; Talreja, Tina; Thomas, Michael P.; Traore, Tary; Vyskocil, Stepan; Weatherhead, Gabriel S.; Yu, Jie; Zhang, Julie; Dick, Lawrence R.; Claiborne, Christopher F.; Rolfe, Mark; Bolen, Joseph B.; Langston, Steven P.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">458</span>
        (<span class="NLM_cas:issue">7239</span>),
    <span class="NLM_cas:pages">732-736</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The clin. development of an inhibitor of cellular proteasome function suggests that compds. targeting other components of the ubiquitin-proteasome system might prove useful for the treatment of human malignancies.  NEDD8-activating enzyme (NAE) is an essential component of the NEDD8 conjugation pathway that controls the activity of the cullin-RING subtype of ubiquitin ligases, thereby regulating the turnover of a subset of proteins upstream of the proteasome.  Substrates of cullin-RING ligases have important roles in cellular processes assocd. with cancer cell growth and survival pathways.  Here we describe MLN4924, a potent and selective inhibitor of NAE.  MLN4924 disrupts cullin-RING ligase-mediated protein turnover leading to apoptotic death in human tumor cells by a new mechanism of action, the deregulation of S-phase DNA synthesis.  MLN4924 suppressed the growth of human tumor xenografts in mice at compd. exposures that were well tolerated.  Our data suggest that NAE inhibitors may hold promise for the treatment of cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr8fx7OnZtYjbVg90H21EOLACvtfcHk0lhA19vQBN6c5Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXksVenu7Y%253D&md5=365d4b4feac3994849050e00f1aae5e0</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1038%2Fnature07884&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature07884%26sid%3Dliteratum%253Aachs%26aulast%3DSoucy%26aufirst%3DT.%2BA.%26aulast%3DSmith%26aufirst%3DP.%2BG.%26aulast%3DMilhollen%26aufirst%3DM.%2BA.%26aulast%3DBerger%26aufirst%3DA.%2BJ.%26aulast%3DGavin%26aufirst%3DJ.%2BM.%26aulast%3DAdhikari%26aufirst%3DS.%26aulast%3DBrownell%26aufirst%3DJ.%2BE.%26aulast%3DBurke%26aufirst%3DK.%2BE.%26aulast%3DCardin%26aufirst%3DD.%2BP.%26aulast%3DCritchley%26aufirst%3DS.%26aulast%3DCullis%26aufirst%3DC.%2BA.%26aulast%3DDoucette%26aufirst%3DA.%26aulast%3DGarnsey%26aufirst%3DJ.%2BJ.%26aulast%3DGaulin%26aufirst%3DJ.%2BL.%26aulast%3DGershman%26aufirst%3DR.%2BE.%26aulast%3DLublinsky%26aufirst%3DA.%2BR.%26aulast%3DMcDonald%26aufirst%3DA.%26aulast%3DMizutani%26aufirst%3DH.%26aulast%3DNarayanan%26aufirst%3DU.%26aulast%3DOlhava%26aufirst%3DE.%2BJ.%26aulast%3DPeluso%26aufirst%3DS.%26aulast%3DRezaei%26aufirst%3DM.%26aulast%3DSintchak%26aufirst%3DM.%2BD.%26aulast%3DTalreja%26aufirst%3DT.%26aulast%3DThomas%26aufirst%3DM.%2BP.%26aulast%3DTraore%26aufirst%3DT.%26aulast%3DVyskocil%26aufirst%3DS.%26aulast%3DWeatherhead%26aufirst%3DG.%2BS.%26aulast%3DYu%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DDick%26aufirst%3DL.%2BR.%26aulast%3DClaiborne%26aufirst%3DC.%2BF.%26aulast%3DRolfe%26aufirst%3DM.%26aulast%3DBolen%26aufirst%3DJ.%2BB.%26aulast%3DLangston%26aufirst%3DS.%2BP.%26atitle%3DAn%2520inhibitor%2520of%2520nedd8-activating%2520enzyme%2520as%2520a%2520new%2520approach%2520to%2520treat%2520cancer%26jtitle%3DNature%26date%3D2009%26volume%3D458%26spage%3D732%26epage%3D736%26doi%3D10.1038%2Fnature07884" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bhatia, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pavlick, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boasberg, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mulligan, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pickard, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Faessel, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dezube, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamid, O.</span></span> <span> </span><span class="NLM_article-title">A phase I study of the investigational NEDD8-activating enzyme inhibitor pevonedistat (TAK-924/MLN4924) in patients with metastatic melanoma</span>. <i>Invest. New Drugs</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>34</i></span>,  <span class="NLM_fpage">439</span>– <span class="NLM_lpage">449</span>, <span class="refDoi"> DOI: 10.1007/s10637-016-0348-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=10.1007%2Fs10637-016-0348-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=27056178" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=1%3ACAS%3A528%3ADC%252BC28XlvV2gs7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=2016&pages=439-449&author=S.+Bhatiaauthor=A.+C.+Pavlickauthor=P.+Boasbergauthor=J.+A.+Thompsonauthor=G.+Mulliganauthor=M.+D.+Pickardauthor=H.+Faesselauthor=B.+J.+Dezubeauthor=O.+Hamid&title=A+phase+I+study+of+the+investigational+NEDD8-activating+enzyme+inhibitor+pevonedistat+%28TAK-924%2FMLN4924%29+in+patients+with+metastatic+melanoma&doi=10.1007%2Fs10637-016-0348-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">A phase I study of the investigational NEDD8-activating enzyme inhibitor pevonedistat (TAK-924/MLN4924) in patients with metastatic melanoma</span></div><div class="casAuthors">Bhatia, Shailender; Pavlick, Anna C.; Boasberg, Peter; Thompson, John A.; Mulligan, George; Pickard, Michael D.; Faessel, Helene; Dezube, Bruce J.; Hamid, Omid</div><div class="citationInfo"><span class="NLM_cas:title">Investigational New Drugs</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">34</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">439-449</span>CODEN:
                <span class="NLM_cas:coden">INNDDK</span>;
        ISSN:<span class="NLM_cas:issn">0167-6997</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Purpose The therapeutic index of proteasome inhibitors may be improved through selective inhibition of a sub-component of the ubiquitin-proteasome system, such as the NEDD8-conjugation pathway.  This multicenter, phase I, dose-escalation study assessed safety and the max. tolerated dose (MTD), pharmacokinetics, pharmacodynamics, and antitumor activity of pevonedistat, an investigational NEDD8-activating enzyme (NAE) inhibitor, in patients with metastatic melanoma.  Methods Patients received i.v. pevonedistat on Days 1, 4, 8, 11 (schedule A) or 1, 8, 15 (schedule B) of 21-day cycles.  Results 26 patients received pevonedistat 50-278 mg/m2 on schedule A; 11 patients received pevonedistat 157 mg/m2 on schedule B.  The schedule A MTD was 209 mg/m2: dose-limiting toxicities (DLTs) included grade 3 hypophosphatemia and grade 3 increased blood creatinine (assocd. with grade 3 hyperbilirubinemia).  Two schedule A patients experienced acute organ failure toxicities, one of whom experienced grade 5 acute renal failure.  Dose escalation did not occur in schedule B: DLTs included grade 3 myocarditis, grade 2 acute renal failure, and grade 2 hyperbilirubinemia in a single patient.  Pevonedistat pharmacokinetics were approx. dose-proportional across the dose range studied, with a biphasic disposition profile characterized by a short elimination half-life (∼10 h).  Pharmacodynamic studies showed increases in NAE-regulated transcripts post-treatment; all post-dose biopsy samples were pos. for pevonedistat-NEDD8 adduct.  One schedule A patient achieved a partial response; 15 patients had stable disease (4 lasting ≥6.5 mo).  Conclusions Pevonedistat was generally well tolerated at the MTD.  Anticipated pharmacodynamic effects of NAE inhibition were obsd. with single-agent pevonedistat in peripheral blood and tumor tissue.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr_SdAW55A9BbVg90H21EOLACvtfcHk0lhA19vQBN6c5Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XlvV2gs7Y%253D&md5=9a8a889697053b5e5a7a9f01e0f70f5a</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1007%2Fs10637-016-0348-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10637-016-0348-5%26sid%3Dliteratum%253Aachs%26aulast%3DBhatia%26aufirst%3DS.%26aulast%3DPavlick%26aufirst%3DA.%2BC.%26aulast%3DBoasberg%26aufirst%3DP.%26aulast%3DThompson%26aufirst%3DJ.%2BA.%26aulast%3DMulligan%26aufirst%3DG.%26aulast%3DPickard%26aufirst%3DM.%2BD.%26aulast%3DFaessel%26aufirst%3DH.%26aulast%3DDezube%26aufirst%3DB.%2BJ.%26aulast%3DHamid%26aufirst%3DO.%26atitle%3DA%2520phase%2520I%2520study%2520of%2520the%2520investigational%2520NEDD8-activating%2520enzyme%2520inhibitor%2520pevonedistat%2520%2528TAK-924%252FMLN4924%2529%2520in%2520patients%2520with%2520metastatic%2520melanoma%26jtitle%3DInvest.%2520New%2520Drugs%26date%3D2016%26volume%3D34%26spage%3D439%26epage%3D449%26doi%3D10.1007%2Fs10637-016-0348-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Monda, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lydeard, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">King, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harper, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bennett, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schulman, B. A.</span></span> <span> </span><span class="NLM_article-title">Structural conservation of distinctive N-terminal acetylation-dependent interactions across a family of mammalian nedd8 ligation enzymes</span>. <i>Structure</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">42</span>– <span class="NLM_lpage">53</span>, <span class="refDoi"> DOI: 10.1016/j.str.2012.10.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=10.1016%2Fj.str.2012.10.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=23201271" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=1%3ACAS%3A528%3ADC%252BC38XhslKqtrjL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2013&pages=42-53&author=J.+K.+Mondaauthor=D.+C.+Scottauthor=D.+J.+Millerauthor=J.+Lydeardauthor=D.+Kingauthor=J.+W.+Harperauthor=E.+J.+Bennettauthor=B.+A.+Schulman&title=Structural+conservation+of+distinctive+N-terminal+acetylation-dependent+interactions+across+a+family+of+mammalian+nedd8+ligation+enzymes&doi=10.1016%2Fj.str.2012.10.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Structural Conservation of Distinctive N-terminal Acetylation-Dependent Interactions across a Family of Mammalian NEDD8 Ligation Enzymes</span></div><div class="casAuthors">Monda, Julie K.; Scott, Daniel C.; Miller, Darcie J.; Lydeard, John; King, David; Harper, J. Wade; Bennett, Eric J.; Schulman, Brenda A.</div><div class="citationInfo"><span class="NLM_cas:title">Structure (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">42-53</span>CODEN:
                <span class="NLM_cas:coden">STRUE6</span>;
        ISSN:<span class="NLM_cas:issn">0969-2126</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Little is known about mol. recognition of acetylated N termini, despite prevalence of this modification among eukaryotic cytosolic proteins.  We report that the family of human DCN-like (DCNL) co-E3s, which promote ligation of the ubiquitin-like protein NEDD8 to cullin targets, recognizes acetylated N termini of the E2 enzymes UBC12 and UBE2F.  Systematic biochem. and biophys. analyses reveal 40- and 10-fold variations in affinities among different DCNL-cullin and DCNL-E2 complexes, contributing to varying efficiencies of different NEDD8 ligation cascades.  Structures of DCNL2 and DCNL3 complexes with N-terminally acetylated peptides from UBC12 and UBE2F illuminate a common mechanism by which DCNL proteins recognize N-terminally acetylated E2s and how selectivity for interactions dependent on N-acetyl-methionine are established through side chains recognizing distal residues.  Distinct preferences of UBC12 and UBE2F peptides for inhibiting different DCNLs, including the oncogenic DCNL1 protein, suggest it may be possible to develop small mols. blocking specific N-acetyl-methionine-dependent protein interactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrlyp13hD0rNbVg90H21EOLACvtfcHk0liNuvFdQeGt6g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhslKqtrjL&md5=98e00537298d81b591d87e8e135e9c3a</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1016%2Fj.str.2012.10.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.str.2012.10.013%26sid%3Dliteratum%253Aachs%26aulast%3DMonda%26aufirst%3DJ.%2BK.%26aulast%3DScott%26aufirst%3DD.%2BC.%26aulast%3DMiller%26aufirst%3DD.%2BJ.%26aulast%3DLydeard%26aufirst%3DJ.%26aulast%3DKing%26aufirst%3DD.%26aulast%3DHarper%26aufirst%3DJ.%2BW.%26aulast%3DBennett%26aufirst%3DE.%2BJ.%26aulast%3DSchulman%26aufirst%3DB.%2BA.%26atitle%3DStructural%2520conservation%2520of%2520distinctive%2520N-terminal%2520acetylation-dependent%2520interactions%2520across%2520a%2520family%2520of%2520mammalian%2520nedd8%2520ligation%2520enzymes%26jtitle%3DStructure%26date%3D2013%26volume%3D21%26spage%3D42%26epage%3D53%26doi%3D10.1016%2Fj.str.2012.10.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sarkaria, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O-charoenrat, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Talbot, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reddy, P. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ngai, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maghami, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, K. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yonekawa, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dudas, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaufman, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ryan, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ghossein, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rao, P. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stoffel, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramanathan, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, B.</span></span> <span> </span><span class="NLM_article-title">Squamous cell carcinoma related oncogene/DCUN1D1 is highly conserved and activated by amplification in squamous cell carcinomas</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">9437</span>– <span class="NLM_lpage">9444</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.CAN-06-2074</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=10.1158%2F0008-5472.CAN-06-2074" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=17018598" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtVamsrjE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2006&pages=9437-9444&author=I.+Sarkariaauthor=P.+O-charoenratauthor=S.+G.+Talbotauthor=P.+G.+Reddyauthor=I.+Ngaiauthor=E.+Maghamiauthor=K.+N.+Patelauthor=B.+Leeauthor=Y.+Yonekawaauthor=M.+Dudasauthor=A.+Kaufmanauthor=R.+Ryanauthor=R.+Ghosseinauthor=P.+H.+Raoauthor=A.+Stoffelauthor=Y.+Ramanathanauthor=B.+Singh&title=Squamous+cell+carcinoma+related+oncogene%2FDCUN1D1+is+highly+conserved+and+activated+by+amplification+in+squamous+cell+carcinomas&doi=10.1158%2F0008-5472.CAN-06-2074"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Squamous Cell Carcinoma Related Oncogene/DCUN1D1 Is Highly Conserved and Activated by Amplification in Squamous Cell Carcinomas</span></div><div class="casAuthors">Sarkaria, Inderpal; O-charoenrat, Pornchai; Talbot, Simon G.; Reddy, Pabbathi G.; Ngai, Ivan; Maghami, Ellie; Patel, Kepal N.; Lee, Benjamin; Yonekawa, Yoshihiro; Dudas, Maria; Kaufman, Andrew; Ryan, Russell; Ghossein, Ronald; Rao, Pulivarthi H.; Stoffel, Archontoula; Ramanathan, Y.; Singh, Bhuvanesh</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">9437-9444</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Chromosomal amplification at 3q is common to multiple human cancers, but has a specific predilection for squamous cell carcinomas (SCC) of mucosal origin.  We identified and characterized a novel oncogene, SCC-related oncogene (SCCRO), which is amplified along the 3q26.3 region in human SCC.  Amplification and overexpression of SCCRO in these tumors correlate with poor clin. outcome.  The importance of SCCRO amplification in malignant transformation is established by the apoptotic response to short hairpin RNA against SCCRO, exclusively in cancer cell lines carrying SCCRO amplification.  The oncogenic potential of SCCRO is underscored by its ability to transform fibroblasts (NIH-3T3 cells) in vitro and in vivo.  We show that SCCRO regulates Gli1-a key regulator of the hedgehog (HH) pathway.  Collectively, these data suggest that SCCRO is a novel component of the HH signaling pathway involved in the malignant transformation of squamous cell lineage.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrzTIFIhphYXrVg90H21EOLACvtfcHk0liNuvFdQeGt6g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtVamsrjE&md5=359faefb6b6305279f3df1ec54a544f7</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-06-2074&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-06-2074%26sid%3Dliteratum%253Aachs%26aulast%3DSarkaria%26aufirst%3DI.%26aulast%3DO-charoenrat%26aufirst%3DP.%26aulast%3DTalbot%26aufirst%3DS.%2BG.%26aulast%3DReddy%26aufirst%3DP.%2BG.%26aulast%3DNgai%26aufirst%3DI.%26aulast%3DMaghami%26aufirst%3DE.%26aulast%3DPatel%26aufirst%3DK.%2BN.%26aulast%3DLee%26aufirst%3DB.%26aulast%3DYonekawa%26aufirst%3DY.%26aulast%3DDudas%26aufirst%3DM.%26aulast%3DKaufman%26aufirst%3DA.%26aulast%3DRyan%26aufirst%3DR.%26aulast%3DGhossein%26aufirst%3DR.%26aulast%3DRao%26aufirst%3DP.%2BH.%26aulast%3DStoffel%26aufirst%3DA.%26aulast%3DRamanathan%26aufirst%3DY.%26aulast%3DSingh%26aufirst%3DB.%26atitle%3DSquamous%2520cell%2520carcinoma%2520related%2520oncogene%252FDCUN1D1%2520is%2520highly%2520conserved%2520and%2520activated%2520by%2520amplification%2520in%2520squamous%2520cell%2520carcinomas%26jtitle%3DCancer%2520Res.%26date%3D2006%26volume%3D66%26spage%3D9437%26epage%3D9444%26doi%3D10.1158%2F0008-5472.CAN-06-2074" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Estilo, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O-charoenrat, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ngai, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reddy, P. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dao, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaha, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kraus, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boyle, J. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pfister, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huryn, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zlotolow, I. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shah, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, B.</span></span> <span> </span><span class="NLM_article-title">The role of novel oncogenes squamous cell carcinoma-related oncogene and phosphatidylinositol 3-kinase p110α in squamous cell carcinoma of the oral tongue</span>. <i>Clin. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">2300</span>– <span class="NLM_lpage">2306</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=12796399" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=1%3ACAS%3A528%3ADC%252BD3sXksVajsrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2003&pages=2300-2306&author=C.+L.+Estiloauthor=P.+O-charoenratauthor=I.+Ngaiauthor=S.+G.+Patelauthor=P.+G.+Reddyauthor=S.+Daoauthor=A.+R.+Shahaauthor=D.+H.+Krausauthor=J.+O.+Boyleauthor=R.+J.+Wongauthor=D.+G.+Pfisterauthor=J.+M.+Hurynauthor=I.+M.+Zlotolowauthor=J.+P.+Shahauthor=B.+Singh&title=The+role+of+novel+oncogenes+squamous+cell+carcinoma-related+oncogene+and+phosphatidylinositol+3-kinase+p110%CE%B1+in+squamous+cell+carcinoma+of+the+oral+tongue"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">The Role of Novel Oncogenes Squamous Cell Carcinoma-related Oncogene and Phosphatidylinositol 3-Kinase p110α in Squamous Cell Carcinoma of the Oral Tongue</span></div><div class="casAuthors">Estilo, Cherry L.; O-Charoenrat, Pornchai; Ngai, Ivan; Patel, Snehal G.; Reddy, Pabbathi G.; Dao, Su; Shaha, Ashok R.; Kraus, Dennis H.; Boyle, Jay O.; Wong, Richard J.; Pfister, David G.; Huryn, Joseph M.; Zlotolow, Ian M.; Shah, Jatin P.; Singh, Bhuvanesh</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2300-2306</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Amplification at chromosome 3q26.3 is a common and crucial event in head and neck squamous cell carcinoma (HNSCC), impacting significantly on tumor progression and clin. outcome.  Two novel oncogenes, namely squamous cell carcinoma (SCC)-related oncogene (SCCRO) and PIK3CA (gene encoding phosphatidylinositol-3 kinase catalytic α-polypeptide), have been identified as targets of 3q26.3 amplification.  This study aimed to delineate the role of SCCRO and PIK3CA in the pathogenesis of oral tongue SCC.  The assocn. between gene copy no. for SCCRO and PIK3CA measured by fluorescence in situ hybridization and level of mRNA expression quantitated by real-time reverse transcription-PCR was assessed in a panel of human HNSCC cell lines.  In addn., gene expression in 49 consecutive primary SCCs of the oral tongue was detd. and correlated with clinicopathol. characteristics and outcome.  The mRNA level of SCCRO and PIK3CA was significantly correlated to the gene copy no. in nine HNSCC cell lines.  In addn., the expression level of SCCRO and PIK3CA was significantly greater in malignant tissues compared with those in histol. normal mucosae (2.17- and 2.46-fold, resp.).  Matched tumor normal control anal. revealed that 24.5 and 69.4% of patients expressed high levels of SCCRO and PIK3CA, resp.  Univariate analyses demonstrated that SCCRO overexpression correlated with nodal metastases and advanced stage, whereas PIK3CA overexpression was assocd. with vascular invasion.  Only SCCRO overexpression was assocd. with disease-specific and overall survival.  Furthermore, SCCRO overexpression remained an independent predictor for cervical nodal metastasis on multivariate regression anal. (χ2 likelihood ratio = 4.38).  Although both SCCRO and PIK3CA may play a role in the pathogenesis of oral tongue SCC through amplification at 3q26, SCCRO appears to be a significant predictor of regional metastasis and may be a marker for tumor aggressiveness and clin. outcome.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpMP7JFlxFFtLVg90H21EOLACvtfcHk0lhinJItNx73Nw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXksVajsrc%253D&md5=ccdd010cd52d1e198259a2edddf93ff7</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DEstilo%26aufirst%3DC.%2BL.%26aulast%3DO-charoenrat%26aufirst%3DP.%26aulast%3DNgai%26aufirst%3DI.%26aulast%3DPatel%26aufirst%3DS.%2BG.%26aulast%3DReddy%26aufirst%3DP.%2BG.%26aulast%3DDao%26aufirst%3DS.%26aulast%3DShaha%26aufirst%3DA.%2BR.%26aulast%3DKraus%26aufirst%3DD.%2BH.%26aulast%3DBoyle%26aufirst%3DJ.%2BO.%26aulast%3DWong%26aufirst%3DR.%2BJ.%26aulast%3DPfister%26aufirst%3DD.%2BG.%26aulast%3DHuryn%26aufirst%3DJ.%2BM.%26aulast%3DZlotolow%26aufirst%3DI.%2BM.%26aulast%3DShah%26aufirst%3DJ.%2BP.%26aulast%3DSingh%26aufirst%3DB.%26atitle%3DThe%2520role%2520of%2520novel%2520oncogenes%2520squamous%2520cell%2520carcinoma-related%2520oncogene%2520and%2520phosphatidylinositol%25203-kinase%2520p110%25CE%25B1%2520in%2520squamous%2520cell%2520carcinoma%2520of%2520the%2520oral%2520tongue%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2003%26volume%3D9%26spage%3D2300%26epage%3D2306" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Broderick, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Golas, Bj</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pham, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Towe, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Talbot, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaufman, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bains, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huryn, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yonekawa, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carlson, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hambardzumyan, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramanathan, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, B.</span></span> <span> </span><span class="NLM_article-title">SCCRO promotes glioma formation and malignant progression in mice</span>. <i>Neoplasia</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">476</span>– <span class="NLM_lpage">484</span>, <span class="refDoi"> DOI: 10.1593/neo.10202</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=10.1593%2Fneo.10202" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=20563250" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=1%3ACAS%3A528%3ADC%252BC3cXntFWjtb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2010&pages=476-484&author=S.+R.+Broderickauthor=Bj+Golasauthor=D.+Phamauthor=C.+W.+Toweauthor=S.+G.+Talbotauthor=A.+Kaufmanauthor=S.+Bainsauthor=L.+A.+Hurynauthor=Y.+Yonekawaauthor=D.+Carlsonauthor=D.+Hambardzumyanauthor=Y.+Ramanathanauthor=B.+Singh&title=SCCRO+promotes+glioma+formation+and+malignant+progression+in+mice&doi=10.1593%2Fneo.10202"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">SCCRO promotes glioma formation and malignant progression in mice</span></div><div class="casAuthors">Broderick, Stephen R.; Golas, Benjamin J.; Pham, Duykhanh; Towe, Christopher W.; Talbot, Simon G.; Kaufman, Andrew; Bains, Sarina; Huryn, Laryssa A.; Yonekawa, Yoshihiro; Carlson, Diane; Hambardzumyan, Dolores; Ramanathan, Yegnanarayana; Singh, Bhuvanesh</div><div class="citationInfo"><span class="NLM_cas:title">Neoplasia (Ann Arbor, MI, United States)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">476-484</span>CODEN:
                <span class="NLM_cas:coden">NEOPFL</span>;
        ISSN:<span class="NLM_cas:issn">1522-8002</span>.
    
            (<span class="NLM_cas:orgname">Neoplasia Press Inc.</span>)
        </div><div class="casAbstract">Originally identified as an oncogene activated by amplification in squamous cell carcinomas, several lines of evidence now suggest that squamous cell carcinoma-related oncogene (SCCRO; aka DCUN1D1) may play a role in the pathogenesis of a wide range of human cancers including gliomas.  SCCRO's oncogenic function is substantiated by its ectopic expression, resulting in transformation of cells in culture and xenograft formation in nude mice.  The aim of this study was to assess the in vivo oncogenicity of SCCRO in a murine model.  Ubiquitous expression of SCCRO resulted in early embryonic lethality.  Because SCCRO overexpression was detected in human gliomas, its in vivo oncogenic activity was assessed in an established murine glioma model.  Conditional expression of SCCRO using a replication-competent ASLV long terminal repeat with splice acceptor/nestin-(tumor virus-A) tv-a model system was not sufficient to induce tumor formation in a wild-type genetic background, but tumors formed with increasing frequency and decreasing latency in facilitated background contg. Ink4a deletion alone or in combination with PTEN loss.  Ectopic expression of SCCRO in glial progenitor cells resulted in lower-grade gliomas in Ink4a-/- mice, whereas its expression in Ink4a-/-/PTEN-/- background produced high-grade glioblastoma-like lesions that were indistinguishable from human tumors.  Expression of SCCRO with platelet-derived growth factor-beta (PDGF-β) resulted in an increased proportion of mice forming glioblastoma-like tumors compared with those induced by PDGF-β alone.  This work substantiates SCCRO's function as an oncogene by showing its ability to facilitate malignant transformation and carcinogenic progression in vivo and supports a role for SCCRO in the pathogenesis of gliomas and other human cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpSMNrn3xVDKrVg90H21EOLACvtfcHk0lhinJItNx73Nw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXntFWjtb0%253D&md5=22114aee39c7f21d5149880d2fbd99f2</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1593%2Fneo.10202&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1593%252Fneo.10202%26sid%3Dliteratum%253Aachs%26aulast%3DBroderick%26aufirst%3DS.%2BR.%26aulast%3DGolas%26aufirst%3DBj%26aulast%3DPham%26aufirst%3DD.%26aulast%3DTowe%26aufirst%3DC.%2BW.%26aulast%3DTalbot%26aufirst%3DS.%2BG.%26aulast%3DKaufman%26aufirst%3DA.%26aulast%3DBains%26aufirst%3DS.%26aulast%3DHuryn%26aufirst%3DL.%2BA.%26aulast%3DYonekawa%26aufirst%3DY.%26aulast%3DCarlson%26aufirst%3DD.%26aulast%3DHambardzumyan%26aufirst%3DD.%26aulast%3DRamanathan%26aufirst%3DY.%26aulast%3DSingh%26aufirst%3DB.%26atitle%3DSCCRO%2520promotes%2520glioma%2520formation%2520and%2520malignant%2520progression%2520in%2520mice%26jtitle%3DNeoplasia%26date%3D2010%26volume%3D12%26spage%3D476%26epage%3D484%26doi%3D10.1593%2Fneo.10202" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yoo, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, S.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lym, K.-I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, S. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, S-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoo, C.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jung, J.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, S.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, C.-S.</span></span> <span> </span><span class="NLM_article-title">Immunohistochemical expression of DCUN1D1 in non-small cell lung carcinoma: its relation to brain metastasis</span>. <i>Cancer Res. Treat.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>44</i></span>,  <span class="NLM_fpage">57</span>– <span class="NLM_lpage">62</span>, <span class="refDoi"> DOI: 10.4143/crt.2012.44.1.57</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=10.4143%2Fcrt.2012.44.1.57" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=22500162" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=1%3ACAS%3A280%3ADC%252BC38rkslChtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2012&pages=57-62&author=J.+Yooauthor=S.-H.+Leeauthor=K.-I.+Lymauthor=S.+Y.+Parkauthor=S-H.+Yangauthor=C.-Y.+Yooauthor=J.-H.+Jungauthor=S.-J.+Kangauthor=C.-S.+Kang&title=Immunohistochemical+expression+of+DCUN1D1+in+non-small+cell+lung+carcinoma%3A+its+relation+to+brain+metastasis&doi=10.4143%2Fcrt.2012.44.1.57"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Immunohistochemical Expression of DCUN1D1 in Non-small Cell Lung Carcinoma: Its Relation to Brain Metastasis</span></div><div class="casAuthors">Yoo Jinyoung; Lee Seong-Hak; Lym Kwang-Il; Park Sonya Youngju; Yang Seung-Ho; Yoo Chang-Young; Jung Ji-Han; Kang Seok-Jin; Kang Chang-Suk</div><div class="citationInfo"><span class="NLM_cas:title">Cancer research and treatment : official journal of Korean Cancer Association</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">57-62</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">PURPOSE:  Non-small cell lung carcinoma (NSCLC) comprises 75-85% of all lung cancers, and approximately 25% of all NSCLC patients develop brain metastasis.  There are no reliable markers for predicting in which patients this metastasis will occur.  DCUN1D1, also known as squamous cell carcinoma-related oncogene, is associated with tumor progression and poor outcomes in NSCLC.  The objective of this study was to investigate the role of DCUN1D1 expression in cases of brain metastasis due to NSCLC.  MATERIALS AND METHODS:  Primary tumor samples from a total of 71 cases of NSCLC, either with (n=40) or without (n=31) brain metastasis, were evaluated for DCUN1D1 expression by immunohistochemistry analysis.  RESULTS:  DCUN1D1 expression was detected in 16 patients (23%) and tended to correlate with T classification (15% of T1-2 tumors vs. 30% of T3-4 tumors, p=0.083).  DCUN1D1 expression was significantly associated with tumor stage.  It was observed in none of the patients with stage I disease, 10% of those with stage II disease, and 29% with stage III disease (p=0.009).  In addition, 14 of 16 DCUN1D1-positive patients resulted in brain metastasis (p=0.01).  The odds ratio of brain metastasis for patients with DCUN1D1 expression was 3.112 (p=0.009).  CONCLUSION:  DCUN1D1 expression may play a role in tumor progression and development of brain metastasis in patients with NSCLC.  Evaluation of DCUN1D1 expression may provide assistance in identifying those patients who are at higher risk for brain metastasis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQCEHsxuDVeOuIHSmpohpd0fW6udTcc2ea6UKR17_FAtbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC38rkslChtQ%253D%253D&md5=408e62c5272bb8a82a26ce06de38145b</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.4143%2Fcrt.2012.44.1.57&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4143%252Fcrt.2012.44.1.57%26sid%3Dliteratum%253Aachs%26aulast%3DYoo%26aufirst%3DJ.%26aulast%3DLee%26aufirst%3DS.-H.%26aulast%3DLym%26aufirst%3DK.-I.%26aulast%3DPark%26aufirst%3DS.%2BY.%26aulast%3DYang%26aufirst%3DS-H.%26aulast%3DYoo%26aufirst%3DC.-Y.%26aulast%3DJung%26aufirst%3DJ.-H.%26aulast%3DKang%26aufirst%3DS.-J.%26aulast%3DKang%26aufirst%3DC.-S.%26atitle%3DImmunohistochemical%2520expression%2520of%2520DCUN1D1%2520in%2520non-small%2520cell%2520lung%2520carcinoma%253A%2520its%2520relation%2520to%2520brain%2520metastasis%26jtitle%3DCancer%2520Res.%2520Treat.%26date%3D2012%26volume%3D44%26spage%3D57%26epage%3D62%26doi%3D10.4143%2Fcrt.2012.44.1.57" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Scott, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hammill, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Min, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rhee, D. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Connelly, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sviderskiy, V. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhasin, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ong, S.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chai, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goktug, A. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Monda, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Low, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paulo, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cannon, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shelat, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kelsall, I. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alpi, A. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pagala, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harper, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schulman, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guy, R. K.</span></span> <span> </span><span class="NLM_article-title">Blocking an N-terminal acetylation–dependent protein interaction inhibits an E3 ligase</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">850</span>– <span class="NLM_lpage">857</span>, <span class="refDoi"> DOI: 10.1038/nchembio.2386</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=10.1038%2Fnchembio.2386" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=28581483" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=1%3ACAS%3A528%3ADC%252BC2sXpt1Omt7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2017&pages=850-857&author=D.+C.+Scottauthor=J.+T.+Hammillauthor=J.+Minauthor=D.+Y.+Rheeauthor=M.+Connellyauthor=V.+O.+Sviderskiyauthor=D.+Bhasinauthor=Y.+Chenauthor=S.-S.+Ongauthor=S.+C.+Chaiauthor=A.+N.+Goktugauthor=G.+Huangauthor=J.+K.+Mondaauthor=J.+Lowauthor=H.+S.+Kimauthor=J.+A.+Pauloauthor=J.+R.+Cannonauthor=A.+A.+Shelatauthor=T.+Chenauthor=I.+R.+Kelsallauthor=A.+F.+Alpiauthor=V.+Pagalaauthor=X.+Wangauthor=J.+Pengauthor=B.+Singhauthor=J.+W.+Harperauthor=B.+A.+Schulmanauthor=R.+K.+Guy&title=Blocking+an+N-terminal+acetylation%E2%80%93dependent+protein+interaction+inhibits+an+E3+ligase&doi=10.1038%2Fnchembio.2386"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Blocking an N-terminal acetylation-dependent protein interaction inhibits an E3 ligase</span></div><div class="casAuthors">Scott, Daniel C.; Hammill, Jared T.; Min, Jaeki; Rhee, David Y.; Connelly, Michele; Sviderskiy, Vladislav O.; Bhasin, Deepak; Chen, Yizhe; Ong, Su-Sien; Chai, Sergio C.; Goktug, Asli N.; Huang, Guochang; Monda, Julie K.; Low, Jonathan; Kim, Ho Shin; Paulo, Joao A.; Cannon, Joe R.; Shelat, Anang A.; Chen, Taosheng; Kelsall, Ian R.; Alpi, Arno F.; Pagala, Vishwajeeth; Wang, Xusheng; Peng, Junmin; Singh, Bhuvanesh; Harper, J. Wade; Schulman, Brenda A.; Guy, R. Kip</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">850-857</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">N-terminal acetylation is an abundant modification influencing protein functions.  Because ∼80% of mammalian cytosolic proteins are N-terminally acetylated, this modification is potentially an untapped target for chem. control of their functions.  Structural studies have revealed that, like lysine acetylation, N-terminal acetylation converts a pos. charged amine into a hydrophobic handle that mediates protein interactions; hence, this modification may be a druggable target.  We report the development of chem. probes targeting the N-terminal acetylation-dependent interaction between an E2 conjugating enzyme (UBE2M or UBC12) and DCN1 (DCUN1D1), a subunit of a multiprotein E3 ligase for the ubiquitin-like protein NEDD8.  The inhibitors are highly selective with respect to other protein acetyl-amide-binding sites, inhibit NEDD8 ligation in vitro and in cells, and suppress anchorage-independent growth of a cell line with DCN1 amplification.  Overall, our data demonstrate that N-terminal acetyl-dependent protein interactions are druggable targets and provide insights into targeting multiprotein E2-E3 ligases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqh7JDyzBEGlrVg90H21EOLACvtfcHk0lgAQ-aW6-g9Eg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXpt1Omt7k%253D&md5=de7519fbd287df91e53f8722d361d69f</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.2386&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.2386%26sid%3Dliteratum%253Aachs%26aulast%3DScott%26aufirst%3DD.%2BC.%26aulast%3DHammill%26aufirst%3DJ.%2BT.%26aulast%3DMin%26aufirst%3DJ.%26aulast%3DRhee%26aufirst%3DD.%2BY.%26aulast%3DConnelly%26aufirst%3DM.%26aulast%3DSviderskiy%26aufirst%3DV.%2BO.%26aulast%3DBhasin%26aufirst%3DD.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DOng%26aufirst%3DS.-S.%26aulast%3DChai%26aufirst%3DS.%2BC.%26aulast%3DGoktug%26aufirst%3DA.%2BN.%26aulast%3DHuang%26aufirst%3DG.%26aulast%3DMonda%26aufirst%3DJ.%2BK.%26aulast%3DLow%26aufirst%3DJ.%26aulast%3DKim%26aufirst%3DH.%2BS.%26aulast%3DPaulo%26aufirst%3DJ.%2BA.%26aulast%3DCannon%26aufirst%3DJ.%2BR.%26aulast%3DShelat%26aufirst%3DA.%2BA.%26aulast%3DChen%26aufirst%3DT.%26aulast%3DKelsall%26aufirst%3DI.%2BR.%26aulast%3DAlpi%26aufirst%3DA.%2BF.%26aulast%3DPagala%26aufirst%3DV.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DPeng%26aufirst%3DJ.%26aulast%3DSingh%26aufirst%3DB.%26aulast%3DHarper%26aufirst%3DJ.%2BW.%26aulast%3DSchulman%26aufirst%3DB.%2BA.%26aulast%3DGuy%26aufirst%3DR.%2BK.%26atitle%3DBlocking%2520an%2520N-terminal%2520acetylation%25E2%2580%2593dependent%2520protein%2520interaction%2520inhibits%2520an%2520E3%2520ligase%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2017%26volume%3D13%26spage%3D850%26epage%3D857%26doi%3D10.1038%2Fnchembio.2386" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kurz, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chou, Y. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Willems, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meyer-Schaller, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hecht, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tyers, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peter, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sicheri, F.</span></span> <span> </span><span class="NLM_article-title">Dcn1 functions as a scaffold-type E3 ligase for cullin neddylation</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">23</span>– <span class="NLM_lpage">35</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2007.12.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=10.1016%2Fj.molcel.2007.12.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=18206966" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtlWjt7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2008&pages=23-35&author=T.+Kurzauthor=Y.+C.+Chouauthor=A.+R.+Willemsauthor=N.+Meyer-Schallerauthor=M.+L.+Hechtauthor=M.+Tyersauthor=M.+Peterauthor=F.+Sicheri&title=Dcn1+functions+as+a+scaffold-type+E3+ligase+for+cullin+neddylation&doi=10.1016%2Fj.molcel.2007.12.012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Dcn1 functions as a scaffold-type E3 ligase for cullin neddylation</span></div><div class="casAuthors">Kurz, Thimo; Chou, Yang-Chieh; Willems, Andrew R.; Meyer-Schaller, Nathalie; Hecht, Marie-Lyn; Tyers, Mike; Peter, Matthias; Sicheri, Frank</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">23-35</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Cullin-based E3 ubiquitin ligases are activated through modification of the cullin subunit with the ubiquitin-like protein Nedd8.  Dcn1 regulates cullin neddylation and thus ubiquitin ligase activity.  Here we describe the 1.9 Å x-ray crystal structure of yeast Dcn1 encompassing an N-terminal ubiquitin-binding (UBA) domain and a C-terminal domain of unique architecture, which we termed PONY domain.  A conserved surface on Dcn1 is required for direct binding to cullins and for neddylation.  The reciprocal binding site for Dcn1 on Cdc53 is located ∼18 Å from the site of neddylation.  Dcn1 does not require cysteine residues for catalytic function, and directly interacts with the Nedd8 E2 Ubc12 on a surface that overlaps with the E1-binding site.  We show that Dcn1 is necessary and sufficient for cullin neddylation in a purified recombinant system.  Taken together, these data demonstrate that Dcn1 is a scaffold-like E3 ligase for cullin neddylation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoUMPyViHz6WLVg90H21EOLACvtfcHk0lgAQ-aW6-g9Eg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtlWjt7k%253D&md5=288151ba7d80c529a546ef2b74274516</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2007.12.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2007.12.012%26sid%3Dliteratum%253Aachs%26aulast%3DKurz%26aufirst%3DT.%26aulast%3DChou%26aufirst%3DY.%2BC.%26aulast%3DWillems%26aufirst%3DA.%2BR.%26aulast%3DMeyer-Schaller%26aufirst%3DN.%26aulast%3DHecht%26aufirst%3DM.%2BL.%26aulast%3DTyers%26aufirst%3DM.%26aulast%3DPeter%26aufirst%3DM.%26aulast%3DSicheri%26aufirst%3DF.%26atitle%3DDcn1%2520functions%2520as%2520a%2520scaffold-type%2520E3%2520ligase%2520for%2520cullin%2520neddylation%26jtitle%3DMol.%2520Cell%26date%3D2008%26volume%3D29%26spage%3D23%26epage%3D35%26doi%3D10.1016%2Fj.molcel.2007.12.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Scott, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Monda, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bennett, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harper, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schulman, B. A.</span></span> <span> </span><span class="NLM_article-title">N-terminal acetylation acts as an avidity enhancer within an interconnected multiprotein complex</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>334</i></span>,  <span class="NLM_fpage">674</span>– <span class="NLM_lpage">678</span>, <span class="refDoi"> DOI: 10.1126/science.1209307</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=10.1126%2Fscience.1209307" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=21940857" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtlyqu7jL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=334&publication_year=2011&pages=674-678&author=D.+C.+Scottauthor=J.+K.+Mondaauthor=E.+J.+Bennettauthor=J.+W.+Harperauthor=B.+A.+Schulman&title=N-terminal+acetylation+acts+as+an+avidity+enhancer+within+an+interconnected+multiprotein+complex&doi=10.1126%2Fscience.1209307"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">N-Terminal Acetylation Acts as an Avidity Enhancer Within an Interconnected Multiprotein Complex</span></div><div class="casAuthors">Scott, Daniel C.; Monda, Julie K.; Bennett, Eric J.; Harper, J. Wade; Schulman, Brenda A.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">334</span>
        (<span class="NLM_cas:issue">6056</span>),
    <span class="NLM_cas:pages">674-678</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Although many eukaryotic proteins are amino (N)-terminally acetylated, structural mechanisms by which N-terminal acetylation mediates protein interactions are largely unknown.  Here, N-terminal acetylation of the E2 enzyme, Ubc12, dictates distinctive E3-dependent ligation of the ubiquitin-like protein Nedd8 to Cul1.  Structural, biochem., biophys., and genetic analyses revealed how complete burial of Ubc12's N-acetyl-methionine in a hydrophobic pocket in the E3, Dcn1, promotes cullin neddylation.  The results suggest that the N-terminal acetyl both directs Ubc12's interactions with Dcn1 and prevents repulsion of a charged N terminus.  The authors' data provide a link between acetylation and ubiquitin-like protein conjugation and define a mechanism for N-terminal acetylation-dependent recognition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq2YZdYRdU_fbVg90H21EOLACvtfcHk0ljBSorQT82LUg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtlyqu7jL&md5=1654bea16702633a9baf611827f60450</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1126%2Fscience.1209307&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1209307%26sid%3Dliteratum%253Aachs%26aulast%3DScott%26aufirst%3DD.%2BC.%26aulast%3DMonda%26aufirst%3DJ.%2BK.%26aulast%3DBennett%26aufirst%3DE.%2BJ.%26aulast%3DHarper%26aufirst%3DJ.%2BW.%26aulast%3DSchulman%26aufirst%3DB.%2BA.%26atitle%3DN-terminal%2520acetylation%2520acts%2520as%2520an%2520avidity%2520enhancer%2520within%2520an%2520interconnected%2520multiprotein%2520complex%26jtitle%3DScience%26date%3D2011%26volume%3D334%26spage%3D674%26epage%3D678%26doi%3D10.1126%2Fscience.1209307" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Scott, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sviderskiy, V. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Monda, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lydeard, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cho, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harper, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schulman, B. A.</span></span> <span> </span><span class="NLM_article-title">Structure of a RING E3 trapped in action reveals ligation mechanism for the ubiquitin-like protein NEDD8</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>157</i></span>,  <span class="NLM_fpage">1671</span>– <span class="NLM_lpage">1684</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2014.04.037</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=10.1016%2Fj.cell.2014.04.037" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=24949976" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtVCisrfP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=157&publication_year=2014&pages=1671-1684&author=D.+C.+Scottauthor=V.+O.+Sviderskiyauthor=J.+K.+Mondaauthor=J.+R.+Lydeardauthor=S.+E.+Choauthor=J.+W.+Harperauthor=B.+A.+Schulman&title=Structure+of+a+RING+E3+trapped+in+action+reveals+ligation+mechanism+for+the+ubiquitin-like+protein+NEDD8&doi=10.1016%2Fj.cell.2014.04.037"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of a RING E3 trapped in action reveals ligation mechanism for the ubiquitin-like protein NEDD8</span></div><div class="casAuthors">Scott, Daniel C.; Sviderskiy, Vladislav O.; Monda, Julie K.; Lydeard, John R.; Cho, Shein Ei; Harper, J. Wade; Schulman, Brenda A.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">157</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1671-1684</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Most E3 ligases use a RING domain to activate a thioester-linked E2∼ubiquitin-like protein (UBL) intermediate and promote UBL transfer to a remotely bound target protein.  Nonetheless, RING E3 mechanisms matching a specific UBL and acceptor lysine remain elusive, including for RBX1, which mediates NEDD8 ligation to cullins and >10% of all ubiquitination.  We report the structure of a trapped RING E3-E2∼UBL-target intermediate representing RBX1-UBC12∼NEDD8-CUL1-DCN1, which reveals the mechanism of NEDD8 ligation and how a particular UBL and acceptor lysine are matched by a multifunctional RING E3.  Numerous mechanisms specify cullin neddylation while preventing noncognate ubiquitin ligation.  Notably, E2-E3-target and RING-E2∼UBL modules are not optimized to function independently, but instead require integration by the UBL and target for maximal reactivity.  The UBL and target regulate the catalytic machinery by positioning the RING-E2∼UBL catalytic center, licensing the acceptor lysine, and influencing E2 reactivity, thereby driving their specific coupling by a multifunctional RING E3.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpVAVv0lWWAVLVg90H21EOLACvtfcHk0ljBSorQT82LUg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtVCisrfP&md5=6674c0e32fba481709a4e1d1932d3127</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2014.04.037&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2014.04.037%26sid%3Dliteratum%253Aachs%26aulast%3DScott%26aufirst%3DD.%2BC.%26aulast%3DSviderskiy%26aufirst%3DV.%2BO.%26aulast%3DMonda%26aufirst%3DJ.%2BK.%26aulast%3DLydeard%26aufirst%3DJ.%2BR.%26aulast%3DCho%26aufirst%3DS.%2BE.%26aulast%3DHarper%26aufirst%3DJ.%2BW.%26aulast%3DSchulman%26aufirst%3DB.%2BA.%26atitle%3DStructure%2520of%2520a%2520RING%2520E3%2520trapped%2520in%2520action%2520reveals%2520ligation%2520mechanism%2520for%2520the%2520ubiquitin-like%2520protein%2520NEDD8%26jtitle%3DCell%26date%3D2014%26volume%3D157%26spage%3D1671%26epage%3D1684%26doi%3D10.1016%2Fj.cell.2014.04.037" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kurz, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ozlu, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rudolf, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Rourke, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luke, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hofmann, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hyman, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bowerman, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peter, M.</span></span> <span> </span><span class="NLM_article-title">The conserved protein DCN-1/Dcn1p is required for cullin neddylation in <i>C. elegans</i> and <i>S. cerevisiae</i></span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>435</i></span>,  <span class="NLM_fpage">1257</span>– <span class="NLM_lpage">1261</span>, <span class="refDoi"> DOI: 10.1038/nature03662</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=10.1038%2Fnature03662" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=15988528" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=1%3ACAS%3A528%3ADC%252BD2MXls1Ontb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=435&publication_year=2005&pages=1257-1261&author=T.+Kurzauthor=N.+Ozluauthor=F.+Rudolfauthor=S.+M.+O%E2%80%99Rourkeauthor=B.+Lukeauthor=K.+Hofmannauthor=A.+A.+Hymanauthor=B.+Bowermanauthor=M.+Peter&title=The+conserved+protein+DCN-1%2FDcn1p+is+required+for+cullin+neddylation+in+C.+elegans+and+S.+cerevisiae&doi=10.1038%2Fnature03662"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">The conserved protein DCN-1/Dcn1p is required for cullin neddylation in C. elegans and S. cerevisiae</span></div><div class="casAuthors">Kurz, Thimo; Oezlue, Nurhan; Rudolf, Fabian; O'Rourke, Sean M.; Luke, Brian; Hofmann, Kay; Hyman, Anthony A.; Bowerman, Bruce; Peter, Matthias</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">435</span>
        (<span class="NLM_cas:issue">7046</span>),
    <span class="NLM_cas:pages">1257-1261</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">SCF-type E3 ubiquitin ligases are multi-protein complexes required for polyubiquitination and subsequent degrdn. of target proteins by the 26S proteasome.  Cullins, together with the RING-finger protein Rbx1, form the catalytic core of the ligase, and recruit the substrate-recognition module.  Cycles of covalent modification of cullins by the ubiquitin-like mol. Nedd8 (neddylation) and removal of Nedd8 by the COP9 signalosome (deneddylation) pos. regulate E3 ligase activity.  Here we report the identification and anal. of a widely conserved protein that is required for cullin neddylation in the nematode Caenorhabditis elegans and the yeast Saccharomyces cerevisiae.  C. elegans DCN-1 and S. cerevisiae Dcn1p (defective in cullin neddylation) are characterized by a novel UBA-like ubiquitin-binding domain and a DUF298 domain of unknown function.  Consistent with their requirements for neddylation, DCN-1 and Dcn1p directly bind Nedd8 and phys. assoc. with cullins in both spp.  Moreover, overexpression of Dcn1p in yeast results in the accumulation of Nedd8-modified cullin Cdc53p.  Both in vivo and in vitro expts. indicate that Dcn1p does not inhibit deneddylation of Cdc53p by the COP9 signalosome, but greatly increases the kinetics of the neddylation reaction.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrgm_SbEnUuBrVg90H21EOLACvtfcHk0ljBSorQT82LUg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXls1Ontb0%253D&md5=545afac53b6c989be8b2fe5c78336d43</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1038%2Fnature03662&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature03662%26sid%3Dliteratum%253Aachs%26aulast%3DKurz%26aufirst%3DT.%26aulast%3DOzlu%26aufirst%3DN.%26aulast%3DRudolf%26aufirst%3DF.%26aulast%3DO%25E2%2580%2599Rourke%26aufirst%3DS.%2BM.%26aulast%3DLuke%26aufirst%3DB.%26aulast%3DHofmann%26aufirst%3DK.%26aulast%3DHyman%26aufirst%3DA.%2BA.%26aulast%3DBowerman%26aufirst%3DB.%26aulast%3DPeter%26aufirst%3DM.%26atitle%3DThe%2520conserved%2520protein%2520DCN-1%252FDcn1p%2520is%2520required%2520for%2520cullin%2520neddylation%2520in%2520C.%2520elegans%2520and%2520S.%2520cerevisiae%26jtitle%3DNature%26date%3D2005%26volume%3D435%26spage%3D1257%26epage%3D1261%26doi%3D10.1038%2Fnature03662" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernard, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fernandez-Salas, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chinnaswamy, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Layton, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stuckey, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pan, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span> <span> </span><span class="NLM_article-title">A potent small-molecule inhibitor of the DCN1-UBC12 interaction that selectively blocks cullin 3 neddylation</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">1150</span>, <span class="refDoi"> DOI: 10.1038/s41467-017-01243-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=10.1038%2Fs41467-017-01243-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=29074978" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=1%3ACAS%3A280%3ADC%252BC1M7kvFOquw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=1150&author=H.+Zhouauthor=J.+Luauthor=L.+Liuauthor=D.+Bernardauthor=C.-Y.+Yangauthor=E.+Fernandez-Salasauthor=K.+Chinnaswamyauthor=S.+Laytonauthor=J.+Stuckeyauthor=Q.+Yuauthor=W.+Zhouauthor=Z.+Panauthor=Y.+Sunauthor=S.+Wang&title=A+potent+small-molecule+inhibitor+of+the+DCN1-UBC12+interaction+that+selectively+blocks+cullin+3+neddylation&doi=10.1038%2Fs41467-017-01243-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">A potent small-molecule inhibitor of the DCN1-UBC12 interaction that selectively blocks cullin 3 neddylation</span></div><div class="casAuthors">Zhou Haibin; Lu Jianfeng; Liu Liu; Bernard Denzil; Yang Chao-Yie; Wang Shaomeng; Fernandez-Salas Ester; Chinnaswamy Krishnapriya; Layton Stephanie; Stuckey Jeanne; Yu Qing; Sun Yi; Zhou Weihua; Sun Yi; Pan Zhenqiang; Wang Shaomeng; Wang Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Nature communications</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1150</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The Cullin-RING E3 ubiquitin ligases (CRLs) regulate homeostasis of ~20% of cellular proteins and their activation require neddylation of their cullin subunit.  Cullin neddylation is modulated by a scaffolding DCN protein through interactions with both the cullin protein and an E2 enzyme such as UBC12.  Here we report the development of DI-591 as a high-affinity, cell-permeable small-molecule inhibitor of the DCN1-UBC12 interaction.  DI-591 binds to purified recombinant human DCN1 and DCN2 proteins with K i values of 10-12 nM, and disrupts the DCN1-UBC12 interaction in cells.  Treatment with DI-591 selectively converts cellular cullin 3 into an un-neddylated inactive form with no or minimum effect on other cullin members.  Our data firmly establish a previously unrecognized specific role of the DCN1-UBC12 interaction for cellular neddylation of cullin 3.  DI-591 is an excellent probe compound to investigate the role of the cullin 3 CRL ligase in biological processes and human diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTn87oIyDIsYOxsqfvU0ziafW6udTcc2eY9bygfRzRUk7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1M7kvFOquw%253D%253D&md5=47a713f31bcad6e0b45196cb3f1c8766</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1038%2Fs41467-017-01243-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41467-017-01243-7%26sid%3Dliteratum%253Aachs%26aulast%3DZhou%26aufirst%3DH.%26aulast%3DLu%26aufirst%3DJ.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DBernard%26aufirst%3DD.%26aulast%3DYang%26aufirst%3DC.-Y.%26aulast%3DFernandez-Salas%26aufirst%3DE.%26aulast%3DChinnaswamy%26aufirst%3DK.%26aulast%3DLayton%26aufirst%3DS.%26aulast%3DStuckey%26aufirst%3DJ.%26aulast%3DYu%26aufirst%3DQ.%26aulast%3DZhou%26aufirst%3DW.%26aulast%3DPan%26aufirst%3DZ.%26aulast%3DSun%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DA%2520potent%2520small-molecule%2520inhibitor%2520of%2520the%2520DCN1-UBC12%2520interaction%2520that%2520selectively%2520blocks%2520cullin%25203%2520neddylation%26jtitle%3DNat.%2520Commun.%26date%3D2017%26volume%3D8%26spage%3D1150%26doi%3D10.1038%2Fs41467-017-01243-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hammill, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Min, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Connelly, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holbrook, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matheny, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schulman, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guy, R. K.</span></span> <span> </span><span class="NLM_article-title">Piperidinyl ureas chemically control defective in cullin neddylation 1 (DCN1)-mediated cullin neddylation</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">2680</span>– <span class="NLM_lpage">2693</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b01277</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b01277" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=1%3ACAS%3A528%3ADC%252BC1cXltVGks70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=2680-2693&author=J.+T.+Hammillauthor=D.+C.+Scottauthor=J.+Minauthor=M.+C.+Connellyauthor=G.+Holbrookauthor=F.+Zhuauthor=A.+Mathenyauthor=L.+Yangauthor=B.+Singhauthor=B.+A.+Schulmanauthor=R.+K.+Guy&title=Piperidinyl+ureas+chemically+control+defective+in+cullin+neddylation+1+%28DCN1%29-mediated+cullin+neddylation&doi=10.1021%2Facs.jmedchem.7b01277"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Piperidinyl Ureas Chemically Control Defective in Cullin Neddylation 1 (DCN1)-Mediated Cullin Neddylation</span></div><div class="casAuthors">Hammill, Jared T.; Scott, Daniel C.; Min, Jaeki; Connelly, Michele C.; Holbrook, Gloria; Zhu, Fangyi; Matheny, Amy; Yang, Lei; Singh, Bhuvanesh; Schulman, Brenda A.; Guy, R. Kiplin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2680-2693</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The authors previously discovered and validated a class of piperidinyl ureas that regulate defective in cullin neddylation 1 (DCN1)-dependent neddylation of cullins.  Here, the authors report preliminary structure-activity relationship studies aimed at advancing the high-throughput screen hit into a tractable tool compd. for dissecting the effects of acute DCN1-UBE2M inhibition on the NEDD8/cullin pathway.  Structure-enabled optimization led to a 100-fold increase in biochem. potency and modestly increased soly. and permeability as compared to the initial hit.  The optimized compds. inhibit the DCN1-UBE2M protein-protein interaction in the TR-FRET binding assay and inhibit cullin neddylation in the pulse-chase NEDD8 transfer assay.  The optimized compds. bind to DCN1 and selectively reduce steady-state levels of neddylated CUL1 and CUL3 in a squamous cell carcinoma cell line.  Ultimately, the authors anticipate that these studies will identify early lead compds. for clin. development for the treatment of lung squamous cell carcinomas and other cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoC1AvhDCXYqrVg90H21EOLACvtfcHk0lh3mSAPQz9aqg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXltVGks70%253D&md5=142c48248f4adcbdceca071790adabbd</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b01277&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b01277%26sid%3Dliteratum%253Aachs%26aulast%3DHammill%26aufirst%3DJ.%2BT.%26aulast%3DScott%26aufirst%3DD.%2BC.%26aulast%3DMin%26aufirst%3DJ.%26aulast%3DConnelly%26aufirst%3DM.%2BC.%26aulast%3DHolbrook%26aufirst%3DG.%26aulast%3DZhu%26aufirst%3DF.%26aulast%3DMatheny%26aufirst%3DA.%26aulast%3DYang%26aufirst%3DL.%26aulast%3DSingh%26aufirst%3DB.%26aulast%3DSchulman%26aufirst%3DB.%2BA.%26aulast%3DGuy%26aufirst%3DR.%2BK.%26atitle%3DPiperidinyl%2520ureas%2520chemically%2520control%2520defective%2520in%2520cullin%2520neddylation%25201%2520%2528DCN1%2529-mediated%2520cullin%2520neddylation%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D2680%26epage%3D2693%26doi%3D10.1021%2Facs.jmedchem.7b01277" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Scott, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sviderskiy, V. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Monda, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lydeard, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cho, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harper, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schulman, B. A.</span></span> <span> </span><span class="NLM_article-title">Structure of a ring E3 trapped in action reveals ligation mechanism for the ubiquitin-like protein Nedd8</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>157</i></span>,  <span class="NLM_fpage">1671</span>– <span class="NLM_lpage">1684</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2014.04.037</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=10.1016%2Fj.cell.2014.04.037" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=24949976" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtVCisrfP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=157&publication_year=2014&pages=1671-1684&author=D.+C.+Scottauthor=V.+O.+Sviderskiyauthor=J.+K.+Mondaauthor=J.+R.+Lydeardauthor=S.+E.+Choauthor=J.+W.+Harperauthor=B.+A.+Schulman&title=Structure+of+a+ring+E3+trapped+in+action+reveals+ligation+mechanism+for+the+ubiquitin-like+protein+Nedd8&doi=10.1016%2Fj.cell.2014.04.037"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of a RING E3 trapped in action reveals ligation mechanism for the ubiquitin-like protein NEDD8</span></div><div class="casAuthors">Scott, Daniel C.; Sviderskiy, Vladislav O.; Monda, Julie K.; Lydeard, John R.; Cho, Shein Ei; Harper, J. Wade; Schulman, Brenda A.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">157</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1671-1684</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Most E3 ligases use a RING domain to activate a thioester-linked E2∼ubiquitin-like protein (UBL) intermediate and promote UBL transfer to a remotely bound target protein.  Nonetheless, RING E3 mechanisms matching a specific UBL and acceptor lysine remain elusive, including for RBX1, which mediates NEDD8 ligation to cullins and >10% of all ubiquitination.  We report the structure of a trapped RING E3-E2∼UBL-target intermediate representing RBX1-UBC12∼NEDD8-CUL1-DCN1, which reveals the mechanism of NEDD8 ligation and how a particular UBL and acceptor lysine are matched by a multifunctional RING E3.  Numerous mechanisms specify cullin neddylation while preventing noncognate ubiquitin ligation.  Notably, E2-E3-target and RING-E2∼UBL modules are not optimized to function independently, but instead require integration by the UBL and target for maximal reactivity.  The UBL and target regulate the catalytic machinery by positioning the RING-E2∼UBL catalytic center, licensing the acceptor lysine, and influencing E2 reactivity, thereby driving their specific coupling by a multifunctional RING E3.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpVAVv0lWWAVLVg90H21EOLACvtfcHk0lh3mSAPQz9aqg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtVCisrfP&md5=6674c0e32fba481709a4e1d1932d3127</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2014.04.037&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2014.04.037%26sid%3Dliteratum%253Aachs%26aulast%3DScott%26aufirst%3DD.%2BC.%26aulast%3DSviderskiy%26aufirst%3DV.%2BO.%26aulast%3DMonda%26aufirst%3DJ.%2BK.%26aulast%3DLydeard%26aufirst%3DJ.%2BR.%26aulast%3DCho%26aufirst%3DS.%2BE.%26aulast%3DHarper%26aufirst%3DJ.%2BW.%26aulast%3DSchulman%26aufirst%3DB.%2BA.%26atitle%3DStructure%2520of%2520a%2520ring%2520E3%2520trapped%2520in%2520action%2520reveals%2520ligation%2520mechanism%2520for%2520the%2520ubiquitin-like%2520protein%2520Nedd8%26jtitle%3DCell%26date%3D2014%26volume%3D157%26spage%3D1671%26epage%3D1684%26doi%3D10.1016%2Fj.cell.2014.04.037" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Scott, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Monda, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bennett, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harper, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schulman, B. A.</span></span> <span> </span><span class="NLM_article-title">N-terminal acetylation acts as an avidity enhancer within an interconnected multiprotein complex</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>334</i></span>,  <span class="NLM_fpage">674</span>– <span class="NLM_lpage">678</span>, <span class="refDoi"> DOI: 10.1126/science.1209307</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=10.1126%2Fscience.1209307" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=21940857" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtlyqu7jL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=334&publication_year=2011&pages=674-678&author=D.+C.+Scottauthor=J.+K.+Mondaauthor=E.+J.+Bennettauthor=J.+W.+Harperauthor=B.+A.+Schulman&title=N-terminal+acetylation+acts+as+an+avidity+enhancer+within+an+interconnected+multiprotein+complex&doi=10.1126%2Fscience.1209307"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">N-Terminal Acetylation Acts as an Avidity Enhancer Within an Interconnected Multiprotein Complex</span></div><div class="casAuthors">Scott, Daniel C.; Monda, Julie K.; Bennett, Eric J.; Harper, J. Wade; Schulman, Brenda A.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">334</span>
        (<span class="NLM_cas:issue">6056</span>),
    <span class="NLM_cas:pages">674-678</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Although many eukaryotic proteins are amino (N)-terminally acetylated, structural mechanisms by which N-terminal acetylation mediates protein interactions are largely unknown.  Here, N-terminal acetylation of the E2 enzyme, Ubc12, dictates distinctive E3-dependent ligation of the ubiquitin-like protein Nedd8 to Cul1.  Structural, biochem., biophys., and genetic analyses revealed how complete burial of Ubc12's N-acetyl-methionine in a hydrophobic pocket in the E3, Dcn1, promotes cullin neddylation.  The results suggest that the N-terminal acetyl both directs Ubc12's interactions with Dcn1 and prevents repulsion of a charged N terminus.  The authors' data provide a link between acetylation and ubiquitin-like protein conjugation and define a mechanism for N-terminal acetylation-dependent recognition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq2YZdYRdU_fbVg90H21EOLACvtfcHk0lh0QhtktbaiXA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtlyqu7jL&md5=1654bea16702633a9baf611827f60450</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1126%2Fscience.1209307&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1209307%26sid%3Dliteratum%253Aachs%26aulast%3DScott%26aufirst%3DD.%2BC.%26aulast%3DMonda%26aufirst%3DJ.%2BK.%26aulast%3DBennett%26aufirst%3DE.%2BJ.%26aulast%3DHarper%26aufirst%3DJ.%2BW.%26aulast%3DSchulman%26aufirst%3DB.%2BA.%26atitle%3DN-terminal%2520acetylation%2520acts%2520as%2520an%2520avidity%2520enhancer%2520within%2520an%2520interconnected%2520multiprotein%2520complex%26jtitle%3DScience%26date%3D2011%26volume%3D334%26spage%3D674%26epage%3D678%26doi%3D10.1126%2Fscience.1209307" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hammill, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhasin, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Min, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Connelly, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hammill, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holbrook, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeffries, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schulman, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guy, R. K.</span></span> <span> </span><span class="NLM_article-title">Discovery of an orally bioavailable inhibitor of defective in cullin neddylation 1 (DCN1)-mediated cullin neddylation</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">2694</span>– <span class="NLM_lpage">2706</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b01282</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b01282" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=1%3ACAS%3A528%3ADC%252BC1cXltVGkt7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=2694-2706&author=J.+T.+Hammillauthor=D.+Bhasinauthor=D.+C.+Scottauthor=J.+Minauthor=Y.+Chenauthor=Y.+Luauthor=L.+Yangauthor=H.+S.+Kimauthor=M.+C.+Connellyauthor=C.+Hammillauthor=G.+Holbrookauthor=C.+Jeffriesauthor=B.+Singhauthor=B.+A.+Schulmanauthor=R.+K.+Guy&title=Discovery+of+an+orally+bioavailable+inhibitor+of+defective+in+cullin+neddylation+1+%28DCN1%29-mediated+cullin+neddylation&doi=10.1021%2Facs.jmedchem.7b01282"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of an Orally Bioavailable Inhibitor of Defective in Cullin Neddylation 1 (DCN1)-Mediated Cullin Neddylation</span></div><div class="casAuthors">Hammill, Jared T.; Bhasin, Deepak; Scott, Daniel C.; Min, Jaeki; Chen, Yizhe; Lu, Yan; Yang, Lei; Kim, Ho Shin; Connelly, Michele C.; Hammill, Courtney; Holbrook, Gloria; Jeffries, Cynthia; Singh, Bhuvanesh; Schulman, Brenda A.; Guy, R. Kiplin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2694-2706</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The authors previously reported the discovery, validation, and structure-activity relationships of a series of piperidinyl ureas that potently inhibit the DCN1-UBE2M interaction.  The authors demonstrated that compd. I inhibits both the DCN1-UBE2M protein-protein interaction and DCN1-mediated cullin neddylation in biochem. assays and reduces levels of steady-state cullin neddylation in a squamous carcinoma cell line harboring DCN1 amplification.  Although compd. I exhibits good soly. and permeability, it is rapidly metabolized in microsomal models (CLint = 170 mL/min/kg).  This work lays out the discovery of an orally bioavailable analog, NAcM-OPT (II).  Compd. II retains the favorable biochem. and cellular activity of compd. I but is significantly more stable both in vitro and in vivo.  Compd. II is orally bioavailable, well tolerated in mice, and currently used to study the effects of acute pharmacol. inhibition of the DCN1-UBE2M interaction on the NEDD8/CUL pathway.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq4fvhBFy2Ux7Vg90H21EOLACvtfcHk0lh0QhtktbaiXA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXltVGkt7c%253D&md5=5775690be4373f8d8a6cb401812363dd</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b01282&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b01282%26sid%3Dliteratum%253Aachs%26aulast%3DHammill%26aufirst%3DJ.%2BT.%26aulast%3DBhasin%26aufirst%3DD.%26aulast%3DScott%26aufirst%3DD.%2BC.%26aulast%3DMin%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DLu%26aufirst%3DY.%26aulast%3DYang%26aufirst%3DL.%26aulast%3DKim%26aufirst%3DH.%2BS.%26aulast%3DConnelly%26aufirst%3DM.%2BC.%26aulast%3DHammill%26aufirst%3DC.%26aulast%3DHolbrook%26aufirst%3DG.%26aulast%3DJeffries%26aufirst%3DC.%26aulast%3DSingh%26aufirst%3DB.%26aulast%3DSchulman%26aufirst%3DB.%2BA.%26aulast%3DGuy%26aufirst%3DR.%2BK.%26atitle%3DDiscovery%2520of%2520an%2520orally%2520bioavailable%2520inhibitor%2520of%2520defective%2520in%2520cullin%2520neddylation%25201%2520%2528DCN1%2529-mediated%2520cullin%2520neddylation%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D2694%26epage%3D2706%26doi%3D10.1021%2Facs.jmedchem.7b01282" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chern, T.-R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chinnaswamy, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernard, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stuckey, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span> <span> </span><span class="NLM_article-title">High-Affinity Peptidomimetic inhibitors of the DCN1-UBC12 protein–protein interaction</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">1934</span>– <span class="NLM_lpage">1950</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b01455</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b01455" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=1%3ACAS%3A528%3ADC%252BC1cXis1Kqsr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=1934-1950&author=H.+Zhouauthor=W.+Zhouauthor=B.+Zhouauthor=L.+Liuauthor=T.-R.+Chernauthor=K.+Chinnaswamyauthor=J.+Luauthor=D.+Bernardauthor=C.-Y.+Yangauthor=S.+Liauthor=M.+Wangauthor=J.+Stuckeyauthor=Y.+Sunauthor=S.+Wang&title=High-Affinity+Peptidomimetic+inhibitors+of+the+DCN1-UBC12+protein%E2%80%93protein+interaction&doi=10.1021%2Facs.jmedchem.7b01455"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">High-affinity peptidomimetic inhibitors of the DCN1-UBC12 protein-protein interaction</span></div><div class="casAuthors">Zhou, Haibin; Zhou, Weihua; Zhou, Bing; Liu, Liu; Chern, Ting-Rong; Chinnaswamy, Krishnapriya; Lu, Jianfeng; Bernard, Denzil; Yang, Chao-Yie; Li, Shasha; Wang, Mi; Stuckey, Jeanne; Sun, Yi; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1934-1950</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The Cullin-RING ligases (CRLs) regulate the turnover of approx. 20% of the proteins in mammalian cells and are emerging therapeutic targets in human diseases.  The activation of CRLs requires the neddylation of their cullin subunit, which is controlled by an activation complex consisting of Cullin-RBX1-UBC12-NEDD8-DCN1.  Herein, we describe the design, synthesis, and evaluation of peptidomimetics targeting the DCN1-UBC12 protein-protein interaction.  Starting from a 12-residue UBC12 peptide, we have successfully obtained a series of peptidomimetic compds. that bind to DCN1 protein with KD values of <10 nM.  Detn. of a cocrystal structure of a potent peptidomimetic inhibitor complexed with DCN1 provides the structural basis for their high-affinity interaction.  Cellular investigation of one potent DCN1 inhibitor, compd. (I) (DI-404), reveals that it effectively and selectively inhibits the neddylation of cullin 3 over other cullin members.  Further optimization of DI-404 may yield a new class of therapeutics for the treatment of human diseases in which cullin 3 CRL plays a key role.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqtfHxzNV467rVg90H21EOLACvtfcHk0lh0QhtktbaiXA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXis1Kqsr0%253D&md5=6489af488ea0608aed7348662e9b261f</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b01455&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b01455%26sid%3Dliteratum%253Aachs%26aulast%3DZhou%26aufirst%3DH.%26aulast%3DZhou%26aufirst%3DW.%26aulast%3DZhou%26aufirst%3DB.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DChern%26aufirst%3DT.-R.%26aulast%3DChinnaswamy%26aufirst%3DK.%26aulast%3DLu%26aufirst%3DJ.%26aulast%3DBernard%26aufirst%3DD.%26aulast%3DYang%26aufirst%3DC.-Y.%26aulast%3DLi%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DStuckey%26aufirst%3DJ.%26aulast%3DSun%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DHigh-Affinity%2520Peptidomimetic%2520inhibitors%2520of%2520the%2520DCN1-UBC12%2520protein%25E2%2580%2593protein%2520interaction%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D1934%26epage%3D1950%26doi%3D10.1021%2Facs.jmedchem.7b01455" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, R.-H.</span></span> <span> </span><span class="NLM_article-title">Cullin 3 ubiquitin ligases in cancer biology: functions and therapeutic implications</span>. <i>Front. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">113</span>, <span class="refDoi"> DOI: 10.3389/fonc.2016.00113</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=10.3389%2Ffonc.2016.00113" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=27200299" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=1%3ACAS%3A280%3ADC%252BC2s%252FhsFKisQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2016&pages=113&author=H.-Y.+Chenauthor=R.-H.+Chen&title=Cullin+3+ubiquitin+ligases+in+cancer+biology%3A+functions+and+therapeutic+implications&doi=10.3389%2Ffonc.2016.00113"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Cullin 3 Ubiquitin Ligases in Cancer Biology: Functions and Therapeutic Implications</span></div><div class="casAuthors">Chen Hsin-Yi; Chen Ruey-Hwa</div><div class="citationInfo"><span class="NLM_cas:title">Frontiers in oncology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">113</span>
        ISSN:<span class="NLM_cas:issn">2234-943X</span>.
    </div><div class="casAbstract">Cullin-RING ubiquitin ligases are the largest E3 ligase family in eukaryotes and are multiprotein complexes.  In these complexes, the Cullin protein serves as a scaffold to connect two functional modules of the ligases, the catalytic subunit and substrate-binding subunit.  To date, eight members of the Cullin family proteins have been identified.  In the Cul3 ubiquitin ligases, Bric-a-brac/Tramtrack/Broad complex (BTB) domain-containing proteins function as a bridge to connect Cul3 and substrates.  While the BTB domain is responsible for Cul3 binding, these proteins usually contain an additional domain for substrate interaction, such as MATH, kelch, Zn finger, and PAM, Highwire, and RPM-1 (PHR domain).  With the existence of a large number of BTB proteins in human, the Cul3 ubiquitin ligases ubiquitinate a wide range of substrates involving in diverse cellular functions.  In this review, we will discuss recent advances on the functions of Cul3 ubiquitin ligases in cancer development, progression, and therapeutic response and the dysregulation of Cul3-mediated ubiquitination events in human malignancies.  In particular, we will focus on three Cul3 substrate adaptors, kelch-like ECH-associated protein (Keap1), kelch-like family member 20 (KLHL20), and speckle type BTB/POZ protein (SPOP), with the intent to highlight novel targets in cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQMbUYXuVDp7K91pGlMMoT5fW6udTcc2eaTA8L4SSAeOrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2s%252FhsFKisQ%253D%253D&md5=4ab4fb3c41c92783f89f2c0910f39399</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.3389%2Ffonc.2016.00113&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3389%252Ffonc.2016.00113%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DH.-Y.%26aulast%3DChen%26aufirst%3DR.-H.%26atitle%3DCullin%25203%2520ubiquitin%2520ligases%2520in%2520cancer%2520biology%253A%2520functions%2520and%2520therapeutic%2520implications%26jtitle%3DFront.%2520Oncol.%26date%3D2016%26volume%3D6%26spage%3D113%26doi%3D10.3389%2Ffonc.2016.00113" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">North, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tao, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dai, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, W.</span></span> <span> </span><span class="NLM_article-title">Functional analysis of cullin 3 E3 ligases in tumorigenesis</span>. <i>Biochim. Biophys. Acta, Rev. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>1869</i></span>,  <span class="NLM_fpage">11</span>– <span class="NLM_lpage">28</span>, <span class="refDoi"> DOI: 10.1016/j.bbcan.2017.11.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=10.1016%2Fj.bbcan.2017.11.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=29128526" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvValt7zO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1869&publication_year=2018&pages=11-28&author=J.+Chengauthor=J.+Guoauthor=Z.+Wangauthor=B.+J.+Northauthor=K.+Taoauthor=X.+Daiauthor=W.+Wei&title=Functional+analysis+of+cullin+3+E3+ligases+in+tumorigenesis&doi=10.1016%2Fj.bbcan.2017.11.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Functional analysis of Cullin 3 E3 ligases in tumorigenesis</span></div><div class="casAuthors">Cheng, Ji; Guo, Jianping; Wang, Zhiwei; North, Brian J.; Tao, Kaixiong; Dai, Xiangpeng; Wei, Wenyi</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, Reviews on Cancer</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">1869</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">11-28</span>CODEN:
                <span class="NLM_cas:coden">BBACEU</span>;
        ISSN:<span class="NLM_cas:issn">0304-419X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Cullin 3-RING ligases (CRL3) play pivotal roles in the regulation of various physiol. and pathol. processes, including neoplastic events.  The substrate adaptors of CRL3 typically contain a BTB domain that mediates the interaction between Cullin 3 and target substrates to promote their ubiquitination and subsequent degrdn.  The biol. implications of CRL3 adaptor proteins have been well described where they have been found to play a role as either an oncogene, tumor suppressor, or can mediate either of these effects in a context-dependent manner.  Among the extensively studied CRL3-based E3 ligases, the role of the adaptor protein SPOP (speckle type BTB/POZ protein) in tumorigenesis appears to be tissue or cellular context dependent.  Specifically, SPOP acts as a tumor suppressor via destabilizing downstream oncoproteins in many malignancies, esp. in prostate cancer.  However, SPOP has largely an oncogenic role in kidney cancer.  Keap1, another well-characterized CRL3 adaptor protein, likely serves as a tumor suppressor within diverse malignancies, mainly due to its specific turnover of its downstream oncogenic substrate, NRF2 (nuclear factor erythroid 2-related factor 2).  In accordance with the physiol. role the various CRL3 adaptors exhibit, several pharmacol. agents have been developed to disrupt its E3 ligase activity, therefore blocking its potential oncogenic activity to mitigate tumorigenesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoiqKWKPAR9CLVg90H21EOLACvtfcHk0lh1gR1y8ouyVg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvValt7zO&md5=acd511a43f773fd6ea65179e18403ffc</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1016%2Fj.bbcan.2017.11.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbcan.2017.11.001%26sid%3Dliteratum%253Aachs%26aulast%3DCheng%26aufirst%3DJ.%26aulast%3DGuo%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DZ.%26aulast%3DNorth%26aufirst%3DB.%2BJ.%26aulast%3DTao%26aufirst%3DK.%26aulast%3DDai%26aufirst%3DX.%26aulast%3DWei%26aufirst%3DW.%26atitle%3DFunctional%2520analysis%2520of%2520cullin%25203%2520E3%2520ligases%2520in%2520tumorigenesis%26jtitle%3DBiochim.%2520Biophys.%2520Acta%252C%2520Rev.%2520Cancer%26date%3D2018%26volume%3D1869%26spage%3D11%26epage%3D28%26doi%3D10.1016%2Fj.bbcan.2017.11.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kumari, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumar, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lynn, A.</span></span> <span> </span><span class="NLM_article-title"><i>g_mmpbsa -</i> a GROMACS tool for high-throughput MM-PBSA calculations</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">1951</span>– <span class="NLM_lpage">1962</span>, <span class="refDoi"> DOI: 10.1021/ci500020m</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci500020m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=1%3ACAS%3A528%3ADC%252BC2cXotlWiu7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2014&pages=1951-1962&author=R.+Kumariauthor=R.+Kumarauthor=A.+Lynn&title=g_mmpbsa+-+a+GROMACS+tool+for+high-throughput+MM-PBSA+calculations&doi=10.1021%2Fci500020m"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">g_mmpbsa-A GROMACS Tool for High-Throughput MM-PBSA Calculations</span></div><div class="casAuthors">Kumari, Rashmi; Kumar, Rajendra; Lynn, Andrew</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1951-1962</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Mol. mechanics Poisson-Boltzmann surface area (MM-PBSA), a method to est. interaction free energies, has been increasingly used in the study of biomol. interactions.  Recently, this method has also been applied as a scoring function in computational drug design.  Here a new tool g_mmpbsa, which implements the MM-PBSA approach using subroutines written inhouse or sourced from the GROMACS and APBS packages is described. g_mmpbsa was developed as part of the Open Source Drug Discovery (OSDD) consortium.  Its aim is to integrate high-throughput mol. dynamics (MD) simulations with binding energy calcns.  The tool provides options to select alternative at. radii and different nonpolar solvation models including models based on the solvent accessible surface area (SASA), solvent accessible vol. (SAV), and a model which contains both repulsive (SASA-SAV) and attractive components (described using a Weeks-Chandler-Andersen like integral method).  We showcase the effectiveness of the tool by comparing the calcd. interaction energy of 37 structurally diverse HIV-1 protease inhibitor complexes with their exptl. binding free energies.  The effect of varying several combinations of input parameters such as at. radii, dielec. const., grid resoln., solute-solvent dielec. boundary definition, and nonpolar models was investigated. g_mmpbsa can also be used to est. the energy contribution per residue to the binding energy.  It has been used to identify those residues in HIV-1 protease that are most crit. for binding a range of inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrckNA7gLtUGrVg90H21EOLACvtfcHk0lh1gR1y8ouyVg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXotlWiu7o%253D&md5=e70a138b71eed07054d58f91ba1a227f</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1021%2Fci500020m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci500020m%26sid%3Dliteratum%253Aachs%26aulast%3DKumari%26aufirst%3DR.%26aulast%3DKumar%26aufirst%3DR.%26aulast%3DLynn%26aufirst%3DA.%26atitle%3Dg_mmpbsa%2520-%2520a%2520GROMACS%2520tool%2520for%2520high-throughput%2520MM-PBSA%2520calculations%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2014%26volume%3D54%26spage%3D1951%26epage%3D1962%26doi%3D10.1021%2Fci500020m" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wu, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nielsen, T. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clausen, M. H.</span></span> <span> </span><span class="NLM_article-title">Small-molecule kinase inhibitors: an analysis of FDA-approved drugs</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">5</span>– <span class="NLM_lpage">10</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2015.07.008</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=10.1016%2Fj.drudis.2015.07.008" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=26210956" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1Gru73P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2016&pages=5-10&author=P.+Wuauthor=T.+E.+Nielsenauthor=M.+H.+Clausen&title=Small-molecule+kinase+inhibitors%3A+an+analysis+of+FDA-approved+drugs&doi=10.1016%2Fj.drudis.2015.07.008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Small-molecule kinase inhibitors: an analysis of FDA-approved drugs</span></div><div class="casAuthors">Wu, Peng; Nielsen, Thomas E.; Clausen, Mads H.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">5-10</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Small-mol. kinase inhibitors (SMKIs), 28 of which are approved by the US Food and Drug Administration (FDA), have been actively pursued as promising targeted therapeutics.  Here, we assess the key structural and physicochem. properties, target selectivity and mechanism of function, and therapeutic indications of these approved inhibitors.  Our anal. showed that >30% of approved SMKIs have a mol. wt. (MW) exceeding 500 and all have a total ring count of between three and five.  The assumption that type II inhibitors tend to be more selective than type I inhibitors has been proved to be unreliable.  Although previous SMKI research was concd. on tyrosine kinase inhibitors for cancer treatment, recent progress indicates diversification of SMKI research in terms of new targets, mechanistic types, and therapeutic indications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpZrY68ZOI2xLVg90H21EOLACvtfcHk0lhJz8E6qg4Scw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht1Gru73P&md5=228524fa464393053ecbfac19e994929</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2015.07.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2015.07.008%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DP.%26aulast%3DNielsen%26aufirst%3DT.%2BE.%26aulast%3DClausen%26aufirst%3DM.%2BH.%26atitle%3DSmall-molecule%2520kinase%2520inhibitors%253A%2520an%2520analysis%2520of%2520FDA-approved%2520drugs%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2016%26volume%3D21%26spage%3D5%26epage%3D10%26doi%3D10.1016%2Fj.drudis.2015.07.008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sun, Q.-A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hess, D. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miyagi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stamler, J. S.</span></span> <span> </span><span class="NLM_article-title">Off-target thiol alkylation by the NADPH oxidase inhibitor 3-benzyl-7-(2-benzoxazolyl)thio-1,2,3-triazolo[4,5-d]pyrimidine (VAS2870)</span>. <i>Free Radical Biol. Med.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">1897</span>– <span class="NLM_lpage">1902</span>, <span class="refDoi"> DOI: 10.1016/j.freeradbiomed.2012.02.046</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=10.1016%2Fj.freeradbiomed.2012.02.046" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=22406319" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=1%3ACAS%3A528%3ADC%252BC38XmtlWmsLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2012&pages=1897-1902&author=Q.-A.+Sunauthor=D.+T.+Hessauthor=B.+Wangauthor=M.+Miyagiauthor=J.+S.+Stamler&title=Off-target+thiol+alkylation+by+the+NADPH+oxidase+inhibitor+3-benzyl-7-%282-benzoxazolyl%29thio-1%2C2%2C3-triazolo%5B4%2C5-d%5Dpyrimidine+%28VAS2870%29&doi=10.1016%2Fj.freeradbiomed.2012.02.046"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Off-target thiol alkylation by the NADPH oxidase inhibitor 3-benzyl-7-(2-benzoxazolyl)thio-1,2,3-triazolo[4,5-d]pyrimidine (VAS2870)</span></div><div class="casAuthors">Sun, Qi-An; Hess, Douglas T.; Wang, Benlian; Miyagi, Masaru; Stamler, Jonathan S.</div><div class="citationInfo"><span class="NLM_cas:title">Free Radical Biology & Medicine</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1897-1902</span>CODEN:
                <span class="NLM_cas:coden">FRBMEH</span>;
        ISSN:<span class="NLM_cas:issn">0891-5849</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Specific inhibitors of the prodn. of reactive oxygen species (ROS) by the NADPH oxidases (Nox's) are potentially important therapeutic agents in the wide range of human diseases that are characterized by excessive ROS prodn.  It has been proposed that VAS2870 (3-benzyl-7-(2-benzoxazolyl)thio-1,2,3- triazolo[4,5-d]pyrimidine), identified as an inhibitor of Nox2 by small-mol. screening, may serve as an example of such an agent.  Here we show that VAS2870 inhibits ROS prodn. in the sarcoplasmic reticulum (SR) of mammalian skeletal muscle, previously identified with Nox4, and thereby abrogates O2-coupled redox regulation of the ryanodine receptor-Ca2+ channel (RyR1).  However, we also find that VAS2870 modifies directly identified cysteine thiols within RyR1.  Mass spectrometric anal. of RyR1 exposed in situ to VAS2870 and of VAS2870-treated glutathione indicated that thiol modification is through alkylation by the benzyltriazolopyrimidine moiety of VAS2870.  Thus, VAS2870 exerts significant off-target effects, and thiol alkylation by VAS2870 (and closely related Nox inhibitors) may in fact replicate some of the effects of ROS on cellular thiol redox status.  In addn., we show that SR-localized Nox4 is inhibited by other thiol-alkylating agents, consistent with a causal role for cysteine modification in the inhibition of ROS prodn. by VAS2870.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoSMAxkJX1GbbVg90H21EOLACvtfcHk0lhJz8E6qg4Scw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XmtlWmsLs%253D&md5=43dae8c22f646bd49a41392cf6d092f9</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1016%2Fj.freeradbiomed.2012.02.046&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.freeradbiomed.2012.02.046%26sid%3Dliteratum%253Aachs%26aulast%3DSun%26aufirst%3DQ.-A.%26aulast%3DHess%26aufirst%3DD.%2BT.%26aulast%3DWang%26aufirst%3DB.%26aulast%3DMiyagi%26aufirst%3DM.%26aulast%3DStamler%26aufirst%3DJ.%2BS.%26atitle%3DOff-target%2520thiol%2520alkylation%2520by%2520the%2520NADPH%2520oxidase%2520inhibitor%25203-benzyl-7-%25282-benzoxazolyl%2529thio-1%252C2%252C3-triazolo%255B4%252C5-d%255Dpyrimidine%2520%2528VAS2870%2529%26jtitle%3DFree%2520Radical%2520Biol.%2520Med.%26date%3D2012%26volume%3D52%26spage%3D1897%26epage%3D1902%26doi%3D10.1016%2Fj.freeradbiomed.2012.02.046" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Singh, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stoffel, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gogineni, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poluri, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pfister, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaha, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pathak, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bosl, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cordon-Cardo, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shah, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rao, P. H.</span></span> <span> </span><span class="NLM_article-title">Amplification of the 3q26.3 locus is associated with progression to invasive cancer and is a negative prognostic factor in head and neck squamous cell carcinomas</span>. <i>Am. J. Pathol.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>161</i></span>,  <span class="NLM_fpage">365</span>– <span class="NLM_lpage">371</span>, <span class="refDoi"> DOI: 10.1016/S0002-9440(10)64191-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=10.1016%2FS0002-9440%2810%2964191-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=12163360" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=1%3ACAS%3A528%3ADC%252BD38XntFektb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=161&publication_year=2002&pages=365-371&author=B.+Singhauthor=A.+Stoffelauthor=S.+Gogineniauthor=A.+Poluriauthor=D.+G.+Pfisterauthor=A.+R.+Shahaauthor=A.+Pathakauthor=G.+Boslauthor=C.+Cordon-Cardoauthor=J.+P.+Shahauthor=P.+H.+Rao&title=Amplification+of+the+3q26.3+locus+is+associated+with+progression+to+invasive+cancer+and+is+a+negative+prognostic+factor+in+head+and+neck+squamous+cell+carcinomas&doi=10.1016%2FS0002-9440%2810%2964191-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Amplification of the 3q26.3 locus is associated with progression to invasive cancer and is a negative prognostic factor in head and neck squamous cell carcinomas</span></div><div class="casAuthors">Singh, Bhuvanesh; Stoffel, Archontoula; Gogineni, Swarna; Poluri, Ashok; Pfister, David G.; Shaha, Ashok R.; Pathak, Alok; Bosl, George; Cordon-Cardo, Carlos; Shah, Jatin P.; Rao, Pulivarthi H.</div><div class="citationInfo"><span class="NLM_cas:title">American Journal of Pathology</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">161</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">365-371</span>CODEN:
                <span class="NLM_cas:coden">AJPAA4</span>;
        ISSN:<span class="NLM_cas:issn">0002-9440</span>.
    
            (<span class="NLM_cas:orgname">American Society for Investigative Pathology</span>)
        </div><div class="casAbstract">Amplification of the 3q26-q27 has a high prevalence in squamous cell carcinomas of mucosal origin, including those originating in the head and neck region.  To elucidate its role as a prognostic tool in head and neck squarnous cell carcinoma, a yeast artificial chromosome (YAC) contig spanning the entire 3q26-27 region was constructed.  The minimal region of amplification was refined within a 1- to 2-Mb genomic segment contained within three overlapping, nonchimeric YAC clones using sequential fluorescent in situ hybridization anal.  These YAC clones contg. the apex of amplification were used to develop a two-color fluorescence in situ hybridization assay and applied to the detection of 3q copy nos. in interphase nuclei on archival tumor tissue from 29 cases of normal mucosa, 20 of premalignant mucosa, and 50 of invasive head and neck squamous cell carcinomas.  The presence of 3q amplification increased from 3% in normal mucosa to 25% in premalignant mucosa and 56% in invasive cancers (P < 0.01).  In invasive tumors, low-level 3q amplification (3 to 4 × copy no.) was identified in 18 of 50 primary head and neck cancers and high-level amplification (>4 × copy no.) in 10 of 50 cases.  With a median follow-up of 82.5 mo, an increasing proportion of recurrences (32%, 72%, and 90%; P = 0.003) and cancer-related deaths (14%, 44%, and 70%; P = 0.006) were seen in patients with normal 3q copy no., low-level amplification, and high-level amplification, resp.  The 3-yr disease-free (69%, 56%, and 10%; P = 0.001) and cause-specific (94%, 83%, and 40%; P = 0.01) survivals also decreased from normal copy no. to low-level and high-level amplification.  Only high-level amplification at 3q remained a significant prognostic variable on multivariate anal. including common prognostic predictors for both disease-free (relative risk, 5.1; 95% confidence interval = 1.9 to 13.9) and cause-specific survival (relative risk, 7.6; 95% confidence interval = 1.9 to 29.6).  The findings suggest that the 3q copy no. status is an important marker for tumor progression and prognostication in patients with head and neck squamous cell carcinoma.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr0R6UH5w2KgrVg90H21EOLACvtfcHk0li8FqqM2Y3oBA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XntFektb4%253D&md5=d9e546b51e67befcde4da23006e3fa86</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1016%2FS0002-9440%2810%2964191-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0002-9440%252810%252964191-0%26sid%3Dliteratum%253Aachs%26aulast%3DSingh%26aufirst%3DB.%26aulast%3DStoffel%26aufirst%3DA.%26aulast%3DGogineni%26aufirst%3DS.%26aulast%3DPoluri%26aufirst%3DA.%26aulast%3DPfister%26aufirst%3DD.%2BG.%26aulast%3DShaha%26aufirst%3DA.%2BR.%26aulast%3DPathak%26aufirst%3DA.%26aulast%3DBosl%26aufirst%3DG.%26aulast%3DCordon-Cardo%26aufirst%3DC.%26aulast%3DShah%26aufirst%3DJ.%2BP.%26aulast%3DRao%26aufirst%3DP.%2BH.%26atitle%3DAmplification%2520of%2520the%25203q26.3%2520locus%2520is%2520associated%2520with%2520progression%2520to%2520invasive%2520cancer%2520and%2520is%2520a%2520negative%2520prognostic%2520factor%2520in%2520head%2520and%2520neck%2520squamous%2520cell%2520carcinomas%26jtitle%3DAm.%2520J.%2520Pathol.%26date%3D2002%26volume%3D161%26spage%3D365%26epage%3D371%26doi%3D10.1016%2FS0002-9440%2810%2964191-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Heselmeyer, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Macville, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schrock, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blegen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hellstrom, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shah, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Auer, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ried, T.</span></span> <span> </span><span class="NLM_article-title">Advanced-stage cervical carcinomas are defined by a recurrent pattern of chromosomal aberrations revealing high genetic instability and a consistent gain of chromosome arm 3q</span>. <i>Genes, Chromosomes Cancer</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">233</span>– <span class="NLM_lpage">240</span>, <span class="refDoi"> DOI: 10.1002/(SICI)1098-2264(199708)19:4<233::AID-GCC5>3.0.CO;2-Y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=10.1002%2F%28SICI%291098-2264%28199708%2919%3A4%3C233%3A%3AAID-GCC5%3E3.0.CO%3B2-Y" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=9258658" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=1%3ACAS%3A280%3ADyaK2svgtlaruw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=1997&pages=233-240&author=K.+Heselmeyerauthor=M.+Macvilleauthor=E.+Schrockauthor=H.+Blegenauthor=A.+C.+Hellstromauthor=K.+Shahauthor=G.+Auerauthor=T.+Ried&title=Advanced-stage+cervical+carcinomas+are+defined+by+a+recurrent+pattern+of+chromosomal+aberrations+revealing+high+genetic+instability+and+a+consistent+gain+of+chromosome+arm+3q&doi=10.1002%2F%28SICI%291098-2264%28199708%2919%3A4%3C233%3A%3AAID-GCC5%3E3.0.CO%3B2-Y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Advanced-stage cervical carcinomas are defined by a recurrent pattern of chromosomal aberrations revealing high genetic instability and a consistent gain of chromosome arm 3q</span></div><div class="casAuthors">Heselmeyer K; Macville M; Schrock E; Blegen H; Hellstrom A C; Shah K; Auer G; Ried T</div><div class="citationInfo"><span class="NLM_cas:title">Genes, chromosomes & cancer</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">233-40</span>
        ISSN:<span class="NLM_cas:issn">1045-2257</span>.
    </div><div class="casAbstract">We have analyzed 30 cases of advanced-stage cervical squamous cell carcinoma (stages IIb-IV) by comparative genomic hybridization (CGH).  The most consistent chromosomal gain in the aneuploid tumors was mapped to chromosome arm 3q in 77% of the cases.  Acquisition of genetic material also occurred frequently on Iq (47%), 5p (30%), 6p (27%), and 20 (23%).  Recurrent losses were mapped on 2q (33%), 3p (50%), 4 (33%), 8p (23%), and 13q (27%).  High-level copy number increases were mapped to chromosome 8, chromosome arms 3q, 5p, 8q, 12p, 14q, 17q, 19q, 20p, and 20q, and chromosomal bands 3q26-27, 9p23-24, 11q22-23, and 12p13.  In the majority of the cases, the presence of high-risk human papilloma virus genomes was detected.  High proliferative activity was accompanied by crude aneuploidy.  Increased p21/WAF-I activity, but low or undetectable expression of TP53 were representative for the immunophenotype.  This study confirms the importance of a gain of chromosome arm 3q in cervical carcinogenesis and identifies additional, recurrent chromosomal aberrations that are required for progression from stage I tumors to advanced-stage carcinomas.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQyf87gxuNQ5dhSDZx9hyjKfW6udTcc2ebOAKlWXhjSpLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADyaK2svgtlaruw%253D%253D&md5=d7db7362d12bf4e3787ea5ad4807e15f</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1002%2F%28SICI%291098-2264%28199708%2919%3A4%3C233%3A%3AAID-GCC5%3E3.0.CO%3B2-Y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F%2528SICI%25291098-2264%2528199708%252919%253A4%253C233%253A%253AAID-GCC5%253E3.0.CO%253B2-Y%26sid%3Dliteratum%253Aachs%26aulast%3DHeselmeyer%26aufirst%3DK.%26aulast%3DMacville%26aufirst%3DM.%26aulast%3DSchrock%26aufirst%3DE.%26aulast%3DBlegen%26aufirst%3DH.%26aulast%3DHellstrom%26aufirst%3DA.%2BC.%26aulast%3DShah%26aufirst%3DK.%26aulast%3DAuer%26aufirst%3DG.%26aulast%3DRied%26aufirst%3DT.%26atitle%3DAdvanced-stage%2520cervical%2520carcinomas%2520are%2520defined%2520by%2520a%2520recurrent%2520pattern%2520of%2520chromosomal%2520aberrations%2520revealing%2520high%2520genetic%2520instability%2520and%2520a%2520consistent%2520gain%2520of%2520chromosome%2520arm%25203q%26jtitle%3DGenes%252C%2520Chromosomes%2520Cancer%26date%3D1997%26volume%3D19%26spage%3D233%26epage%3D240%26doi%3D10.1002%2F%28SICI%291098-2264%28199708%2919%3A4%3C233%3A%3AAID-GCC5%3E3.0.CO%3B2-Y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Petersen, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bujard, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petersen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wolf, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goeze, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwendel, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Langreck, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gellert, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reichel, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Just, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">du Manoir, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cremer, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dietel, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ried, T.</span></span> <span> </span><span class="NLM_article-title">Patterns of chromosomal imbalances in adenocarcinoma and squamous cell carcinoma of the lung</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">2331</span>– <span class="NLM_lpage">2335</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=1997&pages=2331-2335&author=I.+Petersenauthor=M.+Bujardauthor=S.+Petersenauthor=G.+Wolfauthor=A.+Goezeauthor=A.+Schwendelauthor=H.+Langreckauthor=K.+Gellertauthor=M.+Reichelauthor=K.+Justauthor=S.+du+Manoirauthor=T.+Cremerauthor=M.+Dietelauthor=T.+Ried&title=Patterns+of+chromosomal+imbalances+in+adenocarcinoma+and+squamous+cell+carcinoma+of+the+lung"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DPetersen%26aufirst%3DI.%26aulast%3DBujard%26aufirst%3DM.%26aulast%3DPetersen%26aufirst%3DS.%26aulast%3DWolf%26aufirst%3DG.%26aulast%3DGoeze%26aufirst%3DA.%26aulast%3DSchwendel%26aufirst%3DA.%26aulast%3DLangreck%26aufirst%3DH.%26aulast%3DGellert%26aufirst%3DK.%26aulast%3DReichel%26aufirst%3DM.%26aulast%3DJust%26aufirst%3DK.%26aulast%3Ddu%2BManoir%26aufirst%3DS.%26aulast%3DCremer%26aufirst%3DT.%26aulast%3DDietel%26aufirst%3DM.%26aulast%3DRied%26aufirst%3DT.%26atitle%3DPatterns%2520of%2520chromosomal%2520imbalances%2520in%2520adenocarcinoma%2520and%2520squamous%2520cell%2520carcinoma%2520of%2520the%2520lung%26jtitle%3DCancer%2520Res.%26date%3D1997%26volume%3D57%26spage%3D2331%26epage%3D2335" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Z. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y. S.</span></span> <span> </span><span class="NLM_article-title">Clinical significance of SCCRO (DCUN1D1) in prostate cancer and its proliferation-inhibiting effect on Lncap cells</span>. <i>Eur. Rev. Med. Pharm. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">4283</span>– <span class="NLM_lpage">4291</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=29077169" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=1%3ACAS%3A280%3ADC%252BC1M7ltF2qsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2017&pages=4283-4291&author=Z.+H.+Zhangauthor=J.+Liauthor=F.+Luoauthor=Y.+S.+Wang&title=Clinical+significance+of+SCCRO+%28DCUN1D1%29+in+prostate+cancer+and+its+proliferation-inhibiting+effect+on+Lncap+cells"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Clinical significance of SCCRO (DCUN1D1) in prostate cancer and its proliferation-inhibiting effect on Lncap cells</span></div><div class="casAuthors">Zhang Z-H; Li J; Luo F; Wang Y-S</div><div class="citationInfo"><span class="NLM_cas:title">European review for medical and pharmacological sciences</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">4283-4291</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">OBJECTIVE:    SCCRO/DCUN1D1/DCN1 (squamous cell carcinoma-related oncogene/defective in cullin neddylation 1 domain containing 1/defective in cullin neddylation) is considered as an oncogene, but its role in the prostate cancer (PC) is still not clear.  The current study aims to investigate the expression of SCCRO in PC tumor tissues, further its clinical significance, and proliferation inhibiting effect on PC cells in vitro.  PATIENTS AND METHODS:  RT-PCR was used to detect the expression of SCCRO in PC tissue and corresponding adjacent normal tissues from 160 cases, and its relationship with clinical pathological characteristics was analyzed.  Small interfering RNA (siRNA) expression plasmid targeting SCCRO gene was constructed and transferred into PC cell line Lncap.  The effect on proliferation was observed by CCK8 assay, and its influence on invasion and migration of Lncap cells was studied by Transwell Matrigel assay after SCCRO gene was silenced.  The expression of focal adhesion kinase (FAK) and matrix metalloproteinase-2 (MMP-2) influenced by SCCRO silencing were detected by Western blot.  RESULTS:  mRNA expression of SCCRO protein increased significantly in cancer tissues compared to adjacent normal tissue, especially for T3+T4, N+, and III+IV patients (p<0.05).  SCCRO expression was an independent prognostic factor (p<0.05).  After SCCRO gene was knocked down by siRNA, the SCCRO protein level decreased 78.4% in the siRNA-3 group.  By CCK8 assay, knocking down SCCRO in Lncap significantly reduced the cell proliferation, as well as its migration and invasion capability compared to siRNA-control group (p<0.01) by transwell invasion and migration assay.  The expression of FAK and MMP-2 also reduced in siRNA-3 group compared to siRNA control group (p<0.01).  CONCLUSIONS:  SCCRO is associated with progression and prognosis of PC.  After SCCRO gene was transferred, the growth of Lncap cells was inhibited, and ability of invasion and migration decreased by reducing the expression of FAK and MMP-2.  SCCRO has potential to become a new target for the treatment of PC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQUF5iSyBgoY-R_TJajKxJDfW6udTcc2ebOAKlWXhjSpLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1M7ltF2qsg%253D%253D&md5=bc432df7a5551dede3c22a9add47d71b</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DZ.%2BH.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DLuo%26aufirst%3DF.%26aulast%3DWang%26aufirst%3DY.%2BS.%26atitle%3DClinical%2520significance%2520of%2520SCCRO%2520%2528DCUN1D1%2529%2520in%2520prostate%2520cancer%2520and%2520its%2520proliferation-inhibiting%2520effect%2520on%2520Lncap%2520cells%26jtitle%3DEur.%2520Rev.%2520Med.%2520Pharm.%2520Sci.%26date%3D2017%26volume%3D21%26spage%3D4283%26epage%3D4291" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pfleger, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pan, Z. Q.</span></span> <span> </span><span class="NLM_article-title">Mono-ubiquitination drives nuclear export of the human DCN1-like protein hDCNL1</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>286</i></span>,  <span class="NLM_fpage">34060</span>– <span class="NLM_lpage">34070</span>, <span class="refDoi"> DOI: 10.1074/jbc.M111.273045</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=10.1074%2Fjbc.M111.273045" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=286&publication_year=2011&pages=34060-34070&author=K.+Wuauthor=H.+Yanauthor=L.+Fangauthor=X.+Wangauthor=C.+Pflegerauthor=X.+Jiangauthor=L.+Huangauthor=Z.+Q.+Pan&title=Mono-ubiquitination+drives+nuclear+export+of+the+human+DCN1-like+protein+hDCNL1&doi=10.1074%2Fjbc.M111.273045"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M111.273045&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M111.273045%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DK.%26aulast%3DYan%26aufirst%3DH.%26aulast%3DFang%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DPfleger%26aufirst%3DC.%26aulast%3DJiang%26aufirst%3DX.%26aulast%3DHuang%26aufirst%3DL.%26aulast%3DPan%26aufirst%3DZ.%2BQ.%26atitle%3DMono-ubiquitination%2520drives%2520nuclear%2520export%2520of%2520the%2520human%2520DCN1-like%2520protein%2520hDCNL1%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2011%26volume%3D286%26spage%3D34060%26epage%3D34070%26doi%3D10.1074%2Fjbc.M111.273045" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Soucy, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, P. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Milhollen, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berger, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gavin, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adhikari, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brownell, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burke, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cardin, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Critchley, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cullis, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doucette, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garnsey, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaulin, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gershman, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lublinsky, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McDonald, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mizutani, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Narayanan, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olhava, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peluso, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rezaei, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sintchak, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Talreja, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Traore, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vyskocil, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weatherhead, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dick, L. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Claiborne, C. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rolfe, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bolen, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Langston, S. P.</span></span> <span> </span><span class="NLM_article-title">An inhibitor of nedd8-activating enzyme as a new approach to treat cancer</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>458</i></span>,  <span class="NLM_fpage">732</span>– <span class="NLM_lpage">736</span>, <span class="refDoi"> DOI: 10.1038/nature07884</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=10.1038%2Fnature07884" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=19360080" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=1%3ACAS%3A528%3ADC%252BD1MXksVenu7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=458&publication_year=2009&pages=732-736&author=T.+A.+Soucyauthor=P.+G.+Smithauthor=M.+A.+Milhollenauthor=A.+J.+Bergerauthor=J.+M.+Gavinauthor=S.+Adhikariauthor=J.+E.+Brownellauthor=K.+E.+Burkeauthor=D.+P.+Cardinauthor=S.+Critchleyauthor=C.+A.+Cullisauthor=A.+Doucetteauthor=J.+J.+Garnseyauthor=J.+L.+Gaulinauthor=R.+E.+Gershmanauthor=A.+R.+Lublinskyauthor=A.+McDonaldauthor=H.+Mizutaniauthor=U.+Narayananauthor=E.+J.+Olhavaauthor=S.+Pelusoauthor=M.+Rezaeiauthor=M.+D.+Sintchakauthor=T.+Talrejaauthor=M.+P.+Thomasauthor=T.+Traoreauthor=S.+Vyskocilauthor=G.+S.+Weatherheadauthor=J.+Yuauthor=J.+Zhangauthor=L.+R.+Dickauthor=C.+F.+Claiborneauthor=M.+Rolfeauthor=J.+B.+Bolenauthor=S.+P.+Langston&title=An+inhibitor+of+nedd8-activating+enzyme+as+a+new+approach+to+treat+cancer&doi=10.1038%2Fnature07884"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">An inhibitor of NEDD8-activating enzyme as a new approach to treat cancer</span></div><div class="casAuthors">Soucy, Teresa A.; Smith, Peter G.; Milhollen, Michael A.; Berger, Allison J.; Gavin, James M.; Adhikari, Sharmila; Brownell, James E.; Burke, Kristine E.; Cardin, David P.; Critchley, Stephen; Cullis, Courtney A.; Doucette, Amanda; Garnsey, James J.; Gaulin, Jeffrey L.; Gershman, Rachel E.; Lublinsky, Anna R.; McDonald, Alice; Mizutani, Hirotake; Narayanan, Usha; Olhava, Edward J.; Peluso, Stephane; Rezaei, Mansoureh; Sintchak, Michael D.; Talreja, Tina; Thomas, Michael P.; Traore, Tary; Vyskocil, Stepan; Weatherhead, Gabriel S.; Yu, Jie; Zhang, Julie; Dick, Lawrence R.; Claiborne, Christopher F.; Rolfe, Mark; Bolen, Joseph B.; Langston, Steven P.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">458</span>
        (<span class="NLM_cas:issue">7239</span>),
    <span class="NLM_cas:pages">732-736</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The clin. development of an inhibitor of cellular proteasome function suggests that compds. targeting other components of the ubiquitin-proteasome system might prove useful for the treatment of human malignancies.  NEDD8-activating enzyme (NAE) is an essential component of the NEDD8 conjugation pathway that controls the activity of the cullin-RING subtype of ubiquitin ligases, thereby regulating the turnover of a subset of proteins upstream of the proteasome.  Substrates of cullin-RING ligases have important roles in cellular processes assocd. with cancer cell growth and survival pathways.  Here we describe MLN4924, a potent and selective inhibitor of NAE.  MLN4924 disrupts cullin-RING ligase-mediated protein turnover leading to apoptotic death in human tumor cells by a new mechanism of action, the deregulation of S-phase DNA synthesis.  MLN4924 suppressed the growth of human tumor xenografts in mice at compd. exposures that were well tolerated.  Our data suggest that NAE inhibitors may hold promise for the treatment of cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr8fx7OnZtYjbVg90H21EOLACvtfcHk0ljJZPFgvBHNiA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXksVenu7Y%253D&md5=365d4b4feac3994849050e00f1aae5e0</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1038%2Fnature07884&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature07884%26sid%3Dliteratum%253Aachs%26aulast%3DSoucy%26aufirst%3DT.%2BA.%26aulast%3DSmith%26aufirst%3DP.%2BG.%26aulast%3DMilhollen%26aufirst%3DM.%2BA.%26aulast%3DBerger%26aufirst%3DA.%2BJ.%26aulast%3DGavin%26aufirst%3DJ.%2BM.%26aulast%3DAdhikari%26aufirst%3DS.%26aulast%3DBrownell%26aufirst%3DJ.%2BE.%26aulast%3DBurke%26aufirst%3DK.%2BE.%26aulast%3DCardin%26aufirst%3DD.%2BP.%26aulast%3DCritchley%26aufirst%3DS.%26aulast%3DCullis%26aufirst%3DC.%2BA.%26aulast%3DDoucette%26aufirst%3DA.%26aulast%3DGarnsey%26aufirst%3DJ.%2BJ.%26aulast%3DGaulin%26aufirst%3DJ.%2BL.%26aulast%3DGershman%26aufirst%3DR.%2BE.%26aulast%3DLublinsky%26aufirst%3DA.%2BR.%26aulast%3DMcDonald%26aufirst%3DA.%26aulast%3DMizutani%26aufirst%3DH.%26aulast%3DNarayanan%26aufirst%3DU.%26aulast%3DOlhava%26aufirst%3DE.%2BJ.%26aulast%3DPeluso%26aufirst%3DS.%26aulast%3DRezaei%26aufirst%3DM.%26aulast%3DSintchak%26aufirst%3DM.%2BD.%26aulast%3DTalreja%26aufirst%3DT.%26aulast%3DThomas%26aufirst%3DM.%2BP.%26aulast%3DTraore%26aufirst%3DT.%26aulast%3DVyskocil%26aufirst%3DS.%26aulast%3DWeatherhead%26aufirst%3DG.%2BS.%26aulast%3DYu%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DDick%26aufirst%3DL.%2BR.%26aulast%3DClaiborne%26aufirst%3DC.%2BF.%26aulast%3DRolfe%26aufirst%3DM.%26aulast%3DBolen%26aufirst%3DJ.%2BB.%26aulast%3DLangston%26aufirst%3DS.%2BP.%26atitle%3DAn%2520inhibitor%2520of%2520nedd8-activating%2520enzyme%2520as%2520a%2520new%2520approach%2520to%2520treat%2520cancer%26jtitle%3DNature%26date%3D2009%26volume%3D458%26spage%3D732%26epage%3D736%26doi%3D10.1038%2Fnature07884" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tong, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Si, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, W.</span></span> <span> </span><span class="NLM_article-title">MLN4924 (Pevonedistat), a protein neddylation inhibitor, suppresses proliferation and migration of human clear cell renal cell carcinoma</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">5599</span>, <span class="refDoi"> DOI: 10.1038/s41598-017-06098-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=10.1038%2Fs41598-017-06098-y" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=28717191" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=1%3ACAS%3A280%3ADC%252BC1cjosFWksg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2017&pages=5599&author=S.+Tongauthor=Y.+Siauthor=H.+Yuauthor=L.+Zhangauthor=P.+Xieauthor=W.+Jiang&title=MLN4924+%28Pevonedistat%29%2C+a+protein+neddylation+inhibitor%2C+suppresses+proliferation+and+migration+of+human+clear+cell+renal+cell+carcinoma&doi=10.1038%2Fs41598-017-06098-y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">MLN4924 (Pevonedistat), a protein neddylation inhibitor, suppresses proliferation and migration of human clear cell renal cell carcinoma</span></div><div class="casAuthors">Tong Shuai; Si Yang; Yu Hefen; Xie Ping; Jiang Wenguo; Tong Shuai; Si Yang; Yu Hefen; Xie Ping; Jiang Wenguo; Zhang Lingqiang; Jiang Wenguo</div><div class="citationInfo"><span class="NLM_cas:title">Scientific reports</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">5599</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Neddylation is a post-translational protein modification associated with cancer development.  MLN4924 is a neddylation inhibitor currently under investigation in multiple phase I studies on various malignancies, and its clincal name is Pevonedistat.  It has been documented that MLN4924 blocks Cullins neddylation and inactivates CRLs and, in turn, triggers cell-cycle arrest, apoptosis, senescence and autophagy in many cancer cells.  In this study, we investigated the anti-tumor effect of MLN4924 in human clear cell renal carcinoma (ccRCC).  Levels of both Nedd8 activating enzyme E1 and Nedd8-conjugating enzyme E2 were higher in ccRCC tissues and RCC cancer cells than in normal.  Moreover, MLN4924 treatment led to rapid inhibition of Cullin1 neddylation and notably suppressed growth and survival as well as migration in a dose-and time-dependent manner.  Mechanistic studies revealed that MLN4924 induced the accumulation of a number of CRL substrates, including p21, p27 and Wee1 to trigger DNA damage and induce growth arrest at the G2/M phase.  MLN4924 also induced anti-migration and anti-invasion by activating E-cadherin and repressing Vimentin.  Taken together, this study provides the first evidence that neddylation pathway is overactive in ccRCC and that MLN4924 induces dose-dependent anti-proliferation, anti-migration, anti-invasion in ccRCC cells.  The study thus indicates that MLN4924 has potential therapeutic value for the clinical treatment of renal cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQrdwMzHGpO0rrIbt-tGZmxfW6udTcc2ebbPPXIdUvCVrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1cjosFWksg%253D%253D&md5=1e0a80b533adac1674de53b9b68750a7</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1038%2Fs41598-017-06098-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41598-017-06098-y%26sid%3Dliteratum%253Aachs%26aulast%3DTong%26aufirst%3DS.%26aulast%3DSi%26aufirst%3DY.%26aulast%3DYu%26aufirst%3DH.%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DXie%26aufirst%3DP.%26aulast%3DJiang%26aufirst%3DW.%26atitle%3DMLN4924%2520%2528Pevonedistat%2529%252C%2520a%2520protein%2520neddylation%2520inhibitor%252C%2520suppresses%2520proliferation%2520and%2520migration%2520of%2520human%2520clear%2520cell%2520renal%2520cell%2520carcinoma%26jtitle%3DSci.%2520Rep.%26date%3D2017%26volume%3D7%26spage%3D5599%26doi%3D10.1038%2Fs41598-017-06098-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, H. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, G. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, S. S.</span></span> <span> </span><span class="NLM_article-title">MLN4924 suppresses neddylation and induces cell cycle arrest, senescence, and apoptosis in human osteosarcoma</span>. <i>Oncotarget</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">45263</span>– <span class="NLM_lpage">45274</span>, <span class="refDoi"> DOI: 10.18632/oncotarget.9481</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=10.18632%2Foncotarget.9481" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=45263-45274&author=Y.+Zhangauthor=C.+C.+Shiauthor=H.+P.+Zhangauthor=G.+Q.+Liauthor=S.+S.+Li&title=MLN4924+suppresses+neddylation+and+induces+cell+cycle+arrest%2C+senescence%2C+and+apoptosis+in+human+osteosarcoma&doi=10.18632%2Foncotarget.9481"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.9481&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.9481%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DShi%26aufirst%3DC.%2BC.%26aulast%3DZhang%26aufirst%3DH.%2BP.%26aulast%3DLi%26aufirst%3DG.%2BQ.%26aulast%3DLi%26aufirst%3DS.%2BS.%26atitle%3DMLN4924%2520suppresses%2520neddylation%2520and%2520induces%2520cell%2520cycle%2520arrest%252C%2520senescence%252C%2520and%2520apoptosis%2520in%2520human%2520osteosarcoma%26jtitle%3DOncotarget%26date%3D2016%26volume%3D7%26spage%3D45263%26epage%3D45274%26doi%3D10.18632%2Foncotarget.9481" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tsvetkov, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yeh, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, H.</span></span> <span> </span><span class="NLM_article-title">p27(Kip1) ubiquitination and degradation is regulated by the SCF(Skp2) complex through phosphorylated Thr187 in p27</span>. <i>Curr. Biol.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">661</span>– <span class="NLM_lpage">664</span>, <span class="refDoi"> DOI: 10.1016/S0960-9822(99)80290-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=10.1016%2FS0960-9822%2899%2980290-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=10375532" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=1%3ACAS%3A528%3ADyaK1MXjvFWis7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=1999&pages=661-664&author=L.+M.+Tsvetkovauthor=K.+H.+Yehauthor=S.+J.+Leeauthor=H.+Sunauthor=H.+Zhang&title=p27%28Kip1%29+ubiquitination+and+degradation+is+regulated+by+the+SCF%28Skp2%29+complex+through+phosphorylated+Thr187+in+p27&doi=10.1016%2FS0960-9822%2899%2980290-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">p27Kip1 Ubiquitination and degradation is regulated by the SCFSkp2 complex through phosphorylated Thr187 in p27</span></div><div class="casAuthors">Tsvetkov, Lyuben M.; Yeh, Kun-Huei; Lee, Soo-Jung; Sun, Hong; Zhang, Hui</div><div class="citationInfo"><span class="NLM_cas:title">Current Biology</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">661-664</span>CODEN:
                <span class="NLM_cas:coden">CUBLE2</span>;
        ISSN:<span class="NLM_cas:issn">0960-9822</span>.
    
            (<span class="NLM_cas:orgname">Current Biology Publications</span>)
        </div><div class="casAbstract">Many tumorigenic processes affect cell-cycle progression by their effects on the levels of the cyclin-dependent kinase inhibitor p27kip1.  The phosphorylation- and ubiquitination-dependent proteolysis of p27 is implicated in control of the G1-S transition in the cell cycle.  To det. the factors that control p27 stability, the authors established a cell-free ext. assay that recapitulates the degrdn. of p27.  Phosphorylation of p27 at Thr187 was essential for its degrdn.  Degrdn. was also dependent on SCFSkp2, a protein complex implicated in targeting phosphorylated proteins for ubiquitination.  Immunodepletion of components of the complex- Cul-1, Skp1, or Skp2-from the ext. abolished p27 degrdn., while addn. of purified SCFSkp2 to Skp2-depleted ext. restores the capacity to degrade p27.  A specific assocn. was obsd. between Skp2 and a p27 carboxy-terminal peptide contg. phosphorylated Thr187, but not between Skp2 and the non-phosphorylated peptide.  Skp2-dependent assocns. between Skp1 or Cul-1 and the p27 phosphopeptide were also detected.  Isolated SCFSkp2 contained an E3 ubiquitin ligase activity towards p27.  The data thus suggest that SCFSkp2 specifically targets p27 for degrdn. during cell-cycle progression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrOzAo67Ohy7bVg90H21EOLACvtfcHk0ljcXr1-hvENpQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXjvFWis7g%253D&md5=f36415d3077a9434a81cc1c074713340</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1016%2FS0960-9822%2899%2980290-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0960-9822%252899%252980290-5%26sid%3Dliteratum%253Aachs%26aulast%3DTsvetkov%26aufirst%3DL.%2BM.%26aulast%3DYeh%26aufirst%3DK.%2BH.%26aulast%3DLee%26aufirst%3DS.%2BJ.%26aulast%3DSun%26aufirst%3DH.%26aulast%3DZhang%26aufirst%3DH.%26atitle%3Dp27%2528Kip1%2529%2520ubiquitination%2520and%2520degradation%2520is%2520regulated%2520by%2520the%2520SCF%2528Skp2%2529%2520complex%2520through%2520phosphorylated%2520Thr187%2520in%2520p27%26jtitle%3DCurr.%2520Biol.%26date%3D1999%26volume%3D9%26spage%3D661%26epage%3D664%26doi%3D10.1016%2FS0960-9822%2899%2980290-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Carrano, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eytan, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hershko, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pagano, M.</span></span> <span> </span><span class="NLM_article-title">SKP2 is required for ubiquitin-mediated degradation of the CDK inhibitor p27</span>. <i>Nat. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">193</span>– <span class="NLM_lpage">199</span>, <span class="refDoi"> DOI: 10.1038/12013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=10.1038%2F12013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=10559916" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=1%3ACAS%3A528%3ADyaK1MXltlSjtr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=1999&pages=193-199&author=A.+C.+Carranoauthor=E.+Eytanauthor=A.+Hershkoauthor=M.+Pagano&title=SKP2+is+required+for+ubiquitin-mediated+degradation+of+the+CDK+inhibitor+p27&doi=10.1038%2F12013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">SKP2 is required for ubiquitin-mediated degradation of the CDK inhibitor p27</span></div><div class="casAuthors">Carrano, Andrea C.; Eytan, Esther; Hershko, Avram; Pagano, Michele</div><div class="citationInfo"><span class="NLM_cas:title">Nature Cell Biology</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">193-199</span>CODEN:
                <span class="NLM_cas:coden">NCBIFN</span>;
        ISSN:<span class="NLM_cas:issn">1465-7392</span>.
    
            (<span class="NLM_cas:orgname">Macmillan Magazines Ltd</span>)
        </div><div class="casAbstract">Degrdn. of the mammalian cyclin-dependent kinase (CDK) inhibitor p27 is required for the cellular transition from quiescence to the proliferative state.  The ubiquitination and subsequent degrdn. of p27 depend on its phosphorylation by cyclin-CDK complexes.  However, the ubiquitin-protein ligase necessary for p27 ubiquitination has not been identified.  Here we show that the F-box protein SKP2 specifically recognizes p27 in a phosphorylation-dependent manner that is characteristic of an F-box-protein-substrate interaction.  Furthermore, both in vivo and in vitro, SKP2 is a rate-limiting component of the machinery that ubiquitinates and degrades phosphorylated p27.  Thus, p27 degrdn. is subject to dual control by the accumulation of both SKP2 and cyclins following mitogenic stimulation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGobE2CKBARwdLVg90H21EOLACvtfcHk0litJdEvygdGhA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXltlSjtr0%253D&md5=99cb1ae155f7946684733150de62747a</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1038%2F12013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F12013%26sid%3Dliteratum%253Aachs%26aulast%3DCarrano%26aufirst%3DA.%2BC.%26aulast%3DEytan%26aufirst%3DE.%26aulast%3DHershko%26aufirst%3DA.%26aulast%3DPagano%26aufirst%3DM.%26atitle%3DSKP2%2520is%2520required%2520for%2520ubiquitin-mediated%2520degradation%2520of%2520the%2520CDK%2520inhibitor%2520p27%26jtitle%3DNat.%2520Cell%2520Biol.%26date%3D1999%26volume%3D1%26spage%3D193%26epage%3D199%26doi%3D10.1038%2F12013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Morimoto, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nishida, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Honda, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yasuda, H.</span></span> <span> </span><span class="NLM_article-title">Modification of cullin-1 by ubiquitin-like protein nedd8 enhances the activity of SCF(skp2) toward p27(kip1)</span>. <i>Biochem. Biophys. Res. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>270</i></span>,  <span class="NLM_fpage">1093</span>– <span class="NLM_lpage">1096</span>, <span class="refDoi"> DOI: 10.1006/bbrc.2000.2576</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=10.1006%2Fbbrc.2000.2576" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=10772955" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=1%3ACAS%3A528%3ADC%252BD3cXisFersr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=270&publication_year=2000&pages=1093-1096&author=M.+Morimotoauthor=T.+Nishidaauthor=R.+Hondaauthor=H.+Yasuda&title=Modification+of+cullin-1+by+ubiquitin-like+protein+nedd8+enhances+the+activity+of+SCF%28skp2%29+toward+p27%28kip1%29&doi=10.1006%2Fbbrc.2000.2576"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Modification of Cullin-1 by Ubiquitin-like Protein Nedd8 Enhances the Activity of SCFskp2 toward p27kip1</span></div><div class="casAuthors">Morimoto, Mitsuru; Nishida, Tamotsu; Honda, Reiko; Yasuda, Hideyo</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical and Biophysical Research Communications</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">270</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1093-1096</span>CODEN:
                <span class="NLM_cas:coden">BBRCA9</span>;
        ISSN:<span class="NLM_cas:issn">0006-291X</span>.
    
            (<span class="NLM_cas:orgname">Academic Press</span>)
        </div><div class="casAbstract">The periodic expression of cell cycle proteins is important for the regulation of cell cycle progression.  The amt. of CDK inhibitor, p27kip1, one such protein, seems to be regulated by the ubiquitin-proteasome system.  The ubiquitin ligase (E3) toward p27kip1 is thought to be SCFskp2.  The activity of SCFskp2 was increased by the addn. of Roc1 protein to the complex.  Furthermore, the ubiquitination of p27kip1 seemed to be dependent on the phosphorylation of T187 of p27kip1 because the mutant T187A was not ubiquitinated at all in an in vitro ubiquitination system.  Cullin-1, a component of SCF, is modified by ubiquitin-like protein Nedd8.  The modification site of cullin-1 was shown to be K696 because the K696R mutant was not modified.  When the effect of the Nedd8 modification on the SCFskp2 activity toward p27kip1 was investigated, the activity was markedly decreased by using the Nedd8-unmodified mutant cullin-1 (K696R), indicating that the modification may play an important role on the SCFskp2 activity toward p27kip1.  (c) 2000 Academic Press.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr7d8u3xfF9H7Vg90H21EOLACvtfcHk0litJdEvygdGhA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXisFersr0%253D&md5=03979249fae36709bb8a13370e772b50</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1006%2Fbbrc.2000.2576&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Fbbrc.2000.2576%26sid%3Dliteratum%253Aachs%26aulast%3DMorimoto%26aufirst%3DM.%26aulast%3DNishida%26aufirst%3DT.%26aulast%3DHonda%26aufirst%3DR.%26aulast%3DYasuda%26aufirst%3DH.%26atitle%3DModification%2520of%2520cullin-1%2520by%2520ubiquitin-like%2520protein%2520nedd8%2520enhances%2520the%2520activity%2520of%2520SCF%2528skp2%2529%2520toward%2520p27%2528kip1%2529%26jtitle%3DBiochem.%2520Biophys.%2520Res.%2520Commun.%26date%3D2000%26volume%3D270%26spage%3D1093%26epage%3D1096%26doi%3D10.1006%2Fbbrc.2000.2576" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yan, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Legerski, R. J.</span></span> <span> </span><span class="NLM_article-title">Artemis interacts with the Cul4A-DDB1DDB2 ubiquitin E3 ligase and regulates degradation of the CDK inhibitor p27</span>. <i>Cell Cycle</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">4098</span>– <span class="NLM_lpage">4109</span>, <span class="refDoi"> DOI: 10.4161/cc.10.23.18227</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=10.4161%2Fcc.10.23.18227" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=22134138" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsV2mt7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2011&pages=4098-4109&author=Y.+Yanauthor=X.+Zhangauthor=R.+J.+Legerski&title=Artemis+interacts+with+the+Cul4A-DDB1DDB2+ubiquitin+E3+ligase+and+regulates+degradation+of+the+CDK+inhibitor+p27&doi=10.4161%2Fcc.10.23.18227"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Artemis interacts with the Cul4A-DDB1DDB2 ubiquitin E3 ligase and regulates degradation of the CDK inhibitor p27</span></div><div class="casAuthors">Yan, Yiyi; Zhang, Xiaoshan; Legerski, Randy J.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Cycle</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">4098-4109</span>CODEN:
                <span class="NLM_cas:coden">CCEYAS</span>;
        ISSN:<span class="NLM_cas:issn">1538-4101</span>.
    
            (<span class="NLM_cas:orgname">Landes Bioscience</span>)
        </div><div class="casAbstract">Artemis, a member of the SNM1 gene family, is a multifunctional phospho-protein that has been shown to have important roles in V(D)J recombination, DNA double-strand break repair and stress-induced cell cycle checkpoint regulation.  We show here that Artemis interacts with the Cul4A-DDB1 E3 ubiquitin ligase via a direct interaction with the substrate-specificity receptor DDB2.  Furthermore, Artemis also interacts with the CDK inhibitor and tumor suppressor p27, a substrate of the Cul4A-DDB1 ligase, and both DDB2 and Artemis are required for the degrdn. of p27 mediated by this complex.  We also show that the regulation of p27 by Artemis and DDB2 is important for cell cycle progression in normally proliferating cells and in response to serum deprivation.  These findings thus define a function for Artemis as an effector of Cullin-based E3 ligase-mediated ubiquitylation, demonstrate a novel pathway for the regulation of p27 and show that Cul4A-DDB1DDB2-Artemis regulates G1-phase cell cycle progression in mammalian cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGolqogyvDm0e7Vg90H21EOLACvtfcHk0litJdEvygdGhA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsV2mt7Y%253D&md5=f268d7aaaff2e54d3e17b76be09c81f3</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.4161%2Fcc.10.23.18227&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4161%252Fcc.10.23.18227%26sid%3Dliteratum%253Aachs%26aulast%3DYan%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DLegerski%26aufirst%3DR.%2BJ.%26atitle%3DArtemis%2520interacts%2520with%2520the%2520Cul4A-DDB1DDB2%2520ubiquitin%2520E3%2520ligase%2520and%2520regulates%2520degradation%2520of%2520the%2520CDK%2520inhibitor%2520p27%26jtitle%3DCell%2520Cycle%26date%3D2011%26volume%3D10%26spage%3D4098%26epage%3D4109%26doi%3D10.4161%2Fcc.10.23.18227" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Abbas, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sivaprasad, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terai, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amador, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pagano, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dutta, A.</span></span> <span> </span><span class="NLM_article-title">PCNA-dependent regulation of p21 ubiquitylation and degradation via the CRL4Cdt2 ubiquitin ligase complex</span>. <i>Genes Dev.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">2496</span>– <span class="NLM_lpage">2506</span>, <span class="refDoi"> DOI: 10.1101/gad.1676108</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=10.1101%2Fgad.1676108" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2008&pages=2496-2506&author=T.+Abbasauthor=U.+Sivaprasadauthor=K.+Teraiauthor=V.+Amadorauthor=M.+Paganoauthor=A.+Dutta&title=PCNA-dependent+regulation+of+p21+ubiquitylation+and+degradation+via+the+CRL4Cdt2+ubiquitin+ligase+complex&doi=10.1101%2Fgad.1676108"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1101%2Fgad.1676108&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fgad.1676108%26sid%3Dliteratum%253Aachs%26aulast%3DAbbas%26aufirst%3DT.%26aulast%3DSivaprasad%26aufirst%3DU.%26aulast%3DTerai%26aufirst%3DK.%26aulast%3DAmador%26aufirst%3DV.%26aulast%3DPagano%26aufirst%3DM.%26aulast%3DDutta%26aufirst%3DA.%26atitle%3DPCNA-dependent%2520regulation%2520of%2520p21%2520ubiquitylation%2520and%2520degradation%2520via%2520the%2520CRL4Cdt2%2520ubiquitin%2520ligase%2520complex%26jtitle%3DGenes%2520Dev.%26date%3D2008%26volume%3D22%26spage%3D2496%26epage%3D2506%26doi%3D10.1101%2Fgad.1676108" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cullinan, S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordan, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harper, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diehl, J. A.</span></span> <span> </span><span class="NLM_article-title">The Keap1-BTB protein is an adaptor that bridges Nrf2 to a Cul3-based E3 ligase: oxidative stress sensing by a Cul3-Keap1 ligase</span>. <i>Mol. Cell. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">8477</span>– <span class="NLM_lpage">8486</span>, <span class="refDoi"> DOI: 10.1128/MCB.24.19.8477-8486.2004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=10.1128%2FMCB.24.19.8477-8486.2004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=15367669" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=1%3ACAS%3A528%3ADC%252BD2cXnvVakt7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2004&pages=8477-8486&author=S.+B.+Cullinanauthor=J.+D.+Gordanauthor=J.+Jinauthor=J.+W.+Harperauthor=J.+A.+Diehl&title=The+Keap1-BTB+protein+is+an+adaptor+that+bridges+Nrf2+to+a+Cul3-based+E3+ligase%3A+oxidative+stress+sensing+by+a+Cul3-Keap1+ligase&doi=10.1128%2FMCB.24.19.8477-8486.2004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">The Keap1-BTB protein is an adaptor that bridges Nrf2 to a Cul3-based E3 ligase: Oxidative stress sensing by a Cul3-Keap1 ligase</span></div><div class="casAuthors">Cullinan, Sara B.; Gordan, John D.; Jin, Jianping; Harper, J. Wade; Diehl, J. Alan</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">8477-8486</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">0270-7306</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">The Nrf2 transcription factor promotes survival following cellular insults that trigger oxidative damage.  Nrf2 activity is opposed by the BTB/POZ domain protein Keap1.  Keap1 is proposed to regulate Nrf2 activity strictly through its capacity to inhibit Nrf2 nuclear import.  Recent work suggests that inhibition of Nrf2 may also depend upon ubiquitin-mediated proteolysis.  To address the contribution of Keap1-dependent sequestration vs. Nrf2 proteolysis, we identified the E3 ligase that regulates Nrf2 ubiquitination.  We demonstrate that Keap1 is not solely a cytosolic anchor; rather, Keap1 is an adaptor that bridges Nrf2 to Cul3.  We demonstrate that Cul3-Keap1 complexes regulate Nrf2 polyubiquitination both in vitro and in vivo.  Inhibition of either Keap1 or Cul3 increases Nrf2 nuclear accumulation, leading to promiscuous activation of Nrf2-dependent gene expression.  Our data demonstrate that Keap1 restrains Nrf2 activity via its capacity to target Nrf2 to a cytoplasmic Cul3-based E3 ligase and suggest a model in which Keap1 coordinately regulates both Nrf2 accumulation and access to target genes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGokzxnJWH6ecbVg90H21EOLACvtfcHk0li_7Oa-Y1ymYw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXnvVakt7k%253D&md5=4a51087f90e5eb8f1f1cace9278941ad</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1128%2FMCB.24.19.8477-8486.2004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.24.19.8477-8486.2004%26sid%3Dliteratum%253Aachs%26aulast%3DCullinan%26aufirst%3DS.%2BB.%26aulast%3DGordan%26aufirst%3DJ.%2BD.%26aulast%3DJin%26aufirst%3DJ.%26aulast%3DHarper%26aufirst%3DJ.%2BW.%26aulast%3DDiehl%26aufirst%3DJ.%2BA.%26atitle%3DThe%2520Keap1-BTB%2520protein%2520is%2520an%2520adaptor%2520that%2520bridges%2520Nrf2%2520to%2520a%2520Cul3-based%2520E3%2520ligase%253A%2520oxidative%2520stress%2520sensing%2520by%2520a%2520Cul3-Keap1%2520ligase%26jtitle%3DMol.%2520Cell.%2520Biol.%26date%3D2004%26volume%3D24%26spage%3D8477%26epage%3D8486%26doi%3D10.1128%2FMCB.24.19.8477-8486.2004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref55"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref55'); return false;" data-citation="" class="refNumLink">55</a></strong><div class="NLM_citation" id="cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, Y.</span></span> <span> </span><span class="NLM_article-title">Cullin-RING ligases as attractive anti-cancer targets</span>. <i>Curr. Pharm. Des.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">3215</span>– <span class="NLM_lpage">3225</span>, <span class="refDoi"> DOI: 10.2174/13816128113199990300</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=10.2174%2F13816128113199990300" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=23151137" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=1%3ACAS%3A528%3ADC%252BC3sXotFagtL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2013&pages=3215-3225&author=Y.+Zhaoauthor=Y.+Sun&title=Cullin-RING+ligases+as+attractive+anti-cancer+targets&doi=10.2174%2F13816128113199990300"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Cullin-RING ligases as attractive anti-cancer targets</span></div><div class="casAuthors">Zhao, Yongchao; Sun, Yi</div><div class="citationInfo"><span class="NLM_cas:title">Current Pharmaceutical Design</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">3215-3225</span>CODEN:
                <span class="NLM_cas:coden">CPDEFP</span>;
        ISSN:<span class="NLM_cas:issn">1381-6128</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  The ubiquitin-proteasome system (UPS) promotes the timely degrdn. of short-lived proteins with key regulatory roles in a vast array of biol. processes, such as cell cycle progression, oncogenesis and genome integrity.  Thus, abnormal regulation of UPS disrupts the protein homeostasis and causes many human diseases, particularly cancer.  Indeed, the FDA approval of bortezomib, the first class of general proteasome inhibitor, for the treatment of multiple myeloma, demonstrated that the UPS can be an attractive anti-cancer target.  However, normal cell toxicity assocd. with bortezomib, resulting from global inhibition of protein degrdn., promotes the focus of drug discovery efforts on targeting enzymes upstream of the proteasome for better specificity.  E3 ubiquitin ligases, particularly those known to be activated in human cancer, become an attractive choice.  Cullin-RING Ligases (CRLs) with multiple components are the largest family of E3 ubiquitin ligases and are responsible for ubiquitination of ∼20% of cellular proteins degraded through UPS.  Activity of CRLs is dynamically regulated and requires the RING component and cullin neddylation.  In this review, we will introduce the UPS and CRL E3s and discuss the biol. processes regulated by each of eight CRLs through substrate degrdn.  We will further discuss how cullin neddylation controls CRL activity, and how CRLs are being validated as the attractive cancer targets by abrogating the RING component through genetic means and by inhibiting cullin neddylation via MLN4924, a small mol. indirect inhibitor of CRLs, currently in several Phase I clin. trials.  Finally, we will discuss current efforts and future perspectives on the development of addnl. inhibitors of CRLs by targeting E2 and/or E3 of cullin neddylation and CRL-mediated ubiquitination as potential anti-cancer agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp-K-w4MaFbdLVg90H21EOLACvtfcHk0li_7Oa-Y1ymYw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXotFagtL8%253D&md5=4498de520df1c0a24796bdbfd5b3ce17</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.2174%2F13816128113199990300&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F13816128113199990300%26sid%3Dliteratum%253Aachs%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DSun%26aufirst%3DY.%26atitle%3DCullin-RING%2520ligases%2520as%2520attractive%2520anti-cancer%2520targets%26jtitle%3DCurr.%2520Pharm.%2520Des.%26date%3D2013%26volume%3D19%26spage%3D3215%26epage%3D3225%26doi%3D10.2174%2F13816128113199990300" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref56"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref56'); return false;" data-citation="" class="refNumLink">56</a></strong><div class="NLM_citation" id="cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Niture, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaspar, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jaiswal, A. K.</span></span> <span> </span><span class="NLM_article-title">Nrf2 signaling and cell survival</span>. <i>Toxicol. Appl. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>244</i></span>,  <span class="NLM_fpage">37</span>– <span class="NLM_lpage">42</span>, <span class="refDoi"> DOI: 10.1016/j.taap.2009.06.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=10.1016%2Fj.taap.2009.06.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=19538984" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=1%3ACAS%3A528%3ADC%252BC3cXjt1Gkt7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=244&publication_year=2010&pages=37-42&author=S.+K.+Nitureauthor=J.+W.+Kasparauthor=J.+Shenauthor=A.+K.+Jaiswal&title=Nrf2+signaling+and+cell+survival&doi=10.1016%2Fj.taap.2009.06.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">Nrf2 signaling and cell survival</span></div><div class="casAuthors">Niture, Suryakant K.; Kaspar, James W.; Shen, Jun; Jaiswal, Anil K.</div><div class="citationInfo"><span class="NLM_cas:title">Toxicology and Applied Pharmacology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">244</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">37-42</span>CODEN:
                <span class="NLM_cas:coden">TXAPA9</span>;
        ISSN:<span class="NLM_cas:issn">0041-008X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Nrf2:INrf2 (Keap1) acts as a sensor for oxidative/electrophilic stress.  INrf2 serves as an adaptor to link Nrf2 to the ubiquitin ligase Cul3-Rbx1 complex that ubiquitinate and degrade Nrf2.  Under basal conditions, cytosolic INrf2/Cul3-Rbx1 is constantly degrading Nrf2.  When a cell encounters stress, Nrf2 dissocs. from INrf2 and translocates into the nucleus.  Oxidative/electrophilic stress induced modification of INrf2-Cys-151 and/or protein kinase C (PKC)-mediated phosphorylation of Nrf2-Ser-40 controls Nrf2 release from INrf2 followed by stabilization and nuclear translocation of Nrf2.  Nrf2 binds to the antioxidant response element (ARE) and activates a myriad of genes that protect cells against oxidative/electrophilic stress and neoplasia.  A delayed response of oxidative/electrophilic stress activates protein kinase GSK-3β that phosphorylates Fyn kinase at unknown Thr residue(s).  Phosphorylated Fyn translocates to the nucleus and phosphorylates Nrf2-Tyr-568 that leads to nuclear export and degrdn. of Nrf2.  Prothymosin-α mediated nuclear translocation of INrf2 also degrades nuclear Nrf2.  The degrdn. of Nrf2 both in cytosol and nuclear compartments rapidly brings down its levels to normal resulting in suppression of Nrf2 downstream gene expression.  An auto-regulatory loop between Nrf2 and INrf2 controls their cellular abundance.  Nrf2 regulates INrf2 by controlling its transcription, and INrf2 controls Nrf2 by degrading it.  In conclusion, switching on and off of Nrf2 combined with promoting an auto-regulatory loop between them regulates activation/deactivation of defensive genes leading to protection of cells against adverse effects of oxidative and electrophilic stress and promote cell survival.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoIcQsUwXM-PLVg90H21EOLACvtfcHk0li_7Oa-Y1ymYw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXjt1Gkt7Y%253D&md5=c25e3a28939528f3bff36a27be805695</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1016%2Fj.taap.2009.06.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.taap.2009.06.009%26sid%3Dliteratum%253Aachs%26aulast%3DNiture%26aufirst%3DS.%2BK.%26aulast%3DKaspar%26aufirst%3DJ.%2BW.%26aulast%3DShen%26aufirst%3DJ.%26aulast%3DJaiswal%26aufirst%3DA.%2BK.%26atitle%3DNrf2%2520signaling%2520and%2520cell%2520survival%26jtitle%3DToxicol.%2520Appl.%2520Pharmacol.%26date%3D2010%26volume%3D244%26spage%3D37%26epage%3D42%26doi%3D10.1016%2Fj.taap.2009.06.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref57"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref57'); return false;" data-citation="" class="refNumLink">57</a></strong><div class="NLM_citation" id="cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, L.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, Y.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, D.-D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miao, E.-F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qiao, X.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, L.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H.-M.</span></span> <span> </span><span class="NLM_article-title">Design, synthesis and biological evaluation of [1,2,4]triazolo[1,5-<i>a</i>]pyrimidines as potent lysine specific demethylase 1 (LSD1/KDM1A) inhibitors</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>125</i></span>,  <span class="NLM_fpage">940</span>– <span class="NLM_lpage">951</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2016.10.021</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=10.1016%2Fj.ejmech.2016.10.021" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=27769034" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=1%3ACAS%3A528%3ADC%252BC28XhslWju7rJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=125&publication_year=2017&pages=940-951&author=S.+Wangauthor=L.-J.+Zhaoauthor=Y.-C.+Zhengauthor=D.-D.+Shenauthor=E.-F.+Miaoauthor=X.-P.+Qiaoauthor=L.-J.+Zhaoauthor=Y.+Liuauthor=R.+Huangauthor=B.+Yuauthor=H.-M.+Liu&title=Design%2C+synthesis+and+biological+evaluation+of+%5B1%2C2%2C4%5Dtriazolo%5B1%2C5-a%5Dpyrimidines+as+potent+lysine+specific+demethylase+1+%28LSD1%2FKDM1A%29+inhibitors&doi=10.1016%2Fj.ejmech.2016.10.021"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">Design, synthesis and biological evaluation of [1,2,4]triazolo[1,5-a]pyrimidines as potent lysine specific demethylase 1 (LSD1/KDM1A) inhibitors</span></div><div class="casAuthors">Wang, Shuai; Zhao, Li-Jie; Zheng, Yi-Chao; Shen, Dan-Dan; Miao, Er-Fei; Qiao, Xue-Peng; Zhao, Li-Juan; Liu, Ying; Huang, Ruilei; Yu, Bin; Liu, Hong-Min</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">125</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">940-951</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">A new series of [1,2,4]triazolo[1,5-a]pyrimidine-based LSD1 inhibitors was designed, synthesized, and further evaluated for cytotoxicity against MGC-803, EC109, A549 and PC-9 cells as well as the ability of inhibiting LSD1.  Some of these compds. showed potent inhibition toward LSD1 and selectively inhibited growth of A549 and PC-9 cells.  Compd. I potently inhibited growth of PC-9 cells (IC50 = 0.59 μM), about 4-fold more potent than 5-FU.  Further SAR studies led to the identification of compds. I and II, which had good growth inhibition against all the tested cancer cell lines and were much more potent than 5-FU and GSK2879552.  Besides, compds. III-V inhibited LSD1 potently (IC50 = 0.154, 1.19 and 0.557 μM, resp.).  Docking studies revealed that compd. III formed arene-H interactions with Val333 and hydrogen bonds with surrounding Ala331, Met332, and Ala539 residues.  Compd. III significantly inhibited migration of A549 and PC-9 cells in a concn.-dependent manner, but had different effect on the expression of E-cadherin and N-cadherin.  The [1,2,4]triazolo[1,5-a]pyrimidine scaffold may serve as a starting point for developing potent LSD1 inhibitors for cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr0jqW-eLBQObVg90H21EOLACvtfcHk0lj3UHl0fOd9vA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhslWju7rJ&md5=d022678613486890faacfef9c163492b</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2016.10.021&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2016.10.021%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DS.%26aulast%3DZhao%26aufirst%3DL.-J.%26aulast%3DZheng%26aufirst%3DY.-C.%26aulast%3DShen%26aufirst%3DD.-D.%26aulast%3DMiao%26aufirst%3DE.-F.%26aulast%3DQiao%26aufirst%3DX.-P.%26aulast%3DZhao%26aufirst%3DL.-J.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DHuang%26aufirst%3DR.%26aulast%3DYu%26aufirst%3DB.%26aulast%3DLiu%26aufirst%3DH.-M.%26atitle%3DDesign%252C%2520synthesis%2520and%2520biological%2520evaluation%2520of%2520%255B1%252C2%252C4%255Dtriazolo%255B1%252C5-a%255Dpyrimidines%2520as%2520potent%2520lysine%2520specific%2520demethylase%25201%2520%2528LSD1%252FKDM1A%2529%2520inhibitors%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2017%26volume%3D125%26spage%3D940%26epage%3D951%26doi%3D10.1016%2Fj.ejmech.2016.10.021" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54'],'ref55':['cit55'],'ref56':['cit56'],'ref57':['cit57']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 32 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Ho Shin Kim, Jared T. Hammill, Daniel C. Scott, Yizhe Chen, Amy L. Rice, William Pistel, Bhuvanesh Singh, Brenda A. Schulman, <span class="NLM_string-name hlFld-ContribAuthor">R. Kiplin Guy</span>. </span><span class="cited-content_cbyCitation_article-title">Improvement of Oral Bioavailability of Pyrazolo-Pyridone Inhibitors of the Interaction of DCN1/2 and UBE2M. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (9)
                                     , 5850-5862. <a href="https://doi.org/10.1021/acs.jmedchem.1c00035" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.1c00035</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.1c00035&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.1c00035%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DImprovement%252Bof%252BOral%252BBioavailability%252Bof%252BPyrazolo-Pyridone%252BInhibitors%252Bof%252Bthe%252BInteraction%252Bof%252BDCN1%25252F2%252Band%252BUBE2M%26aulast%3DKim%26aufirst%3DHo%2BShin%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D08012021%26date%3D04052021%26volume%3D64%26issue%3D9%26spage%3D5850%26epage%3D5862" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Shuo Yuan, Jingya Zhang, Danqing Zhang, Donghui Wei, Jiahui Zuo, Jian Song, Bin Yu, <span class="NLM_string-name hlFld-ContribAuthor">Hong-Min Liu</span>. </span><span class="cited-content_cbyCitation_article-title">Cu(OTf)2-Catalyzed Intramolecular Radical Cascade Reactions for the Diversity-Oriented Synthesis of Quinoline-Annulated Polyheterocyclic Frameworks. </span><span class="cited-content_cbyCitation_journal-name">Organic Letters</span><span> <strong>2021,</strong> <em>23 </em>
                                    (4)
                                     , 1445-1450. <a href="https://doi.org/10.1021/acs.orglett.1c00129" title="DOI URL">https://doi.org/10.1021/acs.orglett.1c00129</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.orglett.1c00129&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.orglett.1c00129%26sid%3Dliteratum%253Aachs%26jtitle%3DOrganic%2520Letters%26atitle%3DCu%252528OTf%2525292-Catalyzed%252BIntramolecular%252BRadical%252BCascade%252BReactions%252Bfor%252Bthe%252BDiversity-Oriented%252BSynthesis%252Bof%252BQuinoline-Annulated%252BPolyheterocyclic%252BFrameworks%26aulast%3DYuan%26aufirst%3DShuo%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D13012021%26date%3D09022021%26volume%3D23%26issue%3D4%26spage%3D1445%26epage%3D1450" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Shuai Wang, Sai-Qi Wang, Qiu-Xu Teng, Linlin Yang, Zi-Ning Lei, Xiao-Han Yuan, Jun-Feng Huo, Xiao-Bing Chen, Mengru Wang, Bin Yu, Zhe-Sheng Chen, <span class="NLM_string-name hlFld-ContribAuthor">Hong-Min Liu</span>. </span><span class="cited-content_cbyCitation_article-title">Structure-Based Design, Synthesis, and Biological Evaluation of New Triazolo[1,5-a]Pyrimidine Derivatives as Highly Potent and Orally Active ABCB1 Modulators. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (24)
                                     , 15979-15996. <a href="https://doi.org/10.1021/acs.jmedchem.0c01741" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c01741</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c01741&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c01741%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DStructure-Based%252BDesign%25252C%252BSynthesis%25252C%252Band%252BBiological%252BEvaluation%252Bof%252BNew%252BTriazolo%25255B1%25252C5-a%25255DPyrimidine%252BDerivatives%252Bas%252BHighly%252BPotent%252Band%252BOrally%252BActive%252BABCB1%252BModulators%26aulast%3DWang%26aufirst%3DShuai%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D05102020%26date%3D05122020%26volume%3D63%26issue%3D24%26spage%3D15979%26epage%3D15996" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Ho Shin Kim, Jared T. Hammill, Daniel C. Scott, Yizhe Chen, Jaeki Min, Jonah Rector, Bhuvanesh Singh, Brenda A. Schulman, <span class="NLM_string-name hlFld-ContribAuthor">R. Kiplin Guy</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of Novel Pyrazolo-pyridone DCN1 Inhibitors Controlling Cullin Neddylation. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2019,</strong> <em>62 </em>
                                    (18)
                                     , 8429-8442. <a href="https://doi.org/10.1021/acs.jmedchem.9b00410" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b00410</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b00410&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.9b00410%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252BNovel%252BPyrazolo-pyridone%252BDCN1%252BInhibitors%252BControlling%252BCullin%252BNeddylation%26aulast%3DKim%26aufirst%3DHo%2BShin%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D08032019%26date%3D13092019%26date%3D29082019%26volume%3D62%26issue%3D18%26spage%3D8429%26epage%3D8442" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yi-Chao  Zheng</span>, <span class="hlFld-ContribAuthor ">Yan-Jia  Guo</span>, <span class="hlFld-ContribAuthor ">Bo  Wang</span>, <span class="hlFld-ContribAuthor ">Chong  Wang</span>, <span class="hlFld-ContribAuthor ">M. A. A.  Mamun</span>, <span class="hlFld-ContribAuthor ">Ya  Gao</span>, <span class="hlFld-ContribAuthor ">Hong-Min  Liu</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting neddylation E2s: a novel therapeutic strategy in cancer. </span><span class="cited-content_cbyCitation_journal-name">Journal of Hematology & Oncology</span><span> <strong>2021,</strong> <em>14 </em>
                                    (1)
                                     <a href="https://doi.org/10.1186/s13045-021-01070-w" title="DOI URL">https://doi.org/10.1186/s13045-021-01070-w</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1186/s13045-021-01070-w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1186%2Fs13045-021-01070-w%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Hematology%2520%2526%2520Oncology%26atitle%3DTargeting%252Bneddylation%252BE2s%25253A%252Ba%252Bnovel%252Btherapeutic%252Bstrategy%252Bin%252Bcancer%26aulast%3DZheng%26aufirst%3DYi-Chao%26date%3D2021%26date%3D2021%26volume%3D14%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Haibin  Zhou</span>, <span class="hlFld-ContribAuthor ">Jianfeng  Lu</span>, <span class="hlFld-ContribAuthor ">Krishnapriya  Chinnaswamy</span>, <span class="hlFld-ContribAuthor ">Jeanne A.  Stuckey</span>, <span class="hlFld-ContribAuthor ">Liu  Liu</span>, <span class="hlFld-ContribAuthor ">Donna  McEachern</span>, <span class="hlFld-ContribAuthor ">Chao-Yie  Yang</span>, <span class="hlFld-ContribAuthor ">Denzil  Bernard</span>, <span class="hlFld-ContribAuthor ">Hong  Shen</span>, <span class="hlFld-ContribAuthor ">Liangyou  Rui</span>, <span class="hlFld-ContribAuthor ">Yi  Sun</span>, <span class="hlFld-ContribAuthor ">Shaomeng  Wang</span>. </span><span class="cited-content_cbyCitation_article-title">Selective inhibition of cullin 3 neddylation through covalent targeting DCN1 protects mice from acetaminophen-induced liver toxicity. </span><span class="cited-content_cbyCitation_journal-name">Nature Communications</span><span> <strong>2021,</strong> <em>12 </em>
                                    (1)
                                     <a href="https://doi.org/10.1038/s41467-021-22924-4" title="DOI URL">https://doi.org/10.1038/s41467-021-22924-4</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41467-021-22924-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41467-021-22924-4%26sid%3Dliteratum%253Aachs%26jtitle%3DNature%2520Communications%26atitle%3DSelective%252Binhibition%252Bof%252Bcullin%252B3%252Bneddylation%252Bthrough%252Bcovalent%252Btargeting%252BDCN1%252Bprotects%252Bmice%252Bfrom%252Bacetaminophen-induced%252Bliver%252Btoxicity%26aulast%3DZhou%26aufirst%3DHaibin%26date%3D2021%26date%3D2021%26volume%3D12%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Qianqian  Song</span>, <span class="hlFld-ContribAuthor ">Siqi  Feng</span>, <span class="hlFld-ContribAuthor ">Wenjun  Peng</span>, <span class="hlFld-ContribAuthor ">Anqi  Li</span>, <span class="hlFld-ContribAuthor ">Ting  Ma</span>, <span class="hlFld-ContribAuthor ">Bin  Yu</span>, <span class="hlFld-ContribAuthor ">Hong-Min  Liu</span>. </span><span class="cited-content_cbyCitation_article-title">Cullin-RING Ligases as Promising Targets for Gastric Carcinoma Treatment. </span><span class="cited-content_cbyCitation_journal-name">Pharmacological Research</span><span> <strong>2021,</strong> <em>170 </em>, 105493. <a href="https://doi.org/10.1016/j.phrs.2021.105493" title="DOI URL">https://doi.org/10.1016/j.phrs.2021.105493</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.phrs.2021.105493&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.phrs.2021.105493%26sid%3Dliteratum%253Aachs%26jtitle%3DPharmacological%2520Research%26atitle%3DCullin-RING%252BLigases%252Bas%252BPromising%252BTargets%252Bfor%252BGastric%252BCarcinoma%252BTreatment%26aulast%3DSong%26aufirst%3DQianqian%26date%3D2021%26volume%3D170%26spage%3D105493" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xin-Hui  Zhang</span>, <span class="hlFld-ContribAuthor ">Hui-Qin  Kang</span>, <span class="hlFld-ContribAuthor ">Yuan-Yuan  Tao</span>, <span class="hlFld-ContribAuthor ">Yi-Han  Li</span>, <span class="hlFld-ContribAuthor ">Jun-Ru  Zhao</span>, <span class="hlFld-ContribAuthor ">Ya-Gao</span>, <span class="hlFld-ContribAuthor ">Li-Ying  Ma</span>, <span class="hlFld-ContribAuthor ">Hong-Min  Liu</span>. </span><span class="cited-content_cbyCitation_article-title">Identification of novel 1,3-diaryl-1,2,4-triazole-capped histone deacetylase 6 inhibitors with potential anti-gastric cancer activity. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>218 </em>, 113392. <a href="https://doi.org/10.1016/j.ejmech.2021.113392" title="DOI URL">https://doi.org/10.1016/j.ejmech.2021.113392</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2021.113392&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2021.113392%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DIdentification%252Bof%252Bnovel%252B1%25252C3-diaryl-1%25252C2%25252C4-triazole-capped%252Bhistone%252Bdeacetylase%252B6%252Binhibitors%252Bwith%252Bpotential%252Banti-gastric%252Bcancer%252Bactivity%26aulast%3DZhang%26aufirst%3DXin-Hui%26date%3D2021%26volume%3D218%26spage%3D113392" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Wenjuan  Zhou</span>, <span class="hlFld-ContribAuthor ">Chenhao  Xu</span>, <span class="hlFld-ContribAuthor ">Guanjun  Dong</span>, <span class="hlFld-ContribAuthor ">Hui  Qiao</span>, <span class="hlFld-ContribAuthor ">Jing  Yang</span>, <span class="hlFld-ContribAuthor ">Hongmin  Liu</span>, <span class="hlFld-ContribAuthor ">Lina  Ding</span>, <span class="hlFld-ContribAuthor ">Kai  Sun</span>, <span class="hlFld-ContribAuthor ">Wen  Zhao</span>. </span><span class="cited-content_cbyCitation_article-title">Development of phenyltriazole thiol-based derivatives as highly potent inhibitors of DCN1-UBC12 interaction. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>217 </em>, 113326. <a href="https://doi.org/10.1016/j.ejmech.2021.113326" title="DOI URL">https://doi.org/10.1016/j.ejmech.2021.113326</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2021.113326&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2021.113326%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDevelopment%252Bof%252Bphenyltriazole%252Bthiol-based%252Bderivatives%252Bas%252Bhighly%252Bpotent%252Binhibitors%252Bof%252BDCN1-UBC12%252Binteraction%26aulast%3DZhou%26aufirst%3DWenjuan%26date%3D2021%26volume%3D217%26spage%3D113326" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Shuo  Yuan</span>, <span class="hlFld-ContribAuthor ">Si-Qi  Feng</span>, <span class="hlFld-ContribAuthor ">An-Qi  Li</span>, <span class="hlFld-ContribAuthor ">Jia-Hui  Zuo</span>, <span class="hlFld-ContribAuthor ">Dan-Qing  Zhang</span>, <span class="hlFld-ContribAuthor ">Yu-Jie  Xing</span>, <span class="hlFld-ContribAuthor ">Zhiyu  Xie</span>, <span class="hlFld-ContribAuthor ">Bin  Yu</span>, <span class="hlFld-ContribAuthor ">Hong-Min  Liu</span>. </span><span class="cited-content_cbyCitation_article-title">Design and synthesis of new indole containing biaryl derivatives as potent antiproliferative agents. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic Chemistry</span><span> <strong>2021,</strong> <em>110 </em>, 104821. <a href="https://doi.org/10.1016/j.bioorg.2021.104821" title="DOI URL">https://doi.org/10.1016/j.bioorg.2021.104821</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bioorg.2021.104821&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bioorg.2021.104821%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520Chemistry%26atitle%3DDesign%252Band%252Bsynthesis%252Bof%252Bnew%252Bindole%252Bcontaining%252Bbiaryl%252Bderivatives%252Bas%252Bpotent%252Bantiproliferative%252Bagents%26aulast%3DYuan%26aufirst%3DShuo%26date%3D2021%26volume%3D110%26spage%3D104821" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Wenbin  Gai</span>, <span class="hlFld-ContribAuthor ">Zhiqiang  Peng</span>, <span class="hlFld-ContribAuthor ">Cui Hua  Liu</span>, <span class="hlFld-ContribAuthor ">Lingqiang  Zhang</span>, <span class="hlFld-ContribAuthor ">Hong  Jiang</span>. </span><span class="cited-content_cbyCitation_article-title">Advances in Cancer Treatment by Targeting the Neddylation Pathway. </span><span class="cited-content_cbyCitation_journal-name">Frontiers in Cell and Developmental Biology</span><span> <strong>2021,</strong> <em>9 </em><a href="https://doi.org/10.3389/fcell.2021.653882" title="DOI URL">https://doi.org/10.3389/fcell.2021.653882</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3389/fcell.2021.653882&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3389%2Ffcell.2021.653882%26sid%3Dliteratum%253Aachs%26jtitle%3DFrontiers%2520in%2520Cell%2520and%2520Developmental%2520Biology%26atitle%3DAdvances%252Bin%252BCancer%252BTreatment%252Bby%252BTargeting%252Bthe%252BNeddylation%252BPathway%26aulast%3DGai%26aufirst%3DWenbin%26date%3D2021%26date%3D2021%26volume%3D9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Shuai  Wang</span>, <span class="hlFld-ContribAuthor ">Xiao-Han  Yuan</span>, <span class="hlFld-ContribAuthor ">Sai-Qi  Wang</span>, <span class="hlFld-ContribAuthor ">Wen  Zhao</span>, <span class="hlFld-ContribAuthor ">Xiao-Bing  Chen</span>, <span class="hlFld-ContribAuthor ">Bin  Yu</span>. </span><span class="cited-content_cbyCitation_article-title">FDA-approved pyrimidine-fused bicyclic heterocycles for cancer therapy: Synthesis and clinical application. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>214 </em>, 113218. <a href="https://doi.org/10.1016/j.ejmech.2021.113218" title="DOI URL">https://doi.org/10.1016/j.ejmech.2021.113218</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2021.113218&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2021.113218%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DFDA-approved%252Bpyrimidine-fused%252Bbicyclic%252Bheterocycles%252Bfor%252Bcancer%252Btherapy%25253A%252BSynthesis%252Band%252Bclinical%252Bapplication%26aulast%3DWang%26aufirst%3DShuai%26date%3D2021%26volume%3D214%26spage%3D113218" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Konstantin V.  Savateev</span>, <span class="hlFld-ContribAuthor ">Pavel A.  Slepukhin</span>, <span class="hlFld-ContribAuthor ">Svetlana K.  Kotovskaya</span>, <span class="hlFld-ContribAuthor ">Valery N.  Charushin</span>, <span class="hlFld-ContribAuthor ">Vladimir L.  Rusinov</span>, <span class="hlFld-ContribAuthor ">Oleg N.  Chupakhin</span>. </span><span class="cited-content_cbyCitation_article-title">Atom-efficient synthesis of hybrid molecules combining fragments of triazolopyrimidines and 3-ethoxycarbonyl-1-ethyl-6-fluoroquinolin-4(1H)-one through 1,2,3-triazole linker. </span><span class="cited-content_cbyCitation_journal-name">Chemistry of Heterocyclic Compounds</span><span> <strong>2021,</strong> <em>57 </em>
                                    (2)
                                     , 143-153. <a href="https://doi.org/10.1007/s10593-021-02886-7" title="DOI URL">https://doi.org/10.1007/s10593-021-02886-7</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s10593-021-02886-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs10593-021-02886-7%26sid%3Dliteratum%253Aachs%26jtitle%3DChemistry%2520of%2520Heterocyclic%2520Compounds%26atitle%3DAtom-efficient%252Bsynthesis%252Bof%252Bhybrid%252Bmolecules%252Bcombining%252Bfragments%252Bof%252Btriazolopyrimidines%252Band%252B3-ethoxycarbonyl-1-ethyl-6-fluoroquinolin-4%2525281H%252529-one%252Bthrough%252B1%25252C2%25252C3-triazole%252Blinker%26aulast%3DSavateev%26aufirst%3DKonstantin%2BV.%26date%3D2021%26date%3D2021%26volume%3D57%26issue%3D2%26spage%3D143%26epage%3D153" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Dong-Jun  Fu</span>, <span class="hlFld-ContribAuthor ">Xin-Xin  Cui</span>, <span class="hlFld-ContribAuthor ">Ting  Zhu</span>, <span class="hlFld-ContribAuthor ">Yan-Bing  Zhang</span>, <span class="hlFld-ContribAuthor ">Yang-Yang  Hu</span>, <span class="hlFld-ContribAuthor ">Li-Rong  Zhang</span>, <span class="hlFld-ContribAuthor ">Sheng-Hui  Wang</span>, <span class="hlFld-ContribAuthor ">Sai-Yang  Zhang</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of novel indole derivatives that inhibit NEDDylation and MAPK pathways against gastric cancer MGC803 cells. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic Chemistry</span><span> <strong>2021,</strong> <em>107 </em>, 104634. <a href="https://doi.org/10.1016/j.bioorg.2021.104634" title="DOI URL">https://doi.org/10.1016/j.bioorg.2021.104634</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bioorg.2021.104634&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bioorg.2021.104634%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520Chemistry%26atitle%3DDiscovery%252Bof%252Bnovel%252Bindole%252Bderivatives%252Bthat%252Binhibit%252BNEDDylation%252Band%252BMAPK%252Bpathways%252Bagainst%252Bgastric%252Bcancer%252BMGC803%252Bcells%26aulast%3DFu%26aufirst%3DDong-Jun%26date%3D2021%26volume%3D107%26spage%3D104634" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jin-Ling  Huo</span>, <span class="hlFld-ContribAuthor ">Shuai  Wang</span>, <span class="hlFld-ContribAuthor ">Xiao-Han  Yuan</span>, <span class="hlFld-ContribAuthor ">Bin  Yu</span>, <span class="hlFld-ContribAuthor ">Wen  Zhao</span>, <span class="hlFld-ContribAuthor ">Hong-Min  Liu</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of [1,2,4]triazolo[1,5-a]pyrimidines derivatives as potential anticancer agents. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>211 </em>, 113108. <a href="https://doi.org/10.1016/j.ejmech.2020.113108" title="DOI URL">https://doi.org/10.1016/j.ejmech.2020.113108</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2020.113108&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2020.113108%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252B%25255B1%25252C2%25252C4%25255Dtriazolo%25255B1%25252C5-a%25255Dpyrimidines%252Bderivatives%252Bas%252Bpotential%252Banticancer%252Bagents%26aulast%3DHuo%26aufirst%3DJin-Ling%26date%3D2021%26volume%3D211%26spage%3D113108" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Shu-Yu  Wang</span>, <span class="hlFld-ContribAuthor ">Xu  Liu</span>, <span class="hlFld-ContribAuthor ">Yuan  Liu</span>, <span class="hlFld-ContribAuthor ">Hong-Yu  Zhang</span>, <span class="hlFld-ContribAuthor ">Yan-Bing  Zhang</span>, <span class="hlFld-ContribAuthor ">Chong  Liu</span>, <span class="hlFld-ContribAuthor ">Jian  Song</span>, <span class="hlFld-ContribAuthor ">Jin-Bo  Niu</span>, <span class="hlFld-ContribAuthor ">Sai-Yang  Zhang</span>. </span><span class="cited-content_cbyCitation_article-title">Review of NEDDylation inhibition activity detection methods. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry</span><span> <strong>2021,</strong> <em>29 </em>, 115875. <a href="https://doi.org/10.1016/j.bmc.2020.115875" title="DOI URL">https://doi.org/10.1016/j.bmc.2020.115875</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmc.2020.115875&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmc.2020.115875%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%26atitle%3DReview%252Bof%252BNEDDylation%252Binhibition%252Bactivity%252Bdetection%252Bmethods%26aulast%3DWang%26aufirst%3DShu-Yu%26date%3D2021%26volume%3D29%26spage%3D115875" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xin  Chen</span>, <span class="hlFld-ContribAuthor ">Xi  Yang</span>, <span class="hlFld-ContribAuthor ">Fei  Mao</span>, <span class="hlFld-ContribAuthor ">Jinlian  Wei</span>, <span class="hlFld-ContribAuthor ">Yixiang  Xu</span>, <span class="hlFld-ContribAuthor ">Baoli  Li</span>, <span class="hlFld-ContribAuthor ">Jin  Zhu</span>, <span class="hlFld-ContribAuthor ">Shuaishuai  Ni</span>, <span class="hlFld-ContribAuthor ">Lijun  Jia</span>, <span class="hlFld-ContribAuthor ">Jian  Li</span>. </span><span class="cited-content_cbyCitation_article-title">Development of novel benzimidazole-derived neddylation inhibitors for suppressing tumor growth in vitro and in vivo. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>210 </em>, 112964. <a href="https://doi.org/10.1016/j.ejmech.2020.112964" title="DOI URL">https://doi.org/10.1016/j.ejmech.2020.112964</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2020.112964&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2020.112964%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDevelopment%252Bof%252Bnovel%252Bbenzimidazole-derived%252Bneddylation%252Binhibitors%252Bfor%252Bsuppressing%252Btumor%252Bgrowth%252Bin%252Bvitro%252Band%252Bin%252Bvivo%26aulast%3DChen%26aufirst%3DXin%26date%3D2021%26volume%3D210%26spage%3D112964" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xiao-Jing  Shi</span>, <span class="hlFld-ContribAuthor ">Shuai  Wang</span>, <span class="hlFld-ContribAuthor ">Xiao-Jing  Li</span>, <span class="hlFld-ContribAuthor ">Xiao-Han  Yuan</span>, <span class="hlFld-ContribAuthor ">Li-Juan  Cao</span>, <span class="hlFld-ContribAuthor ">Bin  Yu</span>, <span class="hlFld-ContribAuthor ">Hong-Min  Liu</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of tofacitinib derivatives as orally active antitumor agents based on the scaffold hybridization strategy. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>203 </em>, 112601. <a href="https://doi.org/10.1016/j.ejmech.2020.112601" title="DOI URL">https://doi.org/10.1016/j.ejmech.2020.112601</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2020.112601&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2020.112601%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252Btofacitinib%252Bderivatives%252Bas%252Borally%252Bactive%252Bantitumor%252Bagents%252Bbased%252Bon%252Bthe%252Bscaffold%252Bhybridization%252Bstrategy%26aulast%3DShi%26aufirst%3DXiao-Jing%26date%3D2020%26volume%3D203%26spage%3D112601" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Shuai  Wang</span>, <span class="hlFld-ContribAuthor ">Xu-Bin  Ma</span>, <span class="hlFld-ContribAuthor ">Xiao-Han  Yuan</span>, <span class="hlFld-ContribAuthor ">Bin  Yu</span>, <span class="hlFld-ContribAuthor ">Yi-Chao  Xu</span>, <span class="hlFld-ContribAuthor ">Hong-Min  Liu</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of new [1,2,4] Triazolo[1,5-a]Pyrimidine derivatives that Kill gastric cancer cells via the mitochondria pathway. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>203 </em>, 112630. <a href="https://doi.org/10.1016/j.ejmech.2020.112630" title="DOI URL">https://doi.org/10.1016/j.ejmech.2020.112630</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2020.112630&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2020.112630%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252Bnew%252B%25255B1%25252C2%25252C4%25255D%252BTriazolo%25255B1%25252C5-a%25255DPyrimidine%252Bderivatives%252Bthat%252BKill%252Bgastric%252Bcancer%252Bcells%252Bvia%252Bthe%252Bmitochondria%252Bpathway%26aulast%3DWang%26aufirst%3DShuai%26date%3D2020%26volume%3D203%26spage%3D112630" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Nan  Lu</span>, <span class="hlFld-ContribAuthor ">Jin-ling  Huo</span>, <span class="hlFld-ContribAuthor ">Shuai  Wang</span>, <span class="hlFld-ContribAuthor ">Xiao-Han  Yuan</span>, <span class="hlFld-ContribAuthor ">Hong-Min  Liu</span>. </span><span class="cited-content_cbyCitation_article-title">Drug repurposing: Discovery of troxipide analogs as potent antitumor agents. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>202 </em>, 112471. <a href="https://doi.org/10.1016/j.ejmech.2020.112471" title="DOI URL">https://doi.org/10.1016/j.ejmech.2020.112471</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2020.112471&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2020.112471%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDrug%252Brepurposing%25253A%252BDiscovery%252Bof%252Btroxipide%252Banalogs%252Bas%252Bpotent%252Bantitumor%252Bagents%26aulast%3DLu%26aufirst%3DNan%26date%3D2020%26volume%3D202%26spage%3D112471" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Qing  Yu</span>, <span class="hlFld-ContribAuthor ">Yihan  Jiang</span>, <span class="hlFld-ContribAuthor ">Yi  Sun</span>. </span><span class="cited-content_cbyCitation_article-title">Anticancer drug discovery by targeting cullin neddylation. </span><span class="cited-content_cbyCitation_journal-name">Acta Pharmaceutica Sinica B</span><span> <strong>2020,</strong> <em>10 </em>
                                    (5)
                                     , 746-765. <a href="https://doi.org/10.1016/j.apsb.2019.09.005" title="DOI URL">https://doi.org/10.1016/j.apsb.2019.09.005</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.apsb.2019.09.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.apsb.2019.09.005%26sid%3Dliteratum%253Aachs%26jtitle%3DActa%2520Pharmaceutica%2520Sinica%2520B%26atitle%3DAnticancer%252Bdrug%252Bdiscovery%252Bby%252Btargeting%252Bcullin%252Bneddylation%26aulast%3DYu%26aufirst%3DQing%26date%3D2020%26volume%3D10%26issue%3D5%26spage%3D746%26epage%3D765" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Dong-Jun  Fu</span>, <span class="hlFld-ContribAuthor ">Jian  Song</span>, <span class="hlFld-ContribAuthor ">Ting  Zhu</span>, <span class="hlFld-ContribAuthor ">Xiao-Jing  Pang</span>, <span class="hlFld-ContribAuthor ">Sheng-Hui  Wang</span>, <span class="hlFld-ContribAuthor ">Yan-Bing  Zhang</span>, <span class="hlFld-ContribAuthor ">Bo-Wen  Wu</span>, <span class="hlFld-ContribAuthor ">Jun-Wei  Wang</span>, <span class="hlFld-ContribAuthor ">Xiaolin  Zi</span>, <span class="hlFld-ContribAuthor ">Sai-Yang  Zhang</span>, <span class="hlFld-ContribAuthor ">Hong-Min  Liu</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of novel tertiary amide derivatives as NEDDylation pathway activators to inhibit the tumor progression in vitro and in vivo. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>192 </em>, 112153. <a href="https://doi.org/10.1016/j.ejmech.2020.112153" title="DOI URL">https://doi.org/10.1016/j.ejmech.2020.112153</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2020.112153&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2020.112153%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252Bnovel%252Btertiary%252Bamide%252Bderivatives%252Bas%252BNEDDylation%252Bpathway%252Bactivators%252Bto%252Binhibit%252Bthe%252Btumor%252Bprogression%252Bin%2525C2%2525A0vitro%252Band%252Bin%2525C2%2525A0vivo%26aulast%3DFu%26aufirst%3DDong-Jun%26date%3D2020%26volume%3D192%26spage%3D112153" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Qing  Yu</span>, <span class="hlFld-ContribAuthor ">Zhiguo  Hu</span>, <span class="hlFld-ContribAuthor ">Yanwen  Shen</span>, <span class="hlFld-ContribAuthor ">Yihan  Jiang</span>, <span class="hlFld-ContribAuthor ">Peichen  Pan</span>, <span class="hlFld-ContribAuthor ">Tingjun  Hou</span>, <span class="hlFld-ContribAuthor ">Zhen-Qiang  Pan</span>, <span class="hlFld-ContribAuthor ">Jing  Huang</span>, <span class="hlFld-ContribAuthor ">Yi  Sun</span>. </span><span class="cited-content_cbyCitation_article-title">Gossypol inhibits cullin neddylation by targeting SAG-CUL5 and RBX1-CUL1 complexes. </span><span class="cited-content_cbyCitation_journal-name">Neoplasia</span><span> <strong>2020,</strong> <em>22 </em>
                                    (4)
                                     , 179-191. <a href="https://doi.org/10.1016/j.neo.2020.02.003" title="DOI URL">https://doi.org/10.1016/j.neo.2020.02.003</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.neo.2020.02.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.neo.2020.02.003%26sid%3Dliteratum%253Aachs%26jtitle%3DNeoplasia%26atitle%3DGossypol%252Binhibits%252Bcullin%252Bneddylation%252Bby%252Btargeting%252BSAG-CUL5%252Band%252BRBX1-CUL1%252Bcomplexes%26aulast%3DYu%26aufirst%3DQing%26date%3D2020%26volume%3D22%26issue%3D4%26spage%3D179%26epage%3D191" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Shuo  Yuan</span>, <span class="hlFld-ContribAuthor ">Sixi  Wang</span>, <span class="hlFld-ContribAuthor ">Min  Zhao</span>, <span class="hlFld-ContribAuthor ">Danqing  Zhang</span>, <span class="hlFld-ContribAuthor ">Jinjie  Chen</span>, <span class="hlFld-ContribAuthor ">Jian-Xin  Li</span>, <span class="hlFld-ContribAuthor ">Jingya  Zhang</span>, <span class="hlFld-ContribAuthor ">Yihui  Song</span>, <span class="hlFld-ContribAuthor ">Jinyi  Wang</span>, <span class="hlFld-ContribAuthor ">Bin  Yu</span>, <span class="hlFld-ContribAuthor ">Hongmin  Liu</span>. </span><span class="cited-content_cbyCitation_article-title">Brønsted acid-promoted ‘on–water’ C(sp3)-H functionalization for the synthesis of isoindolinone/[1,2,4]triazolo[1,5-a]pyrimidine derivatives targeting the SKP2-CKS1 interaction. </span><span class="cited-content_cbyCitation_journal-name">Chinese Chemical Letters</span><span> <strong>2020,</strong> <em>31 </em>
                                    (2)
                                     , 349-352. <a href="https://doi.org/10.1016/j.cclet.2019.07.019" title="DOI URL">https://doi.org/10.1016/j.cclet.2019.07.019</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.cclet.2019.07.019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.cclet.2019.07.019%26sid%3Dliteratum%253Aachs%26jtitle%3DChinese%2520Chemical%2520Letters%26atitle%3DBr%2525C3%2525B8nsted%252Bacid-promoted%252B%2525E2%252580%252598on%2525E2%252580%252593water%2525E2%252580%252599%252BC%252528sp3%252529-H%252Bfunctionalization%252Bfor%252Bthe%252Bsynthesis%252Bof%252Bisoindolinone%25252F%25255B1%25252C2%25252C4%25255Dtriazolo%25255B1%25252C5-a%25255Dpyrimidine%252Bderivatives%252Btargeting%252Bthe%252BSKP2-CKS1%252Binteraction%26aulast%3DYuan%26aufirst%3DShuo%26date%3D2020%26volume%3D31%26issue%3D2%26spage%3D349%26epage%3D352" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Shuai  Wang</span>, <span class="hlFld-ContribAuthor ">Dandan  Shen</span>, <span class="hlFld-ContribAuthor ">Lijie  Zhao</span>, <span class="hlFld-ContribAuthor ">Xiaohan  Yuan</span>, <span class="hlFld-ContribAuthor ">Jialing  Cheng</span>, <span class="hlFld-ContribAuthor ">Bin  Yu</span>, <span class="hlFld-ContribAuthor ">Yichao  Zheng</span>, <span class="hlFld-ContribAuthor ">Hongmin  Liu</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of [1,2,4]triazolo[1,5-a]pyrimidine derivatives as new bromodomain-containing protein 4 (BRD4) inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Chinese Chemical Letters</span><span> <strong>2020,</strong> <em>31 </em>
                                    (2)
                                     , 418-422. <a href="https://doi.org/10.1016/j.cclet.2019.08.029" title="DOI URL">https://doi.org/10.1016/j.cclet.2019.08.029</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.cclet.2019.08.029&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.cclet.2019.08.029%26sid%3Dliteratum%253Aachs%26jtitle%3DChinese%2520Chemical%2520Letters%26atitle%3DDiscovery%252Bof%252B%25255B1%25252C2%25252C4%25255Dtriazolo%25255B1%25252C5-a%25255Dpyrimidine%252Bderivatives%252Bas%252Bnew%252Bbromodomain-containing%252Bprotein%252B4%252B%252528BRD4%252529%252Binhibitors%26aulast%3DWang%26aufirst%3DShuai%26date%3D2020%26volume%3D31%26issue%3D2%26spage%3D418%26epage%3D422" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Lisha  Zhou</span>, <span class="hlFld-ContribAuthor ">Lijun  Jia</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting Protein Neddylation for Cancer Therapy. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2020,</strong>,, 297-315. <a href="https://doi.org/10.1007/978-981-15-1025-0_18" title="DOI URL">https://doi.org/10.1007/978-981-15-1025-0_18</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/978-981-15-1025-0_18&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2F978-981-15-1025-0_18%26sid%3Dliteratum%253Aachs%26atitle%3DTargeting%252BProtein%252BNeddylation%252Bfor%252BCancer%252BTherapy%26aulast%3DZhou%26aufirst%3DLisha%26date%3D2020%26date%3D2020%26spage%3D297%26epage%3D315%26pub%3DSpringer%2520Singapore%26atitle%3DCullin-RING%252BLigases%252Band%252BProtein%252BNeddylation%26aulast%3DSun%26aufirst%3DYi%26date%3D2020%26volume%3D1217" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Haibin  Zhou</span>, <span class="hlFld-ContribAuthor ">Jianfeng  Lu</span>, <span class="hlFld-ContribAuthor ">Chao-Yie  Yang</span>, <span class="hlFld-ContribAuthor ">Yi  Sun</span>, <span class="hlFld-ContribAuthor ">Shaomeng  Wang</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting DCN1-UBC12 Protein-Protein Interaction for Regulation of Neddylation Pathway. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2020,</strong>,, 349-362. <a href="https://doi.org/10.1007/978-981-15-1025-0_20" title="DOI URL">https://doi.org/10.1007/978-981-15-1025-0_20</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/978-981-15-1025-0_20&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2F978-981-15-1025-0_20%26sid%3Dliteratum%253Aachs%26atitle%3DTargeting%252BDCN1-UBC12%252BProtein-Protein%252BInteraction%252Bfor%252BRegulation%252Bof%252BNeddylation%252BPathway%26aulast%3DZhou%26aufirst%3DHaibin%26date%3D2020%26date%3D2020%26spage%3D349%26epage%3D362%26pub%3DSpringer%2520Singapore%26atitle%3DCullin-RING%252BLigases%252Band%252BProtein%252BNeddylation%26aulast%3DSun%26aufirst%3DYi%26date%3D2020%26volume%3D1217" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jian  Song</span>, <span class="hlFld-ContribAuthor ">Xin-Xin  Cui</span>, <span class="hlFld-ContribAuthor ">Bo-Wen  Wu</span>, <span class="hlFld-ContribAuthor ">Dong  Li</span>, <span class="hlFld-ContribAuthor ">Sheng-Hui  Wang</span>, <span class="hlFld-ContribAuthor ">Lei  Shi</span>, <span class="hlFld-ContribAuthor ">Ting  Zhu</span>, <span class="hlFld-ContribAuthor ">Yan-Bing  Zhang</span>, <span class="hlFld-ContribAuthor ">Sai-Yang  Zhang</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of 1,2,4-triazine-based derivatives as novel neddylation inhibitors and anticancer activity studies against gastric cancer MGC-803 cells. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry Letters</span><span> <strong>2020,</strong> <em>30 </em>
                                    (2)
                                     , 126791. <a href="https://doi.org/10.1016/j.bmcl.2019.126791" title="DOI URL">https://doi.org/10.1016/j.bmcl.2019.126791</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmcl.2019.126791&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmcl.2019.126791%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DDiscovery%252Bof%252B1%25252C2%25252C4-triazine-based%252Bderivatives%252Bas%252Bnovel%252Bneddylation%252Binhibitors%252Band%252Banticancer%252Bactivity%252Bstudies%252Bagainst%252Bgastric%252Bcancer%252BMGC-803%252Bcells%26aulast%3DSong%26aufirst%3DJian%26date%3D2020%26volume%3D30%26issue%3D2%26spage%3D126791" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Pengxing  He</span>, <span class="hlFld-ContribAuthor ">Shenghui  Niu</span>, <span class="hlFld-ContribAuthor ">Shuai  Wang</span>, <span class="hlFld-ContribAuthor ">Xiaojing  Shi</span>, <span class="hlFld-ContribAuthor ">Siqi  Feng</span>, <span class="hlFld-ContribAuthor ">Linna  Du</span>, <span class="hlFld-ContribAuthor ">Xuyang  Zhang</span>, <span class="hlFld-ContribAuthor ">Zhilu  Ma</span>, <span class="hlFld-ContribAuthor ">Bin  Yu</span>, <span class="hlFld-ContribAuthor ">Hongmin  Liu</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of WS-157 as a highly potent, selective and orally active EGFR inhibitor. </span><span class="cited-content_cbyCitation_journal-name">Acta Pharmaceutica Sinica B</span><span> <strong>2019,</strong> <em>9 </em>
                                    (6)
                                     , 1193-1203. <a href="https://doi.org/10.1016/j.apsb.2019.06.010" title="DOI URL">https://doi.org/10.1016/j.apsb.2019.06.010</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.apsb.2019.06.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.apsb.2019.06.010%26sid%3Dliteratum%253Aachs%26jtitle%3DActa%2520Pharmaceutica%2520Sinica%2520B%26atitle%3DDiscovery%252Bof%252BWS-157%252Bas%252Ba%252Bhighly%252Bpotent%25252C%252Bselective%252Band%252Borally%252Bactive%252BEGFR%252Binhibitor%26aulast%3DHe%26aufirst%3DPengxing%26date%3D2019%26volume%3D9%26issue%3D6%26spage%3D1193%26epage%3D1203" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yuan  Fang</span>, <span class="hlFld-ContribAuthor ">Bin  Yu</span>, <span class="hlFld-ContribAuthor ">Guochao  Liao</span>, <span class="hlFld-ContribAuthor ">Hong-Min  Liu</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting the DCN1–UBC12 protein–protein interaction: novel approaches and future directions. </span><span class="cited-content_cbyCitation_journal-name">Future Medicinal Chemistry</span><span> <strong>2019,</strong> <em>11 </em>
                                    (22)
                                     , 2869-2871. <a href="https://doi.org/10.4155/fmc-2019-0253" title="DOI URL">https://doi.org/10.4155/fmc-2019-0253</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.4155/fmc-2019-0253&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.4155%2Ffmc-2019-0253%26sid%3Dliteratum%253Aachs%26jtitle%3DFuture%2520Medicinal%2520Chemistry%26atitle%3DTargeting%252Bthe%252BDCN1%2525E2%252580%252593UBC12%252Bprotein%2525E2%252580%252593protein%252Binteraction%25253A%252Bnovel%252Bapproaches%252Band%252Bfuture%252Bdirections%26aulast%3DFang%26aufirst%3DYuan%26date%3D2019%26volume%3D11%26issue%3D22%26spage%3D2869%26epage%3D2871" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jian-Wei  Zhao</span>, <span class="hlFld-ContribAuthor ">Jia-Wen  Guo</span>, <span class="hlFld-ContribAuthor ">Ming-Jie  Huang</span>, <span class="hlFld-ContribAuthor ">Ya-Zhen  You</span>, <span class="hlFld-ContribAuthor ">Zeng-Hui  Wu</span>, <span class="hlFld-ContribAuthor ">Hong-Min  Liu</span>, <span class="hlFld-ContribAuthor ">Li-Hua  Huang</span>. </span><span class="cited-content_cbyCitation_article-title">Design, synthesis and biological evaluation of new steroidal β-triazoly enones as potent antiproliferative agents. </span><span class="cited-content_cbyCitation_journal-name">Steroids</span><span> <strong>2019,</strong> <em>150 </em>, 108431. <a href="https://doi.org/10.1016/j.steroids.2019.108431" title="DOI URL">https://doi.org/10.1016/j.steroids.2019.108431</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.steroids.2019.108431&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.steroids.2019.108431%26sid%3Dliteratum%253Aachs%26jtitle%3DSteroids%26atitle%3DDesign%25252C%252Bsynthesis%252Band%252Bbiological%252Bevaluation%252Bof%252Bnew%252Bsteroidal%252B%2525CE%2525B2-triazoly%252Benones%252Bas%252Bpotent%252Bantiproliferative%252Bagents%26aulast%3DZhao%26aufirst%3DJian-Wei%26date%3D2019%26volume%3D150%26spage%3D108431" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ting  Ma</span>, <span class="hlFld-ContribAuthor ">Qi-Sheng  Ma</span>, <span class="hlFld-ContribAuthor ">Bin  Yu</span>, <span class="hlFld-ContribAuthor ">Hong-Min  Liu</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of the theobromine derivative MQS-14 that induces death of MGC-803 cells mainly through ROS-mediated mechanisms. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2019,</strong> <em>174 </em>, 76-86. <a href="https://doi.org/10.1016/j.ejmech.2019.04.044" title="DOI URL">https://doi.org/10.1016/j.ejmech.2019.04.044</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2019.04.044&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2019.04.044%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252Bthe%252Btheobromine%252Bderivative%252BMQS-14%252Bthat%252Binduces%252Bdeath%252Bof%252BMGC-803%252Bcells%252Bmainly%252Bthrough%252BROS-mediated%252Bmechanisms%26aulast%3DMa%26aufirst%3DTing%26date%3D2019%26volume%3D174%26spage%3D76%26epage%3D86" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.9b00113/20190308/images/medium/jm-2019-00113s_0023.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.9b00113/20190308/images/large/jm-2019-00113s_0023.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00113&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.9b00113/20190308/images/medium/jm-2019-00113s_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.9b00113/20190308/images/large/jm-2019-00113s_0001.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Enzymatic cascades for protein neddylation.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.9b00113/20190308/images/large/jm-2019-00113s_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00113&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.9b00113/20190308/images/medium/jm-2019-00113s_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.9b00113/20190308/images/large/jm-2019-00113s_0002.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Representative small-molecule DCN1 inhibitors.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.9b00113/20190308/images/large/jm-2019-00113s_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00113&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.9b00113/20190308/images/medium/jm-2019-00113s_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.9b00113/20190308/images/large/jm-2019-00113s_0003.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Structural basis of <b>E1</b>-DCN1 binding. (A) <b>E1</b> (magenta) and <b>DI-591</b> (cyan) occupied the same binding site in DCN1 (green). Protein (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5UFI">5UFI</a>) was shown as cartoon, and small molecules were shown as sticks. The surface (white) of the binding pocket was shown in 20% transparence. (B, C) Interaction models of <b>E1</b> and <b>DI-591</b> in DCN1, respectively. For clarity, DCN1 was shown in 80% transparent cartoon. DCN1 residues interacting with <b>E1</b> were shown as sticks. Hydrogen bonds were shown as dashed lines. (D) 2D-view of the interactions between <b>E1</b> and DCN1.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.9b00113/20190308/images/large/jm-2019-00113s_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00113&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.9b00113/20190308/images/medium/jm-2019-00113s_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.9b00113/20190308/images/large/jm-2019-00113s_0004.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Preliminary SARs studies based on the starting point <b>E1</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.9b00113/20190308/images/large/jm-2019-00113s_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00113&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.9b00113/20190308/images/medium/jm-2019-00113s_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.9b00113/20190308/images/large/jm-2019-00113s_0005.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Scaffold replacement.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.9b00113/20190308/images/large/jm-2019-00113s_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00113&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.9b00113/20190308/images/medium/jm-2019-00113s_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.9b00113/20190308/images/large/jm-2019-00113s_0006.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Docking results of <b>WS</b>-<b>383</b> with DCN1. (A) Predicted binding model of compound <b>WS</b>-<b>383</b> (magenta) in complex with 6B5Q. (B) Predicted binding model of compound <b>WS</b>-<b>383</b> (orange) in complex with 5V83. (C) Predicted binding model of compound <b>WS</b>-<b>383</b> (cyan) in complex with 5V86. (D) Predicted binding model of compound <b>WS</b>-<b>383</b> (dark pink) in complex with 6BG3. (E) Predicted binding model of compound <b>WS</b>-<b>383</b> (green) in complex with 6BG5. The associated residues were shown in white sticks.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.9b00113/20190308/images/large/jm-2019-00113s_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00113&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.9b00113/20190308/images/medium/jm-2019-00113s_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.9b00113/20190308/images/large/jm-2019-00113s_0007.jpeg" id="rightTab-gr10" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Docking results of <b>WS-383</b> and <b>WS-524</b>. (A) Superimposition of <b>WS-383</b> (cyan) and <b>WS-524</b> (orange) with DCN1. Protein (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5UFI">5UFI</a>) was shown in surface, and hydrophobic region was shown in green. (B) Predicted binding model of compound <b>WS-383</b> (cyan) in complex with DCN1. (C) Two dimensional (2D) representation of compound <b>WS-383</b> in complex with DCN1; (D) Detailed views of <b>WS-383</b> and interacting residues in the P1–P3 pockets.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.9b00113/20190308/images/large/jm-2019-00113s_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00113&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig8" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.9b00113/20190308/images/medium/jm-2019-00113s_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.9b00113/20190308/images/large/jm-2019-00113s_0008.jpeg" id="rightTab-gr11" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Docking results of <b>WS</b>-<b>383</b> and <b>WS</b>-<b>524</b> with <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5UFI">5UFI</a>. (A) Superimposition of <b>WS</b>-<b>383</b> (cyan) and <b>WS</b>-<b>524</b> (orange) with DCN1. Protein was shown in surface, and hydrophobic region was shown in green. Compounds <b>WS</b>-<b>383</b> and <b>WS-524</b> were shown in dark cyan and dark pink sticks, respectively. (B) Predicted binding model of compound <b>WS</b>-<b>383</b> (dark cyan) in complex with DCN1, the associated residues were shown in white sticks.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.9b00113/20190308/images/large/jm-2019-00113s_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00113&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig9" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.9b00113/20190308/images/medium/jm-2019-00113s_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.9b00113/20190308/images/large/jm-2019-00113s_0009.jpeg" id="rightTab-gr12" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. (A) RMSD plot of the studied complexes during MD simulations. (B) Superimposition of the start structure (cyan) and that (yellow) after MD of compound <b>WS-383</b> with DCN1. Protein (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5UFI">5UFI</a>) was shown in white surface. (C) Superimposition of the start structure (orange) and that (pink) after MD of compound <b>WS-524</b> with DCN1. Protein was shown in white surface. (D) Interactions between compound <b>WS-524</b> and surrounding residues after the 20 ns MD simulations.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.9b00113/20190308/images/large/jm-2019-00113s_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00113&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig10" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 10</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.9b00113/20190308/images/medium/jm-2019-00113s_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.9b00113/20190308/images/large/jm-2019-00113s_0010.jpeg" id="rightTab-gr13" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 10. <b>WS-383</b> inhibited the DCN1–UBC12 interaction selectively and reversibly. (A) Structure of <b>WS-383</b>. (B) Structure of <b>WS-524</b>. (C) Inactivation curve of <b>WS-383</b>, <b>WS-524</b>, and <b>DI-591</b> against the DCN1–UBC12 interaction performed by the HTRF assay. (D) Inhibitory activity of <b>WS-383</b> (10 μM) against a panel of kinases. (E, F) The reversibility of <b>WS-383</b> to DCN1 was determined by the HTRF based dilution assay with a starting concentration at 500 nM following 80-fold dilution (E) and ultrafiltration experiment starting concentration at 500 nM following six round centrifuge. (F) <b>DI-591</b> was used as a control with a starting concentration at 800 nM. Data are the mean ± SD. <i>P</i> < 0.01 was considered statistically significant. All experiments were carried out at least three times.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.9b00113/20190308/images/large/jm-2019-00113s_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00113&amp;id=fig10"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig11" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 11</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.9b00113/20190308/images/medium/jm-2019-00113s_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.9b00113/20190308/images/large/jm-2019-00113s_0011.jpeg" id="rightTab-gr14" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 11. Protein and mRNA levels of UBC12 and DCN1 in the five gastric cell lines (MGC-803, SGC-7901, HGC-27, BGC-823, and GES-1). (A, B) Expression of UBC12, DCN1, and neddylated/ubiquitinated DCN1 (N8/Ub-DCN1) in the five indicated gastric cell lines. (C, D) Relative amounts of UBC12 (C) and DCN1 (D) mRNA in the five gastric cell lines were quantified with RT-qPCR. GAPDH was used as the loading control. Data are the mean ± SD. <i>P</i> < 0.01 was considered statistically significant. All experiments were carried out three times.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.9b00113/20190308/images/large/jm-2019-00113s_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00113&amp;id=fig11"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig12" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Figure 12</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.9b00113/20190308/images/medium/jm-2019-00113s_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.9b00113/20190308/images/large/jm-2019-00113s_0012.jpeg" id="rightTab-gr15" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 12. Enhancement of thermal stability of DCN1 protein by <b>WS-383</b> and <b>DI-591</b> treatment in MGC-803 cells. (A, B) Protein levels of DCN1 in MGC-803 cells treated with <b>WS-383</b> (A) and <b>DI-591</b> (B) at the indicated concentrations for 1 h, followed by heating at 53 °C for 3 min were analyzed by Western blot. GAPDH was used as a loading control.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.9b00113/20190308/images/large/jm-2019-00113s_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00113&amp;id=fig12"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig13" data-index="13" class="article__inlineFigure"><h2 class="fig-label">Figure 13</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.9b00113/20190308/images/medium/jm-2019-00113s_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.9b00113/20190308/images/large/jm-2019-00113s_0013.jpeg" id="rightTab-gr16" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 13. Cellular effects of <b>WS-383</b>, <b>DI-591</b>, and <b>MLN4924</b> on cullin neddylation and CRL substrates. (A) Deneddylation effects of <b>MLN4924</b>, <b>WS-383</b>, and <b>WS-524</b> on cullin1–5 at indicated concentrations for 24 h treatment in MGC-803 cells. (B) Deneddylation effects on cullin1–5 and CRL substrates accumulation in KYSE70 cells with 24 h treatment of <b>WS-383</b> at indicated concentrations. (C) Deneddylation effects on cullin1–5 and CRL substrates accumulation in MGC-803 cells with 24 h treatment of <b>DI-591</b> at indicated concentrations. (D, E) The expressions of p21, p27, and NRF2 in MGC-803 cells after 24 h treatment of <b>WS-383</b>(D) and <b>MLN4924</b> (E). GAPDH was used as the loading control. All experiments were carried out three times.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.9b00113/20190308/images/large/jm-2019-00113s_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00113&amp;id=fig13"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig14" data-index="14" class="article__inlineFigure"><h2 class="fig-label">Figure 14</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.9b00113/20190308/images/medium/jm-2019-00113s_0014.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.9b00113/20190308/images/large/jm-2019-00113s_0014.jpeg" id="rightTab-gr17" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 14. Effect on ROS generation in MGC-803 cells. MGC-803 cells were treated with indicated concentrations of <b>WS-383</b>, <b>WS-524</b>, <b>NAcM-OPT</b>, or <b>DI-591</b> for 24 h. The levels of ROS were detected by flow cytometry.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.9b00113/20190308/images/large/jm-2019-00113s_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00113&amp;id=fig14"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="15" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.9b00113/20190308/images/medium/jm-2019-00113s_0015.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.9b00113/20190308/images/large/jm-2019-00113s_0015.jpeg" id="rightTab-gr18" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of Compounds <b>E1</b>–<b>E5</b> and <b>E8</b>–<b>E44</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.9b00113/20190308/images/large/jm-2019-00113s_0015.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00113&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) acetic acid, 120 °C, 2–15 h, reflux; (b) POCl<sub>3</sub>, 90 °C, 3–5 h, reflux; (c) EtOH, rt, 5 h; (d) Na<sub>2</sub>CO<sub>3</sub>, acetone, 60 °C, 3–5 h, reflux.</p></p></figure><figure data-id="sch2" data-index="16" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.9b00113/20190308/images/medium/jm-2019-00113s_0016.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.9b00113/20190308/images/large/jm-2019-00113s_0016.jpeg" id="rightTab-gr19" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of Compounds <b>E6</b> and <b>E7</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.9b00113/20190308/images/large/jm-2019-00113s_0016.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00113&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) EtOH, rt, 5 h.</p></p></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i157">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_41362" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_41362" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 57 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Petroski, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deshaies, R. J.</span></span> <span> </span><span class="NLM_article-title">Function and regulation of cullin–RING ubiquitin ligases</span>. <i>Nat. Rev. Mol. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">9</span>– <span class="NLM_lpage">20</span>, <span class="refDoi"> DOI: 10.1038/nrm1547</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=10.1038%2Fnrm1547" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=15688063" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=1%3ACAS%3A528%3ADC%252BD2MXlvVWg" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2005&pages=9-20&author=M.+D.+Petroskiauthor=R.+J.+Deshaies&title=Function+and+regulation+of+cullin%E2%80%93RING+ubiquitin+ligases&doi=10.1038%2Fnrm1547"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Function and regulation of cullin-RING ubiquitin ligases</span></div><div class="casAuthors">Petroski, Matthew D.; Deshaies, Raymond J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Molecular Cell Biology</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">9-20</span>CODEN:
                <span class="NLM_cas:coden">NRMCBP</span>;
        ISSN:<span class="NLM_cas:issn">1471-0072</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Cullin-RING complexes comprise the largest known class of ubiquitin ligases.  Owing to the great diversity of their substrate-receptor subunits, it is possible that there are hundreds of distinct cullin-RING ubiquitin ligases in eukaryotic cells, which establishes these enzymes as key mediators of post-translational protein regulation.  Here, the authors focus on the compn., regulation, and function of cullin-RING ligases, and describe how these enzymes can be characterized by a set of general principles.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqFw5LuMMHZoLVg90H21EOLACvtfcHk0lgt6nEQdfZ14A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXlvVWg&md5=297f142f44f971c6858306b1adb36254</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Fnrm1547&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrm1547%26sid%3Dliteratum%253Aachs%26aulast%3DPetroski%26aufirst%3DM.%2BD.%26aulast%3DDeshaies%26aufirst%3DR.%2BJ.%26atitle%3DFunction%2520and%2520regulation%2520of%2520cullin%25E2%2580%2593RING%2520ubiquitin%2520ligases%26jtitle%3DNat.%2520Rev.%2520Mol.%2520Cell%2520Biol.%26date%3D2005%26volume%3D6%26spage%3D9%26epage%3D20%26doi%3D10.1038%2Fnrm1547" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, Y.</span></span> <span> </span><span class="NLM_article-title">Cullin-RING ligases as attractive anti-cancer targets</span>. <i>Curr. Pharm. Des.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">3215</span>– <span class="NLM_lpage">3225</span>, <span class="refDoi"> DOI: 10.2174/13816128113199990300</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=10.2174%2F13816128113199990300" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=23151137" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=1%3ACAS%3A528%3ADC%252BC3sXotFagtL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2013&pages=3215-3225&author=Y.+Zhaoauthor=Y.+Sun&title=Cullin-RING+ligases+as+attractive+anti-cancer+targets&doi=10.2174%2F13816128113199990300"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Cullin-RING ligases as attractive anti-cancer targets</span></div><div class="casAuthors">Zhao, Yongchao; Sun, Yi</div><div class="citationInfo"><span class="NLM_cas:title">Current Pharmaceutical Design</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">3215-3225</span>CODEN:
                <span class="NLM_cas:coden">CPDEFP</span>;
        ISSN:<span class="NLM_cas:issn">1381-6128</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  The ubiquitin-proteasome system (UPS) promotes the timely degrdn. of short-lived proteins with key regulatory roles in a vast array of biol. processes, such as cell cycle progression, oncogenesis and genome integrity.  Thus, abnormal regulation of UPS disrupts the protein homeostasis and causes many human diseases, particularly cancer.  Indeed, the FDA approval of bortezomib, the first class of general proteasome inhibitor, for the treatment of multiple myeloma, demonstrated that the UPS can be an attractive anti-cancer target.  However, normal cell toxicity assocd. with bortezomib, resulting from global inhibition of protein degrdn., promotes the focus of drug discovery efforts on targeting enzymes upstream of the proteasome for better specificity.  E3 ubiquitin ligases, particularly those known to be activated in human cancer, become an attractive choice.  Cullin-RING Ligases (CRLs) with multiple components are the largest family of E3 ubiquitin ligases and are responsible for ubiquitination of ∼20% of cellular proteins degraded through UPS.  Activity of CRLs is dynamically regulated and requires the RING component and cullin neddylation.  In this review, we will introduce the UPS and CRL E3s and discuss the biol. processes regulated by each of eight CRLs through substrate degrdn.  We will further discuss how cullin neddylation controls CRL activity, and how CRLs are being validated as the attractive cancer targets by abrogating the RING component through genetic means and by inhibiting cullin neddylation via MLN4924, a small mol. indirect inhibitor of CRLs, currently in several Phase I clin. trials.  Finally, we will discuss current efforts and future perspectives on the development of addnl. inhibitors of CRLs by targeting E2 and/or E3 of cullin neddylation and CRL-mediated ubiquitination as potential anti-cancer agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp-K-w4MaFbdLVg90H21EOLACvtfcHk0liCSeUGHc121w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXotFagtL8%253D&md5=4498de520df1c0a24796bdbfd5b3ce17</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.2174%2F13816128113199990300&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F13816128113199990300%26sid%3Dliteratum%253Aachs%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DSun%26aufirst%3DY.%26atitle%3DCullin-RING%2520ligases%2520as%2520attractive%2520anti-cancer%2520targets%26jtitle%3DCurr.%2520Pharm.%2520Des.%26date%3D2013%26volume%3D19%26spage%3D3215%26epage%3D3225%26doi%3D10.2174%2F13816128113199990300" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Andérica-Romero, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">González-Herrera, I. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Santamaría, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pedraza-Chaverri, J.</span></span> <span> </span><span class="NLM_article-title">Cullin 3 as a novel target in diverse pathologies</span>. <i>Redox Biol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">366</span>– <span class="NLM_lpage">372</span>, <span class="refDoi"> DOI: 10.1016/j.redox.2013.07.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=10.1016%2Fj.redox.2013.07.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=24024173" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=1%3ACAS%3A528%3ADC%252BC2cXptFKntA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2013&pages=366-372&author=A.+C.+And%C3%A9rica-Romeroauthor=I.+G.+Gonz%C3%A1lez-Herreraauthor=A.+Santamar%C3%ADaauthor=J.+Pedraza-Chaverri&title=Cullin+3+as+a+novel+target+in+diverse+pathologies&doi=10.1016%2Fj.redox.2013.07.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Cullin 3 as a novel target in diverse pathologies</span></div><div class="casAuthors">Anderica-Romero, Ana Cristina; Gonzalez-Herrera, Irma Gabriela; Santamaria, Abel; Pedraza-Chaverri, Jose</div><div class="citationInfo"><span class="NLM_cas:title">Redox Biology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">366-372</span>CODEN:
                <span class="NLM_cas:coden">RBEIB3</span>;
        ISSN:<span class="NLM_cas:issn">2213-2317</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Recent evidence suggests that the malfunctioning disposal system of cell protein called ubiquitin-proteasome system (UPS) plays an important role in the development of disorders, including cancer and neurodegenerative diseases.  Accumulating evidence suggests that the abnormal regulation of the E3 ubiquitin ligases, essential components of the UPS, contributes to uncontrolled proliferation, genomic instability and cancer, since these ligases and their substrates are involved in the regulation of cell cycle progression, gene transcription, signal transduction, DNA replication and others.  Through selective degrdn. of specific substrates, E3 ligases regulate different biol. processes.  Cullins are a family of proteins that confer substrate specificity to multimeric complex of E3 ligases acting as scaffold proteins.  So far, seven members of the cullin family of proteins have been identified.  Interestingly, the data generated by several groups indicate that cullin 3 (Cul3) has begun to emerge as a protein involved in the etiopathol. of multiple diseases.  In this paper the authors examine the latest advances in basic research on the biol. of Cul3 and how it could help to direct drug discovery efforts on this target.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqQlQLrfY9SwLVg90H21EOLACvtfcHk0liCSeUGHc121w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXptFKntA%253D%253D&md5=7d0b387948eba17c4695183bf5fad3a3</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1016%2Fj.redox.2013.07.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.redox.2013.07.003%26sid%3Dliteratum%253Aachs%26aulast%3DAnd%25C3%25A9rica-Romero%26aufirst%3DA.%2BC.%26aulast%3DGonz%25C3%25A1lez-Herrera%26aufirst%3DI.%2BG.%26aulast%3DSantamar%25C3%25ADa%26aufirst%3DA.%26aulast%3DPedraza-Chaverri%26aufirst%3DJ.%26atitle%3DCullin%25203%2520as%2520a%2520novel%2520target%2520in%2520diverse%2520pathologies%26jtitle%3DRedox%2520Biol.%26date%3D2013%26volume%3D1%26spage%3D366%26epage%3D372%26doi%3D10.1016%2Fj.redox.2013.07.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hori, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Osaka, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiba, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miyamoto, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Okabayashi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shimbara, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kato, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanaka, K.</span></span> <span> </span><span class="NLM_article-title">Covalent modification of all members of human cullin family proteins by NEDD8</span>. <i>Oncogene</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">6829</span>– <span class="NLM_lpage">6834</span>, <span class="refDoi"> DOI: 10.1038/sj.onc.1203093</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=10.1038%2Fsj.onc.1203093" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=10597293" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=1%3ACAS%3A528%3ADyaK1MXotV2ls78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=1999&pages=6829-6834&author=T.+Horiauthor=F.+Osakaauthor=T.+Chibaauthor=C.+Miyamotoauthor=K.+Okabayashiauthor=N.+Shimbaraauthor=S.+Katoauthor=K.+Tanaka&title=Covalent+modification+of+all+members+of+human+cullin+family+proteins+by+NEDD8&doi=10.1038%2Fsj.onc.1203093"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Covalent modification of all members of human cullin family proteins by NEDD8</span></div><div class="casAuthors">Hori, Tomoko; Osaka, Fumio; Chiba, Tomoki; Miyamoto, Chikara; Okabayashi, Ken; Shimbara, Naoki; Kato, Seishi; Tanaka, Keiji</div><div class="citationInfo"><span class="NLM_cas:title">Oncogene</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">48</span>),
    <span class="NLM_cas:pages">6829-6834</span>CODEN:
                <span class="NLM_cas:coden">ONCNES</span>;
        ISSN:<span class="NLM_cas:issn">0950-9232</span>.
    
            (<span class="NLM_cas:orgname">Stockton Press</span>)
        </div><div class="casAbstract">Recently the authors found that NEDD8, a ubiquitin-like protein, was linked covalently to human cullin-4A (abbreviated Cul-4A) by a new ubiquitin-related pathway that is analogous to but distinct from the ligating system for SUMO1, another ubiquitin-like protein.  However, it remained unknown whether the other five members of the family of human cullin/Cdc53 proteins are modified by NEDD8.  Here the authors report that all Hs-Cul family proteins, such as Cul-1, Cul-2, Cul-3, Cul-4B, and Cul-5, in addn. to Cul-4A, were modified by covalent attachment of NEDD8 in rabbit reticulocyte lysates.  Moreover, by comprehensive Northern-blot analyses, the authors examd. multiple tissue distributions of the messages for all Cul-family proteins, NEDD8, and the NEDD8-ligating system consisting of APP-BP1/hUba3, and hUbc12, which function as E1- and E2-like enzymes, resp.  The expressions of Cul-1, Cul-2, and Cul-3 resembled each other and were apparently correlated to those of NEDD8 and the NEDD8-ligating system in various human cells and tissues.  However, the mRNA levels of Cul-4A, Cul-4B, and Cul-5 differed considerably from each other as well as from other Cul-family proteins.  The enhanced expression of all Cul-family proteins except Cul-5 was obsd. in a variety of tumor cell lines.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpJYdFEgvbtMbVg90H21EOLACvtfcHk0ljRwUFrEtvk2Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXotV2ls78%253D&md5=25f186ac16e8875121b06e3e9baf2d18</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1038%2Fsj.onc.1203093&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.onc.1203093%26sid%3Dliteratum%253Aachs%26aulast%3DHori%26aufirst%3DT.%26aulast%3DOsaka%26aufirst%3DF.%26aulast%3DChiba%26aufirst%3DT.%26aulast%3DMiyamoto%26aufirst%3DC.%26aulast%3DOkabayashi%26aufirst%3DK.%26aulast%3DShimbara%26aufirst%3DN.%26aulast%3DKato%26aufirst%3DS.%26aulast%3DTanaka%26aufirst%3DK.%26atitle%3DCovalent%2520modification%2520of%2520all%2520members%2520of%2520human%2520cullin%2520family%2520proteins%2520by%2520NEDD8%26jtitle%3DOncogene%26date%3D1999%26volume%3D18%26spage%3D6829%26epage%3D6834%26doi%3D10.1038%2Fsj.onc.1203093" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Enchev, R. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schulman, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peter, M.</span></span> <span> </span><span class="NLM_article-title">Protein neddylation: beyond cullin–RING ligases</span>. <i>Nat. Rev. Mol. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">30</span>– <span class="NLM_lpage">44</span>, <span class="refDoi"> DOI: 10.1038/nrm3919</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=10.1038%2Fnrm3919" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=25531226" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitFKlt7fL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2015&pages=30-44&author=R.+I.+Enchevauthor=B.+A.+Schulmanauthor=M.+Peter&title=Protein+neddylation%3A+beyond+cullin%E2%80%93RING+ligases&doi=10.1038%2Fnrm3919"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Protein neddylation: beyond cullin-RING ligases</span></div><div class="casAuthors">Enchev, Radoslav I.; Schulman, Brenda A.; Peter, Matthias</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Molecular Cell Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">30-44</span>CODEN:
                <span class="NLM_cas:coden">NRMCBP</span>;
        ISSN:<span class="NLM_cas:issn">1471-0072</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  NEDD8 (neural precursor cell expressed developmentally downregulated protein 8) is a ubiquitin-like protein that activates the largest ubiquitin E3 ligase family, the cullin-RING ligases.  Many non-cullin neddylation targets have been proposed in recent years.  However, overexpression of exogenous NEDD8 can trigger NEDD8 conjugation through the ubiquitylation machinery, which makes validating potential NEDD8 targets challenging.  Here, we re-evaluate studies of non-cullin targets of NEDD8 in light of the current understanding of the neddylation pathway, and suggest criteria for identifying genuine neddylation substrates under homeostatic conditions.  We describe the biol. processes that might be regulated by non-cullin neddylation, and the utility of neddylation inhibitors for research and as potential therapies.  Understanding the biol. significance of non-cullin neddylation is an exciting research prospect primed to reveal fundamental insights.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqTz4DUdIwhJbVg90H21EOLACvtfcHk0ljRwUFrEtvk2Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitFKlt7fL&md5=a73346bd599f5d5c09d983ca8f13f2f4</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1038%2Fnrm3919&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrm3919%26sid%3Dliteratum%253Aachs%26aulast%3DEnchev%26aufirst%3DR.%2BI.%26aulast%3DSchulman%26aufirst%3DB.%2BA.%26aulast%3DPeter%26aufirst%3DM.%26atitle%3DProtein%2520neddylation%253A%2520beyond%2520cullin%25E2%2580%2593RING%2520ligases%26jtitle%3DNat.%2520Rev.%2520Mol.%2520Cell%2520Biol.%26date%3D2015%26volume%3D16%26spage%3D30%26epage%3D44%26doi%3D10.1038%2Fnrm3919" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Deshaies, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Emberley, E. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saha, A.</span></span> <span> </span><span class="NLM_article-title">Control of cullin-ring ubiquitin ligase activity by nedd8</span>. <i>Subcell. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">41</span>– <span class="NLM_lpage">56</span>, <span class="refDoi"> DOI: 10.1007/978-1-4419-6676-6_4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=10.1007%2F978-1-4419-6676-6_4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=21222272" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtV2hsb%252FJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2010&pages=41-56&author=R.+J.+Deshaiesauthor=E.+D.+Emberleyauthor=A.+Saha&title=Control+of+cullin-ring+ubiquitin+ligase+activity+by+nedd8&doi=10.1007%2F978-1-4419-6676-6_4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Control of cullin-RING ubiquitin ligase activity by Nedd8</span></div><div class="casAuthors">Deshaies, Raymond J.; Emberley, Ethan D.; Saha, Anjanabha</div><div class="citationInfo"><span class="NLM_cas:title">Subcellular Biochemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">Conjugation and Deconjugation of Ubiquitin Family Modifiers</span>),
    <span class="NLM_cas:pages">41-56</span>CODEN:
                <span class="NLM_cas:coden">SBCBAG</span>;
        ISSN:<span class="NLM_cas:issn">0306-0225</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">A review.  The Cullin-RING ubiquitin ligase (CRL) family, which may no. as many as 350 different enzymes, has an enormous impact on cellular regulation.  CRL enzymes regulate cell biol. by conjugating ubiquitin onto target proteins that are involved in a multitude of processes.  In most cases, this leads to degrdn. of the target, but in some cases CRL-dependent ubiquitination acts as a switch to activate or repress target function.  The ubiquitin ligase activity of CRLs is controlled by cycles of attachment and removal of the ubiquitin-like protein Nedd8.  Conjugation of Nedd8 onto the cullin subunit of CRLs promotes assembly of an intact CRL complex and switches on ubiquitin ligase activity.  Conversely, removal of Nedd8 switches off ubiquitin ligase activity and initiates CRL disassembly.  Continuous maintenance of CRL function in vivo requires the activities of both the Nedd8-conjugating and deconjugating enzymes, pointing to a crit. role of complex dynamics in CRL function.  Here, we review how the Nedd8 cycle controls CRL activity and how perturbations of this cycle can lead to disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrqPoJEejak6rVg90H21EOLACvtfcHk0ljRwUFrEtvk2Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtV2hsb%252FJ&md5=e6b0afe5b8dfe4864978ecc0b47c17a2</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1007%2F978-1-4419-6676-6_4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252F978-1-4419-6676-6_4%26sid%3Dliteratum%253Aachs%26aulast%3DDeshaies%26aufirst%3DR.%2BJ.%26aulast%3DEmberley%26aufirst%3DE.%2BD.%26aulast%3DSaha%26aufirst%3DA.%26atitle%3DControl%2520of%2520cullin-ring%2520ubiquitin%2520ligase%2520activity%2520by%2520nedd8%26jtitle%3DSubcell.%2520Biochem.%26date%3D2010%26volume%3D54%26spage%3D41%26epage%3D56%26doi%3D10.1007%2F978-1-4419-6676-6_4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morgan, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, Y.</span></span> <span> </span><span class="NLM_article-title">Targeting neddylation pathways to inactivate cullin-RING ligases for anticancer therapy</span>. <i>Antioxid. Redox Signaling</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">2383</span>– <span class="NLM_lpage">2400</span>, <span class="refDoi"> DOI: 10.1089/ars.2013.5795</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=10.1089%2Fars.2013.5795" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=24410571" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvFKltrzK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2014&pages=2383-2400&author=Y.+Zhaoauthor=M.+A.+Morganauthor=Y.+Sun&title=Targeting+neddylation+pathways+to+inactivate+cullin-RING+ligases+for+anticancer+therapy&doi=10.1089%2Fars.2013.5795"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting Neddylation Pathways to Inactivate Cullin-RING Ligases for Anticancer Therapy</span></div><div class="casAuthors">Zhao, Yongchao; Morgan, Meredith A.; Sun, Yi</div><div class="citationInfo"><span class="NLM_cas:title">Antioxidants & Redox Signaling</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">2383-2400</span>CODEN:
                <span class="NLM_cas:coden">ARSIF2</span>;
        ISSN:<span class="NLM_cas:issn">1523-0864</span>.
    
            (<span class="NLM_cas:orgname">Mary Ann Liebert, Inc.</span>)
        </div><div class="casAbstract">A review.  Significance: Protein neddylation is catalyzed by an E1 NEDD8-activating enzyme (NAE), an E2 NEDD8-conjugating enzyme, and an E3 NEDD8 ligase.  Known physiol. substrates of neddylation are cullin family members.  Cullin neddylation leads to activation of cullin-RING ligases (CRLs), the largest family of E3 ubiquitin ligases responsible for ubiquitylation and degrdn. of many key signaling/regulatory proteins.  Thus, through modulating CRLs, neddylation regulates many biol. processes, including cell cycle progression, signal transduction, and tumorigenesis.  Given that NEDD8 is overexpressed and CRLs are abnormally activated in many human cancers, targeting protein neddylation, in general, and cullin neddylation, in particular, appears to be an attractive anticancer approach.  Recent Advances: MLN4924, a small mol. inhibitor of NAE, was discovered that inactivates CRLs and causes accumulation of CRL substrates to suppress tumor cell growth both in vitro and in vivo.  Promising preclin. results advanced MLN4924 to several clin. trials for anticancer therapy.  Crit. Issues: In preclin. settings, MLN4924 effectively suppresses tumor cell growth by inducing apoptosis, senescence, and autophagy, and causes sensitization to chemoradiation therapies in a cellular context-dependent manner.  Signal mols. that det. the cell fate upon MLN4924 treatment, however, remain elusive.  Cancer cells develop MLN4924 resistance by selecting target mutations.  Future Directions: In the clin. side, several Phase 1b trials are under way to det. the safety and efficacy of MLN4924, acting alone or in combination with conventional chemotherapy, against human solid tumors.  In the preclin. side, the efforts are being made to develop addnl. neddylation inhibitors by targeting NEDD8 E2s and E3s.  Antioxid. Redox Signal. 21, 2383-2400.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr8ivf2UQt_DrVg90H21EOLACvtfcHk0lijI9mbO1LVjA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvFKltrzK&md5=42ff82c17092b1d5d6fc3526c9f2241a</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1089%2Fars.2013.5795&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1089%252Fars.2013.5795%26sid%3Dliteratum%253Aachs%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DMorgan%26aufirst%3DM.%2BA.%26aulast%3DSun%26aufirst%3DY.%26atitle%3DTargeting%2520neddylation%2520pathways%2520to%2520inactivate%2520cullin-RING%2520ligases%2520for%2520anticancer%2520therapy%26jtitle%3DAntioxid.%2520Redox%2520Signaling%26date%3D2014%26volume%3D21%26spage%3D2383%26epage%3D2400%26doi%3D10.1089%2Fars.2013.5795" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bulatov, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span> <span> </span><span class="NLM_article-title">Targeting cullin-RING E3 ubiquitin ligases for drug discovery: structure, assembly and small-molecule modulation</span>. <i>Biochem. J.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>467</i></span>,  <span class="NLM_fpage">365</span>– <span class="NLM_lpage">386</span>, <span class="refDoi"> DOI: 10.1042/BJ20141450</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=10.1042%2FBJ20141450" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=25886174" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=1%3ACAS%3A528%3ADC%252BC2MXmslChu7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=467&publication_year=2015&pages=365-386&author=E.+Bulatovauthor=A.+Ciulli&title=Targeting+cullin-RING+E3+ubiquitin+ligases+for+drug+discovery%3A+structure%2C+assembly+and+small-molecule+modulation&doi=10.1042%2FBJ20141450"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting Cullin-RING E3 ubiquitin ligases for drug discovery: structure, assembly and small-molecule modulation</span></div><div class="casAuthors">Bulatov, Emil; Ciulli, Alessio</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical Journal</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">467</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">365-386</span>CODEN:
                <span class="NLM_cas:coden">BIJOAK</span>;
        ISSN:<span class="NLM_cas:issn">0264-6021</span>.
    
            (<span class="NLM_cas:orgname">Portland Press Ltd.</span>)
        </div><div class="casAbstract">A review.  In the last decade, the ubiquitin-proteasome system has emerged as a valid target for the development of novel therapeutics.  E3 ubiquitin ligases are particularly attractive targets because they confer substrate specificity on the ubiquitin system.  CRLs [Cullin-RING (really interesting new gene) E3 ubiquitin ligases] draw particular attention, being the largest family of E3s.  The CRLs assemble into functional multisubunit complexes using a repertoire of substrate receptors, adaptors, Cullin scaffolds and RING-box proteins.  Drug discovery targeting CRLs is growing in importance due to mounting evidence pointing to significant roles of these enzymes in diverse biol. processes and human diseases, including cancer, where CRLs and their substrates often function as tumor suppressors or oncogenes.  In the present review, we provide an account of the assembly and structure of CRL complexes, and outline the current state of the field in terms of available knowledge of small-mol. inhibitors and modulators of CRL activity.  A comprehensive overview of the reported crystal structures of CRL subunits, components and full-size complexes, alone or with bound small mols. and substrate peptides, is included.  This information is providing increasing opportunities to aid the rational structure-based design of chem. probes and potential small-mol. therapeutics targeting CRLs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrgO7lBVCWaQrVg90H21EOLACvtfcHk0lijI9mbO1LVjA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXmslChu7k%253D&md5=8f76267e923c5e8868fa127b828296aa</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1042%2FBJ20141450&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252FBJ20141450%26sid%3Dliteratum%253Aachs%26aulast%3DBulatov%26aufirst%3DE.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DTargeting%2520cullin-RING%2520E3%2520ubiquitin%2520ligases%2520for%2520drug%2520discovery%253A%2520structure%252C%2520assembly%2520and%2520small-molecule%2520modulation%26jtitle%3DBiochem.%2520J.%26date%3D2015%26volume%3D467%26spage%3D365%26epage%3D386%26doi%3D10.1042%2FBJ20141450" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tanaka, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakatani, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kamitani, T.</span></span> <span> </span><span class="NLM_article-title">Inhibition of nedd8-conjugation pathway by novel molecules: potential approaches to anticancer therapy</span>. <i>Mol. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">267</span>– <span class="NLM_lpage">275</span>, <span class="refDoi"> DOI: 10.1016/j.molonc.2012.01.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=10.1016%2Fj.molonc.2012.01.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=22306028" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=1%3ACAS%3A528%3ADC%252BC38Xnslags7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2012&pages=267-275&author=T.+Tanakaauthor=T.+Nakataniauthor=T.+Kamitani&title=Inhibition+of+nedd8-conjugation+pathway+by+novel+molecules%3A+potential+approaches+to+anticancer+therapy&doi=10.1016%2Fj.molonc.2012.01.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of NEDD8-conjugation pathway by novel molecules: Potential approaches to anticancer therapy</span></div><div class="casAuthors">Tanaka, Tomoaki; Nakatani, Tatsuya; Kamitani, Tetsu</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Oncology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">267-275</span>CODEN:
                <span class="NLM_cas:coden">MOONC3</span>;
        ISSN:<span class="NLM_cas:issn">1574-7891</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Cancer cells can survive through the upregulation of cell cycle and the escape from apoptosis induced by numerous cellular stresses.  In the normal cells, these biol. cascades depend on scheduled proteolytic degrdn. of regulatory proteins via the ubiquitin-proteasome pathway.  Therefore, interruption of regulated proteolytic pathways leads to abnormal cell-proliferation.  Ubiquitin ligases called SCF complex (consisting of Skp-1, cullin, and F-box protein) or CRL (cullin-RING ubiquitin ligase) are predominant in a family of E3 ubiquitin ligases that control a final step in ubiquitination of diverse substrates.  To a great extent, the ubiquitin ligase activity of the SCF complex requires the conjugation of NEDD8 to cullins, i.e. scaffold proteins.  This review is anticipated to review the downregulation system of NEDD8 conjugation by several factors including a chem. compd. such as MLN4924 and protein mols. (e.g. COP9 signalosome, inactive mutant of Ubc12, and NUB1/NUB1L).  Since the downregulation of NEDD8 conjugation affects cell-cycle progression by inhibiting the ligase activity of SCF complexes, such knowledge in the NEDD8-conjugation pathway will contribute to the more magnificent therapies that selectively suppress tumorigenesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrtddvnq1BRvrVg90H21EOLACvtfcHk0lirLWlt1GKMIA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xnslags7g%253D&md5=1648458a47d5f38fb9ecf2678d9eff6b</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1016%2Fj.molonc.2012.01.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molonc.2012.01.003%26sid%3Dliteratum%253Aachs%26aulast%3DTanaka%26aufirst%3DT.%26aulast%3DNakatani%26aufirst%3DT.%26aulast%3DKamitani%26aufirst%3DT.%26atitle%3DInhibition%2520of%2520nedd8-conjugation%2520pathway%2520by%2520novel%2520molecules%253A%2520potential%2520approaches%2520to%2520anticancer%2520therapy%26jtitle%3DMol.%2520Oncol.%26date%3D2012%26volume%3D6%26spage%3D267%26epage%3D275%26doi%3D10.1016%2Fj.molonc.2012.01.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Soucy, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, P. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rolfe, M.</span></span> <span> </span><span class="NLM_article-title">Targeting nedd8-activated cullin-RING ligases for the treatment of cancer</span>. <i>Clin. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">3912</span>– <span class="NLM_lpage">3916</span>, <span class="refDoi"> DOI: 10.1158/1078-0432.CCR-09-0343</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=10.1158%2F1078-0432.CCR-09-0343" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=19509147" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=1%3ACAS%3A528%3ADC%252BD1MXntlOisrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2009&pages=3912-3916&author=T.+A.+Soucyauthor=P.+G.+Smithauthor=M.+Rolfe&title=Targeting+nedd8-activated+cullin-RING+ligases+for+the+treatment+of+cancer&doi=10.1158%2F1078-0432.CCR-09-0343"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting NEDD8-Activated Cullin-RING Ligases for the Treatment of Cancer</span></div><div class="casAuthors">Soucy, Teresa A.; Smith, Peter G.; Rolfe, Mark</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">3912-3916</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">A review.  E3 ubiquitin ligases regulate many dynamic cellular processes important for cancer cell survival.  Together with ubiquitin-activating enzyme (E1) and ubiquitin-conjugating enzymes (E2s), E3s catalyze the ubiquitination of numerous protein substrates that are subsequently targeted to the 26S proteasome for degrdn.  The clin. success of the proteasome inhibitor bortezomib has encouraged the evaluation of other components of the ubiquitin proteasome system for pharmaceutical intervention.  Targeting specific E3s is particularly attractive because there is the potential to selectively block the degrdn. of certain cellular proteins and possibly avoid unwanted effects on other proteins.  The cullin-RING ubiquitin E3 ligases (CRLs) represent the largest subfamily of E3s.  The requirement that CRLs be activated by NEDD8 modification on the cullin protein offers an "achilles heel" for modulating this entire subfamily.  NEDD8-activating enzyme (NAE) catalyzes the first step in the NEDD8 pathway and as such controls the activity of CRLs.  In this article, we describe the role of the NEDD8 pathway in activating CRLs and discuss the preclin. findings with a first-in-class NAE inhibitor that is currently in phase I clin. trials for both solid tumor and hematol. malignancies.  In addn., we speculate where NAE inhibitors may find clin. utility.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpt7UX7VycsELVg90H21EOLACvtfcHk0lirLWlt1GKMIA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXntlOisrg%253D&md5=cfbe3fc8c8a630e09fc31fe6e8429eb4</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1158%2F1078-0432.CCR-09-0343&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1078-0432.CCR-09-0343%26sid%3Dliteratum%253Aachs%26aulast%3DSoucy%26aufirst%3DT.%2BA.%26aulast%3DSmith%26aufirst%3DP.%2BG.%26aulast%3DRolfe%26aufirst%3DM.%26atitle%3DTargeting%2520nedd8-activated%2520cullin-RING%2520ligases%2520for%2520the%2520treatment%2520of%2520cancer%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2009%26volume%3D15%26spage%3D3912%26epage%3D3916%26doi%3D10.1158%2F1078-0432.CCR-09-0343" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhong, H.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, D. S.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leung, C.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, D.-L.</span></span> <span> </span><span class="NLM_article-title">A metal-based inhibitor of nedd8-activating enzyme</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>7</i></span>, <span class="NLM_elocation-id">e49574</span> <span class="refDoi"> DOI: 10.1371/journal.pone.0049574</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=10.1371%2Fjournal.pone.0049574" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=23185368" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvVSrtr7N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2012&author=H.-J.+Zhongauthor=H.+Yangauthor=D.+S.-H.+Chanauthor=C.-H.+Leungauthor=H.-M.+Wangauthor=D.-L.+Ma&title=A+metal-based+inhibitor+of+nedd8-activating+enzyme&doi=10.1371%2Fjournal.pone.0049574"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">A metal-based inhibitor of NEDD8-activating enzyme</span></div><div class="casAuthors">Zhong, Hai-Jing; Yang, Hui; Chan, Daniel Shiu-Hin; Leung, Chung-Hang; Wang, Hui-Min; Ma, Dik-Lung</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">e49574</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">A cyclometallated rhodium(III) complex [Rh(ppy)2(dppz)]+ (1) (where ppy = 2-phenylpyridine and dppz = dipyrido[3,2-a:2',3'-c]phenazine dipyridophenazine) has been prepd. and identified as an inhibitor of NEDD8-activating enzyme (NAE).  The complex inhibited NAE activity in cell-free and cell-based assays and suppressed the CRL-regulated substrate degrdn. and NF-κB activation in human cancer cells with potency comparable to known NAE inhibitor MLN4924.  Mol. modeling anal. suggested that the overall binding mode of 1 within the binding pocket of the APPBP1/UBA3 heterodimer resembled that for MLN4924.  Complex 1 is the first metal complex reported to suppress the NEDDylation pathway via inhibition of the NEDD8-activating enzyme.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpDnxHVwEANE7Vg90H21EOLACvtfcHk0lirLWlt1GKMIA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvVSrtr7N&md5=e5e4ac7aec1f46138a15e75f121acf10</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0049574&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0049574%26sid%3Dliteratum%253Aachs%26aulast%3DZhong%26aufirst%3DH.-J.%26aulast%3DYang%26aufirst%3DH.%26aulast%3DChan%26aufirst%3DD.%2BS.-H.%26aulast%3DLeung%26aufirst%3DC.-H.%26aulast%3DWang%26aufirst%3DH.-M.%26aulast%3DMa%26aufirst%3DD.-L.%26atitle%3DA%2520metal-based%2520inhibitor%2520of%2520nedd8-activating%2520enzyme%26jtitle%3DPLoS%2520One%26date%3D2012%26volume%3D7%26doi%3D10.1371%2Fjournal.pone.0049574" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Leung, C.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, D. S.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abagyan, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, S. M.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, G.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fong, W.-F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, D.-L.</span></span> <span> </span><span class="NLM_article-title">A natural product-like inhibitor of nedd8-activating enzyme</span>. <i>Chem. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">2511</span>– <span class="NLM_lpage">2513</span>, <span class="refDoi"> DOI: 10.1039/c0cc04927a</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=10.1039%2Fc0cc04927a" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=21240405" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=1%3ACAS%3A528%3ADC%252BC3MXitVCmt7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2011&pages=2511-2513&author=C.-H.+Leungauthor=D.+S.-H.+Chanauthor=H.+Yangauthor=R.+Abagyanauthor=S.+M.-Y.+Leeauthor=G.-Y.+Zhuauthor=W.-F.+Fongauthor=D.-L.+Ma&title=A+natural+product-like+inhibitor+of+nedd8-activating+enzyme&doi=10.1039%2Fc0cc04927a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">A natural product-like inhibitor of NEDD8-activating enzyme</span></div><div class="casAuthors">Leung, Chung-Hang; Chan, Daniel Shiu-Hin; Yang, Hui; Abagyan, Ruben; Lee, Simon Ming-Yuen; Zhu, Guo-Yuan; Fong, Wang-Fun; Ma, Dik-Lung</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Communications (Cambridge, United Kingdom)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2511-2513</span>CODEN:
                <span class="NLM_cas:coden">CHCOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-7345</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">The natural product-like 6,6''-biapigenin has been identified as only the second inhibitor of NEDD8-activating enzyme using virtual screening.  This compd. was active in enzyme and cell-based assays, with potency in the micromolar range.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoIgh-oaWCxuLVg90H21EOLACvtfcHk0ljveDXeNIAA1Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXitVCmt7s%253D&md5=8783a238deaf9306f1bdaca88ab0575e</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1039%2Fc0cc04927a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc0cc04927a%26sid%3Dliteratum%253Aachs%26aulast%3DLeung%26aufirst%3DC.-H.%26aulast%3DChan%26aufirst%3DD.%2BS.-H.%26aulast%3DYang%26aufirst%3DH.%26aulast%3DAbagyan%26aufirst%3DR.%26aulast%3DLee%26aufirst%3DS.%2BM.-Y.%26aulast%3DZhu%26aufirst%3DG.-Y.%26aulast%3DFong%26aufirst%3DW.-F.%26aulast%3DMa%26aufirst%3DD.-L.%26atitle%3DA%2520natural%2520product-like%2520inhibitor%2520of%2520nedd8-activating%2520enzyme%26jtitle%3DChem.%2520Commun.%26date%3D2011%26volume%3D47%26spage%3D2511%26epage%3D2513%26doi%3D10.1039%2Fc0cc04927a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lu, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ouyang, P.-k.</span></span> <span> </span><span class="NLM_article-title">Discovery of a novel Nedd8 activating enzyme inhibitor with piperidin-4-amine scaffold by structure-based virtual screening</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">1901</span>– <span class="NLM_lpage">1907</span>, <span class="refDoi"> DOI: 10.1021/acschembio.6b00159</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.6b00159" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=1%3ACAS%3A528%3ADC%252BC28XmvFKqurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2016&pages=1901-1907&author=P.+Luauthor=X.+Liuauthor=X.+Yuanauthor=M.+Heauthor=Y.+Wangauthor=Q.+Zhangauthor=P.-k.+Ouyang&title=Discovery+of+a+novel+Nedd8+activating+enzyme+inhibitor+with+piperidin-4-amine+scaffold+by+structure-based+virtual+screening&doi=10.1021%2Facschembio.6b00159"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a novel NEDD8 Activating Enzyme Inhibitor with Piperidin-4-amine Scaffold by Structure-Based Virtual Screening</span></div><div class="casAuthors">Lu, Peng; Liu, Xiaoxin; Yuan, Xinrui; He, Minfang; Wang, Yubin; Zhang, Qi; Ouyang, Ping-kai</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1901-1907</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">NEDD8 activating enzyme (NAE) plays an important role in regulating intracellular proteins with key parts in a broad array of cellular functions.  Here, we report a structure-based virtual screening of a compd. library contg. 50000 small mol. entities against the active site of NAE.  Computational docking and scoring followed by biochem. screening and target validation lead to the identification of 1-benzyl-N-(2,4-dichlorophenethyl) piperidin-4-amine (M22) as a selective NAE inhibitor.  M22 is reversible for NAE, inhibits multiple cancer cell lines with GI50 values in the low micromolar range, and induces apoptosis in A549 cells.  Furthermore, it produces tumor inhibition in AGS xenografts in nude mice and low acute toxicity in a zebrafish model.  M22, a novel NAE inhibitor, represents a promising lead structure for the development of new antitumor agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp3wkfzasykYrVg90H21EOLACvtfcHk0ljveDXeNIAA1Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XmvFKqurg%253D&md5=75b21e45e1698ccca8f4ac4a92a0a3b8</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1021%2Facschembio.6b00159&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.6b00159%26sid%3Dliteratum%253Aachs%26aulast%3DLu%26aufirst%3DP.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DYuan%26aufirst%3DX.%26aulast%3DHe%26aufirst%3DM.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DQ.%26aulast%3DOuyang%26aufirst%3DP.-k.%26atitle%3DDiscovery%2520of%2520a%2520novel%2520Nedd8%2520activating%2520enzyme%2520inhibitor%2520with%2520piperidin-4-amine%2520scaffold%2520by%2520structure-based%2520virtual%2520screening%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2016%26volume%3D11%26spage%3D1901%26epage%3D1907%26doi%3D10.1021%2Facschembio.6b00159" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhong, H.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, T.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, D.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leung, C.-H.</span></span> <span> </span><span class="NLM_article-title">A rhodium(iii) complex as an inhibitor of neural precursor cell expressed, developmentally down-regulated 8-activating enzyme with in vivo activity against inflammatory bowel disease</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">497</span>– <span class="NLM_lpage">503</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b00250</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b00250" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=1%3ACAS%3A528%3ADC%252BC28XitV2nurnK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=497-503&author=H.-J.+Zhongauthor=W.+Wangauthor=T.-S.+Kangauthor=H.+Yanauthor=Y.+Yangauthor=L.+Xuauthor=Y.+Wangauthor=D.-L.+Maauthor=C.-H.+Leung&title=A+rhodium%28iii%29+complex+as+an+inhibitor+of+neural+precursor+cell+expressed%2C+developmentally+down-regulated+8-activating+enzyme+with+in+vivo+activity+against+inflammatory+bowel+disease&doi=10.1021%2Facs.jmedchem.6b00250"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">A Rhodium(III) Complex as an Inhibitor of Neural Precursor Cell Expressed, Developmentally Down-Regulated 8-Activating Enzyme with in Vivo Activity against Inflammatory Bowel Disease</span></div><div class="casAuthors">Zhong, Hai-Jing; Wang, Wanhe; Kang, Tian-Shu; Yan, Hui; Yang, Yali; Xu, Lipeng; Wang, Yuqiang; Ma, Dik-Lung; Leung, Chung-Hang</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">497-503</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The authors report herein the identification of the rhodium(III) complex [Rh(phq)2(MOPIP)]+ (1) as a potent and selective ATP-competitive neural precursor cell expressed, developmentally down-regulated 8 (NEDD8)-activating enzyme (NAE) inhibitor.  Structure-activity relationship anal. indicated that the overall organometallic design of complex (1) was important for anti-inflammatory activity.  Complex (1) showed promising anti-inflammatory activity in vivo for the potential treatment of inflammatory bowel disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqOMmNvY-FeErVg90H21EOLACvtfcHk0lgabKQLZ839Aw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitV2nurnK&md5=23c28735fe2477c54a97e21c956f79f5</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b00250&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b00250%26sid%3Dliteratum%253Aachs%26aulast%3DZhong%26aufirst%3DH.-J.%26aulast%3DWang%26aufirst%3DW.%26aulast%3DKang%26aufirst%3DT.-S.%26aulast%3DYan%26aufirst%3DH.%26aulast%3DYang%26aufirst%3DY.%26aulast%3DXu%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DMa%26aufirst%3DD.-L.%26aulast%3DLeung%26aufirst%3DC.-H.%26atitle%3DA%2520rhodium%2528iii%2529%2520complex%2520as%2520an%2520inhibitor%2520of%2520neural%2520precursor%2520cell%2520expressed%252C%2520developmentally%2520down-regulated%25208-activating%2520enzyme%2520with%2520in%2520vivo%2520activity%2520against%2520inflammatory%2520bowel%2520disease%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D497%26epage%3D503%26doi%3D10.1021%2Facs.jmedchem.6b00250" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schlierf, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Altmann, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quancard, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jefferson, A. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Assenberg, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Renatus, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hassiepen, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schaefer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kiffe, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weiss, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wiesmann, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sedrani, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eder, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martoglio, B.</span></span> <span> </span><span class="NLM_article-title">Targeted inhibition of the COP9 signalosome for treatment of cancer</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">13166</span>, <span class="refDoi"> DOI: 10.1038/ncomms13166</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=10.1038%2Fncomms13166" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=27774986" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=1%3ACAS%3A528%3ADC%252BC28XhslGrt77O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=13166&author=A.+Schlierfauthor=E.+Altmannauthor=J.+Quancardauthor=A.+B.+Jeffersonauthor=R.+Assenbergauthor=M.+Renatusauthor=M.+Jonesauthor=U.+Hassiepenauthor=M.+Schaeferauthor=M.+Kiffeauthor=A.+Weissauthor=C.+Wiesmannauthor=R.+Sedraniauthor=J.+Ederauthor=B.+Martoglio&title=Targeted+inhibition+of+the+COP9+signalosome+for+treatment+of+cancer&doi=10.1038%2Fncomms13166"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Targeted inhibition of the COP9 signalosome for treatment of cancer</span></div><div class="casAuthors">Schlierf, Anita; Altmann, Eva; Quancard, Jean; Jefferson, Anne B.; Assenberg, Rene; Renatus, Martin; Jones, Matthew; Hassiepen, Ulrich; Schaefer, Michael; Kiffe, Michael; Weiss, Andreas; Wiesmann, Christian; Sedrani, Richard; Eder, Jorg; Martoglio, Bruno</div><div class="citationInfo"><span class="NLM_cas:title">Nature Communications</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">13166</span>CODEN:
                <span class="NLM_cas:coden">NCAOBW</span>;
        ISSN:<span class="NLM_cas:issn">2041-1723</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The COP9 signalosome (CSN) is a central component of the activation and remodelling cycle of cullin-RING E3 ubiquitin ligases (CRLs), the largest enzyme family of the ubiquitin-proteasome system in humans.  CRLs are implicated in the regulation of numerous cellular processes, including cell cycle progression and apoptosis, and aberrant CRL activity is frequently assocd. with cancer.  Remodelling of CRLs is initiated by CSN-catalyzed cleavage of the ubiquitin-like activator NEDD8 from CRLs.  Here we describe CSN5i-3, a potent, selective and orally available inhibitor of CSN5, the proteolytic subunit of CSN.  The compd. traps CRLs in the neddylated state, which leads to inactivation of a subset of CRLs by inducing degrdn. of their substrate recognition module.  CSN5i-3 differentially affects the viability of tumor cell lines and suppresses growth of a human xenograft in mice.  Our results provide insights into how CSN regulates CRLs and suggest that CSN5 inhibition has potential for anti-tumor therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGojC7MhqQuitLVg90H21EOLACvtfcHk0lgabKQLZ839Aw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhslGrt77O&md5=320de4d96f3201949335c06d613d78ed</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1038%2Fncomms13166&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms13166%26sid%3Dliteratum%253Aachs%26aulast%3DSchlierf%26aufirst%3DA.%26aulast%3DAltmann%26aufirst%3DE.%26aulast%3DQuancard%26aufirst%3DJ.%26aulast%3DJefferson%26aufirst%3DA.%2BB.%26aulast%3DAssenberg%26aufirst%3DR.%26aulast%3DRenatus%26aufirst%3DM.%26aulast%3DJones%26aufirst%3DM.%26aulast%3DHassiepen%26aufirst%3DU.%26aulast%3DSchaefer%26aufirst%3DM.%26aulast%3DKiffe%26aufirst%3DM.%26aulast%3DWeiss%26aufirst%3DA.%26aulast%3DWiesmann%26aufirst%3DC.%26aulast%3DSedrani%26aufirst%3DR.%26aulast%3DEder%26aufirst%3DJ.%26aulast%3DMartoglio%26aufirst%3DB.%26atitle%3DTargeted%2520inhibition%2520of%2520the%2520COP9%2520signalosome%2520for%2520treatment%2520of%2520cancer%26jtitle%3DNat.%2520Commun.%26date%3D2016%26volume%3D7%26spage%3D13166%26doi%3D10.1038%2Fncomms13166" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhong, H.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pui-Yan Ma, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shiu-Hin Chan, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, H.-Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leung, K.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, D.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leung, C.-H.</span></span> <span> </span><span class="NLM_article-title">Discovery of a natural product inhibitor targeting protein neddylation by structure-based virtual screening</span>. <i>Biochimie</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>94</i></span>,  <span class="NLM_fpage">2457</span>– <span class="NLM_lpage">2460</span>, <span class="refDoi"> DOI: 10.1016/j.biochi.2012.06.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=10.1016%2Fj.biochi.2012.06.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=22709868" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtVOqt7fF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=94&publication_year=2012&pages=2457-2460&author=H.-J.+Zhongauthor=V.+Pui-Yan+Maauthor=Z.+Chengauthor=D.+Shiu-Hin+Chanauthor=H.-Z.+Heauthor=K.-H.+Leungauthor=D.-L.+Maauthor=C.-H.+Leung&title=Discovery+of+a+natural+product+inhibitor+targeting+protein+neddylation+by+structure-based+virtual+screening&doi=10.1016%2Fj.biochi.2012.06.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a natural product inhibitor targeting protein neddylation by structure-based virtual screening</span></div><div class="casAuthors">Zhong, Hai-Jing; Ma, Victor Pui-Yan; Cheng, Zhen; Chan, Daniel Shiu-Hin; He, Hong-Zhang; Leung, Ka-Ho; Ma, Dik-Lung; Leung, Chung-Hang</div><div class="citationInfo"><span class="NLM_cas:title">Biochimie</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">94</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2457-2460</span>CODEN:
                <span class="NLM_cas:coden">BICMBE</span>;
        ISSN:<span class="NLM_cas:issn">0300-9084</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">NEDD8-activating enzyme (NAE) controls the specific degrdn. of proteins regulated by cullin-RING ubiquitin E3 ligase, and has been considered as an attractive mol. target for the development of anti-cancer drugs.  We report herein the identification of the dipeptide-conjugated deoxyvasicinone deriv. (1) as an inhibitor of NAE by virtual screening of over 90,000 compds. from the ZINC database of natural products.  Mol. modeling results suggested that 1 may be a non-covalent competitive inhibitor of NAE by blocking the ATP-binding domain.  Compd. 1 was able to inhibit NAE activity in both cell-free and cell-based assay with potencies in the micromolar range and selectivity over analogous E1 enzymes UAE and SAE.  We envisage that the identification and mol. docking anal. of this bioactive scaffold as an NAE inhibitor would provide the scientific community with useful information in order to generate more potent analogs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpKv_G29iM-W7Vg90H21EOLACvtfcHk0lh2DWml8z2YGA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtVOqt7fF&md5=4500cd49c9ef5eb39a969781eaffe312</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1016%2Fj.biochi.2012.06.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.biochi.2012.06.004%26sid%3Dliteratum%253Aachs%26aulast%3DZhong%26aufirst%3DH.-J.%26aulast%3DPui-Yan%2BMa%26aufirst%3DV.%26aulast%3DCheng%26aufirst%3DZ.%26aulast%3DShiu-Hin%2BChan%26aufirst%3DD.%26aulast%3DHe%26aufirst%3DH.-Z.%26aulast%3DLeung%26aufirst%3DK.-H.%26aulast%3DMa%26aufirst%3DD.-L.%26aulast%3DLeung%26aufirst%3DC.-H.%26atitle%3DDiscovery%2520of%2520a%2520natural%2520product%2520inhibitor%2520targeting%2520protein%2520neddylation%2520by%2520structure-based%2520virtual%2520screening%26jtitle%3DBiochimie%26date%3D2012%26volume%3D94%26spage%3D2457%26epage%3D2460%26doi%3D10.1016%2Fj.biochi.2012.06.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ji, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, Y.</span></span> <span> </span><span class="NLM_article-title">Effective virtual screening strategy toward covalent ligands: identification of novel nedd8-activating enzyme inhibitors</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">1785</span>– <span class="NLM_lpage">1797</span>, <span class="refDoi"> DOI: 10.1021/ci5002058</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci5002058" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=1%3ACAS%3A528%3ADC%252BC2cXosFaitLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2014&pages=1785-1797&author=S.+Zhangauthor=J.+Tanauthor=Z.+Laiauthor=Y.+Liauthor=J.+Pangauthor=J.+Xiaoauthor=Z.+Huangauthor=Y.+Zhangauthor=H.+Jiauthor=Y.+Lai&title=Effective+virtual+screening+strategy+toward+covalent+ligands%3A+identification+of+novel+nedd8-activating+enzyme+inhibitors&doi=10.1021%2Fci5002058"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Effective Virtual Screening Strategy toward Covalent Ligands: Identification of Novel NEDD8-Activating Enzyme Inhibitors</span></div><div class="casAuthors">Zhang, Shengping; Tan, Jiani; Lai, Zhonghui; Li, Ying; Pang, Junxia; Xiao, Jianhu; Huang, Zhangjian; Zhang, Yihua; Ji, Hui; Lai, Yisheng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1785-1797</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The NEDD8-activating enzyme (NAE) is an emerging target for cancer therapy, which regulates the degrdn. and turnover of a variety of cancer-related proteins by activating the cullin-RING E3 ubiquitin ligases.  Among a limited no. of known NAE inhibitors, the covalent inhibitors have demonstrated the most potent efficacy through their covalently linked adducts with NEDD8.  Inspired by this unique mechanism, in this study, a novel combined strategy of virtual screening (VS) was adopted with the aim to identify diverse covalent inhibitors of NAE.  To be specific, a docking-enabled pharmacophore model was first built from the possible active conformations of chosen covalent inhibitors.  Meanwhile, a dynamic structure-based phamacophore was also established based on the snapshots derived from mol. dynamic simulation.  Subsequent screening of a focused ZINC database using these pharmacophore models combined with covalent docking discovered three novel active compds.  Among them, compd. LZ3 exhibited the most potent NAE inhibitory activity with an IC50 value of 1.06 ± 0.18 μM.  Furthermore, a cell-based washout expt. proved the proposed covalent binding mechanism for compd. LZ3, which confirmed the successful application of our combined VS strategy, indicating it may provide a viable soln. to systematically discover novel covalent ligands.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqhRJSduwx3CrVg90H21EOLACvtfcHk0lh2DWml8z2YGA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXosFaitLw%253D&md5=07a5cf406764b37d2f68b8aba55649eb</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1021%2Fci5002058&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci5002058%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DS.%26aulast%3DTan%26aufirst%3DJ.%26aulast%3DLai%26aufirst%3DZ.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DPang%26aufirst%3DJ.%26aulast%3DXiao%26aufirst%3DJ.%26aulast%3DHuang%26aufirst%3DZ.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DJi%26aufirst%3DH.%26aulast%3DLai%26aufirst%3DY.%26atitle%3DEffective%2520virtual%2520screening%2520strategy%2520toward%2520covalent%2520ligands%253A%2520identification%2520of%2520novel%2520nedd8-activating%2520enzyme%2520inhibitors%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2014%26volume%3D54%26spage%3D1785%26epage%3D1797%26doi%3D10.1021%2Fci5002058" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Soucy, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, P. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Milhollen, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berger, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gavin, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adhikari, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brownell, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burke, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cardin, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Critchley, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cullis, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doucette, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garnsey, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaulin, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gershman, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lublinsky, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McDonald, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mizutani, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Narayanan, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olhava, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peluso, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rezaei, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sintchak, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Talreja, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Traore, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vyskocil, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weatherhead, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dick, L. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Claiborne, C. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rolfe, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bolen, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Langston, S. P.</span></span> <span> </span><span class="NLM_article-title">An inhibitor of nedd8-activating enzyme as a new approach to treat cancer</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>458</i></span>,  <span class="NLM_fpage">732</span>– <span class="NLM_lpage">736</span>, <span class="refDoi"> DOI: 10.1038/nature07884</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=10.1038%2Fnature07884" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=19360080" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=1%3ACAS%3A528%3ADC%252BD1MXksVenu7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=458&publication_year=2009&pages=732-736&author=T.+A.+Soucyauthor=P.+G.+Smithauthor=M.+A.+Milhollenauthor=A.+J.+Bergerauthor=J.+M.+Gavinauthor=S.+Adhikariauthor=J.+E.+Brownellauthor=K.+E.+Burkeauthor=D.+P.+Cardinauthor=S.+Critchleyauthor=C.+A.+Cullisauthor=A.+Doucetteauthor=J.+J.+Garnseyauthor=J.+L.+Gaulinauthor=R.+E.+Gershmanauthor=A.+R.+Lublinskyauthor=A.+McDonaldauthor=H.+Mizutaniauthor=U.+Narayananauthor=E.+J.+Olhavaauthor=S.+Pelusoauthor=M.+Rezaeiauthor=M.+D.+Sintchakauthor=T.+Talrejaauthor=M.+P.+Thomasauthor=T.+Traoreauthor=S.+Vyskocilauthor=G.+S.+Weatherheadauthor=J.+Yuauthor=J.+Zhangauthor=L.+R.+Dickauthor=C.+F.+Claiborneauthor=M.+Rolfeauthor=J.+B.+Bolenauthor=S.+P.+Langston&title=An+inhibitor+of+nedd8-activating+enzyme+as+a+new+approach+to+treat+cancer&doi=10.1038%2Fnature07884"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">An inhibitor of NEDD8-activating enzyme as a new approach to treat cancer</span></div><div class="casAuthors">Soucy, Teresa A.; Smith, Peter G.; Milhollen, Michael A.; Berger, Allison J.; Gavin, James M.; Adhikari, Sharmila; Brownell, James E.; Burke, Kristine E.; Cardin, David P.; Critchley, Stephen; Cullis, Courtney A.; Doucette, Amanda; Garnsey, James J.; Gaulin, Jeffrey L.; Gershman, Rachel E.; Lublinsky, Anna R.; McDonald, Alice; Mizutani, Hirotake; Narayanan, Usha; Olhava, Edward J.; Peluso, Stephane; Rezaei, Mansoureh; Sintchak, Michael D.; Talreja, Tina; Thomas, Michael P.; Traore, Tary; Vyskocil, Stepan; Weatherhead, Gabriel S.; Yu, Jie; Zhang, Julie; Dick, Lawrence R.; Claiborne, Christopher F.; Rolfe, Mark; Bolen, Joseph B.; Langston, Steven P.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">458</span>
        (<span class="NLM_cas:issue">7239</span>),
    <span class="NLM_cas:pages">732-736</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The clin. development of an inhibitor of cellular proteasome function suggests that compds. targeting other components of the ubiquitin-proteasome system might prove useful for the treatment of human malignancies.  NEDD8-activating enzyme (NAE) is an essential component of the NEDD8 conjugation pathway that controls the activity of the cullin-RING subtype of ubiquitin ligases, thereby regulating the turnover of a subset of proteins upstream of the proteasome.  Substrates of cullin-RING ligases have important roles in cellular processes assocd. with cancer cell growth and survival pathways.  Here we describe MLN4924, a potent and selective inhibitor of NAE.  MLN4924 disrupts cullin-RING ligase-mediated protein turnover leading to apoptotic death in human tumor cells by a new mechanism of action, the deregulation of S-phase DNA synthesis.  MLN4924 suppressed the growth of human tumor xenografts in mice at compd. exposures that were well tolerated.  Our data suggest that NAE inhibitors may hold promise for the treatment of cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr8fx7OnZtYjbVg90H21EOLACvtfcHk0lirYXEbjdiodg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXksVenu7Y%253D&md5=365d4b4feac3994849050e00f1aae5e0</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1038%2Fnature07884&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature07884%26sid%3Dliteratum%253Aachs%26aulast%3DSoucy%26aufirst%3DT.%2BA.%26aulast%3DSmith%26aufirst%3DP.%2BG.%26aulast%3DMilhollen%26aufirst%3DM.%2BA.%26aulast%3DBerger%26aufirst%3DA.%2BJ.%26aulast%3DGavin%26aufirst%3DJ.%2BM.%26aulast%3DAdhikari%26aufirst%3DS.%26aulast%3DBrownell%26aufirst%3DJ.%2BE.%26aulast%3DBurke%26aufirst%3DK.%2BE.%26aulast%3DCardin%26aufirst%3DD.%2BP.%26aulast%3DCritchley%26aufirst%3DS.%26aulast%3DCullis%26aufirst%3DC.%2BA.%26aulast%3DDoucette%26aufirst%3DA.%26aulast%3DGarnsey%26aufirst%3DJ.%2BJ.%26aulast%3DGaulin%26aufirst%3DJ.%2BL.%26aulast%3DGershman%26aufirst%3DR.%2BE.%26aulast%3DLublinsky%26aufirst%3DA.%2BR.%26aulast%3DMcDonald%26aufirst%3DA.%26aulast%3DMizutani%26aufirst%3DH.%26aulast%3DNarayanan%26aufirst%3DU.%26aulast%3DOlhava%26aufirst%3DE.%2BJ.%26aulast%3DPeluso%26aufirst%3DS.%26aulast%3DRezaei%26aufirst%3DM.%26aulast%3DSintchak%26aufirst%3DM.%2BD.%26aulast%3DTalreja%26aufirst%3DT.%26aulast%3DThomas%26aufirst%3DM.%2BP.%26aulast%3DTraore%26aufirst%3DT.%26aulast%3DVyskocil%26aufirst%3DS.%26aulast%3DWeatherhead%26aufirst%3DG.%2BS.%26aulast%3DYu%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DDick%26aufirst%3DL.%2BR.%26aulast%3DClaiborne%26aufirst%3DC.%2BF.%26aulast%3DRolfe%26aufirst%3DM.%26aulast%3DBolen%26aufirst%3DJ.%2BB.%26aulast%3DLangston%26aufirst%3DS.%2BP.%26atitle%3DAn%2520inhibitor%2520of%2520nedd8-activating%2520enzyme%2520as%2520a%2520new%2520approach%2520to%2520treat%2520cancer%26jtitle%3DNature%26date%3D2009%26volume%3D458%26spage%3D732%26epage%3D736%26doi%3D10.1038%2Fnature07884" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bhatia, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pavlick, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boasberg, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mulligan, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pickard, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Faessel, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dezube, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamid, O.</span></span> <span> </span><span class="NLM_article-title">A phase I study of the investigational NEDD8-activating enzyme inhibitor pevonedistat (TAK-924/MLN4924) in patients with metastatic melanoma</span>. <i>Invest. New Drugs</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>34</i></span>,  <span class="NLM_fpage">439</span>– <span class="NLM_lpage">449</span>, <span class="refDoi"> DOI: 10.1007/s10637-016-0348-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=10.1007%2Fs10637-016-0348-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=27056178" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=1%3ACAS%3A528%3ADC%252BC28XlvV2gs7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=2016&pages=439-449&author=S.+Bhatiaauthor=A.+C.+Pavlickauthor=P.+Boasbergauthor=J.+A.+Thompsonauthor=G.+Mulliganauthor=M.+D.+Pickardauthor=H.+Faesselauthor=B.+J.+Dezubeauthor=O.+Hamid&title=A+phase+I+study+of+the+investigational+NEDD8-activating+enzyme+inhibitor+pevonedistat+%28TAK-924%2FMLN4924%29+in+patients+with+metastatic+melanoma&doi=10.1007%2Fs10637-016-0348-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">A phase I study of the investigational NEDD8-activating enzyme inhibitor pevonedistat (TAK-924/MLN4924) in patients with metastatic melanoma</span></div><div class="casAuthors">Bhatia, Shailender; Pavlick, Anna C.; Boasberg, Peter; Thompson, John A.; Mulligan, George; Pickard, Michael D.; Faessel, Helene; Dezube, Bruce J.; Hamid, Omid</div><div class="citationInfo"><span class="NLM_cas:title">Investigational New Drugs</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">34</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">439-449</span>CODEN:
                <span class="NLM_cas:coden">INNDDK</span>;
        ISSN:<span class="NLM_cas:issn">0167-6997</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Purpose The therapeutic index of proteasome inhibitors may be improved through selective inhibition of a sub-component of the ubiquitin-proteasome system, such as the NEDD8-conjugation pathway.  This multicenter, phase I, dose-escalation study assessed safety and the max. tolerated dose (MTD), pharmacokinetics, pharmacodynamics, and antitumor activity of pevonedistat, an investigational NEDD8-activating enzyme (NAE) inhibitor, in patients with metastatic melanoma.  Methods Patients received i.v. pevonedistat on Days 1, 4, 8, 11 (schedule A) or 1, 8, 15 (schedule B) of 21-day cycles.  Results 26 patients received pevonedistat 50-278 mg/m2 on schedule A; 11 patients received pevonedistat 157 mg/m2 on schedule B.  The schedule A MTD was 209 mg/m2: dose-limiting toxicities (DLTs) included grade 3 hypophosphatemia and grade 3 increased blood creatinine (assocd. with grade 3 hyperbilirubinemia).  Two schedule A patients experienced acute organ failure toxicities, one of whom experienced grade 5 acute renal failure.  Dose escalation did not occur in schedule B: DLTs included grade 3 myocarditis, grade 2 acute renal failure, and grade 2 hyperbilirubinemia in a single patient.  Pevonedistat pharmacokinetics were approx. dose-proportional across the dose range studied, with a biphasic disposition profile characterized by a short elimination half-life (∼10 h).  Pharmacodynamic studies showed increases in NAE-regulated transcripts post-treatment; all post-dose biopsy samples were pos. for pevonedistat-NEDD8 adduct.  One schedule A patient achieved a partial response; 15 patients had stable disease (4 lasting ≥6.5 mo).  Conclusions Pevonedistat was generally well tolerated at the MTD.  Anticipated pharmacodynamic effects of NAE inhibition were obsd. with single-agent pevonedistat in peripheral blood and tumor tissue.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr_SdAW55A9BbVg90H21EOLACvtfcHk0lirYXEbjdiodg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XlvV2gs7Y%253D&md5=9a8a889697053b5e5a7a9f01e0f70f5a</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1007%2Fs10637-016-0348-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10637-016-0348-5%26sid%3Dliteratum%253Aachs%26aulast%3DBhatia%26aufirst%3DS.%26aulast%3DPavlick%26aufirst%3DA.%2BC.%26aulast%3DBoasberg%26aufirst%3DP.%26aulast%3DThompson%26aufirst%3DJ.%2BA.%26aulast%3DMulligan%26aufirst%3DG.%26aulast%3DPickard%26aufirst%3DM.%2BD.%26aulast%3DFaessel%26aufirst%3DH.%26aulast%3DDezube%26aufirst%3DB.%2BJ.%26aulast%3DHamid%26aufirst%3DO.%26atitle%3DA%2520phase%2520I%2520study%2520of%2520the%2520investigational%2520NEDD8-activating%2520enzyme%2520inhibitor%2520pevonedistat%2520%2528TAK-924%252FMLN4924%2529%2520in%2520patients%2520with%2520metastatic%2520melanoma%26jtitle%3DInvest.%2520New%2520Drugs%26date%3D2016%26volume%3D34%26spage%3D439%26epage%3D449%26doi%3D10.1007%2Fs10637-016-0348-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Monda, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lydeard, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">King, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harper, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bennett, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schulman, B. A.</span></span> <span> </span><span class="NLM_article-title">Structural conservation of distinctive N-terminal acetylation-dependent interactions across a family of mammalian nedd8 ligation enzymes</span>. <i>Structure</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">42</span>– <span class="NLM_lpage">53</span>, <span class="refDoi"> DOI: 10.1016/j.str.2012.10.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=10.1016%2Fj.str.2012.10.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=23201271" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=1%3ACAS%3A528%3ADC%252BC38XhslKqtrjL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2013&pages=42-53&author=J.+K.+Mondaauthor=D.+C.+Scottauthor=D.+J.+Millerauthor=J.+Lydeardauthor=D.+Kingauthor=J.+W.+Harperauthor=E.+J.+Bennettauthor=B.+A.+Schulman&title=Structural+conservation+of+distinctive+N-terminal+acetylation-dependent+interactions+across+a+family+of+mammalian+nedd8+ligation+enzymes&doi=10.1016%2Fj.str.2012.10.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Structural Conservation of Distinctive N-terminal Acetylation-Dependent Interactions across a Family of Mammalian NEDD8 Ligation Enzymes</span></div><div class="casAuthors">Monda, Julie K.; Scott, Daniel C.; Miller, Darcie J.; Lydeard, John; King, David; Harper, J. Wade; Bennett, Eric J.; Schulman, Brenda A.</div><div class="citationInfo"><span class="NLM_cas:title">Structure (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">42-53</span>CODEN:
                <span class="NLM_cas:coden">STRUE6</span>;
        ISSN:<span class="NLM_cas:issn">0969-2126</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Little is known about mol. recognition of acetylated N termini, despite prevalence of this modification among eukaryotic cytosolic proteins.  We report that the family of human DCN-like (DCNL) co-E3s, which promote ligation of the ubiquitin-like protein NEDD8 to cullin targets, recognizes acetylated N termini of the E2 enzymes UBC12 and UBE2F.  Systematic biochem. and biophys. analyses reveal 40- and 10-fold variations in affinities among different DCNL-cullin and DCNL-E2 complexes, contributing to varying efficiencies of different NEDD8 ligation cascades.  Structures of DCNL2 and DCNL3 complexes with N-terminally acetylated peptides from UBC12 and UBE2F illuminate a common mechanism by which DCNL proteins recognize N-terminally acetylated E2s and how selectivity for interactions dependent on N-acetyl-methionine are established through side chains recognizing distal residues.  Distinct preferences of UBC12 and UBE2F peptides for inhibiting different DCNLs, including the oncogenic DCNL1 protein, suggest it may be possible to develop small mols. blocking specific N-acetyl-methionine-dependent protein interactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrlyp13hD0rNbVg90H21EOLACvtfcHk0ljzGYRAMDjjuQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhslKqtrjL&md5=98e00537298d81b591d87e8e135e9c3a</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1016%2Fj.str.2012.10.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.str.2012.10.013%26sid%3Dliteratum%253Aachs%26aulast%3DMonda%26aufirst%3DJ.%2BK.%26aulast%3DScott%26aufirst%3DD.%2BC.%26aulast%3DMiller%26aufirst%3DD.%2BJ.%26aulast%3DLydeard%26aufirst%3DJ.%26aulast%3DKing%26aufirst%3DD.%26aulast%3DHarper%26aufirst%3DJ.%2BW.%26aulast%3DBennett%26aufirst%3DE.%2BJ.%26aulast%3DSchulman%26aufirst%3DB.%2BA.%26atitle%3DStructural%2520conservation%2520of%2520distinctive%2520N-terminal%2520acetylation-dependent%2520interactions%2520across%2520a%2520family%2520of%2520mammalian%2520nedd8%2520ligation%2520enzymes%26jtitle%3DStructure%26date%3D2013%26volume%3D21%26spage%3D42%26epage%3D53%26doi%3D10.1016%2Fj.str.2012.10.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sarkaria, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O-charoenrat, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Talbot, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reddy, P. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ngai, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maghami, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, K. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yonekawa, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dudas, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaufman, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ryan, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ghossein, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rao, P. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stoffel, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramanathan, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, B.</span></span> <span> </span><span class="NLM_article-title">Squamous cell carcinoma related oncogene/DCUN1D1 is highly conserved and activated by amplification in squamous cell carcinomas</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">9437</span>– <span class="NLM_lpage">9444</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.CAN-06-2074</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=10.1158%2F0008-5472.CAN-06-2074" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=17018598" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtVamsrjE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2006&pages=9437-9444&author=I.+Sarkariaauthor=P.+O-charoenratauthor=S.+G.+Talbotauthor=P.+G.+Reddyauthor=I.+Ngaiauthor=E.+Maghamiauthor=K.+N.+Patelauthor=B.+Leeauthor=Y.+Yonekawaauthor=M.+Dudasauthor=A.+Kaufmanauthor=R.+Ryanauthor=R.+Ghosseinauthor=P.+H.+Raoauthor=A.+Stoffelauthor=Y.+Ramanathanauthor=B.+Singh&title=Squamous+cell+carcinoma+related+oncogene%2FDCUN1D1+is+highly+conserved+and+activated+by+amplification+in+squamous+cell+carcinomas&doi=10.1158%2F0008-5472.CAN-06-2074"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Squamous Cell Carcinoma Related Oncogene/DCUN1D1 Is Highly Conserved and Activated by Amplification in Squamous Cell Carcinomas</span></div><div class="casAuthors">Sarkaria, Inderpal; O-charoenrat, Pornchai; Talbot, Simon G.; Reddy, Pabbathi G.; Ngai, Ivan; Maghami, Ellie; Patel, Kepal N.; Lee, Benjamin; Yonekawa, Yoshihiro; Dudas, Maria; Kaufman, Andrew; Ryan, Russell; Ghossein, Ronald; Rao, Pulivarthi H.; Stoffel, Archontoula; Ramanathan, Y.; Singh, Bhuvanesh</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">9437-9444</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Chromosomal amplification at 3q is common to multiple human cancers, but has a specific predilection for squamous cell carcinomas (SCC) of mucosal origin.  We identified and characterized a novel oncogene, SCC-related oncogene (SCCRO), which is amplified along the 3q26.3 region in human SCC.  Amplification and overexpression of SCCRO in these tumors correlate with poor clin. outcome.  The importance of SCCRO amplification in malignant transformation is established by the apoptotic response to short hairpin RNA against SCCRO, exclusively in cancer cell lines carrying SCCRO amplification.  The oncogenic potential of SCCRO is underscored by its ability to transform fibroblasts (NIH-3T3 cells) in vitro and in vivo.  We show that SCCRO regulates Gli1-a key regulator of the hedgehog (HH) pathway.  Collectively, these data suggest that SCCRO is a novel component of the HH signaling pathway involved in the malignant transformation of squamous cell lineage.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrzTIFIhphYXrVg90H21EOLACvtfcHk0ljzGYRAMDjjuQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtVamsrjE&md5=359faefb6b6305279f3df1ec54a544f7</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-06-2074&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-06-2074%26sid%3Dliteratum%253Aachs%26aulast%3DSarkaria%26aufirst%3DI.%26aulast%3DO-charoenrat%26aufirst%3DP.%26aulast%3DTalbot%26aufirst%3DS.%2BG.%26aulast%3DReddy%26aufirst%3DP.%2BG.%26aulast%3DNgai%26aufirst%3DI.%26aulast%3DMaghami%26aufirst%3DE.%26aulast%3DPatel%26aufirst%3DK.%2BN.%26aulast%3DLee%26aufirst%3DB.%26aulast%3DYonekawa%26aufirst%3DY.%26aulast%3DDudas%26aufirst%3DM.%26aulast%3DKaufman%26aufirst%3DA.%26aulast%3DRyan%26aufirst%3DR.%26aulast%3DGhossein%26aufirst%3DR.%26aulast%3DRao%26aufirst%3DP.%2BH.%26aulast%3DStoffel%26aufirst%3DA.%26aulast%3DRamanathan%26aufirst%3DY.%26aulast%3DSingh%26aufirst%3DB.%26atitle%3DSquamous%2520cell%2520carcinoma%2520related%2520oncogene%252FDCUN1D1%2520is%2520highly%2520conserved%2520and%2520activated%2520by%2520amplification%2520in%2520squamous%2520cell%2520carcinomas%26jtitle%3DCancer%2520Res.%26date%3D2006%26volume%3D66%26spage%3D9437%26epage%3D9444%26doi%3D10.1158%2F0008-5472.CAN-06-2074" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Estilo, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O-charoenrat, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ngai, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reddy, P. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dao, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaha, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kraus, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boyle, J. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pfister, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huryn, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zlotolow, I. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shah, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, B.</span></span> <span> </span><span class="NLM_article-title">The role of novel oncogenes squamous cell carcinoma-related oncogene and phosphatidylinositol 3-kinase p110α in squamous cell carcinoma of the oral tongue</span>. <i>Clin. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">2300</span>– <span class="NLM_lpage">2306</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=12796399" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=1%3ACAS%3A528%3ADC%252BD3sXksVajsrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2003&pages=2300-2306&author=C.+L.+Estiloauthor=P.+O-charoenratauthor=I.+Ngaiauthor=S.+G.+Patelauthor=P.+G.+Reddyauthor=S.+Daoauthor=A.+R.+Shahaauthor=D.+H.+Krausauthor=J.+O.+Boyleauthor=R.+J.+Wongauthor=D.+G.+Pfisterauthor=J.+M.+Hurynauthor=I.+M.+Zlotolowauthor=J.+P.+Shahauthor=B.+Singh&title=The+role+of+novel+oncogenes+squamous+cell+carcinoma-related+oncogene+and+phosphatidylinositol+3-kinase+p110%CE%B1+in+squamous+cell+carcinoma+of+the+oral+tongue"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">The Role of Novel Oncogenes Squamous Cell Carcinoma-related Oncogene and Phosphatidylinositol 3-Kinase p110α in Squamous Cell Carcinoma of the Oral Tongue</span></div><div class="casAuthors">Estilo, Cherry L.; O-Charoenrat, Pornchai; Ngai, Ivan; Patel, Snehal G.; Reddy, Pabbathi G.; Dao, Su; Shaha, Ashok R.; Kraus, Dennis H.; Boyle, Jay O.; Wong, Richard J.; Pfister, David G.; Huryn, Joseph M.; Zlotolow, Ian M.; Shah, Jatin P.; Singh, Bhuvanesh</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2300-2306</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Amplification at chromosome 3q26.3 is a common and crucial event in head and neck squamous cell carcinoma (HNSCC), impacting significantly on tumor progression and clin. outcome.  Two novel oncogenes, namely squamous cell carcinoma (SCC)-related oncogene (SCCRO) and PIK3CA (gene encoding phosphatidylinositol-3 kinase catalytic α-polypeptide), have been identified as targets of 3q26.3 amplification.  This study aimed to delineate the role of SCCRO and PIK3CA in the pathogenesis of oral tongue SCC.  The assocn. between gene copy no. for SCCRO and PIK3CA measured by fluorescence in situ hybridization and level of mRNA expression quantitated by real-time reverse transcription-PCR was assessed in a panel of human HNSCC cell lines.  In addn., gene expression in 49 consecutive primary SCCs of the oral tongue was detd. and correlated with clinicopathol. characteristics and outcome.  The mRNA level of SCCRO and PIK3CA was significantly correlated to the gene copy no. in nine HNSCC cell lines.  In addn., the expression level of SCCRO and PIK3CA was significantly greater in malignant tissues compared with those in histol. normal mucosae (2.17- and 2.46-fold, resp.).  Matched tumor normal control anal. revealed that 24.5 and 69.4% of patients expressed high levels of SCCRO and PIK3CA, resp.  Univariate analyses demonstrated that SCCRO overexpression correlated with nodal metastases and advanced stage, whereas PIK3CA overexpression was assocd. with vascular invasion.  Only SCCRO overexpression was assocd. with disease-specific and overall survival.  Furthermore, SCCRO overexpression remained an independent predictor for cervical nodal metastasis on multivariate regression anal. (χ2 likelihood ratio = 4.38).  Although both SCCRO and PIK3CA may play a role in the pathogenesis of oral tongue SCC through amplification at 3q26, SCCRO appears to be a significant predictor of regional metastasis and may be a marker for tumor aggressiveness and clin. outcome.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpMP7JFlxFFtLVg90H21EOLACvtfcHk0lhdKQG3DqVdpQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXksVajsrc%253D&md5=ccdd010cd52d1e198259a2edddf93ff7</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DEstilo%26aufirst%3DC.%2BL.%26aulast%3DO-charoenrat%26aufirst%3DP.%26aulast%3DNgai%26aufirst%3DI.%26aulast%3DPatel%26aufirst%3DS.%2BG.%26aulast%3DReddy%26aufirst%3DP.%2BG.%26aulast%3DDao%26aufirst%3DS.%26aulast%3DShaha%26aufirst%3DA.%2BR.%26aulast%3DKraus%26aufirst%3DD.%2BH.%26aulast%3DBoyle%26aufirst%3DJ.%2BO.%26aulast%3DWong%26aufirst%3DR.%2BJ.%26aulast%3DPfister%26aufirst%3DD.%2BG.%26aulast%3DHuryn%26aufirst%3DJ.%2BM.%26aulast%3DZlotolow%26aufirst%3DI.%2BM.%26aulast%3DShah%26aufirst%3DJ.%2BP.%26aulast%3DSingh%26aufirst%3DB.%26atitle%3DThe%2520role%2520of%2520novel%2520oncogenes%2520squamous%2520cell%2520carcinoma-related%2520oncogene%2520and%2520phosphatidylinositol%25203-kinase%2520p110%25CE%25B1%2520in%2520squamous%2520cell%2520carcinoma%2520of%2520the%2520oral%2520tongue%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2003%26volume%3D9%26spage%3D2300%26epage%3D2306" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Broderick, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Golas, Bj</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pham, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Towe, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Talbot, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaufman, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bains, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huryn, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yonekawa, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carlson, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hambardzumyan, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramanathan, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, B.</span></span> <span> </span><span class="NLM_article-title">SCCRO promotes glioma formation and malignant progression in mice</span>. <i>Neoplasia</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">476</span>– <span class="NLM_lpage">484</span>, <span class="refDoi"> DOI: 10.1593/neo.10202</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=10.1593%2Fneo.10202" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=20563250" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=1%3ACAS%3A528%3ADC%252BC3cXntFWjtb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2010&pages=476-484&author=S.+R.+Broderickauthor=Bj+Golasauthor=D.+Phamauthor=C.+W.+Toweauthor=S.+G.+Talbotauthor=A.+Kaufmanauthor=S.+Bainsauthor=L.+A.+Hurynauthor=Y.+Yonekawaauthor=D.+Carlsonauthor=D.+Hambardzumyanauthor=Y.+Ramanathanauthor=B.+Singh&title=SCCRO+promotes+glioma+formation+and+malignant+progression+in+mice&doi=10.1593%2Fneo.10202"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">SCCRO promotes glioma formation and malignant progression in mice</span></div><div class="casAuthors">Broderick, Stephen R.; Golas, Benjamin J.; Pham, Duykhanh; Towe, Christopher W.; Talbot, Simon G.; Kaufman, Andrew; Bains, Sarina; Huryn, Laryssa A.; Yonekawa, Yoshihiro; Carlson, Diane; Hambardzumyan, Dolores; Ramanathan, Yegnanarayana; Singh, Bhuvanesh</div><div class="citationInfo"><span class="NLM_cas:title">Neoplasia (Ann Arbor, MI, United States)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">476-484</span>CODEN:
                <span class="NLM_cas:coden">NEOPFL</span>;
        ISSN:<span class="NLM_cas:issn">1522-8002</span>.
    
            (<span class="NLM_cas:orgname">Neoplasia Press Inc.</span>)
        </div><div class="casAbstract">Originally identified as an oncogene activated by amplification in squamous cell carcinomas, several lines of evidence now suggest that squamous cell carcinoma-related oncogene (SCCRO; aka DCUN1D1) may play a role in the pathogenesis of a wide range of human cancers including gliomas.  SCCRO's oncogenic function is substantiated by its ectopic expression, resulting in transformation of cells in culture and xenograft formation in nude mice.  The aim of this study was to assess the in vivo oncogenicity of SCCRO in a murine model.  Ubiquitous expression of SCCRO resulted in early embryonic lethality.  Because SCCRO overexpression was detected in human gliomas, its in vivo oncogenic activity was assessed in an established murine glioma model.  Conditional expression of SCCRO using a replication-competent ASLV long terminal repeat with splice acceptor/nestin-(tumor virus-A) tv-a model system was not sufficient to induce tumor formation in a wild-type genetic background, but tumors formed with increasing frequency and decreasing latency in facilitated background contg. Ink4a deletion alone or in combination with PTEN loss.  Ectopic expression of SCCRO in glial progenitor cells resulted in lower-grade gliomas in Ink4a-/- mice, whereas its expression in Ink4a-/-/PTEN-/- background produced high-grade glioblastoma-like lesions that were indistinguishable from human tumors.  Expression of SCCRO with platelet-derived growth factor-beta (PDGF-β) resulted in an increased proportion of mice forming glioblastoma-like tumors compared with those induced by PDGF-β alone.  This work substantiates SCCRO's function as an oncogene by showing its ability to facilitate malignant transformation and carcinogenic progression in vivo and supports a role for SCCRO in the pathogenesis of gliomas and other human cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpSMNrn3xVDKrVg90H21EOLACvtfcHk0lhdKQG3DqVdpQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXntFWjtb0%253D&md5=22114aee39c7f21d5149880d2fbd99f2</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1593%2Fneo.10202&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1593%252Fneo.10202%26sid%3Dliteratum%253Aachs%26aulast%3DBroderick%26aufirst%3DS.%2BR.%26aulast%3DGolas%26aufirst%3DBj%26aulast%3DPham%26aufirst%3DD.%26aulast%3DTowe%26aufirst%3DC.%2BW.%26aulast%3DTalbot%26aufirst%3DS.%2BG.%26aulast%3DKaufman%26aufirst%3DA.%26aulast%3DBains%26aufirst%3DS.%26aulast%3DHuryn%26aufirst%3DL.%2BA.%26aulast%3DYonekawa%26aufirst%3DY.%26aulast%3DCarlson%26aufirst%3DD.%26aulast%3DHambardzumyan%26aufirst%3DD.%26aulast%3DRamanathan%26aufirst%3DY.%26aulast%3DSingh%26aufirst%3DB.%26atitle%3DSCCRO%2520promotes%2520glioma%2520formation%2520and%2520malignant%2520progression%2520in%2520mice%26jtitle%3DNeoplasia%26date%3D2010%26volume%3D12%26spage%3D476%26epage%3D484%26doi%3D10.1593%2Fneo.10202" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yoo, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, S.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lym, K.-I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, S. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, S-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoo, C.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jung, J.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, S.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, C.-S.</span></span> <span> </span><span class="NLM_article-title">Immunohistochemical expression of DCUN1D1 in non-small cell lung carcinoma: its relation to brain metastasis</span>. <i>Cancer Res. Treat.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>44</i></span>,  <span class="NLM_fpage">57</span>– <span class="NLM_lpage">62</span>, <span class="refDoi"> DOI: 10.4143/crt.2012.44.1.57</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=10.4143%2Fcrt.2012.44.1.57" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=22500162" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=1%3ACAS%3A280%3ADC%252BC38rkslChtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2012&pages=57-62&author=J.+Yooauthor=S.-H.+Leeauthor=K.-I.+Lymauthor=S.+Y.+Parkauthor=S-H.+Yangauthor=C.-Y.+Yooauthor=J.-H.+Jungauthor=S.-J.+Kangauthor=C.-S.+Kang&title=Immunohistochemical+expression+of+DCUN1D1+in+non-small+cell+lung+carcinoma%3A+its+relation+to+brain+metastasis&doi=10.4143%2Fcrt.2012.44.1.57"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Immunohistochemical Expression of DCUN1D1 in Non-small Cell Lung Carcinoma: Its Relation to Brain Metastasis</span></div><div class="casAuthors">Yoo Jinyoung; Lee Seong-Hak; Lym Kwang-Il; Park Sonya Youngju; Yang Seung-Ho; Yoo Chang-Young; Jung Ji-Han; Kang Seok-Jin; Kang Chang-Suk</div><div class="citationInfo"><span class="NLM_cas:title">Cancer research and treatment : official journal of Korean Cancer Association</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">57-62</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">PURPOSE:  Non-small cell lung carcinoma (NSCLC) comprises 75-85% of all lung cancers, and approximately 25% of all NSCLC patients develop brain metastasis.  There are no reliable markers for predicting in which patients this metastasis will occur.  DCUN1D1, also known as squamous cell carcinoma-related oncogene, is associated with tumor progression and poor outcomes in NSCLC.  The objective of this study was to investigate the role of DCUN1D1 expression in cases of brain metastasis due to NSCLC.  MATERIALS AND METHODS:  Primary tumor samples from a total of 71 cases of NSCLC, either with (n=40) or without (n=31) brain metastasis, were evaluated for DCUN1D1 expression by immunohistochemistry analysis.  RESULTS:  DCUN1D1 expression was detected in 16 patients (23%) and tended to correlate with T classification (15% of T1-2 tumors vs. 30% of T3-4 tumors, p=0.083).  DCUN1D1 expression was significantly associated with tumor stage.  It was observed in none of the patients with stage I disease, 10% of those with stage II disease, and 29% with stage III disease (p=0.009).  In addition, 14 of 16 DCUN1D1-positive patients resulted in brain metastasis (p=0.01).  The odds ratio of brain metastasis for patients with DCUN1D1 expression was 3.112 (p=0.009).  CONCLUSION:  DCUN1D1 expression may play a role in tumor progression and development of brain metastasis in patients with NSCLC.  Evaluation of DCUN1D1 expression may provide assistance in identifying those patients who are at higher risk for brain metastasis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQCEHsxuDVeOuIHSmpohpd0fW6udTcc2eboyL8FMxE9nLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC38rkslChtQ%253D%253D&md5=408e62c5272bb8a82a26ce06de38145b</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.4143%2Fcrt.2012.44.1.57&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4143%252Fcrt.2012.44.1.57%26sid%3Dliteratum%253Aachs%26aulast%3DYoo%26aufirst%3DJ.%26aulast%3DLee%26aufirst%3DS.-H.%26aulast%3DLym%26aufirst%3DK.-I.%26aulast%3DPark%26aufirst%3DS.%2BY.%26aulast%3DYang%26aufirst%3DS-H.%26aulast%3DYoo%26aufirst%3DC.-Y.%26aulast%3DJung%26aufirst%3DJ.-H.%26aulast%3DKang%26aufirst%3DS.-J.%26aulast%3DKang%26aufirst%3DC.-S.%26atitle%3DImmunohistochemical%2520expression%2520of%2520DCUN1D1%2520in%2520non-small%2520cell%2520lung%2520carcinoma%253A%2520its%2520relation%2520to%2520brain%2520metastasis%26jtitle%3DCancer%2520Res.%2520Treat.%26date%3D2012%26volume%3D44%26spage%3D57%26epage%3D62%26doi%3D10.4143%2Fcrt.2012.44.1.57" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Scott, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hammill, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Min, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rhee, D. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Connelly, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sviderskiy, V. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhasin, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ong, S.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chai, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goktug, A. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Monda, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Low, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paulo, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cannon, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shelat, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kelsall, I. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alpi, A. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pagala, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harper, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schulman, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guy, R. K.</span></span> <span> </span><span class="NLM_article-title">Blocking an N-terminal acetylation–dependent protein interaction inhibits an E3 ligase</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">850</span>– <span class="NLM_lpage">857</span>, <span class="refDoi"> DOI: 10.1038/nchembio.2386</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=10.1038%2Fnchembio.2386" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=28581483" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=1%3ACAS%3A528%3ADC%252BC2sXpt1Omt7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2017&pages=850-857&author=D.+C.+Scottauthor=J.+T.+Hammillauthor=J.+Minauthor=D.+Y.+Rheeauthor=M.+Connellyauthor=V.+O.+Sviderskiyauthor=D.+Bhasinauthor=Y.+Chenauthor=S.-S.+Ongauthor=S.+C.+Chaiauthor=A.+N.+Goktugauthor=G.+Huangauthor=J.+K.+Mondaauthor=J.+Lowauthor=H.+S.+Kimauthor=J.+A.+Pauloauthor=J.+R.+Cannonauthor=A.+A.+Shelatauthor=T.+Chenauthor=I.+R.+Kelsallauthor=A.+F.+Alpiauthor=V.+Pagalaauthor=X.+Wangauthor=J.+Pengauthor=B.+Singhauthor=J.+W.+Harperauthor=B.+A.+Schulmanauthor=R.+K.+Guy&title=Blocking+an+N-terminal+acetylation%E2%80%93dependent+protein+interaction+inhibits+an+E3+ligase&doi=10.1038%2Fnchembio.2386"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Blocking an N-terminal acetylation-dependent protein interaction inhibits an E3 ligase</span></div><div class="casAuthors">Scott, Daniel C.; Hammill, Jared T.; Min, Jaeki; Rhee, David Y.; Connelly, Michele; Sviderskiy, Vladislav O.; Bhasin, Deepak; Chen, Yizhe; Ong, Su-Sien; Chai, Sergio C.; Goktug, Asli N.; Huang, Guochang; Monda, Julie K.; Low, Jonathan; Kim, Ho Shin; Paulo, Joao A.; Cannon, Joe R.; Shelat, Anang A.; Chen, Taosheng; Kelsall, Ian R.; Alpi, Arno F.; Pagala, Vishwajeeth; Wang, Xusheng; Peng, Junmin; Singh, Bhuvanesh; Harper, J. Wade; Schulman, Brenda A.; Guy, R. Kip</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">850-857</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">N-terminal acetylation is an abundant modification influencing protein functions.  Because ∼80% of mammalian cytosolic proteins are N-terminally acetylated, this modification is potentially an untapped target for chem. control of their functions.  Structural studies have revealed that, like lysine acetylation, N-terminal acetylation converts a pos. charged amine into a hydrophobic handle that mediates protein interactions; hence, this modification may be a druggable target.  We report the development of chem. probes targeting the N-terminal acetylation-dependent interaction between an E2 conjugating enzyme (UBE2M or UBC12) and DCN1 (DCUN1D1), a subunit of a multiprotein E3 ligase for the ubiquitin-like protein NEDD8.  The inhibitors are highly selective with respect to other protein acetyl-amide-binding sites, inhibit NEDD8 ligation in vitro and in cells, and suppress anchorage-independent growth of a cell line with DCN1 amplification.  Overall, our data demonstrate that N-terminal acetyl-dependent protein interactions are druggable targets and provide insights into targeting multiprotein E2-E3 ligases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqh7JDyzBEGlrVg90H21EOLACvtfcHk0lhBZ1JF2vmLSA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXpt1Omt7k%253D&md5=de7519fbd287df91e53f8722d361d69f</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.2386&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.2386%26sid%3Dliteratum%253Aachs%26aulast%3DScott%26aufirst%3DD.%2BC.%26aulast%3DHammill%26aufirst%3DJ.%2BT.%26aulast%3DMin%26aufirst%3DJ.%26aulast%3DRhee%26aufirst%3DD.%2BY.%26aulast%3DConnelly%26aufirst%3DM.%26aulast%3DSviderskiy%26aufirst%3DV.%2BO.%26aulast%3DBhasin%26aufirst%3DD.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DOng%26aufirst%3DS.-S.%26aulast%3DChai%26aufirst%3DS.%2BC.%26aulast%3DGoktug%26aufirst%3DA.%2BN.%26aulast%3DHuang%26aufirst%3DG.%26aulast%3DMonda%26aufirst%3DJ.%2BK.%26aulast%3DLow%26aufirst%3DJ.%26aulast%3DKim%26aufirst%3DH.%2BS.%26aulast%3DPaulo%26aufirst%3DJ.%2BA.%26aulast%3DCannon%26aufirst%3DJ.%2BR.%26aulast%3DShelat%26aufirst%3DA.%2BA.%26aulast%3DChen%26aufirst%3DT.%26aulast%3DKelsall%26aufirst%3DI.%2BR.%26aulast%3DAlpi%26aufirst%3DA.%2BF.%26aulast%3DPagala%26aufirst%3DV.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DPeng%26aufirst%3DJ.%26aulast%3DSingh%26aufirst%3DB.%26aulast%3DHarper%26aufirst%3DJ.%2BW.%26aulast%3DSchulman%26aufirst%3DB.%2BA.%26aulast%3DGuy%26aufirst%3DR.%2BK.%26atitle%3DBlocking%2520an%2520N-terminal%2520acetylation%25E2%2580%2593dependent%2520protein%2520interaction%2520inhibits%2520an%2520E3%2520ligase%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2017%26volume%3D13%26spage%3D850%26epage%3D857%26doi%3D10.1038%2Fnchembio.2386" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kurz, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chou, Y. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Willems, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meyer-Schaller, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hecht, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tyers, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peter, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sicheri, F.</span></span> <span> </span><span class="NLM_article-title">Dcn1 functions as a scaffold-type E3 ligase for cullin neddylation</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">23</span>– <span class="NLM_lpage">35</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2007.12.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=10.1016%2Fj.molcel.2007.12.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=18206966" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtlWjt7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2008&pages=23-35&author=T.+Kurzauthor=Y.+C.+Chouauthor=A.+R.+Willemsauthor=N.+Meyer-Schallerauthor=M.+L.+Hechtauthor=M.+Tyersauthor=M.+Peterauthor=F.+Sicheri&title=Dcn1+functions+as+a+scaffold-type+E3+ligase+for+cullin+neddylation&doi=10.1016%2Fj.molcel.2007.12.012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Dcn1 functions as a scaffold-type E3 ligase for cullin neddylation</span></div><div class="casAuthors">Kurz, Thimo; Chou, Yang-Chieh; Willems, Andrew R.; Meyer-Schaller, Nathalie; Hecht, Marie-Lyn; Tyers, Mike; Peter, Matthias; Sicheri, Frank</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">23-35</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Cullin-based E3 ubiquitin ligases are activated through modification of the cullin subunit with the ubiquitin-like protein Nedd8.  Dcn1 regulates cullin neddylation and thus ubiquitin ligase activity.  Here we describe the 1.9 Å x-ray crystal structure of yeast Dcn1 encompassing an N-terminal ubiquitin-binding (UBA) domain and a C-terminal domain of unique architecture, which we termed PONY domain.  A conserved surface on Dcn1 is required for direct binding to cullins and for neddylation.  The reciprocal binding site for Dcn1 on Cdc53 is located ∼18 Å from the site of neddylation.  Dcn1 does not require cysteine residues for catalytic function, and directly interacts with the Nedd8 E2 Ubc12 on a surface that overlaps with the E1-binding site.  We show that Dcn1 is necessary and sufficient for cullin neddylation in a purified recombinant system.  Taken together, these data demonstrate that Dcn1 is a scaffold-like E3 ligase for cullin neddylation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoUMPyViHz6WLVg90H21EOLACvtfcHk0lgjKOSf_bXWLQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtlWjt7k%253D&md5=288151ba7d80c529a546ef2b74274516</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2007.12.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2007.12.012%26sid%3Dliteratum%253Aachs%26aulast%3DKurz%26aufirst%3DT.%26aulast%3DChou%26aufirst%3DY.%2BC.%26aulast%3DWillems%26aufirst%3DA.%2BR.%26aulast%3DMeyer-Schaller%26aufirst%3DN.%26aulast%3DHecht%26aufirst%3DM.%2BL.%26aulast%3DTyers%26aufirst%3DM.%26aulast%3DPeter%26aufirst%3DM.%26aulast%3DSicheri%26aufirst%3DF.%26atitle%3DDcn1%2520functions%2520as%2520a%2520scaffold-type%2520E3%2520ligase%2520for%2520cullin%2520neddylation%26jtitle%3DMol.%2520Cell%26date%3D2008%26volume%3D29%26spage%3D23%26epage%3D35%26doi%3D10.1016%2Fj.molcel.2007.12.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Scott, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Monda, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bennett, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harper, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schulman, B. A.</span></span> <span> </span><span class="NLM_article-title">N-terminal acetylation acts as an avidity enhancer within an interconnected multiprotein complex</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>334</i></span>,  <span class="NLM_fpage">674</span>– <span class="NLM_lpage">678</span>, <span class="refDoi"> DOI: 10.1126/science.1209307</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=10.1126%2Fscience.1209307" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=21940857" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtlyqu7jL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=334&publication_year=2011&pages=674-678&author=D.+C.+Scottauthor=J.+K.+Mondaauthor=E.+J.+Bennettauthor=J.+W.+Harperauthor=B.+A.+Schulman&title=N-terminal+acetylation+acts+as+an+avidity+enhancer+within+an+interconnected+multiprotein+complex&doi=10.1126%2Fscience.1209307"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">N-Terminal Acetylation Acts as an Avidity Enhancer Within an Interconnected Multiprotein Complex</span></div><div class="casAuthors">Scott, Daniel C.; Monda, Julie K.; Bennett, Eric J.; Harper, J. Wade; Schulman, Brenda A.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">334</span>
        (<span class="NLM_cas:issue">6056</span>),
    <span class="NLM_cas:pages">674-678</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Although many eukaryotic proteins are amino (N)-terminally acetylated, structural mechanisms by which N-terminal acetylation mediates protein interactions are largely unknown.  Here, N-terminal acetylation of the E2 enzyme, Ubc12, dictates distinctive E3-dependent ligation of the ubiquitin-like protein Nedd8 to Cul1.  Structural, biochem., biophys., and genetic analyses revealed how complete burial of Ubc12's N-acetyl-methionine in a hydrophobic pocket in the E3, Dcn1, promotes cullin neddylation.  The results suggest that the N-terminal acetyl both directs Ubc12's interactions with Dcn1 and prevents repulsion of a charged N terminus.  The authors' data provide a link between acetylation and ubiquitin-like protein conjugation and define a mechanism for N-terminal acetylation-dependent recognition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq2YZdYRdU_fbVg90H21EOLACvtfcHk0lgjKOSf_bXWLQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtlyqu7jL&md5=1654bea16702633a9baf611827f60450</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1126%2Fscience.1209307&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1209307%26sid%3Dliteratum%253Aachs%26aulast%3DScott%26aufirst%3DD.%2BC.%26aulast%3DMonda%26aufirst%3DJ.%2BK.%26aulast%3DBennett%26aufirst%3DE.%2BJ.%26aulast%3DHarper%26aufirst%3DJ.%2BW.%26aulast%3DSchulman%26aufirst%3DB.%2BA.%26atitle%3DN-terminal%2520acetylation%2520acts%2520as%2520an%2520avidity%2520enhancer%2520within%2520an%2520interconnected%2520multiprotein%2520complex%26jtitle%3DScience%26date%3D2011%26volume%3D334%26spage%3D674%26epage%3D678%26doi%3D10.1126%2Fscience.1209307" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Scott, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sviderskiy, V. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Monda, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lydeard, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cho, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harper, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schulman, B. A.</span></span> <span> </span><span class="NLM_article-title">Structure of a RING E3 trapped in action reveals ligation mechanism for the ubiquitin-like protein NEDD8</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>157</i></span>,  <span class="NLM_fpage">1671</span>– <span class="NLM_lpage">1684</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2014.04.037</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=10.1016%2Fj.cell.2014.04.037" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=24949976" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtVCisrfP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=157&publication_year=2014&pages=1671-1684&author=D.+C.+Scottauthor=V.+O.+Sviderskiyauthor=J.+K.+Mondaauthor=J.+R.+Lydeardauthor=S.+E.+Choauthor=J.+W.+Harperauthor=B.+A.+Schulman&title=Structure+of+a+RING+E3+trapped+in+action+reveals+ligation+mechanism+for+the+ubiquitin-like+protein+NEDD8&doi=10.1016%2Fj.cell.2014.04.037"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of a RING E3 trapped in action reveals ligation mechanism for the ubiquitin-like protein NEDD8</span></div><div class="casAuthors">Scott, Daniel C.; Sviderskiy, Vladislav O.; Monda, Julie K.; Lydeard, John R.; Cho, Shein Ei; Harper, J. Wade; Schulman, Brenda A.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">157</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1671-1684</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Most E3 ligases use a RING domain to activate a thioester-linked E2∼ubiquitin-like protein (UBL) intermediate and promote UBL transfer to a remotely bound target protein.  Nonetheless, RING E3 mechanisms matching a specific UBL and acceptor lysine remain elusive, including for RBX1, which mediates NEDD8 ligation to cullins and >10% of all ubiquitination.  We report the structure of a trapped RING E3-E2∼UBL-target intermediate representing RBX1-UBC12∼NEDD8-CUL1-DCN1, which reveals the mechanism of NEDD8 ligation and how a particular UBL and acceptor lysine are matched by a multifunctional RING E3.  Numerous mechanisms specify cullin neddylation while preventing noncognate ubiquitin ligation.  Notably, E2-E3-target and RING-E2∼UBL modules are not optimized to function independently, but instead require integration by the UBL and target for maximal reactivity.  The UBL and target regulate the catalytic machinery by positioning the RING-E2∼UBL catalytic center, licensing the acceptor lysine, and influencing E2 reactivity, thereby driving their specific coupling by a multifunctional RING E3.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpVAVv0lWWAVLVg90H21EOLACvtfcHk0liUiIbN_dHxrg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtVCisrfP&md5=6674c0e32fba481709a4e1d1932d3127</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2014.04.037&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2014.04.037%26sid%3Dliteratum%253Aachs%26aulast%3DScott%26aufirst%3DD.%2BC.%26aulast%3DSviderskiy%26aufirst%3DV.%2BO.%26aulast%3DMonda%26aufirst%3DJ.%2BK.%26aulast%3DLydeard%26aufirst%3DJ.%2BR.%26aulast%3DCho%26aufirst%3DS.%2BE.%26aulast%3DHarper%26aufirst%3DJ.%2BW.%26aulast%3DSchulman%26aufirst%3DB.%2BA.%26atitle%3DStructure%2520of%2520a%2520RING%2520E3%2520trapped%2520in%2520action%2520reveals%2520ligation%2520mechanism%2520for%2520the%2520ubiquitin-like%2520protein%2520NEDD8%26jtitle%3DCell%26date%3D2014%26volume%3D157%26spage%3D1671%26epage%3D1684%26doi%3D10.1016%2Fj.cell.2014.04.037" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kurz, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ozlu, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rudolf, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Rourke, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luke, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hofmann, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hyman, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bowerman, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peter, M.</span></span> <span> </span><span class="NLM_article-title">The conserved protein DCN-1/Dcn1p is required for cullin neddylation in <i>C. elegans</i> and <i>S. cerevisiae</i></span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>435</i></span>,  <span class="NLM_fpage">1257</span>– <span class="NLM_lpage">1261</span>, <span class="refDoi"> DOI: 10.1038/nature03662</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=10.1038%2Fnature03662" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=15988528" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=1%3ACAS%3A528%3ADC%252BD2MXls1Ontb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=435&publication_year=2005&pages=1257-1261&author=T.+Kurzauthor=N.+Ozluauthor=F.+Rudolfauthor=S.+M.+O%E2%80%99Rourkeauthor=B.+Lukeauthor=K.+Hofmannauthor=A.+A.+Hymanauthor=B.+Bowermanauthor=M.+Peter&title=The+conserved+protein+DCN-1%2FDcn1p+is+required+for+cullin+neddylation+in+C.+elegans+and+S.+cerevisiae&doi=10.1038%2Fnature03662"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">The conserved protein DCN-1/Dcn1p is required for cullin neddylation in C. elegans and S. cerevisiae</span></div><div class="casAuthors">Kurz, Thimo; Oezlue, Nurhan; Rudolf, Fabian; O'Rourke, Sean M.; Luke, Brian; Hofmann, Kay; Hyman, Anthony A.; Bowerman, Bruce; Peter, Matthias</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">435</span>
        (<span class="NLM_cas:issue">7046</span>),
    <span class="NLM_cas:pages">1257-1261</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">SCF-type E3 ubiquitin ligases are multi-protein complexes required for polyubiquitination and subsequent degrdn. of target proteins by the 26S proteasome.  Cullins, together with the RING-finger protein Rbx1, form the catalytic core of the ligase, and recruit the substrate-recognition module.  Cycles of covalent modification of cullins by the ubiquitin-like mol. Nedd8 (neddylation) and removal of Nedd8 by the COP9 signalosome (deneddylation) pos. regulate E3 ligase activity.  Here we report the identification and anal. of a widely conserved protein that is required for cullin neddylation in the nematode Caenorhabditis elegans and the yeast Saccharomyces cerevisiae.  C. elegans DCN-1 and S. cerevisiae Dcn1p (defective in cullin neddylation) are characterized by a novel UBA-like ubiquitin-binding domain and a DUF298 domain of unknown function.  Consistent with their requirements for neddylation, DCN-1 and Dcn1p directly bind Nedd8 and phys. assoc. with cullins in both spp.  Moreover, overexpression of Dcn1p in yeast results in the accumulation of Nedd8-modified cullin Cdc53p.  Both in vivo and in vitro expts. indicate that Dcn1p does not inhibit deneddylation of Cdc53p by the COP9 signalosome, but greatly increases the kinetics of the neddylation reaction.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrgm_SbEnUuBrVg90H21EOLACvtfcHk0liUiIbN_dHxrg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXls1Ontb0%253D&md5=545afac53b6c989be8b2fe5c78336d43</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1038%2Fnature03662&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature03662%26sid%3Dliteratum%253Aachs%26aulast%3DKurz%26aufirst%3DT.%26aulast%3DOzlu%26aufirst%3DN.%26aulast%3DRudolf%26aufirst%3DF.%26aulast%3DO%25E2%2580%2599Rourke%26aufirst%3DS.%2BM.%26aulast%3DLuke%26aufirst%3DB.%26aulast%3DHofmann%26aufirst%3DK.%26aulast%3DHyman%26aufirst%3DA.%2BA.%26aulast%3DBowerman%26aufirst%3DB.%26aulast%3DPeter%26aufirst%3DM.%26atitle%3DThe%2520conserved%2520protein%2520DCN-1%252FDcn1p%2520is%2520required%2520for%2520cullin%2520neddylation%2520in%2520C.%2520elegans%2520and%2520S.%2520cerevisiae%26jtitle%3DNature%26date%3D2005%26volume%3D435%26spage%3D1257%26epage%3D1261%26doi%3D10.1038%2Fnature03662" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernard, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fernandez-Salas, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chinnaswamy, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Layton, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stuckey, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pan, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span> <span> </span><span class="NLM_article-title">A potent small-molecule inhibitor of the DCN1-UBC12 interaction that selectively blocks cullin 3 neddylation</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">1150</span>, <span class="refDoi"> DOI: 10.1038/s41467-017-01243-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=10.1038%2Fs41467-017-01243-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=29074978" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=1%3ACAS%3A280%3ADC%252BC1M7kvFOquw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=1150&author=H.+Zhouauthor=J.+Luauthor=L.+Liuauthor=D.+Bernardauthor=C.-Y.+Yangauthor=E.+Fernandez-Salasauthor=K.+Chinnaswamyauthor=S.+Laytonauthor=J.+Stuckeyauthor=Q.+Yuauthor=W.+Zhouauthor=Z.+Panauthor=Y.+Sunauthor=S.+Wang&title=A+potent+small-molecule+inhibitor+of+the+DCN1-UBC12+interaction+that+selectively+blocks+cullin+3+neddylation&doi=10.1038%2Fs41467-017-01243-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">A potent small-molecule inhibitor of the DCN1-UBC12 interaction that selectively blocks cullin 3 neddylation</span></div><div class="casAuthors">Zhou Haibin; Lu Jianfeng; Liu Liu; Bernard Denzil; Yang Chao-Yie; Wang Shaomeng; Fernandez-Salas Ester; Chinnaswamy Krishnapriya; Layton Stephanie; Stuckey Jeanne; Yu Qing; Sun Yi; Zhou Weihua; Sun Yi; Pan Zhenqiang; Wang Shaomeng; Wang Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Nature communications</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1150</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The Cullin-RING E3 ubiquitin ligases (CRLs) regulate homeostasis of ~20% of cellular proteins and their activation require neddylation of their cullin subunit.  Cullin neddylation is modulated by a scaffolding DCN protein through interactions with both the cullin protein and an E2 enzyme such as UBC12.  Here we report the development of DI-591 as a high-affinity, cell-permeable small-molecule inhibitor of the DCN1-UBC12 interaction.  DI-591 binds to purified recombinant human DCN1 and DCN2 proteins with K i values of 10-12 nM, and disrupts the DCN1-UBC12 interaction in cells.  Treatment with DI-591 selectively converts cellular cullin 3 into an un-neddylated inactive form with no or minimum effect on other cullin members.  Our data firmly establish a previously unrecognized specific role of the DCN1-UBC12 interaction for cellular neddylation of cullin 3.  DI-591 is an excellent probe compound to investigate the role of the cullin 3 CRL ligase in biological processes and human diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTn87oIyDIsYOxsqfvU0ziafW6udTcc2eZhIo3NsRgvEbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1M7kvFOquw%253D%253D&md5=47a713f31bcad6e0b45196cb3f1c8766</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1038%2Fs41467-017-01243-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41467-017-01243-7%26sid%3Dliteratum%253Aachs%26aulast%3DZhou%26aufirst%3DH.%26aulast%3DLu%26aufirst%3DJ.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DBernard%26aufirst%3DD.%26aulast%3DYang%26aufirst%3DC.-Y.%26aulast%3DFernandez-Salas%26aufirst%3DE.%26aulast%3DChinnaswamy%26aufirst%3DK.%26aulast%3DLayton%26aufirst%3DS.%26aulast%3DStuckey%26aufirst%3DJ.%26aulast%3DYu%26aufirst%3DQ.%26aulast%3DZhou%26aufirst%3DW.%26aulast%3DPan%26aufirst%3DZ.%26aulast%3DSun%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DA%2520potent%2520small-molecule%2520inhibitor%2520of%2520the%2520DCN1-UBC12%2520interaction%2520that%2520selectively%2520blocks%2520cullin%25203%2520neddylation%26jtitle%3DNat.%2520Commun.%26date%3D2017%26volume%3D8%26spage%3D1150%26doi%3D10.1038%2Fs41467-017-01243-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hammill, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Min, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Connelly, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holbrook, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matheny, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schulman, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guy, R. K.</span></span> <span> </span><span class="NLM_article-title">Piperidinyl ureas chemically control defective in cullin neddylation 1 (DCN1)-mediated cullin neddylation</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">2680</span>– <span class="NLM_lpage">2693</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b01277</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b01277" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=1%3ACAS%3A528%3ADC%252BC1cXltVGks70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=2680-2693&author=J.+T.+Hammillauthor=D.+C.+Scottauthor=J.+Minauthor=M.+C.+Connellyauthor=G.+Holbrookauthor=F.+Zhuauthor=A.+Mathenyauthor=L.+Yangauthor=B.+Singhauthor=B.+A.+Schulmanauthor=R.+K.+Guy&title=Piperidinyl+ureas+chemically+control+defective+in+cullin+neddylation+1+%28DCN1%29-mediated+cullin+neddylation&doi=10.1021%2Facs.jmedchem.7b01277"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Piperidinyl Ureas Chemically Control Defective in Cullin Neddylation 1 (DCN1)-Mediated Cullin Neddylation</span></div><div class="casAuthors">Hammill, Jared T.; Scott, Daniel C.; Min, Jaeki; Connelly, Michele C.; Holbrook, Gloria; Zhu, Fangyi; Matheny, Amy; Yang, Lei; Singh, Bhuvanesh; Schulman, Brenda A.; Guy, R. Kiplin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2680-2693</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The authors previously discovered and validated a class of piperidinyl ureas that regulate defective in cullin neddylation 1 (DCN1)-dependent neddylation of cullins.  Here, the authors report preliminary structure-activity relationship studies aimed at advancing the high-throughput screen hit into a tractable tool compd. for dissecting the effects of acute DCN1-UBE2M inhibition on the NEDD8/cullin pathway.  Structure-enabled optimization led to a 100-fold increase in biochem. potency and modestly increased soly. and permeability as compared to the initial hit.  The optimized compds. inhibit the DCN1-UBE2M protein-protein interaction in the TR-FRET binding assay and inhibit cullin neddylation in the pulse-chase NEDD8 transfer assay.  The optimized compds. bind to DCN1 and selectively reduce steady-state levels of neddylated CUL1 and CUL3 in a squamous cell carcinoma cell line.  Ultimately, the authors anticipate that these studies will identify early lead compds. for clin. development for the treatment of lung squamous cell carcinomas and other cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoC1AvhDCXYqrVg90H21EOLACvtfcHk0lhhz6O5ILU20Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXltVGks70%253D&md5=142c48248f4adcbdceca071790adabbd</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b01277&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b01277%26sid%3Dliteratum%253Aachs%26aulast%3DHammill%26aufirst%3DJ.%2BT.%26aulast%3DScott%26aufirst%3DD.%2BC.%26aulast%3DMin%26aufirst%3DJ.%26aulast%3DConnelly%26aufirst%3DM.%2BC.%26aulast%3DHolbrook%26aufirst%3DG.%26aulast%3DZhu%26aufirst%3DF.%26aulast%3DMatheny%26aufirst%3DA.%26aulast%3DYang%26aufirst%3DL.%26aulast%3DSingh%26aufirst%3DB.%26aulast%3DSchulman%26aufirst%3DB.%2BA.%26aulast%3DGuy%26aufirst%3DR.%2BK.%26atitle%3DPiperidinyl%2520ureas%2520chemically%2520control%2520defective%2520in%2520cullin%2520neddylation%25201%2520%2528DCN1%2529-mediated%2520cullin%2520neddylation%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D2680%26epage%3D2693%26doi%3D10.1021%2Facs.jmedchem.7b01277" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Scott, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sviderskiy, V. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Monda, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lydeard, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cho, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harper, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schulman, B. A.</span></span> <span> </span><span class="NLM_article-title">Structure of a ring E3 trapped in action reveals ligation mechanism for the ubiquitin-like protein Nedd8</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>157</i></span>,  <span class="NLM_fpage">1671</span>– <span class="NLM_lpage">1684</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2014.04.037</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=10.1016%2Fj.cell.2014.04.037" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=24949976" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtVCisrfP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=157&publication_year=2014&pages=1671-1684&author=D.+C.+Scottauthor=V.+O.+Sviderskiyauthor=J.+K.+Mondaauthor=J.+R.+Lydeardauthor=S.+E.+Choauthor=J.+W.+Harperauthor=B.+A.+Schulman&title=Structure+of+a+ring+E3+trapped+in+action+reveals+ligation+mechanism+for+the+ubiquitin-like+protein+Nedd8&doi=10.1016%2Fj.cell.2014.04.037"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of a RING E3 trapped in action reveals ligation mechanism for the ubiquitin-like protein NEDD8</span></div><div class="casAuthors">Scott, Daniel C.; Sviderskiy, Vladislav O.; Monda, Julie K.; Lydeard, John R.; Cho, Shein Ei; Harper, J. Wade; Schulman, Brenda A.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">157</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1671-1684</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Most E3 ligases use a RING domain to activate a thioester-linked E2∼ubiquitin-like protein (UBL) intermediate and promote UBL transfer to a remotely bound target protein.  Nonetheless, RING E3 mechanisms matching a specific UBL and acceptor lysine remain elusive, including for RBX1, which mediates NEDD8 ligation to cullins and >10% of all ubiquitination.  We report the structure of a trapped RING E3-E2∼UBL-target intermediate representing RBX1-UBC12∼NEDD8-CUL1-DCN1, which reveals the mechanism of NEDD8 ligation and how a particular UBL and acceptor lysine are matched by a multifunctional RING E3.  Numerous mechanisms specify cullin neddylation while preventing noncognate ubiquitin ligation.  Notably, E2-E3-target and RING-E2∼UBL modules are not optimized to function independently, but instead require integration by the UBL and target for maximal reactivity.  The UBL and target regulate the catalytic machinery by positioning the RING-E2∼UBL catalytic center, licensing the acceptor lysine, and influencing E2 reactivity, thereby driving their specific coupling by a multifunctional RING E3.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpVAVv0lWWAVLVg90H21EOLACvtfcHk0lhhz6O5ILU20Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtVCisrfP&md5=6674c0e32fba481709a4e1d1932d3127</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2014.04.037&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2014.04.037%26sid%3Dliteratum%253Aachs%26aulast%3DScott%26aufirst%3DD.%2BC.%26aulast%3DSviderskiy%26aufirst%3DV.%2BO.%26aulast%3DMonda%26aufirst%3DJ.%2BK.%26aulast%3DLydeard%26aufirst%3DJ.%2BR.%26aulast%3DCho%26aufirst%3DS.%2BE.%26aulast%3DHarper%26aufirst%3DJ.%2BW.%26aulast%3DSchulman%26aufirst%3DB.%2BA.%26atitle%3DStructure%2520of%2520a%2520ring%2520E3%2520trapped%2520in%2520action%2520reveals%2520ligation%2520mechanism%2520for%2520the%2520ubiquitin-like%2520protein%2520Nedd8%26jtitle%3DCell%26date%3D2014%26volume%3D157%26spage%3D1671%26epage%3D1684%26doi%3D10.1016%2Fj.cell.2014.04.037" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Scott, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Monda, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bennett, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harper, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schulman, B. A.</span></span> <span> </span><span class="NLM_article-title">N-terminal acetylation acts as an avidity enhancer within an interconnected multiprotein complex</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>334</i></span>,  <span class="NLM_fpage">674</span>– <span class="NLM_lpage">678</span>, <span class="refDoi"> DOI: 10.1126/science.1209307</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=10.1126%2Fscience.1209307" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=21940857" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtlyqu7jL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=334&publication_year=2011&pages=674-678&author=D.+C.+Scottauthor=J.+K.+Mondaauthor=E.+J.+Bennettauthor=J.+W.+Harperauthor=B.+A.+Schulman&title=N-terminal+acetylation+acts+as+an+avidity+enhancer+within+an+interconnected+multiprotein+complex&doi=10.1126%2Fscience.1209307"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">N-Terminal Acetylation Acts as an Avidity Enhancer Within an Interconnected Multiprotein Complex</span></div><div class="casAuthors">Scott, Daniel C.; Monda, Julie K.; Bennett, Eric J.; Harper, J. Wade; Schulman, Brenda A.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">334</span>
        (<span class="NLM_cas:issue">6056</span>),
    <span class="NLM_cas:pages">674-678</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Although many eukaryotic proteins are amino (N)-terminally acetylated, structural mechanisms by which N-terminal acetylation mediates protein interactions are largely unknown.  Here, N-terminal acetylation of the E2 enzyme, Ubc12, dictates distinctive E3-dependent ligation of the ubiquitin-like protein Nedd8 to Cul1.  Structural, biochem., biophys., and genetic analyses revealed how complete burial of Ubc12's N-acetyl-methionine in a hydrophobic pocket in the E3, Dcn1, promotes cullin neddylation.  The results suggest that the N-terminal acetyl both directs Ubc12's interactions with Dcn1 and prevents repulsion of a charged N terminus.  The authors' data provide a link between acetylation and ubiquitin-like protein conjugation and define a mechanism for N-terminal acetylation-dependent recognition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq2YZdYRdU_fbVg90H21EOLACvtfcHk0liFVcmpUvVliQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtlyqu7jL&md5=1654bea16702633a9baf611827f60450</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1126%2Fscience.1209307&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1209307%26sid%3Dliteratum%253Aachs%26aulast%3DScott%26aufirst%3DD.%2BC.%26aulast%3DMonda%26aufirst%3DJ.%2BK.%26aulast%3DBennett%26aufirst%3DE.%2BJ.%26aulast%3DHarper%26aufirst%3DJ.%2BW.%26aulast%3DSchulman%26aufirst%3DB.%2BA.%26atitle%3DN-terminal%2520acetylation%2520acts%2520as%2520an%2520avidity%2520enhancer%2520within%2520an%2520interconnected%2520multiprotein%2520complex%26jtitle%3DScience%26date%3D2011%26volume%3D334%26spage%3D674%26epage%3D678%26doi%3D10.1126%2Fscience.1209307" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hammill, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhasin, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Min, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Connelly, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hammill, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holbrook, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeffries, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schulman, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guy, R. K.</span></span> <span> </span><span class="NLM_article-title">Discovery of an orally bioavailable inhibitor of defective in cullin neddylation 1 (DCN1)-mediated cullin neddylation</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">2694</span>– <span class="NLM_lpage">2706</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b01282</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b01282" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=1%3ACAS%3A528%3ADC%252BC1cXltVGkt7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=2694-2706&author=J.+T.+Hammillauthor=D.+Bhasinauthor=D.+C.+Scottauthor=J.+Minauthor=Y.+Chenauthor=Y.+Luauthor=L.+Yangauthor=H.+S.+Kimauthor=M.+C.+Connellyauthor=C.+Hammillauthor=G.+Holbrookauthor=C.+Jeffriesauthor=B.+Singhauthor=B.+A.+Schulmanauthor=R.+K.+Guy&title=Discovery+of+an+orally+bioavailable+inhibitor+of+defective+in+cullin+neddylation+1+%28DCN1%29-mediated+cullin+neddylation&doi=10.1021%2Facs.jmedchem.7b01282"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of an Orally Bioavailable Inhibitor of Defective in Cullin Neddylation 1 (DCN1)-Mediated Cullin Neddylation</span></div><div class="casAuthors">Hammill, Jared T.; Bhasin, Deepak; Scott, Daniel C.; Min, Jaeki; Chen, Yizhe; Lu, Yan; Yang, Lei; Kim, Ho Shin; Connelly, Michele C.; Hammill, Courtney; Holbrook, Gloria; Jeffries, Cynthia; Singh, Bhuvanesh; Schulman, Brenda A.; Guy, R. Kiplin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2694-2706</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The authors previously reported the discovery, validation, and structure-activity relationships of a series of piperidinyl ureas that potently inhibit the DCN1-UBE2M interaction.  The authors demonstrated that compd. I inhibits both the DCN1-UBE2M protein-protein interaction and DCN1-mediated cullin neddylation in biochem. assays and reduces levels of steady-state cullin neddylation in a squamous carcinoma cell line harboring DCN1 amplification.  Although compd. I exhibits good soly. and permeability, it is rapidly metabolized in microsomal models (CLint = 170 mL/min/kg).  This work lays out the discovery of an orally bioavailable analog, NAcM-OPT (II).  Compd. II retains the favorable biochem. and cellular activity of compd. I but is significantly more stable both in vitro and in vivo.  Compd. II is orally bioavailable, well tolerated in mice, and currently used to study the effects of acute pharmacol. inhibition of the DCN1-UBE2M interaction on the NEDD8/CUL pathway.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq4fvhBFy2Ux7Vg90H21EOLACvtfcHk0liFVcmpUvVliQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXltVGkt7c%253D&md5=5775690be4373f8d8a6cb401812363dd</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b01282&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b01282%26sid%3Dliteratum%253Aachs%26aulast%3DHammill%26aufirst%3DJ.%2BT.%26aulast%3DBhasin%26aufirst%3DD.%26aulast%3DScott%26aufirst%3DD.%2BC.%26aulast%3DMin%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DLu%26aufirst%3DY.%26aulast%3DYang%26aufirst%3DL.%26aulast%3DKim%26aufirst%3DH.%2BS.%26aulast%3DConnelly%26aufirst%3DM.%2BC.%26aulast%3DHammill%26aufirst%3DC.%26aulast%3DHolbrook%26aufirst%3DG.%26aulast%3DJeffries%26aufirst%3DC.%26aulast%3DSingh%26aufirst%3DB.%26aulast%3DSchulman%26aufirst%3DB.%2BA.%26aulast%3DGuy%26aufirst%3DR.%2BK.%26atitle%3DDiscovery%2520of%2520an%2520orally%2520bioavailable%2520inhibitor%2520of%2520defective%2520in%2520cullin%2520neddylation%25201%2520%2528DCN1%2529-mediated%2520cullin%2520neddylation%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D2694%26epage%3D2706%26doi%3D10.1021%2Facs.jmedchem.7b01282" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chern, T.-R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chinnaswamy, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernard, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stuckey, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span> <span> </span><span class="NLM_article-title">High-Affinity Peptidomimetic inhibitors of the DCN1-UBC12 protein–protein interaction</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">1934</span>– <span class="NLM_lpage">1950</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b01455</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b01455" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=1%3ACAS%3A528%3ADC%252BC1cXis1Kqsr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=1934-1950&author=H.+Zhouauthor=W.+Zhouauthor=B.+Zhouauthor=L.+Liuauthor=T.-R.+Chernauthor=K.+Chinnaswamyauthor=J.+Luauthor=D.+Bernardauthor=C.-Y.+Yangauthor=S.+Liauthor=M.+Wangauthor=J.+Stuckeyauthor=Y.+Sunauthor=S.+Wang&title=High-Affinity+Peptidomimetic+inhibitors+of+the+DCN1-UBC12+protein%E2%80%93protein+interaction&doi=10.1021%2Facs.jmedchem.7b01455"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">High-affinity peptidomimetic inhibitors of the DCN1-UBC12 protein-protein interaction</span></div><div class="casAuthors">Zhou, Haibin; Zhou, Weihua; Zhou, Bing; Liu, Liu; Chern, Ting-Rong; Chinnaswamy, Krishnapriya; Lu, Jianfeng; Bernard, Denzil; Yang, Chao-Yie; Li, Shasha; Wang, Mi; Stuckey, Jeanne; Sun, Yi; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1934-1950</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The Cullin-RING ligases (CRLs) regulate the turnover of approx. 20% of the proteins in mammalian cells and are emerging therapeutic targets in human diseases.  The activation of CRLs requires the neddylation of their cullin subunit, which is controlled by an activation complex consisting of Cullin-RBX1-UBC12-NEDD8-DCN1.  Herein, we describe the design, synthesis, and evaluation of peptidomimetics targeting the DCN1-UBC12 protein-protein interaction.  Starting from a 12-residue UBC12 peptide, we have successfully obtained a series of peptidomimetic compds. that bind to DCN1 protein with KD values of <10 nM.  Detn. of a cocrystal structure of a potent peptidomimetic inhibitor complexed with DCN1 provides the structural basis for their high-affinity interaction.  Cellular investigation of one potent DCN1 inhibitor, compd. (I) (DI-404), reveals that it effectively and selectively inhibits the neddylation of cullin 3 over other cullin members.  Further optimization of DI-404 may yield a new class of therapeutics for the treatment of human diseases in which cullin 3 CRL plays a key role.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqtfHxzNV467rVg90H21EOLACvtfcHk0liYnDYWHdcayA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXis1Kqsr0%253D&md5=6489af488ea0608aed7348662e9b261f</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b01455&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b01455%26sid%3Dliteratum%253Aachs%26aulast%3DZhou%26aufirst%3DH.%26aulast%3DZhou%26aufirst%3DW.%26aulast%3DZhou%26aufirst%3DB.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DChern%26aufirst%3DT.-R.%26aulast%3DChinnaswamy%26aufirst%3DK.%26aulast%3DLu%26aufirst%3DJ.%26aulast%3DBernard%26aufirst%3DD.%26aulast%3DYang%26aufirst%3DC.-Y.%26aulast%3DLi%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DStuckey%26aufirst%3DJ.%26aulast%3DSun%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DHigh-Affinity%2520Peptidomimetic%2520inhibitors%2520of%2520the%2520DCN1-UBC12%2520protein%25E2%2580%2593protein%2520interaction%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D1934%26epage%3D1950%26doi%3D10.1021%2Facs.jmedchem.7b01455" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, R.-H.</span></span> <span> </span><span class="NLM_article-title">Cullin 3 ubiquitin ligases in cancer biology: functions and therapeutic implications</span>. <i>Front. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">113</span>, <span class="refDoi"> DOI: 10.3389/fonc.2016.00113</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=10.3389%2Ffonc.2016.00113" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=27200299" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=1%3ACAS%3A280%3ADC%252BC2s%252FhsFKisQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2016&pages=113&author=H.-Y.+Chenauthor=R.-H.+Chen&title=Cullin+3+ubiquitin+ligases+in+cancer+biology%3A+functions+and+therapeutic+implications&doi=10.3389%2Ffonc.2016.00113"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Cullin 3 Ubiquitin Ligases in Cancer Biology: Functions and Therapeutic Implications</span></div><div class="casAuthors">Chen Hsin-Yi; Chen Ruey-Hwa</div><div class="citationInfo"><span class="NLM_cas:title">Frontiers in oncology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">113</span>
        ISSN:<span class="NLM_cas:issn">2234-943X</span>.
    </div><div class="casAbstract">Cullin-RING ubiquitin ligases are the largest E3 ligase family in eukaryotes and are multiprotein complexes.  In these complexes, the Cullin protein serves as a scaffold to connect two functional modules of the ligases, the catalytic subunit and substrate-binding subunit.  To date, eight members of the Cullin family proteins have been identified.  In the Cul3 ubiquitin ligases, Bric-a-brac/Tramtrack/Broad complex (BTB) domain-containing proteins function as a bridge to connect Cul3 and substrates.  While the BTB domain is responsible for Cul3 binding, these proteins usually contain an additional domain for substrate interaction, such as MATH, kelch, Zn finger, and PAM, Highwire, and RPM-1 (PHR domain).  With the existence of a large number of BTB proteins in human, the Cul3 ubiquitin ligases ubiquitinate a wide range of substrates involving in diverse cellular functions.  In this review, we will discuss recent advances on the functions of Cul3 ubiquitin ligases in cancer development, progression, and therapeutic response and the dysregulation of Cul3-mediated ubiquitination events in human malignancies.  In particular, we will focus on three Cul3 substrate adaptors, kelch-like ECH-associated protein (Keap1), kelch-like family member 20 (KLHL20), and speckle type BTB/POZ protein (SPOP), with the intent to highlight novel targets in cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQMbUYXuVDp7K91pGlMMoT5fW6udTcc2eZN5u02GjmsHrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2s%252FhsFKisQ%253D%253D&md5=4ab4fb3c41c92783f89f2c0910f39399</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.3389%2Ffonc.2016.00113&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3389%252Ffonc.2016.00113%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DH.-Y.%26aulast%3DChen%26aufirst%3DR.-H.%26atitle%3DCullin%25203%2520ubiquitin%2520ligases%2520in%2520cancer%2520biology%253A%2520functions%2520and%2520therapeutic%2520implications%26jtitle%3DFront.%2520Oncol.%26date%3D2016%26volume%3D6%26spage%3D113%26doi%3D10.3389%2Ffonc.2016.00113" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">North, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tao, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dai, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, W.</span></span> <span> </span><span class="NLM_article-title">Functional analysis of cullin 3 E3 ligases in tumorigenesis</span>. <i>Biochim. Biophys. Acta, Rev. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>1869</i></span>,  <span class="NLM_fpage">11</span>– <span class="NLM_lpage">28</span>, <span class="refDoi"> DOI: 10.1016/j.bbcan.2017.11.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=10.1016%2Fj.bbcan.2017.11.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=29128526" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvValt7zO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1869&publication_year=2018&pages=11-28&author=J.+Chengauthor=J.+Guoauthor=Z.+Wangauthor=B.+J.+Northauthor=K.+Taoauthor=X.+Daiauthor=W.+Wei&title=Functional+analysis+of+cullin+3+E3+ligases+in+tumorigenesis&doi=10.1016%2Fj.bbcan.2017.11.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Functional analysis of Cullin 3 E3 ligases in tumorigenesis</span></div><div class="casAuthors">Cheng, Ji; Guo, Jianping; Wang, Zhiwei; North, Brian J.; Tao, Kaixiong; Dai, Xiangpeng; Wei, Wenyi</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, Reviews on Cancer</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">1869</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">11-28</span>CODEN:
                <span class="NLM_cas:coden">BBACEU</span>;
        ISSN:<span class="NLM_cas:issn">0304-419X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Cullin 3-RING ligases (CRL3) play pivotal roles in the regulation of various physiol. and pathol. processes, including neoplastic events.  The substrate adaptors of CRL3 typically contain a BTB domain that mediates the interaction between Cullin 3 and target substrates to promote their ubiquitination and subsequent degrdn.  The biol. implications of CRL3 adaptor proteins have been well described where they have been found to play a role as either an oncogene, tumor suppressor, or can mediate either of these effects in a context-dependent manner.  Among the extensively studied CRL3-based E3 ligases, the role of the adaptor protein SPOP (speckle type BTB/POZ protein) in tumorigenesis appears to be tissue or cellular context dependent.  Specifically, SPOP acts as a tumor suppressor via destabilizing downstream oncoproteins in many malignancies, esp. in prostate cancer.  However, SPOP has largely an oncogenic role in kidney cancer.  Keap1, another well-characterized CRL3 adaptor protein, likely serves as a tumor suppressor within diverse malignancies, mainly due to its specific turnover of its downstream oncogenic substrate, NRF2 (nuclear factor erythroid 2-related factor 2).  In accordance with the physiol. role the various CRL3 adaptors exhibit, several pharmacol. agents have been developed to disrupt its E3 ligase activity, therefore blocking its potential oncogenic activity to mitigate tumorigenesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoiqKWKPAR9CLVg90H21EOLACvtfcHk0ljeEjnAEHPMsg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvValt7zO&md5=acd511a43f773fd6ea65179e18403ffc</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1016%2Fj.bbcan.2017.11.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbcan.2017.11.001%26sid%3Dliteratum%253Aachs%26aulast%3DCheng%26aufirst%3DJ.%26aulast%3DGuo%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DZ.%26aulast%3DNorth%26aufirst%3DB.%2BJ.%26aulast%3DTao%26aufirst%3DK.%26aulast%3DDai%26aufirst%3DX.%26aulast%3DWei%26aufirst%3DW.%26atitle%3DFunctional%2520analysis%2520of%2520cullin%25203%2520E3%2520ligases%2520in%2520tumorigenesis%26jtitle%3DBiochim.%2520Biophys.%2520Acta%252C%2520Rev.%2520Cancer%26date%3D2018%26volume%3D1869%26spage%3D11%26epage%3D28%26doi%3D10.1016%2Fj.bbcan.2017.11.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kumari, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumar, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lynn, A.</span></span> <span> </span><span class="NLM_article-title"><i>g_mmpbsa -</i> a GROMACS tool for high-throughput MM-PBSA calculations</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">1951</span>– <span class="NLM_lpage">1962</span>, <span class="refDoi"> DOI: 10.1021/ci500020m</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci500020m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=1%3ACAS%3A528%3ADC%252BC2cXotlWiu7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2014&pages=1951-1962&author=R.+Kumariauthor=R.+Kumarauthor=A.+Lynn&title=g_mmpbsa+-+a+GROMACS+tool+for+high-throughput+MM-PBSA+calculations&doi=10.1021%2Fci500020m"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">g_mmpbsa-A GROMACS Tool for High-Throughput MM-PBSA Calculations</span></div><div class="casAuthors">Kumari, Rashmi; Kumar, Rajendra; Lynn, Andrew</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1951-1962</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Mol. mechanics Poisson-Boltzmann surface area (MM-PBSA), a method to est. interaction free energies, has been increasingly used in the study of biomol. interactions.  Recently, this method has also been applied as a scoring function in computational drug design.  Here a new tool g_mmpbsa, which implements the MM-PBSA approach using subroutines written inhouse or sourced from the GROMACS and APBS packages is described. g_mmpbsa was developed as part of the Open Source Drug Discovery (OSDD) consortium.  Its aim is to integrate high-throughput mol. dynamics (MD) simulations with binding energy calcns.  The tool provides options to select alternative at. radii and different nonpolar solvation models including models based on the solvent accessible surface area (SASA), solvent accessible vol. (SAV), and a model which contains both repulsive (SASA-SAV) and attractive components (described using a Weeks-Chandler-Andersen like integral method).  We showcase the effectiveness of the tool by comparing the calcd. interaction energy of 37 structurally diverse HIV-1 protease inhibitor complexes with their exptl. binding free energies.  The effect of varying several combinations of input parameters such as at. radii, dielec. const., grid resoln., solute-solvent dielec. boundary definition, and nonpolar models was investigated. g_mmpbsa can also be used to est. the energy contribution per residue to the binding energy.  It has been used to identify those residues in HIV-1 protease that are most crit. for binding a range of inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrckNA7gLtUGrVg90H21EOLACvtfcHk0ljeEjnAEHPMsg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXotlWiu7o%253D&md5=e70a138b71eed07054d58f91ba1a227f</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1021%2Fci500020m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci500020m%26sid%3Dliteratum%253Aachs%26aulast%3DKumari%26aufirst%3DR.%26aulast%3DKumar%26aufirst%3DR.%26aulast%3DLynn%26aufirst%3DA.%26atitle%3Dg_mmpbsa%2520-%2520a%2520GROMACS%2520tool%2520for%2520high-throughput%2520MM-PBSA%2520calculations%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2014%26volume%3D54%26spage%3D1951%26epage%3D1962%26doi%3D10.1021%2Fci500020m" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wu, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nielsen, T. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clausen, M. H.</span></span> <span> </span><span class="NLM_article-title">Small-molecule kinase inhibitors: an analysis of FDA-approved drugs</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">5</span>– <span class="NLM_lpage">10</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2015.07.008</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=10.1016%2Fj.drudis.2015.07.008" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=26210956" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1Gru73P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2016&pages=5-10&author=P.+Wuauthor=T.+E.+Nielsenauthor=M.+H.+Clausen&title=Small-molecule+kinase+inhibitors%3A+an+analysis+of+FDA-approved+drugs&doi=10.1016%2Fj.drudis.2015.07.008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Small-molecule kinase inhibitors: an analysis of FDA-approved drugs</span></div><div class="casAuthors">Wu, Peng; Nielsen, Thomas E.; Clausen, Mads H.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">5-10</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Small-mol. kinase inhibitors (SMKIs), 28 of which are approved by the US Food and Drug Administration (FDA), have been actively pursued as promising targeted therapeutics.  Here, we assess the key structural and physicochem. properties, target selectivity and mechanism of function, and therapeutic indications of these approved inhibitors.  Our anal. showed that >30% of approved SMKIs have a mol. wt. (MW) exceeding 500 and all have a total ring count of between three and five.  The assumption that type II inhibitors tend to be more selective than type I inhibitors has been proved to be unreliable.  Although previous SMKI research was concd. on tyrosine kinase inhibitors for cancer treatment, recent progress indicates diversification of SMKI research in terms of new targets, mechanistic types, and therapeutic indications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpZrY68ZOI2xLVg90H21EOLACvtfcHk0ljeEjnAEHPMsg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht1Gru73P&md5=228524fa464393053ecbfac19e994929</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2015.07.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2015.07.008%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DP.%26aulast%3DNielsen%26aufirst%3DT.%2BE.%26aulast%3DClausen%26aufirst%3DM.%2BH.%26atitle%3DSmall-molecule%2520kinase%2520inhibitors%253A%2520an%2520analysis%2520of%2520FDA-approved%2520drugs%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2016%26volume%3D21%26spage%3D5%26epage%3D10%26doi%3D10.1016%2Fj.drudis.2015.07.008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sun, Q.-A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hess, D. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miyagi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stamler, J. S.</span></span> <span> </span><span class="NLM_article-title">Off-target thiol alkylation by the NADPH oxidase inhibitor 3-benzyl-7-(2-benzoxazolyl)thio-1,2,3-triazolo[4,5-d]pyrimidine (VAS2870)</span>. <i>Free Radical Biol. Med.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">1897</span>– <span class="NLM_lpage">1902</span>, <span class="refDoi"> DOI: 10.1016/j.freeradbiomed.2012.02.046</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=10.1016%2Fj.freeradbiomed.2012.02.046" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=22406319" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=1%3ACAS%3A528%3ADC%252BC38XmtlWmsLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2012&pages=1897-1902&author=Q.-A.+Sunauthor=D.+T.+Hessauthor=B.+Wangauthor=M.+Miyagiauthor=J.+S.+Stamler&title=Off-target+thiol+alkylation+by+the+NADPH+oxidase+inhibitor+3-benzyl-7-%282-benzoxazolyl%29thio-1%2C2%2C3-triazolo%5B4%2C5-d%5Dpyrimidine+%28VAS2870%29&doi=10.1016%2Fj.freeradbiomed.2012.02.046"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Off-target thiol alkylation by the NADPH oxidase inhibitor 3-benzyl-7-(2-benzoxazolyl)thio-1,2,3-triazolo[4,5-d]pyrimidine (VAS2870)</span></div><div class="casAuthors">Sun, Qi-An; Hess, Douglas T.; Wang, Benlian; Miyagi, Masaru; Stamler, Jonathan S.</div><div class="citationInfo"><span class="NLM_cas:title">Free Radical Biology & Medicine</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1897-1902</span>CODEN:
                <span class="NLM_cas:coden">FRBMEH</span>;
        ISSN:<span class="NLM_cas:issn">0891-5849</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Specific inhibitors of the prodn. of reactive oxygen species (ROS) by the NADPH oxidases (Nox's) are potentially important therapeutic agents in the wide range of human diseases that are characterized by excessive ROS prodn.  It has been proposed that VAS2870 (3-benzyl-7-(2-benzoxazolyl)thio-1,2,3- triazolo[4,5-d]pyrimidine), identified as an inhibitor of Nox2 by small-mol. screening, may serve as an example of such an agent.  Here we show that VAS2870 inhibits ROS prodn. in the sarcoplasmic reticulum (SR) of mammalian skeletal muscle, previously identified with Nox4, and thereby abrogates O2-coupled redox regulation of the ryanodine receptor-Ca2+ channel (RyR1).  However, we also find that VAS2870 modifies directly identified cysteine thiols within RyR1.  Mass spectrometric anal. of RyR1 exposed in situ to VAS2870 and of VAS2870-treated glutathione indicated that thiol modification is through alkylation by the benzyltriazolopyrimidine moiety of VAS2870.  Thus, VAS2870 exerts significant off-target effects, and thiol alkylation by VAS2870 (and closely related Nox inhibitors) may in fact replicate some of the effects of ROS on cellular thiol redox status.  In addn., we show that SR-localized Nox4 is inhibited by other thiol-alkylating agents, consistent with a causal role for cysteine modification in the inhibition of ROS prodn. by VAS2870.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoSMAxkJX1GbbVg90H21EOLACvtfcHk0lhiRUG8IVVRcA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XmtlWmsLs%253D&md5=43dae8c22f646bd49a41392cf6d092f9</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1016%2Fj.freeradbiomed.2012.02.046&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.freeradbiomed.2012.02.046%26sid%3Dliteratum%253Aachs%26aulast%3DSun%26aufirst%3DQ.-A.%26aulast%3DHess%26aufirst%3DD.%2BT.%26aulast%3DWang%26aufirst%3DB.%26aulast%3DMiyagi%26aufirst%3DM.%26aulast%3DStamler%26aufirst%3DJ.%2BS.%26atitle%3DOff-target%2520thiol%2520alkylation%2520by%2520the%2520NADPH%2520oxidase%2520inhibitor%25203-benzyl-7-%25282-benzoxazolyl%2529thio-1%252C2%252C3-triazolo%255B4%252C5-d%255Dpyrimidine%2520%2528VAS2870%2529%26jtitle%3DFree%2520Radical%2520Biol.%2520Med.%26date%3D2012%26volume%3D52%26spage%3D1897%26epage%3D1902%26doi%3D10.1016%2Fj.freeradbiomed.2012.02.046" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Singh, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stoffel, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gogineni, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poluri, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pfister, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaha, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pathak, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bosl, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cordon-Cardo, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shah, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rao, P. H.</span></span> <span> </span><span class="NLM_article-title">Amplification of the 3q26.3 locus is associated with progression to invasive cancer and is a negative prognostic factor in head and neck squamous cell carcinomas</span>. <i>Am. J. Pathol.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>161</i></span>,  <span class="NLM_fpage">365</span>– <span class="NLM_lpage">371</span>, <span class="refDoi"> DOI: 10.1016/S0002-9440(10)64191-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=10.1016%2FS0002-9440%2810%2964191-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=12163360" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=1%3ACAS%3A528%3ADC%252BD38XntFektb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=161&publication_year=2002&pages=365-371&author=B.+Singhauthor=A.+Stoffelauthor=S.+Gogineniauthor=A.+Poluriauthor=D.+G.+Pfisterauthor=A.+R.+Shahaauthor=A.+Pathakauthor=G.+Boslauthor=C.+Cordon-Cardoauthor=J.+P.+Shahauthor=P.+H.+Rao&title=Amplification+of+the+3q26.3+locus+is+associated+with+progression+to+invasive+cancer+and+is+a+negative+prognostic+factor+in+head+and+neck+squamous+cell+carcinomas&doi=10.1016%2FS0002-9440%2810%2964191-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Amplification of the 3q26.3 locus is associated with progression to invasive cancer and is a negative prognostic factor in head and neck squamous cell carcinomas</span></div><div class="casAuthors">Singh, Bhuvanesh; Stoffel, Archontoula; Gogineni, Swarna; Poluri, Ashok; Pfister, David G.; Shaha, Ashok R.; Pathak, Alok; Bosl, George; Cordon-Cardo, Carlos; Shah, Jatin P.; Rao, Pulivarthi H.</div><div class="citationInfo"><span class="NLM_cas:title">American Journal of Pathology</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">161</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">365-371</span>CODEN:
                <span class="NLM_cas:coden">AJPAA4</span>;
        ISSN:<span class="NLM_cas:issn">0002-9440</span>.
    
            (<span class="NLM_cas:orgname">American Society for Investigative Pathology</span>)
        </div><div class="casAbstract">Amplification of the 3q26-q27 has a high prevalence in squamous cell carcinomas of mucosal origin, including those originating in the head and neck region.  To elucidate its role as a prognostic tool in head and neck squarnous cell carcinoma, a yeast artificial chromosome (YAC) contig spanning the entire 3q26-27 region was constructed.  The minimal region of amplification was refined within a 1- to 2-Mb genomic segment contained within three overlapping, nonchimeric YAC clones using sequential fluorescent in situ hybridization anal.  These YAC clones contg. the apex of amplification were used to develop a two-color fluorescence in situ hybridization assay and applied to the detection of 3q copy nos. in interphase nuclei on archival tumor tissue from 29 cases of normal mucosa, 20 of premalignant mucosa, and 50 of invasive head and neck squamous cell carcinomas.  The presence of 3q amplification increased from 3% in normal mucosa to 25% in premalignant mucosa and 56% in invasive cancers (P < 0.01).  In invasive tumors, low-level 3q amplification (3 to 4 × copy no.) was identified in 18 of 50 primary head and neck cancers and high-level amplification (>4 × copy no.) in 10 of 50 cases.  With a median follow-up of 82.5 mo, an increasing proportion of recurrences (32%, 72%, and 90%; P = 0.003) and cancer-related deaths (14%, 44%, and 70%; P = 0.006) were seen in patients with normal 3q copy no., low-level amplification, and high-level amplification, resp.  The 3-yr disease-free (69%, 56%, and 10%; P = 0.001) and cause-specific (94%, 83%, and 40%; P = 0.01) survivals also decreased from normal copy no. to low-level and high-level amplification.  Only high-level amplification at 3q remained a significant prognostic variable on multivariate anal. including common prognostic predictors for both disease-free (relative risk, 5.1; 95% confidence interval = 1.9 to 13.9) and cause-specific survival (relative risk, 7.6; 95% confidence interval = 1.9 to 29.6).  The findings suggest that the 3q copy no. status is an important marker for tumor progression and prognostication in patients with head and neck squamous cell carcinoma.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr0R6UH5w2KgrVg90H21EOLACvtfcHk0lhiRUG8IVVRcA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XntFektb4%253D&md5=d9e546b51e67befcde4da23006e3fa86</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1016%2FS0002-9440%2810%2964191-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0002-9440%252810%252964191-0%26sid%3Dliteratum%253Aachs%26aulast%3DSingh%26aufirst%3DB.%26aulast%3DStoffel%26aufirst%3DA.%26aulast%3DGogineni%26aufirst%3DS.%26aulast%3DPoluri%26aufirst%3DA.%26aulast%3DPfister%26aufirst%3DD.%2BG.%26aulast%3DShaha%26aufirst%3DA.%2BR.%26aulast%3DPathak%26aufirst%3DA.%26aulast%3DBosl%26aufirst%3DG.%26aulast%3DCordon-Cardo%26aufirst%3DC.%26aulast%3DShah%26aufirst%3DJ.%2BP.%26aulast%3DRao%26aufirst%3DP.%2BH.%26atitle%3DAmplification%2520of%2520the%25203q26.3%2520locus%2520is%2520associated%2520with%2520progression%2520to%2520invasive%2520cancer%2520and%2520is%2520a%2520negative%2520prognostic%2520factor%2520in%2520head%2520and%2520neck%2520squamous%2520cell%2520carcinomas%26jtitle%3DAm.%2520J.%2520Pathol.%26date%3D2002%26volume%3D161%26spage%3D365%26epage%3D371%26doi%3D10.1016%2FS0002-9440%2810%2964191-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Heselmeyer, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Macville, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schrock, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blegen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hellstrom, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shah, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Auer, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ried, T.</span></span> <span> </span><span class="NLM_article-title">Advanced-stage cervical carcinomas are defined by a recurrent pattern of chromosomal aberrations revealing high genetic instability and a consistent gain of chromosome arm 3q</span>. <i>Genes, Chromosomes Cancer</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">233</span>– <span class="NLM_lpage">240</span>, <span class="refDoi"> DOI: 10.1002/(SICI)1098-2264(199708)19:4<233::AID-GCC5>3.0.CO;2-Y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=10.1002%2F%28SICI%291098-2264%28199708%2919%3A4%3C233%3A%3AAID-GCC5%3E3.0.CO%3B2-Y" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=9258658" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=1%3ACAS%3A280%3ADyaK2svgtlaruw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=1997&pages=233-240&author=K.+Heselmeyerauthor=M.+Macvilleauthor=E.+Schrockauthor=H.+Blegenauthor=A.+C.+Hellstromauthor=K.+Shahauthor=G.+Auerauthor=T.+Ried&title=Advanced-stage+cervical+carcinomas+are+defined+by+a+recurrent+pattern+of+chromosomal+aberrations+revealing+high+genetic+instability+and+a+consistent+gain+of+chromosome+arm+3q&doi=10.1002%2F%28SICI%291098-2264%28199708%2919%3A4%3C233%3A%3AAID-GCC5%3E3.0.CO%3B2-Y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Advanced-stage cervical carcinomas are defined by a recurrent pattern of chromosomal aberrations revealing high genetic instability and a consistent gain of chromosome arm 3q</span></div><div class="casAuthors">Heselmeyer K; Macville M; Schrock E; Blegen H; Hellstrom A C; Shah K; Auer G; Ried T</div><div class="citationInfo"><span class="NLM_cas:title">Genes, chromosomes & cancer</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">233-40</span>
        ISSN:<span class="NLM_cas:issn">1045-2257</span>.
    </div><div class="casAbstract">We have analyzed 30 cases of advanced-stage cervical squamous cell carcinoma (stages IIb-IV) by comparative genomic hybridization (CGH).  The most consistent chromosomal gain in the aneuploid tumors was mapped to chromosome arm 3q in 77% of the cases.  Acquisition of genetic material also occurred frequently on Iq (47%), 5p (30%), 6p (27%), and 20 (23%).  Recurrent losses were mapped on 2q (33%), 3p (50%), 4 (33%), 8p (23%), and 13q (27%).  High-level copy number increases were mapped to chromosome 8, chromosome arms 3q, 5p, 8q, 12p, 14q, 17q, 19q, 20p, and 20q, and chromosomal bands 3q26-27, 9p23-24, 11q22-23, and 12p13.  In the majority of the cases, the presence of high-risk human papilloma virus genomes was detected.  High proliferative activity was accompanied by crude aneuploidy.  Increased p21/WAF-I activity, but low or undetectable expression of TP53 were representative for the immunophenotype.  This study confirms the importance of a gain of chromosome arm 3q in cervical carcinogenesis and identifies additional, recurrent chromosomal aberrations that are required for progression from stage I tumors to advanced-stage carcinomas.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQyf87gxuNQ5dhSDZx9hyjKfW6udTcc2eYB4CIAqhAeaLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADyaK2svgtlaruw%253D%253D&md5=d7db7362d12bf4e3787ea5ad4807e15f</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1002%2F%28SICI%291098-2264%28199708%2919%3A4%3C233%3A%3AAID-GCC5%3E3.0.CO%3B2-Y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F%2528SICI%25291098-2264%2528199708%252919%253A4%253C233%253A%253AAID-GCC5%253E3.0.CO%253B2-Y%26sid%3Dliteratum%253Aachs%26aulast%3DHeselmeyer%26aufirst%3DK.%26aulast%3DMacville%26aufirst%3DM.%26aulast%3DSchrock%26aufirst%3DE.%26aulast%3DBlegen%26aufirst%3DH.%26aulast%3DHellstrom%26aufirst%3DA.%2BC.%26aulast%3DShah%26aufirst%3DK.%26aulast%3DAuer%26aufirst%3DG.%26aulast%3DRied%26aufirst%3DT.%26atitle%3DAdvanced-stage%2520cervical%2520carcinomas%2520are%2520defined%2520by%2520a%2520recurrent%2520pattern%2520of%2520chromosomal%2520aberrations%2520revealing%2520high%2520genetic%2520instability%2520and%2520a%2520consistent%2520gain%2520of%2520chromosome%2520arm%25203q%26jtitle%3DGenes%252C%2520Chromosomes%2520Cancer%26date%3D1997%26volume%3D19%26spage%3D233%26epage%3D240%26doi%3D10.1002%2F%28SICI%291098-2264%28199708%2919%3A4%3C233%3A%3AAID-GCC5%3E3.0.CO%3B2-Y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Petersen, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bujard, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petersen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wolf, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goeze, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwendel, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Langreck, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gellert, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reichel, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Just, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">du Manoir, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cremer, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dietel, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ried, T.</span></span> <span> </span><span class="NLM_article-title">Patterns of chromosomal imbalances in adenocarcinoma and squamous cell carcinoma of the lung</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">2331</span>– <span class="NLM_lpage">2335</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=1997&pages=2331-2335&author=I.+Petersenauthor=M.+Bujardauthor=S.+Petersenauthor=G.+Wolfauthor=A.+Goezeauthor=A.+Schwendelauthor=H.+Langreckauthor=K.+Gellertauthor=M.+Reichelauthor=K.+Justauthor=S.+du+Manoirauthor=T.+Cremerauthor=M.+Dietelauthor=T.+Ried&title=Patterns+of+chromosomal+imbalances+in+adenocarcinoma+and+squamous+cell+carcinoma+of+the+lung"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DPetersen%26aufirst%3DI.%26aulast%3DBujard%26aufirst%3DM.%26aulast%3DPetersen%26aufirst%3DS.%26aulast%3DWolf%26aufirst%3DG.%26aulast%3DGoeze%26aufirst%3DA.%26aulast%3DSchwendel%26aufirst%3DA.%26aulast%3DLangreck%26aufirst%3DH.%26aulast%3DGellert%26aufirst%3DK.%26aulast%3DReichel%26aufirst%3DM.%26aulast%3DJust%26aufirst%3DK.%26aulast%3Ddu%2BManoir%26aufirst%3DS.%26aulast%3DCremer%26aufirst%3DT.%26aulast%3DDietel%26aufirst%3DM.%26aulast%3DRied%26aufirst%3DT.%26atitle%3DPatterns%2520of%2520chromosomal%2520imbalances%2520in%2520adenocarcinoma%2520and%2520squamous%2520cell%2520carcinoma%2520of%2520the%2520lung%26jtitle%3DCancer%2520Res.%26date%3D1997%26volume%3D57%26spage%3D2331%26epage%3D2335" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Z. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y. S.</span></span> <span> </span><span class="NLM_article-title">Clinical significance of SCCRO (DCUN1D1) in prostate cancer and its proliferation-inhibiting effect on Lncap cells</span>. <i>Eur. Rev. Med. Pharm. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">4283</span>– <span class="NLM_lpage">4291</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=29077169" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=1%3ACAS%3A280%3ADC%252BC1M7ltF2qsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2017&pages=4283-4291&author=Z.+H.+Zhangauthor=J.+Liauthor=F.+Luoauthor=Y.+S.+Wang&title=Clinical+significance+of+SCCRO+%28DCUN1D1%29+in+prostate+cancer+and+its+proliferation-inhibiting+effect+on+Lncap+cells"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Clinical significance of SCCRO (DCUN1D1) in prostate cancer and its proliferation-inhibiting effect on Lncap cells</span></div><div class="casAuthors">Zhang Z-H; Li J; Luo F; Wang Y-S</div><div class="citationInfo"><span class="NLM_cas:title">European review for medical and pharmacological sciences</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">4283-4291</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">OBJECTIVE:    SCCRO/DCUN1D1/DCN1 (squamous cell carcinoma-related oncogene/defective in cullin neddylation 1 domain containing 1/defective in cullin neddylation) is considered as an oncogene, but its role in the prostate cancer (PC) is still not clear.  The current study aims to investigate the expression of SCCRO in PC tumor tissues, further its clinical significance, and proliferation inhibiting effect on PC cells in vitro.  PATIENTS AND METHODS:  RT-PCR was used to detect the expression of SCCRO in PC tissue and corresponding adjacent normal tissues from 160 cases, and its relationship with clinical pathological characteristics was analyzed.  Small interfering RNA (siRNA) expression plasmid targeting SCCRO gene was constructed and transferred into PC cell line Lncap.  The effect on proliferation was observed by CCK8 assay, and its influence on invasion and migration of Lncap cells was studied by Transwell Matrigel assay after SCCRO gene was silenced.  The expression of focal adhesion kinase (FAK) and matrix metalloproteinase-2 (MMP-2) influenced by SCCRO silencing were detected by Western blot.  RESULTS:  mRNA expression of SCCRO protein increased significantly in cancer tissues compared to adjacent normal tissue, especially for T3+T4, N+, and III+IV patients (p<0.05).  SCCRO expression was an independent prognostic factor (p<0.05).  After SCCRO gene was knocked down by siRNA, the SCCRO protein level decreased 78.4% in the siRNA-3 group.  By CCK8 assay, knocking down SCCRO in Lncap significantly reduced the cell proliferation, as well as its migration and invasion capability compared to siRNA-control group (p<0.01) by transwell invasion and migration assay.  The expression of FAK and MMP-2 also reduced in siRNA-3 group compared to siRNA control group (p<0.01).  CONCLUSIONS:  SCCRO is associated with progression and prognosis of PC.  After SCCRO gene was transferred, the growth of Lncap cells was inhibited, and ability of invasion and migration decreased by reducing the expression of FAK and MMP-2.  SCCRO has potential to become a new target for the treatment of PC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQUF5iSyBgoY-R_TJajKxJDfW6udTcc2eYB4CIAqhAeaLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1M7ltF2qsg%253D%253D&md5=bc432df7a5551dede3c22a9add47d71b</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DZ.%2BH.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DLuo%26aufirst%3DF.%26aulast%3DWang%26aufirst%3DY.%2BS.%26atitle%3DClinical%2520significance%2520of%2520SCCRO%2520%2528DCUN1D1%2529%2520in%2520prostate%2520cancer%2520and%2520its%2520proliferation-inhibiting%2520effect%2520on%2520Lncap%2520cells%26jtitle%3DEur.%2520Rev.%2520Med.%2520Pharm.%2520Sci.%26date%3D2017%26volume%3D21%26spage%3D4283%26epage%3D4291" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pfleger, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pan, Z. Q.</span></span> <span> </span><span class="NLM_article-title">Mono-ubiquitination drives nuclear export of the human DCN1-like protein hDCNL1</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>286</i></span>,  <span class="NLM_fpage">34060</span>– <span class="NLM_lpage">34070</span>, <span class="refDoi"> DOI: 10.1074/jbc.M111.273045</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=10.1074%2Fjbc.M111.273045" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=286&publication_year=2011&pages=34060-34070&author=K.+Wuauthor=H.+Yanauthor=L.+Fangauthor=X.+Wangauthor=C.+Pflegerauthor=X.+Jiangauthor=L.+Huangauthor=Z.+Q.+Pan&title=Mono-ubiquitination+drives+nuclear+export+of+the+human+DCN1-like+protein+hDCNL1&doi=10.1074%2Fjbc.M111.273045"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M111.273045&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M111.273045%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DK.%26aulast%3DYan%26aufirst%3DH.%26aulast%3DFang%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DPfleger%26aufirst%3DC.%26aulast%3DJiang%26aufirst%3DX.%26aulast%3DHuang%26aufirst%3DL.%26aulast%3DPan%26aufirst%3DZ.%2BQ.%26atitle%3DMono-ubiquitination%2520drives%2520nuclear%2520export%2520of%2520the%2520human%2520DCN1-like%2520protein%2520hDCNL1%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2011%26volume%3D286%26spage%3D34060%26epage%3D34070%26doi%3D10.1074%2Fjbc.M111.273045" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Soucy, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, P. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Milhollen, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berger, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gavin, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adhikari, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brownell, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burke, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cardin, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Critchley, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cullis, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doucette, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garnsey, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaulin, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gershman, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lublinsky, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McDonald, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mizutani, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Narayanan, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olhava, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peluso, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rezaei, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sintchak, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Talreja, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Traore, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vyskocil, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weatherhead, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dick, L. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Claiborne, C. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rolfe, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bolen, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Langston, S. P.</span></span> <span> </span><span class="NLM_article-title">An inhibitor of nedd8-activating enzyme as a new approach to treat cancer</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>458</i></span>,  <span class="NLM_fpage">732</span>– <span class="NLM_lpage">736</span>, <span class="refDoi"> DOI: 10.1038/nature07884</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=10.1038%2Fnature07884" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=19360080" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=1%3ACAS%3A528%3ADC%252BD1MXksVenu7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=458&publication_year=2009&pages=732-736&author=T.+A.+Soucyauthor=P.+G.+Smithauthor=M.+A.+Milhollenauthor=A.+J.+Bergerauthor=J.+M.+Gavinauthor=S.+Adhikariauthor=J.+E.+Brownellauthor=K.+E.+Burkeauthor=D.+P.+Cardinauthor=S.+Critchleyauthor=C.+A.+Cullisauthor=A.+Doucetteauthor=J.+J.+Garnseyauthor=J.+L.+Gaulinauthor=R.+E.+Gershmanauthor=A.+R.+Lublinskyauthor=A.+McDonaldauthor=H.+Mizutaniauthor=U.+Narayananauthor=E.+J.+Olhavaauthor=S.+Pelusoauthor=M.+Rezaeiauthor=M.+D.+Sintchakauthor=T.+Talrejaauthor=M.+P.+Thomasauthor=T.+Traoreauthor=S.+Vyskocilauthor=G.+S.+Weatherheadauthor=J.+Yuauthor=J.+Zhangauthor=L.+R.+Dickauthor=C.+F.+Claiborneauthor=M.+Rolfeauthor=J.+B.+Bolenauthor=S.+P.+Langston&title=An+inhibitor+of+nedd8-activating+enzyme+as+a+new+approach+to+treat+cancer&doi=10.1038%2Fnature07884"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">An inhibitor of NEDD8-activating enzyme as a new approach to treat cancer</span></div><div class="casAuthors">Soucy, Teresa A.; Smith, Peter G.; Milhollen, Michael A.; Berger, Allison J.; Gavin, James M.; Adhikari, Sharmila; Brownell, James E.; Burke, Kristine E.; Cardin, David P.; Critchley, Stephen; Cullis, Courtney A.; Doucette, Amanda; Garnsey, James J.; Gaulin, Jeffrey L.; Gershman, Rachel E.; Lublinsky, Anna R.; McDonald, Alice; Mizutani, Hirotake; Narayanan, Usha; Olhava, Edward J.; Peluso, Stephane; Rezaei, Mansoureh; Sintchak, Michael D.; Talreja, Tina; Thomas, Michael P.; Traore, Tary; Vyskocil, Stepan; Weatherhead, Gabriel S.; Yu, Jie; Zhang, Julie; Dick, Lawrence R.; Claiborne, Christopher F.; Rolfe, Mark; Bolen, Joseph B.; Langston, Steven P.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">458</span>
        (<span class="NLM_cas:issue">7239</span>),
    <span class="NLM_cas:pages">732-736</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The clin. development of an inhibitor of cellular proteasome function suggests that compds. targeting other components of the ubiquitin-proteasome system might prove useful for the treatment of human malignancies.  NEDD8-activating enzyme (NAE) is an essential component of the NEDD8 conjugation pathway that controls the activity of the cullin-RING subtype of ubiquitin ligases, thereby regulating the turnover of a subset of proteins upstream of the proteasome.  Substrates of cullin-RING ligases have important roles in cellular processes assocd. with cancer cell growth and survival pathways.  Here we describe MLN4924, a potent and selective inhibitor of NAE.  MLN4924 disrupts cullin-RING ligase-mediated protein turnover leading to apoptotic death in human tumor cells by a new mechanism of action, the deregulation of S-phase DNA synthesis.  MLN4924 suppressed the growth of human tumor xenografts in mice at compd. exposures that were well tolerated.  Our data suggest that NAE inhibitors may hold promise for the treatment of cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr8fx7OnZtYjbVg90H21EOLACvtfcHk0lh1OFvI8ndZ1A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXksVenu7Y%253D&md5=365d4b4feac3994849050e00f1aae5e0</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1038%2Fnature07884&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature07884%26sid%3Dliteratum%253Aachs%26aulast%3DSoucy%26aufirst%3DT.%2BA.%26aulast%3DSmith%26aufirst%3DP.%2BG.%26aulast%3DMilhollen%26aufirst%3DM.%2BA.%26aulast%3DBerger%26aufirst%3DA.%2BJ.%26aulast%3DGavin%26aufirst%3DJ.%2BM.%26aulast%3DAdhikari%26aufirst%3DS.%26aulast%3DBrownell%26aufirst%3DJ.%2BE.%26aulast%3DBurke%26aufirst%3DK.%2BE.%26aulast%3DCardin%26aufirst%3DD.%2BP.%26aulast%3DCritchley%26aufirst%3DS.%26aulast%3DCullis%26aufirst%3DC.%2BA.%26aulast%3DDoucette%26aufirst%3DA.%26aulast%3DGarnsey%26aufirst%3DJ.%2BJ.%26aulast%3DGaulin%26aufirst%3DJ.%2BL.%26aulast%3DGershman%26aufirst%3DR.%2BE.%26aulast%3DLublinsky%26aufirst%3DA.%2BR.%26aulast%3DMcDonald%26aufirst%3DA.%26aulast%3DMizutani%26aufirst%3DH.%26aulast%3DNarayanan%26aufirst%3DU.%26aulast%3DOlhava%26aufirst%3DE.%2BJ.%26aulast%3DPeluso%26aufirst%3DS.%26aulast%3DRezaei%26aufirst%3DM.%26aulast%3DSintchak%26aufirst%3DM.%2BD.%26aulast%3DTalreja%26aufirst%3DT.%26aulast%3DThomas%26aufirst%3DM.%2BP.%26aulast%3DTraore%26aufirst%3DT.%26aulast%3DVyskocil%26aufirst%3DS.%26aulast%3DWeatherhead%26aufirst%3DG.%2BS.%26aulast%3DYu%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DDick%26aufirst%3DL.%2BR.%26aulast%3DClaiborne%26aufirst%3DC.%2BF.%26aulast%3DRolfe%26aufirst%3DM.%26aulast%3DBolen%26aufirst%3DJ.%2BB.%26aulast%3DLangston%26aufirst%3DS.%2BP.%26atitle%3DAn%2520inhibitor%2520of%2520nedd8-activating%2520enzyme%2520as%2520a%2520new%2520approach%2520to%2520treat%2520cancer%26jtitle%3DNature%26date%3D2009%26volume%3D458%26spage%3D732%26epage%3D736%26doi%3D10.1038%2Fnature07884" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tong, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Si, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, W.</span></span> <span> </span><span class="NLM_article-title">MLN4924 (Pevonedistat), a protein neddylation inhibitor, suppresses proliferation and migration of human clear cell renal cell carcinoma</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">5599</span>, <span class="refDoi"> DOI: 10.1038/s41598-017-06098-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=10.1038%2Fs41598-017-06098-y" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=28717191" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=1%3ACAS%3A280%3ADC%252BC1cjosFWksg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2017&pages=5599&author=S.+Tongauthor=Y.+Siauthor=H.+Yuauthor=L.+Zhangauthor=P.+Xieauthor=W.+Jiang&title=MLN4924+%28Pevonedistat%29%2C+a+protein+neddylation+inhibitor%2C+suppresses+proliferation+and+migration+of+human+clear+cell+renal+cell+carcinoma&doi=10.1038%2Fs41598-017-06098-y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">MLN4924 (Pevonedistat), a protein neddylation inhibitor, suppresses proliferation and migration of human clear cell renal cell carcinoma</span></div><div class="casAuthors">Tong Shuai; Si Yang; Yu Hefen; Xie Ping; Jiang Wenguo; Tong Shuai; Si Yang; Yu Hefen; Xie Ping; Jiang Wenguo; Zhang Lingqiang; Jiang Wenguo</div><div class="citationInfo"><span class="NLM_cas:title">Scientific reports</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">5599</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Neddylation is a post-translational protein modification associated with cancer development.  MLN4924 is a neddylation inhibitor currently under investigation in multiple phase I studies on various malignancies, and its clincal name is Pevonedistat.  It has been documented that MLN4924 blocks Cullins neddylation and inactivates CRLs and, in turn, triggers cell-cycle arrest, apoptosis, senescence and autophagy in many cancer cells.  In this study, we investigated the anti-tumor effect of MLN4924 in human clear cell renal carcinoma (ccRCC).  Levels of both Nedd8 activating enzyme E1 and Nedd8-conjugating enzyme E2 were higher in ccRCC tissues and RCC cancer cells than in normal.  Moreover, MLN4924 treatment led to rapid inhibition of Cullin1 neddylation and notably suppressed growth and survival as well as migration in a dose-and time-dependent manner.  Mechanistic studies revealed that MLN4924 induced the accumulation of a number of CRL substrates, including p21, p27 and Wee1 to trigger DNA damage and induce growth arrest at the G2/M phase.  MLN4924 also induced anti-migration and anti-invasion by activating E-cadherin and repressing Vimentin.  Taken together, this study provides the first evidence that neddylation pathway is overactive in ccRCC and that MLN4924 induces dose-dependent anti-proliferation, anti-migration, anti-invasion in ccRCC cells.  The study thus indicates that MLN4924 has potential therapeutic value for the clinical treatment of renal cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQrdwMzHGpO0rrIbt-tGZmxfW6udTcc2eaHkuTicH4DuLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1cjosFWksg%253D%253D&md5=1e0a80b533adac1674de53b9b68750a7</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1038%2Fs41598-017-06098-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41598-017-06098-y%26sid%3Dliteratum%253Aachs%26aulast%3DTong%26aufirst%3DS.%26aulast%3DSi%26aufirst%3DY.%26aulast%3DYu%26aufirst%3DH.%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DXie%26aufirst%3DP.%26aulast%3DJiang%26aufirst%3DW.%26atitle%3DMLN4924%2520%2528Pevonedistat%2529%252C%2520a%2520protein%2520neddylation%2520inhibitor%252C%2520suppresses%2520proliferation%2520and%2520migration%2520of%2520human%2520clear%2520cell%2520renal%2520cell%2520carcinoma%26jtitle%3DSci.%2520Rep.%26date%3D2017%26volume%3D7%26spage%3D5599%26doi%3D10.1038%2Fs41598-017-06098-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, H. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, G. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, S. S.</span></span> <span> </span><span class="NLM_article-title">MLN4924 suppresses neddylation and induces cell cycle arrest, senescence, and apoptosis in human osteosarcoma</span>. <i>Oncotarget</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">45263</span>– <span class="NLM_lpage">45274</span>, <span class="refDoi"> DOI: 10.18632/oncotarget.9481</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=10.18632%2Foncotarget.9481" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=45263-45274&author=Y.+Zhangauthor=C.+C.+Shiauthor=H.+P.+Zhangauthor=G.+Q.+Liauthor=S.+S.+Li&title=MLN4924+suppresses+neddylation+and+induces+cell+cycle+arrest%2C+senescence%2C+and+apoptosis+in+human+osteosarcoma&doi=10.18632%2Foncotarget.9481"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.9481&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.9481%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DShi%26aufirst%3DC.%2BC.%26aulast%3DZhang%26aufirst%3DH.%2BP.%26aulast%3DLi%26aufirst%3DG.%2BQ.%26aulast%3DLi%26aufirst%3DS.%2BS.%26atitle%3DMLN4924%2520suppresses%2520neddylation%2520and%2520induces%2520cell%2520cycle%2520arrest%252C%2520senescence%252C%2520and%2520apoptosis%2520in%2520human%2520osteosarcoma%26jtitle%3DOncotarget%26date%3D2016%26volume%3D7%26spage%3D45263%26epage%3D45274%26doi%3D10.18632%2Foncotarget.9481" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tsvetkov, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yeh, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, H.</span></span> <span> </span><span class="NLM_article-title">p27(Kip1) ubiquitination and degradation is regulated by the SCF(Skp2) complex through phosphorylated Thr187 in p27</span>. <i>Curr. Biol.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">661</span>– <span class="NLM_lpage">664</span>, <span class="refDoi"> DOI: 10.1016/S0960-9822(99)80290-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=10.1016%2FS0960-9822%2899%2980290-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=10375532" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=1%3ACAS%3A528%3ADyaK1MXjvFWis7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=1999&pages=661-664&author=L.+M.+Tsvetkovauthor=K.+H.+Yehauthor=S.+J.+Leeauthor=H.+Sunauthor=H.+Zhang&title=p27%28Kip1%29+ubiquitination+and+degradation+is+regulated+by+the+SCF%28Skp2%29+complex+through+phosphorylated+Thr187+in+p27&doi=10.1016%2FS0960-9822%2899%2980290-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">p27Kip1 Ubiquitination and degradation is regulated by the SCFSkp2 complex through phosphorylated Thr187 in p27</span></div><div class="casAuthors">Tsvetkov, Lyuben M.; Yeh, Kun-Huei; Lee, Soo-Jung; Sun, Hong; Zhang, Hui</div><div class="citationInfo"><span class="NLM_cas:title">Current Biology</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">661-664</span>CODEN:
                <span class="NLM_cas:coden">CUBLE2</span>;
        ISSN:<span class="NLM_cas:issn">0960-9822</span>.
    
            (<span class="NLM_cas:orgname">Current Biology Publications</span>)
        </div><div class="casAbstract">Many tumorigenic processes affect cell-cycle progression by their effects on the levels of the cyclin-dependent kinase inhibitor p27kip1.  The phosphorylation- and ubiquitination-dependent proteolysis of p27 is implicated in control of the G1-S transition in the cell cycle.  To det. the factors that control p27 stability, the authors established a cell-free ext. assay that recapitulates the degrdn. of p27.  Phosphorylation of p27 at Thr187 was essential for its degrdn.  Degrdn. was also dependent on SCFSkp2, a protein complex implicated in targeting phosphorylated proteins for ubiquitination.  Immunodepletion of components of the complex- Cul-1, Skp1, or Skp2-from the ext. abolished p27 degrdn., while addn. of purified SCFSkp2 to Skp2-depleted ext. restores the capacity to degrade p27.  A specific assocn. was obsd. between Skp2 and a p27 carboxy-terminal peptide contg. phosphorylated Thr187, but not between Skp2 and the non-phosphorylated peptide.  Skp2-dependent assocns. between Skp1 or Cul-1 and the p27 phosphopeptide were also detected.  Isolated SCFSkp2 contained an E3 ubiquitin ligase activity towards p27.  The data thus suggest that SCFSkp2 specifically targets p27 for degrdn. during cell-cycle progression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrOzAo67Ohy7bVg90H21EOLACvtfcHk0lhUNts5nZuiqw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXjvFWis7g%253D&md5=f36415d3077a9434a81cc1c074713340</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1016%2FS0960-9822%2899%2980290-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0960-9822%252899%252980290-5%26sid%3Dliteratum%253Aachs%26aulast%3DTsvetkov%26aufirst%3DL.%2BM.%26aulast%3DYeh%26aufirst%3DK.%2BH.%26aulast%3DLee%26aufirst%3DS.%2BJ.%26aulast%3DSun%26aufirst%3DH.%26aulast%3DZhang%26aufirst%3DH.%26atitle%3Dp27%2528Kip1%2529%2520ubiquitination%2520and%2520degradation%2520is%2520regulated%2520by%2520the%2520SCF%2528Skp2%2529%2520complex%2520through%2520phosphorylated%2520Thr187%2520in%2520p27%26jtitle%3DCurr.%2520Biol.%26date%3D1999%26volume%3D9%26spage%3D661%26epage%3D664%26doi%3D10.1016%2FS0960-9822%2899%2980290-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Carrano, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eytan, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hershko, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pagano, M.</span></span> <span> </span><span class="NLM_article-title">SKP2 is required for ubiquitin-mediated degradation of the CDK inhibitor p27</span>. <i>Nat. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">193</span>– <span class="NLM_lpage">199</span>, <span class="refDoi"> DOI: 10.1038/12013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=10.1038%2F12013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=10559916" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=1%3ACAS%3A528%3ADyaK1MXltlSjtr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=1999&pages=193-199&author=A.+C.+Carranoauthor=E.+Eytanauthor=A.+Hershkoauthor=M.+Pagano&title=SKP2+is+required+for+ubiquitin-mediated+degradation+of+the+CDK+inhibitor+p27&doi=10.1038%2F12013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">SKP2 is required for ubiquitin-mediated degradation of the CDK inhibitor p27</span></div><div class="casAuthors">Carrano, Andrea C.; Eytan, Esther; Hershko, Avram; Pagano, Michele</div><div class="citationInfo"><span class="NLM_cas:title">Nature Cell Biology</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">193-199</span>CODEN:
                <span class="NLM_cas:coden">NCBIFN</span>;
        ISSN:<span class="NLM_cas:issn">1465-7392</span>.
    
            (<span class="NLM_cas:orgname">Macmillan Magazines Ltd</span>)
        </div><div class="casAbstract">Degrdn. of the mammalian cyclin-dependent kinase (CDK) inhibitor p27 is required for the cellular transition from quiescence to the proliferative state.  The ubiquitination and subsequent degrdn. of p27 depend on its phosphorylation by cyclin-CDK complexes.  However, the ubiquitin-protein ligase necessary for p27 ubiquitination has not been identified.  Here we show that the F-box protein SKP2 specifically recognizes p27 in a phosphorylation-dependent manner that is characteristic of an F-box-protein-substrate interaction.  Furthermore, both in vivo and in vitro, SKP2 is a rate-limiting component of the machinery that ubiquitinates and degrades phosphorylated p27.  Thus, p27 degrdn. is subject to dual control by the accumulation of both SKP2 and cyclins following mitogenic stimulation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGobE2CKBARwdLVg90H21EOLACvtfcHk0lhga1O_9W4wIg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXltlSjtr0%253D&md5=99cb1ae155f7946684733150de62747a</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1038%2F12013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F12013%26sid%3Dliteratum%253Aachs%26aulast%3DCarrano%26aufirst%3DA.%2BC.%26aulast%3DEytan%26aufirst%3DE.%26aulast%3DHershko%26aufirst%3DA.%26aulast%3DPagano%26aufirst%3DM.%26atitle%3DSKP2%2520is%2520required%2520for%2520ubiquitin-mediated%2520degradation%2520of%2520the%2520CDK%2520inhibitor%2520p27%26jtitle%3DNat.%2520Cell%2520Biol.%26date%3D1999%26volume%3D1%26spage%3D193%26epage%3D199%26doi%3D10.1038%2F12013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Morimoto, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nishida, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Honda, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yasuda, H.</span></span> <span> </span><span class="NLM_article-title">Modification of cullin-1 by ubiquitin-like protein nedd8 enhances the activity of SCF(skp2) toward p27(kip1)</span>. <i>Biochem. Biophys. Res. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>270</i></span>,  <span class="NLM_fpage">1093</span>– <span class="NLM_lpage">1096</span>, <span class="refDoi"> DOI: 10.1006/bbrc.2000.2576</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=10.1006%2Fbbrc.2000.2576" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=10772955" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=1%3ACAS%3A528%3ADC%252BD3cXisFersr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=270&publication_year=2000&pages=1093-1096&author=M.+Morimotoauthor=T.+Nishidaauthor=R.+Hondaauthor=H.+Yasuda&title=Modification+of+cullin-1+by+ubiquitin-like+protein+nedd8+enhances+the+activity+of+SCF%28skp2%29+toward+p27%28kip1%29&doi=10.1006%2Fbbrc.2000.2576"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Modification of Cullin-1 by Ubiquitin-like Protein Nedd8 Enhances the Activity of SCFskp2 toward p27kip1</span></div><div class="casAuthors">Morimoto, Mitsuru; Nishida, Tamotsu; Honda, Reiko; Yasuda, Hideyo</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical and Biophysical Research Communications</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">270</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1093-1096</span>CODEN:
                <span class="NLM_cas:coden">BBRCA9</span>;
        ISSN:<span class="NLM_cas:issn">0006-291X</span>.
    
            (<span class="NLM_cas:orgname">Academic Press</span>)
        </div><div class="casAbstract">The periodic expression of cell cycle proteins is important for the regulation of cell cycle progression.  The amt. of CDK inhibitor, p27kip1, one such protein, seems to be regulated by the ubiquitin-proteasome system.  The ubiquitin ligase (E3) toward p27kip1 is thought to be SCFskp2.  The activity of SCFskp2 was increased by the addn. of Roc1 protein to the complex.  Furthermore, the ubiquitination of p27kip1 seemed to be dependent on the phosphorylation of T187 of p27kip1 because the mutant T187A was not ubiquitinated at all in an in vitro ubiquitination system.  Cullin-1, a component of SCF, is modified by ubiquitin-like protein Nedd8.  The modification site of cullin-1 was shown to be K696 because the K696R mutant was not modified.  When the effect of the Nedd8 modification on the SCFskp2 activity toward p27kip1 was investigated, the activity was markedly decreased by using the Nedd8-unmodified mutant cullin-1 (K696R), indicating that the modification may play an important role on the SCFskp2 activity toward p27kip1.  (c) 2000 Academic Press.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr7d8u3xfF9H7Vg90H21EOLACvtfcHk0lhga1O_9W4wIg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXisFersr0%253D&md5=03979249fae36709bb8a13370e772b50</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1006%2Fbbrc.2000.2576&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Fbbrc.2000.2576%26sid%3Dliteratum%253Aachs%26aulast%3DMorimoto%26aufirst%3DM.%26aulast%3DNishida%26aufirst%3DT.%26aulast%3DHonda%26aufirst%3DR.%26aulast%3DYasuda%26aufirst%3DH.%26atitle%3DModification%2520of%2520cullin-1%2520by%2520ubiquitin-like%2520protein%2520nedd8%2520enhances%2520the%2520activity%2520of%2520SCF%2528skp2%2529%2520toward%2520p27%2528kip1%2529%26jtitle%3DBiochem.%2520Biophys.%2520Res.%2520Commun.%26date%3D2000%26volume%3D270%26spage%3D1093%26epage%3D1096%26doi%3D10.1006%2Fbbrc.2000.2576" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yan, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Legerski, R. J.</span></span> <span> </span><span class="NLM_article-title">Artemis interacts with the Cul4A-DDB1DDB2 ubiquitin E3 ligase and regulates degradation of the CDK inhibitor p27</span>. <i>Cell Cycle</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">4098</span>– <span class="NLM_lpage">4109</span>, <span class="refDoi"> DOI: 10.4161/cc.10.23.18227</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=10.4161%2Fcc.10.23.18227" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=22134138" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsV2mt7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2011&pages=4098-4109&author=Y.+Yanauthor=X.+Zhangauthor=R.+J.+Legerski&title=Artemis+interacts+with+the+Cul4A-DDB1DDB2+ubiquitin+E3+ligase+and+regulates+degradation+of+the+CDK+inhibitor+p27&doi=10.4161%2Fcc.10.23.18227"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Artemis interacts with the Cul4A-DDB1DDB2 ubiquitin E3 ligase and regulates degradation of the CDK inhibitor p27</span></div><div class="casAuthors">Yan, Yiyi; Zhang, Xiaoshan; Legerski, Randy J.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Cycle</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">4098-4109</span>CODEN:
                <span class="NLM_cas:coden">CCEYAS</span>;
        ISSN:<span class="NLM_cas:issn">1538-4101</span>.
    
            (<span class="NLM_cas:orgname">Landes Bioscience</span>)
        </div><div class="casAbstract">Artemis, a member of the SNM1 gene family, is a multifunctional phospho-protein that has been shown to have important roles in V(D)J recombination, DNA double-strand break repair and stress-induced cell cycle checkpoint regulation.  We show here that Artemis interacts with the Cul4A-DDB1 E3 ubiquitin ligase via a direct interaction with the substrate-specificity receptor DDB2.  Furthermore, Artemis also interacts with the CDK inhibitor and tumor suppressor p27, a substrate of the Cul4A-DDB1 ligase, and both DDB2 and Artemis are required for the degrdn. of p27 mediated by this complex.  We also show that the regulation of p27 by Artemis and DDB2 is important for cell cycle progression in normally proliferating cells and in response to serum deprivation.  These findings thus define a function for Artemis as an effector of Cullin-based E3 ligase-mediated ubiquitylation, demonstrate a novel pathway for the regulation of p27 and show that Cul4A-DDB1DDB2-Artemis regulates G1-phase cell cycle progression in mammalian cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGolqogyvDm0e7Vg90H21EOLACvtfcHk0lhga1O_9W4wIg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsV2mt7Y%253D&md5=f268d7aaaff2e54d3e17b76be09c81f3</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.4161%2Fcc.10.23.18227&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4161%252Fcc.10.23.18227%26sid%3Dliteratum%253Aachs%26aulast%3DYan%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DLegerski%26aufirst%3DR.%2BJ.%26atitle%3DArtemis%2520interacts%2520with%2520the%2520Cul4A-DDB1DDB2%2520ubiquitin%2520E3%2520ligase%2520and%2520regulates%2520degradation%2520of%2520the%2520CDK%2520inhibitor%2520p27%26jtitle%3DCell%2520Cycle%26date%3D2011%26volume%3D10%26spage%3D4098%26epage%3D4109%26doi%3D10.4161%2Fcc.10.23.18227" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Abbas, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sivaprasad, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terai, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amador, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pagano, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dutta, A.</span></span> <span> </span><span class="NLM_article-title">PCNA-dependent regulation of p21 ubiquitylation and degradation via the CRL4Cdt2 ubiquitin ligase complex</span>. <i>Genes Dev.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">2496</span>– <span class="NLM_lpage">2506</span>, <span class="refDoi"> DOI: 10.1101/gad.1676108</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=10.1101%2Fgad.1676108" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2008&pages=2496-2506&author=T.+Abbasauthor=U.+Sivaprasadauthor=K.+Teraiauthor=V.+Amadorauthor=M.+Paganoauthor=A.+Dutta&title=PCNA-dependent+regulation+of+p21+ubiquitylation+and+degradation+via+the+CRL4Cdt2+ubiquitin+ligase+complex&doi=10.1101%2Fgad.1676108"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1101%2Fgad.1676108&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fgad.1676108%26sid%3Dliteratum%253Aachs%26aulast%3DAbbas%26aufirst%3DT.%26aulast%3DSivaprasad%26aufirst%3DU.%26aulast%3DTerai%26aufirst%3DK.%26aulast%3DAmador%26aufirst%3DV.%26aulast%3DPagano%26aufirst%3DM.%26aulast%3DDutta%26aufirst%3DA.%26atitle%3DPCNA-dependent%2520regulation%2520of%2520p21%2520ubiquitylation%2520and%2520degradation%2520via%2520the%2520CRL4Cdt2%2520ubiquitin%2520ligase%2520complex%26jtitle%3DGenes%2520Dev.%26date%3D2008%26volume%3D22%26spage%3D2496%26epage%3D2506%26doi%3D10.1101%2Fgad.1676108" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cullinan, S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordan, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harper, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diehl, J. A.</span></span> <span> </span><span class="NLM_article-title">The Keap1-BTB protein is an adaptor that bridges Nrf2 to a Cul3-based E3 ligase: oxidative stress sensing by a Cul3-Keap1 ligase</span>. <i>Mol. Cell. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">8477</span>– <span class="NLM_lpage">8486</span>, <span class="refDoi"> DOI: 10.1128/MCB.24.19.8477-8486.2004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=10.1128%2FMCB.24.19.8477-8486.2004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=15367669" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=1%3ACAS%3A528%3ADC%252BD2cXnvVakt7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2004&pages=8477-8486&author=S.+B.+Cullinanauthor=J.+D.+Gordanauthor=J.+Jinauthor=J.+W.+Harperauthor=J.+A.+Diehl&title=The+Keap1-BTB+protein+is+an+adaptor+that+bridges+Nrf2+to+a+Cul3-based+E3+ligase%3A+oxidative+stress+sensing+by+a+Cul3-Keap1+ligase&doi=10.1128%2FMCB.24.19.8477-8486.2004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">The Keap1-BTB protein is an adaptor that bridges Nrf2 to a Cul3-based E3 ligase: Oxidative stress sensing by a Cul3-Keap1 ligase</span></div><div class="casAuthors">Cullinan, Sara B.; Gordan, John D.; Jin, Jianping; Harper, J. Wade; Diehl, J. Alan</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">8477-8486</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">0270-7306</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">The Nrf2 transcription factor promotes survival following cellular insults that trigger oxidative damage.  Nrf2 activity is opposed by the BTB/POZ domain protein Keap1.  Keap1 is proposed to regulate Nrf2 activity strictly through its capacity to inhibit Nrf2 nuclear import.  Recent work suggests that inhibition of Nrf2 may also depend upon ubiquitin-mediated proteolysis.  To address the contribution of Keap1-dependent sequestration vs. Nrf2 proteolysis, we identified the E3 ligase that regulates Nrf2 ubiquitination.  We demonstrate that Keap1 is not solely a cytosolic anchor; rather, Keap1 is an adaptor that bridges Nrf2 to Cul3.  We demonstrate that Cul3-Keap1 complexes regulate Nrf2 polyubiquitination both in vitro and in vivo.  Inhibition of either Keap1 or Cul3 increases Nrf2 nuclear accumulation, leading to promiscuous activation of Nrf2-dependent gene expression.  Our data demonstrate that Keap1 restrains Nrf2 activity via its capacity to target Nrf2 to a cytoplasmic Cul3-based E3 ligase and suggest a model in which Keap1 coordinately regulates both Nrf2 accumulation and access to target genes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGokzxnJWH6ecbVg90H21EOLACvtfcHk0lg9YI0oFN67eA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXnvVakt7k%253D&md5=4a51087f90e5eb8f1f1cace9278941ad</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1128%2FMCB.24.19.8477-8486.2004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.24.19.8477-8486.2004%26sid%3Dliteratum%253Aachs%26aulast%3DCullinan%26aufirst%3DS.%2BB.%26aulast%3DGordan%26aufirst%3DJ.%2BD.%26aulast%3DJin%26aufirst%3DJ.%26aulast%3DHarper%26aufirst%3DJ.%2BW.%26aulast%3DDiehl%26aufirst%3DJ.%2BA.%26atitle%3DThe%2520Keap1-BTB%2520protein%2520is%2520an%2520adaptor%2520that%2520bridges%2520Nrf2%2520to%2520a%2520Cul3-based%2520E3%2520ligase%253A%2520oxidative%2520stress%2520sensing%2520by%2520a%2520Cul3-Keap1%2520ligase%26jtitle%3DMol.%2520Cell.%2520Biol.%26date%3D2004%26volume%3D24%26spage%3D8477%26epage%3D8486%26doi%3D10.1128%2FMCB.24.19.8477-8486.2004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref55"><div class="reference"><strong class="refLabel"><a href="#ref55" class="rightTabRefNumLink">55</a></strong><div class="NLM_citation" id="rightTab-cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, Y.</span></span> <span> </span><span class="NLM_article-title">Cullin-RING ligases as attractive anti-cancer targets</span>. <i>Curr. Pharm. Des.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">3215</span>– <span class="NLM_lpage">3225</span>, <span class="refDoi"> DOI: 10.2174/13816128113199990300</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=10.2174%2F13816128113199990300" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=23151137" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=1%3ACAS%3A528%3ADC%252BC3sXotFagtL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2013&pages=3215-3225&author=Y.+Zhaoauthor=Y.+Sun&title=Cullin-RING+ligases+as+attractive+anti-cancer+targets&doi=10.2174%2F13816128113199990300"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Cullin-RING ligases as attractive anti-cancer targets</span></div><div class="casAuthors">Zhao, Yongchao; Sun, Yi</div><div class="citationInfo"><span class="NLM_cas:title">Current Pharmaceutical Design</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">3215-3225</span>CODEN:
                <span class="NLM_cas:coden">CPDEFP</span>;
        ISSN:<span class="NLM_cas:issn">1381-6128</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  The ubiquitin-proteasome system (UPS) promotes the timely degrdn. of short-lived proteins with key regulatory roles in a vast array of biol. processes, such as cell cycle progression, oncogenesis and genome integrity.  Thus, abnormal regulation of UPS disrupts the protein homeostasis and causes many human diseases, particularly cancer.  Indeed, the FDA approval of bortezomib, the first class of general proteasome inhibitor, for the treatment of multiple myeloma, demonstrated that the UPS can be an attractive anti-cancer target.  However, normal cell toxicity assocd. with bortezomib, resulting from global inhibition of protein degrdn., promotes the focus of drug discovery efforts on targeting enzymes upstream of the proteasome for better specificity.  E3 ubiquitin ligases, particularly those known to be activated in human cancer, become an attractive choice.  Cullin-RING Ligases (CRLs) with multiple components are the largest family of E3 ubiquitin ligases and are responsible for ubiquitination of ∼20% of cellular proteins degraded through UPS.  Activity of CRLs is dynamically regulated and requires the RING component and cullin neddylation.  In this review, we will introduce the UPS and CRL E3s and discuss the biol. processes regulated by each of eight CRLs through substrate degrdn.  We will further discuss how cullin neddylation controls CRL activity, and how CRLs are being validated as the attractive cancer targets by abrogating the RING component through genetic means and by inhibiting cullin neddylation via MLN4924, a small mol. indirect inhibitor of CRLs, currently in several Phase I clin. trials.  Finally, we will discuss current efforts and future perspectives on the development of addnl. inhibitors of CRLs by targeting E2 and/or E3 of cullin neddylation and CRL-mediated ubiquitination as potential anti-cancer agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp-K-w4MaFbdLVg90H21EOLACvtfcHk0lg9YI0oFN67eA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXotFagtL8%253D&md5=4498de520df1c0a24796bdbfd5b3ce17</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.2174%2F13816128113199990300&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F13816128113199990300%26sid%3Dliteratum%253Aachs%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DSun%26aufirst%3DY.%26atitle%3DCullin-RING%2520ligases%2520as%2520attractive%2520anti-cancer%2520targets%26jtitle%3DCurr.%2520Pharm.%2520Des.%26date%3D2013%26volume%3D19%26spage%3D3215%26epage%3D3225%26doi%3D10.2174%2F13816128113199990300" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref56"><div class="reference"><strong class="refLabel"><a href="#ref56" class="rightTabRefNumLink">56</a></strong><div class="NLM_citation" id="rightTab-cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Niture, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaspar, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jaiswal, A. K.</span></span> <span> </span><span class="NLM_article-title">Nrf2 signaling and cell survival</span>. <i>Toxicol. Appl. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>244</i></span>,  <span class="NLM_fpage">37</span>– <span class="NLM_lpage">42</span>, <span class="refDoi"> DOI: 10.1016/j.taap.2009.06.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=10.1016%2Fj.taap.2009.06.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=19538984" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=1%3ACAS%3A528%3ADC%252BC3cXjt1Gkt7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=244&publication_year=2010&pages=37-42&author=S.+K.+Nitureauthor=J.+W.+Kasparauthor=J.+Shenauthor=A.+K.+Jaiswal&title=Nrf2+signaling+and+cell+survival&doi=10.1016%2Fj.taap.2009.06.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">Nrf2 signaling and cell survival</span></div><div class="casAuthors">Niture, Suryakant K.; Kaspar, James W.; Shen, Jun; Jaiswal, Anil K.</div><div class="citationInfo"><span class="NLM_cas:title">Toxicology and Applied Pharmacology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">244</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">37-42</span>CODEN:
                <span class="NLM_cas:coden">TXAPA9</span>;
        ISSN:<span class="NLM_cas:issn">0041-008X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Nrf2:INrf2 (Keap1) acts as a sensor for oxidative/electrophilic stress.  INrf2 serves as an adaptor to link Nrf2 to the ubiquitin ligase Cul3-Rbx1 complex that ubiquitinate and degrade Nrf2.  Under basal conditions, cytosolic INrf2/Cul3-Rbx1 is constantly degrading Nrf2.  When a cell encounters stress, Nrf2 dissocs. from INrf2 and translocates into the nucleus.  Oxidative/electrophilic stress induced modification of INrf2-Cys-151 and/or protein kinase C (PKC)-mediated phosphorylation of Nrf2-Ser-40 controls Nrf2 release from INrf2 followed by stabilization and nuclear translocation of Nrf2.  Nrf2 binds to the antioxidant response element (ARE) and activates a myriad of genes that protect cells against oxidative/electrophilic stress and neoplasia.  A delayed response of oxidative/electrophilic stress activates protein kinase GSK-3β that phosphorylates Fyn kinase at unknown Thr residue(s).  Phosphorylated Fyn translocates to the nucleus and phosphorylates Nrf2-Tyr-568 that leads to nuclear export and degrdn. of Nrf2.  Prothymosin-α mediated nuclear translocation of INrf2 also degrades nuclear Nrf2.  The degrdn. of Nrf2 both in cytosol and nuclear compartments rapidly brings down its levels to normal resulting in suppression of Nrf2 downstream gene expression.  An auto-regulatory loop between Nrf2 and INrf2 controls their cellular abundance.  Nrf2 regulates INrf2 by controlling its transcription, and INrf2 controls Nrf2 by degrading it.  In conclusion, switching on and off of Nrf2 combined with promoting an auto-regulatory loop between them regulates activation/deactivation of defensive genes leading to protection of cells against adverse effects of oxidative and electrophilic stress and promote cell survival.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoIcQsUwXM-PLVg90H21EOLACvtfcHk0liP9jppbpBPiQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXjt1Gkt7Y%253D&md5=c25e3a28939528f3bff36a27be805695</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1016%2Fj.taap.2009.06.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.taap.2009.06.009%26sid%3Dliteratum%253Aachs%26aulast%3DNiture%26aufirst%3DS.%2BK.%26aulast%3DKaspar%26aufirst%3DJ.%2BW.%26aulast%3DShen%26aufirst%3DJ.%26aulast%3DJaiswal%26aufirst%3DA.%2BK.%26atitle%3DNrf2%2520signaling%2520and%2520cell%2520survival%26jtitle%3DToxicol.%2520Appl.%2520Pharmacol.%26date%3D2010%26volume%3D244%26spage%3D37%26epage%3D42%26doi%3D10.1016%2Fj.taap.2009.06.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref57"><div class="reference"><strong class="refLabel"><a href="#ref57" class="rightTabRefNumLink">57</a></strong><div class="NLM_citation" id="rightTab-cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, L.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, Y.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, D.-D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miao, E.-F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qiao, X.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, L.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H.-M.</span></span> <span> </span><span class="NLM_article-title">Design, synthesis and biological evaluation of [1,2,4]triazolo[1,5-<i>a</i>]pyrimidines as potent lysine specific demethylase 1 (LSD1/KDM1A) inhibitors</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>125</i></span>,  <span class="NLM_fpage">940</span>– <span class="NLM_lpage">951</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2016.10.021</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=10.1016%2Fj.ejmech.2016.10.021" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=27769034" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;key=1%3ACAS%3A528%3ADC%252BC28XhslWju7rJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=125&publication_year=2017&pages=940-951&author=S.+Wangauthor=L.-J.+Zhaoauthor=Y.-C.+Zhengauthor=D.-D.+Shenauthor=E.-F.+Miaoauthor=X.-P.+Qiaoauthor=L.-J.+Zhaoauthor=Y.+Liuauthor=R.+Huangauthor=B.+Yuauthor=H.-M.+Liu&title=Design%2C+synthesis+and+biological+evaluation+of+%5B1%2C2%2C4%5Dtriazolo%5B1%2C5-a%5Dpyrimidines+as+potent+lysine+specific+demethylase+1+%28LSD1%2FKDM1A%29+inhibitors&doi=10.1016%2Fj.ejmech.2016.10.021"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">Design, synthesis and biological evaluation of [1,2,4]triazolo[1,5-a]pyrimidines as potent lysine specific demethylase 1 (LSD1/KDM1A) inhibitors</span></div><div class="casAuthors">Wang, Shuai; Zhao, Li-Jie; Zheng, Yi-Chao; Shen, Dan-Dan; Miao, Er-Fei; Qiao, Xue-Peng; Zhao, Li-Juan; Liu, Ying; Huang, Ruilei; Yu, Bin; Liu, Hong-Min</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">125</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">940-951</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">A new series of [1,2,4]triazolo[1,5-a]pyrimidine-based LSD1 inhibitors was designed, synthesized, and further evaluated for cytotoxicity against MGC-803, EC109, A549 and PC-9 cells as well as the ability of inhibiting LSD1.  Some of these compds. showed potent inhibition toward LSD1 and selectively inhibited growth of A549 and PC-9 cells.  Compd. I potently inhibited growth of PC-9 cells (IC50 = 0.59 μM), about 4-fold more potent than 5-FU.  Further SAR studies led to the identification of compds. I and II, which had good growth inhibition against all the tested cancer cell lines and were much more potent than 5-FU and GSK2879552.  Besides, compds. III-V inhibited LSD1 potently (IC50 = 0.154, 1.19 and 0.557 μM, resp.).  Docking studies revealed that compd. III formed arene-H interactions with Val333 and hydrogen bonds with surrounding Ala331, Met332, and Ala539 residues.  Compd. III significantly inhibited migration of A549 and PC-9 cells in a concn.-dependent manner, but had different effect on the expression of E-cadherin and N-cadherin.  The [1,2,4]triazolo[1,5-a]pyrimidine scaffold may serve as a starting point for developing potent LSD1 inhibitors for cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr0jqW-eLBQObVg90H21EOLACvtfcHk0liP9jppbpBPiQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhslWju7rJ&md5=d022678613486890faacfef9c163492b</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2016.10.021&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2016.10.021%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DS.%26aulast%3DZhao%26aufirst%3DL.-J.%26aulast%3DZheng%26aufirst%3DY.-C.%26aulast%3DShen%26aufirst%3DD.-D.%26aulast%3DMiao%26aufirst%3DE.-F.%26aulast%3DQiao%26aufirst%3DX.-P.%26aulast%3DZhao%26aufirst%3DL.-J.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DHuang%26aufirst%3DR.%26aulast%3DYu%26aufirst%3DB.%26aulast%3DLiu%26aufirst%3DH.-M.%26atitle%3DDesign%252C%2520synthesis%2520and%2520biological%2520evaluation%2520of%2520%255B1%252C2%252C4%255Dtriazolo%255B1%252C5-a%255Dpyrimidines%2520as%2520potent%2520lysine%2520specific%2520demethylase%25201%2520%2528LSD1%252FKDM1A%2529%2520inhibitors%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2017%26volume%3D125%26spage%3D940%26epage%3D951%26doi%3D10.1016%2Fj.ejmech.2016.10.021" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5UFI" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5UFI','PDB','5UFI'); return false;">PDB: 5UFI</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i153"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b00113">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_85358"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a class="ext-link" href="/">ACS Publications website</a> at DOI: <a class="ext-link" href="/doi/abs/10.1021/acs.jmedchem.9b00113">10.1021/acs.jmedchem.9b00113</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Coordinates of modeled structures in PDB format: <b>E1</b> in complex with DCN1; <b>DI-591</b> in complex with DCN1 after molecular dynamics simulations (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5UFI">5UFI</a>); <b>WS-383</b> in complex with DCN1; <b>WS-383</b> in complex with DCN1 after molecular dynamics simulations; <b>WS-524</b> in complex with DCN1; <b>WS-524</b> in complex with DCN1 after molecular dynamics simulations (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b00113/suppl_file/jm9b00113_si_001.zip">ZIP</a>)</p></li><li><p class="inline">Molecular formula strings and some data (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b00113/suppl_file/jm9b00113_si_002.csv">CSV</a>)</p></li><li><p class="inline">Assay development results, binding mode, and NMR, HRMS, and HPLC spectra of representative compounds (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b00113/suppl_file/jm9b00113_si_003.pdf">PDF</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00113/suppl_file/jm9b00113_si_001.zip">jm9b00113_si_001.zip (464.83 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00113/suppl_file/jm9b00113_si_002.csv">jm9b00113_si_002.csv (53.56 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00113/suppl_file/jm9b00113_si_003.pdf">jm9b00113_si_003.pdf (30.38 MB)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.9b00113&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.9b00113%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2019.62.issue-5%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.9b00113" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                12MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"677adabb5f362521","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
