
<html lang="en"     class="pb-page"  data-request-id="3e67391d-387e-4fc1-a43f-a2bc33d76c85"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;issue:issue:10.1021/jmcmar.2008.51.issue-5;article:article:10.1021/jm070654j;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="dc.Title" content="Discovery of Kinase Inhibitors by High-Throughput Docking and Scoring Based on a Transferable Linear Interaction Energy Model" /></meta><meta name="dc.Creator" content="Peter  Kolb" /></meta><meta name="dc.Creator" content="Danzhi  Huang" /></meta><meta name="dc.Creator" content="Fabian  Dey" /></meta><meta name="dc.Creator" content="Amedeo  Caflisch" /></meta><meta name="dc.Description" content="The linear interaction energy method with continuum electrostatics (LIECE) is evaluated in depth on five kinases. The two multiplicative coefficients for the van der Waals energy and electrostatic ..." /></meta><meta name="Description" content="The linear interaction energy method with continuum electrostatics (LIECE) is evaluated in depth on five kinases. The two multiplicative coefficients for the van der Waals energy and electrostatic ..." /></meta><meta name="dc.Publisher" content=" American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="February 14, 2008" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/jm070654j" /></meta><meta name="dc.Language" content="EN" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2008 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Fjm070654j" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Fjm070654j" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Fjm070654j" /></link>
        
    
    

<title>Discovery of Kinase Inhibitors by High-Throughput Docking and Scoring Based on a Transferable Linear Interaction Energy Model | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/jm070654j" /></meta><meta property="og:title" content="Discovery of Kinase Inhibitors by High-Throughput Docking and Scoring Based on a Transferable Linear Interaction Energy Model" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2008/jmcmar.2008.51.issue-5/jm070654j/production/images/large/jm-2007-00654j_0007.jpeg" /></meta><meta property="og:description" content=" The linear interaction energy method with continuum electrostatics (LIECE) is evaluated in depth on five kinases. The two multiplicative coefficients for the van der Waals energy and electrostatic free energy are shown to be transferable among different kinases. Moreover, good enrichment factors are obtained for a library of 40375 diverse compounds seeded with 73 known inhibitors of CDK2. Therefore, a general two-parameter LIECE model for kinases is derived by combining large data sets of inhibitors of CDK2, Lck, and p38. This two-parameter model is cross-validated on two kinases not used for fitting; it shows an average error of about 1.5 kcal/mol for the prediction of absolute binding affinity of 37 and 128 known inhibitors of EphB4 and EGFR, respectively. High-throughput docking and ranking by two-parameter LIECE models are shown to be able to identify novel low-micromolar EphB4 and CDK2 inhibitors of low-molecular weight (≤355 g/mol)." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/jm070654j"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/jm070654j">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-14/achre4.2021.54.issue-14/20210720/achre4.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-29/aamick.2021.13.issue-29/20210728/aamick.2021.13.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-14/acncdm.2021.12.issue-14/20210721/acncdm.2021.12.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-7/amlcef.2021.3.issue-7/20210705/amlcef.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-29/acsodf.2021.6.issue-29/20210727/acsodf.2021.6.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-29/ascecg.2021.9.issue-29/20210726/ascecg.2021.9.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-29/ancham.2021.93.issue-29/20210727/ancham.2021.93.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-29/bichaw.2021.60.issue-29/20210727/bichaw.2021.60.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-27/cgeabj.2021.99.issue-27/20210726-01/cgeabj.2021.99.issue-27.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-7/cgdefu.2021.21.issue-7/20210707/cgdefu.2021.21.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-14/esthag.2021.55.issue-14/20210720/esthag.2021.55.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-29/iecred.2021.60.issue-29/20210728/iecred.2021.60.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-14/inocaj.2021.60.issue-14/20210719/inocaj.2021.60.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-29/jafcau.2021.69.issue-29/20210728/jafcau.2021.69.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-29/jacsat.2021.143.issue-29/20210728/jacsat.2021.143.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-7/jamsef.2021.32.issue-7/20210707/jamsef.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-29/langd5.2021.37.issue-29/20210727/langd5.2021.37.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-7/mpohbp.2021.18.issue-7/20210705/mpohbp.2021.18.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Fjm070654j&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Fjm070654j&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/jm070654j&amp;href=/doi/10.1021/jm070654j" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2008</span><span class="cit-fg-volume">, 51</span><span class="cit-fg-issue">, 5</span><span class="cit-fg-pageRange">, 1179-1188</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/51/5" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/jm070543k" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/jm070723u" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Discovery of Kinase Inhibitors by High-Throughput Docking and Scoring Based on a Transferable Linear Interaction Energy Model</span></h1><ul class="loa non-jats-loa"><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Peter++Kolb">Peter Kolb</a></span><span class="author-xref-symbol "><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Danzhi++Huang">Danzhi Huang</a></span><span class="author-xref-symbol "><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Fabian++Dey">Fabian Dey</a></span><span class="author-xref-symbol "><sup>†</sup></span></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Amedeo++Caflisch">Amedeo Caflisch</a></span><span class="author-xref-symbol "><sup>*</sup></span></span></li><span class="comma-separator"></span></ul><div class="affiliations"><div class="view-affs"><span class="title">View Author Information</span><i class="caret-icon icon-angle-down"></i></div><div class="all-aff-infos hlFld-Affiliation"><div class="aff-info"><span class="aff-text">Department of Biochemistry, University of Zürich, Winterthurerstrasse 190, CH-8057 Zürich, Switzerland</span></div><div class="corresp-info"><strong>*</strong> To whom correspondence should be addressed. Phone: (+41 44) 635 55 21 . Fax: (+41 44) 635 68 62. E-mail: <a href="/cdn-cgi/l/email-protection#6b080a0d07021808032b09020408451e1103450803"><span class="__cf_email__" data-cfemail="0f6c6e6963667c6c674f6d66606c217a7567216c67">[email protected]</span></a></div><div class="corresp-fn" id="fn1">†These authors contributed equally to this work.</div></div></div><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Fjm070654j&amp;href=/doi/10.1021%2Fjm070654j" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2008</span></span><span class="cit-volume">, 51</span><span class="cit-issue">, 5</span><span class="cit-pageRange">, 1179–1188</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">February 14, 2008</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>7 June 2007</li><li><span class="item_label"><b>Published</b> online</span>14 February 2008</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 13 March 2008</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/jm070654j" title="DOI URL">https://doi.org/10.1021/jm070654j</a></div><div class="article_header-article-copyright"><strong>Copyright © 2008 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D1179%26pageCount%3D10%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DPeter%2BKolb%252C%2BDanzhi%2BHuang%252C%2BFabian%2BDey%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D51%26issueNum%3D5%26contentID%3Djm070654j%26title%3DDiscovery%2Bof%2BKinase%2BInhibitors%2Bby%2BHigh-Throughput%2BDocking%2Band%2BScoring%2BBased%2Bon%2Ba%2BTransferable%2BLinear%2BInteraction%2BEnergy%2BModel%26numPages%3D10%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D1188%26publicationDate%3DMarch%2B2008">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/jm070654j"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">1207</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">46</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/jm070654j" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Discovery of Kinase Inhibitors by High-Throughput Docking and Scoring Based on a Transferable Linear Interaction Energy Model&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Peter&quot;,&quot;last_name&quot;:&quot;Kolb&quot;},{&quot;first_name&quot;:&quot;Danzhi&quot;,&quot;last_name&quot;:&quot;Huang&quot;},{&quot;first_name&quot;:&quot;Fabian&quot;,&quot;last_name&quot;:&quot;Dey&quot;},{&quot;first_name&quot;:&quot;Amedeo&quot;,&quot;last_name&quot;:&quot;Caflisch&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2008&quot;,&quot;month&quot;:&quot;02&quot;,&quot;day&quot;:&quot;14&quot;,&quot;issue&quot;:&quot;5&quot;,&quot;volume&quot;:&quot;51&quot;,&quot;pages&quot;:&quot;1179-1188&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/jm070654j&quot;},&quot;abstract&quot;:&quot;The linear interaction energy method with continuum electrostatics (LIECE) is evaluated in depth on five kinases. The two multiplicative coefficients for the van der Waals energy and electrostatic free energy are shown to be transferable among different kinases. Moreover, good enrichment factors are obtained for a library of 40375 diverse compounds seeded with 73 known inhibitors of CDK2. Therefore, a general two-parameter LIECE model for kinases is derived by combining large data sets of inhibitors of CDK2, Lck, and p38. This two-parameter model is cross-validated on two kinases not used for fitting; it shows an average error of about 1.5 kcal/mol for the prediction of absolute binding affinity of 37 and 128 known inhibitors of EphB4 and EGFR, respectively. High-throughput docking and ranking by two-parameter LIECE models are shown to be able to identify novel low-micromolar EphB4 and CDK2 inhibitors of low-molecular weight (≤355 g/mol).&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Fjm070654j&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=jm070654j" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Fjm070654j&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=jm070654j" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Fjm070654j&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=jm070654j" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Fjm070654j&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=jm070654j&amp;href=/doi/10.1021/jm070654j" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/jm070654j" /></input><a href="/doi/pdf/10.1021/jm070654j" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (730 KB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/jm070654j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Fjm070654j%26sid%3Dliteratum%253Aachs%26pmid%3D18271520%26genre%3Darticle%26aulast%3DKolb%26date%3D2008%26atitle%3DDiscovery%2Bof%2BKinase%2BInhibitors%2Bby%2BHigh-Throughput%2BDocking%2Band%2BScoring%2BBased%2Bon%2Ba%2BTransferable%2BLinear%2BInteraction%2BEnergy%2BModel%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D51%26issue%3D5%26spage%3D1179%26epage%3D1188%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (1)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291202" title="Electrostatics">Electrostatics</a>,</li><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=291732" title="Interaction energies">Interaction energies</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=291352" title="Screening assays">Screening assays</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/51/5" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="f_null" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2008/jmcmar.2008.51.issue-5/jm070654j/production/images/medium/jm-2007-00654j_0007.gif" alt="" id="_i2" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2008/jmcmar.2008.51.issue-5/jm070654j/production/images/large/jm-2007-00654j_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm070654j&amp;id=f_null"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">The linear interaction energy method with continuum electrostatics (LIECE) is evaluated in depth on five kinases. The two multiplicative coefficients for the van der Waals energy and electrostatic free energy are shown to be transferable among different kinases. Moreover, good enrichment factors are obtained for a library of 40375 diverse compounds seeded with 73 known inhibitors of CDK2. Therefore, a general two-parameter LIECE model for kinases is derived by combining large data sets of inhibitors of CDK2, Lck, and p38. This two-parameter model is cross-validated on two kinases not used for fitting; it shows an average error of about 1.5 kcal/mol for the prediction of absolute binding affinity of 37 and 128 known inhibitors of EphB4 and EGFR, respectively. High-throughput docking and ranking by two-parameter LIECE models are shown to be able to identify novel low-micromolar EphB4 and CDK2 inhibitors of low-molecular weight (≤355 g/mol).</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">1 Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_84302" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_84302" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Accurate and efficient approaches for the evaluation of binding affinities are required for in silico screening of large libraries of compounds by high-throughput docking.<a onclick="showRef(event, 'ref1 ref2 ref3 ref4 ref5 ref6 ref7 ref8 ref9 ref10'); return false;" href="javascript:void(0);" class="ref ref1 ref2 ref3 ref4 ref5 ref6 ref7 ref8 ref9 ref10">(1-10)</a> Rigorous methods based on free energy perturbation molecular dynamics simulations have recently been developed to improve efficiency by enhancing convergence. However, these methods still require about 10–20 days of computer time per compound.<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a></div><div class="NLM_p">The LIE (linear interaction energy) method was proposed to calculate free energies of binding by averaging interaction energies from molecular dynamics simulations of the ligand and the ligand/protein complex.<a onclick="showRef(event, 'ref12 ref13'); return false;" href="javascript:void(0);" class="ref ref12 ref13">(12, 13)</a> In LIE, the free energy of binding is approximated by <span class="NLM_disp-formula" id="eqA"><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2008/jmcmar.2008.51.issue-5/jm070654j/production/images/jm-2007-00654j_m001.gif" alt="" /></img></span> where <i>E</i><sup>vdw</sup> and <i>E</i><sup>elec</sup> are the van der Waals and electrostatic interaction energies between the ligand and its environment. The environment is either the solvent (free) or the solvated ligand/protein complex (bound). The ⟨⟩ denotes an ensemble average sampled over a molecular dynamics<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> or Monte Carlo<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> trajectory. The coefficient α is determined empirically.<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> Originally, α was fixed to a value of <sup>1</sup>/<sub>2</sub>, as predicted by the linear response approximation.<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> Later studies have shown, however, that improved models for a large variety of systems could be obtained by considering β as a free parameter.<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> Consequently, both coefficients are obtained by a fit of experimentally determined values of Δ<i>G</i> to the calculated values of <i>E</i><sup>elec</sup> and <i>E</i><sup>vdw</sup> for a training set of known ligands.</div><div class="NLM_p">The LIE method and modifications thereof have been applied to a large number of existing inhibitor/protein data sets.<a onclick="showRef(event, 'ref12 ref16 ref17 ref18 ref19 ref20 ref21 ref22'); return false;" href="javascript:void(0);" class="ref ref12 ref16 ref17 ref18 ref19 ref20 ref21 ref22">(12, 16-22)</a> Moreover, LIE-based scorings of ligands were shown to perform better than established scoring functions.<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> Interestingly, recent applications to pharmaceutically relevant enzyme targets have documented the predictive ability and usefulness in lead-discovery projects. As an example, the LIE method with explicit water molecular dynamics sampling was successfully used in the design of a series of inhibitors of the malarial aspartic proteases plasmepsin I and II.<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> Unfortunately, LIE cannot be used for high-throughput docking because of its computational requirements (the currently fastest implementation needs about 6 h for each compound<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a>). Therefore, we have replaced the explicit water molecular dynamics (or Monte Carlo) sampling with a simple energy minimization and combined the LIE method with a rigorous treatment of solvation within the continuum electrostatics approximation,<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> i.e., the numerical solution of the Poisson equation by the finite-difference technique.<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> The LIECE<a class="ref internalNav" href="#fn2" aria-label="a">a</a><div class="NLM_fn" id="fn2">a<p class="last">Abbreviations: LIECE, linear interaction energy with continuum electrostatics; CDK2, cyclin-dependent kinase 2; Lck, lymphocyte-specific kinase; p38, p38 mitogen-activated protein kinase; EGFR, epidermal growth factor receptor; EphB4, erythropoietin producing human hepatocellular carcinoma receptor B4.</p></div> approach, where the last two letters stand for continuum electrostatics, is about 2 orders of magnitude faster than previous LIE methods and shows a similar precision on the targets tested. In fact, an error of about 1 kcal/mol was observed for 13 and 29 peptidic inhibitors of β-secretase and HIV-1 protease, respectively.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> Most importantly, the LIECE approach has played a key role in the recent discoveries of three novel series of β-secretase inhibitors (phenylurea derivatives,<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> triazine derivatives,<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> and a set of five cell-permeable, nonpeptidic, low-micromolar inhibitors with a different scaffold (D. Huang and A. Caflisch, unpublished results)).</div><div class="NLM_p last">Here, we further validate the predictive ability of LIECE by a critical assessment of five different protein kinases and 330 inhibitors. Protein kinases are important pharmacological targets,<a onclick="showRef(event, 'ref26 ref27'); return false;" href="javascript:void(0);" class="ref ref26 ref27">(26, 27)</a> and several three-dimensional structures with inhibitors and binding affinity data are available.<a onclick="showRef(event, 'ref28 ref29'); return false;" href="javascript:void(0);" class="ref ref28 ref29">(28, 29)</a> Recently, a generalization of LIE to additional energy terms and parameters (i.e., the extended linear response approach) has been applied to three protein kinases;<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> the results and predictive ability are used as a basis of comparison here. The present work was motivated by three main questions: Are the parameters of the LIECE approach transferable between enzymes of the same class? Which physicochemical properties of the binding site are responsible for the parameter transferability? Is it possible to combine experimental data from different kinases to derive a predictive LIECE model with a minimal number of parameters? Answers to these questions help to determine the usefulness and range of applicability of the LIECE method in structure-based ligand design.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i4">2 Methods</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_46325" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_46325" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5"> Selection of the Kinases</h3><div class="NLM_p last">Three tyrosine kinases, lymphocyte-specific kinase (Lck), erythropoietin producing human hepatocellular carcinoma receptor B4 (EphB4), and epidermal growth factor receptor (EGFR), as well as two Ser-Thr protein kinases, cyclin-dependent kinase 2 (CDK2) and p38 mitogen-activated protein kinase (p38), were selected for the present study. CDK2, Lck, and p38 (with their inhibitors, see below) were chosen as training set to directly compare with the extended linear response approach published recently.<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> Cross-validation was performed on 37 EphB4 inhibitors (in-house data, unpublished) and 128 EGFR inhibitors;<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> these two tyrosine kinases and their inhibitors had not been employed for fitting.</div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i6" class="anchor-spacer"></div><h3 class="article-section__title" id="_i6"> Selection of the Inhibitors for the Training Set</h3><div class="NLM_p">For CDK2, 50 of the 94 inhibitors published by Bramson et al.<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> were used. Tominaga and Jorgensen<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> had used a subset of 57 compounds, which was further reduced in our study by excluding the seven inhibitors for which only lower bounds of the IC<sub>50</sub> were available (compounds <b>C14</b>, <b>C34</b>, <b>C38</b>–<b>40</b>, <b>C48</b>, and <b>C49</b> in ref <a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">21</a>). To extend the range of IC<sub>50</sub> values, this set was supplemented with the 23 inhibitors and IC<sub>50</sub> values described in Gibson et al.<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> Hence, the training set for CDK2 comprised a total of 73 known inhibitors with two different scaffolds. In the case of Lck, the 51 inhibitors used in ref <a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">21</a>, which had previously been reported by Chen et al.,<a onclick="showRef(event, 'ref33 ref34'); return false;" href="javascript:void(0);" class="ref ref33 ref34">(33, 34)</a> were selected. Finally, 41 inhibitors of p38 identified by Stelmach et al.<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> were used (compounds <b>9a</b>, <b>9b</b>, <b>9j</b>, and <b>9s</b> in ref <a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">35</a> were discarded because only a lower bound for the IC<sub>50</sub> had been published). In contrast to the inhibitors used in ref <a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">21</a>, only inhibitors binding to the ATP binding site and forming hydrogen bonds with the hinge region were used. On the other hand, the three ligand sets differ significantly in molecular weight, number of rotatable bonds, and ClogP (Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a>), and despite the similar binding sites and modes, it is not obvious a priori that all LIECE models can be combined into one.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2008/jmcmar.2008.51.issue-5/jm070654j/production/images/medium/jm-2007-00654j_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2008/jmcmar.2008.51.issue-5/jm070654j/production/images/large/jm-2007-00654j_0001.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Distributions of some key properties of the five sets of inhibitors. All values have been calculated with DAIM.<a onclick="showRef(event, 'ref52'); return false;" href="javascript:void(0);" class="ref ref52">(52)</a></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2008/jmcmar.2008.51.issue-5/jm070654j/production/images/large/jm-2007-00654j_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm070654j&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i7" class="anchor-spacer"></div><h3 class="article-section__title" id="_i7"> Selection of the Inhibitors for the Test Set</h3><div class="NLM_p last">For EphB4, 37 inhibitors originating from an imidazo[1,2-<i>a</i>]pyrazine lead identified in a yeast-based high-throughput screen<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> were selected. The inhibitory activity of all compounds had been verified in two enzymatic assays: a fluorescence-based assay (Panvera Z’Lyte Tyr2 kinase assay PV3191, Invitrogen) and a radioactivity-based assay (KinaseProfiler Assay Protocols, Upstate Ltd., Dundee, U.K.). The EGFR test set comprised 128 known inhibitors published by Aparna et al.<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a></div></div><div id="sec2_3a" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8"> Preparation of the Inhibitor and Protein Structures</h3><div class="NLM_p">The three sets of inhibitors used to derive the LIECE models were generated by manual modification of the scaffold present in the respective X-ray structures (PDB codes <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1KE5">1KE5</a> for CDK2, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1QPE">1QPE</a> for Lck, and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1M7Q">1M7Q</a> for p38). Newly added bonds were minimized in the absence of the protein structure. This and all subsequent minimizations were carried out with CHARMM,<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> using the CHARMm22 force field (Accelrys Inc.<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a>). Hydrogen atoms were added to all protein structures according to the protonation states of side chains and termini at pH 7. Partial charges were then assigned to both the inhibitor and protein structures using the MPEOE method.<a onclick="showRef(event, 'ref39 ref40'); return false;" href="javascript:void(0);" class="ref ref39 ref40">(39, 40)</a> Protein hydrogen atoms were minimized in the absence of the inhibitor.</div><div class="NLM_p last">The 3D structures of the inhibitors of the two test sets, EphB4 and EGFR, were obtained from Dr. S. Audétat (Oncalis AG) and Dr. V. Aparna, respectively, and were visually checked for accuracy. For the calculations with EphB4 a modeled protein structure was used; the building of this model is described in the next section. The calculations for EGFR were carried out on the X-ray structure of the complex with erlotinib (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1M17">1M17</a>). The 37 inhibitors of EphB4 and the 128 inhibitors of EGFR were manually docked into the binding site such that at least one hydrogen bond with the hinge region was formed. Each pose was minimized within the rigid protein using CHARMM. For the minimizations of the 128 compounds in EGFR, the water molecule WAT10 was retained. This water molecule is supposed to mediate a hydrogen bond between N(3) of the pyrimidine ring of the inhibitor scaffold and Thr766. It is described as essential for the successful reproduction of the crystallographic binding mode,<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> which is supported by the fact that its inclusion in the calculations decreased the average error for the prediction of this set of inhibitors substantially.</div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i9" class="anchor-spacer"></div><h3 class="article-section__title" id="_i9"> Preparation of EphB4</h3><div class="NLM_p">Since the structure of the kinase domain of EphB4 is not publicly available, a homology model was built using the structure of EphB2 (mouse, PDB entry <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1JPA">1JPA</a>) as template. The overall amino acid sequence identity between the human EphB4 sequence obtained from SWISS-PROT (accession code P54760) and the sequence derived from the mouse EphB2 structure is 88.4%, and there are no gaps or insertions in the aligned region. An additionally generated binary sequence alignment with the sequence derived from the human Eph kinase structure EphA2 (PDB entry <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1MQB">1MQB</a>) revealed a lower sequence identity of 63.1% and also three short regions containing gaps or insertions. Therefore, only the sequence alignment between EphB4 (human) and EphB2 (mouse) was used in the initial phase of the homology modeling procedure. All sequence alignments were performed using the program ClustalW.<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> The <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1JPA">1JPA</a> crystal structure comprises two chemically identical subunits in the crystallographic asymmetric unit. A structural superpostion of the two subunits results in an average root-mean-square deviation of 0.32 Å for 269 C<sub>α</sub> atoms. Since subunit A of the <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1JPA">1JPA</a> crystal structure has better main-chain dihedral angles (φ and ψ) and lower <i>B</i> values, this subunit was initially chosen as the template structure. However, the two subunits of the <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1JPA">1JPA</a> structure show some clear structural differences around the active site region. The differing amino acid residues were analyzed with respect to atomic <i>B</i> values, possible contacts, and stereochemical criteria. Subsequently, some side chain rotamer conformations of the initial template structure (subunit A) were replaced by the corresponding side chain conformations of subunit B of the <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1JPA">1JPA</a> crystal structure. The resulting modified structure was then used as template for homology modeling, and a total of 100 different models were generated using the program Modeller.<a onclick="showRef(event, 'ref42 ref43'); return false;" href="javascript:void(0);" class="ref ref42 ref43">(42, 43)</a></div><div class="NLM_p last">The obtained initial models were ranked and analyzed on the basis of energetical and stereochemical criteria. The best model was manually improved by comparison with the active sites of the two known Eph kinase structures (PDB entry codes <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1JPA">1JPA</a> and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1MQB">1MQB</a>) and other related kinase structures (PDB entry codes <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1BYG">1BYG</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1FMK">1FMK</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1FPU">1FPU</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1IEP">1IEP</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1M14">1M14</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1M17">1M17</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1M52">1M52</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1MP8">1MP8</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1OPK">1OPK</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1OPL">1OPL</a>, and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2SRC">2SRC</a>) found by PSI-BLAST<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> and DALI<a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a> searches of protein structure databases. The conformational information contained in these homologous structures was used for manually adjusting side chain conformations of conserved or similar amino acid residues. For some of the mutated residues, statistically preferred χ-angles were chosen and favorable hydrophobic or polar contacts were also considered. Additionally, an ATP molecule was modeled into the active site to avoid structural changes during minimization. Hydrogen atoms (considering appropriate ionization states for acidic and basic amino acid residues), CHARMm22<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> atom types, and partial charges were assigned to the protein and ATP using the program WITNOTP and the MPEOE method.<a onclick="showRef(event, 'ref39 ref40'); return false;" href="javascript:void(0);" class="ref ref39 ref40">(39, 40)</a> Manual rebuilding and refinement with CHARMM<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> using the CHARMm22<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> force field led to a model exhibiting excellent stereochemical quality, with 92.9% of the φ/ψ values in the most favored regions and 6.3% in additionally allowed regions of the Ramachandran plot as evaluated with the program PROCHECK.<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a> Finally, the ATP molecule was removed.</div></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i10" class="anchor-spacer"></div><h3 class="article-section__title" id="_i10"> Energy Minimization of the Complexes</h3><div class="NLM_p">All inhibitor/protein complexes were minimized by the conjugate gradient algorithm to a root mean square of the energy gradient of 0.01 kcal mol<sup>−1</sup> Å<sup>−1</sup>. During minimization, the electrostatic energy term was screened by a distance-dependent dielectric of 4<i>r</i> to prevent artificial deviations due to vacuum effects, and the default nonbonding cutoff of 14 Å was used. Furthermore, the positions of all protein atoms were fixed. The minimized structures were used for the evaluation of the van der Waals energy and the finite-difference Poisson calculations.</div><div class="NLM_p last">As an illustrative example, the binding mode of the manually placed and minimized compound <b>14f</b> (shown in the <a class="ref internalNav" href="#si1" aria-label="Supporting Information as Figure 1">Supporting Information as Figure 1</a>) is very similar to the X-ray structure of compound <b>14e</b> in p38 (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1M7Q">1M7Q</a>, compound names according to ref <a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">35</a>). As described above, compound <b>14f</b> has been obtained by manual deletion/addition of atoms to the scaffold of the compound in the pose present in the X-ray structure (<b>14e</b>) and subsequent minimization with CHARMM<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> and the CHARMm22 force field (Accelrys Inc.<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a>). Similar small deviations were observed for most compounds.</div></div><div id="sec2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i11" class="anchor-spacer"></div><h3 class="article-section__title" id="_i11"> Binding Energy Evaluation</h3><div class="NLM_p">The van der Waals interaction energy and electrostatic interaction free energy were calculated by subtracting the values of the isolated components from the energy of the complex. The van der Waals energy was calculated with CHARMM and the CHARMm22 force field using the default nonbonding cutoff of 14 Å.</div><div class="NLM_p last">The electrostatic free energy is the sum of the Coulombic energy in vacuo and the solvation energy. The former was calculated with CHARMM using infinite cutoff and neglecting interactions between pairs of atoms separated by one or two covalent bonds. The electrostatic solvation energy was calculated by the finite-difference Poisson approach<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> using the PBEQ module<a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a> in CHARMM and a focusing procedure with a final grid spacing of 0.4 Å. Test calculations with a grid spacing of 0.3 Å yielded similar solvation energy values (results not shown). The size of the initial grid was determined by considering a layer of at least 20 Å around the solute. The dielectric discontinuity surface was delimited by the molecular surface spanned by the surface of a rolling probe of 1.4 Å. The ionic strength was set to zero and the temperature to 300 K. Two finite-difference Poisson calculations were performed for each of the three systems (inhibitor, protein, and inhibitor/protein complex). The exterior dielectric constant was set to 78.5 and 1.0 for the first and second calculation, respectively, while the solute dielectric constant was set to 1.0, which is consistent with the value used for the parametrization of the charges. The solvation energy is the difference between the two calculations.</div></div><div id="sec2_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12"> Experimental Methods</h3><div id="sec2_7_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i13" class="anchor-spacer"></div><h4 class="article-section__title" id="_i13"> Panvera</h4><div class="NLM_p last">In vitro kinase activity was measured using the Panvera Z’Lyte Tyr2 kinase assay PV3191 (Invitrogen) according to the manufacturerʼs instructions. Briefly, five dilutions of compound in a 3-fold series were measured, with the highest concentration being 125 μM. The reaction assay (10 μL) contained 7.5 ng of EphB4 kinase (Proqinase, Germany), 10 μM ATP, and 1% DMSO. The reaction was performed at room temperature for 1 h. Since this assay contains Brij, a nonionic detergent, aggregating compounds (i.e., promiscuous binders) should not show inhibition.<a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a></div></div><div id="sec2_7_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i14" class="anchor-spacer"></div><h4 class="article-section__title" id="_i14"> Cerep</h4><div class="NLM_p last">The assays performed at Cerep (Celle lʼEvescault, France) were done in duplicate at eight different concentrations ranging from 10 nM to 20 μM. The concentrations of ATP were 0.75 and 0.8 μM in the assays for EphB4 and CDK2, respectively. The concentrations of EphB4 and CDK2 were 0.2 and 1.25 μg/mL, respectively.</div></div><div id="sec2_7_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i15" class="anchor-spacer"></div><h4 class="article-section__title" id="_i15"> Biosource</h4><div class="NLM_p last">To provide more evidence against unspecific binding,<a onclick="showRef(event, 'ref49'); return false;" href="javascript:void(0);" class="ref ref49">(49)</a> compound <b>1</b> was tested in the Omnia Tyr Recombinant Kit KNZ4051 (Biosource) twice, i.e., without detergent and with 0.01% Triton X-100. In each of the two experiments, the compound was measured in duplicate at eight different concentrations between 50 nM and 100 μM. EphB4 kinase (Proqinase, Germany) and ATP were used at final concentrations of 25 ng/μL and 125 μM, respectively. The assay was run at 303 K for 1 h.</div></div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i16">3 Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_03104" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_03104" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec3_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> LIECE Models</h3><div class="NLM_p last">The equations used for fitting the calculated energy terms to the experimental free energies of binding (Δ<i>G</i> = <i>RT</i> ln(IC<sub>50</sub>) are a one-parameter model <span class="NLM_disp-formula" id="eq1"><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2008/jmcmar.2008.51.issue-5/jm070654j/production/images/jm-2007-00654j_m002.gif" alt="" /></img><span class="labelSpan">(1)</span></span> a two-parameter model with continuum electrostatics<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a><span class="NLM_disp-formula" id="eq2"><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2008/jmcmar.2008.51.issue-5/jm070654j/production/images/jm-2007-00654j_m003.gif" alt="" /></img><span class="labelSpan">(2)</span></span> and a three-parameter model with decomposed electrostatics <span class="NLM_disp-formula" id="eq3"><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2008/jmcmar.2008.51.issue-5/jm070654j/production/images/jm-2007-00654j_m004.gif" alt="" /></img><span class="labelSpan">(3)</span></span> where, as detailed above, Δ<i>E</i><sub>vdw</sub> is the intermolecular van der Waals energy and Δ<i>G</i><sub>elec</sub> is the sum of two terms: the intermolecular Coulombic energy in vacuo (Δ<i>E</i><sub>coul</sub>) plus the change in solvation energy of inhibitor and protein upon binding (Δ<i>G</i><sub>solv</sub>). Additional models were tested by taking into account the loss of translational and rotational degrees of freedom upon binding and the freezing of rotatable bonds of the inhibitor. No improvement was observed for the kinase inhibitors in this study. This result is consistent with the LIECE models of β-secretase but is in contrast to the results for 24 peptidic inhibitors of HIV-1 protease where a third parameter reflecting the penalty for the loss of translational and rotational entropy improved both fitting and predictive ability.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a></div></div><div id="sec3_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> Predictive Accuracy</h3><div class="NLM_p">The parameters obtained by least-squares fitting are given in Tables <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a>−<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a> for the LIECE models on individual kinases and combined data sets. The single-protein LIECE models derived from the 73 inhibitors of CDK2 show a higher level of predictivity than those derived from Lck or p38 (Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a>) probably because of the large range of activity values (Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>A). Notably, the derivation of predictive models using more than one kinase is possible despite the differing properties of the three ligand sets (Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a>). As an example, the two-parameter CDK2-Lck-p38-model (eq <a class="ref internalNav" href="#eq2" aria-label="2">2</a>) shows a root mean square of the error (rmse) of 1.29 and 1.46 kcal/mol in predicting the binding affinity of the 37 inhibitors of EphB4 and the 128 inhibitors of EGFR, respectively (Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a> and Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>). Most of the 37 inhibitors of EphB4 are predicted to bind more favorably, which might in part be a consequence of the use of a homology model. The cross-validation on EphB4 and EGFR yields rmse < 1.5 kcal/mol for most LIECE models (Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a>). Such robustness of the LIECE models is especially encouraging in the context of high-throughput screening by docking. In fact, the EphB4 LIECE model generated using only its 37 known inhibitors is not predictive (data not shown) because of the small range of IC<sub>50</sub> values (corresponding to binding free energies in the range from −9.4 to −6.3 kcal/mol; Berset et al., Oncalis AG, unpublished results). These results indicate that it is possible to derive predictive LIECE models even in cases where only inhibitors of “cognate” kinases are known.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. LIECE Parameters for the Single-Protein Models<a class="ref internalNav" href="#tbl1-fn3" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table frame_topbot"><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead valign="middle"><tr valign="top" class="rowsep1"><th class="rowsep1" align="center" valign="middle"> </th><th class="rowsep1" align="center" valign="bottom">α</th><th class="rowsep1" align="center" valign="bottom">β or β<sub>1</sub></th><th class="rowsep1" align="center" valign="bottom">β<sub>2</sub></th><th class="rowsep1" align="center" valign="middle" char=".">rms<a class="ref internalNav" href="#tbl1-fn1" aria-label="b">b</a>(kcal/mol)</th><th class="rowsep1" align="center" valign="middle" char=".">LOO<a class="ref internalNav" href="#tbl1-fn2" aria-label="c">c</a>cv <i>q</i><sup>2</sup></th></tr></thead><tbody><tr valign="top"><td colspan="6" align="center" valign="top">CDK2 (73 Inhibitors, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1KE5">1KE5</a>)</td></tr><tr valign="top"><td align="left" valign="top">αΔ<i>E</i><sub>vdW</sub></td><td align="left" valign="top">0.2388</td><td align="left" valign="top"> </td><td align="left" valign="top"> </td><td align="char" valign="top" char=".">0.98</td><td align="char" valign="top" char=".">0.80</td></tr><tr valign="top"><td align="left" valign="top">standard deviation</td><td align="left" valign="top">±0.0030</td><td align="left" valign="top"> </td><td align="left" valign="top"> </td><td align="char" valign="top" char="."> </td><td align="char" valign="top" char="."> </td></tr><tr valign="top"><td align="left" valign="top">αΔ<i>E</i><sub>vdW</sub> + βΔ<i>G<sub>elec</sub></i></td><td align="left" valign="top">0.2866</td><td align="left" valign="top">0.0520</td><td align="left" valign="top"> </td><td align="char" valign="top" char=".">0.93</td><td align="char" valign="top" char=".">0.82</td></tr><tr valign="top"><td align="left" valign="top">standard deviation</td><td align="left" valign="top">±0.0171</td><td align="left" valign="top">±0.0183</td><td align="left" valign="top"> </td><td align="char" valign="top" char="."> </td><td align="char" valign="top" char="."> </td></tr><tr valign="top"><td align="left" valign="top">αΔE<sub>vdW</sub> + β<sub>1</sub>Δ<i>E</i><sub>coul</sub> + β<sub>2</sub>Δ<i>G</i><sub>solv</sub></td><td align="left" valign="top">0.2395</td><td align="left" valign="top">0.0750</td><td align="left" valign="top"><i>0.0294</i></td><td align="char" valign="top" char=".">0.89</td><td align="char" valign="top" char=".">0.83</td></tr><tr valign="top"><td align="left" valign="top">standard deviation</td><td align="left" valign="top">±0.0265</td><td align="left" valign="top">±0.0208</td><td align="left" valign="top">±<i>0.0207</i></td><td align="char" valign="top" char="."> </td><td align="char" valign="top" char="."> </td></tr><tr valign="top"><td colspan="6" align="center" valign="top">Lck (51 Inhibitors, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1QPE">1QPE</a>)</td></tr><tr valign="top"><td align="left" valign="top">αΔ<i>E</i><sub>vdW</sub></td><td align="left" valign="top">0.2700</td><td align="left" valign="top"> </td><td align="left" valign="top"> </td><td align="char" valign="top" char=".">0.93</td><td align="char" valign="top" char=".">0.47</td></tr><tr valign="top"><td align="left" valign="top">standard deviation</td><td align="left" valign="top">±0.0037</td><td align="left" valign="top"> </td><td align="left" valign="top"> </td><td align="char" valign="top" char="."> </td><td align="char" valign="top" char="."> </td></tr><tr valign="top"><td align="left" valign="top">αΔ<i>E</i><sub>vdW</sub> + βΔ<i>G<sub>elec</sub></i></td><td align="left" valign="top">0.2735</td><td align="left" valign="top"><i>0.0046</i></td><td align="left" valign="top"> </td><td align="char" valign="top" char=".">0.93</td><td align="char" valign="top" char=".">0.44</td></tr><tr valign="top"><td align="left" valign="top">standard deviation</td><td align="left" valign="top">±0.0182</td><td align="left" valign="top">±<i>0.0237</i></td><td align="left" valign="top"> </td><td align="char" valign="top" char="."> </td><td align="char" valign="top" char="."> </td></tr><tr valign="top"><td align="left" valign="top">αΔ<i>E</i><sub>vdW</sub> + β<sub>1</sub>Δ<i>E</i><sub>coul</sub> + β<sub>2</sub>Δ<i>G</i><sub>solv</sub></td><td align="left" valign="top">0.2446</td><td align="left" valign="top">0.1528</td><td align="left" valign="top"><i>0.0076</i></td><td align="char" valign="top" char=".">0.84</td><td align="char" valign="top" char=".">0.53</td></tr><tr valign="top"><td align="left" valign="top">standard deviation</td><td align="left" valign="top">±0.0208</td><td align="left" valign="top">±0.0572</td><td align="left" valign="top">±<i>0.0237</i></td><td align="char" valign="top" char="."> </td><td align="char" valign="top" char="."> </td></tr><tr valign="top"><td colspan="6" align="center" valign="top">p38 (41 Inhibitors, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1M7Q">1M7Q</a>)</td></tr><tr valign="top"><td align="left" valign="top">αΔ<i>E</i><sub>vdW</sub></td><td align="left" valign="top">0.2377</td><td align="left" valign="top"> </td><td align="left" valign="top"> </td><td align="char" valign="top" char=".">1.01</td><td align="char" valign="top" char=".">0.40</td></tr><tr valign="top"><td align="left" valign="top">standard deviation</td><td align="left" valign="top">±0.0032</td><td align="left" valign="top"> </td><td align="left" valign="top"> </td><td align="char" valign="top" char="."> </td><td align="char" valign="top" char="."> </td></tr><tr valign="top"><td align="left" valign="top">αΔ<i>E</i><sub>vdW</sub> + βΔ<i>G</i><sub>elec</sub></td><td align="left" valign="top">0.2699</td><td align="left" valign="top">0.0264</td><td align="left" valign="top"> </td><td align="char" valign="top" char=".">0.98</td><td align="char" valign="top" char=".">0.43</td></tr><tr valign="top"><td align="left" valign="top">standard deviation</td><td align="left" valign="top">±0.0210</td><td align="left" valign="top">± 0.0170</td><td align="left" valign="top"> </td><td align="char" valign="top" char="."> </td><td align="char" valign="top" char="."> </td></tr><tr valign="top"><td align="left" valign="top">αΔ<i>E</i><sub>vdW</sub> + β<sub>1</sub>Δ<i>E</i><sub>coul</sub> + β<sub>2</sub>Δ<i>G</i><sub>solv</sub></td><td align="left" valign="top">0.1827</td><td align="left" valign="top">0.1584</td><td align="left" valign="top">−<i>0.0013</i></td><td align="char" valign="top" char=".">0.80</td><td align="char" valign="top" char=".">0.59</td></tr><tr valign="top"><td align="left" valign="top">standard deviation</td><td align="left" valign="top">±0.0316</td><td align="left" valign="top">±0.0397</td><td align="left" valign="top">±<i>0.0186</i></td><td align="char" valign="top" char="."> </td><td align="char" valign="top" char="."> </td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-tbl1-fn3"><div class="footnote" id="tbl1-fn3"><sup>a</sup><p class="last">The LIECE parameters for the single-protein models. Parameters with LOO variation of the same order of magnitude as the parameter itself are statistically not significant and are given in italics.</p></div><div class="footnote" id="tbl1-fn1"><sup>b</sup><p class="last">Root mean square of the error when predicting the Δ<i>G</i> values.</p></div><div class="footnote" id="tbl1-fn2"><sup>c</sup><p class="last">Leave-one-out cross-validated <i>q</i><sup>2</sup>.</p></div></div></div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. LIECE Parameters for the Two-Protein Models<a class="ref internalNav" href="#tbl2-fn4" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table frame_topbot"><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead valign="middle"><tr valign="top" class="rowsep1"><th class="rowsep1" align="center" valign="middle"> </th><th class="rowsep1" align="center" valign="bottom">α</th><th class="rowsep1" align="center" valign="bottom">β or β<sub>1</sub></th><th class="rowsep1" align="center" valign="bottom">β<sub>2</sub></th><th class="rowsep1" align="center" valign="middle" char=".">rms<a class="ref internalNav" href="#tbl2-fn1" aria-label="b">b</a>(kcal/mol)</th><th class="rowsep1" align="center" valign="middle" char=".">LOO<a class="ref internalNav" href="#tbl2-fn2" aria-label="c">c</a>cv <i>q</i><sup>2</sup></th><th class="rowsep1" align="center" valign="middle" char=".">rms<a class="ref internalNav" href="#tbl2-fn1" aria-label="b">b</a>(kcal/mol)</th><th class="rowsep1" align="center" valign="bottom" char="."><i>R</i><a class="ref internalNav" href="#tbl2-fn3" aria-label="d">d</a></th></tr></thead><tbody><tr valign="top"><td colspan="8" align="center">CDK2 + Lck (124 Inhibitors)</td></tr><tr valign="top"><td align="left" valign="top"> </td><td align="left" valign="top"> </td><td align="left" valign="top"> </td><td align="left" valign="top"> </td><td align="char" valign="top" char="."> </td><td align="char" valign="top" char="."> </td><td colspan="2" align="center" valign="top" char=".">prediction of 41 inhibitors of p38</td></tr><tr valign="top"><td align="left" valign="top">αΔ<i>E</i><sub>vdW</sub></td><td align="left" valign="top">0.2510</td><td align="left" valign="top"> </td><td align="left" valign="top"> </td><td align="char" valign="top" char=".">1.13</td><td align="char" valign="top" char=".">0.66</td><td align="char" valign="top" char=".">1.20</td><td align="char" valign="top" char=".">0.66</td></tr><tr valign="top"><td align="left" valign="top">standard deviation</td><td align="left" valign="top">±0.0023</td><td align="left" valign="top"> </td><td align="left" valign="top"> </td><td align="char" valign="top" char="."> </td><td align="char" valign="top" char="."> </td><td align="char" valign="top" char="."> </td><td align="char" valign="top" char="."> </td></tr><tr valign="top"><td align="left" valign="top">αΔ<i>E</i><sub>vdW</sub> + βΔ<i>G</i><sub>elec</sub></td><td align="left" valign="top">0.3072</td><td align="left" valign="top">0.0657</td><td align="left" valign="top"> </td><td align="char" valign="top" char=".">1.03</td><td align="char" valign="top" char=".">0.72</td><td align="char" valign="top" char=".">1.17</td><td align="char" valign="top" char=".">0.65</td></tr><tr valign="top"><td align="left" valign="top">standard deviation</td><td align="left" valign="top">±0.0113</td><td align="left" valign="top">±0.0129</td><td align="left" valign="top"> </td><td align="char" valign="top" char="."> </td><td align="char" valign="top" char="."> </td><td align="char" valign="top" char="."> </td><td align="char" valign="top" char="."> </td></tr><tr valign="top"><td align="left" valign="top">αΔ<i>E</i><sub>vdW</sub> + β<sub>1</sub>Δ<i>E</i><sub>coul</sub> + β<sub>2</sub>Δ<i>G</i><sub>solv</sub></td><td align="left" valign="top">0.3118</td><td align="left" valign="top">0.0440</td><td align="left" valign="top">0.0620</td><td align="char" valign="top" char=".">1.02</td><td align="char" valign="top" char=".">0.72</td><td align="char" valign="top" char=".">1.13</td><td align="char" valign="top" char=".">0.64</td></tr><tr valign="top"><td align="left" valign="top">standard deviation</td><td align="left" valign="top">±0.0116</td><td align="left" valign="top">±0.0180</td><td align="left" valign="top">±0.0131</td><td align="char" valign="top" char="."> </td><td align="char" valign="top" char="."> </td><td align="char" valign="top" char="."> </td><td align="char" valign="top" char="."> </td></tr><tr valign="top"><td colspan="8" align="center">CDK2 + p38 (114 Inhibitors)</td></tr><tr valign="top"><td align="left" valign="top"> </td><td align="left" valign="top"> </td><td align="left" valign="top"> </td><td align="left" valign="top"> </td><td align="char" valign="top" char="."> </td><td align="char" valign="top" char="."> </td><td colspan="2" align="center" valign="top" char=".">prediction of 51 inhibitors of Lck</td></tr><tr valign="top"><td align="left" valign="top">αΔ<i>E</i><sub>vdW</sub></td><td align="left" valign="top">0.2383</td><td align="left" valign="top"> </td><td align="left" valign="top"> </td><td align="char" valign="top" char=".">0.99</td><td align="char" valign="top" char=".">0.81</td><td align="char" valign="top" char=".">1.52</td><td align="char" valign="top" char=".">0.74</td></tr><tr valign="top"><td align="left" valign="top">standard deviation</td><td align="left" valign="top">±0.0022</td><td align="left" valign="top"> </td><td align="left" valign="top"> </td><td align="char" valign="top" char="."> </td><td align="char" valign="top" char="."> </td><td align="char" valign="top" char="."> </td><td align="char" valign="top" char="."> </td></tr><tr valign="top"><td align="left" valign="top">αΔ<i>E</i><sub>vdW</sub> + βΔ<i>G</i><sub>elec</sub></td><td align="left" valign="top">0.2632</td><td align="left" valign="top">0.0235</td><td align="left" valign="top"> </td><td align="char" valign="top" char=".">0.97</td><td align="char" valign="top" char=".">0.82</td><td align="char" valign="top" char=".">1.31</td><td align="char" valign="top" char=".">0.72</td></tr><tr valign="top"><td align="left" valign="top">standard deviation</td><td align="left" valign="top">±0.0103</td><td align="left" valign="top">±0.0095</td><td align="left" valign="top"> </td><td align="char" valign="top" char="."> </td><td align="char" valign="top" char="."> </td><td align="char" valign="top" char="."> </td><td align="char" valign="top" char="."> </td></tr><tr valign="top"><td align="left" valign="top">αΔ<i>E</i><sub>vdW</sub> + β<sub>1</sub>Δ<i>E</i><sub>coul</sub> + β<sub>2</sub>Δ<i>G</i><sub>solv</sub></td><td align="left" valign="top">0.2190</td><td align="left" valign="top">0.0439</td><td align="left" valign="top"><i>0.0006</i></td><td align="char" valign="top" char=".">0.93</td><td align="char" valign="top" char=".">0.83</td><td align="char" valign="top" char=".">1.83</td><td align="char" valign="top" char=".">0.76</td></tr><tr valign="top"><td align="left" valign="top">standard deviation</td><td align="left" valign="top">±0.0193</td><td align="left" valign="top">±0.0121</td><td align="left" valign="top">±<i>0.0127</i></td><td align="char" valign="top" char="."> </td><td align="char" valign="top" char="."> </td><td align="char" valign="top" char="."> </td><td align="char" valign="top" char="."> </td></tr><tr valign="top"><td colspan="8" align="center">Lck + p38 (92 Inhibitors)</td></tr><tr valign="top"><td align="left" valign="top"> </td><td align="left" valign="top"> </td><td align="left" valign="top"> </td><td align="left" valign="top"> </td><td align="char" valign="top" char="."> </td><td align="char" valign="top" char="."> </td><td colspan="2" align="center" valign="top" char=".">prediction of 73 inhibitors of CDK2</td></tr><tr valign="top"><td align="left" valign="top">αΔ<i>E</i><sub>vdW</sub></td><td align="left" valign="top">0.2513</td><td align="left" valign="top"> </td><td align="left" valign="top"> </td><td align="char" valign="top" char=".">1.19</td><td align="char" valign="top" char=".">0.39</td><td align="char" valign="top" char=".">1.10</td><td align="char" valign="top" char=".">0.90</td></tr><tr valign="top"><td align="left" valign="top">standard deviation</td><td align="left" valign="top">±0.0024</td><td align="left" valign="top"> </td><td align="left" valign="top"> </td><td align="char" valign="top" char="."> </td><td align="char" valign="top" char="."> </td><td align="char" valign="top" char="."> </td><td align="char" valign="top" char="."> </td></tr><tr valign="top"><td align="left" valign="top">αΔ<i>E</i><sub>vdW</sub> + βΔ<i>G</i><sub>elec</sub></td><td align="left" valign="top">0.3033</td><td align="left" valign="top">0.0508</td><td align="left" valign="top"> </td><td align="char" valign="top" char=".">1.00</td><td align="char" valign="top" char=".">0.56</td><td align="char" valign="top" char=".">1.16</td><td align="char" valign="top" char=".">0.91</td></tr><tr valign="top"><td align="left" valign="top">standard deviation</td><td align="left" valign="top">±0.0089</td><td align="left" valign="top">±0.0083</td><td align="left" valign="top"> </td><td align="char" valign="top" char="."> </td><td align="char" valign="top" char="."> </td><td align="char" valign="top" char="."> </td><td align="char" valign="top" char="."> </td></tr><tr valign="top"><td align="left" valign="top">αΔ<i>E</i><sub>vdW</sub> + β<sub>1</sub>Δ<i>E</i><sub>coul</sub> + β<sub>2</sub>Δ<i>G</i><sub>solv</sub></td><td align="left" valign="top">0.2939</td><td align="left" valign="top">0.1186</td><td align="left" valign="top">0.0584</td><td align="char" valign="top" char=".">0.98</td><td align="char" valign="top" char=".">0.57</td><td align="char" valign="top" char=".">1.69</td><td align="char" valign="top" char=".">0.92</td></tr><tr valign="top"><td align="left" valign="top">standard deviation</td><td align="left" valign="top">±0.0098</td><td align="left" valign="top">±0.0311</td><td align="left" valign="top">±0.0090</td><td align="char" valign="top" char="."> </td><td align="char" valign="top" char="."> </td><td align="char" valign="top" char="."> </td><td align="char" valign="top" char="."> </td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-tbl2-fn4"><div class="footnote" id="tbl2-fn4"><sup>a</sup><p class="last">The LIECE parameters for the two-protein models as well as the data for the predictions of the third protein not used for the derivation of the parameters. Parameters with LOO variation of the same order of magnitude as the parameter itself are statistically not significant and are given in italics.</p></div><div class="footnote" id="tbl2-fn1"><sup>b</sup><p class="last">Root mean square of the error when predicting the Δ<i>G</i> values.</p></div><div class="footnote" id="tbl2-fn2"><sup>c</sup><p class="last">Leave-one-out cross-validated <i>q</i><sup>2</sup>.</p></div><div class="footnote" id="tbl2-fn3"><sup>d</sup><p class="last">Correlation coefficient.</p></div></div></div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. LIECE Parameters for the Three-Protein Models<a class="ref internalNav" href="#tbl3-fn4" aria-label="a">a</a></div><p class="last"></p></div></div><div class="scrollable-table-wrap"><table class="table frame_topbot"><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead valign="middle"><tr valign="top" class="rowsep1"><th class="rowsep1" align="center" valign="bottom"> </th><th class="rowsep1" align="center" valign="bottom">α</th><th class="rowsep1" align="center" valign="bottom">β or β<sub>1</sub></th><th class="rowsep1" align="center" valign="bottom">β<sub>2</sub></th><th class="rowsep1" align="center" valign="middle" char=".">rms<a class="ref internalNav" href="#tbl3-fn1" aria-label="b">b</a>(kcal/mol)</th><th class="rowsep1" align="center" valign="middle" char=".">LOO<a class="ref internalNav" href="#tbl3-fn2" aria-label="c">c</a>cv <i>q</i><sup>2</sup></th><th class="rowsep1" align="center" valign="middle" char=".">rms<a class="ref internalNav" href="#tbl3-fn1" aria-label="b">b</a>(kcal/mol)</th><th class="rowsep1" align="center" valign="bottom" char="."><i>R</i><a class="ref internalNav" href="#tbl3-fn3" aria-label="d">d</a></th><th class="rowsep1" align="center" valign="middle" char=".">rms<a class="ref internalNav" href="#tbl3-fn1" aria-label="b">b</a>(kcal/mol)</th><th class="rowsep1" align="center" valign="bottom" char="."><i>R</i><a class="ref internalNav" href="#tbl3-fn3" aria-label="d">d</a></th></tr></thead><tbody><tr valign="top"><td colspan="10" align="center">CDK2 + Lck + p38 (165 Inhibitors)</td></tr><tr valign="top"><td align="left" valign="top"> </td><td align="left" valign="top"> </td><td align="left" valign="top"> </td><td align="left" valign="top"> </td><td align="char" valign="top" char="."> </td><td align="char" valign="top" char="."> </td><td colspan="2" align="center" valign="top" char=".">prediction of 128 inhibitors of EGFR</td><td colspan="2" align="center" valign="top" char=".">prediction of 37 inhibitors of EphB4</td></tr><tr valign="top"><td align="left" valign="top">αΔ<i>E</i><sub>vdW</sub></td><td align="left" valign="top">0.2463</td><td align="left" valign="top"> </td><td align="left" valign="top"> </td><td align="char" valign="top" char=".">1.13</td><td align="char" valign="top" char=".">0.69</td><td align="char" valign="top" char=".">1.58</td><td align="char" valign="top" char=".">0.53</td><td align="char" valign="top" char=".">0.92</td><td align="char" valign="top" char=".">0.16</td></tr><tr valign="top"><td align="left" valign="top">standard deviation</td><td align="left" valign="top">±0.0019</td><td align="left" valign="top"> </td><td align="left" valign="top"> </td><td align="char" valign="top" char="."> </td><td align="char" valign="top" char="."> </td><td align="char" valign="top" char="."> </td><td align="char" valign="top" char="."> </td><td align="char" valign="top" char="."> </td><td align="char" valign="top" char="."> </td></tr><tr valign="top"><td align="left" valign="top">αΔ<i>E</i><sub>vdW</sub> + βΔ<i>G</i><sub>elec</sub></td><td align="left" valign="top">0.2898</td><td align="left" valign="top">0.0442</td><td align="left" valign="top"> </td><td align="char" valign="top" char=".">1.03</td><td align="char" valign="top" char=".">0.74</td><td align="char" valign="top" char=".">1.46</td><td align="char" valign="top" char=".">0.52</td><td align="char" valign="top" char=".">1.29</td><td align="char" valign="top" char=".">0.15</td></tr><tr valign="top"><td align="left" valign="top">standard deviation</td><td align="left" valign="top">±0.0075</td><td align="left" valign="top">±0.0074</td><td align="left" valign="top"> </td><td align="char" valign="top" char="."> </td><td align="char" valign="top" char="."> </td><td align="char" valign="top" char="."> </td><td align="char" valign="top" char="."> </td><td align="char" valign="top" char="."> </td><td align="char" valign="top" char="."> </td></tr><tr valign="top"><td align="left" valign="top">αΔ<i>E</i><sub>vdW</sub> + β<sub>1</sub>Δ<i>E</i><sub>coul</sub> + β<sub>2</sub>Δ<i>G</i><sub>solv</sub></td><td align="left" valign="top">0.2961</td><td align="left" valign="top">0.0325</td><td align="left" valign="top">0.0454</td><td align="char" valign="top" char=".">1.03</td><td align="char" valign="top" char=".">0.74</td><td align="char" valign="top" char=".">1.46</td><td align="char" valign="top" char=".">0.53</td><td align="char" valign="top" char=".">1.36</td><td align="char" valign="top" char=".">0.16</td></tr><tr valign="top"><td align="left" valign="top">standard deviation</td><td align="left" valign="top">±0.0089</td><td align="left" valign="top">±0.0115</td><td align="left" valign="top">±0.0075</td><td align="char" valign="top" char="."> </td><td align="char" valign="top" char="."> </td><td align="char" valign="top" char="."> </td><td align="char" valign="top" char="."> </td><td align="char" valign="top" char="."> </td><td align="char" valign="top" char="."> </td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-tbl3-fn4"><div class="footnote" id="tbl3-fn4"><sup>a</sup><p class="last">The LIECE parameters for the three-protein models as well as the data for the predictions of the two test cases.</p></div><div class="footnote" id="tbl3-fn1"><sup>b</sup><p class="last">Root mean square of the error when predicting the Δ<i>G</i> values.</p></div><div class="footnote" id="tbl3-fn2"><sup>c</sup><p class="last">Leave-one-out cross-validated <i>q</i><sup>2</sup>.</p></div><div class="footnote" id="tbl3-fn3"><sup>d</sup><p class="last">Correlation coefficient.</p></div></div></div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2008/jmcmar.2008.51.issue-5/jm070654j/production/images/medium/jm-2007-00654j_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2008/jmcmar.2008.51.issue-5/jm070654j/production/images/large/jm-2007-00654j_0002.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. LIECE two-parameter CDK2-Lck-p38-model, with experimental Δ<i>G</i> = <i>RT</i> ln(IC<sub>50</sub>). (A) Data used for fitting are 73, 51, and 41 inhibitors of CDK2, Lck, and p38, respectively. Cross-validation was done on 37 inhibitors of EphB4 (B) and 128 inhibitors of EGFR (C). The green diagonal is the ideal line of perfect prediction. The black diagonals delimit the 1 kcal/mol error region. All values are in kcal/mol.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2008/jmcmar.2008.51.issue-5/jm070654j/production/images/large/jm-2007-00654j_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm070654j&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. Root Mean Square Errors of the Predictions of the Test Sets for the Different LIECE Models</div></div></div><div class="scrollable-table-wrap"><table class="table frame_topbot"><colgroup><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead valign="middle"><tr valign="top" class="rowsep1"><th class="rowsep1" align="center" valign="bottom">model</th><th class="rowsep1" align="center" valign="middle" char=".">rms EGFR (kcal/mol)</th><th class="rowsep1" align="center" valign="middle" char=".">rms EphB4 (kcal/mol)</th></tr></thead><tbody><tr valign="top"><td align="left" valign="top">αΔ<i>E</i><sub>vdW</sub></td><td align="char" valign="top" char="."> </td><td align="char" valign="top" char="."> </td></tr><tr valign="top"><td align="left" valign="top">CDK2</td><td align="char" valign="top" char=".">1.76</td><td align="char" valign="top" char=".">0.83</td></tr><tr valign="top"><td align="left" valign="top">Lck</td><td align="char" valign="top" char=".">1.37</td><td align="char" valign="top" char=".">1.50</td></tr><tr valign="top"><td align="left" valign="top">p38</td><td align="char" valign="top" char=".">1.78</td><td align="char" valign="top" char=".">0.82</td></tr><tr valign="top"><td align="left" valign="top">CDK2 + Lck</td><td align="char" valign="top" char=".">1.50</td><td align="char" valign="top" char=".">1.01</td></tr><tr valign="top"><td align="left" valign="top">CDK2 + p38</td><td align="char" valign="top" char=".">1.77</td><td align="char" valign="top" char=".">0.82</td></tr><tr valign="top"><td align="left" valign="top">Lck +p38</td><td align="char" valign="top" char=".">1.49</td><td align="char" valign="top" char=".">1.02</td></tr><tr valign="top"><td align="left" valign="top">CDK2 + Lck + p38</td><td align="char" valign="top" char=".">1.58</td><td align="char" valign="top" char=".">0.92</td></tr><tr valign="top"><td align="left" valign="top">αΔ<i>E</i><sub>vdW</sub> + βΔ<i>G</i><sub>elec</sub></td><td align="char" valign="top" char="."> </td><td align="char" valign="top" char="."> </td></tr><tr valign="top"><td align="left" valign="top">CDK2</td><td align="char" valign="top" char=".">1.63</td><td align="char" valign="top" char=".">1.10</td></tr><tr valign="top"><td align="left" valign="top">Lck</td><td align="char" valign="top" char=".">1.37</td><td align="char" valign="top" char=".">1.52</td></tr><tr valign="top"><td align="left" valign="top">p38</td><td align="char" valign="top" char=".">1.54</td><td align="char" valign="top" char=".">1.08</td></tr><tr valign="top"><td align="left" valign="top">CDK2 + Lck</td><td align="char" valign="top" char=".">1.48</td><td align="char" valign="top" char=".">1.40</td></tr><tr valign="top"><td align="left" valign="top">CDK2 + p38</td><td align="char" valign="top" char=".">1.63</td><td align="char" valign="top" char=".">0.97</td></tr><tr valign="top"><td align="left" valign="top">Lck + p38</td><td align="char" valign="top" char=".">1.40</td><td align="char" valign="top" char=".">1.55</td></tr><tr valign="top"><td align="left" valign="top">CDK2 + Lck + p38</td><td align="char" valign="top" char=".">1.46</td><td align="char" valign="top" char=".">1.29</td></tr><tr valign="top"><td align="left" valign="top">αΔ<i>E</i><sub>vdW</sub> + β<sub>1</sub>Δ<i>E</i><sub>coul</sub> + β<sub>2</sub>Δ<i>G</i><sub>solv</sub></td><td align="char" valign="top" char="."> </td><td align="char" valign="top" char="."> </td></tr><tr valign="top"><td align="left" valign="top">CDK2</td><td align="char" valign="top" char=".">1.89</td><td align="char" valign="top" char=".">0.84</td></tr><tr valign="top"><td align="left" valign="top">Lck</td><td align="char" valign="top" char=".">1.98</td><td align="char" valign="top" char=".">1.70</td></tr><tr valign="top"><td align="left" valign="top">p38</td><td align="char" valign="top" char=".">1.73</td><td align="char" valign="top" char=".">1.01</td></tr><tr valign="top"><td align="left" valign="top">CDK2 + Lck</td><td align="char" valign="top" char=".">1.47</td><td align="char" valign="top" char=".">1.48</td></tr><tr valign="top"><td align="left" valign="top">CDK2 + p38</td><td align="char" valign="top" char=".">1.84</td><td align="char" valign="top" char=".">0.82</td></tr><tr valign="top"><td align="left" valign="top">Lck + p38</td><td align="char" valign="top" char=".">1.46</td><td align="char" valign="top" char=".">1.58</td></tr><tr valign="top"><td align="left" valign="top">CDK2 + Lck + p38</td><td align="char" valign="top" char=".">1.46</td><td align="char" valign="top" char=".">1.36</td></tr></tbody></table></div></div><div class="NLM_p last">Interestingly, the three-parameter CDK2-Lck-p38-model does not show an improvement with respect to the two-parameter CDK2-Lck-p38-model in terms of predictivity (i.e., leave-one-out cross-validated <i>q</i><sup>2</sup>, Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a>). The β<sub>1</sub> and β<sub>2</sub> parameters are very close, indicating that the electrostatic term should not be decomposed into Coulombic and solvation terms.</div></div><div id="sec3_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i23" class="anchor-spacer"></div><h3 class="article-section__title" id="_i23"> Importance of van der Waals</h3><div class="NLM_p">It is noteworthy that the van der Waals parameter α varies only in a relatively small range of values, i.e., from 0.219 to 0.312, for 20 of the 21 LIECE models in Tables <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a>−<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a>. The value of α is also much larger than the electrostatic parameter β because of the large correlation between van der Waals energy and experimentally measured binding affinity. As an example, linear regression of these two quantities yields a correlation coefficient of 0.902 for the 73 inhibitors of CDK2. On the same set of inhibitors, there is an anticorrelation (correlation coefficient of −0.414) between the electrostatic free energy (Δ<i>G</i><sub>elec</sub>) and binding affinity. These results for individual energy terms are consistent with α = 0.287 and β = 0.052 in the LIECE two-parameter model of the 73 inhibitors of CDK2. As a basis of comparison, correlation coefficients of 0.829 (Δ<i>E</i><sub>vdw</sub> vs measured binding affinity) and −0.189 (Δ<i>G</i><sub>elec</sub> vs measured binding affinity) were obtained for the 13 peptidic inhibitors of β-secretase whose two-parameter LIECE model has α = 0.274 and β = 0.180.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a></div><div class="NLM_p">Moreover, the van der Waals energy alone can be used for both fitting and prediction of the test sets (Tables <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a>−<a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a>). In fact, the van der Waals CDK2-Lck-p38-model shows an even better predictive ability for the 37 inhibitors of EphB4 (rmse of 0.92 kcal/mol) than both the two- and three-parameter models on the same set (rmse of 1.29 and 1.36 kcal/mol, respectively). Furthermore, only a marginally worse accuracy is observed for the 128 inhibitors of EGFR (rmse of 1.58 kcal/mol for the van der Waals model vs 1.46 kcal/mol for both the two- and three-parameter models). Such predictive ability indicates that the van der Waals energy is the main factor needed to determine the relative binding strength of the known kinase inhibitors used in this study. In addition, it is likely that a very simple energy function is more predictive because of inaccuracies in the binding site structure, especially when a model of the protein is used as in the case of EphB4. On the basis of docking results obtained with a very simple energy function, it has recently been suggested that steric complementarity is a major factor in specific binding to different kinases.<a onclick="showRef(event, 'ref50'); return false;" href="javascript:void(0);" class="ref ref50">(50)</a></div><div class="NLM_p last">However, the relative importance of van der Waals and electrostatics might be significantly different for ranking only known inhibitors as opposed to a large database of (mainly) inactive compounds. In fact, the significantly higher enrichment factors (see below and <a class="ref internalNav" href="#si1" aria-label="Supporting Information">Supporting Information</a>) obtained with the two- and three-parameter models indicate that electrostatics are necessary to reduce false positives in high-throughput docking.</div></div><div id="sec3_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i24" class="anchor-spacer"></div><h3 class="article-section__title" id="_i24"> Parameter Transferability</h3><div class="NLM_p">The two-parameter single-protein models are similar (Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a>), and consequently transferable, because of the predominance of the van der Waals term and the steric similarity of the ATP-binding site in the three kinases. The latter observation is supported by the evaluation of the van der Waals interaction energy between the protein and a probe placed at vertices of a grid spanning the entire binding site. The probe consisted of a carbonyl group with a C=O bond length of 1.22 Å. First, the three kinase structures were overlapped (using the Cα atoms), and then the ATP-binding site was filled by a cubic grid of 1 Å spacing. The geometrical center of the probe was put on each grid point in turn, and the carbonyl was oriented along the positive direction of each of the three grid axes. Grid points for which the van der Waals interaction energy between the probe (in any of its three orientations) and the protein had a positive value were discarded. This procedure yielded 81 grid points that define the empty volume in common to the three kinases. On these grid points, the van der Waals interaction energy between the carbonyl probe and the protein is similar for the three kinases (Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a>). Similar results were obtained for the van der Waals interaction energy of a probe consisting of a single neon atom. These findings explain the parameter transferability if one considers that inhibitors that are energy-minimized in the ATP-binding site do not clash with the protein and therefore occupy a spatial region corresponding to a subset of the 81 grid points. Most importantly, this approach can also be used a priori to judge the similarity of binding sites and hence the transferability of parameters.</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2008/jmcmar.2008.51.issue-5/jm070654j/production/images/medium/jm-2007-00654j_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2008/jmcmar.2008.51.issue-5/jm070654j/production/images/large/jm-2007-00654j_0003.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Scatter plot of the van der Waals interaction energy between a C═O group and a kinase at 81 grid points in the ATP-binding site. Regression lines are solid black, and the values of <i>r</i> are the correlation coefficients. All values are in kcal/mol.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2008/jmcmar.2008.51.issue-5/jm070654j/production/images/large/jm-2007-00654j_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm070654j&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec3_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i25" class="anchor-spacer"></div><h3 class="article-section__title" id="_i25"> False Positives and Enrichment Factors</h3><div class="NLM_p">In the previous sections, the ability of LIECE models for predicting absolute binding free energy values has been tested on known inhibitors. These tests are important but yield only limited information on the usefulness of LIECE for its main application, i.e., ranking of poses of compounds from large libraries. In fact, most of these compounds do not bind at all to the target protein or they bind only weakly and nonspecifically. Therefore, it is essential to estimate the amount of false positives, i.e., compounds whose binding affinity is significantly overestimated by the LIECE approach. A non-negligible amount of false positives is expected to emerge, since the van der Waals interaction energy is a pairwise energy function and thus depends largely on the number of atoms. Moreover, because all docked molecules are subjected to energy minimization, the van der Waals energy is almost always favorable. Consequently, a ranking according to the two-parameter LIECE model will result in large molecules with a high number of atoms in the top ranks, and this ranking will not reflect the binding affinity. An unfeasible pose can only be penalized by the electrostatic term. However, this term has a very small coefficient in the LIECE models presented above. To decrease the number of false positives, poses with unlikely binding modes have to be eliminated before calculation of the LIECE energy (Table <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a>). One way to discard those poses is by means of a cutoff in the “van der Waals efficiency” (the ratio of the van der Waals interaction energy to the molecular weight). This criterion eliminates compounds with high molecular weight, which have a very favorable van der Waals interaction mainly because of their large number of atoms and not because of ideal steric complementarity with the binding site.<a onclick="showRef(event, 'ref50'); return false;" href="javascript:void(0);" class="ref ref50">(50)</a></div><div class="NLM_p">The use of a cutoff in the “Coulombic efficiency” (the ratio of the Coulombic interaction energy [evaluated using a distance-dependent dielectric of 4<i>r</i> and a nonbonding cutoff of 14 Å] to the molecular weight) leads to the elimination of compounds with poor charge complementarity which cannot be discarded by the kinase LIECE models due to the small value of β. Another filter is derived from the fact that inhibitors often show only one binding mode. LIECE models are in general calculated using known inhibitors, and their energy contributions are evaluated using their unique binding mode. In contrast, for the majority of compounds from large libraries, multiple poses are generated by docking programs. To limit the number of possible poses, it is helpful to take advantage of the knowledge from crystal structures. For the ATP-binding site of kinases, the existence of at least one hydrogen bond with the hinge region can be used as a filter, which reduces the average number (±standard deviation) of poses per compound from 11.7 ± 14.0 to 3.3 ± 3.6 (Table <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a>).</div><div class="NLM_table-wrap" id="tbl5"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 5. Enrichment Factors (EF)<a class="ref internalNav" href="#tbl5-fn10" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table frame_topbot"><colgroup><col align="left" /></col><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead valign="middle"><tr valign="top" class="rowsep1"><th class="rowsep1" align="center" valign="middle">filter</th><th class="rowsep1" align="center" valign="middle"><i>N</i><sub>inh</sub><a class="ref internalNav" href="#tbl5-fn11" aria-label="b">b</a></th><th class="rowsep1" align="center" valign="middle" char="."><i>N</i><sub>poses</sub><a class="ref internalNav" href="#tbl5-fn1" aria-label="c">c</a></th><th class="rowsep1" align="center" valign="middle" char="."><i>N</i><sub>cpds</sub><a class="ref internalNav" href="#tbl5-fn2" aria-label="d">d</a></th><th class="rowsep1" align="center" valign="middle"><i>N<sub>random</sub></i><a class="ref internalNav" href="#tbl5-fn3" aria-label="e">e</a></th><th class="rowsep1" align="center" valign="middle" char="."><i>N</i><sub>inh5%</sub><a class="ref internalNav" href="#tbl5-fn4" aria-label="f">f</a></th><th class="rowsep1" align="center" valign="middle" char=".">EF<a class="ref internalNav" href="#tbl5-fn5" aria-label="g">g</a></th></tr></thead><tbody><tr valign="top"><td align="left" valign="top"><i>E</i><sub>coul</sub>/MW ≤ −0.01 kcal/g</td><td align="left" valign="top">48</td><td align="char" valign="top" char=".">62141</td><td align="char" valign="top" char=".">7793</td><td align="left" valign="top">11951.8 ± 3.6</td><td align="char" valign="top" char=".">26</td><td align="char" valign="top" char=".">7.1</td></tr><tr valign="top"><td align="left" valign="top">H-bond to hinge region</td><td align="left" valign="top">48</td><td align="char" valign="top" char=".">22046</td><td align="char" valign="top" char=".">6716</td><td align="left" valign="top">8457.8 ± 4.3</td><td align="char" valign="top" char=".">41</td><td align="char" valign="top" char=".">11.2</td></tr><tr valign="top"><td align="left" valign="top"><i>E</i><sub>vdW</sub>/MW ≤ −0.11 kcal/g</td><td align="left" valign="top">32</td><td align="char" valign="top" char=".">121880</td><td align="char" valign="top" char=".">11842</td><td align="left" valign="top">13859.8 ± 2.4</td><td align="char" valign="top" char=".">16</td><td align="char" valign="top" char=".">4.4</td></tr><tr valign="top"><td align="left" valign="top">all filters</td><td align="left" valign="top">32</td><td align="char" valign="top" char=".">7895</td><td align="char" valign="top" char=".">2654</td><td align="left" valign="top">4914.0 ± 3.1</td><td align="char" valign="top" char=".">32</td><td align="char" valign="top" char=".">8.8</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-tbl5-fn10"><div class="footnote" id="tbl5-fn10"><sup>a</sup><p class="last">Enrichment factors. The initial set of active molecules consists of 73 known inhibitors of CDK2 with a unique binding mode. The initial database of inactive molecules consists of 40 375 ZINC compounds. Therefore, the composite library consists of 40 448 molecules. The LIECE two-parameter CDK2 model was evaluated for the 171 898 poses, of 14 701 ZINC compounds, with <i>E</i><sub>vdw</sub> ≤ −35 kcal/mol and <i>E</i><sub>vdw</sub>/MW ≤ −0.1 kcal/g. Only the pose with the most favorable LIECE energy was taken into account for the ranking used to calculate the values in the last two columns.</p></div><div class="footnote" id="tbl5-fn11"><sup>b</sup><p class="last">Number of known inhibitors that pass the filter.</p></div><div class="footnote" id="tbl5-fn1"><sup>c</sup><p class="last">Number of poses of docked compounds that pass the filter.</p></div><div class="footnote" id="tbl5-fn2"><sup>d</sup><p class="last">Number of unique docked compounds that pass the filter.</p></div><div class="footnote" id="tbl5-fn3"><sup>e</sup><p class="last">Number of compounds ± standard error when randomly picking <i>N</i><sub>poses</sub> poses of the 14 701 compounds (100 separate random pickings). Note that each cutoff filters out a significant amount of compounds and not just a few poses of every compound, as can be deduced from the fact that <i>N</i><sub>random</sub> is significantly larger than <i>N</i><sub>cpds</sub>.</p></div><div class="footnote" id="tbl5-fn4"><sup>f</sup><p class="last">Number of known inhibitors among the first 5% of the LIECE ranking.</p></div><div class="footnote" id="tbl5-fn5"><sup>g</sup><p class="last">Enrichment factors calculated as (<i>N</i><sub>inh5%</sub>/<i>N</i><sub>cpds,5%</sub>)(73/40448)<sup>−1</sup>, where <i>N</i><sub>cpds,5%</sub> = 2022 (5% of 40 448).</p></div></div></div><div class="NLM_p">The usefulness of these filters is illustrated by high-throughput docking into CDK2 using the <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1KE5">1KE5</a> structure.<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> A diverse set of 40 375 compounds has been selected from the ZINC database<a onclick="showRef(event, 'ref51'); return false;" href="javascript:void(0);" class="ref ref51">(51)</a> on the basis of mutual dissimilarity calculated by the program DAIM.<a onclick="showRef(event, 'ref52'); return false;" href="javascript:void(0);" class="ref ref52">(52)</a> By use of a fragment-based high-throughput docking approach,<a onclick="showRef(event, 'ref10 ref52 ref53 ref54'); return false;" href="javascript:void(0);" class="ref ref10 ref52 ref53 ref54">(10, 52-54)</a> a total of 690 530 poses were generated, with an average of 17.1 ± 19.7 poses per compound. Since it is computationally expensive to evaluate the LIECE energy for all poses, the electrostatic solvation free energy was calculated only for the 171 898 poses (of 14 701 unique compounds) with a van der Waals interaction energy more favorable than −35 kcal/mol and a van der Waals efficiency more favorable than −0.1 kcal/g. These cutoffs were determined by choosing values close to the peaks of histograms of the respective properties of the diverse set of 40 735 compounds (<a class="ref internalNav" href="#si1" aria-label="Supporting Information, Figure S2">Supporting Information, Figure S2</a>). Note that the 23 inhibitors of Gibson et al.<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> do not pass this filter because of their poor van der Waals interaction energy. This observation is consistent with their relatively poor binding affinity, which ranges from −6.6 to −5.5 kcal/mol, while the binding-affinity range of the compounds of Bramson et al. is −12.2 to −7.7 kcal/mol.<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a></div><div class="NLM_p">Upon application of the aforementioned filters, the LIECE two-parameter CDK2-model ranks most of the known inhibitors before most of the docked compounds as shown by the receiver-operating characteristic (ROC<a onclick="showRef(event, 'ref55'); return false;" href="javascript:void(0);" class="ref ref55">(55)</a>) plots (Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a>). In other words, the combined use of filters and LIECE model generates few false positives. It is important to note that by keeping the poses with a very favorable van der Waals term there is an enrichment of compounds with favorable Δ<i>G</i> evaluated according to the van der Waals dominated LIECE model. Consequently, the ROC plots shown in Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a> underestimate the enrichments that can be achieved by using a combined filter/LIECE ranking. In light of this, it is worth noting that by use of the filters mentioned above, half of the 32 known inhibitors (which passed the filters) are ranked among the first 300 compounds of an initial library of 40 375 diverse compounds. Furthermore, upon ranking with the LIECE two-parameter CDK2-model, the 5% enrichment factors range from 4.4 to 11.2 depending on the filter (Table <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a>, last column). Interestingly, the enrichment factors are quite insensitive to the choice of the cutoff value as shown in the <a class="ref internalNav" href="#si1" aria-label="Supporting Information (Figure S3)">Supporting Information (Figure S3)</a>. In particular, the enrichment is almost constant for a van der Waals efficiency cutoff between −0.12 and −0.10 kcal/g or a Coulombic efficiency cutoff between −0.06 and −0.01 kcal/g.</div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2008/jmcmar.2008.51.issue-5/jm070654j/production/images/medium/jm-2007-00654j_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2008/jmcmar.2008.51.issue-5/jm070654j/production/images/large/jm-2007-00654j_0004.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. ROC plot for CDK2 showing improvement of the retrieval rate of known inhibitors by using filters. A set of 40 375 compounds from the ZINC library was used as starting database of inactive molecules. This set of 40 375 compounds was seeded with 73 known inhibitors, and the filters described below were applied to both ZINC compounds and inhibitors. Poses were first filtered according to an <i>E</i><sub>vdW</sub> ≤ –35 kcal/mol and an <i>E</i><sub>vdw</sub>/MW ≤ –0.1 kcal/g, which yielded 171 898 poses of 14 701 ZINC compounds. These 14 701 compounds and the 48 inhibitors that passed the above filter make up the set to which all subsequent filters (see also Table <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a>) were applied. (Black curve) The LIECE two-parameter CDK2-model was used to rank the combined library consisting of 48 known inhibitors of CDK2 and the pose with the most favorable LIECE energy of each of the 14 701 compounds. The area under this curve is <i>A</i><sub>ROC</sub> = 0.76. (Red curve) Forty-eight known inhibitors and 7793 compounds with Coulombic efficiency more favorable than −0.01 kcal/g, <i>A</i><sub>ROC</sub> = 0.79. (Green curve) Forty-eight known inhibitors and 6716 compounds with at least one hydrogen bond to the hinge region, <i>A</i><sub>ROC</sub> = 0.82. (Blue curve) Thirty-two known inhibitors and 11 842 compounds with van der Waals efficiency more favorable than −0.11 kcal/g, <i>A</i><sub>ROC</sub> = 0.81. (Orange curve) Thirty-two known inhibitors and 2654 compounds that satisfy all aforementioned filters, <i>A</i><sub>ROC</sub> = 0.86. The dashed line is the random model.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2008/jmcmar.2008.51.issue-5/jm070654j/production/images/large/jm-2007-00654j_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm070654j&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">The ranking obtained by the LIECE one-parameter CDK2-model yielded significantly worse enrichment factors than the two-parameter model (<a class="ref internalNav" href="#si1" aria-label="Supporting Information">Supporting Information</a>). This result indicates that while van der Waals energy alone is sufficient for self-prediction, it is not possible to neglect electrostatic interactions for ranking large libraries of compounds upon high-throughput docking. For the kinase used in this study, the most accurate filter in terms of final enrichment is the orientational filter, which requires at least one hydrogen bond with the hinge region. Importantly, the effect of each of the filters is specific because they tend to eliminate all poses of few compounds rather than few poses of many compounds (Table <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a>).</div></div><div id="sec3_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i27" class="anchor-spacer"></div><h3 class="article-section__title" id="_i27"> Experimental Validation</h3><div class="NLM_p">In the search for EphB4 inhibitors, a two-step procedure has recently been adopted to rank compounds upon fragment-based high-throughput docking. The van der Waals efficiency and presence of at least one hydrogen bond with the hinge region were first used to filter out unlikely poses. Then the poses that passed these filters were ranked according to the two-parameter CDK2-Lck-p38-model (Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a>). In this way, two inhibitors of EphB4 with an IC<sub>50</sub> below 10 μM in two different enzymatic assays and a compound with an IC<sub>50</sub> of 76 μM (tested in one assay) could be identified (Table <a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">6</a>). Notably, only 43 compounds were tested in enzymatic assays. To exclude unspecific inhibition effects due to aggregation, compound <b>1</b> was also tested in an assay with and without nonionic detergent. Encouragingly, very similar values of IC<sub>50</sub> were obtained under both conditions in the Omnia Tyr recombinant kit KNZ4051 (Biosource). Such behavior is generally interpreted as strong evidence for specific inhibition, i.e., for a nonpromiscuous and nonaggregating compound.<a onclick="showRef(event, 'ref49'); return false;" href="javascript:void(0);" class="ref ref49">(49)</a> The same result can also be expected for compound <b>2</b>, since the two compounds share the same scaffold and have 24 of 26 heavy atoms in common. It is important to underline that the “general” LIECE model can successfully be used to identify inhibitors of kinases that were not used to derive it.</div><div class="NLM_table-wrap" id="tbl6"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 6. Experimental Data on Kinase Inhibitors Discovered by High-Throughput Docking and LIECE Rankings</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2008/jmcmar.2008.51.issue-5/jm070654j/production/images/medium/jm-2007-00654j_0006.gif" alt="" id="_i29" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-tbl6-fn1"><div class="footnote" id="tbl6-fn1"><sup>Table a</sup><p class="last">Panvera FRET-based enzymatic assay. Compounds <b>1</b> and <b>2</b> were tested twice. Experiments performed using 5 concentrations (<b>1</b>, <b>3</b>) or 3 and 10 concentrations (<b>2</b>).</p></div><div class="footnote" id="tbl6-fn2"><sup>Table b</sup><p class="last">FRET-based enzymatic assay performed at Cerep (Celle lʼEvescault, France). Duplicate measurements.</p></div><div class="footnote" id="tbl6-fn3"><sup>Table c</sup><p class="last">LIECE two-parameter CDK2-Lck-p38 model for EphB4 ranking and two-parameter CDK2 model for CDK2 ranking.</p></div></div><div></div></div><div class="NLM_p">To further validate the LIECE model for scoring, it was recently used in a docking campaign to identify CDK2 inhibitors. Again, filters were applied to the van der Waals efficiency and the presence of at least one key hydrogen bond. Thereafter, the two-parameter CDK2-model was used to rank the remaining poses. In this docking campaign only 30 compounds were tested in enzymatic assays and compound <b>4</b> emerged with an IC<sub>50</sub> below 10 μM. Its predicted binding mode with three hydrogen bonds to the hinge region is depicted in Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a>.</div><figure id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2008/jmcmar.2008.51.issue-5/jm070654j/production/images/medium/jm-2007-00654j_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2008/jmcmar.2008.51.issue-5/jm070654j/production/images/large/jm-2007-00654j_0005.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Predicted binding mode of compound <b>4</b>. Intermolecular hydrogen bonds to the hinge region are shown by dashed cyan lines. The pose shown is the one with the lowest binding energy according to the two-parameter CDK2 model after docking with DAIM, SEED, and FFLD and minimization with CHARMm22 in CDK2 (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1KE5">1KE5</a>). Figure was prepared with PyMOL (DeLano Scientific, San Carlos, CA).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2008/jmcmar.2008.51.issue-5/jm070654j/production/images/large/jm-2007-00654j_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm070654j&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec3_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i30" class="anchor-spacer"></div><h3 class="article-section__title" id="_i30"> Computational Requirements</h3><div class="NLM_p last">The LIECE approach requires about 5 min (mainly for the finite-difference Poisson calculations) of CPU time on a single Athlon 2.1 GHz for each kinase inhibitor. It is about 2 orders of magnitude faster than the most efficient LIE method reported (about 6 h for each inhibitor<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a>). For a protein of 250 residues, the memory requirement for the finite-difference Poisson calculations is about 0.3 GB.</div></div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i31">4 Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_83268" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_83268" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">The main goal of this work was to evaluate the usefulness of LIECE in structure-based high-throughput docking. Particular emphasis was put on parameter robustness and transferability whose ultimate validation was based on the application of a LIECE model, fitted using inhibitors of CDK2, Lck, and p38, to rank poses of a large library of compounds and identify low-micromolar inhibitors of a kinase (EphB4) not used to derive the model.</div><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i32" class="anchor-spacer"></div><h3 class="article-section__title" id="_i32"> In Silico Validation</h3><div class="NLM_p">Three main results emerge from the computational part of this study. First, a LIECE two-parameter model (van der Waals and electrostatics) based on three kinases shows good predictive ability for a set of 165 inhibitors of two distantly related kinases not used to generate the model. In particular, the binding free energy of 37 inhibitors of EphB4 and 128 inhibitors of EGFR is predicted with an average error of 1.3 and 1.5 kcal/mol, respectively. These results compare favorably with the extended linear response (ELR) approach with Monte Carlo simulations,<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> which has been applied to the same three kinases used as training set in the present work. A good predictive ability has been reported for a five-parameter ELR model, but the cross-validation did not include a completely independent test set of inhibitors and was based solely on the leave-one-out procedure.<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> A leave-one-out cross-validated correlation coefficient of 0.67 was reported for the 148 inhibitors used to derive the five-parameter ELR model,<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> while in the present study a value of 0.74 is obtained for the 165 inhibitors used to derive the two-parameter LIECE model. It has to be mentioned that the main objective of the ELR study was to investigate the significance of individual energetic and structural descriptors (e.g., steric and electrostatic complementarity between kinase and inhibitor were present in all ELR models<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a>). In fact, in contrast to the LIECE model, the ELR approach cannot be used for high-throughput docking because of the very high computational cost required for Monte Carlo sampling with explicit water treatment (22 Å water sphere for the ATP-binding site of kinases).</div><div class="NLM_p">Second, the LIECE parameter transferability among kinases is mainly due to the dominance of the van der Waals interaction energy, whose multiplicative parameter is between 5 and 10 times larger than the one of the electrostatic free energy. A possible explanation is that the shape of the ATP-binding site is highly conserved among the different kinases. In a previous work it was found that LIECE parameters are not transferable between human β-secretase and HIV-1 protease,<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> despite the fact that both enzymes are aspartic proteases. This lack of generality originates from the significant differences between the substrate-binding site of mammalian and viral aspartic proteases which bind different polypeptide substrates. Moreover, the electrostatic interactions play a more important role in the substrate-binding site of β-secretase than HIV-1 protease.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> Although the van der Waals term seems to dominate in the LIECE models of kinases, the ranking of large libraries seeded with a few known inhibitors shows significantly better enrichment factors using LIECE models with both van der Waals and electrostatics. A pure van der Waals model generates many false positives because it favors large compounds.</div><div class="NLM_p last">Third, upon high-throughput docking, it is essential to weed out compounds with unlikely binding modes to decrease the number of false positives in a LIECE ranking. In this work, the application of filters based on the van der Waals efficiency, the Coulombic efficiency, and the presence of key hydrogen bonds between ligand and protein proved useful. Being developed with information of known inhibitors, a LIECE model alone cannot identify all of the unlikely binding modes. Moreover, only energetic criteria are used by LIECE, namely, van der Waals and electrostatics. In this context, it is not surprising that the orientational filter requiring at least one hydrogen bond with the hinge region is most efficient in terms of enrichment of known inhibitors, as it uses information that is not already contained in the LIECE models.</div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i33" class="anchor-spacer"></div><h3 class="article-section__title" id="_i33"> In Vitro Validation</h3><div class="NLM_p last">The ultimate test of any computational approach has to involve an experimental validation. In this work, the efficacy of the combined filter/LIECE approach was demonstrated by scoring the poses resulting from two recent high-throughput docking campaigns for discovering ATP-binding site inhibitors of the kinases EphB4 and CDK2. Overall, three novel inhibitors with IC<sub>50</sub> values below 10 μM were identified with only 73 compounds tested. This shows that the combined use of filters and LIECE is capable of correctly selecting strong binders. It must be stressed again that the three inhibitors of EphB4 were discovered upon ranking with the LIECE two-parameter CDK2-Lck-p38-model. This result provides strong evidence on the transferability of the model. Most importantly, the usefulness of a general LIECE model for kinases is evident if one considers that it is impossible to derive a LIECE model based on the known EphB4 inhibitors, which are few and span a small range of activity values.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="si1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i35"><a href="/doi/suppl/10.1021/jm070654j">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_47836" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_47836" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">Depiction of the predicted binding mode of compound <b>14f</b>, histograms of the van der Waals interaction energy and the van der Waals efficiency, all 2D structures of known inhibitors, and enrichment factors for the three-protein one-, two-, and three-parameter models. This material is available free of charge via the Internet at <a href="http://pubs.acs.org" class="extLink">http://pubs.acs.org</a>.</p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/jm070654j/suppl_file/jm070654j-file002.pdf">jm070654j-file002.pdf (677.13 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/jm070654j" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_47616" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_47616" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Amedeo Caflisch</span> - <span class="hlFld-Affiliation affiliation">Department of Biochemistry, University of Zürich, Winterthurerstrasse 190, CH-8057 Zürich, Switzerland</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#1b787a7d77726878735b79727478356e6173357873"><span class="__cf_email__" data-cfemail="1b787a7d77726878735b79727478356e6173357873">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Peter Kolb</span> - <span class="hlFld-Affiliation affiliation">Department of Biochemistry, University of Zürich, Winterthurerstrasse 190, CH-8057 Zürich, Switzerland</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Danzhi Huang</span> - <span class="hlFld-Affiliation affiliation">Department of Biochemistry, University of Zürich, Winterthurerstrasse 190, CH-8057 Zürich, Switzerland</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Fabian Dey</span> - <span class="hlFld-Affiliation affiliation">Department of Biochemistry, University of Zürich, Winterthurerstrasse 190, CH-8057 Zürich, Switzerland</span></div></li></ul></li><li></li><li></li><li></li></ul></div><div class="ack"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i34">Acknowledgment</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_36411" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_36411" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">We thank Stjepan Jelakovic for the generation of the EphB4 model structure, and we thank Stephan Audétat, Catherine Berset, and Julia Tietz for activity measurements of the EphB4 compounds. We also thank Dariusz Ekonomiuk, Stefanie Muff, and Pietro Alfarano for comments on the manuscript. The calculations were performed on Matterhorn, a Beowulf Linux cluster at the Informatikdienste of the University of Zurich, and we thank C. Bolliger, T. Steenbock, and A. Godknecht for installing and maintaining the Linux cluster. We thank A. Widmer (Novartis Pharma, Basel, Switzerland) for providing a program for multiple linear regression and the molecular modeling program Wit!P, which was used for preparing the structures. This work was supported by the KTI (Kommission Technologie and Innovation) and the National Center of Competence in Research “Neural Plasticity and Repair” (NCCR Neuro).</p></div><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i36">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_37813" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_37813" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 55 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group">Holloway, M. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wai, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Halgren, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fitzgerald, P. M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vacca, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dorsey, B. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Levi, R. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thompson, W. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, L. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">deSolms, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gaffin, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ghosh, A. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Giuliani, E. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Graham, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guare, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hungate, R. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lyle, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sanders, W. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tucker, T. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wiggins, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wiscount, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Woltersdorf, O. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Young, S. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Darke, P. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zugay, J. A.</span><span> </span><span class="NLM_article-title">A priori prediction of activity for HIV-1 protease inhibitors employing energy minimization in the active site</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1995</span><span class="NLM_x">, </span> <span class="NLM_volume">38</span><span class="NLM_x">, </span> <span class="NLM_fpage">305</span><span class="NLM_x">–</span> <span class="NLM_lpage">317</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm00002a012" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=1995&pages=305-317&author=M.+K.+Hollowayauthor=J.+M.+Waiauthor=T.+A.+Halgrenauthor=P.+M.+D.+Fitzgeraldauthor=J.+P.+Vaccaauthor=B.+D.+Dorseyauthor=R.+B.+Leviauthor=W.+J.+Thompsonauthor=L.+J.+Chenauthor=S.+J.+deSolmsauthor=N.+Gaffinauthor=A.+K.+Ghoshauthor=E.+A.+Giulianiauthor=S.+L.+Grahamauthor=J.+P.+Guareauthor=R.+W.+Hungateauthor=T.+A.+Lyleauthor=W.+M.+Sandersauthor=T.+J.+Tuckerauthor=M.+Wigginsauthor=C.+M.+Wiscountauthor=O.+W.+Woltersdorfauthor=S.+D.+Youngauthor=P.+L.+Darkeauthor=J.+A.+Zugay&title=A+priori+prediction+of+activity+for+HIV-1+protease+inhibitors+employing+energy+minimization+in+the+active+site"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1021%2Fjm00002a012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm00002a012%26sid%3Dliteratum%253Aachs%26aulast%3DHolloway%26aufirst%3DM.%2BK.%26aulast%3DWai%26aufirst%3DJ.%2BM.%26aulast%3DHalgren%26aufirst%3DT.%2BA.%26aulast%3DFitzgerald%26aufirst%3DP.%2BM.%2BD.%26aulast%3DVacca%26aufirst%3DJ.%2BP.%26aulast%3DDorsey%26aufirst%3DB.%2BD.%26aulast%3DLevi%26aufirst%3DR.%2BB.%26aulast%3DThompson%26aufirst%3DW.%2BJ.%26aulast%3DChen%26aufirst%3DL.%2BJ.%26aulast%3DdeSolms%26aufirst%3DS.%2BJ.%26aulast%3DGaffin%26aufirst%3DN.%26aulast%3DGhosh%26aufirst%3DA.%2BK.%26aulast%3DGiuliani%26aufirst%3DE.%2BA.%26aulast%3DGraham%26aufirst%3DS.%2BL.%26aulast%3DGuare%26aufirst%3DJ.%2BP.%26aulast%3DHungate%26aufirst%3DR.%2BW.%26aulast%3DLyle%26aufirst%3DT.%2BA.%26aulast%3DSanders%26aufirst%3DW.%2BM.%26aulast%3DTucker%26aufirst%3DT.%2BJ.%26aulast%3DWiggins%26aufirst%3DM.%26aulast%3DWiscount%26aufirst%3DC.%2BM.%26aulast%3DWoltersdorf%26aufirst%3DO.%2BW.%26aulast%3DYoung%26aufirst%3DS.%2BD.%26aulast%3DDarke%26aufirst%3DP.%2BL.%26aulast%3DZugay%26aufirst%3DJ.%2BA.%26atitle%3DA%2520priori%2520prediction%2520of%2520activity%2520for%2520HIV-1%2520protease%2520inhibitors%2520employing%2520energy%2520minimization%2520in%2520the%2520active%2520site%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1995%26volume%3D38%26spage%3D305%26epage%3D317" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group">Apostolakis, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Caflisch, A.</span><span> </span><span class="NLM_article-title">Computational ligand design.</span> <span class="citation_source-journal">Comb. Chem. High Throughput Screening</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1999</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">91</span><span class="NLM_x">–</span> <span class="NLM_lpage">104</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Fjm070654j&amp;key=10420978" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Fjm070654j&amp;key=1%3ACAS%3A528%3ADyaK1MXjsFWmtbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=1999&pages=91-104&author=J.+Apostolakisauthor=A.+Caflisch&title=Computational+ligand+design."target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Computational ligand design</span></div><div class="casAuthors">Apostolakis, J.; Caflisch, A.</div><div class="citationInfo"><span class="NLM_cas:title">Combinatorial Chemistry and High Throughput Screening</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">91-104</span>CODEN:
                <span class="NLM_cas:coden">CCHSFU</span>;
        ISSN:<span class="NLM_cas:issn">1386-2073</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers</span>)
        </div><div class="casAbstract">A review with 108 refs. of variety of computational tools that are used to assist drug design.  Particular emphasis is given to the limitations and merits of different methodologies.  Recently, a no. of general methods have been proposed for clustering compds. in classes of drug-like and non-drug-like mols.  The usefulness of this classification for drug design is discussed.  The estn. of (relative) binding affinities is from a theor. point of view the most challenging part of ligand design.  The authors review three methods for the estn. of binding energies.  Firstly, quant. structure-activity relationships (QSAR) are presented.  These have gained significantly from recent developments of exptl. techniques for combinatorial synthesis and high-throughput screening as well as the use of powerful computational procedures like genetic algorithms and neural networks for the derivation of models.  Secondly, empirical energy functions are shown to lead to more general models than std. QSAR, since they are fitted to a variety of complexes.  They have been used recently with considerable success.  Thirdly, the authors briefly outline free energy calcns. based on mol. dynamics simulations, the method with the most sound theor. foundation.  Recent developments are reestablishing the interest in this approach.  In the last part of this review, structure-based ligand design programs are described.  These are closely related to docking, with the difference that in design, unlike in most docking procedures, ligands are built on a fragment-by-fragment basis.  Finally, a short description of the authors approach to computational combinatorial ligand design is given.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpCT9f0aXUDm7Vg90H21EOLACvtfcHk0ljnS8n20tvCFA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXjsFWmtbs%253D&md5=4d9a30b781baac1571e496bbc49bbd53</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DApostolakis%26aufirst%3DJ.%26aulast%3DCaflisch%26aufirst%3DA.%26atitle%3DComputational%2520ligand%2520design.%26jtitle%3DComb.%2520Chem.%2520High%2520Throughput%2520Screening%26date%3D1999%26volume%3D2%26spage%3D91%26epage%3D104" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group">Doman, T. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McGovern, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Witherbee, B. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kasten, T. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kurumbail, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stallings, W. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Connolly, D. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">Molecular docking and high-throughput screening for novel inhibitors of protein tyrosine phosphatase-1B</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">45</span><span class="NLM_x">, </span> <span class="NLM_fpage">2213</span><span class="NLM_x">–</span> <span class="NLM_lpage">2221</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm010548w" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2002&pages=2213-2221&author=T.+N.+Domanauthor=S.+L.+McGovernauthor=B.+J.+Witherbeeauthor=T.+P.+Kastenauthor=R.+Kurumbailauthor=W.+C.+Stallingsauthor=D.+T.+Connollyauthor=B.+K.+Shoichet&title=Molecular+docking+and+high-throughput+screening+for+novel+inhibitors+of+protein+tyrosine+phosphatase-1B"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1021%2Fjm010548w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm010548w%26sid%3Dliteratum%253Aachs%26aulast%3DDoman%26aufirst%3DT.%2BN.%26aulast%3DMcGovern%26aufirst%3DS.%2BL.%26aulast%3DWitherbee%26aufirst%3DB.%2BJ.%26aulast%3DKasten%26aufirst%3DT.%2BP.%26aulast%3DKurumbail%26aufirst%3DR.%26aulast%3DStallings%26aufirst%3DW.%2BC.%26aulast%3DConnolly%26aufirst%3DD.%2BT.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DMolecular%2520docking%2520and%2520high-throughput%2520screening%2520for%2520novel%2520inhibitors%2520of%2520protein%2520tyrosine%2520phosphatase-1B%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2002%26volume%3D45%26spage%3D2213%26epage%3D2221" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group">Glen, R. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allen, S. C.</span><span> </span><span class="NLM_article-title">Ligand−protein docking: cancer research at the interface between biology and chemistry</span> <span class="citation_source-journal">Curr. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">763</span><span class="NLM_x">–</span> <span class="NLM_lpage">777</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Fjm070654j&amp;key=10.2174%2F0929867033457809" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Fjm070654j&amp;key=12678780" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Fjm070654j&amp;key=1%3ACAS%3A528%3ADC%252BD3sXjtVaju7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2003&pages=763-777&author=R.+C.+Glenauthor=S.+C.+Allen&title=Ligand%E2%88%92protein+docking%3A+cancer+research+at+the+interface+between+biology+and+chemistry"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Ligand-protein docking: Cancer research at the interface between biology and chemistry</span></div><div class="casAuthors">Glen, R. C.; Allen, S. C.</div><div class="citationInfo"><span class="NLM_cas:title">Current Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">763-777</span>CODEN:
                <span class="NLM_cas:coden">CMCHE7</span>;
        ISSN:<span class="NLM_cas:issn">0929-8673</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  In recent years there has been a growing interest in computer-based screening.  One of the driving forces has been the increased efficiency of protein crystallog. leading to the real possibility of using structure-based design as a significant contributor to the discovery of novel ligands.  In 1957 after 22 yr of work the first protein structure, detd. by x-ray crystallog. was produced [1].  Now the process has become increasingly automated and nearly 20,000 protein structures are available in the Protein Data Bank (PDB) [2].  Equally, progress in genomics will result in a great expansion of validated targets for cancer therapy.  The understanding of the relationships between structure and function of gene products will be one of the key routes to new therapeutic advances.  The challenge now is to use this data in the discovery of novel therapeutics.  One approach is obviously to synthesize mols. and co-crystallize or soak them into the protein crystal and so det. the position and interaction of the mol. with the protein.  The structural information obtained (where does the mol. bind; what are the ligand/protein/solvent interactions) can be invaluable in the generation of novel mols. or in the re-design of existing mols. whose drug properties are not optimal.  However, when dealing with large nos. (millions) of mols., when crystn. is difficult or in testing hypotheses, a significant contribution can be made using computer based screening methods.  To use the structural information derived from x-ray crystallog. (or other sources, for example NMR or homol. modeling) when evaluating the utility of a novel ligand, we need to understand where in the protein (or other macromol. such as RNA) the ligand is likely to bind and also if possible, the strength of the binding interactions.  This problem is known as the "docking problem".  There have been many approaches to the soln. of this problem over the last ten years.  For example, some methods rely on complex mol. dynamics simulations while others use less costly graph matching approaches.  There is generally a compromise between speed and accuracy, with some methods giving much more information and insight into the nature of the protein/ligand interactions and other methods optimized for speed of docking thousands of putative ligands.  We will describe some of the more common methods and algorithms used to solve the docking problem and in particular, we will review recent applications in cancer research.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpgEI_kOJr7ILVg90H21EOLACvtfcHk0lh72UA3wGo-Bw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXjtVaju7c%253D&md5=d96c95e03061bdcfffbe1e9e40f3c6ad</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.2174%2F0929867033457809&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F0929867033457809%26sid%3Dliteratum%253Aachs%26aulast%3DGlen%26aufirst%3DR.%2BC.%26aulast%3DAllen%26aufirst%3DS.%2BC.%26atitle%3DLigand%25E2%2588%2592protein%2520docking%253A%2520cancer%2520research%2520at%2520the%2520interface%2520between%2520biology%2520and%2520chemistry%26jtitle%3DCurr.%2520Med.%2520Chem.%26date%3D2003%26volume%3D10%26spage%3D763%26epage%3D777" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group">Walters, W. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Namchuk, M.</span><span> </span><span class="NLM_article-title">Designing screens: how to make your hits a hit</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">259</span><span class="NLM_x">–</span> <span class="NLM_lpage">266</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2003&pages=259-266&author=W.+P.+Waltersauthor=M.+Namchuk&title=Designing+screens%3A+how+to+make+your+hits+a+hit"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DWalters%26aufirst%3DW.%2BP.%26aulast%3DNamchuk%26aufirst%3DM.%26atitle%3DDesigning%2520screens%253A%2520how%2520to%2520make%2520your%2520hits%2520a%2520hit%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2003%26volume%3D2%26spage%3D259%26epage%3D266" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group">Alvarez, J. C.</span><span> </span><span class="NLM_article-title">High-throughput docking as a source of novel drug leads</span> <span class="citation_source-journal">Curr. Opin. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">365</span><span class="NLM_x">–</span> <span class="NLM_lpage">370</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Fjm070654j&amp;key=10.1016%2Fj.cbpa.2004.05.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Fjm070654j&amp;key=15288245" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Fjm070654j&amp;key=1%3ACAS%3A528%3ADC%252BD2cXmt1Omsrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2004&pages=365-370&author=J.+C.+Alvarez&title=High-throughput+docking+as+a+source+of+novel+drug+leads"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">High-throughput docking as a source of novel drug leads</span></div><div class="casAuthors">Alvarez, Juan C.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Chemical Biology</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">365-370</span>CODEN:
                <span class="NLM_cas:coden">COCBF4</span>;
        ISSN:<span class="NLM_cas:issn">1367-5931</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Receptor-based virtual screening has become a viable source of novel leads in the pharmaceutical industry.  The rapidly growing availability of structural information across protein families, the accessibility to increased computational power at affordable cost, as well as an improved understanding on how to effectively apply virtual screening technologies has contributed to their emergence.  Nonetheless, continued improvement in the accuracy of scoring functions and a greater understanding of protein mobility is crit. to advance the technol. further.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqkrrsrGzZ06LVg90H21EOLACvtfcHk0lh72UA3wGo-Bw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXmt1Omsrw%253D&md5=d828a47c7bc00bcc7dc6c39827db94b5</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1016%2Fj.cbpa.2004.05.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cbpa.2004.05.001%26sid%3Dliteratum%253Aachs%26aulast%3DAlvarez%26aufirst%3DJ.%2BC.%26atitle%3DHigh-throughput%2520docking%2520as%2520a%2520source%2520of%2520novel%2520drug%2520leads%26jtitle%3DCurr.%2520Opin.%2520Chem.%2520Biol.%26date%3D2004%26volume%3D8%26spage%3D365%26epage%3D370" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group">Jorgensen, W. L.</span><span> </span><span class="NLM_article-title">The many roles of computation in drug discovery</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">303</span><span class="NLM_x">, </span> <span class="NLM_fpage">1813</span><span class="NLM_x">–</span> <span class="NLM_lpage">1818</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Fjm070654j&amp;key=10.1126%2Fscience.1096361" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Fjm070654j&amp;key=15031495" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Fjm070654j&amp;key=1%3ACAS%3A528%3ADC%252BD2cXitFehu7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=303&publication_year=2004&pages=1813-1818&author=W.+L.+Jorgensen&title=The+many+roles+of+computation+in+drug+discovery"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">The Many Roles of Computation in Drug Discovery</span></div><div class="casAuthors">Jorgensen, William L.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">303</span>
        (<span class="NLM_cas:issue">5665</span>),
    <span class="NLM_cas:pages">1813-1818</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">A review.  An overview is given on the diverse uses of computational chem. in drug discovery.  Particular emphasis is placed on virtual screening, de novo design, evaluation of drug-likeness, and advanced methods for detg. protein-ligand binding.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrPr2xRlx_bxbVg90H21EOLACvtfcHk0lgn5JZa35om8A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXitFehu7w%253D&md5=77b2fac63e750ca7cdc7e4f35654549a</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1126%2Fscience.1096361&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1096361%26sid%3Dliteratum%253Aachs%26aulast%3DJorgensen%26aufirst%3DW.%2BL.%26atitle%3DThe%2520many%2520roles%2520of%2520computation%2520in%2520drug%2520discovery%26jtitle%3DScience%26date%3D2004%26volume%3D303%26spage%3D1813%26epage%3D1818" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">Virtual screening of chemical libraries</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">432</span><span class="NLM_x">, </span> <span class="NLM_fpage">862</span><span class="NLM_x">–</span> <span class="NLM_lpage">865</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Fjm070654j&amp;key=10.1038%2Fnature03197" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Fjm070654j&amp;key=15602552" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Fjm070654j&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVOht7jK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=432&publication_year=2004&pages=862-865&author=B.+K.+Shoichet&title=Virtual+screening+of+chemical+libraries"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Virtual screening of chemical libraries</span></div><div class="casAuthors">Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">432</span>
        (<span class="NLM_cas:issue">7019</span>),
    <span class="NLM_cas:pages">862-865</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Virtual screening uses computer-based methods to discover new ligands on the basis of biol. structures.  Although widely heralded in the 1970s and 1980s, the technique has since struggled to meet its initial promise, and drug discovery remains dominated by empirical screening.  Recent successes in predicting new ligands and their receptor-bound structures, and better rates of ligand discovery compared to empirical screening, have reignited interest in virtual screening, which is now widely used in drug discovery, albeit on a more limited scale than empirical screening.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrSY9IbbFldkrVg90H21EOLACvtfcHk0lgn5JZa35om8A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVOht7jK&md5=e351e226e8985d7c304a05fb67dfdc60</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1038%2Fnature03197&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature03197%26sid%3Dliteratum%253Aachs%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DVirtual%2520screening%2520of%2520chemical%2520libraries%26jtitle%3DNature%26date%3D2004%26volume%3D432%26spage%3D862%26epage%3D865" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group">Huang, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lüthi, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kolb, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Edler, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cecchini, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Audetat, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barberis, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Caflisch, A.</span><span> </span><span class="NLM_article-title">Discovery of cell-permeable non-peptide inhibitors of β-secretase by high-throughput docking and continuum electrostatics calculations</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">48</span><span class="NLM_x">, </span> <span class="NLM_fpage">5108</span><span class="NLM_x">–</span> <span class="NLM_lpage">5111</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm050499d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2005&pages=5108-5111&author=D.+Huangauthor=U.+L%C3%BCthiauthor=P.+Kolbauthor=K.+Edlerauthor=M.+Cecchiniauthor=S.+Audetatauthor=A.+Barberisauthor=A.+Caflisch&title=Discovery+of+cell-permeable+non-peptide+inhibitors+of+%CE%B2-secretase+by+high-throughput+docking+and+continuum+electrostatics+calculations"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1021%2Fjm050499d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm050499d%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DD.%26aulast%3DL%25C3%25BCthi%26aufirst%3DU.%26aulast%3DKolb%26aufirst%3DP.%26aulast%3DEdler%26aufirst%3DK.%26aulast%3DCecchini%26aufirst%3DM.%26aulast%3DAudetat%26aufirst%3DS.%26aulast%3DBarberis%26aufirst%3DA.%26aulast%3DCaflisch%26aufirst%3DA.%26atitle%3DDiscovery%2520of%2520cell-permeable%2520non-peptide%2520inhibitors%2520of%2520%25CE%25B2-secretase%2520by%2520high-throughput%2520docking%2520and%2520continuum%2520electrostatics%2520calculations%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2005%26volume%3D48%26spage%3D5108%26epage%3D5111" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group">Huang, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lüthi, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kolb, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cecchini, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barberis, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Caflisch, A.</span><span> </span><span class="NLM_article-title">In silico discovery of β-secretase inhibitors</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">128</span><span class="NLM_x">, </span> <span class="NLM_fpage">5436</span><span class="NLM_x">–</span> <span class="NLM_lpage">5443</span></span><div class="citationLinks">[<a href="/doi/10.1021/ja0573108" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=128&publication_year=2006&pages=5436-5443&author=D.+Huangauthor=U.+L%C3%BCthiauthor=P.+Kolbauthor=M.+Cecchiniauthor=A.+Barberisauthor=A.+Caflisch&title=In+silico+discovery+of+%CE%B2-secretase+inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1021%2Fja0573108&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja0573108%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DD.%26aulast%3DL%25C3%25BCthi%26aufirst%3DU.%26aulast%3DKolb%26aufirst%3DP.%26aulast%3DCecchini%26aufirst%3DM.%26aulast%3DBarberis%26aufirst%3DA.%26aulast%3DCaflisch%26aufirst%3DA.%26atitle%3DIn%2520silico%2520discovery%2520of%2520%25CE%25B2-secretase%2520inhibitors%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2006%26volume%3D128%26spage%3D5436%26epage%3D5443" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group">Wang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deng, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roux, B.</span><span> </span><span class="NLM_article-title">Absolute binding free energy calculations using molecular dynamics simulations with restraining potentials</span> <span class="citation_source-journal">Biophys. J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">91</span><span class="NLM_x">, </span> <span class="NLM_fpage">2798</span><span class="NLM_x">–</span> <span class="NLM_lpage">2814</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Fjm070654j&amp;key=10.1529%2Fbiophysj.106.084301" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Fjm070654j&amp;key=16844742" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Fjm070654j&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtVGls7vP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=91&publication_year=2006&pages=2798-2814&author=J.+Wangauthor=Y.+Dengauthor=B.+Roux&title=Absolute+binding+free+energy+calculations+using+molecular+dynamics+simulations+with+restraining+potentials"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Absolute binding free energy calculations using molecular dynamics simulations with restraining potentials</span></div><div class="casAuthors">Wang, Jiyao; Deng, Yuqing; Roux, Benoit</div><div class="citationInfo"><span class="NLM_cas:title">Biophysical Journal</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">91</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2798-2814</span>CODEN:
                <span class="NLM_cas:coden">BIOJAU</span>;
        ISSN:<span class="NLM_cas:issn">0006-3495</span>.
    
            (<span class="NLM_cas:orgname">Biophysical Society</span>)
        </div><div class="casAbstract">The abs. (std.) binding free energy of eight FK506-related ligands to FKBP12 is calcd. using free energy perturbation mol. dynamics (FEP/MD) simulations with explicit solvent.  A no. of features are implemented to improve the accuracy and enhance the convergence of the calcns.  First, the abs. binding free energy is decompd. into sequential steps during which the ligand-surrounding interactions as well as various biasing potentials restraining the translation, orientation, and conformation of the ligand are turned "on" and "off.".  Second, sampling of the ligand conformation is enforced by a restraining potential based on the root mean-square deviation relative to the bound state conformation.  The effect of all the restraining potentials is rigorously unbiased, and it is shown explicitly that the final results are independent of all artificial restraints.  Third, the repulsive and dispersive free energy contribution arising from the Lennard-Jones interactions of the ligand with its surrounding (protein and solvent) is calcd. using the Weeks-Chandler-Andersen sepn.  This sepn. also improves convergence of the FEP/MD calcns.  Fourth, to decrease the computational cost, only a small no. of atoms in the vicinity of the binding site are simulated explicitly, while all the influence of the remaining atoms is incorporated implicitly using the generalized solvent boundary potential (GSBP) method.  With GSBP, the size of the simulated FKBP12/ligand systems is significantly reduced, from ∼25,000 to 2500.  The computations are very efficient and the statistical error is small (∼1 kcal/mol).  The calcd. binding free energies are generally in good agreement with available exptl. data and previous calcns. (within ∼2 kcal/mol).  The present results indicate that a strategy based on FEP/MD simulations of a reduced GSBP at. model sampled with conformational, translational, and orientational restraining potentials can be computationally inexpensive and accurate.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqsDftyP7DXobVg90H21EOLACvtfcHk0lgn5JZa35om8A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtVGls7vP&md5=010f74f61048d03fee5dce41172dac70</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1529%2Fbiophysj.106.084301&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1529%252Fbiophysj.106.084301%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DDeng%26aufirst%3DY.%26aulast%3DRoux%26aufirst%3DB.%26atitle%3DAbsolute%2520binding%2520free%2520energy%2520calculations%2520using%2520molecular%2520dynamics%2520simulations%2520with%2520restraining%2520potentials%26jtitle%3DBiophys.%2520J.%26date%3D2006%26volume%3D91%26spage%3D2798%26epage%3D2814" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group">Åqvist, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Medina, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Samuelsson, J.-E.</span><span> </span><span class="NLM_article-title">A new method for predicting binding affinity in computer-aided drug design</span> <span class="citation_source-journal">Protein Eng.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1994</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">385</span><span class="NLM_x">–</span> <span class="NLM_lpage">391</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Fjm070654j&amp;key=10.1093%2Fprotein%2F7.3.385" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Fjm070654j&amp;key=8177887" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Fjm070654j&amp;key=1%3ACAS%3A280%3ADyaK2c3jtlSnsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=1994&pages=385-391&author=J.+%C3%85qvistauthor=C.+Medinaauthor=J.-E.+Samuelsson&title=A+new+method+for+predicting+binding+affinity+in+computer-aided+drug+design"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">A new method for predicting binding affinity in computer-aided drug design</span></div><div class="casAuthors">Aqvist J; Medina C; Samuelsson J E</div><div class="citationInfo"><span class="NLM_cas:title">Protein engineering</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">385-91</span>
        ISSN:<span class="NLM_cas:issn">0269-2139</span>.
    </div><div class="casAbstract">A new semi-empirical method for calculating free energies of binding from molecular dynamics (MD) simulations is presented.  It is based on standard thermodynamic cycles and on a linear approximation of polar and non-polar free energy contributions from the corresponding MD averages.  The method is tested on a set of endothiapepsin inhibitors and found to give accurate results both for absolute as well as relative free energies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTEx3uc8suH_BOk4JA3soTqfW6udTcc2eadDG6u5H0PtLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADyaK2c3jtlSnsw%253D%253D&md5=c45572201328309d4fc849844d1c3b71</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1093%2Fprotein%2F7.3.385&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fprotein%252F7.3.385%26sid%3Dliteratum%253Aachs%26aulast%3D%25C3%2585qvist%26aufirst%3DJ.%26aulast%3DMedina%26aufirst%3DC.%26aulast%3DSamuelsson%26aufirst%3DJ.-E.%26atitle%3DA%2520new%2520method%2520for%2520predicting%2520binding%2520affinity%2520in%2520computer-aided%2520drug%2520design%26jtitle%3DProtein%2520Eng.%26date%3D1994%26volume%3D7%26spage%3D385%26epage%3D391" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group">Hansson, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Åqvist, J.</span><span> </span><span class="NLM_article-title">Estimation of binding free energies for HIV proteinase inhibitors by molecular dynamics simulations.</span> <span class="citation_source-journal">Protein Eng.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1995</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">1137</span><span class="NLM_x">–</span> <span class="NLM_lpage">1144</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Fjm070654j&amp;key=10.1093%2Fprotein%2F8.11.1137" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Fjm070654j&amp;key=8819979" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Fjm070654j&amp;key=1%3ACAS%3A528%3ADyaK28XhsFymtLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=1995&pages=1137-1144&author=T.+Hanssonauthor=J.+%C3%85qvist&title=Estimation+of+binding+free+energies+for+HIV+proteinase+inhibitors+by+molecular+dynamics+simulations."target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Estimation of binding free energies for HIV proteinase inhibitors by molecular dynamics simulations</span></div><div class="casAuthors">Hansson, Tomas; Aaqvist, Johan</div><div class="citationInfo"><span class="NLM_cas:title">Protein Engineering</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1137-44</span>CODEN:
                <span class="NLM_cas:coden">PRENE9</span>;
        ISSN:<span class="NLM_cas:issn">0269-2139</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Abs. binding free energies for inhibitors of HIV-1 proteinase were estd. from mol. dynamics simulations by a recently reported linear approxn. procedure.  The results were in fairly good agreement with exptl. binding data.  Two of the inhibitors were very similar and, for comparison, their relative free energies of binding were also calcd. by free energy perturbation methods, giving virtually the same result.  Effects of cutoff radii and charge states of the protein model were examd.  The effects of pH on binding of one of the inhibitors were predicted.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrcv6yv6JgB1rVg90H21EOLACvtfcHk0liAx9yVuZCzyw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28XhsFymtLY%253D&md5=c77820746da2a692e7543f5f79789cca</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1093%2Fprotein%2F8.11.1137&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fprotein%252F8.11.1137%26sid%3Dliteratum%253Aachs%26aulast%3DHansson%26aufirst%3DT.%26aulast%3D%25C3%2585qvist%26aufirst%3DJ.%26atitle%3DEstimation%2520of%2520binding%2520free%2520energies%2520for%2520HIV%2520proteinase%2520inhibitors%2520by%2520molecular%2520dynamics%2520simulations.%26jtitle%3DProtein%2520Eng.%26date%3D1995%26volume%3D8%26spage%3D1137%26epage%3D1144" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group">Jones-Hertzog, D. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jorgensen, W. L.</span><span> </span><span class="NLM_article-title">Binding affinities for sulfonamide inhibitors with human thrombin using Monte Carlo simulations with a linear response method</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1996</span><span class="NLM_x">, </span> <span class="NLM_volume">40</span><span class="NLM_x">, </span> <span class="NLM_fpage">1539</span><span class="NLM_x">–</span> <span class="NLM_lpage">1549</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=1996&pages=1539-1549&author=D.+K.+Jones-Hertzogauthor=W.+L.+Jorgensen&title=Binding+affinities+for+sulfonamide+inhibitors+with+human+thrombin+using+Monte+Carlo+simulations+with+a+linear+response+method"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DJones-Hertzog%26aufirst%3DD.%2BK.%26aulast%3DJorgensen%26aufirst%3DW.%2BL.%26atitle%3DBinding%2520affinities%2520for%2520sulfonamide%2520inhibitors%2520with%2520human%2520thrombin%2520using%2520Monte%2520Carlo%2520simulations%2520with%2520a%2520linear%2520response%2520method%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1996%26volume%3D40%26spage%3D1539%26epage%3D1549" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group">Hansson, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marelius, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Åqvist, J.</span><span> </span><span class="NLM_article-title">Ligand binding affinity prediction by linear interaction energy methods</span> <span class="citation_source-journal">J. Comput.-Aided Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">12</span><span class="NLM_x">, </span> <span class="NLM_fpage">27</span><span class="NLM_x">–</span> <span class="NLM_lpage">35</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Fjm070654j&amp;key=10.1023%2FA%3A1007930623000" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Fjm070654j&amp;key=9570087" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Fjm070654j&amp;key=1%3ACAS%3A528%3ADyaK1cXisl2qsbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=1998&pages=27-35&author=T.+Hanssonauthor=J.+Mareliusauthor=J.+%C3%85qvist&title=Ligand+binding+affinity+prediction+by+linear+interaction+energy+methods"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Ligand binding affinity prediction by linear interaction energy methods</span></div><div class="casAuthors">Hansson, Tomas; Marelius, John; Aqvist, Johan</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">27-35</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Kluwer Academic Publishers</span>)
        </div><div class="casAbstract">A recent method for estg. ligand binding affinities is extended.  This method employs avs. of interaction potential energy terms from mol. dynamics simulations or other thermal conformational sampling techniques.  Incorporation of systematic deviations from electrostatic linear response, derived from free energy perturbation studies, into the abs. binding free energy expression significantly enhances the accuracy of the approach.  This type of method may be useful for computational prediction of ligand binding strengths, e.g., in drug design applications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpU4E4PGZrXWbVg90H21EOLACvtfcHk0liAx9yVuZCzyw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXisl2qsbc%253D&md5=c66173a2b01b3283be2700c370d0ddea</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1023%2FA%3A1007930623000&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1023%252FA%253A1007930623000%26sid%3Dliteratum%253Aachs%26aulast%3DHansson%26aufirst%3DT.%26aulast%3DMarelius%26aufirst%3DJ.%26aulast%3D%25C3%2585qvist%26aufirst%3DJ.%26atitle%3DLigand%2520binding%2520affinity%2520prediction%2520by%2520linear%2520interaction%2520energy%2520methods%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D1998%26volume%3D12%26spage%3D27%26epage%3D35" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group">Wang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dixon, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kollman, P.</span><span> </span><span class="NLM_article-title">Ranking ligand binding affinities with avidin: a molecular dynamics-based interaction energy study</span> <span class="citation_source-journal">Proteins: Struct., Funct., Bioinf.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1999</span><span class="NLM_x">, </span> <span class="NLM_volume">34</span><span class="NLM_x">, </span> <span class="NLM_fpage">69</span><span class="NLM_x">–</span> <span class="NLM_lpage">81</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=1999&pages=69-81&author=J.+Wangauthor=R.+Dixonauthor=P.+Kollman&title=Ranking+ligand+binding+affinities+with+avidin%3A+a+molecular+dynamics-based+interaction+energy+study"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DDixon%26aufirst%3DR.%26aulast%3DKollman%26aufirst%3DP.%26atitle%3DRanking%2520ligand%2520binding%2520affinities%2520with%2520avidin%253A%2520a%2520molecular%2520dynamics-based%2520interaction%2520energy%2520study%26jtitle%3DProteins%253A%2520Struct.%252C%2520Funct.%252C%2520Bioinf.%26date%3D1999%26volume%3D34%26spage%3D69%26epage%3D81" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group">Carlson, H. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jorgensen, W. L.</span><span> </span><span class="NLM_article-title">An extended linear response method for determining free energies of hydration</span> <span class="citation_source-journal">J. Phys. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1995</span><span class="NLM_x">, </span> <span class="NLM_volume">99</span><span class="NLM_x">, </span> <span class="NLM_fpage">10667</span><span class="NLM_x">–</span> <span class="NLM_lpage">10673</span></span><div class="citationLinks">[<a href="/doi/10.1021/j100026a034" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Fjm070654j&amp;key=1%3ACAS%3A528%3ADyaK2MXmtVyiu7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=99&publication_year=1995&pages=10667-10673&author=H.+A.+Carlsonauthor=W.+L.+Jorgensen&title=An+extended+linear+response+method+for+determining+free+energies+of+hydration"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">An Extended Linear Response Method for Determining Free Energies of Hydration</span></div><div class="casAuthors">Carlson, Heather A.; Jorgensen, William L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Physical Chemistry</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">99</span>
        (<span class="NLM_cas:issue">26</span>),
    <span class="NLM_cas:pages">10667-73</span>CODEN:
                <span class="NLM_cas:coden">JPCHAX</span>;
        ISSN:<span class="NLM_cas:issn">0022-3654</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Fourteen neutral org. mols. with diverse functionalities were used to det. the utility of equations based on linear response theory for the calcn. of free energies of hydration.  The equations contain terms proportional to the Coulombic and van der Waals components of the solute-solvent interaction energy and to an index for the cost of cavity formation such as solvent-accessible surface area.  The energy components can be obtained from Monte Carlo or mol. dynamics simulations of the solutes in water.  The methodol. was tested by using partial charges from fitting to the electrostatic potentials of ab initio 6-31G* wave functions and from OPLS potential functions.  Root-mean-square deviations between the results from the recommended linear response methods and exptl. free energies of hydration are ca. 0.8 kcal/mol.  Substituted benzenes, not included in the parameter development, were used to further evaluate the method performance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoNcOGUicelNrVg90H21EOLACvtfcHk0lgZBmZuijGZNA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXmtVyiu7Y%253D&md5=b91a7f287b63063c81c51082f694a78b</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1021%2Fj100026a034&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fj100026a034%26sid%3Dliteratum%253Aachs%26aulast%3DCarlson%26aufirst%3DH.%2BA.%26aulast%3DJorgensen%26aufirst%3DW.%2BL.%26atitle%3DAn%2520extended%2520linear%2520response%2520method%2520for%2520determining%2520free%2520energies%2520of%2520hydration%26jtitle%3DJ.%2520Phys.%2520Chem.%26date%3D1995%26volume%3D99%26spage%3D10667%26epage%3D10673" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group">Wall, I. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leach, A. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Salt, D. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ford, M. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Essex, J. W.</span><span> </span><span class="NLM_article-title">Binding constants of neuraminidase inhibitors: an investigation of the linear interaction energy method</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1999</span><span class="NLM_x">, </span> <span class="NLM_volume">42</span><span class="NLM_x">, </span> <span class="NLM_fpage">5142</span><span class="NLM_x">–</span> <span class="NLM_lpage">5152</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm990105g" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Fjm070654j&amp;key=1%3ACAS%3A528%3ADyaK1MXnsFOlu70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=1999&pages=5142-5152&author=I.+D.+Wallauthor=A.+R.+Leachauthor=D.+W.+Saltauthor=M.+G.+Fordauthor=J.+W.+Essex&title=Binding+constants+of+neuraminidase+inhibitors%3A+an+investigation+of+the+linear+interaction+energy+method"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Binding Constants of Neuraminidase Inhibitors: An Investigation of the Linear Interaction Energy Method</span></div><div class="casAuthors">Wall, Ian D.; Leach, Andrew R.; Salt, David W.; Ford, Martyn G.; Essex, Jonathan W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">25</span>),
    <span class="NLM_cas:pages">5142-5152</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The linear interaction energy (LIE) method has been applied to the calcn. of the binding free energies of 15 inhibitors of the enzyme neuraminidase.  This is a particularly challenging system for this methodol. since the protein conformation and the no. of tightly bound water mols. in the active site are known to change for different inhibitors.  It is not clear that the basic LIE method will calc. the contributions to the binding free energies arising from these effects correctly.  Application of the basic LIE equation yielded an rms error with respect to expt. of 1.51 kcal mol-1 for the free energies of binding.  To det. whether it is appropriate to include extra terms in the LIE equation, a detailed statistical anal. was undertaken.  Multiple linear regression (MLR) is often used to det. the significance of terms in a fitting equation; this method is inappropriate for the current system owing to the highly correlated nature of the descriptor variables.  Use of MLR in other applications of the LIE equation is therefore not recommended without a correlation anal. being performed first.  Here factor anal. was used to det. the no. of useful dimensions contained within the data and, hence, the max. no. of variables to be considered when specifying a model or equation.  Biased fitting methods using orthogonalized components were then used to generate the most predictive model.  The final model gave a q2 of 0.74 and contained van der Waals and electrostatic energy terms.  This result was obtained without recourse to prior knowledge and was based solely on the information content of the data.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr487StWx1P2rVg90H21EOLACvtfcHk0lgZBmZuijGZNA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXnsFOlu70%253D&md5=466f8bc796cc64e1c3979318e9313daa</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1021%2Fjm990105g&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm990105g%26sid%3Dliteratum%253Aachs%26aulast%3DWall%26aufirst%3DI.%2BD.%26aulast%3DLeach%26aufirst%3DA.%2BR.%26aulast%3DSalt%26aufirst%3DD.%2BW.%26aulast%3DFord%26aufirst%3DM.%2BG.%26aulast%3DEssex%26aufirst%3DJ.%2BW.%26atitle%3DBinding%2520constants%2520of%2520neuraminidase%2520inhibitors%253A%2520an%2520investigation%2520of%2520the%2520linear%2520interaction%2520energy%2520method%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1999%26volume%3D42%26spage%3D5142%26epage%3D5152" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group">Zhou, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Friesner, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ghoshs, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rizzo, R. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jorgensen, W. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Levy, R. M.</span><span> </span><span class="NLM_article-title">New linear interaction method for binding affinity calculations using a continuum solvent model</span> <span class="citation_source-journal">J. Phys. Chem. B</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">102</span><span class="NLM_x">, </span> <span class="NLM_fpage">10388</span><span class="NLM_x">–</span> <span class="NLM_lpage">10397</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=102&publication_year=2001&pages=10388-10397&author=R.+Zhouauthor=R.+A.+Friesnerauthor=A.+Ghoshsauthor=R.+C.+Rizzoauthor=W.+J.+Jorgensenauthor=R.+M.+Levy&title=New+linear+interaction+method+for+binding+affinity+calculations+using+a+continuum+solvent+model"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DZhou%26aufirst%3DR.%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26aulast%3DGhoshs%26aufirst%3DA.%26aulast%3DRizzo%26aufirst%3DR.%2BC.%26aulast%3DJorgensen%26aufirst%3DW.%2BJ.%26aulast%3DLevy%26aufirst%3DR.%2BM.%26atitle%3DNew%2520linear%2520interaction%2520method%2520for%2520binding%2520affinity%2520calculations%2520using%2520a%2520continuum%2520solvent%2520model%26jtitle%3DJ.%2520Phys.%2520Chem.%2520B%26date%3D2001%26volume%3D102%26spage%3D10388%26epage%3D10397" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group">Tounge, B. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reynolds, C. H.</span><span> </span><span class="NLM_article-title">Calculation of the binding affinity of β-secretase inhibitors using the linear interaction energy method</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">46</span><span class="NLM_x">, </span> <span class="NLM_fpage">2074</span><span class="NLM_x">–</span> <span class="NLM_lpage">2082</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm020513b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2003&pages=2074-2082&author=B.+A.+Toungeauthor=C.+H.+Reynolds&title=Calculation+of+the+binding+affinity+of+%CE%B2-secretase+inhibitors+using+the+linear+interaction+energy+method"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Fjm020513b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm020513b%26sid%3Dliteratum%253Aachs%26aulast%3DTounge%26aufirst%3DB.%2BA.%26aulast%3DReynolds%26aufirst%3DC.%2BH.%26atitle%3DCalculation%2520of%2520the%2520binding%2520affinity%2520of%2520%25CE%25B2-secretase%2520inhibitors%2520using%2520the%2520linear%2520interaction%2520energy%2520method%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2003%26volume%3D46%26spage%3D2074%26epage%3D2082" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group">Tominaga, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jorgensen, W. L.</span><span> </span><span class="NLM_article-title">General model for estimation of the inhibition of protein kinases using Monte Carlo simulations</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">2534</span><span class="NLM_x">–</span> <span class="NLM_lpage">2549</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm0304358" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=2534-2549&author=Y.+Tominagaauthor=W.+L.+Jorgensen&title=General+model+for+estimation+of+the+inhibition+of+protein+kinases+using+Monte+Carlo+simulations"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Fjm0304358&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0304358%26sid%3Dliteratum%253Aachs%26aulast%3DTominaga%26aufirst%3DY.%26aulast%3DJorgensen%26aufirst%3DW.%2BL.%26atitle%3DGeneral%2520model%2520for%2520estimation%2520of%2520the%2520inhibition%2520of%2520protein%2520kinases%2520using%2520Monte%2520Carlo%2520simulations%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D2534%26epage%3D2549" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group">Huang, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Caflisch, A.</span><span> </span><span class="NLM_article-title">Efficient evaluation of binding free energy using continuum electrostatic solvation</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">5791</span><span class="NLM_x">–</span> <span class="NLM_lpage">5797</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm049726m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=5791-5797&author=D.+Huangauthor=A.+Caflisch&title=Efficient+evaluation+of+binding+free+energy+using+continuum+electrostatic+solvation"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1021%2Fjm049726m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm049726m%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DD.%26aulast%3DCaflisch%26aufirst%3DA.%26atitle%3DEfficient%2520evaluation%2520of%2520binding%2520free%2520energy%2520using%2520continuum%2520electrostatic%2520solvation%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D5791%26epage%3D5797" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group">Stjernschantz, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marelius, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Medina, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jacobsson, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vermeulen, N. P. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oostenbrink, C.</span><span> </span><span class="NLM_article-title">Are automated molecular dynamics simulations and binding free energy calculations realistic tools in lead optimization? An evaluation of the linear interaction energy (LIE) method</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">46</span><span class="NLM_x">, </span> <span class="NLM_fpage">1972</span><span class="NLM_x">–</span> <span class="NLM_lpage">1983</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci0601214" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Fjm070654j&amp;key=1%3ACAS%3A528%3ADC%252BD28XnsV2hs7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2006&pages=1972-1983&author=E.+Stjernschantzauthor=J.+Mareliusauthor=C.+Medinaauthor=M.+Jacobssonauthor=N.+P.+E.+Vermeulenauthor=C.+Oostenbrink&title=Are+automated+molecular+dynamics+simulations+and+binding+free+energy+calculations+realistic+tools+in+lead+optimization%3F+An+evaluation+of+the+linear+interaction+energy+%28LIE%29+method"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Are Automated Molecular Dynamics Simulations and Binding Free Energy Calculations Realistic Tools in Lead Optimization? An Evaluation of the Linear Interaction Energy (LIE) Method</span></div><div class="casAuthors">Stjernschantz, Eva; Marelius, John; Medina, Carmen; Jacobsson, Micael; Vermeulen, Nico P. E.; Oostenbrink, Chris</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1972-1983</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">An extensive evaluation of the linear interaction energy (LIE) method for the prediction of binding affinity of docked compds. has been performed, with an emphasis on its applicability in lead optimization.  An automated setup is presented, which allows for the use of the method in an industrial setting.  Calcns. are performed for four realistic examples, retinoic acid receptor γ, matrix metalloprotease 3, estrogen receptor α, and dihydrofolate reductase, focusing on different aspects of the procedure.  The obtained LIE models are evaluated in terms of the root-mean-square (RMS) errors from exptl. binding free energies and the ability to rank compds. appropriately.  The results are compared to the best empirical scoring function, selected from a set of 10 scoring functions.  In all cases, good LIE models can be obtained in terms of free-energy RMS errors, although reasonable ranking of the ligands of dihydrofolate reductase proves difficult for both the LIE method and scoring functions.  For the other proteins, the LIE model results in better predictions than the best performing scoring function.  These results indicate that the LIE approach, as a tool to evaluate docking results, can be a valuable asset in computational lead optimization programs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpzvbHPR51x6bVg90H21EOLACvtfcHk0lggSVYy5cJ2tQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XnsV2hs7w%253D&md5=c63a2c9e40c00b85c8c29f04fe996f79</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1021%2Fci0601214&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci0601214%26sid%3Dliteratum%253Aachs%26aulast%3DStjernschantz%26aufirst%3DE.%26aulast%3DMarelius%26aufirst%3DJ.%26aulast%3DMedina%26aufirst%3DC.%26aulast%3DJacobsson%26aufirst%3DM.%26aulast%3DVermeulen%26aufirst%3DN.%2BP.%2BE.%26aulast%3DOostenbrink%26aufirst%3DC.%26atitle%3DAre%2520automated%2520molecular%2520dynamics%2520simulations%2520and%2520binding%2520free%2520energy%2520calculations%2520realistic%2520tools%2520in%2520lead%2520optimization%253F%2520An%2520evaluation%2520of%2520the%2520linear%2520interaction%2520energy%2520%2528LIE%2529%2520method%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2006%26volume%3D46%26spage%3D1972%26epage%3D1983" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group">Ersmark, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nervall, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hamelink, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Janka, L. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clemente, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dunn, B. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blackman, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Samuelsson, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Åqvist, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hallberg, A.</span><span> </span><span class="NLM_article-title">Synthesis of malarial plasmepsin inhibitors and prediction of binding modes by molecular dynamics simulations</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">48</span><span class="NLM_x">, </span> <span class="NLM_fpage">6090</span><span class="NLM_x">–</span> <span class="NLM_lpage">6106</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm050463l" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2005&pages=6090-6106&author=K.+Ersmarkauthor=M.+Nervallauthor=E.+Hamelinkauthor=L.+K.+Jankaauthor=J.+C.+Clementeauthor=B.+M.+Dunnauthor=M.+J.+Blackmanauthor=B.+Samuelssonauthor=J.+%C3%85qvistauthor=A.+Hallberg&title=Synthesis+of+malarial+plasmepsin+inhibitors+and+prediction+of+binding+modes+by+molecular+dynamics+simulations"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1021%2Fjm050463l&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm050463l%26sid%3Dliteratum%253Aachs%26aulast%3DErsmark%26aufirst%3DK.%26aulast%3DNervall%26aufirst%3DM.%26aulast%3DHamelink%26aufirst%3DE.%26aulast%3DJanka%26aufirst%3DL.%2BK.%26aulast%3DClemente%26aufirst%3DJ.%2BC.%26aulast%3DDunn%26aufirst%3DB.%2BM.%26aulast%3DBlackman%26aufirst%3DM.%2BJ.%26aulast%3DSamuelsson%26aufirst%3DB.%26aulast%3D%25C3%2585qvist%26aufirst%3DJ.%26aulast%3DHallberg%26aufirst%3DA.%26atitle%3DSynthesis%2520of%2520malarial%2520plasmepsin%2520inhibitors%2520and%2520prediction%2520of%2520binding%2520modes%2520by%2520molecular%2520dynamics%2520simulations%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2005%26volume%3D48%26spage%3D6090%26epage%3D6106" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group">Warwicker, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Watson, H. C.</span><span> </span><span class="NLM_article-title">Calculation of the electric potential in the active site cleft due to α-helix dipoles</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1982</span><span class="NLM_x">, </span> <span class="NLM_volume">157</span><span class="NLM_x">, </span> <span class="NLM_fpage">671</span><span class="NLM_x">–</span> <span class="NLM_lpage">679</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Fjm070654j&amp;key=10.1016%2F0022-2836%2882%2990505-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Fjm070654j&amp;key=6288964" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Fjm070654j&amp;key=1%3ACAS%3A280%3ADyaL3s%252FhtVartA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=157&publication_year=1982&pages=671-679&author=J.+Warwickerauthor=H.+C.+Watson&title=Calculation+of+the+electric+potential+in+the+active+site+cleft+due+to+%CE%B1-helix+dipoles"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Calculation of the electric potential in the active site cleft due to alpha-helix dipoles</span></div><div class="casAuthors">Warwicker J; Watson H C</div><div class="citationInfo"><span class="NLM_cas:title">Journal of molecular biology</span>
        (<span class="NLM_cas:date">1982</span>),
    <span class="NLM_cas:volume">157</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">671-9</span>
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRKK3_KJDDB2pggzHGM8X5CfW6udTcc2ebNotmZ2XOKyLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADyaL3s%252FhtVartA%253D%253D&md5=4b779b03b1ce72f1a0e062ae9d074178</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1016%2F0022-2836%2882%2990505-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0022-2836%252882%252990505-8%26sid%3Dliteratum%253Aachs%26aulast%3DWarwicker%26aufirst%3DJ.%26aulast%3DWatson%26aufirst%3DH.%2BC.%26atitle%3DCalculation%2520of%2520the%2520electric%2520potential%2520in%2520the%2520active%2520site%2520cleft%2520due%2520to%2520%25CE%25B1-helix%2520dipoles%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D1982%26volume%3D157%26spage%3D671%26epage%3D679" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group">Dancey, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sausville, E. A.</span><span> </span><span class="NLM_article-title">Issues and progress with protein kinase inhibitors for cancer treatment</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">296</span><span class="NLM_x">–</span> <span class="NLM_lpage">313</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Fjm070654j&amp;key=10.1038%2Fnrd1066" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Fjm070654j&amp;key=12669029" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Fjm070654j&amp;key=1%3ACAS%3A528%3ADC%252BD3sXisFWkt70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2003&pages=296-313&author=J.+Danceyauthor=E.+A.+Sausville&title=Issues+and+progress+with+protein+kinase+inhibitors+for+cancer+treatment"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Issues and progress with protein kinase inhibitors for cancer treatment</span></div><div class="casAuthors">Dancey, Janet; Sausville, Edward A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">296-313</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Identification of the key roles of protein kinases in cancer has led to extensive efforts to develop kinase inhibitors for the treatment of a wide range of cancers, and more than 30 such agents are now in clin. trials.  Here, we consider the crucial issues in the development of kinase inhibitors for cancer, and discuss strategies to address the challenges raised by these issues in the light of preclin. and clin. experiences so far.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpH-u-kzpegLrVg90H21EOLACvtfcHk0ljOCZ4SxerF2A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXisFWkt70%253D&md5=fb11803282bcc39b24fa238a6ab973c8</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1038%2Fnrd1066&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd1066%26sid%3Dliteratum%253Aachs%26aulast%3DDancey%26aufirst%3DJ.%26aulast%3DSausville%26aufirst%3DE.%2BA.%26atitle%3DIssues%2520and%2520progress%2520with%2520protein%2520kinase%2520inhibitors%2520for%2520cancer%2520treatment%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2003%26volume%3D2%26spage%3D296%26epage%3D313" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group">Noble, M. E. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Endicott, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johnson, L. N.</span><span> </span><span class="NLM_article-title">Protein kinase inhibitors: insights into drug design from structure</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">303</span><span class="NLM_x">, </span> <span class="NLM_fpage">1800</span><span class="NLM_x">–</span> <span class="NLM_lpage">1805</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Fjm070654j&amp;key=10.1126%2Fscience.1095920" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Fjm070654j&amp;key=15031492" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Fjm070654j&amp;key=1%3ACAS%3A528%3ADC%252BD2cXitFehurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=303&publication_year=2004&pages=1800-1805&author=M.+E.+M.+Nobleauthor=J.+A.+Endicottauthor=L.+N.+Johnson&title=Protein+kinase+inhibitors%3A+insights+into+drug+design+from+structure"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Protein Kinase Inhibitors: Insights into Drug Design from Structure</span></div><div class="casAuthors">Noble, Martin E. M.; Endicott, Jane A.; Johnson, Louise N.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">303</span>
        (<span class="NLM_cas:issue">5665</span>),
    <span class="NLM_cas:pages">1800-1805</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">A review.  Protein kinases are targets for treatment of a no. of diseases.  This review focuses on kinase inhibitors that are in the clinic or in clin. trials and for which structural information is available.  Structures have informed drug design and have illuminated the mechanism of inhibition.  We review progress with the receptor tyrosine kinases (growth factor receptors EGFR, VEGFR, and FGFR) and nonreceptor tyrosine kinases (Bcr-Abl), where advances have been made with cancer therapeutic agents such as Herceptin and Gleevec.  Among the serine-threonine kinases, p38, Rho-kinase, cyclin-dependent kinases, and Chk1 have been targeted with productive results for inflammation and cancer.  Structures have provided insights into targeting the inactive or active form of the kinase, for targeting the global constellation of residues at the ATP site or less conserved addnl. pockets or single residues, and into targeting noncatalytic domains.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWqtKz7cvN_rVg90H21EOLACvtfcHk0ljOCZ4SxerF2A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXitFehurc%253D&md5=7fd3f60bee46e59307436eeed285d6ae</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1126%2Fscience.1095920&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1095920%26sid%3Dliteratum%253Aachs%26aulast%3DNoble%26aufirst%3DM.%2BE.%2BM.%26aulast%3DEndicott%26aufirst%3DJ.%2BA.%26aulast%3DJohnson%26aufirst%3DL.%2BN.%26atitle%3DProtein%2520kinase%2520inhibitors%253A%2520insights%2520into%2520drug%2520design%2520from%2520structure%26jtitle%3DScience%26date%3D2004%26volume%3D303%26spage%3D1800%26epage%3D1805" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group">Smith, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shindyalov, I. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Veretnik, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gribskov, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taylor, S. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ten- Eyck, L. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bourne, P. E.</span><span> </span><span class="NLM_article-title">The protein kinase resource</span> <span class="citation_source-journal">Trends Biochem. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">22</span><span class="NLM_x">, </span> <span class="NLM_fpage">444</span><span class="NLM_x">–</span> <span class="NLM_lpage">446</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=1997&pages=444-446&author=C.+M.+Smithauthor=I.+N.+Shindyalovauthor=S.+Veretnikauthor=M.+Gribskovauthor=S.+S.+Taylorauthor=L.+F.+Ten-+Eyckauthor=P.+E.+Bourne&title=The+protein+kinase+resource"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSmith%26aufirst%3DC.%2BM.%26aulast%3DShindyalov%26aufirst%3DI.%2BN.%26aulast%3DVeretnik%26aufirst%3DS.%26aulast%3DGribskov%26aufirst%3DM.%26aulast%3DTaylor%26aufirst%3DS.%2BS.%26aulast%3DTen-%2BEyck%26aufirst%3DL.%2BF.%26aulast%3DBourne%26aufirst%3DP.%2BE.%26atitle%3DThe%2520protein%2520kinase%2520resource%26jtitle%3DTrends%2520Biochem.%2520Sci.%26date%3D1997%26volume%3D22%26spage%3D444%26epage%3D446" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group">Cohen, P.</span><span> </span><span class="NLM_article-title">Protein kinases—the major drug targets of the twenty-first century?</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">1</span><span class="NLM_x">, </span> <span class="NLM_fpage">309</span><span class="NLM_x">–</span> <span class="NLM_lpage">315</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2002&pages=309-315&author=P.+Cohen&title=Protein+kinases%E2%80%94the+major+drug+targets+of+the+twenty-first+century%3F"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DCohen%26aufirst%3DP.%26atitle%3DProtein%2520kinases%25E2%2580%2594the%2520major%2520drug%2520targets%2520of%2520the%2520twenty-first%2520century%253F%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2002%26volume%3D1%26spage%3D309%26epage%3D315" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group">Aparna, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rambabu, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Panigrahi, S. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sarma, J. A. R. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Desiraju, G. R.</span><span> </span><span class="NLM_article-title">Virtual screening of 4-anilinoquinazoline analogues as EGFR kinase inhibitors: importance of hydrogen bonds in the evaluation of poses and scoring functions</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">45</span><span class="NLM_x">, </span> <span class="NLM_fpage">725</span><span class="NLM_x">–</span> <span class="NLM_lpage">738</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci049676u" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Fjm070654j&amp;key=1%3ACAS%3A528%3ADC%252BD2MXislWqsbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2005&pages=725-738&author=V.+Aparnaauthor=G.+Rambabuauthor=S.+K.+Panigrahiauthor=J.+A.+R.+P.+Sarmaauthor=G.+R.+Desiraju&title=Virtual+screening+of+4-anilinoquinazoline+analogues+as+EGFR+kinase+inhibitors%3A+importance+of+hydrogen+bonds+in+the+evaluation+of+poses+and+scoring+functions"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Virtual Screening of 4-Anilinoquinazoline Analogues as EGFR Kinase Inhibitors: Importance of Hydrogen Bonds in the Evaluation of Poses and Scoring Functions</span></div><div class="casAuthors">Aparna, V.; Rambabu, G.; Panigrahi, S. K.; Sarma, J. A. R. P.; Desiraju, G. R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">725-738</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Virtual Screening (VS) is a computational technique that allows selection and ranking of possible hits from a library of compds.  We have carried out VS on 128 selected EGFR kinase inhibitors with GOLD and LigandFit.  From the exptl. crystal structure of the erlotinib-EGFR complex, three key hydrogen bonds were identified as responsible for anchoring the ligand in the active site.  These are of the N-H···N, Ow-H···N, and C-H···O types.  Failure to include the hydrogen-bonded water mol. that forms the Ow-H···N bond leads to incorrect results.  Of the three interactions, the C-H···O formed by an activated C-H group is the best conserved.  On the basis of the efficacy of these hydrogen bonds, the poses were classified into one of three categories: close, shifted, and misoriented.  In the VS context, all three interactions need to be modeled correctly so that correct poses and affinities are obtained, and this happens in ligands of the close variety.  Cross scoring wherein the poses from one software are input into another for scoring and consensus scoring wherein the scores from various software packages are weighted are also helpful in obtaining better agreements.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp8i6TSN7WStrVg90H21EOLACvtfcHk0ljOCZ4SxerF2A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXislWqsbw%253D&md5=1dc1d041a37424b889937d697337a3d7</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Fci049676u&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci049676u%26sid%3Dliteratum%253Aachs%26aulast%3DAparna%26aufirst%3DV.%26aulast%3DRambabu%26aufirst%3DG.%26aulast%3DPanigrahi%26aufirst%3DS.%2BK.%26aulast%3DSarma%26aufirst%3DJ.%2BA.%2BR.%2BP.%26aulast%3DDesiraju%26aufirst%3DG.%2BR.%26atitle%3DVirtual%2520screening%2520of%25204-anilinoquinazoline%2520analogues%2520as%2520EGFR%2520kinase%2520inhibitors%253A%2520importance%2520of%2520hydrogen%2520bonds%2520in%2520the%2520evaluation%2520of%2520poses%2520and%2520scoring%2520functions%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2005%26volume%3D45%26spage%3D725%26epage%3D738" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group">Bramson, H. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Corona, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davis, S. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dickerson, S. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Edelstein, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frye, S. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gampe, R. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Harris, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hassell, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Holmes, W. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hunter, R. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lackey, K. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lovejoy, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luzzio, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Montana, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rocque, W. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rusnak, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shewchuk, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Veal, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Walker, D. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuyper, L. F.</span><span> </span><span class="NLM_article-title">Oxindole-based inhibitors of cyclin-dependent kinase 2 (CDK2): design, synthesis, enzymatic activities, and X-ray crystallographic analysis</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">44</span><span class="NLM_x">, </span> <span class="NLM_fpage">4339</span><span class="NLM_x">–</span> <span class="NLM_lpage">4358</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm010117d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Fjm070654j&amp;key=1%3ACAS%3A528%3ADC%252BD3MXnvFGnt7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2001&pages=4339-4358&author=H.+N.+Bramsonauthor=J.+Coronaauthor=S.+T.+Davisauthor=S.+H.+Dickersonauthor=M.+Edelsteinauthor=S.+V.+Fryeauthor=R.+T.+Gampeauthor=P.+A.+Harrisauthor=A.+Hassellauthor=W.+D.+Holmesauthor=R.+N.+Hunterauthor=K.+E.+Lackeyauthor=B.+Lovejoyauthor=M.+J.+Luzzioauthor=V.+Montanaauthor=W.+J.+Rocqueauthor=D.+Rusnakauthor=L.+Shewchukauthor=J.+M.+Vealauthor=D.+H.+Walkerauthor=L.+F.+Kuyper&title=Oxindole-based+inhibitors+of+cyclin-dependent+kinase+2+%28CDK2%29%3A+design%2C+synthesis%2C+enzymatic+activities%2C+and+X-ray+crystallographic+analysis"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Oxindole-Based Inhibitors of Cyclin-Dependent Kinase 2 (CDK2): Design, Synthesis, Enzymatic Activities, and X-ray Crystallographic Analysis</span></div><div class="casAuthors">Bramson, H. Neal; Corona, John; Davis, Stephen T.; Dickerson, Scott H.; Edelstein, Mark; Frye, Stephen V.; Gampe, Robert T., Jr.; Harris, Phil A.; Hassell, Anne; Holmes, William D.; Hunter, Robert N.; Lackey, Karen E.; Lovejoy, Brett; Luzzio, Michael J.; Montana, Val; Rocque, Warren J.; Rusnak, David; Shewchuk, Lisa; Veal, James M.; Walker, Duncan H.; Kuyper, Lee F.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">25</span>),
    <span class="NLM_cas:pages">4339-4358</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Two closely related classes of oxindole-based compds., 1H-indole-2,3-dione 3-phenylhydrazones and 3-(anilinomethylene)-1,3-dihydro-2H-indol-2-ones, were shown to potently inhibit cyclin-dependent kinase 2 (CDK2).  The initial lead compd. was prepd. as a homolog of the 3-benzylidene-1,3-dihydro-2H-indol-2-one class of kinase inhibitor.  Crystallog. anal. of the lead compd. bound to CDK2 provided the basis for analog design.  A semiautomated method of ligand docking was used to select compds. for synthesis, and a no. of compds. with low nanomolar inhibitory activity vs. CDK2 were identified.  Enzyme binding determinants for several analogs were evaluated by X-ray crystallog.  Compds. in this series inhibited CDK2 with a potency ∼10-fold greater than that for CDK1.  Members of this class of inhibitor cause an arrest of the cell cycle and have shown potential utility in the prevention of chemotherapy-induced alopecia.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGppOmtXxUWKlrVg90H21EOLACvtfcHk0lgX93pPaCP9pg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXnvFGnt7Y%253D&md5=d628549913ca46f9edf6b4692c82419d</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Fjm010117d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm010117d%26sid%3Dliteratum%253Aachs%26aulast%3DBramson%26aufirst%3DH.%2BN.%26aulast%3DCorona%26aufirst%3DJ.%26aulast%3DDavis%26aufirst%3DS.%2BT.%26aulast%3DDickerson%26aufirst%3DS.%2BH.%26aulast%3DEdelstein%26aufirst%3DM.%26aulast%3DFrye%26aufirst%3DS.%2BV.%26aulast%3DGampe%26aufirst%3DR.%2BT.%26aulast%3DHarris%26aufirst%3DP.%2BA.%26aulast%3DHassell%26aufirst%3DA.%26aulast%3DHolmes%26aufirst%3DW.%2BD.%26aulast%3DHunter%26aufirst%3DR.%2BN.%26aulast%3DLackey%26aufirst%3DK.%2BE.%26aulast%3DLovejoy%26aufirst%3DB.%26aulast%3DLuzzio%26aufirst%3DM.%2BJ.%26aulast%3DMontana%26aufirst%3DV.%26aulast%3DRocque%26aufirst%3DW.%2BJ.%26aulast%3DRusnak%26aufirst%3DD.%26aulast%3DShewchuk%26aufirst%3DL.%26aulast%3DVeal%26aufirst%3DJ.%2BM.%26aulast%3DWalker%26aufirst%3DD.%2BH.%26aulast%3DKuyper%26aufirst%3DL.%2BF.%26atitle%3DOxindole-based%2520inhibitors%2520of%2520cyclin-dependent%2520kinase%25202%2520%2528CDK2%2529%253A%2520design%252C%2520synthesis%252C%2520enzymatic%2520activities%252C%2520and%2520X-ray%2520crystallographic%2520analysis%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2001%26volume%3D44%26spage%3D4339%26epage%3D4358" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group">Gibson, A. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arris, C. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bentley, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boyle, F. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Curtin, N. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davies, T. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Endicott, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Golding, B. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grant, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Griffin, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jewsbury, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johnson, L. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mesguiche, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Newell, D. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Noble, M. E. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tucker, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Whitfield, H. J.</span><span> </span><span class="NLM_article-title">Probing the ATP ribose-binding domain of cyclin-dependent kinases 1 and 2 with O6-substituted guanine derivatives</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">45</span><span class="NLM_x">, </span> <span class="NLM_fpage">3381</span><span class="NLM_x">–</span> <span class="NLM_lpage">3393</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm020056z" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2002&pages=3381-3393&author=A.+E.+Gibsonauthor=C.+E.+Arrisauthor=J.+Bentleyauthor=F.+T.+Boyleauthor=N.+J.+Curtinauthor=T.+G.+Daviesauthor=J.+A.+Endicottauthor=B.+T.+Goldingauthor=S.+Grantauthor=R.+J.+Griffinauthor=P.+Jewsburyauthor=L.+N.+Johnsonauthor=V.+Mesguicheauthor=D.+R.+Newellauthor=M.+E.+M.+Nobleauthor=J.+A.+Tuckerauthor=H.+J.+Whitfield&title=Probing+the+ATP+ribose-binding+domain+of+cyclin-dependent+kinases+1+and+2+with+O6-substituted+guanine+derivatives"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1021%2Fjm020056z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm020056z%26sid%3Dliteratum%253Aachs%26aulast%3DGibson%26aufirst%3DA.%2BE.%26aulast%3DArris%26aufirst%3DC.%2BE.%26aulast%3DBentley%26aufirst%3DJ.%26aulast%3DBoyle%26aufirst%3DF.%2BT.%26aulast%3DCurtin%26aufirst%3DN.%2BJ.%26aulast%3DDavies%26aufirst%3DT.%2BG.%26aulast%3DEndicott%26aufirst%3DJ.%2BA.%26aulast%3DGolding%26aufirst%3DB.%2BT.%26aulast%3DGrant%26aufirst%3DS.%26aulast%3DGriffin%26aufirst%3DR.%2BJ.%26aulast%3DJewsbury%26aufirst%3DP.%26aulast%3DJohnson%26aufirst%3DL.%2BN.%26aulast%3DMesguiche%26aufirst%3DV.%26aulast%3DNewell%26aufirst%3DD.%2BR.%26aulast%3DNoble%26aufirst%3DM.%2BE.%2BM.%26aulast%3DTucker%26aufirst%3DJ.%2BA.%26aulast%3DWhitfield%26aufirst%3DH.%2BJ.%26atitle%3DProbing%2520the%2520ATP%2520ribose-binding%2520domain%2520of%2520cyclin-dependent%2520kinases%25201%2520and%25202%2520with%2520O6-substituted%2520guanine%2520derivatives%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2002%26volume%3D45%26spage%3D3381%26epage%3D3393" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group">Chen, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Norris, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Iwanowicz, E. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spergel, S. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gu, H. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shen, Z. Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wityak, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pang, S. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">de Fex, H. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pitt, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shen, D. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Doweyko, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bassolino, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roberge, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Poss, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, B. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schieven, G. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barrish, J. C.</span><span> </span><span class="NLM_article-title">Discovery and initial SAR of imidazoquinoxalines as inhibitors of the src-family kinase p56(Lck)</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">12</span><span class="NLM_x">, </span> <span class="NLM_fpage">1361</span><span class="NLM_x">–</span> <span class="NLM_lpage">1364</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2002&pages=1361-1364&author=P.+Chenauthor=D.+Norrisauthor=E.+J.+Iwanowiczauthor=S.+H.+Spergelauthor=J.+Linauthor=H.+H.+Guauthor=Z.+Q.+Shenauthor=J.+Wityakauthor=T.+A.+Linauthor=S.+H.+Pangauthor=H.+F.+de+Fexauthor=S.+Pittauthor=D.+R.+Shenauthor=A.+M.+Doweykoauthor=D.+A.+Bassolinoauthor=J.+Robergeauthor=M.+A.+Possauthor=B.+C.+Chenauthor=G.+L.+Schievenauthor=J.+C.+Barrish&title=Discovery+and+initial+SAR+of+imidazoquinoxalines+as+inhibitors+of+the+src-family+kinase+p56%28Lck%29"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DP.%26aulast%3DNorris%26aufirst%3DD.%26aulast%3DIwanowicz%26aufirst%3DE.%2BJ.%26aulast%3DSpergel%26aufirst%3DS.%2BH.%26aulast%3DLin%26aufirst%3DJ.%26aulast%3DGu%26aufirst%3DH.%2BH.%26aulast%3DShen%26aufirst%3DZ.%2BQ.%26aulast%3DWityak%26aufirst%3DJ.%26aulast%3DLin%26aufirst%3DT.%2BA.%26aulast%3DPang%26aufirst%3DS.%2BH.%26aulast%3Dde%2BFex%26aufirst%3DH.%2BF.%26aulast%3DPitt%26aufirst%3DS.%26aulast%3DShen%26aufirst%3DD.%2BR.%26aulast%3DDoweyko%26aufirst%3DA.%2BM.%26aulast%3DBassolino%26aufirst%3DD.%2BA.%26aulast%3DRoberge%26aufirst%3DJ.%26aulast%3DPoss%26aufirst%3DM.%2BA.%26aulast%3DChen%26aufirst%3DB.%2BC.%26aulast%3DSchieven%26aufirst%3DG.%2BL.%26aulast%3DBarrish%26aufirst%3DJ.%2BC.%26atitle%3DDiscovery%2520and%2520initial%2520SAR%2520of%2520imidazoquinoxalines%2520as%2520inhibitors%2520of%2520the%2520src-family%2520kinase%2520p56%2528Lck%2529%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2002%26volume%3D12%26spage%3D1361%26epage%3D1364" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group">Chen, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Iwanowicz, E. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Norris, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gu, H. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moquin, R. V</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Das, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wityak, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spergel, S. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">de Fex, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pang, S. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pitt, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shen, D. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schieven, G. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barrish, J. C.</span><span> </span><span class="NLM_article-title">Synthesis and SAR of novel imidazoquinoxaline-based Lck inhibitors: improvement of cell potency</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">12</span><span class="NLM_x">, </span> <span class="NLM_fpage">3153</span><span class="NLM_x">–</span> <span class="NLM_lpage">3156</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2002&pages=3153-3156&author=P.+Chenauthor=E.+J.+Iwanowiczauthor=D.+Norrisauthor=H.+H.+Guauthor=J.+Linauthor=R.+V+Moquinauthor=J.+Dasauthor=J.+Wityakauthor=S.+H.+Spergelauthor=H.+de+Fexauthor=S.+H.+Pangauthor=S.+Pittauthor=D.+R.+Shenauthor=G.+L.+Schievenauthor=J.+C.+Barrish&title=Synthesis+and+SAR+of+novel+imidazoquinoxaline-based+Lck+inhibitors%3A+improvement+of+cell+potency"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DP.%26aulast%3DIwanowicz%26aufirst%3DE.%2BJ.%26aulast%3DNorris%26aufirst%3DD.%26aulast%3DGu%26aufirst%3DH.%2BH.%26aulast%3DLin%26aufirst%3DJ.%26aulast%3DMoquin%26aufirst%3DR.%2BV%26aulast%3DDas%26aufirst%3DJ.%26aulast%3DWityak%26aufirst%3DJ.%26aulast%3DSpergel%26aufirst%3DS.%2BH.%26aulast%3Dde%2BFex%26aufirst%3DH.%26aulast%3DPang%26aufirst%3DS.%2BH.%26aulast%3DPitt%26aufirst%3DS.%26aulast%3DShen%26aufirst%3DD.%2BR.%26aulast%3DSchieven%26aufirst%3DG.%2BL.%26aulast%3DBarrish%26aufirst%3DJ.%2BC.%26atitle%3DSynthesis%2520and%2520SAR%2520of%2520novel%2520imidazoquinoxaline-based%2520Lck%2520inhibitors%253A%2520improvement%2520of%2520cell%2520potency%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2002%26volume%3D12%26spage%3D3153%26epage%3D3156" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group">Stelmach, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, L. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Patela, S. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pivnichny, J. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scapin, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Singh, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hop, C. E. C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Strauss, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cameron, P. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nichols, E. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">OʼKeefe, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">OʼNeill, E. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schmatz, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schwartz, C. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thompson, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zaller, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Doherty, J. B.</span><span> </span><span class="NLM_article-title">Design and synthesis of potent, orally bioavailable dihydroquinazolinone inhibitors of p38 MAP kinase</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">277</span><span class="NLM_x">–</span> <span class="NLM_lpage">280</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Fjm070654j&amp;key=10.1016%2FS0960-894X%2802%2900752-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Fjm070654j&amp;key=12482439" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Fjm070654j&amp;key=1%3ACAS%3A528%3ADC%252BD38XpsVemu7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2003&pages=277-280&author=J.+E.+Stelmachauthor=L.+P.+Liuauthor=S.+B.+Patelaauthor=J.+V.+Pivnichnyauthor=G.+Scapinauthor=S.+Singhauthor=C.+E.+C.+A.+Hopauthor=Z.+Wangauthor=J.+R.+Straussauthor=P.+M.+Cameronauthor=E.+A.+Nicholsauthor=S.+J.+O%CA%BCKeefeauthor=E.+A.+O%CA%BCNeillauthor=D.+M.+Schmatzauthor=C.+D.+Schwartzauthor=C.+M.+Thompsonauthor=D.+M.+Zallerauthor=J.+B.+Doherty&title=Design+and+synthesis+of+potent%2C+orally+bioavailable+dihydroquinazolinone+inhibitors+of+p38+MAP+kinase"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Design and synthesis of potent, orally bioavailable dihydroquinazolinone inhibitors of p38 MAP kinase</span></div><div class="casAuthors">Stelmach, John E.; Liu, Luping; Patel, Sangita B.; Pivnichny, James V.; Scapin, Giovanna; Singh, Suresh; Hop, Cornelis E. C. A.; Wang, Zhen; Strauss, John R.; Cameron, Patricia M.; Nichols, Elizabeth A.; O'Keefe, Stephen J.; O'Neill, Edward A.; Schmatz, Dennis M.; Schwartz, Cheryl D.; Thompson, Chris M.; Zaller, Dennis M.; Doherty, James B.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">277-280</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ltd.</span>)
        </div><div class="casAbstract">The development of potent, orally bioavailable (in rat) and selective dihydroquinazolinone inhibitors I (R = CO2Me, R1 = H, Br, 2-FC6H4, etc.; R = OMe, R1 = 2-Cl-4-FC6H3, 2,4-F2C6H3, 2-ClC6H4), II (R1 = 2-Cl-4-FC6H3, 2,4-F2C6H3, 2-ClC6H4, X = none, O, NH, CH2, CO, Y = CH, N) and III (R2 = Me, Et, CHMe2, etc.) of p38α MAP kinase is described.  For example, III (R2 = CMe3) was prepd. via the Still coupling of I (R = OSO2CF3, R1 = 2-Cl-4-FC6H3) with 1-tert-butyl-4-(trimethylstannyl)-1,2,3,6-tetrahydropyridine.  These analogs are hybrids of a pyridinylimidazole p38α inhibitor reported by Merck Research Labs. and VX-745.  Optimization of the C-5 Ph and the C-7 piperidinyl substituents led to the identification of III (R2 = CMe3) which gave excellent suppression of TNF-α prodn. in LPS-stimulated whole blood (IC50=10 nM) and good oral exposure in rats (F=68%, AUCn PO=0.58 μM h).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqeEsr0ksZsw7Vg90H21EOLACvtfcHk0lg4BU4BD_J7iQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XpsVemu7c%253D&md5=489ab070411bc48e879737d57d98d745</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1016%2FS0960-894X%2802%2900752-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0960-894X%252802%252900752-7%26sid%3Dliteratum%253Aachs%26aulast%3DStelmach%26aufirst%3DJ.%2BE.%26aulast%3DLiu%26aufirst%3DL.%2BP.%26aulast%3DPatela%26aufirst%3DS.%2BB.%26aulast%3DPivnichny%26aufirst%3DJ.%2BV.%26aulast%3DScapin%26aufirst%3DG.%26aulast%3DSingh%26aufirst%3DS.%26aulast%3DHop%26aufirst%3DC.%2BE.%2BC.%2BA.%26aulast%3DWang%26aufirst%3DZ.%26aulast%3DStrauss%26aufirst%3DJ.%2BR.%26aulast%3DCameron%26aufirst%3DP.%2BM.%26aulast%3DNichols%26aufirst%3DE.%2BA.%26aulast%3DO%25CA%25BCKeefe%26aufirst%3DS.%2BJ.%26aulast%3DO%25CA%25BCNeill%26aufirst%3DE.%2BA.%26aulast%3DSchmatz%26aufirst%3DD.%2BM.%26aulast%3DSchwartz%26aufirst%3DC.%2BD.%26aulast%3DThompson%26aufirst%3DC.%2BM.%26aulast%3DZaller%26aufirst%3DD.%2BM.%26aulast%3DDoherty%26aufirst%3DJ.%2BB.%26atitle%3DDesign%2520and%2520synthesis%2520of%2520potent%252C%2520orally%2520bioavailable%2520dihydroquinazolinone%2520inhibitors%2520of%2520p38%2520MAP%2520kinase%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2003%26volume%3D13%26spage%3D277%26epage%3D280" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group">Berset, C.;; Audétat, S.; Tietz, J.; Gunde, T.; Barberis, A.; Schumacher, A.; Traxler, P. </span><span> </span><span class="NLM_article-title">Protein Kinase Inhibitors</span>. WO/2005/120513,<span class="NLM_x"> </span><span class="NLM_year">2005</span><span class="NLM_x">, </span><div class="note"><p class="first last">(Oncalis AG)</p></div>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2005&author=C.%3B+Berset&author=S.+Aud%C3%A9tat&author=J.+Tietz&author=T.+Gunde&author=A.+Barberis&author=A.+Schumacher&author=P.+Traxler&title=Protein+Kinase+Inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DBerset%26aufirst%3DC.%253B%26atitle%3DProtein%2520Kinase%2520Inhibitors%26date%3D2005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group">Brooks, B. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bruccoleri, R. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Olafson, B. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">States, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Swaminathan, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Karplus, M</span><span> </span><span class="NLM_article-title">A program for macromolecular energy, minimization, and dynamics calculations</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1983</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">187</span><span class="NLM_x">–</span> <span class="NLM_lpage">217</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Fjm070654j&amp;key=10.1002%2Fjcc.540040211" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Fjm070654j&amp;key=1%3ACAS%3A528%3ADyaL3sXit1aiu7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=1983&pages=187-217&author=B.+R.+Brooksauthor=R.+E.+Bruccoleriauthor=B.+D.+Olafsonauthor=D.+J.+Statesauthor=S.+Swaminathanauthor=M+Karplus&title=A+program+for+macromolecular+energy%2C+minimization%2C+and+dynamics+calculations"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">CHARMM:  a program for macromolecular energy, minimization, and dynamics calculations</span></div><div class="casAuthors">Brooks, Bernard R.; Bruccoleri, Robert E.; Olafson, Barry D.; States, David J.; Swaminathan, S.; Karplus, Martin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">1983</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">187-217</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    </div><div class="casAbstract">CHARMM (Chem. at HARvard Macromol. Mechanics) is a highly flexible computer program which uses empirical energy functions to model macromol. systems.  The program can read or model build structures, energy minimize them by first- or second-deriv. techniques, perform a normal mode or mol. dynamics simulation, and analyze the structural, equil., and dynamic properties detd. in these calcns.  The operations that CHARMM can perform are described, and some implementation details are given.  A set of parameters for the empirical energy function and a sample run are included.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqBSFuoT3w87LVg90H21EOLACvtfcHk0lgk-iv_7m_f3A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL3sXit1aiu7w%253D&md5=bd639b4299ac9934f4497c1a9fe750d2</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1002%2Fjcc.540040211&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.540040211%26sid%3Dliteratum%253Aachs%26aulast%3DBrooks%26aufirst%3DB.%2BR.%26aulast%3DBruccoleri%26aufirst%3DR.%2BE.%26aulast%3DOlafson%26aufirst%3DB.%2BD.%26aulast%3DStates%26aufirst%3DD.%2BJ.%26aulast%3DSwaminathan%26aufirst%3DS.%26aulast%3DKarplus%26aufirst%3DM%26atitle%3DA%2520program%2520for%2520macromolecular%2520energy%252C%2520minimization%252C%2520and%2520dynamics%2520calculations%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D1983%26volume%3D4%26spage%3D187%26epage%3D217" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group">Momany, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rone, R.</span><span> </span><span class="NLM_article-title">Validation of the general purpose QUANTA 3.2/CHARMm force field</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1992</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">888</span><span class="NLM_x">–</span> <span class="NLM_lpage">900</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=1992&pages=888-900&author=F.+Momanyauthor=R.+Rone&title=Validation+of+the+general+purpose+QUANTA+3.2%2FCHARMm+force+field"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMomany%26aufirst%3DF.%26aulast%3DRone%26aufirst%3DR.%26atitle%3DValidation%2520of%2520the%2520general%2520purpose%2520QUANTA%25203.2%252FCHARMm%2520force%2520field%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D1992%26volume%3D13%26spage%3D888%26epage%3D900" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group">No, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grant, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scheraga, H.</span><span> </span><span class="NLM_article-title">Determination of net atomic charges using a modified partial equalization of orbital electronegativity method. 1. Application to neutral molecules as models for polypeptides</span> <span class="citation_source-journal">J. Phys. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1990</span><span class="NLM_x">, </span> <span class="NLM_volume">94</span><span class="NLM_x">, </span> <span class="NLM_fpage">4732</span><span class="NLM_x">–</span> <span class="NLM_lpage">4739</span></span><div class="citationLinks">[<a href="/doi/10.1021/j100374a066" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Fjm070654j&amp;key=1%3ACAS%3A528%3ADyaK3cXktVGgtbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=94&publication_year=1990&pages=4732-4739&author=K.+Noauthor=J.+Grantauthor=H.+Scheraga&title=Determination+of+net+atomic+charges+using+a+modified+partial+equalization+of+orbital+electronegativity+method.+1.+Application+to+neutral+molecules+as+models+for+polypeptides"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Determination of net atomic charges using a modified partial equalization of orbital electronegativity method.  1.  Application to neutral molecules as models for polypeptides</span></div><div class="casAuthors">No, Kyoung Tai; Grant, J. Andrew; Scheraga, Harold A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Physical Chemistry</span>
        (<span class="NLM_cas:date">1990</span>),
    <span class="NLM_cas:volume">94</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">4732-9</span>CODEN:
                <span class="NLM_cas:coden">JPCHAX</span>;
        ISSN:<span class="NLM_cas:issn">0022-3654</span>.
    </div><div class="casAbstract">Modified methods are proposed for detg. net at. charges, based on an existing scheme using the method of partial equalization of orbital electronegativity.  The new methods introduce (as constraints) exptl. dipole and, where possible, quadrupole moments of a set of mols. chosen to resemble the different at. environments found in proteins.  As a consequence, the at. charges reproduce accurately the exptl. values of mol. dipole and quadrupole moments.  The electrostatic potential surface, calcd. with the new net-at.-charge set, compares well to a surface obtained from a moderate-level ab initio calcn. (6-31G**).  However, the new charges are double those obtained from studies in which the charges are detd. from a least-squares fit to the calcd. ab initio electrostatic potential (potential-derived method).  The methodol. outlined leads to a direct calcn. of the permanent mol. charge distribution represented as a set of distributed monopoles, dependent only on the connectivity of the mol.  Hence, the method circumvents assumptions concerning the transferability of charges and can therefore approx. describe any redistribution of charge d. upon assembly of amino acid residues to form a polypeptide.  The method may be of use in providing an approx. representation of mol. charge distributions in extended systems, esp. large polypeptide mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr5hUCBNRO0TLVg90H21EOLACvtfcHk0lgk-iv_7m_f3A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3cXktVGgtbw%253D&md5=1783d0a7d1e5c52cb5bf93d96f16d8c6</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1021%2Fj100374a066&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fj100374a066%26sid%3Dliteratum%253Aachs%26aulast%3DNo%26aufirst%3DK.%26aulast%3DGrant%26aufirst%3DJ.%26aulast%3DScheraga%26aufirst%3DH.%26atitle%3DDetermination%2520of%2520net%2520atomic%2520charges%2520using%2520a%2520modified%2520partial%2520equalization%2520of%2520orbital%2520electronegativity%2520method.%25201.%2520Application%2520to%2520neutral%2520molecules%2520as%2520models%2520for%2520polypeptides%26jtitle%3DJ.%2520Phys.%2520Chem.%26date%3D1990%26volume%3D94%26spage%3D4732%26epage%3D4739" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group">No, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grant, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jhon, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scheraga, H.</span><span> </span><span class="NLM_article-title">Determination of net atomic charges using a modified partial equalization of orbital electronegativity method. 2. Application to ionic and aromatic molecules as models for polypeptides</span> <span class="citation_source-journal">J. Phys. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1990</span><span class="NLM_x">, </span> <span class="NLM_volume">94</span><span class="NLM_x">, </span> <span class="NLM_fpage">4740</span><span class="NLM_x">–</span> <span class="NLM_lpage">4746</span></span><div class="citationLinks">[<a href="/doi/10.1021/j100374a067" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Fjm070654j&amp;key=1%3ACAS%3A528%3ADyaK3cXitlCmsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=94&publication_year=1990&pages=4740-4746&author=K.+Noauthor=J.+Grantauthor=M.+Jhonauthor=H.+Scheraga&title=Determination+of+net+atomic+charges+using+a+modified+partial+equalization+of+orbital+electronegativity+method.+2.+Application+to+ionic+and+aromatic+molecules+as+models+for+polypeptides"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Determination of net atomic charges using a modified partial equalization of orbital electronegativity method.  2.  Application to ionic and aromatic molecules as models for polypeptides</span></div><div class="casAuthors">No, Kyoung Tai; Grant, J. Andrew; Jhon, Mu Shik; Scheraga, Harold A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Physical Chemistry</span>
        (<span class="NLM_cas:date">1990</span>),
    <span class="NLM_cas:volume">94</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">4740-6</span>CODEN:
                <span class="NLM_cas:coden">JPCHAX</span>;
        ISSN:<span class="NLM_cas:issn">0022-3654</span>.
    </div><div class="casAbstract">To finalize a scheme intended to calc. directly approx. charge distributions for proteins, parameters for a modified title method (PEOE) are obtained for both ionic and arom. functional groups.  For ionic groups, the electrostatic potentials of several ionic mols. (computed with the 6-31G** basis set) are used as constraints; for arom. groups, the exptl. gas-phase dipole and quadrupole moments of several arom. mols. are used as constraints.  The electrostatic potentials calcd. with the modified PEOE point charges of ionic mols. are quite close to the ab initio electrostatic potential surfaces, and the dipole and quadrupole moments calcd. with the modified PEOE charges of arom. mols. agree well with exptl. data.  The method enables the charge distribution, described as a set of atom-centered monopoles, to be calcd. directly.  This avoids assumptions concerning transferability of charges from small model compds. and describes in an approx. fashion the redistribution of charge upon assembly of amino acid residues into polypeptide mols.  The computational requirements for the procedure are negligible.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrdOLr-aBqHiLVg90H21EOLACvtfcHk0ljAZsOFjxmkmA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3cXitlCmsL8%253D&md5=b819eb54db46af0ad66a22d12d052c4b</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1021%2Fj100374a067&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fj100374a067%26sid%3Dliteratum%253Aachs%26aulast%3DNo%26aufirst%3DK.%26aulast%3DGrant%26aufirst%3DJ.%26aulast%3DJhon%26aufirst%3DM.%26aulast%3DScheraga%26aufirst%3DH.%26atitle%3DDetermination%2520of%2520net%2520atomic%2520charges%2520using%2520a%2520modified%2520partial%2520equalization%2520of%2520orbital%2520electronegativity%2520method.%25202.%2520Application%2520to%2520ionic%2520and%2520aromatic%2520molecules%2520as%2520models%2520for%2520polypeptides%26jtitle%3DJ.%2520Phys.%2520Chem.%26date%3D1990%26volume%3D94%26spage%3D4740%26epage%3D4746" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group">Thompson, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Higgins, D. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gibson, T. J.</span><span> </span><span class="NLM_article-title">CLUSTAL W: improving the sensitivity of progressive multiple sequence alignment through sequence weighting, position-specific gap penalties and weight matrix choice</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1994</span><span class="NLM_x">, </span> <span class="NLM_volume">22</span><span class="NLM_x">, </span> <span class="NLM_fpage">4673</span><span class="NLM_x">–</span> <span class="NLM_lpage">4680</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Fjm070654j&amp;key=10.1093%2Fnar%2F22.22.4673" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Fjm070654j&amp;key=7984417" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Fjm070654j&amp;key=1%3ACAS%3A528%3ADyaK2MXitlSgu74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=1994&pages=4673-4680&author=J.+D.+Thompsonauthor=D.+G.+Higginsauthor=T.+J.+Gibson&title=CLUSTAL+W%3A+improving+the+sensitivity+of+progressive+multiple+sequence+alignment+through+sequence+weighting%2C+position-specific+gap+penalties+and+weight+matrix+choice"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">CLUSTAL W: improving the sensitivity of progressive multiple sequence alignment through sequence weighting, position-specific gap penalties and weight matrix choice</span></div><div class="casAuthors">Thompson, Julie D.; Higgins, Desmond G.; Gibson, Toby J.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">4673-80</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The sensitivity of the commonly used progressive multiple sequence alignment method has been greatly improved for the alignment of divergent protein sequences.  Firstly, individual wts. are assigned to each sequence in a partial alignment in order to down-wt. near-duplicate sequences and up-wt. the most divergent ones.  Secondly, amino acid substitution matrixes are varied at different alignment stages according to the divergence of the sequences to be aligned.  Thirdly, residue-specific gap penalties and locally reduced gap penalties in hydrophilic regions encourage new gaps in potential loop regions rather than regular secondary structure.  Fourthly, positions in early alignments where gaps have been opened receive locally reduced gap penalties to encourage the opening of new gaps at these positions.  These modifications are incorporated into a new program, CLUSTAL W which is freely available.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGox0BWqmybJqbVg90H21EOLACvtfcHk0ljAZsOFjxmkmA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXitlSgu74%253D&md5=b219f8f0a8ea118b20a3e23461b99ccf</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1093%2Fnar%2F22.22.4673&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252F22.22.4673%26sid%3Dliteratum%253Aachs%26aulast%3DThompson%26aufirst%3DJ.%2BD.%26aulast%3DHiggins%26aufirst%3DD.%2BG.%26aulast%3DGibson%26aufirst%3DT.%2BJ.%26atitle%3DCLUSTAL%2520W%253A%2520improving%2520the%2520sensitivity%2520of%2520progressive%2520multiple%2520sequence%2520alignment%2520through%2520sequence%2520weighting%252C%2520position-specific%2520gap%2520penalties%2520and%2520weight%2520matrix%2520choice%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D1994%26volume%3D22%26spage%3D4673%26epage%3D4680" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group">Šali, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blundell, T. L.</span><span> </span><span class="NLM_article-title">Comparative protein modeling by satisfaction of spatial restraints</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1993</span><span class="NLM_x">, </span> <span class="NLM_volume">234</span><span class="NLM_x">, </span> <span class="NLM_fpage">779</span><span class="NLM_x">–</span> <span class="NLM_lpage">815</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=234&publication_year=1993&pages=779-815&author=A.+%C5%A0aliauthor=T.+L.+Blundell&title=Comparative+protein+modeling+by+satisfaction+of+spatial+restraints"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3D%25C5%25A0ali%26aufirst%3DA.%26aulast%3DBlundell%26aufirst%3DT.%2BL.%26atitle%3DComparative%2520protein%2520modeling%2520by%2520satisfaction%2520of%2520spatial%2520restraints%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D1993%26volume%3D234%26spage%3D779%26epage%3D815" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group">Marti-Renom, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stuart, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fiser, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sanchez, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Melo, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Šali, A.</span><span> </span><span class="NLM_article-title">Comparative protein structure modeling of genes and genomes</span> <span class="citation_source-journal">Annu. Rev. Biophys. Biomol. Struct.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">29</span><span class="NLM_x">, </span> <span class="NLM_fpage">291</span><span class="NLM_x">–</span> <span class="NLM_lpage">325</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Fjm070654j&amp;key=10.1146%2Fannurev.biophys.29.1.291" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Fjm070654j&amp;key=10940251" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Fjm070654j&amp;key=1%3ACAS%3A528%3ADC%252BD3cXlsFOnur0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2000&pages=291-325&author=M.+A.+Marti-Renomauthor=A.+C.+Stuartauthor=A.+Fiserauthor=R.+Sanchezauthor=F.+Meloauthor=A.+%C5%A0ali&title=Comparative+protein+structure+modeling+of+genes+and+genomes"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Comparative protein structure modeling of genes and genomes</span></div><div class="casAuthors">Marti-Renom, Marc A.; Stuart, Ashley C.; Fiser, Andras; Sanchez, Roberto; Melo, Francisco; Sali, Andrej</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Biophysics and Biomolecular Structure</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">291-325</span>CODEN:
                <span class="NLM_cas:coden">ABBSE4</span>;
        ISSN:<span class="NLM_cas:issn">1056-8700</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews Inc.</span>)
        </div><div class="casAbstract">A review with 212 refs.  Comparative modeling predicts the three-dimensional structure of a given protein sequence (target) based primarily on its alignment to one or more proteins of known structure (templates).  The prediction process consists of fold assignment, target-template alignment, model building, and model evaluation.  The no. of protein sequences that can be modeled and the accuracy of the predictions are increasing steadily because of the growth in the no. of known protein structures and because of the improvements in the modeling software.  Further advances are necessary in recognizing weak sequence-structure similarities, aligning sequences with structures, modeling of rigid body shifts, distortions, loops and side chains, as well as detecting errors in a model.  Despite these problems, it is currently possible to model with useful accuracy significant parts of approx. one third of all known protein sequences.  The use of individual comparative models in biol. is already rewarding and increasingly widespread.  A major new challenge for comparative modeling is the integration of it with the torrents of data from genome sequencing projects as well as from functional and structural genomics.  In particular, there is a need to develop an automated, rapid, robust, sensitive, and accurate comparative modeling pipeline applicable to whole genomes.  Such large-scale modeling is likely to encourage new kinds of applications for the many resulting models, based on their large no. and completeness at the level of the family, organism, or functional network.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq45B6Amw-zi7Vg90H21EOLACvtfcHk0ljAZsOFjxmkmA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXlsFOnur0%253D&md5=d97ffe6c47c4a3bae2bbbf14c045afdd</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1146%2Fannurev.biophys.29.1.291&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev.biophys.29.1.291%26sid%3Dliteratum%253Aachs%26aulast%3DMarti-Renom%26aufirst%3DM.%2BA.%26aulast%3DStuart%26aufirst%3DA.%2BC.%26aulast%3DFiser%26aufirst%3DA.%26aulast%3DSanchez%26aufirst%3DR.%26aulast%3DMelo%26aufirst%3DF.%26aulast%3D%25C5%25A0ali%26aufirst%3DA.%26atitle%3DComparative%2520protein%2520structure%2520modeling%2520of%2520genes%2520and%2520genomes%26jtitle%3DAnnu.%2520Rev.%2520Biophys.%2520Biomol.%2520Struct.%26date%3D2000%26volume%3D29%26spage%3D291%26epage%3D325" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group">Altschul, S. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Madden, T. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schäffer, A. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miller, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lipman, D. J.</span><span> </span><span class="NLM_article-title">Gapped BLAST and PSI-BLAST: a new generation of protein database search programs</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x">, </span> <span class="NLM_fpage">3389</span><span class="NLM_x">–</span> <span class="NLM_lpage">3402</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Fjm070654j&amp;key=10.1093%2Fnar%2F25.17.3389" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Fjm070654j&amp;key=9254694" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Fjm070654j&amp;key=1%3ACAS%3A528%3ADyaK2sXlvFyhu7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=1997&pages=3389-3402&author=S.+F.+Altschulauthor=T.+L.+Maddenauthor=A.+A.+Sch%C3%A4fferauthor=J.+Zhangauthor=Z.+Zhangauthor=W.+Millerauthor=D.+J.+Lipman&title=Gapped+BLAST+and+PSI-BLAST%3A+a+new+generation+of+protein+database+search+programs"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Gapped BLAST and PSI-BLAST: a new generation of protein database search programs</span></div><div class="casAuthors">Altschul, Stephen F.; Madden, Thomas L.; Schaffer, Alejandro A.; Zhang, Jinghui; Zhang, Zheng; Miller, Webb; Lipman, David J.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">3389-3402</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The BLAST programs are widely used tools for searching protein and DNA databases for sequence similarities.  For protein comparisons, a variety of definitional, algorithmic and statistical refinements described here permits the execution time of the BLAST programs to be decreased substantially while enhancing their sensitivity to weak similarities.  A new criterion for triggering the extension of word hits, combined with a new heuristic for generating gapped alignments, yields a gapped BLAST program that runs at approx. three times the speed of the original.  In addn., a method is introduced for automatically combining statistically significant alignments produced by BLAST into a position-specific score matrix, and searching the database using this matrix.  The resulting Position-Specific Iterated BLAST (PSI-BLAST) program runs at approx. the same speed per iteration as gapped BLAST, but in many cases is much more sensitive to weak but biol. relevant sequence similarities.  PSI-BLAST is used to uncover several new and interesting members of the BRCT superfamily.  The source code for the new BLAST programs is available by anonymous ftp from the machine ncbi.nlm.nih.gov, within the directory 'blast', and the programs may be run from NCBIs web site at http://www.ncbi.nlm.nih.gov/.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrfl-GGLfjeoLVg90H21EOLACvtfcHk0liG5cqBl7sdKQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXlvFyhu7w%253D&md5=4e44123e5984e4aca46a9899d347a176</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1093%2Fnar%2F25.17.3389&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252F25.17.3389%26sid%3Dliteratum%253Aachs%26aulast%3DAltschul%26aufirst%3DS.%2BF.%26aulast%3DMadden%26aufirst%3DT.%2BL.%26aulast%3DSch%25C3%25A4ffer%26aufirst%3DA.%2BA.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DZ.%26aulast%3DMiller%26aufirst%3DW.%26aulast%3DLipman%26aufirst%3DD.%2BJ.%26atitle%3DGapped%2520BLAST%2520and%2520PSI-BLAST%253A%2520a%2520new%2520generation%2520of%2520protein%2520database%2520search%2520programs%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D1997%26volume%3D25%26spage%3D3389%26epage%3D3402" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group">Holm, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sander, C.</span><span> </span><span class="NLM_article-title">Protein structure comparison by alignment of distance matrices</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1993</span><span class="NLM_x">, </span> <span class="NLM_volume">233</span><span class="NLM_x">, </span> <span class="NLM_fpage">123</span><span class="NLM_x">–</span> <span class="NLM_lpage">138</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Fjm070654j&amp;key=10.1006%2Fjmbi.1993.1489" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Fjm070654j&amp;key=8377180" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Fjm070654j&amp;key=1%3ACAS%3A528%3ADyaK3sXms1Gru7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=233&publication_year=1993&pages=123-138&author=L.+Holmauthor=C.+Sander&title=Protein+structure+comparison+by+alignment+of+distance+matrices"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Protein structure comparison by alignment of distance matrixes</span></div><div class="casAuthors">Holm, Liisa; Sander, Chris</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">233</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">123-38</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    </div><div class="casAbstract">With a rapidly growing pool of known tertiary structures, the importance of protein structure comparison parallels that of sequence alignment.  The authors have developed a novel algorithm (Dali) for optimal pairwise alignment of protein structures.  The three-dimensional coordinates of each protein are used to calc. residue-residue (Cα-Cα) distance matrixes.  The distance matrixes are first decompd. into elementary contact patterns, e.g., hexapeptide-hexapeptide submatrixes.  Then, similar contact patterns in the two matrixes are paired and combined into larger consistent sets of pairs.  A Monte Carlo procedure is used to optimize a similarity score defined in terms of equiv. intramol. distances.  Several alignments are optimized in parallel, leading to simultaneous detection of the best, second-best and so on solns.  The method allows sequence gaps of any length, reversal of chain direction and free topol. connectivity of aligned segments.  Sequential connectivity can be imposed as an option.  The method is fully automatic and identifies structural resemblances and common structural cores accurately and sensitively, even in the presence of geometrical distortions.  An all-against-all alignment of over 200 representative protein structures results in an objective classification of known three-dimensional folds in agreement with visual classifications.  Unexpected topol. similarities of biol. interest have been detected, e.g., between the bacterial toxin colicin A and globins and between the eukaryotic POU-specific DNA-binding domain and the bacterial λ repressor.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo_NVXKS4kkyrVg90H21EOLACvtfcHk0liG5cqBl7sdKQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3sXms1Gru7o%253D&md5=0312147da986d6364977108dfbf3f338</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1006%2Fjmbi.1993.1489&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Fjmbi.1993.1489%26sid%3Dliteratum%253Aachs%26aulast%3DHolm%26aufirst%3DL.%26aulast%3DSander%26aufirst%3DC.%26atitle%3DProtein%2520structure%2520comparison%2520by%2520alignment%2520of%2520distance%2520matrices%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D1993%26volume%3D233%26spage%3D123%26epage%3D138" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group">Laskowski, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">MacArthur, M. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moss, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thornton, J. M.</span><span> </span><span class="NLM_article-title">PROCHECK: a program to check the stereochemical quality of protein structures</span> <span class="citation_source-journal">J. Appl. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1993</span><span class="NLM_x">, </span> <span class="NLM_volume">26</span><span class="NLM_x">, </span> <span class="NLM_fpage">283</span><span class="NLM_x">–</span> <span class="NLM_lpage">291</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Fjm070654j&amp;key=10.1107%2FS0021889892009944" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Fjm070654j&amp;key=1%3ACAS%3A528%3ADyaK3sXit12lurY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=1993&pages=283-291&author=R.+A.+Laskowskiauthor=M.+W.+MacArthurauthor=D.+Mossauthor=J.+M.+Thornton&title=PROCHECK%3A+a+program+to+check+the+stereochemical+quality+of+protein+structures"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">PROCHECK: a program to check the stereochemical quality of protein structures</span></div><div class="casAuthors">Laskowski, Roman A.; MacArthur, Malcolm W.; Moss, David S.; Thornton, Janet M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">283-91</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    </div><div class="casAbstract">The PROCHECK suite of programs provides a detailed check on the stereochem. of a protein structure.  Its outputs comprise a no. of plots in PostScript format and a comprehensive residue-by-residue listing.  These give an assessment of the overall quality of the structure as compared with well-refined structures of the same resoln. and also highlight regions that may need further investigation.  The PROCHECK programs are useful for assessing the quality not only of protein structures in the process of being solved but also of existing structures and of those being modeled on known structures.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrCe5FgJf4KH7Vg90H21EOLACvtfcHk0liG5cqBl7sdKQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3sXit12lurY%253D&md5=fb69fc4410cd716aaaa7cc0db06b3ed2</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1107%2FS0021889892009944&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0021889892009944%26sid%3Dliteratum%253Aachs%26aulast%3DLaskowski%26aufirst%3DR.%2BA.%26aulast%3DMacArthur%26aufirst%3DM.%2BW.%26aulast%3DMoss%26aufirst%3DD.%26aulast%3DThornton%26aufirst%3DJ.%2BM.%26atitle%3DPROCHECK%253A%2520a%2520program%2520to%2520check%2520the%2520stereochemical%2520quality%2520of%2520protein%2520structures%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D1993%26volume%3D26%26spage%3D283%26epage%3D291" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group">Im, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beglov, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roux, B.</span><span> </span><span class="NLM_article-title">Continuum solvation model: computation of electrostatic forces from numerical solutions to the poisson-boltzmann equation</span> <span class="citation_source-journal">Comput. Phys. Commun.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">111</span><span class="NLM_x">, </span> <span class="NLM_fpage">59</span><span class="NLM_x">–</span> <span class="NLM_lpage">75</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Fjm070654j&amp;key=10.1016%2FS0010-4655%2898%2900016-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Fjm070654j&amp;key=1%3ACAS%3A528%3ADyaK1cXktFGmtr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=111&publication_year=1998&pages=59-75&author=W.+Imauthor=D.+Beglovauthor=B.+Roux&title=Continuum+solvation+model%3A+computation+of+electrostatic+forces+from+numerical+solutions+to+the+poisson-boltzmann+equation"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Continuum Solvation Model: computation of electrostatic forces from numerical solutions to the Poisson-Boltzmann equation</span></div><div class="casAuthors">Im, Wonpil; Beglov, Dmitrii; Roux, Benoit</div><div class="citationInfo"><span class="NLM_cas:title">Computer Physics Communications</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">111</span>
        (<span class="NLM_cas:issue">1-3</span>),
    <span class="NLM_cas:pages">59-75</span>CODEN:
                <span class="NLM_cas:coden">CPHCBZ</span>;
        ISSN:<span class="NLM_cas:issn">0010-4655</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science B.V.</span>)
        </div><div class="casAbstract">A rigorous formulation of the solvation forces (first derivs.) assocd. with the electrostatic free energy calcd. from numerical solns. of the linearized Poisson-Boltzmann equation on a discrete grid is described.  The solvation forces are obtained from the formal soln. of the linearized Poisson-Boltzmann equation written in terms of the Green function.  An intermediate region for the solute-solvent dielec. boundary is introduced to yield a continuous solvation free energy and accurate solvation forces.  A series of numerical tests show that the calcd. forces agree extremely well with finite-difference derivs. of the solvation free energy.  To gain a max. efficiency, the nonpolar contribution to the free energy is expressed in terms of the discretized grid used for the electrostatic problem.  The current treatment of solvation forces can be used to introduce the influence of a continuum solvation model in mol. mechanics calcns. of large biol. systems.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrtvYWWwNDnfbVg90H21EOLACvtfcHk0li1aJtZkHHwBg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXktFGmtr4%253D&md5=5c6c98db5e70117cacfd4c8eb123a923</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1016%2FS0010-4655%2898%2900016-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0010-4655%252898%252900016-2%26sid%3Dliteratum%253Aachs%26aulast%3DIm%26aufirst%3DW.%26aulast%3DBeglov%26aufirst%3DD.%26aulast%3DRoux%26aufirst%3DB.%26atitle%3DContinuum%2520solvation%2520model%253A%2520computation%2520of%2520electrostatic%2520forces%2520from%2520numerical%2520solutions%2520to%2520the%2520poisson-boltzmann%2520equation%26jtitle%3DComput.%2520Phys.%2520Commun.%26date%3D1998%26volume%3D111%26spage%3D59%26epage%3D75" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group">Feng, B. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simeonov, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jadhav, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Babaoglu, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Inglese, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Austin, C. P. B.</span><span> </span><span class="NLM_article-title">High-throughput screen for aggregation-based inhibition in a large compound library</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x">, </span> <span class="NLM_fpage">2385</span><span class="NLM_x">–</span> <span class="NLM_lpage">2390</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm061317y" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2007&pages=2385-2390&author=B.+Y.+Fengauthor=A.+Simeonovauthor=A.+Jadhavauthor=K.+Babaogluauthor=J.+Ingleseauthor=B.+K.+Shoichetauthor=C.+P.+B.+Austin&title=High-throughput+screen+for+aggregation-based+inhibition+in+a+large+compound+library"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1021%2Fjm061317y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm061317y%26sid%3Dliteratum%253Aachs%26aulast%3DFeng%26aufirst%3DB.%2BY.%26aulast%3DSimeonov%26aufirst%3DA.%26aulast%3DJadhav%26aufirst%3DA.%26aulast%3DBabaoglu%26aufirst%3DK.%26aulast%3DInglese%26aufirst%3DJ.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26aulast%3DAustin%26aufirst%3DC.%2BP.%2BB.%26atitle%3DHigh-throughput%2520screen%2520for%2520aggregation-based%2520inhibition%2520in%2520a%2520large%2520compound%2520library%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2007%26volume%3D50%26spage%3D2385%26epage%3D2390" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">Screening in a spirit haunted world</span> <span class="citation_source-journal">Drug Discovery Today</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">607</span><span class="NLM_x">–</span> <span class="NLM_lpage">615</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Fjm070654j&amp;key=10.1016%2Fj.drudis.2006.05.014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Fjm070654j&amp;key=16793529" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Fjm070654j&amp;key=1%3ACAS%3A528%3ADC%252BD28XlvFGqtLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2006&pages=607-615&author=B.+K.+Shoichet&title=Screening+in+a+spirit+haunted+world"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Screening in a spirit haunted world</span></div><div class="casAuthors">Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">13 & 14</span>),
    <span class="NLM_cas:pages">607-615</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  High-throughput screening (HTS) campaigns can be dominated by hits that ultimately turn out to be non-drug-like.  These 'nuisance' compds. often behave strangely, with steep dose-response curves, absence of clear structure-activity relationships, and high sensitivity to assay conditions.  Several mechanisms contribute to these artifacts, including chem. reactive mols., those that absorb light in assays and those that affect redox conditions.  One of the most common mechanisms behind artifactual inhibition is discussed in this review: at micromolar concns. org. mols. can aggregate to form particles in aq. buffers, and these aggregates can sequester and thereby inhibit protein targets.  Aggregation-based inhibition is baffling from a chem. perspective, but viewed biophys. such behavior is expected.  The range of mols. that behave this way, their rapid detection in a screening environment and their possible biol. implications will be considered here.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpzoEkwI2izy7Vg90H21EOLACvtfcHk0lgeYDtmjXTZDQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XlvFGqtLk%253D&md5=08d22b6b06215ac435824b7664b3674c</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2006.05.014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2006.05.014%26sid%3Dliteratum%253Aachs%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DScreening%2520in%2520a%2520spirit%2520haunted%2520world%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2006%26volume%3D11%26spage%3D607%26epage%3D615" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group">Rockey, W. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Elcock, A. H.</span><span> </span><span class="NLM_article-title">Rapid computational identification of the targets of protein kinase inhibitors</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">48</span><span class="NLM_x">, </span> <span class="NLM_fpage">4138</span><span class="NLM_x">–</span> <span class="NLM_lpage">4152</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm049461b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2005&pages=4138-4152&author=W.+M.+Rockeyauthor=A.+H.+Elcock&title=Rapid+computational+identification+of+the+targets+of+protein+kinase+inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1021%2Fjm049461b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm049461b%26sid%3Dliteratum%253Aachs%26aulast%3DRockey%26aufirst%3DW.%2BM.%26aulast%3DElcock%26aufirst%3DA.%2BH.%26atitle%3DRapid%2520computational%2520identification%2520of%2520the%2520targets%2520of%2520protein%2520kinase%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2005%26volume%3D48%26spage%3D4138%26epage%3D4152" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group">Irwin, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">ZINC—a free database of commercially available compounds for virtual screening</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">45</span><span class="NLM_x">, </span> <span class="NLM_fpage">177</span><span class="NLM_x">–</span> <span class="NLM_lpage">182</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci049714%2B" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Fjm070654j&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVOjt77J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2005&pages=177-182&author=J.+J.+Irwinauthor=B.+K.+Shoichet&title=ZINC%E2%80%94a+free+database+of+commercially+available+compounds+for+virtual+screening"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">ZINC - A Free Database of Commercially Available Compounds for Virtual Screening</span></div><div class="casAuthors">Irwin, John J.; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Computer Sciences</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">177-182</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">0095-2338</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A crit. barrier to entry into structure-based virtual screening is the lack of a suitable, easy to access database of purchasable compds.  We have therefore prepd. a library of 727 842 mols., each with 3D structure, using catalogs of compds. from vendors (the size of this library continues to grow).  The mols. have been assigned biol. relevant protonation states and are annotated with properties such as mol. wt., calcd. LogP, and no. of rotatable bonds.  Each mol. in the library contains vendor and purchasing information and is ready for docking using a no. of popular docking programs.  Within certain limits, the mols. are prepd. in multiple protonation states and multiple tautomeric forms.  In one format, multiple conformations are available for the mols.  This database is available for free download (http://zinc.docking.org) in several common file formats including SMILES, mol2, 3D SDF, and DOCK flexibase format.  A Web-based query tool incorporating a mol. drawing interface enables the database to be searched and browsed and subsets to be created.  Users can process their own mols. by uploading them to a server.  Our hope is that this database will bring virtual screening libraries to a wide community of structural biologists and medicinal chemists.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpPhFw5E2XACbVg90H21EOLACvtfcHk0lgeYDtmjXTZDQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVOjt77J&md5=e3892b7dc8608b17a3e63541a5ed60e6</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1021%2Fci049714%2B&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci049714%252B%26sid%3Dliteratum%253Aachs%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DZINC%25E2%2580%2594a%2520free%2520database%2520of%2520commercially%2520available%2520compounds%2520for%2520virtual%2520screening%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2005%26volume%3D45%26spage%3D177%26epage%3D182" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group">Kolb, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Caflisch, A.</span><span> </span><span class="NLM_article-title">Automatic and efficient decomposition of two-dimensional structures of small molecules for fragment-based high-throughput docking</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">7384</span><span class="NLM_x">–</span> <span class="NLM_lpage">7392</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm060838i" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2006&pages=7384-7392&author=P.+Kolbauthor=A.+Caflisch&title=Automatic+and+efficient+decomposition+of+two-dimensional+structures+of+small+molecules+for+fragment-based+high-throughput+docking"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1021%2Fjm060838i&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm060838i%26sid%3Dliteratum%253Aachs%26aulast%3DKolb%26aufirst%3DP.%26aulast%3DCaflisch%26aufirst%3DA.%26atitle%3DAutomatic%2520and%2520efficient%2520decomposition%2520of%2520two-dimensional%2520structures%2520of%2520small%2520molecules%2520for%2520fragment-based%2520high-throughput%2520docking%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2006%26volume%3D49%26spage%3D7384%26epage%3D7392" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group">Budin, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Majeux, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Caflisch, A.</span><span> </span><span class="NLM_article-title">Fragment-based flexible ligand docking by evolutionary opimization</span> <span class="citation_source-journal">Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">382</span><span class="NLM_x">, </span> <span class="NLM_fpage">1365</span><span class="NLM_x">–</span> <span class="NLM_lpage">1372</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=382&publication_year=2001&pages=1365-1372&author=N.+Budinauthor=N.+Majeuxauthor=A.+Caflisch&title=Fragment-based+flexible+ligand+docking+by+evolutionary+opimization"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBudin%26aufirst%3DN.%26aulast%3DMajeux%26aufirst%3DN.%26aulast%3DCaflisch%26aufirst%3DA.%26atitle%3DFragment-based%2520flexible%2520ligand%2520docking%2520by%2520evolutionary%2520opimization%26jtitle%3DBiol.%2520Chem.%26date%3D2001%26volume%3D382%26spage%3D1365%26epage%3D1372" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54"><span><span class="NLM_contrib-group">Cecchini, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kolb, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Majeux, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Caflisch, A.</span><span> </span><span class="NLM_article-title">Automated docking of highly flexible ligands by genetic algorithms: A critical assessment</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x">, </span> <span class="NLM_fpage">412</span><span class="NLM_x">–</span> <span class="NLM_lpage">422</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2004&pages=412-422&author=M.+Cecchiniauthor=P.+Kolbauthor=N.+Majeuxauthor=A.+Caflisch&title=Automated+docking+of+highly+flexible+ligands+by+genetic+algorithms%3A+A+critical+assessment"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DCecchini%26aufirst%3DM.%26aulast%3DKolb%26aufirst%3DP.%26aulast%3DMajeux%26aufirst%3DN.%26aulast%3DCaflisch%26aufirst%3DA.%26atitle%3DAutomated%2520docking%2520of%2520highly%2520flexible%2520ligands%2520by%2520genetic%2520algorithms%253A%2520A%2520critical%2520assessment%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2004%26volume%3D25%26spage%3D412%26epage%3D422" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref55"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref55'); return false;" data-citation="" class="refNumLink">55</a></strong><div class="NLM_citation" id="cit55"><span><span class="NLM_contrib-group">Zweig, M. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Campbell, G.</span><span> </span><span class="NLM_article-title">Receiver-operating characteristic (ROC) plots—a fundamental evaluation tool in clinical medicine</span> <span class="citation_source-journal">Clin. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1993</span><span class="NLM_x">, </span> <span class="NLM_volume">39</span><span class="NLM_x">, </span> <span class="NLM_fpage">561</span><span class="NLM_x">–</span> <span class="NLM_lpage">577</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Fjm070654j&amp;key=8472349" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Fjm070654j&amp;key=1%3ACAS%3A528%3ADyaK3sXktVCitLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=1993&pages=561-577&author=M.+H.+Zweigauthor=G.+Campbell&title=Receiver-operating+characteristic+%28ROC%29+plots%E2%80%94a+fundamental+evaluation+tool+in+clinical+medicine"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Receiver-operating characteristic (ROC) plots: a fundamdntal evaluation tool in clinical medicine</span></div><div class="casAuthors">Zweig, Mark H.; Campbell, Gregory</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Chemistry (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">39</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">561-77</span>CODEN:
                <span class="NLM_cas:coden">CLCHAU</span>;
        ISSN:<span class="NLM_cas:issn">0009-9147</span>.
    </div><div class="casAbstract">A review with 77 refs.  The clin. performance of a lab. test can be described in terms of diagnostic accuracy, or the ability to correctly classify subjects into clin. relevant subgroupss.  Diagnostic accuracy refers to the quality of the information provided by the classification device and should be distinguished from the usefulness, or actual practical value, of the information.  Receiver-operating characteristics (ROC) plots provide a pure index of accuracy by demonstrating the limits of a test's ability to discriminate between alternative states of health over the complete spectrum of operating conditions.  Furthermore, ROC plots occupy a central or unifying position in the process of assessing and using diagnostic tools.  Once the plot is generated, a user can readily go on to many other activities such as performing quant. ROC anal. and comparisons of tests, using likelihood ratio to revise the probability of disease in individual subjects, selecting decision thresholds, using logistic-regression anal., using discriminant-function anal., or incorporating the tool into a clin. strategy by using decision anal.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoHGuiPFaxD87Vg90H21EOLACvtfcHk0ljVzW6UXomrlw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3sXktVCitLs%253D&md5=0dc685cbe02354d2e65c773355fb5c45</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DZweig%26aufirst%3DM.%2BH.%26aulast%3DCampbell%26aufirst%3DG.%26atitle%3DReceiver-operating%2520characteristic%2520%2528ROC%2529%2520plots%25E2%2580%2594a%2520fundamental%2520evaluation%2520tool%2520in%2520clinical%2520medicine%26jtitle%3DClin.%2520Chem.%26date%3D1993%26volume%3D39%26spage%3D561%26epage%3D577" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54'],'ref55':['cit55']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 46 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Jean-Rémy Marchand, Tim Knehans, Amedeo Caflisch, <span class="NLM_string-name hlFld-ContribAuthor">Andreas Vitalis</span>. </span><span class="cited-content_cbyCitation_article-title">An ABSINTH-Based Protocol for Predicting Binding Affinities between Proteins and Small Molecules. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2020,</strong> <em>60 </em>
                                    (10)
                                     , 5188-5202. <a href="https://doi.org/10.1021/acs.jcim.0c00558" title="DOI URL">https://doi.org/10.1021/acs.jcim.0c00558</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.0c00558&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.0c00558%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DAn%252BABSINTH-Based%252BProtocol%252Bfor%252BPredicting%252BBinding%252BAffinities%252Bbetween%252BProteins%252Band%252BSmall%252BMolecules%26aulast%3DMarchand%26aufirst%3DJean-R%25C3%25A9my%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D19052020%26date%3D23092020%26date%3D08092020%26volume%3D60%26issue%3D10%26spage%3D5188%26epage%3D5202" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jean-Rémy  Marchand</span>, <span class="hlFld-ContribAuthor ">Andrea  Dalle Vedove</span>, <span class="hlFld-ContribAuthor ">Graziano  Lolli</span>, and <span class="hlFld-ContribAuthor ">Amedeo  Caflisch</span>  . </span><span class="cited-content_cbyCitation_article-title">Discovery of Inhibitors of Four Bromodomains by Fragment-Anchored Ligand Docking. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2017,</strong> <em>57 </em>
                                    (10)
                                     , 2584-2597. <a href="https://doi.org/10.1021/acs.jcim.7b00336" title="DOI URL">https://doi.org/10.1021/acs.jcim.7b00336</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.7b00336&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.7b00336%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DDiscovery%252Bof%252BInhibitors%252Bof%252BFour%252BBromodomains%252Bby%252BFragment-Anchored%252BLigand%252BDocking%26aulast%3DMarchand%26aufirst%3DJean-R%25C3%25A9my%26date%3D2017%26date%3D2017%26date%3D2017%26date%3D06062017%26date%3D18092017%26date%3D23102017%26date%3D01092017%26volume%3D57%26issue%3D10%26spage%3D2584%26epage%3D2597" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Praveen  Nedumpully-Govindan</span>, <span class="hlFld-ContribAuthor ">Domen B.  Jemec</span>, and <span class="hlFld-ContribAuthor ">Feng  Ding</span>  . </span><span class="cited-content_cbyCitation_article-title">CSAR Benchmark of Flexible MedusaDock in Affinity Prediction and Nativelike Binding Pose Selection. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2016,</strong> <em>56 </em>
                                    (6)
                                     , 1042-1052. <a href="https://doi.org/10.1021/acs.jcim.5b00303" title="DOI URL">https://doi.org/10.1021/acs.jcim.5b00303</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.5b00303&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.5b00303%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DCSAR%252BBenchmark%252Bof%252BFlexible%252BMedusaDock%252Bin%252BAffinity%252BPrediction%252Band%252BNativelike%252BBinding%252BPose%252BSelection%26aulast%3DNedumpully-Govindan%26aufirst%3DPraveen%26date%3D2016%26date%3D2015%26date%3D2015%26date%3D22052015%26date%3D17082015%26date%3D27062016%26date%3D07082015%26volume%3D56%26issue%3D6%26spage%3D1042%26epage%3D1052" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Kanin  Wichapong</span>, <span class="hlFld-ContribAuthor ">Jean-Eric  Alard</span>, <span class="hlFld-ContribAuthor ">Almudena  Ortega-Gomez</span>, <span class="hlFld-ContribAuthor ">Christian  Weber</span>, <span class="hlFld-ContribAuthor ">Tilman M.  Hackeng</span>, <span class="hlFld-ContribAuthor ">Oliver  Soehnlein</span>, and <span class="hlFld-ContribAuthor ">Gerry A. F.  Nicolaes</span>  . </span><span class="cited-content_cbyCitation_article-title">Structure-Based Design of Peptidic Inhibitors of the Interaction between CC Chemokine Ligand 5 (CCL5) and Human Neutrophil Peptides 1 (HNP1). </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2016,</strong> <em>59 </em>
                                    (9)
                                     , 4289-4301. <a href="https://doi.org/10.1021/acs.jmedchem.5b01952" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.5b01952</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.5b01952&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.5b01952%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DStructure-Based%252BDesign%252Bof%252BPeptidic%252BInhibitors%252Bof%252Bthe%252BInteraction%252Bbetween%252BCC%252BChemokine%252BLigand%252B5%252B%252528CCL5%252529%252Band%252BHuman%252BNeutrophil%252BPeptides%252B1%252B%252528HNP1%252529%26aulast%3DWichapong%26aufirst%3DKanin%26date%3D2016%26date%3D2016%26date%3D2015%26date%3D17122015%26date%3D23022016%26date%3D12052016%26date%3D12022016%26volume%3D59%26issue%3D9%26spage%3D4289%26epage%3D4301" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ariane  Nunes-Alves</span> and <span class="hlFld-ContribAuthor ">Guilherme Menegon  Arantes</span>  . </span><span class="cited-content_cbyCitation_article-title">Ligand–Receptor Affinities Computed by an Adapted Linear Interaction Model for Continuum Electrostatics and by Protein Conformational Averaging. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2014,</strong> <em>54 </em>
                                    (8)
                                     , 2309-2319. <a href="https://doi.org/10.1021/ci500301s" title="DOI URL">https://doi.org/10.1021/ci500301s</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/ci500301s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fci500301s%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DLigand%2525E2%252580%252593Receptor%252BAffinities%252BComputed%252Bby%252Ban%252BAdapted%252BLinear%252BInteraction%252BModel%252Bfor%252BContinuum%252BElectrostatics%252Band%252Bby%252BProtein%252BConformational%252BAveraging%26aulast%3DNunes-Alves%26aufirst%3DAriane%26date%3D2014%26date%3D2014%26date%3D2014%26date%3D19052014%26date%3D06082014%26date%3D25082014%26date%3D30072014%26volume%3D54%26issue%3D8%26spage%3D2309%26epage%3D2319" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Lei  Xu</span>, <span class="hlFld-ContribAuthor ">Yu  Zhang</span>, <span class="hlFld-ContribAuthor ">Longtai  Zheng</span>, <span class="hlFld-ContribAuthor ">Chunhua  Qiao</span>, <span class="hlFld-ContribAuthor ">Youyong  Li</span>, <span class="hlFld-ContribAuthor ">Dan  Li</span>, <span class="hlFld-ContribAuthor ">Xuechu  Zhen</span>, and <span class="hlFld-ContribAuthor ">Tingjun  Hou</span>  . </span><span class="cited-content_cbyCitation_article-title">Discovery of Novel Inhibitors Targeting the Macrophage Migration Inhibitory Factor via Structure-Based Virtual Screening and Bioassays. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2014,</strong> <em>57 </em>
                                    (9)
                                     , 3737-3745. <a href="https://doi.org/10.1021/jm401908w" title="DOI URL">https://doi.org/10.1021/jm401908w</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/jm401908w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fjm401908w%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252BNovel%252BInhibitors%252BTargeting%252Bthe%252BMacrophage%252BMigration%252BInhibitory%252BFactor%252Bvia%252BStructure-Based%252BVirtual%252BScreening%252Band%252BBioassays%26aulast%3DXu%26aufirst%3DLei%26date%3D2014%26date%3D2014%26date%3D2013%26date%3D11122013%26date%3D23042014%26date%3D08052014%26date%3D09042014%26volume%3D57%26issue%3D9%26spage%3D3737%26epage%3D3745" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Kanin  Wichapong</span>, <span class="hlFld-ContribAuthor ">Alexander  Rohe</span>, <span class="hlFld-ContribAuthor ">Charlott  Platzer</span>, <span class="hlFld-ContribAuthor ">Inna  Slynko</span>, <span class="hlFld-ContribAuthor ">Frank  Erdmann</span>, <span class="hlFld-ContribAuthor ">Matthias  Schmidt</span>, and <span class="hlFld-ContribAuthor ">Wolfgang  Sippl</span>  . </span><span class="cited-content_cbyCitation_article-title">Application of Docking and QM/MM-GBSA Rescoring to Screen for Novel Myt1 Kinase Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2014,</strong> <em>54 </em>
                                    (3)
                                     , 881-893. <a href="https://doi.org/10.1021/ci4007326" title="DOI URL">https://doi.org/10.1021/ci4007326</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/ci4007326&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fci4007326%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DApplication%252Bof%252BDocking%252Band%252BQM%25252FMM-GBSA%252BRescoring%252Bto%252BScreen%252Bfor%252BNovel%252BMyt1%252BKinase%252BInhibitors%26aulast%3DWichapong%26aufirst%3DKanin%26date%3D2014%26date%3D2014%26date%3D2013%26date%3D10122013%26date%3D13022014%26date%3D24032014%26date%3D03022014%26volume%3D54%26issue%3D3%26spage%3D881%26epage%3D893" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Obaidur  Rahaman</span>, <span class="hlFld-ContribAuthor ">Trilce P.  Estrada</span>, <span class="hlFld-ContribAuthor ">Douglas J.  Doren</span>, <span class="hlFld-ContribAuthor ">Michela  Taufer</span>, <span class="hlFld-ContribAuthor ">Charles L.  Brooks, III</span>, and <span class="hlFld-ContribAuthor ">Roger S.  Armen</span>  . </span><span class="cited-content_cbyCitation_article-title">Evaluation of Several Two-Step Scoring Functions Based on Linear Interaction Energy, Effective Ligand Size, and Empirical Pair Potentials for Prediction of Protein–Ligand Binding Geometry and Free Energy. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2011,</strong> <em>51 </em>
                                    (9)
                                     , 2047-2065. <a href="https://doi.org/10.1021/ci1003009" title="DOI URL">https://doi.org/10.1021/ci1003009</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/ci1003009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fci1003009%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DEvaluation%252Bof%252BSeveral%252BTwo-Step%252BScoring%252BFunctions%252BBased%252Bon%252BLinear%252BInteraction%252BEnergy%25252C%252BEffective%252BLigand%252BSize%25252C%252Band%252BEmpirical%252BPair%252BPotentials%252Bfor%252BPrediction%252Bof%252BProtein%2525E2%252580%252593Ligand%252BBinding%252BGeometry%252Band%252BFree%252BEnergy%26aulast%3DRahaman%26aufirst%3DObaidur%26date%3D2011%26date%3D2011%26date%3D2010%26date%3D06082010%26date%3D06062011%26date%3D26092011%26volume%3D51%26issue%3D9%26spage%3D2047%26epage%3D2065" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Kanin  Wichapong</span>, <span class="hlFld-ContribAuthor ">Michael  Lawson</span>, <span class="hlFld-ContribAuthor ">Somsak  Pianwanit</span>, <span class="hlFld-ContribAuthor ">Sirirat  Kokpol</span>, and <span class="hlFld-ContribAuthor ">Wolfgang  Sippl</span>  . </span><span class="cited-content_cbyCitation_article-title">Postprocessing of Protein−Ligand Docking Poses Using Linear Response MM-PB/SA: Application to Wee1 Kinase Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2010,</strong> <em>50 </em>
                                    (9)
                                     , 1574-1588. <a href="https://doi.org/10.1021/ci1002153" title="DOI URL">https://doi.org/10.1021/ci1002153</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/ci1002153&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fci1002153%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DPostprocessing%252Bof%252BProtein%2525E2%252588%252592Ligand%252BDocking%252BPoses%252BUsing%252BLinear%252BResponse%252BMM-PB%25252FSA%25253A%252BApplication%252Bto%252BWee1%252BKinase%252BInhibitors%26aulast%3DWichapong%26aufirst%3DKanin%26date%3D2010%26date%3D2010%26date%3D2010%26date%3D31%252005%25202010%26date%3D16082010%26date%3D27092010%26volume%3D50%26issue%3D9%26spage%3D1574%26epage%3D1588" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Roger S.  Armen</span>, <span class="hlFld-ContribAuthor ">Jianhan  Chen</span> and <span class="hlFld-ContribAuthor ">Charles L.  Brooks, III</span>  . </span><span class="cited-content_cbyCitation_article-title">An Evaluation of Explicit Receptor Flexibility in Molecular Docking Using Molecular Dynamics and Torsion Angle Molecular Dynamics. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Theory and Computation</span><span> <strong>2009,</strong> <em>5 </em>
                                    (10)
                                     , 2909-2923. <a href="https://doi.org/10.1021/ct900262t" title="DOI URL">https://doi.org/10.1021/ct900262t</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/ct900262t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fct900262t%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Theory%2520and%2520Computation%26atitle%3DAn%252BEvaluation%252Bof%252BExplicit%252BReceptor%252BFlexibility%252Bin%252BMolecular%252BDocking%252BUsing%252BMolecular%252BDynamics%252Band%252BTorsion%252BAngle%252BMolecular%252BDynamics%26aulast%3DArmen%26aufirst%3DRoger%2BS.%26date%3D2009%26date%3D2009%26date%3D2009%26date%3D21%252005%25202009%26date%3D14092009%26date%3D13102009%26volume%3D5%26issue%3D10%26spage%3D2909%26epage%3D2923" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Göran  Wallin</span>, <span class="hlFld-ContribAuthor ">Martin  Nervall</span>, <span class="hlFld-ContribAuthor ">Jens  Carlsson</span> and <span class="hlFld-ContribAuthor ">Johan  Åqvist</span>. </span><span class="cited-content_cbyCitation_article-title">Charges for Large Scale Binding Free Energy Calculations with the Linear Interaction Energy Method. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Theory and Computation</span><span> <strong>2009,</strong> <em>5 </em>
                                    (2)
                                     , 380-395. <a href="https://doi.org/10.1021/ct800404f" title="DOI URL">https://doi.org/10.1021/ct800404f</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/ct800404f&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fct800404f%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Theory%2520and%2520Computation%26atitle%3DCharges%252Bfor%252BLarge%252BScale%252BBinding%252BFree%252BEnergy%252BCalculations%252Bwith%252Bthe%252BLinear%252BInteraction%252BEnergy%252BMethod%26aulast%3DWallin%26aufirst%3DG%25C3%25B6ran%26date%3D2009%26date%3D2009%26date%3D2008%26date%3D10%252002%25202009%26date%3D14%252001%25202009%26date%3D26%252009%25202008%26volume%3D5%26issue%3D2%26spage%3D380%26epage%3D395" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ting  Zhou</span> and <span class="hlFld-ContribAuthor ">Amedeo  Caflisch</span>. </span><span class="cited-content_cbyCitation_article-title">Data Management System for Distributed Virtual Screening. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2009,</strong> <em>49 </em>
                                    (1)
                                     , 145-152. <a href="https://doi.org/10.1021/ci800295q" title="DOI URL">https://doi.org/10.1021/ci800295q</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/ci800295q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fci800295q%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DData%252BManagement%252BSystem%252Bfor%252BDistributed%252BVirtual%252BScreening%26aulast%3DZhou%26aufirst%3DTing%26date%3D2009%26date%3D2008%26date%3D2008%26date%3D26%252001%25202009%26date%3D12%252012%25202008%26date%3D22%252008%25202008%26volume%3D49%26issue%3D1%26spage%3D145%26epage%3D152" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ting  Zhou</span>, <span class="hlFld-ContribAuthor ">Danzhi  Huang</span> and <span class="hlFld-ContribAuthor ">Amedeo  Caflisch</span>. </span><span class="cited-content_cbyCitation_article-title">Is Quantum Mechanics Necessary for Predicting Binding Free Energy?. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2008,</strong> <em>51 </em>
                                    (14)
                                     , 4280-4288. <a href="https://doi.org/10.1021/jm800242q" title="DOI URL">https://doi.org/10.1021/jm800242q</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/jm800242q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fjm800242q%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DIs%252BQuantum%252BMechanics%252BNecessary%252Bfor%252BPredicting%252BBinding%252BFree%252BEnergy%25253F%26aulast%3DZhou%26aufirst%3DTing%26date%3D2008%26date%3D2008%26date%3D2008%26date%3D24%252007%25202008%26date%3D25%252006%25202008%26date%3D06%252003%25202008%26volume%3D51%26issue%3D14%26spage%3D4280%26epage%3D4288" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Chetan  Chintha</span>, <span class="hlFld-ContribAuthor ">Antonio  Carlesso</span>, <span class="hlFld-ContribAuthor ">Adrienne M.  Gorman</span>, <span class="hlFld-ContribAuthor ">Afshin  Samali</span>, <span class="hlFld-ContribAuthor ">Leif A.  Eriksson</span>. </span><span class="cited-content_cbyCitation_article-title">Molecular modeling provides a structural basis for PERK inhibitor selectivity towards RIPK1. </span><span class="cited-content_cbyCitation_journal-name">RSC Advances</span><span> <strong>2020,</strong> <em>10 </em>
                                    (1)
                                     , 367-375. <a href="https://doi.org/10.1039/C9RA08047C" title="DOI URL">https://doi.org/10.1039/C9RA08047C</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C9RA08047C&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC9RA08047C%26sid%3Dliteratum%253Aachs%26jtitle%3DRSC%2520Advances%26atitle%3DMolecular%252Bmodeling%252Bprovides%252Ba%252Bstructural%252Bbasis%252Bfor%252BPERK%252Binhibitor%252Bselectivity%252Btowards%252BRIPK1%26aulast%3DChintha%26aufirst%3DChetan%26date%3D2020%26date%3D2020%26volume%3D10%26issue%3D1%26spage%3D367%26epage%3D375" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">N.  Abelyan</span>, <span class="hlFld-ContribAuthor ">H.  Grabski</span>, <span class="hlFld-ContribAuthor ">S.  Tiratsuyan</span>. </span><span class="cited-content_cbyCitation_article-title">In silico Screening of Flavones and its Derivatives as Potential Inhibitors of Quorum-Sensing Regulator LasR of Pseudomonas aeruginosa. </span><span class="cited-content_cbyCitation_journal-name">Molecular Biology</span><span> <strong>2020,</strong> <em>54 </em>
                                    (1)
                                     , 134-143. <a href="https://doi.org/10.1134/S0026893320010021" title="DOI URL">https://doi.org/10.1134/S0026893320010021</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1134/S0026893320010021&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1134%2FS0026893320010021%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecular%2520Biology%26atitle%3DIn%252Bsilico%252BScreening%252Bof%252BFlavones%252Band%252Bits%252BDerivatives%252Bas%252BPotential%252BInhibitors%252Bof%252BQuorum-Sensing%252BRegulator%252BLasR%252Bof%252BPseudomonas%252Baeruginosa%26aulast%3DAbelyan%26aufirst%3DN.%26date%3D2020%26date%3D2020%26volume%3D54%26issue%3D1%26spage%3D134%26epage%3D143" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Balaguru  Ravikumar</span>, <span class="hlFld-ContribAuthor ">Sanna  Timonen</span>, <span class="hlFld-ContribAuthor ">Zaid  Alam</span>, <span class="hlFld-ContribAuthor ">Elina  Parri</span>, <span class="hlFld-ContribAuthor ">Krister  Wennerberg</span>, <span class="hlFld-ContribAuthor ">Tero  Aittokallio</span>. </span><span class="cited-content_cbyCitation_article-title">Chemogenomic Analysis of the Druggable Kinome and Its Application to Repositioning and Lead Identification Studies. </span><span class="cited-content_cbyCitation_journal-name">Cell Chemical Biology</span><span> <strong>2019,</strong> <em>26 </em>
                                    (11)
                                     , 1608-1622.e6. <a href="https://doi.org/10.1016/j.chembiol.2019.08.007" title="DOI URL">https://doi.org/10.1016/j.chembiol.2019.08.007</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.chembiol.2019.08.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.chembiol.2019.08.007%26sid%3Dliteratum%253Aachs%26jtitle%3DCell%2520Chemical%2520Biology%26atitle%3DChemogenomic%252BAnalysis%252Bof%252Bthe%252BDruggable%252BKinome%252Band%252BIts%252BApplication%252Bto%252BRepositioning%252Band%252BLead%252BIdentification%252BStudies%26aulast%3DRavikumar%26aufirst%3DBalaguru%26date%3D2019%26volume%3D26%26issue%3D11%26spage%3D1608%26epage%3D1622.e6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Gerhard  König</span>, <span class="hlFld-ContribAuthor ">Bernard R.  Brooks</span>, <span class="hlFld-ContribAuthor ">Walter  Thiel</span>, <span class="hlFld-ContribAuthor ">Darrin M.  York</span>. </span><span class="cited-content_cbyCitation_article-title">On the convergence of multi-scale free energy simulations. </span><span class="cited-content_cbyCitation_journal-name">Molecular Simulation</span><span> <strong>2018,</strong> <em>44 </em>
                                    (13-14)
                                     , 1062-1081. <a href="https://doi.org/10.1080/08927022.2018.1475741" title="DOI URL">https://doi.org/10.1080/08927022.2018.1475741</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/08927022.2018.1475741&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F08927022.2018.1475741%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecular%2520Simulation%26atitle%3DOn%252Bthe%252Bconvergence%252Bof%252Bmulti-scale%252Bfree%252Benergy%252Bsimulations%26aulast%3DK%25C3%25B6nig%26aufirst%3DGerhard%26date%3D2018%26date%3D2018%26volume%3D44%26issue%3D13-14%26spage%3D1062%26epage%3D1081" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jean-Rémy  Marchand</span>, <span class="hlFld-ContribAuthor ">Amedeo  Caflisch</span>. </span><span class="cited-content_cbyCitation_article-title">In silico fragment-based drug design with SEED. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2018,</strong> <em>156 </em>, 907-917. <a href="https://doi.org/10.1016/j.ejmech.2018.07.042" title="DOI URL">https://doi.org/10.1016/j.ejmech.2018.07.042</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2018.07.042&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2018.07.042%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DIn%252Bsilico%252Bfragment-based%252Bdrug%252Bdesign%252Bwith%252BSEED%26aulast%3DMarchand%26aufirst%3DJean-R%25C3%25A9my%26date%3D2018%26volume%3D156%26spage%3D907%26epage%3D917" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Paweł  Śledź</span>, <span class="hlFld-ContribAuthor ">Amedeo  Caflisch</span>. </span><span class="cited-content_cbyCitation_article-title">Protein structure-based drug design: from docking to molecular dynamics. </span><span class="cited-content_cbyCitation_journal-name">Current Opinion in Structural Biology</span><span> <strong>2018,</strong> <em>48 </em>, 93-102. <a href="https://doi.org/10.1016/j.sbi.2017.10.010" title="DOI URL">https://doi.org/10.1016/j.sbi.2017.10.010</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.sbi.2017.10.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.sbi.2017.10.010%26sid%3Dliteratum%253Aachs%26jtitle%3DCurrent%2520Opinion%2520in%2520Structural%2520Biology%26atitle%3DProtein%252Bstructure-based%252Bdrug%252Bdesign%25253A%252Bfrom%252Bdocking%252Bto%252Bmolecular%252Bdynamics%26aulast%3D%25C5%259Aled%25C5%25BA%26aufirst%3DPawe%25C5%2582%26date%3D2018%26volume%3D48%26spage%3D93%26epage%3D102" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Nicolas  Panel</span>, <span class="hlFld-ContribAuthor ">Young Joo  Sun</span>, <span class="hlFld-ContribAuthor ">Ernesto J.  Fuentes</span>, <span class="hlFld-ContribAuthor ">Thomas  Simonson</span>. </span><span class="cited-content_cbyCitation_article-title">A Simple PB/LIE Free Energy Function Accurately Predicts the Peptide Binding Specificity of the Tiam1 PDZ Domain. </span><span class="cited-content_cbyCitation_journal-name">Frontiers in Molecular Biosciences</span><span> <strong>2017,</strong> <em>4 </em><a href="https://doi.org/10.3389/fmolb.2017.00065" title="DOI URL">https://doi.org/10.3389/fmolb.2017.00065</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3389/fmolb.2017.00065&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3389%2Ffmolb.2017.00065%26sid%3Dliteratum%253Aachs%26jtitle%3DFrontiers%2520in%2520Molecular%2520Biosciences%26atitle%3DA%252BSimple%252BPB%25252FLIE%252BFree%252BEnergy%252BFunction%252BAccurately%252BPredicts%252Bthe%252BPeptide%252BBinding%252BSpecificity%252Bof%252Bthe%252BTiam1%252BPDZ%252BDomain%26aulast%3DPanel%26aufirst%3DNicolas%26date%3D2017%26date%3D2017%26volume%3D4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Antonio  Peón</span>, <span class="hlFld-ContribAuthor ">Adrián  Robles</span>, <span class="hlFld-ContribAuthor ">Beatriz  Blanco</span>, <span class="hlFld-ContribAuthor ">Marino  Convertino</span>, <span class="hlFld-ContribAuthor ">Paul  Thompson</span>, <span class="hlFld-ContribAuthor ">Alastair R.  Hawkins</span>, <span class="hlFld-ContribAuthor ">Amedeo  Caflisch</span>, <span class="hlFld-ContribAuthor ">Concepción  González-Bello</span>. </span><span class="cited-content_cbyCitation_article-title">Reducing the Flexibility of Type II Dehydroquinase for Inhibition: A Fragment-Based Approach and Molecular Dynamics Study. </span><span class="cited-content_cbyCitation_journal-name">ChemMedChem</span><span> <strong>2017,</strong> <em>12 </em>
                                    (18)
                                     , 1512-1524. <a href="https://doi.org/10.1002/cmdc.201700396" title="DOI URL">https://doi.org/10.1002/cmdc.201700396</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/cmdc.201700396&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fcmdc.201700396%26sid%3Dliteratum%253Aachs%26jtitle%3DChemMedChem%26atitle%3DReducing%252Bthe%252BFlexibility%252Bof%252BType%2525E2%252580%252585II%252BDehydroquinase%252Bfor%252BInhibition%25253A%252BA%252BFragment-Based%252BApproach%252Band%252BMolecular%252BDynamics%252BStudy%26aulast%3DPe%25C3%25B3n%26aufirst%3DAntonio%26date%3D2017%26date%3D2017%26volume%3D12%26issue%3D18%26spage%3D1512%26epage%3D1524" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Vasanthanathan  Poongavanam</span>, <span class="hlFld-ContribAuthor ">Jacob  Kongsted</span>. </span><span class="cited-content_cbyCitation_article-title">Binding affinity models for Falcipain inhibition based on the Linear Interaction Energy method. </span><span class="cited-content_cbyCitation_journal-name">Journal of Molecular Graphics and Modelling</span><span> <strong>2016,</strong> <em>70 </em>, 236-245. <a href="https://doi.org/10.1016/j.jmgm.2016.06.008" title="DOI URL">https://doi.org/10.1016/j.jmgm.2016.06.008</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.jmgm.2016.06.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.jmgm.2016.06.008%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Molecular%2520Graphics%2520and%2520Modelling%26atitle%3DBinding%252Baffinity%252Bmodels%252Bfor%252BFalcipain%252Binhibition%252Bbased%252Bon%252Bthe%252BLinear%252BInteraction%252BEnergy%252Bmethod%26aulast%3DPoongavanam%26aufirst%3DVasanthanathan%26date%3D2016%26volume%3D70%26spage%3D236%26epage%3D245" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Erik Rosendahl  Kjellgren</span>, <span class="hlFld-ContribAuthor ">Oliver Emil Skytte  Glue</span>, <span class="hlFld-ContribAuthor ">Peter  Reinholdt</span>, <span class="hlFld-ContribAuthor ">Julie Egeskov  Meyer</span>, <span class="hlFld-ContribAuthor ">Jacob  Kongsted</span>, <span class="hlFld-ContribAuthor ">Vasanthanathan  Poongavanam</span>. </span><span class="cited-content_cbyCitation_article-title">A comparative study of binding affinities for 6,7-dimethoxy-4-pyrrolidylquinazolines as phosphodiesterase 10A inhibitors using the linear interaction energy method. </span><span class="cited-content_cbyCitation_journal-name">Journal of Molecular Graphics and Modelling</span><span> <strong>2015,</strong> <em>61 </em>, 44-52. <a href="https://doi.org/10.1016/j.jmgm.2015.06.010" title="DOI URL">https://doi.org/10.1016/j.jmgm.2015.06.010</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.jmgm.2015.06.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.jmgm.2015.06.010%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Molecular%2520Graphics%2520and%2520Modelling%26atitle%3DA%252Bcomparative%252Bstudy%252Bof%252Bbinding%252Baffinities%252Bfor%252B6%25252C7-dimethoxy-4-pyrrolidylquinazolines%252Bas%252Bphosphodiesterase%252B10A%252Binhibitors%252Busing%252Bthe%252Blinear%252Binteraction%252Benergy%252Bmethod%26aulast%3DKjellgren%26aufirst%3DErik%2BRosendahl%26date%3D2015%26volume%3D61%26spage%3D44%26epage%3D52" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Hongtao  Zhao</span>, <span class="hlFld-ContribAuthor ">Amedeo  Caflisch</span>. </span><span class="cited-content_cbyCitation_article-title">Molecular dynamics in drug design. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2015,</strong> <em>91 </em>, 4-14. <a href="https://doi.org/10.1016/j.ejmech.2014.08.004" title="DOI URL">https://doi.org/10.1016/j.ejmech.2014.08.004</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2014.08.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2014.08.004%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DMolecular%252Bdynamics%252Bin%252Bdrug%252Bdesign%26aulast%3DZhao%26aufirst%3DHongtao%26date%3D2015%26volume%3D91%26spage%3D4%26epage%3D14" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Andrej  Perdih</span>, <span class="hlFld-ContribAuthor ">Gerhard  Wolber</span>, <span class="hlFld-ContribAuthor ">Tom  Solmajer</span>. </span><span class="cited-content_cbyCitation_article-title">Molecular dynamics simulation and linear interaction energy study of d-Glu-based inhibitors of the MurD ligase. </span><span class="cited-content_cbyCitation_journal-name">Journal of Computer-Aided Molecular Design</span><span> <strong>2013,</strong> <em>27 </em>
                                    (8)
                                     , 723-738. <a href="https://doi.org/10.1007/s10822-013-9673-3" title="DOI URL">https://doi.org/10.1007/s10822-013-9673-3</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s10822-013-9673-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs10822-013-9673-3%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Computer-Aided%2520Molecular%2520Design%26atitle%3DMolecular%252Bdynamics%252Bsimulation%252Band%252Blinear%252Binteraction%252Benergy%252Bstudy%252Bof%252Bd-Glu-based%252Binhibitors%252Bof%252Bthe%252BMurD%252Bligase%26aulast%3DPerdih%26aufirst%3DAndrej%26date%3D2013%26date%3D2013%26volume%3D27%26issue%3D8%26spage%3D723%26epage%3D738" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Mingyun  Shen</span>, <span class="hlFld-ContribAuthor ">Huidong  Yu</span>, <span class="hlFld-ContribAuthor ">Youyong  Li</span>, <span class="hlFld-ContribAuthor ">Pixu  Li</span>, <span class="hlFld-ContribAuthor ">Peichen  Pan</span>, <span class="hlFld-ContribAuthor ">Shunye  Zhou</span>, <span class="hlFld-ContribAuthor ">Liling  Zhang</span>, <span class="hlFld-ContribAuthor ">Shang  Li</span>, <span class="hlFld-ContribAuthor ">Simon Ming-Yuen  Lee</span>, <span class="hlFld-ContribAuthor ">Tingjun  Hou</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of Rho-kinase inhibitors by docking-based virtual screening. </span><span class="cited-content_cbyCitation_journal-name">Molecular BioSystems</span><span> <strong>2013,</strong> <em>9 </em>
                                    (6)
                                     , 1511. <a href="https://doi.org/10.1039/c3mb00016h" title="DOI URL">https://doi.org/10.1039/c3mb00016h</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/c3mb00016h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2Fc3mb00016h%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecular%2520BioSystems%26atitle%3DDiscovery%252Bof%252BRho-kinase%252Binhibitors%252Bby%252Bdocking-based%252Bvirtual%252Bscreening%26aulast%3DShen%26aufirst%3DMingyun%26date%3D2013%26volume%3D9%26issue%3D6%26spage%3D1511" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Hongtao  Zhao</span>, <span class="hlFld-ContribAuthor ">Danzhi  Huang</span>, <span class="hlFld-ContribAuthor ">Amedeo  Caflisch</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of Tyrosine Kinase Inhibitors by Docking into an Inactive Kinase Conformation Generated by Molecular Dynamics. </span><span class="cited-content_cbyCitation_journal-name">ChemMedChem</span><span> <strong>2012,</strong> <em>7 </em>
                                    (11)
                                     , 1983-1990. <a href="https://doi.org/10.1002/cmdc.201200331" title="DOI URL">https://doi.org/10.1002/cmdc.201200331</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/cmdc.201200331&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fcmdc.201200331%26sid%3Dliteratum%253Aachs%26jtitle%3DChemMedChem%26atitle%3DDiscovery%252Bof%252BTyrosine%252BKinase%252BInhibitors%252Bby%252BDocking%252Binto%252Ban%252BInactive%252BKinase%252BConformation%252BGenerated%252Bby%252BMolecular%252BDynamics%26aulast%3DZhao%26aufirst%3DHongtao%26date%3D2012%26date%3D2012%26volume%3D7%26issue%3D11%26spage%3D1983%26epage%3D1990" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Danzhi  Huang</span>. </span><span class="cited-content_cbyCitation_article-title">High-Throughput Virtual Screening Lead to Discovery of Non-Peptidic Inhibitors of West Nile Virus NS3 Protease. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2012,</strong>,, 615-623. <a href="https://doi.org/10.1007/978-1-61779-465-0_36" title="DOI URL">https://doi.org/10.1007/978-1-61779-465-0_36</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/978-1-61779-465-0_36&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2F978-1-61779-465-0_36%26sid%3Dliteratum%253Aachs%26atitle%3DHigh-Throughput%252BVirtual%252BScreening%252BLead%252Bto%252BDiscovery%252Bof%252BNon-Peptidic%252BInhibitors%252Bof%252BWest%252BNile%252BVirus%252BNS3%252BProtease%26aulast%3DHuang%26aufirst%3DDanzhi%26date%3D2012%26date%3D2011%26spage%3D615%26epage%3D623%26pub%3DSpringer%2520New%2520York%26atitle%3DComputational%252BDrug%252BDiscovery%252Band%252BDesign%26aulast%3DBaron%26aufirst%3DRiccardo%26date%3D2012%26volume%3D819" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Toan B.  Nguyen</span>, <span class="hlFld-ContribAuthor ">Sergio E.  Wong</span>, <span class="hlFld-ContribAuthor ">Felice C.  Lightstone</span>. </span><span class="cited-content_cbyCitation_article-title">Leveraging Structural Information for the Discovery of New Drugs: Computational Methods. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2012,</strong>,, 209-234. <a href="https://doi.org/10.1007/978-1-61779-520-6_9" title="DOI URL">https://doi.org/10.1007/978-1-61779-520-6_9</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/978-1-61779-520-6_9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2F978-1-61779-520-6_9%26sid%3Dliteratum%253Aachs%26atitle%3DLeveraging%252BStructural%252BInformation%252Bfor%252Bthe%252BDiscovery%252Bof%252BNew%252BDrugs%25253A%252BComputational%252BMethods%26aulast%3DNguyen%26aufirst%3DToan%2BB.%26date%3D2012%26date%3D2011%26spage%3D209%26epage%3D234%26pub%3DHumana%2520Press%26atitle%3DStructure-Based%252BDrug%252BDiscovery%26aulast%3DTari%26aufirst%3DLeslie%2BW.%26date%3D2012%26volume%3D841" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Vincent  Zoete</span>, <span class="hlFld-ContribAuthor ">Michel A.  Cuendet</span>, <span class="hlFld-ContribAuthor ">Aurélien  Grosdidier</span>, <span class="hlFld-ContribAuthor ">Olivier  Michielin</span>. </span><span class="cited-content_cbyCitation_article-title">SwissParam: A fast force field generation tool for small organic molecules. </span><span class="cited-content_cbyCitation_journal-name">Journal of Computational Chemistry</span><span> <strong>2011,</strong> <em>32 </em>
                                    (11)
                                     , 2359-2368. <a href="https://doi.org/10.1002/jcc.21816" title="DOI URL">https://doi.org/10.1002/jcc.21816</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/jcc.21816&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fjcc.21816%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Computational%2520Chemistry%26atitle%3DSwissParam%25253A%252BA%252Bfast%252Bforce%252Bfield%252Bgeneration%252Btool%252Bfor%252Bsmall%252Borganic%252Bmolecules%26aulast%3DZoete%26aufirst%3DVincent%26date%3D2011%26date%3D2011%26volume%3D32%26issue%3D11%26spage%3D2359%26epage%3D2368" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Hongtao  Zhao</span>, <span class="hlFld-ContribAuthor ">Danzhi  Huang</span>, . </span><span class="cited-content_cbyCitation_article-title">Hydrogen Bonding Penalty upon Ligand Binding. </span><span class="cited-content_cbyCitation_journal-name">PLoS ONE</span><span> <strong>2011,</strong> <em>6 </em>
                                    (6)
                                     , e19923. <a href="https://doi.org/10.1371/journal.pone.0019923" title="DOI URL">https://doi.org/10.1371/journal.pone.0019923</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1371/journal.pone.0019923&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1371%2Fjournal.pone.0019923%26sid%3Dliteratum%253Aachs%26jtitle%3DPLoS%2520ONE%26atitle%3DHydrogen%252BBonding%252BPenalty%252Bupon%252BLigand%252BBinding%26aulast%3DZhao%26aufirst%3DHongtao%26date%3D2011%26date%3D2011%26volume%3D6%26issue%3D6%26spage%3De19923" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Hassan M.  Shallal</span>, <span class="hlFld-ContribAuthor ">Wade A.  Russu</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery, synthesis, and investigation of the antitumor activity of novel piperazinylpyrimidine derivatives. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2011,</strong> <em>46 </em>
                                    (6)
                                     , 2043-2057. <a href="https://doi.org/10.1016/j.ejmech.2011.02.057" title="DOI URL">https://doi.org/10.1016/j.ejmech.2011.02.057</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2011.02.057&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2011.02.057%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%25252C%252Bsynthesis%25252C%252Band%252Binvestigation%252Bof%252Bthe%252Bantitumor%252Bactivity%252Bof%252Bnovel%252Bpiperazinylpyrimidine%252Bderivatives%26aulast%3DShallal%26aufirst%3DHassan%2BM.%26date%3D2011%26volume%3D46%26issue%3D6%26spage%3D2043%26epage%3D2057" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Danzhi  Huang</span>, <span class="hlFld-ContribAuthor ">Amedeo  Caflisch</span>. </span><span class="cited-content_cbyCitation_article-title">Fragment-Based Approaches in Virtual Screening. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2011,</strong>,, 467-489. <a href="https://doi.org/10.1002/9783527633326.ch17" title="DOI URL">https://doi.org/10.1002/9783527633326.ch17</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/9783527633326.ch17&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2F9783527633326.ch17%26sid%3Dliteratum%253Aachs%26atitle%3DFragment-Based%252BApproaches%252Bin%252BVirtual%252BScreening%26aulast%3DHuang%26aufirst%3DDanzhi%26date%3D2011%26date%3D2011%26spage%3D467%26epage%3D489%26pub%3DWiley-VCH%2520Verlag%2520GmbH%2520%2526%2520Co.%2520KGaA%26atitle%3DVirtual%252BScreening%26aulast%3DSotriffer%26aufirst%3DChristoph%26date%3D2011%26volume%3D51" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Shu  Liu</span>, <span class="hlFld-ContribAuthor ">Li-Hua  Zhou</span>, <span class="hlFld-ContribAuthor ">Hua-Qiao  Wang</span>, <span class="hlFld-ContribAuthor ">Zhi-Bin  Yao</span>. </span><span class="cited-content_cbyCitation_article-title">Superimposing the 27 crystal protein/inhibitor complexes of β-secretase to calculate the binding affinities by the linear interaction energy method. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry Letters</span><span> <strong>2010,</strong> <em>20 </em>
                                    (22)
                                     , 6533-6537. <a href="https://doi.org/10.1016/j.bmcl.2010.09.050" title="DOI URL">https://doi.org/10.1016/j.bmcl.2010.09.050</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmcl.2010.09.050&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmcl.2010.09.050%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DSuperimposing%252Bthe%252B27%252Bcrystal%252Bprotein%25252Finhibitor%252Bcomplexes%252Bof%252B%2525CE%2525B2-secretase%252Bto%252Bcalculate%252Bthe%252Bbinding%252Baffinities%252Bby%252Bthe%252Blinear%252Binteraction%252Benergy%252Bmethod%26aulast%3DLiu%26aufirst%3DShu%26date%3D2010%26volume%3D20%26issue%3D22%26spage%3D6533%26epage%3D6537" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ting  Zhou</span>, <span class="hlFld-ContribAuthor ">Amedeo  Caflisch</span>. </span><span class="cited-content_cbyCitation_article-title">High-Throughput Virtual Screening Using Quantum Mechanical Probes: Discovery of Selective Kinase Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">ChemMedChem</span><span> <strong>2010,</strong> <em>5 </em>
                                    (7)
                                     , 1007-1014. <a href="https://doi.org/10.1002/cmdc.201000085" title="DOI URL">https://doi.org/10.1002/cmdc.201000085</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/cmdc.201000085&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fcmdc.201000085%26sid%3Dliteratum%253Aachs%26jtitle%3DChemMedChem%26atitle%3DHigh-Throughput%252BVirtual%252BScreening%252BUsing%252BQuantum%252BMechanical%252BProbes%25253A%252BDiscovery%252Bof%252BSelective%252BKinase%252BInhibitors%26aulast%3DZhou%26aufirst%3DTing%26date%3D2010%26date%3D2010%26volume%3D5%26issue%3D7%26spage%3D1007%26epage%3D1014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Natasja  Brooijmans</span>, <span class="hlFld-ContribAuthor ">Christine  Humblet</span>. </span><span class="cited-content_cbyCitation_article-title">Chemical space sampling by different scoring functions and crystal structures. </span><span class="cited-content_cbyCitation_journal-name">Journal of Computer-Aided Molecular Design</span><span> <strong>2010,</strong> <em>24 </em>
                                    (5)
                                     , 433-447. <a href="https://doi.org/10.1007/s10822-010-9356-2" title="DOI URL">https://doi.org/10.1007/s10822-010-9356-2</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s10822-010-9356-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs10822-010-9356-2%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Computer-Aided%2520Molecular%2520Design%26atitle%3DChemical%252Bspace%252Bsampling%252Bby%252Bdifferent%252Bscoring%252Bfunctions%252Band%252Bcrystal%252Bstructures%26aulast%3DBrooijmans%26aufirst%3DNatasja%26date%3D2010%26date%3D2010%26volume%3D24%26issue%3D5%26spage%3D433%26epage%3D447" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Orazio  Nicolotti</span>, <span class="hlFld-ContribAuthor ">Ilenia  Giangreco</span>, <span class="hlFld-ContribAuthor ">Teresa Fabiola  Miscioscia</span>, <span class="hlFld-ContribAuthor ">Marino  Convertino</span>, <span class="hlFld-ContribAuthor ">Francesco  Leonetti</span>, <span class="hlFld-ContribAuthor ">Leonardo  Pisani</span>, <span class="hlFld-ContribAuthor ">Angelo  Carotti</span>. </span><span class="cited-content_cbyCitation_article-title">Screening of benzamidine-based thrombin inhibitors via a linear interaction energy in continuum electrostatics model. </span><span class="cited-content_cbyCitation_journal-name">Journal of Computer-Aided Molecular Design</span><span> <strong>2010,</strong> <em>24 </em>
                                    (2)
                                     , 117-129. <a href="https://doi.org/10.1007/s10822-010-9320-1" title="DOI URL">https://doi.org/10.1007/s10822-010-9320-1</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s10822-010-9320-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs10822-010-9320-1%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Computer-Aided%2520Molecular%2520Design%26atitle%3DScreening%252Bof%252Bbenzamidine-based%252Bthrombin%252Binhibitors%252Bvia%252Ba%252Blinear%252Binteraction%252Benergy%252Bin%252Bcontinuum%252Belectrostatics%252Bmodel%26aulast%3DNicolotti%26aufirst%3DOrazio%26date%3D2010%26date%3D2010%26volume%3D24%26issue%3D2%26spage%3D117%26epage%3D129" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Chaya S.  Rapp</span>, <span class="hlFld-ContribAuthor ">Cheryl  Schonbrun</span>, <span class="hlFld-ContribAuthor ">Matthew P.  Jacobson</span>, <span class="hlFld-ContribAuthor ">Chakrapani  Kalyanaraman</span>, <span class="hlFld-ContribAuthor ">Niu  Huang</span>. </span><span class="cited-content_cbyCitation_article-title">Automated site preparation in physics-based rescoring of receptor ligand complexes. </span><span class="cited-content_cbyCitation_journal-name">Proteins: Structure, Function, and Bioinformatics</span><span> <strong>2009,</strong> <em>77 </em>
                                    (1)
                                     , 52-61. <a href="https://doi.org/10.1002/prot.22415" title="DOI URL">https://doi.org/10.1002/prot.22415</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/prot.22415&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fprot.22415%26sid%3Dliteratum%253Aachs%26jtitle%3DProteins%253A%2520Structure%252C%2520Function%252C%2520and%2520Bioinformatics%26atitle%3DAutomated%252Bsite%252Bpreparation%252Bin%252Bphysics-based%252Brescoring%252Bof%252Breceptor%252Bligand%252Bcomplexes%26aulast%3DRapp%26aufirst%3DChaya%2BS.%26date%3D2009%26volume%3D77%26issue%3D1%26spage%3D52%26epage%3D61" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ran  Friedman</span>, <span class="hlFld-ContribAuthor ">Amedeo  Caflisch</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of Plasmepsin Inhibitors by Fragment-Based Docking and Consensus Scoring. </span><span class="cited-content_cbyCitation_journal-name">ChemMedChem</span><span> <strong>2009,</strong> <em>4 </em>
                                    (8)
                                     , 1317-1326. <a href="https://doi.org/10.1002/cmdc.200900078" title="DOI URL">https://doi.org/10.1002/cmdc.200900078</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/cmdc.200900078&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fcmdc.200900078%26sid%3Dliteratum%253Aachs%26jtitle%3DChemMedChem%26atitle%3DDiscovery%252Bof%252BPlasmepsin%252BInhibitors%252Bby%252BFragment-Based%252BDocking%252Band%252BConsensus%252BScoring%26aulast%3DFriedman%26aufirst%3DRan%26date%3D2009%26date%3D2009%26volume%3D4%26issue%3D8%26spage%3D1317%26epage%3D1326" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Andrej  Perdih</span>, <span class="hlFld-ContribAuthor ">Urban  Bren</span>, <span class="hlFld-ContribAuthor ">Tom  Solmajer</span>. </span><span class="cited-content_cbyCitation_article-title">Binding free energy calculations of N-sulphonyl-glutamic acid inhibitors of MurD ligase. </span><span class="cited-content_cbyCitation_journal-name">Journal of Molecular Modeling</span><span> <strong>2009,</strong> <em>15 </em>
                                    (8)
                                     , 983-996. <a href="https://doi.org/10.1007/s00894-009-0455-8" title="DOI URL">https://doi.org/10.1007/s00894-009-0455-8</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s00894-009-0455-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs00894-009-0455-8%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Molecular%2520Modeling%26atitle%3DBinding%252Bfree%252Benergy%252Bcalculations%252Bof%252BN-sulphonyl-glutamic%252Bacid%252Binhibitors%252Bof%252BMurD%252Bligase%26aulast%3DPerdih%26aufirst%3DAndrej%26date%3D2009%26date%3D2009%26volume%3D15%26issue%3D8%26spage%3D983%26epage%3D996" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Vincent  Zoete</span>, <span class="hlFld-ContribAuthor ">Aurélien  Grosdidier</span>, <span class="hlFld-ContribAuthor ">Olivier  Michielin</span>. </span><span class="cited-content_cbyCitation_article-title">Docking, virtual high throughput screening and in silico fragment-based drug design. </span><span class="cited-content_cbyCitation_journal-name">Journal of Cellular and Molecular Medicine</span><span> <strong>2009,</strong> <em>13 </em>
                                    (2)
                                     , 238-248. <a href="https://doi.org/10.1111/j.1582-4934.2008.00665.x" title="DOI URL">https://doi.org/10.1111/j.1582-4934.2008.00665.x</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1111/j.1582-4934.2008.00665.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1111%2Fj.1582-4934.2008.00665.x%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Cellular%2520and%2520Molecular%2520Medicine%26atitle%3DDocking%25252C%252Bvirtual%252Bhigh%252Bthroughput%252Bscreening%252Band%252Bin%252Bsilico%252Bfragment-based%252Bdrug%252Bdesign%26aulast%3DZoete%26aufirst%3DVincent%26date%3D2009%26date%3D2009%26volume%3D13%26issue%3D2%26spage%3D238%26epage%3D248" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Dariusz  Ekonomiuk</span>, <span class="hlFld-ContribAuthor ">Xun-Cheng  Su</span>, <span class="hlFld-ContribAuthor ">Kiyoshi  Ozawa</span>, <span class="hlFld-ContribAuthor ">Christophe  Bodenreider</span>, <span class="hlFld-ContribAuthor ">Siew Pheng  Lim</span>, <span class="hlFld-ContribAuthor ">Zheng  Yin</span>, <span class="hlFld-ContribAuthor ">Thomas H.  Keller</span>, <span class="hlFld-ContribAuthor ">David  Beer</span>, <span class="hlFld-ContribAuthor ">Viral  Patel</span>, <span class="hlFld-ContribAuthor ">Gottfried  Otting</span>, <span class="hlFld-ContribAuthor ">Amedeo  Caflisch</span>, <span class="hlFld-ContribAuthor ">Danzhi  Huang</span>, . </span><span class="cited-content_cbyCitation_article-title">Discovery of a Non-Peptidic Inhibitor of West Nile Virus NS3 Protease by High-Throughput Docking. </span><span class="cited-content_cbyCitation_journal-name">PLoS Neglected Tropical Diseases</span><span> <strong>2009,</strong> <em>3 </em>
                                    (1)
                                     , e356. <a href="https://doi.org/10.1371/journal.pntd.0000356" title="DOI URL">https://doi.org/10.1371/journal.pntd.0000356</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1371/journal.pntd.0000356&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1371%2Fjournal.pntd.0000356%26sid%3Dliteratum%253Aachs%26jtitle%3DPLoS%2520Neglected%2520Tropical%2520Diseases%26atitle%3DDiscovery%252Bof%252Ba%252BNon-Peptidic%252BInhibitor%252Bof%252BWest%252BNile%252BVirus%252BNS3%252BProtease%252Bby%252BHigh-Throughput%252BDocking%26aulast%3DEkonomiuk%26aufirst%3DDariusz%26date%3D2009%26date%3D2009%26volume%3D3%26issue%3D1%26spage%3De356" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Danzhi  Huang</span>, <span class="hlFld-ContribAuthor ">Amedeo  Caflisch</span>. </span><span class="cited-content_cbyCitation_article-title">Library screening by fragment-based docking. </span><span class="cited-content_cbyCitation_journal-name">Journal of Molecular Recognition</span><span> <strong>2009,</strong> <em>11 </em>, n/a-n/a. <a href="https://doi.org/10.1002/jmr.981" title="DOI URL">https://doi.org/10.1002/jmr.981</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/jmr.981&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fjmr.981%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Molecular%2520Recognition%26atitle%3DLibrary%252Bscreening%252Bby%252Bfragment-based%252Bdocking%26aulast%3DHuang%26aufirst%3DDanzhi%26date%3D2009%26volume%3D11%26spage%3Dn%252Fa%26epage%3Dn%252Fa" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Patricia  Schenker</span>, <span class="hlFld-ContribAuthor ">Pietro  Alfarano</span>, <span class="hlFld-ContribAuthor ">Peter  Kolb</span>, <span class="hlFld-ContribAuthor ">Amedeo  Caflisch</span>, <span class="hlFld-ContribAuthor ">Antonio  Baici</span>. </span><span class="cited-content_cbyCitation_article-title">A double‐headed cathepsin B inhibitor devoid of warhead. </span><span class="cited-content_cbyCitation_journal-name">Protein Science</span><span> <strong>2008,</strong> <em>17 </em>
                                    (12)
                                     , 2145-2155. <a href="https://doi.org/10.1110/ps.037341.108" title="DOI URL">https://doi.org/10.1110/ps.037341.108</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1110/ps.037341.108&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1110%2Fps.037341.108%26sid%3Dliteratum%253Aachs%26jtitle%3DProtein%2520Science%26atitle%3DA%252Bdouble%2525E2%252580%252590headed%252Bcathepsin%252BB%252Binhibitor%252Bdevoid%252Bof%252Bwarhead%26aulast%3DSchenker%26aufirst%3DPatricia%26date%3D2008%26date%3D2009%26volume%3D17%26issue%3D12%26spage%3D2145%26epage%3D2155" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Peter  Kolb</span>, <span class="hlFld-ContribAuthor ">Catherine Berset  Kipouros</span>, <span class="hlFld-ContribAuthor ">Danzhi  Huang</span>, <span class="hlFld-ContribAuthor ">Amedeo  Caflisch</span>. </span><span class="cited-content_cbyCitation_article-title">Structure-based tailoring of compound libraries for high-throughput screening: Discovery of novel EphB4 kinase inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Proteins: Structure, Function, and Bioinformatics</span><span> <strong>2008,</strong> <em>73 </em>
                                    (1)
                                     , 11-18. <a href="https://doi.org/10.1002/prot.22028" title="DOI URL">https://doi.org/10.1002/prot.22028</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/prot.22028&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fprot.22028%26sid%3Dliteratum%253Aachs%26jtitle%3DProteins%253A%2520Structure%252C%2520Function%252C%2520and%2520Bioinformatics%26atitle%3DStructure-based%252Btailoring%252Bof%252Bcompound%252Blibraries%252Bfor%252Bhigh-throughput%252Bscreening%25253A%252BDiscovery%252Bof%252Bnovel%252BEphB4%252Bkinase%252Binhibitors%26aulast%3DKolb%26aufirst%3DPeter%26date%3D2008%26date%3D2008%26volume%3D73%26issue%3D1%26spage%3D11%26epage%3D18" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Fabian  Dey</span>, <span class="hlFld-ContribAuthor ">Amedeo  Caflisch</span>. </span><span class="cited-content_cbyCitation_article-title">Fragment-Based de Novo Ligand Design by Multiobjective Evolutionary Optimization. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2008,</strong> <em>48 </em>
                                    (3)
                                     , 679-690. <a href="https://doi.org/10.1021/ci700424b" title="DOI URL">https://doi.org/10.1021/ci700424b</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/ci700424b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fci700424b%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DFragment-Based%252Bde%252BNovo%252BLigand%252BDesign%252Bby%252BMultiobjective%252BEvolutionary%252BOptimization%26aulast%3DDey%26aufirst%3DFabian%26date%3D2008%26date%3D2008%26volume%3D48%26issue%3D3%26spage%3D679%26epage%3D690" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="f_null" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2008/jmcmar.2008.51.issue-5/jm070654j/production/images/medium/jm-2007-00654j_0007.gif" alt="" id="_i2" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2008/jmcmar.2008.51.issue-5/jm070654j/production/images/large/jm-2007-00654j_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm070654j&amp;id=f_null"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2008/jmcmar.2008.51.issue-5/jm070654j/production/images/medium/jm-2007-00654j_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2008/jmcmar.2008.51.issue-5/jm070654j/production/images/large/jm-2007-00654j_0001.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Distributions of some key properties of the five sets of inhibitors. All values have been calculated with DAIM.<a onclick="showRef(event, 'ref52'); return false;" href="javascript:void(0);" class="ref ref52">(52)</a></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2008/jmcmar.2008.51.issue-5/jm070654j/production/images/large/jm-2007-00654j_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm070654j&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2008/jmcmar.2008.51.issue-5/jm070654j/production/images/medium/jm-2007-00654j_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2008/jmcmar.2008.51.issue-5/jm070654j/production/images/large/jm-2007-00654j_0002.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. LIECE two-parameter CDK2-Lck-p38-model, with experimental Δ<i>G</i> = <i>RT</i> ln(IC<sub>50</sub>). (A) Data used for fitting are 73, 51, and 41 inhibitors of CDK2, Lck, and p38, respectively. Cross-validation was done on 37 inhibitors of EphB4 (B) and 128 inhibitors of EGFR (C). The green diagonal is the ideal line of perfect prediction. The black diagonals delimit the 1 kcal/mol error region. All values are in kcal/mol.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2008/jmcmar.2008.51.issue-5/jm070654j/production/images/large/jm-2007-00654j_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm070654j&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2008/jmcmar.2008.51.issue-5/jm070654j/production/images/medium/jm-2007-00654j_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2008/jmcmar.2008.51.issue-5/jm070654j/production/images/large/jm-2007-00654j_0003.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Scatter plot of the van der Waals interaction energy between a C═O group and a kinase at 81 grid points in the ATP-binding site. Regression lines are solid black, and the values of <i>r</i> are the correlation coefficients. All values are in kcal/mol.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2008/jmcmar.2008.51.issue-5/jm070654j/production/images/large/jm-2007-00654j_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm070654j&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2008/jmcmar.2008.51.issue-5/jm070654j/production/images/medium/jm-2007-00654j_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2008/jmcmar.2008.51.issue-5/jm070654j/production/images/large/jm-2007-00654j_0004.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. ROC plot for CDK2 showing improvement of the retrieval rate of known inhibitors by using filters. A set of 40 375 compounds from the ZINC library was used as starting database of inactive molecules. This set of 40 375 compounds was seeded with 73 known inhibitors, and the filters described below were applied to both ZINC compounds and inhibitors. Poses were first filtered according to an <i>E</i><sub>vdW</sub> ≤ –35 kcal/mol and an <i>E</i><sub>vdw</sub>/MW ≤ –0.1 kcal/g, which yielded 171 898 poses of 14 701 ZINC compounds. These 14 701 compounds and the 48 inhibitors that passed the above filter make up the set to which all subsequent filters (see also Table <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a>) were applied. (Black curve) The LIECE two-parameter CDK2-model was used to rank the combined library consisting of 48 known inhibitors of CDK2 and the pose with the most favorable LIECE energy of each of the 14 701 compounds. The area under this curve is <i>A</i><sub>ROC</sub> = 0.76. (Red curve) Forty-eight known inhibitors and 7793 compounds with Coulombic efficiency more favorable than −0.01 kcal/g, <i>A</i><sub>ROC</sub> = 0.79. (Green curve) Forty-eight known inhibitors and 6716 compounds with at least one hydrogen bond to the hinge region, <i>A</i><sub>ROC</sub> = 0.82. (Blue curve) Thirty-two known inhibitors and 11 842 compounds with van der Waals efficiency more favorable than −0.11 kcal/g, <i>A</i><sub>ROC</sub> = 0.81. (Orange curve) Thirty-two known inhibitors and 2654 compounds that satisfy all aforementioned filters, <i>A</i><sub>ROC</sub> = 0.86. The dashed line is the random model.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2008/jmcmar.2008.51.issue-5/jm070654j/production/images/large/jm-2007-00654j_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm070654j&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2008/jmcmar.2008.51.issue-5/jm070654j/production/images/medium/jm-2007-00654j_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2008/jmcmar.2008.51.issue-5/jm070654j/production/images/large/jm-2007-00654j_0005.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Predicted binding mode of compound <b>4</b>. Intermolecular hydrogen bonds to the hinge region are shown by dashed cyan lines. The pose shown is the one with the lowest binding energy according to the two-parameter CDK2 model after docking with DAIM, SEED, and FFLD and minimization with CHARMm22 in CDK2 (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1KE5">1KE5</a>). Figure was prepared with PyMOL (DeLano Scientific, San Carlos, CA).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2008/jmcmar.2008.51.issue-5/jm070654j/production/images/large/jm-2007-00654j_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm070654j&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i36">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_18557" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_18557" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 55 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group">Holloway, M. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wai, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Halgren, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fitzgerald, P. M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vacca, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dorsey, B. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Levi, R. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thompson, W. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, L. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">deSolms, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gaffin, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ghosh, A. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Giuliani, E. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Graham, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guare, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hungate, R. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lyle, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sanders, W. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tucker, T. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wiggins, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wiscount, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Woltersdorf, O. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Young, S. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Darke, P. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zugay, J. A.</span><span> </span><span class="NLM_article-title">A priori prediction of activity for HIV-1 protease inhibitors employing energy minimization in the active site</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1995</span><span class="NLM_x">, </span> <span class="NLM_volume">38</span><span class="NLM_x">, </span> <span class="NLM_fpage">305</span><span class="NLM_x">–</span> <span class="NLM_lpage">317</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm00002a012" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=1995&pages=305-317&author=M.+K.+Hollowayauthor=J.+M.+Waiauthor=T.+A.+Halgrenauthor=P.+M.+D.+Fitzgeraldauthor=J.+P.+Vaccaauthor=B.+D.+Dorseyauthor=R.+B.+Leviauthor=W.+J.+Thompsonauthor=L.+J.+Chenauthor=S.+J.+deSolmsauthor=N.+Gaffinauthor=A.+K.+Ghoshauthor=E.+A.+Giulianiauthor=S.+L.+Grahamauthor=J.+P.+Guareauthor=R.+W.+Hungateauthor=T.+A.+Lyleauthor=W.+M.+Sandersauthor=T.+J.+Tuckerauthor=M.+Wigginsauthor=C.+M.+Wiscountauthor=O.+W.+Woltersdorfauthor=S.+D.+Youngauthor=P.+L.+Darkeauthor=J.+A.+Zugay&title=A+priori+prediction+of+activity+for+HIV-1+protease+inhibitors+employing+energy+minimization+in+the+active+site"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1021%2Fjm00002a012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm00002a012%26sid%3Dliteratum%253Aachs%26aulast%3DHolloway%26aufirst%3DM.%2BK.%26aulast%3DWai%26aufirst%3DJ.%2BM.%26aulast%3DHalgren%26aufirst%3DT.%2BA.%26aulast%3DFitzgerald%26aufirst%3DP.%2BM.%2BD.%26aulast%3DVacca%26aufirst%3DJ.%2BP.%26aulast%3DDorsey%26aufirst%3DB.%2BD.%26aulast%3DLevi%26aufirst%3DR.%2BB.%26aulast%3DThompson%26aufirst%3DW.%2BJ.%26aulast%3DChen%26aufirst%3DL.%2BJ.%26aulast%3DdeSolms%26aufirst%3DS.%2BJ.%26aulast%3DGaffin%26aufirst%3DN.%26aulast%3DGhosh%26aufirst%3DA.%2BK.%26aulast%3DGiuliani%26aufirst%3DE.%2BA.%26aulast%3DGraham%26aufirst%3DS.%2BL.%26aulast%3DGuare%26aufirst%3DJ.%2BP.%26aulast%3DHungate%26aufirst%3DR.%2BW.%26aulast%3DLyle%26aufirst%3DT.%2BA.%26aulast%3DSanders%26aufirst%3DW.%2BM.%26aulast%3DTucker%26aufirst%3DT.%2BJ.%26aulast%3DWiggins%26aufirst%3DM.%26aulast%3DWiscount%26aufirst%3DC.%2BM.%26aulast%3DWoltersdorf%26aufirst%3DO.%2BW.%26aulast%3DYoung%26aufirst%3DS.%2BD.%26aulast%3DDarke%26aufirst%3DP.%2BL.%26aulast%3DZugay%26aufirst%3DJ.%2BA.%26atitle%3DA%2520priori%2520prediction%2520of%2520activity%2520for%2520HIV-1%2520protease%2520inhibitors%2520employing%2520energy%2520minimization%2520in%2520the%2520active%2520site%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1995%26volume%3D38%26spage%3D305%26epage%3D317" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group">Apostolakis, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Caflisch, A.</span><span> </span><span class="NLM_article-title">Computational ligand design.</span> <span class="citation_source-journal">Comb. Chem. High Throughput Screening</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1999</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">91</span><span class="NLM_x">–</span> <span class="NLM_lpage">104</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Fjm070654j&amp;key=10420978" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Fjm070654j&amp;key=1%3ACAS%3A528%3ADyaK1MXjsFWmtbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=1999&pages=91-104&author=J.+Apostolakisauthor=A.+Caflisch&title=Computational+ligand+design."target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Computational ligand design</span></div><div class="casAuthors">Apostolakis, J.; Caflisch, A.</div><div class="citationInfo"><span class="NLM_cas:title">Combinatorial Chemistry and High Throughput Screening</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">91-104</span>CODEN:
                <span class="NLM_cas:coden">CCHSFU</span>;
        ISSN:<span class="NLM_cas:issn">1386-2073</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers</span>)
        </div><div class="casAbstract">A review with 108 refs. of variety of computational tools that are used to assist drug design.  Particular emphasis is given to the limitations and merits of different methodologies.  Recently, a no. of general methods have been proposed for clustering compds. in classes of drug-like and non-drug-like mols.  The usefulness of this classification for drug design is discussed.  The estn. of (relative) binding affinities is from a theor. point of view the most challenging part of ligand design.  The authors review three methods for the estn. of binding energies.  Firstly, quant. structure-activity relationships (QSAR) are presented.  These have gained significantly from recent developments of exptl. techniques for combinatorial synthesis and high-throughput screening as well as the use of powerful computational procedures like genetic algorithms and neural networks for the derivation of models.  Secondly, empirical energy functions are shown to lead to more general models than std. QSAR, since they are fitted to a variety of complexes.  They have been used recently with considerable success.  Thirdly, the authors briefly outline free energy calcns. based on mol. dynamics simulations, the method with the most sound theor. foundation.  Recent developments are reestablishing the interest in this approach.  In the last part of this review, structure-based ligand design programs are described.  These are closely related to docking, with the difference that in design, unlike in most docking procedures, ligands are built on a fragment-by-fragment basis.  Finally, a short description of the authors approach to computational combinatorial ligand design is given.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpCT9f0aXUDm7Vg90H21EOLACvtfcHk0lhL6BmdWMgMXA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXjsFWmtbs%253D&md5=4d9a30b781baac1571e496bbc49bbd53</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DApostolakis%26aufirst%3DJ.%26aulast%3DCaflisch%26aufirst%3DA.%26atitle%3DComputational%2520ligand%2520design.%26jtitle%3DComb.%2520Chem.%2520High%2520Throughput%2520Screening%26date%3D1999%26volume%3D2%26spage%3D91%26epage%3D104" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group">Doman, T. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McGovern, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Witherbee, B. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kasten, T. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kurumbail, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stallings, W. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Connolly, D. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">Molecular docking and high-throughput screening for novel inhibitors of protein tyrosine phosphatase-1B</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">45</span><span class="NLM_x">, </span> <span class="NLM_fpage">2213</span><span class="NLM_x">–</span> <span class="NLM_lpage">2221</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm010548w" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2002&pages=2213-2221&author=T.+N.+Domanauthor=S.+L.+McGovernauthor=B.+J.+Witherbeeauthor=T.+P.+Kastenauthor=R.+Kurumbailauthor=W.+C.+Stallingsauthor=D.+T.+Connollyauthor=B.+K.+Shoichet&title=Molecular+docking+and+high-throughput+screening+for+novel+inhibitors+of+protein+tyrosine+phosphatase-1B"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1021%2Fjm010548w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm010548w%26sid%3Dliteratum%253Aachs%26aulast%3DDoman%26aufirst%3DT.%2BN.%26aulast%3DMcGovern%26aufirst%3DS.%2BL.%26aulast%3DWitherbee%26aufirst%3DB.%2BJ.%26aulast%3DKasten%26aufirst%3DT.%2BP.%26aulast%3DKurumbail%26aufirst%3DR.%26aulast%3DStallings%26aufirst%3DW.%2BC.%26aulast%3DConnolly%26aufirst%3DD.%2BT.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DMolecular%2520docking%2520and%2520high-throughput%2520screening%2520for%2520novel%2520inhibitors%2520of%2520protein%2520tyrosine%2520phosphatase-1B%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2002%26volume%3D45%26spage%3D2213%26epage%3D2221" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group">Glen, R. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allen, S. C.</span><span> </span><span class="NLM_article-title">Ligand−protein docking: cancer research at the interface between biology and chemistry</span> <span class="citation_source-journal">Curr. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">763</span><span class="NLM_x">–</span> <span class="NLM_lpage">777</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Fjm070654j&amp;key=10.2174%2F0929867033457809" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Fjm070654j&amp;key=12678780" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Fjm070654j&amp;key=1%3ACAS%3A528%3ADC%252BD3sXjtVaju7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2003&pages=763-777&author=R.+C.+Glenauthor=S.+C.+Allen&title=Ligand%E2%88%92protein+docking%3A+cancer+research+at+the+interface+between+biology+and+chemistry"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Ligand-protein docking: Cancer research at the interface between biology and chemistry</span></div><div class="casAuthors">Glen, R. C.; Allen, S. C.</div><div class="citationInfo"><span class="NLM_cas:title">Current Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">763-777</span>CODEN:
                <span class="NLM_cas:coden">CMCHE7</span>;
        ISSN:<span class="NLM_cas:issn">0929-8673</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  In recent years there has been a growing interest in computer-based screening.  One of the driving forces has been the increased efficiency of protein crystallog. leading to the real possibility of using structure-based design as a significant contributor to the discovery of novel ligands.  In 1957 after 22 yr of work the first protein structure, detd. by x-ray crystallog. was produced [1].  Now the process has become increasingly automated and nearly 20,000 protein structures are available in the Protein Data Bank (PDB) [2].  Equally, progress in genomics will result in a great expansion of validated targets for cancer therapy.  The understanding of the relationships between structure and function of gene products will be one of the key routes to new therapeutic advances.  The challenge now is to use this data in the discovery of novel therapeutics.  One approach is obviously to synthesize mols. and co-crystallize or soak them into the protein crystal and so det. the position and interaction of the mol. with the protein.  The structural information obtained (where does the mol. bind; what are the ligand/protein/solvent interactions) can be invaluable in the generation of novel mols. or in the re-design of existing mols. whose drug properties are not optimal.  However, when dealing with large nos. (millions) of mols., when crystn. is difficult or in testing hypotheses, a significant contribution can be made using computer based screening methods.  To use the structural information derived from x-ray crystallog. (or other sources, for example NMR or homol. modeling) when evaluating the utility of a novel ligand, we need to understand where in the protein (or other macromol. such as RNA) the ligand is likely to bind and also if possible, the strength of the binding interactions.  This problem is known as the "docking problem".  There have been many approaches to the soln. of this problem over the last ten years.  For example, some methods rely on complex mol. dynamics simulations while others use less costly graph matching approaches.  There is generally a compromise between speed and accuracy, with some methods giving much more information and insight into the nature of the protein/ligand interactions and other methods optimized for speed of docking thousands of putative ligands.  We will describe some of the more common methods and algorithms used to solve the docking problem and in particular, we will review recent applications in cancer research.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpgEI_kOJr7ILVg90H21EOLACvtfcHk0lhe07qXXLV--A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXjtVaju7c%253D&md5=d96c95e03061bdcfffbe1e9e40f3c6ad</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.2174%2F0929867033457809&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F0929867033457809%26sid%3Dliteratum%253Aachs%26aulast%3DGlen%26aufirst%3DR.%2BC.%26aulast%3DAllen%26aufirst%3DS.%2BC.%26atitle%3DLigand%25E2%2588%2592protein%2520docking%253A%2520cancer%2520research%2520at%2520the%2520interface%2520between%2520biology%2520and%2520chemistry%26jtitle%3DCurr.%2520Med.%2520Chem.%26date%3D2003%26volume%3D10%26spage%3D763%26epage%3D777" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group">Walters, W. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Namchuk, M.</span><span> </span><span class="NLM_article-title">Designing screens: how to make your hits a hit</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">259</span><span class="NLM_x">–</span> <span class="NLM_lpage">266</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2003&pages=259-266&author=W.+P.+Waltersauthor=M.+Namchuk&title=Designing+screens%3A+how+to+make+your+hits+a+hit"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DWalters%26aufirst%3DW.%2BP.%26aulast%3DNamchuk%26aufirst%3DM.%26atitle%3DDesigning%2520screens%253A%2520how%2520to%2520make%2520your%2520hits%2520a%2520hit%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2003%26volume%3D2%26spage%3D259%26epage%3D266" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group">Alvarez, J. C.</span><span> </span><span class="NLM_article-title">High-throughput docking as a source of novel drug leads</span> <span class="citation_source-journal">Curr. Opin. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">365</span><span class="NLM_x">–</span> <span class="NLM_lpage">370</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Fjm070654j&amp;key=10.1016%2Fj.cbpa.2004.05.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Fjm070654j&amp;key=15288245" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Fjm070654j&amp;key=1%3ACAS%3A528%3ADC%252BD2cXmt1Omsrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2004&pages=365-370&author=J.+C.+Alvarez&title=High-throughput+docking+as+a+source+of+novel+drug+leads"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">High-throughput docking as a source of novel drug leads</span></div><div class="casAuthors">Alvarez, Juan C.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Chemical Biology</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">365-370</span>CODEN:
                <span class="NLM_cas:coden">COCBF4</span>;
        ISSN:<span class="NLM_cas:issn">1367-5931</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Receptor-based virtual screening has become a viable source of novel leads in the pharmaceutical industry.  The rapidly growing availability of structural information across protein families, the accessibility to increased computational power at affordable cost, as well as an improved understanding on how to effectively apply virtual screening technologies has contributed to their emergence.  Nonetheless, continued improvement in the accuracy of scoring functions and a greater understanding of protein mobility is crit. to advance the technol. further.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqkrrsrGzZ06LVg90H21EOLACvtfcHk0lhe07qXXLV--A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXmt1Omsrw%253D&md5=d828a47c7bc00bcc7dc6c39827db94b5</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1016%2Fj.cbpa.2004.05.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cbpa.2004.05.001%26sid%3Dliteratum%253Aachs%26aulast%3DAlvarez%26aufirst%3DJ.%2BC.%26atitle%3DHigh-throughput%2520docking%2520as%2520a%2520source%2520of%2520novel%2520drug%2520leads%26jtitle%3DCurr.%2520Opin.%2520Chem.%2520Biol.%26date%3D2004%26volume%3D8%26spage%3D365%26epage%3D370" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group">Jorgensen, W. L.</span><span> </span><span class="NLM_article-title">The many roles of computation in drug discovery</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">303</span><span class="NLM_x">, </span> <span class="NLM_fpage">1813</span><span class="NLM_x">–</span> <span class="NLM_lpage">1818</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Fjm070654j&amp;key=10.1126%2Fscience.1096361" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Fjm070654j&amp;key=15031495" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Fjm070654j&amp;key=1%3ACAS%3A528%3ADC%252BD2cXitFehu7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=303&publication_year=2004&pages=1813-1818&author=W.+L.+Jorgensen&title=The+many+roles+of+computation+in+drug+discovery"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">The Many Roles of Computation in Drug Discovery</span></div><div class="casAuthors">Jorgensen, William L.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">303</span>
        (<span class="NLM_cas:issue">5665</span>),
    <span class="NLM_cas:pages">1813-1818</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">A review.  An overview is given on the diverse uses of computational chem. in drug discovery.  Particular emphasis is placed on virtual screening, de novo design, evaluation of drug-likeness, and advanced methods for detg. protein-ligand binding.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrPr2xRlx_bxbVg90H21EOLACvtfcHk0lhe07qXXLV--A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXitFehu7w%253D&md5=77b2fac63e750ca7cdc7e4f35654549a</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1126%2Fscience.1096361&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1096361%26sid%3Dliteratum%253Aachs%26aulast%3DJorgensen%26aufirst%3DW.%2BL.%26atitle%3DThe%2520many%2520roles%2520of%2520computation%2520in%2520drug%2520discovery%26jtitle%3DScience%26date%3D2004%26volume%3D303%26spage%3D1813%26epage%3D1818" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">Virtual screening of chemical libraries</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">432</span><span class="NLM_x">, </span> <span class="NLM_fpage">862</span><span class="NLM_x">–</span> <span class="NLM_lpage">865</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Fjm070654j&amp;key=10.1038%2Fnature03197" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Fjm070654j&amp;key=15602552" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Fjm070654j&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVOht7jK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=432&publication_year=2004&pages=862-865&author=B.+K.+Shoichet&title=Virtual+screening+of+chemical+libraries"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Virtual screening of chemical libraries</span></div><div class="casAuthors">Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">432</span>
        (<span class="NLM_cas:issue">7019</span>),
    <span class="NLM_cas:pages">862-865</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Virtual screening uses computer-based methods to discover new ligands on the basis of biol. structures.  Although widely heralded in the 1970s and 1980s, the technique has since struggled to meet its initial promise, and drug discovery remains dominated by empirical screening.  Recent successes in predicting new ligands and their receptor-bound structures, and better rates of ligand discovery compared to empirical screening, have reignited interest in virtual screening, which is now widely used in drug discovery, albeit on a more limited scale than empirical screening.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrSY9IbbFldkrVg90H21EOLACvtfcHk0lhe07qXXLV--A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVOht7jK&md5=e351e226e8985d7c304a05fb67dfdc60</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1038%2Fnature03197&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature03197%26sid%3Dliteratum%253Aachs%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DVirtual%2520screening%2520of%2520chemical%2520libraries%26jtitle%3DNature%26date%3D2004%26volume%3D432%26spage%3D862%26epage%3D865" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group">Huang, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lüthi, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kolb, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Edler, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cecchini, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Audetat, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barberis, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Caflisch, A.</span><span> </span><span class="NLM_article-title">Discovery of cell-permeable non-peptide inhibitors of β-secretase by high-throughput docking and continuum electrostatics calculations</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">48</span><span class="NLM_x">, </span> <span class="NLM_fpage">5108</span><span class="NLM_x">–</span> <span class="NLM_lpage">5111</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm050499d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2005&pages=5108-5111&author=D.+Huangauthor=U.+L%C3%BCthiauthor=P.+Kolbauthor=K.+Edlerauthor=M.+Cecchiniauthor=S.+Audetatauthor=A.+Barberisauthor=A.+Caflisch&title=Discovery+of+cell-permeable+non-peptide+inhibitors+of+%CE%B2-secretase+by+high-throughput+docking+and+continuum+electrostatics+calculations"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1021%2Fjm050499d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm050499d%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DD.%26aulast%3DL%25C3%25BCthi%26aufirst%3DU.%26aulast%3DKolb%26aufirst%3DP.%26aulast%3DEdler%26aufirst%3DK.%26aulast%3DCecchini%26aufirst%3DM.%26aulast%3DAudetat%26aufirst%3DS.%26aulast%3DBarberis%26aufirst%3DA.%26aulast%3DCaflisch%26aufirst%3DA.%26atitle%3DDiscovery%2520of%2520cell-permeable%2520non-peptide%2520inhibitors%2520of%2520%25CE%25B2-secretase%2520by%2520high-throughput%2520docking%2520and%2520continuum%2520electrostatics%2520calculations%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2005%26volume%3D48%26spage%3D5108%26epage%3D5111" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group">Huang, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lüthi, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kolb, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cecchini, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barberis, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Caflisch, A.</span><span> </span><span class="NLM_article-title">In silico discovery of β-secretase inhibitors</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">128</span><span class="NLM_x">, </span> <span class="NLM_fpage">5436</span><span class="NLM_x">–</span> <span class="NLM_lpage">5443</span></span><div class="citationLinks">[<a href="/doi/10.1021/ja0573108" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=128&publication_year=2006&pages=5436-5443&author=D.+Huangauthor=U.+L%C3%BCthiauthor=P.+Kolbauthor=M.+Cecchiniauthor=A.+Barberisauthor=A.+Caflisch&title=In+silico+discovery+of+%CE%B2-secretase+inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1021%2Fja0573108&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja0573108%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DD.%26aulast%3DL%25C3%25BCthi%26aufirst%3DU.%26aulast%3DKolb%26aufirst%3DP.%26aulast%3DCecchini%26aufirst%3DM.%26aulast%3DBarberis%26aufirst%3DA.%26aulast%3DCaflisch%26aufirst%3DA.%26atitle%3DIn%2520silico%2520discovery%2520of%2520%25CE%25B2-secretase%2520inhibitors%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2006%26volume%3D128%26spage%3D5436%26epage%3D5443" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group">Wang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deng, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roux, B.</span><span> </span><span class="NLM_article-title">Absolute binding free energy calculations using molecular dynamics simulations with restraining potentials</span> <span class="citation_source-journal">Biophys. J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">91</span><span class="NLM_x">, </span> <span class="NLM_fpage">2798</span><span class="NLM_x">–</span> <span class="NLM_lpage">2814</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Fjm070654j&amp;key=10.1529%2Fbiophysj.106.084301" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Fjm070654j&amp;key=16844742" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Fjm070654j&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtVGls7vP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=91&publication_year=2006&pages=2798-2814&author=J.+Wangauthor=Y.+Dengauthor=B.+Roux&title=Absolute+binding+free+energy+calculations+using+molecular+dynamics+simulations+with+restraining+potentials"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Absolute binding free energy calculations using molecular dynamics simulations with restraining potentials</span></div><div class="casAuthors">Wang, Jiyao; Deng, Yuqing; Roux, Benoit</div><div class="citationInfo"><span class="NLM_cas:title">Biophysical Journal</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">91</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2798-2814</span>CODEN:
                <span class="NLM_cas:coden">BIOJAU</span>;
        ISSN:<span class="NLM_cas:issn">0006-3495</span>.
    
            (<span class="NLM_cas:orgname">Biophysical Society</span>)
        </div><div class="casAbstract">The abs. (std.) binding free energy of eight FK506-related ligands to FKBP12 is calcd. using free energy perturbation mol. dynamics (FEP/MD) simulations with explicit solvent.  A no. of features are implemented to improve the accuracy and enhance the convergence of the calcns.  First, the abs. binding free energy is decompd. into sequential steps during which the ligand-surrounding interactions as well as various biasing potentials restraining the translation, orientation, and conformation of the ligand are turned "on" and "off.".  Second, sampling of the ligand conformation is enforced by a restraining potential based on the root mean-square deviation relative to the bound state conformation.  The effect of all the restraining potentials is rigorously unbiased, and it is shown explicitly that the final results are independent of all artificial restraints.  Third, the repulsive and dispersive free energy contribution arising from the Lennard-Jones interactions of the ligand with its surrounding (protein and solvent) is calcd. using the Weeks-Chandler-Andersen sepn.  This sepn. also improves convergence of the FEP/MD calcns.  Fourth, to decrease the computational cost, only a small no. of atoms in the vicinity of the binding site are simulated explicitly, while all the influence of the remaining atoms is incorporated implicitly using the generalized solvent boundary potential (GSBP) method.  With GSBP, the size of the simulated FKBP12/ligand systems is significantly reduced, from ∼25,000 to 2500.  The computations are very efficient and the statistical error is small (∼1 kcal/mol).  The calcd. binding free energies are generally in good agreement with available exptl. data and previous calcns. (within ∼2 kcal/mol).  The present results indicate that a strategy based on FEP/MD simulations of a reduced GSBP at. model sampled with conformational, translational, and orientational restraining potentials can be computationally inexpensive and accurate.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqsDftyP7DXobVg90H21EOLACvtfcHk0lh9EdFBUjKKzw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtVGls7vP&md5=010f74f61048d03fee5dce41172dac70</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1529%2Fbiophysj.106.084301&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1529%252Fbiophysj.106.084301%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DDeng%26aufirst%3DY.%26aulast%3DRoux%26aufirst%3DB.%26atitle%3DAbsolute%2520binding%2520free%2520energy%2520calculations%2520using%2520molecular%2520dynamics%2520simulations%2520with%2520restraining%2520potentials%26jtitle%3DBiophys.%2520J.%26date%3D2006%26volume%3D91%26spage%3D2798%26epage%3D2814" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group">Åqvist, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Medina, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Samuelsson, J.-E.</span><span> </span><span class="NLM_article-title">A new method for predicting binding affinity in computer-aided drug design</span> <span class="citation_source-journal">Protein Eng.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1994</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">385</span><span class="NLM_x">–</span> <span class="NLM_lpage">391</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Fjm070654j&amp;key=10.1093%2Fprotein%2F7.3.385" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Fjm070654j&amp;key=8177887" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Fjm070654j&amp;key=1%3ACAS%3A280%3ADyaK2c3jtlSnsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=1994&pages=385-391&author=J.+%C3%85qvistauthor=C.+Medinaauthor=J.-E.+Samuelsson&title=A+new+method+for+predicting+binding+affinity+in+computer-aided+drug+design"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">A new method for predicting binding affinity in computer-aided drug design</span></div><div class="casAuthors">Aqvist J; Medina C; Samuelsson J E</div><div class="citationInfo"><span class="NLM_cas:title">Protein engineering</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">385-91</span>
        ISSN:<span class="NLM_cas:issn">0269-2139</span>.
    </div><div class="casAbstract">A new semi-empirical method for calculating free energies of binding from molecular dynamics (MD) simulations is presented.  It is based on standard thermodynamic cycles and on a linear approximation of polar and non-polar free energy contributions from the corresponding MD averages.  The method is tested on a set of endothiapepsin inhibitors and found to give accurate results both for absolute as well as relative free energies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTEx3uc8suH_BOk4JA3soTqfW6udTcc2ea9EOSWyvbWgrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADyaK2c3jtlSnsw%253D%253D&md5=c45572201328309d4fc849844d1c3b71</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1093%2Fprotein%2F7.3.385&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fprotein%252F7.3.385%26sid%3Dliteratum%253Aachs%26aulast%3D%25C3%2585qvist%26aufirst%3DJ.%26aulast%3DMedina%26aufirst%3DC.%26aulast%3DSamuelsson%26aufirst%3DJ.-E.%26atitle%3DA%2520new%2520method%2520for%2520predicting%2520binding%2520affinity%2520in%2520computer-aided%2520drug%2520design%26jtitle%3DProtein%2520Eng.%26date%3D1994%26volume%3D7%26spage%3D385%26epage%3D391" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group">Hansson, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Åqvist, J.</span><span> </span><span class="NLM_article-title">Estimation of binding free energies for HIV proteinase inhibitors by molecular dynamics simulations.</span> <span class="citation_source-journal">Protein Eng.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1995</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">1137</span><span class="NLM_x">–</span> <span class="NLM_lpage">1144</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Fjm070654j&amp;key=10.1093%2Fprotein%2F8.11.1137" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Fjm070654j&amp;key=8819979" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Fjm070654j&amp;key=1%3ACAS%3A528%3ADyaK28XhsFymtLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=1995&pages=1137-1144&author=T.+Hanssonauthor=J.+%C3%85qvist&title=Estimation+of+binding+free+energies+for+HIV+proteinase+inhibitors+by+molecular+dynamics+simulations."target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Estimation of binding free energies for HIV proteinase inhibitors by molecular dynamics simulations</span></div><div class="casAuthors">Hansson, Tomas; Aaqvist, Johan</div><div class="citationInfo"><span class="NLM_cas:title">Protein Engineering</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1137-44</span>CODEN:
                <span class="NLM_cas:coden">PRENE9</span>;
        ISSN:<span class="NLM_cas:issn">0269-2139</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Abs. binding free energies for inhibitors of HIV-1 proteinase were estd. from mol. dynamics simulations by a recently reported linear approxn. procedure.  The results were in fairly good agreement with exptl. binding data.  Two of the inhibitors were very similar and, for comparison, their relative free energies of binding were also calcd. by free energy perturbation methods, giving virtually the same result.  Effects of cutoff radii and charge states of the protein model were examd.  The effects of pH on binding of one of the inhibitors were predicted.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrcv6yv6JgB1rVg90H21EOLACvtfcHk0lgRU9s43qIoAg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28XhsFymtLY%253D&md5=c77820746da2a692e7543f5f79789cca</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1093%2Fprotein%2F8.11.1137&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fprotein%252F8.11.1137%26sid%3Dliteratum%253Aachs%26aulast%3DHansson%26aufirst%3DT.%26aulast%3D%25C3%2585qvist%26aufirst%3DJ.%26atitle%3DEstimation%2520of%2520binding%2520free%2520energies%2520for%2520HIV%2520proteinase%2520inhibitors%2520by%2520molecular%2520dynamics%2520simulations.%26jtitle%3DProtein%2520Eng.%26date%3D1995%26volume%3D8%26spage%3D1137%26epage%3D1144" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group">Jones-Hertzog, D. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jorgensen, W. L.</span><span> </span><span class="NLM_article-title">Binding affinities for sulfonamide inhibitors with human thrombin using Monte Carlo simulations with a linear response method</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1996</span><span class="NLM_x">, </span> <span class="NLM_volume">40</span><span class="NLM_x">, </span> <span class="NLM_fpage">1539</span><span class="NLM_x">–</span> <span class="NLM_lpage">1549</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=1996&pages=1539-1549&author=D.+K.+Jones-Hertzogauthor=W.+L.+Jorgensen&title=Binding+affinities+for+sulfonamide+inhibitors+with+human+thrombin+using+Monte+Carlo+simulations+with+a+linear+response+method"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DJones-Hertzog%26aufirst%3DD.%2BK.%26aulast%3DJorgensen%26aufirst%3DW.%2BL.%26atitle%3DBinding%2520affinities%2520for%2520sulfonamide%2520inhibitors%2520with%2520human%2520thrombin%2520using%2520Monte%2520Carlo%2520simulations%2520with%2520a%2520linear%2520response%2520method%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1996%26volume%3D40%26spage%3D1539%26epage%3D1549" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group">Hansson, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marelius, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Åqvist, J.</span><span> </span><span class="NLM_article-title">Ligand binding affinity prediction by linear interaction energy methods</span> <span class="citation_source-journal">J. Comput.-Aided Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">12</span><span class="NLM_x">, </span> <span class="NLM_fpage">27</span><span class="NLM_x">–</span> <span class="NLM_lpage">35</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Fjm070654j&amp;key=10.1023%2FA%3A1007930623000" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Fjm070654j&amp;key=9570087" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Fjm070654j&amp;key=1%3ACAS%3A528%3ADyaK1cXisl2qsbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=1998&pages=27-35&author=T.+Hanssonauthor=J.+Mareliusauthor=J.+%C3%85qvist&title=Ligand+binding+affinity+prediction+by+linear+interaction+energy+methods"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Ligand binding affinity prediction by linear interaction energy methods</span></div><div class="casAuthors">Hansson, Tomas; Marelius, John; Aqvist, Johan</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">27-35</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Kluwer Academic Publishers</span>)
        </div><div class="casAbstract">A recent method for estg. ligand binding affinities is extended.  This method employs avs. of interaction potential energy terms from mol. dynamics simulations or other thermal conformational sampling techniques.  Incorporation of systematic deviations from electrostatic linear response, derived from free energy perturbation studies, into the abs. binding free energy expression significantly enhances the accuracy of the approach.  This type of method may be useful for computational prediction of ligand binding strengths, e.g., in drug design applications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpU4E4PGZrXWbVg90H21EOLACvtfcHk0lgRU9s43qIoAg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXisl2qsbc%253D&md5=c66173a2b01b3283be2700c370d0ddea</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1023%2FA%3A1007930623000&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1023%252FA%253A1007930623000%26sid%3Dliteratum%253Aachs%26aulast%3DHansson%26aufirst%3DT.%26aulast%3DMarelius%26aufirst%3DJ.%26aulast%3D%25C3%2585qvist%26aufirst%3DJ.%26atitle%3DLigand%2520binding%2520affinity%2520prediction%2520by%2520linear%2520interaction%2520energy%2520methods%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D1998%26volume%3D12%26spage%3D27%26epage%3D35" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group">Wang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dixon, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kollman, P.</span><span> </span><span class="NLM_article-title">Ranking ligand binding affinities with avidin: a molecular dynamics-based interaction energy study</span> <span class="citation_source-journal">Proteins: Struct., Funct., Bioinf.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1999</span><span class="NLM_x">, </span> <span class="NLM_volume">34</span><span class="NLM_x">, </span> <span class="NLM_fpage">69</span><span class="NLM_x">–</span> <span class="NLM_lpage">81</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=1999&pages=69-81&author=J.+Wangauthor=R.+Dixonauthor=P.+Kollman&title=Ranking+ligand+binding+affinities+with+avidin%3A+a+molecular+dynamics-based+interaction+energy+study"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DDixon%26aufirst%3DR.%26aulast%3DKollman%26aufirst%3DP.%26atitle%3DRanking%2520ligand%2520binding%2520affinities%2520with%2520avidin%253A%2520a%2520molecular%2520dynamics-based%2520interaction%2520energy%2520study%26jtitle%3DProteins%253A%2520Struct.%252C%2520Funct.%252C%2520Bioinf.%26date%3D1999%26volume%3D34%26spage%3D69%26epage%3D81" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group">Carlson, H. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jorgensen, W. L.</span><span> </span><span class="NLM_article-title">An extended linear response method for determining free energies of hydration</span> <span class="citation_source-journal">J. Phys. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1995</span><span class="NLM_x">, </span> <span class="NLM_volume">99</span><span class="NLM_x">, </span> <span class="NLM_fpage">10667</span><span class="NLM_x">–</span> <span class="NLM_lpage">10673</span></span><div class="citationLinks">[<a href="/doi/10.1021/j100026a034" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Fjm070654j&amp;key=1%3ACAS%3A528%3ADyaK2MXmtVyiu7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=99&publication_year=1995&pages=10667-10673&author=H.+A.+Carlsonauthor=W.+L.+Jorgensen&title=An+extended+linear+response+method+for+determining+free+energies+of+hydration"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">An Extended Linear Response Method for Determining Free Energies of Hydration</span></div><div class="casAuthors">Carlson, Heather A.; Jorgensen, William L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Physical Chemistry</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">99</span>
        (<span class="NLM_cas:issue">26</span>),
    <span class="NLM_cas:pages">10667-73</span>CODEN:
                <span class="NLM_cas:coden">JPCHAX</span>;
        ISSN:<span class="NLM_cas:issn">0022-3654</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Fourteen neutral org. mols. with diverse functionalities were used to det. the utility of equations based on linear response theory for the calcn. of free energies of hydration.  The equations contain terms proportional to the Coulombic and van der Waals components of the solute-solvent interaction energy and to an index for the cost of cavity formation such as solvent-accessible surface area.  The energy components can be obtained from Monte Carlo or mol. dynamics simulations of the solutes in water.  The methodol. was tested by using partial charges from fitting to the electrostatic potentials of ab initio 6-31G* wave functions and from OPLS potential functions.  Root-mean-square deviations between the results from the recommended linear response methods and exptl. free energies of hydration are ca. 0.8 kcal/mol.  Substituted benzenes, not included in the parameter development, were used to further evaluate the method performance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoNcOGUicelNrVg90H21EOLACvtfcHk0lgRU9s43qIoAg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXmtVyiu7Y%253D&md5=b91a7f287b63063c81c51082f694a78b</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1021%2Fj100026a034&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fj100026a034%26sid%3Dliteratum%253Aachs%26aulast%3DCarlson%26aufirst%3DH.%2BA.%26aulast%3DJorgensen%26aufirst%3DW.%2BL.%26atitle%3DAn%2520extended%2520linear%2520response%2520method%2520for%2520determining%2520free%2520energies%2520of%2520hydration%26jtitle%3DJ.%2520Phys.%2520Chem.%26date%3D1995%26volume%3D99%26spage%3D10667%26epage%3D10673" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group">Wall, I. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leach, A. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Salt, D. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ford, M. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Essex, J. W.</span><span> </span><span class="NLM_article-title">Binding constants of neuraminidase inhibitors: an investigation of the linear interaction energy method</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1999</span><span class="NLM_x">, </span> <span class="NLM_volume">42</span><span class="NLM_x">, </span> <span class="NLM_fpage">5142</span><span class="NLM_x">–</span> <span class="NLM_lpage">5152</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm990105g" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Fjm070654j&amp;key=1%3ACAS%3A528%3ADyaK1MXnsFOlu70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=1999&pages=5142-5152&author=I.+D.+Wallauthor=A.+R.+Leachauthor=D.+W.+Saltauthor=M.+G.+Fordauthor=J.+W.+Essex&title=Binding+constants+of+neuraminidase+inhibitors%3A+an+investigation+of+the+linear+interaction+energy+method"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Binding Constants of Neuraminidase Inhibitors: An Investigation of the Linear Interaction Energy Method</span></div><div class="casAuthors">Wall, Ian D.; Leach, Andrew R.; Salt, David W.; Ford, Martyn G.; Essex, Jonathan W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">25</span>),
    <span class="NLM_cas:pages">5142-5152</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The linear interaction energy (LIE) method has been applied to the calcn. of the binding free energies of 15 inhibitors of the enzyme neuraminidase.  This is a particularly challenging system for this methodol. since the protein conformation and the no. of tightly bound water mols. in the active site are known to change for different inhibitors.  It is not clear that the basic LIE method will calc. the contributions to the binding free energies arising from these effects correctly.  Application of the basic LIE equation yielded an rms error with respect to expt. of 1.51 kcal mol-1 for the free energies of binding.  To det. whether it is appropriate to include extra terms in the LIE equation, a detailed statistical anal. was undertaken.  Multiple linear regression (MLR) is often used to det. the significance of terms in a fitting equation; this method is inappropriate for the current system owing to the highly correlated nature of the descriptor variables.  Use of MLR in other applications of the LIE equation is therefore not recommended without a correlation anal. being performed first.  Here factor anal. was used to det. the no. of useful dimensions contained within the data and, hence, the max. no. of variables to be considered when specifying a model or equation.  Biased fitting methods using orthogonalized components were then used to generate the most predictive model.  The final model gave a q2 of 0.74 and contained van der Waals and electrostatic energy terms.  This result was obtained without recourse to prior knowledge and was based solely on the information content of the data.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr487StWx1P2rVg90H21EOLACvtfcHk0ljzqar6h1B-fg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXnsFOlu70%253D&md5=466f8bc796cc64e1c3979318e9313daa</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1021%2Fjm990105g&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm990105g%26sid%3Dliteratum%253Aachs%26aulast%3DWall%26aufirst%3DI.%2BD.%26aulast%3DLeach%26aufirst%3DA.%2BR.%26aulast%3DSalt%26aufirst%3DD.%2BW.%26aulast%3DFord%26aufirst%3DM.%2BG.%26aulast%3DEssex%26aufirst%3DJ.%2BW.%26atitle%3DBinding%2520constants%2520of%2520neuraminidase%2520inhibitors%253A%2520an%2520investigation%2520of%2520the%2520linear%2520interaction%2520energy%2520method%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1999%26volume%3D42%26spage%3D5142%26epage%3D5152" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group">Zhou, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Friesner, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ghoshs, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rizzo, R. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jorgensen, W. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Levy, R. M.</span><span> </span><span class="NLM_article-title">New linear interaction method for binding affinity calculations using a continuum solvent model</span> <span class="citation_source-journal">J. Phys. Chem. B</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">102</span><span class="NLM_x">, </span> <span class="NLM_fpage">10388</span><span class="NLM_x">–</span> <span class="NLM_lpage">10397</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=102&publication_year=2001&pages=10388-10397&author=R.+Zhouauthor=R.+A.+Friesnerauthor=A.+Ghoshsauthor=R.+C.+Rizzoauthor=W.+J.+Jorgensenauthor=R.+M.+Levy&title=New+linear+interaction+method+for+binding+affinity+calculations+using+a+continuum+solvent+model"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DZhou%26aufirst%3DR.%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26aulast%3DGhoshs%26aufirst%3DA.%26aulast%3DRizzo%26aufirst%3DR.%2BC.%26aulast%3DJorgensen%26aufirst%3DW.%2BJ.%26aulast%3DLevy%26aufirst%3DR.%2BM.%26atitle%3DNew%2520linear%2520interaction%2520method%2520for%2520binding%2520affinity%2520calculations%2520using%2520a%2520continuum%2520solvent%2520model%26jtitle%3DJ.%2520Phys.%2520Chem.%2520B%26date%3D2001%26volume%3D102%26spage%3D10388%26epage%3D10397" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group">Tounge, B. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reynolds, C. H.</span><span> </span><span class="NLM_article-title">Calculation of the binding affinity of β-secretase inhibitors using the linear interaction energy method</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">46</span><span class="NLM_x">, </span> <span class="NLM_fpage">2074</span><span class="NLM_x">–</span> <span class="NLM_lpage">2082</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm020513b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2003&pages=2074-2082&author=B.+A.+Toungeauthor=C.+H.+Reynolds&title=Calculation+of+the+binding+affinity+of+%CE%B2-secretase+inhibitors+using+the+linear+interaction+energy+method"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Fjm020513b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm020513b%26sid%3Dliteratum%253Aachs%26aulast%3DTounge%26aufirst%3DB.%2BA.%26aulast%3DReynolds%26aufirst%3DC.%2BH.%26atitle%3DCalculation%2520of%2520the%2520binding%2520affinity%2520of%2520%25CE%25B2-secretase%2520inhibitors%2520using%2520the%2520linear%2520interaction%2520energy%2520method%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2003%26volume%3D46%26spage%3D2074%26epage%3D2082" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group">Tominaga, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jorgensen, W. L.</span><span> </span><span class="NLM_article-title">General model for estimation of the inhibition of protein kinases using Monte Carlo simulations</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">2534</span><span class="NLM_x">–</span> <span class="NLM_lpage">2549</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm0304358" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=2534-2549&author=Y.+Tominagaauthor=W.+L.+Jorgensen&title=General+model+for+estimation+of+the+inhibition+of+protein+kinases+using+Monte+Carlo+simulations"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Fjm0304358&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0304358%26sid%3Dliteratum%253Aachs%26aulast%3DTominaga%26aufirst%3DY.%26aulast%3DJorgensen%26aufirst%3DW.%2BL.%26atitle%3DGeneral%2520model%2520for%2520estimation%2520of%2520the%2520inhibition%2520of%2520protein%2520kinases%2520using%2520Monte%2520Carlo%2520simulations%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D2534%26epage%3D2549" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group">Huang, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Caflisch, A.</span><span> </span><span class="NLM_article-title">Efficient evaluation of binding free energy using continuum electrostatic solvation</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">5791</span><span class="NLM_x">–</span> <span class="NLM_lpage">5797</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm049726m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=5791-5797&author=D.+Huangauthor=A.+Caflisch&title=Efficient+evaluation+of+binding+free+energy+using+continuum+electrostatic+solvation"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1021%2Fjm049726m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm049726m%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DD.%26aulast%3DCaflisch%26aufirst%3DA.%26atitle%3DEfficient%2520evaluation%2520of%2520binding%2520free%2520energy%2520using%2520continuum%2520electrostatic%2520solvation%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D5791%26epage%3D5797" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group">Stjernschantz, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marelius, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Medina, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jacobsson, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vermeulen, N. P. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oostenbrink, C.</span><span> </span><span class="NLM_article-title">Are automated molecular dynamics simulations and binding free energy calculations realistic tools in lead optimization? An evaluation of the linear interaction energy (LIE) method</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">46</span><span class="NLM_x">, </span> <span class="NLM_fpage">1972</span><span class="NLM_x">–</span> <span class="NLM_lpage">1983</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci0601214" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Fjm070654j&amp;key=1%3ACAS%3A528%3ADC%252BD28XnsV2hs7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2006&pages=1972-1983&author=E.+Stjernschantzauthor=J.+Mareliusauthor=C.+Medinaauthor=M.+Jacobssonauthor=N.+P.+E.+Vermeulenauthor=C.+Oostenbrink&title=Are+automated+molecular+dynamics+simulations+and+binding+free+energy+calculations+realistic+tools+in+lead+optimization%3F+An+evaluation+of+the+linear+interaction+energy+%28LIE%29+method"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Are Automated Molecular Dynamics Simulations and Binding Free Energy Calculations Realistic Tools in Lead Optimization? An Evaluation of the Linear Interaction Energy (LIE) Method</span></div><div class="casAuthors">Stjernschantz, Eva; Marelius, John; Medina, Carmen; Jacobsson, Micael; Vermeulen, Nico P. E.; Oostenbrink, Chris</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1972-1983</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">An extensive evaluation of the linear interaction energy (LIE) method for the prediction of binding affinity of docked compds. has been performed, with an emphasis on its applicability in lead optimization.  An automated setup is presented, which allows for the use of the method in an industrial setting.  Calcns. are performed for four realistic examples, retinoic acid receptor γ, matrix metalloprotease 3, estrogen receptor α, and dihydrofolate reductase, focusing on different aspects of the procedure.  The obtained LIE models are evaluated in terms of the root-mean-square (RMS) errors from exptl. binding free energies and the ability to rank compds. appropriately.  The results are compared to the best empirical scoring function, selected from a set of 10 scoring functions.  In all cases, good LIE models can be obtained in terms of free-energy RMS errors, although reasonable ranking of the ligands of dihydrofolate reductase proves difficult for both the LIE method and scoring functions.  For the other proteins, the LIE model results in better predictions than the best performing scoring function.  These results indicate that the LIE approach, as a tool to evaluate docking results, can be a valuable asset in computational lead optimization programs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpzvbHPR51x6bVg90H21EOLACvtfcHk0ljzqar6h1B-fg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XnsV2hs7w%253D&md5=c63a2c9e40c00b85c8c29f04fe996f79</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1021%2Fci0601214&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci0601214%26sid%3Dliteratum%253Aachs%26aulast%3DStjernschantz%26aufirst%3DE.%26aulast%3DMarelius%26aufirst%3DJ.%26aulast%3DMedina%26aufirst%3DC.%26aulast%3DJacobsson%26aufirst%3DM.%26aulast%3DVermeulen%26aufirst%3DN.%2BP.%2BE.%26aulast%3DOostenbrink%26aufirst%3DC.%26atitle%3DAre%2520automated%2520molecular%2520dynamics%2520simulations%2520and%2520binding%2520free%2520energy%2520calculations%2520realistic%2520tools%2520in%2520lead%2520optimization%253F%2520An%2520evaluation%2520of%2520the%2520linear%2520interaction%2520energy%2520%2528LIE%2529%2520method%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2006%26volume%3D46%26spage%3D1972%26epage%3D1983" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group">Ersmark, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nervall, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hamelink, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Janka, L. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clemente, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dunn, B. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blackman, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Samuelsson, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Åqvist, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hallberg, A.</span><span> </span><span class="NLM_article-title">Synthesis of malarial plasmepsin inhibitors and prediction of binding modes by molecular dynamics simulations</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">48</span><span class="NLM_x">, </span> <span class="NLM_fpage">6090</span><span class="NLM_x">–</span> <span class="NLM_lpage">6106</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm050463l" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2005&pages=6090-6106&author=K.+Ersmarkauthor=M.+Nervallauthor=E.+Hamelinkauthor=L.+K.+Jankaauthor=J.+C.+Clementeauthor=B.+M.+Dunnauthor=M.+J.+Blackmanauthor=B.+Samuelssonauthor=J.+%C3%85qvistauthor=A.+Hallberg&title=Synthesis+of+malarial+plasmepsin+inhibitors+and+prediction+of+binding+modes+by+molecular+dynamics+simulations"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1021%2Fjm050463l&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm050463l%26sid%3Dliteratum%253Aachs%26aulast%3DErsmark%26aufirst%3DK.%26aulast%3DNervall%26aufirst%3DM.%26aulast%3DHamelink%26aufirst%3DE.%26aulast%3DJanka%26aufirst%3DL.%2BK.%26aulast%3DClemente%26aufirst%3DJ.%2BC.%26aulast%3DDunn%26aufirst%3DB.%2BM.%26aulast%3DBlackman%26aufirst%3DM.%2BJ.%26aulast%3DSamuelsson%26aufirst%3DB.%26aulast%3D%25C3%2585qvist%26aufirst%3DJ.%26aulast%3DHallberg%26aufirst%3DA.%26atitle%3DSynthesis%2520of%2520malarial%2520plasmepsin%2520inhibitors%2520and%2520prediction%2520of%2520binding%2520modes%2520by%2520molecular%2520dynamics%2520simulations%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2005%26volume%3D48%26spage%3D6090%26epage%3D6106" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group">Warwicker, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Watson, H. C.</span><span> </span><span class="NLM_article-title">Calculation of the electric potential in the active site cleft due to α-helix dipoles</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1982</span><span class="NLM_x">, </span> <span class="NLM_volume">157</span><span class="NLM_x">, </span> <span class="NLM_fpage">671</span><span class="NLM_x">–</span> <span class="NLM_lpage">679</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Fjm070654j&amp;key=10.1016%2F0022-2836%2882%2990505-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Fjm070654j&amp;key=6288964" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Fjm070654j&amp;key=1%3ACAS%3A280%3ADyaL3s%252FhtVartA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=157&publication_year=1982&pages=671-679&author=J.+Warwickerauthor=H.+C.+Watson&title=Calculation+of+the+electric+potential+in+the+active+site+cleft+due+to+%CE%B1-helix+dipoles"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Calculation of the electric potential in the active site cleft due to alpha-helix dipoles</span></div><div class="casAuthors">Warwicker J; Watson H C</div><div class="citationInfo"><span class="NLM_cas:title">Journal of molecular biology</span>
        (<span class="NLM_cas:date">1982</span>),
    <span class="NLM_cas:volume">157</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">671-9</span>
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRKK3_KJDDB2pggzHGM8X5CfW6udTcc2eay78dPyI797Lntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADyaL3s%252FhtVartA%253D%253D&md5=4b779b03b1ce72f1a0e062ae9d074178</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1016%2F0022-2836%2882%2990505-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0022-2836%252882%252990505-8%26sid%3Dliteratum%253Aachs%26aulast%3DWarwicker%26aufirst%3DJ.%26aulast%3DWatson%26aufirst%3DH.%2BC.%26atitle%3DCalculation%2520of%2520the%2520electric%2520potential%2520in%2520the%2520active%2520site%2520cleft%2520due%2520to%2520%25CE%25B1-helix%2520dipoles%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D1982%26volume%3D157%26spage%3D671%26epage%3D679" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group">Dancey, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sausville, E. A.</span><span> </span><span class="NLM_article-title">Issues and progress with protein kinase inhibitors for cancer treatment</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">296</span><span class="NLM_x">–</span> <span class="NLM_lpage">313</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Fjm070654j&amp;key=10.1038%2Fnrd1066" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Fjm070654j&amp;key=12669029" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Fjm070654j&amp;key=1%3ACAS%3A528%3ADC%252BD3sXisFWkt70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2003&pages=296-313&author=J.+Danceyauthor=E.+A.+Sausville&title=Issues+and+progress+with+protein+kinase+inhibitors+for+cancer+treatment"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Issues and progress with protein kinase inhibitors for cancer treatment</span></div><div class="casAuthors">Dancey, Janet; Sausville, Edward A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">296-313</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Identification of the key roles of protein kinases in cancer has led to extensive efforts to develop kinase inhibitors for the treatment of a wide range of cancers, and more than 30 such agents are now in clin. trials.  Here, we consider the crucial issues in the development of kinase inhibitors for cancer, and discuss strategies to address the challenges raised by these issues in the light of preclin. and clin. experiences so far.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpH-u-kzpegLrVg90H21EOLACvtfcHk0lhtPkc1O2EjhA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXisFWkt70%253D&md5=fb11803282bcc39b24fa238a6ab973c8</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1038%2Fnrd1066&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd1066%26sid%3Dliteratum%253Aachs%26aulast%3DDancey%26aufirst%3DJ.%26aulast%3DSausville%26aufirst%3DE.%2BA.%26atitle%3DIssues%2520and%2520progress%2520with%2520protein%2520kinase%2520inhibitors%2520for%2520cancer%2520treatment%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2003%26volume%3D2%26spage%3D296%26epage%3D313" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group">Noble, M. E. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Endicott, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johnson, L. N.</span><span> </span><span class="NLM_article-title">Protein kinase inhibitors: insights into drug design from structure</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">303</span><span class="NLM_x">, </span> <span class="NLM_fpage">1800</span><span class="NLM_x">–</span> <span class="NLM_lpage">1805</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Fjm070654j&amp;key=10.1126%2Fscience.1095920" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Fjm070654j&amp;key=15031492" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Fjm070654j&amp;key=1%3ACAS%3A528%3ADC%252BD2cXitFehurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=303&publication_year=2004&pages=1800-1805&author=M.+E.+M.+Nobleauthor=J.+A.+Endicottauthor=L.+N.+Johnson&title=Protein+kinase+inhibitors%3A+insights+into+drug+design+from+structure"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Protein Kinase Inhibitors: Insights into Drug Design from Structure</span></div><div class="casAuthors">Noble, Martin E. M.; Endicott, Jane A.; Johnson, Louise N.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">303</span>
        (<span class="NLM_cas:issue">5665</span>),
    <span class="NLM_cas:pages">1800-1805</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">A review.  Protein kinases are targets for treatment of a no. of diseases.  This review focuses on kinase inhibitors that are in the clinic or in clin. trials and for which structural information is available.  Structures have informed drug design and have illuminated the mechanism of inhibition.  We review progress with the receptor tyrosine kinases (growth factor receptors EGFR, VEGFR, and FGFR) and nonreceptor tyrosine kinases (Bcr-Abl), where advances have been made with cancer therapeutic agents such as Herceptin and Gleevec.  Among the serine-threonine kinases, p38, Rho-kinase, cyclin-dependent kinases, and Chk1 have been targeted with productive results for inflammation and cancer.  Structures have provided insights into targeting the inactive or active form of the kinase, for targeting the global constellation of residues at the ATP site or less conserved addnl. pockets or single residues, and into targeting noncatalytic domains.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWqtKz7cvN_rVg90H21EOLACvtfcHk0lhtPkc1O2EjhA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXitFehurc%253D&md5=7fd3f60bee46e59307436eeed285d6ae</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1126%2Fscience.1095920&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1095920%26sid%3Dliteratum%253Aachs%26aulast%3DNoble%26aufirst%3DM.%2BE.%2BM.%26aulast%3DEndicott%26aufirst%3DJ.%2BA.%26aulast%3DJohnson%26aufirst%3DL.%2BN.%26atitle%3DProtein%2520kinase%2520inhibitors%253A%2520insights%2520into%2520drug%2520design%2520from%2520structure%26jtitle%3DScience%26date%3D2004%26volume%3D303%26spage%3D1800%26epage%3D1805" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group">Smith, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shindyalov, I. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Veretnik, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gribskov, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taylor, S. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ten- Eyck, L. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bourne, P. E.</span><span> </span><span class="NLM_article-title">The protein kinase resource</span> <span class="citation_source-journal">Trends Biochem. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">22</span><span class="NLM_x">, </span> <span class="NLM_fpage">444</span><span class="NLM_x">–</span> <span class="NLM_lpage">446</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=1997&pages=444-446&author=C.+M.+Smithauthor=I.+N.+Shindyalovauthor=S.+Veretnikauthor=M.+Gribskovauthor=S.+S.+Taylorauthor=L.+F.+Ten-+Eyckauthor=P.+E.+Bourne&title=The+protein+kinase+resource"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSmith%26aufirst%3DC.%2BM.%26aulast%3DShindyalov%26aufirst%3DI.%2BN.%26aulast%3DVeretnik%26aufirst%3DS.%26aulast%3DGribskov%26aufirst%3DM.%26aulast%3DTaylor%26aufirst%3DS.%2BS.%26aulast%3DTen-%2BEyck%26aufirst%3DL.%2BF.%26aulast%3DBourne%26aufirst%3DP.%2BE.%26atitle%3DThe%2520protein%2520kinase%2520resource%26jtitle%3DTrends%2520Biochem.%2520Sci.%26date%3D1997%26volume%3D22%26spage%3D444%26epage%3D446" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group">Cohen, P.</span><span> </span><span class="NLM_article-title">Protein kinases—the major drug targets of the twenty-first century?</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">1</span><span class="NLM_x">, </span> <span class="NLM_fpage">309</span><span class="NLM_x">–</span> <span class="NLM_lpage">315</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2002&pages=309-315&author=P.+Cohen&title=Protein+kinases%E2%80%94the+major+drug+targets+of+the+twenty-first+century%3F"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DCohen%26aufirst%3DP.%26atitle%3DProtein%2520kinases%25E2%2580%2594the%2520major%2520drug%2520targets%2520of%2520the%2520twenty-first%2520century%253F%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2002%26volume%3D1%26spage%3D309%26epage%3D315" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group">Aparna, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rambabu, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Panigrahi, S. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sarma, J. A. R. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Desiraju, G. R.</span><span> </span><span class="NLM_article-title">Virtual screening of 4-anilinoquinazoline analogues as EGFR kinase inhibitors: importance of hydrogen bonds in the evaluation of poses and scoring functions</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">45</span><span class="NLM_x">, </span> <span class="NLM_fpage">725</span><span class="NLM_x">–</span> <span class="NLM_lpage">738</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci049676u" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Fjm070654j&amp;key=1%3ACAS%3A528%3ADC%252BD2MXislWqsbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2005&pages=725-738&author=V.+Aparnaauthor=G.+Rambabuauthor=S.+K.+Panigrahiauthor=J.+A.+R.+P.+Sarmaauthor=G.+R.+Desiraju&title=Virtual+screening+of+4-anilinoquinazoline+analogues+as+EGFR+kinase+inhibitors%3A+importance+of+hydrogen+bonds+in+the+evaluation+of+poses+and+scoring+functions"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Virtual Screening of 4-Anilinoquinazoline Analogues as EGFR Kinase Inhibitors: Importance of Hydrogen Bonds in the Evaluation of Poses and Scoring Functions</span></div><div class="casAuthors">Aparna, V.; Rambabu, G.; Panigrahi, S. K.; Sarma, J. A. R. P.; Desiraju, G. R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">725-738</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Virtual Screening (VS) is a computational technique that allows selection and ranking of possible hits from a library of compds.  We have carried out VS on 128 selected EGFR kinase inhibitors with GOLD and LigandFit.  From the exptl. crystal structure of the erlotinib-EGFR complex, three key hydrogen bonds were identified as responsible for anchoring the ligand in the active site.  These are of the N-H···N, Ow-H···N, and C-H···O types.  Failure to include the hydrogen-bonded water mol. that forms the Ow-H···N bond leads to incorrect results.  Of the three interactions, the C-H···O formed by an activated C-H group is the best conserved.  On the basis of the efficacy of these hydrogen bonds, the poses were classified into one of three categories: close, shifted, and misoriented.  In the VS context, all three interactions need to be modeled correctly so that correct poses and affinities are obtained, and this happens in ligands of the close variety.  Cross scoring wherein the poses from one software are input into another for scoring and consensus scoring wherein the scores from various software packages are weighted are also helpful in obtaining better agreements.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp8i6TSN7WStrVg90H21EOLACvtfcHk0lg71TO3x4RThw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXislWqsbw%253D&md5=1dc1d041a37424b889937d697337a3d7</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Fci049676u&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci049676u%26sid%3Dliteratum%253Aachs%26aulast%3DAparna%26aufirst%3DV.%26aulast%3DRambabu%26aufirst%3DG.%26aulast%3DPanigrahi%26aufirst%3DS.%2BK.%26aulast%3DSarma%26aufirst%3DJ.%2BA.%2BR.%2BP.%26aulast%3DDesiraju%26aufirst%3DG.%2BR.%26atitle%3DVirtual%2520screening%2520of%25204-anilinoquinazoline%2520analogues%2520as%2520EGFR%2520kinase%2520inhibitors%253A%2520importance%2520of%2520hydrogen%2520bonds%2520in%2520the%2520evaluation%2520of%2520poses%2520and%2520scoring%2520functions%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2005%26volume%3D45%26spage%3D725%26epage%3D738" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group">Bramson, H. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Corona, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davis, S. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dickerson, S. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Edelstein, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frye, S. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gampe, R. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Harris, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hassell, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Holmes, W. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hunter, R. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lackey, K. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lovejoy, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luzzio, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Montana, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rocque, W. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rusnak, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shewchuk, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Veal, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Walker, D. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuyper, L. F.</span><span> </span><span class="NLM_article-title">Oxindole-based inhibitors of cyclin-dependent kinase 2 (CDK2): design, synthesis, enzymatic activities, and X-ray crystallographic analysis</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">44</span><span class="NLM_x">, </span> <span class="NLM_fpage">4339</span><span class="NLM_x">–</span> <span class="NLM_lpage">4358</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm010117d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Fjm070654j&amp;key=1%3ACAS%3A528%3ADC%252BD3MXnvFGnt7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2001&pages=4339-4358&author=H.+N.+Bramsonauthor=J.+Coronaauthor=S.+T.+Davisauthor=S.+H.+Dickersonauthor=M.+Edelsteinauthor=S.+V.+Fryeauthor=R.+T.+Gampeauthor=P.+A.+Harrisauthor=A.+Hassellauthor=W.+D.+Holmesauthor=R.+N.+Hunterauthor=K.+E.+Lackeyauthor=B.+Lovejoyauthor=M.+J.+Luzzioauthor=V.+Montanaauthor=W.+J.+Rocqueauthor=D.+Rusnakauthor=L.+Shewchukauthor=J.+M.+Vealauthor=D.+H.+Walkerauthor=L.+F.+Kuyper&title=Oxindole-based+inhibitors+of+cyclin-dependent+kinase+2+%28CDK2%29%3A+design%2C+synthesis%2C+enzymatic+activities%2C+and+X-ray+crystallographic+analysis"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Oxindole-Based Inhibitors of Cyclin-Dependent Kinase 2 (CDK2): Design, Synthesis, Enzymatic Activities, and X-ray Crystallographic Analysis</span></div><div class="casAuthors">Bramson, H. Neal; Corona, John; Davis, Stephen T.; Dickerson, Scott H.; Edelstein, Mark; Frye, Stephen V.; Gampe, Robert T., Jr.; Harris, Phil A.; Hassell, Anne; Holmes, William D.; Hunter, Robert N.; Lackey, Karen E.; Lovejoy, Brett; Luzzio, Michael J.; Montana, Val; Rocque, Warren J.; Rusnak, David; Shewchuk, Lisa; Veal, James M.; Walker, Duncan H.; Kuyper, Lee F.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">25</span>),
    <span class="NLM_cas:pages">4339-4358</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Two closely related classes of oxindole-based compds., 1H-indole-2,3-dione 3-phenylhydrazones and 3-(anilinomethylene)-1,3-dihydro-2H-indol-2-ones, were shown to potently inhibit cyclin-dependent kinase 2 (CDK2).  The initial lead compd. was prepd. as a homolog of the 3-benzylidene-1,3-dihydro-2H-indol-2-one class of kinase inhibitor.  Crystallog. anal. of the lead compd. bound to CDK2 provided the basis for analog design.  A semiautomated method of ligand docking was used to select compds. for synthesis, and a no. of compds. with low nanomolar inhibitory activity vs. CDK2 were identified.  Enzyme binding determinants for several analogs were evaluated by X-ray crystallog.  Compds. in this series inhibited CDK2 with a potency ∼10-fold greater than that for CDK1.  Members of this class of inhibitor cause an arrest of the cell cycle and have shown potential utility in the prevention of chemotherapy-induced alopecia.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGppOmtXxUWKlrVg90H21EOLACvtfcHk0liMuR3mfcMUWg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXnvFGnt7Y%253D&md5=d628549913ca46f9edf6b4692c82419d</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Fjm010117d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm010117d%26sid%3Dliteratum%253Aachs%26aulast%3DBramson%26aufirst%3DH.%2BN.%26aulast%3DCorona%26aufirst%3DJ.%26aulast%3DDavis%26aufirst%3DS.%2BT.%26aulast%3DDickerson%26aufirst%3DS.%2BH.%26aulast%3DEdelstein%26aufirst%3DM.%26aulast%3DFrye%26aufirst%3DS.%2BV.%26aulast%3DGampe%26aufirst%3DR.%2BT.%26aulast%3DHarris%26aufirst%3DP.%2BA.%26aulast%3DHassell%26aufirst%3DA.%26aulast%3DHolmes%26aufirst%3DW.%2BD.%26aulast%3DHunter%26aufirst%3DR.%2BN.%26aulast%3DLackey%26aufirst%3DK.%2BE.%26aulast%3DLovejoy%26aufirst%3DB.%26aulast%3DLuzzio%26aufirst%3DM.%2BJ.%26aulast%3DMontana%26aufirst%3DV.%26aulast%3DRocque%26aufirst%3DW.%2BJ.%26aulast%3DRusnak%26aufirst%3DD.%26aulast%3DShewchuk%26aufirst%3DL.%26aulast%3DVeal%26aufirst%3DJ.%2BM.%26aulast%3DWalker%26aufirst%3DD.%2BH.%26aulast%3DKuyper%26aufirst%3DL.%2BF.%26atitle%3DOxindole-based%2520inhibitors%2520of%2520cyclin-dependent%2520kinase%25202%2520%2528CDK2%2529%253A%2520design%252C%2520synthesis%252C%2520enzymatic%2520activities%252C%2520and%2520X-ray%2520crystallographic%2520analysis%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2001%26volume%3D44%26spage%3D4339%26epage%3D4358" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group">Gibson, A. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arris, C. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bentley, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boyle, F. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Curtin, N. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davies, T. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Endicott, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Golding, B. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grant, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Griffin, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jewsbury, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johnson, L. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mesguiche, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Newell, D. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Noble, M. E. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tucker, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Whitfield, H. J.</span><span> </span><span class="NLM_article-title">Probing the ATP ribose-binding domain of cyclin-dependent kinases 1 and 2 with O6-substituted guanine derivatives</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">45</span><span class="NLM_x">, </span> <span class="NLM_fpage">3381</span><span class="NLM_x">–</span> <span class="NLM_lpage">3393</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm020056z" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2002&pages=3381-3393&author=A.+E.+Gibsonauthor=C.+E.+Arrisauthor=J.+Bentleyauthor=F.+T.+Boyleauthor=N.+J.+Curtinauthor=T.+G.+Daviesauthor=J.+A.+Endicottauthor=B.+T.+Goldingauthor=S.+Grantauthor=R.+J.+Griffinauthor=P.+Jewsburyauthor=L.+N.+Johnsonauthor=V.+Mesguicheauthor=D.+R.+Newellauthor=M.+E.+M.+Nobleauthor=J.+A.+Tuckerauthor=H.+J.+Whitfield&title=Probing+the+ATP+ribose-binding+domain+of+cyclin-dependent+kinases+1+and+2+with+O6-substituted+guanine+derivatives"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1021%2Fjm020056z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm020056z%26sid%3Dliteratum%253Aachs%26aulast%3DGibson%26aufirst%3DA.%2BE.%26aulast%3DArris%26aufirst%3DC.%2BE.%26aulast%3DBentley%26aufirst%3DJ.%26aulast%3DBoyle%26aufirst%3DF.%2BT.%26aulast%3DCurtin%26aufirst%3DN.%2BJ.%26aulast%3DDavies%26aufirst%3DT.%2BG.%26aulast%3DEndicott%26aufirst%3DJ.%2BA.%26aulast%3DGolding%26aufirst%3DB.%2BT.%26aulast%3DGrant%26aufirst%3DS.%26aulast%3DGriffin%26aufirst%3DR.%2BJ.%26aulast%3DJewsbury%26aufirst%3DP.%26aulast%3DJohnson%26aufirst%3DL.%2BN.%26aulast%3DMesguiche%26aufirst%3DV.%26aulast%3DNewell%26aufirst%3DD.%2BR.%26aulast%3DNoble%26aufirst%3DM.%2BE.%2BM.%26aulast%3DTucker%26aufirst%3DJ.%2BA.%26aulast%3DWhitfield%26aufirst%3DH.%2BJ.%26atitle%3DProbing%2520the%2520ATP%2520ribose-binding%2520domain%2520of%2520cyclin-dependent%2520kinases%25201%2520and%25202%2520with%2520O6-substituted%2520guanine%2520derivatives%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2002%26volume%3D45%26spage%3D3381%26epage%3D3393" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group">Chen, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Norris, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Iwanowicz, E. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spergel, S. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gu, H. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shen, Z. Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wityak, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pang, S. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">de Fex, H. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pitt, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shen, D. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Doweyko, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bassolino, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roberge, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Poss, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, B. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schieven, G. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barrish, J. C.</span><span> </span><span class="NLM_article-title">Discovery and initial SAR of imidazoquinoxalines as inhibitors of the src-family kinase p56(Lck)</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">12</span><span class="NLM_x">, </span> <span class="NLM_fpage">1361</span><span class="NLM_x">–</span> <span class="NLM_lpage">1364</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2002&pages=1361-1364&author=P.+Chenauthor=D.+Norrisauthor=E.+J.+Iwanowiczauthor=S.+H.+Spergelauthor=J.+Linauthor=H.+H.+Guauthor=Z.+Q.+Shenauthor=J.+Wityakauthor=T.+A.+Linauthor=S.+H.+Pangauthor=H.+F.+de+Fexauthor=S.+Pittauthor=D.+R.+Shenauthor=A.+M.+Doweykoauthor=D.+A.+Bassolinoauthor=J.+Robergeauthor=M.+A.+Possauthor=B.+C.+Chenauthor=G.+L.+Schievenauthor=J.+C.+Barrish&title=Discovery+and+initial+SAR+of+imidazoquinoxalines+as+inhibitors+of+the+src-family+kinase+p56%28Lck%29"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DP.%26aulast%3DNorris%26aufirst%3DD.%26aulast%3DIwanowicz%26aufirst%3DE.%2BJ.%26aulast%3DSpergel%26aufirst%3DS.%2BH.%26aulast%3DLin%26aufirst%3DJ.%26aulast%3DGu%26aufirst%3DH.%2BH.%26aulast%3DShen%26aufirst%3DZ.%2BQ.%26aulast%3DWityak%26aufirst%3DJ.%26aulast%3DLin%26aufirst%3DT.%2BA.%26aulast%3DPang%26aufirst%3DS.%2BH.%26aulast%3Dde%2BFex%26aufirst%3DH.%2BF.%26aulast%3DPitt%26aufirst%3DS.%26aulast%3DShen%26aufirst%3DD.%2BR.%26aulast%3DDoweyko%26aufirst%3DA.%2BM.%26aulast%3DBassolino%26aufirst%3DD.%2BA.%26aulast%3DRoberge%26aufirst%3DJ.%26aulast%3DPoss%26aufirst%3DM.%2BA.%26aulast%3DChen%26aufirst%3DB.%2BC.%26aulast%3DSchieven%26aufirst%3DG.%2BL.%26aulast%3DBarrish%26aufirst%3DJ.%2BC.%26atitle%3DDiscovery%2520and%2520initial%2520SAR%2520of%2520imidazoquinoxalines%2520as%2520inhibitors%2520of%2520the%2520src-family%2520kinase%2520p56%2528Lck%2529%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2002%26volume%3D12%26spage%3D1361%26epage%3D1364" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group">Chen, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Iwanowicz, E. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Norris, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gu, H. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moquin, R. V</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Das, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wityak, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spergel, S. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">de Fex, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pang, S. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pitt, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shen, D. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schieven, G. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barrish, J. C.</span><span> </span><span class="NLM_article-title">Synthesis and SAR of novel imidazoquinoxaline-based Lck inhibitors: improvement of cell potency</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">12</span><span class="NLM_x">, </span> <span class="NLM_fpage">3153</span><span class="NLM_x">–</span> <span class="NLM_lpage">3156</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2002&pages=3153-3156&author=P.+Chenauthor=E.+J.+Iwanowiczauthor=D.+Norrisauthor=H.+H.+Guauthor=J.+Linauthor=R.+V+Moquinauthor=J.+Dasauthor=J.+Wityakauthor=S.+H.+Spergelauthor=H.+de+Fexauthor=S.+H.+Pangauthor=S.+Pittauthor=D.+R.+Shenauthor=G.+L.+Schievenauthor=J.+C.+Barrish&title=Synthesis+and+SAR+of+novel+imidazoquinoxaline-based+Lck+inhibitors%3A+improvement+of+cell+potency"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DP.%26aulast%3DIwanowicz%26aufirst%3DE.%2BJ.%26aulast%3DNorris%26aufirst%3DD.%26aulast%3DGu%26aufirst%3DH.%2BH.%26aulast%3DLin%26aufirst%3DJ.%26aulast%3DMoquin%26aufirst%3DR.%2BV%26aulast%3DDas%26aufirst%3DJ.%26aulast%3DWityak%26aufirst%3DJ.%26aulast%3DSpergel%26aufirst%3DS.%2BH.%26aulast%3Dde%2BFex%26aufirst%3DH.%26aulast%3DPang%26aufirst%3DS.%2BH.%26aulast%3DPitt%26aufirst%3DS.%26aulast%3DShen%26aufirst%3DD.%2BR.%26aulast%3DSchieven%26aufirst%3DG.%2BL.%26aulast%3DBarrish%26aufirst%3DJ.%2BC.%26atitle%3DSynthesis%2520and%2520SAR%2520of%2520novel%2520imidazoquinoxaline-based%2520Lck%2520inhibitors%253A%2520improvement%2520of%2520cell%2520potency%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2002%26volume%3D12%26spage%3D3153%26epage%3D3156" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group">Stelmach, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, L. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Patela, S. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pivnichny, J. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scapin, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Singh, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hop, C. E. C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Strauss, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cameron, P. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nichols, E. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">OʼKeefe, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">OʼNeill, E. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schmatz, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schwartz, C. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thompson, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zaller, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Doherty, J. B.</span><span> </span><span class="NLM_article-title">Design and synthesis of potent, orally bioavailable dihydroquinazolinone inhibitors of p38 MAP kinase</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">277</span><span class="NLM_x">–</span> <span class="NLM_lpage">280</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Fjm070654j&amp;key=10.1016%2FS0960-894X%2802%2900752-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Fjm070654j&amp;key=12482439" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Fjm070654j&amp;key=1%3ACAS%3A528%3ADC%252BD38XpsVemu7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2003&pages=277-280&author=J.+E.+Stelmachauthor=L.+P.+Liuauthor=S.+B.+Patelaauthor=J.+V.+Pivnichnyauthor=G.+Scapinauthor=S.+Singhauthor=C.+E.+C.+A.+Hopauthor=Z.+Wangauthor=J.+R.+Straussauthor=P.+M.+Cameronauthor=E.+A.+Nicholsauthor=S.+J.+O%CA%BCKeefeauthor=E.+A.+O%CA%BCNeillauthor=D.+M.+Schmatzauthor=C.+D.+Schwartzauthor=C.+M.+Thompsonauthor=D.+M.+Zallerauthor=J.+B.+Doherty&title=Design+and+synthesis+of+potent%2C+orally+bioavailable+dihydroquinazolinone+inhibitors+of+p38+MAP+kinase"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Design and synthesis of potent, orally bioavailable dihydroquinazolinone inhibitors of p38 MAP kinase</span></div><div class="casAuthors">Stelmach, John E.; Liu, Luping; Patel, Sangita B.; Pivnichny, James V.; Scapin, Giovanna; Singh, Suresh; Hop, Cornelis E. C. A.; Wang, Zhen; Strauss, John R.; Cameron, Patricia M.; Nichols, Elizabeth A.; O'Keefe, Stephen J.; O'Neill, Edward A.; Schmatz, Dennis M.; Schwartz, Cheryl D.; Thompson, Chris M.; Zaller, Dennis M.; Doherty, James B.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">277-280</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ltd.</span>)
        </div><div class="casAbstract">The development of potent, orally bioavailable (in rat) and selective dihydroquinazolinone inhibitors I (R = CO2Me, R1 = H, Br, 2-FC6H4, etc.; R = OMe, R1 = 2-Cl-4-FC6H3, 2,4-F2C6H3, 2-ClC6H4), II (R1 = 2-Cl-4-FC6H3, 2,4-F2C6H3, 2-ClC6H4, X = none, O, NH, CH2, CO, Y = CH, N) and III (R2 = Me, Et, CHMe2, etc.) of p38α MAP kinase is described.  For example, III (R2 = CMe3) was prepd. via the Still coupling of I (R = OSO2CF3, R1 = 2-Cl-4-FC6H3) with 1-tert-butyl-4-(trimethylstannyl)-1,2,3,6-tetrahydropyridine.  These analogs are hybrids of a pyridinylimidazole p38α inhibitor reported by Merck Research Labs. and VX-745.  Optimization of the C-5 Ph and the C-7 piperidinyl substituents led to the identification of III (R2 = CMe3) which gave excellent suppression of TNF-α prodn. in LPS-stimulated whole blood (IC50=10 nM) and good oral exposure in rats (F=68%, AUCn PO=0.58 μM h).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqeEsr0ksZsw7Vg90H21EOLACvtfcHk0lhSz7z07-eeJw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XpsVemu7c%253D&md5=489ab070411bc48e879737d57d98d745</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1016%2FS0960-894X%2802%2900752-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0960-894X%252802%252900752-7%26sid%3Dliteratum%253Aachs%26aulast%3DStelmach%26aufirst%3DJ.%2BE.%26aulast%3DLiu%26aufirst%3DL.%2BP.%26aulast%3DPatela%26aufirst%3DS.%2BB.%26aulast%3DPivnichny%26aufirst%3DJ.%2BV.%26aulast%3DScapin%26aufirst%3DG.%26aulast%3DSingh%26aufirst%3DS.%26aulast%3DHop%26aufirst%3DC.%2BE.%2BC.%2BA.%26aulast%3DWang%26aufirst%3DZ.%26aulast%3DStrauss%26aufirst%3DJ.%2BR.%26aulast%3DCameron%26aufirst%3DP.%2BM.%26aulast%3DNichols%26aufirst%3DE.%2BA.%26aulast%3DO%25CA%25BCKeefe%26aufirst%3DS.%2BJ.%26aulast%3DO%25CA%25BCNeill%26aufirst%3DE.%2BA.%26aulast%3DSchmatz%26aufirst%3DD.%2BM.%26aulast%3DSchwartz%26aufirst%3DC.%2BD.%26aulast%3DThompson%26aufirst%3DC.%2BM.%26aulast%3DZaller%26aufirst%3DD.%2BM.%26aulast%3DDoherty%26aufirst%3DJ.%2BB.%26atitle%3DDesign%2520and%2520synthesis%2520of%2520potent%252C%2520orally%2520bioavailable%2520dihydroquinazolinone%2520inhibitors%2520of%2520p38%2520MAP%2520kinase%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2003%26volume%3D13%26spage%3D277%26epage%3D280" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group">Berset, C.;; Audétat, S.; Tietz, J.; Gunde, T.; Barberis, A.; Schumacher, A.; Traxler, P. </span><span> </span><span class="NLM_article-title">Protein Kinase Inhibitors</span>. WO/2005/120513,<span class="NLM_x"> </span><span class="NLM_year">2005</span><span class="NLM_x">, </span><div class="note"><p class="first last">(Oncalis AG)</p></div>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2005&author=C.%3B+Berset&author=S.+Aud%C3%A9tat&author=J.+Tietz&author=T.+Gunde&author=A.+Barberis&author=A.+Schumacher&author=P.+Traxler&title=Protein+Kinase+Inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DBerset%26aufirst%3DC.%253B%26atitle%3DProtein%2520Kinase%2520Inhibitors%26date%3D2005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group">Brooks, B. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bruccoleri, R. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Olafson, B. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">States, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Swaminathan, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Karplus, M</span><span> </span><span class="NLM_article-title">A program for macromolecular energy, minimization, and dynamics calculations</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1983</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">187</span><span class="NLM_x">–</span> <span class="NLM_lpage">217</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Fjm070654j&amp;key=10.1002%2Fjcc.540040211" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Fjm070654j&amp;key=1%3ACAS%3A528%3ADyaL3sXit1aiu7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=1983&pages=187-217&author=B.+R.+Brooksauthor=R.+E.+Bruccoleriauthor=B.+D.+Olafsonauthor=D.+J.+Statesauthor=S.+Swaminathanauthor=M+Karplus&title=A+program+for+macromolecular+energy%2C+minimization%2C+and+dynamics+calculations"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">CHARMM:  a program for macromolecular energy, minimization, and dynamics calculations</span></div><div class="casAuthors">Brooks, Bernard R.; Bruccoleri, Robert E.; Olafson, Barry D.; States, David J.; Swaminathan, S.; Karplus, Martin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">1983</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">187-217</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    </div><div class="casAbstract">CHARMM (Chem. at HARvard Macromol. Mechanics) is a highly flexible computer program which uses empirical energy functions to model macromol. systems.  The program can read or model build structures, energy minimize them by first- or second-deriv. techniques, perform a normal mode or mol. dynamics simulation, and analyze the structural, equil., and dynamic properties detd. in these calcns.  The operations that CHARMM can perform are described, and some implementation details are given.  A set of parameters for the empirical energy function and a sample run are included.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqBSFuoT3w87LVg90H21EOLACvtfcHk0lj2KrqHovT8TA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL3sXit1aiu7w%253D&md5=bd639b4299ac9934f4497c1a9fe750d2</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1002%2Fjcc.540040211&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.540040211%26sid%3Dliteratum%253Aachs%26aulast%3DBrooks%26aufirst%3DB.%2BR.%26aulast%3DBruccoleri%26aufirst%3DR.%2BE.%26aulast%3DOlafson%26aufirst%3DB.%2BD.%26aulast%3DStates%26aufirst%3DD.%2BJ.%26aulast%3DSwaminathan%26aufirst%3DS.%26aulast%3DKarplus%26aufirst%3DM%26atitle%3DA%2520program%2520for%2520macromolecular%2520energy%252C%2520minimization%252C%2520and%2520dynamics%2520calculations%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D1983%26volume%3D4%26spage%3D187%26epage%3D217" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group">Momany, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rone, R.</span><span> </span><span class="NLM_article-title">Validation of the general purpose QUANTA 3.2/CHARMm force field</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1992</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">888</span><span class="NLM_x">–</span> <span class="NLM_lpage">900</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=1992&pages=888-900&author=F.+Momanyauthor=R.+Rone&title=Validation+of+the+general+purpose+QUANTA+3.2%2FCHARMm+force+field"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMomany%26aufirst%3DF.%26aulast%3DRone%26aufirst%3DR.%26atitle%3DValidation%2520of%2520the%2520general%2520purpose%2520QUANTA%25203.2%252FCHARMm%2520force%2520field%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D1992%26volume%3D13%26spage%3D888%26epage%3D900" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group">No, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grant, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scheraga, H.</span><span> </span><span class="NLM_article-title">Determination of net atomic charges using a modified partial equalization of orbital electronegativity method. 1. Application to neutral molecules as models for polypeptides</span> <span class="citation_source-journal">J. Phys. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1990</span><span class="NLM_x">, </span> <span class="NLM_volume">94</span><span class="NLM_x">, </span> <span class="NLM_fpage">4732</span><span class="NLM_x">–</span> <span class="NLM_lpage">4739</span></span><div class="citationLinks">[<a href="/doi/10.1021/j100374a066" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Fjm070654j&amp;key=1%3ACAS%3A528%3ADyaK3cXktVGgtbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=94&publication_year=1990&pages=4732-4739&author=K.+Noauthor=J.+Grantauthor=H.+Scheraga&title=Determination+of+net+atomic+charges+using+a+modified+partial+equalization+of+orbital+electronegativity+method.+1.+Application+to+neutral+molecules+as+models+for+polypeptides"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Determination of net atomic charges using a modified partial equalization of orbital electronegativity method.  1.  Application to neutral molecules as models for polypeptides</span></div><div class="casAuthors">No, Kyoung Tai; Grant, J. Andrew; Scheraga, Harold A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Physical Chemistry</span>
        (<span class="NLM_cas:date">1990</span>),
    <span class="NLM_cas:volume">94</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">4732-9</span>CODEN:
                <span class="NLM_cas:coden">JPCHAX</span>;
        ISSN:<span class="NLM_cas:issn">0022-3654</span>.
    </div><div class="casAbstract">Modified methods are proposed for detg. net at. charges, based on an existing scheme using the method of partial equalization of orbital electronegativity.  The new methods introduce (as constraints) exptl. dipole and, where possible, quadrupole moments of a set of mols. chosen to resemble the different at. environments found in proteins.  As a consequence, the at. charges reproduce accurately the exptl. values of mol. dipole and quadrupole moments.  The electrostatic potential surface, calcd. with the new net-at.-charge set, compares well to a surface obtained from a moderate-level ab initio calcn. (6-31G**).  However, the new charges are double those obtained from studies in which the charges are detd. from a least-squares fit to the calcd. ab initio electrostatic potential (potential-derived method).  The methodol. outlined leads to a direct calcn. of the permanent mol. charge distribution represented as a set of distributed monopoles, dependent only on the connectivity of the mol.  Hence, the method circumvents assumptions concerning the transferability of charges and can therefore approx. describe any redistribution of charge d. upon assembly of amino acid residues to form a polypeptide.  The method may be of use in providing an approx. representation of mol. charge distributions in extended systems, esp. large polypeptide mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr5hUCBNRO0TLVg90H21EOLACvtfcHk0lj2KrqHovT8TA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3cXktVGgtbw%253D&md5=1783d0a7d1e5c52cb5bf93d96f16d8c6</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1021%2Fj100374a066&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fj100374a066%26sid%3Dliteratum%253Aachs%26aulast%3DNo%26aufirst%3DK.%26aulast%3DGrant%26aufirst%3DJ.%26aulast%3DScheraga%26aufirst%3DH.%26atitle%3DDetermination%2520of%2520net%2520atomic%2520charges%2520using%2520a%2520modified%2520partial%2520equalization%2520of%2520orbital%2520electronegativity%2520method.%25201.%2520Application%2520to%2520neutral%2520molecules%2520as%2520models%2520for%2520polypeptides%26jtitle%3DJ.%2520Phys.%2520Chem.%26date%3D1990%26volume%3D94%26spage%3D4732%26epage%3D4739" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group">No, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grant, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jhon, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scheraga, H.</span><span> </span><span class="NLM_article-title">Determination of net atomic charges using a modified partial equalization of orbital electronegativity method. 2. Application to ionic and aromatic molecules as models for polypeptides</span> <span class="citation_source-journal">J. Phys. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1990</span><span class="NLM_x">, </span> <span class="NLM_volume">94</span><span class="NLM_x">, </span> <span class="NLM_fpage">4740</span><span class="NLM_x">–</span> <span class="NLM_lpage">4746</span></span><div class="citationLinks">[<a href="/doi/10.1021/j100374a067" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Fjm070654j&amp;key=1%3ACAS%3A528%3ADyaK3cXitlCmsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=94&publication_year=1990&pages=4740-4746&author=K.+Noauthor=J.+Grantauthor=M.+Jhonauthor=H.+Scheraga&title=Determination+of+net+atomic+charges+using+a+modified+partial+equalization+of+orbital+electronegativity+method.+2.+Application+to+ionic+and+aromatic+molecules+as+models+for+polypeptides"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Determination of net atomic charges using a modified partial equalization of orbital electronegativity method.  2.  Application to ionic and aromatic molecules as models for polypeptides</span></div><div class="casAuthors">No, Kyoung Tai; Grant, J. Andrew; Jhon, Mu Shik; Scheraga, Harold A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Physical Chemistry</span>
        (<span class="NLM_cas:date">1990</span>),
    <span class="NLM_cas:volume">94</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">4740-6</span>CODEN:
                <span class="NLM_cas:coden">JPCHAX</span>;
        ISSN:<span class="NLM_cas:issn">0022-3654</span>.
    </div><div class="casAbstract">To finalize a scheme intended to calc. directly approx. charge distributions for proteins, parameters for a modified title method (PEOE) are obtained for both ionic and arom. functional groups.  For ionic groups, the electrostatic potentials of several ionic mols. (computed with the 6-31G** basis set) are used as constraints; for arom. groups, the exptl. gas-phase dipole and quadrupole moments of several arom. mols. are used as constraints.  The electrostatic potentials calcd. with the modified PEOE point charges of ionic mols. are quite close to the ab initio electrostatic potential surfaces, and the dipole and quadrupole moments calcd. with the modified PEOE charges of arom. mols. agree well with exptl. data.  The method enables the charge distribution, described as a set of atom-centered monopoles, to be calcd. directly.  This avoids assumptions concerning transferability of charges from small model compds. and describes in an approx. fashion the redistribution of charge upon assembly of amino acid residues into polypeptide mols.  The computational requirements for the procedure are negligible.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrdOLr-aBqHiLVg90H21EOLACvtfcHk0lj2KrqHovT8TA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3cXitlCmsL8%253D&md5=b819eb54db46af0ad66a22d12d052c4b</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1021%2Fj100374a067&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fj100374a067%26sid%3Dliteratum%253Aachs%26aulast%3DNo%26aufirst%3DK.%26aulast%3DGrant%26aufirst%3DJ.%26aulast%3DJhon%26aufirst%3DM.%26aulast%3DScheraga%26aufirst%3DH.%26atitle%3DDetermination%2520of%2520net%2520atomic%2520charges%2520using%2520a%2520modified%2520partial%2520equalization%2520of%2520orbital%2520electronegativity%2520method.%25202.%2520Application%2520to%2520ionic%2520and%2520aromatic%2520molecules%2520as%2520models%2520for%2520polypeptides%26jtitle%3DJ.%2520Phys.%2520Chem.%26date%3D1990%26volume%3D94%26spage%3D4740%26epage%3D4746" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group">Thompson, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Higgins, D. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gibson, T. J.</span><span> </span><span class="NLM_article-title">CLUSTAL W: improving the sensitivity of progressive multiple sequence alignment through sequence weighting, position-specific gap penalties and weight matrix choice</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1994</span><span class="NLM_x">, </span> <span class="NLM_volume">22</span><span class="NLM_x">, </span> <span class="NLM_fpage">4673</span><span class="NLM_x">–</span> <span class="NLM_lpage">4680</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Fjm070654j&amp;key=10.1093%2Fnar%2F22.22.4673" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Fjm070654j&amp;key=7984417" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Fjm070654j&amp;key=1%3ACAS%3A528%3ADyaK2MXitlSgu74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=1994&pages=4673-4680&author=J.+D.+Thompsonauthor=D.+G.+Higginsauthor=T.+J.+Gibson&title=CLUSTAL+W%3A+improving+the+sensitivity+of+progressive+multiple+sequence+alignment+through+sequence+weighting%2C+position-specific+gap+penalties+and+weight+matrix+choice"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">CLUSTAL W: improving the sensitivity of progressive multiple sequence alignment through sequence weighting, position-specific gap penalties and weight matrix choice</span></div><div class="casAuthors">Thompson, Julie D.; Higgins, Desmond G.; Gibson, Toby J.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">4673-80</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The sensitivity of the commonly used progressive multiple sequence alignment method has been greatly improved for the alignment of divergent protein sequences.  Firstly, individual wts. are assigned to each sequence in a partial alignment in order to down-wt. near-duplicate sequences and up-wt. the most divergent ones.  Secondly, amino acid substitution matrixes are varied at different alignment stages according to the divergence of the sequences to be aligned.  Thirdly, residue-specific gap penalties and locally reduced gap penalties in hydrophilic regions encourage new gaps in potential loop regions rather than regular secondary structure.  Fourthly, positions in early alignments where gaps have been opened receive locally reduced gap penalties to encourage the opening of new gaps at these positions.  These modifications are incorporated into a new program, CLUSTAL W which is freely available.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGox0BWqmybJqbVg90H21EOLACvtfcHk0liy_Zd4KWNyQg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXitlSgu74%253D&md5=b219f8f0a8ea118b20a3e23461b99ccf</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1093%2Fnar%2F22.22.4673&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252F22.22.4673%26sid%3Dliteratum%253Aachs%26aulast%3DThompson%26aufirst%3DJ.%2BD.%26aulast%3DHiggins%26aufirst%3DD.%2BG.%26aulast%3DGibson%26aufirst%3DT.%2BJ.%26atitle%3DCLUSTAL%2520W%253A%2520improving%2520the%2520sensitivity%2520of%2520progressive%2520multiple%2520sequence%2520alignment%2520through%2520sequence%2520weighting%252C%2520position-specific%2520gap%2520penalties%2520and%2520weight%2520matrix%2520choice%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D1994%26volume%3D22%26spage%3D4673%26epage%3D4680" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group">Šali, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blundell, T. L.</span><span> </span><span class="NLM_article-title">Comparative protein modeling by satisfaction of spatial restraints</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1993</span><span class="NLM_x">, </span> <span class="NLM_volume">234</span><span class="NLM_x">, </span> <span class="NLM_fpage">779</span><span class="NLM_x">–</span> <span class="NLM_lpage">815</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=234&publication_year=1993&pages=779-815&author=A.+%C5%A0aliauthor=T.+L.+Blundell&title=Comparative+protein+modeling+by+satisfaction+of+spatial+restraints"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3D%25C5%25A0ali%26aufirst%3DA.%26aulast%3DBlundell%26aufirst%3DT.%2BL.%26atitle%3DComparative%2520protein%2520modeling%2520by%2520satisfaction%2520of%2520spatial%2520restraints%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D1993%26volume%3D234%26spage%3D779%26epage%3D815" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group">Marti-Renom, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stuart, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fiser, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sanchez, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Melo, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Šali, A.</span><span> </span><span class="NLM_article-title">Comparative protein structure modeling of genes and genomes</span> <span class="citation_source-journal">Annu. Rev. Biophys. Biomol. Struct.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">29</span><span class="NLM_x">, </span> <span class="NLM_fpage">291</span><span class="NLM_x">–</span> <span class="NLM_lpage">325</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Fjm070654j&amp;key=10.1146%2Fannurev.biophys.29.1.291" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Fjm070654j&amp;key=10940251" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Fjm070654j&amp;key=1%3ACAS%3A528%3ADC%252BD3cXlsFOnur0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2000&pages=291-325&author=M.+A.+Marti-Renomauthor=A.+C.+Stuartauthor=A.+Fiserauthor=R.+Sanchezauthor=F.+Meloauthor=A.+%C5%A0ali&title=Comparative+protein+structure+modeling+of+genes+and+genomes"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Comparative protein structure modeling of genes and genomes</span></div><div class="casAuthors">Marti-Renom, Marc A.; Stuart, Ashley C.; Fiser, Andras; Sanchez, Roberto; Melo, Francisco; Sali, Andrej</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Biophysics and Biomolecular Structure</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">291-325</span>CODEN:
                <span class="NLM_cas:coden">ABBSE4</span>;
        ISSN:<span class="NLM_cas:issn">1056-8700</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews Inc.</span>)
        </div><div class="casAbstract">A review with 212 refs.  Comparative modeling predicts the three-dimensional structure of a given protein sequence (target) based primarily on its alignment to one or more proteins of known structure (templates).  The prediction process consists of fold assignment, target-template alignment, model building, and model evaluation.  The no. of protein sequences that can be modeled and the accuracy of the predictions are increasing steadily because of the growth in the no. of known protein structures and because of the improvements in the modeling software.  Further advances are necessary in recognizing weak sequence-structure similarities, aligning sequences with structures, modeling of rigid body shifts, distortions, loops and side chains, as well as detecting errors in a model.  Despite these problems, it is currently possible to model with useful accuracy significant parts of approx. one third of all known protein sequences.  The use of individual comparative models in biol. is already rewarding and increasingly widespread.  A major new challenge for comparative modeling is the integration of it with the torrents of data from genome sequencing projects as well as from functional and structural genomics.  In particular, there is a need to develop an automated, rapid, robust, sensitive, and accurate comparative modeling pipeline applicable to whole genomes.  Such large-scale modeling is likely to encourage new kinds of applications for the many resulting models, based on their large no. and completeness at the level of the family, organism, or functional network.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq45B6Amw-zi7Vg90H21EOLACvtfcHk0liy_Zd4KWNyQg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXlsFOnur0%253D&md5=d97ffe6c47c4a3bae2bbbf14c045afdd</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1146%2Fannurev.biophys.29.1.291&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev.biophys.29.1.291%26sid%3Dliteratum%253Aachs%26aulast%3DMarti-Renom%26aufirst%3DM.%2BA.%26aulast%3DStuart%26aufirst%3DA.%2BC.%26aulast%3DFiser%26aufirst%3DA.%26aulast%3DSanchez%26aufirst%3DR.%26aulast%3DMelo%26aufirst%3DF.%26aulast%3D%25C5%25A0ali%26aufirst%3DA.%26atitle%3DComparative%2520protein%2520structure%2520modeling%2520of%2520genes%2520and%2520genomes%26jtitle%3DAnnu.%2520Rev.%2520Biophys.%2520Biomol.%2520Struct.%26date%3D2000%26volume%3D29%26spage%3D291%26epage%3D325" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group">Altschul, S. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Madden, T. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schäffer, A. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miller, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lipman, D. J.</span><span> </span><span class="NLM_article-title">Gapped BLAST and PSI-BLAST: a new generation of protein database search programs</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x">, </span> <span class="NLM_fpage">3389</span><span class="NLM_x">–</span> <span class="NLM_lpage">3402</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Fjm070654j&amp;key=10.1093%2Fnar%2F25.17.3389" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Fjm070654j&amp;key=9254694" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Fjm070654j&amp;key=1%3ACAS%3A528%3ADyaK2sXlvFyhu7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=1997&pages=3389-3402&author=S.+F.+Altschulauthor=T.+L.+Maddenauthor=A.+A.+Sch%C3%A4fferauthor=J.+Zhangauthor=Z.+Zhangauthor=W.+Millerauthor=D.+J.+Lipman&title=Gapped+BLAST+and+PSI-BLAST%3A+a+new+generation+of+protein+database+search+programs"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Gapped BLAST and PSI-BLAST: a new generation of protein database search programs</span></div><div class="casAuthors">Altschul, Stephen F.; Madden, Thomas L.; Schaffer, Alejandro A.; Zhang, Jinghui; Zhang, Zheng; Miller, Webb; Lipman, David J.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">3389-3402</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The BLAST programs are widely used tools for searching protein and DNA databases for sequence similarities.  For protein comparisons, a variety of definitional, algorithmic and statistical refinements described here permits the execution time of the BLAST programs to be decreased substantially while enhancing their sensitivity to weak similarities.  A new criterion for triggering the extension of word hits, combined with a new heuristic for generating gapped alignments, yields a gapped BLAST program that runs at approx. three times the speed of the original.  In addn., a method is introduced for automatically combining statistically significant alignments produced by BLAST into a position-specific score matrix, and searching the database using this matrix.  The resulting Position-Specific Iterated BLAST (PSI-BLAST) program runs at approx. the same speed per iteration as gapped BLAST, but in many cases is much more sensitive to weak but biol. relevant sequence similarities.  PSI-BLAST is used to uncover several new and interesting members of the BRCT superfamily.  The source code for the new BLAST programs is available by anonymous ftp from the machine ncbi.nlm.nih.gov, within the directory 'blast', and the programs may be run from NCBIs web site at http://www.ncbi.nlm.nih.gov/.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrfl-GGLfjeoLVg90H21EOLACvtfcHk0liy_Zd4KWNyQg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXlvFyhu7w%253D&md5=4e44123e5984e4aca46a9899d347a176</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1093%2Fnar%2F25.17.3389&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252F25.17.3389%26sid%3Dliteratum%253Aachs%26aulast%3DAltschul%26aufirst%3DS.%2BF.%26aulast%3DMadden%26aufirst%3DT.%2BL.%26aulast%3DSch%25C3%25A4ffer%26aufirst%3DA.%2BA.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DZ.%26aulast%3DMiller%26aufirst%3DW.%26aulast%3DLipman%26aufirst%3DD.%2BJ.%26atitle%3DGapped%2520BLAST%2520and%2520PSI-BLAST%253A%2520a%2520new%2520generation%2520of%2520protein%2520database%2520search%2520programs%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D1997%26volume%3D25%26spage%3D3389%26epage%3D3402" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group">Holm, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sander, C.</span><span> </span><span class="NLM_article-title">Protein structure comparison by alignment of distance matrices</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1993</span><span class="NLM_x">, </span> <span class="NLM_volume">233</span><span class="NLM_x">, </span> <span class="NLM_fpage">123</span><span class="NLM_x">–</span> <span class="NLM_lpage">138</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Fjm070654j&amp;key=10.1006%2Fjmbi.1993.1489" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Fjm070654j&amp;key=8377180" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Fjm070654j&amp;key=1%3ACAS%3A528%3ADyaK3sXms1Gru7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=233&publication_year=1993&pages=123-138&author=L.+Holmauthor=C.+Sander&title=Protein+structure+comparison+by+alignment+of+distance+matrices"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Protein structure comparison by alignment of distance matrixes</span></div><div class="casAuthors">Holm, Liisa; Sander, Chris</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">233</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">123-38</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    </div><div class="casAbstract">With a rapidly growing pool of known tertiary structures, the importance of protein structure comparison parallels that of sequence alignment.  The authors have developed a novel algorithm (Dali) for optimal pairwise alignment of protein structures.  The three-dimensional coordinates of each protein are used to calc. residue-residue (Cα-Cα) distance matrixes.  The distance matrixes are first decompd. into elementary contact patterns, e.g., hexapeptide-hexapeptide submatrixes.  Then, similar contact patterns in the two matrixes are paired and combined into larger consistent sets of pairs.  A Monte Carlo procedure is used to optimize a similarity score defined in terms of equiv. intramol. distances.  Several alignments are optimized in parallel, leading to simultaneous detection of the best, second-best and so on solns.  The method allows sequence gaps of any length, reversal of chain direction and free topol. connectivity of aligned segments.  Sequential connectivity can be imposed as an option.  The method is fully automatic and identifies structural resemblances and common structural cores accurately and sensitively, even in the presence of geometrical distortions.  An all-against-all alignment of over 200 representative protein structures results in an objective classification of known three-dimensional folds in agreement with visual classifications.  Unexpected topol. similarities of biol. interest have been detected, e.g., between the bacterial toxin colicin A and globins and between the eukaryotic POU-specific DNA-binding domain and the bacterial λ repressor.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo_NVXKS4kkyrVg90H21EOLACvtfcHk0ljBcMoFWsNXCg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3sXms1Gru7o%253D&md5=0312147da986d6364977108dfbf3f338</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1006%2Fjmbi.1993.1489&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Fjmbi.1993.1489%26sid%3Dliteratum%253Aachs%26aulast%3DHolm%26aufirst%3DL.%26aulast%3DSander%26aufirst%3DC.%26atitle%3DProtein%2520structure%2520comparison%2520by%2520alignment%2520of%2520distance%2520matrices%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D1993%26volume%3D233%26spage%3D123%26epage%3D138" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group">Laskowski, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">MacArthur, M. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moss, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thornton, J. M.</span><span> </span><span class="NLM_article-title">PROCHECK: a program to check the stereochemical quality of protein structures</span> <span class="citation_source-journal">J. Appl. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1993</span><span class="NLM_x">, </span> <span class="NLM_volume">26</span><span class="NLM_x">, </span> <span class="NLM_fpage">283</span><span class="NLM_x">–</span> <span class="NLM_lpage">291</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Fjm070654j&amp;key=10.1107%2FS0021889892009944" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Fjm070654j&amp;key=1%3ACAS%3A528%3ADyaK3sXit12lurY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=1993&pages=283-291&author=R.+A.+Laskowskiauthor=M.+W.+MacArthurauthor=D.+Mossauthor=J.+M.+Thornton&title=PROCHECK%3A+a+program+to+check+the+stereochemical+quality+of+protein+structures"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">PROCHECK: a program to check the stereochemical quality of protein structures</span></div><div class="casAuthors">Laskowski, Roman A.; MacArthur, Malcolm W.; Moss, David S.; Thornton, Janet M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">283-91</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    </div><div class="casAbstract">The PROCHECK suite of programs provides a detailed check on the stereochem. of a protein structure.  Its outputs comprise a no. of plots in PostScript format and a comprehensive residue-by-residue listing.  These give an assessment of the overall quality of the structure as compared with well-refined structures of the same resoln. and also highlight regions that may need further investigation.  The PROCHECK programs are useful for assessing the quality not only of protein structures in the process of being solved but also of existing structures and of those being modeled on known structures.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrCe5FgJf4KH7Vg90H21EOLACvtfcHk0ljBcMoFWsNXCg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3sXit12lurY%253D&md5=fb69fc4410cd716aaaa7cc0db06b3ed2</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1107%2FS0021889892009944&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0021889892009944%26sid%3Dliteratum%253Aachs%26aulast%3DLaskowski%26aufirst%3DR.%2BA.%26aulast%3DMacArthur%26aufirst%3DM.%2BW.%26aulast%3DMoss%26aufirst%3DD.%26aulast%3DThornton%26aufirst%3DJ.%2BM.%26atitle%3DPROCHECK%253A%2520a%2520program%2520to%2520check%2520the%2520stereochemical%2520quality%2520of%2520protein%2520structures%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D1993%26volume%3D26%26spage%3D283%26epage%3D291" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group">Im, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beglov, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roux, B.</span><span> </span><span class="NLM_article-title">Continuum solvation model: computation of electrostatic forces from numerical solutions to the poisson-boltzmann equation</span> <span class="citation_source-journal">Comput. Phys. Commun.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">111</span><span class="NLM_x">, </span> <span class="NLM_fpage">59</span><span class="NLM_x">–</span> <span class="NLM_lpage">75</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Fjm070654j&amp;key=10.1016%2FS0010-4655%2898%2900016-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Fjm070654j&amp;key=1%3ACAS%3A528%3ADyaK1cXktFGmtr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=111&publication_year=1998&pages=59-75&author=W.+Imauthor=D.+Beglovauthor=B.+Roux&title=Continuum+solvation+model%3A+computation+of+electrostatic+forces+from+numerical+solutions+to+the+poisson-boltzmann+equation"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Continuum Solvation Model: computation of electrostatic forces from numerical solutions to the Poisson-Boltzmann equation</span></div><div class="casAuthors">Im, Wonpil; Beglov, Dmitrii; Roux, Benoit</div><div class="citationInfo"><span class="NLM_cas:title">Computer Physics Communications</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">111</span>
        (<span class="NLM_cas:issue">1-3</span>),
    <span class="NLM_cas:pages">59-75</span>CODEN:
                <span class="NLM_cas:coden">CPHCBZ</span>;
        ISSN:<span class="NLM_cas:issn">0010-4655</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science B.V.</span>)
        </div><div class="casAbstract">A rigorous formulation of the solvation forces (first derivs.) assocd. with the electrostatic free energy calcd. from numerical solns. of the linearized Poisson-Boltzmann equation on a discrete grid is described.  The solvation forces are obtained from the formal soln. of the linearized Poisson-Boltzmann equation written in terms of the Green function.  An intermediate region for the solute-solvent dielec. boundary is introduced to yield a continuous solvation free energy and accurate solvation forces.  A series of numerical tests show that the calcd. forces agree extremely well with finite-difference derivs. of the solvation free energy.  To gain a max. efficiency, the nonpolar contribution to the free energy is expressed in terms of the discretized grid used for the electrostatic problem.  The current treatment of solvation forces can be used to introduce the influence of a continuum solvation model in mol. mechanics calcns. of large biol. systems.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrtvYWWwNDnfbVg90H21EOLACvtfcHk0ljBcMoFWsNXCg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXktFGmtr4%253D&md5=5c6c98db5e70117cacfd4c8eb123a923</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1016%2FS0010-4655%2898%2900016-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0010-4655%252898%252900016-2%26sid%3Dliteratum%253Aachs%26aulast%3DIm%26aufirst%3DW.%26aulast%3DBeglov%26aufirst%3DD.%26aulast%3DRoux%26aufirst%3DB.%26atitle%3DContinuum%2520solvation%2520model%253A%2520computation%2520of%2520electrostatic%2520forces%2520from%2520numerical%2520solutions%2520to%2520the%2520poisson-boltzmann%2520equation%26jtitle%3DComput.%2520Phys.%2520Commun.%26date%3D1998%26volume%3D111%26spage%3D59%26epage%3D75" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group">Feng, B. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simeonov, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jadhav, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Babaoglu, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Inglese, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Austin, C. P. B.</span><span> </span><span class="NLM_article-title">High-throughput screen for aggregation-based inhibition in a large compound library</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x">, </span> <span class="NLM_fpage">2385</span><span class="NLM_x">–</span> <span class="NLM_lpage">2390</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm061317y" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2007&pages=2385-2390&author=B.+Y.+Fengauthor=A.+Simeonovauthor=A.+Jadhavauthor=K.+Babaogluauthor=J.+Ingleseauthor=B.+K.+Shoichetauthor=C.+P.+B.+Austin&title=High-throughput+screen+for+aggregation-based+inhibition+in+a+large+compound+library"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1021%2Fjm061317y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm061317y%26sid%3Dliteratum%253Aachs%26aulast%3DFeng%26aufirst%3DB.%2BY.%26aulast%3DSimeonov%26aufirst%3DA.%26aulast%3DJadhav%26aufirst%3DA.%26aulast%3DBabaoglu%26aufirst%3DK.%26aulast%3DInglese%26aufirst%3DJ.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26aulast%3DAustin%26aufirst%3DC.%2BP.%2BB.%26atitle%3DHigh-throughput%2520screen%2520for%2520aggregation-based%2520inhibition%2520in%2520a%2520large%2520compound%2520library%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2007%26volume%3D50%26spage%3D2385%26epage%3D2390" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">Screening in a spirit haunted world</span> <span class="citation_source-journal">Drug Discovery Today</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">607</span><span class="NLM_x">–</span> <span class="NLM_lpage">615</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Fjm070654j&amp;key=10.1016%2Fj.drudis.2006.05.014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Fjm070654j&amp;key=16793529" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Fjm070654j&amp;key=1%3ACAS%3A528%3ADC%252BD28XlvFGqtLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2006&pages=607-615&author=B.+K.+Shoichet&title=Screening+in+a+spirit+haunted+world"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Screening in a spirit haunted world</span></div><div class="casAuthors">Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">13 & 14</span>),
    <span class="NLM_cas:pages">607-615</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  High-throughput screening (HTS) campaigns can be dominated by hits that ultimately turn out to be non-drug-like.  These 'nuisance' compds. often behave strangely, with steep dose-response curves, absence of clear structure-activity relationships, and high sensitivity to assay conditions.  Several mechanisms contribute to these artifacts, including chem. reactive mols., those that absorb light in assays and those that affect redox conditions.  One of the most common mechanisms behind artifactual inhibition is discussed in this review: at micromolar concns. org. mols. can aggregate to form particles in aq. buffers, and these aggregates can sequester and thereby inhibit protein targets.  Aggregation-based inhibition is baffling from a chem. perspective, but viewed biophys. such behavior is expected.  The range of mols. that behave this way, their rapid detection in a screening environment and their possible biol. implications will be considered here.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpzoEkwI2izy7Vg90H21EOLACvtfcHk0ljBcMoFWsNXCg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XlvFGqtLk%253D&md5=08d22b6b06215ac435824b7664b3674c</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2006.05.014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2006.05.014%26sid%3Dliteratum%253Aachs%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DScreening%2520in%2520a%2520spirit%2520haunted%2520world%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2006%26volume%3D11%26spage%3D607%26epage%3D615" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group">Rockey, W. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Elcock, A. H.</span><span> </span><span class="NLM_article-title">Rapid computational identification of the targets of protein kinase inhibitors</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">48</span><span class="NLM_x">, </span> <span class="NLM_fpage">4138</span><span class="NLM_x">–</span> <span class="NLM_lpage">4152</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm049461b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2005&pages=4138-4152&author=W.+M.+Rockeyauthor=A.+H.+Elcock&title=Rapid+computational+identification+of+the+targets+of+protein+kinase+inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1021%2Fjm049461b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm049461b%26sid%3Dliteratum%253Aachs%26aulast%3DRockey%26aufirst%3DW.%2BM.%26aulast%3DElcock%26aufirst%3DA.%2BH.%26atitle%3DRapid%2520computational%2520identification%2520of%2520the%2520targets%2520of%2520protein%2520kinase%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2005%26volume%3D48%26spage%3D4138%26epage%3D4152" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group">Irwin, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">ZINC—a free database of commercially available compounds for virtual screening</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">45</span><span class="NLM_x">, </span> <span class="NLM_fpage">177</span><span class="NLM_x">–</span> <span class="NLM_lpage">182</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci049714%2B" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Fjm070654j&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVOjt77J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2005&pages=177-182&author=J.+J.+Irwinauthor=B.+K.+Shoichet&title=ZINC%E2%80%94a+free+database+of+commercially+available+compounds+for+virtual+screening"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">ZINC - A Free Database of Commercially Available Compounds for Virtual Screening</span></div><div class="casAuthors">Irwin, John J.; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Computer Sciences</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">177-182</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">0095-2338</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A crit. barrier to entry into structure-based virtual screening is the lack of a suitable, easy to access database of purchasable compds.  We have therefore prepd. a library of 727 842 mols., each with 3D structure, using catalogs of compds. from vendors (the size of this library continues to grow).  The mols. have been assigned biol. relevant protonation states and are annotated with properties such as mol. wt., calcd. LogP, and no. of rotatable bonds.  Each mol. in the library contains vendor and purchasing information and is ready for docking using a no. of popular docking programs.  Within certain limits, the mols. are prepd. in multiple protonation states and multiple tautomeric forms.  In one format, multiple conformations are available for the mols.  This database is available for free download (http://zinc.docking.org) in several common file formats including SMILES, mol2, 3D SDF, and DOCK flexibase format.  A Web-based query tool incorporating a mol. drawing interface enables the database to be searched and browsed and subsets to be created.  Users can process their own mols. by uploading them to a server.  Our hope is that this database will bring virtual screening libraries to a wide community of structural biologists and medicinal chemists.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpPhFw5E2XACbVg90H21EOLACvtfcHk0lj4Qi5MWfMuOg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVOjt77J&md5=e3892b7dc8608b17a3e63541a5ed60e6</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1021%2Fci049714%2B&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci049714%252B%26sid%3Dliteratum%253Aachs%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DZINC%25E2%2580%2594a%2520free%2520database%2520of%2520commercially%2520available%2520compounds%2520for%2520virtual%2520screening%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2005%26volume%3D45%26spage%3D177%26epage%3D182" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group">Kolb, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Caflisch, A.</span><span> </span><span class="NLM_article-title">Automatic and efficient decomposition of two-dimensional structures of small molecules for fragment-based high-throughput docking</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">7384</span><span class="NLM_x">–</span> <span class="NLM_lpage">7392</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm060838i" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2006&pages=7384-7392&author=P.+Kolbauthor=A.+Caflisch&title=Automatic+and+efficient+decomposition+of+two-dimensional+structures+of+small+molecules+for+fragment-based+high-throughput+docking"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1021%2Fjm060838i&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm060838i%26sid%3Dliteratum%253Aachs%26aulast%3DKolb%26aufirst%3DP.%26aulast%3DCaflisch%26aufirst%3DA.%26atitle%3DAutomatic%2520and%2520efficient%2520decomposition%2520of%2520two-dimensional%2520structures%2520of%2520small%2520molecules%2520for%2520fragment-based%2520high-throughput%2520docking%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2006%26volume%3D49%26spage%3D7384%26epage%3D7392" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group">Budin, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Majeux, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Caflisch, A.</span><span> </span><span class="NLM_article-title">Fragment-based flexible ligand docking by evolutionary opimization</span> <span class="citation_source-journal">Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">382</span><span class="NLM_x">, </span> <span class="NLM_fpage">1365</span><span class="NLM_x">–</span> <span class="NLM_lpage">1372</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=382&publication_year=2001&pages=1365-1372&author=N.+Budinauthor=N.+Majeuxauthor=A.+Caflisch&title=Fragment-based+flexible+ligand+docking+by+evolutionary+opimization"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBudin%26aufirst%3DN.%26aulast%3DMajeux%26aufirst%3DN.%26aulast%3DCaflisch%26aufirst%3DA.%26atitle%3DFragment-based%2520flexible%2520ligand%2520docking%2520by%2520evolutionary%2520opimization%26jtitle%3DBiol.%2520Chem.%26date%3D2001%26volume%3D382%26spage%3D1365%26epage%3D1372" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54"><span><span class="NLM_contrib-group">Cecchini, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kolb, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Majeux, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Caflisch, A.</span><span> </span><span class="NLM_article-title">Automated docking of highly flexible ligands by genetic algorithms: A critical assessment</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x">, </span> <span class="NLM_fpage">412</span><span class="NLM_x">–</span> <span class="NLM_lpage">422</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2004&pages=412-422&author=M.+Cecchiniauthor=P.+Kolbauthor=N.+Majeuxauthor=A.+Caflisch&title=Automated+docking+of+highly+flexible+ligands+by+genetic+algorithms%3A+A+critical+assessment"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DCecchini%26aufirst%3DM.%26aulast%3DKolb%26aufirst%3DP.%26aulast%3DMajeux%26aufirst%3DN.%26aulast%3DCaflisch%26aufirst%3DA.%26atitle%3DAutomated%2520docking%2520of%2520highly%2520flexible%2520ligands%2520by%2520genetic%2520algorithms%253A%2520A%2520critical%2520assessment%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2004%26volume%3D25%26spage%3D412%26epage%3D422" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref55"><div class="reference"><strong class="refLabel"><a href="#ref55" class="rightTabRefNumLink">55</a></strong><div class="NLM_citation" id="rightTab-cit55"><span><span class="NLM_contrib-group">Zweig, M. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Campbell, G.</span><span> </span><span class="NLM_article-title">Receiver-operating characteristic (ROC) plots—a fundamental evaluation tool in clinical medicine</span> <span class="citation_source-journal">Clin. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1993</span><span class="NLM_x">, </span> <span class="NLM_volume">39</span><span class="NLM_x">, </span> <span class="NLM_fpage">561</span><span class="NLM_x">–</span> <span class="NLM_lpage">577</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Fjm070654j&amp;key=8472349" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Fjm070654j&amp;key=1%3ACAS%3A528%3ADyaK3sXktVCitLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=1993&pages=561-577&author=M.+H.+Zweigauthor=G.+Campbell&title=Receiver-operating+characteristic+%28ROC%29+plots%E2%80%94a+fundamental+evaluation+tool+in+clinical+medicine"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Receiver-operating characteristic (ROC) plots: a fundamdntal evaluation tool in clinical medicine</span></div><div class="casAuthors">Zweig, Mark H.; Campbell, Gregory</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Chemistry (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">39</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">561-77</span>CODEN:
                <span class="NLM_cas:coden">CLCHAU</span>;
        ISSN:<span class="NLM_cas:issn">0009-9147</span>.
    </div><div class="casAbstract">A review with 77 refs.  The clin. performance of a lab. test can be described in terms of diagnostic accuracy, or the ability to correctly classify subjects into clin. relevant subgroupss.  Diagnostic accuracy refers to the quality of the information provided by the classification device and should be distinguished from the usefulness, or actual practical value, of the information.  Receiver-operating characteristics (ROC) plots provide a pure index of accuracy by demonstrating the limits of a test's ability to discriminate between alternative states of health over the complete spectrum of operating conditions.  Furthermore, ROC plots occupy a central or unifying position in the process of assessing and using diagnostic tools.  Once the plot is generated, a user can readily go on to many other activities such as performing quant. ROC anal. and comparisons of tests, using likelihood ratio to revise the probability of disease in individual subjects, selecting decision thresholds, using logistic-regression anal., using discriminant-function anal., or incorporating the tool into a clin. strategy by using decision anal.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoHGuiPFaxD87Vg90H21EOLACvtfcHk0lj4Qi5MWfMuOg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3sXktVCitLs%253D&md5=0dc685cbe02354d2e65c773355fb5c45</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DZweig%26aufirst%3DM.%2BH.%26aulast%3DCampbell%26aufirst%3DG.%26atitle%3DReceiver-operating%2520characteristic%2520%2528ROC%2529%2520plots%25E2%2580%2594a%2520fundamental%2520evaluation%2520tool%2520in%2520clinical%2520medicine%26jtitle%3DClin.%2520Chem.%26date%3D1993%26volume%3D39%26spage%3D561%26epage%3D577" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1KE5" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1KE5','PDB','1KE5'); return false;">PDB: 1KE5</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1QPE" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1QPE','PDB','1QPE'); return false;">PDB: 1QPE</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1M7Q" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1M7Q','PDB','1M7Q'); return false;">PDB: 1M7Q</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1M17" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1M17','PDB','1M17'); return false;">PDB: 1M17</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1JPA" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1JPA','PDB','1JPA'); return false;">PDB: 1JPA</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1MQB" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1MQB','PDB','1MQB'); return false;">PDB: 1MQB</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1BYG" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1BYG','PDB','1BYG'); return false;">PDB: 1BYG</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1FMK" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1FMK','PDB','1FMK'); return false;">PDB: 1FMK</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1FPU" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1FPU','PDB','1FPU'); return false;">PDB: 1FPU</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1IEP" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1IEP','PDB','1IEP'); return false;">PDB: 1IEP</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1M14" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1M14','PDB','1M14'); return false;">PDB: 1M14</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1M52" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1M52','PDB','1M52'); return false;">PDB: 1M52</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1MP8" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1MP8','PDB','1MP8'); return false;">PDB: 1MP8</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1OPK" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1OPK','PDB','1OPK'); return false;">PDB: 1OPK</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1OPL" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1OPL','PDB','1OPL'); return false;">PDB: 1OPL</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2SRC" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2SRC','PDB','2SRC'); return false;">PDB: 2SRC</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-si1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i35"><a href="/doi/suppl/10.1021/jm070654j">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_38166"></div></div></div></div></div><hr /></hr><p class="last">Depiction of the predicted binding mode of compound <b>14f</b>, histograms of the van der Waals interaction energy and the van der Waals efficiency, all 2D structures of known inhibitors, and enrichment factors for the three-protein one-, two-, and three-parameter models. This material is available free of charge via the Internet at <a class="extLink" href="http://pubs.acs.org">http://pubs.acs.org</a>.</p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/jm070654j/suppl_file/jm070654j-file002.pdf">jm070654j-file002.pdf (677.13 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Fjm070654j&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2008.51.issue-5%3Barticle%3Aarticle%3A10.1021%2Fjm070654j%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/jm070654j" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                730KB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"677a6ef648c53d8e","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
