
<html lang="en"     class="pb-page"  data-request-id="79010901-93c4-42f7-ba48-2b8029dcc5de"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;article:article:10.1021/acs.jmedchem.9b00136;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text;issue:issue:10.1021/jmcmar.2019.62.issue-12"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Synthesis and Structure–Activity Relationships of 3,5-Disubstituted-pyrrolo[2,3-b]pyridines as Inhibitors of Adaptor-Associated Kinase 1 with Antiviral Activity" /></meta><meta name="dc.Creator" content="Sven  Verdonck" /></meta><meta name="dc.Creator" content="Szu-Yuan  Pu" /></meta><meta name="dc.Creator" content="Fiona J.  Sorrell" /></meta><meta name="dc.Creator" content="Jon M.  Elkins" /></meta><meta name="dc.Creator" content="Mathy  Froeyen" /></meta><meta name="dc.Creator" content="Ling-Jie  Gao" /></meta><meta name="dc.Creator" content="Laura I.  Prugar" /></meta><meta name="dc.Creator" content="Danielle E.  Dorosky" /></meta><meta name="dc.Creator" content="Jennifer M.  Brannan" /></meta><meta name="dc.Creator" content="Rina  Barouch-Bentov" /></meta><meta name="dc.Creator" content="Stefan  Knapp" /></meta><meta name="dc.Creator" content="John M.  Dye" /></meta><meta name="dc.Creator" content="Piet  Herdewijn" /></meta><meta name="dc.Creator" content="Shirit  Einav" /></meta><meta name="dc.Creator" content="Steven  De Jonghe" /></meta><meta name="dc.Description" content="There are currently no approved drugs for the treatment of emerging viral infections, such as dengue and Ebola. Adaptor-associated kinase 1 (AAK1) is a cellular serine–threonine protein kinase that..." /></meta><meta name="Description" content="There are currently no approved drugs for the treatment of emerging viral infections, such as dengue and Ebola. Adaptor-associated kinase 1 (AAK1) is a cellular serine–threonine protein kinase that..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="May 28, 2019" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.9b00136" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Copyright © 2019 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.9b00136" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.9b00136" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.9b00136" /></link>
        
    
    

<title>Synthesis and Structure–Activity Relationships of 3,5-Disubstituted-pyrrolo[2,3-b]pyridines as Inhibitors of Adaptor-Associated Kinase 1 with Antiviral Activity | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.9b00136" /></meta><meta property="og:title" content="Synthesis and Structure–Activity Relationships of 3,5-Disubstituted-pyrrolo[2,3-b]pyridines as Inhibitors of Adaptor-Associated Kinase 1 with Antiviral Activity" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-12/acs.jmedchem.9b00136/20190621/images/large/jm-2019-00136f_0026.jpeg" /></meta><meta property="og:description" content="There are currently no approved drugs for the treatment of emerging viral infections, such as dengue and Ebola. Adaptor-associated kinase 1 (AAK1) is a cellular serine–threonine protein kinase that functions as a key regulator of the clathrin-associated host adaptor proteins and regulates the intracellular trafficking of multiple unrelated RNA viruses. Moreover, AAK1 is overexpressed specifically in dengue virus-infected but not bystander cells. Because AAK1 is a promising antiviral drug target, we have embarked on an optimization campaign of a previously identified 7-azaindole analogue, yielding novel pyrrolo[2,3-b]pyridines with high AAK1 affinity. The optimized compounds demonstrate improved activity against dengue virus both in vitro and in human primary dendritic cells and the unrelated Ebola virus. These findings demonstrate that targeting cellular AAK1 may represent a promising broad-spectrum antiviral strategy." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.9b00136"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.9b00136">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-14/achre4.2021.54.issue-14/20210720/achre4.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-29/aamick.2021.13.issue-29/20210728/aamick.2021.13.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-14/acncdm.2021.12.issue-14/20210721/acncdm.2021.12.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-7/amlcef.2021.3.issue-7/20210705/amlcef.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-29/acsodf.2021.6.issue-29/20210727/acsodf.2021.6.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-29/ascecg.2021.9.issue-29/20210726/ascecg.2021.9.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-29/ancham.2021.93.issue-29/20210727/ancham.2021.93.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-29/bichaw.2021.60.issue-29/20210727/bichaw.2021.60.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-27/cgeabj.2021.99.issue-27/20210726-01/cgeabj.2021.99.issue-27.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-7/cgdefu.2021.21.issue-7/20210707/cgdefu.2021.21.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-14/esthag.2021.55.issue-14/20210720/esthag.2021.55.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-29/iecred.2021.60.issue-29/20210728/iecred.2021.60.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-14/inocaj.2021.60.issue-14/20210719/inocaj.2021.60.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-29/jafcau.2021.69.issue-29/20210728/jafcau.2021.69.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-29/jacsat.2021.143.issue-29/20210728/jacsat.2021.143.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-7/jamsef.2021.32.issue-7/20210707/jamsef.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-29/langd5.2021.37.issue-29/20210727/langd5.2021.37.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-7/mpohbp.2021.18.issue-7/20210705/mpohbp.2021.18.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.9b00136&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.9b00136&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.9b00136&amp;href=/doi/10.1021/acs.jmedchem.9b00136" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2019</span><span class="cit-fg-volume">, 62</span><span class="cit-fg-issue">, 12</span><span class="cit-fg-pageRange">, 5810-5831</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/62/12" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.9b00026" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.9b00153" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Synthesis and Structure–Activity Relationships of 3,5-Disubstituted-pyrrolo[2,3-<i>b</i>]pyridines as Inhibitors of Adaptor-Associated Kinase 1 with Antiviral Activity</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Sven Verdonck</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Sven Verdonck</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Medicinal Chemistry, Rega Institute for Medical Research, KU Leuven, Herestraat 49—bus 1041, 3000 Leuven, Belgium</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Sven++Verdonck">Sven Verdonck</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Szu-Yuan Pu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Szu-Yuan Pu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Medicine, Division of Infectious Diseases and Geographic Medicine, and Department of Microbiology and Immunology, Stanford University School of Medicine, Stanford, California 94305, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Szu-Yuan++Pu">Szu-Yuan Pu</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Fiona J. Sorrell</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Fiona J. Sorrell</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Nuffield Department of Clinical Medicine, Target Discovery Institute (TDI) and Structural Genomics Consortium (SGC), University of Oxford, Old Road Campus, Roosevelt Drive, Oxford OX3 7DQ, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Fiona+J.++Sorrell">Fiona J. Sorrell</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Jon M. Elkins</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jon M. Elkins</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Nuffield Department of Clinical Medicine, Target Discovery Institute (TDI) and Structural Genomics Consortium (SGC), University of Oxford, Old Road Campus, Roosevelt Drive, Oxford OX3 7DQ, U.K.</div><div class="loa-info-affiliations-info">Structural Genomics Consortium, Universidade Estadual de Campinas, Cidade Universitária Zeferino Vaz, Av. Dr. André Tosello, 550, Barão Geraldo, Campinas, São Paulo 13083-886, Brazil</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jon+M.++Elkins">Jon M. Elkins</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Mathy Froeyen</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Mathy Froeyen</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Medicinal Chemistry, Rega Institute for Medical Research, KU Leuven, Herestraat 49—bus 1041, 3000 Leuven, Belgium</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Mathy++Froeyen">Mathy Froeyen</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-8675-6540" title="Orcid link">http://orcid.org/0000-0002-8675-6540</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Ling-Jie Gao</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Ling-Jie Gao</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Medicinal Chemistry, Rega Institute for Medical Research, KU Leuven, Herestraat 49—bus 1041, 3000 Leuven, Belgium</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Ling-Jie++Gao">Ling-Jie Gao</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Laura I. Prugar</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Laura I. Prugar</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">US Army Medical Research Institute of Infectious Diseases, Viral Immunology Branch, Fort Detrick, Maryland 21702, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Laura+I.++Prugar">Laura I. Prugar</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Danielle E. Dorosky</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Danielle E. Dorosky</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">US Army Medical Research Institute of Infectious Diseases, Viral Immunology Branch, Fort Detrick, Maryland 21702, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Danielle+E.++Dorosky">Danielle E. Dorosky</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Jennifer M. Brannan</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jennifer M. Brannan</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">US Army Medical Research Institute of Infectious Diseases, Viral Immunology Branch, Fort Detrick, Maryland 21702, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jennifer+M.++Brannan">Jennifer M. Brannan</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Rina Barouch-Bentov</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Rina Barouch-Bentov</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Medicine, Division of Infectious Diseases and Geographic Medicine, and Department of Microbiology and Immunology, Stanford University School of Medicine, Stanford, California 94305, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Rina++Barouch-Bentov">Rina Barouch-Bentov</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Stefan Knapp</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Stefan Knapp</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Nuffield Department of Clinical Medicine, Target Discovery Institute (TDI) and Structural Genomics Consortium (SGC), University of Oxford, Old Road Campus, Roosevelt Drive, Oxford OX3 7DQ, U.K.</div><div class="loa-info-affiliations-info">Institute for Pharmaceutical Chemistry, Buchmann Institute for Life Sciences Campus Riedbeerg, Goethe-University Frankfurt, 60438 Frankfurt am Main, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Stefan++Knapp">Stefan Knapp</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-5995-6494" title="Orcid link">http://orcid.org/0000-0001-5995-6494</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">John M. Dye</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">John M. Dye</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">US Army Medical Research Institute of Infectious Diseases, Viral Immunology Branch, Fort Detrick, Maryland 21702, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=John+M.++Dye">John M. Dye</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Piet Herdewijn</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Piet Herdewijn</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Medicinal Chemistry, Rega Institute for Medical Research, KU Leuven, Herestraat 49—bus 1041, 3000 Leuven, Belgium</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Piet++Herdewijn">Piet Herdewijn</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-3589-8503" title="Orcid link">http://orcid.org/0000-0003-3589-8503</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Shirit Einav</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Shirit Einav</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Medicine, Division of Infectious Diseases and Geographic Medicine, and Department of Microbiology and Immunology, Stanford University School of Medicine, Stanford, California 94305, United States</div></div><span class="conrtib-corresp"><strong>*</strong>E-mail: <a href="/cdn-cgi/l/email-protection#e390868a8d8295a39097828d858c9187cd868796"><span class="__cf_email__" data-cfemail="b5c6d0dcdbd4c3f5c6c1d4dbd3dac7d19bd0d1c0">[email protected]</span></a>. Phone: 650 723 8656 (S.E.).</span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Shirit++Einav">Shirit Einav</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-6441-4171" title="Orcid link">http://orcid.org/0000-0001-6441-4171</a></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Steven De Jonghe</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Steven De Jonghe</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Medicinal Chemistry, Rega Institute for Medical Research, KU Leuven, Herestraat 49—bus 1041, 3000 Leuven, Belgium</div></div><span class="conrtib-corresp"><strong>*</strong>E-mail: <a href="/cdn-cgi/l/email-protection#3f4c4b5a495a51115b5a55505158575a7f544a535a4a495a51115d5a"><span class="__cf_email__" data-cfemail="097a7d6c7f6c67276d6c6366676e616c49627c656c7c7f6c67276b6c">[email protected]</span></a>. Phone: +32 16 32 26 62 (S.D.J.).</span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Steven++De+Jonghe">Steven De Jonghe</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-3872-6558" title="Orcid link">http://orcid.org/0000-0002-3872-6558</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.9b00136&amp;href=/doi/10.1021%2Facs.jmedchem.9b00136" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2019</span></span><span class="cit-volume">, 62</span><span class="cit-issue">, 12</span><span class="cit-pageRange">, 5810–5831</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">May 28, 2019</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>23 January 2019</li><li><span class="item_label"><b>Published</b> online</span>28 May 2019</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 27 June 2019</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.9b00136" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b00136</a></div><div class="article_header-article-copyright"><strong>Copyright © 2019 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D5810%26pageCount%3D22%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DSven%2BVerdonck%252C%2BSzu-Yuan%2BPu%252C%2BFiona%2BJ.%2BSorrell%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D62%26issueNum%3D12%26contentID%3Dacs.jmedchem.9b00136%26title%3DSynthesis%2Band%2BStructure%25E2%2580%2593Activity%2BRelationships%2Bof%2B3%252C5-Disubstituted-pyrrolo%255B2%252C3-b%255Dpyridines%2Bas%2BInhibitors%2Bof%2BAdaptor-Associated%2BKinase%2B1%2Bwith%2BAntiviral%2BActivity%26numPages%3D22%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D5831%26publicationDate%3DJune%2B2019">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.9b00136"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">2750</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">13</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.9b00136" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Synthesis and Structure–Activity Relationships of 3,5-Disubstituted-pyrrolo[2,3-b]pyridines as Inhibitors of Adaptor-Associated Kinase 1 with Antiviral Activity&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Sven&quot;,&quot;last_name&quot;:&quot;Verdonck&quot;},{&quot;first_name&quot;:&quot;Szu-Yuan&quot;,&quot;last_name&quot;:&quot;Pu&quot;},{&quot;first_name&quot;:&quot;Fiona&quot;,&quot;last_name&quot;:&quot;J. Sorrell&quot;},{&quot;first_name&quot;:&quot;Jon&quot;,&quot;last_name&quot;:&quot;M. Elkins&quot;},{&quot;first_name&quot;:&quot;Mathy&quot;,&quot;last_name&quot;:&quot;Froeyen&quot;},{&quot;first_name&quot;:&quot;Ling-Jie&quot;,&quot;last_name&quot;:&quot;Gao&quot;},{&quot;first_name&quot;:&quot;Laura&quot;,&quot;last_name&quot;:&quot;I. Prugar&quot;},{&quot;first_name&quot;:&quot;Danielle&quot;,&quot;last_name&quot;:&quot;E. Dorosky&quot;},{&quot;first_name&quot;:&quot;Jennifer&quot;,&quot;last_name&quot;:&quot;M. Brannan&quot;},{&quot;first_name&quot;:&quot;Rina&quot;,&quot;last_name&quot;:&quot;Barouch-Bentov&quot;},{&quot;first_name&quot;:&quot;Stefan&quot;,&quot;last_name&quot;:&quot;Knapp&quot;},{&quot;first_name&quot;:&quot;John&quot;,&quot;last_name&quot;:&quot;M. Dye&quot;},{&quot;first_name&quot;:&quot;Piet&quot;,&quot;last_name&quot;:&quot;Herdewijn&quot;},{&quot;first_name&quot;:&quot;Shirit&quot;,&quot;last_name&quot;:&quot;Einav&quot;},{&quot;first_name&quot;:&quot;Steven&quot;,&quot;last_name&quot;:&quot;De Jonghe&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2019&quot;,&quot;month&quot;:&quot;05&quot;,&quot;day&quot;:&quot;28&quot;,&quot;issue&quot;:&quot;12&quot;,&quot;volume&quot;:&quot;62&quot;,&quot;pages&quot;:&quot;5810-5831&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.9b00136&quot;},&quot;abstract&quot;:&quot;There are currently no approved drugs for the treatment of emerging viral infections, such as dengue and Ebola. Adaptor-associated kinase 1 (AAK1) is a cellular serine–threonine protein kinase that functions as a key regulator of the clathrin-associated host adaptor proteins and regulates the intracellular trafficking of multiple unrelated RNA viruses. Moreover, AAK1 is overexpressed specifically in dengue virus-infected but not bystander cells. Because AAK1 is a promising antiviral drug target, we have embarked on an optimization campaign of a previously identified 7-azaindole analogue, yielding novel pyrrolo[2,3-b]pyridines with high AAK1 affinity. The optimized compounds demonstrate improved activity against dengue virus both in vitro and in human primary dendritic cells and the unrelated Ebola virus. These findings demonstrate that targeting cellular AAK1 may represent a promising broad-spectrum antiviral strategy.&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b00136&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b00136" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b00136&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b00136" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b00136&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b00136" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.9b00136&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b00136&amp;href=/doi/10.1021/acs.jmedchem.9b00136" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.9b00136" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.9b00136" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (4 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b00136&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.9b00136%26sid%3Dliteratum%253Aachs%26pmid%3D31136173%26genre%3Darticle%26aulast%3DVerdonck%26date%3D2019%26atitle%3DSynthesis%2Band%2BStructure%25E2%2580%2593Activity%2BRelationships%2Bof%2B3%252C5-Disubstituted-pyrrolo%255B2%252C3-b%255Dpyridines%2Bas%2BInhibitors%2Bof%2BAdaptor-Associated%2BKinase%2B1%2Bwith%2BAntiviral%2BActivity%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D62%26issue%3D12%26spage%3D5810%26epage%3D5831%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291430" title="Infectious diseases">Infectious diseases</a>,</li><li><a href="/action/doSearch?ConceptID=291362" title="Antimicrobial agents">Antimicrobial agents</a>,</li><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=291350" title="Assays">Assays</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/62/12" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-12/jmcmar.2019.62.issue-12/20190627/jmcmar.2019.62.issue-12.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-12/acs.jmedchem.9b00136/20190621/images/medium/jm-2019-00136f_0026.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-12/acs.jmedchem.9b00136/20190621/images/large/jm-2019-00136f_0026.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00136&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">There are currently no approved drugs for the treatment of emerging viral infections, such as dengue and Ebola. Adaptor-associated kinase 1 (AAK1) is a cellular serine–threonine protein kinase that functions as a key regulator of the clathrin-associated host adaptor proteins and regulates the intracellular trafficking of multiple unrelated RNA viruses. Moreover, AAK1 is overexpressed specifically in dengue virus-infected but not bystander cells. Because AAK1 is a promising antiviral drug target, we have embarked on an optimization campaign of a previously identified 7-azaindole analogue, yielding novel pyrrolo[2,3-<i>b</i>]pyridines with high AAK1 affinity. The optimized compounds demonstrate improved activity against dengue virus both in vitro and in human primary dendritic cells and the unrelated Ebola virus. These findings demonstrate that targeting cellular AAK1 may represent a promising broad-spectrum antiviral strategy.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_62606" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_62606" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Dengue virus (DENV) is an enveloped, positive-sense, single-stranded RNA virus belonging to the Flaviviridae family. DENV is transmitted by the mosquitoes <i>Aedes aegypti</i> and <i>Aedes albopictus</i>, which mainly reside in (sub)tropical climates. Hence, dengue outbreaks are mainly confined to equatorial areas, where more than 100 countries have been declared dengue-endemic.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> The World Health Organization (WHO) estimates that up to 3.9 billion people are at risk of dengue infection at any given time.<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> In 2013, the WHO reported 3.2 million cases of severe dengue and more than 9000 dengue-related deaths worldwide.<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> Up to 80% of DENV-infected patients remain asymptomatic. Symptomatic patients usually experience an acute febrile illness, characterized by high fever, muscle and joint pain, and sometimes rash.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> The likelihood of progression to severe dengue, manifesting by shock, hemorrhage, and organ failure, is greater upon secondary infection with a heterologous dengue serotype (of four that circulate) due to antibody-dependent enhancement.<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a></div><div class="NLM_p">Ebola virus (EBOV) is a member of the Filoviridae family. Four of the five known EBOV species have been responsible for over 20 outbreaks and over 10,000 deaths since their identification in 1976.<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a></div><div class="NLM_p">Current efforts in search for drugs active against DENV focus primarily on viral targets, such as the NS3 helicase, NS2B-NS3 protease, NS4B, NS5 methyltransferase, NS5 polymerase, and the viral envelope.<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> In search for anti-EBOV drugs, the RNA-dependent RNA polymerase L, the viral surface glycoprotein GP, and viral proteins VP24 and VP35 have been explored as candidate targets.<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> However, targeting viral functions is often associated with the rapid emergence of drug resistance and usually provides a “one drug, one bug” approach. DENV and EBOV rely extensively on host factors for their replication and survival. These cellular factors represent attractive candidate targets for antiviral agents, potentially with a higher barrier to resistance. In addition, when targeting a host function required for the replication of several unrelated viruses, such compounds are more likely to exhibit broad-spectrum antiviral activity.<a onclick="showRef(event, 'ref7 ref8'); return false;" href="javascript:void(0);" class="ref ref7 ref8">(7,8)</a></div><div class="NLM_p">Intracellular membrane trafficking is an example of a cellular process that is hijacked by various viruses.<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> Intracellular membrane trafficking depends on the function of tyrosine- and dileucine-based signals in host cargo proteins, which are recognized by μ1–5 subunits of the clathrin adaptor protein (AP) complexes AP1–5. Adaptor complexes mediate the sorting of cargo proteins to specific membrane compartments within the cell. While AP2 sorts in the endocytic pathway, AP1 and AP4 sort in the secretory pathway.<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a></div><div class="NLM_p">The activity of AP2M1 and AP1M1, the μ subunits of AP2 and AP1, respectively, is regulated by two host cell kinases, adaptor-associated kinase 1 (AAK1) and cyclin G-associated kinase (GAK). Phosphorylation of specific threonine residues in AP2M1 and AP1M1 by these kinases is known to stimulate their binding to tyrosine signals in cargo proteins and enhance vesicle assembly and internalization. Both AAK1 and GAK regulate clathrin-mediated endocytosis by recruiting clathrin and AP2 to the plasma membrane. AAK1 also regulates clathrin-mediated endocytosis of cellular receptors via alternative sorting adaptors that collaborate with AP-2, for example, by phosphorylation of NUMB.<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> Additionally, AAK1 has been implicated in the regulation of epidermal growth factor receptor (EGFR) internalization and recycling to the plasma membrane via its effects on and interactions with alternate endocytic adaptors. We have demonstrated that AAK1 and GAK regulate hepatitis C virus (HCV) entry and assembly by modulating AP2 activity<a onclick="showRef(event, 'ref10 ref11'); return false;" href="javascript:void(0);" class="ref ref10 ref11">(10,11)</a> and viral release and cell-to-cell spread via regulation of AP1.<a onclick="showRef(event, 'ref7 ref12'); return false;" href="javascript:void(0);" class="ref ref7 ref12">(7,12)</a> AAK1 and GAK are also required in the life cycles of DENV and EBOV.<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a></div><div class="NLM_p">We have reported that the approved anticancer drugs sunitinib and erlotinib that potently inhibit AAK1 and GAK, respectively, demonstrate broad-spectrum in vitro antiviral activity against different members of the Flaviviridae family (HCV, DENV, Zika virus, West Nile virus), as well as against various unrelated families of RNA viruses.<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> We have also demonstrated that the combination of these two drugs effectively reduces viral load, morbidity, and mortality in mice infected with DENV and EBOV.<a onclick="showRef(event, 'ref7 ref13'); return false;" href="javascript:void(0);" class="ref ref7 ref13">(7,13)</a> These data provide a proof of concept that small-molecule inhibition of AAK1 and GAK can yield broad-spectrum antiviral agents.<a onclick="showRef(event, 'ref7 ref13'); return false;" href="javascript:void(0);" class="ref ref7 ref13">(7,13)</a> Moreover, using single-cell transcriptomic analysis, AAK1 has been validated as a particularly attractive target because it is overexpressed specifically in DENV-infected and not bystander cells (uninfected cells from the same cell culture), and its expression level increases with cellular virus abundance.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a></div><div class="NLM_p">AAK1 has been studied primarily as a drug target for the treatment of neurological disorders, such as schizophrenia, Parkinson’s disease, neuropathic pain,<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> bipolar disorders, and Alzheimer’s disease.<a onclick="showRef(event, 'ref15 ref16'); return false;" href="javascript:void(0);" class="ref ref15 ref16">(15,16)</a> Consequently, very potent AAK1 inhibitors based on different chemotypes have been disclosed in the patent literature. <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a> shows representative examples of these AAK1 inhibitors and their enzymatic inhibition data.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-12/acs.jmedchem.9b00136/20190621/images/medium/jm-2019-00136f_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-12/acs.jmedchem.9b00136/20190621/images/large/jm-2019-00136f_0001.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Known AAK1 inhibitors.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-12/acs.jmedchem.9b00136/20190621/images/large/jm-2019-00136f_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00136&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Despite the fact that AAK1 has emerged as a promising antiviral target, none of these compound classes have been evaluated and/or optimized for antiviral activity, the only exception being a series of imidazo[1,2-<i>b</i>]pyridazines, which were originally developed by Lexicon Pharmaceuticals (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>).<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> We have previously resynthesized these molecules, confirmed their potent AAK1 affinity, and demonstrated their antiviral activity against HCV and DENV.<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a></div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-12/acs.jmedchem.9b00136/20190621/images/medium/jm-2019-00136f_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-12/acs.jmedchem.9b00136/20190621/images/large/jm-2019-00136f_0005.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. AAK1 inhibitors with documented antiviral activity.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-12/acs.jmedchem.9b00136/20190621/images/large/jm-2019-00136f_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00136&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Here, rather than starting from a potent AAK1 inhibitor, we started from a structurally simple compound with reasonable AAK1 activity from which easy structural variation could be introduced. A screening campaign of 577 structurally diverse compounds (representing kinase inhibitor chemical space) across a panel of 203 protein kinases using the DiscoverX binding assay format previously identified a pyrrolo[2,3-<i>b</i>]pyridine or 7-aza-indole derivative (compound <b>1</b>, <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>) as a potent AAK1 inhibitor (<i>K</i><sub><span class="smallcaps smallerCapital">D</span></sub> = 53 nM).<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> In this manuscript, we describe our efforts to optimize the AAK1 affinity and antiviral activity of compound <b>1</b>.</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-12/acs.jmedchem.9b00136/20190621/images/medium/jm-2019-00136f_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-12/acs.jmedchem.9b00136/20190621/images/large/jm-2019-00136f_0006.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Pyrrolo[2,3-<i>b</i>]pyridine (7-aza-indole) based AAK1 inhibitor.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-12/acs.jmedchem.9b00136/20190621/images/large/jm-2019-00136f_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00136&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_58552" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_58552" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> Chemistry</h3><div id="sec2_1_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i5" class="anchor-spacer"></div><h4 class="article-section__title" id="_i5"> Synthesis of 3-Substituted-5-aryl Pyrrolo[2,3-<i>b</i>]pyridines</h4><div class="NLM_p">A regioselective Sonogashira coupling of trimethylsilylacetylene with commercially available 5-bromo-3-iodo-2-aminopyridine <b>2</b> afforded the alkynyl derivative <b>3</b> (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>).<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> Compound <b>3</b> was then reductively ring-closed with a strong base yielding pyrrolo[2,3-<i>b</i>]pyridine <b>4</b>. Initially, NaH was used as base,<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> but later on potassium <i>tert</i>-butoxide<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> was applied as this gave a cleaner reaction outcome and an improved yield. Initial attempts to nitrate position 3 of the 7-azaindole scaffold employed a mixture of a 65% HNO<sub>3</sub> solution and sulfuric acid.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> However, the desired product was difficult to isolate from this reaction mixture, and therefore, the nitration was performed by treatment of compound <b>4</b> with fuming nitric acid.<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> The 3-nitro derivative <b>5</b> precipitated from the reaction mixture and was conveniently isolated by filtration. Suzuki coupling of compound <b>5</b> with a number of arylboronic acids yielded the 3-nitro-5-aryl-pyrrolo[2,3-<i>b</i>]pyridines <b>6a–h</b> in yields ranging from 65 to 85%.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> Catalytic hydrogenation of the nitro moiety yielded the corresponding amino derivatives <b>7a–h</b>. Because of the instability of the 3-amino-pyrrolo[2,3-<i>b</i>]pyridines, these were not purified and used as such for further reaction. Coupling with an acid chloride in a mixture of pyridine and dichloromethane (DCM)<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> or alternatively, reaction with a carboxylic acid using (benzotriazol-1-yloxy)tris(dimethylamino)phosphonium hexafluorophosphate (BOP) as a coupling reagent<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> yielded a small library of pyrrolo[2,3-<i>b</i>]pyridines <b>8a–o</b>. A sulfonamide derivative <b>8p</b> was prepared via reaction of <b>7g</b> with phenylsulfonyl chloride in pyridine.</div><figure id="sch1" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-12/acs.jmedchem.9b00136/20190621/images/medium/jm-2019-00136f_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-12/acs.jmedchem.9b00136/20190621/images/large/jm-2019-00136f_0013.jpeg" id="gr13" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of 3-Substituted-5-aryl Pyrrolo[2,3-<i>b</i>]pyridines <b>8a–p</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-12/acs.jmedchem.9b00136/20190621/images/large/jm-2019-00136f_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00136&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions. (a) TMSA, Pd(PPh<sub>3</sub>)<sub>2</sub>Cl<sub>2</sub>, CuI, Et<sub>3</sub>N, THF, rt; (b) KO<i>t</i>Bu, NMP, 80 °C; (c) HNO<sub>3</sub>, 0 °C to rt; (d) Pd(PPh<sub>3</sub>)<sub>4</sub>, K<sub>2</sub>CO<sub>3</sub>, ArB(OH)<sub>2</sub>, H<sub>2</sub>O, dioxane, 105 °C; (e) H<sub>2</sub>, Pd/C, THF, rt; (f) RCOCl, pyridine, THF, 1 M NaOH, rt; (g) RCOOH, BOP, Et<sub>3</sub>N, DMF, rt; (h) PhSO<sub>2</sub>Cl, pyridine, rt.</p></p></figure></div><div id="sec2_1_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i7" class="anchor-spacer"></div><h4 class="article-section__title" id="_i7"> Synthesis of 3-Benzoyl-5-(3,4-dimethoxyphenyl)-pyrrolo[2,3-<i>b</i>]pyridine</h4><div class="NLM_p">Formylation of compound <b>4</b> by a Duff reaction<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> yielded 3-formyl-5-bromo-azaindole <b>9</b> (<a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>). The pyrrole nitrogen was protected<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> using NaH and tosylchloride yielding compound <b>10</b>. Suzuki coupling reaction with 3,4-dimethoxyphenylboronic acid furnished compound <b>11</b>. Nucleophilic addition<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> of phenylmagnesium bromide to the aldehyde furnished the secondary alcohol <b>12</b>. Oxidation<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> of the benzylic alcohol using MnO<sub>2</sub> afforded ketone <b>13</b>. Finally, alkaline deprotection<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> of the tosyl group yielded the desired compound <b>14</b>.</div><figure id="sch2" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-12/acs.jmedchem.9b00136/20190621/images/medium/jm-2019-00136f_0014.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-12/acs.jmedchem.9b00136/20190621/images/large/jm-2019-00136f_0014.jpeg" id="gr14" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of 3-Benzoyl-5-(3,4-dimethoxyphenyl)-pyrrolo[2,3-<i>b</i>]pyridine <b>14</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-12/acs.jmedchem.9b00136/20190621/images/large/jm-2019-00136f_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00136&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions. (a) Hexamine, H<sub>2</sub>O, CH<sub>3</sub>COOH, 120 °C; (b) NaH, TsCl, 0 °C to rt; (c) 3,4-dimethoxyphenylboronic acid, Pd(PPh<sub>3</sub>)<sub>4</sub>, 2 M K<sub>2</sub>CO<sub>3</sub>, toluene, EtOH, 105 °C; (d) 3 M PhMgBr, THF, rt; (e) MnO<sub>2</sub>, THF, rt; (f) KOH, EtOH, 80 °C.</p></p></figure></div><div id="sec2_1_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i9" class="anchor-spacer"></div><h4 class="article-section__title" id="_i9"> Synthesis of 3-Phenyl-5-(3,4-dimethoxyphenyl)-pyrrolo[2,3-<i>b</i>]pyridine</h4><div class="NLM_p">Iodination of compound <b>4</b> with <i>N</i>-iodosuccinimide<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> afforded compound <b>15</b> (<a class="ref internalNav" href="#sch3" aria-label="Scheme 3">Scheme 3</a>). Reaction of compound <b>15</b> with phenylboronic acid led only to recovery of unreacted starting material. Therefore, the pyrrole nitrogen of the 7-azaindole scaffold was protected as a tosyl group,<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> affording compound <b>16</b>. A regioselective Suzuki coupling reaction using phenylboronic acid furnished the 3-phenyl-pyrrolo[2,3-<i>b</i>]pyridine analogue <b>17</b>. A subsequent Suzuki reaction<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> with 3,4-dimethoxyphenylboronic acid yielded compound <b>18</b>. Finally, alkaline cleavage of the tosyl protecting group afforded the desired target compound <b>19</b>.<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a></div><figure id="sch3" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-12/acs.jmedchem.9b00136/20190621/images/medium/jm-2019-00136f_0015.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-12/acs.jmedchem.9b00136/20190621/images/large/jm-2019-00136f_0015.jpeg" id="gr15" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthesis of 3-Phenyl-5-(3,4-dimethoxyphenyl)-pyrrolo[2,3-<i>b</i>]pyridine <b>19</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-12/acs.jmedchem.9b00136/20190621/images/large/jm-2019-00136f_0015.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00136&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions. (a) NIS, acetone, rt; (b) NaH, TsCl, THF, 0 °C to rt; (c) PhB(OH)<sub>2</sub>, Pd(PPh<sub>3</sub>)<sub>4</sub>, K<sub>2</sub>CO<sub>3</sub>, toluene, EtOH, H<sub>2</sub>O, 90 °C; (d) 3,4-dimethoxyphenylboronic acid, Pd(PPh<sub>3</sub>)<sub>4</sub>, K<sub>2</sub>CO<sub>3</sub>, toluene, EtOH, H<sub>2</sub>O, 105 °C; (e) KOH, EtOH, 80 °C.</p></p></figure></div><div id="sec2_1_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i11" class="anchor-spacer"></div><h4 class="article-section__title" id="_i11"> Synthesis of 3-Alkynyl-5-(3,4-dimethoxyphenyl)-pyrrolo[2,3-<i>b</i>]pyridines</h4><div class="NLM_p">Sonogashira reaction of compound <b>15</b> with a number of (hetero)arylacetylenes yielded regioselectively compounds <b>20a–f</b> in yields varying from 20 to 70% (<a class="ref internalNav" href="#sch4" aria-label="Scheme 4">Scheme 4</a>).<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> In contrast to Suzuki couplings (<a class="ref internalNav" href="#sch3" aria-label="Scheme 3">Scheme 3</a>), protection of the pyrrole nitrogen was not necessary. Subsequent Suzuki coupling<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> with 3,4-dimethoxyphenylboronic acid gave access to final compounds <b>21a–f</b>.</div><figure id="sch4" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-12/acs.jmedchem.9b00136/20190621/images/medium/jm-2019-00136f_0016.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-12/acs.jmedchem.9b00136/20190621/images/large/jm-2019-00136f_0016.jpeg" id="gr16" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 4. Synthesis of 3-Alkynyl-5-(3,4-dimethoxyphenyl)-pyrrolo[2,3-<i>b</i>]pyridines <b>21a–f</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-12/acs.jmedchem.9b00136/20190621/images/large/jm-2019-00136f_0016.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00136&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions. (a) RC≡CH, Pd(PPh<sub>3</sub>)<sub>2</sub>Cl<sub>2</sub>, CuI, THF, Et<sub>3</sub>N, rt; (b) 3,4-dimethoxyphenylboronic acid, Pd(PPh<sub>3</sub>)<sub>4</sub>, K<sub>2</sub>CO<sub>3</sub>, H<sub>2</sub>O, dioxane, 105 °C.</p></p></figure></div><div id="sec2_1_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i13" class="anchor-spacer"></div><h4 class="article-section__title" id="_i13"> Synthesis of 1-Methyl-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine</h4><div class="NLM_p">Methylation<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> of compound <b>4</b> using NaH and MeI furnished compound <b>22</b> (<a class="ref internalNav" href="#sch5" aria-label="Scheme 5">Scheme 5</a>). Nitration,<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> followed by Suzuki coupling,<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> gave access to compound <b>24</b>. Finally, catalytic reduction of the nitro group, followed by condensation<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> of the exocyclic amino group of compound <b>25</b> with nicotinoyl chloride, yielded target compound <b>26</b>.</div><figure id="sch5" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Scheme 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-12/acs.jmedchem.9b00136/20190621/images/medium/jm-2019-00136f_0017.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-12/acs.jmedchem.9b00136/20190621/images/large/jm-2019-00136f_0017.jpeg" id="gr17" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 5. Synthesis of 1-Methyl-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine <b>26</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-12/acs.jmedchem.9b00136/20190621/images/large/jm-2019-00136f_0017.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00136&amp;id=sch5"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions. (a) NaH, MeI, THF, 0 °C to rt; (b) HNO<sub>3</sub>, 0 °C to rt; (c) 3,4-dimethoxyphenylboronic acid, Pd(PPh<sub>3</sub>)<sub>4</sub>, K<sub>2</sub>CO<sub>3</sub>, H<sub>2</sub>O, dioxane, 105 °C; (d) H<sub>2</sub>, THF, rt; (e) nicotinoyl chloride, pyridine, THF, 1 M NaOH, rt.</p></p></figure></div><div id="sec2_1_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i15" class="anchor-spacer"></div><h4 class="article-section__title" id="_i15"> Synthesis of Pyrazolo[3,4-<i>b</i>]pyridine</h4><div class="NLM_p">Suzuki coupling<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> between commercially available 5-bromo-2-chloronicotinonitrile <b>27</b> and 3,4-dimethoxyphenylboronic acid yielded regioselectively compound <b>28</b>. Nucleophilic displacement of the chlorine by hydrazine, with a concomitant nucleophilic addition at the cyano group,<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> allowed to construct the pyrazole moiety, yielding compound <b>29</b>. Finally, amide formation<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> using nicotinoyl chloride yielded the final compound <b>30</b> (<a class="ref internalNav" href="#sch6" aria-label="Scheme 6">Scheme 6</a>).</div><figure id="sch6" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Scheme 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-12/acs.jmedchem.9b00136/20190621/images/medium/jm-2019-00136f_0018.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-12/acs.jmedchem.9b00136/20190621/images/large/jm-2019-00136f_0018.jpeg" id="gr18" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 6. Synthesis of Pyrazolo[3,4-<i>b</i>]pyridine <b>30</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-12/acs.jmedchem.9b00136/20190621/images/large/jm-2019-00136f_0018.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00136&amp;id=sch6"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions (a) 3,4-dimethoxyphenylboronic acid, Pd(PPh<sub>3</sub>)<sub>4</sub>, K<sub>2</sub>CO<sub>3</sub>, H<sub>2</sub>O, dioxane, 105 °C; (b) 35% hydrazine hydrate, EtOH, 80 °C; (c) nicotinoyl chloride, pyridine, rt.</p></p></figure></div><div id="sec2_1_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i17" class="anchor-spacer"></div><h4 class="article-section__title" id="_i17"> Synthesis of Pyrrolo[2,3-<i>b</i>]pyrazine</h4><div class="NLM_p">Treatment of 2-aminopyrazine <b>31</b> with <i>N</i>-bromosuccinimide yielded the 3,5-dibromopyrazine intermediate <b>32</b> (<a class="ref internalNav" href="#sch7" aria-label="Scheme 7">Scheme 7</a>).<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> A regioselective Sonogashira coupling with trimethylsilylacetylene, followed by a reductive ring closure with potassium <i>tert</i>-butoxide, furnished pyrrolo[2,3-<i>b</i>]pyrazine <b>34</b>.<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> Iodination<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> with <i>N</i>-iodosuccinimide yielded the dihalogenated intermediate <b>35</b> that was subsequently treated with 3-ethynylpyridine<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> and 3,4-dimethoxyphenylboronic acid<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> leading to the desired pyrrolo[2,3-<i>b</i>]pyrazine <b>37</b>.</div><figure id="sch7" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Scheme 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-12/acs.jmedchem.9b00136/20190621/images/medium/jm-2019-00136f_0019.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-12/acs.jmedchem.9b00136/20190621/images/large/jm-2019-00136f_0019.jpeg" id="gr19" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 7. Synthesis of Pyrrolo[2,3-<i>b</i>]pyrazine <b>37</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-12/acs.jmedchem.9b00136/20190621/images/large/jm-2019-00136f_0019.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00136&amp;id=sch7"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions (a) NBS, DMSO, rt; (b) Me<sub>3</sub>SiC≡CH, Pd(PPh<sub>3</sub>)<sub>2</sub>Cl<sub>2</sub>, CuI, THF, Et<sub>3</sub>N, rt; (c) KO<i>t</i>Bu, NMP, 100 °C; (d) NIS, acetone, rt; (e) 3-ethynylpyridine, Pd(PPh<sub>3</sub>)<sub>2</sub>Cl<sub>2</sub>, CuI, THF, Et<sub>3</sub>N, rt; (f) 3,4-dimethoxyphenylboronic acid, Pd(PPh<sub>3</sub>)<sub>4</sub>, K<sub>2</sub>CO<sub>3</sub>, H<sub>2</sub>O, dioxane, 105 °C.</p></p></figure></div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i19" class="anchor-spacer"></div><h3 class="article-section__title" id="_i19"> Kinase Profiling and X-ray Crystallography of Compound <b>1</b></h3><div class="NLM_p">AAK1 is a serine–threonine kinase that belongs to the family of NUMB-associated kinases (NAKs). Other members of this kinase family include BIKE/BMP2K (BMP-2 inducible kinase), GAK (cyclin G-associated kinase), and MPSK1 (myristoylated and palmitoylated serine–threonine kinase 1, also known as STK16). As part of an early profiling of hit compound <b>1</b>, we assessed its selectivity by a binding-displacement assay against each of the four NAK family kinases (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>). Conversion of the experimentally determined IC<sub>50</sub> values to <i>K</i><sub>i</sub> values to allow estimation of the selectivity showed that compound <b>1</b> was 3-fold more selective for AAK1 over GAK, and 8-fold and 22-fold more selective for AAK1 over BMP2K and STK16, respectively (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>).</div><figure id="fig4" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-12/acs.jmedchem.9b00136/20190621/images/medium/jm-2019-00136f_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-12/acs.jmedchem.9b00136/20190621/images/large/jm-2019-00136f_0007.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Binding displacement assay of compound <b>1</b> against the NAK family members.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-12/acs.jmedchem.9b00136/20190621/images/large/jm-2019-00136f_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00136&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Selectivity of Compound <b>1</b> for AAK1 against the NAK Family Kinases</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" colspan="4" align="center">binding displacement assay</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">NAK</th><th class="colsep0 rowsep0" align="center" char=".">IC<sub>50</sub> (μM)</th><th class="colsep0 rowsep0" align="center" char="."><i>K</i><sub>i</sub> (μM)</th><th class="colsep0 rowsep0" align="center" char="."><i>K</i><sub>i</sub>/<i>K</i><sub>i</sub> (AAK1)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">AAK1</td><td class="colsep0 rowsep0" align="char" char=".">1.17</td><td class="colsep0 rowsep0" align="char" char=".">0.541</td><td class="colsep0 rowsep0" align="char" char=".">1.0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">BMP2K</td><td class="colsep0 rowsep0" align="char" char=".">10.1</td><td class="colsep0 rowsep0" align="char" char=".">4.40</td><td class="colsep0 rowsep0" align="char" char=".">8.13</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">GAK</td><td class="colsep0 rowsep0" align="char" char=".">3.25</td><td class="colsep0 rowsep0" align="char" char=".">1.75</td><td class="colsep0 rowsep0" align="char" char=".">3.23</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">STK16</td><td class="colsep0 rowsep0" align="char" char=".">20.0</td><td class="colsep0 rowsep0" align="char" char=".">11.9</td><td class="colsep0 rowsep0" align="char" char=".">22.0</td></tr></tbody></table></div></div><div class="NLM_p">To analyze the binding mode of compound <b>1</b>, the crystal structure of compound <b>1</b> bound to AAK1 to 2.0 Å resolution was determined (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>). Data collection and refinement statistics are provided in Table S1 (see <a href="/doi/suppl/10.1021/acs.jmedchem.9b00136/suppl_file/jm9b00136_si_001.pdf" class="ext-link">Supporting Information</a>), and PDB coordinates are deposited under PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5L4Q">5L4Q</a>. Compound <b>1</b> binds in the ATP-binding site of AAK1 in a relatively planar manner (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>), with the pyrrolo[2,3-<i>b</i>]pyridine moiety bound directly between the side chains of two highly conserved residues: Ala72 from β2 in the kinase N-lobe and Leu183 of the C-lobe, the location where the adenine ring of ATP would bind. The nitrogen atoms of the pyrrolo[2,3-<i>b</i>]pyridine moiety form two hydrogen bonds to the peptide backbone of residues Asp127 and Cys129 at the kinase hinge region (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>). The 4-cyanophenyl moiety is oriented toward the solvent, with the phenyl ring directly sandwiched between the backbone of Gly132 of the hinge and Leu52 of β1 in the N-lobe, and the nitrogen of the cyano moiety forming a polar interaction with the side chain of Asn136. The nicotinamide moiety is bound against the “gatekeeper” residue Met126 and forms a hydrogen bond to the side chain of Lys74, another highly conserved residue that normally links the phosphate of ATP to the αC-helix and is required for correct positioning of the N-lobe for efficient catalysis. Compound <b>1</b> also interacts with two water molecules, one situated at the back of the ATP pocket that bridges between the oxygen of the amide moiety and the backbone nitrogen of Asp194, and a second at the front of the adenosine binding site directly below Val60 in the β1 strand that interacts with the amide nitrogen and surrounding solvent. The DFG motif (Asp194) is in the conformation expected of active AAK1 (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>), with the activation loop of AAK1 in the same conformation as seen in previous AAK1 and BMP2K crystal structures, including the activation-segment C-terminal helix (ASCH), a structural element that is rare among other protein kinases, but conserved across the NAK family.<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a></div><figure id="fig5" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-12/acs.jmedchem.9b00136/20190621/images/medium/jm-2019-00136f_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-12/acs.jmedchem.9b00136/20190621/images/large/jm-2019-00136f_0008.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Crystal structure of AAK1 (gray) in complex with compound <b>1</b> (purple), PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5L4Q">5L4Q</a>. (A) Overview of the crystal structure of AAK1. Highlighted are areas that are important for kinase function. Compound <b>1</b> bound at the hinge is shown. ASCH = activation segment C-terminal Helix, a feature unique to kinases of the NAK family. (B) 2<i>F</i><sub>o</sub> – <i>F</i><sub>c</sub> electron density map contoured at 1σ around compound <b>1</b>, showing the fit of the model to the map. (C,D) Detailed views of the interactions of compound <b>1</b> in the ATP-binding site from two orientations. Black dotted lines indicate polar interactions, and red spheres indicate water molecules. Note that residues 48–63 from β1 and β2 are removed from (C) for clarity.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-12/acs.jmedchem.9b00136/20190621/images/large/jm-2019-00136f_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00136&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i21" class="anchor-spacer"></div><h3 class="article-section__title" id="_i21"> Structure–Activity Relationship Study</h3><div class="NLM_p">All compounds synthesized in this study were evaluated for AAK1 affinity using two different, commercially available AAK1 binding assays. In the early stage of the program, the proprietary KINOMEscan screening platform of DiscoverX was used. In this assay, compounds that bind the kinase active site prevent kinase binding to an immobilized ligand and reduce the amount of kinase captured on the solid support. Hits are then identified by measuring the amount of kinase captured in test versus control samples via quantitative polymerase chain reaction (qPCR) that detects an associated DNA label.<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> Later on in the project, compounds were evaluated by the LanthaScreen Eu kinase binding assay (ThermoFisher Scientific), in which binding of an Alexa Fluor conjugate or “tracer” to a kinase is detected by addition of a Eu-labeled anti-tag antibody. Binding of the tracer and antibody to a kinase results in a high degree of fluorescence resonance energy transfer (FRET), whereas displacement of the tracer with a kinase inhibitor results in loss of FRET.</div><div class="NLM_p">The compounds were also assessed for antiviral activity in human hepatoma (Huh7) cells infected with DENV2. Their effect on overall infection was measured at 48 h postinfection with DENV2 via luciferase assays and the half-maximal effective concentration and the 90% effective concentrations (EC<sub>50</sub> and EC<sub>90</sub> values, respectively) were calculated. In parallel, the cytotoxicity of the compounds (expressed as the half-maximal cytotoxic concentration or CC<sub>50</sub> value) was measured via an alamarBlue assay in the DENV-infected Huh7 cells (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Tables <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>–<named-content content-type="anchor" rid="tbl3" type="simple"></named-content><named-content content-type="anchor" rid="tbl4" type="simple"></named-content><named-content content-type="anchor" rid="tbl5" type="simple"></named-content><a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">6</a>).</div><div class="NLM_p">In all these enzymatic and antiviral assays, sunitinib was included as a positive control (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). Sunitinib has potent AAK1 affinity as measured by the KinomeScan format (<i>K</i><sub><span class="smallcaps smallerCapital">D</span></sub> = 11 nM) and LanthaScreen assay (IC<sub>50</sub> = 47 nM) and displayed potent activity against DENV with an EC<sub>50</sub> value of 1.35 μM.</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. SAR at Position 5 of the 7-Aza-indole Scaffold</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-12/acs.jmedchem.9b00136/20190621/images/medium/jm-2019-00136f_0020.gif" alt="" id="GRAPHIC-d7e1465-autogenerated" /></img><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="2" align="center">AAK1 enzymatic data</th><th class="rowsep1 colsep0" colspan="2" align="center" char=".">DENV antiviral activity</th><th class="rowsep1 colsep0" align="center">cytotoxicity</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">Cmpd#</th><th class="colsep0 rowsep0" align="center">R<sub>5</sub></th><th class="colsep0 rowsep0" align="center">AAK1 <i>K</i><sub>D</sub> (μM) (DiscoverX)</th><th class="colsep0 rowsep0" align="center">AAK1 IC<sub>50</sub> (μM) LanthaScreen</th><th class="colsep0 rowsep0" align="center" char=".">EC<sub>50</sub> (μM)</th><th class="colsep0 rowsep0" align="center" char=".">EC<sub>90</sub> (μM)</th><th class="colsep0 rowsep0" align="center">CC<sub>50</sub> (μM)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>1</b></td><td class="colsep0 rowsep0" align="left">4-cyanophenyl</td><td class="colsep0 rowsep0" align="left">0.120</td><td class="colsep0 rowsep0" align="left">ND<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="char" char=".">8.37</td><td class="colsep0 rowsep0" align="char" char=".">46.8</td><td class="colsep0 rowsep0" align="left">>50</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>8a</b></td><td class="colsep0 rowsep0" align="left">phenyl</td><td class="colsep0 rowsep0" align="left">0.0822</td><td class="colsep0 rowsep0" align="left">ND<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="char" char=".">5.29</td><td class="colsep0 rowsep0" align="char" char=".">87.1</td><td class="colsep0 rowsep0" align="left">NE<a class="ref internalNav" href="#t2fn2" aria-label="b">b</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>8b</b></td><td class="colsep0 rowsep0" align="left">4-F-phenyl</td><td class="colsep0 rowsep0" align="left">0.141</td><td class="colsep0 rowsep0" align="left">ND<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="char" char=".">6.43</td><td class="colsep0 rowsep0" align="char" char=".">25.2</td><td class="colsep0 rowsep0" align="left">NE<a class="ref internalNav" href="#t2fn2" aria-label="b">b</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>8c</b></td><td class="colsep0 rowsep0" align="left">4-CF<sub>3</sub>-phenyl</td><td class="colsep0 rowsep0" align="left">ND<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="left">0.23</td><td class="colsep0 rowsep0" align="char" char=".">2.94</td><td class="colsep0 rowsep0" align="char" char=".">12.0</td><td class="colsep0 rowsep0" align="left">57.0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>8d</b></td><td class="colsep0 rowsep0" align="left">4-OCH<sub>3</sub>-phenyl</td><td class="colsep0 rowsep0" align="left">0.0194</td><td class="colsep0 rowsep0" align="left">ND<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="char" char=".">4.39</td><td class="colsep0 rowsep0" align="char" char=".">25.8</td><td class="colsep0 rowsep0" align="left">NE<a class="ref internalNav" href="#t2fn2" aria-label="b">b</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>8e</b></td><td class="colsep0 rowsep0" align="left">4-Me-phenyl</td><td class="colsep0 rowsep0" align="left">0.0532</td><td class="colsep0 rowsep0" align="left">ND<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="char" char=".">2.60</td><td class="colsep0 rowsep0" align="char" char=".">33.6</td><td class="colsep0 rowsep0" align="left">>100</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>8f</b></td><td class="colsep0 rowsep0" align="left">3-Me-phenyl</td><td class="colsep0 rowsep0" align="left">ND<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="left">0.0186</td><td class="colsep0 rowsep0" align="char" char=".">4.86</td><td class="colsep0 rowsep0" align="char" char=".">10.8</td><td class="colsep0 rowsep0" align="left">16.0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>8g</b></td><td class="colsep0 rowsep0" align="left">3,4-di-OCH<sub>3</sub>-phenyl</td><td class="colsep0 rowsep0" align="left">0.00673</td><td class="colsep0 rowsep0" align="left">0.00432</td><td class="colsep0 rowsep0" align="char" char=".">1.64</td><td class="colsep0 rowsep0" align="char" char=".">7.46</td><td class="colsep0 rowsep0" align="left">39.7</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>8h</b></td><td class="colsep0 rowsep0" align="left">3-thienyl</td><td class="colsep0 rowsep0" align="left">0.0161</td><td class="colsep0 rowsep0" align="left">ND<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="char" char=".">3.01</td><td class="colsep0 rowsep0" align="char" char=".">66.8</td><td class="colsep0 rowsep0" align="left">NE<a class="ref internalNav" href="#t2fn2" aria-label="b">b</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">sunitinib</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">0.0110</td><td class="colsep0 rowsep0" align="left">0.0474</td><td class="colsep0 rowsep0" align="char" char=".">1.35</td><td class="colsep0 rowsep0" align="char" char=".">2.71</td><td class="colsep0 rowsep0" align="left">246</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup><sup>a</sup></sup><p class="last">ND: not determined.</p></div><div class="footnote" id="t2fn2"><sup><sup>b</sup></sup><p class="last">NE: No effect (no apparent effect on cellular viability up to a concentration of 10 μM).</p></div></div></div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. SAR of the N-Acyl Moiety</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-12/acs.jmedchem.9b00136/20190621/images/medium/jm-2019-00136f_0021.gif" alt="" id="GRAPHIC-d7e1724-autogenerated" /></img><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="char" char="." /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" align="center" char=".">AAK1 enzymatic data</th><th class="rowsep1 colsep0" colspan="2" align="center">DENV antiviral activity</th><th class="rowsep1 colsep0" align="center">Cytotoxicity</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">Cmpd#</th><th class="colsep0 rowsep0" align="center">R</th><th class="colsep0 rowsep0" align="center" char=".">AAK1 IC<sub>50</sub> (μM) LanthaScreen</th><th class="colsep0 rowsep0" align="center">EC<sub>50</sub> (μM)</th><th class="colsep0 rowsep0" align="center">EC<sub>90</sub> (μM)</th><th class="colsep0 rowsep0" align="center">CC<sub>50</sub> (μM)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>8g</b></td><td class="colsep0 rowsep0" align="left">3-pyridyl</td><td class="colsep0 rowsep0" align="char" char=".">0.00432</td><td class="colsep0 rowsep0" align="left">1.64</td><td class="colsep0 rowsep0" align="left">7.46</td><td class="colsep0 rowsep0" align="left">39.7</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>8i</b></td><td class="colsep0 rowsep0" align="left">phenyl</td><td class="colsep0 rowsep0" align="char" char=".">0.108</td><td class="colsep0 rowsep0" align="left">4.07</td><td class="colsep0 rowsep0" align="left">50.5</td><td class="colsep0 rowsep0" align="left">17.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>8j</b></td><td class="colsep0 rowsep0" align="left">4-F-phenyl</td><td class="colsep0 rowsep0" align="char" char=".">0.161</td><td class="colsep0 rowsep0" align="left">8.34</td><td class="colsep0 rowsep0" align="left">30.2</td><td class="colsep0 rowsep0" align="left">15.9</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>8k</b></td><td class="colsep0 rowsep0" align="left">4-OCH<sub>3</sub>-phenyl</td><td class="colsep0 rowsep0" align="char" char=".">0.0721</td><td class="colsep0 rowsep0" align="left">NE<a class="ref internalNav" href="#t3fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="left">NE<a class="ref internalNav" href="#t3fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="left">23.6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>8l</b></td><td class="colsep0 rowsep0" align="left">4-pyridyl</td><td class="colsep0 rowsep0" align="char" char=".">0.381</td><td class="colsep0 rowsep0" align="left">12.6</td><td class="colsep0 rowsep0" align="left">26.8</td><td class="colsep0 rowsep0" align="left">NE<a class="ref internalNav" href="#t3fn1" aria-label="a">a</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>8m</b></td><td class="colsep0 rowsep0" align="left">benzyl</td><td class="colsep0 rowsep0" align="char" char=".">0.612</td><td class="colsep0 rowsep0" align="left">142</td><td class="colsep0 rowsep0" align="left">NE<a class="ref internalNav" href="#t3fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="left">NE<a class="ref internalNav" href="#t3fn1" aria-label="a">a</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>8n</b></td><td class="colsep0 rowsep0" align="left">cyclopentyl</td><td class="colsep0 rowsep0" align="char" char=".">0.398</td><td class="colsep0 rowsep0" align="left">3.42</td><td class="colsep0 rowsep0" align="left">8.34</td><td class="colsep0 rowsep0" align="left">15.2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>8o</b></td><td class="colsep0 rowsep0" align="left">cyclohexyl</td><td class="colsep0 rowsep0" align="char" char=".">0.319</td><td class="colsep0 rowsep0" align="left">4.27</td><td class="colsep0 rowsep0" align="left">10.8</td><td class="colsep0 rowsep0" align="left">14.7</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t3fn1"><div class="footnote" id="t3fn1"><sup><sup>a</sup></sup><p class="last">NE: no effect (no apparent antiviral effect or effect on cellular viability up to a concentration of 10 μM).</p></div></div></div><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. SAR of the Linker Moiety at Position 3</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-12/acs.jmedchem.9b00136/20190621/images/medium/jm-2019-00136f_0022.gif" alt="" id="GRAPHIC-d7e1905-autogenerated" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-12/acs.jmedchem.9b00136/20190621/images/medium/jm-2019-00136f_0023.gif" alt="" id="gr20" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t4fn1"><div class="footnote" id="t4fn1"><sup><sup>a</sup></sup><p class="last">NE: no effect (no apparent effect on cellular viability up to a concentration of 10 μM).</p></div></div><div></div></div><div class="NLM_table-wrap" id="tbl5"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 5. SAR of Phenylacetylene Moiety</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-12/acs.jmedchem.9b00136/20190621/images/medium/jm-2019-00136f_0024.gif" alt="" id="GRAPHIC-d7e1919-autogenerated" /></img><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="char" char="." /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" align="center" char=".">AAK1 enzymatic data</th><th class="rowsep1 colsep0" colspan="2" align="center">DENV antiviral activity</th><th class="rowsep1 colsep0" align="center">cytotoxicity</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">Cmpd#</th><th class="colsep0 rowsep0" align="center">R</th><th class="colsep0 rowsep0" align="center" char=".">AAK1 IC<sub>50</sub> (μM) LanthaScreen</th><th class="colsep0 rowsep0" align="center">EC<sub>50</sub> (μM)</th><th class="colsep0 rowsep0" align="center">EC<sub>90</sub> (μM)</th><th class="colsep0 rowsep0" align="center">CC<sub>50</sub> (μM)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>21a</b></td><td class="colsep0 rowsep0" align="left">phenyl</td><td class="colsep0 rowsep0" align="char" char=".">0.209</td><td class="colsep0 rowsep0" align="left">5.64</td><td class="colsep0 rowsep0" align="left">10.58</td><td class="colsep0 rowsep0" align="left">20.1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>21b</b></td><td class="colsep0 rowsep0" align="left">3-pyridyl</td><td class="colsep0 rowsep0" align="char" char=".">0.00402</td><td class="colsep0 rowsep0" align="left">0.72</td><td class="colsep0 rowsep0" align="left">4.16</td><td class="colsep0 rowsep0" align="left">17.0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>21c</b></td><td class="colsep0 rowsep0" align="left">3-OCH<sub>3</sub>-phenyl</td><td class="colsep0 rowsep0" align="char" char=".">0.149</td><td class="colsep0 rowsep0" align="left">NE<a class="ref internalNav" href="#t5fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="left">NE<a class="ref internalNav" href="#t5fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="left">19.6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>21d</b></td><td class="colsep0 rowsep0" align="left">4-CH<sub>3</sub>-phenyl</td><td class="colsep0 rowsep0" align="char" char=".">0.420</td><td class="colsep0 rowsep0" align="left">7.45</td><td class="colsep0 rowsep0" align="left">20.5</td><td class="colsep0 rowsep0" align="left">NE<a class="ref internalNav" href="#t5fn1" aria-label="a">a</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>21e</b></td><td class="colsep0 rowsep0" align="left">4-F-phenyl</td><td class="colsep0 rowsep0" align="char" char=".">0.387</td><td class="colsep0 rowsep0" align="left">4.21</td><td class="colsep0 rowsep0" align="left">8.57</td><td class="colsep0 rowsep0" align="left">19.9</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>21f</b></td><td class="colsep0 rowsep0" align="left">3-CN-phenyl</td><td class="colsep0 rowsep0" align="char" char=".">0.381</td><td class="colsep0 rowsep0" align="left">10.2</td><td class="colsep0 rowsep0" align="left">41.2</td><td class="colsep0 rowsep0" align="left">NE<a class="ref internalNav" href="#t5fn1" aria-label="a">a</a></td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t5fn1"><div class="footnote" id="t5fn1"><sup><sup>a</sup></sup><p class="last">NE: no effect (no apparent antiviral effect or effect on cellular viability up to a concentration of 10 μM).</p></div></div></div><div class="NLM_table-wrap" id="tbl6"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 6. Scaffold Modifications</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-12/acs.jmedchem.9b00136/20190621/images/medium/jm-2019-00136f_0025.gif" alt="" id="gr21" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t6fn1"><div class="footnote" id="t6fn1"><sup><sup>a</sup></sup><p class="last">NE: no effect (no apparent antiviral effect or effect on cellular viability up to a concentration of 10 μM).</p></div></div><div></div></div><div id="sec2_3_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i27" class="anchor-spacer"></div><h4 class="article-section__title" id="_i27"> Structure–Activity Relationship at Position 5 of the 7-Aza-indole Scaffold</h4><div class="NLM_p">We confirmed the AAK1 binding affinity of hit compound <b>1</b>, yet in our hands, a <i>K</i><sub><span class="smallcaps smallerCapital">D</span></sub> value of 120 nM was measured (vs the reported <i>K</i><sub>D</sub> = 53 nM).<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> This hit demonstrated antiviral activity, although it was rather weak (EC<sub>50</sub> = 8.37 μM). Substitution of the 5-(4-cyanophenyl) moiety of compound <b>1</b> by phenyl (compound <b>8a</b>), thienyl (compound <b>8h</b>), and substituted phenyl rings with electron-withdrawing groups (compound <b>8c</b>), electron-donating substituents (compounds <b>8d–g</b>), and a halogen (compound <b>8b</b>) gave rise to a series of analogues with structural variety that were at least equipotent to compound <b>1</b>, suggesting that structural modification at this position is tolerated for AAK1 binding (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). The synthesis of this limited number of analogues allowed us to quickly identify the 5-(3,4-dimethoxyphenyl) congener (compound <b>8g</b>) with low nM AAK1 binding affinity in both the KinomeScan and LanthaScreen assay, indicating an excellent correlation between the two assays. As compound <b>8g</b> showed stronger AAK1 affinity than the positive control sunitinib, no additional efforts were done to further explore the structure–activity relationship (SAR) at this position of the 7-aza-indole scaffold.</div><div class="NLM_p">All analogues within this series had an improved antiviral activity against DENV, relative to the original hit <b>1</b>. The compound with the highest binding affinity for AAK1 (compound <b>8g</b>) also demonstrated the most effective anti-DENV activity, with EC<sub>50</sub> and EC<sub>90</sub> values of 1.64 and 7.46 μM, respectively (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>).</div><figure id="fig6" data-index="13" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-12/acs.jmedchem.9b00136/20190621/images/medium/jm-2019-00136f_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-12/acs.jmedchem.9b00136/20190621/images/large/jm-2019-00136f_0009.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Compounds <b>8g</b> and <b>21b</b> suppress DENV infection more effectively than compound <b>1</b>. Dose response of DENV infection (blue) and cell viability (black) to compounds <b>1</b>, <b>8g</b>, and <b>21b</b> measured by luciferase and alamarBlue assays, respectively, 48 h after infection. Data are plotted relative to vehicle control. At least two representative experiments conducted are shown, each with 5 biological replicates, and the values are expressed as means ± SD.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-12/acs.jmedchem.9b00136/20190621/images/large/jm-2019-00136f_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00136&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_3_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i28" class="anchor-spacer"></div><h4 class="article-section__title" id="_i28"> SAR of the N-Acyl Moiety</h4><div class="NLM_p last">Given the improved AAK1 affinity and antiviral activity of compound <b>8g</b>, the SAR of this compound was further examined through the replacement of the 3-pyridyl group by a number of (hetero)aromatics and cycloaliphatic groups, while the 3,4-dimethoxyphenyl residue was kept fixed (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>). Our findings indicate that the 3-pyridyl moiety is critical for AAK1 binding as all analogues showed a 100-fold decreased AAK1 affinity, when compared to compound <b>8g</b>, giving rise to AAK1 IC<sub>50</sub> values in the 0.1–0.6 μM range. Only the congener with a 4-methoxyphenyl residue (compound <b>8k</b>) was endowed with an enhanced AAK1 affinity with an IC<sub>50</sub> value of 0.072 μM. In correlation with their weaker affinity for AAK1, these compounds had a diminished antiviral activity relative to compound <b>8g</b>.</div></div><div id="sec2_3_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i29" class="anchor-spacer"></div><h4 class="article-section__title" id="_i29"> SAR of the Linker Moiety at Position 3</h4><div class="NLM_p last">To evaluate the importance of the amide linker, a number of surrogates were prepared (<a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>). For synthetic feasibility reasons, compound <b>8i</b>, having a phenyl residue instead of a 3-pyridyl ring, and endowed with quite potent AAK1 affinity and moderate antiviral activity, was selected as a reference compound. Removing the amide linker furnished the 3-phenyl substituted analogue <b>19</b>, which was 4-fold more potent as AAK1 ligand than compound <b>8i</b>, and showed a slightly improved antiviral activity against DENV (EC<sub>50</sub> and EC<sub>90</sub> values of 3.02 and 8.34 μM, respectively). When the amide linker of compound <b>8i</b> was replaced by a ketone (compound <b>14</b>) or alkyne (compound <b>21a</b>) functionality, AAK1 affinity was retained. Compound <b>14</b> exhibited an improved antiviral activity in comparison with compound <b>8i</b>. Finally, replacement of the amide moiety by a sulfonamide linker was not well-tolerated, as compound <b>8p</b> showed close to 200-fold drop in AAK1 affinity.</div></div><div id="sec2_3_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i30" class="anchor-spacer"></div><h4 class="article-section__title" id="_i30"> SAR of Phenylacetylene</h4><div class="NLM_p last">The data in <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a> suggest that the amide moiety is not essential for AAK1 binding and can be replaced. Although the 3-phenyl derivative <b>19</b> displayed potent AAK1 affinity, 3,5-diaryl pyrrolo[2,3-<i>b</i>]pyridines are well known in the literature as kinase inhibitors. In contrast, 3-alkynyl-7-aza-indoles are studied to a much less extent. We, therefore, selected the acetylene derivate <b>21a</b> to decipher if it was possible to improve AAK1 affinity and antiviral activity by further modifying the substitution pattern. A number of substituted phenylacetylene derivatives were prepared (compounds <b>21b–f</b>). The SAR in this series was quite flat, as all compounds displayed very similar AAK1 affinity with IC<sub>50</sub> values in the 0.1–0.4 μM range (<a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">Table <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a></a>). In contrast, the introduction of a 3-pyridylacetylene (compound <b>21b</b>) led to a substantial improvement in AAK1 affinity (IC<sub>50</sub> = 0.0042 μM) with a concomitant improvement in antiviral activity (EC<sub>50</sub> = 0.72 μM) and only a moderate cytotoxic effect (CC<sub>50</sub> = 17 μM, <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>).</div></div><div id="sec2_3_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i31" class="anchor-spacer"></div><h4 class="article-section__title" id="_i31"> Scaffold Modifications</h4><div class="NLM_p last">Lastly, our SAR exploration focused on the 7-aza-indole scaffold itself (<a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">Table <a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">6</a></a>). Methylation of the pyrrole nitrogen afforded compound <b>26</b>, displaying a greatly decreased AAK1 affinity (>800-fold loss in activity relative to compound <b>8g</b>) and antiviral activity (EC<sub>50</sub> > 10 μM). Insertion of an additional nitrogen atom in the pyrrole moiety yielded the pyrazolo[3,4-<i>b</i>]pyridine analogue <b>30</b> that is 100-fold less active as an AAK1 ligand relative to the parent compound <b>8g</b>. When the pyridine moiety of compound <b>21b</b> was replaced by a pyrazine ring, the pyrrolo[2,3-<i>b</i>]pyrazine analogue <b>37</b> was obtained. This compound was endowed with very potent AAK1 affinity (IC<sub>50</sub> = 0.00927 μM), comparable to its 7-aza-indole counterpart <b>21b</b>. Unfortunately, compound <b>37</b> demonstrated greater cytotoxicity than compound <b>21b</b> in Huh7 cells with CC<sub>50</sub>’s of 4.81 versus 17.0 μM, respectively.</div></div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i32" class="anchor-spacer"></div><h3 class="article-section__title" id="_i32"> Broad-Spectrum Antiviral Activity</h3><div class="NLM_p">AAK1 has been demonstrated to be important for the regulation of intracellular viral trafficking of multiple unrelated viruses.<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> Sunitinib, an approved anticancer drug with potent anti-AAK1 activity, displayed antiviral activity against RNA viruses from six different families.<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> To evaluate for potential broad-spectrum antiviral coverage of our molecules beyond DENV infection, hit compound <b>1</b> and the optimized congeners (compounds <b>8g</b> and <b>21b</b>) were tested for their activity against the unrelated EBOV. Huh7 cells were infected with EBOV and treated for 48 h with each compound (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>). Whereas compound <b>1</b> did not show effective activity against EBOV, the more potent AAK1 inhibitors <b>8g</b> and <b>21b</b> displayed anti-EBOV activity with EC<sub>50</sub> values in the low μM range and CC<sub>50</sub> > 10–20 μM.</div><figure id="fig7" data-index="14" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-12/acs.jmedchem.9b00136/20190621/images/medium/jm-2019-00136f_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-12/acs.jmedchem.9b00136/20190621/images/large/jm-2019-00136f_0010.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Compounds <b>1</b>, <b>8g</b>, and <b>21b</b> suppress EBOV infection. Dose response of EBOV infection (blue) and cell viability (black) to compounds <b>1</b>, <b>8g</b>, and <b>21b</b> measured by plaque assay (for compound <b>1</b>) or immunofluorescence assays (for compounds <b>8g</b> and <b>21b</b>) and CellTiter-Glo luminescent cell viability assay in Huh7 cells 48 h after infection. Data are plotted relative to vehicle control. A representative experiment, out of at least two conducted, is shown and the values are shown as means ± SD.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-12/acs.jmedchem.9b00136/20190621/images/large/jm-2019-00136f_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00136&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i33" class="anchor-spacer"></div><h3 class="article-section__title" id="_i33"> Correlation of the Antiviral Effect of Compounds <b>1</b>, <b>8g</b>, and <b>21b</b> with Functional AAK1 Inhibition</h3><div class="NLM_p">To confirm that the observed antiviral activity is correlated with functional inhibition of AAK1 activity, we measured levels of the phosphorylated form of the μ subunit of the AP2 complex, AP2M1, upon treatment with compounds <b>1</b>, <b>21b</b>, and <b>8g</b>. Because AP2M1 phosphorylation is transient (due to phosphatase PP2A activity),<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> to allow capturing of the phosphorylated state, Huh7 cells were incubated for 30 min in the presence of the PP2A inhibitor calyculin A prior to lysis. Treatment with compounds <b>1</b>, <b>21b</b>, and <b>8g</b> reduced AP2M1 phosphorylation (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>), indicating modulation of AP2M1 phosphorylation via AAK1 inhibition.</div><figure id="fig8" data-index="15" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-12/acs.jmedchem.9b00136/20190621/images/medium/jm-2019-00136f_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-12/acs.jmedchem.9b00136/20190621/images/large/jm-2019-00136f_0011.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Antiviral effect of compounds <b>1</b>, <b>8g</b>, and <b>21b</b> correlates with functional inhibition of AAK1. Dose response of AP2M1 phosphorylation to treatment with <b>1</b> (A), <b>8g</b> (B), and <b>21b</b> (C) by western analysis in lysates derived from Huh7 cells. Representative membranes (from two independent experiments) blotted with anti-phospho-AP2M1 (pAP2M1), anti-AP2M1 (AP2M1), and anti-actin (actin) antibodies and quantified data of pAP2M1/actin protein ratio normalized to dimethyl sulfoxide (DMSO) controls are shown. **<i>p</i> < 0.01, ***<i>p</i> < 0.001 by 2-tailed unpaired <i>t</i>-test.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-12/acs.jmedchem.9b00136/20190621/images/large/jm-2019-00136f_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00136&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i34" class="anchor-spacer"></div><h3 class="article-section__title" id="_i34"> Inhibition of DENV Infection in Human Primary Monocyte-Derived Dendritic Cells</h3><div class="NLM_p">To determine the therapeutic potential of the AAK1 inhibitors, their antiviral activity was studied in human primary dendritic cells. Primary cells are a physiologically more relevant model for DENV infection than immortalized cell lines and are considered an ex vivo model for DENV infection.<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> Compounds <b>21b</b> and <b>8g</b> showed a dose-dependent inhibition of DENV infection with EC<sub>50</sub> and EC<sub>90</sub> values of 0.0428 and 1.49 μM and 0.739 and 3.32 μM, respectively (<a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a></a>). This very potent activity in monocyte-derived dendritic cells (MDDCs) associated with minimal cytotoxicity (CC<sub>50</sub> > 20 μM), particularly of compound <b>21b</b> demonstrates the potential of AAK1 inhibitors as antiviral agents.</div><figure id="fig9" data-index="16" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-12/acs.jmedchem.9b00136/20190621/images/medium/jm-2019-00136f_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-12/acs.jmedchem.9b00136/20190621/images/large/jm-2019-00136f_0012.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. Ex vivo antiviral activity of <b>21b</b> and <b>8g</b> in human primary dendritic cells. Dose response of DENV infection (blue) and cell viability (black) to compounds <b>21b</b> (A) and <b>8g</b> (B) measured by plaque assays and alamarBlue assays, respectively, 72 h after infection of primary human MDDCs. A representative experiment with cells from a single donor is shown, out of 2 independent experiments conducted with cells derived from 2 donors, each with 6 biological replicates; the values are shown as means ± SD.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-12/acs.jmedchem.9b00136/20190621/images/large/jm-2019-00136f_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00136&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i35" class="anchor-spacer"></div><h3 class="article-section__title" id="_i35"> Kinase Selectivity</h3><div class="NLM_p">To assess the kinase selectivity of the optimized AAK1 inhibitors, compound <b>21b</b> was screened against 468 kinases via the KINOMEScan assay (DiscoverX) at a single concentration of 10 μM. As can be derived from the kinase interaction map (<a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">10</a></a>), compound <b>21b</b> cannot be considered as a selective AAK1 inhibitor. Beyond AAK1, compound <b>21b</b> targets multiple other kinases (including the other members of the NAK family), which might contribute to its antiviral effect.</div><figure id="fig10" data-index="17" class="article__inlineFigure"><h2 class="fig-label">Figure 10</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-12/acs.jmedchem.9b00136/20190621/images/medium/jm-2019-00136f_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-12/acs.jmedchem.9b00136/20190621/images/large/jm-2019-00136f_0002.jpeg" id="gr10" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 10. Kinome tree of compound <b>21b</b>. Kinases that bind compound <b>21b</b> are marked with red circles. The larger the circle, the stronger the binding affinity.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-12/acs.jmedchem.9b00136/20190621/images/large/jm-2019-00136f_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00136&amp;id=fig10"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">To quantitatively characterize the selectivity of compound <b>21b</b>, selectivity scores (S-scores) were calculated at a concentration of 10 μM<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> (<a class="ref showTableEvent internalNav" data-ID="tbl7" href="#tbl7">Table <a class="ref showTableEvent internalNav" data-ID="tbl7" href="#tbl7">7</a></a>). The S(10) score of <b>21b</b> was calculated by dividing the number of kinases that showed binding at 10% of control or less upon drug treatment by the total number of kinases tested, excluding mutant variants and was found to be 0.342. In comparison, the S(3 μM) score of sunitinib, which is the number of kinases found to bind with a dissociation constant of less than 3 μM divided by the total number of non-mutant kinases tested, is 0.57.<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> Based on these data it can be deduced that, while <b>21b</b> is not a highly selective AAK1 inhibitor, its selectivity profile is improved relative to that of sunitinib,  and it may therefore represent a better pharmacological tool to probe the role of AAK1 in viral infection.</div><div class="NLM_table-wrap" id="tbl7"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 7. Selectivity Scores (<i>S</i> Scores) for Compound <b>21b</b> at 10 μM</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"><i>S</i>-score type</th><th class="colsep0 rowsep0" align="center">number of hits</th><th class="colsep0 rowsep0" align="center">number of nonmutant kinases</th><th class="colsep0 rowsep0" align="center" char=".">selectivity score</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>S</i>(35)<a class="ref internalNav" href="#t7fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="left">212</td><td class="colsep0 rowsep0" align="left">403</td><td class="colsep0 rowsep0" align="char" char=".">0.526</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>S</i>(10)<a class="ref internalNav" href="#t7fn2" aria-label="b">b</a></td><td class="colsep0 rowsep0" align="left">138</td><td class="colsep0 rowsep0" align="left">403</td><td class="colsep0 rowsep0" align="char" char=".">0.342</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>S</i>(1)<a class="ref internalNav" href="#t7fn3" aria-label="c">c</a></td><td class="colsep0 rowsep0" align="left">49</td><td class="colsep0 rowsep0" align="left">403</td><td class="colsep0 rowsep0" align="char" char=".">0.122</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t7fn1"><div class="footnote" id="t7fn1"><sup><sup>a</sup></sup><p class="last"><i>S</i>(35) = (number of nonmutant kinases with % ctrl <35)/(number of non-mutant kinases tested).</p></div><div class="footnote" id="t7fn2"><sup><sup>b</sup></sup><p class="last"><i>S</i>(10) = (number of nonmutant kinases with % ctrl <10)/(number of non-mutant kinases tested).</p></div><div class="footnote" id="t7fn3"><sup><sup>c</sup></sup><p class="last"><i>S</i>(1) = (number of nonmutant kinases with % ctrl <1)/(number of non-mutant kinases tested).</p></div></div></div></div><div id="sec2_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i37" class="anchor-spacer"></div><h3 class="article-section__title" id="_i37"> Molecular Modeling</h3><div class="NLM_p">To rationalize the improved AAK1 affinity of compounds <b>8g</b> and <b>21b</b>, when compared to the original hit <b>1</b>, a docking study was performed using AutoDock Vina.<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> A control docking experiment with the original inhibitor lkb (compound <b>1</b>), present in the 5L4Q PDB file, allowed to reproduce the original X-ray position very well. The best Vina docking scores are reported in <a class="ref showTableEvent internalNav" data-ID="tbl8" href="#tbl8">Table <a class="ref showTableEvent internalNav" data-ID="tbl8" href="#tbl8">8</a></a>. Despite the higher AAK1 affinity of compounds <b>8g</b> and <b>21b</b>, their Vina docking scores are worse than reference compound <b>1</b>. This might be due to the fact that a rigid enzyme in the docking process that blocks induced-fit effects is used.</div><div class="NLM_table-wrap" id="tbl8"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 8. Vina Docking and MM/PBSA Results Calculated on Last 4 ns</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="." /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compound</th><th class="colsep0 rowsep0" align="center" char=".">Vina score (kcal/mol)</th><th class="colsep0 rowsep0" align="center">MM<a class="ref internalNav" href="#t8fn1" aria-label="a">a</a>/GBSA<a class="ref internalNav" href="#t8fn2" aria-label="b">b</a> in kcal/mol (SD<a class="ref internalNav" href="#t8fn4" aria-label="d">d</a>)</th><th class="colsep0 rowsep0" align="center">MM<a class="ref internalNav" href="#t8fn1" aria-label="a">a</a>/PBSA<a class="ref internalNav" href="#t8fn3" aria-label="c">c</a> in kcal/mol (SD<a class="ref internalNav" href="#t8fn4" aria-label="d">d</a>)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>1</b></td><td class="colsep0 rowsep0" align="char" char=".">–10.7</td><td class="colsep0 rowsep0" align="left">–31.5 (1.6)</td><td class="colsep0 rowsep0" align="left">–0.7 (2.3)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>8g</b></td><td class="colsep0 rowsep0" align="char" char=".">–10.3</td><td class="colsep0 rowsep0" align="left">–38.6 (2.2)</td><td class="colsep0 rowsep0" align="left">–3.0 (2.9)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>21b</b></td><td class="colsep0 rowsep0" align="char" char=".">–10.1</td><td class="colsep0 rowsep0" align="left">–40.6 (2.2)</td><td class="colsep0 rowsep0" align="left">–3.1 (2.8)</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t8fn1"><div class="footnote" id="t8fn1"><sup><sup>a</sup></sup><p class="last">MM: molecular mechanics.</p></div><div class="footnote" id="t8fn2"><sup><sup>b</sup></sup><p class="last">GB: generalized born surface area.</p></div><div class="footnote" id="t8fn3"><sup><sup>c</sup></sup><p class="last">PBSA: Poisson–Boltzmann surface area.</p></div><div class="footnote" id="t8fn4"><sup><sup>d</sup></sup><p class="last">SD: standard deviation.</p></div></div></div><div class="NLM_p">Therefore, for the three docked systems with the best Vina docking score, a molecular dynamics (MD) simulation using the Amber 18 software<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> was performed. <a data-tab="pane-pcw-Figures" href="#fig11" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig11" id="" class=" internalNav">11</a></a> shows an overlap of representative structures of compounds <b>1</b>, <b>8g</b>, and <b>21b</b>, as extracted from the MD simulations. The three inhibitors form a hydrogen bond between the NH of the pyrrole ring and the backbone carbonyl group of D127. The nitrogen at position 3 of the pyridine ring seems to be essential for strong AAK1 affinity. In the X-ray of AAK1 with compound <b>1</b>, this nitrogen of the nicotinamide moiety makes a hydrogen bond with K74 side chain. However, this hydrogen/ionic bond is not maintained in the MD simulations of AAK1 with compound <b>1</b>, and neither with compounds <b>8g</b> and <b>21b</b>. On the other hand, for the three compounds, frequent van der Waals contacts between this nitrogen and the charged nitrogen in the side chain of K74 (distances < 4 Å) are observed.</div><figure id="fig11" data-index="18" class="article__inlineFigure"><h2 class="fig-label">Figure 11</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-12/acs.jmedchem.9b00136/20190621/images/medium/jm-2019-00136f_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-12/acs.jmedchem.9b00136/20190621/images/large/jm-2019-00136f_0003.jpeg" id="gr11" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 11. AAK1 snapshot structures with 3 ligands, compound <b>1</b> (green, lightgreen), compound <b>8g</b> (magenta, rose), and compound <b>21b</b> (cyan; lightblue), extracted from the last 4 ns of the MD trajectory as a representative structure from the biggest cluster using the average linkage clustering method available in cpptraj. The AAK1 and lkb X-ray structures are also shown (orange, yellow). Image made by the Chimera software.<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-12/acs.jmedchem.9b00136/20190621/images/large/jm-2019-00136f_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00136&amp;id=fig11"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">For the last 4 ns of the trajectories, MM/GBSA and MM/PBSA (molecular mechanics energies with a generalized Born surface area continuum solvation model and molecular mechanics energies combined with the Poisson–Boltzmann surface area continuum solvation model, respectively)<a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a> binding affinity calculations were conducted. No entropy contributions were calculated (<a class="ref showTableEvent internalNav" data-ID="tbl8" href="#tbl8">Table <a class="ref showTableEvent internalNav" data-ID="tbl8" href="#tbl8">8</a></a>). From the data in <a class="ref showTableEvent internalNav" data-ID="tbl8" href="#tbl8">Table <a class="ref showTableEvent internalNav" data-ID="tbl8" href="#tbl8">8</a></a>, it is clear (more from the general born model than from the Poisson–Boltzmann model) that compounds <b>8g</b> and <b>21b</b> are stronger AAK1 binders than reference compound <b>1</b>. Both methoxy groups of compounds <b>8g</b> and <b>21b</b> contribute to the binding energy via van der Waals interactions with specific amino acids of AAK1. Residues L52, L62, F128, C129, R130, G131, G132, Q133, V135, and N136 are involved in binding to compound <b>21b</b>. The same residues plus E50 make contact with compound <b>8g</b>. On the other hand, the nitrile function of compound <b>1</b> shows clearly less van der Waals contacts with surrounding residues (L52, G132, Q133, and N136, <a data-tab="pane-pcw-Figures" href="#fig12" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig12" id="" class=" internalNav">12</a></a>).</div><figure id="fig12" data-index="19" class="article__inlineFigure"><h2 class="fig-label">Figure 12</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-12/acs.jmedchem.9b00136/20190621/images/medium/jm-2019-00136f_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-12/acs.jmedchem.9b00136/20190621/images/large/jm-2019-00136f_0004.jpeg" id="gr12" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 12. AAK1 enzyme with compound <b>21b</b> (same snapshot as in <a data-tab="pane-pcw-Figures" href="#fig11" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig11" id="" class=" internalNav">11</a></a>). Amino acids making contact with the two methoxy groups (heavy atom distances < 4 Å) are colored in yellow and magenta. The magenta color represents also the residues having van der Waals contacts with the nitrile group of compound <b>1</b>. Image made by the Chimera software.<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-12/acs.jmedchem.9b00136/20190621/images/large/jm-2019-00136f_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00136&amp;id=fig12"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i39">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_52821" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_52821" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">AAK1 is a promising host target for the development of broad-spectrum antiviral agents. In this article, the first systematic SAR study of AAK1 inhibitors as antiviral agents is presented. Starting from a known ligand with moderate AAK1 binding affinity and anti-DENV activity, a systematic SAR study was carried out. It led to the discovery of AAK1 ligands with low nM AAK1 binding affinity that display improved antiviral activity against DENV. Moreover, the optimized AAK1 inhibitors exhibit very potent activity in DENV-infected primary dendritic cells and anti-EBOV activity, supporting the potential to develop broad-spectrum antiviral agents based on AAK1 inhibition. Kinase profiling revealed that these compounds have an improved selectivity profile relative to sunitinib, yet further research is necessary to improve their kinase selectivity.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i40">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_62218" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_62218" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">All commercial reagents were obtained via Acros Organics, Sigma-Aldrich, AK Scientific and Fluorochem at least 99% purity unless indicated otherwise. Furthermore, all dry solvents were obtained via Acros Organics with an AcroSeal system, and regular solvents were obtained via Fisher Scientific at technical grade. Thin layer chromatography (TLC) of the reactions was performed on silica gel on aluminum foils (60 Å pore diameter) obtained from Sigma-Aldrich and visualized using ultraviolet light. Recording of the NMR spectra was performed using a Bruker 300, 500, or 600 MHz spectrometer. The chemical shifts are reported in ppm with Me<sub>4</sub>Si as the reference and coupling constants (<i>J</i>) are reported in hertz. Mass spectra were acquired on a quadrupole orthogonal acceleration time-of-flight mass spectrometer (Synapt G2 HDMS, Waters, Milford, MA). Samples were infused at 3 μL/min and spectra were obtained in positive or negative ionization mode with a resolution of 15 000 (full width at half-maximum) using leucine enkephalin as lock mass. Purity of the compounds was analyzed on a Waters 600 HPLC system equipped with a Waters 2487 dual λ absorbance detector set at 256 nm using a 5 μm 4.6 × 150 mm XBridge reversed phase (C<sub>18</sub>) column. The mobile phase was a gradient over 30 min starting from 95% <b>A</b> and 5% <b>B</b> and finishing at 5% <b>A</b> and 95% <b>B</b> with a flow rate of 1 mL per minute (solvent <b>A</b>: Milli-Q water; solvent <b>B</b>: acetonitrile). All synthesized final compounds had a purity of at least 95%. Compound <b>8o</b> was synthesized according to literature procedure.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a></div><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i41" class="anchor-spacer"></div><h3 class="article-section__title" id="_i41"> 5-Bromo-3-((trimethylsilyl)ethynyl)pyridin-2-amine (<b>3</b>)</h3><div class="NLM_p last">To a stirring suspension of 5-bromo-3-iodopyridin-2-amine <b>2</b> (520 mg, 1.74 mmol) in degassed triethylamine (10 mL) were added copper iodide (6.63 mg, 0.034 mmol) and Pd(PPh<sub>3</sub>)<sub>2</sub>Cl<sub>2</sub> (12 mg, 0.017 mmol). The system was flushed with nitrogen, and trimethylsilylacetylene (188 mg, 265 μL, 1.91 mmol) was added dropwise over 5 min. The reaction was allowed to stir for 3 h at room temperature. After reaction completion, the solvent was evaporated and water was added. The resulting suspension was extracted three times using ethyl acetate. The combined organic phases were washed with water and brine, dried over MgSO<sub>4</sub>, and evaporated in vacuo. The crude residue was purified by silica gel flash column chromatography using a mixture of heptane and ethylacetate (in a ratio of 80:20) as a mobile phase, yielding the title compound as a yellow-beige solid (420 mg, 90%). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ (ppm) 8.03 (s, 1H, ArH), 7.69 (s, 1H, ArH), 6.37 (s, 2H, NH<sub>2</sub>), 0.24 (s, 9H, Si(CH<sub>3</sub>)<sub>3</sub>).</div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i42" class="anchor-spacer"></div><h3 class="article-section__title" id="_i42"> 5-Bromo-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine (<b>4</b>)</h3><div class="NLM_p last">To a solution of 5-bromo-3-((trimethylsilyl)ethynyl)pyridin-2-amine <b>3</b> (200 mg, 0.743 mmol) in dry NMP (5 mL) was added portionwise KO<i>t</i>Bu (100 mg, 0.891 mmol). The reaction mixture was heated to 80 °C and stirred for 1 h. After reaction completion, the mixture was extracted with water and ethyl acetate three times. The combined organic phases were washed twice with water and once with brine, dried over MgSO<sub>4</sub>, and evaporated in vacuo. The crude residue was then purified using silica gel flash column chromatography (using a mixture of heptane and ethyl acetate in a ratio of 5:1 as mobile phase), yielding the title compound as a white solid (103 mg, 70%). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ (ppm) 10.33 (br s, 1H, NH), 8.37 (d, <i>J</i> = 2.1 Hz, 1H, ArH), 8.09 (d, <i>J</i> = 2.1 Hz, 1H, ArH), 7.37 (m, 1H, ArH), 6.47 (m, 1H, ArH).</div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i43" class="anchor-spacer"></div><h3 class="article-section__title" id="_i43"> 5-Bromo-3-nitro-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine (<b>5</b>)</h3><div class="NLM_p last">5-Bromo-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine 4 (730 mg, 3.7 mmol) was added portionwise to a stirring solution of fuming nitric acid (2 mL) at 0 °C over 10 min. The reaction was allowed to stir for 30 min at 0 °C. The mixture was poured into ice water and the formed precipitate was collected via vacuum filtration. The filter cake was washed generously with water and heptane yielding the title compound as a yellow solid (853 mg, 95%). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ (ppm) 13.48 (br s, 1H, NH), 8.87 (s, 1H, ArH), 8.51 (s, 2H, ArH). <sup>13</sup>C NMR (75 MHz, DMSO-<i>d</i><sub>6</sub>): δ (ppm) 145.98, 145.26, 132.13, 129.96, 126.32, 115.17, 114.32.</div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i44" class="anchor-spacer"></div><h3 class="article-section__title" id="_i44"> Synthesis of 3-Nitro-5-aryl-pyrrolo[2,3-<i>b</i>]pyridines (<b>6a–h</b>)</h3><div id="sec4_4_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i45" class="anchor-spacer"></div><h4 class="article-section__title" id="_i45"> General Procedure</h4><div class="NLM_p">To a solution of 5-bromo-3-nitro-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine <b>5</b> (1 equiv) in dioxane (8 mL) were added the appropriate boronic acid (1.2 equiv) and 2 mL of a K<sub>2</sub>CO<sub>3</sub> (3 equiv) solution. The system was purged three times with argon and heated to 105 °C. After stirring for 10 min, Pd(PPh<sub>3</sub>)<sub>4</sub> (0.1 equiv) was added and the reaction was purged once more with argon. The reaction mixture was stirred at 105 °C overnight. After completion, the reaction mixture was cooled to room temperature and filtered through Celite. The filtrate was extracted with water and ethyl acetate. The combined organic phases were washed with brine, dried over MgSO<sub>4</sub>, and evaporated in vacuo. Purification of the crude residue was achieved by silica gel flash column chromatography using the appropriate solvent mixture.</div><div class="NLM_p last">The following compounds were made according to this procedure.</div></div><div id="sec4_4_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i46" class="anchor-spacer"></div><h4 class="article-section__title" id="_i46"> 3-Nitro-5-phenyl-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine (<b>6a</b>)</h4><div class="NLM_p last">The title compound was synthesized according to the general procedure using 5-bromo-3-nitro-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine <b>5</b> (100 mg, 0.413 mmol), phenylboronic acid (61 mg, 0.496 mmol), and K<sub>2</sub>CO<sub>3</sub> (171 mg, 1.24 mmol). Purification by silica gel flash column chromatography using a mixture of DCM and ethyl acetate (in a ratio of 90:10) as the mobile phase, yielded the desired compound as a white solid (83 mg, 84%). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ (ppm) 13.39 (br s, 1H, NH), 8.89 (s, 1H, HetH), 8.76 (d, <i>J</i> = 2.1 Hz, 1H, HetH), 8.60 (d, <i>J</i> = 2.2 Hz, 1H, HetH), 7.79 (d, <i>J</i> = 7.2 Hz, 2H, <i>o</i>-PhH), 7.54 (t, <i>J</i> = 7.4 Hz, 2H, <i>m</i>-PhH), 7.45 (t, <i>J</i> = 7.3 Hz, 1H, <i>p</i>-PhH).</div></div><div id="sec4_4_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i47" class="anchor-spacer"></div><h4 class="article-section__title" id="_i47"> 5-(4-Fluorophenyl)-3-nitro-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine (<b>6b</b>)</h4><div class="NLM_p last">The title compound was synthesized according to the general procedure using 5-bromo-3-nitro-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine <b>5</b> (100 mg, 0.413 mmol), 4-fluorophenylboronic acid (69 mg, 0.496 mmol), and K<sub>2</sub>CO<sub>3</sub> (171 mg, 1.24 mmol). Purification by silica gel flash column chromatography using a mixture of DCM and ethyl acetate (in a ratio of 90:10) as the mobile phase yielded the desired compound as a light brown solid (98 mg, 92%). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ (ppm) 13.37 (br s, 1H, NH), 8.90 (s, 1H, HetH), 8.82 (d, <i>J</i> = 2.9 Hz, 1H, HetH), 8.68 (d, <i>J</i> = 3.0 Hz, 1H, HetH), 8.01 (m, 4H, PhH).</div></div><div id="sec4_4_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i48" class="anchor-spacer"></div><h4 class="article-section__title" id="_i48"> 5-(4-(Trifluoromethyl)phenyl)-3-nitro-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine (<b>6c</b>)</h4><div class="NLM_p last">The title compound was synthesized according to the general procedure using 5-bromo-3-nitro-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine <b>5</b> (100 mg, 0.413 mmol), 4-trifluoromethylphenylboronic acid (94 mg, 0.496 mmol), and K<sub>2</sub>CO<sub>3</sub> (171 mg, 1.24 mmol). Purification by silica gel flash column chromatography using a mixture of DCM and ethyl acetate (in a ratio of 90:10) as the mobile phase yielded the desired compound as a white solid (110 mg, 87%). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ (ppm) 13.45 (br s, 1H, NH), 8.93 (s, 1H, HetH), 8.84 (d, <i>J</i> = 3.2 Hz, 1H, HetH), 8.70 (d, <i>J</i> = 3.1 Hz, 1H, HetH), 8.45 (m, 1H, PhH), 8.05 (d, <i>J</i> = 8.8 Hz, 2H, PhH), 7.89 (d, <i>J</i> = 8.9 Hz, 1H, PhH).</div></div><div id="sec4_4_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i49" class="anchor-spacer"></div><h4 class="article-section__title" id="_i49"> 5-(4-Methoxyphenyl)-3-nitro-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine (<b>6d</b>)</h4><div class="NLM_p last">The title compound was synthesized according to the general procedure using 5-bromo-3-nitro-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine <b>5</b> (100 mg, 0.413 mmol), 4-methoxyphenylboronic acid (75 mg, 0.496 mmol), and K<sub>2</sub>CO<sub>3</sub> (171 mg, 1.24 mmol). Purification by silica gel flash column chromatography using a mixture of DCM and ethyl acetate (in a ratio of 90:10) as the mobile phase yielded the desired compound as a yellow solid (104 mg, 94%). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ (ppm) 13.32 (br s, 1H, NH), 8.86 (s, 1H, HetH), 8.72 (d, <i>J</i> = 2.8 Hz, 1H, HetH), 8.55 (d, <i>J</i> = 3.0 Hz, 1H, HetH), 7.73 (d, <i>J</i> = 8.5 Hz, 2H, PhH), 7.10 (d, <i>J</i> = 8.5 Hz, 2H, PhH), 3.83 (s, 3H, OCH<sub>3</sub>).</div></div><div id="sec4_4_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i50" class="anchor-spacer"></div><h4 class="article-section__title" id="_i50"> 3-Nitro-5-(<i>p</i>-tolyl)-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine (<b>6e</b>)</h4><div class="NLM_p last">The title compound was synthesized according to the general procedure using 5-bromo-3-nitro-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine <b>5</b> (100 mg, 0.413 mmol), <i>p</i>-tolylboronic acid (67 mg, 0.496 mmol), and K<sub>2</sub>CO<sub>3</sub> (171 mg, 1.24 mmol). Purification by silica gel flash column chromatography using a mixture of DCM and ethyl acetate (in a ratio of 90:10) as the mobile phase yielded the desired compound as a white solid (83 mg, 74%). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ (ppm) 13.35 (br s, 1H, NH), 8.88 (s, 1H, HetH), 8.74 (d, J = 3.2 Hz, 1H, HetH), 8.58 (d, <i>J</i> = 3.1 Hz, 1H, HetH), 7.68 (d, <i>J</i> = 7.8 Hz, 2H, PhH), 7.35 (d, <i>J</i> = 7.8 Hz, 2H, PhH), 2.38 (s, 3H, CH<sub>3</sub>).</div></div><div id="sec4_4_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i51" class="anchor-spacer"></div><h4 class="article-section__title" id="_i51"> 3-Nitro-5-(<i>m</i>-tolyl)-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine (<b>6f</b>)</h4><div class="NLM_p last">The title compound was synthesized according to the general procedure using 5-bromo-3-nitro-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine <b>5</b> (100 mg, 0.413 mmol), <i>m</i>-tolylboronic acid (67 mg, 0.496 mmol), and K<sub>2</sub>CO<sub>3</sub> (171 mg, 1.24 mmol). Purification by silica gel flash column chromatography using a mixture of DCM and ethyl acetate (in a ratio of 90:10) as the mobile phase yielded the desired compound as a white solid (83 mg, 74%). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ (ppm) 13.35 (br s, 1H, NH), 8.88 (s, 1H, HetH), 8.74 (d, <i>J</i> = 3.1 Hz, 1H, HetH), 8.59 (d, <i>J</i> = 3.1 Hz, 1H, HetH), 7.58 (t, <i>J</i> = 7.2 Hz, 2H, PhH), 7.42 (t, <i>J</i> = 7.3 Hz, 1H, PhH), 7.26 (d, <i>J</i> = 3.3 Hz, 1H, PhH), 2.42 (s, 3H, CH<sub>3</sub>).</div></div><div id="sec4_4_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i52" class="anchor-spacer"></div><h4 class="article-section__title" id="_i52"> 5-(3,4-Dimethoxyphenyl)-3-nitro-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine (<b>6g</b>)</h4><div class="NLM_p last">The title compound was synthesized according to the general procedure using 5-bromo-3-nitro-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine <b>5</b> (100 mg, 0.413 mmol), 3,4-dimethoxyphenylboronic acid (63 mg, 0.496 mmol), and K<sub>2</sub>CO<sub>3</sub> (171 mg, 1.24 mmol). The compound precipitated as a yellow solid that was washed twice with dioxane, followed by washing with water yielding the pure title compound (88 mg, 89%). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ (ppm) 8.82 (s, 1H, HetH), 8.72 (d, <i>J</i> = 2.1 Hz, 1H, HetH), 8.54 (d, <i>J</i> = 2.2 Hz, 1H, HetH), 7.30 (m, 2H, PhH), 7.10 (d, <i>J</i> = 8.3 Hz, 1H, PhH), 3.88 (s, 3H, OCH<sub>3</sub>), 3.82 (s, 3H, OCH<sub>3</sub>).</div></div><div id="sec4_4_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i53" class="anchor-spacer"></div><h4 class="article-section__title" id="_i53"> 3-Nitro-5-(3-thienyl)-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine (<b>6h</b>)</h4><div class="NLM_p last">The title compound was synthesized according to the general procedure using 5-bromo-3-nitro-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine <b>5</b> (100 mg, 0.413 mmol), 3-thienylboronic acid (69 mg, 0.496 mmol), and K<sub>2</sub>CO<sub>3</sub> (171 mg, 1.24 mmol). Purification by silica gel flash column chromatography using a mixture of DCM and ethyl acetate (in a ratio of 90:10) as the mobile phase yielded the title compound as a brown solid (97 mg, 96%). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ (ppm) 13.41 (br s, 1H, NH), 8.89 (d, <i>J</i> = 3.0 Hz, 1H, HetH), 8.83 (d, <i>J</i> = 3.1 Hz, 1H, HetH), 8.57 (d, <i>J</i> = 3.0 Hz, 1H, HetH), 7.69 (m, 2H, ThH) ppm, 7.22 (t, <i>J</i> = 4.4 Hz, 1H, ThH).</div></div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i54" class="anchor-spacer"></div><h3 class="article-section__title" id="_i54"> Synthesis of 5-Aryl-3-amino- and 5-Aryl-3-<i>N</i>-acylamino Pyrrolo[2,3-<i>b</i>]pyridines (<b>7a–f</b> and <b>8a–o</b>)</h3><div id="sec4_5_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i55" class="anchor-spacer"></div><h4 class="article-section__title" id="_i55"> General Procedure</h4><div class="NLM_p last">To a solution of a 5-aryl-3-nitro-pyrrolo[2,3-<i>b</i>]pyridine <b>6a–h</b> (100 mg) in tetrahydrofuran (THF) (5 mL) was added a slurry of Raney nickel in water in catalytic amounts. The reaction vessel was flushed three times with hydrogen gas and was stirred under a hydrogen atmosphere for 3–4 h. Upon completion of the reaction, the catalyst was removed and the solvent evaporated in vacuo. The crude residue was used in the next reaction without any purification because of the rapid decomposition of the 3-amino-pyrrolo[2,3-<i>b</i>]pyridine intermediates <b>7a–h</b>. To a solution of compounds <b>7a–h</b> (1 equiv) in dry pyridine (3 mL) was added a solution of nicotinoyl chloride hydrochloride (1.2 equiv) in CH<sub>2</sub>Cl<sub>2</sub> (2 mL). The reaction was allowed to stir at room temperature for 3 h. The solvent was evaporated in vacuo. THF (5 mL) was added and the resulting solution was stirred for 5 min, followed by the addition of a 1 M NaOH solution in water (5 mL). The resulting suspension was stirred for an additional 10 min. The precipitate was collected via vacuum filtration and purified using flash column chromatography with the appropriate solvent mixture as mobile phase. Compounds <b>8a–h</b> were synthesized according to this procedure.</div></div><div id="sec4_5_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i56" class="anchor-spacer"></div><h4 class="article-section__title" id="_i56"> <i>N</i>-(5-Phenyl-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridin-3-yl)nicotinamide (<b>8a</b>)</h4><div class="NLM_p last">The title compound was synthesized according to the general procedure using 3-nitro-5-phenyl-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine <b>6a</b> (84 mg, 0.351 mmol). Purification by silica gel flash column chromatography (eluting with a mixture of DCM and methanol in a gradient gradually ranging from 98:2 to 90:10) afforded the title compound as a dark brown solid (47 mg, 36%). Purity of 99%. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ 11.62 (br s, 1H, NH), 10.58 (br s, 1H, NH), 9.19 (s, 1H, HetH), 8.79 (d, <i>J</i> = 3.0 Hz, 1H, HetH), 8.66 (s, 1H, ArH), 8.58 (s, 1H, ArH), 8.36 (d, <i>J</i> = 6.1 Hz, 1H, ArH), 7.98 (s, 1H, ArH), 7.75 (d, <i>J</i> = 8.8 Hz, 2H, ArH), 7.60 (t, <i>J</i> = 6.0 Hz, 1H, ArH), 7.51 (t, <i>J</i> = 7.3 Hz, 2H, ArH), 7.38 (t, <i>J</i> = 7.4 Hz, 1H, ArH) ppm. <sup>13</sup>C NMR (75 MHz, DMSO-<i>d</i><sub>6</sub>): δ 163.46, 152.01, 148.87, 145.52, 142.23, 139.16, 135.60, 131.63, 130.52, 129.13, 127.06, 126.93, 125.49, 123.61, 117.52, 114.10, 113.45 ppm. HRMS <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>19</sub>H<sub>14</sub>N<sub>4</sub>O, 315.12403; found, 315.1237.</div></div><div id="sec4_5_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i57" class="anchor-spacer"></div><h4 class="article-section__title" id="_i57"> <i>N</i>-[5-(4-Fluorophenyl)-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridin-3-yl]nicotinamide (<b>8b</b>)</h4><div class="NLM_p last">The title compound was synthesized according to the general procedure using 3-nitro-5-(4-fluorophenyl)-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine <b>6b</b> (98 mg, 0.381 mmol). Purification by silica gel flash column chromatography (using a mixture of DCM/methanol in a ratio gradually ranging from 98:2 to 90:10 as mobile phase) afforded the title compound as a brown solid (44 mg, 34%). Purity of 99%. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ 11.63 (br s, 1H, NH), 10.57 (br s, 1H, NH), 9.17 (s, 1H, HetH), 8.78 (d, <i>J</i> = 5.8 Hz, 1H, ArH), 8.62 (s, 1H, HetH), 8.56 (s, 1H, ArH), 8.36 (d, <i>J</i> = 8.9 Hz, 1H, ArH), 7.98 (d, <i>J</i> = 3.2 Hz, 1H, ArH), 7.77 (t, <i>J</i> = 7.6 Hz, 2H, ArH), 7.61 (d, <i>J</i> = 9.0 Hz, 1H, ArH), 7.35 (t, <i>J</i> = 7.5 Hz, 2H, ArH) ppm. <sup>13</sup>C NMR (75 MHz, DMSO-<i>d</i><sub>6</sub>): δ 162.24, 161.77 (d, <i>J</i><sub>CF</sub> = 242 Hz), 150.23, 148.62, 148.62, 145.58, 142.05, 138.10, 135.66, 128.86 (d, <i>J</i><sub>CF</sub> = 8.0 Hz), 126.77, 126.72, 125.74, 122.38, 117.74, 116.00, 115.85 (d, <i>J</i><sub>CF</sub> = 20.4), 113.75, 113.47 ppm. HRMS <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>19</sub>H<sub>13</sub>N<sub>4</sub>OF, 333.11460; found, 333.1145. HRMS <i>m</i>/<i>z</i>: [M + Na]<sup>+</sup> calcd for C<sub>19</sub>H<sub>13</sub>N<sub>4</sub>OF, 355.09658; found, 355.0950.</div></div><div id="sec4_5_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i58" class="anchor-spacer"></div><h4 class="article-section__title" id="_i58"> <i>N</i>-[5-(4-Trifluoromethylphenyl)-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridin-3-yl]nicotinamide (<b>8c</b>)</h4><div class="NLM_p last">The title compound was synthesized according to the general procedure using 3-nitro-5-(4-trifluoromethylphenyl)-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine <b>6c</b> (116 mg, 377.56 μmol). Purification by silica gel flash column chromatography using a mixture of DCM and methanol (in a ratio gradually ranging from 98:2 to 90:10) as the mobile phase afforded the title compound as a white solid (49 mg, 36%). Purity of 99%. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ 11.73 (br s, 1H, NH), 10.62 (br s, 1H, NH), 9.19 (d, <i>J</i> = 3.2 Hz, 1H, HetH), 8.78 (t, <i>J</i> = 6.1 Hz, 2H, ArH), 8.67 (d, <i>J</i> = 3.1 Hz, 1H, HetH), 8.36 (d, <i>J</i> = 3.0 Hz, 1H, ArH), 7.99 (d, <i>J</i> = 8.9 Hz, 3H, PhH), 7.87 (d, <i>J</i> = 9.0 Hz, 2H, PhH), 7.61 (q, <i>J</i> = 5.9 Hz, 1H, ArH) ppm. <sup>13</sup>C NMR (75 MHz, DMSO-<i>d</i><sub>6</sub>): δ 163.52, 152.05, 148.85, 145.88, 143.25, 142.35, 135.58, 130.46, 127.48, 126.09, 123.61, 122.80, 117.84, 114.28, 113.52 ppm; missing peaks were observed in an APT spectrum. HRMS <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>20</sub>H<sub>13</sub>N<sub>4</sub>F<sub>3</sub>O, 383.11141; found, 383.1114.</div></div><div id="sec4_5_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i59" class="anchor-spacer"></div><h4 class="article-section__title" id="_i59"> <i>N</i>-[5-(4-Methoxyphenyl)-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridin-3-yl]nicotinamide (<b>8d</b>)</h4><div class="NLM_p last">The title compound was synthesized according to the general procedure using 3-nitro-5-(4-methoxyphenyl)-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine <b>6d</b> (105 mg, 0.390 mmol). Purification by silica gel flash column chromatography using a mixture of DCM and methanol (in a ratio gradually ranging from 98:2 to 90:10) as the mobile phase afforded the title compound as a pink solid (43 mg, 34%). Purity of 99% <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ 11.56 (br s, 1H, NH), 10.56 (br s, 1H, NH), 9.18 (s, 1H, ArH), 8.78 (d, <i>J</i> = 6.1 Hz, 1H, ArH), 8.58 (s, 1H, ArH), 8.52 (s, 1H, ArH), 8.36 (d, <i>J</i> = 9.0 Hz, 1H, PhH), 7.96 (s, 1H, ArH), 7.62 (m, 3H, ArH), 7.08 (d, <i>J</i> = 8.9 Hz, 2H, PhH), 3.82 (s, 3H, OCH<sub>3</sub>) ppm. <sup>13</sup>C NMR (75 MHz, DMSO-<i>d</i><sub>6</sub>): δ 163.45, 158.74, 151.99, 148.85, 145.22, 141.99, 135.59, 131.54, 130.50, 128.00, 127.55, 124.87, 123.60, 117.43, 114.64, 113.94, 113.45, 55.34 ppm. HRMS <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>20</sub>H<sub>16</sub>N<sub>4</sub>O<sub>2</sub>, 345.13459; found, 345.1343. HRMS <i>m</i>/<i>z</i>: [M + Na]<sup>+</sup> calcd for C<sub>20</sub>H<sub>16</sub>N<sub>4</sub>O<sub>2</sub>, 367.11656; found, 367.1153.</div></div><div id="sec4_5_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i60" class="anchor-spacer"></div><h4 class="article-section__title" id="_i60"> <i>N</i>-(5-(<i>p</i>-Tolyl)-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridin-3-yl)nicotinamide (<b>8e</b>)</h4><div class="NLM_p last">The title compound was synthesized according to the general procedure using 3-nitro-5-(<i>p</i>-tolyl)-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine <b>6e</b> (96 mg, 0.379 mmol). Purification by silica gel flash column chromatography using a mixture of DCM and methanol (in a ratio gradually ranging from 98:2 to 90:10) as the mobile phase afforded the title compound as a beige solid (48 mg, 37%). Purity of 98%. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ 11.60 (br s, 1H, NH), 10.59 (br s, 1H, NH), 9.19 (s, 1H, ArH), 8.78 (d, <i>J</i> = 2.9 Hz, 1H, ArH), 8.63 (s, 1H, ArH), 8.56 (s, 1H, ArH), 8.36 (d, <i>J</i> = 6.0 Hz, 1H, ArH), 7.99 (s, 1H, ArH), 7.59 (m, 3H, ArH), 7.31 (d, <i>J</i> = 9.3 Hz, 2H, ArH), 2.35 (s, 3H, CH<sub>3</sub>) ppm. <sup>13</sup>C NMR (75 MHz, DMSO-<i>d</i><sub>6</sub>): δ 163.49, 151.99, 148.85, 145.41, 142.10, 136.28, 136.22, 135.59, 130.51, 129.73, 127.68, 126.73, 125.12, 123.60, 117.50, 114.03, 113.47, 20.77 ppm. HRMS <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>20</sub>H<sub>16</sub>N<sub>4</sub>O, 329.13967; found, 329.1398.</div></div><div id="sec4_5_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i61" class="anchor-spacer"></div><h4 class="article-section__title" id="_i61"> <i>N</i>-[5-(<i>m</i>-Tolyl)-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridin-3-yl]nicotinamide (<b>8f</b>)</h4><div class="NLM_p last">The title compound was synthesized according to the general procedure using 3-nitro-5-(<i>m</i>-tolyl)-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine <b>6f</b> (77 mg, 0.304 mmol). Purification by silica gel flash column chromatography using a mixture of DCM and methanol (in a ratio gradually ranging from 98:2 to 90:10) as the mobile phase afforded the title compound as a beige solid (22 mg, 25% yield). Purity of 98%. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ 11.61 (br s, 1H, NH), 10.59 (br s, 1H, NH), 9.19 (s, 1H, ArH), 8.79 (d, <i>J</i> = 2.8 Hz, 1H, ArH), 8.64 (s, 1H, ArH), 8.57 (s, 1H, ArH), 8.37 (d, <i>J</i> = 9.0 Hz, 1H, ArH), 7.99 (s, 1H, ArH), 7.59 (m, 3H, CH<sub>3</sub>), 7.39 (t, <i>J</i> = 7.4 Hz, 1H), 7.19 (d, <i>J</i> = 6.1 Hz, 1H), 2.41 (s, 3H) ppm. <sup>13</sup>C NMR (75 MHz, DMSO-<i>d</i><sub>6</sub>): δ 163.49, 151.99, 148.84, 145.47, 142.25, 139.08, 138.27, 135.59, 130.50, 129.02, 127.84, 127.71, 127.53, 125.39, 124.06, 123.60, 117.47, 114.07, 113.42, 21.28 ppm. HRMS <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>20</sub>H<sub>16</sub>N<sub>4</sub>O, 329.13967; found, 329.1397.</div></div><div id="sec4_5_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i62" class="anchor-spacer"></div><h4 class="article-section__title" id="_i62"> <i>N</i>-[5-(3,4-Dimethoxyphenyl)-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridin-3-yl]nicotinamide (<b>8g</b>)</h4><div class="NLM_p last">The title compound was synthesized according to the general procedure using 3-nitro-5-(3,4-dimethoxyphenyl)-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine <b>6g</b> (80 mg, 0.297 mmol). Purification by silica gel flash column chromatography using a mixture of DCM and methanol (in a ratio gradually ranging from 98:2 to 90:10) as the mobile phase afforded the title compound as a beige solid (56 mg, 50%). Purity of 97%. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ 11.57 (br s, 1H, NH), 10.56 (br s, 1H, NH), 9.20 (s, 1H, ArH), 8.78 (d, <i>J</i> = 6.1 Hz, 1H, ArH), 8.58 (d, <i>J</i> = 3.2 Hz, 2H, ArH), 8.37 (d, <i>J</i> = 9.1 Hz, 1H, ArH), 7.97 (s, 1H, ArH), 7.60 (t, <i>J</i> = 6.0 Hz, 1H, ArH), 7.27 (t, <i>J</i> = 7.4 Hz, 2H, ArH), 7.09 (d, <i>J</i> = 9.0 Hz, 1H, ArH), 3.88 (s, 3H, OCH<sub>3</sub>), 3.81 (s, 3H, OCH<sub>3</sub>) ppm. <sup>13</sup>C NMR (75 MHz, DMSO-<i>d</i><sub>6</sub>): δ 163.45, 152.00, 149.40, 148.88, 148.43, 145.28, 142.26, 135.61, 132.04, 130.51, 127.84, 124.96, 123.61, 119.23, 117.47, 113.93, 113.39, 112.70, 111.09, 55.92, 55.82 ppm. HRMS <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>21</sub>H<sub>18</sub>N<sub>4</sub>O<sub>3</sub>, 375.14515; found, 375.1447, HRMS <i>m</i>/<i>z</i>: [M + Na]<sup>+</sup> calcd for C<sub>21</sub>H<sub>18</sub>N<sub>4</sub>O<sub>3</sub>, 397.12713; found, 397.1264.</div></div><div id="sec4_5_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i63" class="anchor-spacer"></div><h4 class="article-section__title" id="_i63"> <i>N</i>-[5-(3-Thienyl)-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridin-3-yl]nicotinamide (<b>8h</b>)</h4><div class="NLM_p last">The title compound was synthesized according to the general procedure using 3-nitro-5-(3-thienyl)-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine <b>6h</b> (97 mg, 0.395 mmol). Purification by silica gel flash column chromatography using a mixture of DCM and methanol (in a ratio gradually ranging from 98:2 to 90:10) as the mobile phase, afforded the title compound as a brown solid (48 mg, 37%). Purity of 99%. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ 11.58 (br s, 1H, NH), 10.54 (br s, 1H, NH), 9.20 (d, <i>J</i> = 2.8 Hz, 1H, ArH), 8.79 (d, <i>J</i> = 2.9 Hz, 1H, ArH), 8.67 (d, <i>J</i> = 2.9 Hz, 1H, ArH), 8.64 (d, <i>J</i> = 3.0 Hz, 1H, ArH), 8.36 (m, 1H, ArH), 7.93 (s, 1H, ArH), 7.84 (s, 1H, ArH), 7.71 (m, 1H, ArH), 7.60 (t, <i>J</i> = 5.9 Hz, 2H, ArH) ppm. <sup>13</sup>C NMR (75 MHz, DMSO-<i>d</i><sub>6</sub>): δ 163.48, 152.02, 148.86, 145.30, 142.03, 140.21, 135.58, 130.49, 127.30, 126.37, 124.61, 123.60, 123.18, 119.78, 117.63, 113.94, 113.49 ppm. HRMS <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>17</sub>H<sub>12</sub>N<sub>4</sub>OS, 321.08045; found, 321.0804.</div></div><div id="sec4_5_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i64" class="anchor-spacer"></div><h4 class="article-section__title" id="_i64"> <i>N</i>-(5-(3,4-Dimethoxyphenyl)-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridin-3-yl)benzamide (<b>8i</b>)</h4><div class="NLM_p last">To a solution of 3-nitro-5-(3,4-dimethoxyphenyl)-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine <b>6g</b> (100 mg, 0.334 mmol) in THF (4 mL) was added a catalytic amount of a slurry of Raney nickel in water. The vessel was flushed three times with hydrogen gas and kept under a hydrogen atmosphere for 4 h. After complete conversion, the catalyst was removed, and the solvent was evaporated in vacuo. The crude product was immediately used for further reaction without purification. To a solution of the crude residue from the previous reaction in pyridine (5 mL) was added a solution of benzoyl chloride (47 μL, 0.401 mmol) in DCM (0.5 mL). The reaction was stirred overnight at room temperature. After reaction completion, the solvent was evaporated in vacuo, and the crude residue was extracted three times with water and ethyl acetate. The combined organic layers were dried over MgSO<sub>4</sub> and evaporated to dryness. Purification was achieved by silica gel flash column chromatography (using a mixture of DCM and methanol in a ratio of 97:3 as mobile phase), followed by precipitation of the residue from acetone. The precipitate was collected via filtration yielding the title compound as a white solid (27 mg, 22%). Purity of 98%. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ 11.52 (br s, 1H, NH), 10.37 (br s, 1H, NH), 8.56 (d, <i>J</i> = 6.52, 2H, ArH), 8.00 (m, 3H, ArH), 7.56 (d, <i>J</i> = 7.6 Hz, 3H, ArH), 7.25 (m, 2H, ArH), 7.08 (d, <i>J</i> = 8.2 Hz, 1H, ArH), 3.87 (s, 3H, OCH<sub>3</sub>), 3.80 (s, 3H, OCH<sub>3</sub>) ppm. <sup>13</sup>C NMR (75 MHz, DMSO-<i>d</i><sub>6</sub>): δ 165.07, 149.30, 148.30, 145.25, 142.12, 134.88, 132.01, 131.42, 128.48, 127.87, 127.70, 125.04, 119.14, 117.27, 114.22, 113.44, 112.52, 110.87, 55.83, 55.74 ppm. HRMS <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>22</sub>H<sub>19</sub>N<sub>3</sub>O<sub>3</sub>, 374.14990; found, 374.1493.</div></div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i65" class="anchor-spacer"></div><h3 class="article-section__title" id="_i65"> Synthesis of 5-(3,4-Dimethoxyphenyl)-3-<i>N</i>-acylamino Pyrrolo[2,3-<i>b</i>]pyridines</h3><div id="sec4_6_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i66" class="anchor-spacer"></div><h4 class="article-section__title" id="_i66"> General Procedure</h4><div class="NLM_p last">3-Nitro-5-(3,4-dimethoxyphenyl)-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine 6g (100 mg, 0.334 mmol) was hydrogenated with a catalytic amount of Raney nickel as a slurry in water (slurry in water? <i>x</i> mg/mL) in THF (5 mL) for 3 h. The solvents were evaporated in vacuo yielding crude 3-amino-5-(3,4-dimethoxyphenyl)-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine which was used in the next reaction without further purification (90 mg, 99%). To a solution of 3-amino-5-(3,4-dimethoxyphenyl)-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine (90 mg, 0.334 mmol) in dimethylformamide (DMF) (3 mL) was added the appropriate carboxylic acid (1 equiv), BOP (177 mg, 0.401 mmol), and triethylamine (140 μL, 1 mmol). The mixture was stirred overnight. When the reaction reached completion, water was added and the mixture was extracted three times with ethyl acetate. The combined organic layers were washed with brine, dried over Na<sub>2</sub>SO<sub>4</sub>, and evaporated in vacuo. The crude residue was purified by silica gel flash chromatography using an appropriate solvent system as the mobile phase. An additional purification was performed by preparative TLC using a mixture of DCM and acetone (in a ratio of 8:2) as eluent. The following compounds were prepared according to this procedure.</div></div><div id="sec4_6_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i67" class="anchor-spacer"></div><h4 class="article-section__title" id="_i67"> <i>N</i>-(5-(3,4-Dimethoxyphenyl)-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridin-3-yl)-4-fluorobenzamide (<b>8j</b>)</h4><div class="NLM_p last">The title compound was synthesized according to the general procedure using 4-fluorobenzoic acid (47 mg, 0.334 mmol). Purification by silica gel flash column chromatography (using a mixture of DCM and methanol as mobile phase in a ratio of 99:1) yielded the desired compound as a white solid (56 mg, 43%). Purity of 99%. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ 11.53 (br s, 1H, NH), 10.39 (br s, 1H, NH), 8.56 (d, <i>J</i> = 8.5 Hz, 2H, ArH), 8.12 (m, 2H, ArH), 7.93 (s, 1H, ArH), 7.38 (t, <i>J</i> = 8.7 Hz, 2H, ArH), 7.26 (m, 2H, ArH), 7.08 (d, <i>J</i> = 8.3 Hz, 1H, ArH), 3.87 (s, 3H, OCH<sub>3</sub>), 3.81 (s, 3H, OCH<sub>3</sub>) ppm. <sup>13</sup>C NMR (75 MHz, DMSO-<i>d</i><sub>6</sub>): δ 164.07 (<i>J</i><sub>CF</sub> = 245 Hz), 163.93, 149.30, 148.30, 145.25, 142.12, 131.99, 131.27, 130.57 (<i>J</i><sub>CF</sub> = 8.97 Hz), 127.69, 125.05, 119.12, 117.35, 115.39 (<i>J</i><sub>CF</sub> = 22 Hz), 114.11, 113.46, 112.53, 110.88, 55.84, 55.75 ppm. HRMS <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>22</sub>H<sub>18</sub>FN<sub>3</sub>O<sub>3</sub>, 392.14048; found, 392.1399.</div></div><div id="sec4_6_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i68" class="anchor-spacer"></div><h4 class="article-section__title" id="_i68"> <i>N</i>-(5-(3,4-Dimethoxyphenyl)-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridin-3-yl)-4-methoxybenzamide (<b>8k</b>)</h4><div class="NLM_p last">The title compound was synthesized according to the general procedure using 4-methoxybenzoic acid (51 mg, 0.334 mmol). Purification by silica gel flash column chromatography (using a mixture of DCM and methanol as mobile phase in a ratio of 99:1) yielded the desired compound as a white solid (64 mg, 47%). Purity of 97%. <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>): δ 11.46 (s, 1H, NH), 10.16 (s, 1H, NH), 8.55 (d, <i>J</i> = 1.92 Hz, 1H, ArH), 8.53 (d, <i>J</i> = 2.22 Hz, 1H, ArH), 8.01 (m, 2H, ArH), 7.90 (d, <i>J</i> = 2.46 Hz, 1H, ArH), 7.28 (d, <i>J</i> = 2.16 Hz, 1H, ArH), 7.22 (dd, <i>J</i> = 8.22, 2.16 Hz, ArH), 7.08 (m, 3H, ArH), 3.87 (s, 3H, OCH<sub>3</sub>), 3.85 (s,3H, OCH<sub>3</sub>), 3.80 (s, 3H, OCH<sub>3</sub>) ppm. <sup>13</sup>C NMR (150 MHz, DMSO-<i>d</i><sub>6</sub>): δ 164.43, 161.77, 149.25, 148.24, 145.21, 142.00, 131.99, 129.69, 127.58, 126.92, 124.98, 119.08, 117.14, 114.27, 113.65, 113.48, 112.48, 110.82, 55.78, 55.70, 55.50 ppm. HRMS <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>23</sub>H<sub>21</sub>N<sub>3</sub>O<sub>4</sub>, 404.16047; found, 404.1603.</div></div><div id="sec4_6_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i69" class="anchor-spacer"></div><h4 class="article-section__title" id="_i69"> <i>N</i>-(5-(3,4-Dimethoxyphenyl)-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridin-3-yl)-2-phenylacetamide (<b>8m</b>)</h4><div class="NLM_p last">The title compound was synthesized according to the general procedure using phenylacetic acid (45 mg, 0.334 mmol). Purification by silica gel flash column chromatography (using a mixture of DCM and methanol as mobile phase in a ratio of 99:1) yielded the desired compound as a white solid (49 mg, 54%). Purity of 99%. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ 11.36 (s, 1H, NH), 10.45 (s, 1H, NH), 8.53 (m, 2H, ArH), 7.79 (d, <i>J</i> = 2.3 Hz, 1H, ArH), 7.31 (m, 7H, ArH), 7.09 (d, <i>J</i> = 8.4 Hz, 1H, ArH), 3.88 (s, 3H, OCH<sub>3</sub>), 3.81 (s, 3H, OCH<sub>3</sub>), 3.76 (s, 2H, CH<sub>2</sub>) ppm. <sup>13</sup>C NMR (75 MHz, DMSO-<i>d</i><sub>6</sub>): δ 168.21, 157.37, 149.33, 148.30, 145.01, 142.09, 136.54, 131.94, 129.28, 128.38, 127.55, 126.56, 124.40, 119.00, 115.78, 114.37, 112.78, 112.55, 110.80, 55.83, 55.76, 42.41 ppm. HRMS <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>23</sub>H<sub>21</sub>N<sub>3</sub>O<sub>3</sub>, 388.16555; found, 388.1660.</div></div><div id="sec4_6_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i70" class="anchor-spacer"></div><h4 class="article-section__title" id="_i70"> <i>N</i>-(5-(3,4-Dimethoxyphenyl)-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridin-3-yl)cyclopentanecarboxamide (<b>8n</b>)</h4><div class="NLM_p last">To a solution of 3-nitro-5-(3,4-dimethoxyphenyl)-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine <b>6g</b> (100 mg, 0.334 mmol) in THF (5 mL) was added a catalytic amount of a slurry of Raney nickel in water. The vessel was flushed three times with hydrogen gas and kept under a hydrogen atmosphere for 4 h. After complete conversion, the catalyst was removed, and the solvent was evaporated in vacuo. The crude product was immediately used for further reaction without purification. To a solution of the crude residue in pyridine (5 mL) was added a solution of cyclopentylcarbonyl chloride (46 μL, 0.401 mmol) in DCM (2 mL). The reaction was stirred overnight at room temperature. After reaction completion, the solvent was evaporated in vacuo, and the crude residue was extracted three times with water and ethyl acetate. The combined organic layers were dried over MgSO<sub>4</sub> and evaporated to dryness. Purification by silica gel flash column chromatography (using a mixture of DCM and methanol in a ratio of 97:3 as mobile phase) yielded the title compound as a white solid (39 mg, 32%). Purity of 99%. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ 11.33 (br s, 1H, NH), 9.96 (br s, 1H, NH), 8.51 (s, 1H, ArH), 8.44 (s, 1H, ArH), 7.82 (d, <i>J</i> = 2.2 Hz, 1H, ArH), 7.23 (m, 2H, ArH), 7.08 (d, <i>J</i> = 8.3 Hz, 1H, ArH), 3.88 (s, 3H, OCH<sub>3</sub>), 3.81 (s, 3H, OCH<sub>3</sub>), 2.90 (m, 1H, CH<sub>2</sub>), 1.77 (m, 8H, CH<sub>2</sub>) ppm. <sup>13</sup>C NMR (75 MHz, DMSO-<i>d</i><sub>6</sub>): δ 173.48, 149.31, 148.30, 145.00, 142.05, 132.01, 127.51, 124.19, 119.05, 115.69, 114.50, 112.71, 112.55, 110.80, 55.82, 55.75, 44.55, 30.46, 25.85. HRMS <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>21</sub>H<sub>23</sub>N<sub>3</sub>O<sub>3</sub>, 366.18120; found, 366.1808.</div></div><div id="sec4_6_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i71" class="anchor-spacer"></div><h4 class="article-section__title" id="_i71"> <i>N</i>-(5-(3,4-Dimethoxyphenyl)-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridin-3-yl)benzenesulfonamide (<b>8p</b>)</h4><div class="NLM_p last">To a solution of 3-nitro-5-(3,4-dimethoxyphenyl)-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine (100 mg, 0.334 mmol) in THF (5 mL) was added a catalytic amount of a slurry of Raney nickel in water. The vessel was flushed with hydrogen gas three times and kept under a hydrogen atmosphere for 4 h. After complete conversion, the catalyst was removed and the solvent was evaporated. The crude was immediately used in the next reaction without further purification because of the rapid decomposition of the amine. To a solution of the crude from the previous reaction in pyridine (5 mL) was added a solution of benzensulfonyl chloride (47 μL, 0.368 mmol) in DCM (1 mL). The reaction was stirred overnight at room temperature. After reaction completion, the solvent was evaporated and the crude was extracted with water and ethyl acetate three times. The collected organics were dried with MgSO<sub>4</sub> and evaporated to dryness. Flash column chromatography (using a mixture of DCM and methanol as mobile phase in a ratio of 97:3) yielded the desired compound as an off-white solid (34 mg, 25%). Purity of 98%. <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>): δ 11.64 (br s, 1H, NH), 9.87 (br s, 1H, NH), 8.42 (d, <i>J</i> = 1.92 Hz, 1H, ArH), 7.73 (s, 1H, ArH), 7.72 (s, 2H, ArH), 7.55 (t, <i>J</i> = 7.5 Hz, 1H, ArH), 7.51 (t, <i>J</i> = 7.74 Hz, 2H, ArH), 7.20 (d, <i>J</i> = 2.52 Hz, 1H, ArH), 7.05 (m, 2H, ArH), 6.99 (d, <i>J</i> = 1.74 Hz, 1H, ArH), 3.85 (s, 3H, OCH<sub>3</sub>), 3.80 (s, 3H, OCH<sub>3</sub>) ppm. <sup>13</sup>C NMR (150 MHz, DMSO-<i>d</i><sub>6</sub>): δ 149.18, 148.27, 145.66, 142.15, 140.05, 132.66, 131.55, 129.03, 128.23, 126.96, 123.76, 121.94, 118.96, 115.65, 113.91, 112.46, 111.06, 110.50, 55.70, 55.68 ppm. HRMS <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>21</sub>H<sub>19</sub>N<sub>3</sub>O<sub>4</sub>S, 408.10234; found, 408.1024.</div></div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i72" class="anchor-spacer"></div><h3 class="article-section__title" id="_i72"> 5-Bromo-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine-3-carbaldehyde (<b>9</b>)</h3><div class="NLM_p last">To a solution of 5-bromo-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine <b>4</b> (500 mg, 2.54 mmol) in a mixture of water and acetic acid (in a ratio of 3:1, 10 mL) was added hexamine (712 mg, 5.08 mmol). The reaction was heated to 120 °C and refluxed overnight. The formed precipitate was filtered off, affording the title compound (380 mg, 67%). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ 12.93 (br s, 1H, NH), 9.93 (s, 1H, CHO), 8.55 (s, 1H, HetH), 8.53 (d, <i>J</i> = 2.3 Hz, 1H, HetH), 8.46 (d, <i>J</i> = 2.3 Hz, 1H, HetH) ppm. <sup>13</sup>C NMR (75 MHz, DMSO-<i>d</i><sub>6</sub>): δ 185.58, 147.92, 145.09, 139.90, 131.01, 118.32, 116.04, 113.86 ppm.</div></div><div id="sec4_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i73" class="anchor-spacer"></div><h3 class="article-section__title" id="_i73"> 5-Bromo-1-tosyl-pyrrolo[2,3-<i>b</i>]pyridine-3-carbaldehyde (<b>10</b>)</h3><div class="NLM_p last">To a suspension of 60% NaH on mineral oil (80 mg, 2 mmol) in dry THF (6 mL) was added 5-bromo-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine-3-carbaldehyde <b>9</b> (300 mg, 1.33 mmol) at 0 °C. After stirring for 20 min at room temperature, tosyl chloride (381 mg, 2 mmol) was added and the resulting solution was stirred for another 3 h at room temperature. After reaction completion, the solvent was evaporated in vacuo and the crude residue was extracted with water and DCM. The combined organic layers were dried over MgSO<sub>4</sub> and evaporated in vacuo, affording the title compound (460 mg, 91%). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ 10.03 (s, 1H, CHO), 9.02 (s, 1H, HetH), 8.59 (d, <i>J</i> = 2.2 Hz, 1H, HetH), 8.55 (d, <i>J</i> = 2.2 Hz, 1H, HetH), 8.07 (d, <i>J</i> = 8.4 Hz, 2H, TsH), 7.48 (d, <i>J</i> = 8.2 Hz, 2H, TsH), 2.37 (s, 3H, CH<sub>3</sub>) ppm.</div></div><div id="sec4_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i74" class="anchor-spacer"></div><h3 class="article-section__title" id="_i74"> 5-(3,4-Dimethoxyphenyl)-1-tosyl-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine-3-carbaldehyde (<b>11</b>)</h3><div class="NLM_p last">To a solution of 5-bromo-1-tosyl-pyrrolo[2,3-<i>b</i>]pyridine-3-carbaldehyde <b>10</b> (240 mg, 0.633 mmol) in a mixture of toluene and ethanol (in a ratio of 3:1, 4 mL) was added 3,4-dimethoxyphenylboronic acid (138 mg, 0.759 mmol) and a 2 M solution of K<sub>2</sub>CO<sub>3</sub> (950 μL, 1.90 mmol). The system was purged with argon and Pd(PPh<sub>3</sub>)<sub>4</sub> (2 mol %) was added. The reaction was heated to 105 °C and was stirred for 4 h. After completion, the solvent was evaporated in vacuo and the residue was extracted with water and ethyl acetate. Purification by silica gel flash chromatography using a mixture of heptane and ethyl acetate (in a ratio of 7:3) as the mobile phase afforded the title compound (160 mg, 58%). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ 10.09 (s, 1H, CHO), 9.00 (s, 1H, HetH), 8.76 (s, 1H, HetH), 8.55 (s, 1H, HetH), 8.11 (d, <i>J</i> = 7.7 Hz, 2H, TsH), 7.49 (d, <i>J</i> = 7.4 Hz, 2H, TsH), 7.24 (m, 2H, PhH), 7.07 (d, <i>J</i> = 8.2 Hz, 1H, PhH), 3.85 (s, 3H, OCH<sub>3</sub>), 3.79 (s, 3H, OCH<sub>3</sub>), 2.37 (s, 3H, CH<sub>3</sub>) ppm.</div></div><div id="sec4_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i75" class="anchor-spacer"></div><h3 class="article-section__title" id="_i75"> (5-(3,4-Dimethoxyphenyl)-1-tosyl-pyrrolo[2,3-<i>b</i>]pyridin-3-yl)(phenyl)methanol (<b>12</b>)</h3><div class="NLM_p last">To a solution of 5-(3,4-dimethoxyphenyl)-1-tosyl-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine-3-carbaldehyde <b>11</b> (160 mg, 0.367 mmol) in dry THF (5 mL) at 0 °C was added a 3 M solution of phenylmagnesium bromide in diethylether (159 μL, 0.477 mmol). The reaction mixture was stirred for 1 h at 0 °C. After reaction completion, the mixture was quenched with a saturated NH<sub>4</sub>Cl solution and extracted with water and ethyl acetate. The combined organic layers were dried over Na<sub>2</sub>SO<sub>4</sub> and evaporated to dryness yielding the title compound (130 mg, 69%). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.62 (s, 1H, ArH), 7.99 (d, <i>J</i> = 8.8 Hz, 3H, ArH), 7.70 (s, 1H, ArH), 7.59—6.99 (m, 10H, ArH), 6.13 (d, <i>J</i> = 4.4 Hz, 1H, ArH), 6.02 (s, 1H, ArH), 3.82 (s, 3H, OCH<sub>3</sub>), 3.78 (s, 3H, OCH<sub>3</sub>), 2.34 (s, 3H, CH<sub>3</sub>) ppm.</div></div><div id="sec4_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i76" class="anchor-spacer"></div><h3 class="article-section__title" id="_i76"> (5-(3,4-Dimethoxyphenyl)-1-tosyl-pyrrolo[2,3-<i>b</i>]pyridin-3-yl)(phenyl)methanone (<b>13</b>)</h3><div class="NLM_p last">To a solution of (5-(3,4-dimethoxyphenyl)-1-tosyl-pyrrolo[2,3-<i>b</i>]pyridin-3-yl) (phenyl)methanol <b>12</b> (130 mg, 0.253 mmol) in DCM (8 mL) was added manganese dioxide (220 mg, 2.53 mmol), and the reaction was stirred overnight at room temperature. After completion, the mixture was filtered through Celite, and the filtrate was evaporated to dryness. The crude residue was purified by silica gel flash column chromatography using a mixture of heptane and ethyl acetate (in a ratio of 6:4) as the mobile phase yielding the title compound (64 mg, 49%). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.78 (d, <i>J</i> = 2.1 Hz, 1H, HetH), 8.64 (d, <i>J</i> = 2.1 Hz, 1H, HetH), 8.33 (s, 1H), 8.16 (d, <i>J</i> = 8.3 Hz, 2H, ArH), 7.96 (d, <i>J</i> = 7.2 Hz, 2H, ArH), 7.70 (m, 3H, ArH), 7.47 (d, <i>J</i> = 8.2 Hz, 2H, ArH), 7.25 (m, 2H, ArH), 7.08 (d, <i>J</i> = 8.4 Hz, 1H, ArH), 3.86 (s, 3H, OCH<sub>3</sub>), 3.81 (s, 3H, OCH<sub>3</sub>), 2.37 (s, 3H, CH<sub>3</sub>) ppm.</div></div><div id="sec4_12" class="NLM_sec NLM_sec_level_2"><div id="ac_i77" class="anchor-spacer"></div><h3 class="article-section__title" id="_i77"> (5-(3,4-Dimethoxyphenyl)-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridin-3-yl)(phenyl)methanone (<b>14</b>)</h3><div class="NLM_p last">To a solution of (5-(3,4-dimethoxyphenyl)-1-tosyl-pyrrolo[2,3-<i>b</i>]pyridin-3-yl) (phenyl)methanone <b>13</b> (64 mg, 0.124 mmol) in ethanol (10 mL) was added KOH (35 mg, 0.624 mmol), and the mixture was heated to 80 °C. The reaction was stirred for 3 h. After reaction completion, the solvent was evaporated in vacuo and the crude residue was partitioned between water and ethyl acetate. The combined organic layers were dried over MgSO<sub>4</sub> and evaporated to dryness, yielding the title compound as a white solid (34 mg, 76%). Purity of 95%. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ 12.74 (br s, 1H, NH), 8.67 (s, 2H, ArH), 8.12 (s, 1H, ArH), 7.84 (d, <i>J</i> = 6.8 Hz, 2H, ArH), 7.61 (m, 3H, ArH), 7.25 (m, 2H, ArH), 7.10 (d, <i>J</i> = 8.3 Hz, 1H, ArH), 3.88 (s, 3H, OCH<sub>3</sub>), 3.82 (s, 3H, OCH<sub>3</sub>) ppm. <sup>13</sup>C NMR (75 MHz, DMSO-<i>d</i><sub>6</sub>): δ 190.00, 149.38, 148.64, 148.45, 143.66, 139.77, 136.58, 131.13, 127.30, 119.42, 118.86, 113.88, 112.59, 110.91, 55.77, 55.75 ppm. HRMS <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>22</sub>H<sub>18</sub>N<sub>2</sub>O<sub>3</sub>, 359.1390; found, 359.1393.</div></div><div id="sec4_13" class="NLM_sec NLM_sec_level_2"><div id="ac_i78" class="anchor-spacer"></div><h3 class="article-section__title" id="_i78"> 5-Bromo-3-iodo-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine (<b>15</b>)</h3><div class="NLM_p last">To a solution of 5-bromo-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine <b>4</b> (1.00 g, 5.08 mmol) in acetone (16 mL) was added portionwise <i>N</i>-iodosuccinimide (1.26 g, 5.58 mmol). The reaction mixture was stirred for 2 h at room temperature. The formed precipitate was collected via filtration, yielding the title compound as an off-white solid (1.50 g, 92%). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ 12.36 (s, 1H, NH), 8.32 (d, <i>J</i> = 2.0 Hz, 1H, HetH), 7.87 (d, <i>J</i> = 1.7 Hz, 1H, HetH), 7.81 (d, <i>J</i> = 2.4 Hz, 1H, HetH) ppm.</div></div><div id="sec4_14" class="NLM_sec NLM_sec_level_2"><div id="ac_i79" class="anchor-spacer"></div><h3 class="article-section__title" id="_i79"> 5-Bromo-3-iodo-1-tosyl-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine (<b>16</b>)</h3><div class="NLM_p last">To a solution of 5-bromo-3-iodo-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine <b>15</b> (300 mg, 0.929 mmol) in dry THF (10 mL) was added portionwise NaH (41 mg, 1.02 mmol) at 0 °C. After stirring for 20 min at room temperature, tosyl chloride (230 mg, 1.21 mmol) was added, and the reaction was allowed to stir for another 2 h at room temperature. After reaction completion, the reaction was quenched with water and partitioned between water and ethyl acetate. The combined organic phases were dried over MgSO<sub>4</sub> and concentrated in vacuo, yielding the title compound as a white solid (420 mg, 95%). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.51 (d, <i>J</i> = 2.1 Hz, 1H, HetH), 8.22 (s, 1H, HetH), 8.00 (m, 3H, HetH/TsH), 7.43 (d, <i>J</i> = 8.5 Hz, 2H, TsH), 2.34 (s, 3H, CH<sub>3</sub>) ppm.</div></div><div id="sec4_15" class="NLM_sec NLM_sec_level_2"><div id="ac_i80" class="anchor-spacer"></div><h3 class="article-section__title" id="_i80"> 5-Bromo-3-phenyl-1-tosyl-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine (<b>17</b>)</h3><div class="NLM_p last">To a solution of 5-bromo-3-iodo-1-tosyl-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine <b>16</b> (200 mg, 0.419 mmol) in a mixture of toluene and ethanol (in a ratio of 3:1) were added phenylboronic acid (51 mg, 0.419 mmol) and a 2 M solution of K<sub>2</sub>CO<sub>3</sub> (420 μL, 0.838 mmol). The reaction was flushed with argon three times, followed by the addition of Pd(PPh<sub>3</sub>)<sub>4</sub> (10 mg, 0.008 mmol). The reaction was stirred at 90 °C for 3 h. After reaction completion, the reaction was filtered through Celite and the filtrate was washed with water and ethyl acetate. The combined organic phases were dried over MgSO<sub>4</sub> and evaporated in vacuo. Purification by silica gel flash column chromatography using a mixture of heptane and ethyl acetate (in a ratio of 8:2) as the mobile phase yielded the title compound as a white solid (150 mg, 84%). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.54 (d, <i>J</i> = 2.1 Hz, 1H, HetH), 8.50 (d, <i>J</i> = 2.1 Hz, 1H, HetH), 8.29 (s, 1H, HetH), 8.04 (d, <i>J</i> = 8.4 Hz, 2H, TsH), 7.79 (d, <i>J</i> = 7.2 Hz, 2H, TsH), 7.45 (m, 5H, PhH), 2.35 (s, 3H, CH<sub>3</sub>) ppm.</div></div><div id="sec4_16" class="NLM_sec NLM_sec_level_2"><div id="ac_i81" class="anchor-spacer"></div><h3 class="article-section__title" id="_i81"> 5-(3,4-Dimethoxyphenyl)-3-phenyl-1-tosyl-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine (<b>18</b>)</h3><div class="NLM_p last">To a solution of 5-bromo-3-phenyl-1-tosyl-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine (150 mg, 0.351 mmol) in a mixture of toluene and ethanol (in a ratio of 3:1) was added 3,4-dimethoxyphenylboronic acid (77 mg, 0.421 mmol) and a 2 M solution of K<sub>2</sub>CO<sub>3</sub> in water (351 μL, 0.702 mmol). The system was flushed with argon and Pd(PPh<sub>3</sub>)<sub>4</sub> (2 mol %) was added. The reaction was stirred for 4 h at 105 °C. Upon reaction completion, the solvents were evaporated in vacuo, and the crude residue was extracted with water and ethyl acetate. Purification by silica gel flash chromatography using a mixture of heptane and ethyl acetate (in a ratio of 8:2) as the mobile phase yielded the desired compound (100 mg, 59%). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.71 (d, <i>J</i> = 2.0 Hz, 1H, HetH), 8.37 (d, <i>J</i> = 2.0 Hz, 1H, HetH), 8.23 (s, 1H, HetH), 8.08 (d, <i>J</i> = 8.4 Hz, 2H, ArH), 7.85 (d, <i>J</i> = 7.2 Hz, 2H, ArH), 7.46 (m, 5H, ArH), 7.29 (m, 2H, ArH), 7.06 (d, <i>J</i> = 8.3 Hz, 1H, ArH), 3.84 (s, 3H, OCH<sub>3</sub>), 3.80 (s, 3H, OCH<sub>3</sub>), 2.35 (s, 3H, CH<sub>3</sub>) ppm.</div></div><div id="sec4_17" class="NLM_sec NLM_sec_level_2"><div id="ac_i82" class="anchor-spacer"></div><h3 class="article-section__title" id="_i82"> 5-(3,4-Dimethoxyphenyl)-3-phenyl-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine (<b>19</b>)</h3><div class="NLM_p last">To a solution of 5-(3,4-dimethoxyphenyl)-3-phenyl-1-tosyl-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine <b>18</b> (100 mg, 0.309 mmol) in ethanol (5 mL) was added KOH (87 mg, 1.55 mmol). The reaction was allowed to stir for 2 h at 80 °C. After completion, the solvent was evaporated in vacuo and the crude residue was purified by silica gel flash chromatography using a mixture of heptane and ethyl acetate (in a ratio of 7:3) as the mobile phase, affording the title compound (49 mg, 72%). Purity of 97% <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ 11.98 (s, 1H, NH), 8.56 (s, 1H, ArH), 8.39 (s, 1H, ArH), 7.90 (d, <i>J</i> = 2.3 Hz, 1H, ArH), 7.79 (d, <i>J</i> = 7.5 Hz, 2H, ArH), 7.45 (t, <i>J</i> = 7.6 Hz, 2H, ArH), 7.28 (m, 3H, ArH), 7.06 (d, <i>J</i> = 8.3 Hz, 1H, ArH), 3.87 (s, 3H, OCH<sub>3</sub>), 3.80 (s, 3H, OCH<sub>3</sub>) ppm. <sup>13</sup>C NMR (75 MHz, DMSO-<i>d</i><sub>6</sub>): δ 149.32, 148.56, 148.37, 142.16, 135.18, 132.07, 129.15, 129.06, 126.57, 125.83, 124.63, 119.50, 117.37, 114.73, 112.54, 111.27, 55.84, 55.76 ppm. HRMS <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>21</sub>H<sub>18</sub>N<sub>2</sub>O<sub>2</sub>, 331.1440; found, 331.1437.</div></div><div id="sec4_18" class="NLM_sec NLM_sec_level_2"><div id="ac_i83" class="anchor-spacer"></div><h3 class="article-section__title" id="_i83"> Synthesis of 3-Alkynyl-5-bromo-pyrrolo[2,3-<i>b</i>]pyridines (<b>20a–f</b>)</h3><div id="sec4_18_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i84" class="anchor-spacer"></div><h4 class="article-section__title" id="_i84"> General Procedure</h4><div class="NLM_p last">To a degassed solution of 5-bromo-3-iodo-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine <b>15</b> (1 equiv) in THF and triethylamine (3 equiv) were added CuI (0.02 equiv) and Pd(PPh<sub>3</sub>)<sub>2</sub>Cl<sub>2</sub> (0.01 equiv). The resulting mixture was stirred under an inert atmosphere for 10 min at room temperature. A solution of the appropriate alkyne (0.95 equiv) in THF was added to the mixture. The reaction mixture was stirred for an additional 4 h at room temperature. After completion, the reaction was filtered through Celite. The filtrate was extracted with water and ethyl acetate, washed with brine, and dried over MgSO<sub>4</sub>. The crude residue was purified by silica gel flash column chromatography with an appropriate mobile phase. Compounds <b>20a–f</b> were made according to this procedure.</div></div><div id="sec4_18_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i85" class="anchor-spacer"></div><h4 class="article-section__title" id="_i85"> 5-Bromo-3-(phenylethynyl)-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine (<b>20a</b>)</h4><div class="NLM_p last">The title compound was synthesized according to the general procedure using 5-bromo-3-iodo-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine <b>15</b> (200 mg, 0.619 mmol) and phenylacetylene (60 mg, 0.588 mmol). The crude residue was purified by silica gel flash column chromatography using a mixture of heptane and ethyl acetate (in a ratio of 8:2) as the mobile phase yielding the title compound as a white solid (85 mg, 51%). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) 12.40 (br s, 1H, NH), 8.39 (d, <i>J</i> = 2.2 Hz, 1H, HetH), 8.32 (d, <i>J</i> = 2.1 Hz, 1H, HetH), 8.01 (s, 1H, HetH), 7.60 (m, 2H, PhH), 7.42 (m, 3H, PhH) ppm.</div></div><div id="sec4_18_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i86" class="anchor-spacer"></div><h4 class="article-section__title" id="_i86"> 5-Bromo-3-(pyridin-3-ylethynyl)-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine (<b>20b</b>)</h4><div class="NLM_p last">The title compound was synthesized according to the general procedure using 5-bromo-3-iodo-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine <b>15</b> (200 mg, 0.619 mmol) and 3-ethynylpyridine (61 mg, 0.588 mmol). The crude residue was purified by silica gel flash column chromatography using a mixture of heptane and ethyl acetate (in a ratio of 7:3) as the mobile phase yielding the title compound as a white solid (140 mg, 75%). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) 12.49 (br s, 1H, NH), 8.81 (s, 1H, ArH), 8.56 (d, <i>J</i> = 4.8 Hz, 1H, ArH), 8.40 (s, 2H, ArH), 8.04 (m, 2H, ArH), 7.46 (m, 1H, ArH) ppm.</div></div><div id="sec4_18_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i87" class="anchor-spacer"></div><h4 class="article-section__title" id="_i87"> 5-Bromo-3-((3-methoxyphenyl)ethynyl)-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine (<b>20c</b>)</h4><div class="NLM_p last">The title compound was synthesized according to the general procedure using 5-bromo-3-iodo-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine <b>15</b> (200 mg, 0.619 mmol) and 3-methoxyphenylacetylene (60 mg, 0.588 mmol). The crude residue was purified by silica gel flash column chromatography using a mixture of heptane and ethyl acetate (in a ratio of 7:3) as the mobile phase, yielding the title compound as a white solid (105 mg, 51%). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ 12.41 (br s, 1H, NH), 8.39 (d, <i>J</i> = 2.2 Hz, 1H, HetH), 8.34 (d, <i>J</i> = 2.1 Hz, 1H, HetH), 8.00 (s, 1H, HetH), 7.33 (t, <i>J</i> = 8.1 Hz, 1H, PhH), 7.16 (m, 2H, PhH), 6.96 (m, 1H, PhH), 3.81 (s, 3H, OCH<sub>3</sub>) ppm.</div></div><div id="sec4_18_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i88" class="anchor-spacer"></div><h4 class="article-section__title" id="_i88"> 5-Bromo-3-(<i>p</i>-tolylethynyl)-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine (<b>20d</b>)</h4><div class="NLM_p last">The title compound was synthesized according to the general procedure using 5-bromo-3-iodo-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine <b>15</b> (200 mg, 0.619 mmol) and <i>p</i>-tolylacetylene (68 mg, 0.588 mmol). The crude residue was purified by silica gel flash column chromatography using a mixture of heptane and ethyl acetate (in a ratio of 7:3) as the mobile phase, yielding the title compound as a white solid (130 mg, 67%). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ 12.36 (br s, 1H, NH), 8.38 (d<i>, J</i> = 2.2 Hz, 1H, HetH), 8.30 (d, <i>J</i> = 2.2 Hz, 1H, HetH), 7.98 (s, 1H, HetH), 7.48 (d, <i>J</i> = 8.1 Hz, 2H, PhH), 7.23 (d, <i>J</i> = 7.9 Hz, 2H, PhH), 2.34 (s, 3H, CH<sub>3</sub>).</div></div><div id="sec4_18_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i89" class="anchor-spacer"></div><h4 class="article-section__title" id="_i89"> 5-Bromo-3-((4-fluorophenyl)ethynyl)-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine (<b>20e</b>)</h4><div class="NLM_p last">The title compound was synthesized according to the general procedure using 5-bromo-3-iodo-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine <b>15</b> (200 mg, 0.619 mmol) and 4-fluorophenylacetylene (60 mg, 0.588 mmol). The crude residue was purified by silica gel flash column chromatography using a mixture of heptane and ethyl acetate (in a ratio of 8:2) as the mobile phase yielding the title compound as a white solid (110 mg, 56%). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ 12.41 (br s, 1H, NH), 8.38 (d, <i>J</i> = 2.1 Hz, 1H, HetH), 8.34 (d, <i>J</i> = 2.1 Hz, 1H, HetH), 8.00 (s, 1H, HetH), 7.66 (dd, <i>J</i> = 8.7, 5.6 Hz, 2H, PhH), 7.27 (t, <i>J</i> = 8.9 Hz, 2H, PhH) ppm.</div></div><div id="sec4_18_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i90" class="anchor-spacer"></div><h4 class="article-section__title" id="_i90"> 3-((5-Bromo-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridin-3-yl)ethynyl)benzonitrile (<b>20f</b>)</h4><div class="NLM_p last">The title compound was synthesized according to the general procedure using 5-bromo-3-iodo-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine <b>15</b> (200 mg, 0.619 mmol) and 3-cyanophenylacetylene (75 mg, 0.588 mmol). The crude residue was purified by silica gel flash column chromatography using a mixture of heptane and ethyl acetate (in a ratio of 7:3) as the mobile phase, yielding the title compound as a white solid (90 mg, 45%). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.45 (d, <i>J</i> = 2.2 Hz, 1H, HetH), 8.40 (d, <i>J</i> = 2.1 Hz, 1H, HetH), 8.14 (s, 1H, HetH), 8.05 (s, 1H, PhH), 7.91 (d, <i>J</i> = 8.0 Hz, 1H, PhH), 7.84 (d, <i>J</i> = 7.5 Hz, 1H, PhH), 7.63 (t, <i>J</i> = 7.9 Hz, 1H, PhH) ppm.</div></div></div><div id="sec4_19" class="NLM_sec NLM_sec_level_2"><div id="ac_i91" class="anchor-spacer"></div><h3 class="article-section__title" id="_i91"> Synthesis of 3-Alkynyl-5-(3,4-dimethoxyphenyl)-pyrrolo[2,3-<i>b</i>]pyridines (<b>21a–f</b>)</h3><div id="sec4_19_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i92" class="anchor-spacer"></div><h4 class="article-section__title" id="_i92"> General Procedure</h4><div class="NLM_p last">To a solution of a 5-bromo-3-arylethynyl-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine derivatrive (1 equiv) in dioxane (4 mL) were added 3,4-dimethoxyphenylboronic acid (1.2 equiv) and 1 mL of a K<sub>2</sub>CO<sub>3</sub> (3 equiv) solution. The system was purged three times with argon and heated to 105 °C. After stirring for 10 min, Pd(PPh<sub>3</sub>)<sub>4</sub> (0.1 equiv) was added and the reaction was purged once more with argon. The reaction mixture was stirred at 105 °C for 3 h. After completion, the reaction mixture was cooled to room temperature and filtered through Celite. The filtrate was extracted with water and ethyl acetate. The combined organic layers were washed with brine, dried over MgSO<sub>4</sub>, and evaporated in vacuo. The crude residue was purified by silica gel flash column chromatography with an appropriate mobile phase. Compounds <b>21a–f</b> were made according to this procedure.</div></div><div id="sec4_19_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i93" class="anchor-spacer"></div><h4 class="article-section__title" id="_i93"> 5-(3,4-Dimethoxyphenyl)-3-(phenylethynyl)-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine (<b>21a</b>)</h4><div class="NLM_p last">The title compound was synthesized according to the general procedure using 3-(phenylethynyl)-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine <b>20a</b> (80 mg, 0.269 mmol). The crude residue was purified by silica gel flash column chromatography using a mixture of DCM and ethyl acetate (in a ratio of 9:1) as the mobile phase yielding the title compound as a brown solid (36 mg, 38%). Purity of 99%. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ 12.21 (br s, 1H, NH), 8.61 (s, 1H, ArH), 8.25 (s, 1H, ArH), 7.96 (d, <i>J</i> = 2.2 Hz, 1H, ArH), 7.60 (d, <i>J</i> = 6.5 Hz, 1H, ArH), 8.01 (m, 2H, ArH), 7.35 (m, 6H, ArH), 7.06 (d, <i>J</i> = 8.3 Hz, 1H, ArH), 3.88 (s, 3H, OCH<sub>3</sub>), 3.81 (s, 3H, OCH<sub>3</sub>) ppm. <sup>13</sup>C NMR (75 MHz, DMSO-<i>d</i><sub>6</sub>): δ 149.33, 148.49, 147.24, 143.12, 131.42, 131.27, 131.17, 129.60, 128.80, 128.13, 124.93, 123.48, 120.29, 119.44, 112.48, 111.11, 95.31, 90.75, 83.53, 55.82, 55.73 ppm. HRMS <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>23</sub>H<sub>18</sub>N<sub>2</sub>O<sub>2</sub>, 355.14409; found, 355.1435.</div></div><div id="sec4_19_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i94" class="anchor-spacer"></div><h4 class="article-section__title" id="_i94"> 5-(3,4-Dimethoxyphenyl)-3-(pyridin-3-ylethynyl)-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine (<b>21b</b>)</h4><div class="NLM_p last">The title compound was synthesized according to the general procedure using 5-bromo-3-(pyridin-3-ylethynyl)-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine <b>20b</b> (140 mg, 0.470 mmol). The crude residue was purified by silica gel flash column chromatography using a mixture of DCM and ethyl acetate (in a ratio of 9:1) as the mobile phase, yielding the title compound as a white solid (75 mg, 44%). Purity of 98% <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ 12.30 (s, 1H, NH), 8.81 (s, 1H, ArH), 8.63 (d, <i>J</i> = 2.0 Hz, 1H, ArH), 8.55 (d, <i>J</i> = 4.9 Hz, 1H, ArH), 8.30 (d, <i>J</i> = 1.9 Hz, 1H, ArH), 8.01 (m, 2H, ArH), 7.46 (m, 1H, ArH), 7.31 (m, 2H, ArH), 7.07 (d, <i>J</i> = 8.3 Hz, 1H), 3.89 (s, 3H, OCH<sub>3</sub>), 3.81 (s, 3H, OCH<sub>3</sub>) ppm. <sup>13</sup>C NMR (75 MHz, DMSO-<i>d</i><sub>6</sub>): δ 151.46, 149.33, 148.52, 148.34, 147.24, 143.24, 138.19, 131.78, 131.33, 129.73, 125.03, 123.68, 120.59, 120.28, 119.46, 112.47, 111.13, 94.77, 87.69, 86.87, 55.83, 55.74 ppm. HRMS <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>22</sub>H<sub>17</sub>N<sub>3</sub>O<sub>2</sub>, 356.13934; found, 356.1393.</div></div><div id="sec4_19_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i95" class="anchor-spacer"></div><h4 class="article-section__title" id="_i95"> 5-(3,4-Dimethoxyphenyl)-3-(3-methoxyphenylethynyl)-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine (<b>21c</b>)</h4><div class="NLM_p last">The title compound was synthesized according to the general procedure using 3-(3-methoxyphenylethynyl)-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine <b>20c</b> (110 mg, 0.353 mmol). The crude residue was purified by silica gel flash column chromatography using a mixture of DCM and ethyl acetate (in a ratio of 9:1). Further purification by preparative TLC (the mobile phase being a mixture of nitromethane/toluene in a ratio of 6:4) yielded the title compound as a white solid (25 mg, 20%). Purity of 97%. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ 12.21 (s, 1H, NH), 8.61 (d, <i>J</i> = 1.9 Hz, 1H, ArH), 8.25 (d, <i>J</i> = 1.9 Hz, 1H, ArH), 7.96 (d, <i>J</i> = 2.6 Hz, 1H, ArH), 7.32 (m, 3H, ArH), 7.16 (m, 2H, ArH), 7.07 (d, <i>J</i> = 8.3 Hz, 1H, ArH), 6.96 (m, 1H, ArH), 3.89 (s, 3H, OCH<sub>3</sub>), 3.81 (s, 3H, OCH<sub>3</sub>), 3.80 (s, 3H, OCH<sub>3</sub>) ppm. <sup>13</sup>C NMR (75 MHz, DMSO-<i>d</i><sub>6</sub>): δ 159.33, 149.33, 148.50, 147.24, 143.12, 131.41, 131.34, 129.91, 129.60, 124.97, 124.56, 123.61, 120.28, 119.44, 115.95, 114.58, 112.49, 111.12, 95.24, 90.74, 83.43, 55.82, 55.75, 55.34 ppm. HRMS <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>24</sub>H<sub>20</sub>N<sub>2</sub>O<sub>3</sub>, 385.15465; found, 385.1541.</div></div><div id="sec4_19_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i96" class="anchor-spacer"></div><h4 class="article-section__title" id="_i96"> 5-(3,4-Dimethoxyphenyl)-3-(<i>p</i>-tolylethynyl)-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine (<b>21d</b>)</h4><div class="NLM_p last">The title compound was synthesized according to the general procedure using 3-(<i>p</i>-tolylethynyl)-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine <b>20d</b> (110 mg, 0.353 mmol). The crude residue was purified by silica gel flash column chromatography using a mixture of DCM and ethyl acetate (in a ratio of 9:1) as the mobile phase, yielding the title compound as a white solid (25 mg, 20%). Purity of 95%. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ 12.18 (br s, 1H, NH), 8.60 (d, <i>J</i> = 2.0 Hz, 1H, ArH), 8.22 (d, <i>J</i> = 1.8 Hz, 1H, ArH), 7.93 (d, <i>J</i> = 2.6 Hz, 1H, ArH), 7.49 (d, <i>J</i> = 8.0 Hz, 2H, ArH), 7.27 (m, 4H, ArH), 7.03 (m, 2H, ArH), 3.88 (s, 3H, OCH<sub>3</sub>), 3.81 (s, 3H, OCH<sub>3</sub>), 2.34 (s, 3H, CH<sub>3</sub>). <sup>13</sup>C NMR (75 MHz, DMSO-<i>d</i><sub>6</sub>): δ 149.33, 148.48, 147.23, 143.07, 137.78, 131.45, 131.11, 131.04, 129.54, 129.42, 124.91, 120.46, 120.27, 119.43, 112.49, 111.11, 95.49, 90.81, 82.77, 55.83, 55.74, 21.14. HRMS <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>24</sub>H<sub>20</sub>N<sub>2</sub>O<sub>2</sub>, 369.15974; found, 369.1593.</div></div><div id="sec4_19_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i97" class="anchor-spacer"></div><h4 class="article-section__title" id="_i97"> 5-(3,4-Dimethoxyphenyl)-3-(4-fluorophenylethynyl)-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine (<b>21e</b>)</h4><div class="NLM_p last">The title compound was synthesized according to the general procedure using 3-(4-fluorophenylethynyl)-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine <b>20e</b> (80 mg, 0.269 mmol). The crude residue was purified by silica gel flash column chromatography using a mixture of DCM and ethyl acetate (in a ratio of 9:1), yielding the desired compound as a brown solid (34 mg, 36%). Purity of 98%. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ 12.21 (br s, 1H, NH), 8.61 (s, 1H, ArH), 8.25 (s, 1H, ArH), 7.95 (d, <i>J</i> = 2.4 Hz, 1H, ArH), 7.65 (m, 2H, ArH), 7.28 (m, 4H, ArH), 7.06 (d, <i>J</i> = 8.5 Hz, 1H, ArH), 3.88 (s, 3H), 3.81 (s, 3H, OCH<sub>3</sub>) ppm. <sup>13</sup>C NMR (150 MHz, DMSO-<i>d</i><sub>6</sub>): δ 161.68 (d, <i>J</i><sub>CF</sub> = 247 Hz) 149.27, 148.44, 147.17, 143.07, 133.36 (d, <i>J</i><sub>CF</sub> = 8.3 Hz), 131.35, 131.20, 129.54, 124.91, 120.23, 119.90, 119.89, 119.38, 115.95 (d, <i>J</i><sub>CF</sub> = 22 Hz), 112.42, 111.05, 95.11, 89.60, 83.20, 55.76, 55.68 ppm. HRMS <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>23</sub>H<sub>17</sub>FN<sub>2</sub>O<sub>2</sub>, 373.13467; found, 373.1333.</div></div><div id="sec4_19_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i98" class="anchor-spacer"></div><h4 class="article-section__title" id="_i98"> 3-((5-(3,4-Dimethoxyphenyl)-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridin-3-yl)ethynyl)benzonitrile (<b>21f</b>)</h4><div class="NLM_p last">The title compound was synthesized according to the general procedure using 3-(3-cyanophenylethynyl)-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine <b>20f</b> (100 mg, 0.310 mmol). The crude residue was purified by silica gel flash column chromatography using a mixture of DCM and ethyl acetate (in a ratio of 9:1) as the mobile phase, yielding the title compound as a brown solid (44 mg, 37%). Purity of 98%. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ 12.31 (br s, 1H, NH), 8.62 (d, <i>J</i> = 2.0 Hz, 1H, ArH), 8.32 (d, <i>J</i> = 1.9 Hz, 1H, ArH), 8.11 (s, 1H, ArH), 8.00 (d, <i>J</i> = 2.5 Hz, 1H, ArH), 7.91 (d, <i>J</i> = 7.9 Hz, 1H, ArH), 7.82 (d, <i>J</i> = 7.9 Hz, 1H, ArH), 7.62 (t, <i>J</i> = 7.9 Hz, 2H, ArH), 7.31 (m, 2H, ArH), 7.07 (d, <i>J</i> = 8.3 Hz, 1H, ArH), 3.89 (s, 3H, OCH<sub>3</sub>), 3.81 (s, 3H, OCH<sub>3</sub>). <sup>13</sup>C NMR (75 MHz, DMSO-<i>d</i><sub>6</sub>): δ 149.33, 148.54, 147.25, 143.28, 135.52, 134.37, 131.91, 131.43, 131.35, 130.10, 129.81, 125.11, 124.93, 120.34, 119.51, 118.33, 112.48, 112.17, 111.17, 94.64, 89.01, 86.05, 55.84, 55.75. HRMS <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>24</sub>H<sub>17</sub>N<sub>3</sub>O<sub>2</sub>, 380.13934; found, 380.1390.</div></div></div><div id="sec4_20" class="NLM_sec NLM_sec_level_2"><div id="ac_i99" class="anchor-spacer"></div><h3 class="article-section__title" id="_i99"> 5-Bromo-1-methyl-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine (<b>22</b>)</h3><div class="NLM_p last">To a solution of 5-bromo-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine <b>4</b> (200 mg, 1.02 mmol) in THF (5 mL) at 0 °C was added 60% NaH on mineral oil (49 mg, 1.22 mmol). The reaction was stirred at 0 °C for 1.5 h. Methyl iodide (70 μL, 1.12 mmol) was added, and the mixture was allowed to warm at ambient temperature and then stirred at room temperature for 3 h. After completion, the mixture was quenched with water and extracted three times with ethyl acetate. The combined organic layers were washed with brine, dried over MgSO<sub>4</sub>, filtered, and evaporated. The crude residue was purified by silica gel flash column chromatography using a mixture of heptane and ethyl acetate (in a ratio of 8:2) as the mobile phase, yielding the title compound as an oil that subsequently crystallized to an off-white solid (160 mg, 75%). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.31 (d, <i>J</i> = 2.07 Hz, 1H, HetH), 8.20 (d, <i>J</i> = 2.07 Hz, 1H, HetH), 7.59 (d, <i>J</i> = 3.39 Hz, 1H, HetH), 6.46 (d, <i>J</i> = 3.42 Hz, 1H, HetH), 3.81 (s, 3H, CH<sub>3</sub>) ppm. HRMS <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>8</sub>H<sub>7</sub>N<sub>2</sub>Br, 210.98658; found, 210.9866.</div></div><div id="sec4_21" class="NLM_sec NLM_sec_level_2"><div id="ac_i100" class="anchor-spacer"></div><h3 class="article-section__title" id="_i100"> 5-Bromo-1-methyl-3-nitro-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine (<b>23</b>)</h3><div class="NLM_p last">5-Bromo-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine <b>22</b> (130 mg, 0.615 mmol) was added portionwise to a stirring solution of fuming nitric acid (1 mL) at 0 °C over 10 min. The reaction was allowed to stir for 30 min at 0 °C. The mixture was poured out in ice water and the formed precipitate was collected via vacuum filtration. The filter cake was washed generously with water and heptane, yielding the title compound as a pink solid (155 mg, 98%). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.99 (s, 1H, HetH), 8.61 (s, 1H, HetH), 8.57 (m, 1H, HetH), 3.92 (s, 3H, CH<sub>3</sub>) ppm.</div></div><div id="sec4_22" class="NLM_sec NLM_sec_level_2"><div id="ac_i101" class="anchor-spacer"></div><h3 class="article-section__title" id="_i101"> 5-(3,4-Dimethoxyphenyl)-1-methyl-3-nitro-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine (<b>24</b>)</h3><div class="NLM_p last">To a solution of 5-bromo-1-methyl-3-nitro-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine <b>23</b> (100 mg, 0.391 mmol) in dioxane (4 mL) were added 3,4-dimethoxyphenylboronic acid (69 mg, 0.469 mmol) and 1 mL of a K<sub>2</sub>CO<sub>3</sub> solution (161 mg, 1.17 mmol). The system was purged three times with argon and heated to 105 °C. After stirring for 10 min, Pd(PPh<sub>3</sub>)<sub>4</sub> (10 mol %) was added and the reaction was purged once more with argon. The reaction mixture was stirred at 105 °C overnight. After completion, the reaction mixture was cooled to room temperature and filtered through Celite. Extraction was performed with water and ethyl acetate. The combined organic layers were washed with brine, dried over MgSO<sub>4</sub>, and evaporated. Purification by silica gel flash chromatography using a mixture of DCM and ethyl acetate (in a ratio of 9:1) as the mobile phase yielded the title compound as a yellow solid (90 mg, 83%). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.96 (s, 1H, HetH), 8.79 (d, <i>J</i> = 2.07 Hz, 1H, HetH), 8.56 (d, <i>J</i> = 2.01 Hz, 1H, HetH), 7.30 (m, 2H, PhH), 7.10 (d, <i>J</i> = 8.22 Hz, 1H, PhH), 3.96 (s, 3H), 3.88 (s, 3H, OCH<sub>3</sub>), 3.82 (s, 3H) ppm.</div></div><div id="sec4_23" class="NLM_sec NLM_sec_level_2"><div id="ac_i102" class="anchor-spacer"></div><h3 class="article-section__title" id="_i102"> <i>N</i>-(5-(3,4-Dimethoxyphenyl)-1-methyl-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridin-3-yl)nicotinamide (<b>26</b>)</h3><div class="NLM_p last">The title compound was synthesized according to the general procedure described for the synthesis of compounds <b>8a–h</b>, starting from 5-(3,4-dimethoxyphenyl)-1-methyl-3-nitro-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine <b>24</b> (90 mg, 0.287 mmol). Purification by silica gel flash column chromatography using a mixture of DCM and methanol (in a ratio gradually ranging from 98:2 to 95:5) yielded the title compound as a red solid (22 mg, 20%). Purity of 98%. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ 10.60 (br s, 1H, NH), 9.19 (br s, 1H, NH), 8.77 (d, <i>J</i> = 2.6 Hz, 1H, ArH), 8.60 (s, 2H, ArH), 8.36 (d, <i>J</i> = 8.0 Hz, 1H, ArH), 8.07 (s, 1H, ArH), 7.59 (m, 1H, ArH), 7.26 (m, 2H, ArH), 7.08 (d, <i>J</i> = 8.3 Hz, 1H, ArH), 3.87 (s, 6H, OCH<sub>3</sub>/CH<sub>3</sub>), 3.80 (s, 3H, OCH<sub>3</sub>/CH<sub>3</sub>). <sup>13</sup>C NMR (75 MHz, DMSO-<i>d</i><sub>6</sub>): δ 163.33, 152.03, 149.33, 148.87, 148.39, 144.19, 142.14, 135.65, 131.81, 130.43, 127.77, 125.22, 123.63, 121.31, 119.21, 113.40, 113.14, 112.54, 110.93, 55.83, 55.75, 30.91 ppm. HRMS <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>22</sub>H<sub>20</sub>N<sub>4</sub>O<sub>3</sub>, 389.16080; found, 389.1604.</div></div><div id="sec4_24" class="NLM_sec NLM_sec_level_2"><div id="ac_i103" class="anchor-spacer"></div><h3 class="article-section__title" id="_i103"> 2-Chloro-5-(3,4-dimethoxyphenyl)nicotinonitrile (<b>28</b>)</h3><div class="NLM_p last">To a solution of 3,4-dimethoxyphenylboronic acid (460 mg, 2.53 mmol) and 5-bromo-2-chloronicotinonitrile <b>27</b> (500 mg, 2.3 mmol) in isopropanol (12 mL) was added a solution of K<sub>2</sub>CO<sub>3</sub> (953 mg, 6.9 mmol) in water (4 mL). The reaction mixture was flushed three times with argon and heated to 90 °C. Then, Pd(PPh<sub>3</sub>)<sub>4</sub> (10 mol %) was added and the system was flushed once more with argon. After stirring at 90 °C for 2.5 h, the reaction was concentrated under reduced pressure. The crude residue was partitioned between ethyl acetate and water and extracted three times. The combined organic phases were washed with brine and dried over MgSO<sub>4</sub>. The solvent was removed under reduced pressure and the crude residue was purified by silica gel flash column chromatography using a mixture of heptane and ethylacetate (in a ratio of 7:3) as the mobile phase. This was followed by a second purification by silica gel flash column chromatography using a mixture of DCM, heptane, and ethylacetate (in a ratio of 70:25:5) as the mobile phase, yielding the title compound as a white solid (320 mg, 50%). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.06 (d, <i>J</i> = 2.5 Hz, 1H, HetH), 8.85 (d, <i>J</i> = 2.5 Hz, 1H, HetH), 7.41 (m, 2H, PhH), 7.10 (m, 1H, PhH), 3.87 (s, 3H, OCH<sub>3</sub>), 3.82 (s, 3H, OCH<sub>3</sub>).</div></div><div id="sec4_25" class="NLM_sec NLM_sec_level_2"><div id="ac_i104" class="anchor-spacer"></div><h3 class="article-section__title" id="_i104"> 5-(3,4-Dimethoxyphenyl)-1<i>H</i>-pyrazolo[3,4-<i>b</i>]pyridin-3-amine (<b>29</b>)</h3><div class="NLM_p last">To a solution of 2-chloro-5-(3,4-dimethoxyphenyl)nicotinonitrile <b>28</b> (100 mg, 0.364 mmol) in pyridine (3 mL) was added a hydrazine monohydrate solution (65% in water, 103 μL, 0.728 mmol). The reaction was refluxed overnight, and after completion, the solvent was evaporated. The crude residue was purified by silica gel flash column chromatography using a mixture of DCM and methanol (in a ratio of 95:5), yielding the title compound as a yellow solid (90 mg, 91%). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ 11.97 (br s, 1H, NH), 8.66 (d, <i>J</i> = 2.1 Hz, 1H, HetH), 8.36 (d, <i>J</i> = 2.0 Hz, 1H, HetH), 7.21 (m, 2H, PhH), 7.07 (d, <i>J</i> = 8.4 Hz, 1H, PhH), 3.86 (s, 3H, OCH<sub>3</sub>), 3.80 (s, 3H, OCH<sub>3</sub>).</div></div><div id="sec4_26" class="NLM_sec NLM_sec_level_2"><div id="ac_i105" class="anchor-spacer"></div><h3 class="article-section__title" id="_i105"> <i>N</i>-(5-(3,4-Dimethoxyphenyl)-1<i>H</i>-pyrazolo[3,4-<i>b</i>]pyridin-3-yl)nicotinamide (<b>30</b>)</h3><div class="NLM_p last">To a solution of 5-(3,4-dimethoxyphenyl)-1<i>H</i>-pyrazolo[3,4-<i>b</i>]pyridin-3-amine <b>29</b> (90 mg, 0.332 mmol) in pyridine (3 mL) at 0 °C was added nicotinoyl chloride hydrochloride (71 mg, 0.400 mmol). The reaction was stirred at 0 °C for 1 h and then stirred at room temperature overnight. After reaction completion, water was added and the mixture was extracted three times with ethyl acetate. The combined organic phases were washed with brine, dried over MgSO<sub>4</sub>, and evaporated to dryness. The crude residue was purified by silica gel flash chromatography using a mixture of DCM and methanol (in a ratio gradually ranging from 98:2 to 96:4), yielding the title compound as a beige solid (76 mg, 61%). Purity of 99%. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ 13.47 (s, 1H), 11.33 (s, 1H), 9.24 (s, 1H), 8.85 (d, <i>J</i> = 2.1 Hz, 1H), 8.79 (d, <i>J</i> = 3.4 Hz, 1H), 8.50 (d, <i>J</i> = 2.0 Hz, 1H), 8.42 (d, <i>J</i> = 8.1 Hz, 1H), 7.59 (dd, <i>J</i> = 7.7, 4.8 Hz, 1H), 7.28 (s, 1H), 7.23 (d, <i>J</i> = 8.4 Hz, 1H), 7.07 (d, <i>J</i> = 8.4 Hz, 1H), 3.86 (s, 3H), 3.80 (s, 3H). <sup>13</sup>C NMR (75 MHz, DMSO-<i>d</i><sub>6</sub>): δ 164.30, 152.57, 151.36, 149.35, 149.22, 148.89, 148.67, 139.54, 135.92, 130.86, 129.45, 129.25, 129.13, 123.66, 119.50, 112.54, 111.08, 108.70, 55.84, 55.74. HRMS <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>20</sub>H<sub>17</sub>N<sub>5</sub>O<sub>3</sub>, 376.14040; found, 376.1400.</div></div><div id="sec4_27" class="NLM_sec NLM_sec_level_2"><div id="ac_i106" class="anchor-spacer"></div><h3 class="article-section__title" id="_i106"> 2-Amino-3,5-dibromopyrazine (<b>32</b>)</h3><div class="NLM_p last">To a solution of aminopyrazine <b>31</b> (1 g, 10.52 mmol) in DMSO (10 mL) was added <i>N</i>-bromosuccinimide (3.94 g, 22.08 mmol) portionwise over 45 min. The resulting mixture was stirred for 3 h at room temperature. The reaction was poured in ice water and extracted five times with ethyl acetate. The combined organic phases were washed with brine, dried over MgSO<sub>4</sub>, and evaporated to dryness. The crude residue was purified by silica gel flash column chromatography using a mixture of heptane and ethyl acetate (in a ratio of 7:3) as the mobile phase, yielding the desired compound as a fluffy white solid (1.94 g, 73%). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.13 (s, 1H, ArH), 6.99 (br s, 2H, NH<sub>2</sub>) ppm. HRMS <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>4</sub>H<sub>3</sub>N<sub>3</sub>Br<sub>2</sub>, 251.87675; found, 251.8765.</div></div><div id="sec4_28" class="NLM_sec NLM_sec_level_2"><div id="ac_i107" class="anchor-spacer"></div><h3 class="article-section__title" id="_i107"> 5-Bromo-3-((trimethylsilyl)ethynyl)pyrazin-2-amine (<b>33</b>)</h3><div class="NLM_p last">To a solution of 2-amino-3,5-dibromopyrazine <b>32</b> (1 g, 3.95 mmol) in degassed THF (15 mL) was added copper iodide (7.53 mg, 0.040 mmol), Pd(PPh<sub>3</sub>)<sub>2</sub>Cl<sub>2</sub> (45 mg, 0.040 mmol), and triethylamine (1.65 mL, 11.86 mmol). The system was flushed with nitrogen, and trimethylsilylacetylene (534 μL, 1.91 mmol) was added dropwise over 5 min. The reaction was stirred overnight at room temperature. After reaction completion, the solvent was evaporated and water was added. The resulting suspension was extracted three times with ethyl acetate. The combined organic phases were washed with water and brine, dried over MgSO<sub>4</sub>, and evaporated to dryness. The crude residue was purified by silica gel flash column chromatography using a mixture of heptane and ethylacetate (in a ratio of 80:20) as the mobile phase, yielding the title compound as a bright yellow solid (769 mg, 72%). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.12 (s, 1H, ArH), 6.82 (s, 2H, NH<sub>2</sub>), 0.27 (s, 9H, Si(CH<sub>3</sub>)<sub>3</sub>) ppm.</div></div><div id="sec4_29" class="NLM_sec NLM_sec_level_2"><div id="ac_i108" class="anchor-spacer"></div><h3 class="article-section__title" id="_i108"> 2-Bromo-5<i>H</i>-pyrrolo[2,3-<i>b</i>]pyrazine (<b>34</b>)</h3><div class="NLM_p last">To a solution of 5-bromo-3-((trimethylsilyl)ethynyl)pyridin-2-amine <b>33</b> (1 g, 3.7 mmol) in dry NMP (10 mL) was added portionwise KO<i>t</i>Bu (498 mg, 4.44 mmol). The reaction mixture was flushed with nitrogen and stirred for 3 h at 80 °C. After reaction completion, the mixture was extracted three times with water and ethyl acetate. The combined organic layers were washed twice with water and once with brine, dried over MgSO<sub>4</sub>, and evaporated in vacuo. The crude residue was then purified by silica gel flash column chromatography using a mixture of heptane/ethyl acetate (in a ratio of 7:3) as the mobile phase, yielding the title compound as a yellow solid (502 mg, 69%). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ 12.42 (br s, 1H, NH), 8.35 (s, 1H, ArH), 7.97 (d, <i>J</i> = 3.48 Hz, 1H, ArH), 6.63 (d, <i>J</i> = 3.48 Hz, 1H, ArH) ppm.</div></div><div id="sec4_30" class="NLM_sec NLM_sec_level_2"><div id="ac_i109" class="anchor-spacer"></div><h3 class="article-section__title" id="_i109"> 5-Bromo-3-iodo-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyrazine (<b>35</b>)</h3><div class="NLM_p last">To a solution of 5-bromo-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyrazine <b>34</b> (500 mg, 2.52 mmol) in a minimal volume of acetone was added portionwise <i>N</i>-iodosuccinimide (625 g, 2.78 mmol). The reaction mixture was stirred for 2 h at room temperature. The formed precipitate was collected via filtration, yielding the title product as an off-white solid (408 mg, 50%).. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ 12.82 (br s, 1H, NH), 8.40 (d, <i>J</i> = 1.77 Hz, 1H, ArH), 8.19 (s, 1H, ArH) ppm.</div></div><div id="sec4_31" class="NLM_sec NLM_sec_level_2"><div id="ac_i110" class="anchor-spacer"></div><h3 class="article-section__title" id="_i110"> 2-Bromo-7-(pyridin-3-ylethynyl)-5<i>H</i>-pyrrolo[2,3-<i>b</i>]pyrazine (<b>36</b>)</h3><div class="NLM_p last">To a degassed solution of 5-bromo-3-iodo-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyrazine <b>35</b> (200 mg, 0.617 mmol) in THF (10 mL) were added triethylamine (238 μL, 1.85 mmol), Pd(PPh<sub>3</sub>)<sub>2</sub>Cl<sub>2</sub> (4.33 mg, 1 mol %), and CuI (2.35 mg, 2 mol %). The resulting mixture was stirred under inert atmosphere for 10 min at room temperature. Then, a solution of 3-ethynylpyridine (61 mg, 0.586 mmol) in THF (1 mL) was added to the reaction mixture. The reaction was stirred for 4 h at room temperature. After completion, the reaction was filtered through Celite, extracted with water and ethyl acetate, washed with brine, dried over MgSO<sub>4</sub>, and evaporated in vacuo. The crude residue was purified by silica gel flash column chromatography using a mixture of heptane and ethyl acetate (in a ratio of 6:4) as the mobile phase, yielding the title compound as a white solid (130 mg, 71%). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ 12.94 (br s, 1H, NH), 8.76 (s, 1H), 8.59 (d, <i>J</i> = 3.2 Hz, 1H, ArH), 8.51 (s, 1H, ArH), 8.45 (s, 1H, ArH), 7.99 (dt, <i>J</i> = 7.9, 1.8 Hz, 1H, ArH), 7.47 (dd, <i>J</i> = 7.8, 4.8 Hz, 1H, ArH). HRMS <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>13</sub>H<sub>7</sub>N<sub>4</sub>Br, 298.99273; found, 298.9930.</div></div><div id="sec4_32" class="NLM_sec NLM_sec_level_2"><div id="ac_i111" class="anchor-spacer"></div><h3 class="article-section__title" id="_i111"> 2-(3,4-Dimethoxyphenyl)-7-(pyridin-3-ylethynyl)-5<i>H</i>-pyrrolo[2,3-<i>b</i>]pyrazine (<b>37</b>)</h3><div class="NLM_p last">To a solution of 2-bromo-7-(pyridin-3-ylethynyl)-5<i>H</i>-pyrrolo[2,3-<i>b</i>]pyrazine <b>36</b> (110 mg, 0.368 mmol) in dioxane (4 mL) was added 3,4-dimethoxyphenylboronic acid (80 mg, 0.441 mmol) and 1 mL of a K<sub>2</sub>CO<sub>3</sub> solution (152 mg, 1.1 mmol). The system was purged three times with argon and heated to 105 °C. After stirring for 10 min, Pd(PPh<sub>3</sub>)<sub>4</sub> (10 mol %) was added and the reaction was purged once more with argon. The reaction mixture was stirred at 105 °C for 3 h. After completion, the reaction mixture was cooled to room temperature and filtered through Celite. The filtrate was extracted with water and ethyl acetate. The combined organic layers were washed with brine, dried over MgSO<sub>4</sub>, and evaporated in vacuo. Purification by silica gel flash column chromatography using a mixture of DCM and ethylacetate (in a ratio of 1:1) as the mobile phase yielded the title compound as a white solid (37 mg, 28%). Purity of 98%. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ 12.63 (br s, 1H, NH), 8.96 (s, 1H, ArH), 8.78 (s, 1H, ArH), 8.58 (d, <i>J</i> = 3.3 Hz, 1H, ArH), 8.35 (s, 1H, ArH), 8.00 (d, <i>J</i> = 7.8 Hz, 1H, ArH), 7.76 (m, 2H, ArH), 7.48 (m, 1H, ArH), 7.11 (d, <i>J</i> = 9.0 Hz, 1H, ArH), 3.90 (s, 3H, OCH<sub>3</sub>), 3.83 (s, 3H, OCH<sub>3</sub>) ppm. <sup>13</sup>C NMR (75 MHz, DMSO-<i>d</i><sub>6</sub>): δ 151.44, 149.97, 149.32, 148.57, 146.70, 139.95, 138.27, 137.69, 136.06, 135.59, 130.21, 123.77, 120.46, 119.60, 112.33, 110.46, 95.69, 88.10, 85.92, 55.88, 55.81 ppm. HRMS <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>21</sub>H<sub>16</sub>N<sub>4</sub>O<sub>2</sub>, 357.13459; found, 357.1346.</div></div><div id="sec4_33" class="NLM_sec NLM_sec_level_2"><div id="ac_i112" class="anchor-spacer"></div><h3 class="article-section__title" id="_i112"> Binding-Displacement Assay for NAK Family Selectivity</h3><div class="NLM_p last">Inhibitor binding to NAK family kinase domain proteins was determined using a binding-displacement assay which tests the ability of the inhibitors to displace a fluorescent tracer compound from the ATP binding site of the kinase domain. Inhibitors were dissolved in DMSO and dispensed as 16-point, 2× serial dilutions in duplicate into black multiwell plates (Greiner) using an Echo dispenser (Labcyte Inc). Each well contained 1 nM biotinylated AAK1, BMP2K, GAK or STK16 kinase domain protein ligated to streptavidin-Tb-cryptate (Cisbio), either 12.5 nM (for AAK1 or BMP2K) or 25 nM (for GAK or STK16) Kinase Tracer 236 (Thermo Fisher Scientific), 10 mM 4-(2-hydroxyethyl)-1-piperazineethanesulfonic acid (HEPES) pH 7.5, 150 mM NaCl, 2 mM dithiothreitol, 0.01% bovine serum albumin, 0.01% Tween-20. Final assay volume for each data point was 5 μL, and final DMSO concentration was 1%. The plate was incubated at room temperature for 1.5 h and then read using a TR-FRET protocol on a PheraStarFS plate reader (BMG Labtech). The data were normalized to 0 and 100% inhibition control values and fitted to a four parameter dose–response binding curve in GraphPad Software. For the purpose of estimating the selectivity between each kinase domain, the determined IC<sub>50</sub> values were converted to <i>K</i><sub>i</sub> values using the Cheng–Prusoff equation and the concentration and <i>K</i><sub><span class="smallcaps smallerCapital">D</span></sub> values for the tracer (previously determined).</div></div><div id="sec4_34" class="NLM_sec NLM_sec_level_2"><div id="ac_i113" class="anchor-spacer"></div><h3 class="article-section__title" id="_i113"> AAK1 Expression, Purification, and Crystallization</h3><div class="NLM_p last">AAK1 (UniProtKB: Q2M2I8) residues T27-A365 were cloned, expressed, and purified as described,<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> except that the protein was expressed in a cell line together with lambda phosphatase to produce the unphosphorylated protein. The purified protein was concentrated to 12 mg/mL, and compound <b>1</b> dissolved in 100% DMSO was added to a final concentration of 1.5 mM (3% final DMSO concentration). The protein–ligand solution was incubated on ice for 30 min, then centrifuged at 14 000 rpm for 10 min, 4 °C, immediately prior to setting up sitting-drop vapour diffusion crystallization plates. The best-diffracting crystals of the AAK1–compound <b>1</b> complex were obtained using a reservoir solution containing 26% PEG 3350, 0.1 M bis–Tris pH 5.5 by spiking drops with 20 nL of seed-stock solution immediately prior to incubation at 18 °C. Seed stock was prepared from poorly formed crystals of AAK1-compound <b>1</b> grown during previous rounds of crystal optimization, which were diluted in 50–100 μL reservoir solution and vortexed for 2 min in an Eppendorf containing a seed bead. A 1:1 dilution series of seeds was prepared in order to find the optimal seed concentration. Prior to mounting, crystals were cryo-protected in situ by addition of reservoir solution containing an additional 25% ethylene glycol. Crystals were then flash-frozen in liquid nitrogen.</div></div><div id="sec4_35" class="NLM_sec NLM_sec_level_2"><div id="ac_i114" class="anchor-spacer"></div><h3 class="article-section__title" id="_i114"> Data Collection, Structure Solution, and Refinement</h3><div class="NLM_p last">Data were collected at Diamond beamline I02 using monochromatic radiation at wavelength 0.9795 Å. Diffraction data were processed using XDS<a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a> as part of the xia2 pipeline<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a> and scaled using AIMLESS,<a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a> and molecular replacement was carried out in Phaser<a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a> with PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4WSQ">4WSQ</a> as a search model. Data processing and refinement statistics are given in Table S1 from the <a href="/doi/suppl/10.1021/acs.jmedchem.9b00136/suppl_file/jm9b00136_si_001.pdf" class="ext-link">Supporting Informatio</a>n. REFMAC5,<a onclick="showRef(event, 'ref49'); return false;" href="javascript:void(0);" class="ref ref49">(49)</a> PHENIX,<a onclick="showRef(event, 'ref50'); return false;" href="javascript:void(0);" class="ref ref50">(50)</a> and Coot<a onclick="showRef(event, 'ref51'); return false;" href="javascript:void(0);" class="ref ref51">(51)</a> were used for model building and refinement. Coordinates were submitted to the PDB under accession code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5L4Q">5L4Q</a>.</div></div><div id="sec4_36" class="NLM_sec NLM_sec_level_2"><div id="ac_i115" class="anchor-spacer"></div><h3 class="article-section__title" id="_i115"> Virus Construct</h3><div class="NLM_p last">DENV2 (New Guinea C strain)<a onclick="showRef(event, 'ref52 ref53'); return false;" href="javascript:void(0);" class="ref ref52 ref53">(52,53)</a><i>Renilla</i> reporter plasmid used for in vitro assays was a gift from Pei-Yong Shi (The University of Texas Medical Branch). DENV 16681 plasmid (pD2IC-30P-NBX) used for ex vivo experiments was a gift from Claire Huang (CDC).<a onclick="showRef(event, 'ref54'); return false;" href="javascript:void(0);" class="ref ref54">(54)</a></div></div><div id="sec4_37" class="NLM_sec NLM_sec_level_2"><div id="ac_i116" class="anchor-spacer"></div><h3 class="article-section__title" id="_i116"> Cells</h3><div class="NLM_p last">Huh7 (Apath LLC) cells were grown in Dulbecco’s modified Eagle medium (Mediatech) supplemented with 10% fetal bovine serum (FBS; Omega Scientific), nonessential amino acids, 1% <span class="smallcaps smallerCapital">l</span>-glutamine, and 1% penicillin–streptomycin (Thermo Fisher Scientific) and maintained in a humidified incubator with 5% CO<sub>2</sub> at 37 °C. MDDCs were prepared as described with slight modifications.<a onclick="showRef(event, 'ref55'); return false;" href="javascript:void(0);" class="ref ref55">(55)</a> Buffy coats were obtained from the Stanford Blood Center. CD14+ cells were purified by EasySep Human Monocyte Enrichment Kit without CD16 Depletion (Stemcell Technologies). Cells were seeded in 6-well plates (2 × 10<sup>6</sup> cells per well), stimulated with 500 U/mL granulocyte-macrophage colony-stimulating and 1000 U/mL interleukin-4 (PeproTech), and incubated at 37 °C for 6 days prior to DENV infection [multiplicity of infection (MOI) 1].</div></div><div id="sec4_38" class="NLM_sec NLM_sec_level_2"><div id="ac_i117" class="anchor-spacer"></div><h3 class="article-section__title" id="_i117"> Virus Production</h3><div class="NLM_p last">DENV2 RNA was transcribed in vitro using mMESSAGE/mMACHINE (Ambion) kits. DENV was produced by electroporating RNA into BHK-21 cells, harvesting supernatants on day 10 and titering via standard plaque assays on BHK-21 cells. In parallel, on day 2 post-electroporation, a DENV-containing supernatant was used to inoculate C6/36 cells to amplify the virus. EBOV (Kikwit isolate) was grown in Vero E6 cells, and supernatants were collected and clarified and stored at −80 °C until further use. Virus titers were determined via standard plaque assay on Vero E6 cells.</div></div><div id="sec4_39" class="NLM_sec NLM_sec_level_2"><div id="ac_i118" class="anchor-spacer"></div><h3 class="article-section__title" id="_i118"> Infection Assays</h3><div class="NLM_p last">Huh7 cells were infected with DENV in replicates (<i>n</i> = 5) at a MOI of 0.05. Overall infection was measured at 48 h using a <i>Renilla</i> luciferase substrate. MDDCs were infected with DENV2 (16881) at an MOI of 1. Standard plaque assays were conducted following a 72 h incubation. Huh7 cells were infected with EBOV at an MOI of 1 or 0.1 under biosafety level 4 conditions. Forty-eight hours after infection, supernatants were collected and stored at −80 °C until further use. Cells were formalin-fixed for 24 h prior to removal from biosafety level 4. Infected cells were detected using an EBOV glycoprotein-specific monoclonal antibody (KZ52) and quantitated by automated fluorescence microscopy using an Operetta High Content Imaging System and the Harmony software package (PerkinElmer). For select experiments, supernatants were assayed by standard plaque assay, as described previously. Briefly, supernatants were thawed, serially diluted in growth media, added to VeroE6 cells, and incubated for 1 h at 37 °C in a humidified 5% CO<sub>2</sub> incubator, prior to being overlaid with agarose. Infected cells were incubated for 7 days and then stained with neutral red vital dye (Gibco). Plaques were counted, and titers were calculated.</div></div><div id="sec4_40" class="NLM_sec NLM_sec_level_2"><div id="ac_i119" class="anchor-spacer"></div><h3 class="article-section__title" id="_i119"> Viability Assays</h3><div class="NLM_p last">Viability was assessed using alamarBlue reagent (Invitrogen) or CellTiter-Glo reagent (Promega) assay according to manufacturer’s protocol. Fluorescence was detected at 560 nm on an InfiniteM1000 plate reader and luminescence on an InfiniteM1000 plate reader (Tecan) or a SpectraMax 340PC.</div></div><div id="sec4_41" class="NLM_sec NLM_sec_level_2"><div id="ac_i120" class="anchor-spacer"></div><h3 class="article-section__title" id="_i120"> Effect of Compounds <b>1</b>, <b>8g</b>, and <b>21b</b> on AP-2 Phosphorylation</h3><div class="NLM_p last">Huh7 cells were kept in serum-free medium for 1 h and then treated with the compounds or DMSO in complete medium for 4 h at 37 °C. To allow capturing of phosphorylated AP2M1, 100 nM of the PP2A inhibitor calyculin A (Cell Signalling) was added 30 min prior to lysis in M-PER lysis buffer (Thermo Fisher Scientific) with 1× Halt Protease & phosphatase inhibitor cocktail (Thermo Fisher Scientific). Samples were then subjected to sodium dodecyl sulfate-polyacrylamide gel electrophoresis and blotting with antibodies targeting phospho-AP2M1 (Cell Signaling), total AP2M1 (Santa Cruz Biotechnology), and actin (Sigma-Aldrich). Band intensity was measured with NIH ImageJ.</div></div><div id="sec4_42" class="NLM_sec NLM_sec_level_2"><div id="ac_i121" class="anchor-spacer"></div><h3 class="article-section__title" id="_i121"> AAK1 LanthaScreen Eu binding Assay</h3><div class="NLM_p last">The compounds were subjected to a LanthaScreen binding assay in which 10 titrations of dissolved test compound in DMSO are transferred to a 384-well plate. Sequential addition of the kinase buffer (50 mM HEPES pH 7.5, 0.01% BRIJ-35, 10 mM MgCl and 1 mM EGTA), the 2× kinase antibody (Eu Anti GST) mixture, and the 4× Tracer 222 solution was performed. After shaking for 30 s and a 1 h incubation period at room temperature, the plate was read on a fluorescence plate reader. When the bound tracer in the active site was displaced by the test compound, fluorescence was not observed. The collected data were then compared to a 0% displacement control with pure DMSO and a 100% displacement control with sunitinib, a known inhibitor of AAK1, and plotted against the logarithmic concentration parameter. The IC<sub>50</sub> was subsequently extracted.</div></div><div id="sec4_43" class="NLM_sec NLM_sec_level_2"><div id="ac_i122" class="anchor-spacer"></div><h3 class="article-section__title" id="_i122"> AAK1 <i>K</i><sub><span class="smallcaps smallerCapital">D</span></sub> Assay</h3><div class="NLM_p last"><i>K</i><sub><span class="smallcaps smallerCapital">D</span></sub> values for AAK1 were determined as previously described.<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> Briefly, the DNA-tagged AAK1, an immobilized ligand on streptavidin-coated magnetic beads, and the test compound were combined. When binding occurred between AAK1 and a test compound, no binding can occur between AAK1 and the immobilized ligand. Upon washing, the compound-bound, DNA-tagged AAK1 was washed away. The beads carrying the ligands were then resuspended in elution buffer and the remaining kinase concentration was measured by qPCR on the eluate. <i>K</i><sub><span class="smallcaps smallerCapital">D</span></sub> values were determined using dose–response curves.</div></div><div id="sec4_44" class="NLM_sec NLM_sec_level_2"><div id="ac_i123" class="anchor-spacer"></div><h3 class="article-section__title" id="_i123"> Kinase Selectivity Assay</h3><div class="NLM_p last">Compound <b>21b</b> was screened against a diverse panel of 468 kinases (DiscoverX, KINOMEscan) using an in vitro ATP-site competition binding assay at a concentration of 10 μM. The results are reported as the percentage of kinase/phage remaining bound to the ligands/beads, relative to a control. High affinity compounds have % of control values close to zero, while weaker binders have higher % control values.</div></div><div id="sec4_45" class="NLM_sec NLM_sec_level_2"><div id="ac_i124" class="anchor-spacer"></div><h3 class="article-section__title" id="_i124"> Statistical Analysis</h3><div class="NLM_p last">All data were analyzed with GraphPad Prism software. Fifty percent effective concentration (EC<sub>50,</sub> EC<sub>90</sub> and CC<sub>50</sub>) values were measured by fitting data to a three-parameter logistic curve. <i>P</i> values were calculated by two-way ANOVA with Bonferroni’s multiple comparisons tests or by 2-tailed unpaired <i>t</i>-test.</div></div><div id="sec4_46" class="NLM_sec NLM_sec_level_2"><div id="ac_i125" class="anchor-spacer"></div><h3 class="article-section__title" id="_i125"> Molecular Modeling</h3><div class="NLM_p">Docking was initiated from the AAK1 PDB structure <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5L4Q">5L4Q</a>, from which first all ligands and water molecules were removed. The remaining structure was prepared by AutodockTools<a onclick="showRef(event, 'ref56'); return false;" href="javascript:void(0);" class="ref ref56">(56)</a> as a receptor: polar hydrogens were added, Gasteiger charges were defined, the AD4 atom type was assigned, and finally everything was saved in a pdbqt file. The original inhibitor lkb (identical to compound <b>1</b>) from the PDB file <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5L4Q">5L4Q</a> was extracted, processed by AutodockTools, and saved in the pdbqt format. Compounds <b>8g</b> and <b>21b</b> were drawn in ChemDraw and a 3D structure was generated by Chem3D.<a onclick="showRef(event, 'ref57'); return false;" href="javascript:void(0);" class="ref ref57">(57)</a> The amide conformation in compound <b>8g</b> was initially chosen having an anti-orientation. Autodocktools was used again to prepare the corresponding pdbqt files. Autodock Vina was used for the docking experiments.<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> A cubic box was defined 60 × 60 × 60 units (0.375 Å/unit) and the box was centred at the ALA72:CB atom in the first kinase domain (chain A). The docking process used variable dihedral angles in the ligand while the receptor was defined as rigid. Amide bonds in the ligands were not allowed to rotate.</div><div class="NLM_p">A control docking was executed using the original inhibitor lkb present in the <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5L4Q">5L4Q</a> PDB file. Vina reproduced the original X-ray position very well. For the three docked systems with the best Vina docking score, a MD simulation using the Amber 18 software was performed.<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> Enzyme parameters and charges were taken from the default amber ff14sb force field. Parameters and atomic charges were calculated by antechamber (gaff2). The barrier of the dihedral torsion parameters for the amide bond was increased from 2.6 (from antechamber) to 10.0 (in line with a peptide bond amide in the standard amber force field).</div><div class="NLM_p last">Three molecular systems (AAK1/compound <b>1</b>, AAK1/compound <b>8g</b>, and AAK1/compound <b>21b</b>) were solvated with TIP3P and neutralized in charge. Standard <i>NPT</i> simulations of 40 ns were started (300 K, periodic conditions with PME, cutoff 10.0 Å, shake for H bonds constraints, 2 fs time step).</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i126"><a href="/doi/suppl/10.1021/acs.jmedchem.9b00136" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_52285" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_52285" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a href="/" class="ext-link">ACS Publications website</a> at DOI: <a href="/doi/abs/10.1021/acs.jmedchem.9b00136" class="ext-link">10.1021/acs.jmedchem.9b00136</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Copies of <sup>1</sup>H and <sup>13</sup>C NMR spectra of intermediates and final compounds; details of the kinase selectivity profiling of compound <b>21b</b>; and data collection and refinement statistics for cocrystallography of compound <b>1</b> with AAK1 (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00136/suppl_file/jm9b00136_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00136/suppl_file/jm9b00136_si_002.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00136/suppl_file/jm9b00136_si_001.pdf">jm9b00136_si_001.pdf (5.27 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00136/suppl_file/jm9b00136_si_002.csv">jm9b00136_si_002.csv (4.21 kb)</a></li></ul></div></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">The coordinates have been deposited in the PDB with accession code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5L4Q">5L4Q</a>.</p><div class="testing" data-doi="10.1021/acs.jmedchem.9b00136" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_07275" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_07275" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Shirit Einav</span> - <span class="hlFld-Affiliation affiliation">Department
of Medicine, Division of Infectious Diseases and Geographic Medicine,
and Department of Microbiology and Immunology, Stanford University School of Medicine, Stanford, California 94305, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-6441-4171" title="Orcid link">http://orcid.org/0000-0001-6441-4171</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#582b3d3136392e182b2c39363e372a3c763d3c2d"><span class="__cf_email__" data-cfemail="d5a6b0bcbbb4a395a6a1b4bbb3baa7b1fbb0b1a0">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Steven De Jonghe</span> - <span class="hlFld-Affiliation affiliation">Medicinal
Chemistry, Rega Institute for Medical Research, KU Leuven, Herestraat
49—bus 1041, 3000 Leuven, Belgium</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-3872-6558" title="Orcid link">http://orcid.org/0000-0002-3872-6558</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#93e0e7f6e5f6fdbdf7f6f9fcfdf4fbf6d3f8e6fff6e6e5f6fdbdf1f6"><span class="__cf_email__" data-cfemail="c9babdacbfaca7e7adaca3a6a7aea1ac89a2bca5acbcbfaca7e7abac">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Sven Verdonck</span> - <span class="hlFld-Affiliation affiliation">Medicinal
Chemistry, Rega Institute for Medical Research, KU Leuven, Herestraat
49—bus 1041, 3000 Leuven, Belgium</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Szu-Yuan Pu</span> - <span class="hlFld-Affiliation affiliation">Department
of Medicine, Division of Infectious Diseases and Geographic Medicine,
and Department of Microbiology and Immunology, Stanford University School of Medicine, Stanford, California 94305, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Fiona J. Sorrell</span> - <span class="hlFld-Affiliation affiliation">Nuffield
Department of Clinical Medicine, Target Discovery Institute (TDI)
and Structural Genomics Consortium (SGC), University of Oxford, Old Road Campus, Roosevelt Drive, Oxford OX3 7DQ, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jon M. Elkins</span> - <span class="hlFld-Affiliation affiliation">Nuffield
Department of Clinical Medicine, Target Discovery Institute (TDI)
and Structural Genomics Consortium (SGC), University of Oxford, Old Road Campus, Roosevelt Drive, Oxford OX3 7DQ, U.K.</span>; 
    <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium, Universidade Estadual
de Campinas, Cidade Universitária
Zeferino Vaz, Av. Dr. André Tosello, 550,
Barão Geraldo, Campinas, São
Paulo 13083-886, Brazil</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Mathy Froeyen</span> - <span class="hlFld-Affiliation affiliation">Medicinal
Chemistry, Rega Institute for Medical Research, KU Leuven, Herestraat
49—bus 1041, 3000 Leuven, Belgium</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-8675-6540" title="Orcid link">http://orcid.org/0000-0002-8675-6540</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ling-Jie Gao</span> - <span class="hlFld-Affiliation affiliation">Medicinal
Chemistry, Rega Institute for Medical Research, KU Leuven, Herestraat
49—bus 1041, 3000 Leuven, Belgium</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Laura I. Prugar</span> - <span class="hlFld-Affiliation affiliation">US Army
Medical Research Institute of Infectious Diseases, Viral Immunology Branch, Fort Detrick, Maryland 21702, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Danielle E. Dorosky</span> - <span class="hlFld-Affiliation affiliation">US Army
Medical Research Institute of Infectious Diseases, Viral Immunology Branch, Fort Detrick, Maryland 21702, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jennifer M. Brannan</span> - <span class="hlFld-Affiliation affiliation">US Army
Medical Research Institute of Infectious Diseases, Viral Immunology Branch, Fort Detrick, Maryland 21702, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Rina Barouch-Bentov</span> - <span class="hlFld-Affiliation affiliation">Department
of Medicine, Division of Infectious Diseases and Geographic Medicine,
and Department of Microbiology and Immunology, Stanford University School of Medicine, Stanford, California 94305, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Stefan Knapp</span> - <span class="hlFld-Affiliation affiliation">Nuffield
Department of Clinical Medicine, Target Discovery Institute (TDI)
and Structural Genomics Consortium (SGC), University of Oxford, Old Road Campus, Roosevelt Drive, Oxford OX3 7DQ, U.K.</span>; 
    <span class="hlFld-Affiliation affiliation">Institute
for Pharmaceutical Chemistry, Buchmann Institute for Life Sciences
Campus Riedbeerg, Goethe-University Frankfurt, 60438 Frankfurt
am Main, Germany</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-5995-6494" title="Orcid link">http://orcid.org/0000-0001-5995-6494</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">John M. Dye</span> - <span class="hlFld-Affiliation affiliation">US Army
Medical Research Institute of Infectious Diseases, Viral Immunology Branch, Fort Detrick, Maryland 21702, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Piet Herdewijn</span> - <span class="hlFld-Affiliation affiliation">Medicinal
Chemistry, Rega Institute for Medical Research, KU Leuven, Herestraat
49—bus 1041, 3000 Leuven, Belgium</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-3589-8503" title="Orcid link">http://orcid.org/0000-0003-3589-8503</a></span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>S.V. and S-Y.P. contributed equally to this work. S.E. and S.D.J. also contributed equally to this work.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br><p class="inlineNote">Authors will release the atomic coordinates and experimental data upon article publication.</p></div></li></ul></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i130">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_85146" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_85146" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">This work was supported by award number W81XWH-16-1–0691 from the Department of Defense (DoD), Congressionally Directed Medical Research Programs (CDMRP) to S.E., P.H.; Grant 12393481 from the Defense Threat Reduction Agency (DTRA), Fundamental Research to Counter Weapons of Mass Destruction to S.E., P.H.; and seed grant from the Stanford SPARK program. S.V. is the recipient of a doctoral fellowship from the Research Foundation—Flanders (1S00116N). S.P. was supported by the Child Health Research Institute, Lucile Packard Foundation for Children’s Health and the Stanford CSTA (grant number UL1 TR000093). The SGC is a registered charity (number 1097737) that receives funds from AbbVie, Bayer Pharma AG, Boehringer Ingelheim, Canada Foundation for Innovation, Eshelman Institute for Innovation, Genome Canada, Innovative Medicines Initiative (EU/EFPIA) [ULTRA-DD grant no. 115766], Janssen, Merck KGaA Darmstadt Germany, MSD, Novartis Pharma AG, Ontario Ministry of Economic Development and Innovation, Pfizer, São Paulo Research Foundation-FAPESP, Takeda, and Wellcome [106169/ZZ14/Z].</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations</td></tr><tr><td class="NLM_term">AAK1</td><td class="NLM_def"><p class="first last">adaptor-associated kinase 1</p></td></tr><tr><td class="NLM_term">AP</td><td class="NLM_def"><p class="first last">adaptor protein</p></td></tr><tr><td class="NLM_term">BOP</td><td class="NLM_def"><p class="first last">(benzotriazol-1-yloxy)tris(dimethylamino)phosphonium hexafluorophosphate</p></td></tr><tr><td class="NLM_term">CC<sub>50</sub></td><td class="NLM_def"><p class="first last">half-maximal cytotoxic concentration</p></td></tr><tr><td class="NLM_term">CCV</td><td class="NLM_def"><p class="first last">clathrin-coated vesicle</p></td></tr><tr><td class="NLM_term">DENV</td><td class="NLM_def"><p class="first last">Dengue virus</p></td></tr><tr><td class="NLM_term">EBOV</td><td class="NLM_def"><p class="first last">Ebola virus</p></td></tr><tr><td class="NLM_term">EC<sub>50</sub></td><td class="NLM_def"><p class="first last">half-maximal effective concentration</p></td></tr><tr><td class="NLM_term">EC<sub>90</sub></td><td class="NLM_def"><p class="first last">90% effective concentration</p></td></tr><tr><td class="NLM_term">EGFR</td><td class="NLM_def"><p class="first last">epidermal growth factor receptor</p></td></tr><tr><td class="NLM_term">GAK</td><td class="NLM_def"><p class="first last">cyclin G-associated kinase</p></td></tr><tr><td class="NLM_term">HCV</td><td class="NLM_def"><p class="first last">hepatitis C virus</p></td></tr><tr><td class="NLM_term"><i>K</i><sub><span class="smallcaps smallerCapital">D</span></sub></td><td class="NLM_def"><p class="first last">dissociation constant</p></td></tr><tr><td class="NLM_term">MM/GBSA</td><td class="NLM_def"><p class="first last">molecular mechanics with a generalized Born surface area continuum solvation model</p></td></tr><tr><td class="NLM_term">MM/PBSA</td><td class="NLM_def"><p class="first last">molecular mechanics energies combined with the Poisson–Boltzmann surface area continuum solvation model</p></td></tr><tr><td class="NLM_term">NAK</td><td class="NLM_def"><p class="first last">numb-associated kinase</p></td></tr><tr><td class="NLM_term">PDB</td><td class="NLM_def"><p class="first last">protein data bank</p></td></tr><tr><td class="NLM_term">rmsd</td><td class="NLM_def"><p class="first last">root mean square deviation</p></td></tr><tr><td class="NLM_term">TGN</td><td class="NLM_def"><p class="first last"><i>trans</i>-Golgi network</p></td></tr><tr><td class="NLM_term">WHO</td><td class="NLM_def"><p class="first last">World Health Organization</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i132">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_44442" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_44442" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 57 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Murray, N. A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quam, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilder-Smith, A.</span></span> <span> </span><span class="NLM_article-title">Epidemiology of Dengue: Past, Present and Future Prospects</span>. <i>Clin. Epidemiol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">299</span>– <span class="NLM_lpage">309</span>, <span class="refDoi"> DOI: 10.2147/clep.s34440</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=10.2147%2Fclep.s34440" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=23990732" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=1%3ACAS%3A280%3ADC%252BC3sbjs12msg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2013&pages=299-309&author=N.+A.+E.+Murrayauthor=M.+B.+Quamauthor=A.+Wilder-Smith&title=Epidemiology+of+Dengue%3A+Past%2C+Present+and+Future+Prospects&doi=10.2147%2Fclep.s34440"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Epidemiology of dengue: past, present and future prospects</span></div><div class="casAuthors">Murray Natasha Evelyn Anne; Quam Mikkel B; Wilder-Smith Annelies</div><div class="citationInfo"><span class="NLM_cas:title">Clinical epidemiology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">299-309</span>
        ISSN:<span class="NLM_cas:issn">1179-1349</span>.
    </div><div class="casAbstract">Dengue is currently regarded globally as the most important mosquito-borne viral disease.  A history of symptoms compatible with dengue can be traced back to the Chin Dynasty of 265-420 AD.  The virus and its vectors have now become widely distributed throughout tropical and subtropical regions of the world, particularly over the last half-century.  Significant geographic expansion has been coupled with rapid increases in incident cases, epidemics, and hyperendemicity, leading to the more severe forms of dengue.  Transmission of dengue is now present in every World Health Organization (WHO) region of the world and more than 125 countries are known to be dengue endemic.  The true impact of dengue globally is difficult to ascertain due to factors such as inadequate disease surveillance, misdiagnosis, and low levels of reporting.  Currently available data likely grossly underestimates the social, economic, and disease burden.  Estimates of the global incidence of dengue infections per year have ranged between 50 million and 200 million; however, recent estimates using cartographic approaches suggest this number is closer to almost 400 million.  The expansion of dengue is expected to increase due to factors such as the modern dynamics of climate change, globalization, travel, trade, socioeconomics, settlement and also viral evolution.  No vaccine or specific antiviral therapy currently exists to address the growing threat of dengue.  Prompt case detection and appropriate clinical management can reduce the mortality from severe dengue.  Effective vector control is the mainstay of dengue prevention and control.  Surveillance and improved reporting of dengue cases is also essential to gauge the true global situation as indicated in the objectives of the WHO Global Strategy for Dengue Prevention and Control, 2012-2020.  More accurate data will inform the prioritization of research, health policy, and financial resources toward reducing this poorly controlled disease.  The objective of this paper is to review historical and current epidemiology of dengue worldwide and, additionally, reflect on some potential reasons for expansion of dengue into the future.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTTkqBcd5TiZSM8MEzWGjS9fW6udTcc2ea_BnvxCDi4KLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3sbjs12msg%253D%253D&md5=89d2c693ec4bda3679bc24609316e4b5</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.2147%2Fclep.s34440&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2147%252Fclep.s34440%26sid%3Dliteratum%253Aachs%26aulast%3DMurray%26aufirst%3DN.%2BA.%2BE.%26aulast%3DQuam%26aufirst%3DM.%2BB.%26aulast%3DWilder-Smith%26aufirst%3DA.%26atitle%3DEpidemiology%2520of%2520Dengue%253A%2520Past%252C%2520Present%2520and%2520Future%2520Prospects%26jtitle%3DClin.%2520Epidemiol.%26date%3D2013%26volume%3D5%26spage%3D299%26epage%3D309%26doi%3D10.2147%2Fclep.s34440" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group">WHO</span>. <span> </span><span class="NLM_article-title">Epidemiology</span>. <a href="http://www.who.int/denguecontrol/epidemiology/en/" class="extLink">http://www.who.int/denguecontrol/epidemiology/en/</a> (accessed Nov 15, <span class="NLM_year">2018</span>).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2018&author=WHO&title=Epidemiology"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26aulast%3D%26atitle%3DEpidemiology%26date%3D2018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span> <i>Weekly Epidemiological
Record</i>; <span class="NLM_publisher-name">World Health
Organization</span>, <span class="NLM_year">2016</span>; Vol.  <span class="NLM_volume">91</span> (<span class="NLM_issue">30</span>), pp  <span class="NLM_fpage">349</span>– <span class="NLM_lpage">364</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+Weekly+Epidemiological%0ARecord%3B+World+Health%0AOrganization%2C+2016%3B+Vol.+91+%2830%29%2C+pp+349%E2%80%93364."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dreport%26sid%3Dliteratum%253Aachs%26jtitle%3DWeekly%2520Epidemiological%250ARecord%26pub%3DWorld%2520Health%250AOrganization%26date%3D2016%26volume%3D91%26issue%3D30%26spage%3D349%26epage%3D364" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sanyal, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sinha, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Halder, K. K.</span></span> <span> </span><span class="NLM_article-title">Pathogenesis of Dengue Haemorrhagic Fever</span>. <i>J. Indian Med. Assoc.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>89</i></span>,  <span class="NLM_fpage">152</span>– <span class="NLM_lpage">153</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=89&publication_year=2013&pages=152-153&author=S.+Sanyalauthor=S.+Sinhaauthor=K.+K.+Halder&title=Pathogenesis+of+Dengue+Haemorrhagic+Fever"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSanyal%26aufirst%3DS.%26aulast%3DSinha%26aufirst%3DS.%26aulast%3DHalder%26aufirst%3DK.%2BK.%26atitle%3DPathogenesis%2520of%2520Dengue%2520Haemorrhagic%2520Fever%26jtitle%3DJ.%2520Indian%2520Med.%2520Assoc.%26date%3D2013%26volume%3D89%26spage%3D152%26epage%3D153" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group">WHO</span>. <span> </span><span class="NLM_article-title">Ebola virus disease</span> <a href="http://www.who.int/news-room/fact-sheets/detail/ebola-virus-disease" class="extLink">http://www.who.int/news-room/fact-sheets/detail/ebola-virus-disease</a> (accessed Aug 7, <span class="NLM_year">2018</span>).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2018&author=WHO&title=Ebola+virus+disease"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26aulast%3D%26atitle%3DEbola%2520virus%2520disease%26date%3D2018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Behnam, M. A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nitsche, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boldescu, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klein, C. D.</span></span> <span> </span><span class="NLM_article-title">The Medicinal Chemistry of Dengue Virus</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">5622</span>– <span class="NLM_lpage">5649</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01653</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01653" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=1%3ACAS%3A528%3ADC%252BC28XptlGguw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=5622-5649&author=M.+A.+M.+Behnamauthor=C.+Nitscheauthor=V.+Boldescuauthor=C.+D.+Klein&title=The+Medicinal+Chemistry+of+Dengue+Virus&doi=10.1021%2Facs.jmedchem.5b01653"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">The Medicinal Chemistry of Dengue Virus</span></div><div class="casAuthors">Behnam, Mira A. M.; Nitsche, Christoph; Boldescu, Veaceslav; Klein, Christian D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">5622-5649</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The dengue virus and related flaviviruses are an increasing global health threat.  In this perspective, we comment on and review medicinal chem. efforts aimed at the prevention or treatment of dengue infections.  We include target-based approaches aimed at viral or host factors and results from phenotypic screenings in cellular assay systems for viral replication.  This perspective is limited to the discussion of results that provide explicit chem. or structure-activity relationship (SAR), or appear to be of particular interest to the medicinal chemist for other reasons.  The discovery and development efforts discussed here may at least partially be extrapolated toward other emerging flaviviral infections, such as West Nile virus.  Therefore, this perspective, although not aimed at flaviviruses in general, should also be able to provide an overview of the medicinal chem. of these closely related infectious agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr7LeRM_49L7bVg90H21EOLACvtfcHk0ljfCk4PQwQTvg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XptlGguw%253D%253D&md5=a3de42f04cd59a4396a0b020b53c66ee</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01653&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01653%26sid%3Dliteratum%253Aachs%26aulast%3DBehnam%26aufirst%3DM.%2BA.%2BM.%26aulast%3DNitsche%26aufirst%3DC.%26aulast%3DBoldescu%26aufirst%3DV.%26aulast%3DKlein%26aufirst%3DC.%2BD.%26atitle%3DThe%2520Medicinal%2520Chemistry%2520of%2520Dengue%2520Virus%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D5622%26epage%3D5649%26doi%3D10.1021%2Facs.jmedchem.5b01653" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bekerman, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neveu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shulla, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brannan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pu, S.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barouch-Bentov, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bakken, R. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mateo, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Govero, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nagamine, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diamond, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Jonghe, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herdewijn, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dye, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Randall, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Einav, S.</span></span> <span> </span><span class="NLM_article-title">Anticancer Kinase Inhibitors Impair Intracellular Viral Trafficking and Exert Broad-Spectrum Antiviral Effects</span>. <i>J. Clin. Invest.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>127</i></span>,  <span class="NLM_fpage">1338</span>– <span class="NLM_lpage">1352</span>, <span class="refDoi"> DOI: 10.1172/jci89857</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=10.1172%2Fjci89857" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=28240606" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=1%3ACAS%3A280%3ADC%252BC1czgvFajtg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=127&publication_year=2017&pages=1338-1352&author=E.+Bekermanauthor=G.+Neveuauthor=A.+Shullaauthor=J.+Brannanauthor=S.-Y.+Puauthor=S.+Wangauthor=F.+Xiaoauthor=R.+Barouch-Bentovauthor=R.+R.+Bakkenauthor=R.+Mateoauthor=J.+Goveroauthor=C.+M.+Nagamineauthor=M.+S.+Diamondauthor=S.+De+Jongheauthor=P.+Herdewijnauthor=J.+M.+Dyeauthor=G.+Randallauthor=S.+Einav&title=Anticancer+Kinase+Inhibitors+Impair+Intracellular+Viral+Trafficking+and+Exert+Broad-Spectrum+Antiviral+Effects&doi=10.1172%2Fjci89857"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Anticancer kinase inhibitors impair intracellular viral trafficking and exert broad-spectrum antiviral effects</span></div><div class="casAuthors">Bekerman Elena; Neveu Gregory; Shulla Ana; Brannan Jennifer; Pu Szu-Yuan; Wang Stanley; Xiao Fei; Barouch-Bentov Rina; Bakken Russell R; Mateo Roberto; Govero Jennifer; Nagamine Claude M; Diamond Michael S; De Jonghe Steven; Herdewijn Piet; Dye John M; Randall Glenn; Einav Shirit</div><div class="citationInfo"><span class="NLM_cas:title">The Journal of clinical investigation</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">127</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1338-1352</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Global health is threatened by emerging viral infections, which largely lack effective vaccines or therapies.  Targeting host pathways that are exploited by multiple viruses could offer broad-spectrum solutions.  We previously reported that AAK1 and GAK, kinase regulators of the host adaptor proteins AP1 and AP2, are essential for hepatitis C virus (HCV) infection, but the underlying mechanism and relevance to other viruses or in vivo infections remained unknown.  Here, we have discovered that AP1 and AP2 cotraffic with HCV particles in live cells.  Moreover, we found that multiple viruses, including dengue and Ebola, exploit AAK1 and GAK during entry and infectious virus production.  In cultured cells, treatment with sunitinib and erlotinib, approved anticancer drugs that inhibit AAK1 or GAK activity, or with more selective compounds inhibited intracellular trafficking of HCV and multiple unrelated RNA viruses with a high barrier to resistance.  In murine models of dengue and Ebola infection, sunitinib/erlotinib combination protected against morbidity and mortality.  We validated sunitinib- and erlotinib-mediated inhibition of AAK1 and GAK activity as an important mechanism of antiviral action.  Additionally, we revealed potential roles for additional kinase targets.  These findings advance our understanding of virus-host interactions and establish a proof of principle for a repurposed, host-targeted approach to combat emerging viruses.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSeR1dYk1i_czQLBVDQn4bCfW6udTcc2eYdKs-vCwS_nbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1czgvFajtg%253D%253D&md5=2de0a5d7668651f92be3c2e79823ced4</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1172%2Fjci89857&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1172%252Fjci89857%26sid%3Dliteratum%253Aachs%26aulast%3DBekerman%26aufirst%3DE.%26aulast%3DNeveu%26aufirst%3DG.%26aulast%3DShulla%26aufirst%3DA.%26aulast%3DBrannan%26aufirst%3DJ.%26aulast%3DPu%26aufirst%3DS.-Y.%26aulast%3DWang%26aufirst%3DS.%26aulast%3DXiao%26aufirst%3DF.%26aulast%3DBarouch-Bentov%26aufirst%3DR.%26aulast%3DBakken%26aufirst%3DR.%2BR.%26aulast%3DMateo%26aufirst%3DR.%26aulast%3DGovero%26aufirst%3DJ.%26aulast%3DNagamine%26aufirst%3DC.%2BM.%26aulast%3DDiamond%26aufirst%3DM.%2BS.%26aulast%3DDe%2BJonghe%26aufirst%3DS.%26aulast%3DHerdewijn%26aufirst%3DP.%26aulast%3DDye%26aufirst%3DJ.%2BM.%26aulast%3DRandall%26aufirst%3DG.%26aulast%3DEinav%26aufirst%3DS.%26atitle%3DAnticancer%2520Kinase%2520Inhibitors%2520Impair%2520Intracellular%2520Viral%2520Trafficking%2520and%2520Exert%2520Broad-Spectrum%2520Antiviral%2520Effects%26jtitle%3DJ.%2520Clin.%2520Invest.%26date%3D2017%26volume%3D127%26spage%3D1338%26epage%3D1352%26doi%3D10.1172%2Fjci89857" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bekerman, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Einav, S.</span></span> <span> </span><span class="NLM_article-title">Combating Emerging Viral Threats</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>348</i></span>,  <span class="NLM_fpage">282</span>– <span class="NLM_lpage">283</span>, <span class="refDoi"> DOI: 10.1126/science.aaa3778</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=10.1126%2Fscience.aaa3778" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=25883340" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=1%3ACAS%3A528%3ADC%252BC2MXosVKjt7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=348&publication_year=2015&pages=282-283&author=E.+Bekermanauthor=S.+Einav&title=Combating+Emerging+Viral+Threats&doi=10.1126%2Fscience.aaa3778"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Combating emerging viral threats</span></div><div class="casAuthors">Bekerman, Elena; Einav, Shirit</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">348</span>
        (<span class="NLM_cas:issue">6232</span>),
    <span class="NLM_cas:pages">282-283</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGriDTBlEp8j0LVg90H21EOLACvtfcHk0ljYTe-Id0ZkFg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXosVKjt7Y%253D&md5=1cf54cc0a7db0925573f96ed72c27c11</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1126%2Fscience.aaa3778&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.aaa3778%26sid%3Dliteratum%253Aachs%26aulast%3DBekerman%26aufirst%3DE.%26aulast%3DEinav%26aufirst%3DS.%26atitle%3DCombating%2520Emerging%2520Viral%2520Threats%26jtitle%3DScience%26date%3D2015%26volume%3D348%26spage%3D282%26epage%3D283%26doi%3D10.1126%2Fscience.aaa3778" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Grove, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marsh, M.</span></span> <span> </span><span class="NLM_article-title">The Cell Biology of Receptor-Mediated Virus Entry</span>. <i>J. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>195</i></span>,  <span class="NLM_fpage">1071</span>– <span class="NLM_lpage">1082</span>, <span class="refDoi"> DOI: 10.1083/jcb.201108131</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=10.1083%2Fjcb.201108131" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=22123832" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=1%3ACAS%3A528%3ADC%252BC38XktlGksQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=195&publication_year=2011&pages=1071-1082&author=J.+Groveauthor=M.+Marsh&title=The+Cell+Biology+of+Receptor-Mediated+Virus+Entry&doi=10.1083%2Fjcb.201108131"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">The cell biology of receptor-mediated virus entry</span></div><div class="casAuthors">Grove, Joe; Marsh, Mark</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Cell Biology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">195</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1071-1082</span>CODEN:
                <span class="NLM_cas:coden">JCLBA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9525</span>.
    
            (<span class="NLM_cas:orgname">Rockefeller University Press</span>)
        </div><div class="casAbstract">A review.  The cell imposes multiple barriers to virus entry.  However, viruses exploit fundamental cellular processes to gain entry to cells and deliver their genetic cargo.  Virus entry pathways are largely defined by the interactions between virus particles and their receptors at the cell surface.  These interactions det. the mechanisms of virus attachment, uptake, intracellular trafficking, and, ultimately, penetration to the cytosol.  Elucidating the complex interplay between viruses and their receptors is necessary for a full understanding of how these remarkable agents invade their cellular hosts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqJz6iwbcbos7Vg90H21EOLACvtfcHk0ljYTe-Id0ZkFg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XktlGksQ%253D%253D&md5=cb94e5f7845d2efbff2d9f08ed047938</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1083%2Fjcb.201108131&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1083%252Fjcb.201108131%26sid%3Dliteratum%253Aachs%26aulast%3DGrove%26aufirst%3DJ.%26aulast%3DMarsh%26aufirst%3DM.%26atitle%3DThe%2520Cell%2520Biology%2520of%2520Receptor-Mediated%2520Virus%2520Entry%26jtitle%3DJ.%2520Cell%2520Biol.%26date%3D2011%26volume%3D195%26spage%3D1071%26epage%3D1082%26doi%3D10.1083%2Fjcb.201108131" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Neveu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ziv-Av, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barouch-Bentov, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berkerman, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mulholland, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Einav, S.</span></span> <span> </span><span class="NLM_article-title">AP-2-Associated Protein Kinase 1 and Cyclin G-Associated Kinase Regulate Hepatitis C Virus Entry and Are Potential Drug Targets</span>. <i>J. Virol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>89</i></span>,  <span class="NLM_fpage">4387</span>– <span class="NLM_lpage">4404</span>, <span class="refDoi"> DOI: 10.1128/jvi.02705-14</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=10.1128%2Fjvi.02705-14" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=25653444" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=1%3ACAS%3A528%3ADC%252BC2MXlvVWlurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=89&publication_year=2015&pages=4387-4404&author=G.+Neveuauthor=A.+Ziv-Avauthor=R.+Barouch-Bentovauthor=E.+Berkermanauthor=J.+Mulhollandauthor=S.+Einav&title=AP-2-Associated+Protein+Kinase+1+and+Cyclin+G-Associated+Kinase+Regulate+Hepatitis+C+Virus+Entry+and+Are+Potential+Drug+Targets&doi=10.1128%2Fjvi.02705-14"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">AP-2-associated protein kinase 1 and cyclin G-associated kinase regulate hepatitis C virus entry and are potential drug targets</span></div><div class="casAuthors">Neveu, Gregory; Ziv-Av, Amotz; Barouch-Bentov, Rina; Berkerman, Elena; Mulholland, Jon; Einav, Shirit</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Virology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">89</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">4387-4404</span>CODEN:
                <span class="NLM_cas:coden">JOVIAM</span>;
        ISSN:<span class="NLM_cas:issn">1098-5514</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Hepatitis C virus (HCV) enters its target cell via clathrin-mediated endocytosis.  AP-2-assocd. protein kinase 1 (AAK1) and cyclin G-assocd. kinase (GAK) are host kinases that regulate clathrin adaptor protein (AP)-mediated trafficking in the endocytic and secretory pathways.  We previously reported that AAK1 and GAK regulate HCV assembly by stimulating binding of the μ subunit of AP-2, AP2M1, to HCV core protein.  We also discovered that AAK1 and GAK inhibitors, including the approved anticancer drugs sunitinib and erlotinib, could block HCV assembly.  Here, we hypothesized that AAK1 and GAK regulate HCV entry independently of their effect on HCV assembly.  Indeed, silencing AAK1 and GAK expression inhibited entry of pseudoparticles and cell culture grown-HCV and internalization of Dil-labeled HCV particles with no effect on HCV attachment or RNA replication.  AAK1 or GAK depletion impaired epidermal growth factor (EGF)-mediated enhanced HCV entry and endocytosis of EGF receptor (EGFR), an HCV entry cofactor and erlotinib's cancer target.  Moreover, either RNA interference-mediated depletion of AP2M1 or NUMB, each a substrate of AAK1 and/or GAK, or overexpression of either an AP2M1 or NUMB phosphorylation site mutant inhibited HCV entry.  Last, in addn. to affecting assembly, sunitinib and erlotinib inhibited HCV entry at a postbinding step, their combination was synergistic, and their antiviral effect was reversed by either AAK1 or GAK overexpression.  Together, these results validate AAK1 and GAK as crit. regulators of HCV entry that function in part by activating EGFR, AP2M1, and NUMB and as the mol. targets underlying the antiviral effect of sunitinib and erlotinib (in addn. to EGFR), resp.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoB1x5sJzyhrLVg90H21EOLACvtfcHk0lg4lP2rcUR4Rw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXlvVWlurg%253D&md5=cbdd3e12d29f383c73dc53c0e9954300</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1128%2Fjvi.02705-14&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252Fjvi.02705-14%26sid%3Dliteratum%253Aachs%26aulast%3DNeveu%26aufirst%3DG.%26aulast%3DZiv-Av%26aufirst%3DA.%26aulast%3DBarouch-Bentov%26aufirst%3DR.%26aulast%3DBerkerman%26aufirst%3DE.%26aulast%3DMulholland%26aufirst%3DJ.%26aulast%3DEinav%26aufirst%3DS.%26atitle%3DAP-2-Associated%2520Protein%2520Kinase%25201%2520and%2520Cyclin%2520G-Associated%2520Kinase%2520Regulate%2520Hepatitis%2520C%2520Virus%2520Entry%2520and%2520Are%2520Potential%2520Drug%2520Targets%26jtitle%3DJ.%2520Virol.%26date%3D2015%26volume%3D89%26spage%3D4387%26epage%3D4404%26doi%3D10.1128%2Fjvi.02705-14" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Neveu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barouch-Bentov, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ziv-Av, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerber, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacob, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Einav, S.</span></span> <span> </span><span class="NLM_article-title">Identification and Targeting of an Interaction between a Tyrosine Motif within Hepatitis C Virus Core Protein and AP2M1 Essential for Viral Assembly</span>. <i>PLoS Pathog.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>8</i></span>, <span class="NLM_elocation-id">e1002845</span> <span class="refDoi"> DOI: 10.1371/journal.ppat.1002845</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=10.1371%2Fjournal.ppat.1002845" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=22916011" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=1%3ACAS%3A528%3ADC%252BC38Xht1Cns7bF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2012&author=G.+Neveuauthor=R.+Barouch-Bentovauthor=A.+Ziv-Avauthor=D.+Gerberauthor=Y.+Jacobauthor=S.+Einav&title=Identification+and+Targeting+of+an+Interaction+between+a+Tyrosine+Motif+within+Hepatitis+C+Virus+Core+Protein+and+AP2M1+Essential+for+Viral+Assembly&doi=10.1371%2Fjournal.ppat.1002845"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Identification and targeting of an interaction between a tyrosine motif within hepatitis C virus core protein and AP2M1 essential for viral assembly</span></div><div class="casAuthors">Neveu, Gregory; Barouch-Bentov, Rina; Ziv-Av, Amotz; Gerber, Doron; Jacob, Yves; Einav, Shirit</div><div class="citationInfo"><span class="NLM_cas:title">PLoS Pathogens</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">e1002845</span>CODEN:
                <span class="NLM_cas:coden">PPLACN</span>;
        ISSN:<span class="NLM_cas:issn">1553-7374</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Novel therapies are urgently needed against hepatitis C virus infection (HCV), a major global health problem.  The current model of infectious virus prodn. suggests that HCV virions are assembled on or near the surface of lipid droplets, acquire their envelope at the ER, and egress through the secretory pathway.  The mechanisms of HCV assembly and particularly the role of viral-host protein-protein interactions in mediating this process are, however, poorly understood.  We identified a conserved heretofore unrecognized YXXΦ motif (Φ is a bulky hydrophobic residue) within the core protein.  This motif is homologous to sorting signals within host cargo proteins known to mediate binding of AP2M1, the μ subunit of clathrin adaptor protein complex 2 (AP-2), and intracellular trafficking.  Using microfluidics affinity anal., protein-fragment complementation assays, and co-immunopptns. in infected cells, we show that this motif mediates core binding to AP2M1.  YXXΦ mutations, silencing AP2M1 expression or overexpressing a dominant neg. AP2M1 mutant had no effect on HCV RNA replication, however, they dramatically inhibited intra- and extra-cellular infectivity, consistent with a defect in viral assembly.  Quant. confocal immunofluorescence anal. revealed that core's YXXΦ motif mediates recruitment of AP2M1 to lipid droplets and that the obsd. defect in HCV assembly following disruption of core-AP2M1 binding correlates with accumulation of core on lipid droplets, reduced core colocalization with E2 and reduced core localization to trans-Golgi network (TGN), the presumed site of viral particles maturation.  Furthermore, AAK1 and GAK, serine/threonine kinases known to stimulate binding of AP2M1 to host cargo proteins, regulate core-AP2M1 binding and are essential for HCV assembly.  Last, approved anti-cancer drugs that inhibit AAK1 or GAK not only disrupt core-AP2M1 binding, but also significantly inhibit HCV assembly and infectious virus prodn.  These results validate viral-host interactions essential for HCV assembly and yield compds. for pharmaceutical development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpjKaGJVLP2vLVg90H21EOLACvtfcHk0lg4lP2rcUR4Rw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xht1Cns7bF&md5=811c0818d4b2500b1b788076ac4abc5d</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1371%2Fjournal.ppat.1002845&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.ppat.1002845%26sid%3Dliteratum%253Aachs%26aulast%3DNeveu%26aufirst%3DG.%26aulast%3DBarouch-Bentov%26aufirst%3DR.%26aulast%3DZiv-Av%26aufirst%3DA.%26aulast%3DGerber%26aufirst%3DD.%26aulast%3DJacob%26aufirst%3DY.%26aulast%3DEinav%26aufirst%3DS.%26atitle%3DIdentification%2520and%2520Targeting%2520of%2520an%2520Interaction%2520between%2520a%2520Tyrosine%2520Motif%2520within%2520Hepatitis%2520C%2520Virus%2520Core%2520Protein%2520and%2520AP2M1%2520Essential%2520for%2520Viral%2520Assembly%26jtitle%3DPLoS%2520Pathog.%26date%3D2012%26volume%3D8%26doi%3D10.1371%2Fjournal.ppat.1002845" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barouch-Bentov, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neveu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schor, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumar, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nicolaescu, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindenbach, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Randall, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Einav, S.</span></span> <span> </span><span class="NLM_article-title">Interactions between the Hepatitis C Virus Nonstructural 2 Protein and Host Adaptor Proteins 1 and 4 Orchestrate Virus Release</span>. <i>mBio</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">21</span>, <span class="refDoi"> DOI: 10.1128/mbio.02233-17</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=10.1128%2Fmbio.02233-17" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2018&pages=1-21&author=F.+Xiaoauthor=S.+Wangauthor=R.+Barouch-Bentovauthor=G.+Neveuauthor=S.+Puauthor=M.+Beerauthor=S.+Schorauthor=S.+Kumarauthor=V.+Nicolaescuauthor=B.+D.+Lindenbachauthor=G.+Randallauthor=S.+Einav&title=Interactions+between+the+Hepatitis+C+Virus+Nonstructural+2+Protein+and+Host+Adaptor+Proteins+1+and+4+Orchestrate+Virus+Release&doi=10.1128%2Fmbio.02233-17"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1128%2Fmbio.02233-17&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252Fmbio.02233-17%26sid%3Dliteratum%253Aachs%26aulast%3DXiao%26aufirst%3DF.%26aulast%3DWang%26aufirst%3DS.%26aulast%3DBarouch-Bentov%26aufirst%3DR.%26aulast%3DNeveu%26aufirst%3DG.%26aulast%3DPu%26aufirst%3DS.%26aulast%3DBeer%26aufirst%3DM.%26aulast%3DSchor%26aufirst%3DS.%26aulast%3DKumar%26aufirst%3DS.%26aulast%3DNicolaescu%26aufirst%3DV.%26aulast%3DLindenbach%26aufirst%3DB.%2BD.%26aulast%3DRandall%26aufirst%3DG.%26aulast%3DEinav%26aufirst%3DS.%26atitle%3DInteractions%2520between%2520the%2520Hepatitis%2520C%2520Virus%2520Nonstructural%25202%2520Protein%2520and%2520Host%2520Adaptor%2520Proteins%25201%2520and%25204%2520Orchestrate%2520Virus%2520Release%26jtitle%3DmBio%26date%3D2018%26volume%3D9%26spage%3D1%26epage%3D21%26doi%3D10.1128%2Fmbio.02233-17" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pu, S.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schor, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bekerman, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zanini, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barouch-Bentov, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nagamine, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Einav, S.</span></span> <span> </span><span class="NLM_article-title">Feasibility and Biological Rationale of Repurposing Sunitinib and Erlotinib for Dengue Treatment</span>. <i>Antiviral Res.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>155</i></span>,  <span class="NLM_fpage">67</span>– <span class="NLM_lpage">75</span>, <span class="refDoi"> DOI: 10.1016/j.antiviral.2018.05.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=10.1016%2Fj.antiviral.2018.05.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=29753658" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=1%3ACAS%3A528%3ADC%252BC1cXpslaqtbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=155&publication_year=2018&pages=67-75&author=S.-Y.+Puauthor=F.+Xiaoauthor=S.+Schorauthor=E.+Bekermanauthor=F.+Zaniniauthor=R.+Barouch-Bentovauthor=C.+M.+Nagamineauthor=S.+Einav&title=Feasibility+and+Biological+Rationale+of+Repurposing+Sunitinib+and+Erlotinib+for+Dengue+Treatment&doi=10.1016%2Fj.antiviral.2018.05.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Feasibility and biological rationale of repurposing sunitinib and erlotinib for dengue treatment</span></div><div class="casAuthors">Pu, Szu-Yuan; Xiao, Fei; Schor, Stanford; Bekerman, Elena; Zanini, Fabio; Barouch-Bentov, Rina; Nagamine, Claude M.; Einav, Shirit</div><div class="citationInfo"><span class="NLM_cas:title">Antiviral Research</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">155</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">67-75</span>CODEN:
                <span class="NLM_cas:coden">ARSRDR</span>;
        ISSN:<span class="NLM_cas:issn">0166-3542</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">There is an urgent need for strategies to combat dengue virus (DENV) infection; a major global threat.  We reported that the cellular kinases AAK1 and GAK regulate intracellular trafficking of multiple viruses and that sunitinib and erlotinib, approved anticancer drugs with potent activity against these kinases, protect DENV-infected mice from mortality.  Nevertheless, further characterization of the therapeutic potential and underlying mechanism of this approach is required prior to clin. evaluation.  Here, we demonstrate that sunitinib/erlotinib combination achieves sustained suppression of systemic infection at approved dose in DENV-infected IFN-α/β and IFN-γ receptor-deficient mice.  Nevertheless, treatment with these blood-brain barrier impermeable drugs delays, yet does not prevent, late-onset paralysis; a common manifestation in this immunodeficient mouse model but not in humans.  Sunitinib and erlotinib treatment also demonstrates efficacy in human primary monocyte-derived dendritic cells.  Addnl., DENV infection induces expression of AAK1 transcripts, but not GAK, via single-cell transcriptomics, and these kinases are important mol. targets underlying the anti-DENV effect of sunitinib and erlotinib.  Lastly, sunitinib/erlotinib combination alters inflammatory cytokine responses in DENV-infected mice.  These findings support feasibility of repurposing sunitinib/erlotinib combination as a host-targeted antiviral approach and contribute to understanding its mechanism of antiviral action.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp2cs8UpuoykLVg90H21EOLACvtfcHk0liTK7jLgDn_Dw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXpslaqtbk%253D&md5=acde697b6e3c6d9f42a2011be6091e83</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1016%2Fj.antiviral.2018.05.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.antiviral.2018.05.001%26sid%3Dliteratum%253Aachs%26aulast%3DPu%26aufirst%3DS.-Y.%26aulast%3DXiao%26aufirst%3DF.%26aulast%3DSchor%26aufirst%3DS.%26aulast%3DBekerman%26aufirst%3DE.%26aulast%3DZanini%26aufirst%3DF.%26aulast%3DBarouch-Bentov%26aufirst%3DR.%26aulast%3DNagamine%26aufirst%3DC.%2BM.%26aulast%3DEinav%26aufirst%3DS.%26atitle%3DFeasibility%2520and%2520Biological%2520Rationale%2520of%2520Repurposing%2520Sunitinib%2520and%2520Erlotinib%2520for%2520Dengue%2520Treatment%26jtitle%3DAntiviral%2520Res.%26date%3D2018%26volume%3D155%26spage%3D67%26epage%3D75%26doi%3D10.1016%2Fj.antiviral.2018.05.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kostich, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamman, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Naidu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dandapani, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Easton, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bourin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baker, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Savelieva, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Louis, J. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dokania, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elavazhagan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vattikundala, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharma, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Das, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shankar, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holenarsipur, V. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gulianello, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Molski, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewis, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pieschl, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">OMalley, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lippy, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nouraldeen, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lanthorn, T. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ye, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Balakrishnan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Denton, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grace, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lentz, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Santone, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Main, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swaffield, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carson, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mandlekar, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vikramadithyan, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nara, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dzierba, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bronson, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Macor, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zaczek, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Westphal, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kiss, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bristow, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Conway, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zambrowicz, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Albright, C. F.</span></span> <span> </span><span class="NLM_article-title">Inhibition of AAK1 Kinase as a Novel Therapeutic Approach to Treat Neuropathic Pain</span>. <i>J. Pharmacol. Exp. Ther.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>358</i></span>,  <span class="NLM_fpage">371</span>– <span class="NLM_lpage">386</span>, <span class="refDoi"> DOI: 10.1124/jpet.116.235333</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=10.1124%2Fjpet.116.235333" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=27411717" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvVyks7rI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=358&publication_year=2016&pages=371-386&author=W.+Kostichauthor=B.+D.+Hammanauthor=Y.-W.+Liauthor=S.+Naiduauthor=K.+Dandapaniauthor=J.+Fengauthor=A.+Eastonauthor=C.+Bourinauthor=K.+Bakerauthor=J.+Allenauthor=K.+Savelievaauthor=J.+V.+Louisauthor=M.+Dokaniaauthor=S.+Elavazhaganauthor=P.+Vattikundalaauthor=V.+Sharmaauthor=M.+L.+Dasauthor=G.+Shankarauthor=A.+Kumarauthor=V.+K.+Holenarsipurauthor=M.+Gulianelloauthor=T.+Molskiauthor=J.+M.+Brownauthor=M.+Lewisauthor=Y.+Huangauthor=Y.+Luauthor=R.+Pieschlauthor=K.+OMalleyauthor=J.+Lippyauthor=A.+Nouraldeenauthor=T.+H.+Lanthornauthor=G.+Yeauthor=A.+Wilsonauthor=A.+Balakrishnanauthor=R.+Dentonauthor=J.+E.+Graceauthor=K.+A.+Lentzauthor=K.+S.+Santoneauthor=Y.+Biauthor=A.+Mainauthor=J.+Swaffieldauthor=K.+Carsonauthor=S.+Mandlekarauthor=R.+K.+Vikramadithyanauthor=S.+J.+Naraauthor=C.+Dzierbaauthor=J.+Bronsonauthor=J.+E.+Macorauthor=R.+Zaczekauthor=R.+Westphalauthor=L.+Kissauthor=L.+Bristowauthor=C.+M.+Conwayauthor=B.+Zambrowiczauthor=C.+F.+Albright&title=Inhibition+of+AAK1+Kinase+as+a+Novel+Therapeutic+Approach+to+Treat+Neuropathic+Pain&doi=10.1124%2Fjpet.116.235333"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of AAK1 kinase as a novel therapeutic approach to treat neuropathic pain</span></div><div class="casAuthors">Kostich, Walter; Hamman, Brian D.; Li, Yu-Wen; Naidu, Sreenivasulu; Dandapani, Kumaran; Feng, Jianlin; Easton, Amy; Bourin, Clotilde; Baker, Kevin; Allen, Jason; Savelieva, Katerina; Louis, Justin V.; Dokania, Manoj; Elavazhagan, Saravanan; Vattikundala, Pradeep; Sharma, Vivek; Das, Manish Lal; Shankar, Ganesh; Kumar, Anoop; Holenarsipur, Vinay K.; Gulianello, Michael; Molski, Ted; Brown, Jeffrey M.; Lewis, Martin; Huang, Yanling; Lu, Yifeng; Pieschl, Rick; O'Malley, Kevin; Lippy, Jonathan; Nouraldeen, Amr; Lanthorn, Thomas H.; Ye, Guilan; Wilson, Alan; Balakrishnan, Anand; Denton, Rex; Grace, James E.; Lentz, Kimberley A.; Santone, Kenneth S.; Bi, Yingzhi; Main, Alan; Swaffield, Jon; Carson, Ken; Mandlekar, Sandhya; Vikramadithyan, Reeba K.; Nara, Susheel J.; Dzierba, Carolyn; Bronson, Joanne; Macor, John E.; Zaczek, Robert; Westphal, Ryan; Kiss, Laszlo; Bristow, Linda; Conway, Charles M.; Zambrowicz, Brian; Albright, Charles F.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Pharmacology and Experimental Therapeutics</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">358</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">371-386</span>CODEN:
                <span class="NLM_cas:coden">JPETAB</span>;
        ISSN:<span class="NLM_cas:issn">1521-0103</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">To identify novel targets for neuropathic pain, 3097 mouse knockout lines were tested in acute and persistent pain behavior assays.  One of the lines from this screen, which contained a null allele of the adapter protein-2 assocd. kinase 1 (AAK1) gene, had a normal response in acute pain assays (hot plate, phase I formalin), but a markedly reduced response to persistent pain in phase II formalin.  AAK1 knockout mice also failed to develop tactile allodynia following the Chung procedure of spinal nerve ligation (SNL).  Based on these findings, potent, small-mol. inhibitors of AAK1 were identified.  Studies in mice showed that one such inhibitor, LP-935509, caused a reduced pain response in phase II formalin and reversed fully established pain behavior following the SNL procedure.  Further studies showed that the inhibitor also reduced evoked pain responses in the rat chronic constriction injury (CCI) model and the rat streptozotocin model of diabetic peripheral neuropathy.  Using a nonbrain-penetrant AAK1 inhibitor and local administration of an AAK1 inhibitor, the relevant pool of AAK1 for antineuropathic action was found to be in the spinal cord.  Consistent with these results, AAK1 inhibitors dose-dependently reduced the increased spontaneous neural activity in the spinal cord caused by CCI and blocked the development of windup induced by repeated elec. stimulation of the paw.  The mechanism of AAK1 antinociception was further investigated with inhibitors of α2 adrenergic and opioid receptors.  These studies showed that α2 adrenergic receptor inhibitors, but not opioid receptor inhibitors, not only prevented AAK1 inhibitor antineuropathic action in behavioral assays, but also blocked the AAK1 inhibitor-induced redn. in spinal neural activity in the rat CCI model.  Hence, AAK1 inhibitors are a novel therapeutic approach to neuropathic pain with activity in animal models that is mechanistically linked (behaviorally and electrophysiol.) to α2 adrenergic signaling, a pathway known to be antinociceptive in humans.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo5cJpGeOPjm7Vg90H21EOLACvtfcHk0liTK7jLgDn_Dw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvVyks7rI&md5=d4cec6b3681b63ba7d9119424e53ab61</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1124%2Fjpet.116.235333&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fjpet.116.235333%26sid%3Dliteratum%253Aachs%26aulast%3DKostich%26aufirst%3DW.%26aulast%3DHamman%26aufirst%3DB.%2BD.%26aulast%3DLi%26aufirst%3DY.-W.%26aulast%3DNaidu%26aufirst%3DS.%26aulast%3DDandapani%26aufirst%3DK.%26aulast%3DFeng%26aufirst%3DJ.%26aulast%3DEaston%26aufirst%3DA.%26aulast%3DBourin%26aufirst%3DC.%26aulast%3DBaker%26aufirst%3DK.%26aulast%3DAllen%26aufirst%3DJ.%26aulast%3DSavelieva%26aufirst%3DK.%26aulast%3DLouis%26aufirst%3DJ.%2BV.%26aulast%3DDokania%26aufirst%3DM.%26aulast%3DElavazhagan%26aufirst%3DS.%26aulast%3DVattikundala%26aufirst%3DP.%26aulast%3DSharma%26aufirst%3DV.%26aulast%3DDas%26aufirst%3DM.%2BL.%26aulast%3DShankar%26aufirst%3DG.%26aulast%3DKumar%26aufirst%3DA.%26aulast%3DHolenarsipur%26aufirst%3DV.%2BK.%26aulast%3DGulianello%26aufirst%3DM.%26aulast%3DMolski%26aufirst%3DT.%26aulast%3DBrown%26aufirst%3DJ.%2BM.%26aulast%3DLewis%26aufirst%3DM.%26aulast%3DHuang%26aufirst%3DY.%26aulast%3DLu%26aufirst%3DY.%26aulast%3DPieschl%26aufirst%3DR.%26aulast%3DOMalley%26aufirst%3DK.%26aulast%3DLippy%26aufirst%3DJ.%26aulast%3DNouraldeen%26aufirst%3DA.%26aulast%3DLanthorn%26aufirst%3DT.%2BH.%26aulast%3DYe%26aufirst%3DG.%26aulast%3DWilson%26aufirst%3DA.%26aulast%3DBalakrishnan%26aufirst%3DA.%26aulast%3DDenton%26aufirst%3DR.%26aulast%3DGrace%26aufirst%3DJ.%2BE.%26aulast%3DLentz%26aufirst%3DK.%2BA.%26aulast%3DSantone%26aufirst%3DK.%2BS.%26aulast%3DBi%26aufirst%3DY.%26aulast%3DMain%26aufirst%3DA.%26aulast%3DSwaffield%26aufirst%3DJ.%26aulast%3DCarson%26aufirst%3DK.%26aulast%3DMandlekar%26aufirst%3DS.%26aulast%3DVikramadithyan%26aufirst%3DR.%2BK.%26aulast%3DNara%26aufirst%3DS.%2BJ.%26aulast%3DDzierba%26aufirst%3DC.%26aulast%3DBronson%26aufirst%3DJ.%26aulast%3DMacor%26aufirst%3DJ.%2BE.%26aulast%3DZaczek%26aufirst%3DR.%26aulast%3DWestphal%26aufirst%3DR.%26aulast%3DKiss%26aufirst%3DL.%26aulast%3DBristow%26aufirst%3DL.%26aulast%3DConway%26aufirst%3DC.%2BM.%26aulast%3DZambrowicz%26aufirst%3DB.%26aulast%3DAlbright%26aufirst%3DC.%2BF.%26atitle%3DInhibition%2520of%2520AAK1%2520Kinase%2520as%2520a%2520Novel%2520Therapeutic%2520Approach%2520to%2520Treat%2520Neuropathic%2520Pain%26jtitle%3DJ.%2520Pharmacol.%2520Exp.%2520Ther.%26date%3D2016%26volume%3D358%26spage%3D371%26epage%3D386%26doi%3D10.1124%2Fjpet.116.235333" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bronson, J.</span>; <span class="NLM_string-name">Chen, L.</span>; <span class="NLM_string-name">Ditta, J.</span>; <span class="NLM_string-name">Dzierba, C. D.</span>; <span class="NLM_string-name">Jalagam, P. R.</span>; <span class="NLM_string-name">Luo, G.</span>; <span class="NLM_string-name">Macor, J.</span>; <span class="NLM_string-name">Maishal, T. K.</span>; <span class="NLM_string-name">Nara, S. J.</span>; <span class="NLM_string-name">Rajamani, R.</span>; <span class="NLM_string-name">Sistla, R. K.</span>; <span class="NLM_string-name">Thangavel, S.</span></span> <span> </span><span class="NLM_article-title">Biaryl Kinase Inhibitors</span>. WO <span class="NLM_patent">2017/059085</span>, <span class="NLM_year">2017</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2017&author=J.+Bronson&author=L.+Chen&author=J.+Ditta&author=C.+D.+Dzierba&author=P.+R.+Jalagam&author=G.+Luo&author=J.+Macor&author=T.+K.+Maishal&author=S.+J.+Nara&author=R.+Rajamani&author=R.+K.+Sistla&author=S.+Thangavel&title=Biaryl+Kinase+Inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DBronson%26aufirst%3DJ.%26atitle%3DBiaryl%2520Kinase%2520Inhibitors%26date%3D2017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kuai, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ong, S.-E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Madison, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duvall, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewis, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luce, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Conner, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pearlman, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wood, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schreiber, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carr, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scolnick, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haggarty, S. J.</span></span> <span> </span><span class="NLM_article-title">AAK1 Identified as an Inhibitor of Neuregulin-1/ErbB4-Dependent Neurotrophic Factor Signaling Using Integrative Chemical Genomics and Proteomics</span>. <i>Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">891</span>– <span class="NLM_lpage">906</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2011.03.017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=10.1016%2Fj.chembiol.2011.03.017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=21802010" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=1%3ACAS%3A528%3ADC%252BC3MXpsFaruro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2011&pages=891-906&author=L.+Kuaiauthor=S.-E.+Ongauthor=J.+M.+Madisonauthor=X.+Wangauthor=J.+R.+Duvallauthor=T.+A.+Lewisauthor=C.+J.+Luceauthor=S.+D.+Connerauthor=D.+A.+Pearlmanauthor=J.+L.+Woodauthor=S.+L.+Schreiberauthor=S.+A.+Carrauthor=E.+M.+Scolnickauthor=S.+J.+Haggarty&title=AAK1+Identified+as+an+Inhibitor+of+Neuregulin-1%2FErbB4-Dependent+Neurotrophic+Factor+Signaling+Using+Integrative+Chemical+Genomics+and+Proteomics&doi=10.1016%2Fj.chembiol.2011.03.017"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">AAK1 Identified as an Inhibitor of Neuregulin-1/ErbB4-Dependent Neurotrophic Factor Signaling Using Integrative Chemical Genomics and Proteomics</span></div><div class="casAuthors">Kuai, Letian; Ong, Shao-En; Madison, Jon M.; Wang, Xiang; Duvall, Jeremy R.; Lewis, Timothy A.; Luce, Catherine J.; Conner, Sean D.; Pearlman, David A.; Wood, John L.; Schreiber, Stuart L.; Carr, Steven A.; Scolnick, Edward M.; Haggarty, Stephen J.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">891-906</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Summary: Target identification remains challenging for the field of chem. biol.  We describe an integrative chem. genomic and proteomic approach combining the use of differentially active analogs of small mol. probes with stable isotope labeling by amino acids in cell culture-mediated affinity enrichment, followed by subsequent testing of candidate targets using RNA interference-mediated gene silencing.  We applied this approach to characterizing the natural product K252a and its ability to potentiate neuregulin-1 (Nrg1)/ErbB4 (v-erb-a erythroblastic leukemia viral oncogene homolog 4)-dependent neurotrophic factor signaling and neuritogenesis.  We show that AAK1 (adaptor-assocd. kinase 1) is a relevant target of K252a, and that the loss of AAK1 alters ErbB4 trafficking and expression levels, providing evidence for a previously unrecognized role for AAK1 in Nrg1-mediated neurotrophic factor signaling.  Similar strategies should lead to the discovery of novel targets for therapeutic development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo0dEttut6YzLVg90H21EOLACvtfcHk0lg56zzgmfhDvA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXpsFaruro%253D&md5=3fcf22dfc3defc7b684a643444a7838a</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2011.03.017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2011.03.017%26sid%3Dliteratum%253Aachs%26aulast%3DKuai%26aufirst%3DL.%26aulast%3DOng%26aufirst%3DS.-E.%26aulast%3DMadison%26aufirst%3DJ.%2BM.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DDuvall%26aufirst%3DJ.%2BR.%26aulast%3DLewis%26aufirst%3DT.%2BA.%26aulast%3DLuce%26aufirst%3DC.%2BJ.%26aulast%3DConner%26aufirst%3DS.%2BD.%26aulast%3DPearlman%26aufirst%3DD.%2BA.%26aulast%3DWood%26aufirst%3DJ.%2BL.%26aulast%3DSchreiber%26aufirst%3DS.%2BL.%26aulast%3DCarr%26aufirst%3DS.%2BA.%26aulast%3DScolnick%26aufirst%3DE.%2BM.%26aulast%3DHaggarty%26aufirst%3DS.%2BJ.%26atitle%3DAAK1%2520Identified%2520as%2520an%2520Inhibitor%2520of%2520Neuregulin-1%252FErbB4-Dependent%2520Neurotrophic%2520Factor%2520Signaling%2520Using%2520Integrative%2520Chemical%2520Genomics%2520and%2520Proteomics%26jtitle%3DChem.%2520Biol.%26date%3D2011%26volume%3D18%26spage%3D891%26epage%3D906%26doi%3D10.1016%2Fj.chembiol.2011.03.017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bamborough, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drewry, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harper, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, G. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schneider, K.</span></span> <span> </span><span class="NLM_article-title">Assessment of Chemical Coverage of Kinome Space and Its Implications for Kinase Drug Discovery</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>51</i></span>,  <span class="NLM_fpage">7898</span>– <span class="NLM_lpage">7914</span>, <span class="refDoi"> DOI: 10.1021/jm8011036</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm8011036" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhsVequ7zI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2008&pages=7898-7914&author=P.+Bamboroughauthor=D.+Drewryauthor=G.+Harperauthor=G.+K.+Smithauthor=K.+Schneider&title=Assessment+of+Chemical+Coverage+of+Kinome+Space+and+Its+Implications+for+Kinase+Drug+Discovery&doi=10.1021%2Fjm8011036"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Assessment of Chemical Coverage of Kinome Space and Its Implications for Kinase Drug Discovery</span></div><div class="casAuthors">Bamborough, Paul; Drewry, David; Harper, Gavin; Smith, Gary K.; Schneider, Klaus</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">7898-7914</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">More than 500 compds. chosen to represent kinase inhibitor space have been screened against a panel of over 200 protein kinases.  Significant results include the identification of hits against new kinases including PIM1 and MPSK1, and the expansion of the inhibition profiles of several literature compds.  A detailed anal. of the data through the use of affinity fingerprints has produced findings with implications for biol. target selection, the choice of tool compds. for target validation, and lead discovery and optimization.  In a detailed examn. of the tyrosine kinases, interesting relationships have been found between targets and compds.  Taken together, these results show how broad cross-profiling can provide important insights to assist kinase drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrBRI-arQk3t7Vg90H21EOLACvtfcHk0lg56zzgmfhDvA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhsVequ7zI&md5=8d98c80fb157b987a19e32a1efb53103</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1021%2Fjm8011036&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm8011036%26sid%3Dliteratum%253Aachs%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DDrewry%26aufirst%3DD.%26aulast%3DHarper%26aufirst%3DG.%26aulast%3DSmith%26aufirst%3DG.%2BK.%26aulast%3DSchneider%26aufirst%3DK.%26atitle%3DAssessment%2520of%2520Chemical%2520Coverage%2520of%2520Kinome%2520Space%2520and%2520Its%2520Implications%2520for%2520Kinase%2520Drug%2520Discovery%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2008%26volume%3D51%26spage%3D7898%26epage%3D7914%26doi%3D10.1021%2Fjm8011036" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Barl, N. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sansiaume-Dagousset, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karaghiosoff, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knochel, P.</span></span> <span> </span><span class="NLM_article-title">Full Functionalization of the 7-Azaindole Scaffold by Selective Metalation and Sulfoxide/Magnesium Exchange</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">10093</span>– <span class="NLM_lpage">10096</span>, <span class="refDoi"> DOI: 10.1002/anie.201303490</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=10.1002%2Fanie.201303490" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtFOkurnL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2013&pages=10093-10096&author=N.+M.+Barlauthor=E.+Sansiaume-Dagoussetauthor=K.+Karaghiosoffauthor=P.+Knochel&title=Full+Functionalization+of+the+7-Azaindole+Scaffold+by+Selective+Metalation+and+Sulfoxide%2FMagnesium+Exchange&doi=10.1002%2Fanie.201303490"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Full Functionalization of the 7-Azaindole Scaffold by Selective Metalation and Sulfoxide/Magnesium Exchange</span></div><div class="casAuthors">Barl, Nadja M.; Sansiaume-Dagousset, Elodie; Karaghiosoff, Konstantin; Knochel, Paul</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">38</span>),
    <span class="NLM_cas:pages">10093-10096</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">The authors report a novel methodol. for the mild full functionalization of the 7-azaindole scaffold by combining directed metalation and halogen/magnesium and sulfoxide/magnesium exchange using substituted 7-azaindole I as the key intermediate.  Introducing the sulfoxide substituent on C5 was essential for all further transformations.  For example, reacting I with TMPMgCl·LiCl/THF (TMP = 2,2,6,6-tetramethylpiperidine), followed by treatment with ZnCl2, CuCN·2LiCl, and then 2-furancarbonyl chloride gave furanyl ketone deriv. II in 63% yield.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoDM4RBErZ9Q7Vg90H21EOLACvtfcHk0lg56zzgmfhDvA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtFOkurnL&md5=f4a2dd87924c5c092e226b1619d4323a</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1002%2Fanie.201303490&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201303490%26sid%3Dliteratum%253Aachs%26aulast%3DBarl%26aufirst%3DN.%2BM.%26aulast%3DSansiaume-Dagousset%26aufirst%3DE.%26aulast%3DKaraghiosoff%26aufirst%3DK.%26aulast%3DKnochel%26aufirst%3DP.%26atitle%3DFull%2520Functionalization%2520of%2520the%25207-Azaindole%2520Scaffold%2520by%2520Selective%2520Metalation%2520and%2520Sulfoxide%252FMagnesium%2520Exchange%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2013%26volume%3D52%26spage%3D10093%26epage%3D10096%26doi%3D10.1002%2Fanie.201303490" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brumsted, C. J.</span>; <span class="NLM_string-name">Moorlag, H.</span>; <span class="NLM_string-name">Radinov, R. N.</span>; <span class="NLM_string-name">Ren, Y.</span>; <span class="NLM_string-name">Waldmeier, P.</span></span> <span> </span><span class="NLM_article-title">Method for Preparation of N-{3-[5-(4-Chlorophenyl)-1H-Pyrrolo[2,3-b]Pyridine-3-Carbonyl]-2,4-Difluorophenyl} Propane-1-Sulfonamide</span>. WO <span class="NLM_patent">2012/010538 A2</span>, <span class="NLM_year">2012</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2012&author=C.+J.+Brumsted&author=H.+Moorlag&author=R.+N.+Radinov&author=Y.+Ren&author=P.+Waldmeier&title=Method+for+Preparation+of+N-%7B3-%5B5-%284-Chlorophenyl%29-1H-Pyrrolo%5B2%2C3-b%5DPyridine-3-Carbonyl%5D-2%2C4-Difluorophenyl%7D+Propane-1-Sulfonamide"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DBrumsted%26aufirst%3DC.%2BJ.%26atitle%3DMethod%2520for%2520Preparation%2520of%2520N-%257B3-%255B5-%25284-Chlorophenyl%2529-1H-Pyrrolo%255B2%252C3-b%255DPyridine-3-Carbonyl%255D-2%252C4-Difluorophenyl%257D%2520Propane-1-Sulfonamide%26date%3D2012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Han, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Green, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pfeifer, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gosselin, F.</span></span> <span> </span><span class="NLM_article-title">Highly Regioselective and Practical Synthesis of 5-Bromo-4-Chloro-3- Nitro-7-Azaindole</span>. <i>Org. Process Res. Dev.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">664</span>– <span class="NLM_lpage">668</span>, <span class="refDoi"> DOI: 10.1021/acs.oprd.7b00060</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.oprd.7b00060" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=1%3ACAS%3A528%3ADC%252BC2sXltVSrtL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2017&pages=664-668&author=C.+Hanauthor=K.+Greenauthor=E.+Pfeiferauthor=F.+Gosselin&title=Highly+Regioselective+and+Practical+Synthesis+of+5-Bromo-4-Chloro-3-+Nitro-7-Azaindole&doi=10.1021%2Facs.oprd.7b00060"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Highly Regioselective and Practical Synthesis of 5-Bromo-4-chloro-3-nitro-7-azaindole</span></div><div class="casAuthors">Han, Chong; Green, Keena; Pfeifer, Eugen; Gosselin, Francis</div><div class="citationInfo"><span class="NLM_cas:title">Organic Process Research & Development</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">664-668</span>CODEN:
                <span class="NLM_cas:coden">OPRDFK</span>;
        ISSN:<span class="NLM_cas:issn">1083-6160</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We report an efficient and highly regiocontrolled route to prep. a functionalized 7-azaindole deriv. (5-bromo-4-chloro-3-nitro-7-azaindole) from readily available parent 7-azaindole featuring a highly regioselective bromination of the 4-chloro-3-nitro-7-azaindole intermediate.  In addn. to the high efficiency and excellent control of regioisomeric impurities, the process is operationally simple by isolating each product via direct crystn. from the reaction mixt. with no liq.-liq. extns. or distn. steps needed.  We demonstrated the route on >50 kg scale and 46% overall yield to provide the target product in 97% purity by HPLC, which can serve as a useful building block for the prepn. of a series of 3,4,5-substituted-7-azaindole derivs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrhmXJx-GEcXbVg90H21EOLACvtfcHk0ljdxsyft0-lGg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXltVSrtL4%253D&md5=ab19646db3439c8f42596749a30e0df7</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Facs.oprd.7b00060&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.oprd.7b00060%26sid%3Dliteratum%253Aachs%26aulast%3DHan%26aufirst%3DC.%26aulast%3DGreen%26aufirst%3DK.%26aulast%3DPfeifer%26aufirst%3DE.%26aulast%3DGosselin%26aufirst%3DF.%26atitle%3DHighly%2520Regioselective%2520and%2520Practical%2520Synthesis%2520of%25205-Bromo-4-Chloro-3-%2520Nitro-7-Azaindole%26jtitle%3DOrg.%2520Process%2520Res.%2520Dev.%26date%3D2017%26volume%3D21%26spage%3D664%26epage%3D668%26doi%3D10.1021%2Facs.oprd.7b00060" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Stokes, S.</span>; <span class="NLM_string-name">Graham, C. J.</span>; <span class="NLM_string-name">Ray, S. C.</span>; <span class="NLM_string-name">Stefaniak, E. J.</span></span> <span> </span><span class="NLM_article-title">1H-Pyrrolo[2,3-b]Pyridine Derivatives and Their Use as Kinase Inhibitors</span>. WO <span class="NLM_patent">2013/114113 A1</span>, <span class="NLM_year">2013</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2013&author=S.+Stokes&author=C.+J.+Graham&author=S.+C.+Ray&author=E.+J.+Stefaniak&title=1H-Pyrrolo%5B2%2C3-b%5DPyridine+Derivatives+and+Their+Use+as+Kinase+Inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DStokes%26aufirst%3DS.%26atitle%3D1H-Pyrrolo%255B2%252C3-b%255DPyridine%2520Derivatives%2520and%2520Their%2520Use%2520as%2520Kinase%2520Inhibitors%26date%3D2013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gao, L.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kovackova, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Šála, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramadori, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Jonghe, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herdewijn, P.</span></span> <span> </span><span class="NLM_article-title">Discovery of Dual Death-Associated Protein Related Apoptosis Inducing Protein Kinase 1 and 2 Inhibitors by a Scaffold Hopping Approach</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">7624</span>– <span class="NLM_lpage">7643</span>, <span class="refDoi"> DOI: 10.1021/jm5007929</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm5007929" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsVGhtL7P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=7624-7643&author=L.-J.+Gaoauthor=S.+Kovackovaauthor=M.+%C5%A0%C3%A1laauthor=A.+T.+Ramadoriauthor=S.+De+Jongheauthor=P.+Herdewijn&title=Discovery+of+Dual+Death-Associated+Protein+Related+Apoptosis+Inducing+Protein+Kinase+1+and+2+Inhibitors+by+a+Scaffold+Hopping+Approach&doi=10.1021%2Fjm5007929"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Dual Death-Associated Protein Related Apoptosis Inducing Protein Kinase 1 and 2 Inhibitors by a Scaffold Hopping Approach</span></div><div class="casAuthors">Gao, Ling-Jie; Kovackova, Sona; Sala, Michal; Ramadori, Anna Teresa; De Jonghe, Steven; Herdewijn, Piet</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">7624-7643</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">DRAK2 emerged as a promising drug target for the treatment of autoimmune diseases and to prevent graft rejection after organ transplantation.  Screening of a compd. library in a DRAK2 binding assay led to the identification of an isothiazolo[5,4-b]pyridine deriv. as a novel ligand for DRAK2, displaying a Kd value of 1.6 μM.  Subsequent medicinal chem. work led to the discovery of a thieno[2,3-b]pyridine deriv. with strong DRAK2 binding affinity (Kd = 9 nM).  Moreover, this compd. also behaves as a functional inhibitor of DRAK2 enzymic activity, displaying an IC50 value of 0.82 μM, although lacking selectivity, when tested against DRAK1.  This paper describes for the first time functionally active dual DRAK1 and DRAK2 inhibitors that can be used as starting point for the synthesis of chem. tool compds. to study DRAK1 and DRAK2 biol., or they can be considered as hit compds. for hit-to-lead optimization campaigns in drug discovery programs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpkAuMAD_Ac_LVg90H21EOLACvtfcHk0ljdxsyft0-lGg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsVGhtL7P&md5=15353fd901634efed8d2953a1cf72ecf</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1021%2Fjm5007929&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm5007929%26sid%3Dliteratum%253Aachs%26aulast%3DGao%26aufirst%3DL.-J.%26aulast%3DKovackova%26aufirst%3DS.%26aulast%3D%25C5%25A0%25C3%25A1la%26aufirst%3DM.%26aulast%3DRamadori%26aufirst%3DA.%2BT.%26aulast%3DDe%2BJonghe%26aufirst%3DS.%26aulast%3DHerdewijn%26aufirst%3DP.%26atitle%3DDiscovery%2520of%2520Dual%2520Death-Associated%2520Protein%2520Related%2520Apoptosis%2520Inducing%2520Protein%2520Kinase%25201%2520and%25202%2520Inhibitors%2520by%2520a%2520Scaffold%2520Hopping%2520Approach%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D7624%26epage%3D7643%26doi%3D10.1021%2Fjm5007929" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Le Huerou, Y.</span>; <span class="NLM_string-name">Blake, J.</span>; <span class="NLM_string-name">Gunwardana, I.</span>; <span class="NLM_string-name">Mohr, P.</span>; <span class="NLM_string-name">Wallace, E.</span>; <span class="NLM_string-name">Wang, B.</span>; <span class="NLM_string-name">Chicarelli, M.</span>; <span class="NLM_string-name">Lyon, M.</span></span> <span> </span><span class="NLM_article-title">Pyrrolopyridines as Kinase Inhibitors</span>. WO <span class="NLM_patent">2009/140320</span>, <span class="NLM_year">2009</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2009&author=Y.+Le+Huerou&author=J.+Blake&author=I.+Gunwardana&author=P.+Mohr&author=E.+Wallace&author=B.+Wang&author=M.+Chicarelli&author=M.+Lyon&title=Pyrrolopyridines+as+Kinase+Inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DLe%2BHuerou%26aufirst%3DY.%26atitle%3DPyrrolopyridines%2520as%2520Kinase%2520Inhibitors%26date%3D2009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Stavenger, R.</span>; <span class="NLM_string-name">Witherington, J.</span>; <span class="NLM_string-name">Rawlings, D.</span>; <span class="NLM_string-name">Holt, D.</span>; <span class="NLM_string-name">Chan, G.</span></span> <span> </span><span class="NLM_article-title">CHK1 Kinase Inhibitors</span>. WO <span class="NLM_patent">03/028724</span>, <span class="NLM_year">2003</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2003&author=R.+Stavenger&author=J.+Witherington&author=D.+Rawlings&author=D.+Holt&author=G.+Chan&title=CHK1+Kinase+Inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DStavenger%26aufirst%3DR.%26atitle%3DCHK1%2520Kinase%2520Inhibitors%26date%3D2003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bahekar, R.
H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jain, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jadav, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prajapati, V. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, D. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gupta, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharma, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tom, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bandyopadhya, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Modi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, P. R.</span></span> <span> </span><span class="NLM_article-title">Synthesis and antidiabetic activity of 2,5-disubstituted-3-imidazol-2-yl-pyrrolo[2,3-b]pyridines and thieno[2,3-b]pyridines</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">6782</span>– <span class="NLM_lpage">6795</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2007.08.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=10.1016%2Fj.bmc.2007.08.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=17723306" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtVCitr7K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2007&pages=6782-6795&author=R.%0AH.+Bahekarauthor=M.+R.+Jainauthor=P.+A.+Jadavauthor=V.+M.+Prajapatiauthor=D.+N.+Patelauthor=A.+A.+Guptaauthor=A.+Sharmaauthor=R.+Tomauthor=D.+Bandyopadhyaauthor=H.+Modiauthor=P.+R.+Patel&title=Synthesis+and+antidiabetic+activity+of+2%2C5-disubstituted-3-imidazol-2-yl-pyrrolo%5B2%2C3-b%5Dpyridines+and+thieno%5B2%2C3-b%5Dpyridines&doi=10.1016%2Fj.bmc.2007.08.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis and antidiabetic activity of 2,5-disubstituted-3-imidazol-2-yl-pyrrolo[2,3-b]pyridines and thieno[2,3-b]pyridines</span></div><div class="casAuthors">Bahekar, Rajesh H.; Jain, Mukul R.; Jadav, Pradip A.; Prajapati, Vijay M.; Patel, Dipam N.; Gupta, Arun A.; Sharma, Ajay; Tom, Robby; Bandyopadhya, Debdutta; Modi, Honey; Patel, Pankaj R.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">6782-6795</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">In the present investigation, two series of 2,5-disubstituted-3-imidazol-2-yl-pyrrolo[2,3-b]pyridines (2a-l) and thieno[2,3-b]pyridines (3a-l) were designed as analogs of BL 11282 (1).  The in vitro glucose dependent insulinotropic activity of all the test compds. was evaluated using RIN5F cell based assay and all the test compds. showed glucose and concn. dependent insulin secretion.  The in vivo antidiabetic activities of most potent compds. from each series (I and II) were assessed in C57BL/6J mice.  Compds. I and II showed dose dependent insulin secretion and significant glucose redn. in vivo.  In general, compds. I and II were found to be equipotent at all the three different doses selected and with respect to BL 11282, both the test compds. were found to be more potent, at all the time points.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGopXNGDHu3ZVbVg90H21EOLACvtfcHk0lhLaXztneQ7mw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtVCitr7K&md5=8b0133862be93a285639df3c0adbbc95</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2007.08.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2007.08.005%26sid%3Dliteratum%253Aachs%26aulast%3DBahekar%26aufirst%3DR.%2BH.%26aulast%3DJain%26aufirst%3DM.%2BR.%26aulast%3DJadav%26aufirst%3DP.%2BA.%26aulast%3DPrajapati%26aufirst%3DV.%2BM.%26aulast%3DPatel%26aufirst%3DD.%2BN.%26aulast%3DGupta%26aufirst%3DA.%2BA.%26aulast%3DSharma%26aufirst%3DA.%26aulast%3DTom%26aufirst%3DR.%26aulast%3DBandyopadhya%26aufirst%3DD.%26aulast%3DModi%26aufirst%3DH.%26aulast%3DPatel%26aufirst%3DP.%2BR.%26atitle%3DSynthesis%2520and%2520antidiabetic%2520activity%2520of%25202%252C5-disubstituted-3-imidazol-2-yl-pyrrolo%255B2%252C3-b%255Dpyridines%2520and%2520thieno%255B2%252C3-b%255Dpyridines%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2007%26volume%3D15%26spage%3D6782%26epage%3D6795%26doi%3D10.1016%2Fj.bmc.2007.08.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chavan, N. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nayak, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kusurkar, R. S.</span></span> <span> </span><span class="NLM_article-title">A rapid method toward the synthesis of new substituted tetrahydro α-carbolines and α-carbolines</span>. <i>Tetrahedron</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">1827</span>– <span class="NLM_lpage">1831</span>, <span class="refDoi"> DOI: 10.1016/j.tet.2010.01.042</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=10.1016%2Fj.tet.2010.01.042" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=1%3ACAS%3A528%3ADC%252BC3cXitVSitL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=1827-1831&author=N.+L.+Chavanauthor=S.+K.+Nayakauthor=R.+S.+Kusurkar&title=A+rapid+method+toward+the+synthesis+of+new+substituted+tetrahydro+%CE%B1-carbolines+and+%CE%B1-carbolines&doi=10.1016%2Fj.tet.2010.01.042"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">A rapid method toward the synthesis of new substituted tetrahydro α-carbolines and α-carbolines</span></div><div class="casAuthors">Chavan, Neelam L.; Nayak, Sandip K.; Kusurkar, Radhika S.</div><div class="citationInfo"><span class="NLM_cas:title">Tetrahedron</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1827-1831</span>CODEN:
                <span class="NLM_cas:coden">TETRAB</span>;
        ISSN:<span class="NLM_cas:issn">0040-4020</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A rapid and efficient method for stereoselective synthesis of new substituted tetrahydro-α-carbolines using Diels-Alder reaction under microwave irradn. has been developed.  Further, dehydrogenation of these adducts resulted in synthesis of new substituted α-carbolines.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrprueVyzy4rLVg90H21EOLACvtfcHk0lhLaXztneQ7mw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXitVSitL0%253D&md5=e3a285d8ff93115769f926259da3648d</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1016%2Fj.tet.2010.01.042&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tet.2010.01.042%26sid%3Dliteratum%253Aachs%26aulast%3DChavan%26aufirst%3DN.%2BL.%26aulast%3DNayak%26aufirst%3DS.%2BK.%26aulast%3DKusurkar%26aufirst%3DR.%2BS.%26atitle%3DA%2520rapid%2520method%2520toward%2520the%2520synthesis%2520of%2520new%2520substituted%2520tetrahydro%2520%25CE%25B1-carbolines%2520and%2520%25CE%25B1-carbolines%26jtitle%3DTetrahedron%26date%3D2010%26volume%3D66%26spage%3D1827%26epage%3D1831%26doi%3D10.1016%2Fj.tet.2010.01.042" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chinta, B. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baire, B.</span></span> <span> </span><span class="NLM_article-title">Reactivity of indole-3-alkoxides in the absence of acids: Rapid synthesis of homo- bisindolylmethanes</span>. <i>Tetrahedron</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>72</i></span>,  <span class="NLM_fpage">8106</span>– <span class="NLM_lpage">8116</span>, <span class="refDoi"> DOI: 10.1016/j.tet.2016.10.067</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=10.1016%2Fj.tet.2016.10.067" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvVahsrbM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=72&publication_year=2016&pages=8106-8116&author=B.+S.+Chintaauthor=B.+Baire&title=Reactivity+of+indole-3-alkoxides+in+the+absence+of+acids%3A+Rapid+synthesis+of+homo-+bisindolylmethanes&doi=10.1016%2Fj.tet.2016.10.067"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Reactivity of indole-3-alkoxides in the absence of acids: rapid synthesis of homo-bisindolylmethanes</span></div><div class="casAuthors">Chinta, Bhavani Shankar; Baire, Beeraiah</div><div class="citationInfo"><span class="NLM_cas:title">Tetrahedron</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">72</span>
        (<span class="NLM_cas:issue">49</span>),
    <span class="NLM_cas:pages">8106-8116</span>CODEN:
                <span class="NLM_cas:coden">TETRAB</span>;
        ISSN:<span class="NLM_cas:issn">0040-4020</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">An unprecedented behavior of indole-3-alkoxides (MgX or Li) in situ generated from indolylcarbaldehydes and Grignard/lithium reagents was reported.  Tuning the electronic properties of the alkoxides offered the direct and selective construction of bisindolylmethanes I [R = Me, ethynyl, 2-thienyl, etc.; R1 = Me, n-hexyl, Bn; X = H, OMe, Br] and indole-3-carbinols II [R = Me, Ph, 2-thienyl, etc.; R1 = Me, Bn, Ts; Y = CH, N].  This process showed very broad scope and represented the reagent (external) free, greener synthesis of structurally divergent bisindolylmethanes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpIPC5u_aaMT7Vg90H21EOLACvtfcHk0lhLaXztneQ7mw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvVahsrbM&md5=4a8553c51b7809a2d7e11f536fb52e6f</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1016%2Fj.tet.2016.10.067&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tet.2016.10.067%26sid%3Dliteratum%253Aachs%26aulast%3DChinta%26aufirst%3DB.%2BS.%26aulast%3DBaire%26aufirst%3DB.%26atitle%3DReactivity%2520of%2520indole-3-alkoxides%2520in%2520the%2520absence%2520of%2520acids%253A%2520Rapid%2520synthesis%2520of%2520homo-%2520bisindolylmethanes%26jtitle%3DTetrahedron%26date%3D2016%26volume%3D72%26spage%3D8106%26epage%3D8116%26doi%3D10.1016%2Fj.tet.2016.10.067" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McCoull, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hennessy, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blades, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Box, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chuaqui, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dowling, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davies, C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferguson, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goldberg, F. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Howe, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kemmitt, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lamont, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Madden, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McWhirter, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Varnes, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ward, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williams, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, B.</span></span> <span> </span><span class="NLM_article-title">Identification and Optimisation of 7-Azaindole PAK1 Inhibitors with Improved Potency and Kinase Selectivity</span>. <i>MedChemComm</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">1533</span>– <span class="NLM_lpage">1539</span>, <span class="refDoi"> DOI: 10.1039/c4md00280f</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=10.1039%2Fc4md00280f" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtlOksrvL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=1533-1539&author=W.+McCoullauthor=E.+J.+Hennessyauthor=K.+Bladesauthor=M.+R.+Boxauthor=C.+Chuaquiauthor=J.+E.+Dowlingauthor=C.+D.+Daviesauthor=A.+D.+Fergusonauthor=F.+W.+Goldbergauthor=N.+J.+Howeauthor=P.+D.+Kemmittauthor=G.+M.+Lamontauthor=K.+Maddenauthor=C.+McWhirterauthor=J.+G.+Varnesauthor=R.+A.+Wardauthor=J.+D.+Williamsauthor=B.+Yang&title=Identification+and+Optimisation+of+7-Azaindole+PAK1+Inhibitors+with+Improved+Potency+and+Kinase+Selectivity&doi=10.1039%2Fc4md00280f"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Identification and optimisation of 7-azaindole PAK1 inhibitors with improved potency and kinase selectivity</span></div><div class="casAuthors">McCoull, William; Hennessy, Edward J.; Blades, Kevin; Box, Matthew R.; Chuaqui, Claudio; Dowling, James E.; Davies, Christopher D.; Ferguson, Andrew D.; Goldberg, Frederick W.; Howe, Nicholas J.; Kemmitt, Paul D.; Lamont, Gillian M.; Madden, Katrina; McWhirter, Claire; Varnes, Jeffrey G.; Ward, Richard A.; Williams, Jason D.; Yang, Bin</div><div class="citationInfo"><span class="NLM_cas:title">MedChemComm</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1533-1539</span>CODEN:
                <span class="NLM_cas:coden">MCCEAY</span>;
        ISSN:<span class="NLM_cas:issn">2040-2503</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A novel series of PAK1 inhibitors was discovered from a kinase directed screen.  SAR exploration in the selectivity pocket and solvent tail regions was conducted to understand and optimize PAK1 potency and selectivity against targeted kinases.  A liganded PAK1 crystal structure was utilized to guide compd. design.  Permeability and kinase selectivity impacted the translation of enzyme to cellular PAK1 potency.  Compd. 36 (AZ-PAK-36) demonstrated improved Gini coeff., good PAK1 cellular potency and has utility as a tool compd. for target validation studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrFuGgNmSEGu7Vg90H21EOLACvtfcHk0lhLaXztneQ7mw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtlOksrvL&md5=bda79c5a2a067284e8520c8fb97069fd</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1039%2Fc4md00280f&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc4md00280f%26sid%3Dliteratum%253Aachs%26aulast%3DMcCoull%26aufirst%3DW.%26aulast%3DHennessy%26aufirst%3DE.%2BJ.%26aulast%3DBlades%26aufirst%3DK.%26aulast%3DBox%26aufirst%3DM.%2BR.%26aulast%3DChuaqui%26aufirst%3DC.%26aulast%3DDowling%26aufirst%3DJ.%2BE.%26aulast%3DDavies%26aufirst%3DC.%2BD.%26aulast%3DFerguson%26aufirst%3DA.%2BD.%26aulast%3DGoldberg%26aufirst%3DF.%2BW.%26aulast%3DHowe%26aufirst%3DN.%2BJ.%26aulast%3DKemmitt%26aufirst%3DP.%2BD.%26aulast%3DLamont%26aufirst%3DG.%2BM.%26aulast%3DMadden%26aufirst%3DK.%26aulast%3DMcWhirter%26aufirst%3DC.%26aulast%3DVarnes%26aufirst%3DJ.%2BG.%26aulast%3DWard%26aufirst%3DR.%2BA.%26aulast%3DWilliams%26aufirst%3DJ.%2BD.%26aulast%3DYang%26aufirst%3DB.%26atitle%3DIdentification%2520and%2520Optimisation%2520of%25207-Azaindole%2520PAK1%2520Inhibitors%2520with%2520Improved%2520Potency%2520and%2520Kinase%2520Selectivity%26jtitle%3DMedChemComm%26date%3D2014%26volume%3D5%26spage%3D1533%26epage%3D1539%26doi%3D10.1039%2Fc4md00280f" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gourdain, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dairou, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Denhez, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bui, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodrigues-Lima, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Janel, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delabar, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cariou, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dodd, R. H.</span></span> <span> </span><span class="NLM_article-title">Development of DANDYs, New 3,5-Diaryl-7-Azaindoles Demonstrating Potent DYRK1A Kinase Inhibitory Activity</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">9569</span>– <span class="NLM_lpage">9585</span>, <span class="refDoi"> DOI: 10.1021/jm401049v</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm401049v" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhslWrt77K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=9569-9585&author=S.+Gourdainauthor=J.+Dairouauthor=C.+Denhezauthor=L.+C.+Buiauthor=F.+Rodrigues-Limaauthor=N.+Janelauthor=J.+M.+Delabarauthor=K.+Cariouauthor=R.+H.+Dodd&title=Development+of+DANDYs%2C+New+3%2C5-Diaryl-7-Azaindoles+Demonstrating+Potent+DYRK1A+Kinase+Inhibitory+Activity&doi=10.1021%2Fjm401049v"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Development of DANDYs, New 3,5-Diaryl-7-azaindoles Demonstrating Potent DYRK1A Kinase Inhibitory Activity</span></div><div class="casAuthors">Gourdain, Stephanie; Dairou, Julien; Denhez, Clement; Bui, Linh Chi; Rodrigues-Lima, Fernando; Janel, Nathalie; Delabar, Jean M.; Cariou, Kevin; Dodd, Robert H.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">9569-9585</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A series of 3,5-diaryl-1H-pyrrolo[2,3-b]pyridines was synthesized and evaluated for inhibition of DYRK1A kinase in vitro.  Derivs. having hydroxy groups on the aryl moieties demonstrated high inhibitory potencies with Kis in the low nanomolar range.  Their methoxy analogs were up to 100 times less active.  Docking studies at the ATP binding site suggested that these compds. bind tightly to this site via a network of multiple H-bonds with the peptide backbone.  None of the active compds. was cytotoxic to KB cells at 10-6 M.  Kinase profiling revealed that compd. I showed 2-fold selectivity for DYRK1A with respect to DYRK2 and DYRK3.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo1vxODs_tGD7Vg90H21EOLACvtfcHk0ljdU2VQ8w9Kpw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhslWrt77K&md5=27dcc77a1c0d413cf0ab5c999cf9a195</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1021%2Fjm401049v&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm401049v%26sid%3Dliteratum%253Aachs%26aulast%3DGourdain%26aufirst%3DS.%26aulast%3DDairou%26aufirst%3DJ.%26aulast%3DDenhez%26aufirst%3DC.%26aulast%3DBui%26aufirst%3DL.%2BC.%26aulast%3DRodrigues-Lima%26aufirst%3DF.%26aulast%3DJanel%26aufirst%3DN.%26aulast%3DDelabar%26aufirst%3DJ.%2BM.%26aulast%3DCariou%26aufirst%3DK.%26aulast%3DDodd%26aufirst%3DR.%2BH.%26atitle%3DDevelopment%2520of%2520DANDYs%252C%2520New%25203%252C5-Diaryl-7-Azaindoles%2520Demonstrating%2520Potent%2520DYRK1A%2520Kinase%2520Inhibitory%2520Activity%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D9569%26epage%3D9585%26doi%3D10.1021%2Fjm401049v" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gelbard, H.</span>; <span class="NLM_string-name">Dewhurst, S.</span>; <span class="NLM_string-name">Goodfellow, V.</span>; <span class="NLM_string-name">Wiemann, T.</span>; <span class="NLM_string-name">Ravula, S.</span>; <span class="NLM_string-name">Loweth, C.</span></span>. <span> </span><span class="NLM_article-title">Bicyclic Heteroaryl Kinase Inhibitors and Methods of Use</span>. WO <span class="NLM_patent">2011/149950</span>, <span class="NLM_year">2011</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2011&author=H.+Gelbard&author=S.+Dewhurst&author=V.+Goodfellow&author=T.+Wiemann&author=S.+Ravula&author=C.+Loweth&title=Bicyclic+Heteroaryl+Kinase+Inhibitors+and+Methods+of+Use"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DGelbard%26aufirst%3DH.%26atitle%3DBicyclic%2520Heteroaryl%2520Kinase%2520Inhibitors%2520and%2520Methods%2520of%2520Use%26date%3D2011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Knauber, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tucker, J.</span></span> <span> </span><span class="NLM_article-title">Palladium Catalyzed Monoselective α-Arylation of Sulfones and Sulfonamides with 2,2,6,6-Tetramethylpiperidine·ZnCl·LiCl Base and Aryl Bromides</span>. <i>J. Org. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>81</i></span>,  <span class="NLM_fpage">5636</span>– <span class="NLM_lpage">5648</span>, <span class="refDoi"> DOI: 10.1021/acs.joc.6b01062</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.joc.6b01062" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=1%3ACAS%3A528%3ADC%252BC28XpslaqsLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=81&publication_year=2016&pages=5636-5648&author=T.+Knauberauthor=J.+Tucker&title=Palladium+Catalyzed+Monoselective+%CE%B1-Arylation+of+Sulfones+and+Sulfonamides+with+2%2C2%2C6%2C6-Tetramethylpiperidine%C2%B7ZnCl%C2%B7LiCl+Base+and+Aryl+Bromides&doi=10.1021%2Facs.joc.6b01062"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Palladium catalyzed monoselective α-arylation of sulfones and sulfonamides with 2,2,6,6-tetramethylpiperidine·ZnCl·LiCl base and aryl bromides</span></div><div class="casAuthors">Knauber, Thomas; Tucker, Joseph</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Organic Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">81</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">5636-5648</span>CODEN:
                <span class="NLM_cas:coden">JOCEAH</span>;
        ISSN:<span class="NLM_cas:issn">0022-3263</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A palladium catalyzed Negishi-type α-arylation of sulfones and sulfonamides with a broad range of aryl bromides has been developed.  The substrates are selectively metalated in situ with tmp·ZnCl·LiCl base (tmp: 2,2,6,6-tetramethylpiperidine) and cross-coupled in the presence of a catalyst system that is generated from Pd(dba)2 and XPhos.  Electron-deficient, electron-rich, and heterocyclic aryl bromides have been successfully cross-coupled, and sensitive functional groups are well tolerated.  Simple aryl bromides are converted overnight at 60 °C in THF while heteroaryl bromides are efficiently coupled within 2 h at 130 °C in a microwave reactor.  The desired monoarylated α-branched benzyl sulfones and sulfonamides were obtained in good yields, and over-arylation was not detected.  The procedure is ideal for late stage functionalization in parallel medicinal chem.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGowffEziLTrk7Vg90H21EOLACvtfcHk0ljdU2VQ8w9Kpw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XpslaqsLg%253D&md5=b579dc9b2864b5a4f63c31f720218676</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Facs.joc.6b01062&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.joc.6b01062%26sid%3Dliteratum%253Aachs%26aulast%3DKnauber%26aufirst%3DT.%26aulast%3DTucker%26aufirst%3DJ.%26atitle%3DPalladium%2520Catalyzed%2520Monoselective%2520%25CE%25B1-Arylation%2520of%2520Sulfones%2520and%2520Sulfonamides%2520with%25202%252C2%252C6%252C6-Tetramethylpiperidine%25C2%25B7ZnCl%25C2%25B7LiCl%2520Base%2520and%2520Aryl%2520Bromides%26jtitle%3DJ.%2520Org.%2520Chem.%26date%3D2016%26volume%3D81%26spage%3D5636%26epage%3D5648%26doi%3D10.1021%2Facs.joc.6b01062" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dai, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ai, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geng, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duan, W.</span></span> <span> </span><span class="NLM_article-title">Discovery of Substituted 1H-Pyrazolo[3,4-b]pyridine Derivatives as Potent and Selective FGFR Kinase Inhibitors</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">629</span>– <span class="NLM_lpage">634</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.6b00066</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.6b00066" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=1%3ACAS%3A528%3ADC%252BC28XmsVamsb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=629-634&author=B.+Zhaoauthor=Y.+Liauthor=P.+Xuauthor=Y.+Daiauthor=C.+Luoauthor=Y.+Sunauthor=J.+Aiauthor=M.+Gengauthor=W.+Duan&title=Discovery+of+Substituted+1H-Pyrazolo%5B3%2C4-b%5Dpyridine+Derivatives+as+Potent+and+Selective+FGFR+Kinase+Inhibitors&doi=10.1021%2Facsmedchemlett.6b00066"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Substituted 1H-Pyrazolo[3,4-b]pyridine Derivatives as Potent and Selective FGFR Kinase Inhibitors</span></div><div class="casAuthors">Zhao, Bin; Li, Yixuan; Xu, Pan; Dai, Yang; Luo, Cheng; Sun, Yiming; Ai, Jing; Geng, Meiyu; Duan, Wenhu</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">629-634</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Fibroblast growth factor receptors (FGFRs) are important targets for cancer therapy.  Herein, we describe the design, synthesis, and biol. evaluation of a novel series of 1H-pyrazolo[3,4-b]pyridine derivs. as potent and selective FGFR kinase inhibitors.  On the basis of its excellent in vitro potency and favorable pharmacokinetic properties, compd. I was selected for in vivo evaluation and showed significant antitumor activity in a FGFR1-driven H1581 xenograft model.  These results indicated that I would be a promising candidate for further drug development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpt1RCtnNDMAbVg90H21EOLACvtfcHk0ljdU2VQ8w9Kpw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XmsVamsb0%253D&md5=08cf7f710e2dfa738c7904965fae0630</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.6b00066&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.6b00066%26sid%3Dliteratum%253Aachs%26aulast%3DZhao%26aufirst%3DB.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DXu%26aufirst%3DP.%26aulast%3DDai%26aufirst%3DY.%26aulast%3DLuo%26aufirst%3DC.%26aulast%3DSun%26aufirst%3DY.%26aulast%3DAi%26aufirst%3DJ.%26aulast%3DGeng%26aufirst%3DM.%26aulast%3DDuan%26aufirst%3DW.%26atitle%3DDiscovery%2520of%2520Substituted%25201H-Pyrazolo%255B3%252C4-b%255Dpyridine%2520Derivatives%2520as%2520Potent%2520and%2520Selective%2520FGFR%2520Kinase%2520Inhibitors%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2016%26volume%3D7%26spage%3D629%26epage%3D634%26doi%3D10.1021%2Facsmedchemlett.6b00066" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ye, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.</span></span> <span> </span><span class="NLM_article-title">Design and Synthesis of 1,4,5,6-Tetrahydropyrrolo[3,4-c]Pyrazoles and Pyrazolo[3,4-b]Pyridines for Aurora-A Kinase Inhibitors</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">4273</span>– <span class="NLM_lpage">4278</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2010.04.083</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=10.1016%2Fj.bmcl.2010.04.083" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=20621733" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=1%3ACAS%3A528%3ADC%252BC3cXotVKms7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2010&pages=4273-4278&author=J.+Shiauthor=G.+Xuauthor=W.+Zhuauthor=H.+Yeauthor=S.+Yangauthor=Y.+Luoauthor=J.+Hanauthor=J.+Yangauthor=R.+Liauthor=Y.+Weiauthor=L.+Chen&title=Design+and+Synthesis+of+1%2C4%2C5%2C6-Tetrahydropyrrolo%5B3%2C4-c%5DPyrazoles+and+Pyrazolo%5B3%2C4-b%5DPyridines+for+Aurora-A+Kinase+Inhibitors&doi=10.1016%2Fj.bmcl.2010.04.083"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Design and synthesis of 1,4,5,6-tetrahydropyrrolo[3,4-c]pyrazoles and pyrazolo[3,4-b]pyridines for Aurora-A kinase inhibitors</span></div><div class="casAuthors">Shi, Jianyou; Xu, Guobin; Zhu, Wei; Ye, Haoyu; Yang, Shengyong; Luo, Youfu; Han, Jing; Yang, Jincheng; Li, Rui; Wei, Yuquan; Chen, Lijuan</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">4273-4278</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Two series of 3-aminopyrazole compds. including 24 1,4,5,6-tetrahydropyrrolo[3,4-c]pyrazoles and 16 pyrazolo[3,4-b]pyridines were synthesized and evaluated against HCT116, A549, and A2780 tumor cell lines.  Among them, three compds. were found to have the ideal anti-proliferative activities in vitro.  Docking expts. showed that the novel pyrazolo[3,4-b]pyridines share the similar interaction mode with Aurora-A kinase as PHA739358 (I).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoc3mQNCWrj_7Vg90H21EOLACvtfcHk0lgjk7g54P9d4A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXotVKms7Y%253D&md5=edb892e4dec485298d9ed9ff83d345c9</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2010.04.083&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2010.04.083%26sid%3Dliteratum%253Aachs%26aulast%3DShi%26aufirst%3DJ.%26aulast%3DXu%26aufirst%3DG.%26aulast%3DZhu%26aufirst%3DW.%26aulast%3DYe%26aufirst%3DH.%26aulast%3DYang%26aufirst%3DS.%26aulast%3DLuo%26aufirst%3DY.%26aulast%3DHan%26aufirst%3DJ.%26aulast%3DYang%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DR.%26aulast%3DWei%26aufirst%3DY.%26aulast%3DChen%26aufirst%3DL.%26atitle%3DDesign%2520and%2520Synthesis%2520of%25201%252C4%252C5%252C6-Tetrahydropyrrolo%255B3%252C4-c%255DPyrazoles%2520and%2520Pyrazolo%255B3%252C4-b%255DPyridines%2520for%2520Aurora-A%2520Kinase%2520Inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2010%26volume%3D20%26spage%3D4273%26epage%3D4278%26doi%3D10.1016%2Fj.bmcl.2010.04.083" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Van
Mileghem, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Egle, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilles, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Veryser, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Meervelt, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Borggraeve, W. M.</span></span> <span> </span><span class="NLM_article-title">Carbonylation as a Novel Method for the Assembly of Pyrazine Based Oligoamide Alpha-Helix Mimetics</span>. <i>Org. Biomol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">373</span>– <span class="NLM_lpage">378</span>, <span class="refDoi"> DOI: 10.1039/c6ob02358d</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=10.1039%2Fc6ob02358d" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=27910980" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvFKntLrP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2017&pages=373-378&author=S.+Van%0AMileghemauthor=B.+Egleauthor=P.+Gillesauthor=C.+Veryserauthor=L.+Van+Meerveltauthor=W.+M.+De+Borggraeve&title=Carbonylation+as+a+Novel+Method+for+the+Assembly+of+Pyrazine+Based+Oligoamide+Alpha-Helix+Mimetics&doi=10.1039%2Fc6ob02358d"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Carbonylation as a novel method for the assembly of pyrazine based oligoamide alpha-helix mimetics</span></div><div class="casAuthors">Van Mileghem, Seger; Egle, Brecht; Gilles, Philippe; Veryser, Cedrick; Van Meervelt, Luc; De Borggraeve, Wim M.</div><div class="citationInfo"><span class="NLM_cas:title">Organic & Biomolecular Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">373-378</span>CODEN:
                <span class="NLM_cas:coden">OBCRAK</span>;
        ISSN:<span class="NLM_cas:issn">1477-0520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">The design and synthesis of oligoamide α-helix peptidomimetics is reported.  The oligoamide type systems are prepd. in a modular fashion by coupling the monomers using palladium-catalyzed carbonylation chem.  This enabled us to use substrates with a low nucleophilicity, leading to previously unreported pyrazine based oligoamide α-helix mimetics.  The proof of principle is given by synthesizing a small set of compds.  Various end-capping groups were introduced and also a mixed multimer was successfully prepd.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpBhPQhYaXR_rVg90H21EOLACvtfcHk0lgjk7g54P9d4A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvFKntLrP&md5=aa4a28969f01e6076e7f9e5cf700e16f</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1039%2Fc6ob02358d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc6ob02358d%26sid%3Dliteratum%253Aachs%26aulast%3DVan%2BMileghem%26aufirst%3DS.%26aulast%3DEgle%26aufirst%3DB.%26aulast%3DGilles%26aufirst%3DP.%26aulast%3DVeryser%26aufirst%3DC.%26aulast%3DVan%2BMeervelt%26aufirst%3DL.%26aulast%3DDe%2BBorggraeve%26aufirst%3DW.%2BM.%26atitle%3DCarbonylation%2520as%2520a%2520Novel%2520Method%2520for%2520the%2520Assembly%2520of%2520Pyrazine%2520Based%2520Oligoamide%2520Alpha-Helix%2520Mimetics%26jtitle%3DOrg.%2520Biomol.%2520Chem.%26date%3D2017%26volume%3D15%26spage%3D373%26epage%3D378%26doi%3D10.1039%2Fc6ob02358d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Maccormick, S.</span>; <span class="NLM_string-name">Storck, P.-H.</span>; <span class="NLM_string-name">Mertimore, M.</span>; <span class="NLM_string-name">Charrier, J.-D.</span>; <span class="NLM_string-name">Knegtel, R.</span>; <span class="NLM_string-name">Young, S.</span>; <span class="NLM_string-name">Pinder, J.</span>; <span class="NLM_string-name">Durrant, S.</span></span> <span> </span><span class="NLM_article-title">Compounds Useful as Inhibitors of ATR Kinase</span>. WO <span class="NLM_patent">2012/178123</span>, <span class="NLM_year">2012</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2012&author=S.+Maccormick&author=P.-H.+Storck&author=M.+Mertimore&author=J.-D.+Charrier&author=R.+Knegtel&author=S.+Young&author=J.+Pinder&author=S.+Durrant&title=Compounds+Useful+as+Inhibitors+of+ATR+Kinase"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DMaccormick%26aufirst%3DS.%26atitle%3DCompounds%2520Useful%2520as%2520Inhibitors%2520of%2520ATR%2520Kinase%26date%3D2012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gelbard, H.</span>; <span class="NLM_string-name">Dewhurst, S.</span>; <span class="NLM_string-name">Goodfellow, V.</span>; <span class="NLM_string-name">Wiemann, T.</span>; <span class="NLM_string-name">Bennet, D.</span></span> <span> </span><span class="NLM_article-title">MLK Inhibitors and Methods of Use</span>. WO <span class="NLM_patent">2010/068483</span>, <span class="NLM_year">2010</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2010&author=H.+Gelbard&author=S.+Dewhurst&author=V.+Goodfellow&author=T.+Wiemann&author=D.+Bennet&title=MLK+Inhibitors+and+Methods+of+Use"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DGelbard%26aufirst%3DH.%26atitle%3DMLK%2520Inhibitors%2520and%2520Methods%2520of%2520Use%26date%3D2010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sorrell, F. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szklarz, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abdul Azeez, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elkins, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szklarz, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Azeez, K. R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elkins, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span> <span> </span><span class="NLM_article-title">Family-wide Structural Analysis of Human Numb-Associated Protein Kinases</span>. <i>Struct. Des.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">401</span>– <span class="NLM_lpage">411</span>, <span class="refDoi"> DOI: 10.1016/j.str.2015.12.015</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=10.1016%2Fj.str.2015.12.015" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=26853940" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=1%3ACAS%3A528%3ADC%252BC28XitFCktLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2016&pages=401-411&author=F.+J.+Sorrellauthor=M.+Szklarzauthor=K.+R.+Abdul+Azeezauthor=J.+M.+Elkinsauthor=S.+Knappauthor=M.+Szklarzauthor=K.+R.+A.+Azeezauthor=J.+M.+Elkinsauthor=S.+Knapp&title=Family-wide+Structural+Analysis+of+Human+Numb-Associated+Protein+Kinases&doi=10.1016%2Fj.str.2015.12.015"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Family-wide Structural Analysis of Human Numb-Associated Protein Kinases</span></div><div class="casAuthors">Sorrell, Fiona J.; Szklarz, Marta; Abdul Azeez, Kamal R.; Elkins, Jon M.; Knapp, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Structure (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">401-411</span>CODEN:
                <span class="NLM_cas:coden">STRUE6</span>;
        ISSN:<span class="NLM_cas:issn">0969-2126</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">The highly diverse Numb-assocd. kinase (NAK) family has been linked to broad cellular functions including receptor-mediated endocytosis, Notch pathway modulation, osteoblast differentiation, and dendrite morphogenesis.  Consequently, NAK kinases play a key role in a diverse range of diseases from Parkinson's and prostate cancer to HIV.  Due to the plasticity of this kinase family, NAK kinases are often inhibited by approved or investigational drugs and have been assocd. with side effects, but they are also potential drug targets.  The presence of cysteine residues in some NAK family members provides the possibility for selective targeting via covalent inhibition.  Here we report the first high-resoln. structures of kinases AAK1 and BIKE in complex with two drug candidates.  The presented data allow a comprehensive structural characterization of the NAK kinase family and provide the basis for rational design of selective NAK inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGryZi0QmT8YH7Vg90H21EOLACvtfcHk0liqbgqkBT3c2w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitFCktLY%253D&md5=6a353c993ec9f2b7152e459d72bab9d3</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1016%2Fj.str.2015.12.015&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.str.2015.12.015%26sid%3Dliteratum%253Aachs%26aulast%3DSorrell%26aufirst%3DF.%2BJ.%26aulast%3DSzklarz%26aufirst%3DM.%26aulast%3DAbdul%2BAzeez%26aufirst%3DK.%2BR.%26aulast%3DElkins%26aufirst%3DJ.%2BM.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DSzklarz%26aufirst%3DM.%26aulast%3DAzeez%26aufirst%3DK.%2BR.%2BA.%26aulast%3DElkins%26aufirst%3DJ.%2BM.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DFamily-wide%2520Structural%2520Analysis%2520of%2520Human%2520Numb-Associated%2520Protein%2520Kinases%26jtitle%3DStruct.%2520Des.%26date%3D2016%26volume%3D24%26spage%3D401%26epage%3D411%26doi%3D10.1016%2Fj.str.2015.12.015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fabian, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Biggs, W. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Treiber, D. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Atteridge, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Azimioara, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benedetti, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carter, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciceri, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edeen, P. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Floyd, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ford, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galvin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerlach, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grotzfeld, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herrgard, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Insko, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Insko, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lélias, J.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mehta, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Milanov, Z. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Velasco, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wodicka, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, H. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zarrinkar, P. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lockhart, D. J.</span></span> <span> </span><span class="NLM_article-title">A small molecule-kinase interaction map for clinical kinase inhibitors</span>. <i>Nat. Biotechnol.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">329</span>– <span class="NLM_lpage">336</span>, <span class="refDoi"> DOI: 10.1038/nbt1068</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=10.1038%2Fnbt1068" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=15711537" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=1%3ACAS%3A528%3ADC%252BD2MXitF2nt7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2005&pages=329-336&author=M.+A.+Fabianauthor=W.+H.+Biggsauthor=D.+K.+Treiberauthor=C.+E.+Atteridgeauthor=M.+D.+Azimioaraauthor=M.+G.+Benedettiauthor=T.+A.+Carterauthor=P.+Ciceriauthor=P.+T.+Edeenauthor=M.+Floydauthor=J.+M.+Fordauthor=M.+Galvinauthor=J.+L.+Gerlachauthor=R.+M.+Grotzfeldauthor=S.+Herrgardauthor=D.+E.+Inskoauthor=M.+A.+Inskoauthor=A.+G.+Laiauthor=J.-M.+L%C3%A9liasauthor=S.+A.+Mehtaauthor=Z.+V.+Milanovauthor=A.+M.+Velascoauthor=L.+M.+Wodickaauthor=H.+K.+Patelauthor=P.+P.+Zarrinkarauthor=D.+J.+Lockhart&title=A+small+molecule-kinase+interaction+map+for+clinical+kinase+inhibitors&doi=10.1038%2Fnbt1068"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">A small molecule-kinase interaction map for clinical kinase inhibitors</span></div><div class="casAuthors">Fabian, Miles A.; Biggs, William H.; Treiber, Daniel K.; Atteridge, Corey E.; Azimioara, Mihai D.; Benedetti, Michael G.; Carter, Todd A.; Ciceri, Pietro; Edeen, Philip T.; Floyd, Mark; Ford, Julia M.; Galvin, Margaret; Gerlach, Jay L.; Grotzfeld, Robert M.; Herrgard, Sanna; Insko, Darren E.; Insko, Michael A.; Lai, Andiliy G.; Lelias, Jean-Michel; Mehta, Shamal A.; Milanov, Zdravko V.; Velasco, Anne Marie; Wodicka, Lisa M.; Patel, Hitesh K.; Zarrinkar, Patrick P.; Lockhart, David J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">329-336</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Kinase inhibitors show great promise as a new class of therapeutics.  Here the authors describe an efficient way to det. kinase inhibitor specificity by measuring binding of small mols. to the ATP site of kinases.  The authors have profiled 20 kinase inhibitors, including 16 that are approved drugs or in clin. development, against a panel of 119 protein kinases.  The authors find that specificity varies widely and is not strongly correlated with chem. structure or the identity of the intended target.  Many novel interactions were identified, including tight binding of the p38 inhibitor BIRB-796 to an imatinib-resistant variant of the ABL kinase, and binding of imatinib to the SRC-family kinase LCK.  The authors also show that mutations in the epidermal growth factor receptor (EGFR) found in gefitinib-responsive patients do not affect the binding affinity of gefitinib or erlotinib.  Our results represent a systematic small mol.-protein interaction map for clin. compds. across a large no. of related proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqA0vNmMOXrnrVg90H21EOLACvtfcHk0liqbgqkBT3c2w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXitF2nt7w%253D&md5=6aedd5ceb8f77cd26ee50425dcec7bdf</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1038%2Fnbt1068&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt1068%26sid%3Dliteratum%253Aachs%26aulast%3DFabian%26aufirst%3DM.%2BA.%26aulast%3DBiggs%26aufirst%3DW.%2BH.%26aulast%3DTreiber%26aufirst%3DD.%2BK.%26aulast%3DAtteridge%26aufirst%3DC.%2BE.%26aulast%3DAzimioara%26aufirst%3DM.%2BD.%26aulast%3DBenedetti%26aufirst%3DM.%2BG.%26aulast%3DCarter%26aufirst%3DT.%2BA.%26aulast%3DCiceri%26aufirst%3DP.%26aulast%3DEdeen%26aufirst%3DP.%2BT.%26aulast%3DFloyd%26aufirst%3DM.%26aulast%3DFord%26aufirst%3DJ.%2BM.%26aulast%3DGalvin%26aufirst%3DM.%26aulast%3DGerlach%26aufirst%3DJ.%2BL.%26aulast%3DGrotzfeld%26aufirst%3DR.%2BM.%26aulast%3DHerrgard%26aufirst%3DS.%26aulast%3DInsko%26aufirst%3DD.%2BE.%26aulast%3DInsko%26aufirst%3DM.%2BA.%26aulast%3DLai%26aufirst%3DA.%2BG.%26aulast%3DL%25C3%25A9lias%26aufirst%3DJ.-M.%26aulast%3DMehta%26aufirst%3DS.%2BA.%26aulast%3DMilanov%26aufirst%3DZ.%2BV.%26aulast%3DVelasco%26aufirst%3DA.%2BM.%26aulast%3DWodicka%26aufirst%3DL.%2BM.%26aulast%3DPatel%26aufirst%3DH.%2BK.%26aulast%3DZarrinkar%26aufirst%3DP.%2BP.%26aulast%3DLockhart%26aufirst%3DD.%2BJ.%26atitle%3DA%2520small%2520molecule-kinase%2520interaction%2520map%2520for%2520clinical%2520kinase%2520inhibitors%26jtitle%3DNat.%2520Biotechnol.%26date%3D2005%26volume%3D23%26spage%3D329%26epage%3D336%26doi%3D10.1038%2Fnbt1068" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rodriguez-Madoz, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernal-Rubio, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaminski, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boyd, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fernandez-Sesma, A.</span></span> <span> </span><span class="NLM_article-title">Dengue Virus Inhibits the Production of Type I Interferon in Primary Human Dendritic Cells</span>. <i>J. Virol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>84</i></span>,  <span class="NLM_fpage">4845</span>– <span class="NLM_lpage">4850</span>, <span class="refDoi"> DOI: 10.1128/jvi.02514-09</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=10.1128%2Fjvi.02514-09" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=20164230" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=1%3ACAS%3A528%3ADC%252BC3cXlt1KhsLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=84&publication_year=2010&pages=4845-4850&author=J.+R.+Rodriguez-Madozauthor=D.+Bernal-Rubioauthor=D.+Kaminskiauthor=K.+Boydauthor=A.+Fernandez-Sesma&title=Dengue+Virus+Inhibits+the+Production+of+Type+I+Interferon+in+Primary+Human+Dendritic+Cells&doi=10.1128%2Fjvi.02514-09"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Dengue virus inhibits the production of type I interferon in primary human dendritic cells</span></div><div class="casAuthors">Rodriguez-Madoz, Juan R.; Bernal-Rubio, Dabeiba; Kaminski, Dorota; Boyd, Kelley; Fernandez-Sesma, Ana</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Virology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">84</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">4845-4850</span>CODEN:
                <span class="NLM_cas:coden">JOVIAM</span>;
        ISSN:<span class="NLM_cas:issn">0022-538X</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Dengue virus (DENV) infects human immune cells in vitro and likely infects dendritic cells (DCs) in vivo.  DENV-2 productive infection induces activation and release of high levels of chemokines and proinflammatory cytokines in monocyte-derived DCs (moDCs), with the notable exception of alpha/beta interferon (IFN-α/β).  Interestingly, DENV-2-infected moDCs fail to prime T cells, most likely due to the lack of IFN-α/β released by moDCs, since this effect was reversed by addn. of exogenous IFN-β.  Together, the authors' data show that inhibition of IFN-α/β prodn. by DENV in primary human moDCs is a novel mechanism of immune evasion.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpFUNnZLnic7bVg90H21EOLACvtfcHk0liqbgqkBT3c2w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXlt1KhsLk%253D&md5=7d751330acbca6445989821c48ffbe56</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1128%2Fjvi.02514-09&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252Fjvi.02514-09%26sid%3Dliteratum%253Aachs%26aulast%3DRodriguez-Madoz%26aufirst%3DJ.%2BR.%26aulast%3DBernal-Rubio%26aufirst%3DD.%26aulast%3DKaminski%26aufirst%3DD.%26aulast%3DBoyd%26aufirst%3DK.%26aulast%3DFernandez-Sesma%26aufirst%3DA.%26atitle%3DDengue%2520Virus%2520Inhibits%2520the%2520Production%2520of%2520Type%2520I%2520Interferon%2520in%2520Primary%2520Human%2520Dendritic%2520Cells%26jtitle%3DJ.%2520Virol.%26date%3D2010%26volume%3D84%26spage%3D4845%26epage%3D4850%26doi%3D10.1128%2Fjvi.02514-09" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Karaman, M. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herrgard, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Treiber, D. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gallant, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Atteridge, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campbell, B. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, K. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciceri, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, M. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edeen, P. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Faraoni, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Floyd, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hunt, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lockhart, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Milanov, Z. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morrison, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pallares, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, H. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pritchard, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wodicka, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zarrinkar, P. P.</span></span> <span> </span><span class="NLM_article-title">A Quantitative Analysis of Kinase Inhibitor Selectivity</span>. <i>Nat. Biotechnol.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">127</span>– <span class="NLM_lpage">132</span>, <span class="refDoi"> DOI: 10.1038/nbt1358</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=10.1038%2Fnbt1358" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=18183025" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=1%3ACAS%3A528%3ADC%252BD1cXisFGlsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2008&pages=127-132&author=M.+W.+Karamanauthor=S.+Herrgardauthor=D.+K.+Treiberauthor=P.+Gallantauthor=C.+E.+Atteridgeauthor=B.+T.+Campbellauthor=K.+W.+Chanauthor=P.+Ciceriauthor=M.+I.+Davisauthor=P.+T.+Edeenauthor=R.+Faraoniauthor=M.+Floydauthor=J.+P.+Huntauthor=D.+J.+Lockhartauthor=Z.+V.+Milanovauthor=M.+J.+Morrisonauthor=G.+Pallaresauthor=H.+K.+Patelauthor=S.+Pritchardauthor=L.+M.+Wodickaauthor=P.+P.+Zarrinkar&title=A+Quantitative+Analysis+of+Kinase+Inhibitor+Selectivity&doi=10.1038%2Fnbt1358"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">A quantitative analysis of kinase inhibitor selectivity</span></div><div class="casAuthors">Karaman, Mazen W.; Herrgard, Sanna; Treiber, Daniel K.; Gallant, Paul; Atteridge, Corey E.; Campbell, Brian T.; Chan, Katrina W.; Ciceri, Pietro; Davis, Mindy I.; Edeen, Philip T.; Faraoni, Raffaella; Floyd, Mark; Hunt, Jeremy P.; Lockhart, Daniel J.; Milanov, Zdravko V.; Morrison, Michael J.; Pallares, Gabriel; Patel, Hitesh K.; Pritchard, Stephanie; Wodicka, Lisa M.; Zarrinkar, Patrick P.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">127-132</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Kinase inhibitors are a new class of therapeutics with a propensity to inhibit multiple targets.  The biol. consequences of multikinase activity are poorly defined, and an important step toward understanding the relationship between selectivity, efficacy and safety is the exploration of how inhibitors interact with the human kinome.  The authors present interaction maps for 38 kinase inhibitors across a panel of 317 kinases representing >50% of the predicted human protein kinome.  The data constitute the most comprehensive study of kinase inhibitor selectivity to date and reveal a wide diversity of interaction patterns.  To enable a global anal. of the results, the authors introduce the concept of a selectivity score as a general tool to quantify and differentiate the obsd. interaction patterns.  The authors further investigate the impact of panel size and find that small assay panels do not provide a robust measure of selectivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpiArJSq2_lPrVg90H21EOLACvtfcHk0lhy-LSEL3DDMg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXisFGlsQ%253D%253D&md5=346265d412853ced636ad4128ed8a76f</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1038%2Fnbt1358&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt1358%26sid%3Dliteratum%253Aachs%26aulast%3DKaraman%26aufirst%3DM.%2BW.%26aulast%3DHerrgard%26aufirst%3DS.%26aulast%3DTreiber%26aufirst%3DD.%2BK.%26aulast%3DGallant%26aufirst%3DP.%26aulast%3DAtteridge%26aufirst%3DC.%2BE.%26aulast%3DCampbell%26aufirst%3DB.%2BT.%26aulast%3DChan%26aufirst%3DK.%2BW.%26aulast%3DCiceri%26aufirst%3DP.%26aulast%3DDavis%26aufirst%3DM.%2BI.%26aulast%3DEdeen%26aufirst%3DP.%2BT.%26aulast%3DFaraoni%26aufirst%3DR.%26aulast%3DFloyd%26aufirst%3DM.%26aulast%3DHunt%26aufirst%3DJ.%2BP.%26aulast%3DLockhart%26aufirst%3DD.%2BJ.%26aulast%3DMilanov%26aufirst%3DZ.%2BV.%26aulast%3DMorrison%26aufirst%3DM.%2BJ.%26aulast%3DPallares%26aufirst%3DG.%26aulast%3DPatel%26aufirst%3DH.%2BK.%26aulast%3DPritchard%26aufirst%3DS.%26aulast%3DWodicka%26aufirst%3DL.%2BM.%26aulast%3DZarrinkar%26aufirst%3DP.%2BP.%26atitle%3DA%2520Quantitative%2520Analysis%2520of%2520Kinase%2520Inhibitor%2520Selectivity%26jtitle%3DNat.%2520Biotechnol.%26date%3D2008%26volume%3D26%26spage%3D127%26epage%3D132%26doi%3D10.1038%2Fnbt1358" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Trott, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olson, A. J.</span></span> <span> </span><span class="NLM_article-title">AutoDock Vina: improving the speed and accuracy of docking with a new scoring function, efficient optimization, and multithreading</span>. <i>J. Comput. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>31</i></span>,  <span class="NLM_fpage">455</span>– <span class="NLM_lpage">461</span>, <span class="refDoi"> DOI: 10.1002/jcc.21334</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=10.1002%2Fjcc.21334" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=19499576" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsFGnur3O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2010&pages=455-461&author=O.+Trottauthor=A.+J.+Olson&title=AutoDock+Vina%3A+improving+the+speed+and+accuracy+of+docking+with+a+new+scoring+function%2C+efficient+optimization%2C+and+multithreading&doi=10.1002%2Fjcc.21334"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">AutoDock Vina: Improving the speed and accuracy of docking with a new scoring function, efficient optimization, and multithreading</span></div><div class="casAuthors">Trott, Oleg; Olson, Arthur J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">455-461</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">AutoDock Vina, a new program for mol. docking and virtual screening, is presented.  AutoDock Vina achieves an approx. 2 orders of magnitude speed-up compared with the mol. docking software previously developed in the authors' lab (AutoDock 4), while also significantly improving the accuracy of the binding mode predictions, judging by the authors' tests on the training set used in AutoDock 4 development.  Further speed-up is achieved from parallelism, by using multithreading on multicore machines.  AutoDock Vina automatically calcs. the grid maps and clusters the results in a way transparent to the user.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpLecOIkUxIlrVg90H21EOLACvtfcHk0lhy-LSEL3DDMg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsFGnur3O&md5=c6974af8a1235f7aa09918d3e6f70dc4</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1002%2Fjcc.21334&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.21334%26sid%3Dliteratum%253Aachs%26aulast%3DTrott%26aufirst%3DO.%26aulast%3DOlson%26aufirst%3DA.%2BJ.%26atitle%3DAutoDock%2520Vina%253A%2520improving%2520the%2520speed%2520and%2520accuracy%2520of%2520docking%2520with%2520a%2520new%2520scoring%2520function%252C%2520efficient%2520optimization%252C%2520and%2520multithreading%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2010%26volume%3D31%26spage%3D455%26epage%3D461%26doi%3D10.1002%2Fjcc.21334" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Case, D. A.</span>; <span class="NLM_string-name">Ben-Shalom, I.</span>; <span class="NLM_string-name">Brozell, S. R.</span>; <span class="NLM_string-name">Cerutti, D. S.</span>; <span class="NLM_string-name">Cheatham, T.</span>; <span class="NLM_string-name">Cruzeiro, V. W. D.</span>; <span class="NLM_string-name">Darden, T. A.</span>; <span class="NLM_string-name">Duke, R. A.</span>; <span class="NLM_string-name">Ghoreishi, D.</span>; <span class="NLM_string-name">Gilson, M. K.</span>; <span class="NLM_string-name">Gohlke, H.</span>; <span class="NLM_string-name">Goetz, A.
W.</span>; <span class="NLM_string-name">Greene, D.</span>; <span class="NLM_string-name">Harris, R.</span>; <span class="NLM_string-name">Homeyer, N.</span>; <span class="NLM_string-name">Huang, Y.</span>; <span class="NLM_string-name">Izadi, S.</span>; <span class="NLM_string-name">Kovalenko, A.</span>; <span class="NLM_string-name">Kurtzman, T.</span>; <span class="NLM_string-name">Lee, T. S.</span>; <span class="NLM_string-name">LeGrand, S.</span>; <span class="NLM_string-name">Li, P.</span>; <span class="NLM_string-name">Lin, C.</span>; <span class="NLM_string-name">Liu, J.</span>; <span class="NLM_string-name">Luchko, T.</span>; <span class="NLM_string-name">Luo, R.</span>; <span class="NLM_string-name">Mermelstein, D. J.</span>; <span class="NLM_string-name">Merz, K. M.</span>; <span class="NLM_string-name">Miao, Y.</span>; <span class="NLM_string-name">Monard, G.</span>; <span class="NLM_string-name">Nguyen, C.</span>; <span class="NLM_string-name">Nguyen, H.</span>; <span class="NLM_string-name">Omelyan, I.</span>; <span class="NLM_string-name">Onufriev, A.</span>; <span class="NLM_string-name">Pan, F.</span>; <span class="NLM_string-name">Qi, R.</span>; <span class="NLM_string-name">Roe, D. R.</span>; <span class="NLM_string-name">Roitberg, A.</span>; <span class="NLM_string-name">Sagui, C.</span>; <span class="NLM_string-name">Schott-Verdugo, S.</span>; <span class="NLM_string-name">Shen, J.</span>; <span class="NLM_string-name">Simmerling, C. L.</span>; <span class="NLM_string-name">Smith, J.</span>; <span class="NLM_string-name">SalomonFerrer, R.</span>; <span class="NLM_string-name">Swails, J.</span>; <span class="NLM_string-name">Walker, R. C.</span>; <span class="NLM_string-name">Wang, J.</span>; <span class="NLM_string-name">Wei, H.</span>; <span class="NLM_string-name">Wolf, R. M.</span>; <span class="NLM_string-name">Wu, X.</span>; <span class="NLM_string-name">Xiao, L.</span>; <span class="NLM_string-name">York, D. M.</span>; <span class="NLM_string-name">Kollman, P. A.</span></span> <i>AMBER 2018</i>; <span class="NLM_publisher-loc">San Francisco</span>, <span class="NLM_year">2018</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2018&author=D.+A.+Case&author=I.+Ben-Shalom&author=S.+R.+Brozell&author=D.+S.+Cerutti&author=T.+Cheatham&author=V.+W.+D.+Cruzeiro&author=T.+A.+Darden&author=R.+A.+Duke&author=D.+Ghoreishi&author=M.+K.+Gilson&author=H.+Gohlke&author=A.%0AW.+Goetz&author=D.+Greene&author=R.+Harris&author=N.+Homeyer&author=Y.+Huang&author=S.+Izadi&author=A.+Kovalenko&author=T.+Kurtzman&author=T.+S.+Lee&author=S.+LeGrand&author=P.+Li&author=C.+Lin&author=J.+Liu&author=T.+Luchko&author=R.+Luo&author=D.+J.+Mermelstein&author=K.+M.+Merz&author=Y.+Miao&author=G.+Monard&author=C.+Nguyen&author=H.+Nguyen&author=I.+Omelyan&author=A.+Onufriev&author=F.+Pan&author=R.+Qi&author=D.+R.+Roe&author=A.+Roitberg&author=C.+Sagui&author=S.+Schott-Verdugo&author=J.+Shen&author=C.+L.+Simmerling&author=J.+Smith&author=R.+SalomonFerrer&author=J.+Swails&author=R.+C.+Walker&author=J.+Wang&author=H.+Wei&author=R.+M.+Wolf&author=X.+Wu&author=L.+Xiao&author=D.+M.+York&author=P.+A.+Kollman&title=AMBER+2018"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26aulast%3DCase%26aufirst%3DD.%2BA.%26jtitle%3DAMBER%25202018%26date%3D2018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Genheden, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ryde, U.</span></span> <span> </span><span class="NLM_article-title">The MM/PBSA and MM/GBSA Methods to Estimate Ligand-Binding Affinities</span>. <i>Expert Opin. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">449</span>– <span class="NLM_lpage">461</span>, <span class="refDoi"> DOI: 10.1517/17460441.2015.1032936</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=10.1517%2F17460441.2015.1032936" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=25835573" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=1%3ACAS%3A528%3ADC%252BC2MXntFGktr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=449-461&author=S.+Genhedenauthor=U.+Ryde&title=The+MM%2FPBSA+and+MM%2FGBSA+Methods+to+Estimate+Ligand-Binding+Affinities&doi=10.1517%2F17460441.2015.1032936"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">The MM/PBSA and MM/GBSA methods to estimate ligand-binding affinities</span></div><div class="casAuthors">Genheden, Samuel; Ryde, Ulf</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Drug Discovery</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">449-461</span>CODEN:
                <span class="NLM_cas:coden">EODDBX</span>;
        ISSN:<span class="NLM_cas:issn">1746-0441</span>.
    
            (<span class="NLM_cas:orgname">Informa Healthcare</span>)
        </div><div class="casAbstract">Introduction: The mol. mechanics energies combined with the Poisson-Boltzmann or generalized Born and surface area continuum solvation (MM/PBSA and MM/GBSA) methods are popular approaches to est. the free energy of the binding of small ligands to biol. macromols.  They are typically based on mol. dynamics simulations of the receptor-ligand complex and are therefore intermediate in both accuracy and computational effort between empirical scoring and strict alchem. perturbation methods.  They have been applied to a large no. of systems with varying success.  Areas covered: The authors review the use of MM/PBSA and MM/GBSA methods to calc. ligand-binding affinities, with an emphasis on calibration, testing and validation, as well as attempts to improve the methods, rather than on specific applications.  Expert opinion: MM/PBSA and MM/GBSA are attractive approaches owing to their modular nature and that they do not require calcns. on a training set.  They have been used successfully to reproduce and rationalize exptl. findings and to improve the results of virtual screening and docking.  However, they contain several crude and questionable approxns., for example, the lack of conformational entropy and information about the no. and free energy of water mols. in the binding site.  Moreover, there are many variants of the method and their performance varies strongly with the tested system.  Likewise, most attempts to ameliorate the methods with more accurate approaches, for example, quantum-mech. calcns., polarizable force fields or improved solvation have deteriorated the results.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpTBKte1pAZR7Vg90H21EOLACvtfcHk0lhy-LSEL3DDMg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXntFGktr8%253D&md5=b123b88809f275564f95a2271ebd159f</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1517%2F17460441.2015.1032936&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1517%252F17460441.2015.1032936%26sid%3Dliteratum%253Aachs%26aulast%3DGenheden%26aufirst%3DS.%26aulast%3DRyde%26aufirst%3DU.%26atitle%3DThe%2520MM%252FPBSA%2520and%2520MM%252FGBSA%2520Methods%2520to%2520Estimate%2520Ligand-Binding%2520Affinities%26jtitle%3DExpert%2520Opin.%2520Drug%2520Discovery%26date%3D2015%26volume%3D10%26spage%3D449%26epage%3D461%26doi%3D10.1517%2F17460441.2015.1032936" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pettersen, E. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goddard, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Couch, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greenblatt, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meng, E. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferrin, T. E.</span></span> <span> </span><span class="NLM_article-title">UCSF Chimera?A Visualization System for Exploratory Research and Analysis</span>. <i>J. Comput. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">1605</span>– <span class="NLM_lpage">1612</span>, <span class="refDoi"> DOI: 10.1002/jcc.20084</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=10.1002%2Fjcc.20084" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=15264254" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=1%3ACAS%3A528%3ADC%252BD2cXmvVOhsbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2004&pages=1605-1612&author=E.+F.+Pettersenauthor=T.+D.+Goddardauthor=C.+C.+Huangauthor=G.+S.+Couchauthor=D.+M.+Greenblattauthor=E.+C.+Mengauthor=T.+E.+Ferrin&title=UCSF+Chimera%3FA+Visualization+System+for+Exploratory+Research+and+Analysis&doi=10.1002%2Fjcc.20084"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">UCSF Chimera-A visualization system for exploratory research and analysis</span></div><div class="casAuthors">Pettersen, Eric F.; Goddard, Thomas D.; Huang, Conrad C.; Couch, Gregory S.; Greenblatt, Daniel M.; Meng, Elaine C.; Ferrin, Thomas E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">1605-1612</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">The design, implementation, and capabilities of an extensible visualization system, UCSF Chimera, are discussed.  Chimera is segmented into a core that provides basic services and visualization, and extensions that provide most higher level functionality.  This architecture ensures that the extension mechanism satisfies the demands of outside developers who wish to incorporate new features.  Two unusual extensions are presented: Multiscale, which adds the ability to visualize large-scale mol. assemblies such as viral coats, and Collab., which allows researchers to share a Chimera session interactively despite being at sep. locales.  Other extensions include Multalign Viewer, for showing multiple sequence alignments and assocd. structures; ViewDock, for screening docked ligand orientations; Movie, for replaying mol. dynamics trajectories; and Vol. Viewer, for display and anal. of volumetric data.  A discussion of the usage of Chimera in real-world situations is given, along with anticipated future directions.  Chimera includes full user documentation, is free to academic and nonprofit users, and is available for Microsoft Windows, Linux, Apple Mac OS X, SGI IRIX, and HP Tru64 Unix from http://www.cgl.ucsf.edu/chimera/.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrFmD4M5S0cPbVg90H21EOLACvtfcHk0lgbg6fzigr7jg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXmvVOhsbs%253D&md5=944b175f440c1ff323705987cf937ee7</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1002%2Fjcc.20084&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.20084%26sid%3Dliteratum%253Aachs%26aulast%3DPettersen%26aufirst%3DE.%2BF.%26aulast%3DGoddard%26aufirst%3DT.%2BD.%26aulast%3DHuang%26aufirst%3DC.%2BC.%26aulast%3DCouch%26aufirst%3DG.%2BS.%26aulast%3DGreenblatt%26aufirst%3DD.%2BM.%26aulast%3DMeng%26aufirst%3DE.%2BC.%26aulast%3DFerrin%26aufirst%3DT.%2BE.%26atitle%3DUCSF%2520Chimera%253FA%2520Visualization%2520System%2520for%2520Exploratory%2520Research%2520and%2520Analysis%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2004%26volume%3D25%26spage%3D1605%26epage%3D1612%26doi%3D10.1002%2Fjcc.20084" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kabsch, W.</span></span> <span> </span><span class="NLM_article-title">XDS</span>. <i>Acta Crystallogr. Sect. D Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">125</span>– <span class="NLM_lpage">132</span>, <span class="refDoi"> DOI: 10.1107/s0907444909047337</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=10.1107%2FS0907444909047337" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=20124692" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=125-132&author=W.+Kabsch&title=XDS&doi=10.1107%2Fs0907444909047337"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Software XDS for image rotation, recognition and crystal symmetry assignment</span></div><div class="casAuthors">Kabsch, Wolfgang</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">125-132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">The usage and control of recent modifications of the program package XDS for the processing of rotation images are described in the context of previous versions.  New features include automatic detn. of spot size and reflecting range and recognition and assignment of crystal symmetry.  Moreover, the limitations of earlier package versions on the no. of correction/scaling factors and the representation of pixel contents have been removed.  Large program parts have been restructured for parallel processing so that the quality and completeness of collected data can be assessed soon after measurement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpv5AyUw00DYbVg90H21EOLACvtfcHk0lgbg6fzigr7jg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D&md5=1aa9a38aeb3ce95af4ffb7d8b8a142bd</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1107%2FS0907444909047337&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909047337%26sid%3Dliteratum%253Aachs%26aulast%3DKabsch%26aufirst%3DW.%26atitle%3DXDS%26jtitle%3DActa%2520Crystallogr.%2520Sect.%2520D%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D125%26epage%3D132%26doi%3D10.1107%2Fs0907444909047337" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Winter, G.</span></span> <span> </span><span class="NLM_article-title">xia2: an expert system for macromolecular crystallography data reduction</span>. <i>J. Appl. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>43</i></span>,  <span class="NLM_fpage">186</span>– <span class="NLM_lpage">190</span>, <span class="refDoi"> DOI: 10.1107/s0021889809045701</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=10.1107%2FS0021889809045701" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtFalsrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2010&pages=186-190&author=G.+Winter&title=xia2%3A+an+expert+system+for+macromolecular+crystallography+data+reduction&doi=10.1107%2Fs0021889809045701"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">xia2: An expert system for macromolecular crystallography data reduction</span></div><div class="casAuthors">Winter, G.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">43</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">186-190, S186/1-S186/5</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">An expert system for macromol. crystallog. data redn. is presented, which builds on existing software to automate the complete data redn. process from images to merged structure factor amplitudes.  This can automatically identify multi-wedge, multi-pass and multiwavelength data sets and includes explicit procedures to test for crystallog. special cases.  With the push towards high-throughput crystallog. at synchrotron beamlines and automation of structure soln., the ability to reduce data with no user input fills an important gap in the pipeline.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqOi3u0ydXbSrVg90H21EOLACvtfcHk0lgbg6fzigr7jg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtFalsrk%253D&md5=27771caa813f3e828abdf2a331ae55e3</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1107%2FS0021889809045701&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0021889809045701%26sid%3Dliteratum%253Aachs%26aulast%3DWinter%26aufirst%3DG.%26atitle%3Dxia2%253A%2520an%2520expert%2520system%2520for%2520macromolecular%2520crystallography%2520data%2520reduction%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D2010%26volume%3D43%26spage%3D186%26epage%3D190%26doi%3D10.1107%2Fs0021889809045701" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Winn, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ballard, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dodson, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evans, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keegan, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krissinel, E. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leslie, A. G. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McNicholas, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murshudov, G. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pannu, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Potterton, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Powell, H. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vagin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, K. S.</span></span> <span> </span><span class="NLM_article-title">Overview of the CCP4 suite and current developments</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>67</i></span>,  <span class="NLM_fpage">235</span>– <span class="NLM_lpage">242</span>, <span class="refDoi"> DOI: 10.1107/s0907444910045749</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=10.1107%2FS0907444910045749" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=21460441" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktFWqt70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2011&pages=235-242&author=M.+D.+Winnauthor=C.+C.+Ballardauthor=K.+D.+Cowtanauthor=E.+J.+Dodsonauthor=P.+Emsleyauthor=P.+R.+Evansauthor=R.+M.+Keeganauthor=E.+B.+Krissinelauthor=A.+G.+W.+Leslieauthor=A.+McCoyauthor=S.+J.+McNicholasauthor=G.+N.+Murshudovauthor=N.+S.+Pannuauthor=E.+A.+Pottertonauthor=H.+R.+Powellauthor=R.+J.+Readauthor=A.+Vaginauthor=K.+S.+Wilson&title=Overview+of+the+CCP4+suite+and+current+developments&doi=10.1107%2Fs0907444910045749"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Overview of the CCP4 suite and current developments</span></div><div class="casAuthors">Winn, Martyn D.; Ballard, Charles C.; Cowtan, Kevin D.; Dodson, Eleanor J.; Emsley, Paul; Evans, Phil R.; Keegan, Ronan M.; Krissinel, Eugene B.; Leslie, Andrew G. W.; McCoy, Airlie; McNicholas, Stuart J.; Murshudov, Garib N.; Pannu, Navraj S.; Potterton, Elizabeth A.; Powell, Harold R.; Read, Randy J.; Vagin, Alexei; Wilson, Keith S.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">235-242</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A review.  The CCP4 (Collaborative Computational Project, No. 4) software suite is a collection of programs and assocd. data and software libraries which can be used for macromol. structure detn. by X-ray crystallog.  The suite is designed to be flexible, allowing users a no. of methods of achieving their aims.  The programs are from a wide variety of sources but are connected by a common infrastructure provided by std. file formats, data objects and graphical interfaces.  Structure soln. by macromol. crystallog. is becoming increasingly automated and the CCP4 suite includes several automation pipelines.  After giving a brief description of the evolution of CCP4 over the last 30 years, an overview of the current suite is given.  While detailed descriptions are given in the accompanying articles, here it is shown how the individual programs contribute to a complete software package.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp_wWMMIol6UrVg90H21EOLACvtfcHk0li-dY29Crl6nw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktFWqt70%253D&md5=c407e4d47bef46864be336d60147c17d</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1107%2FS0907444910045749&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910045749%26sid%3Dliteratum%253Aachs%26aulast%3DWinn%26aufirst%3DM.%2BD.%26aulast%3DBallard%26aufirst%3DC.%2BC.%26aulast%3DCowtan%26aufirst%3DK.%2BD.%26aulast%3DDodson%26aufirst%3DE.%2BJ.%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DEvans%26aufirst%3DP.%2BR.%26aulast%3DKeegan%26aufirst%3DR.%2BM.%26aulast%3DKrissinel%26aufirst%3DE.%2BB.%26aulast%3DLeslie%26aufirst%3DA.%2BG.%2BW.%26aulast%3DMcCoy%26aufirst%3DA.%26aulast%3DMcNicholas%26aufirst%3DS.%2BJ.%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26aulast%3DPannu%26aufirst%3DN.%2BS.%26aulast%3DPotterton%26aufirst%3DE.%2BA.%26aulast%3DPowell%26aufirst%3DH.%2BR.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DVagin%26aufirst%3DA.%26aulast%3DWilson%26aufirst%3DK.%2BS.%26atitle%3DOverview%2520of%2520the%2520CCP4%2520suite%2520and%2520current%2520developments%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2011%26volume%3D67%26spage%3D235%26epage%3D242%26doi%3D10.1107%2Fs0907444910045749" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mccoy, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosse-kunstleve, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winn, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Storoni, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span> <span> </span><span class="NLM_article-title">Phaser Crystallographic Software Research Papers</span>. <i>J. Appl. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>40</i></span>,  <span class="NLM_fpage">658</span>– <span class="NLM_lpage">674</span>, <span class="refDoi"> DOI: 10.1107/s0021889807021206</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=10.1107%2Fs0021889807021206" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=19461840" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2007&pages=658-674&author=A.+J.+Mccoyauthor=R.+W.+Grosse-kunstleveauthor=P.+D.+Adamsauthor=M.+D.+Winnauthor=L.+C.+Storoniauthor=R.+J.+Read&title=Phaser+Crystallographic+Software+Research+Papers&doi=10.1107%2Fs0021889807021206"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Phaser crystallographic software</span></div><div class="casAuthors">McCoy, Airlie J.; Grosse-Kunstleve, Ralf W.; Adams, Paul D.; Winn, Martyn D.; Storoni, Laurent C.; Read, Randy J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">658-674</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Phaser is a program for phasing macromol. crystal structures by both mol. replacement and exptl. phasing methods.  The novel phasing algorithms implemented in Phaser have been developed using max. likelihood and multivariate statistics.  For mol. replacement, the new algorithms have proved to be significantly better than traditional methods in discriminating correct solns. from noise, and for single-wavelength anomalous dispersion exptl. phasing, the new algorithms, which account for correlations between F+ and F-, give better phases (lower mean phase error with respect to the phases given by the refined structure) than those that use mean F and anomalous differences ΔF.  One of the design concepts of Phaser was that it be capable of a high degree of automation.  To this end, Phaser (written in C++) can be called directly from Python, although it can also be called using traditional CCP4 keyword-style input.  Phaser is a platform for future development of improved phasing methods and their release, including source code, to the crystallog. community.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPeVyjN_FMGbVg90H21EOLACvtfcHk0li-dY29Crl6nw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D&md5=c63b722ae97e0a74e6a5a079d388f09f</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1107%2Fs0021889807021206&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252Fs0021889807021206%26sid%3Dliteratum%253Aachs%26aulast%3DMccoy%26aufirst%3DA.%2BJ.%26aulast%3DGrosse-kunstleve%26aufirst%3DR.%2BW.%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DWinn%26aufirst%3DM.%2BD.%26aulast%3DStoroni%26aufirst%3DL.%2BC.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26atitle%3DPhaser%2520Crystallographic%2520Software%2520Research%2520Papers%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D2007%26volume%3D40%26spage%3D658%26epage%3D674%26doi%3D10.1107%2Fs0021889807021206" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Murshudov, G. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vagin, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dodson, E. J.</span></span> <span> </span><span class="NLM_article-title">Refinement of Macromolecular Structures by the Maximum-Likelihood Method</span>. <i>Acta Crystallogr. Sect. D Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">240</span>– <span class="NLM_lpage">255</span>, <span class="refDoi"> DOI: 10.1107/s0907444996012255</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=10.1107%2FS0907444996012255" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=15299926" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=1%3ACAS%3A528%3ADyaK2sXjs1Gnsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=1997&pages=240-255&author=G.+N.+Murshudovauthor=A.+A.+Vaginauthor=E.+J.+Dodson&title=Refinement+of+Macromolecular+Structures+by+the+Maximum-Likelihood+Method&doi=10.1107%2Fs0907444996012255"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Refinement of macromolecular structures by the maximum-likelihood method</span></div><div class="casAuthors">Murshudov, Garib N.; Vagin, Alexei A.; Dodson, Eleanor J.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">D53</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">240-255</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Munksgaard</span>)
        </div><div class="casAbstract">A review with many refs. on the math. basis of max. likelihood.  The likelihood function for macromol. structures is extended to include prior phase information and exptl. std. uncertainties.  The assumption that different parts of a structure might have different errors is considered.  A method for estg. σA using "free" reflections is described and its effects analyzed.  The derived equations have been implemented in the program REFMAC.  This has been tested on several proteins at different stages of refinement (bacterial α-amylase, cytochrome c', cross-linked insulin and oligopeptide binding protein).  The results derived using the max.-likelihood residual are consistently better than those obtained from least-squares refinement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWg7mzQRNBhbVg90H21EOLACvtfcHk0li-dY29Crl6nw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXjs1Gnsb4%253D&md5=ec7f141ce1542f7ff458b98ecfe3f8af</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1107%2FS0907444996012255&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444996012255%26sid%3Dliteratum%253Aachs%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26aulast%3DVagin%26aufirst%3DA.%2BA.%26aulast%3DDodson%26aufirst%3DE.%2BJ.%26atitle%3DRefinement%2520of%2520Macromolecular%2520Structures%2520by%2520the%2520Maximum-Likelihood%2520Method%26jtitle%3DActa%2520Crystallogr.%2520Sect.%2520D%2520Biol.%2520Crystallogr.%26date%3D1997%26volume%3D53%26spage%3D240%26epage%3D255%26doi%3D10.1107%2Fs0907444996012255" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Afonine, P. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bunkóczi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, V. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, I. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Echols, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Headd, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hung, L.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kapral, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosse-Kunstleve, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moriarty, N. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oeffner, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terwilliger, T. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zwart, P. H.</span></span> <span> </span><span class="NLM_article-title">PHENIX: a comprehensive Python-based system for macromolecular structure solution</span>. <i>Acta Crystallogr. Sect. D Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">213</span>– <span class="NLM_lpage">221</span>, <span class="refDoi"> DOI: 10.1107/s0907444909052925</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=10.1107%2FS0907444909052925" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=20124702" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=213-221&author=P.+D.+Adamsauthor=P.+V.+Afonineauthor=G.+Bunk%C3%B3cziauthor=V.+B.+Chenauthor=I.+W.+Davisauthor=N.+Echolsauthor=J.+J.+Headdauthor=L.-W.+Hungauthor=G.+J.+Kapralauthor=R.+W.+Grosse-Kunstleveauthor=A.+J.+McCoyauthor=N.+W.+Moriartyauthor=R.+Oeffnerauthor=R.+J.+Readauthor=D.+C.+Richardsonauthor=J.+S.+Richardsonauthor=T.+C.+Terwilligerauthor=P.+H.+Zwart&title=PHENIX%3A+a+comprehensive+Python-based+system+for+macromolecular+structure+solution&doi=10.1107%2Fs0907444909052925"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">PHENIX: a comprehensive Python-based system for macromolecular structure solution</span></div><div class="casAuthors">Adams, Paul D.; Afonine, Pavel V.; Bunkoczi, Gabor; Chen, Vincent B.; Davis, Ian W.; Echols, Nathaniel; Headd, Jeffrey J.; Hung, Li Wei; Kapral, Gary J.; Grosse-Kunstleve, Ralf W.; McCoy, Airlie J.; Moriarty, Nigel W.; Oeffner, Robert; Read, Randy J.; Richardson, David C.; Richardson, Jane S.; Terwilliger, Thomas C.; Zwart, Peter H.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">213-221</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A review.  Macromol. X-ray crystallog. is routinely applied to understand biol. processes at a mol. level.  However, significant time and effort are still required to solve and complete many of these structures because of the need for manual interpretation of complex numerical data using many software packages and the repeated use of interactive three-dimensional graphics.  PHENIX has been developed to provide a comprehensive system for macromol. crystallog. structure soln. with an emphasis on the automation of all procedures.  This has relied on the development of algorithms that minimize or eliminate subjective input, the development of algorithms that automate procedures that are traditionally performed by hand and, finally, the development of a framework that allows a tight integration between the algorithms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpm_HgKRpa-ObVg90H21EOLACvtfcHk0lhZ0PGPevKKEQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D&md5=67b439ff4bd61c659cae37ca4209b7bc</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1107%2FS0907444909052925&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909052925%26sid%3Dliteratum%253Aachs%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DAfonine%26aufirst%3DP.%2BV.%26aulast%3DBunk%25C3%25B3czi%26aufirst%3DG.%26aulast%3DChen%26aufirst%3DV.%2BB.%26aulast%3DDavis%26aufirst%3DI.%2BW.%26aulast%3DEchols%26aufirst%3DN.%26aulast%3DHeadd%26aufirst%3DJ.%2BJ.%26aulast%3DHung%26aufirst%3DL.-W.%26aulast%3DKapral%26aufirst%3DG.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DMoriarty%26aufirst%3DN.%2BW.%26aulast%3DOeffner%26aufirst%3DR.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DRichardson%26aufirst%3DD.%2BC.%26aulast%3DRichardson%26aufirst%3DJ.%2BS.%26aulast%3DTerwilliger%26aufirst%3DT.%2BC.%26aulast%3DZwart%26aufirst%3DP.%2BH.%26atitle%3DPHENIX%253A%2520a%2520comprehensive%2520Python-based%2520system%2520for%2520macromolecular%2520structure%2520solution%26jtitle%3DActa%2520Crystallogr.%2520Sect.%2520D%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D213%26epage%3D221%26doi%3D10.1107%2Fs0907444909052925" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lohkamp, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, W. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K.</span></span> <span> </span><span class="NLM_article-title">Features and development ofCoot</span>. <i>Acta Crystallogr. Sect. D Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">486</span>– <span class="NLM_lpage">501</span>, <span class="refDoi"> DOI: 10.1107/s0907444910007493</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=10.1107%2Fs0907444910007493" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=20383002" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=486-501&author=P.+Emsleyauthor=B.+Lohkampauthor=W.+G.+Scottauthor=K.+Cowtan&title=Features+and+development+ofCoot&doi=10.1107%2Fs0907444910007493"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Features and development of Coot</span></div><div class="casAuthors">Emsley, P.; Lohkamp, B.; Scott, W. G.; Cowtan, K.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">486-501</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Coot is a mol.-graphics application for model building and validation of biol. macromols.  The program displays electron-d. maps and at. models and allows model manipulations such as idealization, real-space refinement, manual rotation/translation, rigid-body fitting, ligand search, solvation, mutations, rotamers and Ramachandran idealization.  Furthermore, tools are provided for model validation as well as interfaces to external programs for refinement, validation and graphics.  The software is designed to be easy to learn for novice users, which is achieved by ensuring that tools for common tasks are 'discoverable' through familiar user-interface elements (menus and toolbars) or by intuitive behavior (mouse controls).  Recent developments have focused on providing tools for expert users, with customisable key bindings, extensions and an extensive scripting interface.  The software is under rapid development, but has already achieved very widespread use within the crystallog. community.  The current state of the software is presented, with a description of the facilities available and of some of the underlying methods employed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAKBKszbqb_7Vg90H21EOLACvtfcHk0lhZ0PGPevKKEQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D&md5=67262cbfc60004de5ef962d5c043c910</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1107%2Fs0907444910007493&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252Fs0907444910007493%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DLohkamp%26aufirst%3DB.%26aulast%3DScott%26aufirst%3DW.%2BG.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DFeatures%2520and%2520development%2520ofCoot%26jtitle%3DActa%2520Crystallogr.%2520Sect.%2520D%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D486%26epage%3D501%26doi%3D10.1107%2Fs0907444910007493" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Perera, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khaliq, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuhn, R. J.</span></span> <span> </span><span class="NLM_article-title">Closing the Door on Flaviviruses: Entry as a Target for Antiviral Drug Design</span>. <i>Antiviral Res.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>80</i></span>,  <span class="NLM_fpage">11</span>– <span class="NLM_lpage">22</span>, <span class="refDoi"> DOI: 10.1016/j.antiviral.2008.05.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=10.1016%2Fj.antiviral.2008.05.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=18585795" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtVKlsb7J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=80&publication_year=2008&pages=11-22&author=R.+Pereraauthor=M.+Khaliqauthor=R.+J.+Kuhn&title=Closing+the+Door+on+Flaviviruses%3A+Entry+as+a+Target+for+Antiviral+Drug+Design&doi=10.1016%2Fj.antiviral.2008.05.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Closing the door on flaviviruses: Entry as a target for antiviral drug design</span></div><div class="casAuthors">Perera, Rushika; Khaliq, Mansoora; Kuhn, Richard J.</div><div class="citationInfo"><span class="NLM_cas:title">Antiviral Research</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">80</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">11-22</span>CODEN:
                <span class="NLM_cas:coden">ARSRDR</span>;
        ISSN:<span class="NLM_cas:issn">0166-3542</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  With the emergence and rapid spread of West Nile virus in the United States since 1999, and the 50-100 million infections per yr caused by dengue virus globally, the threat of flaviviruses as re-emerging human pathogens has become a reality.  To support the efforts that are currently being pursued to develop effective vaccines against these viruses, researchers are also actively pursuing the development of small mol. compds. that target various aspects of the virus life cycle.  Recent advances in the structural characterization of the flaviviruses have provided a strong foundation toward these efforts.  These studies have provided the pseudo-at. structures of virions from several members of the genus as well as at. resoln. structures of several viral proteins.  Most importantly, these studies have highlighted specific structural rearrangements that occur within the virion that are necessary for the virus to complete its life cycle.  These rearrangements occur when the virus must transition from immature, to mature, to fusion-active states and rely heavily on the conformational flexibility of the envelope (E) protein that forms the outer glycoprotein shell of the virus.  Anal. of these conformational changes can suggest promising targets for structure-based antiviral design.  For instance, by targeting the flexibility of the E protein, it might be possible to inhibit required rearrangements of this protein and trap the virus in a specific state.  This would interfere with a productive flaviviral infection.  This review presents a structural perspective of the flavivirus life cycle and focuses on the role of the E protein as an opportune target for structure-based antiviral drug design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrQNS6DhW3U3rVg90H21EOLACvtfcHk0lhZ0PGPevKKEQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtVKlsb7J&md5=dfaed4719a04fe40a02b14d14fa42342</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1016%2Fj.antiviral.2008.05.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.antiviral.2008.05.004%26sid%3Dliteratum%253Aachs%26aulast%3DPerera%26aufirst%3DR.%26aulast%3DKhaliq%26aufirst%3DM.%26aulast%3DKuhn%26aufirst%3DR.%2BJ.%26atitle%3DClosing%2520the%2520Door%2520on%2520Flaviviruses%253A%2520Entry%2520as%2520a%2520Target%2520for%2520Antiviral%2520Drug%2520Design%26jtitle%3DAntiviral%2520Res.%26date%3D2008%26volume%3D80%26spage%3D11%26epage%3D22%26doi%3D10.1016%2Fj.antiviral.2008.05.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xie, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gayen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, P.-Y.</span></span> <span> </span><span class="NLM_article-title">Membrane Topology and Function of Dengue Virus NS2A Protein</span>. <i>J. Virol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>87</i></span>,  <span class="NLM_fpage">4609</span>– <span class="NLM_lpage">4622</span>, <span class="refDoi"> DOI: 10.1128/jvi.02424-12</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=10.1128%2Fjvi.02424-12" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=23408612" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=1%3ACAS%3A528%3ADC%252BC3sXotVyqtLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=87&publication_year=2013&pages=4609-4622&author=X.+Xieauthor=S.+Gayenauthor=C.+Kangauthor=Z.+Yuanauthor=P.-Y.+Shi&title=Membrane+Topology+and+Function+of+Dengue+Virus+NS2A+Protein&doi=10.1128%2Fjvi.02424-12"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Membrane topology and function of dengue virus NS2A protein</span></div><div class="casAuthors">Xie, Xuping; Gayen, Shovanlal; Kang, Cong Bao; Yuan, Zhiming; Shi, Pei-Yong</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Virology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">87</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">4609-4622</span>CODEN:
                <span class="NLM_cas:coden">JOVIAM</span>;
        ISSN:<span class="NLM_cas:issn">0022-538X</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Flavivirus nonstructural protein 2A (NS2A) is a component of the viral replication complex that functions in virion assembly and antagonizes the host immune response.  Although flavivirus NS2A is known to assoc. with the endoplasmic reticulum (ER) membrane, the detailed topol. of this protein has not been detd.  Here we report the first topol. model of flavivirus NS2A on the ER membrane.  Using dengue virus (DENV) NS2A as a model, we show that (i) the N-terminal 68 amino acids are located in the ER lumen, with one segment (amino acids 30 to 52) that interacts with ER membrane without traversing the lipid bilayer; (ii) amino acids 69 to 209 form five transmembrane segments, each of which integrally spans the ER membrane; and (iii) the C-terminal tail (amino acids 210 to 218) is located in the cytosol.  NMR structural anal. showed that the first membrane-spanning segment (amino acids 69 to 93) consists of two helixes sepd. by a "helix breaker.".  The helix breaker is formed by amino acid P85 and one pos. charged residue, R84.  Functional anal. using replicon and genome-length RNAs of DENV-2 indicates that P85 is not important for viral replication.  However, when R84 was replaced with E, the mutation attenuated both viral RNA synthesis and virus prodn.  Remarkably, an R84A mutation did not affect viral RNA synthesis but blocked intracellular formation of infectious virions.  Collectively, the mutagenesis results demonstrate that NS2A functions in both DENV RNA synthesis and virion assembly/maturation.  The topol. model of DENV NS2A provides a good starting point for studying how flavivirus NS2A modulates viral replication and evasion of host immune response.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoKckWTzQ8sSLVg90H21EOLACvtfcHk0lhZ0PGPevKKEQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXotVyqtLo%253D&md5=d797704a8f18e5692cd74fc243c62eb1</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1128%2Fjvi.02424-12&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252Fjvi.02424-12%26sid%3Dliteratum%253Aachs%26aulast%3DXie%26aufirst%3DX.%26aulast%3DGayen%26aufirst%3DS.%26aulast%3DKang%26aufirst%3DC.%26aulast%3DYuan%26aufirst%3DZ.%26aulast%3DShi%26aufirst%3DP.-Y.%26atitle%3DMembrane%2520Topology%2520and%2520Function%2520of%2520Dengue%2520Virus%2520NS2A%2520Protein%26jtitle%3DJ.%2520Virol.%26date%3D2013%26volume%3D87%26spage%3D4609%26epage%3D4622%26doi%3D10.1128%2Fjvi.02424-12" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huang, C. Y.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Butrapet, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moss, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Childers, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Erb, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Calvert, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Silengo, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kinney, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blair, C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roehrig, J. T.</span></span> <span> </span><span class="NLM_article-title">The Dengue Virus Type 2 Envelope Protein Fusion Peptide Is Essential for Membrane Fusion</span>. <i>Virology</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>396</i></span>,  <span class="NLM_fpage">305</span>– <span class="NLM_lpage">315</span>, <span class="refDoi"> DOI: 10.1016/j.virol.2009.10.027</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=10.1016%2Fj.virol.2009.10.027" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=19913272" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsFantbzE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=396&publication_year=2010&pages=305-315&author=C.+Y.-H.+Huangauthor=S.+Butrapetauthor=K.+J.+Mossauthor=T.+Childersauthor=S.+M.+Erbauthor=A.+E.+Calvertauthor=S.+J.+Silengoauthor=R.+M.+Kinneyauthor=C.+D.+Blairauthor=J.+T.+Roehrig&title=The+Dengue+Virus+Type+2+Envelope+Protein+Fusion+Peptide+Is+Essential+for+Membrane+Fusion&doi=10.1016%2Fj.virol.2009.10.027"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">The dengue virus type 2 envelope protein fusion peptide is essential for membrane fusion</span></div><div class="casAuthors">Huang, Claire Y.-H.; Butrapet, Siritorn; Moss, Kelly J.; Childers, Thomas; Erb, Steven M.; Calvert, Amanda E.; Silengo, Shawn J.; Kinney, Richard M.; Blair, Carol D.; Roehrig, John T.</div><div class="citationInfo"><span class="NLM_cas:title">Virology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">396</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">305-315</span>CODEN:
                <span class="NLM_cas:coden">VIRLAX</span>;
        ISSN:<span class="NLM_cas:issn">0042-6822</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The flaviviral envelope (E) protein directs virus-mediated membrane fusion.  To investigate membrane fusion as a requirement for virus growth, the authors introduced 27 unique mutations into the fusion peptide of an infectious cDNA clone of dengue 2 virus and recovered seven stable mutant viruses.  The fusion efficiency of the mutants was impaired, demonstrating for the first time the requirement for specific FP AAs in optimal fusion.  Mutant viruses exhibited different growth kinetics and/or genetic stabilities in different cell types and adult mosquitoes.  Virus particles could be recovered following RNA transfection of cells with four lethal mutants; however, recovered viruses could not re-infect cells.  These viruses could enter cells, but internalized virus appeared to be retained in endosomal compartments of infected cells, thus suggesting a fusion blockade.  Mutations of the FP also resulted in reduced virus reactivity with flavivirus group-reactive antibodies, confirming earlier reports using virus-like particles.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGonaQLm21eCQLVg90H21EOLACvtfcHk0ljbFQM-LXyccw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsFantbzE&md5=93b00be0bdebcf5ff749374cea8c355b</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1016%2Fj.virol.2009.10.027&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.virol.2009.10.027%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DC.%2BY.-H.%26aulast%3DButrapet%26aufirst%3DS.%26aulast%3DMoss%26aufirst%3DK.%2BJ.%26aulast%3DChilders%26aufirst%3DT.%26aulast%3DErb%26aufirst%3DS.%2BM.%26aulast%3DCalvert%26aufirst%3DA.%2BE.%26aulast%3DSilengo%26aufirst%3DS.%2BJ.%26aulast%3DKinney%26aufirst%3DR.%2BM.%26aulast%3DBlair%26aufirst%3DC.%2BD.%26aulast%3DRoehrig%26aufirst%3DJ.%2BT.%26atitle%3DThe%2520Dengue%2520Virus%2520Type%25202%2520Envelope%2520Protein%2520Fusion%2520Peptide%2520Is%2520Essential%2520for%2520Membrane%2520Fusion%26jtitle%3DVirology%26date%3D2010%26volume%3D396%26spage%3D305%26epage%3D315%26doi%3D10.1016%2Fj.virol.2009.10.027" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref55"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref55'); return false;" data-citation="" class="refNumLink">55</a></strong><div class="NLM_citation" id="cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rodriguez-Madoz, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernal-Rubio, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaminski, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boyd, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fernandez-Sesma, A.</span></span> <span> </span><span class="NLM_article-title">Dengue Virus Inhibits the Production of Type I Interferon in Primary Human Dendritic Cells</span>. <i>J. Virol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>84</i></span>,  <span class="NLM_fpage">4845</span>– <span class="NLM_lpage">4850</span>, <span class="refDoi"> DOI: 10.1128/jvi.02514-09</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=10.1128%2Fjvi.02514-09" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=20164230" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=1%3ACAS%3A528%3ADC%252BC3cXlt1KhsLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=84&publication_year=2010&pages=4845-4850&author=J.+R.+Rodriguez-Madozauthor=D.+Bernal-Rubioauthor=D.+Kaminskiauthor=K.+Boydauthor=A.+Fernandez-Sesma&title=Dengue+Virus+Inhibits+the+Production+of+Type+I+Interferon+in+Primary+Human+Dendritic+Cells&doi=10.1128%2Fjvi.02514-09"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Dengue virus inhibits the production of type I interferon in primary human dendritic cells</span></div><div class="casAuthors">Rodriguez-Madoz, Juan R.; Bernal-Rubio, Dabeiba; Kaminski, Dorota; Boyd, Kelley; Fernandez-Sesma, Ana</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Virology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">84</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">4845-4850</span>CODEN:
                <span class="NLM_cas:coden">JOVIAM</span>;
        ISSN:<span class="NLM_cas:issn">0022-538X</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Dengue virus (DENV) infects human immune cells in vitro and likely infects dendritic cells (DCs) in vivo.  DENV-2 productive infection induces activation and release of high levels of chemokines and proinflammatory cytokines in monocyte-derived DCs (moDCs), with the notable exception of alpha/beta interferon (IFN-α/β).  Interestingly, DENV-2-infected moDCs fail to prime T cells, most likely due to the lack of IFN-α/β released by moDCs, since this effect was reversed by addn. of exogenous IFN-β.  Together, the authors' data show that inhibition of IFN-α/β prodn. by DENV in primary human moDCs is a novel mechanism of immune evasion.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpFUNnZLnic7bVg90H21EOLACvtfcHk0ljbFQM-LXyccw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXlt1KhsLk%253D&md5=7d751330acbca6445989821c48ffbe56</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1128%2Fjvi.02514-09&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252Fjvi.02514-09%26sid%3Dliteratum%253Aachs%26aulast%3DRodriguez-Madoz%26aufirst%3DJ.%2BR.%26aulast%3DBernal-Rubio%26aufirst%3DD.%26aulast%3DKaminski%26aufirst%3DD.%26aulast%3DBoyd%26aufirst%3DK.%26aulast%3DFernandez-Sesma%26aufirst%3DA.%26atitle%3DDengue%2520Virus%2520Inhibits%2520the%2520Production%2520of%2520Type%2520I%2520Interferon%2520in%2520Primary%2520Human%2520Dendritic%2520Cells%26jtitle%3DJ.%2520Virol.%26date%3D2010%26volume%3D84%26spage%3D4845%26epage%3D4850%26doi%3D10.1128%2Fjvi.02514-09" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref56"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref56'); return false;" data-citation="" class="refNumLink">56</a></strong><div class="NLM_citation" id="cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Morris, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huey, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindstrom, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanner, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Belew, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goodsell, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olson, A. J.</span></span> <span> </span><span class="NLM_article-title">AutoDock4 and AutoDockTools4: Automated Docking with Selective Receptor Flexibility</span>. <i>J. Comput. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">2785</span>– <span class="NLM_lpage">2791</span>, <span class="refDoi"> DOI: 10.1002/jcc.21256</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=10.1002%2Fjcc.21256" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=19399780" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=1%3ACAS%3A528%3ADC%252BD1MXht1GitrnK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2009&pages=2785-2791&author=G.+M.+Morrisauthor=R.+Hueyauthor=W.+Lindstromauthor=M.+F.+Sannerauthor=R.+K.+Belewauthor=D.+S.+Goodsellauthor=A.+J.+Olson&title=AutoDock4+and+AutoDockTools4%3A+Automated+Docking+with+Selective+Receptor+Flexibility&doi=10.1002%2Fjcc.21256"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">AutoDock and AutoDockTools: Automated docking with selective receptor flexibility</span></div><div class="casAuthors">Morris, Garrett M.; Huey, Ruth; Lindstrom, William; Sanner, Michel F.; Belew, Richard K.; Goodsell, David S.; Olson, Arthur J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">2785-2791</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">We describe the testing and release of AutoDock4 and the accompanying graphical user interface AutoDockTools.  AutoDock4 incorporates limited flexibility in the receptor.  Several tests are reported here, including a redocking expt. with 188 diverse ligand-protein complexes and a cross-docking expt. using flexible sidechains in 87 HIV protease complexes.  We also report its utility in anal. of covalently bound ligands, using both a grid-based docking method and a modification of the flexible sidechain technique. © 2009 Wiley Periodicals, Inc.  J Comput Chem, 2009.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqktE6WAo4GEbVg90H21EOLACvtfcHk0ljbFQM-LXyccw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXht1GitrnK&md5=679ce22fc50e9291c9aa16e7a1855845</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1002%2Fjcc.21256&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.21256%26sid%3Dliteratum%253Aachs%26aulast%3DMorris%26aufirst%3DG.%2BM.%26aulast%3DHuey%26aufirst%3DR.%26aulast%3DLindstrom%26aufirst%3DW.%26aulast%3DSanner%26aufirst%3DM.%2BF.%26aulast%3DBelew%26aufirst%3DR.%2BK.%26aulast%3DGoodsell%26aufirst%3DD.%2BS.%26aulast%3DOlson%26aufirst%3DA.%2BJ.%26atitle%3DAutoDock4%2520and%2520AutoDockTools4%253A%2520Automated%2520Docking%2520with%2520Selective%2520Receptor%2520Flexibility%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2009%26volume%3D30%26spage%3D2785%26epage%3D2791%26doi%3D10.1002%2Fjcc.21256" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref57"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref57'); return false;" data-citation="" class="refNumLink">57</a></strong><div class="NLM_citation" id="cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Evans, D. A.</span></span> <span> </span><span class="NLM_article-title">History of the Harvard ChemDraw Project</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">11140</span>– <span class="NLM_lpage">11145</span>, <span class="refDoi"> DOI: 10.1002/anie.201405820</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=10.1002%2Fanie.201405820" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtlejurvK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2014&pages=11140-11145&author=D.+A.+Evans&title=History+of+the+Harvard+ChemDraw+Project&doi=10.1002%2Fanie.201405820"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">History of the Harvard ChemDraw Project</span></div><div class="casAuthors">Evans, David A.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">42</span>),
    <span class="NLM_cas:pages">11140-11145</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Org. chemists routinely communicate concepts through a combination of cursive statements and graphics-based mol. representations commonly referred to as valence bond structures.  This form of written communication may be compared to the structure of the Egyptian hieroglyphic language which combines both cursive and icon-based graphics (glyphs) to communicate a specific thought or statement.  The importance of "chem. glyphs" may be highlighted by the fact that two chemists not sharing a common language may effectively communicate through the use of a series of valence bond structures and assocd. "curved arrow" notations.  This paper discusses the developmental history of ChemDraw, a computer program designed for drawing mol. structures.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo902ZF4RWorbVg90H21EOLACvtfcHk0lgvKCY7_5I2nA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtlejurvK&md5=593906df77924693bbc440decb0a8ec3</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1002%2Fanie.201405820&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201405820%26sid%3Dliteratum%253Aachs%26aulast%3DEvans%26aufirst%3DD.%2BA.%26atitle%3DHistory%2520of%2520the%2520Harvard%2520ChemDraw%2520Project%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2014%26volume%3D53%26spage%3D11140%26epage%3D11145%26doi%3D10.1002%2Fanie.201405820" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54'],'ref55':['cit55'],'ref56':['cit56'],'ref57':['cit57']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 13 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Javier García-Cárceles, Elena Caballero, Carmen Gil, <span class="NLM_string-name hlFld-ContribAuthor">Ana Martínez</span>. </span><span class="cited-content_cbyCitation_article-title">Kinase Inhibitors as Underexplored Antiviral Agents. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021</strong>, Article ASAP.</span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.1c00302&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.1c00302%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DKinase%252BInhibitors%252Bas%252BUnderexplored%252BAntiviral%252BAgents%26aulast%3DGarc%25C3%25ADa-C%25C3%25A1rceles%26aufirst%3DJavier%26date%3D2021%26date%3D2021%26date%3D16022021%26volume%3D0%26issue%3D0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Zhouling Xie, Xiaoxiao Yang, Yajun Duan, Jihong Han, <span class="NLM_string-name hlFld-ContribAuthor">Chenzhong Liao</span>. </span><span class="cited-content_cbyCitation_article-title">Small-Molecule Kinase Inhibitors for the Treatment of Nononcologic Diseases. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (3)
                                     , 1283-1345. <a href="https://doi.org/10.1021/acs.jmedchem.0c01511" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c01511</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c01511&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c01511%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DSmall-Molecule%252BKinase%252BInhibitors%252Bfor%252Bthe%252BTreatment%252Bof%252BNononcologic%252BDiseases%26aulast%3DXie%26aufirst%3DZhouling%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D29082020%26date%3D22012021%26volume%3D64%26issue%3D3%26spage%3D1283%26epage%3D1345" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Rinky Raghuvanshi, <span class="NLM_string-name hlFld-ContribAuthor">Sandip B. Bharate</span>. </span><span class="cited-content_cbyCitation_article-title">Recent Developments in the Use of Kinase Inhibitors for Management of Viral Infections. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021</strong>, Article ASAP.</span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c01467&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c01467%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DRecent%252BDevelopments%252Bin%252Bthe%252BUse%252Bof%252BKinase%252BInhibitors%252Bfor%252BManagement%252Bof%252BViral%252BInfections%26aulast%3DRaghuvanshi%26aufirst%3DRinky%26date%3D2021%26date%3D2020%26date%3D23082020%26volume%3D0%26issue%3D0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Carmen Gil, Tiziana Ginex, Inés Maestro, Vanesa Nozal, Lucía Barrado-Gil, Miguel Ángel Cuesta-Geijo, Jesús Urquiza, David Ramírez, Covadonga Alonso, Nuria E. Campillo, <span class="NLM_string-name hlFld-ContribAuthor">Ana Martinez</span>. </span><span class="cited-content_cbyCitation_article-title">COVID-19: Drug Targets and Potential Treatments. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (21)
                                     , 12359-12386. <a href="https://doi.org/10.1021/acs.jmedchem.0c00606" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c00606</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c00606&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c00606%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DCOVID-19%25253A%252BDrug%252BTargets%252Band%252BPotential%252BTreatments%26aulast%3DGil%26aufirst%3DCarmen%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D13042020%26date%3D26062020%26date%3D08062020%26volume%3D63%26issue%3D21%26spage%3D12359%26epage%3D12386" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Xin Liu, Paul Carr, Michael G. Gardiner, Martin G. Banwell, Ahmed H. Elbanna, Zeinab G. Khalil, <span class="NLM_string-name hlFld-ContribAuthor">Robert J. Capon</span>. </span><span class="cited-content_cbyCitation_article-title">Levoglucosenone and Its Pseudoenantiomer iso-Levoglucosenone as Scaffolds for Drug Discovery and Development. </span><span class="cited-content_cbyCitation_journal-name">ACS Omega</span><span> <strong>2020,</strong> <em>5 </em>
                                    (23)
                                     , 13926-13939. <a href="https://doi.org/10.1021/acsomega.0c01331" title="DOI URL">https://doi.org/10.1021/acsomega.0c01331</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsomega.0c01331&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsomega.0c01331%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Omega%26atitle%3DLevoglucosenone%252Band%252BIts%252BPseudoenantiomer%252Biso-Levoglucosenone%252Bas%252BScaffolds%252Bfor%252BDrug%252BDiscovery%252Band%252BDevelopment%26aulast%3DLiu%26aufirst%3DXin%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D25032020%26date%3D25052020%26date%3D08062020%26volume%3D5%26issue%3D23%26spage%3D13926%26epage%3D13939" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Carrow Wells, Rafael M. Couñago, Juanita C. Limas, Tuanny L. Almeida, Jeanette Gowen Cook, David H. Drewry, Jonathan M. Elkins, Opher Gileadi, Nirav R. Kapadia, Alvaro Lorente-Macias, Julie E. Pickett, Alexander Riemen, Roberta R. Ruela-de-Sousa, Timothy M. Willson, Cunyu Zhang, William J. Zuercher, Reena Zutshi, <span class="NLM_string-name hlFld-ContribAuthor">Alison D. Axtman</span>. </span><span class="cited-content_cbyCitation_article-title">SGC-AAK1-1: A Chemical Probe Targeting AAK1 and BMP2K. </span><span class="cited-content_cbyCitation_journal-name">ACS Medicinal Chemistry Letters</span><span> <strong>2020,</strong> <em>11 </em>
                                    (3)
                                     , 340-345. <a href="https://doi.org/10.1021/acsmedchemlett.9b00399" title="DOI URL">https://doi.org/10.1021/acsmedchemlett.9b00399</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsmedchemlett.9b00399&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsmedchemlett.9b00399%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DSGC-AAK1-1%25253A%252BA%252BChemical%252BProbe%252BTargeting%252BAAK1%252Band%252BBMP2K%26aulast%3DWells%26aufirst%3DCarrow%26date%3D2020%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D29082019%26date%3D23102019%26date%3D29102019%26date%3D23102019%26volume%3D11%26issue%3D3%26spage%3D340%26epage%3D345" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Jimin Xu, Xuping Xie, Na Ye, Jing Zou, Haiying Chen, Mark A. White, Pei-Yong Shi, <span class="NLM_string-name hlFld-ContribAuthor">Jia Zhou</span>. </span><span class="cited-content_cbyCitation_article-title">Design, Synthesis, and Biological Evaluation of Substituted 4,6-Dihydrospiro[[1,2,3]triazolo[4,5-b]pyridine-7,3′-indoline]-2′,5(3H)-dione Analogues as Potent NS4B Inhibitors for the Treatment of Dengue Virus Infection. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2019,</strong> <em>62 </em>
                                    (17)
                                     , 7941-7960. <a href="https://doi.org/10.1021/acs.jmedchem.9b00698" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b00698</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b00698&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.9b00698%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDesign%25252C%252BSynthesis%25252C%252Band%252BBiological%252BEvaluation%252Bof%252BSubstituted%252B4%25252C6-Dihydrospiro%25255B%25255B1%25252C2%25252C3%25255Dtriazolo%25255B4%25252C5-b%25255Dpyridine-7%25252C3%2525E2%252580%2525B2-indoline%25255D-2%2525E2%252580%2525B2%25252C5%2525283H%252529-dione%252BAnalogues%252Bas%252BPotent%252BNS4B%252BInhibitors%252Bfor%252Bthe%252BTreatment%252Bof%252BDengue%252BVirus%252BInfection%26aulast%3DXu%26aufirst%3DJimin%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D29042019%26date%3D23082019%26date%3D12082019%26volume%3D62%26issue%3D17%26spage%3D7941%26epage%3D7960" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Belén  Martinez-Gualda</span>, <span class="hlFld-ContribAuthor ">Dominique  Schols</span>, <span class="hlFld-ContribAuthor ">Steven  De Jonghe</span>. </span><span class="cited-content_cbyCitation_article-title">A patent review of adaptor associated kinase 1 (AAK1) inhibitors (2013-present). </span><span class="cited-content_cbyCitation_journal-name">Expert Opinion on Therapeutic Patents</span><span> <strong>2021,</strong> <em>156 </em>, 1-26. <a href="https://doi.org/10.1080/13543776.2021.1928637" title="DOI URL">https://doi.org/10.1080/13543776.2021.1928637</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/13543776.2021.1928637&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F13543776.2021.1928637%26sid%3Dliteratum%253Aachs%26jtitle%3DExpert%2520Opinion%2520on%2520Therapeutic%2520Patents%26atitle%3DA%252Bpatent%252Breview%252Bof%252Badaptor%252Bassociated%252Bkinase%252B1%252B%252528AAK1%252529%252Binhibitors%252B%2525282013-present%252529%26aulast%3DMartinez-Gualda%26aufirst%3DBel%25C3%25A9n%26date%3D2021%26date%3D2021%26volume%3D156%26spage%3D1%26epage%3D26" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Gaurav  Raina</span>, <span class="hlFld-ContribAuthor ">Prakash  Kannaboina</span>, <span class="hlFld-ContribAuthor ">Qazi Naveed  Ahmed</span>, <span class="hlFld-ContribAuthor ">Krishanu  Mondal</span>, <span class="hlFld-ContribAuthor ">Parthasarathi  Das</span>. </span><span class="cited-content_cbyCitation_article-title">Palladium‐Catalyzed Barluenga‐Valdes Type Cross‐Coupling Reaction: Alkenylation of 7‐Azaindole
              s. </span><span class="cited-content_cbyCitation_journal-name">Asian Journal of Organic Chemistry</span><span> <strong>2021,</strong> <em>10 </em>
                                    (1)
                                     , 251-256. <a href="https://doi.org/10.1002/ajoc.202000516" title="DOI URL">https://doi.org/10.1002/ajoc.202000516</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/ajoc.202000516&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fajoc.202000516%26sid%3Dliteratum%253Aachs%26jtitle%3DAsian%2520Journal%2520of%2520Organic%2520Chemistry%26atitle%3DPalladium%2525E2%252580%252590Catalyzed%252BBarluenga%2525E2%252580%252590Valdes%252BType%252BCross%2525E2%252580%252590Coupling%252BReaction%25253A%252BAlkenylation%252Bof%252B7%2525E2%252580%252590Azaindole%252Bs%26aulast%3DRaina%26aufirst%3DGaurav%26date%3D2021%26date%3D2020%26volume%3D10%26issue%3D1%26spage%3D251%26epage%3D256" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jeanese C.  Badenock</span>. </span><span class="cited-content_cbyCitation_article-title">Six-membered ring systems: pyridines and benzo derivatives. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2021,</strong>,, 397-430. <a href="https://doi.org/10.1016/B978-0-323-89812-6.00012-2" title="DOI URL">https://doi.org/10.1016/B978-0-323-89812-6.00012-2</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/B978-0-323-89812-6.00012-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2FB978-0-323-89812-6.00012-2%26sid%3Dliteratum%253Aachs%26atitle%3DSix-membered%252Bring%252Bsystems%25253A%252Bpyridines%252Band%252Bbenzo%252Bderivatives%26aulast%3DBadenock%26aufirst%3DJeanese%2BC.%26date%3D2021%26spage%3D397%26epage%3D430%26pub%3DElsevier%26date%3D2021%26volume%3D32" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Damoder Reddy  Motati</span>, <span class="hlFld-ContribAuthor ">Radhika  Amaradhi</span>, <span class="hlFld-ContribAuthor ">Thota  Ganesh</span>. </span><span class="cited-content_cbyCitation_article-title">Azaindole therapeutic agents. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry</span><span> <strong>2020,</strong> <em>28 </em>
                                    (24)
                                     , 115830. <a href="https://doi.org/10.1016/j.bmc.2020.115830" title="DOI URL">https://doi.org/10.1016/j.bmc.2020.115830</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmc.2020.115830&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmc.2020.115830%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%26atitle%3DAzaindole%252Btherapeutic%252Bagents%26aulast%3DMotati%26aufirst%3DDamoder%2BReddy%26date%3D2020%26volume%3D28%26issue%3D24%26spage%3D115830" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Szuyuan  Pu</span>, <span class="hlFld-ContribAuthor ">Stanford  Schor</span>, <span class="hlFld-ContribAuthor ">Marwah  Karim</span>, <span class="hlFld-ContribAuthor ">Sirle  Saul</span>, <span class="hlFld-ContribAuthor ">Makeda  Robinson</span>, <span class="hlFld-ContribAuthor ">Sathish  Kumar</span>, <span class="hlFld-ContribAuthor ">Laura I.  Prugar</span>, <span class="hlFld-ContribAuthor ">Danielle E.  Dorosky</span>, <span class="hlFld-ContribAuthor ">Jennifer  Brannan</span>, <span class="hlFld-ContribAuthor ">John M.  Dye</span>, <span class="hlFld-ContribAuthor ">Shirit  Einav</span>. </span><span class="cited-content_cbyCitation_article-title">BIKE regulates dengue virus infection and is a cellular target for broad-spectrum antivirals. </span><span class="cited-content_cbyCitation_journal-name">Antiviral Research</span><span> <strong>2020,</strong> <em>184 </em>, 104966. <a href="https://doi.org/10.1016/j.antiviral.2020.104966" title="DOI URL">https://doi.org/10.1016/j.antiviral.2020.104966</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.antiviral.2020.104966&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.antiviral.2020.104966%26sid%3Dliteratum%253Aachs%26jtitle%3DAntiviral%2520Research%26atitle%3DBIKE%252Bregulates%252Bdengue%252Bvirus%252Binfection%252Band%252Bis%252Ba%252Bcellular%252Btarget%252Bfor%252Bbroad-spectrum%252Bantivirals%26aulast%3DPu%26aufirst%3DSzuyuan%26date%3D2020%26volume%3D184%26spage%3D104966" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Sirle  Saul</span>, <span class="hlFld-ContribAuthor ">Szu-Yuan  Pu</span>, <span class="hlFld-ContribAuthor ">William J.  Zuercher</span>, <span class="hlFld-ContribAuthor ">Shirit  Einav</span>, <span class="hlFld-ContribAuthor ">Christopher R.M.  Asquith</span>. </span><span class="cited-content_cbyCitation_article-title">Potent antiviral activity of novel multi-substituted 4-anilinoquin(az)olines. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry Letters</span><span> <strong>2020,</strong> <em>30 </em>
                                    (16)
                                     , 127284. <a href="https://doi.org/10.1016/j.bmcl.2020.127284" title="DOI URL">https://doi.org/10.1016/j.bmcl.2020.127284</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmcl.2020.127284&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmcl.2020.127284%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DPotent%252Bantiviral%252Bactivity%252Bof%252Bnovel%252Bmulti-substituted%252B4-anilinoquin%252528az%252529olines%26aulast%3DSaul%26aufirst%3DSirle%26date%3D2020%26volume%3D30%26issue%3D16%26spage%3D127284" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-12/acs.jmedchem.9b00136/20190621/images/medium/jm-2019-00136f_0026.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-12/acs.jmedchem.9b00136/20190621/images/large/jm-2019-00136f_0026.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00136&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-12/acs.jmedchem.9b00136/20190621/images/medium/jm-2019-00136f_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-12/acs.jmedchem.9b00136/20190621/images/large/jm-2019-00136f_0001.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Known AAK1 inhibitors.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-12/acs.jmedchem.9b00136/20190621/images/large/jm-2019-00136f_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00136&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-12/acs.jmedchem.9b00136/20190621/images/medium/jm-2019-00136f_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-12/acs.jmedchem.9b00136/20190621/images/large/jm-2019-00136f_0005.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. AAK1 inhibitors with documented antiviral activity.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-12/acs.jmedchem.9b00136/20190621/images/large/jm-2019-00136f_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00136&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-12/acs.jmedchem.9b00136/20190621/images/medium/jm-2019-00136f_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-12/acs.jmedchem.9b00136/20190621/images/large/jm-2019-00136f_0006.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Pyrrolo[2,3-<i>b</i>]pyridine (7-aza-indole) based AAK1 inhibitor.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-12/acs.jmedchem.9b00136/20190621/images/large/jm-2019-00136f_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00136&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-12/acs.jmedchem.9b00136/20190621/images/medium/jm-2019-00136f_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-12/acs.jmedchem.9b00136/20190621/images/large/jm-2019-00136f_0013.jpeg" id="rightTab-gr13" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of 3-Substituted-5-aryl Pyrrolo[2,3-<i>b</i>]pyridines <b>8a–p</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-12/acs.jmedchem.9b00136/20190621/images/large/jm-2019-00136f_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00136&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions. (a) TMSA, Pd(PPh<sub>3</sub>)<sub>2</sub>Cl<sub>2</sub>, CuI, Et<sub>3</sub>N, THF, rt; (b) KO<i>t</i>Bu, NMP, 80 °C; (c) HNO<sub>3</sub>, 0 °C to rt; (d) Pd(PPh<sub>3</sub>)<sub>4</sub>, K<sub>2</sub>CO<sub>3</sub>, ArB(OH)<sub>2</sub>, H<sub>2</sub>O, dioxane, 105 °C; (e) H<sub>2</sub>, Pd/C, THF, rt; (f) RCOCl, pyridine, THF, 1 M NaOH, rt; (g) RCOOH, BOP, Et<sub>3</sub>N, DMF, rt; (h) PhSO<sub>2</sub>Cl, pyridine, rt.</p></p></figure><figure data-id="sch2" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-12/acs.jmedchem.9b00136/20190621/images/medium/jm-2019-00136f_0014.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-12/acs.jmedchem.9b00136/20190621/images/large/jm-2019-00136f_0014.jpeg" id="rightTab-gr14" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of 3-Benzoyl-5-(3,4-dimethoxyphenyl)-pyrrolo[2,3-<i>b</i>]pyridine <b>14</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-12/acs.jmedchem.9b00136/20190621/images/large/jm-2019-00136f_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00136&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions. (a) Hexamine, H<sub>2</sub>O, CH<sub>3</sub>COOH, 120 °C; (b) NaH, TsCl, 0 °C to rt; (c) 3,4-dimethoxyphenylboronic acid, Pd(PPh<sub>3</sub>)<sub>4</sub>, 2 M K<sub>2</sub>CO<sub>3</sub>, toluene, EtOH, 105 °C; (d) 3 M PhMgBr, THF, rt; (e) MnO<sub>2</sub>, THF, rt; (f) KOH, EtOH, 80 °C.</p></p></figure><figure data-id="sch3" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-12/acs.jmedchem.9b00136/20190621/images/medium/jm-2019-00136f_0015.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-12/acs.jmedchem.9b00136/20190621/images/large/jm-2019-00136f_0015.jpeg" id="rightTab-gr15" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthesis of 3-Phenyl-5-(3,4-dimethoxyphenyl)-pyrrolo[2,3-<i>b</i>]pyridine <b>19</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-12/acs.jmedchem.9b00136/20190621/images/large/jm-2019-00136f_0015.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00136&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions. (a) NIS, acetone, rt; (b) NaH, TsCl, THF, 0 °C to rt; (c) PhB(OH)<sub>2</sub>, Pd(PPh<sub>3</sub>)<sub>4</sub>, K<sub>2</sub>CO<sub>3</sub>, toluene, EtOH, H<sub>2</sub>O, 90 °C; (d) 3,4-dimethoxyphenylboronic acid, Pd(PPh<sub>3</sub>)<sub>4</sub>, K<sub>2</sub>CO<sub>3</sub>, toluene, EtOH, H<sub>2</sub>O, 105 °C; (e) KOH, EtOH, 80 °C.</p></p></figure><figure data-id="sch4" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-12/acs.jmedchem.9b00136/20190621/images/medium/jm-2019-00136f_0016.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-12/acs.jmedchem.9b00136/20190621/images/large/jm-2019-00136f_0016.jpeg" id="rightTab-gr16" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 4. Synthesis of 3-Alkynyl-5-(3,4-dimethoxyphenyl)-pyrrolo[2,3-<i>b</i>]pyridines <b>21a–f</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-12/acs.jmedchem.9b00136/20190621/images/large/jm-2019-00136f_0016.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00136&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions. (a) RC≡CH, Pd(PPh<sub>3</sub>)<sub>2</sub>Cl<sub>2</sub>, CuI, THF, Et<sub>3</sub>N, rt; (b) 3,4-dimethoxyphenylboronic acid, Pd(PPh<sub>3</sub>)<sub>4</sub>, K<sub>2</sub>CO<sub>3</sub>, H<sub>2</sub>O, dioxane, 105 °C.</p></p></figure><figure data-id="sch5" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Scheme 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-12/acs.jmedchem.9b00136/20190621/images/medium/jm-2019-00136f_0017.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-12/acs.jmedchem.9b00136/20190621/images/large/jm-2019-00136f_0017.jpeg" id="rightTab-gr17" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 5. Synthesis of 1-Methyl-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine <b>26</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-12/acs.jmedchem.9b00136/20190621/images/large/jm-2019-00136f_0017.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00136&amp;id=sch5"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions. (a) NaH, MeI, THF, 0 °C to rt; (b) HNO<sub>3</sub>, 0 °C to rt; (c) 3,4-dimethoxyphenylboronic acid, Pd(PPh<sub>3</sub>)<sub>4</sub>, K<sub>2</sub>CO<sub>3</sub>, H<sub>2</sub>O, dioxane, 105 °C; (d) H<sub>2</sub>, THF, rt; (e) nicotinoyl chloride, pyridine, THF, 1 M NaOH, rt.</p></p></figure><figure data-id="sch6" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Scheme 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-12/acs.jmedchem.9b00136/20190621/images/medium/jm-2019-00136f_0018.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-12/acs.jmedchem.9b00136/20190621/images/large/jm-2019-00136f_0018.jpeg" id="rightTab-gr18" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 6. Synthesis of Pyrazolo[3,4-<i>b</i>]pyridine <b>30</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-12/acs.jmedchem.9b00136/20190621/images/large/jm-2019-00136f_0018.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00136&amp;id=sch6"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions (a) 3,4-dimethoxyphenylboronic acid, Pd(PPh<sub>3</sub>)<sub>4</sub>, K<sub>2</sub>CO<sub>3</sub>, H<sub>2</sub>O, dioxane, 105 °C; (b) 35% hydrazine hydrate, EtOH, 80 °C; (c) nicotinoyl chloride, pyridine, rt.</p></p></figure><figure data-id="sch7" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Scheme 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-12/acs.jmedchem.9b00136/20190621/images/medium/jm-2019-00136f_0019.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-12/acs.jmedchem.9b00136/20190621/images/large/jm-2019-00136f_0019.jpeg" id="rightTab-gr19" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 7. Synthesis of Pyrrolo[2,3-<i>b</i>]pyrazine <b>37</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-12/acs.jmedchem.9b00136/20190621/images/large/jm-2019-00136f_0019.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00136&amp;id=sch7"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions (a) NBS, DMSO, rt; (b) Me<sub>3</sub>SiC≡CH, Pd(PPh<sub>3</sub>)<sub>2</sub>Cl<sub>2</sub>, CuI, THF, Et<sub>3</sub>N, rt; (c) KO<i>t</i>Bu, NMP, 100 °C; (d) NIS, acetone, rt; (e) 3-ethynylpyridine, Pd(PPh<sub>3</sub>)<sub>2</sub>Cl<sub>2</sub>, CuI, THF, Et<sub>3</sub>N, rt; (f) 3,4-dimethoxyphenylboronic acid, Pd(PPh<sub>3</sub>)<sub>4</sub>, K<sub>2</sub>CO<sub>3</sub>, H<sub>2</sub>O, dioxane, 105 °C.</p></p></figure><figure data-id="fig4" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-12/acs.jmedchem.9b00136/20190621/images/medium/jm-2019-00136f_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-12/acs.jmedchem.9b00136/20190621/images/large/jm-2019-00136f_0007.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Binding displacement assay of compound <b>1</b> against the NAK family members.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-12/acs.jmedchem.9b00136/20190621/images/large/jm-2019-00136f_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00136&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-12/acs.jmedchem.9b00136/20190621/images/medium/jm-2019-00136f_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-12/acs.jmedchem.9b00136/20190621/images/large/jm-2019-00136f_0008.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Crystal structure of AAK1 (gray) in complex with compound <b>1</b> (purple), PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5L4Q">5L4Q</a>. (A) Overview of the crystal structure of AAK1. Highlighted are areas that are important for kinase function. Compound <b>1</b> bound at the hinge is shown. ASCH = activation segment C-terminal Helix, a feature unique to kinases of the NAK family. (B) 2<i>F</i><sub>o</sub> – <i>F</i><sub>c</sub> electron density map contoured at 1σ around compound <b>1</b>, showing the fit of the model to the map. (C,D) Detailed views of the interactions of compound <b>1</b> in the ATP-binding site from two orientations. Black dotted lines indicate polar interactions, and red spheres indicate water molecules. Note that residues 48–63 from β1 and β2 are removed from (C) for clarity.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-12/acs.jmedchem.9b00136/20190621/images/large/jm-2019-00136f_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00136&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="13" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-12/acs.jmedchem.9b00136/20190621/images/medium/jm-2019-00136f_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-12/acs.jmedchem.9b00136/20190621/images/large/jm-2019-00136f_0009.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Compounds <b>8g</b> and <b>21b</b> suppress DENV infection more effectively than compound <b>1</b>. Dose response of DENV infection (blue) and cell viability (black) to compounds <b>1</b>, <b>8g</b>, and <b>21b</b> measured by luciferase and alamarBlue assays, respectively, 48 h after infection. Data are plotted relative to vehicle control. At least two representative experiments conducted are shown, each with 5 biological replicates, and the values are expressed as means ± SD.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-12/acs.jmedchem.9b00136/20190621/images/large/jm-2019-00136f_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00136&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="14" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-12/acs.jmedchem.9b00136/20190621/images/medium/jm-2019-00136f_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-12/acs.jmedchem.9b00136/20190621/images/large/jm-2019-00136f_0010.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Compounds <b>1</b>, <b>8g</b>, and <b>21b</b> suppress EBOV infection. Dose response of EBOV infection (blue) and cell viability (black) to compounds <b>1</b>, <b>8g</b>, and <b>21b</b> measured by plaque assay (for compound <b>1</b>) or immunofluorescence assays (for compounds <b>8g</b> and <b>21b</b>) and CellTiter-Glo luminescent cell viability assay in Huh7 cells 48 h after infection. Data are plotted relative to vehicle control. A representative experiment, out of at least two conducted, is shown and the values are shown as means ± SD.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-12/acs.jmedchem.9b00136/20190621/images/large/jm-2019-00136f_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00136&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig8" data-index="15" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-12/acs.jmedchem.9b00136/20190621/images/medium/jm-2019-00136f_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-12/acs.jmedchem.9b00136/20190621/images/large/jm-2019-00136f_0011.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Antiviral effect of compounds <b>1</b>, <b>8g</b>, and <b>21b</b> correlates with functional inhibition of AAK1. Dose response of AP2M1 phosphorylation to treatment with <b>1</b> (A), <b>8g</b> (B), and <b>21b</b> (C) by western analysis in lysates derived from Huh7 cells. Representative membranes (from two independent experiments) blotted with anti-phospho-AP2M1 (pAP2M1), anti-AP2M1 (AP2M1), and anti-actin (actin) antibodies and quantified data of pAP2M1/actin protein ratio normalized to dimethyl sulfoxide (DMSO) controls are shown. **<i>p</i> < 0.01, ***<i>p</i> < 0.001 by 2-tailed unpaired <i>t</i>-test.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-12/acs.jmedchem.9b00136/20190621/images/large/jm-2019-00136f_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00136&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig9" data-index="16" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-12/acs.jmedchem.9b00136/20190621/images/medium/jm-2019-00136f_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-12/acs.jmedchem.9b00136/20190621/images/large/jm-2019-00136f_0012.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. Ex vivo antiviral activity of <b>21b</b> and <b>8g</b> in human primary dendritic cells. Dose response of DENV infection (blue) and cell viability (black) to compounds <b>21b</b> (A) and <b>8g</b> (B) measured by plaque assays and alamarBlue assays, respectively, 72 h after infection of primary human MDDCs. A representative experiment with cells from a single donor is shown, out of 2 independent experiments conducted with cells derived from 2 donors, each with 6 biological replicates; the values are shown as means ± SD.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-12/acs.jmedchem.9b00136/20190621/images/large/jm-2019-00136f_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00136&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig10" data-index="17" class="article__inlineFigure"><h2 class="fig-label">Figure 10</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-12/acs.jmedchem.9b00136/20190621/images/medium/jm-2019-00136f_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-12/acs.jmedchem.9b00136/20190621/images/large/jm-2019-00136f_0002.jpeg" id="rightTab-gr10" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 10. Kinome tree of compound <b>21b</b>. Kinases that bind compound <b>21b</b> are marked with red circles. The larger the circle, the stronger the binding affinity.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-12/acs.jmedchem.9b00136/20190621/images/large/jm-2019-00136f_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00136&amp;id=fig10"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig11" data-index="18" class="article__inlineFigure"><h2 class="fig-label">Figure 11</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-12/acs.jmedchem.9b00136/20190621/images/medium/jm-2019-00136f_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-12/acs.jmedchem.9b00136/20190621/images/large/jm-2019-00136f_0003.jpeg" id="rightTab-gr11" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 11. AAK1 snapshot structures with 3 ligands, compound <b>1</b> (green, lightgreen), compound <b>8g</b> (magenta, rose), and compound <b>21b</b> (cyan; lightblue), extracted from the last 4 ns of the MD trajectory as a representative structure from the biggest cluster using the average linkage clustering method available in cpptraj. The AAK1 and lkb X-ray structures are also shown (orange, yellow). Image made by the Chimera software.<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-12/acs.jmedchem.9b00136/20190621/images/large/jm-2019-00136f_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00136&amp;id=fig11"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig12" data-index="19" class="article__inlineFigure"><h2 class="fig-label">Figure 12</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-12/acs.jmedchem.9b00136/20190621/images/medium/jm-2019-00136f_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-12/acs.jmedchem.9b00136/20190621/images/large/jm-2019-00136f_0004.jpeg" id="rightTab-gr12" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 12. AAK1 enzyme with compound <b>21b</b> (same snapshot as in <a data-tab="pane-pcw-Figures" href="#fig11" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig11" id="" class=" internalNav">11</a></a>). Amino acids making contact with the two methoxy groups (heavy atom distances < 4 Å) are colored in yellow and magenta. The magenta color represents also the residues having van der Waals contacts with the nitrile group of compound <b>1</b>. Image made by the Chimera software.<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-12/acs.jmedchem.9b00136/20190621/images/large/jm-2019-00136f_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00136&amp;id=fig12"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i132">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_87044" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_87044" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 57 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Murray, N. A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quam, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilder-Smith, A.</span></span> <span> </span><span class="NLM_article-title">Epidemiology of Dengue: Past, Present and Future Prospects</span>. <i>Clin. Epidemiol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">299</span>– <span class="NLM_lpage">309</span>, <span class="refDoi"> DOI: 10.2147/clep.s34440</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=10.2147%2Fclep.s34440" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=23990732" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=1%3ACAS%3A280%3ADC%252BC3sbjs12msg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2013&pages=299-309&author=N.+A.+E.+Murrayauthor=M.+B.+Quamauthor=A.+Wilder-Smith&title=Epidemiology+of+Dengue%3A+Past%2C+Present+and+Future+Prospects&doi=10.2147%2Fclep.s34440"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Epidemiology of dengue: past, present and future prospects</span></div><div class="casAuthors">Murray Natasha Evelyn Anne; Quam Mikkel B; Wilder-Smith Annelies</div><div class="citationInfo"><span class="NLM_cas:title">Clinical epidemiology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">299-309</span>
        ISSN:<span class="NLM_cas:issn">1179-1349</span>.
    </div><div class="casAbstract">Dengue is currently regarded globally as the most important mosquito-borne viral disease.  A history of symptoms compatible with dengue can be traced back to the Chin Dynasty of 265-420 AD.  The virus and its vectors have now become widely distributed throughout tropical and subtropical regions of the world, particularly over the last half-century.  Significant geographic expansion has been coupled with rapid increases in incident cases, epidemics, and hyperendemicity, leading to the more severe forms of dengue.  Transmission of dengue is now present in every World Health Organization (WHO) region of the world and more than 125 countries are known to be dengue endemic.  The true impact of dengue globally is difficult to ascertain due to factors such as inadequate disease surveillance, misdiagnosis, and low levels of reporting.  Currently available data likely grossly underestimates the social, economic, and disease burden.  Estimates of the global incidence of dengue infections per year have ranged between 50 million and 200 million; however, recent estimates using cartographic approaches suggest this number is closer to almost 400 million.  The expansion of dengue is expected to increase due to factors such as the modern dynamics of climate change, globalization, travel, trade, socioeconomics, settlement and also viral evolution.  No vaccine or specific antiviral therapy currently exists to address the growing threat of dengue.  Prompt case detection and appropriate clinical management can reduce the mortality from severe dengue.  Effective vector control is the mainstay of dengue prevention and control.  Surveillance and improved reporting of dengue cases is also essential to gauge the true global situation as indicated in the objectives of the WHO Global Strategy for Dengue Prevention and Control, 2012-2020.  More accurate data will inform the prioritization of research, health policy, and financial resources toward reducing this poorly controlled disease.  The objective of this paper is to review historical and current epidemiology of dengue worldwide and, additionally, reflect on some potential reasons for expansion of dengue into the future.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTTkqBcd5TiZSM8MEzWGjS9fW6udTcc2eZOSVmerfQoUrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3sbjs12msg%253D%253D&md5=89d2c693ec4bda3679bc24609316e4b5</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.2147%2Fclep.s34440&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2147%252Fclep.s34440%26sid%3Dliteratum%253Aachs%26aulast%3DMurray%26aufirst%3DN.%2BA.%2BE.%26aulast%3DQuam%26aufirst%3DM.%2BB.%26aulast%3DWilder-Smith%26aufirst%3DA.%26atitle%3DEpidemiology%2520of%2520Dengue%253A%2520Past%252C%2520Present%2520and%2520Future%2520Prospects%26jtitle%3DClin.%2520Epidemiol.%26date%3D2013%26volume%3D5%26spage%3D299%26epage%3D309%26doi%3D10.2147%2Fclep.s34440" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group">WHO</span>. <span> </span><span class="NLM_article-title">Epidemiology</span>. <a href="http://www.who.int/denguecontrol/epidemiology/en/" class="extLink">http://www.who.int/denguecontrol/epidemiology/en/</a> (accessed Nov 15, <span class="NLM_year">2018</span>).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2018&author=WHO&title=Epidemiology"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26aulast%3D%26atitle%3DEpidemiology%26date%3D2018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span> <i>Weekly Epidemiological
Record</i>; <span class="NLM_publisher-name">World Health
Organization</span>, <span class="NLM_year">2016</span>; Vol.  <span class="NLM_volume">91</span> (<span class="NLM_issue">30</span>), pp  <span class="NLM_fpage">349</span>– <span class="NLM_lpage">364</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+Weekly+Epidemiological%0ARecord%3B+World+Health%0AOrganization%2C+2016%3B+Vol.+91+%2830%29%2C+pp+349%E2%80%93364."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dreport%26sid%3Dliteratum%253Aachs%26jtitle%3DWeekly%2520Epidemiological%250ARecord%26pub%3DWorld%2520Health%250AOrganization%26date%3D2016%26volume%3D91%26issue%3D30%26spage%3D349%26epage%3D364" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sanyal, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sinha, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Halder, K. K.</span></span> <span> </span><span class="NLM_article-title">Pathogenesis of Dengue Haemorrhagic Fever</span>. <i>J. Indian Med. Assoc.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>89</i></span>,  <span class="NLM_fpage">152</span>– <span class="NLM_lpage">153</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=89&publication_year=2013&pages=152-153&author=S.+Sanyalauthor=S.+Sinhaauthor=K.+K.+Halder&title=Pathogenesis+of+Dengue+Haemorrhagic+Fever"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSanyal%26aufirst%3DS.%26aulast%3DSinha%26aufirst%3DS.%26aulast%3DHalder%26aufirst%3DK.%2BK.%26atitle%3DPathogenesis%2520of%2520Dengue%2520Haemorrhagic%2520Fever%26jtitle%3DJ.%2520Indian%2520Med.%2520Assoc.%26date%3D2013%26volume%3D89%26spage%3D152%26epage%3D153" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group">WHO</span>. <span> </span><span class="NLM_article-title">Ebola virus disease</span> <a href="http://www.who.int/news-room/fact-sheets/detail/ebola-virus-disease" class="extLink">http://www.who.int/news-room/fact-sheets/detail/ebola-virus-disease</a> (accessed Aug 7, <span class="NLM_year">2018</span>).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2018&author=WHO&title=Ebola+virus+disease"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26aulast%3D%26atitle%3DEbola%2520virus%2520disease%26date%3D2018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Behnam, M. A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nitsche, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boldescu, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klein, C. D.</span></span> <span> </span><span class="NLM_article-title">The Medicinal Chemistry of Dengue Virus</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">5622</span>– <span class="NLM_lpage">5649</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01653</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01653" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=1%3ACAS%3A528%3ADC%252BC28XptlGguw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=5622-5649&author=M.+A.+M.+Behnamauthor=C.+Nitscheauthor=V.+Boldescuauthor=C.+D.+Klein&title=The+Medicinal+Chemistry+of+Dengue+Virus&doi=10.1021%2Facs.jmedchem.5b01653"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">The Medicinal Chemistry of Dengue Virus</span></div><div class="casAuthors">Behnam, Mira A. M.; Nitsche, Christoph; Boldescu, Veaceslav; Klein, Christian D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">5622-5649</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The dengue virus and related flaviviruses are an increasing global health threat.  In this perspective, we comment on and review medicinal chem. efforts aimed at the prevention or treatment of dengue infections.  We include target-based approaches aimed at viral or host factors and results from phenotypic screenings in cellular assay systems for viral replication.  This perspective is limited to the discussion of results that provide explicit chem. or structure-activity relationship (SAR), or appear to be of particular interest to the medicinal chemist for other reasons.  The discovery and development efforts discussed here may at least partially be extrapolated toward other emerging flaviviral infections, such as West Nile virus.  Therefore, this perspective, although not aimed at flaviviruses in general, should also be able to provide an overview of the medicinal chem. of these closely related infectious agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr7LeRM_49L7bVg90H21EOLACvtfcHk0lgRiknPi0OWXw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XptlGguw%253D%253D&md5=a3de42f04cd59a4396a0b020b53c66ee</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01653&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01653%26sid%3Dliteratum%253Aachs%26aulast%3DBehnam%26aufirst%3DM.%2BA.%2BM.%26aulast%3DNitsche%26aufirst%3DC.%26aulast%3DBoldescu%26aufirst%3DV.%26aulast%3DKlein%26aufirst%3DC.%2BD.%26atitle%3DThe%2520Medicinal%2520Chemistry%2520of%2520Dengue%2520Virus%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D5622%26epage%3D5649%26doi%3D10.1021%2Facs.jmedchem.5b01653" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bekerman, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neveu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shulla, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brannan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pu, S.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barouch-Bentov, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bakken, R. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mateo, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Govero, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nagamine, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diamond, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Jonghe, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herdewijn, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dye, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Randall, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Einav, S.</span></span> <span> </span><span class="NLM_article-title">Anticancer Kinase Inhibitors Impair Intracellular Viral Trafficking and Exert Broad-Spectrum Antiviral Effects</span>. <i>J. Clin. Invest.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>127</i></span>,  <span class="NLM_fpage">1338</span>– <span class="NLM_lpage">1352</span>, <span class="refDoi"> DOI: 10.1172/jci89857</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=10.1172%2Fjci89857" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=28240606" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=1%3ACAS%3A280%3ADC%252BC1czgvFajtg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=127&publication_year=2017&pages=1338-1352&author=E.+Bekermanauthor=G.+Neveuauthor=A.+Shullaauthor=J.+Brannanauthor=S.-Y.+Puauthor=S.+Wangauthor=F.+Xiaoauthor=R.+Barouch-Bentovauthor=R.+R.+Bakkenauthor=R.+Mateoauthor=J.+Goveroauthor=C.+M.+Nagamineauthor=M.+S.+Diamondauthor=S.+De+Jongheauthor=P.+Herdewijnauthor=J.+M.+Dyeauthor=G.+Randallauthor=S.+Einav&title=Anticancer+Kinase+Inhibitors+Impair+Intracellular+Viral+Trafficking+and+Exert+Broad-Spectrum+Antiviral+Effects&doi=10.1172%2Fjci89857"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Anticancer kinase inhibitors impair intracellular viral trafficking and exert broad-spectrum antiviral effects</span></div><div class="casAuthors">Bekerman Elena; Neveu Gregory; Shulla Ana; Brannan Jennifer; Pu Szu-Yuan; Wang Stanley; Xiao Fei; Barouch-Bentov Rina; Bakken Russell R; Mateo Roberto; Govero Jennifer; Nagamine Claude M; Diamond Michael S; De Jonghe Steven; Herdewijn Piet; Dye John M; Randall Glenn; Einav Shirit</div><div class="citationInfo"><span class="NLM_cas:title">The Journal of clinical investigation</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">127</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1338-1352</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Global health is threatened by emerging viral infections, which largely lack effective vaccines or therapies.  Targeting host pathways that are exploited by multiple viruses could offer broad-spectrum solutions.  We previously reported that AAK1 and GAK, kinase regulators of the host adaptor proteins AP1 and AP2, are essential for hepatitis C virus (HCV) infection, but the underlying mechanism and relevance to other viruses or in vivo infections remained unknown.  Here, we have discovered that AP1 and AP2 cotraffic with HCV particles in live cells.  Moreover, we found that multiple viruses, including dengue and Ebola, exploit AAK1 and GAK during entry and infectious virus production.  In cultured cells, treatment with sunitinib and erlotinib, approved anticancer drugs that inhibit AAK1 or GAK activity, or with more selective compounds inhibited intracellular trafficking of HCV and multiple unrelated RNA viruses with a high barrier to resistance.  In murine models of dengue and Ebola infection, sunitinib/erlotinib combination protected against morbidity and mortality.  We validated sunitinib- and erlotinib-mediated inhibition of AAK1 and GAK activity as an important mechanism of antiviral action.  Additionally, we revealed potential roles for additional kinase targets.  These findings advance our understanding of virus-host interactions and establish a proof of principle for a repurposed, host-targeted approach to combat emerging viruses.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSeR1dYk1i_czQLBVDQn4bCfW6udTcc2eZ9_iPInJ94crntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1czgvFajtg%253D%253D&md5=2de0a5d7668651f92be3c2e79823ced4</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1172%2Fjci89857&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1172%252Fjci89857%26sid%3Dliteratum%253Aachs%26aulast%3DBekerman%26aufirst%3DE.%26aulast%3DNeveu%26aufirst%3DG.%26aulast%3DShulla%26aufirst%3DA.%26aulast%3DBrannan%26aufirst%3DJ.%26aulast%3DPu%26aufirst%3DS.-Y.%26aulast%3DWang%26aufirst%3DS.%26aulast%3DXiao%26aufirst%3DF.%26aulast%3DBarouch-Bentov%26aufirst%3DR.%26aulast%3DBakken%26aufirst%3DR.%2BR.%26aulast%3DMateo%26aufirst%3DR.%26aulast%3DGovero%26aufirst%3DJ.%26aulast%3DNagamine%26aufirst%3DC.%2BM.%26aulast%3DDiamond%26aufirst%3DM.%2BS.%26aulast%3DDe%2BJonghe%26aufirst%3DS.%26aulast%3DHerdewijn%26aufirst%3DP.%26aulast%3DDye%26aufirst%3DJ.%2BM.%26aulast%3DRandall%26aufirst%3DG.%26aulast%3DEinav%26aufirst%3DS.%26atitle%3DAnticancer%2520Kinase%2520Inhibitors%2520Impair%2520Intracellular%2520Viral%2520Trafficking%2520and%2520Exert%2520Broad-Spectrum%2520Antiviral%2520Effects%26jtitle%3DJ.%2520Clin.%2520Invest.%26date%3D2017%26volume%3D127%26spage%3D1338%26epage%3D1352%26doi%3D10.1172%2Fjci89857" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bekerman, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Einav, S.</span></span> <span> </span><span class="NLM_article-title">Combating Emerging Viral Threats</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>348</i></span>,  <span class="NLM_fpage">282</span>– <span class="NLM_lpage">283</span>, <span class="refDoi"> DOI: 10.1126/science.aaa3778</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=10.1126%2Fscience.aaa3778" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=25883340" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=1%3ACAS%3A528%3ADC%252BC2MXosVKjt7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=348&publication_year=2015&pages=282-283&author=E.+Bekermanauthor=S.+Einav&title=Combating+Emerging+Viral+Threats&doi=10.1126%2Fscience.aaa3778"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Combating emerging viral threats</span></div><div class="casAuthors">Bekerman, Elena; Einav, Shirit</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">348</span>
        (<span class="NLM_cas:issue">6232</span>),
    <span class="NLM_cas:pages">282-283</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGriDTBlEp8j0LVg90H21EOLACvtfcHk0ljdQSwBnZDAyw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXosVKjt7Y%253D&md5=1cf54cc0a7db0925573f96ed72c27c11</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1126%2Fscience.aaa3778&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.aaa3778%26sid%3Dliteratum%253Aachs%26aulast%3DBekerman%26aufirst%3DE.%26aulast%3DEinav%26aufirst%3DS.%26atitle%3DCombating%2520Emerging%2520Viral%2520Threats%26jtitle%3DScience%26date%3D2015%26volume%3D348%26spage%3D282%26epage%3D283%26doi%3D10.1126%2Fscience.aaa3778" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Grove, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marsh, M.</span></span> <span> </span><span class="NLM_article-title">The Cell Biology of Receptor-Mediated Virus Entry</span>. <i>J. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>195</i></span>,  <span class="NLM_fpage">1071</span>– <span class="NLM_lpage">1082</span>, <span class="refDoi"> DOI: 10.1083/jcb.201108131</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=10.1083%2Fjcb.201108131" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=22123832" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=1%3ACAS%3A528%3ADC%252BC38XktlGksQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=195&publication_year=2011&pages=1071-1082&author=J.+Groveauthor=M.+Marsh&title=The+Cell+Biology+of+Receptor-Mediated+Virus+Entry&doi=10.1083%2Fjcb.201108131"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">The cell biology of receptor-mediated virus entry</span></div><div class="casAuthors">Grove, Joe; Marsh, Mark</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Cell Biology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">195</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1071-1082</span>CODEN:
                <span class="NLM_cas:coden">JCLBA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9525</span>.
    
            (<span class="NLM_cas:orgname">Rockefeller University Press</span>)
        </div><div class="casAbstract">A review.  The cell imposes multiple barriers to virus entry.  However, viruses exploit fundamental cellular processes to gain entry to cells and deliver their genetic cargo.  Virus entry pathways are largely defined by the interactions between virus particles and their receptors at the cell surface.  These interactions det. the mechanisms of virus attachment, uptake, intracellular trafficking, and, ultimately, penetration to the cytosol.  Elucidating the complex interplay between viruses and their receptors is necessary for a full understanding of how these remarkable agents invade their cellular hosts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqJz6iwbcbos7Vg90H21EOLACvtfcHk0ljdQSwBnZDAyw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XktlGksQ%253D%253D&md5=cb94e5f7845d2efbff2d9f08ed047938</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1083%2Fjcb.201108131&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1083%252Fjcb.201108131%26sid%3Dliteratum%253Aachs%26aulast%3DGrove%26aufirst%3DJ.%26aulast%3DMarsh%26aufirst%3DM.%26atitle%3DThe%2520Cell%2520Biology%2520of%2520Receptor-Mediated%2520Virus%2520Entry%26jtitle%3DJ.%2520Cell%2520Biol.%26date%3D2011%26volume%3D195%26spage%3D1071%26epage%3D1082%26doi%3D10.1083%2Fjcb.201108131" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Neveu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ziv-Av, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barouch-Bentov, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berkerman, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mulholland, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Einav, S.</span></span> <span> </span><span class="NLM_article-title">AP-2-Associated Protein Kinase 1 and Cyclin G-Associated Kinase Regulate Hepatitis C Virus Entry and Are Potential Drug Targets</span>. <i>J. Virol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>89</i></span>,  <span class="NLM_fpage">4387</span>– <span class="NLM_lpage">4404</span>, <span class="refDoi"> DOI: 10.1128/jvi.02705-14</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=10.1128%2Fjvi.02705-14" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=25653444" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=1%3ACAS%3A528%3ADC%252BC2MXlvVWlurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=89&publication_year=2015&pages=4387-4404&author=G.+Neveuauthor=A.+Ziv-Avauthor=R.+Barouch-Bentovauthor=E.+Berkermanauthor=J.+Mulhollandauthor=S.+Einav&title=AP-2-Associated+Protein+Kinase+1+and+Cyclin+G-Associated+Kinase+Regulate+Hepatitis+C+Virus+Entry+and+Are+Potential+Drug+Targets&doi=10.1128%2Fjvi.02705-14"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">AP-2-associated protein kinase 1 and cyclin G-associated kinase regulate hepatitis C virus entry and are potential drug targets</span></div><div class="casAuthors">Neveu, Gregory; Ziv-Av, Amotz; Barouch-Bentov, Rina; Berkerman, Elena; Mulholland, Jon; Einav, Shirit</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Virology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">89</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">4387-4404</span>CODEN:
                <span class="NLM_cas:coden">JOVIAM</span>;
        ISSN:<span class="NLM_cas:issn">1098-5514</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Hepatitis C virus (HCV) enters its target cell via clathrin-mediated endocytosis.  AP-2-assocd. protein kinase 1 (AAK1) and cyclin G-assocd. kinase (GAK) are host kinases that regulate clathrin adaptor protein (AP)-mediated trafficking in the endocytic and secretory pathways.  We previously reported that AAK1 and GAK regulate HCV assembly by stimulating binding of the μ subunit of AP-2, AP2M1, to HCV core protein.  We also discovered that AAK1 and GAK inhibitors, including the approved anticancer drugs sunitinib and erlotinib, could block HCV assembly.  Here, we hypothesized that AAK1 and GAK regulate HCV entry independently of their effect on HCV assembly.  Indeed, silencing AAK1 and GAK expression inhibited entry of pseudoparticles and cell culture grown-HCV and internalization of Dil-labeled HCV particles with no effect on HCV attachment or RNA replication.  AAK1 or GAK depletion impaired epidermal growth factor (EGF)-mediated enhanced HCV entry and endocytosis of EGF receptor (EGFR), an HCV entry cofactor and erlotinib's cancer target.  Moreover, either RNA interference-mediated depletion of AP2M1 or NUMB, each a substrate of AAK1 and/or GAK, or overexpression of either an AP2M1 or NUMB phosphorylation site mutant inhibited HCV entry.  Last, in addn. to affecting assembly, sunitinib and erlotinib inhibited HCV entry at a postbinding step, their combination was synergistic, and their antiviral effect was reversed by either AAK1 or GAK overexpression.  Together, these results validate AAK1 and GAK as crit. regulators of HCV entry that function in part by activating EGFR, AP2M1, and NUMB and as the mol. targets underlying the antiviral effect of sunitinib and erlotinib (in addn. to EGFR), resp.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoB1x5sJzyhrLVg90H21EOLACvtfcHk0li0v4-Cf7bXeA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXlvVWlurg%253D&md5=cbdd3e12d29f383c73dc53c0e9954300</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1128%2Fjvi.02705-14&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252Fjvi.02705-14%26sid%3Dliteratum%253Aachs%26aulast%3DNeveu%26aufirst%3DG.%26aulast%3DZiv-Av%26aufirst%3DA.%26aulast%3DBarouch-Bentov%26aufirst%3DR.%26aulast%3DBerkerman%26aufirst%3DE.%26aulast%3DMulholland%26aufirst%3DJ.%26aulast%3DEinav%26aufirst%3DS.%26atitle%3DAP-2-Associated%2520Protein%2520Kinase%25201%2520and%2520Cyclin%2520G-Associated%2520Kinase%2520Regulate%2520Hepatitis%2520C%2520Virus%2520Entry%2520and%2520Are%2520Potential%2520Drug%2520Targets%26jtitle%3DJ.%2520Virol.%26date%3D2015%26volume%3D89%26spage%3D4387%26epage%3D4404%26doi%3D10.1128%2Fjvi.02705-14" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Neveu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barouch-Bentov, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ziv-Av, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerber, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacob, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Einav, S.</span></span> <span> </span><span class="NLM_article-title">Identification and Targeting of an Interaction between a Tyrosine Motif within Hepatitis C Virus Core Protein and AP2M1 Essential for Viral Assembly</span>. <i>PLoS Pathog.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>8</i></span>, <span class="NLM_elocation-id">e1002845</span> <span class="refDoi"> DOI: 10.1371/journal.ppat.1002845</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=10.1371%2Fjournal.ppat.1002845" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=22916011" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=1%3ACAS%3A528%3ADC%252BC38Xht1Cns7bF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2012&author=G.+Neveuauthor=R.+Barouch-Bentovauthor=A.+Ziv-Avauthor=D.+Gerberauthor=Y.+Jacobauthor=S.+Einav&title=Identification+and+Targeting+of+an+Interaction+between+a+Tyrosine+Motif+within+Hepatitis+C+Virus+Core+Protein+and+AP2M1+Essential+for+Viral+Assembly&doi=10.1371%2Fjournal.ppat.1002845"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Identification and targeting of an interaction between a tyrosine motif within hepatitis C virus core protein and AP2M1 essential for viral assembly</span></div><div class="casAuthors">Neveu, Gregory; Barouch-Bentov, Rina; Ziv-Av, Amotz; Gerber, Doron; Jacob, Yves; Einav, Shirit</div><div class="citationInfo"><span class="NLM_cas:title">PLoS Pathogens</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">e1002845</span>CODEN:
                <span class="NLM_cas:coden">PPLACN</span>;
        ISSN:<span class="NLM_cas:issn">1553-7374</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Novel therapies are urgently needed against hepatitis C virus infection (HCV), a major global health problem.  The current model of infectious virus prodn. suggests that HCV virions are assembled on or near the surface of lipid droplets, acquire their envelope at the ER, and egress through the secretory pathway.  The mechanisms of HCV assembly and particularly the role of viral-host protein-protein interactions in mediating this process are, however, poorly understood.  We identified a conserved heretofore unrecognized YXXΦ motif (Φ is a bulky hydrophobic residue) within the core protein.  This motif is homologous to sorting signals within host cargo proteins known to mediate binding of AP2M1, the μ subunit of clathrin adaptor protein complex 2 (AP-2), and intracellular trafficking.  Using microfluidics affinity anal., protein-fragment complementation assays, and co-immunopptns. in infected cells, we show that this motif mediates core binding to AP2M1.  YXXΦ mutations, silencing AP2M1 expression or overexpressing a dominant neg. AP2M1 mutant had no effect on HCV RNA replication, however, they dramatically inhibited intra- and extra-cellular infectivity, consistent with a defect in viral assembly.  Quant. confocal immunofluorescence anal. revealed that core's YXXΦ motif mediates recruitment of AP2M1 to lipid droplets and that the obsd. defect in HCV assembly following disruption of core-AP2M1 binding correlates with accumulation of core on lipid droplets, reduced core colocalization with E2 and reduced core localization to trans-Golgi network (TGN), the presumed site of viral particles maturation.  Furthermore, AAK1 and GAK, serine/threonine kinases known to stimulate binding of AP2M1 to host cargo proteins, regulate core-AP2M1 binding and are essential for HCV assembly.  Last, approved anti-cancer drugs that inhibit AAK1 or GAK not only disrupt core-AP2M1 binding, but also significantly inhibit HCV assembly and infectious virus prodn.  These results validate viral-host interactions essential for HCV assembly and yield compds. for pharmaceutical development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpjKaGJVLP2vLVg90H21EOLACvtfcHk0ljjo7m_YTBK4g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xht1Cns7bF&md5=811c0818d4b2500b1b788076ac4abc5d</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1371%2Fjournal.ppat.1002845&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.ppat.1002845%26sid%3Dliteratum%253Aachs%26aulast%3DNeveu%26aufirst%3DG.%26aulast%3DBarouch-Bentov%26aufirst%3DR.%26aulast%3DZiv-Av%26aufirst%3DA.%26aulast%3DGerber%26aufirst%3DD.%26aulast%3DJacob%26aufirst%3DY.%26aulast%3DEinav%26aufirst%3DS.%26atitle%3DIdentification%2520and%2520Targeting%2520of%2520an%2520Interaction%2520between%2520a%2520Tyrosine%2520Motif%2520within%2520Hepatitis%2520C%2520Virus%2520Core%2520Protein%2520and%2520AP2M1%2520Essential%2520for%2520Viral%2520Assembly%26jtitle%3DPLoS%2520Pathog.%26date%3D2012%26volume%3D8%26doi%3D10.1371%2Fjournal.ppat.1002845" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barouch-Bentov, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neveu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schor, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumar, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nicolaescu, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindenbach, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Randall, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Einav, S.</span></span> <span> </span><span class="NLM_article-title">Interactions between the Hepatitis C Virus Nonstructural 2 Protein and Host Adaptor Proteins 1 and 4 Orchestrate Virus Release</span>. <i>mBio</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">21</span>, <span class="refDoi"> DOI: 10.1128/mbio.02233-17</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=10.1128%2Fmbio.02233-17" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2018&pages=1-21&author=F.+Xiaoauthor=S.+Wangauthor=R.+Barouch-Bentovauthor=G.+Neveuauthor=S.+Puauthor=M.+Beerauthor=S.+Schorauthor=S.+Kumarauthor=V.+Nicolaescuauthor=B.+D.+Lindenbachauthor=G.+Randallauthor=S.+Einav&title=Interactions+between+the+Hepatitis+C+Virus+Nonstructural+2+Protein+and+Host+Adaptor+Proteins+1+and+4+Orchestrate+Virus+Release&doi=10.1128%2Fmbio.02233-17"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1128%2Fmbio.02233-17&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252Fmbio.02233-17%26sid%3Dliteratum%253Aachs%26aulast%3DXiao%26aufirst%3DF.%26aulast%3DWang%26aufirst%3DS.%26aulast%3DBarouch-Bentov%26aufirst%3DR.%26aulast%3DNeveu%26aufirst%3DG.%26aulast%3DPu%26aufirst%3DS.%26aulast%3DBeer%26aufirst%3DM.%26aulast%3DSchor%26aufirst%3DS.%26aulast%3DKumar%26aufirst%3DS.%26aulast%3DNicolaescu%26aufirst%3DV.%26aulast%3DLindenbach%26aufirst%3DB.%2BD.%26aulast%3DRandall%26aufirst%3DG.%26aulast%3DEinav%26aufirst%3DS.%26atitle%3DInteractions%2520between%2520the%2520Hepatitis%2520C%2520Virus%2520Nonstructural%25202%2520Protein%2520and%2520Host%2520Adaptor%2520Proteins%25201%2520and%25204%2520Orchestrate%2520Virus%2520Release%26jtitle%3DmBio%26date%3D2018%26volume%3D9%26spage%3D1%26epage%3D21%26doi%3D10.1128%2Fmbio.02233-17" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pu, S.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schor, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bekerman, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zanini, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barouch-Bentov, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nagamine, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Einav, S.</span></span> <span> </span><span class="NLM_article-title">Feasibility and Biological Rationale of Repurposing Sunitinib and Erlotinib for Dengue Treatment</span>. <i>Antiviral Res.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>155</i></span>,  <span class="NLM_fpage">67</span>– <span class="NLM_lpage">75</span>, <span class="refDoi"> DOI: 10.1016/j.antiviral.2018.05.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=10.1016%2Fj.antiviral.2018.05.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=29753658" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=1%3ACAS%3A528%3ADC%252BC1cXpslaqtbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=155&publication_year=2018&pages=67-75&author=S.-Y.+Puauthor=F.+Xiaoauthor=S.+Schorauthor=E.+Bekermanauthor=F.+Zaniniauthor=R.+Barouch-Bentovauthor=C.+M.+Nagamineauthor=S.+Einav&title=Feasibility+and+Biological+Rationale+of+Repurposing+Sunitinib+and+Erlotinib+for+Dengue+Treatment&doi=10.1016%2Fj.antiviral.2018.05.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Feasibility and biological rationale of repurposing sunitinib and erlotinib for dengue treatment</span></div><div class="casAuthors">Pu, Szu-Yuan; Xiao, Fei; Schor, Stanford; Bekerman, Elena; Zanini, Fabio; Barouch-Bentov, Rina; Nagamine, Claude M.; Einav, Shirit</div><div class="citationInfo"><span class="NLM_cas:title">Antiviral Research</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">155</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">67-75</span>CODEN:
                <span class="NLM_cas:coden">ARSRDR</span>;
        ISSN:<span class="NLM_cas:issn">0166-3542</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">There is an urgent need for strategies to combat dengue virus (DENV) infection; a major global threat.  We reported that the cellular kinases AAK1 and GAK regulate intracellular trafficking of multiple viruses and that sunitinib and erlotinib, approved anticancer drugs with potent activity against these kinases, protect DENV-infected mice from mortality.  Nevertheless, further characterization of the therapeutic potential and underlying mechanism of this approach is required prior to clin. evaluation.  Here, we demonstrate that sunitinib/erlotinib combination achieves sustained suppression of systemic infection at approved dose in DENV-infected IFN-α/β and IFN-γ receptor-deficient mice.  Nevertheless, treatment with these blood-brain barrier impermeable drugs delays, yet does not prevent, late-onset paralysis; a common manifestation in this immunodeficient mouse model but not in humans.  Sunitinib and erlotinib treatment also demonstrates efficacy in human primary monocyte-derived dendritic cells.  Addnl., DENV infection induces expression of AAK1 transcripts, but not GAK, via single-cell transcriptomics, and these kinases are important mol. targets underlying the anti-DENV effect of sunitinib and erlotinib.  Lastly, sunitinib/erlotinib combination alters inflammatory cytokine responses in DENV-infected mice.  These findings support feasibility of repurposing sunitinib/erlotinib combination as a host-targeted antiviral approach and contribute to understanding its mechanism of antiviral action.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp2cs8UpuoykLVg90H21EOLACvtfcHk0ljjo7m_YTBK4g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXpslaqtbk%253D&md5=acde697b6e3c6d9f42a2011be6091e83</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1016%2Fj.antiviral.2018.05.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.antiviral.2018.05.001%26sid%3Dliteratum%253Aachs%26aulast%3DPu%26aufirst%3DS.-Y.%26aulast%3DXiao%26aufirst%3DF.%26aulast%3DSchor%26aufirst%3DS.%26aulast%3DBekerman%26aufirst%3DE.%26aulast%3DZanini%26aufirst%3DF.%26aulast%3DBarouch-Bentov%26aufirst%3DR.%26aulast%3DNagamine%26aufirst%3DC.%2BM.%26aulast%3DEinav%26aufirst%3DS.%26atitle%3DFeasibility%2520and%2520Biological%2520Rationale%2520of%2520Repurposing%2520Sunitinib%2520and%2520Erlotinib%2520for%2520Dengue%2520Treatment%26jtitle%3DAntiviral%2520Res.%26date%3D2018%26volume%3D155%26spage%3D67%26epage%3D75%26doi%3D10.1016%2Fj.antiviral.2018.05.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kostich, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamman, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Naidu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dandapani, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Easton, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bourin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baker, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Savelieva, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Louis, J. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dokania, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elavazhagan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vattikundala, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharma, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Das, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shankar, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holenarsipur, V. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gulianello, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Molski, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewis, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pieschl, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">OMalley, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lippy, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nouraldeen, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lanthorn, T. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ye, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Balakrishnan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Denton, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grace, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lentz, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Santone, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Main, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swaffield, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carson, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mandlekar, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vikramadithyan, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nara, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dzierba, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bronson, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Macor, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zaczek, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Westphal, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kiss, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bristow, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Conway, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zambrowicz, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Albright, C. F.</span></span> <span> </span><span class="NLM_article-title">Inhibition of AAK1 Kinase as a Novel Therapeutic Approach to Treat Neuropathic Pain</span>. <i>J. Pharmacol. Exp. Ther.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>358</i></span>,  <span class="NLM_fpage">371</span>– <span class="NLM_lpage">386</span>, <span class="refDoi"> DOI: 10.1124/jpet.116.235333</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=10.1124%2Fjpet.116.235333" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=27411717" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvVyks7rI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=358&publication_year=2016&pages=371-386&author=W.+Kostichauthor=B.+D.+Hammanauthor=Y.-W.+Liauthor=S.+Naiduauthor=K.+Dandapaniauthor=J.+Fengauthor=A.+Eastonauthor=C.+Bourinauthor=K.+Bakerauthor=J.+Allenauthor=K.+Savelievaauthor=J.+V.+Louisauthor=M.+Dokaniaauthor=S.+Elavazhaganauthor=P.+Vattikundalaauthor=V.+Sharmaauthor=M.+L.+Dasauthor=G.+Shankarauthor=A.+Kumarauthor=V.+K.+Holenarsipurauthor=M.+Gulianelloauthor=T.+Molskiauthor=J.+M.+Brownauthor=M.+Lewisauthor=Y.+Huangauthor=Y.+Luauthor=R.+Pieschlauthor=K.+OMalleyauthor=J.+Lippyauthor=A.+Nouraldeenauthor=T.+H.+Lanthornauthor=G.+Yeauthor=A.+Wilsonauthor=A.+Balakrishnanauthor=R.+Dentonauthor=J.+E.+Graceauthor=K.+A.+Lentzauthor=K.+S.+Santoneauthor=Y.+Biauthor=A.+Mainauthor=J.+Swaffieldauthor=K.+Carsonauthor=S.+Mandlekarauthor=R.+K.+Vikramadithyanauthor=S.+J.+Naraauthor=C.+Dzierbaauthor=J.+Bronsonauthor=J.+E.+Macorauthor=R.+Zaczekauthor=R.+Westphalauthor=L.+Kissauthor=L.+Bristowauthor=C.+M.+Conwayauthor=B.+Zambrowiczauthor=C.+F.+Albright&title=Inhibition+of+AAK1+Kinase+as+a+Novel+Therapeutic+Approach+to+Treat+Neuropathic+Pain&doi=10.1124%2Fjpet.116.235333"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of AAK1 kinase as a novel therapeutic approach to treat neuropathic pain</span></div><div class="casAuthors">Kostich, Walter; Hamman, Brian D.; Li, Yu-Wen; Naidu, Sreenivasulu; Dandapani, Kumaran; Feng, Jianlin; Easton, Amy; Bourin, Clotilde; Baker, Kevin; Allen, Jason; Savelieva, Katerina; Louis, Justin V.; Dokania, Manoj; Elavazhagan, Saravanan; Vattikundala, Pradeep; Sharma, Vivek; Das, Manish Lal; Shankar, Ganesh; Kumar, Anoop; Holenarsipur, Vinay K.; Gulianello, Michael; Molski, Ted; Brown, Jeffrey M.; Lewis, Martin; Huang, Yanling; Lu, Yifeng; Pieschl, Rick; O'Malley, Kevin; Lippy, Jonathan; Nouraldeen, Amr; Lanthorn, Thomas H.; Ye, Guilan; Wilson, Alan; Balakrishnan, Anand; Denton, Rex; Grace, James E.; Lentz, Kimberley A.; Santone, Kenneth S.; Bi, Yingzhi; Main, Alan; Swaffield, Jon; Carson, Ken; Mandlekar, Sandhya; Vikramadithyan, Reeba K.; Nara, Susheel J.; Dzierba, Carolyn; Bronson, Joanne; Macor, John E.; Zaczek, Robert; Westphal, Ryan; Kiss, Laszlo; Bristow, Linda; Conway, Charles M.; Zambrowicz, Brian; Albright, Charles F.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Pharmacology and Experimental Therapeutics</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">358</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">371-386</span>CODEN:
                <span class="NLM_cas:coden">JPETAB</span>;
        ISSN:<span class="NLM_cas:issn">1521-0103</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">To identify novel targets for neuropathic pain, 3097 mouse knockout lines were tested in acute and persistent pain behavior assays.  One of the lines from this screen, which contained a null allele of the adapter protein-2 assocd. kinase 1 (AAK1) gene, had a normal response in acute pain assays (hot plate, phase I formalin), but a markedly reduced response to persistent pain in phase II formalin.  AAK1 knockout mice also failed to develop tactile allodynia following the Chung procedure of spinal nerve ligation (SNL).  Based on these findings, potent, small-mol. inhibitors of AAK1 were identified.  Studies in mice showed that one such inhibitor, LP-935509, caused a reduced pain response in phase II formalin and reversed fully established pain behavior following the SNL procedure.  Further studies showed that the inhibitor also reduced evoked pain responses in the rat chronic constriction injury (CCI) model and the rat streptozotocin model of diabetic peripheral neuropathy.  Using a nonbrain-penetrant AAK1 inhibitor and local administration of an AAK1 inhibitor, the relevant pool of AAK1 for antineuropathic action was found to be in the spinal cord.  Consistent with these results, AAK1 inhibitors dose-dependently reduced the increased spontaneous neural activity in the spinal cord caused by CCI and blocked the development of windup induced by repeated elec. stimulation of the paw.  The mechanism of AAK1 antinociception was further investigated with inhibitors of α2 adrenergic and opioid receptors.  These studies showed that α2 adrenergic receptor inhibitors, but not opioid receptor inhibitors, not only prevented AAK1 inhibitor antineuropathic action in behavioral assays, but also blocked the AAK1 inhibitor-induced redn. in spinal neural activity in the rat CCI model.  Hence, AAK1 inhibitors are a novel therapeutic approach to neuropathic pain with activity in animal models that is mechanistically linked (behaviorally and electrophysiol.) to α2 adrenergic signaling, a pathway known to be antinociceptive in humans.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo5cJpGeOPjm7Vg90H21EOLACvtfcHk0lgy1G0jQlGmNA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvVyks7rI&md5=d4cec6b3681b63ba7d9119424e53ab61</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1124%2Fjpet.116.235333&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fjpet.116.235333%26sid%3Dliteratum%253Aachs%26aulast%3DKostich%26aufirst%3DW.%26aulast%3DHamman%26aufirst%3DB.%2BD.%26aulast%3DLi%26aufirst%3DY.-W.%26aulast%3DNaidu%26aufirst%3DS.%26aulast%3DDandapani%26aufirst%3DK.%26aulast%3DFeng%26aufirst%3DJ.%26aulast%3DEaston%26aufirst%3DA.%26aulast%3DBourin%26aufirst%3DC.%26aulast%3DBaker%26aufirst%3DK.%26aulast%3DAllen%26aufirst%3DJ.%26aulast%3DSavelieva%26aufirst%3DK.%26aulast%3DLouis%26aufirst%3DJ.%2BV.%26aulast%3DDokania%26aufirst%3DM.%26aulast%3DElavazhagan%26aufirst%3DS.%26aulast%3DVattikundala%26aufirst%3DP.%26aulast%3DSharma%26aufirst%3DV.%26aulast%3DDas%26aufirst%3DM.%2BL.%26aulast%3DShankar%26aufirst%3DG.%26aulast%3DKumar%26aufirst%3DA.%26aulast%3DHolenarsipur%26aufirst%3DV.%2BK.%26aulast%3DGulianello%26aufirst%3DM.%26aulast%3DMolski%26aufirst%3DT.%26aulast%3DBrown%26aufirst%3DJ.%2BM.%26aulast%3DLewis%26aufirst%3DM.%26aulast%3DHuang%26aufirst%3DY.%26aulast%3DLu%26aufirst%3DY.%26aulast%3DPieschl%26aufirst%3DR.%26aulast%3DOMalley%26aufirst%3DK.%26aulast%3DLippy%26aufirst%3DJ.%26aulast%3DNouraldeen%26aufirst%3DA.%26aulast%3DLanthorn%26aufirst%3DT.%2BH.%26aulast%3DYe%26aufirst%3DG.%26aulast%3DWilson%26aufirst%3DA.%26aulast%3DBalakrishnan%26aufirst%3DA.%26aulast%3DDenton%26aufirst%3DR.%26aulast%3DGrace%26aufirst%3DJ.%2BE.%26aulast%3DLentz%26aufirst%3DK.%2BA.%26aulast%3DSantone%26aufirst%3DK.%2BS.%26aulast%3DBi%26aufirst%3DY.%26aulast%3DMain%26aufirst%3DA.%26aulast%3DSwaffield%26aufirst%3DJ.%26aulast%3DCarson%26aufirst%3DK.%26aulast%3DMandlekar%26aufirst%3DS.%26aulast%3DVikramadithyan%26aufirst%3DR.%2BK.%26aulast%3DNara%26aufirst%3DS.%2BJ.%26aulast%3DDzierba%26aufirst%3DC.%26aulast%3DBronson%26aufirst%3DJ.%26aulast%3DMacor%26aufirst%3DJ.%2BE.%26aulast%3DZaczek%26aufirst%3DR.%26aulast%3DWestphal%26aufirst%3DR.%26aulast%3DKiss%26aufirst%3DL.%26aulast%3DBristow%26aufirst%3DL.%26aulast%3DConway%26aufirst%3DC.%2BM.%26aulast%3DZambrowicz%26aufirst%3DB.%26aulast%3DAlbright%26aufirst%3DC.%2BF.%26atitle%3DInhibition%2520of%2520AAK1%2520Kinase%2520as%2520a%2520Novel%2520Therapeutic%2520Approach%2520to%2520Treat%2520Neuropathic%2520Pain%26jtitle%3DJ.%2520Pharmacol.%2520Exp.%2520Ther.%26date%3D2016%26volume%3D358%26spage%3D371%26epage%3D386%26doi%3D10.1124%2Fjpet.116.235333" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bronson, J.</span>; <span class="NLM_string-name">Chen, L.</span>; <span class="NLM_string-name">Ditta, J.</span>; <span class="NLM_string-name">Dzierba, C. D.</span>; <span class="NLM_string-name">Jalagam, P. R.</span>; <span class="NLM_string-name">Luo, G.</span>; <span class="NLM_string-name">Macor, J.</span>; <span class="NLM_string-name">Maishal, T. K.</span>; <span class="NLM_string-name">Nara, S. J.</span>; <span class="NLM_string-name">Rajamani, R.</span>; <span class="NLM_string-name">Sistla, R. K.</span>; <span class="NLM_string-name">Thangavel, S.</span></span> <span> </span><span class="NLM_article-title">Biaryl Kinase Inhibitors</span>. WO <span class="NLM_patent">2017/059085</span>, <span class="NLM_year">2017</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2017&author=J.+Bronson&author=L.+Chen&author=J.+Ditta&author=C.+D.+Dzierba&author=P.+R.+Jalagam&author=G.+Luo&author=J.+Macor&author=T.+K.+Maishal&author=S.+J.+Nara&author=R.+Rajamani&author=R.+K.+Sistla&author=S.+Thangavel&title=Biaryl+Kinase+Inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DBronson%26aufirst%3DJ.%26atitle%3DBiaryl%2520Kinase%2520Inhibitors%26date%3D2017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kuai, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ong, S.-E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Madison, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duvall, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewis, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luce, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Conner, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pearlman, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wood, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schreiber, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carr, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scolnick, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haggarty, S. J.</span></span> <span> </span><span class="NLM_article-title">AAK1 Identified as an Inhibitor of Neuregulin-1/ErbB4-Dependent Neurotrophic Factor Signaling Using Integrative Chemical Genomics and Proteomics</span>. <i>Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">891</span>– <span class="NLM_lpage">906</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2011.03.017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=10.1016%2Fj.chembiol.2011.03.017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=21802010" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=1%3ACAS%3A528%3ADC%252BC3MXpsFaruro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2011&pages=891-906&author=L.+Kuaiauthor=S.-E.+Ongauthor=J.+M.+Madisonauthor=X.+Wangauthor=J.+R.+Duvallauthor=T.+A.+Lewisauthor=C.+J.+Luceauthor=S.+D.+Connerauthor=D.+A.+Pearlmanauthor=J.+L.+Woodauthor=S.+L.+Schreiberauthor=S.+A.+Carrauthor=E.+M.+Scolnickauthor=S.+J.+Haggarty&title=AAK1+Identified+as+an+Inhibitor+of+Neuregulin-1%2FErbB4-Dependent+Neurotrophic+Factor+Signaling+Using+Integrative+Chemical+Genomics+and+Proteomics&doi=10.1016%2Fj.chembiol.2011.03.017"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">AAK1 Identified as an Inhibitor of Neuregulin-1/ErbB4-Dependent Neurotrophic Factor Signaling Using Integrative Chemical Genomics and Proteomics</span></div><div class="casAuthors">Kuai, Letian; Ong, Shao-En; Madison, Jon M.; Wang, Xiang; Duvall, Jeremy R.; Lewis, Timothy A.; Luce, Catherine J.; Conner, Sean D.; Pearlman, David A.; Wood, John L.; Schreiber, Stuart L.; Carr, Steven A.; Scolnick, Edward M.; Haggarty, Stephen J.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">891-906</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Summary: Target identification remains challenging for the field of chem. biol.  We describe an integrative chem. genomic and proteomic approach combining the use of differentially active analogs of small mol. probes with stable isotope labeling by amino acids in cell culture-mediated affinity enrichment, followed by subsequent testing of candidate targets using RNA interference-mediated gene silencing.  We applied this approach to characterizing the natural product K252a and its ability to potentiate neuregulin-1 (Nrg1)/ErbB4 (v-erb-a erythroblastic leukemia viral oncogene homolog 4)-dependent neurotrophic factor signaling and neuritogenesis.  We show that AAK1 (adaptor-assocd. kinase 1) is a relevant target of K252a, and that the loss of AAK1 alters ErbB4 trafficking and expression levels, providing evidence for a previously unrecognized role for AAK1 in Nrg1-mediated neurotrophic factor signaling.  Similar strategies should lead to the discovery of novel targets for therapeutic development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo0dEttut6YzLVg90H21EOLACvtfcHk0li6ocTGDvXzzA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXpsFaruro%253D&md5=3fcf22dfc3defc7b684a643444a7838a</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2011.03.017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2011.03.017%26sid%3Dliteratum%253Aachs%26aulast%3DKuai%26aufirst%3DL.%26aulast%3DOng%26aufirst%3DS.-E.%26aulast%3DMadison%26aufirst%3DJ.%2BM.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DDuvall%26aufirst%3DJ.%2BR.%26aulast%3DLewis%26aufirst%3DT.%2BA.%26aulast%3DLuce%26aufirst%3DC.%2BJ.%26aulast%3DConner%26aufirst%3DS.%2BD.%26aulast%3DPearlman%26aufirst%3DD.%2BA.%26aulast%3DWood%26aufirst%3DJ.%2BL.%26aulast%3DSchreiber%26aufirst%3DS.%2BL.%26aulast%3DCarr%26aufirst%3DS.%2BA.%26aulast%3DScolnick%26aufirst%3DE.%2BM.%26aulast%3DHaggarty%26aufirst%3DS.%2BJ.%26atitle%3DAAK1%2520Identified%2520as%2520an%2520Inhibitor%2520of%2520Neuregulin-1%252FErbB4-Dependent%2520Neurotrophic%2520Factor%2520Signaling%2520Using%2520Integrative%2520Chemical%2520Genomics%2520and%2520Proteomics%26jtitle%3DChem.%2520Biol.%26date%3D2011%26volume%3D18%26spage%3D891%26epage%3D906%26doi%3D10.1016%2Fj.chembiol.2011.03.017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bamborough, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drewry, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harper, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, G. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schneider, K.</span></span> <span> </span><span class="NLM_article-title">Assessment of Chemical Coverage of Kinome Space and Its Implications for Kinase Drug Discovery</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>51</i></span>,  <span class="NLM_fpage">7898</span>– <span class="NLM_lpage">7914</span>, <span class="refDoi"> DOI: 10.1021/jm8011036</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm8011036" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhsVequ7zI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2008&pages=7898-7914&author=P.+Bamboroughauthor=D.+Drewryauthor=G.+Harperauthor=G.+K.+Smithauthor=K.+Schneider&title=Assessment+of+Chemical+Coverage+of+Kinome+Space+and+Its+Implications+for+Kinase+Drug+Discovery&doi=10.1021%2Fjm8011036"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Assessment of Chemical Coverage of Kinome Space and Its Implications for Kinase Drug Discovery</span></div><div class="casAuthors">Bamborough, Paul; Drewry, David; Harper, Gavin; Smith, Gary K.; Schneider, Klaus</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">7898-7914</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">More than 500 compds. chosen to represent kinase inhibitor space have been screened against a panel of over 200 protein kinases.  Significant results include the identification of hits against new kinases including PIM1 and MPSK1, and the expansion of the inhibition profiles of several literature compds.  A detailed anal. of the data through the use of affinity fingerprints has produced findings with implications for biol. target selection, the choice of tool compds. for target validation, and lead discovery and optimization.  In a detailed examn. of the tyrosine kinases, interesting relationships have been found between targets and compds.  Taken together, these results show how broad cross-profiling can provide important insights to assist kinase drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrBRI-arQk3t7Vg90H21EOLACvtfcHk0lha8JLZaHVIHA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhsVequ7zI&md5=8d98c80fb157b987a19e32a1efb53103</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1021%2Fjm8011036&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm8011036%26sid%3Dliteratum%253Aachs%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DDrewry%26aufirst%3DD.%26aulast%3DHarper%26aufirst%3DG.%26aulast%3DSmith%26aufirst%3DG.%2BK.%26aulast%3DSchneider%26aufirst%3DK.%26atitle%3DAssessment%2520of%2520Chemical%2520Coverage%2520of%2520Kinome%2520Space%2520and%2520Its%2520Implications%2520for%2520Kinase%2520Drug%2520Discovery%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2008%26volume%3D51%26spage%3D7898%26epage%3D7914%26doi%3D10.1021%2Fjm8011036" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Barl, N. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sansiaume-Dagousset, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karaghiosoff, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knochel, P.</span></span> <span> </span><span class="NLM_article-title">Full Functionalization of the 7-Azaindole Scaffold by Selective Metalation and Sulfoxide/Magnesium Exchange</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">10093</span>– <span class="NLM_lpage">10096</span>, <span class="refDoi"> DOI: 10.1002/anie.201303490</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=10.1002%2Fanie.201303490" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtFOkurnL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2013&pages=10093-10096&author=N.+M.+Barlauthor=E.+Sansiaume-Dagoussetauthor=K.+Karaghiosoffauthor=P.+Knochel&title=Full+Functionalization+of+the+7-Azaindole+Scaffold+by+Selective+Metalation+and+Sulfoxide%2FMagnesium+Exchange&doi=10.1002%2Fanie.201303490"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Full Functionalization of the 7-Azaindole Scaffold by Selective Metalation and Sulfoxide/Magnesium Exchange</span></div><div class="casAuthors">Barl, Nadja M.; Sansiaume-Dagousset, Elodie; Karaghiosoff, Konstantin; Knochel, Paul</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">38</span>),
    <span class="NLM_cas:pages">10093-10096</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">The authors report a novel methodol. for the mild full functionalization of the 7-azaindole scaffold by combining directed metalation and halogen/magnesium and sulfoxide/magnesium exchange using substituted 7-azaindole I as the key intermediate.  Introducing the sulfoxide substituent on C5 was essential for all further transformations.  For example, reacting I with TMPMgCl·LiCl/THF (TMP = 2,2,6,6-tetramethylpiperidine), followed by treatment with ZnCl2, CuCN·2LiCl, and then 2-furancarbonyl chloride gave furanyl ketone deriv. II in 63% yield.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoDM4RBErZ9Q7Vg90H21EOLACvtfcHk0lha8JLZaHVIHA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtFOkurnL&md5=f4a2dd87924c5c092e226b1619d4323a</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1002%2Fanie.201303490&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201303490%26sid%3Dliteratum%253Aachs%26aulast%3DBarl%26aufirst%3DN.%2BM.%26aulast%3DSansiaume-Dagousset%26aufirst%3DE.%26aulast%3DKaraghiosoff%26aufirst%3DK.%26aulast%3DKnochel%26aufirst%3DP.%26atitle%3DFull%2520Functionalization%2520of%2520the%25207-Azaindole%2520Scaffold%2520by%2520Selective%2520Metalation%2520and%2520Sulfoxide%252FMagnesium%2520Exchange%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2013%26volume%3D52%26spage%3D10093%26epage%3D10096%26doi%3D10.1002%2Fanie.201303490" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brumsted, C. J.</span>; <span class="NLM_string-name">Moorlag, H.</span>; <span class="NLM_string-name">Radinov, R. N.</span>; <span class="NLM_string-name">Ren, Y.</span>; <span class="NLM_string-name">Waldmeier, P.</span></span> <span> </span><span class="NLM_article-title">Method for Preparation of N-{3-[5-(4-Chlorophenyl)-1H-Pyrrolo[2,3-b]Pyridine-3-Carbonyl]-2,4-Difluorophenyl} Propane-1-Sulfonamide</span>. WO <span class="NLM_patent">2012/010538 A2</span>, <span class="NLM_year">2012</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2012&author=C.+J.+Brumsted&author=H.+Moorlag&author=R.+N.+Radinov&author=Y.+Ren&author=P.+Waldmeier&title=Method+for+Preparation+of+N-%7B3-%5B5-%284-Chlorophenyl%29-1H-Pyrrolo%5B2%2C3-b%5DPyridine-3-Carbonyl%5D-2%2C4-Difluorophenyl%7D+Propane-1-Sulfonamide"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DBrumsted%26aufirst%3DC.%2BJ.%26atitle%3DMethod%2520for%2520Preparation%2520of%2520N-%257B3-%255B5-%25284-Chlorophenyl%2529-1H-Pyrrolo%255B2%252C3-b%255DPyridine-3-Carbonyl%255D-2%252C4-Difluorophenyl%257D%2520Propane-1-Sulfonamide%26date%3D2012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Han, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Green, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pfeifer, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gosselin, F.</span></span> <span> </span><span class="NLM_article-title">Highly Regioselective and Practical Synthesis of 5-Bromo-4-Chloro-3- Nitro-7-Azaindole</span>. <i>Org. Process Res. Dev.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">664</span>– <span class="NLM_lpage">668</span>, <span class="refDoi"> DOI: 10.1021/acs.oprd.7b00060</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.oprd.7b00060" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=1%3ACAS%3A528%3ADC%252BC2sXltVSrtL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2017&pages=664-668&author=C.+Hanauthor=K.+Greenauthor=E.+Pfeiferauthor=F.+Gosselin&title=Highly+Regioselective+and+Practical+Synthesis+of+5-Bromo-4-Chloro-3-+Nitro-7-Azaindole&doi=10.1021%2Facs.oprd.7b00060"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Highly Regioselective and Practical Synthesis of 5-Bromo-4-chloro-3-nitro-7-azaindole</span></div><div class="casAuthors">Han, Chong; Green, Keena; Pfeifer, Eugen; Gosselin, Francis</div><div class="citationInfo"><span class="NLM_cas:title">Organic Process Research & Development</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">664-668</span>CODEN:
                <span class="NLM_cas:coden">OPRDFK</span>;
        ISSN:<span class="NLM_cas:issn">1083-6160</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We report an efficient and highly regiocontrolled route to prep. a functionalized 7-azaindole deriv. (5-bromo-4-chloro-3-nitro-7-azaindole) from readily available parent 7-azaindole featuring a highly regioselective bromination of the 4-chloro-3-nitro-7-azaindole intermediate.  In addn. to the high efficiency and excellent control of regioisomeric impurities, the process is operationally simple by isolating each product via direct crystn. from the reaction mixt. with no liq.-liq. extns. or distn. steps needed.  We demonstrated the route on >50 kg scale and 46% overall yield to provide the target product in 97% purity by HPLC, which can serve as a useful building block for the prepn. of a series of 3,4,5-substituted-7-azaindole derivs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrhmXJx-GEcXbVg90H21EOLACvtfcHk0liptp9PtUeP6A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXltVSrtL4%253D&md5=ab19646db3439c8f42596749a30e0df7</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Facs.oprd.7b00060&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.oprd.7b00060%26sid%3Dliteratum%253Aachs%26aulast%3DHan%26aufirst%3DC.%26aulast%3DGreen%26aufirst%3DK.%26aulast%3DPfeifer%26aufirst%3DE.%26aulast%3DGosselin%26aufirst%3DF.%26atitle%3DHighly%2520Regioselective%2520and%2520Practical%2520Synthesis%2520of%25205-Bromo-4-Chloro-3-%2520Nitro-7-Azaindole%26jtitle%3DOrg.%2520Process%2520Res.%2520Dev.%26date%3D2017%26volume%3D21%26spage%3D664%26epage%3D668%26doi%3D10.1021%2Facs.oprd.7b00060" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Stokes, S.</span>; <span class="NLM_string-name">Graham, C. J.</span>; <span class="NLM_string-name">Ray, S. C.</span>; <span class="NLM_string-name">Stefaniak, E. J.</span></span> <span> </span><span class="NLM_article-title">1H-Pyrrolo[2,3-b]Pyridine Derivatives and Their Use as Kinase Inhibitors</span>. WO <span class="NLM_patent">2013/114113 A1</span>, <span class="NLM_year">2013</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2013&author=S.+Stokes&author=C.+J.+Graham&author=S.+C.+Ray&author=E.+J.+Stefaniak&title=1H-Pyrrolo%5B2%2C3-b%5DPyridine+Derivatives+and+Their+Use+as+Kinase+Inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DStokes%26aufirst%3DS.%26atitle%3D1H-Pyrrolo%255B2%252C3-b%255DPyridine%2520Derivatives%2520and%2520Their%2520Use%2520as%2520Kinase%2520Inhibitors%26date%3D2013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gao, L.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kovackova, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Šála, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramadori, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Jonghe, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herdewijn, P.</span></span> <span> </span><span class="NLM_article-title">Discovery of Dual Death-Associated Protein Related Apoptosis Inducing Protein Kinase 1 and 2 Inhibitors by a Scaffold Hopping Approach</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">7624</span>– <span class="NLM_lpage">7643</span>, <span class="refDoi"> DOI: 10.1021/jm5007929</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm5007929" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsVGhtL7P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=7624-7643&author=L.-J.+Gaoauthor=S.+Kovackovaauthor=M.+%C5%A0%C3%A1laauthor=A.+T.+Ramadoriauthor=S.+De+Jongheauthor=P.+Herdewijn&title=Discovery+of+Dual+Death-Associated+Protein+Related+Apoptosis+Inducing+Protein+Kinase+1+and+2+Inhibitors+by+a+Scaffold+Hopping+Approach&doi=10.1021%2Fjm5007929"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Dual Death-Associated Protein Related Apoptosis Inducing Protein Kinase 1 and 2 Inhibitors by a Scaffold Hopping Approach</span></div><div class="casAuthors">Gao, Ling-Jie; Kovackova, Sona; Sala, Michal; Ramadori, Anna Teresa; De Jonghe, Steven; Herdewijn, Piet</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">7624-7643</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">DRAK2 emerged as a promising drug target for the treatment of autoimmune diseases and to prevent graft rejection after organ transplantation.  Screening of a compd. library in a DRAK2 binding assay led to the identification of an isothiazolo[5,4-b]pyridine deriv. as a novel ligand for DRAK2, displaying a Kd value of 1.6 μM.  Subsequent medicinal chem. work led to the discovery of a thieno[2,3-b]pyridine deriv. with strong DRAK2 binding affinity (Kd = 9 nM).  Moreover, this compd. also behaves as a functional inhibitor of DRAK2 enzymic activity, displaying an IC50 value of 0.82 μM, although lacking selectivity, when tested against DRAK1.  This paper describes for the first time functionally active dual DRAK1 and DRAK2 inhibitors that can be used as starting point for the synthesis of chem. tool compds. to study DRAK1 and DRAK2 biol., or they can be considered as hit compds. for hit-to-lead optimization campaigns in drug discovery programs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpkAuMAD_Ac_LVg90H21EOLACvtfcHk0lgMSyio7PBb2Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsVGhtL7P&md5=15353fd901634efed8d2953a1cf72ecf</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1021%2Fjm5007929&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm5007929%26sid%3Dliteratum%253Aachs%26aulast%3DGao%26aufirst%3DL.-J.%26aulast%3DKovackova%26aufirst%3DS.%26aulast%3D%25C5%25A0%25C3%25A1la%26aufirst%3DM.%26aulast%3DRamadori%26aufirst%3DA.%2BT.%26aulast%3DDe%2BJonghe%26aufirst%3DS.%26aulast%3DHerdewijn%26aufirst%3DP.%26atitle%3DDiscovery%2520of%2520Dual%2520Death-Associated%2520Protein%2520Related%2520Apoptosis%2520Inducing%2520Protein%2520Kinase%25201%2520and%25202%2520Inhibitors%2520by%2520a%2520Scaffold%2520Hopping%2520Approach%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D7624%26epage%3D7643%26doi%3D10.1021%2Fjm5007929" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Le Huerou, Y.</span>; <span class="NLM_string-name">Blake, J.</span>; <span class="NLM_string-name">Gunwardana, I.</span>; <span class="NLM_string-name">Mohr, P.</span>; <span class="NLM_string-name">Wallace, E.</span>; <span class="NLM_string-name">Wang, B.</span>; <span class="NLM_string-name">Chicarelli, M.</span>; <span class="NLM_string-name">Lyon, M.</span></span> <span> </span><span class="NLM_article-title">Pyrrolopyridines as Kinase Inhibitors</span>. WO <span class="NLM_patent">2009/140320</span>, <span class="NLM_year">2009</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2009&author=Y.+Le+Huerou&author=J.+Blake&author=I.+Gunwardana&author=P.+Mohr&author=E.+Wallace&author=B.+Wang&author=M.+Chicarelli&author=M.+Lyon&title=Pyrrolopyridines+as+Kinase+Inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DLe%2BHuerou%26aufirst%3DY.%26atitle%3DPyrrolopyridines%2520as%2520Kinase%2520Inhibitors%26date%3D2009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Stavenger, R.</span>; <span class="NLM_string-name">Witherington, J.</span>; <span class="NLM_string-name">Rawlings, D.</span>; <span class="NLM_string-name">Holt, D.</span>; <span class="NLM_string-name">Chan, G.</span></span> <span> </span><span class="NLM_article-title">CHK1 Kinase Inhibitors</span>. WO <span class="NLM_patent">03/028724</span>, <span class="NLM_year">2003</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2003&author=R.+Stavenger&author=J.+Witherington&author=D.+Rawlings&author=D.+Holt&author=G.+Chan&title=CHK1+Kinase+Inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DStavenger%26aufirst%3DR.%26atitle%3DCHK1%2520Kinase%2520Inhibitors%26date%3D2003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bahekar, R.
H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jain, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jadav, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prajapati, V. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, D. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gupta, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharma, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tom, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bandyopadhya, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Modi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, P. R.</span></span> <span> </span><span class="NLM_article-title">Synthesis and antidiabetic activity of 2,5-disubstituted-3-imidazol-2-yl-pyrrolo[2,3-b]pyridines and thieno[2,3-b]pyridines</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">6782</span>– <span class="NLM_lpage">6795</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2007.08.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=10.1016%2Fj.bmc.2007.08.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=17723306" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtVCitr7K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2007&pages=6782-6795&author=R.%0AH.+Bahekarauthor=M.+R.+Jainauthor=P.+A.+Jadavauthor=V.+M.+Prajapatiauthor=D.+N.+Patelauthor=A.+A.+Guptaauthor=A.+Sharmaauthor=R.+Tomauthor=D.+Bandyopadhyaauthor=H.+Modiauthor=P.+R.+Patel&title=Synthesis+and+antidiabetic+activity+of+2%2C5-disubstituted-3-imidazol-2-yl-pyrrolo%5B2%2C3-b%5Dpyridines+and+thieno%5B2%2C3-b%5Dpyridines&doi=10.1016%2Fj.bmc.2007.08.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis and antidiabetic activity of 2,5-disubstituted-3-imidazol-2-yl-pyrrolo[2,3-b]pyridines and thieno[2,3-b]pyridines</span></div><div class="casAuthors">Bahekar, Rajesh H.; Jain, Mukul R.; Jadav, Pradip A.; Prajapati, Vijay M.; Patel, Dipam N.; Gupta, Arun A.; Sharma, Ajay; Tom, Robby; Bandyopadhya, Debdutta; Modi, Honey; Patel, Pankaj R.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">6782-6795</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">In the present investigation, two series of 2,5-disubstituted-3-imidazol-2-yl-pyrrolo[2,3-b]pyridines (2a-l) and thieno[2,3-b]pyridines (3a-l) were designed as analogs of BL 11282 (1).  The in vitro glucose dependent insulinotropic activity of all the test compds. was evaluated using RIN5F cell based assay and all the test compds. showed glucose and concn. dependent insulin secretion.  The in vivo antidiabetic activities of most potent compds. from each series (I and II) were assessed in C57BL/6J mice.  Compds. I and II showed dose dependent insulin secretion and significant glucose redn. in vivo.  In general, compds. I and II were found to be equipotent at all the three different doses selected and with respect to BL 11282, both the test compds. were found to be more potent, at all the time points.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGopXNGDHu3ZVbVg90H21EOLACvtfcHk0lgMSyio7PBb2Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtVCitr7K&md5=8b0133862be93a285639df3c0adbbc95</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2007.08.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2007.08.005%26sid%3Dliteratum%253Aachs%26aulast%3DBahekar%26aufirst%3DR.%2BH.%26aulast%3DJain%26aufirst%3DM.%2BR.%26aulast%3DJadav%26aufirst%3DP.%2BA.%26aulast%3DPrajapati%26aufirst%3DV.%2BM.%26aulast%3DPatel%26aufirst%3DD.%2BN.%26aulast%3DGupta%26aufirst%3DA.%2BA.%26aulast%3DSharma%26aufirst%3DA.%26aulast%3DTom%26aufirst%3DR.%26aulast%3DBandyopadhya%26aufirst%3DD.%26aulast%3DModi%26aufirst%3DH.%26aulast%3DPatel%26aufirst%3DP.%2BR.%26atitle%3DSynthesis%2520and%2520antidiabetic%2520activity%2520of%25202%252C5-disubstituted-3-imidazol-2-yl-pyrrolo%255B2%252C3-b%255Dpyridines%2520and%2520thieno%255B2%252C3-b%255Dpyridines%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2007%26volume%3D15%26spage%3D6782%26epage%3D6795%26doi%3D10.1016%2Fj.bmc.2007.08.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chavan, N. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nayak, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kusurkar, R. S.</span></span> <span> </span><span class="NLM_article-title">A rapid method toward the synthesis of new substituted tetrahydro α-carbolines and α-carbolines</span>. <i>Tetrahedron</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">1827</span>– <span class="NLM_lpage">1831</span>, <span class="refDoi"> DOI: 10.1016/j.tet.2010.01.042</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=10.1016%2Fj.tet.2010.01.042" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=1%3ACAS%3A528%3ADC%252BC3cXitVSitL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=1827-1831&author=N.+L.+Chavanauthor=S.+K.+Nayakauthor=R.+S.+Kusurkar&title=A+rapid+method+toward+the+synthesis+of+new+substituted+tetrahydro+%CE%B1-carbolines+and+%CE%B1-carbolines&doi=10.1016%2Fj.tet.2010.01.042"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">A rapid method toward the synthesis of new substituted tetrahydro α-carbolines and α-carbolines</span></div><div class="casAuthors">Chavan, Neelam L.; Nayak, Sandip K.; Kusurkar, Radhika S.</div><div class="citationInfo"><span class="NLM_cas:title">Tetrahedron</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1827-1831</span>CODEN:
                <span class="NLM_cas:coden">TETRAB</span>;
        ISSN:<span class="NLM_cas:issn">0040-4020</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A rapid and efficient method for stereoselective synthesis of new substituted tetrahydro-α-carbolines using Diels-Alder reaction under microwave irradn. has been developed.  Further, dehydrogenation of these adducts resulted in synthesis of new substituted α-carbolines.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrprueVyzy4rLVg90H21EOLACvtfcHk0lhx8sdY0vAXDQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXitVSitL0%253D&md5=e3a285d8ff93115769f926259da3648d</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1016%2Fj.tet.2010.01.042&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tet.2010.01.042%26sid%3Dliteratum%253Aachs%26aulast%3DChavan%26aufirst%3DN.%2BL.%26aulast%3DNayak%26aufirst%3DS.%2BK.%26aulast%3DKusurkar%26aufirst%3DR.%2BS.%26atitle%3DA%2520rapid%2520method%2520toward%2520the%2520synthesis%2520of%2520new%2520substituted%2520tetrahydro%2520%25CE%25B1-carbolines%2520and%2520%25CE%25B1-carbolines%26jtitle%3DTetrahedron%26date%3D2010%26volume%3D66%26spage%3D1827%26epage%3D1831%26doi%3D10.1016%2Fj.tet.2010.01.042" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chinta, B. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baire, B.</span></span> <span> </span><span class="NLM_article-title">Reactivity of indole-3-alkoxides in the absence of acids: Rapid synthesis of homo- bisindolylmethanes</span>. <i>Tetrahedron</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>72</i></span>,  <span class="NLM_fpage">8106</span>– <span class="NLM_lpage">8116</span>, <span class="refDoi"> DOI: 10.1016/j.tet.2016.10.067</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=10.1016%2Fj.tet.2016.10.067" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvVahsrbM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=72&publication_year=2016&pages=8106-8116&author=B.+S.+Chintaauthor=B.+Baire&title=Reactivity+of+indole-3-alkoxides+in+the+absence+of+acids%3A+Rapid+synthesis+of+homo-+bisindolylmethanes&doi=10.1016%2Fj.tet.2016.10.067"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Reactivity of indole-3-alkoxides in the absence of acids: rapid synthesis of homo-bisindolylmethanes</span></div><div class="casAuthors">Chinta, Bhavani Shankar; Baire, Beeraiah</div><div class="citationInfo"><span class="NLM_cas:title">Tetrahedron</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">72</span>
        (<span class="NLM_cas:issue">49</span>),
    <span class="NLM_cas:pages">8106-8116</span>CODEN:
                <span class="NLM_cas:coden">TETRAB</span>;
        ISSN:<span class="NLM_cas:issn">0040-4020</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">An unprecedented behavior of indole-3-alkoxides (MgX or Li) in situ generated from indolylcarbaldehydes and Grignard/lithium reagents was reported.  Tuning the electronic properties of the alkoxides offered the direct and selective construction of bisindolylmethanes I [R = Me, ethynyl, 2-thienyl, etc.; R1 = Me, n-hexyl, Bn; X = H, OMe, Br] and indole-3-carbinols II [R = Me, Ph, 2-thienyl, etc.; R1 = Me, Bn, Ts; Y = CH, N].  This process showed very broad scope and represented the reagent (external) free, greener synthesis of structurally divergent bisindolylmethanes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpIPC5u_aaMT7Vg90H21EOLACvtfcHk0lhx8sdY0vAXDQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvVahsrbM&md5=4a8553c51b7809a2d7e11f536fb52e6f</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1016%2Fj.tet.2016.10.067&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tet.2016.10.067%26sid%3Dliteratum%253Aachs%26aulast%3DChinta%26aufirst%3DB.%2BS.%26aulast%3DBaire%26aufirst%3DB.%26atitle%3DReactivity%2520of%2520indole-3-alkoxides%2520in%2520the%2520absence%2520of%2520acids%253A%2520Rapid%2520synthesis%2520of%2520homo-%2520bisindolylmethanes%26jtitle%3DTetrahedron%26date%3D2016%26volume%3D72%26spage%3D8106%26epage%3D8116%26doi%3D10.1016%2Fj.tet.2016.10.067" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McCoull, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hennessy, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blades, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Box, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chuaqui, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dowling, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davies, C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferguson, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goldberg, F. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Howe, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kemmitt, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lamont, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Madden, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McWhirter, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Varnes, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ward, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williams, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, B.</span></span> <span> </span><span class="NLM_article-title">Identification and Optimisation of 7-Azaindole PAK1 Inhibitors with Improved Potency and Kinase Selectivity</span>. <i>MedChemComm</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">1533</span>– <span class="NLM_lpage">1539</span>, <span class="refDoi"> DOI: 10.1039/c4md00280f</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=10.1039%2Fc4md00280f" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtlOksrvL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=1533-1539&author=W.+McCoullauthor=E.+J.+Hennessyauthor=K.+Bladesauthor=M.+R.+Boxauthor=C.+Chuaquiauthor=J.+E.+Dowlingauthor=C.+D.+Daviesauthor=A.+D.+Fergusonauthor=F.+W.+Goldbergauthor=N.+J.+Howeauthor=P.+D.+Kemmittauthor=G.+M.+Lamontauthor=K.+Maddenauthor=C.+McWhirterauthor=J.+G.+Varnesauthor=R.+A.+Wardauthor=J.+D.+Williamsauthor=B.+Yang&title=Identification+and+Optimisation+of+7-Azaindole+PAK1+Inhibitors+with+Improved+Potency+and+Kinase+Selectivity&doi=10.1039%2Fc4md00280f"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Identification and optimisation of 7-azaindole PAK1 inhibitors with improved potency and kinase selectivity</span></div><div class="casAuthors">McCoull, William; Hennessy, Edward J.; Blades, Kevin; Box, Matthew R.; Chuaqui, Claudio; Dowling, James E.; Davies, Christopher D.; Ferguson, Andrew D.; Goldberg, Frederick W.; Howe, Nicholas J.; Kemmitt, Paul D.; Lamont, Gillian M.; Madden, Katrina; McWhirter, Claire; Varnes, Jeffrey G.; Ward, Richard A.; Williams, Jason D.; Yang, Bin</div><div class="citationInfo"><span class="NLM_cas:title">MedChemComm</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1533-1539</span>CODEN:
                <span class="NLM_cas:coden">MCCEAY</span>;
        ISSN:<span class="NLM_cas:issn">2040-2503</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A novel series of PAK1 inhibitors was discovered from a kinase directed screen.  SAR exploration in the selectivity pocket and solvent tail regions was conducted to understand and optimize PAK1 potency and selectivity against targeted kinases.  A liganded PAK1 crystal structure was utilized to guide compd. design.  Permeability and kinase selectivity impacted the translation of enzyme to cellular PAK1 potency.  Compd. 36 (AZ-PAK-36) demonstrated improved Gini coeff., good PAK1 cellular potency and has utility as a tool compd. for target validation studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrFuGgNmSEGu7Vg90H21EOLACvtfcHk0lhx8sdY0vAXDQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtlOksrvL&md5=bda79c5a2a067284e8520c8fb97069fd</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1039%2Fc4md00280f&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc4md00280f%26sid%3Dliteratum%253Aachs%26aulast%3DMcCoull%26aufirst%3DW.%26aulast%3DHennessy%26aufirst%3DE.%2BJ.%26aulast%3DBlades%26aufirst%3DK.%26aulast%3DBox%26aufirst%3DM.%2BR.%26aulast%3DChuaqui%26aufirst%3DC.%26aulast%3DDowling%26aufirst%3DJ.%2BE.%26aulast%3DDavies%26aufirst%3DC.%2BD.%26aulast%3DFerguson%26aufirst%3DA.%2BD.%26aulast%3DGoldberg%26aufirst%3DF.%2BW.%26aulast%3DHowe%26aufirst%3DN.%2BJ.%26aulast%3DKemmitt%26aufirst%3DP.%2BD.%26aulast%3DLamont%26aufirst%3DG.%2BM.%26aulast%3DMadden%26aufirst%3DK.%26aulast%3DMcWhirter%26aufirst%3DC.%26aulast%3DVarnes%26aufirst%3DJ.%2BG.%26aulast%3DWard%26aufirst%3DR.%2BA.%26aulast%3DWilliams%26aufirst%3DJ.%2BD.%26aulast%3DYang%26aufirst%3DB.%26atitle%3DIdentification%2520and%2520Optimisation%2520of%25207-Azaindole%2520PAK1%2520Inhibitors%2520with%2520Improved%2520Potency%2520and%2520Kinase%2520Selectivity%26jtitle%3DMedChemComm%26date%3D2014%26volume%3D5%26spage%3D1533%26epage%3D1539%26doi%3D10.1039%2Fc4md00280f" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gourdain, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dairou, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Denhez, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bui, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodrigues-Lima, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Janel, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delabar, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cariou, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dodd, R. H.</span></span> <span> </span><span class="NLM_article-title">Development of DANDYs, New 3,5-Diaryl-7-Azaindoles Demonstrating Potent DYRK1A Kinase Inhibitory Activity</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">9569</span>– <span class="NLM_lpage">9585</span>, <span class="refDoi"> DOI: 10.1021/jm401049v</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm401049v" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhslWrt77K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=9569-9585&author=S.+Gourdainauthor=J.+Dairouauthor=C.+Denhezauthor=L.+C.+Buiauthor=F.+Rodrigues-Limaauthor=N.+Janelauthor=J.+M.+Delabarauthor=K.+Cariouauthor=R.+H.+Dodd&title=Development+of+DANDYs%2C+New+3%2C5-Diaryl-7-Azaindoles+Demonstrating+Potent+DYRK1A+Kinase+Inhibitory+Activity&doi=10.1021%2Fjm401049v"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Development of DANDYs, New 3,5-Diaryl-7-azaindoles Demonstrating Potent DYRK1A Kinase Inhibitory Activity</span></div><div class="casAuthors">Gourdain, Stephanie; Dairou, Julien; Denhez, Clement; Bui, Linh Chi; Rodrigues-Lima, Fernando; Janel, Nathalie; Delabar, Jean M.; Cariou, Kevin; Dodd, Robert H.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">9569-9585</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A series of 3,5-diaryl-1H-pyrrolo[2,3-b]pyridines was synthesized and evaluated for inhibition of DYRK1A kinase in vitro.  Derivs. having hydroxy groups on the aryl moieties demonstrated high inhibitory potencies with Kis in the low nanomolar range.  Their methoxy analogs were up to 100 times less active.  Docking studies at the ATP binding site suggested that these compds. bind tightly to this site via a network of multiple H-bonds with the peptide backbone.  None of the active compds. was cytotoxic to KB cells at 10-6 M.  Kinase profiling revealed that compd. I showed 2-fold selectivity for DYRK1A with respect to DYRK2 and DYRK3.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo1vxODs_tGD7Vg90H21EOLACvtfcHk0lhPYAQGg-jklA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhslWrt77K&md5=27dcc77a1c0d413cf0ab5c999cf9a195</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1021%2Fjm401049v&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm401049v%26sid%3Dliteratum%253Aachs%26aulast%3DGourdain%26aufirst%3DS.%26aulast%3DDairou%26aufirst%3DJ.%26aulast%3DDenhez%26aufirst%3DC.%26aulast%3DBui%26aufirst%3DL.%2BC.%26aulast%3DRodrigues-Lima%26aufirst%3DF.%26aulast%3DJanel%26aufirst%3DN.%26aulast%3DDelabar%26aufirst%3DJ.%2BM.%26aulast%3DCariou%26aufirst%3DK.%26aulast%3DDodd%26aufirst%3DR.%2BH.%26atitle%3DDevelopment%2520of%2520DANDYs%252C%2520New%25203%252C5-Diaryl-7-Azaindoles%2520Demonstrating%2520Potent%2520DYRK1A%2520Kinase%2520Inhibitory%2520Activity%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D9569%26epage%3D9585%26doi%3D10.1021%2Fjm401049v" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gelbard, H.</span>; <span class="NLM_string-name">Dewhurst, S.</span>; <span class="NLM_string-name">Goodfellow, V.</span>; <span class="NLM_string-name">Wiemann, T.</span>; <span class="NLM_string-name">Ravula, S.</span>; <span class="NLM_string-name">Loweth, C.</span></span>. <span> </span><span class="NLM_article-title">Bicyclic Heteroaryl Kinase Inhibitors and Methods of Use</span>. WO <span class="NLM_patent">2011/149950</span>, <span class="NLM_year">2011</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2011&author=H.+Gelbard&author=S.+Dewhurst&author=V.+Goodfellow&author=T.+Wiemann&author=S.+Ravula&author=C.+Loweth&title=Bicyclic+Heteroaryl+Kinase+Inhibitors+and+Methods+of+Use"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DGelbard%26aufirst%3DH.%26atitle%3DBicyclic%2520Heteroaryl%2520Kinase%2520Inhibitors%2520and%2520Methods%2520of%2520Use%26date%3D2011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Knauber, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tucker, J.</span></span> <span> </span><span class="NLM_article-title">Palladium Catalyzed Monoselective α-Arylation of Sulfones and Sulfonamides with 2,2,6,6-Tetramethylpiperidine·ZnCl·LiCl Base and Aryl Bromides</span>. <i>J. Org. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>81</i></span>,  <span class="NLM_fpage">5636</span>– <span class="NLM_lpage">5648</span>, <span class="refDoi"> DOI: 10.1021/acs.joc.6b01062</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.joc.6b01062" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=1%3ACAS%3A528%3ADC%252BC28XpslaqsLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=81&publication_year=2016&pages=5636-5648&author=T.+Knauberauthor=J.+Tucker&title=Palladium+Catalyzed+Monoselective+%CE%B1-Arylation+of+Sulfones+and+Sulfonamides+with+2%2C2%2C6%2C6-Tetramethylpiperidine%C2%B7ZnCl%C2%B7LiCl+Base+and+Aryl+Bromides&doi=10.1021%2Facs.joc.6b01062"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Palladium catalyzed monoselective α-arylation of sulfones and sulfonamides with 2,2,6,6-tetramethylpiperidine·ZnCl·LiCl base and aryl bromides</span></div><div class="casAuthors">Knauber, Thomas; Tucker, Joseph</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Organic Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">81</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">5636-5648</span>CODEN:
                <span class="NLM_cas:coden">JOCEAH</span>;
        ISSN:<span class="NLM_cas:issn">0022-3263</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A palladium catalyzed Negishi-type α-arylation of sulfones and sulfonamides with a broad range of aryl bromides has been developed.  The substrates are selectively metalated in situ with tmp·ZnCl·LiCl base (tmp: 2,2,6,6-tetramethylpiperidine) and cross-coupled in the presence of a catalyst system that is generated from Pd(dba)2 and XPhos.  Electron-deficient, electron-rich, and heterocyclic aryl bromides have been successfully cross-coupled, and sensitive functional groups are well tolerated.  Simple aryl bromides are converted overnight at 60 °C in THF while heteroaryl bromides are efficiently coupled within 2 h at 130 °C in a microwave reactor.  The desired monoarylated α-branched benzyl sulfones and sulfonamides were obtained in good yields, and over-arylation was not detected.  The procedure is ideal for late stage functionalization in parallel medicinal chem.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGowffEziLTrk7Vg90H21EOLACvtfcHk0lhPYAQGg-jklA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XpslaqsLg%253D&md5=b579dc9b2864b5a4f63c31f720218676</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Facs.joc.6b01062&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.joc.6b01062%26sid%3Dliteratum%253Aachs%26aulast%3DKnauber%26aufirst%3DT.%26aulast%3DTucker%26aufirst%3DJ.%26atitle%3DPalladium%2520Catalyzed%2520Monoselective%2520%25CE%25B1-Arylation%2520of%2520Sulfones%2520and%2520Sulfonamides%2520with%25202%252C2%252C6%252C6-Tetramethylpiperidine%25C2%25B7ZnCl%25C2%25B7LiCl%2520Base%2520and%2520Aryl%2520Bromides%26jtitle%3DJ.%2520Org.%2520Chem.%26date%3D2016%26volume%3D81%26spage%3D5636%26epage%3D5648%26doi%3D10.1021%2Facs.joc.6b01062" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dai, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ai, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geng, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duan, W.</span></span> <span> </span><span class="NLM_article-title">Discovery of Substituted 1H-Pyrazolo[3,4-b]pyridine Derivatives as Potent and Selective FGFR Kinase Inhibitors</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">629</span>– <span class="NLM_lpage">634</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.6b00066</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.6b00066" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=1%3ACAS%3A528%3ADC%252BC28XmsVamsb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=629-634&author=B.+Zhaoauthor=Y.+Liauthor=P.+Xuauthor=Y.+Daiauthor=C.+Luoauthor=Y.+Sunauthor=J.+Aiauthor=M.+Gengauthor=W.+Duan&title=Discovery+of+Substituted+1H-Pyrazolo%5B3%2C4-b%5Dpyridine+Derivatives+as+Potent+and+Selective+FGFR+Kinase+Inhibitors&doi=10.1021%2Facsmedchemlett.6b00066"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Substituted 1H-Pyrazolo[3,4-b]pyridine Derivatives as Potent and Selective FGFR Kinase Inhibitors</span></div><div class="casAuthors">Zhao, Bin; Li, Yixuan; Xu, Pan; Dai, Yang; Luo, Cheng; Sun, Yiming; Ai, Jing; Geng, Meiyu; Duan, Wenhu</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">629-634</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Fibroblast growth factor receptors (FGFRs) are important targets for cancer therapy.  Herein, we describe the design, synthesis, and biol. evaluation of a novel series of 1H-pyrazolo[3,4-b]pyridine derivs. as potent and selective FGFR kinase inhibitors.  On the basis of its excellent in vitro potency and favorable pharmacokinetic properties, compd. I was selected for in vivo evaluation and showed significant antitumor activity in a FGFR1-driven H1581 xenograft model.  These results indicated that I would be a promising candidate for further drug development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpt1RCtnNDMAbVg90H21EOLACvtfcHk0lhP6pTcKFE0Yg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XmsVamsb0%253D&md5=08cf7f710e2dfa738c7904965fae0630</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.6b00066&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.6b00066%26sid%3Dliteratum%253Aachs%26aulast%3DZhao%26aufirst%3DB.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DXu%26aufirst%3DP.%26aulast%3DDai%26aufirst%3DY.%26aulast%3DLuo%26aufirst%3DC.%26aulast%3DSun%26aufirst%3DY.%26aulast%3DAi%26aufirst%3DJ.%26aulast%3DGeng%26aufirst%3DM.%26aulast%3DDuan%26aufirst%3DW.%26atitle%3DDiscovery%2520of%2520Substituted%25201H-Pyrazolo%255B3%252C4-b%255Dpyridine%2520Derivatives%2520as%2520Potent%2520and%2520Selective%2520FGFR%2520Kinase%2520Inhibitors%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2016%26volume%3D7%26spage%3D629%26epage%3D634%26doi%3D10.1021%2Facsmedchemlett.6b00066" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ye, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.</span></span> <span> </span><span class="NLM_article-title">Design and Synthesis of 1,4,5,6-Tetrahydropyrrolo[3,4-c]Pyrazoles and Pyrazolo[3,4-b]Pyridines for Aurora-A Kinase Inhibitors</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">4273</span>– <span class="NLM_lpage">4278</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2010.04.083</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=10.1016%2Fj.bmcl.2010.04.083" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=20621733" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=1%3ACAS%3A528%3ADC%252BC3cXotVKms7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2010&pages=4273-4278&author=J.+Shiauthor=G.+Xuauthor=W.+Zhuauthor=H.+Yeauthor=S.+Yangauthor=Y.+Luoauthor=J.+Hanauthor=J.+Yangauthor=R.+Liauthor=Y.+Weiauthor=L.+Chen&title=Design+and+Synthesis+of+1%2C4%2C5%2C6-Tetrahydropyrrolo%5B3%2C4-c%5DPyrazoles+and+Pyrazolo%5B3%2C4-b%5DPyridines+for+Aurora-A+Kinase+Inhibitors&doi=10.1016%2Fj.bmcl.2010.04.083"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Design and synthesis of 1,4,5,6-tetrahydropyrrolo[3,4-c]pyrazoles and pyrazolo[3,4-b]pyridines for Aurora-A kinase inhibitors</span></div><div class="casAuthors">Shi, Jianyou; Xu, Guobin; Zhu, Wei; Ye, Haoyu; Yang, Shengyong; Luo, Youfu; Han, Jing; Yang, Jincheng; Li, Rui; Wei, Yuquan; Chen, Lijuan</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">4273-4278</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Two series of 3-aminopyrazole compds. including 24 1,4,5,6-tetrahydropyrrolo[3,4-c]pyrazoles and 16 pyrazolo[3,4-b]pyridines were synthesized and evaluated against HCT116, A549, and A2780 tumor cell lines.  Among them, three compds. were found to have the ideal anti-proliferative activities in vitro.  Docking expts. showed that the novel pyrazolo[3,4-b]pyridines share the similar interaction mode with Aurora-A kinase as PHA739358 (I).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoc3mQNCWrj_7Vg90H21EOLACvtfcHk0lhP6pTcKFE0Yg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXotVKms7Y%253D&md5=edb892e4dec485298d9ed9ff83d345c9</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2010.04.083&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2010.04.083%26sid%3Dliteratum%253Aachs%26aulast%3DShi%26aufirst%3DJ.%26aulast%3DXu%26aufirst%3DG.%26aulast%3DZhu%26aufirst%3DW.%26aulast%3DYe%26aufirst%3DH.%26aulast%3DYang%26aufirst%3DS.%26aulast%3DLuo%26aufirst%3DY.%26aulast%3DHan%26aufirst%3DJ.%26aulast%3DYang%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DR.%26aulast%3DWei%26aufirst%3DY.%26aulast%3DChen%26aufirst%3DL.%26atitle%3DDesign%2520and%2520Synthesis%2520of%25201%252C4%252C5%252C6-Tetrahydropyrrolo%255B3%252C4-c%255DPyrazoles%2520and%2520Pyrazolo%255B3%252C4-b%255DPyridines%2520for%2520Aurora-A%2520Kinase%2520Inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2010%26volume%3D20%26spage%3D4273%26epage%3D4278%26doi%3D10.1016%2Fj.bmcl.2010.04.083" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Van
Mileghem, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Egle, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilles, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Veryser, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Meervelt, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Borggraeve, W. M.</span></span> <span> </span><span class="NLM_article-title">Carbonylation as a Novel Method for the Assembly of Pyrazine Based Oligoamide Alpha-Helix Mimetics</span>. <i>Org. Biomol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">373</span>– <span class="NLM_lpage">378</span>, <span class="refDoi"> DOI: 10.1039/c6ob02358d</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=10.1039%2Fc6ob02358d" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=27910980" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvFKntLrP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2017&pages=373-378&author=S.+Van%0AMileghemauthor=B.+Egleauthor=P.+Gillesauthor=C.+Veryserauthor=L.+Van+Meerveltauthor=W.+M.+De+Borggraeve&title=Carbonylation+as+a+Novel+Method+for+the+Assembly+of+Pyrazine+Based+Oligoamide+Alpha-Helix+Mimetics&doi=10.1039%2Fc6ob02358d"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Carbonylation as a novel method for the assembly of pyrazine based oligoamide alpha-helix mimetics</span></div><div class="casAuthors">Van Mileghem, Seger; Egle, Brecht; Gilles, Philippe; Veryser, Cedrick; Van Meervelt, Luc; De Borggraeve, Wim M.</div><div class="citationInfo"><span class="NLM_cas:title">Organic & Biomolecular Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">373-378</span>CODEN:
                <span class="NLM_cas:coden">OBCRAK</span>;
        ISSN:<span class="NLM_cas:issn">1477-0520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">The design and synthesis of oligoamide α-helix peptidomimetics is reported.  The oligoamide type systems are prepd. in a modular fashion by coupling the monomers using palladium-catalyzed carbonylation chem.  This enabled us to use substrates with a low nucleophilicity, leading to previously unreported pyrazine based oligoamide α-helix mimetics.  The proof of principle is given by synthesizing a small set of compds.  Various end-capping groups were introduced and also a mixed multimer was successfully prepd.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpBhPQhYaXR_rVg90H21EOLACvtfcHk0lhP6pTcKFE0Yg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvFKntLrP&md5=aa4a28969f01e6076e7f9e5cf700e16f</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1039%2Fc6ob02358d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc6ob02358d%26sid%3Dliteratum%253Aachs%26aulast%3DVan%2BMileghem%26aufirst%3DS.%26aulast%3DEgle%26aufirst%3DB.%26aulast%3DGilles%26aufirst%3DP.%26aulast%3DVeryser%26aufirst%3DC.%26aulast%3DVan%2BMeervelt%26aufirst%3DL.%26aulast%3DDe%2BBorggraeve%26aufirst%3DW.%2BM.%26atitle%3DCarbonylation%2520as%2520a%2520Novel%2520Method%2520for%2520the%2520Assembly%2520of%2520Pyrazine%2520Based%2520Oligoamide%2520Alpha-Helix%2520Mimetics%26jtitle%3DOrg.%2520Biomol.%2520Chem.%26date%3D2017%26volume%3D15%26spage%3D373%26epage%3D378%26doi%3D10.1039%2Fc6ob02358d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Maccormick, S.</span>; <span class="NLM_string-name">Storck, P.-H.</span>; <span class="NLM_string-name">Mertimore, M.</span>; <span class="NLM_string-name">Charrier, J.-D.</span>; <span class="NLM_string-name">Knegtel, R.</span>; <span class="NLM_string-name">Young, S.</span>; <span class="NLM_string-name">Pinder, J.</span>; <span class="NLM_string-name">Durrant, S.</span></span> <span> </span><span class="NLM_article-title">Compounds Useful as Inhibitors of ATR Kinase</span>. WO <span class="NLM_patent">2012/178123</span>, <span class="NLM_year">2012</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2012&author=S.+Maccormick&author=P.-H.+Storck&author=M.+Mertimore&author=J.-D.+Charrier&author=R.+Knegtel&author=S.+Young&author=J.+Pinder&author=S.+Durrant&title=Compounds+Useful+as+Inhibitors+of+ATR+Kinase"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DMaccormick%26aufirst%3DS.%26atitle%3DCompounds%2520Useful%2520as%2520Inhibitors%2520of%2520ATR%2520Kinase%26date%3D2012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gelbard, H.</span>; <span class="NLM_string-name">Dewhurst, S.</span>; <span class="NLM_string-name">Goodfellow, V.</span>; <span class="NLM_string-name">Wiemann, T.</span>; <span class="NLM_string-name">Bennet, D.</span></span> <span> </span><span class="NLM_article-title">MLK Inhibitors and Methods of Use</span>. WO <span class="NLM_patent">2010/068483</span>, <span class="NLM_year">2010</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2010&author=H.+Gelbard&author=S.+Dewhurst&author=V.+Goodfellow&author=T.+Wiemann&author=D.+Bennet&title=MLK+Inhibitors+and+Methods+of+Use"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DGelbard%26aufirst%3DH.%26atitle%3DMLK%2520Inhibitors%2520and%2520Methods%2520of%2520Use%26date%3D2010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sorrell, F. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szklarz, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abdul Azeez, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elkins, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szklarz, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Azeez, K. R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elkins, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span> <span> </span><span class="NLM_article-title">Family-wide Structural Analysis of Human Numb-Associated Protein Kinases</span>. <i>Struct. Des.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">401</span>– <span class="NLM_lpage">411</span>, <span class="refDoi"> DOI: 10.1016/j.str.2015.12.015</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=10.1016%2Fj.str.2015.12.015" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=26853940" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=1%3ACAS%3A528%3ADC%252BC28XitFCktLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2016&pages=401-411&author=F.+J.+Sorrellauthor=M.+Szklarzauthor=K.+R.+Abdul+Azeezauthor=J.+M.+Elkinsauthor=S.+Knappauthor=M.+Szklarzauthor=K.+R.+A.+Azeezauthor=J.+M.+Elkinsauthor=S.+Knapp&title=Family-wide+Structural+Analysis+of+Human+Numb-Associated+Protein+Kinases&doi=10.1016%2Fj.str.2015.12.015"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Family-wide Structural Analysis of Human Numb-Associated Protein Kinases</span></div><div class="casAuthors">Sorrell, Fiona J.; Szklarz, Marta; Abdul Azeez, Kamal R.; Elkins, Jon M.; Knapp, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Structure (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">401-411</span>CODEN:
                <span class="NLM_cas:coden">STRUE6</span>;
        ISSN:<span class="NLM_cas:issn">0969-2126</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">The highly diverse Numb-assocd. kinase (NAK) family has been linked to broad cellular functions including receptor-mediated endocytosis, Notch pathway modulation, osteoblast differentiation, and dendrite morphogenesis.  Consequently, NAK kinases play a key role in a diverse range of diseases from Parkinson's and prostate cancer to HIV.  Due to the plasticity of this kinase family, NAK kinases are often inhibited by approved or investigational drugs and have been assocd. with side effects, but they are also potential drug targets.  The presence of cysteine residues in some NAK family members provides the possibility for selective targeting via covalent inhibition.  Here we report the first high-resoln. structures of kinases AAK1 and BIKE in complex with two drug candidates.  The presented data allow a comprehensive structural characterization of the NAK kinase family and provide the basis for rational design of selective NAK inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGryZi0QmT8YH7Vg90H21EOLACvtfcHk0lg74475OyHuzQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitFCktLY%253D&md5=6a353c993ec9f2b7152e459d72bab9d3</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1016%2Fj.str.2015.12.015&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.str.2015.12.015%26sid%3Dliteratum%253Aachs%26aulast%3DSorrell%26aufirst%3DF.%2BJ.%26aulast%3DSzklarz%26aufirst%3DM.%26aulast%3DAbdul%2BAzeez%26aufirst%3DK.%2BR.%26aulast%3DElkins%26aufirst%3DJ.%2BM.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DSzklarz%26aufirst%3DM.%26aulast%3DAzeez%26aufirst%3DK.%2BR.%2BA.%26aulast%3DElkins%26aufirst%3DJ.%2BM.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DFamily-wide%2520Structural%2520Analysis%2520of%2520Human%2520Numb-Associated%2520Protein%2520Kinases%26jtitle%3DStruct.%2520Des.%26date%3D2016%26volume%3D24%26spage%3D401%26epage%3D411%26doi%3D10.1016%2Fj.str.2015.12.015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fabian, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Biggs, W. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Treiber, D. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Atteridge, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Azimioara, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benedetti, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carter, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciceri, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edeen, P. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Floyd, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ford, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galvin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerlach, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grotzfeld, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herrgard, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Insko, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Insko, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lélias, J.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mehta, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Milanov, Z. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Velasco, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wodicka, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, H. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zarrinkar, P. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lockhart, D. J.</span></span> <span> </span><span class="NLM_article-title">A small molecule-kinase interaction map for clinical kinase inhibitors</span>. <i>Nat. Biotechnol.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">329</span>– <span class="NLM_lpage">336</span>, <span class="refDoi"> DOI: 10.1038/nbt1068</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=10.1038%2Fnbt1068" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=15711537" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=1%3ACAS%3A528%3ADC%252BD2MXitF2nt7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2005&pages=329-336&author=M.+A.+Fabianauthor=W.+H.+Biggsauthor=D.+K.+Treiberauthor=C.+E.+Atteridgeauthor=M.+D.+Azimioaraauthor=M.+G.+Benedettiauthor=T.+A.+Carterauthor=P.+Ciceriauthor=P.+T.+Edeenauthor=M.+Floydauthor=J.+M.+Fordauthor=M.+Galvinauthor=J.+L.+Gerlachauthor=R.+M.+Grotzfeldauthor=S.+Herrgardauthor=D.+E.+Inskoauthor=M.+A.+Inskoauthor=A.+G.+Laiauthor=J.-M.+L%C3%A9liasauthor=S.+A.+Mehtaauthor=Z.+V.+Milanovauthor=A.+M.+Velascoauthor=L.+M.+Wodickaauthor=H.+K.+Patelauthor=P.+P.+Zarrinkarauthor=D.+J.+Lockhart&title=A+small+molecule-kinase+interaction+map+for+clinical+kinase+inhibitors&doi=10.1038%2Fnbt1068"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">A small molecule-kinase interaction map for clinical kinase inhibitors</span></div><div class="casAuthors">Fabian, Miles A.; Biggs, William H.; Treiber, Daniel K.; Atteridge, Corey E.; Azimioara, Mihai D.; Benedetti, Michael G.; Carter, Todd A.; Ciceri, Pietro; Edeen, Philip T.; Floyd, Mark; Ford, Julia M.; Galvin, Margaret; Gerlach, Jay L.; Grotzfeld, Robert M.; Herrgard, Sanna; Insko, Darren E.; Insko, Michael A.; Lai, Andiliy G.; Lelias, Jean-Michel; Mehta, Shamal A.; Milanov, Zdravko V.; Velasco, Anne Marie; Wodicka, Lisa M.; Patel, Hitesh K.; Zarrinkar, Patrick P.; Lockhart, David J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">329-336</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Kinase inhibitors show great promise as a new class of therapeutics.  Here the authors describe an efficient way to det. kinase inhibitor specificity by measuring binding of small mols. to the ATP site of kinases.  The authors have profiled 20 kinase inhibitors, including 16 that are approved drugs or in clin. development, against a panel of 119 protein kinases.  The authors find that specificity varies widely and is not strongly correlated with chem. structure or the identity of the intended target.  Many novel interactions were identified, including tight binding of the p38 inhibitor BIRB-796 to an imatinib-resistant variant of the ABL kinase, and binding of imatinib to the SRC-family kinase LCK.  The authors also show that mutations in the epidermal growth factor receptor (EGFR) found in gefitinib-responsive patients do not affect the binding affinity of gefitinib or erlotinib.  Our results represent a systematic small mol.-protein interaction map for clin. compds. across a large no. of related proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqA0vNmMOXrnrVg90H21EOLACvtfcHk0lg74475OyHuzQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXitF2nt7w%253D&md5=6aedd5ceb8f77cd26ee50425dcec7bdf</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1038%2Fnbt1068&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt1068%26sid%3Dliteratum%253Aachs%26aulast%3DFabian%26aufirst%3DM.%2BA.%26aulast%3DBiggs%26aufirst%3DW.%2BH.%26aulast%3DTreiber%26aufirst%3DD.%2BK.%26aulast%3DAtteridge%26aufirst%3DC.%2BE.%26aulast%3DAzimioara%26aufirst%3DM.%2BD.%26aulast%3DBenedetti%26aufirst%3DM.%2BG.%26aulast%3DCarter%26aufirst%3DT.%2BA.%26aulast%3DCiceri%26aufirst%3DP.%26aulast%3DEdeen%26aufirst%3DP.%2BT.%26aulast%3DFloyd%26aufirst%3DM.%26aulast%3DFord%26aufirst%3DJ.%2BM.%26aulast%3DGalvin%26aufirst%3DM.%26aulast%3DGerlach%26aufirst%3DJ.%2BL.%26aulast%3DGrotzfeld%26aufirst%3DR.%2BM.%26aulast%3DHerrgard%26aufirst%3DS.%26aulast%3DInsko%26aufirst%3DD.%2BE.%26aulast%3DInsko%26aufirst%3DM.%2BA.%26aulast%3DLai%26aufirst%3DA.%2BG.%26aulast%3DL%25C3%25A9lias%26aufirst%3DJ.-M.%26aulast%3DMehta%26aufirst%3DS.%2BA.%26aulast%3DMilanov%26aufirst%3DZ.%2BV.%26aulast%3DVelasco%26aufirst%3DA.%2BM.%26aulast%3DWodicka%26aufirst%3DL.%2BM.%26aulast%3DPatel%26aufirst%3DH.%2BK.%26aulast%3DZarrinkar%26aufirst%3DP.%2BP.%26aulast%3DLockhart%26aufirst%3DD.%2BJ.%26atitle%3DA%2520small%2520molecule-kinase%2520interaction%2520map%2520for%2520clinical%2520kinase%2520inhibitors%26jtitle%3DNat.%2520Biotechnol.%26date%3D2005%26volume%3D23%26spage%3D329%26epage%3D336%26doi%3D10.1038%2Fnbt1068" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rodriguez-Madoz, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernal-Rubio, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaminski, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boyd, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fernandez-Sesma, A.</span></span> <span> </span><span class="NLM_article-title">Dengue Virus Inhibits the Production of Type I Interferon in Primary Human Dendritic Cells</span>. <i>J. Virol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>84</i></span>,  <span class="NLM_fpage">4845</span>– <span class="NLM_lpage">4850</span>, <span class="refDoi"> DOI: 10.1128/jvi.02514-09</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=10.1128%2Fjvi.02514-09" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=20164230" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=1%3ACAS%3A528%3ADC%252BC3cXlt1KhsLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=84&publication_year=2010&pages=4845-4850&author=J.+R.+Rodriguez-Madozauthor=D.+Bernal-Rubioauthor=D.+Kaminskiauthor=K.+Boydauthor=A.+Fernandez-Sesma&title=Dengue+Virus+Inhibits+the+Production+of+Type+I+Interferon+in+Primary+Human+Dendritic+Cells&doi=10.1128%2Fjvi.02514-09"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Dengue virus inhibits the production of type I interferon in primary human dendritic cells</span></div><div class="casAuthors">Rodriguez-Madoz, Juan R.; Bernal-Rubio, Dabeiba; Kaminski, Dorota; Boyd, Kelley; Fernandez-Sesma, Ana</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Virology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">84</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">4845-4850</span>CODEN:
                <span class="NLM_cas:coden">JOVIAM</span>;
        ISSN:<span class="NLM_cas:issn">0022-538X</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Dengue virus (DENV) infects human immune cells in vitro and likely infects dendritic cells (DCs) in vivo.  DENV-2 productive infection induces activation and release of high levels of chemokines and proinflammatory cytokines in monocyte-derived DCs (moDCs), with the notable exception of alpha/beta interferon (IFN-α/β).  Interestingly, DENV-2-infected moDCs fail to prime T cells, most likely due to the lack of IFN-α/β released by moDCs, since this effect was reversed by addn. of exogenous IFN-β.  Together, the authors' data show that inhibition of IFN-α/β prodn. by DENV in primary human moDCs is a novel mechanism of immune evasion.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpFUNnZLnic7bVg90H21EOLACvtfcHk0lg74475OyHuzQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXlt1KhsLk%253D&md5=7d751330acbca6445989821c48ffbe56</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1128%2Fjvi.02514-09&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252Fjvi.02514-09%26sid%3Dliteratum%253Aachs%26aulast%3DRodriguez-Madoz%26aufirst%3DJ.%2BR.%26aulast%3DBernal-Rubio%26aufirst%3DD.%26aulast%3DKaminski%26aufirst%3DD.%26aulast%3DBoyd%26aufirst%3DK.%26aulast%3DFernandez-Sesma%26aufirst%3DA.%26atitle%3DDengue%2520Virus%2520Inhibits%2520the%2520Production%2520of%2520Type%2520I%2520Interferon%2520in%2520Primary%2520Human%2520Dendritic%2520Cells%26jtitle%3DJ.%2520Virol.%26date%3D2010%26volume%3D84%26spage%3D4845%26epage%3D4850%26doi%3D10.1128%2Fjvi.02514-09" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Karaman, M. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herrgard, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Treiber, D. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gallant, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Atteridge, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campbell, B. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, K. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciceri, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, M. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edeen, P. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Faraoni, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Floyd, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hunt, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lockhart, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Milanov, Z. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morrison, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pallares, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, H. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pritchard, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wodicka, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zarrinkar, P. P.</span></span> <span> </span><span class="NLM_article-title">A Quantitative Analysis of Kinase Inhibitor Selectivity</span>. <i>Nat. Biotechnol.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">127</span>– <span class="NLM_lpage">132</span>, <span class="refDoi"> DOI: 10.1038/nbt1358</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=10.1038%2Fnbt1358" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=18183025" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=1%3ACAS%3A528%3ADC%252BD1cXisFGlsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2008&pages=127-132&author=M.+W.+Karamanauthor=S.+Herrgardauthor=D.+K.+Treiberauthor=P.+Gallantauthor=C.+E.+Atteridgeauthor=B.+T.+Campbellauthor=K.+W.+Chanauthor=P.+Ciceriauthor=M.+I.+Davisauthor=P.+T.+Edeenauthor=R.+Faraoniauthor=M.+Floydauthor=J.+P.+Huntauthor=D.+J.+Lockhartauthor=Z.+V.+Milanovauthor=M.+J.+Morrisonauthor=G.+Pallaresauthor=H.+K.+Patelauthor=S.+Pritchardauthor=L.+M.+Wodickaauthor=P.+P.+Zarrinkar&title=A+Quantitative+Analysis+of+Kinase+Inhibitor+Selectivity&doi=10.1038%2Fnbt1358"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">A quantitative analysis of kinase inhibitor selectivity</span></div><div class="casAuthors">Karaman, Mazen W.; Herrgard, Sanna; Treiber, Daniel K.; Gallant, Paul; Atteridge, Corey E.; Campbell, Brian T.; Chan, Katrina W.; Ciceri, Pietro; Davis, Mindy I.; Edeen, Philip T.; Faraoni, Raffaella; Floyd, Mark; Hunt, Jeremy P.; Lockhart, Daniel J.; Milanov, Zdravko V.; Morrison, Michael J.; Pallares, Gabriel; Patel, Hitesh K.; Pritchard, Stephanie; Wodicka, Lisa M.; Zarrinkar, Patrick P.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">127-132</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Kinase inhibitors are a new class of therapeutics with a propensity to inhibit multiple targets.  The biol. consequences of multikinase activity are poorly defined, and an important step toward understanding the relationship between selectivity, efficacy and safety is the exploration of how inhibitors interact with the human kinome.  The authors present interaction maps for 38 kinase inhibitors across a panel of 317 kinases representing >50% of the predicted human protein kinome.  The data constitute the most comprehensive study of kinase inhibitor selectivity to date and reveal a wide diversity of interaction patterns.  To enable a global anal. of the results, the authors introduce the concept of a selectivity score as a general tool to quantify and differentiate the obsd. interaction patterns.  The authors further investigate the impact of panel size and find that small assay panels do not provide a robust measure of selectivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpiArJSq2_lPrVg90H21EOLACvtfcHk0lgWnNUm3qSN0g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXisFGlsQ%253D%253D&md5=346265d412853ced636ad4128ed8a76f</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1038%2Fnbt1358&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt1358%26sid%3Dliteratum%253Aachs%26aulast%3DKaraman%26aufirst%3DM.%2BW.%26aulast%3DHerrgard%26aufirst%3DS.%26aulast%3DTreiber%26aufirst%3DD.%2BK.%26aulast%3DGallant%26aufirst%3DP.%26aulast%3DAtteridge%26aufirst%3DC.%2BE.%26aulast%3DCampbell%26aufirst%3DB.%2BT.%26aulast%3DChan%26aufirst%3DK.%2BW.%26aulast%3DCiceri%26aufirst%3DP.%26aulast%3DDavis%26aufirst%3DM.%2BI.%26aulast%3DEdeen%26aufirst%3DP.%2BT.%26aulast%3DFaraoni%26aufirst%3DR.%26aulast%3DFloyd%26aufirst%3DM.%26aulast%3DHunt%26aufirst%3DJ.%2BP.%26aulast%3DLockhart%26aufirst%3DD.%2BJ.%26aulast%3DMilanov%26aufirst%3DZ.%2BV.%26aulast%3DMorrison%26aufirst%3DM.%2BJ.%26aulast%3DPallares%26aufirst%3DG.%26aulast%3DPatel%26aufirst%3DH.%2BK.%26aulast%3DPritchard%26aufirst%3DS.%26aulast%3DWodicka%26aufirst%3DL.%2BM.%26aulast%3DZarrinkar%26aufirst%3DP.%2BP.%26atitle%3DA%2520Quantitative%2520Analysis%2520of%2520Kinase%2520Inhibitor%2520Selectivity%26jtitle%3DNat.%2520Biotechnol.%26date%3D2008%26volume%3D26%26spage%3D127%26epage%3D132%26doi%3D10.1038%2Fnbt1358" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Trott, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olson, A. J.</span></span> <span> </span><span class="NLM_article-title">AutoDock Vina: improving the speed and accuracy of docking with a new scoring function, efficient optimization, and multithreading</span>. <i>J. Comput. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>31</i></span>,  <span class="NLM_fpage">455</span>– <span class="NLM_lpage">461</span>, <span class="refDoi"> DOI: 10.1002/jcc.21334</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=10.1002%2Fjcc.21334" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=19499576" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsFGnur3O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2010&pages=455-461&author=O.+Trottauthor=A.+J.+Olson&title=AutoDock+Vina%3A+improving+the+speed+and+accuracy+of+docking+with+a+new+scoring+function%2C+efficient+optimization%2C+and+multithreading&doi=10.1002%2Fjcc.21334"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">AutoDock Vina: Improving the speed and accuracy of docking with a new scoring function, efficient optimization, and multithreading</span></div><div class="casAuthors">Trott, Oleg; Olson, Arthur J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">455-461</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">AutoDock Vina, a new program for mol. docking and virtual screening, is presented.  AutoDock Vina achieves an approx. 2 orders of magnitude speed-up compared with the mol. docking software previously developed in the authors' lab (AutoDock 4), while also significantly improving the accuracy of the binding mode predictions, judging by the authors' tests on the training set used in AutoDock 4 development.  Further speed-up is achieved from parallelism, by using multithreading on multicore machines.  AutoDock Vina automatically calcs. the grid maps and clusters the results in a way transparent to the user.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpLecOIkUxIlrVg90H21EOLACvtfcHk0lgWnNUm3qSN0g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsFGnur3O&md5=c6974af8a1235f7aa09918d3e6f70dc4</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1002%2Fjcc.21334&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.21334%26sid%3Dliteratum%253Aachs%26aulast%3DTrott%26aufirst%3DO.%26aulast%3DOlson%26aufirst%3DA.%2BJ.%26atitle%3DAutoDock%2520Vina%253A%2520improving%2520the%2520speed%2520and%2520accuracy%2520of%2520docking%2520with%2520a%2520new%2520scoring%2520function%252C%2520efficient%2520optimization%252C%2520and%2520multithreading%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2010%26volume%3D31%26spage%3D455%26epage%3D461%26doi%3D10.1002%2Fjcc.21334" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Case, D. A.</span>; <span class="NLM_string-name">Ben-Shalom, I.</span>; <span class="NLM_string-name">Brozell, S. R.</span>; <span class="NLM_string-name">Cerutti, D. S.</span>; <span class="NLM_string-name">Cheatham, T.</span>; <span class="NLM_string-name">Cruzeiro, V. W. D.</span>; <span class="NLM_string-name">Darden, T. A.</span>; <span class="NLM_string-name">Duke, R. A.</span>; <span class="NLM_string-name">Ghoreishi, D.</span>; <span class="NLM_string-name">Gilson, M. K.</span>; <span class="NLM_string-name">Gohlke, H.</span>; <span class="NLM_string-name">Goetz, A.
W.</span>; <span class="NLM_string-name">Greene, D.</span>; <span class="NLM_string-name">Harris, R.</span>; <span class="NLM_string-name">Homeyer, N.</span>; <span class="NLM_string-name">Huang, Y.</span>; <span class="NLM_string-name">Izadi, S.</span>; <span class="NLM_string-name">Kovalenko, A.</span>; <span class="NLM_string-name">Kurtzman, T.</span>; <span class="NLM_string-name">Lee, T. S.</span>; <span class="NLM_string-name">LeGrand, S.</span>; <span class="NLM_string-name">Li, P.</span>; <span class="NLM_string-name">Lin, C.</span>; <span class="NLM_string-name">Liu, J.</span>; <span class="NLM_string-name">Luchko, T.</span>; <span class="NLM_string-name">Luo, R.</span>; <span class="NLM_string-name">Mermelstein, D. J.</span>; <span class="NLM_string-name">Merz, K. M.</span>; <span class="NLM_string-name">Miao, Y.</span>; <span class="NLM_string-name">Monard, G.</span>; <span class="NLM_string-name">Nguyen, C.</span>; <span class="NLM_string-name">Nguyen, H.</span>; <span class="NLM_string-name">Omelyan, I.</span>; <span class="NLM_string-name">Onufriev, A.</span>; <span class="NLM_string-name">Pan, F.</span>; <span class="NLM_string-name">Qi, R.</span>; <span class="NLM_string-name">Roe, D. R.</span>; <span class="NLM_string-name">Roitberg, A.</span>; <span class="NLM_string-name">Sagui, C.</span>; <span class="NLM_string-name">Schott-Verdugo, S.</span>; <span class="NLM_string-name">Shen, J.</span>; <span class="NLM_string-name">Simmerling, C. L.</span>; <span class="NLM_string-name">Smith, J.</span>; <span class="NLM_string-name">SalomonFerrer, R.</span>; <span class="NLM_string-name">Swails, J.</span>; <span class="NLM_string-name">Walker, R. C.</span>; <span class="NLM_string-name">Wang, J.</span>; <span class="NLM_string-name">Wei, H.</span>; <span class="NLM_string-name">Wolf, R. M.</span>; <span class="NLM_string-name">Wu, X.</span>; <span class="NLM_string-name">Xiao, L.</span>; <span class="NLM_string-name">York, D. M.</span>; <span class="NLM_string-name">Kollman, P. A.</span></span> <i>AMBER 2018</i>; <span class="NLM_publisher-loc">San Francisco</span>, <span class="NLM_year">2018</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2018&author=D.+A.+Case&author=I.+Ben-Shalom&author=S.+R.+Brozell&author=D.+S.+Cerutti&author=T.+Cheatham&author=V.+W.+D.+Cruzeiro&author=T.+A.+Darden&author=R.+A.+Duke&author=D.+Ghoreishi&author=M.+K.+Gilson&author=H.+Gohlke&author=A.%0AW.+Goetz&author=D.+Greene&author=R.+Harris&author=N.+Homeyer&author=Y.+Huang&author=S.+Izadi&author=A.+Kovalenko&author=T.+Kurtzman&author=T.+S.+Lee&author=S.+LeGrand&author=P.+Li&author=C.+Lin&author=J.+Liu&author=T.+Luchko&author=R.+Luo&author=D.+J.+Mermelstein&author=K.+M.+Merz&author=Y.+Miao&author=G.+Monard&author=C.+Nguyen&author=H.+Nguyen&author=I.+Omelyan&author=A.+Onufriev&author=F.+Pan&author=R.+Qi&author=D.+R.+Roe&author=A.+Roitberg&author=C.+Sagui&author=S.+Schott-Verdugo&author=J.+Shen&author=C.+L.+Simmerling&author=J.+Smith&author=R.+SalomonFerrer&author=J.+Swails&author=R.+C.+Walker&author=J.+Wang&author=H.+Wei&author=R.+M.+Wolf&author=X.+Wu&author=L.+Xiao&author=D.+M.+York&author=P.+A.+Kollman&title=AMBER+2018"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26aulast%3DCase%26aufirst%3DD.%2BA.%26jtitle%3DAMBER%25202018%26date%3D2018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Genheden, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ryde, U.</span></span> <span> </span><span class="NLM_article-title">The MM/PBSA and MM/GBSA Methods to Estimate Ligand-Binding Affinities</span>. <i>Expert Opin. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">449</span>– <span class="NLM_lpage">461</span>, <span class="refDoi"> DOI: 10.1517/17460441.2015.1032936</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=10.1517%2F17460441.2015.1032936" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=25835573" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=1%3ACAS%3A528%3ADC%252BC2MXntFGktr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=449-461&author=S.+Genhedenauthor=U.+Ryde&title=The+MM%2FPBSA+and+MM%2FGBSA+Methods+to+Estimate+Ligand-Binding+Affinities&doi=10.1517%2F17460441.2015.1032936"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">The MM/PBSA and MM/GBSA methods to estimate ligand-binding affinities</span></div><div class="casAuthors">Genheden, Samuel; Ryde, Ulf</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Drug Discovery</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">449-461</span>CODEN:
                <span class="NLM_cas:coden">EODDBX</span>;
        ISSN:<span class="NLM_cas:issn">1746-0441</span>.
    
            (<span class="NLM_cas:orgname">Informa Healthcare</span>)
        </div><div class="casAbstract">Introduction: The mol. mechanics energies combined with the Poisson-Boltzmann or generalized Born and surface area continuum solvation (MM/PBSA and MM/GBSA) methods are popular approaches to est. the free energy of the binding of small ligands to biol. macromols.  They are typically based on mol. dynamics simulations of the receptor-ligand complex and are therefore intermediate in both accuracy and computational effort between empirical scoring and strict alchem. perturbation methods.  They have been applied to a large no. of systems with varying success.  Areas covered: The authors review the use of MM/PBSA and MM/GBSA methods to calc. ligand-binding affinities, with an emphasis on calibration, testing and validation, as well as attempts to improve the methods, rather than on specific applications.  Expert opinion: MM/PBSA and MM/GBSA are attractive approaches owing to their modular nature and that they do not require calcns. on a training set.  They have been used successfully to reproduce and rationalize exptl. findings and to improve the results of virtual screening and docking.  However, they contain several crude and questionable approxns., for example, the lack of conformational entropy and information about the no. and free energy of water mols. in the binding site.  Moreover, there are many variants of the method and their performance varies strongly with the tested system.  Likewise, most attempts to ameliorate the methods with more accurate approaches, for example, quantum-mech. calcns., polarizable force fields or improved solvation have deteriorated the results.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpTBKte1pAZR7Vg90H21EOLACvtfcHk0ljB2yfY75Xz4Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXntFGktr8%253D&md5=b123b88809f275564f95a2271ebd159f</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1517%2F17460441.2015.1032936&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1517%252F17460441.2015.1032936%26sid%3Dliteratum%253Aachs%26aulast%3DGenheden%26aufirst%3DS.%26aulast%3DRyde%26aufirst%3DU.%26atitle%3DThe%2520MM%252FPBSA%2520and%2520MM%252FGBSA%2520Methods%2520to%2520Estimate%2520Ligand-Binding%2520Affinities%26jtitle%3DExpert%2520Opin.%2520Drug%2520Discovery%26date%3D2015%26volume%3D10%26spage%3D449%26epage%3D461%26doi%3D10.1517%2F17460441.2015.1032936" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pettersen, E. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goddard, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Couch, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greenblatt, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meng, E. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferrin, T. E.</span></span> <span> </span><span class="NLM_article-title">UCSF Chimera?A Visualization System for Exploratory Research and Analysis</span>. <i>J. Comput. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">1605</span>– <span class="NLM_lpage">1612</span>, <span class="refDoi"> DOI: 10.1002/jcc.20084</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=10.1002%2Fjcc.20084" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=15264254" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=1%3ACAS%3A528%3ADC%252BD2cXmvVOhsbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2004&pages=1605-1612&author=E.+F.+Pettersenauthor=T.+D.+Goddardauthor=C.+C.+Huangauthor=G.+S.+Couchauthor=D.+M.+Greenblattauthor=E.+C.+Mengauthor=T.+E.+Ferrin&title=UCSF+Chimera%3FA+Visualization+System+for+Exploratory+Research+and+Analysis&doi=10.1002%2Fjcc.20084"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">UCSF Chimera-A visualization system for exploratory research and analysis</span></div><div class="casAuthors">Pettersen, Eric F.; Goddard, Thomas D.; Huang, Conrad C.; Couch, Gregory S.; Greenblatt, Daniel M.; Meng, Elaine C.; Ferrin, Thomas E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">1605-1612</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">The design, implementation, and capabilities of an extensible visualization system, UCSF Chimera, are discussed.  Chimera is segmented into a core that provides basic services and visualization, and extensions that provide most higher level functionality.  This architecture ensures that the extension mechanism satisfies the demands of outside developers who wish to incorporate new features.  Two unusual extensions are presented: Multiscale, which adds the ability to visualize large-scale mol. assemblies such as viral coats, and Collab., which allows researchers to share a Chimera session interactively despite being at sep. locales.  Other extensions include Multalign Viewer, for showing multiple sequence alignments and assocd. structures; ViewDock, for screening docked ligand orientations; Movie, for replaying mol. dynamics trajectories; and Vol. Viewer, for display and anal. of volumetric data.  A discussion of the usage of Chimera in real-world situations is given, along with anticipated future directions.  Chimera includes full user documentation, is free to academic and nonprofit users, and is available for Microsoft Windows, Linux, Apple Mac OS X, SGI IRIX, and HP Tru64 Unix from http://www.cgl.ucsf.edu/chimera/.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrFmD4M5S0cPbVg90H21EOLACvtfcHk0ljB2yfY75Xz4Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXmvVOhsbs%253D&md5=944b175f440c1ff323705987cf937ee7</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1002%2Fjcc.20084&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.20084%26sid%3Dliteratum%253Aachs%26aulast%3DPettersen%26aufirst%3DE.%2BF.%26aulast%3DGoddard%26aufirst%3DT.%2BD.%26aulast%3DHuang%26aufirst%3DC.%2BC.%26aulast%3DCouch%26aufirst%3DG.%2BS.%26aulast%3DGreenblatt%26aufirst%3DD.%2BM.%26aulast%3DMeng%26aufirst%3DE.%2BC.%26aulast%3DFerrin%26aufirst%3DT.%2BE.%26atitle%3DUCSF%2520Chimera%253FA%2520Visualization%2520System%2520for%2520Exploratory%2520Research%2520and%2520Analysis%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2004%26volume%3D25%26spage%3D1605%26epage%3D1612%26doi%3D10.1002%2Fjcc.20084" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kabsch, W.</span></span> <span> </span><span class="NLM_article-title">XDS</span>. <i>Acta Crystallogr. Sect. D Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">125</span>– <span class="NLM_lpage">132</span>, <span class="refDoi"> DOI: 10.1107/s0907444909047337</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=10.1107%2FS0907444909047337" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=20124692" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=125-132&author=W.+Kabsch&title=XDS&doi=10.1107%2Fs0907444909047337"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Software XDS for image rotation, recognition and crystal symmetry assignment</span></div><div class="casAuthors">Kabsch, Wolfgang</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">125-132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">The usage and control of recent modifications of the program package XDS for the processing of rotation images are described in the context of previous versions.  New features include automatic detn. of spot size and reflecting range and recognition and assignment of crystal symmetry.  Moreover, the limitations of earlier package versions on the no. of correction/scaling factors and the representation of pixel contents have been removed.  Large program parts have been restructured for parallel processing so that the quality and completeness of collected data can be assessed soon after measurement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpv5AyUw00DYbVg90H21EOLACvtfcHk0ljB2yfY75Xz4Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D&md5=1aa9a38aeb3ce95af4ffb7d8b8a142bd</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1107%2FS0907444909047337&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909047337%26sid%3Dliteratum%253Aachs%26aulast%3DKabsch%26aufirst%3DW.%26atitle%3DXDS%26jtitle%3DActa%2520Crystallogr.%2520Sect.%2520D%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D125%26epage%3D132%26doi%3D10.1107%2Fs0907444909047337" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Winter, G.</span></span> <span> </span><span class="NLM_article-title">xia2: an expert system for macromolecular crystallography data reduction</span>. <i>J. Appl. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>43</i></span>,  <span class="NLM_fpage">186</span>– <span class="NLM_lpage">190</span>, <span class="refDoi"> DOI: 10.1107/s0021889809045701</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=10.1107%2FS0021889809045701" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtFalsrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2010&pages=186-190&author=G.+Winter&title=xia2%3A+an+expert+system+for+macromolecular+crystallography+data+reduction&doi=10.1107%2Fs0021889809045701"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">xia2: An expert system for macromolecular crystallography data reduction</span></div><div class="casAuthors">Winter, G.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">43</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">186-190, S186/1-S186/5</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">An expert system for macromol. crystallog. data redn. is presented, which builds on existing software to automate the complete data redn. process from images to merged structure factor amplitudes.  This can automatically identify multi-wedge, multi-pass and multiwavelength data sets and includes explicit procedures to test for crystallog. special cases.  With the push towards high-throughput crystallog. at synchrotron beamlines and automation of structure soln., the ability to reduce data with no user input fills an important gap in the pipeline.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqOi3u0ydXbSrVg90H21EOLACvtfcHk0ljB2yfY75Xz4Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtFalsrk%253D&md5=27771caa813f3e828abdf2a331ae55e3</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1107%2FS0021889809045701&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0021889809045701%26sid%3Dliteratum%253Aachs%26aulast%3DWinter%26aufirst%3DG.%26atitle%3Dxia2%253A%2520an%2520expert%2520system%2520for%2520macromolecular%2520crystallography%2520data%2520reduction%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D2010%26volume%3D43%26spage%3D186%26epage%3D190%26doi%3D10.1107%2Fs0021889809045701" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Winn, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ballard, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dodson, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evans, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keegan, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krissinel, E. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leslie, A. G. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McNicholas, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murshudov, G. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pannu, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Potterton, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Powell, H. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vagin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, K. S.</span></span> <span> </span><span class="NLM_article-title">Overview of the CCP4 suite and current developments</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>67</i></span>,  <span class="NLM_fpage">235</span>– <span class="NLM_lpage">242</span>, <span class="refDoi"> DOI: 10.1107/s0907444910045749</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=10.1107%2FS0907444910045749" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=21460441" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktFWqt70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2011&pages=235-242&author=M.+D.+Winnauthor=C.+C.+Ballardauthor=K.+D.+Cowtanauthor=E.+J.+Dodsonauthor=P.+Emsleyauthor=P.+R.+Evansauthor=R.+M.+Keeganauthor=E.+B.+Krissinelauthor=A.+G.+W.+Leslieauthor=A.+McCoyauthor=S.+J.+McNicholasauthor=G.+N.+Murshudovauthor=N.+S.+Pannuauthor=E.+A.+Pottertonauthor=H.+R.+Powellauthor=R.+J.+Readauthor=A.+Vaginauthor=K.+S.+Wilson&title=Overview+of+the+CCP4+suite+and+current+developments&doi=10.1107%2Fs0907444910045749"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Overview of the CCP4 suite and current developments</span></div><div class="casAuthors">Winn, Martyn D.; Ballard, Charles C.; Cowtan, Kevin D.; Dodson, Eleanor J.; Emsley, Paul; Evans, Phil R.; Keegan, Ronan M.; Krissinel, Eugene B.; Leslie, Andrew G. W.; McCoy, Airlie; McNicholas, Stuart J.; Murshudov, Garib N.; Pannu, Navraj S.; Potterton, Elizabeth A.; Powell, Harold R.; Read, Randy J.; Vagin, Alexei; Wilson, Keith S.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">235-242</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A review.  The CCP4 (Collaborative Computational Project, No. 4) software suite is a collection of programs and assocd. data and software libraries which can be used for macromol. structure detn. by X-ray crystallog.  The suite is designed to be flexible, allowing users a no. of methods of achieving their aims.  The programs are from a wide variety of sources but are connected by a common infrastructure provided by std. file formats, data objects and graphical interfaces.  Structure soln. by macromol. crystallog. is becoming increasingly automated and the CCP4 suite includes several automation pipelines.  After giving a brief description of the evolution of CCP4 over the last 30 years, an overview of the current suite is given.  While detailed descriptions are given in the accompanying articles, here it is shown how the individual programs contribute to a complete software package.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp_wWMMIol6UrVg90H21EOLACvtfcHk0lhiOKsZVQD0hw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktFWqt70%253D&md5=c407e4d47bef46864be336d60147c17d</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1107%2FS0907444910045749&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910045749%26sid%3Dliteratum%253Aachs%26aulast%3DWinn%26aufirst%3DM.%2BD.%26aulast%3DBallard%26aufirst%3DC.%2BC.%26aulast%3DCowtan%26aufirst%3DK.%2BD.%26aulast%3DDodson%26aufirst%3DE.%2BJ.%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DEvans%26aufirst%3DP.%2BR.%26aulast%3DKeegan%26aufirst%3DR.%2BM.%26aulast%3DKrissinel%26aufirst%3DE.%2BB.%26aulast%3DLeslie%26aufirst%3DA.%2BG.%2BW.%26aulast%3DMcCoy%26aufirst%3DA.%26aulast%3DMcNicholas%26aufirst%3DS.%2BJ.%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26aulast%3DPannu%26aufirst%3DN.%2BS.%26aulast%3DPotterton%26aufirst%3DE.%2BA.%26aulast%3DPowell%26aufirst%3DH.%2BR.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DVagin%26aufirst%3DA.%26aulast%3DWilson%26aufirst%3DK.%2BS.%26atitle%3DOverview%2520of%2520the%2520CCP4%2520suite%2520and%2520current%2520developments%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2011%26volume%3D67%26spage%3D235%26epage%3D242%26doi%3D10.1107%2Fs0907444910045749" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mccoy, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosse-kunstleve, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winn, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Storoni, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span> <span> </span><span class="NLM_article-title">Phaser Crystallographic Software Research Papers</span>. <i>J. Appl. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>40</i></span>,  <span class="NLM_fpage">658</span>– <span class="NLM_lpage">674</span>, <span class="refDoi"> DOI: 10.1107/s0021889807021206</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=10.1107%2Fs0021889807021206" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=19461840" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2007&pages=658-674&author=A.+J.+Mccoyauthor=R.+W.+Grosse-kunstleveauthor=P.+D.+Adamsauthor=M.+D.+Winnauthor=L.+C.+Storoniauthor=R.+J.+Read&title=Phaser+Crystallographic+Software+Research+Papers&doi=10.1107%2Fs0021889807021206"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Phaser crystallographic software</span></div><div class="casAuthors">McCoy, Airlie J.; Grosse-Kunstleve, Ralf W.; Adams, Paul D.; Winn, Martyn D.; Storoni, Laurent C.; Read, Randy J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">658-674</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Phaser is a program for phasing macromol. crystal structures by both mol. replacement and exptl. phasing methods.  The novel phasing algorithms implemented in Phaser have been developed using max. likelihood and multivariate statistics.  For mol. replacement, the new algorithms have proved to be significantly better than traditional methods in discriminating correct solns. from noise, and for single-wavelength anomalous dispersion exptl. phasing, the new algorithms, which account for correlations between F+ and F-, give better phases (lower mean phase error with respect to the phases given by the refined structure) than those that use mean F and anomalous differences ΔF.  One of the design concepts of Phaser was that it be capable of a high degree of automation.  To this end, Phaser (written in C++) can be called directly from Python, although it can also be called using traditional CCP4 keyword-style input.  Phaser is a platform for future development of improved phasing methods and their release, including source code, to the crystallog. community.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPeVyjN_FMGbVg90H21EOLACvtfcHk0lhiOKsZVQD0hw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D&md5=c63b722ae97e0a74e6a5a079d388f09f</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1107%2Fs0021889807021206&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252Fs0021889807021206%26sid%3Dliteratum%253Aachs%26aulast%3DMccoy%26aufirst%3DA.%2BJ.%26aulast%3DGrosse-kunstleve%26aufirst%3DR.%2BW.%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DWinn%26aufirst%3DM.%2BD.%26aulast%3DStoroni%26aufirst%3DL.%2BC.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26atitle%3DPhaser%2520Crystallographic%2520Software%2520Research%2520Papers%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D2007%26volume%3D40%26spage%3D658%26epage%3D674%26doi%3D10.1107%2Fs0021889807021206" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Murshudov, G. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vagin, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dodson, E. J.</span></span> <span> </span><span class="NLM_article-title">Refinement of Macromolecular Structures by the Maximum-Likelihood Method</span>. <i>Acta Crystallogr. Sect. D Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">240</span>– <span class="NLM_lpage">255</span>, <span class="refDoi"> DOI: 10.1107/s0907444996012255</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=10.1107%2FS0907444996012255" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=15299926" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=1%3ACAS%3A528%3ADyaK2sXjs1Gnsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=1997&pages=240-255&author=G.+N.+Murshudovauthor=A.+A.+Vaginauthor=E.+J.+Dodson&title=Refinement+of+Macromolecular+Structures+by+the+Maximum-Likelihood+Method&doi=10.1107%2Fs0907444996012255"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Refinement of macromolecular structures by the maximum-likelihood method</span></div><div class="casAuthors">Murshudov, Garib N.; Vagin, Alexei A.; Dodson, Eleanor J.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">D53</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">240-255</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Munksgaard</span>)
        </div><div class="casAbstract">A review with many refs. on the math. basis of max. likelihood.  The likelihood function for macromol. structures is extended to include prior phase information and exptl. std. uncertainties.  The assumption that different parts of a structure might have different errors is considered.  A method for estg. σA using "free" reflections is described and its effects analyzed.  The derived equations have been implemented in the program REFMAC.  This has been tested on several proteins at different stages of refinement (bacterial α-amylase, cytochrome c', cross-linked insulin and oligopeptide binding protein).  The results derived using the max.-likelihood residual are consistently better than those obtained from least-squares refinement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWg7mzQRNBhbVg90H21EOLACvtfcHk0lhiOKsZVQD0hw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXjs1Gnsb4%253D&md5=ec7f141ce1542f7ff458b98ecfe3f8af</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1107%2FS0907444996012255&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444996012255%26sid%3Dliteratum%253Aachs%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26aulast%3DVagin%26aufirst%3DA.%2BA.%26aulast%3DDodson%26aufirst%3DE.%2BJ.%26atitle%3DRefinement%2520of%2520Macromolecular%2520Structures%2520by%2520the%2520Maximum-Likelihood%2520Method%26jtitle%3DActa%2520Crystallogr.%2520Sect.%2520D%2520Biol.%2520Crystallogr.%26date%3D1997%26volume%3D53%26spage%3D240%26epage%3D255%26doi%3D10.1107%2Fs0907444996012255" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Afonine, P. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bunkóczi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, V. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, I. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Echols, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Headd, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hung, L.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kapral, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosse-Kunstleve, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moriarty, N. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oeffner, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terwilliger, T. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zwart, P. H.</span></span> <span> </span><span class="NLM_article-title">PHENIX: a comprehensive Python-based system for macromolecular structure solution</span>. <i>Acta Crystallogr. Sect. D Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">213</span>– <span class="NLM_lpage">221</span>, <span class="refDoi"> DOI: 10.1107/s0907444909052925</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=10.1107%2FS0907444909052925" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=20124702" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=213-221&author=P.+D.+Adamsauthor=P.+V.+Afonineauthor=G.+Bunk%C3%B3cziauthor=V.+B.+Chenauthor=I.+W.+Davisauthor=N.+Echolsauthor=J.+J.+Headdauthor=L.-W.+Hungauthor=G.+J.+Kapralauthor=R.+W.+Grosse-Kunstleveauthor=A.+J.+McCoyauthor=N.+W.+Moriartyauthor=R.+Oeffnerauthor=R.+J.+Readauthor=D.+C.+Richardsonauthor=J.+S.+Richardsonauthor=T.+C.+Terwilligerauthor=P.+H.+Zwart&title=PHENIX%3A+a+comprehensive+Python-based+system+for+macromolecular+structure+solution&doi=10.1107%2Fs0907444909052925"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">PHENIX: a comprehensive Python-based system for macromolecular structure solution</span></div><div class="casAuthors">Adams, Paul D.; Afonine, Pavel V.; Bunkoczi, Gabor; Chen, Vincent B.; Davis, Ian W.; Echols, Nathaniel; Headd, Jeffrey J.; Hung, Li Wei; Kapral, Gary J.; Grosse-Kunstleve, Ralf W.; McCoy, Airlie J.; Moriarty, Nigel W.; Oeffner, Robert; Read, Randy J.; Richardson, David C.; Richardson, Jane S.; Terwilliger, Thomas C.; Zwart, Peter H.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">213-221</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A review.  Macromol. X-ray crystallog. is routinely applied to understand biol. processes at a mol. level.  However, significant time and effort are still required to solve and complete many of these structures because of the need for manual interpretation of complex numerical data using many software packages and the repeated use of interactive three-dimensional graphics.  PHENIX has been developed to provide a comprehensive system for macromol. crystallog. structure soln. with an emphasis on the automation of all procedures.  This has relied on the development of algorithms that minimize or eliminate subjective input, the development of algorithms that automate procedures that are traditionally performed by hand and, finally, the development of a framework that allows a tight integration between the algorithms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpm_HgKRpa-ObVg90H21EOLACvtfcHk0liQdcH5oi9pZw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D&md5=67b439ff4bd61c659cae37ca4209b7bc</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1107%2FS0907444909052925&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909052925%26sid%3Dliteratum%253Aachs%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DAfonine%26aufirst%3DP.%2BV.%26aulast%3DBunk%25C3%25B3czi%26aufirst%3DG.%26aulast%3DChen%26aufirst%3DV.%2BB.%26aulast%3DDavis%26aufirst%3DI.%2BW.%26aulast%3DEchols%26aufirst%3DN.%26aulast%3DHeadd%26aufirst%3DJ.%2BJ.%26aulast%3DHung%26aufirst%3DL.-W.%26aulast%3DKapral%26aufirst%3DG.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DMoriarty%26aufirst%3DN.%2BW.%26aulast%3DOeffner%26aufirst%3DR.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DRichardson%26aufirst%3DD.%2BC.%26aulast%3DRichardson%26aufirst%3DJ.%2BS.%26aulast%3DTerwilliger%26aufirst%3DT.%2BC.%26aulast%3DZwart%26aufirst%3DP.%2BH.%26atitle%3DPHENIX%253A%2520a%2520comprehensive%2520Python-based%2520system%2520for%2520macromolecular%2520structure%2520solution%26jtitle%3DActa%2520Crystallogr.%2520Sect.%2520D%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D213%26epage%3D221%26doi%3D10.1107%2Fs0907444909052925" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lohkamp, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, W. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K.</span></span> <span> </span><span class="NLM_article-title">Features and development ofCoot</span>. <i>Acta Crystallogr. Sect. D Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">486</span>– <span class="NLM_lpage">501</span>, <span class="refDoi"> DOI: 10.1107/s0907444910007493</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=10.1107%2Fs0907444910007493" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=20383002" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=486-501&author=P.+Emsleyauthor=B.+Lohkampauthor=W.+G.+Scottauthor=K.+Cowtan&title=Features+and+development+ofCoot&doi=10.1107%2Fs0907444910007493"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Features and development of Coot</span></div><div class="casAuthors">Emsley, P.; Lohkamp, B.; Scott, W. G.; Cowtan, K.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">486-501</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Coot is a mol.-graphics application for model building and validation of biol. macromols.  The program displays electron-d. maps and at. models and allows model manipulations such as idealization, real-space refinement, manual rotation/translation, rigid-body fitting, ligand search, solvation, mutations, rotamers and Ramachandran idealization.  Furthermore, tools are provided for model validation as well as interfaces to external programs for refinement, validation and graphics.  The software is designed to be easy to learn for novice users, which is achieved by ensuring that tools for common tasks are 'discoverable' through familiar user-interface elements (menus and toolbars) or by intuitive behavior (mouse controls).  Recent developments have focused on providing tools for expert users, with customisable key bindings, extensions and an extensive scripting interface.  The software is under rapid development, but has already achieved very widespread use within the crystallog. community.  The current state of the software is presented, with a description of the facilities available and of some of the underlying methods employed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAKBKszbqb_7Vg90H21EOLACvtfcHk0liQdcH5oi9pZw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D&md5=67262cbfc60004de5ef962d5c043c910</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1107%2Fs0907444910007493&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252Fs0907444910007493%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DLohkamp%26aufirst%3DB.%26aulast%3DScott%26aufirst%3DW.%2BG.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DFeatures%2520and%2520development%2520ofCoot%26jtitle%3DActa%2520Crystallogr.%2520Sect.%2520D%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D486%26epage%3D501%26doi%3D10.1107%2Fs0907444910007493" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Perera, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khaliq, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuhn, R. J.</span></span> <span> </span><span class="NLM_article-title">Closing the Door on Flaviviruses: Entry as a Target for Antiviral Drug Design</span>. <i>Antiviral Res.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>80</i></span>,  <span class="NLM_fpage">11</span>– <span class="NLM_lpage">22</span>, <span class="refDoi"> DOI: 10.1016/j.antiviral.2008.05.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=10.1016%2Fj.antiviral.2008.05.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=18585795" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtVKlsb7J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=80&publication_year=2008&pages=11-22&author=R.+Pereraauthor=M.+Khaliqauthor=R.+J.+Kuhn&title=Closing+the+Door+on+Flaviviruses%3A+Entry+as+a+Target+for+Antiviral+Drug+Design&doi=10.1016%2Fj.antiviral.2008.05.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Closing the door on flaviviruses: Entry as a target for antiviral drug design</span></div><div class="casAuthors">Perera, Rushika; Khaliq, Mansoora; Kuhn, Richard J.</div><div class="citationInfo"><span class="NLM_cas:title">Antiviral Research</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">80</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">11-22</span>CODEN:
                <span class="NLM_cas:coden">ARSRDR</span>;
        ISSN:<span class="NLM_cas:issn">0166-3542</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  With the emergence and rapid spread of West Nile virus in the United States since 1999, and the 50-100 million infections per yr caused by dengue virus globally, the threat of flaviviruses as re-emerging human pathogens has become a reality.  To support the efforts that are currently being pursued to develop effective vaccines against these viruses, researchers are also actively pursuing the development of small mol. compds. that target various aspects of the virus life cycle.  Recent advances in the structural characterization of the flaviviruses have provided a strong foundation toward these efforts.  These studies have provided the pseudo-at. structures of virions from several members of the genus as well as at. resoln. structures of several viral proteins.  Most importantly, these studies have highlighted specific structural rearrangements that occur within the virion that are necessary for the virus to complete its life cycle.  These rearrangements occur when the virus must transition from immature, to mature, to fusion-active states and rely heavily on the conformational flexibility of the envelope (E) protein that forms the outer glycoprotein shell of the virus.  Anal. of these conformational changes can suggest promising targets for structure-based antiviral design.  For instance, by targeting the flexibility of the E protein, it might be possible to inhibit required rearrangements of this protein and trap the virus in a specific state.  This would interfere with a productive flaviviral infection.  This review presents a structural perspective of the flavivirus life cycle and focuses on the role of the E protein as an opportune target for structure-based antiviral drug design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrQNS6DhW3U3rVg90H21EOLACvtfcHk0liQdcH5oi9pZw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtVKlsb7J&md5=dfaed4719a04fe40a02b14d14fa42342</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1016%2Fj.antiviral.2008.05.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.antiviral.2008.05.004%26sid%3Dliteratum%253Aachs%26aulast%3DPerera%26aufirst%3DR.%26aulast%3DKhaliq%26aufirst%3DM.%26aulast%3DKuhn%26aufirst%3DR.%2BJ.%26atitle%3DClosing%2520the%2520Door%2520on%2520Flaviviruses%253A%2520Entry%2520as%2520a%2520Target%2520for%2520Antiviral%2520Drug%2520Design%26jtitle%3DAntiviral%2520Res.%26date%3D2008%26volume%3D80%26spage%3D11%26epage%3D22%26doi%3D10.1016%2Fj.antiviral.2008.05.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xie, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gayen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, P.-Y.</span></span> <span> </span><span class="NLM_article-title">Membrane Topology and Function of Dengue Virus NS2A Protein</span>. <i>J. Virol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>87</i></span>,  <span class="NLM_fpage">4609</span>– <span class="NLM_lpage">4622</span>, <span class="refDoi"> DOI: 10.1128/jvi.02424-12</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=10.1128%2Fjvi.02424-12" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=23408612" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=1%3ACAS%3A528%3ADC%252BC3sXotVyqtLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=87&publication_year=2013&pages=4609-4622&author=X.+Xieauthor=S.+Gayenauthor=C.+Kangauthor=Z.+Yuanauthor=P.-Y.+Shi&title=Membrane+Topology+and+Function+of+Dengue+Virus+NS2A+Protein&doi=10.1128%2Fjvi.02424-12"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Membrane topology and function of dengue virus NS2A protein</span></div><div class="casAuthors">Xie, Xuping; Gayen, Shovanlal; Kang, Cong Bao; Yuan, Zhiming; Shi, Pei-Yong</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Virology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">87</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">4609-4622</span>CODEN:
                <span class="NLM_cas:coden">JOVIAM</span>;
        ISSN:<span class="NLM_cas:issn">0022-538X</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Flavivirus nonstructural protein 2A (NS2A) is a component of the viral replication complex that functions in virion assembly and antagonizes the host immune response.  Although flavivirus NS2A is known to assoc. with the endoplasmic reticulum (ER) membrane, the detailed topol. of this protein has not been detd.  Here we report the first topol. model of flavivirus NS2A on the ER membrane.  Using dengue virus (DENV) NS2A as a model, we show that (i) the N-terminal 68 amino acids are located in the ER lumen, with one segment (amino acids 30 to 52) that interacts with ER membrane without traversing the lipid bilayer; (ii) amino acids 69 to 209 form five transmembrane segments, each of which integrally spans the ER membrane; and (iii) the C-terminal tail (amino acids 210 to 218) is located in the cytosol.  NMR structural anal. showed that the first membrane-spanning segment (amino acids 69 to 93) consists of two helixes sepd. by a "helix breaker.".  The helix breaker is formed by amino acid P85 and one pos. charged residue, R84.  Functional anal. using replicon and genome-length RNAs of DENV-2 indicates that P85 is not important for viral replication.  However, when R84 was replaced with E, the mutation attenuated both viral RNA synthesis and virus prodn.  Remarkably, an R84A mutation did not affect viral RNA synthesis but blocked intracellular formation of infectious virions.  Collectively, the mutagenesis results demonstrate that NS2A functions in both DENV RNA synthesis and virion assembly/maturation.  The topol. model of DENV NS2A provides a good starting point for studying how flavivirus NS2A modulates viral replication and evasion of host immune response.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoKckWTzQ8sSLVg90H21EOLACvtfcHk0lj2FsWvmYjjQA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXotVyqtLo%253D&md5=d797704a8f18e5692cd74fc243c62eb1</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1128%2Fjvi.02424-12&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252Fjvi.02424-12%26sid%3Dliteratum%253Aachs%26aulast%3DXie%26aufirst%3DX.%26aulast%3DGayen%26aufirst%3DS.%26aulast%3DKang%26aufirst%3DC.%26aulast%3DYuan%26aufirst%3DZ.%26aulast%3DShi%26aufirst%3DP.-Y.%26atitle%3DMembrane%2520Topology%2520and%2520Function%2520of%2520Dengue%2520Virus%2520NS2A%2520Protein%26jtitle%3DJ.%2520Virol.%26date%3D2013%26volume%3D87%26spage%3D4609%26epage%3D4622%26doi%3D10.1128%2Fjvi.02424-12" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huang, C. Y.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Butrapet, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moss, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Childers, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Erb, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Calvert, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Silengo, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kinney, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blair, C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roehrig, J. T.</span></span> <span> </span><span class="NLM_article-title">The Dengue Virus Type 2 Envelope Protein Fusion Peptide Is Essential for Membrane Fusion</span>. <i>Virology</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>396</i></span>,  <span class="NLM_fpage">305</span>– <span class="NLM_lpage">315</span>, <span class="refDoi"> DOI: 10.1016/j.virol.2009.10.027</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=10.1016%2Fj.virol.2009.10.027" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=19913272" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsFantbzE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=396&publication_year=2010&pages=305-315&author=C.+Y.-H.+Huangauthor=S.+Butrapetauthor=K.+J.+Mossauthor=T.+Childersauthor=S.+M.+Erbauthor=A.+E.+Calvertauthor=S.+J.+Silengoauthor=R.+M.+Kinneyauthor=C.+D.+Blairauthor=J.+T.+Roehrig&title=The+Dengue+Virus+Type+2+Envelope+Protein+Fusion+Peptide+Is+Essential+for+Membrane+Fusion&doi=10.1016%2Fj.virol.2009.10.027"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">The dengue virus type 2 envelope protein fusion peptide is essential for membrane fusion</span></div><div class="casAuthors">Huang, Claire Y.-H.; Butrapet, Siritorn; Moss, Kelly J.; Childers, Thomas; Erb, Steven M.; Calvert, Amanda E.; Silengo, Shawn J.; Kinney, Richard M.; Blair, Carol D.; Roehrig, John T.</div><div class="citationInfo"><span class="NLM_cas:title">Virology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">396</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">305-315</span>CODEN:
                <span class="NLM_cas:coden">VIRLAX</span>;
        ISSN:<span class="NLM_cas:issn">0042-6822</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The flaviviral envelope (E) protein directs virus-mediated membrane fusion.  To investigate membrane fusion as a requirement for virus growth, the authors introduced 27 unique mutations into the fusion peptide of an infectious cDNA clone of dengue 2 virus and recovered seven stable mutant viruses.  The fusion efficiency of the mutants was impaired, demonstrating for the first time the requirement for specific FP AAs in optimal fusion.  Mutant viruses exhibited different growth kinetics and/or genetic stabilities in different cell types and adult mosquitoes.  Virus particles could be recovered following RNA transfection of cells with four lethal mutants; however, recovered viruses could not re-infect cells.  These viruses could enter cells, but internalized virus appeared to be retained in endosomal compartments of infected cells, thus suggesting a fusion blockade.  Mutations of the FP also resulted in reduced virus reactivity with flavivirus group-reactive antibodies, confirming earlier reports using virus-like particles.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGonaQLm21eCQLVg90H21EOLACvtfcHk0lj2FsWvmYjjQA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsFantbzE&md5=93b00be0bdebcf5ff749374cea8c355b</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1016%2Fj.virol.2009.10.027&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.virol.2009.10.027%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DC.%2BY.-H.%26aulast%3DButrapet%26aufirst%3DS.%26aulast%3DMoss%26aufirst%3DK.%2BJ.%26aulast%3DChilders%26aufirst%3DT.%26aulast%3DErb%26aufirst%3DS.%2BM.%26aulast%3DCalvert%26aufirst%3DA.%2BE.%26aulast%3DSilengo%26aufirst%3DS.%2BJ.%26aulast%3DKinney%26aufirst%3DR.%2BM.%26aulast%3DBlair%26aufirst%3DC.%2BD.%26aulast%3DRoehrig%26aufirst%3DJ.%2BT.%26atitle%3DThe%2520Dengue%2520Virus%2520Type%25202%2520Envelope%2520Protein%2520Fusion%2520Peptide%2520Is%2520Essential%2520for%2520Membrane%2520Fusion%26jtitle%3DVirology%26date%3D2010%26volume%3D396%26spage%3D305%26epage%3D315%26doi%3D10.1016%2Fj.virol.2009.10.027" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref55"><div class="reference"><strong class="refLabel"><a href="#ref55" class="rightTabRefNumLink">55</a></strong><div class="NLM_citation" id="rightTab-cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rodriguez-Madoz, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernal-Rubio, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaminski, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boyd, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fernandez-Sesma, A.</span></span> <span> </span><span class="NLM_article-title">Dengue Virus Inhibits the Production of Type I Interferon in Primary Human Dendritic Cells</span>. <i>J. Virol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>84</i></span>,  <span class="NLM_fpage">4845</span>– <span class="NLM_lpage">4850</span>, <span class="refDoi"> DOI: 10.1128/jvi.02514-09</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=10.1128%2Fjvi.02514-09" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=20164230" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=1%3ACAS%3A528%3ADC%252BC3cXlt1KhsLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=84&publication_year=2010&pages=4845-4850&author=J.+R.+Rodriguez-Madozauthor=D.+Bernal-Rubioauthor=D.+Kaminskiauthor=K.+Boydauthor=A.+Fernandez-Sesma&title=Dengue+Virus+Inhibits+the+Production+of+Type+I+Interferon+in+Primary+Human+Dendritic+Cells&doi=10.1128%2Fjvi.02514-09"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Dengue virus inhibits the production of type I interferon in primary human dendritic cells</span></div><div class="casAuthors">Rodriguez-Madoz, Juan R.; Bernal-Rubio, Dabeiba; Kaminski, Dorota; Boyd, Kelley; Fernandez-Sesma, Ana</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Virology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">84</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">4845-4850</span>CODEN:
                <span class="NLM_cas:coden">JOVIAM</span>;
        ISSN:<span class="NLM_cas:issn">0022-538X</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Dengue virus (DENV) infects human immune cells in vitro and likely infects dendritic cells (DCs) in vivo.  DENV-2 productive infection induces activation and release of high levels of chemokines and proinflammatory cytokines in monocyte-derived DCs (moDCs), with the notable exception of alpha/beta interferon (IFN-α/β).  Interestingly, DENV-2-infected moDCs fail to prime T cells, most likely due to the lack of IFN-α/β released by moDCs, since this effect was reversed by addn. of exogenous IFN-β.  Together, the authors' data show that inhibition of IFN-α/β prodn. by DENV in primary human moDCs is a novel mechanism of immune evasion.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpFUNnZLnic7bVg90H21EOLACvtfcHk0lj2FsWvmYjjQA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXlt1KhsLk%253D&md5=7d751330acbca6445989821c48ffbe56</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1128%2Fjvi.02514-09&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252Fjvi.02514-09%26sid%3Dliteratum%253Aachs%26aulast%3DRodriguez-Madoz%26aufirst%3DJ.%2BR.%26aulast%3DBernal-Rubio%26aufirst%3DD.%26aulast%3DKaminski%26aufirst%3DD.%26aulast%3DBoyd%26aufirst%3DK.%26aulast%3DFernandez-Sesma%26aufirst%3DA.%26atitle%3DDengue%2520Virus%2520Inhibits%2520the%2520Production%2520of%2520Type%2520I%2520Interferon%2520in%2520Primary%2520Human%2520Dendritic%2520Cells%26jtitle%3DJ.%2520Virol.%26date%3D2010%26volume%3D84%26spage%3D4845%26epage%3D4850%26doi%3D10.1128%2Fjvi.02514-09" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref56"><div class="reference"><strong class="refLabel"><a href="#ref56" class="rightTabRefNumLink">56</a></strong><div class="NLM_citation" id="rightTab-cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Morris, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huey, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindstrom, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanner, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Belew, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goodsell, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olson, A. J.</span></span> <span> </span><span class="NLM_article-title">AutoDock4 and AutoDockTools4: Automated Docking with Selective Receptor Flexibility</span>. <i>J. Comput. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">2785</span>– <span class="NLM_lpage">2791</span>, <span class="refDoi"> DOI: 10.1002/jcc.21256</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=10.1002%2Fjcc.21256" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=19399780" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=1%3ACAS%3A528%3ADC%252BD1MXht1GitrnK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2009&pages=2785-2791&author=G.+M.+Morrisauthor=R.+Hueyauthor=W.+Lindstromauthor=M.+F.+Sannerauthor=R.+K.+Belewauthor=D.+S.+Goodsellauthor=A.+J.+Olson&title=AutoDock4+and+AutoDockTools4%3A+Automated+Docking+with+Selective+Receptor+Flexibility&doi=10.1002%2Fjcc.21256"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">AutoDock and AutoDockTools: Automated docking with selective receptor flexibility</span></div><div class="casAuthors">Morris, Garrett M.; Huey, Ruth; Lindstrom, William; Sanner, Michel F.; Belew, Richard K.; Goodsell, David S.; Olson, Arthur J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">2785-2791</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">We describe the testing and release of AutoDock4 and the accompanying graphical user interface AutoDockTools.  AutoDock4 incorporates limited flexibility in the receptor.  Several tests are reported here, including a redocking expt. with 188 diverse ligand-protein complexes and a cross-docking expt. using flexible sidechains in 87 HIV protease complexes.  We also report its utility in anal. of covalently bound ligands, using both a grid-based docking method and a modification of the flexible sidechain technique. © 2009 Wiley Periodicals, Inc.  J Comput Chem, 2009.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqktE6WAo4GEbVg90H21EOLACvtfcHk0lgvWHH9Q-3AyQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXht1GitrnK&md5=679ce22fc50e9291c9aa16e7a1855845</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1002%2Fjcc.21256&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.21256%26sid%3Dliteratum%253Aachs%26aulast%3DMorris%26aufirst%3DG.%2BM.%26aulast%3DHuey%26aufirst%3DR.%26aulast%3DLindstrom%26aufirst%3DW.%26aulast%3DSanner%26aufirst%3DM.%2BF.%26aulast%3DBelew%26aufirst%3DR.%2BK.%26aulast%3DGoodsell%26aufirst%3DD.%2BS.%26aulast%3DOlson%26aufirst%3DA.%2BJ.%26atitle%3DAutoDock4%2520and%2520AutoDockTools4%253A%2520Automated%2520Docking%2520with%2520Selective%2520Receptor%2520Flexibility%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2009%26volume%3D30%26spage%3D2785%26epage%3D2791%26doi%3D10.1002%2Fjcc.21256" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref57"><div class="reference"><strong class="refLabel"><a href="#ref57" class="rightTabRefNumLink">57</a></strong><div class="NLM_citation" id="rightTab-cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Evans, D. A.</span></span> <span> </span><span class="NLM_article-title">History of the Harvard ChemDraw Project</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">11140</span>– <span class="NLM_lpage">11145</span>, <span class="refDoi"> DOI: 10.1002/anie.201405820</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=10.1002%2Fanie.201405820" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtlejurvK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2014&pages=11140-11145&author=D.+A.+Evans&title=History+of+the+Harvard+ChemDraw+Project&doi=10.1002%2Fanie.201405820"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">History of the Harvard ChemDraw Project</span></div><div class="casAuthors">Evans, David A.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">42</span>),
    <span class="NLM_cas:pages">11140-11145</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Org. chemists routinely communicate concepts through a combination of cursive statements and graphics-based mol. representations commonly referred to as valence bond structures.  This form of written communication may be compared to the structure of the Egyptian hieroglyphic language which combines both cursive and icon-based graphics (glyphs) to communicate a specific thought or statement.  The importance of "chem. glyphs" may be highlighted by the fact that two chemists not sharing a common language may effectively communicate through the use of a series of valence bond structures and assocd. "curved arrow" notations.  This paper discusses the developmental history of ChemDraw, a computer program designed for drawing mol. structures.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo902ZF4RWorbVg90H21EOLACvtfcHk0lgvWHH9Q-3AyQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtlejurvK&md5=593906df77924693bbc440decb0a8ec3</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1002%2Fanie.201405820&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201405820%26sid%3Dliteratum%253Aachs%26aulast%3DEvans%26aufirst%3DD.%2BA.%26atitle%3DHistory%2520of%2520the%2520Harvard%2520ChemDraw%2520Project%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2014%26volume%3D53%26spage%3D11140%26epage%3D11145%26doi%3D10.1002%2Fanie.201405820" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5L4Q" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5L4Q','PDB','5L4Q'); return false;">PDB: 5L4Q</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4WSQ" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4WSQ','PDB','4WSQ'); return false;">PDB: 4WSQ</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i126"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b00136">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_13155"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a class="ext-link" href="/">ACS Publications website</a> at DOI: <a class="ext-link" href="/doi/abs/10.1021/acs.jmedchem.9b00136">10.1021/acs.jmedchem.9b00136</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Copies of <sup>1</sup>H and <sup>13</sup>C NMR spectra of intermediates and final compounds; details of the kinase selectivity profiling of compound <b>21b</b>; and data collection and refinement statistics for cocrystallography of compound <b>1</b> with AAK1 (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b00136/suppl_file/jm9b00136_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b00136/suppl_file/jm9b00136_si_002.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00136/suppl_file/jm9b00136_si_001.pdf">jm9b00136_si_001.pdf (5.27 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00136/suppl_file/jm9b00136_si_002.csv">jm9b00136_si_002.csv (4.21 kb)</a></li></ul></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">The coordinates have been deposited in the PDB with accession code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5L4Q">5L4Q</a>.</p><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.9b00136&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.9b00136%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2019.62.issue-12" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.9b00136" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                4MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"677b74abae8a22f0","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
