
<html lang="en"     class="pb-page"  data-request-id="ecf33664-2207-43b7-8c67-dcdbd9833460"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;article:article:10.1021/jm050563r;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;issue:issue:10.1021/jmcmar.2005.48.issue-22;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="dc.Title" content="Fast Small Molecule Similarity Searching with Multiple Alignment Profiles of Molecules Represented in One-Dimension" /></meta><meta name="dc.Creator" content="Norman  Wang," /></meta><meta name="dc.Creator" content="Robert K.  DeLisle, and" /></meta><meta name="dc.Creator" content="David J.  Diller*" /></meta><meta name="dc.Description" content="Multiple sequence alignment has proven to be a powerful method for creating protein and DNA sequence alignment profiles. These profiles of protein families are useful tools for identifying conserve..." /></meta><meta name="Description" content="Multiple sequence alignment has proven to be a powerful method for creating protein and DNA sequence alignment profiles. These profiles of protein families are useful tools for identifying conserve..." /></meta><meta name="dc.Publisher" content=" American Chemical Society " /></meta><meta name="dc.Date" scheme="WTN8601" content="September 30, 2005" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/jm050563r" /></meta><meta name="dc.Identifier" scheme="crossover-key" content="JMCMAr-48-22-6980" /></meta><meta name="dc.Identifier" scheme="pii" content="S0022-2623(05)00563-7" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Copyright © 2005 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Fjm050563r" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Fjm050563r" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Fjm050563r" /></link>
        
    
    

<title>Fast Small Molecule Similarity Searching with Multiple Alignment Profiles of Molecules Represented in One-Dimension | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/jm050563r" /></meta><meta property="og:title" content="Fast Small Molecule Similarity Searching with Multiple Alignment Profiles of Molecules Represented in One-Dimension" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm050563r/production/images/large/jm050563rn00001.jpeg" /></meta><meta property="og:description" content=" Multiple sequence alignment has proven to be a powerful method for creating protein and DNA sequence alignment profiles. These profiles of protein families are useful tools for identifying conserved motifs, such as the catalytic triad of the serine protease family or the seven transmembrane helices of the G-protein coupled receptor family. Ultimately, the understanding of the critical motifs within a family is useful for identifying new members of the family. Due to the complexity of protein−ligand recognition, no universally accepted method exists for clustering small molecules into families with the same or similar biological activity. A combination of the concept of multiple sequence alignment and the 1-dimensional molecular representation described earlier offers a new method for profiling sets of small molecules with the same biological activity. These small molecule profiles can isolate key commonalties within the set of bioactive compounds much like a multiple sequence alignment can isolate critical motifs within a protein family. The small molecule profiles then make useful tools for searching small molecule databases for new compounds with the same biological activity. The technique is demonstrated here using the human ether-a-go-go potassium channel and the kinase SRC.  " /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/jm050563r"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/jm050563r">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-14/achre4.2021.54.issue-14/20210720/achre4.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-29/aamick.2021.13.issue-29/20210728/aamick.2021.13.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-14/acncdm.2021.12.issue-14/20210721/acncdm.2021.12.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-7/amlcef.2021.3.issue-7/20210705/amlcef.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-29/acsodf.2021.6.issue-29/20210727/acsodf.2021.6.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-29/ascecg.2021.9.issue-29/20210726/ascecg.2021.9.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-29/ancham.2021.93.issue-29/20210727/ancham.2021.93.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-29/bichaw.2021.60.issue-29/20210727/bichaw.2021.60.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-27/cgeabj.2021.99.issue-27/20210726-01/cgeabj.2021.99.issue-27.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-7/cgdefu.2021.21.issue-7/20210707/cgdefu.2021.21.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-14/esthag.2021.55.issue-14/20210720/esthag.2021.55.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-29/iecred.2021.60.issue-29/20210728/iecred.2021.60.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-14/inocaj.2021.60.issue-14/20210719/inocaj.2021.60.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-29/jafcau.2021.69.issue-29/20210728/jafcau.2021.69.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-29/jacsat.2021.143.issue-29/20210728/jacsat.2021.143.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-7/jamsef.2021.32.issue-7/20210707/jamsef.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-29/langd5.2021.37.issue-29/20210727/langd5.2021.37.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-7/mpohbp.2021.18.issue-7/20210705/mpohbp.2021.18.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Fjm050563r&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Fjm050563r&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/jm050563r&amp;href=/doi/10.1021/jm050563r" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2005</span><span class="cit-fg-volume">, 48</span><span class="cit-fg-issue">, 22</span><span class="cit-fg-pageRange">, 6980-6990</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/48/22" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/jm050529c" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/jm0505866" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Fast Small Molecule Similarity Searching with Multiple Alignment Profiles of Molecules Represented in One-Dimension</span></h1><ul class="loa non-jats-loa"><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Norman++Wang">Norman Wang</a></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Robert+K.++DeLisle">Robert K. DeLisle</a></span></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=David+J.++Diller">David J. Diller</a></span></span></li><span class="comma-separator"></span></ul><div class="affiliations"><div class="view-affs"><span class="title">View Author Information</span><i class="caret-icon icon-angle-down"></i></div><div class="all-aff-infos hlFld-Affiliation"><div class="aff-info"><span class="aff-text">Pharmacopeia, Inc., CN5350, Princeton, New Jersey 08543-5350 </span></div></div></div><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="JMCMAr" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Fjm050563r&amp;href=/doi/10.1021%2Fjm050563r" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2005</span></span><span class="cit-volume">, 48</span><span class="cit-issue">, 22</span><span class="cit-pageRange">, 6980–6990</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">September 30, 2005</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>15 June 2005</li><li><span class="item_label"><b>Published</b> online</span>30 September 2005</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 1 November 2005</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/jm050563r" title="DOI URL">https://doi.org/10.1021/jm050563r</a></div><div class="article_header-article-copyright"><strong>Copyright © 2005 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D6980%26pageCount%3D11%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DNorman%2BWang%252C%2BRobert%2BK.%2BDeLisle%252C%2BDavid%2BJ.%2BDiller%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D48%26issueNum%3D22%26contentID%3Djm050563r%26title%3DFast%2BSmall%2BMolecule%2BSimilarity%2BSearching%2Bwith%2BMultiple%2BAlignment%2BProfiles%2Bof%2BMolecules%2BRepresented%2Bin%2BOne-Dimension%26numPages%3D11%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D6990%26publicationDate%3DNovember%2B2005">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/jm050563r"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">277</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">15</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/jm050563r" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Fast Small Molecule Similarity Searching with Multiple Alignment Profiles of Molecules Represented in One-Dimension&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Norman&quot;,&quot;last_name&quot;:&quot;Wang&quot;},{&quot;first_name&quot;:&quot;Robert&quot;,&quot;last_name&quot;:&quot;K. DeLisle&quot;},{&quot;first_name&quot;:&quot;David&quot;,&quot;last_name&quot;:&quot;J. Diller&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2005&quot;,&quot;month&quot;:&quot;09&quot;,&quot;day&quot;:&quot;30&quot;,&quot;issue&quot;:&quot;22&quot;,&quot;volume&quot;:&quot;48&quot;,&quot;pages&quot;:&quot;6980-6990&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/jm050563r&quot;},&quot;abstract&quot;:&quot;Multiple sequence alignment has proven to be a powerful method for creating protein and DNA sequence alignment profiles. These profiles of protein families are useful tools for identifying conserved motifs, such as the catalytic triad of the serine protease family or the seven transmembrane helices of the G-protein coupled receptor family. Ultimately, the understanding of the critical motifs within a family is useful for identifying new members of the family. Due to the complexity of protein−ligand recognition, no universally accepted method exists for clustering small molecules into families with the same or similar biological activity. A combination of the concept of multiple sequence alignment and the 1-dimensional molecular representation described earlier offers a new method for profiling sets of small molecules with the same biological activity. These small molecule profiles can isolate key commonalties within the set of bioactive compounds much like a multiple sequence alignment can isolate critical mo&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Fjm050563r&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=jm050563r" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Fjm050563r&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=jm050563r" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Fjm050563r&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=jm050563r" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Fjm050563r&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=jm050563r&amp;href=/doi/10.1021/jm050563r" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/jm050563r" /></input><a href="/doi/pdf/10.1021/jm050563r" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (467 KB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/jm050563r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Fjm050563r%26sid%3Dliteratum%253Aachs%26pmid%3D16250656%26genre%3Darticle%26aulast%3DWang%26date%3D2005%26atitle%3DFast%2BSmall%2BMolecule%2BSimilarity%2BSearching%2Bwith%2BMultiple%2BAlignment%2BProfiles%2Bof%2BMolecules%2BRepresented%2Bin%2BOne-Dimension%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D48%26issue%3D22%26spage%3D6980%26epage%3D6990%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=290663" title="Pharmacophores">Pharmacophores</a>,</li><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=291754" title="Molecules">Molecules</a>,</li><li><a href="/action/doSearch?ConceptID=292276" title="Small molecules">Small molecules</a>,</li><li><a href="/action/doSearch?ConceptID=292247" title="Sequence alignment">Sequence alignment</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/48/22" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm050563r/production/images/medium/jm050563rn00001.gif" alt="" id="_i2" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm050563r/production/images/large/jm050563rn00001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm050563r&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Multiple sequence alignment has proven to be a powerful method for creating protein and DNA sequence alignment profiles. These profiles of protein families are useful tools for identifying conserved motifs, such as the catalytic triad of the serine protease family or the seven transmembrane helices of the G-protein coupled receptor family. Ultimately, the understanding of the critical motifs within a family is useful for identifying new members of the family. Due to the complexity of protein−ligand recognition, no universally accepted method exists for clustering small molecules into families with the same or similar biological activity. A combination of the concept of multiple sequence alignment and the 1-dimensional molecular representation described earlier offers a new method for profiling sets of small molecules with the same biological activity. These small molecule profiles can isolate key commonalties within the set of bioactive compounds much like a multiple sequence alignment can isolate critical motifs within a protein family. The small molecule profiles then make useful tools for searching small molecule databases for new compounds with the same biological activity. The technique is demonstrated here using the human ether-a-go-go potassium channel and the kinase SRC. </p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div class="extra-info-sec fnGroup"><div class="fnGroupItem" id="jm050563rAF1"><span><h4><span class="NLM_fn-symbol">*</span></h4></span><span><p class="last">  To whom correspondence should be addressed:  <a href="/cdn-cgi/l/email-protection" class="__cf_email__" data-cfemail="412525282d2d243301">[email protected]</a> pharmacop.com, (609) 452-3783 (phone), (609) 655-4187 (fax).</p></span></div></div><div id="" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_52530" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_52530" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">In bioinformatics, DNA and protein sequence alignment is the foundation for predicting structural similarity and thus the likelihood of similarity in function. In cheminformatics, a host of chemical similarity measures are used to assess the similarity between pairs of small molecules and thus the likelihood of them having similar function, i.e., biological activity. The list of small molecule similarity identification techniques includes substructure fingerprints,<sup><a onclick="showRef(event, 'jm050563rb00001 jm050563rb00002'); return false;" href="javascript:void(0);" class="ref jm050563rb00001 jm050563rb00002">1,2</a></sup> pharmacophore fingerprints,<sup><a onclick="showRef(event, 'jm050563rb00003 jm050563rb00004 jm050563rb00005 jm050563rb00006 jm050563rb00007'); return false;" href="javascript:void(0);" class="ref jm050563rb00003 jm050563rb00004 jm050563rb00005 jm050563rb00006 jm050563rb00007">3-7</a></sup> and overall 3-D alignments.<sup><a onclick="showRef(event, 'jm050563rb00008 jm050563rb00009'); return false;" href="javascript:void(0);" class="ref jm050563rb00008 jm050563rb00009">8,9</a></sup> For the purpose of finding molecules with similar function, DNA and protein sequence alignment have several advantages over methods for determining chemical similarity. A protein sequence typically has 100−1000 amino acids. A small molecule on the other hand typically has 20−40 non-hydrogen atoms. Thus, the problem of determining chemical similarity does not have the same statistical power of large numbers. In addition, the biological activity of a small molecule often hinges on a single atom. Indeed, there are numerous cases in which the change, deletion, or addition of a single atom completely abolishes the biological activity of a small molecule. To further complicate matters, a single small molecule will often have multiple seemingly unrelated biological activities. In this sense the small molecule similarity problem is context specific. Figure 1<a data-tab="pane-pcw-Figures" href="#jm050563rf00001" id="" class=" internalNav"></a>a shows a molecule that both inhibits the p38 kinase and acts as an antagonist of the glucagon receptor, a class B GPCR. Figure 1b shows a related molecule that is an inhibitor of the p38 kinase but not an antagonist of the glucagon receptor. Figure 1c shows a second related molecule that retains the glucagon antagonist activity but is devoid of the p38 activity. From the viewpoint of p38 molecule 1a is similar to 1b but not 1c whereas from the perspective of the glucagon receptor 1a is similar to 1c but not 1b. <div id="_i4"><figure id="jm050563rf00001" data-index="1" class="article__inlineFigure"><h2 class="fig-label"></h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm050563r/production/images/medium/jm050563rf00001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm050563r/production/images/large/jm050563rf00001.jpeg" id="_i5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last"> Figure 1 An example of a small molecule showing multiple biological activities and subtle changes that distinguish between these activities.<sup><a onclick="showRef(event, 'jm050563rb00050'); return false;" href="javascript:void(0);" class="ref jm050563rb00050">50</a></sup> This example shows that assigning function to a small molecule, i.e., its biological activity, based on similarity is a context specific problem. (A) Compound <b>A</b> both inhibits the p38 kinase and acts as an antagonist of the glucagon receptor which is a GPCR from the class B family. Compound <b>B</b> is an example of a small molecule that is closely related to compound <b>A</b> and retains the p38 inhibitory activity but is devoid of the glucagon activity. Compound <b>C</b> is an example of a small molecule that is closely related to <b>A</b> and retains the glucagon activity but is devoid of the p38 activity. Thus, from the perspective of p38, molecule <b>A</b> is similar to <b>B</b> but not <b>C</b> whereas from the perspective of the glucagon receptor, <b>A</b> is similar to <b>C</b> but not <b>B</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm050563r/production/images/large/jm050563rf00001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm050563r&amp;id=jm050563rf00001"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div><div class="NLM_p">Despite the aforementioned differences, the problems of assigning function to a protein and assigning function to a small molecule share a common challenge. Often <a name="jm004802206981" id="jm004802206981"></a> two proteins with little or no overall sequence identity share a common function. An example is the serine protease family in which only the catalytic triad (Ser, Asp, and His) is required for function while the remainder of the amino acids largely serve to precisely position the catalytic triad.<sup><a onclick="showRef(event, 'jm050563rb00010 jm050563rb00011 jm050563rb00012 jm050563rb00013'); return false;" href="javascript:void(0);" class="ref jm050563rb00010 jm050563rb00011 jm050563rb00012 jm050563rb00013">10-13</a></sup> Over a few hundred amino acids, three residues have no effect on overall sequence identity. Similarly, two small molecules with little or no obvious similarity often possess the same biological activity. Much like the example of the serine protease family, the binding of a small molecule to a specific protein often relies on a few key interaction points, such as hydrogen bond acceptors, donors, aromatic rings, etc., with the remainder of the molecule serving to precisely position the key interaction points. The set of interaction points and their precise arrangement is referred to as the pharmacophore.<sup><a onclick="showRef(event, 'jm050563rb00014'); return false;" href="javascript:void(0);" class="ref jm050563rb00014">14</a></sup></div><div class="NLM_p">Bioinformatics addresses the issue of identifying relationships between proteins in large part through multiple sequence alignment. The advantage of multiple sequence alignment over pairwise alignment is that multiple sequence alignment can find residues that are conserved over an entire protein family thereby distinguishing the critical amino acids or motifs such as the catalytic triad of the serine protease family. The most comparable technique in computational chemistry is pharmacophore modeling. In this case, multiple small molecules are aligned in 3 dimensions. The features that the majority of the small molecules share are analogous to the conserved amino acids of a multiple sequence alignment. Pharmacophore models are often used to search through a small molecule database for molecules with the same biological activity much like a multiple sequence alignment can be used to search for related family members. </div><div class="NLM_p">A key difference between multiple sequence alignment and pharmacophore modeling is that multiple sequence alignment is inherently a 1-dimensional search problem and is therefore amenable to a host of specialized algorithms whereas pharmacophore modeling is inherently a high dimensional problem involving for each molecule 3 rotational, 3 translational, and the sometimes many conformational degrees of freedom. In computational chemistry, 3-dimensional methods in general and pharmacophore modeling in particular suffer from the problem of conformational explosion as the number of rotatable bonds increases. This conformational explosion makes the initial elucidation of the pharmacophore extremely challenging and affects both the quality and speed of the search of conformational databases. </div><div class="NLM_p">Recently, Dixon and Merz<sup><a onclick="showRef(event, 'jm050563rb00015'); return false;" href="javascript:void(0);" class="ref jm050563rb00015">15</a></sup> developed a method for assessing molecular similarity that is directly analogous to pairwise sequence alignment. In this method the atoms of the small molecule, either from 3-dimensional coordinates or 2-dimensional topological distances, are projected onto 1 dimension using multidimensional scaling.<sup><a onclick="showRef(event, 'jm050563rb00016'); return false;" href="javascript:void(0);" class="ref jm050563rb00016">16</a></sup> This results in the molecule being represented as a 1-dimensional string of atoms where the atom pairwise distances retain as much information as possible concerning their true 2- or 3-dimensional distances. Once two molecules have been projected to 1 dimension they can be rapidly aligned using the techniques of pairwise alignment. The chief differences between pairwise protein sequence alignment and pairwise 1-dimensional molecular alignment are; first, for 1-dimensional molecular alignment, both relative orientations must be considered, second, insertions and deletions do not make sense in the context of a small molecule, and third for small molecule alignment, the 1-dimensional representations can be aligned continuously relative to one another rather than in discrete steps. In a variety of tests Dixon and Merz have shown this molecular similarity method to be superior to a variety of other similarity methods. </div><div class="NLM_p">In this work, we extend the method of Dixon and Merz to multiple ligand alignment. The goal is to create 1-dimensional profiles from many molecules with the same biological activity that identify the features common to all or most of the molecules. Such a profile could potentially isolate the key features of the molecules, much like a pharmacophore model isolates key interaction sites and a multiple sequence alignment isolates conserved amino acids. Furthermore these profiles would avoid the difficulties associated with the high dimensionality encountered in the conformational search problem. </div><div class="NLM_p">To align 1-dimensional representation of K molecules, there are a total of 2<sup>k-1</sup> possible relative orientations. As an example, Figure 2<a data-tab="pane-pcw-Figures" href="#jm050563rf00002" id="" class=" internalNav"></a>a shows the 4 relative orientations of 3 1-dimensional objects. With 10 molecules there are 512 possible relative orientations, and with 20 molecules there are over 50 000 possible relative orientations. To find the global maximal alignment, one must solve a continuous global optimization problem in <i>K</i> − 1 dimensions for each of the possible 2<i><sup>K</sup></i><sup>-1</sup> relative orientations. As an example the 2-dimensional similarity landscape for 3 molecules, Cisapride,<sup><a onclick="showRef(event, 'jm050563rb00017'); return false;" href="javascript:void(0);" class="ref jm050563rb00017">17</a></sup><a name="jm004802206982" id="jm004802206982"></a> Thioridazine,<sup><a onclick="showRef(event, 'jm050563rb00018'); return false;" href="javascript:void(0);" class="ref jm050563rb00018">18</a></sup> and Ziprasidone<sup>18</sup> are shown in Figure 2b. The number of local maxima is certain to increase exponentially with the number of molecules. Because of the complexity of the high dimensional similarity landscape and the large number of discrete relative orientations, a brute force solution to the overall global optimization problem is not practical. In addition, the large number of local maxima and the presence of discrete variables preclude the exclusive use of gradient-based methods. <div id="_i6"><figure id="jm050563rf00002" data-index="2" class="article__inlineFigure"><h2 class="fig-label"></h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm050563r/production/images/medium/jm050563rf00002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm050563r/production/images/large/jm050563rf00002.jpeg" id="_i7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last"> Figure 2 The search space in a multiple 1-dimensional alignment of three molecules. (a) The 4 possible relative orientations of the 3 1-dimensional objects. Note that the four remaining orientations in which the top arrow points to the left are by flipping all three arrows mathematically equivalent to one of the four relative orientations shown in part a. (b) The 2D search space for a single relative orientation of the three molecules Cisapride, Thioridazine, and Ziprasidone.  </p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm050563r/production/images/large/jm050563rf00002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm050563r&amp;id=jm050563rf00002"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div><div class="NLM_p last">To produce a consensus 1-dimensional alignment of multiple molecules, a heuristic approach is used:  evolutionary programming<sup><a onclick="showRef(event, 'jm050563rb00019'); return false;" href="javascript:void(0);" class="ref jm050563rb00019">19</a></sup> to address the continuous variables and genetic algorithms<sup><a onclick="showRef(event, 'jm050563rb00020'); return false;" href="javascript:void(0);" class="ref jm050563rb00020">20</a></sup> to handle the discrete variables. The combination of genetic algorithms and evolutionary programming has been shown through numerous examples to robustly handle optimization problems with continuous and discrete variables<sup><a onclick="showRef(event, 'jm050563rb00021 jm050563rb00022'); return false;" href="javascript:void(0);" class="ref jm050563rb00021 jm050563rb00022">21,22</a></sup> and thus is particularly well suited to the problem of multiple 1-dimensional ligand alignment. Due to the complexity of the optimization problem involved, we adopt the heuristic combination of evolutionary programming and genetic algorithms for building a near optimal multiple ligand alignment profile. We then show these profiles to be useful for rapidly searching large compound databases for novel molecules with the same biological activity. Finally, we compare the results with the 1-dimensional profiles to results using comparable 3-dimensional methods. We demonstrate the utility of this approach with the human ether-a-go-go potassium channel and the kinase SRC. </div></div><div id="" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i8">Methods</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_24608" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_24608" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">To obtain multiple 1-dimensional ligand alignments, a heuristic approach utilizing evolutionary programming and genetic algorithms was used to ‘evolve' a solution, resulting in a consensus multiple alignment profile of the selected set of compounds. Each small molecule to be aligned is first converted, using only 2-dimensional topological distances, to its 1-dimensional representation through the multidimensional scaling and BFGS optimization procedure as described for the pairwise case.<sup>15</sup> Each small molecule's 1-dimensional encoding consists of information about the type of each atom and its 1-dimensional coordinate. </div><div class="NLM_p">For the purposes of this description, it is assumed that each of the 1-dimensional representations has been created such that its center of mass is 0, i.e., the sum over the 1-dimensional coordinates of its atoms is 0. To put this molecule's 1-dimensional representation into a new frame of reference, its orientation and translation must be specified. An orientation takes a value of 1 or −1 where a value of 1 means to make no change in orientation and a value of −1 means to flip the 1-dimensional representation of the molecule. The translation is a continuous variable, takes any real value, and essentially shifts the representation along the 1-dimensional axis. Mathematically, the 1-dimensional coordinates are transformed by<span class="NLM_inline-graphic"><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm050563r/production/images/medium/jm050563re10001.gif" alt="" /></img></span><sup>new</sup> = ρ<span class="NLM_inline-graphic"><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm050563r/production/images/medium/jm050563re10002.gif" alt="" /></img></span><sup>initial</sup> + Δ<i>x</i> where ρ is the orientation and Δ<i>x</i> is the translation. </div><div class="NLM_p"><b>Organism/Gene Encoding.</b> The first step in a genetic algorithm or evolutionary program is the generation of an initial population of potential solutions referred to as organisms. Each organism consists of a set of genes, with each gene representing the translation and orientation of a single molecule. The initial set of generated genes contains translation values distributed via a Gaussian distribution with mean 0 and standard deviation equal to one-quarter of the molecule's 1-dimensional width. The initial orientation of each gene was chosen purely at random. </div><div class="NLM_p"><b>Alignment Scoring/Organism Fitness.</b> To assess the fitness of an organism, the quality of any given multiple alignment must be scored. The score is given by<a class="ref internalNav" href="#jm050563re00001" aria-label=""></a><span class="NLM_disp-formula" id="jm050563re00001"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm050563r/production/images/medium/jm050563re00001.gif" alt="" id="_i9" /></img></span> where the outer sum is over all pairs of molecules in the alignment, the second sum is over the atoms of the ith molecule, the inner sum is over the atoms of the jth molecule, <i>S</i>(a<sub>ik</sub>,a<sub>jm</sub>) is the similarity of the kth atom of the ith molecule to the mth atom of the jth molecule, x<sub>ik</sub> and x<sub>jm</sub> are the 1-dimensional coordinates of the two atoms, and <i>f</i> is a distance dependent measure of the overlap of two atoms. The full description of the scoring scheme requires two components:  the atom pairwise similarity matrix, <i>S</i>, and the distance dependent overlap function, <i>f</i>. These are described below. </div><div class="NLM_p">The overlap function, f, in eq 1 could reasonably be any number of functions. For this work the original overlap function proposed by Dixon and Merz<sup>15</sup> was used:<a class="ref internalNav" href="#jm050563re00002" aria-label=""></a><span class="NLM_disp-formula" id="jm050563re00002"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm050563r/production/images/medium/jm050563re00002.gif" alt="" id="_i10" /></img></span></div><div class="NLM_p">This overlap score is best thought of as each atom having a width of 1 bond unit and the overlap being the area under the intersection of the two atoms. </div><div class="NLM_p"><b>Atom Pairwise Scoring </b><b>Matrixes.</b> There are no well-established atom similarity matrixes such as the Dayhoff,<sup><a onclick="showRef(event, 'jm050563rb00023'); return false;" href="javascript:void(0);" class="ref jm050563rb00023">23</a></sup> PAM,<sup>23</sup> MDM,<sup>23</sup> BLOSUM,<sup><a onclick="showRef(event, 'jm050563rb00024'); return false;" href="javascript:void(0);" class="ref jm050563rb00024">24</a></sup> GCB,<sup><a onclick="showRef(event, 'jm050563rb00025'); return false;" href="javascript:void(0);" class="ref jm050563rb00025">25</a></sup> and PET<sup><a onclick="showRef(event, 'jm050563rb00026'); return false;" href="javascript:void(0);" class="ref jm050563rb00026">26</a></sup> similarity matrixes for amino acids. There are a large number of factors that could be taken into account in determining atom similarity. These factors include, actual atom type, atom hybridization, partial charge, polarizability, aromaticity, hydrophobicity, etc. As many of these factors depend on both the particular atom and its neighbors in the molecule, there could be in essence an infinite number of atom types. A reasonable atom similarity matrix must have a sufficiently detailed atom description without having an unmanageable number of atom types. </div><div class="NLM_p">Ultimately, the atom types used were the same as those used by Dixon and Merz.<sup>15</sup> In this scheme an atom's type is determined by its atomic number, its hybridization, number of bonded hydrogens, and whether it is a member of a ring. The most obvious atom pairwise similarity matrix is the identity matrix, i.e., atoms have a similarity of 1 if they have identical type; otherwise they have a similarity of 0. This matrix proved to be too strict. The analogy for protein sequence alignment would be to classify amino acids such as leucine and isoleucine as completely different. Experience has shown that similarity matrixes such as BLOSUM, PAM, and the hydrophobicity matrixes are significantly more sensitive than the identity matrix. With this in mind similarity matrix 1 (denoted by M below) was developed with the following rules: </div><div class="NLM_p">Except for the halogens, atoms of different atomic number have no similarity. The similarity between atoms of the same atomic number decreases from 1 to a minimum of 0 via the rules:  decrease by 0.4 for each change in hybridization; decrease by 0.2 if one of the atoms is in a ring while the other is not; decrease by 0.2 for the difference in number of hydrogens. The similarity for halogens is 1−0.2 × <i>n</i> where <i>n</i> is the difference in the rows of the periodic table of the two halogens. </div><div class="NLM_p">Ultimately, similarity matrix 1 was not sufficiently discriminating because the alignments were always dominated by the carbon atoms of the molecules. Often, the carbon atoms act more as the framework of the molecule and less as prominent interactions. To overcome this issue, the MDDR<sup><a onclick="showRef(event, 'jm050563rb00027'); return false;" href="javascript:void(0);" class="ref jm050563rb00027">27</a></sup> database was profiled for the frequency of occurrence of each possible atom type. From the vector of occurrences, P, the weighted occurrence vector, W, is defined by W = MP where M is similarity matrix 1. The element W<sub>j</sub>, of the weighted occurrence vector, W, is the expected similarity between an <a name="jm004802206983" id="jm004802206983"></a> atom of type j and a randomly chosen atom from the MDDR database. Similarity matrix 2 is then derived by scaling similarity matrix 1 by the weighted occurrences via the formula<a class="ref internalNav" href="#jm050563re00003" aria-label=""></a><span class="NLM_disp-formula" id="jm050563re00003"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm050563r/production/images/medium/jm050563re00003.gif" alt="" id="_i11" /></img></span> Similarity matrix 2 was used for all atom pairwise similarity calculations. </div><div class="NLM_p"><b>Selection Process.</b> There are many ways in which a new generation of potential solutions can be created from the previous generation. For this work the selection process was done through roulette wheel selection<sup>21,22</sup> with (mu,lambda) population replacement from one generation to the next. Given a population of organisms, the probability <i>Q</i><sub>i</sub> for organism i to have offspring in the next generation is dependent on its fitness score, <i>E</i>, and the temperature constant, <i>T</i>, via the relationship, <i>Q</i><sub>i</sub>∝<i>e</i><sup>(Si/</sup><i><sup>T</sup></i><sup>)</sup>, where the constant of proportionality is chosen so that the sum of the <i>Q</i><sub>i</sub> over the population is 1 and <i>T</i> is a constant used to control the global versus local nature of the search. </div><div class="NLM_p"><b>Recombination and Mutation Processes.</b> A genetic algorithm was used to perform crossover with two parents to yield a single offspring. Each gene of the offspring was inherited at random with uniform crossover with 50% probability. At this point the genome of the offspring is a combination of the genomes of its parents. </div><div class="NLM_p">After the entire offspring generation is created, two types of random mutations were used to further modify each organism:  flip mutation and shift mutation. The flip mutation takes the form of bit flipping which is typical of a genetic algorithm but usually not seen in evolutionary programming. The flip mutation negates the orientation of the 1-dimensional atom coordinates resulting in a 1-dimensional representation that is flipped. The shift mutation uses a Gaussian distribution to change the translation of a 1-dimensional representation essentially shifting the molecule along the 1-dimensional axis. This is typical of evolutionary programming but usually not seen in genetic algorithms. Both the shift and flip mutations were applied to each molecule with a probability of 0.05%. A Gaussian distribution with a standard deviation of 0.5 bond units was used to randomly perturb those molecules selected for a shift mutation. Also, for this work a population size of 128 was used with 500 steps of the evolutionary process. </div><div class="NLM_p"><b>The Compound Database Search.</b> After the multiple alignment of the small molecules with the same biological activity of interest, we wish to search large databases of available or virtual compounds with the intent of finding novel small molecules with the same biological activity. To assess the likelihood of a small molecule having the desired biological activity, its 1-dimensional representation must be aligned to the multiple ligand profile. To find the optimal alignment, two 1-dimensional optimizations (both relative orientations) must be performed. With two straightforward techniques the database search can be extremely fast:  ∼900 molecules per second on a single SGI R10000 processor. </div><div class="NLM_p">The first technique is to create a lookup table for each atom type at the beginning of the database search. To do so, a fixed step size (usually 0.1 bond units) is chosen. Beginning at the lower limit of the profile the score for an atom of type 1 is calculated and stored in the first position of the array. Next the atom is moved one step size to the right, the score for the atom is calculated and stored in the second position of the array. The process is continued until the atom reaches the upper limit of the profile. The process is then repeated for each atom type. When the score for a molecule from the database needs to be calculated, the score for each atom can be found using its 1-dimensional coordinate from the appropriate array. The memory needed to store the arrays is less than 1 Mb, and the time necessary to calculate the lookup table is less than 10 s on an SGI R10000. </div><div class="NLM_p last">The second technique is a standard bracketing technique used for solving 1-dimensional optimization problems.<sup><a onclick="showRef(event, 'jm050563rb00028'); return false;" href="javascript:void(0);" class="ref jm050563rb00028">28</a></sup> Again, a fixed step size (usually 0.5 bond units) and upper and lower limits (determined by the bounds of the profile and the bounds of the molecule) are chosen. Beginning at the lower limit the score for the molecule is calculated. The molecule is moved one step to the right and the score calculated again. The process is continued until the molecule moves past the upper limit. Three consecutive offsets of the molecule (x1 < x2 < x3) are said to bracket a maximum if S(x1) < S(x2) and S(x3) < S(x2):  because the scoring function S is continuous, one can mathematically guarantee that under these conditions S will have a local maximum somewhere between x1 and x3.<sup>28</sup> Standard line optimization techniques can then be used to find any bracketed local maximum to any desired level of accuracy. </div></div><div id="" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i12">Results</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_24883" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_24883" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">All molecules used in these studies were projected into their 1-dimensional representation using only their interatomic topological distances via the procedure outlined by Dixon and Merz.<sup>15</sup> Thus, there is no need to produce intermediate 3-dimensional coordinates at any stage in the process. To quantify the extent of loss of information we calculated the average distortion with the 1-dimensional representation of each molecule in the MDDR database with respect to its 2-dimensional structure. The average distortion of a molecule is calculated via<a class="ref internalNav" href="#jm050563re00004" aria-label=""></a><span class="NLM_disp-formula" id="jm050563re00004"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm050563r/production/images/medium/jm050563re00004.gif" alt="" id="_i13" /></img></span> where <i>N</i> is the number of atoms in the molecule, <i>d</i><i><sub>ij</sub></i> is the topological distance between atoms i and j, and<span class="NLM_inline-graphic"><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm050563r/production/images/medium/jm050563re10003.gif" alt="" /></img></span><sup>D</sup> is the distance between atoms i and j in the 1-dimensional representation. In essence the formula captures the average distortion between all pairs of atoms in the molecule. Figure 3<a data-tab="pane-pcw-Figures" href="#jm050563rf00003" id="" class=" internalNav"></a> shows the distribution of the distortion of the molecules in MDDR in their 1-dimensional representation. 90% of the molecules exhibit a distortion below 9%, 95% exhibit a distortion below 11%, 99% exhibit a distortion below 14%, and essentially all the molecules of the MDDR database exhibit a distortion below 20%. Thus, the 1-dimensional representation <a name="jm004802206984" id="jm004802206984"></a> retains the majority of the information present in the 2-dimesional structure. <div id="_i14"><figure id="jm050563rf00003" data-index="3" class="article__inlineFigure"><h2 class="fig-label"></h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm050563r/production/images/medium/jm050563rf00003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm050563r/production/images/large/jm050563rf00003.jpeg" id="_i15" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last"> Figure 3 The distribution of the extent of distortion of each molecule caused by projecting from two-dimensional topological distances to its 1-dimensional representation. 90% of the molecules exhibit a distortion below 9%, 95% exhibit a distortion below 11%, 99% exhibit a distortion below 14%, and essentially all the molecules of the MDDR database exhibit a distortion below 20%. Thus, the 1-dimensional representation retains the majority of the information present in the 2-dimesional structure.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm050563r/production/images/large/jm050563rf00003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm050563r&amp;id=jm050563rf00003"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div><div class="NLM_p">To demonstrate the utility of the multiple ligand profiles for identifying novel small molecules with similar biological activity, we perform seeding experiments:  a relatively large set of small molecules with<a data-tab="pane-pcw-Figures" href="#jm050563rf00004" id="" class=" internalNav"></a> the same biological activity are collected and split into a training and test set. For the purpose of this work, the training sets consist of 10 molecules and the test sets consist of at least 50 small molecules. The small molecules of the training set are used to build the 1-dimensional profiles. The profiles are then used to rank the molecules of the test set along with those from a database of random compounds. The quality of the search method is judged by the extent to which it differentiates the molecules of the test set from those of the random database. <div id="_i16"><figure id="jm050563rf00004" data-index="4" class="article__inlineFigure"><h2 class="fig-label"></h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm050563r/production/images/medium/jm050563rf00004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm050563r/production/images/large/jm050563rf00004.jpeg" id="_i17" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last"> Figure 4 The SRC 1-dimensional multiple ligand alignment. (a) The compounds used in the alignment. (b) The resulting alignment. Each compound is shown in part a roughly oriented in the same manner as in the alignment in part b. The color indicates the type of atom, i.e., gray is carbon, red is oxygen, blue is nitrogen, etc. The darkness of the color indicates the hybridization:  light colors indicate Sp3 atoms and dark colors indicate Sp2 or Sp1 atoms. The size of the atom is proportional to the amount the atom contributes to the score of the overall alignment.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm050563r/production/images/large/jm050563rf00004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm050563r&amp;id=jm050563rf00004"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div><div class="NLM_p">The random molecules (∼100 000) were drawn from the MDDR database<sup>27</sup> with a molecular weight cutoff of 600 Da. The MDDR database consists of drug-like molecules and is representative of the types of small molecules synthesized in medicinal chemistry programs. Thus, selecting random small molecules from the MDDR database makes these realistic examples. </div><div class="NLM_p">The examples described below are:  the human ether-a-go-go potassium channel (hERG) and the kinase SRC. The kinase SRC has been implicated in a variety of cancers.<sup><a onclick="showRef(event, 'jm050563rb00029'); return false;" href="javascript:void(0);" class="ref jm050563rb00029">29</a></sup> In addition, SRC is used as a selectivity assay in many kinase programs. Thus, a SRC virtual screen would be useful as a means to generate new leads and as a virtual filter to identify potential selectivity issues in kinase programs. As a test case SRC is representative of a medicinal chemistry program in that there are a small handful of chemotypes all of which are represented in the training set. Thus, the test set contains molecules whose chemotypes are all represented in the training set. </div><div class="NLM_p">The second example is the hERG potassium channel. Inhibition of the hERG potassium channel has recently been linked to cardiotoxicity such as prolonged QT <a name="jm004802206985" id="jm004802206985"></a> interval and torsades de pointes.<sup><a onclick="showRef(event, 'jm050563rb00030'); return false;" href="javascript:void(0);" class="ref jm050563rb00030">30</a></sup> Drugs such as cisapride<sup>17</sup> and terfenadine<sup><a onclick="showRef(event, 'jm050563rb00031'); return false;" href="javascript:void(0);" class="ref jm050563rb00031">31</a></sup> have been withdrawn from the market because of their propensity to cause fatal arrhythmias caused by blocking the hERG channel. Thus, the hERG channel is a counter screen in many medicinal chemistry programs. The hERG test set consists largely of chemotypes not represented in the training set. Thus, as an example the hERG data set is more representative of the challenges faced in discovering new chemotypes either during discovery efforts or lead hopping. </div><div class="NLM_p"><b>The SRC Validation Study.</b> The final SRC profile and the 10 compounds used to build the profile are shown in Figure 4a. In this case the test set consists of 142 known SRC inhibitors.<sup><a onclick="showRef(event, 'jm050563rb00032 jm050563rb00033 jm050563rb00034 jm050563rb00035 jm050563rb00036 jm050563rb00037 jm050563rb00038 jm050563rb00039 jm050563rb00040 jm050563rb00041 jm050563rb00042'); return false;" href="javascript:void(0);" class="ref jm050563rb00032 jm050563rb00033 jm050563rb00034 jm050563rb00035 jm050563rb00036 jm050563rb00037 jm050563rb00038 jm050563rb00039 jm050563rb00040 jm050563rb00041 jm050563rb00042">32-42</a></sup> The strongest feature in the SRC profile is a central aromatic nitrogen (see Figure 4b). One could hypothesize that these aligned nitrogens make the hydrogen bond with the backbone NH of the hinge region<sup><a onclick="showRef(event, 'jm050563rb00043'); return false;" href="javascript:void(0);" class="ref jm050563rb00043">43</a></sup> often observed as critical in kinase−ligand interactions. In 9 of the 10 molecules used to make the SRC profile the structure−activity relationships strongly support the aromatic ring aligned to the left of the profile as being the ring that binds in the main hydrophobic pocket. The group that likely binds in the main hydrophobic pocket for molecule SRC-8 (see Figure 4) is the aromatic ring on the same side as the NH2. Thus, this molecule should likely be oriented the same as SRC-9 and SRC-10. The psuedo-2-fold symmetry makes the problem of orienting these three molecules particularly difficult. </div><div class="NLM_p">The overall results of the database search with the SRC 1-dimensional profile are shown in Figure 5<a data-tab="pane-pcw-Figures" href="#jm050563rf00005" id="" class=" internalNav"></a>. Approximately 26% of the known SRC inhibitors are ranked within the top 1% of the MDDR compounds (enrichment = 26), 58% of the known SRC inhibitors are ranked within the top 5% of the MDDR compounds (enrichment of 11.6), and nearly 95% of the known SRC inhibitors are found within the top 17% of the MDDR compounds (enrichment of 5.6). <div id="_i18"><figure id="jm050563rf00005" data-index="5" class="article__inlineFigure"><h2 class="fig-label"></h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm050563r/production/images/medium/jm050563rf00005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm050563r/production/images/large/jm050563rf00005.jpeg" id="_i19" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last"> Figure 5 The performance of the SRC profile. To make these figures, the molecules are ranked and sorted by their score, in this case their similarity to the 1-dimensional profile. The top curve is the fraction of the known SRC actives found (<i>y</i>-axis) versus the fraction of the overall compounds (<i>x</i>-axis). The diagonal line is the expected performance of a method that selected compounds at random.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm050563r/production/images/large/jm050563rf00005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm050563r&amp;id=jm050563rf00005"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div><div class="NLM_p"><b>The hERG Validation Study.</b> The final 1-dimensional profile and the 10 molecules used to build the hERG profile are shown in Figure 6<a data-tab="pane-pcw-Figures" href="#jm050563rf00006" id="" class=" internalNav"></a>. In this case, the hERG test set consists of 92 known hERG inhibitors. The central feature of the hERG profile is a basic amine; only Loratadine lacks this feature. The basic center is immediately surrounded by aliphatic carbons. Both ends of the profile are dominated by aromatic rings. <div id="_i20"><figure id="jm050563rf00006" data-index="6" class="article__inlineFigure"><h2 class="fig-label"></h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm050563r/production/images/medium/jm050563rf00006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm050563r/production/images/large/jm050563rf00006.jpeg" id="_i21" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last"> Figure 6 The hERG 1-dimensional Multiple Ligand Alignment. (a) The compounds used in the alignment. (b) The hERG 1-dimensional alignment. Each compound is shown in <b>6a</b> roughly oriented in the same manner as in the alignment in part b. The atoms are colored and sized as described in Figure 4b.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm050563r/production/images/large/jm050563rf00006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm050563r&amp;id=jm050563rf00006"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div><div class="NLM_p">The performance of the hERG profile, shown in Figure 7<a data-tab="pane-pcw-Figures" href="#jm050563rf00007" id="" class=" internalNav"></a>, is not nearly as crisp as those with the SRC data set. In this case 6% of the known hERG inhibitors are ranked in the top 1% of the MDDR compounds, 25% of the known hERG inhibitors are ranked in the top 3% of the MDDR compounds, and 30% of the known hERG inhibitors are ranked in the top 5% of the MDDR compounds. Thus, the enrichment factors with the hERG data set range from 6 to 8 when considering the top 1−5% of the overall compounds compared to enrichment factors of 11−26 with the SRC data set over the same range. The difference in performance likely stems from the fact that the hERG potassium channel is much less specific in its binding requirements when compared to typical proteins such as SRC. This affects results such as these in numerous ways. First, there are in all likelihood far more hERG inhibitors than SRC inhibitors within the MDDR database. Second, the hERG data set is much more diverse than the SRC data set. Both of these considerations would be expected to increase the difficulty in tests such as these. <div id="_i22"><figure id="jm050563rf00007" data-index="7" class="article__inlineFigure"><h2 class="fig-label"></h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm050563r/production/images/medium/jm050563rf00007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm050563r/production/images/large/jm050563rf00007.jpeg" id="_i23" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last"> Figure 7 The performance of the hERG profile. This figure was created using the hERG 1-dimensional profile to rank the compounds of the MDDR and the hERG training set in the same manner as Figure 5.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm050563r/production/images/large/jm050563rf00007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm050563r&amp;id=jm050563rf00007"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div><div class="NLM_p"><b>A Comparison to 3-Dimensional Methods.</b> Here we compare the results with the 1-dimensional profiles to those with two 3-dimensional methods:  explicit pharmacophore models and 3-point pharmacophore fingerprints. These methods are commonly used and well validated and therefore create good benchmarks for comparison. </div><div class="NLM_p">In all cases, the conformations were generated for the molecules using Catalyst<sup><a onclick="showRef(event, 'jm050563rb00044'); return false;" href="javascript:void(0);" class="ref jm050563rb00044">44</a></sup> with the fast option and a 15 kcal/mol strain cutoff. The pharmacophore fingerprints were generated from within Cerius2<sup><a onclick="showRef(event, 'jm050563rb00045'); return false;" href="javascript:void(0);" class="ref jm050563rb00045">45</a></sup> using a 10 Å grid with a uniform 2.0 Å spacing and using all available features:  negative charge, positive charge, negative ionizable, positive ionizable, hydrogen bond acceptor, hydrogen bond acceptor projection, hydrogen bond donor, hydrogen bond donor projection, aromatic ring, aromatic ring projection, and hydrophobic features. From the pharmacophore fingerprints of two molecules the Tanimoto coefficient was used as the similarity measurement. Finally, to rank the compounds of the MDDR database and the test sets, both the mean and maximum similarity to the compounds in the training set were used. </div><div class="NLM_p">To create a pharmacophore model for hERG, we recreated the model described by Ekins and co-workers.<sup><a onclick="showRef(event, 'jm050563rb00046'); return false;" href="javascript:void(0);" class="ref jm050563rb00046">46</a></sup> This model consists of a positive ionizable feature and four hydrophobic features. To create a pharmacophore model for SRC, the crystal structures 1qcf (HCK),<sup><a onclick="showRef(event, 'jm050563rb00047'); return false;" href="javascript:void(0);" class="ref jm050563rb00047">47</a></sup> 1m17 (EGFr),<sup><a onclick="showRef(event, 'jm050563rb00048'); return false;" href="javascript:void(0);" class="ref jm050563rb00048">48</a></sup> and 1m52(ABL)<sup><a onclick="showRef(event, 'jm050563rb00049'); return false;" href="javascript:void(0);" class="ref jm050563rb00049">49</a></sup> are used, each of which is cocrystalized with a ligand similar to at least one of the compounds in the training set (see Figure 8<a data-tab="pane-pcw-Figures" href="#jm050563rf00008" id="" class=" internalNav"></a>). Two pharmacophore models, a four feature and a six feature model, were then created by manually mapping the observed conserved interactions. The four feature model consists of a hydrogen bond acceptor representing the aromatic nitrogen that interacts with the backbone NH of the hinge region (Met341 of SRC), an aromatic ring feature representing the heteroaromatic core, an aromatic ring representing the group in <a name="jm004802206986" id="jm004802206986"></a> the main hydrophobic pocket, and a hydrophobic feature representing the groups in the secondary hydrophobic patch. The six feature model consists of these same four features with two additional hydrogen bond donors that represent the interactions with the backbone carbonyl oxygens of the SRC hinge region (Glu339 and Met341). The donors are observed in some cocrystallized kinase inhibitors of the same chemotype used in the SRC training set but not in others. With both hERG and SRC pharmacophore models the “Fit Value” produced by the Catalyst utility citest was used to rank the compounds in the MDDR database and in the test sets. <div id="_i24"><figure id="jm050563rf00008" data-index="8" class="article__inlineFigure"><h2 class="fig-label"></h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm050563r/production/images/medium/jm050563rf00008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm050563r/production/images/large/jm050563rf00008.jpeg" id="_i25" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last"> Figure 8 The compounds and the features used to build the SRC pharmacophore model. The three compounds were selected from crystal structures of related kinases because they are representative of the chemotypes found in the training set. <b>A</b> was taken from the crystal structure of 1m17<sup>48</sup> (EGFr). <b>B</b> was take from the pdb structure 1m52<sup>49</sup> (ABL). <b>C</b> was taken from pdb structure 1qcf<sup>47</sup> (HCK). The orange aromatic ring of the three ligands bound in the main hybrophobic pocket of the appropriate kinases. This interaction was represented by an aromatic ring feature. The blue nitrogen of the three ligands was observed to interact with the hinge NH (corresponds to Met341 of SRC) in each case. This interaction was represented by a hydrogen bond acceptor. The substructures highlighted in purple were bound in the secondary hydrophobic site and were represented as a hydrophobic feature. The central heteroaromatic core was represented by an aromatic ring feature. Finally, the NH of <b>B</b> was observed to interact with a backbone carbonyl oxygen of the kinase (corresponds to Met341 of SRC) while the NH2 of <b>C</b> was observed to interact with a second carbonyl oxygen of the kinase (corresponds to Glu339 of SRC). Each of these was represented by a hydrogen bond donor feature in the 6 feature model.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm050563r/production/images/large/jm050563rf00008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm050563r&amp;id=jm050563rf00008"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div><div class="NLM_p">The results from the comparisons of the 1-dimensional profiles to the 3-dimensional methods are shown in Figure 9<a data-tab="pane-pcw-Figures" href="#jm050563rf00009" id="" class=" internalNav"></a> (SRC) and Figure 10<a data-tab="pane-pcw-Figures" href="#jm050563rf00010" id="" class=" internalNav"></a> (hERG). For both data sets the pharmacophore fingerprints and the explicit pharmacophore models produced reasonable enrichments. In both cases, however, the 1-dimensional profiles outperform the 3-dimensional methods. With the SRC data set, the 1-dimensional profile significantly outperforms either the pharmacophore model or the pharmacophore fingerprints. In particular, the 3-dimensional methods show significant enrichment initially but quickly resume random behavior. Compounds for which the bioactive conformation was generated were probably ranked very high whereas those for which the conformational search failed to produce a conformation reasonably close to the bioactive conformation were ranked within the noise produced by the MDDR compounds. Since it avoids any explicit conformation 1-dimensional multiple alignment does not face this problem. <div id="_i26"><figure id="jm050563rf00009" data-index="9" class="article__inlineFigure"><h2 class="fig-label"></h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm050563r/production/images/medium/jm050563rf00009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm050563r/production/images/large/jm050563rf00009.jpeg" id="_i27" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last"> Figure 9 A comparison between the SRC 1-dimensional profile and 3-dimensional methods. (a) The performance of the 1-dimensional profile is in black. The performance of the 4 feature pharmacophore model is in red. The performance of the 6 feature pharmacophore model is in blue. The pharmacophore models are described in Figure 8. (b) The performance of the 3-dimensional profile is in black. The results with the 1-dimensional pharmacophore fingerprint are shown in blue and red. The maximum similarity between the training set compounds was used to create the red curve whereas the average similarity to the training set compounds was used to create the blue compounds.  </p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm050563r/production/images/large/jm050563rf00009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm050563r&amp;id=jm050563rf00009"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="jm050563rf00010" data-index="10" class="article__inlineFigure"><h2 class="fig-label"></h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm050563r/production/images/medium/jm050563rf00010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm050563r/production/images/large/jm050563rf00010.jpeg" id="_i28" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last"> Figure 10 A comparison between the hERG 1-dimensional profile and 3-dimensional methods. (a) The performance of the 1-dimensional profile is in black. The performance of the hERG pharmacophore model is in blue. The hERG pharmacophore model used is a reproduction of the model by Ekins and co-workers.<sup>46</sup> (b) The performance of the 1-dimensional profile is in black. The results with the 1-dimensional pharmacophore fingerprint are shown in blue and red. The maximum similarity between the training set compounds was used to create the red curve whereas the average similarity to the training set compounds was used to create the blue compounds.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm050563r/production/images/large/jm050563rf00010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm050563r&amp;id=jm050563rf00010"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div><div class="NLM_p">Perhaps the largest difference between the methods is in the amount of time required to search a large database. For the discussion below all quoted times are from an SGI R10000. Searches with the 1-dimensional profiles can rank approximately 900 molecules per <a name="jm004802206987" id="jm004802206987"></a> second meaning the search of the approximately 100 000 MDDR compounds takes less than 2 min. The conformation generation for the MDDR database took approximately 15 days of CPU time. Once the conformations had been generated the fingerprint generation took approximately 7 h. Once the conformations and corresponding pharmacophoric fingerprints were produced the pharmacophore similarities to the MDDR compounds were calculated in approximately 2 h. The generation of the pharmacophore fit value for the entire MDDR database took several weeks of CPU time, and as such this is not a practical means to search through large corporate databases. While speed is a secondary consideration it does dictate where the methods can be reasonably applied. Particularly, the speed and quality of searches with the 1-dimensional profiles makes them amenable to searching large databases of available or virtual compounds where 3-dimensional methods are not fast enough. </div><div class="NLM_p"><b>Comparison with Pairwise 1-Dimensional Similarity.</b> To demonstrate the value of the multiple alignment, we compare the performance of the multiple alignment with the 1-dimensional pairwise similarity. To do this, we compute the 1-dimesional similarity<sup>15</sup> between each of the molecules of the training set to each of the molecules of the test set and MDDR database. As with the 3-dimensional methods to rank the molecules of the test set and MDDR database, their maximum similarity and average similarity to the members of the training set were used. </div><div class="NLM_p">For the SRC case in which the chemotypes of the test set are well represented by the training set the maximum similarity outperforms both the average similarity and the 1-dimensional profile (see Figure 11<a data-tab="pane-pcw-Figures" href="#jm050563rf00011" id="" class=" internalNav"></a>a). This result is not all that surprising as molecules of the same chemotype would be expected to show very high similarity to other molecules of the same chemotype. Thus, the maximum similarity metric should be well suited to finding highly related additional inhibitors. The average similarity and the 1-dimensional profile perform very similarly. <div id="_i29"><figure id="jm050563rf00011" data-index="11" class="article__inlineFigure"><h2 class="fig-label"></h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm050563r/production/images/medium/jm050563rf00011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm050563r/production/images/large/jm050563rf00011.jpeg" id="_i30" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last"> Figure 11 A comparison of the 1-dimensional profiles to pairwise 1-dimensional similarity. In both cases the results with the 1-dimensional profile are shown in black, the results with the maximum similarity metric are shown in red, and the results with the average similarity metric are shown in blue. (a) The SRC example. (b) The hERG case.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm050563r/production/images/large/jm050563rf00011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm050563r&amp;id=jm050563rf00011"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div><div class="NLM_p"><a name="jm004802206988" id="jm004802206988"></a>The hERG test case better demonstrates the value of the multiple alignment profiles over the pairwise alignment. In this case the majority of the chemotypes in the test set are not represented in the training set. In this case the multiple alignment profile outperforms both the maximum and average similarity metrics. As expected the maximum similarity metric performs quite poorly. After finding approximately 20% of the known inhibitors, the maximum similarity metric trails off to random performance. This is expected, as most of the test set was not represented in the training set. Interestingly, the average similarity performs significantly better than random even though the maximum similarity does not. The 1-dimensional profile significantly outperforms both the average and maximum similarity. For example the average similarity metric finds 28% of the known inhibitors within the top 10% of the overall compounds compared to 45% for the 1-dimensional profile. </div><div class="NLM_p last">It is also noteworthy that for the pairwise similarity methods the time required to search a large database will increase linearly with the number of molecules in the training set. The methods used to perform the database searches with a 1-dimensional profile are independent of the number of molecules in the training set. Thus, for these cases where we had 10 molecules in the training set the database searches with the 1-dimensional profile were over 10 times faster than the pairwise searches. </div></div><div id="" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i31">Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_36620" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_36620" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Because it involves discrete and continuous variables with a complex landscape, multiple ligand alignment offers a challenging numerical optimization problem. Despite its heuristic nature the combination of a genetic algorithm and evolutionary programming proved to be a robust way to generate near optimal multiple ligand alignments. It is likely that for similar problems with mixed continuous and discrete variables this combination will prove fruitful. </div><div class="NLM_p">Despite the unconventional nature of the approach, the 1-dimensional representation of small molecules retains much of the information present in a small molecule structure. Much like a 3-dimensional pharmacophore model, a 1-dimensional profile can effectively isolate the common features of a set of molecules with the same biological activity. This in turn makes the profiles useful for searching databases of available small molecules for compounds with the same biological activity. In addition, the improvements in running times are accompanied by improvements in enrichment. In <a name="jm004802206989" id="jm004802206989"></a> both cases examined here, the 1D methods outperformed the 3D methods. This is significant considering that virtual screening methods such as these are best positioned prior to synthesis and applied to large virtual chemical libraries. The observed improvements in accuracy and the considerable decrease in time required for virtual screening make searching large virtual collections tractable and allows the profiles to be used multiple times while speeding up the turn-around time in the cyclic design of combinatorial libraries. </div><div class="NLM_p last">Future work in this area will include more rigorous atom similarity calculations. While the atom similarity matrices used for this work showed improvement over the identity matrices there is little reason to believe they are optimal. Improvements could include focusing more on properties of the atoms such as hydrogen bonding capacity, aromaticity, etc., rather than explicit atom types. Furthermore, the alignment procedures could be tailored to include both molecules with the biological activity of interest and molecules known to be devoid of the biological activity of interest. Incorporating the negative information would further focus the profiles to the key features of the molecules and might help to eliminate many of the false positives found in the virtual screening experiments. </div></div><div class="NLM_back"><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_04585" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_04585" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><ul></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Norman Wang</span> - <span class="hlFld-Affiliation affiliation">Pharmacopeia, Inc., CN5350, Princeton, New Jersey 08543-5350 </span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Robert K. DeLisle</span> - <span class="hlFld-Affiliation affiliation">Pharmacopeia, Inc., CN5350, Princeton, New Jersey 08543-5350 </span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">David J. Diller</span> - <span class="hlFld-Affiliation affiliation">Pharmacopeia, Inc., CN5350, Princeton, New Jersey 08543-5350 </span></div></li></ul></li><li></li><li></li><li></li></ul></div><div class="testing" data-doi="10.1021/jm050563r" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="refs-header-label"></div><p class="references-count">
            This article references 50 other publications.
        </p><ol id="references" class="useLabel"><li id="jm050563rb00001"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm050563rb00001'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="jm050563rb00001_1"><span><span class="NLM_string-ref">Martin, Y. C.; Kofron, J. L.; Traphagen, L. M. Do structurally similar molecules have similar biological activity? <i>J. Med. Chem.</i><b>2002</b>,<i> 45</i>, 4350−4358.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2002&pages=4358&author=Y.+C.+Martin&author=J.+L.+Kofron&author=L.+M+Traphagen&title=Do+structurally+similar+molecules+have+similar+biological+activity"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm050563rb00001_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMartin%26aufirst%3DY.%2BC.%26atitle%3DDo%2520structurally%2520similar%2520molecules%2520have%2520similar%2520biological%2520activity%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2002%26volume%3D45%26spage%3D4358" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm050563rb00002"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm050563rb00002'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="jm050563rb00002_1"><span><span class="NLM_string-ref">Durant, J. L.; Leland, B. A.; Henry, D. R.; Nourse, J. G. Reoptimization of MDL keys for use in drug discovery. <i>  J. Chem. </i><i>Inf.</i><i> Comput. Sci</i>. <b>2002</b><i>, 42</i>, 1273−1280.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2002&pages=1280&author=J.+L.+Durant&author=B.+A.+Leland&author=D.+R.+Henry&author=J.+G+Nourse&title=Reoptimization+of+MDL+keys+for+use+in+drug+discovery"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm050563rb00002_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DDurant%26aufirst%3DJ.%2BL.%26atitle%3DReoptimization%2520of%2520MDL%2520keys%2520for%2520use%2520in%2520drug%2520discovery%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Comput.%2520Sci.%26date%3D2002%26volume%3D42%26spage%3D1280" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm050563rb00003"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm050563rb00003'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="jm050563rb00003_1"><span><span class="NLM_string-ref">Makara, G. M. Measuring molecular similarity and diversity:  total pharmacophore diversity. <i>  J. Med. Chem. </i><b>2001</b><i>, 44</i>, 3563−3571.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2001&pages=3571&author=G.+M+Makara&title=Measuring+molecular+similarity+and+diversity%3A%E2%80%89+total+pharmacophore+diversity"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm050563rb00003_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMakara%26aufirst%3DG.%2BM%26atitle%3DMeasuring%2520molecular%2520similarity%2520and%2520diversity%253A%25E2%2580%2589%2520total%2520pharmacophore%2520diversity%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2001%26volume%3D44%26spage%3D3571" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm050563rb00004"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm050563rb00004'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="jm050563rb00004_1"><span><span class="NLM_string-ref">Mason, J. S.; Good, A. C.; Martin, E. J. 3-D pharmacophores in drug discovery. <i>  Curr. Pharm. Des. </i><b>2001</b><i>, 7</i>, 567−597.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2001&pages=597&author=J.+S.+Mason&author=A.+C.+Good&author=E.+J.+Martin&title=pharmacophores+in+drug+discovery"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm050563rb00004_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMason%26aufirst%3DJ.%2BS.%26atitle%3Dpharmacophores%2520in%2520drug%2520discovery%26jtitle%3DCurr.%2520Pharm.%2520Des.%26date%3D2001%26volume%3D7%26spage%3D597" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm050563rb00005"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm050563rb00005'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="jm050563rb00005_1"><span><span class="NLM_string-ref">McGregor, M. J.; Muskal, S. M. Pharmacophore fingerprinting. 1. Application to QSAR and focused library design. <i>  J. Chem. </i><i>Inf.</i><i> Comput. Sci.</i><b>1999</b><i>, 39</i>, 569−574.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=1999&pages=574&author=M.+J.+McGregor&author=S.+M+Muskal&title=Pharmacophore+fingerprinting.+1.+Application+to+QSAR+and+focused+library+design"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm050563rb00005_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMcGregor%26aufirst%3DM.%2BJ.%26atitle%3DPharmacophore%2520fingerprinting.%25201.%2520Application%2520to%2520QSAR%2520and%2520focused%2520library%2520design%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Comput.%2520Sci.%26date%3D1999%26volume%3D39%26spage%3D574" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm050563rb00006"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm050563rb00006'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="jm050563rb00006_1"><span><span class="NLM_string-ref">McGregor, M. J.; Muskal, S. M. Pharmacophore fingerprinting. 2. Application to primary library design. <i>  J. Chem. </i><i>Inf.</i><i> Comput. Sci</i>. <b>2000</b>,<i> 40</i>, 117−125.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2000&pages=125&author=M.+J.+McGregor&author=S.+M+Muskal&title=Pharmacophore+fingerprinting.+2.+Application+to+primary+library+design"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm050563rb00006_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMcGregor%26aufirst%3DM.%2BJ.%26atitle%3DPharmacophore%2520fingerprinting.%25202.%2520Application%2520to%2520primary%2520library%2520design%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Comput.%2520Sci.%26date%3D2000%26volume%3D40%26spage%3D125" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm050563rb00007"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm050563rb00007'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="jm050563rb00007_1"><span><span class="NLM_string-ref">Rarey, M.; Stahl, M. Similarity searching in large combinatorial chemistry spaces. <i>  J. Comput. Aided Mol. Des. </i><b>2001</b>,<i> 15</i>, 497−520.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2001&pages=520&author=M.+Rarey&author=M+Stahl&title=Similarity+searching+in+large+combinatorial+chemistry+spaces"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm050563rb00007_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DRarey%26aufirst%3DM.%26atitle%3DSimilarity%2520searching%2520in%2520large%2520combinatorial%2520chemistry%2520spaces%26jtitle%3DJ.%2520Comput.%2520Aided%2520Mol.%2520Des.%26date%3D2001%26volume%3D15%26spage%3D520" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm050563rb00008"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm050563rb00008'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="jm050563rb00008_1"><span><span class="NLM_string-ref">Lemmen, C.; Lengauer, T.; Klebe, G. FLEXS:  a method for fast flexible ligand superposition. <i>  J. Med. Chem. </i><b>1998</b>,<i> 41</i>, 4502−4520.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=1998&pages=4520&author=C.+Lemmen&author=T.+Lengauer&author=G.+Klebe&title=a+method+for+fast+flexible+ligand+superposition"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm050563rb00008_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DLemmen%26aufirst%3DC.%26atitle%3Da%2520method%2520for%2520fast%2520flexible%2520ligand%2520superposition%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1998%26volume%3D41%26spage%3D4520" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm050563rb00009"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm050563rb00009'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="jm050563rb00009_1"><span><span class="NLM_string-ref">Lemmen, C.; Lengauer, T. Computational methods for the structural alignment of molecules. <i>  J. Comput. Aided Mol. Des. </i><b>2000</b><i>, 14</i>, 215−232.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2000&pages=232&author=C.+Lemmen&author=T+Lengauer&title=Computational+methods+for+the+structural+alignment+of+molecules"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm050563rb00009_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DLemmen%26aufirst%3DC.%26atitle%3DComputational%2520methods%2520for%2520the%2520structural%2520alignment%2520of%2520molecules%26jtitle%3DJ.%2520Comput.%2520Aided%2520Mol.%2520Des.%26date%3D2000%26volume%3D14%26spage%3D232" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm050563rb00010"><div class="reference"><div class="NLM_citation" id="jm050563rb00010_1"><span><span class="NLM_string-ref">Barrett, A. J.; Rawlings, N. D. Families and clans of serine peptidases. <i>  Arch. </i><i>Biochem. Biophys. </i><b>1995</b><i>, 318</i>, 247−250.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=318&publication_year=1995&pages=250&author=A.+J.+Barrett&author=N.+D+Rawlings&title=Families+and+clans+of+serine+peptidases"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm050563rb00010_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBarrett%26aufirst%3DA.%2BJ.%26atitle%3DFamilies%2520and%2520clans%2520of%2520serine%2520peptidases%26jtitle%3DArch.%2520Biochem.%2520Biophys.%26date%3D1995%26volume%3D318%26spage%3D250" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm050563rb00011"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm050563rb00011'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="jm050563rb00011_1"><span><span class="NLM_string-ref">Fischer, D.; Wolfson, H.; Lin, S. L.; Nussinov, R. Three-dimensional, sequence order-independent structural comparison of a serine protease against the crystallographic database reveals active site similarities:  potential implications to evolution and to protein folding. <i>  Protein Sci. </i><b>1994</b>,<i> 3</i>, 769−778.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=1994&pages=778&author=D.+Fischer&author=H.+Wolfson&author=S.+L.+Lin&author=R+Nussinov&title=Three-dimensional%2C+sequence+order-independent+structural+comparison+of+a+serine+protease+against+the+crystallographic+database+reveals+active+site+similarities%3A%E2%80%89+potential+implications+to+evolution+and+to+protein+folding"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm050563rb00011_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DFischer%26aufirst%3DD.%26atitle%3DThree-dimensional%252C%2520sequence%2520order-independent%2520structural%2520comparison%2520of%2520a%2520serine%2520protease%2520against%2520the%2520crystallographic%2520database%2520reveals%2520active%2520site%2520similarities%253A%25E2%2580%2589%2520potential%2520implications%2520to%2520evolution%2520and%2520to%2520protein%2520folding%26jtitle%3DProtein%2520Sci.%26date%3D1994%26volume%3D3%26spage%3D778" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm050563rb00012"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm050563rb00012'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="jm050563rb00012_1"><span><span class="NLM_string-ref">Krem, M. M.; Di Cera, E. Molecular markers of serine protease evolution. <i>  EMBO J. </i><b>2001</b>,<i> 20</i>, 3036−3045.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2001&pages=3045&author=M.+M.+Krem&author=E+Di+Cera&title=Molecular+markers+of+serine+protease+evolution"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm050563rb00012_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DKrem%26aufirst%3DM.%2BM.%26atitle%3DMolecular%2520markers%2520of%2520serine%2520protease%2520evolution%26jtitle%3DEMBO%2520J.%26date%3D2001%26volume%3D20%26spage%3D3045" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm050563rb00013"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm050563rb00013'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="jm050563rb00013_1"><span><span class="NLM_string-ref">Rawlings, N. D.; Barrett, A. J. Families of serine peptidases. <i>  Methods Enzymol. </i><b>1994</b>,<i> 244</i>, 19−61.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=244&publication_year=1994&pages=61&author=N.+D.+Rawlings&author=A.+J+Barrett&title=Families+of+serine+peptidases"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm050563rb00013_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DRawlings%26aufirst%3DN.%2BD.%26atitle%3DFamilies%2520of%2520serine%2520peptidases%26jtitle%3DMethods%2520Enzymol.%26date%3D1994%26volume%3D244%26spage%3D61" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm050563rb00014"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm050563rb00014'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="jm050563rb00014_1"><span><span class="NLM_string-ref">Guner, O. F. History and evolution of the pharmacophore concept in computer-aided drug design. <i>  Curr. Top. Med. Chem.</i><b>2002</b><i>, 2</i>, 1321−1332.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2002&pages=1332&author=O.+F+Guner&title=History+and+evolution+of+the+pharmacophore+concept+in+computer-aided+drug+design"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm050563rb00014_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DGuner%26aufirst%3DO.%2BF%26atitle%3DHistory%2520and%2520evolution%2520of%2520the%2520pharmacophore%2520concept%2520in%2520computer-aided%2520drug%2520design%26jtitle%3DCurr.%2520Top.%2520Med.%2520Chem.%26date%3D2002%26volume%3D2%26spage%3D1332" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm050563rb00015"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm050563rb00015'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="jm050563rb00015_1"><span><span class="NLM_string-ref">Dixon, S. L.; Merz, K. M., Jr. One-dimensional molecular representations and similarity calculations:  methodology and validation. <i>  J. Med. Chem. </i><b>2001</b><i>, </i><i>44</i>, 3795−3809.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2001&pages=3809&author=S.+L.+Dixon&author=K.+M.+Merz&author=+Jr&title=One-dimensional+molecular+representations+and+similarity+calculations%3A%E2%80%89+methodology+and+validation"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm050563rb00015_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DDixon%26aufirst%3DS.%2BL.%26atitle%3DOne-dimensional%2520molecular%2520representations%2520and%2520similarity%2520calculations%253A%25E2%2580%2589%2520methodology%2520and%2520validation%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2001%26volume%3D44%26spage%3D3809" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm050563rb00016"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm050563rb00016'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="jm050563rb00016_1"><span><span class="NLM_string-ref">Borg, I.; Groenen, P. <i>  Modern Multidimensional Scaling:  Theory and </i><i>Applications</i>; Springer:  New York, 1997.</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=jm050563rb00016/jm050563rb00016_1&amp;dbid=16&amp;doi=10.1021%2Fjm050563r&amp;key=10.1007%2F978-1-4757-2711-1" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=1997&author=I.+Borg&author=P.+Groenen&title=Modern+Multidimensional+Scaling%3A%E2%80%89+Theory+and+Applications"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm050563rb00016_1&amp;dbid=16384&amp;doi=10.1007%2F978-1-4757-2711-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1007%252F978-1-4757-2711-1%26sid%3Dliteratum%253Aachs%26aulast%3DBorg%26aufirst%3DI.%26btitle%3DModern%2520Multidimensional%2520Scaling%253A%25E2%2580%2589%2520Theory%2520and%2520Applications%26pub%3DSpringer%26date%3D1997" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm050563rb00017"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm050563rb00017'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="jm050563rb00017_1"><span><span class="NLM_string-ref">Mohammad, S.; Zhou, Z.; Gong, Q.; January, C. T. Blockage of the HERG human cardiac K+ channel by the gastrointestinalprokinetic agent cisapride. <i>  Am. J. Physiol. </i><b>1997</b><i>, 273</i>, 2534−2538.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=273&publication_year=1997&pages=2538&author=S.+Mohammad&author=Z.+Zhou&author=Q.+Gong&author=C.+T+January&title=Blockage+of+the+HERG+human+cardiac+K%2B+channel+by+the+gastrointestinalprokinetic+agent+cisapride"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm050563rb00017_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMohammad%26aufirst%3DS.%26atitle%3DBlockage%2520of%2520the%2520HERG%2520human%2520cardiac%2520K%252B%2520channel%2520by%2520the%2520gastrointestinalprokinetic%2520agent%2520cisapride%26jtitle%3DAm.%2520J.%2520Physiol.%26date%3D1997%26volume%3D273%26spage%3D2538" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm050563rb00018"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm050563rb00018'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="jm050563rb00018_1"><span><span class="NLM_string-ref">Kongsamut, S.; Kang, J.; Chen, X.; Roehr, J.; Rampe, D. A comparison of the receptor binding and HERG channel affinities for a series of antipsychotic drugs. <i>  Eur. J. Pharmacol. </i><b>2002</b>,<i> 450</i>, 37−41.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=450&publication_year=2002&pages=41&author=S.+Kongsamut&author=J.+Kang&author=X.+Chen&author=J.+Roehr&author=D+Rampe&title=A+comparison+of+the+receptor+binding+and+HERG+channel+affinities+for+a+series+of+antipsychotic+drugs"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm050563rb00018_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DKongsamut%26aufirst%3DS.%26atitle%3DA%2520comparison%2520of%2520the%2520receptor%2520binding%2520and%2520HERG%2520channel%2520affinities%2520for%2520a%2520series%2520of%2520antipsychotic%2520drugs%26jtitle%3DEur.%2520J.%2520Pharmacol.%26date%3D2002%26volume%3D450%26spage%3D41" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm050563rb00019"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm050563rb00019'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="jm050563rb00019_1"><span><span class="NLM_string-ref">Fogel, L. J.; Owens, A. J.; Walsh, M. J. <i>  Artificial intelligence through simulated </i><i>evolution</i>; John Wiley and Sons:  New York, 1966.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=1966&author=L.+J.+Fogel&author=A.+J.+Owens&author=M.+J.+Walsh&title=Artificial+intelligence+through+simulated+evolution"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm050563rb00019_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DFogel%26aufirst%3DL.%2BJ.%26btitle%3DArtificial%2520intelligence%2520through%2520simulated%2520evolution%26pub%3DJohn%2520Wiley%2520and%2520Sons%26date%3D1966" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm050563rb00020"><div class="reference"><div class="NLM_citation" id="jm050563rb00020_1"><span><span class="NLM_string-ref">Holland, J. H. <i>  Adaptation is natural and artificial systems</i>; Univesity of Michigan Press:  Ann Arbor, 1975.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=1975&author=J.+H.+Holland&title=Adaptation+is+natural+and+artificial+systems"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm050563rb00020_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DHolland%26aufirst%3DJ.%2BH.%26btitle%3DAdaptation%2520is%2520natural%2520and%2520artificial%2520systems%26pub%3DUnivesity%2520of%2520Michigan%2520Press%26date%3D1975" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm050563rb00021"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm050563rb00021'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="jm050563rb00021_1"><span><span class="NLM_string-ref">Baeck, T.; Fogel, D. B.; Michalewicz, Z.; Back, T.; Fogel, D. R. R. <i>  Evolutionary Computation 1:  Basic Algorithms and </i>Operators; Institute of Physics:  Bristol, UK, 2000.</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=jm050563rb00021/jm050563rb00021_1&amp;dbid=16&amp;doi=10.1021%2Fjm050563r&amp;key=10.1887%2F0750306653" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2000&author=T.+Baeck&author=D.+B.+Fogel&author=Z.+Michalewicz&author=T.+Back&author=D.+R.+R.+Fogel&title=Evolutionary+Computation+1%3A%E2%80%89+Basic+Algorithms+and+Operators"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm050563rb00021_1&amp;dbid=16384&amp;doi=10.1887%2F0750306653&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1887%252F0750306653%26sid%3Dliteratum%253Aachs%26aulast%3DBaeck%26aufirst%3DT.%26btitle%3DEvolutionary%2520Computation%25201%253A%25E2%2580%2589%2520Basic%2520Algorithms%2520and%2520Operators%26pub%3DInstitute%2520of%2520Physics%26date%3D2000" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm050563rb00022"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm050563rb00022'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="jm050563rb00022_1"><span><span class="NLM_string-ref">Baeck, T.; Fogel, D. B.; Michalewicz, Z. <i>  Evolutionary Computation 2:  </i><i>Advanced Algorithms and Operators</i>; Institute of Physics:  Bristol, UK, 2000.</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=jm050563rb00022/jm050563rb00022_1&amp;dbid=16&amp;doi=10.1021%2Fjm050563r&amp;key=10.1887%2F0750306653" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2000&author=T.+Baeck&author=D.+B.+Fogel&author=Z.+Michalewicz&title=Evolutionary+Computation+2%3A%E2%80%89+Advanced+Algorithms+and+Operators"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm050563rb00022_1&amp;dbid=16384&amp;doi=10.1887%2F0750306653&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1887%252F0750306653%26sid%3Dliteratum%253Aachs%26aulast%3DBaeck%26aufirst%3DT.%26btitle%3DEvolutionary%2520Computation%25202%253A%25E2%2580%2589%2520Advanced%2520Algorithms%2520and%2520Operators%26pub%3DInstitute%2520of%2520Physics%26date%3D2000" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm050563rb00023"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm050563rb00023'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="jm050563rb00023_1"><span><span class="NLM_string-ref">Schwartz, R. M.; Dayhoff, M. O. Matrices for detecting distant relationships. <i>  Atlas of Protein Sequence and Structure</i>; National Biomedical Research Foundation. Washington, D.C., 1978.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=1978&author=R.+M.+Schwartz&author=M.+O.+Dayhoff&title=Matrices+for+detecting+distant+relationships.+Atlas+of+Protein+Sequence+and+Structure"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm050563rb00023_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DSchwartz%26aufirst%3DR.%2BM.%26btitle%3DMatrices%2520for%2520detecting%2520distant%2520relationships.%2520Atlas%2520of%2520Protein%2520Sequence%2520and%2520Structure%26pub%3DD.C%26date%3D1978" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm050563rb00024"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm050563rb00024'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="jm050563rb00024_1"><span><span class="NLM_string-ref">Henikoff, S.; Henikoff, J. G. Amino acid substitution matrices from protein blocks. <i>  Proc. Natl. Acad. Sci. U.S.A.</i><b>1992</b><i>, 89</i>, 10915−10919.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=89&publication_year=1992&pages=10919&author=S.+Henikoff&author=J.+G+Henikoff&title=Amino+acid+substitution+matrices+from+protein+blocks"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm050563rb00024_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DHenikoff%26aufirst%3DS.%26atitle%3DAmino%2520acid%2520substitution%2520matrices%2520from%2520protein%2520blocks%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D1992%26volume%3D89%26spage%3D10919" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm050563rb00025"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm050563rb00025'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="jm050563rb00025_1"><span><span class="NLM_string-ref">Gonnet, G. H.; Cohen, M. A.; Benner, S. A. Exhaustive matching of the entire protein sequence database. <i>  Science </i><b>1992</b>, 256, 1443−1445.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=256&publication_year=1992&pages=1445&author=G.+H.+Gonnet&author=M.+A.+Cohen&author=S.+A+Benner&title=Exhaustive+matching+of+the+entire+protein+sequence+database"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm050563rb00025_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DGonnet%26aufirst%3DG.%2BH.%26atitle%3DExhaustive%2520matching%2520of%2520the%2520entire%2520protein%2520sequence%2520database%26jtitle%3DScience%26date%3D1992%26volume%3D256%26spage%3D1445" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm050563rb00026"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm050563rb00026'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="jm050563rb00026_1"><span><span class="NLM_string-ref">Jones, D. T.; Taylor, W. R.; Thornton, J. M. The rapid generation of mutation data matrices from protein sequences. <i>  Comput. Appl. Biosci. </i><b>1992</b>,<i> 8</i>, 275−282.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=1992&pages=282&author=D.+T.+Jones&author=W.+R.+Taylor&author=J.+M+Thornton&title=The+rapid+generation+of+mutation+data+matrices+from+protein+sequences"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm050563rb00026_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DJones%26aufirst%3DD.%2BT.%26atitle%3DThe%2520rapid%2520generation%2520of%2520mutation%2520data%2520matrices%2520from%2520protein%2520sequences%26jtitle%3DComput.%2520Appl.%2520Biosci.%26date%3D1992%26volume%3D8%26spage%3D282" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm050563rb00027"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm050563rb00027'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="jm050563rb00027_1"><span><span class="NLM_string-ref">MDDR <i>MDL Drug Data Report</i>; MDL:  San Leandro, CA.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+MDDR+MDL+Drug+Data+Report%3B+MDL%3A%E2%80%89+San+Leandro%2C+CA.+MDDR+MDL+Drug+Data+Report%3B+MDL%3A%E2%80%89+San+Leandro%2C+CA."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm050563rb00027_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dfootnote%26sid%3Dliteratum%253Aachs" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm050563rb00028"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm050563rb00028'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="jm050563rb00028_1"><span><span class="NLM_string-ref">Press:  W. H.; Flannery, B. P.; Teukolsky, S. A.; Vetterling, W. T. <i>  Numerical </i><i>recipes in C:  The art of scientific computing</i>, 2nd ed.; Cambridge University Press:  New York, 1993; 994.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=1993&author=W.+H.+Press&author=B.+P.+Flannery&author=S.+A.+Teukolsky&author=W.+T.+Vetterling&title=Numerical+recipes+in+C%3A%E2%80%89+The+art+of+scientific+computing"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm050563rb00028_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DPress%26aufirst%3DW.%2BH.%26btitle%3DNumerical%2520recipes%2520in%2520C%253A%25E2%2580%2589%2520The%2520art%2520of%2520scientific%2520computing%26pub%3DCambridge%2520University%2520Press%26date%3D1993" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm050563rb00029"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm050563rb00029'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="jm050563rb00029_1"><span><span class="NLM_string-ref">Frame, M. C. Src in cancer:  deregulation and consequences for cell behaviour. <i>  Biochim. Biophys. Acta </i><b>2002</b>,<i> 1602</i>, 114−130.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1602&publication_year=2002&pages=130&author=M.+C+Frame&title=Src+in+cancer%3A%E2%80%89+deregulation+and+consequences+for+cell+behaviour"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm050563rb00029_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DFrame%26aufirst%3DM.%2BC%26atitle%3DSrc%2520in%2520cancer%253A%25E2%2580%2589%2520deregulation%2520and%2520consequences%2520for%2520cell%2520behaviour%26jtitle%3DBiochim.%2520Biophys.%2520Acta%26date%3D2002%26volume%3D1602%26spage%3D130" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm050563rb00030"><div class="reference"><div class="NLM_citation" id="jm050563rb00030_1"><span><span class="NLM_string-ref">De Ponti, F.; Poluzzi, E.; Cavalli, A.; Recanatini, M.; Montanaro, N. Safety of nonantiarrhythmic drugs that prolong the QT interval or induce torsade de pointes:  an overview. <i>  Drug Safety </i><b>2002</b>,<i> 25</i>, 263−286.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2002&pages=286&author=F.+De+Ponti&author=E.+Poluzzi&author=A.+Cavalli&author=M.+Recanatini&author=N+Montanaro&title=Safety+of+nonantiarrhythmic+drugs+that+prolong+the+QT+interval+or+induce+torsade+de+pointes%3A%E2%80%89+an+overview"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm050563rb00030_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DDe%2BPonti%26aufirst%3DF.%26atitle%3DSafety%2520of%2520nonantiarrhythmic%2520drugs%2520that%2520prolong%2520the%2520QT%2520interval%2520or%2520induce%2520torsade%2520de%2520pointes%253A%25E2%2580%2589%2520an%2520overview%26jtitle%3DDrug%2520Safety%26date%3D2002%26volume%3D25%26spage%3D286" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm050563rb00031"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm050563rb00031'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="jm050563rb00031_1"><span><span class="NLM_string-ref">Roy, M.; Dumaine, R.; Brown, A. M. HERG, a primary human ventricular target of the nonsedating antihistamine terfenadine. <i>  Circulation </i><b>1996</b>,<i> 94</i>, 817−823.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=94&publication_year=1996&pages=823&author=M.+Roy&author=R.+Dumaine&author=A.+M+Brown&title=HERG%2C+a+primary+human+ventricular+target+of+the+nonsedating+antihistamine+terfenadine"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm050563rb00031_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DRoy%26aufirst%3DM.%26atitle%3DHERG%252C%2520a%2520primary%2520human%2520ventricular%2520target%2520of%2520the%2520nonsedating%2520antihistamine%2520terfenadine%26jtitle%3DCirculation%26date%3D1996%26volume%3D94%26spage%3D823" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm050563rb00032"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm050563rb00032'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="jm050563rb00032_1"><span><span class="NLM_string-ref">Wang, Y. D.; Miller, K.; Boschelli, D. H.; Ye, F.; Wu, B. et al. Inhibitors of src tyrosine kinase:  the preparation and structure−activity relationship of 4-anilino-3-cyanoquinolines and 4-anilinoquinazolines. <i>  Bioorg. Med. </i><i>Chem. Lett. </i><b>2000</b>,<i> 10</i>, 2477−2480.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2000&pages=2480&author=Y.+D.+Wang&author=K.+Miller&author=D.+H.+Boschelli&author=F.+Ye&author=B+Wu&title=et+al.+Inhibitors+of+src+tyrosine+kinase%3A%E2%80%89+the+preparation+and+structure%E2%88%92activity+relationship+of+4-anilino-3-cyanoquinolines+and+4-anilinoquinazolines"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm050563rb00032_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DY.%2BD.%26atitle%3Det%2520al.%2520Inhibitors%2520of%2520src%2520tyrosine%2520kinase%253A%25E2%2580%2589%2520the%2520preparation%2520and%2520structure%25E2%2588%2592activity%2520relationship%2520of%25204-anilino-3-cyanoquinolines%2520and%25204-anilinoquinazolines%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2000%26volume%3D10%26spage%3D2480" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm050563rb00033"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm050563rb00033'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="jm050563rb00033_1"><span><span class="NLM_string-ref">Trumpp-Kallmeyer, S.; Rubin, J. R.; Humblet, C.; Hamby, J. M.; Showalter, H. D. H. Development of a binding model to protein tyrosine kinases for substituted pyrido[2,3-d]pyrimidine inhibitors. <i>  J. Med. Chem. </i><b>1998</b>,<i> 41</i>, 1752−1763.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=1998&pages=1763&author=S.+Trumpp-Kallmeyer&author=J.+R.+Rubin&author=C.+Humblet&author=J.+M.+Hamby&author=H.+D.+H+Showalter&title=Development+of+a+binding+model+to+protein+tyrosine+kinases+for+substituted+pyrido%5B2%2C3-d%5Dpyrimidine+inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm050563rb00033_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DTrumpp-Kallmeyer%26aufirst%3DS.%26atitle%3DDevelopment%2520of%2520a%2520binding%2520model%2520to%2520protein%2520tyrosine%2520kinases%2520for%2520substituted%2520pyrido%255B2%252C3-d%255Dpyrimidine%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1998%26volume%3D41%26spage%3D1763" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm050563rb00034"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm050563rb00034'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="jm050563rb00034_1"><span><span class="NLM_string-ref">Traxler, P.; Green, J.; Mett, H.; Sequin, U.; P., F. Use of a pharmacophore model for the design of EGFR tyrosine kinase inhibitors:  isoflavones and 3-phenyl-4(1H)-quinolones. <i>  J. Med. Chem. </i><b>1999</b>,<i> 42</i>, 1018−1026.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=1999&pages=1026&author=P.+Traxler&author=J.+Green&author=H.+Mett&author=U.+Sequin&title=Use+of+a+pharmacophore+model+for+the+design+of+EGFR+tyrosine+kinase+inhibitors%3A%E2%80%89+isoflavones+and+3-phenyl-4%281H%29-quinolones"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm050563rb00034_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DTraxler%26aufirst%3DP.%26atitle%3DUse%2520of%2520a%2520pharmacophore%2520model%2520for%2520the%2520design%2520of%2520EGFR%2520tyrosine%2520kinase%2520inhibitors%253A%25E2%2580%2589%2520isoflavones%2520and%25203-phenyl-4%25281H%2529-quinolones%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1999%26volume%3D42%26spage%3D1026" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm050563rb00035"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm050563rb00035'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="jm050563rb00035_1"><span><span class="NLM_string-ref">Traxler, P.; Bold, G.; Frei, J.; Lang, M.; Lydon, N. et al. Use of a Pharmacophore Model for the Design of EGF−R Tyrosine Kinase Inhibitors:  4-(Phenylamino)pyrazolo[3,4-d]pyrimidines. <i>  J. Med. Chem. </i><b>1997</b>,<i> 40</i>, 3601−3616.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=1997&pages=3616&author=P.+Traxler&author=G.+Bold&author=J.+Frei&author=M.+Lang&author=N+Lydon&title=et+al.+Use+of+a+Pharmacophore+Model+for+the+Design+of+EGF%E2%88%92R+Tyrosine+Kinase+Inhibitors%3A%E2%80%89+4-%28Phenylamino%29pyrazolo%5B3%2C4-d%5Dpyrimidines"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm050563rb00035_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DTraxler%26aufirst%3DP.%26atitle%3Det%2520al.%2520Use%2520of%2520a%2520Pharmacophore%2520Model%2520for%2520the%2520Design%2520of%2520EGF%25E2%2588%2592R%2520Tyrosine%2520Kinase%2520Inhibitors%253A%25E2%2580%2589%25204-%2528Phenylamino%2529pyrazolo%255B3%252C4-d%255Dpyrimidines%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1997%26volume%3D40%26spage%3D3616" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm050563rb00036"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm050563rb00036'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="jm050563rb00036_1"><span><span class="NLM_string-ref">Klutchko, S. R.; Hamby, J. M.; Boschelli, D. H.; Wu, Z.; Kraker, A. J. et al. 2-Substituted Aminopyrido[2,3-d]pyrimidin-7(8H)-ones. Structure−Activity Relationships Against Selected Tyrosine Kinases and in Vitro and in Vivo Anticancer Activity. <i>  J. Med. Chem. </i><b>1998</b>,<i> 41</i>, 3276−3292.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=1998&pages=3292&author=S.+R.+Klutchko&author=J.+M.+Hamby&author=D.+H.+Boschelli&author=Z.+Wu&author=A.+J+Kraker&title=et+al.+2-Substituted+Aminopyrido%5B2%2C3-d%5Dpyrimidin-7%288H%29-ones.+Structure%E2%88%92Activity+Relationships+Against+Selected+Tyrosine+Kinases+and+in+Vitro+and+in+Vivo+Anticancer+Activity"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm050563rb00036_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DKlutchko%26aufirst%3DS.%2BR.%26atitle%3Det%2520al.%25202-Substituted%2520Aminopyrido%255B2%252C3-d%255Dpyrimidin-7%25288H%2529-ones.%2520Structure%25E2%2588%2592Activity%2520Relationships%2520Against%2520Selected%2520Tyrosine%2520Kinases%2520and%2520in%2520Vitro%2520and%2520in%2520Vivo%2520Anticancer%2520Activity%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1998%26volume%3D41%26spage%3D3292" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm050563rb00037"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm050563rb00037'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="jm050563rb00037_1"><span><span class="NLM_string-ref">Hamby, J. M.; Connolly, C. J.; Schroeder, M. C.; Winters, R. T.; Showalter, H. D. et al. Structure−activity relationships for a novel series of pyrido[2,3-d]pyrimidine tyrosine kinase inhibitors. <i>  J. Med. Chem. </i><b>1997</b>, <i>40</i>, 2296−2303.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=1997&pages=2303&author=J.+M.+Hamby&author=C.+J.+Connolly&author=M.+C.+Schroeder&author=R.+T.+Winters&author=H.+D+Showalter&title=et+al.+Structure%E2%88%92activity+relationships+for+a+novel+series+of+pyrido%5B2%2C3-d%5Dpyrimidine+tyrosine+kinase+inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm050563rb00037_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DHamby%26aufirst%3DJ.%2BM.%26atitle%3Det%2520al.%2520Structure%25E2%2588%2592activity%2520relationships%2520for%2520a%2520novel%2520series%2520of%2520pyrido%255B2%252C3-d%255Dpyrimidine%2520tyrosine%2520kinase%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1997%26volume%3D40%26spage%3D2303" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm050563rb00038"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm050563rb00038'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="jm050563rb00038_1"><span><span class="NLM_string-ref">Boschelli, D. H.; Wang, D. Y.; Ye, F.; Yamashita, A.; Zhang, N. et al. Inhibition of Src kinase activity by 4-anilino-7-thienyl-3-quinolinecarbonitriles. <i>  Bioorg. Med. Chem. Lett. </i><b>2002</b>,<i> 12</i>, 2011−2014.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2002&pages=2014&author=D.+H.+Boschelli&author=D.+Y.+Wang&author=F.+Ye&author=A.+Yamashita&author=N+Zhang&title=et+al.+Inhibition+of+Src+kinase+activity+by+4-anilino-7-thienyl-3-quinolinecarbonitriles"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm050563rb00038_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBoschelli%26aufirst%3DD.%2BH.%26atitle%3Det%2520al.%2520Inhibition%2520of%2520Src%2520kinase%2520activity%2520by%25204-anilino-7-thienyl-3-quinolinecarbonitriles%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2002%26volume%3D12%26spage%3D2014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm050563rb00039"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm050563rb00039'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="jm050563rb00039_1"><span><span class="NLM_string-ref">Boschelli, D. H.; Wu, Z.; Klutchko, S. R.; Showalter, H. D.; Hamby, J. M. et al. Synthesis and tyrosine kinase inhibitory activity of a series of 2-amino-8H-pyrido[2,3-d]pyrimidines:  identification of potent, selective platelet-derived growth factor receptor tyrosine kinase inhibitors. <i>  J. Med. </i><i>Chem. </i><b>1998</b>,<i> 41</i>, 4365−4377.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=1998&pages=4377&author=D.+H.+Boschelli&author=Z.+Wu&author=S.+R.+Klutchko&author=H.+D.+Showalter&author=J.+M+Hamby&title=et+al.+Synthesis+and+tyrosine+kinase+inhibitory+activity+of+a+series+of+2-amino-8H-pyrido%5B2%2C3-d%5Dpyrimidines%3A%E2%80%89+identification+of+potent%2C+selective+platelet-derived+growth+factor+receptor+tyrosine+kinase+inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm050563rb00039_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBoschelli%26aufirst%3DD.%2BH.%26atitle%3Det%2520al.%2520Synthesis%2520and%2520tyrosine%2520kinase%2520inhibitory%2520activity%2520of%2520a%2520series%2520of%25202-amino-8H-pyrido%255B2%252C3-d%255Dpyrimidines%253A%25E2%2580%2589%2520identification%2520of%2520potent%252C%2520selective%2520platelet-derived%2520growth%2520factor%2520receptor%2520tyrosine%2520kinase%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1998%26volume%3D41%26spage%3D4377" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm050563rb00040"><div class="reference"><div class="NLM_citation" id="jm050563rb00040_1"><span><span class="NLM_string-ref">Thompson, A. M.; Connolly, C. J.; Hamby, J. M.; Boushelle, S.; Hartl, B. G. et al. 3-(3,5-Dimethoxyphenyl)-1,6-naphthyridine-2,7-diamines and Related 2-Urea Derivatives Are Potent and Selective Inhibitors of the FGF Receptor-1 Tyrosine Kinase. <i>  J. Med. Chem. </i><b>2000</b>,<i> 43</i>, 4200−4211.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2000&pages=4211&author=A.+M.+Thompson&author=C.+J.+Connolly&author=J.+M.+Hamby&author=S.+Boushelle&author=B.+G+Hartl&title=et+al.+3-%283%2C5-Dimethoxyphenyl%29-1%2C6-naphthyridine-2%2C7-diamines+and+Related+2-Urea+Derivatives+Are+Potent+and+Selective+Inhibitors+of+the+FGF+Receptor-1+Tyrosine+Kinase"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm050563rb00040_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DThompson%26aufirst%3DA.%2BM.%26atitle%3Det%2520al.%25203-%25283%252C5-Dimethoxyphenyl%2529-1%252C6-naphthyridine-2%252C7-diamines%2520and%2520Related%25202-Urea%2520Derivatives%2520Are%2520Potent%2520and%2520Selective%2520Inhibitors%2520of%2520the%2520FGF%2520Receptor-1%2520Tyrosine%2520Kinase%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2000%26volume%3D43%26spage%3D4211" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm050563rb00041"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm050563rb00041'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="jm050563rb00041_1"><span><span class="NLM_string-ref">Missbach, M.; Altmann, E.; Widler, L.; Susa, M.; Buchdunger, E. et al. Substituted 5,7-diphenylpyrrolo[2,3-d]pyrimidines:   6990  potent inhibitors of the tyrosine kinase c-Src. <i>  Bioorg. Med. Chem.  Lett. </i><b>2000</b>,<i> 10</i>, 945−949.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2000&pages=949&author=M.+Missbach&author=E.+Altmann&author=L.+Widler&author=M.+Susa&author=E+Buchdunger&title=et+al.+Substituted+5%2C7-diphenylpyrrolo%5B2%2C3-d%5Dpyrimidines%3A%E2%80%89++6990++potent+inhibitors+of+the+tyrosine+kinase+c-Src"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm050563rb00041_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMissbach%26aufirst%3DM.%26atitle%3Det%2520al.%2520Substituted%25205%252C7-diphenylpyrrolo%255B2%252C3-d%255Dpyrimidines%253A%25E2%2580%2589%2520%25206990%2520%2520potent%2520inhibitors%2520of%2520the%2520tyrosine%2520kinase%2520c-Src%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520%2520Lett.%26date%3D2000%26volume%3D10%26spage%3D949" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm050563rb00042"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm050563rb00042'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="jm050563rb00042_1"><span><span class="NLM_string-ref">Connolly, C. J. C.; Hamby, J. M.; Schroeder, M. C.; Barvian, M.; Lu, G. H. et al. Discovery and structure−activity studies of a novel series of pyrido[2,3-d]pyrimidine tyrosine kinase inhibitors. <i>  Bioorg. Med. Chem. </i><i>Lett. </i><b>1997</b>,<i> 7</i>, 2415−2420.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=1997&pages=2420&author=C.+J.+C.+Connolly&author=J.+M.+Hamby&author=M.+C.+Schroeder&author=M.+Barvian&author=G.+H+Lu&title=et+al.+Discovery+and+structure%E2%88%92activity+studies+of+a+novel+series+of+pyrido%5B2%2C3-d%5Dpyrimidine+tyrosine+kinase+inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm050563rb00042_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DConnolly%26aufirst%3DC.%2BJ.%2BC.%26atitle%3Det%2520al.%2520Discovery%2520and%2520structure%25E2%2588%2592activity%2520studies%2520of%2520a%2520novel%2520series%2520of%2520pyrido%255B2%252C3-d%255Dpyrimidine%2520tyrosine%2520kinase%2520inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D1997%26volume%3D7%26spage%3D2420" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm050563rb00043"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm050563rb00043'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="jm050563rb00043_1"><span><span class="NLM_string-ref">Toledo, L. M.; Lydon, N. B.; Elbaum, D. The structure-based design of ATP-site directed protein kinase inhibitors. <i>  Curr. Med. Chem.</i><b>1999</b>,<i> 6</i>, 775−805.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=1999&pages=805&author=L.+M.+Toledo&author=N.+B.+Lydon&author=D+Elbaum&title=The+structure-based+design+of+ATP-site+directed+protein+kinase+inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm050563rb00043_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DToledo%26aufirst%3DL.%2BM.%26atitle%3DThe%2520structure-based%2520design%2520of%2520ATP-site%2520directed%2520protein%2520kinase%2520inhibitors%26jtitle%3DCurr.%2520Med.%2520Chem.%26date%3D1999%26volume%3D6%26spage%3D805" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm050563rb00044"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm050563rb00044'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="jm050563rb00044_1"><span><span class="NLM_string-ref">Catalyst; 4.8 ed.; Accelrys, Inc.:  San Diego, CA.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+Catalyst%3B+4.8+ed.%3B+Accelrys%2C+Inc.%3A%E2%80%89+San+Diego%2C+CA.+Catalyst%3B+4.8+ed.%3B+Accelrys%2C+Inc.%3A%E2%80%89+San+Diego%2C+CA."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm050563rb00044_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dfootnote%26sid%3Dliteratum%253Aachs" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm050563rb00045"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm050563rb00045'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="jm050563rb00045_1"><span><span class="NLM_string-ref">Cerius2; 4.9 ed.; Accelrys Inc.:  San Diego, CA.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+Cerius2%3B+4.9+ed.%3B+Accelrys+Inc.%3A%E2%80%89+San+Diego%2C+CA.+Cerius2%3B+4.9+ed.%3B+Accelrys+Inc.%3A%E2%80%89+San+Diego%2C+CA."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm050563rb00045_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dfootnote%26sid%3Dliteratum%253Aachs" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm050563rb00046"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm050563rb00046'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="jm050563rb00046_1"><span><span class="NLM_string-ref">Ekins, S.; Crumb, W. J.; Sarazan, R. D.; Wikel, J. H.; Wrighton, S. A. Three-dimensional quantitative structure−activity relationship for inhibition of human ether-a-go-go-related gene potassium channel. <i>  J. </i><i>Pharmacol. Exp. Ther. </i><b>2002</b>,<i> 301</i>, 427−434.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=301&publication_year=2002&pages=434&author=S.+Ekins&author=W.+J.+Crumb&author=R.+D.+Sarazan&author=J.+H.+Wikel&author=S.+A+Wrighton&title=Three-dimensional+quantitative+structure%E2%88%92activity+relationship+for+inhibition+of+human+ether-a-go-go-related+gene+potassium+channel"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm050563rb00046_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DEkins%26aufirst%3DS.%26atitle%3DThree-dimensional%2520quantitative%2520structure%25E2%2588%2592activity%2520relationship%2520for%2520inhibition%2520of%2520human%2520ether-a-go-go-related%2520gene%2520potassium%2520channel%26jtitle%3DJ.%2520Pharmacol.%2520Exp.%2520Ther.%26date%3D2002%26volume%3D301%26spage%3D434" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm050563rb00047"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm050563rb00047'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="jm050563rb00047_1"><span><span class="NLM_string-ref">Schindler, T.; Sicheri, F.; Pico, A.; Gazit, A.; Levitzki, A. et al. Crystal structure of Hck in complex with a Src family selective tyrosine kinase inhibitor. <i>  Mol. Cell </i><b>1999</b>,<i> 3</i>, 639−648.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=1999&pages=648&author=T.+Schindler&author=F.+Sicheri&author=A.+Pico&author=A.+Gazit&author=A+Levitzki&title=et+al.+Crystal+structure+of+Hck+in+complex+with+a+Src+family+selective+tyrosine+kinase+inhibitor"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm050563rb00047_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSchindler%26aufirst%3DT.%26atitle%3Det%2520al.%2520Crystal%2520structure%2520of%2520Hck%2520in%2520complex%2520with%2520a%2520Src%2520family%2520selective%2520tyrosine%2520kinase%2520inhibitor%26jtitle%3DMol.%2520Cell%26date%3D1999%26volume%3D3%26spage%3D648" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm050563rb00048"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm050563rb00048'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="jm050563rb00048_1"><span><span class="NLM_string-ref">Stamos, J.; Sliwkowski, M. X.; Eigenbrot, C. Structure of the epidermal growth factor receptor kinase domain alone and in complex with a 4-anilinoquinazoline inhibitor. <i>  J. Biol. Chem. </i><b>2002</b>,<i> 277</i>, 46265−46272.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=277&publication_year=2002&pages=46272&author=J.+Stamos&author=M.+X.+Sliwkowski&author=C+Eigenbrot&title=Structure+of+the+epidermal+growth+factor+receptor+kinase+domain+alone+and+in+complex+with+a+4-anilinoquinazoline+inhibitor"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm050563rb00048_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DStamos%26aufirst%3DJ.%26atitle%3DStructure%2520of%2520the%2520epidermal%2520growth%2520factor%2520receptor%2520kinase%2520domain%2520alone%2520and%2520in%2520complex%2520with%2520a%25204-anilinoquinazoline%2520inhibitor%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2002%26volume%3D277%26spage%3D46272" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm050563rb00049"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm050563rb00049'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="jm050563rb00049_1"><span><span class="NLM_string-ref">Nagar, B.; Bornmann, W. G.; Pellicena, P.; Schindler, T.; Veach, D. R. et al. Crystal structures of the kinase domain of c-Abl in complex with the small molecule inhibitors PD173955 and imatinib (STI-571). <i>  Cancer Res. </i><b>2002</b>, <i>62</i>, 4236−4243.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2002&pages=4243&author=B.+Nagar&author=W.+G.+Bornmann&author=P.+Pellicena&author=T.+Schindler&author=D.+R+Veach&title=et+al.+Crystal+structures+of+the+kinase+domain+of+c-Abl+in+complex+with+the+small+molecule+inhibitors+PD173955+and+imatinib+%28STI-571%29"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm050563rb00049_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DNagar%26aufirst%3DB.%26atitle%3Det%2520al.%2520Crystal%2520structures%2520of%2520the%2520kinase%2520domain%2520of%2520c-Abl%2520in%2520complex%2520with%2520the%2520small%2520molecule%2520inhibitors%2520PD173955%2520and%2520imatinib%2520%2528STI-571%2529%26jtitle%3DCancer%2520Res.%26date%3D2002%26volume%3D62%26spage%3D4243" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm050563rb00050"><div class="reference"><div class="NLM_citation" id="jm050563rb00050_1"><span><span class="NLM_string-ref">de Laszlo, S. E.; Hacker, C.; Li, B.; Kim, D.; MacCoss, M. et al. Potent, orally absorbed glucagon receptor antagonists. <i>  Bioorg. Med. Chem. Lett. </i><b>1999</b>,<i> 9</i>, 641−646.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=1999&pages=646&author=S.+E.+de+Laszlo&author=C.+Hacker&author=B.+Li&author=D.+Kim&author=M+MacCoss&title=et+al.+Potent%2C+orally+absorbed+glucagon+receptor+antagonists"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm050563rb00050_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3Dde%2BLaszlo%26aufirst%3DS.%2BE.%26atitle%3Det%2520al.%2520Potent%252C%2520orally%2520absorbed%2520glucagon%2520receptor%2520antagonists%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D1999%26volume%3D9%26spage%3D646" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'jm050563rb00001':['jm050563rb00001_1'],'jm050563rb00002':['jm050563rb00002_1'],'jm050563rb00003':['jm050563rb00003_1'],'jm050563rb00004':['jm050563rb00004_1'],'jm050563rb00005':['jm050563rb00005_1'],'jm050563rb00006':['jm050563rb00006_1'],'jm050563rb00007':['jm050563rb00007_1'],'jm050563rb00008':['jm050563rb00008_1'],'jm050563rb00009':['jm050563rb00009_1'],'jm050563rb00010':['jm050563rb00010_1'],'jm050563rb00011':['jm050563rb00011_1'],'jm050563rb00012':['jm050563rb00012_1'],'jm050563rb00013':['jm050563rb00013_1'],'jm050563rb00014':['jm050563rb00014_1'],'jm050563rb00015':['jm050563rb00015_1'],'jm050563rb00016':['jm050563rb00016_1'],'jm050563rb00017':['jm050563rb00017_1'],'jm050563rb00018':['jm050563rb00018_1'],'jm050563rb00019':['jm050563rb00019_1'],'jm050563rb00020':['jm050563rb00020_1'],'jm050563rb00021':['jm050563rb00021_1'],'jm050563rb00022':['jm050563rb00022_1'],'jm050563rb00023':['jm050563rb00023_1'],'jm050563rb00024':['jm050563rb00024_1'],'jm050563rb00025':['jm050563rb00025_1'],'jm050563rb00026':['jm050563rb00026_1'],'jm050563rb00027':['jm050563rb00027_1'],'jm050563rb00028':['jm050563rb00028_1'],'jm050563rb00029':['jm050563rb00029_1'],'jm050563rb00030':['jm050563rb00030_1'],'jm050563rb00031':['jm050563rb00031_1'],'jm050563rb00032':['jm050563rb00032_1'],'jm050563rb00033':['jm050563rb00033_1'],'jm050563rb00034':['jm050563rb00034_1'],'jm050563rb00035':['jm050563rb00035_1'],'jm050563rb00036':['jm050563rb00036_1'],'jm050563rb00037':['jm050563rb00037_1'],'jm050563rb00038':['jm050563rb00038_1'],'jm050563rb00039':['jm050563rb00039_1'],'jm050563rb00040':['jm050563rb00040_1'],'jm050563rb00041':['jm050563rb00041_1'],'jm050563rb00042':['jm050563rb00042_1'],'jm050563rb00043':['jm050563rb00043_1'],'jm050563rb00044':['jm050563rb00044_1'],'jm050563rb00045':['jm050563rb00045_1'],'jm050563rb00046':['jm050563rb00046_1'],'jm050563rb00047':['jm050563rb00047_1'],'jm050563rb00048':['jm050563rb00048_1'],'jm050563rb00049':['jm050563rb00049_1'],'jm050563rb00050':['jm050563rb00050_1']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 15 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Anders E.  Berglund</span> and <span class="hlFld-ContribAuthor ">Richard D.  Head</span>. </span><span class="cited-content_cbyCitation_article-title">PZIM: A Method for Similarity Searching Using Atom Environments and 2D Alignment. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2010,</strong> <em>50 </em>
                                    (10)
                                     , 1790-1795. <a href="https://doi.org/10.1021/ci1002075" title="DOI URL">https://doi.org/10.1021/ci1002075</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/ci1002075&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fci1002075%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DPZIM%25253A%252BA%252BMethod%252Bfor%252BSimilarity%252BSearching%252BUsing%252BAtom%252BEnvironments%252Band%252B2D%252BAlignment%26aulast%3DBerglund%26aufirst%3DAnders%2BE.%26date%3D2010%26date%3D2010%26date%3D2010%26date%3D24%252005%25202010%26date%3D01102010%26date%3D25102010%26volume%3D50%26issue%3D10%26spage%3D1790%26epage%3D1795" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Andrei V.  Anghelescu</span>, <span class="hlFld-ContribAuthor ">Robert K.  DeLisle</span>, <span class="hlFld-ContribAuthor ">Jeffrey F.  Lowrie</span>, <span class="hlFld-ContribAuthor ">Anthony E.  Klon</span>, <span class="hlFld-ContribAuthor ">Xiaoming  Xie</span> and <span class="hlFld-ContribAuthor ">David J.  Diller</span>  . </span><span class="cited-content_cbyCitation_article-title">Technique for Generating Three-Dimensional Alignments of Multiple Ligands from One-Dimensional Alignments. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2008,</strong> <em>48 </em>
                                    (5)
                                     , 1041-1054. <a href="https://doi.org/10.1021/ci700395f" title="DOI URL">https://doi.org/10.1021/ci700395f</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/ci700395f&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fci700395f%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DTechnique%252Bfor%252BGenerating%252BThree-Dimensional%252BAlignments%252Bof%252BMultiple%252BLigands%252Bfrom%252BOne-Dimensional%252BAlignments%26aulast%3DAnghelescu%26aufirst%3DAndrei%2BV.%26date%3D2008%26date%3D2008%26date%3D2007%26date%3D27%252005%25202008%26date%3D16%252004%25202008%26date%3D31%252010%25202007%26volume%3D48%26issue%3D5%26spage%3D1041%26epage%3D1054" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Lazaros  Mavridis,,   Brian D.  Hudson, and,   <span class="NLM_string-name hlFld-ContribAuthor">David W.  Ritchie</span>. </span><span class="cited-content_cbyCitation_article-title">Toward High Throughput 3D Virtual Screening Using Spherical Harmonic Surface Representations. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2007,</strong> <em>47 </em>
                                    (5)
                                     , 1787-1796. <a href="https://doi.org/10.1021/ci7001507" title="DOI URL">https://doi.org/10.1021/ci7001507</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/ci7001507&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fci7001507%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DToward%252BHigh%252BThroughput%252B3D%252BVirtual%252BScreening%252BUsing%252BSpherical%252BHarmonic%252BSurface%252BRepresentations%26aulast%3DMavridis%26aufirst%3DLazaros%26date%3D2007%26date%3D2007%26date%3D2007%26date%3D28%252004%25202007%2520Received%2520April%252028%252C%25202007%26volume%3D47%26issue%3D5%26spage%3D1787%26epage%3D1796" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Lorentz  Jäntschi</span>. </span><span class="cited-content_cbyCitation_article-title">The Eigenproblem Translated for Alignment of Molecules. </span><span class="cited-content_cbyCitation_journal-name">Symmetry</span><span> <strong>2019,</strong> <em>11 </em>
                                    (8)
                                     , 1027. <a href="https://doi.org/10.3390/sym11081027" title="DOI URL">https://doi.org/10.3390/sym11081027</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/sym11081027&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fsym11081027%26sid%3Dliteratum%253Aachs%26jtitle%3DSymmetry%26atitle%3DThe%252BEigenproblem%252BTranslated%252Bfor%252BAlignment%252Bof%252BMolecules%26aulast%3DJ%25C3%25A4ntschi%26aufirst%3DLorentz%26date%3D2019%26date%3D2019%26volume%3D11%26issue%3D8%26spage%3D1027" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ahmet  Sacan</span>, <span class="hlFld-ContribAuthor ">Sean  Ekins</span>, <span class="hlFld-ContribAuthor ">Sandhya  Kortagere</span>. </span><span class="cited-content_cbyCitation_article-title">Applications and Limitations of In Silico Models in Drug Discovery. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2012,</strong>,, 87-124. <a href="https://doi.org/10.1007/978-1-61779-965-5_6" title="DOI URL">https://doi.org/10.1007/978-1-61779-965-5_6</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/978-1-61779-965-5_6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2F978-1-61779-965-5_6%26sid%3Dliteratum%253Aachs%26atitle%3DApplications%252Band%252BLimitations%252Bof%252BIn%252BSilico%252BModels%252Bin%252BDrug%252BDiscovery%26aulast%3DSacan%26aufirst%3DAhmet%26date%3D2012%26date%3D2012%26spage%3D87%26epage%3D124%26pub%3DHumana%2520Press%26atitle%3DBioinformatics%252Band%252BDrug%252BDiscovery%26aulast%3DLarson%26aufirst%3DRichard%2BS.%26date%3D2012%26volume%3D910" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Michael C  Hutter</span>. </span><span class="cited-content_cbyCitation_article-title">Graph-based similarity concepts in virtual screening. </span><span class="cited-content_cbyCitation_journal-name">Future Medicinal Chemistry</span><span> <strong>2011,</strong> <em>3 </em>
                                    (4)
                                     , 485-501. <a href="https://doi.org/10.4155/fmc.11.3" title="DOI URL">https://doi.org/10.4155/fmc.11.3</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.4155/fmc.11.3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.4155%2Ffmc.11.3%26sid%3Dliteratum%253Aachs%26jtitle%3DFuture%2520Medicinal%2520Chemistry%26atitle%3DGraph-based%252Bsimilarity%252Bconcepts%252Bin%252Bvirtual%252Bscreening%26aulast%3DHutter%26aufirst%3DMichael%2BC%26date%3D2011%26volume%3D3%26issue%3D4%26spage%3D485%26epage%3D501" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Gyanu  Lamichhane</span>, <span class="hlFld-ContribAuthor ">Joel S.  Freundlich</span>, <span class="hlFld-ContribAuthor ">Sean  Ekins</span>, <span class="hlFld-ContribAuthor ">Niluka  Wickramaratne</span>, <span class="hlFld-ContribAuthor ">Scott T.  Nolan</span>, <span class="hlFld-ContribAuthor ">William R.  Bishai</span>, . </span><span class="cited-content_cbyCitation_article-title">Essential Metabolites of Mycobacterium tuberculosis and Their Mimics. </span><span class="cited-content_cbyCitation_journal-name">mBio</span><span> <strong>2011,</strong> <em>2 </em>
                                    (1)
                                     <a href="https://doi.org/10.1128/mBio.00301-10" title="DOI URL">https://doi.org/10.1128/mBio.00301-10</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1128/mBio.00301-10&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1128%2FmBio.00301-10%26sid%3Dliteratum%253Aachs%26jtitle%3DmBio%26atitle%3DEssential%252BMetabolites%252Bof%252BMycobacterium%252Btuberculosis%252Band%252BTheir%252BMimics%26aulast%3DLamichhane%26aufirst%3DGyanu%26date%3D2011%26volume%3D2%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Matthew D  Krasowski</span>, <span class="hlFld-ContribAuthor ">Mohamed G  Siam</span>, <span class="hlFld-ContribAuthor ">Manisha  Iyer</span>, <span class="hlFld-ContribAuthor ">Sean  Ekins</span>. </span><span class="cited-content_cbyCitation_article-title">Molecular Similarity Methods for Predicting Cross-Reactivity With Therapeutic Drug Monitoring Immunoassays. </span><span class="cited-content_cbyCitation_journal-name">Therapeutic Drug Monitoring</span><span> <strong>2009,</strong> <em>31 </em>
                                    (3)
                                     , 337-344. <a href="https://doi.org/10.1097/FTD.0b013e31819c1b83" title="DOI URL">https://doi.org/10.1097/FTD.0b013e31819c1b83</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1097/FTD.0b013e31819c1b83&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1097%2FFTD.0b013e31819c1b83%26sid%3Dliteratum%253Aachs%26jtitle%3DTherapeutic%2520Drug%2520Monitoring%26atitle%3DMolecular%252BSimilarity%252BMethods%252Bfor%252BPredicting%252BCross-Reactivity%252BWith%252BTherapeutic%252BDrug%252BMonitoring%252BImmunoassays%26aulast%3DKrasowski%26aufirst%3DMatthew%2BD%26date%3D2009%26volume%3D31%26issue%3D3%26spage%3D337%26epage%3D344" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Anton F.  Fliri</span>, <span class="hlFld-ContribAuthor ">William T.  Loging</span>, <span class="hlFld-ContribAuthor ">Robert A.  Volkmann</span>. </span><span class="cited-content_cbyCitation_article-title">Analysis of System Structure–Function Relationships. </span><span class="cited-content_cbyCitation_journal-name">ChemMedChem</span><span> <strong>2007,</strong> <em>2 </em>
                                    (12)
                                     , 1774-1782. <a href="https://doi.org/10.1002/cmdc.200700153" title="DOI URL">https://doi.org/10.1002/cmdc.200700153</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/cmdc.200700153&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fcmdc.200700153%26sid%3Dliteratum%253Aachs%26jtitle%3DChemMedChem%26atitle%3DAnalysis%252Bof%252BSystem%252BStructure%2525E2%252580%252593Function%252BRelationships%26aulast%3DFliri%26aufirst%3DAnton%25E2%2580%2585F.%26date%3D2007%26volume%3D2%26issue%3D12%26spage%3D1774%26epage%3D1782" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">David J.  Diller</span>, <span class="hlFld-ContribAuthor ">Doug W.  Hobbs</span>. </span><span class="cited-content_cbyCitation_article-title">Understanding hERG inhibition with QSAR models based on a one-dimensional molecular representation. </span><span class="cited-content_cbyCitation_journal-name">Journal of Computer-Aided Molecular Design</span><span> <strong>2007,</strong> <em>21 </em>
                                    (7)
                                     , 379-393. <a href="https://doi.org/10.1007/s10822-007-9122-2" title="DOI URL">https://doi.org/10.1007/s10822-007-9122-2</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s10822-007-9122-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs10822-007-9122-2%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Computer-Aided%2520Molecular%2520Design%26atitle%3DUnderstanding%252BhERG%252Binhibition%252Bwith%252BQSAR%252Bmodels%252Bbased%252Bon%252Ba%252Bone-dimensional%252Bmolecular%252Brepresentation%26aulast%3DDiller%26aufirst%3DDavid%2BJ.%26date%3D2007%26date%3D2007%26volume%3D21%26issue%3D7%26spage%3D379%26epage%3D393" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Hanna  Eckert</span>, <span class="hlFld-ContribAuthor ">Jürgen  Bajorath</span>. </span><span class="cited-content_cbyCitation_article-title">Molecular similarity analysis in virtual screening: foundations, limitations and novel approaches. </span><span class="cited-content_cbyCitation_journal-name">Drug Discovery Today</span><span> <strong>2007,</strong> <em>12 </em>
                                    (5-6)
                                     , 225-233. <a href="https://doi.org/10.1016/j.drudis.2007.01.011" title="DOI URL">https://doi.org/10.1016/j.drudis.2007.01.011</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.drudis.2007.01.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.drudis.2007.01.011%26sid%3Dliteratum%253Aachs%26jtitle%3DDrug%2520Discovery%2520Today%26atitle%3DMolecular%252Bsimilarity%252Banalysis%252Bin%252Bvirtual%252Bscreening%25253A%252Bfoundations%25252C%252Blimitations%252Band%252Bnovel%252Bapproaches%26aulast%3DEckert%26aufirst%3DHanna%26date%3D2007%26volume%3D12%26issue%3D5-6%26spage%3D225%26epage%3D233" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">L.H.  Hall</span>, <span class="hlFld-ContribAuthor ">L.B.  Kier</span>, <span class="hlFld-ContribAuthor ">L.M.  Hall</span>. </span><span class="cited-content_cbyCitation_article-title">Topological Quantitative Structure–Activity Relationship Applications: Structure Information Representation in Drug Discovery. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2007,</strong>,, 537-574. <a href="https://doi.org/10.1016/B0-08-045044-X/00265-0" title="DOI URL">https://doi.org/10.1016/B0-08-045044-X/00265-0</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/B0-08-045044-X/00265-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2FB0-08-045044-X%2F00265-0%26sid%3Dliteratum%253Aachs%26atitle%3DTopological%252BQuantitative%252BStructure%2525E2%252580%252593Activity%252BRelationship%252BApplications%25253A%252BStructure%252BInformation%252BRepresentation%252Bin%252BDrug%252BDiscovery%26aulast%3DHall%26aufirst%3DL.H.%26date%3D2007%26spage%3D537%26epage%3D574%26pub%3DElsevier%26atitle%3DComprehensive%252BMedicinal%252BChemistry%252BII%26date%3D2007%26volume%3D11" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Sean  Ekins</span>, <span class="hlFld-ContribAuthor ">Jun  Shimada</span>, <span class="hlFld-ContribAuthor ">Cheng  Chang</span>. </span><span class="cited-content_cbyCitation_article-title">Application of data mining approaches to drug delivery. </span><span class="cited-content_cbyCitation_journal-name">Advanced Drug Delivery Reviews</span><span> <strong>2006,</strong> <em>58 </em>
                                    (12-13)
                                     , 1409-1430. <a href="https://doi.org/10.1016/j.addr.2006.09.005" title="DOI URL">https://doi.org/10.1016/j.addr.2006.09.005</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.addr.2006.09.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.addr.2006.09.005%26sid%3Dliteratum%253Aachs%26jtitle%3DAdvanced%2520Drug%2520Delivery%2520Reviews%26atitle%3DApplication%252Bof%252Bdata%252Bmining%252Bapproaches%252Bto%252Bdrug%252Bdelivery%26aulast%3DEkins%26aufirst%3DSean%26date%3D2006%26volume%3D58%26issue%3D12-13%26spage%3D1409%26epage%3D1430" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Andreas  Bender</span>, <span class="hlFld-ContribAuthor ">Jeremy L.  Jenkins</span>, <span class="hlFld-ContribAuthor ">Qingliang  Li</span>, <span class="hlFld-ContribAuthor ">Sam E.  Adams</span>, <span class="hlFld-ContribAuthor ">Edward O.  Cannon</span>, <span class="hlFld-ContribAuthor ">Robert C.  Glen</span>. </span><span class="cited-content_cbyCitation_article-title">Chapter 9 Molecular Similarity: Advances in Methods, Applications and Validations in Virtual Screening and QSAR. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2006,</strong>,, 141-168. <a href="https://doi.org/10.1016/S1574-1400(06)02009-3" title="DOI URL">https://doi.org/10.1016/S1574-1400(06)02009-3</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/S1574-1400(06)02009-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2FS1574-1400%2806%2902009-3%26sid%3Dliteratum%253Aachs%26atitle%3DChapter%252B9%252BMolecular%252BSimilarity%25253A%252BAdvances%252Bin%252BMethods%25252C%252BApplications%252Band%252BValidations%252Bin%252BVirtual%252BScreening%252Band%252BQSAR%26aulast%3DBender%26aufirst%3DAndreas%26date%3D2006%26spage%3D141%26epage%3D168%26pub%3DElsevier%26date%3D2006%26volume%3D2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Doug  Auld</span>, <span class="hlFld-ContribAuthor ">Anton  Simeonov</span>. </span><span class="cited-content_cbyCitation_article-title">Literature Search and Review. </span><span class="cited-content_cbyCitation_journal-name">ASSAY and Drug Development Technologies</span><span> <strong>2005,</strong> <em>3 </em>
                                    (6)
                                     , 701-713. <a href="https://doi.org/10.1089/adt.2005.3.701" title="DOI URL">https://doi.org/10.1089/adt.2005.3.701</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1089/adt.2005.3.701&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1089%2Fadt.2005.3.701%26sid%3Dliteratum%253Aachs%26jtitle%3DASSAY%2520and%2520Drug%2520Development%2520Technologies%26atitle%3DLiterature%252BSearch%252Band%252BReview%26aulast%3DAuld%26aufirst%3DDoug%26date%3D2005%26volume%3D3%26issue%3D6%26spage%3D701%26epage%3D713" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm050563r/production/images/medium/jm050563rn00001.gif" alt="" id="_i2" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm050563r/production/images/large/jm050563rn00001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm050563r&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="jm050563rf00001" data-index="1" class="article__inlineFigure"><h2 class="fig-label"></h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm050563r/production/images/medium/jm050563rf00001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm050563r/production/images/large/jm050563rf00001.jpeg" id="rightTab-_i5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last"> Figure 1 An example of a small molecule showing multiple biological activities and subtle changes that distinguish between these activities.<sup><a onclick="showRef(event, 'jm050563rb00050'); return false;" href="javascript:void(0);" class="ref jm050563rb00050">50</a></sup> This example shows that assigning function to a small molecule, i.e., its biological activity, based on similarity is a context specific problem. (A) Compound <b>A</b> both inhibits the p38 kinase and acts as an antagonist of the glucagon receptor which is a GPCR from the class B family. Compound <b>B</b> is an example of a small molecule that is closely related to compound <b>A</b> and retains the p38 inhibitory activity but is devoid of the glucagon activity. Compound <b>C</b> is an example of a small molecule that is closely related to <b>A</b> and retains the glucagon activity but is devoid of the p38 activity. Thus, from the perspective of p38, molecule <b>A</b> is similar to <b>B</b> but not <b>C</b> whereas from the perspective of the glucagon receptor, <b>A</b> is similar to <b>C</b> but not <b>B</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm050563r/production/images/large/jm050563rf00001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm050563r&amp;id=jm050563rf00001"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="jm050563rf00002" data-index="2" class="article__inlineFigure"><h2 class="fig-label"></h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm050563r/production/images/medium/jm050563rf00002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm050563r/production/images/large/jm050563rf00002.jpeg" id="rightTab-_i7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last"> Figure 2 The search space in a multiple 1-dimensional alignment of three molecules. (a) The 4 possible relative orientations of the 3 1-dimensional objects. Note that the four remaining orientations in which the top arrow points to the left are by flipping all three arrows mathematically equivalent to one of the four relative orientations shown in part a. (b) The 2D search space for a single relative orientation of the three molecules Cisapride, Thioridazine, and Ziprasidone.  </p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm050563r/production/images/large/jm050563rf00002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm050563r&amp;id=jm050563rf00002"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="jm050563rf00003" data-index="3" class="article__inlineFigure"><h2 class="fig-label"></h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm050563r/production/images/medium/jm050563rf00003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm050563r/production/images/large/jm050563rf00003.jpeg" id="rightTab-_i15" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last"> Figure 3 The distribution of the extent of distortion of each molecule caused by projecting from two-dimensional topological distances to its 1-dimensional representation. 90% of the molecules exhibit a distortion below 9%, 95% exhibit a distortion below 11%, 99% exhibit a distortion below 14%, and essentially all the molecules of the MDDR database exhibit a distortion below 20%. Thus, the 1-dimensional representation retains the majority of the information present in the 2-dimesional structure.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm050563r/production/images/large/jm050563rf00003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm050563r&amp;id=jm050563rf00003"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="jm050563rf00004" data-index="4" class="article__inlineFigure"><h2 class="fig-label"></h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm050563r/production/images/medium/jm050563rf00004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm050563r/production/images/large/jm050563rf00004.jpeg" id="rightTab-_i17" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last"> Figure 4 The SRC 1-dimensional multiple ligand alignment. (a) The compounds used in the alignment. (b) The resulting alignment. Each compound is shown in part a roughly oriented in the same manner as in the alignment in part b. The color indicates the type of atom, i.e., gray is carbon, red is oxygen, blue is nitrogen, etc. The darkness of the color indicates the hybridization:  light colors indicate Sp3 atoms and dark colors indicate Sp2 or Sp1 atoms. The size of the atom is proportional to the amount the atom contributes to the score of the overall alignment.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm050563r/production/images/large/jm050563rf00004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm050563r&amp;id=jm050563rf00004"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="jm050563rf00005" data-index="5" class="article__inlineFigure"><h2 class="fig-label"></h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm050563r/production/images/medium/jm050563rf00005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm050563r/production/images/large/jm050563rf00005.jpeg" id="rightTab-_i19" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last"> Figure 5 The performance of the SRC profile. To make these figures, the molecules are ranked and sorted by their score, in this case their similarity to the 1-dimensional profile. The top curve is the fraction of the known SRC actives found (<i>y</i>-axis) versus the fraction of the overall compounds (<i>x</i>-axis). The diagonal line is the expected performance of a method that selected compounds at random.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm050563r/production/images/large/jm050563rf00005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm050563r&amp;id=jm050563rf00005"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="jm050563rf00006" data-index="6" class="article__inlineFigure"><h2 class="fig-label"></h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm050563r/production/images/medium/jm050563rf00006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm050563r/production/images/large/jm050563rf00006.jpeg" id="rightTab-_i21" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last"> Figure 6 The hERG 1-dimensional Multiple Ligand Alignment. (a) The compounds used in the alignment. (b) The hERG 1-dimensional alignment. Each compound is shown in <b>6a</b> roughly oriented in the same manner as in the alignment in part b. The atoms are colored and sized as described in Figure 4b.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm050563r/production/images/large/jm050563rf00006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm050563r&amp;id=jm050563rf00006"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="jm050563rf00007" data-index="7" class="article__inlineFigure"><h2 class="fig-label"></h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm050563r/production/images/medium/jm050563rf00007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm050563r/production/images/large/jm050563rf00007.jpeg" id="rightTab-_i23" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last"> Figure 7 The performance of the hERG profile. This figure was created using the hERG 1-dimensional profile to rank the compounds of the MDDR and the hERG training set in the same manner as Figure 5.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm050563r/production/images/large/jm050563rf00007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm050563r&amp;id=jm050563rf00007"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="jm050563rf00008" data-index="8" class="article__inlineFigure"><h2 class="fig-label"></h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm050563r/production/images/medium/jm050563rf00008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm050563r/production/images/large/jm050563rf00008.jpeg" id="rightTab-_i25" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last"> Figure 8 The compounds and the features used to build the SRC pharmacophore model. The three compounds were selected from crystal structures of related kinases because they are representative of the chemotypes found in the training set. <b>A</b> was taken from the crystal structure of 1m17<sup>48</sup> (EGFr). <b>B</b> was take from the pdb structure 1m52<sup>49</sup> (ABL). <b>C</b> was taken from pdb structure 1qcf<sup>47</sup> (HCK). The orange aromatic ring of the three ligands bound in the main hybrophobic pocket of the appropriate kinases. This interaction was represented by an aromatic ring feature. The blue nitrogen of the three ligands was observed to interact with the hinge NH (corresponds to Met341 of SRC) in each case. This interaction was represented by a hydrogen bond acceptor. The substructures highlighted in purple were bound in the secondary hydrophobic site and were represented as a hydrophobic feature. The central heteroaromatic core was represented by an aromatic ring feature. Finally, the NH of <b>B</b> was observed to interact with a backbone carbonyl oxygen of the kinase (corresponds to Met341 of SRC) while the NH2 of <b>C</b> was observed to interact with a second carbonyl oxygen of the kinase (corresponds to Glu339 of SRC). Each of these was represented by a hydrogen bond donor feature in the 6 feature model.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm050563r/production/images/large/jm050563rf00008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm050563r&amp;id=jm050563rf00008"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="jm050563rf00009" data-index="9" class="article__inlineFigure"><h2 class="fig-label"></h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm050563r/production/images/medium/jm050563rf00009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm050563r/production/images/large/jm050563rf00009.jpeg" id="rightTab-_i27" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last"> Figure 9 A comparison between the SRC 1-dimensional profile and 3-dimensional methods. (a) The performance of the 1-dimensional profile is in black. The performance of the 4 feature pharmacophore model is in red. The performance of the 6 feature pharmacophore model is in blue. The pharmacophore models are described in Figure 8. (b) The performance of the 3-dimensional profile is in black. The results with the 1-dimensional pharmacophore fingerprint are shown in blue and red. The maximum similarity between the training set compounds was used to create the red curve whereas the average similarity to the training set compounds was used to create the blue compounds.  </p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm050563r/production/images/large/jm050563rf00009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm050563r&amp;id=jm050563rf00009"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="jm050563rf00010" data-index="10" class="article__inlineFigure"><h2 class="fig-label"></h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm050563r/production/images/medium/jm050563rf00010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm050563r/production/images/large/jm050563rf00010.jpeg" id="rightTab-_i28" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last"> Figure 10 A comparison between the hERG 1-dimensional profile and 3-dimensional methods. (a) The performance of the 1-dimensional profile is in black. The performance of the hERG pharmacophore model is in blue. The hERG pharmacophore model used is a reproduction of the model by Ekins and co-workers.<sup>46</sup> (b) The performance of the 1-dimensional profile is in black. The results with the 1-dimensional pharmacophore fingerprint are shown in blue and red. The maximum similarity between the training set compounds was used to create the red curve whereas the average similarity to the training set compounds was used to create the blue compounds.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm050563r/production/images/large/jm050563rf00010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm050563r&amp;id=jm050563rf00010"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="jm050563rf00011" data-index="11" class="article__inlineFigure"><h2 class="fig-label"></h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm050563r/production/images/medium/jm050563rf00011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm050563r/production/images/large/jm050563rf00011.jpeg" id="rightTab-_i30" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last"> Figure 11 A comparison of the 1-dimensional profiles to pairwise 1-dimensional similarity. In both cases the results with the 1-dimensional profile are shown in black, the results with the maximum similarity metric are shown in red, and the results with the average similarity metric are shown in blue. (a) The SRC example. (b) The hERG case.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm050563r/production/images/large/jm050563rf00011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm050563r&amp;id=jm050563rf00011"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"></div><p class="rightTab-references-count">
            This article references 50 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-jm050563rb00001"><div class="reference"><strong class="refLabel"><a href="#jm050563rb00001" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-jm050563rb00001_1"><span><span class="NLM_string-ref">Martin, Y. C.; Kofron, J. L.; Traphagen, L. M. Do structurally similar molecules have similar biological activity? <i>J. Med. Chem.</i><b>2002</b>,<i> 45</i>, 4350−4358.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2002&pages=4358&author=Y.+C.+Martin&author=J.+L.+Kofron&author=L.+M+Traphagen&title=Do+structurally+similar+molecules+have+similar+biological+activity"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm050563rb00001_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMartin%26aufirst%3DY.%2BC.%26atitle%3DDo%2520structurally%2520similar%2520molecules%2520have%2520similar%2520biological%2520activity%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2002%26volume%3D45%26spage%3D4358" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm050563rb00002"><div class="reference"><strong class="refLabel"><a href="#jm050563rb00002" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-jm050563rb00002_1"><span><span class="NLM_string-ref">Durant, J. L.; Leland, B. A.; Henry, D. R.; Nourse, J. G. Reoptimization of MDL keys for use in drug discovery. <i>  J. Chem. </i><i>Inf.</i><i> Comput. Sci</i>. <b>2002</b><i>, 42</i>, 1273−1280.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2002&pages=1280&author=J.+L.+Durant&author=B.+A.+Leland&author=D.+R.+Henry&author=J.+G+Nourse&title=Reoptimization+of+MDL+keys+for+use+in+drug+discovery"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm050563rb00002_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DDurant%26aufirst%3DJ.%2BL.%26atitle%3DReoptimization%2520of%2520MDL%2520keys%2520for%2520use%2520in%2520drug%2520discovery%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Comput.%2520Sci.%26date%3D2002%26volume%3D42%26spage%3D1280" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm050563rb00003"><div class="reference"><strong class="refLabel"><a href="#jm050563rb00003" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-jm050563rb00003_1"><span><span class="NLM_string-ref">Makara, G. M. Measuring molecular similarity and diversity:  total pharmacophore diversity. <i>  J. Med. Chem. </i><b>2001</b><i>, 44</i>, 3563−3571.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2001&pages=3571&author=G.+M+Makara&title=Measuring+molecular+similarity+and+diversity%3A%E2%80%89+total+pharmacophore+diversity"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm050563rb00003_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMakara%26aufirst%3DG.%2BM%26atitle%3DMeasuring%2520molecular%2520similarity%2520and%2520diversity%253A%25E2%2580%2589%2520total%2520pharmacophore%2520diversity%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2001%26volume%3D44%26spage%3D3571" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm050563rb00004"><div class="reference"><strong class="refLabel"><a href="#jm050563rb00004" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-jm050563rb00004_1"><span><span class="NLM_string-ref">Mason, J. S.; Good, A. C.; Martin, E. J. 3-D pharmacophores in drug discovery. <i>  Curr. Pharm. Des. </i><b>2001</b><i>, 7</i>, 567−597.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2001&pages=597&author=J.+S.+Mason&author=A.+C.+Good&author=E.+J.+Martin&title=pharmacophores+in+drug+discovery"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm050563rb00004_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMason%26aufirst%3DJ.%2BS.%26atitle%3Dpharmacophores%2520in%2520drug%2520discovery%26jtitle%3DCurr.%2520Pharm.%2520Des.%26date%3D2001%26volume%3D7%26spage%3D597" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm050563rb00005"><div class="reference"><strong class="refLabel"><a href="#jm050563rb00005" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-jm050563rb00005_1"><span><span class="NLM_string-ref">McGregor, M. J.; Muskal, S. M. Pharmacophore fingerprinting. 1. Application to QSAR and focused library design. <i>  J. Chem. </i><i>Inf.</i><i> Comput. Sci.</i><b>1999</b><i>, 39</i>, 569−574.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=1999&pages=574&author=M.+J.+McGregor&author=S.+M+Muskal&title=Pharmacophore+fingerprinting.+1.+Application+to+QSAR+and+focused+library+design"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm050563rb00005_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMcGregor%26aufirst%3DM.%2BJ.%26atitle%3DPharmacophore%2520fingerprinting.%25201.%2520Application%2520to%2520QSAR%2520and%2520focused%2520library%2520design%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Comput.%2520Sci.%26date%3D1999%26volume%3D39%26spage%3D574" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm050563rb00006"><div class="reference"><strong class="refLabel"><a href="#jm050563rb00006" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-jm050563rb00006_1"><span><span class="NLM_string-ref">McGregor, M. J.; Muskal, S. M. Pharmacophore fingerprinting. 2. Application to primary library design. <i>  J. Chem. </i><i>Inf.</i><i> Comput. Sci</i>. <b>2000</b>,<i> 40</i>, 117−125.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2000&pages=125&author=M.+J.+McGregor&author=S.+M+Muskal&title=Pharmacophore+fingerprinting.+2.+Application+to+primary+library+design"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm050563rb00006_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMcGregor%26aufirst%3DM.%2BJ.%26atitle%3DPharmacophore%2520fingerprinting.%25202.%2520Application%2520to%2520primary%2520library%2520design%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Comput.%2520Sci.%26date%3D2000%26volume%3D40%26spage%3D125" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm050563rb00007"><div class="reference"><strong class="refLabel"><a href="#jm050563rb00007" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-jm050563rb00007_1"><span><span class="NLM_string-ref">Rarey, M.; Stahl, M. Similarity searching in large combinatorial chemistry spaces. <i>  J. Comput. Aided Mol. Des. </i><b>2001</b>,<i> 15</i>, 497−520.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2001&pages=520&author=M.+Rarey&author=M+Stahl&title=Similarity+searching+in+large+combinatorial+chemistry+spaces"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm050563rb00007_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DRarey%26aufirst%3DM.%26atitle%3DSimilarity%2520searching%2520in%2520large%2520combinatorial%2520chemistry%2520spaces%26jtitle%3DJ.%2520Comput.%2520Aided%2520Mol.%2520Des.%26date%3D2001%26volume%3D15%26spage%3D520" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm050563rb00008"><div class="reference"><strong class="refLabel"><a href="#jm050563rb00008" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-jm050563rb00008_1"><span><span class="NLM_string-ref">Lemmen, C.; Lengauer, T.; Klebe, G. FLEXS:  a method for fast flexible ligand superposition. <i>  J. Med. Chem. </i><b>1998</b>,<i> 41</i>, 4502−4520.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=1998&pages=4520&author=C.+Lemmen&author=T.+Lengauer&author=G.+Klebe&title=a+method+for+fast+flexible+ligand+superposition"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm050563rb00008_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DLemmen%26aufirst%3DC.%26atitle%3Da%2520method%2520for%2520fast%2520flexible%2520ligand%2520superposition%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1998%26volume%3D41%26spage%3D4520" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm050563rb00009"><div class="reference"><strong class="refLabel"><a href="#jm050563rb00009" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-jm050563rb00009_1"><span><span class="NLM_string-ref">Lemmen, C.; Lengauer, T. Computational methods for the structural alignment of molecules. <i>  J. Comput. Aided Mol. Des. </i><b>2000</b><i>, 14</i>, 215−232.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2000&pages=232&author=C.+Lemmen&author=T+Lengauer&title=Computational+methods+for+the+structural+alignment+of+molecules"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm050563rb00009_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DLemmen%26aufirst%3DC.%26atitle%3DComputational%2520methods%2520for%2520the%2520structural%2520alignment%2520of%2520molecules%26jtitle%3DJ.%2520Comput.%2520Aided%2520Mol.%2520Des.%26date%3D2000%26volume%3D14%26spage%3D232" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm050563rb00010"><div class="reference"><div class="NLM_citation" id="rightTab-jm050563rb00010_1"><span><span class="NLM_string-ref">Barrett, A. J.; Rawlings, N. D. Families and clans of serine peptidases. <i>  Arch. </i><i>Biochem. Biophys. </i><b>1995</b><i>, 318</i>, 247−250.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=318&publication_year=1995&pages=250&author=A.+J.+Barrett&author=N.+D+Rawlings&title=Families+and+clans+of+serine+peptidases"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm050563rb00010_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBarrett%26aufirst%3DA.%2BJ.%26atitle%3DFamilies%2520and%2520clans%2520of%2520serine%2520peptidases%26jtitle%3DArch.%2520Biochem.%2520Biophys.%26date%3D1995%26volume%3D318%26spage%3D250" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm050563rb00011"><div class="reference"><strong class="refLabel"><a href="#jm050563rb00011" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-jm050563rb00011_1"><span><span class="NLM_string-ref">Fischer, D.; Wolfson, H.; Lin, S. L.; Nussinov, R. Three-dimensional, sequence order-independent structural comparison of a serine protease against the crystallographic database reveals active site similarities:  potential implications to evolution and to protein folding. <i>  Protein Sci. </i><b>1994</b>,<i> 3</i>, 769−778.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=1994&pages=778&author=D.+Fischer&author=H.+Wolfson&author=S.+L.+Lin&author=R+Nussinov&title=Three-dimensional%2C+sequence+order-independent+structural+comparison+of+a+serine+protease+against+the+crystallographic+database+reveals+active+site+similarities%3A%E2%80%89+potential+implications+to+evolution+and+to+protein+folding"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm050563rb00011_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DFischer%26aufirst%3DD.%26atitle%3DThree-dimensional%252C%2520sequence%2520order-independent%2520structural%2520comparison%2520of%2520a%2520serine%2520protease%2520against%2520the%2520crystallographic%2520database%2520reveals%2520active%2520site%2520similarities%253A%25E2%2580%2589%2520potential%2520implications%2520to%2520evolution%2520and%2520to%2520protein%2520folding%26jtitle%3DProtein%2520Sci.%26date%3D1994%26volume%3D3%26spage%3D778" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm050563rb00012"><div class="reference"><strong class="refLabel"><a href="#jm050563rb00012" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-jm050563rb00012_1"><span><span class="NLM_string-ref">Krem, M. M.; Di Cera, E. Molecular markers of serine protease evolution. <i>  EMBO J. </i><b>2001</b>,<i> 20</i>, 3036−3045.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2001&pages=3045&author=M.+M.+Krem&author=E+Di+Cera&title=Molecular+markers+of+serine+protease+evolution"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm050563rb00012_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DKrem%26aufirst%3DM.%2BM.%26atitle%3DMolecular%2520markers%2520of%2520serine%2520protease%2520evolution%26jtitle%3DEMBO%2520J.%26date%3D2001%26volume%3D20%26spage%3D3045" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm050563rb00013"><div class="reference"><strong class="refLabel"><a href="#jm050563rb00013" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-jm050563rb00013_1"><span><span class="NLM_string-ref">Rawlings, N. D.; Barrett, A. J. Families of serine peptidases. <i>  Methods Enzymol. </i><b>1994</b>,<i> 244</i>, 19−61.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=244&publication_year=1994&pages=61&author=N.+D.+Rawlings&author=A.+J+Barrett&title=Families+of+serine+peptidases"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm050563rb00013_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DRawlings%26aufirst%3DN.%2BD.%26atitle%3DFamilies%2520of%2520serine%2520peptidases%26jtitle%3DMethods%2520Enzymol.%26date%3D1994%26volume%3D244%26spage%3D61" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm050563rb00014"><div class="reference"><strong class="refLabel"><a href="#jm050563rb00014" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-jm050563rb00014_1"><span><span class="NLM_string-ref">Guner, O. F. History and evolution of the pharmacophore concept in computer-aided drug design. <i>  Curr. Top. Med. Chem.</i><b>2002</b><i>, 2</i>, 1321−1332.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2002&pages=1332&author=O.+F+Guner&title=History+and+evolution+of+the+pharmacophore+concept+in+computer-aided+drug+design"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm050563rb00014_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DGuner%26aufirst%3DO.%2BF%26atitle%3DHistory%2520and%2520evolution%2520of%2520the%2520pharmacophore%2520concept%2520in%2520computer-aided%2520drug%2520design%26jtitle%3DCurr.%2520Top.%2520Med.%2520Chem.%26date%3D2002%26volume%3D2%26spage%3D1332" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm050563rb00015"><div class="reference"><strong class="refLabel"><a href="#jm050563rb00015" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-jm050563rb00015_1"><span><span class="NLM_string-ref">Dixon, S. L.; Merz, K. M., Jr. One-dimensional molecular representations and similarity calculations:  methodology and validation. <i>  J. Med. Chem. </i><b>2001</b><i>, </i><i>44</i>, 3795−3809.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2001&pages=3809&author=S.+L.+Dixon&author=K.+M.+Merz&author=+Jr&title=One-dimensional+molecular+representations+and+similarity+calculations%3A%E2%80%89+methodology+and+validation"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm050563rb00015_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DDixon%26aufirst%3DS.%2BL.%26atitle%3DOne-dimensional%2520molecular%2520representations%2520and%2520similarity%2520calculations%253A%25E2%2580%2589%2520methodology%2520and%2520validation%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2001%26volume%3D44%26spage%3D3809" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm050563rb00016"><div class="reference"><strong class="refLabel"><a href="#jm050563rb00016" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-jm050563rb00016_1"><span><span class="NLM_string-ref">Borg, I.; Groenen, P. <i>  Modern Multidimensional Scaling:  Theory and </i><i>Applications</i>; Springer:  New York, 1997.</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=jm050563rb00016/jm050563rb00016_1&amp;dbid=16&amp;doi=10.1021%2Fjm050563r&amp;key=10.1007%2F978-1-4757-2711-1" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=1997&author=I.+Borg&author=P.+Groenen&title=Modern+Multidimensional+Scaling%3A%E2%80%89+Theory+and+Applications"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm050563rb00016_1&amp;dbid=16384&amp;doi=10.1007%2F978-1-4757-2711-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1007%252F978-1-4757-2711-1%26sid%3Dliteratum%253Aachs%26aulast%3DBorg%26aufirst%3DI.%26btitle%3DModern%2520Multidimensional%2520Scaling%253A%25E2%2580%2589%2520Theory%2520and%2520Applications%26pub%3DSpringer%26date%3D1997" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm050563rb00017"><div class="reference"><strong class="refLabel"><a href="#jm050563rb00017" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-jm050563rb00017_1"><span><span class="NLM_string-ref">Mohammad, S.; Zhou, Z.; Gong, Q.; January, C. T. Blockage of the HERG human cardiac K+ channel by the gastrointestinalprokinetic agent cisapride. <i>  Am. J. Physiol. </i><b>1997</b><i>, 273</i>, 2534−2538.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=273&publication_year=1997&pages=2538&author=S.+Mohammad&author=Z.+Zhou&author=Q.+Gong&author=C.+T+January&title=Blockage+of+the+HERG+human+cardiac+K%2B+channel+by+the+gastrointestinalprokinetic+agent+cisapride"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm050563rb00017_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMohammad%26aufirst%3DS.%26atitle%3DBlockage%2520of%2520the%2520HERG%2520human%2520cardiac%2520K%252B%2520channel%2520by%2520the%2520gastrointestinalprokinetic%2520agent%2520cisapride%26jtitle%3DAm.%2520J.%2520Physiol.%26date%3D1997%26volume%3D273%26spage%3D2538" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm050563rb00018"><div class="reference"><strong class="refLabel"><a href="#jm050563rb00018" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-jm050563rb00018_1"><span><span class="NLM_string-ref">Kongsamut, S.; Kang, J.; Chen, X.; Roehr, J.; Rampe, D. A comparison of the receptor binding and HERG channel affinities for a series of antipsychotic drugs. <i>  Eur. J. Pharmacol. </i><b>2002</b>,<i> 450</i>, 37−41.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=450&publication_year=2002&pages=41&author=S.+Kongsamut&author=J.+Kang&author=X.+Chen&author=J.+Roehr&author=D+Rampe&title=A+comparison+of+the+receptor+binding+and+HERG+channel+affinities+for+a+series+of+antipsychotic+drugs"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm050563rb00018_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DKongsamut%26aufirst%3DS.%26atitle%3DA%2520comparison%2520of%2520the%2520receptor%2520binding%2520and%2520HERG%2520channel%2520affinities%2520for%2520a%2520series%2520of%2520antipsychotic%2520drugs%26jtitle%3DEur.%2520J.%2520Pharmacol.%26date%3D2002%26volume%3D450%26spage%3D41" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm050563rb00019"><div class="reference"><strong class="refLabel"><a href="#jm050563rb00019" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-jm050563rb00019_1"><span><span class="NLM_string-ref">Fogel, L. J.; Owens, A. J.; Walsh, M. J. <i>  Artificial intelligence through simulated </i><i>evolution</i>; John Wiley and Sons:  New York, 1966.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=1966&author=L.+J.+Fogel&author=A.+J.+Owens&author=M.+J.+Walsh&title=Artificial+intelligence+through+simulated+evolution"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm050563rb00019_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DFogel%26aufirst%3DL.%2BJ.%26btitle%3DArtificial%2520intelligence%2520through%2520simulated%2520evolution%26pub%3DJohn%2520Wiley%2520and%2520Sons%26date%3D1966" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm050563rb00020"><div class="reference"><div class="NLM_citation" id="rightTab-jm050563rb00020_1"><span><span class="NLM_string-ref">Holland, J. H. <i>  Adaptation is natural and artificial systems</i>; Univesity of Michigan Press:  Ann Arbor, 1975.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=1975&author=J.+H.+Holland&title=Adaptation+is+natural+and+artificial+systems"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm050563rb00020_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DHolland%26aufirst%3DJ.%2BH.%26btitle%3DAdaptation%2520is%2520natural%2520and%2520artificial%2520systems%26pub%3DUnivesity%2520of%2520Michigan%2520Press%26date%3D1975" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm050563rb00021"><div class="reference"><strong class="refLabel"><a href="#jm050563rb00021" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-jm050563rb00021_1"><span><span class="NLM_string-ref">Baeck, T.; Fogel, D. B.; Michalewicz, Z.; Back, T.; Fogel, D. R. R. <i>  Evolutionary Computation 1:  Basic Algorithms and </i>Operators; Institute of Physics:  Bristol, UK, 2000.</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=jm050563rb00021/jm050563rb00021_1&amp;dbid=16&amp;doi=10.1021%2Fjm050563r&amp;key=10.1887%2F0750306653" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2000&author=T.+Baeck&author=D.+B.+Fogel&author=Z.+Michalewicz&author=T.+Back&author=D.+R.+R.+Fogel&title=Evolutionary+Computation+1%3A%E2%80%89+Basic+Algorithms+and+Operators"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm050563rb00021_1&amp;dbid=16384&amp;doi=10.1887%2F0750306653&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1887%252F0750306653%26sid%3Dliteratum%253Aachs%26aulast%3DBaeck%26aufirst%3DT.%26btitle%3DEvolutionary%2520Computation%25201%253A%25E2%2580%2589%2520Basic%2520Algorithms%2520and%2520Operators%26pub%3DInstitute%2520of%2520Physics%26date%3D2000" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm050563rb00022"><div class="reference"><strong class="refLabel"><a href="#jm050563rb00022" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-jm050563rb00022_1"><span><span class="NLM_string-ref">Baeck, T.; Fogel, D. B.; Michalewicz, Z. <i>  Evolutionary Computation 2:  </i><i>Advanced Algorithms and Operators</i>; Institute of Physics:  Bristol, UK, 2000.</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=jm050563rb00022/jm050563rb00022_1&amp;dbid=16&amp;doi=10.1021%2Fjm050563r&amp;key=10.1887%2F0750306653" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2000&author=T.+Baeck&author=D.+B.+Fogel&author=Z.+Michalewicz&title=Evolutionary+Computation+2%3A%E2%80%89+Advanced+Algorithms+and+Operators"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm050563rb00022_1&amp;dbid=16384&amp;doi=10.1887%2F0750306653&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1887%252F0750306653%26sid%3Dliteratum%253Aachs%26aulast%3DBaeck%26aufirst%3DT.%26btitle%3DEvolutionary%2520Computation%25202%253A%25E2%2580%2589%2520Advanced%2520Algorithms%2520and%2520Operators%26pub%3DInstitute%2520of%2520Physics%26date%3D2000" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm050563rb00023"><div class="reference"><strong class="refLabel"><a href="#jm050563rb00023" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-jm050563rb00023_1"><span><span class="NLM_string-ref">Schwartz, R. M.; Dayhoff, M. O. Matrices for detecting distant relationships. <i>  Atlas of Protein Sequence and Structure</i>; National Biomedical Research Foundation. Washington, D.C., 1978.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=1978&author=R.+M.+Schwartz&author=M.+O.+Dayhoff&title=Matrices+for+detecting+distant+relationships.+Atlas+of+Protein+Sequence+and+Structure"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm050563rb00023_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DSchwartz%26aufirst%3DR.%2BM.%26btitle%3DMatrices%2520for%2520detecting%2520distant%2520relationships.%2520Atlas%2520of%2520Protein%2520Sequence%2520and%2520Structure%26pub%3DD.C%26date%3D1978" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm050563rb00024"><div class="reference"><strong class="refLabel"><a href="#jm050563rb00024" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-jm050563rb00024_1"><span><span class="NLM_string-ref">Henikoff, S.; Henikoff, J. G. Amino acid substitution matrices from protein blocks. <i>  Proc. Natl. Acad. Sci. U.S.A.</i><b>1992</b><i>, 89</i>, 10915−10919.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=89&publication_year=1992&pages=10919&author=S.+Henikoff&author=J.+G+Henikoff&title=Amino+acid+substitution+matrices+from+protein+blocks"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm050563rb00024_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DHenikoff%26aufirst%3DS.%26atitle%3DAmino%2520acid%2520substitution%2520matrices%2520from%2520protein%2520blocks%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D1992%26volume%3D89%26spage%3D10919" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm050563rb00025"><div class="reference"><strong class="refLabel"><a href="#jm050563rb00025" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-jm050563rb00025_1"><span><span class="NLM_string-ref">Gonnet, G. H.; Cohen, M. A.; Benner, S. A. Exhaustive matching of the entire protein sequence database. <i>  Science </i><b>1992</b>, 256, 1443−1445.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=256&publication_year=1992&pages=1445&author=G.+H.+Gonnet&author=M.+A.+Cohen&author=S.+A+Benner&title=Exhaustive+matching+of+the+entire+protein+sequence+database"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm050563rb00025_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DGonnet%26aufirst%3DG.%2BH.%26atitle%3DExhaustive%2520matching%2520of%2520the%2520entire%2520protein%2520sequence%2520database%26jtitle%3DScience%26date%3D1992%26volume%3D256%26spage%3D1445" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm050563rb00026"><div class="reference"><strong class="refLabel"><a href="#jm050563rb00026" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-jm050563rb00026_1"><span><span class="NLM_string-ref">Jones, D. T.; Taylor, W. R.; Thornton, J. M. The rapid generation of mutation data matrices from protein sequences. <i>  Comput. Appl. Biosci. </i><b>1992</b>,<i> 8</i>, 275−282.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=1992&pages=282&author=D.+T.+Jones&author=W.+R.+Taylor&author=J.+M+Thornton&title=The+rapid+generation+of+mutation+data+matrices+from+protein+sequences"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm050563rb00026_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DJones%26aufirst%3DD.%2BT.%26atitle%3DThe%2520rapid%2520generation%2520of%2520mutation%2520data%2520matrices%2520from%2520protein%2520sequences%26jtitle%3DComput.%2520Appl.%2520Biosci.%26date%3D1992%26volume%3D8%26spage%3D282" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm050563rb00027"><div class="reference"><strong class="refLabel"><a href="#jm050563rb00027" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-jm050563rb00027_1"><span><span class="NLM_string-ref">MDDR <i>MDL Drug Data Report</i>; MDL:  San Leandro, CA.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+MDDR+MDL+Drug+Data+Report%3B+MDL%3A%E2%80%89+San+Leandro%2C+CA.+MDDR+MDL+Drug+Data+Report%3B+MDL%3A%E2%80%89+San+Leandro%2C+CA."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm050563rb00027_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dfootnote%26sid%3Dliteratum%253Aachs" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm050563rb00028"><div class="reference"><strong class="refLabel"><a href="#jm050563rb00028" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-jm050563rb00028_1"><span><span class="NLM_string-ref">Press:  W. H.; Flannery, B. P.; Teukolsky, S. A.; Vetterling, W. T. <i>  Numerical </i><i>recipes in C:  The art of scientific computing</i>, 2nd ed.; Cambridge University Press:  New York, 1993; 994.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=1993&author=W.+H.+Press&author=B.+P.+Flannery&author=S.+A.+Teukolsky&author=W.+T.+Vetterling&title=Numerical+recipes+in+C%3A%E2%80%89+The+art+of+scientific+computing"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm050563rb00028_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DPress%26aufirst%3DW.%2BH.%26btitle%3DNumerical%2520recipes%2520in%2520C%253A%25E2%2580%2589%2520The%2520art%2520of%2520scientific%2520computing%26pub%3DCambridge%2520University%2520Press%26date%3D1993" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm050563rb00029"><div class="reference"><strong class="refLabel"><a href="#jm050563rb00029" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-jm050563rb00029_1"><span><span class="NLM_string-ref">Frame, M. C. Src in cancer:  deregulation and consequences for cell behaviour. <i>  Biochim. Biophys. Acta </i><b>2002</b>,<i> 1602</i>, 114−130.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1602&publication_year=2002&pages=130&author=M.+C+Frame&title=Src+in+cancer%3A%E2%80%89+deregulation+and+consequences+for+cell+behaviour"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm050563rb00029_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DFrame%26aufirst%3DM.%2BC%26atitle%3DSrc%2520in%2520cancer%253A%25E2%2580%2589%2520deregulation%2520and%2520consequences%2520for%2520cell%2520behaviour%26jtitle%3DBiochim.%2520Biophys.%2520Acta%26date%3D2002%26volume%3D1602%26spage%3D130" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm050563rb00030"><div class="reference"><div class="NLM_citation" id="rightTab-jm050563rb00030_1"><span><span class="NLM_string-ref">De Ponti, F.; Poluzzi, E.; Cavalli, A.; Recanatini, M.; Montanaro, N. Safety of nonantiarrhythmic drugs that prolong the QT interval or induce torsade de pointes:  an overview. <i>  Drug Safety </i><b>2002</b>,<i> 25</i>, 263−286.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2002&pages=286&author=F.+De+Ponti&author=E.+Poluzzi&author=A.+Cavalli&author=M.+Recanatini&author=N+Montanaro&title=Safety+of+nonantiarrhythmic+drugs+that+prolong+the+QT+interval+or+induce+torsade+de+pointes%3A%E2%80%89+an+overview"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm050563rb00030_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DDe%2BPonti%26aufirst%3DF.%26atitle%3DSafety%2520of%2520nonantiarrhythmic%2520drugs%2520that%2520prolong%2520the%2520QT%2520interval%2520or%2520induce%2520torsade%2520de%2520pointes%253A%25E2%2580%2589%2520an%2520overview%26jtitle%3DDrug%2520Safety%26date%3D2002%26volume%3D25%26spage%3D286" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm050563rb00031"><div class="reference"><strong class="refLabel"><a href="#jm050563rb00031" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-jm050563rb00031_1"><span><span class="NLM_string-ref">Roy, M.; Dumaine, R.; Brown, A. M. HERG, a primary human ventricular target of the nonsedating antihistamine terfenadine. <i>  Circulation </i><b>1996</b>,<i> 94</i>, 817−823.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=94&publication_year=1996&pages=823&author=M.+Roy&author=R.+Dumaine&author=A.+M+Brown&title=HERG%2C+a+primary+human+ventricular+target+of+the+nonsedating+antihistamine+terfenadine"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm050563rb00031_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DRoy%26aufirst%3DM.%26atitle%3DHERG%252C%2520a%2520primary%2520human%2520ventricular%2520target%2520of%2520the%2520nonsedating%2520antihistamine%2520terfenadine%26jtitle%3DCirculation%26date%3D1996%26volume%3D94%26spage%3D823" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm050563rb00032"><div class="reference"><strong class="refLabel"><a href="#jm050563rb00032" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-jm050563rb00032_1"><span><span class="NLM_string-ref">Wang, Y. D.; Miller, K.; Boschelli, D. H.; Ye, F.; Wu, B. et al. Inhibitors of src tyrosine kinase:  the preparation and structure−activity relationship of 4-anilino-3-cyanoquinolines and 4-anilinoquinazolines. <i>  Bioorg. Med. </i><i>Chem. Lett. </i><b>2000</b>,<i> 10</i>, 2477−2480.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2000&pages=2480&author=Y.+D.+Wang&author=K.+Miller&author=D.+H.+Boschelli&author=F.+Ye&author=B+Wu&title=et+al.+Inhibitors+of+src+tyrosine+kinase%3A%E2%80%89+the+preparation+and+structure%E2%88%92activity+relationship+of+4-anilino-3-cyanoquinolines+and+4-anilinoquinazolines"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm050563rb00032_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DY.%2BD.%26atitle%3Det%2520al.%2520Inhibitors%2520of%2520src%2520tyrosine%2520kinase%253A%25E2%2580%2589%2520the%2520preparation%2520and%2520structure%25E2%2588%2592activity%2520relationship%2520of%25204-anilino-3-cyanoquinolines%2520and%25204-anilinoquinazolines%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2000%26volume%3D10%26spage%3D2480" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm050563rb00033"><div class="reference"><strong class="refLabel"><a href="#jm050563rb00033" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-jm050563rb00033_1"><span><span class="NLM_string-ref">Trumpp-Kallmeyer, S.; Rubin, J. R.; Humblet, C.; Hamby, J. M.; Showalter, H. D. H. Development of a binding model to protein tyrosine kinases for substituted pyrido[2,3-d]pyrimidine inhibitors. <i>  J. Med. Chem. </i><b>1998</b>,<i> 41</i>, 1752−1763.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=1998&pages=1763&author=S.+Trumpp-Kallmeyer&author=J.+R.+Rubin&author=C.+Humblet&author=J.+M.+Hamby&author=H.+D.+H+Showalter&title=Development+of+a+binding+model+to+protein+tyrosine+kinases+for+substituted+pyrido%5B2%2C3-d%5Dpyrimidine+inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm050563rb00033_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DTrumpp-Kallmeyer%26aufirst%3DS.%26atitle%3DDevelopment%2520of%2520a%2520binding%2520model%2520to%2520protein%2520tyrosine%2520kinases%2520for%2520substituted%2520pyrido%255B2%252C3-d%255Dpyrimidine%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1998%26volume%3D41%26spage%3D1763" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm050563rb00034"><div class="reference"><strong class="refLabel"><a href="#jm050563rb00034" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-jm050563rb00034_1"><span><span class="NLM_string-ref">Traxler, P.; Green, J.; Mett, H.; Sequin, U.; P., F. Use of a pharmacophore model for the design of EGFR tyrosine kinase inhibitors:  isoflavones and 3-phenyl-4(1H)-quinolones. <i>  J. Med. Chem. </i><b>1999</b>,<i> 42</i>, 1018−1026.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=1999&pages=1026&author=P.+Traxler&author=J.+Green&author=H.+Mett&author=U.+Sequin&title=Use+of+a+pharmacophore+model+for+the+design+of+EGFR+tyrosine+kinase+inhibitors%3A%E2%80%89+isoflavones+and+3-phenyl-4%281H%29-quinolones"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm050563rb00034_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DTraxler%26aufirst%3DP.%26atitle%3DUse%2520of%2520a%2520pharmacophore%2520model%2520for%2520the%2520design%2520of%2520EGFR%2520tyrosine%2520kinase%2520inhibitors%253A%25E2%2580%2589%2520isoflavones%2520and%25203-phenyl-4%25281H%2529-quinolones%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1999%26volume%3D42%26spage%3D1026" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm050563rb00035"><div class="reference"><strong class="refLabel"><a href="#jm050563rb00035" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-jm050563rb00035_1"><span><span class="NLM_string-ref">Traxler, P.; Bold, G.; Frei, J.; Lang, M.; Lydon, N. et al. Use of a Pharmacophore Model for the Design of EGF−R Tyrosine Kinase Inhibitors:  4-(Phenylamino)pyrazolo[3,4-d]pyrimidines. <i>  J. Med. Chem. </i><b>1997</b>,<i> 40</i>, 3601−3616.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=1997&pages=3616&author=P.+Traxler&author=G.+Bold&author=J.+Frei&author=M.+Lang&author=N+Lydon&title=et+al.+Use+of+a+Pharmacophore+Model+for+the+Design+of+EGF%E2%88%92R+Tyrosine+Kinase+Inhibitors%3A%E2%80%89+4-%28Phenylamino%29pyrazolo%5B3%2C4-d%5Dpyrimidines"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm050563rb00035_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DTraxler%26aufirst%3DP.%26atitle%3Det%2520al.%2520Use%2520of%2520a%2520Pharmacophore%2520Model%2520for%2520the%2520Design%2520of%2520EGF%25E2%2588%2592R%2520Tyrosine%2520Kinase%2520Inhibitors%253A%25E2%2580%2589%25204-%2528Phenylamino%2529pyrazolo%255B3%252C4-d%255Dpyrimidines%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1997%26volume%3D40%26spage%3D3616" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm050563rb00036"><div class="reference"><strong class="refLabel"><a href="#jm050563rb00036" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-jm050563rb00036_1"><span><span class="NLM_string-ref">Klutchko, S. R.; Hamby, J. M.; Boschelli, D. H.; Wu, Z.; Kraker, A. J. et al. 2-Substituted Aminopyrido[2,3-d]pyrimidin-7(8H)-ones. Structure−Activity Relationships Against Selected Tyrosine Kinases and in Vitro and in Vivo Anticancer Activity. <i>  J. Med. Chem. </i><b>1998</b>,<i> 41</i>, 3276−3292.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=1998&pages=3292&author=S.+R.+Klutchko&author=J.+M.+Hamby&author=D.+H.+Boschelli&author=Z.+Wu&author=A.+J+Kraker&title=et+al.+2-Substituted+Aminopyrido%5B2%2C3-d%5Dpyrimidin-7%288H%29-ones.+Structure%E2%88%92Activity+Relationships+Against+Selected+Tyrosine+Kinases+and+in+Vitro+and+in+Vivo+Anticancer+Activity"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm050563rb00036_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DKlutchko%26aufirst%3DS.%2BR.%26atitle%3Det%2520al.%25202-Substituted%2520Aminopyrido%255B2%252C3-d%255Dpyrimidin-7%25288H%2529-ones.%2520Structure%25E2%2588%2592Activity%2520Relationships%2520Against%2520Selected%2520Tyrosine%2520Kinases%2520and%2520in%2520Vitro%2520and%2520in%2520Vivo%2520Anticancer%2520Activity%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1998%26volume%3D41%26spage%3D3292" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm050563rb00037"><div class="reference"><strong class="refLabel"><a href="#jm050563rb00037" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-jm050563rb00037_1"><span><span class="NLM_string-ref">Hamby, J. M.; Connolly, C. J.; Schroeder, M. C.; Winters, R. T.; Showalter, H. D. et al. Structure−activity relationships for a novel series of pyrido[2,3-d]pyrimidine tyrosine kinase inhibitors. <i>  J. Med. Chem. </i><b>1997</b>, <i>40</i>, 2296−2303.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=1997&pages=2303&author=J.+M.+Hamby&author=C.+J.+Connolly&author=M.+C.+Schroeder&author=R.+T.+Winters&author=H.+D+Showalter&title=et+al.+Structure%E2%88%92activity+relationships+for+a+novel+series+of+pyrido%5B2%2C3-d%5Dpyrimidine+tyrosine+kinase+inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm050563rb00037_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DHamby%26aufirst%3DJ.%2BM.%26atitle%3Det%2520al.%2520Structure%25E2%2588%2592activity%2520relationships%2520for%2520a%2520novel%2520series%2520of%2520pyrido%255B2%252C3-d%255Dpyrimidine%2520tyrosine%2520kinase%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1997%26volume%3D40%26spage%3D2303" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm050563rb00038"><div class="reference"><strong class="refLabel"><a href="#jm050563rb00038" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-jm050563rb00038_1"><span><span class="NLM_string-ref">Boschelli, D. H.; Wang, D. Y.; Ye, F.; Yamashita, A.; Zhang, N. et al. Inhibition of Src kinase activity by 4-anilino-7-thienyl-3-quinolinecarbonitriles. <i>  Bioorg. Med. Chem. Lett. </i><b>2002</b>,<i> 12</i>, 2011−2014.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2002&pages=2014&author=D.+H.+Boschelli&author=D.+Y.+Wang&author=F.+Ye&author=A.+Yamashita&author=N+Zhang&title=et+al.+Inhibition+of+Src+kinase+activity+by+4-anilino-7-thienyl-3-quinolinecarbonitriles"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm050563rb00038_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBoschelli%26aufirst%3DD.%2BH.%26atitle%3Det%2520al.%2520Inhibition%2520of%2520Src%2520kinase%2520activity%2520by%25204-anilino-7-thienyl-3-quinolinecarbonitriles%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2002%26volume%3D12%26spage%3D2014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm050563rb00039"><div class="reference"><strong class="refLabel"><a href="#jm050563rb00039" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-jm050563rb00039_1"><span><span class="NLM_string-ref">Boschelli, D. H.; Wu, Z.; Klutchko, S. R.; Showalter, H. D.; Hamby, J. M. et al. Synthesis and tyrosine kinase inhibitory activity of a series of 2-amino-8H-pyrido[2,3-d]pyrimidines:  identification of potent, selective platelet-derived growth factor receptor tyrosine kinase inhibitors. <i>  J. Med. </i><i>Chem. </i><b>1998</b>,<i> 41</i>, 4365−4377.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=1998&pages=4377&author=D.+H.+Boschelli&author=Z.+Wu&author=S.+R.+Klutchko&author=H.+D.+Showalter&author=J.+M+Hamby&title=et+al.+Synthesis+and+tyrosine+kinase+inhibitory+activity+of+a+series+of+2-amino-8H-pyrido%5B2%2C3-d%5Dpyrimidines%3A%E2%80%89+identification+of+potent%2C+selective+platelet-derived+growth+factor+receptor+tyrosine+kinase+inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm050563rb00039_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBoschelli%26aufirst%3DD.%2BH.%26atitle%3Det%2520al.%2520Synthesis%2520and%2520tyrosine%2520kinase%2520inhibitory%2520activity%2520of%2520a%2520series%2520of%25202-amino-8H-pyrido%255B2%252C3-d%255Dpyrimidines%253A%25E2%2580%2589%2520identification%2520of%2520potent%252C%2520selective%2520platelet-derived%2520growth%2520factor%2520receptor%2520tyrosine%2520kinase%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1998%26volume%3D41%26spage%3D4377" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm050563rb00040"><div class="reference"><div class="NLM_citation" id="rightTab-jm050563rb00040_1"><span><span class="NLM_string-ref">Thompson, A. M.; Connolly, C. J.; Hamby, J. M.; Boushelle, S.; Hartl, B. G. et al. 3-(3,5-Dimethoxyphenyl)-1,6-naphthyridine-2,7-diamines and Related 2-Urea Derivatives Are Potent and Selective Inhibitors of the FGF Receptor-1 Tyrosine Kinase. <i>  J. Med. Chem. </i><b>2000</b>,<i> 43</i>, 4200−4211.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2000&pages=4211&author=A.+M.+Thompson&author=C.+J.+Connolly&author=J.+M.+Hamby&author=S.+Boushelle&author=B.+G+Hartl&title=et+al.+3-%283%2C5-Dimethoxyphenyl%29-1%2C6-naphthyridine-2%2C7-diamines+and+Related+2-Urea+Derivatives+Are+Potent+and+Selective+Inhibitors+of+the+FGF+Receptor-1+Tyrosine+Kinase"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm050563rb00040_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DThompson%26aufirst%3DA.%2BM.%26atitle%3Det%2520al.%25203-%25283%252C5-Dimethoxyphenyl%2529-1%252C6-naphthyridine-2%252C7-diamines%2520and%2520Related%25202-Urea%2520Derivatives%2520Are%2520Potent%2520and%2520Selective%2520Inhibitors%2520of%2520the%2520FGF%2520Receptor-1%2520Tyrosine%2520Kinase%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2000%26volume%3D43%26spage%3D4211" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm050563rb00041"><div class="reference"><strong class="refLabel"><a href="#jm050563rb00041" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-jm050563rb00041_1"><span><span class="NLM_string-ref">Missbach, M.; Altmann, E.; Widler, L.; Susa, M.; Buchdunger, E. et al. Substituted 5,7-diphenylpyrrolo[2,3-d]pyrimidines:   6990  potent inhibitors of the tyrosine kinase c-Src. <i>  Bioorg. Med. Chem.  Lett. </i><b>2000</b>,<i> 10</i>, 945−949.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2000&pages=949&author=M.+Missbach&author=E.+Altmann&author=L.+Widler&author=M.+Susa&author=E+Buchdunger&title=et+al.+Substituted+5%2C7-diphenylpyrrolo%5B2%2C3-d%5Dpyrimidines%3A%E2%80%89++6990++potent+inhibitors+of+the+tyrosine+kinase+c-Src"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm050563rb00041_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMissbach%26aufirst%3DM.%26atitle%3Det%2520al.%2520Substituted%25205%252C7-diphenylpyrrolo%255B2%252C3-d%255Dpyrimidines%253A%25E2%2580%2589%2520%25206990%2520%2520potent%2520inhibitors%2520of%2520the%2520tyrosine%2520kinase%2520c-Src%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520%2520Lett.%26date%3D2000%26volume%3D10%26spage%3D949" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm050563rb00042"><div class="reference"><strong class="refLabel"><a href="#jm050563rb00042" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-jm050563rb00042_1"><span><span class="NLM_string-ref">Connolly, C. J. C.; Hamby, J. M.; Schroeder, M. C.; Barvian, M.; Lu, G. H. et al. Discovery and structure−activity studies of a novel series of pyrido[2,3-d]pyrimidine tyrosine kinase inhibitors. <i>  Bioorg. Med. Chem. </i><i>Lett. </i><b>1997</b>,<i> 7</i>, 2415−2420.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=1997&pages=2420&author=C.+J.+C.+Connolly&author=J.+M.+Hamby&author=M.+C.+Schroeder&author=M.+Barvian&author=G.+H+Lu&title=et+al.+Discovery+and+structure%E2%88%92activity+studies+of+a+novel+series+of+pyrido%5B2%2C3-d%5Dpyrimidine+tyrosine+kinase+inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm050563rb00042_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DConnolly%26aufirst%3DC.%2BJ.%2BC.%26atitle%3Det%2520al.%2520Discovery%2520and%2520structure%25E2%2588%2592activity%2520studies%2520of%2520a%2520novel%2520series%2520of%2520pyrido%255B2%252C3-d%255Dpyrimidine%2520tyrosine%2520kinase%2520inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D1997%26volume%3D7%26spage%3D2420" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm050563rb00043"><div class="reference"><strong class="refLabel"><a href="#jm050563rb00043" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-jm050563rb00043_1"><span><span class="NLM_string-ref">Toledo, L. M.; Lydon, N. B.; Elbaum, D. The structure-based design of ATP-site directed protein kinase inhibitors. <i>  Curr. Med. Chem.</i><b>1999</b>,<i> 6</i>, 775−805.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=1999&pages=805&author=L.+M.+Toledo&author=N.+B.+Lydon&author=D+Elbaum&title=The+structure-based+design+of+ATP-site+directed+protein+kinase+inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm050563rb00043_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DToledo%26aufirst%3DL.%2BM.%26atitle%3DThe%2520structure-based%2520design%2520of%2520ATP-site%2520directed%2520protein%2520kinase%2520inhibitors%26jtitle%3DCurr.%2520Med.%2520Chem.%26date%3D1999%26volume%3D6%26spage%3D805" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm050563rb00044"><div class="reference"><strong class="refLabel"><a href="#jm050563rb00044" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-jm050563rb00044_1"><span><span class="NLM_string-ref">Catalyst; 4.8 ed.; Accelrys, Inc.:  San Diego, CA.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+Catalyst%3B+4.8+ed.%3B+Accelrys%2C+Inc.%3A%E2%80%89+San+Diego%2C+CA.+Catalyst%3B+4.8+ed.%3B+Accelrys%2C+Inc.%3A%E2%80%89+San+Diego%2C+CA."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm050563rb00044_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dfootnote%26sid%3Dliteratum%253Aachs" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm050563rb00045"><div class="reference"><strong class="refLabel"><a href="#jm050563rb00045" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-jm050563rb00045_1"><span><span class="NLM_string-ref">Cerius2; 4.9 ed.; Accelrys Inc.:  San Diego, CA.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+Cerius2%3B+4.9+ed.%3B+Accelrys+Inc.%3A%E2%80%89+San+Diego%2C+CA.+Cerius2%3B+4.9+ed.%3B+Accelrys+Inc.%3A%E2%80%89+San+Diego%2C+CA."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm050563rb00045_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dfootnote%26sid%3Dliteratum%253Aachs" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm050563rb00046"><div class="reference"><strong class="refLabel"><a href="#jm050563rb00046" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-jm050563rb00046_1"><span><span class="NLM_string-ref">Ekins, S.; Crumb, W. J.; Sarazan, R. D.; Wikel, J. H.; Wrighton, S. A. Three-dimensional quantitative structure−activity relationship for inhibition of human ether-a-go-go-related gene potassium channel. <i>  J. </i><i>Pharmacol. Exp. Ther. </i><b>2002</b>,<i> 301</i>, 427−434.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=301&publication_year=2002&pages=434&author=S.+Ekins&author=W.+J.+Crumb&author=R.+D.+Sarazan&author=J.+H.+Wikel&author=S.+A+Wrighton&title=Three-dimensional+quantitative+structure%E2%88%92activity+relationship+for+inhibition+of+human+ether-a-go-go-related+gene+potassium+channel"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm050563rb00046_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DEkins%26aufirst%3DS.%26atitle%3DThree-dimensional%2520quantitative%2520structure%25E2%2588%2592activity%2520relationship%2520for%2520inhibition%2520of%2520human%2520ether-a-go-go-related%2520gene%2520potassium%2520channel%26jtitle%3DJ.%2520Pharmacol.%2520Exp.%2520Ther.%26date%3D2002%26volume%3D301%26spage%3D434" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm050563rb00047"><div class="reference"><strong class="refLabel"><a href="#jm050563rb00047" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-jm050563rb00047_1"><span><span class="NLM_string-ref">Schindler, T.; Sicheri, F.; Pico, A.; Gazit, A.; Levitzki, A. et al. Crystal structure of Hck in complex with a Src family selective tyrosine kinase inhibitor. <i>  Mol. Cell </i><b>1999</b>,<i> 3</i>, 639−648.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=1999&pages=648&author=T.+Schindler&author=F.+Sicheri&author=A.+Pico&author=A.+Gazit&author=A+Levitzki&title=et+al.+Crystal+structure+of+Hck+in+complex+with+a+Src+family+selective+tyrosine+kinase+inhibitor"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm050563rb00047_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSchindler%26aufirst%3DT.%26atitle%3Det%2520al.%2520Crystal%2520structure%2520of%2520Hck%2520in%2520complex%2520with%2520a%2520Src%2520family%2520selective%2520tyrosine%2520kinase%2520inhibitor%26jtitle%3DMol.%2520Cell%26date%3D1999%26volume%3D3%26spage%3D648" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm050563rb00048"><div class="reference"><strong class="refLabel"><a href="#jm050563rb00048" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-jm050563rb00048_1"><span><span class="NLM_string-ref">Stamos, J.; Sliwkowski, M. X.; Eigenbrot, C. Structure of the epidermal growth factor receptor kinase domain alone and in complex with a 4-anilinoquinazoline inhibitor. <i>  J. Biol. Chem. </i><b>2002</b>,<i> 277</i>, 46265−46272.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=277&publication_year=2002&pages=46272&author=J.+Stamos&author=M.+X.+Sliwkowski&author=C+Eigenbrot&title=Structure+of+the+epidermal+growth+factor+receptor+kinase+domain+alone+and+in+complex+with+a+4-anilinoquinazoline+inhibitor"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm050563rb00048_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DStamos%26aufirst%3DJ.%26atitle%3DStructure%2520of%2520the%2520epidermal%2520growth%2520factor%2520receptor%2520kinase%2520domain%2520alone%2520and%2520in%2520complex%2520with%2520a%25204-anilinoquinazoline%2520inhibitor%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2002%26volume%3D277%26spage%3D46272" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm050563rb00049"><div class="reference"><strong class="refLabel"><a href="#jm050563rb00049" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-jm050563rb00049_1"><span><span class="NLM_string-ref">Nagar, B.; Bornmann, W. G.; Pellicena, P.; Schindler, T.; Veach, D. R. et al. Crystal structures of the kinase domain of c-Abl in complex with the small molecule inhibitors PD173955 and imatinib (STI-571). <i>  Cancer Res. </i><b>2002</b>, <i>62</i>, 4236−4243.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2002&pages=4243&author=B.+Nagar&author=W.+G.+Bornmann&author=P.+Pellicena&author=T.+Schindler&author=D.+R+Veach&title=et+al.+Crystal+structures+of+the+kinase+domain+of+c-Abl+in+complex+with+the+small+molecule+inhibitors+PD173955+and+imatinib+%28STI-571%29"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm050563rb00049_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DNagar%26aufirst%3DB.%26atitle%3Det%2520al.%2520Crystal%2520structures%2520of%2520the%2520kinase%2520domain%2520of%2520c-Abl%2520in%2520complex%2520with%2520the%2520small%2520molecule%2520inhibitors%2520PD173955%2520and%2520imatinib%2520%2528STI-571%2529%26jtitle%3DCancer%2520Res.%26date%3D2002%26volume%3D62%26spage%3D4243" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm050563rb00050"><div class="reference"><div class="NLM_citation" id="rightTab-jm050563rb00050_1"><span><span class="NLM_string-ref">de Laszlo, S. E.; Hacker, C.; Li, B.; Kim, D.; MacCoss, M. et al. Potent, orally absorbed glucagon receptor antagonists. <i>  Bioorg. Med. Chem. Lett. </i><b>1999</b>,<i> 9</i>, 641−646.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=1999&pages=646&author=S.+E.+de+Laszlo&author=C.+Hacker&author=B.+Li&author=D.+Kim&author=M+MacCoss&title=et+al.+Potent%2C+orally+absorbed+glucagon+receptor+antagonists"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm050563rb00050_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3Dde%2BLaszlo%26aufirst%3DS.%2BE.%26atitle%3Det%2520al.%2520Potent%252C%2520orally%2520absorbed%2520glucagon%2520receptor%2520antagonists%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D1999%26volume%3D9%26spage%3D646" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Fjm050563r&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Barticle%3Aarticle%3A10.1021%2Fjm050563r%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2005.48.issue-22%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/jm050563r" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                467KB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"677aa571d9443dc4","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
