
<html lang="en"     class="pb-page"  data-request-id="11150996-1fd5-4015-8fc2-02aa62be1d54"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;issue:issue:10.1021/jmcmar.2011.54.issue-22;wgroup:string:ACHS website Group;article:article:10.1021/jm200452d;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="dc.Title" content="Matched Molecular Pairs as a Medicinal Chemistry Tool" /></meta><meta name="dc.Creator" content="Ed  Griffen" /></meta><meta name="dc.Creator" content="Andrew G.  Leach" /></meta><meta name="dc.Creator" content="Graeme R.  Robb" /></meta><meta name="dc.Creator" content="Daniel J.  Warner" /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="September 22, 2011" /></meta><meta name="dc.Type" content="review-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/jm200452d" /></meta><meta name="dc.Language" content="EN" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2011 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Fjm200452d" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Fjm200452d" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Fjm200452d" /></link>
        
    
    

<title>Matched Molecular Pairs as a Medicinal Chemistry Tool | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/jm200452d" /></meta><meta property="og:title" content="Matched Molecular Pairs as a Medicinal Chemistry Tool" /></meta><meta property="og:image" content="https://pubs.acs.org/pb-assets/images/CoverImages_2by1Scaled/jmcmar-1595795902140.png" /></meta><meta property="og:description" content="" /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/jm200452d"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/jm200452d">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-14/achre4.2021.54.issue-14/20210720/achre4.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-29/aamick.2021.13.issue-29/20210728/aamick.2021.13.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-14/acncdm.2021.12.issue-14/20210721/acncdm.2021.12.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-7/amlcef.2021.3.issue-7/20210705/amlcef.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-29/acsodf.2021.6.issue-29/20210727/acsodf.2021.6.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-29/ascecg.2021.9.issue-29/20210726/ascecg.2021.9.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-29/ancham.2021.93.issue-29/20210727/ancham.2021.93.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-29/bichaw.2021.60.issue-29/20210727/bichaw.2021.60.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-27/cgeabj.2021.99.issue-27/20210726-01/cgeabj.2021.99.issue-27.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-7/cgdefu.2021.21.issue-7/20210707/cgdefu.2021.21.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-14/esthag.2021.55.issue-14/20210720/esthag.2021.55.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-29/iecred.2021.60.issue-29/20210728/iecred.2021.60.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-14/inocaj.2021.60.issue-14/20210719/inocaj.2021.60.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-29/jafcau.2021.69.issue-29/20210728/jafcau.2021.69.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-29/jacsat.2021.143.issue-29/20210728/jacsat.2021.143.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-7/jamsef.2021.32.issue-7/20210707/jamsef.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-29/langd5.2021.37.issue-29/20210727/langd5.2021.37.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-7/mpohbp.2021.18.issue-7/20210705/mpohbp.2021.18.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Fjm200452d&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Fjm200452d&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/jm200452d&amp;href=/doi/10.1021/jm200452d" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2011</span><span class="cit-fg-volume">, 54</span><span class="cit-fg-issue">, 22</span><span class="cit-fg-pageRange">, 7739-7750</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/54/22" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><span class="content-navigation__contentType">Perspective</span><a href="/doi/10.1021/jm2004128" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Matched Molecular Pairs as a Medicinal Chemistry Tool<a class="ref internalNav" href="#ded" aria-label=""></a></span></h1><h2 class="article_header-subtitle"><span class="hlFld-SubTitle">Miniperspective</span></h2><ul class="loa non-jats-loa"><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Ed++Griffen">Ed Griffen</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Andrew+G.++Leach">Andrew G. Leach</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Graeme+R.++Robb">Graeme R. Robb</a></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Daniel+J.++Warner">Daniel J. Warner</a></span><span class="author-xref-symbol author-aff-symbol"><sup>∥</sup></span></span></li><span class="comma-separator"></span></ul><div class="affiliations"><div class="view-affs"><span class="title">View Author Information</span><i class="caret-icon icon-angle-down"></i></div><div class="all-aff-infos hlFld-Affiliation"><div class="aff-info" id="aff1"><span class="aff-text">Oncology Innovative Medicines Unit, AstraZeneca Pharmaceuticals, Mereside, Alderley Park, Macclesfield, SK10 4TG, U.K.</span></div><div class="aff-info" id="aff2"><span class="aff-text">Cardiovascular and Gastrointestinal Innovative Medicines Unit, AstraZeneca Pharmaceuticals, 30S373 Mereside, Alderley Park, Macclesfield, SK10 4TG, U.K.</span></div><div class="aff-info" id="aff3"><span class="aff-text">Department of Medicinal Chemistry, AstraZeneca R&D Montreal, Montreal, Quebec, H4S 1Z9, Canada</span></div><div class="corresp-info">Phone: +44 1625 231853. E-mail: <a href="/cdn-cgi/l/email-protection#c9a8a7adbbacbee7a5aca8aaa189a8babdbba8b3aca7acaaa8e7aaa6a4"><span class="__cf_email__" data-cfemail="5f3e313b2d3a2871333a3e3c371f3e2c2b2d3e253a313a3c3e713c3032">[email protected]</span></a></div></div></div><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Fjm200452d&amp;href=/doi/10.1021%2Fjm200452d" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2011</span></span><span class="cit-volume">, 54</span><span class="cit-issue">, 22</span><span class="cit-pageRange">, 7739–7750</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">September 22, 2011</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>14 April 2011</li><li><span class="item_label"><b>Published</b> online</span>22 September 2011</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 24 November 2011</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/jm200452d" title="DOI URL">https://doi.org/10.1021/jm200452d</a></div><div class="article_header-article-copyright"><strong>Copyright © 2011 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D7739%26pageCount%3D12%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DEd%2BGriffen%252C%2BAndrew%2BG.%2BLeach%252C%2BGraeme%2BR.%2BRobb%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D54%26issueNum%3D22%26contentID%3Djm200452d%26title%3DMatched%2BMolecular%2BPairs%2Bas%2Ba%2BMedicinal%2BChemistry%2BTool%26numPages%3D12%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D7750%26publicationDate%3DNovember%2B2011">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/jm200452d"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">6673</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">179</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/jm200452d" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Matched Molecular Pairs as a Medicinal Chemistry Tool&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Ed&quot;,&quot;last_name&quot;:&quot;Griffen&quot;},{&quot;first_name&quot;:&quot;Andrew&quot;,&quot;last_name&quot;:&quot;G. Leach&quot;},{&quot;first_name&quot;:&quot;Graeme&quot;,&quot;last_name&quot;:&quot;R. Robb&quot;},{&quot;first_name&quot;:&quot;Daniel&quot;,&quot;last_name&quot;:&quot;J. Warner&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2011&quot;,&quot;month&quot;:&quot;09&quot;,&quot;day&quot;:&quot;22&quot;,&quot;issue&quot;:&quot;22&quot;,&quot;volume&quot;:&quot;54&quot;,&quot;pages&quot;:&quot;7739-7750&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/jm200452d&quot;},&quot;abstract&quot;:&quot;&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Fjm200452d&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=jm200452d" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Fjm200452d&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=jm200452d" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Fjm200452d&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=jm200452d" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Fjm200452d&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=jm200452d&amp;href=/doi/10.1021/jm200452d" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/jm200452d" /></input><a href="/doi/pdf/10.1021/jm200452d" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (3 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/jm200452d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Fjm200452d%26sid%3Dliteratum%253Aachs%26pmid%3D21936582%26genre%3Darticle%26aulast%3DGriffen%26date%3D2011%26atitle%3DMatched%2BMolecular%2BPairs%2Bas%2Ba%2BMedicinal%2BChemistry%2BTool%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D54%26issue%3D22%26spage%3D7739%26epage%3D7750%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291350" title="Assays">Assays</a>,</li><li><a href="/action/doSearch?ConceptID=291454" title="Solubility">Solubility</a>,</li><li><a href="/action/doSearch?ConceptID=291583" title="Mathematical methods">Mathematical methods</a>,</li><li><a href="/action/doSearch?ConceptID=291465" title="Structure activity relationship">Structure activity relationship</a>,</li><li><a href="/action/doSearch?ConceptID=291754" title="Molecules">Molecules</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/54/22" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-22/jmcmar.2011.54.issue-22/production/jmcmar.2011.54.issue-22.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div class="extra-info-sec articleNote"><h4><span>†</span> Dedication</h4><p class="last">This article is dedicated to Dr. Andy Barker on the occasion of his recognition by the American Chemical Society as a Hero of Chemistry and of his retirement from AstraZeneca.</p></div><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_57102" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_57102" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">At the very heart of the role of a medicinal chemist or drug designer is the ability to link chemical structure to molecular properties. This is the structure–activity relationship (SAR) or structure–property relationship (SPR). At the start of understanding SAR is the idea that “adding” a particular group “adds” a degree of potency, stability, or other biological property. Many chemists will use this language, for example: “adding a chlorine at the 2-position adds 3-fold potency but lowers solubility by 10-fold”. This language is derived from the training of many medicinal chemists being based in synthetic chemistry where thinking literally could be to “add” the chlorine. It is a small conceptual step to then think in terms of a virtual transformation where, although it cannot be done in one step synthetically, the two bioactive compounds can be considered as either end of a transformation. Another conceptual step is to then consider all the occasions a particular change has been tried to be examples of the same transform and to ask questions such as “<i>How much</i> potency does adding chlorine at the 2-position generally give?” and “<i>What is the general effect</i> on solubility?”. The equivalent but systematic approach is to describe the initial compound and the new compound as a “matched pair” linked by the addition of a chlorine at the 2-position and then to look at the statistics of the effects of this structural change. Six years ago one of our colleagues coined the phrase “matched molecular pairs analysis” (MMPA) to describe any systematic method of identifying matched molecular pairs from a set of compounds and determining the property change associated.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> One of the key advantages of MMPA over other data analysis and modeling techniques is that it deals directly with chemistry and measured data, ensuring clear interpretation of the results. In recent times MMPA has been usefully applied by numerous workers in a range of disciplines within both academia and the pharmaceutical industry.</div><div class="NLM_p">Drug discovery projects frequently discover that a particular structural change causes a change (for good or ill) to a property of interest, as described above. Two contrasting fates often follow. In the first case, the effect is described to others and what had been an observation in one series is passed on as if a general rule of thumb, whether general or not. In the second case, the effect is not disclosed beyond the original project and the knowledge, either explicit or tacet,<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> does not get passed on. MMPA provides a means to both test the generality of rules of thumb and to extract understanding that might otherwise be lost.</div><div class="NLM_p">The four authors of this Miniperspective have been involved in developing and implementing some variations on the original theme of MMPA and felt this to be the time to highlight some of these and to point toward future developments. All of us are involved in the process of drug design, and it is within this context that we will set our comments. In outline, we shall introduce matched molecular pair analysis and highlight why it fits so well with the kind of problem solving that is required in the process of drug design. This will be followed by some comments concerning when the application of the methodology is appropriate. Then in a section intended for those with more computational interests, we will describe the various ways that have been devised for identifying such matched pairs. Finally, we will present our expectations for where this approach might be developed in the future.</div><div class="NLM_p last">Throughout the paper, we have elected to use a publicly available data set to illustrate some of our comments: the ChEMBL database of inhibitors of the epidermal growth factor receptor tyrosine kinase (EGFR).<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> This consists of 2899 measurements of an IC<sub>50</sub> annotated as a binding measurement on 2348 unique compounds. These measurements are drawn from a number of sources, having been abstracted from the scientific literature. We use examples based on this mixed data set purely for illustrative purposes.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Defining Matched Molecular Pair Analysis and Its Relationship to Other Established Approaches</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_52474" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_52474" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Matched pairs have generally been defined similarly to the definition proffered 5 years ago by one of us as “molecules that differ only by a particular, well-defined, structural transformation”.<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> It is worth noting that while this label emphasizes the two molecules that are related, others have emphasized the transformation linking them.<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> Although the labels vary, the key concept is that the effect of small structural differences between molecules is more easily predicted than the absolute value for the activity or property of each molecule, which is the approach taken in the field of quantitative structure–activity relationships (QSAR) and quantitative structure–property relationships (QSPR). Viewed like this, MMPA and QSAR/QSPR are related in a similar fashion as free energy perturbation methods are to the scoring of bound poses in the field of prediction of protein–ligand binding energies. In that arena as here, the methods focusing on differences as opposed to absolute values tend to be more successful. The origin of this difference in performance owes much to the cancellation of errors and to the averaging out of localized effects present in individual cases.</div><div class="NLM_p">The quest to quantify the effect of structural changes upon properties of pharmaceutical relevance has emerged as a related discipline to the identification of bioisosteres.<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> These can be defined as “the replacement of a part of a bioactive molecule with a substructure that is similar in size and exhibits similar properties.”<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> While the emphasis in bioisosterism has usually been to find groups that are unlikely to change the primary binding potency of a compound, the concept applies quite readily to finding molecular transformations that are likely to leave any given property unchanged. Such transformations are naturally and inevitably identified during the course of most matched molecular pair analyses. For instance, when studying matched molecular pairs that occur in druglike compounds, Sheridan found that phenyl is often exchanged with thiophene. In the context of the EGFR data set, all the pairs in which this transformation has been imposed have been identified and are shown in Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a>. These show that within this data set this particular isostere is valid. For the overall data set, the mean change is −0.12 with a standard error in that mean of 0.16, indicating that an underlying mean change of 0 is not contraindicated. For individual cases, the error bars on the plot indicate 3-fold variation in the IC<sub>50</sub>, a typical experimental uncertainty; most cases do not show differences beyond what can be accurately determined.</div><figure id="fig1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-22/jm200452d/production/images/medium/jm-2011-00452d_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-22/jm200452d/production/images/large/jm-2011-00452d_0007.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Matched molecular pairs in the EGFR data set that differ by transformation of phenyl to thiophenyl. On the <i>x</i>-axis is plotted the pIC<sub>50</sub> value of the phenyl compound and on the <i>y</i>-axis the value for the thiophenyl. The red line is the 1:1 line. Error bars correspond to 3-fold variation in the IC<sub>50</sub> and indicate typical experimental uncertainty.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-22/jm200452d/production/images/large/jm-2011-00452d_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm200452d&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">MMPA might also be considered a relation of Free-Wilson analysis.<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> In this kind of analysis a least-squares fit is used to link different R-groups around a molecular core to contributions to a property of interest. The contrast between the two is illustrated in Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>. A notional series of compounds is illustrated as a simple substituted phenyl ring. Free–Wilson analysis is able to make predictions for the contribution of each of the five substituents at defined positions: A<sub>1</sub>, A<sub>2</sub>, A<sub>3</sub>, B<sub>1</sub>, and B<sub>2</sub>. Meanwhile, matched molecular pair analysis is able to estimate the effect of transforming A<sub>1</sub> to A<sub>2</sub> and of B<sub>1</sub> to B<sub>2</sub> and A<sub>2</sub> to A<sub>3</sub> but not A<sub>1</sub> to A<sub>3</sub>; no pairs exist in which two molecules differ only by the change A<sub>1</sub> to A<sub>3</sub>. The effect of the last change can only be inferred from the sum of the change caused by changing A<sub>1</sub> to A<sub>2</sub> and that of changing A<sub>2</sub> to A<sub>3</sub>. It has been demonstrated that in some cases this sum of pairwise effects is equivalent to the direct transformation.<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> Free–Wilson analysis is a powerful way of understanding the contribution various groups make within a well-defined part of chemical space; the ability to directly link A<sub>1</sub> to A<sub>3</sub> can be viewed as a strength of the analysis. However, if there is substantial interaction between A and B, then the link that is made may be misleading. MMPA would not make that link directly; unlike Free–Wilson analysis additivity of SAR is neither assumed nor required.<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> While there are relationships with the modeling techniques of Free and Wilson, no model is ever built from the data and the link back to measurement remains clear and unambiguous.<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a></div><figure id="fig2" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-22/jm200452d/production/images/medium/jm-2011-00452d_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-22/jm200452d/production/images/large/jm-2011-00452d_0001.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Example illustrating the contrast between Free–Wilson analysis and matched molecular pairs. A series of compounds with defined substitution positions is illustrated as a simple substituted phenyl ring.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-22/jm200452d/production/images/large/jm-2011-00452d_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm200452d&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">In close analogy to the approach of Topliss,<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> MMPA can also be used to derive small sets of substituents to probe a given property. Dossetter studied the groups to which phenyl substituents had been transformed (limited to substituted phenyls and unsubstituted pyridines) in a data set of microsomal metabolic clearance.<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> The range of changes that had historically been brought about by these transformations can set expectations for what might be expected of them in the future, and a representative subset can be picked. This subset should span microsomal clearance in a similar way to the Topliss set of substitution patterns that were selected to span electronic and lipophilic properties. This can guide the design of libraries to probe the changes in clearance that might be expected by modifying certain parts of a chemical series.</div><div class="NLM_p">QSAR or QSPR methods are also frequently used within the pharmaceutical industry and allow properties of the whole molecule (measured or calculated) to be related to the activity or property of interest.<a onclick="showRef(event, 'ref12 ref13'); return false;" href="javascript:void(0);" class="ref ref12 ref13">(12, 13)</a> Though many successful QSAR and QSPR models have been reported, these models have inherent disadvantages. By their very nature they tend to find the most generalized relationships that smooth over interesting substructural effects<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> and tend to be limited in the precision of prediction. Such generalizations in the prediction of primary potency, and other properties, are often insufficient for guiding design in a lead optimization phase. Precision and accuracy are further limited by the quality of descriptors used in the model. Although many thousands of descriptors can be generated, the value of each is not always clear and can lead to misleading relationships and chance correlations, depending on the data and modeling technique used.<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> Descriptors frequently describe chemical structure incompletely, and so information about the structure–activity relationship is lost.</div><div class="NLM_p">To illustrate the differences, the EGFR data set has been treated with a very simple QSAR approach and with MMPA. One of the most commonly occurring transformations in the EGFR data set is that of converting a <i>m</i>-bromoaniline group into a <i>m</i>-chloro-<i>p</i>-fluoroaniline, as shown in Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a>a. There are 34 occurences of this matched pair in the data set. Pairs were assigned a random number between 0 and 1, and where the value was less than 0.5 (18 pairs in this case), the <i>m</i>-chloro-<i>p</i>-fluoroaniline analogue was assigned to a test set. A training set was built in which these 18 <i>m</i>-chloro-<i>p</i>-fluoroaniline containing compounds were excluded. A QSAR model was built for the 2287 remaining compounds using 193 standard descriptors (40 compounds were not amenable to calculation of all of the descriptors, and 3 did not have IC<sub>50</sub> data) as computed by the in-house c-lab tool.<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> The model was built using a simple least-squares fitting approach in the JMP package.<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> An excellent fit is obtained for both the training set and the test set, as shown in parts b and c of Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a>. The training set achieves an <i>R</i><sup>2</sup> of 0.78 and RMSE of 0.66, while the 18-member test set achieves an RMSE of 0.74. All of these are perfectly reasonable values, suggesting a well-behaved QSAR model. By contrast, the 16 matched molecular pairs in the training set show the changes in pIC<sub>50</sub> illustrated in Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a>d. The distribution has a mean of −0.18 and a standard error in that mean of 0.18 and standard deviation of 0.73. Adding −0.18 to the pIC<sub>50</sub> for each of the 18 <i>m</i>-bromoaniline containing compounds corresponding to the test set led to the predicted pIC<sub>50</sub> values plotted in blue in Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a>e. The points marked with black crosses are the corresponding values for the <i>m</i>-bromoaniline from which each prediction is projected. The predictions are generally excellent; the RMSE for the test set is 0.55. The performance of the matched pairs prediction in the test set is unexpectedly better than in the training set, where the RMSE is 0.70. Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a> e illustrates why matched pairs tends to be more successful in terms of predictivity, as it shows that the challenge has been reduced. The potency of the <i>m</i>-bromoaniline compound provides a good starting point for predicting the potency of the transformed compound. Just as important, the outliers away from the matched pairs plot provide insight, as highlighted in many literature studies;<a onclick="showRef(event, 'ref4 ref18'); return false;" href="javascript:void(0);" class="ref ref4 ref18">(4, 18)</a> the outlier in Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a>e is compound <b>1</b>. Knowing that this is an outlier can be a spur to drive hypothesis generation, which in turn leads to design of new compounds or further testing of existing compounds.</div><figure id="fig3" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-22/jm200452d/production/images/medium/jm-2011-00452d_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-22/jm200452d/production/images/large/jm-2011-00452d_0002.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. (a) Structural change being studied for its effect upon pIC<sub>50</sub> against EGFR. (b) Measured values for a training set of compounds plotted against predictions from a simple least-squares QSAR model based upon 193 descriptors for 2287 compounds. (c) Measured values plotted against predictions for a test set of 18 compounds all containing the <i>m</i>-chloro-<i>p</i>-fluoroaniline using the QSAR model built to generate (b). (d) Distribution of changes in measured values between 16 training set molecular pairs differing only by the transformation of <i>m</i>-bromoaniline to <i>m</i>-chloro-<i>p</i>-fluoroaniline. (e) In blue are the measured values of the same test set as in (c) plotted against the prediction based on matched pairs, assuming that all compounds will experience a mean change of −0.18. In black crosses are plotted the measured values for the <i>m</i>-bromoanilines upon which these predictions are made.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-22/jm200452d/production/images/large/jm-2011-00452d_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm200452d&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">Although the overall quality of a QSAR model can be determined, the applicability of the model to specific chemistries cannot easily be assessed. Even if a “good” QSAR model that could accurately predict a given activity was discovered, on its own it can provide no information about what to make in the next cycle of drug design. This “inverse QSAR” problem is one to which MMPA is particularly suited. Changes in chemical structure are linked directly to changes in property; knowing what the desired change in property is, a suitable structural change can be selected. The lessons from the historical data allow medicinal chemists to make the best predictions possible about what particular structural changes will achieve in the future.</div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i4">Considerations When Applying Matched Molecular Pair Analysis</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_06413" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_06413" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">There are a number of issues that ought to be considered carefully when applying MMPA. These are grouped together here into those concerning the chemical structures and those concerning the preparation and analysis of the associated data. The principal concern in the area of chemical structure is how the “tradeoff (sic) between specificity and generalizability”, as highlighted by Papadatos et al. is achieved.<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> Stated in an alternative fashion: the more tightly the context of a given structural change is controlled, the smaller the data set of relevant transformations will be and the less structural diversity will be sampled. These two factors conflate to restrict how generally applicable the derived estimates of the change in the property value will be. This is discussed in many of the papers referred to in this article. Here we illustrate the issue with the EGFR data set. In Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a> a hierarchy of sets of matched molecular pairs is given. All of the pairs correspond to the addition of a hydroxyl group. At the very top is the set of all such matched pairs. Beneath this are two subsets, one in which hydroxyl is added to an aromatic carbon and the other in which it is added to an aliphatic carbon. The distribution in changes in potency for each of these subsets is given along with the corresponding statistics. These two subsets are statistically distinct from one another and suggest that if addition of hydroxyl is being considered, addition to an aromatic carbon should be preferred over addition to an aliphatic carbon. In the bottom layer, the distributions of changes in potency are given for two of the many possible subsets of the aromatic carbons, one in which the hydroxyl is added in the para position of an aniline and a second in which it is added to the 7-position of a quinazoline. These two sets are more likely to represent the kind of structural change of interest in a lead optimization campaign, but neither has enough representatives to be very general and neither is statistically distinct either from each other or from the more general set of changes caused by adding hydroxyl to an aromatic carbon. While the general rule that addition of hydroxyls is likely to be beneficial for EGFR potency is obtained (top layer), this may not be of use to projects limited for some reason to adding hydroxyls either to aliphatic carbons (middle layer) or to particular aromatic carbons (bottom layer).</div><figure id="fig4" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-22/jm200452d/production/images/medium/jm-2011-00452d_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-22/jm200452d/production/images/large/jm-2011-00452d_0003.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Distribution of changes in EGFR pIC<sub>50</sub> caused by adding a hydroxyl group in varying contexts with the most general case at the top and some specific positions at the bottom. The distributions are summarized by mean values, standard errors in those means (SEM), the standard deviation (SD), and the number of representative pairs identified (<i>N</i>). Ar represents any aromatic carbon linked group and R any aliphatic carbon linked group.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-22/jm200452d/production/images/large/jm-2011-00452d_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm200452d&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">When the context is completely proscribed to a particular attachment point on a particular series, the analysis almost reduces to Free–Wilson analysis (see Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a> for differences), which is not necessarily applicable or relevant beyond that context. If the context is completely uncontrolled, then the effect of a particular transformation is likely to reflect only simple trends, most notably lipophilicity determined effects.<a onclick="showRef(event, 'ref4 ref18 ref19'); return false;" href="javascript:void(0);" class="ref ref4 ref18 ref19">(4, 18, 19)</a> However the subsetting is performed, a threshold number of molecular pairs must be included in a set before the mean change becomes stable to the inclusion of additional pairs and a significant degree of molecular diversity is required among the molecules contributing to a set of pairs before chemists using the analysis can have confidence that any prediction is likely to be applicable in a general fashion. The first of these concerns is dependent upon the data type being analyzed, whereas the second requires an estimate of the diversity encompassed within a set. This was achieved by manual inspection of the contributing molecular pairs in the publication of Leach et al. but is automated in internal AstraZeneca Web pages by computing the mean Tanimoto difference among the first members of all pairs in a set.<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> These Tanimoto differences can be benchmarked to identify a cutoff that is felt to represent adequate diversity for general applicability. In their study, Gleeson et al. control for diversity by ensuring that each set of compounds contains representatives of at least five clusters, grouped by Tanimoto comparison of Daylight fingerprints, and ensuring that each set has at least 20 pairs.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> These two considerations of data set size and diversity are too rarely considered in publications in this area. A cautionary note is provided by data related to that originally presented by Leach et al. and shown in Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a>.<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> The addition of methyl sulfone groups to phenyl rings was reported in that paper to increase solubility. The variation of the mean change in solubility as measurements have been added to the database is shown in the period leading up to that publication and since. It is noteworthy that the transformation now is found to correspond to a small reduction in solubility. Meanwhile the structural diversity, as assessed by mean Tanimoto distance among the phenyl sulfones, has remained fairly constant at ∼0.6, having reached that point after about 10 pairs were included. At the time of the publication the mean change in log(solubility) was +0.26 with a standard error in that mean of 0.12; currently the corresponding mean value is −0.03 with a standard error of 0.09. The two mean values are consistent with an underlying population mean in the range +0.02 to +0.15 (see green lines in Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a>), but the variation in the mean from positive to negative is an extreme example of how temporal variation might dent confidence in the value of the analysis. Such variations are less likely the larger the data set is and the greater the structural diversity represented, but this relationship will depend upon both the structural change being studied and the nature of the data being mined.</div><figure id="fig5" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-22/jm200452d/production/images/medium/jm-2011-00452d_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-22/jm200452d/production/images/large/jm-2011-00452d_0004.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Variation with time in the mean change in solubility corresponding to addition of methyl sulfone to phenyl rings. The individual matched pairs are plotted as blue crosses in the order in which they were added to the data set, and the mean following the addition of each point is plotted in red. The green lines are at 2 standard errors in the mean above and below the mean at each point. The green arrow indicates the point in time at which the data in the publication by Leach et al. were extracted.<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-22/jm200452d/production/images/large/jm-2011-00452d_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm200452d&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">A final consideration related to chemical structure that is often neglected is stereochemistry. This can be difficult to treat correctly, particularly if matched pairs are defined by changes at one end or other of a bond to a stereocenter. This is particularly the case in many of the fragmentation approaches discussed below along with MCSS approaches that often use molecular graphs that neglect stereochemical information. For some properties, this is likely to have only a minor influence (lipophilicity for instance), but for others it will have a profound effect upon whether the pairs are appropriately paired or not and hence whether the resulting property change is computed correctly. For all of the reasons outlined in this section, it is beneficial to provide a link to the original data upon which any matched pairs analysis is based such that users can decide for themselves whether the result is relevant. Even rules that appear to be general should be subject to such scrutiny.</div><div class="NLM_p">The requirements of the data to be analyzed should also be considered carefully. Generally, the output from a matched pairs analysis is either a mean change in a particular property or else the proportion of times that a property has changed in a particular way (i.e., increased vs decreased or active vs inactive). The first of these requires that the property being compared in a pairwise fashion is within the dynamic range of the assay and on a linear scale (and preferably normally distributed). For some properties, such as IC<sub>50</sub> values, this might involve log transformation to yield pIC<sub>50</sub> values or equivalents as suggested by Fujita and Ban in their modified version of Free–Wilson analysis.<a onclick="showRef(event, 'ref8 ref20 ref21'); return false;" href="javascript:void(0);" class="ref ref8 ref20 ref21">(8, 20, 21)</a> Properties that are like rates or equilibrium constants should only properly be compared as ratios such that logarithmic transformation should be performed if linear comparisons are to be made, as is the case in MMPA. Logarithmic conversion is also applicable to the analysis of physicochemical and pharmacokinetic parameters, as has been demonstrated by Leach et al. in their study of aqueous solubility, plasma protein binding, and in vivo plasma exposure following oral dosing.<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> A recent paper describes multiplicative transformations, but these result from inappropriately comparing untransformed properties.<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a></div><div class="NLM_p">Careful consideration should also be given at the outset of what types of data should be paired together for comparison; differences between compounds measured in different assay formats may reveal only differences between assays and may not be related to the structural change. For example, Leach et al. only analyzed solubility data from an assay using solids provided by chemists. Data obtained in assays starting from DMSO solubilized samples were not included. If the need for more data to create larger matched pair sets and increase chemical diversity had been felt, these data could have been included with the risk that the extra variation would add more noise from which a signal might be more difficult to detect. Hajduck et al. compared data from a large set of different assays and end points in a way that is not recommended.<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> Careful consideration should also be given to which sets of pairs should be aggregated together to arrive at summary statistics. Even when data are obtained in comparable assays within each pair, in some cases groups of pairs measured in different assay formats might be sensibly aggregated and in others not. The decision about what to group together should ideally be taken in collaboration with those involved in the design and performance of the experiments. It is with these caveats in mind that our findings based on the EGFR data set are illustrative examples only.</div><div class="NLM_p">The mean change ought to be presented alongside the standard error in the mean to determine if it is actually distinct from zero (mean ± 2(SEM) should not include 0 to provide 95% confidence that the structural change generally causes a change in the property). This was illustrated graphically by Dossetter in his analysis of microsomal Cl<sub>int</sub> values where the mean was plotted against the standard error for a set of pairs in which phenyl groups are exchanged for alternative substituted aromatic rings.<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> Pair sets that lie above the <i>y</i> = 2<i>x</i> or below the <i>y</i> = −2<i>x</i> line are the transformations satisfying this criterion. The standard deviation should also be considered in order to provide the likely range of values in the change in property to be expected when a transformation is performed. Two standard deviations about the mean ought to encompass the actual value of the change in 95% of cases. Although testing that a mean is distinct from zero might provide confidence that the change is real, when this statistical test is not passed, the values still facilitate the presentation of the relevant information. In such circumstances, it could be concluded that a bioisostere has been identified or else that more measurements are required to enhance the reliability of the mean value.</div><div class="NLM_p">Many assays have limited dynamic ranges; values outside of the dynamic range cannot be compared to other values (within or without the dynamic range). In this circumstance, it may be that many pairs of interest might be excluded when computing the mean change; oftentimes these are the most interesting pairs because they involve the most substantial changes in a property. This kind of transformation is likely to include the discontinuous changes identified by Wasserman and Bajorath and the switch transformations highlighted by Keefer et al.<a onclick="showRef(event, 'ref2 ref22'); return false;" href="javascript:void(0);" class="ref ref2 ref22">(2, 22)</a> In these cases, it may be appropriate and useful to instead analyze the number of times that a property changes in a particular direction rather than the degree of change. These proportions ought to track with the mean difference. This was shown by Leach et al. for solubility and plasma protein binding and illustrated in Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a>.<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> If the data set is diverse and large enough, the observed proportion ought to be equivalent to the probability that applying the transformation to a new molecule (in the relevant context) will also cause the property to change in the same direction. This is often the kind of information that lead optimization projects are searching for, that transformations are likely to move a property in a desired direction or at the least not move it in an undesired direction. Similarly, data sets in which compounds are classified categorically, either because of the nature of the assay (e.g., Ames testing to identify either active or inactive compounds)<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> or artificially (e.g., Pfizer metabolism index),<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> can be analyzed in a pairwise fashion leading to predicted probabilities for changes among the various classifications. In their analysis of over 150 000 in vitro clearance measurements in human liver microsomes (HLM), Lewis and Cucurull-Sanchez quote 2-fold changes or greater as being significant based on variability in the experimental procedure.<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> Papadatos et al. also used a 2-fold margin in their classification of hERG activity, solubility, and lipophilicity into favorable, unfavorable, and zero effect transformations.<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> Despite loss of some quantitative information, this approach benefits from being able to include additional data points that are out of range of the assay. For example, a molecular transformation corresponding to a change in hERG activity from 7 to >30 μM could be considered as a significant decrease, albeit in the absence of any estimate of the magnitude of the reduction. In her study of Pfizer’s uridine 5′-diphosphoglucuronosyltransferase (UGT) data set, Cucurull-Sanchez adopted a different approach by classifying the compounds themselves as exhibiting either high or low clearance before conducting any analysis.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a></div><figure id="fig6" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-22/jm200452d/production/images/medium/jm-2011-00452d_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-22/jm200452d/production/images/large/jm-2011-00452d_0005.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Proportion of cases in which a property increases plotted against the mean change in the property for log(solubility) on the left and log(<i>K</i><sub>1</sub>) for plasma protein binding on the right. Error bars correspond to 2 standard errors in the mean on the <i>x</i>-axis, and the binomial probability 95% confidence interval is on the <i>y</i>-axis. All data are taken from Leach et al.<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-22/jm200452d/production/images/large/jm-2011-00452d_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm200452d&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">One complication with MMPA is deciding which properties are most useful for analysis. The simpler the property is, the more likely there is to be a straightforward link to molecular structure. This was exemplified by Leach et al.; effects upon solubility and plasma protein binding were shown to frequently occur with tight statistical certainty, whereas those concerning in vivo plasma exposure, which depends upon many other properties of the molecule, were less likely to do so.<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> Similarly, where a measurement relates simply to a binding event between protein and ligand, a better outcome might be expected than where a functional effect (agonism, antagonism, channel blocking, etc.) is measured, although we have found that MMPA can be effective for analyzing these more complex end points.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i5">Methods To Identify Matched Molecular Pairs</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_07617" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_07617" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">This section focuses on the algorithmic details of how several groups have chosen to identify matched pairs, and those less computationally inclined might prefer to skip to the next section. Recent literature has demonstrated that there are a host of different approaches to identifying matched molecular pairs. The choices made by different researchers often provide an insight into the philosophy underpinning the use of matched molecular pairs in these different groups. In this section we highlight some of the key variations.</div><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i6" class="anchor-spacer"></div><h3 class="article-section__title" id="_i6"> (i) Small Data Sets</h3><div class="NLM_p last">There are numerous reports in the medicinal chemistry literature involving “matched pairs” within a series featuring a common core.<a onclick="showRef(event, 'ref27 ref28 ref29 ref30 ref31'); return false;" href="javascript:void(0);" class="ref ref27 ref28 ref29 ref30 ref31">(27-31)</a> This basic form of MMPA is essentially an abstraction of the kind of information provided in many of the tables in publications in this journal where the variation of properties with substituents is detailed. It can be formalized by defining a core and simply plotting the substituent at a given position against the property of interest.<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> The analysis is then completed with the connection of points in which the remainder of the molecule is the same. The magnitude and consistency of property differences between one substituent and another are revealed.<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> Automated MMPA in this form has been implemented at AstraZeneca to support lead optimization projects that work within defined series. A core is defined by each project team, and the various exchanges of substituents around the core are automatically detected and the corresponding changes in properties computed. This is particularly powerful in an environment in which all compounds are subject to parallel testing in a battery of assays such that a fairly comprehensive view of the effect structural changes have on the full range of properties can be detected; this in turn can drive multiparameter optimization.<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> Any new data that are generated are automatically added such that a simple Web tool can provide a project team with the most up to date view of the effect of all structural changes so far tried. This influences the choices that are made about which compound to make next in a dynamic fashion. In this simple form, the overlap with Free–Wilson analysis is clear. For the remainder of this section, only those methods where the formal definition of a core is not required are detailed.</div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i7" class="anchor-spacer"></div><h3 class="article-section__title" id="_i7"> (ii) Supervised Approaches</h3><div class="NLM_p">A number of attempts to correlate changes in molecular structure with changes in properties on a large scale have relied on the investigator, either wholly or in part, to define the transformations themselves prior to conducting the analysis. Researchers at AstraZeneca’s Alderley Park site have used the molecular editor program LEATHERFACE to convert the SMILES for each putative member of a matched pair into its counterpart structure.<a onclick="showRef(event, 'ref1 ref4 ref35 ref36'); return false;" href="javascript:void(0);" class="ref ref1 ref4 ref35 ref36">(1, 4, 35, 36)</a> The use of SMARTS to define the substructure to be modified allows control over the chemical environment in which the structural modification occurs.<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> Further developments in the same group led to the program find_pairs and its successor thrice_pairs.<a onclick="showRef(event, 'ref38 ref39'); return false;" href="javascript:void(0);" class="ref ref38 ref39">(38, 39)</a> These rely upon defining the two end points with SMARTS patterns and defining how the atoms in each SMARTS pattern map onto one another and might be considered to be a basic implementation of SMIRKS.<a onclick="showRef(event, 'ref4 ref11'); return false;" href="javascript:void(0);" class="ref ref4 ref11">(4, 11)</a> Gleeson et al. used related methodology that also allows the second member of the pair to be encoded using SMARTS.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> This extension allows less specific transformations such as ArCl to ArAr′ (Ar′ = any aromatic) to be studied and is achieved by using SMARTS filtering as opposed to SMILES matching as a final step. The diversity of the hit list can be expanded even further by using “X to any” type transformation definitions, where just the initial molecule less fragment X is used as a product substructure filter. Pfizer’s “Buy me Grease” tool allows users to mine for matched molecular pairs via a Web interface.<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> The program is implemented in Pipeline Pilot and requires a RXN file to be provided by the user to define the molecular transformation of interest.<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> The reaction is applied to the entire data set and identifies compounds for which experimental data also exist for the products. Finally, a report is generated with a histogram indicating the percentage of times the transformation increased, decreased, or had no effect on the property, in vitro clearance in this case.</div><div class="NLM_p last">Other authors have used predefined transformation lists to supplement an unsupervised method that is reliant on the heavy atom framework of one member of the matched molecular pair being structurally embedded within the other. Hajduk and Sauer used the Daylight toolkit to identify instances where one member of each putative matched pair was a substructure of the other.<a onclick="showRef(event, 'ref19 ref41'); return false;" href="javascript:void(0);" class="ref ref19 ref41">(19, 41)</a> Cases where only one fragment remained following cleavage of that substructure were considered matched molecular pairs. Cucurull-Sanchez adopted a similar tactic, having identified a collection of substructural features that related to either high or low UGT clearance.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a></div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8"> (iii) Unsupervised Approaches</h3><div class="NLM_p">The fastest and most efficient way to identify matched molecular pairs within a data set in an unsupervised fashion is to decompose the molecules into fragments, which may then be indexed to allow rapid sorting and retrieval from a database. The fragmentation schemes are typically based on the method described by Bemis and Murcko, where acyclic single bonds are cleaved to define a molecular scaffold and its side chain substituents.<a onclick="showRef(event, 'ref42 ref43'); return false;" href="javascript:void(0);" class="ref ref42 ref43">(42, 43)</a> Such methods benefit from the requirement to process each molecule just once, as once the core(s) and side chain(s) for a given structure have been defined, the process of matched pair identification requires simply pairing compounds with a common core.</div><div class="NLM_p">Haubertin and Bruneau used a variant of the RECAP fragmentation process in their analysis of over 50 000 druglike compounds to determine the effect of common structural transformations on solubility, plasma protein binding, and lipophilicity.<a onclick="showRef(event, 'ref44 ref45'); return false;" href="javascript:void(0);" class="ref ref44 ref45">(44, 45)</a> Their definition of a side chain was limited to fragments having an acyclic attachment point and a SMILES string of less than 40 characters, which resulted in a fixed library of 9038 substructures. Their rationale for limiting the size of the side chain library in this way was to expedite the identification of side chains in previously unprocessed structures submitted by users through a Web interface. All side chain occurrences were identified in each of the druglike molecules, generating a total of 386 757 core/side chain combinations, from which the core pairing process identified 733 445 occurrences of side chain substitution. Hussain and Rea were able to address some of the limitations of this approach by removing the 40-character limit from the definition of a fragment and extending the fragmentation procedure to include double and triple cut disconnection patterns.<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a> Their analysis identified all heavy atom molecular pairs arising from acyclic bond disconnection from a set of 333 491 compounds in just under 14 h on a single CPU. Hydrogen substitutions were identified in a second step by capping the fragments arising from single cut fragmentations with hydrogen and then looking up the canonicalized SMILES among the input molecules. This paper reports the first use of SMIRKS to encode the transformations identified from MMPA, although the contextual information was not retained to ensure specificity in their application.</div><div class="NLM_p">This issue of context dependence is one that has recently been studied in some depth at the University of Sheffield, U.K., in collaboration with GlaxoSmithKline.<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> The fragment indexing approach described above was selected for this study because of its computational efficiency. For each transformation, the context was characterized using a total of five additional descriptors, three describing the molecules as a whole (a reduced graph representation of the molecules, a Murcko scaffold, and a fingerprint based cluster membership), with the remaining two based on the local environment of the disconnection point (the nearest node from the reduced graph and a string representing the nearest three layers of SYBYL atom types).<a onclick="showRef(event, 'ref42 ref47'); return false;" href="javascript:void(0);" class="ref ref42 ref47">(42, 47)</a> Papadatos et al. reported a collection of common side chain replacements with well validated effects on hERG inhibition, solubility, and lipophilicity, most of which follow anticipated changes in lipophilicity. However, what makes this paper a must-read for anyone interested in the field of MMPA are the examples where this expectation is defied in a way that can be attributed to the local environment of the attachment point and where there are enough examples to demonstrate that this is a real dependence. This paper offers a vision of how identifying context specific effects might be systematized.</div><div class="NLM_p">A popular alternative to the acyclic disconnection algorithms described above are those that require a maximum common substructure search (MCSS) to be performed on each and every pair of molecules. The advantage of these algorithms is a conceptual one and relates back to the central definition of a matched molecular pair, i.e., two compounds that differ from each other by just a small well-defined structural change. As the name suggests, MCSS based approaches identify the maximum amount of substructure shared between two molecules; thus, only the absolute minimum substructure distinguishing the two structures is identified as a byproduct and forms the basis for molecular pairing.</div><div class="NLM_p">Unfortunately, this method of identifying matched molecular pairs comes at price, as where the fragmentation algorithms require each molecule to be processed only once, a complete MCSS based approach requires a full <i>n</i> × <i>n</i> comparison of all the molecules in a data set. To overcome the computational expense incurred, Sheridan performed clustering on his data set of 100 000 compounds from the MDL Drug Data Report (MDDR), first based on biological activity and then on topological descriptors prior to performing MMPA by MCSS.<a onclick="showRef(event, 'ref48 ref49'); return false;" href="javascript:void(0);" class="ref ref48 ref49">(48, 49)</a> All pairs of compounds within a cluster were compared with each other, with the highest scoring common substructure (HSCS) being used as a core from which the fragments undergoing modification could be identified.<a onclick="showRef(event, 'ref50'); return false;" href="javascript:void(0);" class="ref ref50">(50)</a> Having ensured that a single fragment was all that set the two molecules apart, the remainder of the molecule was pruned according to two methods, one which encoded just the bare minimum number of atoms required to define the transformation (algorithm A) and one where rings directly encompassing the transformation were also preserved (algorithm B). As the compounds had been clustered according to their biological activity at the outset, the most frequently occurring transformations were considered bioisosteres, as they represented groups that could be interchanged with each other, retaining activity across a variety of target classes. Sheridan reports that despite returning larger fragments and fewer observations, algorithm B appeared to be “more appealing because it retains more of the contextual information of the replacement”.</div><div class="NLM_p">A revised version of Merck’s algorithm, published in 2006, went beyond simply identifying molecular pairs with activity against a given receptor and began to associate quantitative changes in activity between the pair members.<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> Dopamine 2 (D2) agonism, dihydrofolate reductase (DHFR) inhibition, and angiotensin converting enzyme inhibition were studied. First, the T-ANALYZE program was used to identify the “remainder after elimination of common substructure” (RECS) for pairs of molecules satisfying a given degree of similarity, which defined the nature of the transformation itself in the previously reported “algorithm A”. Examples of each transformation were then clustered according to atom-pair and topological-torsion descriptors to ensure that the transformation occurred within a similar context in all cases, and the experimental data were added to each of the clusters as a final step. The clusters were scored and ranked allowing users to easily visualize interesting regions of consistent, local SAR. The authors then took the D2 and DHFR data sets and submitted a probe molecule to a second program, T-MORPH, which identified relevant transformations that could be applied to the probe and their likely effects on activity. In AstraZeneca’s most recently published method, the authors also use a MCSS to identify molecular pairs, only having identified the RECS, they proceed to incorporate a series of layers of common substructure in a hierarchical version of Merck’s “algorithm B”.<a onclick="showRef(event, 'ref51'); return false;" href="javascript:void(0);" class="ref ref51">(51)</a> The advantage of this so-called “WizePairZ” approach is that the local environment is encoded as part of the transformation (using nonproprietary SMIRKS notation). Having identified transformations and effects on histone deacetylase inhibition from a small collection of compounds, all were fed back into the program, resulting in the generation of two structures initially omitted from the data set and accurate predictions of their biological activity.</div><div class="NLM_p last">Improvements in computer performance over recent years have seen ever larger data sets mined for molecular transformations by MCSS. While retaining a topological similarity based cutoff, Southall and Ajay were able to do away with biological target-based clustering in their analysis of 116 550 kinase inhibitors from patents and the medicinal chemistry literature.<a onclick="showRef(event, 'ref52'); return false;" href="javascript:void(0);" class="ref ref52">(52)</a> They estimated that by setting the Tanimoto threshold for comparison at 0.5, they were able to capture 90% of the molecular pairs in the data set while reducing the number of MCSS searches by 1000-fold. The calculation took approximately 2 weeks on a 30-processor cluster. Warner et al. have also reported similar performance, performing an entirely unfiltered analysis of over 35 000 compounds in around 3 days on 36 CPUs.<a onclick="showRef(event, 'ref51'); return false;" href="javascript:void(0);" class="ref ref51">(51)</a> In what is undoubtedly the largest such study to date, researchers at Eli Lilly performed around 3.6 × 10<sup>12</sup> MCSS calculations in their analysis of almost 2.7 million structures.<a onclick="showRef(event, 'ref53'); return false;" href="javascript:void(0);" class="ref ref53">(53)</a> Without clustering based on either biological activity or molecular similarity, the time taken to complete this enormous calculation was reduced through use of the RASCAL algorithm for the MCSS,<a onclick="showRef(event, 'ref54'); return false;" href="javascript:void(0);" class="ref ref54">(54)</a> the use of constraints on the extent of common substructure required and shortest path lengths between connection points, the omission of hydrogen replacements, and the use of a 1072 core cluster on which to perform the comparisons.<a onclick="showRef(event, 'ref53 ref54 ref55'); return false;" href="javascript:void(0);" class="ref ref53 ref54 ref55">(53-55)</a> Nevertheless this study marks a significant achievement, as 2.7 million structures are many more than are ever likely to exist in an experimental data set to be mined by MMPA.</div></div></div><div id="sec5" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i9">Future for Matched Molecular Pairs</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_66216" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_66216" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Having outlined some of the principal considerations in how and when to apply MMPA, it is worth considering what new avenues might be followed. The first priority is to apply this kind of analysis to larger and more diverse sets of compounds, now that the feasibility of such comparisons has been demonstrated,<a onclick="showRef(event, 'ref53'); return false;" href="javascript:void(0);" class="ref ref53">(53)</a> and to data sets generated in new assay types not yet considered. This ought to lead to new insights and guidelines of utility to understanding and optimizing compounds. During the course or preparing this manuscript, a number of pertinent publications have appeared: Hu and Bajorath have found transformations that change a compound's biological activity profile (with 754 different target proteins considered); Wassermann and Bajorath identified bioisosteric changes that are target family specific; Keefer et al. studied effects on a range of ADME properties caused by a diverse set of structural changes.<a onclick="showRef(event, 'ref2 ref56 ref57'); return false;" href="javascript:void(0);" class="ref ref2 ref56 ref57">(2, 56, 57)</a> As more parameters are studied, the possibility of identifying transformations that facilitate multiparameter optimization should be feasible. By way of illustration of this and the predictive power of even the so far limited published analyses, we note that in a recent publication from AstraZeneca it is reported that addition of a <i>m</i>-Me group in a particular compound reduces solubility from 9.5 to 8 μM and increases Cl<sub>int</sub> (in microsomes) from 76 to 100 μL min<sup>–1</sup> mg<sup>–1</sup>.<a onclick="showRef(event, 'ref58'); return false;" href="javascript:void(0);" class="ref ref58">(58)</a> These changes correspond to log(Sol) and log(Cl<sub>int</sub>) both decreasing by 0.1 which agrees well with expected values of 0.21 and 0.28 predicted by Leach et al. and Dossetter, respectively.<a onclick="showRef(event, 'ref4 ref11'); return false;" href="javascript:void(0);" class="ref ref4 ref11">(4, 11)</a> We hope that publications concerning a broader range of structural changes relating to more properties will lead to more examples where MMPA has predicted structural changes that have changed a number of properties in such a way as to strike a desired balance.</div><div class="NLM_p">One limitation of the approach, as stressed above, is that it can only make predictions about structural features that have precedent in any given assay. This might be circumvented by introducing levels of abstraction for particular groups; rather than grouping transformations in a way that requires an exact match, this could be loosened. For instance, if a data set does not contain any matched pairs in which an ethyl sulfone has been added to an aromatic ring but does have a set of pairs in which a methyl sulfone has been added, then the prediction might be based on this with a lipophilicity (or alternative substituent related descriptor such as σ) term added to compensate for the larger alkyl group. The challenge will be to ensure that the user is comfortable as the analysis moves away from being a means of organizing and presenting experimental data. The concepts being developed in the area of molecular graph theory describing the amount of editing required to link two structures may contribute to the organizing of transformations both to address the issues highlighted in Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a> and to facilitate the abstraction necessary for extrapolation.<a onclick="showRef(event, 'ref59'); return false;" href="javascript:void(0);" class="ref ref59">(59)</a></div><div class="NLM_p">The pairwise comparison involved in MMPA ought to complement and validate computational modeling methods that either rely upon abbreviating to model systems (as is the case in many quantum mechanical studies) or that have many errors involved that might otherwise mask any signal (as is the case in many docking/scoring or even simulation studies). In the former case, expensive calculations on protein–ligand binding are only currently feasible when focused on small regions of the protein that are in proximity to the ligand, analogous to the theozymes proposed by Houk and co-workers to rationalize enzymatic catalysis.<a onclick="showRef(event, 'ref60'); return false;" href="javascript:void(0);" class="ref ref60">(60)</a> While calculation of individual binding free energies will be impossible, computed changes in binding energy for structural changes ought to relate to the sort of differences that MMPA produces. Equally, the ab initio calculation of changes in intrinsic solubility from lattice and solvation energies could be validated with MMPA.<a onclick="showRef(event, 'ref61'); return false;" href="javascript:void(0);" class="ref ref61">(61)</a> By contrast, modeling all of the atoms in a complex system requires a number of approximations to be used (such as those in pose scoring or in force fields) that reduce accuracy. Computing differences could benefit from cancellation of some of the errors introduced by these approximations, and these differences can be compared to the output of MMPA. The output of MMPA is limited to the observation that a structural change leads to a particular property change but often prompts the question of why this might be; these two approaches can help to address this.</div><div class="NLM_p">A view of how matched molecular pairs and other analyses might contribute most effectively to pharmaceutical research has been described by Griffen, outlined in Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a>.<a onclick="showRef(event, 'ref62'); return false;" href="javascript:void(0);" class="ref ref62">(62)</a> Here, all available and relevant databases can be examined to find rules that link changes in chemical structure with improvements in properties of medicinal chemistry interest; this happens in the dashed box to the left of Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a>. These rules can then be applied to “problem molecules” that have one or more properties preventing them from becoming clinical candidates. The application of the rules is performed in the right-hand section of Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a>. In principle, the identification and application of the rules do not need manual intervention.</div><figure id="fig7" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-22/jm200452d/production/images/medium/jm-2011-00452d_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-22/jm200452d/production/images/large/jm-2011-00452d_0006.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Flow diagram for a potential automated drug design tool. In the left-hand box, databases are mined to obtain rules; matched molecular pair analysis provides a simple and effective way to obtain these rules. In the right-hand section, these rules are applied to compounds that have deficiencies preventing them from becoming clinical candidates to suggest new molecules that might overcome those deficiencies.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-22/jm200452d/production/images/large/jm-2011-00452d_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm200452d&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">MMPA provides a simple way to explore data sets in search of the rules required for this kind of approach. The various unsupervised approaches described above can all be used to do just this. Each will mine a data set and find sets of compounds that differ only by small, well-defined transformations. They can also compute for each the corresponding effect on a property of interest. When such methods are applied to data sets of the size typically available (thousands of compounds), they will discover enormous sets of potential rules (thousands to millions).<a onclick="showRef(event, 'ref51'); return false;" href="javascript:void(0);" class="ref ref51">(51)</a> These rules are subject to the issues highlighted in Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a> of differing degrees of specificity contrasting with different degrees of exemplification. In the context of generating rules, these issues translate to the dilemma of whether high degrees of certainty for rules derived from many diverse molecules are preferred over lower degrees of certainty for larger, structurally more specific effects. The choice will depend upon the philosophy of those involved and is unlikely to have any absolute right answer.</div><div class="NLM_p last">One consequence of taking an approach like that outlined in Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a> is that the nature of the compounds that are represented in the databases of measurements becomes highly significant. Ideally, significant numbers of structural transformations would be represented with each deriving from a diverse and large set of compounds. Most of the measurements are generated without this consideration; they are generated to determine properties of interest to particular drug discovery projects. By identifying compounds that would populate matched pairs sets that are available for testing and generating data on them, MMPA can improve its own utility by enhancing the certainty of rules that appear to be interesting but that do not have enough data to be convincing. In this way, the ideal outcome of providing drug designers with rules that surprise them but that are nonetheless convincing can be achieved.</div></div><div id="sec6" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i10">Summary</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_65782" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_65782" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">MMPA has become accepted as a useful approach in drug discovery. In the absence of experimental data, it can be used to identify the frequency with which certain transformations have been performed. It has been applied across a range of properties of medicinal chemistry interest. It contrasts favorably with some alternative data analysis methods and in particular presents experimental data to drug designers in a way that naturally suggests what compound to make next but within the limit of transformations that have been exemplified in the data set. It can be used to define a set of transformations that have historically modulated a given property that can assist the design of future compounds. Many researchers have developed software for identifying matched molecular pairs, and the approach taken tends to be governed by the motivation behind the analysis. There are a number of important considerations determining when matched pairs are appropriate. There is an inevitable trade-off between structurally specific changes that are less well exemplified and more general changes that are better represented but less likely to be surprising. Data ought to be carefully prepared and generated in assays that are as homogeneous as is practicable. Matched pairs can complement molecular modeling techniques such as quantum mechanics and docking. Matched pairs can be used to obtain rules of thumb that can guide improvements in any measurable property or indeed several properties at once.</div></div><div class="NLM_back"><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_28350" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_28350" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Andrew G. Leach</span> - <span class="hlFld-Affiliation affiliation">Cardiovascular and Gastrointestinal Innovative Medicines Unit, AstraZeneca Pharmaceuticals, 30S373 Mereside, Alderley Park, Macclesfield, SK10 4TG, U.K.</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#4d2c23293f283a6321282c2e250d2c3e393f2c372823282e2c632e2220"><span class="__cf_email__" data-cfemail="62030c061007154c0e0703010a22031116100318070c0701034c010d0f">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ed Griffen</span> - <span class="hlFld-Affiliation affiliation">Oncology Innovative Medicines Unit, AstraZeneca Pharmaceuticals, Mereside, Alderley Park, Macclesfield, SK10 4TG, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Graeme R. Robb</span> - <span class="hlFld-Affiliation affiliation">Cardiovascular and Gastrointestinal Innovative Medicines Unit, AstraZeneca Pharmaceuticals, 30S373 Mereside, Alderley Park, Macclesfield, SK10 4TG, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Daniel J. Warner</span> - <span class="hlFld-Affiliation affiliation">Department of Medicinal Chemistry, AstraZeneca R&D Montreal, Montreal, Quebec, H4S 1Z9, Canada</span></div></li></ul></li><li></li><li></li><li></li></ul></div><div class="testing" data-doi="10.1021/jm200452d" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="bio" rid="ath1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="biographies">Biographies</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_42158" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_42158" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><figure id="ath1" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Ed Griffen</h2><div class="figure-bottom-links"><div class="download-hi-res-img"></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm200452d&amp;id="><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><b>Ed Griffen</b> is a Principal Scientist working in the Oncology Innovative Medicines unit at AstraZeneca. He studied for his Ph.D. at Imperial College, London, in marine alkaloid synthesis with Professors Charles Rees and Christopher Moody and then joined the research group of Professor Victor Snieckus as a Postdoctoral Fellow developing methods in indole metalation and radiometal chelation chemistries. He is a named inventor on 16 patents and is a coauthor of the textbook “On Medicinal Chemistry”. Ed is a Visiting Lecturer at the University of Manchester, U.K., in the medicinal chemistry program. Recent research publications have been in the application of data mining methods to accelerate drug discovery.</p></figure></div><div class="bio" rid="ath2"><figure id="ath2" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Andrew G. Leach</h2><div class="figure-bottom-links"><div class="download-hi-res-img"></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm200452d&amp;id="><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><b>Andrew G. Leach</b> is a computational chemist working in the Cardiovascular and Gastrointestinal Innovative Medicines unit at AstraZeneca. He obtained his Ph.D. in the group of Professor S. V. Ley at the University of Cambridge, U.K., for work developing new techniques for combinatorial chemistry. He undertook postdoctoral studies at the University of California, Los Angeles, in the group of Professor K. N. Houk, investigating several pericyclic reaction mechanisms and the interactions between proteins and transition states.</p></figure></div><div class="bio" rid="ath3"><figure id="ath3" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Graeme R. Robb</h2><div class="figure-bottom-links"><div class="download-hi-res-img"></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm200452d&amp;id="><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><b>Graeme R. Robb</b> is a computational chemist working in the Cardiovascular and Gastrointestinal Innovative Medicines Unit at AstraZeneca. He obtained his Ph.D. in chemical physics under the supervision of Professor A. Harrison, studying the effects of microwave radiation on solid materials using in situ X-ray diffraction. He joined AstraZeneca in 2002 and has worked predominantly in lead optimization programs, with particular interest in GPCR receptors.</p></figure></div><div class="bio" rid="ath4"><figure id="ath4" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Daniel J. Warner</h2><div class="figure-bottom-links"><div class="download-hi-res-img"></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm200452d&amp;id="><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><b>Daniel J. Warner</b> is a computational chemist with a background in biomolecular simulation and analysis. He completed his Ph.D. under the supervision of Drs. Charles Laughton and Stephen Doughty at the University of Nottingham, U.K., before joining AstraZeneca in October 2006. Since then, he has supported predominantly structure-based lead optimization programs, first within respiratory and inflammation at Charnwood in the U.K. and more recently in neuroscience at AstraZeneca R&D Montreal, Canada. He is currently involved in the delivery of a global system for improved exploitation of MMPA across AstraZeneca.</p></figure></div><div id="" class="NLM_sec NLM_sec_level_1"><div id="ac_i15" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i15"> </h2><div class="NLM_p last"><table summary="" class="NLM_def-list" id="dl1"><div id="ac_i16" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i16"> Abbreviations Used</h2><tr><td class="NLM_term">MMPA</td><td class="NLM_def"><p class="first last">matched molecular pair analysis</p></td></tr><tr><td class="NLM_term">SAR</td><td class="NLM_def"><p class="first last">structure–activity relationship</p></td></tr><tr><td class="NLM_term">SPR</td><td class="NLM_def"><p class="first last">structure–property relationship</p></td></tr><tr><td class="NLM_term">QSAR</td><td class="NLM_def"><p class="first last">quantitative structure–activity relationship</p></td></tr><tr><td class="NLM_term">QSPR</td><td class="NLM_def"><p class="first last">quantitative structure–property relationship</p></td></tr><tr><td class="NLM_term">EGFR</td><td class="NLM_def"><p class="first last">epidermal growth factor receptor tyrosine kinase</p></td></tr><tr><td class="NLM_term">RMSE</td><td class="NLM_def"><p class="first last">root-mean-square error</p></td></tr><tr><td class="NLM_term">SEM</td><td class="NLM_def"><p class="first last">standard error in the mean</p></td></tr><tr><td class="NLM_term">SD</td><td class="NLM_def"><p class="first last">standard deviation</p></td></tr><tr><td class="NLM_term">MCSS</td><td class="NLM_def"><p class="first last">maximum common substructure</p></td></tr><tr><td class="NLM_term">hERG</td><td class="NLM_def"><p class="first last">human ether-a-go-go-related gene encoded potassium channel</p></td></tr><tr><td class="NLM_term">HLM</td><td class="NLM_def"><p class="first last">human liver microsomes</p></td></tr><tr><td class="NLM_term">RECS</td><td class="NLM_def"><p class="first last">remainder after elimination of common substructure</p></td></tr></table></div></div><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i17">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_60335" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_60335" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 62 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group">Kenny, P. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sadowski, J.</span><span> </span><span class="NLM_article-title">Structure modification in chemical databases</span> <span class="citation_source-journal">Methods Princ. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">23</span><span class="NLM_x">, </span> <span class="NLM_fpage">271</span><span class="NLM_x">–</span> <span class="NLM_lpage">285</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Fjm200452d&amp;key=10.1002%2F3527603743.ch11" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Fjm200452d&amp;key=1%3ACAS%3A528%3ADC%252BD2MXlt1KksLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2005&pages=271-285&author=P.+W.+Kennyauthor=J.+Sadowski&title=Structure+modification+in+chemical+databases"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Structure modification in chemical databases</span></div><div class="casAuthors">Kenny, Peter W.; Sadowski, Jens</div><div class="citationInfo"><span class="NLM_cas:title">Methods and Principles in Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">Chemoinformatics in Drug Discovery</span>),
    <span class="NLM_cas:pages">271-285</span>CODEN:
                <span class="NLM_cas:coden">MPMCE3</span>;
        ISSN:<span class="NLM_cas:issn">1432-4636</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Two complementary tools for controlled modification of structures and their application to building databases, in particular, generation of appropriate ionization, tautomeric and configurational states for virtual screening, are described.  Permute has been written to exhaustively generate reasonable isomers of druglike mols. with respect to protonation and formal charge, tautomerism, and nitrogen configurations.  The approach applies a set of empirical rules for possible sites of hydrogen placements, for restricting the no. of isomers, and for the prevention of duplicates.  It is optimized for isomer generation for large databases of druglike mols. for virtual screening.  Leatherface is a 2-D mol. editor that modifies properties of atoms and bonds in mol. connection tables according to rules specified by the user.  Unlike Permute, Leatherface encodes no chem. knowledge and neither processes nor generates 3-D structures.  Its real strength is that it allows the user to impose a very detailed and precisely specified chem. view on large nos. of connection tables.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrUAeZZS8UjP7Vg90H21EOLACvtfcHk0liWGP6SvThy_A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXlt1KksLY%253D&md5=5b5e4bdc415573c16622f43333595b6d</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1002%2F3527603743.ch11&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F3527603743.ch11%26sid%3Dliteratum%253Aachs%26aulast%3DKenny%26aufirst%3DP.%2BW.%26aulast%3DSadowski%26aufirst%3DJ.%26atitle%3DStructure%2520modification%2520in%2520chemical%2520databases%26jtitle%3DMethods%2520Princ.%2520Med.%2520Chem.%26date%3D2005%26volume%3D23%26spage%3D271%26epage%3D285" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group">Keefer, C. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chang, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kauffman, G. W.</span><span> </span><span class="NLM_article-title">Extraction of tacit knowledge from large ADME data sets via pairwise analysis</span> <span class="citation_source-journal">Bioorg. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">3739</span><span class="NLM_x">–</span> <span class="NLM_lpage">3749</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Fjm200452d&amp;key=10.1016%2Fj.bmc.2011.05.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Fjm200452d&amp;key=21616672" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Fjm200452d&amp;key=1%3ACAS%3A528%3ADC%252BC3MXnsVyisrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2011&pages=3739-3749&author=C.+E.+Keeferauthor=G.+Changauthor=G.+W.+Kauffman&title=Extraction+of+tacit+knowledge+from+large+ADME+data+sets+via+pairwise+analysis"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Extraction of tacit knowledge from large ADME data sets via pairwise analysis</span></div><div class="casAuthors">Keefer, Christopher E.; Chang, George; Kauffman, Gregory W.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">3739-3749</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Pharmaceutical companies routinely collect data across multiple projects for common ADME endpoints.  Although at the time of collection the data is intended for use in decision making within a specific project, knowledge can be gained by data mining the entire cross-project data set for patterns of structure-activity relationships (SAR) that may be applied to any project.  One such data mining method is pairwise anal.  This method has the advantage of being able to identify small structural changes that lead to significant changes in activity.  In this paper, we describe the process for full pairwise anal. of our high-throughput ADME assays routinely used for compd. discovery efforts at Pfizer (microsomal clearance, passive membrane permeability, P-gp efflux, and lipophilicity).  We also describe multiple strategies for the application of these transforms in a prospective manner during compd. design.  Finally, a detailed anal. of the activity patterns in pairs of compds. that share the same mol. transformation reveals multiple types of transforms from an SAR perspective.  These include bioisosteres, additives, multiplicatives, and a type we call switches as they act to either turn on or turn off an activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrOPDhkbDbMmLVg90H21EOLACvtfcHk0liWGP6SvThy_A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXnsVyisrw%253D&md5=3b1ff499078e50c63935d7c7820134cd</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2011.05.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2011.05.003%26sid%3Dliteratum%253Aachs%26aulast%3DKeefer%26aufirst%3DC.%2BE.%26aulast%3DChang%26aufirst%3DG.%26aulast%3DKauffman%26aufirst%3DG.%2BW.%26atitle%3DExtraction%2520of%2520tacit%2520knowledge%2520from%2520large%2520ADME%2520data%2520sets%2520via%2520pairwise%2520analysis%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2011%26volume%3D19%26spage%3D3739%26epage%3D3749" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group">Overington, J.</span><span> </span><span class="NLM_article-title">ChEMBL. An interview with John Overington, team leader, chemogenomics at the European Bioinformatics Institute Outstation of the European Molecular Biology Laboratory (EMBL-EBI). Interview by Wendy A. Warr</span> <span class="citation_source-journal">J. Comput.-Aided Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">23</span><span class="NLM_x">, </span> <span class="NLM_fpage">195</span><span class="NLM_x">–</span> <span class="NLM_lpage">198</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2009&pages=195-198&author=J.+Overington&title=ChEMBL.+An+interview+with+John+Overington%2C+team+leader%2C+chemogenomics+at+the+European+Bioinformatics+Institute+Outstation+of+the+European+Molecular+Biology+Laboratory+%28EMBL-EBI%29.+Interview+by+Wendy+A.+Warr"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DOverington%26aufirst%3DJ.%26atitle%3DChEMBL.%2520An%2520interview%2520with%2520John%2520Overington%252C%2520team%2520leader%252C%2520chemogenomics%2520at%2520the%2520European%2520Bioinformatics%2520Institute%2520Outstation%2520of%2520the%2520European%2520Molecular%2520Biology%2520Laboratory%2520%2528EMBL-EBI%2529.%2520Interview%2520by%2520Wendy%2520A.%2520Warr%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2009%26volume%3D23%26spage%3D195%26epage%3D198" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group">Leach, A. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jones, H. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cosgrove, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kenny, P. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ruston, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">MacFaul, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wood, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Colclough, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Law, B.</span><span> </span><span class="NLM_article-title">Matched molecular pairs as a guide in the optimization of pharmaceutical properties; a study of aqueous solubility, plasma protein binding and oral exposure</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">6672</span><span class="NLM_x">–</span> <span class="NLM_lpage">6682</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm0605233" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Fjm200452d&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtV2gtLnO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2006&pages=6672-6682&author=A.+G.+Leachauthor=H.+D.+Jonesauthor=D.+A.+Cosgroveauthor=P.+W.+Kennyauthor=L.+Rustonauthor=P.+MacFaulauthor=J.+M.+Woodauthor=N.+Colcloughauthor=B.+Law&title=Matched+molecular+pairs+as+a+guide+in+the+optimization+of+pharmaceutical+properties%3B+a+study+of+aqueous+solubility%2C+plasma+protein+binding+and+oral+exposure"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Matched Molecular Pairs as a Guide in the Optimization of Pharmaceutical Properties; a Study of Aqueous Solubility, Plasma Protein Binding and Oral Exposure</span></div><div class="casAuthors">Leach, Andrew G.; Jones, Huw D.; Cosgrove, David A.; Kenny, Peter W.; Ruston, Linette; MacFaul, Philip; Wood, J. Matthew; Colclough, Nicola; Law, Brian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">6672-6682</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">By identifying every pair of mols. that differ only by a particular, well-defined, structural transformation in a database of measured properties and computing the corresponding change in property, we obtain an overview of the effect that structural change has upon the property and set an expectation for what will happen when that transformation is applied elsewhere.  The mean change indicates the expected magnitude of the change in the property and the no. of cases in which the property increases give the probability that the structural transformation will cause the property to increase.  Outliers indicate potential ways of avoiding the general trend.  Comparing to changes in lipophilicity highlights structural transformations that have unusual effects, some of which can be explained by conformational changes.  In this paper, we focus upon the effects on aq. soly., plasma protein binding and oral exposure of adding substituents to arom. rings and methylating heteroatoms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpcyrloR79Oq7Vg90H21EOLACvtfcHk0ljSFFw4EP4OWg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtV2gtLnO&md5=2c143b424aee3ddb6c34616aff699ee0</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1021%2Fjm0605233&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0605233%26sid%3Dliteratum%253Aachs%26aulast%3DLeach%26aufirst%3DA.%2BG.%26aulast%3DJones%26aufirst%3DH.%2BD.%26aulast%3DCosgrove%26aufirst%3DD.%2BA.%26aulast%3DKenny%26aufirst%3DP.%2BW.%26aulast%3DRuston%26aufirst%3DL.%26aulast%3DMacFaul%26aufirst%3DP.%26aulast%3DWood%26aufirst%3DJ.%2BM.%26aulast%3DColclough%26aufirst%3DN.%26aulast%3DLaw%26aufirst%3DB.%26atitle%3DMatched%2520molecular%2520pairs%2520as%2520a%2520guide%2520in%2520the%2520optimization%2520of%2520pharmaceutical%2520properties%253B%2520a%2520study%2520of%2520aqueous%2520solubility%252C%2520plasma%2520protein%2520binding%2520and%2520oral%2520exposure%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2006%26volume%3D49%26spage%3D6672%26epage%3D6682" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group">Sheridan, R. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hunt, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Culberson, J. C.</span><span> </span><span class="NLM_article-title">Molecular transformations as a way of finding and exploiting consistent local QSAR</span> <span class="citation_source-journal">J. Chem. Inf. Model</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">46</span><span class="NLM_x">, </span> <span class="NLM_fpage">180</span><span class="NLM_x">–</span> <span class="NLM_lpage">192</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci0503208" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Fjm200452d&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtFKrt77N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2006&pages=180-192&author=R.+P.+Sheridanauthor=P.+Huntauthor=J.+C.+Culberson&title=Molecular+transformations+as+a+way+of+finding+and+exploiting+consistent+local+QSAR"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular Transformations as a Way of Finding and Exploiting Consistent Local QSAR</span></div><div class="casAuthors">Sheridan, Robert P.; Hunt, Peter; Culberson, J. Chris</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">180-192</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The idea of a "transformation", making a small change to a chem. structure, for instance removing or replacing a substituent, is familiar to chemists.  We suggest two ways of representing a transformation in silico, as a substructure descriptor difference vector, and as the set of atoms remaining once a max. common substructure is eliminated.  Such transformations can be filtered sensibly, and it is easy to compare one transformation to another.  These representations have two applications.  First, we can use these methods to automatically organize and display sets of closely related compds. such that any consistent local QSAR in a data set can be easily seen, the T-ANALYZE application.  Second, we can suggest to a chemist how to change a mol. "on hand" to a more active one based on local QSAR for that activity, the T-MORPH application.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrM-7rAXqeMnrVg90H21EOLACvtfcHk0ljSFFw4EP4OWg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtFKrt77N&md5=01eff3cb0a2f1a78a18a7c4a3857947b</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1021%2Fci0503208&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci0503208%26sid%3Dliteratum%253Aachs%26aulast%3DSheridan%26aufirst%3DR.%2BP.%26aulast%3DHunt%26aufirst%3DP.%26aulast%3DCulberson%26aufirst%3DJ.%2BC.%26atitle%3DMolecular%2520transformations%2520as%2520a%2520way%2520of%2520finding%2520and%2520exploiting%2520consistent%2520local%2520QSAR%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model%26date%3D2006%26volume%3D46%26spage%3D180%26epage%3D192" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group">Meanwell, N. A.</span><span> </span><span class="NLM_article-title">Synopsis of some recent tactical application of bioisosteres in drug design</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">54</span><span class="NLM_x">, </span> <span class="NLM_fpage">2529</span><span class="NLM_x">–</span> <span class="NLM_lpage">2591</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm1013693" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Fjm200452d&amp;key=1%3ACAS%3A528%3ADC%252BC3MXjsVCnu7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=2529-2591&author=N.+A.+Meanwell&title=Synopsis+of+some+recent+tactical+application+of+bioisosteres+in+drug+design"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Synopsis of Some Recent Tactical Application of Bioisosteres in Drug Design</span></div><div class="casAuthors">Meanwell, Nicholas A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2529-2591</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The established utility of bioisosteres is broad in nature, extending to improving potency, enhancing selectivity, altering phys. properties, reducing or redirecting metab., eliminating or modifying toxicophores, and acquiring novel intellectual property.  In this Perspective, some contemporary themes exploring the role of isosteres in drug design are sampled, with an emphasis placed on tactical applications designed to solve the kinds of problems that impinge on compd. optimization and the long-term success of drug candidates.  Interesting concepts that may have been poorly effective in the context examd. are captured, since the ideas may have merit in alternative circumstances.  A comprehensive cataloging of bioisosteres is beyond the scope of what will be provided, although a synopsis of relevant isosteres of a particular functionality is summarized in a succinct fashion in several sections.  Isosterism has also found productive application in the design and optimization of organocatalysts, and there are several examples in which functional mimicry established initially in a medicinal chem. setting has been adopted by this community.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrZ9z_aPcU4ubVg90H21EOLACvtfcHk0ljSFFw4EP4OWg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXjsVCnu7o%253D&md5=0637ab60c379bef535f3f709b26f3582</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1021%2Fjm1013693&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm1013693%26sid%3Dliteratum%253Aachs%26aulast%3DMeanwell%26aufirst%3DN.%2BA.%26atitle%3DSynopsis%2520of%2520some%2520recent%2520tactical%2520application%2520of%2520bioisosteres%2520in%2520drug%2520design%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D2529%26epage%3D2591" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group">Langdon, S. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ertl, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, N.</span><span> </span><span class="NLM_article-title">Bioisosteric replacement and scaffold hopping in lead generation and optimization</span> <span class="citation_source-journal">Mol. Inf.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">29</span><span class="NLM_x">, </span> <span class="NLM_fpage">366</span><span class="NLM_x">–</span> <span class="NLM_lpage">385</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Fjm200452d&amp;key=10.1002%2Fminf.201000019" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Fjm200452d&amp;key=27463193" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Fjm200452d&amp;key=1%3ACAS%3A528%3ADC%252BC3cXmsVSgur4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2010&pages=366-385&author=S.+R.+Langdonauthor=P.+Ertlauthor=N.+Brown&title=Bioisosteric+replacement+and+scaffold+hopping+in+lead+generation+and+optimization"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Bioisosteric Replacement and Scaffold Hopping in Lead Generation and Optimization</span></div><div class="casAuthors">Langdon, Sarah R.; Ertl, Peter; Brown, Nathan</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Informatics</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">366-385</span>CODEN:
                <span class="NLM_cas:coden">MIONBS</span>;
        ISSN:<span class="NLM_cas:issn">1868-1743</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A review.  Bioisosteric replacement and scaffold hopping are twin methods used in drug design to improve the synthetic accessibility, potency and drug like properties of a compd. and to move into novel chem. space.  Bioisosteric replacement involves swapping functional groups of a mol. with other functional groups that have similar biol. properties.  Scaffold hopping is the replacement of the core framework of a mol. with another scaffold that will improve the properties of the mol. or to find similar potent compds. that exist in novel chem. space.  This review outlines the key concepts, importance and challenges of both methods using examples and comparisons of techniques available for finding bioisosteric replacements and scaffold hops.  There are many methods available for bioisosteric replacement and scaffold hopping, all with their own advantages and disadvantages.  Drug design projects would benefit from a combination of these methods to retrieve diverse and complimentary results.  Continuing progress in these fields will allow further validation of both methods as well as the accumulation of knowledge on bioisosteres and possible scaffold replacements.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpJsbL69-PF-7Vg90H21EOLACvtfcHk0libwGAfJqVBWA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXmsVSgur4%253D&md5=372b9094a34108fb978f5d9fa95831cd</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1002%2Fminf.201000019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fminf.201000019%26sid%3Dliteratum%253Aachs%26aulast%3DLangdon%26aufirst%3DS.%2BR.%26aulast%3DErtl%26aufirst%3DP.%26aulast%3DBrown%26aufirst%3DN.%26atitle%3DBioisosteric%2520replacement%2520and%2520scaffold%2520hopping%2520in%2520lead%2520generation%2520and%2520optimization%26jtitle%3DMol.%2520Inf.%26date%3D2010%26volume%3D29%26spage%3D366%26epage%3D385" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group">Free, S. M.,  Jr.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilson, J. W.</span><span> </span><span class="NLM_article-title">A mathematical contribution to structure–activity studies</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1964</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">395</span><span class="NLM_x">–</span> <span class="NLM_lpage">399</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm00334a001" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=1964&pages=395-399&author=S.+M.+Freeauthor=J.+W.+Wilson&title=A+mathematical+contribution+to+structure%E2%80%93activity+studies"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1021%2Fjm00334a001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm00334a001%26sid%3Dliteratum%253Aachs%26aulast%3DFree%26aufirst%3DS.%2BM.%26aulast%3DWilson%26aufirst%3DJ.%2BW.%26atitle%3DA%2520mathematical%2520contribution%2520to%2520structure%25E2%2580%2593activity%2520studies%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1964%26volume%3D7%26spage%3D395%26epage%3D399" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group">Patel, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gillet, V. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Howe, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pastor, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oyarzabal, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Willett, P.</span><span> </span><span class="NLM_article-title">Assessment of additive/nonadditive effects in structure–activity relationships: implications for iterative drug design</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">7552</span><span class="NLM_x">–</span> <span class="NLM_lpage">7562</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm801070q" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2008&pages=7552-7562&author=Y.+Patelauthor=V.+J.+Gilletauthor=T.+Howeauthor=J.+Pastorauthor=J.+Oyarzabalauthor=P.+Willett&title=Assessment+of+additive%2Fnonadditive+effects+in+structure%E2%80%93activity+relationships%3A+implications+for+iterative+drug+design"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1021%2Fjm801070q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm801070q%26sid%3Dliteratum%253Aachs%26aulast%3DPatel%26aufirst%3DY.%26aulast%3DGillet%26aufirst%3DV.%2BJ.%26aulast%3DHowe%26aufirst%3DT.%26aulast%3DPastor%26aufirst%3DJ.%26aulast%3DOyarzabal%26aufirst%3DJ.%26aulast%3DWillett%26aufirst%3DP.%26atitle%3DAssessment%2520of%2520additive%252Fnonadditive%2520effects%2520in%2520structure%25E2%2580%2593activity%2520relationships%253A%2520implications%2520for%2520iterative%2520drug%2520design%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2008%26volume%3D51%26spage%3D7552%26epage%3D7562" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group">Topliss, J. G.</span><span> </span><span class="NLM_article-title">A manual method for applying the hansch approach to drug design</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1977</span><span class="NLM_x">, </span> <span class="NLM_volume">20</span><span class="NLM_x">, </span> <span class="NLM_fpage">463</span><span class="NLM_x">–</span> <span class="NLM_lpage">469</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm00214a001" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Fjm200452d&amp;key=1%3ACAS%3A528%3ADyaE2sXhtFGgtrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=1977&pages=463-469&author=J.+G.+Topliss&title=A+manual+method+for+applying+the+hansch+approach+to+drug+design"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">A manual method for applying the Hansch approach to drug design</span></div><div class="casAuthors">Topliss, John G.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">1977</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">463-9</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    </div><div class="casAbstract">A procedure is described in which an initial small group of compds. is selected, tested, and ordered according to potency.  The potency order in the group is then compared to the tabulated potency order calcd. for various parameter dependencies relating to hydrophobic, electronic, and steric effects.  From this activity pattern anal. the probable operative parameters can be deduced and a new substituent selection made for the synthesis of potentially more potent analogs.  Application of the method is illustrated with a series of examples.  It differs from a previously described decision tree, single compd. stepwise approach in that it involves the batchwise anal. of small groups of compds., usually the preferred procedure for logistical reasons if the compds. are relatively easy to synthesize.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpILt0RoFSjlbVg90H21EOLACvtfcHk0libwGAfJqVBWA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaE2sXhtFGgtrk%253D&md5=74658b807e3e45cbee88855eda203bdb</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1021%2Fjm00214a001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm00214a001%26sid%3Dliteratum%253Aachs%26aulast%3DTopliss%26aufirst%3DJ.%2BG.%26atitle%3DA%2520manual%2520method%2520for%2520applying%2520the%2520hansch%2520approach%2520to%2520drug%2520design%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1977%26volume%3D20%26spage%3D463%26epage%3D469" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group">Dossetter, A. G.</span><span> </span><span class="NLM_article-title">A statistical analysis of in vitro human microsomal metabolic stability of small phenyl group substituents, leading to improved design sets for parallel SAR exploration of a chemical series</span> <span class="citation_source-journal">Bioorg. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x">, </span> <span class="NLM_fpage">4405</span><span class="NLM_x">–</span> <span class="NLM_lpage">4414</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Fjm200452d&amp;key=10.1016%2Fj.bmc.2010.04.077" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Fjm200452d&amp;key=20510621" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Fjm200452d&amp;key=1%3ACAS%3A528%3ADC%252BC3cXntV2rurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2010&pages=4405-4414&author=A.+G.+Dossetter&title=A+statistical+analysis+of+in+vitro+human+microsomal+metabolic+stability+of+small+phenyl+group+substituents%2C+leading+to+improved+design+sets+for+parallel+SAR+exploration+of+a+chemical+series"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">A statistical analysis of in vitro human microsomal metabolic stability of small phenyl group substituents, leading to improved design sets for parallel SAR exploration of a chemical series</span></div><div class="casAuthors">Dossetter, Alexander G.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">4405-4414</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">An anal. of in vitro human liver microsomal turnover assay results from a large dataset (∼75 K) of exptl. compds. tested is presented.  Combined with an anal. of small (<6 Ha) substituents on known drugs and existing published results a new set of 29 substituents (consensus) is proposed to increase stability and probe SAR (an enhanced Topliss set').  In addn. a different group of 28 substituents are identified as unlikely to change in vitro HLM stability, and a further set of compds. focuses on increasing HLM stability only.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpOwR4TenuEULVg90H21EOLACvtfcHk0lg5GEiPSjWuIg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXntV2rurc%253D&md5=93c9825d5da0b47e0f4c66996c883642</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2010.04.077&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2010.04.077%26sid%3Dliteratum%253Aachs%26aulast%3DDossetter%26aufirst%3DA.%2BG.%26atitle%3DA%2520statistical%2520analysis%2520of%2520in%2520vitro%2520human%2520microsomal%2520metabolic%2520stability%2520of%2520small%2520phenyl%2520group%2520substituents%252C%2520leading%2520to%2520improved%2520design%2520sets%2520for%2520parallel%2520SAR%2520exploration%2520of%2520a%2520chemical%2520series%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2010%26volume%3D18%26spage%3D4405%26epage%3D4414" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group">Sutherland, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">O’Brien, L. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weaver, D. F.</span><span> </span><span class="NLM_article-title">A comparison of methods for modeling quantitative structure–activity relationships</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">5541</span><span class="NLM_x">–</span> <span class="NLM_lpage">5554</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm0497141" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=5541-5554&author=J.+J.+Sutherlandauthor=L.+A.+O%E2%80%99Brienauthor=D.+F.+Weaver&title=A+comparison+of+methods+for+modeling+quantitative+structure%E2%80%93activity+relationships"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1021%2Fjm0497141&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0497141%26sid%3Dliteratum%253Aachs%26aulast%3DSutherland%26aufirst%3DJ.%2BJ.%26aulast%3DO%25E2%2580%2599Brien%26aufirst%3DL.%2BA.%26aulast%3DWeaver%26aufirst%3DD.%2BF.%26atitle%3DA%2520comparison%2520of%2520methods%2520for%2520modeling%2520quantitative%2520structure%25E2%2580%2593activity%2520relationships%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D5541%26epage%3D5554" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group">Katritzky, A. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuanar, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Slavov, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hall, C. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Karelson, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kahn, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dobchev, D. A.</span><span> </span><span class="NLM_article-title">Quantitative correlation of physical and chemical properties with chemical structure: utility for prediction</span> <span class="citation_source-journal">Chem. Rev. (Washington, DC, U. S.)</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">110</span><span class="NLM_x">, </span> <span class="NLM_fpage">5714</span><span class="NLM_x">–</span> <span class="NLM_lpage">5789</span></span><div class="citationLinks">[<a href="/doi/10.1021/cr900238d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Fjm200452d&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtVKmtbnP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=110&publication_year=2010&pages=5714-5789&author=A.+R.+Katritzkyauthor=M.+Kuanarauthor=S.+Slavovauthor=C.+D.+Hallauthor=M.+Karelsonauthor=I.+Kahnauthor=D.+A.+Dobchev&title=Quantitative+correlation+of+physical+and+chemical+properties+with+chemical+structure%3A+utility+for+prediction"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Quantitative Correlation of Physical and Chemical Properties with Chemical Structure: Utility for Prediction</span></div><div class="casAuthors">Katritzky, Alan R.; Kuanar, Minati; Slavov, Svetoslav; Hall, C. Dennis; Karelson, Mati; Kahn, Iiris; Dobchev, Dimitar A.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">110</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">5714-5789</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The present review summarizes recent QSPR research methods and applications.  The main focus is placed on QSPR based upon structural descriptors derived solely from chem. structure for the correlation and prediction of various phys., chem., and physicochem. properties of compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo6wWJhM8cT_7Vg90H21EOLACvtfcHk0lg5GEiPSjWuIg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtVKmtbnP&md5=8fc26f1e548cd8f9e5d3fac6dd4c12bc</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1021%2Fcr900238d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcr900238d%26sid%3Dliteratum%253Aachs%26aulast%3DKatritzky%26aufirst%3DA.%2BR.%26aulast%3DKuanar%26aufirst%3DM.%26aulast%3DSlavov%26aufirst%3DS.%26aulast%3DHall%26aufirst%3DC.%2BD.%26aulast%3DKarelson%26aufirst%3DM.%26aulast%3DKahn%26aufirst%3DI.%26aulast%3DDobchev%26aufirst%3DD.%2BA.%26atitle%3DQuantitative%2520correlation%2520of%2520physical%2520and%2520chemical%2520properties%2520with%2520chemical%2520structure%253A%2520utility%2520for%2520prediction%26jtitle%3DChem.%2520Rev.%2520%2528Washington%252C%2520DC%252C%2520U.%2520S.%2529%26date%3D2010%26volume%3D110%26spage%3D5714%26epage%3D5789" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group">Guha, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Van Drie, J. H.</span><span> </span><span class="NLM_article-title">Assessing how well modeling protocol captures structure–activity landscape</span> <span class="citation_source-journal">J. Chem. Inf. Model</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">48</span><span class="NLM_x">, </span> <span class="NLM_fpage">1716</span><span class="NLM_x">–</span> <span class="NLM_lpage">1728</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci8001414" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Fjm200452d&amp;key=1%3ACAS%3A528%3ADC%252BD1cXps1KnsLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2008&pages=1716-1728&author=R.+Guhaauthor=J.+H.+Van+Drie&title=Assessing+how+well+modeling+protocol+captures+structure%E2%80%93activity+landscape"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Assessing how well modeling protocol captures structure-activity landscape</span></div><div class="casAuthors">Guha, Rajarshi; Van Drie, John H.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1716-1728</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We introduce the notion of structure-activity landscape index (SALI) curves as a way to assess a model and a modeling protocol, applied to structure-activity relationships.  We start from our earlier work [J. Chem. Inf. Model., 2008, 48, 646-658], where we show how to study a structure-activity relationship pairwise, based on the notion of "activity cliffs" - pairs of mols. that are structurally similar but have large differences in activity.  There, we also introduced the SALI parameter, which allows one to identify cliffs easily, and which allows one to represent a structure-activity relationship as a graph.  This graph orders every pair of mols. by their activity.  Here, we introduce the new idea of a SALI curve, which tallies how many of these orderings a model is able to predict.  Empirically, testing these SALI curves against a variety of models, ranging over two-dimensional quant. structure-activity relationship (2D-QSAR), three-dimensional quant. structure-activity relationship (3D-QSAR), and structure-based design models, the utility of a model seems to correspond to characteristics of these curves.  In particular, the integral of these curves, denoted as SCI and being a no. ranging from -1.0 to 1.0, approaches a value of 1.0 for two literature models, which are both known to be prospectively useful.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrWF7pqmO2nIbVg90H21EOLACvtfcHk0lg5GEiPSjWuIg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXps1KnsLY%253D&md5=e2ef0062ee3779ea0ef595dc534d703f</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1021%2Fci8001414&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci8001414%26sid%3Dliteratum%253Aachs%26aulast%3DGuha%26aufirst%3DR.%26aulast%3DVan%2BDrie%26aufirst%3DJ.%2BH.%26atitle%3DAssessing%2520how%2520well%2520modeling%2520protocol%2520captures%2520structure%25E2%2580%2593activity%2520landscape%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model%26date%3D2008%26volume%3D48%26spage%3D1716%26epage%3D1728" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group">Livingstone, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Salt, D. W.</span><span> </span><span class="NLM_article-title">Judging the significance of multiple linear regression models</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">48</span><span class="NLM_x">, </span> <span class="NLM_fpage">661</span><span class="NLM_x">–</span> <span class="NLM_lpage">663</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm049111p" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Fjm200452d&amp;key=1%3ACAS%3A528%3ADC%252BD2MXjtFKmtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2005&pages=661-663&author=D.+J.+Livingstoneauthor=D.+W.+Salt&title=Judging+the+significance+of+multiple+linear+regression+models"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Judging the Significance of Multiple Linear Regression Models</span></div><div class="casAuthors">Livingstone, David J.; Salt, David W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">661-663</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">It is common practice to calc. large nos. of mol. descriptors, apply variable selection procedures to reduce the nos., and then construct multiple linear regression (MLR) models with biol. activity.  The significance of these models is judged using the usual statistical tests.  Unfortunately, these tests are not appropriate under these circumstances since the MLR models suffer from "selection bias".  Expts. with regression using random nos. have generated crit. values (Fmax) with which to assess significance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpZLFVeHNGO77Vg90H21EOLACvtfcHk0lhwWzAew8siSw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXjtFKmtA%253D%253D&md5=784a318f9968b5d65d3cec78b73c3fe0</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1021%2Fjm049111p&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm049111p%26sid%3Dliteratum%253Aachs%26aulast%3DLivingstone%26aufirst%3DD.%2BJ.%26aulast%3DSalt%26aufirst%3DD.%2BW.%26atitle%3DJudging%2520the%2520significance%2520of%2520multiple%2520linear%2520regression%2520models%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2005%26volume%3D48%26spage%3D661%26epage%3D663" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group">Bruneau, P.</span><span> </span><span class="NLM_article-title">Search for predictive generic model of aqueous solubility using Bayesian neural nets</span> <span class="citation_source-journal">J. Chem. Inf. Comput. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">41</span><span class="NLM_x">, </span> <span class="NLM_fpage">1605</span><span class="NLM_x">–</span> <span class="NLM_lpage">1616</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci010363y" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Fjm200452d&amp;key=1%3ACAS%3A528%3ADC%252BD3MXms1Ogtrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2001&pages=1605-1616&author=P.+Bruneau&title=Search+for+predictive+generic+model+of+aqueous+solubility+using+Bayesian+neural+nets"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Search for predictive generic model of aqueous solubility using Bayesian neural nets</span></div><div class="casAuthors">Bruneau, Pierre</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Computer Sciences</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1605-1616</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">0095-2338</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Several predictive models of aq. soly. have been published.  They have good performances on the data sets which have been used for training the models, but usually these data sets do not contain many structures similar to the structures of interest to the drug research and their applicability in drug hunting is questionable.  A very diverse data set has been gathered with compds. issued from literature reports and proprietary compds.  These compds. have been grouped in a so-called literature data set I, a proprietary data set II, and a mixed data set III formed by I and II.  About 100 descriptors emphasizing surface properties were calcd. for every compd.  Bayesian learning of neural nets which cumulates the advantages of neural nets without having their weaknesses was used to select the most parsimonious models and train them, from I, II, and III.  The models were established by either selecting the most efficient descriptors one by one using a modified Gram-Schmidt procedure (GS) or by simplifying a most complete model using automatic relevance procedure (ARD).  The predictive ability of the models was accessed using validation data sets as much unrelated to the training sets as possible, using two new parameters: NDDx,ref the normalized smallest descriptor distance of a compd. x to a ref. data set and CDx,mod the combination of NDDx,ref with the dispersion of the Bayesian neural nets calcns.  The results show that it is possible to obtain a generic predictive model from database I but that the diversity of database II is too restricted to give a model with good generalization ability and that the ARD method applied to the mixed database III gives the best predictive model.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrDYiHCG1pIOrVg90H21EOLACvtfcHk0lhwWzAew8siSw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXms1Ogtrs%253D&md5=b6a2e1c8e3b297b35fd774f7fe3efb33</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1021%2Fci010363y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci010363y%26sid%3Dliteratum%253Aachs%26aulast%3DBruneau%26aufirst%3DP.%26atitle%3DSearch%2520for%2520predictive%2520generic%2520model%2520of%2520aqueous%2520solubility%2520using%2520Bayesian%2520neural%2520nets%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Comput.%2520Sci.%26date%3D2001%26volume%3D41%26spage%3D1605%26epage%3D1616" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span> <span class="citation_source-book">JMP</span>, version <span class="NLM_edition">8.0</span>; <span class="NLM_publisher-name">SAS Institute Inc.</span>: <span class="NLM_publisher-loc">Cary, NC</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=JMP%2C+version+8.0%3B+SAS+Institute+Inc.%3A+Cary%2C+NC."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26btitle%3DJMP%26pub%3DSAS%2520Institute%2520Inc" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group">Papadatos, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Alkarouri, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gillet, V. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Willett, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kadirkamanathan, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luscombe, C. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bravi, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richmond, N. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pickett, S. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hussain, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pritchard, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cooper, A. W. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">MacDonald, S. J. F.</span><span> </span><span class="NLM_article-title">Lead optimization using matched molecular pairs: inclusion of contextual information for enhanced prediction of hERG inhibition, solubility, and lipophilicity</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x">, </span> <span class="NLM_fpage">1872</span><span class="NLM_x">–</span> <span class="NLM_lpage">1886</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci100258p" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Fjm200452d&amp;key=1%3ACAS%3A528%3ADC%252BC3cXht1Whs7jJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2010&pages=1872-1886&author=G.+Papadatosauthor=M.+Alkarouriauthor=V.+J.+Gilletauthor=P.+Willettauthor=V.+Kadirkamanathanauthor=C.+N.+Luscombeauthor=G.+Braviauthor=N.+J.+Richmondauthor=S.+D.+Pickettauthor=J.+Hussainauthor=J.+M.+Pritchardauthor=A.+W.+J.+Cooperauthor=S.+J.+F.+MacDonald&title=Lead+optimization+using+matched+molecular+pairs%3A+inclusion+of+contextual+information+for+enhanced+prediction+of+hERG+inhibition%2C+solubility%2C+and+lipophilicity"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Lead Optimization Using Matched Molecular Pairs: Inclusion of Contextual Information for Enhanced Prediction of hERG Inhibition, Solubility, and Lipophilicity</span></div><div class="casAuthors">Papadatos, George; Alkarouri, Muhammad; Gillet, Valerie J.; Willett, Peter; Kadirkamanathan, Visakan; Luscombe, Christopher N.; Bravi, Gianpaolo; Richmond, Nicola J.; Pickett, Stephen D.; Hussain, Jameed; Pritchard, John M.; Cooper, Anthony W. J.; MacDonald, Simon J. F.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1872-1886</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Previous studies of the anal. of mol. matched pairs (MMPs) have often assumed that the effect of a substructural transformation on a mol. property is independent of the context (i.e., the local structural environment in which that transformation occurs).  Expts. with large sets of hERG, soly., and lipophilicity data demonstrate that the inclusion of contextual information can enhance the predictive power of MMP analyses, with significant trends (both pos. and neg.) being identified that are not apparent when using conventional, context-independent approaches.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoLZfW5E3jgX7Vg90H21EOLACvtfcHk0lhwWzAew8siSw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXht1Whs7jJ&md5=464dff86e047cf1b0273571f82786994</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1021%2Fci100258p&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci100258p%26sid%3Dliteratum%253Aachs%26aulast%3DPapadatos%26aufirst%3DG.%26aulast%3DAlkarouri%26aufirst%3DM.%26aulast%3DGillet%26aufirst%3DV.%2BJ.%26aulast%3DWillett%26aufirst%3DP.%26aulast%3DKadirkamanathan%26aufirst%3DV.%26aulast%3DLuscombe%26aufirst%3DC.%2BN.%26aulast%3DBravi%26aufirst%3DG.%26aulast%3DRichmond%26aufirst%3DN.%2BJ.%26aulast%3DPickett%26aufirst%3DS.%2BD.%26aulast%3DHussain%26aufirst%3DJ.%26aulast%3DPritchard%26aufirst%3DJ.%2BM.%26aulast%3DCooper%26aufirst%3DA.%2BW.%2BJ.%26aulast%3DMacDonald%26aufirst%3DS.%2BJ.%2BF.%26atitle%3DLead%2520optimization%2520using%2520matched%2520molecular%2520pairs%253A%2520inclusion%2520of%2520contextual%2520information%2520for%2520enhanced%2520prediction%2520of%2520hERG%2520inhibition%252C%2520solubility%252C%2520and%2520lipophilicity%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2010%26volume%3D50%26spage%3D1872%26epage%3D1886" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group">Hajduk, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sauer, D. R.</span><span> </span><span class="NLM_article-title">Statistical analysis of the effects of common chemical substituents on ligand potency</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">553</span><span class="NLM_x">–</span> <span class="NLM_lpage">564</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm070838y" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Fjm200452d&amp;key=1%3ACAS%3A528%3ADC%252BD1cXjt1ejtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2008&pages=553-564&author=P.+J.+Hajdukauthor=D.+R.+Sauer&title=Statistical+analysis+of+the+effects+of+common+chemical+substituents+on+ligand+potency"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Statistical Analysis of the Effects of Common Chemical Substituents on Ligand Potency</span></div><div class="casAuthors">Hajduk, Philip J.; Sauer, Daryl R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">553-564</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The results of a statistical anal. of more than 84000 compds. from lead optimization programs against 30 different protein targets is presented, with a focus on the effects that different chem. substituents have on compd. potency.  It is obsd. that the potency changes induced by most chem. groups follows a nearly normal distribution centered near zero (i.e., no effect on potency).  However, the widths of the distributions vary significantly between different substituents, and these effects cannot be rationalized by simple physicochem. parameters.  In addn., certain substituents consistently bias the distribution toward higher or lower potency, suggesting the existence of preferred and nonpreferred chem. groups for lead optimization.  The implications of these results for understanding protein-ligand recognition and for enhancing the efficiency and speed of lead optimization will be discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr9sbNYA_Mzl7Vg90H21EOLACvtfcHk0lgcdqQ2hQN33A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXjt1ejtQ%253D%253D&md5=396d02789aece9be31fe3508fbabca7d</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1021%2Fjm070838y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm070838y%26sid%3Dliteratum%253Aachs%26aulast%3DHajduk%26aufirst%3DP.%2BJ.%26aulast%3DSauer%26aufirst%3DD.%2BR.%26atitle%3DStatistical%2520analysis%2520of%2520the%2520effects%2520of%2520common%2520chemical%2520substituents%2520on%2520ligand%2520potency%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2008%26volume%3D51%26spage%3D553%26epage%3D564" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group">Gleeson, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bravi, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Modi, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lowe, D.</span><span> </span><span class="NLM_article-title">ADMET rules of thumb II: a comparison of the effects of common substituents on a range of ADMET parameters</span> <span class="citation_source-journal">Bioorg. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">17</span><span class="NLM_x">, </span> <span class="NLM_fpage">5906</span><span class="NLM_x">–</span> <span class="NLM_lpage">5919</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Fjm200452d&amp;key=10.1016%2Fj.bmc.2009.07.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Fjm200452d&amp;key=19632124" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Fjm200452d&amp;key=1%3ACAS%3A528%3ADC%252BD1MXps1Crs7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2009&pages=5906-5919&author=P.+Gleesonauthor=G.+Braviauthor=S.+Modiauthor=D.+Lowe&title=ADMET+rules+of+thumb+II%3A+a+comparison+of+the+effects+of+common+substituents+on+a+range+of+ADMET+parameters"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">ADMET rules of thumb II: A comparison of the effects of common substituents on a range of ADMET parameters</span></div><div class="casAuthors">Gleeson, Paul; Bravi, Gianpaolo; Modi, Sandeep; Lowe, Daniel</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">5906-5919</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A systematic anal. of data generated in key in vitro assays within GSK has been undertaken to identify what impact a range of common substituents have on a range of ADMET parameters.  These include; P 450 1A2, 2C9, 2C19, 2D6 and 3A4 inhibition, hERG inhibition, phosphate buffer soly. and artificial membrane permeability.  We do this by identifying all matched mol. pairs, differing by the replacement of a hydrogen atom with a list of predefined substituents.  For each substituent we calc. the mean difference in the ADMET parameter for all the matched mol. pairs identified, making a statistical assessment of the difference, as well as assessing the diversity for each example to ensure that the results can be generalized.  We also relate the change in activity obsd. for each substituent to differences in their mol. properties in an effort to identify any structural alerts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpk7HA_bqdilrVg90H21EOLACvtfcHk0lgcdqQ2hQN33A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXps1Crs7o%253D&md5=fce672cbaadb66c4d7212aa1b92bd7d2</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2009.07.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2009.07.002%26sid%3Dliteratum%253Aachs%26aulast%3DGleeson%26aufirst%3DP.%26aulast%3DBravi%26aufirst%3DG.%26aulast%3DModi%26aufirst%3DS.%26aulast%3DLowe%26aufirst%3DD.%26atitle%3DADMET%2520rules%2520of%2520thumb%2520II%253A%2520a%2520comparison%2520of%2520the%2520effects%2520of%2520common%2520substituents%2520on%2520a%2520range%2520of%2520ADMET%2520parameters%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2009%26volume%3D17%26spage%3D5906%26epage%3D5919" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group">Fujita, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ban, T.</span><span> </span><span class="NLM_article-title">Structure–activity relation. 3. Structure–activity study of phenethylamines as substrates of biosynthetic enzymes of sympathetic transmitters</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1971</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x">, </span> <span class="NLM_fpage">148</span><span class="NLM_x">–</span> <span class="NLM_lpage">152</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm00284a016" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=1971&pages=148-152&author=T.+Fujitaauthor=T.+Ban&title=Structure%E2%80%93activity+relation.+3.+Structure%E2%80%93activity+study+of+phenethylamines+as+substrates+of+biosynthetic+enzymes+of+sympathetic+transmitters"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Fjm00284a016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm00284a016%26sid%3Dliteratum%253Aachs%26aulast%3DFujita%26aufirst%3DT.%26aulast%3DBan%26aufirst%3DT.%26atitle%3DStructure%25E2%2580%2593activity%2520relation.%25203.%2520Structure%25E2%2580%2593activity%2520study%2520of%2520phenethylamines%2520as%2520substrates%2520of%2520biosynthetic%2520enzymes%2520of%2520sympathetic%2520transmitters%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1971%26volume%3D14%26spage%3D148%26epage%3D152" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group">Wassermann, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bajorath, J.</span><span> </span><span class="NLM_article-title">Chemical substitutions that introduce activity cliffs across different compound classes and biological targets</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x">, </span> <span class="NLM_fpage">1248</span><span class="NLM_x">–</span> <span class="NLM_lpage">1256</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci1001845" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Fjm200452d&amp;key=1%3ACAS%3A528%3ADC%252BC3cXosVGhtrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2010&pages=1248-1256&author=A.+M.+Wassermannauthor=J.+Bajorath&title=Chemical+substitutions+that+introduce+activity+cliffs+across+different+compound+classes+and+biological+targets"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Chemical Substitutions That Introduce Activity Cliffs Across Different Compound Classes and Biological Targets</span></div><div class="casAuthors">Wassermann, Anne Mai; Bajorath, Juergen</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1248-1256</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Applying the concept of matched mol. pairs, the authors have systematically analyzed the ability of defined chem. changes to introduce activity cliffs.  Public domain compd. data were systematically screened for matched mol. pairs that were then organized according to chem. transformations they represent and assocd. potency changes.  From vast available chem. transformation space, including both R-group and core substructure changes, ∼250 nonredundant substitutions were identified that displayed a general tendency to form activity cliffs.  These substitutions introduced activity cliffs in the structural context of diverse scaffolds and in compds. active against many different targets.  Activity cliff-forming transformations were often rather simple, including replacements of small functional groups.  Moreover, in many instances, chem. very similar transformations were identified that had a much lower propensity to form activity cliffs or no detectable cliff potential.  Thus, clear preferences emerged for specific transformations.  A compendium of substitutions with general activity cliff-forming potential is provided to aid in compd. optimization efforts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpUSS8Tj-rk8bVg90H21EOLACvtfcHk0lgcdqQ2hQN33A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXosVGhtrY%253D&md5=381262f83abb19d326dbf95ddc367565</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1021%2Fci1001845&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci1001845%26sid%3Dliteratum%253Aachs%26aulast%3DWassermann%26aufirst%3DA.%2BM.%26aulast%3DBajorath%26aufirst%3DJ.%26atitle%3DChemical%2520substitutions%2520that%2520introduce%2520activity%2520cliffs%2520across%2520different%2520compound%2520classes%2520and%2520biological%2520targets%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2010%26volume%3D50%26spage%3D1248%26epage%3D1256" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group">Ames, B. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gurney, E. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miller, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bartsch, H.</span><span> </span><span class="NLM_article-title">Carcinogens as frameshift mutagens. Metabolites and derivatives of 2-acetylaminofluorene and other aromatic amine carcinogens</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U.S.A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1972</span><span class="NLM_x">, </span> <span class="NLM_volume">69</span><span class="NLM_x">, </span> <span class="NLM_fpage">3128</span><span class="NLM_x">–</span> <span class="NLM_lpage">3132</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=69&publication_year=1972&pages=3128-3132&author=B.+N.+Amesauthor=E.+G.+Gurneyauthor=J.+A.+Millerauthor=H.+Bartsch&title=Carcinogens+as+frameshift+mutagens.+Metabolites+and+derivatives+of+2-acetylaminofluorene+and+other+aromatic+amine+carcinogens"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DAmes%26aufirst%3DB.%2BN.%26aulast%3DGurney%26aufirst%3DE.%2BG.%26aulast%3DMiller%26aufirst%3DJ.%2BA.%26aulast%3DBartsch%26aufirst%3DH.%26atitle%3DCarcinogens%2520as%2520frameshift%2520mutagens.%2520Metabolites%2520and%2520derivatives%2520of%25202-acetylaminofluorene%2520and%2520other%2520aromatic%2520amine%2520carcinogens%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D1972%26volume%3D69%26spage%3D3128%26epage%3D3132" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group">Lewis, R. A.</span><span> </span><span class="NLM_article-title">Computer-aided drug design 2005–2007</span> <span class="citation_source-journal">Chem. Modell.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">51</span><span class="NLM_x">–</span> <span class="NLM_lpage">66</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Fjm200452d&amp;key=10.1039%2Fb609116b" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Fjm200452d&amp;key=1%3ACAS%3A528%3ADC%252BD1MXmvFWksg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2008&pages=51-66&author=R.+A.+Lewis&title=Computer-aided+drug+design+2005%E2%80%932007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Computer-aided drug design 2005-2007</span></div><div class="casAuthors">Lewis, Richard A.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Modelling</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">51-66</span>CODEN:
                <span class="NLM_cas:coden">CMHOBG</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A review on QSAR and ADMET, structure-based drug design, virtual screening, de novo structure generation, fragment-based screening, target fishing, and library design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpEqkZ5mtiqf7Vg90H21EOLACvtfcHk0lhwghmQd0-pEA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXmvFWksg%253D%253D&md5=90468069e401676e1774a88c2d2c8129</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1039%2Fb609116b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fb609116b%26sid%3Dliteratum%253Aachs%26aulast%3DLewis%26aufirst%3DR.%2BA.%26atitle%3DComputer-aided%2520drug%2520design%25202005%25E2%2580%25932007%26jtitle%3DChem.%2520Modell.%26date%3D2008%26volume%3D5%26spage%3D51%26epage%3D66" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group">Lewis, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cucurull-Sanchez, L.</span><span> </span><span class="NLM_article-title">Structural pairwise comparisons of HLM stability of phenyl derivatives: introduction of the Pfizer metabolism index (PMI) and metabolism-lipophilicity efficiency (MLE)</span> <span class="citation_source-journal">J. Comput.-Aided Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">23</span><span class="NLM_x">, </span> <span class="NLM_fpage">97</span><span class="NLM_x">–</span> <span class="NLM_lpage">103</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Fjm200452d&amp;key=10.1007%2Fs10822-008-9242-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Fjm200452d&amp;key=18802667" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Fjm200452d&amp;key=1%3ACAS%3A528%3ADC%252BD1MXmtFGg" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2009&pages=97-103&author=M.+L.+Lewisauthor=L.+Cucurull-Sanchez&title=Structural+pairwise+comparisons+of+HLM+stability+of+phenyl+derivatives%3A+introduction+of+the+Pfizer+metabolism+index+%28PMI%29+and+metabolism-lipophilicity+efficiency+%28MLE%29"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Structural pairwise comparisons of HLM stability of phenyl derivatives: Introduction of the Pfizer metabolism index (PMI) and metabolism-lipophilicity efficiency (MLE)</span></div><div class="casAuthors">Lewis, Mark L.; Cucurull-Sanchez, Lourdes</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">97-103</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Data mining by pairwise comparison of over 150,000 human liver microsome (HLM) intrinsic clearance values stored within the internal Pfizer database has been performed by an automated tool.  Systematic probability tables of specific structural changes on the intrinsic clearance of Ph derivs. were generated.  From these data two new parameters, the Pfizer Metab. Index (PMI) and Metab.-Lipophilicity Efficiency (MLE) are introduced for each fragment.  The findings are applied to a Topliss style anal. that focuses on metabolic stability.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp-t3Rk6sR_vLVg90H21EOLACvtfcHk0lhwghmQd0-pEA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXmtFGg&md5=5088f2c94bce164a302df21936da9a43</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1007%2Fs10822-008-9242-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-008-9242-3%26sid%3Dliteratum%253Aachs%26aulast%3DLewis%26aufirst%3DM.%2BL.%26aulast%3DCucurull-Sanchez%26aufirst%3DL.%26atitle%3DStructural%2520pairwise%2520comparisons%2520of%2520HLM%2520stability%2520of%2520phenyl%2520derivatives%253A%2520introduction%2520of%2520the%2520Pfizer%2520metabolism%2520index%2520%2528PMI%2529%2520and%2520metabolism-lipophilicity%2520efficiency%2520%2528MLE%2529%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2009%26volume%3D23%26spage%3D97%26epage%3D103" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group">Cucurull-Sanchez, L.</span><span> </span><span class="NLM_article-title">Successful identification of key chemical structure modifications that lead to improved ADME profiles</span> <span class="citation_source-journal">J. Comput.-Aided Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">24</span><span class="NLM_x">, </span> <span class="NLM_fpage">449</span><span class="NLM_x">–</span> <span class="NLM_lpage">458</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2010&pages=449-458&author=L.+Cucurull-Sanchez&title=Successful+identification+of+key+chemical+structure+modifications+that+lead+to+improved+ADME+profiles"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DCucurull-Sanchez%26aufirst%3DL.%26atitle%3DSuccessful%2520identification%2520of%2520key%2520chemical%2520structure%2520modifications%2520that%2520lead%2520to%2520improved%2520ADME%2520profiles%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2010%26volume%3D24%26spage%3D449%26epage%3D458" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group">Gall, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCall, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">TenBrink, R. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">VonVoigtlander, P. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mohrland, J. S.</span><span> </span><span class="NLM_article-title">Arylformamidines with antinociceptive properties</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1988</span><span class="NLM_x">, </span> <span class="NLM_volume">31</span><span class="NLM_x">, </span> <span class="NLM_fpage">1816</span><span class="NLM_x">–</span> <span class="NLM_lpage">1820</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm00117a023" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=1988&pages=1816-1820&author=M.+Gallauthor=J.+M.+McCallauthor=R.+E.+TenBrinkauthor=P.+F.+VonVoigtlanderauthor=J.+S.+Mohrland&title=Arylformamidines+with+antinociceptive+properties"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1021%2Fjm00117a023&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm00117a023%26sid%3Dliteratum%253Aachs%26aulast%3DGall%26aufirst%3DM.%26aulast%3DMcCall%26aufirst%3DJ.%2BM.%26aulast%3DTenBrink%26aufirst%3DR.%2BE.%26aulast%3DVonVoigtlander%26aufirst%3DP.%2BF.%26aulast%3DMohrland%26aufirst%3DJ.%2BS.%26atitle%3DArylformamidines%2520with%2520antinociceptive%2520properties%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1988%26volume%3D31%26spage%3D1816%26epage%3D1820" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group">Blaszczak, L. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, R. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cook, G. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hornback, W. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Indelicato, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jordan, C. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Katner, A. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kinnick, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McDonald, J. H.,  III</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morin, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Munroe, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pasini, C. E.</span><span> </span><span class="NLM_article-title">Comparative reactivity of 1-carba-1-dethiacephalosporins with cephalosporins</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1990</span><span class="NLM_x">, </span> <span class="NLM_volume">33</span><span class="NLM_x">, </span> <span class="NLM_fpage">1656</span><span class="NLM_x">–</span> <span class="NLM_lpage">1662</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm00168a019" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=1990&pages=1656-1662&author=L.+C.+Blaszczakauthor=R.+F.+Brownauthor=G.+K.+Cookauthor=W.+J.+Hornbackauthor=J.+M.+Indelicatoauthor=C.+L.+Jordanauthor=A.+S.+Katnerauthor=M.+D.+Kinnickauthor=J.+H.+McDonaldauthor=J.+M.+Morinauthor=J.+E.+Munroeauthor=C.+E.+Pasini&title=Comparative+reactivity+of+1-carba-1-dethiacephalosporins+with+cephalosporins"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Fjm00168a019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm00168a019%26sid%3Dliteratum%253Aachs%26aulast%3DBlaszczak%26aufirst%3DL.%2BC.%26aulast%3DBrown%26aufirst%3DR.%2BF.%26aulast%3DCook%26aufirst%3DG.%2BK.%26aulast%3DHornback%26aufirst%3DW.%2BJ.%26aulast%3DIndelicato%26aufirst%3DJ.%2BM.%26aulast%3DJordan%26aufirst%3DC.%2BL.%26aulast%3DKatner%26aufirst%3DA.%2BS.%26aulast%3DKinnick%26aufirst%3DM.%2BD.%26aulast%3DMcDonald%26aufirst%3DJ.%2BH.%26aulast%3DMorin%26aufirst%3DJ.%2BM.%26aulast%3DMunroe%26aufirst%3DJ.%2BE.%26aulast%3DPasini%26aufirst%3DC.%2BE.%26atitle%3DComparative%2520reactivity%2520of%25201-carba-1-dethiacephalosporins%2520with%2520cephalosporins%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1990%26volume%3D33%26spage%3D1656%26epage%3D1662" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group">Herberich, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cao, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chakrabarti, P. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Falsey, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pettus, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rzasa, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reed, A. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reichelt, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sham, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thaman, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wurz, R. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hsieh, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Syed, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grosfeld, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Plant, M. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Henkle, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sherman, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Middleton, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wong, L. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tasker, A. S.</span><span> </span><span class="NLM_article-title">Discovery of highly selective and potent p38 inhibitors based on a phthalazine scaffold</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">6271</span><span class="NLM_x">–</span> <span class="NLM_lpage">6279</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm8005417" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2008&pages=6271-6279&author=B.+Herberichauthor=G.+Caoauthor=P.+P.+Chakrabartiauthor=J.+R.+Falseyauthor=L.+Pettusauthor=R.+M.+Rzasaauthor=A.+B.+Reedauthor=A.+Reicheltauthor=K.+Shamauthor=M.+Thamanauthor=R.+P.+Wurzauthor=S.+Xuauthor=D.+Zhangauthor=F.+Hsiehauthor=M.+R.+Leeauthor=R.+Syedauthor=V.+Liauthor=D.+Grosfeldauthor=M.+H.+Plantauthor=B.+Henkleauthor=L.+Shermanauthor=S.+Middletonauthor=L.+M.+Wongauthor=A.+S.+Tasker&title=Discovery+of+highly+selective+and+potent+p38+inhibitors+based+on+a+phthalazine+scaffold"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1021%2Fjm8005417&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm8005417%26sid%3Dliteratum%253Aachs%26aulast%3DHerberich%26aufirst%3DB.%26aulast%3DCao%26aufirst%3DG.%26aulast%3DChakrabarti%26aufirst%3DP.%2BP.%26aulast%3DFalsey%26aufirst%3DJ.%2BR.%26aulast%3DPettus%26aufirst%3DL.%26aulast%3DRzasa%26aufirst%3DR.%2BM.%26aulast%3DReed%26aufirst%3DA.%2BB.%26aulast%3DReichelt%26aufirst%3DA.%26aulast%3DSham%26aufirst%3DK.%26aulast%3DThaman%26aufirst%3DM.%26aulast%3DWurz%26aufirst%3DR.%2BP.%26aulast%3DXu%26aufirst%3DS.%26aulast%3DZhang%26aufirst%3DD.%26aulast%3DHsieh%26aufirst%3DF.%26aulast%3DLee%26aufirst%3DM.%2BR.%26aulast%3DSyed%26aufirst%3DR.%26aulast%3DLi%26aufirst%3DV.%26aulast%3DGrosfeld%26aufirst%3DD.%26aulast%3DPlant%26aufirst%3DM.%2BH.%26aulast%3DHenkle%26aufirst%3DB.%26aulast%3DSherman%26aufirst%3DL.%26aulast%3DMiddleton%26aufirst%3DS.%26aulast%3DWong%26aufirst%3DL.%2BM.%26aulast%3DTasker%26aufirst%3DA.%2BS.%26atitle%3DDiscovery%2520of%2520highly%2520selective%2520and%2520potent%2520p38%2520inhibitors%2520based%2520on%2520a%2520phthalazine%2520scaffold%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2008%26volume%3D51%26spage%3D6271%26epage%3D6279" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group">Shi, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sitkoff, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klei, H. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kish, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, E. C. -.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hartl, K. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Seiler, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chang, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Youssef, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Steinbacher, T. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schumacher, W. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grazier, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pudzianowski, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Apedo, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Discenza, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yanchunas, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stein, P. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Atwal, K. S.</span><span> </span><span class="NLM_article-title">Design, structure-activity relationships, X-ray crystal structure, and energetic contributions of a critical P1 pharmacophore: 3-chloroindole-7-yl-based factor Xa inhibitors</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">7541</span><span class="NLM_x">–</span> <span class="NLM_lpage">7551</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm800855x" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2008&pages=7541-7551&author=Y.+Shiauthor=D.+Sitkoffauthor=J.+Zhangauthor=H.+E.+Kleiauthor=K.+Kishauthor=E.+C.+-.+Liuauthor=K.+S.+Hartlauthor=S.+M.+Seilerauthor=M.+Changauthor=C.+Huangauthor=S.+Youssefauthor=T.+E.+Steinbacherauthor=W.+A.+Schumacherauthor=N.+Grazierauthor=A.+Pudzianowskiauthor=A.+Apedoauthor=L.+Discenzaauthor=J.+Yanchunasauthor=P.+D.+Steinauthor=K.+S.+Atwal&title=Design%2C+structure-activity+relationships%2C+X-ray+crystal+structure%2C+and+energetic+contributions+of+a+critical+P1+pharmacophore%3A+3-chloroindole-7-yl-based+factor+Xa+inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Fjm800855x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm800855x%26sid%3Dliteratum%253Aachs%26aulast%3DShi%26aufirst%3DY.%26aulast%3DSitkoff%26aufirst%3DD.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DKlei%26aufirst%3DH.%2BE.%26aulast%3DKish%26aufirst%3DK.%26aulast%3DLiu%26aufirst%3DE.%2BC.%2B-.%26aulast%3DHartl%26aufirst%3DK.%2BS.%26aulast%3DSeiler%26aufirst%3DS.%2BM.%26aulast%3DChang%26aufirst%3DM.%26aulast%3DHuang%26aufirst%3DC.%26aulast%3DYoussef%26aufirst%3DS.%26aulast%3DSteinbacher%26aufirst%3DT.%2BE.%26aulast%3DSchumacher%26aufirst%3DW.%2BA.%26aulast%3DGrazier%26aufirst%3DN.%26aulast%3DPudzianowski%26aufirst%3DA.%26aulast%3DApedo%26aufirst%3DA.%26aulast%3DDiscenza%26aufirst%3DL.%26aulast%3DYanchunas%26aufirst%3DJ.%26aulast%3DStein%26aufirst%3DP.%2BD.%26aulast%3DAtwal%26aufirst%3DK.%2BS.%26atitle%3DDesign%252C%2520structure-activity%2520relationships%252C%2520X-ray%2520crystal%2520structure%252C%2520and%2520energetic%2520contributions%2520of%2520a%2520critical%2520P1%2520pharmacophore%253A%25203-chloroindole-7-yl-based%2520factor%2520Xa%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2008%26volume%3D51%26spage%3D7541%26epage%3D7551" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group">Ioannidis, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lamb, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Almeida, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Block, M. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davies, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peng, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Su, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hoffmann, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rivard, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Green, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Howard, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pollard, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Read, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Alimzhanov, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bebernitz, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bell, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ye, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huszar, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zinda, M.</span><span> </span><span class="NLM_article-title">Discovery of 5-chloro-N2-[(1<i>S</i>)-1-(5-fluoropyrimidin-2-yl)ethyl]-N4-(5-methyl-1<i>H</i>-pyrazol-3-yl)pyrimidine-2,4-diamine (AZD1480) as a novel inhibitor of the Jak/Stat pathway</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">54</span><span class="NLM_x">, </span> <span class="NLM_fpage">262</span><span class="NLM_x">–</span> <span class="NLM_lpage">276</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm1011319" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Fjm200452d&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsFarsr%252FN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=262-276&author=S.+Ioannidisauthor=M.+L.+Lambauthor=T.+Wangauthor=L.+Almeidaauthor=M.+H.+Blockauthor=A.+M.+Daviesauthor=B.+Pengauthor=M.+Suauthor=H.+Zhangauthor=E.+Hoffmannauthor=C.+Rivardauthor=I.+Greenauthor=T.+Howardauthor=H.+Pollardauthor=J.+Readauthor=M.+Alimzhanovauthor=G.+Bebernitzauthor=K.+Bellauthor=M.+Yeauthor=D.+Huszarauthor=M.+Zinda&title=Discovery+of+5-chloro-N2-%5B%281S%29-1-%285-fluoropyrimidin-2-yl%29ethyl%5D-N4-%285-methyl-1H-pyrazol-3-yl%29pyrimidine-2%2C4-diamine+%28AZD1480%29+as+a+novel+inhibitor+of+the+Jak%2FStat+pathway"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of 5-Chloro-N2-[(1S)-1-(5-fluoropyrimidin-2-yl)ethyl]-N4-(5-methyl-1H-pyrazol-3-yl)pyrimidine-2,4-diamine (AZD1480) as a Novel Inhibitor of the Jak/Stat Pathway</span></div><div class="casAuthors">Ioannidis, Stephanos; Lamb, Michelle L.; Wang, Tao; Almeida, Lynsie; Block, Michael H.; Davies, Audrey M.; Peng, Bo; Su, Mei; Zhang, Hai-Jun; Hoffmann, Ethan; Rivard, Caroline; Green, Isabelle; Howard, Tina; Pollard, Hannah; Read, Jon; Alimzhanov, Marat; Bebernitz, Geraldine; Bell, Kirsten; Ye, Min-Wei; Huszar, Dennis; Zinda, Michael</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">262-276</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The myeloproliferative neoplasms, polycythemia vera, essential thrombocythemia, and idiopathic myelofibrosis are a heterogeneous but related group of hematol. malignancies characterized by clonal expansion of one or more myeloid lineages.  The discovery of the Jak2 V617F gain of function mutation highlighted Jak2 as a potential therapeutic target in the MPNs.  Herein, we disclose the discovery of a series of pyrazol-3-yl pyrimidin-4-amines and the identification of 9e (AZD1480) as a potent Jak2 inhibitor.  9E inhibits signaling and proliferation of Jak2 V617F cell lines in vitro, demonstrates in vivo efficacy in a TEL-Jak2 model, has excellent phys. properties and preclin. pharmacokinetics, and is currently being evaluated in Phase I clin. trials.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMG6RQUv4HqbVg90H21EOLACvtfcHk0lhwtbNoZeNOeA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsFarsr%252FN&md5=c8141c5b0e25a007e68b6e2567a576fe</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Fjm1011319&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm1011319%26sid%3Dliteratum%253Aachs%26aulast%3DIoannidis%26aufirst%3DS.%26aulast%3DLamb%26aufirst%3DM.%2BL.%26aulast%3DWang%26aufirst%3DT.%26aulast%3DAlmeida%26aufirst%3DL.%26aulast%3DBlock%26aufirst%3DM.%2BH.%26aulast%3DDavies%26aufirst%3DA.%2BM.%26aulast%3DPeng%26aufirst%3DB.%26aulast%3DSu%26aufirst%3DM.%26aulast%3DZhang%26aufirst%3DH.%26aulast%3DHoffmann%26aufirst%3DE.%26aulast%3DRivard%26aufirst%3DC.%26aulast%3DGreen%26aufirst%3DI.%26aulast%3DHoward%26aufirst%3DT.%26aulast%3DPollard%26aufirst%3DH.%26aulast%3DRead%26aufirst%3DJ.%26aulast%3DAlimzhanov%26aufirst%3DM.%26aulast%3DBebernitz%26aufirst%3DG.%26aulast%3DBell%26aufirst%3DK.%26aulast%3DYe%26aufirst%3DM.%26aulast%3DHuszar%26aufirst%3DD.%26aulast%3DZinda%26aufirst%3DM.%26atitle%3DDiscovery%2520of%25205-chloro-N2-%255B%25281S%2529-1-%25285-fluoropyrimidin-2-yl%2529ethyl%255D-N4-%25285-methyl-1H-pyrazol-3-yl%2529pyrimidine-2%252C4-diamine%2520%2528AZD1480%2529%2520as%2520a%2520novel%2520inhibitor%2520of%2520the%2520Jak%252FStat%2520pathway%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D262%26epage%3D276" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group">Andrews, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gibson, K. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Graham, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Matusiak, Z. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roberts, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stokes, E. S. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brady, M. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chresta, C. M.</span><span> </span><span class="NLM_article-title">Design and campaign synthesis of pyridine-based histone deacetylase inhibitors</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x">, </span> <span class="NLM_fpage">2525</span><span class="NLM_x">–</span> <span class="NLM_lpage">2529</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2008&pages=2525-2529&author=D.+M.+Andrewsauthor=K.+M.+Gibsonauthor=M.+A.+Grahamauthor=Z.+S.+Matusiakauthor=C.+A.+Robertsauthor=E.+S.+E.+Stokesauthor=M.+C.+Bradyauthor=C.+M.+Chresta&title=Design+and+campaign+synthesis+of+pyridine-based+histone+deacetylase+inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DAndrews%26aufirst%3DD.%2BM.%26aulast%3DGibson%26aufirst%3DK.%2BM.%26aulast%3DGraham%26aufirst%3DM.%2BA.%26aulast%3DMatusiak%26aufirst%3DZ.%2BS.%26aulast%3DRoberts%26aufirst%3DC.%2BA.%26aulast%3DStokes%26aufirst%3DE.%2BS.%2BE.%26aulast%3DBrady%26aufirst%3DM.%2BC.%26aulast%3DChresta%26aufirst%3DC.%2BM.%26atitle%3DDesign%2520and%2520campaign%2520synthesis%2520of%2520pyridine-based%2520histone%2520deacetylase%2520inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2008%26volume%3D18%26spage%3D2525%26epage%3D2529" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group">Weininger, D.</span><span> </span><span class="NLM_article-title">SMILES, a chemical language and information system. 1. Introduction to methodology and encoding rules</span> <span class="citation_source-journal">J. Chem. Inf. Comput. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1988</span><span class="NLM_x">, </span> <span class="NLM_volume">28</span><span class="NLM_x">, </span> <span class="NLM_fpage">31</span><span class="NLM_x">–</span> <span class="NLM_lpage">36</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci00057a005" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Fjm200452d&amp;key=1%3ACAS%3A528%3ADyaL1cXnsVeqsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=1988&pages=31-36&author=D.+Weininger&title=SMILES%2C+a+chemical+language+and+information+system.+1.+Introduction+to+methodology+and+encoding+rules"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">SMILES, a chemical language and information system.  1.  Introduction to methodology and encoding rules</span></div><div class="casAuthors">Weininger, David</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Computer Sciences</span>
        (<span class="NLM_cas:date">1988</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">31-6</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">0095-2338</span>.
    </div><div class="casAbstract">The SMILES (simplified mol. input line entry system) chem. notation system is described for information processing.  The system is based on principles of mol. graph theory and it allows structure specification by use of a very small and natural grammar well suited for high-speed machine processing.  The system is easy to use, has high machine compatibility, and allows many computer applications, including notation generation, const. speed database retrieval, substructure searching, and property prediction models.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpYNGvUnFxDlLVg90H21EOLACvtfcHk0lhwtbNoZeNOeA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL1cXnsVeqsA%253D%253D&md5=04592975f9dd3c0ce3c1ad618ba2b17d</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1021%2Fci00057a005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci00057a005%26sid%3Dliteratum%253Aachs%26aulast%3DWeininger%26aufirst%3DD.%26atitle%3DSMILES%252C%2520a%2520chemical%2520language%2520and%2520information%2520system.%25201.%2520Introduction%2520to%2520methodology%2520and%2520encoding%2520rules%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Comput.%2520Sci.%26date%3D1988%26volume%3D28%26spage%3D31%26epage%3D36" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group">Andersson, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Armstrong, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bjoere, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bowker, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chapman, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davies, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Donald, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Egner, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Elebring, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Holmqvist, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Inghardt, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johannesson, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johansson, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johnstone, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kemmitt, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kihlberg, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Korsgren, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lemurell, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moore, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pettersson, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pointon, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ponten, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schofield, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Selmi, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Whittamore, P.</span><span> </span><span class="NLM_article-title">Making medicinal chemistry more effective—application of Lean Sigma to improve processes, speed and quality</span> <span class="citation_source-journal">Drug Discovery Today</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x">, </span> <span class="NLM_fpage">598</span><span class="NLM_x">–</span> <span class="NLM_lpage">604</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2009&pages=598-604&author=S.+Anderssonauthor=A.+Armstrongauthor=A.+Bjoereauthor=S.+Bowkerauthor=S.+Chapmanauthor=R.+Daviesauthor=C.+Donaldauthor=B.+Egnerauthor=T.+Elebringauthor=S.+Holmqvistauthor=T.+Inghardtauthor=P.+Johannessonauthor=M.+Johanssonauthor=C.+Johnstoneauthor=P.+Kemmittauthor=J.+Kihlbergauthor=P.+Korsgrenauthor=M.+Lemurellauthor=J.+Mooreauthor=J.+A.+Petterssonauthor=H.+Pointonauthor=F.+Pontenauthor=P.+Schofieldauthor=N.+Selmiauthor=P.+Whittamore&title=Making+medicinal+chemistry+more+effective%E2%80%94application+of+Lean+Sigma+to+improve+processes%2C+speed+and+quality"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DAndersson%26aufirst%3DS.%26aulast%3DArmstrong%26aufirst%3DA.%26aulast%3DBjoere%26aufirst%3DA.%26aulast%3DBowker%26aufirst%3DS.%26aulast%3DChapman%26aufirst%3DS.%26aulast%3DDavies%26aufirst%3DR.%26aulast%3DDonald%26aufirst%3DC.%26aulast%3DEgner%26aufirst%3DB.%26aulast%3DElebring%26aufirst%3DT.%26aulast%3DHolmqvist%26aufirst%3DS.%26aulast%3DInghardt%26aufirst%3DT.%26aulast%3DJohannesson%26aufirst%3DP.%26aulast%3DJohansson%26aufirst%3DM.%26aulast%3DJohnstone%26aufirst%3DC.%26aulast%3DKemmitt%26aufirst%3DP.%26aulast%3DKihlberg%26aufirst%3DJ.%26aulast%3DKorsgren%26aufirst%3DP.%26aulast%3DLemurell%26aufirst%3DM.%26aulast%3DMoore%26aufirst%3DJ.%26aulast%3DPettersson%26aufirst%3DJ.%2BA.%26aulast%3DPointon%26aufirst%3DH.%26aulast%3DPonten%26aufirst%3DF.%26aulast%3DSchofield%26aufirst%3DP.%26aulast%3DSelmi%26aufirst%3DN.%26aulast%3DWhittamore%26aufirst%3DP.%26atitle%3DMaking%2520medicinal%2520chemistry%2520more%2520effective%25E2%2580%2594application%2520of%2520Lean%2520Sigma%2520to%2520improve%2520processes%252C%2520speed%2520and%2520quality%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2009%26volume%3D14%26spage%3D598%26epage%3D604" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group">Birch, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kenny, P. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simpson, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Whittamore, P. R. O.</span><span> </span><span class="NLM_article-title">Matched molecular pair analysis of activity and properties of glycogen phosphorylase inhibitors</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">850</span><span class="NLM_x">–</span> <span class="NLM_lpage">853</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2009&pages=850-853&author=A.+M.+Birchauthor=P.+W.+Kennyauthor=I.+Simpsonauthor=P.+R.+O.+Whittamore&title=Matched+molecular+pair+analysis+of+activity+and+properties+of+glycogen+phosphorylase+inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBirch%26aufirst%3DA.%2BM.%26aulast%3DKenny%26aufirst%3DP.%2BW.%26aulast%3DSimpson%26aufirst%3DI.%26aulast%3DWhittamore%26aufirst%3DP.%2BR.%2BO.%26atitle%3DMatched%2520molecular%2520pair%2520analysis%2520of%2520activity%2520and%2520properties%2520of%2520glycogen%2520phosphorylase%2520inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2009%26volume%3D19%26spage%3D850%26epage%3D853" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group">Morley, A. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kenny, P. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Burton, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heald, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">MacFaul, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mullett, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Page, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Porres, S. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ribeiro, L. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ward, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilkinson, T. J.</span><span> </span><span class="NLM_article-title">5-Aminopyrimidin-2-ylnitriles as cathepsin K inhibitors</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">1658</span><span class="NLM_x">–</span> <span class="NLM_lpage">1661</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2009&pages=1658-1661&author=A.+D.+Morleyauthor=P.+W.+Kennyauthor=B.+Burtonauthor=R.+A.+Healdauthor=P.+A.+MacFaulauthor=J.+Mullettauthor=K.+Pageauthor=S.+S.+Porresauthor=L.+R.+Ribeiroauthor=P.+Smithauthor=S.+Wardauthor=T.+J.+Wilkinson&title=5-Aminopyrimidin-2-ylnitriles+as+cathepsin+K+inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMorley%26aufirst%3DA.%2BD.%26aulast%3DKenny%26aufirst%3DP.%2BW.%26aulast%3DBurton%26aufirst%3DB.%26aulast%3DHeald%26aufirst%3DR.%2BA.%26aulast%3DMacFaul%26aufirst%3DP.%2BA.%26aulast%3DMullett%26aufirst%3DJ.%26aulast%3DPage%26aufirst%3DK.%26aulast%3DPorres%26aufirst%3DS.%2BS.%26aulast%3DRibeiro%26aufirst%3DL.%2BR.%26aulast%3DSmith%26aufirst%3DP.%26aulast%3DWard%26aufirst%3DS.%26aulast%3DWilkinson%26aufirst%3DT.%2BJ.%26atitle%3D5-Aminopyrimidin-2-ylnitriles%2520as%2520cathepsin%2520K%2520inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2009%26volume%3D19%26spage%3D1658%26epage%3D1661" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37a"><span> <span class="citation_source-book">SMARTS</span>; <span class="NLM_publisher-name">Daylight Chemical Information Systems Inc.</span>: <span class="NLM_publisher-loc">Santa Fe, NM</span>, Vol  <span class="NLM_volume">471</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=SMARTS%3B+Daylight+Chemical+Information+Systems+Inc.%3A+Santa+Fe%2C+NM%2C+Vol+471."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit37a&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26btitle%3DSMARTS%26pub%3DDaylight%2520Chemical%2520Information%2520Systems%2520Inc%26volume%3D471" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit37b"><span><a href="http://www.daylight.com/dayhtml/doc/theory/theory.smarts.html" class="extLink">http://www.daylight.com/dayhtml/doc/theory/theory.smarts.html</a>, .</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=http%3A%2F%2Fwww.daylight.com%2Fdayhtml%2Fdoc%2Ftheory%2Ftheory.smarts.html%2C+."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit37b&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweb%26sid%3Dliteratum%253Aachs" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group">Jones, H. D.; Cosgrove, D. A.</span> <span class="citation_source-book">find_pairs</span>; <span class="NLM_publisher-name">AstraZeneca Pharmaceuticals</span>: <span class="NLM_publisher-loc">Macclesfield, U.K.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Jones%2C+H.+D.%3B+Cosgrove%2C+D.+A.find_pairs%3B+AstraZeneca+Pharmaceuticals%3A+Macclesfield%2C+U.K."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DJones%26aufirst%3DH.%2BD.%26btitle%3Dfind_pairs%26pub%3DAstraZeneca%2520Pharmaceuticals" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group">Cosgrove, D. A.</span> <span class="citation_source-book">thrice_pairs</span>; <span class="NLM_publisher-name">AstraZeneca Pharmaceuticals</span>: <span class="NLM_publisher-loc">Macclesfield, U.K.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Cosgrove%2C+D.+A.thrice_pairs%3B+AstraZeneca+Pharmaceuticals%3A+Macclesfield%2C+U.K."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DCosgrove%26aufirst%3DD.%2BA.%26btitle%3Dthrice_pairs%26pub%3DAstraZeneca%2520Pharmaceuticals" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span> <span class="citation_source-book">Pipeline Pilot</span>, version <span class="NLM_edition">6.1.5</span>; <span class="NLM_publisher-name">Accelrys, Inc.</span>: <span class="NLM_publisher-loc">San Diego, CA</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Pipeline+Pilot%2C+version+6.1.5%3B+Accelrys%2C+Inc.%3A+San+Diego%2C+CA."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26btitle%3DPipeline%2520Pilot%26pub%3DAccelrys%252C%2520Inc" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span> <span class="citation_source-book">Daylight</span>, version <span class="NLM_edition">4.83</span>; <span class="NLM_publisher-name">Daylight Chemical Information Systems, Inc.</span>: <span class="NLM_publisher-loc">Laguna Niguel, CA</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Daylight%2C+version+4.83%3B+Daylight+Chemical+Information+Systems%2C+Inc.%3A+Laguna+Niguel%2C+CA."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26btitle%3DDaylight%26pub%3DDaylight%2520Chemical%2520Information%2520Systems%252C%2520Inc" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group">Bemis, G. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murcko, M. A.</span><span> </span><span class="NLM_article-title">The properties of known drugs. 1. Molecular frameworks</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1996</span><span class="NLM_x">, </span> <span class="NLM_volume">39</span><span class="NLM_x">, </span> <span class="NLM_fpage">2887</span><span class="NLM_x">–</span> <span class="NLM_lpage">2893</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm9602928" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Fjm200452d&amp;key=1%3ACAS%3A528%3ADyaK28XjvVejtro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=1996&pages=2887-2893&author=G.+W.+Bemisauthor=M.+A.+Murcko&title=The+properties+of+known+drugs.+1.+Molecular+frameworks"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">The Properties of Known Drugs. 1. Molecular Frameworks</span></div><div class="casAuthors">Bemis, Guy W.; Murcko, Mark A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">39</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">2887-2893</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">To better understand the common features present in drug mols., we use shape description methods to analyze a database of com. available drugs and prep. a list of common drug shapes.  A useful way of organizing this structural data is to group the atoms of each drug mol. into ring, linker, framework, and side chain atoms.  On the basis of the two-dimensional mol. structures (without regard to atom type, hybridization, and bond order), there are 1179 different frameworks among the 5120 compds. analyzed.  However, the shapes of half of the drugs in the database are described by the 32 most frequently occurring frameworks.  This suggests that the diversity of shapes in the set of known drugs is extremely low.  In our second method of anal., in which atom type, hybridization, and bond order are considered, more diversity is seen; there are 2506 different frameworks among the 5120 compds. in the database, and the most frequently occurring 42 frameworks account for only one-fourth of the drugs.  We discuss the possible interpretations of these findings and the way they may be used to guide future drug discovery research.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoodEb3sfDSdrVg90H21EOLACvtfcHk0lgS0H9A2bHPTg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28XjvVejtro%253D&md5=5e2c4fdfea9434456a0cca83de4185b3</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1021%2Fjm9602928&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm9602928%26sid%3Dliteratum%253Aachs%26aulast%3DBemis%26aufirst%3DG.%2BW.%26aulast%3DMurcko%26aufirst%3DM.%2BA.%26atitle%3DThe%2520properties%2520of%2520known%2520drugs.%25201.%2520Molecular%2520frameworks%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1996%26volume%3D39%26spage%3D2887%26epage%3D2893" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group">Bemis, G. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murcko, M. A.</span><span> </span><span class="NLM_article-title">Properties of known drugs. 2. Side chains</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1999</span><span class="NLM_x">, </span> <span class="NLM_volume">42</span><span class="NLM_x">, </span> <span class="NLM_fpage">5095</span><span class="NLM_x">–</span> <span class="NLM_lpage">5099</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm9903996" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Fjm200452d&amp;key=1%3ACAS%3A528%3ADyaK1MXnsFymtLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=1999&pages=5095-5099&author=G.+W.+Bemisauthor=M.+A.+Murcko&title=Properties+of+known+drugs.+2.+Side+chains"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Properties of Known Drugs. 2. Side Chains</span></div><div class="casAuthors">Bemis, Guy W.; Murcko, Mark A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">25</span>),
    <span class="NLM_cas:pages">5095-5099</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We continue our study of the common features present in drug mols. by looking in detail at drug side chains.  Using shape description methods, we divide a database of com. available drugs into a list of common drug side chains.  On the basis of the atom pair shape descriptor (taking into account atom type, hybridization, and bond order), there are 1,246 different side chains among the 5090 compds. analyzed.  The av. no. of side chains per mol. is 4, and the av. no. of heavy atoms per side chain is 2.  If we ignore the carbonyl side chain, then there are approx. 15,000 occurrences of side chains.  Of these 15,000 approx. 11,000 are from the "top 20" group of side chains.  This suggests that the diversity that side chains provide to drug mols. is quite low.  We discuss ways that this work could be used to provide guidance for mol. design efforts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpWUwPfGwYU3rVg90H21EOLACvtfcHk0lgt8ESri_Jz_Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXnsFymtLw%253D&md5=430bd96d4cf3e1b51bcd0a7515c5ccd5</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1021%2Fjm9903996&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm9903996%26sid%3Dliteratum%253Aachs%26aulast%3DBemis%26aufirst%3DG.%2BW.%26aulast%3DMurcko%26aufirst%3DM.%2BA.%26atitle%3DProperties%2520of%2520known%2520drugs.%25202.%2520Side%2520chains%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1999%26volume%3D42%26spage%3D5095%26epage%3D5099" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group">Haubertin, D. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bruneau, P.</span><span> </span><span class="NLM_article-title">A database of historically-observed chemical replacements</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">1294</span><span class="NLM_x">–</span> <span class="NLM_lpage">1302</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci600395u" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Fjm200452d&amp;key=1%3ACAS%3A528%3ADC%252BD2sXlvFehurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2007&pages=1294-1302&author=D.+Y.+Haubertinauthor=P.+Bruneau&title=A+database+of+historically-observed+chemical+replacements"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">A database of historically-observed chemical replacements</span></div><div class="casAuthors">Haubertin, David Y.; Bruneau, Pierre</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1294-1302</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A systematic anal. of one-to-one chem. replacements occurring in a set of 50 000 druglike mols. was performed.  The frequency of occurrence, as well as the av. change in measured and calcd. properties, was computed for each obsd. substitution.  The exptl. properties considered were soly., protein binding, and logD.  The calcd. properties were logP, mol. wt., no. of hydrogen bond donors and acceptors, and polar surface area.  During this anal., in which 9000 different functional groups were considered, 0.7 million substitutions were identified and stored in a database.  As an application, we present a web interface from which users can identify historically obsd. replacements of any functional group on their query mol.  The server returns a list of side-chains, as well as the historically obsd. shift in exptl. properties.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqYd8AQP_E2A7Vg90H21EOLACvtfcHk0lgt8ESri_Jz_Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXlvFehurc%253D&md5=1e3e7ac2992cae3e48a316bdf0a7e0e2</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1021%2Fci600395u&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci600395u%26sid%3Dliteratum%253Aachs%26aulast%3DHaubertin%26aufirst%3DD.%2BY.%26aulast%3DBruneau%26aufirst%3DP.%26atitle%3DA%2520database%2520of%2520historically-observed%2520chemical%2520replacements%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2007%26volume%3D47%26spage%3D1294%26epage%3D1302" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group">Lewell, X. Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Judd, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Watson, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hann, M.</span><span> </span><span class="NLM_article-title">RECAP—retrosynthetic combinatorial analysis procedure: a powerful new technique for identifying privileged molecular fragments with useful applications in combinatorial chemistry</span> <span class="citation_source-journal">J. Chem. Inf. Comput. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">38</span><span class="NLM_x">, </span> <span class="NLM_fpage">511</span><span class="NLM_x">–</span> <span class="NLM_lpage">522</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci970429i" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Fjm200452d&amp;key=1%3ACAS%3A528%3ADyaK1cXis1yrtbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=1998&pages=511-522&author=X.+Q.+Lewellauthor=D.+Juddauthor=S.+Watsonauthor=M.+Hann&title=RECAP%E2%80%94retrosynthetic+combinatorial+analysis+procedure%3A+a+powerful+new+technique+for+identifying+privileged+molecular+fragments+with+useful+applications+in+combinatorial+chemistry"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">RECAP-Retrosynthetic Combinatorial Analysis Procedure: A Powerful New Technique for Identifying Privileged Molecular Fragments with Useful Applications in Combinatorial Chemistry</span></div><div class="casAuthors">Lewell, Xiao Qing; Judd, Duncan; Watson, Steve; Hann, Mike</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Computer Sciences</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">511-522</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">0095-2338</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The use of combinatorial chem. for the generation of new lead mols. is now a well established strategy in the drug discovery process.  Central to the use of combinatorial chem. is the design and availability of high quality building blocks which are likely to afford hits from the libraries that they generate.  Herein the authors describe "RECAP" (Retrosynthetic Combinatorial Anal. Procedure), a new computational technique designed to address this building block issue.  RECAP electronically fragments mols. based on chem. knowledge.  When applied to databases of biol. active mols., this allows the identification of building block fragments rich in biol. recognized elements and privileged motifs and structures.  This allows the design of building blocks and the synthesis of libraries rich in biol. motifs.  Application of RECAP to the Derwent World Drug Index (WDI) and the mol. fragments/building blocks that this generates are discussed.  The authors also describe a WDI fragment knowledge base which the authors have built which stores the drug motifs and mention its potential application in structure based drug design programs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGovNOpJj5LVU7Vg90H21EOLACvtfcHk0lgt8ESri_Jz_Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXis1yrtbg%253D&md5=7445f76f162006c687116e1e539da8be</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1021%2Fci970429i&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci970429i%26sid%3Dliteratum%253Aachs%26aulast%3DLewell%26aufirst%3DX.%2BQ.%26aulast%3DJudd%26aufirst%3DD.%26aulast%3DWatson%26aufirst%3DS.%26aulast%3DHann%26aufirst%3DM.%26atitle%3DRECAP%25E2%2580%2594retrosynthetic%2520combinatorial%2520analysis%2520procedure%253A%2520a%2520powerful%2520new%2520technique%2520for%2520identifying%2520privileged%2520molecular%2520fragments%2520with%2520useful%2520applications%2520in%2520combinatorial%2520chemistry%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Comput.%2520Sci.%26date%3D1998%26volume%3D38%26spage%3D511%26epage%3D522" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group">Hussain, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rea, C.</span><span> </span><span class="NLM_article-title">Computationally efficient algorithm to identify matched molecular pairs (MMPs) in large data sets</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x">, </span> <span class="NLM_fpage">339</span><span class="NLM_x">–</span> <span class="NLM_lpage">348</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci900450m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Fjm200452d&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtlWltr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2010&pages=339-348&author=J.+Hussainauthor=C.+Rea&title=Computationally+efficient+algorithm+to+identify+matched+molecular+pairs+%28MMPs%29+in+large+data+sets"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Computationally Efficient Algorithm to Identify Matched Molecular Pairs (MMPs) in Large Data Sets</span></div><div class="casAuthors">Hussain, Jameed; Rea, Ceara</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">339-348</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Modern drug discovery organizations generate large vols. of SAR data.  A promising methodol. that can be used to mine this chem. data to identify novel structure-activity relationships is the matched mol. pair (MMP) methodol.  However, before the full potential of the MMP methodol. can be utilized, a MMP identification method that is capable of identifying all MMPs in large chem. data sets on modest computational hardware is required.  In this paper we report an algorithm that is capable of systematically generating all MMPs in chem. data sets.  Addnl., the algorithm is computationally efficient enough to be applied on large data sets.  As an example the algorithm was used to identify the MMPs in the ∼300k NIH MLSMR set.  The algorithm identified ∼5.3 million matched mol. pairs in the set.  These pairs cover ∼2.6 million unique mol. transformations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoF0rN8jaKSjrVg90H21EOLACvtfcHk0lgCogpOeIf2Tw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtlWltr4%253D&md5=23964abebc23274f5c36c319bee52bab</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1021%2Fci900450m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci900450m%26sid%3Dliteratum%253Aachs%26aulast%3DHussain%26aufirst%3DJ.%26aulast%3DRea%26aufirst%3DC.%26atitle%3DComputationally%2520efficient%2520algorithm%2520to%2520identify%2520matched%2520molecular%2520pairs%2520%2528MMPs%2529%2520in%2520large%2520data%2520sets%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2010%26volume%3D50%26spage%3D339%26epage%3D348" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group">Bender, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mussa, H. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Glen, R. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reiling, S.</span><span> </span><span class="NLM_article-title">Molecular similarity searching using atom environments, information-based feature selection, and a naive Bayesian classifier</span> <span class="citation_source-journal">J. Chem. Inf. Comput. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">44</span><span class="NLM_x">, </span> <span class="NLM_fpage">170</span><span class="NLM_x">–</span> <span class="NLM_lpage">178</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci034207y" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Fjm200452d&amp;key=1%3ACAS%3A528%3ADC%252BD3sXhtVSis7vM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2004&pages=170-178&author=A.+Benderauthor=H.+Y.+Mussaauthor=R.+C.+Glenauthor=S.+Reiling&title=Molecular+similarity+searching+using+atom+environments%2C+information-based+feature+selection%2C+and+a+naive+Bayesian+classifier"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular Similarity Searching Using Atom Environments, Information-Based Feature Selection, and a Naive Bayesian Classifier</span></div><div class="casAuthors">Bender, Andreas; Mussa, Hamse Y.; Glen, Robert C.; Reiling, Stephan</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Computer Sciences</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">170-178</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">0095-2338</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A novel technique for similarity searching is introduced.  Mols. are represented by atom environments, which are fed into an information-gain-based feature selection.  A naive Bayesian classifier is then employed for compd. classification.  The new method is tested by its ability to retrieve five sets of active mols. seeded in the MDL Drug Data Report (MDDR).  In comparison expts., the algorithm outperforms all current retrieval methods assessed here using two- and three-dimensional descriptors and offers insight into the significance of structural components for binding.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrgoFUDO5FldLVg90H21EOLACvtfcHk0lgCogpOeIf2Tw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXhtVSis7vM&md5=fc422eb289c5bdee1eebe431ef7f336e</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1021%2Fci034207y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci034207y%26sid%3Dliteratum%253Aachs%26aulast%3DBender%26aufirst%3DA.%26aulast%3DMussa%26aufirst%3DH.%2BY.%26aulast%3DGlen%26aufirst%3DR.%2BC.%26aulast%3DReiling%26aufirst%3DS.%26atitle%3DMolecular%2520similarity%2520searching%2520using%2520atom%2520environments%252C%2520information-based%2520feature%2520selection%252C%2520and%2520a%2520naive%2520Bayesian%2520classifier%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Comput.%2520Sci.%26date%3D2004%26volume%3D44%26spage%3D170%26epage%3D178" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span> <span class="citation_source-book">MDDR</span>, version <span class="NLM_edition">1</span>; <span class="NLM_publisher-name">Accelrys, Inc.</span>: <span class="NLM_publisher-loc">San Diego, CA</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=MDDR%2C+version+1%3B+Accelrys%2C+Inc.%3A+San+Diego%2C+CA."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26btitle%3DMDDR%26pub%3DAccelrys%252C%2520Inc" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group">Sheridan, R. P.</span><span> </span><span class="NLM_article-title">The most common chemical replacements in drug-like compounds</span> <span class="citation_source-journal">J. Chem. Inf. Comput. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">42</span><span class="NLM_x">, </span> <span class="NLM_fpage">103</span><span class="NLM_x">–</span> <span class="NLM_lpage">108</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci0100806" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Fjm200452d&amp;key=1%3ACAS%3A528%3ADC%252BD38XlvFWg" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2002&pages=103-108&author=R.+P.+Sheridan&title=The+most+common+chemical+replacements+in+drug-like+compounds"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">The most common chemical replacements in drug-like compounds</span></div><div class="casAuthors">Sheridan, Robert P.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Computer Sciences</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">103-108</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">0095-2338</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We have written a method that exts. one-to-one replacements of chem. groups in pairs of drug-like mols. with the same biol. activity and counts the frequency of the replacements in a large collection of such mols.  There are two variations on the method that differ in their treatment of replacements in rings.  This method is one possible approach to systematically identify candidate bioisosteres.  Here we look at the MDDR database because it has a large diversity of drug-like compds. in a large no. of therapeutic areas.  The most frequent replacements in MDDR seem generally consistent with medicinal chem. intuition about what chem. groups are equiv. or with groups that are easily converted by synthetic or metabolic pathways.  This method can be applied to any set of mols. wherein the mols. can be paired by similar biol. activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoSazOMexul7LVg90H21EOLACvtfcHk0lgCogpOeIf2Tw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XlvFWg&md5=6ca09924b4535fdf3cc9adfb3e70ae3d</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1021%2Fci0100806&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci0100806%26sid%3Dliteratum%253Aachs%26aulast%3DSheridan%26aufirst%3DR.%2BP.%26atitle%3DThe%2520most%2520common%2520chemical%2520replacements%2520in%2520drug-like%2520compounds%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Comput.%2520Sci.%26date%3D2002%26volume%3D42%26spage%3D103%26epage%3D108" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group">Sheridan, R. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miller, M. D.</span><span> </span><span class="NLM_article-title">A method for visualizing recurrent topological substructures in sets of active molecules</span> <span class="citation_source-journal">J. Chem. Inf. Comput. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">38</span><span class="NLM_x">, </span> <span class="NLM_fpage">915</span><span class="NLM_x">–</span> <span class="NLM_lpage">924</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci980044f" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Fjm200452d&amp;key=1%3ACAS%3A528%3ADyaK1cXltlyhs7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=1998&pages=915-924&author=R.+P.+Sheridanauthor=M.+D.+Miller&title=A+method+for+visualizing+recurrent+topological+substructures+in+sets+of+active+molecules"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">A Method for Visualizing Recurrent Topological Substructures in Sets of Active Molecules</span></div><div class="casAuthors">Sheridan, Robert P.; Miller, Michael D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Computer Sciences</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">915-924</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">0095-2338</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We present a method for detecting meaningful common topol. substructures among sets of active compds.  A clique-based subgraph detection method is used to find the highest scoring common substructure (HSCS) for each pair of mols.  Only those HSCSs are kept that have a much larger score than would be expected by chance for a randomly selected pair of mols. of the same size.  Information on these significant HSCSs is visualized in two complementary ways.  Individual HSCSs can be displayed as pairs of mols. with the atoms in the HSCS highlighted.  The HSCSs are presented in order of decreasing statistical significance.  Alternatively, individual mols. can be displayed such that each atom is labeled with the no. of significant HSCSs in which it has appeared.  These are presented in order of decreasing max. no. per mol.  Browsing these data gives an impression of what parts of mols. are conserved among actives.  The mols. can be simplified by deleting parts that are not conserved.  We show three examples taken from the MDDR database.  One example highlights common substructures among diverse CCK antagonists.  Two examples point out substructures common between actives on different receptors: benzodiazepine agonists vs. CCK antagonists and antidepressants vs. antihistamines.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGogQqKtvLFZaLVg90H21EOLACvtfcHk0lgx20AuKIE7lg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXltlyhs7Y%253D&md5=b5267c4b1bfa4d52729ba3cc2be88cc7</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1021%2Fci980044f&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci980044f%26sid%3Dliteratum%253Aachs%26aulast%3DSheridan%26aufirst%3DR.%2BP.%26aulast%3DMiller%26aufirst%3DM.%2BD.%26atitle%3DA%2520method%2520for%2520visualizing%2520recurrent%2520topological%2520substructures%2520in%2520sets%2520of%2520active%2520molecules%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Comput.%2520Sci.%26date%3D1998%26volume%3D38%26spage%3D915%26epage%3D924" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group">Warner, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Griffen, E. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">St-Gallay, S. A.</span><span> </span><span class="NLM_article-title">WizePairZ: a novel algorithm to identify, encode, and exploit matched molecular pairs with unspecified cores in medicinal chemistry</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x">, </span> <span class="NLM_fpage">1350</span><span class="NLM_x">–</span> <span class="NLM_lpage">1357</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci100084s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Fjm200452d&amp;key=1%3ACAS%3A528%3ADC%252BC3cXpvVSgs7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2010&pages=1350-1357&author=D.+J.+Warnerauthor=E.+J.+Griffenauthor=S.+A.+St-Gallay&title=WizePairZ%3A+a+novel+algorithm+to+identify%2C+encode%2C+and+exploit+matched+molecular+pairs+with+unspecified+cores+in+medicinal+chemistry"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">WizePairZ: A Novel Algorithm to Identify, Encode, and Exploit Matched Molecular Pairs with Unspecified Cores in Medicinal Chemistry</span></div><div class="casAuthors">Warner, Daniel J.; Griffen, Edward J.; St-Gallay, Stephen A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1350-1357</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">An algorithm to automatically identify and ext. matched mol. pairs from a collection of compds. has been developed, allowing the learning assocd. with each mol. transformation to be readily exploited in drug discovery projects.  Here, we present the application to an example data set of 11 histone deacetylase inhibitors.  The matched pairs were identified, and corresponding differences in activity and lipophilicity were recorded.  These property differences were assocd. with the chem. transformations encoded in the SMIRKS reaction notation.  The transformations identified a subseries with the optimal balance of these two parameters.  Enumeration of a virtual library of compds. using the extd. transformations identified two addnl. compds. initially excluded from the anal. with an accurate estn. of their biol. activity.  We describe how the WizePairZ system can be used to archive and apply medicinal chem. knowledge from one drug discovery project to another as well as identify common bioisosteres.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpSWafs2qQ4UbVg90H21EOLACvtfcHk0lgx20AuKIE7lg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXpvVSgs7c%253D&md5=9d83eb14cdb56079429761e79beeb292</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1021%2Fci100084s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci100084s%26sid%3Dliteratum%253Aachs%26aulast%3DWarner%26aufirst%3DD.%2BJ.%26aulast%3DGriffen%26aufirst%3DE.%2BJ.%26aulast%3DSt-Gallay%26aufirst%3DS.%2BA.%26atitle%3DWizePairZ%253A%2520a%2520novel%2520algorithm%2520to%2520identify%252C%2520encode%252C%2520and%2520exploit%2520matched%2520molecular%2520pairs%2520with%2520unspecified%2520cores%2520in%2520medicinal%2520chemistry%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2010%26volume%3D50%26spage%3D1350%26epage%3D1357" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group">Southall, N. T.</span><span> </span><span class="NLM_article-title">Ajay kinase patent space visualization using chemical replacements</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">2103</span><span class="NLM_x">–</span> <span class="NLM_lpage">2109</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm051201m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2006&pages=2103-2109&author=N.+T.+Southall&title=Ajay+kinase+patent+space+visualization+using+chemical+replacements"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1021%2Fjm051201m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm051201m%26sid%3Dliteratum%253Aachs%26aulast%3DSouthall%26aufirst%3DN.%2BT.%26atitle%3DAjay%2520kinase%2520patent%2520space%2520visualization%2520using%2520chemical%2520replacements%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2006%26volume%3D49%26spage%3D2103%26epage%3D2109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group">Raymond, J. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Watson, I. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mahoui, A.</span><span> </span><span class="NLM_article-title">Rationalizing lead optimization by associating quantitative relevance with molecular structure modification</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">1952</span><span class="NLM_x">–</span> <span class="NLM_lpage">1962</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci9000426" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Fjm200452d&amp;key=1%3ACAS%3A528%3ADC%252BD1MXosFOjt70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2009&pages=1952-1962&author=J.+W.+Raymondauthor=I.+A.+Watsonauthor=A.+Mahoui&title=Rationalizing+lead+optimization+by+associating+quantitative+relevance+with+molecular+structure+modification"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Rationalizing lead optimization by associating quantitative relevance with molecular structure modification</span></div><div class="casAuthors">Raymond, John W.; Watson, Ian A.; Mahoui, Abdelaziz</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1952-1962</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Historically, one of the characteristic activities of the medicinal chemist has been the iterative improvement of lead compds. until a suitable therapeutic entity is achieved.  Often referred to as lead optimization, this process typically takes the form of minor structural modifications to an existing lead in an attempt to ameliorate deleterious attributes while simultaneously trying to maintain or improve desirable properties.  The cumulative effect of this exercise performed over the course of several decades of pharmaceutical research by thousands of trained researchers has resulted in large collections of pharmaceutically relevant chem. structures.  As far as the authors are aware, this work represents the first attempt to use that data to define a framework to quantifiably catalog and summate this information into a medicinal chem. expert system.  A method is proposed that first comprehensively mines a compendium of chem. structures compiling the structural modifications, abridges them to rectify artificially inflated support levels, and then performs an assocn. rule mining expt. to ascribe relative confidences to each transformation.  The result is a catalog of statistically relevant structural modifications that can potentially be used in a no. of pharmaceutical applications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGprLA8yqcSLSrVg90H21EOLACvtfcHk0lgx20AuKIE7lg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXosFOjt70%253D&md5=327eb03c9ced8e99d8c0964f2c5c226a</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1021%2Fci9000426&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci9000426%26sid%3Dliteratum%253Aachs%26aulast%3DRaymond%26aufirst%3DJ.%2BW.%26aulast%3DWatson%26aufirst%3DI.%2BA.%26aulast%3DMahoui%26aufirst%3DA.%26atitle%3DRationalizing%2520lead%2520optimization%2520by%2520associating%2520quantitative%2520relevance%2520with%2520molecular%2520structure%2520modification%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2009%26volume%3D49%26spage%3D1952%26epage%3D1962" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54"><span><span class="NLM_contrib-group">Raymond, J. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gardiner, E. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Willett, P.</span><span> </span><span class="NLM_article-title">Heuristics for similarity searching of chemical graphs using a maximum common edge subgraph algorithm</span> <span class="citation_source-journal">J. Chem. Inf. Comput. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">42</span><span class="NLM_x">, </span> <span class="NLM_fpage">305</span><span class="NLM_x">–</span> <span class="NLM_lpage">316</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci010381f" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Fjm200452d&amp;key=1%3ACAS%3A528%3ADC%252BD38XhtVKgt78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2002&pages=305-316&author=J.+W.+Raymondauthor=E.+J.+Gardinerauthor=P.+Willett&title=Heuristics+for+similarity+searching+of+chemical+graphs+using+a+maximum+common+edge+subgraph+algorithm"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Heuristics for similarity searching of chemical graphs using a maximum common edge subgraph algorithm</span></div><div class="casAuthors">Raymond, John W.; Gardiner, Eleanor J.; Willett, Peter</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Computer Sciences</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">305-316</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">0095-2338</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Recently a method (RASCAL) for detg. graph similarity using a max. common edge subgraph algorithm has been proposed which has proven to be very efficient when used to calc. the relative similarity of chem. structures represented as graphs.  This paper describes heuristics which simplify a RASCAL similarity calcn. by taking advantage of certain properties specific to chem. graph representations of mol. structure.  These heuristics are shown exptl. to increase the efficiency of the algorithm, esp. at more distant values of chem. graph similarity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqScTcQRk232bVg90H21EOLACvtfcHk0lgM8N2Po-O1jw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XhtVKgt78%253D&md5=b6af8c4dc19d9c46856b1f93f5d8ebbd</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1021%2Fci010381f&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci010381f%26sid%3Dliteratum%253Aachs%26aulast%3DRaymond%26aufirst%3DJ.%2BW.%26aulast%3DGardiner%26aufirst%3DE.%2BJ.%26aulast%3DWillett%26aufirst%3DP.%26atitle%3DHeuristics%2520for%2520similarity%2520searching%2520of%2520chemical%2520graphs%2520using%2520a%2520maximum%2520common%2520edge%2520subgraph%2520algorithm%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Comput.%2520Sci.%26date%3D2002%26volume%3D42%26spage%3D305%26epage%3D316" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref55"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref55'); return false;" data-citation="" class="refNumLink">55</a></strong><div class="NLM_citation" id="cit55"><span><span class="NLM_contrib-group">Raymond, J. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Willett, P.</span><span> </span><span class="NLM_article-title">Maximum common subgraph isomorphism algorithms for the matching of chemical structures</span> <span class="citation_source-journal">J. Comput.-Aided Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">16</span><span class="NLM_x">, </span> <span class="NLM_fpage">521</span><span class="NLM_x">–</span> <span class="NLM_lpage">533</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2002&pages=521-533&author=J.+W.+Raymondauthor=P.+Willett&title=Maximum+common+subgraph+isomorphism+algorithms+for+the+matching+of+chemical+structures"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DRaymond%26aufirst%3DJ.%2BW.%26aulast%3DWillett%26aufirst%3DP.%26atitle%3DMaximum%2520common%2520subgraph%2520isomorphism%2520algorithms%2520for%2520the%2520matching%2520of%2520chemical%2520structures%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2002%26volume%3D16%26spage%3D521%26epage%3D533" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref56"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref56'); return false;" data-citation="" class="refNumLink">56</a></strong><div class="NLM_citation" id="cit56"><span><span class="NLM_contrib-group">Hu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bajorath, J.</span><span> </span><span class="NLM_article-title">Chemical transformations that yield compounds with distinct activity profiles</span> <span class="citation_source-journal">ACS Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">523</span><span class="NLM_x">–</span> <span class="NLM_lpage">527</span></span><div class="citationLinks">[<a href="/doi/10.1021/ml2000609" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2011&pages=523-527&author=Y.+Huauthor=J.+Bajorath&title=Chemical+transformations+that+yield+compounds+with+distinct+activity+profiles"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1021%2Fml2000609&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fml2000609%26sid%3Dliteratum%253Aachs%26aulast%3DHu%26aufirst%3DY.%26aulast%3DBajorath%26aufirst%3DJ.%26atitle%3DChemical%2520transformations%2520that%2520yield%2520compounds%2520with%2520distinct%2520activity%2520profiles%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2011%26volume%3D2%26spage%3D523%26epage%3D527" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref57"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref57'); return false;" data-citation="" class="refNumLink">57</a></strong><div class="NLM_citation" id="cit57"><span><span class="NLM_contrib-group">Wassermann, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bajorath, J.</span><span> </span><span class="NLM_article-title">Identification of target family directed bioisosteric replacements</span> <span class="citation_source-journal">MedChemComm</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">601</span><span class="NLM_x">–</span> <span class="NLM_lpage">606</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2011&pages=601-606&author=A.+M.+Wassermannauthor=J.+Bajorath&title=Identification+of+target+family+directed+bioisosteric+replacements"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DWassermann%26aufirst%3DA.%2BM.%26aulast%3DBajorath%26aufirst%3DJ.%26atitle%3DIdentification%2520of%2520target%2520family%2520directed%2520bioisosteric%2520replacements%26jtitle%3DMedChemComm%26date%3D2011%26volume%3D2%26spage%3D601%26epage%3D606" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref58"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref58'); return false;" data-citation="" class="refNumLink">58</a></strong><div class="NLM_citation" id="cit58"><span><span class="NLM_contrib-group">Alhambra, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Becker, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blake, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chang, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Damewood, J. R.,  Jr.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Daniels, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dembofsky, B. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gurley, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hall, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Herzog, K. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Horchler, C. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ohnmacht, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schmiesing, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dudley, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ribadeneira, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knappenberger, K. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maciag, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stein, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chopra, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, X. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Christian, E. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arriza, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chapdelaine, M. J.</span><span> </span><span class="NLM_article-title">Development and SAR of functionally selective allosteric modulators of GABAA receptors</span> <span class="citation_source-journal">Bioorg. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">2927</span><span class="NLM_x">–</span> <span class="NLM_lpage">2938</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Fjm200452d&amp;key=10.1016%2Fj.bmc.2011.03.035" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=8&amp;doi=10.1021%2Fjm200452d&amp;key=21498079" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Fjm200452d&amp;key=1%3ACAS%3A528%3ADC%252BC3MXltFOrsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2011&pages=2927-2938&author=C.+Alhambraauthor=C.+Beckerauthor=T.+Blakeauthor=A.+Changauthor=J.+R.+Damewoodauthor=T.+Danielsauthor=B.+T.+Dembofskyauthor=D.+A.+Gurleyauthor=J.+E.+Hallauthor=K.+J.+Herzogauthor=C.+L.+Horchlerauthor=C.+J.+Ohnmachtauthor=R.+J.+Schmiesingauthor=A.+Dudleyauthor=M.+D.+Ribadeneiraauthor=K.+S.+Knappenbergerauthor=C.+Maciagauthor=M.+M.+Steinauthor=M.+Chopraauthor=X.+F.+Liuauthor=E.+P.+Christianauthor=J.+L.+Arrizaauthor=M.+J.+Chapdelaine&title=Development+and+SAR+of+functionally+selective+allosteric+modulators+of+GABAA+receptors"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">Development and SAR of functionally selective allosteric modulators of GABAA receptors</span></div><div class="casAuthors">Alhambra, Cristobal; Becker, Chris; Blake, Timothy; Chang, Amy; Damewood, James R., Jr.; Daniels, Thalia; Dembofsky, Bruce T.; Gurley, David A.; Hall, James E.; Herzog, Keith J.; Horchler, Carey L.; Ohnmacht, Cyrus J.; Schmiesing, Richard Jon; Dudley, Adam; Ribadeneira, Maria D.; Knappenberger, Katherine S.; Maciag, Carla; Stein, Mark M.; Chopra, Maninder; Liu, Xiaodong F.; Christian, Edward P.; Arriza, Jeffrey L.; Chapdelaine, Marc J.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2927-2938</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Pos. modulators at the benzodiazepine site of α2- and α3-contg. GABAA receptors are believed to be anxiolytic.  Through oocyte voltage clamp studies, two series of compds. have been discovered that are pos. modulators at α2-/α3-contg. GABAA receptors and that show no functional activity at α1-contg. GABAA receptors.  Studies to improve this functional selectivity and ultimately deliver clin. candidates are reported.  The functional SAR of cinnolines and quinolines that are pos. allosteric modulators of the α2- and α3-contg. GABAA receptors, while simultaneously neutral antagonists at α1-contg. GABAA receptors, is described.  Such functionally selective modulators of GABAA receptors are expected to be useful in the treatment of anxiety and other psychiatric illnesses.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrVYzfPtmlNCrVg90H21EOLACvtfcHk0lgM8N2Po-O1jw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXltFOrsbg%253D&md5=24eaa34a5d28e2c7e2cfce14f02267fc</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2011.03.035&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2011.03.035%26sid%3Dliteratum%253Aachs%26aulast%3DAlhambra%26aufirst%3DC.%26aulast%3DBecker%26aufirst%3DC.%26aulast%3DBlake%26aufirst%3DT.%26aulast%3DChang%26aufirst%3DA.%26aulast%3DDamewood%26aufirst%3DJ.%2BR.%26aulast%3DDaniels%26aufirst%3DT.%26aulast%3DDembofsky%26aufirst%3DB.%2BT.%26aulast%3DGurley%26aufirst%3DD.%2BA.%26aulast%3DHall%26aufirst%3DJ.%2BE.%26aulast%3DHerzog%26aufirst%3DK.%2BJ.%26aulast%3DHorchler%26aufirst%3DC.%2BL.%26aulast%3DOhnmacht%26aufirst%3DC.%2BJ.%26aulast%3DSchmiesing%26aufirst%3DR.%2BJ.%26aulast%3DDudley%26aufirst%3DA.%26aulast%3DRibadeneira%26aufirst%3DM.%2BD.%26aulast%3DKnappenberger%26aufirst%3DK.%2BS.%26aulast%3DMaciag%26aufirst%3DC.%26aulast%3DStein%26aufirst%3DM.%2BM.%26aulast%3DChopra%26aufirst%3DM.%26aulast%3DLiu%26aufirst%3DX.%2BF.%26aulast%3DChristian%26aufirst%3DE.%2BP.%26aulast%3DArriza%26aufirst%3DJ.%2BL.%26aulast%3DChapdelaine%26aufirst%3DM.%2BJ.%26atitle%3DDevelopment%2520and%2520SAR%2520of%2520functionally%2520selective%2520allosteric%2520modulators%2520of%2520GABAA%2520receptors%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2011%26volume%3D19%26spage%3D2927%26epage%3D2938" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref59"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref59'); return false;" data-citation="" class="refNumLink">59</a></strong><div class="NLM_citation" id="cit59"><span><span class="NLM_contrib-group">Heinonen, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lappalainen, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mielikainen, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rousu, J.</span><span> </span><span class="NLM_article-title">Computing atom mappings for biochemical reactions without subgraph isomorphism</span> <span class="citation_source-journal">J. Comput. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x">, </span> <span class="NLM_fpage">43</span><span class="NLM_x">–</span> <span class="NLM_lpage">58</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Fjm200452d&amp;key=10.1089%2Fcmb.2009.0216" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=8&amp;doi=10.1021%2Fjm200452d&amp;key=21210731" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Fjm200452d&amp;key=1%3ACAS%3A528%3ADC%252BC3MXosFemtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2011&pages=43-58&author=M.+Heinonenauthor=S.+Lappalainenauthor=T.+Mielikainenauthor=J.+Rousu&title=Computing+atom+mappings+for+biochemical+reactions+without+subgraph+isomorphism"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">Computing atom mappings for biochemical reactions without subgraph isomorphism</span></div><div class="casAuthors">Heinonen, Markus; Lappalainen, Sampsa; Mielikainen, Taneli; Rousu, Juho</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Biology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">43-58</span>CODEN:
                <span class="NLM_cas:coden">JCOBEM</span>;
        ISSN:<span class="NLM_cas:issn">1557-8666</span>.
    
            (<span class="NLM_cas:orgname">Mary Ann Liebert, Inc.</span>)
        </div><div class="casAbstract">The ability to trace the fate of individual atoms through the metabolic pathways is needed in many applications of systems biol. and drug discovery.  However, this information is not immediately available from the most common metabolome studies and needs to be sep. acquired.  Automatic discovery of correspondence of atoms in biochem. reactions is called the "atom mapping problem.".  We suggest an efficient approach for solving the atom mapping problem exactly-finding mappings of min. edge edit distance.  The algorithm is based on A* search equipped with sophisticated heuristics for pruning the search space.  This approach has clear advantages over the commonly used heuristic approach of iterative max. common subgraph (MCS) algorithm: we explicitly minimize an objective function, and we produce solns. that typically require less manual curation.  The two methods are similar in computational resource demands.  We compare the performance of the proposed algorithm against several alternatives on data obtained from the KEGG LIGAND and RPAIR databases: greedy search, bi-partite graph matching, and the MCS approach.  Our expts. show that alternative approaches often fail in finding mappings with min. edit distance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr6wVpmn_1T2bVg90H21EOLACvtfcHk0ljGQWwOsJ-Skw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXosFemtw%253D%253D&md5=da66598ba846e8da5e02c7ea39fb2c0a</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1089%2Fcmb.2009.0216&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1089%252Fcmb.2009.0216%26sid%3Dliteratum%253Aachs%26aulast%3DHeinonen%26aufirst%3DM.%26aulast%3DLappalainen%26aufirst%3DS.%26aulast%3DMielikainen%26aufirst%3DT.%26aulast%3DRousu%26aufirst%3DJ.%26atitle%3DComputing%2520atom%2520mappings%2520for%2520biochemical%2520reactions%2520without%2520subgraph%2520isomorphism%26jtitle%3DJ.%2520Comput.%2520Biol.%26date%3D2011%26volume%3D18%26spage%3D43%26epage%3D58" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref60"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref60'); return false;" data-citation="" class="refNumLink">60</a></strong><div class="NLM_citation" id="cit60"><span><span class="NLM_contrib-group">Tantillo, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Houk, K. N.</span><span> </span><span class="NLM_article-title">Theozymes and compuzymes: theoretical models for biological catalysis</span> <span class="citation_source-journal">Curr. Opin. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">743</span><span class="NLM_x">–</span> <span class="NLM_lpage">750</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Fjm200452d&amp;key=10.1016%2FS1367-5931%2898%2980112-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=8&amp;doi=10.1021%2Fjm200452d&amp;key=9914196" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Fjm200452d&amp;key=1%3ACAS%3A528%3ADyaK1cXotVWrtbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=1998&pages=743-750&author=D.+J.+Tantilloauthor=J.+Chenauthor=K.+N.+Houk&title=Theozymes+and+compuzymes%3A+theoretical+models+for+biological+catalysis"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">Theozymes and compuzymes: theoretical models for biological catalysis</span></div><div class="casAuthors">Tantillo, Dean J.; Chen, Jiangang; Houk, Kendall N.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Chemical Biology</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">743-750</span>CODEN:
                <span class="NLM_cas:coden">COCBF4</span>;
        ISSN:<span class="NLM_cas:issn">1367-5931</span>.
    
            (<span class="NLM_cas:orgname">Current Biology Publications</span>)
        </div><div class="casAbstract">A review with 32 refs.  A theozyme is a theor. enzyme constructed by computing the optimal geometry for transition-state stabilization by functional groups.  It is created in order to permit quant. assessment of catalytic function.  Theozymes have been used to elucidate the role of transition-state stabilization in the mechanisms underlying enzyme- and antibody-catalyzed hydroxyepoxide cyclizations, eliminations and decarboxylations, peptide and ester hydrolyzes, and pericyclic and radical reactions.  The enzymes studied include orotodine monophosphate decarboxylase, HIV protease and ribonucleotide reductase.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGouKJ8Lt0UwqrVg90H21EOLACvtfcHk0ljGQWwOsJ-Skw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXotVWrtbs%253D&md5=05cf725d2647bdb6d3630dec85311401</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1016%2FS1367-5931%2898%2980112-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1367-5931%252898%252980112-9%26sid%3Dliteratum%253Aachs%26aulast%3DTantillo%26aufirst%3DD.%2BJ.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DHouk%26aufirst%3DK.%2BN.%26atitle%3DTheozymes%2520and%2520compuzymes%253A%2520theoretical%2520models%2520for%2520biological%2520catalysis%26jtitle%3DCurr.%2520Opin.%2520Chem.%2520Biol.%26date%3D1998%26volume%3D2%26spage%3D743%26epage%3D750" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref61"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref61'); return false;" data-citation="" class="refNumLink">61</a></strong><div class="NLM_citation" id="cit61"><span><span class="NLM_contrib-group">Palmer, D. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Llinas, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morao, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Day, G. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Goodman, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Glen, R. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mitchell, J. B. O.</span><span> </span><span class="NLM_article-title">Predicting intrinsic aqueous solubility by a thermodynamic cycle</span> <span class="citation_source-journal">Mol. Pharmaceutics</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">266</span><span class="NLM_x">–</span> <span class="NLM_lpage">279</span></span><div class="citationLinks">[<a href="/doi/10.1021/mp7000878" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Fjm200452d&amp;key=1%3ACAS%3A528%3ADC%252BD1cXitleis74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2008&pages=266-279&author=D.+S.+Palmerauthor=A.+Llinasauthor=I.+Moraoauthor=G.+M.+Dayauthor=J.+M.+Goodmanauthor=R.+C.+Glenauthor=J.+B.+O.+Mitchell&title=Predicting+intrinsic+aqueous+solubility+by+a+thermodynamic+cycle"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">Predicting Intrinsic Aqueous Solubility by a Thermodynamic Cycle</span></div><div class="casAuthors">Palmer, David S.; Llinas, Antonio; Morao, Inaki; Day, Graeme M.; Goodman, Jonathan M.; Glen, Robert C.; Mitchell, John B. O.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Pharmaceutics</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">266-279</span>CODEN:
                <span class="NLM_cas:coden">MPOHBP</span>;
        ISSN:<span class="NLM_cas:issn">1543-8384</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The authors report methods to predict the intrinsic aq. soly. of cryst. org. mols. from two different thermodn. cycles.  Direct computation of soly., via ab initio calcn. of thermodn. quantities at an affordable level of theory, cannot deliver the required accuracy.  Therefore, the authors have turned to a mixt. of direct computation and informatics, using the calcd. thermodn. properties, along with a few other key descriptors, in regression models.  The prediction of log intrinsic soly. (referred to mol/L) by a three-variable linear regression equation gave r2 = 0.77 and RMSE = 0.71 for an external test set comprising drug mols.  The model includes a calcd. crystal lattice energy which provides a computational method to account for the interactions in the solid state.  Probably it is not necessary to know the polymorphic form prior to prediction.  Also, the method developed here may be applicable to other solid-state systems such as salts or cocrystals.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrWUy94arHdGrVg90H21EOLACvtfcHk0ljGQWwOsJ-Skw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXitleis74%253D&md5=71502207fc58378d9cb7e4cb6ad15fd0</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1021%2Fmp7000878&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fmp7000878%26sid%3Dliteratum%253Aachs%26aulast%3DPalmer%26aufirst%3DD.%2BS.%26aulast%3DLlinas%26aufirst%3DA.%26aulast%3DMorao%26aufirst%3DI.%26aulast%3DDay%26aufirst%3DG.%2BM.%26aulast%3DGoodman%26aufirst%3DJ.%2BM.%26aulast%3DGlen%26aufirst%3DR.%2BC.%26aulast%3DMitchell%26aufirst%3DJ.%2BB.%2BO.%26atitle%3DPredicting%2520intrinsic%2520aqueous%2520solubility%2520by%2520a%2520thermodynamic%2520cycle%26jtitle%3DMol.%2520Pharmaceutics%26date%3D2008%26volume%3D5%26spage%3D266%26epage%3D279" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref62"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref62'); return false;" data-citation="" class="refNumLink">62</a></strong><div class="NLM_citation" id="cit62"><span><span class="NLM_contrib-group">Griffen, E.</span><span> </span><span class="NLM_article-title">The rise of the intelligent machines in drug hunting?</span> <span class="citation_source-journal">Future Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">1</span><span class="NLM_x">, </span> <span class="NLM_fpage">405</span><span class="NLM_x">–</span> <span class="NLM_lpage">408</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=16&amp;doi=10.1021%2Fjm200452d&amp;key=10.4155%2Ffmc.09.43" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=8&amp;doi=10.1021%2Fjm200452d&amp;key=21426119" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=32&amp;doi=10.1021%2Fjm200452d&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtFSgur%252FF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2009&pages=405-408&author=E.+Griffen&title=The+rise+of+the+intelligent+machines+in+drug+hunting%3F"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit62R"><div class="casContent"><span class="casTitleNuber">62</span><div class="casTitle"><span class="NLM_cas:atitle">The rise of the intelligent machines in drug hunting?</span></div><div class="casAuthors">Griffen, Ed</div><div class="citationInfo"><span class="NLM_cas:title">Future Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">405-408</span>CODEN:
                <span class="NLM_cas:coden">FMCUA7</span>;
        ISSN:<span class="NLM_cas:issn">1756-8919</span>.
    
            (<span class="NLM_cas:orgname">Future Science Ltd.</span>)
        </div><div class="casAbstract">The impact of automated heuristic generation to drug discovery programs is discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqGdmTklMJKm7Vg90H21EOLACvtfcHk0lglVFIS8WV6bQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtFSgur%252FF&md5=89e9aadb119a28e621e7f50310c433a4</span></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.4155%2Ffmc.09.43&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4155%252Ffmc.09.43%26sid%3Dliteratum%253Aachs%26aulast%3DGriffen%26aufirst%3DE.%26atitle%3DThe%2520rise%2520of%2520the%2520intelligent%2520machines%2520in%2520drug%2520hunting%253F%26jtitle%3DFuture%2520Med.%2520Chem.%26date%3D2009%26volume%3D1%26spage%3D405%26epage%3D408" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37a','cit37b'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54'],'ref55':['cit55'],'ref56':['cit56'],'ref57':['cit57'],'ref58':['cit58'],'ref59':['cit59'],'ref60':['cit60'],'ref61':['cit61'],'ref62':['cit62']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 179 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Richard J. Glyn, <span class="NLM_string-name hlFld-ContribAuthor">Graham Pattison</span>. </span><span class="cited-content_cbyCitation_article-title">Effects of Replacing Oxygenated Functionality with Fluorine on Lipophilicity. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (14)
                                     , 10246-10259. <a href="https://doi.org/10.1021/acs.jmedchem.1c00668" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.1c00668</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.1c00668&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.1c00668%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DEffects%252Bof%252BReplacing%252BOxygenated%252BFunctionality%252Bwith%252BFluorine%252Bon%252BLipophilicity%26aulast%3DGlyn%26aufirst%3DRichard%2BJ.%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D12042021%26date%3D02072021%26volume%3D64%26issue%3D14%26spage%3D10246%26epage%3D10259" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">José Jiménez-Luna, Miha Skalic, Nils Weskamp, <span class="NLM_string-name hlFld-ContribAuthor">Gisbert Schneider</span>. </span><span class="cited-content_cbyCitation_article-title">Coloring Molecules with Explainable Artificial Intelligence for Preclinical Relevance Assessment. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2021,</strong> <em>61 </em>
                                    (3)
                                     , 1083-1094. <a href="https://doi.org/10.1021/acs.jcim.0c01344" title="DOI URL">https://doi.org/10.1021/acs.jcim.0c01344</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.0c01344&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.0c01344%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DColoring%252BMolecules%252Bwith%252BExplainable%252BArtificial%252BIntelligence%252Bfor%252BPreclinical%252BRelevance%252BAssessment%26aulast%3DJim%25C3%25A9nez-Luna%26aufirst%3DJos%25C3%25A9%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D19112020%26date%3D25022021%26volume%3D61%26issue%3D3%26spage%3D1083%26epage%3D1094" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Mahendra Awale, Jérôme Hert, Laura Guasch, Sereina Riniker, <span class="NLM_string-name hlFld-ContribAuthor">Christian Kramer</span>. </span><span class="cited-content_cbyCitation_article-title">The Playbooks of Medicinal Chemistry Design Moves. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2021,</strong> <em>61 </em>
                                    (2)
                                     , 729-742. <a href="https://doi.org/10.1021/acs.jcim.0c01143" title="DOI URL">https://doi.org/10.1021/acs.jcim.0c01143</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.0c01143&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.0c01143%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DThe%252BPlaybooks%252Bof%252BMedicinal%252BChemistry%252BDesign%252BMoves%26aulast%3DAwale%26aufirst%3DMahendra%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D01102020%26date%3D01022021%26volume%3D61%26issue%3D2%26spage%3D729%26epage%3D742" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">W. Patrick Walters, <span class="NLM_string-name hlFld-ContribAuthor">Regina Barzilay</span>. </span><span class="cited-content_cbyCitation_article-title">Applications of Deep Learning in Molecule Generation and Molecular Property Prediction. </span><span class="cited-content_cbyCitation_journal-name">Accounts of Chemical Research</span><span> <strong>2021,</strong> <em>54 </em>
                                    (2)
                                     , 263-270. <a href="https://doi.org/10.1021/acs.accounts.0c00699" title="DOI URL">https://doi.org/10.1021/acs.accounts.0c00699</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.accounts.0c00699&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.accounts.0c00699%26sid%3Dliteratum%253Aachs%26jtitle%3DAccounts%2520of%2520Chemical%2520Research%26atitle%3DApplications%252Bof%252BDeep%252BLearning%252Bin%252BMolecule%252BGeneration%252Band%252BMolecular%252BProperty%252BPrediction%26aulast%3DWalters%26aufirst%3DW.%2BPatrick%26date%3D2021%26date%3D2020%26date%3D2020%26date%3D28102020%26date%3D28122020%26volume%3D54%26issue%3D2%26spage%3D263%26epage%3D270" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Hans Matter, Christian Buning, Dan Dragos Stefanescu, Sven Ruf, <span class="NLM_string-name hlFld-ContribAuthor">Gerhard Hessler</span>. </span><span class="cited-content_cbyCitation_article-title">Using Graph Databases to Investigate Trends in Structure–Activity Relationship Networks. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2020,</strong> <em>60 </em>
                                    (12)
                                     , 6120-6134. <a href="https://doi.org/10.1021/acs.jcim.0c00947" title="DOI URL">https://doi.org/10.1021/acs.jcim.0c00947</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.0c00947&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.0c00947%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DUsing%252BGraph%252BDatabases%252Bto%252BInvestigate%252BTrends%252Bin%252BStructure%2525E2%252580%252593Activity%252BRelationship%252BNetworks%26aulast%3DMatter%26aufirst%3DHans%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D14082020%26date%3D27112020%26volume%3D60%26issue%3D12%26spage%3D6120%26epage%3D6134" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Kosuke Takeuchi, Ryo Kunimoto, <span class="NLM_string-name hlFld-ContribAuthor">Jürgen Bajorath</span>. </span><span class="cited-content_cbyCitation_article-title">Global Assessment of Substituents on the Basis of Analogue Series. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (23)
                                     , 15013-15020. <a href="https://doi.org/10.1021/acs.jmedchem.0c01607" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c01607</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c01607&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c01607%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DGlobal%252BAssessment%252Bof%252BSubstituents%252Bon%252Bthe%252BBasis%252Bof%252BAnalogue%252BSeries%26aulast%3DTakeuchi%26aufirst%3DKosuke%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D14092020%26date%3D30112020%26volume%3D63%26issue%3D23%26spage%3D15013%26epage%3D15020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">James A. Lumley, Prashant Desai, Jibo Wang, Suntara Cahya, <span class="NLM_string-name hlFld-ContribAuthor">Hongzhou Zhang</span>. </span><span class="cited-content_cbyCitation_article-title">The Derivation of a Matched Molecular Pairs Based ADME/Tox Knowledge Base for Compound Optimization. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2020,</strong> <em>60 </em>
                                    (10)
                                     , 4757-4771. <a href="https://doi.org/10.1021/acs.jcim.0c00583" title="DOI URL">https://doi.org/10.1021/acs.jcim.0c00583</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.0c00583&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.0c00583%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DThe%252BDerivation%252Bof%252Ba%252BMatched%252BMolecular%252BPairs%252BBased%252BADME%25252FTox%252BKnowledge%252BBase%252Bfor%252BCompound%252BOptimization%26aulast%3DLumley%26aufirst%3DJames%2BA.%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D26052020%26date%3D06102020%26date%3D25092020%26volume%3D60%26issue%3D10%26spage%3D4757%26epage%3D4771" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="NLM_string-name hlFld-ContribAuthor">Matthew J. Fuchter</span>. </span><span class="cited-content_cbyCitation_article-title">On the Promise of Photopharmacology Using Photoswitches: A Medicinal Chemist’s Perspective. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (20)
                                     , 11436-11447. <a href="https://doi.org/10.1021/acs.jmedchem.0c00629" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c00629</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c00629&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c00629%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DOn%252Bthe%252BPromise%252Bof%252BPhotopharmacology%252BUsing%252BPhotoswitches%25253A%252BA%252BMedicinal%252BChemist%2525E2%252580%252599s%252BPerspective%26aulast%3DFuchter%26aufirst%3DMatthew%2BJ.%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D17042020%26date%3D24062020%26date%3D08062020%26volume%3D63%26issue%3D20%26spage%3D11436%26epage%3D11447" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Kangway V. Chuang, Laura M. Gunsalus, <span class="NLM_string-name hlFld-ContribAuthor">Michael J. Keiser</span>. </span><span class="cited-content_cbyCitation_article-title">Learning Molecular Representations for Medicinal Chemistry. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (16)
                                     , 8705-8722. <a href="https://doi.org/10.1021/acs.jmedchem.0c00385" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c00385</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c00385&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c00385%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DLearning%252BMolecular%252BRepresentations%252Bfor%252BMedicinal%252BChemistry%26aulast%3DChuang%26aufirst%3DKangway%2BV.%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D05032020%26date%3D15052020%26date%3D04052020%26volume%3D63%26issue%3D16%26spage%3D8705%26epage%3D8722" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="NLM_string-name hlFld-ContribAuthor">Peter Ertl</span>. </span><span class="cited-content_cbyCitation_article-title">Identification of Bioisosteric Substituents by a Deep Neural Network. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2020,</strong> <em>60 </em>
                                    (7)
                                     , 3369-3375. <a href="https://doi.org/10.1021/acs.jcim.0c00290" title="DOI URL">https://doi.org/10.1021/acs.jcim.0c00290</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.0c00290&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.0c00290%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DIdentification%252Bof%252BBioisosteric%252BSubstituents%252Bby%252Ba%252BDeep%252BNeural%252BNetwork%26aulast%3DErtl%26aufirst%3DPeter%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D23032020%26date%3D30062020%26date%3D15062020%26volume%3D60%26issue%3D7%26spage%3D3369%26epage%3D3375" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Mahendra Awale, Sereina Riniker, <span class="NLM_string-name hlFld-ContribAuthor">Christian Kramer</span>. </span><span class="cited-content_cbyCitation_article-title">Matched Molecular Series Analysis for ADME Property Prediction. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2020,</strong> <em>60 </em>
                                    (6)
                                     , 2903-2914. <a href="https://doi.org/10.1021/acs.jcim.0c00269" title="DOI URL">https://doi.org/10.1021/acs.jcim.0c00269</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.0c00269&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.0c00269%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DMatched%252BMolecular%252BSeries%252BAnalysis%252Bfor%252BADME%252BProperty%252BPrediction%26aulast%3DAwale%26aufirst%3DMahendra%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D16032020%26date%3D05052020%26volume%3D60%26issue%3D6%26spage%3D2903%26epage%3D2914" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Huabin Hu, <span class="NLM_string-name hlFld-ContribAuthor">Jürgen Bajorath</span>. </span><span class="cited-content_cbyCitation_article-title">Systematic Exploration of Activity Cliffs Containing Privileged Substructures. </span><span class="cited-content_cbyCitation_journal-name">Molecular Pharmaceutics</span><span> <strong>2020,</strong> <em>17 </em>
                                    (3)
                                     , 979-989. <a href="https://doi.org/10.1021/acs.molpharmaceut.9b01236" title="DOI URL">https://doi.org/10.1021/acs.molpharmaceut.9b01236</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.molpharmaceut.9b01236&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.molpharmaceut.9b01236%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecular%2520Pharmaceutics%26atitle%3DSystematic%252BExploration%252Bof%252BActivity%252BCliffs%252BContaining%252BPrivileged%252BSubstructures%26aulast%3DHu%26aufirst%3DHuabin%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D2019%26date%3D2020%26date%3D06122019%26date%3D24012020%26date%3D22012020%26date%3D03022020%26date%3D24012020%26volume%3D17%26issue%3D3%26spage%3D979%26epage%3D989" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Li Fu, Lu Liu, Zhi-Jiang Yang, Pan Li, Jun-Jie Ding, Yong-Huan Yun, Ai-Ping Lu, Ting-Jun Hou, <span class="NLM_string-name hlFld-ContribAuthor">Dong-Sheng Cao</span>. </span><span class="cited-content_cbyCitation_article-title">Systematic Modeling of log D7.4 Based on Ensemble Machine Learning, Group Contribution, and Matched Molecular Pair Analysis. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2020,</strong> <em>60 </em>
                                    (1)
                                     , 63-76. <a href="https://doi.org/10.1021/acs.jcim.9b00718" title="DOI URL">https://doi.org/10.1021/acs.jcim.9b00718</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.9b00718&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.9b00718%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DSystematic%252BModeling%252Bof%252Blog%2525E2%252580%2525AFD7.4%252BBased%252Bon%252BEnsemble%252BMachine%252BLearning%25252C%252BGroup%252BContribution%25252C%252Band%252BMatched%252BMolecular%252BPair%252BAnalysis%26aulast%3DFu%26aufirst%3DLi%26date%3D2020%26date%3D2019%26date%3D2019%26date%3D29082019%26date%3D10012020%26date%3D23122019%26volume%3D60%26issue%3D1%26spage%3D63%26epage%3D76" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Matthew L. Landry, <span class="NLM_string-name hlFld-ContribAuthor">James J. Crawford</span>. </span><span class="cited-content_cbyCitation_article-title">LogD Contributions of Substituents Commonly Used in Medicinal Chemistry. </span><span class="cited-content_cbyCitation_journal-name">ACS Medicinal Chemistry Letters</span><span> <strong>2020,</strong> <em>11 </em>
                                    (1)
                                     , 72-76. <a href="https://doi.org/10.1021/acsmedchemlett.9b00489" title="DOI URL">https://doi.org/10.1021/acsmedchemlett.9b00489</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsmedchemlett.9b00489&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsmedchemlett.9b00489%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DLogD%252BContributions%252Bof%252BSubstituents%252BCommonly%252BUsed%252Bin%252BMedicinal%252BChemistry%26aulast%3DLandry%26aufirst%3DMatthew%2BL.%26date%3D2020%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D25102019%26date%3D11122019%26date%3D23122019%26date%3D11122019%26volume%3D11%26issue%3D1%26spage%3D72%26epage%3D76" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Kavoos Kolahdouzan, Ryan Khalaf, Jessica M. Grandner, Yongsheng Chen, Jack A. Terrett, <span class="NLM_string-name hlFld-ContribAuthor">Malcolm P. Huestis</span>. </span><span class="cited-content_cbyCitation_article-title">Dual Photoredox/Nickel-Catalyzed Conversion of Aryl Halides to Aryl Aminooxetanes: Computational Evidence for a Substrate-Dependent Switch in Mechanism. </span><span class="cited-content_cbyCitation_journal-name">ACS Catalysis</span><span> <strong>2020,</strong> <em>10 </em>
                                    (1)
                                     , 405-411. <a href="https://doi.org/10.1021/acscatal.9b03596" title="DOI URL">https://doi.org/10.1021/acscatal.9b03596</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acscatal.9b03596&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facscatal.9b03596%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Catalysis%26atitle%3DDual%252BPhotoredox%25252FNickel-Catalyzed%252BConversion%252Bof%252BAryl%252BHalides%252Bto%252BAryl%252BAminooxetanes%25253A%252BComputational%252BEvidence%252Bfor%252Ba%252BSubstrate-Dependent%252BSwitch%252Bin%252BMechanism%26aulast%3DKolahdouzan%26aufirst%3DKavoos%26date%3D2020%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D22082019%26date%3D17102019%26date%3D11122019%26date%3D08112019%26volume%3D10%26issue%3D1%26spage%3D405%26epage%3D411" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Atsushi Yoshimori, Yuichi Horita, Toru Tanoue, <span class="NLM_string-name hlFld-ContribAuthor">Jürgen Bajorath</span>. </span><span class="cited-content_cbyCitation_article-title">Method for Systematic Analogue Search Using the Mega SAR Matrix Database. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2019,</strong> <em>59 </em>
                                    (9)
                                     , 3727-3734. <a href="https://doi.org/10.1021/acs.jcim.9b00557" title="DOI URL">https://doi.org/10.1021/acs.jcim.9b00557</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.9b00557&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.9b00557%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DMethod%252Bfor%252BSystematic%252BAnalogue%252BSearch%252BUsing%252Bthe%252BMega%252BSAR%252BMatrix%252BDatabase%26aulast%3DYoshimori%26aufirst%3DAtsushi%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D08072019%26date%3D30082019%26volume%3D59%26issue%3D9%26spage%3D3727%26epage%3D3734" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Linsen Li, Antony A. Okumu, Sheri Nolan, Anthony English, Sandip Vibhute, Yanran Lu, Katherine Hervert-Thomas, Justin T. Seffernick, Lovette Azap, Serena L. Cole, D. Shinabarger, Laura M. Koeth, Steffen Lindert, Jack C. Yalowich, Daniel J. Wozniak, <span class="NLM_string-name hlFld-ContribAuthor">Mark J. Mitton-Fry</span>. </span><span class="cited-content_cbyCitation_article-title">1,3-Dioxane-Linked Bacterial Topoisomerase Inhibitors with Enhanced Antibacterial Activity and Reduced hERG Inhibition. </span><span class="cited-content_cbyCitation_journal-name">ACS Infectious Diseases</span><span> <strong>2019,</strong> <em>5 </em>
                                    (7)
                                     , 1115-1128. <a href="https://doi.org/10.1021/acsinfecdis.8b00375" title="DOI URL">https://doi.org/10.1021/acsinfecdis.8b00375</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsinfecdis.8b00375&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsinfecdis.8b00375%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Infectious%2520Diseases%26atitle%3D1%25252C3-Dioxane-Linked%252BBacterial%252BTopoisomerase%252BInhibitors%252Bwith%252BEnhanced%252BAntibacterial%252BActivity%252Band%252BReduced%252BhERG%252BInhibition%26aulast%3DLi%26aufirst%3DLinsen%26date%3D2019%26date%3D2019%26date%3D2018%26date%3D31122018%26date%3D09052019%26date%3D01052019%26volume%3D5%26issue%3D7%26spage%3D1115%26epage%3D1128" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Shunsuke Tamura, Tomoyuki Miyao, <span class="NLM_string-name hlFld-ContribAuthor">Kimito Funatsu</span>. </span><span class="cited-content_cbyCitation_article-title">Development of R-Group Fingerprints Based on the Local Landscape from an Attachment Point of a Molecular Structure. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2019,</strong> <em>59 </em>
                                    (6)
                                     , 2656-2663. <a href="https://doi.org/10.1021/acs.jcim.9b00122" title="DOI URL">https://doi.org/10.1021/acs.jcim.9b00122</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.9b00122&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.9b00122%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DDevelopment%252Bof%252BR-Group%252BFingerprints%252BBased%252Bon%252Bthe%252BLocal%252BLandscape%252Bfrom%252Ban%252BAttachment%252BPoint%252Bof%252Ba%252BMolecular%252BStructure%26aulast%3DTamura%26aufirst%3DShunsuke%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D10022019%26date%3D06052019%26volume%3D59%26issue%3D6%26spage%3D2656%26epage%3D2663" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">J. Jesús Naveja, Martin Vogt, Dagmar Stumpfe, José L. Medina-Franco, <span class="NLM_string-name hlFld-ContribAuthor">Jürgen Bajorath</span>. </span><span class="cited-content_cbyCitation_article-title">Systematic Extraction of Analogue Series from Large Compound Collections Using a New Computational Compound–Core Relationship Method. </span><span class="cited-content_cbyCitation_journal-name">ACS Omega</span><span> <strong>2019,</strong> <em>4 </em>
                                    (1)
                                     , 1027-1032. <a href="https://doi.org/10.1021/acsomega.8b03390" title="DOI URL">https://doi.org/10.1021/acsomega.8b03390</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsomega.8b03390&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsomega.8b03390%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Omega%26atitle%3DSystematic%252BExtraction%252Bof%252BAnalogue%252BSeries%252Bfrom%252BLarge%252BCompound%252BCollections%252BUsing%252Ba%252BNew%252BComputational%252BCompound%2525E2%252580%252593Core%252BRelationship%252BMethod%26aulast%3DNaveja%26aufirst%3DJ.%2BJes%25C3%25BAs%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D2018%26date%3D03122018%26date%3D03012019%26volume%3D4%26issue%3D1%26spage%3D1027%26epage%3D1032" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Jennifer E. Davoren, Deane Nason, Jotham Coe, Keith Dlugolenski, Christopher Helal, Anthony R. Harris, Erik LaChapelle, Sidney Liang, Yue Liu, Rebecca O’Connor, Christine C. Orozco, Brajesh K. Rai, Michelle Salafia, Brian Samas, Wenjian Xu, Rouba Kozak, <span class="NLM_string-name hlFld-ContribAuthor">David Gray</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery and Lead Optimization of Atropisomer D1 Agonists with Reduced Desensitization. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2018,</strong> <em>61 </em>
                                    (24)
                                     , 11384-11397. <a href="https://doi.org/10.1021/acs.jmedchem.8b01622" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.8b01622</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.8b01622&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.8b01622%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Band%252BLead%252BOptimization%252Bof%252BAtropisomer%252BD1%252BAgonists%252Bwith%252BReduced%252BDesensitization%26aulast%3DDavoren%26aufirst%3DJennifer%2BE.%26date%3D2018%26date%3D2018%26date%3D2018%26date%3D18102018%26date%3D11122018%26date%3D15112018%26volume%3D61%26issue%3D24%26spage%3D11384%26epage%3D11397" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Sébastien L. Degorce, Michael S. Bodnarchuk, Iain A. Cumming, <span class="NLM_string-name hlFld-ContribAuthor">James S. Scott</span>. </span><span class="cited-content_cbyCitation_article-title">Lowering Lipophilicity by Adding Carbon: One-Carbon Bridges of Morpholines and Piperazines. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2018,</strong> <em>61 </em>
                                    (19)
                                     , 8934-8943. <a href="https://doi.org/10.1021/acs.jmedchem.8b01148" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.8b01148</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.8b01148&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.8b01148%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DLowering%252BLipophilicity%252Bby%252BAdding%252BCarbon%25253A%252BOne-Carbon%252BBridges%252Bof%252BMorpholines%252Band%252BPiperazines%26aulast%3DDegorce%26aufirst%3DS%25C3%25A9bastien%2BL.%26date%3D2018%26date%3D2018%26date%3D2018%26date%3D23072018%26date%3D20092018%26date%3D06092018%26volume%3D61%26issue%3D19%26spage%3D8934%26epage%3D8943" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Vinicius
M. Alves, Alexander Golbraikh, Stephen J. Capuzzi, Kammy Liu, Wai In Lam, Daniel Robert Korn, Diane Pozefsky, Carolina Horta Andrade, Eugene N. Muratov, <span class="NLM_string-name hlFld-ContribAuthor">Alexander Tropsha</span>. </span><span class="cited-content_cbyCitation_article-title">Multi-Descriptor Read Across (MuDRA): A Simple and Transparent Approach for Developing Accurate Quantitative Structure–Activity Relationship Models. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2018,</strong> <em>58 </em>
                                    (6)
                                     , 1214-1223. <a href="https://doi.org/10.1021/acs.jcim.8b00124" title="DOI URL">https://doi.org/10.1021/acs.jcim.8b00124</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.8b00124&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.8b00124%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DMulti-Descriptor%252BRead%252BAcross%252B%252528MuDRA%252529%25253A%252BA%252BSimple%252Band%252BTransparent%252BApproach%252Bfor%252BDeveloping%252BAccurate%252BQuantitative%252BStructure%2525E2%252580%252593Activity%252BRelationship%252BModels%26aulast%3DAlves%26aufirst%3DVinicius%2BM.%26date%3D2018%26date%3D2018%26date%3D2018%26date%3D05032018%26date%3D13062018%26date%3D29052018%26volume%3D58%26issue%3D6%26spage%3D1214%26epage%3D1223" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Fabio Broccatelli, Ignacio Aliagas, <span class="NLM_string-name hlFld-ContribAuthor">Hao Zheng</span>. </span><span class="cited-content_cbyCitation_article-title">Why Decreasing Lipophilicity Alone Is Often Not a Reliable Strategy for Extending IV Half-life. </span><span class="cited-content_cbyCitation_journal-name">ACS Medicinal Chemistry Letters</span><span> <strong>2018,</strong> <em>9 </em>
                                    (6)
                                     , 522-527. <a href="https://doi.org/10.1021/acsmedchemlett.8b00047" title="DOI URL">https://doi.org/10.1021/acsmedchemlett.8b00047</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsmedchemlett.8b00047&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsmedchemlett.8b00047%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DWhy%252BDecreasing%252BLipophilicity%252BAlone%252BIs%252BOften%252BNot%252Ba%252BReliable%252BStrategy%252Bfor%252BExtending%252BIV%252BHalf-life%26aulast%3DBroccatelli%26aufirst%3DFabio%26date%3D2018%26date%3D2018%26date%3D2018%26date%3D2018%26date%3D02022018%26date%3D19042018%26date%3D20042018%26date%3D19042018%26volume%3D9%26issue%3D6%26spage%3D522%26epage%3D527" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Andrew Dalke, Jérôme Hert, <span class="NLM_string-name hlFld-ContribAuthor">Christian Kramer</span>. </span><span class="cited-content_cbyCitation_article-title">mmpdb: An Open-Source Matched Molecular Pair Platform for Large Multiproperty Data Sets. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2018,</strong> <em>58 </em>
                                    (5)
                                     , 902-910. <a href="https://doi.org/10.1021/acs.jcim.8b00173" title="DOI URL">https://doi.org/10.1021/acs.jcim.8b00173</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.8b00173&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.8b00173%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3Dmmpdb%25253A%252BAn%252BOpen-Source%252BMatched%252BMolecular%252BPair%252BPlatform%252Bfor%252BLarge%252BMultiproperty%252BData%252BSets%26aulast%3DDalke%26aufirst%3DAndrew%26date%3D2018%26date%3D2018%26date%3D2018%26date%3D22032018%26date%3D17052018%26volume%3D58%26issue%3D5%26spage%3D902%26epage%3D910" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Christian Kramer, Attilla Ting, Hao Zheng, Jérôme Hert, Torsten Schindler, Martin Stahl, Graeme Robb, James J. Crawford, Jeff Blaney, Shane Montague, Andrew G. Leach, Al G. Dossetter, <span class="NLM_string-name hlFld-ContribAuthor">Ed J. Griffen</span>. </span><span class="cited-content_cbyCitation_article-title">Learning Medicinal Chemistry Absorption, Distribution, Metabolism, Excretion, and Toxicity (ADMET) Rules from Cross-Company Matched Molecular Pairs Analysis (MMPA). </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2018,</strong> <em>61 </em>
                                    (8)
                                     , 3277-3292. <a href="https://doi.org/10.1021/acs.jmedchem.7b00935" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.7b00935</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.7b00935&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.7b00935%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DLearning%252BMedicinal%252BChemistry%252BAbsorption%25252C%252BDistribution%25252C%252BMetabolism%25252C%252BExcretion%25252C%252Band%252BToxicity%252B%252528ADMET%252529%252BRules%252Bfrom%252BCross-Company%252BMatched%252BMolecular%252BPairs%252BAnalysis%252B%252528MMPA%252529%26aulast%3DKramer%26aufirst%3DChristian%26date%3D2018%26date%3D2017%26date%3D2017%26date%3D26062017%26date%3D28092017%26volume%3D61%26issue%3D8%26spage%3D3277%26epage%3D3292" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Jean-Philippe Métivier, Bertrand Cuissart, Ronan Bureau, <span class="NLM_string-name hlFld-ContribAuthor">Alban Lepailleur</span>. </span><span class="cited-content_cbyCitation_article-title">The Pharmacophore Network: A Computational Method for Exploring Structure–Activity Relationships from a Large Chemical Data Set. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2018,</strong> <em>61 </em>
                                    (8)
                                     , 3551-3564. <a href="https://doi.org/10.1021/acs.jmedchem.7b01890" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.7b01890</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.7b01890&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.7b01890%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DThe%252BPharmacophore%252BNetwork%25253A%252BA%252BComputational%252BMethod%252Bfor%252BExploring%252BStructure%2525E2%252580%252593Activity%252BRelationships%252Bfrom%252Ba%252BLarge%252BChemical%252BData%252BSet%26aulast%3DM%25C3%25A9tivier%26aufirst%3DJean-Philippe%26date%3D2018%26date%3D2018%26date%3D2017%26date%3D21122017%26date%3D18042018%26date%3D12042018%26volume%3D61%26issue%3D8%26spage%3D3551%26epage%3D3564" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Erik  Gilberg</span>, <span class="hlFld-ContribAuthor ">Dagmar  Stumpfe</span>, and <span class="hlFld-ContribAuthor ">Jürgen  Bajorath</span>  . </span><span class="cited-content_cbyCitation_article-title">X-ray-Structure-Based Identification of Compounds with Activity against Targets from Different Families and Generation of Templates for Multitarget Ligand Design. </span><span class="cited-content_cbyCitation_journal-name">ACS Omega</span><span> <strong>2018,</strong> <em>3 </em>
                                    (1)
                                     , 106-111. <a href="https://doi.org/10.1021/acsomega.7b01849" title="DOI URL">https://doi.org/10.1021/acsomega.7b01849</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsomega.7b01849&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsomega.7b01849%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Omega%26atitle%3DX-ray-Structure-Based%252BIdentification%252Bof%252BCompounds%252Bwith%252BActivity%252Bagainst%252BTargets%252Bfrom%252BDifferent%252BFamilies%252Band%252BGeneration%252Bof%252BTemplates%252Bfor%252BMultitarget%252BLigand%252BDesign%26aulast%3DGilberg%26aufirst%3DErik%26date%3D2018%26date%3D2018%26date%3D2017%26date%3D2017%26date%3D24112017%26date%3D18122017%26date%3D31012018%26volume%3D3%26issue%3D1%26spage%3D106%26epage%3D111" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Tomohiro  Sato</span>, <span class="hlFld-ContribAuthor ">Noriaki  Hashimoto</span>, and <span class="hlFld-ContribAuthor ">Teruki  Honma</span>  . </span><span class="cited-content_cbyCitation_article-title">Bioisostere Identification by Determining the Amino Acid Binding Preferences of Common Chemical Fragments. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2017,</strong> <em>57 </em>
                                    (12)
                                     , 2938-2947. <a href="https://doi.org/10.1021/acs.jcim.7b00092" title="DOI URL">https://doi.org/10.1021/acs.jcim.7b00092</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.7b00092&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.7b00092%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DBioisostere%252BIdentification%252Bby%252BDetermining%252Bthe%252BAmino%252BAcid%252BBinding%252BPreferences%252Bof%252BCommon%252BChemical%252BFragments%26aulast%3DSato%26aufirst%3DTomohiro%26date%3D2017%26date%3D2017%26date%3D2017%26date%3D17022017%26date%3D01122017%26date%3D26122017%26date%3D07112017%26volume%3D57%26issue%3D12%26spage%3D2938%26epage%3D2947" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Samo  Turk</span>, <span class="hlFld-ContribAuthor ">Benjamin  Merget</span>, <span class="hlFld-ContribAuthor ">Friedrich  Rippmann</span>, and <span class="hlFld-ContribAuthor ">Simone  Fulle</span>  . </span><span class="cited-content_cbyCitation_article-title">Coupling Matched Molecular Pairs with Machine Learning for Virtual Compound Optimization. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2017,</strong> <em>57 </em>
                                    (12)
                                     , 3079-3085. <a href="https://doi.org/10.1021/acs.jcim.7b00298" title="DOI URL">https://doi.org/10.1021/acs.jcim.7b00298</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.7b00298&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.7b00298%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DCoupling%252BMatched%252BMolecular%252BPairs%252Bwith%252BMachine%252BLearning%252Bfor%252BVirtual%252BCompound%252BOptimization%26aulast%3DTurk%26aufirst%3DSamo%26date%3D2017%26date%3D2017%26date%3D2017%26date%3D23052017%26date%3D27112017%26date%3D26122017%26date%3D13112017%26volume%3D57%26issue%3D12%26spage%3D3079%26epage%3D3085" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Richard  Lonsdale</span>, <span class="hlFld-ContribAuthor ">Jonathan  Burgess</span>, <span class="hlFld-ContribAuthor ">Nicola  Colclough</span>, <span class="hlFld-ContribAuthor ">Nichola L.  Davies</span>, <span class="hlFld-ContribAuthor ">Eva M.  Lenz</span>, <span class="hlFld-ContribAuthor ">Alexandra L.  Orton</span>, and <span class="hlFld-ContribAuthor ">Richard A.  Ward</span>  . </span><span class="cited-content_cbyCitation_article-title">Expanding the Armory: Predicting and Tuning Covalent Warhead Reactivity. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2017,</strong> <em>57 </em>
                                    (12)
                                     , 3124-3137. <a href="https://doi.org/10.1021/acs.jcim.7b00553" title="DOI URL">https://doi.org/10.1021/acs.jcim.7b00553</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.7b00553&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.7b00553%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DExpanding%252Bthe%252BArmory%25253A%252BPredicting%252Band%252BTuning%252BCovalent%252BWarhead%252BReactivity%26aulast%3DLonsdale%26aufirst%3DRichard%26date%3D2017%26date%3D2017%26date%3D2017%26date%3D13092017%26date%3D21112017%26date%3D26122017%26date%3D13112017%26volume%3D57%26issue%3D12%26spage%3D3124%26epage%3D3137" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Iva  Lukac</span>, <span class="hlFld-ContribAuthor ">Joanna  Zarnecka</span>, <span class="hlFld-ContribAuthor ">Edward J.  Griffen</span>, <span class="hlFld-ContribAuthor ">Alexander G.  Dossetter</span>, <span class="hlFld-ContribAuthor ">Stephen A.  St-Gallay</span>, <span class="hlFld-ContribAuthor ">Steven J.  Enoch</span>, <span class="hlFld-ContribAuthor ">Judith C.  Madden</span>, and <span class="hlFld-ContribAuthor ">Andrew G.  Leach</span>  . </span><span class="cited-content_cbyCitation_article-title">Turbocharging Matched Molecular Pair Analysis: Optimizing the Identification and Analysis of Pairs. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2017,</strong> <em>57 </em>
                                    (10)
                                     , 2424-2436. <a href="https://doi.org/10.1021/acs.jcim.7b00335" title="DOI URL">https://doi.org/10.1021/acs.jcim.7b00335</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.7b00335&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.7b00335%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DTurbocharging%252BMatched%252BMolecular%252BPair%252BAnalysis%25253A%252BOptimizing%252Bthe%252BIdentification%252Band%252BAnalysis%252Bof%252BPairs%26aulast%3DLukac%26aufirst%3DIva%26date%3D2017%26date%3D2017%26date%3D2017%26date%3D06062017%26date%3D02102017%26date%3D23102017%26volume%3D57%26issue%3D10%26spage%3D2424%26epage%3D2436" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ignacio  Aliagas</span>, <span class="hlFld-ContribAuthor ">Raphaëlle  Berger</span>, <span class="hlFld-ContribAuthor ">Kristin  Goldberg</span>, <span class="hlFld-ContribAuthor ">Rachel T.  Nishimura</span>, <span class="hlFld-ContribAuthor ">John  Reilly</span>, <span class="hlFld-ContribAuthor ">Paul  Richardson</span>, <span class="hlFld-ContribAuthor ">Daniel  Richter</span>, <span class="hlFld-ContribAuthor ">Edward C.  Sherer</span>, <span class="hlFld-ContribAuthor ">Brian A.  Sparling</span>, and <span class="hlFld-ContribAuthor ">Marian C.  Bryan</span>  . </span><span class="cited-content_cbyCitation_article-title">Sustainable Practices in Medicinal Chemistry Part 2: Green by Design. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2017,</strong> <em>60 </em>
                                    (14)
                                     , 5955-5968. <a href="https://doi.org/10.1021/acs.jmedchem.6b01837" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.6b01837</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.6b01837&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.6b01837%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DSustainable%252BPractices%252Bin%252BMedicinal%252BChemistry%252BPart%252B2%25253A%252BGreen%252Bby%252BDesign%26aulast%3DAliagas%26aufirst%3DIgnacio%26date%3D2017%26date%3D2017%26date%3D2016%26date%3D14122016%26date%3D12042017%26date%3D27072017%26date%3D04042017%26volume%3D60%26issue%3D14%26spage%3D5955%26epage%3D5968" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Richard J.  Hall</span>, <span class="hlFld-ContribAuthor ">Christopher W.  Murray</span>, and <span class="hlFld-ContribAuthor ">Marcel L.  Verdonk</span>  . </span><span class="cited-content_cbyCitation_article-title">The Fragment Network: A Chemistry Recommendation Engine Built Using a Graph Database. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2017,</strong> <em>60 </em>
                                    (14)
                                     , 6440-6450. <a href="https://doi.org/10.1021/acs.jmedchem.7b00809" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.7b00809</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.7b00809&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.7b00809%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DThe%252BFragment%252BNetwork%25253A%252BA%252BChemistry%252BRecommendation%252BEngine%252BBuilt%252BUsing%252Ba%252BGraph%252BDatabase%26aulast%3DHall%26aufirst%3DRichard%2BJ.%26date%3D2017%26date%3D2017%26date%3D2017%26date%3D02062017%26date%3D15072017%26date%3D27072017%26volume%3D60%26issue%3D14%26spage%3D6440%26epage%3D6450" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jeremy  Ash</span> and <span class="hlFld-ContribAuthor ">Denis  Fourches</span>  . </span><span class="cited-content_cbyCitation_article-title">Characterizing the Chemical Space of ERK2 Kinase Inhibitors Using Descriptors Computed from Molecular Dynamics Trajectories. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2017,</strong> <em>57 </em>
                                    (6)
                                     , 1286-1299. <a href="https://doi.org/10.1021/acs.jcim.7b00048" title="DOI URL">https://doi.org/10.1021/acs.jcim.7b00048</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.7b00048&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.7b00048%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DCharacterizing%252Bthe%252BChemical%252BSpace%252Bof%252BERK2%252BKinase%252BInhibitors%252BUsing%252BDescriptors%252BComputed%252Bfrom%252BMolecular%252BDynamics%252BTrajectories%26aulast%3DAsh%26aufirst%3DJeremy%26date%3D2017%26date%3D2017%26date%3D2017%26date%3D25012017%26date%3D19052017%26date%3D26062017%26date%3D04052017%26volume%3D57%26issue%3D6%26spage%3D1286%26epage%3D1299" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Emanuel S. R.  Ehmki</span> and <span class="hlFld-ContribAuthor ">Christian  Kramer</span>  . </span><span class="cited-content_cbyCitation_article-title">Matched Molecular Series: Measuring SAR Similarity. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2017,</strong> <em>57 </em>
                                    (5)
                                     , 1187-1196. <a href="https://doi.org/10.1021/acs.jcim.6b00709" title="DOI URL">https://doi.org/10.1021/acs.jcim.6b00709</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.6b00709&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.6b00709%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DMatched%252BMolecular%252BSeries%25253A%252BMeasuring%252BSAR%252BSimilarity%26aulast%3DEhmki%26aufirst%3DEmanuel%2BS.%2BR.%26date%3D2017%26date%3D2017%26date%3D2016%26date%3D22112016%26date%3D01052017%26date%3D22052017%26volume%3D57%26issue%3D5%26spage%3D1187%26epage%3D1196" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Dragos  Horvath</span>, <span class="hlFld-ContribAuthor ">Gilles  Marcou</span>, <span class="hlFld-ContribAuthor ">Alexandre  Varnek</span>, <span class="hlFld-ContribAuthor ">Shilva  Kayastha</span>, <span class="hlFld-ContribAuthor ">Antonio  de la Vega de León</span>, and <span class="hlFld-ContribAuthor ">Jürgen  Bajorath</span>  . </span><span class="cited-content_cbyCitation_article-title">Prediction of Activity Cliffs Using Condensed Graphs of Reaction Representations, Descriptor Recombination, Support Vector Machine Classification, and Support Vector Regression. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2016,</strong> <em>56 </em>
                                    (9)
                                     , 1631-1640. <a href="https://doi.org/10.1021/acs.jcim.6b00359" title="DOI URL">https://doi.org/10.1021/acs.jcim.6b00359</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.6b00359&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.6b00359%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DPrediction%252Bof%252BActivity%252BCliffs%252BUsing%252BCondensed%252BGraphs%252Bof%252BReaction%252BRepresentations%25252C%252BDescriptor%252BRecombination%25252C%252BSupport%252BVector%252BMachine%252BClassification%25252C%252Band%252BSupport%252BVector%252BRegression%26aulast%3DHorvath%26aufirst%3DDragos%26date%3D2016%26date%3D2016%26date%3D2016%26date%3D16062016%26date%3D26082016%26date%3D26092016%26volume%3D56%26issue%3D9%26spage%3D1631%26epage%3D1640" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Dagmar  Stumpfe</span>, <span class="hlFld-ContribAuthor ">Dilyana  Dimova</span>, and <span class="hlFld-ContribAuthor ">Jürgen  Bajorath</span>  . </span><span class="cited-content_cbyCitation_article-title">Computational Method for the Systematic Identification of Analog Series and Key Compounds Representing Series and Their Biological Activity Profiles. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2016,</strong> <em>59 </em>
                                    (16)
                                     , 7667-7676. <a href="https://doi.org/10.1021/acs.jmedchem.6b00906" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.6b00906</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.6b00906&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.6b00906%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DComputational%252BMethod%252Bfor%252Bthe%252BSystematic%252BIdentification%252Bof%252BAnalog%252BSeries%252Band%252BKey%252BCompounds%252BRepresenting%252BSeries%252Band%252BTheir%252BBiological%252BActivity%252BProfiles%26aulast%3DStumpfe%26aufirst%3DDagmar%26date%3D2016%26date%3D2016%26date%3D2016%26date%3D16062016%26date%3D08082016%26date%3D25082016%26volume%3D59%26issue%3D16%26spage%3D7667%26epage%3D7676" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ye  Hu</span>, <span class="hlFld-ContribAuthor ">Dagmar  Stumpfe</span>, and <span class="hlFld-ContribAuthor ">Jürgen  Bajorath</span>  . </span><span class="cited-content_cbyCitation_article-title">Computational Exploration of Molecular Scaffolds in Medicinal Chemistry. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2016,</strong> <em>59 </em>
                                    (9)
                                     , 4062-4076. <a href="https://doi.org/10.1021/acs.jmedchem.5b01746" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.5b01746</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.5b01746&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.5b01746%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DComputational%252BExploration%252Bof%252BMolecular%252BScaffolds%252Bin%252BMedicinal%252BChemistry%26aulast%3DHu%26aufirst%3DYe%26date%3D2016%26date%3D2016%26date%3D2015%26date%3D09112015%26date%3D03022016%26date%3D12052016%26volume%3D59%26issue%3D9%26spage%3D4062%26epage%3D4076" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Helen E.  Colley</span>, <span class="hlFld-ContribAuthor ">Munitta  Muthana</span>, <span class="hlFld-ContribAuthor ">Sarah J.  Danson</span>, <span class="hlFld-ContribAuthor ">Lucinda V.  Jackson</span>, <span class="hlFld-ContribAuthor ">Matthew L.  Brett</span>, <span class="hlFld-ContribAuthor ">Joanne  Harrison</span>, <span class="hlFld-ContribAuthor ">Sean F.  Coole</span>, <span class="hlFld-ContribAuthor ">Daniel P.  Mason</span>, <span class="hlFld-ContribAuthor ">Luke R.  Jennings</span>, <span class="hlFld-ContribAuthor ">Melanie  Wong</span>, <span class="hlFld-ContribAuthor ">Vamshi  Tulasi</span>, <span class="hlFld-ContribAuthor ">Dennis  Norman</span>, <span class="hlFld-ContribAuthor ">Peter M.  Lockey</span>, <span class="hlFld-ContribAuthor ">Lynne  Williams</span>, <span class="hlFld-ContribAuthor ">Alexander G.  Dossetter</span>, <span class="hlFld-ContribAuthor ">Edward J.  Griffen</span>, and <span class="hlFld-ContribAuthor ">Mark J.  Thompson</span>  . </span><span class="cited-content_cbyCitation_article-title">An Orally Bioavailable, Indole-3-glyoxylamide Based Series of Tubulin Polymerization Inhibitors Showing Tumor Growth Inhibition in a Mouse Xenograft Model of Head and Neck Cancer. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2015,</strong> <em>58 </em>
                                    (23)
                                     , 9309-9333. <a href="https://doi.org/10.1021/acs.jmedchem.5b01312" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.5b01312</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.5b01312&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.5b01312%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DAn%252BOrally%252BBioavailable%25252C%252BIndole-3-glyoxylamide%252BBased%252BSeries%252Bof%252BTubulin%252BPolymerization%252BInhibitors%252BShowing%252BTumor%252BGrowth%252BInhibition%252Bin%252Ba%252BMouse%252BXenograft%252BModel%252Bof%252BHead%252Band%252BNeck%252BCancer%26aulast%3DColley%26aufirst%3DHelen%2BE.%26date%3D2015%26date%3D2015%26date%3D2015%26date%3D25082015%26date%3D01122015%26date%3D10122015%26date%3D18112015%26volume%3D58%26issue%3D23%26spage%3D9309%26epage%3D9333" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Julian E.  Fuchs</span>, <span class="hlFld-ContribAuthor ">Bernd  Wellenzohn</span>, <span class="hlFld-ContribAuthor ">Nils  Weskamp</span>, and <span class="hlFld-ContribAuthor ">Klaus R.  Liedl</span>  . </span><span class="cited-content_cbyCitation_article-title">Matched Peptides: Tuning Matched Molecular Pair Analysis for Biopharmaceutical Applications. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2015,</strong> <em>55 </em>
                                    (11)
                                     , 2315-2323. <a href="https://doi.org/10.1021/acs.jcim.5b00476" title="DOI URL">https://doi.org/10.1021/acs.jcim.5b00476</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.5b00476&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.5b00476%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DMatched%252BPeptides%25253A%252BTuning%252BMatched%252BMolecular%252BPair%252BAnalysis%252Bfor%252BBiopharmaceutical%252BApplications%26aulast%3DFuchs%26aufirst%3DJulian%2BE.%26date%3D2015%26date%3D2015%26date%3D2015%26date%3D31072015%26date%3D06112015%26date%3D23112015%26date%3D26102015%26volume%3D55%26issue%3D11%26spage%3D2315%26epage%3D2323" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Kyosuke  Ueda</span>, <span class="hlFld-ContribAuthor ">Hirotatsu  Umihara</span>, <span class="hlFld-ContribAuthor ">Satoshi  Yokoshima</span>, and <span class="hlFld-ContribAuthor ">Tohru  Fukuyama</span>  . </span><span class="cited-content_cbyCitation_article-title">Conversion of Ester Moieties to 4-Bromophenyl Groups via Electrocyclic Reaction of Dibromocyclopropanes. </span><span class="cited-content_cbyCitation_journal-name">Organic Letters</span><span> <strong>2015,</strong> <em>17 </em>
                                    (13)
                                     , 3191-3193. <a href="https://doi.org/10.1021/acs.orglett.5b01351" title="DOI URL">https://doi.org/10.1021/acs.orglett.5b01351</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.orglett.5b01351&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.orglett.5b01351%26sid%3Dliteratum%253Aachs%26jtitle%3DOrganic%2520Letters%26atitle%3DConversion%252Bof%252BEster%252BMoieties%252Bto%252B4-Bromophenyl%252BGroups%252Bvia%252BElectrocyclic%252BReaction%252Bof%252BDibromocyclopropanes%26aulast%3DUeda%26aufirst%3DKyosuke%26date%3D2015%26date%3D2015%26date%3D2015%26date%3D07052015%26date%3D12062015%26date%3D02072015%26volume%3D17%26issue%3D13%26spage%3D3191%26epage%3D3193" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jarmila  Husby</span>, <span class="hlFld-ContribAuthor ">Giovanni  Bottegoni</span>, <span class="hlFld-ContribAuthor ">Irina  Kufareva</span>, <span class="hlFld-ContribAuthor ">Ruben  Abagyan</span>, and <span class="hlFld-ContribAuthor ">Andrea  Cavalli</span>  . </span><span class="cited-content_cbyCitation_article-title">Structure-Based Predictions of Activity Cliffs. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2015,</strong> <em>55 </em>
                                    (5)
                                     , 1062-1076. <a href="https://doi.org/10.1021/ci500742b" title="DOI URL">https://doi.org/10.1021/ci500742b</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/ci500742b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fci500742b%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DStructure-Based%252BPredictions%252Bof%252BActivity%252BCliffs%26aulast%3DHusby%26aufirst%3DJarmila%26date%3D2015%26date%3D2015%26date%3D2014%26date%3D16122014%26date%3D11052015%26date%3D26052015%26date%3D28042015%26volume%3D55%26issue%3D5%26spage%3D1062%26epage%3D1076" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Julia  Weber</span>, <span class="hlFld-ContribAuthor ">Janosch  Achenbach</span>, <span class="hlFld-ContribAuthor ">Daniel  Moser</span>, and <span class="hlFld-ContribAuthor ">Ewgenij  Proschak</span>  . </span><span class="cited-content_cbyCitation_article-title">VAMMPIRE-LORD: A Web Server for Straightforward Lead Optimization Using Matched Molecular Pairs. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2015,</strong> <em>55 </em>
                                    (2)
                                     , 207-213. <a href="https://doi.org/10.1021/ci5005256" title="DOI URL">https://doi.org/10.1021/ci5005256</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/ci5005256&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fci5005256%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DVAMMPIRE-LORD%25253A%252BA%252BWeb%252BServer%252Bfor%252BStraightforward%252BLead%252BOptimization%252BUsing%252BMatched%252BMolecular%252BPairs%26aulast%3DWeber%26aufirst%3DJulia%26date%3D2015%26date%3D2015%26date%3D2014%26date%3D30082014%26date%3D10022015%26date%3D23022015%26date%3D28012015%26volume%3D55%26issue%3D2%26spage%3D207%26epage%3D213" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Anthony R.  Bradley</span>, <span class="hlFld-ContribAuthor ">Ian D.  Wall</span>, <span class="hlFld-ContribAuthor ">Darren V. S.  Green</span>, <span class="hlFld-ContribAuthor ">Charlotte M.  Deane</span>, and <span class="hlFld-ContribAuthor ">Brian D.  Marsden</span>  . </span><span class="cited-content_cbyCitation_article-title">OOMMPPAA: A Tool To Aid Directed Synthesis by the Combined Analysis of Activity and Structural Data. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2014,</strong> <em>54 </em>
                                    (10)
                                     , 2636-2646. <a href="https://doi.org/10.1021/ci500245d" title="DOI URL">https://doi.org/10.1021/ci500245d</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/ci500245d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fci500245d%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DOOMMPPAA%25253A%252BA%252BTool%252BTo%252BAid%252BDirected%252BSynthesis%252Bby%252Bthe%252BCombined%252BAnalysis%252Bof%252BActivity%252Band%252BStructural%252BData%26aulast%3DBradley%26aufirst%3DAnthony%2BR.%26date%3D2014%26date%3D2014%26date%3D2014%26date%3D23042014%26date%3D09102014%26date%3D27102014%26date%3D22092014%26volume%3D54%26issue%3D10%26spage%3D2636%26epage%3D2646" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Sabrina  Wollenhaupt</span> and <span class="hlFld-ContribAuthor ">Knut  Baumann</span>  . </span><span class="cited-content_cbyCitation_article-title">inSARa: Intuitive and Interactive SAR Interpretation by Reduced Graphs and Hierarchical MCS-Based Network Navigation. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2014,</strong> <em>54 </em>
                                    (6)
                                     , 1578-1595. <a href="https://doi.org/10.1021/ci4007547" title="DOI URL">https://doi.org/10.1021/ci4007547</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/ci4007547&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fci4007547%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DinSARa%25253A%252BIntuitive%252Band%252BInteractive%252BSAR%252BInterpretation%252Bby%252BReduced%252BGraphs%252Band%252BHierarchical%252BMCS-Based%252BNetwork%252BNavigation%26aulast%3DWollenhaupt%26aufirst%3DSabrina%26date%3D2014%26date%3D2014%26date%3D2013%26date%3D19122013%26date%3D04062014%26date%3D23062014%26date%3D21052014%26volume%3D54%26issue%3D6%26spage%3D1578%26epage%3D1595" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Christian  Kramer</span>, <span class="hlFld-ContribAuthor ">Julian E.  Fuchs</span>, <span class="hlFld-ContribAuthor ">Steven  Whitebread</span>, <span class="hlFld-ContribAuthor ">Peter  Gedeck</span>, and <span class="hlFld-ContribAuthor ">Klaus R.  Liedl</span>  . </span><span class="cited-content_cbyCitation_article-title">Matched Molecular Pair Analysis: Significance and the Impact of Experimental Uncertainty. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2014,</strong> <em>57 </em>
                                    (9)
                                     , 3786-3802. <a href="https://doi.org/10.1021/jm500317a" title="DOI URL">https://doi.org/10.1021/jm500317a</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/jm500317a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fjm500317a%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DMatched%252BMolecular%252BPair%252BAnalysis%25253A%252BSignificance%252Band%252Bthe%252BImpact%252Bof%252BExperimental%252BUncertainty%26aulast%3DKramer%26aufirst%3DChristian%26date%3D2014%26date%3D2014%26date%3D2014%26date%3D17012014%26date%3D16042014%26date%3D08052014%26volume%3D57%26issue%3D9%26spage%3D3786%26epage%3D3802" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Tim  Geppert</span> and <span class="hlFld-ContribAuthor ">Bernd  Beck</span>  . </span><span class="cited-content_cbyCitation_article-title">Fuzzy Matched Pairs: A Means To Determine the Pharmacophore Impact on Molecular Interaction. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2014,</strong> <em>54 </em>
                                    (4)
                                     , 1093-1102. <a href="https://doi.org/10.1021/ci400694q" title="DOI URL">https://doi.org/10.1021/ci400694q</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/ci400694q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fci400694q%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DFuzzy%252BMatched%252BPairs%25253A%252BA%252BMeans%252BTo%252BDetermine%252Bthe%252BPharmacophore%252BImpact%252Bon%252BMolecular%252BInteraction%26aulast%3DGeppert%26aufirst%3DTim%26date%3D2014%26date%3D2014%26date%3D2013%26date%3D25112013%26date%3D17032014%26date%3D28042014%26date%3D04032014%26volume%3D54%26issue%3D4%26spage%3D1093%26epage%3D1102" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Noel M.  O’Boyle</span>, <span class="hlFld-ContribAuthor ">Jonas  Boström</span>, <span class="hlFld-ContribAuthor ">Roger A.  Sayle</span>, and <span class="hlFld-ContribAuthor ">Adrian  Gill</span>  . </span><span class="cited-content_cbyCitation_article-title">Using Matched Molecular Series as a Predictive Tool To Optimize Biological Activity. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2014,</strong> <em>57 </em>
                                    (6)
                                     , 2704-2713. <a href="https://doi.org/10.1021/jm500022q" title="DOI URL">https://doi.org/10.1021/jm500022q</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/jm500022q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fjm500022q%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DUsing%252BMatched%252BMolecular%252BSeries%252Bas%252Ba%252BPredictive%252BTool%252BTo%252BOptimize%252BBiological%252BActivity%26aulast%3DO%25E2%2580%2599Boyle%26aufirst%3DNoel%2BM.%26date%3D2014%26date%3D2014%26date%3D2014%26date%3D06012014%26date%3D14032014%26date%3D27032014%26date%3D06032014%26volume%3D57%26issue%3D6%26spage%3D2704%26epage%3D2713" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Mark D.  Rackham</span>, <span class="hlFld-ContribAuthor ">James A.  Brannigan</span>, <span class="hlFld-ContribAuthor ">Kaveri  Rangachari</span>, <span class="hlFld-ContribAuthor ">Stephan  Meister</span>, <span class="hlFld-ContribAuthor ">Anthony J.  Wilkinson</span>, <span class="hlFld-ContribAuthor ">Anthony A.  Holder</span>, <span class="hlFld-ContribAuthor ">Robin J.  Leatherbarrow</span>, and <span class="hlFld-ContribAuthor ">Edward W.  Tate</span>  . </span><span class="cited-content_cbyCitation_article-title">Design and Synthesis of High Affinity Inhibitors of Plasmodium falciparum and Plasmodium vivax N-Myristoyltransferases Directed by Ligand Efficiency Dependent Lipophilicity (LELP). </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2014,</strong> <em>57 </em>
                                    (6)
                                     , 2773-2788. <a href="https://doi.org/10.1021/jm500066b" title="DOI URL">https://doi.org/10.1021/jm500066b</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/jm500066b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fjm500066b%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDesign%252Band%252BSynthesis%252Bof%252BHigh%252BAffinity%252BInhibitors%252Bof%252BPlasmodium%252Bfalciparum%252Band%252BPlasmodium%252Bvivax%252BN-Myristoyltransferases%252BDirected%252Bby%252BLigand%252BEfficiency%252BDependent%252BLipophilicity%252B%252528LELP%252529%26aulast%3DRackham%26aufirst%3DMark%2BD.%26date%3D2014%26date%3D2014%26date%3D2014%26date%3D13012014%26date%3D18032014%26date%3D27032014%26date%3D05032014%26volume%3D57%26issue%3D6%26spage%3D2773%26epage%3D2788" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Disha  Gupta-Ostermann</span>, <span class="hlFld-ContribAuthor ">Veerabahu  Shanmugasundaram</span>, and <span class="hlFld-ContribAuthor ">Jürgen  Bajorath</span>  . </span><span class="cited-content_cbyCitation_article-title">Neighborhood-Based Prediction of Novel Active Compounds from SAR Matrices. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2014,</strong> <em>54 </em>
                                    (3)
                                     , 801-809. <a href="https://doi.org/10.1021/ci5000483" title="DOI URL">https://doi.org/10.1021/ci5000483</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/ci5000483&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fci5000483%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DNeighborhood-Based%252BPrediction%252Bof%252BNovel%252BActive%252BCompounds%252Bfrom%252BSAR%252BMatrices%26aulast%3DGupta-Ostermann%26aufirst%3DDisha%26date%3D2014%26date%3D2014%26date%3D2014%26date%3D23012014%26date%3D12032014%26date%3D24032014%26date%3D04032014%26volume%3D54%26issue%3D3%26spage%3D801%26epage%3D809" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Anthony A.  Estrada</span>, <span class="hlFld-ContribAuthor ">Bryan K.  Chan</span>, <span class="hlFld-ContribAuthor ">Charles  Baker-Glenn</span>, <span class="hlFld-ContribAuthor ">Alan  Beresford</span>, <span class="hlFld-ContribAuthor ">Daniel J.  Burdick</span>, <span class="hlFld-ContribAuthor ">Mark  Chambers</span>, <span class="hlFld-ContribAuthor ">Huifen  Chen</span>, <span class="hlFld-ContribAuthor ">Sara L.  Dominguez</span>, <span class="hlFld-ContribAuthor ">Jennafer  Dotson</span>, <span class="hlFld-ContribAuthor ">Jason  Drummond</span>, <span class="hlFld-ContribAuthor ">Michael  Flagella</span>, <span class="hlFld-ContribAuthor ">Reina  Fuji</span>, <span class="hlFld-ContribAuthor ">Andrew  Gill</span>, <span class="hlFld-ContribAuthor ">Jason  Halladay</span>, <span class="hlFld-ContribAuthor ">Seth F.  Harris</span>, <span class="hlFld-ContribAuthor ">Timothy P.  Heffron</span>, <span class="hlFld-ContribAuthor ">Tracy  Kleinheinz</span>, <span class="hlFld-ContribAuthor ">Donna W.  Lee</span>, <span class="hlFld-ContribAuthor ">Claire E. Le  Pichon</span>, <span class="hlFld-ContribAuthor ">Xingrong  Liu</span>, <span class="hlFld-ContribAuthor ">Joseph P.  Lyssikatos</span>, <span class="hlFld-ContribAuthor ">Andrew D.  Medhurst</span>, <span class="hlFld-ContribAuthor ">John G.  Moffat</span>, <span class="hlFld-ContribAuthor ">Kevin  Nash</span>, <span class="hlFld-ContribAuthor ">Kimberly  Scearce-Levie</span>, <span class="hlFld-ContribAuthor ">Zejuan  Sheng</span>, <span class="hlFld-ContribAuthor ">Daniel G.  Shore</span>, <span class="hlFld-ContribAuthor ">Susan  Wong</span>, <span class="hlFld-ContribAuthor ">Shuo  Zhang</span>, <span class="hlFld-ContribAuthor ">Xiaolin  Zhang</span>, <span class="hlFld-ContribAuthor ">Haitao  Zhu</span>, and <span class="hlFld-ContribAuthor ">Zachary K.  Sweeney</span>  . </span><span class="cited-content_cbyCitation_article-title">Discovery of Highly Potent, Selective, and Brain-Penetrant Aminopyrazole Leucine-Rich Repeat Kinase 2 (LRRK2) Small Molecule Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2014,</strong> <em>57 </em>
                                    (3)
                                     , 921-936. <a href="https://doi.org/10.1021/jm401654j" title="DOI URL">https://doi.org/10.1021/jm401654j</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/jm401654j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fjm401654j%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252BHighly%252BPotent%25252C%252BSelective%25252C%252Band%252BBrain-Penetrant%252BAminopyrazole%252BLeucine-Rich%252BRepeat%252BKinase%252B2%252B%252528LRRK2%252529%252BSmall%252BMolecule%252BInhibitors%26aulast%3DEstrada%26aufirst%3DAnthony%2BA.%26date%3D2014%26date%3D2014%26date%3D2013%26date%3D25102013%26date%3D15012014%26date%3D13022014%26date%3D19122013%26volume%3D57%26issue%3D3%26spage%3D921%26epage%3D936" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Matthew  Matlock</span> and <span class="hlFld-ContribAuthor ">S. Joshua  Swamidass</span>  . </span><span class="cited-content_cbyCitation_article-title">Sharing Chemical Relationships Does Not Reveal Structures. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2014,</strong> <em>54 </em>
                                    (1)
                                     , 37-48. <a href="https://doi.org/10.1021/ci400399a" title="DOI URL">https://doi.org/10.1021/ci400399a</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/ci400399a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fci400399a%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DSharing%252BChemical%252BRelationships%252BDoes%252BNot%252BReveal%252BStructures%26aulast%3DMatlock%26aufirst%3DMatthew%26date%3D2014%26date%3D2013%26date%3D2013%26date%3D08072013%26date%3D16122013%26date%3D27012014%26date%3D01122013%26volume%3D54%26issue%3D1%26spage%3D37%26epage%3D48" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Antonia F.  Stepan</span>, <span class="hlFld-ContribAuthor ">Gregory W.  Kauffman</span>, <span class="hlFld-ContribAuthor ">Christopher E.  Keefer</span>, <span class="hlFld-ContribAuthor ">Patrick R.  Verhoest</span>, and <span class="hlFld-ContribAuthor ">Martin  Edwards</span>  . </span><span class="cited-content_cbyCitation_article-title">Evaluating the Differences in Cycloalkyl Ether Metabolism Using the Design Parameter “Lipophilic Metabolism Efficiency” (LipMetE) and a Matched Molecular Pairs Analysis. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2013,</strong> <em>56 </em>
                                    (17)
                                     , 6985-6990. <a href="https://doi.org/10.1021/jm4008642" title="DOI URL">https://doi.org/10.1021/jm4008642</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/jm4008642&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fjm4008642%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DEvaluating%252Bthe%252BDifferences%252Bin%252BCycloalkyl%252BEther%252BMetabolism%252BUsing%252Bthe%252BDesign%252BParameter%252B%2525E2%252580%25259CLipophilic%252BMetabolism%252BEfficiency%2525E2%252580%25259D%252B%252528LipMetE%252529%252Band%252Ba%252BMatched%252BMolecular%252BPairs%252BAnalysis%26aulast%3DStepan%26aufirst%3DAntonia%2BF.%26date%3D2013%26date%3D2013%26date%3D2013%26date%3D28052013%26date%3D21082013%26date%3D12092013%26date%3D25072013%26volume%3D56%26issue%3D17%26spage%3D6985%26epage%3D6990" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Michael D.  Shultz</span>, <span class="hlFld-ContribAuthor ">Atwood K.  Cheung</span>, <span class="hlFld-ContribAuthor ">Christina A.  Kirby</span>, <span class="hlFld-ContribAuthor ">Brant  Firestone</span>, <span class="hlFld-ContribAuthor ">Jianmei  Fan</span>, <span class="hlFld-ContribAuthor ">Christine Hiu-Tung  Chen</span>, <span class="hlFld-ContribAuthor ">Zhouliang  Chen</span>, <span class="hlFld-ContribAuthor ">Donovan N.  Chin</span>, <span class="hlFld-ContribAuthor ">Lucian  DiPietro</span>, <span class="hlFld-ContribAuthor ">Aleem  Fazal</span>, <span class="hlFld-ContribAuthor ">Yun  Feng</span>, <span class="hlFld-ContribAuthor ">Pascal D.  Fortin</span>, <span class="hlFld-ContribAuthor ">Ty  Gould</span>, <span class="hlFld-ContribAuthor ">Bharat  Lagu</span>, <span class="hlFld-ContribAuthor ">Huangshu  Lei</span>, <span class="hlFld-ContribAuthor ">Francois  Lenoir</span>, <span class="hlFld-ContribAuthor ">Dyuti  Majumdar</span>, <span class="hlFld-ContribAuthor ">Etienne  Ochala</span>, <span class="hlFld-ContribAuthor ">M. G.  Palermo</span>, <span class="hlFld-ContribAuthor ">Ly  Pham</span>, <span class="hlFld-ContribAuthor ">Minying  Pu</span>, <span class="hlFld-ContribAuthor ">Troy  Smith</span>, <span class="hlFld-ContribAuthor ">Travis  Stams</span>, <span class="hlFld-ContribAuthor ">Ronald C.  Tomlinson</span>, <span class="hlFld-ContribAuthor ">B. Barry  Touré</span>, <span class="hlFld-ContribAuthor ">Michael  Visser</span>, <span class="hlFld-ContribAuthor ">Run Ming  Wang</span>, <span class="hlFld-ContribAuthor ">Nigel J.  Waters</span>, and <span class="hlFld-ContribAuthor ">Wenlin  Shao</span>  . </span><span class="cited-content_cbyCitation_article-title">Identification of NVP-TNKS656: The Use of Structure–Efficiency Relationships To Generate a Highly Potent, Selective, and Orally Active Tankyrase Inhibitor. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2013,</strong> <em>56 </em>
                                    (16)
                                     , 6495-6511. <a href="https://doi.org/10.1021/jm400807n" title="DOI URL">https://doi.org/10.1021/jm400807n</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/jm400807n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fjm400807n%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DIdentification%252Bof%252BNVP-TNKS656%25253A%252BThe%252BUse%252Bof%252BStructure%2525E2%252580%252593Efficiency%252BRelationships%252BTo%252BGenerate%252Ba%252BHighly%252BPotent%25252C%252BSelective%25252C%252Band%252BOrally%252BActive%252BTankyrase%252BInhibitor%26aulast%3DShultz%26aufirst%3DMichael%2BD.%26date%3D2013%26date%3D2013%26date%3D2013%26date%3D30052013%26date%3D13082013%26date%3D22082013%26date%3D11072013%26volume%3D56%26issue%3D16%26spage%3D6495%26epage%3D6511" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Shana L.  Posy</span>, <span class="hlFld-ContribAuthor ">Brian L.  Claus</span>, <span class="hlFld-ContribAuthor ">Matt E.  Pokross</span>, and <span class="hlFld-ContribAuthor ">Stephen R.  Johnson</span>  . </span><span class="cited-content_cbyCitation_article-title">3D Matched Pairs: Integrating Ligand- and Structure-Based Knowledge for Ligand Design and Receptor Annotation. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2013,</strong> <em>53 </em>
                                    (7)
                                     , 1576-1588. <a href="https://doi.org/10.1021/ci400201k" title="DOI URL">https://doi.org/10.1021/ci400201k</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/ci400201k&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fci400201k%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3D3D%252BMatched%252BPairs%25253A%252BIntegrating%252BLigand-%252Band%252BStructure-Based%252BKnowledge%252Bfor%252BLigand%252BDesign%252Band%252BReceptor%252BAnnotation%26aulast%3DPosy%26aufirst%3DShana%2BL.%26date%3D2013%26date%3D2013%26date%3D2013%26date%3D04042013%26date%3D10072013%26date%3D22072013%26date%3D30062013%26volume%3D53%26issue%3D7%26spage%3D1576%26epage%3D1588" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Julia  Weber</span>, <span class="hlFld-ContribAuthor ">Janosch  Achenbach</span>, <span class="hlFld-ContribAuthor ">Daniel  Moser</span>, and <span class="hlFld-ContribAuthor ">Ewgenij  Proschak</span>  . </span><span class="cited-content_cbyCitation_article-title">VAMMPIRE: A Matched Molecular Pairs Database for Structure-Based Drug Design and Optimization. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2013,</strong> <em>56 </em>
                                    (12)
                                     , 5203-5207. <a href="https://doi.org/10.1021/jm400223y" title="DOI URL">https://doi.org/10.1021/jm400223y</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/jm400223y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fjm400223y%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DVAMMPIRE%25253A%252BA%252BMatched%252BMolecular%252BPairs%252BDatabase%252Bfor%252BStructure-Based%252BDrug%252BDesign%252Band%252BOptimization%26aulast%3DWeber%26aufirst%3DJulia%26date%3D2013%26date%3D2013%26date%3D2013%26date%3D13022013%26date%3D18062013%26date%3D27062013%26date%3D04062013%26volume%3D56%26issue%3D12%26spage%3D5203%26epage%3D5207" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Antonio  de la Vega de León</span> and <span class="hlFld-ContribAuthor ">Jürgen  Bajorath</span>  . </span><span class="cited-content_cbyCitation_article-title">Compound Optimization through Data Set-Dependent Chemical Transformations. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2013,</strong> <em>53 </em>
                                    (6)
                                     , 1263-1271. <a href="https://doi.org/10.1021/ci400165a" title="DOI URL">https://doi.org/10.1021/ci400165a</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/ci400165a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fci400165a%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DCompound%252BOptimization%252Bthrough%252BData%252BSet-Dependent%252BChemical%252BTransformations%26aulast%3Dde%2Bla%2BVega%2Bde%2BLe%25C3%25B3n%26aufirst%3DAntonio%26date%3D2013%26date%3D2013%26date%3D2013%26date%3D19032013%26date%3D20052013%26date%3D24062013%26date%3D08052013%26volume%3D53%26issue%3D6%26spage%3D1263%26epage%3D1271" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Anthony A.  Estrada</span>, <span class="hlFld-ContribAuthor ">Daniel G.  Shore</span>, <span class="hlFld-ContribAuthor ">Elizabeth  Blackwood</span>, <span class="hlFld-ContribAuthor ">Yung-Hsiang  Chen</span>, <span class="hlFld-ContribAuthor ">Gauri  Deshmukh</span>, <span class="hlFld-ContribAuthor ">Xiao  Ding</span>, <span class="hlFld-ContribAuthor ">Antonio G.  DiPasquale</span>, <span class="hlFld-ContribAuthor ">Jennifer A.  Epler</span>, <span class="hlFld-ContribAuthor ">Lori S.  Friedman</span>, <span class="hlFld-ContribAuthor ">Michael F. T.  Koehler</span>, <span class="hlFld-ContribAuthor ">Lichuan  Liu</span>, <span class="hlFld-ContribAuthor ">Shiva  Malek</span>, <span class="hlFld-ContribAuthor ">Jim  Nonomiya</span>, <span class="hlFld-ContribAuthor ">Daniel F.  Ortwine</span>, <span class="hlFld-ContribAuthor ">Zhonghua  Pei</span>, <span class="hlFld-ContribAuthor ">Steve  Sideris</span>, <span class="hlFld-ContribAuthor ">Frederic  St-Jean</span>, <span class="hlFld-ContribAuthor ">Lan  Trinh</span>, <span class="hlFld-ContribAuthor ">Tom  Truong</span>, and <span class="hlFld-ContribAuthor ">Joseph P.  Lyssikatos</span>  . </span><span class="cited-content_cbyCitation_article-title">Pyrimidoaminotropanes as Potent, Selective, and Efficacious Small Molecule Kinase Inhibitors of the Mammalian Target of Rapamycin (mTOR). </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2013,</strong> <em>56 </em>
                                    (7)
                                     , 3090-3101. <a href="https://doi.org/10.1021/jm400194n" title="DOI URL">https://doi.org/10.1021/jm400194n</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/jm400194n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fjm400194n%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DPyrimidoaminotropanes%252Bas%252BPotent%25252C%252BSelective%25252C%252Band%252BEfficacious%252BSmall%252BMolecule%252BKinase%252BInhibitors%252Bof%252Bthe%252BMammalian%252BTarget%252Bof%252BRapamycin%252B%252528mTOR%252529%26aulast%3DEstrada%26aufirst%3DAnthony%2BA.%26date%3D2013%26date%3D2013%26date%3D2013%26date%3D06022013%26date%3D29032013%26date%3D11042013%26date%3D08032013%26volume%3D56%26issue%3D7%26spage%3D3090%26epage%3D3101" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Alleyn T.  Plowright</span>, <span class="hlFld-ContribAuthor ">Karolina  Nilsson</span>, <span class="hlFld-ContribAuthor ">Madeleine  Antonsson</span>, <span class="hlFld-ContribAuthor ">Kosrat  Amin</span>, <span class="hlFld-ContribAuthor ">Johan  Broddefalk</span>, <span class="hlFld-ContribAuthor ">Jörgen  Jensen</span>, <span class="hlFld-ContribAuthor ">Anders  Lehmann</span>, <span class="hlFld-ContribAuthor ">Shujuan  Jin</span>, <span class="hlFld-ContribAuthor ">Stephane  St-Onge</span>, <span class="hlFld-ContribAuthor ">Mirosław J.  Tomaszewski</span>, <span class="hlFld-ContribAuthor ">Maxime  Tremblay</span>, <span class="hlFld-ContribAuthor ">Christopher  Walpole</span>, <span class="hlFld-ContribAuthor ">Zhongyong  Wei</span>, <span class="hlFld-ContribAuthor ">Hua  Yang</span>, and <span class="hlFld-ContribAuthor ">Johan  Ulander</span>  . </span><span class="cited-content_cbyCitation_article-title">Discovery of Agonists of Cannabinoid Receptor 1 with Restricted Central Nervous System Penetration Aimed for Treatment of Gastroesophageal Reflux Disease. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2013,</strong> <em>56 </em>
                                    (1)
                                     , 220-240. <a href="https://doi.org/10.1021/jm301511h" title="DOI URL">https://doi.org/10.1021/jm301511h</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/jm301511h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fjm301511h%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252BAgonists%252Bof%252BCannabinoid%252BReceptor%252B1%252Bwith%252BRestricted%252BCentral%252BNervous%252BSystem%252BPenetration%252BAimed%252Bfor%252BTreatment%252Bof%252BGastroesophageal%252BReflux%252BDisease%26aulast%3DPlowright%26aufirst%3DAlleyn%2BT.%26date%3D2013%26date%3D2012%26date%3D2012%26date%3D17102012%26date%3D24122012%26date%3D10012013%26date%3D10122012%26volume%3D56%26issue%3D1%26spage%3D220%26epage%3D240" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Anthony A.  Estrada</span>, <span class="hlFld-ContribAuthor ">Xingrong  Liu</span>, <span class="hlFld-ContribAuthor ">Charles  Baker-Glenn</span>, <span class="hlFld-ContribAuthor ">Alan  Beresford</span>, <span class="hlFld-ContribAuthor ">Daniel J.  Burdick</span>, <span class="hlFld-ContribAuthor ">Mark  Chambers</span>, <span class="hlFld-ContribAuthor ">Bryan K.  Chan</span>, <span class="hlFld-ContribAuthor ">Huifen  Chen</span>, <span class="hlFld-ContribAuthor ">Xiao  Ding</span>, <span class="hlFld-ContribAuthor ">Antonio G.  DiPasquale</span>, <span class="hlFld-ContribAuthor ">Sara L.  Dominguez</span>, <span class="hlFld-ContribAuthor ">Jennafer  Dotson</span>, <span class="hlFld-ContribAuthor ">Jason  Drummond</span>, <span class="hlFld-ContribAuthor ">Michael  Flagella</span>, <span class="hlFld-ContribAuthor ">Sean  Flynn</span>, <span class="hlFld-ContribAuthor ">Reina  Fuji</span>, <span class="hlFld-ContribAuthor ">Andrew  Gill</span>, <span class="hlFld-ContribAuthor ">Janet  Gunzner-Toste</span>, <span class="hlFld-ContribAuthor ">Seth F.  Harris</span>, <span class="hlFld-ContribAuthor ">Timothy P.  Heffron</span>, <span class="hlFld-ContribAuthor ">Tracy  Kleinheinz</span>, <span class="hlFld-ContribAuthor ">Donna W.  Lee</span>, <span class="hlFld-ContribAuthor ">Claire E.  Le Pichon</span>, <span class="hlFld-ContribAuthor ">Joseph P.  Lyssikatos</span>, <span class="hlFld-ContribAuthor ">Andrew D.  Medhurst</span>, <span class="hlFld-ContribAuthor ">John G.  Moffat</span>, <span class="hlFld-ContribAuthor ">Susmith  Mukund</span>, <span class="hlFld-ContribAuthor ">Kevin  Nash</span>, <span class="hlFld-ContribAuthor ">Kimberly  Scearce-Levie</span>, <span class="hlFld-ContribAuthor ">Zejuan  Sheng</span>, <span class="hlFld-ContribAuthor ">Daniel G.  Shore</span>, <span class="hlFld-ContribAuthor ">Thuy  Tran</span>, <span class="hlFld-ContribAuthor ">Naimisha  Trivedi</span>, <span class="hlFld-ContribAuthor ">Shumei  Wang</span>, <span class="hlFld-ContribAuthor ">Shuo  Zhang</span>, <span class="hlFld-ContribAuthor ">Xiaolin  Zhang</span>, <span class="hlFld-ContribAuthor ">Guiling  Zhao</span>, <span class="hlFld-ContribAuthor ">Haitao  Zhu</span>, and <span class="hlFld-ContribAuthor ">Zachary K.  Sweeney</span>  . </span><span class="cited-content_cbyCitation_article-title">Discovery of Highly Potent, Selective, and Brain-Penetrable Leucine-Rich Repeat Kinase 2 (LRRK2) Small Molecule Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2012,</strong> <em>55 </em>
                                    (22)
                                     , 9416-9433. <a href="https://doi.org/10.1021/jm301020q" title="DOI URL">https://doi.org/10.1021/jm301020q</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/jm301020q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fjm301020q%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252BHighly%252BPotent%25252C%252BSelective%25252C%252Band%252BBrain-Penetrable%252BLeucine-Rich%252BRepeat%252BKinase%252B2%252B%252528LRRK2%252529%252BSmall%252BMolecule%252BInhibitors%26aulast%3DEstrada%26aufirst%3DAnthony%2BA.%26date%3D2012%26date%3D2012%26date%3D2012%26date%3D12072012%26date%3D15102012%26date%3D26112012%26date%3D17092012%26volume%3D55%26issue%3D22%26spage%3D9416%26epage%3D9433" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">James J.  Crawford</span>, <span class="hlFld-ContribAuthor ">Peter W.  Kenny</span>, <span class="hlFld-ContribAuthor ">Jonathan  Bowyer</span>, <span class="hlFld-ContribAuthor ">Calum R.  Cook</span>, <span class="hlFld-ContribAuthor ">Jonathan E.  Finlayson</span>, <span class="hlFld-ContribAuthor ">Christine  Heyes</span>, <span class="hlFld-ContribAuthor ">Adrian J.  Highton</span>, <span class="hlFld-ContribAuthor ">Julian A.  Hudson</span>, <span class="hlFld-ContribAuthor ">Anja  Jestel</span>, <span class="hlFld-ContribAuthor ">Stephan  Krapp</span>, <span class="hlFld-ContribAuthor ">Scott  Martin</span>, <span class="hlFld-ContribAuthor ">Philip A.  MacFaul</span>, <span class="hlFld-ContribAuthor ">Benjamin P.  McDermott</span>, <span class="hlFld-ContribAuthor ">Thomas M.  McGuire</span>, <span class="hlFld-ContribAuthor ">Andrew D.  Morley</span>, <span class="hlFld-ContribAuthor ">Jeffrey J.  Morris</span>, <span class="hlFld-ContribAuthor ">Ken M.  Page</span>, <span class="hlFld-ContribAuthor ">Lyn Rosenbrier  Ribeiro</span>, <span class="hlFld-ContribAuthor ">Helen  Sawney</span>, <span class="hlFld-ContribAuthor ">Stefan  Steinbacher</span>, <span class="hlFld-ContribAuthor ">Caroline  Smith</span>, and <span class="hlFld-ContribAuthor ">Alexander G.  Dossetter</span>  . </span><span class="cited-content_cbyCitation_article-title">Pharmacokinetic Benefits of 3,4-Dimethoxy Substitution of a Phenyl Ring and Design of Isosteres Yielding Orally Available Cathepsin K Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2012,</strong> <em>55 </em>
                                    (20)
                                     , 8827-8837. <a href="https://doi.org/10.1021/jm301119s" title="DOI URL">https://doi.org/10.1021/jm301119s</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/jm301119s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fjm301119s%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DPharmacokinetic%252BBenefits%252Bof%252B3%25252C4-Dimethoxy%252BSubstitution%252Bof%252Ba%252BPhenyl%252BRing%252Band%252BDesign%252Bof%252BIsosteres%252BYielding%252BOrally%252BAvailable%252BCathepsin%252BK%252BInhibitors%26aulast%3DCrawford%26aufirst%3DJames%2BJ.%26date%3D2012%26date%3D2012%26date%3D2012%26date%3D31072012%26date%3D11102012%26date%3D25102012%26date%3D17092012%26volume%3D55%26issue%3D20%26spage%3D8827%26epage%3D8837" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Huifen  Chen</span>, <span class="hlFld-ContribAuthor ">Bryan K.  Chan</span>, <span class="hlFld-ContribAuthor ">Jason  Drummond</span>, <span class="hlFld-ContribAuthor ">Anthony A.  Estrada</span>, <span class="hlFld-ContribAuthor ">Janet  Gunzner-Toste</span>, <span class="hlFld-ContribAuthor ">Xingrong  Liu</span>, <span class="hlFld-ContribAuthor ">Yichin  Liu</span>, <span class="hlFld-ContribAuthor ">John  Moffat</span>, <span class="hlFld-ContribAuthor ">Daniel  Shore</span>, <span class="hlFld-ContribAuthor ">Zachary K.  Sweeney</span>, <span class="hlFld-ContribAuthor ">Thuy  Tran</span>, <span class="hlFld-ContribAuthor ">Shumei  Wang</span>, <span class="hlFld-ContribAuthor ">Guiling  Zhao</span>, <span class="hlFld-ContribAuthor ">Haitao  Zhu</span>, and <span class="hlFld-ContribAuthor ">Daniel J.  Burdick</span>  . </span><span class="cited-content_cbyCitation_article-title">Discovery of Selective LRRK2 Inhibitors Guided by Computational Analysis and Molecular Modeling. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2012,</strong> <em>55 </em>
                                    (11)
                                     , 5536-5545. <a href="https://doi.org/10.1021/jm300452p" title="DOI URL">https://doi.org/10.1021/jm300452p</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/jm300452p&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fjm300452p%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252BSelective%252BLRRK2%252BInhibitors%252BGuided%252Bby%252BComputational%252BAnalysis%252Band%252BMolecular%252BModeling%26aulast%3DChen%26aufirst%3DHuifen%26date%3D2012%26date%3D2012%26date%3D2012%26date%3D31032012%26date%3D01062012%26date%3D14062012%26date%3D16052012%26volume%3D55%26issue%3D11%26spage%3D5536%26epage%3D5545" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jonas  Boström</span>, <span class="hlFld-ContribAuthor ">Anders  Hogner</span>, <span class="hlFld-ContribAuthor ">Antonio  Llinàs</span>, <span class="hlFld-ContribAuthor ">Eric  Wellner</span>, and <span class="hlFld-ContribAuthor ">Alleyn T.  Plowright</span>  . </span><span class="cited-content_cbyCitation_article-title">Oxadiazoles in Medicinal Chemistry. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2012,</strong> <em>55 </em>
                                    (5)
                                     , 1817-1830. <a href="https://doi.org/10.1021/jm2013248" title="DOI URL">https://doi.org/10.1021/jm2013248</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/jm2013248&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fjm2013248%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DOxadiazoles%252Bin%252BMedicinal%252BChemistry%26aulast%3DBostr%25C3%25B6m%26aufirst%3DJonas%26date%3D2012%26date%3D2012%26date%3D2011%26date%3D04102011%26date%3D13012012%26date%3D08032012%26date%3D19122011%26volume%3D55%26issue%3D5%26spage%3D1817%26epage%3D1830" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Klaus  Müller</span>  . </span><span class="cited-content_cbyCitation_article-title">The Power of MMPA and a Teaching Lesson in Medicinal Chemistry. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2012,</strong> <em>55 </em>
                                    (5)
                                     , 1815-1816. <a href="https://doi.org/10.1021/jm300163y" title="DOI URL">https://doi.org/10.1021/jm300163y</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/jm300163y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fjm300163y%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DThe%252BPower%252Bof%252BMMPA%252Band%252Ba%252BTeaching%252BLesson%252Bin%252BMedicinal%252BChemistry%26aulast%3DM%25C3%25BCller%26aufirst%3DKlaus%26date%3D2012%26date%3D2012%26date%3D2012%26date%3D06022012%26date%3D15022012%26date%3D08032012%26volume%3D55%26issue%3D5%26spage%3D1815%26epage%3D1816" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jiazhen  He</span>, <span class="hlFld-ContribAuthor ">Huifang  You</span>, <span class="hlFld-ContribAuthor ">Emil  Sandström</span>, <span class="hlFld-ContribAuthor ">Eva  Nittinger</span>, <span class="hlFld-ContribAuthor ">Esben Jannik  Bjerrum</span>, <span class="hlFld-ContribAuthor ">Christian  Tyrchan</span>, <span class="hlFld-ContribAuthor ">Werngard  Czechtizky</span>, <span class="hlFld-ContribAuthor ">Ola  Engkvist</span>. </span><span class="cited-content_cbyCitation_article-title">Molecular optimization by capturing chemist’s intuition using deep neural networks. </span><span class="cited-content_cbyCitation_journal-name">Journal of Cheminformatics</span><span> <strong>2021,</strong> <em>13 </em>
                                    (1)
                                     <a href="https://doi.org/10.1186/s13321-021-00497-0" title="DOI URL">https://doi.org/10.1186/s13321-021-00497-0</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1186/s13321-021-00497-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1186%2Fs13321-021-00497-0%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Cheminformatics%26atitle%3DMolecular%252Boptimization%252Bby%252Bcapturing%252Bchemist%2525E2%252580%252599s%252Bintuition%252Busing%252Bdeep%252Bneural%252Bnetworks%26aulast%3DHe%26aufirst%3DJiazhen%26date%3D2021%26date%3D2021%26volume%3D13%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Candida  Manelfi</span>, <span class="hlFld-ContribAuthor ">Marica  Gemei</span>, <span class="hlFld-ContribAuthor ">Carmine  Talarico</span>, <span class="hlFld-ContribAuthor ">Carmen  Cerchia</span>, <span class="hlFld-ContribAuthor ">Anna  Fava</span>, <span class="hlFld-ContribAuthor ">Filippo  Lunghini</span>, <span class="hlFld-ContribAuthor ">Andrea Rosario  Beccari</span>. </span><span class="cited-content_cbyCitation_article-title">“Molecular Anatomy”: a new multi-dimensional hierarchical scaffold analysis tool. </span><span class="cited-content_cbyCitation_journal-name">Journal of Cheminformatics</span><span> <strong>2021,</strong> <em>13 </em>
                                    (1)
                                     <a href="https://doi.org/10.1186/s13321-021-00526-y" title="DOI URL">https://doi.org/10.1186/s13321-021-00526-y</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1186/s13321-021-00526-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1186%2Fs13321-021-00526-y%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Cheminformatics%26atitle%3D%2525E2%252580%25259CMolecular%252BAnatomy%2525E2%252580%25259D%25253A%252Ba%252Bnew%252Bmulti-dimensional%252Bhierarchical%252Bscaffold%252Banalysis%252Btool%26aulast%3DManelfi%26aufirst%3DCandida%26date%3D2021%26date%3D2021%26volume%3D13%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">O.V.  Tinkov</span>, <span class="hlFld-ContribAuthor ">V.Y.  Grigorev</span>, <span class="hlFld-ContribAuthor ">L.D.  Grigoreva</span>. </span><span class="cited-content_cbyCitation_article-title">QSAR analysis of the acute toxicity of avermectins towards
              Tetrahymena pyriformis. </span><span class="cited-content_cbyCitation_journal-name">SAR and QSAR in Environmental Research</span><span> <strong>2021,</strong> <em>32 </em>
                                    (7)
                                     , 541-571. <a href="https://doi.org/10.1080/1062936X.2021.1932583" title="DOI URL">https://doi.org/10.1080/1062936X.2021.1932583</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/1062936X.2021.1932583&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F1062936X.2021.1932583%26sid%3Dliteratum%253Aachs%26jtitle%3DSAR%2520and%2520QSAR%2520in%2520Environmental%2520Research%26atitle%3DQSAR%252Banalysis%252Bof%252Bthe%252Bacute%252Btoxicity%252Bof%252Bavermectins%252Btowards%252BTetrahymena%252Bpyriformis%26aulast%3DTinkov%26aufirst%3DO.V.%26date%3D2021%26date%3D2021%26volume%3D32%26issue%3D7%26spage%3D541%26epage%3D571" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Bonggun  Shin</span>, <span class="hlFld-ContribAuthor ">Sungsoo  Park</span>, <span class="hlFld-ContribAuthor ">JinYeong  Bak</span>, <span class="hlFld-ContribAuthor ">Joyce C.  Ho</span>. </span><span class="cited-content_cbyCitation_article-title">Controlled molecule generator for optimizing multiple chemical properties. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2021,</strong>,, 146-153. <a href="https://doi.org/10.1145/3450439.3451879" title="DOI URL">https://doi.org/10.1145/3450439.3451879</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1145/3450439.3451879&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1145%2F3450439.3451879%26sid%3Dliteratum%253Aachs%26atitle%3DControlled%252Bmolecule%252Bgenerator%252Bfor%252Boptimizing%252Bmultiple%252Bchemical%252Bproperties%26aulast%3DShin%26aufirst%3DBonggun%26date%3D2021%26date%3D2021%26spage%3D146%26epage%3D153%26pub%3DACM%26atitle%3DProceedings%252Bof%252Bthe%252BConference%252Bon%252BHealth%25252C%252BInference%25252C%252Band%252BLearning%26aulast%3DGhassemi%26aufirst%3DMarzyeh%26date%3D2021%26volume%3D58" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">José  Jiménez-Luna</span>, <span class="hlFld-ContribAuthor ">Francesca  Grisoni</span>, <span class="hlFld-ContribAuthor ">Nils  Weskamp</span>, <span class="hlFld-ContribAuthor ">Gisbert  Schneider</span>. </span><span class="cited-content_cbyCitation_article-title">Artificial intelligence in drug discovery: recent advances and future perspectives. </span><span class="cited-content_cbyCitation_journal-name">Expert Opinion on Drug Discovery</span><span> <strong>2021,</strong> <em>3 </em>, 1-11. <a href="https://doi.org/10.1080/17460441.2021.1909567" title="DOI URL">https://doi.org/10.1080/17460441.2021.1909567</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/17460441.2021.1909567&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F17460441.2021.1909567%26sid%3Dliteratum%253Aachs%26jtitle%3DExpert%2520Opinion%2520on%2520Drug%2520Discovery%26atitle%3DArtificial%252Bintelligence%252Bin%252Bdrug%252Bdiscovery%25253A%252Brecent%252Badvances%252Band%252Bfuture%252Bperspectives%26aulast%3DJim%25C3%25A9nez-Luna%26aufirst%3DJos%25C3%25A9%26date%3D2021%26date%3D2021%26volume%3D3%26spage%3D1%26epage%3D11" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Lewis D.  Pennington</span>, <span class="hlFld-ContribAuthor ">Ingo  Muegge</span>. </span><span class="cited-content_cbyCitation_article-title">Holistic Drug Design for Multiparameter Optimization in Modern Small Molecule Drug Discovery. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry Letters</span><span> <strong>2021,</strong> <em>10 </em>, 128003. <a href="https://doi.org/10.1016/j.bmcl.2021.128003" title="DOI URL">https://doi.org/10.1016/j.bmcl.2021.128003</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmcl.2021.128003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmcl.2021.128003%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DHolistic%252BDrug%252BDesign%252Bfor%252BMultiparameter%252BOptimization%252Bin%252BModern%252BSmall%252BMolecule%252BDrug%252BDiscovery%26aulast%3DPennington%26aufirst%3DLewis%2BD.%26date%3D2021%26volume%3D10%26spage%3D128003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Li  Fu</span>, <span class="hlFld-ContribAuthor ">Zi-Yi  Yang</span>, <span class="hlFld-ContribAuthor ">Zhi-Jiang  Yang</span>, <span class="hlFld-ContribAuthor ">Ming-Zhu  Yin</span>, <span class="hlFld-ContribAuthor ">Ai-Ping  Lu</span>, <span class="hlFld-ContribAuthor ">Xiang  Chen</span>, <span class="hlFld-ContribAuthor ">Shao  Liu</span>, <span class="hlFld-ContribAuthor ">Ting-Jun  Hou</span>, <span class="hlFld-ContribAuthor ">Dong-Sheng  Cao</span>. </span><span class="cited-content_cbyCitation_article-title">QSAR-assisted-MMPA to expand chemical transformation space for lead optimization. </span><span class="cited-content_cbyCitation_journal-name">Briefings in Bioinformatics</span><span> <strong>2021,</strong> <em>23 </em><a href="https://doi.org/10.1093/bib/bbaa374" title="DOI URL">https://doi.org/10.1093/bib/bbaa374</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1093/bib/bbaa374&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1093%2Fbib%2Fbbaa374%26sid%3Dliteratum%253Aachs%26jtitle%3DBriefings%2520in%2520Bioinformatics%26atitle%3DQSAR-assisted-MMPA%252Bto%252Bexpand%252Bchemical%252Btransformation%252Bspace%252Bfor%252Blead%252Boptimization%26aulast%3DFu%26aufirst%3DLi%26date%3D2021%26date%3D2021%26volume%3D23" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jérôme  Cassayre</span>, <span class="hlFld-ContribAuthor ">Tomas  Smejkal</span>, <span class="hlFld-ContribAuthor ">Judith  Blythe</span>, <span class="hlFld-ContribAuthor ">Patrik  Hoegger</span>, <span class="hlFld-ContribAuthor ">Peter  Renold</span>, <span class="hlFld-ContribAuthor ">Thomas  Pitterna</span>, <span class="hlFld-ContribAuthor ">C.S.  Prasanna</span>, <span class="hlFld-ContribAuthor ">Helmars  Smits</span>, <span class="hlFld-ContribAuthor ">Edouard  Godineau</span>, <span class="hlFld-ContribAuthor ">Torsten  Luksch</span>, <span class="hlFld-ContribAuthor ">Guillaume  Berthon</span>, <span class="hlFld-ContribAuthor ">Girish  Rawal</span>, <span class="hlFld-ContribAuthor ">Rupesh  Patre</span>, <span class="hlFld-ContribAuthor ">Mukul  Lal</span>, <span class="hlFld-ContribAuthor ">Mohamed  Boussemghoune</span>, <span class="hlFld-ContribAuthor ">Simonetta  Masala</span>, <span class="hlFld-ContribAuthor ">Fabien  Barreteau</span>, <span class="hlFld-ContribAuthor ">Michael  Flaeschel</span>, <span class="hlFld-ContribAuthor ">Juliane  Vogt</span>, <span class="hlFld-ContribAuthor ">Myriem  El Qacemi</span>. </span><span class="cited-content_cbyCitation_article-title">The discovery of isocycloseram: A novel isoxazoline insecticide. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2021,</strong>,, 165-212. <a href="https://doi.org/10.1016/B978-0-12-821035-2.00008-5" title="DOI URL">https://doi.org/10.1016/B978-0-12-821035-2.00008-5</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/B978-0-12-821035-2.00008-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2FB978-0-12-821035-2.00008-5%26sid%3Dliteratum%253Aachs%26atitle%3DThe%252Bdiscovery%252Bof%252Bisocycloseram%25253A%252BA%252Bnovel%252Bisoxazoline%252Binsecticide%26aulast%3DCassayre%26aufirst%3DJ%25C3%25A9r%25C3%25B4me%26date%3D2021%26spage%3D165%26epage%3D212%26pub%3DElsevier%26atitle%3DRecent%252BHighlights%252Bin%252Bthe%252BDiscovery%252Band%252BOptimization%252Bof%252BCrop%252BProtection%252BProducts%26date%3D2021%26volume%3D43" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Dimitar  Yonchev</span>, <span class="hlFld-ContribAuthor ">Jürgen  Bajorath</span>. </span><span class="cited-content_cbyCitation_article-title">DeepCOMO: from structure-activity relationship diagnostics to generative molecular design using the compound optimization monitor methodology. </span><span class="cited-content_cbyCitation_journal-name">Journal of Computer-Aided Molecular Design</span><span> <strong>2020,</strong> <em>34 </em>
                                    (12)
                                     , 1207-1218. <a href="https://doi.org/10.1007/s10822-020-00349-3" title="DOI URL">https://doi.org/10.1007/s10822-020-00349-3</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s10822-020-00349-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs10822-020-00349-3%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Computer-Aided%2520Molecular%2520Design%26atitle%3DDeepCOMO%25253A%252Bfrom%252Bstructure-activity%252Brelationship%252Bdiagnostics%252Bto%252Bgenerative%252Bmolecular%252Bdesign%252Busing%252Bthe%252Bcompound%252Boptimization%252Bmonitor%252Bmethodology%26aulast%3DYonchev%26aufirst%3DDimitar%26date%3D2020%26date%3D2020%26volume%3D34%26issue%3D12%26spage%3D1207%26epage%3D1218" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Vishal B.  Siramshetty</span>, <span class="hlFld-ContribAuthor ">Pranav  Shah</span>, <span class="hlFld-ContribAuthor ">Edward  Kerns</span>, <span class="hlFld-ContribAuthor ">Kimloan  Nguyen</span>, <span class="hlFld-ContribAuthor ">Kyeong Ri  Yu</span>, <span class="hlFld-ContribAuthor ">Md  Kabir</span>, <span class="hlFld-ContribAuthor ">Jordan  Williams</span>, <span class="hlFld-ContribAuthor ">Jorge  Neyra</span>, <span class="hlFld-ContribAuthor ">Noel  Southall</span>, <span class="hlFld-ContribAuthor ">Ðắc-Trung  Nguyễn</span>, <span class="hlFld-ContribAuthor ">Xin  Xu</span>. </span><span class="cited-content_cbyCitation_article-title">Retrospective assessment of rat liver microsomal stability at NCATS: data and QSAR models. </span><span class="cited-content_cbyCitation_journal-name">Scientific Reports</span><span> <strong>2020,</strong> <em>10 </em>
                                    (1)
                                     <a href="https://doi.org/10.1038/s41598-020-77327-0" title="DOI URL">https://doi.org/10.1038/s41598-020-77327-0</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41598-020-77327-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41598-020-77327-0%26sid%3Dliteratum%253Aachs%26jtitle%3DScientific%2520Reports%26atitle%3DRetrospective%252Bassessment%252Bof%252Brat%252Bliver%252Bmicrosomal%252Bstability%252Bat%252BNCATS%25253A%252Bdata%252Band%252BQSAR%252Bmodels%26aulast%3DSiramshetty%26aufirst%3DVishal%2BB.%26date%3D2020%26date%3D2020%26volume%3D10%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Myungwon  Seo</span>, <span class="hlFld-ContribAuthor ">Hyun Kil  Shin</span>, <span class="hlFld-ContribAuthor ">Yoochan  Myung</span>, <span class="hlFld-ContribAuthor ">Sungbo  Hwang</span>, <span class="hlFld-ContribAuthor ">Kyoung Tai  No</span>. </span><span class="cited-content_cbyCitation_article-title">Development of Natural Compound Molecular Fingerprint (NC-MFP) with the Dictionary of Natural Products (DNP) for natural product-based drug development. </span><span class="cited-content_cbyCitation_journal-name">Journal of Cheminformatics</span><span> <strong>2020,</strong> <em>12 </em>
                                    (1)
                                     <a href="https://doi.org/10.1186/s13321-020-0410-3" title="DOI URL">https://doi.org/10.1186/s13321-020-0410-3</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1186/s13321-020-0410-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1186%2Fs13321-020-0410-3%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Cheminformatics%26atitle%3DDevelopment%252Bof%252BNatural%252BCompound%252BMolecular%252BFingerprint%252B%252528NC-MFP%252529%252Bwith%252Bthe%252BDictionary%252Bof%252BNatural%252BProducts%252B%252528DNP%252529%252Bfor%252Bnatural%252Bproduct-based%252Bdrug%252Bdevelopment%26aulast%3DSeo%26aufirst%3DMyungwon%26date%3D2020%26date%3D2020%26volume%3D12%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Pranav  Shah</span>, <span class="hlFld-ContribAuthor ">Vishal B.  Siramshetty</span>, <span class="hlFld-ContribAuthor ">Alexey V.  Zakharov</span>, <span class="hlFld-ContribAuthor ">Noel T.  Southall</span>, <span class="hlFld-ContribAuthor ">Xin  Xu</span>, <span class="hlFld-ContribAuthor ">Dac-Trung  Nguyen</span>. </span><span class="cited-content_cbyCitation_article-title">Predicting liver cytosol stability of small molecules. </span><span class="cited-content_cbyCitation_journal-name">Journal of Cheminformatics</span><span> <strong>2020,</strong> <em>12 </em>
                                    (1)
                                     <a href="https://doi.org/10.1186/s13321-020-00426-7" title="DOI URL">https://doi.org/10.1186/s13321-020-00426-7</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1186/s13321-020-00426-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1186%2Fs13321-020-00426-7%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Cheminformatics%26atitle%3DPredicting%252Bliver%252Bcytosol%252Bstability%252Bof%252Bsmall%252Bmolecules%26aulast%3DShah%26aufirst%3DPranav%26date%3D2020%26date%3D2020%26volume%3D12%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Shoaib  Manzoor</span>, <span class="hlFld-ContribAuthor ">Nasimul  Hoda</span>. </span><span class="cited-content_cbyCitation_article-title">A comprehensive review of monoamine oxidase inhibitors as Anti-Alzheimer’s disease agents: A review. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>206 </em>, 112787. <a href="https://doi.org/10.1016/j.ejmech.2020.112787" title="DOI URL">https://doi.org/10.1016/j.ejmech.2020.112787</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2020.112787&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2020.112787%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DA%252Bcomprehensive%252Breview%252Bof%252Bmonoamine%252Boxidase%252Binhibitors%252Bas%252BAnti-Alzheimer%2525E2%252580%252599s%252Bdisease%252Bagents%25253A%252BA%252Breview%26aulast%3DManzoor%26aufirst%3DShoaib%26date%3D2020%26volume%3D206%26spage%3D112787" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yue  Zhao</span>, <span class="hlFld-ContribAuthor ">Liu-Xia  Zhang</span>, <span class="hlFld-ContribAuthor ">Ting  Jiang</span>, <span class="hlFld-ContribAuthor ">Jing  Long</span>, <span class="hlFld-ContribAuthor ">Zhong-Ye  Ma</span>, <span class="hlFld-ContribAuthor ">Ai-Ping  Lu</span>, <span class="hlFld-ContribAuthor ">Yan  Cheng</span>, <span class="hlFld-ContribAuthor ">Dong-Sheng  Cao</span>. </span><span class="cited-content_cbyCitation_article-title">The ups and downs of Poly(ADP-ribose) Polymerase-1 inhibitors in cancer therapy–Current progress and future direction. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>203 </em>, 112570. <a href="https://doi.org/10.1016/j.ejmech.2020.112570" title="DOI URL">https://doi.org/10.1016/j.ejmech.2020.112570</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2020.112570&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2020.112570%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DThe%252Bups%252Band%252Bdowns%252Bof%252BPoly%252528ADP-ribose%252529%252BPolymerase-1%252Binhibitors%252Bin%252Bcancer%252Btherapy%2525E2%252580%252593Current%252Bprogress%252Band%252Bfuture%252Bdirection%26aulast%3DZhao%26aufirst%3DYue%26date%3D2020%26volume%3D203%26spage%3D112570" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jinwen  Shan</span>, <span class="hlFld-ContribAuthor ">Changge  Ji</span>. </span><span class="cited-content_cbyCitation_article-title">MolOpt: A Web Server for Drug Design using Bioisosteric Transformation. </span><span class="cited-content_cbyCitation_journal-name">Current Computer-Aided Drug Design</span><span> <strong>2020,</strong> <em>16 </em>
                                    (4)
                                     , 460-466. <a href="https://doi.org/10.2174/1573409915666190704093400" title="DOI URL">https://doi.org/10.2174/1573409915666190704093400</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.2174/1573409915666190704093400&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.2174%2F1573409915666190704093400%26sid%3Dliteratum%253Aachs%26jtitle%3DCurrent%2520Computer-Aided%2520Drug%2520Design%26atitle%3DMolOpt%25253A%252BA%252BWeb%252BServer%252Bfor%252BDrug%252BDesign%252Busing%252BBioisosteric%252BTransformation%26aulast%3DShan%26aufirst%3DJinwen%26date%3D2020%26volume%3D16%26issue%3D4%26spage%3D460%26epage%3D466" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Thiago Kelvin Brito  Matos</span>, <span class="hlFld-ContribAuthor ">Pedro Henrique Jatai  Batista</span>, <span class="hlFld-ContribAuthor ">Fernanda  dos Reis Rocho</span>, <span class="hlFld-ContribAuthor ">Daniela  de Vita</span>, <span class="hlFld-ContribAuthor ">Nicholas  Pearce</span>, <span class="hlFld-ContribAuthor ">Barrie  Kellam</span>, <span class="hlFld-ContribAuthor ">Carlos Alberto  Montanari</span>, <span class="hlFld-ContribAuthor ">Andrei  Leitão</span>. </span><span class="cited-content_cbyCitation_article-title">Synthesis and matched molecular pair analysis of covalent reversible inhibitors of the cysteine protease CPB. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry Letters</span><span> <strong>2020,</strong> <em>30 </em>
                                    (18)
                                     , 127439. <a href="https://doi.org/10.1016/j.bmcl.2020.127439" title="DOI URL">https://doi.org/10.1016/j.bmcl.2020.127439</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmcl.2020.127439&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmcl.2020.127439%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DSynthesis%252Band%252Bmatched%252Bmolecular%252Bpair%252Banalysis%252Bof%252Bcovalent%252Breversible%252Binhibitors%252Bof%252Bthe%252Bcysteine%252Bprotease%252BCPB%26aulast%3DMatos%26aufirst%3DThiago%2BKelvin%2BBrito%26date%3D2020%26volume%3D30%26issue%3D18%26spage%3D127439" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Matthew P.  Baumgartner</span>, <span class="hlFld-ContribAuthor ">David A.  Evans</span>. </span><span class="cited-content_cbyCitation_article-title">Side chain virtual screening of matched molecular pairs: a PDB-wide and ChEMBL-wide analysis. </span><span class="cited-content_cbyCitation_journal-name">Journal of Computer-Aided Molecular Design</span><span> <strong>2020,</strong> <em>34 </em>
                                    (9)
                                     , 953-963. <a href="https://doi.org/10.1007/s10822-020-00313-1" title="DOI URL">https://doi.org/10.1007/s10822-020-00313-1</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s10822-020-00313-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs10822-020-00313-1%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Computer-Aided%2520Molecular%2520Design%26atitle%3DSide%252Bchain%252Bvirtual%252Bscreening%252Bof%252Bmatched%252Bmolecular%252Bpairs%25253A%252Ba%252BPDB-wide%252Band%252BChEMBL-wide%252Banalysis%26aulast%3DBaumgartner%26aufirst%3DMatthew%2BP.%26date%3D2020%26date%3D2020%26volume%3D34%26issue%3D9%26spage%3D953%26epage%3D963" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Huabin  Hu</span>, <span class="hlFld-ContribAuthor ">Jürgen  Bajorath</span>. </span><span class="cited-content_cbyCitation_article-title">Evidence for the presence of core structure-dependent activity cliffs. </span><span class="cited-content_cbyCitation_journal-name">Future Medicinal Chemistry</span><span> <strong>2020,</strong> <em>12 </em>
                                    (16)
                                     , 1451-1455. <a href="https://doi.org/10.4155/fmc-2020-0157" title="DOI URL">https://doi.org/10.4155/fmc-2020-0157</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.4155/fmc-2020-0157&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.4155%2Ffmc-2020-0157%26sid%3Dliteratum%253Aachs%26jtitle%3DFuture%2520Medicinal%2520Chemistry%26atitle%3DEvidence%252Bfor%252Bthe%252Bpresence%252Bof%252Bcore%252Bstructure-dependent%252Bactivity%252Bcliffs%26aulast%3DHu%26aufirst%3DHuabin%26date%3D2020%26volume%3D12%26issue%3D16%26spage%3D1451%26epage%3D1455" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Darren V. S.  Green</span>, <span class="hlFld-ContribAuthor ">Stephen  Pickett</span>, <span class="hlFld-ContribAuthor ">Chris  Luscombe</span>, <span class="hlFld-ContribAuthor ">Stefan  Senger</span>, <span class="hlFld-ContribAuthor ">David  Marcus</span>, <span class="hlFld-ContribAuthor ">Jamel  Meslamani</span>, <span class="hlFld-ContribAuthor ">David  Brett</span>, <span class="hlFld-ContribAuthor ">Adam  Powell</span>, <span class="hlFld-ContribAuthor ">Jonathan  Masson</span>. </span><span class="cited-content_cbyCitation_article-title">BRADSHAW: a system for automated molecular design. </span><span class="cited-content_cbyCitation_journal-name">Journal of Computer-Aided Molecular Design</span><span> <strong>2020,</strong> <em>34 </em>
                                    (7)
                                     , 747-765. <a href="https://doi.org/10.1007/s10822-019-00234-8" title="DOI URL">https://doi.org/10.1007/s10822-019-00234-8</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s10822-019-00234-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs10822-019-00234-8%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Computer-Aided%2520Molecular%2520Design%26atitle%3DBRADSHAW%25253A%252Ba%252Bsystem%252Bfor%252Bautomated%252Bmolecular%252Bdesign%26aulast%3DGreen%26aufirst%3DDarren%2BV.%2BS.%26date%3D2020%26date%3D2019%26volume%3D34%26issue%3D7%26spage%3D747%26epage%3D765" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Eugene N.  Muratov</span>, <span class="hlFld-ContribAuthor ">Jürgen  Bajorath</span>, <span class="hlFld-ContribAuthor ">Robert P.  Sheridan</span>, <span class="hlFld-ContribAuthor ">Igor V.  Tetko</span>, <span class="hlFld-ContribAuthor ">Dmitry  Filimonov</span>, <span class="hlFld-ContribAuthor ">Vladimir  Poroikov</span>, <span class="hlFld-ContribAuthor ">Tudor I.  Oprea</span>, <span class="hlFld-ContribAuthor ">Igor I.  Baskin</span>, <span class="hlFld-ContribAuthor ">Alexandre  Varnek</span>, <span class="hlFld-ContribAuthor ">Adrian  Roitberg</span>, <span class="hlFld-ContribAuthor ">Olexandr  Isayev</span>, <span class="hlFld-ContribAuthor ">Stefano  Curtalolo</span>, <span class="hlFld-ContribAuthor ">Denis  Fourches</span>, <span class="hlFld-ContribAuthor ">Yoram  Cohen</span>, <span class="hlFld-ContribAuthor ">Alan  Aspuru-Guzik</span>, <span class="hlFld-ContribAuthor ">David A.  Winkler</span>, <span class="hlFld-ContribAuthor ">Dimitris  Agrafiotis</span>, <span class="hlFld-ContribAuthor ">Artem  Cherkasov</span>, <span class="hlFld-ContribAuthor ">Alexander  Tropsha</span>. </span><span class="cited-content_cbyCitation_article-title">QSAR without borders. </span><span class="cited-content_cbyCitation_journal-name">Chemical Society Reviews</span><span> <strong>2020,</strong> <em>49 </em>
                                    (11)
                                     , 3525-3564. <a href="https://doi.org/10.1039/D0CS00098A" title="DOI URL">https://doi.org/10.1039/D0CS00098A</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/D0CS00098A&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FD0CS00098A%26sid%3Dliteratum%253Aachs%26jtitle%3DChemical%2520Society%2520Reviews%26atitle%3DQSAR%252Bwithout%252Bborders%26aulast%3DMuratov%26aufirst%3DEugene%2BN.%26date%3D2020%26date%3D2020%26volume%3D49%26issue%3D11%26spage%3D3525%26epage%3D3564" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Petra  Schneider</span>, <span class="hlFld-ContribAuthor ">W. Patrick  Walters</span>, <span class="hlFld-ContribAuthor ">Alleyn T.  Plowright</span>, <span class="hlFld-ContribAuthor ">Norman  Sieroka</span>, <span class="hlFld-ContribAuthor ">Jennifer  Listgarten</span>, <span class="hlFld-ContribAuthor ">Robert A.  Goodnow</span>, <span class="hlFld-ContribAuthor ">Jasmin  Fisher</span>, <span class="hlFld-ContribAuthor ">Johanna M.  Jansen</span>, <span class="hlFld-ContribAuthor ">José S.  Duca</span>, <span class="hlFld-ContribAuthor ">Thomas S.  Rush</span>, <span class="hlFld-ContribAuthor ">Matthias  Zentgraf</span>, <span class="hlFld-ContribAuthor ">John Edward  Hill</span>, <span class="hlFld-ContribAuthor ">Elizabeth  Krutoholow</span>, <span class="hlFld-ContribAuthor ">Matthias  Kohler</span>, <span class="hlFld-ContribAuthor ">Jeff  Blaney</span>, <span class="hlFld-ContribAuthor ">Kimito  Funatsu</span>, <span class="hlFld-ContribAuthor ">Chris  Luebkemann</span>, <span class="hlFld-ContribAuthor ">Gisbert  Schneider</span>. </span><span class="cited-content_cbyCitation_article-title">Rethinking drug design in the artificial intelligence era. </span><span class="cited-content_cbyCitation_journal-name">Nature Reviews Drug Discovery</span><span> <strong>2020,</strong> <em>19 </em>
                                    (5)
                                     , 353-364. <a href="https://doi.org/10.1038/s41573-019-0050-3" title="DOI URL">https://doi.org/10.1038/s41573-019-0050-3</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41573-019-0050-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41573-019-0050-3%26sid%3Dliteratum%253Aachs%26jtitle%3DNature%2520Reviews%2520Drug%2520Discovery%26atitle%3DRethinking%252Bdrug%252Bdesign%252Bin%252Bthe%252Bartificial%252Bintelligence%252Bera%26aulast%3DSchneider%26aufirst%3DPetra%26date%3D2020%26date%3D2019%26volume%3D19%26issue%3D5%26spage%3D353%26epage%3D364" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Philippe  Gantzer</span>, <span class="hlFld-ContribAuthor ">Benoit  Creton</span>, <span class="hlFld-ContribAuthor ">Carlos  Nieto‐Draghi</span>. </span><span class="cited-content_cbyCitation_article-title">Inverse‐QSPR for
              de novo
              Design: A Review. </span><span class="cited-content_cbyCitation_journal-name">Molecular Informatics</span><span> <strong>2020,</strong> <em>39 </em>
                                    (4)
                                     , 1900087. <a href="https://doi.org/10.1002/minf.201900087" title="DOI URL">https://doi.org/10.1002/minf.201900087</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/minf.201900087&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fminf.201900087%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecular%2520Informatics%26atitle%3DInverse%2525E2%252580%252590QSPR%252Bfor%252Bde%252Bnovo%252BDesign%25253A%252BA%252BReview%26aulast%3DGantzer%26aufirst%3DPhilippe%26date%3D2020%26volume%3D39%26issue%3D4%26spage%3D1900087" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Atsushi  Yoshimori</span>, <span class="hlFld-ContribAuthor ">Jürgen  Bajorath</span>. </span><span class="cited-content_cbyCitation_article-title">Deep SAR matrix: SAR matrix expansion for advanced analog design using deep learning architectures. </span><span class="cited-content_cbyCitation_journal-name">Future Drug Discovery</span><span> <strong>2020,</strong> <em>2 </em>
                                    (2)
                                     , FDD36. <a href="https://doi.org/10.4155/fdd-2020-0005" title="DOI URL">https://doi.org/10.4155/fdd-2020-0005</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.4155/fdd-2020-0005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.4155%2Ffdd-2020-0005%26sid%3Dliteratum%253Aachs%26jtitle%3DFuture%2520Drug%2520Discovery%26atitle%3DDeep%252BSAR%252Bmatrix%25253A%252BSAR%252Bmatrix%252Bexpansion%252Bfor%252Badvanced%252Banalog%252Bdesign%252Busing%252Bdeep%252Blearning%252Barchitectures%26aulast%3DYoshimori%26aufirst%3DAtsushi%26date%3D2020%26volume%3D2%26issue%3D2%26spage%3DFDD36" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Dimitar  Yonchev</span>, <span class="hlFld-ContribAuthor ">Jürgen  Bajorath</span>. </span><span class="cited-content_cbyCitation_article-title">Integrating computational lead optimization diagnostics with analog design and candidate selection. </span><span class="cited-content_cbyCitation_journal-name">Future Science OA</span><span> <strong>2020,</strong> <em>6 </em>
                                    (3)
                                     , FSO451. <a href="https://doi.org/10.2144/fsoa-2019-0131" title="DOI URL">https://doi.org/10.2144/fsoa-2019-0131</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.2144/fsoa-2019-0131&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.2144%2Ffsoa-2019-0131%26sid%3Dliteratum%253Aachs%26jtitle%3DFuture%2520Science%2520OA%26atitle%3DIntegrating%252Bcomputational%252Blead%252Boptimization%252Bdiagnostics%252Bwith%252Banalog%252Bdesign%252Band%252Bcandidate%252Bselection%26aulast%3DYonchev%26aufirst%3DDimitar%26date%3D2020%26volume%3D6%26issue%3D3%26spage%3DFSO451" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Dagmar  Stumpfe</span>, <span class="hlFld-ContribAuthor ">Huabin  Hu</span>, <span class="hlFld-ContribAuthor ">Jürgen  Bajorath</span>. </span><span class="cited-content_cbyCitation_article-title">Computational method for the identification of third generation activity cliffs. </span><span class="cited-content_cbyCitation_journal-name">MethodsX</span><span> <strong>2020,</strong> <em>7 </em>, 100793. <a href="https://doi.org/10.1016/j.mex.2020.100793" title="DOI URL">https://doi.org/10.1016/j.mex.2020.100793</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.mex.2020.100793&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.mex.2020.100793%26sid%3Dliteratum%253Aachs%26jtitle%3DMethodsX%26atitle%3DComputational%252Bmethod%252Bfor%252Bthe%252Bidentification%252Bof%252Bthird%252Bgeneration%252Bactivity%252Bcliffs%26aulast%3DStumpfe%26aufirst%3DDagmar%26date%3D2020%26volume%3D7%26spage%3D100793" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Lu  Liang</span>, <span class="hlFld-ContribAuthor ">Chunfeng  Ma</span>, <span class="hlFld-ContribAuthor ">Tengfei  Du</span>, <span class="hlFld-ContribAuthor ">Yufei  Zhao</span>, <span class="hlFld-ContribAuthor ">Xiaoyong  Zhao</span>, <span class="hlFld-ContribAuthor ">Mengmeng  Liu</span>, <span class="hlFld-ContribAuthor ">Zhonghua  Wang</span>, <span class="hlFld-ContribAuthor ">Jianping  Lin</span>. </span><span class="cited-content_cbyCitation_article-title">Bioactivity-explorer: a web application for interactive visualization and exploration of bioactivity data. </span><span class="cited-content_cbyCitation_journal-name">Journal of Cheminformatics</span><span> <strong>2019,</strong> <em>11 </em>
                                    (1)
                                     <a href="https://doi.org/10.1186/s13321-019-0370-7" title="DOI URL">https://doi.org/10.1186/s13321-019-0370-7</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1186/s13321-019-0370-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1186%2Fs13321-019-0370-7%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Cheminformatics%26atitle%3DBioactivity-explorer%25253A%252Ba%252Bweb%252Bapplication%252Bfor%252Binteractive%252Bvisualization%252Band%252Bexploration%252Bof%252Bbioactivity%252Bdata%26aulast%3DLiang%26aufirst%3DLu%26date%3D2019%26date%3D2019%26volume%3D11%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">J. Jesús  Naveja</span>, <span class="hlFld-ContribAuthor ">José L.  Medina-Franco</span>. </span><span class="cited-content_cbyCitation_article-title">Finding Constellations in Chemical Space Through Core Analysis. </span><span class="cited-content_cbyCitation_journal-name">Frontiers in Chemistry</span><span> <strong>2019,</strong> <em>7 </em><a href="https://doi.org/10.3389/fchem.2019.00510" title="DOI URL">https://doi.org/10.3389/fchem.2019.00510</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3389/fchem.2019.00510&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3389%2Ffchem.2019.00510%26sid%3Dliteratum%253Aachs%26jtitle%3DFrontiers%2520in%2520Chemistry%26atitle%3DFinding%252BConstellations%252Bin%252BChemical%252BSpace%252BThrough%252BCore%252BAnalysis%26aulast%3DNaveja%26aufirst%3DJ.%2BJes%25C3%25BAs%26date%3D2019%26date%3D2019%26volume%3D7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Huabin  Hu</span>, <span class="hlFld-ContribAuthor ">Dagmar  Stumpfe</span>, <span class="hlFld-ContribAuthor ">Jürgen  Bajorath</span>. </span><span class="cited-content_cbyCitation_article-title">Second-generation activity cliffs identified on the basis of target set-dependent potency difference criteria. </span><span class="cited-content_cbyCitation_journal-name">Future Medicinal Chemistry</span><span> <strong>2019,</strong> <em>11 </em>
                                    (5)
                                     , 379-394. <a href="https://doi.org/10.4155/fmc-2018-0299" title="DOI URL">https://doi.org/10.4155/fmc-2018-0299</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.4155/fmc-2018-0299&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.4155%2Ffmc-2018-0299%26sid%3Dliteratum%253Aachs%26jtitle%3DFuture%2520Medicinal%2520Chemistry%26atitle%3DSecond-generation%252Bactivity%252Bcliffs%252Bidentified%252Bon%252Bthe%252Bbasis%252Bof%252Btarget%252Bset-dependent%252Bpotency%252Bdifference%252Bcriteria%26aulast%3DHu%26aufirst%3DHuabin%26date%3D2019%26volume%3D11%26issue%3D5%26spage%3D379%26epage%3D394" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Alpha A.  Lee</span>, <span class="hlFld-ContribAuthor ">Qingyi  Yang</span>, <span class="hlFld-ContribAuthor ">Asser  Bassyouni</span>, <span class="hlFld-ContribAuthor ">Christopher R.  Butler</span>, <span class="hlFld-ContribAuthor ">Xinjun  Hou</span>, <span class="hlFld-ContribAuthor ">Stephen  Jenkinson</span>, <span class="hlFld-ContribAuthor ">David A.  Price</span>. </span><span class="cited-content_cbyCitation_article-title">Ligand biological activity predicted by cleaning positive and negative chemical correlations. </span><span class="cited-content_cbyCitation_journal-name">Proceedings of the National Academy of Sciences</span><span> <strong>2019,</strong> <em>116 </em>
                                    (9)
                                     , 3373-3378. <a href="https://doi.org/10.1073/pnas.1810847116" title="DOI URL">https://doi.org/10.1073/pnas.1810847116</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1073/pnas.1810847116&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1073%2Fpnas.1810847116%26sid%3Dliteratum%253Aachs%26jtitle%3DProceedings%2520of%2520the%2520National%2520Academy%2520of%2520Sciences%26atitle%3DLigand%252Bbiological%252Bactivity%252Bpredicted%252Bby%252Bcleaning%252Bpositive%252Band%252Bnegative%252Bchemical%252Bcorrelations%26aulast%3DLee%26aufirst%3DAlpha%2BA.%26date%3D2019%26date%3D2019%26volume%3D116%26issue%3D9%26spage%3D3373%26epage%3D3378" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Renyuan  Hong</span>, <span class="hlFld-ContribAuthor ">Xun  Li</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of monoamine oxidase inhibitors by medicinal chemistry approaches. </span><span class="cited-content_cbyCitation_journal-name">MedChemComm</span><span> <strong>2019,</strong> <em>10 </em>
                                    (1)
                                     , 10-25. <a href="https://doi.org/10.1039/C8MD00446C" title="DOI URL">https://doi.org/10.1039/C8MD00446C</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C8MD00446C&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC8MD00446C%26sid%3Dliteratum%253Aachs%26jtitle%3DMedChemComm%26atitle%3DDiscovery%252Bof%252Bmonoamine%252Boxidase%252Binhibitors%252Bby%252Bmedicinal%252Bchemistry%252Bapproaches%26aulast%3DHong%26aufirst%3DRenyuan%26date%3D2019%26date%3D2019%26volume%3D10%26issue%3D1%26spage%3D10%26epage%3D25" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Klaus  Müller</span>. </span><span class="cited-content_cbyCitation_article-title">Fluorination patterns in small alkyl groups. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2019,</strong>,, 91-139. <a href="https://doi.org/10.1016/B978-0-12-812733-9.00002-7" title="DOI URL">https://doi.org/10.1016/B978-0-12-812733-9.00002-7</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/B978-0-12-812733-9.00002-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2FB978-0-12-812733-9.00002-7%26sid%3Dliteratum%253Aachs%26atitle%3DFluorination%252Bpatterns%252Bin%252Bsmall%252Balkyl%252Bgroups%26aulast%3DM%25C3%25BCller%26aufirst%3DKlaus%26date%3D2019%26spage%3D91%26epage%3D139%26pub%3DElsevier%26atitle%3DFluorine%252Bin%252BLife%252BSciences%25253A%252BPharmaceuticals%25252C%252BMedicinal%252BDiagnostics%25252C%252Band%252BAgrochemicals%26date%3D2019%26volume%3D109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Alexey A.  Orlov</span>, <span class="hlFld-ContribAuthor ">Vladimir P.  Berishvili</span>, <span class="hlFld-ContribAuthor ">Anastasia A.  Nikitina</span>, <span class="hlFld-ContribAuthor ">Dmitry I.  Osolodkin</span>, <span class="hlFld-ContribAuthor ">Eugene V.  Radchenko</span>, <span class="hlFld-ContribAuthor ">Vladimir A.  Palyulin</span>. </span><span class="cited-content_cbyCitation_article-title">Analysis of Chemical Spaces: Implications for Drug Repurposing. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2019,</strong>,, 359-395. <a href="https://doi.org/10.1016/B978-0-12-816125-8.00013-4" title="DOI URL">https://doi.org/10.1016/B978-0-12-816125-8.00013-4</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/B978-0-12-816125-8.00013-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2FB978-0-12-816125-8.00013-4%26sid%3Dliteratum%253Aachs%26atitle%3DAnalysis%252Bof%252BChemical%252BSpaces%25253A%252BImplications%252Bfor%252BDrug%252BRepurposing%26aulast%3DOrlov%26aufirst%3DAlexey%2BA.%26date%3D2019%26spage%3D359%26epage%3D395%26pub%3DElsevier%26atitle%3DIn%252BSilico%252BDrug%252BDesign%26date%3D2019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Michael  Withnall</span>, <span class="hlFld-ContribAuthor ">Hongming  Chen</span>, <span class="hlFld-ContribAuthor ">Igor V.  Tetko</span>. </span><span class="cited-content_cbyCitation_article-title">Matched Molecular Pair Analysis on Large Melting Point Datasets: A Big Data Perspective. </span><span class="cited-content_cbyCitation_journal-name">ChemMedChem</span><span> <strong>2018,</strong> <em>13 </em>
                                    (6)
                                     , 599-606. <a href="https://doi.org/10.1002/cmdc.201700303" title="DOI URL">https://doi.org/10.1002/cmdc.201700303</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/cmdc.201700303&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fcmdc.201700303%26sid%3Dliteratum%253Aachs%26jtitle%3DChemMedChem%26atitle%3DMatched%252BMolecular%252BPair%252BAnalysis%252Bon%252BLarge%252BMelting%252BPoint%252BDatasets%25253A%252BA%252BBig%252BData%252BPerspective%26aulast%3DWithnall%26aufirst%3DMichael%26date%3D2018%26date%3D2017%26volume%3D13%26issue%3D6%26spage%3D599%26epage%3D606" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Edmund  Duesbury</span>, <span class="hlFld-ContribAuthor ">John  Holliday</span>, <span class="hlFld-ContribAuthor ">Peter  Willett</span>. </span><span class="cited-content_cbyCitation_article-title">Comparison of Maximum Common Subgraph Isomorphism Algorithms for the Alignment of 2D Chemical Structures. </span><span class="cited-content_cbyCitation_journal-name">ChemMedChem</span><span> <strong>2018,</strong> <em>13 </em>
                                    (6)
                                     , 588-598. <a href="https://doi.org/10.1002/cmdc.201700482" title="DOI URL">https://doi.org/10.1002/cmdc.201700482</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/cmdc.201700482&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fcmdc.201700482%26sid%3Dliteratum%253Aachs%26jtitle%3DChemMedChem%26atitle%3DComparison%252Bof%252BMaximum%252BCommon%252BSubgraph%252BIsomorphism%252BAlgorithms%252Bfor%252Bthe%252BAlignment%252Bof%252B2D%252BChemical%252BStructures%26aulast%3DDuesbury%26aufirst%3DEdmund%26date%3D2018%26date%3D2017%26volume%3D13%26issue%3D6%26spage%3D588%26epage%3D598" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Tim  James</span>. </span><span class="cited-content_cbyCitation_article-title">Cheminformatics in the Service of GPCR Drug Discovery. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2018,</strong>,, 395-411. <a href="https://doi.org/10.1007/978-1-4939-7465-8_20" title="DOI URL">https://doi.org/10.1007/978-1-4939-7465-8_20</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/978-1-4939-7465-8_20&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2F978-1-4939-7465-8_20%26sid%3Dliteratum%253Aachs%26atitle%3DCheminformatics%252Bin%252Bthe%252BService%252Bof%252BGPCR%252BDrug%252BDiscovery%26aulast%3DJames%26aufirst%3DTim%26date%3D2018%26date%3D2017%26spage%3D395%26epage%3D411%26pub%3DSpringer%2520New%2520York%26atitle%3DComputational%252BMethods%252Bfor%252BGPCR%252BDrug%252BDiscovery%26aulast%3DHeifetz%26aufirst%3DAlexander%26date%3D2018%26volume%3D1705" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Shoki  Ochi</span>, <span class="hlFld-ContribAuthor ">Tomoyuki  Miyao</span>, <span class="hlFld-ContribAuthor ">Kimito  Funatsu</span>. </span><span class="cited-content_cbyCitation_article-title">Structure Modification toward Applicability Domain of a QSAR/QSPR Model Considering Activity/Property. </span><span class="cited-content_cbyCitation_journal-name">Molecular Informatics</span><span> <strong>2017,</strong> <em>36 </em>
                                    (12)
                                     , 1700076. <a href="https://doi.org/10.1002/minf.201700076" title="DOI URL">https://doi.org/10.1002/minf.201700076</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/minf.201700076&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fminf.201700076%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecular%2520Informatics%26atitle%3DStructure%252BModification%252Btoward%252BApplicability%252BDomain%252Bof%252Ba%252BQSAR%25252FQSPR%252BModel%252BConsidering%252BActivity%25252FProperty%26aulast%3DOchi%26aufirst%3DShoki%26date%3D2017%26date%3D2017%26volume%3D36%26issue%3D12%26spage%3D1700076" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div><a href="#" data-source="/pb/widgets/cited-by?doi=10.1021%2Fjm200452d&amp;widgetId=9cf770f4-a9a2-46e0-bd29-3da15bcf4ae0&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2011.54.issue-22%3Bwgroup%3Astring%3AACHS+website+Group%3Barticle%3Aarticle%3A10.1021%2Fjm200452d%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text&amp;citedByCount=179&amp;pagesCount=2&amp;pageNumber=-1" title="Load all citations" class="button_primary float-right cited-by__see-more mb-5"><i aria-hidden="”true”" class="icon-angle-double-down"></i><span>Load all citations</span></a></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="fig1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-22/jm200452d/production/images/medium/jm-2011-00452d_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-22/jm200452d/production/images/large/jm-2011-00452d_0007.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Matched molecular pairs in the EGFR data set that differ by transformation of phenyl to thiophenyl. On the <i>x</i>-axis is plotted the pIC<sub>50</sub> value of the phenyl compound and on the <i>y</i>-axis the value for the thiophenyl. The red line is the 1:1 line. Error bars correspond to 3-fold variation in the IC<sub>50</sub> and indicate typical experimental uncertainty.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-22/jm200452d/production/images/large/jm-2011-00452d_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm200452d&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-22/jm200452d/production/images/medium/jm-2011-00452d_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-22/jm200452d/production/images/large/jm-2011-00452d_0001.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Example illustrating the contrast between Free–Wilson analysis and matched molecular pairs. A series of compounds with defined substitution positions is illustrated as a simple substituted phenyl ring.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-22/jm200452d/production/images/large/jm-2011-00452d_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm200452d&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-22/jm200452d/production/images/medium/jm-2011-00452d_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-22/jm200452d/production/images/large/jm-2011-00452d_0002.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. (a) Structural change being studied for its effect upon pIC<sub>50</sub> against EGFR. (b) Measured values for a training set of compounds plotted against predictions from a simple least-squares QSAR model based upon 193 descriptors for 2287 compounds. (c) Measured values plotted against predictions for a test set of 18 compounds all containing the <i>m</i>-chloro-<i>p</i>-fluoroaniline using the QSAR model built to generate (b). (d) Distribution of changes in measured values between 16 training set molecular pairs differing only by the transformation of <i>m</i>-bromoaniline to <i>m</i>-chloro-<i>p</i>-fluoroaniline. (e) In blue are the measured values of the same test set as in (c) plotted against the prediction based on matched pairs, assuming that all compounds will experience a mean change of −0.18. In black crosses are plotted the measured values for the <i>m</i>-bromoanilines upon which these predictions are made.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-22/jm200452d/production/images/large/jm-2011-00452d_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm200452d&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-22/jm200452d/production/images/medium/jm-2011-00452d_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-22/jm200452d/production/images/large/jm-2011-00452d_0003.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Distribution of changes in EGFR pIC<sub>50</sub> caused by adding a hydroxyl group in varying contexts with the most general case at the top and some specific positions at the bottom. The distributions are summarized by mean values, standard errors in those means (SEM), the standard deviation (SD), and the number of representative pairs identified (<i>N</i>). Ar represents any aromatic carbon linked group and R any aliphatic carbon linked group.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-22/jm200452d/production/images/large/jm-2011-00452d_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm200452d&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-22/jm200452d/production/images/medium/jm-2011-00452d_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-22/jm200452d/production/images/large/jm-2011-00452d_0004.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Variation with time in the mean change in solubility corresponding to addition of methyl sulfone to phenyl rings. The individual matched pairs are plotted as blue crosses in the order in which they were added to the data set, and the mean following the addition of each point is plotted in red. The green lines are at 2 standard errors in the mean above and below the mean at each point. The green arrow indicates the point in time at which the data in the publication by Leach et al. were extracted.<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-22/jm200452d/production/images/large/jm-2011-00452d_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm200452d&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-22/jm200452d/production/images/medium/jm-2011-00452d_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-22/jm200452d/production/images/large/jm-2011-00452d_0005.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Proportion of cases in which a property increases plotted against the mean change in the property for log(solubility) on the left and log(<i>K</i><sub>1</sub>) for plasma protein binding on the right. Error bars correspond to 2 standard errors in the mean on the <i>x</i>-axis, and the binomial probability 95% confidence interval is on the <i>y</i>-axis. All data are taken from Leach et al.<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-22/jm200452d/production/images/large/jm-2011-00452d_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm200452d&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-22/jm200452d/production/images/medium/jm-2011-00452d_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-22/jm200452d/production/images/large/jm-2011-00452d_0006.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Flow diagram for a potential automated drug design tool. In the left-hand box, databases are mined to obtain rules; matched molecular pair analysis provides a simple and effective way to obtain these rules. In the right-hand section, these rules are applied to compounds that have deficiencies preventing them from becoming clinical candidates to suggest new molecules that might overcome those deficiencies.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2011/jmcmar.2011.54.issue-22/jm200452d/production/images/large/jm-2011-00452d_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm200452d&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i17">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_80056" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_80056" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 62 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group">Kenny, P. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sadowski, J.</span><span> </span><span class="NLM_article-title">Structure modification in chemical databases</span> <span class="citation_source-journal">Methods Princ. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">23</span><span class="NLM_x">, </span> <span class="NLM_fpage">271</span><span class="NLM_x">–</span> <span class="NLM_lpage">285</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Fjm200452d&amp;key=10.1002%2F3527603743.ch11" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Fjm200452d&amp;key=1%3ACAS%3A528%3ADC%252BD2MXlt1KksLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2005&pages=271-285&author=P.+W.+Kennyauthor=J.+Sadowski&title=Structure+modification+in+chemical+databases"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Structure modification in chemical databases</span></div><div class="casAuthors">Kenny, Peter W.; Sadowski, Jens</div><div class="citationInfo"><span class="NLM_cas:title">Methods and Principles in Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">Chemoinformatics in Drug Discovery</span>),
    <span class="NLM_cas:pages">271-285</span>CODEN:
                <span class="NLM_cas:coden">MPMCE3</span>;
        ISSN:<span class="NLM_cas:issn">1432-4636</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Two complementary tools for controlled modification of structures and their application to building databases, in particular, generation of appropriate ionization, tautomeric and configurational states for virtual screening, are described.  Permute has been written to exhaustively generate reasonable isomers of druglike mols. with respect to protonation and formal charge, tautomerism, and nitrogen configurations.  The approach applies a set of empirical rules for possible sites of hydrogen placements, for restricting the no. of isomers, and for the prevention of duplicates.  It is optimized for isomer generation for large databases of druglike mols. for virtual screening.  Leatherface is a 2-D mol. editor that modifies properties of atoms and bonds in mol. connection tables according to rules specified by the user.  Unlike Permute, Leatherface encodes no chem. knowledge and neither processes nor generates 3-D structures.  Its real strength is that it allows the user to impose a very detailed and precisely specified chem. view on large nos. of connection tables.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrUAeZZS8UjP7Vg90H21EOLACvtfcHk0lgl_LvOcE__Fw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXlt1KksLY%253D&md5=5b5e4bdc415573c16622f43333595b6d</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1002%2F3527603743.ch11&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F3527603743.ch11%26sid%3Dliteratum%253Aachs%26aulast%3DKenny%26aufirst%3DP.%2BW.%26aulast%3DSadowski%26aufirst%3DJ.%26atitle%3DStructure%2520modification%2520in%2520chemical%2520databases%26jtitle%3DMethods%2520Princ.%2520Med.%2520Chem.%26date%3D2005%26volume%3D23%26spage%3D271%26epage%3D285" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group">Keefer, C. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chang, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kauffman, G. W.</span><span> </span><span class="NLM_article-title">Extraction of tacit knowledge from large ADME data sets via pairwise analysis</span> <span class="citation_source-journal">Bioorg. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">3739</span><span class="NLM_x">–</span> <span class="NLM_lpage">3749</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Fjm200452d&amp;key=10.1016%2Fj.bmc.2011.05.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Fjm200452d&amp;key=21616672" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Fjm200452d&amp;key=1%3ACAS%3A528%3ADC%252BC3MXnsVyisrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2011&pages=3739-3749&author=C.+E.+Keeferauthor=G.+Changauthor=G.+W.+Kauffman&title=Extraction+of+tacit+knowledge+from+large+ADME+data+sets+via+pairwise+analysis"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Extraction of tacit knowledge from large ADME data sets via pairwise analysis</span></div><div class="casAuthors">Keefer, Christopher E.; Chang, George; Kauffman, Gregory W.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">3739-3749</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Pharmaceutical companies routinely collect data across multiple projects for common ADME endpoints.  Although at the time of collection the data is intended for use in decision making within a specific project, knowledge can be gained by data mining the entire cross-project data set for patterns of structure-activity relationships (SAR) that may be applied to any project.  One such data mining method is pairwise anal.  This method has the advantage of being able to identify small structural changes that lead to significant changes in activity.  In this paper, we describe the process for full pairwise anal. of our high-throughput ADME assays routinely used for compd. discovery efforts at Pfizer (microsomal clearance, passive membrane permeability, P-gp efflux, and lipophilicity).  We also describe multiple strategies for the application of these transforms in a prospective manner during compd. design.  Finally, a detailed anal. of the activity patterns in pairs of compds. that share the same mol. transformation reveals multiple types of transforms from an SAR perspective.  These include bioisosteres, additives, multiplicatives, and a type we call switches as they act to either turn on or turn off an activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrOPDhkbDbMmLVg90H21EOLACvtfcHk0lgGHr-E2qYqOw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXnsVyisrw%253D&md5=3b1ff499078e50c63935d7c7820134cd</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2011.05.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2011.05.003%26sid%3Dliteratum%253Aachs%26aulast%3DKeefer%26aufirst%3DC.%2BE.%26aulast%3DChang%26aufirst%3DG.%26aulast%3DKauffman%26aufirst%3DG.%2BW.%26atitle%3DExtraction%2520of%2520tacit%2520knowledge%2520from%2520large%2520ADME%2520data%2520sets%2520via%2520pairwise%2520analysis%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2011%26volume%3D19%26spage%3D3739%26epage%3D3749" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group">Overington, J.</span><span> </span><span class="NLM_article-title">ChEMBL. An interview with John Overington, team leader, chemogenomics at the European Bioinformatics Institute Outstation of the European Molecular Biology Laboratory (EMBL-EBI). Interview by Wendy A. Warr</span> <span class="citation_source-journal">J. Comput.-Aided Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">23</span><span class="NLM_x">, </span> <span class="NLM_fpage">195</span><span class="NLM_x">–</span> <span class="NLM_lpage">198</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2009&pages=195-198&author=J.+Overington&title=ChEMBL.+An+interview+with+John+Overington%2C+team+leader%2C+chemogenomics+at+the+European+Bioinformatics+Institute+Outstation+of+the+European+Molecular+Biology+Laboratory+%28EMBL-EBI%29.+Interview+by+Wendy+A.+Warr"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DOverington%26aufirst%3DJ.%26atitle%3DChEMBL.%2520An%2520interview%2520with%2520John%2520Overington%252C%2520team%2520leader%252C%2520chemogenomics%2520at%2520the%2520European%2520Bioinformatics%2520Institute%2520Outstation%2520of%2520the%2520European%2520Molecular%2520Biology%2520Laboratory%2520%2528EMBL-EBI%2529.%2520Interview%2520by%2520Wendy%2520A.%2520Warr%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2009%26volume%3D23%26spage%3D195%26epage%3D198" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group">Leach, A. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jones, H. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cosgrove, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kenny, P. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ruston, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">MacFaul, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wood, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Colclough, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Law, B.</span><span> </span><span class="NLM_article-title">Matched molecular pairs as a guide in the optimization of pharmaceutical properties; a study of aqueous solubility, plasma protein binding and oral exposure</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">6672</span><span class="NLM_x">–</span> <span class="NLM_lpage">6682</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm0605233" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Fjm200452d&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtV2gtLnO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2006&pages=6672-6682&author=A.+G.+Leachauthor=H.+D.+Jonesauthor=D.+A.+Cosgroveauthor=P.+W.+Kennyauthor=L.+Rustonauthor=P.+MacFaulauthor=J.+M.+Woodauthor=N.+Colcloughauthor=B.+Law&title=Matched+molecular+pairs+as+a+guide+in+the+optimization+of+pharmaceutical+properties%3B+a+study+of+aqueous+solubility%2C+plasma+protein+binding+and+oral+exposure"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Matched Molecular Pairs as a Guide in the Optimization of Pharmaceutical Properties; a Study of Aqueous Solubility, Plasma Protein Binding and Oral Exposure</span></div><div class="casAuthors">Leach, Andrew G.; Jones, Huw D.; Cosgrove, David A.; Kenny, Peter W.; Ruston, Linette; MacFaul, Philip; Wood, J. Matthew; Colclough, Nicola; Law, Brian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">6672-6682</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">By identifying every pair of mols. that differ only by a particular, well-defined, structural transformation in a database of measured properties and computing the corresponding change in property, we obtain an overview of the effect that structural change has upon the property and set an expectation for what will happen when that transformation is applied elsewhere.  The mean change indicates the expected magnitude of the change in the property and the no. of cases in which the property increases give the probability that the structural transformation will cause the property to increase.  Outliers indicate potential ways of avoiding the general trend.  Comparing to changes in lipophilicity highlights structural transformations that have unusual effects, some of which can be explained by conformational changes.  In this paper, we focus upon the effects on aq. soly., plasma protein binding and oral exposure of adding substituents to arom. rings and methylating heteroatoms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpcyrloR79Oq7Vg90H21EOLACvtfcHk0ljhIS_PcUMC1Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtV2gtLnO&md5=2c143b424aee3ddb6c34616aff699ee0</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1021%2Fjm0605233&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0605233%26sid%3Dliteratum%253Aachs%26aulast%3DLeach%26aufirst%3DA.%2BG.%26aulast%3DJones%26aufirst%3DH.%2BD.%26aulast%3DCosgrove%26aufirst%3DD.%2BA.%26aulast%3DKenny%26aufirst%3DP.%2BW.%26aulast%3DRuston%26aufirst%3DL.%26aulast%3DMacFaul%26aufirst%3DP.%26aulast%3DWood%26aufirst%3DJ.%2BM.%26aulast%3DColclough%26aufirst%3DN.%26aulast%3DLaw%26aufirst%3DB.%26atitle%3DMatched%2520molecular%2520pairs%2520as%2520a%2520guide%2520in%2520the%2520optimization%2520of%2520pharmaceutical%2520properties%253B%2520a%2520study%2520of%2520aqueous%2520solubility%252C%2520plasma%2520protein%2520binding%2520and%2520oral%2520exposure%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2006%26volume%3D49%26spage%3D6672%26epage%3D6682" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group">Sheridan, R. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hunt, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Culberson, J. C.</span><span> </span><span class="NLM_article-title">Molecular transformations as a way of finding and exploiting consistent local QSAR</span> <span class="citation_source-journal">J. Chem. Inf. Model</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">46</span><span class="NLM_x">, </span> <span class="NLM_fpage">180</span><span class="NLM_x">–</span> <span class="NLM_lpage">192</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci0503208" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Fjm200452d&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtFKrt77N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2006&pages=180-192&author=R.+P.+Sheridanauthor=P.+Huntauthor=J.+C.+Culberson&title=Molecular+transformations+as+a+way+of+finding+and+exploiting+consistent+local+QSAR"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular Transformations as a Way of Finding and Exploiting Consistent Local QSAR</span></div><div class="casAuthors">Sheridan, Robert P.; Hunt, Peter; Culberson, J. Chris</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">180-192</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The idea of a "transformation", making a small change to a chem. structure, for instance removing or replacing a substituent, is familiar to chemists.  We suggest two ways of representing a transformation in silico, as a substructure descriptor difference vector, and as the set of atoms remaining once a max. common substructure is eliminated.  Such transformations can be filtered sensibly, and it is easy to compare one transformation to another.  These representations have two applications.  First, we can use these methods to automatically organize and display sets of closely related compds. such that any consistent local QSAR in a data set can be easily seen, the T-ANALYZE application.  Second, we can suggest to a chemist how to change a mol. "on hand" to a more active one based on local QSAR for that activity, the T-MORPH application.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrM-7rAXqeMnrVg90H21EOLACvtfcHk0ljhIS_PcUMC1Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtFKrt77N&md5=01eff3cb0a2f1a78a18a7c4a3857947b</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1021%2Fci0503208&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci0503208%26sid%3Dliteratum%253Aachs%26aulast%3DSheridan%26aufirst%3DR.%2BP.%26aulast%3DHunt%26aufirst%3DP.%26aulast%3DCulberson%26aufirst%3DJ.%2BC.%26atitle%3DMolecular%2520transformations%2520as%2520a%2520way%2520of%2520finding%2520and%2520exploiting%2520consistent%2520local%2520QSAR%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model%26date%3D2006%26volume%3D46%26spage%3D180%26epage%3D192" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group">Meanwell, N. A.</span><span> </span><span class="NLM_article-title">Synopsis of some recent tactical application of bioisosteres in drug design</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">54</span><span class="NLM_x">, </span> <span class="NLM_fpage">2529</span><span class="NLM_x">–</span> <span class="NLM_lpage">2591</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm1013693" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Fjm200452d&amp;key=1%3ACAS%3A528%3ADC%252BC3MXjsVCnu7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=2529-2591&author=N.+A.+Meanwell&title=Synopsis+of+some+recent+tactical+application+of+bioisosteres+in+drug+design"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Synopsis of Some Recent Tactical Application of Bioisosteres in Drug Design</span></div><div class="casAuthors">Meanwell, Nicholas A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2529-2591</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The established utility of bioisosteres is broad in nature, extending to improving potency, enhancing selectivity, altering phys. properties, reducing or redirecting metab., eliminating or modifying toxicophores, and acquiring novel intellectual property.  In this Perspective, some contemporary themes exploring the role of isosteres in drug design are sampled, with an emphasis placed on tactical applications designed to solve the kinds of problems that impinge on compd. optimization and the long-term success of drug candidates.  Interesting concepts that may have been poorly effective in the context examd. are captured, since the ideas may have merit in alternative circumstances.  A comprehensive cataloging of bioisosteres is beyond the scope of what will be provided, although a synopsis of relevant isosteres of a particular functionality is summarized in a succinct fashion in several sections.  Isosterism has also found productive application in the design and optimization of organocatalysts, and there are several examples in which functional mimicry established initially in a medicinal chem. setting has been adopted by this community.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrZ9z_aPcU4ubVg90H21EOLACvtfcHk0ljhIS_PcUMC1Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXjsVCnu7o%253D&md5=0637ab60c379bef535f3f709b26f3582</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1021%2Fjm1013693&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm1013693%26sid%3Dliteratum%253Aachs%26aulast%3DMeanwell%26aufirst%3DN.%2BA.%26atitle%3DSynopsis%2520of%2520some%2520recent%2520tactical%2520application%2520of%2520bioisosteres%2520in%2520drug%2520design%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D2529%26epage%3D2591" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group">Langdon, S. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ertl, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, N.</span><span> </span><span class="NLM_article-title">Bioisosteric replacement and scaffold hopping in lead generation and optimization</span> <span class="citation_source-journal">Mol. Inf.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">29</span><span class="NLM_x">, </span> <span class="NLM_fpage">366</span><span class="NLM_x">–</span> <span class="NLM_lpage">385</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Fjm200452d&amp;key=10.1002%2Fminf.201000019" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Fjm200452d&amp;key=27463193" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Fjm200452d&amp;key=1%3ACAS%3A528%3ADC%252BC3cXmsVSgur4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2010&pages=366-385&author=S.+R.+Langdonauthor=P.+Ertlauthor=N.+Brown&title=Bioisosteric+replacement+and+scaffold+hopping+in+lead+generation+and+optimization"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Bioisosteric Replacement and Scaffold Hopping in Lead Generation and Optimization</span></div><div class="casAuthors">Langdon, Sarah R.; Ertl, Peter; Brown, Nathan</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Informatics</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">366-385</span>CODEN:
                <span class="NLM_cas:coden">MIONBS</span>;
        ISSN:<span class="NLM_cas:issn">1868-1743</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A review.  Bioisosteric replacement and scaffold hopping are twin methods used in drug design to improve the synthetic accessibility, potency and drug like properties of a compd. and to move into novel chem. space.  Bioisosteric replacement involves swapping functional groups of a mol. with other functional groups that have similar biol. properties.  Scaffold hopping is the replacement of the core framework of a mol. with another scaffold that will improve the properties of the mol. or to find similar potent compds. that exist in novel chem. space.  This review outlines the key concepts, importance and challenges of both methods using examples and comparisons of techniques available for finding bioisosteric replacements and scaffold hops.  There are many methods available for bioisosteric replacement and scaffold hopping, all with their own advantages and disadvantages.  Drug design projects would benefit from a combination of these methods to retrieve diverse and complimentary results.  Continuing progress in these fields will allow further validation of both methods as well as the accumulation of knowledge on bioisosteres and possible scaffold replacements.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpJsbL69-PF-7Vg90H21EOLACvtfcHk0lhJohdilZ7PpQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXmsVSgur4%253D&md5=372b9094a34108fb978f5d9fa95831cd</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1002%2Fminf.201000019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fminf.201000019%26sid%3Dliteratum%253Aachs%26aulast%3DLangdon%26aufirst%3DS.%2BR.%26aulast%3DErtl%26aufirst%3DP.%26aulast%3DBrown%26aufirst%3DN.%26atitle%3DBioisosteric%2520replacement%2520and%2520scaffold%2520hopping%2520in%2520lead%2520generation%2520and%2520optimization%26jtitle%3DMol.%2520Inf.%26date%3D2010%26volume%3D29%26spage%3D366%26epage%3D385" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group">Free, S. M.,  Jr.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilson, J. W.</span><span> </span><span class="NLM_article-title">A mathematical contribution to structure–activity studies</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1964</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">395</span><span class="NLM_x">–</span> <span class="NLM_lpage">399</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm00334a001" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=1964&pages=395-399&author=S.+M.+Freeauthor=J.+W.+Wilson&title=A+mathematical+contribution+to+structure%E2%80%93activity+studies"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1021%2Fjm00334a001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm00334a001%26sid%3Dliteratum%253Aachs%26aulast%3DFree%26aufirst%3DS.%2BM.%26aulast%3DWilson%26aufirst%3DJ.%2BW.%26atitle%3DA%2520mathematical%2520contribution%2520to%2520structure%25E2%2580%2593activity%2520studies%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1964%26volume%3D7%26spage%3D395%26epage%3D399" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group">Patel, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gillet, V. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Howe, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pastor, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oyarzabal, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Willett, P.</span><span> </span><span class="NLM_article-title">Assessment of additive/nonadditive effects in structure–activity relationships: implications for iterative drug design</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">7552</span><span class="NLM_x">–</span> <span class="NLM_lpage">7562</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm801070q" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2008&pages=7552-7562&author=Y.+Patelauthor=V.+J.+Gilletauthor=T.+Howeauthor=J.+Pastorauthor=J.+Oyarzabalauthor=P.+Willett&title=Assessment+of+additive%2Fnonadditive+effects+in+structure%E2%80%93activity+relationships%3A+implications+for+iterative+drug+design"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1021%2Fjm801070q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm801070q%26sid%3Dliteratum%253Aachs%26aulast%3DPatel%26aufirst%3DY.%26aulast%3DGillet%26aufirst%3DV.%2BJ.%26aulast%3DHowe%26aufirst%3DT.%26aulast%3DPastor%26aufirst%3DJ.%26aulast%3DOyarzabal%26aufirst%3DJ.%26aulast%3DWillett%26aufirst%3DP.%26atitle%3DAssessment%2520of%2520additive%252Fnonadditive%2520effects%2520in%2520structure%25E2%2580%2593activity%2520relationships%253A%2520implications%2520for%2520iterative%2520drug%2520design%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2008%26volume%3D51%26spage%3D7552%26epage%3D7562" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group">Topliss, J. G.</span><span> </span><span class="NLM_article-title">A manual method for applying the hansch approach to drug design</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1977</span><span class="NLM_x">, </span> <span class="NLM_volume">20</span><span class="NLM_x">, </span> <span class="NLM_fpage">463</span><span class="NLM_x">–</span> <span class="NLM_lpage">469</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm00214a001" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Fjm200452d&amp;key=1%3ACAS%3A528%3ADyaE2sXhtFGgtrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=1977&pages=463-469&author=J.+G.+Topliss&title=A+manual+method+for+applying+the+hansch+approach+to+drug+design"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">A manual method for applying the Hansch approach to drug design</span></div><div class="casAuthors">Topliss, John G.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">1977</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">463-9</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    </div><div class="casAbstract">A procedure is described in which an initial small group of compds. is selected, tested, and ordered according to potency.  The potency order in the group is then compared to the tabulated potency order calcd. for various parameter dependencies relating to hydrophobic, electronic, and steric effects.  From this activity pattern anal. the probable operative parameters can be deduced and a new substituent selection made for the synthesis of potentially more potent analogs.  Application of the method is illustrated with a series of examples.  It differs from a previously described decision tree, single compd. stepwise approach in that it involves the batchwise anal. of small groups of compds., usually the preferred procedure for logistical reasons if the compds. are relatively easy to synthesize.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpILt0RoFSjlbVg90H21EOLACvtfcHk0lhJohdilZ7PpQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaE2sXhtFGgtrk%253D&md5=74658b807e3e45cbee88855eda203bdb</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1021%2Fjm00214a001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm00214a001%26sid%3Dliteratum%253Aachs%26aulast%3DTopliss%26aufirst%3DJ.%2BG.%26atitle%3DA%2520manual%2520method%2520for%2520applying%2520the%2520hansch%2520approach%2520to%2520drug%2520design%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1977%26volume%3D20%26spage%3D463%26epage%3D469" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group">Dossetter, A. G.</span><span> </span><span class="NLM_article-title">A statistical analysis of in vitro human microsomal metabolic stability of small phenyl group substituents, leading to improved design sets for parallel SAR exploration of a chemical series</span> <span class="citation_source-journal">Bioorg. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x">, </span> <span class="NLM_fpage">4405</span><span class="NLM_x">–</span> <span class="NLM_lpage">4414</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Fjm200452d&amp;key=10.1016%2Fj.bmc.2010.04.077" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Fjm200452d&amp;key=20510621" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Fjm200452d&amp;key=1%3ACAS%3A528%3ADC%252BC3cXntV2rurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2010&pages=4405-4414&author=A.+G.+Dossetter&title=A+statistical+analysis+of+in+vitro+human+microsomal+metabolic+stability+of+small+phenyl+group+substituents%2C+leading+to+improved+design+sets+for+parallel+SAR+exploration+of+a+chemical+series"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">A statistical analysis of in vitro human microsomal metabolic stability of small phenyl group substituents, leading to improved design sets for parallel SAR exploration of a chemical series</span></div><div class="casAuthors">Dossetter, Alexander G.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">4405-4414</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">An anal. of in vitro human liver microsomal turnover assay results from a large dataset (∼75 K) of exptl. compds. tested is presented.  Combined with an anal. of small (<6 Ha) substituents on known drugs and existing published results a new set of 29 substituents (consensus) is proposed to increase stability and probe SAR (an enhanced Topliss set').  In addn. a different group of 28 substituents are identified as unlikely to change in vitro HLM stability, and a further set of compds. focuses on increasing HLM stability only.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpOwR4TenuEULVg90H21EOLACvtfcHk0lhJohdilZ7PpQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXntV2rurc%253D&md5=93c9825d5da0b47e0f4c66996c883642</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2010.04.077&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2010.04.077%26sid%3Dliteratum%253Aachs%26aulast%3DDossetter%26aufirst%3DA.%2BG.%26atitle%3DA%2520statistical%2520analysis%2520of%2520in%2520vitro%2520human%2520microsomal%2520metabolic%2520stability%2520of%2520small%2520phenyl%2520group%2520substituents%252C%2520leading%2520to%2520improved%2520design%2520sets%2520for%2520parallel%2520SAR%2520exploration%2520of%2520a%2520chemical%2520series%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2010%26volume%3D18%26spage%3D4405%26epage%3D4414" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group">Sutherland, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">O’Brien, L. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weaver, D. F.</span><span> </span><span class="NLM_article-title">A comparison of methods for modeling quantitative structure–activity relationships</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">5541</span><span class="NLM_x">–</span> <span class="NLM_lpage">5554</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm0497141" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=5541-5554&author=J.+J.+Sutherlandauthor=L.+A.+O%E2%80%99Brienauthor=D.+F.+Weaver&title=A+comparison+of+methods+for+modeling+quantitative+structure%E2%80%93activity+relationships"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1021%2Fjm0497141&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0497141%26sid%3Dliteratum%253Aachs%26aulast%3DSutherland%26aufirst%3DJ.%2BJ.%26aulast%3DO%25E2%2580%2599Brien%26aufirst%3DL.%2BA.%26aulast%3DWeaver%26aufirst%3DD.%2BF.%26atitle%3DA%2520comparison%2520of%2520methods%2520for%2520modeling%2520quantitative%2520structure%25E2%2580%2593activity%2520relationships%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D5541%26epage%3D5554" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group">Katritzky, A. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuanar, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Slavov, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hall, C. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Karelson, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kahn, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dobchev, D. A.</span><span> </span><span class="NLM_article-title">Quantitative correlation of physical and chemical properties with chemical structure: utility for prediction</span> <span class="citation_source-journal">Chem. Rev. (Washington, DC, U. S.)</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">110</span><span class="NLM_x">, </span> <span class="NLM_fpage">5714</span><span class="NLM_x">–</span> <span class="NLM_lpage">5789</span></span><div class="citationLinks">[<a href="/doi/10.1021/cr900238d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Fjm200452d&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtVKmtbnP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=110&publication_year=2010&pages=5714-5789&author=A.+R.+Katritzkyauthor=M.+Kuanarauthor=S.+Slavovauthor=C.+D.+Hallauthor=M.+Karelsonauthor=I.+Kahnauthor=D.+A.+Dobchev&title=Quantitative+correlation+of+physical+and+chemical+properties+with+chemical+structure%3A+utility+for+prediction"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Quantitative Correlation of Physical and Chemical Properties with Chemical Structure: Utility for Prediction</span></div><div class="casAuthors">Katritzky, Alan R.; Kuanar, Minati; Slavov, Svetoslav; Hall, C. Dennis; Karelson, Mati; Kahn, Iiris; Dobchev, Dimitar A.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">110</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">5714-5789</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The present review summarizes recent QSPR research methods and applications.  The main focus is placed on QSPR based upon structural descriptors derived solely from chem. structure for the correlation and prediction of various phys., chem., and physicochem. properties of compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo6wWJhM8cT_7Vg90H21EOLACvtfcHk0ljyrlZfGd-fYw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtVKmtbnP&md5=8fc26f1e548cd8f9e5d3fac6dd4c12bc</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1021%2Fcr900238d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcr900238d%26sid%3Dliteratum%253Aachs%26aulast%3DKatritzky%26aufirst%3DA.%2BR.%26aulast%3DKuanar%26aufirst%3DM.%26aulast%3DSlavov%26aufirst%3DS.%26aulast%3DHall%26aufirst%3DC.%2BD.%26aulast%3DKarelson%26aufirst%3DM.%26aulast%3DKahn%26aufirst%3DI.%26aulast%3DDobchev%26aufirst%3DD.%2BA.%26atitle%3DQuantitative%2520correlation%2520of%2520physical%2520and%2520chemical%2520properties%2520with%2520chemical%2520structure%253A%2520utility%2520for%2520prediction%26jtitle%3DChem.%2520Rev.%2520%2528Washington%252C%2520DC%252C%2520U.%2520S.%2529%26date%3D2010%26volume%3D110%26spage%3D5714%26epage%3D5789" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group">Guha, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Van Drie, J. H.</span><span> </span><span class="NLM_article-title">Assessing how well modeling protocol captures structure–activity landscape</span> <span class="citation_source-journal">J. Chem. Inf. Model</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">48</span><span class="NLM_x">, </span> <span class="NLM_fpage">1716</span><span class="NLM_x">–</span> <span class="NLM_lpage">1728</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci8001414" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Fjm200452d&amp;key=1%3ACAS%3A528%3ADC%252BD1cXps1KnsLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2008&pages=1716-1728&author=R.+Guhaauthor=J.+H.+Van+Drie&title=Assessing+how+well+modeling+protocol+captures+structure%E2%80%93activity+landscape"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Assessing how well modeling protocol captures structure-activity landscape</span></div><div class="casAuthors">Guha, Rajarshi; Van Drie, John H.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1716-1728</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We introduce the notion of structure-activity landscape index (SALI) curves as a way to assess a model and a modeling protocol, applied to structure-activity relationships.  We start from our earlier work [J. Chem. Inf. Model., 2008, 48, 646-658], where we show how to study a structure-activity relationship pairwise, based on the notion of "activity cliffs" - pairs of mols. that are structurally similar but have large differences in activity.  There, we also introduced the SALI parameter, which allows one to identify cliffs easily, and which allows one to represent a structure-activity relationship as a graph.  This graph orders every pair of mols. by their activity.  Here, we introduce the new idea of a SALI curve, which tallies how many of these orderings a model is able to predict.  Empirically, testing these SALI curves against a variety of models, ranging over two-dimensional quant. structure-activity relationship (2D-QSAR), three-dimensional quant. structure-activity relationship (3D-QSAR), and structure-based design models, the utility of a model seems to correspond to characteristics of these curves.  In particular, the integral of these curves, denoted as SCI and being a no. ranging from -1.0 to 1.0, approaches a value of 1.0 for two literature models, which are both known to be prospectively useful.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrWF7pqmO2nIbVg90H21EOLACvtfcHk0ljyrlZfGd-fYw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXps1KnsLY%253D&md5=e2ef0062ee3779ea0ef595dc534d703f</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1021%2Fci8001414&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci8001414%26sid%3Dliteratum%253Aachs%26aulast%3DGuha%26aufirst%3DR.%26aulast%3DVan%2BDrie%26aufirst%3DJ.%2BH.%26atitle%3DAssessing%2520how%2520well%2520modeling%2520protocol%2520captures%2520structure%25E2%2580%2593activity%2520landscape%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model%26date%3D2008%26volume%3D48%26spage%3D1716%26epage%3D1728" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group">Livingstone, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Salt, D. W.</span><span> </span><span class="NLM_article-title">Judging the significance of multiple linear regression models</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">48</span><span class="NLM_x">, </span> <span class="NLM_fpage">661</span><span class="NLM_x">–</span> <span class="NLM_lpage">663</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm049111p" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Fjm200452d&amp;key=1%3ACAS%3A528%3ADC%252BD2MXjtFKmtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2005&pages=661-663&author=D.+J.+Livingstoneauthor=D.+W.+Salt&title=Judging+the+significance+of+multiple+linear+regression+models"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Judging the Significance of Multiple Linear Regression Models</span></div><div class="casAuthors">Livingstone, David J.; Salt, David W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">661-663</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">It is common practice to calc. large nos. of mol. descriptors, apply variable selection procedures to reduce the nos., and then construct multiple linear regression (MLR) models with biol. activity.  The significance of these models is judged using the usual statistical tests.  Unfortunately, these tests are not appropriate under these circumstances since the MLR models suffer from "selection bias".  Expts. with regression using random nos. have generated crit. values (Fmax) with which to assess significance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpZLFVeHNGO77Vg90H21EOLACvtfcHk0ljyrlZfGd-fYw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXjtFKmtA%253D%253D&md5=784a318f9968b5d65d3cec78b73c3fe0</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1021%2Fjm049111p&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm049111p%26sid%3Dliteratum%253Aachs%26aulast%3DLivingstone%26aufirst%3DD.%2BJ.%26aulast%3DSalt%26aufirst%3DD.%2BW.%26atitle%3DJudging%2520the%2520significance%2520of%2520multiple%2520linear%2520regression%2520models%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2005%26volume%3D48%26spage%3D661%26epage%3D663" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group">Bruneau, P.</span><span> </span><span class="NLM_article-title">Search for predictive generic model of aqueous solubility using Bayesian neural nets</span> <span class="citation_source-journal">J. Chem. Inf. Comput. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">41</span><span class="NLM_x">, </span> <span class="NLM_fpage">1605</span><span class="NLM_x">–</span> <span class="NLM_lpage">1616</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci010363y" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Fjm200452d&amp;key=1%3ACAS%3A528%3ADC%252BD3MXms1Ogtrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2001&pages=1605-1616&author=P.+Bruneau&title=Search+for+predictive+generic+model+of+aqueous+solubility+using+Bayesian+neural+nets"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Search for predictive generic model of aqueous solubility using Bayesian neural nets</span></div><div class="casAuthors">Bruneau, Pierre</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Computer Sciences</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1605-1616</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">0095-2338</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Several predictive models of aq. soly. have been published.  They have good performances on the data sets which have been used for training the models, but usually these data sets do not contain many structures similar to the structures of interest to the drug research and their applicability in drug hunting is questionable.  A very diverse data set has been gathered with compds. issued from literature reports and proprietary compds.  These compds. have been grouped in a so-called literature data set I, a proprietary data set II, and a mixed data set III formed by I and II.  About 100 descriptors emphasizing surface properties were calcd. for every compd.  Bayesian learning of neural nets which cumulates the advantages of neural nets without having their weaknesses was used to select the most parsimonious models and train them, from I, II, and III.  The models were established by either selecting the most efficient descriptors one by one using a modified Gram-Schmidt procedure (GS) or by simplifying a most complete model using automatic relevance procedure (ARD).  The predictive ability of the models was accessed using validation data sets as much unrelated to the training sets as possible, using two new parameters: NDDx,ref the normalized smallest descriptor distance of a compd. x to a ref. data set and CDx,mod the combination of NDDx,ref with the dispersion of the Bayesian neural nets calcns.  The results show that it is possible to obtain a generic predictive model from database I but that the diversity of database II is too restricted to give a model with good generalization ability and that the ARD method applied to the mixed database III gives the best predictive model.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrDYiHCG1pIOrVg90H21EOLACvtfcHk0ljzudA66WkW4w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXms1Ogtrs%253D&md5=b6a2e1c8e3b297b35fd774f7fe3efb33</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1021%2Fci010363y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci010363y%26sid%3Dliteratum%253Aachs%26aulast%3DBruneau%26aufirst%3DP.%26atitle%3DSearch%2520for%2520predictive%2520generic%2520model%2520of%2520aqueous%2520solubility%2520using%2520Bayesian%2520neural%2520nets%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Comput.%2520Sci.%26date%3D2001%26volume%3D41%26spage%3D1605%26epage%3D1616" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span> <span class="citation_source-book">JMP</span>, version <span class="NLM_edition">8.0</span>; <span class="NLM_publisher-name">SAS Institute Inc.</span>: <span class="NLM_publisher-loc">Cary, NC</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=JMP%2C+version+8.0%3B+SAS+Institute+Inc.%3A+Cary%2C+NC."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26btitle%3DJMP%26pub%3DSAS%2520Institute%2520Inc" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group">Papadatos, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Alkarouri, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gillet, V. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Willett, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kadirkamanathan, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luscombe, C. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bravi, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richmond, N. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pickett, S. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hussain, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pritchard, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cooper, A. W. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">MacDonald, S. J. F.</span><span> </span><span class="NLM_article-title">Lead optimization using matched molecular pairs: inclusion of contextual information for enhanced prediction of hERG inhibition, solubility, and lipophilicity</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x">, </span> <span class="NLM_fpage">1872</span><span class="NLM_x">–</span> <span class="NLM_lpage">1886</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci100258p" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Fjm200452d&amp;key=1%3ACAS%3A528%3ADC%252BC3cXht1Whs7jJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2010&pages=1872-1886&author=G.+Papadatosauthor=M.+Alkarouriauthor=V.+J.+Gilletauthor=P.+Willettauthor=V.+Kadirkamanathanauthor=C.+N.+Luscombeauthor=G.+Braviauthor=N.+J.+Richmondauthor=S.+D.+Pickettauthor=J.+Hussainauthor=J.+M.+Pritchardauthor=A.+W.+J.+Cooperauthor=S.+J.+F.+MacDonald&title=Lead+optimization+using+matched+molecular+pairs%3A+inclusion+of+contextual+information+for+enhanced+prediction+of+hERG+inhibition%2C+solubility%2C+and+lipophilicity"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Lead Optimization Using Matched Molecular Pairs: Inclusion of Contextual Information for Enhanced Prediction of hERG Inhibition, Solubility, and Lipophilicity</span></div><div class="casAuthors">Papadatos, George; Alkarouri, Muhammad; Gillet, Valerie J.; Willett, Peter; Kadirkamanathan, Visakan; Luscombe, Christopher N.; Bravi, Gianpaolo; Richmond, Nicola J.; Pickett, Stephen D.; Hussain, Jameed; Pritchard, John M.; Cooper, Anthony W. J.; MacDonald, Simon J. F.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1872-1886</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Previous studies of the anal. of mol. matched pairs (MMPs) have often assumed that the effect of a substructural transformation on a mol. property is independent of the context (i.e., the local structural environment in which that transformation occurs).  Expts. with large sets of hERG, soly., and lipophilicity data demonstrate that the inclusion of contextual information can enhance the predictive power of MMP analyses, with significant trends (both pos. and neg.) being identified that are not apparent when using conventional, context-independent approaches.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoLZfW5E3jgX7Vg90H21EOLACvtfcHk0ljzudA66WkW4w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXht1Whs7jJ&md5=464dff86e047cf1b0273571f82786994</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1021%2Fci100258p&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci100258p%26sid%3Dliteratum%253Aachs%26aulast%3DPapadatos%26aufirst%3DG.%26aulast%3DAlkarouri%26aufirst%3DM.%26aulast%3DGillet%26aufirst%3DV.%2BJ.%26aulast%3DWillett%26aufirst%3DP.%26aulast%3DKadirkamanathan%26aufirst%3DV.%26aulast%3DLuscombe%26aufirst%3DC.%2BN.%26aulast%3DBravi%26aufirst%3DG.%26aulast%3DRichmond%26aufirst%3DN.%2BJ.%26aulast%3DPickett%26aufirst%3DS.%2BD.%26aulast%3DHussain%26aufirst%3DJ.%26aulast%3DPritchard%26aufirst%3DJ.%2BM.%26aulast%3DCooper%26aufirst%3DA.%2BW.%2BJ.%26aulast%3DMacDonald%26aufirst%3DS.%2BJ.%2BF.%26atitle%3DLead%2520optimization%2520using%2520matched%2520molecular%2520pairs%253A%2520inclusion%2520of%2520contextual%2520information%2520for%2520enhanced%2520prediction%2520of%2520hERG%2520inhibition%252C%2520solubility%252C%2520and%2520lipophilicity%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2010%26volume%3D50%26spage%3D1872%26epage%3D1886" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group">Hajduk, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sauer, D. R.</span><span> </span><span class="NLM_article-title">Statistical analysis of the effects of common chemical substituents on ligand potency</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">553</span><span class="NLM_x">–</span> <span class="NLM_lpage">564</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm070838y" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Fjm200452d&amp;key=1%3ACAS%3A528%3ADC%252BD1cXjt1ejtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2008&pages=553-564&author=P.+J.+Hajdukauthor=D.+R.+Sauer&title=Statistical+analysis+of+the+effects+of+common+chemical+substituents+on+ligand+potency"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Statistical Analysis of the Effects of Common Chemical Substituents on Ligand Potency</span></div><div class="casAuthors">Hajduk, Philip J.; Sauer, Daryl R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">553-564</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The results of a statistical anal. of more than 84000 compds. from lead optimization programs against 30 different protein targets is presented, with a focus on the effects that different chem. substituents have on compd. potency.  It is obsd. that the potency changes induced by most chem. groups follows a nearly normal distribution centered near zero (i.e., no effect on potency).  However, the widths of the distributions vary significantly between different substituents, and these effects cannot be rationalized by simple physicochem. parameters.  In addn., certain substituents consistently bias the distribution toward higher or lower potency, suggesting the existence of preferred and nonpreferred chem. groups for lead optimization.  The implications of these results for understanding protein-ligand recognition and for enhancing the efficiency and speed of lead optimization will be discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr9sbNYA_Mzl7Vg90H21EOLACvtfcHk0ljzudA66WkW4w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXjt1ejtQ%253D%253D&md5=396d02789aece9be31fe3508fbabca7d</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1021%2Fjm070838y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm070838y%26sid%3Dliteratum%253Aachs%26aulast%3DHajduk%26aufirst%3DP.%2BJ.%26aulast%3DSauer%26aufirst%3DD.%2BR.%26atitle%3DStatistical%2520analysis%2520of%2520the%2520effects%2520of%2520common%2520chemical%2520substituents%2520on%2520ligand%2520potency%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2008%26volume%3D51%26spage%3D553%26epage%3D564" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group">Gleeson, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bravi, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Modi, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lowe, D.</span><span> </span><span class="NLM_article-title">ADMET rules of thumb II: a comparison of the effects of common substituents on a range of ADMET parameters</span> <span class="citation_source-journal">Bioorg. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">17</span><span class="NLM_x">, </span> <span class="NLM_fpage">5906</span><span class="NLM_x">–</span> <span class="NLM_lpage">5919</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Fjm200452d&amp;key=10.1016%2Fj.bmc.2009.07.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Fjm200452d&amp;key=19632124" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Fjm200452d&amp;key=1%3ACAS%3A528%3ADC%252BD1MXps1Crs7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2009&pages=5906-5919&author=P.+Gleesonauthor=G.+Braviauthor=S.+Modiauthor=D.+Lowe&title=ADMET+rules+of+thumb+II%3A+a+comparison+of+the+effects+of+common+substituents+on+a+range+of+ADMET+parameters"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">ADMET rules of thumb II: A comparison of the effects of common substituents on a range of ADMET parameters</span></div><div class="casAuthors">Gleeson, Paul; Bravi, Gianpaolo; Modi, Sandeep; Lowe, Daniel</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">5906-5919</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A systematic anal. of data generated in key in vitro assays within GSK has been undertaken to identify what impact a range of common substituents have on a range of ADMET parameters.  These include; P 450 1A2, 2C9, 2C19, 2D6 and 3A4 inhibition, hERG inhibition, phosphate buffer soly. and artificial membrane permeability.  We do this by identifying all matched mol. pairs, differing by the replacement of a hydrogen atom with a list of predefined substituents.  For each substituent we calc. the mean difference in the ADMET parameter for all the matched mol. pairs identified, making a statistical assessment of the difference, as well as assessing the diversity for each example to ensure that the results can be generalized.  We also relate the change in activity obsd. for each substituent to differences in their mol. properties in an effort to identify any structural alerts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpk7HA_bqdilrVg90H21EOLACvtfcHk0ljzudA66WkW4w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXps1Crs7o%253D&md5=fce672cbaadb66c4d7212aa1b92bd7d2</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2009.07.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2009.07.002%26sid%3Dliteratum%253Aachs%26aulast%3DGleeson%26aufirst%3DP.%26aulast%3DBravi%26aufirst%3DG.%26aulast%3DModi%26aufirst%3DS.%26aulast%3DLowe%26aufirst%3DD.%26atitle%3DADMET%2520rules%2520of%2520thumb%2520II%253A%2520a%2520comparison%2520of%2520the%2520effects%2520of%2520common%2520substituents%2520on%2520a%2520range%2520of%2520ADMET%2520parameters%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2009%26volume%3D17%26spage%3D5906%26epage%3D5919" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group">Fujita, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ban, T.</span><span> </span><span class="NLM_article-title">Structure–activity relation. 3. Structure–activity study of phenethylamines as substrates of biosynthetic enzymes of sympathetic transmitters</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1971</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x">, </span> <span class="NLM_fpage">148</span><span class="NLM_x">–</span> <span class="NLM_lpage">152</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm00284a016" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=1971&pages=148-152&author=T.+Fujitaauthor=T.+Ban&title=Structure%E2%80%93activity+relation.+3.+Structure%E2%80%93activity+study+of+phenethylamines+as+substrates+of+biosynthetic+enzymes+of+sympathetic+transmitters"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Fjm00284a016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm00284a016%26sid%3Dliteratum%253Aachs%26aulast%3DFujita%26aufirst%3DT.%26aulast%3DBan%26aufirst%3DT.%26atitle%3DStructure%25E2%2580%2593activity%2520relation.%25203.%2520Structure%25E2%2580%2593activity%2520study%2520of%2520phenethylamines%2520as%2520substrates%2520of%2520biosynthetic%2520enzymes%2520of%2520sympathetic%2520transmitters%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1971%26volume%3D14%26spage%3D148%26epage%3D152" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group">Wassermann, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bajorath, J.</span><span> </span><span class="NLM_article-title">Chemical substitutions that introduce activity cliffs across different compound classes and biological targets</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x">, </span> <span class="NLM_fpage">1248</span><span class="NLM_x">–</span> <span class="NLM_lpage">1256</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci1001845" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Fjm200452d&amp;key=1%3ACAS%3A528%3ADC%252BC3cXosVGhtrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2010&pages=1248-1256&author=A.+M.+Wassermannauthor=J.+Bajorath&title=Chemical+substitutions+that+introduce+activity+cliffs+across+different+compound+classes+and+biological+targets"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Chemical Substitutions That Introduce Activity Cliffs Across Different Compound Classes and Biological Targets</span></div><div class="casAuthors">Wassermann, Anne Mai; Bajorath, Juergen</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1248-1256</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Applying the concept of matched mol. pairs, the authors have systematically analyzed the ability of defined chem. changes to introduce activity cliffs.  Public domain compd. data were systematically screened for matched mol. pairs that were then organized according to chem. transformations they represent and assocd. potency changes.  From vast available chem. transformation space, including both R-group and core substructure changes, ∼250 nonredundant substitutions were identified that displayed a general tendency to form activity cliffs.  These substitutions introduced activity cliffs in the structural context of diverse scaffolds and in compds. active against many different targets.  Activity cliff-forming transformations were often rather simple, including replacements of small functional groups.  Moreover, in many instances, chem. very similar transformations were identified that had a much lower propensity to form activity cliffs or no detectable cliff potential.  Thus, clear preferences emerged for specific transformations.  A compendium of substitutions with general activity cliff-forming potential is provided to aid in compd. optimization efforts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpUSS8Tj-rk8bVg90H21EOLACvtfcHk0lji7rQHD4Vi_w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXosVGhtrY%253D&md5=381262f83abb19d326dbf95ddc367565</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1021%2Fci1001845&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci1001845%26sid%3Dliteratum%253Aachs%26aulast%3DWassermann%26aufirst%3DA.%2BM.%26aulast%3DBajorath%26aufirst%3DJ.%26atitle%3DChemical%2520substitutions%2520that%2520introduce%2520activity%2520cliffs%2520across%2520different%2520compound%2520classes%2520and%2520biological%2520targets%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2010%26volume%3D50%26spage%3D1248%26epage%3D1256" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group">Ames, B. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gurney, E. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miller, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bartsch, H.</span><span> </span><span class="NLM_article-title">Carcinogens as frameshift mutagens. Metabolites and derivatives of 2-acetylaminofluorene and other aromatic amine carcinogens</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U.S.A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1972</span><span class="NLM_x">, </span> <span class="NLM_volume">69</span><span class="NLM_x">, </span> <span class="NLM_fpage">3128</span><span class="NLM_x">–</span> <span class="NLM_lpage">3132</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=69&publication_year=1972&pages=3128-3132&author=B.+N.+Amesauthor=E.+G.+Gurneyauthor=J.+A.+Millerauthor=H.+Bartsch&title=Carcinogens+as+frameshift+mutagens.+Metabolites+and+derivatives+of+2-acetylaminofluorene+and+other+aromatic+amine+carcinogens"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DAmes%26aufirst%3DB.%2BN.%26aulast%3DGurney%26aufirst%3DE.%2BG.%26aulast%3DMiller%26aufirst%3DJ.%2BA.%26aulast%3DBartsch%26aufirst%3DH.%26atitle%3DCarcinogens%2520as%2520frameshift%2520mutagens.%2520Metabolites%2520and%2520derivatives%2520of%25202-acetylaminofluorene%2520and%2520other%2520aromatic%2520amine%2520carcinogens%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D1972%26volume%3D69%26spage%3D3128%26epage%3D3132" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group">Lewis, R. A.</span><span> </span><span class="NLM_article-title">Computer-aided drug design 2005–2007</span> <span class="citation_source-journal">Chem. Modell.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">51</span><span class="NLM_x">–</span> <span class="NLM_lpage">66</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Fjm200452d&amp;key=10.1039%2Fb609116b" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Fjm200452d&amp;key=1%3ACAS%3A528%3ADC%252BD1MXmvFWksg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2008&pages=51-66&author=R.+A.+Lewis&title=Computer-aided+drug+design+2005%E2%80%932007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Computer-aided drug design 2005-2007</span></div><div class="casAuthors">Lewis, Richard A.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Modelling</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">51-66</span>CODEN:
                <span class="NLM_cas:coden">CMHOBG</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A review on QSAR and ADMET, structure-based drug design, virtual screening, de novo structure generation, fragment-based screening, target fishing, and library design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpEqkZ5mtiqf7Vg90H21EOLACvtfcHk0lji7rQHD4Vi_w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXmvFWksg%253D%253D&md5=90468069e401676e1774a88c2d2c8129</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1039%2Fb609116b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fb609116b%26sid%3Dliteratum%253Aachs%26aulast%3DLewis%26aufirst%3DR.%2BA.%26atitle%3DComputer-aided%2520drug%2520design%25202005%25E2%2580%25932007%26jtitle%3DChem.%2520Modell.%26date%3D2008%26volume%3D5%26spage%3D51%26epage%3D66" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group">Lewis, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cucurull-Sanchez, L.</span><span> </span><span class="NLM_article-title">Structural pairwise comparisons of HLM stability of phenyl derivatives: introduction of the Pfizer metabolism index (PMI) and metabolism-lipophilicity efficiency (MLE)</span> <span class="citation_source-journal">J. Comput.-Aided Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">23</span><span class="NLM_x">, </span> <span class="NLM_fpage">97</span><span class="NLM_x">–</span> <span class="NLM_lpage">103</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Fjm200452d&amp;key=10.1007%2Fs10822-008-9242-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Fjm200452d&amp;key=18802667" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Fjm200452d&amp;key=1%3ACAS%3A528%3ADC%252BD1MXmtFGg" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2009&pages=97-103&author=M.+L.+Lewisauthor=L.+Cucurull-Sanchez&title=Structural+pairwise+comparisons+of+HLM+stability+of+phenyl+derivatives%3A+introduction+of+the+Pfizer+metabolism+index+%28PMI%29+and+metabolism-lipophilicity+efficiency+%28MLE%29"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Structural pairwise comparisons of HLM stability of phenyl derivatives: Introduction of the Pfizer metabolism index (PMI) and metabolism-lipophilicity efficiency (MLE)</span></div><div class="casAuthors">Lewis, Mark L.; Cucurull-Sanchez, Lourdes</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">97-103</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Data mining by pairwise comparison of over 150,000 human liver microsome (HLM) intrinsic clearance values stored within the internal Pfizer database has been performed by an automated tool.  Systematic probability tables of specific structural changes on the intrinsic clearance of Ph derivs. were generated.  From these data two new parameters, the Pfizer Metab. Index (PMI) and Metab.-Lipophilicity Efficiency (MLE) are introduced for each fragment.  The findings are applied to a Topliss style anal. that focuses on metabolic stability.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp-t3Rk6sR_vLVg90H21EOLACvtfcHk0lhKAinO9JqZMQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXmtFGg&md5=5088f2c94bce164a302df21936da9a43</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1007%2Fs10822-008-9242-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-008-9242-3%26sid%3Dliteratum%253Aachs%26aulast%3DLewis%26aufirst%3DM.%2BL.%26aulast%3DCucurull-Sanchez%26aufirst%3DL.%26atitle%3DStructural%2520pairwise%2520comparisons%2520of%2520HLM%2520stability%2520of%2520phenyl%2520derivatives%253A%2520introduction%2520of%2520the%2520Pfizer%2520metabolism%2520index%2520%2528PMI%2529%2520and%2520metabolism-lipophilicity%2520efficiency%2520%2528MLE%2529%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2009%26volume%3D23%26spage%3D97%26epage%3D103" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group">Cucurull-Sanchez, L.</span><span> </span><span class="NLM_article-title">Successful identification of key chemical structure modifications that lead to improved ADME profiles</span> <span class="citation_source-journal">J. Comput.-Aided Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">24</span><span class="NLM_x">, </span> <span class="NLM_fpage">449</span><span class="NLM_x">–</span> <span class="NLM_lpage">458</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2010&pages=449-458&author=L.+Cucurull-Sanchez&title=Successful+identification+of+key+chemical+structure+modifications+that+lead+to+improved+ADME+profiles"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DCucurull-Sanchez%26aufirst%3DL.%26atitle%3DSuccessful%2520identification%2520of%2520key%2520chemical%2520structure%2520modifications%2520that%2520lead%2520to%2520improved%2520ADME%2520profiles%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2010%26volume%3D24%26spage%3D449%26epage%3D458" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group">Gall, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCall, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">TenBrink, R. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">VonVoigtlander, P. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mohrland, J. S.</span><span> </span><span class="NLM_article-title">Arylformamidines with antinociceptive properties</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1988</span><span class="NLM_x">, </span> <span class="NLM_volume">31</span><span class="NLM_x">, </span> <span class="NLM_fpage">1816</span><span class="NLM_x">–</span> <span class="NLM_lpage">1820</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm00117a023" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=1988&pages=1816-1820&author=M.+Gallauthor=J.+M.+McCallauthor=R.+E.+TenBrinkauthor=P.+F.+VonVoigtlanderauthor=J.+S.+Mohrland&title=Arylformamidines+with+antinociceptive+properties"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1021%2Fjm00117a023&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm00117a023%26sid%3Dliteratum%253Aachs%26aulast%3DGall%26aufirst%3DM.%26aulast%3DMcCall%26aufirst%3DJ.%2BM.%26aulast%3DTenBrink%26aufirst%3DR.%2BE.%26aulast%3DVonVoigtlander%26aufirst%3DP.%2BF.%26aulast%3DMohrland%26aufirst%3DJ.%2BS.%26atitle%3DArylformamidines%2520with%2520antinociceptive%2520properties%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1988%26volume%3D31%26spage%3D1816%26epage%3D1820" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group">Blaszczak, L. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, R. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cook, G. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hornback, W. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Indelicato, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jordan, C. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Katner, A. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kinnick, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McDonald, J. H.,  III</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morin, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Munroe, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pasini, C. E.</span><span> </span><span class="NLM_article-title">Comparative reactivity of 1-carba-1-dethiacephalosporins with cephalosporins</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1990</span><span class="NLM_x">, </span> <span class="NLM_volume">33</span><span class="NLM_x">, </span> <span class="NLM_fpage">1656</span><span class="NLM_x">–</span> <span class="NLM_lpage">1662</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm00168a019" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=1990&pages=1656-1662&author=L.+C.+Blaszczakauthor=R.+F.+Brownauthor=G.+K.+Cookauthor=W.+J.+Hornbackauthor=J.+M.+Indelicatoauthor=C.+L.+Jordanauthor=A.+S.+Katnerauthor=M.+D.+Kinnickauthor=J.+H.+McDonaldauthor=J.+M.+Morinauthor=J.+E.+Munroeauthor=C.+E.+Pasini&title=Comparative+reactivity+of+1-carba-1-dethiacephalosporins+with+cephalosporins"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Fjm00168a019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm00168a019%26sid%3Dliteratum%253Aachs%26aulast%3DBlaszczak%26aufirst%3DL.%2BC.%26aulast%3DBrown%26aufirst%3DR.%2BF.%26aulast%3DCook%26aufirst%3DG.%2BK.%26aulast%3DHornback%26aufirst%3DW.%2BJ.%26aulast%3DIndelicato%26aufirst%3DJ.%2BM.%26aulast%3DJordan%26aufirst%3DC.%2BL.%26aulast%3DKatner%26aufirst%3DA.%2BS.%26aulast%3DKinnick%26aufirst%3DM.%2BD.%26aulast%3DMcDonald%26aufirst%3DJ.%2BH.%26aulast%3DMorin%26aufirst%3DJ.%2BM.%26aulast%3DMunroe%26aufirst%3DJ.%2BE.%26aulast%3DPasini%26aufirst%3DC.%2BE.%26atitle%3DComparative%2520reactivity%2520of%25201-carba-1-dethiacephalosporins%2520with%2520cephalosporins%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1990%26volume%3D33%26spage%3D1656%26epage%3D1662" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group">Herberich, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cao, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chakrabarti, P. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Falsey, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pettus, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rzasa, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reed, A. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reichelt, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sham, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thaman, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wurz, R. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hsieh, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Syed, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grosfeld, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Plant, M. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Henkle, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sherman, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Middleton, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wong, L. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tasker, A. S.</span><span> </span><span class="NLM_article-title">Discovery of highly selective and potent p38 inhibitors based on a phthalazine scaffold</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">6271</span><span class="NLM_x">–</span> <span class="NLM_lpage">6279</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm8005417" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2008&pages=6271-6279&author=B.+Herberichauthor=G.+Caoauthor=P.+P.+Chakrabartiauthor=J.+R.+Falseyauthor=L.+Pettusauthor=R.+M.+Rzasaauthor=A.+B.+Reedauthor=A.+Reicheltauthor=K.+Shamauthor=M.+Thamanauthor=R.+P.+Wurzauthor=S.+Xuauthor=D.+Zhangauthor=F.+Hsiehauthor=M.+R.+Leeauthor=R.+Syedauthor=V.+Liauthor=D.+Grosfeldauthor=M.+H.+Plantauthor=B.+Henkleauthor=L.+Shermanauthor=S.+Middletonauthor=L.+M.+Wongauthor=A.+S.+Tasker&title=Discovery+of+highly+selective+and+potent+p38+inhibitors+based+on+a+phthalazine+scaffold"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1021%2Fjm8005417&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm8005417%26sid%3Dliteratum%253Aachs%26aulast%3DHerberich%26aufirst%3DB.%26aulast%3DCao%26aufirst%3DG.%26aulast%3DChakrabarti%26aufirst%3DP.%2BP.%26aulast%3DFalsey%26aufirst%3DJ.%2BR.%26aulast%3DPettus%26aufirst%3DL.%26aulast%3DRzasa%26aufirst%3DR.%2BM.%26aulast%3DReed%26aufirst%3DA.%2BB.%26aulast%3DReichelt%26aufirst%3DA.%26aulast%3DSham%26aufirst%3DK.%26aulast%3DThaman%26aufirst%3DM.%26aulast%3DWurz%26aufirst%3DR.%2BP.%26aulast%3DXu%26aufirst%3DS.%26aulast%3DZhang%26aufirst%3DD.%26aulast%3DHsieh%26aufirst%3DF.%26aulast%3DLee%26aufirst%3DM.%2BR.%26aulast%3DSyed%26aufirst%3DR.%26aulast%3DLi%26aufirst%3DV.%26aulast%3DGrosfeld%26aufirst%3DD.%26aulast%3DPlant%26aufirst%3DM.%2BH.%26aulast%3DHenkle%26aufirst%3DB.%26aulast%3DSherman%26aufirst%3DL.%26aulast%3DMiddleton%26aufirst%3DS.%26aulast%3DWong%26aufirst%3DL.%2BM.%26aulast%3DTasker%26aufirst%3DA.%2BS.%26atitle%3DDiscovery%2520of%2520highly%2520selective%2520and%2520potent%2520p38%2520inhibitors%2520based%2520on%2520a%2520phthalazine%2520scaffold%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2008%26volume%3D51%26spage%3D6271%26epage%3D6279" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group">Shi, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sitkoff, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klei, H. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kish, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, E. C. -.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hartl, K. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Seiler, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chang, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Youssef, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Steinbacher, T. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schumacher, W. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grazier, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pudzianowski, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Apedo, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Discenza, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yanchunas, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stein, P. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Atwal, K. S.</span><span> </span><span class="NLM_article-title">Design, structure-activity relationships, X-ray crystal structure, and energetic contributions of a critical P1 pharmacophore: 3-chloroindole-7-yl-based factor Xa inhibitors</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">7541</span><span class="NLM_x">–</span> <span class="NLM_lpage">7551</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm800855x" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2008&pages=7541-7551&author=Y.+Shiauthor=D.+Sitkoffauthor=J.+Zhangauthor=H.+E.+Kleiauthor=K.+Kishauthor=E.+C.+-.+Liuauthor=K.+S.+Hartlauthor=S.+M.+Seilerauthor=M.+Changauthor=C.+Huangauthor=S.+Youssefauthor=T.+E.+Steinbacherauthor=W.+A.+Schumacherauthor=N.+Grazierauthor=A.+Pudzianowskiauthor=A.+Apedoauthor=L.+Discenzaauthor=J.+Yanchunasauthor=P.+D.+Steinauthor=K.+S.+Atwal&title=Design%2C+structure-activity+relationships%2C+X-ray+crystal+structure%2C+and+energetic+contributions+of+a+critical+P1+pharmacophore%3A+3-chloroindole-7-yl-based+factor+Xa+inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Fjm800855x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm800855x%26sid%3Dliteratum%253Aachs%26aulast%3DShi%26aufirst%3DY.%26aulast%3DSitkoff%26aufirst%3DD.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DKlei%26aufirst%3DH.%2BE.%26aulast%3DKish%26aufirst%3DK.%26aulast%3DLiu%26aufirst%3DE.%2BC.%2B-.%26aulast%3DHartl%26aufirst%3DK.%2BS.%26aulast%3DSeiler%26aufirst%3DS.%2BM.%26aulast%3DChang%26aufirst%3DM.%26aulast%3DHuang%26aufirst%3DC.%26aulast%3DYoussef%26aufirst%3DS.%26aulast%3DSteinbacher%26aufirst%3DT.%2BE.%26aulast%3DSchumacher%26aufirst%3DW.%2BA.%26aulast%3DGrazier%26aufirst%3DN.%26aulast%3DPudzianowski%26aufirst%3DA.%26aulast%3DApedo%26aufirst%3DA.%26aulast%3DDiscenza%26aufirst%3DL.%26aulast%3DYanchunas%26aufirst%3DJ.%26aulast%3DStein%26aufirst%3DP.%2BD.%26aulast%3DAtwal%26aufirst%3DK.%2BS.%26atitle%3DDesign%252C%2520structure-activity%2520relationships%252C%2520X-ray%2520crystal%2520structure%252C%2520and%2520energetic%2520contributions%2520of%2520a%2520critical%2520P1%2520pharmacophore%253A%25203-chloroindole-7-yl-based%2520factor%2520Xa%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2008%26volume%3D51%26spage%3D7541%26epage%3D7551" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group">Ioannidis, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lamb, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Almeida, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Block, M. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davies, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peng, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Su, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hoffmann, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rivard, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Green, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Howard, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pollard, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Read, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Alimzhanov, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bebernitz, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bell, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ye, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huszar, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zinda, M.</span><span> </span><span class="NLM_article-title">Discovery of 5-chloro-N2-[(1<i>S</i>)-1-(5-fluoropyrimidin-2-yl)ethyl]-N4-(5-methyl-1<i>H</i>-pyrazol-3-yl)pyrimidine-2,4-diamine (AZD1480) as a novel inhibitor of the Jak/Stat pathway</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">54</span><span class="NLM_x">, </span> <span class="NLM_fpage">262</span><span class="NLM_x">–</span> <span class="NLM_lpage">276</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm1011319" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Fjm200452d&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsFarsr%252FN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=262-276&author=S.+Ioannidisauthor=M.+L.+Lambauthor=T.+Wangauthor=L.+Almeidaauthor=M.+H.+Blockauthor=A.+M.+Daviesauthor=B.+Pengauthor=M.+Suauthor=H.+Zhangauthor=E.+Hoffmannauthor=C.+Rivardauthor=I.+Greenauthor=T.+Howardauthor=H.+Pollardauthor=J.+Readauthor=M.+Alimzhanovauthor=G.+Bebernitzauthor=K.+Bellauthor=M.+Yeauthor=D.+Huszarauthor=M.+Zinda&title=Discovery+of+5-chloro-N2-%5B%281S%29-1-%285-fluoropyrimidin-2-yl%29ethyl%5D-N4-%285-methyl-1H-pyrazol-3-yl%29pyrimidine-2%2C4-diamine+%28AZD1480%29+as+a+novel+inhibitor+of+the+Jak%2FStat+pathway"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of 5-Chloro-N2-[(1S)-1-(5-fluoropyrimidin-2-yl)ethyl]-N4-(5-methyl-1H-pyrazol-3-yl)pyrimidine-2,4-diamine (AZD1480) as a Novel Inhibitor of the Jak/Stat Pathway</span></div><div class="casAuthors">Ioannidis, Stephanos; Lamb, Michelle L.; Wang, Tao; Almeida, Lynsie; Block, Michael H.; Davies, Audrey M.; Peng, Bo; Su, Mei; Zhang, Hai-Jun; Hoffmann, Ethan; Rivard, Caroline; Green, Isabelle; Howard, Tina; Pollard, Hannah; Read, Jon; Alimzhanov, Marat; Bebernitz, Geraldine; Bell, Kirsten; Ye, Min-Wei; Huszar, Dennis; Zinda, Michael</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">262-276</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The myeloproliferative neoplasms, polycythemia vera, essential thrombocythemia, and idiopathic myelofibrosis are a heterogeneous but related group of hematol. malignancies characterized by clonal expansion of one or more myeloid lineages.  The discovery of the Jak2 V617F gain of function mutation highlighted Jak2 as a potential therapeutic target in the MPNs.  Herein, we disclose the discovery of a series of pyrazol-3-yl pyrimidin-4-amines and the identification of 9e (AZD1480) as a potent Jak2 inhibitor.  9E inhibits signaling and proliferation of Jak2 V617F cell lines in vitro, demonstrates in vivo efficacy in a TEL-Jak2 model, has excellent phys. properties and preclin. pharmacokinetics, and is currently being evaluated in Phase I clin. trials.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMG6RQUv4HqbVg90H21EOLACvtfcHk0liAJwxg48Otjw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsFarsr%252FN&md5=c8141c5b0e25a007e68b6e2567a576fe</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Fjm1011319&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm1011319%26sid%3Dliteratum%253Aachs%26aulast%3DIoannidis%26aufirst%3DS.%26aulast%3DLamb%26aufirst%3DM.%2BL.%26aulast%3DWang%26aufirst%3DT.%26aulast%3DAlmeida%26aufirst%3DL.%26aulast%3DBlock%26aufirst%3DM.%2BH.%26aulast%3DDavies%26aufirst%3DA.%2BM.%26aulast%3DPeng%26aufirst%3DB.%26aulast%3DSu%26aufirst%3DM.%26aulast%3DZhang%26aufirst%3DH.%26aulast%3DHoffmann%26aufirst%3DE.%26aulast%3DRivard%26aufirst%3DC.%26aulast%3DGreen%26aufirst%3DI.%26aulast%3DHoward%26aufirst%3DT.%26aulast%3DPollard%26aufirst%3DH.%26aulast%3DRead%26aufirst%3DJ.%26aulast%3DAlimzhanov%26aufirst%3DM.%26aulast%3DBebernitz%26aufirst%3DG.%26aulast%3DBell%26aufirst%3DK.%26aulast%3DYe%26aufirst%3DM.%26aulast%3DHuszar%26aufirst%3DD.%26aulast%3DZinda%26aufirst%3DM.%26atitle%3DDiscovery%2520of%25205-chloro-N2-%255B%25281S%2529-1-%25285-fluoropyrimidin-2-yl%2529ethyl%255D-N4-%25285-methyl-1H-pyrazol-3-yl%2529pyrimidine-2%252C4-diamine%2520%2528AZD1480%2529%2520as%2520a%2520novel%2520inhibitor%2520of%2520the%2520Jak%252FStat%2520pathway%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D262%26epage%3D276" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group">Andrews, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gibson, K. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Graham, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Matusiak, Z. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roberts, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stokes, E. S. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brady, M. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chresta, C. M.</span><span> </span><span class="NLM_article-title">Design and campaign synthesis of pyridine-based histone deacetylase inhibitors</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x">, </span> <span class="NLM_fpage">2525</span><span class="NLM_x">–</span> <span class="NLM_lpage">2529</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2008&pages=2525-2529&author=D.+M.+Andrewsauthor=K.+M.+Gibsonauthor=M.+A.+Grahamauthor=Z.+S.+Matusiakauthor=C.+A.+Robertsauthor=E.+S.+E.+Stokesauthor=M.+C.+Bradyauthor=C.+M.+Chresta&title=Design+and+campaign+synthesis+of+pyridine-based+histone+deacetylase+inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DAndrews%26aufirst%3DD.%2BM.%26aulast%3DGibson%26aufirst%3DK.%2BM.%26aulast%3DGraham%26aufirst%3DM.%2BA.%26aulast%3DMatusiak%26aufirst%3DZ.%2BS.%26aulast%3DRoberts%26aufirst%3DC.%2BA.%26aulast%3DStokes%26aufirst%3DE.%2BS.%2BE.%26aulast%3DBrady%26aufirst%3DM.%2BC.%26aulast%3DChresta%26aufirst%3DC.%2BM.%26atitle%3DDesign%2520and%2520campaign%2520synthesis%2520of%2520pyridine-based%2520histone%2520deacetylase%2520inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2008%26volume%3D18%26spage%3D2525%26epage%3D2529" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group">Weininger, D.</span><span> </span><span class="NLM_article-title">SMILES, a chemical language and information system. 1. Introduction to methodology and encoding rules</span> <span class="citation_source-journal">J. Chem. Inf. Comput. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1988</span><span class="NLM_x">, </span> <span class="NLM_volume">28</span><span class="NLM_x">, </span> <span class="NLM_fpage">31</span><span class="NLM_x">–</span> <span class="NLM_lpage">36</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci00057a005" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Fjm200452d&amp;key=1%3ACAS%3A528%3ADyaL1cXnsVeqsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=1988&pages=31-36&author=D.+Weininger&title=SMILES%2C+a+chemical+language+and+information+system.+1.+Introduction+to+methodology+and+encoding+rules"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">SMILES, a chemical language and information system.  1.  Introduction to methodology and encoding rules</span></div><div class="casAuthors">Weininger, David</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Computer Sciences</span>
        (<span class="NLM_cas:date">1988</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">31-6</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">0095-2338</span>.
    </div><div class="casAbstract">The SMILES (simplified mol. input line entry system) chem. notation system is described for information processing.  The system is based on principles of mol. graph theory and it allows structure specification by use of a very small and natural grammar well suited for high-speed machine processing.  The system is easy to use, has high machine compatibility, and allows many computer applications, including notation generation, const. speed database retrieval, substructure searching, and property prediction models.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpYNGvUnFxDlLVg90H21EOLACvtfcHk0liAJwxg48Otjw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL1cXnsVeqsA%253D%253D&md5=04592975f9dd3c0ce3c1ad618ba2b17d</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1021%2Fci00057a005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci00057a005%26sid%3Dliteratum%253Aachs%26aulast%3DWeininger%26aufirst%3DD.%26atitle%3DSMILES%252C%2520a%2520chemical%2520language%2520and%2520information%2520system.%25201.%2520Introduction%2520to%2520methodology%2520and%2520encoding%2520rules%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Comput.%2520Sci.%26date%3D1988%26volume%3D28%26spage%3D31%26epage%3D36" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group">Andersson, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Armstrong, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bjoere, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bowker, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chapman, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davies, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Donald, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Egner, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Elebring, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Holmqvist, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Inghardt, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johannesson, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johansson, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johnstone, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kemmitt, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kihlberg, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Korsgren, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lemurell, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moore, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pettersson, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pointon, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ponten, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schofield, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Selmi, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Whittamore, P.</span><span> </span><span class="NLM_article-title">Making medicinal chemistry more effective—application of Lean Sigma to improve processes, speed and quality</span> <span class="citation_source-journal">Drug Discovery Today</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x">, </span> <span class="NLM_fpage">598</span><span class="NLM_x">–</span> <span class="NLM_lpage">604</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2009&pages=598-604&author=S.+Anderssonauthor=A.+Armstrongauthor=A.+Bjoereauthor=S.+Bowkerauthor=S.+Chapmanauthor=R.+Daviesauthor=C.+Donaldauthor=B.+Egnerauthor=T.+Elebringauthor=S.+Holmqvistauthor=T.+Inghardtauthor=P.+Johannessonauthor=M.+Johanssonauthor=C.+Johnstoneauthor=P.+Kemmittauthor=J.+Kihlbergauthor=P.+Korsgrenauthor=M.+Lemurellauthor=J.+Mooreauthor=J.+A.+Petterssonauthor=H.+Pointonauthor=F.+Pontenauthor=P.+Schofieldauthor=N.+Selmiauthor=P.+Whittamore&title=Making+medicinal+chemistry+more+effective%E2%80%94application+of+Lean+Sigma+to+improve+processes%2C+speed+and+quality"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DAndersson%26aufirst%3DS.%26aulast%3DArmstrong%26aufirst%3DA.%26aulast%3DBjoere%26aufirst%3DA.%26aulast%3DBowker%26aufirst%3DS.%26aulast%3DChapman%26aufirst%3DS.%26aulast%3DDavies%26aufirst%3DR.%26aulast%3DDonald%26aufirst%3DC.%26aulast%3DEgner%26aufirst%3DB.%26aulast%3DElebring%26aufirst%3DT.%26aulast%3DHolmqvist%26aufirst%3DS.%26aulast%3DInghardt%26aufirst%3DT.%26aulast%3DJohannesson%26aufirst%3DP.%26aulast%3DJohansson%26aufirst%3DM.%26aulast%3DJohnstone%26aufirst%3DC.%26aulast%3DKemmitt%26aufirst%3DP.%26aulast%3DKihlberg%26aufirst%3DJ.%26aulast%3DKorsgren%26aufirst%3DP.%26aulast%3DLemurell%26aufirst%3DM.%26aulast%3DMoore%26aufirst%3DJ.%26aulast%3DPettersson%26aufirst%3DJ.%2BA.%26aulast%3DPointon%26aufirst%3DH.%26aulast%3DPonten%26aufirst%3DF.%26aulast%3DSchofield%26aufirst%3DP.%26aulast%3DSelmi%26aufirst%3DN.%26aulast%3DWhittamore%26aufirst%3DP.%26atitle%3DMaking%2520medicinal%2520chemistry%2520more%2520effective%25E2%2580%2594application%2520of%2520Lean%2520Sigma%2520to%2520improve%2520processes%252C%2520speed%2520and%2520quality%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2009%26volume%3D14%26spage%3D598%26epage%3D604" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group">Birch, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kenny, P. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simpson, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Whittamore, P. R. O.</span><span> </span><span class="NLM_article-title">Matched molecular pair analysis of activity and properties of glycogen phosphorylase inhibitors</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">850</span><span class="NLM_x">–</span> <span class="NLM_lpage">853</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2009&pages=850-853&author=A.+M.+Birchauthor=P.+W.+Kennyauthor=I.+Simpsonauthor=P.+R.+O.+Whittamore&title=Matched+molecular+pair+analysis+of+activity+and+properties+of+glycogen+phosphorylase+inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBirch%26aufirst%3DA.%2BM.%26aulast%3DKenny%26aufirst%3DP.%2BW.%26aulast%3DSimpson%26aufirst%3DI.%26aulast%3DWhittamore%26aufirst%3DP.%2BR.%2BO.%26atitle%3DMatched%2520molecular%2520pair%2520analysis%2520of%2520activity%2520and%2520properties%2520of%2520glycogen%2520phosphorylase%2520inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2009%26volume%3D19%26spage%3D850%26epage%3D853" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group">Morley, A. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kenny, P. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Burton, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heald, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">MacFaul, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mullett, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Page, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Porres, S. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ribeiro, L. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ward, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilkinson, T. J.</span><span> </span><span class="NLM_article-title">5-Aminopyrimidin-2-ylnitriles as cathepsin K inhibitors</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">1658</span><span class="NLM_x">–</span> <span class="NLM_lpage">1661</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2009&pages=1658-1661&author=A.+D.+Morleyauthor=P.+W.+Kennyauthor=B.+Burtonauthor=R.+A.+Healdauthor=P.+A.+MacFaulauthor=J.+Mullettauthor=K.+Pageauthor=S.+S.+Porresauthor=L.+R.+Ribeiroauthor=P.+Smithauthor=S.+Wardauthor=T.+J.+Wilkinson&title=5-Aminopyrimidin-2-ylnitriles+as+cathepsin+K+inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMorley%26aufirst%3DA.%2BD.%26aulast%3DKenny%26aufirst%3DP.%2BW.%26aulast%3DBurton%26aufirst%3DB.%26aulast%3DHeald%26aufirst%3DR.%2BA.%26aulast%3DMacFaul%26aufirst%3DP.%2BA.%26aulast%3DMullett%26aufirst%3DJ.%26aulast%3DPage%26aufirst%3DK.%26aulast%3DPorres%26aufirst%3DS.%2BS.%26aulast%3DRibeiro%26aufirst%3DL.%2BR.%26aulast%3DSmith%26aufirst%3DP.%26aulast%3DWard%26aufirst%3DS.%26aulast%3DWilkinson%26aufirst%3DT.%2BJ.%26atitle%3D5-Aminopyrimidin-2-ylnitriles%2520as%2520cathepsin%2520K%2520inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2009%26volume%3D19%26spage%3D1658%26epage%3D1661" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37a"><span> <span class="citation_source-book">SMARTS</span>; <span class="NLM_publisher-name">Daylight Chemical Information Systems Inc.</span>: <span class="NLM_publisher-loc">Santa Fe, NM</span>, Vol  <span class="NLM_volume">471</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=SMARTS%3B+Daylight+Chemical+Information+Systems+Inc.%3A+Santa+Fe%2C+NM%2C+Vol+471."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit37a&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26btitle%3DSMARTS%26pub%3DDaylight%2520Chemical%2520Information%2520Systems%2520Inc%26volume%3D471" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit37b"><span><a href="http://www.daylight.com/dayhtml/doc/theory/theory.smarts.html" class="extLink">http://www.daylight.com/dayhtml/doc/theory/theory.smarts.html</a>, .</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=http%3A%2F%2Fwww.daylight.com%2Fdayhtml%2Fdoc%2Ftheory%2Ftheory.smarts.html%2C+."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit37b&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweb%26sid%3Dliteratum%253Aachs" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group">Jones, H. D.; Cosgrove, D. A.</span> <span class="citation_source-book">find_pairs</span>; <span class="NLM_publisher-name">AstraZeneca Pharmaceuticals</span>: <span class="NLM_publisher-loc">Macclesfield, U.K.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Jones%2C+H.+D.%3B+Cosgrove%2C+D.+A.find_pairs%3B+AstraZeneca+Pharmaceuticals%3A+Macclesfield%2C+U.K."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DJones%26aufirst%3DH.%2BD.%26btitle%3Dfind_pairs%26pub%3DAstraZeneca%2520Pharmaceuticals" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group">Cosgrove, D. A.</span> <span class="citation_source-book">thrice_pairs</span>; <span class="NLM_publisher-name">AstraZeneca Pharmaceuticals</span>: <span class="NLM_publisher-loc">Macclesfield, U.K.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Cosgrove%2C+D.+A.thrice_pairs%3B+AstraZeneca+Pharmaceuticals%3A+Macclesfield%2C+U.K."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DCosgrove%26aufirst%3DD.%2BA.%26btitle%3Dthrice_pairs%26pub%3DAstraZeneca%2520Pharmaceuticals" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span> <span class="citation_source-book">Pipeline Pilot</span>, version <span class="NLM_edition">6.1.5</span>; <span class="NLM_publisher-name">Accelrys, Inc.</span>: <span class="NLM_publisher-loc">San Diego, CA</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Pipeline+Pilot%2C+version+6.1.5%3B+Accelrys%2C+Inc.%3A+San+Diego%2C+CA."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26btitle%3DPipeline%2520Pilot%26pub%3DAccelrys%252C%2520Inc" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span> <span class="citation_source-book">Daylight</span>, version <span class="NLM_edition">4.83</span>; <span class="NLM_publisher-name">Daylight Chemical Information Systems, Inc.</span>: <span class="NLM_publisher-loc">Laguna Niguel, CA</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Daylight%2C+version+4.83%3B+Daylight+Chemical+Information+Systems%2C+Inc.%3A+Laguna+Niguel%2C+CA."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26btitle%3DDaylight%26pub%3DDaylight%2520Chemical%2520Information%2520Systems%252C%2520Inc" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group">Bemis, G. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murcko, M. A.</span><span> </span><span class="NLM_article-title">The properties of known drugs. 1. Molecular frameworks</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1996</span><span class="NLM_x">, </span> <span class="NLM_volume">39</span><span class="NLM_x">, </span> <span class="NLM_fpage">2887</span><span class="NLM_x">–</span> <span class="NLM_lpage">2893</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm9602928" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Fjm200452d&amp;key=1%3ACAS%3A528%3ADyaK28XjvVejtro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=1996&pages=2887-2893&author=G.+W.+Bemisauthor=M.+A.+Murcko&title=The+properties+of+known+drugs.+1.+Molecular+frameworks"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">The Properties of Known Drugs. 1. Molecular Frameworks</span></div><div class="casAuthors">Bemis, Guy W.; Murcko, Mark A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">39</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">2887-2893</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">To better understand the common features present in drug mols., we use shape description methods to analyze a database of com. available drugs and prep. a list of common drug shapes.  A useful way of organizing this structural data is to group the atoms of each drug mol. into ring, linker, framework, and side chain atoms.  On the basis of the two-dimensional mol. structures (without regard to atom type, hybridization, and bond order), there are 1179 different frameworks among the 5120 compds. analyzed.  However, the shapes of half of the drugs in the database are described by the 32 most frequently occurring frameworks.  This suggests that the diversity of shapes in the set of known drugs is extremely low.  In our second method of anal., in which atom type, hybridization, and bond order are considered, more diversity is seen; there are 2506 different frameworks among the 5120 compds. in the database, and the most frequently occurring 42 frameworks account for only one-fourth of the drugs.  We discuss the possible interpretations of these findings and the way they may be used to guide future drug discovery research.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoodEb3sfDSdrVg90H21EOLACvtfcHk0li2yh45cWt4-g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28XjvVejtro%253D&md5=5e2c4fdfea9434456a0cca83de4185b3</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1021%2Fjm9602928&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm9602928%26sid%3Dliteratum%253Aachs%26aulast%3DBemis%26aufirst%3DG.%2BW.%26aulast%3DMurcko%26aufirst%3DM.%2BA.%26atitle%3DThe%2520properties%2520of%2520known%2520drugs.%25201.%2520Molecular%2520frameworks%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1996%26volume%3D39%26spage%3D2887%26epage%3D2893" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group">Bemis, G. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murcko, M. A.</span><span> </span><span class="NLM_article-title">Properties of known drugs. 2. Side chains</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1999</span><span class="NLM_x">, </span> <span class="NLM_volume">42</span><span class="NLM_x">, </span> <span class="NLM_fpage">5095</span><span class="NLM_x">–</span> <span class="NLM_lpage">5099</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm9903996" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Fjm200452d&amp;key=1%3ACAS%3A528%3ADyaK1MXnsFymtLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=1999&pages=5095-5099&author=G.+W.+Bemisauthor=M.+A.+Murcko&title=Properties+of+known+drugs.+2.+Side+chains"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Properties of Known Drugs. 2. Side Chains</span></div><div class="casAuthors">Bemis, Guy W.; Murcko, Mark A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">25</span>),
    <span class="NLM_cas:pages">5095-5099</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We continue our study of the common features present in drug mols. by looking in detail at drug side chains.  Using shape description methods, we divide a database of com. available drugs into a list of common drug side chains.  On the basis of the atom pair shape descriptor (taking into account atom type, hybridization, and bond order), there are 1,246 different side chains among the 5090 compds. analyzed.  The av. no. of side chains per mol. is 4, and the av. no. of heavy atoms per side chain is 2.  If we ignore the carbonyl side chain, then there are approx. 15,000 occurrences of side chains.  Of these 15,000 approx. 11,000 are from the "top 20" group of side chains.  This suggests that the diversity that side chains provide to drug mols. is quite low.  We discuss ways that this work could be used to provide guidance for mol. design efforts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpWUwPfGwYU3rVg90H21EOLACvtfcHk0lgGL9qVc_FEGQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXnsFymtLw%253D&md5=430bd96d4cf3e1b51bcd0a7515c5ccd5</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1021%2Fjm9903996&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm9903996%26sid%3Dliteratum%253Aachs%26aulast%3DBemis%26aufirst%3DG.%2BW.%26aulast%3DMurcko%26aufirst%3DM.%2BA.%26atitle%3DProperties%2520of%2520known%2520drugs.%25202.%2520Side%2520chains%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1999%26volume%3D42%26spage%3D5095%26epage%3D5099" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group">Haubertin, D. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bruneau, P.</span><span> </span><span class="NLM_article-title">A database of historically-observed chemical replacements</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">1294</span><span class="NLM_x">–</span> <span class="NLM_lpage">1302</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci600395u" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Fjm200452d&amp;key=1%3ACAS%3A528%3ADC%252BD2sXlvFehurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2007&pages=1294-1302&author=D.+Y.+Haubertinauthor=P.+Bruneau&title=A+database+of+historically-observed+chemical+replacements"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">A database of historically-observed chemical replacements</span></div><div class="casAuthors">Haubertin, David Y.; Bruneau, Pierre</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1294-1302</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A systematic anal. of one-to-one chem. replacements occurring in a set of 50 000 druglike mols. was performed.  The frequency of occurrence, as well as the av. change in measured and calcd. properties, was computed for each obsd. substitution.  The exptl. properties considered were soly., protein binding, and logD.  The calcd. properties were logP, mol. wt., no. of hydrogen bond donors and acceptors, and polar surface area.  During this anal., in which 9000 different functional groups were considered, 0.7 million substitutions were identified and stored in a database.  As an application, we present a web interface from which users can identify historically obsd. replacements of any functional group on their query mol.  The server returns a list of side-chains, as well as the historically obsd. shift in exptl. properties.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqYd8AQP_E2A7Vg90H21EOLACvtfcHk0lgGL9qVc_FEGQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXlvFehurc%253D&md5=1e3e7ac2992cae3e48a316bdf0a7e0e2</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1021%2Fci600395u&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci600395u%26sid%3Dliteratum%253Aachs%26aulast%3DHaubertin%26aufirst%3DD.%2BY.%26aulast%3DBruneau%26aufirst%3DP.%26atitle%3DA%2520database%2520of%2520historically-observed%2520chemical%2520replacements%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2007%26volume%3D47%26spage%3D1294%26epage%3D1302" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group">Lewell, X. Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Judd, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Watson, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hann, M.</span><span> </span><span class="NLM_article-title">RECAP—retrosynthetic combinatorial analysis procedure: a powerful new technique for identifying privileged molecular fragments with useful applications in combinatorial chemistry</span> <span class="citation_source-journal">J. Chem. Inf. Comput. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">38</span><span class="NLM_x">, </span> <span class="NLM_fpage">511</span><span class="NLM_x">–</span> <span class="NLM_lpage">522</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci970429i" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Fjm200452d&amp;key=1%3ACAS%3A528%3ADyaK1cXis1yrtbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=1998&pages=511-522&author=X.+Q.+Lewellauthor=D.+Juddauthor=S.+Watsonauthor=M.+Hann&title=RECAP%E2%80%94retrosynthetic+combinatorial+analysis+procedure%3A+a+powerful+new+technique+for+identifying+privileged+molecular+fragments+with+useful+applications+in+combinatorial+chemistry"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">RECAP-Retrosynthetic Combinatorial Analysis Procedure: A Powerful New Technique for Identifying Privileged Molecular Fragments with Useful Applications in Combinatorial Chemistry</span></div><div class="casAuthors">Lewell, Xiao Qing; Judd, Duncan; Watson, Steve; Hann, Mike</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Computer Sciences</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">511-522</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">0095-2338</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The use of combinatorial chem. for the generation of new lead mols. is now a well established strategy in the drug discovery process.  Central to the use of combinatorial chem. is the design and availability of high quality building blocks which are likely to afford hits from the libraries that they generate.  Herein the authors describe "RECAP" (Retrosynthetic Combinatorial Anal. Procedure), a new computational technique designed to address this building block issue.  RECAP electronically fragments mols. based on chem. knowledge.  When applied to databases of biol. active mols., this allows the identification of building block fragments rich in biol. recognized elements and privileged motifs and structures.  This allows the design of building blocks and the synthesis of libraries rich in biol. motifs.  Application of RECAP to the Derwent World Drug Index (WDI) and the mol. fragments/building blocks that this generates are discussed.  The authors also describe a WDI fragment knowledge base which the authors have built which stores the drug motifs and mention its potential application in structure based drug design programs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGovNOpJj5LVU7Vg90H21EOLACvtfcHk0lgGL9qVc_FEGQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXis1yrtbg%253D&md5=7445f76f162006c687116e1e539da8be</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1021%2Fci970429i&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci970429i%26sid%3Dliteratum%253Aachs%26aulast%3DLewell%26aufirst%3DX.%2BQ.%26aulast%3DJudd%26aufirst%3DD.%26aulast%3DWatson%26aufirst%3DS.%26aulast%3DHann%26aufirst%3DM.%26atitle%3DRECAP%25E2%2580%2594retrosynthetic%2520combinatorial%2520analysis%2520procedure%253A%2520a%2520powerful%2520new%2520technique%2520for%2520identifying%2520privileged%2520molecular%2520fragments%2520with%2520useful%2520applications%2520in%2520combinatorial%2520chemistry%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Comput.%2520Sci.%26date%3D1998%26volume%3D38%26spage%3D511%26epage%3D522" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group">Hussain, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rea, C.</span><span> </span><span class="NLM_article-title">Computationally efficient algorithm to identify matched molecular pairs (MMPs) in large data sets</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x">, </span> <span class="NLM_fpage">339</span><span class="NLM_x">–</span> <span class="NLM_lpage">348</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci900450m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Fjm200452d&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtlWltr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2010&pages=339-348&author=J.+Hussainauthor=C.+Rea&title=Computationally+efficient+algorithm+to+identify+matched+molecular+pairs+%28MMPs%29+in+large+data+sets"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Computationally Efficient Algorithm to Identify Matched Molecular Pairs (MMPs) in Large Data Sets</span></div><div class="casAuthors">Hussain, Jameed; Rea, Ceara</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">339-348</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Modern drug discovery organizations generate large vols. of SAR data.  A promising methodol. that can be used to mine this chem. data to identify novel structure-activity relationships is the matched mol. pair (MMP) methodol.  However, before the full potential of the MMP methodol. can be utilized, a MMP identification method that is capable of identifying all MMPs in large chem. data sets on modest computational hardware is required.  In this paper we report an algorithm that is capable of systematically generating all MMPs in chem. data sets.  Addnl., the algorithm is computationally efficient enough to be applied on large data sets.  As an example the algorithm was used to identify the MMPs in the ∼300k NIH MLSMR set.  The algorithm identified ∼5.3 million matched mol. pairs in the set.  These pairs cover ∼2.6 million unique mol. transformations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoF0rN8jaKSjrVg90H21EOLACvtfcHk0lgGL9qVc_FEGQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtlWltr4%253D&md5=23964abebc23274f5c36c319bee52bab</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1021%2Fci900450m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci900450m%26sid%3Dliteratum%253Aachs%26aulast%3DHussain%26aufirst%3DJ.%26aulast%3DRea%26aufirst%3DC.%26atitle%3DComputationally%2520efficient%2520algorithm%2520to%2520identify%2520matched%2520molecular%2520pairs%2520%2528MMPs%2529%2520in%2520large%2520data%2520sets%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2010%26volume%3D50%26spage%3D339%26epage%3D348" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group">Bender, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mussa, H. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Glen, R. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reiling, S.</span><span> </span><span class="NLM_article-title">Molecular similarity searching using atom environments, information-based feature selection, and a naive Bayesian classifier</span> <span class="citation_source-journal">J. Chem. Inf. Comput. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">44</span><span class="NLM_x">, </span> <span class="NLM_fpage">170</span><span class="NLM_x">–</span> <span class="NLM_lpage">178</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci034207y" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Fjm200452d&amp;key=1%3ACAS%3A528%3ADC%252BD3sXhtVSis7vM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2004&pages=170-178&author=A.+Benderauthor=H.+Y.+Mussaauthor=R.+C.+Glenauthor=S.+Reiling&title=Molecular+similarity+searching+using+atom+environments%2C+information-based+feature+selection%2C+and+a+naive+Bayesian+classifier"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular Similarity Searching Using Atom Environments, Information-Based Feature Selection, and a Naive Bayesian Classifier</span></div><div class="casAuthors">Bender, Andreas; Mussa, Hamse Y.; Glen, Robert C.; Reiling, Stephan</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Computer Sciences</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">170-178</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">0095-2338</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A novel technique for similarity searching is introduced.  Mols. are represented by atom environments, which are fed into an information-gain-based feature selection.  A naive Bayesian classifier is then employed for compd. classification.  The new method is tested by its ability to retrieve five sets of active mols. seeded in the MDL Drug Data Report (MDDR).  In comparison expts., the algorithm outperforms all current retrieval methods assessed here using two- and three-dimensional descriptors and offers insight into the significance of structural components for binding.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrgoFUDO5FldLVg90H21EOLACvtfcHk0liaQ12HAAvHpA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXhtVSis7vM&md5=fc422eb289c5bdee1eebe431ef7f336e</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1021%2Fci034207y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci034207y%26sid%3Dliteratum%253Aachs%26aulast%3DBender%26aufirst%3DA.%26aulast%3DMussa%26aufirst%3DH.%2BY.%26aulast%3DGlen%26aufirst%3DR.%2BC.%26aulast%3DReiling%26aufirst%3DS.%26atitle%3DMolecular%2520similarity%2520searching%2520using%2520atom%2520environments%252C%2520information-based%2520feature%2520selection%252C%2520and%2520a%2520naive%2520Bayesian%2520classifier%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Comput.%2520Sci.%26date%3D2004%26volume%3D44%26spage%3D170%26epage%3D178" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span> <span class="citation_source-book">MDDR</span>, version <span class="NLM_edition">1</span>; <span class="NLM_publisher-name">Accelrys, Inc.</span>: <span class="NLM_publisher-loc">San Diego, CA</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=MDDR%2C+version+1%3B+Accelrys%2C+Inc.%3A+San+Diego%2C+CA."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26btitle%3DMDDR%26pub%3DAccelrys%252C%2520Inc" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group">Sheridan, R. P.</span><span> </span><span class="NLM_article-title">The most common chemical replacements in drug-like compounds</span> <span class="citation_source-journal">J. Chem. Inf. Comput. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">42</span><span class="NLM_x">, </span> <span class="NLM_fpage">103</span><span class="NLM_x">–</span> <span class="NLM_lpage">108</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci0100806" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Fjm200452d&amp;key=1%3ACAS%3A528%3ADC%252BD38XlvFWg" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2002&pages=103-108&author=R.+P.+Sheridan&title=The+most+common+chemical+replacements+in+drug-like+compounds"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">The most common chemical replacements in drug-like compounds</span></div><div class="casAuthors">Sheridan, Robert P.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Computer Sciences</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">103-108</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">0095-2338</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We have written a method that exts. one-to-one replacements of chem. groups in pairs of drug-like mols. with the same biol. activity and counts the frequency of the replacements in a large collection of such mols.  There are two variations on the method that differ in their treatment of replacements in rings.  This method is one possible approach to systematically identify candidate bioisosteres.  Here we look at the MDDR database because it has a large diversity of drug-like compds. in a large no. of therapeutic areas.  The most frequent replacements in MDDR seem generally consistent with medicinal chem. intuition about what chem. groups are equiv. or with groups that are easily converted by synthetic or metabolic pathways.  This method can be applied to any set of mols. wherein the mols. can be paired by similar biol. activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoSazOMexul7LVg90H21EOLACvtfcHk0liaQ12HAAvHpA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XlvFWg&md5=6ca09924b4535fdf3cc9adfb3e70ae3d</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1021%2Fci0100806&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci0100806%26sid%3Dliteratum%253Aachs%26aulast%3DSheridan%26aufirst%3DR.%2BP.%26atitle%3DThe%2520most%2520common%2520chemical%2520replacements%2520in%2520drug-like%2520compounds%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Comput.%2520Sci.%26date%3D2002%26volume%3D42%26spage%3D103%26epage%3D108" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group">Sheridan, R. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miller, M. D.</span><span> </span><span class="NLM_article-title">A method for visualizing recurrent topological substructures in sets of active molecules</span> <span class="citation_source-journal">J. Chem. Inf. Comput. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">38</span><span class="NLM_x">, </span> <span class="NLM_fpage">915</span><span class="NLM_x">–</span> <span class="NLM_lpage">924</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci980044f" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Fjm200452d&amp;key=1%3ACAS%3A528%3ADyaK1cXltlyhs7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=1998&pages=915-924&author=R.+P.+Sheridanauthor=M.+D.+Miller&title=A+method+for+visualizing+recurrent+topological+substructures+in+sets+of+active+molecules"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">A Method for Visualizing Recurrent Topological Substructures in Sets of Active Molecules</span></div><div class="casAuthors">Sheridan, Robert P.; Miller, Michael D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Computer Sciences</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">915-924</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">0095-2338</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We present a method for detecting meaningful common topol. substructures among sets of active compds.  A clique-based subgraph detection method is used to find the highest scoring common substructure (HSCS) for each pair of mols.  Only those HSCSs are kept that have a much larger score than would be expected by chance for a randomly selected pair of mols. of the same size.  Information on these significant HSCSs is visualized in two complementary ways.  Individual HSCSs can be displayed as pairs of mols. with the atoms in the HSCS highlighted.  The HSCSs are presented in order of decreasing statistical significance.  Alternatively, individual mols. can be displayed such that each atom is labeled with the no. of significant HSCSs in which it has appeared.  These are presented in order of decreasing max. no. per mol.  Browsing these data gives an impression of what parts of mols. are conserved among actives.  The mols. can be simplified by deleting parts that are not conserved.  We show three examples taken from the MDDR database.  One example highlights common substructures among diverse CCK antagonists.  Two examples point out substructures common between actives on different receptors: benzodiazepine agonists vs. CCK antagonists and antidepressants vs. antihistamines.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGogQqKtvLFZaLVg90H21EOLACvtfcHk0liaQ12HAAvHpA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXltlyhs7Y%253D&md5=b5267c4b1bfa4d52729ba3cc2be88cc7</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1021%2Fci980044f&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci980044f%26sid%3Dliteratum%253Aachs%26aulast%3DSheridan%26aufirst%3DR.%2BP.%26aulast%3DMiller%26aufirst%3DM.%2BD.%26atitle%3DA%2520method%2520for%2520visualizing%2520recurrent%2520topological%2520substructures%2520in%2520sets%2520of%2520active%2520molecules%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Comput.%2520Sci.%26date%3D1998%26volume%3D38%26spage%3D915%26epage%3D924" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group">Warner, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Griffen, E. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">St-Gallay, S. A.</span><span> </span><span class="NLM_article-title">WizePairZ: a novel algorithm to identify, encode, and exploit matched molecular pairs with unspecified cores in medicinal chemistry</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x">, </span> <span class="NLM_fpage">1350</span><span class="NLM_x">–</span> <span class="NLM_lpage">1357</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci100084s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Fjm200452d&amp;key=1%3ACAS%3A528%3ADC%252BC3cXpvVSgs7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2010&pages=1350-1357&author=D.+J.+Warnerauthor=E.+J.+Griffenauthor=S.+A.+St-Gallay&title=WizePairZ%3A+a+novel+algorithm+to+identify%2C+encode%2C+and+exploit+matched+molecular+pairs+with+unspecified+cores+in+medicinal+chemistry"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">WizePairZ: A Novel Algorithm to Identify, Encode, and Exploit Matched Molecular Pairs with Unspecified Cores in Medicinal Chemistry</span></div><div class="casAuthors">Warner, Daniel J.; Griffen, Edward J.; St-Gallay, Stephen A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1350-1357</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">An algorithm to automatically identify and ext. matched mol. pairs from a collection of compds. has been developed, allowing the learning assocd. with each mol. transformation to be readily exploited in drug discovery projects.  Here, we present the application to an example data set of 11 histone deacetylase inhibitors.  The matched pairs were identified, and corresponding differences in activity and lipophilicity were recorded.  These property differences were assocd. with the chem. transformations encoded in the SMIRKS reaction notation.  The transformations identified a subseries with the optimal balance of these two parameters.  Enumeration of a virtual library of compds. using the extd. transformations identified two addnl. compds. initially excluded from the anal. with an accurate estn. of their biol. activity.  We describe how the WizePairZ system can be used to archive and apply medicinal chem. knowledge from one drug discovery project to another as well as identify common bioisosteres.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpSWafs2qQ4UbVg90H21EOLACvtfcHk0ljyFzUkMPaZVA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXpvVSgs7c%253D&md5=9d83eb14cdb56079429761e79beeb292</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1021%2Fci100084s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci100084s%26sid%3Dliteratum%253Aachs%26aulast%3DWarner%26aufirst%3DD.%2BJ.%26aulast%3DGriffen%26aufirst%3DE.%2BJ.%26aulast%3DSt-Gallay%26aufirst%3DS.%2BA.%26atitle%3DWizePairZ%253A%2520a%2520novel%2520algorithm%2520to%2520identify%252C%2520encode%252C%2520and%2520exploit%2520matched%2520molecular%2520pairs%2520with%2520unspecified%2520cores%2520in%2520medicinal%2520chemistry%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2010%26volume%3D50%26spage%3D1350%26epage%3D1357" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group">Southall, N. T.</span><span> </span><span class="NLM_article-title">Ajay kinase patent space visualization using chemical replacements</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">2103</span><span class="NLM_x">–</span> <span class="NLM_lpage">2109</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm051201m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2006&pages=2103-2109&author=N.+T.+Southall&title=Ajay+kinase+patent+space+visualization+using+chemical+replacements"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1021%2Fjm051201m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm051201m%26sid%3Dliteratum%253Aachs%26aulast%3DSouthall%26aufirst%3DN.%2BT.%26atitle%3DAjay%2520kinase%2520patent%2520space%2520visualization%2520using%2520chemical%2520replacements%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2006%26volume%3D49%26spage%3D2103%26epage%3D2109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group">Raymond, J. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Watson, I. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mahoui, A.</span><span> </span><span class="NLM_article-title">Rationalizing lead optimization by associating quantitative relevance with molecular structure modification</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">1952</span><span class="NLM_x">–</span> <span class="NLM_lpage">1962</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci9000426" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Fjm200452d&amp;key=1%3ACAS%3A528%3ADC%252BD1MXosFOjt70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2009&pages=1952-1962&author=J.+W.+Raymondauthor=I.+A.+Watsonauthor=A.+Mahoui&title=Rationalizing+lead+optimization+by+associating+quantitative+relevance+with+molecular+structure+modification"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Rationalizing lead optimization by associating quantitative relevance with molecular structure modification</span></div><div class="casAuthors">Raymond, John W.; Watson, Ian A.; Mahoui, Abdelaziz</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1952-1962</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Historically, one of the characteristic activities of the medicinal chemist has been the iterative improvement of lead compds. until a suitable therapeutic entity is achieved.  Often referred to as lead optimization, this process typically takes the form of minor structural modifications to an existing lead in an attempt to ameliorate deleterious attributes while simultaneously trying to maintain or improve desirable properties.  The cumulative effect of this exercise performed over the course of several decades of pharmaceutical research by thousands of trained researchers has resulted in large collections of pharmaceutically relevant chem. structures.  As far as the authors are aware, this work represents the first attempt to use that data to define a framework to quantifiably catalog and summate this information into a medicinal chem. expert system.  A method is proposed that first comprehensively mines a compendium of chem. structures compiling the structural modifications, abridges them to rectify artificially inflated support levels, and then performs an assocn. rule mining expt. to ascribe relative confidences to each transformation.  The result is a catalog of statistically relevant structural modifications that can potentially be used in a no. of pharmaceutical applications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGprLA8yqcSLSrVg90H21EOLACvtfcHk0ljyFzUkMPaZVA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXosFOjt70%253D&md5=327eb03c9ced8e99d8c0964f2c5c226a</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1021%2Fci9000426&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci9000426%26sid%3Dliteratum%253Aachs%26aulast%3DRaymond%26aufirst%3DJ.%2BW.%26aulast%3DWatson%26aufirst%3DI.%2BA.%26aulast%3DMahoui%26aufirst%3DA.%26atitle%3DRationalizing%2520lead%2520optimization%2520by%2520associating%2520quantitative%2520relevance%2520with%2520molecular%2520structure%2520modification%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2009%26volume%3D49%26spage%3D1952%26epage%3D1962" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54"><span><span class="NLM_contrib-group">Raymond, J. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gardiner, E. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Willett, P.</span><span> </span><span class="NLM_article-title">Heuristics for similarity searching of chemical graphs using a maximum common edge subgraph algorithm</span> <span class="citation_source-journal">J. Chem. Inf. Comput. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">42</span><span class="NLM_x">, </span> <span class="NLM_fpage">305</span><span class="NLM_x">–</span> <span class="NLM_lpage">316</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci010381f" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Fjm200452d&amp;key=1%3ACAS%3A528%3ADC%252BD38XhtVKgt78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2002&pages=305-316&author=J.+W.+Raymondauthor=E.+J.+Gardinerauthor=P.+Willett&title=Heuristics+for+similarity+searching+of+chemical+graphs+using+a+maximum+common+edge+subgraph+algorithm"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Heuristics for similarity searching of chemical graphs using a maximum common edge subgraph algorithm</span></div><div class="casAuthors">Raymond, John W.; Gardiner, Eleanor J.; Willett, Peter</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Computer Sciences</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">305-316</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">0095-2338</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Recently a method (RASCAL) for detg. graph similarity using a max. common edge subgraph algorithm has been proposed which has proven to be very efficient when used to calc. the relative similarity of chem. structures represented as graphs.  This paper describes heuristics which simplify a RASCAL similarity calcn. by taking advantage of certain properties specific to chem. graph representations of mol. structure.  These heuristics are shown exptl. to increase the efficiency of the algorithm, esp. at more distant values of chem. graph similarity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqScTcQRk232bVg90H21EOLACvtfcHk0ljyFzUkMPaZVA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XhtVKgt78%253D&md5=b6af8c4dc19d9c46856b1f93f5d8ebbd</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1021%2Fci010381f&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci010381f%26sid%3Dliteratum%253Aachs%26aulast%3DRaymond%26aufirst%3DJ.%2BW.%26aulast%3DGardiner%26aufirst%3DE.%2BJ.%26aulast%3DWillett%26aufirst%3DP.%26atitle%3DHeuristics%2520for%2520similarity%2520searching%2520of%2520chemical%2520graphs%2520using%2520a%2520maximum%2520common%2520edge%2520subgraph%2520algorithm%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Comput.%2520Sci.%26date%3D2002%26volume%3D42%26spage%3D305%26epage%3D316" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref55"><div class="reference"><strong class="refLabel"><a href="#ref55" class="rightTabRefNumLink">55</a></strong><div class="NLM_citation" id="rightTab-cit55"><span><span class="NLM_contrib-group">Raymond, J. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Willett, P.</span><span> </span><span class="NLM_article-title">Maximum common subgraph isomorphism algorithms for the matching of chemical structures</span> <span class="citation_source-journal">J. Comput.-Aided Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">16</span><span class="NLM_x">, </span> <span class="NLM_fpage">521</span><span class="NLM_x">–</span> <span class="NLM_lpage">533</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2002&pages=521-533&author=J.+W.+Raymondauthor=P.+Willett&title=Maximum+common+subgraph+isomorphism+algorithms+for+the+matching+of+chemical+structures"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DRaymond%26aufirst%3DJ.%2BW.%26aulast%3DWillett%26aufirst%3DP.%26atitle%3DMaximum%2520common%2520subgraph%2520isomorphism%2520algorithms%2520for%2520the%2520matching%2520of%2520chemical%2520structures%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2002%26volume%3D16%26spage%3D521%26epage%3D533" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref56"><div class="reference"><strong class="refLabel"><a href="#ref56" class="rightTabRefNumLink">56</a></strong><div class="NLM_citation" id="rightTab-cit56"><span><span class="NLM_contrib-group">Hu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bajorath, J.</span><span> </span><span class="NLM_article-title">Chemical transformations that yield compounds with distinct activity profiles</span> <span class="citation_source-journal">ACS Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">523</span><span class="NLM_x">–</span> <span class="NLM_lpage">527</span></span><div class="citationLinks">[<a href="/doi/10.1021/ml2000609" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2011&pages=523-527&author=Y.+Huauthor=J.+Bajorath&title=Chemical+transformations+that+yield+compounds+with+distinct+activity+profiles"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1021%2Fml2000609&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fml2000609%26sid%3Dliteratum%253Aachs%26aulast%3DHu%26aufirst%3DY.%26aulast%3DBajorath%26aufirst%3DJ.%26atitle%3DChemical%2520transformations%2520that%2520yield%2520compounds%2520with%2520distinct%2520activity%2520profiles%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2011%26volume%3D2%26spage%3D523%26epage%3D527" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref57"><div class="reference"><strong class="refLabel"><a href="#ref57" class="rightTabRefNumLink">57</a></strong><div class="NLM_citation" id="rightTab-cit57"><span><span class="NLM_contrib-group">Wassermann, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bajorath, J.</span><span> </span><span class="NLM_article-title">Identification of target family directed bioisosteric replacements</span> <span class="citation_source-journal">MedChemComm</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">601</span><span class="NLM_x">–</span> <span class="NLM_lpage">606</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2011&pages=601-606&author=A.+M.+Wassermannauthor=J.+Bajorath&title=Identification+of+target+family+directed+bioisosteric+replacements"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DWassermann%26aufirst%3DA.%2BM.%26aulast%3DBajorath%26aufirst%3DJ.%26atitle%3DIdentification%2520of%2520target%2520family%2520directed%2520bioisosteric%2520replacements%26jtitle%3DMedChemComm%26date%3D2011%26volume%3D2%26spage%3D601%26epage%3D606" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref58"><div class="reference"><strong class="refLabel"><a href="#ref58" class="rightTabRefNumLink">58</a></strong><div class="NLM_citation" id="rightTab-cit58"><span><span class="NLM_contrib-group">Alhambra, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Becker, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blake, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chang, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Damewood, J. R.,  Jr.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Daniels, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dembofsky, B. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gurley, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hall, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Herzog, K. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Horchler, C. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ohnmacht, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schmiesing, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dudley, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ribadeneira, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knappenberger, K. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maciag, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stein, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chopra, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, X. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Christian, E. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arriza, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chapdelaine, M. J.</span><span> </span><span class="NLM_article-title">Development and SAR of functionally selective allosteric modulators of GABAA receptors</span> <span class="citation_source-journal">Bioorg. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">2927</span><span class="NLM_x">–</span> <span class="NLM_lpage">2938</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Fjm200452d&amp;key=10.1016%2Fj.bmc.2011.03.035" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=8&amp;doi=10.1021%2Fjm200452d&amp;key=21498079" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Fjm200452d&amp;key=1%3ACAS%3A528%3ADC%252BC3MXltFOrsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2011&pages=2927-2938&author=C.+Alhambraauthor=C.+Beckerauthor=T.+Blakeauthor=A.+Changauthor=J.+R.+Damewoodauthor=T.+Danielsauthor=B.+T.+Dembofskyauthor=D.+A.+Gurleyauthor=J.+E.+Hallauthor=K.+J.+Herzogauthor=C.+L.+Horchlerauthor=C.+J.+Ohnmachtauthor=R.+J.+Schmiesingauthor=A.+Dudleyauthor=M.+D.+Ribadeneiraauthor=K.+S.+Knappenbergerauthor=C.+Maciagauthor=M.+M.+Steinauthor=M.+Chopraauthor=X.+F.+Liuauthor=E.+P.+Christianauthor=J.+L.+Arrizaauthor=M.+J.+Chapdelaine&title=Development+and+SAR+of+functionally+selective+allosteric+modulators+of+GABAA+receptors"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">Development and SAR of functionally selective allosteric modulators of GABAA receptors</span></div><div class="casAuthors">Alhambra, Cristobal; Becker, Chris; Blake, Timothy; Chang, Amy; Damewood, James R., Jr.; Daniels, Thalia; Dembofsky, Bruce T.; Gurley, David A.; Hall, James E.; Herzog, Keith J.; Horchler, Carey L.; Ohnmacht, Cyrus J.; Schmiesing, Richard Jon; Dudley, Adam; Ribadeneira, Maria D.; Knappenberger, Katherine S.; Maciag, Carla; Stein, Mark M.; Chopra, Maninder; Liu, Xiaodong F.; Christian, Edward P.; Arriza, Jeffrey L.; Chapdelaine, Marc J.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2927-2938</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Pos. modulators at the benzodiazepine site of α2- and α3-contg. GABAA receptors are believed to be anxiolytic.  Through oocyte voltage clamp studies, two series of compds. have been discovered that are pos. modulators at α2-/α3-contg. GABAA receptors and that show no functional activity at α1-contg. GABAA receptors.  Studies to improve this functional selectivity and ultimately deliver clin. candidates are reported.  The functional SAR of cinnolines and quinolines that are pos. allosteric modulators of the α2- and α3-contg. GABAA receptors, while simultaneously neutral antagonists at α1-contg. GABAA receptors, is described.  Such functionally selective modulators of GABAA receptors are expected to be useful in the treatment of anxiety and other psychiatric illnesses.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrVYzfPtmlNCrVg90H21EOLACvtfcHk0ljkt9MkL5jKXA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXltFOrsbg%253D&md5=24eaa34a5d28e2c7e2cfce14f02267fc</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2011.03.035&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2011.03.035%26sid%3Dliteratum%253Aachs%26aulast%3DAlhambra%26aufirst%3DC.%26aulast%3DBecker%26aufirst%3DC.%26aulast%3DBlake%26aufirst%3DT.%26aulast%3DChang%26aufirst%3DA.%26aulast%3DDamewood%26aufirst%3DJ.%2BR.%26aulast%3DDaniels%26aufirst%3DT.%26aulast%3DDembofsky%26aufirst%3DB.%2BT.%26aulast%3DGurley%26aufirst%3DD.%2BA.%26aulast%3DHall%26aufirst%3DJ.%2BE.%26aulast%3DHerzog%26aufirst%3DK.%2BJ.%26aulast%3DHorchler%26aufirst%3DC.%2BL.%26aulast%3DOhnmacht%26aufirst%3DC.%2BJ.%26aulast%3DSchmiesing%26aufirst%3DR.%2BJ.%26aulast%3DDudley%26aufirst%3DA.%26aulast%3DRibadeneira%26aufirst%3DM.%2BD.%26aulast%3DKnappenberger%26aufirst%3DK.%2BS.%26aulast%3DMaciag%26aufirst%3DC.%26aulast%3DStein%26aufirst%3DM.%2BM.%26aulast%3DChopra%26aufirst%3DM.%26aulast%3DLiu%26aufirst%3DX.%2BF.%26aulast%3DChristian%26aufirst%3DE.%2BP.%26aulast%3DArriza%26aufirst%3DJ.%2BL.%26aulast%3DChapdelaine%26aufirst%3DM.%2BJ.%26atitle%3DDevelopment%2520and%2520SAR%2520of%2520functionally%2520selective%2520allosteric%2520modulators%2520of%2520GABAA%2520receptors%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2011%26volume%3D19%26spage%3D2927%26epage%3D2938" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref59"><div class="reference"><strong class="refLabel"><a href="#ref59" class="rightTabRefNumLink">59</a></strong><div class="NLM_citation" id="rightTab-cit59"><span><span class="NLM_contrib-group">Heinonen, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lappalainen, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mielikainen, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rousu, J.</span><span> </span><span class="NLM_article-title">Computing atom mappings for biochemical reactions without subgraph isomorphism</span> <span class="citation_source-journal">J. Comput. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x">, </span> <span class="NLM_fpage">43</span><span class="NLM_x">–</span> <span class="NLM_lpage">58</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Fjm200452d&amp;key=10.1089%2Fcmb.2009.0216" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=8&amp;doi=10.1021%2Fjm200452d&amp;key=21210731" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Fjm200452d&amp;key=1%3ACAS%3A528%3ADC%252BC3MXosFemtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2011&pages=43-58&author=M.+Heinonenauthor=S.+Lappalainenauthor=T.+Mielikainenauthor=J.+Rousu&title=Computing+atom+mappings+for+biochemical+reactions+without+subgraph+isomorphism"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">Computing atom mappings for biochemical reactions without subgraph isomorphism</span></div><div class="casAuthors">Heinonen, Markus; Lappalainen, Sampsa; Mielikainen, Taneli; Rousu, Juho</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Biology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">43-58</span>CODEN:
                <span class="NLM_cas:coden">JCOBEM</span>;
        ISSN:<span class="NLM_cas:issn">1557-8666</span>.
    
            (<span class="NLM_cas:orgname">Mary Ann Liebert, Inc.</span>)
        </div><div class="casAbstract">The ability to trace the fate of individual atoms through the metabolic pathways is needed in many applications of systems biol. and drug discovery.  However, this information is not immediately available from the most common metabolome studies and needs to be sep. acquired.  Automatic discovery of correspondence of atoms in biochem. reactions is called the "atom mapping problem.".  We suggest an efficient approach for solving the atom mapping problem exactly-finding mappings of min. edge edit distance.  The algorithm is based on A* search equipped with sophisticated heuristics for pruning the search space.  This approach has clear advantages over the commonly used heuristic approach of iterative max. common subgraph (MCS) algorithm: we explicitly minimize an objective function, and we produce solns. that typically require less manual curation.  The two methods are similar in computational resource demands.  We compare the performance of the proposed algorithm against several alternatives on data obtained from the KEGG LIGAND and RPAIR databases: greedy search, bi-partite graph matching, and the MCS approach.  Our expts. show that alternative approaches often fail in finding mappings with min. edit distance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr6wVpmn_1T2bVg90H21EOLACvtfcHk0ljkt9MkL5jKXA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXosFemtw%253D%253D&md5=da66598ba846e8da5e02c7ea39fb2c0a</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1089%2Fcmb.2009.0216&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1089%252Fcmb.2009.0216%26sid%3Dliteratum%253Aachs%26aulast%3DHeinonen%26aufirst%3DM.%26aulast%3DLappalainen%26aufirst%3DS.%26aulast%3DMielikainen%26aufirst%3DT.%26aulast%3DRousu%26aufirst%3DJ.%26atitle%3DComputing%2520atom%2520mappings%2520for%2520biochemical%2520reactions%2520without%2520subgraph%2520isomorphism%26jtitle%3DJ.%2520Comput.%2520Biol.%26date%3D2011%26volume%3D18%26spage%3D43%26epage%3D58" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref60"><div class="reference"><strong class="refLabel"><a href="#ref60" class="rightTabRefNumLink">60</a></strong><div class="NLM_citation" id="rightTab-cit60"><span><span class="NLM_contrib-group">Tantillo, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Houk, K. N.</span><span> </span><span class="NLM_article-title">Theozymes and compuzymes: theoretical models for biological catalysis</span> <span class="citation_source-journal">Curr. Opin. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">743</span><span class="NLM_x">–</span> <span class="NLM_lpage">750</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Fjm200452d&amp;key=10.1016%2FS1367-5931%2898%2980112-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=8&amp;doi=10.1021%2Fjm200452d&amp;key=9914196" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Fjm200452d&amp;key=1%3ACAS%3A528%3ADyaK1cXotVWrtbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=1998&pages=743-750&author=D.+J.+Tantilloauthor=J.+Chenauthor=K.+N.+Houk&title=Theozymes+and+compuzymes%3A+theoretical+models+for+biological+catalysis"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">Theozymes and compuzymes: theoretical models for biological catalysis</span></div><div class="casAuthors">Tantillo, Dean J.; Chen, Jiangang; Houk, Kendall N.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Chemical Biology</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">743-750</span>CODEN:
                <span class="NLM_cas:coden">COCBF4</span>;
        ISSN:<span class="NLM_cas:issn">1367-5931</span>.
    
            (<span class="NLM_cas:orgname">Current Biology Publications</span>)
        </div><div class="casAbstract">A review with 32 refs.  A theozyme is a theor. enzyme constructed by computing the optimal geometry for transition-state stabilization by functional groups.  It is created in order to permit quant. assessment of catalytic function.  Theozymes have been used to elucidate the role of transition-state stabilization in the mechanisms underlying enzyme- and antibody-catalyzed hydroxyepoxide cyclizations, eliminations and decarboxylations, peptide and ester hydrolyzes, and pericyclic and radical reactions.  The enzymes studied include orotodine monophosphate decarboxylase, HIV protease and ribonucleotide reductase.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGouKJ8Lt0UwqrVg90H21EOLACvtfcHk0liRacu4ILxM0Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXotVWrtbs%253D&md5=05cf725d2647bdb6d3630dec85311401</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1016%2FS1367-5931%2898%2980112-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1367-5931%252898%252980112-9%26sid%3Dliteratum%253Aachs%26aulast%3DTantillo%26aufirst%3DD.%2BJ.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DHouk%26aufirst%3DK.%2BN.%26atitle%3DTheozymes%2520and%2520compuzymes%253A%2520theoretical%2520models%2520for%2520biological%2520catalysis%26jtitle%3DCurr.%2520Opin.%2520Chem.%2520Biol.%26date%3D1998%26volume%3D2%26spage%3D743%26epage%3D750" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref61"><div class="reference"><strong class="refLabel"><a href="#ref61" class="rightTabRefNumLink">61</a></strong><div class="NLM_citation" id="rightTab-cit61"><span><span class="NLM_contrib-group">Palmer, D. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Llinas, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morao, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Day, G. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Goodman, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Glen, R. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mitchell, J. B. O.</span><span> </span><span class="NLM_article-title">Predicting intrinsic aqueous solubility by a thermodynamic cycle</span> <span class="citation_source-journal">Mol. Pharmaceutics</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">266</span><span class="NLM_x">–</span> <span class="NLM_lpage">279</span></span><div class="citationLinks">[<a href="/doi/10.1021/mp7000878" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Fjm200452d&amp;key=1%3ACAS%3A528%3ADC%252BD1cXitleis74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2008&pages=266-279&author=D.+S.+Palmerauthor=A.+Llinasauthor=I.+Moraoauthor=G.+M.+Dayauthor=J.+M.+Goodmanauthor=R.+C.+Glenauthor=J.+B.+O.+Mitchell&title=Predicting+intrinsic+aqueous+solubility+by+a+thermodynamic+cycle"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">Predicting Intrinsic Aqueous Solubility by a Thermodynamic Cycle</span></div><div class="casAuthors">Palmer, David S.; Llinas, Antonio; Morao, Inaki; Day, Graeme M.; Goodman, Jonathan M.; Glen, Robert C.; Mitchell, John B. O.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Pharmaceutics</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">266-279</span>CODEN:
                <span class="NLM_cas:coden">MPOHBP</span>;
        ISSN:<span class="NLM_cas:issn">1543-8384</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The authors report methods to predict the intrinsic aq. soly. of cryst. org. mols. from two different thermodn. cycles.  Direct computation of soly., via ab initio calcn. of thermodn. quantities at an affordable level of theory, cannot deliver the required accuracy.  Therefore, the authors have turned to a mixt. of direct computation and informatics, using the calcd. thermodn. properties, along with a few other key descriptors, in regression models.  The prediction of log intrinsic soly. (referred to mol/L) by a three-variable linear regression equation gave r2 = 0.77 and RMSE = 0.71 for an external test set comprising drug mols.  The model includes a calcd. crystal lattice energy which provides a computational method to account for the interactions in the solid state.  Probably it is not necessary to know the polymorphic form prior to prediction.  Also, the method developed here may be applicable to other solid-state systems such as salts or cocrystals.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrWUy94arHdGrVg90H21EOLACvtfcHk0liRacu4ILxM0Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXitleis74%253D&md5=71502207fc58378d9cb7e4cb6ad15fd0</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1021%2Fmp7000878&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fmp7000878%26sid%3Dliteratum%253Aachs%26aulast%3DPalmer%26aufirst%3DD.%2BS.%26aulast%3DLlinas%26aufirst%3DA.%26aulast%3DMorao%26aufirst%3DI.%26aulast%3DDay%26aufirst%3DG.%2BM.%26aulast%3DGoodman%26aufirst%3DJ.%2BM.%26aulast%3DGlen%26aufirst%3DR.%2BC.%26aulast%3DMitchell%26aufirst%3DJ.%2BB.%2BO.%26atitle%3DPredicting%2520intrinsic%2520aqueous%2520solubility%2520by%2520a%2520thermodynamic%2520cycle%26jtitle%3DMol.%2520Pharmaceutics%26date%3D2008%26volume%3D5%26spage%3D266%26epage%3D279" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref62"><div class="reference"><strong class="refLabel"><a href="#ref62" class="rightTabRefNumLink">62</a></strong><div class="NLM_citation" id="rightTab-cit62"><span><span class="NLM_contrib-group">Griffen, E.</span><span> </span><span class="NLM_article-title">The rise of the intelligent machines in drug hunting?</span> <span class="citation_source-journal">Future Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">1</span><span class="NLM_x">, </span> <span class="NLM_fpage">405</span><span class="NLM_x">–</span> <span class="NLM_lpage">408</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=16&amp;doi=10.1021%2Fjm200452d&amp;key=10.4155%2Ffmc.09.43" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=8&amp;doi=10.1021%2Fjm200452d&amp;key=21426119" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=32&amp;doi=10.1021%2Fjm200452d&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtFSgur%252FF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2009&pages=405-408&author=E.+Griffen&title=The+rise+of+the+intelligent+machines+in+drug+hunting%3F"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit62R"><div class="casContent"><span class="casTitleNuber">62</span><div class="casTitle"><span class="NLM_cas:atitle">The rise of the intelligent machines in drug hunting?</span></div><div class="casAuthors">Griffen, Ed</div><div class="citationInfo"><span class="NLM_cas:title">Future Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">405-408</span>CODEN:
                <span class="NLM_cas:coden">FMCUA7</span>;
        ISSN:<span class="NLM_cas:issn">1756-8919</span>.
    
            (<span class="NLM_cas:orgname">Future Science Ltd.</span>)
        </div><div class="casAbstract">The impact of automated heuristic generation to drug discovery programs is discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqGdmTklMJKm7Vg90H21EOLACvtfcHk0liRacu4ILxM0Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtFSgur%252FF&md5=89e9aadb119a28e621e7f50310c433a4</span></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.4155%2Ffmc.09.43&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4155%252Ffmc.09.43%26sid%3Dliteratum%253Aachs%26aulast%3DGriffen%26aufirst%3DE.%26atitle%3DThe%2520rise%2520of%2520the%2520intelligent%2520machines%2520in%2520drug%2520hunting%253F%26jtitle%3DFuture%2520Med.%2520Chem.%26date%3D2009%26volume%3D1%26spage%3D405%26epage%3D408" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Fjm200452d&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2011.54.issue-22%3Bwgroup%3Astring%3AACHS+website+Group%3Barticle%3Aarticle%3A10.1021%2Fjm200452d%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/jm200452d" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                3MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"677a7f912c563cc4","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
