
<html lang="en"     class="pb-page"  data-request-id="c8107555-9923-42a4-baa0-17337c4f1053"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;article:article:10.1021/acs.jmedchem.0c01705;wgroup:string:ACHS website Group;website:website:acspubs;issue:issue:10.1021/jmcmar.2021.64.issue-1;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Discovery of Novel Azetidine Amides as Potent Small-Molecule STAT3 Inhibitors" /></meta><meta name="dc.Creator" content="Christine  Brotherton-Pleiss" /></meta><meta name="dc.Creator" content="Peibin  Yue" /></meta><meta name="dc.Creator" content="Yinsong  Zhu" /></meta><meta name="dc.Creator" content="Kayo  Nakamura" /></meta><meta name="dc.Creator" content="Weiliang  Chen" /></meta><meta name="dc.Creator" content="Wenzhen  Fu" /></meta><meta name="dc.Creator" content="Casie  Kubota" /></meta><meta name="dc.Creator" content="Jasmine  Chen" /></meta><meta name="dc.Creator" content="Felix  Alonso-Valenteen" /></meta><meta name="dc.Creator" content="Simoun  Mikhael" /></meta><meta name="dc.Creator" content="Lali  Medina-Kauwe" /></meta><meta name="dc.Creator" content="Marcus A.  Tius" /></meta><meta name="dc.Creator" content="Francisco  Lopez-Tapia" /></meta><meta name="dc.Creator" content="James  Turkson" /></meta><meta name="dc.Description" content="We optimized our previously reported proline-based STAT3 inhibitors into an exciting new series of (R)-azetidine-2-carboxamide analogues that have sub-micromolar potencies. 5a, 5o, and 8i have STAT..." /></meta><meta name="Description" content="We optimized our previously reported proline-based STAT3 inhibitors into an exciting new series of (R)-azetidine-2-carboxamide analogues that have sub-micromolar potencies. 5a, 5o, and 8i have STAT..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="December 22, 2020" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.0c01705" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2020 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c01705" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.0c01705" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c01705" /></link>
        
    
    

<title>Discovery of Novel Azetidine Amides as Potent Small-Molecule STAT3 Inhibitors | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.0c01705" /></meta><meta property="og:title" content="Discovery of Novel Azetidine Amides as Potent Small-Molecule STAT3 Inhibitors" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01705/20210107/images/large/jm0c01705_0025.jpeg" /></meta><meta property="og:description" content="We optimized our previously reported proline-based STAT3 inhibitors into an exciting new series of (R)-azetidine-2-carboxamide analogues that have sub-micromolar potencies. 5a, 5o, and 8i have STAT3-inhibitory potencies (IC50) of 0.55, 0.38, and 0.34 μM, respectively, compared to potencies greater than 18 μM against STAT1 or STAT5 activity. Further modifications derived analogues, including 7e, 7f, 7g, and 9k, that addressed cell membrane permeability and other physicochemical issues. Isothermal titration calorimetry analysis confirmed high-affinity binding to STAT3, with KD of 880 nM (7g) and 960 nM (9k). 7g and 9k inhibited constitutive STAT3 phosphorylation and DNA-binding activity in human breast cancer, MDA-MB-231 or MDA-MB-468 cells. Furthermore, treatment of breast cancer cells with 7e, 7f, 7g, or 9k inhibited viable cells, with an EC50 of 0.9–1.9 μM, cell growth, and colony survival, and induced apoptosis while having relatively weaker effects on normal breast epithelial, MCF-10A or breast cancer, MCF-7 cells that do not harbor constitutively active STAT3." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.0c01705"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c01705">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-14/achre4.2021.54.issue-14/20210720/achre4.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-29/aamick.2021.13.issue-29/20210728/aamick.2021.13.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-14/acncdm.2021.12.issue-14/20210721/acncdm.2021.12.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-7/amlcef.2021.3.issue-7/20210705/amlcef.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-29/acsodf.2021.6.issue-29/20210727/acsodf.2021.6.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-29/ascecg.2021.9.issue-29/20210726/ascecg.2021.9.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-29/ancham.2021.93.issue-29/20210727/ancham.2021.93.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-29/bichaw.2021.60.issue-29/20210727/bichaw.2021.60.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-27/cgeabj.2021.99.issue-27/20210726-01/cgeabj.2021.99.issue-27.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-7/cgdefu.2021.21.issue-7/20210707/cgdefu.2021.21.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-14/esthag.2021.55.issue-14/20210720/esthag.2021.55.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-29/iecred.2021.60.issue-29/20210728/iecred.2021.60.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-14/inocaj.2021.60.issue-14/20210719/inocaj.2021.60.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-29/jafcau.2021.69.issue-29/20210728/jafcau.2021.69.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-29/jacsat.2021.143.issue-29/20210728/jacsat.2021.143.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-7/jamsef.2021.32.issue-7/20210707/jamsef.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-29/langd5.2021.37.issue-29/20210727/langd5.2021.37.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-7/mpohbp.2021.18.issue-7/20210705/mpohbp.2021.18.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.0c01705&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.0c01705&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.0c01705&amp;href=/doi/10.1021/acs.jmedchem.0c01705" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2021</span><span class="cit-fg-volume">, 64</span><span class="cit-fg-issue">, 1</span><span class="cit-fg-pageRange">, 695-710</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/64/1" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.0c01698" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.0c01711" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Discovery of Novel Azetidine Amides as Potent Small-Molecule STAT3 Inhibitors</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Christine Brotherton-Pleiss</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Christine Brotherton-Pleiss</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Cancer Biology Program, University of Hawaii Cancer Center, University of Hawaii, Manoa, Honolulu, Hawaii 96813, United States</div><div class="loa-info-affiliations-info">Medicinal Chemistry Leader, Department of Chemistry, University of Hawaii, Manoa, 2545 McCarthy Mall, Honolulu, Hawaii 9682, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Christine++Brotherton-Pleiss">Christine Brotherton-Pleiss</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Peibin Yue</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Peibin Yue</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Medicine, Division of Oncology,  Cedars-Sinai Cancer, Cedars-Sinai Medical Center, 8700 Beverly Blvd, Davis 5065, Los Angeles, California 90048, United States</div><div class="loa-info-affiliations-info">Cancer Biology Program, University of Hawaii Cancer Center, University of Hawaii, Manoa, Honolulu, Hawaii 96813, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Peibin++Yue">Peibin Yue</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Yinsong Zhu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Yinsong Zhu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Medicine, Division of Oncology,  Cedars-Sinai Cancer, Cedars-Sinai Medical Center, 8700 Beverly Blvd, Davis 5065, Los Angeles, California 90048, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Yinsong++Zhu">Yinsong Zhu</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Kayo Nakamura</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Kayo Nakamura</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemistry, University of Hawaii, Manoa, 2545 McCarthy Mall, Honolulu, Hawaii 9682, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Kayo++Nakamura">Kayo Nakamura</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Weiliang Chen</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Weiliang Chen</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemistry, University of Hawaii, Manoa, 2545 McCarthy Mall, Honolulu, Hawaii 9682, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Weiliang++Chen">Weiliang Chen</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Wenzhen Fu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Wenzhen Fu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Cancer Biology Program, University of Hawaii Cancer Center, University of Hawaii, Manoa, Honolulu, Hawaii 96813, United States</div><div class="loa-info-affiliations-info">Department of Chemistry, University of Hawaii, Manoa, 2545 McCarthy Mall, Honolulu, Hawaii 9682, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Wenzhen++Fu">Wenzhen Fu</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Casie Kubota</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Casie Kubota</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Cancer Biology Program, University of Hawaii Cancer Center, University of Hawaii, Manoa, Honolulu, Hawaii 96813, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Casie++Kubota">Casie Kubota</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Jasmine Chen</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jasmine Chen</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Cancer Biology Program, University of Hawaii Cancer Center, University of Hawaii, Manoa, Honolulu, Hawaii 96813, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jasmine++Chen">Jasmine Chen</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Felix Alonso-Valenteen</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Felix Alonso-Valenteen</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Cedars-Sinai Cancer, Cedars-Sinai Medical Center, 8700 Beverly Blvd, Davis 5065, Los Angeles, California 90048, United States</div><div class="loa-info-affiliations-info">Department of Biomedical Sciences, Cedars-Sinai Medical Center, Los Angeles, California 90048, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Felix++Alonso-Valenteen">Felix Alonso-Valenteen</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Simoun Mikhael</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Simoun Mikhael</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Cedars-Sinai Cancer, Cedars-Sinai Medical Center, 8700 Beverly Blvd, Davis 5065, Los Angeles, California 90048, United States</div><div class="loa-info-affiliations-info">Department of Biomedical Sciences, Cedars-Sinai Medical Center, Los Angeles, California 90048, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Simoun++Mikhael">Simoun Mikhael</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Lali Medina-Kauwe</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Lali Medina-Kauwe</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Cedars-Sinai Cancer, Cedars-Sinai Medical Center, 8700 Beverly Blvd, Davis 5065, Los Angeles, California 90048, United States</div><div class="loa-info-affiliations-info">Department of Biomedical Sciences, Cedars-Sinai Medical Center, Los Angeles, California 90048, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Lali++Medina-Kauwe">Lali Medina-Kauwe</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Marcus A. Tius</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Marcus A. Tius</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Cancer Biology Program, University of Hawaii Cancer Center, University of Hawaii, Manoa, Honolulu, Hawaii 96813, United States</div><div class="loa-info-affiliations-info">Medicinal Chemistry Leader, Department of Chemistry, University of Hawaii, Manoa, 2545 McCarthy Mall, Honolulu, Hawaii 9682, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Marcus+A.++Tius">Marcus A. Tius</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-7092-8993" title="Orcid link">http://orcid.org/0000-0002-7092-8993</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Francisco Lopez-Tapia</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Francisco Lopez-Tapia</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Cancer Biology Program, University of Hawaii Cancer Center, University of Hawaii, Manoa, Honolulu, Hawaii 96813, United States</div><div class="loa-info-affiliations-info">Medicinal Chemistry Leader, Department of Chemistry, University of Hawaii, Manoa, 2545 McCarthy Mall, Honolulu, Hawaii 9682, United States</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#fb9d899a959892889894d597948b9e81bb98889388d594899c"><span class="__cf_email__" data-cfemail="dcbaaebdb2bfb5afbfb3f2b0b3acb9a69cbfafb4aff2b3aebb">[email protected]</span></a></span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Francisco++Lopez-Tapia">Francisco Lopez-Tapia</a></span></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">James Turkson</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">James Turkson</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Medicine, Division of Oncology,  Cedars-Sinai Cancer, Cedars-Sinai Medical Center, 8700 Beverly Blvd, Davis 5065, Los Angeles, California 90048, United States</div><div class="loa-info-affiliations-info">Cancer Biology Program, University of Hawaii Cancer Center, University of Hawaii, Manoa, Honolulu, Hawaii 96813, United States</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#1b717a767e68356f6e69706874755b786873683574697c"><span class="__cf_email__" data-cfemail="402a212d25336e3435322b332f2e00233328336e2f3227">[email protected]</span></a>. Tel: 310-423-6887.</span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=James++Turkson">James Turkson</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-6964-1602" title="Orcid link">http://orcid.org/0000-0002-6964-1602</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c01705&amp;href=/doi/10.1021%2Facs.jmedchem.0c01705" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2021</span></span><span class="cit-volume">, 64</span><span class="cit-issue">, 1</span><span class="cit-pageRange">, 695–710</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">December 22, 2020</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>28 September 2020</li><li><span class="item_label"><b>Published</b> online</span>22 December 2020</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 14 January 2021</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.0c01705" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c01705</a></div><div class="article_header-article-copyright"><strong>Copyright © 2020 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D695%26pageCount%3D16%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DChristine%2BBrotherton-Pleiss%252C%2BPeibin%2BYue%252C%2BYinsong%2BZhu%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D64%26issueNum%3D1%26contentID%3Dacs.jmedchem.0c01705%26title%3DDiscovery%2Bof%2BNovel%2BAzetidine%2BAmides%2Bas%2BPotent%2BSmall-Molecule%2BSTAT3%2BInhibitors%26numPages%3D16%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D710%26publicationDate%3DJanuary%2B2021">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.0c01705"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">1134</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">2</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.0c01705" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Discovery of Novel Azetidine Amides as Potent Small-Molecule STAT3 Inhibitors&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Christine&quot;,&quot;last_name&quot;:&quot;Brotherton-Pleiss&quot;},{&quot;first_name&quot;:&quot;Peibin&quot;,&quot;last_name&quot;:&quot;Yue&quot;},{&quot;first_name&quot;:&quot;Yinsong&quot;,&quot;last_name&quot;:&quot;Zhu&quot;},{&quot;first_name&quot;:&quot;Kayo&quot;,&quot;last_name&quot;:&quot;Nakamura&quot;},{&quot;first_name&quot;:&quot;Weiliang&quot;,&quot;last_name&quot;:&quot;Chen&quot;},{&quot;first_name&quot;:&quot;Wenzhen&quot;,&quot;last_name&quot;:&quot;Fu&quot;},{&quot;first_name&quot;:&quot;Casie&quot;,&quot;last_name&quot;:&quot;Kubota&quot;},{&quot;first_name&quot;:&quot;Jasmine&quot;,&quot;last_name&quot;:&quot;Chen&quot;},{&quot;first_name&quot;:&quot;Felix&quot;,&quot;last_name&quot;:&quot;Alonso-Valenteen&quot;},{&quot;first_name&quot;:&quot;Simoun&quot;,&quot;last_name&quot;:&quot;Mikhael&quot;},{&quot;first_name&quot;:&quot;Lali&quot;,&quot;last_name&quot;:&quot;Medina-Kauwe&quot;},{&quot;first_name&quot;:&quot;Marcus&quot;,&quot;last_name&quot;:&quot;A. Tius&quot;},{&quot;first_name&quot;:&quot;Francisco&quot;,&quot;last_name&quot;:&quot;Lopez-Tapia&quot;},{&quot;first_name&quot;:&quot;James&quot;,&quot;last_name&quot;:&quot;Turkson&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2020&quot;,&quot;month&quot;:&quot;12&quot;,&quot;day&quot;:&quot;22&quot;,&quot;issue&quot;:&quot;1&quot;,&quot;volume&quot;:&quot;64&quot;,&quot;pages&quot;:&quot;695-710&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.0c01705&quot;},&quot;abstract&quot;:&quot;We optimized our previously reported proline-based STAT3 inhibitors into an exciting new series of (R)-azetidine-2-carboxamide analogues that have sub-micromolar potencies. 5a, 5o, and 8i have STAT3-inhibitory potencies (IC50) of 0.55, 0.38, and 0.34 μM, respectively, compared to potencies greater than 18 μM against STAT1 or STAT5 activity. Further modifications derived analogues, including 7e, 7f, 7g, and 9k, that addressed cell membrane permeability and other physicochemical issues. Isothermal titration calorimetry analysis confirmed high-affinity binding to STAT3, with KD of 880 nM (7g) and 960 nM (9k). 7g and 9k inhibited constitutive STAT3 phosphorylation and DNA-binding activity in human breast cancer, MDA-MB-231 or MDA-MB-468 cells. Furthermore, treatment of breast cancer cells with 7e, 7f, 7g, or 9k inhibited viable cells, with an EC50 of 0.9–1.9 μM, cell growth, and colony survival, and induced apoptosis while having relatively weaker effects on normal breast epithelial, MCF-10A or breast cancer, MCF&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c01705&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c01705" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c01705&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c01705" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c01705&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c01705" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c01705&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c01705&amp;href=/doi/10.1021/acs.jmedchem.0c01705" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.0c01705" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.0c01705" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (8 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c01705&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.0c01705%26sid%3Dliteratum%253Aachs%26pmid%3D33352047%26genre%3Darticle%26aulast%3DBrotherton-Pleiss%26date%3D2021%26atitle%3DDiscovery%2Bof%2BNovel%2BAzetidine%2BAmides%2Bas%2BPotent%2BSmall-Molecule%2BSTAT3%2BInhibitors%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D64%26issue%3D1%26spage%3D695%26epage%3D710%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (9)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291350" title="Assays">Assays</a>,</li><li><a href="/action/doSearch?ConceptID=290896" title="Cells">Cells</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=292258" title="Cancer">Cancer</a>,</li><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/64/1" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/jmcmar.2021.64.issue-1/20210114/jmcmar.2021.64.issue-1.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01705/20210107/images/medium/jm0c01705_0025.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01705/20210107/images/large/jm0c01705_0025.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01705&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">We optimized our previously reported proline-based STAT3 inhibitors into an exciting new series of (<i>R</i>)-azetidine-2-carboxamide analogues that have sub-micromolar potencies. <b><u class="uu">5a</u>, <u class="uu">5o</u></b>, and <b><u class="uu">8i</u></b> have STAT3-inhibitory potencies (IC<sub>50</sub>) of 0.55, 0.38, and 0.34 μM, respectively, compared to potencies greater than 18 μM against STAT1 or STAT5 activity. Further modifications derived analogues, including <b><u class="uu">7e</u></b>, <b><u class="uu">7f</u></b>, <b><u class="uu">7g</u></b>, and <b><u class="uu">9k</u></b>, that addressed cell membrane permeability and other physicochemical issues. Isothermal titration calorimetry analysis confirmed high-affinity binding to STAT3, with <i>K</i><sub>D</sub> of 880 nM (<b><u class="uu">7g</u></b>) and 960 nM (<b><u class="uu">9k</u></b>). <b><u class="uu">7g</u></b> and <b><u class="uu">9k</u></b> inhibited constitutive STAT3 phosphorylation and DNA-binding activity in human breast cancer, MDA-MB-231 or MDA-MB-468 cells. Furthermore, treatment of breast cancer cells with <b><u class="uu">7e</u></b>, <b><u class="uu">7f</u></b>, <b><u class="uu">7g</u></b>, or <b><u class="uu">9k</u></b> inhibited viable cells, with an EC<sub>50</sub> of 0.9–1.9 μM, cell growth, and colony survival, and induced apoptosis while having relatively weaker effects on normal breast epithelial, MCF-10A or breast cancer, MCF-7 cells that do not harbor constitutively active STAT3.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_21453" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_21453" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">The signal transducer and activator of transcription (STAT) family of cytoplasmic transcription factors mediate the cellular responses to cytokine and growth factors, including cell growth and differentiation, inflammation, and immune responses.<a onclick="showRef(event, 'ref1 ref2 ref3'); return false;" href="javascript:void(0);" class="ref ref1 ref2 ref3">(1−3)</a> STAT activation is initiated upon receptor–ligand binding that induces STAT recruitment to the receptor and STAT phosphorylation by Janus kinases (JAKs) and Src family kinases. Two phospho-STAT monomer proteins form STAT:STAT dimers through a reciprocal pTyr-Src homology (SH)2 domain interactions, translocate to the nucleus, and bind to specific DNA-response elements in target gene promoters to induce gene transcription.<a onclick="showRef(event, 'ref2 ref3 ref4 ref5'); return false;" href="javascript:void(0);" class="ref ref2 ref3 ref4 ref5">(2−5)</a></div><div class="NLM_p">While normal STAT activation is rapid and transient, aberrant activation of one family member, STAT3, is prevalent and has a causal role in many human cancers.<a onclick="showRef(event, 'ref6 ref7'); return false;" href="javascript:void(0);" class="ref ref6 ref7">(6,7)</a> STAT3 is therefore a valid and an attractive target for the development of novel anticancer therapeutics.<a onclick="showRef(event, 'ref7 ref8 ref9 ref10'); return false;" href="javascript:void(0);" class="ref ref7 ref8 ref9 ref10">(7−10)</a> One of the main approaches to develop inhibitors of STAT3 is focused on targeting the key pTyr:SH2 domain interaction and the STAT3:STAT3 dimerization event.<a onclick="showRef(event, 'ref8 ref9'); return false;" href="javascript:void(0);" class="ref ref8 ref9">(8,9)</a> Several small-molecule inhibitors have been developed that target the STAT3 SH2 domain and disrupt the STAT3:STAT3 dimerization.<a onclick="showRef(event, 'ref11 ref12 ref13 ref14 ref15 ref16 ref17 ref18 ref19 ref20 ref21 ref22 ref23 ref24 ref25 ref26'); return false;" href="javascript:void(0);" class="ref ref11 ref12 ref13 ref14 ref15 ref16 ref17 ref18 ref19 ref20 ref21 ref22 ref23 ref24 ref25 ref26">(11−26)</a> However, there has been limited success in the advancement of these STAT3 inhibitors into clinical application due to low potency and pharmacokinetic (PK) limitations and toxicity.<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> More recently, a new class of inhibitors of the STAT3 signaling pathway has emerged that focuses on protein degradation. The PROTAC-STAT3 degraders, represented by SD-36, promoted STAT3 degradation with nanomolar potency, and SD-36 induced a complete tumor growth inhibition in vivo in multiple tumor models.<a onclick="showRef(event, 'ref28 ref29'); return false;" href="javascript:void(0);" class="ref ref28 ref29">(28,29)</a></div><div class="NLM_p last">We have previously provided proof-of-concept for the in vivo antitumor efficacy of the micromolar potent lead inhibitors, BP-1-102, SH5-07, and SH4-54, which are based on the <i>N</i>-methylglycinamide scaffold, with its two amine moieties condensed with three different functionalities.<a onclick="showRef(event, 'ref24 ref26'); return false;" href="javascript:void(0);" class="ref ref24 ref26">(24,26)</a> We took steps to address the challenges with potency and PK properties and hence advance the development of these amino acid amide-based inhibitors and recently published an extensive study on the structure–activity relationship (SAR) analysis using an iterative medicinal chemistry approach in which the amino acid linker was varied along with the simultaneous optimization of the three functionalities to improve potency and physicochemical properties, and this led to new proline-based analogues.<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> With the focus on potency, we further extended the optimization of the proline-based analogues into other cyclic amino acids and have now derived more exciting new series of (<i>R</i>)-azetidine-2-carboxamide analogues of BP-1-102, including <b><u class="uu">5a</u></b>, <b><u class="uu">5o</u></b>, and <b><u class="uu">8i</u></b>, which show sub-micromolar STAT3-inhibitory activity in vitro (IC<sub>50</sub> values of 0.52, 0.38, and 0.34 μM, respectively). To improve membrane permeability, we further derived analogues containing carboxylic acid surrogates, <b><u class="uu">7e</u></b>, <b><u class="uu">7f</u></b>, <b><u class="uu">7g</u></b>, and <b><u class="uu">9k</u></b>, which at 1 μM or less strongly inhibited the viability, anchorage-dependent and independent growth, and colony formation of MDA-MB-231 or MDA-MB-468 human breast cancer cells that harbor aberrantly active STAT3.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_34344" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_34344" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> Progression from the Proline Linker: SAR of Early Azetidine Analogues Shows Nanomolar Potency in Disrupting STAT3 DNA-Binding Activity In Vitro</h3><div class="NLM_p">Progression from the previously reported proline linker (Supporting Information <a href="/doi/suppl/10.1021/acs.jmedchem.0c01705/suppl_file/jm0c01705_si_001.pdf" class="ext-link">Figure S1</a>, <b><u class="uu">3</u></b>)<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> into other cyclic amino acid linkers led us to discover more potent inhibitors of STAT3 activity, as measured by DNA-binding activity/electrophoretic mobility shift assay (EMSA). In this assay, nuclear extracts containing active STAT3:STAT3 prepared from cancer cells are preincubated with increasing concentrations of the compounds at room temperature for 30 min, prior to incubation with the radiolabeled high-affinity <i>sis</i>-inducible element (hSIE) probe (from the <i>c</i>-fos promoter) that binds active STAT3 and subjecting to EMSA analysis.<a onclick="showRef(event, 'ref23 ref24 ref26'); return false;" href="javascript:void(0);" class="ref ref23 ref24 ref26">(23,24,26)</a> The basic premise is that the binding of the high-affinity compounds to STAT3 inhibits STAT3 DNA-binding activity, as shown in the past.<a onclick="showRef(event, 'ref13 ref14 ref23 ref24 ref26 ref30'); return false;" href="javascript:void(0);" class="ref ref13 ref14 ref23 ref24 ref26 ref30">(13,14,23,24,26,30)</a> The bands corresponding to the DNA-bound STAT3 are scanned, quantified by ImageJ, and represented as percent of control (100%), which are plotted against the concentration of the compounds. Representative plots are shown in the Supporting Information <a href="/doi/suppl/10.1021/acs.jmedchem.0c01705/suppl_file/jm0c01705_si_002.csv" class="ext-link">Figure S2</a>. Although changing the 5-membered proline analogue, <b><u class="uu">3</u></b>, to the corresponding 6-membered, pipecolamide analogue, <b><u class="uu">4</u></b>, decreased STAT3-inhibitory potency from EMSA IC<sub>50</sub> 2.4 μM for <b><u class="uu">3</u></b> to IC<sub>50</sub> 5.4 μM for <b><u class="uu">4</u></b> (Supporting Information <a href="/doi/suppl/10.1021/acs.jmedchem.0c01705/suppl_file/jm0c01705_si_003.pdf" class="ext-link">Figure S3</a>), changing to the 4-membered azetidine-2-carboxamide analogue, <b><u class="uu">5a</u></b> (Supporting Information <a href="/doi/suppl/10.1021/acs.jmedchem.0c01705/suppl_file/jm0c01705_si_003.pdf" class="ext-link">Figure S3</a>), gave over a 4-fold boost in potency in vitro over proline, <b><u class="uu">3</u></b>, against STAT3 DNA-binding activity (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>A). The concentration at which there is 50% inhibition of STAT3 DNA-binding activity relative to the dimethyl sulfoxide (DMSO)-treated control in the EMSA analysis, IC<sub>50</sub>, is 0.52 μM for <b><u class="uu">5a</u></b>. This result not only represented over a log-order improvement in potency from the corresponding glycine-based analogue, BP-1-102, but also represented one of the first cases of small-molecule direct inhibitor of STAT3 DNA-binding activity in vitro, with sub-micromolar potency. To confirm that the DNA-bound protein is STAT3, we performed supershift analysis.<a onclick="showRef(event, 'ref31 ref32 ref33'); return false;" href="javascript:void(0);" class="ref ref31 ref32 ref33">(31−33)</a> The presence of the specific anti-STAT3 antibody with the nuclear extract sample blocked and/or caused a shift in the band for DNA:STAT3 complex (Supporting Information <a href="/doi/suppl/10.1021/acs.jmedchem.0c01705/suppl_file/jm0c01705_si_004.pdf" class="ext-link">Figure S4A</a>). We also tested napabucasin (BBI-608), the stem cell inhibitor, which is also purported to inhibit STAT3 function,<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> and another purported STAT3 inhibitor, C188-9,<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> both of which showed a minimal direct effect on STAT3 activity up to 10 μM in the DNA-binding assay/EMSA analysis (Supporting Information <a href="/doi/suppl/10.1021/acs.jmedchem.0c01705/suppl_file/jm0c01705_si_004.pdf" class="ext-link">Figure S4B</a>), suggesting that these compounds have very little direct effects on STAT3 DNA-binding activity. As previously observed with other chiral inhibitors, such as <b><u class="uu">2</u></b> and <b><u class="uu">3</u></b> (Supporting Information <a href="/doi/suppl/10.1021/acs.jmedchem.0c01705/suppl_file/jm0c01705_si_001.pdf" class="ext-link">Figure S1</a>),<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> the (<i>R</i>)-enantiomer was more potent than the (<i>S</i>)-enantiomer (EMSA IC<sub>50</sub> of 0.52 μM for <b><u class="uu">5a</u></b> vs 2.22 μM for <b><u class="uu">5b</u></b>; <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>A and <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>), and changing the azetidine core from the azetidine-2-carboxamide to the azetidine-3-carboxamide (<b><u class="uu">5c</u></b>) (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>A and <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>) resulted in loss of activity. Our structure–activity exploration thus focused on (<i>R</i>)-azetidine-2-carboxamides.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01705/20210107/images/medium/jm0c01705_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01705/20210107/images/large/jm0c01705_0002.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Azetidine analogues inhibit STAT3 DNA-binding activity in vitro. Nuclear extracts of equal total protein prepared from NIH3T3/v-Src fibroblasts containing activated STAT3 were preincubated with increasing concentrations of the designated versions of azetidines (A) salicylic acids, (B) benzoic acids, (C) methyl esters, (D) hydroxamic acids/methylamines, or (E) heterocycles for 30 min at room temperature prior to incubating with the radiolabeled hSIE probe that binds STAT3 and performing EMSA analysis; bands corresponding to STAT3:DNA complexes in gel were quantified using ImageJ and represented as a percent of control and plotted against the concentration of compounds, from which IC<sub>50</sub> values were determined. Positions of STAT3:DNA complexes in gel are labeled; control lanes (0) represent nuclear extracts pretreated with 10% DMSO. Data are representative of two to three independent determinations.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01705/20210107/images/large/jm0c01705_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01705&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01705/20210107/images/medium/jm0c01705_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01705/20210107/images/large/jm0c01705_0003.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Structures of initial azetidine-based STAT3 inhibitors.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01705/20210107/images/large/jm0c01705_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01705&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">We took an iterative medicinal chemistry approach to design and synthesize small-molecule STAT3 inhibitors of improved physicochemical properties and that are potent and selective against tumor cells harboring constitutively active STAT3. Based on our success in the previous work with the alanine and proline analogues,<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> we systematically varied the benzoic acid and cyclohexylbenzyl moieties of the lead compound to optimize potency and physicochemical properties (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a> and <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Tables <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a> and <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a>). Changes in the cyclohexyl group to decrease lipophilicity were tolerated but with a variable effect on potency (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>, entries: 3–6), suggesting that the increased potency provided by the azetidine scaffold could allow for a balancing of physicochemical properties while maintaining sufficient potency. However, cyclohexyl remained the optimum. Introduction of polarity by changing the phenyl ring of the benzyl portion of the molecule to a heterocycle was successful. Although replacement of the phenyl in the cyclohexylbenzyl moiety with a 3-pyridyl resulted in a slight decrease in potency (EMSA IC<sub>50</sub> of 0.66 μM for <b><u class="uu">5m</u></b> vs IC<sub>50</sub> of 0.52 μM for <b><u class="uu">5a</u></b>; <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>A and <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>, entry 11), its replacement with the 2-pyridyl analogue provided 50% boost in potency (EMSA IC<sub>50</sub> of 0.38 μM for <b><u class="uu">5o</u></b>; <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>A and <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>, entry 13), potentially indicating an additional binding interaction with the STAT3 protein. Moreover, increasing the polarity at this very lipophilic region (i.e., the cyclohexylbenzyl moiety) results in better molecular polarity distribution and thus may improve the drug-like properties. Other modifications of the phenyl ring to pyrazine, pyrimidine, or pyridazine (e.g., <b><u class="uu">5p</u></b>, <b><u class="uu">5q</u></b>, and <b><u class="uu">5r</u></b> with IC<sub>50</sub> values of 0.46, 0.46, and 0.70 μM, respectively; <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>A and <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>) resulted in compounds retaining high affinity. In this heterocyclic series as well, replacement of the cyclohexyl with the less lipophilic cyclopentyl or tetrahydropyranyl resulted in slightly less potent compounds (e.g., <b><u class="uu">5n</u></b>, <b><u class="uu">5s</u></b>, and <b><u class="uu">5t</u></b>; <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). Similar results were seen with the 5-fluorosalicylates (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>, entries 7–10).</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. SAR of Salicylic Acid-Based Analogues</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01705/20210107/images/medium/jm0c01705_0011.gif" alt="" id="fx1" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01705/20210107/images/medium/jm0c01705_0012.gif" alt="" id="fx2" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup><sup>a</sup></sup><p class="last">Bands corresponding to STAT3:DNA complexes in gel were quantified using ImageJ and represented as a percent of control and plotted against the concentration of compounds, from which IC<sub>50</sub> values were determined.</p></div></div><div></div></div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. SAR of Benzoic Acid-Based Analogues</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01705/20210107/images/medium/jm0c01705_0014.gif" alt="" id="fx4" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01705/20210107/images/medium/jm0c01705_0015.gif" alt="" id="fx5" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup><sup>a</sup></sup><p class="last">Bands corresponding to STAT3:DNA complexes in gel were quantified using ImageJ and represented as a percent of control and plotted against the concentration of compounds, from which IC<sub>50</sub> values were determined.</p></div></div><div></div></div><div class="NLM_p last">Benzoic acids, other than salicylic acids, led to less potent compounds (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). However, in the cyclohexylpyridylmethyl subseries, the STAT3 potency was regained when the benzoic acid group was substituted with fluorine at the 2- or 3-position (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>B and <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>; compounds <b><u class="uu">6h</u></b> and <b><u class="uu">6i</u></b> with IC<sub>50</sub> values 0.75 and 0.86 μM, respectively). Replacement of the benzoic acid by a 4-oxazolecarboxylic acid or a 2-pyridinecarboxylic acid led to compounds with weaker activity (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>, entries 4 and 11).</div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i7" class="anchor-spacer"></div><h3 class="article-section__title" id="_i7"> Analogues with Carboxylic Acid Motif Have Low Cellular Activities</h3><div class="NLM_p last">Given that constitutively active STAT3 promotes tumor cell proliferation and survival,<a onclick="showRef(event, 'ref27 ref36'); return false;" href="javascript:void(0);" class="ref ref27 ref36">(27,36)</a> we tested the aforementioned, most active azetidine analogues for their effects on the growth of the human breast cancer MDA-MB-231 and MDA-MB-468 cells that harbor active STAT3.<a onclick="showRef(event, 'ref26 ref30'); return false;" href="javascript:void(0);" class="ref ref26 ref30">(26,30)</a> Despite their sub-micromolar potency in the in vitro cell-free STAT3 DNA-binding/EMSA assay (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Tables <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a> and <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a>), they showed weak activity against the breast cancer cells up to 10 μM (Supporting Information <a href="/doi/suppl/10.1021/acs.jmedchem.0c01705/suppl_file/jm0c01705_si_005.pdf" class="ext-link">Figure S5A,B</a>). This is presumably due to poor cell membrane permeability afforded by the ionized polar carboxylate group.<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> Even tetrazole <b><u class="uu">6e</u></b> (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>), which still partially ionizes at physiological pH, showed weak cellular activity at 10 μM. In this case, the concentration at which there is loss of 50% of DMSO-treated control cell numbers relative to the untreated, control cell numbers, EC<sub>50</sub>, is greater than 10 μM against MDA-MB-231 cells.</div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8"> Carboxylate Methyl Esters, Phthalide, and Methyl Amide Versions Improve Cellular Potency of the Azetidine Analogues</h3><div class="NLM_p">To test whether the carboxylate group was responsible for the low cellular activity, methyl esters were prepared and evaluated (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>C and <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>). Not surprisingly, the cell-free STAT3-inhibitory potencies of the methyl ester versions were lower than their corresponding acid analogues (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>C and <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>), highlighting the importance of the acid motif for inhibiting STAT3 activity.<a onclick="showRef(event, 'ref23 ref24'); return false;" href="javascript:void(0);" class="ref ref23 ref24">(23,24)</a> By contrast, the ester versions generally showed stronger cellular activities (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>A and <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>), presumably by facilitating cell membrane permeability and functioning as prodrugs inside cells. Thus, <b><u class="uu">7a</u></b>, the methyl ester of salicylic acid <b><u class="uu">5a</u></b>, had cell-free EMSA potency of >4 μM (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>C and <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>), as compared to 0.52 μM for <b><u class="uu">5a</u></b> (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>A and <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). However, <b><u class="uu">7a</u></b> showed good cellular activity against breast cancer cells following treatment for 72 h, with EC<sub>50</sub> values of 2.7 and 2.5 μM against MDA-MB-231 and MDA-MB-468 viable cells, respectively (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01705/suppl_file/jm0c01705_si_001.pdf" class="ext-link">Supporting Information Figure S5C</a> and <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>), as compared with <b><u class="uu">5a</u></b>, which showed no activity up to 10 μM (Supporting Information <a href="/doi/suppl/10.1021/acs.jmedchem.0c01705/suppl_file/jm0c01705_si_005.pdf" class="ext-link">Figure S5A</a>). Similar results were seen when comparing the methyl ester <b><u class="uu">7b</u></b> with its corresponding acid, <b><u class="uu">6f</u></b> (EC<sub>50</sub> against MDA-MB-231 cells after 72 h treatment was 4.4 μM for <b><u class="uu">7b</u></b> vs > 10 μM for <b><u class="uu">6f</u></b>) (Supporting Information <a href="/doi/suppl/10.1021/acs.jmedchem.0c01705/suppl_file/jm0c01705_si_005.pdf" class="ext-link">Figure S5B,C</a>). The esters of the more active cyclohexylpyridylmethyl analogues, <b><u class="uu">7c</u></b> and <b><u class="uu">7d</u></b>, also showed weaker STAT3-inhibitory activities in the cell-free EMSA assay (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>C and <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>, entries 3 and 4) than their corresponding acids (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>, entries 11 and 13), whereas they showed good activities against MDA-MB-231 viable cells, with EC<sub>50</sub> of 2.0 μM for <b><u class="uu">7c</u></b> and 1.8 μM for <b><u class="uu">7d</u></b> (Supporting Information <a href="/doi/suppl/10.1021/acs.jmedchem.0c01705/suppl_file/jm0c01705_si_005.pdf" class="ext-link">Figure S5C</a> and <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>).</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01705/20210107/images/medium/jm0c01705_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01705/20210107/images/large/jm0c01705_0004.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. <i>In vitro</i> cell viability and growth studies for effects of active azetidine analogs. (A) Human breast cancer MDA-MB-231 and MDA-MB-468 cells harboring aberrantly active STAT3 and (B) normal breast epithelial MCF-10A or breast cancer MCF-7 cells that do not and growing in 96-well culture were treated once with 0–10 μM of the indicated STAT3 inhibitors, or (C) human breast cancer cells in 96-well culture were first treated with 1 μM <b><u class="uu">7g</u></b> for 6 h followed by treatment with 0–100 nM docetaxel (Doc) or 0–20 μM cisplatin (Cis), or the cells were treated with docetaxel or cisplatin alone. After 72-h culture, cells were harvested and subjected to CyQuant cell proliferation assay for the number of viable cells, which are plotted as % cell viability against concentration from which EC<sub>50</sub> values were derived; or (D) human breast cancer MDA-MB-231 or MCF-7 cells in a 6-well culture plate were untreated or treated once with 2 μM <b><u class="uu">9k</u></b> and every 24 h, cells were harvested and subjected to trypan blue exclusion/phase-contrast microscopy for viable cell counts, which were plotted against the duration of treatment. Values are mean ± SEM of two to three studies each in three replicates.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01705/20210107/images/large/jm0c01705_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01705&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. SAR of Ester Analogues</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01705/20210107/images/medium/jm0c01705_0016.gif" alt="" id="fx6" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01705/20210107/images/medium/jm0c01705_0017.gif" alt="" id="fx7" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t3fn1"><div class="footnote" id="t3fn1"><sup><sup>a</sup></sup><p class="last">Bands corresponding to STAT3:DNA complexes in gel were quantified using ImageJ and represented as a percent of control and plotted against the concentration of compounds, from which IC<sub>50</sub> values were determined.</p></div></div><div></div></div><div class="NLM_p">The interesting phthalide <b><u class="uu">7g</u></b> showed one of the best activities against MDA-MB-231 and MDA-MB-468 viable cells, with EC<sub>50</sub> of 0.9 and 1.0 μM, respectively (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>A and <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Tables <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a> and <a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">6</a>). The esters <b><u class="uu">7e</u></b> (EC<sub>50</sub> 1.4 and 1.4 μM, respectively) and <b>7f</b> (EC<sub>50</sub> 1.6 and 1.6 μM, respectively) also showed relatively better cellular activities than their carboxylic acid versions against the viable cell numbers of MDA-MB-231 and MDA-MB-468 cells (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>A and <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Tables <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a> and <a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">6</a>). Notably, combined treatment with <b><u class="uu">7g</u></b> and docetaxel<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> or cisplatin,<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> both chemotherapeutic agents used to treat TNBC, showed a strong shift of the dose–response curves to the left indicative of an enhanced response (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>C). We note the apparent weaker cell-free EMSA potencies of the methyl esters, which reflect their prodrug properties, and is the reason the esters are more active in cells than outside of cells. Under conditions that promote hydrolysis, as is the case inside cells, the esters will be converted to carboxylates. Moreover, the in vitro activities of these compounds compare favorably to the parent lead compounds, BP-1-102, SH5-07, and SH4-54, with IC<sub>50</sub> of 6.8, 3.9, and 4.7 μM, respectively, and cellular activities at 10–20, 3.8, and 4.5 μM, respectively.<a onclick="showRef(event, 'ref24 ref26'); return false;" href="javascript:void(0);" class="ref ref24 ref26">(24,26)</a></div><div class="NLM_p">With these promising results, our efforts continued also on other bioisosteric replacements of the benzoic acid or salicylic acid moieties that would enable penetration into cells. We first concentrated our attention on benzohydroxamic acids, which had shown promise in our previous work (e.g., SH5-07).<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> The first benzohydroxamic acids we prepared, <b><u class="uu">8a</u></b> to <b><u class="uu">8d</u></b> (<a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>), showed only moderate inhibitory potencies against both the STAT3 DNA-binding activity in the cell-free EMSA assay (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>D and <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>) and the viable cell numbers of MDA-MB-231 and MDA-MB-468 cells (<a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>). However, by introducing the novel cyclohexylpyridylmethyl group, the corresponding benzohydroxamic acid analogue <b><u class="uu">8i</u></b> was found to have the most potent STAT3-inhibitory activity in the whole series in the EMSA assay, with an IC<sub>50</sub> value of 0.34 μM (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>D and <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>, entry 9), though it did not have good cellular activity (EC<sub>50</sub> 6.2 μM against MDA-MB-468; <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>, entry 9). Presumably, the increased polar surface area (PSA, 140 Å<sup>2</sup>) mainly imparted by both the pyridyl ring in one vector and the benzohydroxamic acid in the other vector had an impact on the sluggish cell membrane permeability. Likewise, the benzohydroxamic acid-pyrazine analogue <b><u class="uu">8s</u></b> gave a similar result (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>D, Supporting Information <a href="/doi/suppl/10.1021/acs.jmedchem.0c01705/suppl_file/jm0c01705_si_005.pdf" class="ext-link">Figure S5D</a>, and <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>, entry 19). The O-methyl hydroxamic acid <b><u class="uu">8j</u></b> (PSA 129 Å<sup>2</sup>, EMSA assay IC<sub>50</sub> 0.51 μM; <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>, entry 10) was similarly weak in the assay for viable cell numbers (Supporting Information <a href="/doi/suppl/10.1021/acs.jmedchem.0c01705/suppl_file/jm0c01705_si_005.pdf" class="ext-link">Figure S5D</a> and <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>, entry 10).</div><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. SAR of Benzohydroxamic Acid and Salicylamide Analogues</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01705/20210107/images/medium/jm0c01705_0018.gif" alt="" id="fx8" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01705/20210107/images/medium/jm0c01705_0019.gif" alt="" id="fx9" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t4fn1"><div class="footnote" id="t4fn1"><sup><sup>a</sup></sup><p class="last">Bands corresponding to STAT3:DNA complexes in gel were quantified using ImageJ and represented as a percent of control and plotted against the concentration of compounds, from which IC<sub>50</sub> values were determined. nd: not determined.</p></div></div><div></div></div><div class="NLM_p">Variations of the sub-micromolar 3-fluorobenzoic acid analogue, <b><u class="uu">6h</u></b>, were explored. The 3-fluorobenzohydroxamic acid, <b><u class="uu">8l</u></b>, was weakly active against breast cancer cells (Supporting Information <a href="/doi/suppl/10.1021/acs.jmedchem.0c01705/suppl_file/jm0c01705_si_005.pdf" class="ext-link">Figure S5D</a> and <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>) while maintaining the good STAT3-inhibitory potency in the EMSA assay (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>D, IC<sub>50</sub> of 0.89 μM; <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>, entry 12). We next sought to replace the polar hydroxamic acid group while maintaining favorable PSA and cLogP. The cutoff values that we followed, in accordance with the literature, were PSA < 120 Å<sup>2</sup> <a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> and cLogP ≤ 5.<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> The methyl salicylamide, <b><u class="uu">8q</u></b> (PSA 119 Å<sup>2</sup>), showed good activity against breast cancer cells in both viability assays, with EC<sub>50</sub> 1.8 and 1.8 μM against MDA-MB-231 cells and MDA-MB-468 cells, respectively, and trypan blue exclusion/phase-contrast microscopy cell growth assay (Supporting Information <a href="/doi/suppl/10.1021/acs.jmedchem.0c01705/suppl_file/jm0c01705_si_005.pdf" class="ext-link">Figure S5C,E</a> and <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>, entry 17), and it also showed a high STAT3-inhibitory potency (IC<sub>50</sub> 0.77 μM; <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>D and <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>, entry 17). The more polar primary salicylamide <b><u class="uu">8p</u></b> (PSA 133 Å<sup>2</sup>) showed relatively modest activity against MDA-MB-231 and MDA-MB-468 cells, with EC<sub>50</sub> 2.9 and 2.6 μM, respectively (<a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a> and Supporting Information <a href="/doi/suppl/10.1021/acs.jmedchem.0c01705/suppl_file/jm0c01705_si_005.pdf" class="ext-link">Figure S5C</a>), while strongly inhibiting STAT3 DNA-binding activity in EMSA (IC<sub>50</sub> 0.66 μM).</div><div class="NLM_p">Altogether, the replacement of the salicylic or benzoic acid system with a methyl salicylate or benzoate (<b><u class="uu">7a</u></b>, <b><u class="uu">7b</u></b>, <b><u class="uu">7c</u></b>, <b><u class="uu">7d, 7e, 7f</u></b>), phthalide (<b><u class="uu">7g</u></b>) (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>, <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>A, and Supporting Information <a href="/doi/suppl/10.1021/acs.jmedchem.0c01705/suppl_file/jm0c01705_si_005.pdf" class="ext-link">Figure S5C</a>), or salicylamide (<b><u class="uu">8p</u></b>, <b><u class="uu">8q</u></b>) (<a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a> and Supporting Information <a href="/doi/suppl/10.1021/acs.jmedchem.0c01705/suppl_file/jm0c01705_si_005.pdf" class="ext-link">Figure S5C</a>) gave analogues that showed variable cellular activities. Notably, <b><u class="uu">7e</u></b>, <b><u class="uu">7f</u></b>, and <b><u class="uu">7g</u></b> represent the best in the group to possess high cellular activities, with an EC<sub>50</sub> of 0.9–1.6 μM (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>A and <a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">Table <a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">6</a></a>). Comparatively, <b><u class="uu">7e</u></b>, <b><u class="uu">7f</u></b>, and <b><u class="uu">7g</u></b> showed relatively weaker activities against the normal human breast epithelial MCF-10A, with an EC<sub>50</sub> of 3.8–4.6 μM, and against the breast cancer MCF-7 cells that do not harbor constitutively active STAT3, with an EC<sub>50</sub> of 4.6–8.9 μM (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>B and <a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">Table <a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">6</a></a>).</div><div class="NLM_p last">Additional specificity studies were conducted for the inhibitors <b><u class="uu">7a</u></b>, <u class="uu"><b>7b</b></u>, <b><u class="uu">7c</u></b>, <b><u class="uu">8p</u></b>, and <b><u class="uu">8q</u></b>. The results showed that these compounds are relatively weaker in inhibiting the number of viable cells that do not harbor aberrantly active STAT3, with EC<sub>50</sub> 4.6 and 7.2 μM (<b><u class="uu">8p</u></b>), and 3.8 and 4.6 μM (<b><u class="uu">8q</u></b>) against MCF-7 or MCF-10A cells, respectively, and 4.3 μM (<b><u class="uu">7a</u></b>), 6.0 μM (<b><u class="uu">7b</u></b>) and 3.6 μM (<b><u class="uu">7c</u></b>) against MCF-10A cells that do not harbor aberrantly active STAT3 (Supporting Information <a href="/doi/suppl/10.1021/acs.jmedchem.0c01705/suppl_file/jm0c01705_si_005.pdf" class="ext-link">Figure S5F</a> and <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>). Notably, treatment with 2 μM of <b><u class="uu">8q</u></b> showed no effect on the growth of MCF-7 cells as measured by trypan blue exclusion/phase-contrast microscopy (Supporting Information <a href="/doi/suppl/10.1021/acs.jmedchem.0c01705/suppl_file/jm0c01705_si_005.pdf" class="ext-link">Figure S5E</a>). These inhibitors therefore show varied relative preferences against STAT3-dependent tumor cells over cells that do not depend on STAT3 activity. Notably, the carboxylic acid bioisosteres provided the first evidence that strong cellular activity could be achieved.</div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i11" class="anchor-spacer"></div><h3 class="article-section__title" id="_i11"> Isosteric Replacement of Salicylic Acid Moiety with Benzo-Fused <i>N</i>-Heterocyclic Systems Retained the In Vitro Activity of Analogues and Greatly Improved Their Cellular Activity</h3><div class="NLM_p">A concurrent approach was to incorporate benzo-fused <i>N</i>-heterocyclic systems, which were also successful in providing analogues with good cellular potency (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>E and <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">Table <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a></a>). The benzo-fused <i>N</i>-heterocyclic replacement for the salicylic acids allowed for further fine tuning of the cLogP and especially PSA to achieve the desirable physicochemical properties while maintaining potency. Initial results from the isoquinolinone analogues were encouraging (<a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">Tables <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a></a> and <a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">6</a>); for example, compound <b><u class="uu">9b</u></b>, with a PSA value of 99 Å<sup>2</sup> showed cellular activity against MDA-MB-231 cells, with an EC<sub>50</sub> of 1.2 μM, while retaining STAT3-inhibitory potency (IC<sub>50</sub> 0.79 μM) (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>E and <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">Table <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a></a>, entry 2). The corresponding quinazolinone analogues were moderately active against MDA-MB-231 and MDA-MB-468 cells (<a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">Table <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a></a>). This was especially the case for the <i>N</i>-methyl variant <b><u class="uu">9g</u></b> that was moderately active against MDA-MB-231 and MDA-MB-468 cells (EC<sub>50</sub> 2.1 and 2.2 μM, respectively) while also maintaining good STAT3-inhibitory activity, with an IC<sub>50</sub> of 1.15 μM (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>E and <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">Table <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a></a>, entry 7). As for the analogues with benzo-fused 5-membered <i>N</i>-heterocycles, while the cellular activity of benzotriazole <b><u class="uu">9r</u></b> against MDA-MB-231 and MDA-MB-468 was moderate (EC<sub>50</sub> 2.1 and 2.3, respectively), that of <b><u class="uu">9q</u></b> was rather low (EC<sub>50</sub> of 6.6 and 5.1 μM, respectively), despite both having relatively good PSA values (107 and 119 Å<sup>2</sup>, respectively; <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">Table <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a></a>), a reflection perhaps of the more basic pyridine over the pyrazine systems. The STAT3-inhibitory potencies for both <b><u class="uu">9q</u></b> and <b><u class="uu">9r</u></b>, however, were high (IC<sub>50</sub> 0.61 and 0.63 μM, respectively; <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>E and <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">Table <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a></a>, entries 17 and 18). On the other hand, results for the isoindolinones <b><u class="uu">9i</u></b> and <b><u class="uu">9j</u></b> showed moderate cellular activities (EC<sub>50</sub> 2.0 μM against MDA-MB-231 and PSA 99 Å<sup>2</sup> for both compounds), while <b><u class="uu">9i</u></b> exhibited better potency than <b><u class="uu">9j</u></b> against STAT3 DNA-binding activity (IC<sub>50</sub> 0.64 and 1.09 μM, respectively; <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">Table <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a></a>, entries 9 and 10; <a href="/doi/suppl/10.1021/acs.jmedchem.0c01705/suppl_file/jm0c01705_si_001.pdf" class="ext-link">Supporting Information Table S1</a>). On the other hand, phthalimide <b><u class="uu">9k</u></b> presented good inhibitory activities in the cell-free STAT3 DNA-binding/EMSA (IC<sub>50</sub> 1.18 μM) (PSA 116 Å<sup>2</sup>; <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">Table <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a></a>, entry 11) and in the cell viability assays (EC<sub>50</sub> 1.7 and 1.9 μM against MDA-MB-231 and MDA-MB-468 cells, respectively) (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>A). Moreover, trypan blue exclusion/phase-contrast microscopy showed that 2 μM <b><u class="uu">9k</u></b> treatment of MDA-MB-231 cells strongly inhibited cell growth (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>D). By contrast, the test of <b><u class="uu">9k</u></b> on MCF-10A and MCF-7 cells that do not harbor constitutively active STAT3 showed weaker effects on both cell viability, with EC<sub>50</sub> of 8.1 and 7.0 μM, respectively (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>B, <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">Table <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a></a>, entry 11, and <a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">Table <a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">6</a></a>). Further, trypan blue exclusion/phase-contrast microscopy showed that treatment with 2 μM <b><u class="uu">9k</u></b> of MCF-7 cells had no effect on cell growth (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>D). Furthermore, 72 h treatment of MCF-7 and MCF-10A cells with <b><u class="uu">9a</u></b> or <b><u class="uu">9d</u></b> showed only moderate effects on viable cell numbers (<a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">Table <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a></a> and Supporting Information <a href="/doi/suppl/10.1021/acs.jmedchem.0c01705/suppl_file/jm0c01705_si_005.pdf" class="ext-link">Figure S5F, Table S1</a>). Other systems such as indazole (<b><u class="uu">9l</u>, <u class="uu">9m</u>, <u class="uu">9n</u></b>, and <b><u class="uu">9o</u></b>) or benzimidazole (<b><u class="uu">9p</u></b>) did not look as promising (<a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">Table <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a></a>). Comparatively, our inhibitors show better selectivity against tumor cells harboring aberrantly active STAT3 than the purported STAT3 inhibitors, napabucasin (BBI-608)<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> or C188-9.<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> Treatments with increasing inhibitor concentrations showed weak effects for C188-9 (EC<sub>50</sub> 25.7 μM) against MDA-MB-231 cells but 2-fold stronger activity (EC<sub>50</sub> 13.75 μM) against MCF-7 cells that do not harbor constitutively active STAT3, while napabucasin showed strong effects against both MDA-MB-231 (EC<sub>50</sub> 1.8 μM) and MCF-7 cells (EC<sub>50</sub> 1.49 μM) (Supporting Information <a href="/doi/suppl/10.1021/acs.jmedchem.0c01705/suppl_file/jm0c01705_si_005.pdf" class="ext-link">Figure S5G,H</a>) suggesting the lack of specificity for either inhibitor.</div><div class="NLM_table-wrap" id="tbl5"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 5. SAR of Benzo-Fused <i>N</i>-Heterocyclic Analogues</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01705/20210107/images/medium/jm0c01705_0021.gif" alt="" id="fx11" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01705/20210107/images/medium/jm0c01705_0022.gif" alt="" id="fx12" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01705/20210107/images/medium/jm0c01705_0023.gif" alt="" id="fx13" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t5fn1"><div class="footnote" id="t5fn1"><sup><sup>a</sup></sup><p class="last">Bands corresponding to STAT3:DNA complexes in gel were quantified using ImageJ and represented as a percent of control and plotted against the concentration of compounds, from which IC<sub>50</sub> values were determined. nd: not determined.</p></div></div><div></div></div><div class="NLM_table-wrap" id="tbl6"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 6. EMSA IC<sub>50</sub> and EC<sub>50</sub> Values of Select Esters and Heterocycles</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01705/20210107/images/medium/jm0c01705_0024.gif" alt="" id="fx14" /></img><div></div></div></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14"> Isothermal Titration Calorimetry (ITC) Studies of Inhibitor Binding to STAT3</h3><div class="NLM_p">Given the enhanced potency of the azetidine-based inhibitors against STAT3, we were interested to determine their level of binding to the target in vitro. We performed isothermal titration calorimetry (ITC) studies, as previously described.<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> The binding isotherm from the integrated thermogram fit using the one-site model in the PEAQ-ITC software generated from the titration of the representative inhibitors, <b><u class="uu">7g</u></b> (red) and <b><u class="uu">9k</u></b> (blue), into STAT3 shows <i>K</i><sub>D</sub> of 880 and 960 nM, respectively (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>A). The signature plots showing the thermodynamics parameters for each titration reveal Δ<i>H</i> = −22.7 kJ/mol, Δ<i>G</i> = −34.6 kJ/mol, and −<i>T</i>Δ<i>S</i> = −11.9 kJ/mol for <b><u class="uu">7g</u></b> and Δ<i>H</i> = −20.8 kJ/mol, Δ<i>G</i> = −34.4 kJ/mol, and −<i>T</i>Δ<i>S</i> = −13.6 kJ/mol for <b><u class="uu">9k</u></b> (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>B). Results together show that the azetidine inhibitors directly bind with high affinity to STAT3.</div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01705/20210107/images/medium/jm0c01705_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01705/20210107/images/large/jm0c01705_0005.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Isothermal titration calorimetry (ITC) measurements of <b><u class="uu">7g</u></b>, and <b><u class="uu">9k</u></b> during incubation with STAT3. (A) Binding isotherm from the integrated thermogram fit using the one-site model in the PEAQ-ITC software generated from the titration of the inhibitors, <b><u class="uu">7g</u></b> (red) and <b><u class="uu">9k</u></b> (blue), into STAT3. The <i>K</i><sub>D</sub> was 880 and 960 nM, respectively, and (B) the signature plots showing the thermodynamics parameters for each titration reveal Δ<i>H</i> = −22.7 kJ/mol, Δ<i>G</i> = −34.6 kJ/mol, and −<i>T</i>Δ<i>S</i> = −11.9 kJ/mol for <b><u class="uu">7g</u></b> and Δ<i>H</i> = −20.8 kJ/mol, Δ<i>G</i> = −34.4 kJ/mol, and −<i>T</i>Δ<i>S</i> = −13.6 kJ/mol for <b><u class="uu">9k</u></b>. Data are representative of three independent experiments.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01705/20210107/images/large/jm0c01705_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01705&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i15" class="anchor-spacer"></div><h3 class="article-section__title" id="_i15"> Comparison of the Inhibition of STAT3 DNA-Binding Activity over That of STAT1 and STAT5 In Vitro</h3><div class="NLM_p">To further establish the specificity of select new analogues with potent activity against STAT3, we investigated their effects on other STAT family members, including STAT1 and STAT5 DNA-binding activities in vitro, as previously reported.<a onclick="showRef(event, 'ref13 ref23 ref24 ref26'); return false;" href="javascript:void(0);" class="ref ref13 ref23 ref24 ref26">(13,23,24,26)</a> Nuclear extracts were prepared from epidermal growth factor (EGF)-stimulated fibroblasts overexpressing the EGF receptor (NIH3T3/EGFR) containing active STAT1, STAT3, and STAT5. Extracts of equal total protein were incubated with increasing or a single concentration of the azetidine compounds prior to incubation with the radiolabeled hSIE probe that binds STAT1 and STAT3 or the mammary gland factor element (MGFe) that binds STAT1 and STAT5 and performing the EMSA analysis. Results show that select azetidine analogues had minimum effects on both STAT5 and STAT1 DNA-binding activities (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a> and Supporting Information <a href="/doi/suppl/10.1021/acs.jmedchem.0c01705/suppl_file/jm0c01705_si_006.pdf" class="ext-link">Figure S6</a>). Excluding the esters of the carboxylic acids that in principle are prodrugs, we selected and evaluated the selectivity of the representative analogues from the other subgroups. All compounds tested showed preferentially potent disruption of the DNA-binding activity of STAT3:STAT3 homodimers ahead of STAT1:STAT3 heterodimers, which were inhibited ahead of STAT1:STAT1 homodimers or STAT5:STAT5 homodimers, with potencies (IC<sub>50</sub>) of 0.52, 2.61, 12.0, and 9.3 μM (<b><u class="uu">5a</u></b>), 0.38, 1.46, >20, and >20 μM (<b><u class="uu">5o</u></b>), 1.08, 4.92, >20, and 17.5 μM (<b><u class="uu">6f</u></b>), 0.77, 3.14, >20, and >20 μM (<b><u class="uu">8q</u></b>), and 1.18, 4.71, >20, and >20 μM (<b><u class="uu">9k</u></b>) (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>). Other compounds, including <b><u class="uu">5e</u></b>, <b><u class="uu">5g</u></b>, <b><u class="uu">5i</u></b>, <b><u class="uu">5m</u></b>, <b><u class="uu">5n</u></b>, <b><u class="uu">7a</u></b>, <b><u class="uu">7b</u></b>, <b><u class="uu">8d</u></b>, <b><u class="uu">8f</u></b>, <b><u class="uu">8q</u></b>, and <b><u class="uu">9k</u></b>, also showed minimal effect on the DNA-binding activity of STAT5:STAT5 or STAT1:STAT1 homodimers (Supporting Information <a href="/doi/suppl/10.1021/acs.jmedchem.0c01705/suppl_file/jm0c01705_si_006.pdf" class="ext-link">Figure S6</a>). Altogether, these data show that active azetidine inhibitors tested have preferential effects on the DNA-binding activity of STAT3 over that of STAT1 or STAT5.</div><figure id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01705/20210107/images/medium/jm0c01705_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01705/20210107/images/large/jm0c01705_0006.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Comparing the effect of new analogues on STAT1, STAT3, or STAT5 DNA-binding activity in vitro. Nuclear extracts of equal total protein prepared from epidermal growth factor-stimulated NIH3T3/EGFR fibroblasts containing activated STAT1, STAT3, and STAT5 were preincubated with increasing concentrations of the designated compounds for 30 min at room temperature prior to incubating with the radiolabeled hSIE probe that binds STAT1 and STAT3 (upper panel) or the MGFe probe that binds STAT5 (bottom panel) and performing EMSA analysis. Positions of STAT:DNA complexes in gel are labeled; control lanes (0) represent nuclear extracts preincubated with 10% DMSO. Data are representative of two to three independent determinations.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01705/20210107/images/large/jm0c01705_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01705&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i16" class="anchor-spacer"></div><h3 class="article-section__title" id="_i16"> Analogues Inhibited Constitutive STAT3 Phosphorylation and DNA-Binding Activity in Human Breast Cancer Cells</h3><div class="NLM_p">The human breast cancer MDA-MB-231 and MDA-MB-468 cells harbor aberrantly active STAT3<a onclick="showRef(event, 'ref27 ref42'); return false;" href="javascript:void(0);" class="ref ref27 ref42">(27,42)</a> and are sensitive to the azetidine inhibitors (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>A). We were interested to determine the effect of select active azetidine analogues on the constitutive STAT3 signaling in the breast cancer cells. Cells were treated with inhibitors, including <b><u class="uu">7g</u></b>, <b><u class="uu">8q</u></b>, and <b><u class="uu">9k</u></b>, at 1–5 μM for 0–24 h. Nuclear extracts were prepared and subjected to DNA-binding activity/EMSA analysis, while whole-cell lysates were prepared for sodium dodecyl sulfate polyacrylamide gel electrophoresis (SDS-PAGE)/Western blotting analysis to determine effects on intracellular STAT3 activity.<a onclick="showRef(event, 'ref23 ref24 ref26'); return false;" href="javascript:void(0);" class="ref ref23 ref24 ref26">(23,24,26)</a> Results showed that STAT3 DNA-binding activity was inhibited to variable degrees in MDA-MB-468 cells by <b><u class="uu">7g</u></b>, <b><u class="uu">8q</u></b>, and <b><u class="uu">9k</u></b> and in a time-dependent manner and as early as 5 min for <b><u class="uu">7g</u></b> and <b><u class="uu">9k</u></b> and 6 h for <b><u class="uu">8q</u></b> (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>A and Supporting Information <a href="/doi/suppl/10.1021/acs.jmedchem.0c01705/suppl_file/jm0c01705_si_007.pdf" class="ext-link">Figure S7A</a>). Similarly, treatment with increasing concentrations of <b><u class="uu">7g</u></b>, <b><u class="uu">8q</u></b>, and <b><u class="uu">9k</u></b> inhibited pY705STAT3 in dose- and time-dependent manner in the breast cancer cells (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>B and Supporting Information <a href="/doi/suppl/10.1021/acs.jmedchem.0c01705/suppl_file/jm0c01705_si_007.pdf" class="ext-link">Figure S7B</a>). For the <b><u class="uu">9k</u></b> treatment condition, the pYSTAT3 bounced back by 24 h (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>B-iii). Comparatively, treatment of MDA-MB-468 and MDA-MB-231 cells for 2–3 h with the purported STAT3 inhibitor, BBI-608,<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> at 0.5–5 μM showed weak to moderate effects on STAT3 DNA-binding activity or pY705STAT3 (Supporting Information <a href="/doi/suppl/10.1021/acs.jmedchem.0c01705/suppl_file/jm0c01705_si_007.pdf" class="ext-link">Figure S7C,D</a>). For nonspecific effects, immunoblotting analysis showed that treatment of MDA-MB-231 cells with 1 or 3 μM <b><u class="uu">7g</u></b> had no measurable effect on the tyrosine kinases, EGFR, JAK2, and Src, or on AKT and ERK1/2 (Supporting Information <a href="/doi/suppl/10.1021/acs.jmedchem.0c01705/suppl_file/jm0c01705_si_007.pdf" class="ext-link">Figure S7E</a>). These results show that select cell-permeable azetidine inhibitors are active at 1–3 μM against constitutive STAT3 induction in human breast cancer cells.</div><figure id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01705/20210107/images/medium/jm0c01705_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01705/20210107/images/large/jm0c01705_0007.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Effects of compounds on constitutive STAT3 activation in tumor cells. (A) Nuclear extracts of equal total protein prepared from the human breast cancer, MDA-MB-468 cells untreated (DMSO, 0) or treated with 5 μM of the indicated analogues for 1–3 h were subjected to STAT3 DNA-binding assay using the hSIE probe that binds STAT3, and (B) immunoblotting analysis of whole-cell lysates of equal total protein prepared from (i) and (ii) MDA-MB-231 cells untreated (DMSO, 0) or treated with 1 or 3 μM of <b><u class="uu">7g</u></b> for 3–24 h or (iii) MDA-MB-468 cells untreated (DMSO, 0) or treated with 3 μM of <b><u class="uu">9k</u></b> for 3–24 h and probing for pY705STAT3, STAT3, or tubulin. Positions of STAT3:DNA complex or proteins in gel are shown; control (0 or Con) lane represents whole-cell lysates or nuclear extracts prepared from 0.05% DMSO-treated cells. Data are representative of two to three independent determinations.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01705/20210107/images/large/jm0c01705_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01705&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> Azetidines Inhibited the Colony Survival of Human Breast Cancer Cells</h3><div class="NLM_p">Human MDA-MB-231 breast cancer cells in a single-cell culture were treated once with analogues <b><u class="uu">7e</u></b>, <b><u class="uu">7f</u></b>, <b><u class="uu">7g</u></b>, <b><u class="uu">8q</u></b>, or <b><u class="uu">9k</u></b> at 0.5–1 μM and allowed to culture until colonies were visible, which were stained and imaged. The results indicate that at the concentration of 0.5 μM, <b><u class="uu">7g</u></b> shows significant inhibition of colony formation; however, <b><u class="uu">7e</u></b>, <b><u class="uu">7f</u></b>, and <b><u class="uu">9k</u></b> only show minimal to moderate inhibition of colony formation (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>). Moreover, at the concentration of 1 μM, the results show a complete inhibition of colony formation for <b><u class="uu">7g</u></b>, a near-complete inhibition for <b><u class="uu">7e</u></b>, and a significant inhibition for <b><u class="uu">7f</u></b> and <b><u class="uu">9k</u></b> (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>), whereas, <b><u class="uu">8q</u></b> does not show any inhibitory effect up to 1 μM (Supporting Information <a href="/doi/suppl/10.1021/acs.jmedchem.0c01705/suppl_file/jm0c01705_si_008.pdf" class="ext-link">Figure S8</a>). These results indicate that selected azetidine-based STAT3 inhibitors attenuate the survival of cancer cells that harbor constitutively active STAT3 at concentrations that inhibit STAT3 activity.</div><figure id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01705/20210107/images/medium/jm0c01705_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01705/20210107/images/large/jm0c01705_0008.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Compounds <u class="uu"><b>7e</b></u>, <u class="uu"><b>7f</b></u>, <u class="uu"><b>7g</b></u>, and <u class="uu"><b>9k</b></u> inhibit the colony survival of human breast cancer cells in vitro. Human breast cancer MDA-MB-231 cells were seeded as single-cell culture and treated once with 0–1 μM of the indicated compounds and allowed to culture until large colonies were visible, which were stained with crystal violet and imaged. Data are representative of three independent determinations.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01705/20210107/images/large/jm0c01705_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01705&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> Inhibition of STAT3-Regulated Genes and Induction of Apoptosis</h3><div class="NLM_p">Consistent with the dysregulation of genes that promote tumor cell growth, survival, and malignant phenotype,<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> treatment of breast cancer cells with 1 or 3 μM <b><u class="uu">7g</u></b> for 3–24 h inhibited the expression of STAT3 target genes, c-Myc, vascular endothelial growth factor (VEGF), Bcl-2, and survivin (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>) and induced poly (ADP-ribose) polymerase (PARP) cleavage in parallel with the inhibition of pY705STAT3 (<a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a></a>).</div><figure id="fig8" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01705/20210107/images/medium/jm0c01705_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01705/20210107/images/large/jm0c01705_0009.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Inhibition of STAT3 target gene expression in breast cancer cells. Immunoblotting analysis of whole-cell lysates of equal total protein prepared from MDA-MB-231 cells untreated (DMSO, C) or treated with 1 or 3 μM of <b><u class="uu">7g</u></b> for 3–24 h and probing for c-Myc Bcl-2, VEGF, survivin, or tubulin. Positions of proteins in gel are shown; control (C) lane represents whole-cell lysates prepared from 0.05% DMSO-treated cells. Data are representative of two to three independent determinations.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01705/20210107/images/large/jm0c01705_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01705&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig9" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01705/20210107/images/medium/jm0c01705_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01705/20210107/images/large/jm0c01705_0010.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. Induction of apoptosis of human breast cancer cells. Human breast cancer, MDA-MB-231 and MDA-MB-468 cells in culture were treated with 1 or 3 μM <b><u class="uu">7g</u></b> for 0–24 h, whole-cell lysates were prepared, and samples of equal total protein were subjected to SDS/PAGE–Western blotting analysis probing for pYSTAT3, STAT3, full-length PARP, cleaved PARP, and tubulin. Positions of proteins in gel are shown; control (0) lane represents whole-cell lysates prepared from 0.05% DMSO-treated cells. Data are representative of two independent determinations.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01705/20210107/images/large/jm0c01705_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01705&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i19" class="anchor-spacer"></div><h3 class="article-section__title" id="_i19"> Initial Evaluation of Solubility and Metabolic Characteristics of the Azetidine Compounds</h3><div class="NLM_p last">The in vivo activity of a drug is influenced by parameters, such as solubility, permeability, and metabolism,<a onclick="showRef(event, 'ref43 ref44 ref45'); return false;" href="javascript:void(0);" class="ref ref43 ref44 ref45">(43−45)</a> which are dependent on its physicochemical characteristic. With medicinal chemistry effort focused on the optimization of the physicochemical features, we conducted an industry-standard evaluation of solubility and metabolism assays<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a> through the contract research organization (CRO), Eurofins-CEREP. In general, compounds show good aqueous solubility, as found by simulated gastric fluid (SGF) and simulated intestinal fluid (SIF), above the standard cutoff of 60 μg/mL.<a onclick="showRef(event, 'ref47 ref48'); return false;" href="javascript:void(0);" class="ref ref47 ref48">(47,48)</a> For example, promising phthalide <b><u class="uu">7g</u></b> has a solubility at SGF of 116 μg/mL and at SIF of 200 μg/mL (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01705/suppl_file/jm0c01705_si_002.csv" class="ext-link">Table S2</a>). Preliminary results from a human HLM MetID study of the phthalide <b><u class="uu">7g</u></b> showed both the parent compound and its hydroxy-acid metabolite (Supporting Information <a href="/doi/suppl/10.1021/acs.jmedchem.0c01705/suppl_file/jm0c01705_si_009.pdf" class="ext-link">Figure S9</a>), which is the result of hydrolytic lactone opening, and a presumably more potent version in terms of directly inhibiting STAT3 DNA-binding activity, suggesting that <b><u class="uu">7g</u></b> and its corresponding hydroxy-acid version exist in equilibrium. Additional metabolites of <b><u class="uu">7g</u></b> were also identified (Supporting Information <a href="/doi/suppl/10.1021/acs.jmedchem.0c01705/suppl_file/jm0c01705_si_009.pdf" class="ext-link">Figure S9</a>).</div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i20">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_16263" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_16263" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">Despite the strong validation of STAT3 as a target, as a transcription factor, it has presented significant challenges for drug discovery/development research due to the flat protein surface that lacks deep pockets to target and design high-affinity binders. Much of the earlier efforts to develop inhibitors of STAT3 focused on targeting the SH2 domain to disrupt the interactions with pTyr-containing binding sites and the STAT3:STAT3 dimerization event.<a onclick="showRef(event, 'ref8 ref9'); return false;" href="javascript:void(0);" class="ref ref8 ref9">(8,9)</a> These efforts have generated a number of small-molecule inhibitors,<a onclick="showRef(event, 'ref11 ref12 ref13 ref14 ref15 ref16 ref17 ref18 ref19 ref20 ref21 ref22 ref23 ref24 ref25 ref26 ref35'); return false;" href="javascript:void(0);" class="ref ref11 ref12 ref13 ref14 ref15 ref16 ref17 ref18 ref19 ref20 ref21 ref22 ref23 ref24 ref25 ref26 ref35">(11−26,35)</a> though the clinical development of these inhibitors has been hampered due to their low potency, unclear mechanisms of inhibition of STAT3 signaling, and pharmacokinetic limitations among others.<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> New methods are required to design unique potent small-molecule binders that can potentially overcome the limitations. The nanomolar potent PROTAC-STAT3 degraders, including SD-36, are a new class of inhibitors, and the proof-of-concept studies in which SD-36 inhibited the growth of xenograft models of leukemia and lymphomas show that they hold promise.<a onclick="showRef(event, 'ref28 ref29'); return false;" href="javascript:void(0);" class="ref ref28 ref29">(28,29)</a> In the current study, we have taken a different strategy to address the challenge of potency by creating the new azetidine class of inhibitors. The azetidine series marks a significant advancement in the study of small-molecule STAT3 inhibitors. With the change from <i>R</i>-proline-amides to <i>R</i>-azetidine-2-carboxamides, the first analogues have been realized with sub-micromolar potency in the STAT3 DNA-binding activity/EMSA, i.e., salicylate <b><u class="uu">5a</u></b> (EMSA IC<sub>50</sub> 0.55 μM) among many others. Optimization of the salicylate series provided the 5-cyclohexyl-2-pyridinylmethyl analogues that led to the potent salicylate analogue, <b><u class="uu">5o</u></b> (EMSA IC<sub>50</sub> 0.38 μM), which represented a log-order improvement in potency over earlier published analogues.<a onclick="showRef(event, 'ref24 ref26'); return false;" href="javascript:void(0);" class="ref ref24 ref26">(24,26)</a> Further elaboration of the salicylic acid portion of the molecule provided benzamides and benzo-fused <i>N</i>-heterocycles, which maintained the sub-micromolar potency in the in vitro STAT3 DNA-binding/EMSA assay and furnished the most potent analogue in the whole azetidine series, i.e., benzohydroxamic acid <b><u class="uu">8i</u></b> (EMSA IC<sub>50</sub> 0.34 μM). Having overcome the potency barrier, we focused the medicinal chemistry efforts on optimizing the physicochemical properties while maintaining potency. We were able to trade some potency for improved cell permeability, and several compounds showed improved activity in cell-based assays over previous analogues. In particular, ester versions of the carboxylic acid, including <b><u class="uu">7e</u></b> and <b><u class="uu">7f</u></b>, or the lactone, <b><u class="uu">7g</u></b>, or compounds that contained carboxylic acid bioisosteres, such as salicylamides (<b><u class="uu">8p</u></b> and <b><u class="uu">8q</u></b>) and some of the benzo-fused <i>N</i>-heterocycle analogues (<b><u class="uu">9b</u>, <u class="uu">9f</u></b>, and <b><u class="uu">9k</u></b>), showed good cellular activity. Notably, <b><u class="uu">7e</u></b>, <b><u class="uu">7f</u></b>, <b><u class="uu">7g</u></b>, and <b><u class="uu">9k</u></b> have the best cellular activities against breast tumor cells that harbor aberrantly active STAT3, including inhibiting cell growth, suppressing STAT3 target gene expression, and inducing apoptosis. The current azetidine series of compounds therefore are significantly improved over their leads, BP-1-102, SH5-07, and SH4-54, which had EMSA IC<sub>50</sub> of 6.8, 3.9, and 4.7 μM, respectively, and cellular activities at 10–20, 3.8, and 4.5 μM, respectively. They also compare more favorably to napabucasin (BBI-608) and C188-9. The azetidine compounds therefore represent promising new chemical entities in the search for suitable small-molecule, direct STAT3 inhibitors for further clinical development into new anticancer agents either as standalone or in combination. Their utility in combination therapy with chemotherapy, including docetaxel or cisplatin, is demonstrated herein.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i21">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_07514" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_07514" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i22" class="anchor-spacer"></div><h3 class="article-section__title" id="_i22"> Cell Lines and Reagents</h3><div class="NLM_p last">The human breast cancer MDA-MB-468 and MCF-7 cell lines and the normal breast epithelial line MCF-10A have been reported previously,<a onclick="showRef(event, 'ref24 ref26 ref31 ref32'); return false;" href="javascript:void(0);" class="ref ref24 ref26 ref31 ref32">(24,26,31,32)</a> and MDA-MB-231 was obtained from the National Cancer Institute on December 2, 2015. MCF-10A cells were grown in Dulbecco’s modified Eagle’s medium, DMEM/F12 with 5% horse serum plus EGF (20 ng/mL), insulin (10 μg/mL), hydrocortisone (0.5 mg/mL), and 100 ng/mL cholera toxin. All other cells were grown in DMEM plus 10% heat-inactivated fetal bovine serum (FBS). Cell line authentication was done in December 2015 by American Type Culture Collection (ATCC) for MDA-MB-468, which was found to be authentic. Mycoplasma test was conducted on MDA-MB-231 and MCF-10A IDEXX BioAnalytics (Columbia, MO) in Dec 2019. Both lines are negative, while the test has not been done on MCF-7 and MDA-MB-468. Antibodies against STAT3, pY705STAT3, pY1173EGFR, EGFR, pY1007/1008JAK2, JAK2, pY416Src, Src, pS473AKT, AKT, pT202/Y204ERK1/2 (p44/42), ERK1/2, full-length poly (ADP-ribose) polymerase (PARP), cleaved PARP, tubulin, and glyceraldehyde 3-phosphate dehydrogenase (GAPDH) were purchased from Cell Signaling Technology, Inc. (Danvers, MA). Cisplatin and docetaxel were purchased from Sigma-Aldrich (St. Louis, MO).</div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i23" class="anchor-spacer"></div><h3 class="article-section__title" id="_i23"> Nuclear Extract Preparation, Gel Shift Assays, and Densitometric Analysis</h3><div class="NLM_p last">Nuclear extract preparations and DNA-binding activity/electrophoretic mobility shift assay (EMSA) were carried out as previously described.<a onclick="showRef(event, 'ref13 ref14 ref23 ref24 ref26'); return false;" href="javascript:void(0);" class="ref ref13 ref14 ref23 ref24 ref26">(13,14,23,24,26)</a> The <sup>33</sup>P-labeled oligonucleotide probes used were hSIE (high-affinity <i>sis</i>-inducible element from the c-fos gene, m67 variant, 5′-AGCTTCATTTCCCGTAAATCCCTA) that binds STAT1 and STAT3 and MGFe (mammary gland factor element from the bovine β-casein gene promoter, 5′-AGATTTCTAGGAATTCAA) for STAT1 and STAT5 binding. Except where indicated, nuclear extracts of equal total protein were preincubated with compound for 30 min at room temperature prior to incubation with the radiolabeled probe for 30 min at 30 °C before subjecting to EMSA analysis. Where appropriate, bands corresponding to STAT3:DNA complexes were scanned and quantified for each concentration of compound using ImageJ and plotted as a percent of control (DMSO) against the concentration of compound, from which the IC<sub>50</sub> values were derived.</div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i24" class="anchor-spacer"></div><h3 class="article-section__title" id="_i24"> Immunoblotting Analysis</h3><div class="NLM_p last">Whole-cell lysate preparation and immunoblotting analysis were performed as previously reported.<a onclick="showRef(event, 'ref23 ref24'); return false;" href="javascript:void(0);" class="ref ref23 ref24">(23,24)</a> Briefly, cultured cells treated or not were harvested and whole-cell lysates were prepared in RIPA buffer. Samples of equal total protein were subjected to SDS-PAGE and immunoblotting analysis. Primary antibodies used were anti-STAT3, pY705STAT3, PARP, c-Myc, Bcl-2, VEGF, survivin, tubulin, and GAPDH. All antibodies were purchased from Cell Signaling Technology, Inc. (Danvers, MA), except GAPDH from Santa Cruz Biotechnology (Dallas, Texas).</div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i25" class="anchor-spacer"></div><h3 class="article-section__title" id="_i25"> Cell Proliferation and Viability Assays</h3><div class="NLM_p last">Studies were performed as previously reported.<a onclick="showRef(event, 'ref23 ref24 ref26'); return false;" href="javascript:void(0);" class="ref ref23 ref24 ref26">(23,24,26)</a> Briefly, cultured cells in 6-well or 96-well plates were treated with or without compounds for the indicated concentrations, and cells were harvested every 24 h up to 96 h for viable cell count by trypan blue exclusion phase-contrast microscopy, or after 72 h, cells were subjected to CyQuant cell proliferation assay following the manufacturer’s instructions (Invitrogen/ThermoFisher Scientific). In the case of the combination treatment, cells in culture were first treated with azetidine inhibitor, <b><u class="uu">7g</u></b>, for 6 h followed by treatment with docetaxel or cisplatin and then harvested after a total of 72 h treatment for CyQuant assay. Cell viability was normalized to the percentage of the control groups.</div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i26" class="anchor-spacer"></div><h3 class="article-section__title" id="_i26"> Clonogenic Survival Assays</h3><div class="NLM_p last">Colony survival assay was performed as previously reported.<a onclick="showRef(event, 'ref24 ref26'); return false;" href="javascript:void(0);" class="ref ref24 ref26">(24,26)</a> Briefly, cells were seeded as single-cell cultures in 6-well plates (250 cells per well), treated once the next day with compounds at the indicated concentrations and allowed to culture until large colonies were visible. Colonies were stained with crystal violet for 4 h and photographed.</div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i27" class="anchor-spacer"></div><h3 class="article-section__title" id="_i27"> Isothermal Titration Calorimetry (ITC)</h3><div class="NLM_p last">The ITC experiment was carried out as previously described<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> with some modification using Malvern Panalytical MicroCal PEAQ-ITC (United Kingdom). Studies were done at 25 °C. Briefly, STAT3 inhibitors, previously suspended in 100% DMSO, were diluted in 20 mM 4-(2-hydroxyethyl)-1-piperazineethanesulfonic acid (HEPES), 150 mM KCl buffer, so the final DMSO was 5%. To avoid buffer mismatch, STAT3 in HEPES buffer was diluted in HEPES buffer with 5% DMSO final concentration. Three hundred microliter volumes (300 μL) of 3.0 μM STAT3 were placed in the cell and titrated with 250 μM inhibitors. Titrations took place by injecting 2 μL inhibitor in a 2.5 min injection for the titration peak to return to the baseline. The <i>K</i><sub>D</sub> was calculated using the MicroCal PEAQ-ITC analysis software, as well as Prism GraphPad software, using the one-site model. Control experiments were carried out by titration of the inhibitors into buffer, buffer into STAT3, and buffer into buffer. The three controls were used as a composite for the ITC experiment to subtract the heat of dilution and background noise from the measurements.</div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i28" class="anchor-spacer"></div><h3 class="article-section__title" id="_i28"> General Methods for Chemistry</h3><div class="NLM_p last">All reagents and solvents were purchased from commercial sources and used without further purification. All moisture-sensitive reactions were performed under a static atmosphere of nitrogen or argon in oven-dried glassware. Tetrahydrofuran (THF), dichloromethane (DCM), diethyl ether (Et<sub>2</sub>O), toluene, and dimethylformamide (DMF) used in the reactions were dried by being passed through an SPS system. Other anhydrous solvents were purchased from commercial sources. Thin-layer chromatography (TLC) was performed on glass plates, 250–1000 μm. Flash column chromatography was performed on silica gel, 200–400 mesh. <sup>1</sup>H NMR spectra were obtained as CDCl<sub>3</sub>, CD<sub>3</sub>OD, or (CD<sub>3</sub>)<sub>2</sub>SO solutions using an Agilent 300MHz NMR spectrometer with an Agilent DD2 console, and chemical shifts were expressed in ä (ppm) using residual solvent (CDCl<sub>3</sub>, 7.26 ppm; CD<sub>3</sub>OD, 3.31 ppm; and (CD<sub>3</sub>)<sub>2</sub>SO, 2.50 ppm) as the reference standard. When peak multiplicities are reported, the following abbreviations are used: s (singlet), d (doublet), t (triplet), q (quartet), m (multiplet), br-s (broadened singlet), dd (doublet of doublets), and dt (doublet of triplets). Coupling constants, when reported, are reported in hertz (Hz). All compounds were analyzed by LC/MS (liquid chromatography/mass spectrometry) using an Agilent Triple Quad 640 LC/MS. Ionization was generally achieved via electron spray (ESI) unless otherwise indicated. The LC fraction detection consisted of a variable wavelength detector, and all tested compounds had purity greater than 95%. High-resolution mass spectral (HRMS) data was obtained for all tested compounds using either an Agilent 6200 LC/MSD TOF or an Agilent 6545 Q-TOF LC/MS, and reported exact masses were calculated based on an algorithm using MS (ESI) <i>m</i>/<i>z</i> for [M + H]<sup>+</sup> and [M + Na]<sup>+</sup> adducts and were within 5 ppm of the expected target mass. Chiral molecules were analyzed by chiral HPLC using Chiralpak AD-H or OD-H columns (4.6 mm × 250 mm, UV detection at 254 or 261nm); eluents used were hexane and <i>i</i>-PrOH.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-2"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i29"><a href="/doi/suppl/10.1021/acs.jmedchem.0c01705" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_15882" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_15882" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a href="/doi/10.1021/acs.jmedchem.0c01705?goto=supporting-info" class="ext-link">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c01705</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Supplementary results and discussion; chemistry; early leads (Figure S1); dose–response curves for the cell-free EMSA analysis for the most active compounds (Figure S2); EMSA IC50 and cell viability EC50 values for select esters and heterocyclics (Table S1) (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01705/suppl_file/jm0c01705_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01705/suppl_file/jm0c01705_si_002.csv" class="ext-link">CSV</a>)</p></li><li><p class="inline">Qualitative Analysis Report: Compound <b>5a</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01705/suppl_file/jm0c01705_si_003.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Qualitative Analysis Report: Compound <b>5o</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01705/suppl_file/jm0c01705_si_004.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Qualitative Analysis Report: Compound <b>7e</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01705/suppl_file/jm0c01705_si_005.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Qualitative Analysis Report: Compound <b>7f</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01705/suppl_file/jm0c01705_si_006.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Qualitative Analysis Report: Compound <b>7g</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01705/suppl_file/jm0c01705_si_007.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Qualitative Analysis Report: Compound <b>8i</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01705/suppl_file/jm0c01705_si_008.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Qualitative Analysis Report: Compound <b>9k</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01705/suppl_file/jm0c01705_si_009.pdf" class="ext-link">PDF</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01705/suppl_file/jm0c01705_si_001.pdf">jm0c01705_si_001.pdf (2.93 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01705/suppl_file/jm0c01705_si_002.csv">jm0c01705_si_002.csv (9.91 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01705/suppl_file/jm0c01705_si_003.pdf">jm0c01705_si_003.pdf (123.83 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01705/suppl_file/jm0c01705_si_004.pdf">jm0c01705_si_004.pdf (120.62 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01705/suppl_file/jm0c01705_si_005.pdf">jm0c01705_si_005.pdf (139.66 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01705/suppl_file/jm0c01705_si_006.pdf">jm0c01705_si_006.pdf (122.36 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01705/suppl_file/jm0c01705_si_007.pdf">jm0c01705_si_007.pdf (203.0 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01705/suppl_file/jm0c01705_si_008.pdf">jm0c01705_si_008.pdf (141.25 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01705/suppl_file/jm0c01705_si_009.pdf">jm0c01705_si_009.pdf (195.01 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.0c01705" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_21733" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_21733" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Francisco Lopez-Tapia</span> - <span class="hlFld-Affiliation affiliation">Cancer
Biology Program, University of Hawaii Cancer
Center, University of Hawaii, Manoa, Honolulu, Hawaii 96813, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Medicinal
Chemistry Leader, Department of Chemistry, University of Hawaii, Manoa, 2545 McCarthy Mall, Honolulu, Hawaii 9682, United States</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#086e7a69666b617b6b67266467786d72486b7b607b26677a6f"><span class="__cf_email__" data-cfemail="c4a2b6a5aaa7adb7a7abeaa8abb4a1be84a7b7acb7eaabb6a3">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">James Turkson</span> - <span class="hlFld-Affiliation affiliation">Department of Medicine, Division
of Oncology, 
    Cedars-Sinai Cancer, 
     and , Cedars-Sinai Medical
Center, 8700 Beverly Blvd, Davis 5065, Los Angeles, California 90048, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Cancer
Biology Program, University of Hawaii Cancer
Center, University of Hawaii, Manoa, Honolulu, Hawaii 96813, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-6964-1602" title="Orcid link">http://orcid.org/0000-0002-6964-1602</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#c4aea5a9a1b7eab0b1b6afb7abaa84a7b7acb7eaabb6a3"><span class="__cf_email__" data-cfemail="8ae0ebe7eff9a4fefff8e1f9e5e4cae9f9e2f9a4e5f8ed">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Christine Brotherton-Pleiss</span> - <span class="hlFld-Affiliation affiliation">Cancer
Biology Program, University of Hawaii Cancer
Center, University of Hawaii, Manoa, Honolulu, Hawaii 96813, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Medicinal
Chemistry Leader, Department of Chemistry, University of Hawaii, Manoa, 2545 McCarthy Mall, Honolulu, Hawaii 9682, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Peibin Yue</span> - <span class="hlFld-Affiliation affiliation">Department of Medicine, Division
of Oncology, 
    Cedars-Sinai Cancer, 
     and , Cedars-Sinai Medical
Center, 8700 Beverly Blvd, Davis 5065, Los Angeles, California 90048, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Cancer
Biology Program, University of Hawaii Cancer
Center, University of Hawaii, Manoa, Honolulu, Hawaii 96813, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yinsong Zhu</span> - <span class="hlFld-Affiliation affiliation">Department of Medicine, Division
of Oncology, 
    Cedars-Sinai Cancer, 
     and , Cedars-Sinai Medical
Center, 8700 Beverly Blvd, Davis 5065, Los Angeles, California 90048, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Kayo Nakamura</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry, University of Hawaii, Manoa, 2545 McCarthy Mall, Honolulu, Hawaii 9682, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Weiliang Chen</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry, University of Hawaii, Manoa, 2545 McCarthy Mall, Honolulu, Hawaii 9682, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Wenzhen Fu</span> - <span class="hlFld-Affiliation affiliation">Cancer
Biology Program, University of Hawaii Cancer
Center, University of Hawaii, Manoa, Honolulu, Hawaii 96813, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Chemistry, University of Hawaii, Manoa, 2545 McCarthy Mall, Honolulu, Hawaii 9682, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Casie Kubota</span> - <span class="hlFld-Affiliation affiliation">Cancer
Biology Program, University of Hawaii Cancer
Center, University of Hawaii, Manoa, Honolulu, Hawaii 96813, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jasmine Chen</span> - <span class="hlFld-Affiliation affiliation">Cancer
Biology Program, University of Hawaii Cancer
Center, University of Hawaii, Manoa, Honolulu, Hawaii 96813, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Felix Alonso-Valenteen</span> - <span class="hlFld-Affiliation affiliation">Cedars-Sinai Cancer, 
     and , Cedars-Sinai Medical
Center, 8700 Beverly Blvd, Davis 5065, Los Angeles, California 90048, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Biomedical Sciences, Cedars-Sinai Medical
Center, Los Angeles, California 90048, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Simoun Mikhael</span> - <span class="hlFld-Affiliation affiliation">Cedars-Sinai Cancer, 
     and , Cedars-Sinai Medical
Center, 8700 Beverly Blvd, Davis 5065, Los Angeles, California 90048, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Biomedical Sciences, Cedars-Sinai Medical
Center, Los Angeles, California 90048, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Lali Medina-Kauwe</span> - <span class="hlFld-Affiliation affiliation">Cedars-Sinai Cancer, 
     and , Cedars-Sinai Medical
Center, 8700 Beverly Blvd, Davis 5065, Los Angeles, California 90048, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Biomedical Sciences, Cedars-Sinai Medical
Center, Los Angeles, California 90048, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Marcus A. Tius</span> - <span class="hlFld-Affiliation affiliation">Cancer
Biology Program, University of Hawaii Cancer
Center, University of Hawaii, Manoa, Honolulu, Hawaii 96813, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Medicinal
Chemistry Leader, Department of Chemistry, University of Hawaii, Manoa, 2545 McCarthy Mall, Honolulu, Hawaii 9682, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-7092-8993" title="Orcid link">http://orcid.org/0000-0002-7092-8993</a></span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>C.B.-P. and P.Y. authors contributed equally to this work.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare the following competing financial interest(s): JT is a co-founder of Novella, LLC, which has licensed STAT3 IP. The remaining authors have no competing interests.<br /></br></div></li></ul></div><div class="ack" id="ACK-d7e2750-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i31">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_51115" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_51115" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">We thank all colleagues and members of our laboratory for the stimulating discussions and Dr. Joel Kawakami and Mabel Bernaldez of Chaminade University of Honolulu for their contributions to the design of the Cover Art. This work was supported by NIH/NCI R01 CA208851 (J.T.), LEIDOS Biomedical Research/NCI Contract 19X122Q (J.T.), and Cedars-Sinai Start-up funds (JT).</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations</td></tr><tr><td class="NLM_term">STAT</td><td class="NLM_def"><p class="first last">signal transducer and activator of transcription</p></td></tr><tr><td class="NLM_term">EMSA</td><td class="NLM_def"><p class="first last">electrophoretic mobility shift assay</p></td></tr><tr><td class="NLM_term">PARP</td><td class="NLM_def"><p class="first last">poly (ADP-ribose) polymerase</p></td></tr><tr><td class="NLM_term">VEGF</td><td class="NLM_def"><p class="first last">vascular endothelial growth factor</p></td></tr><tr><td class="NLM_term">GAPDH</td><td class="NLM_def"><p class="first last">glyceraldehyde 3-phosphate dehydrogenase</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i33">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_44247" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_44247" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 48 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Darnell, J. E.,  Jr.</span></span> <span> </span><span class="NLM_article-title">Reflections on Stat3, Stat5, and Stat6 as fat STATs</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>93</i></span>,  <span class="NLM_fpage">6221</span>– <span class="NLM_lpage">6224</span>, <span class="refDoi"> DOI: 10.1073/pnas.93.13.6221</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=10.1073%2Fpnas.93.13.6221" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=8692794" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=1%3ACAS%3A528%3ADyaK28XjvFejtrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=93&publication_year=1996&pages=6221-6224&author=J.+E.+Darnell&title=Reflections+on+Stat3%2C+Stat5%2C+and+Stat6+as+fat+STATs&doi=10.1073%2Fpnas.93.13.6221"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Reflections on STAT3, STAT5, and STAT6 as fat STATs</span></div><div class="casAuthors">Darnell, James E., Jr.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">93</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">6221-6224</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">A review with 45 refs. on the involvement of STAT proteins in wt. control with emphasis on the work of N. Ghilardi et al. entitled "Defective STAT signaling by the leptin receptor in diabetic mice" appearing in this issue of the Proceedings.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqhc3TK1Kwa87Vg90H21EOLACvtfcHk0lg1D9bim2laCQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28XjvFejtrw%253D&md5=2a061011708b2836362022dc78d82c4f</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1073%2Fpnas.93.13.6221&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.93.13.6221%26sid%3Dliteratum%253Aachs%26aulast%3DDarnell%26aufirst%3DJ.%2BE.%26atitle%3DReflections%2520on%2520Stat3%252C%2520Stat5%252C%2520and%2520Stat6%2520as%2520fat%2520STATs%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D1996%26volume%3D93%26spage%3D6221%26epage%3D6224%26doi%3D10.1073%2Fpnas.93.13.6221" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Darnell, J. E.,  Jr.</span></span> <span> </span><span class="NLM_article-title">The Jak-STAT pathway: summary of initial studies and recent advances</span>. <i>Recent Prog. Horm. Res.</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>51</i></span>,  <span class="NLM_fpage">391</span>– <span class="NLM_lpage">403</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=8701087" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=1%3ACAS%3A528%3ADyaK2sXktFKis78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=1996&pages=391-403&author=J.+E.+Darnell&title=The+Jak-STAT+pathway%3A+summary+of+initial+studies+and+recent+advances"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">The JAK-STAT pathway: summary of initial studies and recent advances</span></div><div class="casAuthors">Darnell, James E., Jr.</div><div class="citationInfo"><span class="NLM_cas:title">Recent Progress in Hormone Research</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">391-404</span>CODEN:
                <span class="NLM_cas:coden">RPHRA6</span>;
        ISSN:<span class="NLM_cas:issn">0079-9963</span>.
    
            (<span class="NLM_cas:orgname">Endocrine Society</span>)
        </div><div class="casAbstract">A review and a discussion with many refs.  Polypeptides redirect cells by causing immediate changes in gene transcription.  A direct connection between polypeptide receptors on the cell surface and genes in the nucleus has been provided by the recognition of the JAK-STAT pathway.  Results leading to our present understanding of this pathway are reviewed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpnoxpk0vC5vrVg90H21EOLACvtfcHk0lg1D9bim2laCQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXktFKis78%253D&md5=385fd1dfee38ec19ef9d325a11ab4c58</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DDarnell%26aufirst%3DJ.%2BE.%26atitle%3DThe%2520Jak-STAT%2520pathway%253A%2520summary%2520of%2520initial%2520studies%2520and%2520recent%2520advances%26jtitle%3DRecent%2520Prog.%2520Horm.%2520Res.%26date%3D1996%26volume%3D51%26spage%3D391%26epage%3D403" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Darnell, J. E.,  Jr.</span></span> <span> </span><span class="NLM_article-title">STATs and gene regulation</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>277</i></span>,  <span class="NLM_fpage">1630</span>– <span class="NLM_lpage">1635</span>, <span class="refDoi"> DOI: 10.1126/science.277.5332.1630</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=10.1126%2Fscience.277.5332.1630" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=9287210" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=1%3ACAS%3A528%3ADyaK2sXmtVSisLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=277&publication_year=1997&pages=1630-1635&author=J.+E.+Darnell&title=STATs+and+gene+regulation&doi=10.1126%2Fscience.277.5332.1630"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">STATs and gene regulation</span></div><div class="casAuthors">Darnell, James E., Jr.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, D. C.)</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">277</span>
        (<span class="NLM_cas:issue">5332</span>),
    <span class="NLM_cas:pages">1630-1635</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">A review with 100 refs.  STATs (signal transducers and activators of transcription) are a family of latent cytoplasmic proteins that are activated to participate in gene control when cells encounter various extracellular polypeptides.  Biochem. and mol. genetic explorations have defined a single tyrosine phosphorylation site and, in a dimeric partner mol., a Src homol. 2 (SH2) phosphotyrosine-binding domain, a DNA interaction domain, and a no. of protein-protein interaction domains (with receptors, other transcription factors, the transcription machinery, and perhaps a tyrosine phosphatase).  Mouse genetics expts. have defined crucial roles for each known mammalian STAT.  The discovery of a STAT in Drosophila, and most recently in Dictyostelium discoideum, implies an ancient evolutionary origin for this dual-function set of proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr2cN7IJWCm2rVg90H21EOLACvtfcHk0liVUQXqnadBMA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXmtVSisLo%253D&md5=ce5d495245adb0ccaab93ba990a7e017</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1126%2Fscience.277.5332.1630&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.277.5332.1630%26sid%3Dliteratum%253Aachs%26aulast%3DDarnell%26aufirst%3DJ.%2BE.%26atitle%3DSTATs%2520and%2520gene%2520regulation%26jtitle%3DScience%26date%3D1997%26volume%3D277%26spage%3D1630%26epage%3D1635%26doi%3D10.1126%2Fscience.277.5332.1630" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bromberg, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Darnell, J. E.,  Jr.</span></span> <span> </span><span class="NLM_article-title">The role of STATs in transcriptional control and their impact on cellular function</span>. <i>Oncogene</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">2468</span>– <span class="NLM_lpage">2473</span>, <span class="refDoi"> DOI: 10.1038/sj.onc.1203476</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=10.1038%2Fsj.onc.1203476" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=10851045" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=1%3ACAS%3A528%3ADC%252BD3cXktVGhsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2000&pages=2468-2473&author=J.+Brombergauthor=J.+E.+Darnell&title=The+role+of+STATs+in+transcriptional+control+and+their+impact+on+cellular+function&doi=10.1038%2Fsj.onc.1203476"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">The role of STATs in transcriptional control and their impact on cellular function</span></div><div class="casAuthors">Bromberg, Jacqueline; Darnell, James E, Jr.</div><div class="citationInfo"><span class="NLM_cas:title">Oncogene</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">2468-2473</span>CODEN:
                <span class="NLM_cas:coden">ONCNES</span>;
        ISSN:<span class="NLM_cas:issn">0950-9232</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review with 68 refs.  The STAT proteins (Signal Transducers and Activators of Transcription), were identified in the last decade as transcription factors which were crit. in mediating virtually all cytokine driven signaling.  These proteins are latent in the cytoplasm and become activated through tyrosine phosphorylation which typically occurs through cytokine receptor assocd. kinases (JAKs) or growth factor receptor tyrosine kinases.  Recently a no. of non-receptor tyrosine kinases (for example src and abl) have been found to cause STAT phosphorylation.  Phosphorylated STATs form homo- or hetero-dimers, enter the nucleus and working coordinately with other transcriptional co-activators or transcription factors lead to increased transcriptional initiation.  In normal cells and in animals, ligand dependent activation of the STATs is a transient process, lasting for several minutes to several hours.  In contrast, in many cancerous cell lines and tumors, where growth factor dysregulation is frequently at the heart of cellular transformation, the STAT proteins (in particular Stats 1, 3 and 5) are persistently tyrosine phosphorylated or activated.  The importance of STAT activation to growth control in expts. using anti-sense mols. or dominant neg. STAT protein encoding constructs performed in cell lines or studies in animals lacking specific STATs strongly indicate that STATs play an important role in controlling cell cycle progression and apoptosis.  Stat1 plays an important role in growth arrest, in promoting apoptosis and is implicated as a tumor suppressor; while Stats 3 and 5 are involved in promoting cell cycle progression and cellular transformation and preventing apoptosis.  Many questions remain including: (1) a better understanding of how the STAT proteins through assocn. with other factors increase transcription initiation; (2) a more complete definition of the sets of genes which are activated by different STATs and (3) how these sets of activated genes differ as a function of cell type.  Finally, in the context of many cancers, where STATs are frequently persistently activated, an understanding of the mechanisms leading to their constitutive activation and defining the potential importance of persistent STAT activation in human tumorigenesis remains.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpKoOhp17wSLLVg90H21EOLACvtfcHk0liVUQXqnadBMA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXktVGhsb4%253D&md5=b504533a9132be2af9c0a0a827b93063</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1038%2Fsj.onc.1203476&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.onc.1203476%26sid%3Dliteratum%253Aachs%26aulast%3DBromberg%26aufirst%3DJ.%26aulast%3DDarnell%26aufirst%3DJ.%2BE.%26atitle%3DThe%2520role%2520of%2520STATs%2520in%2520transcriptional%2520control%2520and%2520their%2520impact%2520on%2520cellular%2520function%26jtitle%3DOncogene%26date%3D2000%26volume%3D19%26spage%3D2468%26epage%3D2473%26doi%3D10.1038%2Fsj.onc.1203476" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Darnell, J. E.,  Jr</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kerr, I. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stark, G. R.</span></span> <span> </span><span class="NLM_article-title">Jak-STAT pathways and transcriptional activation in response to IFNs and other extracellular signaling proteins</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">1994</span>,  <span class="NLM_volume"><i>264</i></span>,  <span class="NLM_fpage">1415</span>– <span class="NLM_lpage">1421</span>, <span class="refDoi"> DOI: 10.1126/science.8197455</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=10.1126%2Fscience.8197455" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=8197455" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=1%3ACAS%3A528%3ADyaK2cXktlalsL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=264&publication_year=1994&pages=1415-1421&author=J.+E.+Darnellauthor=I.+M.+Kerrauthor=G.+R.+Stark&title=Jak-STAT+pathways+and+transcriptional+activation+in+response+to+IFNs+and+other+extracellular+signaling+proteins&doi=10.1126%2Fscience.8197455"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Jak-STAT pathways and transcriptional activation in response to IFNs and other extracellular signaling proteins</span></div><div class="casAuthors">Darnell, James E., Jr.; Kerr, Ian M.; Stark, George R.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">264</span>
        (<span class="NLM_cas:issue">5164</span>),
    <span class="NLM_cas:pages">1415-21</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    </div><div class="casAbstract">A review with 97 refs.  Through the study of transcriptional activation in response to interferon α (IFN-α) and interferon γ (IFN-γ), a previously unrecognized direct signal transduction pathway to the nucleus has been uncovered: IFN-receptor interaction at the cell surface leads to the activation of kinases of the Jak family that then phosphorylate substrate proteins called STATs (signal transducers and activators of transcription).  The phosphorylated STAT proteins move to the nucleus, bind specific DNA elements, and direct transcription.  Recognition of the mols. involved in the IFN-α and IFN-γ pathway has led to discoveries that a no. of STAT family members exist and that other polypeptide ligands also use the Jak-STAT mols. in signal transduction.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo64rvN_qWlmLVg90H21EOLACvtfcHk0liVUQXqnadBMA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2cXktlalsL4%253D&md5=f63d2c7bb3c1aeca6f36fef38aff713c</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1126%2Fscience.8197455&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.8197455%26sid%3Dliteratum%253Aachs%26aulast%3DDarnell%26aufirst%3DJ.%2BE.%26aulast%3DKerr%26aufirst%3DI.%2BM.%26aulast%3DStark%26aufirst%3DG.%2BR.%26atitle%3DJak-STAT%2520pathways%2520and%2520transcriptional%2520activation%2520in%2520response%2520to%2520IFNs%2520and%2520other%2520extracellular%2520signaling%2520proteins%26jtitle%3DScience%26date%3D1994%26volume%3D264%26spage%3D1415%26epage%3D1421%26doi%3D10.1126%2Fscience.8197455" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bromberg, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Horvath, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Besser, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lathem, W. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Darnell, J. E.,  Jr.</span></span> <span> </span><span class="NLM_article-title">Stat3 activation is required for cellular transformation by v-src</span>. <i>Mol. Cell. Biol.</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">2553</span>– <span class="NLM_lpage">2558</span>, <span class="refDoi"> DOI: 10.1128/MCB.18.5.2553</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=10.1128%2FMCB.18.5.2553" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=9566875" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=1%3ACAS%3A528%3ADyaK1cXis1Gntb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=1998&pages=2553-2558&author=J.+F.+Brombergauthor=C.+M.+Horvathauthor=D.+Besserauthor=W.+W.+Lathemauthor=J.+E.+Darnell&title=Stat3+activation+is+required+for+cellular+transformation+by+v-src&doi=10.1128%2FMCB.18.5.2553"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Stat3 activation is required for cellular transformation by v-src</span></div><div class="casAuthors">Bromberg, Jacqueline F.; Horvath, Curt M.; Besser, Daniel; Lathem, Wyndham W.; Darnell, James E., Jr.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">2553-2558</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">0270-7306</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Stat3 activation has been assocd. with cytokine-induced proliferation, anti-apoptosis, and transformation.  Constitutively activated Stat3 has been found in many human tumors as well as v-abl- and v-src-transformed cell lines.  Because of these correlations, the authors examd. directly the relation of activated Stat3 to cellular transformation and found that wild-type Stat3 enhances the transforming potential of v-src while three dominant neg. Stat3 mutants inhibit v-src transformation.  Stat3 wild-type or mutant proteins did not affect v-ras transformation.  The authors conclude that Stat3 has a necessary role in v-src transformation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoT_4zAEjnfeLVg90H21EOLACvtfcHk0liVUQXqnadBMA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXis1Gntb4%253D&md5=28a74826798a6e041e2c2eff0f2300ec</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1128%2FMCB.18.5.2553&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.18.5.2553%26sid%3Dliteratum%253Aachs%26aulast%3DBromberg%26aufirst%3DJ.%2BF.%26aulast%3DHorvath%26aufirst%3DC.%2BM.%26aulast%3DBesser%26aufirst%3DD.%26aulast%3DLathem%26aufirst%3DW.%2BW.%26aulast%3DDarnell%26aufirst%3DJ.%2BE.%26atitle%3DStat3%2520activation%2520is%2520required%2520for%2520cellular%2520transformation%2520by%2520v-src%26jtitle%3DMol.%2520Cell.%2520Biol.%26date%3D1998%26volume%3D18%26spage%3D2553%26epage%3D2558%26doi%3D10.1128%2FMCB.18.5.2553" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Darnell, J. E.</span></span> <span> </span><span class="NLM_article-title">Validating Stat3 in cancer therapy</span>. <i>Nat. Med.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">595</span>– <span class="NLM_lpage">596</span>, <span class="refDoi"> DOI: 10.1038/nm0605-595</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=10.1038%2Fnm0605-595" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=15937466" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=1%3ACAS%3A528%3ADC%252BD2MXkslOlu74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2005&pages=595-596&author=J.+E.+Darnell&title=Validating+Stat3+in+cancer+therapy&doi=10.1038%2Fnm0605-595"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Validating Stat3 in cancer therapy</span></div><div class="casAuthors">Darnell, James E.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Medicine (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">595-596</span>CODEN:
                <span class="NLM_cas:coden">NAMEFI</span>;
        ISSN:<span class="NLM_cas:issn">1078-8956</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrRlI633_ngRbVg90H21EOLACvtfcHk0lhdJxXEswsqoQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXkslOlu74%253D&md5=ddf70bdcb2c812f1e42b0f772f3d6be4</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1038%2Fnm0605-595&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnm0605-595%26sid%3Dliteratum%253Aachs%26aulast%3DDarnell%26aufirst%3DJ.%2BE.%26atitle%3DValidating%2520Stat3%2520in%2520cancer%2520therapy%26jtitle%3DNat.%2520Med.%26date%3D2005%26volume%3D11%26spage%3D595%26epage%3D596%26doi%3D10.1038%2Fnm0605-595" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Turkson, J.</span></span> <span> </span><span class="NLM_article-title">STAT proteins as novel targets for cancer drug discovery</span>. <i>Expert Opin. Ther. Targets</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">409</span>– <span class="NLM_lpage">422</span>, <span class="refDoi"> DOI: 10.1517/14728222.8.5.409</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=10.1517%2F14728222.8.5.409" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=15469392" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=1%3ACAS%3A528%3ADC%252BD2cXotFKhtb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2004&pages=409-422&author=J.+Turkson&title=STAT+proteins+as+novel+targets+for+cancer+drug+discovery&doi=10.1517%2F14728222.8.5.409"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">STAT proteins as novel targets for cancer drug discovery</span></div><div class="casAuthors">Turkson, James</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Therapeutic Targets</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">409-422</span>CODEN:
                <span class="NLM_cas:coden">EOTTAO</span>;
        ISSN:<span class="NLM_cas:issn">1472-8222</span>.
    
            (<span class="NLM_cas:orgname">Ashley Publications Ltd.</span>)
        </div><div class="casAbstract">A review with refs.  Signal transducer and activator of transcription (STAT) proteins are latent cytoplasmic transcription factors that were discovered in the context of cytokine and growth factor signalling.  Normal STAT signalling is tightly controlled with finite kinetics, which is in keeping with std. cellular responses.  However, persistent STAT activation has also been obsd. and is frequently assocd. with malignant transformation.  Constitutive activation of STAT proteins, notably of Stat3 and Stat5, is detected in many human tumor cells and cells transformed by oncoproteins that activate tyrosine kinase signalling pathways.  It is well-established that constitutively active Stat3 is one of the mol. abnormalities that has a causal role in oncogenesis.  Aberrant Stat3 promotes uncontrolled growth and survival through dys-regulation of gene expression, including cyclin D1, c-Myc, Bcl-xL, Mcl-1 and survivin genes, and thereby contributes to oncogenesis.  Moreover, recent studies reveal that persistently active Stat3 induces tumor angiogenesis by upregulation of vascular endothelial growth factor induction, and modulates immune functions in favor of tumor immune evasion.  Overall, studies have validated Stat3 as a novel target for cancer therapy, and hence provided the rationale for developing small-mol. Stat3 inhibitors.  This review will discuss current evidence for the crit. role of aberrant STAT signalling in malignant transformation, and examine the validity as well as the therapeutic potential of Stat3 as a cancer target.  An update on the efforts to develop novel Stat3 inhibitors for therapeutic application will also be provided.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpKUj4aPVco_rVg90H21EOLACvtfcHk0lhdJxXEswsqoQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXotFKhtb0%253D&md5=c524b36db31f3de66430595fc55fbe7c</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1517%2F14728222.8.5.409&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1517%252F14728222.8.5.409%26sid%3Dliteratum%253Aachs%26aulast%3DTurkson%26aufirst%3DJ.%26atitle%3DSTAT%2520proteins%2520as%2520novel%2520targets%2520for%2520cancer%2520drug%2520discovery%26jtitle%3DExpert%2520Opin.%2520Ther.%2520Targets%26date%3D2004%26volume%3D8%26spage%3D409%26epage%3D422%26doi%3D10.1517%2F14728222.8.5.409" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Turkson, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jove, R.</span></span> <span> </span><span class="NLM_article-title">STAT proteins: novel molecular targets for cancer drug discovery</span>. <i>Oncogene</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">6613</span>– <span class="NLM_lpage">6626</span>, <span class="refDoi"> DOI: 10.1038/sj.onc.1204086</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=10.1038%2Fsj.onc.1204086" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=11426647" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=1%3ACAS%3A528%3ADC%252BD3MXhsF2rsro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2000&pages=6613-6626&author=J.+Turksonauthor=R.+Jove&title=STAT+proteins%3A+novel+molecular+targets+for+cancer+drug+discovery&doi=10.1038%2Fsj.onc.1204086"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">STAT proteins: novel molecular targets for cancer drug discovery</span></div><div class="casAuthors">Turkson, James; Jove, Richard</div><div class="citationInfo"><span class="NLM_cas:title">Oncogene</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">56</span>),
    <span class="NLM_cas:pages">6613-6626</span>CODEN:
                <span class="NLM_cas:coden">ONCNES</span>;
        ISSN:<span class="NLM_cas:issn">0950-9232</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review with 101 refs.  Signal Transducers and Activators of Transcription (STATs) are a family of cytoplasmic proteins with roles as signal messengers and transcription factors that participate in normal cellular responses to cytokines and growth factors.  Frequently, however, abnormal activity of certain STAT family members, particularly Stat3 and Stat5, is assocd. with a wide variety of human malignancies, including hematol., breast, head and neck, and prostate cancers.  Application of mol. biol. and pharmacol. tools in disease-relevant models has confirmed Stat3 as having a causal role in oncogenesis, and provided validation of Stat3 as a target for cancer drug discovery and therapeutic intervention.  Furthermore, a constitutively-active mutant form of Stat3 is sufficient to induce oncogenic transformation of cells, which form tumors in vivo.  Constitutive activation of Stat3 signaling is accompanied by upregulation of cyclin D1, c-Myc, and Bcl-x, changes consistent with subversion of normal cellular growth and survival control mechanisms.  Block of constitutive Stat3 signaling results in growth inhibition and apoptosis of Stat3-pos. tumor cells in vitro and in vivo.  The obsd. dependence of certain tumors on constitutive Stat3 signaling for growth and survival has wide implications for cancer therapy, offering the potential for preferential tumor cell killing.  This review evaluates constitutive Stat3 activation as a "cancer-causing" factor, and proposes a no. of mol. strategies for targeting Stat3 signaling for therapeutic intervention.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoffTICt_aLhLVg90H21EOLACvtfcHk0lhdJxXEswsqoQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXhsF2rsro%253D&md5=2011fb621ef70547ddbbef2ed253df98</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1038%2Fsj.onc.1204086&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.onc.1204086%26sid%3Dliteratum%253Aachs%26aulast%3DTurkson%26aufirst%3DJ.%26aulast%3DJove%26aufirst%3DR.%26atitle%3DSTAT%2520proteins%253A%2520novel%2520molecular%2520targets%2520for%2520cancer%2520drug%2520discovery%26jtitle%3DOncogene%26date%3D2000%26volume%3D19%26spage%3D6613%26epage%3D6626%26doi%3D10.1038%2Fsj.onc.1204086" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Darnell, J. E.,  Jr.</span></span> <span> </span><span class="NLM_article-title">Transcription factors as targets for cancer therapy</span>. <i>Nat. Rev. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">740</span>– <span class="NLM_lpage">749</span>, <span class="refDoi"> DOI: 10.1038/nrc906</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=10.1038%2Fnrc906" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=12360277" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=1%3ACAS%3A528%3ADC%252BD38XnsVGrtbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2002&pages=740-749&author=J.+E.+Darnell&title=Transcription+factors+as+targets+for+cancer+therapy&doi=10.1038%2Fnrc906"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Transcription factors as targets for cancer therapy</span></div><div class="casAuthors">Darnell, James E., Jr.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">740-749</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  A limited list of transcription factors are overactive in most human cancer cells, which makes them targets for the development of anticancer drugs.  That they are the most direct and hopeful targets for treating cancer is proposed, and this is supported by the fact that there are many more human oncogenes in signaling pathways than there are oncogenic transcription factors.  But how could specific transcription-factor activity be inhibited.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrJxyqucgv0aLVg90H21EOLACvtfcHk0lhdJxXEswsqoQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XnsVGrtbo%253D&md5=239132f634b138965e3a11c63f7cab76</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1038%2Fnrc906&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc906%26sid%3Dliteratum%253Aachs%26aulast%3DDarnell%26aufirst%3DJ.%2BE.%26atitle%3DTranscription%2520factors%2520as%2520targets%2520for%2520cancer%2520therapy%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2002%26volume%3D2%26spage%3D740%26epage%3D749%26doi%3D10.1038%2Fnrc906" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Turkson, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ryan, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haura, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laudano, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sebti, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamilton, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jove, R.</span></span> <span> </span><span class="NLM_article-title">Phosphotyrosyl peptides block Stat3-mediated DNA binding activity, gene regulation, and cell transformation</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>276</i></span>,  <span class="NLM_fpage">45443</span>– <span class="NLM_lpage">45455</span>, <span class="refDoi"> DOI: 10.1074/jbc.M107527200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=10.1074%2Fjbc.M107527200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=11579100" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=1%3ACAS%3A528%3ADC%252BD3MXovFent7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=276&publication_year=2001&pages=45443-45455&author=J.+Turksonauthor=D.+Ryanauthor=J.+S.+Kimauthor=Y.+Zhangauthor=Z.+Chenauthor=E.+Hauraauthor=A.+Laudanoauthor=S.+Sebtiauthor=A.+D.+Hamiltonauthor=R.+Jove&title=Phosphotyrosyl+peptides+block+Stat3-mediated+DNA+binding+activity%2C+gene+regulation%2C+and+cell+transformation&doi=10.1074%2Fjbc.M107527200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Phosphotyrosyl peptides block Stat3-mediated DNA binding activity, gene regulation, and cell transformation</span></div><div class="casAuthors">Turkson, James; Ryan, Declan; Kim, Joon S.; Zhang, Yi; Chen, Zhi; Haura, Eric; Laudano, Andy; Sebti, Said; Hamilton, Andrew D.; Jove, Richard</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">276</span>
        (<span class="NLM_cas:issue">48</span>),
    <span class="NLM_cas:pages">45443-45455</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Signal transducers and activators of transcription (STATs) comprise a family of cytoplasmic signaling proteins that participates in normal cellular responses to cytokines and growth factors.  Frequently, however, constitutive activation of certain STAT family members, particularly Stat3, has accompanied a wide variety of human malignancies.  To identify small mol. inhibitors of Stat3, we investigated the ability of the Stat3 SH2 domain-binding peptide, PY*LKTK (where Y* represents phosphotyrosine), to disrupt Stat3 activity in vitro.  The presence of PY*LKTK, but not PYLKTK or PFLKTK, in nuclear exts. results in significant redn. in the levels of DNA binding activities of Stat3, to a lesser extent of Stat1, and with no effect on that of Stat5.  Analyses of alanine scanning mutagenesis and deletion derivs. of PY*LKTK reveal that the Leu residue at the Y + 1 position and a substituent at the Y - 1 position (but not necessarily Pro) are essential for the disruption of active Stat3, thereby mapping the min. active sequence to the tripeptide, XY*L.  Studies involving bead-coupled PY*LKTK peptide demonstrate that this phosphopeptide directly complexes with Stat3 monomers in vitro, suggesting that PY*LKTK disrupts Stat3:Stat3 dimers.  As evidence for the functional importance of peptide-directed inhibition of Stat3, PY*LKTK-mts (mts, membrane translocating sequence) selectively inhibits constitutive and ligand-induced Stat3 activation in vivo.  Furthermore, PY*LKTK-mts suppresses transformation by the Src oncoprotein, which has been shown previously to require constitutive Stat3 activation.  Altogether, we have identified a minimal peptide that inhibits Stat3 signaling and provides the conceptual basis for use of this peptide as a lead for novel peptidomimetic drug design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo_r1dIkvcURrVg90H21EOLACvtfcHk0lh25S6boXho1g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXovFent7g%253D&md5=1d39e24e12f8af82715fec8d3c466cbd</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M107527200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M107527200%26sid%3Dliteratum%253Aachs%26aulast%3DTurkson%26aufirst%3DJ.%26aulast%3DRyan%26aufirst%3DD.%26aulast%3DKim%26aufirst%3DJ.%2BS.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DHaura%26aufirst%3DE.%26aulast%3DLaudano%26aufirst%3DA.%26aulast%3DSebti%26aufirst%3DS.%26aulast%3DHamilton%26aufirst%3DA.%2BD.%26aulast%3DJove%26aufirst%3DR.%26atitle%3DPhosphotyrosyl%2520peptides%2520block%2520Stat3-mediated%2520DNA%2520binding%2520activity%252C%2520gene%2520regulation%252C%2520and%2520cell%2520transformation%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2001%26volume%3D276%26spage%3D45443%26epage%3D45455%26doi%3D10.1074%2Fjbc.M107527200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Turkson, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glenn, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haura, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sebti, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamilton, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jove, R.</span></span> <span> </span><span class="NLM_article-title">Novel peptidomimetic inhibitors of signal transducer and activator of transcription 3 dimerization and biological activity</span>. <i>Mol. Cancer Ther.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">261</span>– <span class="NLM_lpage">269</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=15026546" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=1%3ACAS%3A528%3ADC%252BD2cXitFCns74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2004&pages=261-269&author=J.+Turksonauthor=J.+S.+Kimauthor=S.+Zhangauthor=J.+Yuanauthor=M.+Huangauthor=M.+Glennauthor=E.+Hauraauthor=S.+Sebtiauthor=A.+D.+Hamiltonauthor=R.+Jove&title=Novel+peptidomimetic+inhibitors+of+signal+transducer+and+activator+of+transcription+3+dimerization+and+biological+activity"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Novel peptidomimetic inhibitors of signal transducer and activator of transcription 3 dimerization and biological activity</span></div><div class="casAuthors">Turkson, James; Kim, Joon S.; Zhang, Shumin; Yuan, Jing; Huang, Mei; Glenn, Matthew; Haura, Eric; Sebti, Said; Hamilton, Andrew D.; Jove, Richard</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer Therapeutics</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">261-269</span>CODEN:
                <span class="NLM_cas:coden">MCTOCF</span>;
        ISSN:<span class="NLM_cas:issn">1535-7163</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">The crit. role of signal transducer and activator of transcription 3 (Stat3) in the growth and survival of human tumor cells identifies it as a promising target for cancer drug discovery.  We previously identified a Stat3 SH2 domain-binding phosphopeptide, PY*LKTK, and its tripeptide derivs., PY*L and AY*L (where Y* represents phosphotyrosine), which inhibit Stat3 biochem. activity and biol. function.  Here, we report novel peptidomimetic compds. based on PY*L (or AY*L) with substitution of the Y-1 residue by benzyl, pyridyl, or pyrazinyl derivs. that are selective and greater than 5-fold more potent in disrupting Stat3 activity in vitro than lead tripeptides.  The biol. activities of these derivs. mirror that originally obsd. for peptides.  In this context, the representative peptidomimetic ISS 610 with 4-cyanobenzoate substitution inhibits constitutive Stat3 activity in Src-transformed mouse fibroblasts and human breast and lung carcinoma cells.  This effect is not evident with the non-phosphorylated counterpart, ISS 610NP, consistent with interaction of peptidomimetics with the SH2 domain of Stat3.  Moreover, ISS 610 induces cell growth inhibition and apoptosis of Src-transformed fibroblasts that contain persistently active Stat3.  We present the first report of a peptidomimetic approach to design of small-mol. inhibitors of Stat3 that are also among the first examples of disruptors of transcription factor dimerization with the potential for novel cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpoTmPO43TLyrVg90H21EOLACvtfcHk0lh25S6boXho1g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXitFCns74%253D&md5=b56e7dec52b6107d7f98d913376bdd0d</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DTurkson%26aufirst%3DJ.%26aulast%3DKim%26aufirst%3DJ.%2BS.%26aulast%3DZhang%26aufirst%3DS.%26aulast%3DYuan%26aufirst%3DJ.%26aulast%3DHuang%26aufirst%3DM.%26aulast%3DGlenn%26aufirst%3DM.%26aulast%3DHaura%26aufirst%3DE.%26aulast%3DSebti%26aufirst%3DS.%26aulast%3DHamilton%26aufirst%3DA.%2BD.%26aulast%3DJove%26aufirst%3DR.%26atitle%3DNovel%2520peptidomimetic%2520inhibitors%2520of%2520signal%2520transducer%2520and%2520activator%2520of%2520transcription%25203%2520dimerization%2520and%2520biological%2520activity%26jtitle%3DMol.%2520Cancer%2520Ther.%26date%3D2004%26volume%3D3%26spage%3D261%26epage%3D269" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Siddiquee, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guida, W. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blaskovich, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greedy, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lawrence, H. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yip, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jove, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McLaughlin, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lawrence, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sebti, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turkson, J.</span></span> <span> </span><span class="NLM_article-title">Selective chemical probe inhibitor of Stat3, identified through structure-based virtual screening, induces antitumor activity</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>104</i></span>,  <span class="NLM_fpage">7391</span>– <span class="NLM_lpage">7396</span>, <span class="refDoi"> DOI: 10.1073/pnas.0609757104</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=10.1073%2Fpnas.0609757104" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=17463090" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=1%3ACAS%3A528%3ADC%252BD2sXlslWqsbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=104&publication_year=2007&pages=7391-7396&author=K.+Siddiqueeauthor=S.+Zhangauthor=W.+C.+Guidaauthor=M.+A.+Blaskovichauthor=B.+Greedyauthor=H.+R.+Lawrenceauthor=M.+L.+Yipauthor=R.+Joveauthor=M.+M.+McLaughlinauthor=N.+J.+Lawrenceauthor=S.+M.+Sebtiauthor=J.+Turkson&title=Selective+chemical+probe+inhibitor+of+Stat3%2C+identified+through+structure-based+virtual+screening%2C+induces+antitumor+activity&doi=10.1073%2Fpnas.0609757104"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Elective chemical probe inhibitor of Stat3, identified through structure-based virtual screening, induces antitumor activity</span></div><div class="casAuthors">Siddiquee, Khandaker; Zhang, Shumin; Guida, Wayne C.; Blaskovich, Michelle A.; Greddy, Benjamin; Lawrence, Harshani R.; Yip, M. L. Richard; Jove, Richard; McLaughlin, Mark M.; Lawrence, Nicholas J.; Sebti, Said M.; Turkson, James</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">104</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">7391-7396</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">531-201 (NSC 74859) is a chem. probe inhibitor of Stat3 activity, which was identified from the National Cancer Institute chem. libraries by using structure-based virtual screening with a computer model of the Stat3 SH2 domain bound to its Stat3 phosphotyrosine peptide derived from the x-ray crystal structure of the Stat3β homodimer.  S3I-201 inhibits Stat3 · Stat3 complex formation and Stat3 DNA-binding and transcriptional activities.  Furthermore, S3I-201 inhibits growth and induces apoptosis preferentially in tumor cells that contain persistently activated Stat3.  Constitutively dimerized and active Stat3C and Stat3 SH2 domain rescue tumor cells from S3I-201-induced apoptosis.  Finally, S3I-201 inhibits the expression of the Stat3-regulated genes encoding cyclin D1, Bcl-xL, and surviving and inhibits the growth of human breast tumors in vivo.  These findings strongly suggest that the antitumor activity of S3I-201 is mediated in part through inhibition of aberrant Stat3 activation and provide the proof-of-concept for the potential clin. use of Stat3 inhibitors such as S3I-201 in tumors harboring aberrant Stat3.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpAfmqUniFENbVg90H21EOLACvtfcHk0lh25S6boXho1g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXlslWqsbY%253D&md5=85decf56fc7852990f56c6f64165ad9d</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0609757104&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0609757104%26sid%3Dliteratum%253Aachs%26aulast%3DSiddiquee%26aufirst%3DK.%26aulast%3DZhang%26aufirst%3DS.%26aulast%3DGuida%26aufirst%3DW.%2BC.%26aulast%3DBlaskovich%26aufirst%3DM.%2BA.%26aulast%3DGreedy%26aufirst%3DB.%26aulast%3DLawrence%26aufirst%3DH.%2BR.%26aulast%3DYip%26aufirst%3DM.%2BL.%26aulast%3DJove%26aufirst%3DR.%26aulast%3DMcLaughlin%26aufirst%3DM.%2BM.%26aulast%3DLawrence%26aufirst%3DN.%2BJ.%26aulast%3DSebti%26aufirst%3DS.%2BM.%26aulast%3DTurkson%26aufirst%3DJ.%26atitle%3DSelective%2520chemical%2520probe%2520inhibitor%2520of%2520Stat3%252C%2520identified%2520through%2520structure-based%2520virtual%2520screening%252C%2520induces%2520antitumor%2520activity%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2007%26volume%3D104%26spage%3D7391%26epage%3D7396%26doi%3D10.1073%2Fpnas.0609757104" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Siddiquee, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gunning, P. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glenn, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Katt, W. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schrock, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sebti, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jove, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamilton, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turkson, J.</span></span> <span> </span><span class="NLM_article-title">An oxazole-based small-molecule Stat3 inhibitor modulates Stat3 stability and processing and induces antitumor cell effects</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">787</span>– <span class="NLM_lpage">798</span>, <span class="refDoi"> DOI: 10.1021/cb7001973</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb7001973" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhsVCrs7jL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2007&pages=787-798&author=K.+A.+Siddiqueeauthor=P.+T.+Gunningauthor=M.+Glennauthor=W.+P.+Kattauthor=S.+Zhangauthor=C.+Schrockauthor=S.+M.+Sebtiauthor=R.+Joveauthor=A.+D.+Hamiltonauthor=J.+Turkson&title=An+oxazole-based+small-molecule+Stat3+inhibitor+modulates+Stat3+stability+and+processing+and+induces+antitumor+cell+effects&doi=10.1021%2Fcb7001973"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">An Oxazole-Based Small-Molecule Stat3 Inhibitor Modulates Stat3 Stability and Processing and Induces Antitumor Cell Effects</span></div><div class="casAuthors">Siddiquee, Khandaker A. Z.; Gunning, Patrick T.; Glenn, Matthew; Katt, William P.; Zhang, Shumin; Schroeck, Christopher; Sebti, Said M.; Jove, Richard; Hamilton, Andrew D.; Turkson, James</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">787-798</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Stat3 is hyperactivated in many human tumors and represents a valid target for anticancer drug design.  The authors present a novel small-mol. Stat3 inhibitor, S3I-M2001, and describe the dynamics of intracellular processing of activated Stat3 within the context of the biochem. and biol. effects of the Stat3 inhibitor.  S3I-M2001 is an oxazole-based peptidomimetic of the Stat3 Src homol. (SH) 2 domain-binding phosphotyrosine peptide that selectively disrupts active Stat3:Stat3 dimers.  Consequently, hyperactivated Stat3, which hitherto occurs as "dotlike" structures of nuclear bodies, undergoes an early aggregation into nonfunctional perinuclear aggresomes and a late-phase proteasome-mediated degrdn. in malignant cells treated with S3I-M2001.  Thus, S3I-M2001 inhibited Stat3-dependent transcriptional regulation of tumor survival genes, such as Bcl-xL.  Furthermore, Stat3-dependent malignant transformation, survival, and migration and invasion of mouse and human cancer cells harboring persistently activated Stat3 were inhibited by S3I-M2001.  Finally, S3I-M2001 inhibited growth of human breast tumor xenografts.  The study identifies a novel Stat3 inhibitor, S3I-M2001, with antitumor cell effects mediated in part through a biphasic loss of functional Stat3.  The study represents the first on intracellular Stat3 stability and processing following inhibition by a small mol. that has significant antitumor activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGojQxo7tV9euLVg90H21EOLACvtfcHk0lgI7bBmED1VuA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhsVCrs7jL&md5=db079dcd7ca8233cc6e62596bb1672f5</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1021%2Fcb7001973&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb7001973%26sid%3Dliteratum%253Aachs%26aulast%3DSiddiquee%26aufirst%3DK.%2BA.%26aulast%3DGunning%26aufirst%3DP.%2BT.%26aulast%3DGlenn%26aufirst%3DM.%26aulast%3DKatt%26aufirst%3DW.%2BP.%26aulast%3DZhang%26aufirst%3DS.%26aulast%3DSchrock%26aufirst%3DC.%26aulast%3DSebti%26aufirst%3DS.%2BM.%26aulast%3DJove%26aufirst%3DR.%26aulast%3DHamilton%26aufirst%3DA.%2BD.%26aulast%3DTurkson%26aufirst%3DJ.%26atitle%3DAn%2520oxazole-based%2520small-molecule%2520Stat3%2520inhibitor%2520modulates%2520Stat3%2520stability%2520and%2520processing%2520and%2520induces%2520antitumor%2520cell%2520effects%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2007%26volume%3D2%26spage%3D787%26epage%3D798%26doi%3D10.1021%2Fcb7001973" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Song, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, J.</span></span> <span> </span><span class="NLM_article-title">A low-molecular-weight compound discovered through virtual database screening inhibits Stat3 function in breast cancer cells</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>102</i></span>,  <span class="NLM_fpage">4700</span>– <span class="NLM_lpage">4705</span>, <span class="refDoi"> DOI: 10.1073/pnas.0409894102</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=10.1073%2Fpnas.0409894102" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=15781862" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=1%3ACAS%3A528%3ADC%252BD2MXjt1Oiu78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=102&publication_year=2005&pages=4700-4705&author=H.+Songauthor=R.+Wangauthor=S.+Wangauthor=J.+Lin&title=A+low-molecular-weight+compound+discovered+through+virtual+database+screening+inhibits+Stat3+function+in+breast+cancer+cells&doi=10.1073%2Fpnas.0409894102"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">A low-molecular-weight compound discovered through virtual database screening inhibits stat3 function in breast cancer cells</span></div><div class="casAuthors">Song, Hui; Wang, Renxiao; Wang, Shaomeng; Lin, Jiayuh</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">102</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">4700-4705</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">This study focused on the screening of small-mol. inhibitors that target signal transducers and activators of transcription 3 (Stat3) in human breast carcinoma.  The constitutive activation of Stat3 is frequently detected in human breast cancer cell lines as well as clin. breast cancer specimens and may play an important role in the oncogenesis of breast carcinoma.  Activated Stat3 may participate in oncogenesis by stimulating cell proliferation, promoting tumor angiogenesis, and resisting apoptosis.  Because a variety of human cancers are assocd. with constitutively active Stat3, Stat3 represents an attractive target for cancer therapy.  In this study, of the nearly 429,000 compds. screened by virtual database screening, chem. samples of top 100 compds. identified as candidate small-mol. inhibitors of Stat3 were evaluated by using Stat3-dependent luciferase reporter as well as other cell-based assays.  Through serial functional evaluation based on our established cell-based assays, one compd., termed STA-21, was identified as the best match for our selection criteria.  Further investigation demonstrated that STA-21 inhibits Stat3 DNA binding activity, Stat3 dimerization, and Stat3-dependent luciferase activity.  Moreover, STA-21 reduces the survival of breast carcinoma cells with constitutive Stat3 signaling but has minimal effect on the cells in which constitutive Stat3 signaling is absent.  Together, these results demonstrate that STA-21 inhibits breast cancer cells that express constitutively active Stat3.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGriWe9nsAbh67Vg90H21EOLACvtfcHk0lgI7bBmED1VuA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXjt1Oiu78%253D&md5=fe7108bee9cddbcdd75a67dd93e19e63</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0409894102&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0409894102%26sid%3Dliteratum%253Aachs%26aulast%3DSong%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DR.%26aulast%3DWang%26aufirst%3DS.%26aulast%3DLin%26aufirst%3DJ.%26atitle%3DA%2520low-molecular-weight%2520compound%2520discovered%2520through%2520virtual%2520database%2520screening%2520inhibits%2520Stat3%2520function%2520in%2520breast%2520cancer%2520cells%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2005%26volume%3D102%26spage%3D4700%26epage%3D4705%26doi%3D10.1073%2Fpnas.0409894102" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McMurray, J. S.</span></span> <span> </span><span class="NLM_article-title">A new small-molecule Stat3 inhibitor</span>. <i>Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">1123</span>– <span class="NLM_lpage">1124</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2006.11.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=10.1016%2Fj.chembiol.2006.11.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=17113993" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=1%3ACAS%3A528%3ADC%252BD28Xht1Cis7bM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2006&pages=1123-1124&author=J.+S.+McMurray&title=A+new+small-molecule+Stat3+inhibitor&doi=10.1016%2Fj.chembiol.2006.11.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">A New Small-Molecule Stat3 Inhibitor</span></div><div class="casAuthors">McMurray, John S.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1123-1124</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">In this issue of Chem. & Biol., Schust et al. report the discovery of a small mol. (Stattic) that inhibits the binding of a high affinity phosphopeptide for the SH2 domain of Stat3.  Stattic is a new tool for studying Stat3 signaling and demonstrates that the SH2 domain is not a dead target.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpznjm1p_Pt8LVg90H21EOLACvtfcHk0lj6-AdmuUYdUg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xht1Cis7bM&md5=c7744bbffb93abfd6a51f838a49b74f0</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2006.11.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2006.11.001%26sid%3Dliteratum%253Aachs%26aulast%3DMcMurray%26aufirst%3DJ.%2BS.%26atitle%3DA%2520new%2520small-molecule%2520Stat3%2520inhibitor%26jtitle%3DChem.%2520Biol.%26date%3D2006%26volume%3D13%26spage%3D1123%26epage%3D1124%26doi%3D10.1016%2Fj.chembiol.2006.11.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ren, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cabell, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schaefer, T. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McMurray, J. S.</span></span> <span> </span><span class="NLM_article-title">Identification of a high-affinity phosphopeptide inhibitor of Stat3</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">633</span>– <span class="NLM_lpage">636</span>, <span class="refDoi"> DOI: 10.1016/S0960-894X(02)01050-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=10.1016%2FS0960-894X%2802%2901050-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=12639546" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=1%3ACAS%3A528%3ADC%252BD3sXhvV2gtLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2003&pages=633-636&author=Z.+Renauthor=L.+A.+Cabellauthor=T.+S.+Schaeferauthor=J.+S.+McMurray&title=Identification+of+a+high-affinity+phosphopeptide+inhibitor+of+Stat3&doi=10.1016%2FS0960-894X%2802%2901050-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of a High-Affinity Phosphopeptide Inhibitor of Stat3</span></div><div class="casAuthors">Ren, Zhiyong; Cabell, Larry A.; Schaefer, Timothy S.; McMurray, John S.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">633-636</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ltd.</span>)
        </div><div class="casAbstract">Stat3 is a latent transcription factor that exhibits elevated activity in a variety of human cancers.  To find a lead peptide for peptidomimetic drug development we synthesized and tested phosphopeptides derived from known receptor docking sites and found Y(p)LPQTV as the optimal sequence.  SAR studies showed that each residue from pY to pY+3 provided binding energy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpKMeS5w7beErVg90H21EOLACvtfcHk0lj6-AdmuUYdUg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXhvV2gtLc%253D&md5=8798d979feeb7f5d8073ce3d43cebaf0</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1016%2FS0960-894X%2802%2901050-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0960-894X%252802%252901050-8%26sid%3Dliteratum%253Aachs%26aulast%3DRen%26aufirst%3DZ.%26aulast%3DCabell%26aufirst%3DL.%2BA.%26aulast%3DSchaefer%26aufirst%3DT.%2BS.%26aulast%3DMcMurray%26aufirst%3DJ.%2BS.%26atitle%3DIdentification%2520of%2520a%2520high-affinity%2520phosphopeptide%2520inhibitor%2520of%2520Stat3%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2003%26volume%3D13%26spage%3D633%26epage%3D636%26doi%3D10.1016%2FS0960-894X%2802%2901050-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McMurray, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mandal, P. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liao, W. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ren, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span> <span> </span><span class="NLM_article-title">Inhibition of Stat3 by cell-permeable peptidomimetic prodrugs targeted to its SH2 domain</span>. <i>Adv. Exp Med. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>611</i></span>,  <span class="NLM_fpage">545</span>– <span class="NLM_lpage">546</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=10.1007%2F978-0-387-73657-0_239" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=19400306" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtlWrsLjL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=611&publication_year=2009&pages=545-546&author=J.+S.+McMurrayauthor=P.+K.+Mandalauthor=W.+S.+Liaoauthor=Z.+Renauthor=X.+Chen&title=Inhibition+of+Stat3+by+cell-permeable+peptidomimetic+prodrugs+targeted+to+its+SH2+domain"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of Stat3 by cell-permeable peptidomimetic prodrugs targeted to its SH2 domain</span></div><div class="casAuthors">McMurray, John S.; Mandal, Pijus K.; Liao, Warren S.; Ren, Zhiyong; Chen, Xiaomin</div><div class="citationInfo"><span class="NLM_cas:title">Advances in Experimental Medicine and Biology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">611</span>
        (<span class="NLM_cas:issue">Peptides for Youth</span>),
    <span class="NLM_cas:pages">545-546</span>CODEN:
                <span class="NLM_cas:coden">AEMBAP</span>;
        ISSN:<span class="NLM_cas:issn">0065-2598</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">This study performed structure-affinity relationship anal. on truncated 2 to det. the pharmacophores and to gain information on the conformation in the bound state.  The Stat3 SH2 domain was targeted to create new anti-cancer chemotherapeutic agents.  A phosphohexapeptide was generally converted into cell-permeable, phosphatase stable peptidomimetics that inhibit their target in the low μM range.  Further studies are in progress to optimize delivery and potency of these compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqQJCPYE_gu4bVg90H21EOLACvtfcHk0lj6-AdmuUYdUg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtlWrsLjL&md5=2187a32637e930f278b3e950152c5e1d</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1007%2F978-0-387-73657-0_239&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252F978-0-387-73657-0_239%26sid%3Dliteratum%253Aachs%26aulast%3DMcMurray%26aufirst%3DJ.%2BS.%26aulast%3DMandal%26aufirst%3DP.%2BK.%26aulast%3DLiao%26aufirst%3DW.%2BS.%26aulast%3DRen%26aufirst%3DZ.%26aulast%3DChen%26aufirst%3DX.%26atitle%3DInhibition%2520of%2520Stat3%2520by%2520cell-permeable%2520peptidomimetic%2520prodrugs%2520targeted%2520to%2520its%2520SH2%2520domain%26jtitle%3DAdv.%2520Exp%2520Med.%2520Biol.%26date%3D2009%26volume%3D611%26spage%3D545%26epage%3D546" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mandal, P. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liao, W. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McMurray, J. S.</span></span> <span> </span><span class="NLM_article-title">Synthesis of phosphatase-stable, cell-permeable peptidomimetic prodrugs that target the SH2 domain of Stat3</span>. <i>Org. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">3394</span>– <span class="NLM_lpage">3397</span>, <span class="refDoi"> DOI: 10.1021/ol9012662</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ol9012662" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=1%3ACAS%3A528%3ADC%252BD1MXotl2ht78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2009&pages=3394-3397&author=P.+K.+Mandalauthor=W.+S.+Liaoauthor=J.+S.+McMurray&title=Synthesis+of+phosphatase-stable%2C+cell-permeable+peptidomimetic+prodrugs+that+target+the+SH2+domain+of+Stat3&doi=10.1021%2Fol9012662"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis of phosphatase-stable, cell-permeable peptidomimetic prodrugs that target the SH2 domain of Stat3</span></div><div class="casAuthors">Mandal, Pijus K.; Liao, Warren S.-L.; McMurray, John S.</div><div class="citationInfo"><span class="NLM_cas:title">Organic Letters</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">3394-3397</span>CODEN:
                <span class="NLM_cas:coden">ORLEF7</span>;
        ISSN:<span class="NLM_cas:issn">1523-7060</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The synthesis of prodrugs targeted to the SH2 domain of Stat3 is reported.  Using a convergent strategy, the pivaloyloxymethyl phosphonodiester of pentachlorophenyl 4-phosphonodifluoromethylcinnamate, a phosphotyrosine surrogate, was synthesized and used to acylate peptidomimetic fragments that were prepd. on solid supports.  Two prodrugs described here inhibited the phosphorylation of Stat3 in breast tumor cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqokZ-epNTxprVg90H21EOLACvtfcHk0lj6-AdmuUYdUg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXotl2ht78%253D&md5=c935049f107f5ab89bf38cd445c845f5</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1021%2Fol9012662&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fol9012662%26sid%3Dliteratum%253Aachs%26aulast%3DMandal%26aufirst%3DP.%2BK.%26aulast%3DLiao%26aufirst%3DW.%2BS.%26aulast%3DMcMurray%26aufirst%3DJ.%2BS.%26atitle%3DSynthesis%2520of%2520phosphatase-stable%252C%2520cell-permeable%2520peptidomimetic%2520prodrugs%2520that%2520target%2520the%2520SH2%2520domain%2520of%2520Stat3%26jtitle%3DOrg.%2520Lett.%26date%3D2009%26volume%3D11%26spage%3D3394%26epage%3D3397%26doi%3D10.1021%2Fol9012662" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bai, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernard, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nikolovska-Coleska, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gomez, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span> <span> </span><span class="NLM_article-title">Structure-based design of conformationally constrained, cell-permeable Stat3 inhibitors</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">85</span>– <span class="NLM_lpage">89</span>, <span class="refDoi"> DOI: 10.1021/ml100010j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ml100010j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=1%3ACAS%3A528%3ADC%252BC3cXivFGiurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2010&pages=85-89&author=J.+Chenauthor=L.+Baiauthor=D.+Bernardauthor=Z.+Nikolovska-Coleskaauthor=C.+Gomezauthor=J.+Zhangauthor=H.+Yiauthor=S.+Wang&title=Structure-based+design+of+conformationally+constrained%2C+cell-permeable+Stat3+inhibitors&doi=10.1021%2Fml100010j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Based Design of Conformationally Constrained, Cell-Permeable STAT3 Inhibitors</span></div><div class="casAuthors">Chen, Jianyong; Bai, Longchuan; Bernard, Denzil; Nikolovska-Coleska, Zaneta; Gomez, Cindy; Zhang, Jian; Yi, Han; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">85-89</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We report herein the structure-based design of a class of conformationally constrained, potent, cell-permeable small-mol. inhibitors to target the SH2 domain in STAT3.  Compd. 11 (CJ-1383) binds to STAT3 with a Ki value of 0.95 μM, dose-dependently inhibits cellular STAT3 signaling and cancer cell growth, and induces apoptosis in the MDA-MB-468 cancer cell line with constitutively activated STAT3.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqMrK-vDy7ISbVg90H21EOLACvtfcHk0lhRsbAmcYWAJQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXivFGiurs%253D&md5=87a0b1a823441940f75fc2af960e0359</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Fml100010j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fml100010j%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DBai%26aufirst%3DL.%26aulast%3DBernard%26aufirst%3DD.%26aulast%3DNikolovska-Coleska%26aufirst%3DZ.%26aulast%3DGomez%26aufirst%3DC.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DYi%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DStructure-based%2520design%2520of%2520conformationally%2520constrained%252C%2520cell-permeable%2520Stat3%2520inhibitors%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2010%26volume%3D1%26spage%3D85%26epage%3D89%26doi%3D10.1021%2Fml100010j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nikolovska-Coleska, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gomez, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krajewski, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roller, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span> <span> </span><span class="NLM_article-title">Design and synthesis of a new, conformationally constrained, macrocyclic small-molecule inhibitor of Stat3 via ‘click chemistry’</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">3939</span>– <span class="NLM_lpage">3942</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2007.04.096</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=10.1016%2Fj.bmcl.2007.04.096" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=17513110" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=1%3ACAS%3A528%3ADC%252BD2sXmvVClurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2007&pages=3939-3942&author=J.+Chenauthor=Z.+Nikolovska-Coleskaauthor=C.-Y.+Yangauthor=C.+Gomezauthor=W.+Gaoauthor=K.+Krajewskiauthor=S.+Jiangauthor=P.+Rollerauthor=S.+Wang&title=Design+and+synthesis+of+a+new%2C+conformationally+constrained%2C+macrocyclic+small-molecule+inhibitor+of+Stat3+via+%E2%80%98click+chemistry%E2%80%99&doi=10.1016%2Fj.bmcl.2007.04.096"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Design and synthesis of a new, conformationally constrained, macrocyclic small-molecule inhibitor of STAT3 via 'click chemistry'</span></div><div class="casAuthors">Chen, Jianyong; Nikolovska-Coleska, Zaneta; Yang, Chao-Yie; Gomez, Cindy; Gao, Wei; Krajewski, Krzysztof; Jiang, Sheng; Roller, Peter; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">3939-3942</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">STAT3 is a promising mol. target for the design of anticancer drugs.  In this paper, we report the design and synthesis of a conformationally constrained macrocyclic peptidomimetic I via click chem.  Compd. I was detd. to bind to STAT3 with a Ki value of 7.3 μM in a competitive fluorescence-polarization-based binding assay, representing a promising initial lead compd. for further optimization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoOR6JsWbNxsbVg90H21EOLACvtfcHk0lhRsbAmcYWAJQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXmvVClurc%253D&md5=93e403fbff688e18539712c00f39a94e</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2007.04.096&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2007.04.096%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DNikolovska-Coleska%26aufirst%3DZ.%26aulast%3DYang%26aufirst%3DC.-Y.%26aulast%3DGomez%26aufirst%3DC.%26aulast%3DGao%26aufirst%3DW.%26aulast%3DKrajewski%26aufirst%3DK.%26aulast%3DJiang%26aufirst%3DS.%26aulast%3DRoller%26aufirst%3DP.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DDesign%2520and%2520synthesis%2520of%2520a%2520new%252C%2520conformationally%2520constrained%252C%2520macrocyclic%2520small-molecule%2520inhibitor%2520of%2520Stat3%2520via%2520%25E2%2580%2598click%2520chemistry%25E2%2580%2599%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2007%26volume%3D17%26spage%3D3939%26epage%3D3942%26doi%3D10.1016%2Fj.bmcl.2007.04.096" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Leung, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhong, H. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Susanti, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rao, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Che, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leung, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, P. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, D. L.</span></span> <span> </span><span class="NLM_article-title">Discovery of a small-molecule inhibitor of Stat3 by ligand-based pharmacophore screening</span>. <i>Methods</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>71</i></span>,  <span class="NLM_fpage">38</span>– <span class="NLM_lpage">43</span>, <span class="refDoi"> DOI: 10.1016/j.ymeth.2014.07.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=10.1016%2Fj.ymeth.2014.07.010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=25160651" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsVKnur7P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=71&publication_year=2015&pages=38-43&author=K.+H.+Leungauthor=L.+J.+Liuauthor=S.+Linauthor=L.+Luauthor=H.+J.+Zhongauthor=D.+Susantiauthor=W.+Raoauthor=M.+Wangauthor=W.+Cheauthor=D.+S.+Chanauthor=C.+H.+Leungauthor=P.+W.+Chanauthor=D.+L.+Ma&title=Discovery+of+a+small-molecule+inhibitor+of+Stat3+by+ligand-based+pharmacophore+screening&doi=10.1016%2Fj.ymeth.2014.07.010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a small-molecule inhibitor of STAT3 by ligand-based pharmacophore screening</span></div><div class="casAuthors">Leung, Ka-Ho; Liu, Li-Juan; Lin, Sheng; Lu, Lihua; Zhong, Hai-Jing; Susanti, Dewi; Rao, Weidong; Wang, Modi; Che, Weng Ian; Chan, Daniel Shiu-Hin; Leung, Chung-Hang; Chan, Philip Wai Hong; Ma, Dik-Lung</div><div class="citationInfo"><span class="NLM_cas:title">Methods (Amsterdam, Netherlands)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">71</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">38-43</span>CODEN:
                <span class="NLM_cas:coden">MTHDE9</span>;
        ISSN:<span class="NLM_cas:issn">1046-2023</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">STAT3 modulates the transcription of a wide variety of regulatory genes involved in cell proliferation, differentiation, migration, apoptosis, and other crit. cellular functions.  Constitutive activation of STAT3 has been detected in a wide spectrum of human malignancies.  A pharmacophore model constructed from a training set of STAT3 inhibitors binding to the SH2 domain was used to screen an inhouse database of compds., from which azepine 1 emerged as a top candidate.  Compd. 1 inhibited STAT3 DNA-binding activity in vitro and attenuated STAT3-directed transcription in cellulo with comparable potency to the well-known STAT3 inhibitor S3I-201.  A fluorescence polarization assay revealed that compd. 1 targeted the SH2 domain of STAT3.  Furthermore, compd. 1 inhibited STAT3 phosphorylation in cells without affecting the total expression of STAT3.  This study also validates the use of pharmacophore modeling to identify inhibitors of protein-protein interactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGphtVAbxIrmB7Vg90H21EOLACvtfcHk0lhRsbAmcYWAJQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsVKnur7P&md5=1a1d6a3ee10efa8f2965b7eaf43b3a4f</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1016%2Fj.ymeth.2014.07.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ymeth.2014.07.010%26sid%3Dliteratum%253Aachs%26aulast%3DLeung%26aufirst%3DK.%2BH.%26aulast%3DLiu%26aufirst%3DL.%2BJ.%26aulast%3DLin%26aufirst%3DS.%26aulast%3DLu%26aufirst%3DL.%26aulast%3DZhong%26aufirst%3DH.%2BJ.%26aulast%3DSusanti%26aufirst%3DD.%26aulast%3DRao%26aufirst%3DW.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DChe%26aufirst%3DW.%26aulast%3DChan%26aufirst%3DD.%2BS.%26aulast%3DLeung%26aufirst%3DC.%2BH.%26aulast%3DChan%26aufirst%3DP.%2BW.%26aulast%3DMa%26aufirst%3DD.%2BL.%26atitle%3DDiscovery%2520of%2520a%2520small-molecule%2520inhibitor%2520of%2520Stat3%2520by%2520ligand-based%2520pharmacophore%2520screening%26jtitle%3DMethods%26date%3D2015%26volume%3D71%26spage%3D38%26epage%3D43%26doi%3D10.1016%2Fj.ymeth.2014.07.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yue, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fletcher, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gunning, P. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turkson, J.</span></span> <span> </span><span class="NLM_article-title">A novel small-molecule disrupts Stat3 SH2 domain-phosphotyrosine interactions and Stat3-dependent tumor processes</span>. <i>Biochem. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>79</i></span>,  <span class="NLM_fpage">1398</span>– <span class="NLM_lpage">409</span>, <span class="refDoi"> DOI: 10.1016/j.bcp.2010.01.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=10.1016%2Fj.bcp.2010.01.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=20067773" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=1%3ACAS%3A528%3ADC%252BC3cXjtVCnsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=79&publication_year=2010&pages=1398-409&author=X.+Zhangauthor=P.+Yueauthor=S.+Fletcherauthor=W.+Zhaoauthor=P.+T.+Gunningauthor=J.+Turkson&title=A+novel+small-molecule+disrupts+Stat3+SH2+domain-phosphotyrosine+interactions+and+Stat3-dependent+tumor+processes&doi=10.1016%2Fj.bcp.2010.01.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">A novel small-molecule disrupts Stat3 SH2 domain-phosphotyrosine interactions and Stat3-dependent tumor processes</span></div><div class="casAuthors">Zhang, Xiaolei; Yue, Peibin; Fletcher, Steven; Zhao, Wei; Gunning, Patrick T.; Turkson, James</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical Pharmacology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">79</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1398-1409</span>CODEN:
                <span class="NLM_cas:coden">BCPCA6</span>;
        ISSN:<span class="NLM_cas:issn">0006-2952</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The mol. modeling of the phosphotyrosine (pTyr)-SH2 domain interaction in the Stat3:Stat3 dimerization, combined with in silico structural anal. of the Stat3 dimerization disruptor, S3I-201, has furnished a diverse set of analogs.  We present evidence from in vitro biochem. and biophys. studies that the structural analog, S3I-201.1066 (I) directly interacts with Stat3 or the SH2 domain, with an affinity (KD) of 2.74 μM, and disrupts the binding of Stat3 to the cognate pTyr-peptide, GpYLPQTV-NH2, with an IC50 of 23 μM.  Moreover, I selectively blocks the assocn. of Stat3 with the epidermal growth factor receptor (EGFR), and inhibits Stat3 tyrosine phosphorylation and nuclear translocation in EGF-stimulated mouse fibroblasts.  In cancer cells that harbor aberrant Stat3 activity, I inhibits constitutive Stat3 DNA-binding and transcriptional activities.  By contrast, I has no effect on Src activation or the EGFR-mediated activation of the Erk1/2MAPK pathway.  I selectively suppresses the viability, survival, and malignant transformation of the human breast and pancreatic cancer lines and the v-Src-transformed mouse fibroblasts harboring persistently active Stat3.  Treatment with I of malignant cells harboring aberrantly active Stat3 down-regulated the expression of c-Myc, Bcl-xL, Survivin, the matrix metalloproteinase 9, and VEGF.  The in vivo administration of I-induced significant antitumor response in mouse models of human breast cancer, which correlates with the inhibition of constitutively active Stat3 and the suppression of known Stat3-regulated genes.  Our studies identify a novel small-mol. that binds with a high affinity to Stat3, blocks Stat3 activation and function, and thereby induces antitumor response in human breast tumor xenografts harboring persistently active Stat3.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoxJzIoqCaQHbVg90H21EOLACvtfcHk0ljT1yX0iPDl7Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXjtVCnsL8%253D&md5=4a196103d1d0f6d057715ff4cd2d7498</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1016%2Fj.bcp.2010.01.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bcp.2010.01.001%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DYue%26aufirst%3DP.%26aulast%3DFletcher%26aufirst%3DS.%26aulast%3DZhao%26aufirst%3DW.%26aulast%3DGunning%26aufirst%3DP.%2BT.%26aulast%3DTurkson%26aufirst%3DJ.%26atitle%3DA%2520novel%2520small-molecule%2520disrupts%2520Stat3%2520SH2%2520domain-phosphotyrosine%2520interactions%2520and%2520Stat3-dependent%2520tumor%2520processes%26jtitle%3DBiochem.%2520Pharmacol.%26date%3D2010%26volume%3D79%26spage%3D1398%26epage%3D409%26doi%3D10.1016%2Fj.bcp.2010.01.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yue, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Page, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Namanja, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paladino, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gunning, P. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turkson, J.</span></span> <span> </span><span class="NLM_article-title">Orally bioavailable small-molecule inhibitor of transcription factor Stat3 regresses human breast and lung cancer xenografts</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>109</i></span>,  <span class="NLM_fpage">9623</span>– <span class="NLM_lpage">9628</span>, <span class="refDoi"> DOI: 10.1073/pnas.1121606109</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=10.1073%2Fpnas.1121606109" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=22623533" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=1%3ACAS%3A528%3ADC%252BC38XptlWjtLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=109&publication_year=2012&pages=9623-9628&author=X.+Zhangauthor=P.+Yueauthor=B.+D.+Pageauthor=T.+Liauthor=W.+Zhaoauthor=A.+T.+Namanjaauthor=D.+Paladinoauthor=J.+Zhaoauthor=Y.+Chenauthor=P.+T.+Gunningauthor=J.+Turkson&title=Orally+bioavailable+small-molecule+inhibitor+of+transcription+factor+Stat3+regresses+human+breast+and+lung+cancer+xenografts&doi=10.1073%2Fpnas.1121606109"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Orally bioavailable small-molecule inhibitor of transcription factor Stat3 regresses human breast and lung cancer xenografts</span></div><div class="casAuthors">Zhang, Xiaolei; Yue, Peibin; Page, Brent D. G.; Li, Tianshu; Zhao, Wei; Namanja, Andrew T.; Paladino, David; Zhao, Jihe; Chen, Yuan; Gunning, Patrick T.; Turkson, James</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">109</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">9623-9628, S9623/1-S9623/13</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Computer-aided lead optimization derives a unique, orally bioavailable inhibitor of the signal transducer and activator of transcription (Stat)3 Src homol. 2 domain.  BP-1-102 binds Stat3 with an affinity (KD) of 504 nM, blocks Stat3-phospho-tyrosine (pTyr) peptide interactions and Stat3 activation at 4-6.8 μM, and selectively inhibits growth, survival, migration, and invasion of Stat3-dependent tumor cells.  BP-1-102-mediated inhibition of aberrantly active Stat3 in tumor cells suppresses the expression of c-Myc, Cyclin D1, Bcl-xL, Survivin, VEGF, and Kruppel-like factor 8, which is identified as a Stat3 target gene that promotes Stat3-mediated breast tumor cell migration and invasion.  Treatment of breast cancer cells with BP-1-102 further blocks Stat3-NF-κB cross-talk, the release of granulocyte colony-stimulating factor, sol. intercellular adhesion mol. 1, macrophage migration-inhibitory factor/glycosylation-inhibiting factor, interleukin 1 receptor antagonist, and serine protease inhibitor protein 1, and the phosphorylation of focal adhesion kinase and paxillin, while enhancing E-cadherin expression.  I.v. or oral gavage delivery of BP-1-102 furnishes micromolar or microgram levels in tumor tissues and inhibits growth of human breast and lung tumor xenografts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrljEafjbznr7Vg90H21EOLACvtfcHk0ljT1yX0iPDl7Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XptlWjtLo%253D&md5=4da78174a7c00eacc1f1a1ce553c5aa7</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1121606109&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1121606109%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DYue%26aufirst%3DP.%26aulast%3DPage%26aufirst%3DB.%2BD.%26aulast%3DLi%26aufirst%3DT.%26aulast%3DZhao%26aufirst%3DW.%26aulast%3DNamanja%26aufirst%3DA.%2BT.%26aulast%3DPaladino%26aufirst%3DD.%26aulast%3DZhao%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DGunning%26aufirst%3DP.%2BT.%26aulast%3DTurkson%26aufirst%3DJ.%26atitle%3DOrally%2520bioavailable%2520small-molecule%2520inhibitor%2520of%2520transcription%2520factor%2520Stat3%2520regresses%2520human%2520breast%2520and%2520lung%2520cancer%2520xenografts%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2012%26volume%3D109%26spage%3D9623%26epage%3D9628%26doi%3D10.1073%2Fpnas.1121606109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pireddu, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Urlam, M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lawrence, H. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guida, W. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lawrence, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sebti, S. M.</span></span> <span> </span><span class="NLM_article-title">A novel inhibitor of Stat3 homodimerization selectively suppresses Stat3 activity and malignant transformation</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>73</i></span>,  <span class="NLM_fpage">1922</span>– <span class="NLM_lpage">1933</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.CAN-12-3175</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=10.1158%2F0008-5472.CAN-12-3175" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=23322008" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=1%3ACAS%3A528%3ADC%252BC3sXktVGisbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=73&publication_year=2013&pages=1922-1933&author=X.+Zhangauthor=Y.+Sunauthor=R.+Piredduauthor=H.+Yangauthor=M.+K.+Urlamauthor=H.+R.+Lawrenceauthor=W.+C.+Guidaauthor=N.+J.+Lawrenceauthor=S.+M.+Sebti&title=A+novel+inhibitor+of+Stat3+homodimerization+selectively+suppresses+Stat3+activity+and+malignant+transformation&doi=10.1158%2F0008-5472.CAN-12-3175"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">A Novel Inhibitor of STAT3 Homodimerization Selectively Suppresses STAT3 Activity and Malignant Transformation</span></div><div class="casAuthors">Zhang, Xiaolei; Sun, Ying; Pireddu, Roberta; Yang, Hua; Urlam, Murali K.; Lawrence, Harshani R.; Guida, Wayne C.; Lawrence, Nicholas J.; Sebti, Said M.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">73</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1922-1933</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">STAT3-STAT3 dimerization, which involves reciprocal binding of the STAT3-SH2 domain to phosphorylated. Tyrosine-705 (Y-705), is required for STAT3 nuclear translocation, DNA binding, and transcriptional regulation of downstream target genes.  Here, we describe a small mol. S3I-1757 capable of disrupting STAT3-STAT3 dimerization, activation, and malignant transforming activity.  Fluorescence polarization assay and mol. modeling suggest that S3I-1757 interacts with the phospho-Y-705-binding site in the SH2 domain and displaces fluorescein-labeled GpYLPQTV phosphotyrosine peptide from binding to STAT3.  We generated hemagglutinin (HA)-tagged STAT3 and FLAG-tagged STAT3 and showed using coimmunopptn. and colocalization studies that S3I-1757 inhibits STAT3 dimerization and STAT3-EGF receptor (EGFR) binding in intact cells.  Treatment of human cancer cells with S31-1757 (but not a closely related analog, S3I-1756, which does not inhibit STAT3 dimerization), inhibits selectively the phosphorylation of STAT3 over AKTl and ERK1/2 (MAPK3/1), nuclear accumulation of P-Y705-STAT3, STAT3-DNA binding, and transcriptional activation and suppresses the expression levels of STAT3 target genes, such as Bcl-xL (BCL2L1), survivin (BIRC5), cyclin D1 (CCND1), and matrix metalloproteinase (MMP)-9.  Furthermore, S3I-1757, but not S3I-1756, inhibits anchorage-dependent and -independent growth, migration, and invasion of human cancer cells, which depend on STAT3.  Finally, STAT3-C, a genetically engineered mutant of STAT3 that forms a constitutively dimerized STAT3, rescues cells from the effects of S3I-1757 inhibition.  Thus, we have developed S3I-1757 as a STAT3-STAT3 dimerization inhibitor capable of blocking hyperactivated STAT3 and suppressing malignant transformation in human cancer cells that depend on STAT3.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpLVvdmwj25LbVg90H21EOLACvtfcHk0ljT1yX0iPDl7Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXktVGisbg%253D&md5=efc26953ace34e1bd3e32810113b7ef5</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-12-3175&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-12-3175%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DSun%26aufirst%3DY.%26aulast%3DPireddu%26aufirst%3DR.%26aulast%3DYang%26aufirst%3DH.%26aulast%3DUrlam%26aufirst%3DM.%2BK.%26aulast%3DLawrence%26aufirst%3DH.%2BR.%26aulast%3DGuida%26aufirst%3DW.%2BC.%26aulast%3DLawrence%26aufirst%3DN.%2BJ.%26aulast%3DSebti%26aufirst%3DS.%2BM.%26atitle%3DA%2520novel%2520inhibitor%2520of%2520Stat3%2520homodimerization%2520selectively%2520suppresses%2520Stat3%2520activity%2520and%2520malignant%2520transformation%26jtitle%3DCancer%2520Res.%26date%3D2013%26volume%3D73%26spage%3D1922%26epage%3D1933%26doi%3D10.1158%2F0008-5472.CAN-12-3175" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yue, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lopez-Tapia, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paladino, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, C.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hilliard, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tius, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turkson, J.</span></span> <span> </span><span class="NLM_article-title">Hydroxamic acid and benzoic acid-based Stat3 inhibitors suppress human glioma and breast cancer phenotypes in vitro and in vivo</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>76</i></span>,  <span class="NLM_fpage">652</span>– <span class="NLM_lpage">663</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.CAN-14-3558</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=10.1158%2F0008-5472.CAN-14-3558" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=26088127" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=1%3ACAS%3A528%3ADC%252BC28XhslSgtrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=76&publication_year=2016&pages=652-663&author=P.+Yueauthor=F.+Lopez-Tapiaauthor=D.+Paladinoauthor=Y.+Liauthor=C.-H.+Chenauthor=T.+Hilliardauthor=Y.+Chenauthor=M.+Tiusauthor=J.+Turkson&title=Hydroxamic+acid+and+benzoic+acid-based+Stat3+inhibitors+suppress+human+glioma+and+breast+cancer+phenotypes+in+vitro+and+in+vivo&doi=10.1158%2F0008-5472.CAN-14-3558"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Hydroxamic Acid and Benzoic Acid-Based STAT3 Inhibitors Suppress Human Glioma and Breast Cancer Phenotypes In Vitro and In Vivo</span></div><div class="casAuthors">Yue, Peibin; Lopez-Tapia, Francisco; Paladino, David; Li, Yifei; Chen, Chih-Hong; Namanja, Andrew T.; Hilliard, Tyvette; Chen, Yuan; Tius, Marcus A.; Turkson, James</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">76</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">652-663</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">STAT3 offers an attractive target for cancer therapy, but small-mol. inhibitors with appealing pharmacol. properties have been elusive.  Here, we report hydroxamic acid-based and benzoic acid-based inhibitors (SH5-07 and SH4-54, resp.) with robust bioactivity.  Both inhibitors blocked STAT3 DNA-binding activity in vitro and in human glioma, breast, and prostate cancer cells and in v-Src-transformed murine fibroblasts.  STAT3-dependent gene transcription was blocked along with Bcl-2, Bcl-xL, Mcl-1, cyclin D1, c-Myc, and survivin expression.  NMR anal. of STAT3-inhibitor complexes defined interactions with the SH2 and DNA-binding domains of STAT3.  Ectopic expression of the SH2 domain in cells was sufficient to counter the STAT3-inhibitory effects of SH4-54.  Neither compd. appreciably affected STAT1 or STAT5 DNA-binding activities, STAT3-independent gene transcription, or activation of a panel of oncogenic kinases in malignant cells.  Each compd. decreased the proliferation and viability of glioma, breast, and prostate cancer cells and v-Src-transformed murine fibroblasts harboring constitutively active STAT3.  Further, in mouse xenograft models of glioma and breast cancer, administration of SH5-07 or SH4-54 effectively inhibited tumor growth.  Our results offer preclin. proof of concept for SH5-07 and SH4-54 as candidates for further development as cancer therapeutics.  Cancer Res; 76(3); 652-63. ©2015 AACR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGohkHZXQZ-69bVg90H21EOLACvtfcHk0lhc3pS4lTEyvA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhslSgtrY%253D&md5=62397d5abc0b6e551289d44e808b6cd1</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-14-3558&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-14-3558%26sid%3Dliteratum%253Aachs%26aulast%3DYue%26aufirst%3DP.%26aulast%3DLopez-Tapia%26aufirst%3DF.%26aulast%3DPaladino%26aufirst%3DD.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DChen%26aufirst%3DC.-H.%26aulast%3DHilliard%26aufirst%3DT.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DTius%26aufirst%3DM.%26aulast%3DTurkson%26aufirst%3DJ.%26atitle%3DHydroxamic%2520acid%2520and%2520benzoic%2520acid-based%2520Stat3%2520inhibitors%2520suppress%2520human%2520glioma%2520and%2520breast%2520cancer%2520phenotypes%2520in%2520vitro%2520and%2520in%2520vivo%26jtitle%3DCancer%2520Res.%26date%3D2016%26volume%3D76%26spage%3D652%26epage%3D663%26doi%3D10.1158%2F0008-5472.CAN-14-3558" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Miklossy, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hilliard, T. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turkson, J.</span></span> <span> </span><span class="NLM_article-title">Therapeutic modulators of Stat signaling for human diseases</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">611</span>– <span class="NLM_lpage">629</span>, <span class="refDoi"> DOI: 10.1038/nrd4088</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=10.1038%2Fnrd4088" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=23903221" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=1%3ACAS%3A528%3ADC%252BC3sXht1SjtbvI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2013&pages=611-629&author=G.+Miklossyauthor=T.+S.+Hilliardauthor=J.+Turkson&title=Therapeutic+modulators+of+Stat+signaling+for+human+diseases&doi=10.1038%2Fnrd4088"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Therapeutic modulators of STAT signalling for human diseases</span></div><div class="casAuthors">Miklossy, Gabriella; Hilliard, Tyvette S.; Turkson, James</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">4_testjo</span>),
    <span class="NLM_cas:pages">611-629</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The signal transducer and activator of transcription (STAT) proteins have important roles in biol. processes.  The abnormal activation of STAT signalling pathways is also implicated in many human diseases, including cancer, autoimmune diseases, rheumatoid arthritis, asthma and diabetes.  Over a decade has passed since the first inhibitor of a STAT protein was reported and efforts to discover modulators of STAT signalling as therapeutics continue.  This Review discusses the outcomes of the ongoing drug discovery research endeavors against STAT proteins, provides perspectives on new directions for accelerating the discovery of drug candidates, and highlights the noteworthy candidate therapeutics that have progressed to clin. trials.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoSgI1shc9cRrVg90H21EOLACvtfcHk0lhc3pS4lTEyvA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXht1SjtbvI&md5=fcf21bda56a222e81ef7c44166f0b20c</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1038%2Fnrd4088&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd4088%26sid%3Dliteratum%253Aachs%26aulast%3DMiklossy%26aufirst%3DG.%26aulast%3DHilliard%26aufirst%3DT.%2BS.%26aulast%3DTurkson%26aufirst%3DJ.%26atitle%3DTherapeutic%2520modulators%2520of%2520Stat%2520signaling%2520for%2520human%2520diseases%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2013%26volume%3D12%26spage%3D611%26epage%3D629%26doi%3D10.1038%2Fnrd4088" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bai, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McEachern, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chinnaswamy, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wen, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dai, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumar, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meagher, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stuckey, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span> <span> </span><span class="NLM_article-title">Structure-based discovery of SD-36 as a potent, selective, and efficacious protac degrader of Stat3 protein</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">11280</span>– <span class="NLM_lpage">11300</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b01530</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b01530" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=1%3ACAS%3A528%3ADC%252BC1MXit1SnsL3J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=11280-11300&author=H.+Zhouauthor=L.+Baiauthor=R.+Xuauthor=Y.+Zhaoauthor=J.+Chenauthor=D.+McEachernauthor=K.+Chinnaswamyauthor=B.+Wenauthor=L.+Daiauthor=P.+Kumarauthor=C.+Y.+Yangauthor=Z.+Liuauthor=M.+Wangauthor=L.+Liuauthor=J.+L.+Meagherauthor=H.+Yiauthor=D.+Sunauthor=J.+A.+Stuckeyauthor=S.+Wang&title=Structure-based+discovery+of+SD-36+as+a+potent%2C+selective%2C+and+efficacious+protac+degrader+of+Stat3+protein&doi=10.1021%2Facs.jmedchem.9b01530"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Based Discovery of SD-36 as a Potent, Selective, and Efficacious PROTAC Degrader of STAT3 Protein</span></div><div class="casAuthors">Zhou, Haibin; Bai, Longchuan; Xu, Renqi; Zhao, Yujun; Chen, Jianyong; McEachern, Donna; Chinnaswamy, Krishnapriya; Wen, Bo; Dai, Lipeng; Kumar, Praveen; Yang, Chao-Yie; Liu, Zhaomin; Wang, Mi; Liu, Liu; Meagher, Jennifer L.; Yi, Han; Sun, Duxin; Stuckey, Jeanne A.; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">11280-11300</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Signal transducer and activator of transcription 3 (STAT3) is a transcription factor and an attractive therapeutic target for cancer and other human diseases.  Despite twenty-years of persistent research efforts, targeting STAT3 has been very challenging.  The authors report herein the structure-based discovery of potent small-mol. STAT3 degraders based upon the proteolysis targeting chimera (PROTAC) concept.  The authors first designed SI-109 as a potent, small-mol. inhibitor of the STAT3 SH2 domain.  Employing ligands for cereblon/Cullin 4A E3 ligase and SI-109, the authors obtained a series of potent PROTAC STAT3 degraders, exemplified by SD-36.  SD-36 induces rapid STAT3 degrdn. at low nanomolar concns. in cells and fails to degrader other STAT proteins.  SD-36 achieves nanomolar cell growth inhibitory activity in leukemia and lymphoma cell lines with high levels of phosphorylated STAT3.  A single dose of SD-36 results in complete STAT3 protein degrdn. in xenograft tumor tissue and normal mouse tissues.  SD-36 achieves complete and long-lasting tumor regression in the Molm-16 xenograft tumor model at well-tolerated dose-schedules.  SD-36 is a potent, selective and efficacious STAT3 degrader.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoIAWd1hiuqy7Vg90H21EOLACvtfcHk0lhfdHMTJw2wYA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXit1SnsL3J&md5=7d0c656a2d96196a3b71829c9efe715a</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b01530%26sid%3Dliteratum%253Aachs%26aulast%3DZhou%26aufirst%3DH.%26aulast%3DBai%26aufirst%3DL.%26aulast%3DXu%26aufirst%3DR.%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DMcEachern%26aufirst%3DD.%26aulast%3DChinnaswamy%26aufirst%3DK.%26aulast%3DWen%26aufirst%3DB.%26aulast%3DDai%26aufirst%3DL.%26aulast%3DKumar%26aufirst%3DP.%26aulast%3DYang%26aufirst%3DC.%2BY.%26aulast%3DLiu%26aufirst%3DZ.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DMeagher%26aufirst%3DJ.%2BL.%26aulast%3DYi%26aufirst%3DH.%26aulast%3DSun%26aufirst%3DD.%26aulast%3DStuckey%26aufirst%3DJ.%2BA.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DStructure-based%2520discovery%2520of%2520SD-36%2520as%2520a%2520potent%252C%2520selective%252C%2520and%2520efficacious%2520protac%2520degrader%2520of%2520Stat3%2520protein%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D11280%26epage%3D11300%26doi%3D10.1021%2Facs.jmedchem.9b01530" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bai, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chinnaswamy, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McEachern, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wen, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumar, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meagher, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stuckey, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span> <span> </span><span class="NLM_article-title">A potent and selective small-molecule degrader of Stat3 achieves complete tumor regression in vivo</span>. <i>Cancer Cell</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>36</i></span>,  <span class="NLM_fpage">498</span>– <span class="NLM_lpage">511.e17</span>, <span class="refDoi"> DOI: 10.1016/j.ccell.2019.10.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=10.1016%2Fj.ccell.2019.10.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=31715132" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitFCjtrjO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2019&pages=498-511.e17&author=L.+Baiauthor=H.+Zhouauthor=R.+Xuauthor=Y.+Zhaoauthor=K.+Chinnaswamyauthor=D.+McEachernauthor=J.+Chenauthor=C.+Y.+Yangauthor=Z.+Liuauthor=M.+Wangauthor=L.+Liuauthor=H.+Jiangauthor=B.+Wenauthor=P.+Kumarauthor=J.+L.+Meagherauthor=D.+Sunauthor=J.+A.+Stuckeyauthor=S.+Wang&title=A+potent+and+selective+small-molecule+degrader+of+Stat3+achieves+complete+tumor+regression+in+vivo&doi=10.1016%2Fj.ccell.2019.10.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">A Potent and Selective Small-Molecule Degrader of STAT3 Achieves Complete Tumor Regression In Vivo</span></div><div class="casAuthors">Bai, Longchuan; Zhou, Haibin; Xu, Renqi; Zhao, Yujun; Chinnaswamy, Krishnapriya; McEachern, Donna; Chen, Jianyong; Yang, Chao-Yie; Liu, Zhaomin; Wang, Mi; Liu, Liu; Jiang, Hui; Wen, Bo; Kumar, Praveen; Meagher, Jennifer L.; Sun, Duxin; Stuckey, Jeanne A.; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">498-511.e17</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Signal transducer and activator of transcription 3 (STAT3) is an attractive cancer therapeutic target.  Here we report the discovery of SD-36(I), a small-mol. degrader of STAT3.  SD-36 potently induces the degrdn. of STAT3 protein in vitro and in vivo and demonstrates high selectivity over other STAT members.  Induced degrdn. of STAT3 results in a strong suppression of its transcription network in leukemia and lymphoma cells.  SD-36 inhibits the growth of a subset of acute myeloid leukemia and anaplastic large-cell lymphoma cell lines by inducing cell-cycle arrest and/or apoptosis.  SD-36 achieves complete and long-lasting tumor regression in multiple xenograft mouse models at well-tolerated dose schedules.  Degrdn. of STAT3 protein, therefore, is a promising cancer therapeutic strategy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoUaP49Ap9DxLVg90H21EOLACvtfcHk0lhfdHMTJw2wYA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitFCjtrjO&md5=71252c17d50a1a8be829352942175bd6</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1016%2Fj.ccell.2019.10.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccell.2019.10.002%26sid%3Dliteratum%253Aachs%26aulast%3DBai%26aufirst%3DL.%26aulast%3DZhou%26aufirst%3DH.%26aulast%3DXu%26aufirst%3DR.%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DChinnaswamy%26aufirst%3DK.%26aulast%3DMcEachern%26aufirst%3DD.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DYang%26aufirst%3DC.%2BY.%26aulast%3DLiu%26aufirst%3DZ.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DWen%26aufirst%3DB.%26aulast%3DKumar%26aufirst%3DP.%26aulast%3DMeagher%26aufirst%3DJ.%2BL.%26aulast%3DSun%26aufirst%3DD.%26aulast%3DStuckey%26aufirst%3DJ.%2BA.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DA%2520potent%2520and%2520selective%2520small-molecule%2520degrader%2520of%2520Stat3%2520achieves%2520complete%2520tumor%2520regression%2520in%2520vivo%26jtitle%3DCancer%2520Cell%26date%3D2019%26volume%3D36%26spage%3D498%26epage%3D511.e17%26doi%3D10.1016%2Fj.ccell.2019.10.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lopez-Tapia, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brotherton-Pleiss, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yue, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murakami, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Costa Araujo, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reis dos Santos, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ichinotsubo, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rabkin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shah, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lantz, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tius, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turkson, J.</span></span> <span> </span><span class="NLM_article-title">Linker variation and structure-activity relationship analyses of car-boxylic acid-based small molecule Stat3 inhibitors</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">250</span>– <span class="NLM_lpage">255</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.7b00544</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.7b00544" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=1%3ACAS%3A528%3ADC%252BC1cXivFCnsLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2018&pages=250-255&author=F.+Lopez-Tapiaauthor=C.+Brotherton-Pleissauthor=P.+Yueauthor=H.+Murakamiauthor=A.+C.+Costa+Araujoauthor=B.+Reis+dos+Santosauthor=E.+Ichinotsuboauthor=A.+Rabkinauthor=R.+Shahauthor=M.+Lantzauthor=S.+Chenauthor=M.+A.+Tiusauthor=J.+Turkson&title=Linker+variation+and+structure-activity+relationship+analyses+of+car-boxylic+acid-based+small+molecule+Stat3+inhibitors&doi=10.1021%2Facsmedchemlett.7b00544"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Linker variation and structure-activity relationship analysis of carboxylic acid-based small molecule STAT3 inhibitors</span></div><div class="casAuthors">Lopez-Tapia, Francisco; Brotherton-Pleiss, Christine; Yue, Peibin; Murakami, Heide; Costa Araujo, Ana Carolina; Reis dos Santos, Bruna; Ichinotsubo, Erin; Rabkin, Anna; Shah, Raj; Lantz, Megan; Chen, Suzie; Tius, Marcus A.; Turkson, James</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">250-255</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The mol. determinants for the activities of the reported benzoic acid (SH4-54), salicylic acid (BP-1-102), and benzohydroxamic acid (SH5-07)-based STAT3 inhibitors were investigated to design optimized analogs.  All three leads are based on an N-methylglycinamide scaffold, with its two amine groups condensed with three different functionalities.  The three functionalities and the CH2 group of the glycinamide scaffold were sep. modified.  The replacement of the pentafluorobenzene or cyclohexylbenzene, or replacing the benzene ring of the arom. carboxylic or hydroxamic acid motif with heterocyclic components (contg. nitrogen and oxygen elements) all decreased potency.  Notably, the Ala-linker analogs, (I) (R1 = H and OH), and the Pro-based deriv. (II) (X = CH2), all with (R)-configuration at the chiral center, had improved inhibitory activity and selectivity against STAT3 DNA-binding activity in vitro, with IC50 of 3.0 ± 0.9, 1.80 ± 0.94, and 2.4 ± 0.2 μM, resp.  Compds. I and II and other analogs inhibited constitutive STAT3 phosphorylation and activation in human breast cancer and melanoma lines, and blocked tumor cell viability, growth, colony formation, and migration in vitro.  Pro-based analog, II (X = O)(sodium salt), with a relatively polar tetrahydropyranyl (THP) ring, instead of the cyclohexyl, showed improved permeability.  In general, the (R)-configuration Pro-based analogs showed the overall best profile, including physicochem. properties (e.g., microsomal metabolic stability, Caco-2 permeability), and in particular, II (X = CH2) showed improved tumor-cell specificity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr6yrAyhBvV47Vg90H21EOLACvtfcHk0lhfdHMTJw2wYA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXivFCnsLs%253D&md5=c80cc2d96017675b30e13a0f0395103a</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.7b00544&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.7b00544%26sid%3Dliteratum%253Aachs%26aulast%3DLopez-Tapia%26aufirst%3DF.%26aulast%3DBrotherton-Pleiss%26aufirst%3DC.%26aulast%3DYue%26aufirst%3DP.%26aulast%3DMurakami%26aufirst%3DH.%26aulast%3DCosta%2BAraujo%26aufirst%3DA.%2BC.%26aulast%3DReis%2Bdos%2BSantos%26aufirst%3DB.%26aulast%3DIchinotsubo%26aufirst%3DE.%26aulast%3DRabkin%26aufirst%3DA.%26aulast%3DShah%26aufirst%3DR.%26aulast%3DLantz%26aufirst%3DM.%26aulast%3DChen%26aufirst%3DS.%26aulast%3DTius%26aufirst%3DM.%2BA.%26aulast%3DTurkson%26aufirst%3DJ.%26atitle%3DLinker%2520variation%2520and%2520structure-activity%2520relationship%2520analyses%2520of%2520car-boxylic%2520acid-based%2520small%2520molecule%2520Stat3%2520inhibitors%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2018%26volume%3D9%26spage%3D250%26epage%3D255%26doi%3D10.1021%2Facsmedchemlett.7b00544" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Garcia, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hudnall, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Catlett, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nelson, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smithgall, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fujita, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ethier, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jove, R.</span></span> <span> </span><span class="NLM_article-title">Constitutive activation of Stat3 in fibroblasts transformed by diverse oncoproteins and in breast carcinoma cells</span>. <i>Cell Growth Diff.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">1267</span>– <span class="NLM_lpage">1276</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=9419415" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=1%3ACAS%3A528%3ADyaK2sXotVWlt7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=1997&pages=1267-1276&author=R.+Garciaauthor=C.+L.+Yuauthor=A.+Hudnallauthor=R.+Catlettauthor=K.+L.+Nelsonauthor=T.+Smithgallauthor=D.+J.+Fujitaauthor=S.+P.+Ethierauthor=R.+Jove&title=Constitutive+activation+of+Stat3+in+fibroblasts+transformed+by+diverse+oncoproteins+and+in+breast+carcinoma+cells"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Constitutive activation of Stat3 in fibroblasts transformed by diverse oncoproteins and in breast carcinoma cells</span></div><div class="casAuthors">Garcia, Roy; Yu, Chao-Lan; Hudnall, Anne; Catlett, Robyn; Nelson, Kristie L.; Smithgall, Thomas; Fujita, Donald J.; Ethier, Stephen P.; Jove, Richard</div><div class="citationInfo"><span class="NLM_cas:title">Cell Growth & Differentiation</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1267-1276</span>CODEN:
                <span class="NLM_cas:coden">CGDIE7</span>;
        ISSN:<span class="NLM_cas:issn">1044-9523</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Signal transducers and activators of transcription (STATs) were originally identified as key components of signaling pathways involved in mediating responses to IFNs.  Previous studies showed that the Src oncoprotein constitutively activates one STAT family member, Stat3.  In this study, we investigated STAT activation in a panel of rodent fibroblast cell lines stably transformed by diverse viral oncoproteins.  Using a temp.-sensitive mutant of v-Src, we detd. that Stat3 is activated within 15 min of shift from nonpermissive to permissive temp. for cell transformation.  This finding indicates that v-Src tyrosine kinase activity is required for Stat3 activation and suggests that Stat3 is proximal to signaling initiated by Src.  In addn., Stat3 activation is induced by another nonreceptor tyrosine kinase, v-Fps; by polyoma virus middle T antigen, which activates Src family kinases; and by v-Sis, which acts as a ligand for the platelet-derived growth factor receptor.  In contrast, SV40 large T antigen, which transforms cells through different mechanisms, and the v-Ras and v-Raf oncoproteins, which lie in signaling pathways downstream of tyrosine kinases, do not activate Stat3.  We did not detect significant activation of Stat1, Stat5, or Stat6 in fibroblasts transformed by the viral oncoproteins investigated.  Moreover, Stat3 is activated in response to epidermal growth factor (EGF) but not heregulins in immortalized normal human breast epithelial cells.  Because constitutive activation of c-Src and EGF receptor kinases is assocd. with the progression of breast cancer, we examd. activation of STATs in human cell lines derived from breast carcinomas.  We detected constitutive activation of Stat3 in five of nine breast carcinoma cell lines but not in normal breast epithelial cells.  Furthermore, expts. with an EGF receptor-specific inhibitor indicated that the constitutive activation of Stat3 in these breast carcinoma cell lines is not necessarily dependent on signaling through the EGF receptor, although EGF stimulation further increases Stat3 activation.  Taken together, our results demonstrate that selective activation of Stat3 is a common event during oncogenic transformation that directly or indirectly involves activation of specific tyrosine kinase signaling pathways.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoqkU_WBMF3KLVg90H21EOLACvtfcHk0li8KFVhIPl_qw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXotVWlt7s%253D&md5=3e6712ee4cfa9196b9bfac89b51b081a</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DGarcia%26aufirst%3DR.%26aulast%3DYu%26aufirst%3DC.%2BL.%26aulast%3DHudnall%26aufirst%3DA.%26aulast%3DCatlett%26aufirst%3DR.%26aulast%3DNelson%26aufirst%3DK.%2BL.%26aulast%3DSmithgall%26aufirst%3DT.%26aulast%3DFujita%26aufirst%3DD.%2BJ.%26aulast%3DEthier%26aufirst%3DS.%2BP.%26aulast%3DJove%26aufirst%3DR.%26atitle%3DConstitutive%2520activation%2520of%2520Stat3%2520in%2520fibroblasts%2520transformed%2520by%2520diverse%2520oncoproteins%2520and%2520in%2520breast%2520carcinoma%2520cells%26jtitle%3DCell%2520Growth%2520Diff.%26date%3D1997%26volume%3D8%26spage%3D1267%26epage%3D1276" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Garcia, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bowman, T. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Niu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minton, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muro-Cacho, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cox, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Falcone, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fairclough, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parson, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laudano, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gazit, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Levitzki, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kraker, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jove, R.</span></span> <span> </span><span class="NLM_article-title">Constitutive activation of Stat3 by the Src and Jak tyrosine kinases participates in growth regulation of human breast carcinoma cells</span>. <i>Oncogene</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">2499</span>– <span class="NLM_lpage">2513</span>, <span class="refDoi"> DOI: 10.1038/sj.onc.1204349</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=10.1038%2Fsj.onc.1204349" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=11420660" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=1%3ACAS%3A528%3ADC%252BD3MXjvFGmt7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2001&pages=2499-2513&author=R.+Garciaauthor=T.+L.+Bowmanauthor=G.+Niuauthor=H.+Yuauthor=S.+Mintonauthor=C.+A.+Muro-Cachoauthor=C.+E.+Coxauthor=R.+Falconeauthor=R.+Faircloughauthor=S.+Parsonauthor=A.+Laudanoauthor=A.+Gazitauthor=A.+Levitzkiauthor=A.+Krakerauthor=R.+Jove&title=Constitutive+activation+of+Stat3+by+the+Src+and+Jak+tyrosine+kinases+participates+in+growth+regulation+of+human+breast+carcinoma+cells&doi=10.1038%2Fsj.onc.1204349"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Constitutive activation of Stat3 by the Src and JAK tyrosine kinases participates in growth regulation of human breast carcinoma cells</span></div><div class="casAuthors">Garcia, Roy; Bowman, Tammy L.; Niu, Guilian; Yu, Hua; Minton, Sue; Muro-Cacho, Carlos A.; Cox, Charles E.; Falcone, Robert; Fairclough, Rita; Parsons, Sarah; Laudano, Andy; Gazit, Aviv; Levitzki, Alexander; Kraker, Alan; Jove, Richard</div><div class="citationInfo"><span class="NLM_cas:title">Oncogene</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">2499-2513</span>CODEN:
                <span class="NLM_cas:coden">ONCNES</span>;
        ISSN:<span class="NLM_cas:issn">0950-9232</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Constitutive activation of signal transducer and activator of transcription (STAT) proteins has been detected in a wide variety of human primary tumor specimens and tumor cell lines including blood malignancies, head and neck cancer, and breast cancer.  We have previously demonstrated a high frequency of Stat3 DNA-binding activity that is constitutively-induced by an unknown mechanism in human breast cancer cell lines possessing elevated EGF receptor (EGF-R) and c-Src kinase activities.  Using tyrosine kinase selective inhibitors, we show here that Src and JAK family tyrosine kinases cooperate to mediate constitutive Stat3 activation in the absence of EGF stimulation in model human breast cancer cell lines.  Inhibition of Src or JAKs results in dose-dependent suppression of Stat3 DNA-binding activity, which is accompanied by growth inhibition and induction of programmed cell death.  In addn., transfection of a dominant-neg. form of Stat3 leads to growth inhibition involving apoptosis of breast cancer cells.  These results indicate that the biol. effects of the Src and JAK tyrosine kinase inhibitors are at least partially mediated by blocking Stat3 signaling.  While EGF-R kinase activity is not required for constitutive Stat3 activation in breast cancer cells, EGF stimulation further increases STAT DNA-binding activity, consistent with an important role for EGF-R in STAT signaling and malignant progression.  Anal. of primary breast tumor specimens from patients with advanced disease revealed that the majority exhibit elevated STAT DNA-binding activity compared to adjacent non-tumor tissues.  Our findings, taken together, suggest that tyrosine kinases transduce signals through Stat3 protein that contribute to the growth and survival of human breast cancer cells in culture and potentially in vivo.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq5BSpinbxg3rVg90H21EOLACvtfcHk0li8KFVhIPl_qw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXjvFGmt7k%253D&md5=a941c45ef8fbf78234c6f66b741e7be2</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1038%2Fsj.onc.1204349&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.onc.1204349%26sid%3Dliteratum%253Aachs%26aulast%3DGarcia%26aufirst%3DR.%26aulast%3DBowman%26aufirst%3DT.%2BL.%26aulast%3DNiu%26aufirst%3DG.%26aulast%3DYu%26aufirst%3DH.%26aulast%3DMinton%26aufirst%3DS.%26aulast%3DMuro-Cacho%26aufirst%3DC.%2BA.%26aulast%3DCox%26aufirst%3DC.%2BE.%26aulast%3DFalcone%26aufirst%3DR.%26aulast%3DFairclough%26aufirst%3DR.%26aulast%3DParson%26aufirst%3DS.%26aulast%3DLaudano%26aufirst%3DA.%26aulast%3DGazit%26aufirst%3DA.%26aulast%3DLevitzki%26aufirst%3DA.%26aulast%3DKraker%26aufirst%3DA.%26aulast%3DJove%26aufirst%3DR.%26atitle%3DConstitutive%2520activation%2520of%2520Stat3%2520by%2520the%2520Src%2520and%2520Jak%2520tyrosine%2520kinases%2520participates%2520in%2520growth%2520regulation%2520of%2520human%2520breast%2520carcinoma%2520cells%26jtitle%3DOncogene%26date%3D2001%26volume%3D20%26spage%3D2499%26epage%3D2513%26doi%3D10.1038%2Fsj.onc.1204349" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Turkson, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bowman, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garcia, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caldenhoven, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Groot, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jove, R.</span></span> <span> </span><span class="NLM_article-title">Stat3 activation by Src induces specific gene regulation and is required for cell transformation</span>. <i>Mol. Cell. Biol.</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">2545</span>– <span class="NLM_lpage">2552</span>, <span class="refDoi"> DOI: 10.1128/MCB.18.5.2545</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=10.1128%2FMCB.18.5.2545" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=9566874" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=1%3ACAS%3A528%3ADyaK1cXis1GntLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=1998&pages=2545-2552&author=J.+Turksonauthor=T.+Bowmanauthor=R.+Garciaauthor=E.+Caldenhovenauthor=R.+P.+De+Grootauthor=R.+Jove&title=Stat3+activation+by+Src+induces+specific+gene+regulation+and+is+required+for+cell+transformation&doi=10.1128%2FMCB.18.5.2545"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Stat3 activation by Src induces specific gene regulation and is required for cell transformation</span></div><div class="casAuthors">Turkson, James; Bowman, Tammy; Garcia, Roy; Caldenhoven, Eric; De Groot, Rolf P.; Jove, Richard</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">2545-2552</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">0270-7306</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">While signal transducers and activators of transcription (STATs) were originally discovered as intracellular effectors of normal signaling by cytokines, increasing evidence also points to a role for STAT transcription factors in oncogenesis.  Previous studies have demonstrated that one STAT family member, Stat3, possesses constitutively elevated tyrosine phosphorylation and DNA-binding activity in fibroblasts stably transformed by the Src oncoprotein.  To det. if this Stat3 activation by Src could induce Stat3-mediated gene expression, luciferase reporter constructs based on synthetic and authentic promoters were transfected into NIH 3T3 cells.  Activation of endogenous cellular Stat3 by the Src oncoprotein induced gene expression through a Stat3-specific binding element (TTCCCGAA) of the C-reactive protein gene promoter.  A naturally occurring splice variant of human Stat3 protein, Stat3β, with a deletion in the C-terminal transactivation domain abolished this gene induction in a dominant neg. manner.  Expression of Stat3β did not have any effect on a reporter construct based on the c-fos serum response element, which is not dependent on Stat3 signaling, indicating that Stat3β does not nonspecifically inhibit other signaling pathways or Src function.  Transfection of vectors expressing Stat3β together with Src blocked cell transformation by Src as measured in a quant. focus formation assay using NIH 3T3 cells.  By contrast, Stat3β had a much less pronounced effect on focus formation induced by the Ras oncoprotein, which does not activate Stat3 signaling.  In addn., three independent clones of NIH 3T3 cells stably overexpressing Stat3β were generated and characterized, demonstrating that Stat3β overexpression does not have a toxic effect on cell viability.  These Stat3β-overexpressing clones were shown to be deficient in Stat3-mediated signaling and refractory to Src-induced cell transformation.  The authors conclude that Stat3 activation by the Src oncoprotein leads to specific gene regulation and that Stat3 is one of the crit. signaling pathways involved in Src oncogenesis.  The authors' findings provide evidence that oncogenesis-assocd. activation of Stat3 signaling is part of the process of malignant transformation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpX3GjARHEUY7Vg90H21EOLACvtfcHk0li8KFVhIPl_qw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXis1GntLc%253D&md5=935fc4454fb988b7e570f15672d2c173</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1128%2FMCB.18.5.2545&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.18.5.2545%26sid%3Dliteratum%253Aachs%26aulast%3DTurkson%26aufirst%3DJ.%26aulast%3DBowman%26aufirst%3DT.%26aulast%3DGarcia%26aufirst%3DR.%26aulast%3DCaldenhoven%26aufirst%3DE.%26aulast%3DDe%2BGroot%26aufirst%3DR.%2BP.%26aulast%3DJove%26aufirst%3DR.%26atitle%3DStat3%2520activation%2520by%2520Src%2520induces%2520specific%2520gene%2520regulation%2520and%2520is%2520required%2520for%2520cell%2520transformation%26jtitle%3DMol.%2520Cell.%2520Biol.%26date%3D1998%26volume%3D18%26spage%3D2545%26epage%3D2552%26doi%3D10.1128%2FMCB.18.5.2545" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, B.</span></span> <span> </span><span class="NLM_article-title">Napabucasin (BBI608) eliminate AML cells in vitro and in vivo via inhibition of Stat3 pathway and induction of DNA damage</span>. <i>Eur. J. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>855</i></span>,  <span class="NLM_fpage">252</span>– <span class="NLM_lpage">261</span>, <span class="refDoi"> DOI: 10.1016/j.ejphar.2019.05.020</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=10.1016%2Fj.ejphar.2019.05.020" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=31085238" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=1%3ACAS%3A528%3ADC%252BC1MXps1Glsr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=855&publication_year=2019&pages=252-261&author=S.+Biauthor=K.+Chenauthor=L.+Fengauthor=G.+Fuauthor=Q.+Yangauthor=M.+Dengauthor=H.+Zhaoauthor=Z.+Liauthor=L.+Yuauthor=Z.+Fangauthor=B.+Xu&title=Napabucasin+%28BBI608%29+eliminate+AML+cells+in+vitro+and+in+vivo+via+inhibition+of+Stat3+pathway+and+induction+of+DNA+damage&doi=10.1016%2Fj.ejphar.2019.05.020"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Napabucasin (BBI608) eliminate AML cells in vitro and in vivo via inhibition of Stat3 pathway and induction of DNA damage</span></div><div class="casAuthors">Bi, Silei; Chen, Kai; Feng, Liying; Fu, Guofeng; Yang, Qianying; Deng, Manman; Zhao, Haijun; Li, Zhifeng; Yu, Lian; Fang, Zhihong; Xu, Bing</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Pharmacology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">855</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">252-261</span>CODEN:
                <span class="NLM_cas:coden">EJPHAZ</span>;
        ISSN:<span class="NLM_cas:issn">0014-2999</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Acute myeloid leukemia (AML) is a heterogeneous malignancy of hematopoietic stem cells with poor clin. outcome despite recent improvements in chemotherapy and stem cell transplantation regimens.  Thus, new therapeutic agents are urgently needed in order to prolong the disease-free survival of AML patients in clinic.  Here, we report that BBI608 is highly active against diverse AML cell lines in vitro and primary samples obtained from patients with AML ex vivo, as well as effective in vivo in AML xenograft models.  Meanwhile, the anti-AML property of BBI608 is closely assocd. with the inhibition of Stat3 pathway and induction of DNA damage.  Of note, BBI608 combined with Bcl-2 inhibitor (i.e., ABT-199) exerts a significantly enhanced anti-leukemia effect in BBI608-resistant cell line Kasumi-1.  Together, the present findings suggest that BBI608 might represent a potential candidate agent for AML treatment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr3ukbC0r0gDLVg90H21EOLACvtfcHk0liqhHxEnMPvLQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXps1Glsr8%253D&md5=1b3a2e0089b0b42eaa1716215bb13335</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1016%2Fj.ejphar.2019.05.020&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejphar.2019.05.020%26sid%3Dliteratum%253Aachs%26aulast%3DBi%26aufirst%3DS.%26aulast%3DChen%26aufirst%3DK.%26aulast%3DFeng%26aufirst%3DL.%26aulast%3DFu%26aufirst%3DG.%26aulast%3DYang%26aufirst%3DQ.%26aulast%3DDeng%26aufirst%3DM.%26aulast%3DZhao%26aufirst%3DH.%26aulast%3DLi%26aufirst%3DZ.%26aulast%3DYu%26aufirst%3DL.%26aulast%3DFang%26aufirst%3DZ.%26aulast%3DXu%26aufirst%3DB.%26atitle%3DNapabucasin%2520%2528BBI608%2529%2520eliminate%2520AML%2520cells%2520in%2520vitro%2520and%2520in%2520vivo%2520via%2520inhibition%2520of%2520Stat3%2520pathway%2520and%2520induction%2520of%2520DNA%2520damage%26jtitle%3DEur.%2520J.%2520Pharmacol.%26date%3D2019%26volume%3D855%26spage%3D252%26epage%3D261%26doi%3D10.1016%2Fj.ejphar.2019.05.020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lewis, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bharadwaj, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eckols, T. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kolosov, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kasembeli, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fridley, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siller, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tweardy, D. J.</span></span> <span> </span><span class="NLM_article-title">Small-molecule targeting of signal transducer and activator of transcription (Stat) 3 to treat non-small cell lung cancer</span>. <i>Lung Cancer</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>90</i></span>,  <span class="NLM_fpage">182</span>– <span class="NLM_lpage">190</span>, <span class="refDoi"> DOI: 10.1016/j.lungcan.2015.09.014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=10.1016%2Fj.lungcan.2015.09.014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=26410177" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=1%3ACAS%3A280%3ADC%252BC283mtVansA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=90&publication_year=2015&pages=182-190&author=K.+M.+Lewisauthor=U.+Bharadwajauthor=T.+K.+Eckolsauthor=M.+Kolosovauthor=M.+M.+Kasembeliauthor=C.+Fridleyauthor=R.+Sillerauthor=D.+J.+Tweardy&title=Small-molecule+targeting+of+signal+transducer+and+activator+of+transcription+%28Stat%29+3+to+treat+non-small+cell+lung+cancer&doi=10.1016%2Fj.lungcan.2015.09.014"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Small-molecule targeting of signal transducer and activator of transcription (STAT) 3 to treat non-small cell lung cancer</span></div><div class="casAuthors">Lewis Katherine M; Bharadwaj Uddalak; Eckols T Kris; Kolosov Mikhail; Kasembeli Moses M; Siller Ricardo; Fridley Colleen; Tweardy David J</div><div class="citationInfo"><span class="NLM_cas:title">Lung cancer (Amsterdam, Netherlands)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">90</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">182-90</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">OBJECTIVE:  Lung cancer is the leading cause of cancer death in both men and women.  Non-small cell lung cancer (NSCLC) has an overall 5-year survival rate of 15%.  While aberrant STAT3 activation has previously been observed in NSCLC, the scope of its contribution is uncertain and agents that target STAT3 for treatment are not available clinically.  METHODS:  We determined levels of activated STAT3 (STAT3 phosphorylated on Y705, pSTAT3) and the two major isoforms of STAT3 (α and β) in protein extracts of 8 NSCLC cell lines, as well as the effects of targeting STAT3 in vitro and in vivo in NSCLC cells using short hairpin (sh) RNA and two novel small-molecule STAT3 inhibitors, C188-9 and piperlongumine (PL).  RESULTS:  Levels of pSTAT3, STAT3α, and STATβ were increased in 7 of 8 NSCLC cell lines.  Of note, levels of pSTAT3 were tightly correlated with levels of STAT3β, but not STAT3α.  Targeting of STAT3 in A549 cells using shRNA decreased tSTAT3 by 75%; this was accompanied by a 47-78% reduction in anchorage-dependent and anchorage-independent growth and a 28-45% reduction in mRNA levels for anti-apoptotic STAT3 gene targets.  C188-9 and PL (@30 μM) each reduced pSTAT3 levels in all NSCLC cell lines tested by ≥50%, reduced anti-apoptotic protein mRNA levels by 25-60%, and reduced both anchorage-dependent and anchorage-independent growth of NSCLC cell lines with IC50 values ranging from 3.06 to 52.44 μM and 0.86 to 11.66 μM, respectively.  Treatment of nude mice bearing A549 tumor xenografts with C188-9 or PL blocked tumor growth and reduced levels of pSTAT3 and mRNA encoding anti-apoptotic proteins.  CONCLUSION:  STAT3 is essential for growth of NSCLC cell lines and tumors and its targeting using C188-9 or PL may be a useful strategy for treatment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSD9LiZcIR8prrIbt-tGZmxfW6udTcc2ea4bOjdujwnarntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC283mtVansA%253D%253D&md5=135571e0e64ed367ef7bb8778ff1da40</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1016%2Fj.lungcan.2015.09.014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.lungcan.2015.09.014%26sid%3Dliteratum%253Aachs%26aulast%3DLewis%26aufirst%3DK.%2BM.%26aulast%3DBharadwaj%26aufirst%3DU.%26aulast%3DEckols%26aufirst%3DT.%2BK.%26aulast%3DKolosov%26aufirst%3DM.%26aulast%3DKasembeli%26aufirst%3DM.%2BM.%26aulast%3DFridley%26aufirst%3DC.%26aulast%3DSiller%26aufirst%3DR.%26aulast%3DTweardy%26aufirst%3DD.%2BJ.%26atitle%3DSmall-molecule%2520targeting%2520of%2520signal%2520transducer%2520and%2520activator%2520of%2520transcription%2520%2528Stat%2529%25203%2520to%2520treat%2520non-small%2520cell%2520lung%2520cancer%26jtitle%3DLung%2520Cancer%26date%3D2015%26volume%3D90%26spage%3D182%26epage%3D190%26doi%3D10.1016%2Fj.lungcan.2015.09.014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yue, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turkson, J.</span></span> <span> </span><span class="NLM_article-title">Targeting STAT3 in cancer: how successful are we?</span>. <i>Expert Opin. Invest. Drugs</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">45</span>– <span class="NLM_lpage">56</span>, <span class="refDoi"> DOI: 10.1517/13543780802565791</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=10.1517%2F13543780802565791" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=19053881" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhsVGhtbvO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2009&pages=45-56&author=P.+Yueauthor=J.+Turkson&title=Targeting+STAT3+in+cancer%3A+how+successful+are+we%3F&doi=10.1517%2F13543780802565791"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting STAT3 in cancer: how successful are we?</span></div><div class="casAuthors">Yue, Peibin; Turkson, James</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Investigational Drugs</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">45-56</span>CODEN:
                <span class="NLM_cas:coden">EOIDER</span>;
        ISSN:<span class="NLM_cas:issn">1354-3784</span>.
    
            (<span class="NLM_cas:orgname">Informa Healthcare</span>)
        </div><div class="casAbstract">A review.  Aberrant activation of the signal transducer and activator of transcription (STAT)3 occurs in many human tumors.  Moreover, studies utilizing genetic and pharmacol. approaches to modulate constitutive STAT3 activity have provided compelling evidence for the crit. role of aberrant STAT3 activity in malignant transformation and tumor progression, and thereby validated STAT3 as a novel cancer drug target.  This review is intended to be a full coverage of the efforts to develop direct STAT3 inhibitors and will provide a discussion on the inhibitory modalities developed to date.  Review of the literature focused on the modalities and mechanisms that directly target and inhibit the STAT protein or its functions.  While a variety of STAT3 inhibitors have been identified that induce antitumor cell effects in vitro and in vivo, the landscape remains murky.  With a few exceptions, most of the STAT3 inhibitors reported to date have not undergone an in vivo efficacy, pharmacol. or toxicity testing.  Also, there is no evidence, per the published literature of an impending clin. development for the few agents that were reported to exhibit in vivo efficacy.  Overall, there is the need for a reassessment of the ongoing strategies to target STAT3 intended not only for refinement, but also for incorporating some new technologies to strengthen our efforts and ensure the success, sooner, rather than later, of identifying suitable anti-STAT3 agents for development into clin. useful anticancer therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoGo96b-3KjRLVg90H21EOLACvtfcHk0liqhHxEnMPvLQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhsVGhtbvO&md5=e0f4fe796305b7e28ff3020f57b5dc8e</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1517%2F13543780802565791&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1517%252F13543780802565791%26sid%3Dliteratum%253Aachs%26aulast%3DYue%26aufirst%3DP.%26aulast%3DTurkson%26aufirst%3DJ.%26atitle%3DTargeting%2520STAT3%2520in%2520cancer%253A%2520how%2520successful%2520are%2520we%253F%26jtitle%3DExpert%2520Opin.%2520Invest.%2520Drugs%26date%3D2009%26volume%3D18%26spage%3D45%26epage%3D56%26doi%3D10.1517%2F13543780802565791" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kelder, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grootenhuis, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bayada, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delbressine, L. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ploemen, J. P.</span></span> <span> </span><span class="NLM_article-title">Polar molecular surface as a dominating determinant for oral absorption and brain penetration of drugs</span>. <i>Pharm. Res.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">1514</span>– <span class="NLM_lpage">1519</span>, <span class="refDoi"> DOI: 10.1023/A:1015040217741</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=10.1023%2FA%3A1015040217741" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=10554091" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=1%3ACAS%3A528%3ADyaK1MXntVOhtb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=1999&pages=1514-1519&author=J.+Kelderauthor=P.+D.+Grootenhuisauthor=D.+M.+Bayadaauthor=L.+P.+Delbressineauthor=J.+P.+Ploemen&title=Polar+molecular+surface+as+a+dominating+determinant+for+oral+absorption+and+brain+penetration+of+drugs&doi=10.1023%2FA%3A1015040217741"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Polar molecular surface as a dominating determinant for oral absorption and brain penetration of drugs</span></div><div class="casAuthors">Kelder, Jan; Grootenhuis, Peter D. J.; Bayada, Denis M.; Delbressine, Leon P. C.; Ploemen, Jan-Peter</div><div class="citationInfo"><span class="NLM_cas:title">Pharmaceutical Research</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1514-1519</span>CODEN:
                <span class="NLM_cas:coden">PHREEB</span>;
        ISSN:<span class="NLM_cas:issn">0724-8741</span>.
    
            (<span class="NLM_cas:orgname">Kluwer Academic/Plenum Publishers</span>)
        </div><div class="casAbstract">Purpose: To study oral absorption and brain penetration as a function of polar mol. surface area.  Methods: Measured brain penetration data of 45 drug mols. were investigated.  The dynamic polar surface areas were calcd. and correlated with the brain penetration data.  Also the static polar surface areas of 776 orally administered CNS drugs that have reached at least Phase II efficacy studies were calcd.  The same was done for a series of 1590 orally administered non-CNS drugs that have reached at least Phase II efficacy studies.  Results: A linear relationship between brain penetration and dynamic polar surface area (A2) was found (n = 45, R = 0.917, F1,43 = 229).  Brain penetration decreases with increasing polar surface area.  A clear difference between the distribution of the polar surface area of the 776 CNS and 1590 non-CNS drugs was found.  It was deduced that orally active drugs that are transported passively by the transcellular route should not exceed a polar surface area of about 120 A2.  They can be tailored to brain penetration by decreasing the polar surface to <60-70 A2.  This conclusion is supported by the inverse linear relationship between exptl. brain penetration data and the dynamic polar surface area of 45 drug mols.  Conclusions: The polar mol. surface area is a dominating determinant for oral absorption and brain penetration of drugs that are transported by the transcellular route.  This property should be considered in the early phase of drug screening.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpmc3v7TzWM4LVg90H21EOLACvtfcHk0lgJWeCmVtQN9g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXntVOhtb8%253D&md5=868fe1e9491dd3b8b92b326d28ee48f9</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1023%2FA%3A1015040217741&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1023%252FA%253A1015040217741%26sid%3Dliteratum%253Aachs%26aulast%3DKelder%26aufirst%3DJ.%26aulast%3DGrootenhuis%26aufirst%3DP.%2BD.%26aulast%3DBayada%26aufirst%3DD.%2BM.%26aulast%3DDelbressine%26aufirst%3DL.%2BP.%26aulast%3DPloemen%26aufirst%3DJ.%2BP.%26atitle%3DPolar%2520molecular%2520surface%2520as%2520a%2520dominating%2520determinant%2520for%2520oral%2520absorption%2520and%2520brain%2520penetration%2520of%2520drugs%26jtitle%3DPharm.%2520Res.%26date%3D1999%26volume%3D16%26spage%3D1514%26epage%3D1519%26doi%3D10.1023%2FA%3A1015040217741" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sharma, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">López-Tarruella, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">García-Saenz, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ward, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Connor, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gómez, H. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prat, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moreno, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jerez-Gilarranz, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barnadas, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Picornell, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Del Monte-Millán, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gonzalez-Rivera, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Massarrah, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pelaez-Lorenzo, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Palomero, M. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">González Del Val, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cortes, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fuentes Rivera, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bretel Morales, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Márquez-Rodas, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perou, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wagner, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mammen, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McGinness, M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klemp, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amin, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fabian, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heldstab, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Godwin, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jensen, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kimler, B. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khan, Q. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, M.</span></span> <span> </span><span class="NLM_article-title">Efficacy of neoadjuvant carboplatin plus docetaxel in triple-negative breast cancer: combined analysis of two cohorts</span>. <i>Clin. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">649</span>– <span class="NLM_lpage">657</span>, <span class="refDoi"> DOI: 10.1158/1078-0432.CCR-16-0162</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=10.1158%2F1078-0432.CCR-16-0162" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=27301700" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvVSjsLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2017&pages=649-657&author=P.+Sharmaauthor=S.+L%C3%B3pez-Tarruellaauthor=J.+A.+Garc%C3%ADa-Saenzauthor=C.+Wardauthor=C.+S.+Connorauthor=H.+L.+G%C3%B3mezauthor=A.+Pratauthor=F.+Morenoauthor=Y.+Jerez-Gilarranzauthor=A.+Barnadasauthor=A.+C.+Picornellauthor=M.+Del+Monte-Mill%C3%A1nauthor=M.+Gonzalez-Riveraauthor=T.+Massarrahauthor=B.+Pelaez-Lorenzoauthor=M.+I.+Palomeroauthor=R.+Gonz%C3%A1lez+Del+Valauthor=J.+Cortesauthor=H.+Fuentes+Riveraauthor=D.+Bretel+Moralesauthor=I.+M%C3%A1rquez-Rodasauthor=C.+M.+Perouauthor=J.+L.+Wagnerauthor=J.+M.+Mammenauthor=M.+K.+McGinnessauthor=J.+R.+Klempauthor=A.+L.+Aminauthor=C.+J.+Fabianauthor=J.+Heldstabauthor=A.+K.+Godwinauthor=R.+A.+Jensenauthor=B.+F.+Kimlerauthor=Q.+J.+Khanauthor=M.+Martin&title=Efficacy+of+neoadjuvant+carboplatin+plus+docetaxel+in+triple-negative+breast+cancer%3A+combined+analysis+of+two+cohorts&doi=10.1158%2F1078-0432.CCR-16-0162"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Efficacy of Neoadjuvant Carboplatin plus Docetaxel in Triple-Negative Breast Cancer: Combined Analysis of Two Cohorts</span></div><div class="casAuthors">Sharma, Priyanka; Lopez-Tarruella, Sara; Garcoia-Saenz, Jose Angel; Ward, Claire; Connor, Carol S.; Goiomez, Henry L.; Prat, Aleix; Moreno, Fernando; Jerez-Gilarranz, Yolanda; Barnadas, Augusti; Picornell, Antoni C.; del Monte-Milloioan, Maria; Gonzalez-Rivera, Milagros; Massarrah, Tatiana; Pelaez-Lorenzo, Beatriz; Palomero, Maria Isabel; Gonzalez del Val, Ricardo; Cortes, Javier; Fuentes Rivera, Hugo; Bretel Morales, Denisse; Marquez-Rodas, Ivan; Perou, Charles M.; Wagner, Jamie L.; Mammen, Joshua M. V.; McGinness, Marilee K.; Klemp, Jennifer R.; Amin, Amanda L.; Fabian, Carol J.; Heldstab, Jaimie; Godwin, Andrew K.; Jensen, Roy A.; Kimler, Bruce F.; Khan, Qamar J.; Martin, Miguel</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">649-657</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Purpose: Recent studies demonstrate that addn. of neoadjuvant (NA) carboplatin to anthracycline/taxane chemotherapy improves pathol. complete response (pCR) in triple-neg. breast cancer (TNBC).  Effectiveness of anthracycline-free platinum combinations in TNBC is not well known.  Here, we report efficacy of NA carboplatin + docetaxel (CbD) in TNBC.  Exptl. Design: The study population includes 190 patients with stage I-III TNBC treated uniformly on two independent prospective cohorts.  All patients were prescribed NA chemotherapy regimen of carboplatin (AUC 6) + docetaxel (75 mg/m2) given every 21 days × 6 cycles. pCR (no evidence of invasive tumor in the breast and axilla) and residual cancer burden (RCB) were evaluated.  Results: Among 190 patients, median tumor size was 35 mm, 52% were lymph node pos., and 16% had germline BRCA1/2 mutation.  The overall pCR and RCB 0 + 1 rates were 55% and 68%, resp. pCRs in patients with BRCA-assocd. and wild-type TNBC were 59% and 56%, resp. (P = 0.83).  On multivariable anal., stage III disease was the only factor assocd. with a lower likelihood of achieving a pCR.  Twenty-one percent and 7% of patients, resp., experienced at least one grade 3 or 4 adverse event.  Conclusions: The CbD regimen was well tolerated and yielded high pCR rates in both BRCA-assocd. and wild-type TNBC.  These results are comparable with pCR achieved with the addn. of carboplatin to anthracycline-taxane chemotherapy.  Our study adds to the existing data on the efficacy of platinum agents in TNBC and supports further exploration of the CbD regimen in randomized studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrO1X6U5BphXrVg90H21EOLACvtfcHk0lgJWeCmVtQN9g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvVSjsLg%253D&md5=c3b5266e1a68b482ccfa9e30049ce215</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1158%2F1078-0432.CCR-16-0162&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1078-0432.CCR-16-0162%26sid%3Dliteratum%253Aachs%26aulast%3DSharma%26aufirst%3DP.%26aulast%3DL%25C3%25B3pez-Tarruella%26aufirst%3DS.%26aulast%3DGarc%25C3%25ADa-Saenz%26aufirst%3DJ.%2BA.%26aulast%3DWard%26aufirst%3DC.%26aulast%3DConnor%26aufirst%3DC.%2BS.%26aulast%3DG%25C3%25B3mez%26aufirst%3DH.%2BL.%26aulast%3DPrat%26aufirst%3DA.%26aulast%3DMoreno%26aufirst%3DF.%26aulast%3DJerez-Gilarranz%26aufirst%3DY.%26aulast%3DBarnadas%26aufirst%3DA.%26aulast%3DPicornell%26aufirst%3DA.%2BC.%26aulast%3DDel%2BMonte-Mill%25C3%25A1n%26aufirst%3DM.%26aulast%3DGonzalez-Rivera%26aufirst%3DM.%26aulast%3DMassarrah%26aufirst%3DT.%26aulast%3DPelaez-Lorenzo%26aufirst%3DB.%26aulast%3DPalomero%26aufirst%3DM.%2BI.%26aulast%3DGonz%25C3%25A1lez%2BDel%2BVal%26aufirst%3DR.%26aulast%3DCortes%26aufirst%3DJ.%26aulast%3DFuentes%2BRivera%26aufirst%3DH.%26aulast%3DBretel%2BMorales%26aufirst%3DD.%26aulast%3DM%25C3%25A1rquez-Rodas%26aufirst%3DI.%26aulast%3DPerou%26aufirst%3DC.%2BM.%26aulast%3DWagner%26aufirst%3DJ.%2BL.%26aulast%3DMammen%26aufirst%3DJ.%2BM.%26aulast%3DMcGinness%26aufirst%3DM.%2BK.%26aulast%3DKlemp%26aufirst%3DJ.%2BR.%26aulast%3DAmin%26aufirst%3DA.%2BL.%26aulast%3DFabian%26aufirst%3DC.%2BJ.%26aulast%3DHeldstab%26aufirst%3DJ.%26aulast%3DGodwin%26aufirst%3DA.%2BK.%26aulast%3DJensen%26aufirst%3DR.%2BA.%26aulast%3DKimler%26aufirst%3DB.%2BF.%26aulast%3DKhan%26aufirst%3DQ.%2BJ.%26aulast%3DMartin%26aufirst%3DM.%26atitle%3DEfficacy%2520of%2520neoadjuvant%2520carboplatin%2520plus%2520docetaxel%2520in%2520triple-negative%2520breast%2520cancer%253A%2520combined%2520analysis%2520of%2520two%2520cohorts%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2017%26volume%3D23%26spage%3D649%26epage%3D657%26doi%3D10.1158%2F1078-0432.CCR-16-0162" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hu, X. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, B. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ragaz, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, B. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Teng, Y. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tong, Z. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pan, Y. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yin, Y. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, C. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, Z. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lou, G. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gu, Y. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shao, Z. M.</span></span> <span> </span><span class="NLM_article-title">Cisplatin plus gemcitabine versus paclitaxel plus gemcitabine as first-line therapy for metastatic triple-negative breast cancer (CBCSG006): a randomised, open-label, multicentre, phase 3 trial</span>. <i>Lancet Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">436</span>– <span class="NLM_lpage">46</span>, <span class="refDoi"> DOI: 10.1016/S1470-2045(15)70064-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=10.1016%2FS1470-2045%2815%2970064-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=25795409" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=1%3ACAS%3A528%3ADC%252BC2MXkvVWgsLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2015&pages=436-46&author=X.+C.+Huauthor=J.+Zhangauthor=B.+H.+Xuauthor=L.+Caiauthor=J.+Ragazauthor=Z.+H.+Wangauthor=B.+Y.+Wangauthor=Y.+E.+Tengauthor=Z.+S.+Tongauthor=Y.+Y.+Panauthor=Y.+M.+Yinauthor=C.+P.+Wuauthor=Z.+F.+Jiangauthor=X.+J.+Wangauthor=G.+Y.+Louauthor=D.+G.+Liuauthor=J.+F.+Fengauthor=J.+F.+Luoauthor=K.+Sunauthor=Y.+J.+Guauthor=J.+Wuauthor=Z.+M.+Shao&title=Cisplatin+plus+gemcitabine+versus+paclitaxel+plus+gemcitabine+as+first-line+therapy+for+metastatic+triple-negative+breast+cancer+%28CBCSG006%29%3A+a+randomised%2C+open-label%2C+multicentre%2C+phase+3+trial&doi=10.1016%2FS1470-2045%2815%2970064-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Cisplatin plus gemcitabine versus paclitaxel plus gemcitabine as first-line therapy for metastatic triple-negative breast cancer (CBCSG006): a randomized, open-label, multicentre, phase 3 trial</span></div><div class="casAuthors">Hu, Xi-Chun; Zhang, Jian; Xu, Bing-He; Cai, Li; Ragaz, Joseph; Wang, Zhong-Hua; Wang, Bi-Yun; Teng, Yue-E.; Tong, Zhong-Sheng; Pan, Yue-Yin; Yin, Yong-Mei; Wu, Chang-Ping; Jiang, Ze-Fei; Wang, Xiao-Jia; Lou, Gu-Yin; Liu, Dong-Geng; Feng, Ji-Feng; Luo, Jian-Feng; Sun, Kang; Gu, Ya-Jia; Wu, Jiong; Shao, Zhi-Min</div><div class="citationInfo"><span class="NLM_cas:title">Lancet Oncology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">436-446</span>CODEN:
                <span class="NLM_cas:coden">LOANBN</span>;
        ISSN:<span class="NLM_cas:issn">1470-2045</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Background: Platinum chemotherapy has a role in the treatment of metastatic triple-neg. breast cancer but its full potential has probably not yet been reached.  We assessed whether a cisplatin plus gemcitabine regimen was non-inferior to or superior to paclitaxel plus gemcitabine as first-line therapy for patients with metastatic triple-neg. breast cancer.  Methods: For this open-label, randomized, phase 3, hybrid-designed trial undertaken at 12 institutions or hospitals in China, we included Chinese patients aged 18-70 years with previously untreated, histol. confirmed metastatic triple-neg. breast cancer, and an ECOG performance status of 0-1.  These patients were randomly assigned (1:1) to receive either cisplatin plus gemcitabine (cisplatin 75 mg/m2 on day 1 and gemcitabine 1250 mg/m2 on days 1 and 8) or paclitaxel plus gemcitabine (paclitaxel 175 mg/m2 on day 1 and gemcitabine 1250 mg/m2 on days 1 and 8) given i.v. every 3 wk for a max. of eight cycles.  Randomisation was done centrally via an interactive web response system using block randomization with a size of eight, with no stratification factors.  Patients and investigator were aware of group assignments.  The primary endpoint was progression-free survival and analyses were based on all patients who received at least one dose of assigned treatment.  The margin used to establish non-inferiority was 1·2.  If non-inferiority of cisplatin plus gemcitabine compared with paclitaxel plus gemcitabine was achieved, we would then test for superiority.  The trial is registered with ClinicalTrials.gov, no. NCT01287624.  Findings: From Jan 14, 2011, to Nov 14, 2013, 240 patients were assessed for eligibility and randomly assigned to treatment (120 in the cisplatin plus gemcitabine group and 120 in the paclitaxel plus gemcitabine group). 236 patients received at least one dose of assigned chemotherapy and were included in the modified intention-to-treat anal. (118 per group).  After a median follow-up of 16·3 mo IQR 14·4-26·8 in the cisplatin plus gemcitabine group and 15·9 mo (10·7-25·4) in the paclitaxel plus gemcitabine group, the hazard ratio for progression-free survival was 0·692 95% CI 0·523-0·915; pnon-inferiority<0·0001, psuperiority=0·009, thus cisplatin plus gemcitabine was both non-inferior to and superior to paclitaxel plus gemcitabine. Median progression-free survival was 7·73 mo (95% CI 6·16-9·30) in the cisplatin plus gemcitabine group and 6·47 mo 5·76-7·18 in the paclitaxel plus gemcitabine group.  Grade 3 or 4 adverse events that differed significantly between the two groups included nausea (eight [7%] vs one [<1%]), vomiting (13 [11%] vs one [<1%]), musculoskeletal pain (none vs ten [8%]), anemia (39 [33%] vs six [5%]), and thrombocytopenia (38 [32%] vs three [3%]), for the cisplatin plus gemcitabine compared with the paclitaxel plus gemcitabine groups, resp.  In addn., patients in the cisplatin plus gemcitabine group had significantly fewer events of grade 1-4 alopecia (12 [10%] vs 42 [36%]) and peripheral neuropathy (27 [23%] vs 60 [51%]), but more grade 1-4 anorexia (33 [28%] vs 10 [8%]), constipation (29 [25%] vs 11 [9%]), hypomagnesemia (27 [23%] vs five [4%]), and hypokalemia (10 [8%] vs two [2%]).  Serious drug-related adverse events were seen in three patients in the paclitaxel plus gemcitabine group (interstitial pneumonia, anaphylaxis, and severe neutropenia) and four in the cisplatin plus gemcitabine group (pathol. bone fracture, thrombocytopenia with s.c. hemorrhage, severe anemia, and cardiogenic syncope).  There were no treatment-related deaths.  Interpretation: Cisplatin plus gemcitabine could be an alternative or even the preferred first-line chemotherapy strategy for patients with metastatic triple-neg. breast cancer.  Shanghai Natural Science Foundation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoak6TclTwMVbVg90H21EOLACvtfcHk0lg0LfFG0-Rppw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXkvVWgsLs%253D&md5=0de66bedf4f2ad138dbe89c99958433b</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1016%2FS1470-2045%2815%2970064-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1470-2045%252815%252970064-1%26sid%3Dliteratum%253Aachs%26aulast%3DHu%26aufirst%3DX.%2BC.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DXu%26aufirst%3DB.%2BH.%26aulast%3DCai%26aufirst%3DL.%26aulast%3DRagaz%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DZ.%2BH.%26aulast%3DWang%26aufirst%3DB.%2BY.%26aulast%3DTeng%26aufirst%3DY.%2BE.%26aulast%3DTong%26aufirst%3DZ.%2BS.%26aulast%3DPan%26aufirst%3DY.%2BY.%26aulast%3DYin%26aufirst%3DY.%2BM.%26aulast%3DWu%26aufirst%3DC.%2BP.%26aulast%3DJiang%26aufirst%3DZ.%2BF.%26aulast%3DWang%26aufirst%3DX.%2BJ.%26aulast%3DLou%26aufirst%3DG.%2BY.%26aulast%3DLiu%26aufirst%3DD.%2BG.%26aulast%3DFeng%26aufirst%3DJ.%2BF.%26aulast%3DLuo%26aufirst%3DJ.%2BF.%26aulast%3DSun%26aufirst%3DK.%26aulast%3DGu%26aufirst%3DY.%2BJ.%26aulast%3DWu%26aufirst%3DJ.%26aulast%3DShao%26aufirst%3DZ.%2BM.%26atitle%3DCisplatin%2520plus%2520gemcitabine%2520versus%2520paclitaxel%2520plus%2520gemcitabine%2520as%2520first-line%2520therapy%2520for%2520metastatic%2520triple-negative%2520breast%2520cancer%2520%2528CBCSG006%2529%253A%2520a%2520randomised%252C%2520open-label%252C%2520multicentre%252C%2520phase%25203%2520trial%26jtitle%3DLancet%2520Oncol.%26date%3D2015%26volume%3D16%26spage%3D436%26epage%3D46%26doi%3D10.1016%2FS1470-2045%2815%2970064-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lipinski, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lombardo, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dominy, B. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feeney, P. J.</span></span> <span> </span><span class="NLM_article-title">Experimental and computational approaches to estimate solubility and permeability in drug discovery and development settings</span>. <i>Adv. Drug Delivery Rev.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>46</i></span>,  <span class="NLM_fpage">3</span>– <span class="NLM_lpage">26</span>, <span class="refDoi"> DOI: 10.1016/S0169-409X(00)00129-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=10.1016%2FS0169-409X%2800%2900129-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=11259830" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=1%3ACAS%3A528%3ADC%252BD3MXitVOhs7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2001&pages=3-26&author=C.+A.+Lipinskiauthor=F.+Lombardoauthor=B.+W.+Dominyauthor=P.+J.+Feeney&title=Experimental+and+computational+approaches+to+estimate+solubility+and+permeability+in+drug+discovery+and+development+settings&doi=10.1016%2FS0169-409X%2800%2900129-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Experimental and computational approaches to estimate solubility and permeability in drug discovery and development settings</span></div><div class="casAuthors">Lipinski, C. A.; Lombardo, F.; Dominy, B. W.; Feeney, P. J.</div><div class="citationInfo"><span class="NLM_cas:title">Advanced Drug Delivery Reviews</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">1-3</span>),
    <span class="NLM_cas:pages">3-26</span>CODEN:
                <span class="NLM_cas:coden">ADDREP</span>;
        ISSN:<span class="NLM_cas:issn">0169-409X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ireland Ltd.</span>)
        </div><div class="casAbstract">A review with 50 refs.  Exptl. and computational approaches to est. soly. and permeability in discovery and development settings are described.  In the discovery setting 'the rule of 5' predicts that poor absorption or permeation is more likely when there are more than 5 H-bond donors, 10 H-bond acceptors, the mol. wt. (MWT) is greater than 500 and the calcd. Log P (CLogP) is greater than 5 (or MlogP >4.15).  Computational methodol. for the rule-based Moriguchi Log P (MLogP) calcn. is described.  Turbidimetric soly. measurement is described and applied to known drugs.  High throughput screening (HTS) leads tend to have higher MWT and Log P and lower turbidimetric soly. than leads in the pre-HTS era.  In the development setting, soly. calcns. focus on exact value prediction and are difficult because of polymorphism.  Recent work on linear free energy relationships and Log P approaches are critically reviewed.  Useful predictions are possible in closely related analog series when coupled with exptl. thermodn. soly. measurements.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoYDqa02g16urVg90H21EOLACvtfcHk0lg0LfFG0-Rppw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXitVOhs7o%253D&md5=c60bb89da68f051c0ee7ac4c0468a0e4</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1016%2FS0169-409X%2800%2900129-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0169-409X%252800%252900129-0%26sid%3Dliteratum%253Aachs%26aulast%3DLipinski%26aufirst%3DC.%2BA.%26aulast%3DLombardo%26aufirst%3DF.%26aulast%3DDominy%26aufirst%3DB.%2BW.%26aulast%3DFeeney%26aufirst%3DP.%2BJ.%26atitle%3DExperimental%2520and%2520computational%2520approaches%2520to%2520estimate%2520solubility%2520and%2520permeability%2520in%2520drug%2520discovery%2520and%2520development%2520settings%26jtitle%3DAdv.%2520Drug%2520Delivery%2520Rev.%26date%3D2001%26volume%3D46%26spage%3D3%26epage%3D26%26doi%3D10.1016%2FS0169-409X%2800%2900129-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Alonso-Valenteen, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pacheco, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Srinivas, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rentsendorj, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lubow, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sims, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miao, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mikhael, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hwang, J. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abrol, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Medina Kauwe, L. K.</span></span> <span> </span><span class="NLM_article-title">HER3-targeted protein chimera forms endosomolytic capsomeres and self-assembles into stealth nucleocapsids for systemic tumor homing of RNA interference in vivo</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">11020</span>– <span class="NLM_lpage">11043</span>, <span class="refDoi"> DOI: 10.1093/nar/gkz900</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=10.1093%2Fnar%2Fgkz900" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=31617560" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhtVGqsr7K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2019&pages=11020-11043&author=F.+Alonso-Valenteenauthor=S.+Pachecoauthor=D.+Srinivasauthor=A.+Rentsendorjauthor=D.+Chuauthor=J.+Lubowauthor=J.+Simsauthor=T.+Miaoauthor=S.+Mikhaelauthor=J.+Y.+Hwangauthor=R.+Abrolauthor=L.+K.+Medina+Kauwe&title=HER3-targeted+protein+chimera+forms+endosomolytic+capsomeres+and+self-assembles+into+stealth+nucleocapsids+for+systemic+tumor+homing+of+RNA+interference+in+vivo&doi=10.1093%2Fnar%2Fgkz900"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">HER3-targeted protein chimera forms endosomolytic capsomeres and self-assembles into stealth nucleocapsids for systemic tumor homing of RNA interference in vivo</span></div><div class="casAuthors">Alonso-Valenteen, Felix; Pacheco, Sayuri; Srinivas, Dustin; Rentsendorj, Altan; Chu, David; Lubow, Jay; Sims, Jessica; Miao, Tianxin; Mikhael, Simoun; Hwang, Jae Youn; Abrol, Ravinder; Kauwe, Lali K. Medina</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">11020-11043</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">1362-4962</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">RNA interference represents a potent intervention for cancer treatment but requires a robust delivery agent for transporting gene-modulating mols., such as small interfering RNAs (siRNAs).  Although numerous mol. approaches for siRNA delivery are adequate in vitro, delivery to therapeutic targets in vivo is limited by payload integrity, cell targeting, efficient cell uptake, and membrane penetration.  We constructed nonviral biomaterials to transport small nucleic acids to cell targets, including tumor cells, on the basis of the self-assembling and cell-penetrating activities of the adenovirus capsid penton base.  Our recombinant penton base chimera contains polypeptide domains designed for noncovalent assembly with anionic mols. and tumor homing.  Here, structural modeling, mol. dynamics simulations, and functional assays suggest that it forms pentameric units resembling viral capsomeres that assemble into larger capsid-like structures when combined with siRNA cargo.  Pentamerization forms a barrel lined with charged residues mediating pH-responsive dissocn. and exposing masked domains, providing insight on the endosomolytic mechanism.  The therapeutic impact was examd. on tumors expressing high levels of HER3/ErbB3 that are resistant to clin. inhibitors.  Our findings suggest that our construct may utilize ligand mimicry to avoid host attack and target the siRNA to HER3+ tumors by forming multivalent capsid-like structures.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo0Nu1sOinyv7Vg90H21EOLACvtfcHk0ljLyqdCVfi7YA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhtVGqsr7K&md5=d8a9f4539ea7889d4088171c64995406</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkz900&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkz900%26sid%3Dliteratum%253Aachs%26aulast%3DAlonso-Valenteen%26aufirst%3DF.%26aulast%3DPacheco%26aufirst%3DS.%26aulast%3DSrinivas%26aufirst%3DD.%26aulast%3DRentsendorj%26aufirst%3DA.%26aulast%3DChu%26aufirst%3DD.%26aulast%3DLubow%26aufirst%3DJ.%26aulast%3DSims%26aufirst%3DJ.%26aulast%3DMiao%26aufirst%3DT.%26aulast%3DMikhael%26aufirst%3DS.%26aulast%3DHwang%26aufirst%3DJ.%2BY.%26aulast%3DAbrol%26aufirst%3DR.%26aulast%3DMedina%2BKauwe%26aufirst%3DL.%2BK.%26atitle%3DHER3-targeted%2520protein%2520chimera%2520forms%2520endosomolytic%2520capsomeres%2520and%2520self-assembles%2520into%2520stealth%2520nucleocapsids%2520for%2520systemic%2520tumor%2520homing%2520of%2520RNA%2520interference%2520in%2520vivo%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2019%26volume%3D47%26spage%3D11020%26epage%3D11043%26doi%3D10.1093%2Fnar%2Fgkz900" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">H, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herrmann, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buettner, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jove, R.</span></span> <span> </span><span class="NLM_article-title">Revisiting Stat3 signalling in cancer: new and unexpected biological functions</span>. <i>Nat. Rev. Cancer.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">736</span>– <span class="NLM_lpage">746</span>, <span class="refDoi"> DOI: 10.1038/nrc3818</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=10.1038%2Fnrc3818" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=25342631" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvVSmtLnF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2014&pages=736-746&author=H.+Yuauthor=L.+Hauthor=A.+Herrmannauthor=R.+Buettnerauthor=R.+Jove&title=Revisiting+Stat3+signalling+in+cancer%3A+new+and+unexpected+biological+functions&doi=10.1038%2Fnrc3818"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Revisiting STAT3 signalling in cancer: new and unexpected biological functions</span></div><div class="casAuthors">Yu, Hua; Lee, Heehyoung; Herrmann, Andreas; Buettner, Ralf; Jove, Richard</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">736-746</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The Janus kinases (JAKs) and signal transducer and activator of transcription (STAT) proteins, particularly STAT3, are among the most promising new targets for cancer therapy.  In addn. to interleukin-6 (IL-6) and its family members, multiple pathways, including G-protein-coupled receptors (GPCRs), Toll-like receptors (TLRs) and microRNAs were recently identified to regulate JAK-STAT signalling in cancer.  Well known for its role in tumor cell proliferation, survival, invasion and immunosuppression, JAK-STAT3 signalling also promotes cancer through inflammation, obesity, stem cells and the pre-metastatic niche.  In addn. to its established role as a transcription factor in cancer, STAT3 regulates mitochondrion functions, as well as gene expression through epigenetic mechanisms.  Newly identified regulators and functions of JAK-STAT3 in tumors are important targets for potential therapeutic strategies in the treatment of cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq3PGL62I5KvrVg90H21EOLACvtfcHk0ljLyqdCVfi7YA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvVSmtLnF&md5=355a07303813d7ecadb4b085c33c5e0a</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1038%2Fnrc3818&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc3818%26sid%3Dliteratum%253Aachs%26aulast%3DYu%26aufirst%3DH.%26aulast%3DH%26aufirst%3DL.%26aulast%3DHerrmann%26aufirst%3DA.%26aulast%3DBuettner%26aufirst%3DR.%26aulast%3DJove%26aufirst%3DR.%26atitle%3DRevisiting%2520Stat3%2520signalling%2520in%2520cancer%253A%2520new%2520and%2520unexpected%2520biological%2520functions%26jtitle%3DNat.%2520Rev.%2520Cancer.%26date%3D2014%26volume%3D14%26spage%3D736%26epage%3D746%26doi%3D10.1038%2Fnrc3818" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Avdeef, A.</span></span> <span> </span><span class="NLM_article-title">Physicochemical profiling (solubility, permeability and charge state)</span>. <i>Curr. Top. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">277</span>– <span class="NLM_lpage">351</span>, <span class="refDoi"> DOI: 10.2174/1568026013395100</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=10.2174%2F1568026013395100" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=11899112" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=1%3ACAS%3A528%3ADC%252BD3MXms1Khs7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2001&pages=277-351&author=A.+Avdeef&title=Physicochemical+profiling+%28solubility%2C+permeability+and+charge+state%29&doi=10.2174%2F1568026013395100"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Physicochemical profiling (solubility, permeability and charge state)</span></div><div class="casAuthors">Avdeef, Alex</div><div class="citationInfo"><span class="NLM_cas:title">Current Topics in Medicinal Chemistry (Hilversum, Netherlands)</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">277-351</span>CODEN:
                <span class="NLM_cas:coden">CTMCCL</span>;
        ISSN:<span class="NLM_cas:issn">1568-0266</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  About 30% of drug candidate mols. are rejected due to pharmacokinetic-related failures.  When poor pharmaceutical properties are discovered in development, the costs of bringing a potent but poorly absorbable mol. to a product stage by "formulation" can become very high.  Fast and reliable in vitro prediction strategies are needed to filter out problematic mols. at the earliest stages of discovery.  This review will consider recent developments in physicochem. profiling used to identify candidate mols. with phys. properties related to good oral absorption.  Poor soly. and poor permeability account for many PK failures.  FDA's Biopharmaceutics Classification System (BCS) is an attempt to rationalize the crit. components related to oral absorption.  The core idea in the BCS is an in vitro transport model, centrally embracing permeability and soly., with qualifications related to pH and dissoln.  The objective of the BCS is to predict in vivo performance of drug products from in vitro measurements of permeability and soly.  In principle, the framework of the BCS could serve the interests of the earliest stages of discovery research.  The BCS can be rationalized by considering Fick's first law, applied to membranes.  When mols. are introduced on one side of a lipid membrane barrier (e.g., epithelial cell wall) and no such mols. are on the other side, passive diffusion will drive the mols. across the membrane.  When certain simplifying assumptions are made, the flux equation in Fick's law reduces simply to a product of permeability and soly.  Many other measurable properties are closely related to permeability and soly.  Permeability (Pe) is a kinetic parameter related to lipophilicity (as indicated by the partition and distribution coeffs., log P and log D).  Retention (R) of lipophilic mols. by the membrane (which is related to lipophilicity and may predict. PK vols. of distribution) influences the characterization of permeability.  Furthermore, strong drug interactions with serum proteins can influence permeability.  The unstirred water layer on both sides of the membrane barrier can impose limits on permeability.  Soly. (S) is a thermodn. parameter, and is closely related to dissoln., a kinetic parameter.  The unstirred water layer on the surfaces of suspended solids imposes limits on dissoln.  Bile acids affect both soly. and dissoln., by a micellization effect.  For ionizable mols., pH plays a crucial role.  The charge state that a mol. exhibits at a particular pH is characterized by the ionization const. (pKa) of the mol.  Buffers effect pH gradients in the unstirred water layers, which can dramatically affect both permeability and dissoln. of ionizable mols.  In this review, we will focus on the emerging instrumental methods for the measurement of the physicochem. parameters Pe, S, pKa, R, log P, and log D (and their pH-profiles).  These physicochem. profiles can be valuable tools for the medicinal chemists, aiding in the prediction of in vivo oral absorption.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqUuP6xv9GKXLVg90H21EOLACvtfcHk0ljLyqdCVfi7YA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXms1Khs7o%253D&md5=df94226deaa13e5ade1caec003ae6733</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.2174%2F1568026013395100&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1568026013395100%26sid%3Dliteratum%253Aachs%26aulast%3DAvdeef%26aufirst%3DA.%26atitle%3DPhysicochemical%2520profiling%2520%2528solubility%252C%2520permeability%2520and%2520charge%2520state%2529%26jtitle%3DCurr.%2520Top.%2520Med.%2520Chem.%26date%3D2001%26volume%3D1%26spage%3D277%26epage%3D351%26doi%3D10.2174%2F1568026013395100" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Martinez, M. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amidon, G. L.</span></span> <span> </span><span class="NLM_article-title">A mechanistic approach to understanding the factors affecting drug absorption: a review of fundamentals</span>. <i>J. Clin. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>42</i></span>,  <span class="NLM_fpage">620</span>– <span class="NLM_lpage">43</span>, <span class="refDoi"> DOI: 10.1177/00970002042006005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=10.1177%2F00970002042006005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=12043951" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=1%3ACAS%3A528%3ADC%252BD38Xks1OgsLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2002&pages=620-43&author=M.+N.+Martinezauthor=G.+L.+Amidon&title=A+mechanistic+approach+to+understanding+the+factors+affecting+drug+absorption%3A+a+review+of+fundamentals&doi=10.1177%2F00970002042006005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">A mechanistic approach to understanding the factors affecting drug absorption: A review of fundamentals</span></div><div class="casAuthors">Martinez, Marilyn N.; Amidon, Gordon L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Clinical Pharmacology</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">620-643</span>CODEN:
                <span class="NLM_cas:coden">JCPCBR</span>;
        ISSN:<span class="NLM_cas:issn">0091-2700</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">A review.  This article provides an overview of the patient-specific and drug-specific variables that can affect drug absorption following oral product administration.  The oral absorption of any chem. entity reflects a complex spectrum of events.  Factors influencing product bioavailability include drug soly., permeability, and the rate of in vivo dissoln.  In this regard, the Biopharmaceutics Classification System has proven to be an important tool for predicting compds. likely to be assocd. with bioavailability problems.  It also helps in identifying those factors that may alter the rate and extent of drug absorption.  Product bioavailability can also be markedly influenced by patient attributes such as the integrity of the gastrointestinal tract, physiol. status, site of drug absorption, membrane transporters, presystemic drug metab. (intrinsic variables), and extrinsic variables such as the effect of food or concomitant medication.  Through an awareness of a drug's physicochem. properties and the physiol. processes affecting drug absorption, the skilled pharmaceutical scientist can develop formulations that will maximize product availability.  By appreciating the potential impact of patient physiol. status, phenotype, age, gender, and lifestyle, dosing regimens can be tailored to better meet the needs of the individual patient.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr7090GVIXnsrVg90H21EOLACvtfcHk0lhkP5MlwAlcJg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38Xks1OgsLc%253D&md5=65abdd33877aa4cd9295c594a338584e</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1177%2F00970002042006005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F00970002042006005%26sid%3Dliteratum%253Aachs%26aulast%3DMartinez%26aufirst%3DM.%2BN.%26aulast%3DAmidon%26aufirst%3DG.%2BL.%26atitle%3DA%2520mechanistic%2520approach%2520to%2520understanding%2520the%2520factors%2520affecting%2520drug%2520absorption%253A%2520a%2520review%2520of%2520fundamentals%26jtitle%3DJ.%2520Clin.%2520Pharmacol.%26date%3D2002%26volume%3D42%26spage%3D620%26epage%3D43%26doi%3D10.1177%2F00970002042006005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Di, L.</span>; <span class="NLM_string-name">Kerns, E.</span></span> <i>Drug-like Properties: Concepts, Structure Design and Methods from ADME to Toxicity Optimization</i>, <span class="NLM_edition">2</span>nd ed.; <span class="NLM_publisher-name">Elsevier</span>: <span class="NLM_publisher-loc">Cambridge, MA</span>, <span class="NLM_year">2016</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2016&author=L.+Di&author=E.+Kerns&title=Drug-like+Properties%3A+Concepts%2C+Structure+Design+and+Methods+from+ADME+to+Toxicity+Optimization"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DDi%26aufirst%3DL.%26btitle%3DDrug-like%2520Properties%253A%2520Concepts%252C%2520Structure%2520Design%2520and%2520Methods%2520from%2520ADME%2520to%2520Toxicity%2520Optimization%26pub%3DElsevier%26date%3D2016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chaturvedi, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Decker, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Odinecs, A.</span></span> <span> </span><span class="NLM_article-title">Prediction of pharmacokinetic properties using experimental approaches during early drug discovery</span>. <i>Curr. Opin. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">452</span>– <span class="NLM_lpage">463</span>, <span class="refDoi"> DOI: 10.1016/S1367-5931(00)00228-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=10.1016%2FS1367-5931%2800%2900228-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=11470610" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=1%3ACAS%3A528%3ADC%252BD3MXltlyqsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2001&pages=452-463&author=P.+R.+Chaturvediauthor=C.+J.+Deckerauthor=A.+Odinecs&title=Prediction+of+pharmacokinetic+properties+using+experimental+approaches+during+early+drug+discovery&doi=10.1016%2FS1367-5931%2800%2900228-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Prediction of pharmacokinetic properties using experimental approaches during early drug discovery</span></div><div class="casAuthors">Chaturvedi, P. R.; Decker, C. J.; Odinecs, A.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Chemical Biology</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">452-463</span>CODEN:
                <span class="NLM_cas:coden">COCBF4</span>;
        ISSN:<span class="NLM_cas:issn">1367-5931</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ltd.</span>)
        </div><div class="casAbstract">A review with refs.  There has been a significant increase in the no. of compds. synthesized in early drug-discovery programs with the advances in combinatorial chem. and high-throughput biol. screening efforts.  Various in silico, in vitro and in situ approaches have been described in literature that achieve higher throughput pharmacokinetic screening.  In silico methodologies have mainly attempted to quantify the prospects of oral absorption of compds. based upon their physico-chem. properties.  There is a greater availability of in vitro and in situ approaches to screen compds. for intestinal permeability (as a surrogate for absorption) and metabolic stability (as a surrogate for clearance).  More recent modifications of the in vitro and in situ approaches to assess the potential of absorption and metab. have enabled a higher throughput and an ability to correlate better with in vivo pharmacokinetics of compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrAQSfVKDr8KrVg90H21EOLACvtfcHk0lhkP5MlwAlcJg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXltlyqsb4%253D&md5=36956032f958260f71e20d2a0939d432</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1016%2FS1367-5931%2800%2900228-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1367-5931%252800%252900228-3%26sid%3Dliteratum%253Aachs%26aulast%3DChaturvedi%26aufirst%3DP.%2BR.%26aulast%3DDecker%26aufirst%3DC.%2BJ.%26aulast%3DOdinecs%26aufirst%3DA.%26atitle%3DPrediction%2520of%2520pharmacokinetic%2520properties%2520using%2520experimental%2520approaches%2520during%2520early%2520drug%2520discovery%26jtitle%3DCurr.%2520Opin.%2520Chem.%2520Biol.%26date%3D2001%26volume%3D5%26spage%3D452%26epage%3D463%26doi%3D10.1016%2FS1367-5931%2800%2900228-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Guha, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dexheimer, T. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kestranek, A. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jadhav, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chervenak, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ford, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simeonov, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roth, G. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, C. J.</span></span> <span> </span><span class="NLM_article-title">Exploratory analysis of kinetic solubility measurements of a small molecule library</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">4127</span>– <span class="NLM_lpage">4134</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2011.05.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=10.1016%2Fj.bmc.2011.05.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=21640593" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=1%3ACAS%3A528%3ADC%252BC3MXotVehsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2011&pages=4127-4134&author=R.+Guhaauthor=T.+S.+Dexheimerauthor=A.+N.+Kestranekauthor=A.+Jadhavauthor=A.+M.+Chervenakauthor=M.+G.+Fordauthor=A.+Simeonovauthor=G.+P.+Rothauthor=C.+J.+Thomas&title=Exploratory+analysis+of+kinetic+solubility+measurements+of+a+small+molecule+library&doi=10.1016%2Fj.bmc.2011.05.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Exploratory analysis of kinetic solubility measurements of a small molecule library</span></div><div class="casAuthors">Guha, Rajarshi; Dexheimer, Thomas S.; Kestranek, Aimee N.; Jadhav, Ajit; Chervenak, Andrew M.; Ford, Michael G.; Simeonov, Anton; Roth, Gregory P.; Thomas, Craig J.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">4127-4134</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Kinetic soly. measurements using prototypical assay buffer conditions are presented for a ∼58,000 member library of small mols.  Analyses of the data based upon phys. and calcd. properties of each individual mol. were performed and resulting trends were considered in the context of commonly held opinions of how physicochem. properties influence aq. soly.  The data was further analyzed using a decision tree model for soly. prediction and via a multi-dimensional assessment of physicochem. relationships to soly. in the context of specific rule-breakers relative to common dogma.  The role of soly. as a determinant of assay outcome is also considered based upon each compd.'s cross-assay activity score for a collection of publicly available screening results.  Further, the role of soly. as a governing factor for colloidal aggregation formation within a specified assay setting is examd. and considered as a possible cause of a high cross-assay activity score.  The results of this soly. profile should aid chemists during library design and optimization efforts and represent a useful training set for computational soly. prediction.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpU9biLJPDeaLVg90H21EOLACvtfcHk0lhkP5MlwAlcJg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXotVehsbg%253D&md5=1f5b67268444cd3911665ea395bbf72f</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2011.05.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2011.05.005%26sid%3Dliteratum%253Aachs%26aulast%3DGuha%26aufirst%3DR.%26aulast%3DDexheimer%26aufirst%3DT.%2BS.%26aulast%3DKestranek%26aufirst%3DA.%2BN.%26aulast%3DJadhav%26aufirst%3DA.%26aulast%3DChervenak%26aufirst%3DA.%2BM.%26aulast%3DFord%26aufirst%3DM.%2BG.%26aulast%3DSimeonov%26aufirst%3DA.%26aulast%3DRoth%26aufirst%3DG.%2BP.%26aulast%3DThomas%26aufirst%3DC.%2BJ.%26atitle%3DExploratory%2520analysis%2520of%2520kinetic%2520solubility%2520measurements%2520of%2520a%2520small%2520molecule%2520library%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2011%26volume%3D19%26spage%3D4127%26epage%3D4134%26doi%3D10.1016%2Fj.bmc.2011.05.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kerns, E. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Di, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carter, G. T.</span></span> <span> </span><span class="NLM_article-title">In vitro solubility assays in drug discovery</span>. <i>Curr. Drug Metab.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">879</span>– <span class="NLM_lpage">885</span>, <span class="refDoi"> DOI: 10.2174/138920008786485100</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=10.2174%2F138920008786485100" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=18991584" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=1%3ACAS%3A528%3ADC%252BD1MXivF2nsb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2008&pages=879-885&author=E.+H.+Kernsauthor=L.+Diauthor=G.+T.+Carter&title=In+vitro+solubility+assays+in+drug+discovery&doi=10.2174%2F138920008786485100"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">In vitro solubility assays in drug discovery</span></div><div class="casAuthors">Kerns, Edward H.; Di, Li; Carter, Guy T.</div><div class="citationInfo"><span class="NLM_cas:title">Current Drug Metabolism</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">879-885</span>CODEN:
                <span class="NLM_cas:coden">CDMUBU</span>;
        ISSN:<span class="NLM_cas:issn">1389-2002</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  The soly. of a compd. depends on its structure and soln. conditions.  Structure dets. the lipophilicity, hydrogen bonding, mol. vol., crystal energy and ionizability, which det. soly.  Soln. conditions are affected by pH, cosolvents, additives, ionic strength, time and temp.  Many drug discovery expts. are conducted under "kinetic" soly. conditions.  In drug discovery, soly. has a major impact on bioassays, formulation for in vivo dosing, and intestinal absorption.  A good goal for the soly. of drug discovery compds. is >60 ug/mL.  Equil. soly. assays can be conducted in moderate throughput, by incubating excess solid with buffer and agitating for several days, prior to filtration and HPLC quantitation.  Kinetic soly. assays are performed in high throughput with shorter incubation times and high throughput analyses using plate readers.  The most frequently used of these are the nephelometric assay and direct UV assay, which begin by adding a small vol. of DMSO stock soln. of each test compd. to buffer.  In nephelometry, this soln. is serially dild. across a microtiter plate and undissolved particles are detected via light scattering.  In direct UV, undissolved particles are sepd. by filtration, after which the dissolved material is quantitated using UV absorption.  Equil. soly. is useful for preformulation.  Kinetic soly. is useful for rapid compd. assessment, guiding optimization via structure modification, and diagnosing bioassays.  It is often useful to customize soly. expts. using conditions that answer specific research questions of drug discovery teams, such as compd. selection and vehicle development for pharmacol. and PK studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq1Pe5z4JU_6LVg90H21EOLACvtfcHk0ljjeqpzp2BHMQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXivF2nsb8%253D&md5=416cf5bfde2278eac1ba2c67feb1f62e</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.2174%2F138920008786485100&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F138920008786485100%26sid%3Dliteratum%253Aachs%26aulast%3DKerns%26aufirst%3DE.%2BH.%26aulast%3DDi%26aufirst%3DL.%26aulast%3DCarter%26aufirst%3DG.%2BT.%26atitle%3DIn%2520vitro%2520solubility%2520assays%2520in%2520drug%2520discovery%26jtitle%3DCurr.%2520Drug%2520Metab.%26date%3D2008%26volume%3D9%26spage%3D879%26epage%3D885%26doi%3D10.2174%2F138920008786485100" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 2 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Jinyun Dong, Xiang-Dong Cheng, Wei-Dong Zhang, <span class="NLM_string-name hlFld-ContribAuthor">Jiang-Jiang Qin</span>. </span><span class="cited-content_cbyCitation_article-title">Recent Update on Development of Small-Molecule STAT3 Inhibitors for Cancer Therapy: From Phosphorylation Inhibition to Protein Degradation. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (13)
                                     , 8884-8915. <a href="https://doi.org/10.1021/acs.jmedchem.1c00629" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.1c00629</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.1c00629&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.1c00629%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DRecent%252BUpdate%252Bon%252BDevelopment%252Bof%252BSmall-Molecule%252BSTAT3%252BInhibitors%252Bfor%252BCancer%252BTherapy%25253A%252BFrom%252BPhosphorylation%252BInhibition%252Bto%252BProtein%252BDegradation%26aulast%3DDong%26aufirst%3DJinyun%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D06042021%26date%3D25062021%26volume%3D64%26issue%3D13%26spage%3D8884%26epage%3D8915" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ana  Čikoš</span>, <span class="hlFld-ContribAuthor ">Snježana  Dragojević</span>, <span class="hlFld-ContribAuthor ">Adrijana  Kubiček</span>. </span><span class="cited-content_cbyCitation_article-title">Degradation Products of Azetidine Core G334089 - Isolation, Structure Elucidation and Pathway. </span><span class="cited-content_cbyCitation_journal-name">Journal of Pharmaceutical and Biomedical Analysis</span><span> <strong>2021,</strong> <em>22 </em>, 114232. <a href="https://doi.org/10.1016/j.jpba.2021.114232" title="DOI URL">https://doi.org/10.1016/j.jpba.2021.114232</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.jpba.2021.114232&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.jpba.2021.114232%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Pharmaceutical%2520and%2520Biomedical%2520Analysis%26atitle%3DDegradation%252BProducts%252Bof%252BAzetidine%252BCore%252BG334089%252B-%252BIsolation%25252C%252BStructure%252BElucidation%252Band%252BPathway%26aulast%3D%25C4%258Ciko%25C5%25A1%26aufirst%3DAna%26date%3D2021%26volume%3D22%26spage%3D114232" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01705/20210107/images/medium/jm0c01705_0025.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01705/20210107/images/large/jm0c01705_0025.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01705&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01705/20210107/images/medium/jm0c01705_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01705/20210107/images/large/jm0c01705_0002.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Azetidine analogues inhibit STAT3 DNA-binding activity in vitro. Nuclear extracts of equal total protein prepared from NIH3T3/v-Src fibroblasts containing activated STAT3 were preincubated with increasing concentrations of the designated versions of azetidines (A) salicylic acids, (B) benzoic acids, (C) methyl esters, (D) hydroxamic acids/methylamines, or (E) heterocycles for 30 min at room temperature prior to incubating with the radiolabeled hSIE probe that binds STAT3 and performing EMSA analysis; bands corresponding to STAT3:DNA complexes in gel were quantified using ImageJ and represented as a percent of control and plotted against the concentration of compounds, from which IC<sub>50</sub> values were determined. Positions of STAT3:DNA complexes in gel are labeled; control lanes (0) represent nuclear extracts pretreated with 10% DMSO. Data are representative of two to three independent determinations.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01705/20210107/images/large/jm0c01705_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01705&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01705/20210107/images/medium/jm0c01705_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01705/20210107/images/large/jm0c01705_0003.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Structures of initial azetidine-based STAT3 inhibitors.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01705/20210107/images/large/jm0c01705_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01705&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01705/20210107/images/medium/jm0c01705_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01705/20210107/images/large/jm0c01705_0004.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. <i>In vitro</i> cell viability and growth studies for effects of active azetidine analogs. (A) Human breast cancer MDA-MB-231 and MDA-MB-468 cells harboring aberrantly active STAT3 and (B) normal breast epithelial MCF-10A or breast cancer MCF-7 cells that do not and growing in 96-well culture were treated once with 0–10 μM of the indicated STAT3 inhibitors, or (C) human breast cancer cells in 96-well culture were first treated with 1 μM <b><u class="uu">7g</u></b> for 6 h followed by treatment with 0–100 nM docetaxel (Doc) or 0–20 μM cisplatin (Cis), or the cells were treated with docetaxel or cisplatin alone. After 72-h culture, cells were harvested and subjected to CyQuant cell proliferation assay for the number of viable cells, which are plotted as % cell viability against concentration from which EC<sub>50</sub> values were derived; or (D) human breast cancer MDA-MB-231 or MCF-7 cells in a 6-well culture plate were untreated or treated once with 2 μM <b><u class="uu">9k</u></b> and every 24 h, cells were harvested and subjected to trypan blue exclusion/phase-contrast microscopy for viable cell counts, which were plotted against the duration of treatment. Values are mean ± SEM of two to three studies each in three replicates.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01705/20210107/images/large/jm0c01705_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01705&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01705/20210107/images/medium/jm0c01705_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01705/20210107/images/large/jm0c01705_0005.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Isothermal titration calorimetry (ITC) measurements of <b><u class="uu">7g</u></b>, and <b><u class="uu">9k</u></b> during incubation with STAT3. (A) Binding isotherm from the integrated thermogram fit using the one-site model in the PEAQ-ITC software generated from the titration of the inhibitors, <b><u class="uu">7g</u></b> (red) and <b><u class="uu">9k</u></b> (blue), into STAT3. The <i>K</i><sub>D</sub> was 880 and 960 nM, respectively, and (B) the signature plots showing the thermodynamics parameters for each titration reveal Δ<i>H</i> = −22.7 kJ/mol, Δ<i>G</i> = −34.6 kJ/mol, and −<i>T</i>Δ<i>S</i> = −11.9 kJ/mol for <b><u class="uu">7g</u></b> and Δ<i>H</i> = −20.8 kJ/mol, Δ<i>G</i> = −34.4 kJ/mol, and −<i>T</i>Δ<i>S</i> = −13.6 kJ/mol for <b><u class="uu">9k</u></b>. Data are representative of three independent experiments.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01705/20210107/images/large/jm0c01705_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01705&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01705/20210107/images/medium/jm0c01705_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01705/20210107/images/large/jm0c01705_0006.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Comparing the effect of new analogues on STAT1, STAT3, or STAT5 DNA-binding activity in vitro. Nuclear extracts of equal total protein prepared from epidermal growth factor-stimulated NIH3T3/EGFR fibroblasts containing activated STAT1, STAT3, and STAT5 were preincubated with increasing concentrations of the designated compounds for 30 min at room temperature prior to incubating with the radiolabeled hSIE probe that binds STAT1 and STAT3 (upper panel) or the MGFe probe that binds STAT5 (bottom panel) and performing EMSA analysis. Positions of STAT:DNA complexes in gel are labeled; control lanes (0) represent nuclear extracts preincubated with 10% DMSO. Data are representative of two to three independent determinations.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01705/20210107/images/large/jm0c01705_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01705&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01705/20210107/images/medium/jm0c01705_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01705/20210107/images/large/jm0c01705_0007.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Effects of compounds on constitutive STAT3 activation in tumor cells. (A) Nuclear extracts of equal total protein prepared from the human breast cancer, MDA-MB-468 cells untreated (DMSO, 0) or treated with 5 μM of the indicated analogues for 1–3 h were subjected to STAT3 DNA-binding assay using the hSIE probe that binds STAT3, and (B) immunoblotting analysis of whole-cell lysates of equal total protein prepared from (i) and (ii) MDA-MB-231 cells untreated (DMSO, 0) or treated with 1 or 3 μM of <b><u class="uu">7g</u></b> for 3–24 h or (iii) MDA-MB-468 cells untreated (DMSO, 0) or treated with 3 μM of <b><u class="uu">9k</u></b> for 3–24 h and probing for pY705STAT3, STAT3, or tubulin. Positions of STAT3:DNA complex or proteins in gel are shown; control (0 or Con) lane represents whole-cell lysates or nuclear extracts prepared from 0.05% DMSO-treated cells. Data are representative of two to three independent determinations.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01705/20210107/images/large/jm0c01705_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01705&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01705/20210107/images/medium/jm0c01705_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01705/20210107/images/large/jm0c01705_0008.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Compounds <u class="uu"><b>7e</b></u>, <u class="uu"><b>7f</b></u>, <u class="uu"><b>7g</b></u>, and <u class="uu"><b>9k</b></u> inhibit the colony survival of human breast cancer cells in vitro. Human breast cancer MDA-MB-231 cells were seeded as single-cell culture and treated once with 0–1 μM of the indicated compounds and allowed to culture until large colonies were visible, which were stained with crystal violet and imaged. Data are representative of three independent determinations.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01705/20210107/images/large/jm0c01705_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01705&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig8" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01705/20210107/images/medium/jm0c01705_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01705/20210107/images/large/jm0c01705_0009.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Inhibition of STAT3 target gene expression in breast cancer cells. Immunoblotting analysis of whole-cell lysates of equal total protein prepared from MDA-MB-231 cells untreated (DMSO, C) or treated with 1 or 3 μM of <b><u class="uu">7g</u></b> for 3–24 h and probing for c-Myc Bcl-2, VEGF, survivin, or tubulin. Positions of proteins in gel are shown; control (C) lane represents whole-cell lysates prepared from 0.05% DMSO-treated cells. Data are representative of two to three independent determinations.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01705/20210107/images/large/jm0c01705_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01705&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig9" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01705/20210107/images/medium/jm0c01705_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01705/20210107/images/large/jm0c01705_0010.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. Induction of apoptosis of human breast cancer cells. Human breast cancer, MDA-MB-231 and MDA-MB-468 cells in culture were treated with 1 or 3 μM <b><u class="uu">7g</u></b> for 0–24 h, whole-cell lysates were prepared, and samples of equal total protein were subjected to SDS/PAGE–Western blotting analysis probing for pYSTAT3, STAT3, full-length PARP, cleaved PARP, and tubulin. Positions of proteins in gel are shown; control (0) lane represents whole-cell lysates prepared from 0.05% DMSO-treated cells. Data are representative of two independent determinations.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01705/20210107/images/large/jm0c01705_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01705&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i33">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_52360" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_52360" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 48 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Darnell, J. E.,  Jr.</span></span> <span> </span><span class="NLM_article-title">Reflections on Stat3, Stat5, and Stat6 as fat STATs</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>93</i></span>,  <span class="NLM_fpage">6221</span>– <span class="NLM_lpage">6224</span>, <span class="refDoi"> DOI: 10.1073/pnas.93.13.6221</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=10.1073%2Fpnas.93.13.6221" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=8692794" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=1%3ACAS%3A528%3ADyaK28XjvFejtrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=93&publication_year=1996&pages=6221-6224&author=J.+E.+Darnell&title=Reflections+on+Stat3%2C+Stat5%2C+and+Stat6+as+fat+STATs&doi=10.1073%2Fpnas.93.13.6221"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Reflections on STAT3, STAT5, and STAT6 as fat STATs</span></div><div class="casAuthors">Darnell, James E., Jr.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">93</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">6221-6224</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">A review with 45 refs. on the involvement of STAT proteins in wt. control with emphasis on the work of N. Ghilardi et al. entitled "Defective STAT signaling by the leptin receptor in diabetic mice" appearing in this issue of the Proceedings.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqhc3TK1Kwa87Vg90H21EOLACvtfcHk0li_72ofY8vPvw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28XjvFejtrw%253D&md5=2a061011708b2836362022dc78d82c4f</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1073%2Fpnas.93.13.6221&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.93.13.6221%26sid%3Dliteratum%253Aachs%26aulast%3DDarnell%26aufirst%3DJ.%2BE.%26atitle%3DReflections%2520on%2520Stat3%252C%2520Stat5%252C%2520and%2520Stat6%2520as%2520fat%2520STATs%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D1996%26volume%3D93%26spage%3D6221%26epage%3D6224%26doi%3D10.1073%2Fpnas.93.13.6221" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Darnell, J. E.,  Jr.</span></span> <span> </span><span class="NLM_article-title">The Jak-STAT pathway: summary of initial studies and recent advances</span>. <i>Recent Prog. Horm. Res.</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>51</i></span>,  <span class="NLM_fpage">391</span>– <span class="NLM_lpage">403</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=8701087" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=1%3ACAS%3A528%3ADyaK2sXktFKis78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=1996&pages=391-403&author=J.+E.+Darnell&title=The+Jak-STAT+pathway%3A+summary+of+initial+studies+and+recent+advances"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">The JAK-STAT pathway: summary of initial studies and recent advances</span></div><div class="casAuthors">Darnell, James E., Jr.</div><div class="citationInfo"><span class="NLM_cas:title">Recent Progress in Hormone Research</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">391-404</span>CODEN:
                <span class="NLM_cas:coden">RPHRA6</span>;
        ISSN:<span class="NLM_cas:issn">0079-9963</span>.
    
            (<span class="NLM_cas:orgname">Endocrine Society</span>)
        </div><div class="casAbstract">A review and a discussion with many refs.  Polypeptides redirect cells by causing immediate changes in gene transcription.  A direct connection between polypeptide receptors on the cell surface and genes in the nucleus has been provided by the recognition of the JAK-STAT pathway.  Results leading to our present understanding of this pathway are reviewed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpnoxpk0vC5vrVg90H21EOLACvtfcHk0li_72ofY8vPvw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXktFKis78%253D&md5=385fd1dfee38ec19ef9d325a11ab4c58</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DDarnell%26aufirst%3DJ.%2BE.%26atitle%3DThe%2520Jak-STAT%2520pathway%253A%2520summary%2520of%2520initial%2520studies%2520and%2520recent%2520advances%26jtitle%3DRecent%2520Prog.%2520Horm.%2520Res.%26date%3D1996%26volume%3D51%26spage%3D391%26epage%3D403" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Darnell, J. E.,  Jr.</span></span> <span> </span><span class="NLM_article-title">STATs and gene regulation</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>277</i></span>,  <span class="NLM_fpage">1630</span>– <span class="NLM_lpage">1635</span>, <span class="refDoi"> DOI: 10.1126/science.277.5332.1630</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=10.1126%2Fscience.277.5332.1630" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=9287210" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=1%3ACAS%3A528%3ADyaK2sXmtVSisLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=277&publication_year=1997&pages=1630-1635&author=J.+E.+Darnell&title=STATs+and+gene+regulation&doi=10.1126%2Fscience.277.5332.1630"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">STATs and gene regulation</span></div><div class="casAuthors">Darnell, James E., Jr.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, D. C.)</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">277</span>
        (<span class="NLM_cas:issue">5332</span>),
    <span class="NLM_cas:pages">1630-1635</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">A review with 100 refs.  STATs (signal transducers and activators of transcription) are a family of latent cytoplasmic proteins that are activated to participate in gene control when cells encounter various extracellular polypeptides.  Biochem. and mol. genetic explorations have defined a single tyrosine phosphorylation site and, in a dimeric partner mol., a Src homol. 2 (SH2) phosphotyrosine-binding domain, a DNA interaction domain, and a no. of protein-protein interaction domains (with receptors, other transcription factors, the transcription machinery, and perhaps a tyrosine phosphatase).  Mouse genetics expts. have defined crucial roles for each known mammalian STAT.  The discovery of a STAT in Drosophila, and most recently in Dictyostelium discoideum, implies an ancient evolutionary origin for this dual-function set of proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr2cN7IJWCm2rVg90H21EOLACvtfcHk0li_72ofY8vPvw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXmtVSisLo%253D&md5=ce5d495245adb0ccaab93ba990a7e017</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1126%2Fscience.277.5332.1630&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.277.5332.1630%26sid%3Dliteratum%253Aachs%26aulast%3DDarnell%26aufirst%3DJ.%2BE.%26atitle%3DSTATs%2520and%2520gene%2520regulation%26jtitle%3DScience%26date%3D1997%26volume%3D277%26spage%3D1630%26epage%3D1635%26doi%3D10.1126%2Fscience.277.5332.1630" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bromberg, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Darnell, J. E.,  Jr.</span></span> <span> </span><span class="NLM_article-title">The role of STATs in transcriptional control and their impact on cellular function</span>. <i>Oncogene</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">2468</span>– <span class="NLM_lpage">2473</span>, <span class="refDoi"> DOI: 10.1038/sj.onc.1203476</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=10.1038%2Fsj.onc.1203476" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=10851045" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=1%3ACAS%3A528%3ADC%252BD3cXktVGhsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2000&pages=2468-2473&author=J.+Brombergauthor=J.+E.+Darnell&title=The+role+of+STATs+in+transcriptional+control+and+their+impact+on+cellular+function&doi=10.1038%2Fsj.onc.1203476"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">The role of STATs in transcriptional control and their impact on cellular function</span></div><div class="casAuthors">Bromberg, Jacqueline; Darnell, James E, Jr.</div><div class="citationInfo"><span class="NLM_cas:title">Oncogene</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">2468-2473</span>CODEN:
                <span class="NLM_cas:coden">ONCNES</span>;
        ISSN:<span class="NLM_cas:issn">0950-9232</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review with 68 refs.  The STAT proteins (Signal Transducers and Activators of Transcription), were identified in the last decade as transcription factors which were crit. in mediating virtually all cytokine driven signaling.  These proteins are latent in the cytoplasm and become activated through tyrosine phosphorylation which typically occurs through cytokine receptor assocd. kinases (JAKs) or growth factor receptor tyrosine kinases.  Recently a no. of non-receptor tyrosine kinases (for example src and abl) have been found to cause STAT phosphorylation.  Phosphorylated STATs form homo- or hetero-dimers, enter the nucleus and working coordinately with other transcriptional co-activators or transcription factors lead to increased transcriptional initiation.  In normal cells and in animals, ligand dependent activation of the STATs is a transient process, lasting for several minutes to several hours.  In contrast, in many cancerous cell lines and tumors, where growth factor dysregulation is frequently at the heart of cellular transformation, the STAT proteins (in particular Stats 1, 3 and 5) are persistently tyrosine phosphorylated or activated.  The importance of STAT activation to growth control in expts. using anti-sense mols. or dominant neg. STAT protein encoding constructs performed in cell lines or studies in animals lacking specific STATs strongly indicate that STATs play an important role in controlling cell cycle progression and apoptosis.  Stat1 plays an important role in growth arrest, in promoting apoptosis and is implicated as a tumor suppressor; while Stats 3 and 5 are involved in promoting cell cycle progression and cellular transformation and preventing apoptosis.  Many questions remain including: (1) a better understanding of how the STAT proteins through assocn. with other factors increase transcription initiation; (2) a more complete definition of the sets of genes which are activated by different STATs and (3) how these sets of activated genes differ as a function of cell type.  Finally, in the context of many cancers, where STATs are frequently persistently activated, an understanding of the mechanisms leading to their constitutive activation and defining the potential importance of persistent STAT activation in human tumorigenesis remains.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpKoOhp17wSLLVg90H21EOLACvtfcHk0lgqpW_S7rkj9A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXktVGhsb4%253D&md5=b504533a9132be2af9c0a0a827b93063</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1038%2Fsj.onc.1203476&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.onc.1203476%26sid%3Dliteratum%253Aachs%26aulast%3DBromberg%26aufirst%3DJ.%26aulast%3DDarnell%26aufirst%3DJ.%2BE.%26atitle%3DThe%2520role%2520of%2520STATs%2520in%2520transcriptional%2520control%2520and%2520their%2520impact%2520on%2520cellular%2520function%26jtitle%3DOncogene%26date%3D2000%26volume%3D19%26spage%3D2468%26epage%3D2473%26doi%3D10.1038%2Fsj.onc.1203476" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Darnell, J. E.,  Jr</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kerr, I. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stark, G. R.</span></span> <span> </span><span class="NLM_article-title">Jak-STAT pathways and transcriptional activation in response to IFNs and other extracellular signaling proteins</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">1994</span>,  <span class="NLM_volume"><i>264</i></span>,  <span class="NLM_fpage">1415</span>– <span class="NLM_lpage">1421</span>, <span class="refDoi"> DOI: 10.1126/science.8197455</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=10.1126%2Fscience.8197455" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=8197455" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=1%3ACAS%3A528%3ADyaK2cXktlalsL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=264&publication_year=1994&pages=1415-1421&author=J.+E.+Darnellauthor=I.+M.+Kerrauthor=G.+R.+Stark&title=Jak-STAT+pathways+and+transcriptional+activation+in+response+to+IFNs+and+other+extracellular+signaling+proteins&doi=10.1126%2Fscience.8197455"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Jak-STAT pathways and transcriptional activation in response to IFNs and other extracellular signaling proteins</span></div><div class="casAuthors">Darnell, James E., Jr.; Kerr, Ian M.; Stark, George R.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">264</span>
        (<span class="NLM_cas:issue">5164</span>),
    <span class="NLM_cas:pages">1415-21</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    </div><div class="casAbstract">A review with 97 refs.  Through the study of transcriptional activation in response to interferon α (IFN-α) and interferon γ (IFN-γ), a previously unrecognized direct signal transduction pathway to the nucleus has been uncovered: IFN-receptor interaction at the cell surface leads to the activation of kinases of the Jak family that then phosphorylate substrate proteins called STATs (signal transducers and activators of transcription).  The phosphorylated STAT proteins move to the nucleus, bind specific DNA elements, and direct transcription.  Recognition of the mols. involved in the IFN-α and IFN-γ pathway has led to discoveries that a no. of STAT family members exist and that other polypeptide ligands also use the Jak-STAT mols. in signal transduction.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo64rvN_qWlmLVg90H21EOLACvtfcHk0lgqpW_S7rkj9A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2cXktlalsL4%253D&md5=f63d2c7bb3c1aeca6f36fef38aff713c</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1126%2Fscience.8197455&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.8197455%26sid%3Dliteratum%253Aachs%26aulast%3DDarnell%26aufirst%3DJ.%2BE.%26aulast%3DKerr%26aufirst%3DI.%2BM.%26aulast%3DStark%26aufirst%3DG.%2BR.%26atitle%3DJak-STAT%2520pathways%2520and%2520transcriptional%2520activation%2520in%2520response%2520to%2520IFNs%2520and%2520other%2520extracellular%2520signaling%2520proteins%26jtitle%3DScience%26date%3D1994%26volume%3D264%26spage%3D1415%26epage%3D1421%26doi%3D10.1126%2Fscience.8197455" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bromberg, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Horvath, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Besser, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lathem, W. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Darnell, J. E.,  Jr.</span></span> <span> </span><span class="NLM_article-title">Stat3 activation is required for cellular transformation by v-src</span>. <i>Mol. Cell. Biol.</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">2553</span>– <span class="NLM_lpage">2558</span>, <span class="refDoi"> DOI: 10.1128/MCB.18.5.2553</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=10.1128%2FMCB.18.5.2553" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=9566875" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=1%3ACAS%3A528%3ADyaK1cXis1Gntb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=1998&pages=2553-2558&author=J.+F.+Brombergauthor=C.+M.+Horvathauthor=D.+Besserauthor=W.+W.+Lathemauthor=J.+E.+Darnell&title=Stat3+activation+is+required+for+cellular+transformation+by+v-src&doi=10.1128%2FMCB.18.5.2553"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Stat3 activation is required for cellular transformation by v-src</span></div><div class="casAuthors">Bromberg, Jacqueline F.; Horvath, Curt M.; Besser, Daniel; Lathem, Wyndham W.; Darnell, James E., Jr.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">2553-2558</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">0270-7306</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Stat3 activation has been assocd. with cytokine-induced proliferation, anti-apoptosis, and transformation.  Constitutively activated Stat3 has been found in many human tumors as well as v-abl- and v-src-transformed cell lines.  Because of these correlations, the authors examd. directly the relation of activated Stat3 to cellular transformation and found that wild-type Stat3 enhances the transforming potential of v-src while three dominant neg. Stat3 mutants inhibit v-src transformation.  Stat3 wild-type or mutant proteins did not affect v-ras transformation.  The authors conclude that Stat3 has a necessary role in v-src transformation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoT_4zAEjnfeLVg90H21EOLACvtfcHk0lgqpW_S7rkj9A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXis1Gntb4%253D&md5=28a74826798a6e041e2c2eff0f2300ec</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1128%2FMCB.18.5.2553&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.18.5.2553%26sid%3Dliteratum%253Aachs%26aulast%3DBromberg%26aufirst%3DJ.%2BF.%26aulast%3DHorvath%26aufirst%3DC.%2BM.%26aulast%3DBesser%26aufirst%3DD.%26aulast%3DLathem%26aufirst%3DW.%2BW.%26aulast%3DDarnell%26aufirst%3DJ.%2BE.%26atitle%3DStat3%2520activation%2520is%2520required%2520for%2520cellular%2520transformation%2520by%2520v-src%26jtitle%3DMol.%2520Cell.%2520Biol.%26date%3D1998%26volume%3D18%26spage%3D2553%26epage%3D2558%26doi%3D10.1128%2FMCB.18.5.2553" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Darnell, J. E.</span></span> <span> </span><span class="NLM_article-title">Validating Stat3 in cancer therapy</span>. <i>Nat. Med.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">595</span>– <span class="NLM_lpage">596</span>, <span class="refDoi"> DOI: 10.1038/nm0605-595</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=10.1038%2Fnm0605-595" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=15937466" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=1%3ACAS%3A528%3ADC%252BD2MXkslOlu74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2005&pages=595-596&author=J.+E.+Darnell&title=Validating+Stat3+in+cancer+therapy&doi=10.1038%2Fnm0605-595"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Validating Stat3 in cancer therapy</span></div><div class="casAuthors">Darnell, James E.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Medicine (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">595-596</span>CODEN:
                <span class="NLM_cas:coden">NAMEFI</span>;
        ISSN:<span class="NLM_cas:issn">1078-8956</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrRlI633_ngRbVg90H21EOLACvtfcHk0lgqpW_S7rkj9A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXkslOlu74%253D&md5=ddf70bdcb2c812f1e42b0f772f3d6be4</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1038%2Fnm0605-595&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnm0605-595%26sid%3Dliteratum%253Aachs%26aulast%3DDarnell%26aufirst%3DJ.%2BE.%26atitle%3DValidating%2520Stat3%2520in%2520cancer%2520therapy%26jtitle%3DNat.%2520Med.%26date%3D2005%26volume%3D11%26spage%3D595%26epage%3D596%26doi%3D10.1038%2Fnm0605-595" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Turkson, J.</span></span> <span> </span><span class="NLM_article-title">STAT proteins as novel targets for cancer drug discovery</span>. <i>Expert Opin. Ther. Targets</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">409</span>– <span class="NLM_lpage">422</span>, <span class="refDoi"> DOI: 10.1517/14728222.8.5.409</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=10.1517%2F14728222.8.5.409" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=15469392" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=1%3ACAS%3A528%3ADC%252BD2cXotFKhtb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2004&pages=409-422&author=J.+Turkson&title=STAT+proteins+as+novel+targets+for+cancer+drug+discovery&doi=10.1517%2F14728222.8.5.409"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">STAT proteins as novel targets for cancer drug discovery</span></div><div class="casAuthors">Turkson, James</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Therapeutic Targets</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">409-422</span>CODEN:
                <span class="NLM_cas:coden">EOTTAO</span>;
        ISSN:<span class="NLM_cas:issn">1472-8222</span>.
    
            (<span class="NLM_cas:orgname">Ashley Publications Ltd.</span>)
        </div><div class="casAbstract">A review with refs.  Signal transducer and activator of transcription (STAT) proteins are latent cytoplasmic transcription factors that were discovered in the context of cytokine and growth factor signalling.  Normal STAT signalling is tightly controlled with finite kinetics, which is in keeping with std. cellular responses.  However, persistent STAT activation has also been obsd. and is frequently assocd. with malignant transformation.  Constitutive activation of STAT proteins, notably of Stat3 and Stat5, is detected in many human tumor cells and cells transformed by oncoproteins that activate tyrosine kinase signalling pathways.  It is well-established that constitutively active Stat3 is one of the mol. abnormalities that has a causal role in oncogenesis.  Aberrant Stat3 promotes uncontrolled growth and survival through dys-regulation of gene expression, including cyclin D1, c-Myc, Bcl-xL, Mcl-1 and survivin genes, and thereby contributes to oncogenesis.  Moreover, recent studies reveal that persistently active Stat3 induces tumor angiogenesis by upregulation of vascular endothelial growth factor induction, and modulates immune functions in favor of tumor immune evasion.  Overall, studies have validated Stat3 as a novel target for cancer therapy, and hence provided the rationale for developing small-mol. Stat3 inhibitors.  This review will discuss current evidence for the crit. role of aberrant STAT signalling in malignant transformation, and examine the validity as well as the therapeutic potential of Stat3 as a cancer target.  An update on the efforts to develop novel Stat3 inhibitors for therapeutic application will also be provided.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpKUj4aPVco_rVg90H21EOLACvtfcHk0lgef7IXqqCrdw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXotFKhtb0%253D&md5=c524b36db31f3de66430595fc55fbe7c</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1517%2F14728222.8.5.409&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1517%252F14728222.8.5.409%26sid%3Dliteratum%253Aachs%26aulast%3DTurkson%26aufirst%3DJ.%26atitle%3DSTAT%2520proteins%2520as%2520novel%2520targets%2520for%2520cancer%2520drug%2520discovery%26jtitle%3DExpert%2520Opin.%2520Ther.%2520Targets%26date%3D2004%26volume%3D8%26spage%3D409%26epage%3D422%26doi%3D10.1517%2F14728222.8.5.409" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Turkson, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jove, R.</span></span> <span> </span><span class="NLM_article-title">STAT proteins: novel molecular targets for cancer drug discovery</span>. <i>Oncogene</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">6613</span>– <span class="NLM_lpage">6626</span>, <span class="refDoi"> DOI: 10.1038/sj.onc.1204086</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=10.1038%2Fsj.onc.1204086" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=11426647" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=1%3ACAS%3A528%3ADC%252BD3MXhsF2rsro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2000&pages=6613-6626&author=J.+Turksonauthor=R.+Jove&title=STAT+proteins%3A+novel+molecular+targets+for+cancer+drug+discovery&doi=10.1038%2Fsj.onc.1204086"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">STAT proteins: novel molecular targets for cancer drug discovery</span></div><div class="casAuthors">Turkson, James; Jove, Richard</div><div class="citationInfo"><span class="NLM_cas:title">Oncogene</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">56</span>),
    <span class="NLM_cas:pages">6613-6626</span>CODEN:
                <span class="NLM_cas:coden">ONCNES</span>;
        ISSN:<span class="NLM_cas:issn">0950-9232</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review with 101 refs.  Signal Transducers and Activators of Transcription (STATs) are a family of cytoplasmic proteins with roles as signal messengers and transcription factors that participate in normal cellular responses to cytokines and growth factors.  Frequently, however, abnormal activity of certain STAT family members, particularly Stat3 and Stat5, is assocd. with a wide variety of human malignancies, including hematol., breast, head and neck, and prostate cancers.  Application of mol. biol. and pharmacol. tools in disease-relevant models has confirmed Stat3 as having a causal role in oncogenesis, and provided validation of Stat3 as a target for cancer drug discovery and therapeutic intervention.  Furthermore, a constitutively-active mutant form of Stat3 is sufficient to induce oncogenic transformation of cells, which form tumors in vivo.  Constitutive activation of Stat3 signaling is accompanied by upregulation of cyclin D1, c-Myc, and Bcl-x, changes consistent with subversion of normal cellular growth and survival control mechanisms.  Block of constitutive Stat3 signaling results in growth inhibition and apoptosis of Stat3-pos. tumor cells in vitro and in vivo.  The obsd. dependence of certain tumors on constitutive Stat3 signaling for growth and survival has wide implications for cancer therapy, offering the potential for preferential tumor cell killing.  This review evaluates constitutive Stat3 activation as a "cancer-causing" factor, and proposes a no. of mol. strategies for targeting Stat3 signaling for therapeutic intervention.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoffTICt_aLhLVg90H21EOLACvtfcHk0lgef7IXqqCrdw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXhsF2rsro%253D&md5=2011fb621ef70547ddbbef2ed253df98</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1038%2Fsj.onc.1204086&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.onc.1204086%26sid%3Dliteratum%253Aachs%26aulast%3DTurkson%26aufirst%3DJ.%26aulast%3DJove%26aufirst%3DR.%26atitle%3DSTAT%2520proteins%253A%2520novel%2520molecular%2520targets%2520for%2520cancer%2520drug%2520discovery%26jtitle%3DOncogene%26date%3D2000%26volume%3D19%26spage%3D6613%26epage%3D6626%26doi%3D10.1038%2Fsj.onc.1204086" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Darnell, J. E.,  Jr.</span></span> <span> </span><span class="NLM_article-title">Transcription factors as targets for cancer therapy</span>. <i>Nat. Rev. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">740</span>– <span class="NLM_lpage">749</span>, <span class="refDoi"> DOI: 10.1038/nrc906</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=10.1038%2Fnrc906" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=12360277" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=1%3ACAS%3A528%3ADC%252BD38XnsVGrtbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2002&pages=740-749&author=J.+E.+Darnell&title=Transcription+factors+as+targets+for+cancer+therapy&doi=10.1038%2Fnrc906"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Transcription factors as targets for cancer therapy</span></div><div class="casAuthors">Darnell, James E., Jr.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">740-749</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  A limited list of transcription factors are overactive in most human cancer cells, which makes them targets for the development of anticancer drugs.  That they are the most direct and hopeful targets for treating cancer is proposed, and this is supported by the fact that there are many more human oncogenes in signaling pathways than there are oncogenic transcription factors.  But how could specific transcription-factor activity be inhibited.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrJxyqucgv0aLVg90H21EOLACvtfcHk0lgef7IXqqCrdw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XnsVGrtbo%253D&md5=239132f634b138965e3a11c63f7cab76</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1038%2Fnrc906&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc906%26sid%3Dliteratum%253Aachs%26aulast%3DDarnell%26aufirst%3DJ.%2BE.%26atitle%3DTranscription%2520factors%2520as%2520targets%2520for%2520cancer%2520therapy%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2002%26volume%3D2%26spage%3D740%26epage%3D749%26doi%3D10.1038%2Fnrc906" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Turkson, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ryan, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haura, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laudano, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sebti, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamilton, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jove, R.</span></span> <span> </span><span class="NLM_article-title">Phosphotyrosyl peptides block Stat3-mediated DNA binding activity, gene regulation, and cell transformation</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>276</i></span>,  <span class="NLM_fpage">45443</span>– <span class="NLM_lpage">45455</span>, <span class="refDoi"> DOI: 10.1074/jbc.M107527200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=10.1074%2Fjbc.M107527200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=11579100" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=1%3ACAS%3A528%3ADC%252BD3MXovFent7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=276&publication_year=2001&pages=45443-45455&author=J.+Turksonauthor=D.+Ryanauthor=J.+S.+Kimauthor=Y.+Zhangauthor=Z.+Chenauthor=E.+Hauraauthor=A.+Laudanoauthor=S.+Sebtiauthor=A.+D.+Hamiltonauthor=R.+Jove&title=Phosphotyrosyl+peptides+block+Stat3-mediated+DNA+binding+activity%2C+gene+regulation%2C+and+cell+transformation&doi=10.1074%2Fjbc.M107527200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Phosphotyrosyl peptides block Stat3-mediated DNA binding activity, gene regulation, and cell transformation</span></div><div class="casAuthors">Turkson, James; Ryan, Declan; Kim, Joon S.; Zhang, Yi; Chen, Zhi; Haura, Eric; Laudano, Andy; Sebti, Said; Hamilton, Andrew D.; Jove, Richard</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">276</span>
        (<span class="NLM_cas:issue">48</span>),
    <span class="NLM_cas:pages">45443-45455</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Signal transducers and activators of transcription (STATs) comprise a family of cytoplasmic signaling proteins that participates in normal cellular responses to cytokines and growth factors.  Frequently, however, constitutive activation of certain STAT family members, particularly Stat3, has accompanied a wide variety of human malignancies.  To identify small mol. inhibitors of Stat3, we investigated the ability of the Stat3 SH2 domain-binding peptide, PY*LKTK (where Y* represents phosphotyrosine), to disrupt Stat3 activity in vitro.  The presence of PY*LKTK, but not PYLKTK or PFLKTK, in nuclear exts. results in significant redn. in the levels of DNA binding activities of Stat3, to a lesser extent of Stat1, and with no effect on that of Stat5.  Analyses of alanine scanning mutagenesis and deletion derivs. of PY*LKTK reveal that the Leu residue at the Y + 1 position and a substituent at the Y - 1 position (but not necessarily Pro) are essential for the disruption of active Stat3, thereby mapping the min. active sequence to the tripeptide, XY*L.  Studies involving bead-coupled PY*LKTK peptide demonstrate that this phosphopeptide directly complexes with Stat3 monomers in vitro, suggesting that PY*LKTK disrupts Stat3:Stat3 dimers.  As evidence for the functional importance of peptide-directed inhibition of Stat3, PY*LKTK-mts (mts, membrane translocating sequence) selectively inhibits constitutive and ligand-induced Stat3 activation in vivo.  Furthermore, PY*LKTK-mts suppresses transformation by the Src oncoprotein, which has been shown previously to require constitutive Stat3 activation.  Altogether, we have identified a minimal peptide that inhibits Stat3 signaling and provides the conceptual basis for use of this peptide as a lead for novel peptidomimetic drug design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo_r1dIkvcURrVg90H21EOLACvtfcHk0liAsBKgvfDudg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXovFent7g%253D&md5=1d39e24e12f8af82715fec8d3c466cbd</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M107527200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M107527200%26sid%3Dliteratum%253Aachs%26aulast%3DTurkson%26aufirst%3DJ.%26aulast%3DRyan%26aufirst%3DD.%26aulast%3DKim%26aufirst%3DJ.%2BS.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DHaura%26aufirst%3DE.%26aulast%3DLaudano%26aufirst%3DA.%26aulast%3DSebti%26aufirst%3DS.%26aulast%3DHamilton%26aufirst%3DA.%2BD.%26aulast%3DJove%26aufirst%3DR.%26atitle%3DPhosphotyrosyl%2520peptides%2520block%2520Stat3-mediated%2520DNA%2520binding%2520activity%252C%2520gene%2520regulation%252C%2520and%2520cell%2520transformation%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2001%26volume%3D276%26spage%3D45443%26epage%3D45455%26doi%3D10.1074%2Fjbc.M107527200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Turkson, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glenn, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haura, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sebti, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamilton, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jove, R.</span></span> <span> </span><span class="NLM_article-title">Novel peptidomimetic inhibitors of signal transducer and activator of transcription 3 dimerization and biological activity</span>. <i>Mol. Cancer Ther.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">261</span>– <span class="NLM_lpage">269</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=15026546" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=1%3ACAS%3A528%3ADC%252BD2cXitFCns74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2004&pages=261-269&author=J.+Turksonauthor=J.+S.+Kimauthor=S.+Zhangauthor=J.+Yuanauthor=M.+Huangauthor=M.+Glennauthor=E.+Hauraauthor=S.+Sebtiauthor=A.+D.+Hamiltonauthor=R.+Jove&title=Novel+peptidomimetic+inhibitors+of+signal+transducer+and+activator+of+transcription+3+dimerization+and+biological+activity"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Novel peptidomimetic inhibitors of signal transducer and activator of transcription 3 dimerization and biological activity</span></div><div class="casAuthors">Turkson, James; Kim, Joon S.; Zhang, Shumin; Yuan, Jing; Huang, Mei; Glenn, Matthew; Haura, Eric; Sebti, Said; Hamilton, Andrew D.; Jove, Richard</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer Therapeutics</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">261-269</span>CODEN:
                <span class="NLM_cas:coden">MCTOCF</span>;
        ISSN:<span class="NLM_cas:issn">1535-7163</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">The crit. role of signal transducer and activator of transcription 3 (Stat3) in the growth and survival of human tumor cells identifies it as a promising target for cancer drug discovery.  We previously identified a Stat3 SH2 domain-binding phosphopeptide, PY*LKTK, and its tripeptide derivs., PY*L and AY*L (where Y* represents phosphotyrosine), which inhibit Stat3 biochem. activity and biol. function.  Here, we report novel peptidomimetic compds. based on PY*L (or AY*L) with substitution of the Y-1 residue by benzyl, pyridyl, or pyrazinyl derivs. that are selective and greater than 5-fold more potent in disrupting Stat3 activity in vitro than lead tripeptides.  The biol. activities of these derivs. mirror that originally obsd. for peptides.  In this context, the representative peptidomimetic ISS 610 with 4-cyanobenzoate substitution inhibits constitutive Stat3 activity in Src-transformed mouse fibroblasts and human breast and lung carcinoma cells.  This effect is not evident with the non-phosphorylated counterpart, ISS 610NP, consistent with interaction of peptidomimetics with the SH2 domain of Stat3.  Moreover, ISS 610 induces cell growth inhibition and apoptosis of Src-transformed fibroblasts that contain persistently active Stat3.  We present the first report of a peptidomimetic approach to design of small-mol. inhibitors of Stat3 that are also among the first examples of disruptors of transcription factor dimerization with the potential for novel cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpoTmPO43TLyrVg90H21EOLACvtfcHk0liAsBKgvfDudg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXitFCns74%253D&md5=b56e7dec52b6107d7f98d913376bdd0d</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DTurkson%26aufirst%3DJ.%26aulast%3DKim%26aufirst%3DJ.%2BS.%26aulast%3DZhang%26aufirst%3DS.%26aulast%3DYuan%26aufirst%3DJ.%26aulast%3DHuang%26aufirst%3DM.%26aulast%3DGlenn%26aufirst%3DM.%26aulast%3DHaura%26aufirst%3DE.%26aulast%3DSebti%26aufirst%3DS.%26aulast%3DHamilton%26aufirst%3DA.%2BD.%26aulast%3DJove%26aufirst%3DR.%26atitle%3DNovel%2520peptidomimetic%2520inhibitors%2520of%2520signal%2520transducer%2520and%2520activator%2520of%2520transcription%25203%2520dimerization%2520and%2520biological%2520activity%26jtitle%3DMol.%2520Cancer%2520Ther.%26date%3D2004%26volume%3D3%26spage%3D261%26epage%3D269" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Siddiquee, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guida, W. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blaskovich, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greedy, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lawrence, H. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yip, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jove, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McLaughlin, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lawrence, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sebti, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turkson, J.</span></span> <span> </span><span class="NLM_article-title">Selective chemical probe inhibitor of Stat3, identified through structure-based virtual screening, induces antitumor activity</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>104</i></span>,  <span class="NLM_fpage">7391</span>– <span class="NLM_lpage">7396</span>, <span class="refDoi"> DOI: 10.1073/pnas.0609757104</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=10.1073%2Fpnas.0609757104" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=17463090" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=1%3ACAS%3A528%3ADC%252BD2sXlslWqsbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=104&publication_year=2007&pages=7391-7396&author=K.+Siddiqueeauthor=S.+Zhangauthor=W.+C.+Guidaauthor=M.+A.+Blaskovichauthor=B.+Greedyauthor=H.+R.+Lawrenceauthor=M.+L.+Yipauthor=R.+Joveauthor=M.+M.+McLaughlinauthor=N.+J.+Lawrenceauthor=S.+M.+Sebtiauthor=J.+Turkson&title=Selective+chemical+probe+inhibitor+of+Stat3%2C+identified+through+structure-based+virtual+screening%2C+induces+antitumor+activity&doi=10.1073%2Fpnas.0609757104"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Elective chemical probe inhibitor of Stat3, identified through structure-based virtual screening, induces antitumor activity</span></div><div class="casAuthors">Siddiquee, Khandaker; Zhang, Shumin; Guida, Wayne C.; Blaskovich, Michelle A.; Greddy, Benjamin; Lawrence, Harshani R.; Yip, M. L. Richard; Jove, Richard; McLaughlin, Mark M.; Lawrence, Nicholas J.; Sebti, Said M.; Turkson, James</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">104</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">7391-7396</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">531-201 (NSC 74859) is a chem. probe inhibitor of Stat3 activity, which was identified from the National Cancer Institute chem. libraries by using structure-based virtual screening with a computer model of the Stat3 SH2 domain bound to its Stat3 phosphotyrosine peptide derived from the x-ray crystal structure of the Stat3β homodimer.  S3I-201 inhibits Stat3 · Stat3 complex formation and Stat3 DNA-binding and transcriptional activities.  Furthermore, S3I-201 inhibits growth and induces apoptosis preferentially in tumor cells that contain persistently activated Stat3.  Constitutively dimerized and active Stat3C and Stat3 SH2 domain rescue tumor cells from S3I-201-induced apoptosis.  Finally, S3I-201 inhibits the expression of the Stat3-regulated genes encoding cyclin D1, Bcl-xL, and surviving and inhibits the growth of human breast tumors in vivo.  These findings strongly suggest that the antitumor activity of S3I-201 is mediated in part through inhibition of aberrant Stat3 activation and provide the proof-of-concept for the potential clin. use of Stat3 inhibitors such as S3I-201 in tumors harboring aberrant Stat3.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpAfmqUniFENbVg90H21EOLACvtfcHk0liAsBKgvfDudg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXlslWqsbY%253D&md5=85decf56fc7852990f56c6f64165ad9d</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0609757104&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0609757104%26sid%3Dliteratum%253Aachs%26aulast%3DSiddiquee%26aufirst%3DK.%26aulast%3DZhang%26aufirst%3DS.%26aulast%3DGuida%26aufirst%3DW.%2BC.%26aulast%3DBlaskovich%26aufirst%3DM.%2BA.%26aulast%3DGreedy%26aufirst%3DB.%26aulast%3DLawrence%26aufirst%3DH.%2BR.%26aulast%3DYip%26aufirst%3DM.%2BL.%26aulast%3DJove%26aufirst%3DR.%26aulast%3DMcLaughlin%26aufirst%3DM.%2BM.%26aulast%3DLawrence%26aufirst%3DN.%2BJ.%26aulast%3DSebti%26aufirst%3DS.%2BM.%26aulast%3DTurkson%26aufirst%3DJ.%26atitle%3DSelective%2520chemical%2520probe%2520inhibitor%2520of%2520Stat3%252C%2520identified%2520through%2520structure-based%2520virtual%2520screening%252C%2520induces%2520antitumor%2520activity%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2007%26volume%3D104%26spage%3D7391%26epage%3D7396%26doi%3D10.1073%2Fpnas.0609757104" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Siddiquee, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gunning, P. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glenn, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Katt, W. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schrock, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sebti, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jove, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamilton, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turkson, J.</span></span> <span> </span><span class="NLM_article-title">An oxazole-based small-molecule Stat3 inhibitor modulates Stat3 stability and processing and induces antitumor cell effects</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">787</span>– <span class="NLM_lpage">798</span>, <span class="refDoi"> DOI: 10.1021/cb7001973</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb7001973" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhsVCrs7jL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2007&pages=787-798&author=K.+A.+Siddiqueeauthor=P.+T.+Gunningauthor=M.+Glennauthor=W.+P.+Kattauthor=S.+Zhangauthor=C.+Schrockauthor=S.+M.+Sebtiauthor=R.+Joveauthor=A.+D.+Hamiltonauthor=J.+Turkson&title=An+oxazole-based+small-molecule+Stat3+inhibitor+modulates+Stat3+stability+and+processing+and+induces+antitumor+cell+effects&doi=10.1021%2Fcb7001973"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">An Oxazole-Based Small-Molecule Stat3 Inhibitor Modulates Stat3 Stability and Processing and Induces Antitumor Cell Effects</span></div><div class="casAuthors">Siddiquee, Khandaker A. Z.; Gunning, Patrick T.; Glenn, Matthew; Katt, William P.; Zhang, Shumin; Schroeck, Christopher; Sebti, Said M.; Jove, Richard; Hamilton, Andrew D.; Turkson, James</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">787-798</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Stat3 is hyperactivated in many human tumors and represents a valid target for anticancer drug design.  The authors present a novel small-mol. Stat3 inhibitor, S3I-M2001, and describe the dynamics of intracellular processing of activated Stat3 within the context of the biochem. and biol. effects of the Stat3 inhibitor.  S3I-M2001 is an oxazole-based peptidomimetic of the Stat3 Src homol. (SH) 2 domain-binding phosphotyrosine peptide that selectively disrupts active Stat3:Stat3 dimers.  Consequently, hyperactivated Stat3, which hitherto occurs as "dotlike" structures of nuclear bodies, undergoes an early aggregation into nonfunctional perinuclear aggresomes and a late-phase proteasome-mediated degrdn. in malignant cells treated with S3I-M2001.  Thus, S3I-M2001 inhibited Stat3-dependent transcriptional regulation of tumor survival genes, such as Bcl-xL.  Furthermore, Stat3-dependent malignant transformation, survival, and migration and invasion of mouse and human cancer cells harboring persistently activated Stat3 were inhibited by S3I-M2001.  Finally, S3I-M2001 inhibited growth of human breast tumor xenografts.  The study identifies a novel Stat3 inhibitor, S3I-M2001, with antitumor cell effects mediated in part through a biphasic loss of functional Stat3.  The study represents the first on intracellular Stat3 stability and processing following inhibition by a small mol. that has significant antitumor activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGojQxo7tV9euLVg90H21EOLACvtfcHk0lhI9lxkzVSNGw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhsVCrs7jL&md5=db079dcd7ca8233cc6e62596bb1672f5</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1021%2Fcb7001973&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb7001973%26sid%3Dliteratum%253Aachs%26aulast%3DSiddiquee%26aufirst%3DK.%2BA.%26aulast%3DGunning%26aufirst%3DP.%2BT.%26aulast%3DGlenn%26aufirst%3DM.%26aulast%3DKatt%26aufirst%3DW.%2BP.%26aulast%3DZhang%26aufirst%3DS.%26aulast%3DSchrock%26aufirst%3DC.%26aulast%3DSebti%26aufirst%3DS.%2BM.%26aulast%3DJove%26aufirst%3DR.%26aulast%3DHamilton%26aufirst%3DA.%2BD.%26aulast%3DTurkson%26aufirst%3DJ.%26atitle%3DAn%2520oxazole-based%2520small-molecule%2520Stat3%2520inhibitor%2520modulates%2520Stat3%2520stability%2520and%2520processing%2520and%2520induces%2520antitumor%2520cell%2520effects%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2007%26volume%3D2%26spage%3D787%26epage%3D798%26doi%3D10.1021%2Fcb7001973" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Song, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, J.</span></span> <span> </span><span class="NLM_article-title">A low-molecular-weight compound discovered through virtual database screening inhibits Stat3 function in breast cancer cells</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>102</i></span>,  <span class="NLM_fpage">4700</span>– <span class="NLM_lpage">4705</span>, <span class="refDoi"> DOI: 10.1073/pnas.0409894102</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=10.1073%2Fpnas.0409894102" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=15781862" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=1%3ACAS%3A528%3ADC%252BD2MXjt1Oiu78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=102&publication_year=2005&pages=4700-4705&author=H.+Songauthor=R.+Wangauthor=S.+Wangauthor=J.+Lin&title=A+low-molecular-weight+compound+discovered+through+virtual+database+screening+inhibits+Stat3+function+in+breast+cancer+cells&doi=10.1073%2Fpnas.0409894102"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">A low-molecular-weight compound discovered through virtual database screening inhibits stat3 function in breast cancer cells</span></div><div class="casAuthors">Song, Hui; Wang, Renxiao; Wang, Shaomeng; Lin, Jiayuh</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">102</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">4700-4705</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">This study focused on the screening of small-mol. inhibitors that target signal transducers and activators of transcription 3 (Stat3) in human breast carcinoma.  The constitutive activation of Stat3 is frequently detected in human breast cancer cell lines as well as clin. breast cancer specimens and may play an important role in the oncogenesis of breast carcinoma.  Activated Stat3 may participate in oncogenesis by stimulating cell proliferation, promoting tumor angiogenesis, and resisting apoptosis.  Because a variety of human cancers are assocd. with constitutively active Stat3, Stat3 represents an attractive target for cancer therapy.  In this study, of the nearly 429,000 compds. screened by virtual database screening, chem. samples of top 100 compds. identified as candidate small-mol. inhibitors of Stat3 were evaluated by using Stat3-dependent luciferase reporter as well as other cell-based assays.  Through serial functional evaluation based on our established cell-based assays, one compd., termed STA-21, was identified as the best match for our selection criteria.  Further investigation demonstrated that STA-21 inhibits Stat3 DNA binding activity, Stat3 dimerization, and Stat3-dependent luciferase activity.  Moreover, STA-21 reduces the survival of breast carcinoma cells with constitutive Stat3 signaling but has minimal effect on the cells in which constitutive Stat3 signaling is absent.  Together, these results demonstrate that STA-21 inhibits breast cancer cells that express constitutively active Stat3.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGriWe9nsAbh67Vg90H21EOLACvtfcHk0lhI9lxkzVSNGw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXjt1Oiu78%253D&md5=fe7108bee9cddbcdd75a67dd93e19e63</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0409894102&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0409894102%26sid%3Dliteratum%253Aachs%26aulast%3DSong%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DR.%26aulast%3DWang%26aufirst%3DS.%26aulast%3DLin%26aufirst%3DJ.%26atitle%3DA%2520low-molecular-weight%2520compound%2520discovered%2520through%2520virtual%2520database%2520screening%2520inhibits%2520Stat3%2520function%2520in%2520breast%2520cancer%2520cells%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2005%26volume%3D102%26spage%3D4700%26epage%3D4705%26doi%3D10.1073%2Fpnas.0409894102" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McMurray, J. S.</span></span> <span> </span><span class="NLM_article-title">A new small-molecule Stat3 inhibitor</span>. <i>Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">1123</span>– <span class="NLM_lpage">1124</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2006.11.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=10.1016%2Fj.chembiol.2006.11.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=17113993" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=1%3ACAS%3A528%3ADC%252BD28Xht1Cis7bM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2006&pages=1123-1124&author=J.+S.+McMurray&title=A+new+small-molecule+Stat3+inhibitor&doi=10.1016%2Fj.chembiol.2006.11.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">A New Small-Molecule Stat3 Inhibitor</span></div><div class="casAuthors">McMurray, John S.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1123-1124</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">In this issue of Chem. & Biol., Schust et al. report the discovery of a small mol. (Stattic) that inhibits the binding of a high affinity phosphopeptide for the SH2 domain of Stat3.  Stattic is a new tool for studying Stat3 signaling and demonstrates that the SH2 domain is not a dead target.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpznjm1p_Pt8LVg90H21EOLACvtfcHk0lhI9lxkzVSNGw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xht1Cis7bM&md5=c7744bbffb93abfd6a51f838a49b74f0</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2006.11.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2006.11.001%26sid%3Dliteratum%253Aachs%26aulast%3DMcMurray%26aufirst%3DJ.%2BS.%26atitle%3DA%2520new%2520small-molecule%2520Stat3%2520inhibitor%26jtitle%3DChem.%2520Biol.%26date%3D2006%26volume%3D13%26spage%3D1123%26epage%3D1124%26doi%3D10.1016%2Fj.chembiol.2006.11.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ren, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cabell, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schaefer, T. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McMurray, J. S.</span></span> <span> </span><span class="NLM_article-title">Identification of a high-affinity phosphopeptide inhibitor of Stat3</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">633</span>– <span class="NLM_lpage">636</span>, <span class="refDoi"> DOI: 10.1016/S0960-894X(02)01050-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=10.1016%2FS0960-894X%2802%2901050-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=12639546" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=1%3ACAS%3A528%3ADC%252BD3sXhvV2gtLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2003&pages=633-636&author=Z.+Renauthor=L.+A.+Cabellauthor=T.+S.+Schaeferauthor=J.+S.+McMurray&title=Identification+of+a+high-affinity+phosphopeptide+inhibitor+of+Stat3&doi=10.1016%2FS0960-894X%2802%2901050-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of a High-Affinity Phosphopeptide Inhibitor of Stat3</span></div><div class="casAuthors">Ren, Zhiyong; Cabell, Larry A.; Schaefer, Timothy S.; McMurray, John S.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">633-636</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ltd.</span>)
        </div><div class="casAbstract">Stat3 is a latent transcription factor that exhibits elevated activity in a variety of human cancers.  To find a lead peptide for peptidomimetic drug development we synthesized and tested phosphopeptides derived from known receptor docking sites and found Y(p)LPQTV as the optimal sequence.  SAR studies showed that each residue from pY to pY+3 provided binding energy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpKMeS5w7beErVg90H21EOLACvtfcHk0lhI9lxkzVSNGw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXhvV2gtLc%253D&md5=8798d979feeb7f5d8073ce3d43cebaf0</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1016%2FS0960-894X%2802%2901050-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0960-894X%252802%252901050-8%26sid%3Dliteratum%253Aachs%26aulast%3DRen%26aufirst%3DZ.%26aulast%3DCabell%26aufirst%3DL.%2BA.%26aulast%3DSchaefer%26aufirst%3DT.%2BS.%26aulast%3DMcMurray%26aufirst%3DJ.%2BS.%26atitle%3DIdentification%2520of%2520a%2520high-affinity%2520phosphopeptide%2520inhibitor%2520of%2520Stat3%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2003%26volume%3D13%26spage%3D633%26epage%3D636%26doi%3D10.1016%2FS0960-894X%2802%2901050-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McMurray, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mandal, P. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liao, W. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ren, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span> <span> </span><span class="NLM_article-title">Inhibition of Stat3 by cell-permeable peptidomimetic prodrugs targeted to its SH2 domain</span>. <i>Adv. Exp Med. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>611</i></span>,  <span class="NLM_fpage">545</span>– <span class="NLM_lpage">546</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=10.1007%2F978-0-387-73657-0_239" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=19400306" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtlWrsLjL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=611&publication_year=2009&pages=545-546&author=J.+S.+McMurrayauthor=P.+K.+Mandalauthor=W.+S.+Liaoauthor=Z.+Renauthor=X.+Chen&title=Inhibition+of+Stat3+by+cell-permeable+peptidomimetic+prodrugs+targeted+to+its+SH2+domain"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of Stat3 by cell-permeable peptidomimetic prodrugs targeted to its SH2 domain</span></div><div class="casAuthors">McMurray, John S.; Mandal, Pijus K.; Liao, Warren S.; Ren, Zhiyong; Chen, Xiaomin</div><div class="citationInfo"><span class="NLM_cas:title">Advances in Experimental Medicine and Biology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">611</span>
        (<span class="NLM_cas:issue">Peptides for Youth</span>),
    <span class="NLM_cas:pages">545-546</span>CODEN:
                <span class="NLM_cas:coden">AEMBAP</span>;
        ISSN:<span class="NLM_cas:issn">0065-2598</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">This study performed structure-affinity relationship anal. on truncated 2 to det. the pharmacophores and to gain information on the conformation in the bound state.  The Stat3 SH2 domain was targeted to create new anti-cancer chemotherapeutic agents.  A phosphohexapeptide was generally converted into cell-permeable, phosphatase stable peptidomimetics that inhibit their target in the low μM range.  Further studies are in progress to optimize delivery and potency of these compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqQJCPYE_gu4bVg90H21EOLACvtfcHk0lgKEKh__AKS7g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtlWrsLjL&md5=2187a32637e930f278b3e950152c5e1d</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1007%2F978-0-387-73657-0_239&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252F978-0-387-73657-0_239%26sid%3Dliteratum%253Aachs%26aulast%3DMcMurray%26aufirst%3DJ.%2BS.%26aulast%3DMandal%26aufirst%3DP.%2BK.%26aulast%3DLiao%26aufirst%3DW.%2BS.%26aulast%3DRen%26aufirst%3DZ.%26aulast%3DChen%26aufirst%3DX.%26atitle%3DInhibition%2520of%2520Stat3%2520by%2520cell-permeable%2520peptidomimetic%2520prodrugs%2520targeted%2520to%2520its%2520SH2%2520domain%26jtitle%3DAdv.%2520Exp%2520Med.%2520Biol.%26date%3D2009%26volume%3D611%26spage%3D545%26epage%3D546" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mandal, P. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liao, W. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McMurray, J. S.</span></span> <span> </span><span class="NLM_article-title">Synthesis of phosphatase-stable, cell-permeable peptidomimetic prodrugs that target the SH2 domain of Stat3</span>. <i>Org. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">3394</span>– <span class="NLM_lpage">3397</span>, <span class="refDoi"> DOI: 10.1021/ol9012662</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ol9012662" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=1%3ACAS%3A528%3ADC%252BD1MXotl2ht78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2009&pages=3394-3397&author=P.+K.+Mandalauthor=W.+S.+Liaoauthor=J.+S.+McMurray&title=Synthesis+of+phosphatase-stable%2C+cell-permeable+peptidomimetic+prodrugs+that+target+the+SH2+domain+of+Stat3&doi=10.1021%2Fol9012662"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis of phosphatase-stable, cell-permeable peptidomimetic prodrugs that target the SH2 domain of Stat3</span></div><div class="casAuthors">Mandal, Pijus K.; Liao, Warren S.-L.; McMurray, John S.</div><div class="citationInfo"><span class="NLM_cas:title">Organic Letters</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">3394-3397</span>CODEN:
                <span class="NLM_cas:coden">ORLEF7</span>;
        ISSN:<span class="NLM_cas:issn">1523-7060</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The synthesis of prodrugs targeted to the SH2 domain of Stat3 is reported.  Using a convergent strategy, the pivaloyloxymethyl phosphonodiester of pentachlorophenyl 4-phosphonodifluoromethylcinnamate, a phosphotyrosine surrogate, was synthesized and used to acylate peptidomimetic fragments that were prepd. on solid supports.  Two prodrugs described here inhibited the phosphorylation of Stat3 in breast tumor cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqokZ-epNTxprVg90H21EOLACvtfcHk0lgKEKh__AKS7g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXotl2ht78%253D&md5=c935049f107f5ab89bf38cd445c845f5</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1021%2Fol9012662&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fol9012662%26sid%3Dliteratum%253Aachs%26aulast%3DMandal%26aufirst%3DP.%2BK.%26aulast%3DLiao%26aufirst%3DW.%2BS.%26aulast%3DMcMurray%26aufirst%3DJ.%2BS.%26atitle%3DSynthesis%2520of%2520phosphatase-stable%252C%2520cell-permeable%2520peptidomimetic%2520prodrugs%2520that%2520target%2520the%2520SH2%2520domain%2520of%2520Stat3%26jtitle%3DOrg.%2520Lett.%26date%3D2009%26volume%3D11%26spage%3D3394%26epage%3D3397%26doi%3D10.1021%2Fol9012662" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bai, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernard, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nikolovska-Coleska, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gomez, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span> <span> </span><span class="NLM_article-title">Structure-based design of conformationally constrained, cell-permeable Stat3 inhibitors</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">85</span>– <span class="NLM_lpage">89</span>, <span class="refDoi"> DOI: 10.1021/ml100010j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ml100010j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=1%3ACAS%3A528%3ADC%252BC3cXivFGiurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2010&pages=85-89&author=J.+Chenauthor=L.+Baiauthor=D.+Bernardauthor=Z.+Nikolovska-Coleskaauthor=C.+Gomezauthor=J.+Zhangauthor=H.+Yiauthor=S.+Wang&title=Structure-based+design+of+conformationally+constrained%2C+cell-permeable+Stat3+inhibitors&doi=10.1021%2Fml100010j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Based Design of Conformationally Constrained, Cell-Permeable STAT3 Inhibitors</span></div><div class="casAuthors">Chen, Jianyong; Bai, Longchuan; Bernard, Denzil; Nikolovska-Coleska, Zaneta; Gomez, Cindy; Zhang, Jian; Yi, Han; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">85-89</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We report herein the structure-based design of a class of conformationally constrained, potent, cell-permeable small-mol. inhibitors to target the SH2 domain in STAT3.  Compd. 11 (CJ-1383) binds to STAT3 with a Ki value of 0.95 μM, dose-dependently inhibits cellular STAT3 signaling and cancer cell growth, and induces apoptosis in the MDA-MB-468 cancer cell line with constitutively activated STAT3.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqMrK-vDy7ISbVg90H21EOLACvtfcHk0lgKEKh__AKS7g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXivFGiurs%253D&md5=87a0b1a823441940f75fc2af960e0359</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Fml100010j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fml100010j%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DBai%26aufirst%3DL.%26aulast%3DBernard%26aufirst%3DD.%26aulast%3DNikolovska-Coleska%26aufirst%3DZ.%26aulast%3DGomez%26aufirst%3DC.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DYi%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DStructure-based%2520design%2520of%2520conformationally%2520constrained%252C%2520cell-permeable%2520Stat3%2520inhibitors%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2010%26volume%3D1%26spage%3D85%26epage%3D89%26doi%3D10.1021%2Fml100010j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nikolovska-Coleska, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gomez, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krajewski, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roller, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span> <span> </span><span class="NLM_article-title">Design and synthesis of a new, conformationally constrained, macrocyclic small-molecule inhibitor of Stat3 via ‘click chemistry’</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">3939</span>– <span class="NLM_lpage">3942</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2007.04.096</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=10.1016%2Fj.bmcl.2007.04.096" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=17513110" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=1%3ACAS%3A528%3ADC%252BD2sXmvVClurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2007&pages=3939-3942&author=J.+Chenauthor=Z.+Nikolovska-Coleskaauthor=C.-Y.+Yangauthor=C.+Gomezauthor=W.+Gaoauthor=K.+Krajewskiauthor=S.+Jiangauthor=P.+Rollerauthor=S.+Wang&title=Design+and+synthesis+of+a+new%2C+conformationally+constrained%2C+macrocyclic+small-molecule+inhibitor+of+Stat3+via+%E2%80%98click+chemistry%E2%80%99&doi=10.1016%2Fj.bmcl.2007.04.096"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Design and synthesis of a new, conformationally constrained, macrocyclic small-molecule inhibitor of STAT3 via 'click chemistry'</span></div><div class="casAuthors">Chen, Jianyong; Nikolovska-Coleska, Zaneta; Yang, Chao-Yie; Gomez, Cindy; Gao, Wei; Krajewski, Krzysztof; Jiang, Sheng; Roller, Peter; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">3939-3942</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">STAT3 is a promising mol. target for the design of anticancer drugs.  In this paper, we report the design and synthesis of a conformationally constrained macrocyclic peptidomimetic I via click chem.  Compd. I was detd. to bind to STAT3 with a Ki value of 7.3 μM in a competitive fluorescence-polarization-based binding assay, representing a promising initial lead compd. for further optimization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoOR6JsWbNxsbVg90H21EOLACvtfcHk0lguVNIINPTnUA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXmvVClurc%253D&md5=93e403fbff688e18539712c00f39a94e</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2007.04.096&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2007.04.096%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DNikolovska-Coleska%26aufirst%3DZ.%26aulast%3DYang%26aufirst%3DC.-Y.%26aulast%3DGomez%26aufirst%3DC.%26aulast%3DGao%26aufirst%3DW.%26aulast%3DKrajewski%26aufirst%3DK.%26aulast%3DJiang%26aufirst%3DS.%26aulast%3DRoller%26aufirst%3DP.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DDesign%2520and%2520synthesis%2520of%2520a%2520new%252C%2520conformationally%2520constrained%252C%2520macrocyclic%2520small-molecule%2520inhibitor%2520of%2520Stat3%2520via%2520%25E2%2580%2598click%2520chemistry%25E2%2580%2599%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2007%26volume%3D17%26spage%3D3939%26epage%3D3942%26doi%3D10.1016%2Fj.bmcl.2007.04.096" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Leung, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhong, H. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Susanti, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rao, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Che, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leung, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, P. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, D. L.</span></span> <span> </span><span class="NLM_article-title">Discovery of a small-molecule inhibitor of Stat3 by ligand-based pharmacophore screening</span>. <i>Methods</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>71</i></span>,  <span class="NLM_fpage">38</span>– <span class="NLM_lpage">43</span>, <span class="refDoi"> DOI: 10.1016/j.ymeth.2014.07.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=10.1016%2Fj.ymeth.2014.07.010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=25160651" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsVKnur7P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=71&publication_year=2015&pages=38-43&author=K.+H.+Leungauthor=L.+J.+Liuauthor=S.+Linauthor=L.+Luauthor=H.+J.+Zhongauthor=D.+Susantiauthor=W.+Raoauthor=M.+Wangauthor=W.+Cheauthor=D.+S.+Chanauthor=C.+H.+Leungauthor=P.+W.+Chanauthor=D.+L.+Ma&title=Discovery+of+a+small-molecule+inhibitor+of+Stat3+by+ligand-based+pharmacophore+screening&doi=10.1016%2Fj.ymeth.2014.07.010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a small-molecule inhibitor of STAT3 by ligand-based pharmacophore screening</span></div><div class="casAuthors">Leung, Ka-Ho; Liu, Li-Juan; Lin, Sheng; Lu, Lihua; Zhong, Hai-Jing; Susanti, Dewi; Rao, Weidong; Wang, Modi; Che, Weng Ian; Chan, Daniel Shiu-Hin; Leung, Chung-Hang; Chan, Philip Wai Hong; Ma, Dik-Lung</div><div class="citationInfo"><span class="NLM_cas:title">Methods (Amsterdam, Netherlands)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">71</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">38-43</span>CODEN:
                <span class="NLM_cas:coden">MTHDE9</span>;
        ISSN:<span class="NLM_cas:issn">1046-2023</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">STAT3 modulates the transcription of a wide variety of regulatory genes involved in cell proliferation, differentiation, migration, apoptosis, and other crit. cellular functions.  Constitutive activation of STAT3 has been detected in a wide spectrum of human malignancies.  A pharmacophore model constructed from a training set of STAT3 inhibitors binding to the SH2 domain was used to screen an inhouse database of compds., from which azepine 1 emerged as a top candidate.  Compd. 1 inhibited STAT3 DNA-binding activity in vitro and attenuated STAT3-directed transcription in cellulo with comparable potency to the well-known STAT3 inhibitor S3I-201.  A fluorescence polarization assay revealed that compd. 1 targeted the SH2 domain of STAT3.  Furthermore, compd. 1 inhibited STAT3 phosphorylation in cells without affecting the total expression of STAT3.  This study also validates the use of pharmacophore modeling to identify inhibitors of protein-protein interactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGphtVAbxIrmB7Vg90H21EOLACvtfcHk0lguVNIINPTnUA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsVKnur7P&md5=1a1d6a3ee10efa8f2965b7eaf43b3a4f</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1016%2Fj.ymeth.2014.07.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ymeth.2014.07.010%26sid%3Dliteratum%253Aachs%26aulast%3DLeung%26aufirst%3DK.%2BH.%26aulast%3DLiu%26aufirst%3DL.%2BJ.%26aulast%3DLin%26aufirst%3DS.%26aulast%3DLu%26aufirst%3DL.%26aulast%3DZhong%26aufirst%3DH.%2BJ.%26aulast%3DSusanti%26aufirst%3DD.%26aulast%3DRao%26aufirst%3DW.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DChe%26aufirst%3DW.%26aulast%3DChan%26aufirst%3DD.%2BS.%26aulast%3DLeung%26aufirst%3DC.%2BH.%26aulast%3DChan%26aufirst%3DP.%2BW.%26aulast%3DMa%26aufirst%3DD.%2BL.%26atitle%3DDiscovery%2520of%2520a%2520small-molecule%2520inhibitor%2520of%2520Stat3%2520by%2520ligand-based%2520pharmacophore%2520screening%26jtitle%3DMethods%26date%3D2015%26volume%3D71%26spage%3D38%26epage%3D43%26doi%3D10.1016%2Fj.ymeth.2014.07.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yue, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fletcher, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gunning, P. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turkson, J.</span></span> <span> </span><span class="NLM_article-title">A novel small-molecule disrupts Stat3 SH2 domain-phosphotyrosine interactions and Stat3-dependent tumor processes</span>. <i>Biochem. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>79</i></span>,  <span class="NLM_fpage">1398</span>– <span class="NLM_lpage">409</span>, <span class="refDoi"> DOI: 10.1016/j.bcp.2010.01.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=10.1016%2Fj.bcp.2010.01.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=20067773" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=1%3ACAS%3A528%3ADC%252BC3cXjtVCnsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=79&publication_year=2010&pages=1398-409&author=X.+Zhangauthor=P.+Yueauthor=S.+Fletcherauthor=W.+Zhaoauthor=P.+T.+Gunningauthor=J.+Turkson&title=A+novel+small-molecule+disrupts+Stat3+SH2+domain-phosphotyrosine+interactions+and+Stat3-dependent+tumor+processes&doi=10.1016%2Fj.bcp.2010.01.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">A novel small-molecule disrupts Stat3 SH2 domain-phosphotyrosine interactions and Stat3-dependent tumor processes</span></div><div class="casAuthors">Zhang, Xiaolei; Yue, Peibin; Fletcher, Steven; Zhao, Wei; Gunning, Patrick T.; Turkson, James</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical Pharmacology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">79</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1398-1409</span>CODEN:
                <span class="NLM_cas:coden">BCPCA6</span>;
        ISSN:<span class="NLM_cas:issn">0006-2952</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The mol. modeling of the phosphotyrosine (pTyr)-SH2 domain interaction in the Stat3:Stat3 dimerization, combined with in silico structural anal. of the Stat3 dimerization disruptor, S3I-201, has furnished a diverse set of analogs.  We present evidence from in vitro biochem. and biophys. studies that the structural analog, S3I-201.1066 (I) directly interacts with Stat3 or the SH2 domain, with an affinity (KD) of 2.74 μM, and disrupts the binding of Stat3 to the cognate pTyr-peptide, GpYLPQTV-NH2, with an IC50 of 23 μM.  Moreover, I selectively blocks the assocn. of Stat3 with the epidermal growth factor receptor (EGFR), and inhibits Stat3 tyrosine phosphorylation and nuclear translocation in EGF-stimulated mouse fibroblasts.  In cancer cells that harbor aberrant Stat3 activity, I inhibits constitutive Stat3 DNA-binding and transcriptional activities.  By contrast, I has no effect on Src activation or the EGFR-mediated activation of the Erk1/2MAPK pathway.  I selectively suppresses the viability, survival, and malignant transformation of the human breast and pancreatic cancer lines and the v-Src-transformed mouse fibroblasts harboring persistently active Stat3.  Treatment with I of malignant cells harboring aberrantly active Stat3 down-regulated the expression of c-Myc, Bcl-xL, Survivin, the matrix metalloproteinase 9, and VEGF.  The in vivo administration of I-induced significant antitumor response in mouse models of human breast cancer, which correlates with the inhibition of constitutively active Stat3 and the suppression of known Stat3-regulated genes.  Our studies identify a novel small-mol. that binds with a high affinity to Stat3, blocks Stat3 activation and function, and thereby induces antitumor response in human breast tumor xenografts harboring persistently active Stat3.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoxJzIoqCaQHbVg90H21EOLACvtfcHk0lgufABPCpKBkQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXjtVCnsL8%253D&md5=4a196103d1d0f6d057715ff4cd2d7498</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1016%2Fj.bcp.2010.01.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bcp.2010.01.001%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DYue%26aufirst%3DP.%26aulast%3DFletcher%26aufirst%3DS.%26aulast%3DZhao%26aufirst%3DW.%26aulast%3DGunning%26aufirst%3DP.%2BT.%26aulast%3DTurkson%26aufirst%3DJ.%26atitle%3DA%2520novel%2520small-molecule%2520disrupts%2520Stat3%2520SH2%2520domain-phosphotyrosine%2520interactions%2520and%2520Stat3-dependent%2520tumor%2520processes%26jtitle%3DBiochem.%2520Pharmacol.%26date%3D2010%26volume%3D79%26spage%3D1398%26epage%3D409%26doi%3D10.1016%2Fj.bcp.2010.01.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yue, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Page, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Namanja, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paladino, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gunning, P. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turkson, J.</span></span> <span> </span><span class="NLM_article-title">Orally bioavailable small-molecule inhibitor of transcription factor Stat3 regresses human breast and lung cancer xenografts</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>109</i></span>,  <span class="NLM_fpage">9623</span>– <span class="NLM_lpage">9628</span>, <span class="refDoi"> DOI: 10.1073/pnas.1121606109</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=10.1073%2Fpnas.1121606109" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=22623533" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=1%3ACAS%3A528%3ADC%252BC38XptlWjtLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=109&publication_year=2012&pages=9623-9628&author=X.+Zhangauthor=P.+Yueauthor=B.+D.+Pageauthor=T.+Liauthor=W.+Zhaoauthor=A.+T.+Namanjaauthor=D.+Paladinoauthor=J.+Zhaoauthor=Y.+Chenauthor=P.+T.+Gunningauthor=J.+Turkson&title=Orally+bioavailable+small-molecule+inhibitor+of+transcription+factor+Stat3+regresses+human+breast+and+lung+cancer+xenografts&doi=10.1073%2Fpnas.1121606109"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Orally bioavailable small-molecule inhibitor of transcription factor Stat3 regresses human breast and lung cancer xenografts</span></div><div class="casAuthors">Zhang, Xiaolei; Yue, Peibin; Page, Brent D. G.; Li, Tianshu; Zhao, Wei; Namanja, Andrew T.; Paladino, David; Zhao, Jihe; Chen, Yuan; Gunning, Patrick T.; Turkson, James</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">109</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">9623-9628, S9623/1-S9623/13</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Computer-aided lead optimization derives a unique, orally bioavailable inhibitor of the signal transducer and activator of transcription (Stat)3 Src homol. 2 domain.  BP-1-102 binds Stat3 with an affinity (KD) of 504 nM, blocks Stat3-phospho-tyrosine (pTyr) peptide interactions and Stat3 activation at 4-6.8 μM, and selectively inhibits growth, survival, migration, and invasion of Stat3-dependent tumor cells.  BP-1-102-mediated inhibition of aberrantly active Stat3 in tumor cells suppresses the expression of c-Myc, Cyclin D1, Bcl-xL, Survivin, VEGF, and Kruppel-like factor 8, which is identified as a Stat3 target gene that promotes Stat3-mediated breast tumor cell migration and invasion.  Treatment of breast cancer cells with BP-1-102 further blocks Stat3-NF-κB cross-talk, the release of granulocyte colony-stimulating factor, sol. intercellular adhesion mol. 1, macrophage migration-inhibitory factor/glycosylation-inhibiting factor, interleukin 1 receptor antagonist, and serine protease inhibitor protein 1, and the phosphorylation of focal adhesion kinase and paxillin, while enhancing E-cadherin expression.  I.v. or oral gavage delivery of BP-1-102 furnishes micromolar or microgram levels in tumor tissues and inhibits growth of human breast and lung tumor xenografts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrljEafjbznr7Vg90H21EOLACvtfcHk0lgufABPCpKBkQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XptlWjtLo%253D&md5=4da78174a7c00eacc1f1a1ce553c5aa7</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1121606109&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1121606109%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DYue%26aufirst%3DP.%26aulast%3DPage%26aufirst%3DB.%2BD.%26aulast%3DLi%26aufirst%3DT.%26aulast%3DZhao%26aufirst%3DW.%26aulast%3DNamanja%26aufirst%3DA.%2BT.%26aulast%3DPaladino%26aufirst%3DD.%26aulast%3DZhao%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DGunning%26aufirst%3DP.%2BT.%26aulast%3DTurkson%26aufirst%3DJ.%26atitle%3DOrally%2520bioavailable%2520small-molecule%2520inhibitor%2520of%2520transcription%2520factor%2520Stat3%2520regresses%2520human%2520breast%2520and%2520lung%2520cancer%2520xenografts%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2012%26volume%3D109%26spage%3D9623%26epage%3D9628%26doi%3D10.1073%2Fpnas.1121606109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pireddu, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Urlam, M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lawrence, H. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guida, W. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lawrence, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sebti, S. M.</span></span> <span> </span><span class="NLM_article-title">A novel inhibitor of Stat3 homodimerization selectively suppresses Stat3 activity and malignant transformation</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>73</i></span>,  <span class="NLM_fpage">1922</span>– <span class="NLM_lpage">1933</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.CAN-12-3175</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=10.1158%2F0008-5472.CAN-12-3175" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=23322008" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=1%3ACAS%3A528%3ADC%252BC3sXktVGisbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=73&publication_year=2013&pages=1922-1933&author=X.+Zhangauthor=Y.+Sunauthor=R.+Piredduauthor=H.+Yangauthor=M.+K.+Urlamauthor=H.+R.+Lawrenceauthor=W.+C.+Guidaauthor=N.+J.+Lawrenceauthor=S.+M.+Sebti&title=A+novel+inhibitor+of+Stat3+homodimerization+selectively+suppresses+Stat3+activity+and+malignant+transformation&doi=10.1158%2F0008-5472.CAN-12-3175"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">A Novel Inhibitor of STAT3 Homodimerization Selectively Suppresses STAT3 Activity and Malignant Transformation</span></div><div class="casAuthors">Zhang, Xiaolei; Sun, Ying; Pireddu, Roberta; Yang, Hua; Urlam, Murali K.; Lawrence, Harshani R.; Guida, Wayne C.; Lawrence, Nicholas J.; Sebti, Said M.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">73</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1922-1933</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">STAT3-STAT3 dimerization, which involves reciprocal binding of the STAT3-SH2 domain to phosphorylated. Tyrosine-705 (Y-705), is required for STAT3 nuclear translocation, DNA binding, and transcriptional regulation of downstream target genes.  Here, we describe a small mol. S3I-1757 capable of disrupting STAT3-STAT3 dimerization, activation, and malignant transforming activity.  Fluorescence polarization assay and mol. modeling suggest that S3I-1757 interacts with the phospho-Y-705-binding site in the SH2 domain and displaces fluorescein-labeled GpYLPQTV phosphotyrosine peptide from binding to STAT3.  We generated hemagglutinin (HA)-tagged STAT3 and FLAG-tagged STAT3 and showed using coimmunopptn. and colocalization studies that S3I-1757 inhibits STAT3 dimerization and STAT3-EGF receptor (EGFR) binding in intact cells.  Treatment of human cancer cells with S31-1757 (but not a closely related analog, S3I-1756, which does not inhibit STAT3 dimerization), inhibits selectively the phosphorylation of STAT3 over AKTl and ERK1/2 (MAPK3/1), nuclear accumulation of P-Y705-STAT3, STAT3-DNA binding, and transcriptional activation and suppresses the expression levels of STAT3 target genes, such as Bcl-xL (BCL2L1), survivin (BIRC5), cyclin D1 (CCND1), and matrix metalloproteinase (MMP)-9.  Furthermore, S3I-1757, but not S3I-1756, inhibits anchorage-dependent and -independent growth, migration, and invasion of human cancer cells, which depend on STAT3.  Finally, STAT3-C, a genetically engineered mutant of STAT3 that forms a constitutively dimerized STAT3, rescues cells from the effects of S3I-1757 inhibition.  Thus, we have developed S3I-1757 as a STAT3-STAT3 dimerization inhibitor capable of blocking hyperactivated STAT3 and suppressing malignant transformation in human cancer cells that depend on STAT3.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpLVvdmwj25LbVg90H21EOLACvtfcHk0lhvPN-hISa7bg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXktVGisbg%253D&md5=efc26953ace34e1bd3e32810113b7ef5</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-12-3175&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-12-3175%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DSun%26aufirst%3DY.%26aulast%3DPireddu%26aufirst%3DR.%26aulast%3DYang%26aufirst%3DH.%26aulast%3DUrlam%26aufirst%3DM.%2BK.%26aulast%3DLawrence%26aufirst%3DH.%2BR.%26aulast%3DGuida%26aufirst%3DW.%2BC.%26aulast%3DLawrence%26aufirst%3DN.%2BJ.%26aulast%3DSebti%26aufirst%3DS.%2BM.%26atitle%3DA%2520novel%2520inhibitor%2520of%2520Stat3%2520homodimerization%2520selectively%2520suppresses%2520Stat3%2520activity%2520and%2520malignant%2520transformation%26jtitle%3DCancer%2520Res.%26date%3D2013%26volume%3D73%26spage%3D1922%26epage%3D1933%26doi%3D10.1158%2F0008-5472.CAN-12-3175" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yue, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lopez-Tapia, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paladino, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, C.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hilliard, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tius, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turkson, J.</span></span> <span> </span><span class="NLM_article-title">Hydroxamic acid and benzoic acid-based Stat3 inhibitors suppress human glioma and breast cancer phenotypes in vitro and in vivo</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>76</i></span>,  <span class="NLM_fpage">652</span>– <span class="NLM_lpage">663</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.CAN-14-3558</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=10.1158%2F0008-5472.CAN-14-3558" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=26088127" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=1%3ACAS%3A528%3ADC%252BC28XhslSgtrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=76&publication_year=2016&pages=652-663&author=P.+Yueauthor=F.+Lopez-Tapiaauthor=D.+Paladinoauthor=Y.+Liauthor=C.-H.+Chenauthor=T.+Hilliardauthor=Y.+Chenauthor=M.+Tiusauthor=J.+Turkson&title=Hydroxamic+acid+and+benzoic+acid-based+Stat3+inhibitors+suppress+human+glioma+and+breast+cancer+phenotypes+in+vitro+and+in+vivo&doi=10.1158%2F0008-5472.CAN-14-3558"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Hydroxamic Acid and Benzoic Acid-Based STAT3 Inhibitors Suppress Human Glioma and Breast Cancer Phenotypes In Vitro and In Vivo</span></div><div class="casAuthors">Yue, Peibin; Lopez-Tapia, Francisco; Paladino, David; Li, Yifei; Chen, Chih-Hong; Namanja, Andrew T.; Hilliard, Tyvette; Chen, Yuan; Tius, Marcus A.; Turkson, James</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">76</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">652-663</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">STAT3 offers an attractive target for cancer therapy, but small-mol. inhibitors with appealing pharmacol. properties have been elusive.  Here, we report hydroxamic acid-based and benzoic acid-based inhibitors (SH5-07 and SH4-54, resp.) with robust bioactivity.  Both inhibitors blocked STAT3 DNA-binding activity in vitro and in human glioma, breast, and prostate cancer cells and in v-Src-transformed murine fibroblasts.  STAT3-dependent gene transcription was blocked along with Bcl-2, Bcl-xL, Mcl-1, cyclin D1, c-Myc, and survivin expression.  NMR anal. of STAT3-inhibitor complexes defined interactions with the SH2 and DNA-binding domains of STAT3.  Ectopic expression of the SH2 domain in cells was sufficient to counter the STAT3-inhibitory effects of SH4-54.  Neither compd. appreciably affected STAT1 or STAT5 DNA-binding activities, STAT3-independent gene transcription, or activation of a panel of oncogenic kinases in malignant cells.  Each compd. decreased the proliferation and viability of glioma, breast, and prostate cancer cells and v-Src-transformed murine fibroblasts harboring constitutively active STAT3.  Further, in mouse xenograft models of glioma and breast cancer, administration of SH5-07 or SH4-54 effectively inhibited tumor growth.  Our results offer preclin. proof of concept for SH5-07 and SH4-54 as candidates for further development as cancer therapeutics.  Cancer Res; 76(3); 652-63. ©2015 AACR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGohkHZXQZ-69bVg90H21EOLACvtfcHk0lhvPN-hISa7bg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhslSgtrY%253D&md5=62397d5abc0b6e551289d44e808b6cd1</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-14-3558&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-14-3558%26sid%3Dliteratum%253Aachs%26aulast%3DYue%26aufirst%3DP.%26aulast%3DLopez-Tapia%26aufirst%3DF.%26aulast%3DPaladino%26aufirst%3DD.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DChen%26aufirst%3DC.-H.%26aulast%3DHilliard%26aufirst%3DT.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DTius%26aufirst%3DM.%26aulast%3DTurkson%26aufirst%3DJ.%26atitle%3DHydroxamic%2520acid%2520and%2520benzoic%2520acid-based%2520Stat3%2520inhibitors%2520suppress%2520human%2520glioma%2520and%2520breast%2520cancer%2520phenotypes%2520in%2520vitro%2520and%2520in%2520vivo%26jtitle%3DCancer%2520Res.%26date%3D2016%26volume%3D76%26spage%3D652%26epage%3D663%26doi%3D10.1158%2F0008-5472.CAN-14-3558" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Miklossy, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hilliard, T. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turkson, J.</span></span> <span> </span><span class="NLM_article-title">Therapeutic modulators of Stat signaling for human diseases</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">611</span>– <span class="NLM_lpage">629</span>, <span class="refDoi"> DOI: 10.1038/nrd4088</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=10.1038%2Fnrd4088" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=23903221" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=1%3ACAS%3A528%3ADC%252BC3sXht1SjtbvI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2013&pages=611-629&author=G.+Miklossyauthor=T.+S.+Hilliardauthor=J.+Turkson&title=Therapeutic+modulators+of+Stat+signaling+for+human+diseases&doi=10.1038%2Fnrd4088"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Therapeutic modulators of STAT signalling for human diseases</span></div><div class="casAuthors">Miklossy, Gabriella; Hilliard, Tyvette S.; Turkson, James</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">4_testjo</span>),
    <span class="NLM_cas:pages">611-629</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The signal transducer and activator of transcription (STAT) proteins have important roles in biol. processes.  The abnormal activation of STAT signalling pathways is also implicated in many human diseases, including cancer, autoimmune diseases, rheumatoid arthritis, asthma and diabetes.  Over a decade has passed since the first inhibitor of a STAT protein was reported and efforts to discover modulators of STAT signalling as therapeutics continue.  This Review discusses the outcomes of the ongoing drug discovery research endeavors against STAT proteins, provides perspectives on new directions for accelerating the discovery of drug candidates, and highlights the noteworthy candidate therapeutics that have progressed to clin. trials.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoSgI1shc9cRrVg90H21EOLACvtfcHk0lhvPN-hISa7bg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXht1SjtbvI&md5=fcf21bda56a222e81ef7c44166f0b20c</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1038%2Fnrd4088&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd4088%26sid%3Dliteratum%253Aachs%26aulast%3DMiklossy%26aufirst%3DG.%26aulast%3DHilliard%26aufirst%3DT.%2BS.%26aulast%3DTurkson%26aufirst%3DJ.%26atitle%3DTherapeutic%2520modulators%2520of%2520Stat%2520signaling%2520for%2520human%2520diseases%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2013%26volume%3D12%26spage%3D611%26epage%3D629%26doi%3D10.1038%2Fnrd4088" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bai, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McEachern, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chinnaswamy, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wen, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dai, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumar, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meagher, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stuckey, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span> <span> </span><span class="NLM_article-title">Structure-based discovery of SD-36 as a potent, selective, and efficacious protac degrader of Stat3 protein</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">11280</span>– <span class="NLM_lpage">11300</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b01530</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b01530" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=1%3ACAS%3A528%3ADC%252BC1MXit1SnsL3J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=11280-11300&author=H.+Zhouauthor=L.+Baiauthor=R.+Xuauthor=Y.+Zhaoauthor=J.+Chenauthor=D.+McEachernauthor=K.+Chinnaswamyauthor=B.+Wenauthor=L.+Daiauthor=P.+Kumarauthor=C.+Y.+Yangauthor=Z.+Liuauthor=M.+Wangauthor=L.+Liuauthor=J.+L.+Meagherauthor=H.+Yiauthor=D.+Sunauthor=J.+A.+Stuckeyauthor=S.+Wang&title=Structure-based+discovery+of+SD-36+as+a+potent%2C+selective%2C+and+efficacious+protac+degrader+of+Stat3+protein&doi=10.1021%2Facs.jmedchem.9b01530"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Based Discovery of SD-36 as a Potent, Selective, and Efficacious PROTAC Degrader of STAT3 Protein</span></div><div class="casAuthors">Zhou, Haibin; Bai, Longchuan; Xu, Renqi; Zhao, Yujun; Chen, Jianyong; McEachern, Donna; Chinnaswamy, Krishnapriya; Wen, Bo; Dai, Lipeng; Kumar, Praveen; Yang, Chao-Yie; Liu, Zhaomin; Wang, Mi; Liu, Liu; Meagher, Jennifer L.; Yi, Han; Sun, Duxin; Stuckey, Jeanne A.; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">11280-11300</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Signal transducer and activator of transcription 3 (STAT3) is a transcription factor and an attractive therapeutic target for cancer and other human diseases.  Despite twenty-years of persistent research efforts, targeting STAT3 has been very challenging.  The authors report herein the structure-based discovery of potent small-mol. STAT3 degraders based upon the proteolysis targeting chimera (PROTAC) concept.  The authors first designed SI-109 as a potent, small-mol. inhibitor of the STAT3 SH2 domain.  Employing ligands for cereblon/Cullin 4A E3 ligase and SI-109, the authors obtained a series of potent PROTAC STAT3 degraders, exemplified by SD-36.  SD-36 induces rapid STAT3 degrdn. at low nanomolar concns. in cells and fails to degrader other STAT proteins.  SD-36 achieves nanomolar cell growth inhibitory activity in leukemia and lymphoma cell lines with high levels of phosphorylated STAT3.  A single dose of SD-36 results in complete STAT3 protein degrdn. in xenograft tumor tissue and normal mouse tissues.  SD-36 achieves complete and long-lasting tumor regression in the Molm-16 xenograft tumor model at well-tolerated dose-schedules.  SD-36 is a potent, selective and efficacious STAT3 degrader.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoIAWd1hiuqy7Vg90H21EOLACvtfcHk0lgjYWFLWS6ESw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXit1SnsL3J&md5=7d0c656a2d96196a3b71829c9efe715a</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b01530%26sid%3Dliteratum%253Aachs%26aulast%3DZhou%26aufirst%3DH.%26aulast%3DBai%26aufirst%3DL.%26aulast%3DXu%26aufirst%3DR.%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DMcEachern%26aufirst%3DD.%26aulast%3DChinnaswamy%26aufirst%3DK.%26aulast%3DWen%26aufirst%3DB.%26aulast%3DDai%26aufirst%3DL.%26aulast%3DKumar%26aufirst%3DP.%26aulast%3DYang%26aufirst%3DC.%2BY.%26aulast%3DLiu%26aufirst%3DZ.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DMeagher%26aufirst%3DJ.%2BL.%26aulast%3DYi%26aufirst%3DH.%26aulast%3DSun%26aufirst%3DD.%26aulast%3DStuckey%26aufirst%3DJ.%2BA.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DStructure-based%2520discovery%2520of%2520SD-36%2520as%2520a%2520potent%252C%2520selective%252C%2520and%2520efficacious%2520protac%2520degrader%2520of%2520Stat3%2520protein%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D11280%26epage%3D11300%26doi%3D10.1021%2Facs.jmedchem.9b01530" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bai, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chinnaswamy, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McEachern, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wen, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumar, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meagher, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stuckey, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span> <span> </span><span class="NLM_article-title">A potent and selective small-molecule degrader of Stat3 achieves complete tumor regression in vivo</span>. <i>Cancer Cell</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>36</i></span>,  <span class="NLM_fpage">498</span>– <span class="NLM_lpage">511.e17</span>, <span class="refDoi"> DOI: 10.1016/j.ccell.2019.10.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=10.1016%2Fj.ccell.2019.10.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=31715132" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitFCjtrjO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2019&pages=498-511.e17&author=L.+Baiauthor=H.+Zhouauthor=R.+Xuauthor=Y.+Zhaoauthor=K.+Chinnaswamyauthor=D.+McEachernauthor=J.+Chenauthor=C.+Y.+Yangauthor=Z.+Liuauthor=M.+Wangauthor=L.+Liuauthor=H.+Jiangauthor=B.+Wenauthor=P.+Kumarauthor=J.+L.+Meagherauthor=D.+Sunauthor=J.+A.+Stuckeyauthor=S.+Wang&title=A+potent+and+selective+small-molecule+degrader+of+Stat3+achieves+complete+tumor+regression+in+vivo&doi=10.1016%2Fj.ccell.2019.10.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">A Potent and Selective Small-Molecule Degrader of STAT3 Achieves Complete Tumor Regression In Vivo</span></div><div class="casAuthors">Bai, Longchuan; Zhou, Haibin; Xu, Renqi; Zhao, Yujun; Chinnaswamy, Krishnapriya; McEachern, Donna; Chen, Jianyong; Yang, Chao-Yie; Liu, Zhaomin; Wang, Mi; Liu, Liu; Jiang, Hui; Wen, Bo; Kumar, Praveen; Meagher, Jennifer L.; Sun, Duxin; Stuckey, Jeanne A.; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">498-511.e17</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Signal transducer and activator of transcription 3 (STAT3) is an attractive cancer therapeutic target.  Here we report the discovery of SD-36(I), a small-mol. degrader of STAT3.  SD-36 potently induces the degrdn. of STAT3 protein in vitro and in vivo and demonstrates high selectivity over other STAT members.  Induced degrdn. of STAT3 results in a strong suppression of its transcription network in leukemia and lymphoma cells.  SD-36 inhibits the growth of a subset of acute myeloid leukemia and anaplastic large-cell lymphoma cell lines by inducing cell-cycle arrest and/or apoptosis.  SD-36 achieves complete and long-lasting tumor regression in multiple xenograft mouse models at well-tolerated dose schedules.  Degrdn. of STAT3 protein, therefore, is a promising cancer therapeutic strategy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoUaP49Ap9DxLVg90H21EOLACvtfcHk0lgjYWFLWS6ESw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitFCjtrjO&md5=71252c17d50a1a8be829352942175bd6</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1016%2Fj.ccell.2019.10.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccell.2019.10.002%26sid%3Dliteratum%253Aachs%26aulast%3DBai%26aufirst%3DL.%26aulast%3DZhou%26aufirst%3DH.%26aulast%3DXu%26aufirst%3DR.%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DChinnaswamy%26aufirst%3DK.%26aulast%3DMcEachern%26aufirst%3DD.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DYang%26aufirst%3DC.%2BY.%26aulast%3DLiu%26aufirst%3DZ.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DWen%26aufirst%3DB.%26aulast%3DKumar%26aufirst%3DP.%26aulast%3DMeagher%26aufirst%3DJ.%2BL.%26aulast%3DSun%26aufirst%3DD.%26aulast%3DStuckey%26aufirst%3DJ.%2BA.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DA%2520potent%2520and%2520selective%2520small-molecule%2520degrader%2520of%2520Stat3%2520achieves%2520complete%2520tumor%2520regression%2520in%2520vivo%26jtitle%3DCancer%2520Cell%26date%3D2019%26volume%3D36%26spage%3D498%26epage%3D511.e17%26doi%3D10.1016%2Fj.ccell.2019.10.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lopez-Tapia, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brotherton-Pleiss, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yue, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murakami, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Costa Araujo, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reis dos Santos, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ichinotsubo, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rabkin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shah, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lantz, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tius, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turkson, J.</span></span> <span> </span><span class="NLM_article-title">Linker variation and structure-activity relationship analyses of car-boxylic acid-based small molecule Stat3 inhibitors</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">250</span>– <span class="NLM_lpage">255</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.7b00544</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.7b00544" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=1%3ACAS%3A528%3ADC%252BC1cXivFCnsLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2018&pages=250-255&author=F.+Lopez-Tapiaauthor=C.+Brotherton-Pleissauthor=P.+Yueauthor=H.+Murakamiauthor=A.+C.+Costa+Araujoauthor=B.+Reis+dos+Santosauthor=E.+Ichinotsuboauthor=A.+Rabkinauthor=R.+Shahauthor=M.+Lantzauthor=S.+Chenauthor=M.+A.+Tiusauthor=J.+Turkson&title=Linker+variation+and+structure-activity+relationship+analyses+of+car-boxylic+acid-based+small+molecule+Stat3+inhibitors&doi=10.1021%2Facsmedchemlett.7b00544"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Linker variation and structure-activity relationship analysis of carboxylic acid-based small molecule STAT3 inhibitors</span></div><div class="casAuthors">Lopez-Tapia, Francisco; Brotherton-Pleiss, Christine; Yue, Peibin; Murakami, Heide; Costa Araujo, Ana Carolina; Reis dos Santos, Bruna; Ichinotsubo, Erin; Rabkin, Anna; Shah, Raj; Lantz, Megan; Chen, Suzie; Tius, Marcus A.; Turkson, James</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">250-255</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The mol. determinants for the activities of the reported benzoic acid (SH4-54), salicylic acid (BP-1-102), and benzohydroxamic acid (SH5-07)-based STAT3 inhibitors were investigated to design optimized analogs.  All three leads are based on an N-methylglycinamide scaffold, with its two amine groups condensed with three different functionalities.  The three functionalities and the CH2 group of the glycinamide scaffold were sep. modified.  The replacement of the pentafluorobenzene or cyclohexylbenzene, or replacing the benzene ring of the arom. carboxylic or hydroxamic acid motif with heterocyclic components (contg. nitrogen and oxygen elements) all decreased potency.  Notably, the Ala-linker analogs, (I) (R1 = H and OH), and the Pro-based deriv. (II) (X = CH2), all with (R)-configuration at the chiral center, had improved inhibitory activity and selectivity against STAT3 DNA-binding activity in vitro, with IC50 of 3.0 ± 0.9, 1.80 ± 0.94, and 2.4 ± 0.2 μM, resp.  Compds. I and II and other analogs inhibited constitutive STAT3 phosphorylation and activation in human breast cancer and melanoma lines, and blocked tumor cell viability, growth, colony formation, and migration in vitro.  Pro-based analog, II (X = O)(sodium salt), with a relatively polar tetrahydropyranyl (THP) ring, instead of the cyclohexyl, showed improved permeability.  In general, the (R)-configuration Pro-based analogs showed the overall best profile, including physicochem. properties (e.g., microsomal metabolic stability, Caco-2 permeability), and in particular, II (X = CH2) showed improved tumor-cell specificity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr6yrAyhBvV47Vg90H21EOLACvtfcHk0lgjYWFLWS6ESw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXivFCnsLs%253D&md5=c80cc2d96017675b30e13a0f0395103a</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.7b00544&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.7b00544%26sid%3Dliteratum%253Aachs%26aulast%3DLopez-Tapia%26aufirst%3DF.%26aulast%3DBrotherton-Pleiss%26aufirst%3DC.%26aulast%3DYue%26aufirst%3DP.%26aulast%3DMurakami%26aufirst%3DH.%26aulast%3DCosta%2BAraujo%26aufirst%3DA.%2BC.%26aulast%3DReis%2Bdos%2BSantos%26aufirst%3DB.%26aulast%3DIchinotsubo%26aufirst%3DE.%26aulast%3DRabkin%26aufirst%3DA.%26aulast%3DShah%26aufirst%3DR.%26aulast%3DLantz%26aufirst%3DM.%26aulast%3DChen%26aufirst%3DS.%26aulast%3DTius%26aufirst%3DM.%2BA.%26aulast%3DTurkson%26aufirst%3DJ.%26atitle%3DLinker%2520variation%2520and%2520structure-activity%2520relationship%2520analyses%2520of%2520car-boxylic%2520acid-based%2520small%2520molecule%2520Stat3%2520inhibitors%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2018%26volume%3D9%26spage%3D250%26epage%3D255%26doi%3D10.1021%2Facsmedchemlett.7b00544" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Garcia, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hudnall, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Catlett, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nelson, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smithgall, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fujita, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ethier, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jove, R.</span></span> <span> </span><span class="NLM_article-title">Constitutive activation of Stat3 in fibroblasts transformed by diverse oncoproteins and in breast carcinoma cells</span>. <i>Cell Growth Diff.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">1267</span>– <span class="NLM_lpage">1276</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=9419415" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=1%3ACAS%3A528%3ADyaK2sXotVWlt7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=1997&pages=1267-1276&author=R.+Garciaauthor=C.+L.+Yuauthor=A.+Hudnallauthor=R.+Catlettauthor=K.+L.+Nelsonauthor=T.+Smithgallauthor=D.+J.+Fujitaauthor=S.+P.+Ethierauthor=R.+Jove&title=Constitutive+activation+of+Stat3+in+fibroblasts+transformed+by+diverse+oncoproteins+and+in+breast+carcinoma+cells"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Constitutive activation of Stat3 in fibroblasts transformed by diverse oncoproteins and in breast carcinoma cells</span></div><div class="casAuthors">Garcia, Roy; Yu, Chao-Lan; Hudnall, Anne; Catlett, Robyn; Nelson, Kristie L.; Smithgall, Thomas; Fujita, Donald J.; Ethier, Stephen P.; Jove, Richard</div><div class="citationInfo"><span class="NLM_cas:title">Cell Growth & Differentiation</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1267-1276</span>CODEN:
                <span class="NLM_cas:coden">CGDIE7</span>;
        ISSN:<span class="NLM_cas:issn">1044-9523</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Signal transducers and activators of transcription (STATs) were originally identified as key components of signaling pathways involved in mediating responses to IFNs.  Previous studies showed that the Src oncoprotein constitutively activates one STAT family member, Stat3.  In this study, we investigated STAT activation in a panel of rodent fibroblast cell lines stably transformed by diverse viral oncoproteins.  Using a temp.-sensitive mutant of v-Src, we detd. that Stat3 is activated within 15 min of shift from nonpermissive to permissive temp. for cell transformation.  This finding indicates that v-Src tyrosine kinase activity is required for Stat3 activation and suggests that Stat3 is proximal to signaling initiated by Src.  In addn., Stat3 activation is induced by another nonreceptor tyrosine kinase, v-Fps; by polyoma virus middle T antigen, which activates Src family kinases; and by v-Sis, which acts as a ligand for the platelet-derived growth factor receptor.  In contrast, SV40 large T antigen, which transforms cells through different mechanisms, and the v-Ras and v-Raf oncoproteins, which lie in signaling pathways downstream of tyrosine kinases, do not activate Stat3.  We did not detect significant activation of Stat1, Stat5, or Stat6 in fibroblasts transformed by the viral oncoproteins investigated.  Moreover, Stat3 is activated in response to epidermal growth factor (EGF) but not heregulins in immortalized normal human breast epithelial cells.  Because constitutive activation of c-Src and EGF receptor kinases is assocd. with the progression of breast cancer, we examd. activation of STATs in human cell lines derived from breast carcinomas.  We detected constitutive activation of Stat3 in five of nine breast carcinoma cell lines but not in normal breast epithelial cells.  Furthermore, expts. with an EGF receptor-specific inhibitor indicated that the constitutive activation of Stat3 in these breast carcinoma cell lines is not necessarily dependent on signaling through the EGF receptor, although EGF stimulation further increases Stat3 activation.  Taken together, our results demonstrate that selective activation of Stat3 is a common event during oncogenic transformation that directly or indirectly involves activation of specific tyrosine kinase signaling pathways.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoqkU_WBMF3KLVg90H21EOLACvtfcHk0ljuftPoiNqo0Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXotVWlt7s%253D&md5=3e6712ee4cfa9196b9bfac89b51b081a</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DGarcia%26aufirst%3DR.%26aulast%3DYu%26aufirst%3DC.%2BL.%26aulast%3DHudnall%26aufirst%3DA.%26aulast%3DCatlett%26aufirst%3DR.%26aulast%3DNelson%26aufirst%3DK.%2BL.%26aulast%3DSmithgall%26aufirst%3DT.%26aulast%3DFujita%26aufirst%3DD.%2BJ.%26aulast%3DEthier%26aufirst%3DS.%2BP.%26aulast%3DJove%26aufirst%3DR.%26atitle%3DConstitutive%2520activation%2520of%2520Stat3%2520in%2520fibroblasts%2520transformed%2520by%2520diverse%2520oncoproteins%2520and%2520in%2520breast%2520carcinoma%2520cells%26jtitle%3DCell%2520Growth%2520Diff.%26date%3D1997%26volume%3D8%26spage%3D1267%26epage%3D1276" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Garcia, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bowman, T. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Niu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minton, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muro-Cacho, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cox, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Falcone, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fairclough, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parson, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laudano, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gazit, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Levitzki, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kraker, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jove, R.</span></span> <span> </span><span class="NLM_article-title">Constitutive activation of Stat3 by the Src and Jak tyrosine kinases participates in growth regulation of human breast carcinoma cells</span>. <i>Oncogene</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">2499</span>– <span class="NLM_lpage">2513</span>, <span class="refDoi"> DOI: 10.1038/sj.onc.1204349</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=10.1038%2Fsj.onc.1204349" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=11420660" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=1%3ACAS%3A528%3ADC%252BD3MXjvFGmt7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2001&pages=2499-2513&author=R.+Garciaauthor=T.+L.+Bowmanauthor=G.+Niuauthor=H.+Yuauthor=S.+Mintonauthor=C.+A.+Muro-Cachoauthor=C.+E.+Coxauthor=R.+Falconeauthor=R.+Faircloughauthor=S.+Parsonauthor=A.+Laudanoauthor=A.+Gazitauthor=A.+Levitzkiauthor=A.+Krakerauthor=R.+Jove&title=Constitutive+activation+of+Stat3+by+the+Src+and+Jak+tyrosine+kinases+participates+in+growth+regulation+of+human+breast+carcinoma+cells&doi=10.1038%2Fsj.onc.1204349"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Constitutive activation of Stat3 by the Src and JAK tyrosine kinases participates in growth regulation of human breast carcinoma cells</span></div><div class="casAuthors">Garcia, Roy; Bowman, Tammy L.; Niu, Guilian; Yu, Hua; Minton, Sue; Muro-Cacho, Carlos A.; Cox, Charles E.; Falcone, Robert; Fairclough, Rita; Parsons, Sarah; Laudano, Andy; Gazit, Aviv; Levitzki, Alexander; Kraker, Alan; Jove, Richard</div><div class="citationInfo"><span class="NLM_cas:title">Oncogene</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">2499-2513</span>CODEN:
                <span class="NLM_cas:coden">ONCNES</span>;
        ISSN:<span class="NLM_cas:issn">0950-9232</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Constitutive activation of signal transducer and activator of transcription (STAT) proteins has been detected in a wide variety of human primary tumor specimens and tumor cell lines including blood malignancies, head and neck cancer, and breast cancer.  We have previously demonstrated a high frequency of Stat3 DNA-binding activity that is constitutively-induced by an unknown mechanism in human breast cancer cell lines possessing elevated EGF receptor (EGF-R) and c-Src kinase activities.  Using tyrosine kinase selective inhibitors, we show here that Src and JAK family tyrosine kinases cooperate to mediate constitutive Stat3 activation in the absence of EGF stimulation in model human breast cancer cell lines.  Inhibition of Src or JAKs results in dose-dependent suppression of Stat3 DNA-binding activity, which is accompanied by growth inhibition and induction of programmed cell death.  In addn., transfection of a dominant-neg. form of Stat3 leads to growth inhibition involving apoptosis of breast cancer cells.  These results indicate that the biol. effects of the Src and JAK tyrosine kinase inhibitors are at least partially mediated by blocking Stat3 signaling.  While EGF-R kinase activity is not required for constitutive Stat3 activation in breast cancer cells, EGF stimulation further increases STAT DNA-binding activity, consistent with an important role for EGF-R in STAT signaling and malignant progression.  Anal. of primary breast tumor specimens from patients with advanced disease revealed that the majority exhibit elevated STAT DNA-binding activity compared to adjacent non-tumor tissues.  Our findings, taken together, suggest that tyrosine kinases transduce signals through Stat3 protein that contribute to the growth and survival of human breast cancer cells in culture and potentially in vivo.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq5BSpinbxg3rVg90H21EOLACvtfcHk0ljuftPoiNqo0Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXjvFGmt7k%253D&md5=a941c45ef8fbf78234c6f66b741e7be2</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1038%2Fsj.onc.1204349&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.onc.1204349%26sid%3Dliteratum%253Aachs%26aulast%3DGarcia%26aufirst%3DR.%26aulast%3DBowman%26aufirst%3DT.%2BL.%26aulast%3DNiu%26aufirst%3DG.%26aulast%3DYu%26aufirst%3DH.%26aulast%3DMinton%26aufirst%3DS.%26aulast%3DMuro-Cacho%26aufirst%3DC.%2BA.%26aulast%3DCox%26aufirst%3DC.%2BE.%26aulast%3DFalcone%26aufirst%3DR.%26aulast%3DFairclough%26aufirst%3DR.%26aulast%3DParson%26aufirst%3DS.%26aulast%3DLaudano%26aufirst%3DA.%26aulast%3DGazit%26aufirst%3DA.%26aulast%3DLevitzki%26aufirst%3DA.%26aulast%3DKraker%26aufirst%3DA.%26aulast%3DJove%26aufirst%3DR.%26atitle%3DConstitutive%2520activation%2520of%2520Stat3%2520by%2520the%2520Src%2520and%2520Jak%2520tyrosine%2520kinases%2520participates%2520in%2520growth%2520regulation%2520of%2520human%2520breast%2520carcinoma%2520cells%26jtitle%3DOncogene%26date%3D2001%26volume%3D20%26spage%3D2499%26epage%3D2513%26doi%3D10.1038%2Fsj.onc.1204349" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Turkson, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bowman, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garcia, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caldenhoven, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Groot, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jove, R.</span></span> <span> </span><span class="NLM_article-title">Stat3 activation by Src induces specific gene regulation and is required for cell transformation</span>. <i>Mol. Cell. Biol.</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">2545</span>– <span class="NLM_lpage">2552</span>, <span class="refDoi"> DOI: 10.1128/MCB.18.5.2545</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=10.1128%2FMCB.18.5.2545" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=9566874" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=1%3ACAS%3A528%3ADyaK1cXis1GntLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=1998&pages=2545-2552&author=J.+Turksonauthor=T.+Bowmanauthor=R.+Garciaauthor=E.+Caldenhovenauthor=R.+P.+De+Grootauthor=R.+Jove&title=Stat3+activation+by+Src+induces+specific+gene+regulation+and+is+required+for+cell+transformation&doi=10.1128%2FMCB.18.5.2545"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Stat3 activation by Src induces specific gene regulation and is required for cell transformation</span></div><div class="casAuthors">Turkson, James; Bowman, Tammy; Garcia, Roy; Caldenhoven, Eric; De Groot, Rolf P.; Jove, Richard</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">2545-2552</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">0270-7306</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">While signal transducers and activators of transcription (STATs) were originally discovered as intracellular effectors of normal signaling by cytokines, increasing evidence also points to a role for STAT transcription factors in oncogenesis.  Previous studies have demonstrated that one STAT family member, Stat3, possesses constitutively elevated tyrosine phosphorylation and DNA-binding activity in fibroblasts stably transformed by the Src oncoprotein.  To det. if this Stat3 activation by Src could induce Stat3-mediated gene expression, luciferase reporter constructs based on synthetic and authentic promoters were transfected into NIH 3T3 cells.  Activation of endogenous cellular Stat3 by the Src oncoprotein induced gene expression through a Stat3-specific binding element (TTCCCGAA) of the C-reactive protein gene promoter.  A naturally occurring splice variant of human Stat3 protein, Stat3β, with a deletion in the C-terminal transactivation domain abolished this gene induction in a dominant neg. manner.  Expression of Stat3β did not have any effect on a reporter construct based on the c-fos serum response element, which is not dependent on Stat3 signaling, indicating that Stat3β does not nonspecifically inhibit other signaling pathways or Src function.  Transfection of vectors expressing Stat3β together with Src blocked cell transformation by Src as measured in a quant. focus formation assay using NIH 3T3 cells.  By contrast, Stat3β had a much less pronounced effect on focus formation induced by the Ras oncoprotein, which does not activate Stat3 signaling.  In addn., three independent clones of NIH 3T3 cells stably overexpressing Stat3β were generated and characterized, demonstrating that Stat3β overexpression does not have a toxic effect on cell viability.  These Stat3β-overexpressing clones were shown to be deficient in Stat3-mediated signaling and refractory to Src-induced cell transformation.  The authors conclude that Stat3 activation by the Src oncoprotein leads to specific gene regulation and that Stat3 is one of the crit. signaling pathways involved in Src oncogenesis.  The authors' findings provide evidence that oncogenesis-assocd. activation of Stat3 signaling is part of the process of malignant transformation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpX3GjARHEUY7Vg90H21EOLACvtfcHk0ljuftPoiNqo0Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXis1GntLc%253D&md5=935fc4454fb988b7e570f15672d2c173</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1128%2FMCB.18.5.2545&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.18.5.2545%26sid%3Dliteratum%253Aachs%26aulast%3DTurkson%26aufirst%3DJ.%26aulast%3DBowman%26aufirst%3DT.%26aulast%3DGarcia%26aufirst%3DR.%26aulast%3DCaldenhoven%26aufirst%3DE.%26aulast%3DDe%2BGroot%26aufirst%3DR.%2BP.%26aulast%3DJove%26aufirst%3DR.%26atitle%3DStat3%2520activation%2520by%2520Src%2520induces%2520specific%2520gene%2520regulation%2520and%2520is%2520required%2520for%2520cell%2520transformation%26jtitle%3DMol.%2520Cell.%2520Biol.%26date%3D1998%26volume%3D18%26spage%3D2545%26epage%3D2552%26doi%3D10.1128%2FMCB.18.5.2545" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, B.</span></span> <span> </span><span class="NLM_article-title">Napabucasin (BBI608) eliminate AML cells in vitro and in vivo via inhibition of Stat3 pathway and induction of DNA damage</span>. <i>Eur. J. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>855</i></span>,  <span class="NLM_fpage">252</span>– <span class="NLM_lpage">261</span>, <span class="refDoi"> DOI: 10.1016/j.ejphar.2019.05.020</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=10.1016%2Fj.ejphar.2019.05.020" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=31085238" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=1%3ACAS%3A528%3ADC%252BC1MXps1Glsr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=855&publication_year=2019&pages=252-261&author=S.+Biauthor=K.+Chenauthor=L.+Fengauthor=G.+Fuauthor=Q.+Yangauthor=M.+Dengauthor=H.+Zhaoauthor=Z.+Liauthor=L.+Yuauthor=Z.+Fangauthor=B.+Xu&title=Napabucasin+%28BBI608%29+eliminate+AML+cells+in+vitro+and+in+vivo+via+inhibition+of+Stat3+pathway+and+induction+of+DNA+damage&doi=10.1016%2Fj.ejphar.2019.05.020"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Napabucasin (BBI608) eliminate AML cells in vitro and in vivo via inhibition of Stat3 pathway and induction of DNA damage</span></div><div class="casAuthors">Bi, Silei; Chen, Kai; Feng, Liying; Fu, Guofeng; Yang, Qianying; Deng, Manman; Zhao, Haijun; Li, Zhifeng; Yu, Lian; Fang, Zhihong; Xu, Bing</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Pharmacology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">855</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">252-261</span>CODEN:
                <span class="NLM_cas:coden">EJPHAZ</span>;
        ISSN:<span class="NLM_cas:issn">0014-2999</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Acute myeloid leukemia (AML) is a heterogeneous malignancy of hematopoietic stem cells with poor clin. outcome despite recent improvements in chemotherapy and stem cell transplantation regimens.  Thus, new therapeutic agents are urgently needed in order to prolong the disease-free survival of AML patients in clinic.  Here, we report that BBI608 is highly active against diverse AML cell lines in vitro and primary samples obtained from patients with AML ex vivo, as well as effective in vivo in AML xenograft models.  Meanwhile, the anti-AML property of BBI608 is closely assocd. with the inhibition of Stat3 pathway and induction of DNA damage.  Of note, BBI608 combined with Bcl-2 inhibitor (i.e., ABT-199) exerts a significantly enhanced anti-leukemia effect in BBI608-resistant cell line Kasumi-1.  Together, the present findings suggest that BBI608 might represent a potential candidate agent for AML treatment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr3ukbC0r0gDLVg90H21EOLACvtfcHk0liOYsNJ64QbEw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXps1Glsr8%253D&md5=1b3a2e0089b0b42eaa1716215bb13335</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1016%2Fj.ejphar.2019.05.020&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejphar.2019.05.020%26sid%3Dliteratum%253Aachs%26aulast%3DBi%26aufirst%3DS.%26aulast%3DChen%26aufirst%3DK.%26aulast%3DFeng%26aufirst%3DL.%26aulast%3DFu%26aufirst%3DG.%26aulast%3DYang%26aufirst%3DQ.%26aulast%3DDeng%26aufirst%3DM.%26aulast%3DZhao%26aufirst%3DH.%26aulast%3DLi%26aufirst%3DZ.%26aulast%3DYu%26aufirst%3DL.%26aulast%3DFang%26aufirst%3DZ.%26aulast%3DXu%26aufirst%3DB.%26atitle%3DNapabucasin%2520%2528BBI608%2529%2520eliminate%2520AML%2520cells%2520in%2520vitro%2520and%2520in%2520vivo%2520via%2520inhibition%2520of%2520Stat3%2520pathway%2520and%2520induction%2520of%2520DNA%2520damage%26jtitle%3DEur.%2520J.%2520Pharmacol.%26date%3D2019%26volume%3D855%26spage%3D252%26epage%3D261%26doi%3D10.1016%2Fj.ejphar.2019.05.020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lewis, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bharadwaj, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eckols, T. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kolosov, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kasembeli, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fridley, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siller, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tweardy, D. J.</span></span> <span> </span><span class="NLM_article-title">Small-molecule targeting of signal transducer and activator of transcription (Stat) 3 to treat non-small cell lung cancer</span>. <i>Lung Cancer</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>90</i></span>,  <span class="NLM_fpage">182</span>– <span class="NLM_lpage">190</span>, <span class="refDoi"> DOI: 10.1016/j.lungcan.2015.09.014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=10.1016%2Fj.lungcan.2015.09.014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=26410177" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=1%3ACAS%3A280%3ADC%252BC283mtVansA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=90&publication_year=2015&pages=182-190&author=K.+M.+Lewisauthor=U.+Bharadwajauthor=T.+K.+Eckolsauthor=M.+Kolosovauthor=M.+M.+Kasembeliauthor=C.+Fridleyauthor=R.+Sillerauthor=D.+J.+Tweardy&title=Small-molecule+targeting+of+signal+transducer+and+activator+of+transcription+%28Stat%29+3+to+treat+non-small+cell+lung+cancer&doi=10.1016%2Fj.lungcan.2015.09.014"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Small-molecule targeting of signal transducer and activator of transcription (STAT) 3 to treat non-small cell lung cancer</span></div><div class="casAuthors">Lewis Katherine M; Bharadwaj Uddalak; Eckols T Kris; Kolosov Mikhail; Kasembeli Moses M; Siller Ricardo; Fridley Colleen; Tweardy David J</div><div class="citationInfo"><span class="NLM_cas:title">Lung cancer (Amsterdam, Netherlands)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">90</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">182-90</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">OBJECTIVE:  Lung cancer is the leading cause of cancer death in both men and women.  Non-small cell lung cancer (NSCLC) has an overall 5-year survival rate of 15%.  While aberrant STAT3 activation has previously been observed in NSCLC, the scope of its contribution is uncertain and agents that target STAT3 for treatment are not available clinically.  METHODS:  We determined levels of activated STAT3 (STAT3 phosphorylated on Y705, pSTAT3) and the two major isoforms of STAT3 (α and β) in protein extracts of 8 NSCLC cell lines, as well as the effects of targeting STAT3 in vitro and in vivo in NSCLC cells using short hairpin (sh) RNA and two novel small-molecule STAT3 inhibitors, C188-9 and piperlongumine (PL).  RESULTS:  Levels of pSTAT3, STAT3α, and STATβ were increased in 7 of 8 NSCLC cell lines.  Of note, levels of pSTAT3 were tightly correlated with levels of STAT3β, but not STAT3α.  Targeting of STAT3 in A549 cells using shRNA decreased tSTAT3 by 75%; this was accompanied by a 47-78% reduction in anchorage-dependent and anchorage-independent growth and a 28-45% reduction in mRNA levels for anti-apoptotic STAT3 gene targets.  C188-9 and PL (@30 μM) each reduced pSTAT3 levels in all NSCLC cell lines tested by ≥50%, reduced anti-apoptotic protein mRNA levels by 25-60%, and reduced both anchorage-dependent and anchorage-independent growth of NSCLC cell lines with IC50 values ranging from 3.06 to 52.44 μM and 0.86 to 11.66 μM, respectively.  Treatment of nude mice bearing A549 tumor xenografts with C188-9 or PL blocked tumor growth and reduced levels of pSTAT3 and mRNA encoding anti-apoptotic proteins.  CONCLUSION:  STAT3 is essential for growth of NSCLC cell lines and tumors and its targeting using C188-9 or PL may be a useful strategy for treatment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSD9LiZcIR8prrIbt-tGZmxfW6udTcc2ebvHSmIeKVmsLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC283mtVansA%253D%253D&md5=135571e0e64ed367ef7bb8778ff1da40</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1016%2Fj.lungcan.2015.09.014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.lungcan.2015.09.014%26sid%3Dliteratum%253Aachs%26aulast%3DLewis%26aufirst%3DK.%2BM.%26aulast%3DBharadwaj%26aufirst%3DU.%26aulast%3DEckols%26aufirst%3DT.%2BK.%26aulast%3DKolosov%26aufirst%3DM.%26aulast%3DKasembeli%26aufirst%3DM.%2BM.%26aulast%3DFridley%26aufirst%3DC.%26aulast%3DSiller%26aufirst%3DR.%26aulast%3DTweardy%26aufirst%3DD.%2BJ.%26atitle%3DSmall-molecule%2520targeting%2520of%2520signal%2520transducer%2520and%2520activator%2520of%2520transcription%2520%2528Stat%2529%25203%2520to%2520treat%2520non-small%2520cell%2520lung%2520cancer%26jtitle%3DLung%2520Cancer%26date%3D2015%26volume%3D90%26spage%3D182%26epage%3D190%26doi%3D10.1016%2Fj.lungcan.2015.09.014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yue, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turkson, J.</span></span> <span> </span><span class="NLM_article-title">Targeting STAT3 in cancer: how successful are we?</span>. <i>Expert Opin. Invest. Drugs</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">45</span>– <span class="NLM_lpage">56</span>, <span class="refDoi"> DOI: 10.1517/13543780802565791</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=10.1517%2F13543780802565791" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=19053881" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhsVGhtbvO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2009&pages=45-56&author=P.+Yueauthor=J.+Turkson&title=Targeting+STAT3+in+cancer%3A+how+successful+are+we%3F&doi=10.1517%2F13543780802565791"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting STAT3 in cancer: how successful are we?</span></div><div class="casAuthors">Yue, Peibin; Turkson, James</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Investigational Drugs</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">45-56</span>CODEN:
                <span class="NLM_cas:coden">EOIDER</span>;
        ISSN:<span class="NLM_cas:issn">1354-3784</span>.
    
            (<span class="NLM_cas:orgname">Informa Healthcare</span>)
        </div><div class="casAbstract">A review.  Aberrant activation of the signal transducer and activator of transcription (STAT)3 occurs in many human tumors.  Moreover, studies utilizing genetic and pharmacol. approaches to modulate constitutive STAT3 activity have provided compelling evidence for the crit. role of aberrant STAT3 activity in malignant transformation and tumor progression, and thereby validated STAT3 as a novel cancer drug target.  This review is intended to be a full coverage of the efforts to develop direct STAT3 inhibitors and will provide a discussion on the inhibitory modalities developed to date.  Review of the literature focused on the modalities and mechanisms that directly target and inhibit the STAT protein or its functions.  While a variety of STAT3 inhibitors have been identified that induce antitumor cell effects in vitro and in vivo, the landscape remains murky.  With a few exceptions, most of the STAT3 inhibitors reported to date have not undergone an in vivo efficacy, pharmacol. or toxicity testing.  Also, there is no evidence, per the published literature of an impending clin. development for the few agents that were reported to exhibit in vivo efficacy.  Overall, there is the need for a reassessment of the ongoing strategies to target STAT3 intended not only for refinement, but also for incorporating some new technologies to strengthen our efforts and ensure the success, sooner, rather than later, of identifying suitable anti-STAT3 agents for development into clin. useful anticancer therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoGo96b-3KjRLVg90H21EOLACvtfcHk0liOYsNJ64QbEw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhsVGhtbvO&md5=e0f4fe796305b7e28ff3020f57b5dc8e</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1517%2F13543780802565791&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1517%252F13543780802565791%26sid%3Dliteratum%253Aachs%26aulast%3DYue%26aufirst%3DP.%26aulast%3DTurkson%26aufirst%3DJ.%26atitle%3DTargeting%2520STAT3%2520in%2520cancer%253A%2520how%2520successful%2520are%2520we%253F%26jtitle%3DExpert%2520Opin.%2520Invest.%2520Drugs%26date%3D2009%26volume%3D18%26spage%3D45%26epage%3D56%26doi%3D10.1517%2F13543780802565791" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kelder, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grootenhuis, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bayada, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delbressine, L. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ploemen, J. P.</span></span> <span> </span><span class="NLM_article-title">Polar molecular surface as a dominating determinant for oral absorption and brain penetration of drugs</span>. <i>Pharm. Res.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">1514</span>– <span class="NLM_lpage">1519</span>, <span class="refDoi"> DOI: 10.1023/A:1015040217741</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=10.1023%2FA%3A1015040217741" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=10554091" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=1%3ACAS%3A528%3ADyaK1MXntVOhtb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=1999&pages=1514-1519&author=J.+Kelderauthor=P.+D.+Grootenhuisauthor=D.+M.+Bayadaauthor=L.+P.+Delbressineauthor=J.+P.+Ploemen&title=Polar+molecular+surface+as+a+dominating+determinant+for+oral+absorption+and+brain+penetration+of+drugs&doi=10.1023%2FA%3A1015040217741"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Polar molecular surface as a dominating determinant for oral absorption and brain penetration of drugs</span></div><div class="casAuthors">Kelder, Jan; Grootenhuis, Peter D. J.; Bayada, Denis M.; Delbressine, Leon P. C.; Ploemen, Jan-Peter</div><div class="citationInfo"><span class="NLM_cas:title">Pharmaceutical Research</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1514-1519</span>CODEN:
                <span class="NLM_cas:coden">PHREEB</span>;
        ISSN:<span class="NLM_cas:issn">0724-8741</span>.
    
            (<span class="NLM_cas:orgname">Kluwer Academic/Plenum Publishers</span>)
        </div><div class="casAbstract">Purpose: To study oral absorption and brain penetration as a function of polar mol. surface area.  Methods: Measured brain penetration data of 45 drug mols. were investigated.  The dynamic polar surface areas were calcd. and correlated with the brain penetration data.  Also the static polar surface areas of 776 orally administered CNS drugs that have reached at least Phase II efficacy studies were calcd.  The same was done for a series of 1590 orally administered non-CNS drugs that have reached at least Phase II efficacy studies.  Results: A linear relationship between brain penetration and dynamic polar surface area (A2) was found (n = 45, R = 0.917, F1,43 = 229).  Brain penetration decreases with increasing polar surface area.  A clear difference between the distribution of the polar surface area of the 776 CNS and 1590 non-CNS drugs was found.  It was deduced that orally active drugs that are transported passively by the transcellular route should not exceed a polar surface area of about 120 A2.  They can be tailored to brain penetration by decreasing the polar surface to <60-70 A2.  This conclusion is supported by the inverse linear relationship between exptl. brain penetration data and the dynamic polar surface area of 45 drug mols.  Conclusions: The polar mol. surface area is a dominating determinant for oral absorption and brain penetration of drugs that are transported by the transcellular route.  This property should be considered in the early phase of drug screening.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpmc3v7TzWM4LVg90H21EOLACvtfcHk0lgxnfIWVFje-A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXntVOhtb8%253D&md5=868fe1e9491dd3b8b92b326d28ee48f9</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1023%2FA%3A1015040217741&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1023%252FA%253A1015040217741%26sid%3Dliteratum%253Aachs%26aulast%3DKelder%26aufirst%3DJ.%26aulast%3DGrootenhuis%26aufirst%3DP.%2BD.%26aulast%3DBayada%26aufirst%3DD.%2BM.%26aulast%3DDelbressine%26aufirst%3DL.%2BP.%26aulast%3DPloemen%26aufirst%3DJ.%2BP.%26atitle%3DPolar%2520molecular%2520surface%2520as%2520a%2520dominating%2520determinant%2520for%2520oral%2520absorption%2520and%2520brain%2520penetration%2520of%2520drugs%26jtitle%3DPharm.%2520Res.%26date%3D1999%26volume%3D16%26spage%3D1514%26epage%3D1519%26doi%3D10.1023%2FA%3A1015040217741" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sharma, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">López-Tarruella, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">García-Saenz, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ward, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Connor, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gómez, H. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prat, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moreno, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jerez-Gilarranz, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barnadas, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Picornell, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Del Monte-Millán, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gonzalez-Rivera, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Massarrah, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pelaez-Lorenzo, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Palomero, M. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">González Del Val, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cortes, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fuentes Rivera, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bretel Morales, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Márquez-Rodas, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perou, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wagner, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mammen, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McGinness, M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klemp, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amin, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fabian, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heldstab, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Godwin, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jensen, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kimler, B. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khan, Q. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, M.</span></span> <span> </span><span class="NLM_article-title">Efficacy of neoadjuvant carboplatin plus docetaxel in triple-negative breast cancer: combined analysis of two cohorts</span>. <i>Clin. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">649</span>– <span class="NLM_lpage">657</span>, <span class="refDoi"> DOI: 10.1158/1078-0432.CCR-16-0162</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=10.1158%2F1078-0432.CCR-16-0162" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=27301700" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvVSjsLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2017&pages=649-657&author=P.+Sharmaauthor=S.+L%C3%B3pez-Tarruellaauthor=J.+A.+Garc%C3%ADa-Saenzauthor=C.+Wardauthor=C.+S.+Connorauthor=H.+L.+G%C3%B3mezauthor=A.+Pratauthor=F.+Morenoauthor=Y.+Jerez-Gilarranzauthor=A.+Barnadasauthor=A.+C.+Picornellauthor=M.+Del+Monte-Mill%C3%A1nauthor=M.+Gonzalez-Riveraauthor=T.+Massarrahauthor=B.+Pelaez-Lorenzoauthor=M.+I.+Palomeroauthor=R.+Gonz%C3%A1lez+Del+Valauthor=J.+Cortesauthor=H.+Fuentes+Riveraauthor=D.+Bretel+Moralesauthor=I.+M%C3%A1rquez-Rodasauthor=C.+M.+Perouauthor=J.+L.+Wagnerauthor=J.+M.+Mammenauthor=M.+K.+McGinnessauthor=J.+R.+Klempauthor=A.+L.+Aminauthor=C.+J.+Fabianauthor=J.+Heldstabauthor=A.+K.+Godwinauthor=R.+A.+Jensenauthor=B.+F.+Kimlerauthor=Q.+J.+Khanauthor=M.+Martin&title=Efficacy+of+neoadjuvant+carboplatin+plus+docetaxel+in+triple-negative+breast+cancer%3A+combined+analysis+of+two+cohorts&doi=10.1158%2F1078-0432.CCR-16-0162"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Efficacy of Neoadjuvant Carboplatin plus Docetaxel in Triple-Negative Breast Cancer: Combined Analysis of Two Cohorts</span></div><div class="casAuthors">Sharma, Priyanka; Lopez-Tarruella, Sara; Garcoia-Saenz, Jose Angel; Ward, Claire; Connor, Carol S.; Goiomez, Henry L.; Prat, Aleix; Moreno, Fernando; Jerez-Gilarranz, Yolanda; Barnadas, Augusti; Picornell, Antoni C.; del Monte-Milloioan, Maria; Gonzalez-Rivera, Milagros; Massarrah, Tatiana; Pelaez-Lorenzo, Beatriz; Palomero, Maria Isabel; Gonzalez del Val, Ricardo; Cortes, Javier; Fuentes Rivera, Hugo; Bretel Morales, Denisse; Marquez-Rodas, Ivan; Perou, Charles M.; Wagner, Jamie L.; Mammen, Joshua M. V.; McGinness, Marilee K.; Klemp, Jennifer R.; Amin, Amanda L.; Fabian, Carol J.; Heldstab, Jaimie; Godwin, Andrew K.; Jensen, Roy A.; Kimler, Bruce F.; Khan, Qamar J.; Martin, Miguel</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">649-657</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Purpose: Recent studies demonstrate that addn. of neoadjuvant (NA) carboplatin to anthracycline/taxane chemotherapy improves pathol. complete response (pCR) in triple-neg. breast cancer (TNBC).  Effectiveness of anthracycline-free platinum combinations in TNBC is not well known.  Here, we report efficacy of NA carboplatin + docetaxel (CbD) in TNBC.  Exptl. Design: The study population includes 190 patients with stage I-III TNBC treated uniformly on two independent prospective cohorts.  All patients were prescribed NA chemotherapy regimen of carboplatin (AUC 6) + docetaxel (75 mg/m2) given every 21 days × 6 cycles. pCR (no evidence of invasive tumor in the breast and axilla) and residual cancer burden (RCB) were evaluated.  Results: Among 190 patients, median tumor size was 35 mm, 52% were lymph node pos., and 16% had germline BRCA1/2 mutation.  The overall pCR and RCB 0 + 1 rates were 55% and 68%, resp. pCRs in patients with BRCA-assocd. and wild-type TNBC were 59% and 56%, resp. (P = 0.83).  On multivariable anal., stage III disease was the only factor assocd. with a lower likelihood of achieving a pCR.  Twenty-one percent and 7% of patients, resp., experienced at least one grade 3 or 4 adverse event.  Conclusions: The CbD regimen was well tolerated and yielded high pCR rates in both BRCA-assocd. and wild-type TNBC.  These results are comparable with pCR achieved with the addn. of carboplatin to anthracycline-taxane chemotherapy.  Our study adds to the existing data on the efficacy of platinum agents in TNBC and supports further exploration of the CbD regimen in randomized studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrO1X6U5BphXrVg90H21EOLACvtfcHk0lgxnfIWVFje-A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvVSjsLg%253D&md5=c3b5266e1a68b482ccfa9e30049ce215</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1158%2F1078-0432.CCR-16-0162&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1078-0432.CCR-16-0162%26sid%3Dliteratum%253Aachs%26aulast%3DSharma%26aufirst%3DP.%26aulast%3DL%25C3%25B3pez-Tarruella%26aufirst%3DS.%26aulast%3DGarc%25C3%25ADa-Saenz%26aufirst%3DJ.%2BA.%26aulast%3DWard%26aufirst%3DC.%26aulast%3DConnor%26aufirst%3DC.%2BS.%26aulast%3DG%25C3%25B3mez%26aufirst%3DH.%2BL.%26aulast%3DPrat%26aufirst%3DA.%26aulast%3DMoreno%26aufirst%3DF.%26aulast%3DJerez-Gilarranz%26aufirst%3DY.%26aulast%3DBarnadas%26aufirst%3DA.%26aulast%3DPicornell%26aufirst%3DA.%2BC.%26aulast%3DDel%2BMonte-Mill%25C3%25A1n%26aufirst%3DM.%26aulast%3DGonzalez-Rivera%26aufirst%3DM.%26aulast%3DMassarrah%26aufirst%3DT.%26aulast%3DPelaez-Lorenzo%26aufirst%3DB.%26aulast%3DPalomero%26aufirst%3DM.%2BI.%26aulast%3DGonz%25C3%25A1lez%2BDel%2BVal%26aufirst%3DR.%26aulast%3DCortes%26aufirst%3DJ.%26aulast%3DFuentes%2BRivera%26aufirst%3DH.%26aulast%3DBretel%2BMorales%26aufirst%3DD.%26aulast%3DM%25C3%25A1rquez-Rodas%26aufirst%3DI.%26aulast%3DPerou%26aufirst%3DC.%2BM.%26aulast%3DWagner%26aufirst%3DJ.%2BL.%26aulast%3DMammen%26aufirst%3DJ.%2BM.%26aulast%3DMcGinness%26aufirst%3DM.%2BK.%26aulast%3DKlemp%26aufirst%3DJ.%2BR.%26aulast%3DAmin%26aufirst%3DA.%2BL.%26aulast%3DFabian%26aufirst%3DC.%2BJ.%26aulast%3DHeldstab%26aufirst%3DJ.%26aulast%3DGodwin%26aufirst%3DA.%2BK.%26aulast%3DJensen%26aufirst%3DR.%2BA.%26aulast%3DKimler%26aufirst%3DB.%2BF.%26aulast%3DKhan%26aufirst%3DQ.%2BJ.%26aulast%3DMartin%26aufirst%3DM.%26atitle%3DEfficacy%2520of%2520neoadjuvant%2520carboplatin%2520plus%2520docetaxel%2520in%2520triple-negative%2520breast%2520cancer%253A%2520combined%2520analysis%2520of%2520two%2520cohorts%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2017%26volume%3D23%26spage%3D649%26epage%3D657%26doi%3D10.1158%2F1078-0432.CCR-16-0162" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hu, X. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, B. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ragaz, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, B. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Teng, Y. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tong, Z. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pan, Y. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yin, Y. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, C. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, Z. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lou, G. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gu, Y. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shao, Z. M.</span></span> <span> </span><span class="NLM_article-title">Cisplatin plus gemcitabine versus paclitaxel plus gemcitabine as first-line therapy for metastatic triple-negative breast cancer (CBCSG006): a randomised, open-label, multicentre, phase 3 trial</span>. <i>Lancet Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">436</span>– <span class="NLM_lpage">46</span>, <span class="refDoi"> DOI: 10.1016/S1470-2045(15)70064-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=10.1016%2FS1470-2045%2815%2970064-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=25795409" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=1%3ACAS%3A528%3ADC%252BC2MXkvVWgsLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2015&pages=436-46&author=X.+C.+Huauthor=J.+Zhangauthor=B.+H.+Xuauthor=L.+Caiauthor=J.+Ragazauthor=Z.+H.+Wangauthor=B.+Y.+Wangauthor=Y.+E.+Tengauthor=Z.+S.+Tongauthor=Y.+Y.+Panauthor=Y.+M.+Yinauthor=C.+P.+Wuauthor=Z.+F.+Jiangauthor=X.+J.+Wangauthor=G.+Y.+Louauthor=D.+G.+Liuauthor=J.+F.+Fengauthor=J.+F.+Luoauthor=K.+Sunauthor=Y.+J.+Guauthor=J.+Wuauthor=Z.+M.+Shao&title=Cisplatin+plus+gemcitabine+versus+paclitaxel+plus+gemcitabine+as+first-line+therapy+for+metastatic+triple-negative+breast+cancer+%28CBCSG006%29%3A+a+randomised%2C+open-label%2C+multicentre%2C+phase+3+trial&doi=10.1016%2FS1470-2045%2815%2970064-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Cisplatin plus gemcitabine versus paclitaxel plus gemcitabine as first-line therapy for metastatic triple-negative breast cancer (CBCSG006): a randomized, open-label, multicentre, phase 3 trial</span></div><div class="casAuthors">Hu, Xi-Chun; Zhang, Jian; Xu, Bing-He; Cai, Li; Ragaz, Joseph; Wang, Zhong-Hua; Wang, Bi-Yun; Teng, Yue-E.; Tong, Zhong-Sheng; Pan, Yue-Yin; Yin, Yong-Mei; Wu, Chang-Ping; Jiang, Ze-Fei; Wang, Xiao-Jia; Lou, Gu-Yin; Liu, Dong-Geng; Feng, Ji-Feng; Luo, Jian-Feng; Sun, Kang; Gu, Ya-Jia; Wu, Jiong; Shao, Zhi-Min</div><div class="citationInfo"><span class="NLM_cas:title">Lancet Oncology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">436-446</span>CODEN:
                <span class="NLM_cas:coden">LOANBN</span>;
        ISSN:<span class="NLM_cas:issn">1470-2045</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Background: Platinum chemotherapy has a role in the treatment of metastatic triple-neg. breast cancer but its full potential has probably not yet been reached.  We assessed whether a cisplatin plus gemcitabine regimen was non-inferior to or superior to paclitaxel plus gemcitabine as first-line therapy for patients with metastatic triple-neg. breast cancer.  Methods: For this open-label, randomized, phase 3, hybrid-designed trial undertaken at 12 institutions or hospitals in China, we included Chinese patients aged 18-70 years with previously untreated, histol. confirmed metastatic triple-neg. breast cancer, and an ECOG performance status of 0-1.  These patients were randomly assigned (1:1) to receive either cisplatin plus gemcitabine (cisplatin 75 mg/m2 on day 1 and gemcitabine 1250 mg/m2 on days 1 and 8) or paclitaxel plus gemcitabine (paclitaxel 175 mg/m2 on day 1 and gemcitabine 1250 mg/m2 on days 1 and 8) given i.v. every 3 wk for a max. of eight cycles.  Randomisation was done centrally via an interactive web response system using block randomization with a size of eight, with no stratification factors.  Patients and investigator were aware of group assignments.  The primary endpoint was progression-free survival and analyses were based on all patients who received at least one dose of assigned treatment.  The margin used to establish non-inferiority was 1·2.  If non-inferiority of cisplatin plus gemcitabine compared with paclitaxel plus gemcitabine was achieved, we would then test for superiority.  The trial is registered with ClinicalTrials.gov, no. NCT01287624.  Findings: From Jan 14, 2011, to Nov 14, 2013, 240 patients were assessed for eligibility and randomly assigned to treatment (120 in the cisplatin plus gemcitabine group and 120 in the paclitaxel plus gemcitabine group). 236 patients received at least one dose of assigned chemotherapy and were included in the modified intention-to-treat anal. (118 per group).  After a median follow-up of 16·3 mo IQR 14·4-26·8 in the cisplatin plus gemcitabine group and 15·9 mo (10·7-25·4) in the paclitaxel plus gemcitabine group, the hazard ratio for progression-free survival was 0·692 95% CI 0·523-0·915; pnon-inferiority<0·0001, psuperiority=0·009, thus cisplatin plus gemcitabine was both non-inferior to and superior to paclitaxel plus gemcitabine. Median progression-free survival was 7·73 mo (95% CI 6·16-9·30) in the cisplatin plus gemcitabine group and 6·47 mo 5·76-7·18 in the paclitaxel plus gemcitabine group.  Grade 3 or 4 adverse events that differed significantly between the two groups included nausea (eight [7%] vs one [<1%]), vomiting (13 [11%] vs one [<1%]), musculoskeletal pain (none vs ten [8%]), anemia (39 [33%] vs six [5%]), and thrombocytopenia (38 [32%] vs three [3%]), for the cisplatin plus gemcitabine compared with the paclitaxel plus gemcitabine groups, resp.  In addn., patients in the cisplatin plus gemcitabine group had significantly fewer events of grade 1-4 alopecia (12 [10%] vs 42 [36%]) and peripheral neuropathy (27 [23%] vs 60 [51%]), but more grade 1-4 anorexia (33 [28%] vs 10 [8%]), constipation (29 [25%] vs 11 [9%]), hypomagnesemia (27 [23%] vs five [4%]), and hypokalemia (10 [8%] vs two [2%]).  Serious drug-related adverse events were seen in three patients in the paclitaxel plus gemcitabine group (interstitial pneumonia, anaphylaxis, and severe neutropenia) and four in the cisplatin plus gemcitabine group (pathol. bone fracture, thrombocytopenia with s.c. hemorrhage, severe anemia, and cardiogenic syncope).  There were no treatment-related deaths.  Interpretation: Cisplatin plus gemcitabine could be an alternative or even the preferred first-line chemotherapy strategy for patients with metastatic triple-neg. breast cancer.  Shanghai Natural Science Foundation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoak6TclTwMVbVg90H21EOLACvtfcHk0lgxnfIWVFje-A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXkvVWgsLs%253D&md5=0de66bedf4f2ad138dbe89c99958433b</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1016%2FS1470-2045%2815%2970064-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1470-2045%252815%252970064-1%26sid%3Dliteratum%253Aachs%26aulast%3DHu%26aufirst%3DX.%2BC.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DXu%26aufirst%3DB.%2BH.%26aulast%3DCai%26aufirst%3DL.%26aulast%3DRagaz%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DZ.%2BH.%26aulast%3DWang%26aufirst%3DB.%2BY.%26aulast%3DTeng%26aufirst%3DY.%2BE.%26aulast%3DTong%26aufirst%3DZ.%2BS.%26aulast%3DPan%26aufirst%3DY.%2BY.%26aulast%3DYin%26aufirst%3DY.%2BM.%26aulast%3DWu%26aufirst%3DC.%2BP.%26aulast%3DJiang%26aufirst%3DZ.%2BF.%26aulast%3DWang%26aufirst%3DX.%2BJ.%26aulast%3DLou%26aufirst%3DG.%2BY.%26aulast%3DLiu%26aufirst%3DD.%2BG.%26aulast%3DFeng%26aufirst%3DJ.%2BF.%26aulast%3DLuo%26aufirst%3DJ.%2BF.%26aulast%3DSun%26aufirst%3DK.%26aulast%3DGu%26aufirst%3DY.%2BJ.%26aulast%3DWu%26aufirst%3DJ.%26aulast%3DShao%26aufirst%3DZ.%2BM.%26atitle%3DCisplatin%2520plus%2520gemcitabine%2520versus%2520paclitaxel%2520plus%2520gemcitabine%2520as%2520first-line%2520therapy%2520for%2520metastatic%2520triple-negative%2520breast%2520cancer%2520%2528CBCSG006%2529%253A%2520a%2520randomised%252C%2520open-label%252C%2520multicentre%252C%2520phase%25203%2520trial%26jtitle%3DLancet%2520Oncol.%26date%3D2015%26volume%3D16%26spage%3D436%26epage%3D46%26doi%3D10.1016%2FS1470-2045%2815%2970064-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lipinski, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lombardo, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dominy, B. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feeney, P. J.</span></span> <span> </span><span class="NLM_article-title">Experimental and computational approaches to estimate solubility and permeability in drug discovery and development settings</span>. <i>Adv. Drug Delivery Rev.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>46</i></span>,  <span class="NLM_fpage">3</span>– <span class="NLM_lpage">26</span>, <span class="refDoi"> DOI: 10.1016/S0169-409X(00)00129-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=10.1016%2FS0169-409X%2800%2900129-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=11259830" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=1%3ACAS%3A528%3ADC%252BD3MXitVOhs7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2001&pages=3-26&author=C.+A.+Lipinskiauthor=F.+Lombardoauthor=B.+W.+Dominyauthor=P.+J.+Feeney&title=Experimental+and+computational+approaches+to+estimate+solubility+and+permeability+in+drug+discovery+and+development+settings&doi=10.1016%2FS0169-409X%2800%2900129-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Experimental and computational approaches to estimate solubility and permeability in drug discovery and development settings</span></div><div class="casAuthors">Lipinski, C. A.; Lombardo, F.; Dominy, B. W.; Feeney, P. J.</div><div class="citationInfo"><span class="NLM_cas:title">Advanced Drug Delivery Reviews</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">1-3</span>),
    <span class="NLM_cas:pages">3-26</span>CODEN:
                <span class="NLM_cas:coden">ADDREP</span>;
        ISSN:<span class="NLM_cas:issn">0169-409X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ireland Ltd.</span>)
        </div><div class="casAbstract">A review with 50 refs.  Exptl. and computational approaches to est. soly. and permeability in discovery and development settings are described.  In the discovery setting 'the rule of 5' predicts that poor absorption or permeation is more likely when there are more than 5 H-bond donors, 10 H-bond acceptors, the mol. wt. (MWT) is greater than 500 and the calcd. Log P (CLogP) is greater than 5 (or MlogP >4.15).  Computational methodol. for the rule-based Moriguchi Log P (MLogP) calcn. is described.  Turbidimetric soly. measurement is described and applied to known drugs.  High throughput screening (HTS) leads tend to have higher MWT and Log P and lower turbidimetric soly. than leads in the pre-HTS era.  In the development setting, soly. calcns. focus on exact value prediction and are difficult because of polymorphism.  Recent work on linear free energy relationships and Log P approaches are critically reviewed.  Useful predictions are possible in closely related analog series when coupled with exptl. thermodn. soly. measurements.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoYDqa02g16urVg90H21EOLACvtfcHk0lhXCJqrcY65vw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXitVOhs7o%253D&md5=c60bb89da68f051c0ee7ac4c0468a0e4</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1016%2FS0169-409X%2800%2900129-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0169-409X%252800%252900129-0%26sid%3Dliteratum%253Aachs%26aulast%3DLipinski%26aufirst%3DC.%2BA.%26aulast%3DLombardo%26aufirst%3DF.%26aulast%3DDominy%26aufirst%3DB.%2BW.%26aulast%3DFeeney%26aufirst%3DP.%2BJ.%26atitle%3DExperimental%2520and%2520computational%2520approaches%2520to%2520estimate%2520solubility%2520and%2520permeability%2520in%2520drug%2520discovery%2520and%2520development%2520settings%26jtitle%3DAdv.%2520Drug%2520Delivery%2520Rev.%26date%3D2001%26volume%3D46%26spage%3D3%26epage%3D26%26doi%3D10.1016%2FS0169-409X%2800%2900129-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Alonso-Valenteen, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pacheco, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Srinivas, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rentsendorj, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lubow, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sims, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miao, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mikhael, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hwang, J. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abrol, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Medina Kauwe, L. K.</span></span> <span> </span><span class="NLM_article-title">HER3-targeted protein chimera forms endosomolytic capsomeres and self-assembles into stealth nucleocapsids for systemic tumor homing of RNA interference in vivo</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">11020</span>– <span class="NLM_lpage">11043</span>, <span class="refDoi"> DOI: 10.1093/nar/gkz900</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=10.1093%2Fnar%2Fgkz900" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=31617560" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhtVGqsr7K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2019&pages=11020-11043&author=F.+Alonso-Valenteenauthor=S.+Pachecoauthor=D.+Srinivasauthor=A.+Rentsendorjauthor=D.+Chuauthor=J.+Lubowauthor=J.+Simsauthor=T.+Miaoauthor=S.+Mikhaelauthor=J.+Y.+Hwangauthor=R.+Abrolauthor=L.+K.+Medina+Kauwe&title=HER3-targeted+protein+chimera+forms+endosomolytic+capsomeres+and+self-assembles+into+stealth+nucleocapsids+for+systemic+tumor+homing+of+RNA+interference+in+vivo&doi=10.1093%2Fnar%2Fgkz900"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">HER3-targeted protein chimera forms endosomolytic capsomeres and self-assembles into stealth nucleocapsids for systemic tumor homing of RNA interference in vivo</span></div><div class="casAuthors">Alonso-Valenteen, Felix; Pacheco, Sayuri; Srinivas, Dustin; Rentsendorj, Altan; Chu, David; Lubow, Jay; Sims, Jessica; Miao, Tianxin; Mikhael, Simoun; Hwang, Jae Youn; Abrol, Ravinder; Kauwe, Lali K. Medina</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">11020-11043</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">1362-4962</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">RNA interference represents a potent intervention for cancer treatment but requires a robust delivery agent for transporting gene-modulating mols., such as small interfering RNAs (siRNAs).  Although numerous mol. approaches for siRNA delivery are adequate in vitro, delivery to therapeutic targets in vivo is limited by payload integrity, cell targeting, efficient cell uptake, and membrane penetration.  We constructed nonviral biomaterials to transport small nucleic acids to cell targets, including tumor cells, on the basis of the self-assembling and cell-penetrating activities of the adenovirus capsid penton base.  Our recombinant penton base chimera contains polypeptide domains designed for noncovalent assembly with anionic mols. and tumor homing.  Here, structural modeling, mol. dynamics simulations, and functional assays suggest that it forms pentameric units resembling viral capsomeres that assemble into larger capsid-like structures when combined with siRNA cargo.  Pentamerization forms a barrel lined with charged residues mediating pH-responsive dissocn. and exposing masked domains, providing insight on the endosomolytic mechanism.  The therapeutic impact was examd. on tumors expressing high levels of HER3/ErbB3 that are resistant to clin. inhibitors.  Our findings suggest that our construct may utilize ligand mimicry to avoid host attack and target the siRNA to HER3+ tumors by forming multivalent capsid-like structures.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo0Nu1sOinyv7Vg90H21EOLACvtfcHk0lhXCJqrcY65vw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhtVGqsr7K&md5=d8a9f4539ea7889d4088171c64995406</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkz900&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkz900%26sid%3Dliteratum%253Aachs%26aulast%3DAlonso-Valenteen%26aufirst%3DF.%26aulast%3DPacheco%26aufirst%3DS.%26aulast%3DSrinivas%26aufirst%3DD.%26aulast%3DRentsendorj%26aufirst%3DA.%26aulast%3DChu%26aufirst%3DD.%26aulast%3DLubow%26aufirst%3DJ.%26aulast%3DSims%26aufirst%3DJ.%26aulast%3DMiao%26aufirst%3DT.%26aulast%3DMikhael%26aufirst%3DS.%26aulast%3DHwang%26aufirst%3DJ.%2BY.%26aulast%3DAbrol%26aufirst%3DR.%26aulast%3DMedina%2BKauwe%26aufirst%3DL.%2BK.%26atitle%3DHER3-targeted%2520protein%2520chimera%2520forms%2520endosomolytic%2520capsomeres%2520and%2520self-assembles%2520into%2520stealth%2520nucleocapsids%2520for%2520systemic%2520tumor%2520homing%2520of%2520RNA%2520interference%2520in%2520vivo%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2019%26volume%3D47%26spage%3D11020%26epage%3D11043%26doi%3D10.1093%2Fnar%2Fgkz900" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">H, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herrmann, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buettner, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jove, R.</span></span> <span> </span><span class="NLM_article-title">Revisiting Stat3 signalling in cancer: new and unexpected biological functions</span>. <i>Nat. Rev. Cancer.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">736</span>– <span class="NLM_lpage">746</span>, <span class="refDoi"> DOI: 10.1038/nrc3818</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=10.1038%2Fnrc3818" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=25342631" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvVSmtLnF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2014&pages=736-746&author=H.+Yuauthor=L.+Hauthor=A.+Herrmannauthor=R.+Buettnerauthor=R.+Jove&title=Revisiting+Stat3+signalling+in+cancer%3A+new+and+unexpected+biological+functions&doi=10.1038%2Fnrc3818"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Revisiting STAT3 signalling in cancer: new and unexpected biological functions</span></div><div class="casAuthors">Yu, Hua; Lee, Heehyoung; Herrmann, Andreas; Buettner, Ralf; Jove, Richard</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">736-746</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The Janus kinases (JAKs) and signal transducer and activator of transcription (STAT) proteins, particularly STAT3, are among the most promising new targets for cancer therapy.  In addn. to interleukin-6 (IL-6) and its family members, multiple pathways, including G-protein-coupled receptors (GPCRs), Toll-like receptors (TLRs) and microRNAs were recently identified to regulate JAK-STAT signalling in cancer.  Well known for its role in tumor cell proliferation, survival, invasion and immunosuppression, JAK-STAT3 signalling also promotes cancer through inflammation, obesity, stem cells and the pre-metastatic niche.  In addn. to its established role as a transcription factor in cancer, STAT3 regulates mitochondrion functions, as well as gene expression through epigenetic mechanisms.  Newly identified regulators and functions of JAK-STAT3 in tumors are important targets for potential therapeutic strategies in the treatment of cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq3PGL62I5KvrVg90H21EOLACvtfcHk0lhXCJqrcY65vw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvVSmtLnF&md5=355a07303813d7ecadb4b085c33c5e0a</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1038%2Fnrc3818&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc3818%26sid%3Dliteratum%253Aachs%26aulast%3DYu%26aufirst%3DH.%26aulast%3DH%26aufirst%3DL.%26aulast%3DHerrmann%26aufirst%3DA.%26aulast%3DBuettner%26aufirst%3DR.%26aulast%3DJove%26aufirst%3DR.%26atitle%3DRevisiting%2520Stat3%2520signalling%2520in%2520cancer%253A%2520new%2520and%2520unexpected%2520biological%2520functions%26jtitle%3DNat.%2520Rev.%2520Cancer.%26date%3D2014%26volume%3D14%26spage%3D736%26epage%3D746%26doi%3D10.1038%2Fnrc3818" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Avdeef, A.</span></span> <span> </span><span class="NLM_article-title">Physicochemical profiling (solubility, permeability and charge state)</span>. <i>Curr. Top. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">277</span>– <span class="NLM_lpage">351</span>, <span class="refDoi"> DOI: 10.2174/1568026013395100</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=10.2174%2F1568026013395100" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=11899112" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=1%3ACAS%3A528%3ADC%252BD3MXms1Khs7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2001&pages=277-351&author=A.+Avdeef&title=Physicochemical+profiling+%28solubility%2C+permeability+and+charge+state%29&doi=10.2174%2F1568026013395100"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Physicochemical profiling (solubility, permeability and charge state)</span></div><div class="casAuthors">Avdeef, Alex</div><div class="citationInfo"><span class="NLM_cas:title">Current Topics in Medicinal Chemistry (Hilversum, Netherlands)</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">277-351</span>CODEN:
                <span class="NLM_cas:coden">CTMCCL</span>;
        ISSN:<span class="NLM_cas:issn">1568-0266</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  About 30% of drug candidate mols. are rejected due to pharmacokinetic-related failures.  When poor pharmaceutical properties are discovered in development, the costs of bringing a potent but poorly absorbable mol. to a product stage by "formulation" can become very high.  Fast and reliable in vitro prediction strategies are needed to filter out problematic mols. at the earliest stages of discovery.  This review will consider recent developments in physicochem. profiling used to identify candidate mols. with phys. properties related to good oral absorption.  Poor soly. and poor permeability account for many PK failures.  FDA's Biopharmaceutics Classification System (BCS) is an attempt to rationalize the crit. components related to oral absorption.  The core idea in the BCS is an in vitro transport model, centrally embracing permeability and soly., with qualifications related to pH and dissoln.  The objective of the BCS is to predict in vivo performance of drug products from in vitro measurements of permeability and soly.  In principle, the framework of the BCS could serve the interests of the earliest stages of discovery research.  The BCS can be rationalized by considering Fick's first law, applied to membranes.  When mols. are introduced on one side of a lipid membrane barrier (e.g., epithelial cell wall) and no such mols. are on the other side, passive diffusion will drive the mols. across the membrane.  When certain simplifying assumptions are made, the flux equation in Fick's law reduces simply to a product of permeability and soly.  Many other measurable properties are closely related to permeability and soly.  Permeability (Pe) is a kinetic parameter related to lipophilicity (as indicated by the partition and distribution coeffs., log P and log D).  Retention (R) of lipophilic mols. by the membrane (which is related to lipophilicity and may predict. PK vols. of distribution) influences the characterization of permeability.  Furthermore, strong drug interactions with serum proteins can influence permeability.  The unstirred water layer on both sides of the membrane barrier can impose limits on permeability.  Soly. (S) is a thermodn. parameter, and is closely related to dissoln., a kinetic parameter.  The unstirred water layer on the surfaces of suspended solids imposes limits on dissoln.  Bile acids affect both soly. and dissoln., by a micellization effect.  For ionizable mols., pH plays a crucial role.  The charge state that a mol. exhibits at a particular pH is characterized by the ionization const. (pKa) of the mol.  Buffers effect pH gradients in the unstirred water layers, which can dramatically affect both permeability and dissoln. of ionizable mols.  In this review, we will focus on the emerging instrumental methods for the measurement of the physicochem. parameters Pe, S, pKa, R, log P, and log D (and their pH-profiles).  These physicochem. profiles can be valuable tools for the medicinal chemists, aiding in the prediction of in vivo oral absorption.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqUuP6xv9GKXLVg90H21EOLACvtfcHk0lgObVIhqnEDCQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXms1Khs7o%253D&md5=df94226deaa13e5ade1caec003ae6733</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.2174%2F1568026013395100&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1568026013395100%26sid%3Dliteratum%253Aachs%26aulast%3DAvdeef%26aufirst%3DA.%26atitle%3DPhysicochemical%2520profiling%2520%2528solubility%252C%2520permeability%2520and%2520charge%2520state%2529%26jtitle%3DCurr.%2520Top.%2520Med.%2520Chem.%26date%3D2001%26volume%3D1%26spage%3D277%26epage%3D351%26doi%3D10.2174%2F1568026013395100" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Martinez, M. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amidon, G. L.</span></span> <span> </span><span class="NLM_article-title">A mechanistic approach to understanding the factors affecting drug absorption: a review of fundamentals</span>. <i>J. Clin. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>42</i></span>,  <span class="NLM_fpage">620</span>– <span class="NLM_lpage">43</span>, <span class="refDoi"> DOI: 10.1177/00970002042006005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=10.1177%2F00970002042006005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=12043951" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=1%3ACAS%3A528%3ADC%252BD38Xks1OgsLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2002&pages=620-43&author=M.+N.+Martinezauthor=G.+L.+Amidon&title=A+mechanistic+approach+to+understanding+the+factors+affecting+drug+absorption%3A+a+review+of+fundamentals&doi=10.1177%2F00970002042006005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">A mechanistic approach to understanding the factors affecting drug absorption: A review of fundamentals</span></div><div class="casAuthors">Martinez, Marilyn N.; Amidon, Gordon L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Clinical Pharmacology</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">620-643</span>CODEN:
                <span class="NLM_cas:coden">JCPCBR</span>;
        ISSN:<span class="NLM_cas:issn">0091-2700</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">A review.  This article provides an overview of the patient-specific and drug-specific variables that can affect drug absorption following oral product administration.  The oral absorption of any chem. entity reflects a complex spectrum of events.  Factors influencing product bioavailability include drug soly., permeability, and the rate of in vivo dissoln.  In this regard, the Biopharmaceutics Classification System has proven to be an important tool for predicting compds. likely to be assocd. with bioavailability problems.  It also helps in identifying those factors that may alter the rate and extent of drug absorption.  Product bioavailability can also be markedly influenced by patient attributes such as the integrity of the gastrointestinal tract, physiol. status, site of drug absorption, membrane transporters, presystemic drug metab. (intrinsic variables), and extrinsic variables such as the effect of food or concomitant medication.  Through an awareness of a drug's physicochem. properties and the physiol. processes affecting drug absorption, the skilled pharmaceutical scientist can develop formulations that will maximize product availability.  By appreciating the potential impact of patient physiol. status, phenotype, age, gender, and lifestyle, dosing regimens can be tailored to better meet the needs of the individual patient.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr7090GVIXnsrVg90H21EOLACvtfcHk0lgObVIhqnEDCQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38Xks1OgsLc%253D&md5=65abdd33877aa4cd9295c594a338584e</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1177%2F00970002042006005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F00970002042006005%26sid%3Dliteratum%253Aachs%26aulast%3DMartinez%26aufirst%3DM.%2BN.%26aulast%3DAmidon%26aufirst%3DG.%2BL.%26atitle%3DA%2520mechanistic%2520approach%2520to%2520understanding%2520the%2520factors%2520affecting%2520drug%2520absorption%253A%2520a%2520review%2520of%2520fundamentals%26jtitle%3DJ.%2520Clin.%2520Pharmacol.%26date%3D2002%26volume%3D42%26spage%3D620%26epage%3D43%26doi%3D10.1177%2F00970002042006005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Di, L.</span>; <span class="NLM_string-name">Kerns, E.</span></span> <i>Drug-like Properties: Concepts, Structure Design and Methods from ADME to Toxicity Optimization</i>, <span class="NLM_edition">2</span>nd ed.; <span class="NLM_publisher-name">Elsevier</span>: <span class="NLM_publisher-loc">Cambridge, MA</span>, <span class="NLM_year">2016</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2016&author=L.+Di&author=E.+Kerns&title=Drug-like+Properties%3A+Concepts%2C+Structure+Design+and+Methods+from+ADME+to+Toxicity+Optimization"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DDi%26aufirst%3DL.%26btitle%3DDrug-like%2520Properties%253A%2520Concepts%252C%2520Structure%2520Design%2520and%2520Methods%2520from%2520ADME%2520to%2520Toxicity%2520Optimization%26pub%3DElsevier%26date%3D2016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chaturvedi, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Decker, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Odinecs, A.</span></span> <span> </span><span class="NLM_article-title">Prediction of pharmacokinetic properties using experimental approaches during early drug discovery</span>. <i>Curr. Opin. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">452</span>– <span class="NLM_lpage">463</span>, <span class="refDoi"> DOI: 10.1016/S1367-5931(00)00228-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=10.1016%2FS1367-5931%2800%2900228-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=11470610" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=1%3ACAS%3A528%3ADC%252BD3MXltlyqsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2001&pages=452-463&author=P.+R.+Chaturvediauthor=C.+J.+Deckerauthor=A.+Odinecs&title=Prediction+of+pharmacokinetic+properties+using+experimental+approaches+during+early+drug+discovery&doi=10.1016%2FS1367-5931%2800%2900228-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Prediction of pharmacokinetic properties using experimental approaches during early drug discovery</span></div><div class="casAuthors">Chaturvedi, P. R.; Decker, C. J.; Odinecs, A.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Chemical Biology</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">452-463</span>CODEN:
                <span class="NLM_cas:coden">COCBF4</span>;
        ISSN:<span class="NLM_cas:issn">1367-5931</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ltd.</span>)
        </div><div class="casAbstract">A review with refs.  There has been a significant increase in the no. of compds. synthesized in early drug-discovery programs with the advances in combinatorial chem. and high-throughput biol. screening efforts.  Various in silico, in vitro and in situ approaches have been described in literature that achieve higher throughput pharmacokinetic screening.  In silico methodologies have mainly attempted to quantify the prospects of oral absorption of compds. based upon their physico-chem. properties.  There is a greater availability of in vitro and in situ approaches to screen compds. for intestinal permeability (as a surrogate for absorption) and metabolic stability (as a surrogate for clearance).  More recent modifications of the in vitro and in situ approaches to assess the potential of absorption and metab. have enabled a higher throughput and an ability to correlate better with in vivo pharmacokinetics of compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrAQSfVKDr8KrVg90H21EOLACvtfcHk0lgObVIhqnEDCQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXltlyqsb4%253D&md5=36956032f958260f71e20d2a0939d432</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1016%2FS1367-5931%2800%2900228-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1367-5931%252800%252900228-3%26sid%3Dliteratum%253Aachs%26aulast%3DChaturvedi%26aufirst%3DP.%2BR.%26aulast%3DDecker%26aufirst%3DC.%2BJ.%26aulast%3DOdinecs%26aufirst%3DA.%26atitle%3DPrediction%2520of%2520pharmacokinetic%2520properties%2520using%2520experimental%2520approaches%2520during%2520early%2520drug%2520discovery%26jtitle%3DCurr.%2520Opin.%2520Chem.%2520Biol.%26date%3D2001%26volume%3D5%26spage%3D452%26epage%3D463%26doi%3D10.1016%2FS1367-5931%2800%2900228-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Guha, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dexheimer, T. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kestranek, A. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jadhav, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chervenak, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ford, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simeonov, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roth, G. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, C. J.</span></span> <span> </span><span class="NLM_article-title">Exploratory analysis of kinetic solubility measurements of a small molecule library</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">4127</span>– <span class="NLM_lpage">4134</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2011.05.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=10.1016%2Fj.bmc.2011.05.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=21640593" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=1%3ACAS%3A528%3ADC%252BC3MXotVehsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2011&pages=4127-4134&author=R.+Guhaauthor=T.+S.+Dexheimerauthor=A.+N.+Kestranekauthor=A.+Jadhavauthor=A.+M.+Chervenakauthor=M.+G.+Fordauthor=A.+Simeonovauthor=G.+P.+Rothauthor=C.+J.+Thomas&title=Exploratory+analysis+of+kinetic+solubility+measurements+of+a+small+molecule+library&doi=10.1016%2Fj.bmc.2011.05.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Exploratory analysis of kinetic solubility measurements of a small molecule library</span></div><div class="casAuthors">Guha, Rajarshi; Dexheimer, Thomas S.; Kestranek, Aimee N.; Jadhav, Ajit; Chervenak, Andrew M.; Ford, Michael G.; Simeonov, Anton; Roth, Gregory P.; Thomas, Craig J.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">4127-4134</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Kinetic soly. measurements using prototypical assay buffer conditions are presented for a ∼58,000 member library of small mols.  Analyses of the data based upon phys. and calcd. properties of each individual mol. were performed and resulting trends were considered in the context of commonly held opinions of how physicochem. properties influence aq. soly.  The data was further analyzed using a decision tree model for soly. prediction and via a multi-dimensional assessment of physicochem. relationships to soly. in the context of specific rule-breakers relative to common dogma.  The role of soly. as a determinant of assay outcome is also considered based upon each compd.'s cross-assay activity score for a collection of publicly available screening results.  Further, the role of soly. as a governing factor for colloidal aggregation formation within a specified assay setting is examd. and considered as a possible cause of a high cross-assay activity score.  The results of this soly. profile should aid chemists during library design and optimization efforts and represent a useful training set for computational soly. prediction.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpU9biLJPDeaLVg90H21EOLACvtfcHk0lh6LLgfSDfGPQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXotVehsbg%253D&md5=1f5b67268444cd3911665ea395bbf72f</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2011.05.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2011.05.005%26sid%3Dliteratum%253Aachs%26aulast%3DGuha%26aufirst%3DR.%26aulast%3DDexheimer%26aufirst%3DT.%2BS.%26aulast%3DKestranek%26aufirst%3DA.%2BN.%26aulast%3DJadhav%26aufirst%3DA.%26aulast%3DChervenak%26aufirst%3DA.%2BM.%26aulast%3DFord%26aufirst%3DM.%2BG.%26aulast%3DSimeonov%26aufirst%3DA.%26aulast%3DRoth%26aufirst%3DG.%2BP.%26aulast%3DThomas%26aufirst%3DC.%2BJ.%26atitle%3DExploratory%2520analysis%2520of%2520kinetic%2520solubility%2520measurements%2520of%2520a%2520small%2520molecule%2520library%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2011%26volume%3D19%26spage%3D4127%26epage%3D4134%26doi%3D10.1016%2Fj.bmc.2011.05.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kerns, E. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Di, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carter, G. T.</span></span> <span> </span><span class="NLM_article-title">In vitro solubility assays in drug discovery</span>. <i>Curr. Drug Metab.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">879</span>– <span class="NLM_lpage">885</span>, <span class="refDoi"> DOI: 10.2174/138920008786485100</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=10.2174%2F138920008786485100" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=18991584" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;key=1%3ACAS%3A528%3ADC%252BD1MXivF2nsb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2008&pages=879-885&author=E.+H.+Kernsauthor=L.+Diauthor=G.+T.+Carter&title=In+vitro+solubility+assays+in+drug+discovery&doi=10.2174%2F138920008786485100"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">In vitro solubility assays in drug discovery</span></div><div class="casAuthors">Kerns, Edward H.; Di, Li; Carter, Guy T.</div><div class="citationInfo"><span class="NLM_cas:title">Current Drug Metabolism</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">879-885</span>CODEN:
                <span class="NLM_cas:coden">CDMUBU</span>;
        ISSN:<span class="NLM_cas:issn">1389-2002</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  The soly. of a compd. depends on its structure and soln. conditions.  Structure dets. the lipophilicity, hydrogen bonding, mol. vol., crystal energy and ionizability, which det. soly.  Soln. conditions are affected by pH, cosolvents, additives, ionic strength, time and temp.  Many drug discovery expts. are conducted under "kinetic" soly. conditions.  In drug discovery, soly. has a major impact on bioassays, formulation for in vivo dosing, and intestinal absorption.  A good goal for the soly. of drug discovery compds. is >60 ug/mL.  Equil. soly. assays can be conducted in moderate throughput, by incubating excess solid with buffer and agitating for several days, prior to filtration and HPLC quantitation.  Kinetic soly. assays are performed in high throughput with shorter incubation times and high throughput analyses using plate readers.  The most frequently used of these are the nephelometric assay and direct UV assay, which begin by adding a small vol. of DMSO stock soln. of each test compd. to buffer.  In nephelometry, this soln. is serially dild. across a microtiter plate and undissolved particles are detected via light scattering.  In direct UV, undissolved particles are sepd. by filtration, after which the dissolved material is quantitated using UV absorption.  Equil. soly. is useful for preformulation.  Kinetic soly. is useful for rapid compd. assessment, guiding optimization via structure modification, and diagnosing bioassays.  It is often useful to customize soly. expts. using conditions that answer specific research questions of drug discovery teams, such as compd. selection and vehicle development for pharmacol. and PK studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq1Pe5z4JU_6LVg90H21EOLACvtfcHk0lh6LLgfSDfGPQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXivF2nsb8%253D&md5=416cf5bfde2278eac1ba2c67feb1f62e</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.2174%2F138920008786485100&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F138920008786485100%26sid%3Dliteratum%253Aachs%26aulast%3DKerns%26aufirst%3DE.%2BH.%26aulast%3DDi%26aufirst%3DL.%26aulast%3DCarter%26aufirst%3DG.%2BT.%26atitle%3DIn%2520vitro%2520solubility%2520assays%2520in%2520drug%2520discovery%26jtitle%3DCurr.%2520Drug%2520Metab.%26date%3D2008%26volume%3D9%26spage%3D879%26epage%3D885%26doi%3D10.2174%2F138920008786485100" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-2"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i29"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c01705">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_47677"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a class="ext-link" href="/doi/10.1021/acs.jmedchem.0c01705?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c01705</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Supplementary results and discussion; chemistry; early leads (Figure S1); dose–response curves for the cell-free EMSA analysis for the most active compounds (Figure S2); EMSA IC50 and cell viability EC50 values for select esters and heterocyclics (Table S1) (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c01705/suppl_file/jm0c01705_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c01705/suppl_file/jm0c01705_si_002.csv">CSV</a>)</p></li><li><p class="inline">Qualitative Analysis Report: Compound <b>5a</b> (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c01705/suppl_file/jm0c01705_si_003.pdf">PDF</a>)</p></li><li><p class="inline">Qualitative Analysis Report: Compound <b>5o</b> (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c01705/suppl_file/jm0c01705_si_004.pdf">PDF</a>)</p></li><li><p class="inline">Qualitative Analysis Report: Compound <b>7e</b> (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c01705/suppl_file/jm0c01705_si_005.pdf">PDF</a>)</p></li><li><p class="inline">Qualitative Analysis Report: Compound <b>7f</b> (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c01705/suppl_file/jm0c01705_si_006.pdf">PDF</a>)</p></li><li><p class="inline">Qualitative Analysis Report: Compound <b>7g</b> (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c01705/suppl_file/jm0c01705_si_007.pdf">PDF</a>)</p></li><li><p class="inline">Qualitative Analysis Report: Compound <b>8i</b> (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c01705/suppl_file/jm0c01705_si_008.pdf">PDF</a>)</p></li><li><p class="inline">Qualitative Analysis Report: Compound <b>9k</b> (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c01705/suppl_file/jm0c01705_si_009.pdf">PDF</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01705/suppl_file/jm0c01705_si_001.pdf">jm0c01705_si_001.pdf (2.93 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01705/suppl_file/jm0c01705_si_002.csv">jm0c01705_si_002.csv (9.91 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01705/suppl_file/jm0c01705_si_003.pdf">jm0c01705_si_003.pdf (123.83 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01705/suppl_file/jm0c01705_si_004.pdf">jm0c01705_si_004.pdf (120.62 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01705/suppl_file/jm0c01705_si_005.pdf">jm0c01705_si_005.pdf (139.66 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01705/suppl_file/jm0c01705_si_006.pdf">jm0c01705_si_006.pdf (122.36 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01705/suppl_file/jm0c01705_si_007.pdf">jm0c01705_si_007.pdf (203.0 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01705/suppl_file/jm0c01705_si_008.pdf">jm0c01705_si_008.pdf (141.25 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01705/suppl_file/jm0c01705_si_009.pdf">jm0c01705_si_009.pdf (195.01 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.0c01705&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.0c01705%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2021.64.issue-1%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.0c01705" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                8MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"677aa0d26d873d42","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
