
<html lang="en"     class="pb-page"  data-request-id="a1078852-d761-4170-8c35-780a48cec4a3"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;article:article:10.1021/acs.jmedchem.9b01783;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text;issue:issue:10.1021/jmcmar.2020.63.issue-6"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Suppression of Drug-Resistant Non-Small-Cell Lung Cancer with Inhibitors Targeting Minichromosomal Maintenance Protein" /></meta><meta name="dc.Creator" content="Chia-Yi  Lin" /></meta><meta name="dc.Creator" content="Hsin-Yi  Wu" /></meta><meta name="dc.Creator" content="Yuan-Ling  Hsu" /></meta><meta name="dc.Creator" content="Ting-Jen Rachel  Cheng" /></meta><meta name="dc.Creator" content="Jyung-Hurng  Liu" /></meta><meta name="dc.Creator" content="Rou-Jie  Huang" /></meta><meta name="dc.Creator" content="Tzu-Hung  Hsiao" /></meta><meta name="dc.Creator" content="Chia-Jen  Wang" /></meta><meta name="dc.Creator" content="Pei-Fang  Hung" /></meta><meta name="dc.Creator" content="Albert  Lan" /></meta><meta name="dc.Creator" content="Szu-Hua  Pan" /></meta><meta name="dc.Creator" content="Rong-Jie  Chein" /></meta><meta name="dc.Creator" content="Chi-Huey  Wong" /></meta><meta name="dc.Creator" content="Pan-Chyr  Yang" /></meta><meta name="dc.Description" content="Drug resistance has been a major threat in cancer therapies that necessitates the development of new strategies to overcome this problem. We report here a cell-based high-throughput screen of a lib..." /></meta><meta name="Description" content="Drug resistance has been a major threat in cancer therapies that necessitates the development of new strategies to overcome this problem. We report here a cell-based high-throughput screen of a lib..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="March 3, 2020" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.9b01783" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Copyright © 2020 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.9b01783" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.9b01783" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.9b01783" /></link>
        
    
    

<title>Suppression of Drug-Resistant Non-Small-Cell Lung Cancer with Inhibitors Targeting Minichromosomal Maintenance Protein | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.9b01783" /></meta><meta property="og:title" content="Suppression of Drug-Resistant Non-Small-Cell Lung Cancer with Inhibitors Targeting Minichromosomal Maintenance Protein" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b01783/20200323/images/large/jm9b01783_0012.jpeg" /></meta><meta property="og:description" content="Drug resistance has been a major threat in cancer therapies that necessitates the development of new strategies to overcome this problem. We report here a cell-based high-throughput screen of a library containing two-million molecules for the compounds that inhibit the proliferation of non-small-cell lung cancer (NSCLC). Through the process of phenotypic screening, target deconvolution, and structure–activity relationship (SAR) analysis, a compound of furanonaphthoquinone-based small molecule, AS4583, was identified that exhibited potent activity in tyrosine kinase inhibitor (TKI)-sensitive and TKI-resistant NSCLC cells (IC50 = 77 nM) and in xenograft mice. The mechanistic studies revealed that AS4583 inhibited cell-cycle progression and reduced DNA replication by disrupting the formation of the minichromosomal maintenance protein (MCM) complex. Subsequent SAR study of AS4583 gave compound RJ-LC-07-48 which exhibited greater potency in drug-resistant NSCLC cells (IC50 = 17 nM) and in mice with H1975 xenograft tumor." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.9b01783"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.9b01783">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-14/achre4.2021.54.issue-14/20210720/achre4.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-29/aamick.2021.13.issue-29/20210728/aamick.2021.13.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-14/acncdm.2021.12.issue-14/20210721/acncdm.2021.12.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-7/amlcef.2021.3.issue-7/20210705/amlcef.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-29/acsodf.2021.6.issue-29/20210727/acsodf.2021.6.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-29/ascecg.2021.9.issue-29/20210726/ascecg.2021.9.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-29/ancham.2021.93.issue-29/20210727/ancham.2021.93.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-29/bichaw.2021.60.issue-29/20210727/bichaw.2021.60.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-27/cgeabj.2021.99.issue-27/20210726-01/cgeabj.2021.99.issue-27.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-7/cgdefu.2021.21.issue-7/20210707/cgdefu.2021.21.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-14/esthag.2021.55.issue-14/20210720/esthag.2021.55.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-29/iecred.2021.60.issue-29/20210728/iecred.2021.60.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-14/inocaj.2021.60.issue-14/20210719/inocaj.2021.60.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-29/jafcau.2021.69.issue-29/20210728/jafcau.2021.69.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-29/jacsat.2021.143.issue-29/20210728/jacsat.2021.143.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-7/jamsef.2021.32.issue-7/20210707/jamsef.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-29/langd5.2021.37.issue-29/20210727/langd5.2021.37.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-7/mpohbp.2021.18.issue-7/20210705/mpohbp.2021.18.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.9b01783&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.9b01783&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.9b01783&amp;href=/doi/10.1021/acs.jmedchem.9b01783" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2020</span><span class="cit-fg-volume">, 63</span><span class="cit-fg-issue">, 6</span><span class="cit-fg-pageRange">, 3172-3187</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/63/6" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.9b01957" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.9b01967" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Suppression of Drug-Resistant Non-Small-Cell Lung Cancer with Inhibitors Targeting Minichromosomal Maintenance Protein</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Chia-Yi Lin</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Chia-Yi Lin</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Genome and Systems Biology Degree Program, National Taiwan University and Academia Sinica, Taipei 100, Taiwan</div><div class="loa-info-affiliations-info">Institute of Biomedical Sciences, Academia Sinica, Taipei 115, Taiwan</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Chia-Yi++Lin">Chia-Yi Lin</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Hsin-Yi Wu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Hsin-Yi Wu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Institute of Chemistry, Academia Sinica, Taipei 115, Taiwan</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Hsin-Yi++Wu">Hsin-Yi Wu</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Yuan-Ling Hsu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Yuan-Ling Hsu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Graduate Institute of Medical Genomics and Proteomics, College of Medicine, National Taiwan University, Taipei 100, Taiwan</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Yuan-Ling++Hsu">Yuan-Ling Hsu</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Ting-Jen Rachel Cheng</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Ting-Jen Rachel Cheng</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">The Genomics Research Center, Academia Sinica, Taipei 115, Taiwan</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Ting-Jen+Rachel++Cheng">Ting-Jen Rachel Cheng</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Jyung-Hurng Liu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jyung-Hurng Liu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Institute of Genomics and Bioinformatics, College of Life Sciences, National Chung Hsing University, Taichung 402, Taiwan</div><div class="loa-info-affiliations-info">Agricultural Biotechnology Center, NCHU, Taichung 402, Taiwan</div><div class="loa-info-affiliations-info">Rong Hsing Research Center for Translational Medicine, NCHU, Taichung 402, Taiwan</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jyung-Hurng++Liu">Jyung-Hurng Liu</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Rou-Jie Huang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Rou-Jie Huang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemistry, National Central University, Jhong-Li 320, Taiwan</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Rou-Jie++Huang">Rou-Jie Huang</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Tzu-Hung Hsiao</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Tzu-Hung Hsiao</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Medical Research, Taichung Veterans General Hospital, Taichung 407, Taiwan</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Tzu-Hung++Hsiao">Tzu-Hung Hsiao</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Chia-Jen Wang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Chia-Jen Wang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Institute of Stem Cell and Translational Cancer Research, Chang Gung Memorial Hospital, Taoyuan 333, Taiwan</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Chia-Jen++Wang">Chia-Jen Wang</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Pei-Fang Hung</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Pei-Fang Hung</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Institute of Biomedical Sciences, Academia Sinica, Taipei 115, Taiwan</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Pei-Fang++Hung">Pei-Fang Hung</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Albert Lan</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Albert Lan</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Graduate Institute of Medical Genomics and Proteomics, College of Medicine, National Taiwan University, Taipei 100, Taiwan</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Albert++Lan">Albert Lan</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Szu-Hua Pan</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Szu-Hua Pan</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Genome and Systems Biology Degree Program, National Taiwan University and Academia Sinica, Taipei 100, Taiwan</div><div class="loa-info-affiliations-info">Graduate Institute of Medical Genomics and Proteomics, College of Medicine, National Taiwan University, Taipei 100, Taiwan</div><div class="loa-info-affiliations-info">Doctoral Degree Program of Translational Medicine, National Taiwan University, Taipei 100, Taiwan</div></div><span class="conrtib-corresp"><strong>*</strong>S.-H.P.: email, <a href="/cdn-cgi/l/email-protection#afdcc7dfcec1efc1dbda81cacbda81dbd8"><span class="__cf_email__" data-cfemail="d2a1baa2b3bc92bca6a7fcb7b6a7fca6a5">[email protected]</span></a>; phone, +886-2-23123456, extension 88661; fax, +886-2-33936523.</span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Szu-Hua++Pan">Szu-Hua Pan</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-0138-0434" title="Orcid link">http://orcid.org/0000-0002-0138-0434</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Rong-Jie Chein</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Rong-Jie Chein</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Institute of Chemistry, Academia Sinica, Taipei 115, Taiwan</div></div><span class="conrtib-corresp"><strong>*</strong>R.-J.C.: email, <a href="/cdn-cgi/l/email-protection#27554d444f424e4967444f424a09544e494e444609424352095350"><span class="__cf_email__" data-cfemail="790b131a111c1017391a111c14570a1017101a18571c1d0c570d0e">[email protected]</span></a>; phone, +886-2-27898526; fax, +886-27831237.</span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Rong-Jie++Chein">Rong-Jie Chein</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Chi-Huey Wong</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Chi-Huey Wong</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">The Genomics Research Center, Academia Sinica, Taipei 115, Taiwan</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Chi-Huey++Wong">Chi-Huey Wong</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-9961-7865" title="Orcid link">http://orcid.org/0000-0002-9961-7865</a></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Pan-Chyr Yang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Pan-Chyr Yang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Genome and Systems Biology Degree Program, National Taiwan University and Academia Sinica, Taipei 100, Taiwan</div><div class="loa-info-affiliations-info">Institute of Biomedical Sciences, Academia Sinica, Taipei 115, Taiwan</div><div class="loa-info-affiliations-info">Department of Internal Medicine, College of Medicine, National Taiwan University, Taipei 100, Taiwan</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Pan-Chyr++Yang">Pan-Chyr Yang</a></span></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.9b01783&amp;href=/doi/10.1021%2Facs.jmedchem.9b01783" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2020</span></span><span class="cit-volume">, 63</span><span class="cit-issue">, 6</span><span class="cit-pageRange">, 3172–3187</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">March 3, 2020</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>26 November 2019</li><li><span class="item_label"><b>Published</b> online</span>3 March 2020</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 26 March 2020</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.9b01783" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b01783</a></div><div class="article_header-article-copyright"><strong>Copyright © 2020 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D3172%26pageCount%3D16%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DChia-Yi%2BLin%252C%2BHsin-Yi%2BWu%252C%2BYuan-Ling%2BHsu%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D63%26issueNum%3D6%26contentID%3Dacs.jmedchem.9b01783%26title%3DSuppression%2Bof%2BDrug-Resistant%2BNon-Small-Cell%2BLung%2BCancer%2Bwith%2BInhibitors%2BTargeting%2BMinichromosomal%2BMaintenance%2BProtein%26numPages%3D16%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D3187%26publicationDate%3DMarch%2B2020">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.9b01783"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">1481</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">5</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.9b01783" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Suppression of Drug-Resistant Non-Small-Cell Lung Cancer with Inhibitors Targeting Minichromosomal Maintenance Protein&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Chia-Yi&quot;,&quot;last_name&quot;:&quot;Lin&quot;},{&quot;first_name&quot;:&quot;Hsin-Yi&quot;,&quot;last_name&quot;:&quot;Wu&quot;},{&quot;first_name&quot;:&quot;Yuan-Ling&quot;,&quot;last_name&quot;:&quot;Hsu&quot;},{&quot;first_name&quot;:&quot;Ting-Jen&quot;,&quot;last_name&quot;:&quot;Rachel Cheng&quot;},{&quot;first_name&quot;:&quot;Jyung-Hurng&quot;,&quot;last_name&quot;:&quot;Liu&quot;},{&quot;first_name&quot;:&quot;Rou-Jie&quot;,&quot;last_name&quot;:&quot;Huang&quot;},{&quot;first_name&quot;:&quot;Tzu-Hung&quot;,&quot;last_name&quot;:&quot;Hsiao&quot;},{&quot;first_name&quot;:&quot;Chia-Jen&quot;,&quot;last_name&quot;:&quot;Wang&quot;},{&quot;first_name&quot;:&quot;Pei-Fang&quot;,&quot;last_name&quot;:&quot;Hung&quot;},{&quot;first_name&quot;:&quot;Albert&quot;,&quot;last_name&quot;:&quot;Lan&quot;},{&quot;first_name&quot;:&quot;Szu-Hua&quot;,&quot;last_name&quot;:&quot;Pan&quot;},{&quot;first_name&quot;:&quot;Rong-Jie&quot;,&quot;last_name&quot;:&quot;Chein&quot;},{&quot;first_name&quot;:&quot;Chi-Huey&quot;,&quot;last_name&quot;:&quot;Wong&quot;},{&quot;first_name&quot;:&quot;Pan-Chyr&quot;,&quot;last_name&quot;:&quot;Yang&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2020&quot;,&quot;month&quot;:&quot;03&quot;,&quot;day&quot;:&quot;03&quot;,&quot;issue&quot;:&quot;6&quot;,&quot;volume&quot;:&quot;63&quot;,&quot;pages&quot;:&quot;3172-3187&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.9b01783&quot;},&quot;abstract&quot;:&quot;Drug resistance has been a major threat in cancer therapies that necessitates the development of new strategies to overcome this problem. We report here a cell-based high-throughput screen of a library containing two-million molecules for the compounds that inhibit the proliferation of non-small-cell lung cancer (NSCLC). Through the process of phenotypic screening, target deconvolution, and structure–activity relationship (SAR) analysis, a compound of furanonaphthoquinone-based small molecule, AS4583, was identified that exhibited potent activity in tyrosine kinase inhibitor (TKI)-sensitive and TKI-resistant NSCLC cells (IC50 = 77 nM) and in xenograft mice. The mechanistic studies revealed that AS4583 inhibited cell-cycle progression and reduced DNA replication by disrupting the formation of the minichromosomal maintenance protein (MCM) complex. Subsequent SAR study of AS4583 gave compound RJ-LC-07-48 which exhibited greater potency in drug-resistant NSCLC cells (IC50 = 17 nM) and in mice with H1975 xenograft&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b01783&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b01783" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b01783&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b01783" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b01783&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b01783" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.9b01783&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b01783&amp;href=/doi/10.1021/acs.jmedchem.9b01783" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.9b01783" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.9b01783" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (6 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b01783&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.9b01783%26sid%3Dliteratum%253Aachs%26pmid%3D32125853%26genre%3Darticle%26aulast%3DLin%26date%3D2020%26atitle%3DSuppression%2Bof%2BDrug-Resistant%2BNon-Small-Cell%2BLung%2BCancer%2Bwith%2BInhibitors%2BTargeting%2BMinichromosomal%2BMaintenance%2BProtein%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D63%26issue%3D6%26spage%3D3172%26epage%3D3187%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (3)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292258" title="Cancer">Cancer</a>,</li><li><a href="/action/doSearch?ConceptID=291011" title="High-performance liquid chromatography">High-performance liquid chromatography</a>,</li><li><a href="/action/doSearch?ConceptID=290659" title="DNA replication">DNA replication</a>,</li><li><a href="/action/doSearch?ConceptID=291439" title="Genetics">Genetics</a>,</li><li><a href="/action/doSearch?ConceptID=290896" title="Cells">Cells</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/63/6" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/jmcmar.2020.63.issue-6/20200326/jmcmar.2020.63.issue-6.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b01783/20200323/images/medium/jm9b01783_0012.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b01783/20200323/images/large/jm9b01783_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01783&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Drug resistance has been a major threat in cancer therapies that necessitates the development of new strategies to overcome this problem. We report here a cell-based high-throughput screen of a library containing two-million molecules for the compounds that inhibit the proliferation of non-small-cell lung cancer (NSCLC). Through the process of phenotypic screening, target deconvolution, and structure–activity relationship (SAR) analysis, a compound of furanonaphthoquinone-based small molecule, AS4583, was identified that exhibited potent activity in tyrosine kinase inhibitor (TKI)-sensitive and TKI-resistant NSCLC cells (IC<sub>50</sub> = 77 nM) and in xenograft mice. The mechanistic studies revealed that AS4583 inhibited cell-cycle progression and reduced DNA replication by disrupting the formation of the minichromosomal maintenance protein (MCM) complex. Subsequent SAR study of AS4583 gave compound RJ-LC-07-48 which exhibited greater potency in drug-resistant NSCLC cells (IC<sub>50</sub> = 17 nM) and in mice with H1975 xenograft tumor.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_27124" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_27124" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Lung cancer remains the leading cause of cancer-related death worldwide, and approximately 1.8 million new lung-cancer cases are diagnosed every year,<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> the majority being non-small-cell lung cancer (NSCLC). Despite advances in cancer treatment have been made over the past 20 years, less than 30% of lung-cancer patients respond to chemotherapy, and prognoses generally remain poor.<a onclick="showRef(event, 'ref2 ref3'); return false;" href="javascript:void(0);" class="ref ref2 ref3">(2,3)</a> Although the development of specific tyrosine kinase inhibitors (TKIs) targeting the epidermal growth factor receptor (EGFR) has improved patient outcomes, drug resistance has become a major concern. Therefore, it is essential to develop new anticancer agents with novel mode of action to overcome the problem of drug resistance, especially of EGFR-TKI resistance.</div><div class="NLM_p">Over the past years, target-based screens and cell-based screens have been the two predominant strategies for drug discovery. With the growing knowledge of cancer biology, many pharmaceutical companies have used target-based screens to develop therapeutic agents that specifically inhibit the key factors associated with cancer progression,<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> e.g., the FDA-approved drugs gefitinib,<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> erlotinib<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> (targeting EGFR), crizotinib<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> (targeting MET and ALK), imatinib<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> (targeting ABL), temsirolimus<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> (targeting mTOR), and dabrafenib<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> (targeting BRAF). However, the problem of drug resistance and the number of known targets that drive cancer progression remains limited, presenting a substantial challenge for target-based drug development.<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> In order to identify new targets for the development of anticancer agents, the strategy of cell-based phenotypic screens has been used and several anticancer agents have been developed and approved by the FDA, e.g., vorinostat,<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> romidepsin,<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> azacitidine,<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> and lenalidomide and pomalidomide.<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a></div><div class="NLM_p last">Recently, the integration of high-throughput or high-content screening of large libraries combined with computational data mining has been used to facilitate the drug discovery process, including target identification and validation as well as the study of structure–activity relationship (SAR) and mechanisms of action.<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> Herein, we present a successful hit-to-lead discovery of a novel anticancer compound with high potency against both TKI-sensitive and TKI-resistant NSCLC using cell-based, high-throughput screening combined with SAR study and <i>in silico</i> modeling and simulation. A high-throughput screen of lung-cancer cells with different EGFR mutation status<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> was performed to identify compound AS4583, a furanonaphthoquinone-based small molecule that interferes with S-phase progression. <i>In vitro</i> analyses indicated that AS4583 interferes with cell-cycle progression and DNA replication by disrupting the formation of the minichromosomal maintenance protein (MCM) complex by promoting the proteasome degradation, especially of MCM2. Further SAR study of AS4583 led to the identification of RJ-LC-07-48 with anticancer efficacy around 10-fold higher than AS4583 against TKI-resistant NSCLC.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_34308" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_34308" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> Identification of AS4583 as an Inhibitor of NSCLC Cell Proliferation</h3><div class="NLM_p">A high-throughput screening of 2 million compounds (each 10 μM) was initially carried out to evaluate their ability to inhibit the proliferation of human H1975 NSCLC cells, which are resistant to EGFR-TKIs and harbor an EGFR-TKI-sensitive mutation (L858R) and an acquired resistance mutation (T790M).<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> Approximately 6800 hit compounds exhibiting ≥80% inhibition at 10 μM were identified, which were further screened against other lung-cancer cells, including clinical isolates (CL1-0 and CL1-5) as well as an ATCC cell line PC9 and its EGFR-TKI-derivative-resistant clone PC9/IR. We then focused on compounds with potent activity against both TKI-sensitive and TKI-resistant cells, such as PC9 and PC9/IR, respectively, to identify compounds with new modes of action. After exclusion of established anticancer drugs, such as topotecan and doxorubicin analogs, a total of 193 hits with ≥80% inhibition at 10 μM were identified and further assessed with respect to their ability to inhibit the proliferation of both normal and cancer cells, especially inhibition of cell-cycle progression. Among them, a furanonaphthoquinone-based small molecule, AS4583 (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>A), was found to inhibit the proliferation of both TKI-sensitive and TKI-resistant lung-cancer cells with IC<sub>50</sub> values of <100 nM and more than 10-fold higher potency for cancer cells than human normal bronchial epithelial cells (NBE, <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>B and <a href="/doi/suppl/10.1021/acs.jmedchem.9b01783/suppl_file/jm9b01783_si_001.pdf" class="ext-link">Supporting Information Tables 1 and 2</a>).</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b01783/20200323/images/medium/jm9b01783_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b01783/20200323/images/large/jm9b01783_0001.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. AS4583 selected from high-throughput screening affects cell proliferation and inhibits tumor growth <i>in vivo</i>. (A) Structure of AS4583. (B) Cytotoxicity assay for AS4583 with various normal and tumor cell lines. Cells were treated with 0–10 μM AS4583 for 72 h, and survival was assessed with the sulforhodamine B assay. (C) Tumor sizes and (D) TUNEL assay results from tumor xenograft models (<i>n</i> = 6 per group). H1975 cells (3 × 10<sup>6</sup>) were subcutaneously injected into male nu/nu mice. Tumor size was measured every 3–4 days. On day 27, the mice were sacrificed to examine tumor size as well as to confirm histology (original magnification 400×) and apoptosis status by hematoxylin and eosin (H&E) and TUNEL staining. (E) Bar graph showing percent tumor growth inhibition (% TGI) on the day of the final measurement for the control group compared across all treatments in the <i>in vivo</i> study.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b01783/20200323/images/large/jm9b01783_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01783&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5"> AS4583 Inhibits Tumor Growth <i>in Vivo</i></h3><div class="NLM_p last">To evaluate the antitumor efficacy of AS4583 <i>in vivo</i>, athymic nude mice bearing established subcutaneous H1975 tumors were 5 days a week treated with 1 mg/kg or 4 mg/kg AS4583 (compared with DMSO control) for 4 weeks (<i>n</i> = 6 per group). The result showed that treatment with AS4583 significantly reduced H1975 xenograft tumor growth on post-treatment day 27 (average tumor size, 1217 ± 516.6 mm<sup>3</sup> for DMSO, 701.2 ± 196.5 mm<sup>3</sup> for 1 mg/kg, and 518.0 ± 93.0 mm<sup>3</sup> for 4 mg/kg, both <i>p</i> < 0.05) and induced apoptosis in the tumor tissues (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>C and <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>D). AS4583 treatment induced tumor growth delay for a longer period of time when compared with DMSO controls. Compared to DMSO-treated animals, AS4583 1 mg/kg and 4 mg/kg showed significant percent tumor growth inhibition (% TGI) of 44 and 59, respectively (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>E). To evaluate the potential cytotoxicity of AS4583, especially in liver and kidney, the body weight of each mouse was measured and the serum was subjected to biochemical analysis including aspartate transaminase, alanine transaminase, blood urea nitrogen, and creatinine (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01783/suppl_file/jm9b01783_si_001.pdf" class="ext-link">Supplementary Figure 1</a>). No significant difference was observed. These results indicated that AS4583 could inhibit tumor growth without side effects.</div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i6" class="anchor-spacer"></div><h3 class="article-section__title" id="_i6"> AS4583 Inhibits Formation of the DNA Replication Fork and Prolongs S Phase in NSCLC Cells</h3><div class="NLM_p">To examine the hypothesis that treatment of cells with AS4583 might disrupt cell cycle, we first performed a flow cytometry analysis to examine the effect of this compound on cell-cycle progression. H1975 cells treated with 140 nM AS4583 for 72 h were significantly arrested in the S phase (18% of the treated vs 2% of the control group) and the G2/M phase (38% of the treated vs 17% of the control group) (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01783/suppl_file/jm9b01783_si_001.pdf" class="ext-link">Supporting Information Table 3</a>). Next, we investigated whether AS4583 inhibits cell proliferation by interfering with DNA replication. The incorporation of bromodeoxyuridine (BrdU) into replicated DNA was monitored in H1975 cells treated with DMSO (control) or AS4583 for 12, 24, or 48 h. The percentage of BrdU-positive cells was lower in the treated group (at 24 h, 27.7 ± 3.9% vs 44.5 ± 2.0% for the control, <i>p</i> = 0.086; at 48 h, 16.0 ± 2.5% vs 34.0 ± 5.6% for the control, <i>p</i> = 0.0411; <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>A and <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>B).</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b01783/20200323/images/medium/jm9b01783_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b01783/20200323/images/large/jm9b01783_0002.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Treatment of cells with AS4583 interrupts DNA replication, causes cell cycle arrest, and induces apoptosis. (A, B) AS4583 inhibits DNA replication in H1975 cells. H1975 cells were pulsed with BrdU and analyzed by flow cytometry. (A) Representative DNA histograms from DMSO- and AS4583-treated H1975 cells over time (three independent experiments). (B) Average percentage of cells in which BrdU was incorporated. (C) AS4583 decreases the number of DNA replication forks. The replication forks were analyzed by immunofluorescence staining for MCM2 (green) and DAPI (blue, for nuclei). Data were quantified by counting clear, green spots using ImageJ software (<i>n</i> = 5 cells per group; original magnification, 1000×). (D) G1/S phase is prolonged in AS4583-treated H1975 cells. A double thymidine block was performed, and the data were collected at the indicated time points. (E) AS4583 induced intrinsic apoptosis pathway. H1975 cells were treated with AS4583 for 24, 48, or 72 h. The apoptotic marker protein expressions were detected by immunoblotting.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b01783/20200323/images/large/jm9b01783_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01783&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">To further assess whether treatment with AS4583 interferes with DNA replication, we examined the formation of replication forks by immunofluorescence staining of the replication marker, MCM2, in both DMSO- and AS4583-treated H1975 cells released from G1 phase. Indeed, fewer replication forks were found in the AS4583-treated group (48.8 ± 18.7) compared with the control (176.8 ± 8.7, <i>p</i> = 0.0007, <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>C). Whether this decrease in the formation of replication forks could alter the distribution of cell-cycle phases was then evaluated. H1975 cells were synchronized in G1/S phase using a double thymidine block followed by release with DMSO and AS4583-containing medium up to 26 h. We observed that a higher percentage of AS4583-treated cells (<i>p</i> = 0.0373) remained in the G1/S phase after 20 h (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>D), demonstrating that AS4583 increased the G1/S phase population, i.e., causing G1/S arrest. These results revealed that AS4583 significantly inhibits replication-fork progression and hence prevents cell-cycle progression.</div><div class="NLM_p last">When further dissecting the potential apoptotic pathways that are activated by AS4583, the expression levels of several specific apoptotic markers were detected by immunoblotting. As expected, the cleaved form of three important regulatory proteins in the apoptotic pathway, caspase-3 and PARP-1, appeared after AS4583 treatment. The appearance of cleaved caspase-8 implied that an intrinsic apoptotic pathway was being activated by AS4583 treatment (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>E).</div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i7" class="anchor-spacer"></div><h3 class="article-section__title" id="_i7"> The Decrease in MCM2 Level Could Lead to Cell Death through Silencing of the Expression of MCM2 via Lentivirus-Based Short Hairpin RNAs</h3><div class="NLM_p">As a significant reduction of MCM2 in AS4583 treated cells was observed in the DNA replication fork study, we were curious about whether the decrease of MCM2 could directly interfere in the cell survival. To test this hypothesis, we silenced the expression of MCM2 via lentivirus-based short hairpin RNAs and then confirmed whether the breakdown of MCM2 accounts for AS4583-induced cell death. The results showed that MCM2 knockdown resulted in a 33% inhibition of cell proliferation (<i>p</i> < 0.0001, <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>A), and the efficiency of knockdown depended on the amount of short hairpin RNA (<i>p</i> < 0.0001, <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>B). These data are consistent with the observations reported by Liang and his colleagues.<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> Moreover, we also found that AS4583-induced cell death correlated positively with the expression of MCM2 in different lung-cancer cell lines and skin fibroblast cell line (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>C,D and <a href="/doi/suppl/10.1021/acs.jmedchem.9b01783/suppl_file/jm9b01783_si_001.pdf" class="ext-link">Supporting Information Table 4</a>). These data implied that MCM complex might be involved in AS4583-induced lung cancer cell death.</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b01783/20200323/images/medium/jm9b01783_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b01783/20200323/images/large/jm9b01783_0003.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Correlation between MCM2 level and cell death via lentivirus-based short hairpin RNAs or AS4583 treatment. (A) Cell survival decreases after MCM2 knockdown. The cells were infected with the indicated lentivirus in medium containing polybrene (8 μg/mL). At 24 h postinfection, the cells were transferred to fresh medium and incubated for 48 h, then analyzed by immunoblotting and the sulforhodamine B assay. (B) MCM2 knockdown decreases cell survival in a dose-dependent manner. The cells were infected with the indicated lentivirus in medium containing polybrene (8 μg/mL). At 24 h postinfection, the cells were incubated in fresh medium lacking lentivirus and polybrene for 48 h and analyzed with the MTS assay over an additional 24–120 h. (C) Cellular levels of MCM2 were detected in different lung-cancer cells as assessed with immunoblotting. (D) Quantification of MCM2 levels and IC<sub>50</sub> values for each cell line were used to calculate the correlation between the two cofactors in these different lung-cancer cell lines (IC<sub>50</sub> versus MCM2, <i>R</i><sup>2</sup> = 0.5282).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b01783/20200323/images/large/jm9b01783_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01783&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8"> High Expression of the MCM Complex Is Associated with Poor Clinical Outcomes of NSCLC Patients</h3><div class="NLM_p">To further validate whether the MCM complex is a potential therapeutic target for NSCLC, we examined the cellular levels of MCM components by analyzing the gene expression data obtained from 226 pathological stage I and II lung adenocarcinomas in the public database GSE31210.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> The results indicated that high expression of MCM2, MCM4, MCM5, or MCM6 is associated with poor overall survival compared with the low-expression group (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>). Thus, the components of the MCM complex may serve as markers for high-risk patients and as potential therapeutic targets for the treatment of early stage lung adenocarcinoma.</div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b01783/20200323/images/medium/jm9b01783_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b01783/20200323/images/large/jm9b01783_0004.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Clinical significance and prognostic value of MCM in NSCLC. Kaplan–Meier analysis of MCM expression and overall survival for 226 human lung adenocarcinoma samples, pathological stage I or II, published as GSE31210. The <i>p</i>-values were obtained with a two-sided log-rank test.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b01783/20200323/images/large/jm9b01783_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01783&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i9" class="anchor-spacer"></div><h3 class="article-section__title" id="_i9"> AS4583 Promotes Ubiquitination of MCM2–7 and Their Degradation in Lung-Cancer Cells</h3><div class="NLM_p">The heterohexametric MCM2–7 protein complex has been identified as an important component for the precisely coordinated replication of the entire genome each time a cell divides.<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> We next determined whether AS4583 treatment could modulate the expression of MCM protein components. As expected, the cellular levels of MCM2, MCM6, and MCM7 decreased significantly (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>A) in an AS4583 dose-dependent manner (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>B) at 24 h after treatment, whereas the levels of MCM3, MCM4, and MCM5 decreased after 72 h (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>A). This decrease in MCM expression appeared to be a consequence at the protein level because the levels of the corresponding mRNAs did not change over the course of AS4583 treatment (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01783/suppl_file/jm9b01783_si_001.pdf" class="ext-link">Supplementary Figure 2</a>). In support of this idea, the observed decrease in MCM2, MCM6, and MCM7 was abrogated after treatment with the proteasome inhibitor MG132 (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>C). Indeed, the MCM2 levels in H1975 cells were almost completely restored to the control level by MG132, whereas partial restoration was observed for MCM6 and MCM7. Further experiments confirmed that ubiquitination of MCM2 increased in the presence of AS4583 (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>D).</div><figure id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b01783/20200323/images/medium/jm9b01783_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b01783/20200323/images/large/jm9b01783_0005.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. AS4583 induces proteasome-mediated degradation of MCM2 after loss chromatin bound ability. (A) Increased degradation of MCM2/6/7 upon treatment of H1975 cells with AS4583. AS4583-treated cells were harvested at 24, 48, and 72 h, and the levels of proteins of the MCM complex were analyzed with immunoblotting. (B) Dose-dependent degradation of MCM2/6/7 in different lung cancer cell lines. All cells were treated with AS4583 for 24 h and then harvested. The levels of MCM2/6/7 were examined with immunoblotting. (C) The proteasome inhibitor MG132 reverses the effects of AS4583 on the cellular levels of MCM2/6/7. H1975 cells were treated with AS4583 for 48 h, then incubated in the absence or presence of MG132 (10 μM) for an additional 12 h, and then harvested. The levels of MCM2/6/7 were examined with immunoblotting. (D) AS4583 induces ubiquitination of MCM2. The cells were transiently transfected with pcDNA3-Myc-Ub and then incubated in the absence or presence of AS4583 (140 nM) and MG132 (10 μM). After 12 h, the cells were harvested, and MCM2 was precipitated with an MCM2-specific antibody. Ubiquitination of MCM2 was detected with immunoblotting. IgG served as the antibody negative control for the immunoprecipitation (IP) experiment. (E) Treatment with AS4583 decreases the level of the chromatin-bound form of MCM2. A double thymidine block was performed, and the data were collected at the indicated time points. (F) MCM2 protein levels were recovered upon treatment of H1975 cells with MLN4924. Cells were treated with DMSO or AS4583 for 24 h and then co-treated with different dose of MLN4924. After 6 h, the cells were harvested for detection of MCM2.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b01783/20200323/images/large/jm9b01783_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01783&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">To explore whether the AS4583-induced MCM protein degradation was due to interference with the DNA binding ability of MCM2, H1975 cells were synchronized using a double thymidine block followed by release and subsequent treatment with DMSO (control) or AS4583 for 9 h. Then, the chromatin-bound MCM2 was detected by immunoblotting. As shown in <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>E, temporal binding of MCM2 to chromatin was observed in DMSO-treated cells. The bound MCM2 level decreased at 4.5 h after release from the block and later returned to the preblock level. In the AS4583-treated group, the dynamic binding was not observed; instead, the level of MCM2 decreased significantly during the course of treatment. Moreover, we also observed that AS4583-induced MCM2 degradation could be restored upon treatment of cells with the cullin inhibitor MLN4924 (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>F), which is consistent with the fact that MCM2/3/5/6/7 have been identified as substrates of cullin-RING ligases.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> Collectively, all these suggest that MCM2 might be the therapeutic target of AS4583 treatment.</div></div><div id="sec2_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i10" class="anchor-spacer"></div><h3 class="article-section__title" id="_i10"> AS4583 Binds the N-Terminal Portion of MCM2</h3><div class="NLM_p">Our results thus far raised an important question: does AS4583 bind directly to MCM2? To address this issue, we incubated H1975 lysates with AS4583-conjugated magnetic beads (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>A) for 24 h at 4 °C, and the results showed that more MCM2 was pulled down by the mag-beads-AS4583 compared with the mag-beads-control (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>B). In addition, we also performed an <i>in silico</i> modeling and simulation to provide structural insights into the binding of AS4583 to MCM2 (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01783/suppl_file/jm9b01783_si_001.pdf" class="ext-link">Supplementary Results and Supplementary Figures 3–5</a>). We modeled the MCM2 structure based on the crystal structure of an archaeal MCM from <i>Sulfolobus solfataricus</i> (ssoMCM, PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3F9V">3F9V</a>). Although the amino acid sequence identity of MCM2 to ssoMCM is only 38%, the confidence estimate of the fold recognition server is 100%, suggesting this template used for the homology modeling is highly reliable. The final structure of MCM2 was optimized by molecular dynamics procedure, and the validation results revealed that our model structure is of good quality for further studies. The protein–ligand docking program iGEMDOCK predicted that the binding site for AS4583 is in the N-terminal domain of MCM2. This site resides within a cavity surrounded by domains A, B, and C (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>C,D). The solvent-accessible portion of the binding site is close to the interface between MCM2 and MCM6 in the MCM hexamer (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>E). The B domain of MCM6 may rotate and insert into this site, thereby promoting the interaction with MCM2. The predicted binding energy is −8.8 kcal/mol for AS4583 in complex with MCM2 (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>F), and residues from the three domains, including His282 and Gln257 in domain A, Gln341 in domain B, and Asn442 in domain C, are involved in the AS4583 interaction. In particular, Gln341 contributes substantially to the intermolecular hydrogen-bond formation.</div><figure id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b01783/20200323/images/medium/jm9b01783_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b01783/20200323/images/large/jm9b01783_0006.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Interaction between AS4583 and MCM2. (A) Structures of mag-beads-control and mag-beads-AS4583. (B) AS4583 binds MCM2. H1975 cell lysates were incubated with magnetic beads-control or magnetic beads-AS4583 overnight, and the bound MCM2 proteins were detected by immunoblotting. (C) Domain organization of MCM2: N–C, N to C linker; α/β, α/β domain of the AAA+ ATPase core structure; α/β–α, linker between subdomains in the ATPase core; α, α domain; WH, winged-helix domain. The domain annotation is based on the structure of <i>Sulfolobus solfataricus</i> (ssoMCM, PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3F9V">3F9V</a>). (D) Model for MCM2 complexed with AS4583. Domains and linkers have the same coloring as depicted in panel C. Helices are depicted as cylinders and β-strands as arrows. The AS4583 structure is shown as a space-filling model (pink, carbon; blue, nitrogen; red, oxygen). (E) Model of AS4583 in the MCM hexamer. Protomers (MCM2–7) are colored differently and labeled. AS4583 is shown as red spheres. (F) AS4583 binding pocket. The domains have the same coloring as in panel C. The residues involved in AS4583 binding are shown as stick structures. The black dashed lines represent polar contacts between amino acid residues and AS4583. (G) Mag-beads-AS4583 pulled down less MCM2 Q341A than MCM2 wild type. Immunoprecipitation (IP) was performed by incubating lysates of cells that overexpressed FLAG, FLAG-MCM2 wild-type, or the Q341A mutant with mag-beads-control or -AS4583.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b01783/20200323/images/large/jm9b01783_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01783&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">To validate the <i>in silico</i> docking results concerning Gln341, we constructed the plasmids p3xFlag-MCM2-WT and p3xFlag-MCM2-Q341A for overexpression in H1975 cells. The Western blotting analysis of MCM2-WT or MCM2-Q341A cells shows that Flag-tagged protein was expressed. The magnetic-bead pulldown assay revealed significantly less binding of MCM2-Q341A to mag-beads-AS4583 compared with the binding of the control, Flag-tagged MCM2-WT (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>G). Next, we investigated if MCM2-Q341A can serve as a resistant clone by testing IC<sub>50</sub> of AS4583 and ubiquitination of MCM2. The IC<sub>50</sub> of AS4583 is 71 nM for MCM2-WT cells and 150 nM for MCM2-Q341A cells (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01783/suppl_file/jm9b01783_si_001.pdf" class="ext-link">Supplementary Figure 6A</a>). We also performed the immunoprecipitation assay to pull down Flag-tagged MCM2-WT or -Q341A with AS4583 treatment for 24 h and then detected the polyubiquitinated species of MCM2 by immunoblotting. In accordance with the previous result, treatment with AS4583 significantly enhanced the ubiquitination of MCM2-WT but reduced ubiquitination of MCM2-Q341A in 293T cells (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01783/suppl_file/jm9b01783_si_001.pdf" class="ext-link">Supplementary Figure 6B</a>). These data indicate that Q341A weakened the cytotoxicity and proteasomal-dependent ubiquitination of AS4583. This result supports the critical contribution of Gln341 for binding of AS4583 to MCM2.</div></div><div id="sec2_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i11" class="anchor-spacer"></div><h3 class="article-section__title" id="_i11"> Cell-Based SAR Study of AS4583 Reveals Essential Molecular Functionality of AS4583 That Mediates the Inhibition of H1975 Cell Proliferation</h3><div class="NLM_p">To identify the structural components of AS4583 that are essential for its ability to inhibit the proliferation of lung-cancer cells, a preliminary SAR study was carried out with modifications of the X, R1, and R2 groups of AS4583 in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a> and the length (<i>n</i>) of the side chain on the furan ring. The ability of each analog to inhibit H1975 cell proliferation was evaluated using the sulforhodamine B assay. As shown in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>, analogs with different amide side-chain lengths (compound <b>1</b>, <i>n</i> = 0, and compound <b>2</b>, <i>n</i> = 2) and R1 group sizes (compound <b>3</b>–<b>5</b>) suppressed H1975 cell proliferation with IC<sub>50</sub> of <1.0 μM. The IC<sub>50</sub> value of the pyrrole analog <b>6</b> was 100-fold higher than that of AS4583, and benzofuran <b>11</b> had little or no effect up to 10 μM. These results suggested that both the furan oxygen and benzoquinone moiety are essential for AS4583 activity. Replacement of the imidazole group at the end of the amide chain of AS4583 with morpholine (compound <b>8</b>) did not affect the IC<sub>50</sub> value (80 nM), and compound <b>10</b>, with a benzyl tail end, was unable to suppress H1975 cell proliferation even at a dose of 10 μM. These results indicated that the tertiary nitrogen of the amide chain is essential for the antiproliferation activity of AS4583. Notably, compound <b>9</b>, with a pyrrolidine moiety instead of an imidazole ring, had the best activity with IC<sub>50</sub> = 24 nM, while the activity was diminished with pyrrolyl side chain (<b>7</b>, IC<sub>50</sub> = 918 nM). According to the SAR study (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>), the furan oxygen is important for AS4583 activity. This finding is coherent with our predicted binding model (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>F), because the furan oxygen forms a hydrogen bond with His282 side chain. The size of R1 moiety is also closely related to AS4583 activity. In our predicted binding model, the R1 methyl moiety is surrounded by Ile281, Val283, and Leu256, and therefore a much bulky moiety cannot fit into this pocket. In addition, the replacement of R1 by a proton would reduce the van der Waals force for the AS4583 binding.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. SAR Study of AS4583</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b01783/20200323/images/medium/jm9b01783_0010.gif" alt="" id="gr7" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b01783/20200323/images/medium/jm9b01783_0011.gif" alt="" id="gr8" /></img><div></div></div></div><div id="sec2_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13"> Synthesis of a More Potent Lead Compound, RJ-LC-07-48</h3><div class="NLM_p">Although compound <b>9</b> was the most potent inhibitor of H1975 cell viability in the aforementioned analysis (IC<sub>50</sub> = 24 nM), it also exhibited toxicity toward skin fibroblast cells (Hs68, IC<sub>50</sub> = 77 nM). To further improve the potency of AS4583 and reduce its toxicity, a new analog, RJ-LC-07-48, was synthesized from 7-fluoro-1-tetralone (<b>12</b>). As shown in <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>, 7-fluoro-1-tetralone was regioselectively oxidized to 2-hydroxynaphthoquinone (<b>13</b>) under an oxygen atmosphere. A three-components reaction<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> was then applied to transfer compound <b>13</b> to a phosphorus zwitterion <b>14</b>. A Wittig reaction between <b>14</b> and acetyl chloride provided furanonaphthoquinone <b>15</b>, which was then hydrolyzed to acid <b>16</b> and then linked with the imidazole side chain to complete the synthesis. Fortunately, the survival assay showed that RJ-LC-07-48 had a better activity in the inhibition of H1975 cell viability (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01783/suppl_file/jm9b01783_si_001.pdf" class="ext-link">Supplementary Figure 7A</a>, IC<sub>50</sub> = 17 nM). In addition, treatment with RJ-LC-07-48 markedly reduced the growth of H1975 xenograft tumors in mice compared with the control group: average tumor size, 546.2 ± 95.2 mm<sup>3</sup> for DMSO/control, 128.2 ± 50.5 mm<sup>3</sup> for 1 mg/kg RJ-LC-07-48, and 109.5 ± 23.1 mm<sup>3</sup> for 4 mg/kg RJ-LC-07-48 on day 24; <i>p</i> < 0.05, <i>n</i> = 6 per group, <a href="/doi/suppl/10.1021/acs.jmedchem.9b01783/suppl_file/jm9b01783_si_001.pdf" class="ext-link">Supplementary Figure 7G</a>). All these results suggested that RJ-LC-07-48 is a potential lead compound for treatment of lung cancer.</div><figure id="sch1" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b01783/20200323/images/medium/jm9b01783_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b01783/20200323/images/large/jm9b01783_0007.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Stereoselective Synthesis of RJ-LC-07-48<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b01783/20200323/images/large/jm9b01783_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01783&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) O<sub>2</sub> atmosphere, room temperature (rt), 50 min, 90%; (b) CHOCO<sub>2</sub>Et, TFA, PBu<sub>3</sub>, THF, 0–64 °C, 3.5 h, 63%; (c) acetyl chloride, Et<sub>3</sub>N, THF, 0 °C to rt, 15 min, 77%; (d) 3 M NaOH, THF/H<sub>2</sub>O (4:1), 0 °C to rt, 5 h, 97%; (e) HBTU, DMAP, 1-(3-aminopropyl)imidazole, DMF, rt, 6 h, 64%.</p></p></figure></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i15">Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_55205" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_55205" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Through the high-throughput cell-based screening and SAR study, we have identified a furanonaphthoquinone-based small molecule, AS4583, which can induce the proteasome degradation of MCM2–7 and ultimately contribute to the death of NSCLC cells. The SAR study of AS4583 further identified the furanonaphthoquinone moiety and the tertiary nitrogen at the end of the side chain as crucial components for the inhibition of tumor growth. Through computer-simulated docking, we found that AS4583 could bind to MCM2 by forming a hydrogen bond with residue Q341, and this result was supported by immunoprecipitation experiments using anti-MCM2. Finally, a more potent compound RJ-LC-07-48 was synthesized with better antitumor effect than AS4583 <i>in vivo</i>.</div><div class="NLM_p">We found that AS4583 contributes to the induction of ubiquitinated-MCM2, leading to degradation of MCM2 by the proteasome. MCM2 degradation may further interfere with the formation of the MCM complex, cause inhibition of DNA replication, and prolong the duration of S phase in cancer cells. MCM2 degradation may ultimately promote apoptosis <i>in vitro</i> and inhibit tumor growth <i>in vivo.</i> Since MCM2–7 must be assembled as a hexamer or tetramer before nuclear import,<a onclick="showRef(event, 'ref19 ref21'); return false;" href="javascript:void(0);" class="ref ref19 ref21">(19,21)</a> the degradation of ubiquitinated MCM2 in proteasomes may interfere with the formation of the MCM complex, perturb its import into the nucleus, and inhibit chromatin loading.</div><div class="NLM_p">The docking analysis revealed that AS4583 does not occupy the ATP binding site in the C-terminal ATPase domain of MCM2 but rather binds to the cavity in the N-terminal domain. Indeed, the N-terminal portion of MCM2 has been shown to be important for the hexamer formation, DNA binding, and enzyme regulation.<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> Mutational and biochemical analyses carried out by others have further suggested that a conformational change in the N-terminal portion (especially the movement of domain A) is required for helicase function.<a onclick="showRef(event, 'ref25 ref26'); return false;" href="javascript:void(0);" class="ref ref25 ref26">(25,26)</a> The binding of AS4583 to MCM2 may alter the rigidity of the N-terminal portion and hinder the movement of domain A. In addition, the bound AS4583 molecule could interfere with the interaction between MCM6 and MCM2 and therefore perturb hexamer assembly and helicase function. On the basis of these findings, we propose a paradigm for AS4583-induced apoptosis. Cells typically progress through the cell cycle using the MCM2–7 helicases to unwind the DNA double helix for DNA replication. In cancer cells, however, the MCM2–7 complex is highly regulated through the G0–G1 to S phase. Once this complex is disrupted, the cancer cells will undergo apoptosis. As important components in the process of DNA replication for malignant proliferation, MCM2–7 represent a potential class of drug targets for lung-cancer treatment in the near future.</div><div class="NLM_p last">This is the first report to show that a small molecule can induce MCM2 ubiquitination and suppress the expression of MCM2. Although previous studies reported that norcantharidin induced the cleavage of MCM2 at the micromolar level<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> and that MCM2 is a therapeutic target of trichostatin A in colon cancer cells,<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> their mechanisms of action remain unclear and need clarification. In addition, high expression levels of MCM complex proteins are associated with poor survival of patients with NSCLC. In summary, our results demonstrate that the MCM complex can serve as a potential target for lung-cancer treatment, and this work paves the way for development of new cancer agents to tackle the problem of drug resistance.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i16">Conclusion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_66082" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_66082" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">The use of high-throughput drug screens for various lung-cancer cells and of <i>in silico</i> docking prediction will provide effective means for identifying small molecules that target the MCM complex, which is important for developing new cancer therapeutics. Our results demonstrate that proteins of the MCM complex can serve as potential targets for lung-cancer treatment, and our SAR analysis paves the way for the design of potent lead compounds.</div></div><div id="sec5" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i17">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_50645" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_50645" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec5_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> Synthesis and Characterization of Chemical Compounds</h3><div class="NLM_p last">Unless stated otherwise, reactions were performed in flame-dried glassware under a positive pressure of nitrogen. Commercial grade reagents and solvents were used without further purification. CH<sub>2</sub>Cl<sub>2</sub>, CH<sub>3</sub>CN, and THF were purified by PS-MD-5 solvent purification system (Innovative Technology, Inc.). Magnetic bead was purchased from TAN Beads, Taiwan. The progress of all the reactions were monitored by TLC, using TLC glass plates precoated with silica gel 60 F<sub>254</sub> (Merck). Flash column chromatography was performed with silica gel Geduran Si 60 (Merck). IR spectra were recorded with Thermo Nicolet iS-5 FT-IR spectrophotometer, ν<sub>max</sub> in cm<sup>–1</sup>. <sup>1</sup>H and <sup>13</sup>C NMR spectra were recorded with Bruker AV-III 400 MHz or Bruker AV-400 MHz spectrometer, and chemical shifts were measured in δ (ppm) with residual solvent peaks as internal standards (CDCl<sub>3</sub>, δ 7.24 ppm in <sup>1</sup>H NMR, δ 77.0 ppm in <sup>13</sup>C NMR; CD<sub>3</sub>OD, δ 3.31 ppm in <sup>1</sup>H NMR, δ 49.0 ppm in <sup>13</sup>C NMR). Coupling constants <i>J</i> were measured in Hz. HR FAB (LR FAB), and HR EI (LR EI) mass spectra were recorded on a JMS-700 double focusing mass spectrometer (JEOL, Tokyo, Japan) with a resolution of 8000 (3000) (5% valley definition), and HR (LR) ESI (electrospray) mass spectra were recorded using dual ionization ESCi (ESI/APCi) source options, Waters LCT Premier XE (Waters Corp., Manchester, U.K.). Melting points were recorded on Büchi M-565 apparatus. Purity of all compounds for biology test was found to be >98% as determined by HPLC (Shimadzu LCMS-2020 with Unitary C18 columns).</div></div><div id="sec5_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i19" class="anchor-spacer"></div><h3 class="article-section__title" id="_i19"> Ethyl 2-Methyl-4,9-dioxo-4,9-dihydronaphtho[2,3-<i>b</i>]furan-3-carboxylate (<b>17</b>)</h3><div class="NLM_p last">A mixture of 2,3-dichloro-1,4-naphthoquione (500 mg, 2.20 mmol), K<sub>2</sub>CO<sub>3</sub> (760 mg, 5.51 mmol), and ethyl acetoacetate (0.31 mL, 2.42 mmol) in MeCN (30 mL) was stirred at reflux temperature for 6 h. After completion, the reaction mixture was diluted with EtOAc and filtered through Celite, and the filtrate was evaporated <i>in vacuo</i> to yield the crude product. The residue was purified by flash column chromatography (EtOAc/hexanes, 1/4 to 2/3) to give <b>17</b> as a yellow solid (305 mg, 48% yield). <i>R<sub>f</sub></i> (30% EtOAc/hexanes) 0.52; mp 149.3–159.9 °C; <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.42–7.99 (m, 2H), 7.98–7.53 (m, 2H), 4.43 (q, <i>J</i> = 7.2 Hz, 2H), 2.70 (s, 3H), 1.43 (t, <i>J</i> = 7.1 Hz, 3H); <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>) δ 178.7, 173.5, 164.4, 162.0, 151.3, 134.1, 133.7, 133.6, 131.5, 128.2, 127.4, 126.5, 113.8, 61.5, 14.2, 14.2.</div></div><div id="sec5_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i20" class="anchor-spacer"></div><h3 class="article-section__title" id="_i20"> 2-Methyl-4,9-dioxo-4,9-dihydronaphtho[2,3-<i>b</i>]furan-3-carboxylic Acid (<b>18</b>)</h3><div class="NLM_p last">To a stirred solution of <b>17</b> (378 mg, 1.33 mmol) in THF/H<sub>2</sub>O (0.2 M, 7/3 v/v) was added LiOH (167 mg, 3.99 mmol) at room temperature. After being stirred at room temperature for 3 h, the reaction mixture was quenched with 1 N HCl (adjust to pH ∼ 1, aqueous layer will show transparent), extracted with EtOAc (15 mL × 3), dried over Na<sub>2</sub>SO<sub>4</sub>, and concentrated <i>in vacuo</i>. The residue was purified by flash column chromatography (MeOH/CH<sub>2</sub>Cl<sub>2</sub>, 0/100 to 1/9) to give <b>18</b> as a yellow solid (324 mg, 95% yield). <i>R<sub>f</sub></i> (5% MeOH/CH<sub>2</sub>Cl<sub>2</sub>) 0.35; IR (neat) 3425, 3022, 2975, 2920, 2712, 1744, 1673, 1581, 1413, 1217, 1192, 995, 721 cm<sup>–1</sup>; <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.25 (td, <i>J</i> = 7.4, 1.2 Hz, 2H), 7.84 (dtd, <i>J</i> = 18.8, 7.5, 1.3 Hz, 2H), 2.88 (s, 3H); <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>) δ 184.9, 172.4, 169.0, 160.9, 150.9, 135.8, 134.5, 131.8, 131.7, 128.0, 127.5, 126.0, 112.8, 14.5.</div></div><div id="sec5_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i21" class="anchor-spacer"></div><h3 class="article-section__title" id="_i21"> <i>N</i>-(3-(1<i>H</i>-Imidazol-1-yl)propyl)-2-methyl-4,9-dioxo-4,9-dihydronaphtho[2,3-<i>b</i>]furan-3-carboxamide (AS4583)</h3><div class="NLM_p">To a stirred solution of <b>18</b> (256.2 mg, 1.0 mmol), DMAP (12.2 mg, 0.1 mmol), and HBTU (455 mg, 1.2 mmol) in DMF (5 mL) was added 1-(3-aminopropyl)imidazole (180 μL, 1.5 mmol) at room temperature (<a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>). The solvent was removed by vacuum after the reaction mixture was stirred at room temperature for 1 day. Then the crude was washed by H<sub>2</sub>O, extracted with CH<sub>2</sub>Cl<sub>2</sub> (10 mL × 3), dried over Na<sub>2</sub>SO<sub>4</sub>, and concentrated <i>in vacuo</i>. The residue was purified by flash column chromatography (MeOH/CH<sub>2</sub>Cl<sub>2</sub>, 0/100 to 1/49) to give AS4583 as a yellow solid (240 mg, 66% yield). <i>R<sub>f</sub></i> (5% MeOH/CH<sub>2</sub>Cl<sub>2</sub>) 0.28; IR (neat) 3285, 3095, 2923, 2848, 1650, 1582, 1213, 991, 844, 715 cm<sup>–1</sup>; <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 9.79 (s, 1H), 8.21 (qd, <i>J</i> = 4.0, 1.7 Hz, 2H), 7.83–7.76 (m, 2H), 7.59 (s, 1H), 7.07 (s, 1H), 7.00 (s, 1H), 4.11 (t, <i>J</i> = 7.0 Hz, 2H), 3.46 (dd, <i>J</i> = 12.5, 6.2 Hz, 2H), 2.87 (s, 3H), 2.16 (quint, <i>J</i> = 6.8 Hz, 2H); <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>) δ 183.3, 172.9, 166.5, 161.5, 151.1, 137.2, 134.8, 134.2, 132.7, 131.4, 129.3, 127.8, 126.9, 125.9, 118.9, 115.1, 44.5, 36.2, 30.9, 14.9; HRMS (ESI<sup>+</sup>, TOF) calcd for C<sub>20</sub>H<sub>18</sub>N<sub>3</sub>O<sub>4</sub> [M + H]<sup>+</sup> 364.1297, found 364.1289.</div><figure id="sch2" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b01783/20200323/images/medium/jm9b01783_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b01783/20200323/images/large/jm9b01783_0008.jpeg" id="gr10" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of AS4583</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b01783/20200323/images/large/jm9b01783_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01783&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec5_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i23" class="anchor-spacer"></div><h3 class="article-section__title" id="_i23"> <i>N</i>-(2-(1<i>H</i>-Imidazol-1-yl)ethyl)-2-methyl-4,9-dioxo-4,9-dihydronaphtho[2,3-<i>b</i>]furan-3-carboxamide (<b>1</b>)</h3><div class="NLM_p last">To a stirred solution of <b>18</b> (0.17 mmol), DMAP (21 mg, 0.17 mmol), and HBTU (97 mg, 0.26 mmol) in DMF (0.85 mL) was added 2-imidazol-1ylethylamine (47 mg, 0.43 mmol) at room temperature. The solvent was removed by vacuum after the reaction mixture was stirred at room temperature for 1 day. Then the crude was washed by H<sub>2</sub>O, extracted with CH<sub>2</sub>Cl<sub>2</sub> (10 mL × 3), dried over Na<sub>2</sub>SO<sub>4</sub>, and concentrated <i>in vacuo</i>. The residue was purified by flash column chromatography (MeOH/CH<sub>2</sub>Cl<sub>2</sub>, 0/100 to 1/49) to give <b>1</b> as a yellow solid (42 mg, 70% yield). <i>R<sub>f</sub></i> (5% MeOH/CH<sub>2</sub>Cl<sub>2</sub>) 0.28; <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD) δ 8.27–8.14 (m, 3H), 7.92–7.81 (m, 2H), 7.43 (s, 1H), 7.22 (s, 1H), 4.41 (t, <i>J</i> = 6.0 Hz, 1H), 3.86 (t, <i>J</i> = 6.0 Hz, 1H), 2.77 (s, 3H); <sup>13</sup>C NMR (100 MHz, CD<sub>3</sub>OD) δ 184.0, 174.2, 167.1, 163.7, 152.6, 138.8, 135.9, 135.4, 134.2, 132.9, 129.3, 128.6, 127.5, 127.2, 120.8, 115.8, 47.2, 41.3 14.7; HRMS (ESI<sup>+</sup>, TOF) calcd for C<sub>19</sub>H<sub>16</sub>N<sub>3</sub>O<sub>4</sub> [M + H]<sup>+</sup> 350.1141, found 350.1135. Sample for biology test was purified by preparative HPLC with HPLC purity of >99%.</div></div><div id="sec5_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i24" class="anchor-spacer"></div><h3 class="article-section__title" id="_i24"> <i>N</i>-(4-(1<i>H</i>-Imidazol-1-yl)butyl)-2-methyl-4,9-dioxo-4,9-dihydronaphtho[2,3-<i>b</i>]furan-3-carboxamide (<b>2</b>)</h3><div class="NLM_p last">To a stirred solution of <b>18</b> (95 mg, 0.37 mmol), DMAP (45 mg, 0.37 mmol), and HBTU (154 mg, 0.41 mmol) in DMF (1.9 mL) was added 4-(1<i>H</i>-imidazol-1-yl)butan-1-amine (77 mg, 0.56 mmol) at room temperature. The solvent was removed by vacuum after the reaction mixture was stirred at room temperature for 18 h. Then the crude was washed by H<sub>2</sub>O, extracted with CH<sub>2</sub>Cl<sub>2</sub> (10 mL × 3), dried over Na<sub>2</sub>SO<sub>4</sub>, and concentrated <i>in vacuo</i>. The residue was purified by flash column chromatography (MeOH/CH<sub>2</sub>Cl<sub>2</sub>, 0/100 to 1/49) to give <b>2</b> as a yellow solid (97 mg, 70% yield). <i>R<sub>f</sub></i> (5% MeOH/CH<sub>2</sub>Cl<sub>2</sub>) 0.25; mp 158.6–164.1 °C; <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 9.70 (s, 1H), 8.18 (d, <i>J</i> = 7.5 Hz, 2H), 7.83–7.75 (m, 2H), 7.49 (s, 1H), 7.03 (s, 1H), 6.93 (s, 1H), 4.01 (t, <i>J</i> = 7.1 Hz, 2H), 3.47 (dd, <i>J</i> = 12.6, 6.4 Hz, 2H), 2.86 (s, 3H), 1.94 (dt, <i>J</i> = 15.1, 7.4 Hz, 2H), 1.69 (dt, <i>J</i> = 14.2, 6.9 Hz, 2H); <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>) δ 183.1, 172.8, 166.4, 161.3, 151.0, 137.0, 134.7, 134.1, 132.7, 131.4, 129.3, 127.6, 126.7, 125.9, 118.7, 115.2, 46.6, 38.5, 28.5, 26.4, 14.9; HRMS (ESI<sup>+</sup>, TOF) calcd for C<sub>21</sub>H<sub>20</sub>N<sub>3</sub>O<sub>4</sub> [M + H]<sup>+</sup> 378.1454, found 378.1448. Sample for biology test was purified by preparative HPLC with HPLC purity of >99%.</div></div><div id="sec5_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i25" class="anchor-spacer"></div><h3 class="article-section__title" id="_i25"> <i>N</i>-(3-(1<i>H</i>-Imidazol-1-yl)propyl)-2-ethyl-4,9-dioxo-4,9-dihydronaphtho[2,3-<i>b</i>]furan-3-carboxamide (<b>3</b>)</h3><div class="NLM_p last">To a stirred solution of 2-ethyl-4,9-dioxo-4,9-dihydronaphtho[2,3-<i>b</i>]furan-3-carboxylic acid (92 mg, 0.34 mmol), DMAP (42 mg, 0.34 mmol), and HBTU (142 mg, 0.37 mmol) in DMF (1.7 mL) was added 1-(3-aminopropyl)imidazole (49 μL, 0.41 mmol) at room temperature. The solvent was removed by vacuum after the reaction mixture was stirred at room temperature for 1 day. Then the crude was washed by H<sub>2</sub>O, extracted with CH<sub>2</sub>Cl<sub>2</sub> (10 mL × 3), dried over Na<sub>2</sub>SO<sub>4</sub>, and concentrated <i>in vacuo</i>. The residue was purified by flash column chromatography (MeOH/CH<sub>2</sub>Cl<sub>2</sub>, 0/100 to 1/19) to give <b>3</b> as a yellow solid (116 mg, 90% yield). <i>R<sub>f</sub></i> (5% MeOH/CH<sub>2</sub>Cl<sub>2</sub>) 0.31; <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 9.82 (s, 1H), 8.26–8.18 (m, 2H), 7.84–7.74 (m, 2H), 7.66 (s, 1H), 7.08 (s, 1H), 7.02 (s, 1H), 4.12 (t, <i>J</i> = 7.0 Hz, 2H), 3.46 (dd, <i>J</i> = 12.5, 6.2 Hz, 2H), 3.35 (q, <i>J</i> = 7.5 Hz, 2H), 2.24–2.10 (m, 2H), 1.36 (t, <i>J</i> = 7.5 Hz, 3H); <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>) δ 183.5, 173.0, 171.2, 161.5, 151.3, 137.3, 134.8, 134.2, 132.8, 131.6, 129.5, 127.8, 126.9, 118.9, 114.4, 44.6, 36.3, 31.0, 22.1, 12.0; HRMS (MALDI<sup>+</sup>, TOF) calcd for C<sub>21</sub>H<sub>20</sub>N<sub>3</sub>O<sub>4</sub> [M + H]<sup>+</sup> 378.1448, found 378.1457. Sample for biology test was purified by preparative HPLC with HPLC purity of >99%.</div></div><div id="sec5_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i26" class="anchor-spacer"></div><h3 class="article-section__title" id="_i26"> <i>N</i>-(3-(1<i>H</i>-Imidazol-1-yl)propyl)-2-isopropyl-4,9-dioxo-4,9-dihydronaphtho[2,3-<i>b</i>]furan-3-carboxamide (<b>4</b>)</h3><div class="NLM_p last">To a stirred solution of 2-isopropyl-4,9-dioxo-4,9-dihydronaphtho[2,3-<i>b</i>]furan-3-carboxylic acid (41.8 mg, 0.15 mmol), DMAP (18 mg, 0.15 mmol), and HBTU (61 mg, 0.16 mmol) in DMF (0.74 mL) was added 1-(3-aminopropyl)imidazole (21 μL, 0.18 mmol) at room temperature. The solvent was removed by vacuum after the reaction mixture was stirred at room temperature for 1 day. Then the crude was washed by H<sub>2</sub>O, extracted with CH<sub>2</sub>Cl<sub>2</sub> (10 mL × 3), dried over Na<sub>2</sub>SO<sub>4</sub>, and concentrated <i>in vacuo</i>. The residue was purified by flash column chromatography (MeOH/CH<sub>2</sub>Cl<sub>2</sub>, 1/199 to 1/19) to give <b>4</b> as a yellow solid (52 mg, 90% yield). <i>R<sub>f</sub></i> (5% MeOH/CH<sub>2</sub>Cl<sub>2</sub>) 0.32; <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 9.83 (s, 1H), 8.20 (td, <i>J</i> = 5.3, 1.8 Hz, 2H), 7.83–7.73 (m, 2H), 7.72 (s, 1H), 7.08 (s, 1H), 7.05 (s, 1H), 4.34–4.27 (m, 1H), 4.14 (t, <i>J</i> = 7.0 Hz, 2H), 3.46 (dd, <i>J</i> = 12.4, 6.2 Hz, 2H), 2.17 (quint, <i>J</i> = 6.8 Hz, 2H), 1.37 (s, 3H), 1.36 (s, 3H); <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>) δ 183.6, 174.1, 172.9, 161.6, 151.3, 134.8, 134.2, 132.8, 131.6, 128.7, 127.7, 126.8, 125.9, 113.5, 44.8, 36.2, 30.9, 27.7, 20.4; HRMS (ESI<sup>+</sup>, TOF) calcd for C<sub>22</sub>H<sub>22</sub>N<sub>3</sub>O<sub>4</sub> [M + H]<sup>+</sup> 392.1610, found 392.1605. Sample for biology test was purified by preparative HPLC with HPLC purity of >98%.</div></div><div id="sec5_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i27" class="anchor-spacer"></div><h3 class="article-section__title" id="_i27"> <i>N</i>-(3-(1<i>H</i>-Imidazol-1-yl)propyl)-4,9-dioxo-2-phenyl-4,9-dihydronaphtho[2,3-<i>b</i>]furan-3-carboxamide (<b>5</b>)</h3><div class="NLM_p last">To a stirred solution of 4,9-dioxo-2-phenyl-4,9-dihydronaphtho[2,3-<i>b</i>]furan-3-carboxylic acid (50.9 mg, 0.16 mmol), DMAP (19.5 mg, 0.16 mmol), and HBTU (73 mg, 0.19 mmol) in DMF (0.8 mL) was added 1-(3-aminopropyl)imidazole (29 μL, 0.24 mmol) at room temperature. The solvent was removed by vacuum after the reaction mixture was stirred at room temperature for 1 day. Then the crude was washed by H<sub>2</sub>O, extracted with CH<sub>2</sub>Cl<sub>2</sub> (10 mL × 3), dried over Na<sub>2</sub>SO<sub>4</sub>, and concentrated <i>in vacuo</i>. The residue was purified by flash column chromatography (MeOH/CH<sub>2</sub>Cl<sub>2</sub>, 0/100 to 1/19) to give <b>5</b> as a yellow solid (38 mg, 56% yield). <i>R<sub>f</sub></i> (5% MeOH/CH<sub>2</sub>Cl<sub>2</sub>) 0.32; <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 9.22 (s, 1H), 8.26–8.22 (m, 1H), 8.12–8.10 (m, 1H), 7.92–7.89 (m, 2H), 7.84–7.80 (m, 2H), 7.62 (s, 1H), 7.50–7.44 (m, 3H), 7.06 (s, 1H), 7.01 (s, 1H), 4.14 (t, <i>J</i> = 7.0 Hz, 2H), 3.48 (dd, <i>J</i> = 12.5, 6.3 Hz, 2H), 2.18 (quint, <i>J</i> = 6.8 Hz, 2H); HRMS (ESI<sup>+</sup>, TOF) calcd for C<sub>25</sub>H<sub>20</sub>N<sub>3</sub>O<sub>4</sub> [M + H]<sup>+</sup> 426.1454, found 426.1451. Sample for biology test was purified by preparative HPLC with HPLC purity of >99%.</div></div><div id="sec5_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i28" class="anchor-spacer"></div><h3 class="article-section__title" id="_i28"> <i>N</i>-(3-(1<i>H</i>-Imidazol-1-yl)propyl)-2-methyl-4,9-dioxo-4,9-dihydro-1<i>H</i>-benzo[<i>f</i>]indole-3-carboxamide (<b>6</b>)</h3><div class="NLM_p last">To a stirred solution of 2-methyl-4,9-dioxo-4,9-dihydro-1<i>H</i>-benzo[<i>f</i>]indole-3-carboxylic acid (23 mg, 0.09 mmol), DMAP (17 mg, 0.14 mmol), and HBTU (51 mg, 0.14 mmol) in DMF (0.45 mL) was added 1-(3-aminopropyl)imidazole (13 μL, 0.11 mmol) at 0 °C. The solvent was removed by vacuum after the reaction mixture was stirred at room temperature for 2 days. The residue was purified by flash column chromatography to give <b>6</b> as a yellow solid (9.1 mg, 28% yield). <i>R<sub>f</sub></i> (5% MeOH/CH<sub>2</sub>Cl<sub>2</sub>) 0.20; <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 10.33 (t, <i>J</i> = 5.4 Hz, 1H), 8.24–8.10 (m, 1H), 8.10–7.98 (m, 1H), 7.75–7.63 (m, 2H), 7.60 (s, 1H), 7.00 (s, 1H), 6.99 (s, 1H), 4.10 (t, <i>J</i> = 7.1 Hz, 2H), 3.47 (q, <i>J</i> = 6.1 Hz, 2H), 2.67 (s, 3H), 2.17–2.05 (m, 2H); HRMS (ESI<sup>+</sup>, TOF) calcd for C<sub>20</sub>H<sub>19</sub>N<sub>4</sub>O<sub>3</sub> [M + H]<sup>+</sup> 363.1457, found 363.1454. Sample for biology test was purified by preparative HPLC with HPLC purity of >99%.</div></div><div id="sec5_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i29" class="anchor-spacer"></div><h3 class="article-section__title" id="_i29"> <i>N</i>-(3-(1<i>H</i>-Pyrrol-1-yl)propyl)-2-methyl-4,9-dioxo-4,9-dihydronaphtho[2,3-<i>b</i>]furan-3-carboxamide (<b>7</b>)</h3><div class="NLM_p last">To a stirred solution of <b>18</b> (100 mg, 0.39 mmol), DMAP (47 mg, 0.39 mmol), and HBTU (163 mg, 0.43 mmol) in DMF (1.95 mL) was added 3-(1<i>H</i>-pyrrol-1-yl)propan-1-amine (58 mg, 0.47 mmol) at room temperature. The solvent was removed by vacuum after the reaction mixture was stirred at room temperature for 1 day. Then the crude was washed by H<sub>2</sub>O, extracted with CH<sub>2</sub>Cl<sub>2</sub> (10 mL × 3), dried over Na<sub>2</sub>SO<sub>4</sub>, and concentrated <i>in vacuo</i>. The residue was purified by flash column chromatography (MeOH/CH<sub>2</sub>Cl<sub>2</sub>, 0/100 to 1/49) to give <b>7</b> as a yellow solid (67 mg, 47% yield). <i>R<sub>f</sub></i> (CH<sub>2</sub>Cl<sub>2</sub>) 0.07; mp 143.5–150.7 °C; <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 9.73 (s, 1H), 8.26–8.13 (m, 2H), 7.84–7.71 (m, 2H), 6.74 (t, <i>J</i> = 2.1 Hz, 2H), 6.14 (t, <i>J</i> = 2.1 Hz, 2H), 4.05 (t, <i>J</i> = 6.9 Hz, 2H), 3.42 (dd, <i>J</i> = 12.3, 6.6 Hz, 2H), 2.86 (s, 3H), 2.14 (quint, <i>J</i> = 6.8 Hz, 2H); <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>) δ 183.1, 172.9, 166.5, 161.3, 151.0, 134.7, 134.1, 132.8, 131.5, 127.7, 126.8, 126.0, 120.6, 115.3, 108.2, 47.0, 36.6, 31.2, 14.9; HRMS (MALDI<sup>+</sup>, TOF) calcd for C<sub>21</sub>H<sub>19</sub>N<sub>2</sub>O<sub>4</sub> [M + H]<sup>+</sup> 363.1339, found 363.1325. Sample for biology test was purified by preparative HPLC with HPLC purity of >99%.</div></div><div id="sec5_12" class="NLM_sec NLM_sec_level_2"><div id="ac_i30" class="anchor-spacer"></div><h3 class="article-section__title" id="_i30"> 2<b>-</b>Methyl<b>-</b><i>N</i><b>-</b>(3-morpholinopropyl)<b>-</b>4,9<b>-</b>dioxo<b>-</b>4,9<b>-</b>dihydronaphtho[2,3<b>-</b><i>b</i>]furan-3-carboxamide (<b>8</b>)</h3><div class="NLM_p last">To a stirred solution of <b>18</b> (23.8 mg, 0.093 mmol), DMAP (17 mg, 0.14 mmol), and HBTU (54 mg, 0.14 mmol) in DMF (0.47 mL) was added 3-morpholinopropylamine (20.4 μL, 0.14 mmol) at 0 °C. The solvent was removed by vacuum after the reaction mixture was stirred at room temperature for 2 days. The residue was purified by flash column chromatography (MeOH/CH<sub>2</sub>Cl<sub>2</sub>, 0/100 to 1/9) to give <b>8</b> as a yellow solid. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 9.69 (s, 1H), 8.26–8.14 (m, 2H), 7.83–7.71 (m, 2H), 3.81–3.66 (m, 4H), 3.51 (dd, <i>J</i> = 12.5, 6.8 Hz, 2H), 2.87 (s, 3H), 2.60–2.43 (m, 6H), 1.88 (quint, <i>J</i> = 7.2 Hz, 2H); HRMS (ESI<sup>+</sup>, TOF) calcd for C<sub>21</sub>H<sub>23</sub>N<sub>2</sub>O<sub>5</sub> [M + H]<sup>+</sup> 383.1607, found 383.1604. Sample for biology test was purified by preparative HPLC with HPLC purity of >99%.</div></div><div id="sec5_13" class="NLM_sec NLM_sec_level_2"><div id="ac_i31" class="anchor-spacer"></div><h3 class="article-section__title" id="_i31"> 2<b>-</b>Methyl-4,9-dioxo-<i>N</i>-(3-(pyrrolidin-1-yl)propyl)-4,9-dihydronaphtho[2,3-<i>b</i>]furan-3-carboxamide (<b>9</b>)</h3><div class="NLM_p last">To a stirred solution of <b>18</b> (80 mg, 0.31 mmol), DMAP (57 mg, 0.47 mmol), and HBTU (177 mg, 0.47 mmol) in DMF (1.6 mL) was added 1-(3-aminopropyl)pyrrolidine (59 μL, 0.60 mmol) at 0 °C. The solvent was removed by vacuum after the reaction mixture was stirred at room temperature for 36 h. The residue was purified by flash column chromatography (MeOH/EtOAc/CH<sub>2</sub>Cl<sub>2</sub>, 0/1/1 to 1/4/5) to give <b>9</b> as a yellow solid (47 mg, 41% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 9.72 (s, 1H), 8.20–8.18 (m, 2H), 7.84–7.68 (m, 2H), 3.51 (dd, <i>J</i> = 12.5, 6.7 Hz, 2H), 2.86 (s, 3H), 2.66 (t, <i>J</i> = 7.6 Hz, 2H), 2.62 (brs, 4H), 1.96–1.86 (m, 2H), 1.81 (brs, 4H); HRMS (ESI<sup>+</sup>, TOF) calcd for C<sub>21</sub>H<sub>23</sub>N<sub>2</sub>O<sub>4</sub> [M + H]<sup>+</sup> 367.1658, found 367.1657. Sample for biology test was purified by preparative HPLC with HPLC purity of >99%.</div></div><div id="sec5_14" class="NLM_sec NLM_sec_level_2"><div id="ac_i32" class="anchor-spacer"></div><h3 class="article-section__title" id="_i32"> 2-Methyl-4,9-dioxo-<i>N</i>-(4-phenylbutyl)-4,9-dihydronaphtho[2,3-<i>b</i>]furan-3-carboxamide (<b>10</b>)</h3><div class="NLM_p last">To a stirred solution of <b>18</b> (90 mg, 0.35 mmol), DMAP (43 mg, 0.35 mmol), and HBTU (146 mg, 0.39 mmol) in DMF (1.8 mL) was added 4-phenylbutylamine (72 μL, 0.46 mmol) at 0 °C. The solvent was removed by vacuum after the reaction mixture was stirred at room temperature for 20 h. The residue was purified by flash column chromatography (EtOAc/hexanes, 1/49 to 1/9) to give <b>10</b> as a yellow solid (78 mg, 58% yield). <i>R<sub>f</sub></i> (20% EtOAc/hexanes) 0.35; mp 151.4–152.7 °C; <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 9.66 (s, 1H), 8.26–8.11 (m, 2H), 7.87–7.68 (m, 2H), 7.34–7.09 (m, 5H), 3.47 (dd, <i>J</i> = 12.2, 6.6 Hz, 2H), 2.87 (s, 3H), 2.69 (t, <i>J</i> = 7.3 Hz, 2H), 1.88–1.66 (m, 4H); <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>) δ 182.9, 172.9, 166.3, 161.1, 150.9, 142.2, 134.5, 134.0, 132.8, 131.5, 128.4, 128.2, 127.6, 126.7, 126.0, 125.7, 115.5, 39.2, 35.5, 28.9, 28.8, 14.9; HRMS (ESI<sup>+</sup>, TOF) calcd for C<sub>24</sub>H<sub>21</sub>NO<sub>4</sub>Na [M + Na]<sup>+</sup> 410.1368, found 410.1374. Sample for biology test was purified by preparative HPLC with HPLC purity of >99%.</div></div><div id="sec5_15" class="NLM_sec NLM_sec_level_2"><div id="ac_i33" class="anchor-spacer"></div><h3 class="article-section__title" id="_i33"> <i>N</i>-(3-(1<i>H</i>-Imidazol-1-yl) propyl)-2-methylbenzofuran-3-carboxamide (<b>11</b>)</h3><div class="NLM_p last">To a stirred solution of 2-methylbenzofuran-3-carboxylic acid (25 mg, 0.14 mmol), DMAP (25.7 mg, 0.21 mmol), and HBTU (80 mg, 0.21 mmol) in DMF (0.7 mL) was added 1-(3-aminopropyl)imidazole (19 μL, 0.16 mmol) at 0 °C. The solvent was removed by vacuum after the reaction mixture was stirred at room temperature for 1 day. The residue was purified by flash column chromatography (MeOH/CH<sub>2</sub>Cl<sub>2</sub>, 0/100 to 1/9) to give <b>11</b> as a colorless oil (37 mg, 95% yield). <i>R<sub>f</sub></i> (5% MeOH/CH<sub>2</sub>Cl<sub>2</sub>) 0.29; <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.58–7.48 (m, 2H), 7.48–7.41 (m, 1H), 7.32–7.25 (m, 2H), 7.05 (s, 1H), 6.97 (s, 1H), 5.97 (s, 1H), 4.07 (t, <i>J</i> = 7.0 Hz, 2H), 3.51 (dd, <i>J</i> = 13.1, 6.7 Hz, 2H), 2.71 (s, 3H), 2.15 (quint, <i>J</i> = 6.9 Hz, 2H); HRMS (ESI<sup>+</sup>, TOF) calcd for C<sub>16</sub>H<sub>18</sub>N<sub>3</sub>O<sub>2</sub> [M + H]<sup>+</sup> 284.1399, found 284.1392. Sample for biology test was purified by preparative HPLC with HPLC purity of >99%.</div></div><div id="sec5_16" class="NLM_sec NLM_sec_level_2"><div id="ac_i34" class="anchor-spacer"></div><h3 class="article-section__title" id="_i34"> 7-Fluoro-2-hydroxynaphthalene-1,4-dione (<b>13</b>)</h3><div class="NLM_p last">To a solution of KO<i><sup>t</sup></i>Bu (547 mg, 4.873 mmol) in anhydrous <i>t</i>-BuOH (7.6 mL) which had previously been saturated with O<sub>2</sub> (bubbled for 10 min) was added a solution of 7-fluoro-1-tetralone <b>12</b> (100 mg, 0.609 mmol) in anhydrous <i>t</i>-BuOH (2.4 mL), and the resulting red reaction mixture was stirred at ambient temperature for 30 min under O<sub>2</sub> atmosphere. Then, the reaction mixture was cooled down to 0 °C and acidified with 3 M HCl until the pH of the mixture became 1–2. The volatiles were removed in vacuo, and the crude material was extracted with CH<sub>2</sub>Cl<sub>2</sub>. Combined organic layers were dried over anhydrous sodium sulfate and then filtered and concentrated under vacuum to afford 7-fluoro-2-hydroxynaphthalene-1,4-dione <b>13</b> as a yellow solid (108 mg, 92% yield). <i>R<sub>f</sub></i> (EtOAc) 0.38; mp 131.6–167.6 °C (decomposed); IR (neat) 3070, 1685, 1650, 1598, 1384, 1349, 1312, 1205, 897, 888, 876, 844, 772 cm<sup>–1</sup>; <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>) δ 8.16 (dd, <i>J</i> = 8.5, 5.2 Hz, 1H), 7.77 (dd, <i>J</i> = 8.1, 2.5 Hz, 1H), 7.46 (td, <i>J</i> = 8.4, 2.6 Hz, 1H), 7.27 (brs, 1H), 6.37 (s, 1H); <sup>13</sup>C NMR (150 MHz, CDCl<sub>3</sub>) δ 183.8, 181.2, 166.4, 164.7, 156.4, 131.8, 131.7, 129.8, 129.8, 129.4, 122.3, 122.1, 113.4, 113.2, 110.9; <sup>19</sup>F NMR (565 MHz, CDCl<sub>3</sub>) δ −104.04; HRMS (EI<sup>+</sup>, TOF) calcd for C<sub>10</sub>H<sub>5</sub>FO<sub>3</sub> [M]<sup>+</sup> 192.0223, found 192.0221.</div></div><div id="sec5_17" class="NLM_sec NLM_sec_level_2"><div id="ac_i35" class="anchor-spacer"></div><h3 class="article-section__title" id="_i35"> 3-(2-Ethoxy-2-oxo-1-(tributylphosphonio)ethyl)-7-fluoro-1,4-dioxo-1,4-dihydronaphthalen-2-olate (<b>14</b>)</h3><div class="NLM_p last">To compound <b>13</b> (138 mg, 0.718 mmol) with THF (1.0 mL) in a dry and nitrogen-flushed sealed tube at 0 °C were sequentially added ethyl glyoxalate (50% solution in toluene) (263 μL, 1.8 equiv), trifluoroacetic acid (96 μL, 1.8 equiv), and Bu<sub>3</sub>P (210 μL, 1.2 equiv), and then the reaction mixture was stirred for 3.5 h at 60–64 °C. Solvent was removed under vacuum to obtain the crude, which was then purified by flash chromatography in neutral Al<sub>2</sub>O<sub>3</sub> (EtOAc/<i>n</i>-hexane, 1/1) to provide compound <b>14</b> as a red gummy liquid (190.0 mg, 63% yield). <i>R<sub>f</sub></i> (1% MeOH/CH<sub>2</sub>Cl<sub>2,</sub> neutral Al<sub>2</sub>O<sub>3</sub>) 0.52; IR (neat) 2960, 2933, 2873, 1718, 1686, 1595, 1377, 1266, 1228 cm<sup>–1</sup>; <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.05–7.98 (m, 1H), 7.65 (dd, <i>J</i> = 6.6, 2.0 Hz, 1H), 7.26 (dd, <i>J</i> = 11.2, 5.4 Hz, 1H), 5.31 (dd, <i>J</i> = 15.6, 1.1 Hz, 1H), 4.17–4.09 (m, 2H), 2.40–2.21 (m, 3H), 2.21–2.03 (m, 3H), 1.61–1.32 (m, 12H), 1.13 (td, <i>J</i> = 7.0, 1.3 Hz, 3H), 0.91–0.83 (m, 9H); <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>) δ 183.4, 178.5, 172.0, 169.4, 165.7, 163.1, 134.0, 133.9, 131.3, 128.2, 128.1, 120.2, 120.0, 112.9, 112.6, 105.6, 105.5, 62.2, 37.2, 36.7, 24.1, 24.1, 24.0, 23.9, 20.0, 19.6, 13.8, 13.2; <sup>19</sup>F NMR (376 MHz, CDCl<sub>3</sub>) δ −108.70; HRMS (MALDI<sup>+</sup>, TOF) calcd for C<sub>26</sub>H<sub>36</sub>FO<sub>5</sub>P [M + H]<sup>+</sup> 479.2362, found 479.2376.</div></div><div id="sec5_18" class="NLM_sec NLM_sec_level_2"><div id="ac_i36" class="anchor-spacer"></div><h3 class="article-section__title" id="_i36"> Ethyl-7-fluoro-2-methyl-4,9-dioxo-4,9-dihydronaphtho[2,3-<i>b</i>]furan-3-carboxylate (<b>15</b>)</h3><div class="NLM_p last">To compound <b>14</b> (393 mg, 0.821 mmol) with THF (4.1 mL) in a dry and nitrogen-flushed round-bottom flask at 0 °C was slowly added acetyl chloride (76 μL, 1.3 equiv) and Et<sub>3</sub>N (171 μL, 1.5 equiv), and then the mixture was stirred for 15 min. Solvent was removed under vacuum to obtain the crude, which was then purified by flash chromatography (EtOAc/<i>n</i>-hexane, 1/9) to provide compound <b>15</b> as pale yellow solid (199.0 mg, 61% yield). <i>R<sub>f</sub></i> (20% EtOAc/hexanes) 0.53; mp 156.5–160.2 °C; IR (neat) 2986, 2937, 1716, 1688, 1597, 1542, 1264, 1234, 1157, 1115 cm<sup>–1</sup>; <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>) δ 8.25–8.11 (m, 1H), 7.86–7.69 (m, 1H), 7.38 (t, <i>J</i> = 8.0 Hz, 1H), 4.46–4.35 (q, <i>J</i> = 7.2 Hz, 2H), 2.70 (s, 3H), 1.43 (t, <i>J</i> = 7.2 Hz, 3H); <sup>13</sup>C NMR (150 MHz, CDCl<sub>3</sub>) δ 177.4, 172.2, 166.9, 165.2, 164.8, 161.8, 151.2, 134.2, 134.2, 130.5, 130.4, 130.1, 128.4, 121.0, 120.8, 113.9, 113.5, 113.4, 61.6, 14.2; <sup>19</sup>F NMR (565 MHz, CDCl<sub>3</sub>) δ −103.31; HRMS (EI<sup>+</sup>, TOF) calcd for C<sub>16</sub>H<sub>11</sub>FO<sub>5</sub> [M]<sup>+</sup> 302.0591, found 302.0594.</div></div><div id="sec5_19" class="NLM_sec NLM_sec_level_2"><div id="ac_i37" class="anchor-spacer"></div><h3 class="article-section__title" id="_i37"> 7-Fluoro-2-methyl-4,9-dioxo-4,9-dihydronaphtho[2,3-<i>b</i>]furan-3-carboxylic Acid (<b>16</b>)</h3><div class="NLM_p last">To compound <b>15</b> (355 mg, 1.175 mmol) with THF (9.4 mL) and H<sub>2</sub>O (2.35 mL) in a round-bottom flask at 0 °C was added 3 M NaOH<sub>(aq)</sub> (1.17 mL, 4.5 equiv), and then the mixture was allowed warm to room temperature for a period of 5 h. Solvent was removed under vacuum, and the sample was then diluted with H<sub>2</sub>O and washed with CH<sub>2</sub>Cl<sub>2</sub>. The aqueous layer was added 3 M HCl<sub>(aq)</sub> until pH = 1∼2, then extracted with EA and Brine. Combined organic layers were dried over anhydrous sodium sulfate and then filtered and concentrated under vacuum to afford the crude carboxylic acid <b>16</b> (313 mg, 97% yield). <i>R<sub>f</sub></i> (10% MeOH/CH<sub>2</sub>Cl<sub>2</sub>) 0.38; mp 254.5–260.0 °C (decomposed); IR (neat) 3447, 2978, 1684, 1647, 1586, 1550, 1354, 1236 cm<sup>–1</sup>; <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>) δ 12.87 (s, 1H), 8.45–8.23 (m, 1H), 7.91 (d, <i>J</i> = 6.9 Hz, 1H), 7.48 (s, 1H), 2.90 (s, 3H); <sup>19</sup>F NMR (565 MHz, CDCl<sub>3</sub>) δ −98.39; HRMS (EI<sup>+</sup>, TOF) calcd for C<sub>14</sub>H<sub>7</sub>FO<sub>5</sub> [M]<sup>+</sup> 274.0278, found 274.0280.</div></div><div id="sec5_20" class="NLM_sec NLM_sec_level_2"><div id="ac_i38" class="anchor-spacer"></div><h3 class="article-section__title" id="_i38"> <i>N</i>-(3-(1<i>H</i>-Imidazol-1-yl)propyl)-7-fluoro-2-methyl-4,9-dioxo-4,9-dihydronaphtho[2,3-b]furan-3-carboxamide (RJ-LC-07-48)</h3><div class="NLM_p last">To a solution of carboxylic acid <b>16</b> (282 mg, 1.028 mmol) with DMF (10.3 mL) in a round-bottom flask at room temperature was added DMAP (126 mg, 1.028 mmol), HBTU (468 mg, 1.234 mmol), 3-(1<i>H</i>-imidazol-1-yl)propan-1-amine (184 μL, 1.542 mmol), and then the mixture was stirred for 6 h. Solvent was removed under vacuum to obtain the crude material, which was then purified by flash chromatography (MeOH/CH<sub>2</sub>Cl<sub>2,</sub> 1/9) to provide product RJ-LC-07-48 (251 mg, 64% yield). <i>R<sub>f</sub></i> (10% MeOH/CH<sub>2</sub>Cl<sub>2</sub>) 0.30; mp 153.8–171.2 °C (decomposed); IR (neat) 3289, 3101, 1680, 1581, 1266, 1226, 740 cm<sup>–1</sup>; <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>) δ 9.74 (s, 1H), 8.27 (dd, <i>J</i> = 8.6, 5.1 Hz, 1H), 7.87 (dd, <i>J</i> = 8.2, 2.4 Hz, 1H), 7.60 (s, 1H), 7.45 (td, <i>J</i> = 8.3, 2.4 Hz, 1H), 7.08 (s, 1H), 7.01 (s, 1H), 4.12 (t, <i>J</i> = 7.0 Hz, 2H), 3.47 (t, <i>J</i> = 6.0 Hz, 2H), 2.89 (s, 3H), 2.17 (m, 2H); <sup>13</sup>C NMR (150 MHz, CDCl<sub>3</sub>) δ 182.1, 171.7, 167.6, 167.1, 165.9, 161.4, 151.1, 137.3, 134.5, 134.4, 131.0, 131.0, 129.8, 129.2, 126.2, 121.2, 121.1, 118.8, 115.4, 114.2, 114.1, 44.5, 36.3, 30.9, 15.1; <sup>19</sup>F NMR (565 MHz, CDCl<sub>3</sub>) δ −100.63; HRMS (MALDI<sup>+</sup>, TOF) calcd for C<sub>20</sub>H<sub>16</sub>FN<sub>3</sub>O<sub>4</sub> [M + H]<sup>+</sup> 382.1203, found 382.1201. Sample for biology test was purified by preparative HPLC with HPLC purity of >99%.</div></div><div id="sec5_21" class="NLM_sec NLM_sec_level_2"><div id="ac_i39" class="anchor-spacer"></div><h3 class="article-section__title" id="_i39"> <i>N</i>-(3-Azidopropyl)-2-methyl-4,9-dioxo-4,9-dihydronaphtho[2,3-<i>b</i>]furan-3-carboxamide (<b>19</b>)</h3><div class="NLM_p last">To a stirred solution of <b>18</b> (30.7 mg, 0.12 mmol), HBTU (54.6 mg, 0.144 mmol), and DMAP (14.7 mg, 0.12 mmol) in DMF (0.6 mL) was added 3-azido-1-propylamine (30 μL, 0.3 mmol) at room temperature. The solvent was removed by vacuum after the reaction mixture was stirred at room temperature for 3 days. Then the crude was washed by H<sub>2</sub>O, extracted with CH<sub>2</sub>Cl<sub>2</sub> (10 mL × 3), dried over Na<sub>2</sub>SO<sub>4</sub>, and concentrated <i>in vacuo</i>. The residue was purified by flash column chromatography (MeOH/CH<sub>2</sub>Cl<sub>2</sub>, 0/100 to 1/49) to give <b>19</b> as a yellow solid (18 mg, 43% yield). <i>R<sub>f</sub></i> (CH<sub>2</sub>Cl<sub>2</sub>) 0.52; mp 105.5–109.3 °C; IR (neat) 3275, 3084, 2925, 2853, 2099, 1660, 1644, 1580, 1537, 1376, 1260, 1212, 1143, 991, 717 cm<sup>–1</sup>; <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 9.75 (t, <i>J</i> = 4.8 Hz, 1H), 8.24–8.18 (m, 2H), 7.82–7.75 (m, 2H), 3.55 (dd, <i>J</i> = 12.4, 6.6 Hz, 2H), 3.46 (t, <i>J</i> = 6.8 Hz, 2H), 2.87 (s, 3H), 1.96 (quint, <i>J</i> = 6.7 Hz, 2H); <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>) δ 183.1, 172.9, 166.5, 161.4, 151.0, 134.7, 134.2, 132.8, 131.5, 127.7, 126.8, 126.0, 115.2, 49.2, 36.6, 28.7, 14.9; HRMS (ESI<sup>+</sup>, TOF) calcd for C<sub>17</sub>H<sub>14</sub>N<sub>4</sub>Na [M + Na]<sup>+</sup> 361.0913, found 361.0917.</div></div><div id="sec5_22" class="NLM_sec NLM_sec_level_2"><div id="ac_i40" class="anchor-spacer"></div><h3 class="article-section__title" id="_i40"> Mag-Beads-Control</h3><div class="NLM_p">Carboxylate magnetic beads (10 mg) were dispersed into MES (50 mM, pH 6.0, 300 μL). <i>N</i>-Hydroxysuccinimide (NHS, 3.5 mg, 0.03 mmol) and 1-ethyl-3-(3-dimethylaminopropyl)carbodiimide hydrochloride (EDC-HCl, 5.7 mg, 0.03 mmol) were added to the solution (<a class="ref internalNav" href="#sch3" aria-label="Scheme 3">Scheme 3</a>) and stirred for 1 h at room temperature. The resulting beads were washed with PBS (50 mM, pH 7.0, 300 μL × 2) to remove excess NHS and EDC-HCl. 200 μL of 1 mM carboxylated-PEG<sub>12</sub>-amine (pH 7.8 in 50 mM HEPES) was added to the beads and then stirred for 6 h at room temperature. After separation with a magnet, the beads were washed with MES (50 mM, pH 6.0) to give carboxyl-PEG<sub>12</sub>-MBs.</div><figure id="sch3" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b01783/20200323/images/medium/jm9b01783_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b01783/20200323/images/large/jm9b01783_0009.jpeg" id="gr11" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Conjugation of AS4583 Analog onto Magnetic Beads</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b01783/20200323/images/large/jm9b01783_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01783&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec5_23" class="NLM_sec NLM_sec_level_2"><div id="ac_i42" class="anchor-spacer"></div><h3 class="article-section__title" id="_i42"> Alkynyl-PEG<sub>12</sub>-MBs</h3><div class="NLM_p last">Carboxyl-PEG<sub>12</sub>-MBs (10 mg) were dispersed into MES (50 mM, pH 6.0, 300 μL). <i>N</i>-Hydroxysuccinimide (NHS, 3.5 mg, 0.03 mmol) and 1-ethyl-3-(3-dimethylaminopropyl)carbodiimide hydrochloride (EDC-HCl, 5.7 mg, 0.03 mmol) were added to the solution and stirred for 3 h at room temperature. The resulting beads were washed with PBS (50 mM, pH 7.0, 300 μL × 2) to remove excess NHS and EDC-HCl. 200 μL of 1 mM propargylamine (pH 7.0 in 50 mM PBS) was added to the beads and then stirred for 6 h at room temperature. After separation with a magnet, the beads were washed with MES (50 mM, pH 6.0) to give alkynyl-PEG<sub>12</sub>-MBs.</div></div><div id="sec5_24" class="NLM_sec NLM_sec_level_2"><div id="ac_i43" class="anchor-spacer"></div><h3 class="article-section__title" id="_i43"> Mag-Beads-AS4583</h3><div class="NLM_p last">To a solution of alkynyl-PEG<sub>12</sub>-MBs (10 mg), azide <b>19</b> (1.3 mg, 4.0 μmol), CuSO<sub>4</sub> in 0.1 M H<sub>2</sub>O (5 μL, 0.5 μmol), and sodium ascorbate in 0.1 M H<sub>2</sub>O (2 μL, 0.2 μmol) was added <i>t</i>-butanol/H<sub>2</sub>O (0.2 mL, 1/1, v/v). Microwave conditions were the following: 100 W, 60 °C for 30 min. After separation with a magnet, the beads were washed with MES (50 mM, pH 6.0) to give Mag-beads-AS4583.</div></div><div id="sec5_25" class="NLM_sec NLM_sec_level_2"><div id="ac_i44" class="anchor-spacer"></div><h3 class="article-section__title" id="_i44"> High-Throughput Screening (HTS)</h3><div class="NLM_p last">HTS was performed against a collection of the two-million compound library. The synthetic molecules in the two-million compound library were clustered into approximately 5300 groups based on 85% structure similarity. The diversity was approximately 0.87, as calculated by a modified centroid-diversity sorting algorithm. Screens were performed in 1536-well plates using the HTS system manufactured by GNF systems (San Diego, CA). For primary screening, H1975 cells were dispensed into the wells of a 1536-well plate at 400 cells/well. The cells were incubated for 3 days in environmentally controlled incubators (37 °C, 5% CO<sub>2</sub>, 80% humidity), and cell viability was monitored using Cell-TiterGlo (Promega). The compounds resulting in a >80% decrease in luminescence were selected using a hit-picking system (GNF System, San Diego, CA) and collected in 1536-well plates. Serial dilutions of hit plates were performed with an automatic liquid handling system (Bravo, Agilent) and used for downstream screening with other lung cancer cells. IC<sub>50</sub> values were calculated using a proprietary software program (GNF Systems, San Diego).</div></div><div id="sec5_26" class="NLM_sec NLM_sec_level_2"><div id="ac_i45" class="anchor-spacer"></div><h3 class="article-section__title" id="_i45"> Cell Lines and Culture Conditions</h3><div class="NLM_p last">The human lung adenocarcinoma cell lines CL1-0 and CL1-5 were established in our laboratory as previously described.<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> The human lung cancer cell line NCI-H1975, A549 (a human lung adenocarcinoma epithelial cell line), and the BEAS-2B and Hs68 cell lines were purchased from the American Type Culture Collection (Manassas, VA). The cell line PC9 and derivative PC9/IR clones were kind gifts from Dr. C. H. Yang (Graduate Institute of Oncology, Cancer Research Center, and National Taiwan University). NBE cultured in BEBM (bronchial epithelial basal media) was kindly provided by Dr. Reen Wu (Department of Anatomy, Physiology and Cell Biology, University of California Davis). Cells were grown in RPMI or DMEM medium containing 10% FBS and 2 mM <span class="smallcaps smallerCapital">l</span>-glutamine (all from Invitrogen, Eugene, OR) at 37 °C in a humidified atmosphere of 5% CO<sub>2</sub>–95% air.</div></div><div id="sec5_27" class="NLM_sec NLM_sec_level_2"><div id="ac_i46" class="anchor-spacer"></div><h3 class="article-section__title" id="_i46"> Sulforhodamine B Colorimetric Assay for Cytotoxicity Screening</h3><div class="NLM_p last">Prior to the experiment, 2 × 10<sup>3</sup> cells were cultured in 96-well plates for 24 h. The culture medium was replaced with fresh medium containing the appropriate concentration of compound ranging from 0.005 μM to 10 μM for 72 h. After an incubation period, the cells were fixed with 10% trichloroacetic acid and stained with 0.4% (w/v) sulforhodamine B dissolved in 1% acetic acid for 30 min. Then, the excess dye was removed by washing the samples repeatedly with 1% acetic acid. The protein-bound dye was dissolved in 10 mM Tris-base solution for the determination of optical density at 510 nm using a microplate reader. The cell growth curve was plotted using GraphPad software (San Diego, CA).</div></div><div id="sec5_28" class="NLM_sec NLM_sec_level_2"><div id="ac_i47" class="anchor-spacer"></div><h3 class="article-section__title" id="_i47"> MTS Assay</h3><div class="NLM_p last">For the MTS assay, freshly prepared 20 μL of MTS stock solution mix with 100 μL of culture media was added to each well. After more than 1 h of incubation, absorption was measured at 490 nm by a spectrophotometer (Molecular Devices).</div></div><div id="sec5_29" class="NLM_sec NLM_sec_level_2"><div id="ac_i48" class="anchor-spacer"></div><h3 class="article-section__title" id="_i48"> Xenograft Tumor Growth <i>in Vivo</i></h3><div class="NLM_p last">The <i>in vivo</i> Xenograft tumor growth assay was performed as described previously.<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> Briefly, male BALB/c-nu mice (5 weeks of age) were obtained from National Laboratory Animal Center, Taiwan, and acclimatized to laboratory conditions for 1 week before tumor implantation. A total of 3 × 10<sup>6</sup> H1975 cells were subcutaneously injected into the flank of the BALB/c-nu mice, and the drug treatment was initiated at 4 days after the inoculation of cancer cells. The control group was 5 days a week treated intraperitoneally with DMSO in 50% PEG400, whereas the compound group was treated with 1 mg/kg or 4 mg/kg in 50% PEG400 for 4 weeks (<i>n</i> = 6 per group). During the treatment, tumor size and body weights were measured every 3–4 days, and the tumor size was calculated as <i>V</i> = <sup>1</sup>/<sub>2</sub> × (length) × (width)<sup>2</sup>. Tumor growth curves are presented as the mean ± standard error of the mean (SEM). The % tumor growth inhibition (TGI) was calculated as follows: [1 – ((<i>T</i><sub>final</sub> – <i>T</i><sub>initial</sub>)/ (<i>C</i><sub>final</sub> – <i>C</i><sub>initial</sub>))] × 100, where <i>T</i> = AS4583-treated tumor volumes and <i>C</i> = control-treated tumor volumes at treatment start and after 27 days’ treatment. All mouse experiments were performed in accordance with the animal guidelines and with the approval of the Laboratory Animal Center, National Taiwan University College of Medicine (IACUC No. 20150378).</div></div><div id="sec5_30" class="NLM_sec NLM_sec_level_2"><div id="ac_i49" class="anchor-spacer"></div><h3 class="article-section__title" id="_i49"> Cell-Cycle Analysis</h3><div class="NLM_p last">H1975 cells were seeded at a density of 5 × 10<sup>4</sup> in 60 mm culture dishes for 24 h before the experiment. The cells were treated with 0–140 nM of compound for 72 h in complete medium. The cells were harvested with 0.1% trypsin solution containing 0.05% EDTA in PBS, centrifuged, washed with PBS, and resuspended in cold 70% ethanol. The cells were then stained with propidium iodide (50 μg/mL; Sigma) for 30 min and subjected to flow cytometric analysis with a FACStar Plus (Becton Dickinson, San Francisco, CA).</div></div><div id="sec5_31" class="NLM_sec NLM_sec_level_2"><div id="ac_i50" class="anchor-spacer"></div><h3 class="article-section__title" id="_i50"> BrdU Incorporation</h3><div class="NLM_p last">For BrdU (5′-bromo-2′-deoxyuridine) labeling, the cells were precultured for 24 h, treated for 0, 3, 6, 12, 24, or 48 h, and pulse-labeled for 30 min with 10 μM BrdU (BioVision, Inc., Milpitas, CA). Cells were fixed and permeabilized with Cytofix/Cytoperm buffer (BD Pharmingen BrdU flow kit staining, BD Bioscience, San Jose, CA) and treated with DNase to expose incorporated BrdU. The BrdU content was assessed using fluorescent antibodies. DNA was stained with 7-aminoactinomycin D (7-AAD). The samples were analyzed in a FACS caliber flow cytometer (Becton Dickinson, Sweden) for dot plot histogram analysis (BrdU incorporation vs DNA content).</div></div><div id="sec5_32" class="NLM_sec NLM_sec_level_2"><div id="ac_i51" class="anchor-spacer"></div><h3 class="article-section__title" id="_i51"> Cell Synchronization and Immunofluorescence</h3><div class="NLM_p last">H1975 cells were arrested in G1 by contact inhibition. After 3 d, the cells were replated at a low density on coverslips and allowed to progress through the cell cycle. The cells were pulsed with 140 nM compound for 48 h and were fixed for 10 min at room temperature in 3.7% cold paraformaldehyde in PBS (pH 7.2), washed 3 times with PBS, and permeabilized for 10 min at room temperature in PBS containing 0.1% Triton X-100. The cells were blocked with PBS containing 3% bovine serum albumin and stained overnight at 4 °C with primary antibodies against MCM2, followed by incubation for 1 h at 37 °C with Alexa Fluor 488 goat anti-mouse IgG (Life Technologies Corporation). The cells were mounted onto microscope slides with ProLong Gold Antifade reagent with DAPI (Life Technologies Corporation) and then examined and photographed using a Zeiss LSM 700 confocal microscope (Zeiss, Urbana, IL).</div></div><div id="sec5_33" class="NLM_sec NLM_sec_level_2"><div id="ac_i52" class="anchor-spacer"></div><h3 class="article-section__title" id="_i52"> Lentiviral shRNA Induction</h3><div class="NLM_p last">The lentiviruses used in this study were generated by co-transfection of HEK293T cells with the lentiviral vector pLKO.1-shMCM2 from the National RNAi Core Facility (Academia Sinica, Taiwan) and two helper plasmids (pCMV△R8.91, pMD.G) using Lipofectamine 2000. The virus-containing medium was collected at 24, 48, and 72 h post-transfection, centrifuged, and filtered through a 0.45 mm pore-size filter. The cells were infected with the various individual lentiviruses in medium containing polybrene (8 μg/mL). At 24 h postinfection, the cells were incubated in fresh medium for 48 h and used for other experiments.</div></div><div id="sec5_34" class="NLM_sec NLM_sec_level_2"><div id="ac_i53" class="anchor-spacer"></div><h3 class="article-section__title" id="_i53"> Plasmids and Antibodies</h3><div class="NLM_p last">Briefly, the MCM2 full-length DNA fragments were amplified via RT-PCR from H1975 cDNA and subcloned into pJET1.2 (Thermo) to generate pJET-MCM2. The MCM2 DNA fragment was inserted between the NheI and EcoRI sites of p3xFlag to generate p3xFlag-MCM2. Expression vectors for myc-tagged ubiquitin have been described.<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> The primary antibodies specific for Myc, ubiquitin, cyclins E, A, B, and D1, and MCM2, -3, -4, -5, -6, and -7 were purchased from Santa Cruz Biotechnology (Santa Cruz, CA).</div></div><div id="sec5_35" class="NLM_sec NLM_sec_level_2"><div id="ac_i54" class="anchor-spacer"></div><h3 class="article-section__title" id="_i54"> Transfection</h3><div class="NLM_p last">Briefly, cells were seeded in 10 cm plates at a density of 3 × 10<sup>5</sup> cells/mL in RPMI medium with 10% fetal bovine serum for 24 h. The 70% confluent cells were transfected with plasmids (5 μg each) using 20 U Lipofectamine 2000 reagent (Invitrogen) in a total volume of 1 mL of Opti-MEM (all from Invitrogen). At 36 h post-transfection, the cells were incubated in the absence or presence of each compound (140 nM) and MG132 (10 μM; Sigma, St. Louis, MO). After 12 h, the cells were harvested for further analysis.</div></div><div id="sec5_36" class="NLM_sec NLM_sec_level_2"><div id="ac_i55" class="anchor-spacer"></div><h3 class="article-section__title" id="_i55"> Immunoprecipitation and Immunoblotting</h3><div class="NLM_p last">Briefly, H1975 cells were plated and cultured in complete RPMI medium and allowed to attach for 24 h, followed by the addition of 140 nM compound. The incubation was continued for 24, 48, or 72 h. Control cells were maintained in regular medium. The cells were lysed on ice for 30 min in ice-cold immunoprecipitation buffer containing 20 mM Tris (pH 8.0), 150 mM NaCl, 100 μM Na<sub>3</sub>VO<sub>4</sub>, 50 mM NaF, 30 mM sodium pyrophosphate, 0.5% (v/v) NP-40 (Sigma), and a 25-fold dilution of a stock solution treated with one mini-protease inhibitor cocktail tablet (Roche Diagnostics, Basel, Switzerland) dissolved in 2 mL of distilled water. The cell lysates were passed several times through a 21-gauge needle and clarified by centrifugation at 8000<i>g</i> for 30 min at 4 °C. The supernatants were collected, and the protein content of each lysate was determined with the BCA protein assay reagent (Pierce, Rockford, IL, USA) using bovine serum albumin as the standard. Aliquots of the lysates were incubated in immunoprecipitation buffer with respective antibodies overnight at 4 °C. Dynabeads Protein A/G (Invitrogen) was added and incubated for 1 h, then washed six times with PBS (phosphate-buffered saline; 0.01 M sodium phosphate, 0.14 M NaCl, pH 7.4; Sigma). For immunoblotting, the proteins were resolved using SDS–PAGE (8% or 10% acrylamide). The proteins were transferred to a polyvinylidene difluoride membrane and blocked in TBST buffer (10 mM Tris, pH 7.5, 100 mM NaCl, and 0.05% (v/v) Tween 20) containing 3% nonfat dried milk. The blots were incubated with various primary antibodies and incubated for 1 h with appropriate secondary antibodies conjugated to horseradish peroxidase in TBST. The intensity of each specific immunoreactive band was detected by enhanced chemiluminescence according to the manufacturer’s protocol (PerkinElmer, Inc., USA), quantified by densitometry, and normalized against the loading control (β-actin) with ImageJ software.</div></div><div id="sec5_37" class="NLM_sec NLM_sec_level_2"><div id="ac_i56" class="anchor-spacer"></div><h3 class="article-section__title" id="_i56"> Chromatin Loading Assay</h3><div class="NLM_p last">After washing with cold PBS, H1975 cells were harvested using 0.1% trypsin, and cell pellets were lysed by incubating the pellets in cytoskeleton buffer (20 mM HEPES, pH 7.4, 100 mM NaCl, 3 mM MgCl<sub>2</sub>, 300 mM sucrose, and 0.1% (v/v) NP-40) for 15 min on ice. Detergent-soluble fractions were obtained after low-speed centrifugation (3000<i>g</i>) at 4 °C. The pellets were rinsed with cytoskeleton buffer for 10 min on ice and recentrifuged to obtain a chromatin-enriched fraction. The pellets were then sonicated for 5 s in cytoskeleton buffer for 30 min and subjected to high-speed centrifugation (16 000<i>g</i>). The postsonication supernatant was designated as the chromatin-bound fraction.</div></div><div id="sec5_38" class="NLM_sec NLM_sec_level_2"><div id="ac_i57" class="anchor-spacer"></div><h3 class="article-section__title" id="_i57"> Site-Directed Mutagenesis</h3><div class="NLM_p last">Briefly, p3xFlag-MCM2 plasmids were used as templates for PCR-based site-directed mutagenesis with the QuikChange Lightning site-directed mutagenesis kit (Stratagene). The primers used to generate p3xFlag-MCM2-Q341A (mutated nucleotides are underlined) were 5′-CTG GGT CCT TTC TGC <u class="uu">GC</u>G TCC CAG AAC CAG G-3′ (forward) and 5′-CCT GGT TCT GGG AC<u class="uu">GC</u>GC AGA AAG GAC CCA G-3′ (reverse).</div></div><div id="sec5_39" class="NLM_sec NLM_sec_level_2"><div id="ac_i58" class="anchor-spacer"></div><h3 class="article-section__title" id="_i58"> Homology Modeling, MD, and Ligand Docking</h3><div class="NLM_p">We used the Phyre2 server<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> to search the initial template for modeling and to calculate the target–template sequence alignment. The structure of human MCM2 (residues 192–804) was modeled using MODELLER 9v12.<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> To improve the model quality, the model with the best discrete optimized protein energy (DOPE) score was subjected to molecular dynamics simulation using GROMACS version 4.6.2<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> with an OPLS-AA force field as a preparatory process for docking. The initial structure was immersed in an orthorhombic water box, and the net charge was neutralized by adding sodium or chloride ions (at 150 mM salt concentration). Long-range electrostatics were handled using the particle mesh Ewald method. The steepest descent energy minimization was used to remove possible bad contacts from the initial structures until energy convergence reached 1000 kJ/(mol·nm). The systems were subjected to equilibration at 300 K and a normal pressure constant (1 bar) for 100 ps under the conditions of position restraints for heavy atoms and LINCS constraints. The equilibrated structures were used to perform the production run. The time step of the simulation was set to 2 fs, and the coordinates were saved for analysis every 100 ps. ProSA<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> and PROCHECK<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> were used to validate the stereochemical qualities of the final model.</div><div class="NLM_p">The coordinates of compound AS4583 were obtained from the PubChem database<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> through the unique chemical structure identifier CID: 3585487. The potential AS4583 binding pocket was predicted using iGEMDOCK.<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> AutoDock Vina version 1.1.2<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> was used to construct the MCM2–ligand complex structure. We centered the grid box (30 Å × 30 Å × 30 Å) at the putative ligand-binding pocket. Because each docking run produces 20 binding modes, to increase the chance of finding the best binding mode, we repeated each docking run 10 times using an exhaustiveness value of 64 and ranked the resulting 200 poses based on their predicted binding affinity (in kcal/mol). The top ranked results were visualized and analyzed with PyMOL<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> and the AutoDock plugin.<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a></div><div class="NLM_p last">For the MCM2–7 hexamer structure, each protomer was roughly built using the Phyre2 server and then assembled into a hexamer by applying the 6-fold symmetry from the crystal structure of <i>Methanothermobacter</i> MCM (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1LTL">1LTL</a>).<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> The clashes between neighboring protomers were gradually removed using the loop optimization protocol in MODELLER<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a>.</div></div><div id="sec5_40" class="NLM_sec NLM_sec_level_2"><div id="ac_i59" class="anchor-spacer"></div><h3 class="article-section__title" id="_i59"> Statistical Analysis</h3><div class="NLM_p last">The data are shown as the mean ± SEM. Statistical analyses were performed by Student’s <i>t</i> test or Pearson’s <i>x</i><sup>2</sup> test. The overall survival rates for patient groups with different expression signatures were determined using SPSS software (v10.0; SPSS, Inc., Chicago, IL) to apply the Kaplan–Meier method and two-sided log-rank tests. <i>P</i> values of 0.05 were considered statistically significant.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i60"><a href="/doi/suppl/10.1021/acs.jmedchem.9b01783" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_02740" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_02740" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a href="/doi/10.1021/acs.jmedchem.9b01783?goto=supporting-info" class="ext-link">https://pubs.acs.org/doi/10.1021/acs.jmedchem.9b01783</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Modeling and simulation information, analysis of trajectories, and supplementary figures and tables (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01783/suppl_file/jm9b01783_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">NMR spectra (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01783/suppl_file/jm9b01783_si_002.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular formula strings and some data (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01783/suppl_file/jm9b01783_si_003.xlsx" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01783/suppl_file/jm9b01783_si_001.pdf">jm9b01783_si_001.pdf (995.44 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01783/suppl_file/jm9b01783_si_002.pdf">jm9b01783_si_002.pdf (2.98 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01783/suppl_file/jm9b01783_si_003.xlsx">jm9b01783_si_003.xlsx (12.77 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.9b01783" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_30321" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_30321" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Szu-Hua Pan</span> - <span class="hlFld-Affiliation affiliation">Genome
and Systems Biology Degree Program, National
Taiwan University and Academia Sinica, Taipei 100, Taiwan</span>; 
    <span class="hlFld-Affiliation affiliation">Graduate
Institute of Medical Genomics and Proteomics, College of Medicine, National Taiwan University, Taipei 100, Taiwan</span>; 
    <span class="hlFld-Affiliation affiliation">Doctoral
Degree Program of Translational Medicine, National Taiwan University, Taipei 100, Taiwan</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-0138-0434" title="Orcid link">http://orcid.org/0000-0002-0138-0434</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#27544f5746496749535209424352095350"><span class="__cf_email__" data-cfemail="7e0d160e1f103e100a0b501b1a0b500a09">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Rong-Jie Chein</span> - <span class="hlFld-Affiliation affiliation">Institute
of Chemistry, Academia Sinica, Taipei 115, Taiwan</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#8bf9e1e8e3eee2e5cbe8e3eee6a5f8e2e5e2e8eaa5eeeffea5fffc"><span class="__cf_email__" data-cfemail="7c0e161f141915123c1f141911520f1512151f1d5219180952080b">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Chia-Yi Lin</span> - <span class="hlFld-Affiliation affiliation">Genome
and Systems Biology Degree Program, National
Taiwan University and Academia Sinica, Taipei 100, Taiwan</span>; 
    <span class="hlFld-Affiliation affiliation">Institute
of Biomedical Sciences, Academia Sinica, Taipei 115, Taiwan</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Hsin-Yi Wu</span> - <span class="hlFld-Affiliation affiliation">Institute
of Chemistry, Academia Sinica, Taipei 115, Taiwan</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yuan-Ling Hsu</span> - <span class="hlFld-Affiliation affiliation">Graduate
Institute of Medical Genomics and Proteomics, College of Medicine, National Taiwan University, Taipei 100, Taiwan</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ting-Jen Rachel Cheng</span> - <span class="hlFld-Affiliation affiliation">The
Genomics Research Center, Academia Sinica, Taipei 115, Taiwan</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jyung-Hurng Liu</span> - <span class="hlFld-Affiliation affiliation">Institute
of Genomics and Bioinformatics, College of Life Sciences, National Chung Hsing University, Taichung 402, Taiwan</span>; 
    <span class="hlFld-Affiliation affiliation">Agricultural
Biotechnology Center, NCHU, Taichung 402, Taiwan</span>; 
    <span class="hlFld-Affiliation affiliation">Rong
Hsing Research Center for Translational Medicine, NCHU, Taichung 402, Taiwan</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Rou-Jie Huang</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry, National Central University, Jhong-Li 320, Taiwan</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Tzu-Hung Hsiao</span> - <span class="hlFld-Affiliation affiliation">Department
of Medical Research, Taichung Veterans General
Hospital, Taichung 407, Taiwan</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Chia-Jen Wang</span> - <span class="hlFld-Affiliation affiliation">Institute
of Stem Cell and Translational Cancer Research, Chang Gung Memorial Hospital, Taoyuan 333, Taiwan</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Pei-Fang Hung</span> - <span class="hlFld-Affiliation affiliation">Institute
of Biomedical Sciences, Academia Sinica, Taipei 115, Taiwan</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Albert Lan</span> - <span class="hlFld-Affiliation affiliation">Graduate
Institute of Medical Genomics and Proteomics, College of Medicine, National Taiwan University, Taipei 100, Taiwan</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Chi-Huey Wong</span> - <span class="hlFld-Affiliation affiliation">The
Genomics Research Center, Academia Sinica, Taipei 115, Taiwan</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-9961-7865" title="Orcid link">http://orcid.org/0000-0002-9961-7865</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Pan-Chyr Yang</span> - <span class="hlFld-Affiliation affiliation">Genome
and Systems Biology Degree Program, National
Taiwan University and Academia Sinica, Taipei 100, Taiwan</span>; 
    <span class="hlFld-Affiliation affiliation">Institute
of Biomedical Sciences, Academia Sinica, Taipei 115, Taiwan</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Internal Medicine, College of Medicine, National Taiwan University, Taipei 100, Taiwan</span></div></li></ul></li><li></li><li><span class="author-information-subsection-header">Funding</span><p>The authors thank National Taiwan University (Grants NTU-CDP-104R7879, 105R7879), Academia Sinica (Grant AS-SUMMIT-108), the Ministry of Science and Technology (Taiwan) (Grants NSC 102-2314-B-002-046-MY3, 102-2923-B-002-004, 103-2923-B-002-003, MOST 104-0210-01-09-02, 105-2628-B-002 -007-MY3, 105-0210-01-13-01, 106-0210-01-15-02, 107-0210-01-19-01, 108-3114-Y-001-002, 108-2314-B-002-191-MY3), and the National Research Program for Biopharmaceuticals (ChemBank and High-Throughput Screening Resource Center, Grants NSC-100-2325-B-001-022, 101-2325-B-001-029, and 102-2325-B-001-041) for supporting this study.</p></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i61">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_28780" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_28780" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The authors thank the following individuals for providing technical support: Dr. Wen-Long Wang (Department of Internal Medicine, National Cheng Kung University, Tainan, Taiwan), Dr. Yih-Leong Chang and Dr. Chen-Tu Wu (Department of Pathology and Graduate Institute of Pathology, College of Medicine, National Taiwan University), Dr. Lee-Jene Teng and Miss Yu-Tzu Lin (Department of Clinical Laboratory Sciences and Medical Biotechnology, College of Medicine, National Taiwan University).</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations Used</td></tr><tr><td class="NLM_term">NSCLC</td><td class="NLM_def"><p class="first last">non-small-cell lung cancer</p></td></tr><tr><td class="NLM_term">HTS</td><td class="NLM_def"><p class="first last">high-throughput screening</p></td></tr><tr><td class="NLM_term">MCM</td><td class="NLM_def"><p class="first last">minichromosomal maintenance protein</p></td></tr><tr><td class="NLM_term">TKI</td><td class="NLM_def"><p class="first last">tyrosine kinase inhibitor</p></td></tr><tr><td class="NLM_term">EGFR</td><td class="NLM_def"><p class="first last">epithelial growth factor receptor</p></td></tr><tr><td class="NLM_term">Cdc6</td><td class="NLM_def"><p class="first last">cell division cycle 6</p></td></tr><tr><td class="NLM_term">GINS</td><td class="NLM_def"><p class="first last">go, ichi, nii, and san (five, one, two, and three in Japanese)</p></td></tr><tr><td class="NLM_term">ORC</td><td class="NLM_def"><p class="first last">origin recognition complex</p></td></tr><tr><td class="NLM_term">SAR</td><td class="NLM_def"><p class="first last">structure–activity relationship</p></td></tr><tr><td class="NLM_term">CV</td><td class="NLM_def"><p class="first last">coefficient of variation</p></td></tr><tr><td class="NLM_term">WH</td><td class="NLM_def"><p class="first last">winged-helix</p></td></tr><tr><td class="NLM_term">MD</td><td class="NLM_def"><p class="first last">molecular dynamics</p></td></tr><tr><td class="NLM_term">Chk1</td><td class="NLM_def"><p class="first last">checkpoint kinase 1</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i63">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_70338" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_70338" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 39 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Siegel, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jemal, A.</span></span> <span> </span><span class="NLM_article-title">Cancer Statistics, 2019</span>. <i>Ca-Cancer J. Clin.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>69</i></span>,  <span class="NLM_fpage">7</span>– <span class="NLM_lpage">34</span>, <span class="refDoi"> DOI: 10.3322/caac.21551</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=10.3322%2Fcaac.21551" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=30620402" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=1%3ACAS%3A280%3ADC%252BB3cnptFCntA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=69&publication_year=2019&pages=7-34&author=R.+L.+Siegelauthor=K.+D.+Millerauthor=A.+Jemal&title=Cancer+Statistics%2C+2019&doi=10.3322%2Fcaac.21551"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Cancer statistics, 2019</span></div><div class="casAuthors">Siegel Rebecca L; Miller Kimberly D; Jemal Ahmedin</div><div class="citationInfo"><span class="NLM_cas:title">CA: a cancer journal for clinicians</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">69</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">7-34</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Each year, the American Cancer Society estimates the numbers of new cancer cases and deaths that will occur in the United States and compiles the most recent data on cancer incidence, mortality, and survival.  Incidence data, available through 2015, were collected by the Surveillance, Epidemiology, and End Results Program; the National Program of Cancer Registries; and the North American Association of Central Cancer Registries.  Mortality data, available through 2016, were collected by the National Center for Health Statistics.  In 2019, 1,762,450 new cancer cases and 606,880 cancer deaths are projected to occur in the United States.  Over the past decade of data, the cancer incidence rate (2006-2015) was stable in women and declined by approximately 2% per year in men, whereas the cancer death rate (2007-2016) declined annually by 1.4% and 1.8%, respectively.  The overall cancer death rate dropped continuously from 1991 to 2016 by a total of 27%, translating into approximately 2,629,200 fewer cancer deaths than would have been expected if death rates had remained at their peak.  Although the racial gap in cancer mortality is slowly narrowing, socioeconomic inequalities are widening, with the most notable gaps for the most preventable cancers.  For example, compared with the most affluent counties, mortality rates in the poorest counties were 2-fold higher for cervical cancer and 40% higher for male lung and liver cancers during 2012-2016.  Some states are home to both the wealthiest and the poorest counties, suggesting the opportunity for more equitable dissemination of effective cancer prevention, early detection, and treatment strategies.  A broader application of existing cancer control knowledge with an emphasis on disadvantaged groups would undoubtedly accelerate progress against cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRBHJFurk9wJJp6t8nS6Nc3fW6udTcc2eZ2nhmBRoxiy7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3cnptFCntA%253D%253D&md5=02ba708e57d7573ebe0649182325b52c</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.3322%2Fcaac.21551&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3322%252Fcaac.21551%26sid%3Dliteratum%253Aachs%26aulast%3DSiegel%26aufirst%3DR.%2BL.%26aulast%3DMiller%26aufirst%3DK.%2BD.%26aulast%3DJemal%26aufirst%3DA.%26atitle%3DCancer%2520Statistics%252C%25202019%26jtitle%3DCa-Cancer%2520J.%2520Clin.%26date%3D2019%26volume%3D69%26spage%3D7%26epage%3D34%26doi%3D10.3322%2Fcaac.21551" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kobayashi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Costa, D. B.</span></span> <span> </span><span class="NLM_article-title">Acquired Resistance to Epidermal Growth Factor Receptor Tyrosine Kinase Inhibitors in Non-Small-Cell Lung Cancers Dependent on the Epidermal Growth Factor Receptor Pathway</span>. <i>Clin. Lung Cancer</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">281</span>– <span class="NLM_lpage">289</span>, <span class="refDoi"> DOI: 10.3816/CLC.2009.n.039</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=10.3816%2FCLC.2009.n.039" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=19632948" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtVSms7bO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2009&pages=281-289&author=K.+S.+Nguyenauthor=S.+Kobayashiauthor=D.+B.+Costa&title=Acquired+Resistance+to+Epidermal+Growth+Factor+Receptor+Tyrosine+Kinase+Inhibitors+in+Non-Small-Cell+Lung+Cancers+Dependent+on+the+Epidermal+Growth+Factor+Receptor+Pathway&doi=10.3816%2FCLC.2009.n.039"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Acquired resistance to epidermal growth factor receptor tyrosine kinase inhibitors in non-small-cell lung cancers dependent on the epidermal growth factor receptor pathway</span></div><div class="casAuthors">Nguyen, Kim-Son H.; Kobayashi, Susumu; Costa, Daniel B.</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Lung Cancer</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">281-289</span>CODEN:
                <span class="NLM_cas:coden">CLCLCA</span>;
        ISSN:<span class="NLM_cas:issn">1525-7304</span>.
    
            (<span class="NLM_cas:orgname">CIG Media Group, LP</span>)
        </div><div class="casAbstract">A review.  Most advanced non-small-cell lung cancers (NSCLCs) with activating epidermal growth factor receptor (EGFR) mutations (exon 19 deletions or L858R) initially respond to the EGFR tyrosine kinase inhibitors (TKIs) gefitinib and erlotinib.  However, over time (median of 6-12 mo), most tumors develop acquired resistance to EGFR TKIs.  Intense research in these NSCLCs has identified two major mechanisms of resistance to gefitinib/erlotinib: secondary resistance mutations and "oncogene kinase switch" systems.  The secondary T790M mutation occurs in 50% of EGFR-mutated patients with TKI resistance, and in vitro, this mutation negates the hypersensitivity of activating EGFR mutations.  Sensitive detection methods have identified a proportion of TKI-naive tumors that carry T790M, and these resistant clones may be selected after exposure to gefitinib or erlotinib.  Other secondary resistance mutations (D761Y, L747S, T854A) seem to be rare.  The amplification of the MET oncogene is present in 20% of TKI-resistant tumors; however, in half of the cases with this "oncogene kinase switch" mechanism the T790M is coexistent.  It is possible that other kinases (such as insulin-like growth factor-1 receptor [IGF-1R]) might also be selected to bypass EGFR pathways in resistant tumors.  The growing preclin. data in EGFR-mutated NSCLCs with acquired resistance to gefitinib or erlotinib has spawned the initiation or conception of clin. trials testing novel EGFR inhibitors that in vitro inhibit T790M (neratinib, XL647, BIBW 2992, and PF-00299804), MET, or IGF-1R inhibitors in combination with EGFR TKIs, and heat shock protein 90 inhibitors.  Ongoing preclin. and clin. research in EGFR-mutated NSCLC has the potential to significantly improve the outcomes of patients with these somatic mutations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrzBC9bsn7YdrVg90H21EOLACvtfcHk0liKEhlRid6Qtw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtVSms7bO&md5=3522508ea4f2761e4dcedea18d914fe7</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.3816%2FCLC.2009.n.039&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3816%252FCLC.2009.n.039%26sid%3Dliteratum%253Aachs%26aulast%3DNguyen%26aufirst%3DK.%2BS.%26aulast%3DKobayashi%26aufirst%3DS.%26aulast%3DCosta%26aufirst%3DD.%2BB.%26atitle%3DAcquired%2520Resistance%2520to%2520Epidermal%2520Growth%2520Factor%2520Receptor%2520Tyrosine%2520Kinase%2520Inhibitors%2520in%2520Non-Small-Cell%2520Lung%2520Cancers%2520Dependent%2520on%2520the%2520Epidermal%2520Growth%2520Factor%2520Receptor%2520Pathway%26jtitle%3DClin.%2520Lung%2520Cancer%26date%3D2009%26volume%3D10%26spage%3D281%26epage%3D289%26doi%3D10.3816%2FCLC.2009.n.039" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neal, J. W.</span></span> <span> </span><span class="NLM_article-title">First-Line Treatment of Egfr-Mutant Non-Small-Cell Lung Cancer: The Role of Erlotinib and Other Tyrosine Kinase Inhibitors</span>. <i>Biologics</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">337</span>– <span class="NLM_lpage">345</span>, <span class="refDoi"> DOI: 10.2147/BTT.S26558</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=10.2147%2FBTT.S26558" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=23055691" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsFCkurnL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2012&pages=337-345&author=K.+S.+Nguyenauthor=J.+W.+Neal&title=First-Line+Treatment+of+Egfr-Mutant+Non-Small-Cell+Lung+Cancer%3A+The+Role+of+Erlotinib+and+Other+Tyrosine+Kinase+Inhibitors&doi=10.2147%2FBTT.S26558"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">First-line treatment of EGFR-mutant non-small-cell lung cancer: the role of erlotinib and other tyrosine kinase inhibitors</span></div><div class="casAuthors">Nguyen, Kim-Son H.; Neal, Joel W.</div><div class="citationInfo"><span class="NLM_cas:title">Biologics: Targets & Therapy</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">337-345</span>CODEN:
                <span class="NLM_cas:coden">BTTICT</span>;
        ISSN:<span class="NLM_cas:issn">1177-5491</span>.
    
            (<span class="NLM_cas:orgname">Dove Medical Press Ltd.</span>)
        </div><div class="casAbstract">A review.  Epidermal growth factor receptor tyrosine kinase inhibitors (EGFR TKIs) were initially established as second- or third-line treatment of advanced non-small-cell lung cancer (NSCLC).  Subsequent studies, including IPASS, OPTIMAL and EURTAC, have demonstrated that these TKIs are effective first-line therapeutic options in patients with tumors harboring activating mutations in the EGFR gene.  The TKIs are better tolerated than conventional chemotherapy, with frequent yet mild side effects such as rash and diarrhea and rarely interstitial lung disease.  Because most patients on TKIs develop resistance due to a variety of mechanisms, the use of TKIs in the acquired-resistance setting and in the setting of earlier-staged cancers is being extensively studied.  Here we review the major trials leading to the established use of EGFR TKIs in NSCLC, followed by discussion of recently completed and ongoing trials using the next-generation EGFR inhibitor afatinib.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr8uje4RXXGS7Vg90H21EOLACvtfcHk0liKEhlRid6Qtw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsFCkurnL&md5=94eecdd1672d7412f0b318440ae3bb38</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.2147%2FBTT.S26558&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2147%252FBTT.S26558%26sid%3Dliteratum%253Aachs%26aulast%3DNguyen%26aufirst%3DK.%2BS.%26aulast%3DNeal%26aufirst%3DJ.%2BW.%26atitle%3DFirst-Line%2520Treatment%2520of%2520Egfr-Mutant%2520Non-Small-Cell%2520Lung%2520Cancer%253A%2520The%2520Role%2520of%2520Erlotinib%2520and%2520Other%2520Tyrosine%2520Kinase%2520Inhibitors%26jtitle%3DBiologics%26date%3D2012%26volume%3D6%26spage%3D337%26epage%3D345%26doi%3D10.2147%2FBTT.S26558" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Swinney, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anthony, J.</span></span> <span> </span><span class="NLM_article-title">How Were New Medicines Discovered?</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">507</span>– <span class="NLM_lpage">519</span>, <span class="refDoi"> DOI: 10.1038/nrd3480</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=10.1038%2Fnrd3480" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=21701501" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=1%3ACAS%3A528%3ADC%252BC3MXotlCmsbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2011&pages=507-519&author=D.+C.+Swinneyauthor=J.+Anthony&title=How+Were+New+Medicines+Discovered%3F&doi=10.1038%2Fnrd3480"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">How were new medicines discovered?</span></div><div class="casAuthors">Swinney, David C.; Anthony, Jason</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">507-519</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Preclin. strategies that are used to identify potential drug candidates include target-based screening, phenotypic screening, modification of natural substances and biol.-based approaches.  To investigate whether some strategies have been more successful than others in the discovery of new drugs, we analyzed the discovery strategies and the mol. mechanism of action (MMOA) for new mol. entities and new biologics that were approved by the US Food and Drug Administration between 1999 and 2008.  Out of the 259 agents that were approved, 75 were first-in-class drugs with new MMOAs, and out of these, 50 (67%) were small mols. and 25 (33%) were biologics.  The results also show that the contribution of phenotypic screening to the discovery of first-in-class small-mol. drugs exceeded that of target-based approaches - with 28 and 17 of these drugs coming from the two approaches, resp. - in an era in which the major focus was on target-based approaches.  We postulate that a target-centric approach for first-in-class drugs, without consideration of an optimal MMOA, may contribute to the current high attrition rates and low productivity in pharmaceutical research and development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqE8Ah1iNz76bVg90H21EOLACvtfcHk0liKEhlRid6Qtw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXotlCmsbs%253D&md5=7503620f8fbadb0a829f6ea2029936a2</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1038%2Fnrd3480&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd3480%26sid%3Dliteratum%253Aachs%26aulast%3DSwinney%26aufirst%3DD.%2BC.%26aulast%3DAnthony%26aufirst%3DJ.%26atitle%3DHow%2520Were%2520New%2520Medicines%2520Discovered%253F%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2011%26volume%3D10%26spage%3D507%26epage%3D519%26doi%3D10.1038%2Fnrd3480" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Arteaga, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, D. H.</span></span> <span> </span><span class="NLM_article-title">Tyrosine Kinase Inhibitors-Zd1839 (Iressa)</span>. <i>Curr. Opin. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">491</span>– <span class="NLM_lpage">498</span>, <span class="refDoi"> DOI: 10.1097/00001622-200111000-00012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=10.1097%2F00001622-200111000-00012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=11673690" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=1%3ACAS%3A528%3ADC%252BD3MXos1Oqs7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2001&pages=491-498&author=C.+L.+Arteagaauthor=D.+H.+Johnson&title=Tyrosine+Kinase+Inhibitors-Zd1839+%28Iressa%29&doi=10.1097%2F00001622-200111000-00012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Tyrosine kinase inhibitors-ZD1839 (Iressa)</span></div><div class="casAuthors">Arteaga, Carlos L.; Johnson, David H.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Oncology</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">491-498</span>CODEN:
                <span class="NLM_cas:coden">CUOOE8</span>;
        ISSN:<span class="NLM_cas:issn">1040-8746</span>.
    
            (<span class="NLM_cas:orgname">Lippincott Williams & Wilkins</span>)
        </div><div class="casAbstract">Several epithelial tumors display epidermal growth factor receptor (EGFR) overexpression (with or without EGFR gene amplification) that is often assocd. with increased prodn. of EGFR ligands.  This permits the activation of endogenous tumor EGFR via autocrine mechanisms, resulting in cellular proliferation and tumor growth.  Interruption of receptor signaling with bivalent EGFR antibodies or with small mol. inhibitors of the EGFR tyrosine kinase results in inhibition of tumor cell proliferation or viability in vitro and in vivo.  One small mol. currently undergoing preclin. and clin. investigation is ZD1839 (Iressa), a synthetic anilinoquinazoline capable of inhibiting EGFR tyrosine kinase in vitro.  The early results of clin. trials indicate this drug possesses antitumor activity in certain malignancies of the upper aerodigestive tract.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpUp52ELUA8HLVg90H21EOLACvtfcHk0liKEhlRid6Qtw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXos1Oqs7k%253D&md5=ed22721cbb3560ac54e438d363f8a59e</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1097%2F00001622-200111000-00012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1097%252F00001622-200111000-00012%26sid%3Dliteratum%253Aachs%26aulast%3DArteaga%26aufirst%3DC.%2BL.%26aulast%3DJohnson%26aufirst%3DD.%2BH.%26atitle%3DTyrosine%2520Kinase%2520Inhibitors-Zd1839%2520%2528Iressa%2529%26jtitle%3DCurr.%2520Opin.%2520Oncol.%26date%3D2001%26volume%3D13%26spage%3D491%26epage%3D498%26doi%3D10.1097%2F00001622-200111000-00012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Moyer, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barbacci, E. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iwata, K. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arnold, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boman, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cunningham, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DiOrio, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doty, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morin, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moyer, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neveu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pollack, V. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pustilnik, L. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reynolds, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sloan, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Theleman, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, P.</span></span> <span> </span><span class="NLM_article-title">Induction of Apoptosis and Cell Cycle Arrest by Cp-358,774, an Inhibitor of Epidermal Growth Factor Receptor Tyrosine Kinase</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">4838</span>– <span class="NLM_lpage">4848</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=9354447" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=1%3ACAS%3A528%3ADyaK2sXnt12nsr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=1997&pages=4838-4848&author=J.+D.+Moyerauthor=E.+G.+Barbacciauthor=K.+K.+Iwataauthor=L.+Arnoldauthor=B.+Bomanauthor=A.+Cunninghamauthor=C.+DiOrioauthor=J.+Dotyauthor=M.+J.+Morinauthor=M.+P.+Moyerauthor=M.+Neveuauthor=V.+A.+Pollackauthor=L.+R.+Pustilnikauthor=M.+M.+Reynoldsauthor=D.+Sloanauthor=A.+Thelemanauthor=P.+Miller&title=Induction+of+Apoptosis+and+Cell+Cycle+Arrest+by+Cp-358%2C774%2C+an+Inhibitor+of+Epidermal+Growth+Factor+Receptor+Tyrosine+Kinase"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Induction of apoptosis and cell cycle arrest by CP-358774, an inhibitor of epidermal growth factor receptor tyrosine kinase</span></div><div class="casAuthors">Moyer, James D.; Barbacci, Elsa G.; Iwata, Kenneth K.; Arnold, Lee; Boman, Bruce; Cunningham, Ann; Diorio, Catherine; Doty, Jonathan; Morin, Michael J.; Moyer, Mikel P.; Neveu, Mark; Pollack, Vincent A.; Pustilnik, Leslie R.; Reynolds, Margaret M.; Sloan, Don; Theleman, April; Miller, Penny</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">4838-4848</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">The epidermal growth factor receptor (EGFR) is overexpressed in a significant percentage of carcinomas and contributes to the malignant phenotype.  CP-358774 is a directly acting inhibitor of human EGFR tyrosine kinase with an IC50 of 2 nM and reduces EGFR autophosphorylation in intact tumor cells with an IC50 of 20 nM.  This inhibition is selective for EGFR tyrosine kinase relative to other tyrosine kinases we have examd., both in assays of isolated kinases and whole cells.  At doses of 100 mg/kg, CP-358774 completely prevents EGF-induced autophosphorylation of EGFR in human HN5 tumors growing as xenografts in athymic mice and of the hepatic EGFR of the treated mice.  CP-358774 inhibits the proliferation of DiFi human colon tumor cells at submicromolar concns. in cell culture and blocks cell cycle progression at the G1 phase.  This inhibitor produces a marked accumulation of retinoblastoma protein in its underphosphorylated form and accumulation of p27KIP1 in DiFi cells, which may contribute to the cell cycle block.  Inhibition of the EGFR also triggers apoptosis in these cells as detd. by formation of DNA fragments and other criteria.  These results indicate that CP-358774 has potential for the treatment of tumors that are dependent on the EGFR pathway for proliferation or survival.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpghArQOnGlkbVg90H21EOLACvtfcHk0lj_I4gmcOALug"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXnt12nsr4%253D&md5=0a6e9697bf290d8e7c44cbe2a125e0b1</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMoyer%26aufirst%3DJ.%2BD.%26aulast%3DBarbacci%26aufirst%3DE.%2BG.%26aulast%3DIwata%26aufirst%3DK.%2BK.%26aulast%3DArnold%26aufirst%3DL.%26aulast%3DBoman%26aufirst%3DB.%26aulast%3DCunningham%26aufirst%3DA.%26aulast%3DDiOrio%26aufirst%3DC.%26aulast%3DDoty%26aufirst%3DJ.%26aulast%3DMorin%26aufirst%3DM.%2BJ.%26aulast%3DMoyer%26aufirst%3DM.%2BP.%26aulast%3DNeveu%26aufirst%3DM.%26aulast%3DPollack%26aufirst%3DV.%2BA.%26aulast%3DPustilnik%26aufirst%3DL.%2BR.%26aulast%3DReynolds%26aufirst%3DM.%2BM.%26aulast%3DSloan%26aufirst%3DD.%26aulast%3DTheleman%26aufirst%3DA.%26aulast%3DMiller%26aufirst%3DP.%26atitle%3DInduction%2520of%2520Apoptosis%2520and%2520Cell%2520Cycle%2520Arrest%2520by%2520Cp-358%252C774%252C%2520an%2520Inhibitor%2520of%2520Epidermal%2520Growth%2520Factor%2520Receptor%2520Tyrosine%2520Kinase%26jtitle%3DCancer%2520Res.%26date%3D1997%26volume%3D57%26spage%3D4838%26epage%3D4848" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cui, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tran-Dube, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nambu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kung, P. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pairish, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jia, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Funk, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Botrous, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McTigue, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grodsky, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ryan, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Padrique, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alton, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Timofeevski, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamazaki, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zou, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christensen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mroczkowski, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bender, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kania, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edwards, M. P.</span></span> <span> </span><span class="NLM_article-title">Structure Based Drug Design of Crizotinib (Pf-02341066), a Potent and Selective Dual Inhibitor of Mesenchymal-Epithelial Transition Factor (C-Met) Kinase and Anaplastic Lymphoma Kinase (Alk)</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">6342</span>– <span class="NLM_lpage">6363</span>, <span class="refDoi"> DOI: 10.1021/jm2007613</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm2007613" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtVegtLrN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=6342-6363&author=J.+J.+Cuiauthor=M.+Tran-Dubeauthor=H.+Shenauthor=M.+Nambuauthor=P.+P.+Kungauthor=M.+Pairishauthor=L.+Jiaauthor=J.+Mengauthor=L.+Funkauthor=I.+Botrousauthor=M.+McTigueauthor=N.+Grodskyauthor=K.+Ryanauthor=E.+Padriqueauthor=G.+Altonauthor=S.+Timofeevskiauthor=S.+Yamazakiauthor=Q.+Liauthor=H.+Zouauthor=J.+Christensenauthor=B.+Mroczkowskiauthor=S.+Benderauthor=R.+S.+Kaniaauthor=M.+P.+Edwards&title=Structure+Based+Drug+Design+of+Crizotinib+%28Pf-02341066%29%2C+a+Potent+and+Selective+Dual+Inhibitor+of+Mesenchymal-Epithelial+Transition+Factor+%28C-Met%29+Kinase+and+Anaplastic+Lymphoma+Kinase+%28Alk%29&doi=10.1021%2Fjm2007613"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Structure Based Drug Design of Crizotinib (PF-02341066), a Potent and Selective Dual Inhibitor of Mesenchymal-Epithelial Transition Factor (c-MET) Kinase and Anaplastic Lymphoma Kinase (ALK)</span></div><div class="casAuthors">Cui, J. Jean; Tran-Dube, Michelle; Shen, Hong; Nambu, Mitchell; Kung, Pei-Pei; Pairish, Mason; Jia, Lei; Meng, Jerry; Funk, Lee; Botrous, Iriny; McTigue, Michele; Grodsky, Neil; Ryan, Kevin; Padrique, Ellen; Alton, Gordon; Timofeevski, Sergei; Yamazaki, Shinji; Li, Qiuhua; Zou, Helen; Christensen, James; Mroczkowski, Barbara; Bender, Steve; Kania, Robert S.; Edwards, Martin P.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">6342-6363</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Because of the crit. roles of aberrant signaling in cancer, both c-MET and ALK receptor tyrosine kinases are attractive oncol. targets for therapeutic intervention.  The cocrystal structure of 3 (PHA-665752), bound to c-MET kinase domain, revealed a novel ATP site environment, which served as the target to guide parallel, multiattribute drug design.  A novel 2-amino-5-aryl-3-benzyloxypyridine series was created to more effectively make the key interactions achieved with 3.  In the novel series, the 2-aminopyridine core allowed a 3-benzyloxy group to reach into the same pocket as the 2,6-dichlorophenyl group of 3 via a more direct vector and thus with a better ligand efficiency (LE).  Further optimization of the lead series generated the clin. candidate crizotinib (PF-02341066), which demonstrated potent in vitro and in vivo c-MET kinase and ALK inhibition, effective tumor growth inhibition, and good pharmaceutical properties.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoHs_cKJ-U3t7Vg90H21EOLACvtfcHk0lj_I4gmcOALug"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtVegtLrN&md5=dbab4c1b3e7c76180dbad04be2b6a474</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1021%2Fjm2007613&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm2007613%26sid%3Dliteratum%253Aachs%26aulast%3DCui%26aufirst%3DJ.%2BJ.%26aulast%3DTran-Dube%26aufirst%3DM.%26aulast%3DShen%26aufirst%3DH.%26aulast%3DNambu%26aufirst%3DM.%26aulast%3DKung%26aufirst%3DP.%2BP.%26aulast%3DPairish%26aufirst%3DM.%26aulast%3DJia%26aufirst%3DL.%26aulast%3DMeng%26aufirst%3DJ.%26aulast%3DFunk%26aufirst%3DL.%26aulast%3DBotrous%26aufirst%3DI.%26aulast%3DMcTigue%26aufirst%3DM.%26aulast%3DGrodsky%26aufirst%3DN.%26aulast%3DRyan%26aufirst%3DK.%26aulast%3DPadrique%26aufirst%3DE.%26aulast%3DAlton%26aufirst%3DG.%26aulast%3DTimofeevski%26aufirst%3DS.%26aulast%3DYamazaki%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DQ.%26aulast%3DZou%26aufirst%3DH.%26aulast%3DChristensen%26aufirst%3DJ.%26aulast%3DMroczkowski%26aufirst%3DB.%26aulast%3DBender%26aufirst%3DS.%26aulast%3DKania%26aufirst%3DR.%2BS.%26aulast%3DEdwards%26aufirst%3DM.%2BP.%26atitle%3DStructure%2520Based%2520Drug%2520Design%2520of%2520Crizotinib%2520%2528Pf-02341066%2529%252C%2520a%2520Potent%2520and%2520Selective%2520Dual%2520Inhibitor%2520of%2520Mesenchymal-Epithelial%2520Transition%2520Factor%2520%2528C-Met%2529%2520Kinase%2520and%2520Anaplastic%2520Lymphoma%2520Kinase%2520%2528Alk%2529%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D6342%26epage%3D6363%26doi%3D10.1021%2Fjm2007613" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Buchdunger, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zimmermann, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mett, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meyer, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muller, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Druker, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lydon, N. B.</span></span> <span> </span><span class="NLM_article-title">Inhibition of the Abl Protein-Tyrosine Kinase in Vitro and in Vivo by a 2-Phenylaminopyrimidine Derivative</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">100</span>– <span class="NLM_lpage">104</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=8548747" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=1%3ACAS%3A528%3ADyaK28Xit1Chsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=1996&pages=100-104&author=E.+Buchdungerauthor=J.+Zimmermannauthor=H.+Mettauthor=T.+Meyerauthor=M.+Mullerauthor=B.+J.+Drukerauthor=N.+B.+Lydon&title=Inhibition+of+the+Abl+Protein-Tyrosine+Kinase+in+Vitro+and+in+Vivo+by+a+2-Phenylaminopyrimidine+Derivative"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of the Abl protein-tyrosine kinase in vitro and in vivo by a 2-phenylaminopyrimidine derivative</span></div><div class="casAuthors">Buchdunger, Elisabeth; Zimmermann, Juerg; Mett, Helmut; Meyer, Thomas; Mueller, Marcel; Druker, Brian J.; Lydon, Nicholas B.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">100-4</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Oncogenic activation of Abl proteins due to structural modifications can occur as a result of viral transduction or chromosomal translocation.  The tyrosine protein kinase activity of oncogenic Abl proteins is known to be essential for their transforming activity.  Therefore, we have attempted to identify selective inhibitors of the Abl tyrosine protein kinase.  Herein we describe an inhibitor (CGP 57148) of the Abl and platelet-derived growth factor (PDGF) receptor protein-tyrosine kinases from the 2-phenylaminopyrimidine class, which is highly active in vitro and in vivo.  Submicromolar concns. of the compd. inhibited both v-Abl and PDGF receptor autophosphorylation and PDGF-induced c-fos mRNA expression selectively in intact cells.  In contrast, ligand-induced growth factor receptor autophosphorylation in response to epidermal growth factor (EGF), insulin-like growth factor-I, and insulin showed no or weak inhibition by high concns. of CGP 57148.  C-fos mRNA expression induced by EGF, fibroblast growth factor, or phorbol ester was also insensitive to inhibition by CGP 57148.  In antiproliferative assays, the compd. was more than 30-100-fold more potent in inhibiting growth of v-abl-transformed PB-3c cells and v-sis-transformed BALB/c 3T3 cells relative to inhibition of EGF-dependent BaLB/MK cells, interleukin-3-dependent FDC-P1 cells, and the T24 bladder carcinoma line.  Furthermore, anchorage-independent growth of v-abl- and v-sis-transformed BALB/c 3T3 cells was inhibited potently by CGP 57148.  When tested in vivo, CGP 57148 showed antitumor activity at tolerated doses against tumorigenic v-Abl- and v-sis-transformed BALB/c 3T3 cells.  In contrast, CGP 57148 had no antitumor activity when tested using src-transformed BALB/c 3T3 cells.  These findings suggest that CGP 57148 may have therapeutic potential for the treatment of diseases that involve abnormal cellular proliferation induced by Abl protein-tyrosine kinase deregulation or PDGF receptor activation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpwOOoQlhPp6LVg90H21EOLACvtfcHk0lisYP1W77dw9A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28Xit1Chsg%253D%253D&md5=b311c951f52565f11d39901fe5a8cce6</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBuchdunger%26aufirst%3DE.%26aulast%3DZimmermann%26aufirst%3DJ.%26aulast%3DMett%26aufirst%3DH.%26aulast%3DMeyer%26aufirst%3DT.%26aulast%3DMuller%26aufirst%3DM.%26aulast%3DDruker%26aufirst%3DB.%2BJ.%26aulast%3DLydon%26aufirst%3DN.%2BB.%26atitle%3DInhibition%2520of%2520the%2520Abl%2520Protein-Tyrosine%2520Kinase%2520in%2520Vitro%2520and%2520in%2520Vivo%2520by%2520a%25202-Phenylaminopyrimidine%2520Derivative%26jtitle%3DCancer%2520Res.%26date%3D1996%26volume%3D56%26spage%3D100%26epage%3D104" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hartford, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ratain, M. J.</span></span> <span> </span><span class="NLM_article-title">Rapamycin: Something Old, Something New, Sometimes Borrowed and Now Renewed</span>. <i>Clin. Pharmacol. Ther.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>82</i></span>,  <span class="NLM_fpage">381</span>– <span class="NLM_lpage">388</span>, <span class="refDoi"> DOI: 10.1038/sj.clpt.6100317</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=10.1038%2Fsj.clpt.6100317" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=17728765" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtFGqu7fE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=82&publication_year=2007&pages=381-388&author=C.+M.+Hartfordauthor=M.+J.+Ratain&title=Rapamycin%3A+Something+Old%2C+Something+New%2C+Sometimes+Borrowed+and+Now+Renewed&doi=10.1038%2Fsj.clpt.6100317"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Rapamycin: something old, something new, sometimes borrowed and now renewed</span></div><div class="casAuthors">Hartford, C. M.; Ratain, M. J.</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Pharmacology & Therapeutics (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">82</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">381-388</span>CODEN:
                <span class="NLM_cas:coden">CLPTAT</span>;
        ISSN:<span class="NLM_cas:issn">0009-9236</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The mol. target of rapamycin (mTOR) is central to a complex intracellular signaling pathway and is involved in diverse processes including cell growth and proliferation, angiogenesis, autophagy, and metab.  Although sirolimus (rapamycin), the oldest inhibitor of mTOR, was discovered more than 30 years ago, renewed interest in this pathway is evident by the numerous rapalogs recently developed.  These newer agents borrow from the structure of sirolimus and, although there are some pharmacokinetic differences, they appear to differ little in terms of pharmacodynamic effects and overall tolerability.  Given the multitude of potential applications for this class of agents and the decrease in cost that can be expected upon the expiration of sirolimus patents, renewed focus on this agent is warranted.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqbjZFHK2UEtLVg90H21EOLACvtfcHk0lisYP1W77dw9A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtFGqu7fE&md5=53c9582651f57cec29399b4fd5161bdc</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1038%2Fsj.clpt.6100317&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.clpt.6100317%26sid%3Dliteratum%253Aachs%26aulast%3DHartford%26aufirst%3DC.%2BM.%26aulast%3DRatain%26aufirst%3DM.%2BJ.%26atitle%3DRapamycin%253A%2520Something%2520Old%252C%2520Something%2520New%252C%2520Sometimes%2520Borrowed%2520and%2520Now%2520Renewed%26jtitle%3DClin.%2520Pharmacol.%2520Ther.%26date%3D2007%26volume%3D82%26spage%3D381%26epage%3D388%26doi%3D10.1038%2Fsj.clpt.6100317" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Medina, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amaria, M. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jimeno, A.</span></span> <span> </span><span class="NLM_article-title">Dabrafenib in the Treatment of Advanced Melanoma</span>. <i>Drugs Today</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>49</i></span>,  <span class="NLM_fpage">377</span>– <span class="NLM_lpage">385</span>, <span class="refDoi"> DOI: 10.1358/dot.2013.49.6.1968669</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=10.1358%2Fdot.2013.49.6.1968669" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=1%3ACAS%3A280%3ADC%252BC3sjmtlOnsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2013&pages=377-385&author=T.+Medinaauthor=M.+N.+Amariaauthor=A.+Jimeno&title=Dabrafenib+in+the+Treatment+of+Advanced+Melanoma&doi=10.1358%2Fdot.2013.49.6.1968669"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Dabrafenib in the treatment of advanced melanoma</span></div><div class="casAuthors">Medina T; Amaria M N; Jimeno A</div><div class="citationInfo"><span class="NLM_cas:title">Drugs of today (Barcelona, Spain : 1998)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">377-85</span>
        ISSN:<span class="NLM_cas:issn">1699-3993</span>.
    </div><div class="casAbstract">Advanced melanoma has long been a challenging malignancy to treat due to a relative paucity of efficacious therapeutic options.  However, the identification of activating BRAF mutations in approximately 50% of patients with cutaneous melanoma has ushered in the era of targeted therapy for melanoma patients.  Similar to the first-in-class selective serine/threonine-protein kinase B-raf inhibitor vemurafenib, dabrafenib is highly efficacious in melanoma patients with BRAF V600E mutations, with response rates of approximately 50% and progression-free survival of 6 months.  There is data to suggest that dabrafenib not only shows activity in V600E-mutated melanoma, but also in non-V600E BRAF-mutated disease such as V600K.  There is also early data to suggest that dabrafenib is effective in controlling metastases in the brain.  Combining dabrafenib with the selective mitogen-activated protein kinase kinase (MEK) inhibitor trametinib has been effective in improving both the progression-free survival and overall survival of melanoma patients over those patients treated with dabrafenib alone.  Dabrafenib is still being evaluated in several clinical trials in melanoma as well as a variety of other solid tumors with BRAF mutations.  The U.S.  Food and Drug Administration has recently approved dabrafenib as a single agent for the treatment of unresectable or metastatic melanoma in adult patients with BRAF V600E mutation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRdCKpggTCUg-kdrW1FkXG-fW6udTcc2ebvEez9qE7GErntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3sjmtlOnsg%253D%253D&md5=3df12b8590957ab17336aec2086a37fb</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1358%2Fdot.2013.49.6.1968669&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1358%252Fdot.2013.49.6.1968669%26sid%3Dliteratum%253Aachs%26aulast%3DMedina%26aufirst%3DT.%26aulast%3DAmaria%26aufirst%3DM.%2BN.%26aulast%3DJimeno%26aufirst%3DA.%26atitle%3DDabrafenib%2520in%2520the%2520Treatment%2520of%2520Advanced%2520Melanoma%26jtitle%3DDrugs%2520Today%26date%3D2013%26volume%3D49%26spage%3D377%26epage%3D385%26doi%3D10.1358%2Fdot.2013.49.6.1968669" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Moffat, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rudolph, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bailey, D.</span></span> <span> </span><span class="NLM_article-title">Phenotypic Screening in Cancer Drug Discovery - Past, Present and Future</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">588</span>– <span class="NLM_lpage">602</span>, <span class="refDoi"> DOI: 10.1038/nrd4366</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=10.1038%2Fnrd4366" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=25033736" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtFygsr7L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2014&pages=588-602&author=J.+G.+Moffatauthor=J.+Rudolphauthor=D.+Bailey&title=Phenotypic+Screening+in+Cancer+Drug+Discovery+-+Past%2C+Present+and+Future&doi=10.1038%2Fnrd4366"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Phenotypic screening in cancer drug discovery - past, present and future</span></div><div class="casAuthors">Moffat, John G.; Rudolph, Joachim; Bailey, David</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">588-602</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  There has been a resurgence of interest in the use of phenotypic screens in drug discovery as an alternative to target-focused approaches.  Given that oncol. is currently the most active therapeutic area, and also one in which target-focused approaches have been particularly prominent in the past two decades, we investigated the contribution of phenotypic assays to oncol. drug discovery by analyzing the origins of all new small-mol. cancer drugs approved by the US Food and Drug Administration (FDA) over the past 15 years and those currently in clin. development.  Although the majority of these drugs originated from target-based discovery, we identified a significant no. whose discovery depended on phenotypic screening approaches.  We postulate that the contribution of phenotypic screening to cancer drug discovery has been hampered by a reliance on 'classical' nonspecific drug effects such as cytotoxicity and mitotic arrest, exacerbated by a paucity of mechanistically defined cellular models for therapeutically translatable cancer phenotypes.  However, tech. and biol. advances that enable such mechanistically informed phenotypic models have the potential to empower phenotypic drug discovery in oncol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoTWtoTIgWqvbVg90H21EOLACvtfcHk0lhnGrL_Y83dkA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtFygsr7L&md5=0e8efed90e5bc4b47f959b83a6ab5b96</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1038%2Fnrd4366&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd4366%26sid%3Dliteratum%253Aachs%26aulast%3DMoffat%26aufirst%3DJ.%2BG.%26aulast%3DRudolph%26aufirst%3DJ.%26aulast%3DBailey%26aufirst%3DD.%26atitle%3DPhenotypic%2520Screening%2520in%2520Cancer%2520Drug%2520Discovery%2520-%2520Past%252C%2520Present%2520and%2520Future%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2014%26volume%3D13%26spage%3D588%26epage%3D602%26doi%3D10.1038%2Fnrd4366" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Marks, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Breslow, R.</span></span> <span> </span><span class="NLM_article-title">Dimethyl Sulfoxide to Vorinostat: Development of This Histone Deacetylase Inhibitor as an Anticancer Drug</span>. <i>Nat. Biotechnol.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">84</span>– <span class="NLM_lpage">90</span>, <span class="refDoi"> DOI: 10.1038/nbt1272</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=10.1038%2Fnbt1272" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=17211407" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=1%3ACAS%3A528%3ADC%252BD2sXis1GrsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2007&pages=84-90&author=P.+A.+Marksauthor=R.+Breslow&title=Dimethyl+Sulfoxide+to+Vorinostat%3A+Development+of+This+Histone+Deacetylase+Inhibitor+as+an+Anticancer+Drug&doi=10.1038%2Fnbt1272"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Dimethyl sulfoxide to vorinostat: development of this histone deacetylase inhibitor as an anticancer drug</span></div><div class="casAuthors">Marks, Paul A.; Breslow, Ronald</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">84-90</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  In our quest to understand why DMSO (DMSO) can cause growth arrest and terminal differentiation of transformed cells, we followed a path that led us to discover suberoylanilide hydroxamic acid (SAHA; vorinostat (Zolinza)), which is a histone deacetylase inhibitor.  SAHA reacts with and blocks the catalytic site of these enzymes.  Extensive structure-activity studies were done along the path from DMSO to SAHA.  SAHA can cause growth arrest and death of a broad variety of transformed cells both in vitro and in tumor-bearing animals at concns. not toxic to normal cells.  SAHA has many protein targets whose structure and function are altered by acetylation, including chromatin-assocd. histones, nonhistone gene transcription factors and proteins involved in regulation of cell proliferation, migration and death.  In clin. trials, SAHA has shown significant anticancer activity against both hematol. and solid tumors at doses well tolerated by patients.  A new drug application was approved by the US Food and Drug Administration for vorinostat for treatment of cutaneous T-cell lymphoma.  More potent analogs of SAHA have shown unacceptable toxicity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoK3endE54rQ7Vg90H21EOLACvtfcHk0lhnGrL_Y83dkA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXis1GrsQ%253D%253D&md5=cc2a39d80d5d5bea0a1377bdf48a957d</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1038%2Fnbt1272&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt1272%26sid%3Dliteratum%253Aachs%26aulast%3DMarks%26aufirst%3DP.%2BA.%26aulast%3DBreslow%26aufirst%3DR.%26atitle%3DDimethyl%2520Sulfoxide%2520to%2520Vorinostat%253A%2520Development%2520of%2520This%2520Histone%2520Deacetylase%2520Inhibitor%2520as%2520an%2520Anticancer%2520Drug%26jtitle%3DNat.%2520Biotechnol.%26date%3D2007%26volume%3D25%26spage%3D84%26epage%3D90%26doi%3D10.1038%2Fnbt1272" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nakajima, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, Y. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terano, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoshida, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Horinouchi, S.</span></span> <span> </span><span class="NLM_article-title">Fr901228, a Potent Antitumor Antibiotic, Is a Novel Histone Deacetylase Inhibitor</span>. <i>Exp. Cell Res.</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>241</i></span>,  <span class="NLM_fpage">126</span>– <span class="NLM_lpage">133</span>, <span class="refDoi"> DOI: 10.1006/excr.1998.4027</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=10.1006%2Fexcr.1998.4027" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=9633520" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=1%3ACAS%3A528%3ADyaK1cXjvVGjtLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=241&publication_year=1998&pages=126-133&author=H.+Nakajimaauthor=Y.+B.+Kimauthor=H.+Teranoauthor=M.+Yoshidaauthor=S.+Horinouchi&title=Fr901228%2C+a+Potent+Antitumor+Antibiotic%2C+Is+a+Novel+Histone+Deacetylase+Inhibitor&doi=10.1006%2Fexcr.1998.4027"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">FR901228, a potent antitumor antibiotic, is a novel histone deacetylase inhibitor</span></div><div class="casAuthors">Nakajima, Hidenori; Kim, Young Bae; Terano, Hiroshi; Yoshida, Minoru; Horinouchi, Sueharu</div><div class="citationInfo"><span class="NLM_cas:title">Experimental Cell Research</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">241</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">126-133</span>CODEN:
                <span class="NLM_cas:coden">ECREAL</span>;
        ISSN:<span class="NLM_cas:issn">0014-4827</span>.
    
            (<span class="NLM_cas:orgname">Academic Press</span>)
        </div><div class="casAbstract">Screening for microbial metabolites that induce transcriptional activation of the SV40 promoter resulted in the identification of two known compds., FR901228 and trichostatin A (TSA).  FR901228 is a potent antitumor drug that is currently under clin. investigation.  TSA is a specific inhibitor of histone deacetylase.  Despite structural differences, both FR901228 and TSA greatly enhanced the transcriptional activity of the SV40 promoter in an enhancer-dependent manner.  The effects of FR901228 on the cell cycle, chromatin structure, and histone acetylation were examd. and compared to the effects of TSA.  Both compds. arrested the cell cycle at both G1 and G2/M phases and induced internucleosomal breakdown of chromatin.  FR901228, like TSA, inhibited the intracellular histone deacetylase activity, as a result of which marked amts. of acetylated histone species accumulated.  Thus, FR901228 is a new type of histone deacetylase inhibitor whose chem. structure is unrelated to known inhibitors such as trichostatins and trapoxins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqRDyUNz818nbVg90H21EOLACvtfcHk0lhnGrL_Y83dkA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXjvVGjtLY%253D&md5=f0e72c561596900036c9062bf2db2990</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1006%2Fexcr.1998.4027&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Fexcr.1998.4027%26sid%3Dliteratum%253Aachs%26aulast%3DNakajima%26aufirst%3DH.%26aulast%3DKim%26aufirst%3DY.%2BB.%26aulast%3DTerano%26aufirst%3DH.%26aulast%3DYoshida%26aufirst%3DM.%26aulast%3DHorinouchi%26aufirst%3DS.%26atitle%3DFr901228%252C%2520a%2520Potent%2520Antitumor%2520Antibiotic%252C%2520Is%2520a%2520Novel%2520Histone%2520Deacetylase%2520Inhibitor%26jtitle%3DExp.%2520Cell%2520Res.%26date%3D1998%26volume%3D241%26spage%3D126%26epage%3D133%26doi%3D10.1006%2Fexcr.1998.4027" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">O’Dwyer, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maslak, P.</span></span> <span> </span><span class="NLM_article-title">Azacitidine and the Beginnings of Therapeutic Epigenetic Modulation</span>. <i>Expert Opin. Pharmacother.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">1981</span>– <span class="NLM_lpage">1986</span>, <span class="refDoi"> DOI: 10.1517/14656566.9.11.1981</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=10.1517%2F14656566.9.11.1981" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=18627335" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=1%3ACAS%3A528%3ADC%252BD1cXos1Chu7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2008&pages=1981-1986&author=K.+O%E2%80%99Dwyerauthor=P.+Maslak&title=Azacitidine+and+the+Beginnings+of+Therapeutic+Epigenetic+Modulation&doi=10.1517%2F14656566.9.11.1981"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Azacitidine and the beginnings of therapeutic epigenetic modulation</span></div><div class="casAuthors">O'Dwyer, Kristen; Maslak, Peter</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Pharmacotherapy</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1981-1986</span>CODEN:
                <span class="NLM_cas:coden">EOPHF7</span>;
        ISSN:<span class="NLM_cas:issn">1465-6566</span>.
    
            (<span class="NLM_cas:orgname">Informa Healthcare</span>)
        </div><div class="casAbstract">A review.  Although originally developed as a cytarabine analog more than 40 years ago, azacitidine has been the subject of renewed interest in the era of cancer epigenetics.  What is the history of the clin. development of azacitidine and how has it been applied successfully to the treatment of myelodysplastic syndromes (MDS)  We review the evolution of the use of azacitidine for the therapy of human disease and review the major studies that have laid the groundwork for its current clin. indication.  The use of azacitidine has changed the approach to the treatment of MDS and has resulted in improved outcomes for patients.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGplzDnVBQUa-LVg90H21EOLACvtfcHk0ljMfkWvsk0uag"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXos1Chu7w%253D&md5=f421e7829f1b34e7086b0c11319d98a3</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1517%2F14656566.9.11.1981&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1517%252F14656566.9.11.1981%26sid%3Dliteratum%253Aachs%26aulast%3DO%25E2%2580%2599Dwyer%26aufirst%3DK.%26aulast%3DMaslak%26aufirst%3DP.%26atitle%3DAzacitidine%2520and%2520the%2520Beginnings%2520of%2520Therapeutic%2520Epigenetic%2520Modulation%26jtitle%3DExpert%2520Opin.%2520Pharmacother.%26date%3D2008%26volume%3D9%26spage%3D1981%26epage%3D1986%26doi%3D10.1517%2F14656566.9.11.1981" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kotla, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goel, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nischal, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heuck, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vivek, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Das, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verma, A.</span></span> <span> </span><span class="NLM_article-title">Mechanism of Action of Lenalidomide in Hematological Malignancies</span>. <i>J. Hematol. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">36</span>, <span class="refDoi"> DOI: 10.1186/1756-8722-2-36</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=10.1186%2F1756-8722-2-36" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=19674465" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=1%3ACAS%3A280%3ADC%252BD1Mrps1Smug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2009&pages=36&author=V.+Kotlaauthor=S.+Goelauthor=S.+Nischalauthor=C.+Heuckauthor=K.+Vivekauthor=B.+Dasauthor=A.+Verma&title=Mechanism+of+Action+of+Lenalidomide+in+Hematological+Malignancies&doi=10.1186%2F1756-8722-2-36"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Mechanism of action of lenalidomide in hematological malignancies</span></div><div class="casAuthors">Kotla Venumadhav; Goel Swati; Nischal Sangeeta; Heuck Christoph; Vivek Kumar; Das Bhaskar; Verma Amit</div><div class="citationInfo"><span class="NLM_cas:title">Journal of hematology & oncology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">36</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Immunomodulatory drugs lenalidomide and pomalidomide are synthetic compounds derived by modifying the chemical structure of thalidomide to improve its potency and reduce its side effects.  Lenalidomide is a 4-amino-glutamyl analogue of thalidomide that lacks the neurologic side effects of sedation and neuropathy and has emerged as a drug with activity against various hematological and solid malignancies.  It is approved by FDA for clinical use in myelodysplastic syndromes with deletion of chromosome 5q and multiple myeloma.  Lenalidomide has been shown to be an immunomodulator, affecting both cellular and humoral limbs of the immune system.  It has also been shown to have anti-angiogenic properties.  Newer studies demonstrate its effects on signal transduction that can partly explain its selective efficacy in subsets of MDS.  Even though the exact molecular targets of lenalidomide are not well known, its activity across a spectrum of neoplastic conditions highlights the possibility of multiple target sites of action.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRjgPlh7Q_C41VwdslNBF-tfW6udTcc2eb5NVL5gb5njbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD1Mrps1Smug%253D%253D&md5=9615a7d7c2871ab6cb710c8ea896404c</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1186%2F1756-8722-2-36&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1756-8722-2-36%26sid%3Dliteratum%253Aachs%26aulast%3DKotla%26aufirst%3DV.%26aulast%3DGoel%26aufirst%3DS.%26aulast%3DNischal%26aufirst%3DS.%26aulast%3DHeuck%26aufirst%3DC.%26aulast%3DVivek%26aufirst%3DK.%26aulast%3DDas%26aufirst%3DB.%26aulast%3DVerma%26aufirst%3DA.%26atitle%3DMechanism%2520of%2520Action%2520of%2520Lenalidomide%2520in%2520Hematological%2520Malignancies%26jtitle%3DJ.%2520Hematol.%2520Oncol.%26date%3D2009%26volume%3D2%26spage%3D36%26doi%3D10.1186%2F1756-8722-2-36" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gintant, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">George, C. H.</span></span> <span> </span><span class="NLM_article-title">Introduction to Biological Complexity as a Missing Link in Drug Discovery</span>. <i>Expert Opin. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">753</span>– <span class="NLM_lpage">763</span>, <span class="refDoi"> DOI: 10.1080/17460441.2018.1480608</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=10.1080%2F17460441.2018.1480608" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=29871539" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtVOqu7nK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2018&pages=753-763&author=G.+A.+Gintantauthor=C.+H.+George&title=Introduction+to+Biological+Complexity+as+a+Missing+Link+in+Drug+Discovery&doi=10.1080%2F17460441.2018.1480608"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Introduction to biological complexity as a missing link in drug discovery</span></div><div class="casAuthors">Gintant, Gary A.; George, Christopher H.</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Drug Discovery</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">753-763</span>CODEN:
                <span class="NLM_cas:coden">EODDBX</span>;
        ISSN:<span class="NLM_cas:issn">1746-0441</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">A review.  Despite a burgeoning knowledge of the intricacies and mechanisms responsible for human disease, technol. advances in medicinal chem., and more efficient assays used for drug screening, it remains difficult to discover novel and effective pharmacol. therapies.: By ref. to the primary literature and concepts emerging from academic and industrial drug screening landscapes, the authors propose that this disconnect arises from the inability to scale and integrate responses from simpler model systems to outcomes from more complex and human-based biol. systems.: Further collaborative efforts combining target-based and phenotypic-based screening along with systems-based pharmacol. and informatics will be necessary to harness the technol. breakthroughs of today to derive the novel drug candidates of tomorrow.  New questions must be asked of enabling technologies-while recognizing inherent limitations-in a way that moves drug development forward.  Attempts to integrate mechanistic and observational information acquired across multiple scales frequently expose the gap between our knowledge and our understanding as the level of complexity increases.  We hope that the thoughts and actionable items highlighted will help to inform the directed evolution of the drug discovery process.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGospFb4tctmaLVg90H21EOLACvtfcHk0ljMfkWvsk0uag"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtVOqu7nK&md5=74f5b440fe75340e28c1cd594d0600c0</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1080%2F17460441.2018.1480608&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F17460441.2018.1480608%26sid%3Dliteratum%253Aachs%26aulast%3DGintant%26aufirst%3DG.%2BA.%26aulast%3DGeorge%26aufirst%3DC.%2BH.%26atitle%3DIntroduction%2520to%2520Biological%2520Complexity%2520as%2520a%2520Missing%2520Link%2520in%2520Drug%2520Discovery%26jtitle%3DExpert%2520Opin.%2520Drug%2520Discovery%26date%3D2018%26volume%3D13%26spage%3D753%26epage%3D763%26doi%3D10.1080%2F17460441.2018.1480608" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, W. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ho, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, Y. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ling, T. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, C. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pan, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chou, H. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, G. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chu, W. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, Y. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, K. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, P. C.</span></span> <span> </span><span class="NLM_article-title">Cancer-Associated Fibroblasts Regulate the Plasticity of Lung Cancer Stemness Via Paracrine Signalling</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">3472</span>, <span class="refDoi"> DOI: 10.1038/ncomms4472</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=10.1038%2Fncomms4472" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=24668028" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=1%3ACAS%3A280%3ADC%252BC2crnt12rsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=3472&author=W.+J.+Chenauthor=C.+C.+Hoauthor=Y.+L.+Changauthor=H.+Y.+Chenauthor=C.+A.+Linauthor=T.+Y.+Lingauthor=S.+L.+Yuauthor=S.+S.+Yuanauthor=Y.+J.+Chenauthor=C.+Y.+Linauthor=S.+H.+Panauthor=H.+Y.+Chouauthor=Y.+J.+Chenauthor=G.+C.+Changauthor=W.+C.+Chuauthor=Y.+M.+Leeauthor=J.+Y.+Leeauthor=P.+J.+Leeauthor=K.+C.+Liauthor=H.+W.+Chenauthor=P.+C.+Yang&title=Cancer-Associated+Fibroblasts+Regulate+the+Plasticity+of+Lung+Cancer+Stemness+Via+Paracrine+Signalling&doi=10.1038%2Fncomms4472"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Cancer-associated fibroblasts regulate the plasticity of lung cancer stemness via paracrine signalling</span></div><div class="casAuthors">Chen Wan-Jiun; Ho Chao-Chi; Chang Yih-Leong; Chen Hsuan-Yu; Yuan Shin-Sheng; Chen Yu-Ju Louisa; Lin Chien-Yu; Lin Chih-An; Yu Sung-Liang; Ling Thai-Yen; Pan Szu-Hua; Chou Han-Yi Elizabeth; Chen Yu-Ju; Chang Gee-Chen; Chu Wen-Cheng; Lee Yee-Ming; Lee Jen-Yi; Lee Pei-Jung; Li Ker-Chau; Chen Huei-Wen; Yang Pan-Chyr</div><div class="citationInfo"><span class="NLM_cas:title">Nature communications</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">3472</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Cancer stem cells (CSCs) are a promising target for treating cancer, yet how CSC plasticity is maintained in vivo is unclear and is difficult to study in vitro.  Here we establish a sustainable primary culture of Oct3/4(+)/Nanog(+) lung CSCs fed with CD90(+) cancer-associated fibroblasts (CAFs) to further advance our knowledge of preserving stem cells in the tumour microenvironment.  Using transcriptomics we identify the paracrine network by which CAFs enrich CSCs through de-differentiation and reacquisition of stem cell-like properties.  Specifically, we find that IGF1R signalling activation in cancer cells in the presence of CAFs expressing IGF-II can induce Nanog expression and promote stemness.  Moreover, this paracrine signalling predicts overall and relapse-free survival in stage I non-small cell lung cancer (NSCLC) patients.  IGF-II/IGF1R signalling blockade inhibits Nanog expression and attenuates cancer stem cell features.  Our data demonstrate that CAFs constitute a supporting niche for cancer stemness, and targeting this paracrine signalling may present a new therapeutic strategy for NSCLC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTm3hC1BFRecNy3lHz94U-GfW6udTcc2eagU1RaubHPl7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2crnt12rsQ%253D%253D&md5=361b9cb8eee2a5a5b521f2d8a7ae7992</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1038%2Fncomms4472&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms4472%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DW.%2BJ.%26aulast%3DHo%26aufirst%3DC.%2BC.%26aulast%3DChang%26aufirst%3DY.%2BL.%26aulast%3DChen%26aufirst%3DH.%2BY.%26aulast%3DLin%26aufirst%3DC.%2BA.%26aulast%3DLing%26aufirst%3DT.%2BY.%26aulast%3DYu%26aufirst%3DS.%2BL.%26aulast%3DYuan%26aufirst%3DS.%2BS.%26aulast%3DChen%26aufirst%3DY.%2BJ.%26aulast%3DLin%26aufirst%3DC.%2BY.%26aulast%3DPan%26aufirst%3DS.%2BH.%26aulast%3DChou%26aufirst%3DH.%2BY.%26aulast%3DChen%26aufirst%3DY.%2BJ.%26aulast%3DChang%26aufirst%3DG.%2BC.%26aulast%3DChu%26aufirst%3DW.%2BC.%26aulast%3DLee%26aufirst%3DY.%2BM.%26aulast%3DLee%26aufirst%3DJ.%2BY.%26aulast%3DLee%26aufirst%3DP.%2BJ.%26aulast%3DLi%26aufirst%3DK.%2BC.%26aulast%3DChen%26aufirst%3DH.%2BW.%26aulast%3DYang%26aufirst%3DP.%2BC.%26atitle%3DCancer-Associated%2520Fibroblasts%2520Regulate%2520the%2520Plasticity%2520of%2520Lung%2520Cancer%2520Stemness%2520Via%2520Paracrine%2520Signalling%26jtitle%3DNat.%2520Commun.%26date%3D2014%26volume%3D5%26spage%3D3472%26doi%3D10.1038%2Fncomms4472" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kuo, T. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pan, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fang, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hung, P. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hong, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hsu, Y. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, P. C.</span></span> <span> </span><span class="NLM_article-title">Purine-Type Compounds Induce Microtubule Fragmentation and Lung Cancer Cell Death through Interaction with Katanin</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">8521</span>– <span class="NLM_lpage">8534</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b00797</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b00797" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtlOgtrnI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=8521-8534&author=T.+C.+Kuoauthor=L.+W.+Liauthor=S.+H.+Panauthor=J.+M.+Fangauthor=J.+H.+Liuauthor=T.+J.+Chengauthor=C.+J.+Wangauthor=P.+F.+Hungauthor=H.+Y.+Chenauthor=T.+M.+Hongauthor=Y.+L.+Hsuauthor=C.+H.+Wongauthor=P.+C.+Yang&title=Purine-Type+Compounds+Induce+Microtubule+Fragmentation+and+Lung+Cancer+Cell+Death+through+Interaction+with+Katanin&doi=10.1021%2Facs.jmedchem.6b00797"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Purine-Type Compounds Induce Microtubule Fragmentation and Lung Cancer Cell Death through Interaction with Katanin</span></div><div class="casAuthors">Kuo, Ting-Chun; Li, Ling-Wei; Pan, Szu-Hua; Fang, Jim-Min; Liu, Jyung-Hurng; Cheng, Ting-Jen; Wang, Chia-Jen; Hung, Pei-Fang; Chen, Hsuan-Yu; Hong, Tse-Ming; Hsu, Yuan-Ling; Wong, Chi-Huey; Yang, Pan-Chyr</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">8521-8534</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Microtubule targeting agents (MTAs) constitute a class of drugs for cancer treatment.  Despite many MTAs have been proven to significantly improve the treatment outcomes of various malignancies, resistance has usually occurred.  By selection from a 2-million entry chem. library based on the efficacy and safety, the authors identified purine-type compds. that were active against lung small cell lung cancer (NSCLC).  The purine compd. I (GRC0321) was an MTA with good effects against NSCLC.  Lung cancer cells H1975 treated with I could induce microtubule fragmentation, leading to G2/M cell cycle arrest and intrinsic apoptosis.  Compd. I directly targeted katanin and regulated the severing activity of katanin, which cut the cellular microtubules into short pieces and activated c-Jun N-terminal kinases (JNK).  The microtubule fragmenting effect of I is a unique mechanism in MTAs.  It might overcome the resistance problems that most of the MTAs have faced.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrHYxyvm45m4bVg90H21EOLACvtfcHk0lgpz5S7K110-Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtlOgtrnI&md5=069f8498e6fd14cac3f86324b77e09cb</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b00797&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b00797%26sid%3Dliteratum%253Aachs%26aulast%3DKuo%26aufirst%3DT.%2BC.%26aulast%3DLi%26aufirst%3DL.%2BW.%26aulast%3DPan%26aufirst%3DS.%2BH.%26aulast%3DFang%26aufirst%3DJ.%2BM.%26aulast%3DLiu%26aufirst%3DJ.%2BH.%26aulast%3DCheng%26aufirst%3DT.%2BJ.%26aulast%3DWang%26aufirst%3DC.%2BJ.%26aulast%3DHung%26aufirst%3DP.%2BF.%26aulast%3DChen%26aufirst%3DH.%2BY.%26aulast%3DHong%26aufirst%3DT.%2BM.%26aulast%3DHsu%26aufirst%3DY.%2BL.%26aulast%3DWong%26aufirst%3DC.%2BH.%26aulast%3DYang%26aufirst%3DP.%2BC.%26atitle%3DPurine-Type%2520Compounds%2520Induce%2520Microtubule%2520Fragmentation%2520and%2520Lung%2520Cancer%2520Cell%2520Death%2520through%2520Interaction%2520with%2520Katanin%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D8521%26epage%3D8534%26doi%3D10.1021%2Facs.jmedchem.6b00797" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Feng, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, C.</span></span> <span> </span><span class="NLM_article-title">Inhibiting the Expression of DNA Replication-Initiation Proteins Induces Apoptosis in Human Cancer Cells</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">7356</span>– <span class="NLM_lpage">7364</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=14612534" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=1%3ACAS%3A528%3ADC%252BD3sXovVGltL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2003&pages=7356-7364&author=D.+Fengauthor=Z.+Tuauthor=W.+Wuauthor=C.+Liang&title=Inhibiting+the+Expression+of+DNA+Replication-Initiation+Proteins+Induces+Apoptosis+in+Human+Cancer+Cells"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibiting the expression of DNA replication-initiation proteins induces apoptosis in human cancer cells</span></div><div class="casAuthors">Feng, Daorong; Tu, Zheng; Wu, Wenyan; Liang, Chun</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">7356-7364</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">DNA replication-initiation proteins are expressed in cancer cells, whereas some of these proteins are not expressed in nonproliferating normal cells.  Therefore, replication-initiation proteins may present attractive targets for anticancer therapy.  Using selected antisense oligodeoxynucleotides and small interfering RNA mols. targeted to the mRNA encoding the DNA replication-initiation proteins hCdc6p, hMcm2p, and hCdc45p, we show that the target genes could be effectively and specifically silenced and that, consequently, DNA replication and cell proliferation were inhibited in cultured human cells.  In addn., silencing of these genes resulted in apoptosis in both p53-pos. and -neg. cancer cells but not in normal cells: cancer cells entered an abortive S-phase, whereas normal cells arrested mainly in G1 phase.  Our studies are the first to suggest that inhibiting the expression of selective replication-initiation proteins is a novel and effective anticancer strategy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGorSWr21O_JrrVg90H21EOLACvtfcHk0lgpz5S7K110-Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXovVGltL8%253D&md5=2839ea7e0507d2ce35cf69c90204636b</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DFeng%26aufirst%3DD.%26aulast%3DTu%26aufirst%3DZ.%26aulast%3DWu%26aufirst%3DW.%26aulast%3DLiang%26aufirst%3DC.%26atitle%3DInhibiting%2520the%2520Expression%2520of%2520DNA%2520Replication-Initiation%2520Proteins%2520Induces%2520Apoptosis%2520in%2520Human%2520Cancer%2520Cells%26jtitle%3DCancer%2520Res.%26date%3D2003%26volume%3D63%26spage%3D7356%26epage%3D7364" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yamauchi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamaguchi, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakata, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kohno, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nagasaki, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shimamura, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Imoto, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saito, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ueno, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hatanaka, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoshida, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Higuchi, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nomura, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beer, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yokota, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miyano, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gotoh, N.</span></span> <span> </span><span class="NLM_article-title">Epidermal Growth Factor Receptor Tyrosine Kinase Defines Critical Prognostic Genes of Stage I Lung Adenocarcinoma</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>7</i></span>, <span class="NLM_elocation-id">e43923</span> <span class="refDoi"> DOI: 10.1371/journal.pone.0043923</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=10.1371%2Fjournal.pone.0043923" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=23028479" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsVaktrbL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2012&author=M.+Yamauchiauthor=R.+Yamaguchiauthor=A.+Nakataauthor=T.+Kohnoauthor=M.+Nagasakiauthor=T.+Shimamuraauthor=S.+Imotoauthor=A.+Saitoauthor=K.+Uenoauthor=Y.+Hatanakaauthor=R.+Yoshidaauthor=T.+Higuchiauthor=M.+Nomuraauthor=D.+G.+Beerauthor=J.+Yokotaauthor=S.+Miyanoauthor=N.+Gotoh&title=Epidermal+Growth+Factor+Receptor+Tyrosine+Kinase+Defines+Critical+Prognostic+Genes+of+Stage+I+Lung+Adenocarcinoma&doi=10.1371%2Fjournal.pone.0043923"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Epidermal growth factor receptor tyrosine kinase defines critical prognostic genes of stage I lung adenocarcinoma</span></div><div class="casAuthors">Yamauchi, Mai; Yamaguchi, Rui; Nakata, Asuka; Kohno, Takashi; Nagasaki, Masao; Shimamura, Teppei; Imoto, Seiya; Saito, Ayumu; Ueno, Kazuko; Hatanaka, Yousuke; Yoshida, Ryo; Higuchi, Tomoyuki; Nomura, Masaharu; Beer, David G.; Yokota, Jun; Miyano, Satoru; Gotoh, Noriko</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">e43923</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Purpose: To identify stage I lung adenocarcinoma patients with a poor prognosis who will benefit from adjuvant therapy.  Patients and Methods: Whole gene expression profiles were obtained at 19 time points over a 48-h time course from human primary lung epithelial cells that were stimulated with epidermal growth factor (EGF) in the presence or absence of a clin. used EGF receptor tyrosine kinase (RTK)-specific inhibitor, gefitinib.  The data were subjected to a math. simulation using the State Space Model (SSM).  "Gefitinib-sensitive" genes, the expressional dynamics of which were altered by addn. of gefitinib, were identified.  A risk scoring model was constructed to classify high- or low-risk patients based on expression signatures of 139 gefitinib-sensitive genes in lung cancer using a training data set of 253 lung adenocarcinomas of North American cohort.  The predictive ability of the risk scoring model was examd. in independent cohorts of surgical specimens of lung cancer.  Results: The risk scoring model enabled the identification of high-risk stage IA and IB cases in another North American cohort for overall survival (OS) with a hazard ratio (HR) of 7.16 (P = 0.029) and 3.26 (P = 0.0072), resp.  It also enabled the identification of high-risk stage I cases without bronchioalveolar carcinoma (BAC) histol. in a Japanese cohort for OS and recurrence-free survival (RFS) with HRs of 8.79 (P = 0.001) and 3.72 (P = 0.0049), resp.  Conclusion: The set of 139 gefitinib-sensitive genes includes many genes known to be involved in biol. aspects of cancer phenotypes, but not known to be involved in EGF signaling.  The present result strongly re-emphasizes that EGF signaling status in cancer cells underlies an aggressive phenotype of cancer cells, which is useful for the selection of early-stage lung adenocarcinoma patients with a poor prognosis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGolLhiLU3WpObVg90H21EOLACvtfcHk0liKW8fiIBiGYw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsVaktrbL&md5=4b2ef5e7e98350134ce442b47d57df87</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0043923&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0043923%26sid%3Dliteratum%253Aachs%26aulast%3DYamauchi%26aufirst%3DM.%26aulast%3DYamaguchi%26aufirst%3DR.%26aulast%3DNakata%26aufirst%3DA.%26aulast%3DKohno%26aufirst%3DT.%26aulast%3DNagasaki%26aufirst%3DM.%26aulast%3DShimamura%26aufirst%3DT.%26aulast%3DImoto%26aufirst%3DS.%26aulast%3DSaito%26aufirst%3DA.%26aulast%3DUeno%26aufirst%3DK.%26aulast%3DHatanaka%26aufirst%3DY.%26aulast%3DYoshida%26aufirst%3DR.%26aulast%3DHiguchi%26aufirst%3DT.%26aulast%3DNomura%26aufirst%3DM.%26aulast%3DBeer%26aufirst%3DD.%2BG.%26aulast%3DYokota%26aufirst%3DJ.%26aulast%3DMiyano%26aufirst%3DS.%26aulast%3DGotoh%26aufirst%3DN.%26atitle%3DEpidermal%2520Growth%2520Factor%2520Receptor%2520Tyrosine%2520Kinase%2520Defines%2520Critical%2520Prognostic%2520Genes%2520of%2520Stage%2520I%2520Lung%2520Adenocarcinoma%26jtitle%3DPLoS%2520One%26date%3D2012%26volume%3D7%26doi%3D10.1371%2Fjournal.pone.0043923" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Forsburg, S. L.</span></span> <span> </span><span class="NLM_article-title">Eukaryotic Mcm Proteins: Beyond Replication Initiation</span>. <i>Microbiol Mol. Biol. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>68</i></span>,  <span class="NLM_fpage">109</span>– <span class="NLM_lpage">131</span>, <span class="refDoi"> DOI: 10.1128/MMBR.68.1.109-131.2004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=10.1128%2FMMBR.68.1.109-131.2004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=15007098" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=1%3ACAS%3A528%3ADC%252BD2cXjtFehtr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=68&publication_year=2004&pages=109-131&author=S.+L.+Forsburg&title=Eukaryotic+Mcm+Proteins%3A+Beyond+Replication+Initiation&doi=10.1128%2FMMBR.68.1.109-131.2004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Eukaryotic MCM proteins: beyond replication initiation</span></div><div class="casAuthors">Forsburg, Susan L.</div><div class="citationInfo"><span class="NLM_cas:title">Microbiology and Molecular Biology Reviews</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">68</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">109-131</span>CODEN:
                <span class="NLM_cas:coden">MMBRF7</span>;
        ISSN:<span class="NLM_cas:issn">1092-2172</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">A review.  The minichromosome maintenance (or MCM) protein family is composed of six related proteins that are conserved in all eukaryotes.  They were first identified by genetic screens in yeast and subsequently analyzed in other exptl. systems using mol. and biochem. methods.  Early data led to the identification of MCMs as central players in the initiation of DNA replication.  More recent studies have shown that MCM proteins also function in replication elongation, probably as a DNA helicase.  This is consistent with structural anal. showing that the proteins interact together in a heterohexameric ring.  However, MCMs are strikingly abundant and far exceed the stoichiometry of replication origins; they are widely distributed on unreplicated chromatin.  Anal. of mcm mutant phenotypes and interactions with other factors have now implicated the MCM proteins in other chromosome transactions including damage response, transcription, and chromatin structure.  These expts. indicate that the MCMs are central players in many aspects of genome stability.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr-4X_4eAIXyrVg90H21EOLACvtfcHk0liKW8fiIBiGYw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXjtFehtr8%253D&md5=f01251abb8eaad801108d1737ae4e652</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1128%2FMMBR.68.1.109-131.2004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMMBR.68.1.109-131.2004%26sid%3Dliteratum%253Aachs%26aulast%3DForsburg%26aufirst%3DS.%2BL.%26atitle%3DEukaryotic%2520Mcm%2520Proteins%253A%2520Beyond%2520Replication%2520Initiation%26jtitle%3DMicrobiol%2520Mol.%2520Biol.%2520Rev.%26date%3D2004%26volume%3D68%26spage%3D109%26epage%3D131%26doi%3D10.1128%2FMMBR.68.1.109-131.2004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Emanuele, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elia, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thoma, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Izhar, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leng, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rush, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hsu, P. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yen, H. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elledge, S. J.</span></span> <span> </span><span class="NLM_article-title">Global Identification of Modular Cullin-Ring Ligase Substrates</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>147</i></span>,  <span class="NLM_fpage">459</span>– <span class="NLM_lpage">474</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2011.09.019</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=10.1016%2Fj.cell.2011.09.019" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=21963094" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtlaksbvJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=147&publication_year=2011&pages=459-474&author=M.+J.+Emanueleauthor=A.+E.+Eliaauthor=Q.+Xuauthor=C.+R.+Thomaauthor=L.+Izharauthor=Y.+Lengauthor=A.+Guoauthor=Y.+N.+Chenauthor=J.+Rushauthor=P.+W.+Hsuauthor=H.+C.+Yenauthor=S.+J.+Elledge&title=Global+Identification+of+Modular+Cullin-Ring+Ligase+Substrates&doi=10.1016%2Fj.cell.2011.09.019"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Global Identification of Modular Cullin-RING Ligase Substrates</span></div><div class="casAuthors">Emanuele, Michael J.; Elia, Andrew E. H.; Xu, Qikai; Thoma, Claudio R.; Izhar, Lior; Leng, Yumei; Guo, Ailan; Chen, Yi-Ning; Rush, John; Hsu, Paul Wei-Che; Yen, Hsueh-Chi Sherry; Elledge, Stephen J.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">147</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">459-474</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Cullin-RING ligases (CRLs) represent the largest E3 ubiquitin ligase family in eukaryotes, and the identification of their substrates is crit. to understanding regulation of the proteome.  Using genetic and pharmacol. Cullin inactivation coupled with genetic (GPS) and proteomic (QUAINT) assays, we have identified hundreds of proteins whose stabilities or ubiquitylation status are regulated by CRLs.  Together, these approaches yielded many known CRL substrates as well as a multitude of previously unknown putative substrates.  We demonstrate that one substrate, NUSAP1, is an SCFCyclin F substrate during S and G2 phases of the cell cycle and is also degraded in response to DNA damage.  This collection of regulated substrates is highly enriched for nodes in protein interaction networks, representing crit. connections between regulatory pathways.  This demonstrates the broad role of CRL ubiquitylation in all aspects of cellular biol. and provides a set of proteins likely to be key indicators of cellular physiol.  A combination of genetic and proteomic screens reveals cullin-RING ligase substrates and places these factors at key regulatory nodes in cellular networks.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpr1k89XZLzgrVg90H21EOLACvtfcHk0liKW8fiIBiGYw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtlaksbvJ&md5=294df7034b0127b3a6f2f26977634f81</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2011.09.019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2011.09.019%26sid%3Dliteratum%253Aachs%26aulast%3DEmanuele%26aufirst%3DM.%2BJ.%26aulast%3DElia%26aufirst%3DA.%2BE.%26aulast%3DXu%26aufirst%3DQ.%26aulast%3DThoma%26aufirst%3DC.%2BR.%26aulast%3DIzhar%26aufirst%3DL.%26aulast%3DLeng%26aufirst%3DY.%26aulast%3DGuo%26aufirst%3DA.%26aulast%3DChen%26aufirst%3DY.%2BN.%26aulast%3DRush%26aufirst%3DJ.%26aulast%3DHsu%26aufirst%3DP.%2BW.%26aulast%3DYen%26aufirst%3DH.%2BC.%26aulast%3DElledge%26aufirst%3DS.%2BJ.%26atitle%3DGlobal%2520Identification%2520of%2520Modular%2520Cullin-Ring%2520Ligase%2520Substrates%26jtitle%3DCell%26date%3D2011%26volume%3D147%26spage%3D459%26epage%3D474%26doi%3D10.1016%2Fj.cell.2011.09.019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Z. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jang, Y. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, Y. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, W.</span></span> <span> </span><span class="NLM_article-title">A Versatile and Practical Method for Regioselective Synthesis of Polysubstituted Furanonaphthoquinones</span>. <i>Org. Biomol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">828</span>– <span class="NLM_lpage">834</span>, <span class="refDoi"> DOI: 10.1039/C2OB26986D</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=10.1039%2FC2OB26986D" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=23235915" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=1%3ACAS%3A528%3ADC%252BC3sXjtlGjtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2013&pages=828-834&author=Z.+Z.+Wuauthor=Y.+J.+Jangauthor=C.+J.+Leeauthor=Y.+T.+Leeauthor=W.+Lin&title=A+Versatile+and+Practical+Method+for+Regioselective+Synthesis+of+Polysubstituted+Furanonaphthoquinones&doi=10.1039%2FC2OB26986D"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">A versatile and practical method for regioselective synthesis of polysubstituted furanonaphthoquinones</span></div><div class="casAuthors">Wu, Zong-Ze; Jang, Yeong-Jiunn; Lee, Chia-Jui; Lee, Yen-Te; Lin, Wenwei</div><div class="citationInfo"><span class="NLM_cas:title">Organic & Biomolecular Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">828-834</span>CODEN:
                <span class="NLM_cas:coden">OBCRAK</span>;
        ISSN:<span class="NLM_cas:issn">1477-0520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">An efficient and attractive synthesis of a series of novel poly-functionalized phosphorus zwitterions was achieved via three-component reactions of the corresponding functional nucleophiles, aldehydes, and Bu3P in the presence of acidic promoter.  E.g., in presence of TFA, reaction of 2-hydroxynaphthoquinone, 4-BrC6H4CHO, and Bu3P gave 83% phosphorus zwitterion (I).  These polysubstituted zwitterions could regioselectively undergo further transformations to synthetically important furanonaphthoquinones and related products via the intramol. Wittig reaction.  E.g., reaction of I with PhCOCl gave 92% furanonaphthoquinone (II).  These methods could have potential application in synthetic and pharmaceutical chem. for its facilitation and easily accessible com. materials.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoa0oHaxvHZCbVg90H21EOLACvtfcHk0liyGvWTEjC8jg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXjtlGjtQ%253D%253D&md5=d67158fbebdf005883d5b02c98f79d73</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1039%2FC2OB26986D&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC2OB26986D%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DZ.%2BZ.%26aulast%3DJang%26aufirst%3DY.%2BJ.%26aulast%3DLee%26aufirst%3DC.%2BJ.%26aulast%3DLee%26aufirst%3DY.%2BT.%26aulast%3DLin%26aufirst%3DW.%26atitle%3DA%2520Versatile%2520and%2520Practical%2520Method%2520for%2520Regioselective%2520Synthesis%2520of%2520Polysubstituted%2520Furanonaphthoquinones%26jtitle%3DOrg.%2520Biomol.%2520Chem.%26date%3D2013%26volume%3D11%26spage%3D828%26epage%3D834%26doi%3D10.1039%2FC2OB26986D" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sakakibara, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kelman, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kelman, Z.</span></span> <span> </span><span class="NLM_article-title">Unwinding the Structure and Function of the Archaeal Mcm Helicase</span>. <i>Mol. Microbiol.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>72</i></span>,  <span class="NLM_fpage">286</span>– <span class="NLM_lpage">296</span>, <span class="refDoi"> DOI: 10.1111/j.1365-2958.2009.06663.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=10.1111%2Fj.1365-2958.2009.06663.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=19415794" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=1%3ACAS%3A528%3ADC%252BD1MXlsFSrtrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=72&publication_year=2009&pages=286-296&author=N.+Sakakibaraauthor=L.+M.+Kelmanauthor=Z.+Kelman&title=Unwinding+the+Structure+and+Function+of+the+Archaeal+Mcm+Helicase&doi=10.1111%2Fj.1365-2958.2009.06663.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Unwinding the structure and function of the archaeal MCM helicase</span></div><div class="casAuthors">Sakakibara, Nozomi; Kelman, Lori M.; Kelman, Zvi</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Microbiology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">72</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">286-296</span>CODEN:
                <span class="NLM_cas:coden">MOMIEE</span>;
        ISSN:<span class="NLM_cas:issn">0950-382X</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">A review.  During chromosomal DNA replication, the replicative helicase unwinds the duplex DNA to provide the single-stranded DNA substrate for the polymerase.  In archaea, the replicative helicase is the minichromosome maintenance (MCM) complex.  The enzyme utilizes the energy of ATP hydrolysis to translocate along one strand of the duplex and unwind the complementary strand.  Much progress has been made in elucidating structure and function since the 1st report on the biochem. properties of an archaeal MCM protein in 1999.  The biochem. and structural properties of the enzyme from several archaeal species and some of the mechanisms by which the enzyme is regulated are now known.  Here, the authors summarize recent studies on the archaeal MCM protein and discusses the implications for helicase function and DNA replication in archaea.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrga7VKoWZbO7Vg90H21EOLACvtfcHk0liyGvWTEjC8jg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXlsFSrtrk%253D&md5=6ff2a1537d6775376390450e7ebd580a</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1111%2Fj.1365-2958.2009.06663.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1365-2958.2009.06663.x%26sid%3Dliteratum%253Aachs%26aulast%3DSakakibara%26aufirst%3DN.%26aulast%3DKelman%26aufirst%3DL.%2BM.%26aulast%3DKelman%26aufirst%3DZ.%26atitle%3DUnwinding%2520the%2520Structure%2520and%2520Function%2520of%2520the%2520Archaeal%2520Mcm%2520Helicase%26jtitle%3DMol.%2520Microbiol.%26date%3D2009%26volume%3D72%26spage%3D286%26epage%3D296%26doi%3D10.1111%2Fj.1365-2958.2009.06663.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fletcher, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bishop, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leon, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sclafani, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ogata, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X. S.</span></span> <span> </span><span class="NLM_article-title">The Structure and Function of Mcm from Archaeal M. Thermoautotrophicum</span>. <i>Nat. Struct. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">160</span>– <span class="NLM_lpage">167</span>, <span class="refDoi"> DOI: 10.1038/nsb893</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=10.1038%2Fnsb893" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2003&pages=160-167&author=R.+J.+Fletcherauthor=B.+E.+Bishopauthor=R.+P.+Leonauthor=R.+A.+Sclafaniauthor=C.+M.+Ogataauthor=X.+S.+Chen&title=The+Structure+and+Function+of+Mcm+from+Archaeal+M.+Thermoautotrophicum&doi=10.1038%2Fnsb893"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1038%2Fnsb893&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnsb893%26sid%3Dliteratum%253Aachs%26aulast%3DFletcher%26aufirst%3DR.%2BJ.%26aulast%3DBishop%26aufirst%3DB.%2BE.%26aulast%3DLeon%26aufirst%3DR.%2BP.%26aulast%3DSclafani%26aufirst%3DR.%2BA.%26aulast%3DOgata%26aufirst%3DC.%2BM.%26aulast%3DChen%26aufirst%3DX.%2BS.%26atitle%3DThe%2520Structure%2520and%2520Function%2520of%2520Mcm%2520from%2520Archaeal%2520M.%2520Thermoautotrophicum%26jtitle%3DNat.%2520Struct.%2520Mol.%2520Biol.%26date%3D2003%26volume%3D10%26spage%3D160%26epage%3D167%26doi%3D10.1038%2Fnsb893" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fletcher, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X. S.</span></span> <span> </span><span class="NLM_article-title">Biochemical Activities of the Bob1 Mutant in Methanobacterium Thermoautotrophicum Mcm</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>45</i></span>,  <span class="NLM_fpage">462</span>– <span class="NLM_lpage">467</span>, <span class="refDoi"> DOI: 10.1021/bi051754z</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi051754z" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtlWmurvM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2006&pages=462-467&author=R.+J.+Fletcherauthor=X.+S.+Chen&title=Biochemical+Activities+of+the+Bob1+Mutant+in+Methanobacterium+Thermoautotrophicum+Mcm&doi=10.1021%2Fbi051754z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Biochemical Activities of the BOB1 Mutant in Methanobacterium thermoautotrophicum MCM</span></div><div class="casAuthors">Fletcher, Ryan J.; Chen, Xiaojiang S.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">462-467</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Minichromosomal maintenance proteins (MCMs) are considered to be the replicative helicase.  Methanobacterium thermoautotrophicum has a single MCM gene (mtMCM).  The crystal structure of the mtMCM N-terminal region is a double hexamer.  Structure-guided sequence alignment indicates a structural conservation of this fragment across archaeal and eukaryotic MCMs.  The mtMCM structure was successfully used to analyze a Saccharomyces cerevisiae MCM5 mutant, called BOB1, which contains a single residue change from Pro to Leu and bypasses a kinase normally required for initiation of DNA replication.  A domain-push model was proposed to explain the BOB1 bypass activity.  Here we investigate the effects of BOB1 mutation on the biochem. activities of mtMCM.  Surprisingly, the BOB1 mutation (P62L) had a major effect on the helicase activity but had no significant impact on DNA binding and ATPase activities.  These results will contribute to a more detailed understanding of the BOB1 bypass activity and other aspects of DNA replication control.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrKMkxzfOUFS7Vg90H21EOLACvtfcHk0lhYIu8a3OjMsg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtlWmurvM&md5=4d5c1ad5e48a699718a4a8ebf1fdc9bc</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1021%2Fbi051754z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi051754z%26sid%3Dliteratum%253Aachs%26aulast%3DFletcher%26aufirst%3DR.%2BJ.%26aulast%3DChen%26aufirst%3DX.%2BS.%26atitle%3DBiochemical%2520Activities%2520of%2520the%2520Bob1%2520Mutant%2520in%2520Methanobacterium%2520Thermoautotrophicum%2520Mcm%26jtitle%3DBiochemistry%26date%3D2006%26volume%3D45%26spage%3D462%26epage%3D467%26doi%3D10.1021%2Fbi051754z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wan, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Q. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bai, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span> <span> </span><span class="NLM_article-title">Norcantharidin Inhibits Pre-Replicative Complexes Assembly of Hepg2 Cells</span>. <i>Am. J. Chin. Med.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>41</i></span>,  <span class="NLM_fpage">665</span>– <span class="NLM_lpage">682</span>, <span class="refDoi"> DOI: 10.1142/S0192415X13500468</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=10.1142%2FS0192415X13500468" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=23711148" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=1%3ACAS%3A528%3ADC%252BC3sXosFaqt7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2013&pages=665-682&author=S.+Chenauthor=X.+Quauthor=P.+Wanauthor=Q.+W.+Liauthor=Z.+Wangauthor=F.+Guoauthor=L.+Baiauthor=Z.+Huauthor=W.+Tanauthor=J.+Li&title=Norcantharidin+Inhibits+Pre-Replicative+Complexes+Assembly+of+Hepg2+Cells&doi=10.1142%2FS0192415X13500468"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Norcantharidin Inhibits Pre-Replicative Complexes Assembly of HepG2 Cells</span></div><div class="casAuthors">Chen, Sansan; Qu, Xinming; Wan, Pei; Li, Qing Wen; Wang, Ziyi; Guo, Fangfang; Bai, Li; Hu, Zhiming; Tan, Wanlong; Li, Jinlong</div><div class="citationInfo"><span class="NLM_cas:title">American Journal of Chinese Medicine</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">665-682</span>CODEN:
                <span class="NLM_cas:coden">AJCMBA</span>;
        ISSN:<span class="NLM_cas:issn">0192-415X</span>.
    
            (<span class="NLM_cas:orgname">World Scientific Publishing Co. Pte. Ltd.</span>)
        </div><div class="casAbstract">Norcantharidin (NCTD) is currently used for anticancer therapy but the exact mechanism of action remains unknown.  Pre-replicative complexes (pre-RCs) are essential for cell DNA replication and highly related to malignant proliferation.  Here, we examd. the inhibitory effect of NCTD on pre-RC components in HepG2 cells.  We showed that NCTD induced degrdn. of Cdc6 and Mcm2 in a dose-dependent manner.  Under 100 μM NCTD concn., about 70% of Cdc6 and 50% of Mcm2 were degraded.  In addn., the nuclear translocation of Mcm6 was inhibited by NCTD.  Further studies aiming at G1 synchronous cells showed that, NCTD reduced the chromatin-bound Cdc6, Mcm2 and Mcm6.  Moreover, the cells were blocked from entering the S phase and accumulated at the G1 phase when released synchronously into the cell cycle.  Consistently, the DNA replication was inhibited by NCTD.  Finally, the combination NCTD with Cdc6 depletion lead to more severe cytotoxicity (88%) than NCTD (52%) and Cdc6 depletion (39%) alone.  A synergic cytotoxicity was obsd. between Cdc6 depletion and NCTD.  In conclusion, our results demonstrate that NCTD inhibits pre-RC assembly; subsequently blocks the G1 to S transition; and inhibits DNA replication in HepG2 cells.  Pre-RCs are an intriguing target for cancer therapy, which merits further investigations for anticancer development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpPKHJs6CODj7Vg90H21EOLACvtfcHk0lhYIu8a3OjMsg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXosFaqt7k%253D&md5=85ba7ab64ff2f650bf202414de1a35ba</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1142%2FS0192415X13500468&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1142%252FS0192415X13500468%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DS.%26aulast%3DQu%26aufirst%3DX.%26aulast%3DWan%26aufirst%3DP.%26aulast%3DLi%26aufirst%3DQ.%2BW.%26aulast%3DWang%26aufirst%3DZ.%26aulast%3DGuo%26aufirst%3DF.%26aulast%3DBai%26aufirst%3DL.%26aulast%3DHu%26aufirst%3DZ.%26aulast%3DTan%26aufirst%3DW.%26aulast%3DLi%26aufirst%3DJ.%26atitle%3DNorcantharidin%2520Inhibits%2520Pre-Replicative%2520Complexes%2520Assembly%2520of%2520Hepg2%2520Cells%26jtitle%3DAm.%2520J.%2520Chin.%2520Med.%26date%3D2013%26volume%3D41%26spage%3D665%26epage%3D682%26doi%3D10.1142%2FS0192415X13500468" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guan, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, B.</span></span> <span> </span><span class="NLM_article-title">Mcm-2 Is a Therapeutic Target of Trichostatin a in Colon Cancer Cells</span>. <i>Toxicol. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>221</i></span>,  <span class="NLM_fpage">23</span>– <span class="NLM_lpage">30</span>, <span class="refDoi"> DOI: 10.1016/j.toxlet.2013.05.643</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=10.1016%2Fj.toxlet.2013.05.643" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=23770000" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtVKms7jN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=221&publication_year=2013&pages=23-30&author=Y.+Liuauthor=G.+Heauthor=Y.+Wangauthor=X.+Guanauthor=X.+Pangauthor=B.+Zhang&title=Mcm-2+Is+a+Therapeutic+Target+of+Trichostatin+a+in+Colon+Cancer+Cells&doi=10.1016%2Fj.toxlet.2013.05.643"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">MCM-2 is a therapeutic target of Trichostatin A in colon cancer cells</span></div><div class="casAuthors">Liu, Yangbo; He, Gang; Wang, Yan; Guan, Xinging; Pang, Xueli; Zhang, Bo</div><div class="citationInfo"><span class="NLM_cas:title">Toxicology Letters</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">221</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">23-30</span>CODEN:
                <span class="NLM_cas:coden">TOLED5</span>;
        ISSN:<span class="NLM_cas:issn">0378-4274</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ireland Ltd.</span>)
        </div><div class="casAbstract">Histone deacetylase (HDAC) inhibitors have recently emerged as a new class of anti-cancer agents.  Trichostatin A (TSA), a classical HDAC inhibitor, has been demonstrated to induce cell cycle arrest, promote cell apoptosis, and inhibit metastasis.  However, the mol. mechanism underlying TSA function has not been fully elucidated.  In the current study, we found that TSA treatment induced altered expression of cell cycle-assocd. genes in HCT116 cells by RT-PCR array.  Among the 84 genes related to cell cycle control, 34 genes were significantly altered by TSA treatment, with 7 genes upregulated and 27 genes downregulated.  Interestingly, gene expression of minichromosome maintenance protein-2 (MCM-2) was significantly downregulated by TSA treatment.  This was confirmed by quant. RT-PCR and Western blotting.  Moreover, silencing of MCM-2 by siRNA led to cell cycle arrest and apoptosis in HCT116 cells.  In addn., TSA caused an increase of phosphorylated JNK, which was involved in downregulation of MCM-2.  Together, our results suggest that MCM-2 is a novel therapeutic target of TSA in colon cancer cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo7hNc0Yjj8NLVg90H21EOLACvtfcHk0lhYIu8a3OjMsg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtVKms7jN&md5=098a399e3f5445d2a9429428550ae861</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1016%2Fj.toxlet.2013.05.643&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.toxlet.2013.05.643%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DHe%26aufirst%3DG.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DGuan%26aufirst%3DX.%26aulast%3DPang%26aufirst%3DX.%26aulast%3DZhang%26aufirst%3DB.%26atitle%3DMcm-2%2520Is%2520a%2520Therapeutic%2520Target%2520of%2520Trichostatin%2520a%2520in%2520Colon%2520Cancer%2520Cells%26jtitle%3DToxicol.%2520Lett.%26date%3D2013%26volume%3D221%26spage%3D23%26epage%3D30%26doi%3D10.1016%2Fj.toxlet.2013.05.643" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chu, Y. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, P. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shyu, Y. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hendrix, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, C. W.</span></span> <span> </span><span class="NLM_article-title">Selection of Invasive and Metastatic Subpopulations from a Human Lung Adenocarcinoma Cell Line</span>. <i>Am. J. Respir. Cell Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">353</span>– <span class="NLM_lpage">360</span>, <span class="refDoi"> DOI: 10.1165/ajrcmb.17.3.2837</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=10.1165%2Fajrcmb.17.3.2837" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=9308922" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=1%3ACAS%3A528%3ADyaK2sXmtVOrs7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=1997&pages=353-360&author=Y.+W.+Chuauthor=P.+C.+Yangauthor=S.+C.+Yangauthor=Y.+C.+Shyuauthor=M.+J.+Hendrixauthor=R.+Wuauthor=C.+W.+Wu&title=Selection+of+Invasive+and+Metastatic+Subpopulations+from+a+Human+Lung+Adenocarcinoma+Cell+Line&doi=10.1165%2Fajrcmb.17.3.2837"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Selection of invasive and metastatic subpopulations from a human lung adenocarcinoma cell line</span></div><div class="casAuthors">Chu, Yi-Wen; Yang, Pan-Chyr; Yang, Shuenn-Chen; Shyu, Yu-Chiau; Hendrix, Mary J. C.; Wu, Reen; Wu, Cheng-Wen</div><div class="citationInfo"><span class="NLM_cas:title">American Journal of Respiratory Cell and Molecular Biology</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">353-360</span>CODEN:
                <span class="NLM_cas:coden">AJRBEL</span>;
        ISSN:<span class="NLM_cas:issn">1044-1549</span>.
    
            (<span class="NLM_cas:orgname">American Lung Association</span>)
        </div><div class="casAbstract">To better understand the mechanism(s) underlying lung cancer invasion and metastasis, a Transwell invasion chamber was used to select progressively more invasive cancer cell populations from a clonal cell line of human lung adenocarcinoma, CL1.  Five sublines with progressive invasiveness, designated CL1-1, CL1-2, CL1-3, CL1-4, and CL1-5, were obtained through this in vitro selection process.  Their invasive abilities through basement membrane matrix showed a 4-6-fold increase over that of the parental cells.  Moreover, the sublines manifested an increase in their colony-forming ability on soft agar, tumorigenicity, and metastatic potency in severe combined immunodeficiency (SCID) mice.  Examg. the phenotypes of the cell lines revealed increased expression of 92-kDa gelatinase and an increase in the cell population stained with anti-keratin-8 and -18 antibodies.  Clonal isolation of anti-keratin-18-antibody-pos. and -neg. cell populations demonstrated a correlated enhancement of the invasiveness of these cells and their expression of keratin-18.  Thus, the metastatic behavior of lung cancer cells can be characterized with this in vitro system, and the properties of these progressively invasive cancer cells can be clonally studied.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoqf3X5hlC0wLVg90H21EOLACvtfcHk0lhYIu8a3OjMsg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXmtVOrs7c%253D&md5=d5ea3f2aa0517622c6e6026663cc1a03</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1165%2Fajrcmb.17.3.2837&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1165%252Fajrcmb.17.3.2837%26sid%3Dliteratum%253Aachs%26aulast%3DChu%26aufirst%3DY.%2BW.%26aulast%3DYang%26aufirst%3DP.%2BC.%26aulast%3DYang%26aufirst%3DS.%2BC.%26aulast%3DShyu%26aufirst%3DY.%2BC.%26aulast%3DHendrix%26aufirst%3DM.%2BJ.%26aulast%3DWu%26aufirst%3DR.%26aulast%3DWu%26aufirst%3DC.%2BW.%26atitle%3DSelection%2520of%2520Invasive%2520and%2520Metastatic%2520Subpopulations%2520from%2520a%2520Human%2520Lung%2520Adenocarcinoma%2520Cell%2520Line%26jtitle%3DAm.%2520J.%2520Respir.%2520Cell%2520Mol.%2520Biol.%26date%3D1997%26volume%3D17%26spage%3D353%26epage%3D360%26doi%3D10.1165%2Fajrcmb.17.3.2837" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, W. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, Y. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, C. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chao, Y. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kao, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, W. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, K. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hong, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, P. C.</span></span> <span> </span><span class="NLM_article-title">P53 Controls Cancer Cell Invasion by Inducing the Mdm2-Mediated Degradation of Slug</span>. <i>Nat. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">694</span>– <span class="NLM_lpage">704</span>, <span class="refDoi"> DOI: 10.1038/ncb1875</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=10.1038%2Fncb1875" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=19448627" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=1%3ACAS%3A528%3ADC%252BD1MXms1als7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2009&pages=694-704&author=S.+P.+Wangauthor=W.+L.+Wangauthor=Y.+L.+Changauthor=C.+T.+Wuauthor=Y.+C.+Chaoauthor=S.+H.+Kaoauthor=A.+Yuanauthor=C.+W.+Linauthor=S.+C.+Yangauthor=W.+K.+Chanauthor=K.+C.+Liauthor=T.+M.+Hongauthor=P.+C.+Yang&title=P53+Controls+Cancer+Cell+Invasion+by+Inducing+the+Mdm2-Mediated+Degradation+of+Slug&doi=10.1038%2Fncb1875"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">p53 controls cancer cell invasion by inducing the MDM2-mediated degradation of Slug</span></div><div class="casAuthors">Wang, Shu-Ping; Wang, Wen-Lung; Chang, Yih-Leong; Wu, Chen-Tu; Chao, Yu-Chih; Kao, Shih-Han; Yuan, Ang; Lin, Chung-Wu; Yang, Shuenn-Chen; Chan, Wing-Kai; Li, Ker-Chau; Hong, Tse-Ming; Yang, Pan-Chyr</div><div class="citationInfo"><span class="NLM_cas:title">Nature Cell Biology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">694-704</span>CODEN:
                <span class="NLM_cas:coden">NCBIFN</span>;
        ISSN:<span class="NLM_cas:issn">1465-7392</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The tumor suppressor p53 is known to prevent cancer progression by inhibiting proliferation and inducing apoptosis of tumor cells.  Slug, an invasion promoter, exerts its effects by repressing E-cadherin transcription.  Here we show that wild-type p53 (wtp53) suppresses cancer invasion by inducing Slug degrdn., whereas mutant p53 may stabilize Slug protein.  In non-small-cell lung cancer (NSCLC), mutation of p53 correlates with low MDM2, high Slug and low E-cadherin expression.  This expression profile is assocd. with poor overall survival and short metastasis-free survival in patients with NSCLC. wtp53 upregulates MDM2 and forms a wtp53-MDM2-Slug complex that facilitates MDM2-mediated Slug degrdn.  Downregulation of Slug by wtp53 or MDM2 enhances E-cadherin expression and represses cancer cell invasiveness.  In contrast, mutant p53 inactivates Slug degrdn. and leads to Slug accumulation and increased cancer cell invasiveness.  Our findings indicate that wtp53 and p53 mutants may differentially control cancer invasion and metastasis through the p53-MDM2-Slug pathway.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpv0v8DzU0h_7Vg90H21EOLACvtfcHk0lgy7HADpO7c1w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXms1als7g%253D&md5=71ce0ce22e87b86faf76c355afbcaff6</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1038%2Fncb1875&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncb1875%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DS.%2BP.%26aulast%3DWang%26aufirst%3DW.%2BL.%26aulast%3DChang%26aufirst%3DY.%2BL.%26aulast%3DWu%26aufirst%3DC.%2BT.%26aulast%3DChao%26aufirst%3DY.%2BC.%26aulast%3DKao%26aufirst%3DS.%2BH.%26aulast%3DYuan%26aufirst%3DA.%26aulast%3DLin%26aufirst%3DC.%2BW.%26aulast%3DYang%26aufirst%3DS.%2BC.%26aulast%3DChan%26aufirst%3DW.%2BK.%26aulast%3DLi%26aufirst%3DK.%2BC.%26aulast%3DHong%26aufirst%3DT.%2BM.%26aulast%3DYang%26aufirst%3DP.%2BC.%26atitle%3DP53%2520Controls%2520Cancer%2520Cell%2520Invasion%2520by%2520Inducing%2520the%2520Mdm2-Mediated%2520Degradation%2520of%2520Slug%26jtitle%3DNat.%2520Cell%2520Biol.%26date%3D2009%26volume%3D11%26spage%3D694%26epage%3D704%26doi%3D10.1038%2Fncb1875" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kelley, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sternberg, M. J.</span></span> <span> </span><span class="NLM_article-title">Protein Structure Prediction on the Web: A Case Study Using the Phyre Server</span>. <i>Nat. Protoc.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">363</span>– <span class="NLM_lpage">371</span>, <span class="refDoi"> DOI: 10.1038/nprot.2009.2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=10.1038%2Fnprot.2009.2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=19247286" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=1%3ACAS%3A528%3ADC%252BD1MXivF2itbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2009&pages=363-371&author=L.+A.+Kelleyauthor=M.+J.+Sternberg&title=Protein+Structure+Prediction+on+the+Web%3A+A+Case+Study+Using+the+Phyre+Server&doi=10.1038%2Fnprot.2009.2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Protein structure prediction on the Web: a case study using the Phyre server</span></div><div class="casAuthors">Kelley, Lawrence A.; Sternberg, Michael J. E.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Protocols</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">363-371</span>CODEN:
                <span class="NLM_cas:coden">NPARDW</span>;
        ISSN:<span class="NLM_cas:issn">1750-2799</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Detg. the structure and function of a novel protein is a cornerstone of many aspects of modern biol.  Over the past decades, a no. of computational tools for structure prediction have been developed.  It is crit. that the biol. community is aware of such tools and is able to interpret their results in an informed way.  This protocol provides a guide to interpreting the output of structure prediction servers in general and one such tool in particular, the protein homol./analogy recognition engine (Phyre).  New profile-profile matching algorithms have improved structure prediction considerably in recent years.  Although the performance of Phyre is typical of many structure prediction systems using such algorithms, all these systems can reliably detect up to twice as many remote homologies as std. sequence-profile searching.  Phyre is widely used by the biol. community, with >150 submissions per day, and provides a simple interface to results.  Phyre takes 30 min to predict the structure of a 250-residue protein.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqkM_7GFRXSorVg90H21EOLACvtfcHk0lgy7HADpO7c1w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXivF2itbs%253D&md5=6613ba0bebfbba8594b5ceb7d9345bca</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1038%2Fnprot.2009.2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnprot.2009.2%26sid%3Dliteratum%253Aachs%26aulast%3DKelley%26aufirst%3DL.%2BA.%26aulast%3DSternberg%26aufirst%3DM.%2BJ.%26atitle%3DProtein%2520Structure%2520Prediction%2520on%2520the%2520Web%253A%2520A%2520Case%2520Study%2520Using%2520the%2520Phyre%2520Server%26jtitle%3DNat.%2520Protoc.%26date%3D2009%26volume%3D4%26spage%3D363%26epage%3D371%26doi%3D10.1038%2Fnprot.2009.2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Eswar, N.</span>; <span class="NLM_string-name">Webb, B.</span>; <span class="NLM_string-name">Marti-Renom, M. A.</span>; <span class="NLM_string-name">Madhusudhan, M. S.</span>; <span class="NLM_string-name">Eramian, D.</span>; <span class="NLM_string-name">Shen, M. Y.</span>; <span class="NLM_string-name">Pieper, U.</span>; <span class="NLM_string-name">Sali, A.</span></span> <span> </span><span class="NLM_article-title">Comparative Protein Structure Modeling Using Modeller</span>.  <i>Current Protocols in Protein Science</i>; <span class="NLM_publisher-name">Wiley</span>, <span class="NLM_year">2007</span>; Chapter 2, Unit 2.9, <span class="refDoi"> DOI: 10.1002/0471140864.ps0209s50</span> .</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=10.1002%2F0471140864.ps0209s50" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2007&author=N.+Eswar&author=B.+Webb&author=M.+A.+Marti-Renom&author=M.+S.+Madhusudhan&author=D.+Eramian&author=M.+Y.+Shen&author=U.+Pieper&author=A.+Sali&title=Current+Protocols+in+Protein+Science"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1002%2F0471140864.ps0209s50&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1002%252F0471140864.ps0209s50%26sid%3Dliteratum%253Aachs%26aulast%3DEswar%26aufirst%3DN.%26atitle%3DComparative%2520Protein%2520Structure%2520Modeling%2520Using%2520Modeller%26btitle%3DCurrent%2520Protocols%2520in%2520Protein%2520Science%26pub%3DWiley%26date%3D2007%26doi%3D10.1002%2F0471140864.ps0209s50" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pronk, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pall, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schulz, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Larsson, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bjelkmar, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Apostolov, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shirts, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kasson, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van der
Spoel, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hess, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindahl, E.</span></span> <span> </span><span class="NLM_article-title">Gromacs 4.5: A High-Throughput and Highly Parallel Open Source Molecular Simulation Toolkit</span>. <i>Bioinformatics</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">845</span>– <span class="NLM_lpage">854</span>, <span class="refDoi"> DOI: 10.1093/bioinformatics/btt055</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=10.1093%2Fbioinformatics%2Fbtt055" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=23407358" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=1%3ACAS%3A528%3ADC%252BC3sXksFWmsrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2013&pages=845-854&author=S.+Pronkauthor=S.+Pallauthor=R.+Schulzauthor=P.+Larssonauthor=P.+Bjelkmarauthor=R.+Apostolovauthor=M.+R.+Shirtsauthor=J.+C.+Smithauthor=P.+M.+Kassonauthor=D.+van+der%0ASpoelauthor=B.+Hessauthor=E.+Lindahl&title=Gromacs+4.5%3A+A+High-Throughput+and+Highly+Parallel+Open+Source+Molecular+Simulation+Toolkit&doi=10.1093%2Fbioinformatics%2Fbtt055"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">GROMACS 4.5: a high-throughput and highly parallel open source molecular simulation toolkit</span></div><div class="casAuthors">Pronk, Sander; Pall, Szilard; Schulz, Roland; Larsson, Per; Bjelkmar, Paer; Apostolov, Rossen; Shirts, Michael R.; Smith, Jeremy C.; Kasson, Peter M.; van der Spoel, David; Hess, Berk; Lindahl, Erik</div><div class="citationInfo"><span class="NLM_cas:title">Bioinformatics</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">845-854</span>CODEN:
                <span class="NLM_cas:coden">BOINFP</span>;
        ISSN:<span class="NLM_cas:issn">1367-4803</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Motivation: Mol. simulation has historically been a low-throughput technique, but faster computers and increasing amts. of genomic and structural data are changing this by enabling large-scale automated simulation of, for instance, many conformers or mutants of biomols. with or without a range of ligands.  At the same time, advances in performance and scaling now make it possible to model complex biomol. interaction and function in a manner directly testable by expt.  These applications share a need for fast and efficient software that can be deployed on massive scale in clusters, web servers, distributed computing or cloud resources.  Results: Here, we present a range of new simulation algorithms and features developed during the past 4 years, leading up to the GROMACS 4.5 software package.  The software now automatically handles wide classes of biomols., such as proteins, nucleic acids and lipids, and comes with all commonly used force fields for these mols. built-in.  GROMACS supports several implicit solvent models, as well as new free-energy algorithms, and the software now uses multithreading for efficient parallelization even on low-end systems, including windows-based workstations.  Together with hand-tuned assembly kernels and state-of-the-art parallelization, this provides extremely high performance and cost efficiency for high-throughput as well as massively parallel simulations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr71lQgYMm017Vg90H21EOLACvtfcHk0lgm1amt9lX_FA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXksFWmsrg%253D&md5=4b25fd6ab4e33725ae56b5da63f4ad68</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1093%2Fbioinformatics%2Fbtt055&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fbioinformatics%252Fbtt055%26sid%3Dliteratum%253Aachs%26aulast%3DPronk%26aufirst%3DS.%26aulast%3DPall%26aufirst%3DS.%26aulast%3DSchulz%26aufirst%3DR.%26aulast%3DLarsson%26aufirst%3DP.%26aulast%3DBjelkmar%26aufirst%3DP.%26aulast%3DApostolov%26aufirst%3DR.%26aulast%3DShirts%26aufirst%3DM.%2BR.%26aulast%3DSmith%26aufirst%3DJ.%2BC.%26aulast%3DKasson%26aufirst%3DP.%2BM.%26aulast%3Dvan%2Bder%2BSpoel%26aufirst%3DD.%26aulast%3DHess%26aufirst%3DB.%26aulast%3DLindahl%26aufirst%3DE.%26atitle%3DGromacs%25204.5%253A%2520A%2520High-Throughput%2520and%2520Highly%2520Parallel%2520Open%2520Source%2520Molecular%2520Simulation%2520Toolkit%26jtitle%3DBioinformatics%26date%3D2013%26volume%3D29%26spage%3D845%26epage%3D854%26doi%3D10.1093%2Fbioinformatics%2Fbtt055" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wiederstein, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sippl, M. J.</span></span> <span> </span><span class="NLM_article-title">Prosa-Web: Interactive Web Service for the Recognition of Errors in Three-Dimensional Structures of Proteins</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>35</i></span>,  <span class="NLM_fpage">W407</span>– <span class="NLM_lpage">410</span>, <span class="refDoi"> DOI: 10.1093/nar/gkm290</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=10.1093%2Fnar%2Fgkm290" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=17517781" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=1%3ACAS%3A280%3ADC%252BD2svhsVKjug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2007&pages=W407-410&author=M.+Wiedersteinauthor=M.+J.+Sippl&title=Prosa-Web%3A+Interactive+Web+Service+for+the+Recognition+of+Errors+in+Three-Dimensional+Structures+of+Proteins&doi=10.1093%2Fnar%2Fgkm290"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">ProSA-web: interactive web service for the recognition of errors in three-dimensional structures of proteins</span></div><div class="casAuthors">Wiederstein Markus; Sippl Manfred J</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic acids research</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">Web Server issue</span>),
    <span class="NLM_cas:pages">W407-10</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">A major problem in structural biology is the recognition of errors in experimental and theoretical models of protein structures.  The ProSA program (Protein Structure Analysis) is an established tool which has a large user base and is frequently employed in the refinement and validation of experimental protein structures and in structure prediction and modeling.  The analysis of protein structures is generally a difficult and cumbersome exercise.  The new service presented here is a straightforward and easy to use extension of the classic ProSA program which exploits the advantages of interactive web-based applications for the display of scores and energy plots that highlight potential problems spotted in protein structures.  In particular, the quality scores of a protein are displayed in the context of all known protein structures and problematic parts of a structure are shown and highlighted in a 3D molecule viewer.  The service specifically addresses the needs encountered in the validation of protein structures obtained from X-ray analysis, NMR spectroscopy and theoretical calculations.  ProSA-web is accessible at https://prosa.services.came.sbg.ac.at.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSKn5zO1OOtqL5oR8RchjrrfW6udTcc2eZVEfkCj88UY7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2svhsVKjug%253D%253D&md5=e2aba9fe44147a9048f37c77306edea6</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkm290&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkm290%26sid%3Dliteratum%253Aachs%26aulast%3DWiederstein%26aufirst%3DM.%26aulast%3DSippl%26aufirst%3DM.%2BJ.%26atitle%3DProsa-Web%253A%2520Interactive%2520Web%2520Service%2520for%2520the%2520Recognition%2520of%2520Errors%2520in%2520Three-Dimensional%2520Structures%2520of%2520Proteins%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2007%26volume%3D35%26spage%3DW407%26epage%3D410%26doi%3D10.1093%2Fnar%2Fgkm290" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Laskowski, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MacArthur, M. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moss, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thornton, J. M.</span></span> <span> </span><span class="NLM_article-title">Procheck: A Program to Check the Stereochemical Quality of Protein Structures</span>. <i>J. Appl. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">1993</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">283</span>– <span class="NLM_lpage">291</span>, <span class="refDoi"> DOI: 10.1107/S0021889892009944</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=10.1107%2FS0021889892009944" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=1%3ACAS%3A528%3ADyaK3sXit12lurY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=1993&pages=283-291&author=R.+A.+Laskowskiauthor=M.+W.+MacArthurauthor=D.+S.+Mossauthor=J.+M.+Thornton&title=Procheck%3A+A+Program+to+Check+the+Stereochemical+Quality+of+Protein+Structures&doi=10.1107%2FS0021889892009944"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">PROCHECK: a program to check the stereochemical quality of protein structures</span></div><div class="casAuthors">Laskowski, Roman A.; MacArthur, Malcolm W.; Moss, David S.; Thornton, Janet M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">283-91</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    </div><div class="casAbstract">The PROCHECK suite of programs provides a detailed check on the stereochem. of a protein structure.  Its outputs comprise a no. of plots in PostScript format and a comprehensive residue-by-residue listing.  These give an assessment of the overall quality of the structure as compared with well-refined structures of the same resoln. and also highlight regions that may need further investigation.  The PROCHECK programs are useful for assessing the quality not only of protein structures in the process of being solved but also of existing structures and of those being modeled on known structures.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrCe5FgJf4KH7Vg90H21EOLACvtfcHk0lgm1amt9lX_FA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3sXit12lurY%253D&md5=fb69fc4410cd716aaaa7cc0db06b3ed2</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1107%2FS0021889892009944&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0021889892009944%26sid%3Dliteratum%253Aachs%26aulast%3DLaskowski%26aufirst%3DR.%2BA.%26aulast%3DMacArthur%26aufirst%3DM.%2BW.%26aulast%3DMoss%26aufirst%3DD.%2BS.%26aulast%3DThornton%26aufirst%3DJ.%2BM.%26atitle%3DProcheck%253A%2520A%2520Program%2520to%2520Check%2520the%2520Stereochemical%2520Quality%2520of%2520Protein%2520Structures%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D1993%26volume%3D26%26spage%3D283%26epage%3D291%26doi%3D10.1107%2FS0021889892009944" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suzek, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shoemaker, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gindulyte, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bryant, S. H.</span></span> <span> </span><span class="NLM_article-title">Pubchem Bioassay: 2014 Update</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>42</i></span>,  <span class="NLM_fpage">D1075</span>– <span class="NLM_lpage">1082</span>, <span class="refDoi"> DOI: 10.1093/nar/gkt978</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=10.1093%2Fnar%2Fgkt978" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=24198245" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=1%3ACAS%3A528%3ADC%252BC2cXoslaq" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2014&pages=D1075-1082&author=Y.+Wangauthor=T.+Suzekauthor=J.+Zhangauthor=J.+Wangauthor=S.+Heauthor=T.+Chengauthor=B.+A.+Shoemakerauthor=A.+Gindulyteauthor=S.+H.+Bryant&title=Pubchem+Bioassay%3A+2014+Update&doi=10.1093%2Fnar%2Fgkt978"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">PubChem BioAssay: 2014 update</span></div><div class="casAuthors">Wang, Yanli; Suzek, Tugba; Zhang, Jian; Wang, Jiyao; He, Siqian; Cheng, Tiejun; Shoemaker, Benjamin A.; Gindulyte, Asta; Bryant, Stephen H.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">D1</span>),
    <span class="NLM_cas:pages">D1075-D1082</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">PubChem's BioAssay database (http://pubchem.ncbi.nlm.nih.gov) is a public repository for archiving biol. tests of small mols. generated through high-throughput screening expts., medicinal chem. studies, chem. biol. research and drug discovery programs.  In addn., the BioAssay database contains data from high-throughput RNA interference screening aimed at identifying crit. genes responsible for a biol. process or disease condition.  The mission of PubChem is to serve the community by providing free and easy access to all deposited data.  To this end, PubChem BioAssay is integrated into the National Center for Biotechnol. Information retrieval system, making them searchable by Entrez queries and cross-linked to other biomedical information archived at National Center for Biotechnol. Information.  Moreover, PubChem BioAssay provides web-based and programmatic tools allowing users to search, access and analyze bioassay test results and metadata.  In this work, we provide an update for the PubChem BioAssay resource, such as information content growth, new developments supporting data integration and search, and the recently deployed PubChem Upload to streamline chem. structure and bioassay submissions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo1tnO28g86-rVg90H21EOLACvtfcHk0ljjuVyAiMTDeg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXoslaq&md5=66caa053197bcaf40b49535b0a3e0c27</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkt978&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkt978%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DY.%26aulast%3DSuzek%26aufirst%3DT.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DHe%26aufirst%3DS.%26aulast%3DCheng%26aufirst%3DT.%26aulast%3DShoemaker%26aufirst%3DB.%2BA.%26aulast%3DGindulyte%26aufirst%3DA.%26aulast%3DBryant%26aufirst%3DS.%2BH.%26atitle%3DPubchem%2520Bioassay%253A%25202014%2520Update%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2014%26volume%3D42%26spage%3DD1075%26epage%3D1082%26doi%3D10.1093%2Fnar%2Fgkt978" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Trott, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olson, A. J.</span></span> <span> </span><span class="NLM_article-title">Autodock Vina: Improving the Speed and Accuracy of Docking with a New Scoring Function, Efficient Optimization, and Multithreading</span>. <i>J. Comput. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>31</i></span>,  <span class="NLM_fpage">455</span>– <span class="NLM_lpage">461</span>, <span class="refDoi"> DOI: 10.1002/jcc.21334</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=10.1002%2Fjcc.21334" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=19499576" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsFGnur3O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2010&pages=455-461&author=O.+Trottauthor=A.+J.+Olson&title=Autodock+Vina%3A+Improving+the+Speed+and+Accuracy+of+Docking+with+a+New+Scoring+Function%2C+Efficient+Optimization%2C+and+Multithreading&doi=10.1002%2Fjcc.21334"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">AutoDock Vina: Improving the speed and accuracy of docking with a new scoring function, efficient optimization, and multithreading</span></div><div class="casAuthors">Trott, Oleg; Olson, Arthur J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">455-461</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">AutoDock Vina, a new program for mol. docking and virtual screening, is presented.  AutoDock Vina achieves an approx. 2 orders of magnitude speed-up compared with the mol. docking software previously developed in the authors' lab (AutoDock 4), while also significantly improving the accuracy of the binding mode predictions, judging by the authors' tests on the training set used in AutoDock 4 development.  Further speed-up is achieved from parallelism, by using multithreading on multicore machines.  AutoDock Vina automatically calcs. the grid maps and clusters the results in a way transparent to the user.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpLecOIkUxIlrVg90H21EOLACvtfcHk0ljjuVyAiMTDeg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsFGnur3O&md5=c6974af8a1235f7aa09918d3e6f70dc4</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1002%2Fjcc.21334&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.21334%26sid%3Dliteratum%253Aachs%26aulast%3DTrott%26aufirst%3DO.%26aulast%3DOlson%26aufirst%3DA.%2BJ.%26atitle%3DAutodock%2520Vina%253A%2520Improving%2520the%2520Speed%2520and%2520Accuracy%2520of%2520Docking%2520with%2520a%2520New%2520Scoring%2520Function%252C%2520Efficient%2520Optimization%252C%2520and%2520Multithreading%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2010%26volume%3D31%26spage%3D455%26epage%3D461%26doi%3D10.1002%2Fjcc.21334" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hsu, K. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, J. M.</span></span> <span> </span><span class="NLM_article-title">iGEMDOCK: A Graphical Environment of Enhancing GEMDOCK Using Pharmacological Interactions and Post-Screening Analysis</span>. <i>BMC Bioinf.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>12</i></span> (<span class="NLM_issue">Suppl. 1</span>),  <span class="NLM_fpage">S33</span>, <span class="refDoi"> DOI: 10.1186/1471-2105-12-S1-S33</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=10.1186%2F1471-2105-12-S1-S33" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=21342564" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2011&pages=S33&issue=Suppl.+1&author=K.+C.+Hsuauthor=Y.+F.+Chenauthor=S.+R.+Linauthor=J.+M.+Yang&title=iGEMDOCK%3A+A+Graphical+Environment+of+Enhancing+GEMDOCK+Using+Pharmacological+Interactions+and+Post-Screening+Analysis&doi=10.1186%2F1471-2105-12-S1-S33"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1186%2F1471-2105-12-S1-S33&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1471-2105-12-S1-S33%26sid%3Dliteratum%253Aachs%26aulast%3DHsu%26aufirst%3DK.%2BC.%26aulast%3DChen%26aufirst%3DY.%2BF.%26aulast%3DLin%26aufirst%3DS.%2BR.%26aulast%3DYang%26aufirst%3DJ.%2BM.%26atitle%3DiGEMDOCK%253A%2520A%2520Graphical%2520Environment%2520of%2520Enhancing%2520GEMDOCK%2520Using%2520Pharmacological%2520Interactions%2520and%2520Post-Screening%2520Analysis%26jtitle%3DBMC%2520Bioinf.%26date%3D2011%26volume%3D12%26issue%3DSuppl.%25201%26spage%3DS33%26doi%3D10.1186%2F1471-2105-12-S1-S33" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">DeLano, W. L.</span></span> <i>Pymol Molecular Graphics System</i>, version <span class="NLM_edition">1.5.0.4</span>; <span class="NLM_publisher-name">Schrödinger, LLC.</span>, <span class="NLM_year">2004</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2004&author=W.+L.+DeLano&title=Pymol+Molecular+Graphics+System"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26aulast%3DDeLano%26aufirst%3DW.%2BL.%26jtitle%3DPymol%2520Molecular%2520Graphics%2520System%26pub%3DSchr%25C3%25B6dinger%252C%2520LLC%26date%3D2004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 5 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Pankaj  Sharma</span>, <span class="hlFld-ContribAuthor ">Chris  LaRosa</span>, <span class="hlFld-ContribAuthor ">Janet  Antwi</span>, <span class="hlFld-ContribAuthor ">Rajgopal  Govindarajan</span>, <span class="hlFld-ContribAuthor ">Karl A.  Werbovetz</span>. </span><span class="cited-content_cbyCitation_article-title">Imidazoles as Potential Anticancer Agents: An Update on Recent Studies. </span><span class="cited-content_cbyCitation_journal-name">Molecules</span><span> <strong>2021,</strong> <em>26 </em>
                                    (14)
                                     , 4213. <a href="https://doi.org/10.3390/molecules26144213" title="DOI URL">https://doi.org/10.3390/molecules26144213</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/molecules26144213&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fmolecules26144213%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecules%26atitle%3DImidazoles%252Bas%252BPotential%252BAnticancer%252BAgents%25253A%252BAn%252BUpdate%252Bon%252BRecent%252BStudies%26aulast%3DSharma%26aufirst%3DPankaj%26date%3D2021%26date%3D2021%26volume%3D26%26issue%3D14%26spage%3D4213" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Hao  Yang</span>, <span class="hlFld-ContribAuthor ">Xiao-Qing  Zhu</span>, <span class="hlFld-ContribAuthor ">Wenjie  Wang</span>, <span class="hlFld-ContribAuthor ">Yu  Chen</span>, <span class="hlFld-ContribAuthor ">Zhu  Hu</span>, <span class="hlFld-ContribAuthor ">Yu  Zhang</span>, <span class="hlFld-ContribAuthor ">De-Xuan  Hu</span>, <span class="hlFld-ContribAuthor ">Le-Mao  Yu</span>, <span class="hlFld-ContribAuthor ">Keli  Agama</span>, <span class="hlFld-ContribAuthor ">Yves  Pommier</span>, <span class="hlFld-ContribAuthor ">Lin-Kun  An</span>. </span><span class="cited-content_cbyCitation_article-title">The synthesis of furoquinolinedione and isoxazoloquinolinedione derivatives as selective Tyrosyl-DNA phosphodiesterase 2 (TDP2) inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic Chemistry</span><span> <strong>2021,</strong> <em>111 </em>, 104881. <a href="https://doi.org/10.1016/j.bioorg.2021.104881" title="DOI URL">https://doi.org/10.1016/j.bioorg.2021.104881</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bioorg.2021.104881&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bioorg.2021.104881%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520Chemistry%26atitle%3DThe%252Bsynthesis%252Bof%252Bfuroquinolinedione%252Band%252Bisoxazoloquinolinedione%252Bderivatives%252Bas%252Bselective%252BTyrosyl-DNA%252Bphosphodiesterase%252B2%252B%252528TDP2%252529%252Binhibitors%26aulast%3DYang%26aufirst%3DHao%26date%3D2021%26volume%3D111%26spage%3D104881" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Wai-Kok  Choong</span>, <span class="hlFld-ContribAuthor ">Ting-Yi  Sung</span>. </span><span class="cited-content_cbyCitation_article-title">Somatic mutation subtypes of lung adenocarcinoma in East Asian reveal divergent biological characteristics and therapeutic vulnerabilities. </span><span class="cited-content_cbyCitation_journal-name">iScience</span><span> <strong>2021,</strong> <em>24 </em>
                                    (6)
                                     , 102522. <a href="https://doi.org/10.1016/j.isci.2021.102522" title="DOI URL">https://doi.org/10.1016/j.isci.2021.102522</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.isci.2021.102522&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.isci.2021.102522%26sid%3Dliteratum%253Aachs%26jtitle%3DiScience%26atitle%3DSomatic%252Bmutation%252Bsubtypes%252Bof%252Blung%252Badenocarcinoma%252Bin%252BEast%252BAsian%252Breveal%252Bdivergent%252Bbiological%252Bcharacteristics%252Band%252Btherapeutic%252Bvulnerabilities%26aulast%3DChoong%26aufirst%3DWai-Kok%26date%3D2021%26volume%3D24%26issue%3D6%26spage%3D102522" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Shunsuke  Misono</span>, <span class="hlFld-ContribAuthor ">Keiko  Mizuno</span>, <span class="hlFld-ContribAuthor ">Takayuki  Suetsugu</span>, <span class="hlFld-ContribAuthor ">Kengo  Tanigawa</span>, <span class="hlFld-ContribAuthor ">Nijiro  Nohata</span>, <span class="hlFld-ContribAuthor ">Akifumi  Uchida</span>, <span class="hlFld-ContribAuthor ">Hiroki  Sanada</span>, <span class="hlFld-ContribAuthor ">Reona  Okada</span>, <span class="hlFld-ContribAuthor ">Shogo  Moriya</span>, <span class="hlFld-ContribAuthor ">Hiromasa  Inoue</span>, <span class="hlFld-ContribAuthor ">Naohiko  Seki</span>. </span><span class="cited-content_cbyCitation_article-title">Molecular Signature of Small Cell Lung Cancer after Treatment Failure: The MCM Complex as Therapeutic Target. </span><span class="cited-content_cbyCitation_journal-name">Cancers</span><span> <strong>2021,</strong> <em>13 </em>
                                    (6)
                                     , 1187. <a href="https://doi.org/10.3390/cancers13061187" title="DOI URL">https://doi.org/10.3390/cancers13061187</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/cancers13061187&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fcancers13061187%26sid%3Dliteratum%253Aachs%26jtitle%3DCancers%26atitle%3DMolecular%252BSignature%252Bof%252BSmall%252BCell%252BLung%252BCancer%252Bafter%252BTreatment%252BFailure%25253A%252BThe%252BMCM%252BComplex%252Bas%252BTherapeutic%252BTarget%26aulast%3DMisono%26aufirst%3DShunsuke%26date%3D2021%26date%3D2021%26volume%3D13%26issue%3D6%26spage%3D1187" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ting  Cao</span>, <span class="hlFld-ContribAuthor ">Shi-jie  Yi</span>, <span class="hlFld-ContribAuthor ">Li-xin  Wang</span>, <span class="hlFld-ContribAuthor ">Juan-xia  Zhao</span>, <span class="hlFld-ContribAuthor ">Jiao  Xiao</span>, <span class="hlFld-ContribAuthor ">Ni  Xie</span>, <span class="hlFld-ContribAuthor ">Zhi  Zeng</span>, <span class="hlFld-ContribAuthor ">Qi  Han</span>, <span class="hlFld-ContribAuthor ">Hai-ou  Tang</span>, <span class="hlFld-ContribAuthor ">Yu-kun  Li</span>, <span class="hlFld-ContribAuthor ">Juan  Zou</span>, <span class="hlFld-ContribAuthor ">Qing  Wu</span>. </span><span class="cited-content_cbyCitation_article-title">Identification of the DNA Replication Regulator MCM Complex Expression and Prognostic Significance in Hepatic Carcinoma. </span><span class="cited-content_cbyCitation_journal-name">BioMed Research International</span><span> <strong>2020,</strong> <em>2020 </em>, 1-18. <a href="https://doi.org/10.1155/2020/3574261" title="DOI URL">https://doi.org/10.1155/2020/3574261</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1155/2020/3574261&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1155%2F2020%2F3574261%26sid%3Dliteratum%253Aachs%26jtitle%3DBioMed%2520Research%2520International%26atitle%3DIdentification%252Bof%252Bthe%252BDNA%252BReplication%252BRegulator%252BMCM%252BComplex%252BExpression%252Band%252BPrognostic%252BSignificance%252Bin%252BHepatic%252BCarcinoma%26aulast%3DCao%26aufirst%3DTing%26date%3D2020%26volume%3D2020%26spage%3D1%26epage%3D18" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b01783/20200323/images/medium/jm9b01783_0012.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b01783/20200323/images/large/jm9b01783_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01783&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b01783/20200323/images/medium/jm9b01783_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b01783/20200323/images/large/jm9b01783_0001.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. AS4583 selected from high-throughput screening affects cell proliferation and inhibits tumor growth <i>in vivo</i>. (A) Structure of AS4583. (B) Cytotoxicity assay for AS4583 with various normal and tumor cell lines. Cells were treated with 0–10 μM AS4583 for 72 h, and survival was assessed with the sulforhodamine B assay. (C) Tumor sizes and (D) TUNEL assay results from tumor xenograft models (<i>n</i> = 6 per group). H1975 cells (3 × 10<sup>6</sup>) were subcutaneously injected into male nu/nu mice. Tumor size was measured every 3–4 days. On day 27, the mice were sacrificed to examine tumor size as well as to confirm histology (original magnification 400×) and apoptosis status by hematoxylin and eosin (H&E) and TUNEL staining. (E) Bar graph showing percent tumor growth inhibition (% TGI) on the day of the final measurement for the control group compared across all treatments in the <i>in vivo</i> study.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b01783/20200323/images/large/jm9b01783_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01783&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b01783/20200323/images/medium/jm9b01783_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b01783/20200323/images/large/jm9b01783_0002.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Treatment of cells with AS4583 interrupts DNA replication, causes cell cycle arrest, and induces apoptosis. (A, B) AS4583 inhibits DNA replication in H1975 cells. H1975 cells were pulsed with BrdU and analyzed by flow cytometry. (A) Representative DNA histograms from DMSO- and AS4583-treated H1975 cells over time (three independent experiments). (B) Average percentage of cells in which BrdU was incorporated. (C) AS4583 decreases the number of DNA replication forks. The replication forks were analyzed by immunofluorescence staining for MCM2 (green) and DAPI (blue, for nuclei). Data were quantified by counting clear, green spots using ImageJ software (<i>n</i> = 5 cells per group; original magnification, 1000×). (D) G1/S phase is prolonged in AS4583-treated H1975 cells. A double thymidine block was performed, and the data were collected at the indicated time points. (E) AS4583 induced intrinsic apoptosis pathway. H1975 cells were treated with AS4583 for 24, 48, or 72 h. The apoptotic marker protein expressions were detected by immunoblotting.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b01783/20200323/images/large/jm9b01783_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01783&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b01783/20200323/images/medium/jm9b01783_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b01783/20200323/images/large/jm9b01783_0003.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Correlation between MCM2 level and cell death via lentivirus-based short hairpin RNAs or AS4583 treatment. (A) Cell survival decreases after MCM2 knockdown. The cells were infected with the indicated lentivirus in medium containing polybrene (8 μg/mL). At 24 h postinfection, the cells were transferred to fresh medium and incubated for 48 h, then analyzed by immunoblotting and the sulforhodamine B assay. (B) MCM2 knockdown decreases cell survival in a dose-dependent manner. The cells were infected with the indicated lentivirus in medium containing polybrene (8 μg/mL). At 24 h postinfection, the cells were incubated in fresh medium lacking lentivirus and polybrene for 48 h and analyzed with the MTS assay over an additional 24–120 h. (C) Cellular levels of MCM2 were detected in different lung-cancer cells as assessed with immunoblotting. (D) Quantification of MCM2 levels and IC<sub>50</sub> values for each cell line were used to calculate the correlation between the two cofactors in these different lung-cancer cell lines (IC<sub>50</sub> versus MCM2, <i>R</i><sup>2</sup> = 0.5282).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b01783/20200323/images/large/jm9b01783_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01783&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b01783/20200323/images/medium/jm9b01783_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b01783/20200323/images/large/jm9b01783_0004.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Clinical significance and prognostic value of MCM in NSCLC. Kaplan–Meier analysis of MCM expression and overall survival for 226 human lung adenocarcinoma samples, pathological stage I or II, published as GSE31210. The <i>p</i>-values were obtained with a two-sided log-rank test.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b01783/20200323/images/large/jm9b01783_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01783&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b01783/20200323/images/medium/jm9b01783_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b01783/20200323/images/large/jm9b01783_0005.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. AS4583 induces proteasome-mediated degradation of MCM2 after loss chromatin bound ability. (A) Increased degradation of MCM2/6/7 upon treatment of H1975 cells with AS4583. AS4583-treated cells were harvested at 24, 48, and 72 h, and the levels of proteins of the MCM complex were analyzed with immunoblotting. (B) Dose-dependent degradation of MCM2/6/7 in different lung cancer cell lines. All cells were treated with AS4583 for 24 h and then harvested. The levels of MCM2/6/7 were examined with immunoblotting. (C) The proteasome inhibitor MG132 reverses the effects of AS4583 on the cellular levels of MCM2/6/7. H1975 cells were treated with AS4583 for 48 h, then incubated in the absence or presence of MG132 (10 μM) for an additional 12 h, and then harvested. The levels of MCM2/6/7 were examined with immunoblotting. (D) AS4583 induces ubiquitination of MCM2. The cells were transiently transfected with pcDNA3-Myc-Ub and then incubated in the absence or presence of AS4583 (140 nM) and MG132 (10 μM). After 12 h, the cells were harvested, and MCM2 was precipitated with an MCM2-specific antibody. Ubiquitination of MCM2 was detected with immunoblotting. IgG served as the antibody negative control for the immunoprecipitation (IP) experiment. (E) Treatment with AS4583 decreases the level of the chromatin-bound form of MCM2. A double thymidine block was performed, and the data were collected at the indicated time points. (F) MCM2 protein levels were recovered upon treatment of H1975 cells with MLN4924. Cells were treated with DMSO or AS4583 for 24 h and then co-treated with different dose of MLN4924. After 6 h, the cells were harvested for detection of MCM2.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b01783/20200323/images/large/jm9b01783_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01783&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b01783/20200323/images/medium/jm9b01783_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b01783/20200323/images/large/jm9b01783_0006.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Interaction between AS4583 and MCM2. (A) Structures of mag-beads-control and mag-beads-AS4583. (B) AS4583 binds MCM2. H1975 cell lysates were incubated with magnetic beads-control or magnetic beads-AS4583 overnight, and the bound MCM2 proteins were detected by immunoblotting. (C) Domain organization of MCM2: N–C, N to C linker; α/β, α/β domain of the AAA+ ATPase core structure; α/β–α, linker between subdomains in the ATPase core; α, α domain; WH, winged-helix domain. The domain annotation is based on the structure of <i>Sulfolobus solfataricus</i> (ssoMCM, PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3F9V">3F9V</a>). (D) Model for MCM2 complexed with AS4583. Domains and linkers have the same coloring as depicted in panel C. Helices are depicted as cylinders and β-strands as arrows. The AS4583 structure is shown as a space-filling model (pink, carbon; blue, nitrogen; red, oxygen). (E) Model of AS4583 in the MCM hexamer. Protomers (MCM2–7) are colored differently and labeled. AS4583 is shown as red spheres. (F) AS4583 binding pocket. The domains have the same coloring as in panel C. The residues involved in AS4583 binding are shown as stick structures. The black dashed lines represent polar contacts between amino acid residues and AS4583. (G) Mag-beads-AS4583 pulled down less MCM2 Q341A than MCM2 wild type. Immunoprecipitation (IP) was performed by incubating lysates of cells that overexpressed FLAG, FLAG-MCM2 wild-type, or the Q341A mutant with mag-beads-control or -AS4583.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b01783/20200323/images/large/jm9b01783_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01783&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b01783/20200323/images/medium/jm9b01783_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b01783/20200323/images/large/jm9b01783_0007.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Stereoselective Synthesis of RJ-LC-07-48<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b01783/20200323/images/large/jm9b01783_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01783&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) O<sub>2</sub> atmosphere, room temperature (rt), 50 min, 90%; (b) CHOCO<sub>2</sub>Et, TFA, PBu<sub>3</sub>, THF, 0–64 °C, 3.5 h, 63%; (c) acetyl chloride, Et<sub>3</sub>N, THF, 0 °C to rt, 15 min, 77%; (d) 3 M NaOH, THF/H<sub>2</sub>O (4:1), 0 °C to rt, 5 h, 97%; (e) HBTU, DMAP, 1-(3-aminopropyl)imidazole, DMF, rt, 6 h, 64%.</p></p></figure><figure data-id="sch2" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b01783/20200323/images/medium/jm9b01783_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b01783/20200323/images/large/jm9b01783_0008.jpeg" id="rightTab-gr10" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of AS4583</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b01783/20200323/images/large/jm9b01783_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01783&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch3" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b01783/20200323/images/medium/jm9b01783_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b01783/20200323/images/large/jm9b01783_0009.jpeg" id="rightTab-gr11" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Conjugation of AS4583 Analog onto Magnetic Beads</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b01783/20200323/images/large/jm9b01783_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01783&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i63">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_76647" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_76647" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 39 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Siegel, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jemal, A.</span></span> <span> </span><span class="NLM_article-title">Cancer Statistics, 2019</span>. <i>Ca-Cancer J. Clin.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>69</i></span>,  <span class="NLM_fpage">7</span>– <span class="NLM_lpage">34</span>, <span class="refDoi"> DOI: 10.3322/caac.21551</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=10.3322%2Fcaac.21551" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=30620402" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=1%3ACAS%3A280%3ADC%252BB3cnptFCntA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=69&publication_year=2019&pages=7-34&author=R.+L.+Siegelauthor=K.+D.+Millerauthor=A.+Jemal&title=Cancer+Statistics%2C+2019&doi=10.3322%2Fcaac.21551"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Cancer statistics, 2019</span></div><div class="casAuthors">Siegel Rebecca L; Miller Kimberly D; Jemal Ahmedin</div><div class="citationInfo"><span class="NLM_cas:title">CA: a cancer journal for clinicians</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">69</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">7-34</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Each year, the American Cancer Society estimates the numbers of new cancer cases and deaths that will occur in the United States and compiles the most recent data on cancer incidence, mortality, and survival.  Incidence data, available through 2015, were collected by the Surveillance, Epidemiology, and End Results Program; the National Program of Cancer Registries; and the North American Association of Central Cancer Registries.  Mortality data, available through 2016, were collected by the National Center for Health Statistics.  In 2019, 1,762,450 new cancer cases and 606,880 cancer deaths are projected to occur in the United States.  Over the past decade of data, the cancer incidence rate (2006-2015) was stable in women and declined by approximately 2% per year in men, whereas the cancer death rate (2007-2016) declined annually by 1.4% and 1.8%, respectively.  The overall cancer death rate dropped continuously from 1991 to 2016 by a total of 27%, translating into approximately 2,629,200 fewer cancer deaths than would have been expected if death rates had remained at their peak.  Although the racial gap in cancer mortality is slowly narrowing, socioeconomic inequalities are widening, with the most notable gaps for the most preventable cancers.  For example, compared with the most affluent counties, mortality rates in the poorest counties were 2-fold higher for cervical cancer and 40% higher for male lung and liver cancers during 2012-2016.  Some states are home to both the wealthiest and the poorest counties, suggesting the opportunity for more equitable dissemination of effective cancer prevention, early detection, and treatment strategies.  A broader application of existing cancer control knowledge with an emphasis on disadvantaged groups would undoubtedly accelerate progress against cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRBHJFurk9wJJp6t8nS6Nc3fW6udTcc2ebzkWaU3t0BFbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3cnptFCntA%253D%253D&md5=02ba708e57d7573ebe0649182325b52c</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.3322%2Fcaac.21551&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3322%252Fcaac.21551%26sid%3Dliteratum%253Aachs%26aulast%3DSiegel%26aufirst%3DR.%2BL.%26aulast%3DMiller%26aufirst%3DK.%2BD.%26aulast%3DJemal%26aufirst%3DA.%26atitle%3DCancer%2520Statistics%252C%25202019%26jtitle%3DCa-Cancer%2520J.%2520Clin.%26date%3D2019%26volume%3D69%26spage%3D7%26epage%3D34%26doi%3D10.3322%2Fcaac.21551" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kobayashi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Costa, D. B.</span></span> <span> </span><span class="NLM_article-title">Acquired Resistance to Epidermal Growth Factor Receptor Tyrosine Kinase Inhibitors in Non-Small-Cell Lung Cancers Dependent on the Epidermal Growth Factor Receptor Pathway</span>. <i>Clin. Lung Cancer</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">281</span>– <span class="NLM_lpage">289</span>, <span class="refDoi"> DOI: 10.3816/CLC.2009.n.039</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=10.3816%2FCLC.2009.n.039" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=19632948" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtVSms7bO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2009&pages=281-289&author=K.+S.+Nguyenauthor=S.+Kobayashiauthor=D.+B.+Costa&title=Acquired+Resistance+to+Epidermal+Growth+Factor+Receptor+Tyrosine+Kinase+Inhibitors+in+Non-Small-Cell+Lung+Cancers+Dependent+on+the+Epidermal+Growth+Factor+Receptor+Pathway&doi=10.3816%2FCLC.2009.n.039"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Acquired resistance to epidermal growth factor receptor tyrosine kinase inhibitors in non-small-cell lung cancers dependent on the epidermal growth factor receptor pathway</span></div><div class="casAuthors">Nguyen, Kim-Son H.; Kobayashi, Susumu; Costa, Daniel B.</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Lung Cancer</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">281-289</span>CODEN:
                <span class="NLM_cas:coden">CLCLCA</span>;
        ISSN:<span class="NLM_cas:issn">1525-7304</span>.
    
            (<span class="NLM_cas:orgname">CIG Media Group, LP</span>)
        </div><div class="casAbstract">A review.  Most advanced non-small-cell lung cancers (NSCLCs) with activating epidermal growth factor receptor (EGFR) mutations (exon 19 deletions or L858R) initially respond to the EGFR tyrosine kinase inhibitors (TKIs) gefitinib and erlotinib.  However, over time (median of 6-12 mo), most tumors develop acquired resistance to EGFR TKIs.  Intense research in these NSCLCs has identified two major mechanisms of resistance to gefitinib/erlotinib: secondary resistance mutations and "oncogene kinase switch" systems.  The secondary T790M mutation occurs in 50% of EGFR-mutated patients with TKI resistance, and in vitro, this mutation negates the hypersensitivity of activating EGFR mutations.  Sensitive detection methods have identified a proportion of TKI-naive tumors that carry T790M, and these resistant clones may be selected after exposure to gefitinib or erlotinib.  Other secondary resistance mutations (D761Y, L747S, T854A) seem to be rare.  The amplification of the MET oncogene is present in 20% of TKI-resistant tumors; however, in half of the cases with this "oncogene kinase switch" mechanism the T790M is coexistent.  It is possible that other kinases (such as insulin-like growth factor-1 receptor [IGF-1R]) might also be selected to bypass EGFR pathways in resistant tumors.  The growing preclin. data in EGFR-mutated NSCLCs with acquired resistance to gefitinib or erlotinib has spawned the initiation or conception of clin. trials testing novel EGFR inhibitors that in vitro inhibit T790M (neratinib, XL647, BIBW 2992, and PF-00299804), MET, or IGF-1R inhibitors in combination with EGFR TKIs, and heat shock protein 90 inhibitors.  Ongoing preclin. and clin. research in EGFR-mutated NSCLC has the potential to significantly improve the outcomes of patients with these somatic mutations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrzBC9bsn7YdrVg90H21EOLACvtfcHk0liHUMGQf5dgmA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtVSms7bO&md5=3522508ea4f2761e4dcedea18d914fe7</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.3816%2FCLC.2009.n.039&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3816%252FCLC.2009.n.039%26sid%3Dliteratum%253Aachs%26aulast%3DNguyen%26aufirst%3DK.%2BS.%26aulast%3DKobayashi%26aufirst%3DS.%26aulast%3DCosta%26aufirst%3DD.%2BB.%26atitle%3DAcquired%2520Resistance%2520to%2520Epidermal%2520Growth%2520Factor%2520Receptor%2520Tyrosine%2520Kinase%2520Inhibitors%2520in%2520Non-Small-Cell%2520Lung%2520Cancers%2520Dependent%2520on%2520the%2520Epidermal%2520Growth%2520Factor%2520Receptor%2520Pathway%26jtitle%3DClin.%2520Lung%2520Cancer%26date%3D2009%26volume%3D10%26spage%3D281%26epage%3D289%26doi%3D10.3816%2FCLC.2009.n.039" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neal, J. W.</span></span> <span> </span><span class="NLM_article-title">First-Line Treatment of Egfr-Mutant Non-Small-Cell Lung Cancer: The Role of Erlotinib and Other Tyrosine Kinase Inhibitors</span>. <i>Biologics</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">337</span>– <span class="NLM_lpage">345</span>, <span class="refDoi"> DOI: 10.2147/BTT.S26558</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=10.2147%2FBTT.S26558" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=23055691" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsFCkurnL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2012&pages=337-345&author=K.+S.+Nguyenauthor=J.+W.+Neal&title=First-Line+Treatment+of+Egfr-Mutant+Non-Small-Cell+Lung+Cancer%3A+The+Role+of+Erlotinib+and+Other+Tyrosine+Kinase+Inhibitors&doi=10.2147%2FBTT.S26558"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">First-line treatment of EGFR-mutant non-small-cell lung cancer: the role of erlotinib and other tyrosine kinase inhibitors</span></div><div class="casAuthors">Nguyen, Kim-Son H.; Neal, Joel W.</div><div class="citationInfo"><span class="NLM_cas:title">Biologics: Targets & Therapy</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">337-345</span>CODEN:
                <span class="NLM_cas:coden">BTTICT</span>;
        ISSN:<span class="NLM_cas:issn">1177-5491</span>.
    
            (<span class="NLM_cas:orgname">Dove Medical Press Ltd.</span>)
        </div><div class="casAbstract">A review.  Epidermal growth factor receptor tyrosine kinase inhibitors (EGFR TKIs) were initially established as second- or third-line treatment of advanced non-small-cell lung cancer (NSCLC).  Subsequent studies, including IPASS, OPTIMAL and EURTAC, have demonstrated that these TKIs are effective first-line therapeutic options in patients with tumors harboring activating mutations in the EGFR gene.  The TKIs are better tolerated than conventional chemotherapy, with frequent yet mild side effects such as rash and diarrhea and rarely interstitial lung disease.  Because most patients on TKIs develop resistance due to a variety of mechanisms, the use of TKIs in the acquired-resistance setting and in the setting of earlier-staged cancers is being extensively studied.  Here we review the major trials leading to the established use of EGFR TKIs in NSCLC, followed by discussion of recently completed and ongoing trials using the next-generation EGFR inhibitor afatinib.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr8uje4RXXGS7Vg90H21EOLACvtfcHk0ljvNzn8jbhXyA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsFCkurnL&md5=94eecdd1672d7412f0b318440ae3bb38</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.2147%2FBTT.S26558&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2147%252FBTT.S26558%26sid%3Dliteratum%253Aachs%26aulast%3DNguyen%26aufirst%3DK.%2BS.%26aulast%3DNeal%26aufirst%3DJ.%2BW.%26atitle%3DFirst-Line%2520Treatment%2520of%2520Egfr-Mutant%2520Non-Small-Cell%2520Lung%2520Cancer%253A%2520The%2520Role%2520of%2520Erlotinib%2520and%2520Other%2520Tyrosine%2520Kinase%2520Inhibitors%26jtitle%3DBiologics%26date%3D2012%26volume%3D6%26spage%3D337%26epage%3D345%26doi%3D10.2147%2FBTT.S26558" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Swinney, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anthony, J.</span></span> <span> </span><span class="NLM_article-title">How Were New Medicines Discovered?</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">507</span>– <span class="NLM_lpage">519</span>, <span class="refDoi"> DOI: 10.1038/nrd3480</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=10.1038%2Fnrd3480" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=21701501" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=1%3ACAS%3A528%3ADC%252BC3MXotlCmsbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2011&pages=507-519&author=D.+C.+Swinneyauthor=J.+Anthony&title=How+Were+New+Medicines+Discovered%3F&doi=10.1038%2Fnrd3480"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">How were new medicines discovered?</span></div><div class="casAuthors">Swinney, David C.; Anthony, Jason</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">507-519</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Preclin. strategies that are used to identify potential drug candidates include target-based screening, phenotypic screening, modification of natural substances and biol.-based approaches.  To investigate whether some strategies have been more successful than others in the discovery of new drugs, we analyzed the discovery strategies and the mol. mechanism of action (MMOA) for new mol. entities and new biologics that were approved by the US Food and Drug Administration between 1999 and 2008.  Out of the 259 agents that were approved, 75 were first-in-class drugs with new MMOAs, and out of these, 50 (67%) were small mols. and 25 (33%) were biologics.  The results also show that the contribution of phenotypic screening to the discovery of first-in-class small-mol. drugs exceeded that of target-based approaches - with 28 and 17 of these drugs coming from the two approaches, resp. - in an era in which the major focus was on target-based approaches.  We postulate that a target-centric approach for first-in-class drugs, without consideration of an optimal MMOA, may contribute to the current high attrition rates and low productivity in pharmaceutical research and development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqE8Ah1iNz76bVg90H21EOLACvtfcHk0ljvNzn8jbhXyA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXotlCmsbs%253D&md5=7503620f8fbadb0a829f6ea2029936a2</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1038%2Fnrd3480&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd3480%26sid%3Dliteratum%253Aachs%26aulast%3DSwinney%26aufirst%3DD.%2BC.%26aulast%3DAnthony%26aufirst%3DJ.%26atitle%3DHow%2520Were%2520New%2520Medicines%2520Discovered%253F%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2011%26volume%3D10%26spage%3D507%26epage%3D519%26doi%3D10.1038%2Fnrd3480" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Arteaga, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, D. H.</span></span> <span> </span><span class="NLM_article-title">Tyrosine Kinase Inhibitors-Zd1839 (Iressa)</span>. <i>Curr. Opin. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">491</span>– <span class="NLM_lpage">498</span>, <span class="refDoi"> DOI: 10.1097/00001622-200111000-00012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=10.1097%2F00001622-200111000-00012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=11673690" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=1%3ACAS%3A528%3ADC%252BD3MXos1Oqs7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2001&pages=491-498&author=C.+L.+Arteagaauthor=D.+H.+Johnson&title=Tyrosine+Kinase+Inhibitors-Zd1839+%28Iressa%29&doi=10.1097%2F00001622-200111000-00012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Tyrosine kinase inhibitors-ZD1839 (Iressa)</span></div><div class="casAuthors">Arteaga, Carlos L.; Johnson, David H.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Oncology</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">491-498</span>CODEN:
                <span class="NLM_cas:coden">CUOOE8</span>;
        ISSN:<span class="NLM_cas:issn">1040-8746</span>.
    
            (<span class="NLM_cas:orgname">Lippincott Williams & Wilkins</span>)
        </div><div class="casAbstract">Several epithelial tumors display epidermal growth factor receptor (EGFR) overexpression (with or without EGFR gene amplification) that is often assocd. with increased prodn. of EGFR ligands.  This permits the activation of endogenous tumor EGFR via autocrine mechanisms, resulting in cellular proliferation and tumor growth.  Interruption of receptor signaling with bivalent EGFR antibodies or with small mol. inhibitors of the EGFR tyrosine kinase results in inhibition of tumor cell proliferation or viability in vitro and in vivo.  One small mol. currently undergoing preclin. and clin. investigation is ZD1839 (Iressa), a synthetic anilinoquinazoline capable of inhibiting EGFR tyrosine kinase in vitro.  The early results of clin. trials indicate this drug possesses antitumor activity in certain malignancies of the upper aerodigestive tract.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpUp52ELUA8HLVg90H21EOLACvtfcHk0ljvNzn8jbhXyA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXos1Oqs7k%253D&md5=ed22721cbb3560ac54e438d363f8a59e</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1097%2F00001622-200111000-00012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1097%252F00001622-200111000-00012%26sid%3Dliteratum%253Aachs%26aulast%3DArteaga%26aufirst%3DC.%2BL.%26aulast%3DJohnson%26aufirst%3DD.%2BH.%26atitle%3DTyrosine%2520Kinase%2520Inhibitors-Zd1839%2520%2528Iressa%2529%26jtitle%3DCurr.%2520Opin.%2520Oncol.%26date%3D2001%26volume%3D13%26spage%3D491%26epage%3D498%26doi%3D10.1097%2F00001622-200111000-00012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Moyer, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barbacci, E. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iwata, K. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arnold, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boman, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cunningham, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DiOrio, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doty, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morin, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moyer, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neveu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pollack, V. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pustilnik, L. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reynolds, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sloan, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Theleman, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, P.</span></span> <span> </span><span class="NLM_article-title">Induction of Apoptosis and Cell Cycle Arrest by Cp-358,774, an Inhibitor of Epidermal Growth Factor Receptor Tyrosine Kinase</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">4838</span>– <span class="NLM_lpage">4848</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=9354447" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=1%3ACAS%3A528%3ADyaK2sXnt12nsr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=1997&pages=4838-4848&author=J.+D.+Moyerauthor=E.+G.+Barbacciauthor=K.+K.+Iwataauthor=L.+Arnoldauthor=B.+Bomanauthor=A.+Cunninghamauthor=C.+DiOrioauthor=J.+Dotyauthor=M.+J.+Morinauthor=M.+P.+Moyerauthor=M.+Neveuauthor=V.+A.+Pollackauthor=L.+R.+Pustilnikauthor=M.+M.+Reynoldsauthor=D.+Sloanauthor=A.+Thelemanauthor=P.+Miller&title=Induction+of+Apoptosis+and+Cell+Cycle+Arrest+by+Cp-358%2C774%2C+an+Inhibitor+of+Epidermal+Growth+Factor+Receptor+Tyrosine+Kinase"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Induction of apoptosis and cell cycle arrest by CP-358774, an inhibitor of epidermal growth factor receptor tyrosine kinase</span></div><div class="casAuthors">Moyer, James D.; Barbacci, Elsa G.; Iwata, Kenneth K.; Arnold, Lee; Boman, Bruce; Cunningham, Ann; Diorio, Catherine; Doty, Jonathan; Morin, Michael J.; Moyer, Mikel P.; Neveu, Mark; Pollack, Vincent A.; Pustilnik, Leslie R.; Reynolds, Margaret M.; Sloan, Don; Theleman, April; Miller, Penny</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">4838-4848</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">The epidermal growth factor receptor (EGFR) is overexpressed in a significant percentage of carcinomas and contributes to the malignant phenotype.  CP-358774 is a directly acting inhibitor of human EGFR tyrosine kinase with an IC50 of 2 nM and reduces EGFR autophosphorylation in intact tumor cells with an IC50 of 20 nM.  This inhibition is selective for EGFR tyrosine kinase relative to other tyrosine kinases we have examd., both in assays of isolated kinases and whole cells.  At doses of 100 mg/kg, CP-358774 completely prevents EGF-induced autophosphorylation of EGFR in human HN5 tumors growing as xenografts in athymic mice and of the hepatic EGFR of the treated mice.  CP-358774 inhibits the proliferation of DiFi human colon tumor cells at submicromolar concns. in cell culture and blocks cell cycle progression at the G1 phase.  This inhibitor produces a marked accumulation of retinoblastoma protein in its underphosphorylated form and accumulation of p27KIP1 in DiFi cells, which may contribute to the cell cycle block.  Inhibition of the EGFR also triggers apoptosis in these cells as detd. by formation of DNA fragments and other criteria.  These results indicate that CP-358774 has potential for the treatment of tumors that are dependent on the EGFR pathway for proliferation or survival.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpghArQOnGlkbVg90H21EOLACvtfcHk0ljRS490erzMiw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXnt12nsr4%253D&md5=0a6e9697bf290d8e7c44cbe2a125e0b1</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMoyer%26aufirst%3DJ.%2BD.%26aulast%3DBarbacci%26aufirst%3DE.%2BG.%26aulast%3DIwata%26aufirst%3DK.%2BK.%26aulast%3DArnold%26aufirst%3DL.%26aulast%3DBoman%26aufirst%3DB.%26aulast%3DCunningham%26aufirst%3DA.%26aulast%3DDiOrio%26aufirst%3DC.%26aulast%3DDoty%26aufirst%3DJ.%26aulast%3DMorin%26aufirst%3DM.%2BJ.%26aulast%3DMoyer%26aufirst%3DM.%2BP.%26aulast%3DNeveu%26aufirst%3DM.%26aulast%3DPollack%26aufirst%3DV.%2BA.%26aulast%3DPustilnik%26aufirst%3DL.%2BR.%26aulast%3DReynolds%26aufirst%3DM.%2BM.%26aulast%3DSloan%26aufirst%3DD.%26aulast%3DTheleman%26aufirst%3DA.%26aulast%3DMiller%26aufirst%3DP.%26atitle%3DInduction%2520of%2520Apoptosis%2520and%2520Cell%2520Cycle%2520Arrest%2520by%2520Cp-358%252C774%252C%2520an%2520Inhibitor%2520of%2520Epidermal%2520Growth%2520Factor%2520Receptor%2520Tyrosine%2520Kinase%26jtitle%3DCancer%2520Res.%26date%3D1997%26volume%3D57%26spage%3D4838%26epage%3D4848" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cui, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tran-Dube, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nambu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kung, P. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pairish, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jia, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Funk, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Botrous, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McTigue, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grodsky, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ryan, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Padrique, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alton, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Timofeevski, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamazaki, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zou, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christensen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mroczkowski, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bender, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kania, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edwards, M. P.</span></span> <span> </span><span class="NLM_article-title">Structure Based Drug Design of Crizotinib (Pf-02341066), a Potent and Selective Dual Inhibitor of Mesenchymal-Epithelial Transition Factor (C-Met) Kinase and Anaplastic Lymphoma Kinase (Alk)</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">6342</span>– <span class="NLM_lpage">6363</span>, <span class="refDoi"> DOI: 10.1021/jm2007613</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm2007613" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtVegtLrN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=6342-6363&author=J.+J.+Cuiauthor=M.+Tran-Dubeauthor=H.+Shenauthor=M.+Nambuauthor=P.+P.+Kungauthor=M.+Pairishauthor=L.+Jiaauthor=J.+Mengauthor=L.+Funkauthor=I.+Botrousauthor=M.+McTigueauthor=N.+Grodskyauthor=K.+Ryanauthor=E.+Padriqueauthor=G.+Altonauthor=S.+Timofeevskiauthor=S.+Yamazakiauthor=Q.+Liauthor=H.+Zouauthor=J.+Christensenauthor=B.+Mroczkowskiauthor=S.+Benderauthor=R.+S.+Kaniaauthor=M.+P.+Edwards&title=Structure+Based+Drug+Design+of+Crizotinib+%28Pf-02341066%29%2C+a+Potent+and+Selective+Dual+Inhibitor+of+Mesenchymal-Epithelial+Transition+Factor+%28C-Met%29+Kinase+and+Anaplastic+Lymphoma+Kinase+%28Alk%29&doi=10.1021%2Fjm2007613"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Structure Based Drug Design of Crizotinib (PF-02341066), a Potent and Selective Dual Inhibitor of Mesenchymal-Epithelial Transition Factor (c-MET) Kinase and Anaplastic Lymphoma Kinase (ALK)</span></div><div class="casAuthors">Cui, J. Jean; Tran-Dube, Michelle; Shen, Hong; Nambu, Mitchell; Kung, Pei-Pei; Pairish, Mason; Jia, Lei; Meng, Jerry; Funk, Lee; Botrous, Iriny; McTigue, Michele; Grodsky, Neil; Ryan, Kevin; Padrique, Ellen; Alton, Gordon; Timofeevski, Sergei; Yamazaki, Shinji; Li, Qiuhua; Zou, Helen; Christensen, James; Mroczkowski, Barbara; Bender, Steve; Kania, Robert S.; Edwards, Martin P.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">6342-6363</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Because of the crit. roles of aberrant signaling in cancer, both c-MET and ALK receptor tyrosine kinases are attractive oncol. targets for therapeutic intervention.  The cocrystal structure of 3 (PHA-665752), bound to c-MET kinase domain, revealed a novel ATP site environment, which served as the target to guide parallel, multiattribute drug design.  A novel 2-amino-5-aryl-3-benzyloxypyridine series was created to more effectively make the key interactions achieved with 3.  In the novel series, the 2-aminopyridine core allowed a 3-benzyloxy group to reach into the same pocket as the 2,6-dichlorophenyl group of 3 via a more direct vector and thus with a better ligand efficiency (LE).  Further optimization of the lead series generated the clin. candidate crizotinib (PF-02341066), which demonstrated potent in vitro and in vivo c-MET kinase and ALK inhibition, effective tumor growth inhibition, and good pharmaceutical properties.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoHs_cKJ-U3t7Vg90H21EOLACvtfcHk0ljRS490erzMiw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtVegtLrN&md5=dbab4c1b3e7c76180dbad04be2b6a474</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1021%2Fjm2007613&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm2007613%26sid%3Dliteratum%253Aachs%26aulast%3DCui%26aufirst%3DJ.%2BJ.%26aulast%3DTran-Dube%26aufirst%3DM.%26aulast%3DShen%26aufirst%3DH.%26aulast%3DNambu%26aufirst%3DM.%26aulast%3DKung%26aufirst%3DP.%2BP.%26aulast%3DPairish%26aufirst%3DM.%26aulast%3DJia%26aufirst%3DL.%26aulast%3DMeng%26aufirst%3DJ.%26aulast%3DFunk%26aufirst%3DL.%26aulast%3DBotrous%26aufirst%3DI.%26aulast%3DMcTigue%26aufirst%3DM.%26aulast%3DGrodsky%26aufirst%3DN.%26aulast%3DRyan%26aufirst%3DK.%26aulast%3DPadrique%26aufirst%3DE.%26aulast%3DAlton%26aufirst%3DG.%26aulast%3DTimofeevski%26aufirst%3DS.%26aulast%3DYamazaki%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DQ.%26aulast%3DZou%26aufirst%3DH.%26aulast%3DChristensen%26aufirst%3DJ.%26aulast%3DMroczkowski%26aufirst%3DB.%26aulast%3DBender%26aufirst%3DS.%26aulast%3DKania%26aufirst%3DR.%2BS.%26aulast%3DEdwards%26aufirst%3DM.%2BP.%26atitle%3DStructure%2520Based%2520Drug%2520Design%2520of%2520Crizotinib%2520%2528Pf-02341066%2529%252C%2520a%2520Potent%2520and%2520Selective%2520Dual%2520Inhibitor%2520of%2520Mesenchymal-Epithelial%2520Transition%2520Factor%2520%2528C-Met%2529%2520Kinase%2520and%2520Anaplastic%2520Lymphoma%2520Kinase%2520%2528Alk%2529%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D6342%26epage%3D6363%26doi%3D10.1021%2Fjm2007613" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Buchdunger, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zimmermann, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mett, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meyer, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muller, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Druker, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lydon, N. B.</span></span> <span> </span><span class="NLM_article-title">Inhibition of the Abl Protein-Tyrosine Kinase in Vitro and in Vivo by a 2-Phenylaminopyrimidine Derivative</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">100</span>– <span class="NLM_lpage">104</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=8548747" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=1%3ACAS%3A528%3ADyaK28Xit1Chsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=1996&pages=100-104&author=E.+Buchdungerauthor=J.+Zimmermannauthor=H.+Mettauthor=T.+Meyerauthor=M.+Mullerauthor=B.+J.+Drukerauthor=N.+B.+Lydon&title=Inhibition+of+the+Abl+Protein-Tyrosine+Kinase+in+Vitro+and+in+Vivo+by+a+2-Phenylaminopyrimidine+Derivative"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of the Abl protein-tyrosine kinase in vitro and in vivo by a 2-phenylaminopyrimidine derivative</span></div><div class="casAuthors">Buchdunger, Elisabeth; Zimmermann, Juerg; Mett, Helmut; Meyer, Thomas; Mueller, Marcel; Druker, Brian J.; Lydon, Nicholas B.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">100-4</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Oncogenic activation of Abl proteins due to structural modifications can occur as a result of viral transduction or chromosomal translocation.  The tyrosine protein kinase activity of oncogenic Abl proteins is known to be essential for their transforming activity.  Therefore, we have attempted to identify selective inhibitors of the Abl tyrosine protein kinase.  Herein we describe an inhibitor (CGP 57148) of the Abl and platelet-derived growth factor (PDGF) receptor protein-tyrosine kinases from the 2-phenylaminopyrimidine class, which is highly active in vitro and in vivo.  Submicromolar concns. of the compd. inhibited both v-Abl and PDGF receptor autophosphorylation and PDGF-induced c-fos mRNA expression selectively in intact cells.  In contrast, ligand-induced growth factor receptor autophosphorylation in response to epidermal growth factor (EGF), insulin-like growth factor-I, and insulin showed no or weak inhibition by high concns. of CGP 57148.  C-fos mRNA expression induced by EGF, fibroblast growth factor, or phorbol ester was also insensitive to inhibition by CGP 57148.  In antiproliferative assays, the compd. was more than 30-100-fold more potent in inhibiting growth of v-abl-transformed PB-3c cells and v-sis-transformed BALB/c 3T3 cells relative to inhibition of EGF-dependent BaLB/MK cells, interleukin-3-dependent FDC-P1 cells, and the T24 bladder carcinoma line.  Furthermore, anchorage-independent growth of v-abl- and v-sis-transformed BALB/c 3T3 cells was inhibited potently by CGP 57148.  When tested in vivo, CGP 57148 showed antitumor activity at tolerated doses against tumorigenic v-Abl- and v-sis-transformed BALB/c 3T3 cells.  In contrast, CGP 57148 had no antitumor activity when tested using src-transformed BALB/c 3T3 cells.  These findings suggest that CGP 57148 may have therapeutic potential for the treatment of diseases that involve abnormal cellular proliferation induced by Abl protein-tyrosine kinase deregulation or PDGF receptor activation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpwOOoQlhPp6LVg90H21EOLACvtfcHk0lhPONvHbwFuMQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28Xit1Chsg%253D%253D&md5=b311c951f52565f11d39901fe5a8cce6</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBuchdunger%26aufirst%3DE.%26aulast%3DZimmermann%26aufirst%3DJ.%26aulast%3DMett%26aufirst%3DH.%26aulast%3DMeyer%26aufirst%3DT.%26aulast%3DMuller%26aufirst%3DM.%26aulast%3DDruker%26aufirst%3DB.%2BJ.%26aulast%3DLydon%26aufirst%3DN.%2BB.%26atitle%3DInhibition%2520of%2520the%2520Abl%2520Protein-Tyrosine%2520Kinase%2520in%2520Vitro%2520and%2520in%2520Vivo%2520by%2520a%25202-Phenylaminopyrimidine%2520Derivative%26jtitle%3DCancer%2520Res.%26date%3D1996%26volume%3D56%26spage%3D100%26epage%3D104" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hartford, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ratain, M. J.</span></span> <span> </span><span class="NLM_article-title">Rapamycin: Something Old, Something New, Sometimes Borrowed and Now Renewed</span>. <i>Clin. Pharmacol. Ther.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>82</i></span>,  <span class="NLM_fpage">381</span>– <span class="NLM_lpage">388</span>, <span class="refDoi"> DOI: 10.1038/sj.clpt.6100317</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=10.1038%2Fsj.clpt.6100317" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=17728765" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtFGqu7fE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=82&publication_year=2007&pages=381-388&author=C.+M.+Hartfordauthor=M.+J.+Ratain&title=Rapamycin%3A+Something+Old%2C+Something+New%2C+Sometimes+Borrowed+and+Now+Renewed&doi=10.1038%2Fsj.clpt.6100317"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Rapamycin: something old, something new, sometimes borrowed and now renewed</span></div><div class="casAuthors">Hartford, C. M.; Ratain, M. J.</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Pharmacology & Therapeutics (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">82</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">381-388</span>CODEN:
                <span class="NLM_cas:coden">CLPTAT</span>;
        ISSN:<span class="NLM_cas:issn">0009-9236</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The mol. target of rapamycin (mTOR) is central to a complex intracellular signaling pathway and is involved in diverse processes including cell growth and proliferation, angiogenesis, autophagy, and metab.  Although sirolimus (rapamycin), the oldest inhibitor of mTOR, was discovered more than 30 years ago, renewed interest in this pathway is evident by the numerous rapalogs recently developed.  These newer agents borrow from the structure of sirolimus and, although there are some pharmacokinetic differences, they appear to differ little in terms of pharmacodynamic effects and overall tolerability.  Given the multitude of potential applications for this class of agents and the decrease in cost that can be expected upon the expiration of sirolimus patents, renewed focus on this agent is warranted.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqbjZFHK2UEtLVg90H21EOLACvtfcHk0lhPONvHbwFuMQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtFGqu7fE&md5=53c9582651f57cec29399b4fd5161bdc</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1038%2Fsj.clpt.6100317&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.clpt.6100317%26sid%3Dliteratum%253Aachs%26aulast%3DHartford%26aufirst%3DC.%2BM.%26aulast%3DRatain%26aufirst%3DM.%2BJ.%26atitle%3DRapamycin%253A%2520Something%2520Old%252C%2520Something%2520New%252C%2520Sometimes%2520Borrowed%2520and%2520Now%2520Renewed%26jtitle%3DClin.%2520Pharmacol.%2520Ther.%26date%3D2007%26volume%3D82%26spage%3D381%26epage%3D388%26doi%3D10.1038%2Fsj.clpt.6100317" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Medina, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amaria, M. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jimeno, A.</span></span> <span> </span><span class="NLM_article-title">Dabrafenib in the Treatment of Advanced Melanoma</span>. <i>Drugs Today</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>49</i></span>,  <span class="NLM_fpage">377</span>– <span class="NLM_lpage">385</span>, <span class="refDoi"> DOI: 10.1358/dot.2013.49.6.1968669</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=10.1358%2Fdot.2013.49.6.1968669" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=1%3ACAS%3A280%3ADC%252BC3sjmtlOnsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2013&pages=377-385&author=T.+Medinaauthor=M.+N.+Amariaauthor=A.+Jimeno&title=Dabrafenib+in+the+Treatment+of+Advanced+Melanoma&doi=10.1358%2Fdot.2013.49.6.1968669"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Dabrafenib in the treatment of advanced melanoma</span></div><div class="casAuthors">Medina T; Amaria M N; Jimeno A</div><div class="citationInfo"><span class="NLM_cas:title">Drugs of today (Barcelona, Spain : 1998)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">377-85</span>
        ISSN:<span class="NLM_cas:issn">1699-3993</span>.
    </div><div class="casAbstract">Advanced melanoma has long been a challenging malignancy to treat due to a relative paucity of efficacious therapeutic options.  However, the identification of activating BRAF mutations in approximately 50% of patients with cutaneous melanoma has ushered in the era of targeted therapy for melanoma patients.  Similar to the first-in-class selective serine/threonine-protein kinase B-raf inhibitor vemurafenib, dabrafenib is highly efficacious in melanoma patients with BRAF V600E mutations, with response rates of approximately 50% and progression-free survival of 6 months.  There is data to suggest that dabrafenib not only shows activity in V600E-mutated melanoma, but also in non-V600E BRAF-mutated disease such as V600K.  There is also early data to suggest that dabrafenib is effective in controlling metastases in the brain.  Combining dabrafenib with the selective mitogen-activated protein kinase kinase (MEK) inhibitor trametinib has been effective in improving both the progression-free survival and overall survival of melanoma patients over those patients treated with dabrafenib alone.  Dabrafenib is still being evaluated in several clinical trials in melanoma as well as a variety of other solid tumors with BRAF mutations.  The U.S.  Food and Drug Administration has recently approved dabrafenib as a single agent for the treatment of unresectable or metastatic melanoma in adult patients with BRAF V600E mutation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRdCKpggTCUg-kdrW1FkXG-fW6udTcc2ebtmNx63KJX6bntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3sjmtlOnsg%253D%253D&md5=3df12b8590957ab17336aec2086a37fb</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1358%2Fdot.2013.49.6.1968669&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1358%252Fdot.2013.49.6.1968669%26sid%3Dliteratum%253Aachs%26aulast%3DMedina%26aufirst%3DT.%26aulast%3DAmaria%26aufirst%3DM.%2BN.%26aulast%3DJimeno%26aufirst%3DA.%26atitle%3DDabrafenib%2520in%2520the%2520Treatment%2520of%2520Advanced%2520Melanoma%26jtitle%3DDrugs%2520Today%26date%3D2013%26volume%3D49%26spage%3D377%26epage%3D385%26doi%3D10.1358%2Fdot.2013.49.6.1968669" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Moffat, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rudolph, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bailey, D.</span></span> <span> </span><span class="NLM_article-title">Phenotypic Screening in Cancer Drug Discovery - Past, Present and Future</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">588</span>– <span class="NLM_lpage">602</span>, <span class="refDoi"> DOI: 10.1038/nrd4366</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=10.1038%2Fnrd4366" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=25033736" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtFygsr7L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2014&pages=588-602&author=J.+G.+Moffatauthor=J.+Rudolphauthor=D.+Bailey&title=Phenotypic+Screening+in+Cancer+Drug+Discovery+-+Past%2C+Present+and+Future&doi=10.1038%2Fnrd4366"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Phenotypic screening in cancer drug discovery - past, present and future</span></div><div class="casAuthors">Moffat, John G.; Rudolph, Joachim; Bailey, David</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">588-602</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  There has been a resurgence of interest in the use of phenotypic screens in drug discovery as an alternative to target-focused approaches.  Given that oncol. is currently the most active therapeutic area, and also one in which target-focused approaches have been particularly prominent in the past two decades, we investigated the contribution of phenotypic assays to oncol. drug discovery by analyzing the origins of all new small-mol. cancer drugs approved by the US Food and Drug Administration (FDA) over the past 15 years and those currently in clin. development.  Although the majority of these drugs originated from target-based discovery, we identified a significant no. whose discovery depended on phenotypic screening approaches.  We postulate that the contribution of phenotypic screening to cancer drug discovery has been hampered by a reliance on 'classical' nonspecific drug effects such as cytotoxicity and mitotic arrest, exacerbated by a paucity of mechanistically defined cellular models for therapeutically translatable cancer phenotypes.  However, tech. and biol. advances that enable such mechanistically informed phenotypic models have the potential to empower phenotypic drug discovery in oncol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoTWtoTIgWqvbVg90H21EOLACvtfcHk0lhAmCmZlC9w2g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtFygsr7L&md5=0e8efed90e5bc4b47f959b83a6ab5b96</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1038%2Fnrd4366&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd4366%26sid%3Dliteratum%253Aachs%26aulast%3DMoffat%26aufirst%3DJ.%2BG.%26aulast%3DRudolph%26aufirst%3DJ.%26aulast%3DBailey%26aufirst%3DD.%26atitle%3DPhenotypic%2520Screening%2520in%2520Cancer%2520Drug%2520Discovery%2520-%2520Past%252C%2520Present%2520and%2520Future%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2014%26volume%3D13%26spage%3D588%26epage%3D602%26doi%3D10.1038%2Fnrd4366" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Marks, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Breslow, R.</span></span> <span> </span><span class="NLM_article-title">Dimethyl Sulfoxide to Vorinostat: Development of This Histone Deacetylase Inhibitor as an Anticancer Drug</span>. <i>Nat. Biotechnol.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">84</span>– <span class="NLM_lpage">90</span>, <span class="refDoi"> DOI: 10.1038/nbt1272</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=10.1038%2Fnbt1272" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=17211407" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=1%3ACAS%3A528%3ADC%252BD2sXis1GrsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2007&pages=84-90&author=P.+A.+Marksauthor=R.+Breslow&title=Dimethyl+Sulfoxide+to+Vorinostat%3A+Development+of+This+Histone+Deacetylase+Inhibitor+as+an+Anticancer+Drug&doi=10.1038%2Fnbt1272"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Dimethyl sulfoxide to vorinostat: development of this histone deacetylase inhibitor as an anticancer drug</span></div><div class="casAuthors">Marks, Paul A.; Breslow, Ronald</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">84-90</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  In our quest to understand why DMSO (DMSO) can cause growth arrest and terminal differentiation of transformed cells, we followed a path that led us to discover suberoylanilide hydroxamic acid (SAHA; vorinostat (Zolinza)), which is a histone deacetylase inhibitor.  SAHA reacts with and blocks the catalytic site of these enzymes.  Extensive structure-activity studies were done along the path from DMSO to SAHA.  SAHA can cause growth arrest and death of a broad variety of transformed cells both in vitro and in tumor-bearing animals at concns. not toxic to normal cells.  SAHA has many protein targets whose structure and function are altered by acetylation, including chromatin-assocd. histones, nonhistone gene transcription factors and proteins involved in regulation of cell proliferation, migration and death.  In clin. trials, SAHA has shown significant anticancer activity against both hematol. and solid tumors at doses well tolerated by patients.  A new drug application was approved by the US Food and Drug Administration for vorinostat for treatment of cutaneous T-cell lymphoma.  More potent analogs of SAHA have shown unacceptable toxicity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoK3endE54rQ7Vg90H21EOLACvtfcHk0lhAmCmZlC9w2g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXis1GrsQ%253D%253D&md5=cc2a39d80d5d5bea0a1377bdf48a957d</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1038%2Fnbt1272&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt1272%26sid%3Dliteratum%253Aachs%26aulast%3DMarks%26aufirst%3DP.%2BA.%26aulast%3DBreslow%26aufirst%3DR.%26atitle%3DDimethyl%2520Sulfoxide%2520to%2520Vorinostat%253A%2520Development%2520of%2520This%2520Histone%2520Deacetylase%2520Inhibitor%2520as%2520an%2520Anticancer%2520Drug%26jtitle%3DNat.%2520Biotechnol.%26date%3D2007%26volume%3D25%26spage%3D84%26epage%3D90%26doi%3D10.1038%2Fnbt1272" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nakajima, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, Y. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terano, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoshida, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Horinouchi, S.</span></span> <span> </span><span class="NLM_article-title">Fr901228, a Potent Antitumor Antibiotic, Is a Novel Histone Deacetylase Inhibitor</span>. <i>Exp. Cell Res.</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>241</i></span>,  <span class="NLM_fpage">126</span>– <span class="NLM_lpage">133</span>, <span class="refDoi"> DOI: 10.1006/excr.1998.4027</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=10.1006%2Fexcr.1998.4027" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=9633520" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=1%3ACAS%3A528%3ADyaK1cXjvVGjtLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=241&publication_year=1998&pages=126-133&author=H.+Nakajimaauthor=Y.+B.+Kimauthor=H.+Teranoauthor=M.+Yoshidaauthor=S.+Horinouchi&title=Fr901228%2C+a+Potent+Antitumor+Antibiotic%2C+Is+a+Novel+Histone+Deacetylase+Inhibitor&doi=10.1006%2Fexcr.1998.4027"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">FR901228, a potent antitumor antibiotic, is a novel histone deacetylase inhibitor</span></div><div class="casAuthors">Nakajima, Hidenori; Kim, Young Bae; Terano, Hiroshi; Yoshida, Minoru; Horinouchi, Sueharu</div><div class="citationInfo"><span class="NLM_cas:title">Experimental Cell Research</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">241</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">126-133</span>CODEN:
                <span class="NLM_cas:coden">ECREAL</span>;
        ISSN:<span class="NLM_cas:issn">0014-4827</span>.
    
            (<span class="NLM_cas:orgname">Academic Press</span>)
        </div><div class="casAbstract">Screening for microbial metabolites that induce transcriptional activation of the SV40 promoter resulted in the identification of two known compds., FR901228 and trichostatin A (TSA).  FR901228 is a potent antitumor drug that is currently under clin. investigation.  TSA is a specific inhibitor of histone deacetylase.  Despite structural differences, both FR901228 and TSA greatly enhanced the transcriptional activity of the SV40 promoter in an enhancer-dependent manner.  The effects of FR901228 on the cell cycle, chromatin structure, and histone acetylation were examd. and compared to the effects of TSA.  Both compds. arrested the cell cycle at both G1 and G2/M phases and induced internucleosomal breakdown of chromatin.  FR901228, like TSA, inhibited the intracellular histone deacetylase activity, as a result of which marked amts. of acetylated histone species accumulated.  Thus, FR901228 is a new type of histone deacetylase inhibitor whose chem. structure is unrelated to known inhibitors such as trichostatins and trapoxins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqRDyUNz818nbVg90H21EOLACvtfcHk0lhAmCmZlC9w2g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXjvVGjtLY%253D&md5=f0e72c561596900036c9062bf2db2990</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1006%2Fexcr.1998.4027&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Fexcr.1998.4027%26sid%3Dliteratum%253Aachs%26aulast%3DNakajima%26aufirst%3DH.%26aulast%3DKim%26aufirst%3DY.%2BB.%26aulast%3DTerano%26aufirst%3DH.%26aulast%3DYoshida%26aufirst%3DM.%26aulast%3DHorinouchi%26aufirst%3DS.%26atitle%3DFr901228%252C%2520a%2520Potent%2520Antitumor%2520Antibiotic%252C%2520Is%2520a%2520Novel%2520Histone%2520Deacetylase%2520Inhibitor%26jtitle%3DExp.%2520Cell%2520Res.%26date%3D1998%26volume%3D241%26spage%3D126%26epage%3D133%26doi%3D10.1006%2Fexcr.1998.4027" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">O’Dwyer, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maslak, P.</span></span> <span> </span><span class="NLM_article-title">Azacitidine and the Beginnings of Therapeutic Epigenetic Modulation</span>. <i>Expert Opin. Pharmacother.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">1981</span>– <span class="NLM_lpage">1986</span>, <span class="refDoi"> DOI: 10.1517/14656566.9.11.1981</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=10.1517%2F14656566.9.11.1981" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=18627335" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=1%3ACAS%3A528%3ADC%252BD1cXos1Chu7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2008&pages=1981-1986&author=K.+O%E2%80%99Dwyerauthor=P.+Maslak&title=Azacitidine+and+the+Beginnings+of+Therapeutic+Epigenetic+Modulation&doi=10.1517%2F14656566.9.11.1981"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Azacitidine and the beginnings of therapeutic epigenetic modulation</span></div><div class="casAuthors">O'Dwyer, Kristen; Maslak, Peter</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Pharmacotherapy</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1981-1986</span>CODEN:
                <span class="NLM_cas:coden">EOPHF7</span>;
        ISSN:<span class="NLM_cas:issn">1465-6566</span>.
    
            (<span class="NLM_cas:orgname">Informa Healthcare</span>)
        </div><div class="casAbstract">A review.  Although originally developed as a cytarabine analog more than 40 years ago, azacitidine has been the subject of renewed interest in the era of cancer epigenetics.  What is the history of the clin. development of azacitidine and how has it been applied successfully to the treatment of myelodysplastic syndromes (MDS)  We review the evolution of the use of azacitidine for the therapy of human disease and review the major studies that have laid the groundwork for its current clin. indication.  The use of azacitidine has changed the approach to the treatment of MDS and has resulted in improved outcomes for patients.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGplzDnVBQUa-LVg90H21EOLACvtfcHk0lhz9rQijfQwUQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXos1Chu7w%253D&md5=f421e7829f1b34e7086b0c11319d98a3</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1517%2F14656566.9.11.1981&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1517%252F14656566.9.11.1981%26sid%3Dliteratum%253Aachs%26aulast%3DO%25E2%2580%2599Dwyer%26aufirst%3DK.%26aulast%3DMaslak%26aufirst%3DP.%26atitle%3DAzacitidine%2520and%2520the%2520Beginnings%2520of%2520Therapeutic%2520Epigenetic%2520Modulation%26jtitle%3DExpert%2520Opin.%2520Pharmacother.%26date%3D2008%26volume%3D9%26spage%3D1981%26epage%3D1986%26doi%3D10.1517%2F14656566.9.11.1981" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kotla, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goel, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nischal, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heuck, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vivek, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Das, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verma, A.</span></span> <span> </span><span class="NLM_article-title">Mechanism of Action of Lenalidomide in Hematological Malignancies</span>. <i>J. Hematol. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">36</span>, <span class="refDoi"> DOI: 10.1186/1756-8722-2-36</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=10.1186%2F1756-8722-2-36" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=19674465" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=1%3ACAS%3A280%3ADC%252BD1Mrps1Smug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2009&pages=36&author=V.+Kotlaauthor=S.+Goelauthor=S.+Nischalauthor=C.+Heuckauthor=K.+Vivekauthor=B.+Dasauthor=A.+Verma&title=Mechanism+of+Action+of+Lenalidomide+in+Hematological+Malignancies&doi=10.1186%2F1756-8722-2-36"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Mechanism of action of lenalidomide in hematological malignancies</span></div><div class="casAuthors">Kotla Venumadhav; Goel Swati; Nischal Sangeeta; Heuck Christoph; Vivek Kumar; Das Bhaskar; Verma Amit</div><div class="citationInfo"><span class="NLM_cas:title">Journal of hematology & oncology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">36</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Immunomodulatory drugs lenalidomide and pomalidomide are synthetic compounds derived by modifying the chemical structure of thalidomide to improve its potency and reduce its side effects.  Lenalidomide is a 4-amino-glutamyl analogue of thalidomide that lacks the neurologic side effects of sedation and neuropathy and has emerged as a drug with activity against various hematological and solid malignancies.  It is approved by FDA for clinical use in myelodysplastic syndromes with deletion of chromosome 5q and multiple myeloma.  Lenalidomide has been shown to be an immunomodulator, affecting both cellular and humoral limbs of the immune system.  It has also been shown to have anti-angiogenic properties.  Newer studies demonstrate its effects on signal transduction that can partly explain its selective efficacy in subsets of MDS.  Even though the exact molecular targets of lenalidomide are not well known, its activity across a spectrum of neoplastic conditions highlights the possibility of multiple target sites of action.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRjgPlh7Q_C41VwdslNBF-tfW6udTcc2ebzJUEMZp3gA7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD1Mrps1Smug%253D%253D&md5=9615a7d7c2871ab6cb710c8ea896404c</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1186%2F1756-8722-2-36&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1756-8722-2-36%26sid%3Dliteratum%253Aachs%26aulast%3DKotla%26aufirst%3DV.%26aulast%3DGoel%26aufirst%3DS.%26aulast%3DNischal%26aufirst%3DS.%26aulast%3DHeuck%26aufirst%3DC.%26aulast%3DVivek%26aufirst%3DK.%26aulast%3DDas%26aufirst%3DB.%26aulast%3DVerma%26aufirst%3DA.%26atitle%3DMechanism%2520of%2520Action%2520of%2520Lenalidomide%2520in%2520Hematological%2520Malignancies%26jtitle%3DJ.%2520Hematol.%2520Oncol.%26date%3D2009%26volume%3D2%26spage%3D36%26doi%3D10.1186%2F1756-8722-2-36" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gintant, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">George, C. H.</span></span> <span> </span><span class="NLM_article-title">Introduction to Biological Complexity as a Missing Link in Drug Discovery</span>. <i>Expert Opin. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">753</span>– <span class="NLM_lpage">763</span>, <span class="refDoi"> DOI: 10.1080/17460441.2018.1480608</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=10.1080%2F17460441.2018.1480608" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=29871539" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtVOqu7nK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2018&pages=753-763&author=G.+A.+Gintantauthor=C.+H.+George&title=Introduction+to+Biological+Complexity+as+a+Missing+Link+in+Drug+Discovery&doi=10.1080%2F17460441.2018.1480608"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Introduction to biological complexity as a missing link in drug discovery</span></div><div class="casAuthors">Gintant, Gary A.; George, Christopher H.</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Drug Discovery</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">753-763</span>CODEN:
                <span class="NLM_cas:coden">EODDBX</span>;
        ISSN:<span class="NLM_cas:issn">1746-0441</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">A review.  Despite a burgeoning knowledge of the intricacies and mechanisms responsible for human disease, technol. advances in medicinal chem., and more efficient assays used for drug screening, it remains difficult to discover novel and effective pharmacol. therapies.: By ref. to the primary literature and concepts emerging from academic and industrial drug screening landscapes, the authors propose that this disconnect arises from the inability to scale and integrate responses from simpler model systems to outcomes from more complex and human-based biol. systems.: Further collaborative efforts combining target-based and phenotypic-based screening along with systems-based pharmacol. and informatics will be necessary to harness the technol. breakthroughs of today to derive the novel drug candidates of tomorrow.  New questions must be asked of enabling technologies-while recognizing inherent limitations-in a way that moves drug development forward.  Attempts to integrate mechanistic and observational information acquired across multiple scales frequently expose the gap between our knowledge and our understanding as the level of complexity increases.  We hope that the thoughts and actionable items highlighted will help to inform the directed evolution of the drug discovery process.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGospFb4tctmaLVg90H21EOLACvtfcHk0lhz9rQijfQwUQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtVOqu7nK&md5=74f5b440fe75340e28c1cd594d0600c0</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1080%2F17460441.2018.1480608&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F17460441.2018.1480608%26sid%3Dliteratum%253Aachs%26aulast%3DGintant%26aufirst%3DG.%2BA.%26aulast%3DGeorge%26aufirst%3DC.%2BH.%26atitle%3DIntroduction%2520to%2520Biological%2520Complexity%2520as%2520a%2520Missing%2520Link%2520in%2520Drug%2520Discovery%26jtitle%3DExpert%2520Opin.%2520Drug%2520Discovery%26date%3D2018%26volume%3D13%26spage%3D753%26epage%3D763%26doi%3D10.1080%2F17460441.2018.1480608" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, W. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ho, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, Y. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ling, T. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, C. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pan, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chou, H. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, G. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chu, W. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, Y. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, K. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, P. C.</span></span> <span> </span><span class="NLM_article-title">Cancer-Associated Fibroblasts Regulate the Plasticity of Lung Cancer Stemness Via Paracrine Signalling</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">3472</span>, <span class="refDoi"> DOI: 10.1038/ncomms4472</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=10.1038%2Fncomms4472" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=24668028" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=1%3ACAS%3A280%3ADC%252BC2crnt12rsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=3472&author=W.+J.+Chenauthor=C.+C.+Hoauthor=Y.+L.+Changauthor=H.+Y.+Chenauthor=C.+A.+Linauthor=T.+Y.+Lingauthor=S.+L.+Yuauthor=S.+S.+Yuanauthor=Y.+J.+Chenauthor=C.+Y.+Linauthor=S.+H.+Panauthor=H.+Y.+Chouauthor=Y.+J.+Chenauthor=G.+C.+Changauthor=W.+C.+Chuauthor=Y.+M.+Leeauthor=J.+Y.+Leeauthor=P.+J.+Leeauthor=K.+C.+Liauthor=H.+W.+Chenauthor=P.+C.+Yang&title=Cancer-Associated+Fibroblasts+Regulate+the+Plasticity+of+Lung+Cancer+Stemness+Via+Paracrine+Signalling&doi=10.1038%2Fncomms4472"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Cancer-associated fibroblasts regulate the plasticity of lung cancer stemness via paracrine signalling</span></div><div class="casAuthors">Chen Wan-Jiun; Ho Chao-Chi; Chang Yih-Leong; Chen Hsuan-Yu; Yuan Shin-Sheng; Chen Yu-Ju Louisa; Lin Chien-Yu; Lin Chih-An; Yu Sung-Liang; Ling Thai-Yen; Pan Szu-Hua; Chou Han-Yi Elizabeth; Chen Yu-Ju; Chang Gee-Chen; Chu Wen-Cheng; Lee Yee-Ming; Lee Jen-Yi; Lee Pei-Jung; Li Ker-Chau; Chen Huei-Wen; Yang Pan-Chyr</div><div class="citationInfo"><span class="NLM_cas:title">Nature communications</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">3472</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Cancer stem cells (CSCs) are a promising target for treating cancer, yet how CSC plasticity is maintained in vivo is unclear and is difficult to study in vitro.  Here we establish a sustainable primary culture of Oct3/4(+)/Nanog(+) lung CSCs fed with CD90(+) cancer-associated fibroblasts (CAFs) to further advance our knowledge of preserving stem cells in the tumour microenvironment.  Using transcriptomics we identify the paracrine network by which CAFs enrich CSCs through de-differentiation and reacquisition of stem cell-like properties.  Specifically, we find that IGF1R signalling activation in cancer cells in the presence of CAFs expressing IGF-II can induce Nanog expression and promote stemness.  Moreover, this paracrine signalling predicts overall and relapse-free survival in stage I non-small cell lung cancer (NSCLC) patients.  IGF-II/IGF1R signalling blockade inhibits Nanog expression and attenuates cancer stem cell features.  Our data demonstrate that CAFs constitute a supporting niche for cancer stemness, and targeting this paracrine signalling may present a new therapeutic strategy for NSCLC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTm3hC1BFRecNy3lHz94U-GfW6udTcc2eapEd8VQLRD8Lntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2crnt12rsQ%253D%253D&md5=361b9cb8eee2a5a5b521f2d8a7ae7992</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1038%2Fncomms4472&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms4472%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DW.%2BJ.%26aulast%3DHo%26aufirst%3DC.%2BC.%26aulast%3DChang%26aufirst%3DY.%2BL.%26aulast%3DChen%26aufirst%3DH.%2BY.%26aulast%3DLin%26aufirst%3DC.%2BA.%26aulast%3DLing%26aufirst%3DT.%2BY.%26aulast%3DYu%26aufirst%3DS.%2BL.%26aulast%3DYuan%26aufirst%3DS.%2BS.%26aulast%3DChen%26aufirst%3DY.%2BJ.%26aulast%3DLin%26aufirst%3DC.%2BY.%26aulast%3DPan%26aufirst%3DS.%2BH.%26aulast%3DChou%26aufirst%3DH.%2BY.%26aulast%3DChen%26aufirst%3DY.%2BJ.%26aulast%3DChang%26aufirst%3DG.%2BC.%26aulast%3DChu%26aufirst%3DW.%2BC.%26aulast%3DLee%26aufirst%3DY.%2BM.%26aulast%3DLee%26aufirst%3DJ.%2BY.%26aulast%3DLee%26aufirst%3DP.%2BJ.%26aulast%3DLi%26aufirst%3DK.%2BC.%26aulast%3DChen%26aufirst%3DH.%2BW.%26aulast%3DYang%26aufirst%3DP.%2BC.%26atitle%3DCancer-Associated%2520Fibroblasts%2520Regulate%2520the%2520Plasticity%2520of%2520Lung%2520Cancer%2520Stemness%2520Via%2520Paracrine%2520Signalling%26jtitle%3DNat.%2520Commun.%26date%3D2014%26volume%3D5%26spage%3D3472%26doi%3D10.1038%2Fncomms4472" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kuo, T. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pan, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fang, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hung, P. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hong, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hsu, Y. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, P. C.</span></span> <span> </span><span class="NLM_article-title">Purine-Type Compounds Induce Microtubule Fragmentation and Lung Cancer Cell Death through Interaction with Katanin</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">8521</span>– <span class="NLM_lpage">8534</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b00797</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b00797" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtlOgtrnI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=8521-8534&author=T.+C.+Kuoauthor=L.+W.+Liauthor=S.+H.+Panauthor=J.+M.+Fangauthor=J.+H.+Liuauthor=T.+J.+Chengauthor=C.+J.+Wangauthor=P.+F.+Hungauthor=H.+Y.+Chenauthor=T.+M.+Hongauthor=Y.+L.+Hsuauthor=C.+H.+Wongauthor=P.+C.+Yang&title=Purine-Type+Compounds+Induce+Microtubule+Fragmentation+and+Lung+Cancer+Cell+Death+through+Interaction+with+Katanin&doi=10.1021%2Facs.jmedchem.6b00797"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Purine-Type Compounds Induce Microtubule Fragmentation and Lung Cancer Cell Death through Interaction with Katanin</span></div><div class="casAuthors">Kuo, Ting-Chun; Li, Ling-Wei; Pan, Szu-Hua; Fang, Jim-Min; Liu, Jyung-Hurng; Cheng, Ting-Jen; Wang, Chia-Jen; Hung, Pei-Fang; Chen, Hsuan-Yu; Hong, Tse-Ming; Hsu, Yuan-Ling; Wong, Chi-Huey; Yang, Pan-Chyr</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">8521-8534</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Microtubule targeting agents (MTAs) constitute a class of drugs for cancer treatment.  Despite many MTAs have been proven to significantly improve the treatment outcomes of various malignancies, resistance has usually occurred.  By selection from a 2-million entry chem. library based on the efficacy and safety, the authors identified purine-type compds. that were active against lung small cell lung cancer (NSCLC).  The purine compd. I (GRC0321) was an MTA with good effects against NSCLC.  Lung cancer cells H1975 treated with I could induce microtubule fragmentation, leading to G2/M cell cycle arrest and intrinsic apoptosis.  Compd. I directly targeted katanin and regulated the severing activity of katanin, which cut the cellular microtubules into short pieces and activated c-Jun N-terminal kinases (JNK).  The microtubule fragmenting effect of I is a unique mechanism in MTAs.  It might overcome the resistance problems that most of the MTAs have faced.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrHYxyvm45m4bVg90H21EOLACvtfcHk0lhGbub-vrrJqw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtlOgtrnI&md5=069f8498e6fd14cac3f86324b77e09cb</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b00797&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b00797%26sid%3Dliteratum%253Aachs%26aulast%3DKuo%26aufirst%3DT.%2BC.%26aulast%3DLi%26aufirst%3DL.%2BW.%26aulast%3DPan%26aufirst%3DS.%2BH.%26aulast%3DFang%26aufirst%3DJ.%2BM.%26aulast%3DLiu%26aufirst%3DJ.%2BH.%26aulast%3DCheng%26aufirst%3DT.%2BJ.%26aulast%3DWang%26aufirst%3DC.%2BJ.%26aulast%3DHung%26aufirst%3DP.%2BF.%26aulast%3DChen%26aufirst%3DH.%2BY.%26aulast%3DHong%26aufirst%3DT.%2BM.%26aulast%3DHsu%26aufirst%3DY.%2BL.%26aulast%3DWong%26aufirst%3DC.%2BH.%26aulast%3DYang%26aufirst%3DP.%2BC.%26atitle%3DPurine-Type%2520Compounds%2520Induce%2520Microtubule%2520Fragmentation%2520and%2520Lung%2520Cancer%2520Cell%2520Death%2520through%2520Interaction%2520with%2520Katanin%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D8521%26epage%3D8534%26doi%3D10.1021%2Facs.jmedchem.6b00797" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Feng, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, C.</span></span> <span> </span><span class="NLM_article-title">Inhibiting the Expression of DNA Replication-Initiation Proteins Induces Apoptosis in Human Cancer Cells</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">7356</span>– <span class="NLM_lpage">7364</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=14612534" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=1%3ACAS%3A528%3ADC%252BD3sXovVGltL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2003&pages=7356-7364&author=D.+Fengauthor=Z.+Tuauthor=W.+Wuauthor=C.+Liang&title=Inhibiting+the+Expression+of+DNA+Replication-Initiation+Proteins+Induces+Apoptosis+in+Human+Cancer+Cells"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibiting the expression of DNA replication-initiation proteins induces apoptosis in human cancer cells</span></div><div class="casAuthors">Feng, Daorong; Tu, Zheng; Wu, Wenyan; Liang, Chun</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">7356-7364</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">DNA replication-initiation proteins are expressed in cancer cells, whereas some of these proteins are not expressed in nonproliferating normal cells.  Therefore, replication-initiation proteins may present attractive targets for anticancer therapy.  Using selected antisense oligodeoxynucleotides and small interfering RNA mols. targeted to the mRNA encoding the DNA replication-initiation proteins hCdc6p, hMcm2p, and hCdc45p, we show that the target genes could be effectively and specifically silenced and that, consequently, DNA replication and cell proliferation were inhibited in cultured human cells.  In addn., silencing of these genes resulted in apoptosis in both p53-pos. and -neg. cancer cells but not in normal cells: cancer cells entered an abortive S-phase, whereas normal cells arrested mainly in G1 phase.  Our studies are the first to suggest that inhibiting the expression of selective replication-initiation proteins is a novel and effective anticancer strategy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGorSWr21O_JrrVg90H21EOLACvtfcHk0lgRs3xv5IAyVw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXovVGltL8%253D&md5=2839ea7e0507d2ce35cf69c90204636b</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DFeng%26aufirst%3DD.%26aulast%3DTu%26aufirst%3DZ.%26aulast%3DWu%26aufirst%3DW.%26aulast%3DLiang%26aufirst%3DC.%26atitle%3DInhibiting%2520the%2520Expression%2520of%2520DNA%2520Replication-Initiation%2520Proteins%2520Induces%2520Apoptosis%2520in%2520Human%2520Cancer%2520Cells%26jtitle%3DCancer%2520Res.%26date%3D2003%26volume%3D63%26spage%3D7356%26epage%3D7364" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yamauchi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamaguchi, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakata, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kohno, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nagasaki, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shimamura, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Imoto, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saito, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ueno, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hatanaka, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoshida, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Higuchi, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nomura, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beer, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yokota, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miyano, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gotoh, N.</span></span> <span> </span><span class="NLM_article-title">Epidermal Growth Factor Receptor Tyrosine Kinase Defines Critical Prognostic Genes of Stage I Lung Adenocarcinoma</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>7</i></span>, <span class="NLM_elocation-id">e43923</span> <span class="refDoi"> DOI: 10.1371/journal.pone.0043923</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=10.1371%2Fjournal.pone.0043923" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=23028479" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsVaktrbL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2012&author=M.+Yamauchiauthor=R.+Yamaguchiauthor=A.+Nakataauthor=T.+Kohnoauthor=M.+Nagasakiauthor=T.+Shimamuraauthor=S.+Imotoauthor=A.+Saitoauthor=K.+Uenoauthor=Y.+Hatanakaauthor=R.+Yoshidaauthor=T.+Higuchiauthor=M.+Nomuraauthor=D.+G.+Beerauthor=J.+Yokotaauthor=S.+Miyanoauthor=N.+Gotoh&title=Epidermal+Growth+Factor+Receptor+Tyrosine+Kinase+Defines+Critical+Prognostic+Genes+of+Stage+I+Lung+Adenocarcinoma&doi=10.1371%2Fjournal.pone.0043923"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Epidermal growth factor receptor tyrosine kinase defines critical prognostic genes of stage I lung adenocarcinoma</span></div><div class="casAuthors">Yamauchi, Mai; Yamaguchi, Rui; Nakata, Asuka; Kohno, Takashi; Nagasaki, Masao; Shimamura, Teppei; Imoto, Seiya; Saito, Ayumu; Ueno, Kazuko; Hatanaka, Yousuke; Yoshida, Ryo; Higuchi, Tomoyuki; Nomura, Masaharu; Beer, David G.; Yokota, Jun; Miyano, Satoru; Gotoh, Noriko</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">e43923</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Purpose: To identify stage I lung adenocarcinoma patients with a poor prognosis who will benefit from adjuvant therapy.  Patients and Methods: Whole gene expression profiles were obtained at 19 time points over a 48-h time course from human primary lung epithelial cells that were stimulated with epidermal growth factor (EGF) in the presence or absence of a clin. used EGF receptor tyrosine kinase (RTK)-specific inhibitor, gefitinib.  The data were subjected to a math. simulation using the State Space Model (SSM).  "Gefitinib-sensitive" genes, the expressional dynamics of which were altered by addn. of gefitinib, were identified.  A risk scoring model was constructed to classify high- or low-risk patients based on expression signatures of 139 gefitinib-sensitive genes in lung cancer using a training data set of 253 lung adenocarcinomas of North American cohort.  The predictive ability of the risk scoring model was examd. in independent cohorts of surgical specimens of lung cancer.  Results: The risk scoring model enabled the identification of high-risk stage IA and IB cases in another North American cohort for overall survival (OS) with a hazard ratio (HR) of 7.16 (P = 0.029) and 3.26 (P = 0.0072), resp.  It also enabled the identification of high-risk stage I cases without bronchioalveolar carcinoma (BAC) histol. in a Japanese cohort for OS and recurrence-free survival (RFS) with HRs of 8.79 (P = 0.001) and 3.72 (P = 0.0049), resp.  Conclusion: The set of 139 gefitinib-sensitive genes includes many genes known to be involved in biol. aspects of cancer phenotypes, but not known to be involved in EGF signaling.  The present result strongly re-emphasizes that EGF signaling status in cancer cells underlies an aggressive phenotype of cancer cells, which is useful for the selection of early-stage lung adenocarcinoma patients with a poor prognosis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGolLhiLU3WpObVg90H21EOLACvtfcHk0lgRs3xv5IAyVw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsVaktrbL&md5=4b2ef5e7e98350134ce442b47d57df87</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0043923&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0043923%26sid%3Dliteratum%253Aachs%26aulast%3DYamauchi%26aufirst%3DM.%26aulast%3DYamaguchi%26aufirst%3DR.%26aulast%3DNakata%26aufirst%3DA.%26aulast%3DKohno%26aufirst%3DT.%26aulast%3DNagasaki%26aufirst%3DM.%26aulast%3DShimamura%26aufirst%3DT.%26aulast%3DImoto%26aufirst%3DS.%26aulast%3DSaito%26aufirst%3DA.%26aulast%3DUeno%26aufirst%3DK.%26aulast%3DHatanaka%26aufirst%3DY.%26aulast%3DYoshida%26aufirst%3DR.%26aulast%3DHiguchi%26aufirst%3DT.%26aulast%3DNomura%26aufirst%3DM.%26aulast%3DBeer%26aufirst%3DD.%2BG.%26aulast%3DYokota%26aufirst%3DJ.%26aulast%3DMiyano%26aufirst%3DS.%26aulast%3DGotoh%26aufirst%3DN.%26atitle%3DEpidermal%2520Growth%2520Factor%2520Receptor%2520Tyrosine%2520Kinase%2520Defines%2520Critical%2520Prognostic%2520Genes%2520of%2520Stage%2520I%2520Lung%2520Adenocarcinoma%26jtitle%3DPLoS%2520One%26date%3D2012%26volume%3D7%26doi%3D10.1371%2Fjournal.pone.0043923" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Forsburg, S. L.</span></span> <span> </span><span class="NLM_article-title">Eukaryotic Mcm Proteins: Beyond Replication Initiation</span>. <i>Microbiol Mol. Biol. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>68</i></span>,  <span class="NLM_fpage">109</span>– <span class="NLM_lpage">131</span>, <span class="refDoi"> DOI: 10.1128/MMBR.68.1.109-131.2004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=10.1128%2FMMBR.68.1.109-131.2004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=15007098" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=1%3ACAS%3A528%3ADC%252BD2cXjtFehtr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=68&publication_year=2004&pages=109-131&author=S.+L.+Forsburg&title=Eukaryotic+Mcm+Proteins%3A+Beyond+Replication+Initiation&doi=10.1128%2FMMBR.68.1.109-131.2004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Eukaryotic MCM proteins: beyond replication initiation</span></div><div class="casAuthors">Forsburg, Susan L.</div><div class="citationInfo"><span class="NLM_cas:title">Microbiology and Molecular Biology Reviews</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">68</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">109-131</span>CODEN:
                <span class="NLM_cas:coden">MMBRF7</span>;
        ISSN:<span class="NLM_cas:issn">1092-2172</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">A review.  The minichromosome maintenance (or MCM) protein family is composed of six related proteins that are conserved in all eukaryotes.  They were first identified by genetic screens in yeast and subsequently analyzed in other exptl. systems using mol. and biochem. methods.  Early data led to the identification of MCMs as central players in the initiation of DNA replication.  More recent studies have shown that MCM proteins also function in replication elongation, probably as a DNA helicase.  This is consistent with structural anal. showing that the proteins interact together in a heterohexameric ring.  However, MCMs are strikingly abundant and far exceed the stoichiometry of replication origins; they are widely distributed on unreplicated chromatin.  Anal. of mcm mutant phenotypes and interactions with other factors have now implicated the MCM proteins in other chromosome transactions including damage response, transcription, and chromatin structure.  These expts. indicate that the MCMs are central players in many aspects of genome stability.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr-4X_4eAIXyrVg90H21EOLACvtfcHk0lgotNZE6lfMcg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXjtFehtr8%253D&md5=f01251abb8eaad801108d1737ae4e652</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1128%2FMMBR.68.1.109-131.2004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMMBR.68.1.109-131.2004%26sid%3Dliteratum%253Aachs%26aulast%3DForsburg%26aufirst%3DS.%2BL.%26atitle%3DEukaryotic%2520Mcm%2520Proteins%253A%2520Beyond%2520Replication%2520Initiation%26jtitle%3DMicrobiol%2520Mol.%2520Biol.%2520Rev.%26date%3D2004%26volume%3D68%26spage%3D109%26epage%3D131%26doi%3D10.1128%2FMMBR.68.1.109-131.2004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Emanuele, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elia, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thoma, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Izhar, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leng, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rush, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hsu, P. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yen, H. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elledge, S. J.</span></span> <span> </span><span class="NLM_article-title">Global Identification of Modular Cullin-Ring Ligase Substrates</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>147</i></span>,  <span class="NLM_fpage">459</span>– <span class="NLM_lpage">474</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2011.09.019</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=10.1016%2Fj.cell.2011.09.019" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=21963094" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtlaksbvJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=147&publication_year=2011&pages=459-474&author=M.+J.+Emanueleauthor=A.+E.+Eliaauthor=Q.+Xuauthor=C.+R.+Thomaauthor=L.+Izharauthor=Y.+Lengauthor=A.+Guoauthor=Y.+N.+Chenauthor=J.+Rushauthor=P.+W.+Hsuauthor=H.+C.+Yenauthor=S.+J.+Elledge&title=Global+Identification+of+Modular+Cullin-Ring+Ligase+Substrates&doi=10.1016%2Fj.cell.2011.09.019"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Global Identification of Modular Cullin-RING Ligase Substrates</span></div><div class="casAuthors">Emanuele, Michael J.; Elia, Andrew E. H.; Xu, Qikai; Thoma, Claudio R.; Izhar, Lior; Leng, Yumei; Guo, Ailan; Chen, Yi-Ning; Rush, John; Hsu, Paul Wei-Che; Yen, Hsueh-Chi Sherry; Elledge, Stephen J.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">147</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">459-474</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Cullin-RING ligases (CRLs) represent the largest E3 ubiquitin ligase family in eukaryotes, and the identification of their substrates is crit. to understanding regulation of the proteome.  Using genetic and pharmacol. Cullin inactivation coupled with genetic (GPS) and proteomic (QUAINT) assays, we have identified hundreds of proteins whose stabilities or ubiquitylation status are regulated by CRLs.  Together, these approaches yielded many known CRL substrates as well as a multitude of previously unknown putative substrates.  We demonstrate that one substrate, NUSAP1, is an SCFCyclin F substrate during S and G2 phases of the cell cycle and is also degraded in response to DNA damage.  This collection of regulated substrates is highly enriched for nodes in protein interaction networks, representing crit. connections between regulatory pathways.  This demonstrates the broad role of CRL ubiquitylation in all aspects of cellular biol. and provides a set of proteins likely to be key indicators of cellular physiol.  A combination of genetic and proteomic screens reveals cullin-RING ligase substrates and places these factors at key regulatory nodes in cellular networks.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpr1k89XZLzgrVg90H21EOLACvtfcHk0lgotNZE6lfMcg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtlaksbvJ&md5=294df7034b0127b3a6f2f26977634f81</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2011.09.019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2011.09.019%26sid%3Dliteratum%253Aachs%26aulast%3DEmanuele%26aufirst%3DM.%2BJ.%26aulast%3DElia%26aufirst%3DA.%2BE.%26aulast%3DXu%26aufirst%3DQ.%26aulast%3DThoma%26aufirst%3DC.%2BR.%26aulast%3DIzhar%26aufirst%3DL.%26aulast%3DLeng%26aufirst%3DY.%26aulast%3DGuo%26aufirst%3DA.%26aulast%3DChen%26aufirst%3DY.%2BN.%26aulast%3DRush%26aufirst%3DJ.%26aulast%3DHsu%26aufirst%3DP.%2BW.%26aulast%3DYen%26aufirst%3DH.%2BC.%26aulast%3DElledge%26aufirst%3DS.%2BJ.%26atitle%3DGlobal%2520Identification%2520of%2520Modular%2520Cullin-Ring%2520Ligase%2520Substrates%26jtitle%3DCell%26date%3D2011%26volume%3D147%26spage%3D459%26epage%3D474%26doi%3D10.1016%2Fj.cell.2011.09.019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Z. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jang, Y. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, Y. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, W.</span></span> <span> </span><span class="NLM_article-title">A Versatile and Practical Method for Regioselective Synthesis of Polysubstituted Furanonaphthoquinones</span>. <i>Org. Biomol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">828</span>– <span class="NLM_lpage">834</span>, <span class="refDoi"> DOI: 10.1039/C2OB26986D</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=10.1039%2FC2OB26986D" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=23235915" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=1%3ACAS%3A528%3ADC%252BC3sXjtlGjtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2013&pages=828-834&author=Z.+Z.+Wuauthor=Y.+J.+Jangauthor=C.+J.+Leeauthor=Y.+T.+Leeauthor=W.+Lin&title=A+Versatile+and+Practical+Method+for+Regioselective+Synthesis+of+Polysubstituted+Furanonaphthoquinones&doi=10.1039%2FC2OB26986D"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">A versatile and practical method for regioselective synthesis of polysubstituted furanonaphthoquinones</span></div><div class="casAuthors">Wu, Zong-Ze; Jang, Yeong-Jiunn; Lee, Chia-Jui; Lee, Yen-Te; Lin, Wenwei</div><div class="citationInfo"><span class="NLM_cas:title">Organic & Biomolecular Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">828-834</span>CODEN:
                <span class="NLM_cas:coden">OBCRAK</span>;
        ISSN:<span class="NLM_cas:issn">1477-0520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">An efficient and attractive synthesis of a series of novel poly-functionalized phosphorus zwitterions was achieved via three-component reactions of the corresponding functional nucleophiles, aldehydes, and Bu3P in the presence of acidic promoter.  E.g., in presence of TFA, reaction of 2-hydroxynaphthoquinone, 4-BrC6H4CHO, and Bu3P gave 83% phosphorus zwitterion (I).  These polysubstituted zwitterions could regioselectively undergo further transformations to synthetically important furanonaphthoquinones and related products via the intramol. Wittig reaction.  E.g., reaction of I with PhCOCl gave 92% furanonaphthoquinone (II).  These methods could have potential application in synthetic and pharmaceutical chem. for its facilitation and easily accessible com. materials.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoa0oHaxvHZCbVg90H21EOLACvtfcHk0lgotNZE6lfMcg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXjtlGjtQ%253D%253D&md5=d67158fbebdf005883d5b02c98f79d73</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1039%2FC2OB26986D&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC2OB26986D%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DZ.%2BZ.%26aulast%3DJang%26aufirst%3DY.%2BJ.%26aulast%3DLee%26aufirst%3DC.%2BJ.%26aulast%3DLee%26aufirst%3DY.%2BT.%26aulast%3DLin%26aufirst%3DW.%26atitle%3DA%2520Versatile%2520and%2520Practical%2520Method%2520for%2520Regioselective%2520Synthesis%2520of%2520Polysubstituted%2520Furanonaphthoquinones%26jtitle%3DOrg.%2520Biomol.%2520Chem.%26date%3D2013%26volume%3D11%26spage%3D828%26epage%3D834%26doi%3D10.1039%2FC2OB26986D" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sakakibara, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kelman, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kelman, Z.</span></span> <span> </span><span class="NLM_article-title">Unwinding the Structure and Function of the Archaeal Mcm Helicase</span>. <i>Mol. Microbiol.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>72</i></span>,  <span class="NLM_fpage">286</span>– <span class="NLM_lpage">296</span>, <span class="refDoi"> DOI: 10.1111/j.1365-2958.2009.06663.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=10.1111%2Fj.1365-2958.2009.06663.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=19415794" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=1%3ACAS%3A528%3ADC%252BD1MXlsFSrtrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=72&publication_year=2009&pages=286-296&author=N.+Sakakibaraauthor=L.+M.+Kelmanauthor=Z.+Kelman&title=Unwinding+the+Structure+and+Function+of+the+Archaeal+Mcm+Helicase&doi=10.1111%2Fj.1365-2958.2009.06663.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Unwinding the structure and function of the archaeal MCM helicase</span></div><div class="casAuthors">Sakakibara, Nozomi; Kelman, Lori M.; Kelman, Zvi</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Microbiology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">72</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">286-296</span>CODEN:
                <span class="NLM_cas:coden">MOMIEE</span>;
        ISSN:<span class="NLM_cas:issn">0950-382X</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">A review.  During chromosomal DNA replication, the replicative helicase unwinds the duplex DNA to provide the single-stranded DNA substrate for the polymerase.  In archaea, the replicative helicase is the minichromosome maintenance (MCM) complex.  The enzyme utilizes the energy of ATP hydrolysis to translocate along one strand of the duplex and unwind the complementary strand.  Much progress has been made in elucidating structure and function since the 1st report on the biochem. properties of an archaeal MCM protein in 1999.  The biochem. and structural properties of the enzyme from several archaeal species and some of the mechanisms by which the enzyme is regulated are now known.  Here, the authors summarize recent studies on the archaeal MCM protein and discusses the implications for helicase function and DNA replication in archaea.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrga7VKoWZbO7Vg90H21EOLACvtfcHk0lhsM1er4stARw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXlsFSrtrk%253D&md5=6ff2a1537d6775376390450e7ebd580a</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1111%2Fj.1365-2958.2009.06663.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1365-2958.2009.06663.x%26sid%3Dliteratum%253Aachs%26aulast%3DSakakibara%26aufirst%3DN.%26aulast%3DKelman%26aufirst%3DL.%2BM.%26aulast%3DKelman%26aufirst%3DZ.%26atitle%3DUnwinding%2520the%2520Structure%2520and%2520Function%2520of%2520the%2520Archaeal%2520Mcm%2520Helicase%26jtitle%3DMol.%2520Microbiol.%26date%3D2009%26volume%3D72%26spage%3D286%26epage%3D296%26doi%3D10.1111%2Fj.1365-2958.2009.06663.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fletcher, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bishop, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leon, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sclafani, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ogata, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X. S.</span></span> <span> </span><span class="NLM_article-title">The Structure and Function of Mcm from Archaeal M. Thermoautotrophicum</span>. <i>Nat. Struct. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">160</span>– <span class="NLM_lpage">167</span>, <span class="refDoi"> DOI: 10.1038/nsb893</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=10.1038%2Fnsb893" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2003&pages=160-167&author=R.+J.+Fletcherauthor=B.+E.+Bishopauthor=R.+P.+Leonauthor=R.+A.+Sclafaniauthor=C.+M.+Ogataauthor=X.+S.+Chen&title=The+Structure+and+Function+of+Mcm+from+Archaeal+M.+Thermoautotrophicum&doi=10.1038%2Fnsb893"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1038%2Fnsb893&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnsb893%26sid%3Dliteratum%253Aachs%26aulast%3DFletcher%26aufirst%3DR.%2BJ.%26aulast%3DBishop%26aufirst%3DB.%2BE.%26aulast%3DLeon%26aufirst%3DR.%2BP.%26aulast%3DSclafani%26aufirst%3DR.%2BA.%26aulast%3DOgata%26aufirst%3DC.%2BM.%26aulast%3DChen%26aufirst%3DX.%2BS.%26atitle%3DThe%2520Structure%2520and%2520Function%2520of%2520Mcm%2520from%2520Archaeal%2520M.%2520Thermoautotrophicum%26jtitle%3DNat.%2520Struct.%2520Mol.%2520Biol.%26date%3D2003%26volume%3D10%26spage%3D160%26epage%3D167%26doi%3D10.1038%2Fnsb893" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fletcher, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X. S.</span></span> <span> </span><span class="NLM_article-title">Biochemical Activities of the Bob1 Mutant in Methanobacterium Thermoautotrophicum Mcm</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>45</i></span>,  <span class="NLM_fpage">462</span>– <span class="NLM_lpage">467</span>, <span class="refDoi"> DOI: 10.1021/bi051754z</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi051754z" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtlWmurvM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2006&pages=462-467&author=R.+J.+Fletcherauthor=X.+S.+Chen&title=Biochemical+Activities+of+the+Bob1+Mutant+in+Methanobacterium+Thermoautotrophicum+Mcm&doi=10.1021%2Fbi051754z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Biochemical Activities of the BOB1 Mutant in Methanobacterium thermoautotrophicum MCM</span></div><div class="casAuthors">Fletcher, Ryan J.; Chen, Xiaojiang S.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">462-467</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Minichromosomal maintenance proteins (MCMs) are considered to be the replicative helicase.  Methanobacterium thermoautotrophicum has a single MCM gene (mtMCM).  The crystal structure of the mtMCM N-terminal region is a double hexamer.  Structure-guided sequence alignment indicates a structural conservation of this fragment across archaeal and eukaryotic MCMs.  The mtMCM structure was successfully used to analyze a Saccharomyces cerevisiae MCM5 mutant, called BOB1, which contains a single residue change from Pro to Leu and bypasses a kinase normally required for initiation of DNA replication.  A domain-push model was proposed to explain the BOB1 bypass activity.  Here we investigate the effects of BOB1 mutation on the biochem. activities of mtMCM.  Surprisingly, the BOB1 mutation (P62L) had a major effect on the helicase activity but had no significant impact on DNA binding and ATPase activities.  These results will contribute to a more detailed understanding of the BOB1 bypass activity and other aspects of DNA replication control.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrKMkxzfOUFS7Vg90H21EOLACvtfcHk0lhsM1er4stARw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtlWmurvM&md5=4d5c1ad5e48a699718a4a8ebf1fdc9bc</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1021%2Fbi051754z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi051754z%26sid%3Dliteratum%253Aachs%26aulast%3DFletcher%26aufirst%3DR.%2BJ.%26aulast%3DChen%26aufirst%3DX.%2BS.%26atitle%3DBiochemical%2520Activities%2520of%2520the%2520Bob1%2520Mutant%2520in%2520Methanobacterium%2520Thermoautotrophicum%2520Mcm%26jtitle%3DBiochemistry%26date%3D2006%26volume%3D45%26spage%3D462%26epage%3D467%26doi%3D10.1021%2Fbi051754z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wan, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Q. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bai, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span> <span> </span><span class="NLM_article-title">Norcantharidin Inhibits Pre-Replicative Complexes Assembly of Hepg2 Cells</span>. <i>Am. J. Chin. Med.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>41</i></span>,  <span class="NLM_fpage">665</span>– <span class="NLM_lpage">682</span>, <span class="refDoi"> DOI: 10.1142/S0192415X13500468</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=10.1142%2FS0192415X13500468" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=23711148" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=1%3ACAS%3A528%3ADC%252BC3sXosFaqt7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2013&pages=665-682&author=S.+Chenauthor=X.+Quauthor=P.+Wanauthor=Q.+W.+Liauthor=Z.+Wangauthor=F.+Guoauthor=L.+Baiauthor=Z.+Huauthor=W.+Tanauthor=J.+Li&title=Norcantharidin+Inhibits+Pre-Replicative+Complexes+Assembly+of+Hepg2+Cells&doi=10.1142%2FS0192415X13500468"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Norcantharidin Inhibits Pre-Replicative Complexes Assembly of HepG2 Cells</span></div><div class="casAuthors">Chen, Sansan; Qu, Xinming; Wan, Pei; Li, Qing Wen; Wang, Ziyi; Guo, Fangfang; Bai, Li; Hu, Zhiming; Tan, Wanlong; Li, Jinlong</div><div class="citationInfo"><span class="NLM_cas:title">American Journal of Chinese Medicine</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">665-682</span>CODEN:
                <span class="NLM_cas:coden">AJCMBA</span>;
        ISSN:<span class="NLM_cas:issn">0192-415X</span>.
    
            (<span class="NLM_cas:orgname">World Scientific Publishing Co. Pte. Ltd.</span>)
        </div><div class="casAbstract">Norcantharidin (NCTD) is currently used for anticancer therapy but the exact mechanism of action remains unknown.  Pre-replicative complexes (pre-RCs) are essential for cell DNA replication and highly related to malignant proliferation.  Here, we examd. the inhibitory effect of NCTD on pre-RC components in HepG2 cells.  We showed that NCTD induced degrdn. of Cdc6 and Mcm2 in a dose-dependent manner.  Under 100 μM NCTD concn., about 70% of Cdc6 and 50% of Mcm2 were degraded.  In addn., the nuclear translocation of Mcm6 was inhibited by NCTD.  Further studies aiming at G1 synchronous cells showed that, NCTD reduced the chromatin-bound Cdc6, Mcm2 and Mcm6.  Moreover, the cells were blocked from entering the S phase and accumulated at the G1 phase when released synchronously into the cell cycle.  Consistently, the DNA replication was inhibited by NCTD.  Finally, the combination NCTD with Cdc6 depletion lead to more severe cytotoxicity (88%) than NCTD (52%) and Cdc6 depletion (39%) alone.  A synergic cytotoxicity was obsd. between Cdc6 depletion and NCTD.  In conclusion, our results demonstrate that NCTD inhibits pre-RC assembly; subsequently blocks the G1 to S transition; and inhibits DNA replication in HepG2 cells.  Pre-RCs are an intriguing target for cancer therapy, which merits further investigations for anticancer development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpPKHJs6CODj7Vg90H21EOLACvtfcHk0lhsM1er4stARw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXosFaqt7k%253D&md5=85ba7ab64ff2f650bf202414de1a35ba</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1142%2FS0192415X13500468&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1142%252FS0192415X13500468%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DS.%26aulast%3DQu%26aufirst%3DX.%26aulast%3DWan%26aufirst%3DP.%26aulast%3DLi%26aufirst%3DQ.%2BW.%26aulast%3DWang%26aufirst%3DZ.%26aulast%3DGuo%26aufirst%3DF.%26aulast%3DBai%26aufirst%3DL.%26aulast%3DHu%26aufirst%3DZ.%26aulast%3DTan%26aufirst%3DW.%26aulast%3DLi%26aufirst%3DJ.%26atitle%3DNorcantharidin%2520Inhibits%2520Pre-Replicative%2520Complexes%2520Assembly%2520of%2520Hepg2%2520Cells%26jtitle%3DAm.%2520J.%2520Chin.%2520Med.%26date%3D2013%26volume%3D41%26spage%3D665%26epage%3D682%26doi%3D10.1142%2FS0192415X13500468" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guan, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, B.</span></span> <span> </span><span class="NLM_article-title">Mcm-2 Is a Therapeutic Target of Trichostatin a in Colon Cancer Cells</span>. <i>Toxicol. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>221</i></span>,  <span class="NLM_fpage">23</span>– <span class="NLM_lpage">30</span>, <span class="refDoi"> DOI: 10.1016/j.toxlet.2013.05.643</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=10.1016%2Fj.toxlet.2013.05.643" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=23770000" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtVKms7jN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=221&publication_year=2013&pages=23-30&author=Y.+Liuauthor=G.+Heauthor=Y.+Wangauthor=X.+Guanauthor=X.+Pangauthor=B.+Zhang&title=Mcm-2+Is+a+Therapeutic+Target+of+Trichostatin+a+in+Colon+Cancer+Cells&doi=10.1016%2Fj.toxlet.2013.05.643"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">MCM-2 is a therapeutic target of Trichostatin A in colon cancer cells</span></div><div class="casAuthors">Liu, Yangbo; He, Gang; Wang, Yan; Guan, Xinging; Pang, Xueli; Zhang, Bo</div><div class="citationInfo"><span class="NLM_cas:title">Toxicology Letters</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">221</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">23-30</span>CODEN:
                <span class="NLM_cas:coden">TOLED5</span>;
        ISSN:<span class="NLM_cas:issn">0378-4274</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ireland Ltd.</span>)
        </div><div class="casAbstract">Histone deacetylase (HDAC) inhibitors have recently emerged as a new class of anti-cancer agents.  Trichostatin A (TSA), a classical HDAC inhibitor, has been demonstrated to induce cell cycle arrest, promote cell apoptosis, and inhibit metastasis.  However, the mol. mechanism underlying TSA function has not been fully elucidated.  In the current study, we found that TSA treatment induced altered expression of cell cycle-assocd. genes in HCT116 cells by RT-PCR array.  Among the 84 genes related to cell cycle control, 34 genes were significantly altered by TSA treatment, with 7 genes upregulated and 27 genes downregulated.  Interestingly, gene expression of minichromosome maintenance protein-2 (MCM-2) was significantly downregulated by TSA treatment.  This was confirmed by quant. RT-PCR and Western blotting.  Moreover, silencing of MCM-2 by siRNA led to cell cycle arrest and apoptosis in HCT116 cells.  In addn., TSA caused an increase of phosphorylated JNK, which was involved in downregulation of MCM-2.  Together, our results suggest that MCM-2 is a novel therapeutic target of TSA in colon cancer cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo7hNc0Yjj8NLVg90H21EOLACvtfcHk0lhzwNKK9DUqSQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtVKms7jN&md5=098a399e3f5445d2a9429428550ae861</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1016%2Fj.toxlet.2013.05.643&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.toxlet.2013.05.643%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DHe%26aufirst%3DG.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DGuan%26aufirst%3DX.%26aulast%3DPang%26aufirst%3DX.%26aulast%3DZhang%26aufirst%3DB.%26atitle%3DMcm-2%2520Is%2520a%2520Therapeutic%2520Target%2520of%2520Trichostatin%2520a%2520in%2520Colon%2520Cancer%2520Cells%26jtitle%3DToxicol.%2520Lett.%26date%3D2013%26volume%3D221%26spage%3D23%26epage%3D30%26doi%3D10.1016%2Fj.toxlet.2013.05.643" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chu, Y. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, P. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shyu, Y. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hendrix, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, C. W.</span></span> <span> </span><span class="NLM_article-title">Selection of Invasive and Metastatic Subpopulations from a Human Lung Adenocarcinoma Cell Line</span>. <i>Am. J. Respir. Cell Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">353</span>– <span class="NLM_lpage">360</span>, <span class="refDoi"> DOI: 10.1165/ajrcmb.17.3.2837</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=10.1165%2Fajrcmb.17.3.2837" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=9308922" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=1%3ACAS%3A528%3ADyaK2sXmtVOrs7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=1997&pages=353-360&author=Y.+W.+Chuauthor=P.+C.+Yangauthor=S.+C.+Yangauthor=Y.+C.+Shyuauthor=M.+J.+Hendrixauthor=R.+Wuauthor=C.+W.+Wu&title=Selection+of+Invasive+and+Metastatic+Subpopulations+from+a+Human+Lung+Adenocarcinoma+Cell+Line&doi=10.1165%2Fajrcmb.17.3.2837"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Selection of invasive and metastatic subpopulations from a human lung adenocarcinoma cell line</span></div><div class="casAuthors">Chu, Yi-Wen; Yang, Pan-Chyr; Yang, Shuenn-Chen; Shyu, Yu-Chiau; Hendrix, Mary J. C.; Wu, Reen; Wu, Cheng-Wen</div><div class="citationInfo"><span class="NLM_cas:title">American Journal of Respiratory Cell and Molecular Biology</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">353-360</span>CODEN:
                <span class="NLM_cas:coden">AJRBEL</span>;
        ISSN:<span class="NLM_cas:issn">1044-1549</span>.
    
            (<span class="NLM_cas:orgname">American Lung Association</span>)
        </div><div class="casAbstract">To better understand the mechanism(s) underlying lung cancer invasion and metastasis, a Transwell invasion chamber was used to select progressively more invasive cancer cell populations from a clonal cell line of human lung adenocarcinoma, CL1.  Five sublines with progressive invasiveness, designated CL1-1, CL1-2, CL1-3, CL1-4, and CL1-5, were obtained through this in vitro selection process.  Their invasive abilities through basement membrane matrix showed a 4-6-fold increase over that of the parental cells.  Moreover, the sublines manifested an increase in their colony-forming ability on soft agar, tumorigenicity, and metastatic potency in severe combined immunodeficiency (SCID) mice.  Examg. the phenotypes of the cell lines revealed increased expression of 92-kDa gelatinase and an increase in the cell population stained with anti-keratin-8 and -18 antibodies.  Clonal isolation of anti-keratin-18-antibody-pos. and -neg. cell populations demonstrated a correlated enhancement of the invasiveness of these cells and their expression of keratin-18.  Thus, the metastatic behavior of lung cancer cells can be characterized with this in vitro system, and the properties of these progressively invasive cancer cells can be clonally studied.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoqf3X5hlC0wLVg90H21EOLACvtfcHk0lhzwNKK9DUqSQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXmtVOrs7c%253D&md5=d5ea3f2aa0517622c6e6026663cc1a03</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1165%2Fajrcmb.17.3.2837&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1165%252Fajrcmb.17.3.2837%26sid%3Dliteratum%253Aachs%26aulast%3DChu%26aufirst%3DY.%2BW.%26aulast%3DYang%26aufirst%3DP.%2BC.%26aulast%3DYang%26aufirst%3DS.%2BC.%26aulast%3DShyu%26aufirst%3DY.%2BC.%26aulast%3DHendrix%26aufirst%3DM.%2BJ.%26aulast%3DWu%26aufirst%3DR.%26aulast%3DWu%26aufirst%3DC.%2BW.%26atitle%3DSelection%2520of%2520Invasive%2520and%2520Metastatic%2520Subpopulations%2520from%2520a%2520Human%2520Lung%2520Adenocarcinoma%2520Cell%2520Line%26jtitle%3DAm.%2520J.%2520Respir.%2520Cell%2520Mol.%2520Biol.%26date%3D1997%26volume%3D17%26spage%3D353%26epage%3D360%26doi%3D10.1165%2Fajrcmb.17.3.2837" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, W. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, Y. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, C. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chao, Y. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kao, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, W. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, K. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hong, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, P. C.</span></span> <span> </span><span class="NLM_article-title">P53 Controls Cancer Cell Invasion by Inducing the Mdm2-Mediated Degradation of Slug</span>. <i>Nat. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">694</span>– <span class="NLM_lpage">704</span>, <span class="refDoi"> DOI: 10.1038/ncb1875</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=10.1038%2Fncb1875" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=19448627" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=1%3ACAS%3A528%3ADC%252BD1MXms1als7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2009&pages=694-704&author=S.+P.+Wangauthor=W.+L.+Wangauthor=Y.+L.+Changauthor=C.+T.+Wuauthor=Y.+C.+Chaoauthor=S.+H.+Kaoauthor=A.+Yuanauthor=C.+W.+Linauthor=S.+C.+Yangauthor=W.+K.+Chanauthor=K.+C.+Liauthor=T.+M.+Hongauthor=P.+C.+Yang&title=P53+Controls+Cancer+Cell+Invasion+by+Inducing+the+Mdm2-Mediated+Degradation+of+Slug&doi=10.1038%2Fncb1875"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">p53 controls cancer cell invasion by inducing the MDM2-mediated degradation of Slug</span></div><div class="casAuthors">Wang, Shu-Ping; Wang, Wen-Lung; Chang, Yih-Leong; Wu, Chen-Tu; Chao, Yu-Chih; Kao, Shih-Han; Yuan, Ang; Lin, Chung-Wu; Yang, Shuenn-Chen; Chan, Wing-Kai; Li, Ker-Chau; Hong, Tse-Ming; Yang, Pan-Chyr</div><div class="citationInfo"><span class="NLM_cas:title">Nature Cell Biology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">694-704</span>CODEN:
                <span class="NLM_cas:coden">NCBIFN</span>;
        ISSN:<span class="NLM_cas:issn">1465-7392</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The tumor suppressor p53 is known to prevent cancer progression by inhibiting proliferation and inducing apoptosis of tumor cells.  Slug, an invasion promoter, exerts its effects by repressing E-cadherin transcription.  Here we show that wild-type p53 (wtp53) suppresses cancer invasion by inducing Slug degrdn., whereas mutant p53 may stabilize Slug protein.  In non-small-cell lung cancer (NSCLC), mutation of p53 correlates with low MDM2, high Slug and low E-cadherin expression.  This expression profile is assocd. with poor overall survival and short metastasis-free survival in patients with NSCLC. wtp53 upregulates MDM2 and forms a wtp53-MDM2-Slug complex that facilitates MDM2-mediated Slug degrdn.  Downregulation of Slug by wtp53 or MDM2 enhances E-cadherin expression and represses cancer cell invasiveness.  In contrast, mutant p53 inactivates Slug degrdn. and leads to Slug accumulation and increased cancer cell invasiveness.  Our findings indicate that wtp53 and p53 mutants may differentially control cancer invasion and metastasis through the p53-MDM2-Slug pathway.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpv0v8DzU0h_7Vg90H21EOLACvtfcHk0lhzwNKK9DUqSQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXms1als7g%253D&md5=71ce0ce22e87b86faf76c355afbcaff6</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1038%2Fncb1875&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncb1875%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DS.%2BP.%26aulast%3DWang%26aufirst%3DW.%2BL.%26aulast%3DChang%26aufirst%3DY.%2BL.%26aulast%3DWu%26aufirst%3DC.%2BT.%26aulast%3DChao%26aufirst%3DY.%2BC.%26aulast%3DKao%26aufirst%3DS.%2BH.%26aulast%3DYuan%26aufirst%3DA.%26aulast%3DLin%26aufirst%3DC.%2BW.%26aulast%3DYang%26aufirst%3DS.%2BC.%26aulast%3DChan%26aufirst%3DW.%2BK.%26aulast%3DLi%26aufirst%3DK.%2BC.%26aulast%3DHong%26aufirst%3DT.%2BM.%26aulast%3DYang%26aufirst%3DP.%2BC.%26atitle%3DP53%2520Controls%2520Cancer%2520Cell%2520Invasion%2520by%2520Inducing%2520the%2520Mdm2-Mediated%2520Degradation%2520of%2520Slug%26jtitle%3DNat.%2520Cell%2520Biol.%26date%3D2009%26volume%3D11%26spage%3D694%26epage%3D704%26doi%3D10.1038%2Fncb1875" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kelley, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sternberg, M. J.</span></span> <span> </span><span class="NLM_article-title">Protein Structure Prediction on the Web: A Case Study Using the Phyre Server</span>. <i>Nat. Protoc.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">363</span>– <span class="NLM_lpage">371</span>, <span class="refDoi"> DOI: 10.1038/nprot.2009.2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=10.1038%2Fnprot.2009.2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=19247286" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=1%3ACAS%3A528%3ADC%252BD1MXivF2itbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2009&pages=363-371&author=L.+A.+Kelleyauthor=M.+J.+Sternberg&title=Protein+Structure+Prediction+on+the+Web%3A+A+Case+Study+Using+the+Phyre+Server&doi=10.1038%2Fnprot.2009.2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Protein structure prediction on the Web: a case study using the Phyre server</span></div><div class="casAuthors">Kelley, Lawrence A.; Sternberg, Michael J. E.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Protocols</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">363-371</span>CODEN:
                <span class="NLM_cas:coden">NPARDW</span>;
        ISSN:<span class="NLM_cas:issn">1750-2799</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Detg. the structure and function of a novel protein is a cornerstone of many aspects of modern biol.  Over the past decades, a no. of computational tools for structure prediction have been developed.  It is crit. that the biol. community is aware of such tools and is able to interpret their results in an informed way.  This protocol provides a guide to interpreting the output of structure prediction servers in general and one such tool in particular, the protein homol./analogy recognition engine (Phyre).  New profile-profile matching algorithms have improved structure prediction considerably in recent years.  Although the performance of Phyre is typical of many structure prediction systems using such algorithms, all these systems can reliably detect up to twice as many remote homologies as std. sequence-profile searching.  Phyre is widely used by the biol. community, with >150 submissions per day, and provides a simple interface to results.  Phyre takes 30 min to predict the structure of a 250-residue protein.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqkM_7GFRXSorVg90H21EOLACvtfcHk0ljbYlD7y5Z02A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXivF2itbs%253D&md5=6613ba0bebfbba8594b5ceb7d9345bca</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1038%2Fnprot.2009.2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnprot.2009.2%26sid%3Dliteratum%253Aachs%26aulast%3DKelley%26aufirst%3DL.%2BA.%26aulast%3DSternberg%26aufirst%3DM.%2BJ.%26atitle%3DProtein%2520Structure%2520Prediction%2520on%2520the%2520Web%253A%2520A%2520Case%2520Study%2520Using%2520the%2520Phyre%2520Server%26jtitle%3DNat.%2520Protoc.%26date%3D2009%26volume%3D4%26spage%3D363%26epage%3D371%26doi%3D10.1038%2Fnprot.2009.2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Eswar, N.</span>; <span class="NLM_string-name">Webb, B.</span>; <span class="NLM_string-name">Marti-Renom, M. A.</span>; <span class="NLM_string-name">Madhusudhan, M. S.</span>; <span class="NLM_string-name">Eramian, D.</span>; <span class="NLM_string-name">Shen, M. Y.</span>; <span class="NLM_string-name">Pieper, U.</span>; <span class="NLM_string-name">Sali, A.</span></span> <span> </span><span class="NLM_article-title">Comparative Protein Structure Modeling Using Modeller</span>.  <i>Current Protocols in Protein Science</i>; <span class="NLM_publisher-name">Wiley</span>, <span class="NLM_year">2007</span>; Chapter 2, Unit 2.9, <span class="refDoi"> DOI: 10.1002/0471140864.ps0209s50</span> .</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=10.1002%2F0471140864.ps0209s50" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2007&author=N.+Eswar&author=B.+Webb&author=M.+A.+Marti-Renom&author=M.+S.+Madhusudhan&author=D.+Eramian&author=M.+Y.+Shen&author=U.+Pieper&author=A.+Sali&title=Current+Protocols+in+Protein+Science"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1002%2F0471140864.ps0209s50&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1002%252F0471140864.ps0209s50%26sid%3Dliteratum%253Aachs%26aulast%3DEswar%26aufirst%3DN.%26atitle%3DComparative%2520Protein%2520Structure%2520Modeling%2520Using%2520Modeller%26btitle%3DCurrent%2520Protocols%2520in%2520Protein%2520Science%26pub%3DWiley%26date%3D2007%26doi%3D10.1002%2F0471140864.ps0209s50" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pronk, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pall, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schulz, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Larsson, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bjelkmar, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Apostolov, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shirts, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kasson, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van der
Spoel, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hess, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindahl, E.</span></span> <span> </span><span class="NLM_article-title">Gromacs 4.5: A High-Throughput and Highly Parallel Open Source Molecular Simulation Toolkit</span>. <i>Bioinformatics</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">845</span>– <span class="NLM_lpage">854</span>, <span class="refDoi"> DOI: 10.1093/bioinformatics/btt055</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=10.1093%2Fbioinformatics%2Fbtt055" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=23407358" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=1%3ACAS%3A528%3ADC%252BC3sXksFWmsrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2013&pages=845-854&author=S.+Pronkauthor=S.+Pallauthor=R.+Schulzauthor=P.+Larssonauthor=P.+Bjelkmarauthor=R.+Apostolovauthor=M.+R.+Shirtsauthor=J.+C.+Smithauthor=P.+M.+Kassonauthor=D.+van+der%0ASpoelauthor=B.+Hessauthor=E.+Lindahl&title=Gromacs+4.5%3A+A+High-Throughput+and+Highly+Parallel+Open+Source+Molecular+Simulation+Toolkit&doi=10.1093%2Fbioinformatics%2Fbtt055"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">GROMACS 4.5: a high-throughput and highly parallel open source molecular simulation toolkit</span></div><div class="casAuthors">Pronk, Sander; Pall, Szilard; Schulz, Roland; Larsson, Per; Bjelkmar, Paer; Apostolov, Rossen; Shirts, Michael R.; Smith, Jeremy C.; Kasson, Peter M.; van der Spoel, David; Hess, Berk; Lindahl, Erik</div><div class="citationInfo"><span class="NLM_cas:title">Bioinformatics</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">845-854</span>CODEN:
                <span class="NLM_cas:coden">BOINFP</span>;
        ISSN:<span class="NLM_cas:issn">1367-4803</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Motivation: Mol. simulation has historically been a low-throughput technique, but faster computers and increasing amts. of genomic and structural data are changing this by enabling large-scale automated simulation of, for instance, many conformers or mutants of biomols. with or without a range of ligands.  At the same time, advances in performance and scaling now make it possible to model complex biomol. interaction and function in a manner directly testable by expt.  These applications share a need for fast and efficient software that can be deployed on massive scale in clusters, web servers, distributed computing or cloud resources.  Results: Here, we present a range of new simulation algorithms and features developed during the past 4 years, leading up to the GROMACS 4.5 software package.  The software now automatically handles wide classes of biomols., such as proteins, nucleic acids and lipids, and comes with all commonly used force fields for these mols. built-in.  GROMACS supports several implicit solvent models, as well as new free-energy algorithms, and the software now uses multithreading for efficient parallelization even on low-end systems, including windows-based workstations.  Together with hand-tuned assembly kernels and state-of-the-art parallelization, this provides extremely high performance and cost efficiency for high-throughput as well as massively parallel simulations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr71lQgYMm017Vg90H21EOLACvtfcHk0lh6XYJrEL9G4g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXksFWmsrg%253D&md5=4b25fd6ab4e33725ae56b5da63f4ad68</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1093%2Fbioinformatics%2Fbtt055&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fbioinformatics%252Fbtt055%26sid%3Dliteratum%253Aachs%26aulast%3DPronk%26aufirst%3DS.%26aulast%3DPall%26aufirst%3DS.%26aulast%3DSchulz%26aufirst%3DR.%26aulast%3DLarsson%26aufirst%3DP.%26aulast%3DBjelkmar%26aufirst%3DP.%26aulast%3DApostolov%26aufirst%3DR.%26aulast%3DShirts%26aufirst%3DM.%2BR.%26aulast%3DSmith%26aufirst%3DJ.%2BC.%26aulast%3DKasson%26aufirst%3DP.%2BM.%26aulast%3Dvan%2Bder%2BSpoel%26aufirst%3DD.%26aulast%3DHess%26aufirst%3DB.%26aulast%3DLindahl%26aufirst%3DE.%26atitle%3DGromacs%25204.5%253A%2520A%2520High-Throughput%2520and%2520Highly%2520Parallel%2520Open%2520Source%2520Molecular%2520Simulation%2520Toolkit%26jtitle%3DBioinformatics%26date%3D2013%26volume%3D29%26spage%3D845%26epage%3D854%26doi%3D10.1093%2Fbioinformatics%2Fbtt055" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wiederstein, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sippl, M. J.</span></span> <span> </span><span class="NLM_article-title">Prosa-Web: Interactive Web Service for the Recognition of Errors in Three-Dimensional Structures of Proteins</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>35</i></span>,  <span class="NLM_fpage">W407</span>– <span class="NLM_lpage">410</span>, <span class="refDoi"> DOI: 10.1093/nar/gkm290</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=10.1093%2Fnar%2Fgkm290" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=17517781" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=1%3ACAS%3A280%3ADC%252BD2svhsVKjug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2007&pages=W407-410&author=M.+Wiedersteinauthor=M.+J.+Sippl&title=Prosa-Web%3A+Interactive+Web+Service+for+the+Recognition+of+Errors+in+Three-Dimensional+Structures+of+Proteins&doi=10.1093%2Fnar%2Fgkm290"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">ProSA-web: interactive web service for the recognition of errors in three-dimensional structures of proteins</span></div><div class="casAuthors">Wiederstein Markus; Sippl Manfred J</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic acids research</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">Web Server issue</span>),
    <span class="NLM_cas:pages">W407-10</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">A major problem in structural biology is the recognition of errors in experimental and theoretical models of protein structures.  The ProSA program (Protein Structure Analysis) is an established tool which has a large user base and is frequently employed in the refinement and validation of experimental protein structures and in structure prediction and modeling.  The analysis of protein structures is generally a difficult and cumbersome exercise.  The new service presented here is a straightforward and easy to use extension of the classic ProSA program which exploits the advantages of interactive web-based applications for the display of scores and energy plots that highlight potential problems spotted in protein structures.  In particular, the quality scores of a protein are displayed in the context of all known protein structures and problematic parts of a structure are shown and highlighted in a 3D molecule viewer.  The service specifically addresses the needs encountered in the validation of protein structures obtained from X-ray analysis, NMR spectroscopy and theoretical calculations.  ProSA-web is accessible at https://prosa.services.came.sbg.ac.at.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSKn5zO1OOtqL5oR8RchjrrfW6udTcc2eZzcmbAoYCpcbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2svhsVKjug%253D%253D&md5=e2aba9fe44147a9048f37c77306edea6</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkm290&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkm290%26sid%3Dliteratum%253Aachs%26aulast%3DWiederstein%26aufirst%3DM.%26aulast%3DSippl%26aufirst%3DM.%2BJ.%26atitle%3DProsa-Web%253A%2520Interactive%2520Web%2520Service%2520for%2520the%2520Recognition%2520of%2520Errors%2520in%2520Three-Dimensional%2520Structures%2520of%2520Proteins%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2007%26volume%3D35%26spage%3DW407%26epage%3D410%26doi%3D10.1093%2Fnar%2Fgkm290" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Laskowski, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MacArthur, M. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moss, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thornton, J. M.</span></span> <span> </span><span class="NLM_article-title">Procheck: A Program to Check the Stereochemical Quality of Protein Structures</span>. <i>J. Appl. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">1993</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">283</span>– <span class="NLM_lpage">291</span>, <span class="refDoi"> DOI: 10.1107/S0021889892009944</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=10.1107%2FS0021889892009944" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=1%3ACAS%3A528%3ADyaK3sXit12lurY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=1993&pages=283-291&author=R.+A.+Laskowskiauthor=M.+W.+MacArthurauthor=D.+S.+Mossauthor=J.+M.+Thornton&title=Procheck%3A+A+Program+to+Check+the+Stereochemical+Quality+of+Protein+Structures&doi=10.1107%2FS0021889892009944"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">PROCHECK: a program to check the stereochemical quality of protein structures</span></div><div class="casAuthors">Laskowski, Roman A.; MacArthur, Malcolm W.; Moss, David S.; Thornton, Janet M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">283-91</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    </div><div class="casAbstract">The PROCHECK suite of programs provides a detailed check on the stereochem. of a protein structure.  Its outputs comprise a no. of plots in PostScript format and a comprehensive residue-by-residue listing.  These give an assessment of the overall quality of the structure as compared with well-refined structures of the same resoln. and also highlight regions that may need further investigation.  The PROCHECK programs are useful for assessing the quality not only of protein structures in the process of being solved but also of existing structures and of those being modeled on known structures.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrCe5FgJf4KH7Vg90H21EOLACvtfcHk0lh6XYJrEL9G4g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3sXit12lurY%253D&md5=fb69fc4410cd716aaaa7cc0db06b3ed2</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1107%2FS0021889892009944&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0021889892009944%26sid%3Dliteratum%253Aachs%26aulast%3DLaskowski%26aufirst%3DR.%2BA.%26aulast%3DMacArthur%26aufirst%3DM.%2BW.%26aulast%3DMoss%26aufirst%3DD.%2BS.%26aulast%3DThornton%26aufirst%3DJ.%2BM.%26atitle%3DProcheck%253A%2520A%2520Program%2520to%2520Check%2520the%2520Stereochemical%2520Quality%2520of%2520Protein%2520Structures%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D1993%26volume%3D26%26spage%3D283%26epage%3D291%26doi%3D10.1107%2FS0021889892009944" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suzek, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shoemaker, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gindulyte, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bryant, S. H.</span></span> <span> </span><span class="NLM_article-title">Pubchem Bioassay: 2014 Update</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>42</i></span>,  <span class="NLM_fpage">D1075</span>– <span class="NLM_lpage">1082</span>, <span class="refDoi"> DOI: 10.1093/nar/gkt978</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=10.1093%2Fnar%2Fgkt978" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=24198245" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=1%3ACAS%3A528%3ADC%252BC2cXoslaq" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2014&pages=D1075-1082&author=Y.+Wangauthor=T.+Suzekauthor=J.+Zhangauthor=J.+Wangauthor=S.+Heauthor=T.+Chengauthor=B.+A.+Shoemakerauthor=A.+Gindulyteauthor=S.+H.+Bryant&title=Pubchem+Bioassay%3A+2014+Update&doi=10.1093%2Fnar%2Fgkt978"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">PubChem BioAssay: 2014 update</span></div><div class="casAuthors">Wang, Yanli; Suzek, Tugba; Zhang, Jian; Wang, Jiyao; He, Siqian; Cheng, Tiejun; Shoemaker, Benjamin A.; Gindulyte, Asta; Bryant, Stephen H.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">D1</span>),
    <span class="NLM_cas:pages">D1075-D1082</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">PubChem's BioAssay database (http://pubchem.ncbi.nlm.nih.gov) is a public repository for archiving biol. tests of small mols. generated through high-throughput screening expts., medicinal chem. studies, chem. biol. research and drug discovery programs.  In addn., the BioAssay database contains data from high-throughput RNA interference screening aimed at identifying crit. genes responsible for a biol. process or disease condition.  The mission of PubChem is to serve the community by providing free and easy access to all deposited data.  To this end, PubChem BioAssay is integrated into the National Center for Biotechnol. Information retrieval system, making them searchable by Entrez queries and cross-linked to other biomedical information archived at National Center for Biotechnol. Information.  Moreover, PubChem BioAssay provides web-based and programmatic tools allowing users to search, access and analyze bioassay test results and metadata.  In this work, we provide an update for the PubChem BioAssay resource, such as information content growth, new developments supporting data integration and search, and the recently deployed PubChem Upload to streamline chem. structure and bioassay submissions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo1tnO28g86-rVg90H21EOLACvtfcHk0lhkIx2fyOZTag"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXoslaq&md5=66caa053197bcaf40b49535b0a3e0c27</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkt978&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkt978%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DY.%26aulast%3DSuzek%26aufirst%3DT.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DHe%26aufirst%3DS.%26aulast%3DCheng%26aufirst%3DT.%26aulast%3DShoemaker%26aufirst%3DB.%2BA.%26aulast%3DGindulyte%26aufirst%3DA.%26aulast%3DBryant%26aufirst%3DS.%2BH.%26atitle%3DPubchem%2520Bioassay%253A%25202014%2520Update%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2014%26volume%3D42%26spage%3DD1075%26epage%3D1082%26doi%3D10.1093%2Fnar%2Fgkt978" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Trott, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olson, A. J.</span></span> <span> </span><span class="NLM_article-title">Autodock Vina: Improving the Speed and Accuracy of Docking with a New Scoring Function, Efficient Optimization, and Multithreading</span>. <i>J. Comput. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>31</i></span>,  <span class="NLM_fpage">455</span>– <span class="NLM_lpage">461</span>, <span class="refDoi"> DOI: 10.1002/jcc.21334</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=10.1002%2Fjcc.21334" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=19499576" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsFGnur3O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2010&pages=455-461&author=O.+Trottauthor=A.+J.+Olson&title=Autodock+Vina%3A+Improving+the+Speed+and+Accuracy+of+Docking+with+a+New+Scoring+Function%2C+Efficient+Optimization%2C+and+Multithreading&doi=10.1002%2Fjcc.21334"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">AutoDock Vina: Improving the speed and accuracy of docking with a new scoring function, efficient optimization, and multithreading</span></div><div class="casAuthors">Trott, Oleg; Olson, Arthur J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">455-461</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">AutoDock Vina, a new program for mol. docking and virtual screening, is presented.  AutoDock Vina achieves an approx. 2 orders of magnitude speed-up compared with the mol. docking software previously developed in the authors' lab (AutoDock 4), while also significantly improving the accuracy of the binding mode predictions, judging by the authors' tests on the training set used in AutoDock 4 development.  Further speed-up is achieved from parallelism, by using multithreading on multicore machines.  AutoDock Vina automatically calcs. the grid maps and clusters the results in a way transparent to the user.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpLecOIkUxIlrVg90H21EOLACvtfcHk0lhkIx2fyOZTag"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsFGnur3O&md5=c6974af8a1235f7aa09918d3e6f70dc4</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1002%2Fjcc.21334&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.21334%26sid%3Dliteratum%253Aachs%26aulast%3DTrott%26aufirst%3DO.%26aulast%3DOlson%26aufirst%3DA.%2BJ.%26atitle%3DAutodock%2520Vina%253A%2520Improving%2520the%2520Speed%2520and%2520Accuracy%2520of%2520Docking%2520with%2520a%2520New%2520Scoring%2520Function%252C%2520Efficient%2520Optimization%252C%2520and%2520Multithreading%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2010%26volume%3D31%26spage%3D455%26epage%3D461%26doi%3D10.1002%2Fjcc.21334" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hsu, K. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, J. M.</span></span> <span> </span><span class="NLM_article-title">iGEMDOCK: A Graphical Environment of Enhancing GEMDOCK Using Pharmacological Interactions and Post-Screening Analysis</span>. <i>BMC Bioinf.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>12</i></span> (<span class="NLM_issue">Suppl. 1</span>),  <span class="NLM_fpage">S33</span>, <span class="refDoi"> DOI: 10.1186/1471-2105-12-S1-S33</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=10.1186%2F1471-2105-12-S1-S33" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;key=21342564" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2011&pages=S33&issue=Suppl.+1&author=K.+C.+Hsuauthor=Y.+F.+Chenauthor=S.+R.+Linauthor=J.+M.+Yang&title=iGEMDOCK%3A+A+Graphical+Environment+of+Enhancing+GEMDOCK+Using+Pharmacological+Interactions+and+Post-Screening+Analysis&doi=10.1186%2F1471-2105-12-S1-S33"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1186%2F1471-2105-12-S1-S33&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1471-2105-12-S1-S33%26sid%3Dliteratum%253Aachs%26aulast%3DHsu%26aufirst%3DK.%2BC.%26aulast%3DChen%26aufirst%3DY.%2BF.%26aulast%3DLin%26aufirst%3DS.%2BR.%26aulast%3DYang%26aufirst%3DJ.%2BM.%26atitle%3DiGEMDOCK%253A%2520A%2520Graphical%2520Environment%2520of%2520Enhancing%2520GEMDOCK%2520Using%2520Pharmacological%2520Interactions%2520and%2520Post-Screening%2520Analysis%26jtitle%3DBMC%2520Bioinf.%26date%3D2011%26volume%3D12%26issue%3DSuppl.%25201%26spage%3DS33%26doi%3D10.1186%2F1471-2105-12-S1-S33" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">DeLano, W. L.</span></span> <i>Pymol Molecular Graphics System</i>, version <span class="NLM_edition">1.5.0.4</span>; <span class="NLM_publisher-name">Schrödinger, LLC.</span>, <span class="NLM_year">2004</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2004&author=W.+L.+DeLano&title=Pymol+Molecular+Graphics+System"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26aulast%3DDeLano%26aufirst%3DW.%2BL.%26jtitle%3DPymol%2520Molecular%2520Graphics%2520System%26pub%3DSchr%25C3%25B6dinger%252C%2520LLC%26date%3D2004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3F9V" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3F9V','PDB','3F9V'); return false;">PDB: 3F9V</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1LTL" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1LTL','PDB','1LTL'); return false;">PDB: 1LTL</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i60"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b01783">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_27208"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a class="ext-link" href="/doi/10.1021/acs.jmedchem.9b01783?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acs.jmedchem.9b01783</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Modeling and simulation information, analysis of trajectories, and supplementary figures and tables (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b01783/suppl_file/jm9b01783_si_001.pdf">PDF</a>)</p></li><li><p class="inline">NMR spectra (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b01783/suppl_file/jm9b01783_si_002.pdf">PDF</a>)</p></li><li><p class="inline">Molecular formula strings and some data (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b01783/suppl_file/jm9b01783_si_003.xlsx">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01783/suppl_file/jm9b01783_si_001.pdf">jm9b01783_si_001.pdf (995.44 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01783/suppl_file/jm9b01783_si_002.pdf">jm9b01783_si_002.pdf (2.98 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01783/suppl_file/jm9b01783_si_003.xlsx">jm9b01783_si_003.xlsx (12.77 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.9b01783&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.9b01783%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2020.63.issue-6" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.9b01783" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                6MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"677ac4626f863de3","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
