
<html lang="en"     class="pb-page"  data-request-id="f3e872a0-48c9-42ee-9dc3-90bd613b0a2a"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;article:article:10.1021/jm051256o;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;issue:issue:10.1021/jmcmar.2006.49.issue-21;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="dc.Title" content="Extra Precision Glide:  Docking and Scoring Incorporating a Model of Hydrophobic Enclosure for Protein−Ligand Complexes" /></meta><meta name="dc.Creator" content="Richard A.  Friesner,*" /></meta><meta name="dc.Creator" content="Robert B.  Murphy,†" /></meta><meta name="dc.Creator" content="Matthew P.  Repasky,†" /></meta><meta name="dc.Creator" content="Leah L.  Frye,‡" /></meta><meta name="dc.Creator" content="Jeremy R.  Greenwood,†" /></meta><meta name="dc.Creator" content="Thomas A.  Halgren,†" /></meta><meta name="dc.Creator" content="Paul C.  Sanschagrin,† and" /></meta><meta name="dc.Creator" content="Daniel T.  Mainz†" /></meta><meta name="dc.Description" content="A novel scoring function to estimate protein−ligand binding affinities has been developed and implemented as the Glide 4.0 XP scoring function and docking protocol. In addition to unique water deso..." /></meta><meta name="Description" content="A novel scoring function to estimate protein−ligand binding affinities has been developed and implemented as the Glide 4.0 XP scoring function and docking protocol. In addition to unique water deso..." /></meta><meta name="dc.Publisher" content=" American Chemical Society " /></meta><meta name="dc.Date" scheme="WTN8601" content="September 23, 2006" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/jm051256o" /></meta><meta name="dc.Identifier" scheme="crossover-key" content="JMCMAr-49-21-6177" /></meta><meta name="dc.Identifier" scheme="pii" content="S0022-2623(05)01256-2" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Copyright © 2006 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Fjm051256o" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Fjm051256o" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Fjm051256o" /></link>
        
    
    

<title>Extra Precision Glide:  Docking and Scoring Incorporating a Model of Hydrophobic Enclosure for Protein−Ligand Complexes | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/jm051256o" /></meta><meta property="og:title" content="Extra Precision Glide:  Docking and Scoring Incorporating a Model of Hydrophobic Enclosure for Protein−Ligand Complexes" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2006/jmcmar.2006.49.issue-21/jm051256o/production/images/large/jm051256on00001.jpeg" /></meta><meta property="og:description" content=" A novel scoring function to estimate protein−ligand binding affinities has been developed and implemented as the Glide 4.0 XP scoring function and docking protocol. In addition to unique water desolvation energy terms, protein−ligand structural motifs leading to enhanced binding affinity are included:  (1) hydrophobic enclosure where groups of lipophilic ligand atoms are enclosed on opposite faces by lipophilic protein atoms, (2) neutral−neutral single or correlated hydrogen bonds in a hydrophobically enclosed environment, and (3) five categories of charged−charged hydrogen bonds. The XP scoring function and docking protocol have been developed to reproduce experimental binding affinities for a set of 198 complexes (RMSDs of 2.26 and 1.73 kcal/mol over all and well-docked ligands, respectively) and to yield quality enrichments for a set of fifteen screens of pharmaceutical importance. Enrichment results demonstrate the importance of the novel XP molecular recognition and water scoring in separating active and inactive ligands and avoiding false positives.  " /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/jm051256o"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/jm051256o">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-14/achre4.2021.54.issue-14/20210720/achre4.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-29/aamick.2021.13.issue-29/20210728/aamick.2021.13.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-14/acncdm.2021.12.issue-14/20210721/acncdm.2021.12.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-7/amlcef.2021.3.issue-7/20210705/amlcef.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-29/acsodf.2021.6.issue-29/20210727/acsodf.2021.6.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-29/ascecg.2021.9.issue-29/20210726/ascecg.2021.9.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-29/ancham.2021.93.issue-29/20210727/ancham.2021.93.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-29/bichaw.2021.60.issue-29/20210727/bichaw.2021.60.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-27/cgeabj.2021.99.issue-27/20210726-01/cgeabj.2021.99.issue-27.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-7/cgdefu.2021.21.issue-7/20210707/cgdefu.2021.21.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-14/esthag.2021.55.issue-14/20210720/esthag.2021.55.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-29/iecred.2021.60.issue-29/20210728/iecred.2021.60.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-14/inocaj.2021.60.issue-14/20210719/inocaj.2021.60.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-29/jafcau.2021.69.issue-29/20210728/jafcau.2021.69.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-29/jacsat.2021.143.issue-29/20210728/jacsat.2021.143.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-7/jamsef.2021.32.issue-7/20210707/jamsef.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-29/langd5.2021.37.issue-29/20210727/langd5.2021.37.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-7/mpohbp.2021.18.issue-7/20210705/mpohbp.2021.18.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Fjm051256o&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Fjm051256o&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/jm051256o&amp;href=/doi/10.1021/jm051256o" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2006</span><span class="cit-fg-volume">, 49</span><span class="cit-fg-issue">, 21</span><span class="cit-fg-pageRange">, 6177-6196</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/49/21" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/jm060580w" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/jm051076e" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Extra Precision Glide:  Docking and Scoring Incorporating a Model of Hydrophobic Enclosure for Protein−Ligand Complexes</span></h1><ul class="loa non-jats-loa"><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Richard+A.++Friesner">Richard A. Friesner</a></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Robert+B.++Murphy">Robert B. Murphy</a></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Matthew+P.++Repasky">Matthew P. Repasky</a></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Leah+L.++Frye">Leah L. Frye</a></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Jeremy+R.++Greenwood">Jeremy R. Greenwood</a></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Thomas+A.++Halgren">Thomas A. Halgren</a></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Paul+C.++Sanschagrin">Paul C. Sanschagrin</a></span></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Daniel+T.++Mainz">Daniel T. Mainz</a></span></span></li><span class="comma-separator"></span></ul><div class="affiliations"><div class="view-affs"><span class="title">View Author Information</span><i class="caret-icon icon-angle-down"></i></div><div class="all-aff-infos hlFld-Affiliation"><div class="aff-info"><span class="aff-text">Department of Chemistry, Columbia University, New York, New York 10027, Schrödinger, Limited Liability Company, 120 West 45th Street, New York, New York 10036, Schrödinger, Limited Liability Company, 101 SW Main Street, Portland, Oregon 97204 </span></div></div></div><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="JMCMAr" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Fjm051256o&amp;href=/doi/10.1021%2Fjm051256o" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2006</span></span><span class="cit-volume">, 49</span><span class="cit-issue">, 21</span><span class="cit-pageRange">, 6177–6196</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">September 23, 2006</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>16 December 2005</li><li><span class="item_label"><b>Published</b> online</span>23 September 2006</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 1 October 2006</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/jm051256o" title="DOI URL">https://doi.org/10.1021/jm051256o</a></div><div class="article_header-article-copyright"><strong>Copyright © 2006 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D6177%26pageCount%3D20%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DRichard%2BA.%2BFriesner%252C%2BRobert%2BB.%2BMurphy%252C%2BMatthew%2BP.%2BRepasky%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D49%26issueNum%3D21%26contentID%3Djm051256o%26title%3DExtra%2BPrecision%2BGlide%253A%25E2%2580%2589%2BDocking%2Band%2BScoring%2BIncorporating%2Ba%2BModel%2Bof%2BHydrophobic%2BEnclosure%2Bfor%2BProtein%25E2%2588%2592Ligand%2BComplexes%26numPages%3D20%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D6196%26publicationDate%3DOctober%2B2006">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/jm051256o"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">12226</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">3115</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/jm051256o" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Extra Precision Glide:  Docking and Scoring Incorporating a Model of Hydrophobic Enclosure for Protein−Ligand Complexes&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Richard&quot;,&quot;last_name&quot;:&quot;A. Friesner&quot;},{&quot;first_name&quot;:&quot;Robert&quot;,&quot;last_name&quot;:&quot;B. Murphy&quot;},{&quot;first_name&quot;:&quot;Matthew&quot;,&quot;last_name&quot;:&quot;P. Repasky&quot;},{&quot;first_name&quot;:&quot;Leah&quot;,&quot;last_name&quot;:&quot;L. Frye&quot;},{&quot;first_name&quot;:&quot;Jeremy&quot;,&quot;last_name&quot;:&quot;R. Greenwood&quot;},{&quot;first_name&quot;:&quot;Thomas&quot;,&quot;last_name&quot;:&quot;A. Halgren&quot;},{&quot;first_name&quot;:&quot;Paul&quot;,&quot;last_name&quot;:&quot;C. Sanschagrin&quot;},{&quot;first_name&quot;:&quot;Daniel&quot;,&quot;last_name&quot;:&quot;T. Mainz&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2006&quot;,&quot;month&quot;:&quot;09&quot;,&quot;day&quot;:&quot;23&quot;,&quot;issue&quot;:&quot;21&quot;,&quot;volume&quot;:&quot;49&quot;,&quot;pages&quot;:&quot;6177-6196&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/jm051256o&quot;},&quot;abstract&quot;:&quot;A novel scoring function to estimate protein−ligand binding affinities has been developed and implemented as the Glide 4.0 XP scoring function and docking protocol. In addition to unique water desolvation energy terms, protein−ligand structural motifs leading to enhanced binding affinity are included:  (1) hydrophobic enclosure where groups of lipophilic ligand atoms are enclosed on opposite faces by lipophilic protein atoms, (2) neutral−neutral single or correlated hydrogen bonds in a hydrophobically enclosed environment, and (3) five categories of charged−charged hydrogen bonds. The XP scoring function and docking protocol have been developed to reproduce experimental binding affinities for a set of 198 complexes (RMSDs of 2.26 and 1.73 kcal/mol over all and well-docked ligands, respectively) and to yield quality enrichments for a set of fifteen screens of pharmaceutical importance. Enrichment results demonstrate the importance of the novel XP molecular recognition and water scoring in separating active and&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Fjm051256o&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=jm051256o" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Fjm051256o&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=jm051256o" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Fjm051256o&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=jm051256o" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Fjm051256o&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=jm051256o&amp;href=/doi/10.1021/jm051256o" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/jm051256o" /></input><a href="/doi/pdf/10.1021/jm051256o" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (799 KB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/jm051256o&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Fjm051256o%26sid%3Dliteratum%253Aachs%26pmid%3D17034125%26genre%3Darticle%26aulast%3DFriesner%26date%3D2006%26atitle%3DExtra%2BPrecision%2BGlide%253A%25E2%2580%2589%2BDocking%2Band%2BScoring%2BIncorporating%2Ba%2BModel%2Bof%2BHydrophobic%2BEnclosure%2Bfor%2BProtein%25E2%2588%2592Ligand%2BComplexes%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D49%26issue%3D21%26spage%3D6177%26epage%3D6196%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (1)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292366" title="Ligands">Ligands</a>,</li><li><a href="/action/doSearch?ConceptID=291120" title="Hydrophobicity">Hydrophobicity</a>,</li><li><a href="/action/doSearch?ConceptID=291763" title="Noncovalent interactions">Noncovalent interactions</a>,</li><li><a href="/action/doSearch?ConceptID=291352" title="Screening assays">Screening assays</a>,</li><li><a href="/action/doSearch?ConceptID=292186" title="Receptors">Receptors</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/49/21" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2006/jmcmar.2006.49.issue-21/jm051256o/production/images/medium/jm051256on00001.gif" alt="" id="_i2" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2006/jmcmar.2006.49.issue-21/jm051256o/production/images/large/jm051256on00001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm051256o&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">A novel scoring function to estimate protein−ligand binding affinities has been developed and implemented as the Glide 4.0 XP scoring function and docking protocol. In addition to unique water desolvation energy terms, protein−ligand structural motifs leading to enhanced binding affinity are included:  (1) hydrophobic enclosure where groups of lipophilic ligand atoms are enclosed on opposite faces by lipophilic protein atoms, (2) neutral−neutral single or correlated hydrogen bonds in a hydrophobically enclosed environment, and (3) five categories of charged−charged hydrogen bonds. The XP scoring function and docking protocol have been developed to reproduce experimental binding affinities for a set of 198 complexes (RMSDs of 2.26 and 1.73 kcal/mol over all and well-docked ligands, respectively) and to yield quality enrichments for a set of fifteen screens of pharmaceutical importance. Enrichment results demonstrate the importance of the novel XP molecular recognition and water scoring in separating active and inactive ligands and avoiding false positives. </p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div class="extra-info-sec fnGroup"><div class="fnGroupItem" id="jm051256oAF1"><span><h4><span class="NLM_fn-symbol">*</span></h4></span><span><p class="last">  To whom correspondence should be addressed. Phone:  212-854-7606. Fax:  212-854-7454. E-mail:  <a href="/cdn-cgi/l/email-protection" class="__cf_email__" data-cfemail="16647f757e56757e737b3875797a637b747f773873726338">[email protected]</a></p></span></div><div class="fnGroupItem" id="jm051256oAF2"><span><h4><span class="NLM_fn-symbol">†</span></h4></span><span><p class="last">  Schrödinger, L.L.C., NY.</p></span></div><div class="fnGroupItem" id="jm051256oAF3"><span><h4><span class="NLM_fn-symbol">‡</span></h4></span><span><p class="last">  Schrödinger, L.L.C., OR.</p></span></div></div><div id="" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">1. Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_28706" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_28706" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">In two previous papers<sup><a onclick="showRef(event, 'jm051256ob00001 jm051256ob00002'); return false;" href="javascript:void(0);" class="ref jm051256ob00001 jm051256ob00002">1,2</a></sup> we have described the Glide high throughput docking program and provided performance benchmarks for docking and scoring capabilities. These results have established Glide as a competitive methodology in both areas.<sup><a onclick="showRef(event, 'jm051256ob00003 jm051256ob00004 jm051256ob00005'); return false;" href="javascript:void(0);" class="ref jm051256ob00003 jm051256ob00004 jm051256ob00005">2-5</a></sup> However, it is clear from enrichment results (ref 2) that there remains substantial room for improvement in separating “active” from “inactive” compounds. In this paper we outline and present results obtained from significantly enhanced sampling methods and scoring functions, hereafter collectively referred to as “extra-precision” (XP) Glide. The key novel features characterizing XP Glide scoring are (1) the application of large desolvation penalties to both ligand and protein polar and charged groups in appropriate cases and (2) the identification of specific structural motifs that provide exceptionally large contributions to enhanced binding affinity. Accurate assignment of these desolvation penalties and molecular recognition motifs requires an expanded sampling methodology for optimal performance. Thus, XP Glide represents a single, coherent approach in which the sampling algorithms and the scoring function have been optimized simultaneously. </div><div class="NLM_p">The goal of the XP Glide methodology is to semiquantitatively rank the ability of candidate ligands to bind to <i>a specified conformation of the protein receptor.</i> Because of the rigid receptor approximation utilized in Glide and other high throughput docking programs, ligands that exhibit significant steric clashes with the specified receptor conformation cannot be expected to achieve good scores, even if they in reality bind effectively to an alternative conformation of the same receptor. Such ligands may be thought of as unable to “fit” into that specified conformation of the protein. For docking protocols to function effectively within the rigid-receptor approximation, some ability to deviate from the restrictions of the hard wall van der Waals potential of the receptor conformation used in docking must be built into the potential energy function employed to predict the ligand binding mode. In XP and SP Glide, this is accomplished by scaling the van der Waals radii of nonpolar protein and/or ligand atoms; scaling the vdW radii effectively introduces a modest “induced fit” effect. However, it is clear that there are many cases in which a reasonable degree of scaling will not enable the ligand to be docked correctly. For example, a side chain in a rotamer state that is very different from that of the native protein−ligand complex may block the ligand atoms from occupying their preferred location in the binding pocket. There will always be borderline situations, but in practice we have found it possible to classify the great majority of cases in cross-docking experiments as either “fitting” or “not fitting”. The former are expected to be properly ranked by XP Glide (within the limitation of noise in the scoring function), while the latter require an induced-fit protocol<sup><a onclick="showRef(event, 'jm051256ob00006 jm051256ob00007'); return false;" href="javascript:void(0);" class="ref jm051256ob00006 jm051256ob00007">6,7</a></sup> to correctly assess their binding affinity.<sup>4</sup> In the present paper, we focus on complexes where the ligand fits appropriately into the receptor, as judged by two factors:  (1) the ability to make key hydrogen bonding and hydrophobic contacts and (2) the ability to achieve a reasonable root-mean-square deviation (RMSD), as compared to the native complex or as obtained by analogy with the native complex of a related ligand. Comparison by analogy is often necessary when dealing with a large dataset of active ligands, only a few of which may have available crystal structures. </div><div class="NLM_p">Our discussion of XP Glide is divided into four different sections. First, in section 2, we describe the novel terms leading to enhanced binding affinity that have been introduced to account for our observations with regard to protein−ligand binding in a wide range of systems. The origin of these terms lies in the theoretical physical chemistry of protein−ligand interactions; however, developing heuristic mathematical representations that can be used effectively in an empirical scoring function, taking into account imperfections in structures due to the rigid receptor approximation and/or limitations of the docking algorithm, requires extensive analysis of, and fitting <a name="jm004902106178" id="jm004902106178"></a> to, experimental data. Key aspects of this analysis, along with illustrative examples, are provided in section 2 in an effort to provide physical insight as well as formal justification for the model. In developing XP Glide, we have attempted to identify the principal driving forces and structural motifs for achieving significant binding affinity contributions with specific protein−ligand interactions, above and beyond the generic terms that have appeared repeatedly in prior scoring functions. We have found that a relatively small number of such motifs are dominant over a wide range of test cases; the ability to automatically recognize these motifs, and assign binding affinity contributions, potentially represents an advance in the modeling of protein−ligand interactions based on an empirical scheme. </div><div class="NLM_p">In section 3, we evaluate the performance of our methodology in self-docking, with regard to both the ability to generate the correct binding mode of the complex and the prediction of binding affinity, using docked XP structures for the complexes. In section 4, the performance of the scoring function in enrichment studies (ability to rank known active compounds ahead of random database ligands) for a substantial number of targets, containing qualitatively different types of active sites, is investigated. Our treatment of the data differs significantly from what has generally prevailed in previous papers in the literature; in evaluating scoring accuracy, we distinguish cases where there are significant errors in structural prediction, as opposed to systems where the structural prediction is reasonably good, but the scoring function fails to assign the appropriate binding affinity. By using only well-docked structures to parametrize and assess scoring functions, a way forward toward a globally accurate method, in which multiple structures are employed in docking and/or induced fit methods are utilized to directly incorporate protein flexibility, is facilitated. </div><div class="NLM_p">The parameterization of XP Glide is carried out using a large and diverse training set comprising 15 different receptors and between 4 and 106 well-docked ligands per receptor. A separately developed test set incorporating four new receptors, and additional ligands for two receptors already in the training set, is also defined. All of the receptor and ligand data is publicly available (as is our decoy set, which has been posted on the Schrodinger Web site and is freely available for downloading) and we provide extensive references documenting the origin of each ligand. The results reported below have been obtained with the Glide 4.0 release. </div><div class="NLM_p last">The development of data sets suitable for the analysis described above is highly labor intensive; consequently, our current test set is too small to draw robust conclusions, and the results reported herein must be regarded as preliminary. While the test set results are encouraging with regard to demonstration of a respectable degree of transferability, a rigorous assessment of the performance to be expected on a novel receptor will have to be performed in future publications. Nevertheless, qualitative and consistent improvement in the results for both training and test set, at least as compared to the alternative scoring functions available in Glide, is demonstrated. Finally, in the conclusion, we summarize our results and discuss future directions. </div></div><div id="" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i4">2. Glide XP Scoring Function</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_76532" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_76532" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">The major potential contributors to protein−ligand binding affinity can readily be enumerated as follows: </div><div class="NLM_p"><b>(1) Displacement of Waters by the Ligand from “Hydrophobic Regions” of </b><b>the Protein Active Site.</b> Displacement of these waters into the bulk by a suitably designed ligand group will lower the overall free energy of the system. Waters in such regions may not be able to make the full complement of hydrogen bonds that would be available in solution. There are also entropic considerations; if a water molecule is restricted in mobility in the protein cavity, release into solvent via ligand-induced displacement will result in an entropy gain. As one ligand releases many water molecules, this term will contribute favorably to the free energy. Replacement of a water molecule by a hydrophobic group of the ligand retains favorable van der Waals interactions, while eliminating issues concerning the availability of hydrogen bonds. Transfer of a hydrophobic moiety on the ligand from solvent exposure to a hydrophobic pocket can also contribute favorably to binding by withdrawing said hydrophobic group from the bulk solution. </div><div class="NLM_p"><b>(2) Protein</b><b>−</b><b>Ligand Hydrogen-Bonding Interactions, as well as Other </b><b>Strong Electrostatic Interactions Such as Salt Bridges.</b> In making these interactions, the ligand displaces waters in the protein cavity, which can lead to favorable entropic terms of the type discussed above in (1). Contributions to binding affinity (favorable or unfavorable) will also depend on the quality and type of hydrogen bonds formed, net electrostatic interaction energies (possibly including long range effects, although these generally are considered small and typically are neglected in empirical scoring functions), and specialized features of the hydrogen-bonding geometry, such as bidendate salt bridge formation by groups such as carboxylates or guanidium ions. Finally, differences in the interactions of the displaced waters, as compared to the ligand groups replacing them, with the protein environment proximate to the hydrogen bond, can have a major effect on binding affinity, as is discussed in greater detail below. </div><div class="NLM_p"><b>(3) Desolvation Effects.</b> Polar or charged groups of either the ligand or protein that formerly were exposed to solvent may become desolvated by being placed in contact with groups to which they cannot hydrogen bond effectively. In contrast to the two terms described above, such effects can only reduce binding affinity. </div><div class="NLM_p"><b>(4) Entropic Effects Due to the Restriction on Binding of the Motion of </b><b>Flexible Protein or Ligand Groups.</b> The largest contributions are due to restriction of ligand translational/orientational motion and protein and ligand torsions, but modification of vibrational entropies can also contribute. As in the case of desolvation terms, such effects will serve exclusively to reduce binding affinity. </div><div class="NLM_p"><b>(5) Metal−Ligand Interactions.</b> Specialized terms are needed to describe the interaction of the ligand with metal ions. We shall defer the discussion of metal-specific parameterization to another publication, as this is a complex subject in its own right, requiring considerable effort to treat in a robust fashion. </div><div class="NLM_p">A large number of empirical scoring functions for predicting protein−ligand binding affinities have been developed.<sup><a onclick="showRef(event, 'jm051256ob00008 jm051256ob00009 jm051256ob00010 jm051256ob00011 jm051256ob00012 jm051256ob00013 jm051256ob00014 jm051256ob00015 jm051256ob00016 jm051256ob00017 jm051256ob00018 jm051256ob00019'); return false;" href="javascript:void(0);" class="ref jm051256ob00008 jm051256ob00009 jm051256ob00010 jm051256ob00011 jm051256ob00012 jm051256ob00013 jm051256ob00014 jm051256ob00015 jm051256ob00016 jm051256ob00017 jm051256ob00018 jm051256ob00019">8-19</a></sup> While differing somewhat in detail, these scoring functions are broadly similar. A representative example, the ChemScore<sup>8</sup> scoring function, is discussed in our comments below, though similar comments would apply to many of the other scoring functions cited in refs 8−19. We briefly summarize how ChemScore treats the first four potential contributors to the binding affinity presented above: </div><div class="NLM_p">(1) ChemScore<sup>8</sup> contains a hydrophobic atom−atom pair energy term of the form<a class="ref internalNav" href="#jm051256oe00001" aria-label=""></a><span class="NLM_disp-formula" id="jm051256oe00001"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2006/jmcmar.2006.49.issue-21/jm051256o/production/images/medium/jm051256oe00001.gif" alt="" id="_i5" /></img></span></div><div class="NLM_p">Here, <i>i</i> and <i>j</i> refer to lipophilic atoms, generally carbon, and <i>f(r</i><i><sub>ij</sub></i><i>)</i> is a linear function of the interatomic distance, <i>r</i><i><sub>ij</sub></i>. For <i>r</i><i><sub>ij</sub></i> less than the sum of the atomic vdW radii plus 0.5 Å, <i>f</i> is 1.0. <a name="jm004902106179" id="jm004902106179"></a> Between this value and the sum of atomic vdW radii plus 3.0 Å, <i>f</i> ramps linearly from 1.0 to zero. Beyond the sum of atomic vdW radii plus 3.0 Å, <i>f </i>is assigned a value of zero<i>.</i></div><div class="NLM_p">This term heuristically represents the displacement of waters from hydrophobic regions by lipophilic ligand atoms. Numerous close contacts between the lipophilic ligand and protein atoms indicate that poorly solvated waters have been displaced by lipophilic atoms of the ligand that themselves were previously exposed to water. The resulting segregation of lipophilic atoms, and concomitant release of waters from the active site, lowers the free energy via the hydrophobic effect, which is approximately captured by the pair scoring function above. Terms based on contact of the hydrophobic surface area of the protein and ligand, while differing in details, essentially measure the same free energy change and have a similar physical and mathematical basis. </div><div class="NLM_p">Various parameterizations of the atom−atom pair term have been attempted, including efforts such as PLP,<sup>9</sup> in which every pair of atom types is assigned a different empirical pair potential. However, it is unclear whether this more detailed parameterization yields increased accuracy in predicting binding affinities. A key issue is whether a correct description of the hydrophobic effect can be achieved in all cases by using a linearly additive, pairwise decomposable functional form. </div><div class="NLM_p">(2) ChemScore evaluates protein−ligand hydrogen-bond quality based on geometric criteria, but otherwise does not distinguish between different types of hydrogen bonds or among the differing protein environments in which those hydrogen bonds are embedded. </div><div class="NLM_p">(3) ChemScore does not treat desolvation effects. </div><div class="NLM_p">(4) ChemScore uses a simple rotatable-bond term to treat conformation entropy effects arising from restricted motion of the ligand. </div><div class="NLM_p">The new XP Glide scoring function starts from the “standard” terms discussed above, though the functional form of the first three terms have been significantly revised and the parameterization of all terms is specific to our scoring function. In the remainder of this section, the functional form and physical rationale for the novel scoring terms we have developed are described with examples from pharmaceutically relevant test cases provided to illustrate how the various terms arise from consideration of the underlying physical theory and experimental data. </div><div class="NLM_p"><b>Form of the XP Glide Scoring Function.</b> The XP Glide scoring function is presented in eq 2. The principal terms that favor binding are presented in eq 3, while those that hinder binding are presented in eq 4. A description of each of the following terms besides <i>E</i><sub>hb_pair</sub> and <i>E</i><sub>phobic_pair</sub>, which are standard ChemScore-like hydrogen bond and lipophilic pair terms, respectively, follows.<a class="ref internalNav" href="#jm051256oe00002" aria-label=""></a><span class="NLM_disp-formula" id="jm051256oe00002"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2006/jmcmar.2006.49.issue-21/jm051256o/production/images/medium/jm051256oe00002.gif" alt="" id="_i6" /></img></span><a class="ref internalNav" href="#jm051256oe00003" aria-label=""></a><span class="NLM_disp-formula" id="jm051256oe00003"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2006/jmcmar.2006.49.issue-21/jm051256o/production/images/medium/jm051256oe00003.gif" alt="" id="_i7" /></img></span><a class="ref internalNav" href="#jm051256oe00004" aria-label=""></a><span class="NLM_disp-formula" id="jm051256oe00004"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2006/jmcmar.2006.49.issue-21/jm051256o/production/images/medium/jm051256oe00004.gif" alt="" id="_i8" /></img></span></div><div class="NLM_p"><b>Improved Model of Hydrophobic Interactions:  Hydrophobic Enclosure </b><b>(</b><b><i>E</i></b><b><sub>hyd_enclosure</sub></b><b>).</b> The ChemScore atom−atom pair function, <i>E</i><sub>phobic_pair</sub> described above, assigns scores to lipophilic ligand atoms based on summation over a pair function, each term of which depends on the interatomic distance between a ligand atom and a neighboring lipophilic protein atom. This clearly captures a significant component of the physics of the hydrophobic component of ligand binding. It is assumed that the displacement of water molecules from areas with many proximal lipophilic protein atoms will result in lower free energy than displacement from areas with fewer such atoms. As a crude example, it is clear that if the ligand is placed in an active-site cavity, as opposed to on the surface of the protein, the lipophilic atoms of the ligand are likely to receive better scores. If they are located in a “hydrophobic pocket” of the protein, scores should be better than in a location surrounded primarily by polar or charged groups. Furthermore, these improved scores are likely to be correlated with improvements in ligand binding affinity. </div><div class="NLM_p">However, a function dependent only on the sum of interatomic pair functions is potentially inadequately sensitive to details of the local geometry of the lipophilic protein atoms relative to the ligand lipophilic atom in question. As an example, consider the two model distributions shown in Figure 1<a data-tab="pane-pcw-Figures" href="#jm051256of00001" id="" class=" internalNav"></a>. In one case (A), a lipophilic ligand group is placed at a hydrophobic “wall” with lipophilic protein atoms on only a single face of the hydrophobic group. In the second case (B), the lipophilic ligand group is placed into a tight pocket, with lipophilic protein atoms contacting the two faces of the ligand group. As suggested above, one would normally expect a larger contribution to binding in the second case than in the first. However, this does not fully settle the question, which at root is whether the atom−atom pair contribution for a given ligand-atom/protein-atom distance should be identical when the ligand atom is enclosed by protein hydrophobic atoms, as opposed to when it is not, or whether there can be expected to be nonadditive effects. <div id="_i9"><figure id="jm051256of00001" data-index="1" class="article__inlineFigure"><h2 class="fig-label"></h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2006/jmcmar.2006.49.issue-21/jm051256o/production/images/medium/jm051256of00001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2006/jmcmar.2006.49.issue-21/jm051256o/production/images/large/jm051256of00001.jpeg" id="_i10" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last"> Figure 1 Schematic of a ligand group interacting with two distinct hydrophobic environments:  above a hydrophobic “plane” (A) and enclosed in a hydrophobic cavity (B).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2006/jmcmar.2006.49.issue-21/jm051256o/production/images/large/jm051256of00001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm051256o&amp;id=jm051256of00001"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div><div class="NLM_p">From a rigorous point of view, the answer depends principally upon the free energy to be gained by displacing a water molecule at a given location. This in turn depends on how successfully that water molecule is able to satisfy its hydrogen-bonding requirements at that location, while retaining orientational flexibility. In the extreme case in which a single water molecule is placed in a protein cavity that can accommodate only one water molecule and is surrounded on all sides by lipophilic atoms that cannot make hydrogen bonds, the enthalpy gain of transferring the water to bulk solution is enormously favorable. In such a case it is not clear that a water molecule would occupy such a cavity in preference to leaving a vacuum, despite the statistical terms favoring occupancy. However, this is a rare situation not particularly relevant to the binding of a large ligand, whereas structural motifs similar to the examples in Figure 1 are quite common. </div><div class="NLM_p">There have been a large number of papers in the literature studying, via molecular dynamics simulations, the behavior of <a name="jm004902106180" id="jm004902106180"></a> water in contact with various types of hydrophobic structures, including flat and curved surfaces,<sup><a onclick="showRef(event, 'jm051256ob00020 jm051256ob00021 jm051256ob00022'); return false;" href="javascript:void(0);" class="ref jm051256ob00020 jm051256ob00021 jm051256ob00022">20-22</a></sup> parallel plates,<sup><a onclick="showRef(event, 'jm051256ob00023 jm051256ob00024'); return false;" href="javascript:void(0);" class="ref jm051256ob00023 jm051256ob00024">23,24</a></sup> nanotubes,<sup><a onclick="showRef(event, 'jm051256ob00025'); return false;" href="javascript:void(0);" class="ref jm051256ob00025">25</a></sup> and recently more realistic systems such as the hydrophobic surfaces of a protein or the interface between two protein domains.<sup><a onclick="showRef(event, 'jm051256ob00026 jm051256ob00027 jm051256ob00028'); return false;" href="javascript:void(0);" class="ref jm051256ob00026 jm051256ob00027 jm051256ob00028">26-28</a></sup> There have also been attempts to develop general theories as to how the hydrophobic effect depends on the size and shape of the hydrophobic structure presented to the water molecules.<sup><a onclick="showRef(event, 'jm051256ob00029'); return false;" href="javascript:void(0);" class="ref jm051256ob00029">25,29</a></sup></div><div class="NLM_p">A number of concepts that are clearly related to the proposals in the present paper have emerged from this work:  evacuation of water (dewetting), under the appropriate conditions, from regions between two predominantly hydrophobic surfaces<sup>1,2,9</sup> and a model for the curvature dependence of the hydrophobic energy in which concave regions are argued to have greater hydrophobicity than convex ones.<sup>29</sup> However, while this work provides useful ideas and general background, development of a scoring function that can be used to quantitatively predict protein−ligand binding in the highly heterogeneous and complex environment of a protein active site requires direct engagement with a critical mass of experimental data as well as extensive parameterization and investigation of a variety of specific functional forms. In what follows, we describe the results of our investigations along these lines. </div><div class="NLM_p">A large number of computational experiments involving modifications of the hydrophobic scoring term designed to discriminate between different geometrical protein environments have been performed. The criterion for success in these experiments is the ability of any proposed new term to fit a wide range of experimental binding free energy data and yield good predictions in enrichment studies. Key findings are summarized as follows: </div><div class="NLM_p">(1) Ligand hydrophobic atoms must be considered in groups, as opposed to individually. The free energy of water molecules in the protein cavity is adversely affected beyond the norm primarily when placed in an enclosed hydrophobic microenvironment that extends over the dimension of several atoms. If there are individual isolated hydrophobic contacts, the water will typically be able to make its complement of hydrogen bonds anyway by partnering with neighboring waters as in clathrate structures surrounding small hydrocarbons in water.<sup>28</sup> After empirical experimentation, the minimum group size of connected ligand lipophilic atoms has been set at three. </div><div class="NLM_p">(2) When a group of lipophilic ligand atoms is enclosed on two sides (at a 180 degree angle) by lipophilic protein atoms, this type of structure contributes to the binding free energy beyond what is encoded in the atom−atom pair term. We refer to this situation as <i>hydrophobic enclosure</i> of the ligand. There is some analogy here to the parallel plate, nanotube (with some sets of parameters), and protein systems in which dewetting has been observed, although the length scale of the region under consideration is smaller and (likely) more heterogeneous. The pair hydrophobic term in eq 1 is generally fit to data from a wide range of experimental protein−ligand complexes. As such, it represents the behavior of individual lipophilic ligand atoms in an “average” environment. Our new terms utilize specific molecular recognition motifs and are designed to capture deviations from this average that lead to substantial increases in potency for lipophilic ligand groups of types that are typically targeted in medicinal chemistry optimization programs. That is, placing an appropriate hydrophobic ligand group within the specified protein region leads to substantial increases in potency. Indeed, the data enabling development of this term was primarily obtained from a wide range of published medicinal chemistry efforts that provided examples of lipophilic groups that yielded exceptional increases in potency, as well as those yielding minimal increases. Our objective has been to explain these results on the basis of physical chemical principles and to develop empirical scoring terms that captured the essential physics while rejecting false positives, even with imperfect docking and the neglect of induced fit effects. </div><div class="NLM_p">Calculation of the hydrophobic enclosure score, <i>E</i><sub>hyd_enclosure</sub>, is summarized below with a more detailed description of the algorithm provided in Supporting Information: </div><div class="NLM_p">(1) Lipophilic protein atoms near the surface of the active site and lipophilic ligand atoms are divided into connected groups. There are a set of rules specifying which atoms count as lipophilic and what delimits a group. </div><div class="NLM_p">(2) For each atom in a group on the ligand, lipophilic protein atoms are enumerated at various distances. </div><div class="NLM_p">(3) For each lipophilic ligand atom, the closest lipophilic protein atom is selected and a vector is drawn between it and the ligand atom. This is the protein “anchor” atom for that ligand atom. Vectors for all other suitably close lipophilic protein atoms are drawn to the ligand atom and their angles with the anchor-atom vector are determined. To be considered on the “opposite side” of the anchor atom, the angle between vectors must exceed a cutoff value that depends on the pair distance, with shorter distances requiring that the angle be closer to 180°. If the angle is close to zero degrees, the atom is on the “same side”, and is at right angles to the anchor if the angle is close to 90°. When the angle between lipophilic protein atoms is close to 180°, we have argued this leads to an especially poor environment for waters. </div><div class="NLM_p">(4) Each lipophilic ligand atom is assigned a score based on the number of total lipophilic contacts with protein atoms, weighted by the angle term. If no protein atom is greater than 90 degrees from the anchor atom, the angle term is zero and the atom contributes zero to the group's <i>E</i><sub>hyd_enclosure</sub> term. The overall score for a group is the sum over all atoms in that group of the product of the angular factor and a distance dependent factor. </div><div class="NLM_p">(5) If the score for any ligand group is greater than 4.5 kcal/mol, the penalty is capped at 4.5 kcal/mol. This was an empirical determination based on investigating many test cases and comparing the results with experimental data. The capping is rationalized by arguing that if a very large region of this type leads to a score greater than 4.5 kcal/mol, there is probably some ability of the water molecules to compensate by interacting with each other. </div><div class="NLM_p">An experimentally validated example of the gain in binding affinity from placing a large hydrophobic group in a pocket in which lipophilic protein atoms are present on both sides of the pocket (rings in both cases) is shown in Figure 2<a data-tab="pane-pcw-Figures" href="#jm051256of00002" id="" class=" internalNav"></a>. Here, replacing a phenyl substituent with a naphthyl group was shown<sup><a onclick="showRef(event, 'jm051256ob00030'); return false;" href="javascript:void(0);" class="ref jm051256ob00030">30</a></sup> to result in a 21-fold improvement in experimentally measured affinity (<i>K</i><sub>d</sub>). The naphthyl is required to fully occupy the hydrophobic pocket depicted in Figure 2. <div id="_i11"><figure id="jm051256of00002" data-index="2" class="article__inlineFigure"><h2 class="fig-label"></h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2006/jmcmar.2006.49.issue-21/jm051256o/production/images/medium/jm051256of00002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2006/jmcmar.2006.49.issue-21/jm051256o/production/images/large/jm051256of00002.jpeg" id="_i12" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last"> Figure 2 Boehringer active for <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1kv2">1kv2</a> bound to human p38 map kinase. The naphthyl group receives a −4.5 kcal/mol hydrophobic enclosure packing reward.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2006/jmcmar.2006.49.issue-21/jm051256o/production/images/large/jm051256of00002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm051256o&amp;id=jm051256of00002"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div><div class="NLM_p">As indicated above, the surrounding of ligand lipophilic atoms or groups by lipophilic protein atoms is referred to as hydrophobic enclosure. Our contention, here and in much of the following discussion of hydrogen bonding, is that proper treatment of hydrophobic enclosure is the key to discrimination of highly and weakly potent binding motifs and compounds. The underlying mathematical framework for describing enclosure, discussed above, could be cast in other forms, but the essential idea would remain unchanged. Detailed optimization of the numerical criteria for recognizing enclosure, and assigning <a name="jm004902106181" id="jm004902106181"></a> a specific contribution to the binding affinity for each motif is vital to developing methods with predictive capability. </div><div class="NLM_p"><b>Improved Model of Protein-Ligand Hydrogen Bonding.</b> In developing a refined model of hydrogen bonding, we divide hydrogen bonds into three types, neutral−neutral, neutral−charged, and charged−charged. The analysis of each type of hydrogen bonding is different due to issues associated with the long-range solvation energy (Born energy) of charged groups. An initial step is to assign different default values (assuming optimal geometric features) to each of the three types of hydrogen bonds. The default values assigned are neutral−neutral, 1.0 kcal/mol, neutral−charged, 0.5 kcal/mol, and charged−charged, 0.0 kcal/mol. These assignments are based on a combination of physical reasoning and empirical observation from fitting to reported binding affinities of a wide range of PDB complexes. </div><div class="NLM_p">The rationale for rewarding protein−ligand hydrogen bonds at all is subtle, because any such hydrogen bonds are replacing hydrogen bonds that the protein and ligand make with water. At best the net number of total hydrogen bonds on average will remain the same in the bound complex as compared to solution. However, the liberation of waters to the bulk can be argued to result in an increase in entropy, and liberation of waters around a polar protein group requires that a protein−ligand hydrogen bond with similar strength be made for a desolvation penalty to be avoided. This analysis is most plausible when both groups are neutral. The formation of a salt bridge between protein and ligand involves very different types of hydrogen bonding from what is found in solution. The thermodynamics of salt bridge formation in proteins has been studied extensively, both theoretically and experimentally,<sup><a onclick="showRef(event, 'jm051256ob00031 jm051256ob00032 jm051256ob00033 jm051256ob00034'); return false;" href="javascript:void(0);" class="ref jm051256ob00031 jm051256ob00032 jm051256ob00033 jm051256ob00034">31-34</a></sup> and depends on many factors such as the degree of solvent exposure of the groups involved in the salt bridge. The default value of zero that we assign is based on the presence of many protein−ligand complexes in the PDB with very low binding affinities in which solvent-exposed protein−ligand salt bridges are formed. Assigning the contributions of these salt bridges to the binding affinity would lead to systematically worse agreement with experimental enrichment data. In XP scoring, certain features of a salt bridge are required for this type of structure to contribute to binding affinity in XP scoring. Finally, the charged−neutral default value represents an interpolation between the neutral−neutral and charged−charged value that appears to be consistent with the empirical data. </div><div class="NLM_p">Hydrogen-bond scores are diminished from their default values as the geometry deviates from an ideal hydrogen-bonding geometry, based on both the angles between the donor and acceptor atoms and the distance. The function that we use to evaluate quality is similar to that used in ChemScore. </div><div class="NLM_p">In what follows, specialized hydrogen-bonding motifs are described in which additional increments of binding affinity are assigned in addition to those from the ChemScore-like pairwise hydrogen-bond term. Our investigations indicate that these situations can arise for neutral−neutral or charged−charged hydrogen bonds, but not for charged−neutral hydrogen bonds. The exclusion of charged−neutral hydrogen-bond special rewards has principally been driven by our failure to date to identify motifs of this type that help to improve the agreement with experimental data. One can speculate that the lack of charge complementarity in charged−neutral hydrogen bonding precludes such structures from being major molecular recognition motifs, though further investigations with larger data sets will be needed to resolve this issue. </div><div class="NLM_p"><b>Special Neutral</b><b>−</b><b>Neutral Hydrogen-Bond Motifs (</b><b><i>E</i></b><b><sub>hb_nn_motif</sub></b><b>).</b> In this section, neutral−neutral hydrogen-bonding motifs are described that were identified, based on both theoretical and empirical considerations, as making exceptional contributions to binding affinity. Such “special” hydrogen bonds represent key molecular recognition motifs that are found in many if not most pharmaceutical targets. Targeting such motifs is a central strategy in increasing the potency and specificity of medicinal compounds. Identifying such motifs through their incorporation in the scoring function should enable a dramatic improvement in both qualitative and quantitative predictions. </div><div class="NLM_p">The critical idea in our recognition of special hydrogen bonds is to locate positions in the active-site cavity at which a water molecule forming a hydrogen bond to the protein would have particular difficulty in making its complement of <i>additional</i> hydrogen bonds. Forming such a hydrogen bond imposes nontrivial geometrical constraints on the water molecule. This is the basis for the default hydrogen-bond score, but such constraints become more problematic when the environment of the water molecule is challenging with respect to making additional hydrogen bonds such as those found in the bulk environment. </div><div class="NLM_p">Our previous analysis of hydrophobic interactions suggests that the environment will be significantly more challenging if the water molecule has hydrophobic protein atoms on two faces, as opposed to a single face, and if few neighboring waters are available to readjust themselves to the constrained geometry of the protein−water hydrogen bond. Geometries of this type are identified using a modified version of the hydrophobic enclosure detection algorithm described previously. Replacement of such water molecules by the ligand will be particularly favorable if the donor or acceptor atom of the ligand achieves its full complement of hydrogen bonds by making the single targeted hydrogen bond with the protein group in question so that satisfaction of additional hydrogen bonds is not an issue. An example of a suitable group would be a planar nitrogen in an aromatic ring binding for example to a protein N−H backbone group. This has been observed to be essential to achieving high potency experimentally in the <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1bl7">1bl7</a> ligand binding to p38 MAP kinase, as shown in Figure 3<a data-tab="pane-pcw-Figures" href="#jm051256of00003" id="" class=" internalNav"></a>. Here, the Met 109 hydrogen bond is known to be important for potency. Analogous hydrogen bonds have been found to be important in other kinases. In the absence of rigorous physical chemical simulations, we have used the experimental data from a significant number of diverse protein−ligand complexes to guide the development of a set of empirical rules, outlined below, for the types of ligand and <a name="jm004902106182" id="jm004902106182"></a> receptor chemistries that receive this type of reward. These rules will likely evolve as more data is considered, and further simulations are undertaken. <div id="_i13"><figure id="jm051256of00003" data-index="3" class="article__inlineFigure"><h2 class="fig-label"></h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2006/jmcmar.2006.49.issue-21/jm051256o/production/images/medium/jm051256of00003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2006/jmcmar.2006.49.issue-21/jm051256o/production/images/large/jm051256of00003.jpeg" id="_i14" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last"> Figure 3 The <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1bl7">1bl7</a> ligand interacts with p38 MAP kinase through a neutral−neutral hydrogen bond between the ligand's aromatic nitrogen and the Met 109 N−H group.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2006/jmcmar.2006.49.issue-21/jm051256o/production/images/large/jm051256of00003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm051256o&amp;id=jm051256of00003"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div><div class="NLM_p">The scoring-function term outlined above enables such hydrogen bonds to be detected automatically in advance of experimental measurement. Of equal importance, false positives, which superficially share some characteristics of the required structural motif but lack a key component, can be rejected automatically as well. Rejection of false positives has been optimized by running a given variant of the scoring function, identifying high scoring database ligands with special hydrogen bonds in locations not seen in known actives, examining the resulting structure, and altering the recognition function to eliminate the reward for such test cases. </div><div class="NLM_p">In designing a detailed set of rules to implement the ideas outlined above, we have attempted to generalize results obtained from a wide variety of ligand−receptor systems, while at the same time avoiding false positives and respecting the basic physical chemistry principles that form the basis of the model. The detailed algorithm for detecting a single hydrogen bond in a hydrophobic environment is outlined as follows. In our implementation, the donor or acceptor atom must be in a ring with the exception of nitrogen, which is allowed to be a nonring atom. If the ligand atom is in a donor group, then all other donors of the group (e.g., the two hydrogen atoms of NH<sub>2</sub>) must be hydrogen bonded to the protein. Only backbone protein atoms can participate in this type of special hydrogen bond. The unligated protein donor or acceptor must have fewer than three first-shell solvating waters where the waters are placed as outlined in section 3. If these criteria are met, the sum of the angular factor of the hydrophobic enclosure packing score described in this section is made over the hydrogen-bonded ligand heavy atoms and the carbon atoms attached to this ligand atom. The hydrophobic enclosure packing score used in this sum contains only the angular weight of the score described earlier and not the distance-based weight. If the absolute hydrophobic enclosure packing sum is above a defined cutoff, the hydrogen bond is considered to be in a hydrophobically constrained environment, and a special hydrogen-bond reward of 1.5 kcal/mol is applied to this hydrogen bond. In some instances, it is found that a small perturbation of the ligand can move the hydrophobic sum above the cutoff. Therefore, if the hydrophobic sum is below the cutoff, small rigid body perturbations of the ligand are made of 0.3 Å in magnitude. At each perturbed geometry, the sum is recalculated and the reward is applied if at some geometry the sum exceeds the cutoff. This procedure helps to avoid discontinuities inherent in the use of a cutoff. </div><div class="NLM_p">The situation described above identifies a structure in which a single hydrogen bond should be assigned a “special” reward. A second situation occurs when there are multiple correlated hydrogen bonds between the protein and the ligand. The physical argument is that the organization of water molecules to effectively solvate a structure of this type in the confined geometry of the active site can be even more problematic than that for the single hydrogen-bond situation described above. However, this will occur only if the waters involved in such solvation are in a challenging hydrophobic environment, with hydrophobic groups on two sides. The coupling of the special hydrogen-bond identification with the hydrophobic enclosure motif is critical if false positives are to be rejected. Correlated hydrogen bonds are routinely formed in docking with highly solvent-exposed backbone pairs, but there is no evidence from the experimental data we have examined that such structures contribute to enhanced potency. </div><div class="NLM_p">Pair-correlated hydrogen bonds are defined as a donor/acceptor, donor/donor, or acceptor/acceptor pair of ligand atoms (referred to as “ligand atom pair”) that are separated by no more than one rotatable bond (hydroxyl groups count as nonrotatable in this calculation). Several restrictions on the types of pairs that can be considered are made as detailed in Table 1<a class="ref showTableEvent internalNav" data-ID="jm051256ot00001" href="#jm051256ot00001"></a>. If the ligand atoms of the pair individually have zero net formal charge, they must satisfy the following hydrophobicity criterion to achieve a special hydrogen-bond reward. First, the ligand atoms must be part of the same ring or be directly connected to the same ring. Assuming the pair satisfies these restrictions, the hydrophobicity of the hydrogen-bond region is detected in a manner similar though not identical to that for a single special hydrogen bond. A sum of the hydrophobic enclosure packing score described previously is made for the pair of hydrogen-bonding ligand heavy atoms and the ring atoms directly connected to the ligand pair atoms. If a ligand atom of the pair is not a ring atom but is connected to a ring, the sum includes atoms of the ring that are nearest neighbors to the nonring ligand atom. If the absolute hydrophobic sum is above a cutoff, the hydrogen-bonded pair is given a special reward of 3 kcal/mol. Note that double counting of pair and single special hydrogen bonds is avoided by checking pairs first and excluding any rewarded hydrogen bonds found from single hydrogen-bond consideration. <div class="NLM_table-wrap" id="jm051256ot00001"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><p class="first last"> Table 1.  </p></div></div><div class="scrollable-table-wrap"><table class="table " width="100%"><colgroup><col /></col></colgroup><tbody><tr valign="top" class="colsep2"><td class="colsep2 rowsep2"> conditions for not applying the special pair hydrogen-bond scores </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">ignore poor quality hydrogen bonds (<0.05 <i>E</i><sub>hb_pair</sub>) </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">ignore pairs involving the same neutral protein atom </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">ignore pairs involved in a salt bridge if the electrostic potential at either ligand atom is above the cutoff </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">ignore salt bridge pairs if either protein atom is involved in a protein−protein salt bridge </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">ignore ligand donor/donor pairs that come from NH<i><sub>x</sub></i>, where <i>x</i> ≥ 2 groups, the nitrogen atom is not in a ring and has no formal charge </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">ignore formally charged protein atoms with more than eight second-shell waters in the unligated state </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">ignore charged−neutral hydrogen bonds unless the protein atom is in a salt bridge </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">ignore pairs of different neutral acceptor atoms on the ligand </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">neutral hydrogen-bond pairs must satisfy ring atom and hydrophobicity environment criteria as outlined in section 2 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">ignore ligand hydroxyl to protein hydrogen bonds if the protein atom has zero formal charge</td></tr></tbody></table></div></div></div><div class="NLM_p">Finally, the special hydrogen-bond rewards are linearly reduced with the quality of the hydrogen bond. The hydrogen-<a name="jm004902106183" id="jm004902106183"></a> bond quality is measured in the sense of the pair hydrogen-bond score using the donor/acceptor distance and the angle made by the donor-heavy-atom-H vector and the H-acceptor vector. For ligand acceptor atoms in rings, the extent to which the acceptor lone pair vector is aligned with the donor-heavy-atom-H vector is evaluated. A detailed description of the algorithm for scaling the special hydrogen-bond reward is given in Supporting Information. </div><div class="NLM_p">A substantial number of protein−ligand complexes in which motifs containing correlated hydrogen bonds that satisfy the above criteria, including the requisite hydrophobic enclosure, have been identified. A number of examples are shown below. Figure 4<a data-tab="pane-pcw-Figures" href="#jm051256of00004" id="" class=" internalNav"></a> depicts the <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1aq1">1aq1</a> structure of staurosporine bound to human cyclin-dependent kinase (CDK2). This type of correlated pair is also found in a number of other kinases. Some of the CDK2 actives, such as AG12073, make three correlated hydrogen bonds; this structure is shown in Figure 5<a data-tab="pane-pcw-Figures" href="#jm051256of00005" id="" class=" internalNav"></a>. A second example is streptavidin bound to the <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1stp">1stp</a> structure of biotin (Figure 6<a data-tab="pane-pcw-Figures" href="#jm051256of00006" id="" class=" internalNav"></a>), with three correlated hydrogen bonds in a hydrophobically enclosed region. To our knowledge, no empirical scoring function has explained the exceptionally large binding affinity of streptavidin to biotin. However, once the correlated hydrophobically enclosed hydrogen-bonding motif is recognized and assigned an appropriate score (a reward of 3 kcal/mol, consistent with other examples), the deviation between calculated and experimental binding affinity, using a docked structure, is only 0.1 kcal/mol. It should be noted that the high accuracy of this prediction is fortuitous and is not intended to suggest an ability to rigorously rank order compounds. Instead, the intent is to contrast the qualitatively reasonable prediction with that of alternative scoring functions, which typically yield results for this complex in error by ∼5−10 kcal/mol, for example, as reported in ref 35. The triply correlated, enclosed hydrogen-bonding motif also explains the low nanomolar binding affinity in the binding of fidarestat to the <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1ef3">1ef3</a> structure of aldose reductase (Figure 7<a data-tab="pane-pcw-Figures" href="#jm051256of00007" id="" class=" internalNav"></a>) relative to a large number of ligands that achieve similarly large lipophilic scores in the highly hydrophobic active site, yet have only micromolar affinity. <div id="_i15"><figure id="jm051256of00004" data-index="4" class="article__inlineFigure"><h2 class="fig-label"></h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2006/jmcmar.2006.49.issue-21/jm051256o/production/images/medium/jm051256of00004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2006/jmcmar.2006.49.issue-21/jm051256o/production/images/large/jm051256of00004.jpeg" id="_i16" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last"> Figure 4 Staurosporine bound to human cyclin dependent kinase (CDK2). The pair of correlated hydrogen bonds receives a −3 kcal/mol reward, while the central component of the ring is given a −3 kcal/mol hydrophobic enclosure packing reward.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2006/jmcmar.2006.49.issue-21/jm051256o/production/images/large/jm051256of00004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm051256o&amp;id=jm051256of00004"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="jm051256of00005" data-index="5" class="article__inlineFigure"><h2 class="fig-label"></h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2006/jmcmar.2006.49.issue-21/jm051256o/production/images/medium/jm051256of00005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2006/jmcmar.2006.49.issue-21/jm051256o/production/images/large/jm051256of00005.jpeg" id="_i17" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last"> Figure 5 AG12073 bound to human cyclin dependent kinase (CDK2). The three correlated hydrogen bonds receive a −4.2 kcal/mol reward. </p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2006/jmcmar.2006.49.issue-21/jm051256o/production/images/large/jm051256of00005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm051256o&amp;id=jm051256of00005"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="jm051256of00006" data-index="6" class="article__inlineFigure"><h2 class="fig-label"></h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2006/jmcmar.2006.49.issue-21/jm051256o/production/images/medium/jm051256of00006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2006/jmcmar.2006.49.issue-21/jm051256o/production/images/large/jm051256of00006.jpeg" id="_i18" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last"> Figure 6 Biotin bound to streptavidin. The identification of a triplet of correlated hydrogen bonds in the ring in a hydrophobically enclosed region, and the three hydrogen bonds to the ligand carbonyl within that ring each contribute −3 kcal/mol rewards to this tightly bound complex (Δ<i>G</i><sub>exp</sub> = −18.3 kcal/mol, XP binding = −18.2 kcal/mol).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2006/jmcmar.2006.49.issue-21/jm051256o/production/images/large/jm051256of00006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm051256o&amp;id=jm051256of00006"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="jm051256of00007" data-index="7" class="article__inlineFigure"><h2 class="fig-label"></h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2006/jmcmar.2006.49.issue-21/jm051256o/production/images/medium/jm051256of00007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2006/jmcmar.2006.49.issue-21/jm051256o/production/images/large/jm051256of00007.jpeg" id="_i19" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last"> Figure 7 Fidarestat bound to aldose reductase. The triplet of special hydrogen bonds to the ring contributes −5.0 kcal/mol to the binding energy of this 9 nM inhibitor. </p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2006/jmcmar.2006.49.issue-21/jm051256o/production/images/large/jm051256of00007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm051256o&amp;id=jm051256of00007"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div><div class="NLM_p">In our studies of various pharmaceutically relevant targets, the combination of hydrophobic enclosure with one to three correctly positioned hydrogen bonds is characteristic of every “special” neutral−neutral hydrogen-bond motif that leads to an exceptional increase in experimentally measured potency. However, additional characteristics are required to eliminate false positives. In particular, if the hydrogen-bond partner in the protein is highly solvent exposed, formation of a structure capable of solvating the group in question, while still allowing the waters involved to form a suitable number of additional <a name="jm004902106184" id="jm004902106184"></a> hydrogen bonds, becomes easier. Thus, we require that the protein group(s) involved in the special hydrogen bond(s) have a limited number of waters in the first or second shell. Determination of the number of surrounding waters is carried out via the water addition code described later in this section. </div><div class="NLM_p">The magnitude of the rewards associated with the special hydrogen bonds has been determined by optimization against a large experimental database containing a significant number of examples of each type of structure. Values are given in Table 2<a class="ref showTableEvent internalNav" data-ID="jm051256ot00002" href="#jm051256ot00002"></a>. It is conceivable that finer discriminations, depending upon the details of the donors and acceptors, hydrophobic environment, bound waters, and so on, could be developed, along with a correspondingly more elaborate scoring scheme. However, the present relatively simple scheme appears to work remarkably well, at least at the level of discriminating active from inactive compounds (as opposed to rank ordering, which we have not yet examined in detail) in a wide variety of test cases. <div class="NLM_table-wrap" id="jm051256ot00002"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><p class="first last"> Table 2.  Special Hydrogen-Bond Reward Values</p></div></div><div class="scrollable-table-wrap"><table class="table " width="100%"><colgroup><col /></col><col /></col></colgroup><tbody><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">hydrogen-bond moiety</td><td class="colsep2 rowsep2">reward (kcal/mol) </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">single bond to ring in hydrophobic environments </td><td class="colsep2 rowsep2">1.5 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">neutral pair in hydrophobic environments </td><td class="colsep2 rowsep2">3.0</td></tr></tbody></table></div></div></div><div class="NLM_p"><b>Special Charged</b><b>−</b><b>Charged Hydrogen-Bond Motifs (</b><b><i>E</i></b><b><sub>hb_cc_motif</sub></b><b>).</b> We have identified the following features that signal enhanced binding affinity from charged−charged hydrogen bonds: </div><div class="NLM_p">(1) The number of waters surrounding the protein component of the salt bridge. Charged groups that are fully exposed to solvent are unlikely to participate in enhanced charged−charged hydrogen bonding because the cost of displacing the solvent is simply too large. Solvent exposure is calibrated using our water scoring code, described later in this section (see <i>E</i><sub>desolv</sub>), by examining the number of waters in the first two shells surrounding the charged protein group. </div><div class="NLM_p">(2) The number of charged−charged hydrogen bonds made by the charged ligand group. Three different types of salt-bridge structures have been observed:  (a) Monodentate (single hydrogen bond) between the ligand group and a protein group. (b) Bidentate (two hydrogen bonds) between the ligand group and a protein group. An example of a bidentate salt bridge occurs in the 1ett structure of thrombin between a positively charged amine group and a recessed Asp 189 carboxylate in the relevant specificity pocket, as displayed in Figure 8<a data-tab="pane-pcw-Figures" href="#jm051256of00008" id="" class=" internalNav"></a>. (c) Hydrogen bonds of one ligand group to two different protein groups. This requires having two like-charged protein groups in close proximity. This structure, which presumably creates strain energy in the apo protein, occurs with a greater frequency than might be expected. Figure 9<a data-tab="pane-pcw-Figures" href="#jm051256of00009" id="" class=" internalNav"></a> presents an example showing ligand Gr217029 binding to the tern N9 influenza virus of the neuramidase receptor (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1bji">1bji</a>) with a distance between carboxylate oxygen atoms of only 4.5 Å. <div id="_i20"><figure id="jm051256of00008" data-index="8" class="article__inlineFigure"><h2 class="fig-label"></h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2006/jmcmar.2006.49.issue-21/jm051256o/production/images/medium/jm051256of00008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2006/jmcmar.2006.49.issue-21/jm051256o/production/images/large/jm051256of00008.jpeg" id="_i21" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last"> Figure 8 The bidentate hydrogen bonds in this thrombin complex bridge the ligand and Asp 189.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2006/jmcmar.2006.49.issue-21/jm051256o/production/images/large/jm051256of00008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm051256o&amp;id=jm051256of00008"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="jm051256of00009" data-index="9" class="article__inlineFigure"><h2 class="fig-label"></h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2006/jmcmar.2006.49.issue-21/jm051256o/production/images/medium/jm051256of00009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2006/jmcmar.2006.49.issue-21/jm051256o/production/images/large/jm051256of00009.jpeg" id="_i22" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last"> Figure 9 Gr217029 forms hydrogen bonds with two nearby Asp residues when bound to tern N9 influenza virus neuraminidase.  </p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2006/jmcmar.2006.49.issue-21/jm051256o/production/images/large/jm051256of00009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm051256o&amp;id=jm051256of00009"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div><div class="NLM_p">Empirical observations, such as the unexpectedly high potency of several neuramidase ligands including Gr217029 cited above, and physical chemical reasoning in that the electric field from the two nonsalt-bridged, proximate carboxylates is highly negative and interacts more favorably with a ligand positive charge than is typical for a salt bridge suggest that (c) provides less stabilization energy than (b), which in turn provides less stabilization energy than (a). Similarly, one would expect that a bidentate structure is more favorable electrostatically than a monodentate structure. Note, however, that unless consideration (1) is properly satisfied, none of the three structures is likely to be favorable from a free energy point of view. It is the combination of restricted water access for the protein group and an exceptionally strong electrostatic interaction between the ligand and protein group that creates the molecular recognition motif. </div><div class="NLM_p">(3) Zwitterion ligands. A principal reason that the default value for charged−charged hydrogen bonds is set at zero is that, in forming a salt bridge, both the protein and ligand must surrender long-range contributions to the Born energy (i.e., those beyond the first shell). Satisfying the first-shell complement of hydrogen bonds is quite possible when forming a salt bridge, but the replacement of bulk water with the protein, or bound waters, clearly reduces the possible dielectric response to the ion. For a monovalent ion, the unscreened Coulomb field decreases as 1/r. Even though past the second shell dielectric screening substantially reduces further contributions, long-range effects make a nontrivial contribution to the total solvation-free energy. However, for a zwitterion, the fields from the positive and negative charges to some extent cancel at long range, yielding a dipolar field for which the second- and higher-shell contributions to the solvation free energy are significantly reduced. This cancellation depends on the separation of the two charged groups. Thus, formation of two salt bridges by the zwitterion, particularly if the two oppositely charged moieties in the ligand are spatially proximate, should be more favorable than binding a single ion. An example of a zwitterion binding in this fashion is shown in Figure 9. </div><div class="NLM_p"><a name="jm004902106185" id="jm004902106185"></a>(4) Cases where the ligand is positively charged and the protein is negatively charged are distinguished from those in which the charge states are reversed. </div><div class="NLM_p">(5) Strength of the electrostatic field at the ligand. An enhanced binding affinity for a salt bridge is assigned if the site at which the ligand charge is placed is sufficiently electrostatically favorable. The electrostatic field at the ligand site is summed using constant and distance-dependent dielectric models, and cutoffs are imposed for assigning rewards based on empirical optimization over our suite of test cases. These cutoffs help reduce the number of false positives receiving special charged−charged rewards. </div><div class="NLM_p">Table 3<a class="ref showTableEvent internalNav" data-ID="jm051256ot00003" href="#jm051256ot00003"></a> enumerates the various special charged−charged rewards for motifs based on the five categories discussed above. The numerical values have been optimized based on fitting to our entire test suite. <div class="NLM_table-wrap" id="jm051256ot00003"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><p class="first last"> Table 3.  Electrostatic Rewards; Note that Double Counting Is Avoided</p></div></div><div class="scrollable-table-wrap"><table class="table " width="100%"><colgroup><col /></col><col /></col></colgroup><tbody><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">charge interaction</td><td class="colsep2 rowsep2">reward (kcal/mol) </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">charged ligand atom in low electrostatic potential environments </td><td class="colsep2 rowsep2">1.5 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">zwitterion configuration, range of rewards increasing with electrostatic attraction </td><td class="colsep2 rowsep2">3.0 to 4.7 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">positive ligand group binding to weakly solvated negative protein group </td><td class="colsep2 rowsep2">0.5 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">ligand CO<sub>2</sub>− group hydrogen bound to multiple proximate positive protein residues </td><td class="colsep2 rowsep2">1.0 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">salt bridge pair in low solvation environment (less than nine second-shell waters about pre-ligated charge protein atom) </td><td class="colsep2 rowsep2">2.0</td></tr></tbody></table></div></div></div><div class="NLM_p">Table 4<a class="ref showTableEvent internalNav" data-ID="jm051256ot00004" href="#jm051256ot00004"></a> displays the XP active scores for a series of Glur2 receptors versus the experimental binding energies. Along with neuramididase, this is a system for which electrostatic interactions are particularly important. As such, it provides an important contribution to the training set. The good agreement displayed was achieved by using a combination of the electrostatic terms discussed above. <div class="NLM_table-wrap" id="jm051256ot00004"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><p class="first last"> Table 4.  4.0 XP Binding Energies for Docking into Various GluR2 Receptors Compared to Experiment<i><sup>a</sup></i><sup></sup></p></div></div><div class="scrollable-table-wrap"><table class="table " width="100%"><colgroup><col /></col><col /></col><col /></col></colgroup><tbody><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">ligand</td><td class="colsep2 rowsep2">XP score</td><td class="colsep2 rowsep2">Δ<i>G</i><sub>exp</sub> (kcal/mol) </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1ftl </td><td class="colsep2 rowsep2">−8.9 </td><td class="colsep2 rowsep2">−8.3 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1pwr </td><td class="colsep2 rowsep2">−12.6 </td><td class="colsep2 rowsep2">−13.0 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1ftj </td><td class="colsep2 rowsep2">−6.6 </td><td class="colsep2 rowsep2">−8.5 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1mm7 </td><td class="colsep2 rowsep2">−10.7 </td><td class="colsep2 rowsep2">−12.7 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1mqi </td><td class="colsep2 rowsep2">−11.9 </td><td class="colsep2 rowsep2">−11.9 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1ftm </td><td class="colsep2 rowsep2">−11.3 </td><td class="colsep2 rowsep2">−8.8 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1n0t </td><td class="colsep2 rowsep2">−6.6 </td><td class="colsep2 rowsep2">−5.7 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1m5b </td><td class="colsep2 rowsep2">−9.3 </td><td class="colsep2 rowsep2">−9.3 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1m5c </td><td class="colsep2 rowsep2">−9.4 </td><td class="colsep2 rowsep2">−9.3 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1m5e </td><td class="colsep2 rowsep2">−6.4 </td><td class="colsep2 rowsep2">−6.3</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-"><p class="first last"><i><sup>a</sup></i><sup></sup> These systems were used to calibrate the charged−charged hydrogen-bond recognition motif (<i>E</i><sub>h</sub><sub>b_</sub><sub>c</sub><sub>c_</sub><sub>m</sub><sub>otif</sub>).</p></div></div></div><div class="NLM_p"><b>Other Terms.</b> A number of other types of specialized terms have been investigated. These include terms rewarding pi stacking and pi-cation interactions (<i>E</i><sub>PI</sub>), rewards for halogen atoms placed in hydrophobic regions, and an empirical correction enhancing the binding affinity of smaller ligands relative to larger ones. These parameterizations were in many cases performed using limited data, and we do not view them yet as fully mature. As such, details will not be presented in the present publication. These terms are relatively small compared to the enclosure and charged−charged terms discussed above, but can have a nontrivial impact in specific cases. For example, the pi-cation term, for which a reward of 1.5 kcal/mol is assigned, is important for the acetylcholinesterase test case discussed below. </div><div class="NLM_p"><b>Terms that Penalize Binding in the XP Scoring Function.</b> The most important physical effects that oppose binding are strain energy of the ligand, protein, or both, loss of entropy of ligand and protein, and desolvation of the ligand or protein. The penalty terms developed are targeted in all three areas, although terms addressing strain energy and entropic loss do not necessarily represent a significant advance as compared to previous terms described in the literature. In developing the penalty terms, some fundamental limitations arise from the rigid-receptor approximation and the use of empirical scoring functions rather than full energy expressions. One consequence is that, in our view, it is not possible to completely reject false positives with an empirical approach. However, significant improvements are possible as compared to previous efforts, as we shall demonstrate below. </div><div class="NLM_p"><b>Water Scoring:  Rapid Docking of Explicit Waters (</b><b><i>E</i></b><b><sub>desolv</sub></b><b>).</b> A number of approaches to incorporating desolvation penalties into a high throughput docking code have been presented in the literature.<sup><a onclick="showRef(event, 'jm051256ob00035 jm051256ob00036'); return false;" href="javascript:void(0);" class="ref jm051256ob00035 jm051256ob00036">35,36</a></sup> However, the methods in these papers are based on continuum-solvation approaches. For computing protein−ligand binding affinities, the role of individual waters can be critical, and continuum models often provide poor results in treating bound waters in a confined cavity. Therefore, we have chosen to implement a crude explicit water model that can be rapidly evaluated yet captures the basic physics of solvation within the confines of the protein−ligand complex active-site region. </div><div class="NLM_p">The approach employed is to use a grid-based methodology and add 2.8 Å spheres, approximating water molecules, to high-scoring docked poses emerging from the initial round of XP docking. In principle, this methodology is similar to that used in the GRID program, though, algorithmic details have been optimized to achieve speed, critical in the present application. The CPU time for water addition to a single pose is 3−8 CPU seconds (AMD Athelon MP 1800+ processor running Linux) on average depending on ligand size. </div><div class="NLM_p">Once waters have been added, statistics are tabulated with regard to the number of waters surrounding each hydrophobic, polar, and charged group of the ligand and active site of the protein. When a polar or charged ligand or protein group is judged to be inadequately solvated, an appropriate desolvation penalty is assessed. Additionally, the environment of each active-site water is itself probed to search for cases in which waters make an unusual number of hydrophobic contacts. A penalty is assigned if the number of such contacts for an individual water molecule exceeds a given threshold. Water scoring statistics are also used to determine whether special hydrogen-bonding rewards should be assigned, as was discussed previously. </div><div class="NLM_p"><b>Contact Penalties (</b><b><i>E</i></b><b><sub>lig_strain</sub></b><b>).</b> Penalizing strain energy in rigid-receptor docking is probably the single most difficult component of an empirical scoring function. The problem arises from the fact that, in a typical cross-docking situation, the ligand has to adjust to fit into an imperfect (from its point of view) and rigid cavity. This often requires ligands to adopt higher-energy, nonideal torsion angles. Considering the rigid-receptor approximation that is made, it is difficult to determine whether strained ligand geometries would arise if induced-fit effects were properly accounted for or whether strained ligand geometries would be a true requirement for docking to that receptor. Furthermore, even native ligand geometries have been found to exhibit high strain energies.<sup><a onclick="showRef(event, 'jm051256ob00037'); return false;" href="javascript:void(0);" class="ref jm051256ob00037">37</a></sup> Given this limitation, the function used to penalize poses with close internal contacts is fairly lenient and only looks for severe cases of bad internal <a name="jm004902106186" id="jm004902106186"></a> contacts. One function simply counts the number of intramolecular heavy atom contacts below roughly 2.2 Å and rejects a pose entirely if there are more than three such contacts. A second, more sophisticated function assembles the contacts into groups and evaluates a penalty based on the size of the contacting groups, the range of contacts, and the extent to which the groups lie on the periphery of the molecule. Empirically, it has been found that peripheral groups are more difficult to penalize for intraligand contacts than are more centrally located groups. </div><div class="NLM_p"><b>Implementation Issues.</b> Application of XP penalty terms, particularly those related to desolvation, imposes hurdles that make it difficult for random database ligands to achieve good scores in virtual screening. These hurdles do not exist in alternative programs that ignore desolvation effects and strain energy. On the other hand, if the terms are inaccurately defined, they will adversely affect active compounds. Furthermore, a definition that would be “accurate” for a high-resolution structure may function poorly for docked structures, particularly in the rigid-receptor framework, due to inaccuracies in sampling. This issue arises routinely in practice, such as when an active ligand could avoid a penalty by moving a few tenths of an Angstrom in some direction but is blocked by the rigid protein. Similarly, the sampling algorithm may simply fail to find the superior pose. </div><div class="NLM_p">We have found that extensive sampling to enable ligands to avoid penalties when possible is an essential component of Glide XP scoring. If the penalties are due to limitations in the rigid-receptor approximation, the only solutions are (a) to adjust the parameters so that they allow the test case in question to escape penalization or (b) to accept that the active compound does not “fit” into the particular version of the receptor being used and to dock into multiple structures and/or employ induced-fit methodologies. For many cases, however, better sampling in the relevant phase space can locate ligand geometries that are able to avoid the penalties. The XP Glide sampling algorithm was explicitly designed with this objective in mind. </div><div class="NLM_p"><b>XP Glide Sampling Methodology.</b> XP Glide sampling begins with SP Glide docking, as described in refs 1 and 2, but using a wider “docking funnel” so that a greater diversity of docked structures is obtained. For XP docking to succeed, SP docking must produce at least one structure in which a key fragment of the molecule is properly docked. This has been the case in the great majority of systems that have been investigated. </div><div class="NLM_p">The second step in XP sampling is to assign various fragments of the molecule as “anchors” and to attempt to build a better-scoring pose for the ligand starting from each anchor. Typical anchors are rings, but can be other rigid fragments as well. Various positions of the anchors are clustered, representative members of each cluster are chosen, and the growing of side chains from relevant positions on the anchor is initiated. </div><div class="NLM_p">The growing algorithm proceeds one side chain at a time, thereby avoiding the combinatorial explosion of total molecular conformations that occurs when all side chains are considered together. Because the anchor fragment is already positioned in the protein, most side-chain conformations can be trivially rejected based on steric clashes. The Glide “rough scoring” function is used to screen the initial side chain conformations. Most importantly, these conformations can be grown at extremely high resolution (4 degrees for each rotatable bond) because the total number of conformations considered at any one time is being constantly pruned via screening and clustering algorithms. It is this high-resolution sampling that enables difficult cross-docking cases to be effectively addressed, and that ultimately allows penalties to be avoided when possible. </div><div class="NLM_p">After the individual side chains are grown, a set of candidate complete molecules is selected by combining high-scoring individual conformations at each position and eliminating structures with significant steric clashes between side chains. Candidate structures are minimized using the standard Glide total-energy function, which employs a distance-dependent dielectric to screen electrostatic interactions and are ranked according to the Emodel Glide pose-selection function,<sup>1</sup> comprising the molecular mechanics energy plus empirical scoring terms. Then, the grid-based water addition technology is applied to a subset of the top scoring structures, penalties are assessed as discussed above, and the full XP scoring function is computed. </div><div class="NLM_p">At this stage, structures with the largest contributions from terms that promote binding may have penalties of various types. The next stage of the algorithm, critical to obtaining suitable results, is to attempt to evade penalties by regrowing specific side chains from such poses. The side chain that is the cause of the penalty can be targeted and, by focusing on this region of the ligand exclusively, a much larger number of candidate structures covering this region of phase space can be retained and minimized. The algorithm results in a significant increase in the density of poses and locates penalty-free structures when possible, despite the fact that the penalty terms are not at present encoded in the energy gradient. Finally, a single pose is selected based on a scoring function that combines weighted Coulomb/van der Waals protein−ligand interaction energies, the terms favoring binding affinity, and the various penalty terms. </div><div class="NLM_p"><b>XP Glide Parameterization:  Philosophy and Implementation.</b> The novel terms that we have described above have been developed via a combination of reasoning from basic physical chemistry principles and examining a large set of empirical data, as discussed further below. Because the terms are calculated via fast empirical functions (as opposed to rigorous atomistic simulations), extensive parameterization is required to obtain results in reasonable agreement with experiment. The large number of parameters employed in turn necessitates the use of a large number of examples in the training set; to avoid overfitting, the training set must be substantially larger than the number of parameters that are adjusted. </div><div class="NLM_p">The total number of parameters in the current XP scoring function is on the order of 80; this includes parameters for desolvation penalties, hydrophobic enclosure, special neutral−neutral and charge−charge hydrogen bonds, and pi-cation and pi-stacking interactions. Parameters are required to convert various geometrical criteria into specific scores. The PDB complexes below, as well as the enrichment studies in the training set, were used to develop the parameter values. False positives, as well as known actives, were incorporated into the optimization protocol, so the data in the training set exceeds the total number of PDB complexes and known actives by a considerable margin (although a precise calculation of the total number of data points in the training set is very difficult to produce, as for example not every database ligand was competitive with known actives in ranking, and noncompetitive compounds played no role in parameter optimization. </div><div class="NLM_p last">Because the scoring function contains nonlinear functional forms, a rigorous optimization algorithm would also be nonlinear (rather than a simple least-squares fit); furthermore, constraints would be imposed based on physically reasonable values of the various parameters. The present set of parameters was in fact determined by a heuristic approach; a small number of <a name="jm004902106187" id="jm004902106187"></a> paradigmatic test cases were identified for each type of term, initial values were fit to yield reasonable results for these parameters, and the parameter set was then tested on the entire training set. Problem cases were then identified by these tests, and reoptimization was carried out to improve the worst outliers. A fully numerical optimization protocol would quite likely improve results for the training set, but it is unclear whether a corresponding improvement in the test set would result (note that test set results were not generated until the current parameter set was frozen in the Glide 4.0 release). As we develop a larger test set, we will investigate the use of more automated optimization algorithms, with likely quantitative improvements in the predictive capabilities of the scoring function. </div></div><div id="" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i23">3. Structural and Binding Affinity Prediction Results for PDB Complexes</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_81885" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_81885" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">A set of 268 complexes from the PDB, which we have previously used to assess the docking accuracy and scoring capabilities of Glide SP,<sup>1</sup> have been selected, of which 198 have reliable experimentally determined binding affinities as determined by our extensive examination of the literature for each case. This set of complexes displays a wide range of active-site cavities and protein−ligand interactions. The parameters of the scoring function were simultaneously optimized to reproduce the experimental binding affinity data and yield quality enrichment factors/binding affinities for the database screening tests that are discussed below. </div><div class="NLM_p">An evaluation of the docking accuracy of Glide SP was presented in ref 1, and the XP results from docking MMFFs<sup><a onclick="showRef(event, 'jm051256ob00038'); return false;" href="javascript:void(0);" class="ref jm051256ob00038">38</a></sup> optimized ligand structures shown in Table 5<a class="ref showTableEvent internalNav" data-ID="jm051256ot00005" href="#jm051256ot00005"></a> are very similar.<a class="ref showTableEvent internalNav" data-ID="jm051256ot00020" href="#jm051256ot00020"></a> This suggests that the sources of error in docking accuracy are due to issues other than those addressed by the XP scoring-function modifications. In some cases, near symmetry in the ligand leads to docked poses that are functionally equivalent to those in the crystal structure, for example in terms of protein−ligand contacts, but that exhibit a large RMSD; such cases are identified in Table 5. A principal source of errors in pose RMSD appears to be the charge distribution of the ligand, which, in a standard force field representation, may not accurately distribute formal ionic charges and does not incorporate polarization effects. Cho and co-workers have demonstrated that low RMSD ligand poses can be reliably generated by utilizing more accurate polarized charges, where the ligand charge distribution is computed in the protein environment via QM/MM methods.<sup><a onclick="showRef(event, 'jm051256ob00039'); return false;" href="javascript:void(0);" class="ref jm051256ob00039">39</a></sup> We may incorporate this methodology into future XP Glide releases for optional use. The quality of structural prediction shown by XP Glide is sufficient for the great majority of ligands to enable an adequate assessment of the scoring function to be made. <div class="NLM_table-wrap" id="jm051256ot00005"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><p class="first last"> Table 5.  4.0 XP and SP Binding Scores and Heavy Atom RMS (Å) Values for Docking into PDB Entries<i><sup>a</sup></i><sup></sup></p></div></div><div class="scrollable-table-wrap"><table class="table " width="100%"><colgroup><col /></col><col /></col><col /></col><col /></col><col /></col><col /></col><col /></col><col /></col><col /></col><col /></col><col /></col><col /></col><col /></col><col /></col></colgroup><tbody><tr valign="top" class="colsep2"><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2" colspan="3">GlideScore (kcal/mol)</td><td class="colsep2 rowsep2" colspan="2">RMS (Å)</td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2" colspan="3">GlideScore (kcal/mol)</td><td class="colsep2 rowsep2" colspan="2">RMS (Å)</td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">PDB</td><td class="colsep2 rowsep2">Δ<i>G</i><sub>exp</sub></td><td class="colsep2 rowsep2">XP</td><td class="colsep2 rowsep2">XP-corr</td><td class="colsep2 rowsep2">SP</td><td class="colsep2 rowsep2">XP</td><td class="colsep2 rowsep2">SP</td><td class="colsep2 rowsep2">PDB</td><td class="colsep2 rowsep2">Δ<i>G</i><sub>exp</sub></td><td class="colsep2 rowsep2">XP</td><td class="colsep2 rowsep2">XP-corr</td><td class="colsep2 rowsep2">SP</td><td class="colsep2 rowsep2">XP</td><td class="colsep2 rowsep2">SP </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1aaq </td><td class="colsep2 rowsep2">−11.5 </td><td class="colsep2 rowsep2">−10.6 </td><td class="colsep2 rowsep2">−10.6 </td><td class="colsep2 rowsep2">−11.5 </td><td class="colsep2 rowsep2">2.01 </td><td class="colsep2 rowsep2">1.40 </td><td class="colsep2 rowsep2">1e5i </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2">−7.4 </td><td class="colsep2 rowsep2">−7.4 </td><td class="colsep2 rowsep2">−11.2 </td><td class="colsep2 rowsep2">0.28 </td><td class="colsep2 rowsep2">0.17 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1abe </td><td class="colsep2 rowsep2">−8.9 </td><td class="colsep2 rowsep2">−7.8 </td><td class="colsep2 rowsep2">−7.8 </td><td class="colsep2 rowsep2">−8.8 </td><td class="colsep2 rowsep2">0.31 </td><td class="colsep2 rowsep2">0.40 </td><td class="colsep2 rowsep2">1eap </td><td class="colsep2 rowsep2">−8.5 </td><td class="colsep2 rowsep2">−12.6 </td><td class="colsep2 rowsep2">−11.8 </td><td class="colsep2 rowsep2">−9.2 </td><td class="colsep2 rowsep2">0.65 </td><td class="colsep2 rowsep2">2.38 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1abf </td><td class="colsep2 rowsep2">−7.4 </td><td class="colsep2 rowsep2">−8.0 </td><td class="colsep2 rowsep2">−8.0 </td><td class="colsep2 rowsep2">−9.3 </td><td class="colsep2 rowsep2">0.17 </td><td class="colsep2 rowsep2">0.14 </td><td class="colsep2 rowsep2">1ebg </td><td class="colsep2 rowsep2">−14.8 </td><td class="colsep2 rowsep2">−11.0 </td><td class="colsep2 rowsep2">−11.0 </td><td class="colsep2 rowsep2">−18.3 </td><td class="colsep2 rowsep2">0.34 </td><td class="colsep2 rowsep2">0.26 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1acj </td><td class="colsep2 rowsep2">−10.0 </td><td class="colsep2 rowsep2">−11.4 </td><td class="colsep2 rowsep2">−9.9 </td><td class="colsep2 rowsep2">−8.4 </td><td class="colsep2 rowsep2">2.81 </td><td class="colsep2 rowsep2">4.61 </td><td class="colsep2 rowsep2">1ecv </td><td class="colsep2 rowsep2">−6.6 </td><td class="colsep2 rowsep2">−7.4 </td><td class="colsep2 rowsep2">−7.4 </td><td class="colsep2 rowsep2">−9.4 </td><td class="colsep2 rowsep2">0.24 </td><td class="colsep2 rowsep2">0.18 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1acm </td><td class="colsep2 rowsep2">−10.3 </td><td class="colsep2 rowsep2">−12.2 </td><td class="colsep2 rowsep2">−12.2 </td><td class="colsep2 rowsep2">−13.1 </td><td class="colsep2 rowsep2">0.40 </td><td class="colsep2 rowsep2">0.32 </td><td class="colsep2 rowsep2">1eed </td><td class="colsep2 rowsep2">−6.5 </td><td class="colsep2 rowsep2">−12.0 </td><td class="colsep2 rowsep2">−12.0 </td><td class="colsep2 rowsep2">−8.3 </td><td class="colsep2 rowsep2">11.29 </td><td class="colsep2 rowsep2">1.58 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1aco </td><td class="colsep2 rowsep2">−4.9 </td><td class="colsep2 rowsep2">−2.3 </td><td class="colsep2 rowsep2">−2.3 </td><td class="colsep2 rowsep2">−10.1 </td><td class="colsep2 rowsep2">0.34 </td><td class="colsep2 rowsep2">0.29 </td><td class="colsep2 rowsep2">1ejn </td><td class="colsep2 rowsep2">−7.7 </td><td class="colsep2 rowsep2">−9.8 </td><td class="colsep2 rowsep2">−7.8 </td><td class="colsep2 rowsep2">−10.0 </td><td class="colsep2 rowsep2">0.34 </td><td class="colsep2 rowsep2">0.12 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1add </td><td class="colsep2 rowsep2">−9.2 </td><td class="colsep2 rowsep2">−10.3 </td><td class="colsep2 rowsep2">−8.3 </td><td class="colsep2 rowsep2">−9.9 </td><td class="colsep2 rowsep2">0.83 </td><td class="colsep2 rowsep2">0.70 </td><td class="colsep2 rowsep2">1ela </td><td class="colsep2 rowsep2">−8.7 </td><td class="colsep2 rowsep2">−8.4 </td><td class="colsep2 rowsep2">−8.4 </td><td class="colsep2 rowsep2">−5.1 </td><td class="colsep2 rowsep2">0.39 </td><td class="colsep2 rowsep2">6.01 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1adf </td><td class="colsep2 rowsep2">−6.2 </td><td class="colsep2 rowsep2">−7.9 </td><td class="colsep2 rowsep2">−7.9 </td><td class="colsep2 rowsep2">−11.8 </td><td class="colsep2 rowsep2">3.03 </td><td class="colsep2 rowsep2">9.92 </td><td class="colsep2 rowsep2">1elb </td><td class="colsep2 rowsep2">−9.8 </td><td class="colsep2 rowsep2">−6.3 </td><td class="colsep2 rowsep2">−6.3 </td><td class="colsep2 rowsep2">−7.0 </td><td class="colsep2 rowsep2">5.42 </td><td class="colsep2 rowsep2">4.29 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1aha </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2">−7.9 </td><td class="colsep2 rowsep2">−7.9 </td><td class="colsep2 rowsep2">−8.2 </td><td class="colsep2 rowsep2">0.36 </td><td class="colsep2 rowsep2">0.11 </td><td class="colsep2 rowsep2">1elc </td><td class="colsep2 rowsep2">−9.4 </td><td class="colsep2 rowsep2">−7.3 </td><td class="colsep2 rowsep2">−7.3 </td><td class="colsep2 rowsep2">−6.0 </td><td class="colsep2 rowsep2">6.53 </td><td class="colsep2 rowsep2">7.92 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1ake </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2">−14.0 </td><td class="colsep2 rowsep2">−14.0 </td><td class="colsep2 rowsep2">−3.9 </td><td class="colsep2 rowsep2">15.45 </td><td class="colsep2 rowsep2">14.95 </td><td class="colsep2 rowsep2">1eld </td><td class="colsep2 rowsep2">−9.1 </td><td class="colsep2 rowsep2">−6.5 </td><td class="colsep2 rowsep2">−6.5 </td><td class="colsep2 rowsep2">−4.8 </td><td class="colsep2 rowsep2">0.32 </td><td class="colsep2 rowsep2">3.94 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1apb </td><td class="colsep2 rowsep2">−7.9 </td><td class="colsep2 rowsep2">−7.7 </td><td class="colsep2 rowsep2">−7.7 </td><td class="colsep2 rowsep2">−9.4 </td><td class="colsep2 rowsep2">0.06 </td><td class="colsep2 rowsep2">0.10 </td><td class="colsep2 rowsep2">1ele </td><td class="colsep2 rowsep2">−9.3 </td><td class="colsep2 rowsep2">−8.0 </td><td class="colsep2 rowsep2">−8.0 </td><td class="colsep2 rowsep2">−6.4 </td><td class="colsep2 rowsep2">0.36 </td><td class="colsep2 rowsep2">0.38 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1apt </td><td class="colsep2 rowsep2">−12.8 </td><td class="colsep2 rowsep2">−11.9 </td><td class="colsep2 rowsep2">−11.9 </td><td class="colsep2 rowsep2">−11.0 </td><td class="colsep2 rowsep2">1.48 </td><td class="colsep2 rowsep2">1.24 </td><td class="colsep2 rowsep2">1epb </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2">−13.6 </td><td class="colsep2 rowsep2">−11.1 </td><td class="colsep2 rowsep2">−8.0 </td><td class="colsep2 rowsep2">2.24 </td><td class="colsep2 rowsep2">1.89 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1apu </td><td class="colsep2 rowsep2">−10.2 </td><td class="colsep2 rowsep2">−8.7 </td><td class="colsep2 rowsep2">−8.7 </td><td class="colsep2 rowsep2">−7.6 </td><td class="colsep2 rowsep2">0.61 </td><td class="colsep2 rowsep2">1.24 </td><td class="colsep2 rowsep2">1eta </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2">−3.9 </td><td class="colsep2 rowsep2">−3.9 </td><td class="colsep2 rowsep2">−3.5 </td><td class="colsep2 rowsep2">8.69 </td><td class="colsep2 rowsep2">1.85 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1apv </td><td class="colsep2 rowsep2">−12.3 </td><td class="colsep2 rowsep2">−11.2 </td><td class="colsep2 rowsep2">−11.2 </td><td class="colsep2 rowsep2">−8.5 </td><td class="colsep2 rowsep2">0.63 </td><td class="colsep2 rowsep2">0.48 </td><td class="colsep2 rowsep2">1etr </td><td class="colsep2 rowsep2">−10.1 </td><td class="colsep2 rowsep2">−11.7 </td><td class="colsep2 rowsep2">−9.7 </td><td class="colsep2 rowsep2">−9.4 </td><td class="colsep2 rowsep2">0.71 </td><td class="colsep2 rowsep2">0.68 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1apw </td><td class="colsep2 rowsep2">−10.9 </td><td class="colsep2 rowsep2">−11.0 </td><td class="colsep2 rowsep2">−11.0 </td><td class="colsep2 rowsep2">−9.0 </td><td class="colsep2 rowsep2">0.97 </td><td class="colsep2 rowsep2">0.32 </td><td class="colsep2 rowsep2">1ets </td><td class="colsep2 rowsep2">−11.2 </td><td class="colsep2 rowsep2">−13.7 </td><td class="colsep2 rowsep2">−11.7 </td><td class="colsep2 rowsep2">−11.7 </td><td class="colsep2 rowsep2">1.32 </td><td class="colsep2 rowsep2">1.44 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1atl </td><td class="colsep2 rowsep2">−8.6 </td><td class="colsep2 rowsep2">−10.6 </td><td class="colsep2 rowsep2">−10.6 </td><td class="colsep2 rowsep2">−7.8 </td><td class="colsep2 rowsep2">0.87 </td><td class="colsep2 rowsep2">3.47 </td><td class="colsep2 rowsep2">1ett </td><td class="colsep2 rowsep2">−8.0 </td><td class="colsep2 rowsep2">−12.6 </td><td class="colsep2 rowsep2">−10.6 </td><td class="colsep2 rowsep2">−9.5 </td><td class="colsep2 rowsep2">0.62 </td><td class="colsep2 rowsep2">0.58 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1avd </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2">−16.4 </td><td class="colsep2 rowsep2">−16.4 </td><td class="colsep2 rowsep2">−10.4 </td><td class="colsep2 rowsep2">0.82 </td><td class="colsep2 rowsep2">0.55 </td><td class="colsep2 rowsep2">1ezq </td><td class="colsep2 rowsep2">−12.3 </td><td class="colsep2 rowsep2">−11.4 </td><td class="colsep2 rowsep2">−9.4 </td><td class="colsep2 rowsep2">−12.6 </td><td class="colsep2 rowsep2">0.75 </td><td class="colsep2 rowsep2">0.21 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1azm </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2">−5.1 </td><td class="colsep2 rowsep2">−5.1 </td><td class="colsep2 rowsep2">−6.2 </td><td class="colsep2 rowsep2">1.89 </td><td class="colsep2 rowsep2">2.51 </td><td class="colsep2 rowsep2">1f0r </td><td class="colsep2 rowsep2">−10.4 </td><td class="colsep2 rowsep2">−13.3 </td><td class="colsep2 rowsep2">−11.3 </td><td class="colsep2 rowsep2">−10.1 </td><td class="colsep2 rowsep2">2.11 </td><td class="colsep2 rowsep2">0.59 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1b6j </td><td class="colsep2 rowsep2">−10.8 </td><td class="colsep2 rowsep2">−18.3 </td><td class="colsep2 rowsep2">−18.3 </td><td class="colsep2 rowsep2">−15.8 </td><td class="colsep2 rowsep2">2.98 </td><td class="colsep2 rowsep2">0.43 </td><td class="colsep2 rowsep2">1f0s </td><td class="colsep2 rowsep2">−10.6 </td><td class="colsep2 rowsep2">−11.3 </td><td class="colsep2 rowsep2">−11.3 </td><td class="colsep2 rowsep2">−9.4 </td><td class="colsep2 rowsep2">2.08 </td><td class="colsep2 rowsep2">0.35 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1b6k </td><td class="colsep2 rowsep2">−11.9 </td><td class="colsep2 rowsep2">−14.8 </td><td class="colsep2 rowsep2">−12.8 </td><td class="colsep2 rowsep2">−13.8 </td><td class="colsep2 rowsep2">1.04 </td><td class="colsep2 rowsep2">1.06 </td><td class="colsep2 rowsep2">1f0t </td><td class="colsep2 rowsep2">−8.2 </td><td class="colsep2 rowsep2">−7.7 </td><td class="colsep2 rowsep2">−7.7 </td><td class="colsep2 rowsep2">−9.8 </td><td class="colsep2 rowsep2">0.38 </td><td class="colsep2 rowsep2">0.24 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1b6l </td><td class="colsep2 rowsep2">−11.3 </td><td class="colsep2 rowsep2">−11.1 </td><td class="colsep2 rowsep2">−11.1 </td><td class="colsep2 rowsep2">−8.7 </td><td class="colsep2 rowsep2">0.92 </td><td class="colsep2 rowsep2">1.18 </td><td class="colsep2 rowsep2">1f0u </td><td class="colsep2 rowsep2">−9.8 </td><td class="colsep2 rowsep2">−10.8 </td><td class="colsep2 rowsep2">−8.8 </td><td class="colsep2 rowsep2">−10.5 </td><td class="colsep2 rowsep2">1.56 </td><td class="colsep2 rowsep2">1.54 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1b6m </td><td class="colsep2 rowsep2">−11.5 </td><td class="colsep2 rowsep2">−13.9 </td><td class="colsep2 rowsep2">−11.9 </td><td class="colsep2 rowsep2">−11.4 </td><td class="colsep2 rowsep2">0.73 </td><td class="colsep2 rowsep2">3.17 </td><td class="colsep2 rowsep2">1fen </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2">−12.8 </td><td class="colsep2 rowsep2">−12.1 </td><td class="colsep2 rowsep2">−8.4 </td><td class="colsep2 rowsep2">1.10 </td><td class="colsep2 rowsep2">0.40 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1baf </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2">−9.5 </td><td class="colsep2 rowsep2">−9.5 </td><td class="colsep2 rowsep2">−8.3 </td><td class="colsep2 rowsep2">1.17 </td><td class="colsep2 rowsep2">1.08 </td><td class="colsep2 rowsep2">1fh8 </td><td class="colsep2 rowsep2">−9.4 </td><td class="colsep2 rowsep2">−10.7 </td><td class="colsep2 rowsep2">−10.7 </td><td class="colsep2 rowsep2">−10.3 </td><td class="colsep2 rowsep2">0.20 </td><td class="colsep2 rowsep2">0.20 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1bap </td><td class="colsep2 rowsep2">−9.3 </td><td class="colsep2 rowsep2">−8.2 </td><td class="colsep2 rowsep2">−8.2 </td><td class="colsep2 rowsep2">−9.1 </td><td class="colsep2 rowsep2">0.39 </td><td class="colsep2 rowsep2">0.38 </td><td class="colsep2 rowsep2">1fh9 </td><td class="colsep2 rowsep2">−8.8 </td><td class="colsep2 rowsep2">−6.0 </td><td class="colsep2 rowsep2">−6.0 </td><td class="colsep2 rowsep2">−5.3 </td><td class="colsep2 rowsep2">2.11 </td><td class="colsep2 rowsep2">1.99 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1bbp </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2">−14.3 </td><td class="colsep2 rowsep2">−12.4 </td><td class="colsep2 rowsep2">−11.8 </td><td class="colsep2 rowsep2">5.28 </td><td class="colsep2 rowsep2">5.11 </td><td class="colsep2 rowsep2">1fhd </td><td class="colsep2 rowsep2">−9.3 </td><td class="colsep2 rowsep2">−8.1 </td><td class="colsep2 rowsep2">−8.1 </td><td class="colsep2 rowsep2">−8.7 </td><td class="colsep2 rowsep2">5.37 </td><td class="colsep2 rowsep2">0.46 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1bkm </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2">−11.6 </td><td class="colsep2 rowsep2">−11.6 </td><td class="colsep2 rowsep2">−14.6 </td><td class="colsep2 rowsep2">4.77 </td><td class="colsep2 rowsep2">2.36 </td><td class="colsep2 rowsep2">1fjs </td><td class="colsep2 rowsep2">−13.6 </td><td class="colsep2 rowsep2">−13.6 </td><td class="colsep2 rowsep2">−11.6 </td><td class="colsep2 rowsep2">−12.5 </td><td class="colsep2 rowsep2">2.06 </td><td class="colsep2 rowsep2">2.42 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1bma </td><td class="colsep2 rowsep2">−6.3 </td><td class="colsep2 rowsep2">−7.9 </td><td class="colsep2 rowsep2">−7.9 </td><td class="colsep2 rowsep2">−7.5 </td><td class="colsep2 rowsep2">0.68 </td><td class="colsep2 rowsep2">1.94 </td><td class="colsep2 rowsep2">1fkg </td><td class="colsep2 rowsep2">−10.9 </td><td class="colsep2 rowsep2">−13.1 </td><td class="colsep2 rowsep2">−12.6 </td><td class="colsep2 rowsep2">−8.4 </td><td class="colsep2 rowsep2">1.21 </td><td class="colsep2 rowsep2">1.33 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1bra </td><td class="colsep2 rowsep2">−2.5 </td><td class="colsep2 rowsep2">−5.5 </td><td class="colsep2 rowsep2">−3.5 </td><td class="colsep2 rowsep2">−7.8 </td><td class="colsep2 rowsep2">2.26 </td><td class="colsep2 rowsep2">0.32 </td><td class="colsep2 rowsep2">1fki </td><td class="colsep2 rowsep2">−9.5 </td><td class="colsep2 rowsep2">−10.2 </td><td class="colsep2 rowsep2">−10.2 </td><td class="colsep2 rowsep2">−7.7 </td><td class="colsep2 rowsep2">1.30 </td><td class="colsep2 rowsep2">1.29 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1byb </td><td class="colsep2 rowsep2">−19.0 </td><td class="colsep2 rowsep2">−14.2 </td><td class="colsep2 rowsep2">−14.2 </td><td class="colsep2 rowsep2">−11.3 </td><td class="colsep2 rowsep2">0.56 </td><td class="colsep2 rowsep2">0.46 </td><td class="colsep2 rowsep2">1fq5 </td><td class="colsep2 rowsep2">−11.5 </td><td class="colsep2 rowsep2">−17.0 </td><td class="colsep2 rowsep2">−17.0 </td><td class="colsep2 rowsep2">−14.0 </td><td class="colsep2 rowsep2">1.96 </td><td class="colsep2 rowsep2">2.43 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1c1b </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2">−18.2 </td><td class="colsep2 rowsep2">−15.7 </td><td class="colsep2 rowsep2">−10.8 </td><td class="colsep2 rowsep2">0.91 </td><td class="colsep2 rowsep2">0.45 </td><td class="colsep2 rowsep2">1fvt </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2">−13.2 </td><td class="colsep2 rowsep2">−13.2 </td><td class="colsep2 rowsep2">−8.4 </td><td class="colsep2 rowsep2">0.88 </td><td class="colsep2 rowsep2">0.88 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1c3i </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2">−12.6 </td><td class="colsep2 rowsep2">−12.6 </td><td class="colsep2 rowsep2">−11.9 </td><td class="colsep2 rowsep2">0.61 </td><td class="colsep2 rowsep2">0.43 </td><td class="colsep2 rowsep2">1g45 </td><td class="colsep2 rowsep2">−11.8 </td><td class="colsep2 rowsep2">−8.2 </td><td class="colsep2 rowsep2">−8.2 </td><td class="colsep2 rowsep2">−6.0 </td><td class="colsep2 rowsep2">7.88 </td><td class="colsep2 rowsep2">4.02 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1c5p </td><td class="colsep2 rowsep2">−6.4 </td><td class="colsep2 rowsep2">−6.2 </td><td class="colsep2 rowsep2">−6.2 </td><td class="colsep2 rowsep2">−8.6 </td><td class="colsep2 rowsep2">0.27 </td><td class="colsep2 rowsep2">0.25 </td><td class="colsep2 rowsep2">1g46 </td><td class="colsep2 rowsep2">−12.1 </td><td class="colsep2 rowsep2">−8.2 </td><td class="colsep2 rowsep2">−8.2 </td><td class="colsep2 rowsep2">−6.3 </td><td class="colsep2 rowsep2">8.06 </td><td class="colsep2 rowsep2">4.50 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1c83 </td><td class="colsep2 rowsep2">−6.6 </td><td class="colsep2 rowsep2">−7.3 </td><td class="colsep2 rowsep2">−7.3 </td><td class="colsep2 rowsep2">−9.9 </td><td class="colsep2 rowsep2">0.17 </td><td class="colsep2 rowsep2">0.14 </td><td class="colsep2 rowsep2">1g48 </td><td class="colsep2 rowsep2">−11.5 </td><td class="colsep2 rowsep2">−7.0 </td><td class="colsep2 rowsep2">−7.0 </td><td class="colsep2 rowsep2">−5.9 </td><td class="colsep2 rowsep2">1.88 </td><td class="colsep2 rowsep2">3.77 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1c84 </td><td class="colsep2 rowsep2">−6.8 </td><td class="colsep2 rowsep2">−7.5 </td><td class="colsep2 rowsep2">−7.5 </td><td class="colsep2 rowsep2">−8.8 </td><td class="colsep2 rowsep2">0.26 </td><td class="colsep2 rowsep2">0.32 </td><td class="colsep2 rowsep2">1g4j </td><td class="colsep2 rowsep2">−11.9 </td><td class="colsep2 rowsep2">−6.9 </td><td class="colsep2 rowsep2">−6.9 </td><td class="colsep2 rowsep2">−6.9 </td><td class="colsep2 rowsep2">5.56 </td><td class="colsep2 rowsep2">3.51 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1c86 </td><td class="colsep2 rowsep2">−6.4 </td><td class="colsep2 rowsep2">−8.4 </td><td class="colsep2 rowsep2">−8.4 </td><td class="colsep2 rowsep2">−10.9 </td><td class="colsep2 rowsep2">0.19 </td><td class="colsep2 rowsep2">0.18 </td><td class="colsep2 rowsep2">1g4o </td><td class="colsep2 rowsep2">−11.3 </td><td class="colsep2 rowsep2">−7.6 </td><td class="colsep2 rowsep2">−7.6 </td><td class="colsep2 rowsep2">−6.0 </td><td class="colsep2 rowsep2">3.39 </td><td class="colsep2 rowsep2">4.21 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1c87 </td><td class="colsep2 rowsep2">−5.7 </td><td class="colsep2 rowsep2">−8.1 </td><td class="colsep2 rowsep2">−8.1 </td><td class="colsep2 rowsep2">−10.9 </td><td class="colsep2 rowsep2">0.28 </td><td class="colsep2 rowsep2">0.21 </td><td class="colsep2 rowsep2">1g52 </td><td class="colsep2 rowsep2">−13.0 </td><td class="colsep2 rowsep2">−8.2 </td><td class="colsep2 rowsep2">−8.2 </td><td class="colsep2 rowsep2">−5.7 </td><td class="colsep2 rowsep2">8.01 </td><td class="colsep2 rowsep2">4.26 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1c88 </td><td class="colsep2 rowsep2">−7.2 </td><td class="colsep2 rowsep2">−7.5 </td><td class="colsep2 rowsep2">−7.5 </td><td class="colsep2 rowsep2">−11.8 </td><td class="colsep2 rowsep2">0.25 </td><td class="colsep2 rowsep2">0.22 </td><td class="colsep2 rowsep2">1g53 </td><td class="colsep2 rowsep2">−12.3 </td><td class="colsep2 rowsep2">−8.4 </td><td class="colsep2 rowsep2">−8.4 </td><td class="colsep2 rowsep2">−6.4 </td><td class="colsep2 rowsep2">7.88 </td><td class="colsep2 rowsep2">4.53 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1c8k </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2">−8.1 </td><td class="colsep2 rowsep2">−8.1 </td><td class="colsep2 rowsep2">−7.3 </td><td class="colsep2 rowsep2">3.28 </td><td class="colsep2 rowsep2">5.50 </td><td class="colsep2 rowsep2">1g54 </td><td class="colsep2 rowsep2">−12.0 </td><td class="colsep2 rowsep2">−7.2 </td><td class="colsep2 rowsep2">−7.2 </td><td class="colsep2 rowsep2">−5.4 </td><td class="colsep2 rowsep2">8.45 </td><td class="colsep2 rowsep2">5.14 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1cbs </td><td class="colsep2 rowsep2">−9.8 </td><td class="colsep2 rowsep2">−7.5 </td><td class="colsep2 rowsep2">−7.5 </td><td class="colsep2 rowsep2">−7.5 </td><td class="colsep2 rowsep2">0.63 </td><td class="colsep2 rowsep2">0.39 </td><td class="colsep2 rowsep2">1ghb </td><td class="colsep2 rowsep2">−1.7 </td><td class="colsep2 rowsep2">−3.2 </td><td class="colsep2 rowsep2">−3.2 </td><td class="colsep2 rowsep2">−9.7 </td><td class="colsep2 rowsep2">0.45 </td><td class="colsep2 rowsep2">0.30 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1cbx </td><td class="colsep2 rowsep2">−8.7 </td><td class="colsep2 rowsep2">−7.8 </td><td class="colsep2 rowsep2">−7.8 </td><td class="colsep2 rowsep2">−13.1 </td><td class="colsep2 rowsep2">0.28 </td><td class="colsep2 rowsep2">0.48 </td><td class="colsep2 rowsep2">1glp </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2">−4.5 </td><td class="colsep2 rowsep2">−4.5 </td><td class="colsep2 rowsep2">−8.1 </td><td class="colsep2 rowsep2">0.75 </td><td class="colsep2 rowsep2">0.32 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1cde </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2">−15.2 </td><td class="colsep2 rowsep2">−15.2 </td><td class="colsep2 rowsep2">−11.5 </td><td class="colsep2 rowsep2">1.62 </td><td class="colsep2 rowsep2">1.71 </td><td class="colsep2 rowsep2">1glq </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2">−6.2 </td><td class="colsep2 rowsep2">−6.2 </td><td class="colsep2 rowsep2">−9.7 </td><td class="colsep2 rowsep2">0.46 </td><td class="colsep2 rowsep2">0.32 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1cdg </td><td class="colsep2 rowsep2">−3.3 </td><td class="colsep2 rowsep2">−2.4 </td><td class="colsep2 rowsep2">−2.4 </td><td class="colsep2 rowsep2">−4.8 </td><td class="colsep2 rowsep2">6.48 </td><td class="colsep2 rowsep2">9.83 </td><td class="colsep2 rowsep2">1gsp </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2">−7.7 </td><td class="colsep2 rowsep2">−7.7 </td><td class="colsep2 rowsep2">−7.8 </td><td class="colsep2 rowsep2">1.10 </td><td class="colsep2 rowsep2">2.79 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1cil </td><td class="colsep2 rowsep2">−12.9 </td><td class="colsep2 rowsep2">−5.4 </td><td class="colsep2 rowsep2">−5.4 </td><td class="colsep2 rowsep2">−6.2 </td><td class="colsep2 rowsep2">3.61 </td><td class="colsep2 rowsep2">3.92 </td><td class="colsep2 rowsep2">1hbv </td><td class="colsep2 rowsep2">−8.7 </td><td class="colsep2 rowsep2">−12.1 </td><td class="colsep2 rowsep2">−12.1 </td><td class="colsep2 rowsep2">−5.5 </td><td class="colsep2 rowsep2">2.19 </td><td class="colsep2 rowsep2">2.07 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1cnx </td><td class="colsep2 rowsep2">−10.0 </td><td class="colsep2 rowsep2">−6.8 </td><td class="colsep2 rowsep2">−6.8 </td><td class="colsep2 rowsep2">−7.5 </td><td class="colsep2 rowsep2">6.54 </td><td class="colsep2 rowsep2">6.53 </td><td class="colsep2 rowsep2">1hdc </td><td class="colsep2 rowsep2">−8.2 </td><td class="colsep2 rowsep2">−9.8 </td><td class="colsep2 rowsep2">−9.8 </td><td class="colsep2 rowsep2">−8.3 </td><td class="colsep2 rowsep2">0.56 </td><td class="colsep2 rowsep2">0.35 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1com </td><td class="colsep2 rowsep2">−5.4 </td><td class="colsep2 rowsep2">−7.3 </td><td class="colsep2 rowsep2">−7.3 </td><td class="colsep2 rowsep2">−9.0 </td><td class="colsep2 rowsep2">0.55 </td><td class="colsep2 rowsep2">3.74 </td><td class="colsep2 rowsep2">1hdy </td><td class="colsep2 rowsep2">−7.8 </td><td class="colsep2 rowsep2">−4.8 </td><td class="colsep2 rowsep2">−4.8 </td><td class="colsep2 rowsep2">−4.2 </td><td class="colsep2 rowsep2">1.65 </td><td class="colsep2 rowsep2">1.70 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1coy </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2">−8.7 </td><td class="colsep2 rowsep2">−8.7 </td><td class="colsep2 rowsep2">−9.1 </td><td class="colsep2 rowsep2">0.44 </td><td class="colsep2 rowsep2">0.29 </td><td class="colsep2 rowsep2">1hef </td><td class="colsep2 rowsep2">−12.1 </td><td class="colsep2 rowsep2">−11.2 </td><td class="colsep2 rowsep2">−11.2 </td><td class="colsep2 rowsep2">−10.3 </td><td class="colsep2 rowsep2">6.24 </td><td class="colsep2 rowsep2">6.43 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1ctr </td><td class="colsep2 rowsep2">−5.8 </td><td class="colsep2 rowsep2">−7.5 </td><td class="colsep2 rowsep2">−7.5 </td><td class="colsep2 rowsep2">−5.7 </td><td class="colsep2 rowsep2">2.27 </td><td class="colsep2 rowsep2">2.59 </td><td class="colsep2 rowsep2">1hfc </td><td class="colsep2 rowsep2">−7.5 </td><td class="colsep2 rowsep2">−8.9 </td><td class="colsep2 rowsep2">−8.9 </td><td class="colsep2 rowsep2">−9.4 </td><td class="colsep2 rowsep2">2.25 </td><td class="colsep2 rowsep2">2.26 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1ctt </td><td class="colsep2 rowsep2">−6.2 </td><td class="colsep2 rowsep2">−6.9 </td><td class="colsep2 rowsep2">−6.9 </td><td class="colsep2 rowsep2">−7.0 </td><td class="colsep2 rowsep2">0.60 </td><td class="colsep2 rowsep2">5.04 </td><td class="colsep2 rowsep2">1hgg </td><td class="colsep2 rowsep2">−3.4 </td><td class="colsep2 rowsep2">−6.4 </td><td class="colsep2 rowsep2">−6.4 </td><td class="colsep2 rowsep2">−7.2 </td><td class="colsep2 rowsep2">1.37 </td><td class="colsep2 rowsep2">1.47 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1d3d </td><td class="colsep2 rowsep2">−12.4 </td><td class="colsep2 rowsep2">−12.7 </td><td class="colsep2 rowsep2">−12.7 </td><td class="colsep2 rowsep2">−10.4 </td><td class="colsep2 rowsep2">1.52 </td><td class="colsep2 rowsep2">2.74 </td><td class="colsep2 rowsep2">1hgh </td><td class="colsep2 rowsep2">−3.9 </td><td class="colsep2 rowsep2">−5.7 </td><td class="colsep2 rowsep2">−5.7 </td><td class="colsep2 rowsep2">−6.2 </td><td class="colsep2 rowsep2">4.70 </td><td class="colsep2 rowsep2">0.49 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1d3p </td><td class="colsep2 rowsep2">−8.9 </td><td class="colsep2 rowsep2">−9.6 </td><td class="colsep2 rowsep2">−9.6 </td><td class="colsep2 rowsep2">−10.3 </td><td class="colsep2 rowsep2">1.72 </td><td class="colsep2 rowsep2">2.05 </td><td class="colsep2 rowsep2">1hgi </td><td class="colsep2 rowsep2">−3.7 </td><td class="colsep2 rowsep2">−3.9 </td><td class="colsep2 rowsep2">−3.9 </td><td class="colsep2 rowsep2">−6.9 </td><td class="colsep2 rowsep2">0.37 </td><td class="colsep2 rowsep2">0.23 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1d7x </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2">−7.2 </td><td class="colsep2 rowsep2">−7.2 </td><td class="colsep2 rowsep2">−9.1 </td><td class="colsep2 rowsep2">0.58 </td><td class="colsep2 rowsep2">0.44 </td><td class="colsep2 rowsep2">1hgj </td><td class="colsep2 rowsep2">−2.3 </td><td class="colsep2 rowsep2">−4.8 </td><td class="colsep2 rowsep2">−4.8 </td><td class="colsep2 rowsep2">−4.0 </td><td class="colsep2 rowsep2">0.64 </td><td class="colsep2 rowsep2">0.34 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1d8f </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2">−6.8 </td><td class="colsep2 rowsep2">−6.8 </td><td class="colsep2 rowsep2">−6.8 </td><td class="colsep2 rowsep2">4.25 </td><td class="colsep2 rowsep2">4.31 </td><td class="colsep2 rowsep2">1hih </td><td class="colsep2 rowsep2">−11.0 </td><td class="colsep2 rowsep2">−11.8 </td><td class="colsep2 rowsep2">−11.8 </td><td class="colsep2 rowsep2">−11.0 </td><td class="colsep2 rowsep2">1.26 </td><td class="colsep2 rowsep2">1.29 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1dbb </td><td class="colsep2 rowsep2">−12.3 </td><td class="colsep2 rowsep2">−13.2 </td><td class="colsep2 rowsep2">−13.0 </td><td class="colsep2 rowsep2">−10.3 </td><td class="colsep2 rowsep2">0.37 </td><td class="colsep2 rowsep2">0.41 </td><td class="colsep2 rowsep2">1hps </td><td class="colsep2 rowsep2">−12.6 </td><td class="colsep2 rowsep2">−12.2 </td><td class="colsep2 rowsep2">−12.2 </td><td class="colsep2 rowsep2">−12.8 </td><td class="colsep2 rowsep2">11.93* </td><td class="colsep2 rowsep2">2.09 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1dbj </td><td class="colsep2 rowsep2">−10.4 </td><td class="colsep2 rowsep2">−15.9 </td><td class="colsep2 rowsep2">−13.4 </td><td class="colsep2 rowsep2">−9.5 </td><td class="colsep2 rowsep2">0.32 </td><td class="colsep2 rowsep2">0.21 </td><td class="colsep2 rowsep2">1hpv </td><td class="colsep2 rowsep2">−12.6 </td><td class="colsep2 rowsep2">−9.3 </td><td class="colsep2 rowsep2">−9.3 </td><td class="colsep2 rowsep2">−10.0 </td><td class="colsep2 rowsep2">1.05 </td><td class="colsep2 rowsep2">0.93 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1dbk </td><td class="colsep2 rowsep2">−11.0 </td><td class="colsep2 rowsep2">−15.2 </td><td class="colsep2 rowsep2">−12.9 </td><td class="colsep2 rowsep2">−9.0 </td><td class="colsep2 rowsep2">0.57 </td><td class="colsep2 rowsep2">0.40 </td><td class="colsep2 rowsep2">1hpx </td><td class="colsep2 rowsep2">−12.7 </td><td class="colsep2 rowsep2">−11.5 </td><td class="colsep2 rowsep2">−11.5 </td><td class="colsep2 rowsep2">−13.1 </td><td class="colsep2 rowsep2">3.34 </td><td class="colsep2 rowsep2">3.31 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1dbm </td><td class="colsep2 rowsep2">−12.9 </td><td class="colsep2 rowsep2">−15.3 </td><td class="colsep2 rowsep2">−13.1 </td><td class="colsep2 rowsep2">−9.2 </td><td class="colsep2 rowsep2">1.95 </td><td class="colsep2 rowsep2">1.97 </td><td class="colsep2 rowsep2">1hri </td><td class="colsep2 rowsep2">−5.9 </td><td class="colsep2 rowsep2">−8.9 </td><td class="colsep2 rowsep2">−8.9 </td><td class="colsep2 rowsep2">−7.3 </td><td class="colsep2 rowsep2">10.09 </td><td class="colsep2 rowsep2">2.26 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1dd6 </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2">−13.5 </td><td class="colsep2 rowsep2">−13.5 </td><td class="colsep2 rowsep2">−9.1 </td><td class="colsep2 rowsep2">1.36 </td><td class="colsep2 rowsep2">8.27 </td><td class="colsep2 rowsep2">1hsg </td><td class="colsep2 rowsep2">−12.8 </td><td class="colsep2 rowsep2">−12.5 </td><td class="colsep2 rowsep2">−12.5 </td><td class="colsep2 rowsep2">−13.3 </td><td class="colsep2 rowsep2">0.41 </td><td class="colsep2 rowsep2">0.35 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1dds </td><td class="colsep2 rowsep2">−11.3 </td><td class="colsep2 rowsep2">−10.8 </td><td class="colsep2 rowsep2">−10.8 </td><td class="colsep2 rowsep2">−7.4 </td><td class="colsep2 rowsep2">1.75 </td><td class="colsep2 rowsep2">2.38 </td><td class="colsep2 rowsep2">1hsl </td><td class="colsep2 rowsep2">−9.8 </td><td class="colsep2 rowsep2">−8.4 </td><td class="colsep2 rowsep2">−6.4 </td><td class="colsep2 rowsep2">−9.4 </td><td class="colsep2 rowsep2">1.31 </td><td class="colsep2 rowsep2">1.31 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1dhf </td><td class="colsep2 rowsep2">−10.1 </td><td class="colsep2 rowsep2">−8.7 </td><td class="colsep2 rowsep2">−8.7 </td><td class="colsep2 rowsep2">−8.5 </td><td class="colsep2 rowsep2">6.31 </td><td class="colsep2 rowsep2">5.62 </td><td class="colsep2 rowsep2">1hte </td><td class="colsep2 rowsep2">−8.6 </td><td class="colsep2 rowsep2">−9.9 </td><td class="colsep2 rowsep2">−9.9 </td><td class="colsep2 rowsep2">−9.6 </td><td class="colsep2 rowsep2">7.32* </td><td class="colsep2 rowsep2">7.36 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1did </td><td class="colsep2 rowsep2">−4.8 </td><td class="colsep2 rowsep2">−3.8 </td><td class="colsep2 rowsep2">−3.8 </td><td class="colsep2 rowsep2">−6.6 </td><td class="colsep2 rowsep2">3.27 </td><td class="colsep2 rowsep2">4.15 </td><td class="colsep2 rowsep2">1htf </td><td class="colsep2 rowsep2">−9.3 </td><td class="colsep2 rowsep2">−10.9 </td><td class="colsep2 rowsep2">−10.9 </td><td class="colsep2 rowsep2">−9.4 </td><td class="colsep2 rowsep2">2.19 </td><td class="colsep2 rowsep2">2.74 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1die </td><td class="colsep2 rowsep2">−2.9 </td><td class="colsep2 rowsep2">−5.8 </td><td class="colsep2 rowsep2">−5.8 </td><td class="colsep2 rowsep2">−6.9 </td><td class="colsep2 rowsep2">0.34 </td><td class="colsep2 rowsep2">0.77 </td><td class="colsep2 rowsep2">1hti </td><td class="colsep2 rowsep2">−7.0 </td><td class="colsep2 rowsep2">−4.5 </td><td class="colsep2 rowsep2">−4.5 </td><td class="colsep2 rowsep2">−5.7 </td><td class="colsep2 rowsep2">4.40 </td><td class="colsep2 rowsep2">1.60 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1dih </td><td class="colsep2 rowsep2">−7.8 </td><td class="colsep2 rowsep2">−8.2 </td><td class="colsep2 rowsep2">−8.2 </td><td class="colsep2 rowsep2">−14.3 </td><td class="colsep2 rowsep2">2.62 </td><td class="colsep2 rowsep2">1.78 </td><td class="colsep2 rowsep2">1hvr </td><td class="colsep2 rowsep2">−13.0 </td><td class="colsep2 rowsep2">−13.7 </td><td class="colsep2 rowsep2">−13.7 </td><td class="colsep2 rowsep2">−7.8 </td><td class="colsep2 rowsep2">1.60 </td><td class="colsep2 rowsep2">1.75 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1dm2 </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2">−13.9 </td><td class="colsep2 rowsep2">−13.9 </td><td class="colsep2 rowsep2">−10.3 </td><td class="colsep2 rowsep2">0.66 </td><td class="colsep2 rowsep2">0.69 </td><td class="colsep2 rowsep2">1hyt </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2">−8.0 </td><td class="colsep2 rowsep2">−8.0 </td><td class="colsep2 rowsep2">−10.7 </td><td class="colsep2 rowsep2">2.65 </td><td class="colsep2 rowsep2">0.43 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1dog </td><td class="colsep2 rowsep2">−5.5 </td><td class="colsep2 rowsep2">−6.5 </td><td class="colsep2 rowsep2">−6.5 </td><td class="colsep2 rowsep2">−9.5 </td><td class="colsep2 rowsep2">3.77 </td><td class="colsep2 rowsep2">3.77 </td><td class="colsep2 rowsep2">1icn </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2">−10.3 </td><td class="colsep2 rowsep2">−7.8 </td><td class="colsep2 rowsep2">−1.7 </td><td class="colsep2 rowsep2">9.06 </td><td class="colsep2 rowsep2">2.04 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1dr1 </td><td class="colsep2 rowsep2">−7.4 </td><td class="colsep2 rowsep2">−7.6 </td><td class="colsep2 rowsep2">−7.6 </td><td class="colsep2 rowsep2">−7.0 </td><td class="colsep2 rowsep2">0.37 </td><td class="colsep2 rowsep2">1.46 </td><td class="colsep2 rowsep2">1ida </td><td class="colsep2 rowsep2">−11.9 </td><td class="colsep2 rowsep2">−13.1 </td><td class="colsep2 rowsep2">−13.1 </td><td class="colsep2 rowsep2">−12.2 </td><td class="colsep2 rowsep2">1.95 </td><td class="colsep2 rowsep2">2.12 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1dwb </td><td class="colsep2 rowsep2">−4.0 </td><td class="colsep2 rowsep2">−6.0 </td><td class="colsep2 rowsep2">−4.0 </td><td class="colsep2 rowsep2">−7.7 </td><td class="colsep2 rowsep2">0.29 </td><td class="colsep2 rowsep2">0.32 </td><td class="colsep2 rowsep2">1igj </td><td class="colsep2 rowsep2"><i><sup>b</sup></i><sup></sup></td><td class="colsep2 rowsep2">−9.2 </td><td class="colsep2 rowsep2">−9.2 </td><td class="colsep2 rowsep2">−7.0 </td><td class="colsep2 rowsep2">0.72 </td><td class="colsep2 rowsep2">0.46 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1dwc </td><td class="colsep2 rowsep2">−10.3 </td><td class="colsep2 rowsep2">−9.3 </td><td class="colsep2 rowsep2">−9.3 </td><td class="colsep2 rowsep2">−8.8 </td><td class="colsep2 rowsep2">2.06 </td><td class="colsep2 rowsep2">0.89 </td><td class="colsep2 rowsep2">1imb </td><td class="colsep2 rowsep2">−5.7 </td><td class="colsep2 rowsep2">−6.6 </td><td class="colsep2 rowsep2">−6.6 </td><td class="colsep2 rowsep2">−10.0 </td><td class="colsep2 rowsep2">1.84 </td><td class="colsep2 rowsep2">1.64 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1dwd </td><td class="colsep2 rowsep2">−11.4 </td><td class="colsep2 rowsep2">−13.2 </td><td class="colsep2 rowsep2">−11.2 </td><td class="colsep2 rowsep2">−11.2 </td><td class="colsep2 rowsep2">0.47 </td><td class="colsep2 rowsep2">1.43 </td><td class="colsep2 rowsep2">1ivb </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2">−5.2 </td><td class="colsep2 rowsep2">−5.2 </td><td class="colsep2 rowsep2">−7.0 </td><td class="colsep2 rowsep2">3.27 </td><td class="colsep2 rowsep2">0.47</td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1ivc </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2">−3.4 </td><td class="colsep2 rowsep2">−3.4 </td><td class="colsep2 rowsep2">−5.0 </td><td class="colsep2 rowsep2">2.05 </td><td class="colsep2 rowsep2">1.89 </td><td class="colsep2 rowsep2">1wap </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2">−8.1 </td><td class="colsep2 rowsep2">−8.1 </td><td class="colsep2 rowsep2">−10.2 </td><td class="colsep2 rowsep2">0.23 </td><td class="colsep2 rowsep2">0.19 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1ivd </td><td class="colsep2 rowsep2">−4.3 </td><td class="colsep2 rowsep2">−6.3 </td><td class="colsep2 rowsep2">−6.3 </td><td class="colsep2 rowsep2">−6.2 </td><td class="colsep2 rowsep2">0.73 </td><td class="colsep2 rowsep2">0.73 </td><td class="colsep2 rowsep2">1xid </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2">−5.7 </td><td class="colsep2 rowsep2">−5.7 </td><td class="colsep2 rowsep2">−6.6 </td><td class="colsep2 rowsep2">4.02 </td><td class="colsep2 rowsep2">4.32 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1ive </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2">−3.6 </td><td class="colsep2 rowsep2">−3.6 </td><td class="colsep2 rowsep2">−5.6 </td><td class="colsep2 rowsep2">5.11 </td><td class="colsep2 rowsep2">5.17 </td><td class="colsep2 rowsep2">1xie </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2">−4.8 </td><td class="colsep2 rowsep2">−4.8 </td><td class="colsep2 rowsep2">−6.8 </td><td class="colsep2 rowsep2">2.60 </td><td class="colsep2 rowsep2">3.91 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1ivf </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2">−7.7 </td><td class="colsep2 rowsep2">−7.7 </td><td class="colsep2 rowsep2">−6.8 </td><td class="colsep2 rowsep2">0.61 </td><td class="colsep2 rowsep2">0.59 </td><td class="colsep2 rowsep2">2ack </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2">−9.3 </td><td class="colsep2 rowsep2">−9.3 </td><td class="colsep2 rowsep2">−7.0 </td><td class="colsep2 rowsep2">1.07 </td><td class="colsep2 rowsep2">0.88</td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1lah </td><td class="colsep2 rowsep2">−10.3 </td><td class="colsep2 rowsep2">−7.3 </td><td class="colsep2 rowsep2">−7.3 </td><td class="colsep2 rowsep2">−9.6 </td><td class="colsep2 rowsep2">0.52 </td><td class="colsep2 rowsep2">0.19 </td><td class="colsep2 rowsep2">2ada </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2">−9.5 </td><td class="colsep2 rowsep2">−9.5 </td><td class="colsep2 rowsep2">−9.0 </td><td class="colsep2 rowsep2">0.59 </td><td class="colsep2 rowsep2">0.46 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1lcp </td><td class="colsep2 rowsep2">−9.1 </td><td class="colsep2 rowsep2">−7.7 </td><td class="colsep2 rowsep2">−7.7 </td><td class="colsep2 rowsep2">−8.8 </td><td class="colsep2 rowsep2">1.82 </td><td class="colsep2 rowsep2">1.06 </td><td class="colsep2 rowsep2">2cgr </td><td class="colsep2 rowsep2">−9.9 </td><td class="colsep2 rowsep2">−6.5 </td><td class="colsep2 rowsep2">−6.5 </td><td class="colsep2 rowsep2">−10.8 </td><td class="colsep2 rowsep2">0.56 </td><td class="colsep2 rowsep2">0.52 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1ldm </td><td class="colsep2 rowsep2">−7.4 </td><td class="colsep2 rowsep2">−6.3 </td><td class="colsep2 rowsep2">−6.3 </td><td class="colsep2 rowsep2">−7.3 </td><td class="colsep2 rowsep2">1.34 </td><td class="colsep2 rowsep2">1.35 </td><td class="colsep2 rowsep2">2cht </td><td class="colsep2 rowsep2">−7.5 </td><td class="colsep2 rowsep2">−7.7 </td><td class="colsep2 rowsep2">−7.7 </td><td class="colsep2 rowsep2">−11.4 </td><td class="colsep2 rowsep2">0.48 </td><td class="colsep2 rowsep2">0.51</td></tr></tbody></table></div></div><div class="NLM_table-wrap" id="jm051256ot00020"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><p class="first last"> Table 5.  Continued</p></div></div><div class="scrollable-table-wrap"><table class="table " width="100%"><colgroup><col /></col><col /></col><col /></col><col /></col><col /></col><col /></col><col /></col><col /></col><col /></col><col /></col><col /></col><col /></col><col /></col><col /></col></colgroup><tbody><tr valign="top" class="colsep2"><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2" colspan="3">GlideScore (kcal/mol)</td><td class="colsep2 rowsep2" colspan="2">RMS (Å)</td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2" colspan="3">GlideScore (kcal/mol)</td><td class="colsep2 rowsep2" colspan="2">RMS (Å)</td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">PDB</td><td class="colsep2 rowsep2">Δ<i>G</i><sub>exp</sub></td><td class="colsep2 rowsep2">XP</td><td class="colsep2 rowsep2">XP-corr</td><td class="colsep2 rowsep2">SP</td><td class="colsep2 rowsep2">XP</td><td class="colsep2 rowsep2">SP</td><td class="colsep2 rowsep2">PDB</td><td class="colsep2 rowsep2">Δ<i>G</i><sub>exp</sub></td><td class="colsep2 rowsep2">XP</td><td class="colsep2 rowsep2">XP-corr</td><td class="colsep2 rowsep2">SP</td><td class="colsep2 rowsep2">XP</td><td class="colsep2 rowsep2">SP </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1lic </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2">−6.1 </td><td class="colsep2 rowsep2">−6.1 </td><td class="colsep2 rowsep2">−5.3 </td><td class="colsep2 rowsep2">3.96 </td><td class="colsep2 rowsep2">4.99 </td><td class="colsep2 rowsep2">2cmd </td><td class="colsep2 rowsep2">−6.2 </td><td class="colsep2 rowsep2">−8.2 </td><td class="colsep2 rowsep2">−8.2 </td><td class="colsep2 rowsep2">−10.3 </td><td class="colsep2 rowsep2">0.65 </td><td class="colsep2 rowsep2">0.34 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1lmo </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2">−7.6 </td><td class="colsep2 rowsep2">−7.6 </td><td class="colsep2 rowsep2">−6.8 </td><td class="colsep2 rowsep2">8.40 </td><td class="colsep2 rowsep2">0.87 </td><td class="colsep2 rowsep2">2cpp </td><td class="colsep2 rowsep2">−8.3 </td><td class="colsep2 rowsep2">−8.2 </td><td class="colsep2 rowsep2">−8.2 </td><td class="colsep2 rowsep2">−6.8 </td><td class="colsep2 rowsep2">0.15 </td><td class="colsep2 rowsep2">3.04 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1lna </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2">−6.6 </td><td class="colsep2 rowsep2">−6.6 </td><td class="colsep2 rowsep2">−6.8 </td><td class="colsep2 rowsep2">1.50 </td><td class="colsep2 rowsep2">0.90 </td><td class="colsep2 rowsep2">2ctc </td><td class="colsep2 rowsep2">−5.3 </td><td class="colsep2 rowsep2">−7.4 </td><td class="colsep2 rowsep2">−7.4 </td><td class="colsep2 rowsep2">−10.1 </td><td class="colsep2 rowsep2">1.43 </td><td class="colsep2 rowsep2">1.58 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1lst </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2">−6.1 </td><td class="colsep2 rowsep2">−4.1 </td><td class="colsep2 rowsep2">−7.7 </td><td class="colsep2 rowsep2">0.75 </td><td class="colsep2 rowsep2">0.27 </td><td class="colsep2 rowsep2">2dbl </td><td class="colsep2 rowsep2">−11.8 </td><td class="colsep2 rowsep2">−12.2 </td><td class="colsep2 rowsep2">−11.0 </td><td class="colsep2 rowsep2">−8.8 </td><td class="colsep2 rowsep2">2.40 </td><td class="colsep2 rowsep2">0.81 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1mbi </td><td class="colsep2 rowsep2">−2.6 </td><td class="colsep2 rowsep2">−3.9 </td><td class="colsep2 rowsep2">−3.9 </td><td class="colsep2 rowsep2">−4.1 </td><td class="colsep2 rowsep2">1.92 </td><td class="colsep2 rowsep2">1.65 </td><td class="colsep2 rowsep2">2gbp </td><td class="colsep2 rowsep2">−10.1 </td><td class="colsep2 rowsep2">−9.0 </td><td class="colsep2 rowsep2">−9.0 </td><td class="colsep2 rowsep2">−12.5 </td><td class="colsep2 rowsep2">0.61 </td><td class="colsep2 rowsep2">0.14 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1mcr </td><td class="colsep2 rowsep2">−4.3 </td><td class="colsep2 rowsep2">−8.3 </td><td class="colsep2 rowsep2">−8.3 </td><td class="colsep2 rowsep2">−7.5 </td><td class="colsep2 rowsep2">5.82 </td><td class="colsep2 rowsep2">4.33 </td><td class="colsep2 rowsep2">2ifb </td><td class="colsep2 rowsep2">−7.4 </td><td class="colsep2 rowsep2">−8.7 </td><td class="colsep2 rowsep2">−6.2 </td><td class="colsep2 rowsep2">−2.4 </td><td class="colsep2 rowsep2">2.27 </td><td class="colsep2 rowsep2">1.77 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1mdr </td><td class="colsep2 rowsep2">−5.4 </td><td class="colsep2 rowsep2">−8.2 </td><td class="colsep2 rowsep2">−8.2 </td><td class="colsep2 rowsep2">−9.6 </td><td class="colsep2 rowsep2">1.95 </td><td class="colsep2 rowsep2">0.54 </td><td class="colsep2 rowsep2">2lgs </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2">−3.4 </td><td class="colsep2 rowsep2">−3.4 </td><td class="colsep2 rowsep2">−9.5 </td><td class="colsep2 rowsep2">0.88 </td><td class="colsep2 rowsep2">0.53 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1mfe </td><td class="colsep2 rowsep2">−7.2 </td><td class="colsep2 rowsep2">−8.0 </td><td class="colsep2 rowsep2">−8.0 </td><td class="colsep2 rowsep2">−7.4 </td><td class="colsep2 rowsep2">6.09 </td><td class="colsep2 rowsep2">1.78 </td><td class="colsep2 rowsep2">2mcp </td><td class="colsep2 rowsep2">−7.1 </td><td class="colsep2 rowsep2">−5.6 </td><td class="colsep2 rowsep2">−5.6 </td><td class="colsep2 rowsep2">−5.6 </td><td class="colsep2 rowsep2">1.54 </td><td class="colsep2 rowsep2">1.25 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1mld </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2">−6.6 </td><td class="colsep2 rowsep2">−6.6 </td><td class="colsep2 rowsep2">−10.6 </td><td class="colsep2 rowsep2">0.25 </td><td class="colsep2 rowsep2">0.22 </td><td class="colsep2 rowsep2">2phh </td><td class="colsep2 rowsep2">−6.4 </td><td class="colsep2 rowsep2">−7.9 </td><td class="colsep2 rowsep2">−7.9 </td><td class="colsep2 rowsep2">−8.7 </td><td class="colsep2 rowsep2">0.47 </td><td class="colsep2 rowsep2">0.41 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1mmq </td><td class="colsep2 rowsep2">−10.3 </td><td class="colsep2 rowsep2">−13.8 </td><td class="colsep2 rowsep2">−13.8 </td><td class="colsep2 rowsep2">−11.0 </td><td class="colsep2 rowsep2">0.67 </td><td class="colsep2 rowsep2">0.33 </td><td class="colsep2 rowsep2">2pk4 </td><td class="colsep2 rowsep2">−5.9 </td><td class="colsep2 rowsep2">−8.8 </td><td class="colsep2 rowsep2">−6.8 </td><td class="colsep2 rowsep2">−5.8 </td><td class="colsep2 rowsep2">0.65 </td><td class="colsep2 rowsep2">0.85 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1mnc </td><td class="colsep2 rowsep2">−12.3 </td><td class="colsep2 rowsep2">−11.9 </td><td class="colsep2 rowsep2">−11.9 </td><td class="colsep2 rowsep2">−11.1 </td><td class="colsep2 rowsep2">0.33 </td><td class="colsep2 rowsep2">0.73 </td><td class="colsep2 rowsep2">2plv </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2">−14.3 </td><td class="colsep2 rowsep2">−11.8 </td><td class="colsep2 rowsep2">−7.3 </td><td class="colsep2 rowsep2">1.78 </td><td class="colsep2 rowsep2">1.90 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1mrg </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2">−7.8 </td><td class="colsep2 rowsep2">−7.8 </td><td class="colsep2 rowsep2">−7.8 </td><td class="colsep2 rowsep2">0.15 </td><td class="colsep2 rowsep2">0.12 </td><td class="colsep2 rowsep2">2r04 </td><td class="colsep2 rowsep2">−8.0 </td><td class="colsep2 rowsep2">−10.6 </td><td class="colsep2 rowsep2">−8.1 </td><td class="colsep2 rowsep2">−8.9 </td><td class="colsep2 rowsep2">1.44 </td><td class="colsep2 rowsep2">0.75 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1mrk </td><td class="colsep2 rowsep2">−6.2 </td><td class="colsep2 rowsep2">−11.3 </td><td class="colsep2 rowsep2">−11.3 </td><td class="colsep2 rowsep2">−9.4 </td><td class="colsep2 rowsep2">1.21 </td><td class="colsep2 rowsep2">1.17 </td><td class="colsep2 rowsep2">2r07 </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2">−10.9 </td><td class="colsep2 rowsep2">−8.4 </td><td class="colsep2 rowsep2">−8.7 </td><td class="colsep2 rowsep2">0.92 </td><td class="colsep2 rowsep2">0.67 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1mup </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2">−9.3 </td><td class="colsep2 rowsep2">−7.6 </td><td class="colsep2 rowsep2">−6.2 </td><td class="colsep2 rowsep2">4.50 </td><td class="colsep2 rowsep2">4.05 </td><td class="colsep2 rowsep2">2sim </td><td class="colsep2 rowsep2">−4.7 </td><td class="colsep2 rowsep2">−7.1 </td><td class="colsep2 rowsep2">−7.1 </td><td class="colsep2 rowsep2">−10.5 </td><td class="colsep2 rowsep2">0.82 </td><td class="colsep2 rowsep2">0.94 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1nco </td><td class="colsep2 rowsep2">−10.6 </td><td class="colsep2 rowsep2">−10.4 </td><td class="colsep2 rowsep2">−10.4 </td><td class="colsep2 rowsep2">−12.1 </td><td class="colsep2 rowsep2">0.60 </td><td class="colsep2 rowsep2">0.33 </td><td class="colsep2 rowsep2">2tmn </td><td class="colsep2 rowsep2">−8.0 </td><td class="colsep2 rowsep2">−10.1 </td><td class="colsep2 rowsep2">−10.1 </td><td class="colsep2 rowsep2">−10.4 </td><td class="colsep2 rowsep2">0.65 </td><td class="colsep2 rowsep2">0.50 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1nis </td><td class="colsep2 rowsep2">−4.1 </td><td class="colsep2 rowsep2">−4.0 </td><td class="colsep2 rowsep2">−4.0 </td><td class="colsep2 rowsep2">−7.5 </td><td class="colsep2 rowsep2">0.26 </td><td class="colsep2 rowsep2">0.45 </td><td class="colsep2 rowsep2">2tpi </td><td class="colsep2 rowsep2">−5.9 </td><td class="colsep2 rowsep2">−8.6 </td><td class="colsep2 rowsep2">−6.6 </td><td class="colsep2 rowsep2">−9.5 </td><td class="colsep2 rowsep2">0.26 </td><td class="colsep2 rowsep2">1.15 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1nnb </td><td class="colsep2 rowsep2">−7.2 </td><td class="colsep2 rowsep2">−5.5 </td><td class="colsep2 rowsep2">−5.5 </td><td class="colsep2 rowsep2">−8.7 </td><td class="colsep2 rowsep2">1.39 </td><td class="colsep2 rowsep2">0.25 </td><td class="colsep2 rowsep2">2upj </td><td class="colsep2 rowsep2">−14.2 </td><td class="colsep2 rowsep2">−11.1 </td><td class="colsep2 rowsep2">−11.1 </td><td class="colsep2 rowsep2">−10.7 </td><td class="colsep2 rowsep2">3.24 </td><td class="colsep2 rowsep2">2.69 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1nsc </td><td class="colsep2 rowsep2">−4.1 </td><td class="colsep2 rowsep2">−5.7 </td><td class="colsep2 rowsep2">−5.7 </td><td class="colsep2 rowsep2">−10.5 </td><td class="colsep2 rowsep2">0.66 </td><td class="colsep2 rowsep2">0.24 </td><td class="colsep2 rowsep2">2xis </td><td class="colsep2 rowsep2">−7.9 </td><td class="colsep2 rowsep2">−4.9 </td><td class="colsep2 rowsep2">−4.9 </td><td class="colsep2 rowsep2">−7.3 </td><td class="colsep2 rowsep2">2.05 </td><td class="colsep2 rowsep2">2.40 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1nsd </td><td class="colsep2 rowsep2">−7.2 </td><td class="colsep2 rowsep2">−6.3 </td><td class="colsep2 rowsep2">−6.3 </td><td class="colsep2 rowsep2">−9.0 </td><td class="colsep2 rowsep2">0.74 </td><td class="colsep2 rowsep2">0.22 </td><td class="colsep2 rowsep2">2yhx </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2">−4.3 </td><td class="colsep2 rowsep2">−4.3 </td><td class="colsep2 rowsep2">−5.6 </td><td class="colsep2 rowsep2">1.91 </td><td class="colsep2 rowsep2">2.19 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1odw </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2">−11.5 </td><td class="colsep2 rowsep2">−11.5 </td><td class="colsep2 rowsep2">−5.7 </td><td class="colsep2 rowsep2">3.91 </td><td class="colsep2 rowsep2">2.59 </td><td class="colsep2 rowsep2">3cla </td><td class="colsep2 rowsep2">−6.7 </td><td class="colsep2 rowsep2">−6.4 </td><td class="colsep2 rowsep2">−6.4 </td><td class="colsep2 rowsep2">−5.4 </td><td class="colsep2 rowsep2">5.09 </td><td class="colsep2 rowsep2">6.06 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1okl </td><td class="colsep2 rowsep2">−8.2 </td><td class="colsep2 rowsep2">−7.2 </td><td class="colsep2 rowsep2">−7.2 </td><td class="colsep2 rowsep2">−5.8 </td><td class="colsep2 rowsep2">0.38 </td><td class="colsep2 rowsep2">3.14 </td><td class="colsep2 rowsep2">3dfr </td><td class="colsep2 rowsep2">−14.0 </td><td class="colsep2 rowsep2">−15.3 </td><td class="colsep2 rowsep2">−13.3 </td><td class="colsep2 rowsep2">−11.3 </td><td class="colsep2 rowsep2">0.51 </td><td class="colsep2 rowsep2">0.70 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1pbd </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2">−11.4 </td><td class="colsep2 rowsep2">−11.4 </td><td class="colsep2 rowsep2">−9.9 </td><td class="colsep2 rowsep2">0.32 </td><td class="colsep2 rowsep2">0.26 </td><td class="colsep2 rowsep2">3hvt </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2">−11.9 </td><td class="colsep2 rowsep2">−11.4 </td><td class="colsep2 rowsep2">−9.0 </td><td class="colsep2 rowsep2">0.72 </td><td class="colsep2 rowsep2">0.79 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1pgp </td><td class="colsep2 rowsep2">−7.8 </td><td class="colsep2 rowsep2">−6.3 </td><td class="colsep2 rowsep2">−6.3 </td><td class="colsep2 rowsep2">−8.8 </td><td class="colsep2 rowsep2">2.23 </td><td class="colsep2 rowsep2">1.83 </td><td class="colsep2 rowsep2">3mth </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2">−5.4 </td><td class="colsep2 rowsep2">−5.4 </td><td class="colsep2 rowsep2">−5.9 </td><td class="colsep2 rowsep2">1.23 </td><td class="colsep2 rowsep2">5.62 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1pha </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2">−12.2 </td><td class="colsep2 rowsep2">−12.2 </td><td class="colsep2 rowsep2">−8.8 </td><td class="colsep2 rowsep2">1.04 </td><td class="colsep2 rowsep2">0.52 </td><td class="colsep2 rowsep2">3ptb </td><td class="colsep2 rowsep2">−6.1 </td><td class="colsep2 rowsep2">−6.5 </td><td class="colsep2 rowsep2">−6.5 </td><td class="colsep2 rowsep2">−8.8 </td><td class="colsep2 rowsep2">0.23 </td><td class="colsep2 rowsep2">0.16 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1phd </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2">−7.4 </td><td class="colsep2 rowsep2">−7.4 </td><td class="colsep2 rowsep2">−6.5 </td><td class="colsep2 rowsep2">1.13 </td><td class="colsep2 rowsep2">0.30 </td><td class="colsep2 rowsep2">3tpi </td><td class="colsep2 rowsep2">−5.9 </td><td class="colsep2 rowsep2">−9.8 </td><td class="colsep2 rowsep2">−7.8 </td><td class="colsep2 rowsep2">−9.0 </td><td class="colsep2 rowsep2">0.47 </td><td class="colsep2 rowsep2">0.51 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1phf </td><td class="colsep2 rowsep2">−6.0 </td><td class="colsep2 rowsep2">−8.2 </td><td class="colsep2 rowsep2">−8.1 </td><td class="colsep2 rowsep2">−6.3 </td><td class="colsep2 rowsep2">1.46 </td><td class="colsep2 rowsep2">1.11 </td><td class="colsep2 rowsep2">4aah </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2">−7.6 </td><td class="colsep2 rowsep2">−7.6 </td><td class="colsep2 rowsep2">−11.0 </td><td class="colsep2 rowsep2">0.25 </td><td class="colsep2 rowsep2">0.24 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1phg </td><td class="colsep2 rowsep2">−11.8 </td><td class="colsep2 rowsep2">−10.9 </td><td class="colsep2 rowsep2">−10.9 </td><td class="colsep2 rowsep2">−8.1 </td><td class="colsep2 rowsep2">4.29 </td><td class="colsep2 rowsep2">1.20 </td><td class="colsep2 rowsep2">4cts </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2">−4.6 </td><td class="colsep2 rowsep2">−4.6 </td><td class="colsep2 rowsep2">−8.8 </td><td class="colsep2 rowsep2">0.25 </td><td class="colsep2 rowsep2">0.19 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1poc </td><td class="colsep2 rowsep2">−10.7 </td><td class="colsep2 rowsep2">−8.4 </td><td class="colsep2 rowsep2">−8.4 </td><td class="colsep2 rowsep2">−10.1 </td><td class="colsep2 rowsep2">1.44 </td><td class="colsep2 rowsep2">1.52 </td><td class="colsep2 rowsep2">4dfr </td><td class="colsep2 rowsep2">−11.8 </td><td class="colsep2 rowsep2">−10.1 </td><td class="colsep2 rowsep2">−10.1 </td><td class="colsep2 rowsep2">−9.3 </td><td class="colsep2 rowsep2">0.92 </td><td class="colsep2 rowsep2">5.17 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1ppc </td><td class="colsep2 rowsep2">−8.4 </td><td class="colsep2 rowsep2">−11.9 </td><td class="colsep2 rowsep2">−9.9 </td><td class="colsep2 rowsep2">−9.8 </td><td class="colsep2 rowsep2">1.40 </td><td class="colsep2 rowsep2">6.31 </td><td class="colsep2 rowsep2">4fab </td><td class="colsep2 rowsep2">−11.0 </td><td class="colsep2 rowsep2">−13.7 </td><td class="colsep2 rowsep2">−12.9 </td><td class="colsep2 rowsep2">−9.1 </td><td class="colsep2 rowsep2">4.44 </td><td class="colsep2 rowsep2">0.82 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1pph </td><td class="colsep2 rowsep2">−8.1 </td><td class="colsep2 rowsep2">−10.5 </td><td class="colsep2 rowsep2">−8.5 </td><td class="colsep2 rowsep2">−10.4 </td><td class="colsep2 rowsep2">0.70 </td><td class="colsep2 rowsep2">0.58 </td><td class="colsep2 rowsep2">4fbp </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2">−9.7 </td><td class="colsep2 rowsep2">−9.7 </td><td class="colsep2 rowsep2">−12.1 </td><td class="colsep2 rowsep2">2.03 </td><td class="colsep2 rowsep2">0.55 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1ppi </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2">−16.7 </td><td class="colsep2 rowsep2">−14.7 </td><td class="colsep2 rowsep2">−8.5 </td><td class="colsep2 rowsep2">1.01 </td><td class="colsep2 rowsep2">2.80 </td><td class="colsep2 rowsep2">4fxn </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2">−17.3 </td><td class="colsep2 rowsep2">−17.3 </td><td class="colsep2 rowsep2">−13.2 </td><td class="colsep2 rowsep2">0.50 </td><td class="colsep2 rowsep2">0.49 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1ppk </td><td class="colsep2 rowsep2">−10.4 </td><td class="colsep2 rowsep2">−9.9 </td><td class="colsep2 rowsep2">−9.9 </td><td class="colsep2 rowsep2">−10.1 </td><td class="colsep2 rowsep2">0.73 </td><td class="colsep2 rowsep2">0.27 </td><td class="colsep2 rowsep2">4hmg </td><td class="colsep2 rowsep2">−3.5 </td><td class="colsep2 rowsep2">−6.6 </td><td class="colsep2 rowsep2">−6.6 </td><td class="colsep2 rowsep2">−6.7 </td><td class="colsep2 rowsep2">0.54 </td><td class="colsep2 rowsep2">0.67 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1ppl </td><td class="colsep2 rowsep2">−11.7 </td><td class="colsep2 rowsep2">−10.3 </td><td class="colsep2 rowsep2">−10.3 </td><td class="colsep2 rowsep2">−11.6 </td><td class="colsep2 rowsep2">0.70 </td><td class="colsep2 rowsep2">2.55 </td><td class="colsep2 rowsep2">4phv </td><td class="colsep2 rowsep2">−12.5 </td><td class="colsep2 rowsep2">−14.5 </td><td class="colsep2 rowsep2">−14.5 </td><td class="colsep2 rowsep2">−11.8 </td><td class="colsep2 rowsep2">0.55 </td><td class="colsep2 rowsep2">4.22 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1ppm </td><td class="colsep2 rowsep2">−7.9 </td><td class="colsep2 rowsep2">−12.5 </td><td class="colsep2 rowsep2">−12.5 </td><td class="colsep2 rowsep2">−11.7 </td><td class="colsep2 rowsep2">0.99 </td><td class="colsep2 rowsep2">0.62 </td><td class="colsep2 rowsep2">4tim </td><td class="colsep2 rowsep2">−2.9 </td><td class="colsep2 rowsep2">−5.3 </td><td class="colsep2 rowsep2">−5.3 </td><td class="colsep2 rowsep2">−10.2 </td><td class="colsep2 rowsep2">1.32 </td><td class="colsep2 rowsep2">1.32 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1pro </td><td class="colsep2 rowsep2">−15.4 </td><td class="colsep2 rowsep2">−15.6 </td><td class="colsep2 rowsep2">−15.6 </td><td class="colsep2 rowsep2">−12.7 </td><td class="colsep2 rowsep2">1.50 </td><td class="colsep2 rowsep2">1.51 </td><td class="colsep2 rowsep2">4tln </td><td class="colsep2 rowsep2">−5.1 </td><td class="colsep2 rowsep2">−7.1 </td><td class="colsep2 rowsep2">−7.1 </td><td class="colsep2 rowsep2">−6.6 </td><td class="colsep2 rowsep2">1.43 </td><td class="colsep2 rowsep2">2.67 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1pso </td><td class="colsep2 rowsep2">−14.1 </td><td class="colsep2 rowsep2">−10.9 </td><td class="colsep2 rowsep2">−10.9 </td><td class="colsep2 rowsep2">−9.6 </td><td class="colsep2 rowsep2">5.15 </td><td class="colsep2 rowsep2">6.12 </td><td class="colsep2 rowsep2">4tmn </td><td class="colsep2 rowsep2">−13.9 </td><td class="colsep2 rowsep2">−11.9 </td><td class="colsep2 rowsep2">−11.9 </td><td class="colsep2 rowsep2">−12.1 </td><td class="colsep2 rowsep2">1.29 </td><td class="colsep2 rowsep2">0.73 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1sbg </td><td class="colsep2 rowsep2">−10.6 </td><td class="colsep2 rowsep2">−11.1 </td><td class="colsep2 rowsep2">−11.1 </td><td class="colsep2 rowsep2">−10.9 </td><td class="colsep2 rowsep2">0.88 </td><td class="colsep2 rowsep2">0.40 </td><td class="colsep2 rowsep2">4tpi </td><td class="colsep2 rowsep2">−4.0 </td><td class="colsep2 rowsep2">−6.6 </td><td class="colsep2 rowsep2">−6.6 </td><td class="colsep2 rowsep2">−8.5 </td><td class="colsep2 rowsep2">0.88 </td><td class="colsep2 rowsep2">0.56 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1slt </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2">−5.8 </td><td class="colsep2 rowsep2">−5.8 </td><td class="colsep2 rowsep2">−6.2 </td><td class="colsep2 rowsep2">1.03 </td><td class="colsep2 rowsep2">0.57 </td><td class="colsep2 rowsep2">4ts1 </td><td class="colsep2 rowsep2">−6.7 </td><td class="colsep2 rowsep2">−8.6 </td><td class="colsep2 rowsep2">−8.6 </td><td class="colsep2 rowsep2">−8.8 </td><td class="colsep2 rowsep2">0.89 </td><td class="colsep2 rowsep2">0.85 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1snc </td><td class="colsep2 rowsep2">−9.1 </td><td class="colsep2 rowsep2">−9.6 </td><td class="colsep2 rowsep2">−9.6 </td><td class="colsep2 rowsep2">−9.8 </td><td class="colsep2 rowsep2">2.06 </td><td class="colsep2 rowsep2">1.12 </td><td class="colsep2 rowsep2">5abp </td><td class="colsep2 rowsep2">−9.1 </td><td class="colsep2 rowsep2">−7.6 </td><td class="colsep2 rowsep2">−7.6 </td><td class="colsep2 rowsep2">−8.5 </td><td class="colsep2 rowsep2">0.11 </td><td class="colsep2 rowsep2">0.20 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1sre </td><td class="colsep2 rowsep2">−5.3 </td><td class="colsep2 rowsep2">−8.9 </td><td class="colsep2 rowsep2">−8.5 </td><td class="colsep2 rowsep2">−10.0 </td><td class="colsep2 rowsep2">0.30 </td><td class="colsep2 rowsep2">0.36 </td><td class="colsep2 rowsep2">5cpp </td><td class="colsep2 rowsep2">−8.0 </td><td class="colsep2 rowsep2">−8.2 </td><td class="colsep2 rowsep2">−8.2 </td><td class="colsep2 rowsep2">−6.8 </td><td class="colsep2 rowsep2">0.11 </td><td class="colsep2 rowsep2">2.65 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1srj </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2">−13.7 </td><td class="colsep2 rowsep2">−12.9 </td><td class="colsep2 rowsep2">−10.0 </td><td class="colsep2 rowsep2">0.47 </td><td class="colsep2 rowsep2">0.49 </td><td class="colsep2 rowsep2">5cts </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2">−3.2 </td><td class="colsep2 rowsep2">−3.2 </td><td class="colsep2 rowsep2">−8.1 </td><td class="colsep2 rowsep2">0.27 </td><td class="colsep2 rowsep2">0.27 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1stp </td><td class="colsep2 rowsep2">−18.3 </td><td class="colsep2 rowsep2">−18.2 </td><td class="colsep2 rowsep2">−18.2 </td><td class="colsep2 rowsep2">−10.7 </td><td class="colsep2 rowsep2">0.60 </td><td class="colsep2 rowsep2">0.58 </td><td class="colsep2 rowsep2">5p2p </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2">−9.0 </td><td class="colsep2 rowsep2">−9.0 </td><td class="colsep2 rowsep2">−5.4 </td><td class="colsep2 rowsep2">4.95 </td><td class="colsep2 rowsep2">6.18 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1tdb </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2">−7.9 </td><td class="colsep2 rowsep2">−7.9 </td><td class="colsep2 rowsep2">−7.6 </td><td class="colsep2 rowsep2">7.34 </td><td class="colsep2 rowsep2">7.50 </td><td class="colsep2 rowsep2">5tim </td><td class="colsep2 rowsep2">−3.1 </td><td class="colsep2 rowsep2">−3.2 </td><td class="colsep2 rowsep2">−3.2 </td><td class="colsep2 rowsep2">−7.3 </td><td class="colsep2 rowsep2">1.32 </td><td class="colsep2 rowsep2">0.69 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1thy </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2">−7.0 </td><td class="colsep2 rowsep2">−7.0 </td><td class="colsep2 rowsep2">−7.2 </td><td class="colsep2 rowsep2">1.98 </td><td class="colsep2 rowsep2">4.21 </td><td class="colsep2 rowsep2">5tln </td><td class="colsep2 rowsep2">−8.7 </td><td class="colsep2 rowsep2">−12.4 </td><td class="colsep2 rowsep2">−12.4 </td><td class="colsep2 rowsep2">−9.9 </td><td class="colsep2 rowsep2">2.37 </td><td class="colsep2 rowsep2">1.01 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1tka </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2">−7.7 </td><td class="colsep2 rowsep2">−7.7 </td><td class="colsep2 rowsep2">−11.5 </td><td class="colsep2 rowsep2">2.28 </td><td class="colsep2 rowsep2">2.28 </td><td class="colsep2 rowsep2">5tmn </td><td class="colsep2 rowsep2">−11.0 </td><td class="colsep2 rowsep2">−12.0 </td><td class="colsep2 rowsep2">−12.0 </td><td class="colsep2 rowsep2">−12.1 </td><td class="colsep2 rowsep2">2.87 </td><td class="colsep2 rowsep2">2.50 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1tlp </td><td class="colsep2 rowsep2">−10.3 </td><td class="colsep2 rowsep2">−10.0 </td><td class="colsep2 rowsep2">−10.0 </td><td class="colsep2 rowsep2">−10.4 </td><td class="colsep2 rowsep2">7.70 </td><td class="colsep2 rowsep2">7.33 </td><td class="colsep2 rowsep2">6abp </td><td class="colsep2 rowsep2">−7.7 </td><td class="colsep2 rowsep2">−7.7 </td><td class="colsep2 rowsep2">−7.7 </td><td class="colsep2 rowsep2">−8.8 </td><td class="colsep2 rowsep2">0.33 </td><td class="colsep2 rowsep2">0.36 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1tmn </td><td class="colsep2 rowsep2">−10.0 </td><td class="colsep2 rowsep2">−10.2 </td><td class="colsep2 rowsep2">−10.2 </td><td class="colsep2 rowsep2">−10.4 </td><td class="colsep2 rowsep2">3.65 </td><td class="colsep2 rowsep2">1.90 </td><td class="colsep2 rowsep2">6cpa </td><td class="colsep2 rowsep2">−15.7 </td><td class="colsep2 rowsep2">−10.8 </td><td class="colsep2 rowsep2">−10.8 </td><td class="colsep2 rowsep2">−10.8 </td><td class="colsep2 rowsep2">3.93 </td><td class="colsep2 rowsep2">4.29 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1tng </td><td class="colsep2 rowsep2">−4.0 </td><td class="colsep2 rowsep2">−3.9 </td><td class="colsep2 rowsep2">−3.9 </td><td class="colsep2 rowsep2">−8.3 </td><td class="colsep2 rowsep2">0.26 </td><td class="colsep2 rowsep2">0.21 </td><td class="colsep2 rowsep2">6rnt </td><td class="colsep2 rowsep2">−3.2 </td><td class="colsep2 rowsep2">−3.7 </td><td class="colsep2 rowsep2">−3.7 </td><td class="colsep2 rowsep2">−8.2 </td><td class="colsep2 rowsep2">0.63 </td><td class="colsep2 rowsep2">0.63 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1tnh </td><td class="colsep2 rowsep2">−4.6 </td><td class="colsep2 rowsep2">−3.9 </td><td class="colsep2 rowsep2">−3.9 </td><td class="colsep2 rowsep2">−8.7 </td><td class="colsep2 rowsep2">0.39 </td><td class="colsep2 rowsep2">0.28 </td><td class="colsep2 rowsep2">6tim </td><td class="colsep2 rowsep2">−4.4 </td><td class="colsep2 rowsep2">−5.9 </td><td class="colsep2 rowsep2">−5.9 </td><td class="colsep2 rowsep2">−8.3 </td><td class="colsep2 rowsep2">0.59 </td><td class="colsep2 rowsep2">0.42 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1tni </td><td class="colsep2 rowsep2">−5.5 </td><td class="colsep2 rowsep2">−3.7 </td><td class="colsep2 rowsep2">−3.7 </td><td class="colsep2 rowsep2">−6.7 </td><td class="colsep2 rowsep2">2.12 </td><td class="colsep2 rowsep2">2.03 </td><td class="colsep2 rowsep2">6tmn </td><td class="colsep2 rowsep2">−6.9 </td><td class="colsep2 rowsep2">−10.4 </td><td class="colsep2 rowsep2">−10.4 </td><td class="colsep2 rowsep2">−11.6 </td><td class="colsep2 rowsep2">2.53 </td><td class="colsep2 rowsep2">2.57 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1tnj </td><td class="colsep2 rowsep2">−2.7 </td><td class="colsep2 rowsep2">−4.1 </td><td class="colsep2 rowsep2">−4.1 </td><td class="colsep2 rowsep2">−7.8 </td><td class="colsep2 rowsep2">0.43 </td><td class="colsep2 rowsep2">0.36 </td><td class="colsep2 rowsep2">7abp </td><td class="colsep2 rowsep2">−8.6 </td><td class="colsep2 rowsep2">−8.0 </td><td class="colsep2 rowsep2">−8.0 </td><td class="colsep2 rowsep2">−9.4 </td><td class="colsep2 rowsep2">0.15 </td><td class="colsep2 rowsep2">0.17 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1tnk </td><td class="colsep2 rowsep2">−2.0 </td><td class="colsep2 rowsep2">−3.8 </td><td class="colsep2 rowsep2">−3.8 </td><td class="colsep2 rowsep2">−6.9 </td><td class="colsep2 rowsep2">1.17 </td><td class="colsep2 rowsep2">0.98 </td><td class="colsep2 rowsep2">7cpa </td><td class="colsep2 rowsep2">−19.0 </td><td class="colsep2 rowsep2">−13.1 </td><td class="colsep2 rowsep2">−13.1 </td><td class="colsep2 rowsep2">−10.5 </td><td class="colsep2 rowsep2">3.91 </td><td class="colsep2 rowsep2">2.87 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1tnl </td><td class="colsep2 rowsep2">−2.6 </td><td class="colsep2 rowsep2">−4.0 </td><td class="colsep2 rowsep2">−4.0 </td><td class="colsep2 rowsep2">−7.4 </td><td class="colsep2 rowsep2">0.54 </td><td class="colsep2 rowsep2">0.24 </td><td class="colsep2 rowsep2">7cpp </td><td class="colsep2 rowsep2">−5.2 </td><td class="colsep2 rowsep2">−6.2 </td><td class="colsep2 rowsep2">−6.2 </td><td class="colsep2 rowsep2">−5.7 </td><td class="colsep2 rowsep2">1.99 </td><td class="colsep2 rowsep2">3.23 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1tph </td><td class="colsep2 rowsep2">−3.1 </td><td class="colsep2 rowsep2">−5.3 </td><td class="colsep2 rowsep2">−5.3 </td><td class="colsep2 rowsep2">−7.5 </td><td class="colsep2 rowsep2">0.22 </td><td class="colsep2 rowsep2">0.23 </td><td class="colsep2 rowsep2">7tim </td><td class="colsep2 rowsep2">−7.4 </td><td class="colsep2 rowsep2">−5.4 </td><td class="colsep2 rowsep2">−5.4 </td><td class="colsep2 rowsep2">−7.7 </td><td class="colsep2 rowsep2">0.20 </td><td class="colsep2 rowsep2">0.19 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1tpp </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2">−7.1 </td><td class="colsep2 rowsep2">−5.1 </td><td class="colsep2 rowsep2">−7.9 </td><td class="colsep2 rowsep2">0.43 </td><td class="colsep2 rowsep2">1.07 </td><td class="colsep2 rowsep2">8abp </td><td class="colsep2 rowsep2">−10.7 </td><td class="colsep2 rowsep2">−7.6 </td><td class="colsep2 rowsep2">−7.6 </td><td class="colsep2 rowsep2">−8.4 </td><td class="colsep2 rowsep2">0.10 </td><td class="colsep2 rowsep2">0.21 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1trk </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2">−9.6 </td><td class="colsep2 rowsep2">−9.6 </td><td class="colsep2 rowsep2">−11.2 </td><td class="colsep2 rowsep2">1.63 </td><td class="colsep2 rowsep2">2.15 </td><td class="colsep2 rowsep2">8atc </td><td class="colsep2 rowsep2">−10.3 </td><td class="colsep2 rowsep2">−9.5 </td><td class="colsep2 rowsep2">−9.5 </td><td class="colsep2 rowsep2">−10.7 </td><td class="colsep2 rowsep2">0.41 </td><td class="colsep2 rowsep2">0.38 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1tyl </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2">−5.2 </td><td class="colsep2 rowsep2">−5.2 </td><td class="colsep2 rowsep2">−6.7 </td><td class="colsep2 rowsep2">5.20 </td><td class="colsep2 rowsep2">1.08 </td><td class="colsep2 rowsep2">8gch </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2">−7.9 </td><td class="colsep2 rowsep2">−7.9 </td><td class="colsep2 rowsep2">−9.6 </td><td class="colsep2 rowsep2">0.32 </td><td class="colsep2 rowsep2">0.30 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1ukz </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2">−11.3 </td><td class="colsep2 rowsep2">−11.3 </td><td class="colsep2 rowsep2">−13.6 </td><td class="colsep2 rowsep2">0.55 </td><td class="colsep2 rowsep2">0.41 </td><td class="colsep2 rowsep2">9abp </td><td class="colsep2 rowsep2">−10.9 </td><td class="colsep2 rowsep2">−7.2 </td><td class="colsep2 rowsep2">−7.2 </td><td class="colsep2 rowsep2">−10.4 </td><td class="colsep2 rowsep2">0.23 </td><td class="colsep2 rowsep2">0.13 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1ulb </td><td class="colsep2 rowsep2">−6.0 </td><td class="colsep2 rowsep2">−6.7 </td><td class="colsep2 rowsep2">−6.7 </td><td class="colsep2 rowsep2">−8.7 </td><td class="colsep2 rowsep2">0.34 </td><td class="colsep2 rowsep2">0.35 </td><td class="colsep2 rowsep2">9hvp </td><td class="colsep2 rowsep2">−11.4 </td><td class="colsep2 rowsep2">−11.9 </td><td class="colsep2 rowsep2">−11.9 </td><td class="colsep2 rowsep2">−12.9 </td><td class="colsep2 rowsep2">1.44 </td><td class="colsep2 rowsep2">1.47</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-"><p class="first last"><i><sup>a</sup></i><sup></sup> RMS values are computed relative to the native cocrystallized ligand. Scores are compared to available experimental binding energies, Δ<i>G</i><sub>exp</sub> (in kcal/mol). Note that the XP scores have been adjusted to cap the hydrophobic enclosure packing term at a value of −2.0 to bring this term closer to an absolute energy scale. Larger RMS values are indicated with a (*) in cases where a nearly chemically symmetric solution was found by XP docking.<i><sup>b</sup></i><sup></sup> The pdbbind dataset,<sup><a onclick="showRef(event, 'jm051256ob00042'); return false;" href="javascript:void(0);" class="ref jm051256ob00042">42</a></sup> v2002, lists a binding affinity of −13.6 kcal/mol, but this is for the entire antigen−antibody complex, whereas the structure provided is for a fragment of the complex that appears to lack essential antigen−antibody interactions.  </p></div></div></div><div class="NLM_p">In our optimization protocol, we employed docked protein−ligand complexes, retaining only those with protein−ligand contacts that predominantly agree with those in crystal structures. In this fashion, we avoid corrupting the fitting process with irrelevant data such as would be provided by a grossly incorrect pose, yet include a realistic level of variation in the input structure. This is particularly critical in optimization of the penalty terms. If the sampling algorithm cannot avoid incorrect penalties in self-docking, it is unlikely to be able to do so in a much more challenging cross-docking situation. By incorporating docked poses of PDB complexes into the optimization process, the penalty function can be tuned to improve the agreement with experimental binding affinities while avoiding inappropriately penalizing active compounds, keeping in mind that there are also cases where the penalty terms are in fact appropriate. </div><div class="NLM_p">Before discussing binding affinity predictions, a key point that has generally been neglected previously should be noted. An empirical scoring function that considers only protein−ligand interactions with no a priori information concerning the apo structure of the protein cannot, by definition, take into account the reorganization energy of the protein required to accommodate the ligand. In many cases, the protein is relatively rigid, the ligand fits without major rearrangements, and neglect of this term is acceptable. However, there are cases where it is overwhelmingly likely that the induced-fit energies are substantial. The most obvious cases are those in which an allosteric pocket is created to accommodate the ligand. This occurs, for example, in nonnucleoside reverse transcriptase inhibitors (NNRTIs) of HIV reverse transcriptase (HIV-RT) and is also manifested in the large-scale motion of the activation loop in p38 MAP kinases required to produce the DFG-out conformation to which inhibitors such as BIRB796 bind. However, there are many more subtle cases in which side chains or backbone groups alter their positions nontrivially, and this should introduce some energetic cost. </div><div class="NLM_p">The goal of an empirical scoring function should be to predict the binding affinity of the ligand to the <i>structure with which it is presented.</i> That is, given the formal impossibility of predicting reorganization energy in any such scheme, this value should be removed from the experimental binding affinity. Otherwise, one will be fitting to an incorrect experimental number, given the goal of the exercise. A perfect scoring function of this type will correctly rank-order candidate ligands in their ability to bind to the structure at hand. If this structure is known to have a low reorganization energy, compounds with good scores when docked into that structure should yield satisfactory experimental binding affinities. For allosteric pockets and other sites with larger reorganization energies, one would expect that more favorable scores would be needed to yield the desired experimental binding affinity. The problem of comparing scores between ligands docked into different conformations of the receptor can then be treated separately. The problem is highly nontrivial, requiring either a heuristic procedure incorporating experimental information or the brute force ability to compare free energies of different protein conformations. </div><div class="NLM_p">A qualitative observation that we have made, confirmed in a large number of examples, is that a large hydrophobic enclosure score is a signature of significant protein rearrangement and possibly creation of an allosteric pocket. Ligands binding tightly to both the HIV-RT NNRTI site and p38 DFG-out conformation, for example, generally receive maximal hydrophobic-enclosure scores. Furthermore, the total XP scores of these ligands are substantially higher in absolute terms than their experimental binding affinities would mandate. This is completely consistent with the ideas discussed above, in which the reorganization energy of the protein must be subtracted from the empirical binding affinity score to produce the correct experimental binding affinity. </div><div class="NLM_p">In the initial Glide XP parameterization with PDB cocrystallized structures, systems with exceptionally large protein reorganization energies, as predicted by large hydrophobic enclosure contributions, have been intentionally omitted. This omission includes allosteric sites such as the HIV-RT and p38 structures mentioned above, and complexes such as staurosporine/CDK2 (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1aq1">1aq1</a>), in which the CDK2 pocket must expand substantially to accommodate the unusually large staurosporine ligand. If one believes that the scoring function is accurate, the protein reorganization energy can be inferred from the computed rigid receptor and experimental binding <a name="jm004902106189" id="jm004902106189"></a> affinities. This idea could form the basis for an approach enabling scores achieved in different protein conformations for a given receptor to be related based on experimental calibration of reorganization energy. Performance of the methodology when there is substantial reorganization is addressed in a preliminary fashion by the enrichment studies in section 4, where a number of such ligand−receptor pairs are considered. In these cases, knowledge of the reorganization energy of the receptor is not necessary to rank order the binding of compounds to a particular form of the receptor. </div><div class="NLM_p">To address less extreme yet still nontrivial reorganization effects present in our data set, we define “average” adjustable parameters to convert calculated empirical scores into predicted experimental binding affinities. In particular, we estimate the <a name="jm004902106190" id="jm004902106190"></a> hydrophobic-enclosure reorganization energy by capping the maximum assignable enclosure energy at a value of 2.0 kcal/mol, as compared to the maximum scoring-function value of 4.5 kcal/mol. It also appears as though the special bidentate charged−charged reward when the ligand group is positively charged involves some degree of protein reorganization, presumably to enable the bidentate salt bridge to be made properly. Thus, this parameter, which normally is 2.0 kcal/mol, was set to 1.0 kcal/mol for comparison of predicted binding affinities to experiment. Other such average effects could be defined, but due to the limited experimental data considered in this paper, no further attempts were made to determine additional parameters. </div><div class="NLM_p">Tables 6 and 7<a class="ref showTableEvent internalNav" data-ID="jm051256ot00006" href="#jm051256ot00006"></a><a class="ref showTableEvent internalNav" data-ID="jm051256ot00007" href="#jm051256ot00007"></a> present results for both 4.0 XP and 4.0 SP Glide for structural RMSDs (taken with respect to the refined structure of the native ligand generated by our standard protein-preparation procedure,<sup><a onclick="showRef(event, 'jm051256ob00040'); return false;" href="javascript:void(0);" class="ref jm051256ob00040">1,40</a></sup> using heavy atoms only) and for binding affinities of the various complexes discussed herein. Table 6 further divides the comparison according to whether the XP complex is roughly correctly docked, that is, has a structural RMSD of 2.5 Å or less. Over all 198 complexes examined, the average RMSD in predicted binding affinity for the 4.0 XP scoring function is 2.3 kcal/mol and the average unsigned error is 1.8 kcal/mol. When only well-docked ligands are examined, the average RMSD for the 4.0 XP scoring function is 1.7 kcal/mol and the average unsigned error is 1.3 kcal/mol. This is a significant improvement over the performance of the 4.0 SP scoring function and is also an improvement over the performances of other empirical scoring functions in the literature of which we are aware. In only 11% (15 of 136) of well-docked ligands were errors greater than 3 kcal/mol, suggesting, at least when the appropriately fitting protein structure is presented to the ligand and the ligand is well-docked, the present scoring function has a respectable ability to distinguish weak (mM), moderate (μM), and strong (nM) binders. This capability is essential to the principal task in virtual screening, yet is only marginally present in prior scoring functions. <div class="NLM_table-wrap" id="jm051256ot00006"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><p class="first last"> Table 6.  RMS and Average Absolute Deviations (Avg) in Predicted Binding Affinities for the XP 4.0 and SP 4.0 Scoring Functions<i><sup>a</sup></i><sup></sup></p></div></div><div class="scrollable-table-wrap"><table class="table " width="100%"><colgroup><col /></col><col /></col><col /></col><col /></col><col /></col><col /></col></colgroup><tbody><tr valign="top" class="colsep2"><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2" colspan="2">XP 4.0</td><td class="colsep2 rowsep2" colspan="2">SP 4.0</td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">comparison</td><td class="colsep2 rowsep2">number</td><td class="colsep2 rowsep2">RMSD</td><td class="colsep2 rowsep2">avg</td><td class="colsep2 rowsep2">RMSD</td><td class="colsep2 rowsep2">avg </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">all ligands with Δ<i>G</i><sub>exp</sub></td><td class="colsep2 rowsep2">198 </td><td class="colsep2 rowsep2">2.26 </td><td class="colsep2 rowsep2">1.75 </td><td class="colsep2 rowsep2">3.18 </td><td class="colsep2 rowsep2">2.51 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">all well-docked ligands </td><td class="colsep2 rowsep2">136 </td><td class="colsep2 rowsep2">1.73 </td><td class="colsep2 rowsep2">1.34 </td><td class="colsep2 rowsep2">2.80 </td><td class="colsep2 rowsep2">2.23 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">all poorly docked ligands </td><td class="colsep2 rowsep2">62 </td><td class="colsep2 rowsep2">3.02 </td><td class="colsep2 rowsep2">2.49 </td><td class="colsep2 rowsep2">3.70 </td><td class="colsep2 rowsep2">3.11</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-"><p class="first last"><i><sup>a</sup></i><sup></sup> RMS and average absolute deviations are presented in kcal/mol. Well-docked ligands are defined as those having an RMSD to the cocrystallized pose of 2.5 Å or less, plus those identified in Table 5 as being docked appropriately but in a symmetry-related orientation.</p></div></div><div class="NLM_table-wrap" id="jm051256ot00007"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><p class="first last"> Table 7.  Training Set Used to Characterize XP Virtual Screening<i><sup>a</sup></i><sup></sup></p></div></div><div class="scrollable-table-wrap"><table class="table " width="100%"><colgroup><col /></col><col /></col><col /></col><col /></col></colgroup><tbody><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">PDB code</td><td class="colsep2 rowsep2">description</td><td class="colsep2 rowsep2">no. actives</td><td class="colsep2 rowsep2">no. well-docked actives </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1e66 </td><td class="colsep2 rowsep2">acetylcholinesterase </td><td class="colsep2 rowsep2">20 </td><td class="colsep2 rowsep2">20 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1bji </td><td class="colsep2 rowsep2">neuraminidase </td><td class="colsep2 rowsep2">9 </td><td class="colsep2 rowsep2">9 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1fjs </td><td class="colsep2 rowsep2">factor Xa </td><td class="colsep2 rowsep2">13 </td><td class="colsep2 rowsep2">8 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1kv2 </td><td class="colsep2 rowsep2">human p38 map kinase </td><td class="colsep2 rowsep2">10 </td><td class="colsep2 rowsep2">10 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1bl7 </td><td class="colsep2 rowsep2">p38 map kinase </td><td class="colsep2 rowsep2">36 </td><td class="colsep2 rowsep2">27 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1rt1 </td><td class="colsep2 rowsep2">HIV-RT </td><td class="colsep2 rowsep2">29 </td><td class="colsep2 rowsep2">23 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1cx2 </td><td class="colsep2 rowsep2">cyclooxygenase-2 </td><td class="colsep2 rowsep2">13 </td><td class="colsep2 rowsep2">13 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1aq1 </td><td class="colsep2 rowsep2">human cyclin dep. kinase </td><td class="colsep2 rowsep2">10 </td><td class="colsep2 rowsep2">6 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1ett </td><td class="colsep2 rowsep2">thrombin </td><td class="colsep2 rowsep2">16 </td><td class="colsep2 rowsep2">15 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1hpx </td><td class="colsep2 rowsep2">HIV-1 protease </td><td class="colsep2 rowsep2">14 </td><td class="colsep2 rowsep2">9 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">3ert </td><td class="colsep2 rowsep2">human estrogen receptor </td><td class="colsep2 rowsep2">10 </td><td class="colsep2 rowsep2">8 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1qpe </td><td class="colsep2 rowsep2">lck kinase </td><td class="colsep2 rowsep2">121 </td><td class="colsep2 rowsep2">87 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1m17 </td><td class="colsep2 rowsep2">EGRF tyrosine kinase </td><td class="colsep2 rowsep2">117 </td><td class="colsep2 rowsep2">106 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1tmn </td><td class="colsep2 rowsep2">thermolysin </td><td class="colsep2 rowsep2">6 </td><td class="colsep2 rowsep2">5 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1kim </td><td class="colsep2 rowsep2">thymidine kinase </td><td class="colsep2 rowsep2">4 </td><td class="colsep2 rowsep2">4</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-"><p class="first last"><i><sup>a</sup></i><sup></sup> All correctly docked ligands have experimental activities <10 μM except those for neuraminidase.  </p></div></div></div><div class="NLM_p">To improve beyond this level with regard to precision, we believe that receptor flexibility must be introduced and an additional level of detail with regard to the protein−ligand interactions must be incorporated. Robustness is another matter; new moieties and chemistries seem to emerge as additional receptors are added to the test suite. Thus, we cannot claim to have reached convergence in this regard with our current data sets. On the other hand, the amount of detailed medicinal-chemistry information incorporated into the current scoring function is a substantial advance as compared to alternative scoring functions in the literature.<sup>8-19</sup></div><div class="NLM_p last">A final important point is that the accuracy cited above may quantitatively degrade in cross-docking calculations, even when the ligand is able to assume a qualitatively correct pose in the receptor, as of course would the accuracy of other docking/scoring methods, for similar reasons. The enrichment studies presented below address this question to some extent, but at present do not consider the relative rankings of different active compounds. We have carried out some preliminary investigations (data not shown) that suggest qualitatively reasonable results can be obtained in a limited number of test cases for ranking active compounds when cross docking is required, but these results are not yet robust across a wide range of receptors, and the quantitative precision with which this can be done is not yet clear. </div></div><div id="" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i24">4. Enrichment Studies</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_63064" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_63064" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">The goal of the present work is to optimize a scoring function that will properly assign binding affinities to active compounds if the compound is docked in a sufficiently native-like pose and that will minimize the number of inactive database ligands that score well. The question of what is sufficiently native-like is a heuristic one, but we would argue that precision in drawing the line is not critical. If a few additional compounds are included, or excluded, this will have minimal effect on the overall optimization process. Therefore, visual inspection has been employed to construct data sets of active compounds that “fit” into the particular versions of the receptors employed. Typically, 60−100% of the available active compounds per receptor fell into this category, so the current protocol is not simply cherry-picking of a small number of compounds. On the other hand, the fact that poorly fitting compounds are not included in the data set must be taken into account when comparing with other results reported in the literature. </div><div class="NLM_p">We have divided our data set into a training set and a test set. The training set screens have been used in parametrizing the XP scoring function, while the test set screens have not. The training set contains 15 receptors and various numbers of ligands for each receptor, as enumerated in Table 7. A large and diverse training set is essential to address the range of chemical motifs identified in the XP Glide scoring function, as noted previously. Our present test set is relatively small and less diverse than the training set with regard to the number of new receptors considered. Therefore, the validation implied by the results must be considered preliminary. Six receptors are considered in the test set, four of which were not included in the training set:  vascular endothelial growth factor receptor 2 (Vegfr2), peroxisome proliferators activated receptor γ (PPARγ), β-secretase (BACE), and blood coagulation factor VIIa (factor VIIa). For two training set receptors, CDK2 and thrombin, we <a name="jm004902106191" id="jm004902106191"></a> have located a significant number of additional ligands to include as part of the test set. </div><div class="NLM_p"><b>Training Set Results.</b> Table 7 lists, for the training set, the receptors investigated, the number of known active ligands available with affinities better than 10 μM, and the number of ligands deemed to dock correctly into the chosen receptor. There is one exception; active neuraminidase ligands are relatively weak binders that do not have activities better than 10 μM. As suggested above, we assume that a random database will contain relatively few compounds with potency greater than this value. Given the process defined above, a key objective of the paper is to demonstrate the improvement that is obtained from employing the new terms defined above. Comparison of version 4.0 XP Glide is made with the following: </div><div class="NLM_p">(1) Version 4.0 SP Glide, which has been optimized using a similar training set. The 4.0 SP Glide scoring function includes some XP terms, but with very small coefficients. In addition to terms such as those in ChemScore, from which SP Glide was originally derived, this scoring function also includes contributions from the Coulomb and vdW protein−ligand interaction energies and from Schrödinger's “active-site mapping” technology. </div><div class="NLM_p">(2) A preliminary version of XP Glide (v2.7). This version of XP had a nonoptimal set of penalty terms, an initial version of the hydrophobic-enclosure term, a crude representation of the special hydrogen-bond reward term without the crucial coupling to the hydrophobic-enclosure term, and a sampling methodology significantly inferior to that in Glide XP 4.0. This comparison enables assessment of the impact of the improved sampling and scoring methodologies used in Glide XP 4.0. </div><div class="NLM_p"><b>Protein and Ligand Preparation.</b> Because the XP Glide scoring function is based on enforcement of physical chemical principles to a much greater degree than is employed in many other scoring functions, appropriate protein and ligand preparation is particularly critical. In practical applications, it is often necessary to carry out such preparation without prior knowledge of the binding mode of the complex. However, for the present purposes, the objective is to optimize and evaluate the scoring function for correctly docked compounds. Therefore, we have endeavored to use all available information in preparing ligand and protein structures. </div><div class="NLM_p">The most problematic aspect of protein and ligand preparation is the assignment of protonation states of ligand and protein in the protein active site (note that we neutralize ionizable residues distant from the active site to mimic the effects of dielectric screening by solvent and counterions). In the absence of structural data, correctly making such assignments can be a very challenging task. However, when structural data is available, the most likely protonation states of both protein and ligand can usually be deduced from the structure of the complex. In cases for which we do not have a PDB structure, the correct binding mode of the ligand can typically be inferred by analogy. With a binding mode and solution-phase p<i>K</i><sub>a</sub>s of the ligand, the correct protonation states can then be assigned. It should be emphasized that the results shown here require accurate protonation-state assignment, and substantial degradation can result from incorrect assignments in unfavorable cases. </div><div class="NLM_p">A second aspect of protein preparation is relaxation of the receptor structure so that it at least accommodates the native ligand. We employ the standard Schrödinger protein preparation utility<sup>1,40</sup> for this purpose. A related issue is the use of van der Waals scaling of nonpolar ligand and protein atoms to take minor induced-fit effects into account in an approximate fashion. Various scalings have been examined though, with the exception of the human estrogen receptor (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3ert">3ert</a>), which used a scaling of 0.8 on the ligand and 0.9 on the protein, the “standard” scaling of 0.8 on the ligand and no protein scaling has been applied. Only active ligands that succeed in 4.0 XP docking with the chosen scaling parameters have been retained in our enrichment studies. </div><div class="NLM_p"><b>Comparison Database.</b> Our methods for generating comparison databases are outlined in ref 1. Molecules are selected from a purchasable-compound library of about one million compounds that have been filtered for predicted pharmacokinetic properties using the QikProp program.<sup><a onclick="showRef(event, 'jm051256ob00041'); return false;" href="javascript:void(0);" class="ref jm051256ob00041">41</a></sup> Selection protocols are then applied to ensure a distribution of rings, acceptors, donors, molecular weight, and so on in line with averages determined for drug-like molecules. </div><div class="NLM_p"><b>Computed Binding Affinities of Known Actives for a Wide Range of </b><b>Targets.</b> As stated above, our expectation is that only a small number of database ligands will be competitive with active compounds whose experimental binding affinities are better than 10 μM. An ensemble of such compounds can, therefore, be used to optimize the scoring function. Because of the wide range of novel terms that have been incorporated, it has been necessary to perform optimizations using a wide variety of receptors and active compounds. The data set used to date is far from complete in covering the range of potential binding motifs, but is significantly larger and more diverse than any previous data set used in the literature for this purpose. </div><div class="NLM_p">The optimization process consists of adjusting parameters so that as few database ligands as possible achieve better scores than the tight-binding (better than 10 μM) known actives for the term or terms under consideration. In practice, it is not possible to achieve perfect rank ordering in this regard, for the reasons discussed previously. The average error in binding affinity prediction in the PDB data set is about 1.7 kcal/mol for properly docked ligands, with some outliers with errors of more than 3 kcal/mol. These errors may be somewhat larger when cross docking, rather than self-docking, is performed. On the basis of this analysis, some database ligands with experimental binding affinities in the 10−100 μM range are likely to be computed as having low micromolar or even nanomolar binding affinity, while at the same time some of the active compounds will be underpredicted by similar amounts. As a result, overpredicted database ligands can score ahead of underpredicted active compounds. The crucial goal, however, is not to achieve perfection, but rather to eliminate systematic errors that can lead to large numbers of false positives and little enrichment. Neglect of special neutral−neutral hydrogen-bond terms such as those discussed in section 2 is an example of a systematic error of this nature. One would predict that any method that neglects this term should exhibit poor enrichment factors for kinases such as CDK2, where such hydrogen bonds are a crucial component of the molecular-recognition motif. This is because we have found that large hydrophobic compounds with a few strategically placed polar groups can “fit” into the active site and form structures that, based on the usual pair-scoring terms, are highly competitive with the known actives. </div><div class="NLM_p">Another, and perhaps the ultimate, measure of performance of the scoring function is whether high-ranking database ligands are in fact active. The best way to address this issue is via experimental testing of such database ligands. We are in the process of carrying out such tests. </div><div class="NLM_p"><b>Training Set Composition.</b> Our training set, primarily focused on pharmaceutical targets of current interest, is presented in Table 7. This suite represents a wide variety of <a name="jm004902106192" id="jm004902106192"></a> different types of active sites and binding motifs. A rough classification of these is as follows: </div><div class="NLM_p">(1) Small hydrophobic sites:  HIV-RT (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1rt1">1rt1</a>) and cyclooxygenase-2 (Cox-2, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1cx2">1cx2</a>). The HIV-RT NNRTI site is an allosteric pocket that opens to accommodate the ligand; the Cox-2 site does not display as dramatic a structural rearrangement, but is also highly hydrophobic. The dominant terms of the scoring function are the pair hydrophobic term and the hydrophobic-enclosure term. The hydrophobic enclosure is quite large in both cases for known active compounds. In HIV-RT, the active compounds typically make a single hydrogen bond to a backbone carbonyl that receives the special single neutral−neutral hydrogen-bond reward discussed in section 2. In Cox-2, there is also typically a single hydrogen bond, though it does not receive a special reward. </div><div class="NLM_p">(2) Medium-sized sites making a single special hydrogen bond. This category includes EGFR tyrosine kinase and the <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1bl7">1bl7</a> form of p38 MAP kinase. This motif is one of the two typical kinase binding motifs, in which there is a special hydrogen-bonding site in the hinge region of the kinase. These systems allow only a single hydrogen bond in this site, typically involving a ring nitrogen atom, whereas other kinases form a correlated pair or triplet of hydrogen bonds, discussed in (3) below. </div><div class="NLM_p">(3) Sites making correlated hydrogen bonds. This category includes thymidine kinase (TK), CDK2, and lck kinase (LCK). TK and LCK form a pair of correlated hydrogen bonds in the standard kinase hinge region, while CDK2 actives form either a pair or a triplet, and neutral actives binding to aldose reductase form a correlated triplet similar to that in the streptavidin/biotin pair mentioned above. TK is a small, relatively polar site, with the binding driven primarily by the special hydrogen bonding, whereas CDK2 and LCK are medium-sized and more hydrophobic, binding ligands in the 400−500 molecular weight range, although smaller ligands can bind as well. </div><div class="NLM_p">(4) Large, buried predominantly hydrophobic sites. This category includes the human estrogen receptor and the <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1kv2">1kv2</a> conformation of p38 MAP kinase. The estrogen receptor binds large, flat steroid-type molecules and exhibits a medium-sized hydrophobic enclosure term, with the binding driven by this term and by the pair hydrophobic score. The <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1kv2">1kv2</a> active site is created by an allosteric rearrangement of the p38 activation loop. It has a large hydrophobic enclosure term, and many active compounds make one to two hydrogen bonds. The most active compound, ligand <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1kv2">1kv2</a>, also makes a special hydrogen bond in the hinge region, but binding is primarily driven by hydrophobic terms. The p38 pocket in particular is quite large and, in the absence of the hydrophobic enclosure term, will display a significant number of false positives that bind alternative motifs in the cavity in database screening. </div><div class="NLM_p">(5) Large, open sites with relatively shallow cavities in the active site pocket. This category includes thrombin (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1ett">1ett</a>), HIV protease (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1hpx">1hpx</a>), and factor Xa (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1fjs">1fjs</a>). Active ligands in this case are invariably large and fill multiple pockets in the active site. HIV protease actives make a substantial number of hydrogen bonds and derive their binding affinity from this and the hydrophobic pair term. Remarkably, there is no contribution from the hydrophobic enclosure. Thrombin ligands typically form a salt bridge with a buried Asp 189 carboxylate in one of the small available pockets and form other hydrogen bonds as well. Hydrophobic enclosure also makes no contribution for thrombin ligands. Factor Xa exhibits the same salt-bridge motif but also has an unusual hydrophobic location in which a ring moiety of the ligand is sandwiched between a number of aromatic rings in a location near the surface of the protein. This pocket provides some hydrophobic enclosure, although not to the same degree as a deeply buried pocket like that in <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1kv2">1kv2</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1cx2">1cx2</a>, or <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1rt1">1rt1</a> and can also accommodate pi-cation and pi stacking interactions. Discriminating false positives that display interactions with the protein surface rather than in binding pockets is important for these receptors, particularly factor Xa. </div><div class="NLM_p">(6) Small hydrophilic sites in which strong electrostatic interactions are important. This category includes the neuramididase (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1bji">1bji</a>) and GluR2 receptors. In neuramididase, much of the binding affinity derives from salt bridges, and the various ligands serve as important test cases for the rules for charged−charged interactions laid out in section 2. In GluR2, an unusual set of charged ligands in which charge is distributed over a ring system, are buried in the active site. This system was used to calibrate the use of electrostatic terms to turn off buried-charge penalties and to assign an additional contribution to binding affinity in exceptional cases, as discussed in section 2. </div><div class="NLM_p">(7) The acetylcholinesterase (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1e66">1e66</a>) receptor was included to incorporate the well-known pi-cation motif of the active compounds in the parameterization process. </div><div class="NLM_p">Table 8<a class="ref showTableEvent internalNav" data-ID="jm051256ot00008" href="#jm051256ot00008"></a> lists the average contribution of the hydrophobic enclosure and special hydrogen-bonding terms to the scores for known active compounds that bind to each of the above targets, as well as the average total score. Note that the total score cannot be directly translated into the predicted binding affinity in all cases because there are a number of targets where allosteric rearrangement of the binding pocket leads to substantial protein <a name="jm004902106193" id="jm004902106193"></a> reorganization energy. Figures 2−13<a data-tab="pane-pcw-Figures" href="#jm051256of00010" id="" class=" internalNav"></a><a data-tab="pane-pcw-Figures" href="#jm051256of00011" id="" class=" internalNav"></a><a data-tab="pane-pcw-Figures" href="#jm051256of00012" id="" class=" internalNav"></a><a data-tab="pane-pcw-Figures" href="#jm051256of00013" id="" class=" internalNav"></a> provide illustrative examples of a “typical” active ligand binding to the various receptors illustrating features that contribute to the specialized scoring-function terms as appropriate. <div id="_i25"><figure id="jm051256of00010" data-index="10" class="article__inlineFigure"><h2 class="fig-label"></h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2006/jmcmar.2006.49.issue-21/jm051256o/production/images/medium/jm051256of00010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2006/jmcmar.2006.49.issue-21/jm051256o/production/images/large/jm051256of00010.jpeg" id="_i26" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last"> Figure 10 SCX-001 bound to cyclooxygenase-2. The three phenyl rings obtain a 3.7 kcal/mol hydrophobic enclosure packing reward for occupying the large hydrophobic pocket.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2006/jmcmar.2006.49.issue-21/jm051256o/production/images/large/jm051256of00010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm051256o&amp;id=jm051256of00010"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="jm051256of00011" data-index="11" class="article__inlineFigure"><h2 class="fig-label"></h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2006/jmcmar.2006.49.issue-21/jm051256o/production/images/medium/jm051256of00011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2006/jmcmar.2006.49.issue-21/jm051256o/production/images/large/jm051256of00011.jpeg" id="_i27" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last"> Figure 11 The <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1lpk">1lpk</a> ligand bound to factor Xa. A special salt bridge pair with ASP 189, analogous to that in thrombin (Figure 8), is observed. A hydrophobic enclosure packing reward of −0.8 kcal/mol is achieved by the phenyl ring occupying the Tyr 99/Trp 215/Phe 174 pocket. Also, a pi-cation interaction is received by the charged end of the ligand.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2006/jmcmar.2006.49.issue-21/jm051256o/production/images/large/jm051256of00011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm051256o&amp;id=jm051256of00011"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="jm051256of00012" data-index="12" class="article__inlineFigure"><h2 class="fig-label"></h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2006/jmcmar.2006.49.issue-21/jm051256o/production/images/medium/jm051256of00012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2006/jmcmar.2006.49.issue-21/jm051256o/production/images/large/jm051256of00012.jpeg" id="_i28" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last"> Figure 12 The <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1hpx">1hpx</a> ligand bound to HIV-1 protease. Hydrophobic groups of the ligand are not hydrophobically enclosed and do not receive a hydrophobic enclosure packing reward. For example, the phenyl ring faces hydrophobic residues on only one face.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2006/jmcmar.2006.49.issue-21/jm051256o/production/images/large/jm051256of00012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm051256o&amp;id=jm051256of00012"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="jm051256of00013" data-index="13" class="article__inlineFigure"><h2 class="fig-label"></h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2006/jmcmar.2006.49.issue-21/jm051256o/production/images/medium/jm051256of00013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2006/jmcmar.2006.49.issue-21/jm051256o/production/images/large/jm051256of00013.jpeg" id="_i29" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last"> Figure 13 4-Hydroxytamoxifen bound to the human estrogen receptor. Hydrophobic enclosure about the phenoxy group is illustrated by displaying lipophilic protein atoms as green spheres.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2006/jmcmar.2006.49.issue-21/jm051256o/production/images/large/jm051256of00013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm051256o&amp;id=jm051256of00013"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div class="NLM_table-wrap" id="jm051256ot00008"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><p class="first last"> Table 8.  Average Attractive Components of 4.0 XP Score from Eq 3 of Correctly Docked Active Ligands in the Training Set<i><sup>a</sup></i><sup></sup></p></div></div><div class="scrollable-table-wrap"><table class="table " width="100%"><colgroup><col /></col><col /></col><col /></col><col /></col><col /></col><col /></col><col /></col><col /></col></colgroup><tbody><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">screen</td><td class="colsep2 rowsep2">〈<i>E</i><sub>phobic_pair</sub>〉</td><td class="colsep2 rowsep2">〈<i>E</i><sub>hb_pair</sub>〉</td><td class="colsep2 rowsep2">〈<i>E</i><sub>hb_nn_motif</sub>〉</td><td class="colsep2 rowsep2">〈<i>E</i><sub>hb_cc_motif</sub>〉</td><td class="colsep2 rowsep2">〈<i>E</i><sub>hyd_enclosure</sub>〉</td><td class="colsep2 rowsep2">〈<i>E</i><sub>PI</sub>〉</td><td class="colsep2 rowsep2">〈<i>E</i><sub>bind</sub>〉 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">acetylcholinesterase </td><td class="colsep2 rowsep2">−12.1 </td><td class="colsep2 rowsep2">−0.3 </td><td class="colsep2 rowsep2">0.0 </td><td class="colsep2 rowsep2">−1.9 </td><td class="colsep2 rowsep2">−4.4 </td><td class="colsep2 rowsep2">−1.1 </td><td class="colsep2 rowsep2">−20.8 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">neuraminidase </td><td class="colsep2 rowsep2">−3.8 </td><td class="colsep2 rowsep2">−0.7 </td><td class="colsep2 rowsep2">−0.4 </td><td class="colsep2 rowsep2">−3.6 </td><td class="colsep2 rowsep2">0.0 </td><td class="colsep2 rowsep2">0.0 </td><td class="colsep2 rowsep2">−8.6 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">factor Xa </td><td class="colsep2 rowsep2">−8.3 </td><td class="colsep2 rowsep2">−0.8 </td><td class="colsep2 rowsep2">−1.8 </td><td class="colsep2 rowsep2">−1.2 </td><td class="colsep2 rowsep2">−0.8 </td><td class="colsep2 rowsep2">−0.8 </td><td class="colsep2 rowsep2">−14.1 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">human p38 map kinase </td><td class="colsep2 rowsep2">−10.2 </td><td class="colsep2 rowsep2">−1.6 </td><td class="colsep2 rowsep2">−0.3 </td><td class="colsep2 rowsep2">0.0 </td><td class="colsep2 rowsep2">−2.6 </td><td class="colsep2 rowsep2">0.0 </td><td class="colsep2 rowsep2">−14.7 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">p38 map kinase </td><td class="colsep2 rowsep2">−7.4 </td><td class="colsep2 rowsep2">−1.2 </td><td class="colsep2 rowsep2">0.0 </td><td class="colsep2 rowsep2">0.0 </td><td class="colsep2 rowsep2">−0.9 </td><td class="colsep2 rowsep2">0.0 </td><td class="colsep2 rowsep2">−9.5 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">HIV-RT </td><td class="colsep2 rowsep2">−9.1 </td><td class="colsep2 rowsep2">−1.0 </td><td class="colsep2 rowsep2">−1.4 </td><td class="colsep2 rowsep2">0.0 </td><td class="colsep2 rowsep2">−4.2 </td><td class="colsep2 rowsep2">0.0 </td><td class="colsep2 rowsep2">−15.7 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">cyclooxygenase-2 </td><td class="colsep2 rowsep2">−8.9 </td><td class="colsep2 rowsep2">−0.3 </td><td class="colsep2 rowsep2">0.0 </td><td class="colsep2 rowsep2">0.0 </td><td class="colsep2 rowsep2">−3.2 </td><td class="colsep2 rowsep2">0.0 </td><td class="colsep2 rowsep2">−12.5 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">human cyclin dep. kinase </td><td class="colsep2 rowsep2">−7.8 </td><td class="colsep2 rowsep2">−2.1 </td><td class="colsep2 rowsep2">−3.2 </td><td class="colsep2 rowsep2">0.0 </td><td class="colsep2 rowsep2">−1.5 </td><td class="colsep2 rowsep2">0.0 </td><td class="colsep2 rowsep2">−14.7 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">thrombin </td><td class="colsep2 rowsep2">−8.4 </td><td class="colsep2 rowsep2">−1.7 </td><td class="colsep2 rowsep2">0.0 </td><td class="colsep2 rowsep2">−3.0 </td><td class="colsep2 rowsep2">−0.4 </td><td class="colsep2 rowsep2">0.0 </td><td class="colsep2 rowsep2">−13.1 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">HIV-1 protease </td><td class="colsep2 rowsep2">−10.2 </td><td class="colsep2 rowsep2">−2.6 </td><td class="colsep2 rowsep2">−0.2 </td><td class="colsep2 rowsep2">0.0 </td><td class="colsep2 rowsep2">0.0 </td><td class="colsep2 rowsep2">0.0 </td><td class="colsep2 rowsep2">−13.1 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">human estrogen receptor </td><td class="colsep2 rowsep2">−10.6 </td><td class="colsep2 rowsep2">−1.2 </td><td class="colsep2 rowsep2">−0.5 </td><td class="colsep2 rowsep2">0.0 </td><td class="colsep2 rowsep2">−2.0 </td><td class="colsep2 rowsep2">0.0 </td><td class="colsep2 rowsep2">−14.3 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">lck kinase </td><td class="colsep2 rowsep2">−8.1 </td><td class="colsep2 rowsep2">−1.4 </td><td class="colsep2 rowsep2">−1.3 </td><td class="colsep2 rowsep2">0.0 </td><td class="colsep2 rowsep2">0.0 </td><td class="colsep2 rowsep2">0.0 </td><td class="colsep2 rowsep2">−10.8 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">EGRF tyrosine kinase </td><td class="colsep2 rowsep2">−6.7 </td><td class="colsep2 rowsep2">−1.3 </td><td class="colsep2 rowsep2">−1.5 </td><td class="colsep2 rowsep2">0.0 </td><td class="colsep2 rowsep2">0.0 </td><td class="colsep2 rowsep2">0.0 </td><td class="colsep2 rowsep2">−9.5 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">thermolysin </td><td class="colsep2 rowsep2">−8.7 </td><td class="colsep2 rowsep2">−2.3 </td><td class="colsep2 rowsep2">0.0 </td><td class="colsep2 rowsep2">−0.3 </td><td class="colsep2 rowsep2">−0.1 </td><td class="colsep2 rowsep2">0.0 </td><td class="colsep2 rowsep2">−11.4 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">thymidine kinase </td><td class="colsep2 rowsep2">−5.5 </td><td class="colsep2 rowsep2">−2.3 </td><td class="colsep2 rowsep2">−3.0 </td><td class="colsep2 rowsep2">0.0 </td><td class="colsep2 rowsep2">−1.4 </td><td class="colsep2 rowsep2">0.0 </td><td class="colsep2 rowsep2">−12.6</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-"><p class="first last"><i><sup>a</sup></i><sup></sup> <i>E</i><sub>p</sub><sub>hobic_</sub><sub>p</sub><sub>air</sub> is the pair lipophilic term (eq 1), <i>E</i><sub>h</sub><sub>b_</sub><sub>p</sub><sub>air</sub> is the Chemscore-like pair hydrogen-bond term, <i>E</i><sub>h</sub><sub>b_</sub><sub>n</sub><sub>n_</sub><sub>m</sub><sub>otif</sub> is the term for neutral−neutral hydrogen bonds in a hydrophobically enclosed environment, <i>E</i><sub>h</sub><sub>b_</sub><sub>c</sub><sub>c_</sub><sub>m</sub><sub>otif</sub> is the term for special charged−charged hydrogen bonds, <i>E</i><sub>h</sub><sub>yd_</sub><sub>e</sub><sub>nclosure</sub> is the hydrophobic enclosure reward, <i>E</i><sub>PI</sub> is the pi-stacking/pi-cation reward, and <i>E</i><sub>b</sub><sub>ind</sub> is the sum of all terms in eq 3 that account for favorable binding affinities.</p></div></div></div><div class="NLM_p"><b>Results of Training Set Enrichment Studies.</b> Table 9<a class="ref showTableEvent internalNav" data-ID="jm051256ot00009" href="#jm051256ot00009"></a> reports a measure of enrichment defined as the average number of database ligands outranking the active compounds in the database. Specifically, the number of database ligands with a GlideScore that is superior to each active is tabulated, these values are summed, and the result is then divided by the total number of active compounds in the data set. We believe that this metric is superior to standard definitions of enrichment, which punish active ligands when they are outranked by other active ligands; this is a particularly serious problem when the active test suite contains a large number of compounds. A “perfect” score based on this metric would thus be zero (no database ligands outranking any active compounds), and smaller numbers are better. These values are also presented for the older 2.7 XP and 4.0 SP Glide results. As noted previously, only active ligands that successfully docked in 4.0 XP Glide were considered. In a small number of cases, active ligands failed to dock with 4.0 SP or 2.7 XP. For a calculation of the number of outranking decoy ligands, such ligands were ranked lower than all successfully docked active and decoy ligands. <div class="NLM_table-wrap" id="jm051256ot00009"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><p class="first last"> Table 9.  Average Enrichments Defined as the Average Number of Outranking Decoy Ligands over Correctly Docked Actives in the Training Set<i><sup>a</sup></i><sup></sup></p></div></div><div class="scrollable-table-wrap"><table class="table " width="100%"><colgroup><col /></col><col /></col><col /></col><col /></col></colgroup><tbody><tr valign="top" class="colsep2"><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2" colspan="3">avg number of outranking decoys</td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">screen</td><td class="colsep2 rowsep2">v4.0 XP</td><td class="colsep2 rowsep2">v2.7 XP</td><td class="colsep2 rowsep2">v4.0 SP </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">acetylcholinesterase </td><td class="colsep2 rowsep2">111 </td><td class="colsep2 rowsep2">580 </td><td class="colsep2 rowsep2">344 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">neuramididase </td><td class="colsep2 rowsep2">25 </td><td class="colsep2 rowsep2">411 </td><td class="colsep2 rowsep2">37 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">factor Xa </td><td class="colsep2 rowsep2">1 </td><td class="colsep2 rowsep2">196 </td><td class="colsep2 rowsep2">187 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">human p38 map kinase </td><td class="colsep2 rowsep2">26 </td><td class="colsep2 rowsep2">30 </td><td class="colsep2 rowsep2">57 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">p38 map kinase </td><td class="colsep2 rowsep2">7 </td><td class="colsep2 rowsep2">93 </td><td class="colsep2 rowsep2">183 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">HIV-RT </td><td class="colsep2 rowsep2">11 </td><td class="colsep2 rowsep2">26 </td><td class="colsep2 rowsep2">83 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">cyclooxygenase-2 </td><td class="colsep2 rowsep2">24 </td><td class="colsep2 rowsep2">12 </td><td class="colsep2 rowsep2">22 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">human cyclin dep. kinase </td><td class="colsep2 rowsep2">3 </td><td class="colsep2 rowsep2">77 </td><td class="colsep2 rowsep2">216 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">thrombin </td><td class="colsep2 rowsep2">2 </td><td class="colsep2 rowsep2">57 </td><td class="colsep2 rowsep2">70 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">HIV-1 protease </td><td class="colsep2 rowsep2">16 </td><td class="colsep2 rowsep2">60 </td><td class="colsep2 rowsep2">167 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">human estrogen receptor </td><td class="colsep2 rowsep2">14 </td><td class="colsep2 rowsep2">2 </td><td class="colsep2 rowsep2">23 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">lck kinase </td><td class="colsep2 rowsep2">13 </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2">157 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">EGRF tyrosine kinase </td><td class="colsep2 rowsep2">41 </td><td class="colsep2 rowsep2">411 </td><td class="colsep2 rowsep2">279 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">thermolysin </td><td class="colsep2 rowsep2">9 </td><td class="colsep2 rowsep2">107 </td><td class="colsep2 rowsep2">32 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">thymidine kinase </td><td class="colsep2 rowsep2">0 </td><td class="colsep2 rowsep2">1 </td><td class="colsep2 rowsep2">29</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-"><p class="first last"><i><sup>a</sup></i><sup></sup> Active ligands have at least 10 μM activity except those for neuraminidase, as described in Section 4.</p></div></div></div><div class="NLM_p">Table 10<a class="ref showTableEvent internalNav" data-ID="jm051256ot00010" href="#jm051256ot00010"></a> is the corresponding table constructed using a more standard definition of enrichment that we have employed <a name="jm004902106194" id="jm004902106194"></a> previously.<sup>2</sup> The results in Table 10 include the same sets of active ligands as in Table 9, that is, those whose activities are better than 10 μM and have been judged to fit more or less correctly into the specified conformation of the receptor. Table 11<a class="ref showTableEvent internalNav" data-ID="jm051256ot00011" href="#jm051256ot00011"></a> presents results using all active compounds with binding affinities better than 10 μM, whether the binding mode is judged to be correct. Results are presented for recovering 40, 70, and 100% of the considered active ligands in each case. This type of analysis corresponds to the approach taken by us in ref 2, as well as to other work in the literature. While we believe that the analysis in Table 9 is the appropriate one to use in assessing the quality of a scoring function to be used in rigid-receptor docking, the results presented in Table 11 enable a direct connection to be made with alternative viewpoints. In what follows, our discussion is focused on the results in Table 9 for the reasons given above. <div class="NLM_table-wrap" id="jm051256ot00010"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><p class="first last"> Table 10.  Standard Enrichment Factors for Recovering 40% of the Correctly Docked Active Ligands in the Training Set<i><sup>a</sup></i><sup></sup></p></div></div><div class="scrollable-table-wrap"><table class="table " width="100%"><colgroup><col /></col><col /></col><col /></col><col /></col></colgroup><tbody><tr valign="top" class="colsep2"><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2" colspan="3">enrichment factors</td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">screen</td><td class="colsep2 rowsep2">v4.0 XP</td><td class="colsep2 rowsep2">v2.7 XP</td><td class="colsep2 rowsep2">v4.0 SP </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">acetylcholinesterase </td><td class="colsep2 rowsep2">37 </td><td class="colsep2 rowsep2">1 </td><td class="colsep2 rowsep2">2 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">neuramididase </td><td class="colsep2 rowsep2">64 </td><td class="colsep2 rowsep2">2 </td><td class="colsep2 rowsep2">112 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">factor Xa </td><td class="colsep2 rowsep2">126 </td><td class="colsep2 rowsep2">42 </td><td class="colsep2 rowsep2">63 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">human p38 map kinase </td><td class="colsep2 rowsep2">81 </td><td class="colsep2 rowsep2">58 </td><td class="colsep2 rowsep2">51 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">p38 map kinase </td><td class="colsep2 rowsep2">35 </td><td class="colsep2 rowsep2">18 </td><td class="colsep2 rowsep2">11 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">HIV-RT </td><td class="colsep2 rowsep2">36 </td><td class="colsep2 rowsep2">14 </td><td class="colsep2 rowsep2">18 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">cyclooxygenase-2 </td><td class="colsep2 rowsep2">35 </td><td class="colsep2 rowsep2">56 </td><td class="colsep2 rowsep2">78 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">human cyclin dep. kinase </td><td class="colsep2 rowsep2">168 </td><td class="colsep2 rowsep2">168 </td><td class="colsep2 rowsep2">8 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">thrombin </td><td class="colsep2 rowsep2">68 </td><td class="colsep2 rowsep2">58 </td><td class="colsep2 rowsep2">12 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">HIV-1 protease </td><td class="colsep2 rowsep2">90 </td><td class="colsep2 rowsep2">32 </td><td class="colsep2 rowsep2">112 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">human estrogen receptor </td><td class="colsep2 rowsep2">126 </td><td class="colsep2 rowsep2">126 </td><td class="colsep2 rowsep2">126 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">lck kinase </td><td class="colsep2 rowsep2">12 </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2">6 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">EGRF tyrosine kinase </td><td class="colsep2 rowsep2">6 </td><td class="colsep2 rowsep2">1 </td><td class="colsep2 rowsep2">2 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">thermolysin </td><td class="colsep2 rowsep2">50 </td><td class="colsep2 rowsep2">134 </td><td class="colsep2 rowsep2">201 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">thymidine kinase </td><td class="colsep2 rowsep2">251 </td><td class="colsep2 rowsep2">251 </td><td class="colsep2 rowsep2">17</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-"><p class="first last"><i><sup>a</sup></i><sup></sup> Active ligands have at least 10 μM activity, except those for neuraminidase, as described in section 4.</p></div></div><div class="NLM_table-wrap" id="jm051256ot00011"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><p class="first last"> Table 11.  Standard Enrichment Factors for Recovering 40, 70, and 100% of Known Active Ligands in the Training Set, <i>I</i><i>ncluding</i> Misdocked Cases</p></div></div><div class="scrollable-table-wrap"><table class="table " width="100%"><colgroup><col /></col><col /></col><col /></col><col /></col><col /></col><col /></col><col /></col><col /></col><col /></col><col /></col></colgroup><tbody><tr valign="top" class="colsep2"><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2" colspan="9">enrichment factors</td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2" colspan="3">v4.0 XP</td><td class="colsep2 rowsep2" colspan="3">v2.7 XP</td><td class="colsep2 rowsep2" colspan="3">v4.0 SP</td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">screen</td><td class="colsep2 rowsep2">40%</td><td class="colsep2 rowsep2">70%</td><td class="colsep2 rowsep2">100%</td><td class="colsep2 rowsep2">40%</td><td class="colsep2 rowsep2">70%</td><td class="colsep2 rowsep2">100%</td><td class="colsep2 rowsep2">40%</td><td class="colsep2 rowsep2">70%</td><td class="colsep2 rowsep2">100% </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">acetylcholinesterase </td><td class="colsep2 rowsep2">37 </td><td class="colsep2 rowsep2">19 </td><td class="colsep2 rowsep2">1 </td><td class="colsep2 rowsep2">1 </td><td class="colsep2 rowsep2">1 </td><td class="colsep2 rowsep2">1 </td><td class="colsep2 rowsep2">2 </td><td class="colsep2 rowsep2">2 </td><td class="colsep2 rowsep2">1 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">neuramididase </td><td class="colsep2 rowsep2">64 </td><td class="colsep2 rowsep2">34 </td><td class="colsep2 rowsep2">6 </td><td class="colsep2 rowsep2">2 </td><td class="colsep2 rowsep2">1 </td><td class="colsep2 rowsep2">1 </td><td class="colsep2 rowsep2">112 </td><td class="colsep2 rowsep2">23 </td><td class="colsep2 rowsep2">4 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">factor Xa </td><td class="colsep2 rowsep2">78 </td><td class="colsep2 rowsep2">58 </td><td class="colsep2 rowsep2">3 </td><td class="colsep2 rowsep2">10 </td><td class="colsep2 rowsep2">1 </td><td class="colsep2 rowsep2">1 </td><td class="colsep2 rowsep2">22 </td><td class="colsep2 rowsep2">9 </td><td class="colsep2 rowsep2">1 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">human p38 map kinase </td><td class="colsep2 rowsep2">81 </td><td class="colsep2 rowsep2">59 </td><td class="colsep2 rowsep2">4 </td><td class="colsep2 rowsep2">58 </td><td class="colsep2 rowsep2">11 </td><td class="colsep2 rowsep2">9 </td><td class="colsep2 rowsep2">51 </td><td class="colsep2 rowsep2">21 </td><td class="colsep2 rowsep2">3 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">p38 map kinase </td><td class="colsep2 rowsep2">27 </td><td class="colsep2 rowsep2">14 </td><td class="colsep2 rowsep2">1 </td><td class="colsep2 rowsep2">13 </td><td class="colsep2 rowsep2">3 </td><td class="colsep2 rowsep2">1 </td><td class="colsep2 rowsep2">3 </td><td class="colsep2 rowsep2">2 </td><td class="colsep2 rowsep2">2 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">HIV-RT </td><td class="colsep2 rowsep2">27 </td><td class="colsep2 rowsep2">19 </td><td class="colsep2 rowsep2">3 </td><td class="colsep2 rowsep2">12 </td><td class="colsep2 rowsep2">12 </td><td class="colsep2 rowsep2">0 </td><td class="colsep2 rowsep2">11 </td><td class="colsep2 rowsep2">6 </td><td class="colsep2 rowsep2">0 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">cyclooxygenase-2 </td><td class="colsep2 rowsep2">35 </td><td class="colsep2 rowsep2">29 </td><td class="colsep2 rowsep2">7 </td><td class="colsep2 rowsep2">56 </td><td class="colsep2 rowsep2">29 </td><td class="colsep2 rowsep2">0 </td><td class="colsep2 rowsep2">78 </td><td class="colsep2 rowsep2">58 </td><td class="colsep2 rowsep2">9 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">human cyclin dep. kinase </td><td class="colsep2 rowsep2">81 </td><td class="colsep2 rowsep2">20 </td><td class="colsep2 rowsep2">2 </td><td class="colsep2 rowsep2">51 </td><td class="colsep2 rowsep2">4 </td><td class="colsep2 rowsep2">2 </td><td class="colsep2 rowsep2">5 </td><td class="colsep2 rowsep2">3 </td><td class="colsep2 rowsep2">2 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">thrombin </td><td class="colsep2 rowsep2">64 </td><td class="colsep2 rowsep2">64 </td><td class="colsep2 rowsep2">2 </td><td class="colsep2 rowsep2">54 </td><td class="colsep2 rowsep2">7 </td><td class="colsep2 rowsep2">0 </td><td class="colsep2 rowsep2">11 </td><td class="colsep2 rowsep2">10 </td><td class="colsep2 rowsep2">3 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">HIV-1 protease </td><td class="colsep2 rowsep2">72 </td><td class="colsep2 rowsep2">32 </td><td class="colsep2 rowsep2">14 </td><td class="colsep2 rowsep2">27 </td><td class="colsep2 rowsep2">16 </td><td class="colsep2 rowsep2">2 </td><td class="colsep2 rowsep2">72 </td><td class="colsep2 rowsep2">60 </td><td class="colsep2 rowsep2">1 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">human estrogen receptor </td><td class="colsep2 rowsep2">101 </td><td class="colsep2 rowsep2">101 </td><td class="colsep2 rowsep2">2 </td><td class="colsep2 rowsep2">101 </td><td class="colsep2 rowsep2">88 </td><td class="colsep2 rowsep2">2 </td><td class="colsep2 rowsep2">101 </td><td class="colsep2 rowsep2">17 </td><td class="colsep2 rowsep2">2 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">lck kinase </td><td class="colsep2 rowsep2">9 </td><td class="colsep2 rowsep2">8 </td><td class="colsep2 rowsep2">2 </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2">4 </td><td class="colsep2 rowsep2">3 </td><td class="colsep2 rowsep2">1 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">EGRF tyrosine kinase </td><td class="colsep2 rowsep2">5 </td><td class="colsep2 rowsep2">6 </td><td class="colsep2 rowsep2">1 </td><td class="colsep2 rowsep2">1 </td><td class="colsep2 rowsep2">1 </td><td class="colsep2 rowsep2">1 </td><td class="colsep2 rowsep2">2 </td><td class="colsep2 rowsep2">2 </td><td class="colsep2 rowsep2">1 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">thermolysin </td><td class="colsep2 rowsep2">42 </td><td class="colsep2 rowsep2">52 </td><td class="colsep2 rowsep2">23 </td><td class="colsep2 rowsep2">112 </td><td class="colsep2 rowsep2">37 </td><td class="colsep2 rowsep2">2 </td><td class="colsep2 rowsep2">168 </td><td class="colsep2 rowsep2">34 </td><td class="colsep2 rowsep2">7 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">thymidine kinase </td><td class="colsep2 rowsep2">251 </td><td class="colsep2 rowsep2">251 </td><td class="colsep2 rowsep2">201 </td><td class="colsep2 rowsep2">251 </td><td class="colsep2 rowsep2">188 </td><td class="colsep2 rowsep2">201 </td><td class="colsep2 rowsep2">17 </td><td class="colsep2 rowsep2">24 </td><td class="colsep2 rowsep2">17</td></tr></tbody></table></div></div></div><div class="NLM_p">The XP 4.0 results are nearly uniformly comparable to or better than those of either SP 4.0 or XP 2.7 and, in many cases, are significantly better, as is manifested with particular clarity using the new definition of enrichment. There is a slight degradation for the estrogen receptor from XP 4.0 for cyclooxygenase-2 relative to both XP 2.7 and SP 4.0, but all of the results for these test cases are very good. The real question with regard to scoring-function effectiveness is the ability to prevent false positives from ranking ahead of active compounds. XP 4.0 displays an ability to reduce the average number of false positives ranking ahead of actives in many cases by an order of magnitude and in some cases by nearly 2 orders of magnitude, as compared to both 2.7 XP and 4.0 SP. This same effect is also reflected in the more common definition of enrichment factor (Table 10), but the improvement is quantitatively obscured by the definition of enrichment employed, particularly for the data sets containing larger numbers of actives. For example, in EFGR kinase the number of actives is greater than 10% of the random database, and standard enrichment measures that effectively penalize active compounds for having other active compounds ranking ahead of other actives can yield enrichment factors of at most 10.3 for Table 10 and 9.5 for Table 11. </div><div class="NLM_p">The results shown in Table 9 are not perfect. However, until intrinsic RMS fluctuations in the scoring function can be reduced from the present average of 1.7 kcal/mol for well-docked ligands, the scoring function seems unlikely to systematically perform significantly better without overfitting. The number of high-scoring database ligands reflected in this table is consistent with the estimated experimental population of low micromolar hits in a 1000 molecule random database of drug-like molecules. The acetylcholinesterase receptor appears to manifest the largest systematic errors. This is likely due to our inability to optimize the pi-cation and pi-stacking scoring function terms with high precision because we lack sufficiently diverse examples manifesting these terms. There also remain some difficulties associated with smaller, highly hydrophobic sites, such as Cox-2 and in medium-sized sites with a single special hydrogen bond, such as EGFR. Overall though, the results are reasonably robust across the entire data set and clearly represent a major advance over the results obtained using 4.0 SP or 2.7 XP. Direct comparisons with other codes would require using the same sets of actives and database ligands. Based on anecdotal reports from various sources and from comparison with published data,<sup>1</sup> Glide SP has generally performed at least as well in enrichment studies as, if not better than, alternatives such as GOLD and FlexX. One would therefore expect 4.0 XP to outperform these methods by a margin similar to that seen in Table 9 for 4.0 SP. </div><div class="NLM_p"><b>Results of Test Set Enrichment Studies.</b> A summary of key data for our test set, including the receptor crystal structures used, and the number of known and well-docked actives again restricted to ligands with experimental binding affinities better than 10 μM are presented in Table 12<a class="ref showTableEvent internalNav" data-ID="jm051256ot00012" href="#jm051256ot00012"></a>. The test set includes two kinases (CDK2, Vegfr2), three proteases (thrombin, BACE, factor VIIa), and one nuclear hormone receptor (PPARγ) and, hence, is reasonably diverse with regard to function; all of the receptors in the test set are drug targets of current or recent interest. There is less diversity with regard to active site size and hydrophobicity than in the training set. As discussed above, validation with a larger test set will be addressed in future publications. All test set calculations were performed with the released versions of Glide 4.0 XP and SP, with no parameter adjustment being made to improve results for any targets. <div class="NLM_table-wrap" id="jm051256ot00012"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><p class="first last"> Table 12.  Test Set Used to Validate XP Virtual Screening<i><sup>a</sup></i><sup></sup></p></div></div><div class="scrollable-table-wrap"><table class="table " width="100%"><colgroup><col /></col><col /></col><col /></col><col /></col></colgroup><tbody><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">PDB code</td><td class="colsep2 rowsep2">description</td><td class="colsep2 rowsep2">no. actives</td><td class="colsep2 rowsep2">No. well-docked actives </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1m4h </td><td class="colsep2 rowsep2">BACE </td><td class="colsep2 rowsep2">77 </td><td class="colsep2 rowsep2">34 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1dan </td><td class="colsep2 rowsep2">factor VIIa </td><td class="colsep2 rowsep2">93 </td><td class="colsep2 rowsep2">40 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1fm6 </td><td class="colsep2 rowsep2">PPARγ (closed form) </td><td class="colsep2 rowsep2">93<i><sup>b</sup></i><sup></sup></td><td class="colsep2 rowsep2">32<i><sup>c</sup></i><sup></sup></td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1fm9 </td><td class="colsep2 rowsep2">PPARγ (open form) </td><td class="colsep2 rowsep2">93<i><sup>b</sup></i><sup></sup></td><td class="colsep2 rowsep2">25<i><sup>c</sup></i><sup></sup></td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1y6b </td><td class="colsep2 rowsep2">Vegfr2 (closed form) </td><td class="colsep2 rowsep2">111<i><sup>b</sup></i><sup></sup></td><td class="colsep2 rowsep2">21<i><sup>c</sup></i><sup></sup></td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1ywn </td><td class="colsep2 rowsep2">Vegfr2 (open form) </td><td class="colsep2 rowsep2">111<i><sup>b</sup></i><sup></sup></td><td class="colsep2 rowsep2">26<i><sup>c</sup></i><sup></sup></td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1aq1 </td><td class="colsep2 rowsep2">human cyclin dep. kinase </td><td class="colsep2 rowsep2">253 </td><td class="colsep2 rowsep2">143 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1ett </td><td class="colsep2 rowsep2">thrombin </td><td class="colsep2 rowsep2">40 </td><td class="colsep2 rowsep2">15</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-"><p class="first last"><i><sup>a</sup></i><sup></sup> All correctly docked ligands have experimental activities <10 μM.<i><sup>b</sup></i><sup></sup> When multiple forms of a receptor are utilized, the number of actives reported here is all known actives with affinities <10 μM.<i><sup>c</sup></i><sup></sup> Ligands are assessed as optimally fitting into either the open or closed form of the receptor. For example, in PPARγ, 61% of known active ligands (57 of the 93) were assessed as fitting into either the open or the closed form of the receptor.</p></div></div></div><div class="NLM_p">For two of the receptors (Vegfr2 and PPARγ) we utilize two different forms of the receptor structures. These are highly flexible active sites, and a significant fraction of ligands in both cases can be divided into groups that clearly fit better into one version of the receptor or the other. For PPARγ, for example, one class of ligands requires opening of an allosteric pocket (primarily via motion of a phenylalanine residue), while the second class is smaller and does not protrude into this pocket. Comparing scores of these two ligand classes using a single receptor structure does not make sense. If the pocket is closed, the larger ligands will not fit at all, whereas if the pocket is open, the larger ligands will unfairly score better, as the reorganization energy of the receptor required to engender the needed side chain motion will not have been included. There is no overlap between the ligands associated with the two receptor forms. This partitioning is meant as an introductory exploration of enrichment studies using multiple receptor conformations, a topic we intend to pursue more intensively in the future. </div><div class="NLM_p">Enrichment metrics to recover well-docked active ligands based on number of outranking decoys and standard enrichment measures (as for the training set) are presented in Tables 13 and 14<a class="ref showTableEvent internalNav" data-ID="jm051256ot00013" href="#jm051256ot00013"></a><a class="ref showTableEvent internalNav" data-ID="jm051256ot00014" href="#jm051256ot00014"></a>, respectively. For all known active ligands, standard enrichment measures are presented in Table 15<a class="ref showTableEvent internalNav" data-ID="jm051256ot00015" href="#jm051256ot00015"></a>. The same comparison database of 1000 decoy ligands employed in training set enrichment studies has been used. As expected, there is some quantitative degradation of the XP results from the training set, but overall the results are qualitatively comparable to the training set results using the outranking decoy metric (which we have argued is the most meaningful for our purposes), and the improvements as compared to SP Glide are, on average, significant. For PPARγ, both methods do reasonably well; this <a name="jm004902106195" id="jm004902106195"></a> is a case where SP scoring performs unexpectedly well, as opposed to suggesting a particular problem with the XP scoring function. <div class="NLM_table-wrap" id="jm051256ot00013"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><p class="first last"> Table 13.  Average Enrichments Defined as the Average Number of Outranking Decoy Ligands over Correctly Docked Actives in the Test Set<i><sup>a</sup></i><sup></sup></p></div></div><div class="scrollable-table-wrap"><table class="table " width="100%"><colgroup><col /></col><col /></col><col /></col></colgroup><tbody><tr valign="top" class="colsep2"><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2" colspan="2">avg no. of outranking decoys</td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">screen</td><td class="colsep2 rowsep2">v4.0 XP</td><td class="colsep2 rowsep2">v4.0 SP </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">BACE </td><td class="colsep2 rowsep2">35 </td><td class="colsep2 rowsep2">342 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">factor VIIa </td><td class="colsep2 rowsep2">30 </td><td class="colsep2 rowsep2">75 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">PPARγ (closed form) </td><td class="colsep2 rowsep2">45 </td><td class="colsep2 rowsep2">48 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">PPARγ (open form) </td><td class="colsep2 rowsep2">44 </td><td class="colsep2 rowsep2">76 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">Vegfr2 (closed form) </td><td class="colsep2 rowsep2">52 </td><td class="colsep2 rowsep2">222 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">Vegfr2 (open form) </td><td class="colsep2 rowsep2">69 </td><td class="colsep2 rowsep2">310 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">human cyclin dep. kinase </td><td class="colsep2 rowsep2">25 </td><td class="colsep2 rowsep2">206 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">thrombin </td><td class="colsep2 rowsep2">9 </td><td class="colsep2 rowsep2">52</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-"><p class="first last"><i><sup>a</sup></i><sup></sup> Active ligands have at least 10 μM activity.  </p></div></div><div class="NLM_table-wrap" id="jm051256ot00014"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><p class="first last"> Table 14.  Standard Enrichment Factors for Recovering 40% of the Correctly Docked Active Ligands in the Test Set<i><sup>a</sup></i><sup></sup></p></div></div><div class="scrollable-table-wrap"><table class="table " width="100%"><colgroup><col /></col><col /></col><col /></col></colgroup><tbody><tr valign="top" class="colsep2"><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2" colspan="2">enrichment factors</td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">screen</td><td class="colsep2 rowsep2">v4.0 XP</td><td class="colsep2 rowsep2">v4.0 SP </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">BACE </td><td class="colsep2 rowsep2">30 </td><td class="colsep2 rowsep2">25 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">factor VIIa </td><td class="colsep2 rowsep2">19 </td><td class="colsep2 rowsep2">15 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">PPARγ (closed form) </td><td class="colsep2 rowsep2">9 </td><td class="colsep2 rowsep2">19 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">PPARγ (open form) </td><td class="colsep2 rowsep2">14 </td><td class="colsep2 rowsep2">40 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">Vegfr2 (closed form) </td><td class="colsep2 rowsep2">25 </td><td class="colsep2 rowsep2">10 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">Vegfr2 (open form) </td><td class="colsep2 rowsep2">11 </td><td class="colsep2 rowsep2">2 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">human cyclin dep. kinase </td><td class="colsep2 rowsep2">8 </td><td class="colsep2 rowsep2">5 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">thrombin </td><td class="colsep2 rowsep2">64 </td><td class="colsep2 rowsep2">25</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-"><p class="first last"><i><sup>a</sup></i><sup></sup> Active ligands have at least 10 μM activity.</p></div></div><div class="NLM_table-wrap" id="jm051256ot00015"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><p class="first last"> Table 15.  Standard Enrichment Factors for Recovering 40 and 70% of Known Active Ligands in the Test Set, <i>I</i><i>ncluding</i> Misdocked Cases</p></div></div><div class="scrollable-table-wrap"><table class="table " width="100%"><colgroup><col /></col><col /></col><col /></col><col /></col><col /></col></colgroup><tbody><tr valign="top" class="colsep2"><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2" colspan="4">enrichment factors</td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2" colspan="2">v4.0 XP</td><td class="colsep2 rowsep2" colspan="2">v4.0 SP</td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">screen</td><td class="colsep2 rowsep2">40%</td><td class="colsep2 rowsep2">70%</td><td class="colsep2 rowsep2">40%</td><td class="colsep2 rowsep2">70% </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">BACE </td><td class="colsep2 rowsep2">12 </td><td class="colsep2 rowsep2">3 </td><td class="colsep2 rowsep2">11 </td><td class="colsep2 rowsep2">0 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">factor VIIa </td><td class="colsep2 rowsep2">10 </td><td class="colsep2 rowsep2">6 </td><td class="colsep2 rowsep2">7 </td><td class="colsep2 rowsep2">5 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">PPARγ (closed form) </td><td class="colsep2 rowsep2">5 </td><td class="colsep2 rowsep2">2 </td><td class="colsep2 rowsep2">7 </td><td class="colsep2 rowsep2">4 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">PPARγ (open form) </td><td class="colsep2 rowsep2">3 </td><td class="colsep2 rowsep2">3 </td><td class="colsep2 rowsep2">11 </td><td class="colsep2 rowsep2">7 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">Vegfr2 (closed form) </td><td class="colsep2 rowsep2">2 </td><td class="colsep2 rowsep2">1 </td><td class="colsep2 rowsep2">1 </td><td class="colsep2 rowsep2">1 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">Vegfr2 (open form) </td><td class="colsep2 rowsep2">3 </td><td class="colsep2 rowsep2">2 </td><td class="colsep2 rowsep2">1 </td><td class="colsep2 rowsep2">1 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">human cyclin dep. kinase </td><td class="colsep2 rowsep2">4 </td><td class="colsep2 rowsep2">3 </td><td class="colsep2 rowsep2">2 </td><td class="colsep2 rowsep2">2 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">thrombin </td><td class="colsep2 rowsep2">19 </td><td class="colsep2 rowsep2">3 </td><td class="colsep2 rowsep2">11 </td><td class="colsep2 rowsep2">6</td></tr></tbody></table></div></div></div><div class="NLM_p last">A number of caveats should be emphasized with regard to these results. The test set is small, and there are almost certainly cases where enrichment performance will not be as good as that indicated in Tables 13 and 14. Furthermore, high enrichment with few false positives can only be expected when the active ligands are properly docked. The fact that a significant fraction of actives are not well-docked, even when two receptor conformations are used, indicates that if a diverse set of active compounds is to be robustly separated from a random database without false positives or false negatives, significant work needs to be done to better treat receptor flexibility and ensure reliable docking accuracy. However, we believe that the attempt in this paper to separate scoring function accuracy, docking accuracy, and reorganization energy effects is an essential starting point if truly robust approaches are to be developed. </div></div><div id="" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i30">5. Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_37587" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_37587" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">We have described a novel scoring function and an enhanced sampling algorithm, the combination of which constitutes the Glide 4.0 XP docking methodology. The methodology has been tested with a diverse set of ligands and receptors, and has produced large improvements in binding affinity prediction and database enrichment as compared to other scoring functions within Glide. </div><div class="NLM_p last">The potential for providing physical insight into the origins of enhanced binding affinity is, in our view, as important as quantitative improvement of enrichment factors. Visualization of XP Glide terms, as is presented in the Figures of the present paper, can be utilized by modelers and medicinal chemists in the design of new inhibitors. The success of design efforts along these lines in the context of lead optimization would provide <a name="jm004902106196" id="jm004902106196"></a> the most convincing evidence that the underlying model of molecular recognition proposed herein has substantial validity. Our hope is that the present paper will facilitate work along these lines by describing in considerable detail the theory that underlies the XP Glide implementation. </div></div><div class="NLM_back"><div class="article_supporting-info" id=""><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/jm051256o/suppl_file/jm051256osi20060602_023733.pdf">jm051256osi20060602_023733.pdf (112.11 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/jm051256o" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_12582" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_12582" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><ul></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Richard A. Friesner</span> - <span class="hlFld-Affiliation affiliation">Department of Chemistry, Columbia University, New York, New York 10027, Schrödinger, Limited Liability Company, 120 West 45th Street, New York, New York 10036, Schrödinger, Limited Liability Company, 101 SW Main Street, Portland, Oregon 97204 </span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Robert B. Murphy</span> - <span class="hlFld-Affiliation affiliation">Department of Chemistry, Columbia University, New York, New York 10027, Schrödinger, Limited Liability Company, 120 West 45th Street, New York, New York 10036, Schrödinger, Limited Liability Company, 101 SW Main Street, Portland, Oregon 97204 </span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Matthew P. Repasky</span> - <span class="hlFld-Affiliation affiliation">Department of Chemistry, Columbia University, New York, New York 10027, Schrödinger, Limited Liability Company, 120 West 45th Street, New York, New York 10036, Schrödinger, Limited Liability Company, 101 SW Main Street, Portland, Oregon 97204 </span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Leah L. Frye</span> - <span class="hlFld-Affiliation affiliation">Department of Chemistry, Columbia University, New York, New York 10027, Schrödinger, Limited Liability Company, 120 West 45th Street, New York, New York 10036, Schrödinger, Limited Liability Company, 101 SW Main Street, Portland, Oregon 97204 </span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jeremy R. Greenwood</span> - <span class="hlFld-Affiliation affiliation">Department of Chemistry, Columbia University, New York, New York 10027, Schrödinger, Limited Liability Company, 120 West 45th Street, New York, New York 10036, Schrödinger, Limited Liability Company, 101 SW Main Street, Portland, Oregon 97204 </span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Thomas A. Halgren</span> - <span class="hlFld-Affiliation affiliation">Department of Chemistry, Columbia University, New York, New York 10027, Schrödinger, Limited Liability Company, 120 West 45th Street, New York, New York 10036, Schrödinger, Limited Liability Company, 101 SW Main Street, Portland, Oregon 97204 </span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Paul C. Sanschagrin</span> - <span class="hlFld-Affiliation affiliation">Department of Chemistry, Columbia University, New York, New York 10027, Schrödinger, Limited Liability Company, 120 West 45th Street, New York, New York 10036, Schrödinger, Limited Liability Company, 101 SW Main Street, Portland, Oregon 97204 </span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Daniel T. Mainz</span> - <span class="hlFld-Affiliation affiliation">Department of Chemistry, Columbia University, New York, New York 10027, Schrödinger, Limited Liability Company, 120 West 45th Street, New York, New York 10036, Schrödinger, Limited Liability Company, 101 SW Main Street, Portland, Oregon 97204 </span></div></li></ul></li><li></li><li></li><li></li></ul></div><div class="ack"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i31">Acknowledgment</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_78077" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_78077" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">We thank Mark Murcko, Bob Pearlstein, and Barry Honig for reading preliminary versions of this manuscript and providing useful feedback. We also thank Mike Campbell for assistance in generating graphics. </p></div><div class="refs-header-label"></div><p class="references-count">
            This article references 42 other publications.
        </p><ol id="references" class="useLabel"><li id="jm051256ob00001"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm051256ob00001'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="jm051256ob00001_1"><span><span class="NLM_string-ref">Friesner, R. A.; Banks, J. L.; Murphy, R. B.; Halgren, T. A.; Klicic, J. J.; Mainz, D. T.; Repasky, M. P.; Knoll, E. H.; Shelley, M.; Perry, J. K.; Shaw, D. E.; Francis, P.; Shenkin, P. S. Glide:  A New Approach for Rapid, Accurate Docking and Scoring. 1. Method and Assessment of Docking Accuracy. <i>  J. Med. Chem.</i><b>2004</b>, <i>47</i>, 1739−1749.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=1749&author=R.+A.+Friesner&author=J.+L.+Banks&author=R.+B.+Murphy&author=T.+A.+Halgren&author=J.+J.+Klicic&author=D.+T.+Mainz&author=M.+P.+Repasky&author=E.+H.+Knoll&author=M.+Shelley&author=J.+K.+Perry&author=D.+E.+Shaw&author=P.+Francis&author=P.+S.+Shenkin&author=A+Glide&author=+New+Approach&author=+Accurate+Docking&title=Method+and+Assessment+of+Docking+Accuracy"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm051256ob00001_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26atitle%3DMethod%2520and%2520Assessment%2520of%2520Docking%2520Accuracy%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D1749" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm051256ob00002"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm051256ob00002'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="jm051256ob00002_1"><span><span class="NLM_string-ref">Halgren, T. A.; Murphy, R. B.; Friesner, R. A.; Beard, H. S.; Frye, L. L.; Pollard, W. T.; Banks, J. L. Glide:  A New Approach for Rapid, Accurate Docking and Scoring. 2. Enrichment Factors in Database Screening. <i>  J. </i><i>Med. Chem.</i><b>2004</b>, <i>47</i>, 1750−1759.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=1759&author=T.+A.+Halgren&author=R.+B.+Murphy&author=R.+A.+Friesner&author=H.+S.+Beard&author=L.+L.+Frye&author=W.+T.+Pollard&author=J.+L.+Banks&author=A+Glide&author=+New+Approach&author=+Accurate+Docking&title=Enrichment+Factors+in+Database+Screening"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm051256ob00002_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DHalgren%26aufirst%3DT.%2BA.%26atitle%3DEnrichment%2520Factors%2520in%2520Database%2520Screening%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D1759" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm051256ob00003"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm051256ob00003'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="jm051256ob00003_1"><span><span class="NLM_string-ref">Perola, E.; Walters, W. P.; Charifson, P. S. A Detailed Comparison of Current Docking and Scoring Methods on Systems of Pharmaceutical Relevance. <i>  Proteins</i><b>2004</b>, <i>56</i>, 235−249.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2004&pages=249&author=E.+Perola&author=W.+P.+Walters&author=P.+S+Charifson&title=A+Detailed+Comparison+of+Current+Docking+and+Scoring+Methods+on+Systems+of+Pharmaceutical+Relevance"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm051256ob00003_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DPerola%26aufirst%3DE.%26atitle%3DA%2520Detailed%2520Comparison%2520of%2520Current%2520Docking%2520and%2520Scoring%2520Methods%2520on%2520Systems%2520of%2520Pharmaceutical%2520Relevance%26jtitle%3DProteins%26date%3D2004%26volume%3D56%26spage%3D249" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm051256ob00004"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm051256ob00004'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="jm051256ob00004_1"><span><span class="NLM_string-ref">Krovat, E. M.; Steindl, T.; Langer, T. Recent Advances in Docking and Scoring. <i>  Curr. Comput.-Aided Drug Des. </i><b>2005</b>, <i>1</i>, 93−102.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2005&pages=102&author=E.+M.+Krovat&author=T.+Steindl&author=T+Langer&title=Recent+Advances+in+Docking+and+Scoring"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm051256ob00004_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DKrovat%26aufirst%3DE.%2BM.%26atitle%3DRecent%2520Advances%2520in%2520Docking%2520and%2520Scoring%26jtitle%3DCurr.%2520Comput.-Aided%2520Drug%2520Des.%26date%3D2005%26volume%3D1%26spage%3D102" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm051256ob00005"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm051256ob00005'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="jm051256ob00005_1"><span><span class="NLM_string-ref">Kontoyianni, M.; McClellan, L. M.; Sokol, G. S. Evaluation of Docking Performance:  Comparative Data on Docking Algorithms. <i>  J. Med. Chem</i>. <b>2004</b>, <i>47</i>, 558−565.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=565&author=M.+Kontoyianni&author=L.+M.+McClellan&author=G.+S+Sokol&title=Evaluation+of+Docking+Performance%3A%E2%80%89+Comparative+Data+on+Docking+Algorithms"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm051256ob00005_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DKontoyianni%26aufirst%3DM.%26atitle%3DEvaluation%2520of%2520Docking%2520Performance%253A%25E2%2580%2589%2520Comparative%2520Data%2520on%2520Docking%2520Algorithms%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D565" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm051256ob00006"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm051256ob00006'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="jm051256ob00006_1"><span><span class="NLM_string-ref">Sherman, W.; Day, T.; Jacobson, M. P.; Friesner, R. A.; Farid, R. Novel Procedure for Modeling Ligand/Receptor Induced Fit Effects. <i>  J. Med. </i><i>Chem.</i><b>2006</b>, <i>49</i>, 534−553.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2006&pages=553&author=W.+Sherman&author=T.+Day&author=M.+P.+Jacobson&author=R.+A.+Friesner&author=R+Farid&title=Novel+Procedure+for+Modeling+Ligand%2FReceptor+Induced+Fit+Effects"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm051256ob00006_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSherman%26aufirst%3DW.%26atitle%3DNovel%2520Procedure%2520for%2520Modeling%2520Ligand%252FReceptor%2520Induced%2520Fit%2520Effects%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2006%26volume%3D49%26spage%3D553" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm051256ob00007"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm051256ob00007'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="jm051256ob00007_1"><span><span class="NLM_string-ref">Farid, R.; Day, T.; Friesner, R. A.; Pearlstein, R. A. New Insights about HERG Blockade Obtained from Protein Modeling, Potential Energy Mapping, and Docking Studies. <i>  Bioorg. Med. Chem.</i><b>2006</b>, <i>14</i>, 3160−3173.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2006&pages=3173&author=R.+Farid&author=T.+Day&author=R.+A.+Friesner&author=R.+A+Pearlstein&title=New+Insights+about+HERG+Blockade+Obtained+from+Protein+Modeling%2C+Potential+Energy+Mapping%2C+and+Docking+Studies"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm051256ob00007_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DFarid%26aufirst%3DR.%26atitle%3DNew%2520Insights%2520about%2520HERG%2520Blockade%2520Obtained%2520from%2520Protein%2520Modeling%252C%2520Potential%2520Energy%2520Mapping%252C%2520and%2520Docking%2520Studies%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2006%26volume%3D14%26spage%3D3173" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm051256ob00008"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm051256ob00008'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="jm051256ob00008_1"><span><span class="NLM_string-ref">Eldridge, M. D.; Murray, C. W.; Auton, T. R.; Paolinine, G. V.; Mee, R. P. Empirical Scoring Functions:  I. The Development of a Fast Empirical Scoring Function to Estimate the Binding Affinity of Ligands in Receptor Complexes. <i>  J. Comput.-Aid. Mol. Des.</i><b>1997</b>, <i>11</i>, 425−445.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=1997&pages=445&author=M.+D.+Eldridge&author=C.+W.+Murray&author=T.+R.+Auton&author=G.+V.+Paolinine&author=R.+P+Mee&title=Empirical+Scoring+Functions%3A%E2%80%89+I.+The+Development+of+a+Fast+Empirical+Scoring+Function+to+Estimate+the+Binding+Affinity+of+Ligands+in+Receptor+Complexes"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm051256ob00008_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DEldridge%26aufirst%3DM.%2BD.%26atitle%3DEmpirical%2520Scoring%2520Functions%253A%25E2%2580%2589%2520I.%2520The%2520Development%2520of%2520a%2520Fast%2520Empirical%2520Scoring%2520Function%2520to%2520Estimate%2520the%2520Binding%2520Affinity%2520of%2520Ligands%2520in%2520Receptor%2520Complexes%26jtitle%3DJ.%2520Comput.-Aid.%2520Mol.%2520Des.%26date%3D1997%26volume%3D11%26spage%3D445" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm051256ob00009"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm051256ob00009'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="jm051256ob00009_1"><span><span class="NLM_string-ref">Gehlhaar, D. K.; Verkhivker, G. M.; Rejto, P. A.; Sherman, C. J.; Fogel, L. J.; Freer, S. T. Molecular Recognition of the Inhibitor AG-1343 by HIV-1 ProteaseConformationally Flexible Docking by Evolutionary Programming. <i>  Chem. Biol.</i><b>1995</b>, <i>2</i>, 317−324.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=1995&pages=324&author=D.+K.+Gehlhaar&author=G.+M.+Verkhivker&author=P.+A.+Rejto&author=C.+J.+Sherman&author=L.+J.+Fogel&author=S.+T+Freer&title=Molecular+Recognition+of+the+Inhibitor+AG-1343+by+HIV-1+ProteaseConformationally+Flexible+Docking+by+Evolutionary+Programming"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm051256ob00009_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DGehlhaar%26aufirst%3DD.%2BK.%26atitle%3DMolecular%2520Recognition%2520of%2520the%2520Inhibitor%2520AG-1343%2520by%2520HIV-1%2520ProteaseConformationally%2520Flexible%2520Docking%2520by%2520Evolutionary%2520Programming%26jtitle%3DChem.%2520Biol.%26date%3D1995%26volume%3D2%26spage%3D324" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm051256ob00010"><div class="reference"><div class="NLM_citation" id="jm051256ob00010_1"><span><span class="NLM_string-ref">Morris, G. M.; Goodsell, D. S.; Halliday, R. S.; Huey, R.; Hart, W. E.; Belew, R. K.; Olsen, A. J. Automated Docking Using a Lamarckian Genetic Algorithm and an Empirical Binding Free Energy Function. <i>  J. </i><i>Comput. Chem.</i><b>1998</b>, <i>19</i>, 1639−1662.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=1998&pages=1662&author=G.+M.+Morris&author=D.+S.+Goodsell&author=R.+S.+Halliday&author=R.+Huey&author=W.+E.+Hart&author=R.+K.+Belew&author=A.+J+Olsen&title=Automated+Docking+Using+a+Lamarckian+Genetic+Algorithm+and+an+Empirical+Binding+Free+Energy+Function"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm051256ob00010_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMorris%26aufirst%3DG.%2BM.%26atitle%3DAutomated%2520Docking%2520Using%2520a%2520Lamarckian%2520Genetic%2520Algorithm%2520and%2520an%2520Empirical%2520Binding%2520Free%2520Energy%2520Function%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D1998%26volume%3D19%26spage%3D1662" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm051256ob00011"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm051256ob00011'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="jm051256ob00011_1"><span><span class="NLM_string-ref">Muegge, I.; Martin, Y. C. A General and Fast Scoring Function for Protein−Ligand Interactions:  A Simplified Potential Approach. <i>  J. Med. </i><i>Chem.</i><b>1999</b>, <i>42</i>, 791−804.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=1999&pages=804&author=I.+Muegge&author=Y.+C+Martin&title=A+General+and+Fast+Scoring+Function+for+Protein%E2%88%92Ligand+Interactions%3A%E2%80%89+A+Simplified+Potential+Approach"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm051256ob00011_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMuegge%26aufirst%3DI.%26atitle%3DA%2520General%2520and%2520Fast%2520Scoring%2520Function%2520for%2520Protein%25E2%2588%2592Ligand%2520Interactions%253A%25E2%2580%2589%2520A%2520Simplified%2520Potential%2520Approach%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1999%26volume%3D42%26spage%3D804" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm051256ob00012"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm051256ob00012'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="jm051256ob00012_1"><span><span class="NLM_string-ref">Bohm, H. J. The Development of a Simple Empirical Scoring Function to Estimate the Binding Constant for a Protein Ligand Complex of Known 3-Dimensional Structure. <i>  J. Comput.-Aided Mol. Des.</i><b>1994</b>, <i>8</i>, 243−256.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=1994&pages=256&author=H.+J+Bohm&title=The+Development+of+a+Simple+Empirical+Scoring+Function+to+Estimate+the+Binding+Constant+for+a+Protein+Ligand+Complex+of+Known+3-Dimensional+Structure"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm051256ob00012_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBohm%26aufirst%3DH.%2BJ%26atitle%3DThe%2520Development%2520of%2520a%2520Simple%2520Empirical%2520Scoring%2520Function%2520to%2520Estimate%2520the%2520Binding%2520Constant%2520for%2520a%2520Protein%2520Ligand%2520Complex%2520of%2520Known%25203-Dimensional%2520Structure%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D1994%26volume%3D8%26spage%3D256" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm051256ob00013"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm051256ob00013'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="jm051256ob00013_1"><span><span class="NLM_string-ref">Bohm, H. J. Prediction of Binding Constants of Protein Ligands:  A Fast Method for the Prioritization of Hits Obtained from De Novo Design or 3D Database Search Programs. <i>  J. Comput.-Aided Mol. Des.</i><b>1998</b>, <i>12</i>, 309−323.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=1998&pages=323&author=H.+J+Bohm&title=Prediction+of+Binding+Constants+of+Protein+Ligands%3A%E2%80%89+A+Fast+Method+for+the+Prioritization+of+Hits+Obtained+from+De+Novo+Design+or+3D+Database+Search+Programs"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm051256ob00013_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBohm%26aufirst%3DH.%2BJ%26atitle%3DPrediction%2520of%2520Binding%2520Constants%2520of%2520Protein%2520Ligands%253A%25E2%2580%2589%2520A%2520Fast%2520Method%2520for%2520the%2520Prioritization%2520of%2520Hits%2520Obtained%2520from%2520De%2520Novo%2520Design%2520or%25203D%2520Database%2520Search%2520Programs%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D1998%26volume%3D12%26spage%3D323" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm051256ob00014"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm051256ob00014'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="jm051256ob00014_1"><span><span class="NLM_string-ref">Rarey, M.; Kramer, B.; Lengauer, T.; Klebe, G. A. A Fast Flexible Docking Method Using an Incremental Construction Algorithm. <i>  J. Mol. </i><i>Biol.</i><b>1996</b>, <i>261</i>, 470−489.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=261&publication_year=1996&pages=489&author=M.+Rarey&author=B.+Kramer&author=T.+Lengauer&author=G.+A+Klebe&title=A+Fast+Flexible+Docking+Method+Using+an+Incremental+Construction+Algorithm"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm051256ob00014_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DRarey%26aufirst%3DM.%26atitle%3DA%2520Fast%2520Flexible%2520Docking%2520Method%2520Using%2520an%2520Incremental%2520Construction%2520Algorithm%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D1996%26volume%3D261%26spage%3D489" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm051256ob00015"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm051256ob00015'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="jm051256ob00015_1"><span><span class="NLM_string-ref">Jones, G.; Willet, P.; Glen, R. C.; Leach, A. R.; Taylor, R. Development and Validation of a Genetic Algorithm for Flexible Docking. <i>  J. Mol. Biol. </i><b>1997</b>, <i>267</i>, 727−748.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=267&publication_year=1997&pages=748&author=G.+Jones&author=P.+Willet&author=R.+C.+Glen&author=A.+R.+Leach&author=R+Taylor&title=Development+and+Validation+of+a+Genetic+Algorithm+for+Flexible+Docking"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm051256ob00015_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DJones%26aufirst%3DG.%26atitle%3DDevelopment%2520and%2520Validation%2520of%2520a%2520Genetic%2520Algorithm%2520for%2520Flexible%2520Docking%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D1997%26volume%3D267%26spage%3D748" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm051256ob00016"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm051256ob00016'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="jm051256ob00016_1"><span><span class="NLM_string-ref">Ewing, T. J.; Makino, S.; Skillman, A. G.; Kuntz, I. D. DOCK 4.0:  Search Strategies for Automated Molecular Docking of Flexible Molecule Databases. <i>  J. Comput.-Aided Mol. Des.</i><b>2001</b>, <i>15</i>, 411−428.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+Ewing%2C+T.+J.%3B+Makino%2C+S.%3B+Skillman%2C+A.+G.%3B+Kuntz%2C+I.+D.+DOCK+4.0%3A%E2%80%89+Search+Strategies+for+Automated+Molecular+Docking+of+Flexible+Molecule+Databases.+J.+Comput.-Aided+Mol.+Des.+2001%2C+15%2C+411%E2%88%92428.+Ewing%2C+T.+J.%3B+Makino%2C+S.%3B+Skillman%2C+A.+G.%3B+Kuntz%2C+I.+D.+DOCK+4.0%3A%E2%80%89+Search+Strategies+for+Automated+Molecular+Docking+of+Flexible+Molecule+Databases.+J.+Comput.-Aided+Mol.+Des.2001%2C+15%2C+411%E2%88%92428."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm051256ob00016_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DEwing%26aufirst%3DT.%2BJ.%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2001%26volume%3D15%26spage%3D428" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm051256ob00017"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm051256ob00017'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="jm051256ob00017_1"><span><span class="NLM_string-ref">Golke, H.; Hendlich, M.; Kelbe, G. Knowledge-Based Scoring Function to Predict Protein−Ligand Interactions. <i>  J. Mol. Biol.</i><b>2000</b>, <i>295</i>, 337−356.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=295&publication_year=2000&pages=356&author=H.+Golke&author=M.+Hendlich&author=G+Kelbe&title=Knowledge-Based+Scoring+Function+to+Predict+Protein%E2%88%92Ligand+Interactions"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm051256ob00017_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DGolke%26aufirst%3DH.%26atitle%3DKnowledge-Based%2520Scoring%2520Function%2520to%2520Predict%2520Protein%25E2%2588%2592Ligand%2520Interactions%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2000%26volume%3D295%26spage%3D356" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm051256ob00018"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm051256ob00018'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="jm051256ob00018_1"><span><span class="NLM_string-ref">Wang, R.; Lai, L.; Wang, S. Further Development and Validation of Empirical Scoring Functions for Structure-Based Binding Affinity Prediction. <i>  J. Comput.-Aided Mol. Des.</i><b>2002</b>, <i>16</i>, 11−26.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2002&pages=26&author=R.+Wang&author=L.+Lai&author=S+Wang&title=Further+Development+and+Validation+of+Empirical+Scoring+Functions+for+Structure-Based+Binding+Affinity+Prediction"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm051256ob00018_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DR.%26atitle%3DFurther%2520Development%2520and%2520Validation%2520of%2520Empirical%2520Scoring%2520Functions%2520for%2520Structure-Based%2520Binding%2520Affinity%2520Prediction%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2002%26volume%3D16%26spage%3D26" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm051256ob00019"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm051256ob00019'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="jm051256ob00019_1"><span><span class="NLM_string-ref">Wang, R.; Lu, Y.; Wang, S. Comparative Evaluation of 11 Scoring Functions for Molecular Docking. <i>  J. Med. Chem.</i><b>2003</b>, <i>46</i>, 2287−2303.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2003&pages=2303&author=R.+Wang&author=Y.+Lu&author=S+Wang&title=Comparative+Evaluation+of+11+Scoring+Functions+for+Molecular+Docking"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm051256ob00019_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DR.%26atitle%3DComparative%2520Evaluation%2520of%252011%2520Scoring%2520Functions%2520for%2520Molecular%2520Docking%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2003%26volume%3D46%26spage%3D2303" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm051256ob00020"><div class="reference"><div class="NLM_citation" id="jm051256ob00020_1"><span><span class="NLM_string-ref">Wallqvist, A.; Berne, B. J. Computer-Simulation of Hydrophobic Hydration Forces on Stacked Planes at Short-Range. <i>  J. Phys. Chem.</i><b>1995</b>, <i>99</i>, 2893−2899.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=99&publication_year=1995&pages=2899&author=A.+Wallqvist&author=B.+J+Berne&title=Computer-Simulation+of+Hydrophobic+Hydration+Forces+on+Stacked+Planes+at+Short-Range"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm051256ob00020_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DWallqvist%26aufirst%3DA.%26atitle%3DComputer-Simulation%2520of%2520Hydrophobic%2520Hydration%2520Forces%2520on%2520Stacked%2520Planes%2520at%2520Short-Range%26jtitle%3DJ.%2520Phys.%2520Chem.%26date%3D1995%26volume%3D99%26spage%3D2899" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm051256ob00021"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm051256ob00021'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="jm051256ob00021_1"><span><span class="NLM_string-ref">Lum, K.; Chandler, D.; Weeks, J. D. Hydrophobicity at Small and Large Length Scales. <i>  J. Phys. Chem. B</i><b>1999</b>, <i>103</i>, 4570−4577.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=103&publication_year=1999&pages=4577&author=K.+Lum&author=D.+Chandler&author=J.+D+Weeks&title=Hydrophobicity+at+Small+and+Large+Length+Scales"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm051256ob00021_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DLum%26aufirst%3DK.%26atitle%3DHydrophobicity%2520at%2520Small%2520and%2520Large%2520Length%2520Scales%26jtitle%3DJ.%2520Phys.%2520Chem.%2520B%26date%3D1999%26volume%3D103%26spage%3D4577" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm051256ob00022"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm051256ob00022'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="jm051256ob00022_1"><span><span class="NLM_string-ref">Nicholls, A.; Sharp, K. A.; Honig, B. Protein Folding and AssociationInsights from the Interfacial and Thermodynamic Properties of Hydrocarbons. <i>  Proteins </i><b>1991</b>, <i>11</i>, 281−296.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=1991&pages=296&author=A.+Nicholls&author=K.+A.+Sharp&author=B+Honig&title=Protein+Folding+and+AssociationInsights+from+the+Interfacial+and+Thermodynamic+Properties+of+Hydrocarbons"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm051256ob00022_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DNicholls%26aufirst%3DA.%26atitle%3DProtein%2520Folding%2520and%2520AssociationInsights%2520from%2520the%2520Interfacial%2520and%2520Thermodynamic%2520Properties%2520of%2520Hydrocarbons%26jtitle%3DProteins%26date%3D1991%26volume%3D11%26spage%3D296" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm051256ob00023"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm051256ob00023'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="jm051256ob00023_1"><span><span class="NLM_string-ref">Zhou, R. H.; Huang, X. H.; Margulis, C. J.; Berne, B. J. Hydrophobic collapse in multidomain protein folding. <i>  Science</i><b>2004</b>, <i>305</i>, 1605−1609. Huang, X. H.; Zhou, R. H.; Berne, B. J. Drying and hydrophobic collapse of paraffin plates. <i>J. Phys. Chem. B</i><b>2005</b>, <i>109</i>, 3546−3552.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=305&publication_year=2004&pages=1609&author=R.+H.+Zhou&author=X.+H.+Huang&author=C.+J.+Margulis&author=B.+J+Berne&title=Hydrophobic+collapse+in+multidomain+protein+folding"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm051256ob00023_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DZhou%26aufirst%3DR.%2BH.%26atitle%3DHydrophobic%2520collapse%2520in%2520multidomain%2520protein%2520folding%26jtitle%3DScience%26date%3D2004%26volume%3D305%26spage%3D1609" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm051256ob00024"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm051256ob00024'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="jm051256ob00024_1"><span><span class="NLM_string-ref">Cheng, Y. K.; Rossky, P. J. Surface Topography Dependence of Biomolecular Hydrophobic Hydration. <i>  Nature</i><b>1998</b>, <i>392</i>, 696−699.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=392&publication_year=1998&pages=699&author=Y.+K.+Cheng&author=P.+J+Rossky&title=Surface+Topography+Dependence+of+Biomolecular+Hydrophobic+Hydration"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm051256ob00024_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DCheng%26aufirst%3DY.%2BK.%26atitle%3DSurface%2520Topography%2520Dependence%2520of%2520Biomolecular%2520Hydrophobic%2520Hydration%26jtitle%3DNature%26date%3D1998%26volume%3D392%26spage%3D699" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm051256ob00025"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm051256ob00025'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="jm051256ob00025_1"><span><span class="NLM_string-ref">Hummer, G.; Rasaiah, J. C.; Noworyta, J. P. Water Conduction Through the Hydrophobic Channel of a Carbon Nanotube. <i>  Nature</i><b>2001</b>, <i>414</i>, 188−190.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=414&publication_year=2001&pages=190&author=G.+Hummer&author=J.+C.+Rasaiah&author=J.+P+Noworyta&title=Water+Conduction+Through+the+Hydrophobic+Channel+of+a+Carbon+Nanotube"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm051256ob00025_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DHummer%26aufirst%3DG.%26atitle%3DWater%2520Conduction%2520Through%2520the%2520Hydrophobic%2520Channel%2520of%2520a%2520Carbon%2520Nanotube%26jtitle%3DNature%26date%3D2001%26volume%3D414%26spage%3D190" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm051256ob00026"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm051256ob00026'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="jm051256ob00026_1"><span><span class="NLM_string-ref">Liu, P.; Huang, X. H.; Zhou, R. H.; Berne, B. J. Observation of a dewetting transition in the collapse of the melittin tetramer. <i>  Nature</i><b>2005</b>, <i>437</i>, 159−162.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=437&publication_year=2005&pages=162&author=P.+Liu&author=X.+H.+Huang&author=R.+H.+Zhou&author=B.+J+Berne&title=Observation+of+a+dewetting+transition+in+the+collapse+of+the+melittin+tetramer"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm051256ob00026_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DP.%26atitle%3DObservation%2520of%2520a%2520dewetting%2520transition%2520in%2520the%2520collapse%2520of%2520the%2520melittin%2520tetramer%26jtitle%3DNature%26date%3D2005%26volume%3D437%26spage%3D162" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm051256ob00027"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm051256ob00027'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="jm051256ob00027_1"><span><span class="NLM_string-ref">Wallqvist, A.; Berne, B. J. Molecular-Dynamics Study of the Dependence of Water Solvation Free-Energy on Solute Curvature and Surface-Area. <i>  J. </i><i>Phys. Chem.</i><b>1995</b>, <i>99</i>, 2885−2892.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=99&publication_year=1995&pages=2892&author=A.+Wallqvist&author=B.+J+Berne&title=Molecular-Dynamics+Study+of+the+Dependence+of+Water+Solvation+Free-Energy+on+Solute+Curvature+and+Surface-Area"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm051256ob00027_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DWallqvist%26aufirst%3DA.%26atitle%3DMolecular-Dynamics%2520Study%2520of%2520the%2520Dependence%2520of%2520Water%2520Solvation%2520Free-Energy%2520on%2520Solute%2520Curvature%2520and%2520Surface-Area%26jtitle%3DJ.%2520Phys.%2520Chem.%26date%3D1995%26volume%3D99%26spage%3D2892" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm051256ob00028"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm051256ob00028'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="jm051256ob00028_1"><span><span class="NLM_string-ref">Lee, S. H.; Rossky, P. J. A Comparison of the Structure and Dynamics of Liquid Water at Hydrophobic and Hydrophilic SurfacesA Molecular-Dynamics Simulation Study. <i>  J. Chem. Phys.</i><b>1994</b>, <i>100</i>, 3334−3345. Lee, C. Y.; McCammon, J. A.; Rossky, P. J. The Structure of Liquid Water at an Extended Hydrophobic Surface. <i>J. Chem. Phys.</i><b>1984</b>, <i>80</i>, 4448−4455.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=100&publication_year=1994&pages=3345&author=S.+H.+Lee&author=P.+J+Rossky&title=A+Comparison+of+the+Structure+and+Dynamics+of+Liquid+Water+at+Hydrophobic+and+Hydrophilic+SurfacesA+Molecular-Dynamics+Simulation+Study"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm051256ob00028_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DS.%2BH.%26atitle%3DA%2520Comparison%2520of%2520the%2520Structure%2520and%2520Dynamics%2520of%2520Liquid%2520Water%2520at%2520Hydrophobic%2520and%2520Hydrophilic%2520SurfacesA%2520Molecular-Dynamics%2520Simulation%2520Study%26jtitle%3DJ.%2520Chem.%2520Phys.%26date%3D1994%26volume%3D100%26spage%3D3345" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm051256ob00029"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm051256ob00029'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="jm051256ob00029_1"><span><span class="NLM_string-ref">Sharp, K. A.; Nicholls, A.; Fine, R. F.; Honig, B. Reconciling the Magnitude of the Microscopic and Macroscopic Hydrophobic Effects. <i>  Science</i><b>1991</b>, <i>252</i>, 106−109.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=252&publication_year=1991&pages=109&author=K.+A.+Sharp&author=A.+Nicholls&author=R.+F.+Fine&author=B+Honig&title=Reconciling+the+Magnitude+of+the+Microscopic+and+Macroscopic+Hydrophobic+Effects"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm051256ob00029_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSharp%26aufirst%3DK.%2BA.%26atitle%3DReconciling%2520the%2520Magnitude%2520of%2520the%2520Microscopic%2520and%2520Macroscopic%2520Hydrophobic%2520Effects%26jtitle%3DScience%26date%3D1991%26volume%3D252%26spage%3D109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm051256ob00030"><div class="reference"><div class="NLM_citation" id="jm051256ob00030_1"><span><span class="NLM_string-ref">Regan, J.; Breitfelder, S.; Cirillo, P.; Gilmore, T.; Graham, A. G.; Hickey, E.; Klaus, B.; Madwed, J.; Moriak, M.; Moss, N.; Pargellis, C.; Pay, S.; Proto, A.; Swinamer, A.; Tong, L.; Torcellini, C. Pyrazole Urea-Based Inhibitors of p38 MAP Kinase:  From Lead Compound to Clinical Candidate. <i>  J. Med. Chem.</i><b>2002</b>, <i>45</i>, 2994−3008.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2002&pages=3008&author=J.+Regan&author=S.+Breitfelder&author=P.+Cirillo&author=T.+Gilmore&author=A.+G.+Graham&author=E.+Hickey&author=B.+Klaus&author=J.+Madwed&author=M.+Moriak&author=N.+Moss&author=C.+Pargellis&author=S.+Pay&author=A.+Proto&author=A.+Swinamer&author=L.+Tong&author=C+Torcellini&title=Pyrazole+Urea-Based+Inhibitors+of+p38+MAP+Kinase%3A%E2%80%89+From+Lead+Compound+to+Clinical+Candidate"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm051256ob00030_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DRegan%26aufirst%3DJ.%26atitle%3DPyrazole%2520Urea-Based%2520Inhibitors%2520of%2520p38%2520MAP%2520Kinase%253A%25E2%2580%2589%2520From%2520Lead%2520Compound%2520to%2520Clinical%2520Candidate%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2002%26volume%3D45%26spage%3D3008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm051256ob00031"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm051256ob00031'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="jm051256ob00031_1"><span><span class="NLM_string-ref">Hendsch, Z. S.; Tidor, B. Do Salt Bridges Stabilize ProteinsA Continuum Electrostatic Analysis. <i>  Protein Sci.</i><b>1994</b>, <i>3</i>, 211−226.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=1994&pages=226&author=Z.+S.+Hendsch&author=B+Tidor&title=Do+Salt+Bridges+Stabilize+ProteinsA+Continuum+Electrostatic+Analysis"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm051256ob00031_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DHendsch%26aufirst%3DZ.%2BS.%26atitle%3DDo%2520Salt%2520Bridges%2520Stabilize%2520ProteinsA%2520Continuum%2520Electrostatic%2520Analysis%26jtitle%3DProtein%2520Sci.%26date%3D1994%26volume%3D3%26spage%3D226" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm051256ob00032"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm051256ob00032'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="jm051256ob00032_1"><span><span class="NLM_string-ref">Waldburger, C. D.; Schildbach, J. F.; Sauer, R. T. Are Buried Salt Bridges Important for Protein Stability and Conformational Specificity. <i>  Nat. Struct. </i><i>Biol.</i><b>1995</b>, <i>2</i>, 122−128.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=1995&pages=128&author=C.+D.+Waldburger&author=J.+F.+Schildbach&author=R.+T+Sauer&title=Are+Buried+Salt+Bridges+Important+for+Protein+Stability+and+Conformational+Specificity"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm051256ob00032_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DWaldburger%26aufirst%3DC.%2BD.%26atitle%3DAre%2520Buried%2520Salt%2520Bridges%2520Important%2520for%2520Protein%2520Stability%2520and%2520Conformational%2520Specificity%26jtitle%3DNat.%2520Struct.%2520Biol.%26date%3D1995%26volume%3D2%26spage%3D128" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm051256ob00033"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm051256ob00033'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="jm051256ob00033_1"><span><span class="NLM_string-ref">Marqusee, S.; Sauer, R. T. Contributions of a Hydrogen-Bond Salt Bridge Network to the Stability of Secondary and Tertiary Structure in Lambda-Repressor. <i>  Protein Sci.</i><b>1994</b>, <i>3</i>, 2217−2225.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=1994&pages=2225&author=S.+Marqusee&author=R.+T+Sauer&title=Contributions+of+a+Hydrogen-Bond+Salt+Bridge+Network+to+the+Stability+of+Secondary+and+Tertiary+Structure+in+Lambda-Repressor"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm051256ob00033_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMarqusee%26aufirst%3DS.%26atitle%3DContributions%2520of%2520a%2520Hydrogen-Bond%2520Salt%2520Bridge%2520Network%2520to%2520the%2520Stability%2520of%2520Secondary%2520and%2520Tertiary%2520Structure%2520in%2520Lambda-Repressor%26jtitle%3DProtein%2520Sci.%26date%3D1994%26volume%3D3%26spage%3D2225" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm051256ob00034"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm051256ob00034'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="jm051256ob00034_1"><span><span class="NLM_string-ref">Luo, R.; David, L.; Hung, H.; Devaney, J.; Gilson, M. K. Strength of Solvent-Exposed Salt-Bridges. Strength of Solvent-Exposed Salt-Bridges. <i>  J. Phys. Chem. B</i><b>1999</b>, <i>103</i>, 727−736.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=103&publication_year=1999&pages=736&author=R.+Luo&author=L.+David&author=H.+Hung&author=J.+Devaney&author=M.+K+Gilson&title=Strength+of+Solvent-Exposed+Salt-Bridges.+Strength+of+Solvent-Exposed+Salt-Bridges"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm051256ob00034_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DLuo%26aufirst%3DR.%26atitle%3DStrength%2520of%2520Solvent-Exposed%2520Salt-Bridges.%2520Strength%2520of%2520Solvent-Exposed%2520Salt-Bridges%26jtitle%3DJ.%2520Phys.%2520Chem.%2520B%26date%3D1999%26volume%3D103%26spage%3D736" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm051256ob00035"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm051256ob00035'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="jm051256ob00035_1"><span><span class="NLM_string-ref">Krammer, A.; Kirchhoff, P. D.; Jiang, X.; Venkatachalam, C. M.; Waldman, M. LigScore:  A Novel Scoring Function for Predicting Binding Affinities. <i>  J. Mol. Graphics Modell.</i><b>2005</b>, <i>23</i>, 395−407.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+Krammer%2C+A.%3B+Kirchhoff%2C+P.+D.%3B+Jiang%2C+X.%3B+Venkatachalam%2C+C.+M.%3B+Waldman%2C+M.+LigScore%3A%E2%80%89+A+Novel+Scoring+Function+for+Predicting+Binding+Affinities.+J.+Mol.+Graphics+Modell.+2005%2C+23%2C+395%E2%88%92407.+Krammer%2C+A.%3B+Kirchhoff%2C+P.+D.%3B+Jiang%2C+X.%3B+Venkatachalam%2C+C.+M.%3B+Waldman%2C+M.+LigScore%3A%E2%80%89+A+Novel+Scoring+Function+for+Predicting+Binding+Affinities.+J.+Mol.+Graphics+Modell.2005%2C+23%2C+395%E2%88%92407."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm051256ob00035_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DKrammer%26aufirst%3DA.%26jtitle%3DJ.%2520Mol.%2520Graphics%2520Modell.%26date%3D2005%26volume%3D23%26spage%3D407" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm051256ob00036"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm051256ob00036'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="jm051256ob00036_1"><span><span class="NLM_string-ref">Wei, B. Q. Q.; Baase, W. A.; Weaver, L. H.; Matthews, B. W.; Shoichet, B. K. A Model Binding Site for Testing Scoring Functions in Molecular Docking. <i>  J. Mol. Biol.</i><b>2002</b>, <i>322</i>, 339−355.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=322&publication_year=2002&pages=355&author=B.+Q.+Q.+Wei&author=W.+A.+Baase&author=L.+H.+Weaver&author=B.+W.+Matthews&author=B.+K+Shoichet&title=A+Model+Binding+Site+for+Testing+Scoring+Functions+in+Molecular+Docking"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm051256ob00036_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DWei%26aufirst%3DB.%2BQ.%2BQ.%26atitle%3DA%2520Model%2520Binding%2520Site%2520for%2520Testing%2520Scoring%2520Functions%2520in%2520Molecular%2520Docking%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2002%26volume%3D322%26spage%3D355" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm051256ob00037"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm051256ob00037'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="jm051256ob00037_1"><span><span class="NLM_string-ref">Perola, E.; Charifson, P. S. Conformational Analysis of Drug-Like Molecules Bound to Proteins:  An Extensive Study of Ligand Reorganization upon Binding. <i>  J. Med. Chem.</i><b>2004</b>, <i>47</i>, 2499−2510.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=2510&author=E.+Perola&author=P.+S+Charifson&title=Conformational+Analysis+of+Drug-Like+Molecules+Bound+to+Proteins%3A%E2%80%89+An+Extensive+Study+of+Ligand+Reorganization+upon+Binding"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm051256ob00037_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DPerola%26aufirst%3DE.%26atitle%3DConformational%2520Analysis%2520of%2520Drug-Like%2520Molecules%2520Bound%2520to%2520Proteins%253A%25E2%2580%2589%2520An%2520Extensive%2520Study%2520of%2520Ligand%2520Reorganization%2520upon%2520Binding%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D2510" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm051256ob00038"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm051256ob00038'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="jm051256ob00038_1"><span><span class="NLM_string-ref">Halgren, T. A. MMFF VI. MMFF94s Option for Energy Minimization Studies. <i>  J. Comput. Chem. </i><b>1999</b>, <i>20</i>, 720−729.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+Halgren%2C+T.+A.+MMFF+VI.+MMFF94s+Option+for+Energy+Minimization+Studies.+J.+Comput.+Chem.+1999%2C+20%2C+720%E2%88%92729.+Halgren%2C+T.+A.+MMFF+VI.+MMFF94s+Option+for+Energy+Minimization+Studies.+J.+Comput.+Chem.+1999%2C+20%2C+720%E2%88%92729."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm051256ob00038_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DHalgren%26aufirst%3DT.%2BA.%2BMMFF%2BVI.%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D1999%26volume%3D20%26spage%3D729" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm051256ob00039"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm051256ob00039'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="jm051256ob00039_1"><span><span class="NLM_string-ref">Cho, A. E.; Guallar, V.; Berne, B. J.; Friesner, R. A. Importance of Accurate Charges in Molecular Docking:  Quantum Mechanical/Molecular Mechanical (QM/MM) Approach. <i>  J. Comput. Chem. </i><b>2005</b>, <i>26</i>, 915−931.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2005&pages=931&author=A.+E.+Cho&author=V.+Guallar&author=B.+J.+Berne&author=R.+A+Friesner&title=Importance+of+Accurate+Charges+in+Molecular+Docking%3A%E2%80%89+Quantum+Mechanical%2FMolecular+Mechanical+%28QM%2FMM%29+Approach"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm051256ob00039_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DCho%26aufirst%3DA.%2BE.%26atitle%3DImportance%2520of%2520Accurate%2520Charges%2520in%2520Molecular%2520Docking%253A%25E2%2580%2589%2520Quantum%2520Mechanical%252FMolecular%2520Mechanical%2520%2528QM%252FMM%2529%2520Approach%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2005%26volume%3D26%26spage%3D931" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm051256ob00040"><div class="reference"><div class="NLM_citation" id="jm051256ob00040_1"><span><span class="NLM_string-ref"><i>Schrödinger User Manuals</i>, Glide v3.0; Schrödinger, L.L.C.:  New York, NY, 1994.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=1994&author=+Schr%C3%B6dinger+User+Manuals&title=Glide+v3.0"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm051256ob00040_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DSchr%25C3%25B6dinger%2BUser%2BManuals%26btitle%3DGlide%2520v3.0%26pub%3DNY%26date%3D1994" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm051256ob00041"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm051256ob00041'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="jm051256ob00041_1"><span><span class="NLM_string-ref"><i>QikProp v2.3</i>; Schrödinger, L.L.C.:  New York, NY, 2005.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+QikProp+v2.3%3B+Schr%C3%B6dinger%2C+L.L.C.%3A%E2%80%89+New+York%2C+NY%2C+2005.+QikProp+v2.3%3B+Schr%C3%B6dinger%2C+L.L.C.%3A%E2%80%89+New+York%2C+NY%2C+2005."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm051256ob00041_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dother%26sid%3Dliteratum%253Aachs" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm051256ob00042"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm051256ob00042'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="jm051256ob00042_1"><span><span class="NLM_string-ref">Wang, R.; Fang, X.; Lu, Y.; Wang, S. The PDBbind Database:  Collection of Binding Affinities for Protein−Ligand Complexes with Known Three-Dimensional Structures. <i>  J. Med. Chem.</i><b>2004</b>, <i>47</i>, 2977−2980.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=2980&author=R.+Wang&author=X.+Fang&author=Y.+Lu&author=S+Wang&title=The+PDBbind+Database%3A%E2%80%89+Collection+of+Binding+Affinities+for+Protein%E2%88%92Ligand+Complexes+with+Known+Three-Dimensional+Structures"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm051256ob00042_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DR.%26atitle%3DThe%2520PDBbind%2520Database%253A%25E2%2580%2589%2520Collection%2520of%2520Binding%2520Affinities%2520for%2520Protein%25E2%2588%2592Ligand%2520Complexes%2520with%2520Known%2520Three-Dimensional%2520Structures%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D2980" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'jm051256ob00001':['jm051256ob00001_1'],'jm051256ob00002':['jm051256ob00002_1'],'jm051256ob00003':['jm051256ob00003_1'],'jm051256ob00004':['jm051256ob00004_1'],'jm051256ob00005':['jm051256ob00005_1'],'jm051256ob00006':['jm051256ob00006_1'],'jm051256ob00007':['jm051256ob00007_1'],'jm051256ob00008':['jm051256ob00008_1'],'jm051256ob00009':['jm051256ob00009_1'],'jm051256ob00010':['jm051256ob00010_1'],'jm051256ob00011':['jm051256ob00011_1'],'jm051256ob00012':['jm051256ob00012_1'],'jm051256ob00013':['jm051256ob00013_1'],'jm051256ob00014':['jm051256ob00014_1'],'jm051256ob00015':['jm051256ob00015_1'],'jm051256ob00016':['jm051256ob00016_1'],'jm051256ob00017':['jm051256ob00017_1'],'jm051256ob00018':['jm051256ob00018_1'],'jm051256ob00019':['jm051256ob00019_1'],'jm051256ob00020':['jm051256ob00020_1'],'jm051256ob00021':['jm051256ob00021_1'],'jm051256ob00022':['jm051256ob00022_1'],'jm051256ob00023':['jm051256ob00023_1'],'jm051256ob00024':['jm051256ob00024_1'],'jm051256ob00025':['jm051256ob00025_1'],'jm051256ob00026':['jm051256ob00026_1'],'jm051256ob00027':['jm051256ob00027_1'],'jm051256ob00028':['jm051256ob00028_1'],'jm051256ob00029':['jm051256ob00029_1'],'jm051256ob00030':['jm051256ob00030_1'],'jm051256ob00031':['jm051256ob00031_1'],'jm051256ob00032':['jm051256ob00032_1'],'jm051256ob00033':['jm051256ob00033_1'],'jm051256ob00034':['jm051256ob00034_1'],'jm051256ob00035':['jm051256ob00035_1'],'jm051256ob00036':['jm051256ob00036_1'],'jm051256ob00037':['jm051256ob00037_1'],'jm051256ob00038':['jm051256ob00038_1'],'jm051256ob00039':['jm051256ob00039_1'],'jm051256ob00040':['jm051256ob00040_1'],'jm051256ob00041':['jm051256ob00041_1'],'jm051256ob00042':['jm051256ob00042_1']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 3115 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Senthil Renganathan, Sakthivel Manokaran, Preethi Vasanthakumar, Usha Singaravelu, Pok-Son Kim, Arne Kutzner, <span class="NLM_string-name hlFld-ContribAuthor">Klaus Heese</span>. </span><span class="cited-content_cbyCitation_article-title">Phytochemical Profiling in Conjunction with In Vitro and In Silico Studies to Identify Human α-Amylase Inhibitors in Leucaena leucocephala (Lam.) De Wit for the Treatment of Diabetes Mellitus. </span><span class="cited-content_cbyCitation_journal-name">ACS Omega</span><span> <strong>2021,</strong> <em>6 </em>
                                    (29)
                                     , 19045-19057. <a href="https://doi.org/10.1021/acsomega.1c02350" title="DOI URL">https://doi.org/10.1021/acsomega.1c02350</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsomega.1c02350&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsomega.1c02350%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Omega%26atitle%3DPhytochemical%252BProfiling%252Bin%252BConjunction%252Bwith%252BIn%252BVitro%252Band%252BIn%252BSilico%252BStudies%252Bto%252BIdentify%252BHuman%252B%2525CE%2525B1-Amylase%252BInhibitors%252Bin%252BLeucaena%252Bleucocephala%252B%252528Lam.%252529%252BDe%252BWit%252Bfor%252Bthe%252BTreatment%252Bof%252BDiabetes%252BMellitus%26aulast%3DRenganathan%26aufirst%3DSenthil%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D05052021%26date%3D05072021%26date%3D15072021%26volume%3D6%26issue%3D29%26spage%3D19045%26epage%3D19057" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Lauren Leung, Siyan Liao, <span class="NLM_string-name hlFld-ContribAuthor">Chun Wu</span>. </span><span class="cited-content_cbyCitation_article-title">To Probe the Binding Interactions between Two FDA Approved Migraine Drugs (Ubrogepant and Rimegepant) and Calcitonin-Gene Related Peptide Receptor (CGRPR) Using Molecular Dynamics Simulations. </span><span class="cited-content_cbyCitation_journal-name">ACS Chemical Neuroscience</span><span> <strong>2021,</strong> <em>12 </em>
                                    (14)
                                     , 2629-2642. <a href="https://doi.org/10.1021/acschemneuro.1c00135" title="DOI URL">https://doi.org/10.1021/acschemneuro.1c00135</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acschemneuro.1c00135&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facschemneuro.1c00135%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Chemical%2520Neuroscience%26atitle%3DTo%252BProbe%252Bthe%252BBinding%252BInteractions%252Bbetween%252BTwo%252BFDA%252BApproved%252BMigraine%252BDrugs%252B%252528Ubrogepant%252Band%252BRimegepant%252529%252Band%252BCalcitonin-Gene%252BRelated%252BPeptide%252BReceptor%252B%252528CGRPR%252529%252BUsing%252BMolecular%252BDynamics%252BSimulations%26aulast%3DLeung%26aufirst%3DLauren%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D09032021%26date%3D22062021%26date%3D29062021%26volume%3D12%26issue%3D14%26spage%3D2629%26epage%3D2642" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Lara Callea, Laura Bonati, <span class="NLM_string-name hlFld-ContribAuthor">Stefano Motta</span>. </span><span class="cited-content_cbyCitation_article-title">Metadynamics-Based Approaches for Modeling the Hypoxia-Inducible Factor 2α Ligand Binding Process. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Theory and Computation</span><span> <strong>2021,</strong> <em>17 </em>
                                    (7)
                                     , 3841-3851. <a href="https://doi.org/10.1021/acs.jctc.1c00114" title="DOI URL">https://doi.org/10.1021/acs.jctc.1c00114</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jctc.1c00114&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jctc.1c00114%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Theory%2520and%2520Computation%26atitle%3DMetadynamics-Based%252BApproaches%252Bfor%252BModeling%252Bthe%252BHypoxia-Inducible%252BFactor%252B2%2525CE%2525B1%252BLigand%252BBinding%252BProcess%26aulast%3DCallea%26aufirst%3DLara%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D01022021%26date%3D04062021%26volume%3D17%26issue%3D7%26spage%3D3841%26epage%3D3851" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Safia Deddouche-Grass, Cyrielle Andouche, Felix Bärenz, Célia Halter, Arnaud Hohwald, Louison Lebrun, Nathalie Membré, Renaud Morales, Nicolas Muzet, Matthieu Poirot, Morgane Reynaud, Véronique Roujean, Fabienne Weber, André Zimmermann, Rama Heng, <span class="NLM_string-name hlFld-ContribAuthor">Nicolas Basse</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery and Optimization of a Series of Benzofuran Selective ERAP1 Inhibitors: Biochemical and In Silico Studies. </span><span class="cited-content_cbyCitation_journal-name">ACS Medicinal Chemistry Letters</span><span> <strong>2021,</strong> <em>12 </em>
                                    (7)
                                     , 1137-1142. <a href="https://doi.org/10.1021/acsmedchemlett.1c00235" title="DOI URL">https://doi.org/10.1021/acsmedchemlett.1c00235</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsmedchemlett.1c00235&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsmedchemlett.1c00235%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DDiscovery%252Band%252BOptimization%252Bof%252Ba%252BSeries%252Bof%252BBenzofuran%252BSelective%252BERAP1%252BInhibitors%25253A%252BBiochemical%252Band%252BIn%252BSilico%252BStudies%26aulast%3DDeddouche-Grass%26aufirst%3DSafia%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D20042021%26date%3D03062021%26date%3D04062021%26volume%3D12%26issue%3D7%26spage%3D1137%26epage%3D1142" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Marek Staszewski, Dominik Nelic, Jakub Jończyk, Mariam Dubiel, Annika Frank, Holger Stark, Marek Bajda, Jan Jakubik, <span class="NLM_string-name hlFld-ContribAuthor">Krzysztof Walczyński</span>. </span><span class="cited-content_cbyCitation_article-title">Guanidine Derivatives: How Simple Structural Modification of Histamine H3R Antagonists Has Led to the Discovery of Potent Muscarinic M2R/M4R Antagonists. </span><span class="cited-content_cbyCitation_journal-name">ACS Chemical Neuroscience</span><span> <strong>2021,</strong> <em>12 </em>
                                    (13)
                                     , 2503-2519. <a href="https://doi.org/10.1021/acschemneuro.1c00237" title="DOI URL">https://doi.org/10.1021/acschemneuro.1c00237</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acschemneuro.1c00237&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facschemneuro.1c00237%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Chemical%2520Neuroscience%26atitle%3DGuanidine%252BDerivatives%25253A%252BHow%252BSimple%252BStructural%252BModification%252Bof%252BHistamine%252BH3R%252BAntagonists%252BHas%252BLed%252Bto%252Bthe%252BDiscovery%252Bof%252BPotent%252BMuscarinic%252BM2R%25252FM4R%252BAntagonists%26aulast%3DStaszewski%26aufirst%3DMarek%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D14042021%26date%3D25052021%26date%3D08062021%26volume%3D12%26issue%3D13%26spage%3D2503%26epage%3D2519" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Morgan L. Nance, Jason W. Labonte, Jared Adolf-Bryfogle, <span class="NLM_string-name hlFld-ContribAuthor">Jeffrey J. Gray</span>. </span><span class="cited-content_cbyCitation_article-title">Development and Evaluation of GlycanDock: A Protein–Glycoligand Docking Refinement Algorithm in Rosetta. </span><span class="cited-content_cbyCitation_journal-name">The Journal of Physical Chemistry B</span><span> <strong>2021,</strong> <em>125 </em>
                                    (25)
                                     , 6807-6820. <a href="https://doi.org/10.1021/acs.jpcb.1c00910" title="DOI URL">https://doi.org/10.1021/acs.jpcb.1c00910</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jpcb.1c00910&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jpcb.1c00910%26sid%3Dliteratum%253Aachs%26jtitle%3DThe%2520Journal%2520of%2520Physical%2520Chemistry%2520B%26atitle%3DDevelopment%252Band%252BEvaluation%252Bof%252BGlycanDock%25253A%252BA%252BProtein%2525E2%252580%252593Glycoligand%252BDocking%252BRefinement%252BAlgorithm%252Bin%252BRosetta%26aulast%3DNance%26aufirst%3DMorgan%2BL.%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D31012021%26date%3D30052021%26date%3D16062021%26volume%3D125%26issue%3D25%26spage%3D6807%26epage%3D6820" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Shani Zev, Keren Raz, Renana Schwartz, Reem Tarabeh, Prashant Kumar Gupta, <span class="NLM_string-name hlFld-ContribAuthor">Dan T. Major</span>. </span><span class="cited-content_cbyCitation_article-title">Benchmarking the Ability of Common Docking Programs to Correctly Reproduce and Score Binding Modes in SARS-CoV-2 Protease Mpro. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2021,</strong> <em>61 </em>
                                    (6)
                                     , 2957-2966. <a href="https://doi.org/10.1021/acs.jcim.1c00263" title="DOI URL">https://doi.org/10.1021/acs.jcim.1c00263</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.1c00263&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.1c00263%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DBenchmarking%252Bthe%252BAbility%252Bof%252BCommon%252BDocking%252BPrograms%252Bto%252BCorrectly%252BReproduce%252Band%252BScore%252BBinding%252BModes%252Bin%252BSARS-CoV-2%252BProtease%252BMpro%26aulast%3DZev%26aufirst%3DShani%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D06032021%26date%3D28052021%26volume%3D61%26issue%3D6%26spage%3D2957%26epage%3D2966" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Krimo Toutah, Nabanita Nawar, Sanna Timonen, Helena Sorger, Yasir S. Raouf, Shazreh Bukhari, Jana von Jan, Aleksandr Ianevski, Justyna M. Gawel, Olasunkanmi O. Olaoye, Mulu Geletu, Ayah Abdeldayem, Johan Israelian, Tudor B. Radu, Abootaleb Sedighi, Muzaffar N. Bhatti, Muhammad Murtaza Hassan, Pimyupa Manaswiyoungkul, Andrew E. Shouksmith, Heidi A. Neubauer, Elvin D. de Araujo, Tero Aittokallio, Oliver H. Krämer, Richard Moriggl, Satu Mustjoki, Marco Herling, <span class="NLM_string-name hlFld-ContribAuthor">Patrick T. Gunning</span>. </span><span class="cited-content_cbyCitation_article-title">Development of HDAC Inhibitors Exhibiting Therapeutic Potential in T-Cell Prolymphocytic Leukemia. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (12)
                                     , 8486-8509. <a href="https://doi.org/10.1021/acs.jmedchem.1c00420" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.1c00420</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.1c00420&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.1c00420%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDevelopment%252Bof%252BHDAC%252BInhibitors%252BExhibiting%252BTherapeutic%252BPotential%252Bin%252BT-Cell%252BProlymphocytic%252BLeukemia%26aulast%3DToutah%26aufirst%3DKrimo%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D08032021%26date%3D08062021%26volume%3D64%26issue%3D12%26spage%3D8486%26epage%3D8509" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Dehui Zhang, David A. Perrey, Ann M. Decker, Tiffany L. Langston, Vijayakumar Mavanji, Danni L. Harris, Catherine M. Kotz, <span class="NLM_string-name hlFld-ContribAuthor">Yanan Zhang</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of Arylsulfonamides as Dual Orexin Receptor Agonists. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (12)
                                     , 8806-8825. <a href="https://doi.org/10.1021/acs.jmedchem.1c00841" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.1c00841</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.1c00841&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.1c00841%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252BArylsulfonamides%252Bas%252BDual%252BOrexin%252BReceptor%252BAgonists%26aulast%3DZhang%26aufirst%3DDehui%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D09052021%26date%3D08062021%26volume%3D64%26issue%3D12%26spage%3D8806%26epage%3D8825" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Kader Sahin, Muge Didem Orhan, Timucin Avsar, <span class="NLM_string-name hlFld-ContribAuthor">Serdar Durdagi</span>. </span><span class="cited-content_cbyCitation_article-title">Hybrid In Silico and TR-FRET-Guided Discovery of Novel BCL-2 Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">ACS Pharmacology & Translational Science</span><span> <strong>2021,</strong> <em>4 </em>
                                    (3)
                                     , 1111-1123. <a href="https://doi.org/10.1021/acsptsci.0c00210" title="DOI URL">https://doi.org/10.1021/acsptsci.0c00210</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsptsci.0c00210&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsptsci.0c00210%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Pharmacology%2520%2526%2520Translational%2520Science%26atitle%3DHybrid%252BIn%252BSilico%252Band%252BTR-FRET-Guided%252BDiscovery%252Bof%252BNovel%252BBCL-2%252BInhibitors%26aulast%3DSahin%26aufirst%3DKader%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D09122020%26date%3D15042021%26volume%3D4%26issue%3D3%26spage%3D1111%26epage%3D1123" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Liberty François-Moutal, Victor G. Miranda, Niloufar Mollasalehi, Vijay Gokhale, <span class="NLM_string-name hlFld-ContribAuthor">May Khanna</span>. </span><span class="cited-content_cbyCitation_article-title">In Silico Targeting of the Long Noncoding RNA MALAT1. </span><span class="cited-content_cbyCitation_journal-name">ACS Medicinal Chemistry Letters</span><span> <strong>2021,</strong> <em>12 </em>
                                    (6)
                                     , 915-921. <a href="https://doi.org/10.1021/acsmedchemlett.1c00060" title="DOI URL">https://doi.org/10.1021/acsmedchemlett.1c00060</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsmedchemlett.1c00060&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsmedchemlett.1c00060%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DIn%252BSilico%252BTargeting%252Bof%252Bthe%252BLong%252BNoncoding%252BRNA%252BMALAT1%26aulast%3DFran%25C3%25A7ois-Moutal%26aufirst%3DLiberty%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D25012021%26date%3D24032021%26date%3D01042021%26volume%3D12%26issue%3D6%26spage%3D915%26epage%3D921" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Qiaojing Huang, Pengbo Song, Yixin Chen, Zhirong Liu, <span class="NLM_string-name hlFld-ContribAuthor">Luhua Lai</span>. </span><span class="cited-content_cbyCitation_article-title">Allosteric Type and Pathways Are Governed by the Forces of Protein–Ligand Binding. </span><span class="cited-content_cbyCitation_journal-name">The Journal of Physical Chemistry Letters</span><span> <strong>2021,</strong> <em>12 </em>
                                    (22)
                                     , 5404-5412. <a href="https://doi.org/10.1021/acs.jpclett.1c01253" title="DOI URL">https://doi.org/10.1021/acs.jpclett.1c01253</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jpclett.1c01253&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jpclett.1c01253%26sid%3Dliteratum%253Aachs%26jtitle%3DThe%2520Journal%2520of%2520Physical%2520Chemistry%2520Letters%26atitle%3DAllosteric%252BType%252Band%252BPathways%252BAre%252BGoverned%252Bby%252Bthe%252BForces%252Bof%252BProtein%2525E2%252580%252593Ligand%252BBinding%26aulast%3DHuang%26aufirst%3DQiaojing%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D18042021%26date%3D02062021%26date%3D03062021%26volume%3D12%26issue%3D22%26spage%3D5404%26epage%3D5412" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Esther C. Y. Lee, Andrew J. McRiner, Katy E. Georgiadis, Julie Liu, Zooey Wang, Andrew D. Ferguson, Benjamin Levin, Moritz von Rechenberg, Christopher D. Hupp, Michael I. Monteiro, Anthony D. Keefe, Allison Olszewski, Charles J. Eyermann, Paolo Centrella, Yanbin Liu, Shilpi Arora, John W. Cuozzo, Ying Zhang, Matthew A. Clark, Christelle Huguet, <span class="NLM_string-name hlFld-ContribAuthor">Anna Kohlmann</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of Novel, Potent Inhibitors of Hydroxy Acid Oxidase 1 (HAO1) Using DNA-Encoded Chemical Library Screening. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (10)
                                     , 6730-6744. <a href="https://doi.org/10.1021/acs.jmedchem.0c02271" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c02271</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c02271&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c02271%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252BNovel%25252C%252BPotent%252BInhibitors%252Bof%252BHydroxy%252BAcid%252BOxidase%252B1%252B%252528HAO1%252529%252BUsing%252BDNA-Encoded%252BChemical%252BLibrary%252BScreening%26aulast%3DLee%26aufirst%3DEsther%2BC.%2BY.%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D31122020%26date%3D06052021%26volume%3D64%26issue%3D10%26spage%3D6730%26epage%3D6744" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Jingxiao Bao, Xiao He, <span class="NLM_string-name hlFld-ContribAuthor">John Z. H. Zhang</span>. </span><span class="cited-content_cbyCitation_article-title">DeepBSP—a Machine Learning Method for Accurate Prediction of Protein–Ligand Docking Structures. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2021,</strong> <em>61 </em>
                                    (5)
                                     , 2231-2240. <a href="https://doi.org/10.1021/acs.jcim.1c00334" title="DOI URL">https://doi.org/10.1021/acs.jcim.1c00334</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.1c00334&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.1c00334%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DDeepBSP%2525E2%252580%252594a%252BMachine%252BLearning%252BMethod%252Bfor%252BAccurate%252BPrediction%252Bof%252BProtein%2525E2%252580%252593Ligand%252BDocking%252BStructures%26aulast%3DBao%26aufirst%3DJingxiao%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D20032021%26date%3D12052021%26volume%3D61%26issue%3D5%26spage%3D2231%26epage%3D2240" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Francois Berenger, Ashutosh Kumar, Kam Y. J. Zhang, <span class="NLM_string-name hlFld-ContribAuthor">Yoshihiro Yamanishi</span>. </span><span class="cited-content_cbyCitation_article-title">Lean-Docking: Exploiting Ligands’ Predicted Docking Scores to Accelerate Molecular Docking. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2021,</strong> <em>61 </em>
                                    (5)
                                     , 2341-2352. <a href="https://doi.org/10.1021/acs.jcim.0c01452" title="DOI URL">https://doi.org/10.1021/acs.jcim.0c01452</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.0c01452&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.0c01452%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DLean-Docking%25253A%252BExploiting%252BLigands%2525E2%252580%252599%252BPredicted%252BDocking%252BScores%252Bto%252BAccelerate%252BMolecular%252BDocking%26aulast%3DBerenger%26aufirst%3DFrancois%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D17122020%26date%3D16042021%26volume%3D61%26issue%3D5%26spage%3D2341%26epage%3D2352" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Michael S. Malamas, Spiro Pavlopoulos, Shakiru O. Alapafuja, Shrouq I. Farah, Alexander Zvonok, Khadijah A. Mohammad, Jay West, Nicholas Thomas Perry, Dimitrios N. Pelekoudas, Girija Rajarshi, Christina Shields, Honrao Chandrashekhar, Jodi Wood, <span class="NLM_string-name hlFld-ContribAuthor">Alexandros Makriyannis</span>. </span><span class="cited-content_cbyCitation_article-title">Design and Structure–Activity Relationships of Isothiocyanates as Potent and Selective N-Acylethanolamine-Hydrolyzing Acid Amidase Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (9)
                                     , 5956-5972. <a href="https://doi.org/10.1021/acs.jmedchem.1c00076" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.1c00076</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.1c00076&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.1c00076%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDesign%252Band%252BStructure%2525E2%252580%252593Activity%252BRelationships%252Bof%252BIsothiocyanates%252Bas%252BPotent%252Band%252BSelective%252BN-Acylethanolamine-Hydrolyzing%252BAcid%252BAmidase%252BInhibitors%26aulast%3DMalamas%26aufirst%3DMichael%2BS.%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D15012021%26date%3D26042021%26volume%3D64%26issue%3D9%26spage%3D5956%26epage%3D5972" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Stefano Sainas, Marta Giorgis, Paola Circosta, Valentina Gaidano, Davide Bonanni, Agnese C. Pippione, Renzo Bagnati, Alice Passoni, Yaqi Qiu, Carina Florina Cojocaru, Barbara Canepa, Alessandro Bona, Barbara Rolando, Mariia Mishina, Cristina Ramondetti, Barbara Buccinnà, Marco Piccinini, Mohammad Houshmand, Alessandro Cignetti, Enrico Giraudo, Salam Al-Karadaghi, Donatella Boschi, Giuseppe Saglio, <span class="NLM_string-name hlFld-ContribAuthor">Marco L. Lolli</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting Acute Myelogenous Leukemia Using Potent Human Dihydroorotate Dehydrogenase Inhibitors Based on the 2-Hydroxypyrazolo[1,5-a]pyridine Scaffold: SAR of the Biphenyl Moiety. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (9)
                                     , 5404-5428. <a href="https://doi.org/10.1021/acs.jmedchem.0c01549" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c01549</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c01549&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c01549%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DTargeting%252BAcute%252BMyelogenous%252BLeukemia%252BUsing%252BPotent%252BHuman%252BDihydroorotate%252BDehydrogenase%252BInhibitors%252BBased%252Bon%252Bthe%252B2-Hydroxypyrazolo%25255B1%25252C5-a%25255Dpyridine%252BScaffold%25253A%252BSAR%252Bof%252Bthe%252BBiphenyl%252BMoiety%26aulast%3DSainas%26aufirst%3DStefano%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D04092020%26date%3D12042021%26volume%3D64%26issue%3D9%26spage%3D5404%26epage%3D5428" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Mohammad Islamuddin, Obaid Afzal, Wajihul Hasan Khan, Malik Hisamuddin, Abdulmalik Saleh Alfawaz Altamimi, Ibraheem Husain, Kentaro Kato, Mubarak A. Alamri, <span class="NLM_string-name hlFld-ContribAuthor">Shama Parveen</span>. </span><span class="cited-content_cbyCitation_article-title">Inhibition of Chikungunya Virus Infection by 4-Hydroxy-1-Methyl-3-(3-morpholinopropanoyl)quinoline-2(1H)-one (QVIR) Targeting nsP2 and E2 Proteins. </span><span class="cited-content_cbyCitation_journal-name">ACS Omega</span><span> <strong>2021,</strong> <em>6 </em>
                                    (14)
                                     , 9791-9803. <a href="https://doi.org/10.1021/acsomega.1c00447" title="DOI URL">https://doi.org/10.1021/acsomega.1c00447</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsomega.1c00447&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsomega.1c00447%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Omega%26atitle%3DInhibition%252Bof%252BChikungunya%252BVirus%252BInfection%252Bby%252B4-Hydroxy-1-Methyl-3-%2525283-morpholinopropanoyl%252529quinoline-2%2525281H%252529-one%252B%252528QVIR%252529%252BTargeting%252BnsP2%252Band%252BE2%252BProteins%26aulast%3DIslamuddin%26aufirst%3DMohammad%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D25012021%26date%3D23032021%26date%3D31032021%26volume%3D6%26issue%3D14%26spage%3D9791%26epage%3D9803" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Shan Jiang, Christos Iliopoulos-Tsoutsouvas, Fei Tong, Christina A. Brust, Catherine M. Keenan, Jimit Girish Raghav, Tian Hua, Simiao Wu, Jo-Hao Ho, Yiran Wu, Travis W. Grim, Nikolai Zvonok, Ganesh A. Thakur, Zhi-Jie Liu, Keith A. Sharkey, Laura M. Bohn, Spyros P. Nikas, <span class="NLM_string-name hlFld-ContribAuthor">Alexandros Makriyannis</span>. </span><span class="cited-content_cbyCitation_article-title">Novel Functionalized Cannabinoid Receptor Probes: Development of Exceptionally Potent Agonists. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (7)
                                     , 3870-3884. <a href="https://doi.org/10.1021/acs.jmedchem.0c02053" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c02053</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c02053&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c02053%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DNovel%252BFunctionalized%252BCannabinoid%252BReceptor%252BProbes%25253A%252BDevelopment%252Bof%252BExceptionally%252BPotent%252BAgonists%26aulast%3DJiang%26aufirst%3DShan%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D25112020%26date%3D24032021%26volume%3D64%26issue%3D7%26spage%3D3870%26epage%3D3884" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Johnny Birch, Sanaullah Khan, Mikkel Madsen, Christian Kjeldsen, Marie Sofie Møller, Emil G. P. Stender, Günther H. J. Peters, Jens Ø. Duus, Birthe B. Kragelund, <span class="NLM_string-name hlFld-ContribAuthor">Birte Svensson</span>. </span><span class="cited-content_cbyCitation_article-title">Binding Sites for Oligosaccharide Repeats from Lactic Acid Bacteria Exopolysaccharides on Bovine β-Lactoglobulin Identified by NMR Spectroscopy. </span><span class="cited-content_cbyCitation_journal-name">ACS Omega</span><span> <strong>2021,</strong> <em>6 </em>
                                    (13)
                                     , 9039-9052. <a href="https://doi.org/10.1021/acsomega.1c00060" title="DOI URL">https://doi.org/10.1021/acsomega.1c00060</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsomega.1c00060&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsomega.1c00060%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Omega%26atitle%3DBinding%252BSites%252Bfor%252BOligosaccharide%252BRepeats%252Bfrom%252BLactic%252BAcid%252BBacteria%252BExopolysaccharides%252Bon%252BBovine%252B%2525CE%2525B2-Lactoglobulin%252BIdentified%252Bby%252BNMR%252BSpectroscopy%26aulast%3DBirch%26aufirst%3DJohnny%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D05012021%26date%3D11032021%26date%3D23032021%26volume%3D6%26issue%3D13%26spage%3D9039%26epage%3D9052" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Qi Chen, Wei Chen, Ashutosh Kumar, Xi Jiang, Matej Janezic, Kam Y. J. Zhang, <span class="NLM_string-name hlFld-ContribAuthor">Qing Yang</span>. </span><span class="cited-content_cbyCitation_article-title">Crystal Structure and Structure-Based Discovery of Inhibitors of the Nematode Chitinase CeCht1. </span><span class="cited-content_cbyCitation_journal-name">Journal of Agricultural and Food Chemistry</span><span> <strong>2021,</strong> <em>69 </em>
                                    (11)
                                     , 3519-3526. <a href="https://doi.org/10.1021/acs.jafc.1c00162" title="DOI URL">https://doi.org/10.1021/acs.jafc.1c00162</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jafc.1c00162&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jafc.1c00162%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Agricultural%2520and%2520Food%2520Chemistry%26atitle%3DCrystal%252BStructure%252Band%252BStructure-Based%252BDiscovery%252Bof%252BInhibitors%252Bof%252Bthe%252BNematode%252BChitinase%252BCeCht1%26aulast%3DChen%26aufirst%3DQi%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D10012021%26date%3D02032021%26date%3D26022021%26date%3D11032021%26volume%3D69%26issue%3D11%26spage%3D3519%26epage%3D3526" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Lamya H. Al-Wahaibi, Bavanandan Rahul, Ahmed A. B. Mohamed, Mohammed S. M. Abdelbaky, Santiago Garcia-Granda, Ali A. El-Emam, M. Judith Percino, <span class="NLM_string-name hlFld-ContribAuthor">Subbiah Thamotharan</span>. </span><span class="cited-content_cbyCitation_article-title">Supramolecular Self-Assembly Built by Weak Hydrogen, Chalcogen, and Unorthodox Nonbonded Motifs in 4-(4-Chlorophenyl)-3-[(4-fluorobenzyl)sulfanyl]-5-(thiophen-2-yl)-4H-1,2,4-triazole, a Selective COX-2 Inhibitor: Insights from X-ray and Theoretical Studies. </span><span class="cited-content_cbyCitation_journal-name">ACS Omega</span><span> <strong>2021,</strong> <em>6 </em>
                                    (10)
                                     , 6996-7007. <a href="https://doi.org/10.1021/acsomega.0c06287" title="DOI URL">https://doi.org/10.1021/acsomega.0c06287</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsomega.0c06287&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsomega.0c06287%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Omega%26atitle%3DSupramolecular%252BSelf-Assembly%252BBuilt%252Bby%252BWeak%252BHydrogen%25252C%252BChalcogen%25252C%252Band%252BUnorthodox%252BNonbonded%252BMotifs%252Bin%252B4-%2525284-Chlorophenyl%252529-3-%25255B%2525284-fluorobenzyl%252529sulfanyl%25255D-5-%252528thiophen-2-yl%252529-4H-1%25252C2%25252C4-triazole%25252C%252Ba%252BSelective%252BCOX-2%252BInhibitor%25253A%252BInsights%252Bfrom%252BX-ray%252Band%252BTheoretical%252BStudies%26aulast%3DAl-Wahaibi%26aufirst%3DLamya%2BH.%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D26122020%26date%3D22022021%26date%3D03032021%26volume%3D6%26issue%3D10%26spage%3D6996%26epage%3D7007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Jeffrey Y. W. Mak, Kai-Chen Wu, Praveer K. Gupta, Sheila Barbero, Maddison G. McLaughlin, Andrew J. Lucke, Jiahui Tng, Junxian Lim, Zhixuan Loh, Matthew J. Sweet, Robert C. Reid, Ligong Liu, <span class="NLM_string-name hlFld-ContribAuthor">David P. Fairlie</span>. </span><span class="cited-content_cbyCitation_article-title">HDAC7 Inhibition by Phenacetyl and Phenylbenzoyl Hydroxamates. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (4)
                                     , 2186-2204. <a href="https://doi.org/10.1021/acs.jmedchem.0c01967" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c01967</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c01967&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c01967%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DHDAC7%252BInhibition%252Bby%252BPhenacetyl%252Band%252BPhenylbenzoyl%252BHydroxamates%26aulast%3DMak%26aufirst%3DJeffrey%2BY.%2BW.%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D13112020%26date%3D11022021%26volume%3D64%26issue%3D4%26spage%3D2186%26epage%3D2204" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Jovan Damjanovic, Jiayuan Miao, He Huang, <span class="NLM_string-name hlFld-ContribAuthor">Yu-Shan Lin</span>. </span><span class="cited-content_cbyCitation_article-title">Elucidating Solution Structures of Cyclic Peptides Using Molecular Dynamics Simulations. </span><span class="cited-content_cbyCitation_journal-name">Chemical Reviews</span><span> <strong>2021,</strong> <em>121 </em>
                                    (4)
                                     , 2292-2324. <a href="https://doi.org/10.1021/acs.chemrev.0c01087" title="DOI URL">https://doi.org/10.1021/acs.chemrev.0c01087</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.chemrev.0c01087&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.chemrev.0c01087%26sid%3Dliteratum%253Aachs%26jtitle%3DChemical%2520Reviews%26atitle%3DElucidating%252BSolution%252BStructures%252Bof%252BCyclic%252BPeptides%252BUsing%252BMolecular%252BDynamics%252BSimulations%26aulast%3DDamjanovic%26aufirst%3DJovan%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D05102020%26date%3D11012021%26volume%3D121%26issue%3D4%26spage%3D2292%26epage%3D2324" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Anjela Manandhar, Benjamin E. Blass, Dennis J. Colussi, Imane Almi, Magid Abou-Gharbia, Michael L. Klein, <span class="NLM_string-name hlFld-ContribAuthor">Khaled M. Elokely</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting SARS-CoV-2 M3CLpro by HCV NS3/4a Inhibitors: In Silico Modeling and In Vitro Screening. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2021,</strong> <em>61 </em>
                                    (2)
                                     , 1020-1032. <a href="https://doi.org/10.1021/acs.jcim.0c01457" title="DOI URL">https://doi.org/10.1021/acs.jcim.0c01457</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.0c01457&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.0c01457%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DTargeting%252BSARS-CoV-2%252BM3CLpro%252Bby%252BHCV%252BNS3%25252F4a%252BInhibitors%25253A%252BIn%252BSilico%252BModeling%252Band%252BIn%252BVitro%252BScreening%26aulast%3DManandhar%26aufirst%3DAnjela%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D18122020%26date%3D04022021%26volume%3D61%26issue%3D2%26spage%3D1020%26epage%3D1032" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Vanessa Jane Banun, Prarthana Rewatkar, Zanib Chaudhary, Zhi Qu, Taskeen Janjua, Anuja Patil, Yuao Wu, Hang T. Ta, Nidhi Bansal, Jared A. Miles, Benjamin P. Ross, Tushar Kumeria, <span class="NLM_string-name hlFld-ContribAuthor">Amirali Popat</span>. </span><span class="cited-content_cbyCitation_article-title">Protein Nanoparticles for Enhanced Oral Delivery of Coenzyme-Q10: in Vitro and in Silico Studies. </span><span class="cited-content_cbyCitation_journal-name">ACS Biomaterials Science & Engineering</span><span> <strong>2021</strong>, Article ASAP.</span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsbiomaterials.0c01354&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsbiomaterials.0c01354%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Biomaterials%2520Science%2520%2526%2520Engineering%26atitle%3DProtein%252BNanoparticles%252Bfor%252BEnhanced%252BOral%252BDelivery%252Bof%252BCoenzyme-Q10%25253A%252Bin%252BVitro%252Band%252Bin%252BSilico%252BStudies%26aulast%3DBanun%26aufirst%3DVanessa%2BJane%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D11092020%26date%3D09022021%26volume%3D0%26issue%3D0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Sarah Huff, Shashi Kant Tiwari, Gwendolyn M. Gonzalez, Yinsheng Wang, <span class="NLM_string-name hlFld-ContribAuthor">Tariq M. Rana</span>. </span><span class="cited-content_cbyCitation_article-title">m6A-RNA Demethylase FTO Inhibitors Impair Self-Renewal in Glioblastoma Stem Cells. </span><span class="cited-content_cbyCitation_journal-name">ACS Chemical Biology</span><span> <strong>2021,</strong> <em>16 </em>
                                    (2)
                                     , 324-333. <a href="https://doi.org/10.1021/acschembio.0c00841" title="DOI URL">https://doi.org/10.1021/acschembio.0c00841</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acschembio.0c00841&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facschembio.0c00841%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Chemical%2520Biology%26atitle%3Dm6A-RNA%252BDemethylase%252BFTO%252BInhibitors%252BImpair%252BSelf-Renewal%252Bin%252BGlioblastoma%252BStem%252BCells%26aulast%3DHuff%26aufirst%3DSarah%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D2020%26date%3D28102020%26date%3D29122020%26date%3D07012021%26volume%3D16%26issue%3D2%26spage%3D324%26epage%3D333" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Amit Kumawat, Shabnam Raheem, Fasil Ali, Tanveer Ali Dar, Suman Chakrabarty, <span class="NLM_string-name hlFld-ContribAuthor">Masood Ahmad Rizvi</span>. </span><span class="cited-content_cbyCitation_article-title">Organoselenium Compounds as Acetylcholinesterase Inhibitors: Evidence and Mechanism of Mixed Inhibition. </span><span class="cited-content_cbyCitation_journal-name">The Journal of Physical Chemistry B</span><span> <strong>2021,</strong> <em>125 </em>
                                    (6)
                                     , 1531-1541. <a href="https://doi.org/10.1021/acs.jpcb.0c08111" title="DOI URL">https://doi.org/10.1021/acs.jpcb.0c08111</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jpcb.0c08111&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jpcb.0c08111%26sid%3Dliteratum%253Aachs%26jtitle%3DThe%2520Journal%2520of%2520Physical%2520Chemistry%2520B%26atitle%3DOrganoselenium%252BCompounds%252Bas%252BAcetylcholinesterase%252BInhibitors%25253A%252BEvidence%252Band%252BMechanism%252Bof%252BMixed%252BInhibition%26aulast%3DKumawat%26aufirst%3DAmit%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D2021%26date%3D05092020%26date%3D21012021%26date%3D04022021%26volume%3D125%26issue%3D6%26spage%3D1531%26epage%3D1541" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Prayasee Baruah, Abhinandan Das, Debojit Paul, Suman Chakrabarty, Kripamoy Aguan, <span class="NLM_string-name hlFld-ContribAuthor">Sivaprasad Mitra</span>. </span><span class="cited-content_cbyCitation_article-title">Sulfonylurea Class of Antidiabetic Drugs Inhibit Acetylcholinesterase Activity: Unexplored Auxiliary Pharmacological Benefit toward Alzheimer’s Disease. </span><span class="cited-content_cbyCitation_journal-name">ACS Pharmacology & Translational Science</span><span> <strong>2021,</strong> <em>4 </em>
                                    (1)
                                     , 193-205. <a href="https://doi.org/10.1021/acsptsci.0c00168" title="DOI URL">https://doi.org/10.1021/acsptsci.0c00168</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsptsci.0c00168&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsptsci.0c00168%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Pharmacology%2520%2526%2520Translational%2520Science%26atitle%3DSulfonylurea%252BClass%252Bof%252BAntidiabetic%252BDrugs%252BInhibit%252BAcetylcholinesterase%252BActivity%25253A%252BUnexplored%252BAuxiliary%252BPharmacological%252BBenefit%252Btoward%252BAlzheimer%2525E2%252580%252599s%252BDisease%26aulast%3DBaruah%26aufirst%3DPrayasee%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D13102020%26date%3D06012021%26volume%3D4%26issue%3D1%26spage%3D193%26epage%3D205" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Ajit Kandale, Khushboo Patel, Waleed M. Hussein, Shun Jie Wun, Shan Zheng, Lendl Tan, Nicholas P. West, Gerhard Schenk, Luke W. Guddat, <span class="NLM_string-name hlFld-ContribAuthor">Ross P. McGeary</span>. </span><span class="cited-content_cbyCitation_article-title">Analogues of the Herbicide, N-Hydroxy-N-isopropyloxamate, Inhibit Mycobacterium tuberculosis Ketol-Acid Reductoisomerase and Their Prodrugs Are Promising Anti-TB Drug Leads. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (3)
                                     , 1670-1684. <a href="https://doi.org/10.1021/acs.jmedchem.0c01919" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c01919</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c01919&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c01919%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DAnalogues%252Bof%252Bthe%252BHerbicide%25252C%252BN-Hydroxy-N-isopropyloxamate%25252C%252BInhibit%252BMycobacterium%252Btuberculosis%252BKetol-Acid%252BReductoisomerase%252Band%252BTheir%252BProdrugs%252BAre%252BPromising%252BAnti-TB%252BDrug%252BLeads%26aulast%3DKandale%26aufirst%3DAjit%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D05112020%26date%3D29012021%26volume%3D64%26issue%3D3%26spage%3D1670%26epage%3D1684" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Miaomiao Li, <span class="NLM_string-name hlFld-ContribAuthor">Jingjing Guo</span>. </span><span class="cited-content_cbyCitation_article-title">Deciphering the T790M/L858R-Selective Inhibition Mechanism of an Allosteric Inhibitor of EGFR: Insights from Molecular Simulations. </span><span class="cited-content_cbyCitation_journal-name">ACS Chemical Neuroscience</span><span> <strong>2021,</strong> <em>12 </em>
                                    (3)
                                     , 462-472. <a href="https://doi.org/10.1021/acschemneuro.0c00633" title="DOI URL">https://doi.org/10.1021/acschemneuro.0c00633</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acschemneuro.0c00633&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facschemneuro.0c00633%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Chemical%2520Neuroscience%26atitle%3DDeciphering%252Bthe%252BT790M%25252FL858R-Selective%252BInhibition%252BMechanism%252Bof%252Ban%252BAllosteric%252BInhibitor%252Bof%252BEGFR%25253A%252BInsights%252Bfrom%252BMolecular%252BSimulations%26aulast%3DLi%26aufirst%3DMiaomiao%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D03102020%26date%3D07012021%26date%3D13012021%26volume%3D12%26issue%3D3%26spage%3D462%26epage%3D472" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Qingxia Huang, Hang Su, Bin Qi, Ying Wang, Kaili Yan, Xinglin Wang, Xiangyan Li, <span class="NLM_string-name hlFld-ContribAuthor">Daqing Zhao</span>. </span><span class="cited-content_cbyCitation_article-title">A SIRT1 Activator, Ginsenoside Rc, Promotes Energy Metabolism in Cardiomyocytes and Neurons. </span><span class="cited-content_cbyCitation_journal-name">Journal of the American Chemical Society</span><span> <strong>2021,</strong> <em>143 </em>
                                    (3)
                                     , 1416-1427. <a href="https://doi.org/10.1021/jacs.0c10836" title="DOI URL">https://doi.org/10.1021/jacs.0c10836</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/jacs.0c10836&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fjacs.0c10836%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520the%2520American%2520Chemical%2520Society%26atitle%3DA%252BSIRT1%252BActivator%25252C%252BGinsenoside%252BRc%25252C%252BPromotes%252BEnergy%252BMetabolism%252Bin%252BCardiomyocytes%252Band%252BNeurons%26aulast%3DHuang%26aufirst%3DQingxia%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D13102020%26date%3D13012021%26volume%3D143%26issue%3D3%26spage%3D1416%26epage%3D1427" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Yu Yang, Timothy Borel, Francisco de Azambuja, David Johnson, Jacob P. Sorrentino, Chinedum Udokwu, Ian Davis, Aimin Liu, <span class="NLM_string-name hlFld-ContribAuthor">Ryan A. Altman</span>. </span><span class="cited-content_cbyCitation_article-title">Diflunisal Derivatives as Modulators of ACMS Decarboxylase Targeting the Tryptophan–Kynurenine Pathway. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (1)
                                     , 797-811. <a href="https://doi.org/10.1021/acs.jmedchem.0c01762" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c01762</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c01762&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c01762%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiflunisal%252BDerivatives%252Bas%252BModulators%252Bof%252BACMS%252BDecarboxylase%252BTargeting%252Bthe%252BTryptophan%2525E2%252580%252593Kynurenine%252BPathway%26aulast%3DYang%26aufirst%3DYu%26date%3D2021%26date%3D2020%26date%3D2020%26date%3D07102020%26date%3D28122020%26volume%3D64%26issue%3D1%26spage%3D797%26epage%3D811" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Akshita Gupta, Chitra Rani, Pradeep Pant, Viswanathan Vijayan, Naval Vikram, Punit Kaur, Tej Pal Singh, Sujata Sharma, <span class="NLM_string-name hlFld-ContribAuthor">Pradeep Sharma</span>. </span><span class="cited-content_cbyCitation_article-title">Structure-Based Virtual Screening and Biochemical Validation to Discover a Potential Inhibitor of the SARS-CoV-2 Main Protease. </span><span class="cited-content_cbyCitation_journal-name">ACS Omega</span><span> <strong>2020,</strong> <em>5 </em>
                                    (51)
                                     , 33151-33161. <a href="https://doi.org/10.1021/acsomega.0c04808" title="DOI URL">https://doi.org/10.1021/acsomega.0c04808</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsomega.0c04808&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsomega.0c04808%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Omega%26atitle%3DStructure-Based%252BVirtual%252BScreening%252Band%252BBiochemical%252BValidation%252Bto%252BDiscover%252Ba%252BPotential%252BInhibitor%252Bof%252Bthe%252BSARS-CoV-2%252BMain%252BProtease%26aulast%3DGupta%26aufirst%3DAkshita%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D01102020%26date%3D09122020%26date%3D17122020%26volume%3D5%26issue%3D51%26spage%3D33151%26epage%3D33161" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Nurlybek Amangeldiuly, Dmitry Karlov, <span class="NLM_string-name hlFld-ContribAuthor">Maxim V. Fedorov</span>. </span><span class="cited-content_cbyCitation_article-title">Baseline Model for Predicting Protein–Ligand Unbinding Kinetics through Machine Learning. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2020,</strong> <em>60 </em>
                                    (12)
                                     , 5946-5956. <a href="https://doi.org/10.1021/acs.jcim.0c00450" title="DOI URL">https://doi.org/10.1021/acs.jcim.0c00450</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.0c00450&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.0c00450%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DBaseline%252BModel%252Bfor%252BPredicting%252BProtein%2525E2%252580%252593Ligand%252BUnbinding%252BKinetics%252Bthrough%252BMachine%252BLearning%26aulast%3DAmangeldiuly%26aufirst%3DNurlybek%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D29042020%26date%3D13112020%26volume%3D60%26issue%3D12%26spage%3D5946%26epage%3D5956" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Jingxiao Bao, Xiao He, <span class="NLM_string-name hlFld-ContribAuthor">John Z.H. Zhang</span>. </span><span class="cited-content_cbyCitation_article-title">Development of a New Scoring Function for Virtual Screening: APBScore. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2020,</strong> <em>60 </em>
                                    (12)
                                     , 6355-6365. <a href="https://doi.org/10.1021/acs.jcim.0c00474" title="DOI URL">https://doi.org/10.1021/acs.jcim.0c00474</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.0c00474&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.0c00474%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DDevelopment%252Bof%252Ba%252BNew%252BScoring%252BFunction%252Bfor%252BVirtual%252BScreening%25253A%252BAPBScore%26aulast%3DBao%26aufirst%3DJingxiao%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D05052020%26date%3D14102020%26volume%3D60%26issue%3D12%26spage%3D6355%26epage%3D6365" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Anuj Gahlawat, Navneet Kumar, Rajender Kumar, Hardeep Sandhu, Inder Pal Singh, Saranjit Singh, Anders Sjöstedt, <span class="NLM_string-name hlFld-ContribAuthor">Prabha Garg</span>. </span><span class="cited-content_cbyCitation_article-title">Structure-Based Virtual Screening to Discover Potential Lead Molecules for the SARS-CoV-2 Main Protease. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2020,</strong> <em>60 </em>
                                    (12)
                                     , 5781-5793. <a href="https://doi.org/10.1021/acs.jcim.0c00546" title="DOI URL">https://doi.org/10.1021/acs.jcim.0c00546</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.0c00546&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.0c00546%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DStructure-Based%252BVirtual%252BScreening%252Bto%252BDiscover%252BPotential%252BLead%252BMolecules%252Bfor%252Bthe%252BSARS-CoV-2%252BMain%252BProtease%26aulast%3DGahlawat%26aufirst%3DAnuj%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D16052020%26date%3D04082020%26date%3D20072020%26volume%3D60%26issue%3D12%26spage%3D5781%26epage%3D5793" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Mahdi Mousaei, Meruyert Kudaibergenova, Alexander D. MacKerell, Jr., <span class="NLM_string-name hlFld-ContribAuthor">Sergei Noskov</span>. </span><span class="cited-content_cbyCitation_article-title">Assessing hERG1 Blockade from Bayesian Machine-Learning-Optimized Site Identification by Ligand Competitive Saturation Simulations. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2020,</strong> <em>60 </em>
                                    (12)
                                     , 6489-6501. <a href="https://doi.org/10.1021/acs.jcim.0c01065" title="DOI URL">https://doi.org/10.1021/acs.jcim.0c01065</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.0c01065&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.0c01065%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DAssessing%252BhERG1%252BBlockade%252Bfrom%252BBayesian%252BMachine-Learning-Optimized%252BSite%252BIdentification%252Bby%252BLigand%252BCompetitive%252BSaturation%252BSimulations%26aulast%3DMousaei%26aufirst%3DMahdi%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D11092020%26date%3D16112020%26volume%3D60%26issue%3D12%26spage%3D6489%26epage%3D6501" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Francesco Bavo, Marco Pallavicini, Cecilia Gotti, Rebecca Appiani, Milena Moretti, Sara Francesca Colombo, Susanna Pucci, Paola Viani, Roberta Budriesi, Massimiliano Renzi, Sergio Fucile, <span class="NLM_string-name hlFld-ContribAuthor">Cristiano Bolchi</span>. </span><span class="cited-content_cbyCitation_article-title">Modifications at C(5) of 2-(2-Pyrrolidinyl)-Substituted 1,4-Benzodioxane Elicit Potent α4β2 Nicotinic Acetylcholine Receptor Partial Agonism with High Selectivity over the α3β4 Subtype. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (24)
                                     , 15668-15692. <a href="https://doi.org/10.1021/acs.jmedchem.0c01150" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c01150</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c01150&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c01150%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DModifications%252Bat%252BC%2525285%252529%252Bof%252B2-%2525282-Pyrrolidinyl%252529-Substituted%252B1%25252C4-Benzodioxane%252BElicit%252BPotent%252B%2525CE%2525B14%2525CE%2525B22%252BNicotinic%252BAcetylcholine%252BReceptor%252BPartial%252BAgonism%252Bwith%252BHigh%252BSelectivity%252Bover%252Bthe%252B%2525CE%2525B13%2525CE%2525B24%252BSubtype%26aulast%3DBavo%26aufirst%3DFrancesco%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D17072020%26date%3D16122020%26volume%3D63%26issue%3D24%26spage%3D15668%26epage%3D15692" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Patrick G. Dougherty, Jack H. Wellmerling, Amritendu Koley, Jessica K. Lukowski, Amanda B. Hummon, Estelle Cormet-Boyaka, <span class="NLM_string-name hlFld-ContribAuthor">Dehua Pei</span>. </span><span class="cited-content_cbyCitation_article-title">Cyclic Peptidyl Inhibitors against CAL/CFTR Interaction for Treatment of Cystic Fibrosis. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (24)
                                     , 15773-15784. <a href="https://doi.org/10.1021/acs.jmedchem.0c01528" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c01528</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c01528&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c01528%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DCyclic%252BPeptidyl%252BInhibitors%252Bagainst%252BCAL%25252FCFTR%252BInteraction%252Bfor%252BTreatment%252Bof%252BCystic%252BFibrosis%26aulast%3DDougherty%26aufirst%3DPatrick%2BG.%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D01092020%26date%3D14122020%26volume%3D63%26issue%3D24%26spage%3D15773%26epage%3D15784" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Christine Robinson, Valeria Gradinati, Fatima Hamid, Carly Baehr, Bethany Crouse, Saadyah Averick, Marina Kovaliov, Danni Harris, Scott Runyon, Federico Baruffaldi, Mark LeSage, Sandra Comer, <span class="NLM_string-name hlFld-ContribAuthor">Marco Pravetoni</span>. </span><span class="cited-content_cbyCitation_article-title">Therapeutic and Prophylactic Vaccines to Counteract Fentanyl Use Disorders and Toxicity. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (23)
                                     , 14647-14667. <a href="https://doi.org/10.1021/acs.jmedchem.0c01042" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c01042</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c01042&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c01042%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DTherapeutic%252Band%252BProphylactic%252BVaccines%252Bto%252BCounteract%252BFentanyl%252BUse%252BDisorders%252Band%252BToxicity%26aulast%3DRobinson%26aufirst%3DChristine%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D13072020%26date%3D20112020%26volume%3D63%26issue%3D23%26spage%3D14647%26epage%3D14667" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Martí Municoy, Sergi Roda, Daniel Soler, Alberto Soutullo, <span class="NLM_string-name hlFld-ContribAuthor">Victor Guallar</span>. </span><span class="cited-content_cbyCitation_article-title">aquaPELE: A Monte Carlo-Based Algorithm to Sample the Effects of Buried Water Molecules in Proteins. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Theory and Computation</span><span> <strong>2020,</strong> <em>16 </em>
                                    (12)
                                     , 7655-7670. <a href="https://doi.org/10.1021/acs.jctc.0c00925" title="DOI URL">https://doi.org/10.1021/acs.jctc.0c00925</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jctc.0c00925&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jctc.0c00925%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Theory%2520and%2520Computation%26atitle%3DaquaPELE%25253A%252BA%252BMonte%252BCarlo-Based%252BAlgorithm%252Bto%252BSample%252Bthe%252BEffects%252Bof%252BBuried%252BWater%252BMolecules%252Bin%252BProteins%26aulast%3DMunicoy%26aufirst%3DMart%25C3%25AD%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D07092020%26date%3D17112020%26volume%3D16%26issue%3D12%26spage%3D7655%26epage%3D7670" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Satish N. Dighe, Mangapathiraju Tippana, Suzannah van Akker, <span class="NLM_string-name hlFld-ContribAuthor">Trudi A. Collet</span>. </span><span class="cited-content_cbyCitation_article-title">Structure-Based Scaffold Repurposing toward the Discovery of Novel Cholinesterase Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">ACS Omega</span><span> <strong>2020,</strong> <em>5 </em>
                                    (48)
                                     , 30971-30979. <a href="https://doi.org/10.1021/acsomega.0c03848" title="DOI URL">https://doi.org/10.1021/acsomega.0c03848</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsomega.0c03848&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsomega.0c03848%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Omega%26atitle%3DStructure-Based%252BScaffold%252BRepurposing%252Btoward%252Bthe%252BDiscovery%252Bof%252BNovel%252BCholinesterase%252BInhibitors%26aulast%3DDighe%26aufirst%3DSatish%2BN.%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D10082020%26date%3D12112020%26date%3D24112020%26volume%3D5%26issue%3D48%26spage%3D30971%26epage%3D30979" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Hiraku Oshima, Suyong Re, <span class="NLM_string-name hlFld-ContribAuthor">Yuji Sugita</span>. </span><span class="cited-content_cbyCitation_article-title">Prediction of Protein–Ligand Binding Pose and Affinity Using the gREST+FEP Method. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2020,</strong> <em>60 </em>
                                    (11)
                                     , 5382-5394. <a href="https://doi.org/10.1021/acs.jcim.0c00338" title="DOI URL">https://doi.org/10.1021/acs.jcim.0c00338</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.0c00338&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.0c00338%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DPrediction%252Bof%252BProtein%2525E2%252580%252593Ligand%252BBinding%252BPose%252Band%252BAffinity%252BUsing%252Bthe%252BgREST%25252BFEP%252BMethod%26aulast%3DOshima%26aufirst%3DHiraku%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D07042020%26date%3D17082020%26date%3D31072020%26volume%3D60%26issue%3D11%26spage%3D5382%26epage%3D5394" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Patrick G. Dougherty, Manjula Karpurapu, Amritendu Koley, Jessica K. Lukowski, Ziqing Qian, Teja Srinivas Nirujogi, Luiza Rusu, Sangwoon Chung, Amanda B. Hummon, Hao W. Li, John W. Christman, <span class="NLM_string-name hlFld-ContribAuthor">Dehua Pei</span>. </span><span class="cited-content_cbyCitation_article-title">A Peptidyl Inhibitor that Blocks Calcineurin–NFAT Interaction and Prevents Acute Lung Injury. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (21)
                                     , 12853-12872. <a href="https://doi.org/10.1021/acs.jmedchem.0c01236" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c01236</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c01236&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c01236%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DA%252BPeptidyl%252BInhibitor%252Bthat%252BBlocks%252BCalcineurin%2525E2%252580%252593NFAT%252BInteraction%252Band%252BPrevents%252BAcute%252BLung%252BInjury%26aulast%3DDougherty%26aufirst%3DPatrick%2BG.%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D16072020%26date%3D19102020%26volume%3D63%26issue%3D21%26spage%3D12853%26epage%3D12872" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Matthew P. Baumgartner, <span class="NLM_string-name hlFld-ContribAuthor">Hongzhou Zhang</span>. </span><span class="cited-content_cbyCitation_article-title">Building Admiral, an Automated Molecular Dynamics and Analysis Platform. </span><span class="cited-content_cbyCitation_journal-name">ACS Medicinal Chemistry Letters</span><span> <strong>2020,</strong> <em>11 </em>
                                    (11)
                                     , 2331-2335. <a href="https://doi.org/10.1021/acsmedchemlett.0c00458" title="DOI URL">https://doi.org/10.1021/acsmedchemlett.0c00458</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsmedchemlett.0c00458&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsmedchemlett.0c00458%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DBuilding%252BAdmiral%25252C%252Ban%252BAutomated%252BMolecular%252BDynamics%252Band%252BAnalysis%252BPlatform%26aulast%3DBaumgartner%26aufirst%3DMatthew%2BP.%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D19082020%26date%3D28092020%26date%3D09102020%26date%3D28092020%26volume%3D11%26issue%3D11%26spage%3D2331%26epage%3D2335" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Xinrui Yuan, Hong Bu, Jinpei Zhou, Chao-Yie Yang, <span class="NLM_string-name hlFld-ContribAuthor">Huibin Zhang</span>. </span><span class="cited-content_cbyCitation_article-title">Recent Advances of SHP2 Inhibitors in Cancer Therapy: Current Development and Clinical Application. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (20)
                                     , 11368-11396. <a href="https://doi.org/10.1021/acs.jmedchem.0c00249" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c00249</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c00249&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c00249%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DRecent%252BAdvances%252Bof%252BSHP2%252BInhibitors%252Bin%252BCancer%252BTherapy%25253A%252BCurrent%252BDevelopment%252Band%252BClinical%252BApplication%26aulast%3DYuan%26aufirst%3DXinrui%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D10022020%26date%3D10062020%26date%3D27052020%26volume%3D63%26issue%3D20%26spage%3D11368%26epage%3D11396" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Atharva S. Kelkar, Bradley C. Dallin, <span class="NLM_string-name hlFld-ContribAuthor">Reid C. Van Lehn</span>. </span><span class="cited-content_cbyCitation_article-title">Predicting Hydrophobicity by Learning Spatiotemporal Features of Interfacial Water Structure: Combining Molecular Dynamics Simulations with Convolutional Neural Networks. </span><span class="cited-content_cbyCitation_journal-name">The Journal of Physical Chemistry B</span><span> <strong>2020,</strong> <em>124 </em>
                                    (41)
                                     , 9103-9114. <a href="https://doi.org/10.1021/acs.jpcb.0c05977" title="DOI URL">https://doi.org/10.1021/acs.jpcb.0c05977</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jpcb.0c05977&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jpcb.0c05977%26sid%3Dliteratum%253Aachs%26jtitle%3DThe%2520Journal%2520of%2520Physical%2520Chemistry%2520B%26atitle%3DPredicting%252BHydrophobicity%252Bby%252BLearning%252BSpatiotemporal%252BFeatures%252Bof%252BInterfacial%252BWater%252BStructure%25253A%252BCombining%252BMolecular%252BDynamics%252BSimulations%252Bwith%252BConvolutional%252BNeural%252BNetworks%26aulast%3DKelkar%26aufirst%3DAtharva%2BS.%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D30062020%26date%3D22092020%26date%3D02102020%26date%3D23092020%26volume%3D124%26issue%3D41%26spage%3D9103%26epage%3D9114" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Mahendrarajan Venkatramanan, Pitchaipillai Sankar Ganesh, Renganathan Senthil, Jeyachandran Akshay, Arumugam Veera Ravi, Kulanthaivel Langeswaran, Jamuna Vadivelu, Samuthira Nagarajan, Kaliaperumal Rajendran, <span class="NLM_string-name hlFld-ContribAuthor">Esaki Muthu Shankar</span>. </span><span class="cited-content_cbyCitation_article-title">Inhibition of Quorum Sensing and Biofilm Formation in Chromobacterium violaceum by Fruit Extracts of Passiflora edulis. </span><span class="cited-content_cbyCitation_journal-name">ACS Omega</span><span> <strong>2020,</strong> <em>5 </em>
                                    (40)
                                     , 25605-25616. <a href="https://doi.org/10.1021/acsomega.0c02483" title="DOI URL">https://doi.org/10.1021/acsomega.0c02483</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsomega.0c02483&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsomega.0c02483%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Omega%26atitle%3DInhibition%252Bof%252BQuorum%252BSensing%252Band%252BBiofilm%252BFormation%252Bin%252BChromobacterium%252Bviolaceum%252Bby%252BFruit%252BExtracts%252Bof%252BPassiflora%252Bedulis%26aulast%3DVenkatramanan%26aufirst%3DMahendrarajan%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D27052020%26date%3D12082020%26date%3D29092020%26volume%3D5%26issue%3D40%26spage%3D25605%26epage%3D25616" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Katsuhiko Mitachi, Rita G. Kansal, Kirk E. Hevener, Cody D. Gillman, Syed M. Hussain, Hyun Gi Yun, Gustavo A. Miranda-Carboni, Evan S. Glazer, William M. Clemons, Jr., <span class="NLM_string-name hlFld-ContribAuthor">Michio Kurosu</span>. </span><span class="cited-content_cbyCitation_article-title">DPAGT1 Inhibitors of Capuramycin Analogues and Their Antimigratory Activities of Solid Tumors. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (19)
                                     , 10855-10878. <a href="https://doi.org/10.1021/acs.jmedchem.0c00545" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c00545</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c00545&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c00545%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDPAGT1%252BInhibitors%252Bof%252BCapuramycin%252BAnalogues%252Band%252BTheir%252BAntimigratory%252BActivities%252Bof%252BSolid%252BTumors%26aulast%3DMitachi%26aufirst%3DKatsuhiko%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D16042020%26date%3D18092020%26date%3D04092020%26volume%3D63%26issue%3D19%26spage%3D10855%26epage%3D10878" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Hao Liang, Hongbo Liu, Yi Kuang, Limin Chen, Min Ye, <span class="NLM_string-name hlFld-ContribAuthor">Luhua Lai</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of Targeted Covalent Natural Products against PLK1 by Herb-Based Screening. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2020,</strong> <em>60 </em>
                                    (9)
                                     , 4350-4358. <a href="https://doi.org/10.1021/acs.jcim.0c00074" title="DOI URL">https://doi.org/10.1021/acs.jcim.0c00074</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.0c00074&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.0c00074%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DDiscovery%252Bof%252BTargeted%252BCovalent%252BNatural%252BProducts%252Bagainst%252BPLK1%252Bby%252BHerb-Based%252BScreening%26aulast%3DLiang%26aufirst%3DHao%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D22012020%26date%3D28052020%26date%3D14052020%26volume%3D60%26issue%3D9%26spage%3D4350%26epage%3D4358" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Kara J. Cutrona, Ana S. Newton, Stefan G. Krimmer, Julian Tirado-Rives, <span class="NLM_string-name hlFld-ContribAuthor">William L. Jorgensen</span>. </span><span class="cited-content_cbyCitation_article-title">Metadynamics as a Postprocessing Method for Virtual Screening with Application to the Pseudokinase Domain of JAK2. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2020,</strong> <em>60 </em>
                                    (9)
                                     , 4403-4415. <a href="https://doi.org/10.1021/acs.jcim.0c00276" title="DOI URL">https://doi.org/10.1021/acs.jcim.0c00276</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.0c00276&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.0c00276%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DMetadynamics%252Bas%252Ba%252BPostprocessing%252BMethod%252Bfor%252BVirtual%252BScreening%252Bwith%252BApplication%252Bto%252Bthe%252BPseudokinase%252BDomain%252Bof%252BJAK2%26aulast%3DCutrona%26aufirst%3DKara%2BJ.%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D18032020%26date%3D27052020%26date%3D08052020%26volume%3D60%26issue%3D9%26spage%3D4403%26epage%3D4415" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Kasi Viswanatharaju Ruddraraju, Devesh Aggarwal, Congwei Niu, Erica Anne Baker, Ruo-yu Zhang, Li Wu, <span class="NLM_string-name hlFld-ContribAuthor">Zhong-Yin Zhang</span>. </span><span class="cited-content_cbyCitation_article-title">Highly Potent and Selective N-Aryl Oxamic Acid-Based Inhibitors for Mycobacterium tuberculosis Protein Tyrosine Phosphatase B. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (17)
                                     , 9212-9227. <a href="https://doi.org/10.1021/acs.jmedchem.0c00302" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c00302</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c00302&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c00302%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DHighly%252BPotent%252Band%252BSelective%252BN-Aryl%252BOxamic%252BAcid-Based%252BInhibitors%252Bfor%252BMycobacterium%252Btuberculosis%252BProtein%252BTyrosine%252BPhosphatase%252BB%26aulast%3DRuddraraju%26aufirst%3DKasi%2BViswanatharaju%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D19022020%26date%3D21082020%26date%3D05082020%26volume%3D63%26issue%3D17%26spage%3D9212%26epage%3D9227" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Christoph Grebner, Hans Matter, Alleyn T. Plowright, <span class="NLM_string-name hlFld-ContribAuthor">Gerhard Hessler</span>. </span><span class="cited-content_cbyCitation_article-title">Automated De Novo Design in Medicinal Chemistry: Which Types of Chemistry Does a Generative Neural Network Learn?. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (16)
                                     , 8809-8823. <a href="https://doi.org/10.1021/acs.jmedchem.9b02044" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b02044</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b02044&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.9b02044%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DAutomated%252BDe%252BNovo%252BDesign%252Bin%252BMedicinal%252BChemistry%25253A%252BWhich%252BTypes%252Bof%252BChemistry%252BDoes%252Ba%252BGenerative%252BNeural%252BNetwork%252BLearn%25253F%26aulast%3DGrebner%26aufirst%3DChristoph%26date%3D2020%26date%3D2020%26date%3D2019%26date%3D10122019%26date%3D20032020%26date%3D05032020%26volume%3D63%26issue%3D16%26spage%3D8809%26epage%3D8823" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Irina V. Gushchina, Aleksandra M. Polenova, Dmitry A. Suplatov, Vytas K. Švedas, <span class="NLM_string-name hlFld-ContribAuthor">Dmitry K. Nilov</span>. </span><span class="cited-content_cbyCitation_article-title">vsFilt: A Tool to Improve Virtual Screening by Structural Filtration of Docking Poses. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2020,</strong> <em>60 </em>
                                    (8)
                                     , 3692-3696. <a href="https://doi.org/10.1021/acs.jcim.0c00303" title="DOI URL">https://doi.org/10.1021/acs.jcim.0c00303</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.0c00303&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.0c00303%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DvsFilt%25253A%252BA%252BTool%252Bto%252BImprove%252BVirtual%252BScreening%252Bby%252BStructural%252BFiltration%252Bof%252BDocking%252BPoses%26aulast%3DGushchina%26aufirst%3DIrina%2BV.%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D26032020%26date%3D11082020%26volume%3D60%26issue%3D8%26spage%3D3692%26epage%3D3696" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Mariia Radaeva, Xuesen Dong, <span class="NLM_string-name hlFld-ContribAuthor">Artem Cherkasov</span>. </span><span class="cited-content_cbyCitation_article-title">The Use of Methods of Computer-Aided Drug Discovery in the Development of Topoisomerase II Inhibitors: Applications and Future Directions. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2020,</strong> <em>60 </em>
                                    (8)
                                     , 3703-3721. <a href="https://doi.org/10.1021/acs.jcim.0c00325" title="DOI URL">https://doi.org/10.1021/acs.jcim.0c00325</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.0c00325&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.0c00325%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DThe%252BUse%252Bof%252BMethods%252Bof%252BComputer-Aided%252BDrug%252BDiscovery%252Bin%252Bthe%252BDevelopment%252Bof%252BTopoisomerase%252BII%252BInhibitors%25253A%252BApplications%252Band%252BFuture%252BDirections%26aulast%3DRadaeva%26aufirst%3DMariia%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D03042020%26date%3D03082020%26date%3D20072020%26volume%3D60%26issue%3D8%26spage%3D3703%26epage%3D3721" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Muhammad Murtaza Hassan, Johan Israelian, Nabanita Nawar, Giovanni Ganda, Pimyupa Manaswiyoungkul, Yasir S. Raouf, David Armstrong, Abootaleb Sedighi, Olasunkanmi O. Olaoye, Fettah Erdogan, Aaron D. Cabral, Fabrizio Angeles, Rabia Altintas, Elvin D. de Araujo, <span class="NLM_string-name hlFld-ContribAuthor">Patrick T. Gunning</span>. </span><span class="cited-content_cbyCitation_article-title">Characterization of Conformationally Constrained Benzanilide Scaffolds for Potent and Selective HDAC8 Targeting. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (15)
                                     , 8634-8648. <a href="https://doi.org/10.1021/acs.jmedchem.0c01025" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c01025</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c01025&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c01025%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DCharacterization%252Bof%252BConformationally%252BConstrained%252BBenzanilide%252BScaffolds%252Bfor%252BPotent%252Band%252BSelective%252BHDAC8%252BTargeting%26aulast%3DHassan%26aufirst%3DMuhammad%2BMurtaza%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D15062020%26date%3D31072020%26date%3D16072020%26volume%3D63%26issue%3D15%26spage%3D8634%26epage%3D8648" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Daniel A. Greenfield, Hayden R. Schmidt, Meredith A. Skiba, Michael D. Mandler, Jacob R. Anderson, Piotr Sliz, <span class="NLM_string-name hlFld-ContribAuthor">Andrew C. Kruse</span>. </span><span class="cited-content_cbyCitation_article-title">Virtual Screening for Ligand Discovery at the σ1 Receptor. </span><span class="cited-content_cbyCitation_journal-name">ACS Medicinal Chemistry Letters</span><span> <strong>2020,</strong> <em>11 </em>
                                    (8)
                                     , 1555-1561. <a href="https://doi.org/10.1021/acsmedchemlett.9b00314" title="DOI URL">https://doi.org/10.1021/acsmedchemlett.9b00314</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsmedchemlett.9b00314&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsmedchemlett.9b00314%26sid%3Dliteratum%253Aachs%26jtitle%3DJ.%2520Pharmacol.%2520Sci.%26date%3D2015%26volume%3D127%26issue%3D1%26spage%3D17%26epage%3D29%26jtitle%3DBehav.%2520Pharmacol.%26date%3D2016%26volume%3D27%26issue%3D2%25E2%2580%25933%2520Spec%2520Issue%26spage%3D100%26epage%3D15%26jtitle%3DHandb.%2520Exp.%2520Pharmacol.%26date%3D2017%26volume%3D244%26spage%3D237%26epage%3D308%26jtitle%3DNeural%2520Regener.%2520Res.%26date%3D2018%26volume%3D13%26issue%3D5%26spage%3D775%26epage%3D778%26jtitle%3DACS%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DVirtual%252BScreening%252Bfor%252BLigand%252BDiscovery%252Bat%252Bthe%252B%2525CF%2525831%252BReceptor%26aulast%3DGreenfield%26aufirst%3DDaniel%2BA.%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D20042020%26date%3D27072020%26date%3D31072020%26date%3D27072020%26volume%3D11%26issue%3D8%26spage%3D1555%26epage%3D1561" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Philip Ryan, Mingming Xu, Kousar Jahan, Andrew K. Davey, Prasad V. Bharatam, Shailendra Anoopkumar-Dukie, Michael Kassiou, George D. Mellick, <span class="NLM_string-name hlFld-ContribAuthor">Santosh Rudrawar</span>. </span><span class="cited-content_cbyCitation_article-title">Novel Furan-2-yl-1H-pyrazoles Possess Inhibitory Activity against α-Synuclein Aggregation. </span><span class="cited-content_cbyCitation_journal-name">ACS Chemical Neuroscience</span><span> <strong>2020,</strong> <em>11 </em>
                                    (15)
                                     , 2303-2315. <a href="https://doi.org/10.1021/acschemneuro.0c00252" title="DOI URL">https://doi.org/10.1021/acschemneuro.0c00252</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acschemneuro.0c00252&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facschemneuro.0c00252%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Chemical%2520Neuroscience%26atitle%3DNovel%252BFuran-2-yl-1H-pyrazoles%252BPossess%252BInhibitory%252BActivity%252Bagainst%252B%2525CE%2525B1-Synuclein%252BAggregation%26aulast%3DRyan%26aufirst%3DPhilip%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D28042020%26date%3D18062020%26date%3D08072020%26date%3D18062020%26volume%3D11%26issue%3D15%26spage%3D2303%26epage%3D2315" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Leila Tabrizi, Kerry Thompson, Katarzyna Mnich, Chetan Chintha, Adrienne M. Gorman, Liam Morrison, Janna Luessing, Noel F. Lowndes, Peter Dockery, Afshin Samali, <span class="NLM_string-name hlFld-ContribAuthor">Andrea Erxleben</span>. </span><span class="cited-content_cbyCitation_article-title">Novel Pt(IV) Prodrugs Displaying Antimitochondrial Effects. </span><span class="cited-content_cbyCitation_journal-name">Molecular Pharmaceutics</span><span> <strong>2020,</strong> <em>17 </em>
                                    (8)
                                     , 3009-3023. <a href="https://doi.org/10.1021/acs.molpharmaceut.0c00417" title="DOI URL">https://doi.org/10.1021/acs.molpharmaceut.0c00417</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.molpharmaceut.0c00417&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.molpharmaceut.0c00417%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecular%2520Pharmaceutics%26atitle%3DNovel%252BPt%252528IV%252529%252BProdrugs%252BDisplaying%252BAntimitochondrial%252BEffects%26aulast%3DTabrizi%26aufirst%3DLeila%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D18042020%26date%3D06072020%26date%3D01072020%26date%3D15072020%26date%3D06072020%26volume%3D17%26issue%3D8%26spage%3D3009%26epage%3D3023" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Phyo Phyo Kyaw Zin, Alexandre Borrel, <span class="NLM_string-name hlFld-ContribAuthor">Denis Fourches</span>. </span><span class="cited-content_cbyCitation_article-title">Benchmarking 2D/3D/MD-QSAR Models for Imatinib Derivatives: How Far Can We Predict?. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2020,</strong> <em>60 </em>
                                    (7)
                                     , 3342-3360. <a href="https://doi.org/10.1021/acs.jcim.0c00200" title="DOI URL">https://doi.org/10.1021/acs.jcim.0c00200</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.0c00200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.0c00200%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DBenchmarking%252B2D%25252F3D%25252FMD-QSAR%252BModels%252Bfor%252BImatinib%252BDerivatives%25253A%252BHow%252BFar%252BCan%252BWe%252BPredict%25253F%26aulast%3DKyaw%2BZin%26aufirst%3DPhyo%2BPhyo%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D27022020%26date%3D05072020%26volume%3D60%26issue%3D7%26spage%3D3342%26epage%3D3360" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">William H. Coldren, Svetlana B. Tikunova, Jonathan P. Davis, <span class="NLM_string-name hlFld-ContribAuthor">Steffen Lindert</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of Novel Small-Molecule Calcium Sensitizers for Cardiac Troponin C: A Combined Virtual and Experimental Screening Approach. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2020,</strong> <em>60 </em>
                                    (7)
                                     , 3648-3661. <a href="https://doi.org/10.1021/acs.jcim.0c00452" title="DOI URL">https://doi.org/10.1021/acs.jcim.0c00452</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.0c00452&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.0c00452%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DDiscovery%252Bof%252BNovel%252BSmall-Molecule%252BCalcium%252BSensitizers%252Bfor%252BCardiac%252BTroponin%252BC%25253A%252BA%252BCombined%252BVirtual%252Band%252BExperimental%252BScreening%252BApproach%26aulast%3DColdren%26aufirst%3DWilliam%2BH.%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D29042020%26date%3D07072020%26volume%3D60%26issue%3D7%26spage%3D3648%26epage%3D3661" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Bishnu Thapa, Jon Erickson, <span class="NLM_string-name hlFld-ContribAuthor">Krishnan Raghavachari</span>. </span><span class="cited-content_cbyCitation_article-title">Quantum Mechanical Investigation of Three-Dimensional Activity Cliffs Using the Molecules-in-Molecules Fragmentation-Based Method. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2020,</strong> <em>60 </em>
                                    (6)
                                     , 2924-2938. <a href="https://doi.org/10.1021/acs.jcim.9b01123" title="DOI URL">https://doi.org/10.1021/acs.jcim.9b01123</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.9b01123&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.9b01123%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DQuantum%252BMechanical%252BInvestigation%252Bof%252BThree-Dimensional%252BActivity%252BCliffs%252BUsing%252Bthe%252BMolecules-in-Molecules%252BFragmentation-Based%252BMethod%26aulast%3DThapa%26aufirst%3DBishnu%26date%3D2020%26date%3D2020%26date%3D2019%26date%3D03122019%26date%3D14052020%26volume%3D60%26issue%3D6%26spage%3D2924%26epage%3D2938" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Verrill M. Norwood, IV, Ariana C. Brice-Tutt, Shainnel O. Eans, Heather M. Stacy, Guqin Shi, Ranjala Ratnayake, James R. Rocca, Khalil A. Abboud, Chenglong Li, Hendrik Luesch, Jay P. McLaughlin, <span class="NLM_string-name hlFld-ContribAuthor">Robert W. Huigens, III</span>. </span><span class="cited-content_cbyCitation_article-title">Preventing Morphine-Seeking Behavior through the Re-Engineering of Vincamine’s Biological Activity. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (10)
                                     , 5119-5138. <a href="https://doi.org/10.1021/acs.jmedchem.9b01924" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b01924</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b01924&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.9b01924%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DPreventing%252BMorphine-Seeking%252BBehavior%252Bthrough%252Bthe%252BRe-Engineering%252Bof%252BVincamine%2525E2%252580%252599s%252BBiological%252BActivity%26aulast%3DNorwood%26aufirst%3DVerrill%2BM.%26date%3D2020%26date%3D2020%26date%3D2019%26date%3D26112019%26date%3D23032020%26date%3D08012020%26volume%3D63%26issue%3D10%26spage%3D5119%26epage%3D5138" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">John W. Vant, Shae-Lynn J. Lahey, Kalyanashis Jana, Mrinal Shekhar, Daipayan Sarkar, Barbara H. Munk, Ulrich Kleinekathöfer, Sumit Mittal, Christopher Rowley, <span class="NLM_string-name hlFld-ContribAuthor">Abhishek Singharoy</span>. </span><span class="cited-content_cbyCitation_article-title">Flexible Fitting of Small Molecules into Electron Microscopy Maps Using Molecular Dynamics Simulations with Neural Network Potentials. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2020,</strong> <em>60 </em>
                                    (5)
                                     , 2591-2604. <a href="https://doi.org/10.1021/acs.jcim.9b01167" title="DOI URL">https://doi.org/10.1021/acs.jcim.9b01167</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.9b01167&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.9b01167%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DFlexible%252BFitting%252Bof%252BSmall%252BMolecules%252Binto%252BElectron%252BMicroscopy%252BMaps%252BUsing%252BMolecular%252BDynamics%252BSimulations%252Bwith%252BNeural%252BNetwork%252BPotentials%26aulast%3DVant%26aufirst%3DJohn%2BW.%26date%3D2020%26date%3D2020%26date%3D2019%26date%3D17122019%26date%3D30032020%26date%3D24032020%26volume%3D60%26issue%3D5%26spage%3D2591%26epage%3D2604" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Mohamad Sabbah, Vitor Mendes, Robert G. Vistal, David M. G. Dias, Monika Záhorszká, Katarína Mikušová, Jana Korduláková, Anthony G. Coyne, Tom L. Blundell, <span class="NLM_string-name hlFld-ContribAuthor">Chris Abell</span>. </span><span class="cited-content_cbyCitation_article-title">Fragment-Based Design of Mycobacterium tuberculosis InhA Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (9)
                                     , 4749-4761. <a href="https://doi.org/10.1021/acs.jmedchem.0c00007" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c00007</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c00007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c00007%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DFragment-Based%252BDesign%252Bof%252BMycobacterium%252Btuberculosis%252BInhA%252BInhibitors%26aulast%3DSabbah%26aufirst%3DMohamad%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D02012020%26date%3D15042020%26date%3D02042020%26volume%3D63%26issue%3D9%26spage%3D4749%26epage%3D4761" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Ana S. Newton, John C. Faver, Goran Micevic, Viswanathan Muthusamy, Shalley N. Kudalkar, Nicole Bertoletti, Karen S. Anderson, Marcus W. Bosenberg, <span class="NLM_string-name hlFld-ContribAuthor">William L. Jorgensen</span>. </span><span class="cited-content_cbyCitation_article-title">Structure-Guided Identification of DNMT3B Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">ACS Medicinal Chemistry Letters</span><span> <strong>2020,</strong> <em>11 </em>
                                    (5)
                                     , 971-976. <a href="https://doi.org/10.1021/acsmedchemlett.0c00011" title="DOI URL">https://doi.org/10.1021/acsmedchemlett.0c00011</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsmedchemlett.0c00011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsmedchemlett.0c00011%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DStructure-Guided%252BIdentification%252Bof%252BDNMT3B%252BInhibitors%26aulast%3DNewton%26aufirst%3DAna%2BS.%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D08012020%26date%3D07022020%26date%3D11022020%26date%3D07022020%26volume%3D11%26issue%3D5%26spage%3D971%26epage%3D976" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Marco Catto, Leonardo Pisani, Eugenio de la Mora, Benny Danilo Belviso, Giuseppe Felice Mangiatordi, Andrea Pinto, Annalisa De Palma, Nunzio Denora, Rocco Caliandro, Jacques-Philippe Colletier, Israel Silman, Orazio Nicolotti, <span class="NLM_string-name hlFld-ContribAuthor">Cosimo Damiano Altomare</span>. </span><span class="cited-content_cbyCitation_article-title">Chiral Separation, X-ray Structure, and Biological Evaluation of a Potent and Reversible Dual Binding Site AChE Inhibitor. </span><span class="cited-content_cbyCitation_journal-name">ACS Medicinal Chemistry Letters</span><span> <strong>2020,</strong> <em>11 </em>
                                    (5)
                                     , 869-876. <a href="https://doi.org/10.1021/acsmedchemlett.9b00656" title="DOI URL">https://doi.org/10.1021/acsmedchemlett.9b00656</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsmedchemlett.9b00656&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsmedchemlett.9b00656%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DChiral%252BSeparation%25252C%252BX-ray%252BStructure%25252C%252Band%252BBiological%252BEvaluation%252Bof%252Ba%252BPotent%252Band%252BReversible%252BDual%252BBinding%252BSite%252BAChE%252BInhibitor%26aulast%3DCatto%26aufirst%3DMarco%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D2019%26date%3D27122019%26date%3D07022020%26date%3D12022020%26date%3D07022020%26volume%3D11%26issue%3D5%26spage%3D869%26epage%3D876" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Jonathan L. Chen, Peiyuan Zhang, Masahito Abe, Haruo Aikawa, Liying Zhang, Alexander J. Frank, Timothy Zembryski, Christopher Hubbs, HaJeung Park, Jane Withka, Claire Steppan, Lucy Rogers, Shawn Cabral, Martin Pettersson, Travis T. Wager, Matthew A. Fountain, Gavin Rumbaugh, Jessica L. Childs-Disney, <span class="NLM_string-name hlFld-ContribAuthor">Matthew D. Disney</span>. </span><span class="cited-content_cbyCitation_article-title">Design, Optimization, and Study of Small Molecules That Target Tau Pre-mRNA and Affect Splicing. </span><span class="cited-content_cbyCitation_journal-name">Journal of the American Chemical Society</span><span> <strong>2020,</strong> <em>142 </em>
                                    (19)
                                     , 8706-8727. <a href="https://doi.org/10.1021/jacs.0c00768" title="DOI URL">https://doi.org/10.1021/jacs.0c00768</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/jacs.0c00768&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fjacs.0c00768%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520the%2520American%2520Chemical%2520Society%26atitle%3DDesign%25252C%252BOptimization%25252C%252Band%252BStudy%252Bof%252BSmall%252BMolecules%252BThat%252BTarget%252BTau%252BPre-mRNA%252Band%252BAffect%252BSplicing%26aulast%3DChen%26aufirst%3DJonathan%2BL.%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D20012020%26date%3D04052020%26volume%3D142%26issue%3D19%26spage%3D8706%26epage%3D8727" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Doris Mia Taylor, Justin Anglin, Suhyeorn Park, Melek N. Ucisik, John C. Faver, Nicholas Simmons, Zhuang Jin, Murugesan Palaniappan, Pranavanand Nyshadham, Feng Li, James Campbell, Liya Hu, Banumathi Sankaran, B. V. Venkataram Prasad, Hongbing Huang, Martin M. Matzuk, <span class="NLM_string-name hlFld-ContribAuthor">Timothy Palzkill</span>. </span><span class="cited-content_cbyCitation_article-title">Identifying Oxacillinase-48 Carbapenemase Inhibitors Using DNA-Encoded Chemical Libraries. </span><span class="cited-content_cbyCitation_journal-name">ACS Infectious Diseases</span><span> <strong>2020,</strong> <em>6 </em>
                                    (5)
                                     , 1214-1227. <a href="https://doi.org/10.1021/acsinfecdis.0c00015" title="DOI URL">https://doi.org/10.1021/acsinfecdis.0c00015</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsinfecdis.0c00015&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsinfecdis.0c00015%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Infectious%2520Diseases%26atitle%3DIdentifying%252BOxacillinase-48%252BCarbapenemase%252BInhibitors%252BUsing%252BDNA-Encoded%252BChemical%252BLibraries%26aulast%3DTaylor%26aufirst%3DDoris%2BMia%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D12012020%26date%3D25032020%26date%3D17032020%26volume%3D6%26issue%3D5%26spage%3D1214%26epage%3D1227" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Xiaolin Li, Yalong Cong, Mingzhe Ma, Zhi-Neng You, Bei Gao, John Z. H. Zhang, <span class="NLM_string-name hlFld-ContribAuthor">Lujia Zhang</span>. </span><span class="cited-content_cbyCitation_article-title">An Energy Optimization Strategy Based on the Perfect Conformation of Prolyl Endopeptidase for Improving Catalytic Efficiency. </span><span class="cited-content_cbyCitation_journal-name">Journal of Agricultural and Food Chemistry</span><span> <strong>2020,</strong> <em>68 </em>
                                    (18)
                                     , 5129-5137. <a href="https://doi.org/10.1021/acs.jafc.0c00731" title="DOI URL">https://doi.org/10.1021/acs.jafc.0c00731</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jafc.0c00731&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jafc.0c00731%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Agricultural%2520and%2520Food%2520Chemistry%26atitle%3DAn%252BEnergy%252BOptimization%252BStrategy%252BBased%252Bon%252Bthe%252BPerfect%252BConformation%252Bof%252BProlyl%252BEndopeptidase%252Bfor%252BImproving%252BCatalytic%252BEfficiency%26aulast%3DLi%26aufirst%3DXiaolin%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D02022020%26date%3D16042020%26date%3D15042020%26date%3D27042020%26date%3D16042020%26volume%3D68%26issue%3D18%26spage%3D5129%26epage%3D5137" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Peter R. Curran, Chris J. Radoux, Mihaela D. Smilova, Richard A. Sykes, Alicia P. Higueruelo, Anthony R. Bradley, Brian D. Marsden, David R. Spring, Tom L. Blundell, Andrew R. Leach, William R. Pitt, <span class="NLM_string-name hlFld-ContribAuthor">Jason C. Cole</span>. </span><span class="cited-content_cbyCitation_article-title">Hotspots API: A Python Package for the Detection of Small Molecule Binding Hotspots and Application to Structure-Based Drug Design. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2020,</strong> <em>60 </em>
                                    (4)
                                     , 1911-1916. <a href="https://doi.org/10.1021/acs.jcim.9b00996" title="DOI URL">https://doi.org/10.1021/acs.jcim.9b00996</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.9b00996&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.9b00996%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DHotspots%252BAPI%25253A%252BA%252BPython%252BPackage%252Bfor%252Bthe%252BDetection%252Bof%252BSmall%252BMolecule%252BBinding%252BHotspots%252Band%252BApplication%252Bto%252BStructure-Based%252BDrug%252BDesign%26aulast%3DCurran%26aufirst%3DPeter%2BR.%26date%3D2020%26date%3D2020%26date%3D2019%26date%3D28102019%26date%3D02042020%26date%3D24032020%26volume%3D60%26issue%3D4%26spage%3D1911%26epage%3D1916" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Jianjiao Wang, Zhi Liang, Kunlong Li, Bin Yang, Yonghong Liu, Wei Fang, Lan Tang, <span class="NLM_string-name hlFld-ContribAuthor">Xuefeng Zhou</span>. </span><span class="cited-content_cbyCitation_article-title">Ene-yne Hydroquinones from a Marine-derived Strain of the Fungus Pestalotiopsis neglecta with Effects on Liver X Receptor Alpha. </span><span class="cited-content_cbyCitation_journal-name">Journal of Natural Products</span><span> <strong>2020,</strong> <em>83 </em>
                                    (4)
                                     , 1258-1264. <a href="https://doi.org/10.1021/acs.jnatprod.0c00050" title="DOI URL">https://doi.org/10.1021/acs.jnatprod.0c00050</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jnatprod.0c00050&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jnatprod.0c00050%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Natural%2520Products%26atitle%3DEne-yne%252BHydroquinones%252Bfrom%252Ba%252BMarine-derived%252BStrain%252Bof%252Bthe%252BFungus%252BPestalotiopsis%252Bneglecta%252Bwith%252BEffects%252Bon%252BLiver%252BX%252BReceptor%252BAlpha%26aulast%3DWang%26aufirst%3DJianjiao%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D14012020%26date%3D13042020%26volume%3D83%26issue%3D4%26spage%3D1258%26epage%3D1264" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Yifan Fu, Xiaoyu Ding, Xianglei Zhang, Xingcheng Shao, Jihui Zhao, Yechun Xu, Xiaomin Luo, <span class="NLM_string-name hlFld-ContribAuthor">Weimin Zhao</span>. </span><span class="cited-content_cbyCitation_article-title">Diterpenoids from the Root Bark of Pinus massoniana and Evaluation of Their Phosphodiesterase Type 4D Inhibitory Activity. </span><span class="cited-content_cbyCitation_journal-name">Journal of Natural Products</span><span> <strong>2020,</strong> <em>83 </em>
                                    (4)
                                     , 1229-1237. <a href="https://doi.org/10.1021/acs.jnatprod.9b01269" title="DOI URL">https://doi.org/10.1021/acs.jnatprod.9b01269</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jnatprod.9b01269&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jnatprod.9b01269%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Natural%2520Products%26atitle%3DDiterpenoids%252Bfrom%252Bthe%252BRoot%252BBark%252Bof%252BPinus%252Bmassoniana%252Band%252BEvaluation%252Bof%252BTheir%252BPhosphodiesterase%252BType%252B4D%252BInhibitory%252BActivity%26aulast%3DFu%26aufirst%3DYifan%26date%3D2020%26date%3D2020%26date%3D2019%26date%3D26122019%26date%3D26022020%26volume%3D83%26issue%3D4%26spage%3D1229%26epage%3D1237" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Dilip K. Tosh, Veronica Salmaso, Harsha Rao, Amelia Bitant, Courtney L. Fisher, David I. Lieberman, Helmut Vorbrüggen, Marc L. Reitman, Oksana Gavrilova, Zhan-Guo Gao, John A. Auchampach, <span class="NLM_string-name hlFld-ContribAuthor">Kenneth A. Jacobson</span>. </span><span class="cited-content_cbyCitation_article-title">Truncated (N)-Methanocarba Nucleosides as Partial Agonists at Mouse and Human A3 Adenosine Receptors: Affinity Enhancement by N6-(2-Phenylethyl) Substitution. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (8)
                                     , 4334-4348. <a href="https://doi.org/10.1021/acs.jmedchem.0c00235" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c00235</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c00235&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c00235%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DTruncated%252B%252528N%252529-Methanocarba%252BNucleosides%252Bas%252BPartial%252BAgonists%252Bat%252BMouse%252Band%252BHuman%252BA3%252BAdenosine%252BReceptors%25253A%252BAffinity%252BEnhancement%252Bby%252BN6-%2525282-Phenylethyl%252529%252BSubstitution%26aulast%3DTosh%26aufirst%3DDilip%2BK.%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D09022020%26date%3D09042020%26volume%3D63%26issue%3D8%26spage%3D4334%26epage%3D4348" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Masatake Sugita, Masataka Hamano, Kota Kasahara, Takeshi Kikuchi, <span class="NLM_string-name hlFld-ContribAuthor">Fumio Hirata</span>. </span><span class="cited-content_cbyCitation_article-title">New Protocol for Predicting the Ligand-Binding Site and Mode Based on the 3D-RISM/KH Theory. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Theory and Computation</span><span> <strong>2020,</strong> <em>16 </em>
                                    (4)
                                     , 2864-2876. <a href="https://doi.org/10.1021/acs.jctc.9b01069" title="DOI URL">https://doi.org/10.1021/acs.jctc.9b01069</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jctc.9b01069&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jctc.9b01069%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Theory%2520and%2520Computation%26atitle%3DNew%252BProtocol%252Bfor%252BPredicting%252Bthe%252BLigand-Binding%252BSite%252Band%252BMode%252BBased%252Bon%252Bthe%252B3D-RISM%25252FKH%252BTheory%26aulast%3DSugita%26aufirst%3DMasatake%26date%3D2020%26date%3D2020%26date%3D2019%26date%3D28102019%26date%3D27032020%26date%3D16032020%26volume%3D16%26issue%3D4%26spage%3D2864%26epage%3D2876" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Jeffrey Cruz, Lauren Wickstrom, Danzhou Yang, Emilio Gallicchio, <span class="NLM_string-name hlFld-ContribAuthor">Nanjie Deng</span>. </span><span class="cited-content_cbyCitation_article-title">Combining Alchemical Transformation with a Physical Pathway to Accelerate Absolute Binding Free Energy Calculations of Charged Ligands to Enclosed Binding Sites. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Theory and Computation</span><span> <strong>2020,</strong> <em>16 </em>
                                    (4)
                                     , 2803-2813. <a href="https://doi.org/10.1021/acs.jctc.9b01119" title="DOI URL">https://doi.org/10.1021/acs.jctc.9b01119</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jctc.9b01119&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jctc.9b01119%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Theory%2520and%2520Computation%26atitle%3DCombining%252BAlchemical%252BTransformation%252Bwith%252Ba%252BPhysical%252BPathway%252Bto%252BAccelerate%252BAbsolute%252BBinding%252BFree%252BEnergy%252BCalculations%252Bof%252BCharged%252BLigands%252Bto%252BEnclosed%252BBinding%252BSites%26aulast%3DCruz%26aufirst%3DJeffrey%26date%3D2020%26date%3D2020%26date%3D2019%26date%3D10112019%26date%3D09032020%26date%3D26022020%26volume%3D16%26issue%3D4%26spage%3D2803%26epage%3D2813" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Xiaozheng Dou, Dinesh Nath, Henry Shin, Elmar Nurmemmedov, Philip C. Bourne, Jian-Xing Ma, <span class="NLM_string-name hlFld-ContribAuthor">Adam S. Duerfeldt</span>. </span><span class="cited-content_cbyCitation_article-title">Evolution of a 4-Benzyloxy-benzylamino Chemotype to Provide Efficacious, Potent, and Isoform Selective PPARα Agonists as Leads for Retinal Disorders. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (6)
                                     , 2854-2876. <a href="https://doi.org/10.1021/acs.jmedchem.9b01189" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b01189</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b01189&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.9b01189%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DEvolution%252Bof%252Ba%252B4-Benzyloxy-benzylamino%252BChemotype%252Bto%252BProvide%252BEfficacious%25252C%252BPotent%25252C%252Band%252BIsoform%252BSelective%252BPPAR%2525CE%2525B1%252BAgonists%252Bas%252BLeads%252Bfor%252BRetinal%252BDisorders%26aulast%3DDou%26aufirst%3DXiaozheng%26date%3D2020%26date%3D2020%26date%3D2019%26date%3D31072019%26date%3D10032020%26date%3D25022020%26volume%3D63%26issue%3D6%26spage%3D2854%26epage%3D2876" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Kunal Kumar, Peng Wang, Jessica Wilson, Viktor Zlatanic, Cecilia Berrouet, Susmita Khamrui, Cody Secor, Ethan A. Swartz, Michael Lazarus, Roberto Sanchez, Andrew F. Stewart, Adolfo Garcia-Ocana, <span class="NLM_string-name hlFld-ContribAuthor">Robert J. DeVita</span>. </span><span class="cited-content_cbyCitation_article-title">Synthesis and Biological Validation of a Harmine-Based, Central Nervous System (CNS)-Avoidant, Selective, Human β-Cell Regenerative Dual-Specificity Tyrosine Phosphorylation-Regulated Kinase A (DYRK1A) Inhibitor. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (6)
                                     , 2986-3003. <a href="https://doi.org/10.1021/acs.jmedchem.9b01379" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b01379</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b01379&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.9b01379%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DSynthesis%252Band%252BBiological%252BValidation%252Bof%252Ba%252BHarmine-Based%25252C%252BCentral%252BNervous%252BSystem%252B%252528CNS%252529-Avoidant%25252C%252BSelective%25252C%252BHuman%252B%2525CE%2525B2-Cell%252BRegenerative%252BDual-Specificity%252BTyrosine%252BPhosphorylation-Regulated%252BKinase%252BA%252B%252528DYRK1A%252529%252BInhibitor%26aulast%3DKumar%26aufirst%3DKunal%26date%3D2020%26date%3D2020%26date%3D2019%26date%3D03102019%26date%3D19022020%26date%3D31012020%26volume%3D63%26issue%3D6%26spage%3D2986%26epage%3D3003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Bala C. Chenna, Linfeng Li, Drake M. Mellott, Xiang Zhai, Jair L. Siqueira-Neto, Claudia Calvet Alvarez, Jean A. Bernatchez, Emily Desormeaux, Elizabeth Alvarez Hernandez, Jana Gomez, James H. McKerrow, Jorge Cruz-Reyes, <span class="NLM_string-name hlFld-ContribAuthor">Thomas D. Meek</span>. </span><span class="cited-content_cbyCitation_article-title">Peptidomimetic Vinyl Heterocyclic Inhibitors of Cruzain Effect Antitrypanosomal Activity. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (6)
                                     , 3298-3316. <a href="https://doi.org/10.1021/acs.jmedchem.9b02078" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b02078</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b02078&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.9b02078%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DPeptidomimetic%252BVinyl%252BHeterocyclic%252BInhibitors%252Bof%252BCruzain%252BEffect%252BAntitrypanosomal%252BActivity%26aulast%3DChenna%26aufirst%3DBala%2BC.%26date%3D2020%26date%3D2020%26date%3D2019%26date%3D13122019%26date%3D17032020%26date%3D03032020%26volume%3D63%26issue%3D6%26spage%3D3298%26epage%3D3316" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Xiaozheng Zhang, Yibo Wang, Hongshuang Wang, Hongyuan Li, Tianshu Zhang, Yinghua Peng, <span class="NLM_string-name hlFld-ContribAuthor">Xiaohui Wang</span>. </span><span class="cited-content_cbyCitation_article-title">Exploring Methamphetamine Nonenantioselectively Targeting Toll-like Receptor 4/Myeloid Differentiation Protein 2 by in Silico Simulations and Wet-Lab Techniques. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2020,</strong> <em>60 </em>
                                    (3)
                                     , 1607-1613. <a href="https://doi.org/10.1021/acs.jcim.9b01040" title="DOI URL">https://doi.org/10.1021/acs.jcim.9b01040</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.9b01040&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.9b01040%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DExploring%252BMethamphetamine%252BNonenantioselectively%252BTargeting%252BToll-like%252BReceptor%252B4%25252FMyeloid%252BDifferentiation%252BProtein%252B2%252Bby%252Bin%252BSilico%252BSimulations%252Band%252BWet-Lab%252BTechniques%26aulast%3DZhang%26aufirst%3DXiaozheng%26date%3D2020%26date%3D2020%26date%3D2019%26date%3D08112019%26date%3D23012020%26date%3D14012020%26volume%3D60%26issue%3D3%26spage%3D1607%26epage%3D1613" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Maciej Majewski, <span class="NLM_string-name hlFld-ContribAuthor">Xavier Barril</span>. </span><span class="cited-content_cbyCitation_article-title">Structural Stability Predicts the Binding Mode of Protein–Ligand Complexes. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2020,</strong> <em>60 </em>
                                    (3)
                                     , 1644-1651. <a href="https://doi.org/10.1021/acs.jcim.9b01062" title="DOI URL">https://doi.org/10.1021/acs.jcim.9b01062</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.9b01062&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.9b01062%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DStructural%252BStability%252BPredicts%252Bthe%252BBinding%252BMode%252Bof%252BProtein%2525E2%252580%252593Ligand%252BComplexes%26aulast%3DMajewski%26aufirst%3DMaciej%26date%3D2020%26date%3D2020%26date%3D2019%26date%3D14112019%26date%3D28022020%26date%3D13022020%26volume%3D60%26issue%3D3%26spage%3D1644%26epage%3D1651" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Daniel Cappel, Steven Jerome, Gerhard Hessler, <span class="NLM_string-name hlFld-ContribAuthor">Hans Matter</span>. </span><span class="cited-content_cbyCitation_article-title">Impact of Different Automated Binding Pose Generation Approaches on Relative Binding Free Energy Simulations. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2020,</strong> <em>60 </em>
                                    (3)
                                     , 1432-1444. <a href="https://doi.org/10.1021/acs.jcim.9b01118" title="DOI URL">https://doi.org/10.1021/acs.jcim.9b01118</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.9b01118&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.9b01118%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DImpact%252Bof%252BDifferent%252BAutomated%252BBinding%252BPose%252BGeneration%252BApproaches%252Bon%252BRelative%252BBinding%252BFree%252BEnergy%252BSimulations%26aulast%3DCappel%26aufirst%3DDaniel%26date%3D2020%26date%3D2020%26date%3D2019%26date%3D03122019%26date%3D04022020%26date%3D28012020%26volume%3D60%26issue%3D3%26spage%3D1432%26epage%3D1444" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Rick C. Betori, Yue Liu, Rama K. Mishra, Scott B. Cohen, Stephen J. Kron, <span class="NLM_string-name hlFld-ContribAuthor">Karl A. Scheidt</span>. </span><span class="cited-content_cbyCitation_article-title">Targeted Covalent Inhibition of Telomerase. </span><span class="cited-content_cbyCitation_journal-name">ACS Chemical Biology</span><span> <strong>2020,</strong> <em>15 </em>
                                    (3)
                                     , 706-717. <a href="https://doi.org/10.1021/acschembio.9b00945" title="DOI URL">https://doi.org/10.1021/acschembio.9b00945</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acschembio.9b00945&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facschembio.9b00945%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Chemical%2520Biology%26atitle%3DTargeted%252BCovalent%252BInhibition%252Bof%252BTelomerase%26aulast%3DBetori%26aufirst%3DRick%2BC.%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D2019%26date%3D23112019%26date%3D04022020%26date%3D24022020%26date%3D04022020%26volume%3D15%26issue%3D3%26spage%3D706%26epage%3D717" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Azzurra Stefanucci, Marilisa Pia Dimmito, Giorgia Macedonio, Laura Ciarlo, Stefano Pieretti, Ettore Novellino, Wei Lei, Deborah Barlow, Karen L. Houseknecht, John M. Streicher, <span class="NLM_string-name hlFld-ContribAuthor">Adriano Mollica</span>. </span><span class="cited-content_cbyCitation_article-title">Potent, Efficacious, and Stable Cyclic Opioid Peptides with Long Lasting Antinociceptive Effect after Peripheral Administration. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (5)
                                     , 2673-2687. <a href="https://doi.org/10.1021/acs.jmedchem.9b01963" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b01963</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b01963&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.9b01963%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DPotent%25252C%252BEfficacious%25252C%252Band%252BStable%252BCyclic%252BOpioid%252BPeptides%252Bwith%252BLong%252BLasting%252BAntinociceptive%252BEffect%252Bafter%252BPeripheral%252BAdministration%26aulast%3DStefanucci%26aufirst%3DAzzurra%26date%3D2020%26date%3D2019%26date%3D2019%26date%3D26112019%26date%3D27122019%26date%3D13122019%26volume%3D63%26issue%3D5%26spage%3D2673%26epage%3D2687" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Arun John, Umashankar Vetrivel, Muthukumaran Sivashanmugam, <span class="NLM_string-name hlFld-ContribAuthor">Sulochana Konerirajapuram Natarajan</span>. </span><span class="cited-content_cbyCitation_article-title">Microsecond Simulation of the Proteoglycan-like Region of Carbonic Anhydrase IX and Design of Chemical Inhibitors Targeting pH Homeostasis in Cancer Cells. </span><span class="cited-content_cbyCitation_journal-name">ACS Omega</span><span> <strong>2020,</strong> <em>5 </em>
                                    (8)
                                     , 4270-4281. <a href="https://doi.org/10.1021/acsomega.9b04203" title="DOI URL">https://doi.org/10.1021/acsomega.9b04203</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsomega.9b04203&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsomega.9b04203%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Omega%26atitle%3DMicrosecond%252BSimulation%252Bof%252Bthe%252BProteoglycan-like%252BRegion%252Bof%252BCarbonic%252BAnhydrase%252BIX%252Band%252BDesign%252Bof%252BChemical%252BInhibitors%252BTargeting%252BpH%252BHomeostasis%252Bin%252BCancer%252BCells%26aulast%3DJohn%26aufirst%3DArun%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D2019%26date%3D09122019%26date%3D08012020%26date%3D20022020%26volume%3D5%26issue%3D8%26spage%3D4270%26epage%3D4281" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Gergő Dargó, Dávid Bajusz, Kristóf Simon, Judit Müller, <span class="NLM_string-name hlFld-ContribAuthor">György T. Balogh</span>. </span><span class="cited-content_cbyCitation_article-title">Human Serum Albumin Binding in a Vial: A Novel UV-pH Titration Method To Assist Drug Design. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (4)
                                     , 1763-1774. <a href="https://doi.org/10.1021/acs.jmedchem.0c00046" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c00046</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c00046&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c00046%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DHuman%252BSerum%252BAlbumin%252BBinding%252Bin%252Ba%252BVial%25253A%252BA%252BNovel%252BUV-pH%252BTitration%252BMethod%252BTo%252BAssist%252BDrug%252BDesign%26aulast%3DDarg%25C3%25B3%26aufirst%3DGerg%25C5%2591%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D10012020%26date%3D11022020%26date%3D29012020%26volume%3D63%26issue%3D4%26spage%3D1763%26epage%3D1774" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Karina B. Santos, Isabella A. Guedes, Ana L. M. Karl, <span class="NLM_string-name hlFld-ContribAuthor">Laurent E. Dardenne</span>. </span><span class="cited-content_cbyCitation_article-title">Highly Flexible Ligand Docking: Benchmarking of the DockThor Program on the LEADS-PEP Protein–Peptide Data Set. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2020,</strong> <em>60 </em>
                                    (2)
                                     , 667-683. <a href="https://doi.org/10.1021/acs.jcim.9b00905" title="DOI URL">https://doi.org/10.1021/acs.jcim.9b00905</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.9b00905&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.9b00905%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DHighly%252BFlexible%252BLigand%252BDocking%25253A%252BBenchmarking%252Bof%252Bthe%252BDockThor%252BProgram%252Bon%252Bthe%252BLEADS-PEP%252BProtein%2525E2%252580%252593Peptide%252BData%252BSet%26aulast%3DSantos%26aufirst%3DKarina%2BB.%26date%3D2020%26date%3D2020%26date%3D2019%26date%3D30092019%26date%3D27012020%26date%3D10012020%26volume%3D60%26issue%3D2%26spage%3D667%26epage%3D683" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Xi Jiang, Ashutosh Kumar, Yasutaka Motomura, Tian Liu, Yong Zhou, Kazuyo Moro, Kam Y. J. Zhang, <span class="NLM_string-name hlFld-ContribAuthor">Qing Yang</span>. </span><span class="cited-content_cbyCitation_article-title">A Series of Compounds Bearing a Dipyrido-Pyrimidine Scaffold Acting as Novel Human and Insect Pest Chitinase Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (3)
                                     , 987-1001. <a href="https://doi.org/10.1021/acs.jmedchem.9b01154" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b01154</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b01154&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.9b01154%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DA%252BSeries%252Bof%252BCompounds%252BBearing%252Ba%252BDipyrido-Pyrimidine%252BScaffold%252BActing%252Bas%252BNovel%252BHuman%252Band%252BInsect%252BPest%252BChitinase%252BInhibitors%26aulast%3DJiang%26aufirst%3DXi%26date%3D2020%26date%3D2020%26date%3D2019%26date%3D17072019%26date%3D03022020%26date%3D13012020%26volume%3D63%26issue%3D3%26spage%3D987%26epage%3D1001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">William C. Wright, Jude Chenge, Jingheng Wang, Hazel M. Girvan, Lei Yang, Sergio C. Chai, Andrew D. Huber, Jing Wu, Peter O. Oladimeji, Andrew W. Munro, <span class="NLM_string-name hlFld-ContribAuthor">Taosheng Chen</span>. </span><span class="cited-content_cbyCitation_article-title">Clobetasol Propionate Is a Heme-Mediated Selective Inhibitor of Human Cytochrome P450 3A5. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (3)
                                     , 1415-1433. <a href="https://doi.org/10.1021/acs.jmedchem.9b02067" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b02067</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b02067&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.9b02067%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DClobetasol%252BPropionate%252BIs%252Ba%252BHeme-Mediated%252BSelective%252BInhibitor%252Bof%252BHuman%252BCytochrome%252BP450%252B3A5%26aulast%3DWright%26aufirst%3DWilliam%2BC.%26date%3D2020%26date%3D2020%26date%3D2019%26date%3D12122019%26date%3D22012020%26volume%3D63%26issue%3D3%26spage%3D1415%26epage%3D1433" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Richard Mariadasse, Sanjay Kumar Choubey, <span class="NLM_string-name hlFld-ContribAuthor">Jeyaraman Jeyakanthan</span>. </span><span class="cited-content_cbyCitation_article-title">Insights into Exogenous Tryptophan-Mediated Allosteric Communication and Helical Transition of TRP Protein for Transcription Regulation. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2020,</strong> <em>60 </em>
                                    (1)
                                     , 175-191. <a href="https://doi.org/10.1021/acs.jcim.9b00755" title="DOI URL">https://doi.org/10.1021/acs.jcim.9b00755</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.9b00755&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.9b00755%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DInsights%252Binto%252BExogenous%252BTryptophan-Mediated%252BAllosteric%252BCommunication%252Band%252BHelical%252BTransition%252Bof%252BTRP%252BProtein%252Bfor%252BTranscription%252BRegulation%26aulast%3DMariadasse%26aufirst%3DRichard%26date%3D2020%26date%3D2019%26date%3D2019%26date%3D08092019%26date%3D24122019%26date%3D19112019%26volume%3D60%26issue%3D1%26spage%3D175%26epage%3D191" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Pharit Kamsri, Chayanin Hanwarinroj, Naruedon Phusi, Thimpika Pornprom, Kampanart Chayajarus, Auradee Punkvang, Nitima Suttipanta, Potjanee Srimanote, Khomson Suttisintong, Chomphunuch Songsiriritthigul, Patchreenart Saparpakorn, Supa Hannongbua, Siriluk Rattanabunyong, Supaporn Seetaha, Kiattawee Choowongkomon, Sanya Sureram, Prasat Kittakoop, Poonpilas Hongmanee, Pitak Santanirand, Zhaoqiang Chen, Weiliang Zhu, Rosemary A. Blood, Yuiko Takebayashi, Philip Hinchliffe, Adrian J. Mulholland, James Spencer, <span class="NLM_string-name hlFld-ContribAuthor">Pornpan Pungpo</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of New and Potent InhA Inhibitors as Antituberculosis Agents: Structure-Based Virtual Screening Validated by Biological Assays and X-ray Crystallography. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2020,</strong> <em>60 </em>
                                    (1)
                                     , 226-234. <a href="https://doi.org/10.1021/acs.jcim.9b00918" title="DOI URL">https://doi.org/10.1021/acs.jcim.9b00918</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.9b00918&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.9b00918%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DDiscovery%252Bof%252BNew%252Band%252BPotent%252BInhA%252BInhibitors%252Bas%252BAntituberculosis%252BAgents%25253A%252BStructure-Based%252BVirtual%252BScreening%252BValidated%252Bby%252BBiological%252BAssays%252Band%252BX-ray%252BCrystallography%26aulast%3DKamsri%26aufirst%3DPharit%26date%3D2020%26date%3D2019%26date%3D2019%26date%3D03102019%26date%3D27122019%26date%3D10122019%26volume%3D60%26issue%3D1%26spage%3D226%26epage%3D234" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">David J. Huggins, Bryn S. Hardwick, Pooja Sharma, Amy Emery, Luca Laraia, Fengzhi Zhang, Ana J. Narvaez, Meredith Roberts-Thomson, Alex T. Crooks, Robert G. Boyle, Richard Boyce, David W. Walker, Natalia Mateu, Grahame J. McKenzie, David R. Spring, <span class="NLM_string-name hlFld-ContribAuthor">Ashok R. Venkitaraman</span>. </span><span class="cited-content_cbyCitation_article-title">Development of a Novel Cell-Permeable Protein–Protein Interaction Inhibitor for the Polo-box Domain of Polo-like Kinase 1. </span><span class="cited-content_cbyCitation_journal-name">ACS Omega</span><span> <strong>2020,</strong> <em>5 </em>
                                    (1)
                                     , 822-831. <a href="https://doi.org/10.1021/acsomega.9b03626" title="DOI URL">https://doi.org/10.1021/acsomega.9b03626</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsomega.9b03626&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsomega.9b03626%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Omega%26atitle%3DDevelopment%252Bof%252Ba%252BNovel%252BCell-Permeable%252BProtein%2525E2%252580%252593Protein%252BInteraction%252BInhibitor%252Bfor%252Bthe%252BPolo-box%252BDomain%252Bof%252BPolo-like%252BKinase%252B1%26aulast%3DHuggins%26aufirst%3DDavid%2BJ.%26date%3D2020%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D28102019%26date%3D13122019%26date%3D24122019%26volume%3D5%26issue%3D1%26spage%3D822%26epage%3D831" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Andrew E. Shouksmith, Justyna M. Gawel, Nabanita Nawar, Diana Sina, Yasir S. Raouf, Shazreh Bukhari, Liying He, Alexandra E. Johns, Pimyupa Manaswiyoungkul, Olasunkanmi O. Olaoye, Aaron D. Cabral, Abootaleb Sedighi, Elvin D. de Araujo, <span class="NLM_string-name hlFld-ContribAuthor">Patrick T. Gunning</span>. </span><span class="cited-content_cbyCitation_article-title">Class I/IIb-Selective HDAC Inhibitor Exhibits Oral Bioavailability and Therapeutic Efficacy in Acute Myeloid Leukemia. </span><span class="cited-content_cbyCitation_journal-name">ACS Medicinal Chemistry Letters</span><span> <strong>2020,</strong> <em>11 </em>
                                    (1)
                                     , 56-64. <a href="https://doi.org/10.1021/acsmedchemlett.9b00471" title="DOI URL">https://doi.org/10.1021/acsmedchemlett.9b00471</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsmedchemlett.9b00471&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsmedchemlett.9b00471%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DClass%252BI%25252FIIb-Selective%252BHDAC%252BInhibitor%252BExhibits%252BOral%252BBioavailability%252Band%252BTherapeutic%252BEfficacy%252Bin%252BAcute%252BMyeloid%252BLeukemia%26aulast%3DShouksmith%26aufirst%3DAndrew%2BE.%26date%3D2020%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D13102019%26date%3D13122019%26date%3D30122019%26date%3D13122019%26volume%3D11%26issue%3D1%26spage%3D56%26epage%3D64" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor "> Juan Manuel    Gutierrez-Villagomez    </span><span class="hlFld-ContribAuthor "> Enrique    Ramírez-Chávez </span><span class="hlFld-ContribAuthor "> Jorge    Molina-Torres </span><span class="hlFld-ContribAuthor "> Juan    Vázquez-Martínez    </span>. </span><span class="cited-content_cbyCitation_article-title">From Natural to Synthetic Quorum Sensing Active Compounds: Insights to Develop Specific Quorum Sensing Modulators for Microbe-Plant Interaction. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2020,</strong>,, 87-113. <a href="https://doi.org/10.1021/bk-2020-1374.ch006" title="DOI URL">https://doi.org/10.1021/bk-2020-1374.ch006</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/bk-2020-1374.ch006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fbk-2020-1374.ch006%26sid%3Dliteratum%253Aachs%26atitle%3DFrom%252BNatural%252Bto%252BSynthetic%252BQuorum%252BSensing%252BActive%252BCompounds%25253A%252BInsights%252Bto%252BDevelop%252BSpecific%252BQuorum%252BSensing%252BModulators%252Bfor%252BMicrobe-Plant%252BInteraction%26aulast%3DGutierrez-Villagomez%26aufirst%3DJuan%2BManuel%26date%3D2020%26date%3D2020%26spage%3D87%26epage%3D113%26atitle%3DQuorum%252BSensing%25253A%252BMicrobial%252BRules%252Bof%252BLife%26aulast%3DDhiman%26aufirst%3DSaurabh%2BSudha%26date%3D2020%26date%3D2020%26volume%3D1374" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Christian J. Suess, Floriane L. Martins, Anna K. Croft, <span class="NLM_string-name hlFld-ContribAuthor">Christof M. Jäger</span>. </span><span class="cited-content_cbyCitation_article-title">Radical Stabilization Energies for Enzyme Engineering: Tackling the Substrate Scope of the Radical Enzyme QueE. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2019,</strong> <em>59 </em>
                                    (12)
                                     , 5111-5125. <a href="https://doi.org/10.1021/acs.jcim.9b00017" title="DOI URL">https://doi.org/10.1021/acs.jcim.9b00017</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.9b00017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.9b00017%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DRadical%252BStabilization%252BEnergies%252Bfor%252BEnzyme%252BEngineering%25253A%252BTackling%252Bthe%252BSubstrate%252BScope%252Bof%252Bthe%252BRadical%252BEnzyme%252BQueE%26aulast%3DSuess%26aufirst%3DChristian%2BJ.%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D03012019%26date%3D03122019%26date%3D15112019%26volume%3D59%26issue%3D12%26spage%3D5111%26epage%3D5125" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Levente M. Mihalovits, György G. Ferenczy, <span class="NLM_string-name hlFld-ContribAuthor">György M. Keserű</span>. </span><span class="cited-content_cbyCitation_article-title">Catalytic Mechanism and Covalent Inhibition of UDP-N-Acetylglucosamine Enolpyruvyl Transferase (MurA): Implications to the Design of Novel Antibacterials. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2019,</strong> <em>59 </em>
                                    (12)
                                     , 5161-5173. <a href="https://doi.org/10.1021/acs.jcim.9b00691" title="DOI URL">https://doi.org/10.1021/acs.jcim.9b00691</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.9b00691&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.9b00691%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DCatalytic%252BMechanism%252Band%252BCovalent%252BInhibition%252Bof%252BUDP-N-Acetylglucosamine%252BEnolpyruvyl%252BTransferase%252B%252528MurA%252529%25253A%252BImplications%252Bto%252Bthe%252BDesign%252Bof%252BNovel%252BAntibacterials%26aulast%3DMihalovits%26aufirst%3DLevente%2BM.%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D20082019%26date%3D26112019%26date%3D12112019%26volume%3D59%26issue%3D12%26spage%3D5161%26epage%3D5173" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Klemens Hoegenauer, Joerg Kallen, Eloísa Jiménez-Núñez, Ross Strang, Peter Ertl, Nigel G. Cooke, Samuel Hintermann, Markus Voegtle, Claudia Betschart, Daniel J. J. McKay, Juergen Wagner, Johannes Ottl, Christian Beerli, Andreas Billich, Janet Dawson, Klemens Kaupmann, Markus Streiff, Nathalie Gobeau, Stephanie Harlfinger, Rowan Stringer, <span class="NLM_string-name hlFld-ContribAuthor">Christine Guntermann</span>. </span><span class="cited-content_cbyCitation_article-title">Structure-Based and Property-Driven Optimization of N-Aryl Imidazoles toward Potent and Selective Oral RORγt Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2019,</strong> <em>62 </em>
                                    (23)
                                     , 10816-10832. <a href="https://doi.org/10.1021/acs.jmedchem.9b01291" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b01291</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b01291&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.9b01291%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DStructure-Based%252Band%252BProperty-Driven%252BOptimization%252Bof%252BN-Aryl%252BImidazoles%252Btoward%252BPotent%252Band%252BSelective%252BOral%252BROR%2525CE%2525B3t%252BInhibitors%26aulast%3DHoegenauer%26aufirst%3DKlemens%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D08082019%26date%3D03122019%26date%3D15112019%26volume%3D62%26issue%3D23%26spage%3D10816%26epage%3D10832" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Patrick G. Dougherty, Jin Wen, Xiaoyan Pan, Amritendu Koley, Jian-Guo Ren, Ashweta Sahni, Ruchira Basu, Heba Salim, George Appiah Kubi, Ziqing Qian, <span class="NLM_string-name hlFld-ContribAuthor">Dehua Pei</span>. </span><span class="cited-content_cbyCitation_article-title">Enhancing the Cell Permeability of Stapled Peptides with a Cyclic Cell-Penetrating Peptide. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2019,</strong> <em>62 </em>
                                    (22)
                                     , 10098-10107. <a href="https://doi.org/10.1021/acs.jmedchem.9b00456" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b00456</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b00456&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.9b00456%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DEnhancing%252Bthe%252BCell%252BPermeability%252Bof%252BStapled%252BPeptides%252Bwith%252Ba%252BCyclic%252BCell-Penetrating%252BPeptide%26aulast%3DDougherty%26aufirst%3DPatrick%2BG.%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D15032019%26date%3D08112019%26date%3D28102019%26volume%3D62%26issue%3D22%26spage%3D10098%26epage%3D10107" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Simona De Vita, Gianluigi Lauro, Dafne Ruggiero, Stefania Terracciano, Raffaele Riccio, <span class="NLM_string-name hlFld-ContribAuthor">Giuseppe Bifulco</span>. </span><span class="cited-content_cbyCitation_article-title">Protein Preparation Automatic Protocol for High-Throughput Inverse Virtual Screening: Accelerating the Target Identification by Computational Methods. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2019,</strong> <em>59 </em>
                                    (11)
                                     , 4678-4690. <a href="https://doi.org/10.1021/acs.jcim.9b00428" title="DOI URL">https://doi.org/10.1021/acs.jcim.9b00428</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.9b00428&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.9b00428%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DProtein%252BPreparation%252BAutomatic%252BProtocol%252Bfor%252BHigh-Throughput%252BInverse%252BVirtual%252BScreening%25253A%252BAccelerating%252Bthe%252BTarget%252BIdentification%252Bby%252BComputational%252BMethods%26aulast%3DDe%2BVita%26aufirst%3DSimona%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D23052019%26date%3D01112019%26date%3D08102019%26volume%3D59%26issue%3D11%26spage%3D4678%26epage%3D4690" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div><a href="#" data-source="/pb/widgets/cited-by?doi=10.1021%2Fjm051256o&amp;widgetId=9cf770f4-a9a2-46e0-bd29-3da15bcf4ae0&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Barticle%3Aarticle%3A10.1021%2Fjm051256o%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2006.49.issue-21%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text&amp;citedByCount=3115&amp;pagesCount=6&amp;pageNumber=2" title="Load more citations" class="button_primary float-right cited-by__see-more mb-5"><i aria-hidden="”true”" class="icon-angle-double-down"></i><span>Load more citations</span></a></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2006/jmcmar.2006.49.issue-21/jm051256o/production/images/medium/jm051256on00001.gif" alt="" id="_i2" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2006/jmcmar.2006.49.issue-21/jm051256o/production/images/large/jm051256on00001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm051256o&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="jm051256of00001" data-index="1" class="article__inlineFigure"><h2 class="fig-label"></h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2006/jmcmar.2006.49.issue-21/jm051256o/production/images/medium/jm051256of00001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2006/jmcmar.2006.49.issue-21/jm051256o/production/images/large/jm051256of00001.jpeg" id="rightTab-_i10" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last"> Figure 1 Schematic of a ligand group interacting with two distinct hydrophobic environments:  above a hydrophobic “plane” (A) and enclosed in a hydrophobic cavity (B).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2006/jmcmar.2006.49.issue-21/jm051256o/production/images/large/jm051256of00001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm051256o&amp;id=jm051256of00001"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="jm051256of00002" data-index="2" class="article__inlineFigure"><h2 class="fig-label"></h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2006/jmcmar.2006.49.issue-21/jm051256o/production/images/medium/jm051256of00002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2006/jmcmar.2006.49.issue-21/jm051256o/production/images/large/jm051256of00002.jpeg" id="rightTab-_i12" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last"> Figure 2 Boehringer active for <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1kv2">1kv2</a> bound to human p38 map kinase. The naphthyl group receives a −4.5 kcal/mol hydrophobic enclosure packing reward.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2006/jmcmar.2006.49.issue-21/jm051256o/production/images/large/jm051256of00002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm051256o&amp;id=jm051256of00002"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="jm051256of00003" data-index="3" class="article__inlineFigure"><h2 class="fig-label"></h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2006/jmcmar.2006.49.issue-21/jm051256o/production/images/medium/jm051256of00003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2006/jmcmar.2006.49.issue-21/jm051256o/production/images/large/jm051256of00003.jpeg" id="rightTab-_i14" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last"> Figure 3 The <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1bl7">1bl7</a> ligand interacts with p38 MAP kinase through a neutral−neutral hydrogen bond between the ligand's aromatic nitrogen and the Met 109 N−H group.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2006/jmcmar.2006.49.issue-21/jm051256o/production/images/large/jm051256of00003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm051256o&amp;id=jm051256of00003"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="jm051256of00004" data-index="4" class="article__inlineFigure"><h2 class="fig-label"></h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2006/jmcmar.2006.49.issue-21/jm051256o/production/images/medium/jm051256of00004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2006/jmcmar.2006.49.issue-21/jm051256o/production/images/large/jm051256of00004.jpeg" id="rightTab-_i16" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last"> Figure 4 Staurosporine bound to human cyclin dependent kinase (CDK2). The pair of correlated hydrogen bonds receives a −3 kcal/mol reward, while the central component of the ring is given a −3 kcal/mol hydrophobic enclosure packing reward.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2006/jmcmar.2006.49.issue-21/jm051256o/production/images/large/jm051256of00004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm051256o&amp;id=jm051256of00004"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="jm051256of00005" data-index="5" class="article__inlineFigure"><h2 class="fig-label"></h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2006/jmcmar.2006.49.issue-21/jm051256o/production/images/medium/jm051256of00005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2006/jmcmar.2006.49.issue-21/jm051256o/production/images/large/jm051256of00005.jpeg" id="rightTab-_i17" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last"> Figure 5 AG12073 bound to human cyclin dependent kinase (CDK2). The three correlated hydrogen bonds receive a −4.2 kcal/mol reward. </p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2006/jmcmar.2006.49.issue-21/jm051256o/production/images/large/jm051256of00005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm051256o&amp;id=jm051256of00005"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="jm051256of00006" data-index="6" class="article__inlineFigure"><h2 class="fig-label"></h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2006/jmcmar.2006.49.issue-21/jm051256o/production/images/medium/jm051256of00006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2006/jmcmar.2006.49.issue-21/jm051256o/production/images/large/jm051256of00006.jpeg" id="rightTab-_i18" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last"> Figure 6 Biotin bound to streptavidin. The identification of a triplet of correlated hydrogen bonds in the ring in a hydrophobically enclosed region, and the three hydrogen bonds to the ligand carbonyl within that ring each contribute −3 kcal/mol rewards to this tightly bound complex (Δ<i>G</i><sub>exp</sub> = −18.3 kcal/mol, XP binding = −18.2 kcal/mol).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2006/jmcmar.2006.49.issue-21/jm051256o/production/images/large/jm051256of00006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm051256o&amp;id=jm051256of00006"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="jm051256of00007" data-index="7" class="article__inlineFigure"><h2 class="fig-label"></h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2006/jmcmar.2006.49.issue-21/jm051256o/production/images/medium/jm051256of00007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2006/jmcmar.2006.49.issue-21/jm051256o/production/images/large/jm051256of00007.jpeg" id="rightTab-_i19" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last"> Figure 7 Fidarestat bound to aldose reductase. The triplet of special hydrogen bonds to the ring contributes −5.0 kcal/mol to the binding energy of this 9 nM inhibitor. </p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2006/jmcmar.2006.49.issue-21/jm051256o/production/images/large/jm051256of00007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm051256o&amp;id=jm051256of00007"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="jm051256of00008" data-index="8" class="article__inlineFigure"><h2 class="fig-label"></h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2006/jmcmar.2006.49.issue-21/jm051256o/production/images/medium/jm051256of00008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2006/jmcmar.2006.49.issue-21/jm051256o/production/images/large/jm051256of00008.jpeg" id="rightTab-_i21" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last"> Figure 8 The bidentate hydrogen bonds in this thrombin complex bridge the ligand and Asp 189.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2006/jmcmar.2006.49.issue-21/jm051256o/production/images/large/jm051256of00008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm051256o&amp;id=jm051256of00008"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="jm051256of00009" data-index="9" class="article__inlineFigure"><h2 class="fig-label"></h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2006/jmcmar.2006.49.issue-21/jm051256o/production/images/medium/jm051256of00009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2006/jmcmar.2006.49.issue-21/jm051256o/production/images/large/jm051256of00009.jpeg" id="rightTab-_i22" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last"> Figure 9 Gr217029 forms hydrogen bonds with two nearby Asp residues when bound to tern N9 influenza virus neuraminidase.  </p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2006/jmcmar.2006.49.issue-21/jm051256o/production/images/large/jm051256of00009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm051256o&amp;id=jm051256of00009"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="jm051256of00010" data-index="10" class="article__inlineFigure"><h2 class="fig-label"></h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2006/jmcmar.2006.49.issue-21/jm051256o/production/images/medium/jm051256of00010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2006/jmcmar.2006.49.issue-21/jm051256o/production/images/large/jm051256of00010.jpeg" id="rightTab-_i26" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last"> Figure 10 SCX-001 bound to cyclooxygenase-2. The three phenyl rings obtain a 3.7 kcal/mol hydrophobic enclosure packing reward for occupying the large hydrophobic pocket.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2006/jmcmar.2006.49.issue-21/jm051256o/production/images/large/jm051256of00010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm051256o&amp;id=jm051256of00010"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="jm051256of00011" data-index="11" class="article__inlineFigure"><h2 class="fig-label"></h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2006/jmcmar.2006.49.issue-21/jm051256o/production/images/medium/jm051256of00011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2006/jmcmar.2006.49.issue-21/jm051256o/production/images/large/jm051256of00011.jpeg" id="rightTab-_i27" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last"> Figure 11 The <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1lpk">1lpk</a> ligand bound to factor Xa. A special salt bridge pair with ASP 189, analogous to that in thrombin (Figure 8), is observed. A hydrophobic enclosure packing reward of −0.8 kcal/mol is achieved by the phenyl ring occupying the Tyr 99/Trp 215/Phe 174 pocket. Also, a pi-cation interaction is received by the charged end of the ligand.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2006/jmcmar.2006.49.issue-21/jm051256o/production/images/large/jm051256of00011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm051256o&amp;id=jm051256of00011"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="jm051256of00012" data-index="12" class="article__inlineFigure"><h2 class="fig-label"></h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2006/jmcmar.2006.49.issue-21/jm051256o/production/images/medium/jm051256of00012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2006/jmcmar.2006.49.issue-21/jm051256o/production/images/large/jm051256of00012.jpeg" id="rightTab-_i28" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last"> Figure 12 The <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1hpx">1hpx</a> ligand bound to HIV-1 protease. Hydrophobic groups of the ligand are not hydrophobically enclosed and do not receive a hydrophobic enclosure packing reward. For example, the phenyl ring faces hydrophobic residues on only one face.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2006/jmcmar.2006.49.issue-21/jm051256o/production/images/large/jm051256of00012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm051256o&amp;id=jm051256of00012"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="jm051256of00013" data-index="13" class="article__inlineFigure"><h2 class="fig-label"></h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2006/jmcmar.2006.49.issue-21/jm051256o/production/images/medium/jm051256of00013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2006/jmcmar.2006.49.issue-21/jm051256o/production/images/large/jm051256of00013.jpeg" id="rightTab-_i29" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last"> Figure 13 4-Hydroxytamoxifen bound to the human estrogen receptor. Hydrophobic enclosure about the phenoxy group is illustrated by displaying lipophilic protein atoms as green spheres.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2006/jmcmar.2006.49.issue-21/jm051256o/production/images/large/jm051256of00013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm051256o&amp;id=jm051256of00013"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"></div><p class="rightTab-references-count">
            This article references 42 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-jm051256ob00001"><div class="reference"><strong class="refLabel"><a href="#jm051256ob00001" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-jm051256ob00001_1"><span><span class="NLM_string-ref">Friesner, R. A.; Banks, J. L.; Murphy, R. B.; Halgren, T. A.; Klicic, J. J.; Mainz, D. T.; Repasky, M. P.; Knoll, E. H.; Shelley, M.; Perry, J. K.; Shaw, D. E.; Francis, P.; Shenkin, P. S. Glide:  A New Approach for Rapid, Accurate Docking and Scoring. 1. Method and Assessment of Docking Accuracy. <i>  J. Med. Chem.</i><b>2004</b>, <i>47</i>, 1739−1749.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=1749&author=R.+A.+Friesner&author=J.+L.+Banks&author=R.+B.+Murphy&author=T.+A.+Halgren&author=J.+J.+Klicic&author=D.+T.+Mainz&author=M.+P.+Repasky&author=E.+H.+Knoll&author=M.+Shelley&author=J.+K.+Perry&author=D.+E.+Shaw&author=P.+Francis&author=P.+S.+Shenkin&author=A+Glide&author=+New+Approach&author=+Accurate+Docking&title=Method+and+Assessment+of+Docking+Accuracy"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm051256ob00001_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26atitle%3DMethod%2520and%2520Assessment%2520of%2520Docking%2520Accuracy%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D1749" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm051256ob00002"><div class="reference"><strong class="refLabel"><a href="#jm051256ob00002" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-jm051256ob00002_1"><span><span class="NLM_string-ref">Halgren, T. A.; Murphy, R. B.; Friesner, R. A.; Beard, H. S.; Frye, L. L.; Pollard, W. T.; Banks, J. L. Glide:  A New Approach for Rapid, Accurate Docking and Scoring. 2. Enrichment Factors in Database Screening. <i>  J. </i><i>Med. Chem.</i><b>2004</b>, <i>47</i>, 1750−1759.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=1759&author=T.+A.+Halgren&author=R.+B.+Murphy&author=R.+A.+Friesner&author=H.+S.+Beard&author=L.+L.+Frye&author=W.+T.+Pollard&author=J.+L.+Banks&author=A+Glide&author=+New+Approach&author=+Accurate+Docking&title=Enrichment+Factors+in+Database+Screening"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm051256ob00002_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DHalgren%26aufirst%3DT.%2BA.%26atitle%3DEnrichment%2520Factors%2520in%2520Database%2520Screening%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D1759" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm051256ob00003"><div class="reference"><strong class="refLabel"><a href="#jm051256ob00003" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-jm051256ob00003_1"><span><span class="NLM_string-ref">Perola, E.; Walters, W. P.; Charifson, P. S. A Detailed Comparison of Current Docking and Scoring Methods on Systems of Pharmaceutical Relevance. <i>  Proteins</i><b>2004</b>, <i>56</i>, 235−249.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2004&pages=249&author=E.+Perola&author=W.+P.+Walters&author=P.+S+Charifson&title=A+Detailed+Comparison+of+Current+Docking+and+Scoring+Methods+on+Systems+of+Pharmaceutical+Relevance"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm051256ob00003_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DPerola%26aufirst%3DE.%26atitle%3DA%2520Detailed%2520Comparison%2520of%2520Current%2520Docking%2520and%2520Scoring%2520Methods%2520on%2520Systems%2520of%2520Pharmaceutical%2520Relevance%26jtitle%3DProteins%26date%3D2004%26volume%3D56%26spage%3D249" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm051256ob00004"><div class="reference"><strong class="refLabel"><a href="#jm051256ob00004" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-jm051256ob00004_1"><span><span class="NLM_string-ref">Krovat, E. M.; Steindl, T.; Langer, T. Recent Advances in Docking and Scoring. <i>  Curr. Comput.-Aided Drug Des. </i><b>2005</b>, <i>1</i>, 93−102.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2005&pages=102&author=E.+M.+Krovat&author=T.+Steindl&author=T+Langer&title=Recent+Advances+in+Docking+and+Scoring"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm051256ob00004_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DKrovat%26aufirst%3DE.%2BM.%26atitle%3DRecent%2520Advances%2520in%2520Docking%2520and%2520Scoring%26jtitle%3DCurr.%2520Comput.-Aided%2520Drug%2520Des.%26date%3D2005%26volume%3D1%26spage%3D102" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm051256ob00005"><div class="reference"><strong class="refLabel"><a href="#jm051256ob00005" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-jm051256ob00005_1"><span><span class="NLM_string-ref">Kontoyianni, M.; McClellan, L. M.; Sokol, G. S. Evaluation of Docking Performance:  Comparative Data on Docking Algorithms. <i>  J. Med. Chem</i>. <b>2004</b>, <i>47</i>, 558−565.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=565&author=M.+Kontoyianni&author=L.+M.+McClellan&author=G.+S+Sokol&title=Evaluation+of+Docking+Performance%3A%E2%80%89+Comparative+Data+on+Docking+Algorithms"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm051256ob00005_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DKontoyianni%26aufirst%3DM.%26atitle%3DEvaluation%2520of%2520Docking%2520Performance%253A%25E2%2580%2589%2520Comparative%2520Data%2520on%2520Docking%2520Algorithms%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D565" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm051256ob00006"><div class="reference"><strong class="refLabel"><a href="#jm051256ob00006" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-jm051256ob00006_1"><span><span class="NLM_string-ref">Sherman, W.; Day, T.; Jacobson, M. P.; Friesner, R. A.; Farid, R. Novel Procedure for Modeling Ligand/Receptor Induced Fit Effects. <i>  J. Med. </i><i>Chem.</i><b>2006</b>, <i>49</i>, 534−553.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2006&pages=553&author=W.+Sherman&author=T.+Day&author=M.+P.+Jacobson&author=R.+A.+Friesner&author=R+Farid&title=Novel+Procedure+for+Modeling+Ligand%2FReceptor+Induced+Fit+Effects"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm051256ob00006_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSherman%26aufirst%3DW.%26atitle%3DNovel%2520Procedure%2520for%2520Modeling%2520Ligand%252FReceptor%2520Induced%2520Fit%2520Effects%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2006%26volume%3D49%26spage%3D553" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm051256ob00007"><div class="reference"><strong class="refLabel"><a href="#jm051256ob00007" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-jm051256ob00007_1"><span><span class="NLM_string-ref">Farid, R.; Day, T.; Friesner, R. A.; Pearlstein, R. A. New Insights about HERG Blockade Obtained from Protein Modeling, Potential Energy Mapping, and Docking Studies. <i>  Bioorg. Med. Chem.</i><b>2006</b>, <i>14</i>, 3160−3173.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2006&pages=3173&author=R.+Farid&author=T.+Day&author=R.+A.+Friesner&author=R.+A+Pearlstein&title=New+Insights+about+HERG+Blockade+Obtained+from+Protein+Modeling%2C+Potential+Energy+Mapping%2C+and+Docking+Studies"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm051256ob00007_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DFarid%26aufirst%3DR.%26atitle%3DNew%2520Insights%2520about%2520HERG%2520Blockade%2520Obtained%2520from%2520Protein%2520Modeling%252C%2520Potential%2520Energy%2520Mapping%252C%2520and%2520Docking%2520Studies%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2006%26volume%3D14%26spage%3D3173" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm051256ob00008"><div class="reference"><strong class="refLabel"><a href="#jm051256ob00008" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-jm051256ob00008_1"><span><span class="NLM_string-ref">Eldridge, M. D.; Murray, C. W.; Auton, T. R.; Paolinine, G. V.; Mee, R. P. Empirical Scoring Functions:  I. The Development of a Fast Empirical Scoring Function to Estimate the Binding Affinity of Ligands in Receptor Complexes. <i>  J. Comput.-Aid. Mol. Des.</i><b>1997</b>, <i>11</i>, 425−445.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=1997&pages=445&author=M.+D.+Eldridge&author=C.+W.+Murray&author=T.+R.+Auton&author=G.+V.+Paolinine&author=R.+P+Mee&title=Empirical+Scoring+Functions%3A%E2%80%89+I.+The+Development+of+a+Fast+Empirical+Scoring+Function+to+Estimate+the+Binding+Affinity+of+Ligands+in+Receptor+Complexes"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm051256ob00008_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DEldridge%26aufirst%3DM.%2BD.%26atitle%3DEmpirical%2520Scoring%2520Functions%253A%25E2%2580%2589%2520I.%2520The%2520Development%2520of%2520a%2520Fast%2520Empirical%2520Scoring%2520Function%2520to%2520Estimate%2520the%2520Binding%2520Affinity%2520of%2520Ligands%2520in%2520Receptor%2520Complexes%26jtitle%3DJ.%2520Comput.-Aid.%2520Mol.%2520Des.%26date%3D1997%26volume%3D11%26spage%3D445" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm051256ob00009"><div class="reference"><strong class="refLabel"><a href="#jm051256ob00009" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-jm051256ob00009_1"><span><span class="NLM_string-ref">Gehlhaar, D. K.; Verkhivker, G. M.; Rejto, P. A.; Sherman, C. J.; Fogel, L. J.; Freer, S. T. Molecular Recognition of the Inhibitor AG-1343 by HIV-1 ProteaseConformationally Flexible Docking by Evolutionary Programming. <i>  Chem. Biol.</i><b>1995</b>, <i>2</i>, 317−324.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=1995&pages=324&author=D.+K.+Gehlhaar&author=G.+M.+Verkhivker&author=P.+A.+Rejto&author=C.+J.+Sherman&author=L.+J.+Fogel&author=S.+T+Freer&title=Molecular+Recognition+of+the+Inhibitor+AG-1343+by+HIV-1+ProteaseConformationally+Flexible+Docking+by+Evolutionary+Programming"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm051256ob00009_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DGehlhaar%26aufirst%3DD.%2BK.%26atitle%3DMolecular%2520Recognition%2520of%2520the%2520Inhibitor%2520AG-1343%2520by%2520HIV-1%2520ProteaseConformationally%2520Flexible%2520Docking%2520by%2520Evolutionary%2520Programming%26jtitle%3DChem.%2520Biol.%26date%3D1995%26volume%3D2%26spage%3D324" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm051256ob00010"><div class="reference"><div class="NLM_citation" id="rightTab-jm051256ob00010_1"><span><span class="NLM_string-ref">Morris, G. M.; Goodsell, D. S.; Halliday, R. S.; Huey, R.; Hart, W. E.; Belew, R. K.; Olsen, A. J. Automated Docking Using a Lamarckian Genetic Algorithm and an Empirical Binding Free Energy Function. <i>  J. </i><i>Comput. Chem.</i><b>1998</b>, <i>19</i>, 1639−1662.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=1998&pages=1662&author=G.+M.+Morris&author=D.+S.+Goodsell&author=R.+S.+Halliday&author=R.+Huey&author=W.+E.+Hart&author=R.+K.+Belew&author=A.+J+Olsen&title=Automated+Docking+Using+a+Lamarckian+Genetic+Algorithm+and+an+Empirical+Binding+Free+Energy+Function"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm051256ob00010_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMorris%26aufirst%3DG.%2BM.%26atitle%3DAutomated%2520Docking%2520Using%2520a%2520Lamarckian%2520Genetic%2520Algorithm%2520and%2520an%2520Empirical%2520Binding%2520Free%2520Energy%2520Function%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D1998%26volume%3D19%26spage%3D1662" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm051256ob00011"><div class="reference"><strong class="refLabel"><a href="#jm051256ob00011" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-jm051256ob00011_1"><span><span class="NLM_string-ref">Muegge, I.; Martin, Y. C. A General and Fast Scoring Function for Protein−Ligand Interactions:  A Simplified Potential Approach. <i>  J. Med. </i><i>Chem.</i><b>1999</b>, <i>42</i>, 791−804.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=1999&pages=804&author=I.+Muegge&author=Y.+C+Martin&title=A+General+and+Fast+Scoring+Function+for+Protein%E2%88%92Ligand+Interactions%3A%E2%80%89+A+Simplified+Potential+Approach"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm051256ob00011_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMuegge%26aufirst%3DI.%26atitle%3DA%2520General%2520and%2520Fast%2520Scoring%2520Function%2520for%2520Protein%25E2%2588%2592Ligand%2520Interactions%253A%25E2%2580%2589%2520A%2520Simplified%2520Potential%2520Approach%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1999%26volume%3D42%26spage%3D804" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm051256ob00012"><div class="reference"><strong class="refLabel"><a href="#jm051256ob00012" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-jm051256ob00012_1"><span><span class="NLM_string-ref">Bohm, H. J. The Development of a Simple Empirical Scoring Function to Estimate the Binding Constant for a Protein Ligand Complex of Known 3-Dimensional Structure. <i>  J. Comput.-Aided Mol. Des.</i><b>1994</b>, <i>8</i>, 243−256.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=1994&pages=256&author=H.+J+Bohm&title=The+Development+of+a+Simple+Empirical+Scoring+Function+to+Estimate+the+Binding+Constant+for+a+Protein+Ligand+Complex+of+Known+3-Dimensional+Structure"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm051256ob00012_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBohm%26aufirst%3DH.%2BJ%26atitle%3DThe%2520Development%2520of%2520a%2520Simple%2520Empirical%2520Scoring%2520Function%2520to%2520Estimate%2520the%2520Binding%2520Constant%2520for%2520a%2520Protein%2520Ligand%2520Complex%2520of%2520Known%25203-Dimensional%2520Structure%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D1994%26volume%3D8%26spage%3D256" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm051256ob00013"><div class="reference"><strong class="refLabel"><a href="#jm051256ob00013" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-jm051256ob00013_1"><span><span class="NLM_string-ref">Bohm, H. J. Prediction of Binding Constants of Protein Ligands:  A Fast Method for the Prioritization of Hits Obtained from De Novo Design or 3D Database Search Programs. <i>  J. Comput.-Aided Mol. Des.</i><b>1998</b>, <i>12</i>, 309−323.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=1998&pages=323&author=H.+J+Bohm&title=Prediction+of+Binding+Constants+of+Protein+Ligands%3A%E2%80%89+A+Fast+Method+for+the+Prioritization+of+Hits+Obtained+from+De+Novo+Design+or+3D+Database+Search+Programs"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm051256ob00013_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBohm%26aufirst%3DH.%2BJ%26atitle%3DPrediction%2520of%2520Binding%2520Constants%2520of%2520Protein%2520Ligands%253A%25E2%2580%2589%2520A%2520Fast%2520Method%2520for%2520the%2520Prioritization%2520of%2520Hits%2520Obtained%2520from%2520De%2520Novo%2520Design%2520or%25203D%2520Database%2520Search%2520Programs%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D1998%26volume%3D12%26spage%3D323" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm051256ob00014"><div class="reference"><strong class="refLabel"><a href="#jm051256ob00014" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-jm051256ob00014_1"><span><span class="NLM_string-ref">Rarey, M.; Kramer, B.; Lengauer, T.; Klebe, G. A. A Fast Flexible Docking Method Using an Incremental Construction Algorithm. <i>  J. Mol. </i><i>Biol.</i><b>1996</b>, <i>261</i>, 470−489.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=261&publication_year=1996&pages=489&author=M.+Rarey&author=B.+Kramer&author=T.+Lengauer&author=G.+A+Klebe&title=A+Fast+Flexible+Docking+Method+Using+an+Incremental+Construction+Algorithm"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm051256ob00014_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DRarey%26aufirst%3DM.%26atitle%3DA%2520Fast%2520Flexible%2520Docking%2520Method%2520Using%2520an%2520Incremental%2520Construction%2520Algorithm%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D1996%26volume%3D261%26spage%3D489" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm051256ob00015"><div class="reference"><strong class="refLabel"><a href="#jm051256ob00015" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-jm051256ob00015_1"><span><span class="NLM_string-ref">Jones, G.; Willet, P.; Glen, R. C.; Leach, A. R.; Taylor, R. Development and Validation of a Genetic Algorithm for Flexible Docking. <i>  J. Mol. Biol. </i><b>1997</b>, <i>267</i>, 727−748.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=267&publication_year=1997&pages=748&author=G.+Jones&author=P.+Willet&author=R.+C.+Glen&author=A.+R.+Leach&author=R+Taylor&title=Development+and+Validation+of+a+Genetic+Algorithm+for+Flexible+Docking"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm051256ob00015_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DJones%26aufirst%3DG.%26atitle%3DDevelopment%2520and%2520Validation%2520of%2520a%2520Genetic%2520Algorithm%2520for%2520Flexible%2520Docking%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D1997%26volume%3D267%26spage%3D748" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm051256ob00016"><div class="reference"><strong class="refLabel"><a href="#jm051256ob00016" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-jm051256ob00016_1"><span><span class="NLM_string-ref">Ewing, T. J.; Makino, S.; Skillman, A. G.; Kuntz, I. D. DOCK 4.0:  Search Strategies for Automated Molecular Docking of Flexible Molecule Databases. <i>  J. Comput.-Aided Mol. Des.</i><b>2001</b>, <i>15</i>, 411−428.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+Ewing%2C+T.+J.%3B+Makino%2C+S.%3B+Skillman%2C+A.+G.%3B+Kuntz%2C+I.+D.+DOCK+4.0%3A%E2%80%89+Search+Strategies+for+Automated+Molecular+Docking+of+Flexible+Molecule+Databases.+J.+Comput.-Aided+Mol.+Des.+2001%2C+15%2C+411%E2%88%92428.+Ewing%2C+T.+J.%3B+Makino%2C+S.%3B+Skillman%2C+A.+G.%3B+Kuntz%2C+I.+D.+DOCK+4.0%3A%E2%80%89+Search+Strategies+for+Automated+Molecular+Docking+of+Flexible+Molecule+Databases.+J.+Comput.-Aided+Mol.+Des.2001%2C+15%2C+411%E2%88%92428."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm051256ob00016_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DEwing%26aufirst%3DT.%2BJ.%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2001%26volume%3D15%26spage%3D428" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm051256ob00017"><div class="reference"><strong class="refLabel"><a href="#jm051256ob00017" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-jm051256ob00017_1"><span><span class="NLM_string-ref">Golke, H.; Hendlich, M.; Kelbe, G. Knowledge-Based Scoring Function to Predict Protein−Ligand Interactions. <i>  J. Mol. Biol.</i><b>2000</b>, <i>295</i>, 337−356.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=295&publication_year=2000&pages=356&author=H.+Golke&author=M.+Hendlich&author=G+Kelbe&title=Knowledge-Based+Scoring+Function+to+Predict+Protein%E2%88%92Ligand+Interactions"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm051256ob00017_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DGolke%26aufirst%3DH.%26atitle%3DKnowledge-Based%2520Scoring%2520Function%2520to%2520Predict%2520Protein%25E2%2588%2592Ligand%2520Interactions%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2000%26volume%3D295%26spage%3D356" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm051256ob00018"><div class="reference"><strong class="refLabel"><a href="#jm051256ob00018" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-jm051256ob00018_1"><span><span class="NLM_string-ref">Wang, R.; Lai, L.; Wang, S. Further Development and Validation of Empirical Scoring Functions for Structure-Based Binding Affinity Prediction. <i>  J. Comput.-Aided Mol. Des.</i><b>2002</b>, <i>16</i>, 11−26.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2002&pages=26&author=R.+Wang&author=L.+Lai&author=S+Wang&title=Further+Development+and+Validation+of+Empirical+Scoring+Functions+for+Structure-Based+Binding+Affinity+Prediction"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm051256ob00018_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DR.%26atitle%3DFurther%2520Development%2520and%2520Validation%2520of%2520Empirical%2520Scoring%2520Functions%2520for%2520Structure-Based%2520Binding%2520Affinity%2520Prediction%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2002%26volume%3D16%26spage%3D26" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm051256ob00019"><div class="reference"><strong class="refLabel"><a href="#jm051256ob00019" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-jm051256ob00019_1"><span><span class="NLM_string-ref">Wang, R.; Lu, Y.; Wang, S. Comparative Evaluation of 11 Scoring Functions for Molecular Docking. <i>  J. Med. Chem.</i><b>2003</b>, <i>46</i>, 2287−2303.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2003&pages=2303&author=R.+Wang&author=Y.+Lu&author=S+Wang&title=Comparative+Evaluation+of+11+Scoring+Functions+for+Molecular+Docking"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm051256ob00019_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DR.%26atitle%3DComparative%2520Evaluation%2520of%252011%2520Scoring%2520Functions%2520for%2520Molecular%2520Docking%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2003%26volume%3D46%26spage%3D2303" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm051256ob00020"><div class="reference"><div class="NLM_citation" id="rightTab-jm051256ob00020_1"><span><span class="NLM_string-ref">Wallqvist, A.; Berne, B. J. Computer-Simulation of Hydrophobic Hydration Forces on Stacked Planes at Short-Range. <i>  J. Phys. Chem.</i><b>1995</b>, <i>99</i>, 2893−2899.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=99&publication_year=1995&pages=2899&author=A.+Wallqvist&author=B.+J+Berne&title=Computer-Simulation+of+Hydrophobic+Hydration+Forces+on+Stacked+Planes+at+Short-Range"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm051256ob00020_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DWallqvist%26aufirst%3DA.%26atitle%3DComputer-Simulation%2520of%2520Hydrophobic%2520Hydration%2520Forces%2520on%2520Stacked%2520Planes%2520at%2520Short-Range%26jtitle%3DJ.%2520Phys.%2520Chem.%26date%3D1995%26volume%3D99%26spage%3D2899" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm051256ob00021"><div class="reference"><strong class="refLabel"><a href="#jm051256ob00021" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-jm051256ob00021_1"><span><span class="NLM_string-ref">Lum, K.; Chandler, D.; Weeks, J. D. Hydrophobicity at Small and Large Length Scales. <i>  J. Phys. Chem. B</i><b>1999</b>, <i>103</i>, 4570−4577.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=103&publication_year=1999&pages=4577&author=K.+Lum&author=D.+Chandler&author=J.+D+Weeks&title=Hydrophobicity+at+Small+and+Large+Length+Scales"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm051256ob00021_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DLum%26aufirst%3DK.%26atitle%3DHydrophobicity%2520at%2520Small%2520and%2520Large%2520Length%2520Scales%26jtitle%3DJ.%2520Phys.%2520Chem.%2520B%26date%3D1999%26volume%3D103%26spage%3D4577" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm051256ob00022"><div class="reference"><strong class="refLabel"><a href="#jm051256ob00022" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-jm051256ob00022_1"><span><span class="NLM_string-ref">Nicholls, A.; Sharp, K. A.; Honig, B. Protein Folding and AssociationInsights from the Interfacial and Thermodynamic Properties of Hydrocarbons. <i>  Proteins </i><b>1991</b>, <i>11</i>, 281−296.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=1991&pages=296&author=A.+Nicholls&author=K.+A.+Sharp&author=B+Honig&title=Protein+Folding+and+AssociationInsights+from+the+Interfacial+and+Thermodynamic+Properties+of+Hydrocarbons"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm051256ob00022_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DNicholls%26aufirst%3DA.%26atitle%3DProtein%2520Folding%2520and%2520AssociationInsights%2520from%2520the%2520Interfacial%2520and%2520Thermodynamic%2520Properties%2520of%2520Hydrocarbons%26jtitle%3DProteins%26date%3D1991%26volume%3D11%26spage%3D296" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm051256ob00023"><div class="reference"><strong class="refLabel"><a href="#jm051256ob00023" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-jm051256ob00023_1"><span><span class="NLM_string-ref">Zhou, R. H.; Huang, X. H.; Margulis, C. J.; Berne, B. J. Hydrophobic collapse in multidomain protein folding. <i>  Science</i><b>2004</b>, <i>305</i>, 1605−1609. Huang, X. H.; Zhou, R. H.; Berne, B. J. Drying and hydrophobic collapse of paraffin plates. <i>J. Phys. Chem. B</i><b>2005</b>, <i>109</i>, 3546−3552.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=305&publication_year=2004&pages=1609&author=R.+H.+Zhou&author=X.+H.+Huang&author=C.+J.+Margulis&author=B.+J+Berne&title=Hydrophobic+collapse+in+multidomain+protein+folding"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm051256ob00023_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DZhou%26aufirst%3DR.%2BH.%26atitle%3DHydrophobic%2520collapse%2520in%2520multidomain%2520protein%2520folding%26jtitle%3DScience%26date%3D2004%26volume%3D305%26spage%3D1609" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm051256ob00024"><div class="reference"><strong class="refLabel"><a href="#jm051256ob00024" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-jm051256ob00024_1"><span><span class="NLM_string-ref">Cheng, Y. K.; Rossky, P. J. Surface Topography Dependence of Biomolecular Hydrophobic Hydration. <i>  Nature</i><b>1998</b>, <i>392</i>, 696−699.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=392&publication_year=1998&pages=699&author=Y.+K.+Cheng&author=P.+J+Rossky&title=Surface+Topography+Dependence+of+Biomolecular+Hydrophobic+Hydration"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm051256ob00024_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DCheng%26aufirst%3DY.%2BK.%26atitle%3DSurface%2520Topography%2520Dependence%2520of%2520Biomolecular%2520Hydrophobic%2520Hydration%26jtitle%3DNature%26date%3D1998%26volume%3D392%26spage%3D699" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm051256ob00025"><div class="reference"><strong class="refLabel"><a href="#jm051256ob00025" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-jm051256ob00025_1"><span><span class="NLM_string-ref">Hummer, G.; Rasaiah, J. C.; Noworyta, J. P. Water Conduction Through the Hydrophobic Channel of a Carbon Nanotube. <i>  Nature</i><b>2001</b>, <i>414</i>, 188−190.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=414&publication_year=2001&pages=190&author=G.+Hummer&author=J.+C.+Rasaiah&author=J.+P+Noworyta&title=Water+Conduction+Through+the+Hydrophobic+Channel+of+a+Carbon+Nanotube"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm051256ob00025_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DHummer%26aufirst%3DG.%26atitle%3DWater%2520Conduction%2520Through%2520the%2520Hydrophobic%2520Channel%2520of%2520a%2520Carbon%2520Nanotube%26jtitle%3DNature%26date%3D2001%26volume%3D414%26spage%3D190" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm051256ob00026"><div class="reference"><strong class="refLabel"><a href="#jm051256ob00026" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-jm051256ob00026_1"><span><span class="NLM_string-ref">Liu, P.; Huang, X. H.; Zhou, R. H.; Berne, B. J. Observation of a dewetting transition in the collapse of the melittin tetramer. <i>  Nature</i><b>2005</b>, <i>437</i>, 159−162.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=437&publication_year=2005&pages=162&author=P.+Liu&author=X.+H.+Huang&author=R.+H.+Zhou&author=B.+J+Berne&title=Observation+of+a+dewetting+transition+in+the+collapse+of+the+melittin+tetramer"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm051256ob00026_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DP.%26atitle%3DObservation%2520of%2520a%2520dewetting%2520transition%2520in%2520the%2520collapse%2520of%2520the%2520melittin%2520tetramer%26jtitle%3DNature%26date%3D2005%26volume%3D437%26spage%3D162" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm051256ob00027"><div class="reference"><strong class="refLabel"><a href="#jm051256ob00027" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-jm051256ob00027_1"><span><span class="NLM_string-ref">Wallqvist, A.; Berne, B. J. Molecular-Dynamics Study of the Dependence of Water Solvation Free-Energy on Solute Curvature and Surface-Area. <i>  J. </i><i>Phys. Chem.</i><b>1995</b>, <i>99</i>, 2885−2892.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=99&publication_year=1995&pages=2892&author=A.+Wallqvist&author=B.+J+Berne&title=Molecular-Dynamics+Study+of+the+Dependence+of+Water+Solvation+Free-Energy+on+Solute+Curvature+and+Surface-Area"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm051256ob00027_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DWallqvist%26aufirst%3DA.%26atitle%3DMolecular-Dynamics%2520Study%2520of%2520the%2520Dependence%2520of%2520Water%2520Solvation%2520Free-Energy%2520on%2520Solute%2520Curvature%2520and%2520Surface-Area%26jtitle%3DJ.%2520Phys.%2520Chem.%26date%3D1995%26volume%3D99%26spage%3D2892" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm051256ob00028"><div class="reference"><strong class="refLabel"><a href="#jm051256ob00028" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-jm051256ob00028_1"><span><span class="NLM_string-ref">Lee, S. H.; Rossky, P. J. A Comparison of the Structure and Dynamics of Liquid Water at Hydrophobic and Hydrophilic SurfacesA Molecular-Dynamics Simulation Study. <i>  J. Chem. Phys.</i><b>1994</b>, <i>100</i>, 3334−3345. Lee, C. Y.; McCammon, J. A.; Rossky, P. J. The Structure of Liquid Water at an Extended Hydrophobic Surface. <i>J. Chem. Phys.</i><b>1984</b>, <i>80</i>, 4448−4455.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=100&publication_year=1994&pages=3345&author=S.+H.+Lee&author=P.+J+Rossky&title=A+Comparison+of+the+Structure+and+Dynamics+of+Liquid+Water+at+Hydrophobic+and+Hydrophilic+SurfacesA+Molecular-Dynamics+Simulation+Study"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm051256ob00028_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DS.%2BH.%26atitle%3DA%2520Comparison%2520of%2520the%2520Structure%2520and%2520Dynamics%2520of%2520Liquid%2520Water%2520at%2520Hydrophobic%2520and%2520Hydrophilic%2520SurfacesA%2520Molecular-Dynamics%2520Simulation%2520Study%26jtitle%3DJ.%2520Chem.%2520Phys.%26date%3D1994%26volume%3D100%26spage%3D3345" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm051256ob00029"><div class="reference"><strong class="refLabel"><a href="#jm051256ob00029" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-jm051256ob00029_1"><span><span class="NLM_string-ref">Sharp, K. A.; Nicholls, A.; Fine, R. F.; Honig, B. Reconciling the Magnitude of the Microscopic and Macroscopic Hydrophobic Effects. <i>  Science</i><b>1991</b>, <i>252</i>, 106−109.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=252&publication_year=1991&pages=109&author=K.+A.+Sharp&author=A.+Nicholls&author=R.+F.+Fine&author=B+Honig&title=Reconciling+the+Magnitude+of+the+Microscopic+and+Macroscopic+Hydrophobic+Effects"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm051256ob00029_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSharp%26aufirst%3DK.%2BA.%26atitle%3DReconciling%2520the%2520Magnitude%2520of%2520the%2520Microscopic%2520and%2520Macroscopic%2520Hydrophobic%2520Effects%26jtitle%3DScience%26date%3D1991%26volume%3D252%26spage%3D109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm051256ob00030"><div class="reference"><div class="NLM_citation" id="rightTab-jm051256ob00030_1"><span><span class="NLM_string-ref">Regan, J.; Breitfelder, S.; Cirillo, P.; Gilmore, T.; Graham, A. G.; Hickey, E.; Klaus, B.; Madwed, J.; Moriak, M.; Moss, N.; Pargellis, C.; Pay, S.; Proto, A.; Swinamer, A.; Tong, L.; Torcellini, C. Pyrazole Urea-Based Inhibitors of p38 MAP Kinase:  From Lead Compound to Clinical Candidate. <i>  J. Med. Chem.</i><b>2002</b>, <i>45</i>, 2994−3008.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2002&pages=3008&author=J.+Regan&author=S.+Breitfelder&author=P.+Cirillo&author=T.+Gilmore&author=A.+G.+Graham&author=E.+Hickey&author=B.+Klaus&author=J.+Madwed&author=M.+Moriak&author=N.+Moss&author=C.+Pargellis&author=S.+Pay&author=A.+Proto&author=A.+Swinamer&author=L.+Tong&author=C+Torcellini&title=Pyrazole+Urea-Based+Inhibitors+of+p38+MAP+Kinase%3A%E2%80%89+From+Lead+Compound+to+Clinical+Candidate"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm051256ob00030_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DRegan%26aufirst%3DJ.%26atitle%3DPyrazole%2520Urea-Based%2520Inhibitors%2520of%2520p38%2520MAP%2520Kinase%253A%25E2%2580%2589%2520From%2520Lead%2520Compound%2520to%2520Clinical%2520Candidate%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2002%26volume%3D45%26spage%3D3008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm051256ob00031"><div class="reference"><strong class="refLabel"><a href="#jm051256ob00031" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-jm051256ob00031_1"><span><span class="NLM_string-ref">Hendsch, Z. S.; Tidor, B. Do Salt Bridges Stabilize ProteinsA Continuum Electrostatic Analysis. <i>  Protein Sci.</i><b>1994</b>, <i>3</i>, 211−226.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=1994&pages=226&author=Z.+S.+Hendsch&author=B+Tidor&title=Do+Salt+Bridges+Stabilize+ProteinsA+Continuum+Electrostatic+Analysis"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm051256ob00031_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DHendsch%26aufirst%3DZ.%2BS.%26atitle%3DDo%2520Salt%2520Bridges%2520Stabilize%2520ProteinsA%2520Continuum%2520Electrostatic%2520Analysis%26jtitle%3DProtein%2520Sci.%26date%3D1994%26volume%3D3%26spage%3D226" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm051256ob00032"><div class="reference"><strong class="refLabel"><a href="#jm051256ob00032" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-jm051256ob00032_1"><span><span class="NLM_string-ref">Waldburger, C. D.; Schildbach, J. F.; Sauer, R. T. Are Buried Salt Bridges Important for Protein Stability and Conformational Specificity. <i>  Nat. Struct. </i><i>Biol.</i><b>1995</b>, <i>2</i>, 122−128.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=1995&pages=128&author=C.+D.+Waldburger&author=J.+F.+Schildbach&author=R.+T+Sauer&title=Are+Buried+Salt+Bridges+Important+for+Protein+Stability+and+Conformational+Specificity"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm051256ob00032_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DWaldburger%26aufirst%3DC.%2BD.%26atitle%3DAre%2520Buried%2520Salt%2520Bridges%2520Important%2520for%2520Protein%2520Stability%2520and%2520Conformational%2520Specificity%26jtitle%3DNat.%2520Struct.%2520Biol.%26date%3D1995%26volume%3D2%26spage%3D128" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm051256ob00033"><div class="reference"><strong class="refLabel"><a href="#jm051256ob00033" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-jm051256ob00033_1"><span><span class="NLM_string-ref">Marqusee, S.; Sauer, R. T. Contributions of a Hydrogen-Bond Salt Bridge Network to the Stability of Secondary and Tertiary Structure in Lambda-Repressor. <i>  Protein Sci.</i><b>1994</b>, <i>3</i>, 2217−2225.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=1994&pages=2225&author=S.+Marqusee&author=R.+T+Sauer&title=Contributions+of+a+Hydrogen-Bond+Salt+Bridge+Network+to+the+Stability+of+Secondary+and+Tertiary+Structure+in+Lambda-Repressor"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm051256ob00033_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMarqusee%26aufirst%3DS.%26atitle%3DContributions%2520of%2520a%2520Hydrogen-Bond%2520Salt%2520Bridge%2520Network%2520to%2520the%2520Stability%2520of%2520Secondary%2520and%2520Tertiary%2520Structure%2520in%2520Lambda-Repressor%26jtitle%3DProtein%2520Sci.%26date%3D1994%26volume%3D3%26spage%3D2225" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm051256ob00034"><div class="reference"><strong class="refLabel"><a href="#jm051256ob00034" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-jm051256ob00034_1"><span><span class="NLM_string-ref">Luo, R.; David, L.; Hung, H.; Devaney, J.; Gilson, M. K. Strength of Solvent-Exposed Salt-Bridges. Strength of Solvent-Exposed Salt-Bridges. <i>  J. Phys. Chem. B</i><b>1999</b>, <i>103</i>, 727−736.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=103&publication_year=1999&pages=736&author=R.+Luo&author=L.+David&author=H.+Hung&author=J.+Devaney&author=M.+K+Gilson&title=Strength+of+Solvent-Exposed+Salt-Bridges.+Strength+of+Solvent-Exposed+Salt-Bridges"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm051256ob00034_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DLuo%26aufirst%3DR.%26atitle%3DStrength%2520of%2520Solvent-Exposed%2520Salt-Bridges.%2520Strength%2520of%2520Solvent-Exposed%2520Salt-Bridges%26jtitle%3DJ.%2520Phys.%2520Chem.%2520B%26date%3D1999%26volume%3D103%26spage%3D736" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm051256ob00035"><div class="reference"><strong class="refLabel"><a href="#jm051256ob00035" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-jm051256ob00035_1"><span><span class="NLM_string-ref">Krammer, A.; Kirchhoff, P. D.; Jiang, X.; Venkatachalam, C. M.; Waldman, M. LigScore:  A Novel Scoring Function for Predicting Binding Affinities. <i>  J. Mol. Graphics Modell.</i><b>2005</b>, <i>23</i>, 395−407.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+Krammer%2C+A.%3B+Kirchhoff%2C+P.+D.%3B+Jiang%2C+X.%3B+Venkatachalam%2C+C.+M.%3B+Waldman%2C+M.+LigScore%3A%E2%80%89+A+Novel+Scoring+Function+for+Predicting+Binding+Affinities.+J.+Mol.+Graphics+Modell.+2005%2C+23%2C+395%E2%88%92407.+Krammer%2C+A.%3B+Kirchhoff%2C+P.+D.%3B+Jiang%2C+X.%3B+Venkatachalam%2C+C.+M.%3B+Waldman%2C+M.+LigScore%3A%E2%80%89+A+Novel+Scoring+Function+for+Predicting+Binding+Affinities.+J.+Mol.+Graphics+Modell.2005%2C+23%2C+395%E2%88%92407."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm051256ob00035_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DKrammer%26aufirst%3DA.%26jtitle%3DJ.%2520Mol.%2520Graphics%2520Modell.%26date%3D2005%26volume%3D23%26spage%3D407" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm051256ob00036"><div class="reference"><strong class="refLabel"><a href="#jm051256ob00036" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-jm051256ob00036_1"><span><span class="NLM_string-ref">Wei, B. Q. Q.; Baase, W. A.; Weaver, L. H.; Matthews, B. W.; Shoichet, B. K. A Model Binding Site for Testing Scoring Functions in Molecular Docking. <i>  J. Mol. Biol.</i><b>2002</b>, <i>322</i>, 339−355.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=322&publication_year=2002&pages=355&author=B.+Q.+Q.+Wei&author=W.+A.+Baase&author=L.+H.+Weaver&author=B.+W.+Matthews&author=B.+K+Shoichet&title=A+Model+Binding+Site+for+Testing+Scoring+Functions+in+Molecular+Docking"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm051256ob00036_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DWei%26aufirst%3DB.%2BQ.%2BQ.%26atitle%3DA%2520Model%2520Binding%2520Site%2520for%2520Testing%2520Scoring%2520Functions%2520in%2520Molecular%2520Docking%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2002%26volume%3D322%26spage%3D355" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm051256ob00037"><div class="reference"><strong class="refLabel"><a href="#jm051256ob00037" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-jm051256ob00037_1"><span><span class="NLM_string-ref">Perola, E.; Charifson, P. S. Conformational Analysis of Drug-Like Molecules Bound to Proteins:  An Extensive Study of Ligand Reorganization upon Binding. <i>  J. Med. Chem.</i><b>2004</b>, <i>47</i>, 2499−2510.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=2510&author=E.+Perola&author=P.+S+Charifson&title=Conformational+Analysis+of+Drug-Like+Molecules+Bound+to+Proteins%3A%E2%80%89+An+Extensive+Study+of+Ligand+Reorganization+upon+Binding"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm051256ob00037_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DPerola%26aufirst%3DE.%26atitle%3DConformational%2520Analysis%2520of%2520Drug-Like%2520Molecules%2520Bound%2520to%2520Proteins%253A%25E2%2580%2589%2520An%2520Extensive%2520Study%2520of%2520Ligand%2520Reorganization%2520upon%2520Binding%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D2510" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm051256ob00038"><div class="reference"><strong class="refLabel"><a href="#jm051256ob00038" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-jm051256ob00038_1"><span><span class="NLM_string-ref">Halgren, T. A. MMFF VI. MMFF94s Option for Energy Minimization Studies. <i>  J. Comput. Chem. </i><b>1999</b>, <i>20</i>, 720−729.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+Halgren%2C+T.+A.+MMFF+VI.+MMFF94s+Option+for+Energy+Minimization+Studies.+J.+Comput.+Chem.+1999%2C+20%2C+720%E2%88%92729.+Halgren%2C+T.+A.+MMFF+VI.+MMFF94s+Option+for+Energy+Minimization+Studies.+J.+Comput.+Chem.+1999%2C+20%2C+720%E2%88%92729."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm051256ob00038_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DHalgren%26aufirst%3DT.%2BA.%2BMMFF%2BVI.%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D1999%26volume%3D20%26spage%3D729" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm051256ob00039"><div class="reference"><strong class="refLabel"><a href="#jm051256ob00039" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-jm051256ob00039_1"><span><span class="NLM_string-ref">Cho, A. E.; Guallar, V.; Berne, B. J.; Friesner, R. A. Importance of Accurate Charges in Molecular Docking:  Quantum Mechanical/Molecular Mechanical (QM/MM) Approach. <i>  J. Comput. Chem. </i><b>2005</b>, <i>26</i>, 915−931.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2005&pages=931&author=A.+E.+Cho&author=V.+Guallar&author=B.+J.+Berne&author=R.+A+Friesner&title=Importance+of+Accurate+Charges+in+Molecular+Docking%3A%E2%80%89+Quantum+Mechanical%2FMolecular+Mechanical+%28QM%2FMM%29+Approach"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm051256ob00039_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DCho%26aufirst%3DA.%2BE.%26atitle%3DImportance%2520of%2520Accurate%2520Charges%2520in%2520Molecular%2520Docking%253A%25E2%2580%2589%2520Quantum%2520Mechanical%252FMolecular%2520Mechanical%2520%2528QM%252FMM%2529%2520Approach%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2005%26volume%3D26%26spage%3D931" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm051256ob00040"><div class="reference"><div class="NLM_citation" id="rightTab-jm051256ob00040_1"><span><span class="NLM_string-ref"><i>Schrödinger User Manuals</i>, Glide v3.0; Schrödinger, L.L.C.:  New York, NY, 1994.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=1994&author=+Schr%C3%B6dinger+User+Manuals&title=Glide+v3.0"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm051256ob00040_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DSchr%25C3%25B6dinger%2BUser%2BManuals%26btitle%3DGlide%2520v3.0%26pub%3DNY%26date%3D1994" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm051256ob00041"><div class="reference"><strong class="refLabel"><a href="#jm051256ob00041" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-jm051256ob00041_1"><span><span class="NLM_string-ref"><i>QikProp v2.3</i>; Schrödinger, L.L.C.:  New York, NY, 2005.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+QikProp+v2.3%3B+Schr%C3%B6dinger%2C+L.L.C.%3A%E2%80%89+New+York%2C+NY%2C+2005.+QikProp+v2.3%3B+Schr%C3%B6dinger%2C+L.L.C.%3A%E2%80%89+New+York%2C+NY%2C+2005."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm051256ob00041_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dother%26sid%3Dliteratum%253Aachs" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm051256ob00042"><div class="reference"><strong class="refLabel"><a href="#jm051256ob00042" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-jm051256ob00042_1"><span><span class="NLM_string-ref">Wang, R.; Fang, X.; Lu, Y.; Wang, S. The PDBbind Database:  Collection of Binding Affinities for Protein−Ligand Complexes with Known Three-Dimensional Structures. <i>  J. Med. Chem.</i><b>2004</b>, <i>47</i>, 2977−2980.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=2980&author=R.+Wang&author=X.+Fang&author=Y.+Lu&author=S+Wang&title=The+PDBbind+Database%3A%E2%80%89+Collection+of+Binding+Affinities+for+Protein%E2%88%92Ligand+Complexes+with+Known+Three-Dimensional+Structures"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm051256ob00042_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DR.%26atitle%3DThe%2520PDBbind%2520Database%253A%25E2%2580%2589%2520Collection%2520of%2520Binding%2520Affinities%2520for%2520Protein%25E2%2588%2592Ligand%2520Complexes%2520with%2520Known%2520Three-Dimensional%2520Structures%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D2980" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1kv2" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1kv2','PDB','1kv2'); return false;">PDB: 1kv2</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1bl7" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1bl7','PDB','1bl7'); return false;">PDB: 1bl7</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1aq1" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1aq1','PDB','1aq1'); return false;">PDB: 1aq1</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1stp" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1stp','PDB','1stp'); return false;">PDB: 1stp</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1ef3" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1ef3','PDB','1ef3'); return false;">PDB: 1ef3</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1bji" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1bji','PDB','1bji'); return false;">PDB: 1bji</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3ert" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3ert','PDB','3ert'); return false;">PDB: 3ert</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1rt1" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1rt1','PDB','1rt1'); return false;">PDB: 1rt1</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1cx2" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1cx2','PDB','1cx2'); return false;">PDB: 1cx2</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1ett" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1ett','PDB','1ett'); return false;">PDB: 1ett</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1hpx" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1hpx','PDB','1hpx'); return false;">PDB: 1hpx</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1fjs" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1fjs','PDB','1fjs'); return false;">PDB: 1fjs</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1e66" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1e66','PDB','1e66'); return false;">PDB: 1e66</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1lpk" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1lpk','PDB','1lpk'); return false;">PDB: 1lpk</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="NLM_notes" id="rightTab-"><div id="" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i32"><a href="/doi/suppl/10.1021%2Fjm051256o">Supporting Information Available</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_46443" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_46443" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">Detailed descriptions of the algorithms used in hydrophobic enclosure scoring and in scaling special neutral−neutral hydrogen-bond motif rewards. References to experimental binding affinities for all test and training set ligands are also included. This material is available free of charge via the Internet at <a class="ext-link" href="http://pubs.acs.org" target="_blank">http://pubs.acs.org</a>. </div></div></div><div class="article_supporting-info" id=""><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/jm051256o/suppl_file/jm051256osi20060602_023733.pdf">jm051256osi20060602_023733.pdf (112.11 kb)</a></li></ul></div></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Fjm051256o&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Barticle%3Aarticle%3A10.1021%2Fjm051256o%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2006.49.issue-21%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/jm051256o" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                799KB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"677b0f164d973d36","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
