
<html lang="en"     class="pb-page"  data-request-id="4b74abd4-a99c-45ea-9c04-966375279d2b"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;website:website:acspubs;issue:issue:10.1021/jmcmar.2021.64.issue-13;pageGroup:string:Publication Pages;subPage:string:Full Text;article:article:10.1021/acs.jmedchem.1c00373"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Discovery of a Brigatinib Degrader SIAIS164018 with Destroying Metastasis-Related Oncoproteins and a Reshuffling Kinome Profile" /></meta><meta name="dc.Creator" content="Chaowei  Ren" /></meta><meta name="dc.Creator" content="Ning  Sun" /></meta><meta name="dc.Creator" content="Haixia  Liu" /></meta><meta name="dc.Creator" content="Ying  Kong" /></meta><meta name="dc.Creator" content="Renhong  Sun" /></meta><meta name="dc.Creator" content="Xing  Qiu" /></meta><meta name="dc.Creator" content="Jinju  Chen" /></meta><meta name="dc.Creator" content="Yan  Li" /></meta><meta name="dc.Creator" content="Jianshui  Zhang" /></meta><meta name="dc.Creator" content="Yuedong  Zhou" /></meta><meta name="dc.Creator" content="Hui  Zhong" /></meta><meta name="dc.Creator" content="Qianqian  Yin" /></meta><meta name="dc.Creator" content="Xiaoling  Song" /></meta><meta name="dc.Creator" content="Xiaobao  Yang" /></meta><meta name="dc.Creator" content="Biao  Jiang" /></meta><meta name="dc.Description" content="Proteolysis-targeting chimera (PROTAC) is an attractive technology in drug discovery. Canonically, targets act as a basic starting point in the most previous PROTAC design. Here, we designed degrad..." /></meta><meta name="Description" content="Proteolysis-targeting chimera (PROTAC) is an attractive technology in drug discovery. Canonically, targets act as a basic starting point in the most previous PROTAC design. Here, we designed degrad..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="June 17, 2021" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.1c00373" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2021 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.1c00373" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.1c00373" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.1c00373" /></link>
        
    
    

<title>Discovery of a Brigatinib Degrader SIAIS164018 with Destroying Metastasis-Related Oncoproteins and a Reshuffling Kinome Profile | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.1c00373" /></meta><meta property="og:title" content="Discovery of a Brigatinib Degrader SIAIS164018 with Destroying Metastasis-Related Oncoproteins and a Reshuffling Kinome Profile" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-13/acs.jmedchem.1c00373/20210629/images/large/jm1c00373_0012.jpeg" /></meta><meta property="og:description" content="Proteolysis-targeting chimera (PROTAC) is an attractive technology in drug discovery. Canonically, targets act as a basic starting point in the most previous PROTAC design. Here, we designed degraders considering from the view of clinical benefits. With this novel design, Brigatinib was turned into a degrader SIAIS164018 and endowed with unique features. First, SIAIS164018 could degrade not only ALK fusion proteins in activating or G1202R-mutated form but also mutant EGFR with L858R + T790M, which are two most important targets in non-small-cell lung cancer. Second, SIAIS164018 strongly inhibited cell migration and invasion of Calu-1 and MDA-MB-231. Third and surprisingly, SIAIS164018 degrades several important oncoproteins involved in metastasis such as FAK, PYK2, and PTK6. Interestingly, SIAIS164018 reshuffled the kinome ranking profile when compared to Brigatinib. Finally, SIAIS164018 is orally bioavailable and well tolerated in vivo. SIAIS164018 is an enlightening degrader for us to excavate the charm of protein degradation." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.1c00373"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.1c00373">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-14/achre4.2021.54.issue-14/20210720/achre4.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-29/aamick.2021.13.issue-29/20210728/aamick.2021.13.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-14/acncdm.2021.12.issue-14/20210721/acncdm.2021.12.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-7/amlcef.2021.3.issue-7/20210705/amlcef.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-29/acsodf.2021.6.issue-29/20210727/acsodf.2021.6.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-29/ascecg.2021.9.issue-29/20210726/ascecg.2021.9.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-29/ancham.2021.93.issue-29/20210727/ancham.2021.93.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-29/bichaw.2021.60.issue-29/20210727/bichaw.2021.60.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-27/cgeabj.2021.99.issue-27/20210726-01/cgeabj.2021.99.issue-27.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-7/cgdefu.2021.21.issue-7/20210707/cgdefu.2021.21.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-14/esthag.2021.55.issue-14/20210720/esthag.2021.55.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-29/iecred.2021.60.issue-29/20210728/iecred.2021.60.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-14/inocaj.2021.60.issue-14/20210719/inocaj.2021.60.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-29/jafcau.2021.69.issue-29/20210728/jafcau.2021.69.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-29/jacsat.2021.143.issue-29/20210728/jacsat.2021.143.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-7/jamsef.2021.32.issue-7/20210707/jamsef.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-29/langd5.2021.37.issue-29/20210727/langd5.2021.37.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-7/mpohbp.2021.18.issue-7/20210705/mpohbp.2021.18.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.1c00373&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.1c00373&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.1c00373&amp;href=/doi/10.1021/acs.jmedchem.1c00373" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2021</span><span class="cit-fg-volume">, 64</span><span class="cit-fg-issue">, 13</span><span class="cit-fg-pageRange">, 9152-9165</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/64/13" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.1c00362" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.1c00384" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Discovery of a Brigatinib Degrader SIAIS164018 with Destroying Metastasis-Related Oncoproteins and a Reshuffling Kinome Profile</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Chaowei Ren</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Chaowei Ren</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Shanghai Institute for Advanced Immunochemical Studies, ShanghaiTech University, Shanghai 201210, China</div><div class="loa-info-affiliations-info">School of Life Science and Technology, ShanghaiTech University, Shanghai 201210, China</div><div class="loa-info-affiliations-info">University of Chinese Academy of Sciences, Beijing 100049, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Chaowei++Ren">Chaowei Ren</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Ning Sun</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Ning Sun</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Shanghai Institute for Advanced Immunochemical Studies, ShanghaiTech University, Shanghai 201210, China</div><div class="loa-info-affiliations-info">Shanghai Institutes for Biological Sciences, Chinese Academy of Sciences, Shanghai 200031, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Ning++Sun">Ning Sun</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Haixia Liu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Haixia Liu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Shanghai Institute for Advanced Immunochemical Studies, ShanghaiTech University, Shanghai 201210, China</div><div class="loa-info-affiliations-info">School of Physical Science and Technology, ShanghaiTech University, Shanghai 201210, China</div><div class="loa-info-affiliations-info">University of Chinese Academy of Sciences, Beijing 100049, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Haixia++Liu">Haixia Liu</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Ying Kong</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Ying Kong</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Jing Medicine Technology (Shanghai), Ltd., Y Building, Shanghai 201210, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Ying++Kong">Ying Kong</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Renhong Sun</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Renhong Sun</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Shanghai Institute for Advanced Immunochemical Studies, ShanghaiTech University, Shanghai 201210, China</div><div class="loa-info-affiliations-info">Shanghai Institutes for Biological Sciences, Chinese Academy of Sciences, Shanghai 200031, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Renhong++Sun">Renhong Sun</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Xing Qiu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Xing Qiu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">CAS Key Laboratory of Synthetic Chemistry of Natural Substances, Shanghai Institute of Organic Chemistry, Chinese Academy of Sciences, Shanghai 200032, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Xing++Qiu">Xing Qiu</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Jinju Chen</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jinju Chen</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Jing Medicine Technology (Shanghai), Ltd., Y Building, Shanghai 201210, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jinju++Chen">Jinju Chen</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Yan Li</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Yan Li</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Jing Medicine Technology (Shanghai), Ltd., Y Building, Shanghai 201210, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Yan++Li">Yan Li</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Jianshui Zhang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jianshui Zhang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Jing Medicine Technology (Shanghai), Ltd., Y Building, Shanghai 201210, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jianshui++Zhang">Jianshui Zhang</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Yuedong Zhou</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Yuedong Zhou</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Jing Medicine Technology (Shanghai), Ltd., Y Building, Shanghai 201210, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Yuedong++Zhou">Yuedong Zhou</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Hui Zhong</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Hui Zhong</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Shanghai Institute for Advanced Immunochemical Studies, ShanghaiTech University, Shanghai 201210, China</div><div class="loa-info-affiliations-info">School of Pharmaceutical Engineering and Life Science, Changzhou University, Changzhou, Jiangsu 213164, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Hui++Zhong">Hui Zhong</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Qianqian Yin</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Qianqian Yin</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Shanghai Institute for Advanced Immunochemical Studies, ShanghaiTech University, Shanghai 201210, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Qianqian++Yin">Qianqian Yin</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Xiaoling Song</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Xiaoling Song</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Shanghai Institute for Advanced Immunochemical Studies, ShanghaiTech University, Shanghai 201210, China</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#acdfc3c2cbd4c0ecdfc4cdc2cbc4cdc5d8c9cfc482c9c8d982cfc2"><span class="__cf_email__" data-cfemail="b4c7dbdad3ccd8f4c7dcd5dad3dcd5ddc0d1d7dc9ad1d0c19ad7da">[email protected]</span></a></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Xiaoling++Song">Xiaoling Song</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Xiaobao Yang</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Xiaobao Yang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Shanghai Institute for Advanced Immunochemical Studies, ShanghaiTech University, Shanghai 201210, China</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#364e5f57595457594f59435851760704001855595b"><span class="__cf_email__" data-cfemail="631b0a020c01020c1a0c160d04235251554d000c0e">[email protected]</span></a>, <a href="/cdn-cgi/l/email-protection#8af3ebe4edf2e8caf9e2ebe4ede2ebe3feefe9e2a4efeeffa4e9e4"><span class="__cf_email__" data-cfemail="e59c848b829d87a5968d848b828d848c9180868dcb808190cb868b">[email protected]</span></a></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Xiaobao++Yang">Xiaobao Yang</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0001-5266-7673" title="Orcid link">https://orcid.org/0000-0001-5266-7673</a></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Biao Jiang</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Biao Jiang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Shanghai Institute for Advanced Immunochemical Studies, ShanghaiTech University, Shanghai 201210, China</div><div class="loa-info-affiliations-info">School of Physical Science and Technology, ShanghaiTech University, Shanghai 201210, China</div><div class="loa-info-affiliations-info">CAS Key Laboratory of Synthetic Chemistry of Natural Substances, Shanghai Institute of Organic Chemistry, Chinese Academy of Sciences, Shanghai 200032, China</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#28424149464f4a414947685b4049464f4049415c4d4b40064d4c5d064b46"><span class="__cf_email__" data-cfemail="355f5c545b52575c545a75465d545b525d545c4150565d1b5051401b565b">[email protected]</span></a></span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Biao++Jiang">Biao Jiang</a></span></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.1c00373&amp;href=/doi/10.1021%2Facs.jmedchem.1c00373" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2021</span></span><span class="cit-volume">, 64</span><span class="cit-issue">, 13</span><span class="cit-pageRange">, 9152–9165</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">June 17, 2021</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>1 March 2021</li><li><span class="item_label"><b>Published</b> online</span>17 June 2021</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 8 July 2021</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.1c00373" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.1c00373</a></div><div class="article_header-article-copyright"><strong>Copyright © 2021 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D9152%26pageCount%3D14%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DChaowei%2BRen%252C%2BNing%2BSun%252C%2BHaixia%2BLiu%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D64%26issueNum%3D13%26contentID%3Dacs.jmedchem.1c00373%26title%3DDiscovery%2Bof%2Ba%2BBrigatinib%2BDegrader%2BSIAIS164018%2Bwith%2BDestroying%2BMetastasis-Related%2BOncoproteins%2Band%2Ba%2BReshuffling%2BKinome%2BProfile%26numPages%3D14%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D9165%26publicationDate%3DJuly%2B2021">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.1c00373"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">1264</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val">-</div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.1c00373" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Discovery of a Brigatinib Degrader SIAIS164018 with Destroying Metastasis-Related Oncoproteins and a Reshuffling Kinome Profile&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Chaowei&quot;,&quot;last_name&quot;:&quot;Ren&quot;},{&quot;first_name&quot;:&quot;Ning&quot;,&quot;last_name&quot;:&quot;Sun&quot;},{&quot;first_name&quot;:&quot;Haixia&quot;,&quot;last_name&quot;:&quot;Liu&quot;},{&quot;first_name&quot;:&quot;Ying&quot;,&quot;last_name&quot;:&quot;Kong&quot;},{&quot;first_name&quot;:&quot;Renhong&quot;,&quot;last_name&quot;:&quot;Sun&quot;},{&quot;first_name&quot;:&quot;Xing&quot;,&quot;last_name&quot;:&quot;Qiu&quot;},{&quot;first_name&quot;:&quot;Jinju&quot;,&quot;last_name&quot;:&quot;Chen&quot;},{&quot;first_name&quot;:&quot;Yan&quot;,&quot;last_name&quot;:&quot;Li&quot;},{&quot;first_name&quot;:&quot;Jianshui&quot;,&quot;last_name&quot;:&quot;Zhang&quot;},{&quot;first_name&quot;:&quot;Yuedong&quot;,&quot;last_name&quot;:&quot;Zhou&quot;},{&quot;first_name&quot;:&quot;Hui&quot;,&quot;last_name&quot;:&quot;Zhong&quot;},{&quot;first_name&quot;:&quot;Qianqian&quot;,&quot;last_name&quot;:&quot;Yin&quot;},{&quot;first_name&quot;:&quot;Xiaoling&quot;,&quot;last_name&quot;:&quot;Song&quot;},{&quot;first_name&quot;:&quot;Xiaobao&quot;,&quot;last_name&quot;:&quot;Yang&quot;},{&quot;first_name&quot;:&quot;Biao&quot;,&quot;last_name&quot;:&quot;Jiang&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2021&quot;,&quot;month&quot;:&quot;06&quot;,&quot;day&quot;:&quot;17&quot;,&quot;issue&quot;:&quot;13&quot;,&quot;volume&quot;:&quot;64&quot;,&quot;pages&quot;:&quot;9152-9165&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.1c00373&quot;},&quot;abstract&quot;:&quot;Proteolysis-targeting chimera (PROTAC) is an attractive technology in drug discovery. Canonically, targets act as a basic starting point in the most previous PROTAC design. Here, we designed degraders considering from the view of clinical benefits. With this novel design, Brigatinib was turned into a degrader SIAIS164018 and endowed with unique features. First, SIAIS164018 could degrade not only ALK fusion proteins in activating or G1202R-mutated form but also mutant EGFR with L858R + T790M, which are two most important targets in non-small-cell lung cancer. Second, SIAIS164018 strongly inhibited cell migration and invasion of Calu-1 and MDA-MB-231. Third and surprisingly, SIAIS164018 degrades several important oncoproteins involved in metastasis such as FAK, PYK2, and PTK6. Interestingly, SIAIS164018 reshuffled the kinome ranking profile when compared to Brigatinib. Finally, SIAIS164018 is orally bioavailable and well tolerated in vivo. SIAIS164018 is an enlightening degrader for us to excavate the charm of &quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.1c00373&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.1c00373" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.1c00373&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.1c00373" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.1c00373&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.1c00373" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.1c00373&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.1c00373&amp;href=/doi/10.1021/acs.jmedchem.1c00373" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.1c00373" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.1c00373" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (8 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.1c00373&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.1c00373%26sid%3Dliteratum%253Aachs%26pmid%3D34138566%26genre%3Darticle%26aulast%3DRen%26date%3D2021%26atitle%3DDiscovery%2Bof%2Ba%2BBrigatinib%2BDegrader%2BSIAIS164018%2Bwith%2BDestroying%2BMetastasis-Related%2BOncoproteins%2Band%2Ba%2BReshuffling%2BKinome%2BProfile%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D64%26issue%3D13%26spage%3D9152%26epage%3D9165%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (5)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291781" title="Degradation">Degradation</a>,</li><li><a href="/action/doSearch?ConceptID=292258" title="Cancer">Cancer</a>,</li><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=291305" title="Cell physiology">Cell physiology</a>,</li><li><a href="/action/doSearch?ConceptID=291793" title="Inhibition">Inhibition</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/64/13" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-13/jmcmar.2021.64.issue-13/20210708/jmcmar.2021.64.issue-13.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-13/acs.jmedchem.1c00373/20210629/images/medium/jm1c00373_0012.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-13/acs.jmedchem.1c00373/20210629/images/large/jm1c00373_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00373&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Proteolysis-targeting chimera (PROTAC) is an attractive technology in drug discovery. Canonically, targets act as a basic starting point in the most previous PROTAC design. Here, we designed degraders considering from the view of clinical benefits. With this novel design, Brigatinib was turned into a degrader SIAIS164018 and endowed with unique features. First, SIAIS164018 could degrade not only ALK fusion proteins in activating or G1202R-mutated form but also mutant EGFR with L858R + T790M, which are two most important targets in non-small-cell lung cancer. Second, SIAIS164018 strongly inhibited cell migration and invasion of Calu-1 and MDA-MB-231. Third and surprisingly, SIAIS164018 degrades several important oncoproteins involved in metastasis such as FAK, PYK2, and PTK6. Interestingly, SIAIS164018 reshuffled the kinome ranking profile when compared to Brigatinib. Finally, SIAIS164018 is orally bioavailable and well tolerated in vivo. SIAIS164018 is an enlightening degrader for us to excavate the charm of protein degradation.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_23386" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_23386" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Proteolysis-targeting chimeras (PROTACs) are bifunctional molecules that induce protein degradation by recruiting targeted proteins to a specific E3 ligase.<a onclick="showRef(event, 'ref1 ref2'); return false;" href="javascript:void(0);" class="ref ref1 ref2">(1,2)</a> It shows many advantages over kinase inhibitors such as substoichiometry,<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> overcoming drug resistance,<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> and targeting undruggable targets.<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> Many targets have been proven to be degradable by turning the warheads into degraders.<a onclick="showRef(event, 'ref4 ref5 ref6 ref7 ref8 ref9 ref10 ref11 ref12 ref13 ref14 ref15 ref16 ref17 ref18 ref19'); return false;" href="javascript:void(0);" class="ref ref4 ref5 ref6 ref7 ref8 ref9 ref10 ref11 ref12 ref13 ref14 ref15 ref16 ref17 ref18 ref19">(4−19)</a> Due to its mechanism of action, PROTAC has attracted great attention in both academia and industry<a onclick="showRef(event, 'ref20 ref21'); return false;" href="javascript:void(0);" class="ref ref20 ref21">(20,21)</a> and has become an attractive therapeutic strategy in drug discovery in recent years. In 2019, the PROTAC field generated the first phase I clinical ARV-110 study,<a onclick="showRef(event, 'ref21 ref22'); return false;" href="javascript:void(0);" class="ref ref21 ref22">(21,22)</a> which is an orally bioavailable AR degrader. ARV-110 turned out to be safe and efficacious in phase I data reported at ASCO in May 2020. PROTACs will provide more opportunities in cancer therapy.</div><div class="NLM_p">However, canonically, the protein of interest acts as the starting points in designing PROTACs, which indicates that degrading a specific target is the fundamental purpose. Here, noncanonically, based on the clinical behavior and applications of inhibitors, we chose inhibitors and designed PROTAC oriented from clinical behaviors and studied if extra properties could be achieved.</div><div class="NLM_p">Among approved drugs in treating ALK- or EGFR-positive non-small-cell lung cancer (NSCLC), Brigatinib is the only drug that inhibits both ALK (anaplastic lymphoma kinase) and EGFR (epidermal growth factor receptor; especially EGFR L858R mutation).<a onclick="showRef(event, 'ref23 ref24 ref25'); return false;" href="javascript:void(0);" class="ref ref23 ref24 ref25">(23−25)</a> Additionally, Brigatinib has been studied clinically in advanced and metastatic NSCLC.<a onclick="showRef(event, 'ref26 ref27'); return false;" href="javascript:void(0);" class="ref ref26 ref27">(26,27)</a> In real-world treatment, Brigatinib is effective and tolerable in real-world practice regardless of prior treatment with first or other ALK inhibitors.<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a></div><div class="NLM_p last">In this study, considering the clinical benefits and target profile of Brigatinib, we explored Brigatinib-based degraders by recruiting Cereblon. It deserves to study if extra anti-cancer properties distinguishing from Brigatinib could be achieved. As a result, we successfully discovered an orally available degrader SIAIS164018 which degrades not only ALK or mutant EGFR but also oncoproteins involved in metastasis. In our finding, interestingly, SIAIS164018 also rearranged kinase selectivity, which is different from Brigatinib.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_67732" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_67732" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> SIAIS164018 Effectively Degrades ALK</h3><div class="NLM_p">Previously, we had designed Brigatinib PROTACs based on VHL ligands.<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> Here, Brigatinib PROTACs based on CRBN ligands had been designed and evaluated in ALK-positive cell line SR (NPM-ALK fusion). From the crystal structure of ALK bound by Brigatinib, Brigatinib CRBN PROTACs were generated from linking suitable position (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>). We developed a structure–activity relationship study to optimize the ALK PROTAC molecules. Various PEG and alkyl linkers with different lengths were incorporated to connect the ALK inhibitor precursor Brigatinib C, with the CRBN ligand pomalidomide (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00373/suppl_file/jm1c00373_si_001.pdf" class="ext-link">Figure S1</a>). Compounds <b>1–11</b> were synthesized using the synthetic routes shown in <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>. The preparation of compound Brigatinib C was synthesized according to ref <a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a>, and a linker library of CRBN E3 ligase ligand pomalidomide was built according to ref <a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a>. Finally, compound Brigatinib C condensed with various CRBN ligand pomalidomide-based linkers <b>s-1</b> to generate the corresponding ALK degraders <b>1–11</b> and negative compound <b>12</b> (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>). Based on these compounds, we investigated the proliferation inhibition results, where compound <b>6</b> was selected as the most potent compound with the least IC<sub>50</sub> (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). From the western blots of ALK, compound <b>6</b> (SIAIS164018) degraded ALK significantly at 10 nM and it had been identified as a potent Brigatinib-based degrader (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>a). The Hook effect was also observed at 10 μM (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00373/suppl_file/jm1c00373_si_001.pdf" class="ext-link">Figure S2</a>). N-Methylated modification in the glutarimide moiety of degraders was generally used as non-active compounds because methylated CRBN ligands could not bind to the CRBN ligand<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>b). After treating SR cells for 16 h, the negative compound 018NC could not degrade the ALK protein even at 500 nM, which indicates that degradation could not proceed without the recruitment of E3 ligase (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>c). SIAIS164018 significantly inhibited SR cell proliferation with an IC<sub>50</sub> value of 2 nM while that for Brigatinib analogue was 6.8 nM and for 018NC was 16.3 nM (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>d).</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-13/acs.jmedchem.1c00373/20210629/images/medium/jm1c00373_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-13/acs.jmedchem.1c00373/20210629/images/large/jm1c00373_0002.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Crystal structure of ALK bound by Brigatinib (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6MX8">6MX8</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-13/acs.jmedchem.1c00373/20210629/images/large/jm1c00373_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00373&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-13/acs.jmedchem.1c00373/20210629/images/medium/jm1c00373_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-13/acs.jmedchem.1c00373/20210629/images/large/jm1c00373_0003.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Degradation of ALK of synthesized ALK degraders. (a) Western blots of Brigatinib CRBN degraders treated for 16 h. (b) Structure of compound <b>6</b> (SIAIS164018) and 018NC. (c) Western blots of ALK after treatment with 018NC. (d) Growth inhibition of Brigatinib analogue, SIAIS164018, and 018NC hours (error bars are shown).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-13/acs.jmedchem.1c00373/20210629/images/large/jm1c00373_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00373&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="sch1" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-13/acs.jmedchem.1c00373/20210629/images/medium/jm1c00373_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-13/acs.jmedchem.1c00373/20210629/images/large/jm1c00373_0009.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Syntheses of Compounds <b>1–12</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-13/acs.jmedchem.1c00373/20210629/images/large/jm1c00373_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00373&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Research on Structure–Activity Relationship among Brigatinib CRBN Degraders</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-13/acs.jmedchem.1c00373/20210629/images/medium/jm1c00373_0010.gif" alt="" id="GRAPHIC-d7e583-autogenerated" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-13/acs.jmedchem.1c00373/20210629/images/medium/jm1c00373_0011.gif" alt="" id="gr9" /></img><div></div></div><div class="NLM_p">Degradation of ALK protein started at 8 h after treatment and reached to maximum after a 24 h treatment under the concentration of 100 nM SIAIS164018 (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>a). It indicated that longer time beyond the mostly used 16 h presented more efficacious degradation. To confirm the degradation pathway through proteasome, cells were pretreated with the proteasome inhibitor MG132 or the NEDD8 inhibitor MLN4924 for 4 h, and the degradation effect on ALK protein was blocked obviously (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>a). The durable degradation property of PROTAC has been reported.<a onclick="showRef(event, 'ref15 ref16 ref32 ref33'); return false;" href="javascript:void(0);" class="ref ref15 ref16 ref32 ref33">(15,16,32,33)</a> In our study, the degradation effect of SIAIS164018 on the ALK protein was also durable as observed during 72 h after a pulse treatment of cells with 100 nM SIAIS164018 (about 10 times of DC<sub>50</sub> to ALK) and followed by washout (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>b). After a further 5 days of culture before washout, the proliferation of SR cells was still inhibited by SIAIS164018 but not Brigatinib (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>c). SIAIS164018 induced the degradation of both NPM-ALK and EML4-ALK proteins (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>d and <a href="/doi/suppl/10.1021/acs.jmedchem.1c00373/suppl_file/jm1c00373_si_001.pdf" class="ext-link">S3A</a>). Moreover, pALK was blocked significantly at 1 nM SIAIS164018 with a ratio of pALK to 0.18. pSTAT3 was obviously inhibited at 10 nM SIAIS164018, similar to Brigatinib in SR. However, pSTAT3 did not show inhibition in NCI-H2228 after treating with SIAIS164018 or Brigatinib (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00373/suppl_file/jm1c00373_si_001.pdf" class="ext-link">Figure S3A</a>).</div><figure id="fig3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-13/acs.jmedchem.1c00373/20210629/images/medium/jm1c00373_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-13/acs.jmedchem.1c00373/20210629/images/large/jm1c00373_0004.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Inhibition and degradation properties of SIAIS164018 in ALK-positive cell line. (a) Degradation of ALK at different times under 100 nM SIAIS164018. SR was pretreated under 2.5 μM MG (MG132) or 0.5 μM MLN (MLN4924) for 4 h; then, SR was treated for 16 h with compounds. DC<sub>50</sub> was the concentration at which degraders induced 50% protein degradation. (b) Western blots: when washing out 100 nM SIAIS164018 after a 24 h treatment, then after 24, 48, and 72 h, the ALK protein was detected in the SR cell line. (c) The SR cell was treated for 5 days at 10 nM SIAIS164018 or Brigatinib, and then the cell density was recorded after removal of SIAIS164018 at 7 days and 9 days (error bars are shown). (d) Downstream signaling pathways after treatment of SIAIS164018 in the SR cell line. Ratio of pALK or ALK was normalized to GAPDH and quantified using ImageJ. (e) Degradation of ALK(G1202R) at indicated concentrations and inhibition cell proliferation in 293T-ALK(G1202R) cell line after a 72 h treatment (error bars are shown). (f) Kinase IC<sub>50</sub> of ALK and ALK (G1202R).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-13/acs.jmedchem.1c00373/20210629/images/large/jm1c00373_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00373&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">As a conclusion, SIAIS164018-degraded ALK proteins showed long-lasting inhibition effect. Additionally, SIAIS164018 showed superiority in inhibiting pALK than Brigatinib in ALK-positive cell line SR.</div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i7" class="anchor-spacer"></div><h3 class="article-section__title" id="_i7"> SIAIS164018 Degrades ALK and EGFR Drug-Resistant Mutants</h3><div class="NLM_p">Overcoming drug resistances is a goal in drug discovery. Acquired point mutations in cancer driver proteins are major reasons of drug resistance. In our study, SIAIS164018 degraded the G1202R-baring EML4-ALK fusion protein and the mutant EGFR protein with resistant mutations (L858R + T790M) at approximately 100 nM (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>e and <a href="/doi/suppl/10.1021/acs.jmedchem.1c00373/suppl_file/jm1c00373_si_001.pdf" class="ext-link">S3B</a>). It showed much better cell proliferation inhibition than Brigatinib did in ALK(G1202R) over-expressing 293T and EGFR expressing H1975 cell lines, and the IC<sub>50</sub> values in these two cells were 21 and 42 nM, respectively, which were much lower than that of Brigatinib.</div><div class="NLM_p last">To examine whether turning Brigatinib into SIAIS164018 changed the kinetics for ALK kinase binding, <i>in vitro</i> kinase assay was performed. It showed that the binding affinity of ALK or ALK(G1202R) protein was slightly weaker for SIAIS164018 than that for Brigatinib (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>g). Thus, it indicated that degradation of ALK could be achieved even if binding affinity was reduced when turning Brigatinib into SIAIS164018.</div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8"> SIAIS164018 Induces Multiple Phenotype Alterations</h3><div class="NLM_p">To explore more advantages of the degrader SIAIS164018 in the aspect of anti-cancer function besides cell proliferation, we examined the effects of SIAIS164018 on both ALK-positive and ALK-negative cell lines, and the results were listed as follows.</div><div class="NLM_p">First, SIAIS164018 induced a stronger G1 cell cycle arrest compared to Brigatinib. After treating ALK-positive ALCL cell line SR for 24 h or 48 h, both Brigatinib and SIAIS164018 induced G1 cell cycle arrest. Moreover, it showed that SIAIS164018 enhanced the cell cycle arrest much better than Brigatinib. In ALK-negative Calu-1 and MDA-MB-231 cells treated for 24 h or 48 h, 100 nM SIAIS164018 induced a significant G1 cell cycle arrest compared to dimethyl sulfoxide (DMSO), but Brigatinib did not have such effect (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>a). It showed no surprising apoptosis analyzed in Calu-1 and MDA-MB-231 cell lines (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00373/suppl_file/jm1c00373_si_001.pdf" class="ext-link">Figure S4</a>).</div><figure id="fig4" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-13/acs.jmedchem.1c00373/20210629/images/medium/jm1c00373_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-13/acs.jmedchem.1c00373/20210629/images/large/jm1c00373_0005.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Cell cycle, migration, and invasion differences induced by SIAIS164018. (a) Cell cycle analysis after 24 h or 48 h of treatment with 100 nM SIAIS164018 or Brigatinib in SR, Calu-1, or MDA-MB-231 cell line (error bars are shown. ** <i>p</i> value < 0.01; *** <i>p</i> value < 0.001; **** <i>p</i> value < 0.0001, <i>T</i> test). (b) Wound healing assay in Calu-1 after 12 h or 24 h of treatment with 100 nM SIAIS164018 or Brigatinib (error bars are shown). (c) Transwell invasion assay in Calu-1 GFP after 24 h of treatment with 100 nM SIAIS164018 or Brigatinib (error bars are shown, <i>N</i> = 15). (d) Quantified wound confluence ratio in MDA-MB-231 during 48 h of treatment with SIAIS164018 or Brigatinib under indicated concentrations and time scale (error bars are shown).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-13/acs.jmedchem.1c00373/20210629/images/large/jm1c00373_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00373&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Second, cancer metastasis often leads to treatment failure and cancer-related deaths.<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> To examine whether SIAIS164018 could enhance the behavior of anti-metastasis, we found that it inhibited migration or invasion in two highly metastatic cell lines: lung squamous cell Calu-1 and triple negative cancer cell line MDA-MB-231. SIAIS164018, not Brigatinib, significantly inhibited Calu-1 cell migration at 100 nM in wound healing assay (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>b). The wound width was dramatically wider when exposed to SIAIS164018. Furthermore, after being exposed to SIAIS164018, the invasive ability of Calu-1 GFP cells was also significantly inhibited in the transwell invasion assay (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>c). In a real-time migration assay in MDA-MB-231 (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>d), inhibition of migration was obviously observed at 10 nM under the presence of SIAIS164018. In comparison, at a concentration of 100 nM, Brigatinib has no effect on cell migration and the wound was almost healed as control. Although Brigatinib could inhibit migration of MDA-MB-231 at 500 nM, the wound confluence ratio was 56% while the wound confluence ratio was 48% exposed to 10 nM SIAIS164018.</div><div class="NLM_p last">At clinically achievable concentrations, the clinical usage of Brigatinib was less than 500 nM (<a href="https://www.accessdata.fda.gov/drugsatfda_docs/label/2017/208772lbl.pdf" class="extLink">https://www.accessdata.fda.gov/drugsatfda_docs/label/2017/208772lbl.pdf</a>). Brigatinib, although modulating FAK kinase activity, might not inhibit cancer cell migration significantly in clinical studies. Inhibition of brain metastasis by Brigatinib might be mostly contributed from penetrating the blood–brain barrier. However, turning Brigatinib into a degrader SIAIS164018 endowed the new compound with strong capability of inhibition of migration and invasion <i>in vitro</i>.</div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i9" class="anchor-spacer"></div><h3 class="article-section__title" id="_i9"> SIAIS164018 Induces Effective FAK and PYK2 Degradation</h3><div class="NLM_p">To investigate the reasons for multiple differences especially for the inhibition of migration and invasion between Brigatinib and SIAIS164018, we performed quantitative proteomics analysis in SR (7207 proteins were identified) and Calu-1 (6313 proteins were identified). Red points represented up-regulated proteins with p value less than 0.05; blue points represented down-regulated proteins with <i>p</i> value less than 0.05. Both red and blue points represented that the fold change between SIAIS164018 and Brigatinib was more than 2-fold (or log<sub>2</sub> FC > 1). Data showed that SIAIS164018 down-regulated the protein level of FAK, PYK2, FER, RSK1, and GAK in ALK-positive SR and ALK-negative Calu-1 cell lines (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>a). However, ALK was down-regulated compared to DMSO with fold change less than 0.5 times. Besides, ZFP91, also down-regulated, was the traditional substrate of iMiDs (immunomodulators). The abundance of these proteins was examined in different cell lines to validate the proteomic results. Western blotting validated that FAK, PYK2, FER, and RSK1 were down-regulated induced by SIAIS164018 treating for 16 h (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>b,c).</div><figure id="fig5" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-13/acs.jmedchem.1c00373/20210629/images/medium/jm1c00373_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-13/acs.jmedchem.1c00373/20210629/images/large/jm1c00373_0006.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. FAK and PYK2 degradation identified from proteomics in SR and Calu-1 cancer cells. (a) TMT-labeled proteomics data in SR and Calu-1 cell lines after treatment with DMSO, 100 nM Brigatinib, or 100 nM SIAIS164018. 7207 proteins were identified in SR and 6313 proteins were identified in Calu-1. Red dots: <i>p</i> value < 0.05 and log<sub>2</sub> FC > 1; blue dots: <i>p</i> value < 0.05 and log<sub>2</sub> FC < 1. The <i>p</i> value was calculated using <i>t</i>-test. (b) Western blotting of PYK2, FER, and ALK in SR cell line after 24 h of treatment with different compounds. (c) Western blotting of GAK, RSK1, and FAK in NCI-H2228, A549, and Calu-1 cell lines after treatment for 16 h. (d) Western blotting: FAK and PYK2 were detected at different times under 10 nM compounds in MDA-MB-231. (e) FAK and PYK2 abundance was detected at 0 h, 24 h, and 48 h after removal of 10 nM compounds treating for 16 h in MDA-MB-231. (f) Under the treatment of 50 μg/mL CHX (cycloheximide), ALK was detected at different times in SR. FAK and PYK2 were detected at different times in MDA-MB-231.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-13/acs.jmedchem.1c00373/20210629/images/large/jm1c00373_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00373&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">Among these proteins, FAK and PYK2 were the most effectively degraded proteins at 1 nM SIAIS164018. Studies showed that FAK and PYK2 are very important in cell migration and invasion.<a onclick="showRef(event, 'ref35 ref36 ref37 ref38 ref39'); return false;" href="javascript:void(0);" class="ref ref35 ref36 ref37 ref38 ref39">(35−39)</a> Therefore, the unique function in inhibition of cell migration and invasion was likely due to the degradation of FAK and PYK2 proteins.<a onclick="showRef(event, 'ref9 ref40'); return false;" href="javascript:void(0);" class="ref ref9 ref40">(9,40)</a> Thus, turning the warhead of ALK drug Brigatinib, not a clinical FAK inhibitor, to a PROTAC molecule endowed it with an extra function of blocking cell migration and invasion, potentially through degradation of FAK and PYK2. This demonstrated our assumptions of enhancing anti-metastasis through PROTAC.</div></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i10" class="anchor-spacer"></div><h3 class="article-section__title" id="_i10"> Fast Action and Long-Lasting Degradation on FAK and PYK2</h3><div class="NLM_p">Previous studies have shown that the effect of PROTACs on protein degradation was time and dose dependent. Significant protein degradation might initiate from 1 to 16 h after exposed to degraders.<a onclick="showRef(event, 'ref4 ref31 ref41'); return false;" href="javascript:void(0);" class="ref ref4 ref31 ref41">(4,31,41)</a> The protein degradation effect of PROTAC was sustained even after the washout of PROTAC compound.<a onclick="showRef(event, 'ref16 ref33 ref42'); return false;" href="javascript:void(0);" class="ref ref16 ref33 ref42">(16,33,42)</a></div><div class="NLM_p">From the above studies, effective degradation of FAK and PYK2 was identified. To know the degradation properties of FAK and PYK2 besides ALK, we performed time-dependent assay. When treating cells at a concentration of 10 nM (about 10 times of DC<sub>50</sub> to FAK or PYK2), protein degradation started at 1 or 2 h for PYK2 or FAK, respectively. The FAK or PYK2 protein abundance returned to the normal state within 48 h after the washout (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>d,e).</div><div class="NLM_p last">To exclude the degradation property caused by natural protein stability, ALK, FAK, and PYK2 were detected at different times when treating with 50 μg/mL of CHX. ALK was stable within 24 h, and FAK or PYK2 was stable within 8 h (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>f). It was first reported that degradable targets FAK, PYK2, and ALK showed different degradation times and recovery properties under the same treatment condition exposed to SIAIS164018.</div></div><div id="sec2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i11" class="anchor-spacer"></div><h3 class="article-section__title" id="_i11"> SIAIS164018 Downregulates MYC and Cell Cycle Gene Sets</h3><div class="NLM_p">To find out whether SIAIS164018 affected the gene expression of metastatic cells and which genes were affected, RNA-sequencing was performed in Calu-1 cells after 48 h of treatment. Results showed that the gene expression profile of SIAIS164018 was significantly different from that of Brigatinib in clustering and GO analysis (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>a and <a href="/doi/suppl/10.1021/acs.jmedchem.1c00373/suppl_file/jm1c00373_si_001.pdf" class="ext-link">S5A</a>). There was no significant difference between the profile of Brigatinib and that of DMSO. Differently, the degrader SIAIS164018 led to 96 downregulations and 267 upregulations (fold change > 2, <i>q</i> value < 0.05) when compared to Brigatinib. Furthermore, there were no changes in FAK, PYK2, FER, and RSK1 from mRNA levels (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00373/suppl_file/jm1c00373_si_001.pdf" class="ext-link">Figure S5B</a>). Degradation of these proteins was also inhibited by adding MLN4924, MG132, or negative compound 018NC (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00373/suppl_file/jm1c00373_si_001.pdf" class="ext-link">Figure S9</a>). This demonstrated that downregulations of FAK, PYK2, FER, and RSK1 were mediated through the ubiquitin–proteasome system (UPS).</div><figure id="fig6" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-13/acs.jmedchem.1c00373/20210629/images/medium/jm1c00373_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-13/acs.jmedchem.1c00373/20210629/images/large/jm1c00373_0007.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Transcriptional differences induced by SIAIS164018. (a) Overview of gene expression after 48 h of treatment with DMSO, 100 nM Brigatinib, and 100 nM SIAIS164018 in the Calu-1 cell line. (b) Enrichment of gene sets which were analyzed by GSEA. Transcriptions of cyclin D1 and MYC were shown (error bars are shown). (c) Western blotting of cyclin D1 and c-MYC at an indicated concentration after 48 h of treatment in Calu-1 and MDA-MB-231 cell lines.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-13/acs.jmedchem.1c00373/20210629/images/large/jm1c00373_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00373&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">GSEA (Gene Set Enrichment Analysis) showed that the RNA levels of both oncogenic MYC gene set and cell cycle progression gene set were down-regulated after SIAIS164018 treatment (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>b). There was about 0.5- or 0.65-times fold change of cyclin D1 or MYC RNA, respectively, when comparing SIAIS164018 with Brigatinib. The protein levels of cyclin D1 and c-MYC were also examined in Calu-1 and MDA-MB-231 cells (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>c). After 48 h of treatment with SIAIS164018 at indicated concentrations, cyclin D1 and c-MYC protein levels were also significantly decreased in both Calu-1 and MDA-MB-231. To conclude, SIAIS164018 induced an alteration of gene expression involved in MYC signaling and cell cycle, which was distinct from Brigatinib.</div><div class="NLM_p last">To our knowledge, we supposed that degradation of kinases such as FAK or PYK2 induced the downregulation of MYC and cyclin D genes due to their transcription behavior in cell nucleus.<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> Downregulation of MYC gene set also made contributions to cell migration and invasion.<a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a></div></div><div id="sec2_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12"> SIAIS164018 Modulates Kinase Selectivity</h3><div class="NLM_p">To study the relations of degradation and kinase profiles when turning Brigatinib into the degrader SIAIS164018, a high-throughput competitive binding assay (KINOMEScan) was performed.</div><div class="NLM_p">The kinome of SIAIS164018 was determined at 100 nM against 468 kinases including mutant kinases. SIAIS164018 showed a promiscuous inhibition of a number of kinases (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>a). Kinases such as FAK, PYK2, and FER were more preferentially inhibited than ALK by SIAIS164018 while ALK was most preferentially inhibited by Brigatinib in the kinase panel (kinase number: 289).<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> This indicated that SIAIS164018 changed the kinome profile of Brigatinib. Then, we compared the targets of Brigatinib of which the IC<sub>50</sub> value is less than 100 nM (reported data<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a>) with the targets of SIAIS164018 of which the percent control is less than 50%. From our comparison, only 18 kinases (16.5%) were shared in both (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>b). The remaining kinases inhibited by Brigatinib or SIAIS164018 mostly belonged to tyrosine kinases (TKs) and calcium-/calmodulin-dependent protein kinases (CAMKs) (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00373/suppl_file/jm1c00373_si_001.pdf" class="ext-link">Figure S8</a>). Among overlapped kinases, SIAIS164018 showed different ranking preference from Brigatinib. Brigatinib preferably inhibited ALK, FER, and ROS1, while SIAIS164018 preferably inhibited CHEK2, LRRK2, and FAK. This indicated that the kinase inhibition of SIAIS164018 was different from that of Brigatinib and SIAIS164018 rearranged the kinase inhibition of Brigatinib.</div><figure id="fig7" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-13/acs.jmedchem.1c00373/20210629/images/medium/jm1c00373_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-13/acs.jmedchem.1c00373/20210629/images/large/jm1c00373_0008.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Comparison of kinome data with the degradation profile of SIAIS164018 and survival analysis of degraded targets. (a) Kinome of SIAIS164018 at 100 nM (left) and listed targets with high selectivity whose percent control was below 10 (right). (b) Comparison of selectivity between SIAIS164018 and Brigatinib (data of Brigatinib from the reported paper<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a>). Selected targets of SIAIS164018 were percent control below 50; targets of Brigatinib were selected with IC<sub>50</sub> below 100 nM. Among overlapped targets, selectivity difference was drawn using Chordal Graph. Difference targets were shown in the kinome map. Blue dots were targets from Brigatinib, while yellow dots were targets from SIAIS164018. A Venn diagram was plotted using online VENNY 2.1. Chordal Graph was generated using online tools (<a href="https://www.tubiaoxiu.com" class="extLink">https://www.tubiaoxiu.com</a>). Wider arc represented higher selectivity for compounds. (c) Proteomics after 16 h of treatment with 100 nM compounds in MDA-MB-231. 7355 proteins were identified in MDA-MD-231. Red dots: <i>p</i> value < 0.05 and log<sub>2</sub> FC > 1; blue dots: <i>p</i> value < 0.05 and log<sub>2</sub> FC < 1. <i>p</i> value was calculated using <i>T</i> test. (d) Comparing kinome and proteomic in SR, Calu-1, and MDA-MB-231. RSK1(N) was the N terminal kinase domain of RSK1 and RSK1(C) was the C terminal kinase domain of RSK1. Red dots: high selective and degraded; blue dots: high selective and not degraded; and gray dots: low selective and degraded. (e) Survival analysis plots generated from GEPIA (<a href="http://gepia.cancer-pku.cn/index.html" class="extLink">http://gepia.cancer-pku.cn/index.html</a>). LAML (Acute Myelo id Leukemia), BRCA (Breast Invasive Carcinoma), BLCA (Bladder Urothelial Carcinoma), and ACC (Adrenocortical Carcinoma).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-13/acs.jmedchem.1c00373/20210629/images/large/jm1c00373_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00373&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Next, it was important to make it clear that whether kinase inhibition of SIAIS164018 was correlated with its degradation or not. To seek for the answer, we performed proteomics analysis of SIAIS164018 treatment in the metastatic MDA-MB-231 cell line (7355 proteins were identified), in which a well-defined tyrosine kinase PTK6 was also identified besides kinases mentioned above (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>c). PTK6 was strongly degraded as a potent as FAK or PYK2 in a post-translational way (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00373/suppl_file/jm1c00373_si_001.pdf" class="ext-link">Figure S5B</a>).</div><div class="NLM_p last">In the SR proteomic data, degraded kinases exhibited high kinase inhibition except GAK. GAK was highly inhibited but not degraded (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>d). However, in Calu-1 and MDA-MB-231 cell lines, RSK1 was degraded regardless of low kinase inhibition of RSK1(C) or high kinase inhibition of RSK(N). Moreover, it was intriguing that Brigatinib inhibited PTK6 with IC<sub>50</sub> at 4.1 nM, whereas SIAIS164018 showed low kinase inhibition to PTK6 but degraded PTK6 in MDA-MB-231 proteomics data (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>d). Our results indicated that SIAIS164018 modulated the profile of kinase inhibition compared to Brigatinib. It owned better kinase inhibition for many kinases than that for ALK.</div></div><div id="sec2_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13"> Survival Analysis of Kinases Degraded by SIAIS164018</h3><div class="NLM_p last">From online survival analysis<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> (<a href="http://gepia.cancer-pku.cn/index.html" class="extLink">http://gepia.cancer-pku.cn/index.html</a>), genes such as PTK2 (FAK), PTK2B (PYK2), PTK6, FER, and RPS6KA1 (RSK1) were highly related to overall survival in many cancer types such as acute myeloid leukemia, breast invasive carcinoma, bladder urothelial carcinoma, and adrenocortical carcinoma (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>e). This indicated that these genes were clinically related and made contributions to cancer progression in many cancer types. As a summary, we found that FAK, PYK2, and PTK6 were degraded with DC<sub>50</sub> less than 1 nM, and ALK or FER was degraded with DC<sub>50</sub> less than 10 nM. RSK1 and EGFR were degraded with DC<sub>50</sub> less than 100 nM.</div></div><div id="sec2_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14"> SIAIS164018 Is Bioavailable and Well Tolerated In Vivo</h3><div class="NLM_p">At last, we evaluated the metabolism and pharmacokinetic of SIAIS164018 in an animal model (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). From our metabolism results of SIAIS164018, the microsomal stability was less than 9.6 μL/min/mg in both rats and humans. However, it was much higher in mice with 16.7 μL/min/mg. The clearance of SIAIS164018 is 22.6 mL/min/kg with a half-time of 4.65 h, which was well acceptable. SIAIS164018 possessed oral availability with 18.4% post-dosing 10 mg/kg in SD rats. Moreover, the half-life of SIAIS164018 was 7.1 h. From MTD (maximal tolerated dose) assay, the relative change of body weight was relatively normal post-dosing 100 mg/kg SIAIS164018 per day (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00373/suppl_file/jm1c00373_si_001.pdf" class="ext-link">Figure S6</a>). No obvious adverse effects were observed. This provoked preclinical application of SIAIS164018 originated from and beyond Brigatinib. In the next step, we would perform in vivo animal models to explore anti-resistance and anti-metastasis for further studies.</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Metabolism and Pharmacokinetics of SIAIS164018<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th><th class="colsep1 rowsep1" colspan="3" align="center">SIAIS164018</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">study</th><th class="colsep0 rowsep0" align="center">parameters</th><th class="colsep0 rowsep0" align="center">M</th><th class="colsep0 rowsep0" align="center">R</th><th class="colsep0 rowsep0" align="center">H</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" rowspan="2" align="left">in vitro metabolism</td><td class="colsep0 rowsep0" align="left">microsomal stability (μL/min/mg)</td><td class="colsep0 rowsep0" align="left">16.7</td><td class="colsep0 rowsep0" align="left"><9.6</td><td class="colsep0 rowsep0" align="left"><9.6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">CL (mL/min/kg)/<i>T</i><sub>1/2</sub> (h)</td><td class="colsep0 rowsep0" colspan="3" align="center">22.6/4.65</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" rowspan="2" align="left">in vivo PK:IV (2 mg/kg); PO (10 mg/kg)</td><td class="colsep0 rowsep0" align="left"><i>F</i> % (po)/<i>T</i><sub>1/2</sub> (h)</td><td class="colsep0 rowsep0" colspan="3" align="center">18.4/7.1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">AUC (h·ng/mL)</td><td class="colsep0 rowsep0" colspan="3" align="center">1556.1</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup><sup>a</sup></sup><p class="last">M: mouse; R: rat; H: human.</p></div></div></div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i16">Conclusions and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_66481" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_66481" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">UPS has been vital in controlling protein homeostasis. PROTAC intelligently utilized UPS and facilitated the understanding of UPS. The fundamental theory of PROTAC was developed based on the necessities of degradation such as ternary crystal structure complex formation,<a onclick="showRef(event, 'ref45 ref46'); return false;" href="javascript:void(0);" class="ref ref45 ref46">(45,46)</a> selection of warheads, magical linker, and E3s.<a onclick="showRef(event, 'ref6 ref16 ref33'); return false;" href="javascript:void(0);" class="ref ref6 ref16 ref33">(6,16,33)</a> It has been widely developed not only in drug discovery but also provided techniques which are different from RNAi or genome editing to pursuing new biological process.<a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a> In our study, considering clinical benefits of Brigatinib into designing PROTAC, we explored Brigatinib-based degraders and had successfully synthesized and evaluated the degrader SIAIS164018.</div><div class="NLM_p">Degradation of mutant ALK (G1202R) or EGFR (L858R + T790M) had been achieved by SIAIS164018. From other biological phenotypes, SIAIS164018 induced G1 cell cycle arrest distinguishing from Brigatinib in lung cancer cell Calu-1 and triple negative breast cancer MDA-MB-231; inhibition of cell migration and invasion were also achieved post-treating SIAIS164018 in both metastatic Calu-1 and MDA-MB-231 cell lines. Transcriptional down-regulation of oncogenic MYC and cyclin D1 might implicate in enhancing anti-proliferation and cell cycle arrest. This indicated that the degrader SIAIS164018 provided more opportunities in further studies other than protein inhibition.</div><div class="NLM_p">Among clinical or approved drugs,<a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a> drugs with multi-targets seemed to be usual. In addition, it deserved many efforts to repurpose the pharmacology of the approved drugs.<a onclick="showRef(event, 'ref49'); return false;" href="javascript:void(0);" class="ref ref49">(49)</a> This would provide unexpected harvest in the guidance of drug discovery. Moreover, Caner heterogeneity made it complex and difficult to inhibit cancer progression due to heterogeneous driving genes from patient samples. In our understanding, multi-target drugs possessed their own promising therapies. Therefore, when referring to PROTAC, pursuing multi-target degraders could also be promising in further studies.<a onclick="showRef(event, 'ref50'); return false;" href="javascript:void(0);" class="ref ref50">(50)</a> At post-translational regulation, FAK, PYK2, and PTK6 were shown as the most effectively degradable targets besides ALK, FER, EGFR, and RSK1. SIAIS164018 exhibited multifunctional degradable targets closely related to cancer progression and metastasis. These targeted degradations were concentration and time dependent, together with different protein recovery properties.</div><div class="NLM_p last">Referring to affinity and degradation related to SIAIS164018, PTK6 was degraded effectively but with low affinity, while GAK showed no degradation, although with high affinity. Therefore, the affinity of SIAIS164018 to kinases could not predict the potent degradable kinases consistently. However, higher affinity to targets was prone to be degraded. SIAIS164018-modulated kinase selectivity differed from Brigatinib. This reminds us that PROTACs make it more complex to understand. Overall, reshuffle of kinase selectivity could be achieved by turning an inhibitor into a degrader, which may provide more opportunity in treating cancer.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i17">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_33245" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_33245" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> Chemistry</h3><div id="sec4_1_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i19" class="anchor-spacer"></div><h4 class="article-section__title" id="_i19"> General Experiment and Information</h4><div class="NLM_p last">Unless otherwise noted, all purchased reagents were used as received without further purification. <i>N</i>,<i>N</i>-Dimethylformamide (DMF) was dried by a 4 Å MS. Flash chromatography was carried out on silica gel (200–300 mesh). <sup>1</sup>H NMR and <sup>13</sup>CNMR spectra were recorded on a Bruker AVANCE III 500 MHZ (operating at 500 MHz for <sup>1</sup>H NMR and 126 MHz for <sup>13</sup>CNMR), and chemical shift was reported in ppm relative to the residual DMSO-<i>d</i><sub>6</sub> (δ 2.50 ppm <sup>1</sup>H NMR) or CD<sub>3</sub>OD (δ 3.31 ppm <sup>1</sup>H NMR). High-resolution mass spectra were recorded on a AB Triple 4600 spectrometer (QTOF) with acetonitrile and water as solvents. Preparative HPLC was performed on a SHIMADZU LC-20AP series with a UV detector set to 254 nm. The final compounds were all purified by a C18 reverse-phase preparative HPLC column with solvent A (0.5% HCl in H<sub>2</sub>O) and solvent B (MeCN) as eluents. The purity of all the final compounds was confirmed to be >95% purity by HPLC (SHIMADZU). Compound Brigatinib C (2-((5-chloro-2-((2-methoxy-4-(4-(piperazin-1-yl)piperidin-1-yl)phenyl)amino)pyrimidin-4-yl)amino)phenyl)dimethylphosphine oxide was synthesized according to the reference (<i>Eur. J. Med. Chem.</i><b>2020,</b><i>193,</i> 112190).</div></div><div id="sec4_1_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i20" class="anchor-spacer"></div><h4 class="article-section__title" id="_i20"> General Procedure for the Synthesis of ALK Degraders</h4><div class="NLM_p last">In a 25 mL round-bottom flask, to a stirred solution of Brigatinib C (0.02 mmol, 1 equiv) in DMF (1 mL) were added linker <b>s-1</b> (0.02 mmol, 1 equiv), HOAt (0.04 mmol, 2 equiv), EDCI (0.04 mmol, 2 equiv), and NMM (0.2 mmol, 10 equiv) sequentially. Then, the resulting mixture was stirred for 12 h at room temperature. The reaction was quenched with water (0.5 mL), followed by purification via preparative HPLC [C18 column, eluent (v/v)MeCN/(H<sub>2</sub>O + 0.05% HCl) = 10–100%] to afford the desired degraders.</div></div><div id="sec4_1_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i21" class="anchor-spacer"></div><h4 class="article-section__title" id="_i21"> 4-((2-(3-(4-(1-(4-((5-Chloro-4-((2-(dimethylphosphoryl)phenyl)amino)pyrimidin-2-yl)amino)-3-methoxyphenyl)piperidin-4-yl)piperazin-1-yl)-3-oxopropoxy)ethyl)amino)-2-(2,6-dioxopiperidin-3-yl)isoindoline-1,3-dione (<b>1</b>)</h4><div class="NLM_p last">(Yellow solid, 11.4 mg, 59%). <sup>1</sup>H NMR (500 MHz, CD<sub>3</sub>OD): δ 8.38 (s, 1H), 8.06 (s, 1H), 7.72–7.65 (m, 1H), 7.61–7.54 (m, 2H), 7.39 (t, <i>J</i> = 7.5 Hz, 1H), 7.32 (d, <i>J</i> = 8.1 Hz, 1H), 7.11 (d, <i>J</i> = 8.5 Hz, 1H), 7.05 (d, <i>J</i> = 7.0 Hz, 1H), 6.68 (d, <i>J</i> = 2.4 Hz, 1H), 6.52 (d, <i>J</i> = 8.5 Hz, 1H), 5.10 (dd, <i>J</i> = 12.7, 5.5 Hz, 1H), 3.86 (s, 3H), 3.84–3.75 (m, 6H), 3.53–3.51 (m, 4H), 3.43–3.32 (m, 2H), 3.30–2.95 (m, 5H), 2.90–2.83 (m, 2H), 2.81–2.54 (m, 5H), 2.21–2.01 (m, 3H), 1.89 (d, <i>J</i> = 13.6 Hz, 6H), 1.83–1.63 (m, 2H). HRMS (ESI): calcd for C<sub>46</sub>H<sub>55</sub>ClN<sub>10</sub>O<sub>8</sub>P<sup>+</sup> [M + H]<sup>+</sup>, 941.3625; found, 941.3621. Purity: 99.2%.</div></div><div id="sec4_1_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i22" class="anchor-spacer"></div><h4 class="article-section__title" id="_i22"> 4-((2-(2-(3-(4-(1-(4-((5-Chloro-4-((2-(dimethylphosphoryl)phenyl)amino)pyrimidin-2-yl)amino)-3-methoxyphenyl)piperidin-4-yl)piperazin-1-yl)-3-oxopropoxy)ethoxy)ethyl)amino)-2-(2,6-dioxopiperidin-3-yl)isoindoline-1,3-dione (<b>2</b>)</h4><div class="NLM_p last">(Yellow solid, 12.6 mg, 64%). <sup>1</sup>H NMR (500 MHz, CD<sub>3</sub>OD): δ 8.38 (s, 1H), 8.06 (s, 1H), 7.68 (dd, <i>J</i> = 14.7, 7.1 Hz, 1H), 7.60–7.54 (m, 2H), 7.37 (t, <i>J</i> = 7.0 Hz, 1H), 7.32 (d, <i>J</i> = 8.2 Hz, 1H), 7.11 (d, <i>J</i> = 8.5 Hz, 1H), 7.06 (d, <i>J</i> = 7.0 Hz, 1H), 6.69 (d, <i>J</i> = 2.4 Hz, 1H), 6.52 (d, <i>J</i> = 8.2 Hz, 1H), 5.07 (dd, <i>J</i> = 12.5, 5.5 Hz, 1H), 3.89 (s, 1H), 3.87 (s, 1H), 3.84 (s, 3H), 3.78 (t, <i>J</i> = 5.8 Hz, 2H), 3.73 (t, <i>J</i> = 5.1 Hz, 2H), 3.69–3.68 (m, 2H), 3.67–3.63 (m, 2H), 3.51 (t, <i>J</i> = 5.1 Hz, 3H), 3.38–3.31 (m, 8H), 2.91–2.63 (m, 7H), 2.18 (d, <i>J</i> = 10.2 Hz, 2H), 2.15–2.05 (m, 1H), 1.88 (d, <i>J</i> = 13.6 Hz, 6H), 1.86–1.73 (m, 2H). HRMS (ESI): calcd for C<sub>48</sub>H<sub>59</sub>ClN<sub>10</sub>O<sub>9</sub>P <sup>+</sup> [M + H]<sup>+</sup>, 985.3887; found, 985.3883. Purity: 99.3%.</div></div><div id="sec4_1_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i23" class="anchor-spacer"></div><h4 class="article-section__title" id="_i23"> 4-((2-(2-(2-(3-(4-(1-(4-((5-Chloro-4-((2-(dimethylphosphoryl)phenyl)amino)pyrimidin-2-yl)amino)-3-methoxyphenyl)piperidin-4-yl)piperazin-1-yl)-3-oxopropoxy)ethoxy)ethoxy)ethyl)amino)-2-(2,6-dioxopiperidin-3-yl)isoindoline-1,3-dione (<b>3</b>)</h4><div class="NLM_p last">(Yellow solid, 11.5 mg, 56%). <sup>1</sup>H NMR (500 MHz, CD<sub>3</sub>OD): δ 8.38 (s, 1H), 8.07 (d, <i>J</i> = 14.7 Hz, 1H), 7.67 (dd, <i>J</i> = 14.7, 7.1 Hz, 1H), 7.61–7.51 (m, 2H), 7.37 (t, <i>J</i> = 7.1 Hz, 1H), 7.32 (d, <i>J</i> = 8.2 Hz, 1H), 7.08 (t, <i>J</i> = 6.2 Hz, 1H), 7.05 (d, <i>J</i> = 7.0 Hz, 1H), 6.70 (d, <i>J</i> = 2.4 Hz, 1H), 6.53 (d, <i>J</i> = 9.0 Hz, 1H), 5.05 (dd, <i>J</i> = 12.5, 5.5 Hz, 1H), 3.91 (d, <i>J</i> = 12.4 Hz, 2H), 3.84 (s, 3H), 3.80–3.70 (m, 5H), 3.70–3.63 (m, 7H), 3.63–3.57 (m, 3H), 3.51–3.49 (m, 3H), 3.46–3.35 (m, 2H), 3.32–3.31 (m, 3H), 2.92–2.77 (m, 4H), 2.77–2.62 (m, 3H), 2.22 (d, <i>J</i> = 12.1 Hz, 2H), 2.12–2.08 (m, 1H), 1.88 (d, <i>J</i> = 13.6 Hz, 6H), 1.86–1.79 (m, 2H). HRMS (ESI): calcd for C<sub>50</sub>H<sub>63</sub>ClN<sub>10</sub>O<sub>10</sub>P<sup>+</sup> [M + H]<sup>+</sup>, 1029.4149; found, 1029.4139. Purity: 99.7%.</div></div><div id="sec4_1_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i24" class="anchor-spacer"></div><h4 class="article-section__title" id="_i24"> 4-((15-(4-(1-(4-((5-Chloro-4-((2-(dimethylphosphoryl)phenyl)amino)pyrimidin-2-yl)amino)-3-methoxyphenyl)piperidin-4-yl)piperazin-1-yl)-15-oxo-3,6,9,12-tetraoxapentadecyl)amino)-2-(2,6-dioxopiperidin-3-yl)isoindoline-1,3-dione (<b>4</b>)</h4><div class="NLM_p last">(Yellow solid, 9.9 mg, 46%). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>): δ 12.07 (s, 1H), 11.79 (s, 1H), 11.10 (s, 1H), 9.60 (s, 1H), 8.32 (s, 1H), 7.68–7.62 (m, 1H), 7.58 (dd, <i>J</i> = 8.6, 7.0 Hz, 1H), 7.46 (s, 2H), 7.27 (t, <i>J</i> = 7.5 Hz, 1H), 7.15 (d, <i>J</i> = 8.6 Hz, 1H), 7.04 (d, <i>J</i> = 7.1 Hz, 1H), 6.99 (s, 1H), 6.82 (s, 1H), 6.60 (s, 1H), 5.05 (dd, <i>J</i> = 12.8, 5.4 Hz, 1H), 4.48 (d, <i>J</i> = 13.8 Hz, 1H), 4.10 (d, <i>J</i> = 14.0 Hz, 1H), 3.92–3.88 (m, 4H), 3.82 (s, 3H), 3.66–3.61 (m, 4H), 3.58–3.55 (m, 2H), 3.55–3.41 (m, 16H), 3.21 (t, <i>J</i> = 12.9 Hz, 1H), 3.08–2.86 (m, 4H), 2.69–2.56 (m, 3H), 2.29 (d, <i>J</i> = 11.7 Hz, 2H), 2.05–2.01 (m, 3H), 1.80 (d, <i>J</i> = 13.6 Hz, 6H). HRMS (ESI): calcd for C<sub>52</sub>H<sub>67</sub>ClN<sub>10</sub>O<sub>11</sub>P [M + H]<sup>+</sup>, 1073.4411; found, 1073.4405. Purity: 99.5%.</div></div><div id="sec4_1_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i25" class="anchor-spacer"></div><h4 class="article-section__title" id="_i25"> 4-((18-(4-(1-(4-((5-Chloro-4-((2-(dimethylphosphoryl)phenyl)amino)pyrimidin-2-yl)amino)-3-methoxyphenyl)piperidin-4-yl)piperazin-1-yl)-18-oxo-3,6,9,12,15-pentaoxaoctadecyl)amino)-2-(2,6-dioxopiperidin-3-yl)isoindoline-1,3-dione (<b>5</b>)</h4><div class="NLM_p last">(Yellow solid, 13.8 mg, 62%). <sup>1</sup>H NMR (500 MHz, CD<sub>3</sub>OD): δ 8.38 (s, 1H), 8.05 (s, 1H), 7.67 (dd, <i>J</i> = 14.2, 6.5 Hz, 1H), 7.60–7.52 (m, 2H), 7.37 (t, <i>J</i> = 6.9 Hz, 1H), 7.32 (d, <i>J</i> = 8.9 Hz, 1H), 7.08 (d, <i>J</i> = 8.5 Hz, 1H), 7.04 (d, <i>J</i> = 7.0 Hz, 1H), 6.72 (d, <i>J</i> = 2.4 Hz, 1H), 6.57 (d, <i>J</i> = 8.8 Hz, 1H), 5.04 (dd, <i>J</i> = 12.6, 5.5 Hz, 1H), 3.95 (d, <i>J</i> = 12.2 Hz, 2H), 3.85 (s, 3H), 3.75 (dd, <i>J</i> = 12.5, 6.6 Hz, 2H), 3.71 (t, <i>J</i> = 5.2 Hz, 2H), 3.68–3.56 (m, 18H), 3.56–3.37 (m, 5H), 3.32–3.31 (m, 4H), 2.93–2.78 (m, 4H), 2.78–2.60 (m, 3H), 2.25 (d, <i>J</i> = 10.6 Hz, 2H), 2.11–2.07 (m, 1H), 1.90–1.87 (m, 8H). HRMS (ESI): calcd for C<sub>54</sub>H<sub>71</sub>ClN<sub>10</sub>O<sub>12</sub>P [M + H]<sup>+</sup>, 1117.4674; found, 1117.4682. Purity: 99.6%.</div></div><div id="sec4_1_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i26" class="anchor-spacer"></div><h4 class="article-section__title" id="_i26"> 4-((2-(4-(1-(4-((5-Chloro-4-((2-(dimethylphosphoryl)phenyl)amino) pyrimidin-2-yl)amino)-3-methoxyphenyl)piperidin-4-yl)piperazin-1-yl)-2-oxoethyl)amino)-2-(2,6-dioxopiperidin-3-yl)isoindoline-1,3-dione (SIAIS164018, <b>6</b>)</h4><div class="NLM_p last">(Yellow solid, 11.8 mg, 67%). <sup>1</sup>H NMR (500 MHz, CD<sub>3</sub>OD): δ 8.35 (s, 1H), 8.04 (s, 1H), 7.64 (dd, <i>J</i> = 14.1, 7.8 Hz, 1H), 7.58–7.50 (m, 3H), 7.32 (t, <i>J</i> = 7.2 Hz, 1H), 7.08 (d, <i>J</i> = 7.1 Hz, 1H), 6.98 (d, <i>J</i> = 8.5 Hz, 1H), 6.72 (s, 1H), 6.53 (d, <i>J</i> = 8.8 Hz, 1H), 5.08 (dd, <i>J</i> = 12.7, 5.5 Hz, 1H), 4.25 (s, 2H), 3.90 (d, <i>J</i> = 12.5 Hz, 2H), 3.86 (s, 3H), 3.79–3.32 (m, 7H), 3.17 (s, 2H), 2.94–2.83 (m, 3H), 2.80–2.66 (m, 2H), 2.27 (d, <i>J</i> = 11.8 Hz, 2H), 2.18–2.09 (m, 1H), 1.98–1.93 (m, 2H), 1.87 (s, 3H), 1.84 (s, 3H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>): δ 173.05, 170.26, 168.97, 167.48, 166.96, 156.72, 152.87, 145.56, 141.33, 136.38, 132.25, 132.05, 131.16, 131.08, 125.51, 124.94, 124.85, 123.16, 123.11, 122.40, 118.40, 111.13, 109.67, 105.50, 102.80, 61.11, 56.16, 50.18, 48.73, 47.93, 47.86, 43.78, 31.12, 24.64, 22.29, 18.52, 17.96. <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>): δ 173.05, 170.26, 168.97, 167.48, 166.96, 156.72, 152.87, 145.56, 141.33, 136.38, 132.25, 132.05, 131.16, 131.08, 125.51, 124.94, 124.85, 123.16, 123.11, 122.40, 118.40, 111.13, 109.67, 105.50, 102.80, 61.11, 56.16, 50.18, 48.73, 47.93, 47.86, 43.78, 31.12, 24.64, 22.29, 18.52, 17.96. HRMS (ESI): calcd for C<sub>43</sub>H<sub>49</sub>ClN<sub>10</sub>O<sub>7</sub>P [M + H]<sup>+</sup>, 883.3206; found, 883.3204. Purity: 99.2%.</div></div><div id="sec4_1_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i27" class="anchor-spacer"></div><h4 class="article-section__title" id="_i27"> 4-((3-(4-(1-(4-((5-Chloro-4-((2-(dimethylphosphoryl)phenyl)amino) pyrimidin-2-yl)amino)-3-methoxyphenyl)piperidin-4-yl)piperazin-1-yl)-3-oxopropyl)amino)-2-(2,6-dioxopiperidin-3-yl)isoindoline-1,3-dione (<b>7</b>)</h4><div class="NLM_p last">(Yellow solid, 10.7 mg, 60%). <sup>1</sup>H NMR (500 MHz, CD<sub>3</sub>OD): δ 8.39 (s, 1H), 8.06 (s, 1H), 7.68 (dd, <i>J</i> = 13.4, 8.4 Hz, 1H), 7.63–7.53 (m, 2H), 7.38 (t, <i>J</i> = 6.8 Hz, 1H), 7.33 (d, <i>J</i> = 7.4 Hz, 1H), 7.15 (d, <i>J</i> = 8.6 Hz, 1H), 7.10 (d, <i>J</i> = 7.0 Hz, 1H), 6.72 (d, <i>J</i> = 2.4 Hz, 1H), 6.57 (d, <i>J</i> = 8.5 Hz, 1H), 5.05 (dd, <i>J</i> = 12.5, 5.4 Hz, 1H), 3.93 (s, 2H), 3.86 (s, 3H), 3.72 (t, <i>J</i> = 5.2 Hz, 2H), 3.40 (t, <i>J</i> = 12.2 Hz, 2H), 3.32–3.31 (m, 6H), 2.86–2.79 (m, 5H), 2.77–2.64 (m, 3H), 2.17 (d, <i>J</i> = 12.1 Hz, 2H), 2.14–2.06 (m, 1H), 1.90–1.83 (m, 8H). HRMS (ESI): calcd for C<sub>44</sub>H<sub>51</sub>ClN<sub>10</sub>O<sub>7</sub>P [M + H]<sup>+</sup>, 897.3363; found, 897.3366. Purity: 100%.</div></div><div id="sec4_1_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i28" class="anchor-spacer"></div><h4 class="article-section__title" id="_i28"> 4-((4-(4-(1-(4-((5-Chloro-4-((2-(dimethylphosphoryl)phenyl)amino) pyrimidin-2-yl)amino)-3-methoxyphenyl)piperidin-4-yl)piperazin-1-yl)-4-oxobutyl)amino)-2-(2,6-dioxopiperidin-3-yl)isoindoline-1,3-dione (<b>8</b>)</h4><div class="NLM_p last">(Yellow solid, 10.6 mg, 59%). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>): δ 11.10 (s, 1H), 8.44 (s, 1H), 8.18 (s, 1H), 7.63–7.58 (m, 2H), 7.41 (s, 2H), 7.18 (d, <i>J</i> = 8.7 Hz, 2H), 7.04 (d, <i>J</i> = 7.0 Hz, 1H), 6.77 (s, 1H), 6.67–6.63 (m, 2H), 5.06 (dd, <i>J</i> = 12.8, 5.5 Hz, 1H), 4.52 (d, <i>J</i> = 13.9 Hz, 1H), 4.07 (d, <i>J</i> = 14.2 Hz, 1H), 3.90 (d, <i>J</i> = 11.9 Hz, 2H), 3.80 (s, 3H), 3.59 (s, 4H), 3.16–3.11 (m, 4H), 2.99–2.70 (m, 5H), 2.62–2.57 (m, 1H), 2.20 (s, 2H), 2.06–2.01 (m, 1H), 2.01–1.60 (m, 12H). HRMS (ESI): calcd for C<sub>45</sub>H<sub>53</sub>ClN<sub>10</sub>O<sub>7</sub>P [M + H]<sup>+</sup>, 911.3519; found, 911.3528. Purity: 99.9%.</div></div><div id="sec4_1_11" class="NLM_sec NLM_sec_level_3"><div id="ac_i29" class="anchor-spacer"></div><h4 class="article-section__title" id="_i29"> 4-((5-(4-(1-(4-((5-Chloro-4-((2-(dimethylphosphoryl)phenyl)amino)pyrimidin-2-yl)amino)-3-methoxyphenyl)piperidin-4-yl)piperazin-1-yl)-5-oxopentyl)amino)-2-(2,6-dioxopiperidin-3-yl)isoindoline-1,3-dione (<b>9</b>)</h4><div class="NLM_p last">(Yellow solid, 10.8 mg, 58%). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>): δ 12.06 (s, 1H), 11.81 (s, 1H), 11.10 (s, 1H), 9.58 (s, 1H), 8.32 (s, 2H), 7.67–7.63 (m, 1H), 7.59 (dd, <i>J</i> = 8.6, 7.0 Hz, 1H), 7.46 (s, 2H), 7.27 (t, <i>J</i> = 7.5 Hz, 1H), 7.11 (d, <i>J</i> = 8.6 Hz, 1H), 7.03 (d, <i>J</i> = 7.0 Hz, 1H), 6.82 (s, 1H), 6.57 (s, 1H), 5.06 (dd, <i>J</i> = 12.8, 5.4 Hz, 1H), 4.49 (d, <i>J</i> = 13.7 Hz, 1H), 4.09 (d, <i>J</i> = 14.1 Hz, 1H), 3.90 (d, <i>J</i> = 12.0 Hz, 4H), 3.82 (s, 3H), 3.69–3.65 (m, 2H), 3.49–3.46 (m, 4H), 3.24–3.19 (m, 1H), 3.09–2.86 (m, 4H), 2.61–2.57 (m, 1H), 2.44 (d, <i>J</i> = 7.1 Hz, 2H), 2.29 (d, <i>J</i> = 11.8 Hz, 2H), 2.11–1.93 (m, 3H), 1.80 (d, <i>J</i> = 13.6 Hz, 6H), 1.67–1.55 (m, 4H). HRMS (ESI): calcd for C<sub>46</sub>H<sub>55</sub>ClN<sub>10</sub>O<sub>7</sub>P [M + H]<sup>+</sup>, 925.3676; found, 925.3669. Purity: 98.7%.</div></div><div id="sec4_1_12" class="NLM_sec NLM_sec_level_3"><div id="ac_i30" class="anchor-spacer"></div><h4 class="article-section__title" id="_i30"> 4-((6-(4-(1-(4-((5-Chloro-4-((2-(dimethylphosphoryl)phenyl)amino)pyrimidin-2-yl)amino)-3-methoxyphenyl)piperidin-4-yl)piperazin-1-yl)-6-oxohexyl)amino)-2-(2,6-dioxopiperidin-3-yl)isoindoline-1,3-dione (<b>10</b>)</h4><div class="NLM_p last">(Yellow solid, 11.9 mg, 63%). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>): δ 11.10 (s, 1H), 8.46 (s, 1H), 8.15 (s, 1H), 7.63–7.55 (m, 2H), 7.40 (s, 2H), 7.17 (s, 1H), 7.10 (d, <i>J</i> = 8.6 Hz, 1H), 7.03 (d, <i>J</i> = 7.0 Hz, 1H), 6.74 (s, 1H), 6.58–6.52 (m, 2H), 5.05 (dd, <i>J</i> = 12.8, 5.4 Hz, 1H), 4.49 (d, <i>J</i> = 14.0 Hz, 1H), 4.08 (d, <i>J</i> = 14.3 Hz, 1H), 3.89 (d, <i>J</i> = 12.2 Hz, 2H), 3.79 (s, 3H), 3.54 (s, 4H), 3.09–3.03 (m, 4H), 2.99–2.84 (m, 3H), 2.76 (s, 2H), 2.62–2.57 (m, 1H), 2.39 (d, <i>J</i> = 7.4 Hz, 2H), 2.18 (d, <i>J</i> = 11.5 Hz, 2H), 2.05–2.01 (m, 1H), 1.81–1.75 (m, 8H), 1.59–1.55 (m, 4H), 1.41–1.34 (m, 2H). HRMS (ESI): calcd for C<sub>47</sub>H<sub>57</sub>ClN<sub>10</sub>O<sub>7</sub>P [M + H]<sup>+</sup>, 939.3832; found, 939.3827. Purity: 96.9%.</div></div><div id="sec4_1_13" class="NLM_sec NLM_sec_level_3"><div id="ac_i31" class="anchor-spacer"></div><h4 class="article-section__title" id="_i31"> 4-((7-(4-(1-(4-((5-Chloro-4-((2-(dimethylphosphoryl)phenyl)amino)pyrimidin-2-yl)amino)-3-methoxyphenyl)piperidin-4-yl)piperazin-1-yl)-7-oxoheptyl)amino)-2-(2,6-dioxopiperidin-3-yl)isoindoline-1,3-dione (<b>11</b>)</h4><div class="NLM_p last">(Yellow solid, 10.7 mg, 56%). <sup>1</sup>H NMR (500 MHz, CD<sub>3</sub>OD): δ 8.39 (s, 1H), 8.05 (s, 1H), 7.68 (dd, <i>J</i> = 12.8, 7.8 Hz, 1H), 7.56 (dd, <i>J</i> = 8.6, 7.1 Hz, 2H), 7.38 (t, <i>J</i> = 7.0 Hz, 1H), 7.33 (d, <i>J</i> = 8.4 Hz, 1H), 7.05 (t, <i>J</i> = 7.8 Hz, 2H), 6.74 (d, <i>J</i> = 2.4 Hz, 1H), 6.59 (d, <i>J</i> = 8.3 Hz, 1H), 5.06 (dd, <i>J</i> = 12.5, 5.5 Hz, 1H), 3.96 (d, <i>J</i> = 12.7 Hz, 2H), 3.87 (s, 3H), 3.71–3.46 (m, 4H), 3.40–3.31 (m, 7H), 2.93–2.80 (m, 3H), 2.80–2.65 (m, 2H), 2.45 (t, <i>J</i> = 7.5 Hz, 2H), 2.25 (d, <i>J</i> = 11.5 Hz, 2H), 2.15–2.06 (m, 1H), 1.90–1.87 (m, 8H), 1.71–1.62 (m, 4H), 1.58–1.39 (m, 4H). HRMS (ESI): calcd for C<sub>48</sub>H<sub>59</sub>ClN<sub>10</sub>O<sub>7</sub>P [M + H]<sup>+</sup>, 953.3989; found, 953.3983. Purity: 100%.</div></div><div id="sec4_1_14" class="NLM_sec NLM_sec_level_3"><div id="ac_i32" class="anchor-spacer"></div><h4 class="article-section__title" id="_i32"> 4-((2-(4-(1-(4-((5-Chloro-4-((2-(dimethylphosphoryl)phenyl)amino)pyrimidin-2-yl)amino)-3-methoxyphenyl)piperidin-4-yl)piperazin-1-yl)-2-oxoethyl)amino)-2-(1-methyl-2,6-dioxopiperidin-3-yl)isoindoline-1,3-dione (018NC, <b>12</b>)</h4><div class="NLM_p last">(Yellow solid, 9.7 mg, 54%). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>): δ 12.08 (s, 1H), 11.77 (s, 1H), 9.58 (s, 1H), 8.37 (s, 1H), 8.30 (s, 1H), 7.69–7.61 (m, 2H), 7.46 (s, 2H), 7.27 (t, <i>J</i> = 7.5 Hz, 1H), 7.14 (d, <i>J</i> = 8.7 Hz, 1H), 7.10 (d, <i>J</i> = 7.0 Hz, 1H), 7.04–6.95 (m, 1H), 6.80 (s, 1H), 5.14 (dd, <i>J</i> = 13.0, 5.4 Hz, 1H), 4.51 (d, <i>J</i> = 13.9 Hz, 1H), 4.36–4.13 (m, 4H), 3.96–3.91 (m, 4H), 3.82 (s, 3H), 3.39–3.19 (m, 4H), 3.02 (s, 5H), 2.97–2.92 (m, 1H), 2.82–2.75 (m, 1H), 2.60–2.54 (m, 1H), 2.29 (s, 2H), 2.10–1.98 (m, 3H), 1.82 (s, 3H), 1.79 (s, 3H). HRMS (ESI): calcd for C<sub>44</sub>H<sub>51</sub>ClN<sub>10</sub>O<sub>7</sub>P<sup>+</sup> [M + H]<sup>+</sup>, 897.3363; found, 897.3367.</div></div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i33" class="anchor-spacer"></div><h3 class="article-section__title" id="_i33"> Cell Culture</h3><div class="NLM_p last">SR, NCI-H2228, A549, and H1975 cell lines were purchased from American Type Culture Collection. Calu-1 was purchased from Shanghai Cell Bank of the National Science Academy of China (Shanghai). MDA-MB-231 was purchased from Dalian Meilun Biotechnology. Cells were cultured as recommended from the providers. The Calu-1 GFP cell line was stably infected with the lentiviral vector pLVX-IRES-ZsGreen1 carrying the firefly luciferase cDNA. All cell lines were examined as mycoplasma free.</div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i34" class="anchor-spacer"></div><h3 class="article-section__title" id="_i34"> Generation of Mutant Cell Line</h3><div class="NLM_p last">The full length of EML4-ALK cDNA from NCI-H3122 was PCR amplified and cloned into pLVX-Puro plasmid. G1202R point mutation was generated using the KOD Mutagenesis kit (Toyobo Co., Ltd., Osaka, Japan). Mutant ALK vectors were packaged into lentivirus and then transduced into 293T cells. Puromycin was added to the culture medium to select stable clones. The mutant ALK gene was validated by PCR, followed by sequencing (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00373/suppl_file/jm1c00373_si_001.pdf" class="ext-link">Figure S7</a>).</div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i35" class="anchor-spacer"></div><h3 class="article-section__title" id="_i35"> Western Blotting</h3><div class="NLM_p last">Cancer cells were seeded in 1 mL of culture medium in a 24-well plate or a 12-well plate at a density of 3 × 10<sup>5</sup> or 1.5 × 10<sup>5</sup> cell/mL. When cells were treated with different concentrations of drugs for indicated hours, cells were collected, washed with cold 1× PBS, and lysed in 1× SDS buffer containing protease inhibitor cocktails (539134, Merck). The primary antibodies used in this study included ALK (3333), p-ALK (6962), STAT3 (4904), p-STAT3 (9145), FER (4268), PTK6 (55174), c-MYC (5605), cyclin D1 (2978), beta-actin (5125), and GAPDH (8884) and were purchased from Cell Signaling Technology. RSK1 (ab32114) was purchased from Abcam. FAK (A5734), PYK2 (A5653), and GAK (CPA3952) were purchased from Bimake. Anti-rabbit IgG, HRP-linked (7074), and anti-rat IgG, HRP-linked (7077) were purchased from Cell Signaling Technology. The Millipore immobilon western chemiluminescence substrate was used for signal development. Blots were imaged in an Amersham Imager 600 (GE Healthcare). MG132 (MB5137) was purchased from Dalian Meilun Biotechnology. MLN4924 (HY-70062) was purchased from MCE (MedChemExpress).</div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i36" class="anchor-spacer"></div><h3 class="article-section__title" id="_i36"> Cell Growth Inhibition</h3><div class="NLM_p last">Cells were seeded at a specific density in 96-well plates 1 day before treatment. The cells were treated with a series concentration of different compounds and incubated for 72 h. Viable cells were measured via the CCK8 reagent (Meilunbio, MA0218; TargetMol, C0005), and IC<sub>50</sub> values were calculated via Prism Graphpad. The experiments were repeated at least three times.</div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i37" class="anchor-spacer"></div><h3 class="article-section__title" id="_i37"> Cell Migration</h3><div class="NLM_p">Calu-1 cells (5 × 10<sup>5</sup>) were seeded in a 24-well plate, and scratches were made using a 200 μL tip. Pictures were taken using a fluorescent inverted microscope (EVOS fl) at indicated times. Wound widths were analyzed. The experiments were repeated three times.</div><div class="NLM_p last">MDA-MB-231 cells (6 × 10<sup>4</sup>) were seeded in a 96-well plate; on the second day, scratches were made by a nick instrument. Pictures were taken every 4 h within 48 h using Incucyte. Data from three repeats were analyzed.</div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i38" class="anchor-spacer"></div><h3 class="article-section__title" id="_i38"> Transwell Invasion Assay</h3><div class="NLM_p last">To make the coating buffer, Matrigel (Biocoat 354234) was diluted 50 times using the culture medium without FBS. The coating buffer (100 μL) was added into a transwell insert (a Corning FluoroBlok 24-well plate permeable support with an 8.0 μm colored PET membrane, 351152) and incubated at 37 °C for 1 h. Calu-1 GFP cells were digested and washed with PBS. Then, 2.8 × 10<sup>4</sup> cells were resuspended in 150 μL of serum-free medium containing different compounds and added in the transwell insert. The lower chamber was filled with 10% FBS containing medium, and then a 24-well plate was incubated at 37 °C and 5% CO<sub>2</sub>. Permeable Calu-1 GFP cells were visible with green under a fluorescent inverted microscope. Five pictures were taken from each transwell. Then, the pictures were analyzed using ImageJ to count the cell number per 1 mm<sup>3</sup>. Statistical analysis was performed using via Prism Graphpad. The experiments were repeated three times.</div></div><div id="sec4_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i39" class="anchor-spacer"></div><h3 class="article-section__title" id="_i39"> Cell Cycle</h3><div class="NLM_p last">Cells (1.2 × 10<sup>5</sup>) were treated with compounds in a 12-well plate for indicated times, then cells were treated according to the instruction of the protocol (Meilunbio, MA0334). The cell cycle was measured using a CytoFLEX (Beckman Coulter). The cell cycle was analyzed using ModFit LT 3.1.</div></div><div id="sec4_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i40" class="anchor-spacer"></div><h3 class="article-section__title" id="_i40"> Cell Apoptosis</h3><div class="NLM_p last">Cells (2 × 10<sup>4</sup>) per well were plated in a 24-well plate. Different compounds were added in the cell medium on the second day. After treating for 72 h, the cells were treated according to the instruction of the protocol (Meilunbio, MA0220). Apoptosis was analyzed using CytoFLEX (Beckman Coulter).</div></div><div id="sec4_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i41" class="anchor-spacer"></div><h3 class="article-section__title" id="_i41"> Quantitative Mass Spectrometry Analysis</h3><div class="NLM_p last">SR, Calu-1, or MDA-MB-231 cells were treated with different compounds for 16 h and cells were harvested by centrifugation. Lysis buffer (4% SDS, 100 mM Tris, pH 7.6) was added to the cell pellets and lysed by sonication. Approximately 200 μg of total protein from each sample was proteolyzed on a 10 kDa Filter (PALL Life Sciences, Shanghai, China) using a Filter Aided Sample Preparation (FASP) protocol. The methods of sample preparation and LC–MS data analysis were the same as mentioned in the reference (Zhao, Quanju, et al. <i>J. Med. Chem.</i><b>2019</b>, <i>62</i> (20), 9281–9298). Data analyses were performed using the two-tailed Student <i>t</i>-test for independent samples.</div></div><div id="sec4_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i42" class="anchor-spacer"></div><h3 class="article-section__title" id="_i42"> RNA-Seq Analysis</h3><div class="NLM_p last">Calu-1 GFP cells (1.5 × 10<sup>6</sup>) were planted in a 10 cm dish. On the second day, 100 nM compounds were added to the culture medium. After 48 h, the cells were harvested. RNA was extracted using the Trizol reagent (Life 15596026). NGS was sequenced and analyzed in GENEWIZ, Suzhou, China. Triplicate samples were used. GSEA was analyzed using the software downloaded from the website of GSEA (<a href="https://www.gsea-msigdb.org/gsea/index.jsp" class="extLink">https://www.gsea-msigdb.org/gsea/index.jsp</a>).</div></div><div id="sec4_12" class="NLM_sec NLM_sec_level_2"><div id="ac_i43" class="anchor-spacer"></div><h3 class="article-section__title" id="_i43"> Binding Affinity and Kinome</h3><div class="NLM_p last">Compounds binding affinities (<i>K</i><sub>d</sub>) to phosphorylated and non-phosphorylated kinase domain were determined using the KinomeScan platform (DiscoverRx Corporation). The compounds were solubilized in DMSO as a 10 μM stock solution and a series of three-fold dilutions were utilized for affinity analysis. Kinome of 100 nM SIAIS164018 was tested using the KINOMEscan screening platform.</div></div><div id="sec4_13" class="NLM_sec NLM_sec_level_2"><div id="ac_i44" class="anchor-spacer"></div><h3 class="article-section__title" id="_i44"> <i>In Vitro</i> Metabolism Assay</h3><div class="NLM_p">Liver microsome preparation: HLM: human cat no. 452117; RLM: SD rat cat no. BQR1000; and MLM:CD-1 mouse cat no. BQM1000.</div><div class="NLM_p last">Compound stock solution (5 μL, 10 mM in DMSO) was diluted with 495 μL of methanol (MeOH) (intermediate solution concentration: 100 μM, 99% MeOH). The appropriate amount of NADPH powder was weighed and diluted into a 2 mM MgCl<sub>2</sub> solution (working solution concentration: 2 unit/mL; final concentration in reaction system: 1 unit/mL). The appropriate concentrations of microsome working solutions were prepared in 100 mM potassium phosphate buffer. Cold (4 °C) acetonitrile containing 200 ng/mL of tolbutamide and 200 ng/mL of labetalol as internal standards was used as the stop solution. Using an Apricot automation workstation, 2 μL/well of compound working solution was added to all 96-well reaction plates except the blank (T0, T5, T10, T20, T30, T60, and NCF60). An Apricot automation workstation was used to add 100 μL/well of microsome solution to all reaction plates (blank, T0, T5, T10, T20, T30, T60, and NCF60). All reaction plates containing mixtures of compound and microsomes were pre-incubated at 37 °C for 10 min. An Apricot automation workstation was used to add 98 μL/well of 100 mM potassium phosphate buffer to the reaction plate NCF60. The reaction plate NCF60 was incubated at 37 °C, and timer 1 was started. After pre-incubation, an Apricot automation workstation was used to add 98 μL/well of NADPH regenerating system to every reaction plate except NCF60 (Blank, T0, T5, T10, T20, T30, and T60) to start the reaction. The reaction plates were incubated at 37 °C, and timer 2 was started. An Apricot automation workstation was used to add 600 μL/well of stop solution to each reaction plate at its appropriate end time point to terminate the reaction. Each plate was sealed and shaken for 10 min. After shaking, each plate was centrifuged at 4000 rpm and 4 °C for 20 min. After centrifugation, an Apricot automation workstation was used to transfer 300 μL of supernatant from each reaction plate to eight new 96-well plates to LC–MS/MS analysis. The equation of first-order kinetics was used to calculate <i>T</i><sub>1/2</sub> and <i>C</i><sub>lint</sub> (mic) (μL/min/mg).</div></div><div id="sec4_14" class="NLM_sec NLM_sec_level_2"><div id="ac_i45" class="anchor-spacer"></div><h3 class="article-section__title" id="_i45"> Pharmacokinetic and MTD (Maximal Tolerated Dose) Assay</h3><div class="NLM_p">All animal experiments performed in this study were conducted in compliance with the guidelines of Institutional Animal Care & Use Committee (IACUC) and Guide for the Care and Use of Laboratory Animals.</div><div class="NLM_p last">SD rats were purchased from Sino-British SIPPR/BK Lab Animal Ltd., Shanghai, China. PK profiles after IV and PO administrations to SD rats were studied. Plasma concentrations of compound were recorded at each of the nine time points (5 min, 0.25 h, 0.5 h, 1 h, 2 h, 4 h, 6 h, 8 h, and 24 h post-dosing) and the data were the average values from three tests. All procedures involving animals followed protocols approved by the committee for animal research of Medicilon (Shanghai) and conformed to the guide for the care and use of laboratory animals. The MTD (maximal tolerance dose) study of SIAIS164018 was conducted in NU/NU mice. Doses were 20, 100 mg/kg. All procedures involving animals followed protocols approved by the committee for animal research of LIDE (Shanghai) and conformed to the guide for the care and use of laboratory animals.</div></div><div id="sec4_15" class="NLM_sec NLM_sec_level_2"><div id="ac_i46" class="anchor-spacer"></div><h3 class="article-section__title" id="_i46"> Statistical Analysis</h3><div class="NLM_p last">Plots of proteomics and relations between degradation and kinome were generated using R version 3.5.1. Package ggplot2 was used.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i47"><a href="/doi/suppl/10.1021/acs.jmedchem.1c00373" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_08115" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_08115" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a href="/doi/10.1021/acs.jmedchem.1c00373?goto=supporting-info" class="ext-link">https://pubs.acs.org/doi/10.1021/acs.jmedchem.1c00373</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">NMR and HPLC of compounds <b>1–12</b>; hook effect of SIAIS164018; signaling pathway; apoptosis analysis; gene ontology analysis; maximal tolerance dose study; AlK mutant identification; kinome map; and degradation through UPS (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00373/suppl_file/jm1c00373_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00373/suppl_file/jm1c00373_si_002.csv" class="ext-link">CSV</a>)</p></li><li><p class="inline">Proteomics data (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00373/suppl_file/jm1c00373_si_003.xlsx" class="ext-link">XLSX</a>)</p></li><li><p class="inline">RNA-seq data (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00373/suppl_file/jm1c00373_si_004.xlsx" class="ext-link">XLSX</a>)</p></li><li><p class="inline">Kinome data (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00373/suppl_file/jm1c00373_si_005.xlsx" class="ext-link">XLSX</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00373/suppl_file/jm1c00373_si_001.pdf">jm1c00373_si_001.pdf (2.24 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00373/suppl_file/jm1c00373_si_002.csv">jm1c00373_si_002.csv (1.95 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00373/suppl_file/jm1c00373_si_003.xlsx">jm1c00373_si_003.xlsx (26.74 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00373/suppl_file/jm1c00373_si_004.xlsx">jm1c00373_si_004.xlsx (2.38 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00373/suppl_file/jm1c00373_si_005.xlsx">jm1c00373_si_005.xlsx (12.98 MB)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.1c00373" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_34263" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_34263" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Xiaoling Song</span> - <span class="hlFld-Affiliation affiliation">Shanghai
Institute for Advanced Immunochemical Studies, ShanghaiTech University, Shanghai 201210, China</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#99eaf6f7fee1f5d9eaf1f8f7fef1f8f0edfcfaf1b7fcfdecb7faf7"><span class="__cf_email__" data-cfemail="d3a0bcbdb4abbf93a0bbb2bdb4bbb2baa7b6b0bbfdb6b7a6fdb0bd">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Xiaobao Yang</span> - <span class="hlFld-Affiliation affiliation">Shanghai
Institute for Advanced Immunochemical Studies, ShanghaiTech University, Shanghai 201210, China</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0001-5266-7673" title="Orcid link">https://orcid.org/0000-0001-5266-7673</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#81f9e8e0eee3e0eef8eef4efe6c1b0b3b7afe2eeeca1f8e0efe6f9e3c1f2e9e0efe6e9e0e8f5e4e2e9afe4e5f4afe2ef"><span class="__cf_email__" data-cfemail="0a72636b65686b6573657f646d4a3b383c24696567">[email protected]</span> <span class="__cf_email__" data-cfemail="235a424d445b4163504b424d444b424a5746404b0d4647560d404d">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Biao Jiang</span> - <span class="hlFld-Affiliation affiliation">Shanghai
Institute for Advanced Immunochemical Studies, ShanghaiTech University, Shanghai 201210, China</span>; 
    <span class="hlFld-Affiliation affiliation">School
of Physical Science and Technology, ShanghaiTech
University, Shanghai 201210, China</span>; 
    <span class="hlFld-Affiliation affiliation">CAS
Key Laboratory
of Synthetic Chemistry of Natural Substances, Shanghai Institute of
Organic Chemistry, Chinese Academy of Sciences, Shanghai 200032, China</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#dab0b3bbb4bdb8b3bbb59aa9b2bbb4bdb2bbb3aebfb9b2f4bfbeaff4b9b4"><span class="__cf_email__" data-cfemail="f3999a929d94919a929cb3809b929d949b929a8796909bdd969786dd909d">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Chaowei Ren</span> - <span class="hlFld-Affiliation affiliation">Shanghai
Institute for Advanced Immunochemical Studies, ShanghaiTech University, Shanghai 201210, China</span>; 
    <span class="hlFld-Affiliation affiliation">School
of Life Science and Technology, ShanghaiTech
University, Shanghai 201210, China</span>; 
    <span class="hlFld-Affiliation affiliation">University
of Chinese Academy of Sciences, Beijing 100049, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ning Sun</span> - <span class="hlFld-Affiliation affiliation">Shanghai
Institute for Advanced Immunochemical Studies, ShanghaiTech University, Shanghai 201210, China</span>; 
    <span class="hlFld-Affiliation affiliation">Shanghai
Institutes for Biological Sciences, Chinese
Academy of Sciences, Shanghai 200031, China</span>; 
    <span>Present Address:
                        Icahn School of Medicine at Mount Sinai, New York, USA</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Haixia Liu</span> - <span class="hlFld-Affiliation affiliation">Shanghai
Institute for Advanced Immunochemical Studies, ShanghaiTech University, Shanghai 201210, China</span>; 
    <span class="hlFld-Affiliation affiliation">School
of Physical Science and Technology, ShanghaiTech
University, Shanghai 201210, China</span>; 
    <span class="hlFld-Affiliation affiliation">University
of Chinese Academy of Sciences, Beijing 100049, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ying Kong</span> - <span class="hlFld-Affiliation affiliation">Jing
Medicine Technology (Shanghai), Ltd., Y Building, Shanghai 201210, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Renhong Sun</span> - <span class="hlFld-Affiliation affiliation">Shanghai
Institute for Advanced Immunochemical Studies, ShanghaiTech University, Shanghai 201210, China</span>; 
    <span class="hlFld-Affiliation affiliation">Shanghai
Institutes for Biological Sciences, Chinese
Academy of Sciences, Shanghai 200031, China</span>; 
    <span>Present Address:
                        Icahn School of Medicine at Mount Sinai, New York, USA</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Xing Qiu</span> - <span class="hlFld-Affiliation affiliation">CAS
Key Laboratory
of Synthetic Chemistry of Natural Substances, Shanghai Institute of
Organic Chemistry, Chinese Academy of Sciences, Shanghai 200032, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jinju Chen</span> - <span class="hlFld-Affiliation affiliation">Jing
Medicine Technology (Shanghai), Ltd., Y Building, Shanghai 201210, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yan Li</span> - <span class="hlFld-Affiliation affiliation">Jing
Medicine Technology (Shanghai), Ltd., Y Building, Shanghai 201210, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jianshui Zhang</span> - <span class="hlFld-Affiliation affiliation">Jing
Medicine Technology (Shanghai), Ltd., Y Building, Shanghai 201210, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yuedong Zhou</span> - <span class="hlFld-Affiliation affiliation">Jing
Medicine Technology (Shanghai), Ltd., Y Building, Shanghai 201210, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Hui Zhong</span> - <span class="hlFld-Affiliation affiliation">Shanghai
Institute for Advanced Immunochemical Studies, ShanghaiTech University, Shanghai 201210, China</span>; 
    <span class="hlFld-Affiliation affiliation">School
of Pharmaceutical Engineering and Life Science, Changzhou University, Changzhou, Jiangsu 213164, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Qianqian Yin</span> - <span class="hlFld-Affiliation affiliation">Shanghai
Institute for Advanced Immunochemical Studies, ShanghaiTech University, Shanghai 201210, China</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>C.R. and N.S. contributed equally as co-first authors.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br><p class="inlineNote">All data needed to evaluate the conclusions in the paper are present in the paper and/or in the <a href="/doi/suppl/10.1021/acs.jmedchem.1c00373/suppl_file/jm1c00373_si_001.pdf" class="ext-link">Supporting Information</a>.</p></div></li></ul></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i51">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_06748" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_06748" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">This research was supported by the National Natural Science Foundation of China (grant nos. 82072592 and 81602602 to X.S.), the Shanghai Natural Science Foundation (grant no.19ZR1433600 to X.Y.), the Shanghai Science and Technology Innovation Fund (grant no. 20S11903100 to X.S.), the ShanghaiTech University Financial support (to B.J.), the China Postdoctoral Science Foundation (2018M632181 to N.S.), the National Natural Science Foundation of China (grant 81702600 to Q.-Q.Y.), and the Jing Medicine support. We thank Haifan Lin for giving advices. We thank Wenzhang Chen, Hongwei Zhang, Jiakang Chen, and Wei Zhu from Analytic platform and Juan Kong from High Throughput Screening Platform from the Shanghai Institute for Advanced Immunochemical Studies (SIAIS). We also thank Jing Medicine Technology (Shanghai) for metabolism and PK assays. Moreover, the scientific infrastructures of SIAIS facilitate our scientific research studies.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations</td></tr><tr><td class="NLM_term">ALK</td><td class="NLM_def"><p class="first last">anaplastic lymphoma kinase</p></td></tr><tr><td class="NLM_term">DC<sub>50</sub></td><td class="NLM_def"><p class="first last">the concentration of degrading 50% protein</p></td></tr><tr><td class="NLM_term">EGFR</td><td class="NLM_def"><p class="first last">epidermal growth factor receptor</p></td></tr><tr><td class="NLM_term">FAK</td><td class="NLM_def"><p class="first last">focal adhesion kinase 1</p></td></tr><tr><td class="NLM_term">FER</td><td class="NLM_def"><p class="first last">tyrosine-protein kinase Fer</p></td></tr><tr><td class="NLM_term">GAK</td><td class="NLM_def"><p class="first last">cyclin-G-associated kinase</p></td></tr><tr><td class="NLM_term">IC<sub>50</sub></td><td class="NLM_def"><p class="first last">the concentration of half inhibition</p></td></tr><tr><td class="NLM_term">PROTAC</td><td class="NLM_def"><p class="first last">proteolysis-targeting chimera</p></td></tr><tr><td class="NLM_term">PTK6</td><td class="NLM_def"><p class="first last">protein-tyrosine kinase 6</p></td></tr><tr><td class="NLM_term">PYK2</td><td class="NLM_def"><p class="first last">protein-tyrosine kinase 2-beta</p></td></tr><tr><td class="NLM_term">RSK1</td><td class="NLM_def"><p class="first last">ribosomal protein S6 kinase alpha-1</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i53">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_30607" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_30607" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 50 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lai, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Induced protein degradation: an emerging drug discovery paradigm</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">101</span>– <span class="NLM_lpage">114</span>, <span class="refDoi"> DOI: 10.1038/nrd.2016.211</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=10.1038%2Fnrd.2016.211" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=27885283" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvFCit7rP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2017&pages=101-114&author=A.+C.+Laiauthor=C.+M.+Crews&title=Induced+protein+degradation%3A+an+emerging+drug+discovery+paradigm&doi=10.1038%2Fnrd.2016.211"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Induced protein degradation: an emerging drug discovery paradigm</span></div><div class="casAuthors">Lai, Ashton C.; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">101-114</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Small-mol. drug discovery has traditionally focused on occupancy of a binding site that directly affects protein function, and this approach typically precludes targeting proteins that lack such amenable sites.  Furthermore, high systemic drug exposures may be needed to maintain sufficient target inhibition in vivo, increasing the risk of undesirable off-target effects.  Induced protein degrdn. is an alternative approach that is event-driven: upon drug binding, the target protein is tagged for elimination.  Emerging technologies based on proteolysis-targeting chimaeras (PROTACs) that exploit cellular quality control machinery to selectively degrade target proteins are attracting considerable attention in the pharmaceutical industry owing to the advantages they could offer over traditional small-mol. strategies.  These advantages include the potential to reduce systemic drug exposure, the ability to counteract increased target protein expression that often accompanies inhibition of protein function and the potential ability to target proteins that are not currently therapeutically tractable, such as transcription factors, scaffolding and regulatory proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoj-O7zXGOpXLVg90H21EOLACvtfcHk0lixSEcxvn62Bg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvFCit7rP&md5=a01ff43e9b696fe4dc03f0815158a98f</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2016.211&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2016.211%26sid%3Dliteratum%253Aachs%26aulast%3DLai%26aufirst%3DA.%2BC.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DInduced%2520protein%2520degradation%253A%2520an%2520emerging%2520drug%2520discovery%2520paradigm%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2017%26volume%3D16%26spage%3D101%26epage%3D114%26doi%3D10.1038%2Fnrd.2016.211" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Burslem, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Small-Molecule Modulation of Protein Homeostasis</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>117</i></span>,  <span class="NLM_fpage">11269</span>– <span class="NLM_lpage">11301</span>, <span class="refDoi"> DOI: 10.1021/acs.chemrev.7b00077</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.chemrev.7b00077" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=1%3ACAS%3A528%3ADC%252BC2sXht1Ohur3K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=117&publication_year=2017&pages=11269-11301&author=G.+M.+Burslemauthor=C.+M.+Crews&title=Small-Molecule+Modulation+of+Protein+Homeostasis&doi=10.1021%2Facs.chemrev.7b00077"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Small-Molecule Modulation of Protein Homeostasis</span></div><div class="casAuthors">Burslem, George M.; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">117</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">11269-11301</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Control of protein levels by nucleic-acid-based technologies has proven to be a useful research tool but lacks the advantages of small mols. with respect to cell permeability, temporal control, and the potential generation of therapeutics.  In this Review, we discuss the technologies available for the control of intracellular protein levels with small mols. and compare the various systems available.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpcARydbgPK9bVg90H21EOLACvtfcHk0lixSEcxvn62Bg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXht1Ohur3K&md5=f9f8dd98e2c71c7166d2d500b514d578</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1021%2Facs.chemrev.7b00077&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.chemrev.7b00077%26sid%3Dliteratum%253Aachs%26aulast%3DBurslem%26aufirst%3DG.%2BM.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DSmall-Molecule%2520Modulation%2520of%2520Protein%2520Homeostasis%26jtitle%3DChem.%2520Rev.%26date%3D2017%26volume%3D117%26spage%3D11269%26epage%3D11301%26doi%3D10.1021%2Facs.chemrev.7b00077" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bondeson, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mares, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, I. E. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ko, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campos, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miah, A. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mulholland, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Routly, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buckley, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gustafson, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zinn, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grandi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shimamura, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bergamini, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Faelth-Savitski, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bantscheff, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cox, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Willard, R. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flanagan, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Casillas, L. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Votta, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">den Besten, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Famm, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kruidenier, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carter, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harling, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Churcher, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Catalytic in vivo protein knockdown by small-molecule PROTACs</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">611</span>– <span class="NLM_lpage">617</span>, <span class="refDoi"> DOI: 10.1038/nchembio.1858</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=10.1038%2Fnchembio.1858" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=26075522" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFeju7rP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2015&pages=611-617&author=D.+P.+Bondesonauthor=A.+Maresauthor=I.+E.+D.+Smithauthor=E.+Koauthor=S.+Camposauthor=A.+H.+Miahauthor=K.+E.+Mulhollandauthor=N.+Routlyauthor=D.+L.+Buckleyauthor=J.+L.+Gustafsonauthor=N.+Zinnauthor=P.+Grandiauthor=S.+Shimamuraauthor=G.+Bergaminiauthor=M.+Faelth-Savitskiauthor=M.+Bantscheffauthor=C.+Coxauthor=D.+A.+Gordonauthor=R.+R.+Willardauthor=J.+J.+Flanaganauthor=L.+N.+Casillasauthor=B.+J.+Vottaauthor=W.+den+Bestenauthor=K.+Fammauthor=L.+Kruidenierauthor=P.+S.+Carterauthor=J.+D.+Harlingauthor=I.+Churcherauthor=C.+M.+Crews&title=Catalytic+in+vivo+protein+knockdown+by+small-molecule+PROTACs&doi=10.1038%2Fnchembio.1858"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Catalytic in vivo protein knockdown by small-molecule PROTACs</span></div><div class="casAuthors">Bondeson, Daniel P.; Mares, Alina; Smith, Ian E. D.; Ko, Eunhwa; Campos, Sebastien; Miah, Afjal H.; Mulholland, Katie E.; Routly, Natasha; Buckley, Dennis L.; Gustafson, Jeffrey L.; Zinn, Nico; Grandi, Paola; Shimamura, Satoko; Bergamini, Giovanna; Faelth-Savitski, Maria; Bantscheff, Marcus; Cox, Carly; Gordon, Deborah A.; Willard, Ryan R.; Flanagan, John J.; Casillas, Linda N.; Votta, Bartholomew J.; den Besten, Willem; Famm, Kristoffer; Kruidenier, Laurens; Carter, Paul S.; Harling, John D.; Churcher, Ian; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">611-617</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The current predominant therapeutic paradigm is based on maximizing drug-receptor occupancy to achieve clin. benefit.  This strategy, however, generally requires excessive drug concns. to ensure sufficient occupancy, often leading to adverse side effects.  Here, we describe major improvements to the proteolysis targeting chimeras (PROTACs) method, a chem. knockdown strategy in which a heterobifunctional mol. recruits a specific protein target to an E3 ubiquitin ligase, resulting in the target's ubiquitination and degrdn.  These compds. behave catalytically in their ability to induce the ubiquitination of super-stoichiometric quantities of proteins, providing efficacy that is not limited by equil. occupancy.  We present two PROTACs that are capable of specifically reducing protein levels by >90% at nanomolar concns.  In addn., mouse studies indicate that they provide broad tissue distribution and knockdown of the targeted protein in tumor xenografts.  Together, these data demonstrate a protein knockdown system combining many of the favorable properties of small-mol. agents with the potent protein knockdown of RNAi and CRISPR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGonhnajD3CxS7Vg90H21EOLACvtfcHk0lixSEcxvn62Bg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtFeju7rP&md5=31eb8756c5661f78ab4527a50dc1e251</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1858&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1858%26sid%3Dliteratum%253Aachs%26aulast%3DBondeson%26aufirst%3DD.%2BP.%26aulast%3DMares%26aufirst%3DA.%26aulast%3DSmith%26aufirst%3DI.%2BE.%2BD.%26aulast%3DKo%26aufirst%3DE.%26aulast%3DCampos%26aufirst%3DS.%26aulast%3DMiah%26aufirst%3DA.%2BH.%26aulast%3DMulholland%26aufirst%3DK.%2BE.%26aulast%3DRoutly%26aufirst%3DN.%26aulast%3DBuckley%26aufirst%3DD.%2BL.%26aulast%3DGustafson%26aufirst%3DJ.%2BL.%26aulast%3DZinn%26aufirst%3DN.%26aulast%3DGrandi%26aufirst%3DP.%26aulast%3DShimamura%26aufirst%3DS.%26aulast%3DBergamini%26aufirst%3DG.%26aulast%3DFaelth-Savitski%26aufirst%3DM.%26aulast%3DBantscheff%26aufirst%3DM.%26aulast%3DCox%26aufirst%3DC.%26aulast%3DGordon%26aufirst%3DD.%2BA.%26aulast%3DWillard%26aufirst%3DR.%2BR.%26aulast%3DFlanagan%26aufirst%3DJ.%2BJ.%26aulast%3DCasillas%26aufirst%3DL.%2BN.%26aulast%3DVotta%26aufirst%3DB.%2BJ.%26aulast%3Dden%2BBesten%26aufirst%3DW.%26aulast%3DFamm%26aufirst%3DK.%26aulast%3DKruidenier%26aufirst%3DL.%26aulast%3DCarter%26aufirst%3DP.%2BS.%26aulast%3DHarling%26aufirst%3DJ.%2BD.%26aulast%3DChurcher%26aufirst%3DI.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DCatalytic%2520in%2520vivo%2520protein%2520knockdown%2520by%2520small-molecule%2520PROTACs%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2015%26volume%3D11%26spage%3D611%26epage%3D617%26doi%3D10.1038%2Fnchembio.1858" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sun, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hwang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rao, Y.</span></span> <span> </span><span class="NLM_article-title">PROTAC-induced BTK degradation as a novel therapy for mutated BTK C481S induced ibrutinib-resistant B-cell malignancies</span>. <i>Cell Res.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">779</span>– <span class="NLM_lpage">781</span>, <span class="refDoi"> DOI: 10.1038/s41422-018-0055-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=10.1038%2Fs41422-018-0055-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=29875397" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=1%3ACAS%3A528%3ADC%252BC1cXht1ektbzO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2018&pages=779-781&author=Y.+Sunauthor=X.+Zhaoauthor=N.+Dingauthor=H.+Gaoauthor=Y.+Wuauthor=Y.+Yangauthor=M.+Zhaoauthor=J.+Hwangauthor=Y.+Songauthor=W.+Liuauthor=Y.+Rao&title=PROTAC-induced+BTK+degradation+as+a+novel+therapy+for+mutated+BTK+C481S+induced+ibrutinib-resistant+B-cell+malignancies&doi=10.1038%2Fs41422-018-0055-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">PROTAC-induced BTK degradation as a novel therapy for mutated BTK C481S induced ibrutinib-resistant B-cell malignancies</span></div><div class="casAuthors">Sun, Yonghui; Zhao, Xingwang; Ding, Ning; Gao, Hongying; Wu, Yue; Yang, Yiqing; Zhao, Meng; Hwang, Jinseok; Song, Yuqin; Liu, Wanli; Rao, Yu</div><div class="citationInfo"><span class="NLM_cas:title">Cell Research</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">779-781</span>CODEN:
                <span class="NLM_cas:coden">CREEB6</span>;
        ISSN:<span class="NLM_cas:issn">1001-0602</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">In this study, for the first time, we report the development of BTK-targeting degraders using the PROTAC strategy.  These PROTACs could ef fi ciently degrade ibrutinib-sensitive BTK-WT (wild type).  More importantly, our newly designed PROTACs also significantly induced the degrdn. of ibrutinib resistant BTK-C481S (50% degrdn. efficiency at 30 nM).  Furthermore, our PROTAC mols. ef fi ciently inhibited cell proliferation and colony formation, while exhibited no obvious inhibition (>1000 nM) of ITK, EGFR, and TEC, which are major off-targets of ibrutinib.  These data demonstrate the strong potential for developing PROTAC-based therapeutic mols.  The results indicate that the PROTAC strategy (protein degrdn. instead of inhibition) could be adopted as a general and powerful therapeutic treatment for drug-resistant cancers in the future.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqg_FLdaUP_HLVg90H21EOLACvtfcHk0liJ1rYlTOuaAw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXht1ektbzO&md5=1cca51ac76ecb3b491da0117b21271fb</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1038%2Fs41422-018-0055-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41422-018-0055-1%26sid%3Dliteratum%253Aachs%26aulast%3DSun%26aufirst%3DY.%26aulast%3DZhao%26aufirst%3DX.%26aulast%3DDing%26aufirst%3DN.%26aulast%3DGao%26aufirst%3DH.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DYang%26aufirst%3DY.%26aulast%3DZhao%26aufirst%3DM.%26aulast%3DHwang%26aufirst%3DJ.%26aulast%3DSong%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DW.%26aulast%3DRao%26aufirst%3DY.%26atitle%3DPROTAC-induced%2520BTK%2520degradation%2520as%2520a%2520novel%2520therapy%2520for%2520mutated%2520BTK%2520C481S%2520induced%2520ibrutinib-resistant%2520B-cell%2520malignancies%26jtitle%3DCell%2520Res.%26date%3D2018%26volume%3D28%26spage%3D779%26epage%3D781%26doi%3D10.1038%2Fs41422-018-0055-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bai, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chinnaswamy, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McEachern, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wen, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumar, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meagher, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stuckey, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span> <span> </span><span class="NLM_article-title">A Potent and Selective Small-Molecule Degrader of STAT3 Achieves Complete Tumor Regression In Vivo</span>. <i>Cancer Cell</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>36</i></span>,  <span class="NLM_fpage">498</span>– <span class="NLM_lpage">511</span>, <span class="refDoi"> DOI: 10.1016/j.ccell.2019.10.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=10.1016%2Fj.ccell.2019.10.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=31715132" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitFCjtrjO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2019&pages=498-511&author=L.+Baiauthor=H.+Zhouauthor=R.+Xuauthor=Y.+Zhaoauthor=K.+Chinnaswamyauthor=D.+McEachernauthor=J.+Chenauthor=C.-Y.+Yangauthor=Z.+Liuauthor=M.+Wangauthor=L.+Liuauthor=H.+Jiangauthor=B.+Wenauthor=P.+Kumarauthor=J.+L.+Meagherauthor=D.+Sunauthor=J.+A.+Stuckeyauthor=S.+Wang&title=A+Potent+and+Selective+Small-Molecule+Degrader+of+STAT3+Achieves+Complete+Tumor+Regression+In+Vivo&doi=10.1016%2Fj.ccell.2019.10.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">A Potent and Selective Small-Molecule Degrader of STAT3 Achieves Complete Tumor Regression In Vivo</span></div><div class="casAuthors">Bai, Longchuan; Zhou, Haibin; Xu, Renqi; Zhao, Yujun; Chinnaswamy, Krishnapriya; McEachern, Donna; Chen, Jianyong; Yang, Chao-Yie; Liu, Zhaomin; Wang, Mi; Liu, Liu; Jiang, Hui; Wen, Bo; Kumar, Praveen; Meagher, Jennifer L.; Sun, Duxin; Stuckey, Jeanne A.; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">498-511.e17</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Signal transducer and activator of transcription 3 (STAT3) is an attractive cancer therapeutic target.  Here we report the discovery of SD-36(I), a small-mol. degrader of STAT3.  SD-36 potently induces the degrdn. of STAT3 protein in vitro and in vivo and demonstrates high selectivity over other STAT members.  Induced degrdn. of STAT3 results in a strong suppression of its transcription network in leukemia and lymphoma cells.  SD-36 inhibits the growth of a subset of acute myeloid leukemia and anaplastic large-cell lymphoma cell lines by inducing cell-cycle arrest and/or apoptosis.  SD-36 achieves complete and long-lasting tumor regression in multiple xenograft mouse models at well-tolerated dose schedules.  Degrdn. of STAT3 protein, therefore, is a promising cancer therapeutic strategy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoUaP49Ap9DxLVg90H21EOLACvtfcHk0liJ1rYlTOuaAw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitFCjtrjO&md5=71252c17d50a1a8be829352942175bd6</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1016%2Fj.ccell.2019.10.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccell.2019.10.002%26sid%3Dliteratum%253Aachs%26aulast%3DBai%26aufirst%3DL.%26aulast%3DZhou%26aufirst%3DH.%26aulast%3DXu%26aufirst%3DR.%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DChinnaswamy%26aufirst%3DK.%26aulast%3DMcEachern%26aufirst%3DD.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DYang%26aufirst%3DC.-Y.%26aulast%3DLiu%26aufirst%3DZ.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DWen%26aufirst%3DB.%26aulast%3DKumar%26aufirst%3DP.%26aulast%3DMeagher%26aufirst%3DJ.%2BL.%26aulast%3DSun%26aufirst%3DD.%26aulast%3DStuckey%26aufirst%3DJ.%2BA.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DA%2520Potent%2520and%2520Selective%2520Small-Molecule%2520Degrader%2520of%2520STAT3%2520Achieves%2520Complete%2520Tumor%2520Regression%2520In%2520Vivo%26jtitle%3DCancer%2520Cell%26date%3D2019%26volume%3D36%26spage%3D498%26epage%3D511%26doi%3D10.1016%2Fj.ccell.2019.10.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lai, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toure, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hellerschmied, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salami, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jaime-Figueroa, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ko, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hines, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Modular PROTAC Design for the Degradation of Oncogenic BCR-ABL</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">807</span>– <span class="NLM_lpage">810</span>, <span class="refDoi"> DOI: 10.1002/anie.201507634</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=10.1002%2Fanie.201507634" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvVOlt7rN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2016&pages=807-810&author=A.+C.+Laiauthor=M.+Toureauthor=D.+Hellerschmiedauthor=J.+Salamiauthor=S.+Jaime-Figueroaauthor=E.+Koauthor=J.+Hinesauthor=C.+M.+Crews&title=Modular+PROTAC+Design+for+the+Degradation+of+Oncogenic+BCR-ABL&doi=10.1002%2Fanie.201507634"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Modular PROTAC Design for the Degradation of Oncogenic BCR-ABL</span></div><div class="casAuthors">Lai, Ashton C.; Toure, Momar; Hellerschmied, Doris; Salami, Jemilat; Jaime-Figueroa, Saul; Ko, Eunhwa; Hines, John; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">807-810</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Proteolysis Targeting Chimera (PROTAC) technol. is a rapidly emerging alternative therapeutic strategy with the potential to address many of the challenges currently faced in modern drug development programs.  PROTAC technol. employs small mols. that recruit target proteins for ubiquitination and removal by the proteasome.  The synthesis of PROTAC compds. that mediate the degrdn. of c-ABL and BCR-ABL by recruiting either Cereblon or Von Hippel Lindau E3 ligases is reported.  During the course of their development, the authors discovered that the capacity of a PROTAC to induce degrdn. involves more than just target binding: the identity of the inhibitor warhead and the recruited E3 ligase largely det. the degrdn. profiles of the compds.; thus, as a starting point for PROTAC development, both the target ligand and the recruited E3 ligase should be varied to rapidly generate a PROTAC with the desired degrdn. profile.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqdcSTHt5WfILVg90H21EOLACvtfcHk0ljsyCXFEc4CXw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvVOlt7rN&md5=58a5d78f32bc8a21fd1a6e0c38b8979c</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1002%2Fanie.201507634&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201507634%26sid%3Dliteratum%253Aachs%26aulast%3DLai%26aufirst%3DA.%2BC.%26aulast%3DToure%26aufirst%3DM.%26aulast%3DHellerschmied%26aufirst%3DD.%26aulast%3DSalami%26aufirst%3DJ.%26aulast%3DJaime-Figueroa%26aufirst%3DS.%26aulast%3DKo%26aufirst%3DE.%26aulast%3DHines%26aufirst%3DJ.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DModular%2520PROTAC%2520Design%2520for%2520the%2520Degradation%2520of%2520Oncogenic%2520BCR-ABL%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2016%26volume%3D55%26spage%3D807%26epage%3D810%26doi%3D10.1002%2Fanie.201507634" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kang, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, C. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Du Ha, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hwang, J. Y.</span></span> <span> </span><span class="NLM_article-title">Induced protein degradation of anaplastic lymphoma kinase (ALK) by proteolysis targeting chimera (PROTAC)</span>. <i>Biochem. Biophys. Res. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>505</i></span>,  <span class="NLM_fpage">542</span>– <span class="NLM_lpage">547</span>, <span class="refDoi"> DOI: 10.1016/j.bbrc.2018.09.169</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=10.1016%2Fj.bbrc.2018.09.169" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=30274779" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhvVejs7fE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=505&publication_year=2018&pages=542-547&author=C.+H.+Kangauthor=D.+H.+Leeauthor=C.+O.+Leeauthor=J.+Du+Haauthor=C.+H.+Parkauthor=J.+Y.+Hwang&title=Induced+protein+degradation+of+anaplastic+lymphoma+kinase+%28ALK%29+by+proteolysis+targeting+chimera+%28PROTAC%29&doi=10.1016%2Fj.bbrc.2018.09.169"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Induced protein degradation of anaplastic lymphoma kinase (ALK) by proteolysis targeting chimera (PROTAC)</span></div><div class="casAuthors">Kang, Chung Hyo; Lee, Dong Ho; Lee, Chong Ock; Du Ha, Jae; Park, Chi Hoon; Hwang, Jong Yeon</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical and Biophysical Research Communications</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">505</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">542-547</span>CODEN:
                <span class="NLM_cas:coden">BBRCA9</span>;
        ISSN:<span class="NLM_cas:issn">0006-291X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Recently, proteolysis targeting chimera (PROTAC) technol. is highlighted in drug discovery area as a new therapeutic approach.  PROTAC as a heterobifunctional mol. is comprised of two ligands, which recruit target protein and E3 ligase, resp.  To degrade the anaplastic lymphoma kinase (ALK) fusion protein, such as NPM-ALK or EML4-ALK, we generated several ALK-PROTAC mols. consisted of ceritinib, one of the ALK inhibitors, and ligand of von Hippel-Lindau (VHL) E3 ligase.  Among these mols., TD-004 effectively induced ALK degrdn. and inhibited the growth of ALK fusion pos. cell lines, SU-DHL-1 and H3122.  We also confirmed that TD-004 significantly reduced the tumor growth in H3122 xenograft model.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq8JUOZ5fYNW7Vg90H21EOLACvtfcHk0ljsyCXFEc4CXw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhvVejs7fE&md5=cac9252fca98ebec4ad8e5aeed7b7550</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1016%2Fj.bbrc.2018.09.169&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbrc.2018.09.169%26sid%3Dliteratum%253Aachs%26aulast%3DKang%26aufirst%3DC.%2BH.%26aulast%3DLee%26aufirst%3DD.%2BH.%26aulast%3DLee%26aufirst%3DC.%2BO.%26aulast%3DDu%2BHa%26aufirst%3DJ.%26aulast%3DPark%26aufirst%3DC.%2BH.%26aulast%3DHwang%26aufirst%3DJ.%2BY.%26atitle%3DInduced%2520protein%2520degradation%2520of%2520anaplastic%2520lymphoma%2520kinase%2520%2528ALK%2529%2520by%2520proteolysis%2520targeting%2520chimera%2520%2528PROTAC%2529%26jtitle%3DBiochem.%2520Biophys.%2520Res.%2520Commun.%26date%3D2018%26volume%3D505%26spage%3D542%26epage%3D547%26doi%3D10.1016%2Fj.bbrc.2018.09.169" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Burslem, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hines, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Enhancing Antiproliferative Activity and Selectivity of a FLT-3 Inhibitor by Proteolysis Targeting Chimera Conversion</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>140</i></span>,  <span class="NLM_fpage">16428</span>– <span class="NLM_lpage">16432</span>, <span class="refDoi"> DOI: 10.1021/jacs.8b10320</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.8b10320" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitF2qtr7P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=140&publication_year=2018&pages=16428-16432&author=G.+M.+Burslemauthor=J.+Songauthor=X.+Chenauthor=J.+Hinesauthor=C.+M.+Crews&title=Enhancing+Antiproliferative+Activity+and+Selectivity+of+a+FLT-3+Inhibitor+by+Proteolysis+Targeting+Chimera+Conversion&doi=10.1021%2Fjacs.8b10320"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Enhancing Antiproliferative Activity and Selectivity of a FLT-3 Inhibitor by Proteolysis Targeting Chimera Conversion</span></div><div class="casAuthors">Burslem, George M.; Song, Jayoung; Chen, Xin; Hines, John; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">140</span>
        (<span class="NLM_cas:issue">48</span>),
    <span class="NLM_cas:pages">16428-16432</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The receptor tyrosine kinase FLT-3 is frequently mutated in acute myeloid leukemia; however, current small mol. inhibitors suffer from limited efficacy in the clinic.  Conversion of a FLT-3 inhibitor (quizartinib) into a proteolysis targeting chimera (PROTAC) results in a compd. that induces degrdn. of FLT-3 ITD mutant at low nanomolar concns.  Furthermore, the PROTAC is capable of inhibiting cell growth more potently than the warhead alone while inhibiting fewer off-target kinases.  This enhanced antiproliferative activity occurs, despite a slight redn. in the PROTAC's kinase inhibitory activity, via an increased level of apoptosis induction suggesting nonkinase roles for the FLT-3 ITD protein.  Addnl., the PROTAC is capable of inducing FLT-3 ITD degrdn. in vivo.  These results suggest that degrdn. of FLT-3 ITD may provide a useful method for therapeutic intervention.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpBxw_nAEQydLVg90H21EOLACvtfcHk0ljsyCXFEc4CXw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitF2qtr7P&md5=52529d0c3f49052bd2a66c893e98e694</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1021%2Fjacs.8b10320&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.8b10320%26sid%3Dliteratum%253Aachs%26aulast%3DBurslem%26aufirst%3DG.%2BM.%26aulast%3DSong%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DHines%26aufirst%3DJ.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DEnhancing%2520Antiproliferative%2520Activity%2520and%2520Selectivity%2520of%2520a%2520FLT-3%2520Inhibitor%2520by%2520Proteolysis%2520Targeting%2520Chimera%2520Conversion%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2018%26volume%3D140%26spage%3D16428%26epage%3D16432%26doi%3D10.1021%2Fjacs.8b10320" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cromm, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Samarasinghe, K. T. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hines, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Addressing Kinase-Independent Functions of Fak via PROTAC-Mediated Degradation</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>140</i></span>,  <span class="NLM_fpage">17019</span>– <span class="NLM_lpage">17026</span>, <span class="refDoi"> DOI: 10.1021/jacs.8b08008</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.8b08008" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=1%3ACAS%3A528%3ADC%252BC1cXit1CitLnF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=140&publication_year=2018&pages=17019-17026&author=P.+M.+Crommauthor=K.+T.+G.+Samarasingheauthor=J.+Hinesauthor=C.+M.+Crews&title=Addressing+Kinase-Independent+Functions+of+Fak+via+PROTAC-Mediated+Degradation&doi=10.1021%2Fjacs.8b08008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Addressing Kinase-Independent Functions of Fak via PROTAC-Mediated Degradation</span></div><div class="casAuthors">Cromm, Philipp M.; Samarasinghe, Kusal T. G.; Hines, John; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">140</span>
        (<span class="NLM_cas:issue">49</span>),
    <span class="NLM_cas:pages">17019-17026</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Enzymic inhibition proved to be a successful modality for the development of many small-mol. drugs.  In recent years, small-mol.-induced protein degrdn. has emerged as an orthogonal therapeutic strategy that has the potential to expand the druggable target space.  Focal adhesion kinase (Fak) is a key player in tumor invasion and metastasis, acting simultaneously as a kinase and a scaffold for several signaling proteins.  While previous efforts to modulate Fak activity were limited to kinase inhibitors with low success in clin. studies, protein degrdn. offers a possibility to simultaneously block Fak's kinase signaling and scaffolding capabilities.  Here, the authors report the development of a selective and potent Fak degrader, PROTAC-3, which outperforms a clin. candidate, defactinib, with respect to Fak activation as well as Fak-mediated cell migration and invasion.  These results underline the potential that PROTACs offer in expanding the druggable space and controlling protein functions that are not easily addressed by traditional small-mol. therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqTn8r4NmDDObVg90H21EOLACvtfcHk0ljwY9_p2mohsw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXit1CitLnF&md5=b97b1a646a8f9dbf7a2bed640d0f001d</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1021%2Fjacs.8b08008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.8b08008%26sid%3Dliteratum%253Aachs%26aulast%3DCromm%26aufirst%3DP.%2BM.%26aulast%3DSamarasinghe%26aufirst%3DK.%2BT.%2BG.%26aulast%3DHines%26aufirst%3DJ.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DAddressing%2520Kinase-Independent%2520Functions%2520of%2520Fak%2520via%2520PROTAC-Mediated%2520Degradation%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2018%26volume%3D140%26spage%3D17019%26epage%3D17026%26doi%3D10.1021%2Fjacs.8b08008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Powell, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Donovan, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nowak, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loehr, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bahcall, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jänne, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">George, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span> <span> </span><span class="NLM_article-title">Chemically Induced Degradation of Anaplastic Lymphoma Kinase (ALK)</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">4249</span>– <span class="NLM_lpage">4255</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b01655</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b01655" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=1%3ACAS%3A528%3ADC%252BC1cXns1OitrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=4249-4255&author=C.+E.+Powellauthor=Y.+Gaoauthor=L.+Tanauthor=K.+A.+Donovanauthor=R.+P.+Nowakauthor=A.+Loehrauthor=M.+Bahcallauthor=E.+S.+Fischerauthor=P.+A.+J%C3%A4nneauthor=R.+E.+Georgeauthor=N.+S.+Gray&title=Chemically+Induced+Degradation+of+Anaplastic+Lymphoma+Kinase+%28ALK%29&doi=10.1021%2Facs.jmedchem.7b01655"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Chemically Induced Degradation of Anaplastic Lymphoma Kinase (ALK)</span></div><div class="casAuthors">Powell, Chelsea E.; Gao, Yang; Tan, Li; Donovan, Katherine A.; Nowak, Radoslaw P.; Loehr, Amanda; Bahcall, Magda; Fischer, Eric S.; Janne, Pasi A.; George, Rani E.; Gray, Nathanael S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">4249-4255</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We present the development of the first small mol. degraders that can induce anaplastic lymphoma kinase (ALK) degrdn., including in non-small-cell lung cancer (NSCLC), anaplastic large-cell lymphoma (ALCL), and neuroblastoma (NB) cell lines.  These degraders were developed through conjugation of known pyrimidine-based ALK inhibitors, TAE684 or LDK378, and the cereblon ligand pomalidomide.  We demonstrate that in some cell types degrader potency is compromised by expression of drug transporter ABCB1.  In addn., proteomic profiling demonstrated that these compds. also promote the degrdn. of addnl. kinases including PTK2 (FAK), Aurora A, FER, and RPS6KA1 (RSK1).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp7mDH7Sj8EqbVg90H21EOLACvtfcHk0ljwY9_p2mohsw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXns1OitrY%253D&md5=486db11b3fbc2283700f38b09cc39bd7</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b01655&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b01655%26sid%3Dliteratum%253Aachs%26aulast%3DPowell%26aufirst%3DC.%2BE.%26aulast%3DGao%26aufirst%3DY.%26aulast%3DTan%26aufirst%3DL.%26aulast%3DDonovan%26aufirst%3DK.%2BA.%26aulast%3DNowak%26aufirst%3DR.%2BP.%26aulast%3DLoehr%26aufirst%3DA.%26aulast%3DBahcall%26aufirst%3DM.%26aulast%3DFischer%26aufirst%3DE.%2BS.%26aulast%3DJ%25C3%25A4nne%26aufirst%3DP.%2BA.%26aulast%3DGeorge%26aufirst%3DR.%2BE.%26aulast%3DGray%26aufirst%3DN.%2BS.%26atitle%3DChemically%2520Induced%2520Degradation%2520of%2520Anaplastic%2520Lymphoma%2520Kinase%2520%2528ALK%2529%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D4249%26epage%3D4255%26doi%3D10.1021%2Facs.jmedchem.7b01655" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brand, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bauer, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Donovan, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nowak, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kwiatkowski, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winter, G. E.</span></span> <span> </span><span class="NLM_article-title">Homolog-Selective Degradation as a Strategy to Probe the Function of CDK6 in AML</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">300</span>– <span class="NLM_lpage">306</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2018.11.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=10.1016%2Fj.chembiol.2018.11.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=30595531" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=1%3ACAS%3A528%3ADC%252BC1MXnsVSj" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2019&pages=300-306&author=M.+Brandauthor=B.+Jiangauthor=S.+Bauerauthor=K.+A.+Donovanauthor=Y.+Liangauthor=E.+S.+Wangauthor=R.+P.+Nowakauthor=J.+C.+Yuanauthor=T.+Zhangauthor=N.+Kwiatkowskiauthor=A.+C.+M%C3%BCllerauthor=E.+S.+Fischerauthor=N.+S.+Grayauthor=G.+E.+Winter&title=Homolog-Selective+Degradation+as+a+Strategy+to+Probe+the+Function+of+CDK6+in+AML&doi=10.1016%2Fj.chembiol.2018.11.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Homolog-Selective Degradation as a Strategy to Probe the Function of CDK6 in AML</span></div><div class="casAuthors">Brand, Matthias; Jiang, Baishan; Bauer, Sophie; Donovan, Katherine A.; Liang, Yanke; Wang, Eric S.; Nowak, Radoslaw P.; Yuan, Jingting C.; Zhang, Tinghu; Kwiatkowski, Nicholas; Mueller, Andre C.; Fischer, Eric S.; Gray, Nathanael S.; Winter, Georg E.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">300-306.e9</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The design of selective small mols. is often stymied by similar ligand binding pockets.  Here, we report BSJ-03-123, a phthalimide-based degrader that exploits protein-interface determinants to achieve proteome-wide selectivity for the degrdn. of cyclin-dependent kinase 6 (CDK6).  Pharmacol. CDK6 degrdn. targets a selective dependency of acute myeloid leukemia cells, and transcriptomics and phosphoproteomics profiling of acute degrdn. of CDK6 enabled dynamic mapping of its immediate role in coordinating signaling and transcription.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGognoz8FFoR9LVg90H21EOLACvtfcHk0ljwY9_p2mohsw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXnsVSj&md5=f8c1c3520c6c0218763d8769c59b4342</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2018.11.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2018.11.006%26sid%3Dliteratum%253Aachs%26aulast%3DBrand%26aufirst%3DM.%26aulast%3DJiang%26aufirst%3DB.%26aulast%3DBauer%26aufirst%3DS.%26aulast%3DDonovan%26aufirst%3DK.%2BA.%26aulast%3DLiang%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DE.%2BS.%26aulast%3DNowak%26aufirst%3DR.%2BP.%26aulast%3DYuan%26aufirst%3DJ.%2BC.%26aulast%3DZhang%26aufirst%3DT.%26aulast%3DKwiatkowski%26aufirst%3DN.%26aulast%3DM%25C3%25BCller%26aufirst%3DA.%2BC.%26aulast%3DFischer%26aufirst%3DE.%2BS.%26aulast%3DGray%26aufirst%3DN.%2BS.%26aulast%3DWinter%26aufirst%3DG.%2BE.%26atitle%3DHomolog-Selective%2520Degradation%2520as%2520a%2520Strategy%2520to%2520Probe%2520the%2520Function%2520of%2520CDK6%2520in%2520AML%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2019%26volume%3D26%26spage%3D300%26epage%3D306%26doi%3D10.1016%2Fj.chembiol.2018.11.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Khan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lv, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thummuri, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wiegand, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pei, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharma, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCurdy, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuruvilla, V. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baran, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferrando, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, Y.-m.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rogojina, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Houghton, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hromas, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Konopleva, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, D.</span></span> <span> </span><span class="NLM_article-title">A selective BCL-XL PROTAC degrader achieves safe and potent antitumor activity</span>. <i>Nat. Med.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">1938</span>– <span class="NLM_lpage">1947</span>, <span class="refDoi"> DOI: 10.1038/s41591-019-0668-z</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=10.1038%2Fs41591-019-0668-z" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=31792461" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=1%3ACAS%3A528%3ADC%252BC1MXit1yjsrrO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2019&pages=1938-1947&author=S.+Khanauthor=X.+Zhangauthor=D.+Lvauthor=Q.+Zhangauthor=Y.+Heauthor=P.+Zhangauthor=X.+Liuauthor=D.+Thummuriauthor=Y.+Yuanauthor=J.+S.+Wiegandauthor=J.+Peiauthor=W.+Zhangauthor=A.+Sharmaauthor=C.+R.+McCurdyauthor=V.+M.+Kuruvillaauthor=N.+Baranauthor=A.+A.+Ferrandoauthor=Y.-m.+Kimauthor=A.+Rogojinaauthor=P.+J.+Houghtonauthor=G.+Huangauthor=R.+Hromasauthor=M.+Konoplevaauthor=G.+Zhengauthor=D.+Zhou&title=A+selective+BCL-XL+PROTAC+degrader+achieves+safe+and+potent+antitumor+activity&doi=10.1038%2Fs41591-019-0668-z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">A selective BCL-XL PROTAC degrader achieves safe and potent antitumor activity</span></div><div class="casAuthors">Khan, Sajid; Zhang, Xuan; Lv, Dongwen; Zhang, Qi; He, Yonghan; Zhang, Peiyi; Liu, Xingui; Thummuri, Dinesh; Yuan, Yaxia; Wiegand, Janet S.; Pei, Jing; Zhang, Weizhou; Sharma, Abhisheak; McCurdy, Christopher R.; Kuruvilla, Vinitha M.; Baran, Natalia; Ferrando, Adolfo A.; Kim, Yong-mi; Rogojina, Anna; Houghton, Peter J.; Huang, Guangcun; Hromas, Robert; Konopleva, Marina; Zheng, Guangrong; Zhou, Daohong</div><div class="citationInfo"><span class="NLM_cas:title">Nature Medicine (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1938-1947</span>CODEN:
                <span class="NLM_cas:coden">NAMEFI</span>;
        ISSN:<span class="NLM_cas:issn">1078-8956</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">B-cell lymphoma extra large (BCL-XL) is a well-validated cancer target.  However, the on-target and dose-limiting thrombocytopenia limits the use of BCL-XL inhibitors, such as ABT263, as safe and effective anticancer agents.  To reduce the toxicity of ABT263, we converted it into DT2216, a BCL-XL proteolysis-targeting chimera (PROTAC), that targets BCL-XL to the Von Hippel-Lindau (VHL) E3 ligase for degrdn.  We found that DT2216 was more potent against various BCL-XL-dependent leukemia and cancer cells but considerably less toxic to platelets than ABT263 in vitro because VHL is poorly expressed in platelets.  In vivo, DT2216 effectively inhibits the growth of several xenograft tumors as a single agent or in combination with other chemotherapeutic agents, without causing appreciable thrombocytopenia.  These findings demonstrate the potential to use PROTAC technol. to reduce on-target drug toxicities and rescue the therapeutic potential of previously undruggable targets.  Furthermore, DT2216 may be developed as a safe first-in-class anticancer agent targeting BCL-XL.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGposmmgxVNCRLVg90H21EOLACvtfcHk0lj9-pfWgH5GVg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXit1yjsrrO&md5=b07678c880569acac428ec479f002974</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1038%2Fs41591-019-0668-z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41591-019-0668-z%26sid%3Dliteratum%253Aachs%26aulast%3DKhan%26aufirst%3DS.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DLv%26aufirst%3DD.%26aulast%3DZhang%26aufirst%3DQ.%26aulast%3DHe%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DP.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DThummuri%26aufirst%3DD.%26aulast%3DYuan%26aufirst%3DY.%26aulast%3DWiegand%26aufirst%3DJ.%2BS.%26aulast%3DPei%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DW.%26aulast%3DSharma%26aufirst%3DA.%26aulast%3DMcCurdy%26aufirst%3DC.%2BR.%26aulast%3DKuruvilla%26aufirst%3DV.%2BM.%26aulast%3DBaran%26aufirst%3DN.%26aulast%3DFerrando%26aufirst%3DA.%2BA.%26aulast%3DKim%26aufirst%3DY.-m.%26aulast%3DRogojina%26aufirst%3DA.%26aulast%3DHoughton%26aufirst%3DP.%2BJ.%26aulast%3DHuang%26aufirst%3DG.%26aulast%3DHromas%26aufirst%3DR.%26aulast%3DKonopleva%26aufirst%3DM.%26aulast%3DZheng%26aufirst%3DG.%26aulast%3DZhou%26aufirst%3DD.%26atitle%3DA%2520selective%2520BCL-XL%2520PROTAC%2520degrader%2520achieves%2520safe%2520and%2520potent%2520antitumor%2520activity%26jtitle%3DNat.%2520Med.%26date%3D2019%26volume%3D25%26spage%3D1938%26epage%3D1947%26doi%3D10.1038%2Fs41591-019-0668-z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ma, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stratikopoulos, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, T. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwarz, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leshchenko, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rialdi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dale, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lagana, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guccione, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parekh, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parsons, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span> <span> </span><span class="NLM_article-title">Discovery of a first-in-class EZH2 selective degrader</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">214</span>, <span class="refDoi"> DOI: 10.1038/s41589-019-0421-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=10.1038%2Fs41589-019-0421-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=31819273" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2019&pages=214&author=A.+Maauthor=E.+Stratikopoulosauthor=K.+S.+Parkauthor=J.+Weiauthor=T.+C.+Martinauthor=X.+Yangauthor=M.+Schwarzauthor=V.+Leshchenkoauthor=A.+Rialdiauthor=B.+Daleauthor=A.+Laganaauthor=E.+Guccioneauthor=S.+Parekhauthor=R.+Parsonsauthor=J.+Jin&title=Discovery+of+a+first-in-class+EZH2+selective+degrader&doi=10.1038%2Fs41589-019-0421-4"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1038%2Fs41589-019-0421-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41589-019-0421-4%26sid%3Dliteratum%253Aachs%26aulast%3DMa%26aufirst%3DA.%26aulast%3DStratikopoulos%26aufirst%3DE.%26aulast%3DPark%26aufirst%3DK.%2BS.%26aulast%3DWei%26aufirst%3DJ.%26aulast%3DMartin%26aufirst%3DT.%2BC.%26aulast%3DYang%26aufirst%3DX.%26aulast%3DSchwarz%26aufirst%3DM.%26aulast%3DLeshchenko%26aufirst%3DV.%26aulast%3DRialdi%26aufirst%3DA.%26aulast%3DDale%26aufirst%3DB.%26aulast%3DLagana%26aufirst%3DA.%26aulast%3DGuccione%26aufirst%3DE.%26aulast%3DParekh%26aufirst%3DS.%26aulast%3DParsons%26aufirst%3DR.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DDiscovery%2520of%2520a%2520first-in-class%2520EZH2%2520selective%2520degrader%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2019%26volume%3D16%26spage%3D214%26doi%3D10.1038%2Fs41589-019-0421-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ren, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span> <span> </span><span class="NLM_article-title">Discovery of novel resorcinol diphenyl ether-based PROTAC-like molecules as dual inhibitors and degraders of PD-L1</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>199</i></span>,  <span class="NLM_fpage">112377</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2020.112377</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=10.1016%2Fj.ejmech.2020.112377" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=32388281" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=1%3ACAS%3A528%3ADC%252BB3cXptFGisLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=199&publication_year=2020&pages=112377&author=B.+Chengauthor=Y.+Renauthor=H.+Caoauthor=J.+Chen&title=Discovery+of+novel+resorcinol+diphenyl+ether-based+PROTAC-like+molecules+as+dual+inhibitors+and+degraders+of+PD-L1&doi=10.1016%2Fj.ejmech.2020.112377"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of novel resorcinol diphenyl ether-based PROTAC-like molecules as dual inhibitors and degraders of PD-L1</span></div><div class="casAuthors">Cheng, Binbin; Ren, Yichang; Cao, Hao; Chen, Jianjun</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">199</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">112377</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">Novel resorcinol di-Ph ether-based PROTACs (PROteolysis TArgeting Chimeras) were designed and evaluated for their inhibitory activity against the programmed cell death-1/programmed cell death-ligand 1 (PD-1/PD-L1) pathway and their ability to degrade PD-L1 protein.  Most of the compds. displayed excellent inhibitory activities against PD-1/PD-L1, as assessed by the homogeneous time-resolved fluorescence (HTRF) binding assay, with IC50 values ranging from 25 nM to 200 nM.  Among them, compd. P22 (I) is one of the best with an IC50 value of 39.2 nM.  In addn. to inhibiting PD-1/PD-L1 interaction, P22 also significantly restored the immunity repressed in a co-culture model of Hep3B/OS-8/hPD-L1 and CD3 T cells.  Furthermore, flow cytometry (FCM) and western-blot data demonstrated that P22 could moderately reduce the protein levels of PD-L1 in a lysosome-dependent manner, which may contribute to its immune effects.  Preliminary FCM and western-blot data suggest that it is possible to build PD-L1-targeting PROTAC-like mols. based on PD-1/PD-L1 small mol. inhibitors, though these compds. showed only modest degrdn. efficiencies.  Collectively, this work suggests that P22 may serve as a starting point for exploring the degrdn. of PD-L1 by PROTAC-like strategy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoiG4nI2klh77Vg90H21EOLACvtfcHk0lj9-pfWgH5GVg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXptFGisLk%253D&md5=ec722d2e118668f601c0f9648e6f893b</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2020.112377&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2020.112377%26sid%3Dliteratum%253Aachs%26aulast%3DCheng%26aufirst%3DB.%26aulast%3DRen%26aufirst%3DY.%26aulast%3DCao%26aufirst%3DH.%26aulast%3DChen%26aufirst%3DJ.%26atitle%3DDiscovery%2520of%2520novel%2520resorcinol%2520diphenyl%2520ether-based%2520PROTAC-like%2520molecules%2520as%2520dual%2520inhibitors%2520and%2520degraders%2520of%2520PD-L1%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2020%26volume%3D199%26spage%3D112377%26doi%3D10.1016%2Fj.ejmech.2020.112377" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sun, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ren, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kong, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhong, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qiu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, B.</span></span> <span> </span><span class="NLM_article-title">Development of a Brigatinib degrader (SIAIS117) as a potential treatment for ALK positive cancer resistance</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>193</i></span>,  <span class="NLM_fpage">112190</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2020.112190</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=10.1016%2Fj.ejmech.2020.112190" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=32179332" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=1%3ACAS%3A528%3ADC%252BB3cXkvF2qu70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=193&publication_year=2020&pages=112190&author=N.+Sunauthor=C.+Renauthor=Y.+Kongauthor=H.+Zhongauthor=J.+Chenauthor=Y.+Liauthor=J.+Zhangauthor=Y.+Zhouauthor=X.+Qiuauthor=H.+Linauthor=X.+Songauthor=X.+Yangauthor=B.+Jiang&title=Development+of+a+Brigatinib+degrader+%28SIAIS117%29+as+a+potential+treatment+for+ALK+positive+cancer+resistance&doi=10.1016%2Fj.ejmech.2020.112190"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Development of a Brigatinib degrader (SIAIS117) as a potential treatment for ALK positive cancer resistance</span></div><div class="casAuthors">Sun, Ning; Ren, Chaowei; Kong, Ying; Zhong, Hui; Chen, Jinju; Li, Yan; Zhang, Jianshui; Zhou, Yuedong; Qiu, Xing; Lin, Haifan; Song, Xiaoling; Yang, Xiaobao; Jiang, Biao</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">193</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">112190</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">EML4-ALK and NPM-ALK fusion proteins possess constitutively activated ALK (anaplastic lymphoma kinase) activity, which in turn leads to the development of non-small cell lung cancer and anaplastic large-cell lymphomas (ALCLs).  FDA-approved ALK inhibitor drugs cause significant cancer regression.  However, drug resistance eventually occurs and it becomes a big obstacle in clinic.  Novel proteolysis targeting chimera (PROTAC) technol. platform provides a potential therapeutic strategy for drug resistance.  Herein, we designed and synthesized a series of ALK PROTACs based on Brigatinib and VHL-1 conjunction, and screened SIAIS117 as the best degrader which not only blocked the growth of SR and H2228 cancer cell lines, but also degraded ALK protein.  In addn., SIAIS117 also showed much better growth inhibition effect than Brigatinib on 293T cell line that exogenously expressed G1202R-resistant ALK proteins.  Furthermore, it also degraded G1202R mutant ALK protein in vitro.  At last, it has the potentially anti-proliferation ability of small cell lung cancer.  Thus, we have successfully generated the degrader SIAIS117 that can potentially overcome resistance in cancer targeted therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrX9rzk88HMe7Vg90H21EOLACvtfcHk0lh1DN1MBsAiyA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXkvF2qu70%253D&md5=61f0228111c806ae62473fc450fce276</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2020.112190&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2020.112190%26sid%3Dliteratum%253Aachs%26aulast%3DSun%26aufirst%3DN.%26aulast%3DRen%26aufirst%3DC.%26aulast%3DKong%26aufirst%3DY.%26aulast%3DZhong%26aufirst%3DH.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DZhou%26aufirst%3DY.%26aulast%3DQiu%26aufirst%3DX.%26aulast%3DLin%26aufirst%3DH.%26aulast%3DSong%26aufirst%3DX.%26aulast%3DYang%26aufirst%3DX.%26aulast%3DJiang%26aufirst%3DB.%26atitle%3DDevelopment%2520of%2520a%2520Brigatinib%2520degrader%2520%2528SIAIS117%2529%2520as%2520a%2520potential%2520treatment%2520for%2520ALK%2520positive%2520cancer%2520resistance%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2020%26volume%3D193%26spage%3D112190%26doi%3D10.1016%2Fj.ejmech.2020.112190" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ren, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kong, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yin, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, B.</span></span> <span> </span><span class="NLM_article-title">Discovery of SIAIS178 as an Effective BCR-ABL Degrader by Recruiting Von Hippel-Lindau (VHL) E3 Ubiquitin Ligase</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">9281</span>– <span class="NLM_lpage">9298</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b01264</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b01264" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhvVantrnN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=9281-9298&author=Q.+Zhaoauthor=C.+Renauthor=L.+Liuauthor=J.+Chenauthor=Y.+Shaoauthor=N.+Sunauthor=R.+Sunauthor=Y.+Kongauthor=X.+Dingauthor=X.+Zhangauthor=Y.+Xuauthor=B.+Yangauthor=Q.+Yinauthor=X.+Yangauthor=B.+Jiang&title=Discovery+of+SIAIS178+as+an+Effective+BCR-ABL+Degrader+by+Recruiting+Von+Hippel-Lindau+%28VHL%29+E3+Ubiquitin+Ligase&doi=10.1021%2Facs.jmedchem.9b01264"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of SIAIS178 as an Effective BCR-ABL Degrader by Recruiting Von Hippel-Lindau (VHL) E3 Ubiquitin Ligase</span></div><div class="casAuthors">Zhao, Quanju; Ren, Chaowei; Liu, Linyi; Chen, Jinju; Shao, Yubao; Sun, Ning; Sun, Renhong; Kong, Ying; Ding, Xinyu; Zhang, Xianfang; Xu, Youwei; Yang, Bei; Yin, Qianqian; Yang, Xiaobao; Jiang, Biao</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">9281-9298</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The oncogenic fusion protein BCR-ABL is the driving force of leukemogenesis in chronic myeloid leukemia (CML).  Despite great progress for CML treatment through application of tyrosine kinase inhibitors (TKIs) against BCR-ABL, long-term drug administration and clin. resistance continue to be an issue.  Herein, we described the design, synthesis, and evaluation of novel proteolysis-targeting chimeric (PROTAC) small mols. targeting BCR-ABL which connect dasatinib and VHL E3 ubiquitin ligase ligand by extensive optimization of linkers.  Our efforts have yielded SIAIS178 (19), which induces proper interaction between BCR-ABL and VHL ligase leading to effective degrdn. of BCR-ABL protein, achieves significant growth inhibition of BCR-ABL+ leukemic cells in vitro, and induces substantial tumor regression against K562 xenograft tumors in vivo.  In addn., SIAIS178 also degrades several clin. relevant resistance-conferring mutations.  Our data indicate that SIAIS178 as efficacious BCR-ABL degrader warrants extensive further investigation for the treatment of BCR-ABL+ leukemia.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrLBFfujSx6ZbVg90H21EOLACvtfcHk0lh1DN1MBsAiyA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhvVantrnN&md5=50ace274dbc6ef3931f86504a1be93a7</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b01264&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b01264%26sid%3Dliteratum%253Aachs%26aulast%3DZhao%26aufirst%3DQ.%26aulast%3DRen%26aufirst%3DC.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DShao%26aufirst%3DY.%26aulast%3DSun%26aufirst%3DN.%26aulast%3DSun%26aufirst%3DR.%26aulast%3DKong%26aufirst%3DY.%26aulast%3DDing%26aufirst%3DX.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DXu%26aufirst%3DY.%26aulast%3DYang%26aufirst%3DB.%26aulast%3DYin%26aufirst%3DQ.%26aulast%3DYang%26aufirst%3DX.%26aulast%3DJiang%26aufirst%3DB.%26atitle%3DDiscovery%2520of%2520SIAIS178%2520as%2520an%2520Effective%2520BCR-ABL%2520Degrader%2520by%2520Recruiting%2520Von%2520Hippel-Lindau%2520%2528VHL%2529%2520E3%2520Ubiquitin%2520Ligase%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D9281%26epage%3D9298%26doi%3D10.1021%2Facs.jmedchem.9b01264" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaniskan, H. Ü.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, W.</span></span> <span> </span><span class="NLM_article-title">Light-induced control of protein destruction by opto-PROTAC</span>. <i>Sci. Adv.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>6</i></span>, <span class="NLM_elocation-id">eaay5154</span> <span class="refDoi"> DOI: 10.1126/sciadv.aay5154</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=10.1126%2Fsciadv.aay5154" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=32128407" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2020&author=J.+Liuauthor=H.+Chenauthor=L.+Maauthor=Z.+Heauthor=D.+Wangauthor=Y.+Liuauthor=Q.+Linauthor=T.+Zhangauthor=N.+Grayauthor=H.+%C3%9C.+Kaniskanauthor=J.+Jinauthor=W.+Wei&title=Light-induced+control+of+protein+destruction+by+opto-PROTAC&doi=10.1126%2Fsciadv.aay5154"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1126%2Fsciadv.aay5154&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fsciadv.aay5154%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DH.%26aulast%3DMa%26aufirst%3DL.%26aulast%3DHe%26aufirst%3DZ.%26aulast%3DWang%26aufirst%3DD.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DLin%26aufirst%3DQ.%26aulast%3DZhang%26aufirst%3DT.%26aulast%3DGray%26aufirst%3DN.%26aulast%3DKaniskan%26aufirst%3DH.%2B%25C3%259C.%26aulast%3DJin%26aufirst%3DJ.%26aulast%3DWei%26aufirst%3DW.%26atitle%3DLight-induced%2520control%2520of%2520protein%2520destruction%2520by%2520opto-PROTAC%26jtitle%3DSci.%2520Adv.%26date%3D2020%26volume%3D6%26doi%3D10.1126%2Fsciadv.aay5154" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Reynders, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsuura, B. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bérouti, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simoneschi, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marzio, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pagano, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trauner, D.</span></span> <span> </span><span class="NLM_article-title">PHOTACs enable optical control of protein degradation</span>. <i>Sci. Adv.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>6</i></span>, <span class="NLM_elocation-id">eaay5064</span> <span class="refDoi"> DOI: 10.1126/sciadv.aay5064</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=10.1126%2Fsciadv.aay5064" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=32128406" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2020&author=M.+Reyndersauthor=B.+S.+Matsuuraauthor=M.+B%C3%A9routiauthor=D.+Simoneschiauthor=A.+Marzioauthor=M.+Paganoauthor=D.+Trauner&title=PHOTACs+enable+optical+control+of+protein+degradation&doi=10.1126%2Fsciadv.aay5064"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1126%2Fsciadv.aay5064&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fsciadv.aay5064%26sid%3Dliteratum%253Aachs%26aulast%3DReynders%26aufirst%3DM.%26aulast%3DMatsuura%26aufirst%3DB.%2BS.%26aulast%3DB%25C3%25A9routi%26aufirst%3DM.%26aulast%3DSimoneschi%26aufirst%3DD.%26aulast%3DMarzio%26aufirst%3DA.%26aulast%3DPagano%26aufirst%3DM.%26aulast%3DTrauner%26aufirst%3DD.%26atitle%3DPHOTACs%2520enable%2520optical%2520control%2520of%2520protein%2520degradation%26jtitle%3DSci.%2520Adv.%26date%3D2020%26volume%3D6%26doi%3D10.1126%2Fsciadv.aay5064" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ren, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kong, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhong, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, B.</span></span> <span> </span><span class="NLM_article-title">Structure-based discovery of SIAIS001 as an oral bioavailability ALK degrader constructed from Alectinib</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2021</span>,  <span class="NLM_volume"><i>217</i></span>,  <span class="NLM_fpage">113335</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2021.113335</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=10.1016%2Fj.ejmech.2021.113335" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=33751979" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=1%3ACAS%3A528%3ADC%252BB3MXnt1Ohsrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=217&publication_year=2021&pages=113335&author=C.+Renauthor=N.+Sunauthor=Y.+Kongauthor=X.+Quauthor=H.+Liuauthor=H.+Zhongauthor=X.+Songauthor=X.+Yangauthor=B.+Jiang&title=Structure-based+discovery+of+SIAIS001+as+an+oral+bioavailability+ALK+degrader+constructed+from+Alectinib&doi=10.1016%2Fj.ejmech.2021.113335"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-based discovery of SIAIS001 as an oral bioavailability ALK degrader constructed from Alectinib</span></div><div class="casAuthors">Ren, Chaowei; Sun, Ning; Kong, Ying; Qu, Xiaojuan; Liu, Haixia; Zhong, Hui; Song, Xiaoling; Yang, Xiaobao; Jiang, Biao</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2021</span>),
    <span class="NLM_cas:volume">217</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">113335</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">Fusion proteins of the anaplastic lymphoma kinase (ALK) are promising therapeutic targets for cancer and other human diseases, esp. for non-small cell lung cancer (NSCLC) and anaplastic large-cell lymphomas (ALCLs).  We described herein a structure-based design, synthesis, and evaluation of ALK PROTACs (proteolysis-targeting chimeras) based on Alectinib as the warhead.  We firstly screened CRBN ligands as the E3 ligase moiety, then obtained a series of potent ALK degraders based on different CRBN ligands, exemplified by SIAIS091 and SIAIS001 with lenalidomide/thalidomide-based linkers.  Both of them induced effective ALK degrdn. at low nanomolar concns. in cells, and showed much better growth inhibition effects than Alectinib.  SIAIS091 or SIAIS001 also promoted cell cycle arrest in G1/S phase.  Finally, SIAIS001 exhibited good oral bioavailability in Pharmacokinetics study.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoZW4fwPhRGNLVg90H21EOLACvtfcHk0ljX1FRheRfO5Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3MXnt1Ohsrs%253D&md5=b1dc676e47efb1fdf8e9fb50cafec635</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2021.113335&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2021.113335%26sid%3Dliteratum%253Aachs%26aulast%3DRen%26aufirst%3DC.%26aulast%3DSun%26aufirst%3DN.%26aulast%3DKong%26aufirst%3DY.%26aulast%3DQu%26aufirst%3DX.%26aulast%3DLiu%26aufirst%3DH.%26aulast%3DZhong%26aufirst%3DH.%26aulast%3DSong%26aufirst%3DX.%26aulast%3DYang%26aufirst%3DX.%26aulast%3DJiang%26aufirst%3DB.%26atitle%3DStructure-based%2520discovery%2520of%2520SIAIS001%2520as%2520an%2520oral%2520bioavailability%2520ALK%2520degrader%2520constructed%2520from%2520Alectinib%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2021%26volume%3D217%26spage%3D113335%26doi%3D10.1016%2Fj.ejmech.2021.113335" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sun, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tong, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rao, Y.</span></span> <span> </span><span class="NLM_article-title">PROTACs: great opportunities for academia and industry</span>. <i>Signal Transduction Targeted Ther.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">64</span>, <span class="refDoi"> DOI: 10.1038/s41392-019-0101-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=10.1038%2Fs41392-019-0101-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=31885879" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=1%3ACAS%3A280%3ADC%252BB3MbkvFCrtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2019&pages=64&author=X.+Sunauthor=H.+Gaoauthor=Y.+Yangauthor=M.+Heauthor=Y.+Wuauthor=Y.+Songauthor=Y.+Tongauthor=Y.+Rao&title=PROTACs%3A+great+opportunities+for+academia+and+industry&doi=10.1038%2Fs41392-019-0101-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">PROTACs: great opportunities for academia and industry</span></div><div class="casAuthors">Sun Xiuyun; Gao Hongying; Yang Yiqing; He Ming; Wu Yue; Song Yugang; Tong Yan; Rao Yu; Sun Xiuyun; Gao Hongying; Yang Yiqing; Rao Yu</div><div class="citationInfo"><span class="NLM_cas:title">Signal transduction and targeted therapy</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">64</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Although many kinds of therapies are applied in the clinic, drug-resistance is a major and unavoidable problem.  Another disturbing statistic is the limited number of drug targets, which are presently only 20-25% of all protein targets that are currently being studied.  Moreover, the focus of current explorations of targets are their enzymatic functions, which ignores the functions from their scaffold moiety.  As a promising and appealing technology, PROteolysis TArgeting Chimeras (PROTACs) have attracted great attention both from academia and industry for finding available approaches to solve the above problems.  PROTACs regulate protein function by degrading target proteins instead of inhibiting them, providing more sensitivity to drug-resistant targets and a greater chance to affect the nonenzymatic functions.  PROTACs have been proven to show better selectivity compared to classic inhibitors.  PROTACs can be described as a chemical knockdown approach with rapidity and reversibility, which presents new and different biology compared to other gene editing tools by avoiding misinterpretations that arise from potential genetic compensation and/or spontaneous mutations.  PRTOACs have been widely explored throughout the world and have outperformed not only in cancer diseases, but also in immune disorders, viral infections and neurodegenerative diseases.  Although PROTACs present a very promising and powerful approach for crossing the hurdles of present drug discovery and tool development in biology, more efforts are needed to gain to get deeper insight into the efficacy and safety of PROTACs in the clinic.  More target binders and more E3 ligases applicable for developing PROTACs are waiting for exploration.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQretvERU2xT-_Q9URG6Dx8fW6udTcc2ebJ2ETDMPKgO7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3MbkvFCrtA%253D%253D&md5=7d9545a21e3a78cededf2977a31e8f7d</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1038%2Fs41392-019-0101-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41392-019-0101-6%26sid%3Dliteratum%253Aachs%26aulast%3DSun%26aufirst%3DX.%26aulast%3DGao%26aufirst%3DH.%26aulast%3DYang%26aufirst%3DY.%26aulast%3DHe%26aufirst%3DM.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DSong%26aufirst%3DY.%26aulast%3DTong%26aufirst%3DY.%26aulast%3DRao%26aufirst%3DY.%26atitle%3DPROTACs%253A%2520great%2520opportunities%2520for%2520academia%2520and%2520industry%26jtitle%3DSignal%2520Transduction%2520Targeted%2520Ther.%26date%3D2019%26volume%3D4%26spage%3D64%26doi%3D10.1038%2Fs41392-019-0101-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mullard, A.</span></span> <span> </span><span class="NLM_article-title">Arvinas’s PROTACs pass first safety and PK analysis</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">895</span>, <span class="refDoi"> DOI: 10.1038/d41573-019-00188-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=10.1038%2Fd41573-019-00188-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=1%3ACAS%3A528%3ADC%252BC1MXit1Ohs7zO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2019&pages=895&author=A.+Mullard&title=Arvinas%E2%80%99s+PROTACs+pass+first+safety+and+PK+analysis&doi=10.1038%2Fd41573-019-00188-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">NIH launches open science Alzheimer initiative</span></div><div class="casAuthors">Mullard, Asher</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">895</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqKr7PwgMO_fLVg90H21EOLACvtfcHk0ljX1FRheRfO5Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXit1Ohs7zO&md5=2abfeaf7ac16a726ceff22cb0164347c</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1038%2Fd41573-019-00188-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fd41573-019-00188-4%26sid%3Dliteratum%253Aachs%26aulast%3DMullard%26aufirst%3DA.%26atitle%3DArvinas%25E2%2580%2599s%2520PROTACs%2520pass%2520first%2520safety%2520and%2520PK%2520analysis%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2019%26volume%3D18%26spage%3D895%26doi%3D10.1038%2Fd41573-019-00188-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mullard, A.</span></span> <span> </span><span class="NLM_article-title">Targeted degraders clear first safety hurdles</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">435</span>, <span class="refDoi"> DOI: 10.1038/d41573-020-00109-w</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=10.1038%2Fd41573-020-00109-w" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhtlShtbrM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2020&pages=435&author=A.+Mullard&title=Targeted+degraders+clear+first+safety+hurdles&doi=10.1038%2Fd41573-020-00109-w"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Targeted degraders clear first safety hurdles</span></div><div class="casAuthors">Mullard, Asher</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">435</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp7nHwwDZKMr7Vg90H21EOLACvtfcHk0ljypSldiIttKw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhtlShtbrM&md5=122b44714339e8b38fc418c2f8732706</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1038%2Fd41573-020-00109-w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fd41573-020-00109-w%26sid%3Dliteratum%253Aachs%26aulast%3DMullard%26aufirst%3DA.%26atitle%3DTargeted%2520degraders%2520clear%2520first%2520safety%2520hurdles%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2020%26volume%3D19%26spage%3D435%26doi%3D10.1038%2Fd41573-020-00109-w" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anjum, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Squillace, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nadworny, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keats, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ning, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wardwell, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eichinger, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moran, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, W.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zou, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mohemmad, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jang, H. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ye, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Narasimhan, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miret, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clackson, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dalgarno, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shakespeare, W. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rivera, V. M.</span></span> <span> </span><span class="NLM_article-title">The Potent ALK Inhibitor Brigatinib (AP26113) Overcomes Mechanisms of Resistance to First- and Second-Generation ALK Inhibitors in Preclinical Models</span>. <i>Clin. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">5527</span>– <span class="NLM_lpage">5538</span>, <span class="refDoi"> DOI: 10.1158/1078-0432.ccr-16-0569</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=10.1158%2F1078-0432.CCR-16-0569" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=27780853" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvVCqur3O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2016&pages=5527-5538&author=S.+Zhangauthor=R.+Anjumauthor=R.+Squillaceauthor=S.+Nadwornyauthor=T.+Zhouauthor=J.+Keatsauthor=Y.+Ningauthor=S.+D.+Wardwellauthor=D.+Millerauthor=Y.+Songauthor=L.+Eichingerauthor=L.+Moranauthor=W.-S.+Huangauthor=S.+Liuauthor=D.+Zouauthor=Y.+Wangauthor=Q.+Mohemmadauthor=H.+G.+Jangauthor=E.+Yeauthor=N.+Narasimhanauthor=F.+Wangauthor=J.+Miretauthor=X.+Zhuauthor=T.+Clacksonauthor=D.+Dalgarnoauthor=W.+C.+Shakespeareauthor=V.+M.+Rivera&title=The+Potent+ALK+Inhibitor+Brigatinib+%28AP26113%29+Overcomes+Mechanisms+of+Resistance+to+First-+and+Second-Generation+ALK+Inhibitors+in+Preclinical+Models&doi=10.1158%2F1078-0432.ccr-16-0569"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">The Potent ALK Inhibitor Brigatinib (AP26113) Overcomes Mechanisms of Resistance to First- and Second-Generation ALK Inhibitors in Preclinical Models</span></div><div class="casAuthors">Zhang, Sen; Anjum, Rana; Squillace, Rachel; Nadworny, Sara; Zhou, Tianjun; Keats, Jeff; Ning, Yaoyu; Wardwell, Scott D.; Miller, David; Song, Youngchul; Eichinger, Lindsey; Moran, Lauren; Huang, Wei-Sheng; Liu, Shuangying; Zou, Dong; Wang, Yihan; Mohemmad, Qurish; Jang, Hyun Gyung; Ye, Emily; Narasimhan, Narayana; Wang, Frank; Miret, Juan; Zhu, Xiaotian; Clackson, Tim; Dalgarno, David; Shakespeare, William C.; Rivera, Victor M.</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">5527-5538</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Purpose: Non-small cell lung cancers (NSCLCs) harboring ALK gene rearrangements (ALK+) typically become resistant to the first-generation anaplastic lymphoma kinase (ALK) tyrosine kinase inhibitor (TKI) crizotinib through development of secondary resistance mutations in ALK or disease progression in the brain.  Mutations that confer resistance to second-generation ALK TKIs ceritinib and alectinib have also been identified.  Here, we report the structure and first comprehensive preclin. evaluation of the next-generation ALK TKI brigatinib.  Exptl. Design: A kinase screen was performed to evaluate the selectivity profile of brigatinib.  The cellular and in vivo activities of ALK TKIs were compared using engineered and cancer-derived cell lines.  The brigatinib-ALK co-structure was detd.  Results: Brigatinib potently inhibits ALK and ROS1, with a high degree of selectivity over more than 250 kinases.  Across a panel of ALK+ cell lines, brigatinib inhibited native ALK (IC50, 10 nmol/L) with 12-fold greater potency than crizotinib.  Superior efficacy of brigatinib was also obsd. in mice with ALK+ tumors implanted s.c. or intracranially.  Brigatinib maintained substantial activity against all 17 secondary ALK mutants tested in cellular assays and exhibited a superior inhibitory profile compared with crizotinib, ceritinib, and alectinib at clin. achievable concns.  Brigatinib was the only TKI to maintain substantial activity against the most recalcitrant ALK resistance mutation, G1202R.  The unique, potent, and pan-ALK mutant activity of brigatinib could be rationalized by structural analyses.  Conclusions: Brigatinib is a highly potent and selective ALK inhibitor.  These findings provide the mol. basis for the promising activity being obsd. in ALK+, crizotinib-resistant patients with NSCLC being treated with brigatinib in clin. trials.  Clin Cancer Res; 22(22); 5527-38. ©2016 AACR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoVwQUn_qLy7bVg90H21EOLACvtfcHk0ljypSldiIttKw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvVCqur3O&md5=1ece23bf36e807562023d0b814c990c7</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1158%2F1078-0432.CCR-16-0569&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1078-0432.CCR-16-0569%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DS.%26aulast%3DAnjum%26aufirst%3DR.%26aulast%3DSquillace%26aufirst%3DR.%26aulast%3DNadworny%26aufirst%3DS.%26aulast%3DZhou%26aufirst%3DT.%26aulast%3DKeats%26aufirst%3DJ.%26aulast%3DNing%26aufirst%3DY.%26aulast%3DWardwell%26aufirst%3DS.%2BD.%26aulast%3DMiller%26aufirst%3DD.%26aulast%3DSong%26aufirst%3DY.%26aulast%3DEichinger%26aufirst%3DL.%26aulast%3DMoran%26aufirst%3DL.%26aulast%3DHuang%26aufirst%3DW.-S.%26aulast%3DLiu%26aufirst%3DS.%26aulast%3DZou%26aufirst%3DD.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DMohemmad%26aufirst%3DQ.%26aulast%3DJang%26aufirst%3DH.%2BG.%26aulast%3DYe%26aufirst%3DE.%26aulast%3DNarasimhan%26aufirst%3DN.%26aulast%3DWang%26aufirst%3DF.%26aulast%3DMiret%26aufirst%3DJ.%26aulast%3DZhu%26aufirst%3DX.%26aulast%3DClackson%26aufirst%3DT.%26aulast%3DDalgarno%26aufirst%3DD.%26aulast%3DShakespeare%26aufirst%3DW.%2BC.%26aulast%3DRivera%26aufirst%3DV.%2BM.%26atitle%3DThe%2520Potent%2520ALK%2520Inhibitor%2520Brigatinib%2520%2528AP26113%2529%2520Overcomes%2520Mechanisms%2520of%2520Resistance%2520to%2520First-%2520and%2520Second-Generation%2520ALK%2520Inhibitors%2520in%2520Preclinical%2520Models%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2016%26volume%3D22%26spage%3D5527%26epage%3D5538%26doi%3D10.1158%2F1078-0432.ccr-16-0569" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huang, W.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zou, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Romero, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kohlmann, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dwight, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dodd, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toms, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parillon, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anjum, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keats, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wardwell, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ning, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moran, L. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mohemmad, Q. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jang, H. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clackson, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Narasimhan, N. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rivera, V. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dalgarno, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shakespeare, W. C.</span></span> <span> </span><span class="NLM_article-title">Discovery of Brigatinib (AP26113), a Phosphine Oxide-Containing, Potent, Orally Active Inhibitor of Anaplastic Lymphoma Kinase</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">4948</span>– <span class="NLM_lpage">4964</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b00306</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b00306" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=1%3ACAS%3A528%3ADC%252BC28XntlCgur8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=4948-4964&author=W.-S.+Huangauthor=S.+Liuauthor=D.+Zouauthor=M.+Thomasauthor=Y.+Wangauthor=T.+Zhouauthor=J.+Romeroauthor=A.+Kohlmannauthor=F.+Liauthor=J.+Qiauthor=L.+Caiauthor=T.+A.+Dwightauthor=Y.+Xuauthor=R.+Xuauthor=R.+Doddauthor=A.+Tomsauthor=L.+Parillonauthor=X.+Luauthor=R.+Anjumauthor=S.+Zhangauthor=F.+Wangauthor=J.+Keatsauthor=S.+D.+Wardwellauthor=Y.+Ningauthor=Q.+Xuauthor=L.+E.+Moranauthor=Q.+K.+Mohemmadauthor=H.+G.+Jangauthor=T.+Clacksonauthor=N.+I.+Narasimhanauthor=V.+M.+Riveraauthor=X.+Zhuauthor=D.+Dalgarnoauthor=W.+C.+Shakespeare&title=Discovery+of+Brigatinib+%28AP26113%29%2C+a+Phosphine+Oxide-Containing%2C+Potent%2C+Orally+Active+Inhibitor+of+Anaplastic+Lymphoma+Kinase&doi=10.1021%2Facs.jmedchem.6b00306"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Brigatinib (AP26113), a Phosphine Oxide-Containing, Potent, Orally Active Inhibitor of Anaplastic Lymphoma Kinase</span></div><div class="casAuthors">Huang, Wei-Sheng; Liu, Shuangying; Zou, Dong; Thomas, Mathew; Wang, Yihan; Zhou, Tianjun; Romero, Jan; Kohlmann, Anna; Li, Feng; Qi, Jiwei; Cai, Lisi; Dwight, Timothy A.; Xu, Yongjin; Xu, Rongsong; Dodd, Rory; Toms, Angela; Parillon, Lois; Lu, Xiaohui; Anjum, Rana; Zhang, Sen; Wang, Frank; Keats, Jeffrey; Wardwell, Scott D.; Ning, Yaoyu; Xu, Qihong; Moran, Lauren E.; Mohemmad, Qurish K.; Jang, Hyun Gyung; Clackson, Tim; Narasimhan, Narayana I.; Rivera, Victor M.; Zhu, Xiaotian; Dalgarno, David; Shakespeare, William C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">4948-4964</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In the treatment of echinoderm microtubule-assocd. protein-like 4 (EML4)-anaplastic lymphoma kinase pos. (ALK+) non-small-cell lung cancer (NSCLC), secondary mutations within the ALK kinase domain have emerged as a major resistance mechanism to both first- and second-generation ALK inhibitors.  This report describes the design and synthesis of a series of 2,4-diarylaminopyrimidine-based potent and selective ALK inhibitors culminating in identification of the investigational clin. candidate brigatinib.  A unique structural feature of brigatinib is a phosphine oxide, an overlooked but novel hydrogen-bond acceptor that drives potency and selectivity in addn. to favorable ADME properties.  Brigatinib displayed low nanomolar IC50s against native ALK and all tested clin. relevant ALK mutants in both enzyme-based biochem. and cell-based viability assays and demonstrated efficacy in multiple ALK+ xenografts in mice, including Karpas-299 (anaplastic large-cell lymphomas [ALCL]) and H3122 (NSCLC).  Brigatinib represents the most clin. advanced phosphine oxide-contg. drug candidate to date and is currently being evaluated in a global phase 2 registration trial.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr5_iqprSA4QbVg90H21EOLACvtfcHk0ljypSldiIttKw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XntlCgur8%253D&md5=4cb93ff587579c97fda8aff0c7d43317</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b00306&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b00306%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DW.-S.%26aulast%3DLiu%26aufirst%3DS.%26aulast%3DZou%26aufirst%3DD.%26aulast%3DThomas%26aufirst%3DM.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DZhou%26aufirst%3DT.%26aulast%3DRomero%26aufirst%3DJ.%26aulast%3DKohlmann%26aufirst%3DA.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DQi%26aufirst%3DJ.%26aulast%3DCai%26aufirst%3DL.%26aulast%3DDwight%26aufirst%3DT.%2BA.%26aulast%3DXu%26aufirst%3DY.%26aulast%3DXu%26aufirst%3DR.%26aulast%3DDodd%26aufirst%3DR.%26aulast%3DToms%26aufirst%3DA.%26aulast%3DParillon%26aufirst%3DL.%26aulast%3DLu%26aufirst%3DX.%26aulast%3DAnjum%26aufirst%3DR.%26aulast%3DZhang%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DF.%26aulast%3DKeats%26aufirst%3DJ.%26aulast%3DWardwell%26aufirst%3DS.%2BD.%26aulast%3DNing%26aufirst%3DY.%26aulast%3DXu%26aufirst%3DQ.%26aulast%3DMoran%26aufirst%3DL.%2BE.%26aulast%3DMohemmad%26aufirst%3DQ.%2BK.%26aulast%3DJang%26aufirst%3DH.%2BG.%26aulast%3DClackson%26aufirst%3DT.%26aulast%3DNarasimhan%26aufirst%3DN.%2BI.%26aulast%3DRivera%26aufirst%3DV.%2BM.%26aulast%3DZhu%26aufirst%3DX.%26aulast%3DDalgarno%26aufirst%3DD.%26aulast%3DShakespeare%26aufirst%3DW.%2BC.%26atitle%3DDiscovery%2520of%2520Brigatinib%2520%2528AP26113%2529%252C%2520a%2520Phosphine%2520Oxide-Containing%252C%2520Potent%252C%2520Orally%2520Active%2520Inhibitor%2520of%2520Anaplastic%2520Lymphoma%2520Kinase%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D4948%26epage%3D4964%26doi%3D10.1021%2Facs.jmedchem.6b00306" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bedi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khan, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">AbuKhader, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alam, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siddiqui, N. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Husain, A.</span></span> <span> </span><span class="NLM_article-title">A comprehensive review on Brigatinib - A wonder drug for targeted cancer therapy in non-small cell lung cancer</span>. <i>Saudi Pharm. J.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">755</span>– <span class="NLM_lpage">763</span>, <span class="refDoi"> DOI: 10.1016/j.jsps.2018.04.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=10.1016%2Fj.jsps.2018.04.010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=30202213" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=1%3ACAS%3A280%3ADC%252BB3c3nsFWqtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2018&pages=755-763&author=S.+Bediauthor=S.+A.+Khanauthor=M.+M.+AbuKhaderauthor=P.+Alamauthor=N.+A.+Siddiquiauthor=A.+Husain&title=A+comprehensive+review+on+Brigatinib+-+A+wonder+drug+for+targeted+cancer+therapy+in+non-small+cell+lung+cancer&doi=10.1016%2Fj.jsps.2018.04.010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">A comprehensive review on Brigatinib - A wonder drug for targeted cancer therapy in non-small cell lung cancer</span></div><div class="casAuthors">Bedi Silky; Husain Asif; Khan Shah A; AbuKhader Majed M; Alam Perwez; Siddiqui Nasir A</div><div class="citationInfo"><span class="NLM_cas:title">Saudi pharmaceutical journal : SPJ : the official publication of the Saudi Pharmaceutical Society</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">755-763</span>
        ISSN:<span class="NLM_cas:issn">1319-0164</span>.
    </div><div class="casAbstract">The mortality rate in patients suffering from non-small cell lung cancer (NSCLC) is quite high.  This type of cancer mainly occurs due to rearrangements in the anaplastic lymphoma kinase (ALK) gene which leads to form an oncogene of fused gene NPM-ALK.  Brigatinib is recently approved by FDA in April 2017 as a potent tyrosine kinase inhibitor (TKI) for the NSCLC therapy.  In the present scenario, it is no less than a wonder drug because it is indicated for the treatment of advanced stages of metastatic ALK positive NSCLC, a fatal disease to overcome the resistance of various other ALK inhibitors such as crizotinib, ceritinib and alectinib.  In addition to ALK, it is also active against multiple types of kinases such as ROS1, Insulin like growth factor-1Receptor and EGFR.  It can be synthesized by using N-[2-methoxy-4-[4-(dimethylamino) piperidin-1-yl] aniline] guanidine and 2,4,5-trichloropyrimidine respectively in two different ways.  Its structure consists of mainly dimethylphosphine oxide group which is responsible for its pharmacological activity.  It is active against various cell lines such as HCC78, H2228, H23, H358, H838, U937, HepG2 and Karpas- 299.  Results of ALTA (ALK in Lung Cancer Trial of AP26113) phase 1/2 trial shows that 90 mg of brigatinib for 7 days and then 180 mg for next days is effective in the treatment of NSCLC.  Brigatinib has been shown to have favorable risk benefit profile and is a safer drug than the available cytotoxic chemotherapeutic agents.  In comparison to other FDA approved drugs for the same condition, it causes fewer minor adverse reactions which can be easily managed either by changing the dose or by providing good supportive care.  This article is intended to provide readers with an overview of chemistry, pharmacokinetic, pharmacodynamic and safety profile of brigatinib, which addresses an unmet medical need.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcS1tFvEwEman42y4bB4mnvYfW6udTcc2eYk9GiYMKTGYbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3c3nsFWqtA%253D%253D&md5=6202b67fb7713e61af3a5744ced333b3</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1016%2Fj.jsps.2018.04.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jsps.2018.04.010%26sid%3Dliteratum%253Aachs%26aulast%3DBedi%26aufirst%3DS.%26aulast%3DKhan%26aufirst%3DS.%2BA.%26aulast%3DAbuKhader%26aufirst%3DM.%2BM.%26aulast%3DAlam%26aufirst%3DP.%26aulast%3DSiddiqui%26aufirst%3DN.%2BA.%26aulast%3DHusain%26aufirst%3DA.%26atitle%3DA%2520comprehensive%2520review%2520on%2520Brigatinib%2520-%2520A%2520wonder%2520drug%2520for%2520targeted%2520cancer%2520therapy%2520in%2520non-small%2520cell%2520lung%2520cancer%26jtitle%3DSaudi%2520Pharm.%2520J.%26date%3D2018%26volume%3D26%26spage%3D755%26epage%3D763%26doi%3D10.1016%2Fj.jsps.2018.04.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kim, D.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tiseo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ahn, M.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reckamp, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hansen, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, S.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huber, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">West, H. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Groen, H. J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hochmair, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leighl, N. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gettinger, S. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Langer, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paz-Ares Rodríguez, L. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smit, E. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reichmann, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haluska, F. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kerstein, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Camidge, D. R.</span></span> <span> </span><span class="NLM_article-title">Brigatinib in Patients With Crizotinib-Refractory Anaplastic Lymphoma Kinase-Positive Non-Small-Cell Lung Cancer: A Randomized, Multicenter Phase II Trial</span>. <i>J. Clin. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>35</i></span>,  <span class="NLM_fpage">2490</span>– <span class="NLM_lpage">2498</span>, <span class="refDoi"> DOI: 10.1200/jco.2016.71.5904</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=10.1200%2FJCO.2016.71.5904" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=28475456" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitVSqsbfK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2017&pages=2490-2498&author=D.-W.+Kimauthor=M.+Tiseoauthor=M.-J.+Ahnauthor=K.+L.+Reckampauthor=K.+H.+Hansenauthor=S.-W.+Kimauthor=R.+M.+Huberauthor=H.+L.+Westauthor=H.+J.+M.+Groenauthor=M.+J.+Hochmairauthor=N.+B.+Leighlauthor=S.+N.+Gettingerauthor=C.+J.+Langerauthor=L.+G.+Paz-Ares+Rodr%C3%ADguezauthor=E.+F.+Smitauthor=E.+S.+Kimauthor=W.+Reichmannauthor=F.+G.+Haluskaauthor=D.+Kersteinauthor=D.+R.+Camidge&title=Brigatinib+in+Patients+With+Crizotinib-Refractory+Anaplastic+Lymphoma+Kinase-Positive+Non-Small-Cell+Lung+Cancer%3A+A+Randomized%2C+Multicenter+Phase+II+Trial&doi=10.1200%2Fjco.2016.71.5904"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Brigatinib in patients with crizotinib-refractory anaplastic lymphoma kinase-positive non-small-cell lung cancer: a randomized, multicenter phase II trial</span></div><div class="casAuthors">Kim, Dong-Wan; Tiseo, Marcello; Ahn, Myung-Ju; Reckamp, Karen L.; Hansen, Karin Holmskov; Kim, Sang-We; Huber, Rudolf M.; West, Howard L.; Groen, Harry J. M.; Hochmair, Maximilian J.; Leighl, Natasha B.; Gettinger, Scott N.; Langer, Corey J.; Rodr'iguez, Luis G. Paz-Ares; Smit, Egbert F.; Kim, Edward S.; Reichmann, William; Haluska, Frank G.; Kerstein, David; Camidge, D. Ross</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Clinical Oncology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">2490-2500</span>CODEN:
                <span class="NLM_cas:coden">JCONDN</span>;
        ISSN:<span class="NLM_cas:issn">1527-7755</span>.
    
            (<span class="NLM_cas:orgname">American Society of Clinical Oncology</span>)
        </div><div class="casAbstract">Purpose: Most crizotinib-treated patients with anaplastic lymphoma kinase gene (ALK)-rearranged non-smallcell lung cancer (ALK-pos. NSCLC) eventually experience disease progression.  We evaluated two regimens of brigatinib, an investigational next-generation ALK inhibitor, in crizotinib-refractory ALK-pos. NSCLC.  Patients and Methods Patients were stratified by brain metastases and best response to crizotinib.  They were randomly assigned (1:1) to oral brigatinib 90 mg once daily (arm A) or 180 mg once daily with a 7-day lead-in at 90 mg (180 mg once daily [with lead-in]; arm B).  Investigator-assessed confirmed objective response rate (ORR) was the primary end point.  Results Of 222 patients enrolled (arm A: n = 112, 109 treated; arm B: n = 110, 110 treated), 154 (69%) had baseline brain metastases and 164 of 222 (74%) had received prior chemotherapy.  With 8.0-mo median follow-up, investigator-assessed confirmed ORR was 45% (97.5% CI, 34% to 56%) in arm A and 54% (97.5% CI, 43% to 65%) in arm B.  Investigator-assessed median progression-free survival was 9.2 mo (95% CI, 7.4 to 15.6) and 12.9 mo (95% CI, 11.1 to not reached) in arms A and B, resp.  Independent review committee-assessed intracranial ORR in patients with measurable brain metastases at baseline was 42% (11 of 26 patients) in arm A and 67% (12 of 18 patients) in arm B.  Common treatment-emergent adverse events were nausea (arm A/B, 33%/ 40%), diarrhea (arm A/B, 19%/38%), headache (arm A/B, 28%/27%), and cough (arm A/B, 18%/ 34%), and were mainly grades 1 to 2.  A subset of pulmonary adverse events with early onset (median onset: day 2) occurred in 14 of 219 treated patients (all grades, 6%; grade $ 3, 3%); none occurred after escalation to 180 mg in arm B.  Seven of 14 patients were successfully retreated with brigatinib.  Conclusion Brigatinib yielded substantial whole-body and intracranial responses as well as robust progressionfree survival; 180 mg (with lead-in) showed consistently better efficacy than 90 mg, with acceptable safety.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq4RtdJwkNkyLVg90H21EOLACvtfcHk0ljwbEsFT8BU-g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitVSqsbfK&md5=efe68d35d297c3d9e70023f887c23a2c</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1200%2FJCO.2016.71.5904&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1200%252FJCO.2016.71.5904%26sid%3Dliteratum%253Aachs%26aulast%3DKim%26aufirst%3DD.-W.%26aulast%3DTiseo%26aufirst%3DM.%26aulast%3DAhn%26aufirst%3DM.-J.%26aulast%3DReckamp%26aufirst%3DK.%2BL.%26aulast%3DHansen%26aufirst%3DK.%2BH.%26aulast%3DKim%26aufirst%3DS.-W.%26aulast%3DHuber%26aufirst%3DR.%2BM.%26aulast%3DWest%26aufirst%3DH.%2BL.%26aulast%3DGroen%26aufirst%3DH.%2BJ.%2BM.%26aulast%3DHochmair%26aufirst%3DM.%2BJ.%26aulast%3DLeighl%26aufirst%3DN.%2BB.%26aulast%3DGettinger%26aufirst%3DS.%2BN.%26aulast%3DLanger%26aufirst%3DC.%2BJ.%26aulast%3DPaz-Ares%2BRodr%25C3%25ADguez%26aufirst%3DL.%2BG.%26aulast%3DSmit%26aufirst%3DE.%2BF.%26aulast%3DKim%26aufirst%3DE.%2BS.%26aulast%3DReichmann%26aufirst%3DW.%26aulast%3DHaluska%26aufirst%3DF.%2BG.%26aulast%3DKerstein%26aufirst%3DD.%26aulast%3DCamidge%26aufirst%3DD.%2BR.%26atitle%3DBrigatinib%2520in%2520Patients%2520With%2520Crizotinib-Refractory%2520Anaplastic%2520Lymphoma%2520Kinase-Positive%2520Non-Small-Cell%2520Lung%2520Cancer%253A%2520A%2520Randomized%252C%2520Multicenter%2520Phase%2520II%2520Trial%26jtitle%3DJ.%2520Clin.%2520Oncol.%26date%3D2017%26volume%3D35%26spage%3D2490%26epage%3D2498%26doi%3D10.1200%2Fjco.2016.71.5904" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gaye, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geier, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bore, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guilloïque, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lucia, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quéré, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gouva, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robinet, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Descourt, R.</span></span> <span> </span><span class="NLM_article-title">Intra-cranial efficacy of brigatinib in an ALK-positive non-small cell lung cancer patient presenting leptomeningeal carcinomatosis</span>. <i>Lung Cancer</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>133</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">3</span>, <span class="refDoi"> DOI: 10.1016/j.lungcan.2019.04.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=10.1016%2Fj.lungcan.2019.04.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=31200813" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=1%3ACAS%3A280%3ADC%252BB3M3mvFWgtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=133&publication_year=2019&pages=1-3&author=E.+Gayeauthor=M.+Geierauthor=P.+Boreauthor=M.+Guillo%C3%AFqueauthor=F.+Luciaauthor=G.+Qu%C3%A9r%C3%A9author=S.+Gouvaauthor=G.+Robinetauthor=R.+Descourt&title=Intra-cranial+efficacy+of+brigatinib+in+an+ALK-positive+non-small+cell+lung+cancer+patient+presenting+leptomeningeal+carcinomatosis&doi=10.1016%2Fj.lungcan.2019.04.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Intra-cranial efficacy of brigatinib in an ALK-positive non-small cell lung cancer patient presenting leptomeningeal carcinomatosis</span></div><div class="casAuthors">Gaye Elisabeth; Geier Margaux; Bore Paul; Quere Gilles; Gouva Sylvie; Robinet Gilles; Descourt Renaud; Guilloique Marine; Lucia Francois</div><div class="citationInfo"><span class="NLM_cas:title">Lung cancer (Amsterdam, Netherlands)</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">133</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">1-3</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">OBJECTIVES:  Brigatinib is a second-generation ALK inhibitor which demonstrated activity over crizotinib-resistance, especially on brain metastasis by increased blood-brain penetration.  However, its activity on lepto-meningeal disease is unknown and scarcely reported.  MATERIALS AND METHODS:  We hereby report the case of lepto-meningeal disease in crizotinib- and ceretinib- treated patient who was successfully treated by brigatinib.  RESULTS:  The patient achieved intracranial response to brigatinib more than 14 months.  CONCLUSION:  Our case provides additional data on brigatinib's intracranial activity, not only on brain metastasis but also on leptomeningeal disease, after experiencing resistance to both crizotinib and ceretinib, 1(st) and 2(nd) generation ALK inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQCRHissLLXoQMthz5_Rx5rfW6udTcc2eabycaUhLbpcbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3M3mvFWgtQ%253D%253D&md5=b0af2a7cc0f91694fd8ea6b73c9af473</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1016%2Fj.lungcan.2019.04.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.lungcan.2019.04.013%26sid%3Dliteratum%253Aachs%26aulast%3DGaye%26aufirst%3DE.%26aulast%3DGeier%26aufirst%3DM.%26aulast%3DBore%26aufirst%3DP.%26aulast%3DGuillo%25C3%25AFque%26aufirst%3DM.%26aulast%3DLucia%26aufirst%3DF.%26aulast%3DQu%25C3%25A9r%25C3%25A9%26aufirst%3DG.%26aulast%3DGouva%26aufirst%3DS.%26aulast%3DRobinet%26aufirst%3DG.%26aulast%3DDescourt%26aufirst%3DR.%26atitle%3DIntra-cranial%2520efficacy%2520of%2520brigatinib%2520in%2520an%2520ALK-positive%2520non-small%2520cell%2520lung%2520cancer%2520patient%2520presenting%2520leptomeningeal%2520carcinomatosis%26jtitle%3DLung%2520Cancer%26date%3D2019%26volume%3D133%26spage%3D1%26epage%3D3%26doi%3D10.1016%2Fj.lungcan.2019.04.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lin, H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pan, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hou, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baumann, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hochmair, M. J.</span></span> <span> </span><span class="NLM_article-title">Real-world treatment duration in ALK-positive non-small-cell lung cancer patients receiving brigatinib through the early access program</span>. <i>Future Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">1031</span>– <span class="NLM_lpage">1041</span>, <span class="refDoi"> DOI: 10.2217/fon-2019-0849</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=10.2217%2Ffon-2019-0849" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=32338548" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=1%3ACAS%3A528%3ADC%252BB3cXptVaks70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2020&pages=1031-1041&author=H.+M.+Linauthor=X.+Panauthor=P.+Houauthor=S.+Allenauthor=P.+Baumannauthor=M.+J.+Hochmair&title=Real-world+treatment+duration+in+ALK-positive+non-small-cell+lung+cancer+patients+receiving+brigatinib+through+the+early+access+program&doi=10.2217%2Ffon-2019-0849"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Real-world treatment duration in ALK-positive non-small-cell lung cancer patients receiving brigatinib through the early access program</span></div><div class="casAuthors">Lin, Huamao M.; Pan, Xiaoyun; Hou, Peijie; Allen, Susan; Baumann, Pia; Hochmair, Maximilian J.</div><div class="citationInfo"><span class="NLM_cas:title">Future Oncology</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">1031-1041</span>CODEN:
                <span class="NLM_cas:coden">FOUNBN</span>;
        ISSN:<span class="NLM_cas:issn">1744-8301</span>.
    
            (<span class="NLM_cas:orgname">Future Medicine Ltd.</span>)
        </div><div class="casAbstract">Aim: To assess time-to-treatment discontinuation (TTD) of brigatinib following treatment with ALK tyrosine kinase inhibitor(s) (TKIs) in patients with ALK-pos. (ALK+) non-small-cell lung cancer (NSCLC) receiving brigatinib through the international early access program.  Patients & anal.: Anal. was performed for patients with ALK+ NSCLC treated with prior ALK TKIs, including next-generation ALK TKIs.  Results: Data for 604 patients (21 countries), including patients with prior next-generation ALK TKIs, were reported.  The median TTD of brigatinib in patients with prior crizotinib, alectinib, ceritinib or lorlatinib was 10.0, 8.7, 10.3 and 7.5 mo, resp.  Conclusion: Brigatinib appears to be effective and tolerable in real-world clin. practice regardless of prior treatment with first or NG ALK TKIs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqP0lzoeDBC8bVg90H21EOLACvtfcHk0lidjSh8-Ow-4A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXptVaks70%253D&md5=b7677b498bf8359cb0144167bab75ebe</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.2217%2Ffon-2019-0849&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2217%252Ffon-2019-0849%26sid%3Dliteratum%253Aachs%26aulast%3DLin%26aufirst%3DH.%2BM.%26aulast%3DPan%26aufirst%3DX.%26aulast%3DHou%26aufirst%3DP.%26aulast%3DAllen%26aufirst%3DS.%26aulast%3DBaumann%26aufirst%3DP.%26aulast%3DHochmair%26aufirst%3DM.%2BJ.%26atitle%3DReal-world%2520treatment%2520duration%2520in%2520ALK-positive%2520non-small-cell%2520lung%2520cancer%2520patients%2520receiving%2520brigatinib%2520through%2520the%2520early%2520access%2520program%26jtitle%3DFuture%2520Oncol.%26date%3D2020%26volume%3D16%26spage%3D1031%26epage%3D1041%26doi%3D10.2217%2Ffon-2019-0849" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sun, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ren, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kong, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhong, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qiu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, B.</span></span> <span> </span><span class="NLM_article-title">Development of a Brigatinib degrader (SIAIS117) as a potential treatment for ALK positive cancer resistance</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>193</i></span>,  <span class="NLM_fpage">112190</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2020.112190</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=10.1016%2Fj.ejmech.2020.112190" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=32179332" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=1%3ACAS%3A528%3ADC%252BB3cXkvF2qu70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=193&publication_year=2020&pages=112190&author=N.+Sunauthor=C.+Renauthor=Y.+Kongauthor=H.+Zhongauthor=J.+Chenauthor=Y.+Liauthor=J.+Zhangauthor=Y.+Zhouauthor=X.+Qiuauthor=H.+Linauthor=X.+Songauthor=X.+Yangauthor=B.+Jiang&title=Development+of+a+Brigatinib+degrader+%28SIAIS117%29+as+a+potential+treatment+for+ALK+positive+cancer+resistance&doi=10.1016%2Fj.ejmech.2020.112190"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Development of a Brigatinib degrader (SIAIS117) as a potential treatment for ALK positive cancer resistance</span></div><div class="casAuthors">Sun, Ning; Ren, Chaowei; Kong, Ying; Zhong, Hui; Chen, Jinju; Li, Yan; Zhang, Jianshui; Zhou, Yuedong; Qiu, Xing; Lin, Haifan; Song, Xiaoling; Yang, Xiaobao; Jiang, Biao</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">193</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">112190</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">EML4-ALK and NPM-ALK fusion proteins possess constitutively activated ALK (anaplastic lymphoma kinase) activity, which in turn leads to the development of non-small cell lung cancer and anaplastic large-cell lymphomas (ALCLs).  FDA-approved ALK inhibitor drugs cause significant cancer regression.  However, drug resistance eventually occurs and it becomes a big obstacle in clinic.  Novel proteolysis targeting chimera (PROTAC) technol. platform provides a potential therapeutic strategy for drug resistance.  Herein, we designed and synthesized a series of ALK PROTACs based on Brigatinib and VHL-1 conjunction, and screened SIAIS117 as the best degrader which not only blocked the growth of SR and H2228 cancer cell lines, but also degraded ALK protein.  In addn., SIAIS117 also showed much better growth inhibition effect than Brigatinib on 293T cell line that exogenously expressed G1202R-resistant ALK proteins.  Furthermore, it also degraded G1202R mutant ALK protein in vitro.  At last, it has the potentially anti-proliferation ability of small cell lung cancer.  Thus, we have successfully generated the degrader SIAIS117 that can potentially overcome resistance in cancer targeted therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrX9rzk88HMe7Vg90H21EOLACvtfcHk0lgGGQGjVM6uNQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXkvF2qu70%253D&md5=61f0228111c806ae62473fc450fce276</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2020.112190&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2020.112190%26sid%3Dliteratum%253Aachs%26aulast%3DSun%26aufirst%3DN.%26aulast%3DRen%26aufirst%3DC.%26aulast%3DKong%26aufirst%3DY.%26aulast%3DZhong%26aufirst%3DH.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DZhou%26aufirst%3DY.%26aulast%3DQiu%26aufirst%3DX.%26aulast%3DLin%26aufirst%3DH.%26aulast%3DSong%26aufirst%3DX.%26aulast%3DYang%26aufirst%3DX.%26aulast%3DJiang%26aufirst%3DB.%26atitle%3DDevelopment%2520of%2520a%2520Brigatinib%2520degrader%2520%2528SIAIS117%2529%2520as%2520a%2520potential%2520treatment%2520for%2520ALK%2520positive%2520cancer%2520resistance%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2020%26volume%3D193%26spage%3D112190%26doi%3D10.1016%2Fj.ejmech.2020.112190" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Qiu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kong, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, B.</span></span> <span> </span><span class="NLM_article-title">Chemoselective Synthesis of Lenalidomide-Based PROTAC Library Using Alkylation Reaction</span>. <i>Org. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">3838</span>– <span class="NLM_lpage">3841</span>, <span class="refDoi"> DOI: 10.1021/acs.orglett.9b01326</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.orglett.9b01326" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=1%3ACAS%3A528%3ADC%252BC1MXpt1WjsbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2019&pages=3838-3841&author=X.+Qiuauthor=N.+Sunauthor=Y.+Kongauthor=Y.+Liauthor=X.+Yangauthor=B.+Jiang&title=Chemoselective+Synthesis+of+Lenalidomide-Based+PROTAC+Library+Using+Alkylation+Reaction&doi=10.1021%2Facs.orglett.9b01326"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Chemoselective Synthesis of Lenalidomide-Based PROTAC Library Using Alkylation Reaction</span></div><div class="casAuthors">Qiu, Xing; Sun, Ning; Kong, Ying; Li, Yan; Yang, Xiaobao; Jiang, Biao</div><div class="citationInfo"><span class="NLM_cas:title">Organic Letters</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">3838-3841</span>CODEN:
                <span class="NLM_cas:coden">ORLEF7</span>;
        ISSN:<span class="NLM_cas:issn">1523-7052</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Using DIPEA as base in N-methyl-2-pyrrolidinone (NMP), lenalidomide underwent regioselective alkylation with bromoesters and Boc-protected bromoamines followed by deprotection to yield lenalidomide derivs. as a library of potential PROTAC reagents.  One of the products was coupled to the known BET binding agent JQ1 to yield I; I degraded BET protein in cells and effectively inhibited cancer cell proliferation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWwAx0Aar9k7Vg90H21EOLACvtfcHk0lgGGQGjVM6uNQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXpt1WjsbY%253D&md5=b185afe1107a03f13560e4cb44223026</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Facs.orglett.9b01326&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.orglett.9b01326%26sid%3Dliteratum%253Aachs%26aulast%3DQiu%26aufirst%3DX.%26aulast%3DSun%26aufirst%3DN.%26aulast%3DKong%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DYang%26aufirst%3DX.%26aulast%3DJiang%26aufirst%3DB.%26atitle%3DChemoselective%2520Synthesis%2520of%2520Lenalidomide-Based%2520PROTAC%2520Library%2520Using%2520Alkylation%2520Reaction%26jtitle%3DOrg.%2520Lett.%26date%3D2019%26volume%3D21%26spage%3D3838%26epage%3D3841%26doi%3D10.1021%2Facs.orglett.9b01326" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, X.-R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiong, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span> <span> </span><span class="NLM_article-title">Proteolysis Targeting Chimeras (PROTACs) of Anaplastic Lymphoma Kinase (ALK)</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>151</i></span>,  <span class="NLM_fpage">304</span>– <span class="NLM_lpage">314</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2018.03.071</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=10.1016%2Fj.ejmech.2018.03.071" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=29627725" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=1%3ACAS%3A528%3ADC%252BC1cXnt1ejs7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=151&publication_year=2018&pages=304-314&author=C.+Zhangauthor=X.-R.+Hanauthor=X.+Yangauthor=B.+Jiangauthor=J.+Liuauthor=Y.+Xiongauthor=J.+Jin&title=Proteolysis+Targeting+Chimeras+%28PROTACs%29+of+Anaplastic+Lymphoma+Kinase+%28ALK%29&doi=10.1016%2Fj.ejmech.2018.03.071"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Proteolysis Targeting Chimeras (PROTACs) of Anaplastic Lymphoma Kinase (ALK)</span></div><div class="casAuthors">Zhang, Chengwei; Han, Xiao-Ran; Yang, Xiaobao; Jiang, Biao; Liu, Jing; Xiong, Yue; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">151</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">304-314</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">Anaplastic lymphoma kinase (ALK) activation has been assocd. with many types of human cancer.  Significant efforts have been devoted to the development of ALK inhibitors to antagonize the kinase activity of ALK.  Four ALK inhibitors have been approved by the FDA to date for treating patients with ALK-pos. nonsmall cell lung cancers (NSCLC).  However, drug resistance has been obsd. in the majority of patients treated with these inhibitors.  New therapeutic strategies (e.g., compds. with novel mechanisms of action) are needed to overcome the drug resistance issue.  The emerging PROTAC (Proteolysis Targeting Chimera) technol. has been successfully applied to selective degrdn. of multiple protein targets, but not ALK.  Since ALK protein levels are not important for viability in mammals, ALK PROTACs could lead to novel therapeutics with minimal toxicity.  Here the authors report the design, synthesis and biol. evaluation of novel PROTACs (degraders) of ALK.  MS4077 (I) and MS4078 (II) potently decreased cellular levels of oncogenic active ALK fusion proteins in a concn.- and time-dependent manner in SU-DHL-1 lymphoma and NCI-H2228 lung cancer cells.  The ALK protein degrdn. induced by compds. I and II was cereblon and proteasome dependent.  In addn., compds. I and II potently inhibited proliferation of SU-DHL-1 cells.  Furthermore, compd. II displayed good plasma exposure in a mouse pharmacokinetic study, thus is suitable for in vivo efficacy studies.  The authors also developed MS4748 and MS4740 , very close analogs of I and II resp., which are incapable to degrade the ALK fusion proteins, as neg. controls.  All 4 compds. are valuable chem. tools for investigating effects of ALK pharmacol. degrdn.  The study paved the way for developing the next generation of ALK PROTACs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPGM3aAZZ6h7Vg90H21EOLACvtfcHk0lgGGQGjVM6uNQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXnt1ejs7Y%253D&md5=804324b95034f38b46e45f89e18e55be</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2018.03.071&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2018.03.071%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DC.%26aulast%3DHan%26aufirst%3DX.-R.%26aulast%3DYang%26aufirst%3DX.%26aulast%3DJiang%26aufirst%3DB.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DXiong%26aufirst%3DY.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DProteolysis%2520Targeting%2520Chimeras%2520%2528PROTACs%2529%2520of%2520Anaplastic%2520Lymphoma%2520Kinase%2520%2528ALK%2529%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2018%26volume%3D151%26spage%3D304%26epage%3D314%26doi%3D10.1016%2Fj.ejmech.2018.03.071" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Piya, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhattacharya, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lorenzi, P. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McQueen, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruvolo, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baran, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qian, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Konopleva, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ishizawa, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">You, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kantarjian, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andreeff, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borthakur, G.</span></span> <span> </span><span class="NLM_article-title">BETP degradation simultaneously targets acute myelogenous leukemia stem cells and the microenvironment</span>. <i>J. Clin. Invest.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>129</i></span>,  <span class="NLM_fpage">1878</span>– <span class="NLM_lpage">1894</span>, <span class="refDoi"> DOI: 10.1172/jci120654</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=10.1172%2FJCI120654" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=30829648" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=1%3ACAS%3A280%3ADC%252BB3cbgtVSiuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=129&publication_year=2019&pages=1878-1894&author=S.+Piyaauthor=H.+Muauthor=S.+Bhattacharyaauthor=P.+L.+Lorenziauthor=R.+E.+Davisauthor=T.+McQueenauthor=V.+Ruvoloauthor=N.+Baranauthor=Z.+Wangauthor=Y.+Qianauthor=C.+M.+Crewsauthor=M.+Konoplevaauthor=J.+Ishizawaauthor=M.+J.+Youauthor=H.+Kantarjianauthor=M.+Andreeffauthor=G.+Borthakur&title=BETP+degradation+simultaneously+targets+acute+myelogenous+leukemia+stem+cells+and+the+microenvironment&doi=10.1172%2Fjci120654"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">BETP degradation simultaneously targets acute myelogenous leukemia stem cells and the microenvironment</span></div><div class="casAuthors">Piya Sujan; Mu Hong; Bhattacharya Seemana; McQueen Teresa; Ruvolo Vivian; Baran Natalia; Andreeff Michael; Piya Sujan; Mu Hong; Bhattacharya Seemana; McQueen Teresa; Ruvolo Vivian; Baran Natalia; Konopleva Marina; Ishizawa Jo; Kantarjian Hagop; Andreeff Michael; Borthakur Gautam; Lorenzi Philip L; Davis R Eric; Wang Zhiqiang; Qian Yimin; Crews Craig M; You M James</div><div class="citationInfo"><span class="NLM_cas:title">The Journal of clinical investigation</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">129</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1878-1894</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Anti-leukemic effect of BET/BRD4 (BETP) protein inhibition has been largely attributed to transcriptional downregulation of cellular anabolic/anti-apoptotic processes but its effect on bone marrow microenvironment, a sanctuary favoring persistence of leukemia stem/progenitor cells, is unexplored.  Sustained degradation of BETP with small-molecule BET proteolysis-targeting chimera (PROTAC), ARV-825, resulted in marked downregulation of surface CXCR4 and CD44, key proteins in leukemia-microenvironment interaction, in AML cells.  Abrogation of surface CXCR4 expression impaired SDF-1α directed migration and was mediated through transcriptional down-regulation of PIM1 kinase that in turn phosphorylates CXCR4 and facilitates its surface localization.  Down-regulation of CD44/CD44v8-10 impaired cystine uptake, lowered intracellular reduced glutathione and increased oxidative stress.  More importantly, BETP degradation markedly decreased CD34+CD38-CD90-CD45RA+ leukemic stem cell population and alone or in combination with Cytarabine, prolonged survival in mouse model of human leukemia including AML-PDX.  Gene expression profiling and single cell proteomics confirmed down regulation of the gene signatures associated with 'stemness' in AML and Wnt/β-catenin, Myc pathways.  Hence, BETP degradation by ARV-825 simultaneously targets cell intrinsic signaling, stromal interactions and metabolism in AML.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcT9Wy3XYe-G80bxh97K6Vt6fW6udTcc2ebDMaUpfRCmy7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3cbgtVSiuw%253D%253D&md5=6f60455bcebfab2cba3ad3b5e68b022a</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1172%2FJCI120654&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1172%252FJCI120654%26sid%3Dliteratum%253Aachs%26aulast%3DPiya%26aufirst%3DS.%26aulast%3DMu%26aufirst%3DH.%26aulast%3DBhattacharya%26aufirst%3DS.%26aulast%3DLorenzi%26aufirst%3DP.%2BL.%26aulast%3DDavis%26aufirst%3DR.%2BE.%26aulast%3DMcQueen%26aufirst%3DT.%26aulast%3DRuvolo%26aufirst%3DV.%26aulast%3DBaran%26aufirst%3DN.%26aulast%3DWang%26aufirst%3DZ.%26aulast%3DQian%26aufirst%3DY.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26aulast%3DKonopleva%26aufirst%3DM.%26aulast%3DIshizawa%26aufirst%3DJ.%26aulast%3DYou%26aufirst%3DM.%2BJ.%26aulast%3DKantarjian%26aufirst%3DH.%26aulast%3DAndreeff%26aufirst%3DM.%26aulast%3DBorthakur%26aufirst%3DG.%26atitle%3DBETP%2520degradation%2520simultaneously%2520targets%2520acute%2520myelogenous%2520leukemia%2520stem%2520cells%2520and%2520the%2520microenvironment%26jtitle%3DJ.%2520Clin.%2520Invest.%26date%3D2019%26volume%3D129%26spage%3D1878%26epage%3D1894%26doi%3D10.1172%2Fjci120654" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shibata, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shimokawa, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nagai, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohoka, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hattori, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miyamoto, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ujikawa, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sameshima, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nara, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cho, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Naito, M.</span></span> <span> </span><span class="NLM_article-title">Pharmacological difference between degrader and inhibitor against oncogenic BCR-ABL kinase</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">13549</span>, <span class="refDoi"> DOI: 10.1038/s41598-018-31913-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=10.1038%2Fs41598-018-31913-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=30202081" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=1%3ACAS%3A280%3ADC%252BB3c3nsFSltg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2018&pages=13549&author=N.+Shibataauthor=K.+Shimokawaauthor=K.+Nagaiauthor=N.+Ohokaauthor=T.+Hattoriauthor=N.+Miyamotoauthor=O.+Ujikawaauthor=T.+Sameshimaauthor=H.+Naraauthor=N.+Choauthor=M.+Naito&title=Pharmacological+difference+between+degrader+and+inhibitor+against+oncogenic+BCR-ABL+kinase&doi=10.1038%2Fs41598-018-31913-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Pharmacological difference between degrader and inhibitor against oncogenic BCR-ABL kinase</span></div><div class="casAuthors">Shibata Norihito; Ohoka Nobumichi; Hattori Takayuki; Naito Mikihiko; Shimokawa Kenichiro; Nagai Katsunori; Miyamoto Naoki; Ujikawa Osamu; Sameshima Tomoya; Nara Hiroshi; Cho Nobuo; Nagai Katsunori; Ujikawa Osamu; Nara Hiroshi; Cho Nobuo</div><div class="citationInfo"><span class="NLM_cas:title">Scientific reports</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">13549</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Chronic myelogenous leukemia (CML) is characterized by the oncogenic fusion protein, BCR-ABL protein kinase, against which clinically useful inhibitors have been developed.  An alternative approach to treat CML is to degrade the BCR-ABL protein.  Recently, potent degraders against BCR-ABL have been developed by conjugating dasatinib to ligands for E3 ubiquitin ligases.  Since the degraders contain the dasatinib moiety, they also inhibit BCR-ABL kinase activity, which complicates our understanding of the impact of BCR-ABL degradation by degraders in CML growth inhibition.  To address this issue, we chose DAS-IAP, as a potent BCR-ABL degrader, and developed a structurally related inactive degrader, DAS-meIAP, which inhibits kinase activity but does not degrade the BCR-ABL protein.  DAS-IAP showed slightly weaker activity than DAS-meIAP in inhibiting cell growth when CML cells were treated for 48 h.  However, DAS-IAP showed sustained growth inhibition even when the drug was removed after short-term treatment, whereas CML cell growth rapidly resumed following removal of DAS-meIAP and dasatinib.  Consistently, suppression of BCR-ABL levels and downstream kinase signaling were maintained after DAS-IAP removal, whereas kinase signaling rapidly recovered following removal of DAS-meIAP and dasatinib.  These results indicate that BCR-ABL degrader shows more sustained inhibition of CML cell growth than ABL kinase inhibitor.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcS1tFvEwEman0cLvDNbOD4DfW6udTcc2eZWbh9AQXs-L7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3c3nsFSltg%253D%253D&md5=f55ad999fd61e89babe3410fbcd763e4</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1038%2Fs41598-018-31913-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41598-018-31913-5%26sid%3Dliteratum%253Aachs%26aulast%3DShibata%26aufirst%3DN.%26aulast%3DShimokawa%26aufirst%3DK.%26aulast%3DNagai%26aufirst%3DK.%26aulast%3DOhoka%26aufirst%3DN.%26aulast%3DHattori%26aufirst%3DT.%26aulast%3DMiyamoto%26aufirst%3DN.%26aulast%3DUjikawa%26aufirst%3DO.%26aulast%3DSameshima%26aufirst%3DT.%26aulast%3DNara%26aufirst%3DH.%26aulast%3DCho%26aufirst%3DN.%26aulast%3DNaito%26aufirst%3DM.%26atitle%3DPharmacological%2520difference%2520between%2520degrader%2520and%2520inhibitor%2520against%2520oncogenic%2520BCR-ABL%2520kinase%26jtitle%3DSci.%2520Rep.%26date%3D2018%26volume%3D8%26spage%3D13549%26doi%3D10.1038%2Fs41598-018-31913-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Steeg, P. S.</span></span> <span> </span><span class="NLM_article-title">Targeting metastasis</span>. <i>Nat. Rev. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">201</span>– <span class="NLM_lpage">218</span>, <span class="refDoi"> DOI: 10.1038/nrc.2016.25</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=10.1038%2Fnrc.2016.25" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=27009393" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=1%3ACAS%3A528%3ADC%252BC28XkslKnsLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2016&pages=201-218&author=P.+S.+Steeg&title=Targeting+metastasis&doi=10.1038%2Fnrc.2016.25"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting metastasis</span></div><div class="casAuthors">Steeg, Patricia S.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">201-218</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Tumor metastasis, the movement of tumor cells from a primary site to progressively colonize distant organs, is a major contributor to the deaths of cancer patients.  Therapeutic goals are the prevention of an initial metastasis in high-risk patients, shrinkage of established lesions and prevention of addnl. metastases in patients with limited disease.  Instead of being autonomous, tumor cells engage in bidirectional interactions with metastatic microenvironments to alter antitumor immunity, the extracellular milieu, genomic stability, survival signalling, chemotherapeutic resistance and proliferative cycles.  Can targeting of these interactions significantly improve patient outcomes.  In this Review preclin. research, combination therapies and clin. trial designs are re-examd.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqjUyWsbBasb7Vg90H21EOLACvtfcHk0ljvqChVpn91NA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XkslKnsLo%253D&md5=86df7851bdfa0d97e351494a876e9bb7</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1038%2Fnrc.2016.25&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc.2016.25%26sid%3Dliteratum%253Aachs%26aulast%3DSteeg%26aufirst%3DP.%2BS.%26atitle%3DTargeting%2520metastasis%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2016%26volume%3D16%26spage%3D201%26epage%3D218%26doi%3D10.1038%2Fnrc.2016.25" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Roelle, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosse, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buech, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chubanov, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gudermann, T.</span></span> <span> </span><span class="NLM_article-title">Essential role of Pyk2 and Src kinase activation in neuropeptide-induced proliferation of small cell lung cancer cells</span>. <i>Oncogene</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">1737</span>– <span class="NLM_lpage">1748</span>, <span class="refDoi"> DOI: 10.1038/sj.onc.1210819</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=10.1038%2Fsj.onc.1210819" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=17906699" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=1%3ACAS%3A528%3ADC%252BD1cXjtFOnsLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2008&pages=1737-1748&author=S.+Roelleauthor=R.+Grosseauthor=T.+Buechauthor=V.+Chubanovauthor=T.+Gudermann&title=Essential+role+of+Pyk2+and+Src+kinase+activation+in+neuropeptide-induced+proliferation+of+small+cell+lung+cancer+cells&doi=10.1038%2Fsj.onc.1210819"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Essential role of Pyk2 and Src kinase activation in neuropeptide-induced proliferation of small cell lung cancer cells</span></div><div class="casAuthors">Roelle, S.; Grosse, R.; Buech, T.; Chubanov, V.; Gudermann, T.</div><div class="citationInfo"><span class="NLM_cas:title">Oncogene</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1737-1748</span>CODEN:
                <span class="NLM_cas:coden">ONCNES</span>;
        ISSN:<span class="NLM_cas:issn">0950-9232</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Neuropeptide hormones like bombesin/gastrin-releasing peptide, galanin or bradykinin, acting via auto and paracrine growth loops, represent the principal mitogens of small cell lung cancer (SCLC).  These mitogenic neuropeptides activate Gq/11-coupled receptors which stimulate phospholipase Cβ activity, followed by rises of the intracellular calcium concn. ([Ca2+]i) and activation of protein kinase C (PKC).  We report here that proline-rich tyrosine kinase Pyk2 is highly expressed in SCLC cells and provides a functional link between neuropeptide-induced increases in [Ca2+]i and tumor cell proliferation.  Activation of Pyk2 and its assocn. with Src kinases critically depends on the elevation of [Ca2+]i, but is independent of PKC.  Src kinase activities are crucial for neuropeptide-mediated GTP-loading of Ras and activation of extracellular signal-regulated kinases in SCLC cells.  Pyk2 and Src kinases essentially contribute to anchorage-independent proliferation of SCLC cells.  Inhibition of either Pyk2 or Src kinases by lentiviral RNAi or pharmacol. inhibition with PP2, resp., attenuated basal and neuropeptide-elicited survival and proliferation of SCLC cells in liq. culture and in soft agar.  Thus, neuropeptides stimulate anchorage-independent survival and proliferation of SCLC cells via pathways involving Pyk2 and Src kinases.  Therefore, Ca2+-induced Pyk2/Src complex formation may be a rewarding mol. target for novel therapeutic strategies in SCLC cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpbVsMd34QCG7Vg90H21EOLACvtfcHk0ljvqChVpn91NA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXjtFOnsLs%253D&md5=f8470f0d6c8bf1df80ca003fe507f726</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1038%2Fsj.onc.1210819&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.onc.1210819%26sid%3Dliteratum%253Aachs%26aulast%3DRoelle%26aufirst%3DS.%26aulast%3DGrosse%26aufirst%3DR.%26aulast%3DBuech%26aufirst%3DT.%26aulast%3DChubanov%26aufirst%3DV.%26aulast%3DGudermann%26aufirst%3DT.%26atitle%3DEssential%2520role%2520of%2520Pyk2%2520and%2520Src%2520kinase%2520activation%2520in%2520neuropeptide-induced%2520proliferation%2520of%2520small%2520cell%2520lung%2520cancer%2520cells%26jtitle%3DOncogene%26date%3D2008%26volume%3D27%26spage%3D1737%26epage%3D1748%26doi%3D10.1038%2Fsj.onc.1210819" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kuang, B.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, M.-Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, L.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, L.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H.-B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, W.-F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Du, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span> <span> </span><span class="NLM_article-title">Proline-rich tyrosine kinase 2 and its phosphorylated form pY881 are novel prognostic markers for non-small-cell lung cancer progression and patients’ overall survival</span>. <i>Br. J. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>109</i></span>,  <span class="NLM_fpage">1252</span>– <span class="NLM_lpage">1263</span>, <span class="refDoi"> DOI: 10.1038/bjc.2013.439</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=10.1038%2Fbjc.2013.439" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=23922106" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsVSktbjI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=109&publication_year=2013&pages=1252-1263&author=B.-H.+Kuangauthor=M.-Q.+Zhangauthor=L.-H.+Xuauthor=L.-J.+Huauthor=H.-B.+Wangauthor=W.-F.+Zhaoauthor=Y.+Duauthor=X.+Zhang&title=Proline-rich+tyrosine+kinase+2+and+its+phosphorylated+form+pY881+are+novel+prognostic+markers+for+non-small-cell+lung+cancer+progression+and+patients%E2%80%99+overall+survival&doi=10.1038%2Fbjc.2013.439"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Proline-rich tyrosine kinase 2 and its phosphorylated form pY881 are novel prognostic markers for non-small-cell lung cancer progression and patients' overall survival</span></div><div class="casAuthors">Kuang, B.-H.; Zhang, M.-Q.; Xu, L.-H.; Hu, L.-J.; Wang, H.-B.; Zhao, W.-F.; Du, Y.; Zhang, X.</div><div class="citationInfo"><span class="NLM_cas:title">British Journal of Cancer</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">109</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1252-1263</span>CODEN:
                <span class="NLM_cas:coden">BJCAAI</span>;
        ISSN:<span class="NLM_cas:issn">0007-0920</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Background: Our previous study revealed that proline-rich tyrosine kinase 2 (Pyk2) is implicated in both anchorage-independent growth and anoikis resistance in lung cancer cells.  This study aims to explore the expression and clin. significance of Pyk2 and its phosphorylated forms in non-small-cell lung cancer (NSCLC).  Methods: The mRNA and protein levels of Pyk2 or cancer stem cell markers (ALDH1a1, ABCG2 and Bmi-1) were either examd. by reverse transcription-PCR or western blotting.  An immunohistochem. (IHC) assay was conducted to analyze the expression of Pyk2 and its phosphorylated forms in 128 NSCLC cases.  Results: The levels of Pyk2 mRNA, total protein, and its phosphorylated form pY881 were higher in lung cancer lesions than in the paired noncancerous tissues.  The IHC anal. showed the levels of the Pyk2 and Pyk2[pY881] proteins were highly expressed in 70 (54.7%) and 77 (60.2%) cases, resp.  Both Pyk2 and Pyk2[pY881] were independent prognostic factors for NSCLC patients.  The gain and loss study of Pyk2 function revealed that Pyk2 could upregulate the expression of ALDH1a1, ABCG2 and Bmi-1 and enhance the ability of colony formation in soft agar assay in A549 and H460 cells.  Conclusion: Both Pyk2 and phosphorylated Pyk2[pY881] are potential prognostic factors and therapeutic targets for NSCLC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrBW_TkawEATLVg90H21EOLACvtfcHk0li-zfOZQgzdEg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsVSktbjI&md5=ca5d562b59b1c6fce42ef30acf669807</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1038%2Fbjc.2013.439&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fbjc.2013.439%26sid%3Dliteratum%253Aachs%26aulast%3DKuang%26aufirst%3DB.-H.%26aulast%3DZhang%26aufirst%3DM.-Q.%26aulast%3DXu%26aufirst%3DL.-H.%26aulast%3DHu%26aufirst%3DL.-J.%26aulast%3DWang%26aufirst%3DH.-B.%26aulast%3DZhao%26aufirst%3DW.-F.%26aulast%3DDu%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DX.%26atitle%3DProline-rich%2520tyrosine%2520kinase%25202%2520and%2520its%2520phosphorylated%2520form%2520pY881%2520are%2520novel%2520prognostic%2520markers%2520for%2520non-small-cell%2520lung%2520cancer%2520progression%2520and%2520patients%25E2%2580%2599%2520overall%2520survival%26jtitle%3DBr.%2520J.%2520Cancer%26date%3D2013%26volume%3D109%26spage%3D1252%26epage%3D1263%26doi%3D10.1038%2Fbjc.2013.439" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moschetta, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huynh, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tai, Y.-T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mishima, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ring, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tam, W. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maiso, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reagan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sahin, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sacco, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manier, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aljawai, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glavey, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Munshi, N. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anderson, K. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pachter, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roccaro, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ghobrial, I. M.</span></span> <span> </span><span class="NLM_article-title">Pyk2 promotes tumor progression in multiple myeloma</span>. <i>Blood</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>124</i></span>,  <span class="NLM_fpage">2675</span>– <span class="NLM_lpage">2686</span>, <span class="refDoi"> DOI: 10.1182/blood-2014-03-563981</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=10.1182%2Fblood-2014-03-563981" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=25217697" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvVygsrjO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=124&publication_year=2014&pages=2675-2686&author=Y.+Zhangauthor=M.+Moschettaauthor=D.+Huynhauthor=Y.-T.+Taiauthor=Y.+Zhangauthor=W.+Zhangauthor=Y.+Mishimaauthor=J.+E.+Ringauthor=W.+F.+Tamauthor=Q.+Xuauthor=P.+Maisoauthor=M.+Reaganauthor=I.+Sahinauthor=A.+Saccoauthor=S.+Manierauthor=Y.+Aljawaiauthor=S.+Glaveyauthor=N.+C.+Munshiauthor=K.+C.+Andersonauthor=J.+Pachterauthor=A.+M.+Roccaroauthor=I.+M.+Ghobrial&title=Pyk2+promotes+tumor+progression+in+multiple+myeloma&doi=10.1182%2Fblood-2014-03-563981"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Pyk2 promotes tumor progression in multiple myeloma</span></div><div class="casAuthors">Zhang, Yu; Moschetta, Michele; Huynh, Daisy; Tai, Yu-Tzu; Zhang, Yong; Zhang, Wenjing; Mishima, Yuji; Ring, Jennifer E.; Tam, Winnie F.; Xu, Qunli; Maiso, Patricia; Reagan, Michaela; Sahin, Ilyas; Sacco, Antonio; Manier, Salomon; Aljawai, Yosra; Glavey, Siobhan; Munshi, Nikhil C.; Anderson, Kenneth C.; Pachter, Jonathan; Roccaro, Aldo M.; Ghobrial, Irene M.</div><div class="citationInfo"><span class="NLM_cas:title">Blood</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">124</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">2675-2686</span>CODEN:
                <span class="NLM_cas:coden">BLOOAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-4971</span>.
    
            (<span class="NLM_cas:orgname">American Society of Hematology</span>)
        </div><div class="casAbstract">Proline-rich tyrosine kinase 2 (Pyk2) is a member of the focal adhesion kinase family that has been recently linked to tumor development.  However, its role in modulating multiple myeloma (MM) biol. and disease progression remains unexplored.  We first demonstrated that patients with MM present with higher expression of Pyk2 compared with healthy individuals.  By using loss-of-function approaches, we found that Pyk2 inhibition led to redn. of MM tumor growth in vivo as well as decreased cell proliferation, cell-cycle progression, and adhesion ability in vitro.  In turn, overexpression of Pyk2 promoted the malignant phenotype, substantiated by enhanced tumor growth and reduced survival.  Mechanistically, inhibition of Pyk2 reduced activation of Wnt/β-catenin signaling by destabilizing β-catenin, leading to downregulation of c-Myc and Cyclin D1.  Furthermore, treatment of MM cells with the FAK/Pyk2 inhibitor VS-4718 effectively inhibited MM cell growth both in vitro and in vivo.  Collectively, our findings describe the tumor-promoting role of Pyk2 in MM, thus providing mol. evidence for a novel tyrosine kinase inhibitor as a new therapeutic option in MM.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq7K0Ip0dTGuLVg90H21EOLACvtfcHk0li-zfOZQgzdEg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvVygsrjO&md5=a1af2c2927709cfed992c8626738c849</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1182%2Fblood-2014-03-563981&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1182%252Fblood-2014-03-563981%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DMoschetta%26aufirst%3DM.%26aulast%3DHuynh%26aufirst%3DD.%26aulast%3DTai%26aufirst%3DY.-T.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DW.%26aulast%3DMishima%26aufirst%3DY.%26aulast%3DRing%26aufirst%3DJ.%2BE.%26aulast%3DTam%26aufirst%3DW.%2BF.%26aulast%3DXu%26aufirst%3DQ.%26aulast%3DMaiso%26aufirst%3DP.%26aulast%3DReagan%26aufirst%3DM.%26aulast%3DSahin%26aufirst%3DI.%26aulast%3DSacco%26aufirst%3DA.%26aulast%3DManier%26aufirst%3DS.%26aulast%3DAljawai%26aufirst%3DY.%26aulast%3DGlavey%26aufirst%3DS.%26aulast%3DMunshi%26aufirst%3DN.%2BC.%26aulast%3DAnderson%26aufirst%3DK.%2BC.%26aulast%3DPachter%26aufirst%3DJ.%26aulast%3DRoccaro%26aufirst%3DA.%2BM.%26aulast%3DGhobrial%26aufirst%3DI.%2BM.%26atitle%3DPyk2%2520promotes%2520tumor%2520progression%2520in%2520multiple%2520myeloma%26jtitle%3DBlood%26date%3D2014%26volume%3D124%26spage%3D2675%26epage%3D2686%26doi%3D10.1182%2Fblood-2014-03-563981" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hegde, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knolhoff, B. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herndon, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meyer, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nywening, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hawkins, W. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shapiro, I. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weaver, D. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pachter, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang-Gillam, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DeNardo, D. G.</span></span> <span> </span><span class="NLM_article-title">Targeting focal adhesion kinase renders pancreatic cancers responsive to checkpoint immunotherapy</span>. <i>Nat. Med.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">851</span>– <span class="NLM_lpage">860</span>, <span class="refDoi"> DOI: 10.1038/nm.4123</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=10.1038%2Fnm.4123" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=27376576" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtV2msLfN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2016&pages=851-860&author=H.+Jiangauthor=S.+Hegdeauthor=B.+L.+Knolhoffauthor=Y.+Zhuauthor=J.+M.+Herndonauthor=M.+A.+Meyerauthor=T.+M.+Nyweningauthor=W.+G.+Hawkinsauthor=I.+M.+Shapiroauthor=D.+T.+Weaverauthor=J.+A.+Pachterauthor=A.+Wang-Gillamauthor=D.+G.+DeNardo&title=Targeting+focal+adhesion+kinase+renders+pancreatic+cancers+responsive+to+checkpoint+immunotherapy&doi=10.1038%2Fnm.4123"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting focal adhesion kinase renders pancreatic cancers responsive to checkpoint immunotherapy</span></div><div class="casAuthors">Jiang, Hong; Hegde, Samarth; Knolhoff, Brett L.; Zhu, Yu; Herndon, John M.; Meyer, Melissa A.; Nywening, Timothy M.; Hawkins, William G.; Shapiro, Irina M.; Weaver, David T.; Pachter, Jonathan A.; Wang-Gillam, Andrea; DeNardo, David G.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Medicine (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">851-860</span>CODEN:
                <span class="NLM_cas:coden">NAMEFI</span>;
        ISSN:<span class="NLM_cas:issn">1078-8956</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Single-agent immunotherapy has achieved limited clin. benefit to date in patients with pancreatic ductal adenocarcinoma (PDAC).  This may be a result of the presence of a uniquely immunosuppressive tumor microenvironment (TME).  Crit. obstacles to immunotherapy in PDAC tumors include a high no. of tumor-assocd. immunosuppressive cells and a uniquely desmoplastic stroma that functions as a barrier to T cell infiltration.  We identified hyperactivated focal adhesion kinase (FAK) activity in neoplastic PDAC cells as an important regulator of the fibrotic and immunosuppressive TME.  We found that FAK activity was elevated in human PDAC tissues and correlated with high levels of fibrosis and poor CD8+ cytotoxic T cell infiltration.  Single-agent FAK inhibition using the selective FAK inhibitor VS-4718 substantially limited tumor progression, resulting in a doubling of survival in the p48-Cre;LSL-KrasG12D;Trp53flox/+ (KPC) mouse model of human PDAC.  This delay in tumor progression was assocd. with markedly reduced tumor fibrosis and decreased nos. of tumor-infiltrating immunosuppressive cells.  We also found that FAK inhibition rendered the previously unresponsive KPC mouse model responsive to T cell immunotherapy and PD-1 antagonists.  These data suggest that FAK inhibition increases immune surveillance by overcoming the fibrotic and immunosuppressive PDAC TME and renders tumors responsive to immunotherapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp7vp95t6jl9bVg90H21EOLACvtfcHk0lh_xHJuK_VVhA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtV2msLfN&md5=1f715175ab16c97f696999001422df4c</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1038%2Fnm.4123&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnm.4123%26sid%3Dliteratum%253Aachs%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DHegde%26aufirst%3DS.%26aulast%3DKnolhoff%26aufirst%3DB.%2BL.%26aulast%3DZhu%26aufirst%3DY.%26aulast%3DHerndon%26aufirst%3DJ.%2BM.%26aulast%3DMeyer%26aufirst%3DM.%2BA.%26aulast%3DNywening%26aufirst%3DT.%2BM.%26aulast%3DHawkins%26aufirst%3DW.%2BG.%26aulast%3DShapiro%26aufirst%3DI.%2BM.%26aulast%3DWeaver%26aufirst%3DD.%2BT.%26aulast%3DPachter%26aufirst%3DJ.%2BA.%26aulast%3DWang-Gillam%26aufirst%3DA.%26aulast%3DDeNardo%26aufirst%3DD.%2BG.%26atitle%3DTargeting%2520focal%2520adhesion%2520kinase%2520renders%2520pancreatic%2520cancers%2520responsive%2520to%2520checkpoint%2520immunotherapy%26jtitle%3DNat.%2520Med.%26date%3D2016%26volume%3D22%26spage%3D851%26epage%3D860%26doi%3D10.1038%2Fnm.4123" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sulzmaier, F. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jean, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schlaepfer, D. D.</span></span> <span> </span><span class="NLM_article-title">FAK in cancer: mechanistic findings and clinical applications</span>. <i>Nat. Rev. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">598</span>– <span class="NLM_lpage">610</span>, <span class="refDoi"> DOI: 10.1038/nrc3792</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=10.1038%2Fnrc3792" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=25098269" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=1%3ACAS%3A528%3ADC%252BC2cXht12nu7vJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2014&pages=598-610&author=F.+J.+Sulzmaierauthor=C.+Jeanauthor=D.+D.+Schlaepfer&title=FAK+in+cancer%3A+mechanistic+findings+and+clinical+applications&doi=10.1038%2Fnrc3792"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">FAK in cancer: mechanistic findings and clinical applications</span></div><div class="casAuthors">Sulzmaier, Florian J.; Jean, Christine; Schlaepfer, David D.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">598-610</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Focal adhesion kinase (FAK) is a cytoplasmic protein tyrosine kinase that is overexpressed and activated in several advanced-stage solid cancers.  FAK promotes tumor progression and metastasis through effects on cancer cells, as well as stromal cells of the tumor microenvironment.  The kinase-dependent and kinase-independent functions of FAK control cell movement, invasion, survival, gene expression and cancer stem cell self-renewal.  Small mol. FAK inhibitors decrease tumor growth and metastasis in several preclin. models and have initial clin. activity in patients with limited adverse events.  In this Review, we discuss FAK signalling effects on both tumor and stromal cell biol. that provide rationale and support for future therapeutic opportunities.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrmKc7m7dsY67Vg90H21EOLACvtfcHk0lh_xHJuK_VVhA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXht12nu7vJ&md5=f498bcf7af890d2f92eda68e42f979d1</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1038%2Fnrc3792&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc3792%26sid%3Dliteratum%253Aachs%26aulast%3DSulzmaier%26aufirst%3DF.%2BJ.%26aulast%3DJean%26aufirst%3DC.%26aulast%3DSchlaepfer%26aufirst%3DD.%2BD.%26atitle%3DFAK%2520in%2520cancer%253A%2520mechanistic%2520findings%2520and%2520clinical%2520applications%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2014%26volume%3D14%26spage%3D598%26epage%3D610%26doi%3D10.1038%2Fnrc3792" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Popow, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arnhof, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bader, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berger, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Covini, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dank, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gmaschitz, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greb, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karolyi-Özguer, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koegl, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McConnell, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pearson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rieger, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rinnenthal, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roessler, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schrenk, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spina, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steurer, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trainor, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Traxler, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wieshofer, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zoephel, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ettmayer, P.</span></span> <span> </span><span class="NLM_article-title">Highly Selective PTK2 Proteolysis Targeting Chimeras to Probe Focal Adhesion Kinase Scaffolding Functions</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">2508</span>– <span class="NLM_lpage">2520</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b01826</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b01826" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=1%3ACAS%3A528%3ADC%252BC1MXisl2ls78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=2508-2520&author=J.+Popowauthor=H.+Arnhofauthor=G.+Baderauthor=H.+Bergerauthor=A.+Ciulliauthor=D.+Coviniauthor=C.+Dankauthor=T.+Gmaschitzauthor=P.+Grebauthor=J.+Karolyi-%C3%96zguerauthor=M.+Koeglauthor=D.+B.+McConnellauthor=M.+Pearsonauthor=M.+Riegerauthor=J.+Rinnenthalauthor=V.+Roesslerauthor=A.+Schrenkauthor=M.+Spinaauthor=S.+Steurerauthor=N.+Trainorauthor=E.+Traxlerauthor=C.+Wieshoferauthor=A.+Zoephelauthor=P.+Ettmayer&title=Highly+Selective+PTK2+Proteolysis+Targeting+Chimeras+to+Probe+Focal+Adhesion+Kinase+Scaffolding+Functions&doi=10.1021%2Facs.jmedchem.8b01826"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Highly Selective PTK2 Proteolysis Targeting Chimeras to Probe Focal Adhesion Kinase Scaffolding Functions</span></div><div class="casAuthors">Popow, Johannes; Arnhof, Heribert; Bader, Gerd; Berger, Helmut; Ciulli, Alessio; Covini, David; Dank, Christian; Gmaschitz, Teresa; Greb, Peter; Karolyi-Oezguer, Jale; Koegl, Manfred; McConnell, Darryl B.; Pearson, Mark; Rieger, Maria; Rinnenthal, Joerg; Roessler, Vanessa; Schrenk, Andreas; Spina, Markus; Steurer, Steffen; Trainor, Nicole; Traxler, Elisabeth; Wieshofer, Corinna; Zoephel, Andreas; Ettmayer, Peter</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">2508-2520</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Focal adhesion tyrosine kinase (PTK2) is often overexpressed in human hepatocellular carcinoma (HCC), and several reports have linked PTK2 depletion and/or pharmacol. inhibition to reduced tumorigenicity.  However, the clin. relevance of targeting PTK2 still remains to be proven.  Here, we present two highly selective and functional PTK2 proteolysis-targeting chimeras utilizing von Hippel-Lindau and cereblon ligands to hijack E3 ligases for PTK2 degrdn.  BI-3663 (cereblon-based) degrades PTK2 with a median DC50 of 30 nM to >80% across a panel of 11 HCC cell lines.  Despite effective PTK2 degrdn., these compds. did not phenocopy the reported antiproliferative effects of PTK2 depletion in any of the cell lines tested.  By disclosing these compds., we hope to provide valuable tools for the study of PTK2 degrdn. across different biol. systems.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqsL2R_QBrfk7Vg90H21EOLACvtfcHk0lh_xHJuK_VVhA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXisl2ls78%253D&md5=f0ad27d3ff46ab12e8a90e8d40a6f3c6</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b01826%26sid%3Dliteratum%253Aachs%26aulast%3DPopow%26aufirst%3DJ.%26aulast%3DArnhof%26aufirst%3DH.%26aulast%3DBader%26aufirst%3DG.%26aulast%3DBerger%26aufirst%3DH.%26aulast%3DCiulli%26aufirst%3DA.%26aulast%3DCovini%26aufirst%3DD.%26aulast%3DDank%26aufirst%3DC.%26aulast%3DGmaschitz%26aufirst%3DT.%26aulast%3DGreb%26aufirst%3DP.%26aulast%3DKarolyi-%25C3%2596zguer%26aufirst%3DJ.%26aulast%3DKoegl%26aufirst%3DM.%26aulast%3DMcConnell%26aufirst%3DD.%2BB.%26aulast%3DPearson%26aufirst%3DM.%26aulast%3DRieger%26aufirst%3DM.%26aulast%3DRinnenthal%26aufirst%3DJ.%26aulast%3DRoessler%26aufirst%3DV.%26aulast%3DSchrenk%26aufirst%3DA.%26aulast%3DSpina%26aufirst%3DM.%26aulast%3DSteurer%26aufirst%3DS.%26aulast%3DTrainor%26aufirst%3DN.%26aulast%3DTraxler%26aufirst%3DE.%26aulast%3DWieshofer%26aufirst%3DC.%26aulast%3DZoephel%26aufirst%3DA.%26aulast%3DEttmayer%26aufirst%3DP.%26atitle%3DHighly%2520Selective%2520PTK2%2520Proteolysis%2520Targeting%2520Chimeras%2520to%2520Probe%2520Focal%2520Adhesion%2520Kinase%2520Scaffolding%2520Functions%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D2508%26epage%3D2520%26doi%3D10.1021%2Facs.jmedchem.8b01826" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Winter, G. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buckley, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paulk, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Souza, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dhe-Paganon, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span> <span> </span><span class="NLM_article-title">Phthalimide conjugation as a strategy for in vivo target protein degradation</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>348</i></span>,  <span class="NLM_fpage">1376</span>– <span class="NLM_lpage">1381</span>, <span class="refDoi"> DOI: 10.1126/science.aab1433</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=10.1126%2Fscience.aab1433" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=25999370" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=1%3ACAS%3A528%3ADC%252BC2MXpvFansLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=348&publication_year=2015&pages=1376-1381&author=G.+E.+Winterauthor=D.+L.+Buckleyauthor=J.+Paulkauthor=J.+M.+Robertsauthor=A.+Souzaauthor=S.+Dhe-Paganonauthor=J.+E.+Bradner&title=Phthalimide+conjugation+as+a+strategy+for+in+vivo+target+protein+degradation&doi=10.1126%2Fscience.aab1433"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Phthalimide conjugation as a strategy for in vivo target protein degradation</span></div><div class="casAuthors">Winter, Georg E.; Buckley, Dennis L.; Paulk, Joshiawa; Roberts, Justin M.; Souza, Amanda; Dhe-Paganon, Sirano; Bradner, James E.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">348</span>
        (<span class="NLM_cas:issue">6241</span>),
    <span class="NLM_cas:pages">1376-1381</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">The development of effective pharmacol. inhibitors of multidomain scaffold proteins, notably transcription factors, is a particularly challenging problem.  In part, this is because many small-mol. antagonists disrupt the activity of only one domain in the target protein.  The authors devised a chem. strategy that promotes ligand-dependent target protein degrdn. using as an example the transcriptional coactivator BRD4, a protein crit. for cancer cell growth and survival.  The authors appended a competitive antagonist of BET bromodomains to a phthalimide moiety to hijack the cereblon E3 ubiquitin ligase complex.  The resultant compd., dBET1, induced highly selective cereblon-dependent BET protein degrdn. in vitro and in vivo and delayed leukemia progression in mice.  A second series of probes resulted in selective degrdn. of the cytosolic protein FKBP12.  This chem. strategy for controlling target protein stability may have implications for therapeutically targeting previously intractable proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp1ZW4zjRjXi7Vg90H21EOLACvtfcHk0ljJmjJ6f2aHKQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXpvFansLk%253D&md5=b0277cc95d318d926b96e7a119066e3b</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1126%2Fscience.aab1433&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.aab1433%26sid%3Dliteratum%253Aachs%26aulast%3DWinter%26aufirst%3DG.%2BE.%26aulast%3DBuckley%26aufirst%3DD.%2BL.%26aulast%3DPaulk%26aufirst%3DJ.%26aulast%3DRoberts%26aufirst%3DJ.%2BM.%26aulast%3DSouza%26aufirst%3DA.%26aulast%3DDhe-Paganon%26aufirst%3DS.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26atitle%3DPhthalimide%2520conjugation%2520as%2520a%2520strategy%2520for%2520in%2520vivo%2520target%2520protein%2520degradation%26jtitle%3DScience%26date%3D2015%26volume%3D348%26spage%3D1376%26epage%3D1381%26doi%3D10.1126%2Fscience.aab1433" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Burslem, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jaime-Figueroa, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McQuaid, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bondeson, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toure, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qian, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crew, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hines, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">The Advantages of Targeted Protein Degradation Over Inhibition: An RTK Case Study</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">67</span>– <span class="NLM_lpage">77</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2017.09.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=10.1016%2Fj.chembiol.2017.09.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=29129716" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsl2lsrjM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2018&pages=67-77&author=G.+M.+Burslemauthor=B.+E.+Smithauthor=A.+C.+Laiauthor=S.+Jaime-Figueroaauthor=D.+C.+McQuaidauthor=D.+P.+Bondesonauthor=M.+Toureauthor=H.+Dongauthor=Y.+Qianauthor=J.+Wangauthor=A.+P.+Crewauthor=J.+Hinesauthor=C.+M.+Crews&title=The+Advantages+of+Targeted+Protein+Degradation+Over+Inhibition%3A+An+RTK+Case+Study&doi=10.1016%2Fj.chembiol.2017.09.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">The Advantages of Targeted Protein Degradation Over Inhibition: An RTK Case Study</span></div><div class="casAuthors">Burslem, George M.; Smith, Blake E.; Lai, Ashton C.; Jaime-Figueroa, Saul; McQuaid, Daniel C.; Bondeson, Daniel P.; Toure, Momar; Dong, Hanqing; Qian, Yimin; Wang, Jing; Crew, Andrew P.; Hines, John; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">67-77.e3</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Proteolysis targeting chimera (PROTAC) technol. has emerged over the last two decades as a powerful tool for targeted degrdn. of endogenous proteins.  Herein we describe the development of PROTACs for receptor tyrosine kinases, a protein family yet to be targeted for induced protein degrdn.  The use of VHL-recruiting PROTACs against this protein family reveals several advantages of degrdn. over inhibition alone: direct comparisons of fully functional, target-degrading PROTACs with target-inhibiting variants that contain an inactivated E3 ligase-recruiting ligand show that degrdn. leads to more potent inhibition of cell proliferation and a more durable and sustained downstream signaling response, and thus addresses the kinome rewiring challenge seen with many receptor tyrosine kinase inhibitors.  Combined, these findings demonstrate the ability to target receptor tyrosine kinases for degrdn. using the PROTAC technol. and outline the advantages of this degrdn.-based approach.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr4yDhRDWqNv7Vg90H21EOLACvtfcHk0ljJmjJ6f2aHKQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsl2lsrjM&md5=0ac39a6f5836ea3772a0396356efb630</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2017.09.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2017.09.009%26sid%3Dliteratum%253Aachs%26aulast%3DBurslem%26aufirst%3DG.%2BM.%26aulast%3DSmith%26aufirst%3DB.%2BE.%26aulast%3DLai%26aufirst%3DA.%2BC.%26aulast%3DJaime-Figueroa%26aufirst%3DS.%26aulast%3DMcQuaid%26aufirst%3DD.%2BC.%26aulast%3DBondeson%26aufirst%3DD.%2BP.%26aulast%3DToure%26aufirst%3DM.%26aulast%3DDong%26aufirst%3DH.%26aulast%3DQian%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DCrew%26aufirst%3DA.%2BP.%26aulast%3DHines%26aufirst%3DJ.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DThe%2520Advantages%2520of%2520Targeted%2520Protein%2520Degradation%2520Over%2520Inhibition%253A%2520An%2520RTK%2520Case%2520Study%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2018%26volume%3D25%26spage%3D67%26epage%3D77%26doi%3D10.1016%2Fj.chembiol.2017.09.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wolfer, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramaswamy, S.</span></span> <span> </span><span class="NLM_article-title">MYC and Metastasis: Figure 1</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>71</i></span>,  <span class="NLM_fpage">2034</span>– <span class="NLM_lpage">2037</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.can-10-3776</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=10.1158%2F0008-5472.CAN-10-3776" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=21406394" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=1%3ACAS%3A528%3ADC%252BC3MXjtFWgtro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=71&publication_year=2011&pages=2034-2037&author=A.+Wolferauthor=S.+Ramaswamy&title=MYC+and+Metastasis%3A+Figure+1&doi=10.1158%2F0008-5472.can-10-3776"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">MYC and Metastasis</span></div><div class="casAuthors">Wolfer, Anita; Ramaswamy, Sridhar</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">71</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2034-2037</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">A review.  Aggressive primary tumors express transcriptional signatures that correlate with their metastatic propensity.  A no. of these signatures have been deployed in the clinic as risk stratification tools.  However, the mol. basis of these clin. useful prognostic signatures has remained a largely unresolved area of controversy.  We recently found that many prognostic signatures reflect the activity of the MYC oncogene, which in turn regulates tumor metastasis through specific effects on cancer cell invasion and migration.  These findings offer a general framework for understanding the mol. basis of clin. prognostic transcriptional signatures and suggest potentially new avenues for studying metastasis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpw1WcZTRPskLVg90H21EOLACvtfcHk0ljJmjJ6f2aHKQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXjtFWgtro%253D&md5=26098aabe90bca765bfa743ef5d01909</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-10-3776&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-10-3776%26sid%3Dliteratum%253Aachs%26aulast%3DWolfer%26aufirst%3DA.%26aulast%3DRamaswamy%26aufirst%3DS.%26atitle%3DMYC%2520and%2520Metastasis%253A%2520Figure%25201%26jtitle%3DCancer%2520Res.%26date%3D2011%26volume%3D71%26spage%3D2034%26epage%3D2037%26doi%3D10.1158%2F0008-5472.can-10-3776" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Z.</span></span> <span> </span><span class="NLM_article-title">GEPIA: a web server for cancer and normal gene expression profiling and interactive analyses</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>45</i></span>,  <span class="NLM_fpage">W98</span>– <span class="NLM_lpage">W102</span>, <span class="refDoi"> DOI: 10.1093/nar/gkx247</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=10.1093%2Fnar%2Fgkx247" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=28407145" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhs1aitr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2017&pages=W98-W102&author=Z.+Tangauthor=C.+Liauthor=B.+Kangauthor=G.+Gaoauthor=C.+Liauthor=Z.+Zhang&title=GEPIA%3A+a+web+server+for+cancer+and+normal+gene+expression+profiling+and+interactive+analyses&doi=10.1093%2Fnar%2Fgkx247"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">GEPIA: a web server for cancer and normal gene expression profiling and interactive analyses</span></div><div class="casAuthors">Tang, Zefang; Li, Chenwei; Kang, Boxi; Gao, Ge; Li, Cheng; Zhang, Zemin</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">W1</span>),
    <span class="NLM_cas:pages">W98-W102</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">1362-4962</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Tremendous amt. of RNA sequencing data have been produced by large consortium projects such as TCGA and GTEx, creating new opportunities for data mining and deeper understanding of gene functions.  While certain existing web servers are valuable and widely used, many expression anal. functions needed by exptl. biologists are still not adequately addressed by these tools.  We introduce GEPIA (Gene Expression Profiling Interactive Anal.), a web-based tool to deliver fast and customizable functionalities based on TCGA and GTEx data.  GEPIA provides key interactive and customizable functions including differential expression anal., profiling plotting, correlation anal., patient survival anal., similar gene detection and dimensionality redn. anal.  The comprehensive expression analyses with simple clicking through GEPIA greatly facilitate data mining in wide research areas, scientific discussion and the therapeutic discovery process.  GEPIA fills in the gap between cancer genomics big data and the delivery of integrated information to end users, thus helping unleash the value of the current data resources.  GEPIA is available at http://gepia.cancer-pku.cn/.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoVU8vOW02u7bVg90H21EOLACvtfcHk0ljJmjJ6f2aHKQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhs1aitr8%253D&md5=3153a0801287ea9e24841f0fa24e9358</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkx247&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkx247%26sid%3Dliteratum%253Aachs%26aulast%3DTang%26aufirst%3DZ.%26aulast%3DLi%26aufirst%3DC.%26aulast%3DKang%26aufirst%3DB.%26aulast%3DGao%26aufirst%3DG.%26aulast%3DLi%26aufirst%3DC.%26aulast%3DZhang%26aufirst%3DZ.%26atitle%3DGEPIA%253A%2520a%2520web%2520server%2520for%2520cancer%2520and%2520normal%2520gene%2520expression%2520profiling%2520and%2520interactive%2520analyses%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2017%26volume%3D45%26spage%3DW98%26epage%3DW102%26doi%3D10.1093%2Fnar%2Fgkx247" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gadd, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Testa, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lucas, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, K.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lamont, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zengerle, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span> <span> </span><span class="NLM_article-title">Structural basis of PROTAC cooperative recognition for selective protein degradation</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">514</span>– <span class="NLM_lpage">521</span>, <span class="refDoi"> DOI: 10.1038/nchembio.2329</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=10.1038%2Fnchembio.2329" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=28288108" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=1%3ACAS%3A528%3ADC%252BC2sXkvFahtbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2017&pages=514-521&author=M.+S.+Gaddauthor=A.+Testaauthor=X.+Lucasauthor=K.-H.+Chanauthor=W.+Chenauthor=D.+J.+Lamontauthor=M.+Zengerleauthor=A.+Ciulli&title=Structural+basis+of+PROTAC+cooperative+recognition+for+selective+protein+degradation&doi=10.1038%2Fnchembio.2329"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Structural basis of PROTAC cooperative recognition for selective protein degradation</span></div><div class="casAuthors">Gadd, Morgan S.; Testa, Andrea; Lucas, Xavier; Chan, Kwok-Ho; Chen, Wenzhang; Lamont, Douglas J.; Zengerle, Michael; Ciulli, Alessio</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">514-521</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Inducing macromol. interactions with small mols. to activate cellular signaling is a challenging goal.  PROTACs (proteolysis-targeting chimeras) are bifunctional mols. that recruit a target protein in proximity to an E3 ubiquitin ligase to trigger protein degrdn.  Structural elucidation of the key ternary ligase-PROTAC-target species and its impact on target degrdn. selectivity remain elusive.  Here, we solved the crystal structure of Brd4 degrader MZ1 in complex with human VHL and the Brd4 bromodomain (Brd4BD2).  The ligand folds into itself to allow formation of specific intermol. interactions in the ternary complex.  Isothermal titrn. calorimetry studies, supported by surface mutagenesis and proximity assays, were consistent with pronounced cooperative formation of ternary complexes with Brd4BD2.  Structure-based-designed compd. AT1 exhibited highly selective depletion of Brd4 in cells.  These results elucidate how PROTAC-induced de novo contacts dictate preferential recruitment of a target protein into a stable and cooperative complex with an E3 ligase for selective degrdn.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoJri00jB5YJLVg90H21EOLACvtfcHk0liNAzASEcHUGg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXkvFahtbg%253D&md5=dfee13913106d362d0972d7b1a62d700</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.2329&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.2329%26sid%3Dliteratum%253Aachs%26aulast%3DGadd%26aufirst%3DM.%2BS.%26aulast%3DTesta%26aufirst%3DA.%26aulast%3DLucas%26aufirst%3DX.%26aulast%3DChan%26aufirst%3DK.-H.%26aulast%3DChen%26aufirst%3DW.%26aulast%3DLamont%26aufirst%3DD.%2BJ.%26aulast%3DZengerle%26aufirst%3DM.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DStructural%2520basis%2520of%2520PROTAC%2520cooperative%2520recognition%2520for%2520selective%2520protein%2520degradation%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2017%26volume%3D13%26spage%3D514%26epage%3D521%26doi%3D10.1038%2Fnchembio.2329" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nowak, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DeAngelo, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buckley, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Donovan, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">An, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Safaee, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jedrychowski, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ponthier, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ishoey, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mancias, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, E. S.</span></span> <span> </span><span class="NLM_article-title">Plasticity in binding confers selectivity in ligand-induced protein degradation</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">706</span>– <span class="NLM_lpage">714</span>, <span class="refDoi"> DOI: 10.1038/s41589-018-0055-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=10.1038%2Fs41589-018-0055-y" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=29892083" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtFWlurbK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2018&pages=706-714&author=R.+P.+Nowakauthor=S.+L.+DeAngeloauthor=D.+Buckleyauthor=Z.+Heauthor=K.+A.+Donovanauthor=J.+Anauthor=N.+Safaeeauthor=M.+P.+Jedrychowskiauthor=C.+M.+Ponthierauthor=M.+Ishoeyauthor=T.+Zhangauthor=J.+D.+Manciasauthor=N.+S.+Grayauthor=J.+E.+Bradnerauthor=E.+S.+Fischer&title=Plasticity+in+binding+confers+selectivity+in+ligand-induced+protein+degradation&doi=10.1038%2Fs41589-018-0055-y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Plasticity in binding confers selectivity in ligand-induced protein degradation</span></div><div class="casAuthors">Nowak, Radoslaw P.; De Angelo, Stephen L.; Buckley, Dennis; He, Zhixiang; Donovan, Katherine A.; An, Jian; Safaee, Nozhat; Jedrychowski, Mark P.; Ponthier, Charles M.; Ishoey, Mette; Zhang, Tinghu; Mancias, Joseph D.; Gray, Nathanael S.; Bradner, James E.; Fischer, Eric S.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">706-714</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Heterobifunctional small-mol. degraders that induce protein degrdn. through ligase-mediated ubiquitination have shown considerable promise as a new pharmacol. modality.  However, we currently lack a detailed understanding of the mol. basis for target recruitment and selectivity, which is critically required to enable rational design of degraders.  Here we utilized a comprehensive characterization of the ligand-dependent CRBN-BRD4 interaction to demonstrate that binding between proteins that have not evolved to interact is plastic.  Multiple x-ray crystal structures showed that plasticity resulted in several distinct low-energy binding conformations that were selectively bound by ligands.  We demonstrated that computational protein-protein docking can reveal the underlying interprotein contacts and inform the design of a BRD4 selective degrader that can discriminate between highly homologous BET bromodomains.  These findings that plastic interprotein contacts confer selectivity for ligand-induced protein dimerization provide a conceptual framework for the development of heterobifunctional ligands.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGreQasCW8bz2LVg90H21EOLACvtfcHk0liNAzASEcHUGg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtFWlurbK&md5=e4876c43b1eeb4558c23121b7a9fac27</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1038%2Fs41589-018-0055-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41589-018-0055-y%26sid%3Dliteratum%253Aachs%26aulast%3DNowak%26aufirst%3DR.%2BP.%26aulast%3DDeAngelo%26aufirst%3DS.%2BL.%26aulast%3DBuckley%26aufirst%3DD.%26aulast%3DHe%26aufirst%3DZ.%26aulast%3DDonovan%26aufirst%3DK.%2BA.%26aulast%3DAn%26aufirst%3DJ.%26aulast%3DSafaee%26aufirst%3DN.%26aulast%3DJedrychowski%26aufirst%3DM.%2BP.%26aulast%3DPonthier%26aufirst%3DC.%2BM.%26aulast%3DIshoey%26aufirst%3DM.%26aulast%3DZhang%26aufirst%3DT.%26aulast%3DMancias%26aufirst%3DJ.%2BD.%26aulast%3DGray%26aufirst%3DN.%2BS.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DFischer%26aufirst%3DE.%2BS.%26atitle%3DPlasticity%2520in%2520binding%2520confers%2520selectivity%2520in%2520ligand-induced%2520protein%2520degradation%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2018%26volume%3D14%26spage%3D706%26epage%3D714%26doi%3D10.1038%2Fs41589-018-0055-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Burslem, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Proteolysis-Targeting Chimeras as Therapeutics and Tools for Biological Discovery</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>181</i></span>,  <span class="NLM_fpage">102</span>– <span class="NLM_lpage">114</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2019.11.031</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=10.1016%2Fj.cell.2019.11.031" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=31955850" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=1%3ACAS%3A528%3ADC%252BB3cXht1Sitrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=181&publication_year=2020&pages=102-114&author=G.+M.+Burslemauthor=C.+M.+Crews&title=Proteolysis-Targeting+Chimeras+as+Therapeutics+and+Tools+for+Biological+Discovery&doi=10.1016%2Fj.cell.2019.11.031"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Proteolysis-Targeting Chimeras as Therapeutics and Tools for Biological Discovery</span></div><div class="casAuthors">Burslem, George M.; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">181</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">102-114</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  New biol. tools provide new techniques to probe fundamental biol. processes.  Here we describe the burgeoning field of proteolysis-targeting chimeras (PROTACs), which are capable of modulating protein concns. at a post-translational level by co-opting the ubiquitin-proteasome system.  We describe the PROTAC technol. and its application to drug discovery and provide examples where PROTACs have enabled novel biol. insights.  Furthermore, we provide a workflow for PROTAC development and use and discuss the benefits and issues assocd. with PROTACs.  Finally, we compare PROTAC-mediated protein-level modulation with other technologies, such as RNAi and genome editing.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrXgumsEakVWrVg90H21EOLACvtfcHk0liNAzASEcHUGg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXht1Sitrk%253D&md5=3010929d2979cf04dc6003e66575f1b0</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2019.11.031&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2019.11.031%26sid%3Dliteratum%253Aachs%26aulast%3DBurslem%26aufirst%3DG.%2BM.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DProteolysis-Targeting%2520Chimeras%2520as%2520Therapeutics%2520and%2520Tools%2520for%2520Biological%2520Discovery%26jtitle%3DCell%26date%3D2020%26volume%3D181%26spage%3D102%26epage%3D114%26doi%3D10.1016%2Fj.cell.2019.11.031" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Klaeger, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heinzlmeir, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilhelm, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Polzer, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vick, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koenig, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reinecke, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruprecht, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petzoldt, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meng, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zecha, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reiter, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qiao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Helm, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koch, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schoof, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Canevari, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Casale, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Depaolini, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feuchtinger, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmidt, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rueckert, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Becker, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huenges, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garz, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gohlke, B. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zolg, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kayser, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vooder, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Preissner, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hahne, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tonisson, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kramer, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gotze, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bassermann, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schlegl, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ehrlich, H. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aiche, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walch, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greif, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schneider, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Felder, E. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruland, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Medard, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeremias, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spiekermann, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuster, B.</span></span> <span> </span><span class="NLM_article-title">The target landscape of clinical kinase drugs</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>358</i></span>, <span class="NLM_elocation-id">eaan4368</span> <span class="refDoi"> DOI: 10.1126/science.aan4368</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=10.1126%2Fscience.aan4368" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=29191878" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=358&publication_year=2017&author=S.+Klaegerauthor=S.+Heinzlmeirauthor=M.+Wilhelmauthor=H.+Polzerauthor=B.+Vickauthor=P.+A.+Koenigauthor=M.+Reineckeauthor=B.+Ruprechtauthor=S.+Petzoldtauthor=C.+Mengauthor=J.+Zechaauthor=K.+Reiterauthor=H.+Qiaoauthor=D.+Helmauthor=H.+Kochauthor=M.+Schoofauthor=G.+Canevariauthor=E.+Casaleauthor=S.+R.+Depaoliniauthor=A.+Feuchtingerauthor=Z.+Wuauthor=T.+Schmidtauthor=L.+Rueckertauthor=W.+Beckerauthor=J.+Huengesauthor=A.+K.+Garzauthor=B.+O.+Gohlkeauthor=D.+P.+Zolgauthor=G.+Kayserauthor=T.+Vooderauthor=R.+Preissnerauthor=H.+Hahneauthor=N.+Tonissonauthor=K.+Kramerauthor=K.+Gotzeauthor=F.+Bassermannauthor=J.+Schleglauthor=H.+C.+Ehrlichauthor=S.+Aicheauthor=A.+Walchauthor=P.+A.+Greifauthor=S.+Schneiderauthor=E.+R.+Felderauthor=J.+Rulandauthor=G.+Medardauthor=I.+Jeremiasauthor=K.+Spiekermannauthor=B.+Kuster&title=The+target+landscape+of+clinical+kinase+drugs&doi=10.1126%2Fscience.aan4368"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1126%2Fscience.aan4368&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.aan4368%26sid%3Dliteratum%253Aachs%26aulast%3DKlaeger%26aufirst%3DS.%26aulast%3DHeinzlmeir%26aufirst%3DS.%26aulast%3DWilhelm%26aufirst%3DM.%26aulast%3DPolzer%26aufirst%3DH.%26aulast%3DVick%26aufirst%3DB.%26aulast%3DKoenig%26aufirst%3DP.%2BA.%26aulast%3DReinecke%26aufirst%3DM.%26aulast%3DRuprecht%26aufirst%3DB.%26aulast%3DPetzoldt%26aufirst%3DS.%26aulast%3DMeng%26aufirst%3DC.%26aulast%3DZecha%26aufirst%3DJ.%26aulast%3DReiter%26aufirst%3DK.%26aulast%3DQiao%26aufirst%3DH.%26aulast%3DHelm%26aufirst%3DD.%26aulast%3DKoch%26aufirst%3DH.%26aulast%3DSchoof%26aufirst%3DM.%26aulast%3DCanevari%26aufirst%3DG.%26aulast%3DCasale%26aufirst%3DE.%26aulast%3DDepaolini%26aufirst%3DS.%2BR.%26aulast%3DFeuchtinger%26aufirst%3DA.%26aulast%3DWu%26aufirst%3DZ.%26aulast%3DSchmidt%26aufirst%3DT.%26aulast%3DRueckert%26aufirst%3DL.%26aulast%3DBecker%26aufirst%3DW.%26aulast%3DHuenges%26aufirst%3DJ.%26aulast%3DGarz%26aufirst%3DA.%2BK.%26aulast%3DGohlke%26aufirst%3DB.%2BO.%26aulast%3DZolg%26aufirst%3DD.%2BP.%26aulast%3DKayser%26aufirst%3DG.%26aulast%3DVooder%26aufirst%3DT.%26aulast%3DPreissner%26aufirst%3DR.%26aulast%3DHahne%26aufirst%3DH.%26aulast%3DTonisson%26aufirst%3DN.%26aulast%3DKramer%26aufirst%3DK.%26aulast%3DGotze%26aufirst%3DK.%26aulast%3DBassermann%26aufirst%3DF.%26aulast%3DSchlegl%26aufirst%3DJ.%26aulast%3DEhrlich%26aufirst%3DH.%2BC.%26aulast%3DAiche%26aufirst%3DS.%26aulast%3DWalch%26aufirst%3DA.%26aulast%3DGreif%26aufirst%3DP.%2BA.%26aulast%3DSchneider%26aufirst%3DS.%26aulast%3DFelder%26aufirst%3DE.%2BR.%26aulast%3DRuland%26aufirst%3DJ.%26aulast%3DMedard%26aufirst%3DG.%26aulast%3DJeremias%26aufirst%3DI.%26aulast%3DSpiekermann%26aufirst%3DK.%26aulast%3DKuster%26aufirst%3DB.%26atitle%3DThe%2520target%2520landscape%2520of%2520clinical%2520kinase%2520drugs%26jtitle%3DScience%26date%3D2017%26volume%3D358%26doi%3D10.1126%2Fscience.aan4368" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kuenzi, B. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Remsing Rix, L. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stewart, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kinose, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bryant, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boyle, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koomen, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haura, E. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rix, U.</span></span> <span> </span><span class="NLM_article-title">Polypharmacology-based ceritinib repurposing using integrated functional proteomics</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">1222</span>– <span class="NLM_lpage">1231</span>, <span class="refDoi"> DOI: 10.1038/nchembio.2489</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=10.1038%2Fnchembio.2489" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=28991240" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhs1aisrzL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2017&pages=1222-1231&author=B.+M.+Kuenziauthor=L.+L.+Remsing+Rixauthor=P.+A.+Stewartauthor=B.+Fangauthor=F.+Kinoseauthor=A.+T.+Bryantauthor=T.+A.+Boyleauthor=J.+M.+Koomenauthor=E.+B.+Hauraauthor=U.+Rix&title=Polypharmacology-based+ceritinib+repurposing+using+integrated+functional+proteomics&doi=10.1038%2Fnchembio.2489"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Polypharmacology-based ceritinib repurposing using integrated functional proteomics</span></div><div class="casAuthors">Kuenzi, Brent M.; Remsing Rix, Lily L.; Stewart, Paul A.; Fang, Bin; Kinose, Fumi; Bryant, Annamarie T.; Boyle, Theresa A.; Koomen, John M.; Haura, Eric B.; Rix, Uwe</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1222-1231</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Targeted drugs are effective when they directly inhibit strong disease drivers, but only a small fraction of diseases feature defined actionable drivers.  Alternatively, network-based approaches can uncover new therapeutic opportunities.  Applying an integrated phenotypic screening, chem. and phosphoproteomics strategy, here the authors describe the anaplastic lymphoma kinase (ALK) inhibitor ceritinib as having activity across several ALK-neg. lung cancer cell lines and identify new targets and network-wide signaling effects.  Combining pharmacol. inhibitors and RNA interference revealed a polypharmacol. mechanism involving the noncanonical targets IGF1R, FAK1, RSK1 and RSK2.  Mutating the downstream signaling hub YB1 protected cells from ceritinib.  Consistent with YB1 signaling being known to cause taxol resistance, combination of ceritinib with paclitaxel displayed strong synergy, particularly in cells expressing high FAK autophosphorylation, which the authors show to be prevalent in lung cancer.  Together, the authors present a systems chem. biol. platform for elucidating multikinase inhibitor polypharmacol. mechanisms, subsequent design of synergistic drug combinations, and identification of mechanistic biomarker candidates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoECSbklR701rVg90H21EOLACvtfcHk0lgd-ms8WOMckA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhs1aisrzL&md5=8783f3602d51663cde58ed735a294dc2</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.2489&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.2489%26sid%3Dliteratum%253Aachs%26aulast%3DKuenzi%26aufirst%3DB.%2BM.%26aulast%3DRemsing%2BRix%26aufirst%3DL.%2BL.%26aulast%3DStewart%26aufirst%3DP.%2BA.%26aulast%3DFang%26aufirst%3DB.%26aulast%3DKinose%26aufirst%3DF.%26aulast%3DBryant%26aufirst%3DA.%2BT.%26aulast%3DBoyle%26aufirst%3DT.%2BA.%26aulast%3DKoomen%26aufirst%3DJ.%2BM.%26aulast%3DHaura%26aufirst%3DE.%2BB.%26aulast%3DRix%26aufirst%3DU.%26atitle%3DPolypharmacology-based%2520ceritinib%2520repurposing%2520using%2520integrated%2520functional%2520proteomics%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2017%26volume%3D13%26spage%3D1222%26epage%3D1231%26doi%3D10.1038%2Fnchembio.2489" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Donovan, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferguson, F. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bushman, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eleuteri, N. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhunia, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ryu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yue, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dobrovolsky, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hao, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">You, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Teng, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hatcher, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manz, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Groendyke, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nam, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sengupta, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cho, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shin, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Agius, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ghobrial, I. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, M. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Che, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buhrlage, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sim, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, E. S.</span></span> <span> </span><span class="NLM_article-title">Mapping the Degradable Kinome Provides a Resource for Expedited Degrader Development</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>183</i></span>,  <span class="NLM_fpage">1714</span>– <span class="NLM_lpage">1731</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2020.10.038</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=10.1016%2Fj.cell.2020.10.038" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=33275901" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=1%3ACAS%3A528%3ADC%252BB3cXisV2nsbjL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=183&publication_year=2020&pages=1714-1731&author=K.+A.+Donovanauthor=F.+M.+Fergusonauthor=J.+W.+Bushmanauthor=N.+A.+Eleuteriauthor=D.+Bhuniaauthor=S.+Ryuauthor=L.+Tanauthor=K.+Shiauthor=H.+Yueauthor=X.+Liuauthor=D.+Dobrovolskyauthor=B.+Jiangauthor=J.+Wangauthor=M.+Haoauthor=I.+Youauthor=M.+Tengauthor=Y.+Liangauthor=J.+Hatcherauthor=Z.+Liauthor=T.+D.+Manzauthor=B.+Groendykeauthor=W.+Huauthor=Y.+Namauthor=S.+Senguptaauthor=H.+Choauthor=I.+Shinauthor=M.+P.+Agiusauthor=I.+M.+Ghobrialauthor=M.+W.+Maauthor=J.+Cheauthor=S.+J.+Buhrlageauthor=T.+Simauthor=N.+S.+Grayauthor=E.+S.+Fischer&title=Mapping+the+Degradable+Kinome+Provides+a+Resource+for+Expedited+Degrader+Development&doi=10.1016%2Fj.cell.2020.10.038"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Mapping the Degradable Kinome Provides a Resource for Expedited Degrader Development</span></div><div class="casAuthors">Donovan, Katherine A.; Ferguson, Fleur M.; Bushman, Jonathan W.; Eleuteri, Nicholas A.; Bhunia, Debabrata; Ryu, SeongShick; Tan, Li; Shi, Kun; Yue, Hong; Liu, Xiaoxi; Dobrovolsky, Dennis; Jiang, Baishan; Wang, Jinhua; Hao, Mingfeng; You, Inchul; Teng, Mingxing; Liang, Yanke; Hatcher, John; Li, Zhengnian; Manz, Theresa D.; Groendyke, Brian; Hu, Wanyi; Nam, Yunju; Sengupta, Sandip; Cho, Hanna; Shin, Injae; Agius, Michael P.; Ghobrial, Irene M.; Ma, Michelle W.; Che, Jianwei; Buhrlage, Sara J.; Sim, Taebo; Gray, Nathanael S.; Fischer, Eric S.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">183</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1714-1731.e10</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Targeted protein degrdn. (TPD) refers to the use of small mols. to induce ubiquitin-dependent degrdn. of proteins.  TPD is of interest in drug development, as it can address previously inaccessible targets.  However, degrader discovery and optimization remains an inefficient process due to a lack of understanding of the relative importance of the key mol. events required to induce target degrdn.  Here, we use chemo-proteomics to annotate the degradable kinome.  Our expansive dataset provides chem. leads for ∼200 kinases and demonstrates that the current practice of starting from the highest potency binder is an ineffective method for discovering active compds.  We develop multitargeted degraders to answer fundamental questions about the ubiquitin proteasome system, uncovering that kinase degrdn. is p97 dependent.  This work will not only fuel kinase degrader discovery, but also provides a blueprint for evaluating targeted degrdn. across entire gene families to accelerate understanding of TPD beyond the kinome.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoCV_iCTOWvN7Vg90H21EOLACvtfcHk0ljZ_LYuYThV7Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXisV2nsbjL&md5=380302305a3749c52492ebd82bf1dabf</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2020.10.038&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2020.10.038%26sid%3Dliteratum%253Aachs%26aulast%3DDonovan%26aufirst%3DK.%2BA.%26aulast%3DFerguson%26aufirst%3DF.%2BM.%26aulast%3DBushman%26aufirst%3DJ.%2BW.%26aulast%3DEleuteri%26aufirst%3DN.%2BA.%26aulast%3DBhunia%26aufirst%3DD.%26aulast%3DRyu%26aufirst%3DS.%26aulast%3DTan%26aufirst%3DL.%26aulast%3DShi%26aufirst%3DK.%26aulast%3DYue%26aufirst%3DH.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DDobrovolsky%26aufirst%3DD.%26aulast%3DJiang%26aufirst%3DB.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DHao%26aufirst%3DM.%26aulast%3DYou%26aufirst%3DI.%26aulast%3DTeng%26aufirst%3DM.%26aulast%3DLiang%26aufirst%3DY.%26aulast%3DHatcher%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DZ.%26aulast%3DManz%26aufirst%3DT.%2BD.%26aulast%3DGroendyke%26aufirst%3DB.%26aulast%3DHu%26aufirst%3DW.%26aulast%3DNam%26aufirst%3DY.%26aulast%3DSengupta%26aufirst%3DS.%26aulast%3DCho%26aufirst%3DH.%26aulast%3DShin%26aufirst%3DI.%26aulast%3DAgius%26aufirst%3DM.%2BP.%26aulast%3DGhobrial%26aufirst%3DI.%2BM.%26aulast%3DMa%26aufirst%3DM.%2BW.%26aulast%3DChe%26aufirst%3DJ.%26aulast%3DBuhrlage%26aufirst%3DS.%2BJ.%26aulast%3DSim%26aufirst%3DT.%26aulast%3DGray%26aufirst%3DN.%2BS.%26aulast%3DFischer%26aufirst%3DE.%2BS.%26atitle%3DMapping%2520the%2520Degradable%2520Kinome%2520Provides%2520a%2520Resource%2520for%2520Expedited%2520Degrader%2520Development%26jtitle%3DCell%26date%3D2020%26volume%3D183%26spage%3D1714%26epage%3D1731%26doi%3D10.1016%2Fj.cell.2020.10.038" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article has not yet been cited by other publications.</p></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-13/acs.jmedchem.1c00373/20210629/images/medium/jm1c00373_0012.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-13/acs.jmedchem.1c00373/20210629/images/large/jm1c00373_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00373&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-13/acs.jmedchem.1c00373/20210629/images/medium/jm1c00373_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-13/acs.jmedchem.1c00373/20210629/images/large/jm1c00373_0002.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Crystal structure of ALK bound by Brigatinib (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6MX8">6MX8</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-13/acs.jmedchem.1c00373/20210629/images/large/jm1c00373_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00373&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-13/acs.jmedchem.1c00373/20210629/images/medium/jm1c00373_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-13/acs.jmedchem.1c00373/20210629/images/large/jm1c00373_0003.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Degradation of ALK of synthesized ALK degraders. (a) Western blots of Brigatinib CRBN degraders treated for 16 h. (b) Structure of compound <b>6</b> (SIAIS164018) and 018NC. (c) Western blots of ALK after treatment with 018NC. (d) Growth inhibition of Brigatinib analogue, SIAIS164018, and 018NC hours (error bars are shown).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-13/acs.jmedchem.1c00373/20210629/images/large/jm1c00373_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00373&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-13/acs.jmedchem.1c00373/20210629/images/medium/jm1c00373_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-13/acs.jmedchem.1c00373/20210629/images/large/jm1c00373_0009.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Syntheses of Compounds <b>1–12</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-13/acs.jmedchem.1c00373/20210629/images/large/jm1c00373_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00373&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-13/acs.jmedchem.1c00373/20210629/images/medium/jm1c00373_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-13/acs.jmedchem.1c00373/20210629/images/large/jm1c00373_0004.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Inhibition and degradation properties of SIAIS164018 in ALK-positive cell line. (a) Degradation of ALK at different times under 100 nM SIAIS164018. SR was pretreated under 2.5 μM MG (MG132) or 0.5 μM MLN (MLN4924) for 4 h; then, SR was treated for 16 h with compounds. DC<sub>50</sub> was the concentration at which degraders induced 50% protein degradation. (b) Western blots: when washing out 100 nM SIAIS164018 after a 24 h treatment, then after 24, 48, and 72 h, the ALK protein was detected in the SR cell line. (c) The SR cell was treated for 5 days at 10 nM SIAIS164018 or Brigatinib, and then the cell density was recorded after removal of SIAIS164018 at 7 days and 9 days (error bars are shown). (d) Downstream signaling pathways after treatment of SIAIS164018 in the SR cell line. Ratio of pALK or ALK was normalized to GAPDH and quantified using ImageJ. (e) Degradation of ALK(G1202R) at indicated concentrations and inhibition cell proliferation in 293T-ALK(G1202R) cell line after a 72 h treatment (error bars are shown). (f) Kinase IC<sub>50</sub> of ALK and ALK (G1202R).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-13/acs.jmedchem.1c00373/20210629/images/large/jm1c00373_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00373&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-13/acs.jmedchem.1c00373/20210629/images/medium/jm1c00373_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-13/acs.jmedchem.1c00373/20210629/images/large/jm1c00373_0005.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Cell cycle, migration, and invasion differences induced by SIAIS164018. (a) Cell cycle analysis after 24 h or 48 h of treatment with 100 nM SIAIS164018 or Brigatinib in SR, Calu-1, or MDA-MB-231 cell line (error bars are shown. ** <i>p</i> value < 0.01; *** <i>p</i> value < 0.001; **** <i>p</i> value < 0.0001, <i>T</i> test). (b) Wound healing assay in Calu-1 after 12 h or 24 h of treatment with 100 nM SIAIS164018 or Brigatinib (error bars are shown). (c) Transwell invasion assay in Calu-1 GFP after 24 h of treatment with 100 nM SIAIS164018 or Brigatinib (error bars are shown, <i>N</i> = 15). (d) Quantified wound confluence ratio in MDA-MB-231 during 48 h of treatment with SIAIS164018 or Brigatinib under indicated concentrations and time scale (error bars are shown).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-13/acs.jmedchem.1c00373/20210629/images/large/jm1c00373_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00373&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-13/acs.jmedchem.1c00373/20210629/images/medium/jm1c00373_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-13/acs.jmedchem.1c00373/20210629/images/large/jm1c00373_0006.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. FAK and PYK2 degradation identified from proteomics in SR and Calu-1 cancer cells. (a) TMT-labeled proteomics data in SR and Calu-1 cell lines after treatment with DMSO, 100 nM Brigatinib, or 100 nM SIAIS164018. 7207 proteins were identified in SR and 6313 proteins were identified in Calu-1. Red dots: <i>p</i> value < 0.05 and log<sub>2</sub> FC > 1; blue dots: <i>p</i> value < 0.05 and log<sub>2</sub> FC < 1. The <i>p</i> value was calculated using <i>t</i>-test. (b) Western blotting of PYK2, FER, and ALK in SR cell line after 24 h of treatment with different compounds. (c) Western blotting of GAK, RSK1, and FAK in NCI-H2228, A549, and Calu-1 cell lines after treatment for 16 h. (d) Western blotting: FAK and PYK2 were detected at different times under 10 nM compounds in MDA-MB-231. (e) FAK and PYK2 abundance was detected at 0 h, 24 h, and 48 h after removal of 10 nM compounds treating for 16 h in MDA-MB-231. (f) Under the treatment of 50 μg/mL CHX (cycloheximide), ALK was detected at different times in SR. FAK and PYK2 were detected at different times in MDA-MB-231.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-13/acs.jmedchem.1c00373/20210629/images/large/jm1c00373_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00373&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-13/acs.jmedchem.1c00373/20210629/images/medium/jm1c00373_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-13/acs.jmedchem.1c00373/20210629/images/large/jm1c00373_0007.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Transcriptional differences induced by SIAIS164018. (a) Overview of gene expression after 48 h of treatment with DMSO, 100 nM Brigatinib, and 100 nM SIAIS164018 in the Calu-1 cell line. (b) Enrichment of gene sets which were analyzed by GSEA. Transcriptions of cyclin D1 and MYC were shown (error bars are shown). (c) Western blotting of cyclin D1 and c-MYC at an indicated concentration after 48 h of treatment in Calu-1 and MDA-MB-231 cell lines.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-13/acs.jmedchem.1c00373/20210629/images/large/jm1c00373_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00373&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-13/acs.jmedchem.1c00373/20210629/images/medium/jm1c00373_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-13/acs.jmedchem.1c00373/20210629/images/large/jm1c00373_0008.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Comparison of kinome data with the degradation profile of SIAIS164018 and survival analysis of degraded targets. (a) Kinome of SIAIS164018 at 100 nM (left) and listed targets with high selectivity whose percent control was below 10 (right). (b) Comparison of selectivity between SIAIS164018 and Brigatinib (data of Brigatinib from the reported paper<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a>). Selected targets of SIAIS164018 were percent control below 50; targets of Brigatinib were selected with IC<sub>50</sub> below 100 nM. Among overlapped targets, selectivity difference was drawn using Chordal Graph. Difference targets were shown in the kinome map. Blue dots were targets from Brigatinib, while yellow dots were targets from SIAIS164018. A Venn diagram was plotted using online VENNY 2.1. Chordal Graph was generated using online tools (<a href="https://www.tubiaoxiu.com" class="extLink">https://www.tubiaoxiu.com</a>). Wider arc represented higher selectivity for compounds. (c) Proteomics after 16 h of treatment with 100 nM compounds in MDA-MB-231. 7355 proteins were identified in MDA-MD-231. Red dots: <i>p</i> value < 0.05 and log<sub>2</sub> FC > 1; blue dots: <i>p</i> value < 0.05 and log<sub>2</sub> FC < 1. <i>p</i> value was calculated using <i>T</i> test. (d) Comparing kinome and proteomic in SR, Calu-1, and MDA-MB-231. RSK1(N) was the N terminal kinase domain of RSK1 and RSK1(C) was the C terminal kinase domain of RSK1. Red dots: high selective and degraded; blue dots: high selective and not degraded; and gray dots: low selective and degraded. (e) Survival analysis plots generated from GEPIA (<a href="http://gepia.cancer-pku.cn/index.html" class="extLink">http://gepia.cancer-pku.cn/index.html</a>). LAML (Acute Myelo id Leukemia), BRCA (Breast Invasive Carcinoma), BLCA (Bladder Urothelial Carcinoma), and ACC (Adrenocortical Carcinoma).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-13/acs.jmedchem.1c00373/20210629/images/large/jm1c00373_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00373&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i53">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_42700" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_42700" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 50 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lai, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Induced protein degradation: an emerging drug discovery paradigm</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">101</span>– <span class="NLM_lpage">114</span>, <span class="refDoi"> DOI: 10.1038/nrd.2016.211</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=10.1038%2Fnrd.2016.211" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=27885283" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvFCit7rP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2017&pages=101-114&author=A.+C.+Laiauthor=C.+M.+Crews&title=Induced+protein+degradation%3A+an+emerging+drug+discovery+paradigm&doi=10.1038%2Fnrd.2016.211"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Induced protein degradation: an emerging drug discovery paradigm</span></div><div class="casAuthors">Lai, Ashton C.; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">101-114</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Small-mol. drug discovery has traditionally focused on occupancy of a binding site that directly affects protein function, and this approach typically precludes targeting proteins that lack such amenable sites.  Furthermore, high systemic drug exposures may be needed to maintain sufficient target inhibition in vivo, increasing the risk of undesirable off-target effects.  Induced protein degrdn. is an alternative approach that is event-driven: upon drug binding, the target protein is tagged for elimination.  Emerging technologies based on proteolysis-targeting chimaeras (PROTACs) that exploit cellular quality control machinery to selectively degrade target proteins are attracting considerable attention in the pharmaceutical industry owing to the advantages they could offer over traditional small-mol. strategies.  These advantages include the potential to reduce systemic drug exposure, the ability to counteract increased target protein expression that often accompanies inhibition of protein function and the potential ability to target proteins that are not currently therapeutically tractable, such as transcription factors, scaffolding and regulatory proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoj-O7zXGOpXLVg90H21EOLACvtfcHk0liXDE9DxArU-Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvFCit7rP&md5=a01ff43e9b696fe4dc03f0815158a98f</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2016.211&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2016.211%26sid%3Dliteratum%253Aachs%26aulast%3DLai%26aufirst%3DA.%2BC.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DInduced%2520protein%2520degradation%253A%2520an%2520emerging%2520drug%2520discovery%2520paradigm%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2017%26volume%3D16%26spage%3D101%26epage%3D114%26doi%3D10.1038%2Fnrd.2016.211" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Burslem, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Small-Molecule Modulation of Protein Homeostasis</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>117</i></span>,  <span class="NLM_fpage">11269</span>– <span class="NLM_lpage">11301</span>, <span class="refDoi"> DOI: 10.1021/acs.chemrev.7b00077</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.chemrev.7b00077" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=1%3ACAS%3A528%3ADC%252BC2sXht1Ohur3K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=117&publication_year=2017&pages=11269-11301&author=G.+M.+Burslemauthor=C.+M.+Crews&title=Small-Molecule+Modulation+of+Protein+Homeostasis&doi=10.1021%2Facs.chemrev.7b00077"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Small-Molecule Modulation of Protein Homeostasis</span></div><div class="casAuthors">Burslem, George M.; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">117</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">11269-11301</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Control of protein levels by nucleic-acid-based technologies has proven to be a useful research tool but lacks the advantages of small mols. with respect to cell permeability, temporal control, and the potential generation of therapeutics.  In this Review, we discuss the technologies available for the control of intracellular protein levels with small mols. and compare the various systems available.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpcARydbgPK9bVg90H21EOLACvtfcHk0liXDE9DxArU-Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXht1Ohur3K&md5=f9f8dd98e2c71c7166d2d500b514d578</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1021%2Facs.chemrev.7b00077&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.chemrev.7b00077%26sid%3Dliteratum%253Aachs%26aulast%3DBurslem%26aufirst%3DG.%2BM.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DSmall-Molecule%2520Modulation%2520of%2520Protein%2520Homeostasis%26jtitle%3DChem.%2520Rev.%26date%3D2017%26volume%3D117%26spage%3D11269%26epage%3D11301%26doi%3D10.1021%2Facs.chemrev.7b00077" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bondeson, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mares, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, I. E. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ko, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campos, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miah, A. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mulholland, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Routly, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buckley, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gustafson, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zinn, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grandi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shimamura, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bergamini, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Faelth-Savitski, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bantscheff, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cox, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Willard, R. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flanagan, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Casillas, L. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Votta, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">den Besten, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Famm, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kruidenier, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carter, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harling, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Churcher, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Catalytic in vivo protein knockdown by small-molecule PROTACs</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">611</span>– <span class="NLM_lpage">617</span>, <span class="refDoi"> DOI: 10.1038/nchembio.1858</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=10.1038%2Fnchembio.1858" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=26075522" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFeju7rP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2015&pages=611-617&author=D.+P.+Bondesonauthor=A.+Maresauthor=I.+E.+D.+Smithauthor=E.+Koauthor=S.+Camposauthor=A.+H.+Miahauthor=K.+E.+Mulhollandauthor=N.+Routlyauthor=D.+L.+Buckleyauthor=J.+L.+Gustafsonauthor=N.+Zinnauthor=P.+Grandiauthor=S.+Shimamuraauthor=G.+Bergaminiauthor=M.+Faelth-Savitskiauthor=M.+Bantscheffauthor=C.+Coxauthor=D.+A.+Gordonauthor=R.+R.+Willardauthor=J.+J.+Flanaganauthor=L.+N.+Casillasauthor=B.+J.+Vottaauthor=W.+den+Bestenauthor=K.+Fammauthor=L.+Kruidenierauthor=P.+S.+Carterauthor=J.+D.+Harlingauthor=I.+Churcherauthor=C.+M.+Crews&title=Catalytic+in+vivo+protein+knockdown+by+small-molecule+PROTACs&doi=10.1038%2Fnchembio.1858"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Catalytic in vivo protein knockdown by small-molecule PROTACs</span></div><div class="casAuthors">Bondeson, Daniel P.; Mares, Alina; Smith, Ian E. D.; Ko, Eunhwa; Campos, Sebastien; Miah, Afjal H.; Mulholland, Katie E.; Routly, Natasha; Buckley, Dennis L.; Gustafson, Jeffrey L.; Zinn, Nico; Grandi, Paola; Shimamura, Satoko; Bergamini, Giovanna; Faelth-Savitski, Maria; Bantscheff, Marcus; Cox, Carly; Gordon, Deborah A.; Willard, Ryan R.; Flanagan, John J.; Casillas, Linda N.; Votta, Bartholomew J.; den Besten, Willem; Famm, Kristoffer; Kruidenier, Laurens; Carter, Paul S.; Harling, John D.; Churcher, Ian; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">611-617</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The current predominant therapeutic paradigm is based on maximizing drug-receptor occupancy to achieve clin. benefit.  This strategy, however, generally requires excessive drug concns. to ensure sufficient occupancy, often leading to adverse side effects.  Here, we describe major improvements to the proteolysis targeting chimeras (PROTACs) method, a chem. knockdown strategy in which a heterobifunctional mol. recruits a specific protein target to an E3 ubiquitin ligase, resulting in the target's ubiquitination and degrdn.  These compds. behave catalytically in their ability to induce the ubiquitination of super-stoichiometric quantities of proteins, providing efficacy that is not limited by equil. occupancy.  We present two PROTACs that are capable of specifically reducing protein levels by >90% at nanomolar concns.  In addn., mouse studies indicate that they provide broad tissue distribution and knockdown of the targeted protein in tumor xenografts.  Together, these data demonstrate a protein knockdown system combining many of the favorable properties of small-mol. agents with the potent protein knockdown of RNAi and CRISPR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGonhnajD3CxS7Vg90H21EOLACvtfcHk0liXDE9DxArU-Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtFeju7rP&md5=31eb8756c5661f78ab4527a50dc1e251</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1858&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1858%26sid%3Dliteratum%253Aachs%26aulast%3DBondeson%26aufirst%3DD.%2BP.%26aulast%3DMares%26aufirst%3DA.%26aulast%3DSmith%26aufirst%3DI.%2BE.%2BD.%26aulast%3DKo%26aufirst%3DE.%26aulast%3DCampos%26aufirst%3DS.%26aulast%3DMiah%26aufirst%3DA.%2BH.%26aulast%3DMulholland%26aufirst%3DK.%2BE.%26aulast%3DRoutly%26aufirst%3DN.%26aulast%3DBuckley%26aufirst%3DD.%2BL.%26aulast%3DGustafson%26aufirst%3DJ.%2BL.%26aulast%3DZinn%26aufirst%3DN.%26aulast%3DGrandi%26aufirst%3DP.%26aulast%3DShimamura%26aufirst%3DS.%26aulast%3DBergamini%26aufirst%3DG.%26aulast%3DFaelth-Savitski%26aufirst%3DM.%26aulast%3DBantscheff%26aufirst%3DM.%26aulast%3DCox%26aufirst%3DC.%26aulast%3DGordon%26aufirst%3DD.%2BA.%26aulast%3DWillard%26aufirst%3DR.%2BR.%26aulast%3DFlanagan%26aufirst%3DJ.%2BJ.%26aulast%3DCasillas%26aufirst%3DL.%2BN.%26aulast%3DVotta%26aufirst%3DB.%2BJ.%26aulast%3Dden%2BBesten%26aufirst%3DW.%26aulast%3DFamm%26aufirst%3DK.%26aulast%3DKruidenier%26aufirst%3DL.%26aulast%3DCarter%26aufirst%3DP.%2BS.%26aulast%3DHarling%26aufirst%3DJ.%2BD.%26aulast%3DChurcher%26aufirst%3DI.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DCatalytic%2520in%2520vivo%2520protein%2520knockdown%2520by%2520small-molecule%2520PROTACs%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2015%26volume%3D11%26spage%3D611%26epage%3D617%26doi%3D10.1038%2Fnchembio.1858" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sun, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hwang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rao, Y.</span></span> <span> </span><span class="NLM_article-title">PROTAC-induced BTK degradation as a novel therapy for mutated BTK C481S induced ibrutinib-resistant B-cell malignancies</span>. <i>Cell Res.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">779</span>– <span class="NLM_lpage">781</span>, <span class="refDoi"> DOI: 10.1038/s41422-018-0055-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=10.1038%2Fs41422-018-0055-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=29875397" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=1%3ACAS%3A528%3ADC%252BC1cXht1ektbzO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2018&pages=779-781&author=Y.+Sunauthor=X.+Zhaoauthor=N.+Dingauthor=H.+Gaoauthor=Y.+Wuauthor=Y.+Yangauthor=M.+Zhaoauthor=J.+Hwangauthor=Y.+Songauthor=W.+Liuauthor=Y.+Rao&title=PROTAC-induced+BTK+degradation+as+a+novel+therapy+for+mutated+BTK+C481S+induced+ibrutinib-resistant+B-cell+malignancies&doi=10.1038%2Fs41422-018-0055-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">PROTAC-induced BTK degradation as a novel therapy for mutated BTK C481S induced ibrutinib-resistant B-cell malignancies</span></div><div class="casAuthors">Sun, Yonghui; Zhao, Xingwang; Ding, Ning; Gao, Hongying; Wu, Yue; Yang, Yiqing; Zhao, Meng; Hwang, Jinseok; Song, Yuqin; Liu, Wanli; Rao, Yu</div><div class="citationInfo"><span class="NLM_cas:title">Cell Research</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">779-781</span>CODEN:
                <span class="NLM_cas:coden">CREEB6</span>;
        ISSN:<span class="NLM_cas:issn">1001-0602</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">In this study, for the first time, we report the development of BTK-targeting degraders using the PROTAC strategy.  These PROTACs could ef fi ciently degrade ibrutinib-sensitive BTK-WT (wild type).  More importantly, our newly designed PROTACs also significantly induced the degrdn. of ibrutinib resistant BTK-C481S (50% degrdn. efficiency at 30 nM).  Furthermore, our PROTAC mols. ef fi ciently inhibited cell proliferation and colony formation, while exhibited no obvious inhibition (>1000 nM) of ITK, EGFR, and TEC, which are major off-targets of ibrutinib.  These data demonstrate the strong potential for developing PROTAC-based therapeutic mols.  The results indicate that the PROTAC strategy (protein degrdn. instead of inhibition) could be adopted as a general and powerful therapeutic treatment for drug-resistant cancers in the future.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqg_FLdaUP_HLVg90H21EOLACvtfcHk0ljlNKDEtal1qQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXht1ektbzO&md5=1cca51ac76ecb3b491da0117b21271fb</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1038%2Fs41422-018-0055-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41422-018-0055-1%26sid%3Dliteratum%253Aachs%26aulast%3DSun%26aufirst%3DY.%26aulast%3DZhao%26aufirst%3DX.%26aulast%3DDing%26aufirst%3DN.%26aulast%3DGao%26aufirst%3DH.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DYang%26aufirst%3DY.%26aulast%3DZhao%26aufirst%3DM.%26aulast%3DHwang%26aufirst%3DJ.%26aulast%3DSong%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DW.%26aulast%3DRao%26aufirst%3DY.%26atitle%3DPROTAC-induced%2520BTK%2520degradation%2520as%2520a%2520novel%2520therapy%2520for%2520mutated%2520BTK%2520C481S%2520induced%2520ibrutinib-resistant%2520B-cell%2520malignancies%26jtitle%3DCell%2520Res.%26date%3D2018%26volume%3D28%26spage%3D779%26epage%3D781%26doi%3D10.1038%2Fs41422-018-0055-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bai, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chinnaswamy, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McEachern, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wen, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumar, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meagher, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stuckey, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span> <span> </span><span class="NLM_article-title">A Potent and Selective Small-Molecule Degrader of STAT3 Achieves Complete Tumor Regression In Vivo</span>. <i>Cancer Cell</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>36</i></span>,  <span class="NLM_fpage">498</span>– <span class="NLM_lpage">511</span>, <span class="refDoi"> DOI: 10.1016/j.ccell.2019.10.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=10.1016%2Fj.ccell.2019.10.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=31715132" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitFCjtrjO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2019&pages=498-511&author=L.+Baiauthor=H.+Zhouauthor=R.+Xuauthor=Y.+Zhaoauthor=K.+Chinnaswamyauthor=D.+McEachernauthor=J.+Chenauthor=C.-Y.+Yangauthor=Z.+Liuauthor=M.+Wangauthor=L.+Liuauthor=H.+Jiangauthor=B.+Wenauthor=P.+Kumarauthor=J.+L.+Meagherauthor=D.+Sunauthor=J.+A.+Stuckeyauthor=S.+Wang&title=A+Potent+and+Selective+Small-Molecule+Degrader+of+STAT3+Achieves+Complete+Tumor+Regression+In+Vivo&doi=10.1016%2Fj.ccell.2019.10.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">A Potent and Selective Small-Molecule Degrader of STAT3 Achieves Complete Tumor Regression In Vivo</span></div><div class="casAuthors">Bai, Longchuan; Zhou, Haibin; Xu, Renqi; Zhao, Yujun; Chinnaswamy, Krishnapriya; McEachern, Donna; Chen, Jianyong; Yang, Chao-Yie; Liu, Zhaomin; Wang, Mi; Liu, Liu; Jiang, Hui; Wen, Bo; Kumar, Praveen; Meagher, Jennifer L.; Sun, Duxin; Stuckey, Jeanne A.; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">498-511.e17</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Signal transducer and activator of transcription 3 (STAT3) is an attractive cancer therapeutic target.  Here we report the discovery of SD-36(I), a small-mol. degrader of STAT3.  SD-36 potently induces the degrdn. of STAT3 protein in vitro and in vivo and demonstrates high selectivity over other STAT members.  Induced degrdn. of STAT3 results in a strong suppression of its transcription network in leukemia and lymphoma cells.  SD-36 inhibits the growth of a subset of acute myeloid leukemia and anaplastic large-cell lymphoma cell lines by inducing cell-cycle arrest and/or apoptosis.  SD-36 achieves complete and long-lasting tumor regression in multiple xenograft mouse models at well-tolerated dose schedules.  Degrdn. of STAT3 protein, therefore, is a promising cancer therapeutic strategy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoUaP49Ap9DxLVg90H21EOLACvtfcHk0ljlNKDEtal1qQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitFCjtrjO&md5=71252c17d50a1a8be829352942175bd6</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1016%2Fj.ccell.2019.10.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccell.2019.10.002%26sid%3Dliteratum%253Aachs%26aulast%3DBai%26aufirst%3DL.%26aulast%3DZhou%26aufirst%3DH.%26aulast%3DXu%26aufirst%3DR.%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DChinnaswamy%26aufirst%3DK.%26aulast%3DMcEachern%26aufirst%3DD.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DYang%26aufirst%3DC.-Y.%26aulast%3DLiu%26aufirst%3DZ.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DWen%26aufirst%3DB.%26aulast%3DKumar%26aufirst%3DP.%26aulast%3DMeagher%26aufirst%3DJ.%2BL.%26aulast%3DSun%26aufirst%3DD.%26aulast%3DStuckey%26aufirst%3DJ.%2BA.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DA%2520Potent%2520and%2520Selective%2520Small-Molecule%2520Degrader%2520of%2520STAT3%2520Achieves%2520Complete%2520Tumor%2520Regression%2520In%2520Vivo%26jtitle%3DCancer%2520Cell%26date%3D2019%26volume%3D36%26spage%3D498%26epage%3D511%26doi%3D10.1016%2Fj.ccell.2019.10.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lai, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toure, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hellerschmied, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salami, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jaime-Figueroa, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ko, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hines, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Modular PROTAC Design for the Degradation of Oncogenic BCR-ABL</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">807</span>– <span class="NLM_lpage">810</span>, <span class="refDoi"> DOI: 10.1002/anie.201507634</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=10.1002%2Fanie.201507634" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvVOlt7rN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2016&pages=807-810&author=A.+C.+Laiauthor=M.+Toureauthor=D.+Hellerschmiedauthor=J.+Salamiauthor=S.+Jaime-Figueroaauthor=E.+Koauthor=J.+Hinesauthor=C.+M.+Crews&title=Modular+PROTAC+Design+for+the+Degradation+of+Oncogenic+BCR-ABL&doi=10.1002%2Fanie.201507634"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Modular PROTAC Design for the Degradation of Oncogenic BCR-ABL</span></div><div class="casAuthors">Lai, Ashton C.; Toure, Momar; Hellerschmied, Doris; Salami, Jemilat; Jaime-Figueroa, Saul; Ko, Eunhwa; Hines, John; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">807-810</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Proteolysis Targeting Chimera (PROTAC) technol. is a rapidly emerging alternative therapeutic strategy with the potential to address many of the challenges currently faced in modern drug development programs.  PROTAC technol. employs small mols. that recruit target proteins for ubiquitination and removal by the proteasome.  The synthesis of PROTAC compds. that mediate the degrdn. of c-ABL and BCR-ABL by recruiting either Cereblon or Von Hippel Lindau E3 ligases is reported.  During the course of their development, the authors discovered that the capacity of a PROTAC to induce degrdn. involves more than just target binding: the identity of the inhibitor warhead and the recruited E3 ligase largely det. the degrdn. profiles of the compds.; thus, as a starting point for PROTAC development, both the target ligand and the recruited E3 ligase should be varied to rapidly generate a PROTAC with the desired degrdn. profile.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqdcSTHt5WfILVg90H21EOLACvtfcHk0ljlNKDEtal1qQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvVOlt7rN&md5=58a5d78f32bc8a21fd1a6e0c38b8979c</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1002%2Fanie.201507634&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201507634%26sid%3Dliteratum%253Aachs%26aulast%3DLai%26aufirst%3DA.%2BC.%26aulast%3DToure%26aufirst%3DM.%26aulast%3DHellerschmied%26aufirst%3DD.%26aulast%3DSalami%26aufirst%3DJ.%26aulast%3DJaime-Figueroa%26aufirst%3DS.%26aulast%3DKo%26aufirst%3DE.%26aulast%3DHines%26aufirst%3DJ.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DModular%2520PROTAC%2520Design%2520for%2520the%2520Degradation%2520of%2520Oncogenic%2520BCR-ABL%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2016%26volume%3D55%26spage%3D807%26epage%3D810%26doi%3D10.1002%2Fanie.201507634" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kang, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, C. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Du Ha, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hwang, J. Y.</span></span> <span> </span><span class="NLM_article-title">Induced protein degradation of anaplastic lymphoma kinase (ALK) by proteolysis targeting chimera (PROTAC)</span>. <i>Biochem. Biophys. Res. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>505</i></span>,  <span class="NLM_fpage">542</span>– <span class="NLM_lpage">547</span>, <span class="refDoi"> DOI: 10.1016/j.bbrc.2018.09.169</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=10.1016%2Fj.bbrc.2018.09.169" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=30274779" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhvVejs7fE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=505&publication_year=2018&pages=542-547&author=C.+H.+Kangauthor=D.+H.+Leeauthor=C.+O.+Leeauthor=J.+Du+Haauthor=C.+H.+Parkauthor=J.+Y.+Hwang&title=Induced+protein+degradation+of+anaplastic+lymphoma+kinase+%28ALK%29+by+proteolysis+targeting+chimera+%28PROTAC%29&doi=10.1016%2Fj.bbrc.2018.09.169"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Induced protein degradation of anaplastic lymphoma kinase (ALK) by proteolysis targeting chimera (PROTAC)</span></div><div class="casAuthors">Kang, Chung Hyo; Lee, Dong Ho; Lee, Chong Ock; Du Ha, Jae; Park, Chi Hoon; Hwang, Jong Yeon</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical and Biophysical Research Communications</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">505</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">542-547</span>CODEN:
                <span class="NLM_cas:coden">BBRCA9</span>;
        ISSN:<span class="NLM_cas:issn">0006-291X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Recently, proteolysis targeting chimera (PROTAC) technol. is highlighted in drug discovery area as a new therapeutic approach.  PROTAC as a heterobifunctional mol. is comprised of two ligands, which recruit target protein and E3 ligase, resp.  To degrade the anaplastic lymphoma kinase (ALK) fusion protein, such as NPM-ALK or EML4-ALK, we generated several ALK-PROTAC mols. consisted of ceritinib, one of the ALK inhibitors, and ligand of von Hippel-Lindau (VHL) E3 ligase.  Among these mols., TD-004 effectively induced ALK degrdn. and inhibited the growth of ALK fusion pos. cell lines, SU-DHL-1 and H3122.  We also confirmed that TD-004 significantly reduced the tumor growth in H3122 xenograft model.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq8JUOZ5fYNW7Vg90H21EOLACvtfcHk0liEwALcFh9I9Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhvVejs7fE&md5=cac9252fca98ebec4ad8e5aeed7b7550</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1016%2Fj.bbrc.2018.09.169&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbrc.2018.09.169%26sid%3Dliteratum%253Aachs%26aulast%3DKang%26aufirst%3DC.%2BH.%26aulast%3DLee%26aufirst%3DD.%2BH.%26aulast%3DLee%26aufirst%3DC.%2BO.%26aulast%3DDu%2BHa%26aufirst%3DJ.%26aulast%3DPark%26aufirst%3DC.%2BH.%26aulast%3DHwang%26aufirst%3DJ.%2BY.%26atitle%3DInduced%2520protein%2520degradation%2520of%2520anaplastic%2520lymphoma%2520kinase%2520%2528ALK%2529%2520by%2520proteolysis%2520targeting%2520chimera%2520%2528PROTAC%2529%26jtitle%3DBiochem.%2520Biophys.%2520Res.%2520Commun.%26date%3D2018%26volume%3D505%26spage%3D542%26epage%3D547%26doi%3D10.1016%2Fj.bbrc.2018.09.169" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Burslem, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hines, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Enhancing Antiproliferative Activity and Selectivity of a FLT-3 Inhibitor by Proteolysis Targeting Chimera Conversion</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>140</i></span>,  <span class="NLM_fpage">16428</span>– <span class="NLM_lpage">16432</span>, <span class="refDoi"> DOI: 10.1021/jacs.8b10320</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.8b10320" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitF2qtr7P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=140&publication_year=2018&pages=16428-16432&author=G.+M.+Burslemauthor=J.+Songauthor=X.+Chenauthor=J.+Hinesauthor=C.+M.+Crews&title=Enhancing+Antiproliferative+Activity+and+Selectivity+of+a+FLT-3+Inhibitor+by+Proteolysis+Targeting+Chimera+Conversion&doi=10.1021%2Fjacs.8b10320"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Enhancing Antiproliferative Activity and Selectivity of a FLT-3 Inhibitor by Proteolysis Targeting Chimera Conversion</span></div><div class="casAuthors">Burslem, George M.; Song, Jayoung; Chen, Xin; Hines, John; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">140</span>
        (<span class="NLM_cas:issue">48</span>),
    <span class="NLM_cas:pages">16428-16432</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The receptor tyrosine kinase FLT-3 is frequently mutated in acute myeloid leukemia; however, current small mol. inhibitors suffer from limited efficacy in the clinic.  Conversion of a FLT-3 inhibitor (quizartinib) into a proteolysis targeting chimera (PROTAC) results in a compd. that induces degrdn. of FLT-3 ITD mutant at low nanomolar concns.  Furthermore, the PROTAC is capable of inhibiting cell growth more potently than the warhead alone while inhibiting fewer off-target kinases.  This enhanced antiproliferative activity occurs, despite a slight redn. in the PROTAC's kinase inhibitory activity, via an increased level of apoptosis induction suggesting nonkinase roles for the FLT-3 ITD protein.  Addnl., the PROTAC is capable of inducing FLT-3 ITD degrdn. in vivo.  These results suggest that degrdn. of FLT-3 ITD may provide a useful method for therapeutic intervention.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpBxw_nAEQydLVg90H21EOLACvtfcHk0liEwALcFh9I9Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitF2qtr7P&md5=52529d0c3f49052bd2a66c893e98e694</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1021%2Fjacs.8b10320&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.8b10320%26sid%3Dliteratum%253Aachs%26aulast%3DBurslem%26aufirst%3DG.%2BM.%26aulast%3DSong%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DHines%26aufirst%3DJ.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DEnhancing%2520Antiproliferative%2520Activity%2520and%2520Selectivity%2520of%2520a%2520FLT-3%2520Inhibitor%2520by%2520Proteolysis%2520Targeting%2520Chimera%2520Conversion%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2018%26volume%3D140%26spage%3D16428%26epage%3D16432%26doi%3D10.1021%2Fjacs.8b10320" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cromm, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Samarasinghe, K. T. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hines, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Addressing Kinase-Independent Functions of Fak via PROTAC-Mediated Degradation</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>140</i></span>,  <span class="NLM_fpage">17019</span>– <span class="NLM_lpage">17026</span>, <span class="refDoi"> DOI: 10.1021/jacs.8b08008</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.8b08008" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=1%3ACAS%3A528%3ADC%252BC1cXit1CitLnF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=140&publication_year=2018&pages=17019-17026&author=P.+M.+Crommauthor=K.+T.+G.+Samarasingheauthor=J.+Hinesauthor=C.+M.+Crews&title=Addressing+Kinase-Independent+Functions+of+Fak+via+PROTAC-Mediated+Degradation&doi=10.1021%2Fjacs.8b08008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Addressing Kinase-Independent Functions of Fak via PROTAC-Mediated Degradation</span></div><div class="casAuthors">Cromm, Philipp M.; Samarasinghe, Kusal T. G.; Hines, John; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">140</span>
        (<span class="NLM_cas:issue">49</span>),
    <span class="NLM_cas:pages">17019-17026</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Enzymic inhibition proved to be a successful modality for the development of many small-mol. drugs.  In recent years, small-mol.-induced protein degrdn. has emerged as an orthogonal therapeutic strategy that has the potential to expand the druggable target space.  Focal adhesion kinase (Fak) is a key player in tumor invasion and metastasis, acting simultaneously as a kinase and a scaffold for several signaling proteins.  While previous efforts to modulate Fak activity were limited to kinase inhibitors with low success in clin. studies, protein degrdn. offers a possibility to simultaneously block Fak's kinase signaling and scaffolding capabilities.  Here, the authors report the development of a selective and potent Fak degrader, PROTAC-3, which outperforms a clin. candidate, defactinib, with respect to Fak activation as well as Fak-mediated cell migration and invasion.  These results underline the potential that PROTACs offer in expanding the druggable space and controlling protein functions that are not easily addressed by traditional small-mol. therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqTn8r4NmDDObVg90H21EOLACvtfcHk0liEwALcFh9I9Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXit1CitLnF&md5=b97b1a646a8f9dbf7a2bed640d0f001d</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1021%2Fjacs.8b08008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.8b08008%26sid%3Dliteratum%253Aachs%26aulast%3DCromm%26aufirst%3DP.%2BM.%26aulast%3DSamarasinghe%26aufirst%3DK.%2BT.%2BG.%26aulast%3DHines%26aufirst%3DJ.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DAddressing%2520Kinase-Independent%2520Functions%2520of%2520Fak%2520via%2520PROTAC-Mediated%2520Degradation%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2018%26volume%3D140%26spage%3D17019%26epage%3D17026%26doi%3D10.1021%2Fjacs.8b08008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Powell, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Donovan, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nowak, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loehr, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bahcall, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jänne, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">George, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span> <span> </span><span class="NLM_article-title">Chemically Induced Degradation of Anaplastic Lymphoma Kinase (ALK)</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">4249</span>– <span class="NLM_lpage">4255</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b01655</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b01655" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=1%3ACAS%3A528%3ADC%252BC1cXns1OitrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=4249-4255&author=C.+E.+Powellauthor=Y.+Gaoauthor=L.+Tanauthor=K.+A.+Donovanauthor=R.+P.+Nowakauthor=A.+Loehrauthor=M.+Bahcallauthor=E.+S.+Fischerauthor=P.+A.+J%C3%A4nneauthor=R.+E.+Georgeauthor=N.+S.+Gray&title=Chemically+Induced+Degradation+of+Anaplastic+Lymphoma+Kinase+%28ALK%29&doi=10.1021%2Facs.jmedchem.7b01655"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Chemically Induced Degradation of Anaplastic Lymphoma Kinase (ALK)</span></div><div class="casAuthors">Powell, Chelsea E.; Gao, Yang; Tan, Li; Donovan, Katherine A.; Nowak, Radoslaw P.; Loehr, Amanda; Bahcall, Magda; Fischer, Eric S.; Janne, Pasi A.; George, Rani E.; Gray, Nathanael S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">4249-4255</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We present the development of the first small mol. degraders that can induce anaplastic lymphoma kinase (ALK) degrdn., including in non-small-cell lung cancer (NSCLC), anaplastic large-cell lymphoma (ALCL), and neuroblastoma (NB) cell lines.  These degraders were developed through conjugation of known pyrimidine-based ALK inhibitors, TAE684 or LDK378, and the cereblon ligand pomalidomide.  We demonstrate that in some cell types degrader potency is compromised by expression of drug transporter ABCB1.  In addn., proteomic profiling demonstrated that these compds. also promote the degrdn. of addnl. kinases including PTK2 (FAK), Aurora A, FER, and RPS6KA1 (RSK1).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp7mDH7Sj8EqbVg90H21EOLACvtfcHk0lhQO-rMFQccQQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXns1OitrY%253D&md5=486db11b3fbc2283700f38b09cc39bd7</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b01655&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b01655%26sid%3Dliteratum%253Aachs%26aulast%3DPowell%26aufirst%3DC.%2BE.%26aulast%3DGao%26aufirst%3DY.%26aulast%3DTan%26aufirst%3DL.%26aulast%3DDonovan%26aufirst%3DK.%2BA.%26aulast%3DNowak%26aufirst%3DR.%2BP.%26aulast%3DLoehr%26aufirst%3DA.%26aulast%3DBahcall%26aufirst%3DM.%26aulast%3DFischer%26aufirst%3DE.%2BS.%26aulast%3DJ%25C3%25A4nne%26aufirst%3DP.%2BA.%26aulast%3DGeorge%26aufirst%3DR.%2BE.%26aulast%3DGray%26aufirst%3DN.%2BS.%26atitle%3DChemically%2520Induced%2520Degradation%2520of%2520Anaplastic%2520Lymphoma%2520Kinase%2520%2528ALK%2529%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D4249%26epage%3D4255%26doi%3D10.1021%2Facs.jmedchem.7b01655" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brand, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bauer, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Donovan, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nowak, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kwiatkowski, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winter, G. E.</span></span> <span> </span><span class="NLM_article-title">Homolog-Selective Degradation as a Strategy to Probe the Function of CDK6 in AML</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">300</span>– <span class="NLM_lpage">306</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2018.11.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=10.1016%2Fj.chembiol.2018.11.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=30595531" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=1%3ACAS%3A528%3ADC%252BC1MXnsVSj" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2019&pages=300-306&author=M.+Brandauthor=B.+Jiangauthor=S.+Bauerauthor=K.+A.+Donovanauthor=Y.+Liangauthor=E.+S.+Wangauthor=R.+P.+Nowakauthor=J.+C.+Yuanauthor=T.+Zhangauthor=N.+Kwiatkowskiauthor=A.+C.+M%C3%BCllerauthor=E.+S.+Fischerauthor=N.+S.+Grayauthor=G.+E.+Winter&title=Homolog-Selective+Degradation+as+a+Strategy+to+Probe+the+Function+of+CDK6+in+AML&doi=10.1016%2Fj.chembiol.2018.11.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Homolog-Selective Degradation as a Strategy to Probe the Function of CDK6 in AML</span></div><div class="casAuthors">Brand, Matthias; Jiang, Baishan; Bauer, Sophie; Donovan, Katherine A.; Liang, Yanke; Wang, Eric S.; Nowak, Radoslaw P.; Yuan, Jingting C.; Zhang, Tinghu; Kwiatkowski, Nicholas; Mueller, Andre C.; Fischer, Eric S.; Gray, Nathanael S.; Winter, Georg E.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">300-306.e9</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The design of selective small mols. is often stymied by similar ligand binding pockets.  Here, we report BSJ-03-123, a phthalimide-based degrader that exploits protein-interface determinants to achieve proteome-wide selectivity for the degrdn. of cyclin-dependent kinase 6 (CDK6).  Pharmacol. CDK6 degrdn. targets a selective dependency of acute myeloid leukemia cells, and transcriptomics and phosphoproteomics profiling of acute degrdn. of CDK6 enabled dynamic mapping of its immediate role in coordinating signaling and transcription.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGognoz8FFoR9LVg90H21EOLACvtfcHk0lhQO-rMFQccQQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXnsVSj&md5=f8c1c3520c6c0218763d8769c59b4342</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2018.11.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2018.11.006%26sid%3Dliteratum%253Aachs%26aulast%3DBrand%26aufirst%3DM.%26aulast%3DJiang%26aufirst%3DB.%26aulast%3DBauer%26aufirst%3DS.%26aulast%3DDonovan%26aufirst%3DK.%2BA.%26aulast%3DLiang%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DE.%2BS.%26aulast%3DNowak%26aufirst%3DR.%2BP.%26aulast%3DYuan%26aufirst%3DJ.%2BC.%26aulast%3DZhang%26aufirst%3DT.%26aulast%3DKwiatkowski%26aufirst%3DN.%26aulast%3DM%25C3%25BCller%26aufirst%3DA.%2BC.%26aulast%3DFischer%26aufirst%3DE.%2BS.%26aulast%3DGray%26aufirst%3DN.%2BS.%26aulast%3DWinter%26aufirst%3DG.%2BE.%26atitle%3DHomolog-Selective%2520Degradation%2520as%2520a%2520Strategy%2520to%2520Probe%2520the%2520Function%2520of%2520CDK6%2520in%2520AML%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2019%26volume%3D26%26spage%3D300%26epage%3D306%26doi%3D10.1016%2Fj.chembiol.2018.11.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Khan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lv, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thummuri, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wiegand, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pei, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharma, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCurdy, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuruvilla, V. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baran, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferrando, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, Y.-m.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rogojina, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Houghton, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hromas, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Konopleva, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, D.</span></span> <span> </span><span class="NLM_article-title">A selective BCL-XL PROTAC degrader achieves safe and potent antitumor activity</span>. <i>Nat. Med.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">1938</span>– <span class="NLM_lpage">1947</span>, <span class="refDoi"> DOI: 10.1038/s41591-019-0668-z</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=10.1038%2Fs41591-019-0668-z" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=31792461" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=1%3ACAS%3A528%3ADC%252BC1MXit1yjsrrO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2019&pages=1938-1947&author=S.+Khanauthor=X.+Zhangauthor=D.+Lvauthor=Q.+Zhangauthor=Y.+Heauthor=P.+Zhangauthor=X.+Liuauthor=D.+Thummuriauthor=Y.+Yuanauthor=J.+S.+Wiegandauthor=J.+Peiauthor=W.+Zhangauthor=A.+Sharmaauthor=C.+R.+McCurdyauthor=V.+M.+Kuruvillaauthor=N.+Baranauthor=A.+A.+Ferrandoauthor=Y.-m.+Kimauthor=A.+Rogojinaauthor=P.+J.+Houghtonauthor=G.+Huangauthor=R.+Hromasauthor=M.+Konoplevaauthor=G.+Zhengauthor=D.+Zhou&title=A+selective+BCL-XL+PROTAC+degrader+achieves+safe+and+potent+antitumor+activity&doi=10.1038%2Fs41591-019-0668-z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">A selective BCL-XL PROTAC degrader achieves safe and potent antitumor activity</span></div><div class="casAuthors">Khan, Sajid; Zhang, Xuan; Lv, Dongwen; Zhang, Qi; He, Yonghan; Zhang, Peiyi; Liu, Xingui; Thummuri, Dinesh; Yuan, Yaxia; Wiegand, Janet S.; Pei, Jing; Zhang, Weizhou; Sharma, Abhisheak; McCurdy, Christopher R.; Kuruvilla, Vinitha M.; Baran, Natalia; Ferrando, Adolfo A.; Kim, Yong-mi; Rogojina, Anna; Houghton, Peter J.; Huang, Guangcun; Hromas, Robert; Konopleva, Marina; Zheng, Guangrong; Zhou, Daohong</div><div class="citationInfo"><span class="NLM_cas:title">Nature Medicine (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1938-1947</span>CODEN:
                <span class="NLM_cas:coden">NAMEFI</span>;
        ISSN:<span class="NLM_cas:issn">1078-8956</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">B-cell lymphoma extra large (BCL-XL) is a well-validated cancer target.  However, the on-target and dose-limiting thrombocytopenia limits the use of BCL-XL inhibitors, such as ABT263, as safe and effective anticancer agents.  To reduce the toxicity of ABT263, we converted it into DT2216, a BCL-XL proteolysis-targeting chimera (PROTAC), that targets BCL-XL to the Von Hippel-Lindau (VHL) E3 ligase for degrdn.  We found that DT2216 was more potent against various BCL-XL-dependent leukemia and cancer cells but considerably less toxic to platelets than ABT263 in vitro because VHL is poorly expressed in platelets.  In vivo, DT2216 effectively inhibits the growth of several xenograft tumors as a single agent or in combination with other chemotherapeutic agents, without causing appreciable thrombocytopenia.  These findings demonstrate the potential to use PROTAC technol. to reduce on-target drug toxicities and rescue the therapeutic potential of previously undruggable targets.  Furthermore, DT2216 may be developed as a safe first-in-class anticancer agent targeting BCL-XL.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGposmmgxVNCRLVg90H21EOLACvtfcHk0lhQO-rMFQccQQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXit1yjsrrO&md5=b07678c880569acac428ec479f002974</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1038%2Fs41591-019-0668-z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41591-019-0668-z%26sid%3Dliteratum%253Aachs%26aulast%3DKhan%26aufirst%3DS.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DLv%26aufirst%3DD.%26aulast%3DZhang%26aufirst%3DQ.%26aulast%3DHe%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DP.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DThummuri%26aufirst%3DD.%26aulast%3DYuan%26aufirst%3DY.%26aulast%3DWiegand%26aufirst%3DJ.%2BS.%26aulast%3DPei%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DW.%26aulast%3DSharma%26aufirst%3DA.%26aulast%3DMcCurdy%26aufirst%3DC.%2BR.%26aulast%3DKuruvilla%26aufirst%3DV.%2BM.%26aulast%3DBaran%26aufirst%3DN.%26aulast%3DFerrando%26aufirst%3DA.%2BA.%26aulast%3DKim%26aufirst%3DY.-m.%26aulast%3DRogojina%26aufirst%3DA.%26aulast%3DHoughton%26aufirst%3DP.%2BJ.%26aulast%3DHuang%26aufirst%3DG.%26aulast%3DHromas%26aufirst%3DR.%26aulast%3DKonopleva%26aufirst%3DM.%26aulast%3DZheng%26aufirst%3DG.%26aulast%3DZhou%26aufirst%3DD.%26atitle%3DA%2520selective%2520BCL-XL%2520PROTAC%2520degrader%2520achieves%2520safe%2520and%2520potent%2520antitumor%2520activity%26jtitle%3DNat.%2520Med.%26date%3D2019%26volume%3D25%26spage%3D1938%26epage%3D1947%26doi%3D10.1038%2Fs41591-019-0668-z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ma, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stratikopoulos, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, T. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwarz, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leshchenko, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rialdi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dale, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lagana, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guccione, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parekh, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parsons, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span> <span> </span><span class="NLM_article-title">Discovery of a first-in-class EZH2 selective degrader</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">214</span>, <span class="refDoi"> DOI: 10.1038/s41589-019-0421-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=10.1038%2Fs41589-019-0421-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=31819273" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2019&pages=214&author=A.+Maauthor=E.+Stratikopoulosauthor=K.+S.+Parkauthor=J.+Weiauthor=T.+C.+Martinauthor=X.+Yangauthor=M.+Schwarzauthor=V.+Leshchenkoauthor=A.+Rialdiauthor=B.+Daleauthor=A.+Laganaauthor=E.+Guccioneauthor=S.+Parekhauthor=R.+Parsonsauthor=J.+Jin&title=Discovery+of+a+first-in-class+EZH2+selective+degrader&doi=10.1038%2Fs41589-019-0421-4"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1038%2Fs41589-019-0421-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41589-019-0421-4%26sid%3Dliteratum%253Aachs%26aulast%3DMa%26aufirst%3DA.%26aulast%3DStratikopoulos%26aufirst%3DE.%26aulast%3DPark%26aufirst%3DK.%2BS.%26aulast%3DWei%26aufirst%3DJ.%26aulast%3DMartin%26aufirst%3DT.%2BC.%26aulast%3DYang%26aufirst%3DX.%26aulast%3DSchwarz%26aufirst%3DM.%26aulast%3DLeshchenko%26aufirst%3DV.%26aulast%3DRialdi%26aufirst%3DA.%26aulast%3DDale%26aufirst%3DB.%26aulast%3DLagana%26aufirst%3DA.%26aulast%3DGuccione%26aufirst%3DE.%26aulast%3DParekh%26aufirst%3DS.%26aulast%3DParsons%26aufirst%3DR.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DDiscovery%2520of%2520a%2520first-in-class%2520EZH2%2520selective%2520degrader%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2019%26volume%3D16%26spage%3D214%26doi%3D10.1038%2Fs41589-019-0421-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ren, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span> <span> </span><span class="NLM_article-title">Discovery of novel resorcinol diphenyl ether-based PROTAC-like molecules as dual inhibitors and degraders of PD-L1</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>199</i></span>,  <span class="NLM_fpage">112377</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2020.112377</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=10.1016%2Fj.ejmech.2020.112377" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=32388281" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=1%3ACAS%3A528%3ADC%252BB3cXptFGisLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=199&publication_year=2020&pages=112377&author=B.+Chengauthor=Y.+Renauthor=H.+Caoauthor=J.+Chen&title=Discovery+of+novel+resorcinol+diphenyl+ether-based+PROTAC-like+molecules+as+dual+inhibitors+and+degraders+of+PD-L1&doi=10.1016%2Fj.ejmech.2020.112377"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of novel resorcinol diphenyl ether-based PROTAC-like molecules as dual inhibitors and degraders of PD-L1</span></div><div class="casAuthors">Cheng, Binbin; Ren, Yichang; Cao, Hao; Chen, Jianjun</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">199</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">112377</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">Novel resorcinol di-Ph ether-based PROTACs (PROteolysis TArgeting Chimeras) were designed and evaluated for their inhibitory activity against the programmed cell death-1/programmed cell death-ligand 1 (PD-1/PD-L1) pathway and their ability to degrade PD-L1 protein.  Most of the compds. displayed excellent inhibitory activities against PD-1/PD-L1, as assessed by the homogeneous time-resolved fluorescence (HTRF) binding assay, with IC50 values ranging from 25 nM to 200 nM.  Among them, compd. P22 (I) is one of the best with an IC50 value of 39.2 nM.  In addn. to inhibiting PD-1/PD-L1 interaction, P22 also significantly restored the immunity repressed in a co-culture model of Hep3B/OS-8/hPD-L1 and CD3 T cells.  Furthermore, flow cytometry (FCM) and western-blot data demonstrated that P22 could moderately reduce the protein levels of PD-L1 in a lysosome-dependent manner, which may contribute to its immune effects.  Preliminary FCM and western-blot data suggest that it is possible to build PD-L1-targeting PROTAC-like mols. based on PD-1/PD-L1 small mol. inhibitors, though these compds. showed only modest degrdn. efficiencies.  Collectively, this work suggests that P22 may serve as a starting point for exploring the degrdn. of PD-L1 by PROTAC-like strategy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoiG4nI2klh77Vg90H21EOLACvtfcHk0lj4xlCZ_fHoIQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXptFGisLk%253D&md5=ec722d2e118668f601c0f9648e6f893b</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2020.112377&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2020.112377%26sid%3Dliteratum%253Aachs%26aulast%3DCheng%26aufirst%3DB.%26aulast%3DRen%26aufirst%3DY.%26aulast%3DCao%26aufirst%3DH.%26aulast%3DChen%26aufirst%3DJ.%26atitle%3DDiscovery%2520of%2520novel%2520resorcinol%2520diphenyl%2520ether-based%2520PROTAC-like%2520molecules%2520as%2520dual%2520inhibitors%2520and%2520degraders%2520of%2520PD-L1%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2020%26volume%3D199%26spage%3D112377%26doi%3D10.1016%2Fj.ejmech.2020.112377" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sun, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ren, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kong, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhong, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qiu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, B.</span></span> <span> </span><span class="NLM_article-title">Development of a Brigatinib degrader (SIAIS117) as a potential treatment for ALK positive cancer resistance</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>193</i></span>,  <span class="NLM_fpage">112190</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2020.112190</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=10.1016%2Fj.ejmech.2020.112190" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=32179332" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=1%3ACAS%3A528%3ADC%252BB3cXkvF2qu70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=193&publication_year=2020&pages=112190&author=N.+Sunauthor=C.+Renauthor=Y.+Kongauthor=H.+Zhongauthor=J.+Chenauthor=Y.+Liauthor=J.+Zhangauthor=Y.+Zhouauthor=X.+Qiuauthor=H.+Linauthor=X.+Songauthor=X.+Yangauthor=B.+Jiang&title=Development+of+a+Brigatinib+degrader+%28SIAIS117%29+as+a+potential+treatment+for+ALK+positive+cancer+resistance&doi=10.1016%2Fj.ejmech.2020.112190"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Development of a Brigatinib degrader (SIAIS117) as a potential treatment for ALK positive cancer resistance</span></div><div class="casAuthors">Sun, Ning; Ren, Chaowei; Kong, Ying; Zhong, Hui; Chen, Jinju; Li, Yan; Zhang, Jianshui; Zhou, Yuedong; Qiu, Xing; Lin, Haifan; Song, Xiaoling; Yang, Xiaobao; Jiang, Biao</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">193</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">112190</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">EML4-ALK and NPM-ALK fusion proteins possess constitutively activated ALK (anaplastic lymphoma kinase) activity, which in turn leads to the development of non-small cell lung cancer and anaplastic large-cell lymphomas (ALCLs).  FDA-approved ALK inhibitor drugs cause significant cancer regression.  However, drug resistance eventually occurs and it becomes a big obstacle in clinic.  Novel proteolysis targeting chimera (PROTAC) technol. platform provides a potential therapeutic strategy for drug resistance.  Herein, we designed and synthesized a series of ALK PROTACs based on Brigatinib and VHL-1 conjunction, and screened SIAIS117 as the best degrader which not only blocked the growth of SR and H2228 cancer cell lines, but also degraded ALK protein.  In addn., SIAIS117 also showed much better growth inhibition effect than Brigatinib on 293T cell line that exogenously expressed G1202R-resistant ALK proteins.  Furthermore, it also degraded G1202R mutant ALK protein in vitro.  At last, it has the potentially anti-proliferation ability of small cell lung cancer.  Thus, we have successfully generated the degrader SIAIS117 that can potentially overcome resistance in cancer targeted therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrX9rzk88HMe7Vg90H21EOLACvtfcHk0lj4xlCZ_fHoIQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXkvF2qu70%253D&md5=61f0228111c806ae62473fc450fce276</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2020.112190&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2020.112190%26sid%3Dliteratum%253Aachs%26aulast%3DSun%26aufirst%3DN.%26aulast%3DRen%26aufirst%3DC.%26aulast%3DKong%26aufirst%3DY.%26aulast%3DZhong%26aufirst%3DH.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DZhou%26aufirst%3DY.%26aulast%3DQiu%26aufirst%3DX.%26aulast%3DLin%26aufirst%3DH.%26aulast%3DSong%26aufirst%3DX.%26aulast%3DYang%26aufirst%3DX.%26aulast%3DJiang%26aufirst%3DB.%26atitle%3DDevelopment%2520of%2520a%2520Brigatinib%2520degrader%2520%2528SIAIS117%2529%2520as%2520a%2520potential%2520treatment%2520for%2520ALK%2520positive%2520cancer%2520resistance%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2020%26volume%3D193%26spage%3D112190%26doi%3D10.1016%2Fj.ejmech.2020.112190" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ren, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kong, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yin, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, B.</span></span> <span> </span><span class="NLM_article-title">Discovery of SIAIS178 as an Effective BCR-ABL Degrader by Recruiting Von Hippel-Lindau (VHL) E3 Ubiquitin Ligase</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">9281</span>– <span class="NLM_lpage">9298</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b01264</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b01264" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhvVantrnN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=9281-9298&author=Q.+Zhaoauthor=C.+Renauthor=L.+Liuauthor=J.+Chenauthor=Y.+Shaoauthor=N.+Sunauthor=R.+Sunauthor=Y.+Kongauthor=X.+Dingauthor=X.+Zhangauthor=Y.+Xuauthor=B.+Yangauthor=Q.+Yinauthor=X.+Yangauthor=B.+Jiang&title=Discovery+of+SIAIS178+as+an+Effective+BCR-ABL+Degrader+by+Recruiting+Von+Hippel-Lindau+%28VHL%29+E3+Ubiquitin+Ligase&doi=10.1021%2Facs.jmedchem.9b01264"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of SIAIS178 as an Effective BCR-ABL Degrader by Recruiting Von Hippel-Lindau (VHL) E3 Ubiquitin Ligase</span></div><div class="casAuthors">Zhao, Quanju; Ren, Chaowei; Liu, Linyi; Chen, Jinju; Shao, Yubao; Sun, Ning; Sun, Renhong; Kong, Ying; Ding, Xinyu; Zhang, Xianfang; Xu, Youwei; Yang, Bei; Yin, Qianqian; Yang, Xiaobao; Jiang, Biao</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">9281-9298</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The oncogenic fusion protein BCR-ABL is the driving force of leukemogenesis in chronic myeloid leukemia (CML).  Despite great progress for CML treatment through application of tyrosine kinase inhibitors (TKIs) against BCR-ABL, long-term drug administration and clin. resistance continue to be an issue.  Herein, we described the design, synthesis, and evaluation of novel proteolysis-targeting chimeric (PROTAC) small mols. targeting BCR-ABL which connect dasatinib and VHL E3 ubiquitin ligase ligand by extensive optimization of linkers.  Our efforts have yielded SIAIS178 (19), which induces proper interaction between BCR-ABL and VHL ligase leading to effective degrdn. of BCR-ABL protein, achieves significant growth inhibition of BCR-ABL+ leukemic cells in vitro, and induces substantial tumor regression against K562 xenograft tumors in vivo.  In addn., SIAIS178 also degrades several clin. relevant resistance-conferring mutations.  Our data indicate that SIAIS178 as efficacious BCR-ABL degrader warrants extensive further investigation for the treatment of BCR-ABL+ leukemia.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrLBFfujSx6ZbVg90H21EOLACvtfcHk0li1zMv5bFEFwA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhvVantrnN&md5=50ace274dbc6ef3931f86504a1be93a7</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b01264&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b01264%26sid%3Dliteratum%253Aachs%26aulast%3DZhao%26aufirst%3DQ.%26aulast%3DRen%26aufirst%3DC.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DShao%26aufirst%3DY.%26aulast%3DSun%26aufirst%3DN.%26aulast%3DSun%26aufirst%3DR.%26aulast%3DKong%26aufirst%3DY.%26aulast%3DDing%26aufirst%3DX.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DXu%26aufirst%3DY.%26aulast%3DYang%26aufirst%3DB.%26aulast%3DYin%26aufirst%3DQ.%26aulast%3DYang%26aufirst%3DX.%26aulast%3DJiang%26aufirst%3DB.%26atitle%3DDiscovery%2520of%2520SIAIS178%2520as%2520an%2520Effective%2520BCR-ABL%2520Degrader%2520by%2520Recruiting%2520Von%2520Hippel-Lindau%2520%2528VHL%2529%2520E3%2520Ubiquitin%2520Ligase%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D9281%26epage%3D9298%26doi%3D10.1021%2Facs.jmedchem.9b01264" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaniskan, H. Ü.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, W.</span></span> <span> </span><span class="NLM_article-title">Light-induced control of protein destruction by opto-PROTAC</span>. <i>Sci. Adv.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>6</i></span>, <span class="NLM_elocation-id">eaay5154</span> <span class="refDoi"> DOI: 10.1126/sciadv.aay5154</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=10.1126%2Fsciadv.aay5154" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=32128407" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2020&author=J.+Liuauthor=H.+Chenauthor=L.+Maauthor=Z.+Heauthor=D.+Wangauthor=Y.+Liuauthor=Q.+Linauthor=T.+Zhangauthor=N.+Grayauthor=H.+%C3%9C.+Kaniskanauthor=J.+Jinauthor=W.+Wei&title=Light-induced+control+of+protein+destruction+by+opto-PROTAC&doi=10.1126%2Fsciadv.aay5154"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1126%2Fsciadv.aay5154&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fsciadv.aay5154%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DH.%26aulast%3DMa%26aufirst%3DL.%26aulast%3DHe%26aufirst%3DZ.%26aulast%3DWang%26aufirst%3DD.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DLin%26aufirst%3DQ.%26aulast%3DZhang%26aufirst%3DT.%26aulast%3DGray%26aufirst%3DN.%26aulast%3DKaniskan%26aufirst%3DH.%2B%25C3%259C.%26aulast%3DJin%26aufirst%3DJ.%26aulast%3DWei%26aufirst%3DW.%26atitle%3DLight-induced%2520control%2520of%2520protein%2520destruction%2520by%2520opto-PROTAC%26jtitle%3DSci.%2520Adv.%26date%3D2020%26volume%3D6%26doi%3D10.1126%2Fsciadv.aay5154" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Reynders, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsuura, B. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bérouti, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simoneschi, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marzio, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pagano, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trauner, D.</span></span> <span> </span><span class="NLM_article-title">PHOTACs enable optical control of protein degradation</span>. <i>Sci. Adv.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>6</i></span>, <span class="NLM_elocation-id">eaay5064</span> <span class="refDoi"> DOI: 10.1126/sciadv.aay5064</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=10.1126%2Fsciadv.aay5064" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=32128406" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2020&author=M.+Reyndersauthor=B.+S.+Matsuuraauthor=M.+B%C3%A9routiauthor=D.+Simoneschiauthor=A.+Marzioauthor=M.+Paganoauthor=D.+Trauner&title=PHOTACs+enable+optical+control+of+protein+degradation&doi=10.1126%2Fsciadv.aay5064"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1126%2Fsciadv.aay5064&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fsciadv.aay5064%26sid%3Dliteratum%253Aachs%26aulast%3DReynders%26aufirst%3DM.%26aulast%3DMatsuura%26aufirst%3DB.%2BS.%26aulast%3DB%25C3%25A9routi%26aufirst%3DM.%26aulast%3DSimoneschi%26aufirst%3DD.%26aulast%3DMarzio%26aufirst%3DA.%26aulast%3DPagano%26aufirst%3DM.%26aulast%3DTrauner%26aufirst%3DD.%26atitle%3DPHOTACs%2520enable%2520optical%2520control%2520of%2520protein%2520degradation%26jtitle%3DSci.%2520Adv.%26date%3D2020%26volume%3D6%26doi%3D10.1126%2Fsciadv.aay5064" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ren, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kong, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhong, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, B.</span></span> <span> </span><span class="NLM_article-title">Structure-based discovery of SIAIS001 as an oral bioavailability ALK degrader constructed from Alectinib</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2021</span>,  <span class="NLM_volume"><i>217</i></span>,  <span class="NLM_fpage">113335</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2021.113335</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=10.1016%2Fj.ejmech.2021.113335" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=33751979" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=1%3ACAS%3A528%3ADC%252BB3MXnt1Ohsrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=217&publication_year=2021&pages=113335&author=C.+Renauthor=N.+Sunauthor=Y.+Kongauthor=X.+Quauthor=H.+Liuauthor=H.+Zhongauthor=X.+Songauthor=X.+Yangauthor=B.+Jiang&title=Structure-based+discovery+of+SIAIS001+as+an+oral+bioavailability+ALK+degrader+constructed+from+Alectinib&doi=10.1016%2Fj.ejmech.2021.113335"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-based discovery of SIAIS001 as an oral bioavailability ALK degrader constructed from Alectinib</span></div><div class="casAuthors">Ren, Chaowei; Sun, Ning; Kong, Ying; Qu, Xiaojuan; Liu, Haixia; Zhong, Hui; Song, Xiaoling; Yang, Xiaobao; Jiang, Biao</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2021</span>),
    <span class="NLM_cas:volume">217</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">113335</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">Fusion proteins of the anaplastic lymphoma kinase (ALK) are promising therapeutic targets for cancer and other human diseases, esp. for non-small cell lung cancer (NSCLC) and anaplastic large-cell lymphomas (ALCLs).  We described herein a structure-based design, synthesis, and evaluation of ALK PROTACs (proteolysis-targeting chimeras) based on Alectinib as the warhead.  We firstly screened CRBN ligands as the E3 ligase moiety, then obtained a series of potent ALK degraders based on different CRBN ligands, exemplified by SIAIS091 and SIAIS001 with lenalidomide/thalidomide-based linkers.  Both of them induced effective ALK degrdn. at low nanomolar concns. in cells, and showed much better growth inhibition effects than Alectinib.  SIAIS091 or SIAIS001 also promoted cell cycle arrest in G1/S phase.  Finally, SIAIS001 exhibited good oral bioavailability in Pharmacokinetics study.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoZW4fwPhRGNLVg90H21EOLACvtfcHk0liTrF_9NjLB2A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3MXnt1Ohsrs%253D&md5=b1dc676e47efb1fdf8e9fb50cafec635</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2021.113335&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2021.113335%26sid%3Dliteratum%253Aachs%26aulast%3DRen%26aufirst%3DC.%26aulast%3DSun%26aufirst%3DN.%26aulast%3DKong%26aufirst%3DY.%26aulast%3DQu%26aufirst%3DX.%26aulast%3DLiu%26aufirst%3DH.%26aulast%3DZhong%26aufirst%3DH.%26aulast%3DSong%26aufirst%3DX.%26aulast%3DYang%26aufirst%3DX.%26aulast%3DJiang%26aufirst%3DB.%26atitle%3DStructure-based%2520discovery%2520of%2520SIAIS001%2520as%2520an%2520oral%2520bioavailability%2520ALK%2520degrader%2520constructed%2520from%2520Alectinib%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2021%26volume%3D217%26spage%3D113335%26doi%3D10.1016%2Fj.ejmech.2021.113335" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sun, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tong, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rao, Y.</span></span> <span> </span><span class="NLM_article-title">PROTACs: great opportunities for academia and industry</span>. <i>Signal Transduction Targeted Ther.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">64</span>, <span class="refDoi"> DOI: 10.1038/s41392-019-0101-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=10.1038%2Fs41392-019-0101-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=31885879" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=1%3ACAS%3A280%3ADC%252BB3MbkvFCrtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2019&pages=64&author=X.+Sunauthor=H.+Gaoauthor=Y.+Yangauthor=M.+Heauthor=Y.+Wuauthor=Y.+Songauthor=Y.+Tongauthor=Y.+Rao&title=PROTACs%3A+great+opportunities+for+academia+and+industry&doi=10.1038%2Fs41392-019-0101-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">PROTACs: great opportunities for academia and industry</span></div><div class="casAuthors">Sun Xiuyun; Gao Hongying; Yang Yiqing; He Ming; Wu Yue; Song Yugang; Tong Yan; Rao Yu; Sun Xiuyun; Gao Hongying; Yang Yiqing; Rao Yu</div><div class="citationInfo"><span class="NLM_cas:title">Signal transduction and targeted therapy</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">64</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Although many kinds of therapies are applied in the clinic, drug-resistance is a major and unavoidable problem.  Another disturbing statistic is the limited number of drug targets, which are presently only 20-25% of all protein targets that are currently being studied.  Moreover, the focus of current explorations of targets are their enzymatic functions, which ignores the functions from their scaffold moiety.  As a promising and appealing technology, PROteolysis TArgeting Chimeras (PROTACs) have attracted great attention both from academia and industry for finding available approaches to solve the above problems.  PROTACs regulate protein function by degrading target proteins instead of inhibiting them, providing more sensitivity to drug-resistant targets and a greater chance to affect the nonenzymatic functions.  PROTACs have been proven to show better selectivity compared to classic inhibitors.  PROTACs can be described as a chemical knockdown approach with rapidity and reversibility, which presents new and different biology compared to other gene editing tools by avoiding misinterpretations that arise from potential genetic compensation and/or spontaneous mutations.  PRTOACs have been widely explored throughout the world and have outperformed not only in cancer diseases, but also in immune disorders, viral infections and neurodegenerative diseases.  Although PROTACs present a very promising and powerful approach for crossing the hurdles of present drug discovery and tool development in biology, more efforts are needed to gain to get deeper insight into the efficacy and safety of PROTACs in the clinic.  More target binders and more E3 ligases applicable for developing PROTACs are waiting for exploration.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQretvERU2xT-_Q9URG6Dx8fW6udTcc2eanH46HK4pRvLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3MbkvFCrtA%253D%253D&md5=7d9545a21e3a78cededf2977a31e8f7d</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1038%2Fs41392-019-0101-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41392-019-0101-6%26sid%3Dliteratum%253Aachs%26aulast%3DSun%26aufirst%3DX.%26aulast%3DGao%26aufirst%3DH.%26aulast%3DYang%26aufirst%3DY.%26aulast%3DHe%26aufirst%3DM.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DSong%26aufirst%3DY.%26aulast%3DTong%26aufirst%3DY.%26aulast%3DRao%26aufirst%3DY.%26atitle%3DPROTACs%253A%2520great%2520opportunities%2520for%2520academia%2520and%2520industry%26jtitle%3DSignal%2520Transduction%2520Targeted%2520Ther.%26date%3D2019%26volume%3D4%26spage%3D64%26doi%3D10.1038%2Fs41392-019-0101-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mullard, A.</span></span> <span> </span><span class="NLM_article-title">Arvinas’s PROTACs pass first safety and PK analysis</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">895</span>, <span class="refDoi"> DOI: 10.1038/d41573-019-00188-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=10.1038%2Fd41573-019-00188-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=1%3ACAS%3A528%3ADC%252BC1MXit1Ohs7zO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2019&pages=895&author=A.+Mullard&title=Arvinas%E2%80%99s+PROTACs+pass+first+safety+and+PK+analysis&doi=10.1038%2Fd41573-019-00188-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">NIH launches open science Alzheimer initiative</span></div><div class="casAuthors">Mullard, Asher</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">895</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqKr7PwgMO_fLVg90H21EOLACvtfcHk0ljXulMWAxe9sw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXit1Ohs7zO&md5=2abfeaf7ac16a726ceff22cb0164347c</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1038%2Fd41573-019-00188-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fd41573-019-00188-4%26sid%3Dliteratum%253Aachs%26aulast%3DMullard%26aufirst%3DA.%26atitle%3DArvinas%25E2%2580%2599s%2520PROTACs%2520pass%2520first%2520safety%2520and%2520PK%2520analysis%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2019%26volume%3D18%26spage%3D895%26doi%3D10.1038%2Fd41573-019-00188-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mullard, A.</span></span> <span> </span><span class="NLM_article-title">Targeted degraders clear first safety hurdles</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">435</span>, <span class="refDoi"> DOI: 10.1038/d41573-020-00109-w</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=10.1038%2Fd41573-020-00109-w" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhtlShtbrM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2020&pages=435&author=A.+Mullard&title=Targeted+degraders+clear+first+safety+hurdles&doi=10.1038%2Fd41573-020-00109-w"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Targeted degraders clear first safety hurdles</span></div><div class="casAuthors">Mullard, Asher</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">435</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp7nHwwDZKMr7Vg90H21EOLACvtfcHk0ljXulMWAxe9sw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhtlShtbrM&md5=122b44714339e8b38fc418c2f8732706</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1038%2Fd41573-020-00109-w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fd41573-020-00109-w%26sid%3Dliteratum%253Aachs%26aulast%3DMullard%26aufirst%3DA.%26atitle%3DTargeted%2520degraders%2520clear%2520first%2520safety%2520hurdles%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2020%26volume%3D19%26spage%3D435%26doi%3D10.1038%2Fd41573-020-00109-w" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anjum, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Squillace, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nadworny, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keats, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ning, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wardwell, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eichinger, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moran, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, W.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zou, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mohemmad, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jang, H. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ye, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Narasimhan, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miret, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clackson, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dalgarno, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shakespeare, W. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rivera, V. M.</span></span> <span> </span><span class="NLM_article-title">The Potent ALK Inhibitor Brigatinib (AP26113) Overcomes Mechanisms of Resistance to First- and Second-Generation ALK Inhibitors in Preclinical Models</span>. <i>Clin. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">5527</span>– <span class="NLM_lpage">5538</span>, <span class="refDoi"> DOI: 10.1158/1078-0432.ccr-16-0569</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=10.1158%2F1078-0432.CCR-16-0569" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=27780853" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvVCqur3O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2016&pages=5527-5538&author=S.+Zhangauthor=R.+Anjumauthor=R.+Squillaceauthor=S.+Nadwornyauthor=T.+Zhouauthor=J.+Keatsauthor=Y.+Ningauthor=S.+D.+Wardwellauthor=D.+Millerauthor=Y.+Songauthor=L.+Eichingerauthor=L.+Moranauthor=W.-S.+Huangauthor=S.+Liuauthor=D.+Zouauthor=Y.+Wangauthor=Q.+Mohemmadauthor=H.+G.+Jangauthor=E.+Yeauthor=N.+Narasimhanauthor=F.+Wangauthor=J.+Miretauthor=X.+Zhuauthor=T.+Clacksonauthor=D.+Dalgarnoauthor=W.+C.+Shakespeareauthor=V.+M.+Rivera&title=The+Potent+ALK+Inhibitor+Brigatinib+%28AP26113%29+Overcomes+Mechanisms+of+Resistance+to+First-+and+Second-Generation+ALK+Inhibitors+in+Preclinical+Models&doi=10.1158%2F1078-0432.ccr-16-0569"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">The Potent ALK Inhibitor Brigatinib (AP26113) Overcomes Mechanisms of Resistance to First- and Second-Generation ALK Inhibitors in Preclinical Models</span></div><div class="casAuthors">Zhang, Sen; Anjum, Rana; Squillace, Rachel; Nadworny, Sara; Zhou, Tianjun; Keats, Jeff; Ning, Yaoyu; Wardwell, Scott D.; Miller, David; Song, Youngchul; Eichinger, Lindsey; Moran, Lauren; Huang, Wei-Sheng; Liu, Shuangying; Zou, Dong; Wang, Yihan; Mohemmad, Qurish; Jang, Hyun Gyung; Ye, Emily; Narasimhan, Narayana; Wang, Frank; Miret, Juan; Zhu, Xiaotian; Clackson, Tim; Dalgarno, David; Shakespeare, William C.; Rivera, Victor M.</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">5527-5538</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Purpose: Non-small cell lung cancers (NSCLCs) harboring ALK gene rearrangements (ALK+) typically become resistant to the first-generation anaplastic lymphoma kinase (ALK) tyrosine kinase inhibitor (TKI) crizotinib through development of secondary resistance mutations in ALK or disease progression in the brain.  Mutations that confer resistance to second-generation ALK TKIs ceritinib and alectinib have also been identified.  Here, we report the structure and first comprehensive preclin. evaluation of the next-generation ALK TKI brigatinib.  Exptl. Design: A kinase screen was performed to evaluate the selectivity profile of brigatinib.  The cellular and in vivo activities of ALK TKIs were compared using engineered and cancer-derived cell lines.  The brigatinib-ALK co-structure was detd.  Results: Brigatinib potently inhibits ALK and ROS1, with a high degree of selectivity over more than 250 kinases.  Across a panel of ALK+ cell lines, brigatinib inhibited native ALK (IC50, 10 nmol/L) with 12-fold greater potency than crizotinib.  Superior efficacy of brigatinib was also obsd. in mice with ALK+ tumors implanted s.c. or intracranially.  Brigatinib maintained substantial activity against all 17 secondary ALK mutants tested in cellular assays and exhibited a superior inhibitory profile compared with crizotinib, ceritinib, and alectinib at clin. achievable concns.  Brigatinib was the only TKI to maintain substantial activity against the most recalcitrant ALK resistance mutation, G1202R.  The unique, potent, and pan-ALK mutant activity of brigatinib could be rationalized by structural analyses.  Conclusions: Brigatinib is a highly potent and selective ALK inhibitor.  These findings provide the mol. basis for the promising activity being obsd. in ALK+, crizotinib-resistant patients with NSCLC being treated with brigatinib in clin. trials.  Clin Cancer Res; 22(22); 5527-38. ©2016 AACR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoVwQUn_qLy7bVg90H21EOLACvtfcHk0ljXulMWAxe9sw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvVCqur3O&md5=1ece23bf36e807562023d0b814c990c7</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1158%2F1078-0432.CCR-16-0569&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1078-0432.CCR-16-0569%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DS.%26aulast%3DAnjum%26aufirst%3DR.%26aulast%3DSquillace%26aufirst%3DR.%26aulast%3DNadworny%26aufirst%3DS.%26aulast%3DZhou%26aufirst%3DT.%26aulast%3DKeats%26aufirst%3DJ.%26aulast%3DNing%26aufirst%3DY.%26aulast%3DWardwell%26aufirst%3DS.%2BD.%26aulast%3DMiller%26aufirst%3DD.%26aulast%3DSong%26aufirst%3DY.%26aulast%3DEichinger%26aufirst%3DL.%26aulast%3DMoran%26aufirst%3DL.%26aulast%3DHuang%26aufirst%3DW.-S.%26aulast%3DLiu%26aufirst%3DS.%26aulast%3DZou%26aufirst%3DD.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DMohemmad%26aufirst%3DQ.%26aulast%3DJang%26aufirst%3DH.%2BG.%26aulast%3DYe%26aufirst%3DE.%26aulast%3DNarasimhan%26aufirst%3DN.%26aulast%3DWang%26aufirst%3DF.%26aulast%3DMiret%26aufirst%3DJ.%26aulast%3DZhu%26aufirst%3DX.%26aulast%3DClackson%26aufirst%3DT.%26aulast%3DDalgarno%26aufirst%3DD.%26aulast%3DShakespeare%26aufirst%3DW.%2BC.%26aulast%3DRivera%26aufirst%3DV.%2BM.%26atitle%3DThe%2520Potent%2520ALK%2520Inhibitor%2520Brigatinib%2520%2528AP26113%2529%2520Overcomes%2520Mechanisms%2520of%2520Resistance%2520to%2520First-%2520and%2520Second-Generation%2520ALK%2520Inhibitors%2520in%2520Preclinical%2520Models%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2016%26volume%3D22%26spage%3D5527%26epage%3D5538%26doi%3D10.1158%2F1078-0432.ccr-16-0569" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huang, W.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zou, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Romero, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kohlmann, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dwight, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dodd, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toms, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parillon, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anjum, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keats, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wardwell, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ning, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moran, L. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mohemmad, Q. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jang, H. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clackson, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Narasimhan, N. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rivera, V. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dalgarno, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shakespeare, W. C.</span></span> <span> </span><span class="NLM_article-title">Discovery of Brigatinib (AP26113), a Phosphine Oxide-Containing, Potent, Orally Active Inhibitor of Anaplastic Lymphoma Kinase</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">4948</span>– <span class="NLM_lpage">4964</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b00306</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b00306" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=1%3ACAS%3A528%3ADC%252BC28XntlCgur8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=4948-4964&author=W.-S.+Huangauthor=S.+Liuauthor=D.+Zouauthor=M.+Thomasauthor=Y.+Wangauthor=T.+Zhouauthor=J.+Romeroauthor=A.+Kohlmannauthor=F.+Liauthor=J.+Qiauthor=L.+Caiauthor=T.+A.+Dwightauthor=Y.+Xuauthor=R.+Xuauthor=R.+Doddauthor=A.+Tomsauthor=L.+Parillonauthor=X.+Luauthor=R.+Anjumauthor=S.+Zhangauthor=F.+Wangauthor=J.+Keatsauthor=S.+D.+Wardwellauthor=Y.+Ningauthor=Q.+Xuauthor=L.+E.+Moranauthor=Q.+K.+Mohemmadauthor=H.+G.+Jangauthor=T.+Clacksonauthor=N.+I.+Narasimhanauthor=V.+M.+Riveraauthor=X.+Zhuauthor=D.+Dalgarnoauthor=W.+C.+Shakespeare&title=Discovery+of+Brigatinib+%28AP26113%29%2C+a+Phosphine+Oxide-Containing%2C+Potent%2C+Orally+Active+Inhibitor+of+Anaplastic+Lymphoma+Kinase&doi=10.1021%2Facs.jmedchem.6b00306"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Brigatinib (AP26113), a Phosphine Oxide-Containing, Potent, Orally Active Inhibitor of Anaplastic Lymphoma Kinase</span></div><div class="casAuthors">Huang, Wei-Sheng; Liu, Shuangying; Zou, Dong; Thomas, Mathew; Wang, Yihan; Zhou, Tianjun; Romero, Jan; Kohlmann, Anna; Li, Feng; Qi, Jiwei; Cai, Lisi; Dwight, Timothy A.; Xu, Yongjin; Xu, Rongsong; Dodd, Rory; Toms, Angela; Parillon, Lois; Lu, Xiaohui; Anjum, Rana; Zhang, Sen; Wang, Frank; Keats, Jeffrey; Wardwell, Scott D.; Ning, Yaoyu; Xu, Qihong; Moran, Lauren E.; Mohemmad, Qurish K.; Jang, Hyun Gyung; Clackson, Tim; Narasimhan, Narayana I.; Rivera, Victor M.; Zhu, Xiaotian; Dalgarno, David; Shakespeare, William C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">4948-4964</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In the treatment of echinoderm microtubule-assocd. protein-like 4 (EML4)-anaplastic lymphoma kinase pos. (ALK+) non-small-cell lung cancer (NSCLC), secondary mutations within the ALK kinase domain have emerged as a major resistance mechanism to both first- and second-generation ALK inhibitors.  This report describes the design and synthesis of a series of 2,4-diarylaminopyrimidine-based potent and selective ALK inhibitors culminating in identification of the investigational clin. candidate brigatinib.  A unique structural feature of brigatinib is a phosphine oxide, an overlooked but novel hydrogen-bond acceptor that drives potency and selectivity in addn. to favorable ADME properties.  Brigatinib displayed low nanomolar IC50s against native ALK and all tested clin. relevant ALK mutants in both enzyme-based biochem. and cell-based viability assays and demonstrated efficacy in multiple ALK+ xenografts in mice, including Karpas-299 (anaplastic large-cell lymphomas [ALCL]) and H3122 (NSCLC).  Brigatinib represents the most clin. advanced phosphine oxide-contg. drug candidate to date and is currently being evaluated in a global phase 2 registration trial.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr5_iqprSA4QbVg90H21EOLACvtfcHk0ljdC9KRUKqTpA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XntlCgur8%253D&md5=4cb93ff587579c97fda8aff0c7d43317</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b00306&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b00306%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DW.-S.%26aulast%3DLiu%26aufirst%3DS.%26aulast%3DZou%26aufirst%3DD.%26aulast%3DThomas%26aufirst%3DM.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DZhou%26aufirst%3DT.%26aulast%3DRomero%26aufirst%3DJ.%26aulast%3DKohlmann%26aufirst%3DA.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DQi%26aufirst%3DJ.%26aulast%3DCai%26aufirst%3DL.%26aulast%3DDwight%26aufirst%3DT.%2BA.%26aulast%3DXu%26aufirst%3DY.%26aulast%3DXu%26aufirst%3DR.%26aulast%3DDodd%26aufirst%3DR.%26aulast%3DToms%26aufirst%3DA.%26aulast%3DParillon%26aufirst%3DL.%26aulast%3DLu%26aufirst%3DX.%26aulast%3DAnjum%26aufirst%3DR.%26aulast%3DZhang%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DF.%26aulast%3DKeats%26aufirst%3DJ.%26aulast%3DWardwell%26aufirst%3DS.%2BD.%26aulast%3DNing%26aufirst%3DY.%26aulast%3DXu%26aufirst%3DQ.%26aulast%3DMoran%26aufirst%3DL.%2BE.%26aulast%3DMohemmad%26aufirst%3DQ.%2BK.%26aulast%3DJang%26aufirst%3DH.%2BG.%26aulast%3DClackson%26aufirst%3DT.%26aulast%3DNarasimhan%26aufirst%3DN.%2BI.%26aulast%3DRivera%26aufirst%3DV.%2BM.%26aulast%3DZhu%26aufirst%3DX.%26aulast%3DDalgarno%26aufirst%3DD.%26aulast%3DShakespeare%26aufirst%3DW.%2BC.%26atitle%3DDiscovery%2520of%2520Brigatinib%2520%2528AP26113%2529%252C%2520a%2520Phosphine%2520Oxide-Containing%252C%2520Potent%252C%2520Orally%2520Active%2520Inhibitor%2520of%2520Anaplastic%2520Lymphoma%2520Kinase%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D4948%26epage%3D4964%26doi%3D10.1021%2Facs.jmedchem.6b00306" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bedi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khan, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">AbuKhader, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alam, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siddiqui, N. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Husain, A.</span></span> <span> </span><span class="NLM_article-title">A comprehensive review on Brigatinib - A wonder drug for targeted cancer therapy in non-small cell lung cancer</span>. <i>Saudi Pharm. J.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">755</span>– <span class="NLM_lpage">763</span>, <span class="refDoi"> DOI: 10.1016/j.jsps.2018.04.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=10.1016%2Fj.jsps.2018.04.010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=30202213" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=1%3ACAS%3A280%3ADC%252BB3c3nsFWqtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2018&pages=755-763&author=S.+Bediauthor=S.+A.+Khanauthor=M.+M.+AbuKhaderauthor=P.+Alamauthor=N.+A.+Siddiquiauthor=A.+Husain&title=A+comprehensive+review+on+Brigatinib+-+A+wonder+drug+for+targeted+cancer+therapy+in+non-small+cell+lung+cancer&doi=10.1016%2Fj.jsps.2018.04.010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">A comprehensive review on Brigatinib - A wonder drug for targeted cancer therapy in non-small cell lung cancer</span></div><div class="casAuthors">Bedi Silky; Husain Asif; Khan Shah A; AbuKhader Majed M; Alam Perwez; Siddiqui Nasir A</div><div class="citationInfo"><span class="NLM_cas:title">Saudi pharmaceutical journal : SPJ : the official publication of the Saudi Pharmaceutical Society</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">755-763</span>
        ISSN:<span class="NLM_cas:issn">1319-0164</span>.
    </div><div class="casAbstract">The mortality rate in patients suffering from non-small cell lung cancer (NSCLC) is quite high.  This type of cancer mainly occurs due to rearrangements in the anaplastic lymphoma kinase (ALK) gene which leads to form an oncogene of fused gene NPM-ALK.  Brigatinib is recently approved by FDA in April 2017 as a potent tyrosine kinase inhibitor (TKI) for the NSCLC therapy.  In the present scenario, it is no less than a wonder drug because it is indicated for the treatment of advanced stages of metastatic ALK positive NSCLC, a fatal disease to overcome the resistance of various other ALK inhibitors such as crizotinib, ceritinib and alectinib.  In addition to ALK, it is also active against multiple types of kinases such as ROS1, Insulin like growth factor-1Receptor and EGFR.  It can be synthesized by using N-[2-methoxy-4-[4-(dimethylamino) piperidin-1-yl] aniline] guanidine and 2,4,5-trichloropyrimidine respectively in two different ways.  Its structure consists of mainly dimethylphosphine oxide group which is responsible for its pharmacological activity.  It is active against various cell lines such as HCC78, H2228, H23, H358, H838, U937, HepG2 and Karpas- 299.  Results of ALTA (ALK in Lung Cancer Trial of AP26113) phase 1/2 trial shows that 90 mg of brigatinib for 7 days and then 180 mg for next days is effective in the treatment of NSCLC.  Brigatinib has been shown to have favorable risk benefit profile and is a safer drug than the available cytotoxic chemotherapeutic agents.  In comparison to other FDA approved drugs for the same condition, it causes fewer minor adverse reactions which can be easily managed either by changing the dose or by providing good supportive care.  This article is intended to provide readers with an overview of chemistry, pharmacokinetic, pharmacodynamic and safety profile of brigatinib, which addresses an unmet medical need.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcS1tFvEwEman42y4bB4mnvYfW6udTcc2eYijx79-4PJH7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3c3nsFWqtA%253D%253D&md5=6202b67fb7713e61af3a5744ced333b3</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1016%2Fj.jsps.2018.04.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jsps.2018.04.010%26sid%3Dliteratum%253Aachs%26aulast%3DBedi%26aufirst%3DS.%26aulast%3DKhan%26aufirst%3DS.%2BA.%26aulast%3DAbuKhader%26aufirst%3DM.%2BM.%26aulast%3DAlam%26aufirst%3DP.%26aulast%3DSiddiqui%26aufirst%3DN.%2BA.%26aulast%3DHusain%26aufirst%3DA.%26atitle%3DA%2520comprehensive%2520review%2520on%2520Brigatinib%2520-%2520A%2520wonder%2520drug%2520for%2520targeted%2520cancer%2520therapy%2520in%2520non-small%2520cell%2520lung%2520cancer%26jtitle%3DSaudi%2520Pharm.%2520J.%26date%3D2018%26volume%3D26%26spage%3D755%26epage%3D763%26doi%3D10.1016%2Fj.jsps.2018.04.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kim, D.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tiseo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ahn, M.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reckamp, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hansen, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, S.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huber, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">West, H. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Groen, H. J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hochmair, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leighl, N. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gettinger, S. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Langer, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paz-Ares Rodríguez, L. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smit, E. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reichmann, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haluska, F. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kerstein, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Camidge, D. R.</span></span> <span> </span><span class="NLM_article-title">Brigatinib in Patients With Crizotinib-Refractory Anaplastic Lymphoma Kinase-Positive Non-Small-Cell Lung Cancer: A Randomized, Multicenter Phase II Trial</span>. <i>J. Clin. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>35</i></span>,  <span class="NLM_fpage">2490</span>– <span class="NLM_lpage">2498</span>, <span class="refDoi"> DOI: 10.1200/jco.2016.71.5904</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=10.1200%2FJCO.2016.71.5904" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=28475456" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitVSqsbfK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2017&pages=2490-2498&author=D.-W.+Kimauthor=M.+Tiseoauthor=M.-J.+Ahnauthor=K.+L.+Reckampauthor=K.+H.+Hansenauthor=S.-W.+Kimauthor=R.+M.+Huberauthor=H.+L.+Westauthor=H.+J.+M.+Groenauthor=M.+J.+Hochmairauthor=N.+B.+Leighlauthor=S.+N.+Gettingerauthor=C.+J.+Langerauthor=L.+G.+Paz-Ares+Rodr%C3%ADguezauthor=E.+F.+Smitauthor=E.+S.+Kimauthor=W.+Reichmannauthor=F.+G.+Haluskaauthor=D.+Kersteinauthor=D.+R.+Camidge&title=Brigatinib+in+Patients+With+Crizotinib-Refractory+Anaplastic+Lymphoma+Kinase-Positive+Non-Small-Cell+Lung+Cancer%3A+A+Randomized%2C+Multicenter+Phase+II+Trial&doi=10.1200%2Fjco.2016.71.5904"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Brigatinib in patients with crizotinib-refractory anaplastic lymphoma kinase-positive non-small-cell lung cancer: a randomized, multicenter phase II trial</span></div><div class="casAuthors">Kim, Dong-Wan; Tiseo, Marcello; Ahn, Myung-Ju; Reckamp, Karen L.; Hansen, Karin Holmskov; Kim, Sang-We; Huber, Rudolf M.; West, Howard L.; Groen, Harry J. M.; Hochmair, Maximilian J.; Leighl, Natasha B.; Gettinger, Scott N.; Langer, Corey J.; Rodr'iguez, Luis G. Paz-Ares; Smit, Egbert F.; Kim, Edward S.; Reichmann, William; Haluska, Frank G.; Kerstein, David; Camidge, D. Ross</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Clinical Oncology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">2490-2500</span>CODEN:
                <span class="NLM_cas:coden">JCONDN</span>;
        ISSN:<span class="NLM_cas:issn">1527-7755</span>.
    
            (<span class="NLM_cas:orgname">American Society of Clinical Oncology</span>)
        </div><div class="casAbstract">Purpose: Most crizotinib-treated patients with anaplastic lymphoma kinase gene (ALK)-rearranged non-smallcell lung cancer (ALK-pos. NSCLC) eventually experience disease progression.  We evaluated two regimens of brigatinib, an investigational next-generation ALK inhibitor, in crizotinib-refractory ALK-pos. NSCLC.  Patients and Methods Patients were stratified by brain metastases and best response to crizotinib.  They were randomly assigned (1:1) to oral brigatinib 90 mg once daily (arm A) or 180 mg once daily with a 7-day lead-in at 90 mg (180 mg once daily [with lead-in]; arm B).  Investigator-assessed confirmed objective response rate (ORR) was the primary end point.  Results Of 222 patients enrolled (arm A: n = 112, 109 treated; arm B: n = 110, 110 treated), 154 (69%) had baseline brain metastases and 164 of 222 (74%) had received prior chemotherapy.  With 8.0-mo median follow-up, investigator-assessed confirmed ORR was 45% (97.5% CI, 34% to 56%) in arm A and 54% (97.5% CI, 43% to 65%) in arm B.  Investigator-assessed median progression-free survival was 9.2 mo (95% CI, 7.4 to 15.6) and 12.9 mo (95% CI, 11.1 to not reached) in arms A and B, resp.  Independent review committee-assessed intracranial ORR in patients with measurable brain metastases at baseline was 42% (11 of 26 patients) in arm A and 67% (12 of 18 patients) in arm B.  Common treatment-emergent adverse events were nausea (arm A/B, 33%/ 40%), diarrhea (arm A/B, 19%/38%), headache (arm A/B, 28%/27%), and cough (arm A/B, 18%/ 34%), and were mainly grades 1 to 2.  A subset of pulmonary adverse events with early onset (median onset: day 2) occurred in 14 of 219 treated patients (all grades, 6%; grade $ 3, 3%); none occurred after escalation to 180 mg in arm B.  Seven of 14 patients were successfully retreated with brigatinib.  Conclusion Brigatinib yielded substantial whole-body and intracranial responses as well as robust progressionfree survival; 180 mg (with lead-in) showed consistently better efficacy than 90 mg, with acceptable safety.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq4RtdJwkNkyLVg90H21EOLACvtfcHk0ljsWTczp5A4iw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitVSqsbfK&md5=efe68d35d297c3d9e70023f887c23a2c</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1200%2FJCO.2016.71.5904&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1200%252FJCO.2016.71.5904%26sid%3Dliteratum%253Aachs%26aulast%3DKim%26aufirst%3DD.-W.%26aulast%3DTiseo%26aufirst%3DM.%26aulast%3DAhn%26aufirst%3DM.-J.%26aulast%3DReckamp%26aufirst%3DK.%2BL.%26aulast%3DHansen%26aufirst%3DK.%2BH.%26aulast%3DKim%26aufirst%3DS.-W.%26aulast%3DHuber%26aufirst%3DR.%2BM.%26aulast%3DWest%26aufirst%3DH.%2BL.%26aulast%3DGroen%26aufirst%3DH.%2BJ.%2BM.%26aulast%3DHochmair%26aufirst%3DM.%2BJ.%26aulast%3DLeighl%26aufirst%3DN.%2BB.%26aulast%3DGettinger%26aufirst%3DS.%2BN.%26aulast%3DLanger%26aufirst%3DC.%2BJ.%26aulast%3DPaz-Ares%2BRodr%25C3%25ADguez%26aufirst%3DL.%2BG.%26aulast%3DSmit%26aufirst%3DE.%2BF.%26aulast%3DKim%26aufirst%3DE.%2BS.%26aulast%3DReichmann%26aufirst%3DW.%26aulast%3DHaluska%26aufirst%3DF.%2BG.%26aulast%3DKerstein%26aufirst%3DD.%26aulast%3DCamidge%26aufirst%3DD.%2BR.%26atitle%3DBrigatinib%2520in%2520Patients%2520With%2520Crizotinib-Refractory%2520Anaplastic%2520Lymphoma%2520Kinase-Positive%2520Non-Small-Cell%2520Lung%2520Cancer%253A%2520A%2520Randomized%252C%2520Multicenter%2520Phase%2520II%2520Trial%26jtitle%3DJ.%2520Clin.%2520Oncol.%26date%3D2017%26volume%3D35%26spage%3D2490%26epage%3D2498%26doi%3D10.1200%2Fjco.2016.71.5904" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gaye, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geier, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bore, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guilloïque, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lucia, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quéré, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gouva, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robinet, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Descourt, R.</span></span> <span> </span><span class="NLM_article-title">Intra-cranial efficacy of brigatinib in an ALK-positive non-small cell lung cancer patient presenting leptomeningeal carcinomatosis</span>. <i>Lung Cancer</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>133</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">3</span>, <span class="refDoi"> DOI: 10.1016/j.lungcan.2019.04.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=10.1016%2Fj.lungcan.2019.04.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=31200813" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=1%3ACAS%3A280%3ADC%252BB3M3mvFWgtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=133&publication_year=2019&pages=1-3&author=E.+Gayeauthor=M.+Geierauthor=P.+Boreauthor=M.+Guillo%C3%AFqueauthor=F.+Luciaauthor=G.+Qu%C3%A9r%C3%A9author=S.+Gouvaauthor=G.+Robinetauthor=R.+Descourt&title=Intra-cranial+efficacy+of+brigatinib+in+an+ALK-positive+non-small+cell+lung+cancer+patient+presenting+leptomeningeal+carcinomatosis&doi=10.1016%2Fj.lungcan.2019.04.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Intra-cranial efficacy of brigatinib in an ALK-positive non-small cell lung cancer patient presenting leptomeningeal carcinomatosis</span></div><div class="casAuthors">Gaye Elisabeth; Geier Margaux; Bore Paul; Quere Gilles; Gouva Sylvie; Robinet Gilles; Descourt Renaud; Guilloique Marine; Lucia Francois</div><div class="citationInfo"><span class="NLM_cas:title">Lung cancer (Amsterdam, Netherlands)</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">133</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">1-3</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">OBJECTIVES:  Brigatinib is a second-generation ALK inhibitor which demonstrated activity over crizotinib-resistance, especially on brain metastasis by increased blood-brain penetration.  However, its activity on lepto-meningeal disease is unknown and scarcely reported.  MATERIALS AND METHODS:  We hereby report the case of lepto-meningeal disease in crizotinib- and ceretinib- treated patient who was successfully treated by brigatinib.  RESULTS:  The patient achieved intracranial response to brigatinib more than 14 months.  CONCLUSION:  Our case provides additional data on brigatinib's intracranial activity, not only on brain metastasis but also on leptomeningeal disease, after experiencing resistance to both crizotinib and ceretinib, 1(st) and 2(nd) generation ALK inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQCRHissLLXoQMthz5_Rx5rfW6udTcc2eZZbUEyqvcbp7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3M3mvFWgtQ%253D%253D&md5=b0af2a7cc0f91694fd8ea6b73c9af473</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1016%2Fj.lungcan.2019.04.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.lungcan.2019.04.013%26sid%3Dliteratum%253Aachs%26aulast%3DGaye%26aufirst%3DE.%26aulast%3DGeier%26aufirst%3DM.%26aulast%3DBore%26aufirst%3DP.%26aulast%3DGuillo%25C3%25AFque%26aufirst%3DM.%26aulast%3DLucia%26aufirst%3DF.%26aulast%3DQu%25C3%25A9r%25C3%25A9%26aufirst%3DG.%26aulast%3DGouva%26aufirst%3DS.%26aulast%3DRobinet%26aufirst%3DG.%26aulast%3DDescourt%26aufirst%3DR.%26atitle%3DIntra-cranial%2520efficacy%2520of%2520brigatinib%2520in%2520an%2520ALK-positive%2520non-small%2520cell%2520lung%2520cancer%2520patient%2520presenting%2520leptomeningeal%2520carcinomatosis%26jtitle%3DLung%2520Cancer%26date%3D2019%26volume%3D133%26spage%3D1%26epage%3D3%26doi%3D10.1016%2Fj.lungcan.2019.04.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lin, H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pan, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hou, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baumann, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hochmair, M. J.</span></span> <span> </span><span class="NLM_article-title">Real-world treatment duration in ALK-positive non-small-cell lung cancer patients receiving brigatinib through the early access program</span>. <i>Future Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">1031</span>– <span class="NLM_lpage">1041</span>, <span class="refDoi"> DOI: 10.2217/fon-2019-0849</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=10.2217%2Ffon-2019-0849" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=32338548" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=1%3ACAS%3A528%3ADC%252BB3cXptVaks70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2020&pages=1031-1041&author=H.+M.+Linauthor=X.+Panauthor=P.+Houauthor=S.+Allenauthor=P.+Baumannauthor=M.+J.+Hochmair&title=Real-world+treatment+duration+in+ALK-positive+non-small-cell+lung+cancer+patients+receiving+brigatinib+through+the+early+access+program&doi=10.2217%2Ffon-2019-0849"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Real-world treatment duration in ALK-positive non-small-cell lung cancer patients receiving brigatinib through the early access program</span></div><div class="casAuthors">Lin, Huamao M.; Pan, Xiaoyun; Hou, Peijie; Allen, Susan; Baumann, Pia; Hochmair, Maximilian J.</div><div class="citationInfo"><span class="NLM_cas:title">Future Oncology</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">1031-1041</span>CODEN:
                <span class="NLM_cas:coden">FOUNBN</span>;
        ISSN:<span class="NLM_cas:issn">1744-8301</span>.
    
            (<span class="NLM_cas:orgname">Future Medicine Ltd.</span>)
        </div><div class="casAbstract">Aim: To assess time-to-treatment discontinuation (TTD) of brigatinib following treatment with ALK tyrosine kinase inhibitor(s) (TKIs) in patients with ALK-pos. (ALK+) non-small-cell lung cancer (NSCLC) receiving brigatinib through the international early access program.  Patients & anal.: Anal. was performed for patients with ALK+ NSCLC treated with prior ALK TKIs, including next-generation ALK TKIs.  Results: Data for 604 patients (21 countries), including patients with prior next-generation ALK TKIs, were reported.  The median TTD of brigatinib in patients with prior crizotinib, alectinib, ceritinib or lorlatinib was 10.0, 8.7, 10.3 and 7.5 mo, resp.  Conclusion: Brigatinib appears to be effective and tolerable in real-world clin. practice regardless of prior treatment with first or NG ALK TKIs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqP0lzoeDBC8bVg90H21EOLACvtfcHk0ljyUfNv0OekvQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXptVaks70%253D&md5=b7677b498bf8359cb0144167bab75ebe</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.2217%2Ffon-2019-0849&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2217%252Ffon-2019-0849%26sid%3Dliteratum%253Aachs%26aulast%3DLin%26aufirst%3DH.%2BM.%26aulast%3DPan%26aufirst%3DX.%26aulast%3DHou%26aufirst%3DP.%26aulast%3DAllen%26aufirst%3DS.%26aulast%3DBaumann%26aufirst%3DP.%26aulast%3DHochmair%26aufirst%3DM.%2BJ.%26atitle%3DReal-world%2520treatment%2520duration%2520in%2520ALK-positive%2520non-small-cell%2520lung%2520cancer%2520patients%2520receiving%2520brigatinib%2520through%2520the%2520early%2520access%2520program%26jtitle%3DFuture%2520Oncol.%26date%3D2020%26volume%3D16%26spage%3D1031%26epage%3D1041%26doi%3D10.2217%2Ffon-2019-0849" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sun, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ren, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kong, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhong, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qiu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, B.</span></span> <span> </span><span class="NLM_article-title">Development of a Brigatinib degrader (SIAIS117) as a potential treatment for ALK positive cancer resistance</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>193</i></span>,  <span class="NLM_fpage">112190</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2020.112190</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=10.1016%2Fj.ejmech.2020.112190" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=32179332" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=1%3ACAS%3A528%3ADC%252BB3cXkvF2qu70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=193&publication_year=2020&pages=112190&author=N.+Sunauthor=C.+Renauthor=Y.+Kongauthor=H.+Zhongauthor=J.+Chenauthor=Y.+Liauthor=J.+Zhangauthor=Y.+Zhouauthor=X.+Qiuauthor=H.+Linauthor=X.+Songauthor=X.+Yangauthor=B.+Jiang&title=Development+of+a+Brigatinib+degrader+%28SIAIS117%29+as+a+potential+treatment+for+ALK+positive+cancer+resistance&doi=10.1016%2Fj.ejmech.2020.112190"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Development of a Brigatinib degrader (SIAIS117) as a potential treatment for ALK positive cancer resistance</span></div><div class="casAuthors">Sun, Ning; Ren, Chaowei; Kong, Ying; Zhong, Hui; Chen, Jinju; Li, Yan; Zhang, Jianshui; Zhou, Yuedong; Qiu, Xing; Lin, Haifan; Song, Xiaoling; Yang, Xiaobao; Jiang, Biao</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">193</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">112190</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">EML4-ALK and NPM-ALK fusion proteins possess constitutively activated ALK (anaplastic lymphoma kinase) activity, which in turn leads to the development of non-small cell lung cancer and anaplastic large-cell lymphomas (ALCLs).  FDA-approved ALK inhibitor drugs cause significant cancer regression.  However, drug resistance eventually occurs and it becomes a big obstacle in clinic.  Novel proteolysis targeting chimera (PROTAC) technol. platform provides a potential therapeutic strategy for drug resistance.  Herein, we designed and synthesized a series of ALK PROTACs based on Brigatinib and VHL-1 conjunction, and screened SIAIS117 as the best degrader which not only blocked the growth of SR and H2228 cancer cell lines, but also degraded ALK protein.  In addn., SIAIS117 also showed much better growth inhibition effect than Brigatinib on 293T cell line that exogenously expressed G1202R-resistant ALK proteins.  Furthermore, it also degraded G1202R mutant ALK protein in vitro.  At last, it has the potentially anti-proliferation ability of small cell lung cancer.  Thus, we have successfully generated the degrader SIAIS117 that can potentially overcome resistance in cancer targeted therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrX9rzk88HMe7Vg90H21EOLACvtfcHk0ljyUfNv0OekvQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXkvF2qu70%253D&md5=61f0228111c806ae62473fc450fce276</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2020.112190&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2020.112190%26sid%3Dliteratum%253Aachs%26aulast%3DSun%26aufirst%3DN.%26aulast%3DRen%26aufirst%3DC.%26aulast%3DKong%26aufirst%3DY.%26aulast%3DZhong%26aufirst%3DH.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DZhou%26aufirst%3DY.%26aulast%3DQiu%26aufirst%3DX.%26aulast%3DLin%26aufirst%3DH.%26aulast%3DSong%26aufirst%3DX.%26aulast%3DYang%26aufirst%3DX.%26aulast%3DJiang%26aufirst%3DB.%26atitle%3DDevelopment%2520of%2520a%2520Brigatinib%2520degrader%2520%2528SIAIS117%2529%2520as%2520a%2520potential%2520treatment%2520for%2520ALK%2520positive%2520cancer%2520resistance%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2020%26volume%3D193%26spage%3D112190%26doi%3D10.1016%2Fj.ejmech.2020.112190" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Qiu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kong, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, B.</span></span> <span> </span><span class="NLM_article-title">Chemoselective Synthesis of Lenalidomide-Based PROTAC Library Using Alkylation Reaction</span>. <i>Org. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">3838</span>– <span class="NLM_lpage">3841</span>, <span class="refDoi"> DOI: 10.1021/acs.orglett.9b01326</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.orglett.9b01326" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=1%3ACAS%3A528%3ADC%252BC1MXpt1WjsbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2019&pages=3838-3841&author=X.+Qiuauthor=N.+Sunauthor=Y.+Kongauthor=Y.+Liauthor=X.+Yangauthor=B.+Jiang&title=Chemoselective+Synthesis+of+Lenalidomide-Based+PROTAC+Library+Using+Alkylation+Reaction&doi=10.1021%2Facs.orglett.9b01326"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Chemoselective Synthesis of Lenalidomide-Based PROTAC Library Using Alkylation Reaction</span></div><div class="casAuthors">Qiu, Xing; Sun, Ning; Kong, Ying; Li, Yan; Yang, Xiaobao; Jiang, Biao</div><div class="citationInfo"><span class="NLM_cas:title">Organic Letters</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">3838-3841</span>CODEN:
                <span class="NLM_cas:coden">ORLEF7</span>;
        ISSN:<span class="NLM_cas:issn">1523-7052</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Using DIPEA as base in N-methyl-2-pyrrolidinone (NMP), lenalidomide underwent regioselective alkylation with bromoesters and Boc-protected bromoamines followed by deprotection to yield lenalidomide derivs. as a library of potential PROTAC reagents.  One of the products was coupled to the known BET binding agent JQ1 to yield I; I degraded BET protein in cells and effectively inhibited cancer cell proliferation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWwAx0Aar9k7Vg90H21EOLACvtfcHk0ljyUfNv0OekvQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXpt1WjsbY%253D&md5=b185afe1107a03f13560e4cb44223026</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Facs.orglett.9b01326&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.orglett.9b01326%26sid%3Dliteratum%253Aachs%26aulast%3DQiu%26aufirst%3DX.%26aulast%3DSun%26aufirst%3DN.%26aulast%3DKong%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DYang%26aufirst%3DX.%26aulast%3DJiang%26aufirst%3DB.%26atitle%3DChemoselective%2520Synthesis%2520of%2520Lenalidomide-Based%2520PROTAC%2520Library%2520Using%2520Alkylation%2520Reaction%26jtitle%3DOrg.%2520Lett.%26date%3D2019%26volume%3D21%26spage%3D3838%26epage%3D3841%26doi%3D10.1021%2Facs.orglett.9b01326" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, X.-R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiong, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span> <span> </span><span class="NLM_article-title">Proteolysis Targeting Chimeras (PROTACs) of Anaplastic Lymphoma Kinase (ALK)</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>151</i></span>,  <span class="NLM_fpage">304</span>– <span class="NLM_lpage">314</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2018.03.071</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=10.1016%2Fj.ejmech.2018.03.071" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=29627725" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=1%3ACAS%3A528%3ADC%252BC1cXnt1ejs7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=151&publication_year=2018&pages=304-314&author=C.+Zhangauthor=X.-R.+Hanauthor=X.+Yangauthor=B.+Jiangauthor=J.+Liuauthor=Y.+Xiongauthor=J.+Jin&title=Proteolysis+Targeting+Chimeras+%28PROTACs%29+of+Anaplastic+Lymphoma+Kinase+%28ALK%29&doi=10.1016%2Fj.ejmech.2018.03.071"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Proteolysis Targeting Chimeras (PROTACs) of Anaplastic Lymphoma Kinase (ALK)</span></div><div class="casAuthors">Zhang, Chengwei; Han, Xiao-Ran; Yang, Xiaobao; Jiang, Biao; Liu, Jing; Xiong, Yue; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">151</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">304-314</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">Anaplastic lymphoma kinase (ALK) activation has been assocd. with many types of human cancer.  Significant efforts have been devoted to the development of ALK inhibitors to antagonize the kinase activity of ALK.  Four ALK inhibitors have been approved by the FDA to date for treating patients with ALK-pos. nonsmall cell lung cancers (NSCLC).  However, drug resistance has been obsd. in the majority of patients treated with these inhibitors.  New therapeutic strategies (e.g., compds. with novel mechanisms of action) are needed to overcome the drug resistance issue.  The emerging PROTAC (Proteolysis Targeting Chimera) technol. has been successfully applied to selective degrdn. of multiple protein targets, but not ALK.  Since ALK protein levels are not important for viability in mammals, ALK PROTACs could lead to novel therapeutics with minimal toxicity.  Here the authors report the design, synthesis and biol. evaluation of novel PROTACs (degraders) of ALK.  MS4077 (I) and MS4078 (II) potently decreased cellular levels of oncogenic active ALK fusion proteins in a concn.- and time-dependent manner in SU-DHL-1 lymphoma and NCI-H2228 lung cancer cells.  The ALK protein degrdn. induced by compds. I and II was cereblon and proteasome dependent.  In addn., compds. I and II potently inhibited proliferation of SU-DHL-1 cells.  Furthermore, compd. II displayed good plasma exposure in a mouse pharmacokinetic study, thus is suitable for in vivo efficacy studies.  The authors also developed MS4748 and MS4740 , very close analogs of I and II resp., which are incapable to degrade the ALK fusion proteins, as neg. controls.  All 4 compds. are valuable chem. tools for investigating effects of ALK pharmacol. degrdn.  The study paved the way for developing the next generation of ALK PROTACs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPGM3aAZZ6h7Vg90H21EOLACvtfcHk0liNBJ2AZP5mbw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXnt1ejs7Y%253D&md5=804324b95034f38b46e45f89e18e55be</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2018.03.071&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2018.03.071%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DC.%26aulast%3DHan%26aufirst%3DX.-R.%26aulast%3DYang%26aufirst%3DX.%26aulast%3DJiang%26aufirst%3DB.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DXiong%26aufirst%3DY.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DProteolysis%2520Targeting%2520Chimeras%2520%2528PROTACs%2529%2520of%2520Anaplastic%2520Lymphoma%2520Kinase%2520%2528ALK%2529%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2018%26volume%3D151%26spage%3D304%26epage%3D314%26doi%3D10.1016%2Fj.ejmech.2018.03.071" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Piya, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhattacharya, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lorenzi, P. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McQueen, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruvolo, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baran, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qian, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Konopleva, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ishizawa, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">You, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kantarjian, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andreeff, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borthakur, G.</span></span> <span> </span><span class="NLM_article-title">BETP degradation simultaneously targets acute myelogenous leukemia stem cells and the microenvironment</span>. <i>J. Clin. Invest.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>129</i></span>,  <span class="NLM_fpage">1878</span>– <span class="NLM_lpage">1894</span>, <span class="refDoi"> DOI: 10.1172/jci120654</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=10.1172%2FJCI120654" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=30829648" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=1%3ACAS%3A280%3ADC%252BB3cbgtVSiuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=129&publication_year=2019&pages=1878-1894&author=S.+Piyaauthor=H.+Muauthor=S.+Bhattacharyaauthor=P.+L.+Lorenziauthor=R.+E.+Davisauthor=T.+McQueenauthor=V.+Ruvoloauthor=N.+Baranauthor=Z.+Wangauthor=Y.+Qianauthor=C.+M.+Crewsauthor=M.+Konoplevaauthor=J.+Ishizawaauthor=M.+J.+Youauthor=H.+Kantarjianauthor=M.+Andreeffauthor=G.+Borthakur&title=BETP+degradation+simultaneously+targets+acute+myelogenous+leukemia+stem+cells+and+the+microenvironment&doi=10.1172%2Fjci120654"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">BETP degradation simultaneously targets acute myelogenous leukemia stem cells and the microenvironment</span></div><div class="casAuthors">Piya Sujan; Mu Hong; Bhattacharya Seemana; McQueen Teresa; Ruvolo Vivian; Baran Natalia; Andreeff Michael; Piya Sujan; Mu Hong; Bhattacharya Seemana; McQueen Teresa; Ruvolo Vivian; Baran Natalia; Konopleva Marina; Ishizawa Jo; Kantarjian Hagop; Andreeff Michael; Borthakur Gautam; Lorenzi Philip L; Davis R Eric; Wang Zhiqiang; Qian Yimin; Crews Craig M; You M James</div><div class="citationInfo"><span class="NLM_cas:title">The Journal of clinical investigation</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">129</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1878-1894</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Anti-leukemic effect of BET/BRD4 (BETP) protein inhibition has been largely attributed to transcriptional downregulation of cellular anabolic/anti-apoptotic processes but its effect on bone marrow microenvironment, a sanctuary favoring persistence of leukemia stem/progenitor cells, is unexplored.  Sustained degradation of BETP with small-molecule BET proteolysis-targeting chimera (PROTAC), ARV-825, resulted in marked downregulation of surface CXCR4 and CD44, key proteins in leukemia-microenvironment interaction, in AML cells.  Abrogation of surface CXCR4 expression impaired SDF-1α directed migration and was mediated through transcriptional down-regulation of PIM1 kinase that in turn phosphorylates CXCR4 and facilitates its surface localization.  Down-regulation of CD44/CD44v8-10 impaired cystine uptake, lowered intracellular reduced glutathione and increased oxidative stress.  More importantly, BETP degradation markedly decreased CD34+CD38-CD90-CD45RA+ leukemic stem cell population and alone or in combination with Cytarabine, prolonged survival in mouse model of human leukemia including AML-PDX.  Gene expression profiling and single cell proteomics confirmed down regulation of the gene signatures associated with 'stemness' in AML and Wnt/β-catenin, Myc pathways.  Hence, BETP degradation by ARV-825 simultaneously targets cell intrinsic signaling, stromal interactions and metabolism in AML.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcT9Wy3XYe-G80bxh97K6Vt6fW6udTcc2ebgBC_EDegyXLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3cbgtVSiuw%253D%253D&md5=6f60455bcebfab2cba3ad3b5e68b022a</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1172%2FJCI120654&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1172%252FJCI120654%26sid%3Dliteratum%253Aachs%26aulast%3DPiya%26aufirst%3DS.%26aulast%3DMu%26aufirst%3DH.%26aulast%3DBhattacharya%26aufirst%3DS.%26aulast%3DLorenzi%26aufirst%3DP.%2BL.%26aulast%3DDavis%26aufirst%3DR.%2BE.%26aulast%3DMcQueen%26aufirst%3DT.%26aulast%3DRuvolo%26aufirst%3DV.%26aulast%3DBaran%26aufirst%3DN.%26aulast%3DWang%26aufirst%3DZ.%26aulast%3DQian%26aufirst%3DY.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26aulast%3DKonopleva%26aufirst%3DM.%26aulast%3DIshizawa%26aufirst%3DJ.%26aulast%3DYou%26aufirst%3DM.%2BJ.%26aulast%3DKantarjian%26aufirst%3DH.%26aulast%3DAndreeff%26aufirst%3DM.%26aulast%3DBorthakur%26aufirst%3DG.%26atitle%3DBETP%2520degradation%2520simultaneously%2520targets%2520acute%2520myelogenous%2520leukemia%2520stem%2520cells%2520and%2520the%2520microenvironment%26jtitle%3DJ.%2520Clin.%2520Invest.%26date%3D2019%26volume%3D129%26spage%3D1878%26epage%3D1894%26doi%3D10.1172%2Fjci120654" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shibata, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shimokawa, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nagai, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohoka, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hattori, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miyamoto, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ujikawa, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sameshima, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nara, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cho, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Naito, M.</span></span> <span> </span><span class="NLM_article-title">Pharmacological difference between degrader and inhibitor against oncogenic BCR-ABL kinase</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">13549</span>, <span class="refDoi"> DOI: 10.1038/s41598-018-31913-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=10.1038%2Fs41598-018-31913-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=30202081" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=1%3ACAS%3A280%3ADC%252BB3c3nsFSltg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2018&pages=13549&author=N.+Shibataauthor=K.+Shimokawaauthor=K.+Nagaiauthor=N.+Ohokaauthor=T.+Hattoriauthor=N.+Miyamotoauthor=O.+Ujikawaauthor=T.+Sameshimaauthor=H.+Naraauthor=N.+Choauthor=M.+Naito&title=Pharmacological+difference+between+degrader+and+inhibitor+against+oncogenic+BCR-ABL+kinase&doi=10.1038%2Fs41598-018-31913-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Pharmacological difference between degrader and inhibitor against oncogenic BCR-ABL kinase</span></div><div class="casAuthors">Shibata Norihito; Ohoka Nobumichi; Hattori Takayuki; Naito Mikihiko; Shimokawa Kenichiro; Nagai Katsunori; Miyamoto Naoki; Ujikawa Osamu; Sameshima Tomoya; Nara Hiroshi; Cho Nobuo; Nagai Katsunori; Ujikawa Osamu; Nara Hiroshi; Cho Nobuo</div><div class="citationInfo"><span class="NLM_cas:title">Scientific reports</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">13549</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Chronic myelogenous leukemia (CML) is characterized by the oncogenic fusion protein, BCR-ABL protein kinase, against which clinically useful inhibitors have been developed.  An alternative approach to treat CML is to degrade the BCR-ABL protein.  Recently, potent degraders against BCR-ABL have been developed by conjugating dasatinib to ligands for E3 ubiquitin ligases.  Since the degraders contain the dasatinib moiety, they also inhibit BCR-ABL kinase activity, which complicates our understanding of the impact of BCR-ABL degradation by degraders in CML growth inhibition.  To address this issue, we chose DAS-IAP, as a potent BCR-ABL degrader, and developed a structurally related inactive degrader, DAS-meIAP, which inhibits kinase activity but does not degrade the BCR-ABL protein.  DAS-IAP showed slightly weaker activity than DAS-meIAP in inhibiting cell growth when CML cells were treated for 48 h.  However, DAS-IAP showed sustained growth inhibition even when the drug was removed after short-term treatment, whereas CML cell growth rapidly resumed following removal of DAS-meIAP and dasatinib.  Consistently, suppression of BCR-ABL levels and downstream kinase signaling were maintained after DAS-IAP removal, whereas kinase signaling rapidly recovered following removal of DAS-meIAP and dasatinib.  These results indicate that BCR-ABL degrader shows more sustained inhibition of CML cell growth than ABL kinase inhibitor.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcS1tFvEwEman0cLvDNbOD4DfW6udTcc2ebgBC_EDegyXLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3c3nsFSltg%253D%253D&md5=f55ad999fd61e89babe3410fbcd763e4</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1038%2Fs41598-018-31913-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41598-018-31913-5%26sid%3Dliteratum%253Aachs%26aulast%3DShibata%26aufirst%3DN.%26aulast%3DShimokawa%26aufirst%3DK.%26aulast%3DNagai%26aufirst%3DK.%26aulast%3DOhoka%26aufirst%3DN.%26aulast%3DHattori%26aufirst%3DT.%26aulast%3DMiyamoto%26aufirst%3DN.%26aulast%3DUjikawa%26aufirst%3DO.%26aulast%3DSameshima%26aufirst%3DT.%26aulast%3DNara%26aufirst%3DH.%26aulast%3DCho%26aufirst%3DN.%26aulast%3DNaito%26aufirst%3DM.%26atitle%3DPharmacological%2520difference%2520between%2520degrader%2520and%2520inhibitor%2520against%2520oncogenic%2520BCR-ABL%2520kinase%26jtitle%3DSci.%2520Rep.%26date%3D2018%26volume%3D8%26spage%3D13549%26doi%3D10.1038%2Fs41598-018-31913-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Steeg, P. S.</span></span> <span> </span><span class="NLM_article-title">Targeting metastasis</span>. <i>Nat. Rev. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">201</span>– <span class="NLM_lpage">218</span>, <span class="refDoi"> DOI: 10.1038/nrc.2016.25</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=10.1038%2Fnrc.2016.25" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=27009393" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=1%3ACAS%3A528%3ADC%252BC28XkslKnsLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2016&pages=201-218&author=P.+S.+Steeg&title=Targeting+metastasis&doi=10.1038%2Fnrc.2016.25"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting metastasis</span></div><div class="casAuthors">Steeg, Patricia S.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">201-218</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Tumor metastasis, the movement of tumor cells from a primary site to progressively colonize distant organs, is a major contributor to the deaths of cancer patients.  Therapeutic goals are the prevention of an initial metastasis in high-risk patients, shrinkage of established lesions and prevention of addnl. metastases in patients with limited disease.  Instead of being autonomous, tumor cells engage in bidirectional interactions with metastatic microenvironments to alter antitumor immunity, the extracellular milieu, genomic stability, survival signalling, chemotherapeutic resistance and proliferative cycles.  Can targeting of these interactions significantly improve patient outcomes.  In this Review preclin. research, combination therapies and clin. trial designs are re-examd.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqjUyWsbBasb7Vg90H21EOLACvtfcHk0ljGAQhTEn2a4w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XkslKnsLo%253D&md5=86df7851bdfa0d97e351494a876e9bb7</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1038%2Fnrc.2016.25&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc.2016.25%26sid%3Dliteratum%253Aachs%26aulast%3DSteeg%26aufirst%3DP.%2BS.%26atitle%3DTargeting%2520metastasis%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2016%26volume%3D16%26spage%3D201%26epage%3D218%26doi%3D10.1038%2Fnrc.2016.25" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Roelle, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosse, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buech, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chubanov, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gudermann, T.</span></span> <span> </span><span class="NLM_article-title">Essential role of Pyk2 and Src kinase activation in neuropeptide-induced proliferation of small cell lung cancer cells</span>. <i>Oncogene</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">1737</span>– <span class="NLM_lpage">1748</span>, <span class="refDoi"> DOI: 10.1038/sj.onc.1210819</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=10.1038%2Fsj.onc.1210819" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=17906699" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=1%3ACAS%3A528%3ADC%252BD1cXjtFOnsLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2008&pages=1737-1748&author=S.+Roelleauthor=R.+Grosseauthor=T.+Buechauthor=V.+Chubanovauthor=T.+Gudermann&title=Essential+role+of+Pyk2+and+Src+kinase+activation+in+neuropeptide-induced+proliferation+of+small+cell+lung+cancer+cells&doi=10.1038%2Fsj.onc.1210819"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Essential role of Pyk2 and Src kinase activation in neuropeptide-induced proliferation of small cell lung cancer cells</span></div><div class="casAuthors">Roelle, S.; Grosse, R.; Buech, T.; Chubanov, V.; Gudermann, T.</div><div class="citationInfo"><span class="NLM_cas:title">Oncogene</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1737-1748</span>CODEN:
                <span class="NLM_cas:coden">ONCNES</span>;
        ISSN:<span class="NLM_cas:issn">0950-9232</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Neuropeptide hormones like bombesin/gastrin-releasing peptide, galanin or bradykinin, acting via auto and paracrine growth loops, represent the principal mitogens of small cell lung cancer (SCLC).  These mitogenic neuropeptides activate Gq/11-coupled receptors which stimulate phospholipase Cβ activity, followed by rises of the intracellular calcium concn. ([Ca2+]i) and activation of protein kinase C (PKC).  We report here that proline-rich tyrosine kinase Pyk2 is highly expressed in SCLC cells and provides a functional link between neuropeptide-induced increases in [Ca2+]i and tumor cell proliferation.  Activation of Pyk2 and its assocn. with Src kinases critically depends on the elevation of [Ca2+]i, but is independent of PKC.  Src kinase activities are crucial for neuropeptide-mediated GTP-loading of Ras and activation of extracellular signal-regulated kinases in SCLC cells.  Pyk2 and Src kinases essentially contribute to anchorage-independent proliferation of SCLC cells.  Inhibition of either Pyk2 or Src kinases by lentiviral RNAi or pharmacol. inhibition with PP2, resp., attenuated basal and neuropeptide-elicited survival and proliferation of SCLC cells in liq. culture and in soft agar.  Thus, neuropeptides stimulate anchorage-independent survival and proliferation of SCLC cells via pathways involving Pyk2 and Src kinases.  Therefore, Ca2+-induced Pyk2/Src complex formation may be a rewarding mol. target for novel therapeutic strategies in SCLC cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpbVsMd34QCG7Vg90H21EOLACvtfcHk0ljGAQhTEn2a4w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXjtFOnsLs%253D&md5=f8470f0d6c8bf1df80ca003fe507f726</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1038%2Fsj.onc.1210819&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.onc.1210819%26sid%3Dliteratum%253Aachs%26aulast%3DRoelle%26aufirst%3DS.%26aulast%3DGrosse%26aufirst%3DR.%26aulast%3DBuech%26aufirst%3DT.%26aulast%3DChubanov%26aufirst%3DV.%26aulast%3DGudermann%26aufirst%3DT.%26atitle%3DEssential%2520role%2520of%2520Pyk2%2520and%2520Src%2520kinase%2520activation%2520in%2520neuropeptide-induced%2520proliferation%2520of%2520small%2520cell%2520lung%2520cancer%2520cells%26jtitle%3DOncogene%26date%3D2008%26volume%3D27%26spage%3D1737%26epage%3D1748%26doi%3D10.1038%2Fsj.onc.1210819" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kuang, B.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, M.-Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, L.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, L.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H.-B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, W.-F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Du, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span> <span> </span><span class="NLM_article-title">Proline-rich tyrosine kinase 2 and its phosphorylated form pY881 are novel prognostic markers for non-small-cell lung cancer progression and patients’ overall survival</span>. <i>Br. J. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>109</i></span>,  <span class="NLM_fpage">1252</span>– <span class="NLM_lpage">1263</span>, <span class="refDoi"> DOI: 10.1038/bjc.2013.439</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=10.1038%2Fbjc.2013.439" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=23922106" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsVSktbjI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=109&publication_year=2013&pages=1252-1263&author=B.-H.+Kuangauthor=M.-Q.+Zhangauthor=L.-H.+Xuauthor=L.-J.+Huauthor=H.-B.+Wangauthor=W.-F.+Zhaoauthor=Y.+Duauthor=X.+Zhang&title=Proline-rich+tyrosine+kinase+2+and+its+phosphorylated+form+pY881+are+novel+prognostic+markers+for+non-small-cell+lung+cancer+progression+and+patients%E2%80%99+overall+survival&doi=10.1038%2Fbjc.2013.439"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Proline-rich tyrosine kinase 2 and its phosphorylated form pY881 are novel prognostic markers for non-small-cell lung cancer progression and patients' overall survival</span></div><div class="casAuthors">Kuang, B.-H.; Zhang, M.-Q.; Xu, L.-H.; Hu, L.-J.; Wang, H.-B.; Zhao, W.-F.; Du, Y.; Zhang, X.</div><div class="citationInfo"><span class="NLM_cas:title">British Journal of Cancer</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">109</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1252-1263</span>CODEN:
                <span class="NLM_cas:coden">BJCAAI</span>;
        ISSN:<span class="NLM_cas:issn">0007-0920</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Background: Our previous study revealed that proline-rich tyrosine kinase 2 (Pyk2) is implicated in both anchorage-independent growth and anoikis resistance in lung cancer cells.  This study aims to explore the expression and clin. significance of Pyk2 and its phosphorylated forms in non-small-cell lung cancer (NSCLC).  Methods: The mRNA and protein levels of Pyk2 or cancer stem cell markers (ALDH1a1, ABCG2 and Bmi-1) were either examd. by reverse transcription-PCR or western blotting.  An immunohistochem. (IHC) assay was conducted to analyze the expression of Pyk2 and its phosphorylated forms in 128 NSCLC cases.  Results: The levels of Pyk2 mRNA, total protein, and its phosphorylated form pY881 were higher in lung cancer lesions than in the paired noncancerous tissues.  The IHC anal. showed the levels of the Pyk2 and Pyk2[pY881] proteins were highly expressed in 70 (54.7%) and 77 (60.2%) cases, resp.  Both Pyk2 and Pyk2[pY881] were independent prognostic factors for NSCLC patients.  The gain and loss study of Pyk2 function revealed that Pyk2 could upregulate the expression of ALDH1a1, ABCG2 and Bmi-1 and enhance the ability of colony formation in soft agar assay in A549 and H460 cells.  Conclusion: Both Pyk2 and phosphorylated Pyk2[pY881] are potential prognostic factors and therapeutic targets for NSCLC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrBW_TkawEATLVg90H21EOLACvtfcHk0ljGAQhTEn2a4w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsVSktbjI&md5=ca5d562b59b1c6fce42ef30acf669807</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1038%2Fbjc.2013.439&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fbjc.2013.439%26sid%3Dliteratum%253Aachs%26aulast%3DKuang%26aufirst%3DB.-H.%26aulast%3DZhang%26aufirst%3DM.-Q.%26aulast%3DXu%26aufirst%3DL.-H.%26aulast%3DHu%26aufirst%3DL.-J.%26aulast%3DWang%26aufirst%3DH.-B.%26aulast%3DZhao%26aufirst%3DW.-F.%26aulast%3DDu%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DX.%26atitle%3DProline-rich%2520tyrosine%2520kinase%25202%2520and%2520its%2520phosphorylated%2520form%2520pY881%2520are%2520novel%2520prognostic%2520markers%2520for%2520non-small-cell%2520lung%2520cancer%2520progression%2520and%2520patients%25E2%2580%2599%2520overall%2520survival%26jtitle%3DBr.%2520J.%2520Cancer%26date%3D2013%26volume%3D109%26spage%3D1252%26epage%3D1263%26doi%3D10.1038%2Fbjc.2013.439" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moschetta, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huynh, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tai, Y.-T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mishima, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ring, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tam, W. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maiso, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reagan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sahin, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sacco, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manier, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aljawai, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glavey, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Munshi, N. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anderson, K. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pachter, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roccaro, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ghobrial, I. M.</span></span> <span> </span><span class="NLM_article-title">Pyk2 promotes tumor progression in multiple myeloma</span>. <i>Blood</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>124</i></span>,  <span class="NLM_fpage">2675</span>– <span class="NLM_lpage">2686</span>, <span class="refDoi"> DOI: 10.1182/blood-2014-03-563981</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=10.1182%2Fblood-2014-03-563981" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=25217697" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvVygsrjO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=124&publication_year=2014&pages=2675-2686&author=Y.+Zhangauthor=M.+Moschettaauthor=D.+Huynhauthor=Y.-T.+Taiauthor=Y.+Zhangauthor=W.+Zhangauthor=Y.+Mishimaauthor=J.+E.+Ringauthor=W.+F.+Tamauthor=Q.+Xuauthor=P.+Maisoauthor=M.+Reaganauthor=I.+Sahinauthor=A.+Saccoauthor=S.+Manierauthor=Y.+Aljawaiauthor=S.+Glaveyauthor=N.+C.+Munshiauthor=K.+C.+Andersonauthor=J.+Pachterauthor=A.+M.+Roccaroauthor=I.+M.+Ghobrial&title=Pyk2+promotes+tumor+progression+in+multiple+myeloma&doi=10.1182%2Fblood-2014-03-563981"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Pyk2 promotes tumor progression in multiple myeloma</span></div><div class="casAuthors">Zhang, Yu; Moschetta, Michele; Huynh, Daisy; Tai, Yu-Tzu; Zhang, Yong; Zhang, Wenjing; Mishima, Yuji; Ring, Jennifer E.; Tam, Winnie F.; Xu, Qunli; Maiso, Patricia; Reagan, Michaela; Sahin, Ilyas; Sacco, Antonio; Manier, Salomon; Aljawai, Yosra; Glavey, Siobhan; Munshi, Nikhil C.; Anderson, Kenneth C.; Pachter, Jonathan; Roccaro, Aldo M.; Ghobrial, Irene M.</div><div class="citationInfo"><span class="NLM_cas:title">Blood</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">124</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">2675-2686</span>CODEN:
                <span class="NLM_cas:coden">BLOOAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-4971</span>.
    
            (<span class="NLM_cas:orgname">American Society of Hematology</span>)
        </div><div class="casAbstract">Proline-rich tyrosine kinase 2 (Pyk2) is a member of the focal adhesion kinase family that has been recently linked to tumor development.  However, its role in modulating multiple myeloma (MM) biol. and disease progression remains unexplored.  We first demonstrated that patients with MM present with higher expression of Pyk2 compared with healthy individuals.  By using loss-of-function approaches, we found that Pyk2 inhibition led to redn. of MM tumor growth in vivo as well as decreased cell proliferation, cell-cycle progression, and adhesion ability in vitro.  In turn, overexpression of Pyk2 promoted the malignant phenotype, substantiated by enhanced tumor growth and reduced survival.  Mechanistically, inhibition of Pyk2 reduced activation of Wnt/β-catenin signaling by destabilizing β-catenin, leading to downregulation of c-Myc and Cyclin D1.  Furthermore, treatment of MM cells with the FAK/Pyk2 inhibitor VS-4718 effectively inhibited MM cell growth both in vitro and in vivo.  Collectively, our findings describe the tumor-promoting role of Pyk2 in MM, thus providing mol. evidence for a novel tyrosine kinase inhibitor as a new therapeutic option in MM.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq7K0Ip0dTGuLVg90H21EOLACvtfcHk0li1S21C0WQTGQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvVygsrjO&md5=a1af2c2927709cfed992c8626738c849</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1182%2Fblood-2014-03-563981&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1182%252Fblood-2014-03-563981%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DMoschetta%26aufirst%3DM.%26aulast%3DHuynh%26aufirst%3DD.%26aulast%3DTai%26aufirst%3DY.-T.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DW.%26aulast%3DMishima%26aufirst%3DY.%26aulast%3DRing%26aufirst%3DJ.%2BE.%26aulast%3DTam%26aufirst%3DW.%2BF.%26aulast%3DXu%26aufirst%3DQ.%26aulast%3DMaiso%26aufirst%3DP.%26aulast%3DReagan%26aufirst%3DM.%26aulast%3DSahin%26aufirst%3DI.%26aulast%3DSacco%26aufirst%3DA.%26aulast%3DManier%26aufirst%3DS.%26aulast%3DAljawai%26aufirst%3DY.%26aulast%3DGlavey%26aufirst%3DS.%26aulast%3DMunshi%26aufirst%3DN.%2BC.%26aulast%3DAnderson%26aufirst%3DK.%2BC.%26aulast%3DPachter%26aufirst%3DJ.%26aulast%3DRoccaro%26aufirst%3DA.%2BM.%26aulast%3DGhobrial%26aufirst%3DI.%2BM.%26atitle%3DPyk2%2520promotes%2520tumor%2520progression%2520in%2520multiple%2520myeloma%26jtitle%3DBlood%26date%3D2014%26volume%3D124%26spage%3D2675%26epage%3D2686%26doi%3D10.1182%2Fblood-2014-03-563981" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hegde, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knolhoff, B. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herndon, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meyer, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nywening, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hawkins, W. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shapiro, I. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weaver, D. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pachter, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang-Gillam, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DeNardo, D. G.</span></span> <span> </span><span class="NLM_article-title">Targeting focal adhesion kinase renders pancreatic cancers responsive to checkpoint immunotherapy</span>. <i>Nat. Med.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">851</span>– <span class="NLM_lpage">860</span>, <span class="refDoi"> DOI: 10.1038/nm.4123</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=10.1038%2Fnm.4123" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=27376576" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtV2msLfN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2016&pages=851-860&author=H.+Jiangauthor=S.+Hegdeauthor=B.+L.+Knolhoffauthor=Y.+Zhuauthor=J.+M.+Herndonauthor=M.+A.+Meyerauthor=T.+M.+Nyweningauthor=W.+G.+Hawkinsauthor=I.+M.+Shapiroauthor=D.+T.+Weaverauthor=J.+A.+Pachterauthor=A.+Wang-Gillamauthor=D.+G.+DeNardo&title=Targeting+focal+adhesion+kinase+renders+pancreatic+cancers+responsive+to+checkpoint+immunotherapy&doi=10.1038%2Fnm.4123"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting focal adhesion kinase renders pancreatic cancers responsive to checkpoint immunotherapy</span></div><div class="casAuthors">Jiang, Hong; Hegde, Samarth; Knolhoff, Brett L.; Zhu, Yu; Herndon, John M.; Meyer, Melissa A.; Nywening, Timothy M.; Hawkins, William G.; Shapiro, Irina M.; Weaver, David T.; Pachter, Jonathan A.; Wang-Gillam, Andrea; DeNardo, David G.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Medicine (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">851-860</span>CODEN:
                <span class="NLM_cas:coden">NAMEFI</span>;
        ISSN:<span class="NLM_cas:issn">1078-8956</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Single-agent immunotherapy has achieved limited clin. benefit to date in patients with pancreatic ductal adenocarcinoma (PDAC).  This may be a result of the presence of a uniquely immunosuppressive tumor microenvironment (TME).  Crit. obstacles to immunotherapy in PDAC tumors include a high no. of tumor-assocd. immunosuppressive cells and a uniquely desmoplastic stroma that functions as a barrier to T cell infiltration.  We identified hyperactivated focal adhesion kinase (FAK) activity in neoplastic PDAC cells as an important regulator of the fibrotic and immunosuppressive TME.  We found that FAK activity was elevated in human PDAC tissues and correlated with high levels of fibrosis and poor CD8+ cytotoxic T cell infiltration.  Single-agent FAK inhibition using the selective FAK inhibitor VS-4718 substantially limited tumor progression, resulting in a doubling of survival in the p48-Cre;LSL-KrasG12D;Trp53flox/+ (KPC) mouse model of human PDAC.  This delay in tumor progression was assocd. with markedly reduced tumor fibrosis and decreased nos. of tumor-infiltrating immunosuppressive cells.  We also found that FAK inhibition rendered the previously unresponsive KPC mouse model responsive to T cell immunotherapy and PD-1 antagonists.  These data suggest that FAK inhibition increases immune surveillance by overcoming the fibrotic and immunosuppressive PDAC TME and renders tumors responsive to immunotherapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp7vp95t6jl9bVg90H21EOLACvtfcHk0li1S21C0WQTGQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtV2msLfN&md5=1f715175ab16c97f696999001422df4c</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1038%2Fnm.4123&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnm.4123%26sid%3Dliteratum%253Aachs%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DHegde%26aufirst%3DS.%26aulast%3DKnolhoff%26aufirst%3DB.%2BL.%26aulast%3DZhu%26aufirst%3DY.%26aulast%3DHerndon%26aufirst%3DJ.%2BM.%26aulast%3DMeyer%26aufirst%3DM.%2BA.%26aulast%3DNywening%26aufirst%3DT.%2BM.%26aulast%3DHawkins%26aufirst%3DW.%2BG.%26aulast%3DShapiro%26aufirst%3DI.%2BM.%26aulast%3DWeaver%26aufirst%3DD.%2BT.%26aulast%3DPachter%26aufirst%3DJ.%2BA.%26aulast%3DWang-Gillam%26aufirst%3DA.%26aulast%3DDeNardo%26aufirst%3DD.%2BG.%26atitle%3DTargeting%2520focal%2520adhesion%2520kinase%2520renders%2520pancreatic%2520cancers%2520responsive%2520to%2520checkpoint%2520immunotherapy%26jtitle%3DNat.%2520Med.%26date%3D2016%26volume%3D22%26spage%3D851%26epage%3D860%26doi%3D10.1038%2Fnm.4123" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sulzmaier, F. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jean, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schlaepfer, D. D.</span></span> <span> </span><span class="NLM_article-title">FAK in cancer: mechanistic findings and clinical applications</span>. <i>Nat. Rev. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">598</span>– <span class="NLM_lpage">610</span>, <span class="refDoi"> DOI: 10.1038/nrc3792</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=10.1038%2Fnrc3792" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=25098269" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=1%3ACAS%3A528%3ADC%252BC2cXht12nu7vJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2014&pages=598-610&author=F.+J.+Sulzmaierauthor=C.+Jeanauthor=D.+D.+Schlaepfer&title=FAK+in+cancer%3A+mechanistic+findings+and+clinical+applications&doi=10.1038%2Fnrc3792"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">FAK in cancer: mechanistic findings and clinical applications</span></div><div class="casAuthors">Sulzmaier, Florian J.; Jean, Christine; Schlaepfer, David D.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">598-610</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Focal adhesion kinase (FAK) is a cytoplasmic protein tyrosine kinase that is overexpressed and activated in several advanced-stage solid cancers.  FAK promotes tumor progression and metastasis through effects on cancer cells, as well as stromal cells of the tumor microenvironment.  The kinase-dependent and kinase-independent functions of FAK control cell movement, invasion, survival, gene expression and cancer stem cell self-renewal.  Small mol. FAK inhibitors decrease tumor growth and metastasis in several preclin. models and have initial clin. activity in patients with limited adverse events.  In this Review, we discuss FAK signalling effects on both tumor and stromal cell biol. that provide rationale and support for future therapeutic opportunities.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrmKc7m7dsY67Vg90H21EOLACvtfcHk0li1S21C0WQTGQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXht12nu7vJ&md5=f498bcf7af890d2f92eda68e42f979d1</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1038%2Fnrc3792&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc3792%26sid%3Dliteratum%253Aachs%26aulast%3DSulzmaier%26aufirst%3DF.%2BJ.%26aulast%3DJean%26aufirst%3DC.%26aulast%3DSchlaepfer%26aufirst%3DD.%2BD.%26atitle%3DFAK%2520in%2520cancer%253A%2520mechanistic%2520findings%2520and%2520clinical%2520applications%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2014%26volume%3D14%26spage%3D598%26epage%3D610%26doi%3D10.1038%2Fnrc3792" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Popow, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arnhof, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bader, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berger, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Covini, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dank, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gmaschitz, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greb, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karolyi-Özguer, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koegl, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McConnell, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pearson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rieger, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rinnenthal, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roessler, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schrenk, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spina, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steurer, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trainor, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Traxler, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wieshofer, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zoephel, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ettmayer, P.</span></span> <span> </span><span class="NLM_article-title">Highly Selective PTK2 Proteolysis Targeting Chimeras to Probe Focal Adhesion Kinase Scaffolding Functions</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">2508</span>– <span class="NLM_lpage">2520</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b01826</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b01826" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=1%3ACAS%3A528%3ADC%252BC1MXisl2ls78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=2508-2520&author=J.+Popowauthor=H.+Arnhofauthor=G.+Baderauthor=H.+Bergerauthor=A.+Ciulliauthor=D.+Coviniauthor=C.+Dankauthor=T.+Gmaschitzauthor=P.+Grebauthor=J.+Karolyi-%C3%96zguerauthor=M.+Koeglauthor=D.+B.+McConnellauthor=M.+Pearsonauthor=M.+Riegerauthor=J.+Rinnenthalauthor=V.+Roesslerauthor=A.+Schrenkauthor=M.+Spinaauthor=S.+Steurerauthor=N.+Trainorauthor=E.+Traxlerauthor=C.+Wieshoferauthor=A.+Zoephelauthor=P.+Ettmayer&title=Highly+Selective+PTK2+Proteolysis+Targeting+Chimeras+to+Probe+Focal+Adhesion+Kinase+Scaffolding+Functions&doi=10.1021%2Facs.jmedchem.8b01826"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Highly Selective PTK2 Proteolysis Targeting Chimeras to Probe Focal Adhesion Kinase Scaffolding Functions</span></div><div class="casAuthors">Popow, Johannes; Arnhof, Heribert; Bader, Gerd; Berger, Helmut; Ciulli, Alessio; Covini, David; Dank, Christian; Gmaschitz, Teresa; Greb, Peter; Karolyi-Oezguer, Jale; Koegl, Manfred; McConnell, Darryl B.; Pearson, Mark; Rieger, Maria; Rinnenthal, Joerg; Roessler, Vanessa; Schrenk, Andreas; Spina, Markus; Steurer, Steffen; Trainor, Nicole; Traxler, Elisabeth; Wieshofer, Corinna; Zoephel, Andreas; Ettmayer, Peter</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">2508-2520</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Focal adhesion tyrosine kinase (PTK2) is often overexpressed in human hepatocellular carcinoma (HCC), and several reports have linked PTK2 depletion and/or pharmacol. inhibition to reduced tumorigenicity.  However, the clin. relevance of targeting PTK2 still remains to be proven.  Here, we present two highly selective and functional PTK2 proteolysis-targeting chimeras utilizing von Hippel-Lindau and cereblon ligands to hijack E3 ligases for PTK2 degrdn.  BI-3663 (cereblon-based) degrades PTK2 with a median DC50 of 30 nM to >80% across a panel of 11 HCC cell lines.  Despite effective PTK2 degrdn., these compds. did not phenocopy the reported antiproliferative effects of PTK2 depletion in any of the cell lines tested.  By disclosing these compds., we hope to provide valuable tools for the study of PTK2 degrdn. across different biol. systems.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqsL2R_QBrfk7Vg90H21EOLACvtfcHk0lgjvB-piizYGQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXisl2ls78%253D&md5=f0ad27d3ff46ab12e8a90e8d40a6f3c6</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b01826%26sid%3Dliteratum%253Aachs%26aulast%3DPopow%26aufirst%3DJ.%26aulast%3DArnhof%26aufirst%3DH.%26aulast%3DBader%26aufirst%3DG.%26aulast%3DBerger%26aufirst%3DH.%26aulast%3DCiulli%26aufirst%3DA.%26aulast%3DCovini%26aufirst%3DD.%26aulast%3DDank%26aufirst%3DC.%26aulast%3DGmaschitz%26aufirst%3DT.%26aulast%3DGreb%26aufirst%3DP.%26aulast%3DKarolyi-%25C3%2596zguer%26aufirst%3DJ.%26aulast%3DKoegl%26aufirst%3DM.%26aulast%3DMcConnell%26aufirst%3DD.%2BB.%26aulast%3DPearson%26aufirst%3DM.%26aulast%3DRieger%26aufirst%3DM.%26aulast%3DRinnenthal%26aufirst%3DJ.%26aulast%3DRoessler%26aufirst%3DV.%26aulast%3DSchrenk%26aufirst%3DA.%26aulast%3DSpina%26aufirst%3DM.%26aulast%3DSteurer%26aufirst%3DS.%26aulast%3DTrainor%26aufirst%3DN.%26aulast%3DTraxler%26aufirst%3DE.%26aulast%3DWieshofer%26aufirst%3DC.%26aulast%3DZoephel%26aufirst%3DA.%26aulast%3DEttmayer%26aufirst%3DP.%26atitle%3DHighly%2520Selective%2520PTK2%2520Proteolysis%2520Targeting%2520Chimeras%2520to%2520Probe%2520Focal%2520Adhesion%2520Kinase%2520Scaffolding%2520Functions%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D2508%26epage%3D2520%26doi%3D10.1021%2Facs.jmedchem.8b01826" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Winter, G. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buckley, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paulk, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Souza, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dhe-Paganon, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span> <span> </span><span class="NLM_article-title">Phthalimide conjugation as a strategy for in vivo target protein degradation</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>348</i></span>,  <span class="NLM_fpage">1376</span>– <span class="NLM_lpage">1381</span>, <span class="refDoi"> DOI: 10.1126/science.aab1433</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=10.1126%2Fscience.aab1433" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=25999370" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=1%3ACAS%3A528%3ADC%252BC2MXpvFansLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=348&publication_year=2015&pages=1376-1381&author=G.+E.+Winterauthor=D.+L.+Buckleyauthor=J.+Paulkauthor=J.+M.+Robertsauthor=A.+Souzaauthor=S.+Dhe-Paganonauthor=J.+E.+Bradner&title=Phthalimide+conjugation+as+a+strategy+for+in+vivo+target+protein+degradation&doi=10.1126%2Fscience.aab1433"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Phthalimide conjugation as a strategy for in vivo target protein degradation</span></div><div class="casAuthors">Winter, Georg E.; Buckley, Dennis L.; Paulk, Joshiawa; Roberts, Justin M.; Souza, Amanda; Dhe-Paganon, Sirano; Bradner, James E.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">348</span>
        (<span class="NLM_cas:issue">6241</span>),
    <span class="NLM_cas:pages">1376-1381</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">The development of effective pharmacol. inhibitors of multidomain scaffold proteins, notably transcription factors, is a particularly challenging problem.  In part, this is because many small-mol. antagonists disrupt the activity of only one domain in the target protein.  The authors devised a chem. strategy that promotes ligand-dependent target protein degrdn. using as an example the transcriptional coactivator BRD4, a protein crit. for cancer cell growth and survival.  The authors appended a competitive antagonist of BET bromodomains to a phthalimide moiety to hijack the cereblon E3 ubiquitin ligase complex.  The resultant compd., dBET1, induced highly selective cereblon-dependent BET protein degrdn. in vitro and in vivo and delayed leukemia progression in mice.  A second series of probes resulted in selective degrdn. of the cytosolic protein FKBP12.  This chem. strategy for controlling target protein stability may have implications for therapeutically targeting previously intractable proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp1ZW4zjRjXi7Vg90H21EOLACvtfcHk0lgjvB-piizYGQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXpvFansLk%253D&md5=b0277cc95d318d926b96e7a119066e3b</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1126%2Fscience.aab1433&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.aab1433%26sid%3Dliteratum%253Aachs%26aulast%3DWinter%26aufirst%3DG.%2BE.%26aulast%3DBuckley%26aufirst%3DD.%2BL.%26aulast%3DPaulk%26aufirst%3DJ.%26aulast%3DRoberts%26aufirst%3DJ.%2BM.%26aulast%3DSouza%26aufirst%3DA.%26aulast%3DDhe-Paganon%26aufirst%3DS.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26atitle%3DPhthalimide%2520conjugation%2520as%2520a%2520strategy%2520for%2520in%2520vivo%2520target%2520protein%2520degradation%26jtitle%3DScience%26date%3D2015%26volume%3D348%26spage%3D1376%26epage%3D1381%26doi%3D10.1126%2Fscience.aab1433" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Burslem, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jaime-Figueroa, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McQuaid, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bondeson, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toure, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qian, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crew, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hines, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">The Advantages of Targeted Protein Degradation Over Inhibition: An RTK Case Study</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">67</span>– <span class="NLM_lpage">77</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2017.09.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=10.1016%2Fj.chembiol.2017.09.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=29129716" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsl2lsrjM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2018&pages=67-77&author=G.+M.+Burslemauthor=B.+E.+Smithauthor=A.+C.+Laiauthor=S.+Jaime-Figueroaauthor=D.+C.+McQuaidauthor=D.+P.+Bondesonauthor=M.+Toureauthor=H.+Dongauthor=Y.+Qianauthor=J.+Wangauthor=A.+P.+Crewauthor=J.+Hinesauthor=C.+M.+Crews&title=The+Advantages+of+Targeted+Protein+Degradation+Over+Inhibition%3A+An+RTK+Case+Study&doi=10.1016%2Fj.chembiol.2017.09.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">The Advantages of Targeted Protein Degradation Over Inhibition: An RTK Case Study</span></div><div class="casAuthors">Burslem, George M.; Smith, Blake E.; Lai, Ashton C.; Jaime-Figueroa, Saul; McQuaid, Daniel C.; Bondeson, Daniel P.; Toure, Momar; Dong, Hanqing; Qian, Yimin; Wang, Jing; Crew, Andrew P.; Hines, John; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">67-77.e3</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Proteolysis targeting chimera (PROTAC) technol. has emerged over the last two decades as a powerful tool for targeted degrdn. of endogenous proteins.  Herein we describe the development of PROTACs for receptor tyrosine kinases, a protein family yet to be targeted for induced protein degrdn.  The use of VHL-recruiting PROTACs against this protein family reveals several advantages of degrdn. over inhibition alone: direct comparisons of fully functional, target-degrading PROTACs with target-inhibiting variants that contain an inactivated E3 ligase-recruiting ligand show that degrdn. leads to more potent inhibition of cell proliferation and a more durable and sustained downstream signaling response, and thus addresses the kinome rewiring challenge seen with many receptor tyrosine kinase inhibitors.  Combined, these findings demonstrate the ability to target receptor tyrosine kinases for degrdn. using the PROTAC technol. and outline the advantages of this degrdn.-based approach.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr4yDhRDWqNv7Vg90H21EOLACvtfcHk0lgjvB-piizYGQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsl2lsrjM&md5=0ac39a6f5836ea3772a0396356efb630</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2017.09.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2017.09.009%26sid%3Dliteratum%253Aachs%26aulast%3DBurslem%26aufirst%3DG.%2BM.%26aulast%3DSmith%26aufirst%3DB.%2BE.%26aulast%3DLai%26aufirst%3DA.%2BC.%26aulast%3DJaime-Figueroa%26aufirst%3DS.%26aulast%3DMcQuaid%26aufirst%3DD.%2BC.%26aulast%3DBondeson%26aufirst%3DD.%2BP.%26aulast%3DToure%26aufirst%3DM.%26aulast%3DDong%26aufirst%3DH.%26aulast%3DQian%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DCrew%26aufirst%3DA.%2BP.%26aulast%3DHines%26aufirst%3DJ.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DThe%2520Advantages%2520of%2520Targeted%2520Protein%2520Degradation%2520Over%2520Inhibition%253A%2520An%2520RTK%2520Case%2520Study%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2018%26volume%3D25%26spage%3D67%26epage%3D77%26doi%3D10.1016%2Fj.chembiol.2017.09.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wolfer, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramaswamy, S.</span></span> <span> </span><span class="NLM_article-title">MYC and Metastasis: Figure 1</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>71</i></span>,  <span class="NLM_fpage">2034</span>– <span class="NLM_lpage">2037</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.can-10-3776</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=10.1158%2F0008-5472.CAN-10-3776" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=21406394" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=1%3ACAS%3A528%3ADC%252BC3MXjtFWgtro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=71&publication_year=2011&pages=2034-2037&author=A.+Wolferauthor=S.+Ramaswamy&title=MYC+and+Metastasis%3A+Figure+1&doi=10.1158%2F0008-5472.can-10-3776"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">MYC and Metastasis</span></div><div class="casAuthors">Wolfer, Anita; Ramaswamy, Sridhar</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">71</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2034-2037</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">A review.  Aggressive primary tumors express transcriptional signatures that correlate with their metastatic propensity.  A no. of these signatures have been deployed in the clinic as risk stratification tools.  However, the mol. basis of these clin. useful prognostic signatures has remained a largely unresolved area of controversy.  We recently found that many prognostic signatures reflect the activity of the MYC oncogene, which in turn regulates tumor metastasis through specific effects on cancer cell invasion and migration.  These findings offer a general framework for understanding the mol. basis of clin. prognostic transcriptional signatures and suggest potentially new avenues for studying metastasis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpw1WcZTRPskLVg90H21EOLACvtfcHk0ljdOgc4mdrbyA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXjtFWgtro%253D&md5=26098aabe90bca765bfa743ef5d01909</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-10-3776&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-10-3776%26sid%3Dliteratum%253Aachs%26aulast%3DWolfer%26aufirst%3DA.%26aulast%3DRamaswamy%26aufirst%3DS.%26atitle%3DMYC%2520and%2520Metastasis%253A%2520Figure%25201%26jtitle%3DCancer%2520Res.%26date%3D2011%26volume%3D71%26spage%3D2034%26epage%3D2037%26doi%3D10.1158%2F0008-5472.can-10-3776" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Z.</span></span> <span> </span><span class="NLM_article-title">GEPIA: a web server for cancer and normal gene expression profiling and interactive analyses</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>45</i></span>,  <span class="NLM_fpage">W98</span>– <span class="NLM_lpage">W102</span>, <span class="refDoi"> DOI: 10.1093/nar/gkx247</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=10.1093%2Fnar%2Fgkx247" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=28407145" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhs1aitr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2017&pages=W98-W102&author=Z.+Tangauthor=C.+Liauthor=B.+Kangauthor=G.+Gaoauthor=C.+Liauthor=Z.+Zhang&title=GEPIA%3A+a+web+server+for+cancer+and+normal+gene+expression+profiling+and+interactive+analyses&doi=10.1093%2Fnar%2Fgkx247"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">GEPIA: a web server for cancer and normal gene expression profiling and interactive analyses</span></div><div class="casAuthors">Tang, Zefang; Li, Chenwei; Kang, Boxi; Gao, Ge; Li, Cheng; Zhang, Zemin</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">W1</span>),
    <span class="NLM_cas:pages">W98-W102</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">1362-4962</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Tremendous amt. of RNA sequencing data have been produced by large consortium projects such as TCGA and GTEx, creating new opportunities for data mining and deeper understanding of gene functions.  While certain existing web servers are valuable and widely used, many expression anal. functions needed by exptl. biologists are still not adequately addressed by these tools.  We introduce GEPIA (Gene Expression Profiling Interactive Anal.), a web-based tool to deliver fast and customizable functionalities based on TCGA and GTEx data.  GEPIA provides key interactive and customizable functions including differential expression anal., profiling plotting, correlation anal., patient survival anal., similar gene detection and dimensionality redn. anal.  The comprehensive expression analyses with simple clicking through GEPIA greatly facilitate data mining in wide research areas, scientific discussion and the therapeutic discovery process.  GEPIA fills in the gap between cancer genomics big data and the delivery of integrated information to end users, thus helping unleash the value of the current data resources.  GEPIA is available at http://gepia.cancer-pku.cn/.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoVU8vOW02u7bVg90H21EOLACvtfcHk0ljdOgc4mdrbyA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhs1aitr8%253D&md5=3153a0801287ea9e24841f0fa24e9358</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkx247&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkx247%26sid%3Dliteratum%253Aachs%26aulast%3DTang%26aufirst%3DZ.%26aulast%3DLi%26aufirst%3DC.%26aulast%3DKang%26aufirst%3DB.%26aulast%3DGao%26aufirst%3DG.%26aulast%3DLi%26aufirst%3DC.%26aulast%3DZhang%26aufirst%3DZ.%26atitle%3DGEPIA%253A%2520a%2520web%2520server%2520for%2520cancer%2520and%2520normal%2520gene%2520expression%2520profiling%2520and%2520interactive%2520analyses%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2017%26volume%3D45%26spage%3DW98%26epage%3DW102%26doi%3D10.1093%2Fnar%2Fgkx247" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gadd, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Testa, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lucas, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, K.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lamont, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zengerle, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span> <span> </span><span class="NLM_article-title">Structural basis of PROTAC cooperative recognition for selective protein degradation</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">514</span>– <span class="NLM_lpage">521</span>, <span class="refDoi"> DOI: 10.1038/nchembio.2329</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=10.1038%2Fnchembio.2329" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=28288108" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=1%3ACAS%3A528%3ADC%252BC2sXkvFahtbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2017&pages=514-521&author=M.+S.+Gaddauthor=A.+Testaauthor=X.+Lucasauthor=K.-H.+Chanauthor=W.+Chenauthor=D.+J.+Lamontauthor=M.+Zengerleauthor=A.+Ciulli&title=Structural+basis+of+PROTAC+cooperative+recognition+for+selective+protein+degradation&doi=10.1038%2Fnchembio.2329"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Structural basis of PROTAC cooperative recognition for selective protein degradation</span></div><div class="casAuthors">Gadd, Morgan S.; Testa, Andrea; Lucas, Xavier; Chan, Kwok-Ho; Chen, Wenzhang; Lamont, Douglas J.; Zengerle, Michael; Ciulli, Alessio</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">514-521</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Inducing macromol. interactions with small mols. to activate cellular signaling is a challenging goal.  PROTACs (proteolysis-targeting chimeras) are bifunctional mols. that recruit a target protein in proximity to an E3 ubiquitin ligase to trigger protein degrdn.  Structural elucidation of the key ternary ligase-PROTAC-target species and its impact on target degrdn. selectivity remain elusive.  Here, we solved the crystal structure of Brd4 degrader MZ1 in complex with human VHL and the Brd4 bromodomain (Brd4BD2).  The ligand folds into itself to allow formation of specific intermol. interactions in the ternary complex.  Isothermal titrn. calorimetry studies, supported by surface mutagenesis and proximity assays, were consistent with pronounced cooperative formation of ternary complexes with Brd4BD2.  Structure-based-designed compd. AT1 exhibited highly selective depletion of Brd4 in cells.  These results elucidate how PROTAC-induced de novo contacts dictate preferential recruitment of a target protein into a stable and cooperative complex with an E3 ligase for selective degrdn.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoJri00jB5YJLVg90H21EOLACvtfcHk0ljdOgc4mdrbyA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXkvFahtbg%253D&md5=dfee13913106d362d0972d7b1a62d700</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.2329&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.2329%26sid%3Dliteratum%253Aachs%26aulast%3DGadd%26aufirst%3DM.%2BS.%26aulast%3DTesta%26aufirst%3DA.%26aulast%3DLucas%26aufirst%3DX.%26aulast%3DChan%26aufirst%3DK.-H.%26aulast%3DChen%26aufirst%3DW.%26aulast%3DLamont%26aufirst%3DD.%2BJ.%26aulast%3DZengerle%26aufirst%3DM.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DStructural%2520basis%2520of%2520PROTAC%2520cooperative%2520recognition%2520for%2520selective%2520protein%2520degradation%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2017%26volume%3D13%26spage%3D514%26epage%3D521%26doi%3D10.1038%2Fnchembio.2329" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nowak, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DeAngelo, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buckley, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Donovan, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">An, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Safaee, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jedrychowski, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ponthier, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ishoey, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mancias, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, E. S.</span></span> <span> </span><span class="NLM_article-title">Plasticity in binding confers selectivity in ligand-induced protein degradation</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">706</span>– <span class="NLM_lpage">714</span>, <span class="refDoi"> DOI: 10.1038/s41589-018-0055-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=10.1038%2Fs41589-018-0055-y" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=29892083" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtFWlurbK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2018&pages=706-714&author=R.+P.+Nowakauthor=S.+L.+DeAngeloauthor=D.+Buckleyauthor=Z.+Heauthor=K.+A.+Donovanauthor=J.+Anauthor=N.+Safaeeauthor=M.+P.+Jedrychowskiauthor=C.+M.+Ponthierauthor=M.+Ishoeyauthor=T.+Zhangauthor=J.+D.+Manciasauthor=N.+S.+Grayauthor=J.+E.+Bradnerauthor=E.+S.+Fischer&title=Plasticity+in+binding+confers+selectivity+in+ligand-induced+protein+degradation&doi=10.1038%2Fs41589-018-0055-y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Plasticity in binding confers selectivity in ligand-induced protein degradation</span></div><div class="casAuthors">Nowak, Radoslaw P.; De Angelo, Stephen L.; Buckley, Dennis; He, Zhixiang; Donovan, Katherine A.; An, Jian; Safaee, Nozhat; Jedrychowski, Mark P.; Ponthier, Charles M.; Ishoey, Mette; Zhang, Tinghu; Mancias, Joseph D.; Gray, Nathanael S.; Bradner, James E.; Fischer, Eric S.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">706-714</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Heterobifunctional small-mol. degraders that induce protein degrdn. through ligase-mediated ubiquitination have shown considerable promise as a new pharmacol. modality.  However, we currently lack a detailed understanding of the mol. basis for target recruitment and selectivity, which is critically required to enable rational design of degraders.  Here we utilized a comprehensive characterization of the ligand-dependent CRBN-BRD4 interaction to demonstrate that binding between proteins that have not evolved to interact is plastic.  Multiple x-ray crystal structures showed that plasticity resulted in several distinct low-energy binding conformations that were selectively bound by ligands.  We demonstrated that computational protein-protein docking can reveal the underlying interprotein contacts and inform the design of a BRD4 selective degrader that can discriminate between highly homologous BET bromodomains.  These findings that plastic interprotein contacts confer selectivity for ligand-induced protein dimerization provide a conceptual framework for the development of heterobifunctional ligands.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGreQasCW8bz2LVg90H21EOLACvtfcHk0liVEO2-k9JwZg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtFWlurbK&md5=e4876c43b1eeb4558c23121b7a9fac27</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1038%2Fs41589-018-0055-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41589-018-0055-y%26sid%3Dliteratum%253Aachs%26aulast%3DNowak%26aufirst%3DR.%2BP.%26aulast%3DDeAngelo%26aufirst%3DS.%2BL.%26aulast%3DBuckley%26aufirst%3DD.%26aulast%3DHe%26aufirst%3DZ.%26aulast%3DDonovan%26aufirst%3DK.%2BA.%26aulast%3DAn%26aufirst%3DJ.%26aulast%3DSafaee%26aufirst%3DN.%26aulast%3DJedrychowski%26aufirst%3DM.%2BP.%26aulast%3DPonthier%26aufirst%3DC.%2BM.%26aulast%3DIshoey%26aufirst%3DM.%26aulast%3DZhang%26aufirst%3DT.%26aulast%3DMancias%26aufirst%3DJ.%2BD.%26aulast%3DGray%26aufirst%3DN.%2BS.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DFischer%26aufirst%3DE.%2BS.%26atitle%3DPlasticity%2520in%2520binding%2520confers%2520selectivity%2520in%2520ligand-induced%2520protein%2520degradation%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2018%26volume%3D14%26spage%3D706%26epage%3D714%26doi%3D10.1038%2Fs41589-018-0055-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Burslem, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Proteolysis-Targeting Chimeras as Therapeutics and Tools for Biological Discovery</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>181</i></span>,  <span class="NLM_fpage">102</span>– <span class="NLM_lpage">114</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2019.11.031</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=10.1016%2Fj.cell.2019.11.031" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=31955850" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=1%3ACAS%3A528%3ADC%252BB3cXht1Sitrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=181&publication_year=2020&pages=102-114&author=G.+M.+Burslemauthor=C.+M.+Crews&title=Proteolysis-Targeting+Chimeras+as+Therapeutics+and+Tools+for+Biological+Discovery&doi=10.1016%2Fj.cell.2019.11.031"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Proteolysis-Targeting Chimeras as Therapeutics and Tools for Biological Discovery</span></div><div class="casAuthors">Burslem, George M.; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">181</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">102-114</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  New biol. tools provide new techniques to probe fundamental biol. processes.  Here we describe the burgeoning field of proteolysis-targeting chimeras (PROTACs), which are capable of modulating protein concns. at a post-translational level by co-opting the ubiquitin-proteasome system.  We describe the PROTAC technol. and its application to drug discovery and provide examples where PROTACs have enabled novel biol. insights.  Furthermore, we provide a workflow for PROTAC development and use and discuss the benefits and issues assocd. with PROTACs.  Finally, we compare PROTAC-mediated protein-level modulation with other technologies, such as RNAi and genome editing.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrXgumsEakVWrVg90H21EOLACvtfcHk0liVEO2-k9JwZg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXht1Sitrk%253D&md5=3010929d2979cf04dc6003e66575f1b0</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2019.11.031&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2019.11.031%26sid%3Dliteratum%253Aachs%26aulast%3DBurslem%26aufirst%3DG.%2BM.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DProteolysis-Targeting%2520Chimeras%2520as%2520Therapeutics%2520and%2520Tools%2520for%2520Biological%2520Discovery%26jtitle%3DCell%26date%3D2020%26volume%3D181%26spage%3D102%26epage%3D114%26doi%3D10.1016%2Fj.cell.2019.11.031" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Klaeger, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heinzlmeir, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilhelm, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Polzer, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vick, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koenig, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reinecke, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruprecht, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petzoldt, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meng, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zecha, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reiter, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qiao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Helm, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koch, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schoof, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Canevari, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Casale, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Depaolini, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feuchtinger, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmidt, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rueckert, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Becker, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huenges, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garz, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gohlke, B. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zolg, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kayser, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vooder, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Preissner, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hahne, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tonisson, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kramer, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gotze, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bassermann, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schlegl, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ehrlich, H. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aiche, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walch, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greif, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schneider, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Felder, E. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruland, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Medard, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeremias, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spiekermann, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuster, B.</span></span> <span> </span><span class="NLM_article-title">The target landscape of clinical kinase drugs</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>358</i></span>, <span class="NLM_elocation-id">eaan4368</span> <span class="refDoi"> DOI: 10.1126/science.aan4368</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=10.1126%2Fscience.aan4368" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=29191878" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=358&publication_year=2017&author=S.+Klaegerauthor=S.+Heinzlmeirauthor=M.+Wilhelmauthor=H.+Polzerauthor=B.+Vickauthor=P.+A.+Koenigauthor=M.+Reineckeauthor=B.+Ruprechtauthor=S.+Petzoldtauthor=C.+Mengauthor=J.+Zechaauthor=K.+Reiterauthor=H.+Qiaoauthor=D.+Helmauthor=H.+Kochauthor=M.+Schoofauthor=G.+Canevariauthor=E.+Casaleauthor=S.+R.+Depaoliniauthor=A.+Feuchtingerauthor=Z.+Wuauthor=T.+Schmidtauthor=L.+Rueckertauthor=W.+Beckerauthor=J.+Huengesauthor=A.+K.+Garzauthor=B.+O.+Gohlkeauthor=D.+P.+Zolgauthor=G.+Kayserauthor=T.+Vooderauthor=R.+Preissnerauthor=H.+Hahneauthor=N.+Tonissonauthor=K.+Kramerauthor=K.+Gotzeauthor=F.+Bassermannauthor=J.+Schleglauthor=H.+C.+Ehrlichauthor=S.+Aicheauthor=A.+Walchauthor=P.+A.+Greifauthor=S.+Schneiderauthor=E.+R.+Felderauthor=J.+Rulandauthor=G.+Medardauthor=I.+Jeremiasauthor=K.+Spiekermannauthor=B.+Kuster&title=The+target+landscape+of+clinical+kinase+drugs&doi=10.1126%2Fscience.aan4368"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1126%2Fscience.aan4368&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.aan4368%26sid%3Dliteratum%253Aachs%26aulast%3DKlaeger%26aufirst%3DS.%26aulast%3DHeinzlmeir%26aufirst%3DS.%26aulast%3DWilhelm%26aufirst%3DM.%26aulast%3DPolzer%26aufirst%3DH.%26aulast%3DVick%26aufirst%3DB.%26aulast%3DKoenig%26aufirst%3DP.%2BA.%26aulast%3DReinecke%26aufirst%3DM.%26aulast%3DRuprecht%26aufirst%3DB.%26aulast%3DPetzoldt%26aufirst%3DS.%26aulast%3DMeng%26aufirst%3DC.%26aulast%3DZecha%26aufirst%3DJ.%26aulast%3DReiter%26aufirst%3DK.%26aulast%3DQiao%26aufirst%3DH.%26aulast%3DHelm%26aufirst%3DD.%26aulast%3DKoch%26aufirst%3DH.%26aulast%3DSchoof%26aufirst%3DM.%26aulast%3DCanevari%26aufirst%3DG.%26aulast%3DCasale%26aufirst%3DE.%26aulast%3DDepaolini%26aufirst%3DS.%2BR.%26aulast%3DFeuchtinger%26aufirst%3DA.%26aulast%3DWu%26aufirst%3DZ.%26aulast%3DSchmidt%26aufirst%3DT.%26aulast%3DRueckert%26aufirst%3DL.%26aulast%3DBecker%26aufirst%3DW.%26aulast%3DHuenges%26aufirst%3DJ.%26aulast%3DGarz%26aufirst%3DA.%2BK.%26aulast%3DGohlke%26aufirst%3DB.%2BO.%26aulast%3DZolg%26aufirst%3DD.%2BP.%26aulast%3DKayser%26aufirst%3DG.%26aulast%3DVooder%26aufirst%3DT.%26aulast%3DPreissner%26aufirst%3DR.%26aulast%3DHahne%26aufirst%3DH.%26aulast%3DTonisson%26aufirst%3DN.%26aulast%3DKramer%26aufirst%3DK.%26aulast%3DGotze%26aufirst%3DK.%26aulast%3DBassermann%26aufirst%3DF.%26aulast%3DSchlegl%26aufirst%3DJ.%26aulast%3DEhrlich%26aufirst%3DH.%2BC.%26aulast%3DAiche%26aufirst%3DS.%26aulast%3DWalch%26aufirst%3DA.%26aulast%3DGreif%26aufirst%3DP.%2BA.%26aulast%3DSchneider%26aufirst%3DS.%26aulast%3DFelder%26aufirst%3DE.%2BR.%26aulast%3DRuland%26aufirst%3DJ.%26aulast%3DMedard%26aufirst%3DG.%26aulast%3DJeremias%26aufirst%3DI.%26aulast%3DSpiekermann%26aufirst%3DK.%26aulast%3DKuster%26aufirst%3DB.%26atitle%3DThe%2520target%2520landscape%2520of%2520clinical%2520kinase%2520drugs%26jtitle%3DScience%26date%3D2017%26volume%3D358%26doi%3D10.1126%2Fscience.aan4368" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kuenzi, B. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Remsing Rix, L. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stewart, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kinose, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bryant, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boyle, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koomen, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haura, E. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rix, U.</span></span> <span> </span><span class="NLM_article-title">Polypharmacology-based ceritinib repurposing using integrated functional proteomics</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">1222</span>– <span class="NLM_lpage">1231</span>, <span class="refDoi"> DOI: 10.1038/nchembio.2489</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=10.1038%2Fnchembio.2489" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=28991240" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhs1aisrzL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2017&pages=1222-1231&author=B.+M.+Kuenziauthor=L.+L.+Remsing+Rixauthor=P.+A.+Stewartauthor=B.+Fangauthor=F.+Kinoseauthor=A.+T.+Bryantauthor=T.+A.+Boyleauthor=J.+M.+Koomenauthor=E.+B.+Hauraauthor=U.+Rix&title=Polypharmacology-based+ceritinib+repurposing+using+integrated+functional+proteomics&doi=10.1038%2Fnchembio.2489"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Polypharmacology-based ceritinib repurposing using integrated functional proteomics</span></div><div class="casAuthors">Kuenzi, Brent M.; Remsing Rix, Lily L.; Stewart, Paul A.; Fang, Bin; Kinose, Fumi; Bryant, Annamarie T.; Boyle, Theresa A.; Koomen, John M.; Haura, Eric B.; Rix, Uwe</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1222-1231</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Targeted drugs are effective when they directly inhibit strong disease drivers, but only a small fraction of diseases feature defined actionable drivers.  Alternatively, network-based approaches can uncover new therapeutic opportunities.  Applying an integrated phenotypic screening, chem. and phosphoproteomics strategy, here the authors describe the anaplastic lymphoma kinase (ALK) inhibitor ceritinib as having activity across several ALK-neg. lung cancer cell lines and identify new targets and network-wide signaling effects.  Combining pharmacol. inhibitors and RNA interference revealed a polypharmacol. mechanism involving the noncanonical targets IGF1R, FAK1, RSK1 and RSK2.  Mutating the downstream signaling hub YB1 protected cells from ceritinib.  Consistent with YB1 signaling being known to cause taxol resistance, combination of ceritinib with paclitaxel displayed strong synergy, particularly in cells expressing high FAK autophosphorylation, which the authors show to be prevalent in lung cancer.  Together, the authors present a systems chem. biol. platform for elucidating multikinase inhibitor polypharmacol. mechanisms, subsequent design of synergistic drug combinations, and identification of mechanistic biomarker candidates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoECSbklR701rVg90H21EOLACvtfcHk0lguHhonroGRfg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhs1aisrzL&md5=8783f3602d51663cde58ed735a294dc2</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.2489&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.2489%26sid%3Dliteratum%253Aachs%26aulast%3DKuenzi%26aufirst%3DB.%2BM.%26aulast%3DRemsing%2BRix%26aufirst%3DL.%2BL.%26aulast%3DStewart%26aufirst%3DP.%2BA.%26aulast%3DFang%26aufirst%3DB.%26aulast%3DKinose%26aufirst%3DF.%26aulast%3DBryant%26aufirst%3DA.%2BT.%26aulast%3DBoyle%26aufirst%3DT.%2BA.%26aulast%3DKoomen%26aufirst%3DJ.%2BM.%26aulast%3DHaura%26aufirst%3DE.%2BB.%26aulast%3DRix%26aufirst%3DU.%26atitle%3DPolypharmacology-based%2520ceritinib%2520repurposing%2520using%2520integrated%2520functional%2520proteomics%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2017%26volume%3D13%26spage%3D1222%26epage%3D1231%26doi%3D10.1038%2Fnchembio.2489" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Donovan, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferguson, F. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bushman, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eleuteri, N. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhunia, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ryu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yue, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dobrovolsky, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hao, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">You, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Teng, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hatcher, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manz, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Groendyke, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nam, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sengupta, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cho, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shin, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Agius, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ghobrial, I. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, M. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Che, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buhrlage, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sim, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, E. S.</span></span> <span> </span><span class="NLM_article-title">Mapping the Degradable Kinome Provides a Resource for Expedited Degrader Development</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>183</i></span>,  <span class="NLM_fpage">1714</span>– <span class="NLM_lpage">1731</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2020.10.038</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=10.1016%2Fj.cell.2020.10.038" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=33275901" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;key=1%3ACAS%3A528%3ADC%252BB3cXisV2nsbjL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=183&publication_year=2020&pages=1714-1731&author=K.+A.+Donovanauthor=F.+M.+Fergusonauthor=J.+W.+Bushmanauthor=N.+A.+Eleuteriauthor=D.+Bhuniaauthor=S.+Ryuauthor=L.+Tanauthor=K.+Shiauthor=H.+Yueauthor=X.+Liuauthor=D.+Dobrovolskyauthor=B.+Jiangauthor=J.+Wangauthor=M.+Haoauthor=I.+Youauthor=M.+Tengauthor=Y.+Liangauthor=J.+Hatcherauthor=Z.+Liauthor=T.+D.+Manzauthor=B.+Groendykeauthor=W.+Huauthor=Y.+Namauthor=S.+Senguptaauthor=H.+Choauthor=I.+Shinauthor=M.+P.+Agiusauthor=I.+M.+Ghobrialauthor=M.+W.+Maauthor=J.+Cheauthor=S.+J.+Buhrlageauthor=T.+Simauthor=N.+S.+Grayauthor=E.+S.+Fischer&title=Mapping+the+Degradable+Kinome+Provides+a+Resource+for+Expedited+Degrader+Development&doi=10.1016%2Fj.cell.2020.10.038"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Mapping the Degradable Kinome Provides a Resource for Expedited Degrader Development</span></div><div class="casAuthors">Donovan, Katherine A.; Ferguson, Fleur M.; Bushman, Jonathan W.; Eleuteri, Nicholas A.; Bhunia, Debabrata; Ryu, SeongShick; Tan, Li; Shi, Kun; Yue, Hong; Liu, Xiaoxi; Dobrovolsky, Dennis; Jiang, Baishan; Wang, Jinhua; Hao, Mingfeng; You, Inchul; Teng, Mingxing; Liang, Yanke; Hatcher, John; Li, Zhengnian; Manz, Theresa D.; Groendyke, Brian; Hu, Wanyi; Nam, Yunju; Sengupta, Sandip; Cho, Hanna; Shin, Injae; Agius, Michael P.; Ghobrial, Irene M.; Ma, Michelle W.; Che, Jianwei; Buhrlage, Sara J.; Sim, Taebo; Gray, Nathanael S.; Fischer, Eric S.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">183</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1714-1731.e10</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Targeted protein degrdn. (TPD) refers to the use of small mols. to induce ubiquitin-dependent degrdn. of proteins.  TPD is of interest in drug development, as it can address previously inaccessible targets.  However, degrader discovery and optimization remains an inefficient process due to a lack of understanding of the relative importance of the key mol. events required to induce target degrdn.  Here, we use chemo-proteomics to annotate the degradable kinome.  Our expansive dataset provides chem. leads for ∼200 kinases and demonstrates that the current practice of starting from the highest potency binder is an ineffective method for discovering active compds.  We develop multitargeted degraders to answer fundamental questions about the ubiquitin proteasome system, uncovering that kinase degrdn. is p97 dependent.  This work will not only fuel kinase degrader discovery, but also provides a blueprint for evaluating targeted degrdn. across entire gene families to accelerate understanding of TPD beyond the kinome.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoCV_iCTOWvN7Vg90H21EOLACvtfcHk0lguHhonroGRfg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXisV2nsbjL&md5=380302305a3749c52492ebd82bf1dabf</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2020.10.038&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2020.10.038%26sid%3Dliteratum%253Aachs%26aulast%3DDonovan%26aufirst%3DK.%2BA.%26aulast%3DFerguson%26aufirst%3DF.%2BM.%26aulast%3DBushman%26aufirst%3DJ.%2BW.%26aulast%3DEleuteri%26aufirst%3DN.%2BA.%26aulast%3DBhunia%26aufirst%3DD.%26aulast%3DRyu%26aufirst%3DS.%26aulast%3DTan%26aufirst%3DL.%26aulast%3DShi%26aufirst%3DK.%26aulast%3DYue%26aufirst%3DH.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DDobrovolsky%26aufirst%3DD.%26aulast%3DJiang%26aufirst%3DB.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DHao%26aufirst%3DM.%26aulast%3DYou%26aufirst%3DI.%26aulast%3DTeng%26aufirst%3DM.%26aulast%3DLiang%26aufirst%3DY.%26aulast%3DHatcher%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DZ.%26aulast%3DManz%26aufirst%3DT.%2BD.%26aulast%3DGroendyke%26aufirst%3DB.%26aulast%3DHu%26aufirst%3DW.%26aulast%3DNam%26aufirst%3DY.%26aulast%3DSengupta%26aufirst%3DS.%26aulast%3DCho%26aufirst%3DH.%26aulast%3DShin%26aufirst%3DI.%26aulast%3DAgius%26aufirst%3DM.%2BP.%26aulast%3DGhobrial%26aufirst%3DI.%2BM.%26aulast%3DMa%26aufirst%3DM.%2BW.%26aulast%3DChe%26aufirst%3DJ.%26aulast%3DBuhrlage%26aufirst%3DS.%2BJ.%26aulast%3DSim%26aufirst%3DT.%26aulast%3DGray%26aufirst%3DN.%2BS.%26aulast%3DFischer%26aufirst%3DE.%2BS.%26atitle%3DMapping%2520the%2520Degradable%2520Kinome%2520Provides%2520a%2520Resource%2520for%2520Expedited%2520Degrader%2520Development%26jtitle%3DCell%26date%3D2020%26volume%3D183%26spage%3D1714%26epage%3D1731%26doi%3D10.1016%2Fj.cell.2020.10.038" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6MX8" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6MX8','PDB','6MX8'); return false;">PDB: 6MX8</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i47"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.1c00373">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_31570"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a class="ext-link" href="/doi/10.1021/acs.jmedchem.1c00373?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acs.jmedchem.1c00373</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">NMR and HPLC of compounds <b>1–12</b>; hook effect of SIAIS164018; signaling pathway; apoptosis analysis; gene ontology analysis; maximal tolerance dose study; AlK mutant identification; kinome map; and degradation through UPS (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.1c00373/suppl_file/jm1c00373_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.1c00373/suppl_file/jm1c00373_si_002.csv">CSV</a>)</p></li><li><p class="inline">Proteomics data (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.1c00373/suppl_file/jm1c00373_si_003.xlsx">XLSX</a>)</p></li><li><p class="inline">RNA-seq data (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.1c00373/suppl_file/jm1c00373_si_004.xlsx">XLSX</a>)</p></li><li><p class="inline">Kinome data (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.1c00373/suppl_file/jm1c00373_si_005.xlsx">XLSX</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00373/suppl_file/jm1c00373_si_001.pdf">jm1c00373_si_001.pdf (2.24 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00373/suppl_file/jm1c00373_si_002.csv">jm1c00373_si_002.csv (1.95 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00373/suppl_file/jm1c00373_si_003.xlsx">jm1c00373_si_003.xlsx (26.74 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00373/suppl_file/jm1c00373_si_004.xlsx">jm1c00373_si_004.xlsx (2.38 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00373/suppl_file/jm1c00373_si_005.xlsx">jm1c00373_si_005.xlsx (12.98 MB)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.1c00373&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2021.64.issue-13%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.1c00373" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.1c00373" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                8MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"677a72e0a90a3d48","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
