
<html lang="en"     class="pb-page"  data-request-id="2a2cdada-3043-4908-aa26-d4868fb8758d"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;article:article:10.1021/acs.jmedchem.1c00607;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Discovery of Cyclic Peptidomimetic Ligands Targeting the Extracellular Domain of EGFR" /></meta><meta name="dc.Creator" content="Mengmeng  Zheng" /></meta><meta name="dc.Creator" content="Chunpu  Li" /></meta><meta name="dc.Creator" content="Mi  Zhou" /></meta><meta name="dc.Creator" content="Ru  Jia" /></meta><meta name="dc.Creator" content="Gang  Cai" /></meta><meta name="dc.Creator" content="Fengyu  She" /></meta><meta name="dc.Creator" content="Lulu  Wei" /></meta><meta name="dc.Creator" content="Shaohui  Wang" /></meta><meta name="dc.Creator" content="Jie  Yu" /></meta><meta name="dc.Creator" content="Dingyan  Wang" /></meta><meta name="dc.Creator" content="Laurent  Calcul" /></meta><meta name="dc.Creator" content="Xingmin  Sun" /></meta><meta name="dc.Creator" content="Xiaomin  Luo" /></meta><meta name="dc.Creator" content="Feng  Cheng" /></meta><meta name="dc.Creator" content="Qi  Li" /></meta><meta name="dc.Creator" content="Yan  Wang" /></meta><meta name="dc.Creator" content="Jianfeng  Cai" /></meta><meta name="dc.Description" content="It is very promising to target the extracellular domain of epidermal growth factor receptor (EGFR) for developing novel and selective anticancer therapies. Herein, we report the discovery of a nove..." /></meta><meta name="Description" content="It is very promising to target the extracellular domain of epidermal growth factor receptor (EGFR) for developing novel and selective anticancer therapies. Herein, we report the discovery of a nove..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="July 23, 2021" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.1c00607" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2021 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.1c00607" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.1c00607" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.1c00607" /></link>
        
    
    

<title>Discovery of Cyclic Peptidomimetic Ligands Targeting the Extracellular Domain of EGFR | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.1c00607" /></meta><meta property="og:title" content="Discovery of Cyclic Peptidomimetic Ligands Targeting the Extracellular Domain of EGFR" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00607/20210723/images/large/jm1c00607_0008.jpeg" /></meta><meta property="og:description" content="It is very promising to target the extracellular domain of epidermal growth factor receptor (EGFR) for developing novel and selective anticancer therapies. Herein, we report the discovery of a novel small molecule, M-2-5, from a one-bead-two-compound (OBTC) cyclic γ-AApeptide library. The molecule was found to bind tightly to the extracellular domain of EGFR. Intriguingly, this molecule could also effectively antagonize EGF-stimulated EGFR phosphorylation and downstream signal transduction. Furthermore, together with its remarkable resistance to proteolytic degradation, M-2-5 was shown to effectively inhibit cell proliferation and migration in vitro and suppresses the growth of tumor in the A549 xenograft model in vivo, highlighting its potential therapeutic application for cancer treatment." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.1c00607"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.1c00607">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-14/achre4.2021.54.issue-14/20210720/achre4.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-29/aamick.2021.13.issue-29/20210728/aamick.2021.13.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-14/acncdm.2021.12.issue-14/20210721/acncdm.2021.12.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-7/amlcef.2021.3.issue-7/20210705/amlcef.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-29/acsodf.2021.6.issue-29/20210727/acsodf.2021.6.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-29/ascecg.2021.9.issue-29/20210726/ascecg.2021.9.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-29/ancham.2021.93.issue-29/20210727/ancham.2021.93.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-29/bichaw.2021.60.issue-29/20210727/bichaw.2021.60.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-27/cgeabj.2021.99.issue-27/20210726-01/cgeabj.2021.99.issue-27.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-7/cgdefu.2021.21.issue-7/20210707/cgdefu.2021.21.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-14/esthag.2021.55.issue-14/20210720/esthag.2021.55.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-29/iecred.2021.60.issue-29/20210728/iecred.2021.60.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-14/inocaj.2021.60.issue-14/20210719/inocaj.2021.60.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-29/jafcau.2021.69.issue-29/20210728/jafcau.2021.69.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-29/jacsat.2021.143.issue-29/20210728/jacsat.2021.143.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-7/jamsef.2021.32.issue-7/20210707/jamsef.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-29/langd5.2021.37.issue-29/20210727/langd5.2021.37.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-7/mpohbp.2021.18.issue-7/20210705/mpohbp.2021.18.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.1c00607&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.1c00607&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.1c00607&amp;href=/doi/10.1021/acs.jmedchem.1c00607" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2021</span><span class="cit-fg-volume">, XXXX</span><span class="cit-fg-issue">, XXX</span><span class="cit-fg-pageRange">, XXX-XXX</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/0/0" title="Return to Articles ASAP" id="returnToASAP" class="content-navigation__btn--return"><span>RETURN TO ARTICLES ASAP</span></a><a href="/doi/10.1021/acs.jmedchem.1c00277" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.1c01033" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Discovery of Cyclic Peptidomimetic Ligands Targeting the Extracellular Domain of EGFR</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Mengmeng Zheng</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Mengmeng Zheng</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemistry, University of South Florida, 4202 E. Fowler Avenue, Tampa, Florida 33620, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Mengmeng++Zheng">Mengmeng Zheng</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Chunpu Li</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Chunpu Li</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemistry, University of South Florida, 4202 E. Fowler Avenue, Tampa, Florida 33620, United States</div><div class="loa-info-affiliations-info">Department of Medical Oncology & Cancer Institute of Medicine, Shuguang Hospital, Shanghai University of Traditional Chinese Medicine, Shanghai 201203, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Chunpu++Li">Chunpu Li</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Mi Zhou</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Mi Zhou</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemistry, University of South Florida, 4202 E. Fowler Avenue, Tampa, Florida 33620, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Mi++Zhou">Mi Zhou</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Ru Jia</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Ru Jia</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Medical Oncology & Cancer Institute of Medicine, Shuguang Hospital, Shanghai University of Traditional Chinese Medicine, Shanghai 201203, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Ru++Jia">Ru Jia</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Gang Cai</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Gang Cai</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Medical Oncology & Cancer Institute of Medicine, Shuguang Hospital, Shanghai University of Traditional Chinese Medicine, Shanghai 201203, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Gang++Cai">Gang Cai</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Fengyu She</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Fengyu She</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemistry, University of South Florida, 4202 E. Fowler Avenue, Tampa, Florida 33620, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Fengyu++She">Fengyu She</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Lulu Wei</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Lulu Wei</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemistry, University of South Florida, 4202 E. Fowler Avenue, Tampa, Florida 33620, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Lulu++Wei">Lulu Wei</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0003-4657-9479" title="Orcid link">https://orcid.org/0000-0003-4657-9479</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Shaohui Wang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Shaohui Wang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Molecular Medicine, Morsani College of Medicine, University of South Florida, Tampa, Florida 33612, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Shaohui++Wang">Shaohui Wang</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Jie Yu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jie Yu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Drug Discovery and Design Center, Shanghai Institute of Materia Medica, Chinese Academy of Sciences, Shanghai 201203, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jie++Yu">Jie Yu</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Dingyan Wang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Dingyan Wang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Drug Discovery and Design Center, Shanghai Institute of Materia Medica, Chinese Academy of Sciences, Shanghai 201203, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Dingyan++Wang">Dingyan Wang</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Laurent Calcul</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Laurent Calcul</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemistry, University of South Florida, 4202 E. Fowler Avenue, Tampa, Florida 33620, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Laurent++Calcul">Laurent Calcul</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Xingmin Sun</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Xingmin Sun</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Molecular Medicine, Morsani College of Medicine, University of South Florida, Tampa, Florida 33612, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Xingmin++Sun">Xingmin Sun</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Xiaomin Luo</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Xiaomin Luo</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Drug Discovery and Design Center, Shanghai Institute of Materia Medica, Chinese Academy of Sciences, Shanghai 201203, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Xiaomin++Luo">Xiaomin Luo</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0003-0426-3417" title="Orcid link">https://orcid.org/0000-0003-0426-3417</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Feng Cheng</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Feng Cheng</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Pharmaceutical Sciences, College of Pharmacy, University of South Florida, Tampa, Florida 33612, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Feng++Cheng">Feng Cheng</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Qi Li</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Qi Li</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Medical Oncology & Cancer Institute of Medicine, Shuguang Hospital, Shanghai University of Traditional Chinese Medicine, Shanghai 201203, China</div><div class="loa-info-affiliations-info">Academy of Integrative Medicine, Shanghai University of Traditional Chinese Medicine, Shanghai 201203, China</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#ef9e868386af9c879a9b8c82c18a8b9ac18c81"><span class="__cf_email__" data-cfemail="d9a8b0b5b099aab1acadbab4f7bcbdacf7bab7">[email protected]</span></a></span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Qi++Li">Qi Li</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Yan Wang</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Yan Wang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Medical Oncology & Cancer Institute of Medicine, Shuguang Hospital, Shanghai University of Traditional Chinese Medicine, Shanghai 201203, China</div><div class="loa-info-affiliations-info">Academy of Integrative Medicine, Shanghai University of Traditional Chinese Medicine, Shanghai 201203, China</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#ceb7afa0b9afa0a98ebda6bbbaada3e0abaabbe0ada0"><span class="__cf_email__" data-cfemail="a7dec6c9d0c6c9c0e7d4cfd2d3c4ca89c2c3d289c4c9">[email protected]</span></a></span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Yan++Wang">Yan Wang</a></span></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Jianfeng Cai</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jianfeng Cai</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemistry, University of South Florida, 4202 E. Fowler Avenue, Tampa, Florida 33620, United States</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#19537078777f7c777e7a7870596c6a7f377c7d6c"><span class="__cf_email__" data-cfemail="bff5d6ded1d9dad1d8dcded6ffcaccd991dadbca">[email protected]</span></a></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jianfeng++Cai">Jianfeng Cai</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0003-3106-3306" title="Orcid link">https://orcid.org/0000-0003-3106-3306</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.1c00607&amp;href=/doi/10.1021%2Facs.jmedchem.1c00607" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2021</span></span><span class="cit-volume">, XXXX</span><span class="cit-issue">, XXX</span><span class="cit-pageRange">, XXX-XXX</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">July 23, 2021</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>1 April 2021</li><li><span class="item_label"><b>Published</b> online</span>23 July 2021</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.1c00607" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.1c00607</a></div><div class="article_header-article-copyright"><strong>© 2021 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D1%26pageCount%3D10%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DMengmeng%2BZheng%252C%2BChunpu%2BLi%252C%2BMi%2BZhou%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D0%26issueNum%3D0%26contentID%3Dacs.jmedchem.1c00607%26title%3DDiscovery%2Bof%2BCyclic%2BPeptidomimetic%2BLigands%2BTargeting%2Bthe%2BExtracellular%2BDomain%2Bof%2BEGFR%26numPages%3D10%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D10%26publicationDate%3DJuly%2B2021">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.1c00607"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">367</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val">-</div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.1c00607" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Discovery of Cyclic Peptidomimetic Ligands Targeting the Extracellular Domain of EGFR&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Mengmeng&quot;,&quot;last_name&quot;:&quot;Zheng&quot;},{&quot;first_name&quot;:&quot;Chunpu&quot;,&quot;last_name&quot;:&quot;Li&quot;},{&quot;first_name&quot;:&quot;Mi&quot;,&quot;last_name&quot;:&quot;Zhou&quot;},{&quot;first_name&quot;:&quot;Ru&quot;,&quot;last_name&quot;:&quot;Jia&quot;},{&quot;first_name&quot;:&quot;Gang&quot;,&quot;last_name&quot;:&quot;Cai&quot;},{&quot;first_name&quot;:&quot;Fengyu&quot;,&quot;last_name&quot;:&quot;She&quot;},{&quot;first_name&quot;:&quot;Lulu&quot;,&quot;last_name&quot;:&quot;Wei&quot;},{&quot;first_name&quot;:&quot;Shaohui&quot;,&quot;last_name&quot;:&quot;Wang&quot;},{&quot;first_name&quot;:&quot;Jie&quot;,&quot;last_name&quot;:&quot;Yu&quot;},{&quot;first_name&quot;:&quot;Dingyan&quot;,&quot;last_name&quot;:&quot;Wang&quot;},{&quot;first_name&quot;:&quot;Laurent&quot;,&quot;last_name&quot;:&quot;Calcul&quot;},{&quot;first_name&quot;:&quot;Xingmin&quot;,&quot;last_name&quot;:&quot;Sun&quot;},{&quot;first_name&quot;:&quot;Xiaomin&quot;,&quot;last_name&quot;:&quot;Luo&quot;},{&quot;first_name&quot;:&quot;Feng&quot;,&quot;last_name&quot;:&quot;Cheng&quot;},{&quot;first_name&quot;:&quot;Qi&quot;,&quot;last_name&quot;:&quot;Li&quot;},{&quot;first_name&quot;:&quot;Yan&quot;,&quot;last_name&quot;:&quot;Wang&quot;},{&quot;first_name&quot;:&quot;Jianfeng&quot;,&quot;last_name&quot;:&quot;Cai&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2021&quot;,&quot;month&quot;:&quot;07&quot;,&quot;day&quot;:&quot;23&quot;,&quot;issue&quot;:&quot;0&quot;,&quot;volume&quot;:&quot;0&quot;,&quot;pages&quot;:&quot;&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.1c00607&quot;},&quot;abstract&quot;:&quot;It is very promising to target the extracellular domain of epidermal growth factor receptor (EGFR) for developing novel and selective anticancer therapies. Herein, we report the discovery of a novel small molecule, M-2-5, from a one-bead-two-compound (OBTC) cyclic γ-AApeptide library. The molecule was found to bind tightly to the extracellular domain of EGFR. Intriguingly, this molecule could also effectively antagonize EGF-stimulated EGFR phosphorylation and downstream signal transduction. Furthermore, together with its remarkable resistance to proteolytic degradation, M-2-5 was shown to effectively inhibit cell proliferation and migration in vitro and suppresses the growth of tumor in the A549 xenograft model in vivo, highlighting its potential therapeutic application for cancer treatment.&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.1c00607&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.1c00607" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.1c00607&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.1c00607" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.1c00607&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.1c00607" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.1c00607&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.1c00607&amp;href=/doi/10.1021/acs.jmedchem.1c00607" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.1c00607" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.1c00607" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (8 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.1c00607&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.1c00607%26sid%3Dliteratum%253Aachs%26pmid%3D34297567%26genre%3Darticle%26aulast%3DZheng%26date%3D2021%26atitle%3DDiscovery%2Bof%2BCyclic%2BPeptidomimetic%2BLigands%2BTargeting%2Bthe%2BExtracellular%2BDomain%2Bof%2BEGFR%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D0%26issue%3D0%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291909" title="Immunology">Immunology</a>,</li><li><a href="/action/doSearch?ConceptID=291350" title="Assays">Assays</a>,</li><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=290975" title="Biopolymers">Biopolymers</a>,</li><li><a href="/action/doSearch?ConceptID=291305" title="Cell physiology">Cell physiology</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/0/0" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00607/20210723/images/medium/jm1c00607_0008.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00607/20210723/images/large/jm1c00607_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00607&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">It is very promising to target the extracellular domain of epidermal growth factor receptor (EGFR) for developing novel and selective anticancer therapies. Herein, we report the discovery of a novel small molecule, <b>M-2-5</b>, from a one-bead-two-compound (OBTC) cyclic γ-AApeptide library. The molecule was found to bind tightly to the extracellular domain of EGFR. Intriguingly, this molecule could also effectively antagonize EGF-stimulated EGFR phosphorylation and downstream signal transduction. Furthermore, together with its remarkable resistance to proteolytic degradation, <b>M-2-5</b> was shown to effectively inhibit cell proliferation and migration in vitro and suppresses the growth of tumor in the A549 xenograft model in vivo, highlighting its potential therapeutic application for cancer treatment.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_22437" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_22437" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">As a member of the receptor tyrosine kinase family, epidermal growth factor receptor (EGFR) plays a crucial role in key cellular transduction pathways and regulation of growth and differentiation of many cell types.<a onclick="showRef(event, 'ref1 ref2'); return false;" href="javascript:void(0);" class="ref ref1 ref2">(1,2)</a> Previous studies suggest that EGFR is abnormally expressed in a wide variety of human epithelial malignancies, such as glioma, lung cancer, and colon cancer, and therefore has been extensively used as the target for the development of drugs for the treatment of cancer, which is caused by epidermal growth factor receptor upregulation.<a onclick="showRef(event, 'ref3 ref4 ref5 ref6'); return false;" href="javascript:void(0);" class="ref ref3 ref4 ref5 ref6">(3−6)</a></div><div class="NLM_p">EGFR, also known as ErbB-1, is a transmembrane protein that has a ligand-binding extracellular domain, a transmembrane domain, and an intracellular domain with kinase activity. As such, monoclonal antibodies that bind to the extracellular domain, as well as small molecules targeting the intracellular tyrosine kinase domain of EGFR, are the two main approaches currently undertaken to inhibit EGFR functions for cancer therapy.<a onclick="showRef(event, 'ref7 ref8'); return false;" href="javascript:void(0);" class="ref ref7 ref8">(7,8)</a> However, small-molecule EGFR inhibitors such as erlotinib, gefitinib, afatinib, and so on,<a onclick="showRef(event, 'ref9 ref10 ref11'); return false;" href="javascript:void(0);" class="ref ref9 ref10 ref11">(9−11)</a> which are known to target the intracellular tyrosine kinase domain of EGFR and thereby inhibiting EGFR autophosphorylation and downstream signal transduction, lead to rapid intrinsic or acquired resistance. Meanwhile, these small tyrosine kinase inhibitors lack specificity, and as a result, they generally exhibit severe toxic side effects, limiting the prolonged use of these drugs in clinical therapy. On the other hand, anti-EGFR monoclonal antibodies, such as cetuximab<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> and matuzumab,<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> which could recognize the extracellular domain of EGFR exclusively with high affinity and specificity, have already been successfully developed for clinical therapeutic strategies.</div><div class="NLM_p last">Despite the success of EGFR-targeted drugs, there are high demands for EGFR therapeutics with lower cost and improved efficacy. Peptides that bind to the extracellular domain of EGFR have been developed through phage display or virtual screening;<a onclick="showRef(event, 'ref14 ref15'); return false;" href="javascript:void(0);" class="ref ref14 ref15">(14,15)</a> however, they were subsequently investigated as EGFR-targeting ligands, whereas their own ability to mimic monoclonal antibody and affect EGFR signaling was not reported. Meanwhile, the peptides are susceptible to rapid proteolytic degradation. An alternative strategy could take advantage of peptidomimetics that mimic the structure and function of peptides. Recently, we have designed a class of peptidomimetics, γ-AApeptides, which are resistant to enzymatic hydrolysis.<a onclick="showRef(event, 'ref16 ref17 ref18'); return false;" href="javascript:void(0);" class="ref ref16 ref17 ref18">(16−18)</a> γ-AApeptides are able to adopt robust helix stabilized by sulfonamido moieties and hydrogen bonding,<a onclick="showRef(event, 'ref19 ref20 ref21 ref22'); return false;" href="javascript:void(0);" class="ref ref19 ref20 ref21 ref22">(19−22)</a> allowing the rational design of protein domain mimetics for the recognition of protein surface and disruption of crucial protein–protein interactions.<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> Additionally, their vast chemodiversity and modular synthetic capability warrant γ-AApeptides excellent candidates for the creation of combinatorial libraries, which could provide unnatural ligands binding to biomacromolecules including nucleic acids and proteins with high affinity and specificity.<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> Indeed, the unique macrocyclic γ-AApeptides framework provided a novel molecular scaffold with good biocompatibility and high stability for studying the biomacromolecular interaction and developing potential drug candidates and molecular probes.<a onclick="showRef(event, 'ref24 ref25 ref26'); return false;" href="javascript:void(0);" class="ref ref24 ref25 ref26">(24−26)</a> In this work, we report our effort in the identification of γ-AApeptides-based ligands that recognize the EGFR extracellular domain through the screening of a one-bead-two-compound (OBTC) γ-AApeptides combinatorial library with a diversity of 320 000. Fortunately, we were able to discover novel cyclic γ-AApeptides that exhibited excellent binding activity to EGFR. It is intriguing that one cyclic γ-AApeptide could effectively inhibit EGF-stimulated EGFR phosphorylation and block its downstream signaling pathway. Moreover, this compound could efficiently inhibit cell proliferation and cell migration and exhibited potent efficacy in the inhibition of tumor growth on an A549 lung cancer xenograft model.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_25508" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_25508" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> Library Synthesis and Screening</h3><div class="NLM_p">The OBTC γ-AApeptide library was prepared on the solid phase following the previously reported protocol.<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> In the library, each TentaGel bead was manipulated to bear two layers, with a cyclic γ-AApeptide on the outer layer and a unique peptide coding tag attached onto the inner layer. The cyclic γ-AApeptide was constructed through combinational synthesis using five γ-AApeptides building blocks and eight side chains, and the cyclization was achieved through the formation of the thioether bridge by sulfur-mediated S<sub>N</sub>2 reaction (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>A). As a result, the OBTC combinatorial library was expected to have a theoretical diversity of 320 000 compounds which were displayed in triple. The linear peptide coding tag contained seven α-amino acid residues, which specifically codes every cyclic γ-AApeptide on the same bead.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00607/20210723/images/medium/jm1c00607_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00607/20210723/images/large/jm1c00607_0002.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. (A) Scheme for the design of γ-AApeptides OBTC library. The binding affinity of <b>M-2-2-F</b> (B) and <b>M-2-5-F</b> (C) to EGFR measured by fluorescence polarization (FP) assay.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00607/20210723/images/large/jm1c00607_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00607&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">The library was first assessed for its quality after preparation. The MALDI-TOF MS/MS analysis of 10 randomly selected beads revealed that nine beads give unambiguous fragmentation patterns, indicating that the library quality is satisfactory. The on-bead screening for the extracellular domain of EGFR was subsequently conducted.<a onclick="showRef(event, 'ref24 ref25 ref26'); return false;" href="javascript:void(0);" class="ref ref24 ref25 ref26">(24−26)</a> In brief, the OBTC library was first incubated with Fc-Tagged EGFR extracellular domain and then incubated with Dylight 549 conjugated Goat antihuman IgG Fc secondary antibody. Beads showing strong red fluorescence (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00607/suppl_file/jm1c00607_si_001.pdf" class="ext-link">Figure S1</a>, Supporting Information) were fished out from the library and then treated with guanidium chloride (GdmCl) to denature any binding proteins. Next, the beads were treated with CNBr to cleave off the encoding peptide tags from the inner layers, which were sequenced by tandem MS/MS of MALDI afterwards. The chemical structures of five beads could be unambiguously determined.</div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5"> Cyclic Peptides M-2-5 Binds to EGFR and Inhibits EGFR, AKT, and ERK Phosphorylation In Vitro</h3><div class="NLM_p">We first determined whether the identified hits could bind to EGFR in vitro. To this end, fluorescein isothiocyanate (FITC) labeled hits (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00607/suppl_file/jm1c00607_si_001.pdf" class="ext-link">Figure S2</a> in the Supporting Information) were resynthesized and tested for their binding affinity toward the extracellular domain of EGFR using a fluorescence polarization (FP) assay. To our delight, two exhibited high binding affinity toward EGFR; <b>M-2-2-F</b> bound to EGFR with a <i>K</i><sub>D</sub> of 0.55 μM (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>B) while <b>M-2-5-F</b> had a relatively better binding with a <i>K</i><sub>D</sub> of 0.43 μM (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>C), and the others show a quite weak binding to EGFR (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00607/suppl_file/jm1c00607_si_001.pdf" class="ext-link">Figure S3</a> in the Supporting Information).</div><div class="NLM_p">The promising binding affinity prompted us to test the activity of the hits <b>M-2-2</b> (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>B) and <b>M-2-5</b> (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>C) at the cellular level. It is recognized that the extracellular domain of EGFR contains four subdomains, with two of them being used for ligands binding and one of them is engaged in heterodimerization and homodimerization.<a onclick="showRef(event, 'ref27 ref28'); return false;" href="javascript:void(0);" class="ref ref27 ref28">(27,28)</a> The EGFR ligand family is known to include seven transmembrane precursor proteins, including epidermal growth factor (EGF), transforming growth factor-α (TGF-α), and others.<a onclick="showRef(event, 'ref29 ref30'); return false;" href="javascript:void(0);" class="ref ref29 ref30">(29,30)</a> Our cyclic AApeptides, which may bind to any sits on the extracellular domain of EGFR, may compete with natural ligands involved in EGFR binding, resulting in inhibition of the EGFR-mediated signaling pathway including tyrosine kinase activation (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>A).</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00607/20210723/images/medium/jm1c00607_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00607/20210723/images/large/jm1c00607_0003.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. (A) Scheme illustrating the proposed mechanism of cyclic peptides to inhibit EGFR phosphorylation. Chemical structure of (B) <b>M-2-2</b> and (C) <b>M-2-5</b>. (D) Effect of 20 μM <b>M-2-2</b> and <b>M-2-5</b> on EGF-induced EGFR phosphorylation determined using western blot analysis. GAPDH was used as an internal control. Data are representative of three independent experiments. (E) Expression ratio of relative P-EGFR/GAPDH is presented. *<i>P</i> < 0.05 vs EGF stimulation. (F) Starved A549 cells were pretreated with <b>M-2-5</b> (0–50 μM) for 4 h before stimulated with EGF (100 ng/mL) for 5 min. Cell lysates were analyzed by Western blots. Data are representative of three independent experiments. (G) Relative band density of P-EGFR/GAPDH, P-AKT/GAPDH, and P-ERK/GAPDH. The intensity of each strip was analyzed by ImageJ software. (H) Effect of <b>M-2-5</b> on cell proliferation inhibition in vitro. A549 cells were treated with various concentrations of <b>M-2-5</b> in the presence of EGF for 48 h, and cell viability was determined with CCK-8 kits. Means were derived from four replicates (<i>n</i> = 4). **<i>P</i> < 0.05 vs EGF stimulation; ***<i>P</i> < 0.001 vs EGF stimulation.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00607/20210723/images/large/jm1c00607_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00607&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">To determine the effect of both <b>M-2-2</b> and <b>M-2-5</b> on the phosphorylation level of EGFR (P-EGFR), starved A549 cells were pretreated with <b>M-2-2</b> or <b>M-2-5</b> for 4 h before further stimulation with 100 ng/mL of the natural ligand EGF, and cell lysates were analyzed by western blotting. A549 cells were chosen due to their elevated expression of EGFR. As shown in <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>D, stimulation of cells with EGF led to significant tyrosine phosphorylation of EGFR. Interestingly, <b>M-2-2</b> could enhance the activation of EGFR in the presence of EGF at a concentration of 20 μM (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>D,E), suggesting that <b>M-2-2</b> is a potential activator for EGFR phosphorylation by acting as functional mimetics of EGFR ligands. Intriguingly, at the same concentration, <b>M-2-5</b> blocked a vast majority of EGFR activation (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>D,E), which indicated that <b>M-2-5</b> is a potent inhibitor of EGFR. Although <b>M-2-2</b> is of considerable interest for EGFR activation, which is beyond the scope of this article, we focused on attention on <b>M-2-5</b> by investigating the ability of <b>M-2-5</b> to inhibit EGF-stimulated tyrosine phosphorylation of EGFR and downstream ERK and AKT pathways in a dose-dependent manner. Pretreatment of starved A549 cells with <b>M-2-5</b> (0–50 μM) for 4 h (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>F) resulted in a dose-dependent inhibition of EGF stimulation of EGFR phosphorylation (IC<sub>50</sub> ∼ 17.7 μM) and an almost full blockage at 25 μM or more. Moreover, downstream phosphorylation of both ERK and AKT were found to be inhibited by <b>M-2-5</b> as well with a similar potency (IC<sub>50</sub> for ERK is 11.2 μM; IC<sub>50</sub> for AKT is 8.75 μM) in a concentration-dependent manner (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>F,G). Together, these data indicated that <b>M-2-5</b> is an effective inhibitor of EGFR and its downstream signaling pathways.</div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i6" class="anchor-spacer"></div><h3 class="article-section__title" id="_i6"> M-2-5 Inhibits Cancer Cell Proliferation and Cell Migration In Vitro</h3><div class="NLM_p">As cell proliferation is an important aspect of tumorigenesis, the cell proliferation assay was conducted to analyze the impact of <b>M-2-5</b> on EGF-driven proliferation of A549 cells in vitro. A549 cells were treated with serum-reduced EGF-containing (100 ng/mL) DMEM in the presence of <b>M-2-5</b> at different concentrations (0–25 μM) for 48 h. As shown in <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>H, upon the stimulation of EGF, A549 cells demonstrated a significantly enhanced proliferation; however, <b>M-2-5</b> exhibited a strong antiproliferative activity with an IC<sub>50</sub> of 8.23 μM (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>H) and a complete suppression of cell proliferation at ∼10 μM compared to the control group that without EGF stimulation. Indeed, M-2-5 exhibited excellent selectivity toward cancer cells over normal cells (∼8 fold), as its IC<sub>50</sub> for human embryonic kidney HEK293 cells was found to be 62.1 μM.</div><div class="NLM_p">We next investigated <b>M-2-5</b>’s inhibitory effect on cell migration of A549 cells, which plays an important role in tumor metastatic activity. First, a scratch-wound assay was established to investigate cell migratory behaviors. As expected, the percent of the wound closed was notably decreased upon treatment with 25 μM <b>M-2-5</b> (15.2%) for 30 h, leading to 40% decrease compared with untreated control A549 cells (55.3%) (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>A,B). Similar suppression in migration ability was also observed in transwell assay, as shown in <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>C. Compared with the untreated control group, the number of migrating cells were distinctly increased to 133.9% in the presence of EGF (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>D), which were dramatically decreased to 64.8% with <b>M-2-5</b> (25 μM) treatment. Taken together, <b>M-2-5</b> has been shown to significantly inhibit A549 cell proliferation and cell migration ability.</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00607/20210723/images/medium/jm1c00607_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00607/20210723/images/large/jm1c00607_0004.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. <b>M-2-5</b> inhibits cell migration in vitro. (A) Wound-healing assay was performed on A549 cells in the absence or presence of <b>M-2-5</b> (10 and 25 μM). The experiments were repeated three times with similar results, and images of one representative experiment taken at 0 and 30 h after wound scratching are shown. A549 cells were photographed at 4× magnification. (B) Percentage of wound healing covered at 30 h. **<i>P</i> < 0.01 compared with control at 30 h; ***<i>P</i> < 0.001 compared with control at 30 h. (C) Transwell assay migration assay of A549 cells upon treatment of EGF alone or EGF+ <b>M-2-5</b> (25 μM). The A549 cells without EGF and <b>M-2-5</b> treatment were used as control. After stained with crystal violet, A549 cells were photographed at 20× magnification. (D) Quantitative results of migration assays. The columns represent the mean of cell numbers from at least three independent experiments. The relative cell migration was counted manually, and the number of migration cells in the control group was set to 100%. *<i>P</i> < 0.05 compared with control at 12 h, ***<i>P</i> < 0.001 compared with EGF stimulation at 12 h.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00607/20210723/images/large/jm1c00607_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00607&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i7" class="anchor-spacer"></div><h3 class="article-section__title" id="_i7"> Stability of M-2-5 toward Enzymatic Hydrolysis</h3><div class="NLM_p last">As <b>M-2-5</b> is a promising ligand to block EGFR signaling in vitro, we thus set out to further evaluate its enzymatic stability against pronase (the enzyme that is capable of digesting regular peptides into single amino acids), which could provide a strong basis for their potential biological activity in vivo. As anticipated, incubation of <b>M-2-5</b> with pronase for 20 h did not lead to detectable degradation (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00607/suppl_file/jm1c00607_si_001.pdf" class="ext-link">Figure S4</a> in the Supporting Information), demonstrating its remarkable resistance to proteolytic degradation.</div></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8"> Parallel Artificial Membrane Permeability Assays (PAMPA) of M-2-5</h3><div class="NLM_p">The parallel artificial membrane permeability assay (PAMPA) is widely used as a high-throughput permeability assay to evaluate drug candidates.<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> As such, we conducted both the blood–brain barrier (BBB) parallel artificial membrane permeability assay (PAMPA-BBB) and the gastrointestinal tract (GIT) parallel artificial membrane permeability assay (PAMPA-GIT) to further evaluate the potential of <b>M-2-5</b> as a drug candidate. The PAMPA-BBB is used to predict the ability of a drug to cross the BBB.<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> The PAMPA-GIT is used for the evaluation of how the oral drug candidate might be absorbed across the entire GIT.<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> In the PAMPA-BBB assay, we used Verapamil and Lidocaine as the positive and negative control, respectively. Verapamil can easily cross the BBB,<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> exhibiting a very high permeability (<i>P</i><sub>app</sub>) value in permeability assay (<i>P</i><sub>app</sub> > 20 × 10<sup>–6</sup> cm/s), whereas Lidocaine could hardly pass through the BBB. As shown in <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>A, Verapamil in the concentration of 50 μM exhibited a favorable <i>P</i><sub>app</sub> ((169 ± 4) × 10<sup>–6</sup> cm/s), as we expected, Lidocaine displayed a lower <i>P</i><sub>app</sub> value ((94 ± 3) × 10<sup>–6</sup> cm/s) even with a quite high concentration. Surprisingly, <b>M-2-5</b> showed a fabulous high <i>P</i><sub>app</sub> value of 266 (±6) × 10<sup>–6</sup> cm/s even in the concentration of 50 μM. The result demonstrated that <b>M-2-5</b> possesses the promising potential to be further investigated as a novel drug candidate. For the PAMPA-GIT assay, it is significant to test different PH conditions since human intraluminal pH is different in stomach, duodenum, ileum, cecum, and rectum.<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> Carbamazepine and antipyrine were included as the positive and negative control for the assay. Carbamazepine is fully orally bioavailable (>90% absorption in the human gastrointestinal tract); however, antipyrine is considered as a poorly orally bioavailable compound that gives low <i>P</i><sub>app</sub> values. As shown in <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>B, <b>M-2-5</b> exhibited favorable <i>P</i><sub>app</sub> without pH-dependence. The findings indicated the <b>M-2-5</b> has the potential of oral bioavailability for future drug development.</div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00607/20210723/images/medium/jm1c00607_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00607/20210723/images/large/jm1c00607_0005.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. PAMPA assays of <b>M-2-5</b>. All compounds were tested in triplicate. (A) PAMPA-BBB assay on standards and <b>M-2-5</b> at pH 7.4. (B) PAMPA-GIT assay on standards and <b>M-2-5</b> at different pH conditions.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00607/20210723/images/large/jm1c00607_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00607&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00607/20210723/images/medium/jm1c00607_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00607/20210723/images/large/jm1c00607_0006.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Therapeutic efficacy of <b>M-2-5</b> in A549 xenograft models. (A) Total tumor volume assessed on a time course. The day when drugs were initially administered was marked as day 0, and arrows show the injection time. Tumor volume was obtained once every 2 days until day 18. (B) Western blot analysis of phosphor-EGFR and downstream signaling pathway in tumor lysates. GAPDH was used as a loading control. (C) Immunohistochemical staining for P-EGFR, P-ERK, and P-AKT.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00607/20210723/images/large/jm1c00607_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00607&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i9" class="anchor-spacer"></div><h3 class="article-section__title" id="_i9"> Short-Term Therapy of M-2-5 In Vivo</h3><div class="NLM_p last">Based upon the observed strong inhibition of cell proliferation and migration, the ability to inhibit EGFR cell signaling pathways in vitro, as well as the remarkable stability against proteolytic degradation, we next evaluated the antitumor activity of <b>M-2-5</b> in a A549 lung tumor xenograft mouse model in vivo. PBS was used as a negative control, while <b>C225</b> (cetuximab), a monoclonal antibody, was used as a positive control, which was approved by FDA for treating lungs, head, and neck cancer. Mice were intraperitoneal administered with PBS, <b>C225</b> (40 mg/kg) and <b>M-2-5</b> (10, 50 mg/kg), respectively, every 2 days for seven times. After another 6 days post injection, mice were then sacrificed, from which tumors were resected. As shown in <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>A, a substantial reduction in the tumor volume was observed comparing the PBS group with the group receiving <b>C225</b> therapy. Though no significant differences were found for 10 mg/kg of <b>M-2-5</b> treatment, the injection of <b>M-2-5</b> in high concentration (50 mg/kg) demonstrated significant suppression of tumor growth. In addition, injection of the compound did not affect the body weight gain of A549 tumor xenografted mice (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00607/suppl_file/jm1c00607_si_001.pdf" class="ext-link">Figure S5</a> in the Supporting Information). To confirm the expression level of P-EGFR and downstream signaling, total proteins were then extracted from the tumor tissues in each group and subjected to immunoblotting of P-EGFR, P-ARK, and P-ERK. Western blot analysis (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>B) show that either <b>C225</b> or <b>M-2-5</b> (50 mg/kg) causes a dramatic decrease in P-EGFR and P-AKT, whereas <b>M-2-5</b> (50 mg/kg) treatment exhibited slightly weaker efficiency in the inhibition of P-ERK compared to <b>C225</b>, which may account for the difference on inhibition of tumor growth. In addition, the level of phosphorylation for EGFR in A549 tumor tissues was investigated by immunohistochemical staining. Consistent with western blot analysis, the staining intensities of P-EGFR, P-AKT, and P-ERK were decreased, respectively, compared with the PBS group (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>C). Overall, the in vivo studies revealed that <b>M-2-5</b> exhibited similar antitumor efficacy compared with cetuximab in the A549 xenografted model. Compared to cetuximab, <b>M-2-5</b> has much smaller molecular weight and therefore low production cost. Together with its similar function to monoclonal antibody and its high resistance toward proteolytic degradation, it could be further explored as promising anticancer agents for clinical drug development.</div></div><div id="sec2_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i10" class="anchor-spacer"></div><h3 class="article-section__title" id="_i10"> Molecular Docking Study</h3><div class="NLM_p">As <b>M-2-5</b> could directly target the extracellular domain of EGFR and show comparable in vivo efficacy, its binding mode maybe analogous to that of cetuximab. As such, we carried a molecular docking study and tried to gain insight into the binding of <b>M-2-5</b> toward EGFR. As shown in <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>, the docking of <b>M-2-5</b> with the extracellular domain of EGFR (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1YY9">1YY9</a>) suggested that <b>M-2-5</b> could occupy the cetuximab binding site of EGFR. It was observed that an amino group and a carbonyl group of <b>M-2-5</b>, respectively, oriented to form two hydrogen bonds with glutamine side chains in EGFR (Q384 and Q411). It is noticed that Q384 is also the key residue for the interaction between EGFR and cetuximab.<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> The docking results strongly supported the potential of <b>M-2-5</b> as a novel artificial antibody targeting EGFR.</div><figure id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00607/20210723/images/medium/jm1c00607_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00607/20210723/images/large/jm1c00607_0007.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Molecular docking of <b>M-2-5</b> with EGFR extracellular domain (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1YY9">1YY9</a>). The docking was performed using the Glide module in Schrödinger (2015) with default parameters, and the docking results were visualized in Pymol.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00607/20210723/images/large/jm1c00607_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00607&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i11">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_64407" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_64407" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">In conclusion, through screening an OBTC combinatorial library, a novel cyclic γ-AApeptides <b>M-2-5</b> was identified to inhibit EGFR phosphorylation. Our result implicated that <b>M-2-5</b> could interfere EGFR-EGF binding, resulting in inhibition of EGFR activation and downstream signaling pathways, as well as suppression of cell proliferation and migration. In addition, unlike regular peptides, <b>M-2-5</b> is completely resistant to proteolytic degradation. More importantly, <b>M-2-5</b> also shows excellent antitumor efficacy in vivo. Compared with known small molecular inhibitors of EGFR, which are mainly tyrosine kinase inhibitors (TKIs), <b>M-2-5</b> has a few advantages. First, unlike those tyrosine kinase inhibitors which targeting kinase site in the intracellular domain of EGFR, <b>M-2-5</b> targets the extracellular domain of EGFR, and therefore it does not have to have good cell permeability. Meanwhile, it is known that tyrosine kinase inhibitors (TKIs) lack good selectivity as the kinase sites are highly conserved; this is why generally TKIs exhibit high toxicity and frequently lead to drug resistance. On the contrary, <b>M-2-5</b> shows excellent selectivity between cancer cells and normal cells. The ability of <b>M-2-5</b> to mimic monoclonal antibody, as well as its small molecular weight and high stability, could enable it a promising candidate for the discovery of novel anticancer agents. The result also suggested that the approach of OBTC γ-AApeptide library provides an effective strategy for discovering human cancer therapy.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i12">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_15732" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_15732" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13"> Materials</h3><div class="NLM_p">All chemicals were purchased from commercial suppliers and used without further purification. Fmoc-protected amino acids were purchased from Chem-impex. TentaGel resin (0.23 mmol/g) was purchased from RAPP Polymere. Rink Amide-MBHA resin (0.55 mmol/g) was purchased from GL Biochem. Solid-phase synthesis was conducted in peptide synthesis vessels on a Burrell Wrist-Action shaker. Cyclic γ-AApeptides were analyzed and purified on a Waters Breeze 2 HPLC system and then lyophilized on a Labcono lyophilizer. The purity of the compounds was determined to be >95% by analytical HPLC. Masses of γ-AApeptides and the MS/MS analysis were obtained on an Applied Biosystems 4700 Proteomics analyzer.</div><div class="NLM_p last">The A549 cell line was kindly provided by Prof. Lixin Wan at the Moffitt Cancer Center, Tampa. EGF was purchased from Fisher Scientific; the extracellular domain of EGFR was purchased from Creative BioMart; anti-phospho-EGFR antibody was purchased from Life Technologies; anti-phospho-AKT and anti-phospho-ERK antibodies were from Cell Signaling Technology; and GAPDH Loading control monoclonal antibody was purchased from Invitrogen. PAMPA assays were performed on a TECAN Fredom EVO150 robot and analyzed by the pION’s PAMPA evolution software.</div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14"> One-Bead-Two-Compound Library Synthesis, Screening, and Analysis</h3><div class="NLM_p">The one-bead-two-compound library was synthesized as our previous work<a onclick="showRef(event, 'ref24 ref25 ref26'); return false;" href="javascript:void(0);" class="ref ref24 ref25 ref26">(24−26)</a> and the scheme for library synthesis is provided in the <a href="/doi/suppl/10.1021/acs.jmedchem.1c00607/suppl_file/jm1c00607_si_001.pdf" class="ext-link">Supporting Information</a>.</div><div class="NLM_p">For the EGFR-targeted library screening, it contains two main parts, prescreening and screening.<a onclick="showRef(event, 'ref37 ref38'); return false;" href="javascript:void(0);" class="ref ref37 ref38">(37,38)</a> First, for the prescreening, all of the TentaGel beads were swelled in DMF for 1 h. After being washed five times with Tris buffer, the beads were equilibrated in Tris buffer overnight at room temperature, followed by incubation with the blocking buffer (1% BSA in Tris buffer with a 1000× excess of <i>Escherichia coli</i> lysate) for 1 h. After a thorough wash with Tris buffer, the beads were incubated with Goat antihuman IgG Fc cross-adsorbed secondary antibody, Dylight 549 (1:1000 dilution), for 2 h at room temperature. The beads were washed five times with the Tris buffer and then the beads emitting red fluorescence were picked up manually and excluded from formal screening.</div><div class="NLM_p">Second, for the screening, the rest of the beads after prescreening were washed with Tris buffer and treated with 8 M guandine·HCl at room temperature, after 1 h, the beads were washed by DI water (5×), tris buffer (5×), and DMF (5×). The beads were then incubated in DMF for 1 h, followed by washing and equilibration in Tris buffer overnight. The beads were incubated in 1% BSA/Tris buffer and 1000× excess of <i>E. coli</i> lysate for 1 h at room temperature. After washed five times Tris buffer, the beads were incubated with EGFR protein at a concentration of 50 nM for 4 h at room temperature with a 1000× excess of <i>E. coli</i> lysate. After the thorough wash with Tris buffer, the library beads were incubated with and Goat antihuman IgG Fc cross-adsorbed secondary antibody, Dylight 549 (1:1000 dilution), for 2 h at room temperature. The beads were washed five times with the Tris buffer and then the beads emitting red fluorescence were picked up for future analysis.</div><div class="NLM_p last">For the hit structure analysis, each hit was transferred to an Eppendorf microtube and denatured in 100 μL 8 M guanidine·HCl for 1 h at room temperature. The bead was rinsed with Tris buffer 3 × 10 min, water 3 × 10 min, DMF 3 × 10 min, and ACN 3 × 10 min. At last, the resin was placed in ACN overnight in each microtube and then ACN was evaporated. The bead was incubated in the cocktail of 5:4:1 (v/v/v) of ACN/glacial acetic acid/H<sub>2</sub>O containing cyanogen bromide (CNBr) at a concentration of 50 mg/mL overnight at room temperature. The cleavage solution was then evaporated, and the cleaved peptide was dissolved in ACN/H<sub>2</sub>O (4:1) and subjected to MALDI-TOF for MS/MS analysis.</div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i15" class="anchor-spacer"></div><h3 class="article-section__title" id="_i15"> Synthesis of Cyclic γ-AApeptides</h3><div class="NLM_p">The FITC-labeled hits were resynthesized on the Rink Amide resin.<a onclick="showRef(event, 'ref25 ref26'); return false;" href="javascript:void(0);" class="ref ref25 ref26">(25,26)</a> Briefly, the Fmoc-Lys (Dde)-OH was first attached to the Rink amide resin. The Fmoc protection group was then removed, followed by the desired building blocks needed for the sequence synthesis. After the γ-AApeptides were cyclized, the Dde group was removed and Fmoc-β-Ala was added, the Fmoc protection group was then removed and FITC (2 equiv) and DIPEA (6 equiv) in DMF were added to the resin and shaken for 12 h at room temperature. The FITC-labeled cyclic γ-AApeptides was cleaved by 1:1 (v/v) DCM/TFA containing 2% triisopropylsilane. The crude was purified by the Waters HPLC system.</div><div class="NLM_p"><b>M-2-1-F</b>: MS: calcd. For C<sub>105</sub>H<sub>125</sub>N<sub>15</sub>NaO<sub>18</sub>S<sub>2</sub><sup>+</sup> [(M + Na)<sup>+</sup>]: 1870.8661; MALDI-TOF found: <i>m</i>/<i>z</i> 1971.2394.</div><div class="NLM_p"><b>M-2-2-F</b>: MS: calcd. For C<sub>106</sub>H<sub>125</sub>N<sub>13</sub>NaO<sub>20</sub>S<sub>2</sub><sup>+</sup> [(M + Na)<sup>+</sup>]: 1986.8497; MALDI-TOF found: <i>m</i>/<i>z</i> 1987.3434.</div><div class="NLM_p"><b>M-2-3-F</b>: MS: calcd. For C<sub>101</sub>H<sub>122</sub>N<sub>15</sub>O<sub>18</sub>S<sub>2</sub><sup>+</sup> [(M + H)<sup>+</sup>]: 1896.8528; MALDI-TOF found: <i>m</i>/<i>z</i> 1897.3221.</div><div class="NLM_p"><b>M-2-4-F</b>: MS: calcd. For C<sub>109</sub>H<sub>120</sub>N<sub>14</sub>NaO<sub>22</sub>S<sub>2</sub><sup>+</sup> [(M + Na)<sup>+</sup>]: 2063.8035; MALDI-TOF found: <i>m</i>/<i>z</i> 2064.2830.</div><div class="NLM_p"><b>M-2-5-F</b>: MS: calcd. For C<sub>110</sub>H<sub>130</sub>N<sub>14</sub>NaO<sub>18</sub>S<sub>2</sub><sup>+</sup> [(M + Na)<sup>+</sup>]: 2023.4392; MALDI-TOF found: <i>m</i>/<i>z</i> 2023.5481.</div><div class="NLM_p">The synthesis of <b>M-2-2</b> and <b>M-2-5</b> was conducted on the Rink Amide resin with general solid-phase synthesis. After the γ-AApeptides were cyclized, the compounds were cleaved by 1:1 (v/v) DCM/TFA containing 2% triisopropylsilane and purified by the Waters HPLC system.</div><div class="NLM_p"><b>M-2-2</b>: MS: calcd. For C<sub>76</sub>H<sub>97</sub>N<sub>6</sub>NaO<sub>13</sub>S<sup>+</sup> [(M + Na)<sup>+</sup>]: 1398.6819; MALDI-TOF found: <i>m</i>/<i>z</i> 1398.8224.</div><div class="NLM_p last"><b>M-2-5</b>: MS: calcd. For C<sub>80</sub>H<sub>102</sub>N<sub>10</sub>NaO<sub>11</sub>S<sup>+</sup> [(M + Na)<sup>+</sup>]: 1434.8042; MALDI-TOF found: <i>m</i>/<i>z</i> 1434.1172.</div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i16" class="anchor-spacer"></div><h3 class="article-section__title" id="_i16"> Fluorescence Polarization Assay</h3><div class="NLM_p last">The FP experiment was performed by incubating 50 nM FITC-labeled AApeptides with EGFR (0–2 μM) in PBS. Dissociation constants (<i>K</i><sub>d</sub>) were determined by plotting fluorescence anisotropy values as a function of protein concentration, and the plots were fitted to the following equation. Lst is the concentration of the AApeptides and <i>x</i> stands for the concentration of the protein. The experiments were conducted in triplicates and repeated three times.<span class="NLM_disp-formula" id="ueq1"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00607/20210723/images/medium/jm1c00607_m001.gif" alt="" id="_i17" /></img></span></div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> Enzymatic Stability Assay</h3><div class="NLM_p last">Cyclic γ-AApeptides <b>M-2-5</b> (0.1 mg/mL) were incubated with 0.1 mg/mL protease in 100 mM ammonium bicarbonate buffer (pH 7.8) at 37 °C for 20 h. Then, the reaction mixtures were concentrated in a speed vacuum to remove water and ammonium bicarbonate. The resulting residues were redissolved in H<sub>2</sub>O/MeCN and analyzed on a Waters analytical HPLC system.</div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i19" class="anchor-spacer"></div><h3 class="article-section__title" id="_i19"> Cell Cultures and Inhibition of EGF-Induced Cell Proliferation</h3><div class="NLM_p last">A549 cells were cultured in a high-glucose DMEM (Gibco) medium containing 10% fetal bovine serum and 1% penicillin/streptomycin in an atmosphere of 5% CO<sub>2</sub> at 37 °C. A549 cells in good condition were seeded into a 96-well plate at a concentration of 1 × 10<sup>3</sup> cells/well in 100 μL of complete growth medium. After 24 h of attachment at 37 °C and 5% CO<sub>2</sub>, the medium was replaced by fresh serum-reduced DMEM and cells were serum-starved overnight. Then, serum-reduced EGF (100 ng/mL) DMEM in the presence of different concentrations of <b>M-2-5</b> was added to the cells in hexa-duplicate. After 48 h, CCK-8 reagents were added according to the manufacturer’s recommendations. The cytotoxicity of <b>M-2-5</b> toward HEK293 cells was conducted in a similar fashion, except no EGF was used to stimulate the cell growth.</div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i20" class="anchor-spacer"></div><h3 class="article-section__title" id="_i20"> Western Blot Assay</h3><div class="NLM_p last">A549 cells were seeded into a six-well plate at a concentration of 1 × 10<sup>5</sup> cells/well. After 12 h attachment at 37 °C and 5% CO<sub>2</sub>, the cells were starved overnight in serum-reduced DMEM followed by treatment with different concentrations of <b>M-2-5</b> for 4 h. The cells were further stimulated with 100 ng/mL EGF for 10 min, washed with ice-cold PBS, and resuspended in ice-cold RIPA buffer supplemented with Halt Protease and Phosphatase Inhibitor Cocktail. Subsequently, the cells were incubated on ice for 10 min and centrifuged at 14 000<i>g</i> at 4 °C for 10 min. An equal amount of protein was run on 4–2% Bis-Tris gels, transferred to poly(vinylidene difluoride) membranes (Millipore) and western blotted with antiphosphorylated EGFR, antiphosphorylated AKT, antiphosphorylated ERK, and GAPDH. The experiments were conducted in triplicates and repeated three times.</div></div><div id="sec4_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i21" class="anchor-spacer"></div><h3 class="article-section__title" id="_i21"> Cell Wound-Healing Assay</h3><div class="NLM_p last">A549 cells in good condition were seeded into a 12-well plate at a concentration of 5 × 10<sup>4</sup> cells/well. After 24 h attachment at 37 °C and 5% CO<sub>2</sub>, they reached confluence as a monolayer. The monolayer was scratched gently and slowly with a new 200 μL pipette tip across the center of the well straightly. After scratching, the well was washed twice with medium to remove the detached cells. The well was then replenished with a fresh medium containing different concentrations of <b>M-2-5</b>. The photo was taken for the scratched monolayer and then captured again after 30 h for the same wound. The wound width was measured by ImageJ. The experiments were conducted in triplicates and repeated three times.</div></div><div id="sec4_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i22" class="anchor-spacer"></div><h3 class="article-section__title" id="_i22"> Transwell Assay</h3><div class="NLM_p last">A549 cells in good condition were seeded into a 100 mm dish at a concentration of 5 × 10<sup>5</sup> cells. After overnight attachment at 37 °C and 5% CO<sub>2</sub>, the culture medium was replaced with a serum-reduced DMEM, and the dish was returned to the incubator for 24 h. Cells were detached and resuspended in DMEM at a concentration of 1 × 10<sup>6</sup> cells/mL. Next, 100 μL of the cell solution was plated on top of the filter membrane in a transwell insert and incubated for 10 min at 37 °C and 5% CO<sub>2</sub> to allow the cells to settle down. Subsequently, 600 μL of the desired chemoattractant was added very carefully into the bottom of the lower chamber in a 24-well plate. The culture was incubated for 12 h, and then the transwell insert was removed from the plate. A cotton-tipped applicator was used as many times as needed to carefully swab the inside of each insert. Cells were fixed by 70% ethanol, stained by 0.2% crystal violet, and visualized with a microscope. Transwell data were determined by the number of the migrated cells, and the value from parental cells was arbitrarily set at 100%. The experiments were conducted in triplicates and repeated three times.</div></div><div id="sec4_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i23" class="anchor-spacer"></div><h3 class="article-section__title" id="_i23"> PAMPA-BBB Assay</h3><div class="NLM_p last">The PAMPA-BBB assay procedure was realized using a method developed by pION. All liquid handling steps for the PAMPA assay are performed on a TECAN Fredom EVO150 robot and analyzed by the pION’s PAMPA evolution software. The pION’s PAMPA-BBB includes the brain sink buffer (BSB), the BBB-1 lipid solution, and the StirwellTM PAMPA sandwich plate, preloaded with magnetic disks. Four microliters of lipid were transferred into the acceptor well, followed by the addition of 200 μL of BSB (pH 7.4). Then, 180 μL of the diluted test compound (15–250 μM in system buffer at pH 7.4 from a 10 mM DMSO solution) was added to the donor wells. The PAMPA sandwich plate was assembled and placed on the Gut-BoxTM and stirred with 60 μm aqueous boundary layer (ABL) settings for 1 h. The distribution of the compounds in the donor and acceptor buffers (150 μL aliquot) was determined by UV spectra measurement from 200 to 500 nm using the Tecan infinite M-1000 pro microplate reader. Then, the permeability (<i>P</i><sub>app</sub>, 10<sup>–6</sup> cm/s) of each compound was calculated by the pion PAMPA evolution software. Experiments were repeated three times.</div></div><div id="sec4_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i24" class="anchor-spacer"></div><h3 class="article-section__title" id="_i24"> PAMPA-GIT Assay</h3><div class="NLM_p last">The PAMPA-GIT assay procedure was realized using a method developed by pION. All liquid handling steps for the PAMPA assay are performed on a TECAN Fredom EVO150 robot and analyzed by the pION’s PAMPA evolution software. The pION’s GIT PAMPA includes the acceptor sink buffer (ASB), the GIT-0 lipid solution, and the StirwellTM PAMPA sandwich plate, preloaded with magnetic disks. Four microliters of lipid were transferred into the acceptor well, followed by the addition of 200 μL of ASB (pH 7.4). Then, 180 μL of the diluted test compound (50–250 μM in system buffer at pH 5.0, 6.2, and 7.4 from a 10 mM DMSO solution) was added to the donor wells. The PAMPA sandwich plate was assembled and placed on the Gut-BoxTM and stirred with 40 μm ABL settings for 30 min. The distribution of the compounds in the donor and acceptor buffers (150 μL aliquot) was determined by UV spectra measurement from 200 to 498 nm using the Tecan infinite M-1000 pro microplate reader. Then, the permeability (<i>P</i><sub>app</sub>, 10<sup>–6</sup> cm/s) of each compound was calculated by the pion PAMPA evolution software. The assays were repeated three times.</div></div><div id="sec4_12" class="NLM_sec NLM_sec_level_2"><div id="ac_i25" class="anchor-spacer"></div><h3 class="article-section__title" id="_i25"> Antitumor Studies in Nude Mice</h3><div class="NLM_p">Male athymic nude mice (NCr-nu), 8–12 weeks old, were provided by the Shanghai Bikai Experimental Animal Center, with the license number SCXK (Hu) 2008-0016, and maintained under specific-pathogen-free conditions. All animal protocols were approved by the Institutional Animal Use and Care Committee. All of the experiments and animal care were approved by Shanghai Medical Experimental Animal Care Commission and in accordance with the Provision and General Recommendation of Chinese Experimental Animals Administration Legislation.</div><div class="NLM_p">A549 cells were harvested, resuspended in PBS, and injected subcutaneously into 4–6 weeks old male BALB/c nude mice. When the tumors reach an average size of 100 mm<sup>3</sup>, the mice were then randomized divided into four treatment groups: control (PBS), <b>C-225</b> (40 mg/kg), <b>M-2-5</b> (10 mg/kg), and <b>M-2-5</b> (50 mg/kg). Mice were injected intraperitoneally every 2 days for seven injections; 6 days after injection, the mice in each group were killed, and the tumors were removed for examination. The tumor volumes were determined by measuring length (<i>l</i>) and width (<i>w</i>) and calculating volume (<i>V</i> = <i>lw</i><sup>2</sup>/2).</div><div class="NLM_p">For the detection of phosphorylated EGFR, AKT, and ERK in tumors in vivo, a portion of the tumor (80 mg) was immersed in 500 μL of the extraction buffer (50 mM Tris–HCl (pH 7.4), 150 mM NaCl, 1% NP-40, 0.25% Na-deoxycholate, protease inhibitor, SDS 0.1–2%), and tubes were vortexed briefly. The supernatant was transferred to a new tube and samples were centrifuged at 14 000 rpm for 15 min at 4 °C to remove any remaining insoluble material. The supernatant containing soluble proteins was transferred to a new tube and an aliquot was taken for the quantification and further western blot analysis.</div><div class="NLM_p last">For the immunohistochemical analysis, the hydrated paraffin section was incubated in a blocking solution (10% donkey serum + 5% nonfat dry milk + 4% BSA + 0.1% Triton X-100) for 10 min and then incubated at 4 °C overnight with anti-P-EGFR, P-AKT, and P-ERK. After washing with PBS, the sections were incubated with diluted (1:200) biotinylated secondary antibody for 30 min. Subsequently, the sections were washed again in PBS and incubated for 30 min with the preformed avidin–horseradish peroxidase macromolecular complex. Development of peroxidase reaction was achieved by incubation in 0.01% 3,3-diaminobenzidine tetrahydrochloride (DAB) in PBS containing 0.01% hydrogen peroxide for approximately 5 min at room temperature. Sections were then washed thoroughly in tap water, counterstained in haematoxylin, dehydrated in absolute alcohol, cleared in xylene, and mounted in synthetic resin for microscopic examination.</div></div><div id="sec4_13" class="NLM_sec NLM_sec_level_2"><div id="ac_i26" class="anchor-spacer"></div><h3 class="article-section__title" id="_i26"> Molecular Docking Methods</h3><div class="NLM_p last">The three-dimensional structure of the molecule <b>M-2-5</b> was constructed using RDKit package.<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> The conformational search of <b>M-2-5</b> was carried out using mixed torsional/low-mode sampling as implemented in Schrödinger (2015) with AMBER force field. The default parameters were used for the rest of the settings. The protein complex, showing the structure of the extracellular domain of the epidermal growth factor receptor in complex with the Fab fragment of cetuximab (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1YY9">1YY9</a>), was selected for docking. After removed waters, cofactors, oligosaccharides, salt anion, metal ions, or redundant chains using PyMol,<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1YY9">1YY9</a> was prepared using Schrödinger Protein Preparation Wizard<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> with default settings. Grid was generated using the centroid of the interaction surface as the center for each crystal structure. The docking was performed using the Glide module in Schrödinger (2015) with default parameters.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i27"><a href="/doi/suppl/10.1021/acs.jmedchem.1c00607" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_57852" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_57852" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a href="/doi/10.1021/acs.jmedchem.1c00607?goto=supporting-info" class="ext-link">https://pubs.acs.org/doi/10.1021/acs.jmedchem.1c00607</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Representative picture of fluorescent beads; chemical structures of FITC-labeled hits; binding affinity of hits determined by FP; HPLC traces of hits; stability study toward enzyme hydrolysis; one-bead-two-compound bead screening protocol; and MALDI data for hits (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00607/suppl_file/jm1c00607_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular formula strings for final compounds (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00607/suppl_file/jm1c00607_si_002.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00607/suppl_file/jm1c00607_si_001.pdf">jm1c00607_si_001.pdf (1.28 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00607/suppl_file/jm1c00607_si_002.csv">jm1c00607_si_002.csv (1.91 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.1c00607" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_16725" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_16725" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Qi Li</span> - <span class="hlFld-Affiliation affiliation">Department
of Medical Oncology & Cancer Institute of Medicine, Shuguang Hospital, Shanghai University of Traditional Chinese Medicine, Shanghai 201203, China</span>; 
    <span class="hlFld-Affiliation affiliation">Academy
of Integrative Medicine, Shanghai University
of Traditional Chinese Medicine, Shanghai 201203, China</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#8ffee6e3e6cffce7fafbece2a1eaebfaa1ece1"><span class="__cf_email__" data-cfemail="87f6eeebeec7f4eff2f3e4eaa9e2e3f2a9e4e9">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yan Wang</span> - <span class="hlFld-Affiliation affiliation">Department
of Medical Oncology & Cancer Institute of Medicine, Shuguang Hospital, Shanghai University of Traditional Chinese Medicine, Shanghai 201203, China</span>; 
    <span class="hlFld-Affiliation affiliation">Academy
of Integrative Medicine, Shanghai University
of Traditional Chinese Medicine, Shanghai 201203, China</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#bac3dbd4cddbd4ddfac9d2cfced9d794dfdecf94d9d4"><span class="__cf_email__" data-cfemail="3148505f46505f567142594445525c1f5455441f525f">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jianfeng Cai</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry, University of South Florida, 4202 E. Fowler Avenue, Tampa, Florida 33620, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0003-3106-3306" title="Orcid link">https://orcid.org/0000-0003-3106-3306</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#743e1d151a12111a1317151d340107125a111001"><span class="__cf_email__" data-cfemail="db91b2bab5bdbeb5bcb8bab29baea8bdf5bebfae">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Mengmeng Zheng</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry, University of South Florida, 4202 E. Fowler Avenue, Tampa, Florida 33620, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Chunpu Li</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry, University of South Florida, 4202 E. Fowler Avenue, Tampa, Florida 33620, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Medical Oncology & Cancer Institute of Medicine, Shuguang Hospital, Shanghai University of Traditional Chinese Medicine, Shanghai 201203, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Mi Zhou</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry, University of South Florida, 4202 E. Fowler Avenue, Tampa, Florida 33620, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ru Jia</span> - <span class="hlFld-Affiliation affiliation">Department
of Medical Oncology & Cancer Institute of Medicine, Shuguang Hospital, Shanghai University of Traditional Chinese Medicine, Shanghai 201203, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Gang Cai</span> - <span class="hlFld-Affiliation affiliation">Department
of Medical Oncology & Cancer Institute of Medicine, Shuguang Hospital, Shanghai University of Traditional Chinese Medicine, Shanghai 201203, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Fengyu She</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry, University of South Florida, 4202 E. Fowler Avenue, Tampa, Florida 33620, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Lulu Wei</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry, University of South Florida, 4202 E. Fowler Avenue, Tampa, Florida 33620, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0003-4657-9479" title="Orcid link">https://orcid.org/0000-0003-4657-9479</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Shaohui Wang</span> - <span class="hlFld-Affiliation affiliation">Department
of Molecular Medicine, Morsani College of Medicine, University of South Florida, Tampa, Florida 33612, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jie Yu</span> - <span class="hlFld-Affiliation affiliation">Drug
Discovery and Design Center, Shanghai Institute
of Materia Medica, Chinese Academy of Sciences, Shanghai 201203, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Dingyan Wang</span> - <span class="hlFld-Affiliation affiliation">Drug
Discovery and Design Center, Shanghai Institute
of Materia Medica, Chinese Academy of Sciences, Shanghai 201203, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Laurent Calcul</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry, University of South Florida, 4202 E. Fowler Avenue, Tampa, Florida 33620, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Xingmin Sun</span> - <span class="hlFld-Affiliation affiliation">Department
of Molecular Medicine, Morsani College of Medicine, University of South Florida, Tampa, Florida 33612, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Xiaomin Luo</span> - <span class="hlFld-Affiliation affiliation">Drug
Discovery and Design Center, Shanghai Institute
of Materia Medica, Chinese Academy of Sciences, Shanghai 201203, China</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0003-0426-3417" title="Orcid link">https://orcid.org/0000-0003-0426-3417</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Feng Cheng</span> - <span class="hlFld-Affiliation affiliation">Department
of Pharmaceutical Sciences, College of Pharmacy, University of South Florida, Tampa, Florida 33612, United States</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>Mengmeng Zheng, C.L., and Mi Zhou contributed equally to this work.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ACK-d7e1638-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i29">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_36775" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_36775" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">This work was supported by the USF start-up fund (J.C.) and the National Natural Science Foundation of China (Q.L.) (81520108031, 81573764, 81774095), a Municipal Human Resources Development Program for Outstanding Leaders in Medical Disciplines in Shanghai (Q.L.) (2017BR031). Three-year Plan for the Development of TCM (ZY (2018-2020)—CCCX-2003-03).</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations</td></tr><tr><td class="NLM_term">γ-AApeptides</td><td class="NLM_def"><p class="first last">γ-substituted-N-acylated-<i>N</i>-aminoethyl peptides</p></td></tr><tr><td class="NLM_term">DMEM</td><td class="NLM_def"><p class="first last">Dulbecco’s modified Eagle’s medium</p></td></tr><tr><td class="NLM_term">EGF</td><td class="NLM_def"><p class="first last">epidermal growth factor</p></td></tr><tr><td class="NLM_term">EGFR</td><td class="NLM_def"><p class="first last">epidermal growth factor receptor</p></td></tr><tr><td class="NLM_term">OBTC</td><td class="NLM_def"><p class="first last">one-bead-two-compound</p></td></tr><tr><td class="NLM_term">GAPDH</td><td class="NLM_def"><p class="first last">glyceraldehyde 3-phosphate dehydrogenase</p></td></tr><tr><td class="NLM_term">CCK-8</td><td class="NLM_def"><p class="first last">Cell Counting Kit-8</p></td></tr><tr><td class="NLM_term">ERK, AKT, DAB</td><td class="NLM_def"><p class="first last">3,3-diaminobenzidine tetrahydrochloride</p></td></tr><tr><td class="NLM_term">TKI</td><td class="NLM_def"><p class="first last">tyrosine kinase inhibitor</p></td></tr><tr><td class="NLM_term">ECD</td><td class="NLM_def"><p class="first last">extracellular domain</p></td></tr><tr><td class="NLM_term">PAMPA</td><td class="NLM_def"><p class="first last">parallel artificial membrane permeability assays</p></td></tr><tr><td class="NLM_term">CNS</td><td class="NLM_def"><p class="first last">central nervous system</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i31">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_74323" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_74323" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 41 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yarden, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sliwkowski, M. X.</span></span> <span> </span><span class="NLM_article-title">Untangling the ErbB signalling network</span>. <i>Nat. Rev. Mol. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">127</span>– <span class="NLM_lpage">137</span>, <span class="refDoi"> DOI: 10.1038/35052073</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=10.1038%2F35052073" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=11252954" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=1%3ACAS%3A528%3ADC%252BD3MXivVWnt7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2001&pages=127-137&author=Y.+Yardenauthor=M.+X.+Sliwkowski&title=Untangling+the+ErbB+signalling+network&doi=10.1038%2F35052073"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Untangling the ErbB signalling network</span></div><div class="casAuthors">Yarden, Yosef; Sliwkowski, Mark X.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Molecular Cell Biology</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">127-137</span>CODEN:
                <span class="NLM_cas:coden">NRMCBP</span>;
        ISSN:<span class="NLM_cas:issn">1471-0072</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review, with 112 refs.  When epidermal growth factor and its relatives bind the ErbB family of receptors, they trigger a rich network of signaling pathways, culminating in responses ranging from cell division to death, motility to adhesion.  The network is often dysregulated in cancer and lends credence to the mantra that mol. understanding yields clin. benefit: over 25,000 women with breast cancer have now been treated with Herceptin, a recombinant antibody designed to block the receptor ErbB2.  Likewise, small-mol. enzyme inhibitors and monoclonal antibodies to ErbB1 are in advanced phases of clin. testing.  What can this pathway teach us about translating basic science into clin. use.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGovhWL1HObvNrVg90H21EOLACvtfcHk0ljvZVSEfYsoMw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXivVWnt7k%253D&md5=1561a9666f77fce888be821399870cd6</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2F35052073&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F35052073%26sid%3Dliteratum%253Aachs%26aulast%3DYarden%26aufirst%3DY.%26aulast%3DSliwkowski%26aufirst%3DM.%2BX.%26atitle%3DUntangling%2520the%2520ErbB%2520signalling%2520network%26jtitle%3DNat.%2520Rev.%2520Mol.%2520Cell%2520Biol.%26date%3D2001%26volume%3D2%26spage%3D127%26epage%3D137%26doi%3D10.1038%2F35052073" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hynes, N. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MacDonald, G.</span></span> <span> </span><span class="NLM_article-title">ErbB receptors and signaling pathways in cancer</span>. <i>Curr. Opin. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">177</span>– <span class="NLM_lpage">184</span>, <span class="refDoi"> DOI: 10.1016/j.ceb.2008.12.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=10.1016%2Fj.ceb.2008.12.010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=19208461" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=1%3ACAS%3A528%3ADC%252BD1MXksVClurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2009&pages=177-184&author=N.+E.+Hynesauthor=G.+MacDonald&title=ErbB+receptors+and+signaling+pathways+in+cancer&doi=10.1016%2Fj.ceb.2008.12.010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">ErbB receptors and signaling pathways in cancer</span></div><div class="casAuthors">Hynes, Nancy E.; MacDonald, Gwen</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Cell Biology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">177-184</span>CODEN:
                <span class="NLM_cas:coden">COCBE3</span>;
        ISSN:<span class="NLM_cas:issn">0955-0674</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  The ErbB receptor tyrosine kinases play important roles in normal physiol. and in cancer.  Epidermal growth factor receptor (EGFR) and ErbB2 in particular are mutated in many epithelial tumors, and clin. studies suggest that they play roles in cancer development and progression.  These receptors were intensely studied, not only to understand the mechanisms underlying their oncogenic potential, but also to exploit them as therapeutic targets.  ErbB receptors activate a multiplicity of intracellular pathways via their ability to interact with numerous signal transducers.  Furthermore, there are now many ErbB-targeted inhibitors used in the clinic.  In this review we will conc. on breast tumors with ERBB2 gene amplification/receptor overexpression and non-small cell lung cancer (NSCLC) with activating EGFR mutations.  We will discuss data showing the important role that the PI3K/Akt pathway plays, not only in cancer development, but also in response to targeted therapies.  Finally, mechanisms contributing to resistance to ErbB-targeted therapeutics will also be discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr0lju0mdtYpbVg90H21EOLACvtfcHk0ljvZVSEfYsoMw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXksVClurc%253D&md5=217703dc2eff6a8157ce74a7c100a3fe</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1016%2Fj.ceb.2008.12.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ceb.2008.12.010%26sid%3Dliteratum%253Aachs%26aulast%3DHynes%26aufirst%3DN.%2BE.%26aulast%3DMacDonald%26aufirst%3DG.%26atitle%3DErbB%2520receptors%2520and%2520signaling%2520pathways%2520in%2520cancer%26jtitle%3DCurr.%2520Opin.%2520Cell%2520Biol.%26date%3D2009%26volume%3D21%26spage%3D177%26epage%3D184%26doi%3D10.1016%2Fj.ceb.2008.12.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yarden, Y.</span></span> <span> </span><span class="NLM_article-title">The EGFR family and its ligands in human cancer: signalling mechanisms and therapeutic opportunities</span>. <i>Eur. J. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>37</i></span>,  <span class="NLM_fpage">3</span>– <span class="NLM_lpage">8</span>, <span class="refDoi"> DOI: 10.1016/S0959-8049(01)00230-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=10.1016%2FS0959-8049%2801%2900230-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=11342194" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=1%3ACAS%3A528%3ADC%252BD3MXntlOku78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=2001&pages=3-8&author=Y.+Yarden&title=The+EGFR+family+and+its+ligands+in+human+cancer%3A+signalling+mechanisms+and+therapeutic+opportunities&doi=10.1016%2FS0959-8049%2801%2900230-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">The EGFR family and its ligands in human cancer signalling mechanisms and therapeutic opportunities</span></div><div class="casAuthors">Yarden, Y.</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Cancer</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">37</span>
        (<span class="NLM_cas:issue">Suppl. 4</span>),
    <span class="NLM_cas:pages">S3-S8</span>CODEN:
                <span class="NLM_cas:coden">EJCAEL</span>;
        ISSN:<span class="NLM_cas:issn">0959-8049</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ltd.</span>)
        </div><div class="casAbstract">A review.  Growth factors and their transmembrane receptor tyrosine kinases play important roles in cell proliferation, survival, migration and differentiation.  One group of growth factors, comprising epidermal growth factor (EGF)-like proteins and neuregulins, stimulates cells to divide by activating members of the EGF receptor (EGFR) family, which consists of the EGFR itself and the receptors known as HER2-4.  This highly conserved signaling module plays a fundamental role in the morphogenesis of a diverse spectrum of organisms, ranging from humans to nematodes, and has also been implicated in the development and growth of many types of human tumor cells.  In humans, more than 30 ligands and the EGFR family of four receptors lie at the head of a complex, multi-layered signal-transduction network.  Different activated receptor-ligand complexes vary in both the strength and type of cellular responses that they induce.  Anal. of the multiple processes that modulate EGFR signal transduction, such as receptor heterodimerization and endocytosis, has revealed new therapeutic opportunities and elucidated mechanisms contributing to the efficacy of existing anticancer treatments.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGruCkMkD5i497Vg90H21EOLACvtfcHk0liDmhAr_txn_w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXntlOku78%253D&md5=b512e9006ed63e9f01f2cdd525dc4d96</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1016%2FS0959-8049%2801%2900230-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0959-8049%252801%252900230-1%26sid%3Dliteratum%253Aachs%26aulast%3DYarden%26aufirst%3DY.%26atitle%3DThe%2520EGFR%2520family%2520and%2520its%2520ligands%2520in%2520human%2520cancer%253A%2520signalling%2520mechanisms%2520and%2520therapeutic%2520opportunities%26jtitle%3DEur.%2520J.%2520Cancer%26date%3D2001%26volume%3D37%26spage%3D3%26epage%3D8%26doi%3D10.1016%2FS0959-8049%2801%2900230-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Holbro, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hynes, N. E.</span></span> <span> </span><span class="NLM_article-title">ErbB Receptors: directing key signaling networks throughout life</span>. <i>Annu. Rev. Pharmacol. Toxicol.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>44</i></span>,  <span class="NLM_fpage">195</span>– <span class="NLM_lpage">217</span>, <span class="refDoi"> DOI: 10.1146/annurev.pharmtox.44.101802.121440</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=10.1146%2Fannurev.pharmtox.44.101802.121440" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=14744244" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhvVCkt7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2004&pages=195-217&author=T.+Holbroauthor=N.+E.+Hynes&title=ErbB+Receptors%3A+directing+key+signaling+networks+throughout+life&doi=10.1146%2Fannurev.pharmtox.44.101802.121440"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">ErbB receptors: Directing key signaling networks throughout life</span></div><div class="casAuthors">Holbro, Thomas; Hynes, Nancy E.</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Pharmacology and Toxicology</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">195-217</span>CODEN:
                <span class="NLM_cas:coden">ARPTDI</span>;
        ISSN:<span class="NLM_cas:issn">0362-1642</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews Inc.</span>)
        </div><div class="casAbstract">A review.  The EGF-related peptides bind the ErbB receptors, inducing the formation of different homo- and heterodimers.  Receptor dimerization promotes activation of the intrinsic kinase, leading to phosphorylation of specific tyrosines located in the ErbB's cytoplasmic region.  These phosphorylated residues serve as docking sites for a variety of signaling mols. whose recruitment stimulates intracellular signaling cascades, which ultimately control diverse genetic programs.  Particular ligand-receptor complexes have essential roles in embryonic development as well as in the adult.  Finally, ErbB receptors are being pursued as therapeutic targets because aberrant ErbB activity has been obsd. in many human cancers.  In this review, we discuss these data in more detail, illustrating the importance of tightly regulated ErbB signaling throughout life.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoGr3YQEvuFpLVg90H21EOLACvtfcHk0liDmhAr_txn_w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhvVCkt7c%253D&md5=cd88afae65520994f9a7b4916cc74576</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1146%2Fannurev.pharmtox.44.101802.121440&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev.pharmtox.44.101802.121440%26sid%3Dliteratum%253Aachs%26aulast%3DHolbro%26aufirst%3DT.%26aulast%3DHynes%26aufirst%3DN.%2BE.%26atitle%3DErbB%2520Receptors%253A%2520directing%2520key%2520signaling%2520networks%2520throughout%2520life%26jtitle%3DAnnu.%2520Rev.%2520Pharmacol.%2520Toxicol.%26date%3D2004%26volume%3D44%26spage%3D195%26epage%3D217%26doi%3D10.1146%2Fannurev.pharmtox.44.101802.121440" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rocha-Lima, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soares, H. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raez, L. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singal, R.</span></span> <span> </span><span class="NLM_article-title">EGFR targeting of solid tumors</span>. <i>Cancer Control</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">295</span>– <span class="NLM_lpage">304</span>, <span class="refDoi"> DOI: 10.1177/107327480701400313</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=10.1177%2F107327480701400313" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=17615536" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=1%3ACAS%3A280%3ADC%252BD2szpslWksA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2007&pages=295-304&author=C.+M.+Rocha-Limaauthor=H.+P.+Soaresauthor=L.+E.+Raezauthor=R.+Singal&title=EGFR+targeting+of+solid+tumors&doi=10.1177%2F107327480701400313"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">EGFR targeting of solid tumors</span></div><div class="casAuthors">Rocha-Lima Caio M; Soares Heloisa P; Raez Luis E; Singal Rakesh</div><div class="citationInfo"><span class="NLM_cas:title">Cancer control : journal of the Moffitt Cancer Center</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">295-304</span>
        ISSN:<span class="NLM_cas:issn">1073-2748</span>.
    </div><div class="casAbstract">BACKGROUND:  Recent clinical trials suggest that epidermal growth factor receptor (EGFR)-targeted agents could benefit many patients with cancer.  METHODS:  We review the current status of several EGFR-targeted therapies in cancer patients and address the efficacy of theses drugs as monotherapy or in combination with other drugs and/or treatments.  RESULTS:  Cetuximab is the most widely studied anti-EGFR monoclonal antibody.  Other monoclonal antibody agents under investigation are panitumumab, matuzumab, MDX-447, nimutozumab, and mAb806.  Extensive research has also evaluated the efficacy of EGFR tyrosine kinase inhibitors such as erlotinib, gefitinib, EKB-569, lapatinib (GW572016), PKI-166, and canertinib (CI-1033).  All of these agents have been studied for the treatment of colorectal, lung, breast, pancreatic, renal, head and neck, gynecologic, and prostate cancer.  Currently, cetuximab and panitumumab are FDA approved for the treatment of metastatic colorectal cancer.  Additionally, cetuximab is approved for head and neck cancer.  Erlotinib is FDA approved for advanced/metastatic lung cancer.  Erlotinib in combination with gemcitabine is approved for advanced/metastatic pancreatic cancer treatment.  CONCLUSIONS:  EGFR-targeted agents have already shown utility in different scenarios.  Researchers are continuously investigating additional cancer types and combined treatment modalities that could also benefit from the use of EGFR-targeted agents.  Careful patient selection through the identification of specific biologic markers, such as gene expression, genomic polymorphism, and posttranslational modifications of EGFR downstream effectors, most likely will contribute to the successful use of these agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQTb3BRcWgtjBWfpMXQLRstfW6udTcc2ealuf5Cfj0qa7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2szpslWksA%253D%253D&md5=d22e4054f58010acebdabccc58635364</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1177%2F107327480701400313&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F107327480701400313%26sid%3Dliteratum%253Aachs%26aulast%3DRocha-Lima%26aufirst%3DC.%2BM.%26aulast%3DSoares%26aufirst%3DH.%2BP.%26aulast%3DRaez%26aufirst%3DL.%2BE.%26aulast%3DSingal%26aufirst%3DR.%26atitle%3DEGFR%2520targeting%2520of%2520solid%2520tumors%26jtitle%3DCancer%2520Control%26date%3D2007%26volume%3D14%26spage%3D295%26epage%3D304%26doi%3D10.1177%2F107327480701400313" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nicholas, M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lukas, R. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jafri, N. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Faoro, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salgia, R.</span></span> <span> </span><span class="NLM_article-title">Epidermal growth factor receptor–mediated signal transduction in the development and therapy of gliomas</span>. <i>Clin. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">7261</span>– <span class="NLM_lpage">7270</span>, <span class="refDoi"> DOI: 10.1158/1078-0432.CCR-06-0874</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=10.1158%2F1078-0432.CCR-06-0874" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=17189397" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtlCrtr7O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2006&pages=7261-7270&author=M.+K.+Nicholasauthor=R.+V.+Lukasauthor=N.+F.+Jafriauthor=L.+Faoroauthor=R.+Salgia&title=Epidermal+growth+factor+receptor%E2%80%93mediated+signal+transduction+in+the+development+and+therapy+of+gliomas&doi=10.1158%2F1078-0432.CCR-06-0874"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Epidermal Growth Factor Receptor-Mediated Signal Transduction in the Development and Therapy of Gliomas</span></div><div class="casAuthors">Nicholas, M. Kelly; Lukas, Rimas V.; Jafri, Nazia F.; Faoro, Leonardo; Salgia, Ravi</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">7261-7270</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">A review.  The epidermal growth factor receptor (EGFR) and its ligands figure prominently in the biol. of gliomas, the most common tumors of the central nervous system (CNS).  Although their histol. classification seems to be straightforward, these tumors constitute a heterogeneous class of related neoplasms.  They are assocd. with a variety of mol. abnormalities affecting signal transduction, transcription factors, apoptosis, angiogenesis, and the extracellular matrix.  Under normal conditions, these same interacting factors drive CNS growth and development.  We are now recognizing the diverse mol. genetic heterogeneity that underlies tumors classified histol. into three distinct grades.  This recognition is leading to new therapeutic strategies targeted directly at specific mol. subtypes.  In this article, we will review the role of EGFR and related mol. pathways in the genesis of the normal CNS and their relationship to glial tumorigenesis.  We will discuss barriers to effective treatment as they relate to anat. specialization of the CNS.  We will also consider the ways in which specific EGFR alterations common to glioma reflect outcomes following treatment with targeted therapies, all with an eye towards applying this understanding to improved patient outcomes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo9JmBdtem1mbVg90H21EOLACvtfcHk0lj3GHj2_aMibA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtlCrtr7O&md5=9649bd7ee74f26a8f8bfd00fb60ca2c8</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1158%2F1078-0432.CCR-06-0874&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1078-0432.CCR-06-0874%26sid%3Dliteratum%253Aachs%26aulast%3DNicholas%26aufirst%3DM.%2BK.%26aulast%3DLukas%26aufirst%3DR.%2BV.%26aulast%3DJafri%26aufirst%3DN.%2BF.%26aulast%3DFaoro%26aufirst%3DL.%26aulast%3DSalgia%26aufirst%3DR.%26atitle%3DEpidermal%2520growth%2520factor%2520receptor%25E2%2580%2593mediated%2520signal%2520transduction%2520in%2520the%2520development%2520and%2520therapy%2520of%2520gliomas%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2006%26volume%3D12%26spage%3D7261%26epage%3D7270%26doi%3D10.1158%2F1078-0432.CCR-06-0874" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rowinsky, E. K.</span></span> <span> </span><span class="NLM_article-title">The erbB Family: Targets for therapeutic development against cancer and therapeutic strategies using monoclonal antibodies and tyrosine kinase inhibitors</span>. <i>Annu. Rev. Med.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">433</span>– <span class="NLM_lpage">457</span>, <span class="refDoi"> DOI: 10.1146/annurev.med.55.091902.104433</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=10.1146%2Fannurev.med.55.091902.104433" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=14746530" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=1%3ACAS%3A528%3ADC%252BD2cXitVWru74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2004&pages=433-457&author=E.+K.+Rowinsky&title=The+erbB+Family%3A+Targets+for+therapeutic+development+against+cancer+and+therapeutic+strategies+using+monoclonal+antibodies+and+tyrosine+kinase+inhibitors&doi=10.1146%2Fannurev.med.55.091902.104433"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">The erbB family: targets for therapeutic development against cancer and therapeutic strategies using monoclonal antibodies and tyrosine kinase inhibitors</span></div><div class="casAuthors">Rowinsky, Eric K.</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Medicine</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">433-457</span>CODEN:
                <span class="NLM_cas:coden">ARMCAH</span>;
        ISSN:<span class="NLM_cas:issn">0066-4219</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews Inc.</span>)
        </div><div class="casAbstract">A review.  The overexpression and aberrant function of members of the erbB family of receptors, particularly erbB1 (also known as epidermal growth factor receptor), and its ligands in many human cancers have provided a rationale for targeting this signaling network with novel approaches.  ErbB1 is a selective target for inhibiting cancers because its activation often confers a proliferative advantage.  Activation of the erbB1 tyrosine kinase provides signals that drive dysregulated proliferation, invasion, metastasis, angiogenesis, and cell survival, and its inhibition has potential in both the treatment and prevention of these malignancies.  Based on the structure and function of erbB1, two therapeutic strategies have been developed.  The first uses human monoclonal antibodies (MAbs) generated against the receptor's ligand-binding extracellular domain.  These MAbs block binding of receptor-activating ligands, and, in some cases, can induce receptor endocytosis and downregulation.  The second uses small mols. that compete with ATP for binding to the receptor's kinase pocket, thereby blocking receptor activation and the transduction of postreceptor signals.  Early clin. studies suggest that both approaches are well tolerated and can induce clin. activity in many common malignancies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGran3L8bVTvFrVg90H21EOLACvtfcHk0lj3GHj2_aMibA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXitVWru74%253D&md5=d44bcf33198e16162432c87a016a192c</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1146%2Fannurev.med.55.091902.104433&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev.med.55.091902.104433%26sid%3Dliteratum%253Aachs%26aulast%3DRowinsky%26aufirst%3DE.%2BK.%26atitle%3DThe%2520erbB%2520Family%253A%2520Targets%2520for%2520therapeutic%2520development%2520against%2520cancer%2520and%2520therapeutic%2520strategies%2520using%2520monoclonal%2520antibodies%2520and%2520tyrosine%2520kinase%2520inhibitors%26jtitle%3DAnnu.%2520Rev.%2520Med.%26date%3D2004%26volume%3D55%26spage%3D433%26epage%3D457%26doi%3D10.1146%2Fannurev.med.55.091902.104433" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mosesson, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yarden, Y.</span></span> <span> </span><span class="NLM_article-title">Oncogenic growth factor receptors: implications for signal transduction therapy</span>. <i>Semin. Cancer Biol.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">262</span>– <span class="NLM_lpage">270</span>, <span class="refDoi"> DOI: 10.1016/j.semcancer.2004.04.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=10.1016%2Fj.semcancer.2004.04.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=15219619" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=1%3ACAS%3A528%3ADC%252BD2cXlt1agu78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2004&pages=262-270&author=Y.+Mosessonauthor=Y.+Yarden&title=Oncogenic+growth+factor+receptors%3A+implications+for+signal+transduction+therapy&doi=10.1016%2Fj.semcancer.2004.04.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Oncogenic growth factor receptors: implications for signal transduction therapy</span></div><div class="casAuthors">Mosesson, Yaron; Yarden, Yosef</div><div class="citationInfo"><span class="NLM_cas:title">Seminars in Cancer Biology</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">262-270</span>CODEN:
                <span class="NLM_cas:coden">SECBE7</span>;
        ISSN:<span class="NLM_cas:issn">1044-579X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science B.V.</span>)
        </div><div class="casAbstract">A review.  Growth factors of the EGF family and their resp. ErbB/HER receptor tyrosine kinases underlie many landmarks of tumor cells, including excessive growth, invasive behavior and attraction of blood vessels.  Enhanced expression of ErbB proteins, existence of permanently active receptor mutants and occurrence of autocrine loops are frequently obsd. in human cancer, and in some cases they assoc. with poor disease outcome.  The four ErbB proteins and their 11 ligands act within a layered signaling network coordinated by ErbB-2/HER2, the most oncogenic family member.  Drugs that intercept signals emanating from ErbB-2 and ErbB-1 are already in routine clin. application.  Here the authors review three major strategies to develop new ErbB-targeted therapies.  These are monoclonal anti-receptor antibodies, specific tyrosine kinase inhibitors and antagonists of heat shock protein 90.  The underlying mechanisms are critically examd., with an emphasis on potential drug combinations, which hold promise for enhanced clin. efficacy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp-JnXVxWxJ9LVg90H21EOLACvtfcHk0ljfQlnUvxll3Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXlt1agu78%253D&md5=dff2f3a6fdf347b0ffa2c979871abbda</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1016%2Fj.semcancer.2004.04.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.semcancer.2004.04.005%26sid%3Dliteratum%253Aachs%26aulast%3DMosesson%26aufirst%3DY.%26aulast%3DYarden%26aufirst%3DY.%26atitle%3DOncogenic%2520growth%2520factor%2520receptors%253A%2520implications%2520for%2520signal%2520transduction%2520therapy%26jtitle%3DSemin.%2520Cancer%2520Biol.%26date%3D2004%26volume%3D14%26spage%3D262%26epage%3D270%26doi%3D10.1016%2Fj.semcancer.2004.04.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Paez, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jänne, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tracy, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greulich, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gabriel, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herman, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaye, F. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindeman, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boggon, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Naoki, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sasaki, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fujii, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eck, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sellers, W. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meyerson, M.</span></span> <span> </span><span class="NLM_article-title">EGFR mutations in lung cancer: correlation with clinical response to gefitinib therapy</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>304</i></span>,  <span class="NLM_fpage">1497</span>– <span class="NLM_lpage">1500</span>, <span class="refDoi"> DOI: 10.1126/science.1099314</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=10.1126%2Fscience.1099314" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=15118125" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=1%3ACAS%3A528%3ADC%252BD2cXksVGmsbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=304&publication_year=2004&pages=1497-1500&author=J.+G.+Paezauthor=P.+A.+J%C3%A4nneauthor=J.+C.+Leeauthor=S.+Tracyauthor=H.+Greulichauthor=S.+Gabrielauthor=P.+Hermanauthor=F.+J.+Kayeauthor=N.+Lindemanauthor=T.+J.+Boggonauthor=K.+Naokiauthor=H.+Sasakiauthor=Y.+Fujiiauthor=M.+J.+Eckauthor=W.+R.+Sellersauthor=B.+E.+Johnsonauthor=M.+Meyerson&title=EGFR+mutations+in+lung+cancer%3A+correlation+with+clinical+response+to+gefitinib+therapy&doi=10.1126%2Fscience.1099314"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">EGFR Mutations in Lung Cancer: Correlation with Clinical Response to Gefitinib Therapy</span></div><div class="casAuthors">Paez, J. Guillermo; Jaenne, Pasi A.; Lee, Jeffrey C.; Tracy, Sean; Greulich, Heidi; Gabriel, Stacey; Herman, Paula; Kaye, Frederic J.; Lindeman, Neal; Boggon, Titus J.; Naoki, Katsuhiko; Sasaki, Hidefumi; Fujii, Yoshitaka; Eck, Michael J.; Sellers, William R.; Johnson, Bruce E.; Meyerson, Matthew</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">304</span>
        (<span class="NLM_cas:issue">5676</span>),
    <span class="NLM_cas:pages">1497-1500</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Receptor tyrosine kinase genes were sequenced in non-small cell lung cancer (NSCLC) and matched normal tissue.  Somatic mutations of the epidermal growth factor receptor gene EGFR were found in 15 of 58 unselected tumors from Japan and 1 of 61 from the United States.  Treatment with the EGFR kinase inhibitor gefitinib (Iressa) causes tumor regression in some patients with NSCLC, more frequently in Japan.  EGFR mutations were found in addnl. lung cancer samples from U.S. patients who responded to gefitinib therapy and in a lung adenocarcinoma cell line that was hypersensitive to growth inhibition by gefitinib, but not in gefitinib-insensitive tumors or cell lines.  These results suggest that EGFR mutations may predict sensitivity to gefitinib.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoGxLJH5f3Kb7Vg90H21EOLACvtfcHk0ljfQlnUvxll3Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXksVGmsbs%253D&md5=6df1b11bf9b3c284fbe000e34c61ebac</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1126%2Fscience.1099314&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1099314%26sid%3Dliteratum%253Aachs%26aulast%3DPaez%26aufirst%3DJ.%2BG.%26aulast%3DJ%25C3%25A4nne%26aufirst%3DP.%2BA.%26aulast%3DLee%26aufirst%3DJ.%2BC.%26aulast%3DTracy%26aufirst%3DS.%26aulast%3DGreulich%26aufirst%3DH.%26aulast%3DGabriel%26aufirst%3DS.%26aulast%3DHerman%26aufirst%3DP.%26aulast%3DKaye%26aufirst%3DF.%2BJ.%26aulast%3DLindeman%26aufirst%3DN.%26aulast%3DBoggon%26aufirst%3DT.%2BJ.%26aulast%3DNaoki%26aufirst%3DK.%26aulast%3DSasaki%26aufirst%3DH.%26aulast%3DFujii%26aufirst%3DY.%26aulast%3DEck%26aufirst%3DM.%2BJ.%26aulast%3DSellers%26aufirst%3DW.%2BR.%26aulast%3DJohnson%26aufirst%3DB.%2BE.%26aulast%3DMeyerson%26aufirst%3DM.%26atitle%3DEGFR%2520mutations%2520in%2520lung%2520cancer%253A%2520correlation%2520with%2520clinical%2520response%2520to%2520gefitinib%2520therapy%26jtitle%3DScience%26date%3D2004%26volume%3D304%26spage%3D1497%26epage%3D1500%26doi%3D10.1126%2Fscience.1099314" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sharma, S. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bell, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Settleman, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haber, D. A.</span></span> <span> </span><span class="NLM_article-title">Epidermal growth factor receptor mutations in lung cancer</span>. <i>Nat. Rev. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">169</span>, <span class="refDoi"> DOI: 10.1038/nrc2088</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=10.1038%2Fnrc2088" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=17318210" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=1%3ACAS%3A528%3ADC%252BD2sXitVGrsbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2007&pages=169&author=S.+V.+Sharmaauthor=D.+W.+Bellauthor=J.+Settlemanauthor=D.+A.+Haber&title=Epidermal+growth+factor+receptor+mutations+in+lung+cancer&doi=10.1038%2Fnrc2088"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Epidermal growth factor receptor mutations in lung cancer</span></div><div class="casAuthors">Sharma, Sreenath V.; Bell, Daphne W.; Settleman, Jeffrey; Haber, Daniel A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">169-181</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The development and clin. application of inhibitors that target the epidermal growth factor receptor (EGFR) provide important insights for new lung cancer therapies, as well as for the broader field of targeted cancer therapies.  We review the results of genetic, biochem. and clin. studies focused on somatic mutations of EGFR that are assocd. with the phenomenon of oncogene addiction, describing 'oncogenic shock' as a mechanistic explanation for the apoptosis that follows the acute treatment of susceptible cells with kinase inhibitors.  Understanding the genetic heterogeneity of epithelial tumors and devising strategies to circumvent their rapid acquisition of resistance to targeted kinase inhibitors are essential to the successful use of targeted therapies in common epithelial cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrMKw7Y4FUFM7Vg90H21EOLACvtfcHk0lhjejqWZvgGDw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXitVGrsbo%253D&md5=46219ea3ae7b9f0977e4a4b7243fd875</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1038%2Fnrc2088&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc2088%26sid%3Dliteratum%253Aachs%26aulast%3DSharma%26aufirst%3DS.%2BV.%26aulast%3DBell%26aufirst%3DD.%2BW.%26aulast%3DSettleman%26aufirst%3DJ.%26aulast%3DHaber%26aufirst%3DD.%2BA.%26atitle%3DEpidermal%2520growth%2520factor%2520receptor%2520mutations%2520in%2520lung%2520cancer%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2007%26volume%3D7%26spage%3D169%26doi%3D10.1038%2Fnrc2088" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Köhler, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schuler, M.</span></span> <span> </span><span class="NLM_article-title">Afatinib, Erlotinib and Gefitinib in the first-line therapy of egfr mutation-positive lung adenocarcinoma: A Review</span>. <i>Oncol. Res. Treat.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>36</i></span>,  <span class="NLM_fpage">510</span>– <span class="NLM_lpage">518</span>, <span class="refDoi"> DOI: 10.1159/000354627</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=10.1159%2F000354627" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2013&pages=510-518&author=J.+K%C3%B6hlerauthor=M.+Schuler&title=Afatinib%2C+Erlotinib+and+Gefitinib+in+the+first-line+therapy+of+egfr+mutation-positive+lung+adenocarcinoma%3A+A+Review&doi=10.1159%2F000354627"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1159%2F000354627&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1159%252F000354627%26sid%3Dliteratum%253Aachs%26aulast%3DK%25C3%25B6hler%26aufirst%3DJ.%26aulast%3DSchuler%26aufirst%3DM.%26atitle%3DAfatinib%252C%2520Erlotinib%2520and%2520Gefitinib%2520in%2520the%2520first-line%2520therapy%2520of%2520egfr%2520mutation-positive%2520lung%2520adenocarcinoma%253A%2520A%2520Review%26jtitle%3DOncol.%2520Res.%2520Treat.%26date%3D2013%26volume%3D36%26spage%3D510%26epage%3D518%26doi%3D10.1159%2F000354627" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmitz, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeffrey, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wiltzius, J. J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kussie, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferguson, K. M.</span></span> <span> </span><span class="NLM_article-title">Structural basis for inhibition of the epidermal growth factor receptor by cetuximab</span>. <i>Cancer Cell</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">301</span>– <span class="NLM_lpage">311</span>, <span class="refDoi"> DOI: 10.1016/j.ccr.2005.03.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=10.1016%2Fj.ccr.2005.03.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=15837620" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=1%3ACAS%3A528%3ADC%252BD2MXjvFSkurk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2005&pages=301-311&author=S.+Liauthor=K.+R.+Schmitzauthor=P.+D.+Jeffreyauthor=J.+J.+W.+Wiltziusauthor=P.+Kussieauthor=K.+M.+Ferguson&title=Structural+basis+for+inhibition+of+the+epidermal+growth+factor+receptor+by+cetuximab&doi=10.1016%2Fj.ccr.2005.03.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Structural basis for inhibition of the epidermal growth factor receptor by cetuximab</span></div><div class="casAuthors">Li, Shiqing; Schmitz, Karl R.; Jeffrey, Philip D.; Wiltzius, Jed J. W.; Kussie, Paul; Ferguson, Kathryn M.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">301-311</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Recent structural studies of epidermal growth factor receptor (EGFR) family extracellular regions have identified an unexpected mechanism for ligand-induced receptor dimerization that has important implications for activation and inhibition of these receptors.  Here we describe the 2.8 Å resoln. x-ray crystal structure of the antigen binding (Fab) fragment from cetuximab (Erbitux), an inhibitory anti-EGFR antibody, in complex with the sol. extracellular region of EGFR (sEGFR).  The sEGFR is in the characteristic "autoinhibited" or "tethered" inactive configuration.  Cetuximab interacts exclusively with domain III of sEGFR, partially occluding the ligand binding region on this domain and sterically preventing the receptor from adopting the extended conformation required for dimerization.  We suggest that both these effects contribute to potent inhibition of EGFR activation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrfRvCB5ETMNLVg90H21EOLACvtfcHk0lhjejqWZvgGDw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXjvFSkurk%253D&md5=7d7b01d331206edd65f0d2c5c69785dd</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1016%2Fj.ccr.2005.03.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccr.2005.03.003%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DS.%26aulast%3DSchmitz%26aufirst%3DK.%2BR.%26aulast%3DJeffrey%26aufirst%3DP.%2BD.%26aulast%3DWiltzius%26aufirst%3DJ.%2BJ.%2BW.%26aulast%3DKussie%26aufirst%3DP.%26aulast%3DFerguson%26aufirst%3DK.%2BM.%26atitle%3DStructural%2520basis%2520for%2520inhibition%2520of%2520the%2520epidermal%2520growth%2520factor%2520receptor%2520by%2520cetuximab%26jtitle%3DCancer%2520Cell%26date%3D2005%26volume%3D7%26spage%3D301%26epage%3D311%26doi%3D10.1016%2Fj.ccr.2005.03.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schmiedel, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blaukat, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knöchel, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferguson, K. M.</span></span> <span> </span><span class="NLM_article-title">Matuzumab binding to egfr prevents the conformational rearrangement required for dimerization</span>. <i>Cancer Cell</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">365</span>– <span class="NLM_lpage">373</span>, <span class="refDoi"> DOI: 10.1016/j.ccr.2008.02.019</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=10.1016%2Fj.ccr.2008.02.019" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=18394559" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=1%3ACAS%3A528%3ADC%252BD1cXkslSjt74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2008&pages=365-373&author=J.+Schmiedelauthor=A.+Blaukatauthor=S.+Liauthor=T.+Kn%C3%B6chelauthor=K.+M.+Ferguson&title=Matuzumab+binding+to+egfr+prevents+the+conformational+rearrangement+required+for+dimerization&doi=10.1016%2Fj.ccr.2008.02.019"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Matuzumab binding to EGFR prevents the conformational rearrangement required for dimerization</span></div><div class="casAuthors">Schmiedel, Judith; Blaukat, Andree; Li, Shiqing; Knochel, Thorsten; Ferguson, Kathryn M.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">365-373</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">An increasing no. of therapeutic antibodies targeting tumors that express the epidermal growth factor receptor (EGFR) are in clin. use or late stages of clin. development.  Here the authors investigate the mol. basis for inhibition of EGFR activation by the therapeutic antibody matuzumab (EMD72000).  They describe the x-ray crystal structure of the Fab fragment of matuzumab (Fab72000) in complex with isolated domain III from the extracellular region of EGFR.  Fab72000 interacts with an epitope on EGFR that is distinct from the ligand-binding region on domain III and from the cetuximab/Erbitux epitope.  Matuzumab blocks ligand-induced receptor activation indirectly by sterically preventing the domain rearrangement and local conformational changes that must occur for high-affinity ligand binding and receptor dimerization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr07Gswzbba6LVg90H21EOLACvtfcHk0lhjejqWZvgGDw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXkslSjt74%253D&md5=97386aedd55b986dde6c0d87abe786e3</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1016%2Fj.ccr.2008.02.019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccr.2008.02.019%26sid%3Dliteratum%253Aachs%26aulast%3DSchmiedel%26aufirst%3DJ.%26aulast%3DBlaukat%26aufirst%3DA.%26aulast%3DLi%26aufirst%3DS.%26aulast%3DKn%25C3%25B6chel%26aufirst%3DT.%26aulast%3DFerguson%26aufirst%3DK.%2BM.%26atitle%3DMatuzumab%2520binding%2520to%2520egfr%2520prevents%2520the%2520conformational%2520rearrangement%2520required%2520for%2520dimerization%26jtitle%3DCancer%2520Cell%26date%3D2008%26volume%3D13%26spage%3D365%26epage%3D373%26doi%3D10.1016%2Fj.ccr.2008.02.019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gu, J.</span></span> <span> </span><span class="NLM_article-title">Identification and characterization of a novel peptide ligand of epidermal growth factor receptor for targeted delivery of therapeutics</span>. <i>FASEB J.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">1978</span>– <span class="NLM_lpage">1985</span>, <span class="refDoi"> DOI: 10.1096/fj.05-4058com</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=10.1096%2Ffj.05-4058com" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=16319141" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtlSju7zK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2005&pages=1978-1985&author=Z.+Liauthor=R.+Zhaoauthor=X.+Wuauthor=Y.+Sunauthor=M.+Yaoauthor=J.+Liauthor=Y.+Xuauthor=J.+Gu&title=Identification+and+characterization+of+a+novel+peptide+ligand+of+epidermal+growth+factor+receptor+for+targeted+delivery+of+therapeutics&doi=10.1096%2Ffj.05-4058com"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Identification and characterization of a novel peptide ligand of epidermal growth factor receptor for targeted delivery of therapeutics</span></div><div class="casAuthors">Li, Zonghai; Zhao, Ruijiao; Wu, Xianghua; Sun, Ye; Yao, Ming; Li, Jinjun; Xu, Yuhong; Gu, Jianren</div><div class="citationInfo"><span class="NLM_cas:title">FASEB Journal</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">1978-1985, 10.1096/fj.05-4058com</span>CODEN:
                <span class="NLM_cas:coden">FAJOEC</span>;
        ISSN:<span class="NLM_cas:issn">0892-6638</span>.
    
            (<span class="NLM_cas:orgname">Federation of American Societies for Experimental Biology</span>)
        </div><div class="casAbstract">Epidermal growth factor receptor (ErbB1, EGFR) is overexpressed in a variety of human cancer cells.  It has been considered as a rational target for drug delivery.  To identify novel ligands with specific binding capabilities to EGFR, we screened a phage display peptide library and found an enriched phage clone encoding the amino acid sequence YHWGYTPQNVI (designated as GE11).  Competitive binding assay and Scatchard anal. revealed that GE11 peptide bound specifically and efficiently to EGFR with a dissocn. const. of ∼22 nM, but with much lower mitogenic activity than with EGF.  We showed that the peptides were internalized preferentially into EGFR highly expressing cells, and they accumulated in EGFR overexpressing tumor xenografts after i.v. delivery in vivo.  In gene delivery studies, GE11-conjugated polyethylenimine (PEI) vectors were less mitogenic, but still quite efficient at transfecting genes into EGFR highly expressing cells and tumor xenografts.  Taken together, GE11 is a potentially safe and efficient targeting moiety for selective drug delivery systems mediated through EGFR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo7wWbZJf-gVrVg90H21EOLACvtfcHk0lg_qi1ojc3l0Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtlSju7zK&md5=bb5769496ea0cb1514861ff932b06e30</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1096%2Ffj.05-4058com&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1096%252Ffj.05-4058com%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DZ.%26aulast%3DZhao%26aufirst%3DR.%26aulast%3DWu%26aufirst%3DX.%26aulast%3DSun%26aufirst%3DY.%26aulast%3DYao%26aufirst%3DM.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DXu%26aufirst%3DY.%26aulast%3DGu%26aufirst%3DJ.%26atitle%3DIdentification%2520and%2520characterization%2520of%2520a%2520novel%2520peptide%2520ligand%2520of%2520epidermal%2520growth%2520factor%2520receptor%2520for%2520targeted%2520delivery%2520of%2520therapeutics%26jtitle%3DFASEB%2520J.%26date%3D2005%26volume%3D19%26spage%3D1978%26epage%3D1985%26doi%3D10.1096%2Ffj.05-4058com" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Song, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, L. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Y.</span></span> <span> </span><span class="NLM_article-title">Novel peptide ligand directs liposomes toward EGF-R high-expressing cancer cells in vitro and in vivo</span>. <i>FASEB J.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">1396</span>– <span class="NLM_lpage">1404</span>, <span class="refDoi"> DOI: 10.1096/fj.08-117002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=10.1096%2Ffj.08-117002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=19124558" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=1%3ACAS%3A528%3ADC%252BD1MXlsFSmsrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2009&pages=1396-1404&author=S.+Songauthor=D.+Liuauthor=J.+Pengauthor=H.+Dengauthor=Y.+Guoauthor=L.+X.+Xuauthor=A.+D.+Millerauthor=Y.+Xu&title=Novel+peptide+ligand+directs+liposomes+toward+EGF-R+high-expressing+cancer+cells+in+vitro+and+in+vivo&doi=10.1096%2Ffj.08-117002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Novel peptide ligand directs liposomes toward EGF-R high-expressing cancer cells in vitro and in vivo</span></div><div class="casAuthors">Song, Shuxian; Liu, Dan; Peng, Jinliang; Deng, Hongwei; Guo, Yan; Xu, Lisa X.; Miller, Andrew D.; Xu, Yuhong</div><div class="citationInfo"><span class="NLM_cas:title">FASEB Journal</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1396-1404, 10.1096/fj.08-117002</span>CODEN:
                <span class="NLM_cas:coden">FAJOEC</span>;
        ISSN:<span class="NLM_cas:issn">0892-6638</span>.
    
            (<span class="NLM_cas:orgname">Federation of American Societies for Experimental Biology</span>)
        </div><div class="casAbstract">Epidermal growth factor receptor (EGF-R) is an important target in anticancer therapy.  Here we report how a novel EGF-R peptide ligand (D4: Leu-Ala-Arg-Leu-Leu-Thr) is identified using a computer-aided design approach from a virtual peptide library of putative EGF-R binding peptides by screening against the EGF-R X-ray crystal structure in silico and in vitro.  The selected peptide is conjugated with a polyethylene glycol (PEG) lipid, and the lipid moiety of the peptide-PEG-lipid conjugate is inserted into liposome membranes by a postmodification process.  D4 peptide-conjugated liposomes are found to bind to and enter cells by endocytosis specifically and efficiently in vitro in a process apparently mediated by EGF-R high-expressing cancer cells (H1299).  In vivo, the D4 peptide-conjugated liposomes are found to accumulate in EGF-R-expressing xenograft tumor tissue up to 80 h after i.v. delivery, in marked contrast to controls.  These results demonstrate how structure-based peptide design can be an efficient approach to identify highly novel binding ligands against important receptors.  These data could have important consequences for the development of peptide-directed drug delivery systems with engineered specificities and prolonged times of action.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrVWhU7TdAMrLVg90H21EOLACvtfcHk0lg_qi1ojc3l0Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXlsFSmsrs%253D&md5=e5e299248cb417b0fdf16cde5238332b</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1096%2Ffj.08-117002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1096%252Ffj.08-117002%26sid%3Dliteratum%253Aachs%26aulast%3DSong%26aufirst%3DS.%26aulast%3DLiu%26aufirst%3DD.%26aulast%3DPeng%26aufirst%3DJ.%26aulast%3DDeng%26aufirst%3DH.%26aulast%3DGuo%26aufirst%3DY.%26aulast%3DXu%26aufirst%3DL.%2BX.%26aulast%3DMiller%26aufirst%3DA.%2BD.%26aulast%3DXu%26aufirst%3DY.%26atitle%3DNovel%2520peptide%2520ligand%2520directs%2520liposomes%2520toward%2520EGF-R%2520high-expressing%2520cancer%2520cells%2520in%2520vitro%2520and%2520in%2520vivo%26jtitle%3DFASEB%2520J.%26date%3D2009%26volume%3D23%26spage%3D1396%26epage%3D1404%26doi%3D10.1096%2Ffj.08-117002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Teng, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, J.</span></span> <span> </span><span class="NLM_article-title">γ-AApeptides as a new class of peptidomimetics</span>. <i>Chem. - Eur. J.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">5458</span>– <span class="NLM_lpage">5466</span>, <span class="refDoi"> DOI: 10.1002/chem.201504936</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=10.1002%2Fchem.201504936" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=26945679" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=1%3ACAS%3A528%3ADC%252BC28XjslGitbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2016&pages=5458-5466&author=P.+Tengauthor=Y.+Shiauthor=P.+Sangauthor=J.+Cai&title=%CE%B3-AApeptides+as+a+new+class+of+peptidomimetics&doi=10.1002%2Fchem.201504936"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">γ-AApeptides as a New Class of Peptidomimetics</span></div><div class="casAuthors">Teng, Peng; Shi, Yan; Sang, Peng; Cai, Jianfeng</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry - A European Journal</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">5458-5466</span>CODEN:
                <span class="NLM_cas:coden">CEUJED</span>;
        ISSN:<span class="NLM_cas:issn">0947-6539</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A review.  Sequence-specific peptidomimetics are mols. that mimic the structure and function of peptides and proteins.  With new backbones and mol. frameworks, peptidomimetics are of considerable interest in addressing challenges encountered in chem. biol. and biomedical sciences.  Based on the γ-PNA backbone, a new class of peptidomimetics - "γ-AApeptides" were recently developed.  Both linear and cyclic γ-AApeptides can be synthesized with high efficiency.  Compared with α-peptides, γ-AApeptides are resistant to enzymic degrdn., and amendable to diversification with a variety of chem. groups.  Moreover, they could mimic primary and secondary structure, as well as the function of peptides, and show promise in biol. applications, such as the development of new agents combating bacteria, cancer, and Alzheimer's disease.  A few research outcomes of γ-AApeptides are highlighted in this Concept article, and a future perspective is also proposed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrj0LfZU9KRa7Vg90H21EOLACvtfcHk0lg_qi1ojc3l0Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XjslGitbk%253D&md5=6f8385337d870a546e1e6b56fd8ef3f1</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1002%2Fchem.201504936&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fchem.201504936%26sid%3Dliteratum%253Aachs%26aulast%3DTeng%26aufirst%3DP.%26aulast%3DShi%26aufirst%3DY.%26aulast%3DSang%26aufirst%3DP.%26aulast%3DCai%26aufirst%3DJ.%26atitle%3D%25CE%25B3-AApeptides%2520as%2520a%2520new%2520class%2520of%2520peptidomimetics%26jtitle%3DChem.%2520-%2520Eur.%2520J.%26date%3D2016%26volume%3D22%26spage%3D5458%26epage%3D5466%26doi%3D10.1002%2Fchem.201504936" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Teng, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">She, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, J.</span></span> <span> </span><span class="NLM_article-title">γ-AApeptides: design, structure, and applications</span>. <i>Acc. Chem. Res.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>49</i></span>,  <span class="NLM_fpage">428</span>– <span class="NLM_lpage">441</span>, <span class="refDoi"> DOI: 10.1021/acs.accounts.5b00492</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.accounts.5b00492" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=1%3ACAS%3A528%3ADC%252BC28XivVWkt7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2016&pages=428-441&author=Y.+Shiauthor=P.+Tengauthor=P.+Sangauthor=F.+Sheauthor=L.+Weiauthor=J.+Cai&title=%CE%B3-AApeptides%3A+design%2C+structure%2C+and+applications&doi=10.1021%2Facs.accounts.5b00492"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">γ-AApeptides: Design, structure, and applications</span></div><div class="casAuthors">Shi, Yan; Teng, Peng; Sang, Peng; She, Fengyu; Wei, Lulu; Cai, Jianfeng</div><div class="citationInfo"><span class="NLM_cas:title">Accounts of Chemical Research</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">428-441</span>CODEN:
                <span class="NLM_cas:coden">ACHRE4</span>;
        ISSN:<span class="NLM_cas:issn">0001-4842</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The development of sequence-specific peptidomimetics has led to a variety of fascinating discoveries in chem. biol.  Many peptidomimetics can mimic primary, secondary, and even tertiary structure of peptides and proteins, and because of their unnatural backbones, they also possess significantly enhanced resistance to enzymic hydrolysis, improved bioavailability, and chemodiversity.  It is known that peptide nucleic acids (PNAs) are peptidic sequences developed for the mimicry of nucleic acids; however, their unique backbone as the mol. scaffold of peptidomimetics to mimic structure and function of bioactive peptides has not been investigated systematically.  As such, we recently developed a new class of peptidomimetics, "γ-AApeptides", based on the chiral γ-PNA backbone.  They are termed γ-AApeptides because they are the oligomers of γ-substituted-N-acylated-N-aminoethyl amino acids.  Similar to other classes of peptidomimetics, γ-AApeptides are also resistant to proteolytic degrdn. and possess the potential to enhance chemodiversity.  Moreover, in our scientific journey on the exploration of this class of peptidomimetics, we have discovered some intriguing structures and functions of γ-AApeptides.  In this account, we summarize the current development and application of γ-AApeptides with biol. potential.  Briefly, both linear and cyclic (either through head-to-tail or head-to-side-chain cyclization) γ-AApeptides with diverse functional groups can be synthesized easily on the solid phase using the synthetic protocol we developed. γ-AApeptides could mimic the primary structure of peptides, as they project the same no. of side chains as peptides of the same lengths.  For instance, they could mimic the Tat peptide to permeate cell membranes and bind to HIV RNA with high specificity and affinity.  Certain γ-AApeptides show similar activity to the RGD peptide and target integrin specifically on the cell surface.  γ-AApeptides with function akin to fMLF peptides are also identified.  More importantly, we found that γ-AApeptides can fold into discrete secondary structures, such as helical and β-turn-like structures.  Therefore, they could be rationally designed for a range of biol. applications.  For instance, γ-AApeptides can mimic host-defense peptides and display potent and broad-spectrum activity toward a panel of drug-resistant bacterial pathogens.  Meanwhile, because of their stability against proteolysis and their chemodiversity, γ-AApeptides are also amenable for combinatorial screening.  We demonstrate that, through combinatorial selection, certain γ-AApeptides are identified to inhibit Aβ40 peptide aggregation, suggesting their potential use as a mol. probe to intervene in Alzheimer's disease.  In addn., a few γ-AApeptides identified from the γ-AApeptide library have been shown to bind to the DNA-binding domain of STAT3 and antagonize STAT3/DNA interactions.  Our studies suggest that, with further studies and exploration on both structures and functions, γ-AApeptides may emerge to be a new class of peptidomimetics that play an important role in chem. biol. and biomedical sciences.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqrk8PLbKcOxrVg90H21EOLACvtfcHk0lgZvT7NQsxaZg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XivVWkt7w%253D&md5=414ab3568ccfdcebb5863d876218804c</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1021%2Facs.accounts.5b00492&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.accounts.5b00492%26sid%3Dliteratum%253Aachs%26aulast%3DShi%26aufirst%3DY.%26aulast%3DTeng%26aufirst%3DP.%26aulast%3DSang%26aufirst%3DP.%26aulast%3DShe%26aufirst%3DF.%26aulast%3DWei%26aufirst%3DL.%26aulast%3DCai%26aufirst%3DJ.%26atitle%3D%25CE%25B3-AApeptides%253A%2520design%252C%2520structure%252C%2520and%2520applications%26jtitle%3DAcc.%2520Chem.%2520Res.%26date%3D2016%26volume%3D49%26spage%3D428%26epage%3D441%26doi%3D10.1021%2Facs.accounts.5b00492" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abdulkadir, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ji, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, J.</span></span> <span> </span><span class="NLM_article-title">Inhibition of β-catenin/B cell lymphoma 9 protein–protein interaction using α-helix–mimicking sulfono-γ-AApeptide inhibitors</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>116</i></span>,  <span class="NLM_fpage">10757</span>– <span class="NLM_lpage">10762</span>, <span class="refDoi"> DOI: 10.1073/pnas.1819663116</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=10.1073%2Fpnas.1819663116" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=31088961" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhtVChsr%252FE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=116&publication_year=2019&pages=10757-10762&author=P.+Sangauthor=M.+Zhangauthor=Y.+Shiauthor=C.+Liauthor=S.+Abdulkadirauthor=Q.+Liauthor=H.+Jiauthor=J.+Cai&title=Inhibition+of+%CE%B2-catenin%2FB+cell+lymphoma+9+protein%E2%80%93protein+interaction+using+%CE%B1-helix%E2%80%93mimicking+sulfono-%CE%B3-AApeptide+inhibitors&doi=10.1073%2Fpnas.1819663116"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of β-catenin/B cell lymphoma 9 protein-protein interaction using α-helix-mimicking sulfono-γ-AApeptide inhibitors</span></div><div class="casAuthors">Sang, Peng; Zhang, Min; Shi, Yan; Li, Chunpu; Abdulkadir, Sami; Li, Qi; Ji, Haitao; Cai, Jianfeng</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">116</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">10757-10762</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The rational design of α-helix-mimicking peptidomimetics provides a streamlined approach to discover potent inhibitors for protein-protein interactions (PPIs).  However, designing cell-penetrating long peptidomimetic scaffolds equipped with various functional groups necessary for interacting with large protein-binding interfaces remains challenging.  This is particularly true for targeting β-catenin/BCL9 PPIs.  Here the authors designed a series of unprecedented helical sulfono-γ-AApeptides that mimic the binding mode of the α-helical HD2 domain of B Cell Lymphoma 9 (BCL9).  The studies show that sulfono-γ-AApeptides can structurally and functionally mimic the α-helical domain of BCL9 and selectively disrupt β-catenin/BCL9 PPIs with even higher potency.  More intriguingly, these sulfono-γ-AApeptides can enter cancer cells, bind with β-catenin and disrupt β-catenin/BCL9 PPIs, and exhibit excellent cellular activity, which is much more potent than the BCL9 peptide.  Furthermore, the enzymic stability studies demonstrate the remarkable stability of the helical sulfono-γ-AApeptides, with no degrdn. in the presence of pronase for 24 h, augmenting their biol. potential.  This work represents not only an example of helical sulfono-γ-AApeptides that mimic α-helix and disrupt protein-protein interactions, but also an excellent example of potent, selective, and cell-permeable unnatural foldameric peptidomimetics that disrupt the β-catenin/BCL9 PPI.  The design of helical sulfono-γ-AApeptides may lead to a new strategy to modulate a myriad of protein-protein interactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo_pKlgpb-_U7Vg90H21EOLACvtfcHk0lgZvT7NQsxaZg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhtVChsr%252FE&md5=145172aa8475bfe4a20b97d6b88d12dd</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1819663116&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1819663116%26sid%3Dliteratum%253Aachs%26aulast%3DSang%26aufirst%3DP.%26aulast%3DZhang%26aufirst%3DM.%26aulast%3DShi%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DC.%26aulast%3DAbdulkadir%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DQ.%26aulast%3DJi%26aufirst%3DH.%26aulast%3DCai%26aufirst%3DJ.%26atitle%3DInhibition%2520of%2520%25CE%25B2-catenin%252FB%2520cell%2520lymphoma%25209%2520protein%25E2%2580%2593protein%2520interaction%2520using%2520%25CE%25B1-helix%25E2%2580%2593mimicking%2520sulfono-%25CE%25B3-AApeptide%2520inhibitors%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2019%26volume%3D116%26spage%3D10757%26epage%3D10762%26doi%3D10.1073%2Fpnas.1819663116" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Teng, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cerrato, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">She, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Odom, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ming, L.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van der Vaart, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wojtas, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, J.</span></span> <span> </span><span class="NLM_article-title">Right-handed helical foldamers consisting of de novo d-aapeptides</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>139</i></span>,  <span class="NLM_fpage">7363</span>– <span class="NLM_lpage">7369</span>, <span class="refDoi"> DOI: 10.1021/jacs.7b03007</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.7b03007" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=1%3ACAS%3A528%3ADC%252BC2sXnt1Chu7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=139&publication_year=2017&pages=7363-7369&author=P.+Tengauthor=N.+Maauthor=D.+C.+Cerratoauthor=F.+Sheauthor=T.+Odomauthor=X.+Wangauthor=L.-J.+Mingauthor=A.+van+der+Vaartauthor=L.+Wojtasauthor=H.+Xuauthor=J.+Cai&title=Right-handed+helical+foldamers+consisting+of+de+novo+d-aapeptides&doi=10.1021%2Fjacs.7b03007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Right-handed helical foldamers consisting of de novo D-AA-peptides</span></div><div class="casAuthors">Teng, Peng; Ma, Ning; Cerrato, Darrell Cole; She, Fengyu; Odom, Timothy; Wang, Xiang; Ming, Li-June; van der Vaart, Arjan; Wojtas, Lukasz; Xu, Hai; Cai, Jianfeng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">139</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">7363-7369</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">New types of foldamer scaffolds are formidably challenging to design and synthesize, yet highly desirable as structural mimics of peptides/proteins with a wide repertoire of functions.  In particular, the development of peptidomimetic helical foldamers holds promise for new biomaterials, catalysts, and drug mols.  Unnatural L-sulfono-γ-AApeptides were recently developed and shown to have potential applications in both biomedical and material sciences.  However, D-sulfono-γ-AApeptides, the enantiomers of L-sulfono-γ-AApeptides, have never been studied due to the lack of high-resoln. three-dimensional structures to guide structure-based design.  Herein, we report the first synthesis and X-ray crystal structures of a series of 2:1 L-amino acid/D-sulfono-γ-AApeptide hybrid foldamers, and elucidate their folded conformation at the at. level.  Single-crystal X-ray crystallog. indicates that this class of oligomers folds into well-defined right-handed helixes with unique helical parameters.  The helical structures were consistent with data obtained from soln. 2D NMR, CD studies, and mol. dynamics simulations.  Our findings are expected to inspire the structure-based design of this type of unique folding biopolymers for biomaterials and biomedical applications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqaGg_OJWsqYbVg90H21EOLACvtfcHk0lgZvT7NQsxaZg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXnt1Chu7w%253D&md5=fef3f80500953f027687001f3e67f14b</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1021%2Fjacs.7b03007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.7b03007%26sid%3Dliteratum%253Aachs%26aulast%3DTeng%26aufirst%3DP.%26aulast%3DMa%26aufirst%3DN.%26aulast%3DCerrato%26aufirst%3DD.%2BC.%26aulast%3DShe%26aufirst%3DF.%26aulast%3DOdom%26aufirst%3DT.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DMing%26aufirst%3DL.-J.%26aulast%3Dvan%2Bder%2BVaart%26aufirst%3DA.%26aulast%3DWojtas%26aufirst%3DL.%26aulast%3DXu%26aufirst%3DH.%26aulast%3DCai%26aufirst%3DJ.%26atitle%3DRight-handed%2520helical%2520foldamers%2520consisting%2520of%2520de%2520novo%2520d-aapeptides%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2017%26volume%3D139%26spage%3D7363%26epage%3D7369%26doi%3D10.1021%2Fjacs.7b03007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Teng, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Niu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">She, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verma, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wojtas, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van der Vaart, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, J.</span></span> <span> </span><span class="NLM_article-title">Hydrogen-bonding-driven 3d supramolecular assembly of peptidomimetic zipper</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>140</i></span>,  <span class="NLM_fpage">5661</span>– <span class="NLM_lpage">5665</span>, <span class="refDoi"> DOI: 10.1021/jacs.7b11997</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.7b11997" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=1%3ACAS%3A528%3ADC%252BC1cXmsVahsLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=140&publication_year=2018&pages=5661-5665&author=P.+Tengauthor=Z.+Niuauthor=F.+Sheauthor=M.+Zhouauthor=P.+Sangauthor=G.+M.+Grayauthor=G.+Vermaauthor=L.+Wojtasauthor=A.+van+der+Vaartauthor=S.+Maauthor=J.+Cai&title=Hydrogen-bonding-driven+3d+supramolecular+assembly+of+peptidomimetic+zipper&doi=10.1021%2Fjacs.7b11997"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Hydrogen-Bonding-Driven 3D Supramolecular Assembly of Peptidomimetic Zipper</span></div><div class="casAuthors">Teng, Peng; Niu, Zheng; She, Fengyu; Zhou, Mi; Sang, Peng; Gray, Geoffrey M.; Verma, Gaurav; Wojtas, Lukasz; van der Vaart, Arjan; Ma, Shengqian; Cai, Jianfeng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">140</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">5661-5665</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Hydrogen-bonding-driven three-dimensional (3D) assembly of a peptidomimetic zipper has been established for the first time by using an α/AApeptide zipper that assembles into a de novo lattice arrangement through two layers of hydrogen-bonded linker-directed interactions.  Via a covalently bridged 1D 413-helix, drastic enhancement in stability has been achieved in the formed 3D cryst. supramol. architecture as evidenced by gas-sorption studies.  As the first example of an unnatural peptidic zipper, the dimensional augmentation of the zipper differs from metal-coordinated strategies, and may have general implications for the prepn. of peptidic functional materials for a variety of future applications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq6NOPVSQVnN7Vg90H21EOLACvtfcHk0liKYY20onei9g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXmsVahsLY%253D&md5=75ec49a44f96107f9a2472698475157d</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Fjacs.7b11997&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.7b11997%26sid%3Dliteratum%253Aachs%26aulast%3DTeng%26aufirst%3DP.%26aulast%3DNiu%26aufirst%3DZ.%26aulast%3DShe%26aufirst%3DF.%26aulast%3DZhou%26aufirst%3DM.%26aulast%3DSang%26aufirst%3DP.%26aulast%3DGray%26aufirst%3DG.%2BM.%26aulast%3DVerma%26aufirst%3DG.%26aulast%3DWojtas%26aufirst%3DL.%26aulast%3Dvan%2Bder%2BVaart%26aufirst%3DA.%26aulast%3DMa%26aufirst%3DS.%26aulast%3DCai%26aufirst%3DJ.%26atitle%3DHydrogen-bonding-driven%25203d%2520supramolecular%2520assembly%2520of%2520peptidomimetic%2520zipper%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2018%26volume%3D140%26spage%3D5661%26epage%3D5665%26doi%3D10.1021%2Fjacs.7b11997" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">She, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Teng, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peguero-Tejada, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Odom, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gjonaj, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wojtas, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van der Vaart, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, J.</span></span> <span> </span><span class="NLM_article-title">De novo left-handed synthetic peptidomimetic foldamers</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">9916</span>– <span class="NLM_lpage">9920</span>, <span class="refDoi"> DOI: 10.1002/anie.201805184</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=10.1002%2Fanie.201805184" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=1%3ACAS%3A528%3ADC%252BC1cXht1yjt7bL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2018&pages=9916-9920&author=F.+Sheauthor=P.+Tengauthor=A.+Peguero-Tejadaauthor=M.+Wangauthor=N.+Maauthor=T.+Odomauthor=M.+Zhouauthor=E.+Gjonajauthor=L.+Wojtasauthor=A.+van+der+Vaartauthor=J.+Cai&title=De+novo+left-handed+synthetic+peptidomimetic+foldamers&doi=10.1002%2Fanie.201805184"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">De novo left-handed synthetic peptidomimetic foldamers</span></div><div class="casAuthors">She, Fengyu; Teng, Peng; Peguero-Tejada, Alfredo; Wang, Minghui; Ma, Ning; Odom, Timothy; Zhou, Mi; Gjonaj, Erald; Wojtas, Lukasz; van der Vaart, Arjan; Cai, Jianfeng</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">31</span>),
    <span class="NLM_cas:pages">9916-9920</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">The development of peptidomimetic helical foldamers with a wide repertoire of functions is of significant interest.  Herein, we report the X-ray crystal structures of a series of homogeneous L-sulfono-γ-AA foldamers and elucidate their folding conformation at the at. level.  Single-crystal X-ray crystallog. revealed that this class of oligomers fold into unprecedented dragon-boat-shaped and unexpected left-handed helixes, which are stabilized by the 14-hydrogen-bonding pattern present in all sequences.  These L-sulfono-γ-AA-peptides have a helical pitch of 5.1 Å and exactly four side chains per turn, and the side chains lie perfectly on top of each other along the helical axis.  2D NMR spectroscopy, computational simulations, and CD studies support the folding conformation in soln.  Our results provide a structural basis at the at. level for the design of novel biomimetics with a precise arrangement of functional groups in three dimensions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGru2vbXzMss1rVg90H21EOLACvtfcHk0liKYY20onei9g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXht1yjt7bL&md5=c45cc7e77dfa2887f5fec63d8e3dbe7e</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1002%2Fanie.201805184&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201805184%26sid%3Dliteratum%253Aachs%26aulast%3DShe%26aufirst%3DF.%26aulast%3DTeng%26aufirst%3DP.%26aulast%3DPeguero-Tejada%26aufirst%3DA.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DMa%26aufirst%3DN.%26aulast%3DOdom%26aufirst%3DT.%26aulast%3DZhou%26aufirst%3DM.%26aulast%3DGjonaj%26aufirst%3DE.%26aulast%3DWojtas%26aufirst%3DL.%26aulast%3Dvan%2Bder%2BVaart%26aufirst%3DA.%26aulast%3DCai%26aufirst%3DJ.%26atitle%3DDe%2520novo%2520left-handed%2520synthetic%2520peptidomimetic%2520foldamers%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2018%26volume%3D57%26spage%3D9916%26epage%3D9920%26doi%3D10.1002%2Fanie.201805184" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Teng, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wojtas, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van der Vaart, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, J.</span></span> <span> </span><span class="NLM_article-title">Orthogonal halogen-bonding-driven 3d supramolecular assembly of right-handed synthetic helical peptides</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">7778</span>– <span class="NLM_lpage">7782</span>, <span class="refDoi"> DOI: 10.1002/anie.201903259</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=10.1002%2Fanie.201903259" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=1%3ACAS%3A528%3ADC%252BC1MXos1Sjsbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2019&pages=7778-7782&author=P.+Tengauthor=G.+M.+Grayauthor=M.+Zhengauthor=S.+Singhauthor=X.+Liauthor=L.+Wojtasauthor=A.+van+der+Vaartauthor=J.+Cai&title=Orthogonal+halogen-bonding-driven+3d+supramolecular+assembly+of+right-handed+synthetic+helical+peptides&doi=10.1002%2Fanie.201903259"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Orthogonal Halogen-Bonding-Driven 3D Supramolecular Assembly of Right-Handed Synthetic Helical Peptides</span></div><div class="casAuthors">Teng, Peng; Gray, Geoffrey M.; Zheng, Mengmeng; Singh, Sylvia; Li, Xiaopeng; Wojtas, Lukasz; van der Vaart, Arjan; Cai, Jianfeng</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">7778-7782</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Peptide-mediated self-assembly is a prevalent method for creating highly ordered supramol. architectures.  Herein, we report the first example of orthogonal C-X···X-C/C-X···π halogen bonding and hydrogen bonding driven cryst. architectures based on synthetic helical peptides bearing hybrids of L-sulfono-γ-AApeptides and natural amino acids.  The combination of halogen bonding, intra-/intermol. hydrogen bonding, and intermol. hydrophobic interactions enabled novel 3D supramol. assembly.  The orthogonal halogen bonding in the supramol. architecture exerts a novel mechanism for the self-assembly of synthetic peptide foldamers and gives new insights into mol. recognition, supramol. design, and rational design of biomimetic structures.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGphs8_rPqx-zrVg90H21EOLACvtfcHk0liKYY20onei9g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXos1Sjsbc%253D&md5=c61d38ec3f3a4e3d79d716a1c88a7ec3</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1002%2Fanie.201903259&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201903259%26sid%3Dliteratum%253Aachs%26aulast%3DTeng%26aufirst%3DP.%26aulast%3DGray%26aufirst%3DG.%2BM.%26aulast%3DZheng%26aufirst%3DM.%26aulast%3DSingh%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DWojtas%26aufirst%3DL.%26aulast%3Dvan%2Bder%2BVaart%26aufirst%3DA.%26aulast%3DCai%26aufirst%3DJ.%26atitle%3DOrthogonal%2520halogen-bonding-driven%25203d%2520supramolecular%2520assembly%2520of%2520right-handed%2520synthetic%2520helical%2520peptides%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2019%26volume%3D58%26spage%3D7778%26epage%3D7782%26doi%3D10.1002%2Fanie.201903259" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Teng, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qiao, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sebti, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, J.</span></span> <span> </span><span class="NLM_article-title">Identification of novel inhibitors that disrupt STAT3–DNA interaction from a γ-aapeptide oboc combinatorial library</span>. <i>Chem. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">8739</span>– <span class="NLM_lpage">8742</span>, <span class="refDoi"> DOI: 10.1039/C4CC03909B</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=10.1039%2FC4CC03909B" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=24964402" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtVGnsrjM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2014&pages=8739-8742&author=P.+Tengauthor=X.+Zhangauthor=H.+Wuauthor=Q.+Qiaoauthor=S.+M.+Sebtiauthor=J.+Cai&title=Identification+of+novel+inhibitors+that+disrupt+STAT3%E2%80%93DNA+interaction+from+a+%CE%B3-aapeptide+oboc+combinatorial+library&doi=10.1039%2FC4CC03909B"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of novel inhibitors that disrupt STAT3-DNA interaction from a γ-AApeptide OBOC combinatorial library</span></div><div class="casAuthors">Teng, Peng; Zhang, Xiaolei; Wu, Haifan; Qiao, Qiao; Sebti, Said M.; Cai, Jianfeng</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Communications (Cambridge, United Kingdom)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">63</span>),
    <span class="NLM_cas:pages">8739-8742</span>CODEN:
                <span class="NLM_cas:coden">CHCOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-7345</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">From a γ-AApeptide-based one-bead-one-compd. (OBOC) combinatorial library, the authors identified γ-AApeptides that can selectively inhibit STAT3-DNA interaction and suppress the expression levels of STAT3 target genes in intact cells.  The authors' results demonstrate that in addn. to the SH2 domain, the DNA binding domain of STAT3 is targetable for the development of a new generation of anti-cancer therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqVN0NpRV6tXrVg90H21EOLACvtfcHk0liEJNZdg6gOHA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtVGnsrjM&md5=41c994aa9a985107fc1402b58d38223a</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1039%2FC4CC03909B&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC4CC03909B%26sid%3Dliteratum%253Aachs%26aulast%3DTeng%26aufirst%3DP.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DWu%26aufirst%3DH.%26aulast%3DQiao%26aufirst%3DQ.%26aulast%3DSebti%26aufirst%3DS.%2BM.%26aulast%3DCai%26aufirst%3DJ.%26atitle%3DIdentification%2520of%2520novel%2520inhibitors%2520that%2520disrupt%2520STAT3%25E2%2580%2593DNA%2520interaction%2520from%2520a%2520%25CE%25B3-aapeptide%2520oboc%2520combinatorial%2520library%26jtitle%3DChem.%2520Commun.%26date%3D2014%26volume%3D50%26spage%3D8739%26epage%3D8742%26doi%3D10.1039%2FC4CC03909B" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Challa, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">She, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nimmagadda, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Teng, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Odom, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van der
Vaart, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, J.</span></span> <span> </span><span class="NLM_article-title">One-bead–two-compound thioether bridged macrocyclic γ-aapeptide screening library against EphA2</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">9290</span>– <span class="NLM_lpage">9298</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b01280</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b01280" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhslKgtLnO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=9290-9298&author=Y.+Shiauthor=S.+Challaauthor=P.+Sangauthor=F.+Sheauthor=C.+Liauthor=G.+M.+Grayauthor=A.+Nimmagaddaauthor=P.+Tengauthor=T.+Odomauthor=Y.+Wangauthor=A.+van+der%0AVaartauthor=Q.+Liauthor=J.+Cai&title=One-bead%E2%80%93two-compound+thioether+bridged+macrocyclic+%CE%B3-aapeptide+screening+library+against+EphA2&doi=10.1021%2Facs.jmedchem.7b01280"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">One-bead-two-compound thioether bridged macrocyclic γ-AApeptide screening library against EphA2</span></div><div class="casAuthors">Shi, Yan; Challa, Sridevi; Sang, Peng; She, Fengyu; Li, Chunpu; Gray, Geoffrey M.; Nimmagadda, Alekhya; Teng, Peng; Odom, Timothy; Wang, Yan; van der Vaart, Arjan; Li, Qi; Cai, Jianfeng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">9290-9298</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Identification of mol. ligands that recognize peptides or proteins is significant but poses a fundamental challenge in chem. biol. and biomedical sciences.  Development of cyclic peptidomimetic library is scarce, and thus discovery of cyclic peptidomimetic ligands for protein targets is rare.  Herein we report the unprecedented one-bead-two-compd. (OBTC) combinatorial library based on a novel class of the macrocyclic peptidomimetics γ-AApeptides.  In the library, we utilized the coding peptide tags synthesized with Dde-protected α-amino acids, which were orthogonal to solid phase synthesis of γ-AApeptides.  Employing the thioether linkage, the desired macrocyclic γ-AApeptides were found to be effective for ligand identification.  Screening the library against the receptor tyrosine kinase EphA2 led to the discovery of one lead compd. that tightly bound to EphA2 (Kd = 81 nM) and potently antagonized EphA2-mediated signaling.  This new approach of macrocyclic peptidomimetic library may lead to a novel platform for biomacromol. surface recognition and function modulation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrE8_skSAKNvbVg90H21EOLACvtfcHk0liEJNZdg6gOHA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhslKgtLnO&md5=0c5c86232aa6afd699fd6de5128a1ecf</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b01280&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b01280%26sid%3Dliteratum%253Aachs%26aulast%3DShi%26aufirst%3DY.%26aulast%3DChalla%26aufirst%3DS.%26aulast%3DSang%26aufirst%3DP.%26aulast%3DShe%26aufirst%3DF.%26aulast%3DLi%26aufirst%3DC.%26aulast%3DGray%26aufirst%3DG.%2BM.%26aulast%3DNimmagadda%26aufirst%3DA.%26aulast%3DTeng%26aufirst%3DP.%26aulast%3DOdom%26aufirst%3DT.%26aulast%3DWang%26aufirst%3DY.%26aulast%3Dvan%2Bder%2BVaart%26aufirst%3DA.%26aulast%3DLi%26aufirst%3DQ.%26aulast%3DCai%26aufirst%3DJ.%26atitle%3DOne-bead%25E2%2580%2593two-compound%2520thioether%2520bridged%2520macrocyclic%2520%25CE%25B3-aapeptide%2520screening%2520library%2520against%2520EphA2%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D9290%26epage%3D9298%26doi%3D10.1021%2Facs.jmedchem.7b01280" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yan, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhattarai, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, F.-S.</span></span> <span> </span><span class="NLM_article-title">Cyclic Peptidomimetics as Inhibitor for miR-155 Biogenesis</span>. <i>Mol. Pharmaceutics</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">914</span>– <span class="NLM_lpage">920</span>, <span class="refDoi"> DOI: 10.1021/acs.molpharmaceut.8b01247</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.molpharmaceut.8b01247" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhtlSmsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2019&pages=914-920&author=H.+Yanauthor=M.+Zhouauthor=U.+Bhattaraiauthor=Y.+Songauthor=M.+Zhengauthor=J.+Caiauthor=F.-S.+Liang&title=Cyclic+Peptidomimetics+as+Inhibitor+for+miR-155+Biogenesis&doi=10.1021%2Facs.molpharmaceut.8b01247"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Cyclic Peptidomimetics as Inhibitor for miR-155 Biogenesis</span></div><div class="casAuthors">Yan, Hao; Zhou, Mi; Bhattarai, Umesh; Song, Yabin; Zheng, Mengmeng; Cai, Jianfeng; Liang, Fu-Sen</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Pharmaceutics</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">914-920</span>CODEN:
                <span class="NLM_cas:coden">MPOHBP</span>;
        ISSN:<span class="NLM_cas:issn">1543-8384</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">MiR-155 plays key promoting roles in several cancers and emerges as an important anticancer therapeutic target.  However, the discovery of small mols. that target RNAs is challenging.  Peptidomimetics have been shown to be a rich source for discovering novel ligands to regulate cellular proteins.  However, the potential of using peptidomimetics for RNA targeting is relatively unexplored.  To this end, we designed and synthesized members of a novel 320 000 compd. macrocyclic peptidomimetic library.  An affinity-based screening protocol led to the identification of a pre-miR-155 binder that inhibits oncogenic miR-155 maturation in vitro and in cell and induces cancer cell apoptosis.  The results of this investigation demonstrate that macrocyclic peptidomimetics could serve as a new scaffold for RNA targeting.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqa1182LkJrALVg90H21EOLACvtfcHk0liEJNZdg6gOHA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhtlSmsg%253D%253D&md5=0ac2dec99b95d9efbbaef045bf9e9e4f</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Facs.molpharmaceut.8b01247&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.molpharmaceut.8b01247%26sid%3Dliteratum%253Aachs%26aulast%3DYan%26aufirst%3DH.%26aulast%3DZhou%26aufirst%3DM.%26aulast%3DBhattarai%26aufirst%3DU.%26aulast%3DSong%26aufirst%3DY.%26aulast%3DZheng%26aufirst%3DM.%26aulast%3DCai%26aufirst%3DJ.%26aulast%3DLiang%26aufirst%3DF.-S.%26atitle%3DCyclic%2520Peptidomimetics%2520as%2520Inhibitor%2520for%2520miR-155%2520Biogenesis%26jtitle%3DMol.%2520Pharmaceutics%26date%3D2019%26volume%3D16%26spage%3D914%26epage%3D920%26doi%3D10.1021%2Facs.molpharmaceut.8b01247" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parag, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lui, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murr, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, N. A.</span></span> <span> </span><span class="NLM_article-title">Stabilization of lncrna gas5 by a small molecule and its implications in diabetic adipocytes</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">319</span>– <span class="NLM_lpage">330</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2018.11.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=10.1016%2Fj.chembiol.2018.11.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=30661991" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhsVartb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2019&pages=319-330&author=Y.+Shiauthor=S.+Paragauthor=R.+Patelauthor=A.+Luiauthor=M.+Murrauthor=J.+Caiauthor=N.+A.+Patel&title=Stabilization+of+lncrna+gas5+by+a+small+molecule+and+its+implications+in+diabetic+adipocytes&doi=10.1016%2Fj.chembiol.2018.11.012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Stabilization of lncRNA GAS5 by a Small Molecule and Its Implications in Diabetic Adipocytes</span></div><div class="casAuthors">Shi, Yan; Parag, Sajan; Patel, Rekha; Lui, Ashley; Murr, Michel; Cai, Jianfeng; Patel, Niketa A.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">319-330.e6</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Long noncoding RNA (lncRNA) are regulatory RNAs >200 nt.  We previously showed that lncRNA GAS5 decreases significantly in serum of type 2 diabetes mellitus (T2DM) patients.  Hence, we sought to decipher the mol. mechanisms underlying the role of GAS5 in T2DM in adipose tissue.  Using CHIP-RIP, we demonstrate that GAS5 binds to promoter of insulin receptor to regulate its expression, and its depletion inhibits glucose uptake and insulin signaling.  Toward stabilizing GAS5 levels in T2DM, we incorporated a strategy to limit the degrdn. of GAS5 by blocking the interaction of GAS5 and UPF1 with a small mol. identified using OBTC screening strategy.  NP-C86 binds to GAS5 with high affinity, and increases GAS5 levels and glucose uptake in diabetic patient adipocytes.  As a broader impact, NP-C86 may be used as a mol. probe to investigate the intricacies of GAS5 in relevant biol. systems as it offers specificity, efficient cellular uptake and is non-cytotoxic.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpk-KPkGlzXi7Vg90H21EOLACvtfcHk0lgfP4zArcFyzQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhsVartb8%253D&md5=5f7d6288486103bfe129bdd1a7d9e05d</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2018.11.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2018.11.012%26sid%3Dliteratum%253Aachs%26aulast%3DShi%26aufirst%3DY.%26aulast%3DParag%26aufirst%3DS.%26aulast%3DPatel%26aufirst%3DR.%26aulast%3DLui%26aufirst%3DA.%26aulast%3DMurr%26aufirst%3DM.%26aulast%3DCai%26aufirst%3DJ.%26aulast%3DPatel%26aufirst%3DN.%2BA.%26atitle%3DStabilization%2520of%2520lncrna%2520gas5%2520by%2520a%2520small%2520molecule%2520and%2520its%2520implications%2520in%2520diabetic%2520adipocytes%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2019%26volume%3D26%26spage%3D319%26epage%3D330%26doi%3D10.1016%2Fj.chembiol.2018.11.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gureasko, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cole, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuriyan, J.</span></span> <span> </span><span class="NLM_article-title">An Allosteric Mechanism for activation of the kinase domain of epidermal growth factor receptor</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>125</i></span>,  <span class="NLM_fpage">1137</span>– <span class="NLM_lpage">1149</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2006.05.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=10.1016%2Fj.cell.2006.05.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=16777603" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=1%3ACAS%3A528%3ADC%252BD28XmsVOms7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=125&publication_year=2006&pages=1137-1149&author=X.+Zhangauthor=J.+Gureaskoauthor=K.+Shenauthor=P.+A.+Coleauthor=J.+Kuriyan&title=An+Allosteric+Mechanism+for+activation+of+the+kinase+domain+of+epidermal+growth+factor+receptor&doi=10.1016%2Fj.cell.2006.05.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">An allosteric mechanism for activation of the kinase domain of epidermal growth factor receptor</span></div><div class="casAuthors">Zhang, Xuewu; Gureasko, Jodi; Shen, Kui; Cole, Philip A.; Kuriyan, John</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">125</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1137-1149</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The mechanism by which the epidermal growth factor receptor (EGFR) is activated upon dimerization has eluded definition.  We find that the EGFR kinase domain can be activated by increasing its local concn. or by mutating a leucine (L834R) in the activation loop, the phosphorylation of which is not required for activation.  This suggests that the kinase domain is intrinsically auto-inhibited, and an intermol. interaction promotes its activation.  Using further mutational anal. and crystallog. we demonstrate that the auto-inhibited conformation of the EGFR kinase domain resembles that of Src and cyclin-dependent kinases (CDKs).  EGFR activation results from the formation of an asym. dimer in which the C-terminal lobe of one kinase domain plays a role analogous to that of cyclin in activated CDK/cyclin complexes.  The CDK/cyclin-like complex formed by two kinase domains thus explains the activation of EGFR-family receptors by homo- or heterodimerization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoQsRAzLF-wK7Vg90H21EOLACvtfcHk0lgfP4zArcFyzQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XmsVOms7g%253D&md5=f7b6795851425346f588cbd682125c93</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2006.05.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2006.05.013%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DGureasko%26aufirst%3DJ.%26aulast%3DShen%26aufirst%3DK.%26aulast%3DCole%26aufirst%3DP.%2BA.%26aulast%3DKuriyan%26aufirst%3DJ.%26atitle%3DAn%2520Allosteric%2520Mechanism%2520for%2520activation%2520of%2520the%2520kinase%2520domain%2520of%2520epidermal%2520growth%2520factor%2520receptor%26jtitle%3DCell%26date%3D2006%26volume%3D125%26spage%3D1137%26epage%3D1149%26doi%3D10.1016%2Fj.cell.2006.05.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ogiso, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ishitani, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nureki, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fukai, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamanaka, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, J.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saito, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sakamoto, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Inoue, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shirouzu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yokoyama, S.</span></span> <span> </span><span class="NLM_article-title">Crystal structure of the complex of human epidermal growth factor and receptor extracellular domains</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>110</i></span>,  <span class="NLM_fpage">775</span>– <span class="NLM_lpage">787</span>, <span class="refDoi"> DOI: 10.1016/S0092-8674(02)00963-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=10.1016%2FS0092-8674%2802%2900963-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=12297050" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=1%3ACAS%3A528%3ADC%252BD38XnsFKisL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=110&publication_year=2002&pages=775-787&author=H.+Ogisoauthor=R.+Ishitaniauthor=O.+Nurekiauthor=S.+Fukaiauthor=M.+Yamanakaauthor=J.-H.+Kimauthor=K.+Saitoauthor=A.+Sakamotoauthor=M.+Inoueauthor=M.+Shirouzuauthor=S.+Yokoyama&title=Crystal+structure+of+the+complex+of+human+epidermal+growth+factor+and+receptor+extracellular+domains&doi=10.1016%2FS0092-8674%2802%2900963-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal structure of the complex of human epidermal growth factor and receptor extracellular domains</span></div><div class="casAuthors">Ogiso, Hideo; Ishitani, Ryuichiro; Nureki, Osamu; Fukai, Shuya; Yamanaka, Mari; Kim, Jae-Hoon; Saito, Kazuki; Sakamoto, Ayako; Inoue, Mio; Shirouzu, Mikako; Yokoyama, Shigeyuki</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">110</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">775-787</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Epidermal growth factor (EGF) regulates cell proliferation and differentiation by binding to the EGF receptor (EGFR) extracellular region, comprising domains I-IV, with the resultant dimerization of the receptor tyrosine kinase.  In this study, the crystal structure of a 2:2 complex of human EGF and the EGFR extracellular region has been detd. at 3.3 Å resoln.  EGFR domains I-III are arranged in a C shape, and EGF is docked between domains I and III.  The 1:1 EGF•EGFR complex dimerizes through a direct receptor•receptor interaction, in which a protruding β-hairpin arm of each domain II holds the body of the other.  The unique "receptor-mediated dimerization" was verified by EGFR mutagenesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqILejwMfkBaLVg90H21EOLACvtfcHk0lgfP4zArcFyzQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XnsFKisL8%253D&md5=f909ae76aad06fb0a8704ee101e76310</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1016%2FS0092-8674%2802%2900963-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0092-8674%252802%252900963-7%26sid%3Dliteratum%253Aachs%26aulast%3DOgiso%26aufirst%3DH.%26aulast%3DIshitani%26aufirst%3DR.%26aulast%3DNureki%26aufirst%3DO.%26aulast%3DFukai%26aufirst%3DS.%26aulast%3DYamanaka%26aufirst%3DM.%26aulast%3DKim%26aufirst%3DJ.-H.%26aulast%3DSaito%26aufirst%3DK.%26aulast%3DSakamoto%26aufirst%3DA.%26aulast%3DInoue%26aufirst%3DM.%26aulast%3DShirouzu%26aufirst%3DM.%26aulast%3DYokoyama%26aufirst%3DS.%26atitle%3DCrystal%2520structure%2520of%2520the%2520complex%2520of%2520human%2520epidermal%2520growth%2520factor%2520and%2520receptor%2520extracellular%2520domains%26jtitle%3DCell%26date%3D2002%26volume%3D110%26spage%3D775%26epage%3D787%26doi%3D10.1016%2FS0092-8674%2802%2900963-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sanders, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wampole, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thakur, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wickstrom, E.</span></span> <span> </span><span class="NLM_article-title">Molecular determinants of epidermal growth factor binding: a molecular dynamics study</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>8</i></span>, <span class="NLM_elocation-id">e54136</span> <span class="refDoi"> DOI: 10.1371/journal.pone.0054136</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=10.1371%2Fjournal.pone.0054136" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=23382875" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=1%3ACAS%3A528%3ADC%252BC3sXit1ChsrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&author=J.+M.+Sandersauthor=M.+E.+Wampoleauthor=M.+L.+Thakurauthor=E.+Wickstrom&title=Molecular+determinants+of+epidermal+growth+factor+binding%3A+a+molecular+dynamics+study&doi=10.1371%2Fjournal.pone.0054136"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular determinants of epidermal growth factor binding: a molecular dynamics study</span></div><div class="casAuthors">Sanders, Jeffrey M.; Wampole, Matthew E.; Thakur, Mathew L.; Wickstrom, Eric</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">e54136</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">The epidermal growth factor receptor (EGFR) is a member of the receptor tyrosine kinase family that plays a role in multiple cellular processes.  Activation of EGFR requires binding of a ligand on the extracellular domain to promote conformational changes leading to dimerization and transphosphorylation of intracellular kinase domains.  Seven ligands are known to bind EGFR with affinities ranging from sub-nanomolar to near micromolar dissocn. consts.  In the case of EGFR, distinct conformational states assumed upon binding a ligand is thought to be a detg. factor in activation of a downstream signaling network.  Previous biochem. studies suggest the existence of both low affinity and high affinity EGFR ligands.  While these studies have identified functional effects of ligand binding, high-resoln. structural data are lacking.  To gain a better understanding of the mol. basis of EGFR binding affinities, we docked each EGFR ligand to the putative active state extracellular domain dimer and 25.0 ns mol. dynamics simulations were performed.  MM-PBSA/GBSA are efficient computational approaches to approx. free energies of protein-protein interactions and decomp. the free energy at the amino acid level.  We applied these methods to the last 6.0 ns of each ligand-receptor simulation.  MM-PBSA calcns. were able to successfully rank all seven of the EGFR ligands based on the two affinity classes: EGF>HB-EGF>TGF-α>BTC>EPR>EPG>AR.  Results from energy decompn. identified several interactions that are common among binding ligands.  These findings reveal that while several residues are conserved among the EGFR ligand family, no single set of residues dets. the affinity class.  Instead we found heterogeneous sets of interactions that were driven primarily by electrostatic and Van der Waals forces.  These results not only illustrate the complexity of EGFR dynamics but also pave the way for structure-based design of therapeutics targeting EGF ligands or the receptor itself.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqTdLUJ_FXUTbVg90H21EOLACvtfcHk0liq6CYh9S74BQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXit1ChsrY%253D&md5=f2630ac7526af1a73890557bea731b79</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0054136&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0054136%26sid%3Dliteratum%253Aachs%26aulast%3DSanders%26aufirst%3DJ.%2BM.%26aulast%3DWampole%26aufirst%3DM.%2BE.%26aulast%3DThakur%26aufirst%3DM.%2BL.%26aulast%3DWickstrom%26aufirst%3DE.%26atitle%3DMolecular%2520determinants%2520of%2520epidermal%2520growth%2520factor%2520binding%253A%2520a%2520molecular%2520dynamics%2520study%26jtitle%3DPLoS%2520One%26date%3D2013%26volume%3D8%26doi%3D10.1371%2Fjournal.pone.0054136" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jones, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Akita, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sliwkowski, M. X.</span></span> <span> </span><span class="NLM_article-title">Binding specificities and affinities of egf domains for ErbB receptors</span>. <i>FEBS Lett.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>447</i></span>,  <span class="NLM_fpage">227</span>– <span class="NLM_lpage">231</span>, <span class="refDoi"> DOI: 10.1016/S0014-5793(99)00283-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=10.1016%2FS0014-5793%2899%2900283-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=10214951" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=1%3ACAS%3A528%3ADyaK1MXis1ars7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=447&publication_year=1999&pages=227-231&author=J.+T.+Jonesauthor=R.+W.+Akitaauthor=M.+X.+Sliwkowski&title=Binding+specificities+and+affinities+of+egf+domains+for+ErbB+receptors&doi=10.1016%2FS0014-5793%2899%2900283-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Binding specificities and affinities of egf domains for ErbB receptors</span></div><div class="casAuthors">Jones, Jennifer T.; Akita, Robert W.; Sliwkowski, Mark X.</div><div class="citationInfo"><span class="NLM_cas:title">FEBS Letters</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">447</span>
        (<span class="NLM_cas:issue">2,3</span>),
    <span class="NLM_cas:pages">227-231</span>CODEN:
                <span class="NLM_cas:coden">FEBLAL</span>;
        ISSN:<span class="NLM_cas:issn">0014-5793</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science B.V.</span>)
        </div><div class="casAbstract">ErbB receptor activation is a complex process and is dependent upon the type and no. of receptors expressed on a given cell.  Previous studies with defined combinations of ErbB receptors expressed in mammalian cells have helped elucidate specific biol. responses for many of the recognized gene products that serve as ligands for these receptors.  However, no study has examd. the binding of these ligands in a defined exptl. system.  To address this issue, the relative binding affinities of the egf domains of eleven ErbB ligands were measured on six ErbB receptor combinations using a sol. receptor-ligand binding format.  The ErbB2/4 heterodimer was shown to bind all ligands tested with moderate to very high affinity.  In contrast, ErbB3 showed much more restrictive ligand binding specificity and measurable binding was obsd. only with heregulin, neuregulin2β, epiregulin and the synthetic heregulin/egf chimera, biregulin.  These studies also revealed that ErbB2 preferentially enhances ligand binding to ErbB3 or ErbB4 and to a lesser degree to ErbB1.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGodjee8uZQb57Vg90H21EOLACvtfcHk0liq6CYh9S74BQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXis1ars7w%253D&md5=247b1b73bd015dcda612eebd820f6c4d</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1016%2FS0014-5793%2899%2900283-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0014-5793%252899%252900283-5%26sid%3Dliteratum%253Aachs%26aulast%3DJones%26aufirst%3DJ.%2BT.%26aulast%3DAkita%26aufirst%3DR.%2BW.%26aulast%3DSliwkowski%26aufirst%3DM.%2BX.%26atitle%3DBinding%2520specificities%2520and%2520affinities%2520of%2520egf%2520domains%2520for%2520ErbB%2520receptors%26jtitle%3DFEBS%2520Lett.%26date%3D1999%26volume%3D447%26spage%3D227%26epage%3D231%26doi%3D10.1016%2FS0014-5793%2899%2900283-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Di, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kerns, E. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fan, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McConnell, O. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carter, G. T.</span></span> <span> </span><span class="NLM_article-title">High throughput artificial membrane permeability assay for blood–brain barrier</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>38</i></span>,  <span class="NLM_fpage">223</span>– <span class="NLM_lpage">232</span>, <span class="refDoi"> DOI: 10.1016/S0223-5234(03)00012-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=10.1016%2FS0223-5234%2803%2900012-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=12667689" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=1%3ACAS%3A528%3ADC%252BD3sXitlKju7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2003&pages=223-232&author=L.+Diauthor=E.+H.+Kernsauthor=K.+Fanauthor=O.+J.+McConnellauthor=G.+T.+Carter&title=High+throughput+artificial+membrane+permeability+assay+for+blood%E2%80%93brain+barrier&doi=10.1016%2FS0223-5234%2803%2900012-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">High throughput artificial membrane permeability assay for blood-brain barrier</span></div><div class="casAuthors">Di, Li; Kerns, Edward H.; Fan, Kristi; McConnell, Oliver J.; Carter, Guy T.</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">223-232</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Editions Scientifiques et Medicales Elsevier</span>)
        </div><div class="casAbstract">The recent advances in high throughput screening for biol. activities and combinatorial chem. have greatly expanded the no. of drug candidates.  Rapid screening for BBB penetration potential early in drug discovery programs provides important information for compd. selection and guidance of synthesis for desirable CNS properties.  In this paper, we discuss a modification of the parallel artificial membrane permeation assay (PAMPA) for the prediction of blood-brain barrier penetration (PAMPA-BBB).  The assay was developed with 30 structurally diverse com. drugs and validated with 14 Wyeth Research compds.  The PAMPA-BBB assay has the advantages of: predicting passive blood-brain barrier penetration with high success, high throughput, low cost, and reproducibility.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqZCg9Ka4mziLVg90H21EOLACvtfcHk0liq6CYh9S74BQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXitlKju7c%253D&md5=d228d5808ba8df23deed92c5e2bf3863</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1016%2FS0223-5234%2803%2900012-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0223-5234%252803%252900012-6%26sid%3Dliteratum%253Aachs%26aulast%3DDi%26aufirst%3DL.%26aulast%3DKerns%26aufirst%3DE.%2BH.%26aulast%3DFan%26aufirst%3DK.%26aulast%3DMcConnell%26aufirst%3DO.%2BJ.%26aulast%3DCarter%26aufirst%3DG.%2BT.%26atitle%3DHigh%2520throughput%2520artificial%2520membrane%2520permeability%2520assay%2520for%2520blood%25E2%2580%2593brain%2520barrier%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2003%26volume%3D38%26spage%3D223%26epage%3D232%26doi%3D10.1016%2FS0223-5234%2803%2900012-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H.</span></span> <span> </span><span class="NLM_article-title">Prediction of the blood–brain barrier (BBB) permeability of chemicals based on machine-learning and ensemble methods</span>. <i>Chem. Res. Toxicol.</i> <span class="NLM_year" style="font-weight: bold;">2021</span>,  <span class="NLM_volume"><i>34</i></span>,  <span class="NLM_fpage">1456</span>– <span class="NLM_lpage">1467</span>, <span class="refDoi"> DOI: 10.1021/acs.chemrestox.0c00343</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.chemrestox.0c00343" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=1%3ACAS%3A528%3ADC%252BB3MXhtFKqs7fF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=2021&pages=1456-1467&author=L.+Liuauthor=L.+Zhangauthor=H.+Fengauthor=S.+Liauthor=M.+Liuauthor=J.+Zhaoauthor=H.+Liu&title=Prediction+of+the+blood%E2%80%93brain+barrier+%28BBB%29+permeability+of+chemicals+based+on+machine-learning+and+ensemble+methods&doi=10.1021%2Facs.chemrestox.0c00343"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Prediction of the Blood-Brain Barrier (BBB) Permeability of Chemicals Based on Machine-Learning and Ensemble Methods</span></div><div class="casAuthors">Liu, Lili; Zhang, Li; Feng, Huawei; Li, Shimeng; Liu, Miao; Zhao, Jian; Liu, Hongsheng</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Research in Toxicology</span>
        (<span class="NLM_cas:date">2021</span>),
    <span class="NLM_cas:volume">34</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1456-1467</span>CODEN:
                <span class="NLM_cas:coden">CRTOEC</span>;
        ISSN:<span class="NLM_cas:issn">0893-228X</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The ability of chems. to enter the blood-brain barrier (BBB) is a key factor for central nervous system (CNS) drug development.  Although many models for BBB permeability prediction have been developed, they have insufficient accuracy (ACC) and sensitivity (SEN).  To improve performance, ensemble models were built to predict the BBB permeability of compds.  In this study, in silico ensemble-learning models were developed using 3 machine-learning algorithms and 9 mol. fingerprints from 1757 chems. (integrated from 2 published data sets) to predict BBB permeability.  The best prediction performance of the base classifier models was achieved by a prediction model based on an random forest (RF) and a MACCS mol. fingerprint with an ACC of 0.910, an area under the receiver-operating characteristic (ROC) curve (AUC) of 0.957, a SEN of 0.927, and a specificity of 0.867 in 5-fold cross-validation.  The prediction performance of the ensemble models is better than that of most of the base classifiers.  The final ensemble model has also demonstrated good accuracy for an external validation and can be used for the early screening of CNS drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpbmzj6JQ-zmLVg90H21EOLACvtfcHk0ljMTfTC-MeETA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3MXhtFKqs7fF&md5=c0c982566369b519805742a31f0214f6</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1021%2Facs.chemrestox.0c00343&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.chemrestox.0c00343%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DFeng%26aufirst%3DH.%26aulast%3DLi%26aufirst%3DS.%26aulast%3DLiu%26aufirst%3DM.%26aulast%3DZhao%26aufirst%3DJ.%26aulast%3DLiu%26aufirst%3DH.%26atitle%3DPrediction%2520of%2520the%2520blood%25E2%2580%2593brain%2520barrier%2520%2528BBB%2529%2520permeability%2520of%2520chemicals%2520based%2520on%2520machine-learning%2520and%2520ensemble%2520methods%26jtitle%3DChem.%2520Res.%2520Toxicol.%26date%3D2021%26volume%3D34%26spage%3D1456%26epage%3D1467%26doi%3D10.1021%2Facs.chemrestox.0c00343" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bujard, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petit, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carrupt, P.-A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rudaz, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schappler, J.</span></span> <span> </span><span class="NLM_article-title">HDM-PAMPA to predict gastrointestinal absorption, binding percentage, equilibrium and kinetics constants with human serum albumin and using 2 end-point measurements</span>. <i>Eur. J. Pharm. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>97</i></span>,  <span class="NLM_fpage">143</span>– <span class="NLM_lpage">150</span>, <span class="refDoi"> DOI: 10.1016/j.ejps.2016.11.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=10.1016%2Fj.ejps.2016.11.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=27816629" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvFWnsrbJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=97&publication_year=2017&pages=143-150&author=A.+Bujardauthor=C.+Petitauthor=P.-A.+Carruptauthor=S.+Rudazauthor=J.+Schappler&title=HDM-PAMPA+to+predict+gastrointestinal+absorption%2C+binding+percentage%2C+equilibrium+and+kinetics+constants+with+human+serum+albumin+and+using+2+end-point+measurements&doi=10.1016%2Fj.ejps.2016.11.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">HDM-PAMPA to predict gastrointestinal absorption, binding percentage, equilibrium and kinetics constants with human serum albumin and using 2 end-point measurements</span></div><div class="casAuthors">Bujard, Alban; Petit, Charlotte; Carrupt, Pierre-Alain; Rudaz, Serge; Schappler, Julie</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Pharmaceutical Sciences</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">97</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">143-150</span>CODEN:
                <span class="NLM_cas:coden">EPSCED</span>;
        ISSN:<span class="NLM_cas:issn">0928-0987</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The parallel artificial membrane permeability assay (PAMPA) is a high-throughput screening (HTS) technique developed to predict passive permeability through numerous different biol. membranes, such as the gastrointestinal tract (GIT), the blood brain barrier (BBB), and the dermal layer.  PAMPA is based on an artificial membrane, such as hexadecane (HDM), which separates two compartments (i.e., a donor and an acceptor compartment).  In the present study, an HDM-PAMPA method was developed with human serum albumin (HSA) under iso-pH and gradient-pH conditions to predict the percentage of binding, dissocn./assocn. consts. (Kd and Ka, resp.) and dissocn./assocn. kinetic rates (koff and kon, resp.) between a given drug and HSA.  Thanks to the kinetic properties of PAMPA, a two end-point assay was implemented to obtain all three properties.  The assay was used to measure basic, acidic, and amphoteric compds.  The protein was free in soln., allowing a direct comparison between this assay and equil. dialysis (ED).  The developed PAMPA enabled screening of up to 96 compds. in a single run, generating valuable information on absorption and distribution in a high-throughput and high-repeatable manner.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq1Yoxw5vh9ybVg90H21EOLACvtfcHk0ljMTfTC-MeETA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvFWnsrbJ&md5=8d6ec5bcba80401d7d30f32474bbceb6</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1016%2Fj.ejps.2016.11.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejps.2016.11.001%26sid%3Dliteratum%253Aachs%26aulast%3DBujard%26aufirst%3DA.%26aulast%3DPetit%26aufirst%3DC.%26aulast%3DCarrupt%26aufirst%3DP.-A.%26aulast%3DRudaz%26aufirst%3DS.%26aulast%3DSchappler%26aufirst%3DJ.%26atitle%3DHDM-PAMPA%2520to%2520predict%2520gastrointestinal%2520absorption%252C%2520binding%2520percentage%252C%2520equilibrium%2520and%2520kinetics%2520constants%2520with%2520human%2520serum%2520albumin%2520and%2520using%25202%2520end-point%2520measurements%26jtitle%3DEur.%2520J.%2520Pharm.%2520Sci.%26date%3D2017%26volume%3D97%26spage%3D143%26epage%3D150%26doi%3D10.1016%2Fj.ejps.2016.11.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lv, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shang, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sha, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span> <span> </span><span class="NLM_article-title">Penetration of verapamil across blood brain barrier following cerebral ischemia depending on both paracellular pathway and P-glycoprotein transportation</span>. <i>Neurochem. Int.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">23</span>– <span class="NLM_lpage">30</span>, <span class="refDoi"> DOI: 10.1016/j.neuint.2012.10.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=10.1016%2Fj.neuint.2012.10.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=23142723" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvV2ktLjN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2013&pages=23-30&author=W.+Fangauthor=P.+Lvauthor=X.+Gengauthor=E.+Shangauthor=Q.+Yangauthor=L.+Shaauthor=Y.+Li&title=Penetration+of+verapamil+across+blood+brain+barrier+following+cerebral+ischemia+depending+on+both+paracellular+pathway+and+P-glycoprotein+transportation&doi=10.1016%2Fj.neuint.2012.10.012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Penetration of verapamil across blood brain barrier following cerebral ischemia depending on both paracellular pathway and P-glycoprotein transportation</span></div><div class="casAuthors">Fang, Weirong; Lv, Peng; Geng, Xiaohan; Shang, Erxin; Yang, Qichuan; Sha, Lan; Li, Yunman</div><div class="citationInfo"><span class="NLM_cas:title">Neurochemistry International</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">23-30</span>CODEN:
                <span class="NLM_cas:coden">NEUIDS</span>;
        ISSN:<span class="NLM_cas:issn">0197-0186</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Blood brain barrier (BBB) dysfunction is a common facet of cerebral ischemia, and the alteration of drug transporter, P-glycoprotein (P-gp), has been documented.  This study explores influence of damaged BBB and elevated P-gp on cerebral verapamil penetration after ischemia both in vivo and in vitro.  Middle cerebral artery occlusion (MCAO) induced ischemia/reperfusion (I/R) of rats, and Na2S2O4 induced hypoxia/reoxygenation (H/R) damage of rat brain mirovessel endothelial cells (RBMECs) resp., served as BBB breakdown model in vivo and in vitro.  Evans-Blue (EB) extravagation and 125I-albumin were used to quantify BBB dysfunction; UPLC-MS/MS anal. method was performed to det. accurately the concn. of verapamil in brain tissue and cell.  Flow cytometry, immunohistochem. and western blotting were applied to evaluate transport function and protein expression of P-gp.  Overexpressed ICAM-1 and MMP-9 mediated BBB dysfunction after ischemia, which induced EB leakage and 125I-albumin uptake increase.  Enhanced accumulation of verapamil in brain tissue, but intracellular concn. reduced evidently after H/R injury.  Transcellular transportation of verapamil elevated when P-gp function or expression was inhibited after H/R injury.  These data indicated that BBB penetration of verapamil under ischemia condition was not only depending on BBB breakdown, but also regulated by P-gp.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGokw3azr0cCY7Vg90H21EOLACvtfcHk0ljMTfTC-MeETA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvV2ktLjN&md5=977b7e2390458e929243c6150cc3424f</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1016%2Fj.neuint.2012.10.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.neuint.2012.10.012%26sid%3Dliteratum%253Aachs%26aulast%3DFang%26aufirst%3DW.%26aulast%3DLv%26aufirst%3DP.%26aulast%3DGeng%26aufirst%3DX.%26aulast%3DShang%26aufirst%3DE.%26aulast%3DYang%26aufirst%3DQ.%26aulast%3DSha%26aufirst%3DL.%26aulast%3DLi%26aufirst%3DY.%26atitle%3DPenetration%2520of%2520verapamil%2520across%2520blood%2520brain%2520barrier%2520following%2520cerebral%2520ischemia%2520depending%2520on%2520both%2520paracellular%2520pathway%2520and%2520P-glycoprotein%2520transportation%26jtitle%3DNeurochem.%2520Int.%26date%3D2013%26volume%3D62%26spage%3D23%26epage%3D30%26doi%3D10.1016%2Fj.neuint.2012.10.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fallingborg, J.</span></span> <span> </span><span class="NLM_article-title">Intraluminal pH of the human gastrointestinal tract</span>. <i>Dan. Med. Bull.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>46</i></span>,  <span class="NLM_fpage">183</span>– <span class="NLM_lpage">196</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=10421978" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=1%3ACAS%3A280%3ADyaK1MzltVykug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=1999&pages=183-196&author=J.+Fallingborg&title=Intraluminal+pH+of+the+human+gastrointestinal+tract"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Intraluminal pH of the human gastrointestinal tract</span></div><div class="casAuthors">Fallingborg J</div><div class="citationInfo"><span class="NLM_cas:title">Danish medical bulletin</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">183-96</span>
        ISSN:<span class="NLM_cas:issn">0907-8916</span>.
    </div><div class="casAbstract">After a short introduction (chapter 1) methods of measuring gastrointestinal pH are described in chapter 2.  The methods are divided into intubation techniques and tubeless methods, and the advantages and disadvantages are discussed.  Measurements with pH-sensitive, radiotransmitting capsules are highlighted, and methodological problems with these capsules are described.  Chapter 3 concerns the gastrointestinal pH profile of healthy subjects.  The intraluminal pH is rapidly changed from highly acid in the stomach to about pH 6 in the duodenum.  The pH gradually increases in the small intestine from pH 6 to about pH 7.4 in the terminal ileum.  The pH drops to 5.7 in the caecum, but again gradually increases, reaching pH 6.7 in the rectum.  The physiological background of these pH values is discussed.  Chapter 4 describes the effect of gastrointestinal pH on bacterial flora, absorption of vitamins and electrolytes, and on the activity of digestive enzymes.  The pH-profile in children is described in chapter 5.  The profile is identical with that of adults, and it is therefore concluded that the release of a drug from pH-dependent, controlled-release preparations is also probably identical with that of adults.  Chapter 6 describes the correlation between certain diseases and the gastrointestinal pH.  A resection of the colon and the creation of an ileostomy do not affect the pH of the remaining gut.  An ileocaecal resection shortens the small intestinal transit time, increases pH of the proximal colon, but does not change the pH-profile of the small intestine.  Chronic pancreatitis and cystic fibrosis seem to decrease pH of the proximal small intestine.  Very low colonic pH values have been observed in severe active ulcerative colitis and in Crohn's disease, but the background and clinical implication of this phenomenon are not clear.  Chapter 7 describes the modulating effect of diet and drugs on gastrointestinal pH.  Diet primarily has an effect on the colonic pH, whereas drugs might affect both small intestinal and colonic pH.  The different effects are described.  Finally, chapter 8 summarizes the present knowledge about gastrointestinal pH, and future investigations are proposed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTo6G-nALPYxUkUaC_kzhvyfW6udTcc2ebGc9VTlog-nLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADyaK1MzltVykug%253D%253D&md5=caa72e8a9c3e34a923320c5ec31d958e</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DFallingborg%26aufirst%3DJ.%26atitle%3DIntraluminal%2520pH%2520of%2520the%2520human%2520gastrointestinal%2520tract%26jtitle%3DDan.%2520Med.%2520Bull.%26date%3D1999%26volume%3D46%26spage%3D183%26epage%3D196" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmitz, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeffrey, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wiltzius, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kussie, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferguson, K. M.</span></span> <span> </span><span class="NLM_article-title">Structural basis for inhibition of the epidermal growth factor receptor by cetuximab</span>. <i>Cancer Cell</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">301</span>– <span class="NLM_lpage">311</span>, <span class="refDoi"> DOI: 10.1016/j.ccr.2005.03.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=10.1016%2Fj.ccr.2005.03.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=15837620" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=1%3ACAS%3A528%3ADC%252BD2MXjvFSkurk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2005&pages=301-311&author=S.+Liauthor=K.+R.+Schmitzauthor=P.+D.+Jeffreyauthor=J.+J.+Wiltziusauthor=P.+Kussieauthor=K.+M.+Ferguson&title=Structural+basis+for+inhibition+of+the+epidermal+growth+factor+receptor+by+cetuximab&doi=10.1016%2Fj.ccr.2005.03.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Structural basis for inhibition of the epidermal growth factor receptor by cetuximab</span></div><div class="casAuthors">Li, Shiqing; Schmitz, Karl R.; Jeffrey, Philip D.; Wiltzius, Jed J. W.; Kussie, Paul; Ferguson, Kathryn M.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">301-311</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Recent structural studies of epidermal growth factor receptor (EGFR) family extracellular regions have identified an unexpected mechanism for ligand-induced receptor dimerization that has important implications for activation and inhibition of these receptors.  Here we describe the 2.8 Å resoln. x-ray crystal structure of the antigen binding (Fab) fragment from cetuximab (Erbitux), an inhibitory anti-EGFR antibody, in complex with the sol. extracellular region of EGFR (sEGFR).  The sEGFR is in the characteristic "autoinhibited" or "tethered" inactive configuration.  Cetuximab interacts exclusively with domain III of sEGFR, partially occluding the ligand binding region on this domain and sterically preventing the receptor from adopting the extended conformation required for dimerization.  We suggest that both these effects contribute to potent inhibition of EGFR activation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrfRvCB5ETMNLVg90H21EOLACvtfcHk0liIX1gbgOukeg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXjvFSkurk%253D&md5=7d7b01d331206edd65f0d2c5c69785dd</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1016%2Fj.ccr.2005.03.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccr.2005.03.003%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DS.%26aulast%3DSchmitz%26aufirst%3DK.%2BR.%26aulast%3DJeffrey%26aufirst%3DP.%2BD.%26aulast%3DWiltzius%26aufirst%3DJ.%2BJ.%26aulast%3DKussie%26aufirst%3DP.%26aulast%3DFerguson%26aufirst%3DK.%2BM.%26atitle%3DStructural%2520basis%2520for%2520inhibition%2520of%2520the%2520epidermal%2520growth%2520factor%2520receptor%2520by%2520cetuximab%26jtitle%3DCancer%2520Cell%26date%3D2005%26volume%3D7%26spage%3D301%26epage%3D311%26doi%3D10.1016%2Fj.ccr.2005.03.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Udugamasooriya, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dineen, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brekken, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kodadek, T.</span></span> <span> </span><span class="NLM_article-title">A peptoid ″antibody surrogate″ that antagonizes VEGF receptor 2 activity</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>130</i></span>,  <span class="NLM_fpage">5744</span>– <span class="NLM_lpage">5752</span>, <span class="refDoi"> DOI: 10.1021/ja711193x</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja711193x" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=1%3ACAS%3A528%3ADC%252BD1cXktFShsbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=130&publication_year=2008&pages=5744-5752&author=D.+G.+Udugamasooriyaauthor=S.+P.+Dineenauthor=R.+A.+Brekkenauthor=T.+Kodadek&title=A+peptoid+%E2%80%B3antibody+surrogate%E2%80%B3+that+antagonizes+VEGF+receptor+2+activity&doi=10.1021%2Fja711193x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">A Peptoid "Antibody Surrogate" That Antagonizes VEGF Receptor 2 Activity</span></div><div class="casAuthors">Udugamasooriya, D. Gomika; Dineen, Sean P.; Brekken, Rolf A.; Kodadek, Thomas</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">130</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">5744-5752</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The authors report a two-color, cell-based screen to identify specific receptor-binding compds. in a combinatorial library of peptoids displayed on beads.  The authors apply this strategy to the isolation of vascular endothelial growth factor receptor 2 (VEGFR2)-binding peptoids.  A dimeric deriv. of one of these lead compds. is shown to be an antagonist of VEGFR2 activity both in vitro and in vivo.  This methodol. provides a potentially general route to synthetic mols. that bind integral membrane receptors with affinities and specificities similar to those of antibodies, but which are far smaller and easier to make and manipulate.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp6yicSigByj7Vg90H21EOLACvtfcHk0lg4LgtKmi5mqg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXktFShsbc%253D&md5=b15f3f4eade6beff3a80142c481dab50</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1021%2Fja711193x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja711193x%26sid%3Dliteratum%253Aachs%26aulast%3DUdugamasooriya%26aufirst%3DD.%2BG.%26aulast%3DDineen%26aufirst%3DS.%2BP.%26aulast%3DBrekken%26aufirst%3DR.%2BA.%26aulast%3DKodadek%26aufirst%3DT.%26atitle%3DA%2520peptoid%2520%25E2%2580%25B3antibody%2520surrogate%25E2%2580%25B3%2520that%2520antagonizes%2520VEGF%2520receptor%25202%2520activity%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2008%26volume%3D130%26spage%3D5744%26epage%3D5752%26doi%3D10.1021%2Fja711193x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kodadek, T.</span></span> <span> </span><span class="NLM_article-title">Development of antibody surrogates for the treatment of cancers and autoimmune disease</span>. <i>Curr. Opin. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">721</span>– <span class="NLM_lpage">727</span>, <span class="refDoi"> DOI: 10.1016/j.cbpa.2010.07.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=10.1016%2Fj.cbpa.2010.07.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=20674470" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsFamur7K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2010&pages=721-727&author=T.+Kodadek&title=Development+of+antibody+surrogates+for+the+treatment+of+cancers+and+autoimmune+disease&doi=10.1016%2Fj.cbpa.2010.07.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Development of antibody surrogates for the treatment of cancers and autoimmune disease</span></div><div class="casAuthors">Kodadek, Thomas</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Chemical Biology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">721-727</span>CODEN:
                <span class="NLM_cas:coden">COCBF4</span>;
        ISSN:<span class="NLM_cas:issn">1367-5931</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Some therapeutic monoclonal antibodies function by focusing the killing power of the immune system on particular cellular targets, a process known as antibody-dependent cell-mediated cytotoxicity (ADCC).  There is considerable interest in developing small-mol. therapeutics that utilize this mechanism of action and so act as a kind of antibody surrogate.  Progress and challenges in this interesting field are reviewed here.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGor5Ws7XJiFubVg90H21EOLACvtfcHk0lg4LgtKmi5mqg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsFamur7K&md5=6388752708bb215fd8e66756ac139264</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1016%2Fj.cbpa.2010.07.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cbpa.2010.07.002%26sid%3Dliteratum%253Aachs%26aulast%3DKodadek%26aufirst%3DT.%26atitle%3DDevelopment%2520of%2520antibody%2520surrogates%2520for%2520the%2520treatment%2520of%2520cancers%2520and%2520autoimmune%2520disease%26jtitle%3DCurr.%2520Opin.%2520Chem.%2520Biol.%26date%3D2010%26volume%3D14%26spage%3D721%26epage%3D727%26doi%3D10.1016%2Fj.cbpa.2010.07.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Landrum, G.</span></span> <span> </span><span class="NLM_article-title">RDKit: Open-Source Cheminformatics from Machine Learning to Chemical Registration</span>. In  <i>Abstracts of Papers of the American Chemical Society</i>; <span class="NLM_publisher-name">American Chemical Society</span>: <span class="NLM_publisher-loc">Washington, DC</span>, <span class="NLM_year">2019</span>; Vol.  <span class="NLM_volume">258</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2019&author=G.+Landrum&title=Abstracts+of+Papers+of+the+American+Chemical+Society"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DLandrum%26aufirst%3DG.%26atitle%3DRDKit%253A%2520Open-Source%2520Cheminformatics%2520from%2520Machine%2520Learning%2520to%2520Chemical%2520Registration%26btitle%3DAbstracts%2520of%2520Papers%2520of%2520the%2520American%2520Chemical%2520Society%26pub%3DAmerican%2520Chemical%2520Society%26date%3D2019%26volume%3D258" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span> <i>The PyMOL Molecular
Graphics System</i>, Version 1.3r1, <span class="NLM_year">2010</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+The+PyMOL+Molecular%0AGraphics+System%2C+Version+1.3r1%2C+2010."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DThe%2520PyMOL%2520Molecular%250AGraphics%2520System%26date%3D2010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sastry, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adzhigirey, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Day, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Annabhimoju, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sherman, W.</span></span> <span> </span><span class="NLM_article-title">Protein and ligand preparation: parameters, protocols, and influence on virtual screening enrichments</span>. <i>J. Comput.-Aided Mol. Des.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">221</span>– <span class="NLM_lpage">234</span>, <span class="refDoi"> DOI: 10.1007/s10822-013-9644-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=10.1007%2Fs10822-013-9644-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=23579614" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=1%3ACAS%3A280%3ADC%252BC3srksFekug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2013&pages=221-234&author=G.+M.+Sastryauthor=M.+Adzhigireyauthor=T.+Dayauthor=R.+Annabhimojuauthor=W.+Sherman&title=Protein+and+ligand+preparation%3A+parameters%2C+protocols%2C+and+influence+on+virtual+screening+enrichments&doi=10.1007%2Fs10822-013-9644-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Protein and ligand preparation: parameters, protocols, and influence on virtual screening enrichments</span></div><div class="casAuthors">Sastry G Madhavi; Adzhigirey Matvey; Day Tyler; Annabhimoju Ramakrishna; Sherman Woody</div><div class="citationInfo"><span class="NLM_cas:title">Journal of computer-aided molecular design</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">221-34</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Structure-based virtual screening plays an important role in drug discovery and complements other screening approaches.  In general, protein crystal structures are prepared prior to docking in order to add hydrogen atoms, optimize hydrogen bonds, remove atomic clashes, and perform other operations that are not part of the x-ray crystal structure refinement process.  In addition, ligands must be prepared to create 3-dimensional geometries, assign proper bond orders, and generate accessible tautomer and ionization states prior to virtual screening.  While the prerequisite for proper system preparation is generally accepted in the field, an extensive study of the preparation steps and their effect on virtual screening enrichments has not been performed.  In this work, we systematically explore each of the steps involved in preparing a system for virtual screening.  We first explore a large number of parameters using the Glide validation set of 36 crystal structures and 1,000 decoys.  We then apply a subset of protocols to the DUD database.  We show that database enrichment is improved with proper preparation and that neglecting certain steps of the preparation process produces a systematic degradation in enrichments, which can be large for some targets.  We provide examples illustrating the structural changes introduced by the preparation that impact database enrichment.  While the work presented here was performed with the Protein Preparation Wizard and Glide, the insights and guidance are expected to be generalizable to structure-based virtual screening with other docking methods.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQd2Qiai-OqeXqj_yg7twvpfW6udTcc2eZv_zC9KFPCQrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3srksFekug%253D%253D&md5=9eb1a820e121aca2742ed53f24481ace</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1007%2Fs10822-013-9644-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-013-9644-8%26sid%3Dliteratum%253Aachs%26aulast%3DSastry%26aufirst%3DG.%2BM.%26aulast%3DAdzhigirey%26aufirst%3DM.%26aulast%3DDay%26aufirst%3DT.%26aulast%3DAnnabhimoju%26aufirst%3DR.%26aulast%3DSherman%26aufirst%3DW.%26atitle%3DProtein%2520and%2520ligand%2520preparation%253A%2520parameters%252C%2520protocols%252C%2520and%2520influence%2520on%2520virtual%2520screening%2520enrichments%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2013%26volume%3D27%26spage%3D221%26epage%3D234%26doi%3D10.1007%2Fs10822-013-9644-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article has not yet been cited by other publications.</p></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00607/20210723/images/medium/jm1c00607_0008.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00607/20210723/images/large/jm1c00607_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00607&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00607/20210723/images/medium/jm1c00607_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00607/20210723/images/large/jm1c00607_0002.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. (A) Scheme for the design of γ-AApeptides OBTC library. The binding affinity of <b>M-2-2-F</b> (B) and <b>M-2-5-F</b> (C) to EGFR measured by fluorescence polarization (FP) assay.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00607/20210723/images/large/jm1c00607_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00607&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00607/20210723/images/medium/jm1c00607_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00607/20210723/images/large/jm1c00607_0003.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. (A) Scheme illustrating the proposed mechanism of cyclic peptides to inhibit EGFR phosphorylation. Chemical structure of (B) <b>M-2-2</b> and (C) <b>M-2-5</b>. (D) Effect of 20 μM <b>M-2-2</b> and <b>M-2-5</b> on EGF-induced EGFR phosphorylation determined using western blot analysis. GAPDH was used as an internal control. Data are representative of three independent experiments. (E) Expression ratio of relative P-EGFR/GAPDH is presented. *<i>P</i> < 0.05 vs EGF stimulation. (F) Starved A549 cells were pretreated with <b>M-2-5</b> (0–50 μM) for 4 h before stimulated with EGF (100 ng/mL) for 5 min. Cell lysates were analyzed by Western blots. Data are representative of three independent experiments. (G) Relative band density of P-EGFR/GAPDH, P-AKT/GAPDH, and P-ERK/GAPDH. The intensity of each strip was analyzed by ImageJ software. (H) Effect of <b>M-2-5</b> on cell proliferation inhibition in vitro. A549 cells were treated with various concentrations of <b>M-2-5</b> in the presence of EGF for 48 h, and cell viability was determined with CCK-8 kits. Means were derived from four replicates (<i>n</i> = 4). **<i>P</i> < 0.05 vs EGF stimulation; ***<i>P</i> < 0.001 vs EGF stimulation.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00607/20210723/images/large/jm1c00607_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00607&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00607/20210723/images/medium/jm1c00607_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00607/20210723/images/large/jm1c00607_0004.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. <b>M-2-5</b> inhibits cell migration in vitro. (A) Wound-healing assay was performed on A549 cells in the absence or presence of <b>M-2-5</b> (10 and 25 μM). The experiments were repeated three times with similar results, and images of one representative experiment taken at 0 and 30 h after wound scratching are shown. A549 cells were photographed at 4× magnification. (B) Percentage of wound healing covered at 30 h. **<i>P</i> < 0.01 compared with control at 30 h; ***<i>P</i> < 0.001 compared with control at 30 h. (C) Transwell assay migration assay of A549 cells upon treatment of EGF alone or EGF+ <b>M-2-5</b> (25 μM). The A549 cells without EGF and <b>M-2-5</b> treatment were used as control. After stained with crystal violet, A549 cells were photographed at 20× magnification. (D) Quantitative results of migration assays. The columns represent the mean of cell numbers from at least three independent experiments. The relative cell migration was counted manually, and the number of migration cells in the control group was set to 100%. *<i>P</i> < 0.05 compared with control at 12 h, ***<i>P</i> < 0.001 compared with EGF stimulation at 12 h.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00607/20210723/images/large/jm1c00607_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00607&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00607/20210723/images/medium/jm1c00607_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00607/20210723/images/large/jm1c00607_0005.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. PAMPA assays of <b>M-2-5</b>. All compounds were tested in triplicate. (A) PAMPA-BBB assay on standards and <b>M-2-5</b> at pH 7.4. (B) PAMPA-GIT assay on standards and <b>M-2-5</b> at different pH conditions.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00607/20210723/images/large/jm1c00607_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00607&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00607/20210723/images/medium/jm1c00607_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00607/20210723/images/large/jm1c00607_0006.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Therapeutic efficacy of <b>M-2-5</b> in A549 xenograft models. (A) Total tumor volume assessed on a time course. The day when drugs were initially administered was marked as day 0, and arrows show the injection time. Tumor volume was obtained once every 2 days until day 18. (B) Western blot analysis of phosphor-EGFR and downstream signaling pathway in tumor lysates. GAPDH was used as a loading control. (C) Immunohistochemical staining for P-EGFR, P-ERK, and P-AKT.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00607/20210723/images/large/jm1c00607_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00607&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00607/20210723/images/medium/jm1c00607_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00607/20210723/images/large/jm1c00607_0007.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Molecular docking of <b>M-2-5</b> with EGFR extracellular domain (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1YY9">1YY9</a>). The docking was performed using the Glide module in Schrödinger (2015) with default parameters, and the docking results were visualized in Pymol.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00607/20210723/images/large/jm1c00607_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00607&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i31">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_86125" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_86125" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 41 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yarden, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sliwkowski, M. X.</span></span> <span> </span><span class="NLM_article-title">Untangling the ErbB signalling network</span>. <i>Nat. Rev. Mol. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">127</span>– <span class="NLM_lpage">137</span>, <span class="refDoi"> DOI: 10.1038/35052073</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=10.1038%2F35052073" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=11252954" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=1%3ACAS%3A528%3ADC%252BD3MXivVWnt7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2001&pages=127-137&author=Y.+Yardenauthor=M.+X.+Sliwkowski&title=Untangling+the+ErbB+signalling+network&doi=10.1038%2F35052073"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Untangling the ErbB signalling network</span></div><div class="casAuthors">Yarden, Yosef; Sliwkowski, Mark X.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Molecular Cell Biology</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">127-137</span>CODEN:
                <span class="NLM_cas:coden">NRMCBP</span>;
        ISSN:<span class="NLM_cas:issn">1471-0072</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review, with 112 refs.  When epidermal growth factor and its relatives bind the ErbB family of receptors, they trigger a rich network of signaling pathways, culminating in responses ranging from cell division to death, motility to adhesion.  The network is often dysregulated in cancer and lends credence to the mantra that mol. understanding yields clin. benefit: over 25,000 women with breast cancer have now been treated with Herceptin, a recombinant antibody designed to block the receptor ErbB2.  Likewise, small-mol. enzyme inhibitors and monoclonal antibodies to ErbB1 are in advanced phases of clin. testing.  What can this pathway teach us about translating basic science into clin. use.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGovhWL1HObvNrVg90H21EOLACvtfcHk0lgUPKrHOfr_ZQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXivVWnt7k%253D&md5=1561a9666f77fce888be821399870cd6</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2F35052073&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F35052073%26sid%3Dliteratum%253Aachs%26aulast%3DYarden%26aufirst%3DY.%26aulast%3DSliwkowski%26aufirst%3DM.%2BX.%26atitle%3DUntangling%2520the%2520ErbB%2520signalling%2520network%26jtitle%3DNat.%2520Rev.%2520Mol.%2520Cell%2520Biol.%26date%3D2001%26volume%3D2%26spage%3D127%26epage%3D137%26doi%3D10.1038%2F35052073" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hynes, N. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MacDonald, G.</span></span> <span> </span><span class="NLM_article-title">ErbB receptors and signaling pathways in cancer</span>. <i>Curr. Opin. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">177</span>– <span class="NLM_lpage">184</span>, <span class="refDoi"> DOI: 10.1016/j.ceb.2008.12.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=10.1016%2Fj.ceb.2008.12.010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=19208461" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=1%3ACAS%3A528%3ADC%252BD1MXksVClurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2009&pages=177-184&author=N.+E.+Hynesauthor=G.+MacDonald&title=ErbB+receptors+and+signaling+pathways+in+cancer&doi=10.1016%2Fj.ceb.2008.12.010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">ErbB receptors and signaling pathways in cancer</span></div><div class="casAuthors">Hynes, Nancy E.; MacDonald, Gwen</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Cell Biology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">177-184</span>CODEN:
                <span class="NLM_cas:coden">COCBE3</span>;
        ISSN:<span class="NLM_cas:issn">0955-0674</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  The ErbB receptor tyrosine kinases play important roles in normal physiol. and in cancer.  Epidermal growth factor receptor (EGFR) and ErbB2 in particular are mutated in many epithelial tumors, and clin. studies suggest that they play roles in cancer development and progression.  These receptors were intensely studied, not only to understand the mechanisms underlying their oncogenic potential, but also to exploit them as therapeutic targets.  ErbB receptors activate a multiplicity of intracellular pathways via their ability to interact with numerous signal transducers.  Furthermore, there are now many ErbB-targeted inhibitors used in the clinic.  In this review we will conc. on breast tumors with ERBB2 gene amplification/receptor overexpression and non-small cell lung cancer (NSCLC) with activating EGFR mutations.  We will discuss data showing the important role that the PI3K/Akt pathway plays, not only in cancer development, but also in response to targeted therapies.  Finally, mechanisms contributing to resistance to ErbB-targeted therapeutics will also be discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr0lju0mdtYpbVg90H21EOLACvtfcHk0lhY-irp0bQRSg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXksVClurc%253D&md5=217703dc2eff6a8157ce74a7c100a3fe</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1016%2Fj.ceb.2008.12.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ceb.2008.12.010%26sid%3Dliteratum%253Aachs%26aulast%3DHynes%26aufirst%3DN.%2BE.%26aulast%3DMacDonald%26aufirst%3DG.%26atitle%3DErbB%2520receptors%2520and%2520signaling%2520pathways%2520in%2520cancer%26jtitle%3DCurr.%2520Opin.%2520Cell%2520Biol.%26date%3D2009%26volume%3D21%26spage%3D177%26epage%3D184%26doi%3D10.1016%2Fj.ceb.2008.12.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yarden, Y.</span></span> <span> </span><span class="NLM_article-title">The EGFR family and its ligands in human cancer: signalling mechanisms and therapeutic opportunities</span>. <i>Eur. J. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>37</i></span>,  <span class="NLM_fpage">3</span>– <span class="NLM_lpage">8</span>, <span class="refDoi"> DOI: 10.1016/S0959-8049(01)00230-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=10.1016%2FS0959-8049%2801%2900230-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=11342194" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=1%3ACAS%3A528%3ADC%252BD3MXntlOku78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=2001&pages=3-8&author=Y.+Yarden&title=The+EGFR+family+and+its+ligands+in+human+cancer%3A+signalling+mechanisms+and+therapeutic+opportunities&doi=10.1016%2FS0959-8049%2801%2900230-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">The EGFR family and its ligands in human cancer signalling mechanisms and therapeutic opportunities</span></div><div class="casAuthors">Yarden, Y.</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Cancer</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">37</span>
        (<span class="NLM_cas:issue">Suppl. 4</span>),
    <span class="NLM_cas:pages">S3-S8</span>CODEN:
                <span class="NLM_cas:coden">EJCAEL</span>;
        ISSN:<span class="NLM_cas:issn">0959-8049</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ltd.</span>)
        </div><div class="casAbstract">A review.  Growth factors and their transmembrane receptor tyrosine kinases play important roles in cell proliferation, survival, migration and differentiation.  One group of growth factors, comprising epidermal growth factor (EGF)-like proteins and neuregulins, stimulates cells to divide by activating members of the EGF receptor (EGFR) family, which consists of the EGFR itself and the receptors known as HER2-4.  This highly conserved signaling module plays a fundamental role in the morphogenesis of a diverse spectrum of organisms, ranging from humans to nematodes, and has also been implicated in the development and growth of many types of human tumor cells.  In humans, more than 30 ligands and the EGFR family of four receptors lie at the head of a complex, multi-layered signal-transduction network.  Different activated receptor-ligand complexes vary in both the strength and type of cellular responses that they induce.  Anal. of the multiple processes that modulate EGFR signal transduction, such as receptor heterodimerization and endocytosis, has revealed new therapeutic opportunities and elucidated mechanisms contributing to the efficacy of existing anticancer treatments.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGruCkMkD5i497Vg90H21EOLACvtfcHk0lhY-irp0bQRSg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXntlOku78%253D&md5=b512e9006ed63e9f01f2cdd525dc4d96</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1016%2FS0959-8049%2801%2900230-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0959-8049%252801%252900230-1%26sid%3Dliteratum%253Aachs%26aulast%3DYarden%26aufirst%3DY.%26atitle%3DThe%2520EGFR%2520family%2520and%2520its%2520ligands%2520in%2520human%2520cancer%253A%2520signalling%2520mechanisms%2520and%2520therapeutic%2520opportunities%26jtitle%3DEur.%2520J.%2520Cancer%26date%3D2001%26volume%3D37%26spage%3D3%26epage%3D8%26doi%3D10.1016%2FS0959-8049%2801%2900230-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Holbro, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hynes, N. E.</span></span> <span> </span><span class="NLM_article-title">ErbB Receptors: directing key signaling networks throughout life</span>. <i>Annu. Rev. Pharmacol. Toxicol.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>44</i></span>,  <span class="NLM_fpage">195</span>– <span class="NLM_lpage">217</span>, <span class="refDoi"> DOI: 10.1146/annurev.pharmtox.44.101802.121440</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=10.1146%2Fannurev.pharmtox.44.101802.121440" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=14744244" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhvVCkt7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2004&pages=195-217&author=T.+Holbroauthor=N.+E.+Hynes&title=ErbB+Receptors%3A+directing+key+signaling+networks+throughout+life&doi=10.1146%2Fannurev.pharmtox.44.101802.121440"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">ErbB receptors: Directing key signaling networks throughout life</span></div><div class="casAuthors">Holbro, Thomas; Hynes, Nancy E.</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Pharmacology and Toxicology</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">195-217</span>CODEN:
                <span class="NLM_cas:coden">ARPTDI</span>;
        ISSN:<span class="NLM_cas:issn">0362-1642</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews Inc.</span>)
        </div><div class="casAbstract">A review.  The EGF-related peptides bind the ErbB receptors, inducing the formation of different homo- and heterodimers.  Receptor dimerization promotes activation of the intrinsic kinase, leading to phosphorylation of specific tyrosines located in the ErbB's cytoplasmic region.  These phosphorylated residues serve as docking sites for a variety of signaling mols. whose recruitment stimulates intracellular signaling cascades, which ultimately control diverse genetic programs.  Particular ligand-receptor complexes have essential roles in embryonic development as well as in the adult.  Finally, ErbB receptors are being pursued as therapeutic targets because aberrant ErbB activity has been obsd. in many human cancers.  In this review, we discuss these data in more detail, illustrating the importance of tightly regulated ErbB signaling throughout life.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoGr3YQEvuFpLVg90H21EOLACvtfcHk0lhTaiRGZ_tPZw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhvVCkt7c%253D&md5=cd88afae65520994f9a7b4916cc74576</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1146%2Fannurev.pharmtox.44.101802.121440&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev.pharmtox.44.101802.121440%26sid%3Dliteratum%253Aachs%26aulast%3DHolbro%26aufirst%3DT.%26aulast%3DHynes%26aufirst%3DN.%2BE.%26atitle%3DErbB%2520Receptors%253A%2520directing%2520key%2520signaling%2520networks%2520throughout%2520life%26jtitle%3DAnnu.%2520Rev.%2520Pharmacol.%2520Toxicol.%26date%3D2004%26volume%3D44%26spage%3D195%26epage%3D217%26doi%3D10.1146%2Fannurev.pharmtox.44.101802.121440" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rocha-Lima, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soares, H. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raez, L. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singal, R.</span></span> <span> </span><span class="NLM_article-title">EGFR targeting of solid tumors</span>. <i>Cancer Control</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">295</span>– <span class="NLM_lpage">304</span>, <span class="refDoi"> DOI: 10.1177/107327480701400313</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=10.1177%2F107327480701400313" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=17615536" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=1%3ACAS%3A280%3ADC%252BD2szpslWksA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2007&pages=295-304&author=C.+M.+Rocha-Limaauthor=H.+P.+Soaresauthor=L.+E.+Raezauthor=R.+Singal&title=EGFR+targeting+of+solid+tumors&doi=10.1177%2F107327480701400313"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">EGFR targeting of solid tumors</span></div><div class="casAuthors">Rocha-Lima Caio M; Soares Heloisa P; Raez Luis E; Singal Rakesh</div><div class="citationInfo"><span class="NLM_cas:title">Cancer control : journal of the Moffitt Cancer Center</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">295-304</span>
        ISSN:<span class="NLM_cas:issn">1073-2748</span>.
    </div><div class="casAbstract">BACKGROUND:  Recent clinical trials suggest that epidermal growth factor receptor (EGFR)-targeted agents could benefit many patients with cancer.  METHODS:  We review the current status of several EGFR-targeted therapies in cancer patients and address the efficacy of theses drugs as monotherapy or in combination with other drugs and/or treatments.  RESULTS:  Cetuximab is the most widely studied anti-EGFR monoclonal antibody.  Other monoclonal antibody agents under investigation are panitumumab, matuzumab, MDX-447, nimutozumab, and mAb806.  Extensive research has also evaluated the efficacy of EGFR tyrosine kinase inhibitors such as erlotinib, gefitinib, EKB-569, lapatinib (GW572016), PKI-166, and canertinib (CI-1033).  All of these agents have been studied for the treatment of colorectal, lung, breast, pancreatic, renal, head and neck, gynecologic, and prostate cancer.  Currently, cetuximab and panitumumab are FDA approved for the treatment of metastatic colorectal cancer.  Additionally, cetuximab is approved for head and neck cancer.  Erlotinib is FDA approved for advanced/metastatic lung cancer.  Erlotinib in combination with gemcitabine is approved for advanced/metastatic pancreatic cancer treatment.  CONCLUSIONS:  EGFR-targeted agents have already shown utility in different scenarios.  Researchers are continuously investigating additional cancer types and combined treatment modalities that could also benefit from the use of EGFR-targeted agents.  Careful patient selection through the identification of specific biologic markers, such as gene expression, genomic polymorphism, and posttranslational modifications of EGFR downstream effectors, most likely will contribute to the successful use of these agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQTb3BRcWgtjBWfpMXQLRstfW6udTcc2eZS5GfqxMnZErntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2szpslWksA%253D%253D&md5=d22e4054f58010acebdabccc58635364</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1177%2F107327480701400313&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F107327480701400313%26sid%3Dliteratum%253Aachs%26aulast%3DRocha-Lima%26aufirst%3DC.%2BM.%26aulast%3DSoares%26aufirst%3DH.%2BP.%26aulast%3DRaez%26aufirst%3DL.%2BE.%26aulast%3DSingal%26aufirst%3DR.%26atitle%3DEGFR%2520targeting%2520of%2520solid%2520tumors%26jtitle%3DCancer%2520Control%26date%3D2007%26volume%3D14%26spage%3D295%26epage%3D304%26doi%3D10.1177%2F107327480701400313" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nicholas, M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lukas, R. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jafri, N. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Faoro, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salgia, R.</span></span> <span> </span><span class="NLM_article-title">Epidermal growth factor receptor–mediated signal transduction in the development and therapy of gliomas</span>. <i>Clin. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">7261</span>– <span class="NLM_lpage">7270</span>, <span class="refDoi"> DOI: 10.1158/1078-0432.CCR-06-0874</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=10.1158%2F1078-0432.CCR-06-0874" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=17189397" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtlCrtr7O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2006&pages=7261-7270&author=M.+K.+Nicholasauthor=R.+V.+Lukasauthor=N.+F.+Jafriauthor=L.+Faoroauthor=R.+Salgia&title=Epidermal+growth+factor+receptor%E2%80%93mediated+signal+transduction+in+the+development+and+therapy+of+gliomas&doi=10.1158%2F1078-0432.CCR-06-0874"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Epidermal Growth Factor Receptor-Mediated Signal Transduction in the Development and Therapy of Gliomas</span></div><div class="casAuthors">Nicholas, M. Kelly; Lukas, Rimas V.; Jafri, Nazia F.; Faoro, Leonardo; Salgia, Ravi</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">7261-7270</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">A review.  The epidermal growth factor receptor (EGFR) and its ligands figure prominently in the biol. of gliomas, the most common tumors of the central nervous system (CNS).  Although their histol. classification seems to be straightforward, these tumors constitute a heterogeneous class of related neoplasms.  They are assocd. with a variety of mol. abnormalities affecting signal transduction, transcription factors, apoptosis, angiogenesis, and the extracellular matrix.  Under normal conditions, these same interacting factors drive CNS growth and development.  We are now recognizing the diverse mol. genetic heterogeneity that underlies tumors classified histol. into three distinct grades.  This recognition is leading to new therapeutic strategies targeted directly at specific mol. subtypes.  In this article, we will review the role of EGFR and related mol. pathways in the genesis of the normal CNS and their relationship to glial tumorigenesis.  We will discuss barriers to effective treatment as they relate to anat. specialization of the CNS.  We will also consider the ways in which specific EGFR alterations common to glioma reflect outcomes following treatment with targeted therapies, all with an eye towards applying this understanding to improved patient outcomes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo9JmBdtem1mbVg90H21EOLACvtfcHk0lhTaiRGZ_tPZw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtlCrtr7O&md5=9649bd7ee74f26a8f8bfd00fb60ca2c8</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1158%2F1078-0432.CCR-06-0874&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1078-0432.CCR-06-0874%26sid%3Dliteratum%253Aachs%26aulast%3DNicholas%26aufirst%3DM.%2BK.%26aulast%3DLukas%26aufirst%3DR.%2BV.%26aulast%3DJafri%26aufirst%3DN.%2BF.%26aulast%3DFaoro%26aufirst%3DL.%26aulast%3DSalgia%26aufirst%3DR.%26atitle%3DEpidermal%2520growth%2520factor%2520receptor%25E2%2580%2593mediated%2520signal%2520transduction%2520in%2520the%2520development%2520and%2520therapy%2520of%2520gliomas%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2006%26volume%3D12%26spage%3D7261%26epage%3D7270%26doi%3D10.1158%2F1078-0432.CCR-06-0874" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rowinsky, E. K.</span></span> <span> </span><span class="NLM_article-title">The erbB Family: Targets for therapeutic development against cancer and therapeutic strategies using monoclonal antibodies and tyrosine kinase inhibitors</span>. <i>Annu. Rev. Med.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">433</span>– <span class="NLM_lpage">457</span>, <span class="refDoi"> DOI: 10.1146/annurev.med.55.091902.104433</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=10.1146%2Fannurev.med.55.091902.104433" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=14746530" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=1%3ACAS%3A528%3ADC%252BD2cXitVWru74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2004&pages=433-457&author=E.+K.+Rowinsky&title=The+erbB+Family%3A+Targets+for+therapeutic+development+against+cancer+and+therapeutic+strategies+using+monoclonal+antibodies+and+tyrosine+kinase+inhibitors&doi=10.1146%2Fannurev.med.55.091902.104433"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">The erbB family: targets for therapeutic development against cancer and therapeutic strategies using monoclonal antibodies and tyrosine kinase inhibitors</span></div><div class="casAuthors">Rowinsky, Eric K.</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Medicine</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">433-457</span>CODEN:
                <span class="NLM_cas:coden">ARMCAH</span>;
        ISSN:<span class="NLM_cas:issn">0066-4219</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews Inc.</span>)
        </div><div class="casAbstract">A review.  The overexpression and aberrant function of members of the erbB family of receptors, particularly erbB1 (also known as epidermal growth factor receptor), and its ligands in many human cancers have provided a rationale for targeting this signaling network with novel approaches.  ErbB1 is a selective target for inhibiting cancers because its activation often confers a proliferative advantage.  Activation of the erbB1 tyrosine kinase provides signals that drive dysregulated proliferation, invasion, metastasis, angiogenesis, and cell survival, and its inhibition has potential in both the treatment and prevention of these malignancies.  Based on the structure and function of erbB1, two therapeutic strategies have been developed.  The first uses human monoclonal antibodies (MAbs) generated against the receptor's ligand-binding extracellular domain.  These MAbs block binding of receptor-activating ligands, and, in some cases, can induce receptor endocytosis and downregulation.  The second uses small mols. that compete with ATP for binding to the receptor's kinase pocket, thereby blocking receptor activation and the transduction of postreceptor signals.  Early clin. studies suggest that both approaches are well tolerated and can induce clin. activity in many common malignancies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGran3L8bVTvFrVg90H21EOLACvtfcHk0lhlATyAJuexSQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXitVWru74%253D&md5=d44bcf33198e16162432c87a016a192c</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1146%2Fannurev.med.55.091902.104433&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev.med.55.091902.104433%26sid%3Dliteratum%253Aachs%26aulast%3DRowinsky%26aufirst%3DE.%2BK.%26atitle%3DThe%2520erbB%2520Family%253A%2520Targets%2520for%2520therapeutic%2520development%2520against%2520cancer%2520and%2520therapeutic%2520strategies%2520using%2520monoclonal%2520antibodies%2520and%2520tyrosine%2520kinase%2520inhibitors%26jtitle%3DAnnu.%2520Rev.%2520Med.%26date%3D2004%26volume%3D55%26spage%3D433%26epage%3D457%26doi%3D10.1146%2Fannurev.med.55.091902.104433" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mosesson, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yarden, Y.</span></span> <span> </span><span class="NLM_article-title">Oncogenic growth factor receptors: implications for signal transduction therapy</span>. <i>Semin. Cancer Biol.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">262</span>– <span class="NLM_lpage">270</span>, <span class="refDoi"> DOI: 10.1016/j.semcancer.2004.04.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=10.1016%2Fj.semcancer.2004.04.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=15219619" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=1%3ACAS%3A528%3ADC%252BD2cXlt1agu78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2004&pages=262-270&author=Y.+Mosessonauthor=Y.+Yarden&title=Oncogenic+growth+factor+receptors%3A+implications+for+signal+transduction+therapy&doi=10.1016%2Fj.semcancer.2004.04.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Oncogenic growth factor receptors: implications for signal transduction therapy</span></div><div class="casAuthors">Mosesson, Yaron; Yarden, Yosef</div><div class="citationInfo"><span class="NLM_cas:title">Seminars in Cancer Biology</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">262-270</span>CODEN:
                <span class="NLM_cas:coden">SECBE7</span>;
        ISSN:<span class="NLM_cas:issn">1044-579X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science B.V.</span>)
        </div><div class="casAbstract">A review.  Growth factors of the EGF family and their resp. ErbB/HER receptor tyrosine kinases underlie many landmarks of tumor cells, including excessive growth, invasive behavior and attraction of blood vessels.  Enhanced expression of ErbB proteins, existence of permanently active receptor mutants and occurrence of autocrine loops are frequently obsd. in human cancer, and in some cases they assoc. with poor disease outcome.  The four ErbB proteins and their 11 ligands act within a layered signaling network coordinated by ErbB-2/HER2, the most oncogenic family member.  Drugs that intercept signals emanating from ErbB-2 and ErbB-1 are already in routine clin. application.  Here the authors review three major strategies to develop new ErbB-targeted therapies.  These are monoclonal anti-receptor antibodies, specific tyrosine kinase inhibitors and antagonists of heat shock protein 90.  The underlying mechanisms are critically examd., with an emphasis on potential drug combinations, which hold promise for enhanced clin. efficacy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp-JnXVxWxJ9LVg90H21EOLACvtfcHk0lhlATyAJuexSQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXlt1agu78%253D&md5=dff2f3a6fdf347b0ffa2c979871abbda</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1016%2Fj.semcancer.2004.04.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.semcancer.2004.04.005%26sid%3Dliteratum%253Aachs%26aulast%3DMosesson%26aufirst%3DY.%26aulast%3DYarden%26aufirst%3DY.%26atitle%3DOncogenic%2520growth%2520factor%2520receptors%253A%2520implications%2520for%2520signal%2520transduction%2520therapy%26jtitle%3DSemin.%2520Cancer%2520Biol.%26date%3D2004%26volume%3D14%26spage%3D262%26epage%3D270%26doi%3D10.1016%2Fj.semcancer.2004.04.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Paez, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jänne, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tracy, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greulich, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gabriel, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herman, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaye, F. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindeman, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boggon, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Naoki, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sasaki, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fujii, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eck, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sellers, W. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meyerson, M.</span></span> <span> </span><span class="NLM_article-title">EGFR mutations in lung cancer: correlation with clinical response to gefitinib therapy</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>304</i></span>,  <span class="NLM_fpage">1497</span>– <span class="NLM_lpage">1500</span>, <span class="refDoi"> DOI: 10.1126/science.1099314</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=10.1126%2Fscience.1099314" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=15118125" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=1%3ACAS%3A528%3ADC%252BD2cXksVGmsbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=304&publication_year=2004&pages=1497-1500&author=J.+G.+Paezauthor=P.+A.+J%C3%A4nneauthor=J.+C.+Leeauthor=S.+Tracyauthor=H.+Greulichauthor=S.+Gabrielauthor=P.+Hermanauthor=F.+J.+Kayeauthor=N.+Lindemanauthor=T.+J.+Boggonauthor=K.+Naokiauthor=H.+Sasakiauthor=Y.+Fujiiauthor=M.+J.+Eckauthor=W.+R.+Sellersauthor=B.+E.+Johnsonauthor=M.+Meyerson&title=EGFR+mutations+in+lung+cancer%3A+correlation+with+clinical+response+to+gefitinib+therapy&doi=10.1126%2Fscience.1099314"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">EGFR Mutations in Lung Cancer: Correlation with Clinical Response to Gefitinib Therapy</span></div><div class="casAuthors">Paez, J. Guillermo; Jaenne, Pasi A.; Lee, Jeffrey C.; Tracy, Sean; Greulich, Heidi; Gabriel, Stacey; Herman, Paula; Kaye, Frederic J.; Lindeman, Neal; Boggon, Titus J.; Naoki, Katsuhiko; Sasaki, Hidefumi; Fujii, Yoshitaka; Eck, Michael J.; Sellers, William R.; Johnson, Bruce E.; Meyerson, Matthew</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">304</span>
        (<span class="NLM_cas:issue">5676</span>),
    <span class="NLM_cas:pages">1497-1500</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Receptor tyrosine kinase genes were sequenced in non-small cell lung cancer (NSCLC) and matched normal tissue.  Somatic mutations of the epidermal growth factor receptor gene EGFR were found in 15 of 58 unselected tumors from Japan and 1 of 61 from the United States.  Treatment with the EGFR kinase inhibitor gefitinib (Iressa) causes tumor regression in some patients with NSCLC, more frequently in Japan.  EGFR mutations were found in addnl. lung cancer samples from U.S. patients who responded to gefitinib therapy and in a lung adenocarcinoma cell line that was hypersensitive to growth inhibition by gefitinib, but not in gefitinib-insensitive tumors or cell lines.  These results suggest that EGFR mutations may predict sensitivity to gefitinib.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoGxLJH5f3Kb7Vg90H21EOLACvtfcHk0lhlATyAJuexSQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXksVGmsbs%253D&md5=6df1b11bf9b3c284fbe000e34c61ebac</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1126%2Fscience.1099314&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1099314%26sid%3Dliteratum%253Aachs%26aulast%3DPaez%26aufirst%3DJ.%2BG.%26aulast%3DJ%25C3%25A4nne%26aufirst%3DP.%2BA.%26aulast%3DLee%26aufirst%3DJ.%2BC.%26aulast%3DTracy%26aufirst%3DS.%26aulast%3DGreulich%26aufirst%3DH.%26aulast%3DGabriel%26aufirst%3DS.%26aulast%3DHerman%26aufirst%3DP.%26aulast%3DKaye%26aufirst%3DF.%2BJ.%26aulast%3DLindeman%26aufirst%3DN.%26aulast%3DBoggon%26aufirst%3DT.%2BJ.%26aulast%3DNaoki%26aufirst%3DK.%26aulast%3DSasaki%26aufirst%3DH.%26aulast%3DFujii%26aufirst%3DY.%26aulast%3DEck%26aufirst%3DM.%2BJ.%26aulast%3DSellers%26aufirst%3DW.%2BR.%26aulast%3DJohnson%26aufirst%3DB.%2BE.%26aulast%3DMeyerson%26aufirst%3DM.%26atitle%3DEGFR%2520mutations%2520in%2520lung%2520cancer%253A%2520correlation%2520with%2520clinical%2520response%2520to%2520gefitinib%2520therapy%26jtitle%3DScience%26date%3D2004%26volume%3D304%26spage%3D1497%26epage%3D1500%26doi%3D10.1126%2Fscience.1099314" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sharma, S. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bell, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Settleman, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haber, D. A.</span></span> <span> </span><span class="NLM_article-title">Epidermal growth factor receptor mutations in lung cancer</span>. <i>Nat. Rev. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">169</span>, <span class="refDoi"> DOI: 10.1038/nrc2088</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=10.1038%2Fnrc2088" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=17318210" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=1%3ACAS%3A528%3ADC%252BD2sXitVGrsbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2007&pages=169&author=S.+V.+Sharmaauthor=D.+W.+Bellauthor=J.+Settlemanauthor=D.+A.+Haber&title=Epidermal+growth+factor+receptor+mutations+in+lung+cancer&doi=10.1038%2Fnrc2088"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Epidermal growth factor receptor mutations in lung cancer</span></div><div class="casAuthors">Sharma, Sreenath V.; Bell, Daphne W.; Settleman, Jeffrey; Haber, Daniel A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">169-181</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The development and clin. application of inhibitors that target the epidermal growth factor receptor (EGFR) provide important insights for new lung cancer therapies, as well as for the broader field of targeted cancer therapies.  We review the results of genetic, biochem. and clin. studies focused on somatic mutations of EGFR that are assocd. with the phenomenon of oncogene addiction, describing 'oncogenic shock' as a mechanistic explanation for the apoptosis that follows the acute treatment of susceptible cells with kinase inhibitors.  Understanding the genetic heterogeneity of epithelial tumors and devising strategies to circumvent their rapid acquisition of resistance to targeted kinase inhibitors are essential to the successful use of targeted therapies in common epithelial cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrMKw7Y4FUFM7Vg90H21EOLACvtfcHk0lhlATyAJuexSQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXitVGrsbo%253D&md5=46219ea3ae7b9f0977e4a4b7243fd875</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1038%2Fnrc2088&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc2088%26sid%3Dliteratum%253Aachs%26aulast%3DSharma%26aufirst%3DS.%2BV.%26aulast%3DBell%26aufirst%3DD.%2BW.%26aulast%3DSettleman%26aufirst%3DJ.%26aulast%3DHaber%26aufirst%3DD.%2BA.%26atitle%3DEpidermal%2520growth%2520factor%2520receptor%2520mutations%2520in%2520lung%2520cancer%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2007%26volume%3D7%26spage%3D169%26doi%3D10.1038%2Fnrc2088" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Köhler, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schuler, M.</span></span> <span> </span><span class="NLM_article-title">Afatinib, Erlotinib and Gefitinib in the first-line therapy of egfr mutation-positive lung adenocarcinoma: A Review</span>. <i>Oncol. Res. Treat.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>36</i></span>,  <span class="NLM_fpage">510</span>– <span class="NLM_lpage">518</span>, <span class="refDoi"> DOI: 10.1159/000354627</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=10.1159%2F000354627" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2013&pages=510-518&author=J.+K%C3%B6hlerauthor=M.+Schuler&title=Afatinib%2C+Erlotinib+and+Gefitinib+in+the+first-line+therapy+of+egfr+mutation-positive+lung+adenocarcinoma%3A+A+Review&doi=10.1159%2F000354627"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1159%2F000354627&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1159%252F000354627%26sid%3Dliteratum%253Aachs%26aulast%3DK%25C3%25B6hler%26aufirst%3DJ.%26aulast%3DSchuler%26aufirst%3DM.%26atitle%3DAfatinib%252C%2520Erlotinib%2520and%2520Gefitinib%2520in%2520the%2520first-line%2520therapy%2520of%2520egfr%2520mutation-positive%2520lung%2520adenocarcinoma%253A%2520A%2520Review%26jtitle%3DOncol.%2520Res.%2520Treat.%26date%3D2013%26volume%3D36%26spage%3D510%26epage%3D518%26doi%3D10.1159%2F000354627" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmitz, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeffrey, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wiltzius, J. J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kussie, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferguson, K. M.</span></span> <span> </span><span class="NLM_article-title">Structural basis for inhibition of the epidermal growth factor receptor by cetuximab</span>. <i>Cancer Cell</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">301</span>– <span class="NLM_lpage">311</span>, <span class="refDoi"> DOI: 10.1016/j.ccr.2005.03.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=10.1016%2Fj.ccr.2005.03.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=15837620" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=1%3ACAS%3A528%3ADC%252BD2MXjvFSkurk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2005&pages=301-311&author=S.+Liauthor=K.+R.+Schmitzauthor=P.+D.+Jeffreyauthor=J.+J.+W.+Wiltziusauthor=P.+Kussieauthor=K.+M.+Ferguson&title=Structural+basis+for+inhibition+of+the+epidermal+growth+factor+receptor+by+cetuximab&doi=10.1016%2Fj.ccr.2005.03.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Structural basis for inhibition of the epidermal growth factor receptor by cetuximab</span></div><div class="casAuthors">Li, Shiqing; Schmitz, Karl R.; Jeffrey, Philip D.; Wiltzius, Jed J. W.; Kussie, Paul; Ferguson, Kathryn M.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">301-311</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Recent structural studies of epidermal growth factor receptor (EGFR) family extracellular regions have identified an unexpected mechanism for ligand-induced receptor dimerization that has important implications for activation and inhibition of these receptors.  Here we describe the 2.8 Å resoln. x-ray crystal structure of the antigen binding (Fab) fragment from cetuximab (Erbitux), an inhibitory anti-EGFR antibody, in complex with the sol. extracellular region of EGFR (sEGFR).  The sEGFR is in the characteristic "autoinhibited" or "tethered" inactive configuration.  Cetuximab interacts exclusively with domain III of sEGFR, partially occluding the ligand binding region on this domain and sterically preventing the receptor from adopting the extended conformation required for dimerization.  We suggest that both these effects contribute to potent inhibition of EGFR activation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrfRvCB5ETMNLVg90H21EOLACvtfcHk0ljLUL-AAc-zlQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXjvFSkurk%253D&md5=7d7b01d331206edd65f0d2c5c69785dd</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1016%2Fj.ccr.2005.03.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccr.2005.03.003%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DS.%26aulast%3DSchmitz%26aufirst%3DK.%2BR.%26aulast%3DJeffrey%26aufirst%3DP.%2BD.%26aulast%3DWiltzius%26aufirst%3DJ.%2BJ.%2BW.%26aulast%3DKussie%26aufirst%3DP.%26aulast%3DFerguson%26aufirst%3DK.%2BM.%26atitle%3DStructural%2520basis%2520for%2520inhibition%2520of%2520the%2520epidermal%2520growth%2520factor%2520receptor%2520by%2520cetuximab%26jtitle%3DCancer%2520Cell%26date%3D2005%26volume%3D7%26spage%3D301%26epage%3D311%26doi%3D10.1016%2Fj.ccr.2005.03.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schmiedel, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blaukat, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knöchel, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferguson, K. M.</span></span> <span> </span><span class="NLM_article-title">Matuzumab binding to egfr prevents the conformational rearrangement required for dimerization</span>. <i>Cancer Cell</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">365</span>– <span class="NLM_lpage">373</span>, <span class="refDoi"> DOI: 10.1016/j.ccr.2008.02.019</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=10.1016%2Fj.ccr.2008.02.019" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=18394559" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=1%3ACAS%3A528%3ADC%252BD1cXkslSjt74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2008&pages=365-373&author=J.+Schmiedelauthor=A.+Blaukatauthor=S.+Liauthor=T.+Kn%C3%B6chelauthor=K.+M.+Ferguson&title=Matuzumab+binding+to+egfr+prevents+the+conformational+rearrangement+required+for+dimerization&doi=10.1016%2Fj.ccr.2008.02.019"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Matuzumab binding to EGFR prevents the conformational rearrangement required for dimerization</span></div><div class="casAuthors">Schmiedel, Judith; Blaukat, Andree; Li, Shiqing; Knochel, Thorsten; Ferguson, Kathryn M.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">365-373</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">An increasing no. of therapeutic antibodies targeting tumors that express the epidermal growth factor receptor (EGFR) are in clin. use or late stages of clin. development.  Here the authors investigate the mol. basis for inhibition of EGFR activation by the therapeutic antibody matuzumab (EMD72000).  They describe the x-ray crystal structure of the Fab fragment of matuzumab (Fab72000) in complex with isolated domain III from the extracellular region of EGFR.  Fab72000 interacts with an epitope on EGFR that is distinct from the ligand-binding region on domain III and from the cetuximab/Erbitux epitope.  Matuzumab blocks ligand-induced receptor activation indirectly by sterically preventing the domain rearrangement and local conformational changes that must occur for high-affinity ligand binding and receptor dimerization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr07Gswzbba6LVg90H21EOLACvtfcHk0ljLUL-AAc-zlQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXkslSjt74%253D&md5=97386aedd55b986dde6c0d87abe786e3</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1016%2Fj.ccr.2008.02.019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccr.2008.02.019%26sid%3Dliteratum%253Aachs%26aulast%3DSchmiedel%26aufirst%3DJ.%26aulast%3DBlaukat%26aufirst%3DA.%26aulast%3DLi%26aufirst%3DS.%26aulast%3DKn%25C3%25B6chel%26aufirst%3DT.%26aulast%3DFerguson%26aufirst%3DK.%2BM.%26atitle%3DMatuzumab%2520binding%2520to%2520egfr%2520prevents%2520the%2520conformational%2520rearrangement%2520required%2520for%2520dimerization%26jtitle%3DCancer%2520Cell%26date%3D2008%26volume%3D13%26spage%3D365%26epage%3D373%26doi%3D10.1016%2Fj.ccr.2008.02.019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gu, J.</span></span> <span> </span><span class="NLM_article-title">Identification and characterization of a novel peptide ligand of epidermal growth factor receptor for targeted delivery of therapeutics</span>. <i>FASEB J.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">1978</span>– <span class="NLM_lpage">1985</span>, <span class="refDoi"> DOI: 10.1096/fj.05-4058com</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=10.1096%2Ffj.05-4058com" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=16319141" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtlSju7zK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2005&pages=1978-1985&author=Z.+Liauthor=R.+Zhaoauthor=X.+Wuauthor=Y.+Sunauthor=M.+Yaoauthor=J.+Liauthor=Y.+Xuauthor=J.+Gu&title=Identification+and+characterization+of+a+novel+peptide+ligand+of+epidermal+growth+factor+receptor+for+targeted+delivery+of+therapeutics&doi=10.1096%2Ffj.05-4058com"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Identification and characterization of a novel peptide ligand of epidermal growth factor receptor for targeted delivery of therapeutics</span></div><div class="casAuthors">Li, Zonghai; Zhao, Ruijiao; Wu, Xianghua; Sun, Ye; Yao, Ming; Li, Jinjun; Xu, Yuhong; Gu, Jianren</div><div class="citationInfo"><span class="NLM_cas:title">FASEB Journal</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">1978-1985, 10.1096/fj.05-4058com</span>CODEN:
                <span class="NLM_cas:coden">FAJOEC</span>;
        ISSN:<span class="NLM_cas:issn">0892-6638</span>.
    
            (<span class="NLM_cas:orgname">Federation of American Societies for Experimental Biology</span>)
        </div><div class="casAbstract">Epidermal growth factor receptor (ErbB1, EGFR) is overexpressed in a variety of human cancer cells.  It has been considered as a rational target for drug delivery.  To identify novel ligands with specific binding capabilities to EGFR, we screened a phage display peptide library and found an enriched phage clone encoding the amino acid sequence YHWGYTPQNVI (designated as GE11).  Competitive binding assay and Scatchard anal. revealed that GE11 peptide bound specifically and efficiently to EGFR with a dissocn. const. of ∼22 nM, but with much lower mitogenic activity than with EGF.  We showed that the peptides were internalized preferentially into EGFR highly expressing cells, and they accumulated in EGFR overexpressing tumor xenografts after i.v. delivery in vivo.  In gene delivery studies, GE11-conjugated polyethylenimine (PEI) vectors were less mitogenic, but still quite efficient at transfecting genes into EGFR highly expressing cells and tumor xenografts.  Taken together, GE11 is a potentially safe and efficient targeting moiety for selective drug delivery systems mediated through EGFR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo7wWbZJf-gVrVg90H21EOLACvtfcHk0liQbcqjhececg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtlSju7zK&md5=bb5769496ea0cb1514861ff932b06e30</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1096%2Ffj.05-4058com&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1096%252Ffj.05-4058com%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DZ.%26aulast%3DZhao%26aufirst%3DR.%26aulast%3DWu%26aufirst%3DX.%26aulast%3DSun%26aufirst%3DY.%26aulast%3DYao%26aufirst%3DM.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DXu%26aufirst%3DY.%26aulast%3DGu%26aufirst%3DJ.%26atitle%3DIdentification%2520and%2520characterization%2520of%2520a%2520novel%2520peptide%2520ligand%2520of%2520epidermal%2520growth%2520factor%2520receptor%2520for%2520targeted%2520delivery%2520of%2520therapeutics%26jtitle%3DFASEB%2520J.%26date%3D2005%26volume%3D19%26spage%3D1978%26epage%3D1985%26doi%3D10.1096%2Ffj.05-4058com" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Song, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, L. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Y.</span></span> <span> </span><span class="NLM_article-title">Novel peptide ligand directs liposomes toward EGF-R high-expressing cancer cells in vitro and in vivo</span>. <i>FASEB J.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">1396</span>– <span class="NLM_lpage">1404</span>, <span class="refDoi"> DOI: 10.1096/fj.08-117002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=10.1096%2Ffj.08-117002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=19124558" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=1%3ACAS%3A528%3ADC%252BD1MXlsFSmsrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2009&pages=1396-1404&author=S.+Songauthor=D.+Liuauthor=J.+Pengauthor=H.+Dengauthor=Y.+Guoauthor=L.+X.+Xuauthor=A.+D.+Millerauthor=Y.+Xu&title=Novel+peptide+ligand+directs+liposomes+toward+EGF-R+high-expressing+cancer+cells+in+vitro+and+in+vivo&doi=10.1096%2Ffj.08-117002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Novel peptide ligand directs liposomes toward EGF-R high-expressing cancer cells in vitro and in vivo</span></div><div class="casAuthors">Song, Shuxian; Liu, Dan; Peng, Jinliang; Deng, Hongwei; Guo, Yan; Xu, Lisa X.; Miller, Andrew D.; Xu, Yuhong</div><div class="citationInfo"><span class="NLM_cas:title">FASEB Journal</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1396-1404, 10.1096/fj.08-117002</span>CODEN:
                <span class="NLM_cas:coden">FAJOEC</span>;
        ISSN:<span class="NLM_cas:issn">0892-6638</span>.
    
            (<span class="NLM_cas:orgname">Federation of American Societies for Experimental Biology</span>)
        </div><div class="casAbstract">Epidermal growth factor receptor (EGF-R) is an important target in anticancer therapy.  Here we report how a novel EGF-R peptide ligand (D4: Leu-Ala-Arg-Leu-Leu-Thr) is identified using a computer-aided design approach from a virtual peptide library of putative EGF-R binding peptides by screening against the EGF-R X-ray crystal structure in silico and in vitro.  The selected peptide is conjugated with a polyethylene glycol (PEG) lipid, and the lipid moiety of the peptide-PEG-lipid conjugate is inserted into liposome membranes by a postmodification process.  D4 peptide-conjugated liposomes are found to bind to and enter cells by endocytosis specifically and efficiently in vitro in a process apparently mediated by EGF-R high-expressing cancer cells (H1299).  In vivo, the D4 peptide-conjugated liposomes are found to accumulate in EGF-R-expressing xenograft tumor tissue up to 80 h after i.v. delivery, in marked contrast to controls.  These results demonstrate how structure-based peptide design can be an efficient approach to identify highly novel binding ligands against important receptors.  These data could have important consequences for the development of peptide-directed drug delivery systems with engineered specificities and prolonged times of action.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrVWhU7TdAMrLVg90H21EOLACvtfcHk0liQbcqjhececg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXlsFSmsrs%253D&md5=e5e299248cb417b0fdf16cde5238332b</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1096%2Ffj.08-117002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1096%252Ffj.08-117002%26sid%3Dliteratum%253Aachs%26aulast%3DSong%26aufirst%3DS.%26aulast%3DLiu%26aufirst%3DD.%26aulast%3DPeng%26aufirst%3DJ.%26aulast%3DDeng%26aufirst%3DH.%26aulast%3DGuo%26aufirst%3DY.%26aulast%3DXu%26aufirst%3DL.%2BX.%26aulast%3DMiller%26aufirst%3DA.%2BD.%26aulast%3DXu%26aufirst%3DY.%26atitle%3DNovel%2520peptide%2520ligand%2520directs%2520liposomes%2520toward%2520EGF-R%2520high-expressing%2520cancer%2520cells%2520in%2520vitro%2520and%2520in%2520vivo%26jtitle%3DFASEB%2520J.%26date%3D2009%26volume%3D23%26spage%3D1396%26epage%3D1404%26doi%3D10.1096%2Ffj.08-117002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Teng, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, J.</span></span> <span> </span><span class="NLM_article-title">γ-AApeptides as a new class of peptidomimetics</span>. <i>Chem. - Eur. J.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">5458</span>– <span class="NLM_lpage">5466</span>, <span class="refDoi"> DOI: 10.1002/chem.201504936</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=10.1002%2Fchem.201504936" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=26945679" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=1%3ACAS%3A528%3ADC%252BC28XjslGitbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2016&pages=5458-5466&author=P.+Tengauthor=Y.+Shiauthor=P.+Sangauthor=J.+Cai&title=%CE%B3-AApeptides+as+a+new+class+of+peptidomimetics&doi=10.1002%2Fchem.201504936"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">γ-AApeptides as a New Class of Peptidomimetics</span></div><div class="casAuthors">Teng, Peng; Shi, Yan; Sang, Peng; Cai, Jianfeng</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry - A European Journal</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">5458-5466</span>CODEN:
                <span class="NLM_cas:coden">CEUJED</span>;
        ISSN:<span class="NLM_cas:issn">0947-6539</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A review.  Sequence-specific peptidomimetics are mols. that mimic the structure and function of peptides and proteins.  With new backbones and mol. frameworks, peptidomimetics are of considerable interest in addressing challenges encountered in chem. biol. and biomedical sciences.  Based on the γ-PNA backbone, a new class of peptidomimetics - "γ-AApeptides" were recently developed.  Both linear and cyclic γ-AApeptides can be synthesized with high efficiency.  Compared with α-peptides, γ-AApeptides are resistant to enzymic degrdn., and amendable to diversification with a variety of chem. groups.  Moreover, they could mimic primary and secondary structure, as well as the function of peptides, and show promise in biol. applications, such as the development of new agents combating bacteria, cancer, and Alzheimer's disease.  A few research outcomes of γ-AApeptides are highlighted in this Concept article, and a future perspective is also proposed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrj0LfZU9KRa7Vg90H21EOLACvtfcHk0liQbcqjhececg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XjslGitbk%253D&md5=6f8385337d870a546e1e6b56fd8ef3f1</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1002%2Fchem.201504936&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fchem.201504936%26sid%3Dliteratum%253Aachs%26aulast%3DTeng%26aufirst%3DP.%26aulast%3DShi%26aufirst%3DY.%26aulast%3DSang%26aufirst%3DP.%26aulast%3DCai%26aufirst%3DJ.%26atitle%3D%25CE%25B3-AApeptides%2520as%2520a%2520new%2520class%2520of%2520peptidomimetics%26jtitle%3DChem.%2520-%2520Eur.%2520J.%26date%3D2016%26volume%3D22%26spage%3D5458%26epage%3D5466%26doi%3D10.1002%2Fchem.201504936" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Teng, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">She, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, J.</span></span> <span> </span><span class="NLM_article-title">γ-AApeptides: design, structure, and applications</span>. <i>Acc. Chem. Res.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>49</i></span>,  <span class="NLM_fpage">428</span>– <span class="NLM_lpage">441</span>, <span class="refDoi"> DOI: 10.1021/acs.accounts.5b00492</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.accounts.5b00492" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=1%3ACAS%3A528%3ADC%252BC28XivVWkt7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2016&pages=428-441&author=Y.+Shiauthor=P.+Tengauthor=P.+Sangauthor=F.+Sheauthor=L.+Weiauthor=J.+Cai&title=%CE%B3-AApeptides%3A+design%2C+structure%2C+and+applications&doi=10.1021%2Facs.accounts.5b00492"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">γ-AApeptides: Design, structure, and applications</span></div><div class="casAuthors">Shi, Yan; Teng, Peng; Sang, Peng; She, Fengyu; Wei, Lulu; Cai, Jianfeng</div><div class="citationInfo"><span class="NLM_cas:title">Accounts of Chemical Research</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">428-441</span>CODEN:
                <span class="NLM_cas:coden">ACHRE4</span>;
        ISSN:<span class="NLM_cas:issn">0001-4842</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The development of sequence-specific peptidomimetics has led to a variety of fascinating discoveries in chem. biol.  Many peptidomimetics can mimic primary, secondary, and even tertiary structure of peptides and proteins, and because of their unnatural backbones, they also possess significantly enhanced resistance to enzymic hydrolysis, improved bioavailability, and chemodiversity.  It is known that peptide nucleic acids (PNAs) are peptidic sequences developed for the mimicry of nucleic acids; however, their unique backbone as the mol. scaffold of peptidomimetics to mimic structure and function of bioactive peptides has not been investigated systematically.  As such, we recently developed a new class of peptidomimetics, "γ-AApeptides", based on the chiral γ-PNA backbone.  They are termed γ-AApeptides because they are the oligomers of γ-substituted-N-acylated-N-aminoethyl amino acids.  Similar to other classes of peptidomimetics, γ-AApeptides are also resistant to proteolytic degrdn. and possess the potential to enhance chemodiversity.  Moreover, in our scientific journey on the exploration of this class of peptidomimetics, we have discovered some intriguing structures and functions of γ-AApeptides.  In this account, we summarize the current development and application of γ-AApeptides with biol. potential.  Briefly, both linear and cyclic (either through head-to-tail or head-to-side-chain cyclization) γ-AApeptides with diverse functional groups can be synthesized easily on the solid phase using the synthetic protocol we developed. γ-AApeptides could mimic the primary structure of peptides, as they project the same no. of side chains as peptides of the same lengths.  For instance, they could mimic the Tat peptide to permeate cell membranes and bind to HIV RNA with high specificity and affinity.  Certain γ-AApeptides show similar activity to the RGD peptide and target integrin specifically on the cell surface.  γ-AApeptides with function akin to fMLF peptides are also identified.  More importantly, we found that γ-AApeptides can fold into discrete secondary structures, such as helical and β-turn-like structures.  Therefore, they could be rationally designed for a range of biol. applications.  For instance, γ-AApeptides can mimic host-defense peptides and display potent and broad-spectrum activity toward a panel of drug-resistant bacterial pathogens.  Meanwhile, because of their stability against proteolysis and their chemodiversity, γ-AApeptides are also amenable for combinatorial screening.  We demonstrate that, through combinatorial selection, certain γ-AApeptides are identified to inhibit Aβ40 peptide aggregation, suggesting their potential use as a mol. probe to intervene in Alzheimer's disease.  In addn., a few γ-AApeptides identified from the γ-AApeptide library have been shown to bind to the DNA-binding domain of STAT3 and antagonize STAT3/DNA interactions.  Our studies suggest that, with further studies and exploration on both structures and functions, γ-AApeptides may emerge to be a new class of peptidomimetics that play an important role in chem. biol. and biomedical sciences.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqrk8PLbKcOxrVg90H21EOLACvtfcHk0liQbcqjhececg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XivVWkt7w%253D&md5=414ab3568ccfdcebb5863d876218804c</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1021%2Facs.accounts.5b00492&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.accounts.5b00492%26sid%3Dliteratum%253Aachs%26aulast%3DShi%26aufirst%3DY.%26aulast%3DTeng%26aufirst%3DP.%26aulast%3DSang%26aufirst%3DP.%26aulast%3DShe%26aufirst%3DF.%26aulast%3DWei%26aufirst%3DL.%26aulast%3DCai%26aufirst%3DJ.%26atitle%3D%25CE%25B3-AApeptides%253A%2520design%252C%2520structure%252C%2520and%2520applications%26jtitle%3DAcc.%2520Chem.%2520Res.%26date%3D2016%26volume%3D49%26spage%3D428%26epage%3D441%26doi%3D10.1021%2Facs.accounts.5b00492" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abdulkadir, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ji, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, J.</span></span> <span> </span><span class="NLM_article-title">Inhibition of β-catenin/B cell lymphoma 9 protein–protein interaction using α-helix–mimicking sulfono-γ-AApeptide inhibitors</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>116</i></span>,  <span class="NLM_fpage">10757</span>– <span class="NLM_lpage">10762</span>, <span class="refDoi"> DOI: 10.1073/pnas.1819663116</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=10.1073%2Fpnas.1819663116" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=31088961" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhtVChsr%252FE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=116&publication_year=2019&pages=10757-10762&author=P.+Sangauthor=M.+Zhangauthor=Y.+Shiauthor=C.+Liauthor=S.+Abdulkadirauthor=Q.+Liauthor=H.+Jiauthor=J.+Cai&title=Inhibition+of+%CE%B2-catenin%2FB+cell+lymphoma+9+protein%E2%80%93protein+interaction+using+%CE%B1-helix%E2%80%93mimicking+sulfono-%CE%B3-AApeptide+inhibitors&doi=10.1073%2Fpnas.1819663116"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of β-catenin/B cell lymphoma 9 protein-protein interaction using α-helix-mimicking sulfono-γ-AApeptide inhibitors</span></div><div class="casAuthors">Sang, Peng; Zhang, Min; Shi, Yan; Li, Chunpu; Abdulkadir, Sami; Li, Qi; Ji, Haitao; Cai, Jianfeng</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">116</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">10757-10762</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The rational design of α-helix-mimicking peptidomimetics provides a streamlined approach to discover potent inhibitors for protein-protein interactions (PPIs).  However, designing cell-penetrating long peptidomimetic scaffolds equipped with various functional groups necessary for interacting with large protein-binding interfaces remains challenging.  This is particularly true for targeting β-catenin/BCL9 PPIs.  Here the authors designed a series of unprecedented helical sulfono-γ-AApeptides that mimic the binding mode of the α-helical HD2 domain of B Cell Lymphoma 9 (BCL9).  The studies show that sulfono-γ-AApeptides can structurally and functionally mimic the α-helical domain of BCL9 and selectively disrupt β-catenin/BCL9 PPIs with even higher potency.  More intriguingly, these sulfono-γ-AApeptides can enter cancer cells, bind with β-catenin and disrupt β-catenin/BCL9 PPIs, and exhibit excellent cellular activity, which is much more potent than the BCL9 peptide.  Furthermore, the enzymic stability studies demonstrate the remarkable stability of the helical sulfono-γ-AApeptides, with no degrdn. in the presence of pronase for 24 h, augmenting their biol. potential.  This work represents not only an example of helical sulfono-γ-AApeptides that mimic α-helix and disrupt protein-protein interactions, but also an excellent example of potent, selective, and cell-permeable unnatural foldameric peptidomimetics that disrupt the β-catenin/BCL9 PPI.  The design of helical sulfono-γ-AApeptides may lead to a new strategy to modulate a myriad of protein-protein interactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo_pKlgpb-_U7Vg90H21EOLACvtfcHk0lghmCqzgEbC1g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhtVChsr%252FE&md5=145172aa8475bfe4a20b97d6b88d12dd</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1819663116&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1819663116%26sid%3Dliteratum%253Aachs%26aulast%3DSang%26aufirst%3DP.%26aulast%3DZhang%26aufirst%3DM.%26aulast%3DShi%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DC.%26aulast%3DAbdulkadir%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DQ.%26aulast%3DJi%26aufirst%3DH.%26aulast%3DCai%26aufirst%3DJ.%26atitle%3DInhibition%2520of%2520%25CE%25B2-catenin%252FB%2520cell%2520lymphoma%25209%2520protein%25E2%2580%2593protein%2520interaction%2520using%2520%25CE%25B1-helix%25E2%2580%2593mimicking%2520sulfono-%25CE%25B3-AApeptide%2520inhibitors%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2019%26volume%3D116%26spage%3D10757%26epage%3D10762%26doi%3D10.1073%2Fpnas.1819663116" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Teng, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cerrato, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">She, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Odom, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ming, L.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van der Vaart, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wojtas, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, J.</span></span> <span> </span><span class="NLM_article-title">Right-handed helical foldamers consisting of de novo d-aapeptides</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>139</i></span>,  <span class="NLM_fpage">7363</span>– <span class="NLM_lpage">7369</span>, <span class="refDoi"> DOI: 10.1021/jacs.7b03007</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.7b03007" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=1%3ACAS%3A528%3ADC%252BC2sXnt1Chu7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=139&publication_year=2017&pages=7363-7369&author=P.+Tengauthor=N.+Maauthor=D.+C.+Cerratoauthor=F.+Sheauthor=T.+Odomauthor=X.+Wangauthor=L.-J.+Mingauthor=A.+van+der+Vaartauthor=L.+Wojtasauthor=H.+Xuauthor=J.+Cai&title=Right-handed+helical+foldamers+consisting+of+de+novo+d-aapeptides&doi=10.1021%2Fjacs.7b03007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Right-handed helical foldamers consisting of de novo D-AA-peptides</span></div><div class="casAuthors">Teng, Peng; Ma, Ning; Cerrato, Darrell Cole; She, Fengyu; Odom, Timothy; Wang, Xiang; Ming, Li-June; van der Vaart, Arjan; Wojtas, Lukasz; Xu, Hai; Cai, Jianfeng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">139</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">7363-7369</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">New types of foldamer scaffolds are formidably challenging to design and synthesize, yet highly desirable as structural mimics of peptides/proteins with a wide repertoire of functions.  In particular, the development of peptidomimetic helical foldamers holds promise for new biomaterials, catalysts, and drug mols.  Unnatural L-sulfono-γ-AApeptides were recently developed and shown to have potential applications in both biomedical and material sciences.  However, D-sulfono-γ-AApeptides, the enantiomers of L-sulfono-γ-AApeptides, have never been studied due to the lack of high-resoln. three-dimensional structures to guide structure-based design.  Herein, we report the first synthesis and X-ray crystal structures of a series of 2:1 L-amino acid/D-sulfono-γ-AApeptide hybrid foldamers, and elucidate their folded conformation at the at. level.  Single-crystal X-ray crystallog. indicates that this class of oligomers folds into well-defined right-handed helixes with unique helical parameters.  The helical structures were consistent with data obtained from soln. 2D NMR, CD studies, and mol. dynamics simulations.  Our findings are expected to inspire the structure-based design of this type of unique folding biopolymers for biomaterials and biomedical applications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqaGg_OJWsqYbVg90H21EOLACvtfcHk0lghmCqzgEbC1g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXnt1Chu7w%253D&md5=fef3f80500953f027687001f3e67f14b</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1021%2Fjacs.7b03007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.7b03007%26sid%3Dliteratum%253Aachs%26aulast%3DTeng%26aufirst%3DP.%26aulast%3DMa%26aufirst%3DN.%26aulast%3DCerrato%26aufirst%3DD.%2BC.%26aulast%3DShe%26aufirst%3DF.%26aulast%3DOdom%26aufirst%3DT.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DMing%26aufirst%3DL.-J.%26aulast%3Dvan%2Bder%2BVaart%26aufirst%3DA.%26aulast%3DWojtas%26aufirst%3DL.%26aulast%3DXu%26aufirst%3DH.%26aulast%3DCai%26aufirst%3DJ.%26atitle%3DRight-handed%2520helical%2520foldamers%2520consisting%2520of%2520de%2520novo%2520d-aapeptides%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2017%26volume%3D139%26spage%3D7363%26epage%3D7369%26doi%3D10.1021%2Fjacs.7b03007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Teng, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Niu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">She, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verma, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wojtas, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van der Vaart, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, J.</span></span> <span> </span><span class="NLM_article-title">Hydrogen-bonding-driven 3d supramolecular assembly of peptidomimetic zipper</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>140</i></span>,  <span class="NLM_fpage">5661</span>– <span class="NLM_lpage">5665</span>, <span class="refDoi"> DOI: 10.1021/jacs.7b11997</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.7b11997" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=1%3ACAS%3A528%3ADC%252BC1cXmsVahsLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=140&publication_year=2018&pages=5661-5665&author=P.+Tengauthor=Z.+Niuauthor=F.+Sheauthor=M.+Zhouauthor=P.+Sangauthor=G.+M.+Grayauthor=G.+Vermaauthor=L.+Wojtasauthor=A.+van+der+Vaartauthor=S.+Maauthor=J.+Cai&title=Hydrogen-bonding-driven+3d+supramolecular+assembly+of+peptidomimetic+zipper&doi=10.1021%2Fjacs.7b11997"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Hydrogen-Bonding-Driven 3D Supramolecular Assembly of Peptidomimetic Zipper</span></div><div class="casAuthors">Teng, Peng; Niu, Zheng; She, Fengyu; Zhou, Mi; Sang, Peng; Gray, Geoffrey M.; Verma, Gaurav; Wojtas, Lukasz; van der Vaart, Arjan; Ma, Shengqian; Cai, Jianfeng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">140</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">5661-5665</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Hydrogen-bonding-driven three-dimensional (3D) assembly of a peptidomimetic zipper has been established for the first time by using an α/AApeptide zipper that assembles into a de novo lattice arrangement through two layers of hydrogen-bonded linker-directed interactions.  Via a covalently bridged 1D 413-helix, drastic enhancement in stability has been achieved in the formed 3D cryst. supramol. architecture as evidenced by gas-sorption studies.  As the first example of an unnatural peptidic zipper, the dimensional augmentation of the zipper differs from metal-coordinated strategies, and may have general implications for the prepn. of peptidic functional materials for a variety of future applications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq6NOPVSQVnN7Vg90H21EOLACvtfcHk0lghmCqzgEbC1g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXmsVahsLY%253D&md5=75ec49a44f96107f9a2472698475157d</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Fjacs.7b11997&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.7b11997%26sid%3Dliteratum%253Aachs%26aulast%3DTeng%26aufirst%3DP.%26aulast%3DNiu%26aufirst%3DZ.%26aulast%3DShe%26aufirst%3DF.%26aulast%3DZhou%26aufirst%3DM.%26aulast%3DSang%26aufirst%3DP.%26aulast%3DGray%26aufirst%3DG.%2BM.%26aulast%3DVerma%26aufirst%3DG.%26aulast%3DWojtas%26aufirst%3DL.%26aulast%3Dvan%2Bder%2BVaart%26aufirst%3DA.%26aulast%3DMa%26aufirst%3DS.%26aulast%3DCai%26aufirst%3DJ.%26atitle%3DHydrogen-bonding-driven%25203d%2520supramolecular%2520assembly%2520of%2520peptidomimetic%2520zipper%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2018%26volume%3D140%26spage%3D5661%26epage%3D5665%26doi%3D10.1021%2Fjacs.7b11997" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">She, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Teng, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peguero-Tejada, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Odom, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gjonaj, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wojtas, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van der Vaart, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, J.</span></span> <span> </span><span class="NLM_article-title">De novo left-handed synthetic peptidomimetic foldamers</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">9916</span>– <span class="NLM_lpage">9920</span>, <span class="refDoi"> DOI: 10.1002/anie.201805184</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=10.1002%2Fanie.201805184" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=1%3ACAS%3A528%3ADC%252BC1cXht1yjt7bL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2018&pages=9916-9920&author=F.+Sheauthor=P.+Tengauthor=A.+Peguero-Tejadaauthor=M.+Wangauthor=N.+Maauthor=T.+Odomauthor=M.+Zhouauthor=E.+Gjonajauthor=L.+Wojtasauthor=A.+van+der+Vaartauthor=J.+Cai&title=De+novo+left-handed+synthetic+peptidomimetic+foldamers&doi=10.1002%2Fanie.201805184"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">De novo left-handed synthetic peptidomimetic foldamers</span></div><div class="casAuthors">She, Fengyu; Teng, Peng; Peguero-Tejada, Alfredo; Wang, Minghui; Ma, Ning; Odom, Timothy; Zhou, Mi; Gjonaj, Erald; Wojtas, Lukasz; van der Vaart, Arjan; Cai, Jianfeng</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">31</span>),
    <span class="NLM_cas:pages">9916-9920</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">The development of peptidomimetic helical foldamers with a wide repertoire of functions is of significant interest.  Herein, we report the X-ray crystal structures of a series of homogeneous L-sulfono-γ-AA foldamers and elucidate their folding conformation at the at. level.  Single-crystal X-ray crystallog. revealed that this class of oligomers fold into unprecedented dragon-boat-shaped and unexpected left-handed helixes, which are stabilized by the 14-hydrogen-bonding pattern present in all sequences.  These L-sulfono-γ-AA-peptides have a helical pitch of 5.1 Å and exactly four side chains per turn, and the side chains lie perfectly on top of each other along the helical axis.  2D NMR spectroscopy, computational simulations, and CD studies support the folding conformation in soln.  Our results provide a structural basis at the at. level for the design of novel biomimetics with a precise arrangement of functional groups in three dimensions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGru2vbXzMss1rVg90H21EOLACvtfcHk0lhkIQNoIs5EUw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXht1yjt7bL&md5=c45cc7e77dfa2887f5fec63d8e3dbe7e</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1002%2Fanie.201805184&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201805184%26sid%3Dliteratum%253Aachs%26aulast%3DShe%26aufirst%3DF.%26aulast%3DTeng%26aufirst%3DP.%26aulast%3DPeguero-Tejada%26aufirst%3DA.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DMa%26aufirst%3DN.%26aulast%3DOdom%26aufirst%3DT.%26aulast%3DZhou%26aufirst%3DM.%26aulast%3DGjonaj%26aufirst%3DE.%26aulast%3DWojtas%26aufirst%3DL.%26aulast%3Dvan%2Bder%2BVaart%26aufirst%3DA.%26aulast%3DCai%26aufirst%3DJ.%26atitle%3DDe%2520novo%2520left-handed%2520synthetic%2520peptidomimetic%2520foldamers%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2018%26volume%3D57%26spage%3D9916%26epage%3D9920%26doi%3D10.1002%2Fanie.201805184" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Teng, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wojtas, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van der Vaart, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, J.</span></span> <span> </span><span class="NLM_article-title">Orthogonal halogen-bonding-driven 3d supramolecular assembly of right-handed synthetic helical peptides</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">7778</span>– <span class="NLM_lpage">7782</span>, <span class="refDoi"> DOI: 10.1002/anie.201903259</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=10.1002%2Fanie.201903259" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=1%3ACAS%3A528%3ADC%252BC1MXos1Sjsbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2019&pages=7778-7782&author=P.+Tengauthor=G.+M.+Grayauthor=M.+Zhengauthor=S.+Singhauthor=X.+Liauthor=L.+Wojtasauthor=A.+van+der+Vaartauthor=J.+Cai&title=Orthogonal+halogen-bonding-driven+3d+supramolecular+assembly+of+right-handed+synthetic+helical+peptides&doi=10.1002%2Fanie.201903259"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Orthogonal Halogen-Bonding-Driven 3D Supramolecular Assembly of Right-Handed Synthetic Helical Peptides</span></div><div class="casAuthors">Teng, Peng; Gray, Geoffrey M.; Zheng, Mengmeng; Singh, Sylvia; Li, Xiaopeng; Wojtas, Lukasz; van der Vaart, Arjan; Cai, Jianfeng</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">7778-7782</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Peptide-mediated self-assembly is a prevalent method for creating highly ordered supramol. architectures.  Herein, we report the first example of orthogonal C-X···X-C/C-X···π halogen bonding and hydrogen bonding driven cryst. architectures based on synthetic helical peptides bearing hybrids of L-sulfono-γ-AApeptides and natural amino acids.  The combination of halogen bonding, intra-/intermol. hydrogen bonding, and intermol. hydrophobic interactions enabled novel 3D supramol. assembly.  The orthogonal halogen bonding in the supramol. architecture exerts a novel mechanism for the self-assembly of synthetic peptide foldamers and gives new insights into mol. recognition, supramol. design, and rational design of biomimetic structures.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGphs8_rPqx-zrVg90H21EOLACvtfcHk0lhkIQNoIs5EUw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXos1Sjsbc%253D&md5=c61d38ec3f3a4e3d79d716a1c88a7ec3</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1002%2Fanie.201903259&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201903259%26sid%3Dliteratum%253Aachs%26aulast%3DTeng%26aufirst%3DP.%26aulast%3DGray%26aufirst%3DG.%2BM.%26aulast%3DZheng%26aufirst%3DM.%26aulast%3DSingh%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DWojtas%26aufirst%3DL.%26aulast%3Dvan%2Bder%2BVaart%26aufirst%3DA.%26aulast%3DCai%26aufirst%3DJ.%26atitle%3DOrthogonal%2520halogen-bonding-driven%25203d%2520supramolecular%2520assembly%2520of%2520right-handed%2520synthetic%2520helical%2520peptides%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2019%26volume%3D58%26spage%3D7778%26epage%3D7782%26doi%3D10.1002%2Fanie.201903259" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Teng, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qiao, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sebti, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, J.</span></span> <span> </span><span class="NLM_article-title">Identification of novel inhibitors that disrupt STAT3–DNA interaction from a γ-aapeptide oboc combinatorial library</span>. <i>Chem. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">8739</span>– <span class="NLM_lpage">8742</span>, <span class="refDoi"> DOI: 10.1039/C4CC03909B</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=10.1039%2FC4CC03909B" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=24964402" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtVGnsrjM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2014&pages=8739-8742&author=P.+Tengauthor=X.+Zhangauthor=H.+Wuauthor=Q.+Qiaoauthor=S.+M.+Sebtiauthor=J.+Cai&title=Identification+of+novel+inhibitors+that+disrupt+STAT3%E2%80%93DNA+interaction+from+a+%CE%B3-aapeptide+oboc+combinatorial+library&doi=10.1039%2FC4CC03909B"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of novel inhibitors that disrupt STAT3-DNA interaction from a γ-AApeptide OBOC combinatorial library</span></div><div class="casAuthors">Teng, Peng; Zhang, Xiaolei; Wu, Haifan; Qiao, Qiao; Sebti, Said M.; Cai, Jianfeng</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Communications (Cambridge, United Kingdom)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">63</span>),
    <span class="NLM_cas:pages">8739-8742</span>CODEN:
                <span class="NLM_cas:coden">CHCOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-7345</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">From a γ-AApeptide-based one-bead-one-compd. (OBOC) combinatorial library, the authors identified γ-AApeptides that can selectively inhibit STAT3-DNA interaction and suppress the expression levels of STAT3 target genes in intact cells.  The authors' results demonstrate that in addn. to the SH2 domain, the DNA binding domain of STAT3 is targetable for the development of a new generation of anti-cancer therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqVN0NpRV6tXrVg90H21EOLACvtfcHk0lhkIQNoIs5EUw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtVGnsrjM&md5=41c994aa9a985107fc1402b58d38223a</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1039%2FC4CC03909B&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC4CC03909B%26sid%3Dliteratum%253Aachs%26aulast%3DTeng%26aufirst%3DP.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DWu%26aufirst%3DH.%26aulast%3DQiao%26aufirst%3DQ.%26aulast%3DSebti%26aufirst%3DS.%2BM.%26aulast%3DCai%26aufirst%3DJ.%26atitle%3DIdentification%2520of%2520novel%2520inhibitors%2520that%2520disrupt%2520STAT3%25E2%2580%2593DNA%2520interaction%2520from%2520a%2520%25CE%25B3-aapeptide%2520oboc%2520combinatorial%2520library%26jtitle%3DChem.%2520Commun.%26date%3D2014%26volume%3D50%26spage%3D8739%26epage%3D8742%26doi%3D10.1039%2FC4CC03909B" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Challa, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">She, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nimmagadda, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Teng, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Odom, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van der
Vaart, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, J.</span></span> <span> </span><span class="NLM_article-title">One-bead–two-compound thioether bridged macrocyclic γ-aapeptide screening library against EphA2</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">9290</span>– <span class="NLM_lpage">9298</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b01280</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b01280" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhslKgtLnO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=9290-9298&author=Y.+Shiauthor=S.+Challaauthor=P.+Sangauthor=F.+Sheauthor=C.+Liauthor=G.+M.+Grayauthor=A.+Nimmagaddaauthor=P.+Tengauthor=T.+Odomauthor=Y.+Wangauthor=A.+van+der%0AVaartauthor=Q.+Liauthor=J.+Cai&title=One-bead%E2%80%93two-compound+thioether+bridged+macrocyclic+%CE%B3-aapeptide+screening+library+against+EphA2&doi=10.1021%2Facs.jmedchem.7b01280"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">One-bead-two-compound thioether bridged macrocyclic γ-AApeptide screening library against EphA2</span></div><div class="casAuthors">Shi, Yan; Challa, Sridevi; Sang, Peng; She, Fengyu; Li, Chunpu; Gray, Geoffrey M.; Nimmagadda, Alekhya; Teng, Peng; Odom, Timothy; Wang, Yan; van der Vaart, Arjan; Li, Qi; Cai, Jianfeng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">9290-9298</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Identification of mol. ligands that recognize peptides or proteins is significant but poses a fundamental challenge in chem. biol. and biomedical sciences.  Development of cyclic peptidomimetic library is scarce, and thus discovery of cyclic peptidomimetic ligands for protein targets is rare.  Herein we report the unprecedented one-bead-two-compd. (OBTC) combinatorial library based on a novel class of the macrocyclic peptidomimetics γ-AApeptides.  In the library, we utilized the coding peptide tags synthesized with Dde-protected α-amino acids, which were orthogonal to solid phase synthesis of γ-AApeptides.  Employing the thioether linkage, the desired macrocyclic γ-AApeptides were found to be effective for ligand identification.  Screening the library against the receptor tyrosine kinase EphA2 led to the discovery of one lead compd. that tightly bound to EphA2 (Kd = 81 nM) and potently antagonized EphA2-mediated signaling.  This new approach of macrocyclic peptidomimetic library may lead to a novel platform for biomacromol. surface recognition and function modulation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrE8_skSAKNvbVg90H21EOLACvtfcHk0liW06e9do9epQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhslKgtLnO&md5=0c5c86232aa6afd699fd6de5128a1ecf</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b01280&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b01280%26sid%3Dliteratum%253Aachs%26aulast%3DShi%26aufirst%3DY.%26aulast%3DChalla%26aufirst%3DS.%26aulast%3DSang%26aufirst%3DP.%26aulast%3DShe%26aufirst%3DF.%26aulast%3DLi%26aufirst%3DC.%26aulast%3DGray%26aufirst%3DG.%2BM.%26aulast%3DNimmagadda%26aufirst%3DA.%26aulast%3DTeng%26aufirst%3DP.%26aulast%3DOdom%26aufirst%3DT.%26aulast%3DWang%26aufirst%3DY.%26aulast%3Dvan%2Bder%2BVaart%26aufirst%3DA.%26aulast%3DLi%26aufirst%3DQ.%26aulast%3DCai%26aufirst%3DJ.%26atitle%3DOne-bead%25E2%2580%2593two-compound%2520thioether%2520bridged%2520macrocyclic%2520%25CE%25B3-aapeptide%2520screening%2520library%2520against%2520EphA2%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D9290%26epage%3D9298%26doi%3D10.1021%2Facs.jmedchem.7b01280" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yan, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhattarai, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, F.-S.</span></span> <span> </span><span class="NLM_article-title">Cyclic Peptidomimetics as Inhibitor for miR-155 Biogenesis</span>. <i>Mol. Pharmaceutics</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">914</span>– <span class="NLM_lpage">920</span>, <span class="refDoi"> DOI: 10.1021/acs.molpharmaceut.8b01247</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.molpharmaceut.8b01247" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhtlSmsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2019&pages=914-920&author=H.+Yanauthor=M.+Zhouauthor=U.+Bhattaraiauthor=Y.+Songauthor=M.+Zhengauthor=J.+Caiauthor=F.-S.+Liang&title=Cyclic+Peptidomimetics+as+Inhibitor+for+miR-155+Biogenesis&doi=10.1021%2Facs.molpharmaceut.8b01247"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Cyclic Peptidomimetics as Inhibitor for miR-155 Biogenesis</span></div><div class="casAuthors">Yan, Hao; Zhou, Mi; Bhattarai, Umesh; Song, Yabin; Zheng, Mengmeng; Cai, Jianfeng; Liang, Fu-Sen</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Pharmaceutics</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">914-920</span>CODEN:
                <span class="NLM_cas:coden">MPOHBP</span>;
        ISSN:<span class="NLM_cas:issn">1543-8384</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">MiR-155 plays key promoting roles in several cancers and emerges as an important anticancer therapeutic target.  However, the discovery of small mols. that target RNAs is challenging.  Peptidomimetics have been shown to be a rich source for discovering novel ligands to regulate cellular proteins.  However, the potential of using peptidomimetics for RNA targeting is relatively unexplored.  To this end, we designed and synthesized members of a novel 320 000 compd. macrocyclic peptidomimetic library.  An affinity-based screening protocol led to the identification of a pre-miR-155 binder that inhibits oncogenic miR-155 maturation in vitro and in cell and induces cancer cell apoptosis.  The results of this investigation demonstrate that macrocyclic peptidomimetics could serve as a new scaffold for RNA targeting.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqa1182LkJrALVg90H21EOLACvtfcHk0liW06e9do9epQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhtlSmsg%253D%253D&md5=0ac2dec99b95d9efbbaef045bf9e9e4f</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Facs.molpharmaceut.8b01247&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.molpharmaceut.8b01247%26sid%3Dliteratum%253Aachs%26aulast%3DYan%26aufirst%3DH.%26aulast%3DZhou%26aufirst%3DM.%26aulast%3DBhattarai%26aufirst%3DU.%26aulast%3DSong%26aufirst%3DY.%26aulast%3DZheng%26aufirst%3DM.%26aulast%3DCai%26aufirst%3DJ.%26aulast%3DLiang%26aufirst%3DF.-S.%26atitle%3DCyclic%2520Peptidomimetics%2520as%2520Inhibitor%2520for%2520miR-155%2520Biogenesis%26jtitle%3DMol.%2520Pharmaceutics%26date%3D2019%26volume%3D16%26spage%3D914%26epage%3D920%26doi%3D10.1021%2Facs.molpharmaceut.8b01247" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parag, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lui, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murr, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, N. A.</span></span> <span> </span><span class="NLM_article-title">Stabilization of lncrna gas5 by a small molecule and its implications in diabetic adipocytes</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">319</span>– <span class="NLM_lpage">330</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2018.11.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=10.1016%2Fj.chembiol.2018.11.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=30661991" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhsVartb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2019&pages=319-330&author=Y.+Shiauthor=S.+Paragauthor=R.+Patelauthor=A.+Luiauthor=M.+Murrauthor=J.+Caiauthor=N.+A.+Patel&title=Stabilization+of+lncrna+gas5+by+a+small+molecule+and+its+implications+in+diabetic+adipocytes&doi=10.1016%2Fj.chembiol.2018.11.012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Stabilization of lncRNA GAS5 by a Small Molecule and Its Implications in Diabetic Adipocytes</span></div><div class="casAuthors">Shi, Yan; Parag, Sajan; Patel, Rekha; Lui, Ashley; Murr, Michel; Cai, Jianfeng; Patel, Niketa A.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">319-330.e6</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Long noncoding RNA (lncRNA) are regulatory RNAs >200 nt.  We previously showed that lncRNA GAS5 decreases significantly in serum of type 2 diabetes mellitus (T2DM) patients.  Hence, we sought to decipher the mol. mechanisms underlying the role of GAS5 in T2DM in adipose tissue.  Using CHIP-RIP, we demonstrate that GAS5 binds to promoter of insulin receptor to regulate its expression, and its depletion inhibits glucose uptake and insulin signaling.  Toward stabilizing GAS5 levels in T2DM, we incorporated a strategy to limit the degrdn. of GAS5 by blocking the interaction of GAS5 and UPF1 with a small mol. identified using OBTC screening strategy.  NP-C86 binds to GAS5 with high affinity, and increases GAS5 levels and glucose uptake in diabetic patient adipocytes.  As a broader impact, NP-C86 may be used as a mol. probe to investigate the intricacies of GAS5 in relevant biol. systems as it offers specificity, efficient cellular uptake and is non-cytotoxic.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpk-KPkGlzXi7Vg90H21EOLACvtfcHk0liW06e9do9epQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhsVartb8%253D&md5=5f7d6288486103bfe129bdd1a7d9e05d</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2018.11.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2018.11.012%26sid%3Dliteratum%253Aachs%26aulast%3DShi%26aufirst%3DY.%26aulast%3DParag%26aufirst%3DS.%26aulast%3DPatel%26aufirst%3DR.%26aulast%3DLui%26aufirst%3DA.%26aulast%3DMurr%26aufirst%3DM.%26aulast%3DCai%26aufirst%3DJ.%26aulast%3DPatel%26aufirst%3DN.%2BA.%26atitle%3DStabilization%2520of%2520lncrna%2520gas5%2520by%2520a%2520small%2520molecule%2520and%2520its%2520implications%2520in%2520diabetic%2520adipocytes%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2019%26volume%3D26%26spage%3D319%26epage%3D330%26doi%3D10.1016%2Fj.chembiol.2018.11.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gureasko, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cole, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuriyan, J.</span></span> <span> </span><span class="NLM_article-title">An Allosteric Mechanism for activation of the kinase domain of epidermal growth factor receptor</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>125</i></span>,  <span class="NLM_fpage">1137</span>– <span class="NLM_lpage">1149</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2006.05.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=10.1016%2Fj.cell.2006.05.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=16777603" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=1%3ACAS%3A528%3ADC%252BD28XmsVOms7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=125&publication_year=2006&pages=1137-1149&author=X.+Zhangauthor=J.+Gureaskoauthor=K.+Shenauthor=P.+A.+Coleauthor=J.+Kuriyan&title=An+Allosteric+Mechanism+for+activation+of+the+kinase+domain+of+epidermal+growth+factor+receptor&doi=10.1016%2Fj.cell.2006.05.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">An allosteric mechanism for activation of the kinase domain of epidermal growth factor receptor</span></div><div class="casAuthors">Zhang, Xuewu; Gureasko, Jodi; Shen, Kui; Cole, Philip A.; Kuriyan, John</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">125</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1137-1149</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The mechanism by which the epidermal growth factor receptor (EGFR) is activated upon dimerization has eluded definition.  We find that the EGFR kinase domain can be activated by increasing its local concn. or by mutating a leucine (L834R) in the activation loop, the phosphorylation of which is not required for activation.  This suggests that the kinase domain is intrinsically auto-inhibited, and an intermol. interaction promotes its activation.  Using further mutational anal. and crystallog. we demonstrate that the auto-inhibited conformation of the EGFR kinase domain resembles that of Src and cyclin-dependent kinases (CDKs).  EGFR activation results from the formation of an asym. dimer in which the C-terminal lobe of one kinase domain plays a role analogous to that of cyclin in activated CDK/cyclin complexes.  The CDK/cyclin-like complex formed by two kinase domains thus explains the activation of EGFR-family receptors by homo- or heterodimerization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoQsRAzLF-wK7Vg90H21EOLACvtfcHk0lgvCZccmnKEMQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XmsVOms7g%253D&md5=f7b6795851425346f588cbd682125c93</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2006.05.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2006.05.013%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DGureasko%26aufirst%3DJ.%26aulast%3DShen%26aufirst%3DK.%26aulast%3DCole%26aufirst%3DP.%2BA.%26aulast%3DKuriyan%26aufirst%3DJ.%26atitle%3DAn%2520Allosteric%2520Mechanism%2520for%2520activation%2520of%2520the%2520kinase%2520domain%2520of%2520epidermal%2520growth%2520factor%2520receptor%26jtitle%3DCell%26date%3D2006%26volume%3D125%26spage%3D1137%26epage%3D1149%26doi%3D10.1016%2Fj.cell.2006.05.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ogiso, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ishitani, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nureki, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fukai, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamanaka, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, J.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saito, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sakamoto, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Inoue, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shirouzu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yokoyama, S.</span></span> <span> </span><span class="NLM_article-title">Crystal structure of the complex of human epidermal growth factor and receptor extracellular domains</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>110</i></span>,  <span class="NLM_fpage">775</span>– <span class="NLM_lpage">787</span>, <span class="refDoi"> DOI: 10.1016/S0092-8674(02)00963-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=10.1016%2FS0092-8674%2802%2900963-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=12297050" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=1%3ACAS%3A528%3ADC%252BD38XnsFKisL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=110&publication_year=2002&pages=775-787&author=H.+Ogisoauthor=R.+Ishitaniauthor=O.+Nurekiauthor=S.+Fukaiauthor=M.+Yamanakaauthor=J.-H.+Kimauthor=K.+Saitoauthor=A.+Sakamotoauthor=M.+Inoueauthor=M.+Shirouzuauthor=S.+Yokoyama&title=Crystal+structure+of+the+complex+of+human+epidermal+growth+factor+and+receptor+extracellular+domains&doi=10.1016%2FS0092-8674%2802%2900963-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal structure of the complex of human epidermal growth factor and receptor extracellular domains</span></div><div class="casAuthors">Ogiso, Hideo; Ishitani, Ryuichiro; Nureki, Osamu; Fukai, Shuya; Yamanaka, Mari; Kim, Jae-Hoon; Saito, Kazuki; Sakamoto, Ayako; Inoue, Mio; Shirouzu, Mikako; Yokoyama, Shigeyuki</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">110</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">775-787</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Epidermal growth factor (EGF) regulates cell proliferation and differentiation by binding to the EGF receptor (EGFR) extracellular region, comprising domains I-IV, with the resultant dimerization of the receptor tyrosine kinase.  In this study, the crystal structure of a 2:2 complex of human EGF and the EGFR extracellular region has been detd. at 3.3 Å resoln.  EGFR domains I-III are arranged in a C shape, and EGF is docked between domains I and III.  The 1:1 EGF•EGFR complex dimerizes through a direct receptor•receptor interaction, in which a protruding β-hairpin arm of each domain II holds the body of the other.  The unique "receptor-mediated dimerization" was verified by EGFR mutagenesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqILejwMfkBaLVg90H21EOLACvtfcHk0lgvCZccmnKEMQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XnsFKisL8%253D&md5=f909ae76aad06fb0a8704ee101e76310</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1016%2FS0092-8674%2802%2900963-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0092-8674%252802%252900963-7%26sid%3Dliteratum%253Aachs%26aulast%3DOgiso%26aufirst%3DH.%26aulast%3DIshitani%26aufirst%3DR.%26aulast%3DNureki%26aufirst%3DO.%26aulast%3DFukai%26aufirst%3DS.%26aulast%3DYamanaka%26aufirst%3DM.%26aulast%3DKim%26aufirst%3DJ.-H.%26aulast%3DSaito%26aufirst%3DK.%26aulast%3DSakamoto%26aufirst%3DA.%26aulast%3DInoue%26aufirst%3DM.%26aulast%3DShirouzu%26aufirst%3DM.%26aulast%3DYokoyama%26aufirst%3DS.%26atitle%3DCrystal%2520structure%2520of%2520the%2520complex%2520of%2520human%2520epidermal%2520growth%2520factor%2520and%2520receptor%2520extracellular%2520domains%26jtitle%3DCell%26date%3D2002%26volume%3D110%26spage%3D775%26epage%3D787%26doi%3D10.1016%2FS0092-8674%2802%2900963-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sanders, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wampole, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thakur, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wickstrom, E.</span></span> <span> </span><span class="NLM_article-title">Molecular determinants of epidermal growth factor binding: a molecular dynamics study</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>8</i></span>, <span class="NLM_elocation-id">e54136</span> <span class="refDoi"> DOI: 10.1371/journal.pone.0054136</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=10.1371%2Fjournal.pone.0054136" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=23382875" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=1%3ACAS%3A528%3ADC%252BC3sXit1ChsrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&author=J.+M.+Sandersauthor=M.+E.+Wampoleauthor=M.+L.+Thakurauthor=E.+Wickstrom&title=Molecular+determinants+of+epidermal+growth+factor+binding%3A+a+molecular+dynamics+study&doi=10.1371%2Fjournal.pone.0054136"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular determinants of epidermal growth factor binding: a molecular dynamics study</span></div><div class="casAuthors">Sanders, Jeffrey M.; Wampole, Matthew E.; Thakur, Mathew L.; Wickstrom, Eric</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">e54136</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">The epidermal growth factor receptor (EGFR) is a member of the receptor tyrosine kinase family that plays a role in multiple cellular processes.  Activation of EGFR requires binding of a ligand on the extracellular domain to promote conformational changes leading to dimerization and transphosphorylation of intracellular kinase domains.  Seven ligands are known to bind EGFR with affinities ranging from sub-nanomolar to near micromolar dissocn. consts.  In the case of EGFR, distinct conformational states assumed upon binding a ligand is thought to be a detg. factor in activation of a downstream signaling network.  Previous biochem. studies suggest the existence of both low affinity and high affinity EGFR ligands.  While these studies have identified functional effects of ligand binding, high-resoln. structural data are lacking.  To gain a better understanding of the mol. basis of EGFR binding affinities, we docked each EGFR ligand to the putative active state extracellular domain dimer and 25.0 ns mol. dynamics simulations were performed.  MM-PBSA/GBSA are efficient computational approaches to approx. free energies of protein-protein interactions and decomp. the free energy at the amino acid level.  We applied these methods to the last 6.0 ns of each ligand-receptor simulation.  MM-PBSA calcns. were able to successfully rank all seven of the EGFR ligands based on the two affinity classes: EGF>HB-EGF>TGF-α>BTC>EPR>EPG>AR.  Results from energy decompn. identified several interactions that are common among binding ligands.  These findings reveal that while several residues are conserved among the EGFR ligand family, no single set of residues dets. the affinity class.  Instead we found heterogeneous sets of interactions that were driven primarily by electrostatic and Van der Waals forces.  These results not only illustrate the complexity of EGFR dynamics but also pave the way for structure-based design of therapeutics targeting EGF ligands or the receptor itself.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqTdLUJ_FXUTbVg90H21EOLACvtfcHk0lgvCZccmnKEMQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXit1ChsrY%253D&md5=f2630ac7526af1a73890557bea731b79</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0054136&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0054136%26sid%3Dliteratum%253Aachs%26aulast%3DSanders%26aufirst%3DJ.%2BM.%26aulast%3DWampole%26aufirst%3DM.%2BE.%26aulast%3DThakur%26aufirst%3DM.%2BL.%26aulast%3DWickstrom%26aufirst%3DE.%26atitle%3DMolecular%2520determinants%2520of%2520epidermal%2520growth%2520factor%2520binding%253A%2520a%2520molecular%2520dynamics%2520study%26jtitle%3DPLoS%2520One%26date%3D2013%26volume%3D8%26doi%3D10.1371%2Fjournal.pone.0054136" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jones, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Akita, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sliwkowski, M. X.</span></span> <span> </span><span class="NLM_article-title">Binding specificities and affinities of egf domains for ErbB receptors</span>. <i>FEBS Lett.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>447</i></span>,  <span class="NLM_fpage">227</span>– <span class="NLM_lpage">231</span>, <span class="refDoi"> DOI: 10.1016/S0014-5793(99)00283-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=10.1016%2FS0014-5793%2899%2900283-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=10214951" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=1%3ACAS%3A528%3ADyaK1MXis1ars7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=447&publication_year=1999&pages=227-231&author=J.+T.+Jonesauthor=R.+W.+Akitaauthor=M.+X.+Sliwkowski&title=Binding+specificities+and+affinities+of+egf+domains+for+ErbB+receptors&doi=10.1016%2FS0014-5793%2899%2900283-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Binding specificities and affinities of egf domains for ErbB receptors</span></div><div class="casAuthors">Jones, Jennifer T.; Akita, Robert W.; Sliwkowski, Mark X.</div><div class="citationInfo"><span class="NLM_cas:title">FEBS Letters</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">447</span>
        (<span class="NLM_cas:issue">2,3</span>),
    <span class="NLM_cas:pages">227-231</span>CODEN:
                <span class="NLM_cas:coden">FEBLAL</span>;
        ISSN:<span class="NLM_cas:issn">0014-5793</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science B.V.</span>)
        </div><div class="casAbstract">ErbB receptor activation is a complex process and is dependent upon the type and no. of receptors expressed on a given cell.  Previous studies with defined combinations of ErbB receptors expressed in mammalian cells have helped elucidate specific biol. responses for many of the recognized gene products that serve as ligands for these receptors.  However, no study has examd. the binding of these ligands in a defined exptl. system.  To address this issue, the relative binding affinities of the egf domains of eleven ErbB ligands were measured on six ErbB receptor combinations using a sol. receptor-ligand binding format.  The ErbB2/4 heterodimer was shown to bind all ligands tested with moderate to very high affinity.  In contrast, ErbB3 showed much more restrictive ligand binding specificity and measurable binding was obsd. only with heregulin, neuregulin2β, epiregulin and the synthetic heregulin/egf chimera, biregulin.  These studies also revealed that ErbB2 preferentially enhances ligand binding to ErbB3 or ErbB4 and to a lesser degree to ErbB1.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGodjee8uZQb57Vg90H21EOLACvtfcHk0lgeEpggAWR_bg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXis1ars7w%253D&md5=247b1b73bd015dcda612eebd820f6c4d</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1016%2FS0014-5793%2899%2900283-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0014-5793%252899%252900283-5%26sid%3Dliteratum%253Aachs%26aulast%3DJones%26aufirst%3DJ.%2BT.%26aulast%3DAkita%26aufirst%3DR.%2BW.%26aulast%3DSliwkowski%26aufirst%3DM.%2BX.%26atitle%3DBinding%2520specificities%2520and%2520affinities%2520of%2520egf%2520domains%2520for%2520ErbB%2520receptors%26jtitle%3DFEBS%2520Lett.%26date%3D1999%26volume%3D447%26spage%3D227%26epage%3D231%26doi%3D10.1016%2FS0014-5793%2899%2900283-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Di, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kerns, E. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fan, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McConnell, O. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carter, G. T.</span></span> <span> </span><span class="NLM_article-title">High throughput artificial membrane permeability assay for blood–brain barrier</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>38</i></span>,  <span class="NLM_fpage">223</span>– <span class="NLM_lpage">232</span>, <span class="refDoi"> DOI: 10.1016/S0223-5234(03)00012-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=10.1016%2FS0223-5234%2803%2900012-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=12667689" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=1%3ACAS%3A528%3ADC%252BD3sXitlKju7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2003&pages=223-232&author=L.+Diauthor=E.+H.+Kernsauthor=K.+Fanauthor=O.+J.+McConnellauthor=G.+T.+Carter&title=High+throughput+artificial+membrane+permeability+assay+for+blood%E2%80%93brain+barrier&doi=10.1016%2FS0223-5234%2803%2900012-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">High throughput artificial membrane permeability assay for blood-brain barrier</span></div><div class="casAuthors">Di, Li; Kerns, Edward H.; Fan, Kristi; McConnell, Oliver J.; Carter, Guy T.</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">223-232</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Editions Scientifiques et Medicales Elsevier</span>)
        </div><div class="casAbstract">The recent advances in high throughput screening for biol. activities and combinatorial chem. have greatly expanded the no. of drug candidates.  Rapid screening for BBB penetration potential early in drug discovery programs provides important information for compd. selection and guidance of synthesis for desirable CNS properties.  In this paper, we discuss a modification of the parallel artificial membrane permeation assay (PAMPA) for the prediction of blood-brain barrier penetration (PAMPA-BBB).  The assay was developed with 30 structurally diverse com. drugs and validated with 14 Wyeth Research compds.  The PAMPA-BBB assay has the advantages of: predicting passive blood-brain barrier penetration with high success, high throughput, low cost, and reproducibility.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqZCg9Ka4mziLVg90H21EOLACvtfcHk0lgeEpggAWR_bg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXitlKju7c%253D&md5=d228d5808ba8df23deed92c5e2bf3863</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1016%2FS0223-5234%2803%2900012-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0223-5234%252803%252900012-6%26sid%3Dliteratum%253Aachs%26aulast%3DDi%26aufirst%3DL.%26aulast%3DKerns%26aufirst%3DE.%2BH.%26aulast%3DFan%26aufirst%3DK.%26aulast%3DMcConnell%26aufirst%3DO.%2BJ.%26aulast%3DCarter%26aufirst%3DG.%2BT.%26atitle%3DHigh%2520throughput%2520artificial%2520membrane%2520permeability%2520assay%2520for%2520blood%25E2%2580%2593brain%2520barrier%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2003%26volume%3D38%26spage%3D223%26epage%3D232%26doi%3D10.1016%2FS0223-5234%2803%2900012-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H.</span></span> <span> </span><span class="NLM_article-title">Prediction of the blood–brain barrier (BBB) permeability of chemicals based on machine-learning and ensemble methods</span>. <i>Chem. Res. Toxicol.</i> <span class="NLM_year" style="font-weight: bold;">2021</span>,  <span class="NLM_volume"><i>34</i></span>,  <span class="NLM_fpage">1456</span>– <span class="NLM_lpage">1467</span>, <span class="refDoi"> DOI: 10.1021/acs.chemrestox.0c00343</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.chemrestox.0c00343" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=1%3ACAS%3A528%3ADC%252BB3MXhtFKqs7fF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=2021&pages=1456-1467&author=L.+Liuauthor=L.+Zhangauthor=H.+Fengauthor=S.+Liauthor=M.+Liuauthor=J.+Zhaoauthor=H.+Liu&title=Prediction+of+the+blood%E2%80%93brain+barrier+%28BBB%29+permeability+of+chemicals+based+on+machine-learning+and+ensemble+methods&doi=10.1021%2Facs.chemrestox.0c00343"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Prediction of the Blood-Brain Barrier (BBB) Permeability of Chemicals Based on Machine-Learning and Ensemble Methods</span></div><div class="casAuthors">Liu, Lili; Zhang, Li; Feng, Huawei; Li, Shimeng; Liu, Miao; Zhao, Jian; Liu, Hongsheng</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Research in Toxicology</span>
        (<span class="NLM_cas:date">2021</span>),
    <span class="NLM_cas:volume">34</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1456-1467</span>CODEN:
                <span class="NLM_cas:coden">CRTOEC</span>;
        ISSN:<span class="NLM_cas:issn">0893-228X</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The ability of chems. to enter the blood-brain barrier (BBB) is a key factor for central nervous system (CNS) drug development.  Although many models for BBB permeability prediction have been developed, they have insufficient accuracy (ACC) and sensitivity (SEN).  To improve performance, ensemble models were built to predict the BBB permeability of compds.  In this study, in silico ensemble-learning models were developed using 3 machine-learning algorithms and 9 mol. fingerprints from 1757 chems. (integrated from 2 published data sets) to predict BBB permeability.  The best prediction performance of the base classifier models was achieved by a prediction model based on an random forest (RF) and a MACCS mol. fingerprint with an ACC of 0.910, an area under the receiver-operating characteristic (ROC) curve (AUC) of 0.957, a SEN of 0.927, and a specificity of 0.867 in 5-fold cross-validation.  The prediction performance of the ensemble models is better than that of most of the base classifiers.  The final ensemble model has also demonstrated good accuracy for an external validation and can be used for the early screening of CNS drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpbmzj6JQ-zmLVg90H21EOLACvtfcHk0lgeEpggAWR_bg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3MXhtFKqs7fF&md5=c0c982566369b519805742a31f0214f6</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1021%2Facs.chemrestox.0c00343&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.chemrestox.0c00343%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DFeng%26aufirst%3DH.%26aulast%3DLi%26aufirst%3DS.%26aulast%3DLiu%26aufirst%3DM.%26aulast%3DZhao%26aufirst%3DJ.%26aulast%3DLiu%26aufirst%3DH.%26atitle%3DPrediction%2520of%2520the%2520blood%25E2%2580%2593brain%2520barrier%2520%2528BBB%2529%2520permeability%2520of%2520chemicals%2520based%2520on%2520machine-learning%2520and%2520ensemble%2520methods%26jtitle%3DChem.%2520Res.%2520Toxicol.%26date%3D2021%26volume%3D34%26spage%3D1456%26epage%3D1467%26doi%3D10.1021%2Facs.chemrestox.0c00343" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bujard, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petit, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carrupt, P.-A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rudaz, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schappler, J.</span></span> <span> </span><span class="NLM_article-title">HDM-PAMPA to predict gastrointestinal absorption, binding percentage, equilibrium and kinetics constants with human serum albumin and using 2 end-point measurements</span>. <i>Eur. J. Pharm. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>97</i></span>,  <span class="NLM_fpage">143</span>– <span class="NLM_lpage">150</span>, <span class="refDoi"> DOI: 10.1016/j.ejps.2016.11.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=10.1016%2Fj.ejps.2016.11.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=27816629" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvFWnsrbJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=97&publication_year=2017&pages=143-150&author=A.+Bujardauthor=C.+Petitauthor=P.-A.+Carruptauthor=S.+Rudazauthor=J.+Schappler&title=HDM-PAMPA+to+predict+gastrointestinal+absorption%2C+binding+percentage%2C+equilibrium+and+kinetics+constants+with+human+serum+albumin+and+using+2+end-point+measurements&doi=10.1016%2Fj.ejps.2016.11.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">HDM-PAMPA to predict gastrointestinal absorption, binding percentage, equilibrium and kinetics constants with human serum albumin and using 2 end-point measurements</span></div><div class="casAuthors">Bujard, Alban; Petit, Charlotte; Carrupt, Pierre-Alain; Rudaz, Serge; Schappler, Julie</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Pharmaceutical Sciences</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">97</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">143-150</span>CODEN:
                <span class="NLM_cas:coden">EPSCED</span>;
        ISSN:<span class="NLM_cas:issn">0928-0987</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The parallel artificial membrane permeability assay (PAMPA) is a high-throughput screening (HTS) technique developed to predict passive permeability through numerous different biol. membranes, such as the gastrointestinal tract (GIT), the blood brain barrier (BBB), and the dermal layer.  PAMPA is based on an artificial membrane, such as hexadecane (HDM), which separates two compartments (i.e., a donor and an acceptor compartment).  In the present study, an HDM-PAMPA method was developed with human serum albumin (HSA) under iso-pH and gradient-pH conditions to predict the percentage of binding, dissocn./assocn. consts. (Kd and Ka, resp.) and dissocn./assocn. kinetic rates (koff and kon, resp.) between a given drug and HSA.  Thanks to the kinetic properties of PAMPA, a two end-point assay was implemented to obtain all three properties.  The assay was used to measure basic, acidic, and amphoteric compds.  The protein was free in soln., allowing a direct comparison between this assay and equil. dialysis (ED).  The developed PAMPA enabled screening of up to 96 compds. in a single run, generating valuable information on absorption and distribution in a high-throughput and high-repeatable manner.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq1Yoxw5vh9ybVg90H21EOLACvtfcHk0lg8IA3mpGyv6A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvFWnsrbJ&md5=8d6ec5bcba80401d7d30f32474bbceb6</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1016%2Fj.ejps.2016.11.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejps.2016.11.001%26sid%3Dliteratum%253Aachs%26aulast%3DBujard%26aufirst%3DA.%26aulast%3DPetit%26aufirst%3DC.%26aulast%3DCarrupt%26aufirst%3DP.-A.%26aulast%3DRudaz%26aufirst%3DS.%26aulast%3DSchappler%26aufirst%3DJ.%26atitle%3DHDM-PAMPA%2520to%2520predict%2520gastrointestinal%2520absorption%252C%2520binding%2520percentage%252C%2520equilibrium%2520and%2520kinetics%2520constants%2520with%2520human%2520serum%2520albumin%2520and%2520using%25202%2520end-point%2520measurements%26jtitle%3DEur.%2520J.%2520Pharm.%2520Sci.%26date%3D2017%26volume%3D97%26spage%3D143%26epage%3D150%26doi%3D10.1016%2Fj.ejps.2016.11.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lv, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shang, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sha, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span> <span> </span><span class="NLM_article-title">Penetration of verapamil across blood brain barrier following cerebral ischemia depending on both paracellular pathway and P-glycoprotein transportation</span>. <i>Neurochem. Int.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">23</span>– <span class="NLM_lpage">30</span>, <span class="refDoi"> DOI: 10.1016/j.neuint.2012.10.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=10.1016%2Fj.neuint.2012.10.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=23142723" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvV2ktLjN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2013&pages=23-30&author=W.+Fangauthor=P.+Lvauthor=X.+Gengauthor=E.+Shangauthor=Q.+Yangauthor=L.+Shaauthor=Y.+Li&title=Penetration+of+verapamil+across+blood+brain+barrier+following+cerebral+ischemia+depending+on+both+paracellular+pathway+and+P-glycoprotein+transportation&doi=10.1016%2Fj.neuint.2012.10.012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Penetration of verapamil across blood brain barrier following cerebral ischemia depending on both paracellular pathway and P-glycoprotein transportation</span></div><div class="casAuthors">Fang, Weirong; Lv, Peng; Geng, Xiaohan; Shang, Erxin; Yang, Qichuan; Sha, Lan; Li, Yunman</div><div class="citationInfo"><span class="NLM_cas:title">Neurochemistry International</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">23-30</span>CODEN:
                <span class="NLM_cas:coden">NEUIDS</span>;
        ISSN:<span class="NLM_cas:issn">0197-0186</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Blood brain barrier (BBB) dysfunction is a common facet of cerebral ischemia, and the alteration of drug transporter, P-glycoprotein (P-gp), has been documented.  This study explores influence of damaged BBB and elevated P-gp on cerebral verapamil penetration after ischemia both in vivo and in vitro.  Middle cerebral artery occlusion (MCAO) induced ischemia/reperfusion (I/R) of rats, and Na2S2O4 induced hypoxia/reoxygenation (H/R) damage of rat brain mirovessel endothelial cells (RBMECs) resp., served as BBB breakdown model in vivo and in vitro.  Evans-Blue (EB) extravagation and 125I-albumin were used to quantify BBB dysfunction; UPLC-MS/MS anal. method was performed to det. accurately the concn. of verapamil in brain tissue and cell.  Flow cytometry, immunohistochem. and western blotting were applied to evaluate transport function and protein expression of P-gp.  Overexpressed ICAM-1 and MMP-9 mediated BBB dysfunction after ischemia, which induced EB leakage and 125I-albumin uptake increase.  Enhanced accumulation of verapamil in brain tissue, but intracellular concn. reduced evidently after H/R injury.  Transcellular transportation of verapamil elevated when P-gp function or expression was inhibited after H/R injury.  These data indicated that BBB penetration of verapamil under ischemia condition was not only depending on BBB breakdown, but also regulated by P-gp.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGokw3azr0cCY7Vg90H21EOLACvtfcHk0lg8IA3mpGyv6A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvV2ktLjN&md5=977b7e2390458e929243c6150cc3424f</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1016%2Fj.neuint.2012.10.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.neuint.2012.10.012%26sid%3Dliteratum%253Aachs%26aulast%3DFang%26aufirst%3DW.%26aulast%3DLv%26aufirst%3DP.%26aulast%3DGeng%26aufirst%3DX.%26aulast%3DShang%26aufirst%3DE.%26aulast%3DYang%26aufirst%3DQ.%26aulast%3DSha%26aufirst%3DL.%26aulast%3DLi%26aufirst%3DY.%26atitle%3DPenetration%2520of%2520verapamil%2520across%2520blood%2520brain%2520barrier%2520following%2520cerebral%2520ischemia%2520depending%2520on%2520both%2520paracellular%2520pathway%2520and%2520P-glycoprotein%2520transportation%26jtitle%3DNeurochem.%2520Int.%26date%3D2013%26volume%3D62%26spage%3D23%26epage%3D30%26doi%3D10.1016%2Fj.neuint.2012.10.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fallingborg, J.</span></span> <span> </span><span class="NLM_article-title">Intraluminal pH of the human gastrointestinal tract</span>. <i>Dan. Med. Bull.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>46</i></span>,  <span class="NLM_fpage">183</span>– <span class="NLM_lpage">196</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=10421978" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=1%3ACAS%3A280%3ADyaK1MzltVykug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=1999&pages=183-196&author=J.+Fallingborg&title=Intraluminal+pH+of+the+human+gastrointestinal+tract"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Intraluminal pH of the human gastrointestinal tract</span></div><div class="casAuthors">Fallingborg J</div><div class="citationInfo"><span class="NLM_cas:title">Danish medical bulletin</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">183-96</span>
        ISSN:<span class="NLM_cas:issn">0907-8916</span>.
    </div><div class="casAbstract">After a short introduction (chapter 1) methods of measuring gastrointestinal pH are described in chapter 2.  The methods are divided into intubation techniques and tubeless methods, and the advantages and disadvantages are discussed.  Measurements with pH-sensitive, radiotransmitting capsules are highlighted, and methodological problems with these capsules are described.  Chapter 3 concerns the gastrointestinal pH profile of healthy subjects.  The intraluminal pH is rapidly changed from highly acid in the stomach to about pH 6 in the duodenum.  The pH gradually increases in the small intestine from pH 6 to about pH 7.4 in the terminal ileum.  The pH drops to 5.7 in the caecum, but again gradually increases, reaching pH 6.7 in the rectum.  The physiological background of these pH values is discussed.  Chapter 4 describes the effect of gastrointestinal pH on bacterial flora, absorption of vitamins and electrolytes, and on the activity of digestive enzymes.  The pH-profile in children is described in chapter 5.  The profile is identical with that of adults, and it is therefore concluded that the release of a drug from pH-dependent, controlled-release preparations is also probably identical with that of adults.  Chapter 6 describes the correlation between certain diseases and the gastrointestinal pH.  A resection of the colon and the creation of an ileostomy do not affect the pH of the remaining gut.  An ileocaecal resection shortens the small intestinal transit time, increases pH of the proximal colon, but does not change the pH-profile of the small intestine.  Chronic pancreatitis and cystic fibrosis seem to decrease pH of the proximal small intestine.  Very low colonic pH values have been observed in severe active ulcerative colitis and in Crohn's disease, but the background and clinical implication of this phenomenon are not clear.  Chapter 7 describes the modulating effect of diet and drugs on gastrointestinal pH.  Diet primarily has an effect on the colonic pH, whereas drugs might affect both small intestinal and colonic pH.  The different effects are described.  Finally, chapter 8 summarizes the present knowledge about gastrointestinal pH, and future investigations are proposed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTo6G-nALPYxUkUaC_kzhvyfW6udTcc2ebwagkxpvX0_7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADyaK1MzltVykug%253D%253D&md5=caa72e8a9c3e34a923320c5ec31d958e</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DFallingborg%26aufirst%3DJ.%26atitle%3DIntraluminal%2520pH%2520of%2520the%2520human%2520gastrointestinal%2520tract%26jtitle%3DDan.%2520Med.%2520Bull.%26date%3D1999%26volume%3D46%26spage%3D183%26epage%3D196" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmitz, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeffrey, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wiltzius, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kussie, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferguson, K. M.</span></span> <span> </span><span class="NLM_article-title">Structural basis for inhibition of the epidermal growth factor receptor by cetuximab</span>. <i>Cancer Cell</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">301</span>– <span class="NLM_lpage">311</span>, <span class="refDoi"> DOI: 10.1016/j.ccr.2005.03.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=10.1016%2Fj.ccr.2005.03.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=15837620" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=1%3ACAS%3A528%3ADC%252BD2MXjvFSkurk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2005&pages=301-311&author=S.+Liauthor=K.+R.+Schmitzauthor=P.+D.+Jeffreyauthor=J.+J.+Wiltziusauthor=P.+Kussieauthor=K.+M.+Ferguson&title=Structural+basis+for+inhibition+of+the+epidermal+growth+factor+receptor+by+cetuximab&doi=10.1016%2Fj.ccr.2005.03.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Structural basis for inhibition of the epidermal growth factor receptor by cetuximab</span></div><div class="casAuthors">Li, Shiqing; Schmitz, Karl R.; Jeffrey, Philip D.; Wiltzius, Jed J. W.; Kussie, Paul; Ferguson, Kathryn M.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">301-311</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Recent structural studies of epidermal growth factor receptor (EGFR) family extracellular regions have identified an unexpected mechanism for ligand-induced receptor dimerization that has important implications for activation and inhibition of these receptors.  Here we describe the 2.8 Å resoln. x-ray crystal structure of the antigen binding (Fab) fragment from cetuximab (Erbitux), an inhibitory anti-EGFR antibody, in complex with the sol. extracellular region of EGFR (sEGFR).  The sEGFR is in the characteristic "autoinhibited" or "tethered" inactive configuration.  Cetuximab interacts exclusively with domain III of sEGFR, partially occluding the ligand binding region on this domain and sterically preventing the receptor from adopting the extended conformation required for dimerization.  We suggest that both these effects contribute to potent inhibition of EGFR activation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrfRvCB5ETMNLVg90H21EOLACvtfcHk0ljfNwtXKjWdEw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXjvFSkurk%253D&md5=7d7b01d331206edd65f0d2c5c69785dd</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1016%2Fj.ccr.2005.03.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccr.2005.03.003%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DS.%26aulast%3DSchmitz%26aufirst%3DK.%2BR.%26aulast%3DJeffrey%26aufirst%3DP.%2BD.%26aulast%3DWiltzius%26aufirst%3DJ.%2BJ.%26aulast%3DKussie%26aufirst%3DP.%26aulast%3DFerguson%26aufirst%3DK.%2BM.%26atitle%3DStructural%2520basis%2520for%2520inhibition%2520of%2520the%2520epidermal%2520growth%2520factor%2520receptor%2520by%2520cetuximab%26jtitle%3DCancer%2520Cell%26date%3D2005%26volume%3D7%26spage%3D301%26epage%3D311%26doi%3D10.1016%2Fj.ccr.2005.03.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Udugamasooriya, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dineen, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brekken, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kodadek, T.</span></span> <span> </span><span class="NLM_article-title">A peptoid ″antibody surrogate″ that antagonizes VEGF receptor 2 activity</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>130</i></span>,  <span class="NLM_fpage">5744</span>– <span class="NLM_lpage">5752</span>, <span class="refDoi"> DOI: 10.1021/ja711193x</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja711193x" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=1%3ACAS%3A528%3ADC%252BD1cXktFShsbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=130&publication_year=2008&pages=5744-5752&author=D.+G.+Udugamasooriyaauthor=S.+P.+Dineenauthor=R.+A.+Brekkenauthor=T.+Kodadek&title=A+peptoid+%E2%80%B3antibody+surrogate%E2%80%B3+that+antagonizes+VEGF+receptor+2+activity&doi=10.1021%2Fja711193x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">A Peptoid "Antibody Surrogate" That Antagonizes VEGF Receptor 2 Activity</span></div><div class="casAuthors">Udugamasooriya, D. Gomika; Dineen, Sean P.; Brekken, Rolf A.; Kodadek, Thomas</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">130</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">5744-5752</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The authors report a two-color, cell-based screen to identify specific receptor-binding compds. in a combinatorial library of peptoids displayed on beads.  The authors apply this strategy to the isolation of vascular endothelial growth factor receptor 2 (VEGFR2)-binding peptoids.  A dimeric deriv. of one of these lead compds. is shown to be an antagonist of VEGFR2 activity both in vitro and in vivo.  This methodol. provides a potentially general route to synthetic mols. that bind integral membrane receptors with affinities and specificities similar to those of antibodies, but which are far smaller and easier to make and manipulate.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp6yicSigByj7Vg90H21EOLACvtfcHk0ljfNwtXKjWdEw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXktFShsbc%253D&md5=b15f3f4eade6beff3a80142c481dab50</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1021%2Fja711193x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja711193x%26sid%3Dliteratum%253Aachs%26aulast%3DUdugamasooriya%26aufirst%3DD.%2BG.%26aulast%3DDineen%26aufirst%3DS.%2BP.%26aulast%3DBrekken%26aufirst%3DR.%2BA.%26aulast%3DKodadek%26aufirst%3DT.%26atitle%3DA%2520peptoid%2520%25E2%2580%25B3antibody%2520surrogate%25E2%2580%25B3%2520that%2520antagonizes%2520VEGF%2520receptor%25202%2520activity%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2008%26volume%3D130%26spage%3D5744%26epage%3D5752%26doi%3D10.1021%2Fja711193x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kodadek, T.</span></span> <span> </span><span class="NLM_article-title">Development of antibody surrogates for the treatment of cancers and autoimmune disease</span>. <i>Curr. Opin. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">721</span>– <span class="NLM_lpage">727</span>, <span class="refDoi"> DOI: 10.1016/j.cbpa.2010.07.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=10.1016%2Fj.cbpa.2010.07.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=20674470" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsFamur7K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2010&pages=721-727&author=T.+Kodadek&title=Development+of+antibody+surrogates+for+the+treatment+of+cancers+and+autoimmune+disease&doi=10.1016%2Fj.cbpa.2010.07.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Development of antibody surrogates for the treatment of cancers and autoimmune disease</span></div><div class="casAuthors">Kodadek, Thomas</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Chemical Biology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">721-727</span>CODEN:
                <span class="NLM_cas:coden">COCBF4</span>;
        ISSN:<span class="NLM_cas:issn">1367-5931</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Some therapeutic monoclonal antibodies function by focusing the killing power of the immune system on particular cellular targets, a process known as antibody-dependent cell-mediated cytotoxicity (ADCC).  There is considerable interest in developing small-mol. therapeutics that utilize this mechanism of action and so act as a kind of antibody surrogate.  Progress and challenges in this interesting field are reviewed here.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGor5Ws7XJiFubVg90H21EOLACvtfcHk0ljfNwtXKjWdEw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsFamur7K&md5=6388752708bb215fd8e66756ac139264</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1016%2Fj.cbpa.2010.07.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cbpa.2010.07.002%26sid%3Dliteratum%253Aachs%26aulast%3DKodadek%26aufirst%3DT.%26atitle%3DDevelopment%2520of%2520antibody%2520surrogates%2520for%2520the%2520treatment%2520of%2520cancers%2520and%2520autoimmune%2520disease%26jtitle%3DCurr.%2520Opin.%2520Chem.%2520Biol.%26date%3D2010%26volume%3D14%26spage%3D721%26epage%3D727%26doi%3D10.1016%2Fj.cbpa.2010.07.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Landrum, G.</span></span> <span> </span><span class="NLM_article-title">RDKit: Open-Source Cheminformatics from Machine Learning to Chemical Registration</span>. In  <i>Abstracts of Papers of the American Chemical Society</i>; <span class="NLM_publisher-name">American Chemical Society</span>: <span class="NLM_publisher-loc">Washington, DC</span>, <span class="NLM_year">2019</span>; Vol.  <span class="NLM_volume">258</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2019&author=G.+Landrum&title=Abstracts+of+Papers+of+the+American+Chemical+Society"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DLandrum%26aufirst%3DG.%26atitle%3DRDKit%253A%2520Open-Source%2520Cheminformatics%2520from%2520Machine%2520Learning%2520to%2520Chemical%2520Registration%26btitle%3DAbstracts%2520of%2520Papers%2520of%2520the%2520American%2520Chemical%2520Society%26pub%3DAmerican%2520Chemical%2520Society%26date%3D2019%26volume%3D258" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span> <i>The PyMOL Molecular
Graphics System</i>, Version 1.3r1, <span class="NLM_year">2010</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+The+PyMOL+Molecular%0AGraphics+System%2C+Version+1.3r1%2C+2010."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DThe%2520PyMOL%2520Molecular%250AGraphics%2520System%26date%3D2010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sastry, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adzhigirey, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Day, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Annabhimoju, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sherman, W.</span></span> <span> </span><span class="NLM_article-title">Protein and ligand preparation: parameters, protocols, and influence on virtual screening enrichments</span>. <i>J. Comput.-Aided Mol. Des.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">221</span>– <span class="NLM_lpage">234</span>, <span class="refDoi"> DOI: 10.1007/s10822-013-9644-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=10.1007%2Fs10822-013-9644-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=23579614" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;key=1%3ACAS%3A280%3ADC%252BC3srksFekug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2013&pages=221-234&author=G.+M.+Sastryauthor=M.+Adzhigireyauthor=T.+Dayauthor=R.+Annabhimojuauthor=W.+Sherman&title=Protein+and+ligand+preparation%3A+parameters%2C+protocols%2C+and+influence+on+virtual+screening+enrichments&doi=10.1007%2Fs10822-013-9644-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Protein and ligand preparation: parameters, protocols, and influence on virtual screening enrichments</span></div><div class="casAuthors">Sastry G Madhavi; Adzhigirey Matvey; Day Tyler; Annabhimoju Ramakrishna; Sherman Woody</div><div class="citationInfo"><span class="NLM_cas:title">Journal of computer-aided molecular design</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">221-34</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Structure-based virtual screening plays an important role in drug discovery and complements other screening approaches.  In general, protein crystal structures are prepared prior to docking in order to add hydrogen atoms, optimize hydrogen bonds, remove atomic clashes, and perform other operations that are not part of the x-ray crystal structure refinement process.  In addition, ligands must be prepared to create 3-dimensional geometries, assign proper bond orders, and generate accessible tautomer and ionization states prior to virtual screening.  While the prerequisite for proper system preparation is generally accepted in the field, an extensive study of the preparation steps and their effect on virtual screening enrichments has not been performed.  In this work, we systematically explore each of the steps involved in preparing a system for virtual screening.  We first explore a large number of parameters using the Glide validation set of 36 crystal structures and 1,000 decoys.  We then apply a subset of protocols to the DUD database.  We show that database enrichment is improved with proper preparation and that neglecting certain steps of the preparation process produces a systematic degradation in enrichments, which can be large for some targets.  We provide examples illustrating the structural changes introduced by the preparation that impact database enrichment.  While the work presented here was performed with the Protein Preparation Wizard and Glide, the insights and guidance are expected to be generalizable to structure-based virtual screening with other docking methods.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQd2Qiai-OqeXqj_yg7twvpfW6udTcc2ea3qA4nE18Sqrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3srksFekug%253D%253D&md5=9eb1a820e121aca2742ed53f24481ace</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1007%2Fs10822-013-9644-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-013-9644-8%26sid%3Dliteratum%253Aachs%26aulast%3DSastry%26aufirst%3DG.%2BM.%26aulast%3DAdzhigirey%26aufirst%3DM.%26aulast%3DDay%26aufirst%3DT.%26aulast%3DAnnabhimoju%26aufirst%3DR.%26aulast%3DSherman%26aufirst%3DW.%26atitle%3DProtein%2520and%2520ligand%2520preparation%253A%2520parameters%252C%2520protocols%252C%2520and%2520influence%2520on%2520virtual%2520screening%2520enrichments%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2013%26volume%3D27%26spage%3D221%26epage%3D234%26doi%3D10.1007%2Fs10822-013-9644-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1YY9" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1YY9','PDB','1YY9'); return false;">PDB: 1YY9</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i27"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.1c00607">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_10178"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a class="ext-link" href="/doi/10.1021/acs.jmedchem.1c00607?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acs.jmedchem.1c00607</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Representative picture of fluorescent beads; chemical structures of FITC-labeled hits; binding affinity of hits determined by FP; HPLC traces of hits; stability study toward enzyme hydrolysis; one-bead-two-compound bead screening protocol; and MALDI data for hits (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.1c00607/suppl_file/jm1c00607_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Molecular formula strings for final compounds (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.1c00607/suppl_file/jm1c00607_si_002.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00607/suppl_file/jm1c00607_si_001.pdf">jm1c00607_si_001.pdf (1.28 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00607/suppl_file/jm1c00607_si_002.csv">jm1c00607_si_002.csv (1.91 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.1c00607&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.1c00607%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.1c00607" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                8MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"677a5fce38373d3c","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
