
<html lang="en"     class="pb-page"  data-request-id="f62b8411-1736-49a1-9b20-54828ffb9db8"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;issue:issue:10.1021/jmcmar.2017.60.issue-20;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;article:article:10.1021/acs.jmedchem.7b00840;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="dc.Title" content="Discovery of 1-(4-(4-Amino-3-(4-(2-morpholinoethoxy)phenyl)-1H-pyrazolo[3,4-d]pyrimidin-1-yl)phenyl)-3-(5-(tert-butyl)isoxazol-3-yl)urea (CHMFL-FLT3-213) as a Highly Potent Type II FLT3 Kinase Inhibitor Capable of Overcoming a Variety of FLT3 Kinase Mutants in FLT3-ITD Positive AML" /></meta><meta name="dc.Creator" content="Aoli  Wang" /></meta><meta name="dc.Creator" content="Xixiang  Li" /></meta><meta name="dc.Creator" content="Cheng  Chen" /></meta><meta name="dc.Creator" content="Hong  Wu" /></meta><meta name="dc.Creator" content="Ziping  Qi" /></meta><meta name="dc.Creator" content="Chen  Hu" /></meta><meta name="dc.Creator" content="Kailin  Yu" /></meta><meta name="dc.Creator" content="Jiaxin  Wu" /></meta><meta name="dc.Creator" content="Juan  Liu" /></meta><meta name="dc.Creator" content="Xiaochuan  Liu" /></meta><meta name="dc.Creator" content="Zhenquan  Hu" /></meta><meta name="dc.Creator" content="Wei  Wang" /></meta><meta name="dc.Creator" content="Wenliang  Wang" /></meta><meta name="dc.Creator" content="Wenchao  Wang" /></meta><meta name="dc.Creator" content="Li  Wang" /></meta><meta name="dc.Creator" content="Beilei  Wang" /></meta><meta name="dc.Creator" content="Qingwang  Liu" /></meta><meta name="dc.Creator" content="Lili  Li" /></meta><meta name="dc.Creator" content="Jian  Ge" /></meta><meta name="dc.Creator" content="Tao  Ren" /></meta><meta name="dc.Creator" content="Shanchun  Zhang" /></meta><meta name="dc.Creator" content="Ruixiang  Xia" /></meta><meta name="dc.Creator" content="Jing  Liu" /></meta><meta name="dc.Creator" content="Qingsong  Liu" /></meta><meta name="dc.Description" content="FLT3-ITD mutant has been observed in about 30% of AML patients and extensively studied as a drug discovery target. On the basis of our previous study that ibrutinib (9) exhibited selective and mode..." /></meta><meta name="Description" content="FLT3-ITD mutant has been observed in about 30% of AML patients and extensively studied as a drug discovery target. On the basis of our previous study that ibrutinib (9) exhibited selective and mode..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="October 17, 2017" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.7b00840" /></meta><meta name="dc.Language" content="EN" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2017 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.7b00840" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.7b00840" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.7b00840" /></link>
        
    
    

<title>Discovery of 1-(4-(4-Amino-3-(4-(2-morpholinoethoxy)phenyl)-1H-pyrazolo[3,4-d]pyrimidin-1-yl)phenyl)-3-(5-(tert-butyl)isoxazol-3-yl)urea (CHMFL-FLT3-213) as a Highly Potent Type II FLT3 Kinase Inhibitor Capable of Overcoming a Variety of FLT3 Kinase Mutants in FLT3-ITD Positive AML | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.7b00840" /></meta><meta property="og:title" content="Discovery of 1-(4-(4-Amino-3-(4-(2-morpholinoethoxy)phenyl)-1H-pyrazolo[3,4-d]pyrimidin-1-yl)phenyl)-3-(5-(tert-butyl)isoxazol-3-yl)urea (CHMFL-FLT3-213) as a Highly Potent Type II FLT3 Kinase Inhibitor Capable of Overcoming a Variety of FLT3 Kinase Mutants in FLT3-ITD Positive AML" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-20/acs.jmedchem.7b00840/20171020/images/large/jm-2017-008406_0014.jpeg" /></meta><meta property="og:description" content="FLT3-ITD mutant has been observed in about 30% of AML patients and extensively studied as a drug discovery target. On the basis of our previous study that ibrutinib (9) exhibited selective and moderate inhibitory activity against FLT3-ITD positive AML cells, through a structure-guided drug design approach, we have discovered a new type II FLT3 kinase inhibitor, compound 14 (CHMFL-FLT3-213), which exhibited highly potent inhibitory effects against FLT3-ITD mutant and associated oncogenic mutations (including FLT3-D835Y/H/V, FLT3-ITD-D835Y/I/N/A/G/Del, and FLT3-ITD-F691L). In the cellular context 14 strongly affected FLT3-ITD mediated signaling pathways and induced apoptosis by arresting cell cycle into G0/G1 phase. In the in vivo studies 14 demonstrated an acceptable bioavailability (F = 19%) and significantly suppressed the tumor growth in MV4-11 cell inoculated xenograft model (15 mg kg–1 day–1, TGI = 97%) without exhibiting obvious toxicity. Compound 14 might be a potential drug candidate for FLT3-ITD positive AML." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.7b00840"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.7b00840">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-14/achre4.2021.54.issue-14/20210720/achre4.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-29/aamick.2021.13.issue-29/20210728/aamick.2021.13.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-14/acncdm.2021.12.issue-14/20210721/acncdm.2021.12.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-7/amlcef.2021.3.issue-7/20210705/amlcef.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-29/acsodf.2021.6.issue-29/20210727/acsodf.2021.6.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-29/ascecg.2021.9.issue-29/20210726/ascecg.2021.9.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-29/ancham.2021.93.issue-29/20210727/ancham.2021.93.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-29/bichaw.2021.60.issue-29/20210727/bichaw.2021.60.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-27/cgeabj.2021.99.issue-27/20210726-01/cgeabj.2021.99.issue-27.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-7/cgdefu.2021.21.issue-7/20210707/cgdefu.2021.21.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-14/esthag.2021.55.issue-14/20210720/esthag.2021.55.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-29/iecred.2021.60.issue-29/20210728/iecred.2021.60.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-14/inocaj.2021.60.issue-14/20210719/inocaj.2021.60.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-29/jafcau.2021.69.issue-29/20210728/jafcau.2021.69.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-29/jacsat.2021.143.issue-29/20210728/jacsat.2021.143.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-7/jamsef.2021.32.issue-7/20210707/jamsef.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-29/langd5.2021.37.issue-29/20210727/langd5.2021.37.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-7/mpohbp.2021.18.issue-7/20210705/mpohbp.2021.18.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.7b00840&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.7b00840&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.7b00840&amp;href=/doi/10.1021/acs.jmedchem.7b00840" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2017</span><span class="cit-fg-volume">, 60</span><span class="cit-fg-issue">, 20</span><span class="cit-fg-pageRange">, 8407-8424</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/60/20" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.7b00819" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.7b00854" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Discovery of 1-(4-(4-Amino-3-(4-(2-morpholinoethoxy)phenyl)-1<i>H</i>-pyrazolo[3,4-<i>d</i>]pyrimidin-1-yl)phenyl)-3-(5-(<i>tert</i>-butyl)isoxazol-3-yl)urea (CHMFL-FLT3-213) as a Highly Potent Type II FLT3 Kinase Inhibitor Capable of Overcoming a Variety of FLT3 Kinase Mutants in FLT3-ITD Positive AML</span></h1><ul class="loa non-jats-loa"><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Aoli++Wang">Aoli Wang</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span><span class="author-xref-symbol "><sup>∇</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Xixiang++Li">Xixiang Li</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span><span class="author-xref-symbol "><sup>∇</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Cheng++Chen">Cheng Chen</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span><span class="author-xref-symbol "><sup>∇</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Hong++Wu">Hong Wu</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span><span class="author-xref-symbol "><sup>∇</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Ziping++Qi">Ziping Qi</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span><span class="author-xref-symbol "><sup>∇</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Chen++Hu">Chen Hu</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span><span class="author-xref-symbol "><sup>∇</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Kailin++Yu">Kailin Yu</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Jiaxin++Wu">Jiaxin Wu</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Juan++Liu">Juan Liu</a></span><span class="author-xref-symbol author-aff-symbol"><sup>∥</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Xiaochuan++Liu">Xiaochuan Liu</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Zhenquan++Hu">Zhenquan Hu</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Wei++Wang">Wei Wang</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Wenliang++Wang">Wenliang Wang</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Wenchao++Wang">Wenchao Wang</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Li++Wang">Li Wang</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Beilei++Wang">Beilei Wang</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Qingwang++Liu">Qingwang Liu</a></span><span class="author-xref-symbol author-aff-symbol"><sup>∥</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Lili++Li">Lili Li</a></span><span class="author-xref-symbol author-aff-symbol"><sup>⊥</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Jian++Ge">Jian Ge</a></span><span class="author-xref-symbol author-aff-symbol"><sup>⊥</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Tao++Ren">Tao Ren</a></span><span class="author-xref-symbol author-aff-symbol"><sup>∥</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Shanchun++Zhang">Shanchun Zhang</a></span><span class="author-xref-symbol author-aff-symbol"><sup>#</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Ruixiang++Xia">Ruixiang Xia</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>⊥</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Jing++Liu">Jing Liu</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span><span class="loa-info-orcid"><a href="http://orcid.org/0000-0001-9513-3591" title="Orcid link"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img></a></span></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Qingsong++Liu">Qingsong Liu</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>∥</sup></span><span class="loa-info-orcid"><a href="http://orcid.org/0000-0002-7829-2547" title="Orcid link"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img></a></span></span></li><span class="comma-separator"></span></ul><div class="affiliations"><div class="view-affs"><span class="title">View Author Information</span><i class="caret-icon icon-angle-down"></i></div><div class="all-aff-infos hlFld-Affiliation"><div class="aff-info" id="aff1"><span class="aff-symbol">†</span> <span class="aff-text">High Magnetic Field Laboratory, Chinese Academy of Sciences, Mailbox 1110, 350 Shushanhu Road, Hefei, Anhui 230031, P. R. China</span></div><div class="aff-info" id="aff2"><span class="aff-symbol">‡</span> <span class="aff-text">CHMFL-HCMTC Target Therapy Joint Laboratory, 350 Shushanhu Road, Hefei, Anhui 230031, P. R. China</span></div><div class="aff-info" id="aff3"><span class="aff-symbol">§</span> <span class="aff-text">University of Science and Technology of China, Hefei, Anhui 230036, P. R. China</span></div><div class="aff-info" id="aff4"><span class="aff-symbol">∥</span> <span class="aff-text">Precision Targeted Therapy Discovery Center, Institute of Technology Innovation, Hefei Institutes of Physical Science, Chinese Academy of Sciences, Hefei, Anhui 230088, P. R. China</span></div><div class="aff-info" id="aff5"><span class="aff-symbol">⊥</span> <span class="aff-text">Department of Hematology, The First Affiliated Hospital of Anhui Medical University, Hefei, Anhui 230022, P. R. China</span></div><div class="aff-info" id="aff6"><span class="aff-symbol">#</span> <span class="aff-text">Hefei Cosource Medicine Technology Co. Ltd., 358 Ganquan Road, Hefei, Anhui 230031, P. R. China</span></div><div class="corresp-info"><strong>*</strong>R.X: phone, 86-551-62923145; e-mail, <a href="/cdn-cgi/l/email-protection#730b010b41434742334245405d101c1e"><span class="__cf_email__" data-cfemail="58202a206a686c6918696e6b763b3735">[email protected]</span></a></div><div class="corresp-info"><strong>*</strong>J.L.: phone, 86-551-65593186; e-mail, <a href="/cdn-cgi/l/email-protection#e78d8e89808b8e92a78f8a818bc98684c98489"><span class="__cf_email__" data-cfemail="bdd7d4d3dad1d4c8fdd5d0dbd193dcde93ded3">[email protected]</span></a></div><div class="corresp-info"><strong>*</strong>Q.L.: phone, 86-551-65595161; e-mail, <a href="/cdn-cgi/l/email-protection#dcadafb0b5a9e5eb9cb4b1bab0f2bdbff2bfb2"><span class="__cf_email__" data-cfemail="e293918e8b97dbd5a28a8f848ecc8381cc818c">[email protected]</span></a></div></div></div><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.7b00840&amp;href=/doi/10.1021%2Facs.jmedchem.7b00840" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2017</span></span><span class="cit-volume">, 60</span><span class="cit-issue">, 20</span><span class="cit-pageRange">, 8407–8424</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">September 28, 2017</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>8 June 2017</li><li><span class="item_label"><b>Published</b> online</span>17 October 2017</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 26 October 2017</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.7b00840" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.7b00840</a></div><div class="article_header-article-copyright"><strong>Copyright © 2017 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D8407%26pageCount%3D18%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DAoli%2BWang%252C%2BXixiang%2BLi%252C%2BCheng%2BChen%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D60%26issueNum%3D20%26contentID%3Dacs.jmedchem.7b00840%26title%3DDiscovery%2Bof%2B1-%25284-%25284-Amino-3-%25284-%25282-morpholinoethoxy%2529phenyl%2529-1H-pyrazolo%255B3%252C4-d%255Dpyrimidin-1-yl%2529phenyl%2529-3-%25285-%2528tert-butyl%2529isoxazol-3-yl%2529urea%2B%2528CHMFL-FLT3-213%2529%2Bas%2Ba%2BHighly%2BPotent%2BType%2BII%2BFLT3%2BKinase%2BInhibitor%2BCapable%2Bof%2BOvercoming%2Ba%2BVariety%2Bof%2BFLT3%2BKinase%2BMutants%2Bin%2BFLT3-ITD%2BPositive%2BAML%26numPages%3D18%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D8424%26publicationDate%3DOctober%2B2017">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.7b00840"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">2868</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">15</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.7b00840" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Discovery of 1-(4-(4-Amino-3-(4-(2-morpholinoethoxy)phenyl)-1H-pyrazolo[3,4-d]pyrimidin-1-yl)phenyl)-3-(5-(tert-butyl)isoxazol-3-yl)urea (CHMFL-FLT3-213) as a Highly Potent Type II FLT3 Kinase Inhibitor Capable of Overcoming a Variety of FLT3 Kinase Mutants in FLT3-ITD Positive AML&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Aoli&quot;,&quot;last_name&quot;:&quot;Wang&quot;},{&quot;first_name&quot;:&quot;Xixiang&quot;,&quot;last_name&quot;:&quot;Li&quot;},{&quot;first_name&quot;:&quot;Cheng&quot;,&quot;last_name&quot;:&quot;Chen&quot;},{&quot;first_name&quot;:&quot;Hong&quot;,&quot;last_name&quot;:&quot;Wu&quot;},{&quot;first_name&quot;:&quot;Ziping&quot;,&quot;last_name&quot;:&quot;Qi&quot;},{&quot;first_name&quot;:&quot;Chen&quot;,&quot;last_name&quot;:&quot;Hu&quot;},{&quot;first_name&quot;:&quot;Kailin&quot;,&quot;last_name&quot;:&quot;Yu&quot;},{&quot;first_name&quot;:&quot;Jiaxin&quot;,&quot;last_name&quot;:&quot;Wu&quot;},{&quot;first_name&quot;:&quot;Juan&quot;,&quot;last_name&quot;:&quot;Liu&quot;},{&quot;first_name&quot;:&quot;Xiaochuan&quot;,&quot;last_name&quot;:&quot;Liu&quot;},{&quot;first_name&quot;:&quot;Zhenquan&quot;,&quot;last_name&quot;:&quot;Hu&quot;},{&quot;first_name&quot;:&quot;Wei&quot;,&quot;last_name&quot;:&quot;Wang&quot;},{&quot;first_name&quot;:&quot;Wenliang&quot;,&quot;last_name&quot;:&quot;Wang&quot;},{&quot;first_name&quot;:&quot;Wenchao&quot;,&quot;last_name&quot;:&quot;Wang&quot;},{&quot;first_name&quot;:&quot;Li&quot;,&quot;last_name&quot;:&quot;Wang&quot;},{&quot;first_name&quot;:&quot;Beilei&quot;,&quot;last_name&quot;:&quot;Wang&quot;},{&quot;first_name&quot;:&quot;Qingwang&quot;,&quot;last_name&quot;:&quot;Liu&quot;},{&quot;first_name&quot;:&quot;Lili&quot;,&quot;last_name&quot;:&quot;Li&quot;},{&quot;first_name&quot;:&quot;Jian&quot;,&quot;last_name&quot;:&quot;Ge&quot;},{&quot;first_name&quot;:&quot;Tao&quot;,&quot;last_name&quot;:&quot;Ren&quot;},{&quot;first_name&quot;:&quot;Shanchun&quot;,&quot;last_name&quot;:&quot;Zhang&quot;},{&quot;first_name&quot;:&quot;Ruixiang&quot;,&quot;last_name&quot;:&quot;Xia&quot;},{&quot;first_name&quot;:&quot;Jing&quot;,&quot;last_name&quot;:&quot;Liu&quot;},{&quot;first_name&quot;:&quot;Qingsong&quot;,&quot;last_name&quot;:&quot;Liu&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2017&quot;,&quot;month&quot;:&quot;10&quot;,&quot;day&quot;:&quot;17&quot;,&quot;issue&quot;:&quot;20&quot;,&quot;volume&quot;:&quot;60&quot;,&quot;pages&quot;:&quot;8407-8424&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.7b00840&quot;},&quot;abstract&quot;:&quot;FLT3-ITD mutant has been observed in about 30% of AML patients and extensively studied as a drug discovery target. On the basis of our previous study that ibrutinib (9) exhibited selective and moderate inhibitory activity against FLT3-ITD positive AML cells, through a structure-guided drug design approach, we have discovered a new type II FLT3 kinase inhibitor, compound 14 (CHMFL-FLT3-213), which exhibited highly potent inhibitory effects against FLT3-ITD mutant and associated oncogenic mutations (including FLT3-D835Y/H/V, FLT3-ITD-D835Y/I/N/A/G/Del, and FLT3-ITD-F691L). In the cellular context 14 strongly affected FLT3-ITD mediated signaling pathways and induced apoptosis by arresting cell cycle into G0/G1 phase. In the in vivo studies 14 demonstrated an acceptable bioavailability (F = 19%) and significantly suppressed the tumor growth in MV4-11 cell inoculated xenograft model (15 mg kg–1 day–1, TGI = 97%) without exhibiting obvious toxicity. Compound 14 might be a potential drug candidate for FLT3-ITD posit&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.7b00840&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.7b00840" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.7b00840&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.7b00840" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.7b00840&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.7b00840" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.7b00840&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.7b00840&amp;href=/doi/10.1021/acs.jmedchem.7b00840" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.7b00840" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.7b00840" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (10 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.7b00840&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.7b00840%26sid%3Dliteratum%253Aachs%26pmid%3D28956923%26genre%3Darticle%26aulast%3DWang%26date%3D2017%26atitle%3DDiscovery%2Bof%2B1-%25284-%25284-Amino-3-%25284-%25282-morpholinoethoxy%2529phenyl%2529-1H-pyrazolo%255B3%252C4-d%255Dpyrimidin-1-yl%2529phenyl%2529-3-%25285-%2528tert-butyl%2529isoxazol-3-yl%2529urea%2B%2528CHMFL-FLT3-213%2529%2Bas%2Ba%2BHighly%2BPotent%2BType%2BII%2BFLT3%2BKinase%2BInhibitor%2BCapable%2Bof%2BOvercoming%2Ba%2BVariety%2Bof%2BFLT3%2BKinase%2BMutants%2Bin%2BFLT3-ITD%2BPositive%2BAML%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D60%26issue%3D20%26spage%3D8407%26epage%3D8424%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (11)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=291439" title="Genetics">Genetics</a>,</li><li><a href="/action/doSearch?ConceptID=291763" title="Noncovalent interactions">Noncovalent interactions</a>,</li><li><a href="/action/doSearch?ConceptID=291711" title="Selectivity">Selectivity</a>,</li><li><a href="/action/doSearch?ConceptID=292153" title="Tumors">Tumors</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/60/20" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-20/jmcmar.2017.60.issue-20/20171026/jmcmar.2017.60.issue-20.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="f_tgr1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-20/acs.jmedchem.7b00840/20171020/images/medium/jm-2017-008406_0014.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-20/acs.jmedchem.7b00840/20171020/images/large/jm-2017-008406_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00840&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">FLT3-ITD mutant has been observed in about 30% of AML patients and extensively studied as a drug discovery target. On the basis of our previous study that ibrutinib (<b>9</b>) exhibited selective and moderate inhibitory activity against FLT3-ITD positive AML cells, through a structure-guided drug design approach, we have discovered a new type II FLT3 kinase inhibitor, compound <b>14</b> (CHMFL-FLT3-213), which exhibited highly potent inhibitory effects against FLT3-ITD mutant and associated oncogenic mutations (including FLT3-D835Y/H/V, FLT3-ITD-D835Y/I/N/A/G/Del, and FLT3-ITD-F691L). In the cellular context <b>14</b> strongly affected FLT3-ITD mediated signaling pathways and induced apoptosis by arresting cell cycle into G0/G1 phase. In the in vivo studies <b>14</b> demonstrated an acceptable bioavailability (<i>F</i> = 19%) and significantly suppressed the tumor growth in MV4-11 cell inoculated xenograft model (15 mg kg<sup>–1</sup> day<sup>–1</sup>, TGI = 97%) without exhibiting obvious toxicity. Compound <b>14</b> might be a potential drug candidate for FLT3-ITD positive AML.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i1">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_21821" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_21821" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Acute myeloid leukemia (AML) is one of the most common and lethal malignancies among children and young adults. Epidemiologic studies show that AML accounts for at least 3% of all human cancers, causing over 250 thousand cancer-related deaths annually. Fms-like tyrosine receptor kinase 3 (FLT3) plays an important role in normal hematopoiesis and leukemogenesis and is expressed in most AML blasts.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> Mutation in the <i>FLT3</i> gene is the most frequent genetic alteration in AML and is well-known as an important driver mutation for the development of myeloid malignancies.<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a></div><div class="NLM_p">The most prevalent FLT3 mutant is the internal tandem duplication alteration (FLT3<i>-</i>ITD).<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> Other important mutations occur in the tyrosine kinase domain (TKD), typically at the activation loop (AL) residue D835.<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> These mutations are present in approximately 30% of all AML patients and lead to a far worse prognosis. Therefore, targeting FLT3 kinase is one of the current major efforts to fight against AML.</div><div class="NLM_p">Over the past decade, many agents have been investigated as FLT3 inhibitors and tested in clinical trials. On the basis of the binding modes, these inhibitors could be classified as type I or type II<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> depending on whether the compound binds to the “DFG-in” active kinase conformation (type I) or the “DFG-out” inactive kinase conformation (type II).<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> As shown in <a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>, compounds <b>1</b> (sunitinib),<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a><b>2</b> (gilteritinib),<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a><b>3</b> (crenolanib),<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> and <b>4</b> (midostaurin)<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> are type I inhibitors, and recently compound <b>4</b> received a breakthrough therapy designation from the FDA for newly diagnosed FLT3-ITD positive AML. Compounds <b>5</b> (pexidartinib),<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a><b>6</b> (ponatinib),<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a><b>7</b> (quizartinib),<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> and <b>8</b> (sorafenib)<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> are type II inhibitors bearing FLT3 kinase activity, and currently they are at different stages of clinical evaluation for AML.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-20/acs.jmedchem.7b00840/20171020/images/medium/jm-2017-008406_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-20/acs.jmedchem.7b00840/20171020/images/large/jm-2017-008406_0001.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Chemical structures of representative FLT3 kinase inhibitors.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-20/acs.jmedchem.7b00840/20171020/images/large/jm-2017-008406_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00840&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">In our previous studies, we found that BTK kinase inhibitor <b>9</b> (ibrutinib), which has been approved for the treatment of mantle cell lymphoma (MCL)<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> and chronic lymphocytic leukemia (CLL),<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> exhibited selective and moderate inhibitory activity against FLT3-ITD mutant positive AML.<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> On the basis of this observation, we have discovered a potent and selective type I FLT3 kinase inhibitor <b>10</b> (CHMFL-FLT3-122) that displayed impressive in vitro and in vivo activities against FLT3-ITD positive AML.<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> However, compound <b>10</b> could not overcome the secondary mutations in the FLT3 kinase domain, e.g., ITD-F691L, which is located at the “gatekeeper” position in FLT3 that is analogous to residues in other kinases that are hotspots for drug resistance to tyrosine kinase inhibitors (TKIs).<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> Therefore, we continue to explore this pharmacophore, and medicinal chemistry efforts led to the discovery of a novel type II FLT3 kinase inhibitor compound <b>14</b> (CHMFL-FLT3-213), which displayed potent in vitro and in vivo activities against FLT3-ITD and a variety of mutants including gatekeeper FLT3-ITD-F691L, FLT3-ITD-D835Y, as well as FLT3-D835Y/H/V mutants (<a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>).</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-20/acs.jmedchem.7b00840/20171020/images/medium/jm-2017-008406_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-20/acs.jmedchem.7b00840/20171020/images/large/jm-2017-008406_0002.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Schematic illustration of discovery of compound <b>14</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-20/acs.jmedchem.7b00840/20171020/images/large/jm-2017-008406_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00840&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_12024" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_12024" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2-1" class="NLM_sec NLM_sec_level_2"><div id="ac_i3" class="anchor-spacer"></div><h3 class="article-section__title" id="_i3"> Design Rationale and Structure–Activity Relationship (SAR) Exploration</h3><div class="NLM_p">We first examined the binding mode of <b>9</b> with FLT3 kinase to understand the structural resistance mechanism. The result showed that the pyrazolopyrimidine core formed two hydrogen bonds with Cys694 and Glu692, respectively. The gatekeeper residue Phe691 was positioned in parallel with the O-linked benzene ring which formed partial π–π interaction and strengthened the binding (<a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>A). However, when Phe691 is mutated to leucine (F691L), the beneficial π–π interaction was lost and steric interaction between the benzene ring and leucine was introduced which weakened the binding (<a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>B). This was confirmed by the observation that the antiproliferative efficacy (GI<sub>50</sub>) of compound <b>9</b> was significantly reduced from 0.12 μM (BaF3-FLT3-ITD cell) to 2.5 μM (BaF3-FLT3-ITD-F691L cell) (<a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>). In order to overcome this gatekeeper mutation induced drug resistance, such steric interaction should be avoided. This made us turn to the type II inhibitors because usually for type II inhibitors an amide or urea bond is required at this position to form the canonical hydrogen bonds with the amino acid residues located in the c-helix and this would shift this moiety away from the interaction with leucine (F691L). We also noticed that the pyrazolopyrimidine could form the hinge binding if it was flipped 180°, although in this case it will only form two hydrogen bonds with Cys694 but not with Cys694 and Glu692. Hence we reasoned that introduction of proper type II binding elements at R<sub>2</sub> position (<a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>) would be able to achieve the binding affinity.</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-20/acs.jmedchem.7b00840/20171020/images/medium/jm-2017-008406_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-20/acs.jmedchem.7b00840/20171020/images/large/jm-2017-008406_0003.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Design rationale of new FLT3 kinase inhibitor. (A) Docking of <b>9</b> into the active conformation of FLT3 wt kinase (homology model built from PDB codes <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4RT7">4RT7</a> and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3LCD">3LCD</a>). (B) Docking of <b>9</b> into the active conformation of FLT3-F691L kinase (site mutation from FLT3 wt homology model). (C) Docking of <b>12</b> into the DFG-out inactive conformation of FLT3 wt kinase (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4XUF">4XUF</a>). (D) Docking of <b>12</b> into the active conformation of FLT3-F691L kinase (site mutation from PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4XUF">4XUF</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-20/acs.jmedchem.7b00840/20171020/images/large/jm-2017-008406_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00840&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. SAR Exploration at R<sub>1</sub>/R<sub>2</sub> Positions by Examination of the Antiproliferation Efficacies against FLT3-ITD/F691L Transformed BaF3 and MV4-11 Cells<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-20/acs.jmedchem.7b00840/20171020/images/medium/jm-2017-008406_0011.gif" alt="" id="GRAPHIC-d136e722-autogenerated" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-20/acs.jmedchem.7b00840/20171020/images/medium/jm-2017-008406_0012.gif" alt="" id="gr11" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-20/acs.jmedchem.7b00840/20171020/images/medium/jm-2017-008406_0013.gif" alt="" id="gr12" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup>Table a</sup><p class="last">All GI<sub>50</sub> values were obtained by triplet testing (±SD).</p></div></div><div></div></div><div class="NLM_p">On the basis of this postulation, we prepared compounds <b>11</b> and <b>12</b> which bear the urea linkage at R<sub>2</sub> (<a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>). Further testing showed that <b>11</b> exhibited weak activity against BaF3-FLT3-ITD cell (GI<sub>50</sub> = 0.96 μM) and MV4-11 cell (GI<sub>50</sub> = 0.65 μM). When changing the cyclohexane ring to benzene ring for the connection of the amide (<b>12</b>), it gained potent antiproliferative efficacy against BaF3-FLT3-ITD (GI<sub>50</sub> = 0.001 μM). In addition, it exhibited good selectivity over parental BaF3 cells (GI<sub>50</sub> > 10 μM) and potent antiproliferative effect against FLT3-ITD positive AML cells MV4-11 (GI<sub>50</sub> = 0.021 μM). Although compound <b>12</b> still could not overcome the FLT3-ITD-F691L mutation, its potent inhibitory activity against FLT3-ITD indicated that type II binding might be achieved, and this encouraged us to study in detail this new class of compounds. Docking of <b>12</b> into the DFG-out inactive conformation of FLT3 kinase (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4XUF">4XUF</a>) showed that the aminopyrimidine moiety formed two hydrogen bonds with Cys694 in the hinge binding area (<a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>C). As expected, the newly introduced urea linkage formed two canonical hydrogen bonds with Glu661 in the c-helix and Asp829 in the DFG motif. The <i>tert</i>-butyl substituted isoxazole occupied the hydrophobic pocket generated by the DFG-motif shift. In this binding mode, the diphenyl ether was directed to the solvent exposed area near the hinge binding area.</div><div class="NLM_p">When the gatekeeper residue Phe691 is mutated to Leu691, the new leucine residue was away from the urea and should not be the hindrance for binding. However, the partial π–π interaction was lost and this could partially explain why compound <b>12</b> did not display inhibitory effect against FLT3-ITD-F691L mutant (<a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>D). In addition, the polar environment formed by Asp698 and Asn701 surrounded the hydrophobic nonpolar diphenyl ether moiety in the hinge binding area, which may also disfavor this binding. Therefore, in order to further gain activity against FLT3-ITD-F691L, we removed one benzene ring from the R<sub>1</sub> moiety (<b>13</b>, <a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>), which significantly improved its activity to the mutant (GI<sub>50</sub> = 0.11 μM for BaF3-FLT3-ITD-F691L cell). In addition, it also increased the activity against FLT3-ITD (GI<sub>50</sub> < 0.0003 μM for BaF3-FLT3-ITD cell and GI<sub>50</sub> = 0.001 μM for MV4-11 cell) and meanwhile kept a good selectivity window to parental BaF3 cell (GI<sub>50</sub> > 10 μM). This result indicated that our postulation of binding analysis was reasonable, and we next further explored the SAR on the basis of this type II binding mode.</div><div class="NLM_p">Given the fact that R<sub>1</sub> moiety (<a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>) is exposed to the solvent area and more hydrophilic group would be preferred, we then installed O-linked morpholine (<b>14</b>) into compound <b>13</b>, which led to potent inhibition activity against FLT3-ITD (GI<sub>50</sub> < 0.0003 μM for BaF3-FLT3-ITD and MV4-11 cells) and FLT3-ITD-F691L (GI<sub>50</sub> = 0.002 μM) and kept good selectivity over parental BaF3 cell (GI<sub>50</sub> > 10 μM). Switching the connection site of the benzene ring at R<sub>1</sub> from para to meta position (<b>15</b>) retained the activity against FLT3-ITD but lost about 40-fold activity against FLT3-ITD-F691L. Switching the connection site of the urea linkage at R<sub>2</sub> from para to meta position (<b>16</b>) resulted in significant activity loss to both FLT3-ITD and F691L mutants. Interestingly, changing the <i>tert</i>-butyl group at R<sub>2</sub> position that occupied the hydrophobic pocket generated by the DFG motif shift to a methyl group (<b>17</b>) caused over 150-fold activity loss against FLT3-ITD. Removal of the <i>tert</i>-butyl group (<b>18</b>) resulted in further activity loss. Replacement of the isoxazole with thiazole (<b>19</b>) retained good inhibitory activity against FLT3-ITD (GI<sub>50</sub> = 0.021 μM) but lost activity to F691L mutant (GI<sub>50</sub> = 5.6 μM). Changing the <i>tert</i>-butyl group to a smaller CF<sub>3</sub> group (<b>20</b>) further decreased activity to FLT3-ITD (GI<sub>50</sub> = 0.17 μM) and completely lost activity to F691L mutant (GI<sub>50</sub> > 10 μM). Replacement of the <i>tert</i>-butylisoxazole to <i>n</i>-butylbenzene (<b>21</b>) also retained good activity against FLT3-ITD (GI<sub>50</sub> = 0.024 μM) but lost activity to F691L mutant (GI<sub>50</sub> = 8.7 μM). Increasing the hydrophobicity by installment of the 1-methyl-2-(trifluoromethyl)benzene (<b>22</b>) enhanced activity to FLT3-ITD (GI<sub>50</sub> = 0.002 μM) and started to gain activity to F691L mutant (GI<sub>50</sub> = 0.63 μM), although it was still much less potent compared to <b>14</b>. A larger hydrophobic moiety 1,2,3-trimethoxybenzene (<b>23</b>) also retained activity to FLT3-ITD (GI<sub>50</sub> = 0.013 μM) but lost activity to F691L mutant (GI<sub>50</sub> = 3.8 μM). Although introduction of a methyl group at the ortho position of urea linkage (<b>24</b>) retained activity to FLT3-ITD (GI<sub>50</sub> = 0.046 μM) but not F691L mutant (GI<sub>50</sub> = 1.4 μM), shifting the methyl group to meta position (<b>25</b>) displayed good activity to both FLT3-ITD (GI<sub>50</sub> = 0.002 μM) and F691L mutants (GI<sub>50</sub> = 0.013 μM). Elongation of the O-linked morpholine with one more carbon at R<sub>1</sub> position (<b>26</b>) retained activity to FLT3-ITD (GI<sub>50</sub> < 0.0003 μM) but lost activity to F691L mutant (GI<sub>50</sub> = 0.77 μM). Changing the morpholine to piperidine (<b>27</b>) resulted in potent inhibition activity against FLT3-ITD (GI<sub>50</sub> < 0.0003 μM) and F691L mutants (GI<sub>50</sub> = 0.023 μM). Installation of the hydrophilic morpholine directly into the para position of the benzene ring at R<sub>1</sub> position (<b>28</b>) or via a carbonyl linkage (<b>29</b>), or a propionamide linkage (<b>30</b>) all retained potent activities to FLT3-ITD but lost potency to F691L mutant. Attachment of hydrophilic groups such as piperazine (<b>31</b>), methylpiperazine (<b>32</b>), ethylpiperazine (<b>33</b>), ethanolpiperazine (<b>34</b>), and ethylpiperidine (<b>35</b>) at R<sub>1</sub> position all resulted in high potencies against both FLT3-ITD and F691L mutants, although they all displayed some activities against parental BaF3 cells (GI<sub>50</sub> values of 1.0–3.3 μM). These results indicated that a polar hydrophilic group was preferred to achieve both FLT3-ITD and gatekeeper F691L mutant activities.</div><div class="NLM_p">Since compound <b>14</b> displayed best potencies against FLT3-ITD and FLT3-ITD-F691L gatekeeper mutants and exhibited best selectivity to parental BaF3 cells, we then further examined <b>14</b>’s activity against a panel of FLT3 mutants transformed BaF3 cells (<a class="ref internalNav" href="#tbl2" aria-label="Table 2">Table 2</a>). Not surprisingly, <b>14</b> displayed potent inhibitory activities against FLT3 wt, FLT3-ITD associated mutants including gatekeeper F691L, D835Y/I/N/A/G/del as well as gain of function mutants including FLT3-D835Y/V/H. Compared to well-established type II FLT3 inhibitor <b>7</b>, <b>14</b> was more potent against FLT3-ITD-F691L/D835Y. Interestingly, neither <b>14</b> nor <b>7</b> was potent to FLT3-K663Q mutant.</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Antiproliferative Effects of Compounds <b>14</b> and <b>7</b> against a Panel of FLT3 wt/Mutants Transformed BaF3 Cells<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="2" align="center">GI<sub>50</sub>, μM</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">cell line</th><th class="colsep0 rowsep0" align="center"><b>14</b></th><th class="colsep0 rowsep0" align="center"><b>7</b></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">parental BaF3</td><td class="colsep0 rowsep0" align="left">>10</td><td class="colsep0 rowsep0" align="left">>10</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">BaF3-TEL-FLT3 wt</td><td class="colsep0 rowsep0" align="left"><0.0003</td><td class="colsep0 rowsep0" align="left"><0.0003</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">BaF3-FLT3-ITD</td><td class="colsep0 rowsep0" align="left"><0.0003</td><td class="colsep0 rowsep0" align="left"><0.0003</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">BaF3-FLT3-ITD-F691L</td><td class="colsep0 rowsep0" align="left">0.002 ± 0.0009</td><td class="colsep0 rowsep0" align="left">0.021 ± 0.002</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">BaF3-FLT3-ITD-D835Y</td><td class="colsep0 rowsep0" align="left">0.002 ± 0.0008</td><td class="colsep0 rowsep0" align="left">0.028 ± 0.002</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">BaF3-FLT3-ITD-D835I</td><td class="colsep0 rowsep0" align="left">0.056 ± 0.008</td><td class="colsep0 rowsep0" align="left">0.056 ± 0.018</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">BaF3-FLT3-ITD-D835N</td><td class="colsep0 rowsep0" align="left">0.001 ± 0.0005</td><td class="colsep0 rowsep0" align="left"><0.0003</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">BaF3-FLT3-ITD-D835G</td><td class="colsep0 rowsep0" align="left"><0.0003</td><td class="colsep0 rowsep0" align="left"><0.0003</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">BaF3-FLT3-ITD-D835A</td><td class="colsep0 rowsep0" align="left">0.0093 ± 0.00006</td><td class="colsep0 rowsep0" align="left"><0.0003</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">BaF3-FLT3-ITD-D835del</td><td class="colsep0 rowsep0" align="left">0.046 ± 0.0032</td><td class="colsep0 rowsep0" align="left">0.012 ± 0.0015</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">BaF3-FLT3-D835Y</td><td class="colsep0 rowsep0" align="left">0.012 ± 0.003</td><td class="colsep0 rowsep0" align="left">0.002 ± 0.0001</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">BaF3-FLT3-D835V</td><td class="colsep0 rowsep0" align="left">0.003 ± 0.0001</td><td class="colsep0 rowsep0" align="left">0.011 ± 0.0005</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">BaF3-FLT3-D835H</td><td class="colsep0 rowsep0" align="left"><0.0003</td><td class="colsep0 rowsep0" align="left"><0.0003</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">BaF3-FLT3-K663Q</td><td class="colsep0 rowsep0" align="left">3.5 ± 0.46</td><td class="colsep0 rowsep0" align="left">0.37 ± 0.02</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup>a</sup><p class="last">All GI<sub>50</sub> values were obtained by triple testing (±SD).</p></div></div></div></div><div id="sec2-2" class="NLM_sec NLM_sec_level_2"><div id="ac_i6" class="anchor-spacer"></div><h3 class="article-section__title" id="_i6"> Compound <b>14</b>’s Binding Mode</h3><div class="NLM_p">In order to better understand compound <b>14</b>’s binding mechanism, we then docked it into FLT3 wt X-ray structure (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4XUF">4XUF</a>)<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> (<a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>A). The modeling results showed that it adopted a typical type II binding mode with FLT3 kinase. The aminopyrimidine formed two hydrogen bonds with Cys694 in the hinge binding area. The urea moiety formed two signature type II binding hydrogen bonds with Gly661 in the c-helix and Asp829 in the DFG motif. The <i>tert</i>-butylisoxazole occupied the hydrophobic pocket generated by the DFG motif shift. In addition, the morpholine moiety that was directed into solvent exposed area formed a hydrogen bond with the Leu616 backbone carbonyl located in the P-loop after being protonated. This may explain why compound <b>14</b> was more potent than <b>12</b> which is because the latter compound lacks such hydrogen bond. In the F691L mutant model (site mutation from PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4XUF">4XUF</a>) compound <b>14</b> adopted a similar type II binding mode (<a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>B). In this binding model, three hydrogen bonds formed by the urea moiety turned the benzene ring away from this position, which generated enough space to accommodate the Leu691 residue. This may explain why compound <b>14</b> was still sensitive to the F691L mutant. The hydrogen bond between the nitrogen atom in the morpholine and Leu616 played critical roles for the binding because compounds like <b>31</b>–<b>35</b> that bore a similar nitrogen atom at this position all displayed potent inhibition activity to both FLT3-ITD and F691L mutants. However, compounds <b>28</b>–<b>30</b> which lack this hydrogen bond formation capability lost activities to F691L mutant. In addition, when a methyl group was introduced to the ortho position of this benzene ring (<b>24</b>), a hindrance interaction between Leu691 or Asp829 disfavored the binding (<a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>C). However, when the methyl group was introduced to the meta position, there was enough space to accommodate it, which explained why compound <b>25</b> was still potent to both FLT3-ITD and F691L mutants (<a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>D). Comparably, compound <b>7</b> adopted a similar type II binding mode except for the hinge region, where the sulfur atom in the imidazobenzothiazole ring forms a weak hydrogen bond with the backbone nitrogen of Cys694. In addition, the weak π–π stacking interactions between Tyr693 in the hinge region and the imidazobenzothiazole ring also increased the binding affinity.<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a></div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-20/acs.jmedchem.7b00840/20171020/images/medium/jm-2017-008406_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-20/acs.jmedchem.7b00840/20171020/images/large/jm-2017-008406_0004.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Binding mode examination of compound <b>14</b>. (A) Docking of <b>14</b> into the DFG-out conformation of FLT3 kinase (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4XUF">4XUF</a>). (B) Docking of <b>14</b> into the homology model of DFG-out conformation of FLT3-F691L mutant (site mutation from PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4XUF">4XUF</a>). (C) Docking of <b>24</b> into the homology model of DFG-out conformation of FLT3-F691L mutant (green and blue colors represent two possible directions of methyl group respectively). (D) Docking of <b>25</b> into the homology model of DFG-out conformation of FLT3-F691L mutant.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-20/acs.jmedchem.7b00840/20171020/images/large/jm-2017-008406_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00840&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2-3" class="NLM_sec NLM_sec_level_2"><div id="ac_i7" class="anchor-spacer"></div><h3 class="article-section__title" id="_i7"> Compound <b>14</b>’s Selectivity Profiling</h3><div class="NLM_p">In order to further understand compound <b>14</b>’s selectivity, we then examined its kinome wide selectivity profile with KINOMEScan technology.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> The results showed that <b>14</b> possessed a good selectivity (<i>S</i> score (1) = 0.04) in a panel of 468 kinases and mutants at 1 μM concentration (<a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>A and <a href="/doi/suppl/10.1021/acs.jmedchem.7b00840/suppl_file/jm7b00840_si_001.pdf" class="ext-link">Supporting Information Table 1</a>). Not surprisingly, <b>14</b> displayed strong binding affinities against FLT3 wt and FLT3 (ITD) as well as a number of mutants including FLT3 D835H/K663Q/R834Q. In addition, it presented strong binding activities to CDK11, CDK7, CDK8, CDKL2, CIT, CSF1R, DDR1, DDR2, FLT4, KIT, LCK, LOK, MKNK2, PDGFRA, PDGFRB, and RET kinases (percent activity remaining less than 1% at 1 μM) (<a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>B). Given the fact that KINOMEScan is a binding assay and may not fully reflect the inhibitory activities, we then used Invitrogen’s Z’Lyte based and ADP-Glo based biochemical activity assays to further confirm these potential targets (<a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>C). The results demonstrated that <b>14</b> inhibited FLT3 wt (IC<sub>50</sub> = 91 nM), FLT3-ITD (IC<sub>50</sub> = 9 nM), CDK11 (IC<sub>50</sub> = 36 nM), CDK8 (IC<sub>50</sub> = 48 nM), DDR1 (IC<sub>50</sub> = 63 nM), DDR2 (IC<sub>50</sub> = 61 nM), LCK (IC<sub>50</sub> = 425 nM), FLT4 (IC<sub>50</sub> = 141 nM), PDGFRα (IC<sub>50</sub> = 426 nM), PDGFRβ (IC<sub>50</sub> = 606 nM), RET (IC<sub>50</sub> = 32 nM), KIT (IC<sub>50</sub> = 83 nM), and CSF1R (IC<sub>50</sub> = 154 nM) kinases, respectively. We also used ADP-Glo based biochemical activity assays to test positive compound <b>7</b>, and the results showed that <b>7</b> inhibited FLT3 wt (IC<sub>50</sub> = 30.4 nM) as well as FLT3-ITD (IC<sub>50</sub> = 16.8 nM). Moreover, <b>7</b>’s selectivity profile has been published,<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> and the results demonstrated that <b>7</b> also showed strong binding constants against DDR1 (<i>K</i><sub>d</sub> = 81 nM), DDR2 (<i>K</i><sub>d</sub> = 690 nM), FLT4 (<i>K</i><sub>d</sub> = 41 nM), PDGFRα (<i>K</i><sub>d</sub> = 11 nM), PDGFRβ (<i>K</i><sub>d</sub> = 7.7 nM), RET (<i>K</i><sub>d</sub> = 9.9 nM), KIT (<i>K</i><sub>d</sub> = 4.8 nM), CSF1R (<i>K</i><sub>d</sub> = 12 nM), and FLT1 (<i>K</i><sub>d</sub> = 41 nM) kinases. We then used the TEL transformed BaF3 system to further confirm the on-target activity and selectivity against other potential off-targets revealed by KINOMEScan binding assay (<a class="ref internalNav" href="#tbl3" aria-label="Table 3">Table 3</a>). The data showed that <b>14</b> could potently inhibit PDGFRα (GI<sub>50</sub> < 0.0003 μM), PDGFRβ (GI<sub>50</sub><b><</b>0.0003 μM), KIT (GI<sub>50</sub> = 0.001 μM) as well as RET (GI<sub>50</sub> = 0.04 μM) kinase transformed BaF3 cells. This is not surprising because cKIT, FLT3, and PDGFR kinases all belong to the type III receptor tyrosine kinase family and the ATP binding pocket of these kinases are highly conserved. In addition, CSF1R, DDR1/2, FLT4, and LCK kinase transformed BaF3 cells were not potently inhibited which indicated that <b>14</b> might not really potently inhibit those kinases in the cell. In comparison, compound <b>7</b> displayed more potent activities than compound <b>14</b> against CSF1R, DDR1, DDR2, FLT4, and LCK kinases. Both of them potently inhibited cKIT and PDGFRα/β kinases besides FLT3-ITD kinase. In addition, compound <b>14</b> was more potent against RET kinase than compound <b>7</b> (<a class="ref internalNav" href="#tbl3" aria-label="Table 3">Table 3</a>).</div><figure id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-20/acs.jmedchem.7b00840/20171020/images/medium/jm-2017-008406_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-20/acs.jmedchem.7b00840/20171020/images/large/jm-2017-008406_0005.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Kinome wide selectivity profiling of compound <b>14</b> with DiscoveRx KINOMEScan technology. Measurements were performed at a concentration of 1 μM of the inhibitor. The affinity was defined with respect to a DMSO control. (A) Treespot demonstration of <b>14</b>’s selectivity in 468 kinase targets. (B) Other targets that demonstrated strong binding to <b>14</b> with a percent control number less than 1. (C) ADP-Glo assay and Invitrogen’s biochemical characterization of <b>14</b>. <sup><i>a</i></sup>The biochemical tests used ADP-Glo assay (Promega, Madison, WI). <sup><i>b</i></sup>The biochemical tests were provided by Invitrogen (Carlsbad, CA, USA). All data were obtained by triplet testing (±SD).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-20/acs.jmedchem.7b00840/20171020/images/large/jm-2017-008406_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00840&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. Antiproliferative Efficacy of Compounds <b>14</b> and <b>7</b> against a Panel of Kinase Transformed BaF3 Cells<a class="ref internalNav" href="#t3fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="2" align="center">GI<sub>50</sub>, μM</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">cell line</th><th class="colsep0 rowsep0" align="center"><b>14</b></th><th class="colsep0 rowsep0" align="center"><b>7</b></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">parental BaF3</td><td class="colsep0 rowsep0" align="left">>10</td><td class="colsep0 rowsep0" align="left">>10</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">BaF3-TEL-CSF1R</td><td class="colsep0 rowsep0" align="left">0.29 ± 0.026</td><td class="colsep0 rowsep0" align="left">0.024 ± 0.003</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">BaF3-TEL-DDR1</td><td class="colsep0 rowsep0" align="left">>10</td><td class="colsep0 rowsep0" align="left">0.12 ± 0.02</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">BaF3- TEL-DDR2</td><td class="colsep0 rowsep0" align="left">6.4 ± 0.23</td><td class="colsep0 rowsep0" align="left">0.11 ± 0.02</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">BaF3-TEL-FLT4</td><td class="colsep0 rowsep0" align="left">2.0 ± 0.056</td><td class="colsep0 rowsep0" align="left">0.27 ± 0.06</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">BaF3-TEL-cKIT</td><td class="colsep0 rowsep0" align="left">0.001 ± 0.0007</td><td class="colsep0 rowsep0" align="left">0.017 ± 0.001</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">BaF3-TEL-LCK</td><td class="colsep0 rowsep0" align="left">3.7 ± 0.1</td><td class="colsep0 rowsep0" align="left">0.4 ± 0.06</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">BaF3-TEL-PDGFRα</td><td class="colsep0 rowsep0" align="left"><0.0003</td><td class="colsep0 rowsep0" align="left"><0.0003</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">BaF3-TEL-PDGFRβ</td><td class="colsep0 rowsep0" align="left"><0.0003</td><td class="colsep0 rowsep0" align="left">0.0044 ± 0.0001</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">BaF3-TEL-RET</td><td class="colsep0 rowsep0" align="left">0.04 ± 0.002</td><td class="colsep0 rowsep0" align="left">0.13 ± 0.01</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t3fn1"><div class="footnote" id="t3fn1"><sup>a</sup><p class="last">All GI<sub>50</sub> values were obtained by triple testing (±SD).</p></div></div></div></div><div id="sec2-4" class="NLM_sec NLM_sec_level_2"><div id="ac_i9" class="anchor-spacer"></div><h3 class="article-section__title" id="_i9"> Compound <b>14</b>’s In Cell Activity Evaluation</h3><div class="NLM_p">We next tested compound <b>14</b> against a panel of established AML cancer cell lines. The results showed that it could selectively and strongly inhibit the proliferation of FLT3-ITD positive AML cell lines such as MOLM13, MOLM14, and MV4-11 (GI<sub>50</sub> values of <0.0003 μM) meanwhile exhibited good selectivity over FLT3 wt expressing cell lines such as U937 (GI<sub>50</sub> = 4.2 μM), SKM-1 (GI<sub>50</sub> = 8.9 μM), NB4 (GI<sub>50</sub> = 5.4 μM), OCI-AML-2 (GI<sub>50</sub> = 8.5 μM), HL-60 (GI<sub>50</sub> = 1.1 μM), as well as CMK (GI<sub>50</sub> = 1.1 μM). The well established type II FLT3 inhibitor compound <b>7</b> also displayed similar potencies against FLT3-ITD positive cell lines, but for FLT3-wt expressing cell lines it bore narrower selectivity window compared to compound <b>14</b> (<a class="ref internalNav" href="#tbl4" aria-label="Table 4">Table 4</a>).</div><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. Antiproliferative Effects of Compounds <b>14</b> and <b>7</b> against a Panel of AML Cancer Cell Lines<a class="ref internalNav" href="#t4fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="2" align="center">GI<sub>50</sub>, μM</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">cell line</th><th class="colsep0 rowsep0" align="center"><b>14</b></th><th class="colsep0 rowsep0" align="center"><b>7</b></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">MOLM-13 (FLT3-ITD)</td><td class="colsep0 rowsep0" align="left"><0.0003</td><td class="colsep0 rowsep0" align="left"><0.0003</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">MOLM-14 (FLT3-ITD)</td><td class="colsep0 rowsep0" align="left"><0.0003</td><td class="colsep0 rowsep0" align="left"><0.0003</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">MV4-11 (FLT3-ITD)</td><td class="colsep0 rowsep0" align="left"><0.0003</td><td class="colsep0 rowsep0" align="left"><0.0003</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">U937 (FLT3 wt)</td><td class="colsep0 rowsep0" align="left">4.2 ± 0.057</td><td class="colsep0 rowsep0" align="left">1.2 ± 0.1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">SKM-1 (FLT3 wt)</td><td class="colsep0 rowsep0" align="left">8.9 ± 0.11</td><td class="colsep0 rowsep0" align="left">1.3 ± 0.058</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">NB4 (FLT3 wt)</td><td class="colsep0 rowsep0" align="left">5.4 ± 0.15</td><td class="colsep0 rowsep0" align="left">7.3 ± 0.15</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">OCI-AML-2 (FLT3 wt)</td><td class="colsep0 rowsep0" align="left">8.5 ± 0.11</td><td class="colsep0 rowsep0" align="left">>10</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">HL-60 (FLT3 wt)</td><td class="colsep0 rowsep0" align="left">1.1 ± 0.057</td><td class="colsep0 rowsep0" align="left">0.13 ± 0.035</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">CMK (FLT3 wt)</td><td class="colsep0 rowsep0" align="left">1.1 ± 0.058</td><td class="colsep0 rowsep0" align="left">0.12 ± 0.0056</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t4fn1"><div class="footnote" id="t4fn1"><sup>a</sup><p class="last">All GI<sub>50</sub> values were obtained by triple testing (±SD).</p></div></div></div><div class="NLM_p">We then examined compound <b>14</b>’s effects on the FLT3 mediated signaling pathways in MOLM-13, MOLM-14, and MV4-11 cells (<a class="ref internalNav" href="#fig6" aria-label="Figure 6">Figure 6</a>A). As the results demonstrated, compound <b>14</b> completely blocked FLT3 autophosphorylation at Tyr589/591 site at the concentration of 1 nM and also remarkably inhibited the downstream signaling mediators pSTAT5 (Tyr694), pERK (Thr202/Tyr204), and pAKT (Thr308/Ser473) (EC<sub>50</sub> less than 10 nM). In addition, the expression of c-Myc was also significantly down-regulated. Compound <b>14</b> could also effectively arrest cell cycle into the G0/G1 phase starting from a concentration of 10 nM in these cells (<a class="ref internalNav" href="#fig6" aria-label="Figure 6">Figure 6</a>B). Furthermore, apparent apoptosis was observed by examining the cleavage of PARP and caspase-3 proteins (<a class="ref internalNav" href="#fig6" aria-label="Figure 6">Figure 6</a>C). Comparably, similar dose-responsive results were observed for positive control compound <b>7</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.7b00840/suppl_file/jm7b00840_si_001.pdf" class="ext-link">Supporting Information Figure 1</a>).</div><figure id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-20/acs.jmedchem.7b00840/20171020/images/medium/jm-2017-008406_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-20/acs.jmedchem.7b00840/20171020/images/large/jm-2017-008406_0006.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Cellular effects of compound <b>14</b> in FLT3-ITD positive AML cell lines. (A) Effects of <b>14</b> on FLT3 mediated signaling pathways. (B) Effects of <b>14</b> on cell cycle progression. (C) Effects of <b>14</b> on apoptosis. All data were obtained by duplicate runs, and only one run was shown as representative.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-20/acs.jmedchem.7b00840/20171020/images/large/jm-2017-008406_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00840&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2-5" class="NLM_sec NLM_sec_level_2"><div id="ac_i11" class="anchor-spacer"></div><h3 class="article-section__title" id="_i11"> In Vivo PK/PD Evaluation</h3><div class="NLM_p">We next evaluated compound <b>14</b> and <b>7</b>’s PK properties in rats following intravenous and oral administration (<a class="ref internalNav" href="#tbl5" aria-label="Table 5">Table 5</a>). Upon oral administration it showed a half-life (<i>T</i><sub>1/2</sub>) of 3.61 h and a bioavailability (<i>F</i>) of 18.8%, which indicated that <b>14</b> was suitable for oral application in the following animal efficacy study. In comparison, compound <b>7</b> exhibited much better PK profile than compound <b>14</b> regarding to the AUC, <i>C</i><sub>max</sub>, and bioavailability.</div><div class="NLM_table-wrap" id="tbl5"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 5. Pharmacokinetic Study of Compounds <b>14</b> and <b>7</b> on Sprague Dawley Rats<a class="ref internalNav" href="#t5fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="rowsep1 colsep0" colspan="3" align="center"><b>14</b></th><th class="rowsep1 colsep0" colspan="3" align="center"><b>7</b></th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">parameter</th><th class="colsep0 rowsep0" align="center">iv (1 mg/mg)</th><th class="colsep0 rowsep0" align="center">po (10 mg/mg)</th><th class="colsep0 rowsep0" align="center">parameter</th><th class="colsep0 rowsep0" align="center">iv (1 mg/mg)</th><th class="colsep0 rowsep0" align="center">po (10 mg/mg)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">AUC<sub>(0–<i>t</i>)</sub> (ng h mL<sup>–1</sup>)</td><td class="colsep0 rowsep0" align="left">519.81 ± 165.64</td><td class="colsep0 rowsep0" align="left">978.32 ± 482.72</td><td class="colsep0 rowsep0" align="left">AUC<sub>(0–<i>t</i>)</sub> (ng h mL<sup>–1</sup>)</td><td class="colsep0 rowsep0" align="left">1928 ± 754</td><td class="colsep0 rowsep0" align="left">14467 ± 6599</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">AUC<sub>(0–∞)</sub> (ng h mL<sup>–1</sup>)</td><td class="colsep0 rowsep0" align="left">524.69 ± 170.77</td><td class="colsep0 rowsep0" align="left">990.719 ± 489.86</td><td class="colsep0 rowsep0" align="left">AUC<sub>(0–∞)</sub> (ng h mL<sup>–1</sup>)</td><td class="colsep0 rowsep0" align="left">1973 ± 785</td><td class="colsep0 rowsep0" align="left">14970 ± 6980</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>C</i><sub>max</sub> (ng/mL)</td><td class="colsep0 rowsep0" align="left">839.39 ± 111.18</td><td class="colsep0 rowsep0" align="left">83.50 ± 50.90</td><td class="colsep0 rowsep0" align="left"><i>C</i><sub>max</sub> (ng/mL)</td><td class="colsep0 rowsep0" align="left">728 ± 117</td><td class="colsep0 rowsep0" align="left">1142 ± 495</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>T</i><sub>max</sub> (h)</td><td class="colsep0 rowsep0" align="left">0.017 ± 0</td><td class="colsep0 rowsep0" align="left">6.66 ± 1.15</td><td class="colsep0 rowsep0" align="left"><i>T</i><sub>max</sub> (h)</td><td class="colsep0 rowsep0" align="left">0.03 ± 0</td><td class="colsep0 rowsep0" align="left">8.00 ± 3.46</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>T</i><sub>1/2</sub> (h)</td><td class="colsep0 rowsep0" align="left">3.34 ± 1.22</td><td class="colsep0 rowsep0" align="left">3.61 ± 0.46</td><td class="colsep0 rowsep0" align="left"><i>T</i><sub>1/2</sub> (h)</td><td class="colsep0 rowsep0" align="left">4.38 ± 0.24</td><td class="colsep0 rowsep0" align="left">3.73 ± 0.15</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>V</i><sub>z</sub> (mL/kg)</td><td class="colsep0 rowsep0" align="left">9.09 ± 0.72</td><td class="colsep0 rowsep0" align="left">60.82 ± 26.87</td><td class="colsep0 rowsep0" align="left"><i>V</i><sub>z</sub> (mL/kg)</td><td class="colsep0 rowsep0" align="left">3532 ± 1244</td><td class="colsep0 rowsep0" align="left">4078 ± 1539</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">CL<sub>z</sub> (mL h<sup>–1</sup> kg<sup>–1</sup>)</td><td class="colsep0 rowsep0" align="left">2.07 ± 0.78</td><td class="colsep0 rowsep0" align="left">11.56 ± 4.45</td><td class="colsep0 rowsep0" align="left">CL<sub>z</sub> (mL h<sup>–1</sup> kg<sup>–1</sup>)</td><td class="colsep0 rowsep0" align="left">561 ± 208</td><td class="colsep0 rowsep0" align="left">756 ± 284</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">MRT<sub>(0–∞)</sub> (h)</td><td class="colsep0 rowsep0" align="left">3.13 ± 0.70</td><td class="colsep0 rowsep0" align="left">7.22 ± 0.12</td><td class="colsep0 rowsep0" align="left">MRT<sub>(0–∞)</sub> (h)</td><td class="colsep0 rowsep0" align="left">5.83 ± 1.47</td><td class="colsep0 rowsep0" align="left">10.11 ± 1.27</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>F</i> (%)</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">18.8</td><td class="colsep0 rowsep0" align="left"><i>F</i> (%)</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">75.9</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t5fn1"><div class="footnote" id="t5fn1"><sup>a</sup><p class="last">All testing data were obtained from three independent mice (±SD).</p></div></div></div><div class="NLM_p">The antitumor efficacy of compound <b>14</b> was tested in MV4-11 cell (which bears exclusive FLT3-ITD mutation genes) inoculated xenograft mouse model. The results demonstrated that dosages of 3.75, 7.5, and 15 mg kg<sup>–1</sup> day<sup>–1</sup> through oral administration did not significantly affect the mice body weight (<a class="ref internalNav" href="#fig7" aria-label="Figure 7">Figure 7</a>A). During 28 days of continuous treatment, <b>14</b> dose-dependently suppressed the MV4-11 tumor progression and a dosage of 15 mg kg<sup>–1</sup> day<sup>–1</sup> exhibited the TGI (tumor growth inhibition) of 97% (<a class="ref internalNav" href="#fig7" aria-label="Figure 7">Figure 7</a>B–D). However, compound <b>7</b> showed better activity at a dosage of 10 mg kg<sup>–1</sup> day<sup>–1</sup> with 100% TGI, which reflected its superior PK profile compared with compound <b>14</b>. Immunohistochemistry stain results demonstrated that <b>14</b> could dose dependently inhibit cancer cell proliferation (Ki-67 stain) and induce apoptosis (TUNEL stain) in the tumor tissues (<a class="ref internalNav" href="#fig7" aria-label="Figure 7">Figure 7</a>E). In addition, immunoblotting analysis of the extracted tumor cells from the tumor tissues after drug treatment showed that FLT3 mediated signaling pathways were potently inhibited, which further confirmed that compound <b>14</b> exhibited its antitumor efficacy through FLT3 on-target effect (<a class="ref internalNav" href="#fig7" aria-label="Figure 7">Figure 7</a>F). Since the clinically advanced FLT3 inhibitor compound <b>7</b> is known to prevent the tumor regrowth upon stopping the treatment, we also examined this trend for compound <b>14</b>. At 25 mg kg<sup>–1</sup> day<sup>–1</sup>, tumors appeared to regrow after stoppage of <b>14</b> treatment. In contrast, the tumors only slightly regrew after day 38 (17 days after last dose) with 50 mg/kg dose of <b>14</b>. Moreover, the 100 mg kg<sup>–1</sup> day<sup>–1</sup> dose of <b>14</b> completely inhibited tumor regrowth in the 50 days of observation (<a class="ref internalNav" href="#fig7" aria-label="Figure 7">Figure 7</a>G).</div><figure id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-20/acs.jmedchem.7b00840/20171020/images/medium/jm-2017-008406_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-20/acs.jmedchem.7b00840/20171020/images/large/jm-2017-008406_0007.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Compound <b>14</b>’s antitumor efficacy in MV4-11 cell (FLT3-ITD) inoculated xenograft mouse model. Female nu/nu mice bearing established MV4-11 tumor xenografts were treated with <b>14</b> at 3.75, 7.5, and 15 mg kg<sup>–1</sup> day<sup>–1</sup> dosage or <b>7</b> at 10 mg kg<sup>–1</sup> day<sup>–1</sup> or vehicle. Daily oral administration was initiated when MV4-11 tumors had reached a size of 200–400 mm<sup>3</sup>. Each group contained 5–7 animals. Data, mean ± SEM mpk, mg kg<sup>–1</sup> day<sup>–1</sup>. (A) Body weight and (B) tumor size measurements from MV4-11 xenograft mice after 28 days compound administration. Initial body weight and tumor size were set as 100%. (C) Representative photographs of tumors in each group after 0, 3.75, 7.5, and 15 mg kg<sup>–1</sup> day<sup>–1</sup><b>14</b> or 10 mg kg<sup>–1</sup> day<sup>–1</sup><b>7</b> treatments. (D) Comparison of the final tumor weights in each group after 28-day treatment period. Numbers in columns indicate the mean tumor weight in each group. (∗∗∗∗) <i>p</i> < 0.0001. (E) Representative micrographs of hematoxylin and eosin (HE), Ki-67, and TUNEL staining of tumor tissues with <b>14</b> treatment groups in comparison with the vehicle treatment group. Note the specific nuclear staining of cells with morphology consistent with proliferation and apoptosis (E, blue arrows). (F) Dose-responsive immunoblotting of the signaling pathway with the cells extracted from tumor tissues of the experimental animal after the <b>14</b> and <b>7</b> treatments. (G) Tumor progression examination after stopping compound <b>14</b> treatment.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-20/acs.jmedchem.7b00840/20171020/images/large/jm-2017-008406_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00840&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i13">Chemistry</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_47225" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_47225" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Compounds <b>14</b>–<b>27</b> were prepared as shown in <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>. The synthesis started from O-esterification between R<sub>1</sub>-substituted alcohol and methanesulfonyl chloride, which generated intermediates <b>36a</b>–<b>c</b>. Then O-alkylation between <b>36a</b>–<b>c</b> and 4- or 3-(4,4,5,5-tetramethyl-1,3,2-dioxaborolan-2-yl)phenol provided intermediates <b>37a</b>–<b>d</b>. Nucleophilic substitution of 3-iodo-1<i>H</i>-pyrazolo[3,4-<i>d</i>]pyrimidin-4-amine with substituted fluoronitrobenzene afforded intermediates <b>38a</b>–<b>d</b>. Suzuki coupling between <b>38a</b>–<b>d</b> and pinacol boronic ester <b>37a</b>–<b>d</b> followed by hydrogenation of the nitro group yielded compounds <b>39a</b>–<b>d</b>, which then reacted with substituted amines to afford the target molecules.</div><figure id="sch1" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-20/acs.jmedchem.7b00840/20171020/images/medium/jm-2017-008406_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-20/acs.jmedchem.7b00840/20171020/images/large/jm-2017-008406_0008.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of Compounds <b>14</b>–<b>27</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-20/acs.jmedchem.7b00840/20171020/images/large/jm-2017-008406_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00840&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) methanesulfonyl chloride, DIEPA, THF, 0 °C to rt, overnight, yield 87.3%; (b) K<sub>2</sub>CO<sub>3</sub>, DMF, 70 °C, 8 h, yield 74.3–75.6%; (c) for <b>38a</b>,<b>c</b>,<b>d</b>, K<sub>2</sub>CO<sub>3</sub>, DMF, 100 °C, overnight; for <b>38b</b>, K<sub>2</sub>CO<sub>3</sub>, DMF, 120 °C, overnight, yield 80.1–82.5%; (d) <b>37a</b>–<b>d</b>, Pd(PPh<sub>3</sub>)<sub>4</sub>, K<sub>2</sub>CO<sub>3</sub>, 1,4-dioxane/H<sub>2</sub>O, 90 °C, 12 h; (e) 10% Pd/C, H<sub>2</sub>, MeOH, rt, 4 h, two-step yield 85.4–87.8%; (f) triphosgene, Et<sub>3</sub>N, CH<sub>2</sub>Cl<sub>2</sub>, −10 to 25 °C, 16 h, yield 48.9–55.3%.</p></p></figure><div class="NLM_p">Similarly, Pd-catalyzed Suzuki coupling between <b>38a</b> and a series of substituted pinacol boronic esters following by hydrogenation reaction generated intermediates <b>40a</b>–<b>h</b>. Then compounds <b>40a</b>–<b>h</b> or <b>41</b> reacted with 5-(<i>tert</i>-butyl)isoxazol-3-amine to yield the desired products as shown in <a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>. Compound <b>31</b> was prepared following the same synthetic procedure except that the Boc group was deprotected under acidic condition as the last step (<a class="ref internalNav" href="#sch3" aria-label="Scheme 3">Scheme 3</a>). Compound <b>34</b> was obtained from nucleophilic substitution of <b>31</b> with 2-iodoethanol.</div><figure id="sch2" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-20/acs.jmedchem.7b00840/20171020/images/medium/jm-2017-008406_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-20/acs.jmedchem.7b00840/20171020/images/large/jm-2017-008406_0009.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of Compounds <b>11</b>–<b>13</b>, <b>28</b>–<b>30</b>, <b>32</b>, <b>33</b> ,and <b>35</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-20/acs.jmedchem.7b00840/20171020/images/large/jm-2017-008406_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00840&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) Pd(PPh<sub>3</sub>)<sub>4</sub>, K<sub>2</sub>CO<sub>3</sub>, 1,4-dioxane/H<sub>2</sub>O, 90 °C, 12 h; (b) 10% Pd/C, H<sub>2</sub>, MeOH, rt, 2 h, two-step yield 84.6–88.2%; (c) 5-(<i>tert</i>-butyl)isoxazol-3-amine, triphosgene, Et<sub>3</sub>N, CH<sub>2</sub>Cl<sub>2</sub>, −10 to 25 °C, 16 h, yield 50.4–70.1%.</p></p></figure><figure id="sch3" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-20/acs.jmedchem.7b00840/20171020/images/medium/jm-2017-008406_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-20/acs.jmedchem.7b00840/20171020/images/large/jm-2017-008406_0010.jpeg" id="gr10" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthesis of Compounds <b>31</b> and <b>34</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-20/acs.jmedchem.7b00840/20171020/images/large/jm-2017-008406_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00840&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) Pd(PPh<sub>3</sub>)<sub>4</sub>, K<sub>2</sub>CO<sub>3</sub>, 1,4-dioxane/H<sub>2</sub>O, 90 °C, 12 h; (b) 10% Pd/C, H<sub>2</sub>, MeOH, rt, 2 h, two-step yield 52.7%; (c) 5-(<i>tert</i>-butyl)isoxazol-3-amine, triphosgene, Et<sub>3</sub>N, CH<sub>2</sub>Cl<sub>2</sub>, −10 to 25 °C, 16 h; (d) 4 N HCl in EtOAc, rt, 2 h, two-step yield 52.7%; (e) 2-iodoethanol, K<sub>2</sub>CO<sub>3</sub>, DMF, rt, 3 h, yield 71.4%.</p></p></figure></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i17">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_17272" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_17272" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">Starting from BTK kinase inhibitor compound <b>9</b>, which adopted a type I binding mode with FLT3 kinase and exhibited selective moderate activity against FLT3-ITD mutant in AML cancer cell lines, through a structure-guided drug design approach, we have discovered compound <b>14</b>, a novel type II FLT3 kinase inhibitor which exhibited high potency against FLT3-ITD mutant and other oncogenic mutations of FLT3 kinase especially drug resistant gatekeeper mutant F691L. Compound <b>14</b> selectively inhibited the growth of FLT3-ITD positive AML cancer cell lines through strong suppression of the FLT3 mediated signaling pathways, arresting of cell cycle, and induction of apoptosis. Compound <b>14</b> also displayed suitable PK properties and strong in vivo antitumor activity. Meanwhile, it is worthy to note that compound <b>14</b> could also potently inhibit cKIT kinase and CDK8/11 kinases. However, simultaneous inhibition of FLT3 kinase and cKIT kinase might induce the myelosuppression toxicity and inhibition of CDK8/11 kinase has the risk of pleiotropic toxicity. Therefore, careful safety attention should be paid when the inhibitor is moved to the clinical trial. In summary, all these reported results suggested that compound <b>14</b> might be a potential drug candidate for FLT3-ITD driven AML.</div></div><div id="sec5" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i18">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_11757" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_11757" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec5-1" class="NLM_sec NLM_sec_level_2"><div id="ac_i19" class="anchor-spacer"></div><h3 class="article-section__title" id="_i19"> Chemistry</h3><div class="NLM_p">All solvents and reagents were used as obtained. <sup>1</sup>H and <sup>13</sup>C NMR spectra were recorded with a Bruker 400 MHz NMR spectrometer and referenced to deuterium dimethyl sulfoxide (DMSO-<i>d</i><sub>6</sub>) or deuterium chloroform (CDCl<sub>3</sub>). Chemical shifts are expressed in ppm. In the NMR tabulation, s indicates singlet; d, doublet; t, triplet; q, quartet; m, multiplet; and br, broad peak. LC/MS experiments were performed on an Agilent 6224 TOF using an ESI source coupled to an Agilent 1260 Infinity HPLC system operating in reverse mode with an Agilent Eclipse Plus C18 1.8 μm, 3.0 mm × 50 mm column. Purification of the final compounds were performed with an Isco CombiFlash Rf+ system (silica flash column 4 g) using a gradient of 0–10% MeOH in DCM over 20 min at a flow rate of 18 mL/min. The purities of all final compounds were above 95%.</div><div class="NLM_p">Compounds <b>11</b>–<b>30</b>, <b>32</b>–<b>33</b>, <b>35</b> were prepared following the synthetic procedure of compound <b>14</b>.</div><div id="sec5-1-1" class="NLM_sec NLM_sec_level_3"><div id="ac_i20" class="anchor-spacer"></div><h4 class="article-section__title" id="_i20"> 1-(4-(4-Amino-3-(4-phenoxyphenyl)-1<i>H</i>-pyrazolo[3,4-<i>d</i>]pyrimidin-1-yl)cyclohexyl)-3-(5-(<i>tert</i>-butyl)isoxazol-3-yl)urea (<b>11</b>)</h4><div class="NLM_p last">Yield = 70.1%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.24 (s, 1H), 8.26 (s, 1H), 7.68 (s, 2H), 7.45 (s, 2H), 7.17 (s, 5H), 6.95 (s, 1H), 6.39 (s, 1H), 4.77 (s, 1H), 3.93 (s, 2H), 2.16 (s, 2H), 1.87 (s, 5H), 1.27 (s, 9H); LC/MS (ESI, <i>m</i>/<i>z</i>) 567.28 [M + H]<sup>+</sup>.</div></div><div id="sec5-1-2" class="NLM_sec NLM_sec_level_3"><div id="ac_i21" class="anchor-spacer"></div><h4 class="article-section__title" id="_i21"> 1-(4-(4-Amino-3-(4-phenoxyphenyl)-1<i>H</i>-pyrazolo[3,4-<i>d</i>]pyrimidin-1-yl)phenyl)-3-(5-(<i>tert</i>-butyl)isoxazol-3-yl)urea (<b>12</b>)</h4><div class="NLM_p last">Yield = 52.5%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.70 (s, 1H), 9.37 (s, 1H), 8.37 (s, 1H), 8.13 (d, <i>J</i> = 8.4 Hz, 2H), 7.77 (d, <i>J</i> = 8.5 Hz, 3H), 7.65 (d, <i>J</i> = 9.0 Hz, 1H), 7.59–7.41 (m, 2H), 7.18 (dd, <i>J</i> = 15.5, 7.9 Hz, 5H), 6.55 (s, 1H), 1.31 (s, 9H); LC/MS (ESI, <i>m</i>/<i>z</i>) 563.23 [M + H]<sup>+</sup>.</div></div><div id="sec5-1-3" class="NLM_sec NLM_sec_level_3"><div id="ac_i22" class="anchor-spacer"></div><h4 class="article-section__title" id="_i22"> 1-(4-(4-Amino-3-phenyl-1<i>H</i>-pyrazolo[3,4-<i>d</i>]pyrimidin-1-yl)phenyl)-3-(5-(<i>tert</i>-butyl)isoxazol-3-yl)urea (<b>13</b>)</h4><div class="NLM_p last">Yield = 53.3%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.58 (s, 1H), 9.04 (s, 1H), 8.39 (s, 1H), 8.14 (s, 2H), 7.77 (s, 2H), 7.62 (d, <i>J</i> = 14.9 Hz, 5H), 6.55 (s, 1H), 1.31 (s, 9H); LC/MS (ESI, <i>m</i>/<i>z</i>) 469.21 [M + H]<sup>+</sup>.</div></div><div id="sec5-1-4" class="NLM_sec NLM_sec_level_3"><div id="ac_i23" class="anchor-spacer"></div><h4 class="article-section__title" id="_i23"> 1-(4-(4-Amino-3-(4-(2-morpholinoethoxy)phenyl)-1<i>H</i>-pyrazolo[3,4-<i>d</i>]pyrimidin-1-yl)phenyl)-3-(5-(<i>tert</i>-butyl)isoxazol-3-yl)urea (<b>14</b>)</h4><div class="NLM_p last">A solution of <b>39a</b> (50 mg, 0.12 mmol) and Et<sub>3</sub>N (14 mg, 0.14 mmol) in DCM (2 mL) was added dropwise to a cooled solution (−10 °C) of triphosgene (11 mg, 0.04 mmol) in DCM (2 mL). The mixture was stirred for 1 h at room temperature followed by addition of a solution of 5-(<i>tert</i>-butyl)isoxazol-3-amine (16 mg, 0.12 mmol) and Et<sub>3</sub>N (14 mg, 0.14 mmol) in DCM (2 mL). The mixture was then stirred for 16 h at room temperature. The resulting mixture was concentrated and purified with the CombiFlash system (0–10% MeOH in DCM) to afford title compound <b>14</b> (35 mg, 50.7%) as a white solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.55 (s, 1H), 8.99 (s, 1H), 8.36 (s, 1H), 8.13 (s, 2H), 7.65 (s, 4H), 7.14 (s, 2H), 6.54 (s, 1H), 4.17 (s, 2H), 3.59 (s, 4H), 2.73 (s, 2H), 2.50 (s, 4H), 1.30 (s, 9H); <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 180.69, 159.60, 158.83, 156.88, 154.43, 151.82, 145.46, 137.53, 134.00, 130.25, 125.12, 122.09, 119.34, 115.67, 99.03, 92.95, 66.66, 65.98, 57.45, 54.10, 32.95, 28.83; LC/MS (ESI, <i>m</i>/<i>z</i>) 598.28 [M + H]<sup>+</sup>.</div></div><div id="sec5-1-5" class="NLM_sec NLM_sec_level_3"><div id="ac_i24" class="anchor-spacer"></div><h4 class="article-section__title" id="_i24"> 1-(4-(4-Amino-3-(3-(2-morpholinoethoxy)phenyl)-1<i>H</i>-pyrazolo[3,4-<i>d</i>]pyrimidin-1-yl)phenyl)-3-(5-(<i>tert</i>-butyl)isoxazol-3-yl)urea (<b>15</b>)</h4><div class="NLM_p last">Yield = 48.9%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.71 (s, 1H), 9.35 (s, 1H), 8.38 (s, 1H), 8.12 (s, 2H), 7.65 (s, 2H), 7.50 (s, 1H), 7.32 (s, 2H), 7.13 (s, 1H), 6.55 (s, 1H), 4.24 (s, 2H), 3.63 (s, 4H), 2.85 (s, 2H), 2.61 (s, 4H), 1.31 (s, 9H); LC/MS (ESI, <i>m</i>/<i>z</i>) 598.28 [M + H]<sup>+</sup>.</div></div><div id="sec5-1-6" class="NLM_sec NLM_sec_level_3"><div id="ac_i25" class="anchor-spacer"></div><h4 class="article-section__title" id="_i25"> 1-(3-(4-Amino-3-(4-(2-morpholinoethoxy)phenyl)-1<i>H</i>-pyrazolo[3,4-<i>d</i>]pyrimidin-1-yl)phenyl)-3-(5-(<i>tert</i>-butyl)isoxazol-3-yl)urea (<b>16</b>)</h4><div class="NLM_p last">Yield = 54.6%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.64 (s, 1H), 9.44 (s, 1H), 8.38 (s, 2H), 7.96 (s, 1H), 7.69 (s, 2H), 7.44 (s, 2H), 7.19 (s, 2H), 6.55 (s, 1H), 4.25 (s, 2H), 3.66 (s, 4H), 2.83 (s, 2H), 2.63 (s, 4H), 1.30 (s, 9H); LC/MS (ESI, <i>m</i>/<i>z</i>) 598.28 [M + H]<sup>+</sup>.</div></div><div id="sec5-1-7" class="NLM_sec NLM_sec_level_3"><div id="ac_i26" class="anchor-spacer"></div><h4 class="article-section__title" id="_i26"> 1-(4-(4-Amino-3-(4-(2-morpholinoethoxy)phenyl)-1<i>H</i>-pyrazolo[3,4-<i>d</i>]pyrimidin-1-yl)phenyl)-3-(5-methylisoxazol-3-yl)urea (<b>17</b>)</h4><div class="NLM_p last">Yield = 51.3%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.72 (s, 1H), 9.53 (s, 1H), 8.37 (s, 1H), 8.24 (s, 2H), 8.12 (s, 2H), 7.69 (s, 2H), 7.18 (s, 2H), 6.59 (s, 1H), 4.26 (s, 2H), 3.66 (s, 4H), 2.90 (s, 2H), 2.66 (s, 4H), 2.38 (s, 3H); LC/MS (ESI, <i>m</i>/<i>z</i>) 556.24 [M + H]<sup>+</sup>.</div></div><div id="sec5-1-8" class="NLM_sec NLM_sec_level_3"><div id="ac_i27" class="anchor-spacer"></div><h4 class="article-section__title" id="_i27"> 1-(4-(4-Amino-3-(4-(2-morpholinoethoxy)phenyl)-1<i>H</i>-pyrazolo[3,4-<i>d</i>]pyrimidin-1-yl)phenyl)-3-(isoxazol-3-yl)urea (<b>18</b>)</h4><div class="NLM_p last">Yield = 54.2%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.99 (s, 1H), 9.80 (s, 1H), 8.75 (s, 1H), 8.37 (s, 1H), 8.12 (s, 2H), 7.66 (s, 2H), 7.44 (s, 2H), 7.20 (s, 2H), 6.89 (s, 1H), 4.48 (s, 2H), 3.85 (s, 4H), 2.81 (s, 2H), 2.62 (s, 4H); LC/MS (ESI, <i>m</i>/<i>z</i>) 542.22 [M + H]<sup>+</sup>.</div></div><div id="sec5-1-9" class="NLM_sec NLM_sec_level_3"><div id="ac_i28" class="anchor-spacer"></div><h4 class="article-section__title" id="_i28"> 1-(4-(4-Amino-3-(4-(2-morpholinoethoxy)phenyl)-1<i>H</i>-pyrazolo[3,4-<i>d</i>]pyrimidin-1-yl)phenyl)-3-(4-(<i>tert</i>-butyl)thiazol-2-yl)urea (<b>19</b>)</h4><div class="NLM_p last">Yield = 48.1%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.71 (s, 1H), 8.37 (s, 1H), 8.14 (s, 2H), 7.67 (s, 4H), 7.17 (s, 3H), 6.65 (s, 1H), 4.22 (s, 2H), 3.62 (s, 4H), 2.80 (s, 2H), 2.50 (s, 4H), 1.26 (s, 9H); LC/MS (ESI, <i>m</i>/<i>z</i>) 614.26 [M + H]<sup>+</sup>.</div></div><div id="sec5-1-10" class="NLM_sec NLM_sec_level_3"><div id="ac_i29" class="anchor-spacer"></div><h4 class="article-section__title" id="_i29"> 1-(4-(4-Amino-3-(4-(2-morpholinoethoxy)phenyl)-1<i>H</i>-pyrazolo[3,4-<i>d</i>]pyrimidin-1-yl)phenyl)-3-(4-(trifluoromethyl)thiazol-2-yl)urea (<b>20</b>)</h4><div class="NLM_p last">Yield = 48.9%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.17 (s, 1H), 8.38 (s, 1H), 8.17 (s, 2H), 7.87 (s, 1H), 7.70 (s, 4H), 7.20 (s, 3H), 4.36 (s, 2H), 3.75 (s, 4H), 3.13 (s, 2H), 2.90 (s, 4H); LC/MS (ESI, <i>m</i>/<i>z</i>) 626.19 [M + H]<sup>+</sup>.</div></div><div id="sec5-1-11" class="NLM_sec NLM_sec_level_3"><div id="ac_i30" class="anchor-spacer"></div><h4 class="article-section__title" id="_i30"> 1-(4-(4-Amino-3-(4-(2-morpholinoethoxy)phenyl)-1<i>H</i>-pyrazolo[3,4-<i>d</i>]pyrimidin-1-yl)phenyl)-3-(4-butylphenyl)urea (<b>21</b>)</h4><div class="NLM_p last">Yield = 50.5%. <sup>1</sup>H NMR (400 MHz, DMSO) δ 9.89 (s, 1H), 9.66 (s, 1H), 8.35 (s, 1H), 8.04 (s, 2H), 7.68–7.49 (m, 6H), 7.18 (s, 2H), 7.08 (s, 2H), 4.36 (s, 2H), 3.75 (s, 4H), 3.40 (s, 5H), 3.16 (s, 1H), 2.50 (s, 2H),1.52 (s, 2H), 1.25 (s, 2H), 0.89 (s, 3H); LC/MS (ESI, <i>m</i>/<i>z</i>) 607.31 [M + H]<sup>+</sup>.</div></div><div id="sec5-1-12" class="NLM_sec NLM_sec_level_3"><div id="ac_i31" class="anchor-spacer"></div><h4 class="article-section__title" id="_i31"> 1-(4-(4-Amino-3-(4-(2-morpholinoethoxy)phenyl)-1<i>H</i>-pyrazolo[3,4-<i>d</i>]pyrimidin-1-yl)phenyl)-3-(4-methyl-3-(trifluoromethyl)phenyl)urea (<b>22</b>)</h4><div class="NLM_p last">Yield = 47.5%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.03 (s, 1H), 9.85 (s, 1H), 8.35 (s, 1H), 8.11 (s, 2H), 7.95 (s, 1H), 7.66 (s, 4H), 7.54 (s, 1H), 7.20 (s, 3H), 4.23 (s, 2H), 3.64 (s, 4H), 2.86 (s, 2H), 2.62 (s, 4H), 2.36 (s, 3H); LC/MS (ESI, <i>m</i>/<i>z</i>) 633.25 [M + H]<sup>+</sup>.</div></div><div id="sec5-1-13" class="NLM_sec NLM_sec_level_3"><div id="ac_i32" class="anchor-spacer"></div><h4 class="article-section__title" id="_i32"> 1-(4-(4-Amino-3-(4-(2-morpholinoethoxy)phenyl)-1<i>H</i>-pyrazolo[3,4-<i>d</i>]pyrimidin-1-yl)phenyl)-3-(3,4,5-trimethoxyphenyl)urea (<b>23</b>)</h4><div class="NLM_p last">Yield = 55.3%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.33 (s, 1H), 9.17 (s, 1H), 8.36 (s, 1H), 8.22 (s, 2H), 8.08 (s, 2H), 7.69 (s, 2H), 7.18 (s, 2H), 6.84 (s, 2H), 4.26 (s, 2H), 3.77 (s, 6H), 3.64 (d, <i>J</i> = 13.6 Hz, 4H), 2.90 (s, 2H), 2.66 (s, 4H); LC/MS (ESI, <i>m</i>/<i>z</i>) 641.28 [M + H]<sup>+</sup>.</div></div><div id="sec5-1-14" class="NLM_sec NLM_sec_level_3"><div id="ac_i33" class="anchor-spacer"></div><h4 class="article-section__title" id="_i33"> 1-(4-(4-Amino-3-(4-(2-morpholinoethoxy)phenyl)-1<i>H</i>-pyrazolo[3,4-<i>d</i>]pyrimidin-1-yl)-2-methylphenyl)-3-(5-(<i>tert</i>-butyl)isoxazol-3-yl)urea (<b>24</b>)</h4><div class="NLM_p last">Yield = 51.8%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.09 (s, 1H), 8.59 (s, 1H), 8.37 (s, 1H), 8.02 (s, 3H), 7.69 (s, 2H), 7.17 (s, 2H), 6.51 (s, 1H), 4.24 (s, 2H), 3.65 (s, 4H), 2.86 (s, 2H), 2.62 (s, 4H), 2.36 (s, 3H), 1.31 (s, 9H); LC/MS (ESI, <i>m</i>/<i>z</i>) 612.30 [M + H]<sup>+</sup>.</div></div><div id="sec5-1-15" class="NLM_sec NLM_sec_level_3"><div id="ac_i34" class="anchor-spacer"></div><h4 class="article-section__title" id="_i34"> 1-(4-(4-Amino-3-(4-(2-morpholinoethoxy)phenyl)-1<i>H</i>-pyrazolo[3,4-<i>d</i>]pyrimidin-1-yl)-3-methylphenyl)-3-(5-(<i>tert</i>-butyl)isoxazol-3-yl)urea (<b>25</b>)</h4><div class="NLM_p last">Yield = 51.4%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.77 (s, 1H), 9.42 (s, 1H), 8.24 (s, 1H), 7.68 (s, 2H), 7.55 (s, 1H), 7.46 (s, 1H), 7.35 (s, 1H), 7.17 (s, 2H), 6.56 (s, 1H), 4.33 (s, 2H), 3.73 (s, 4H), 3.10 (s, 2H), 2.85 (s, 4H), 2.09 (s, 3H), 1.29 (s, 9H); LC/MS (ESI, <i>m</i>/<i>z</i>) 612.30 [M + H]<sup>+</sup>.</div></div><div id="sec5-1-16" class="NLM_sec NLM_sec_level_3"><div id="ac_i35" class="anchor-spacer"></div><h4 class="article-section__title" id="_i35"> 1-(4-(4-Amino-3-(4-(3-morpholinopropoxy)phenyl)-1<i>H</i>-pyrazolo[3,4-<i>d</i>]pyrimidin-1-yl)phenyl)-3-(5-(<i>tert</i>-butyl)isoxazol-3-yl)urea (<b>26</b>)</h4><div class="NLM_p last">Yield = 52.7%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.70 (s, 1H), 9.34 (s, 1H), 8.37 (s, 1H), 8.12 (s, 2H), 7.71 (s, 2H), 7.65 (s, 2H), 7.17 (s, 2H), 6.55 (s, 1H), 4.17 (s, 2H), 3.86 (s, 4H), 3.21 (s, 4H), 2.21 (s, 2H), 1.29 (s, 9H); LC/MS (ESI, <i>m</i>/<i>z</i>) 612.30 [M + H]<sup>+</sup>.</div></div><div id="sec5-1-17" class="NLM_sec NLM_sec_level_3"><div id="ac_i36" class="anchor-spacer"></div><h4 class="article-section__title" id="_i36"> 1-(4-(4-Amino-3-(4-(2-(piperidin-1-yl)ethoxy)phenyl)-1<i>H</i>-pyrazolo[3,4-<i>d</i>]pyrimidin-1-yl)phenyl)-3-(5-(<i>tert</i>-butyl)isoxazol-3-yl)urea (<b>27</b>)</h4><div class="NLM_p last">Yield = 53.4%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.31 (s, 1H), 10.06 (s, 1H), 8.35 (s, 1H), 8.09 (s, 2H), 7.70–7.49 (m, 4H), 7.21 (s, 2H), 6.55 (s, 1H), 4.56 (s, 2H), 3.49–3.40 (m, 6H), 1.83 (s, 4H), 1.55 (s, 2H), 1.29 (s, 9H); LC/MS (ESI, <i>m</i>/<i>z</i>) 596.30 [M + H]<sup>+</sup>.</div></div><div id="sec5-1-18" class="NLM_sec NLM_sec_level_3"><div id="ac_i37" class="anchor-spacer"></div><h4 class="article-section__title" id="_i37"> 1-(4-(4-Amino-3-(4-morpholinophenyl)-1<i>H</i>-pyrazolo[3,4-<i>d</i>]pyrimidin-1-yl)phenyl)-3-(5-(<i>tert</i>-butyl)isoxazol-3-yl)urea (<b>28</b>)</h4><div class="NLM_p last">Yield = 50.6%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.75 (s, 1H), 9.50 (s, 1H), 8.36 (s, 1H), 8.13 (s, 2H), 7.64 (s, 3H), 7.16 (s, 3H), 6.56 (s, 1H), 3.79 (s, 3H), 3.33 (s, 2H), 3.25 (s, 3H), 1.32 (s, 9H); LC/MS (ESI, <i>m</i>/<i>z</i>) 554.26 [M + H]<sup>+</sup>.</div></div><div id="sec5-1-19" class="NLM_sec NLM_sec_level_3"><div id="ac_i38" class="anchor-spacer"></div><h4 class="article-section__title" id="_i38"> 1-(4-(4-Amino-3-(4-(morpholine-4-carbonyl)phenyl)-1<i>H</i>-pyrazolo[3,4-<i>d</i>]pyrimidin-1-yl)phenyl)-3-(5-(<i>tert</i>-butyl)isoxazol-3-yl)urea (<b>29</b>)</h4><div class="NLM_p last">Yield = 52.4%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.83 (s, 1H), 9.41 (s, 1H), 8.39 (s, 1H), 8.11 (s, 2H), 7.82 (s, 2H), 7.65 (s, 2H), 7.60 (s, 2H), 6.56 (s, 1H), 6.36 (s, 1H), 3.65–3.48 (m, 8H), 1.29 (s, 9H); LC/MS (ESI, <i>m</i>/<i>z</i>) 582.25 [M + H]<sup>+</sup>.</div></div><div id="sec5-1-20" class="NLM_sec NLM_sec_level_3"><div id="ac_i39" class="anchor-spacer"></div><h4 class="article-section__title" id="_i39"> 1-(4-(4-Amino-3-(4-(2-morpholino-2-oxoethyl)phenyl)-1<i>H</i>-pyrazolo[3,4-<i>d</i>]pyrimidin-1-yl)phenyl)-3-(5-(<i>tert</i>-butyl)isoxazol-3-yl)urea (<b>30</b>)</h4><div class="NLM_p last">Yield = 51.5%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.90 (s, 2H), 8.38 (s, 1H), 8.11 (s, 2H), 7.72 (s, 2H), 7.66 (s, 2H), 7.45 (s, 2H), 6.56 (s, 1H), 3.84 (s, 2H), 3.55–3.50 (m, 8H), 1.31 (s, 9H); LC/MS (ESI, <i>m</i>/<i>z</i>) 596.27 [M + H]<sup>+</sup>.</div></div><div id="sec5-1-21" class="NLM_sec NLM_sec_level_3"><div id="ac_i40" class="anchor-spacer"></div><h4 class="article-section__title" id="_i40"> 1-(4-(4-Amino-3-(4-(piperazin-1-yl)phenyl)-1<i>H</i>-pyrazolo[3,4-<i>d</i>]pyrimidin-1-yl)phenyl)-3-(5-(<i>tert</i>-butyl)isoxazol-3-yl)urea (<b>31</b>)</h4><div class="NLM_p last">A solution of <b>42</b> (100 mg, 0.20 mmol) and Et<sub>3</sub>N (24 mg, 0.24 mmol) in DCM (2 mL) was added dropwise to a cooled solution (−10 °C) of triphosgene (18.2 mg, 0.06 mmol) in DCM (4 mL). The mixture was stirred for 1 h at room temperature followed by addition of a solution of 5-(<i>tert</i>-butyl)isoxazol-3-amine (24 mg, 0.24 mmol) and Et<sub>3</sub>N (24 mg, 0.24 mmol) in DCM (4 mL). The mixture was then stirred for 16 h at room temperature. The resulting mixture was concentrated and purified with the CombiFlash system (0–10% MeOH in DCM) to afford the desired Boc-protected intermediate <b>42</b> (61 mg) as an off-white solid. Then, to the solid in EtOAc (2 mL) was added 4 N HCl in EtOAc (4 mL). The reaction mixture was stirred at room temperature for 1 h. After complete conversion of the starting material, excess EtOAc was removed under vacuum. The residue was diluted in EtOAc and water. The water layer was basified with 2 N NaHCO<sub>3</sub> solution and extracted with EtOAc (3 × 30 mL). The organic layers were then washed with water followed by brine. The organic layers were dried over sodium sulfate, filtered, concentrated, and purified with the CombiFlash system (0–10% MeOH in DCM) to afford compound <b>31</b> (27 mg, 51.9%) as a white solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.90 (s, 2H), 8.36 (s, 1H), 8.11 (s, 2H), 7.65 (s, 3H), 7.21 (s, 3H), 6.56 (s, 1H), 3.55 (s, 4H), 3.24 (s, 3H), 3.12 (s, 2H), 1.31 (s, 9H); LC/MS (ESI, <i>m</i>/<i>z</i>) 533.27 [M + H]<sup>+</sup>.</div></div><div id="sec5-1-22" class="NLM_sec NLM_sec_level_3"><div id="ac_i41" class="anchor-spacer"></div><h4 class="article-section__title" id="_i41"> 1-(4-(4-Amino-3-(4-(4-methylpiperazin-1-yl)phenyl)-1<i>H</i>-pyrazolo[3,4-<i>d</i>]pyrimidin-1-yl)phenyl)-3-(5-(<i>tert</i>-butyl)isoxazol-3-yl)urea (<b>32</b>)</h4><div class="NLM_p last">Yield = 50.7%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.94 (s, 1H), 8.55 (s, 1H), 8.10 (s, 1H), 7.65 (s, 3H), 7.42 (s, 3H), 7.21 (s, H), 6.55 (s, 1H), 3.60 (s, 3H), 3.27 (s, 3H), 2.76 (s, 2H), 2.51 (s, 3H) 1.30 (s, 9H); LC/MS (ESI, <i>m</i>/<i>z</i>) 567.29 [M + H]<sup>+</sup>.</div></div><div id="sec5-1-23" class="NLM_sec NLM_sec_level_3"><div id="ac_i42" class="anchor-spacer"></div><h4 class="article-section__title" id="_i42"> 1-(4-(4-Amino-3-(4-(4-ethylpiperazin-1-yl)phenyl)-1<i>H</i>-pyrazolo[3,4-<i>d</i>]pyrimidin-1-yl)phenyl)-3-(5-(<i>tert</i>-butyl)isoxazol-3-yl)urea (<b>33</b>)</h4><div class="NLM_p last">Yield = 54.1%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.86 (s, 1H), 9.78 (s, 1H), 8.56 (s, 2H), 8.36 (s, 1H), 8.12 (s, 2H), 7.65 (s, 2H), 7.22 (s, 2H), 6.55 (s, 1H), 3.62 (s, 4H), 3.29 (s, 4H), 3.13 (s, 2H), 1.31 (s, 12H); LC/MS (ESI, <i>m</i>/<i>z</i>) 581.31 [M + H]<sup>+</sup>.</div></div><div id="sec5-1-24" class="NLM_sec NLM_sec_level_3"><div id="ac_i43" class="anchor-spacer"></div><h4 class="article-section__title" id="_i43"> 1-(4-(4-Amino-3-(4-(4-(2-hydroxyethyl)piperazin-1-yl)phenyl)-1<i>H</i>-pyrazolo[3,4-<i>d</i>]pyrimidin-1-yl)phenyl)-3-(5-(<i>tert</i>-butyl)isoxazol-3-yl)urea (<b>34</b>)</h4><div class="NLM_p last">To a solution of <b>31</b> (20 mg, 0.03 mmol) in DMF (2 mL) were added K<sub>2</sub>CO<sub>3</sub> (10 mg, 0.06 mmol) and 2-iodoethanol (6.5 mg, 0.03 mmol). The resulting mixture was stirred at room temperature for 3 h and then diluted with EtOAc and washed with water followed by brine. The organic layers were dried over sodium sulfate, filtered, concentrated, and purified with the CombiFlash system (0–10% MeOH in DCM) to afford title compound <b>34</b> (15 mg, 71.4%) as a white solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.76 (s, 1H), 9.54 (s, 1H), 8.36 (s, 1H), 8.12 (s, 2H), 7.65 (s, 4H), 7.20 (s, 3H), 6.55 (s, 1H), 3.77 (s, 2H), 3.34 (s, 6H), 3.10 (s, 4H), 1.32 (s, 9H); LC/MS (ESI, <i>m</i>/<i>z</i>) 597.30 [M + H]<sup>+</sup>.</div></div><div id="sec5-1-25" class="NLM_sec NLM_sec_level_3"><div id="ac_i44" class="anchor-spacer"></div><h4 class="article-section__title" id="_i44"> 1-(4-(4-Amino-3-(4-(1-ethylpiperidin-4-yl)phenyl)-1<i>H</i>-pyrazolo[3,4-<i>d</i>]pyrimidin-1-yl)phenyl)-3-(5-(<i>tert</i>-butyl)isoxazol-3-yl)urea (<b>35</b>)</h4><div class="NLM_p last">Yield = 50.4%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.62 (s, 1H), 9.19 (s, 1H), 8.38 (s, 1H), 8.14 (s, 2H), 7.74 (s, 2H), 7.66 (s, 2H), 7.49 (s, 2H), 6.54 (s, 1H), 2.77 (s, 3H), 1.94 (s, 4H), 1.31 (s, 13H) 1.16 (s, 3H); LC/MS (ESI, <i>m</i>/<i>z</i>) 580.31 [M + H]<sup>+</sup>.</div></div><div id="sec5-1-26" class="NLM_sec NLM_sec_level_3"><div id="ac_i45" class="anchor-spacer"></div><h4 class="article-section__title" id="_i45"> 2-Morpholinoethyl Methanesulfonate (<b>36a</b>)</h4><div class="NLM_p">To a solution of 2-morpholinoethanol (5 g, 38.1 mmol) in THF (30 mL) were added DIEPA (5.9 g, 45.8 mmol) and methanesulfonyl chloride (4.8 g, 41.9 mol) at 0 °C. The resulting mixture was stirred overnight at room temperature. Then it was diluted with EtOAc and washed with water followed by brine. The organic layers were dried over sodium sulfate, filtered, and concentrated to provide <b>36a</b> as a light yellow oil (6.9 g, 87.3%), which was used directly without further purification.</div><div class="NLM_p last">Compounds <b>36b</b>–<b>c</b> were prepared following the synthetic procedure of <b>36a</b>.</div></div><div id="sec5-1-27" class="NLM_sec NLM_sec_level_3"><div id="ac_i46" class="anchor-spacer"></div><h4 class="article-section__title" id="_i46"> 4-(2-(4-(4,4,5,5-Tetramethyl-1,3,2-dioxaborolan-2-yl)phenoxy)ethyl)morpholine (<b>37a</b>)</h4><div class="NLM_p">A mixture of <b>36a</b> (6.9 g, 33.0 mmol), 4-(4,4,5,5-tetramethyl-1,3,2-dioxaborolan-2-yl)phenol (8.7 g, 39.6 mmol), and K<sub>2</sub>CO<sub>3</sub> (9.1 g, 66.0 mmol) in DMF (80 mL) was stirred at 70 °C for 8 h. The resulting mixture was diluted with 500 mL of EtOAc and then washed with water (3 × 80 mL) followed by brine. The organic layers were dried over sodium sulfate, filtered, concentrated, and purified by silica gel column chromatography (eluting with 0–2% MeOH in DCM) to afford <b>37a</b> (8.1 g, 74.3%) as a light purple solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 7.60 (s, 2H), 6.93 (s, 2H), 4.11 (s, 2H), 3.57 (s, 4H), 2.89 (s, 1H), 2.74–2.69 (m, 3H), 1.28 (s, 12H); LC/MS (ESI, <i>m</i>/<i>z</i>) 334.21 [M + H]<sup>+</sup>.</div><div class="NLM_p last">Compounds <b>37b</b>–<b>d</b> were prepared following the synthetic procedure of <b>37a</b>.</div></div><div id="sec5-1-28" class="NLM_sec NLM_sec_level_3"><div id="ac_i47" class="anchor-spacer"></div><h4 class="article-section__title" id="_i47"> 4-(3-(4-(4,4,5,5-Tetramethyl-1,3,2-dioxaborolan-2-yl)phenoxy)propyl)morpholine (<b>37b</b>)</h4><div class="NLM_p last">Yield = 73.5%. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.76 (s, 2H), 6.91 (s, 2H), 4.07 (s, 2H), 3.74 (s, 4H), 2.54–2.49 (m, 6H), 1.99 (s, 2H), 1.35 (s, 12H); LC/MS (ESI, <i>m</i>/<i>z</i>) 338.23 [M + H]<sup>+</sup>.</div></div><div id="sec5-1-29" class="NLM_sec NLM_sec_level_3"><div id="ac_i48" class="anchor-spacer"></div><h4 class="article-section__title" id="_i48"> 1-(2-(4-(4,4,5,5-Tetramethyl-1,3,2-dioxaborolan-2-yl)phenoxy)ethyl)piperidine (<b>37c</b>)</h4><div class="NLM_p last">Yield = 75.6%. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.71 (s, 2H), 6.87 (s, 2H), 4.28 (s, 2H), 3.01 (s, 2H), 2.77 (s, 4H), 1.76 (s, 4H), 1.50 (s, 2H), 1.30 (s, 12H); LC/MS (ESI, <i>m</i>/<i>z</i>) 332.23 [M + H]<sup>+</sup>.</div></div><div id="sec5-1-30" class="NLM_sec NLM_sec_level_3"><div id="ac_i49" class="anchor-spacer"></div><h4 class="article-section__title" id="_i49"> 4-(2-(3-(4,4,5,5-Tetramethyl-1,3,2-dioxaborolan-2-yl)phenoxy)ethyl)morpholine (<b>37d</b>)</h4><div class="NLM_p last">Yield = 74.7%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 7.27–7.05 (m, 4H), 4.07 (s, 2H), 3.57 (s, 6H), 2.95–2.59 (m, 4H), 1.27 (s, 12H); LC/MS (ESI, <i>m</i>/<i>z</i>) 334.21 [M + H]<sup>+</sup>.</div></div><div id="sec5-1-31" class="NLM_sec NLM_sec_level_3"><div id="ac_i50" class="anchor-spacer"></div><h4 class="article-section__title" id="_i50"> 3-Iodo-1-(4-nitrophenyl)-1<i>H</i>-pyrazolo[3,4-<i>d</i>]pyrimidin-4-amine (<b>38a</b>)</h4><div class="NLM_p">To a solution of 3-iodo-1<i>H</i>-pyrazolo[3,4-<i>d</i>]pyrimidin-4-amine (10.0 g, 38.3 mmol) in DMF (100 mL) were added K<sub>2</sub>CO<sub>3</sub> (15.8 g, 114.9 mmol) and 1-fluoro-4-nitrobenzene (5.9 g, 42.1 mmol). After the resulting mixture was stirred overnight at 100 °C under N<sub>2</sub> atmosphere, the reaction mixture was cooled to rt before pouring into water (500 mL). The precipitate was collected, washed with water, and dried to provide <b>38a</b> (11.8 g, 80.8%) as a light yellow solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.50 (s, 2H), 8.39 (s, 2H), 8.20 (s, 1H); LC/MS (ESI, <i>m</i>/<i>z</i>) 382.97 [M + H]<sup>+</sup>.</div><div class="NLM_p last">Compounds <b>38b</b>–<b>d</b> were prepared following the synthetic procedure of <b>38a</b>.</div></div><div id="sec5-1-32" class="NLM_sec NLM_sec_level_3"><div id="ac_i51" class="anchor-spacer"></div><h4 class="article-section__title" id="_i51"> 3-Iodo-1-(3-nitrophenyl)-1<i>H</i>-pyrazolo[3,4-<i>d</i>]pyrimidin-4-amine (<b>38b</b>)</h4><div class="NLM_p last">Yield = 81.6%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.06 (s, 1H), 8.60 (s, 1H), 8.39 (s, 1H), 8.17 (s, 1H), 7.83 (s, 1H); LC/MS (ESI, <i>m</i>/<i>z</i>) 382.97 [M + H]<sup>+</sup>.</div></div><div id="sec5-1-33" class="NLM_sec NLM_sec_level_3"><div id="ac_i52" class="anchor-spacer"></div><h4 class="article-section__title" id="_i52"> 3-Iodo-1-(3-methyl-4-nitrophenyl)-1<i>H</i>-pyrazolo[3,4-<i>d</i>]pyrimidin-4-amine (<b>38c</b>)</h4><div class="NLM_p last">Yield = 80.1%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.37 (s, 1H), 8.28 (d, <i>J</i> = 12.1 Hz, 2H), 8.19 (s, 1H), 2.62 (s, 3H); LC/MS (ESI, <i>m</i>/<i>z</i>) 396.99 [M + H]<sup>+</sup>.</div></div><div id="sec5-1-34" class="NLM_sec NLM_sec_level_3"><div id="ac_i53" class="anchor-spacer"></div><h4 class="article-section__title" id="_i53"> 3-Iodo-1-(2-methyl-4-nitrophenyl)-1<i>H</i>-pyrazolo[3,4-<i>d</i>]pyrimidin-4-amine (<b>38d</b>)</h4><div class="NLM_p last">Yield = 82.5%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.36 (s, 1H), 8.24 (s, 2H), 7.73 (s, 1H), 2.28 (s, 3H); LC/MS (ESI, <i>m</i>/<i>z</i>) 396.99 [M + H]<sup>+</sup>.</div></div><div id="sec5-1-35" class="NLM_sec NLM_sec_level_3"><div id="ac_i54" class="anchor-spacer"></div><h4 class="article-section__title" id="_i54"> 1-(4-Aminophenyl)-3-(4-(2-morpholinoethoxy)phenyl)-1<i>H</i>-pyrazolo[3,4-<i>d</i>]pyrimidin-4-amine (<b>39a</b>)</h4><div class="NLM_p">To a mixture of <b>37a</b> (7.0 g, 20.0 mmol) and 1,4-dioxane/H<sub>2</sub>O (50 mL, v/v, 5:1) were added <b>38a</b> (7.3 g, 19.1 mmol), K<sub>2</sub>CO<sub>3</sub> (5.3 g, 38.2 mmol), and Pd(PPh<sub>3</sub>)<sub>4</sub> (1.1 g, 0.9 mmol). The reaction mixture was placed into an oil bath preheated to 90 °C with stirring at this temperature for 12 h under argon. Then the reaction mixture was cooled to rt before pouring into water (200 mL). The precipitate was collected, washed with water, MeOH, and dried to provide the nitrobenzene intermediate as a yellow solid (7.1 g). Then to the solid (7.0 g, 15.1 mmol) in MeOH (50 mL) was added Pd/C (1.0 g, 10%) under H<sub>2</sub> atmosphere. The resulting mixture was stirred at rt for 4 h and then diluted with 100 mL of DCM. The soild was filtered, and the filtrate was evaporated. The residue was purified by silica gel column chromatography (eluting with 0–4% MeOH in DCM) to provide <b>39a</b> (5.8 g, 87.8%) as an off-white solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.29 (s, 1H), 7.66 (s, 4H), 7.16 (s, 2H), 6.70 (s, 2H), 5.37 (s, 2H), 4.38 (s, 2H), 3.78 (s, 4H), 3.40 (s, 2H), 3.16 (s, 2H), 2.96 (s, 2H); LC/MS (ESI, <i>m</i>/<i>z</i>) 432.21 [M + H]<sup>+</sup>.</div><div class="NLM_p last">Compounds <b>39b</b>–<b>d</b>, <b>40a</b>–<b>h</b>, and <b>42</b> were prepared following the synthetic procedure of <b>39a</b>.</div></div><div id="sec5-1-36" class="NLM_sec NLM_sec_level_3"><div id="ac_i55" class="anchor-spacer"></div><h4 class="article-section__title" id="_i55"> 1-(3-Aminophenyl)-3-(4-(2-morpholinoethoxy)phenyl)-1<i>H</i>-pyrazolo[3,4-<i>d</i>]pyrimidin-4-amine (<b>39b</b>)</h4><div class="NLM_p last">Yield = 86.3%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.34 (s, 1H), 7.68 (s, 2H), 7.42 (s, 4H), 7.17 (s, 2H), 5.39 (s, 2H), 4.39 (s, 2H), 3.78 (s, 4H), 3.45 (s, 2H), 2.96 (s, 4H); LC/MS (ESI, <i>m</i>/<i>z</i>) 432.21 [M + H]<sup>+</sup>.</div></div><div id="sec5-1-37" class="NLM_sec NLM_sec_level_3"><div id="ac_i56" class="anchor-spacer"></div><h4 class="article-section__title" id="_i56"> 1-(4-Amino-3-methylphenyl)-3-(4-(2-morpholinoethoxy)phenyl)-1<i>H</i>-pyrazolo[3,4-<i>d</i>]pyrimidin-4-amine (<b>39c</b>)</h4><div class="NLM_p last">Yield = 85.4%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.29 (s, 1H), 7.66–7.59 (m, 5H), 7.16 (s, 2H), 6.75 (s, 1H), 5.11 (s, 2H), 4.34 (s, 4H), 3.73 (s, 15H), 3.45 (s, 2H), 2.85 (s, 4H), 2.14 (s, 3H); LC/MS (ESI, <i>m</i>/<i>z</i>) 446.23 [M + H]<sup>+</sup>.</div></div><div id="sec5-1-38" class="NLM_sec NLM_sec_level_3"><div id="ac_i57" class="anchor-spacer"></div><h4 class="article-section__title" id="_i57"> 1-(4-Amino-2-methylphenyl)-3-(4-(2-morpholinoethoxy)phenyl)-1<i>H</i>-pyrazolo[3,4-<i>d</i>]pyrimidin-4-amine (<b>39d</b>)</h4><div class="NLM_p last">Yield = 86.7%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.19 (s, 1H), 7.64 (s, 2H), 7.15 (s, 2H), 6.99 (s, 1H), 6.55 (s, 2H), 5.36 (s, 2H), 4.35 (s, 2H), 3.75 (s, 2H), 3.37 (s, 2H), 3.15–2.89 (m, 4H), 1.89 (s, 3H); LC/MS (ESI, <i>m</i>/<i>z</i>) 446.23 [M + H]<sup>+</sup>.</div></div><div id="sec5-1-39" class="NLM_sec NLM_sec_level_3"><div id="ac_i58" class="anchor-spacer"></div><h4 class="article-section__title" id="_i58"> 1-(4-Aminophenyl)-3-(4-phenoxyphenyl)-1<i>H</i>-pyrazolo[3,4-<i>d</i>]pyrimidin-4-amine (<b>40a</b>)</h4><div class="NLM_p last">Yield = 84.6%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.30 (s, 1H), 7.72 (s, 4H), 7.44 (s, 4H), 7.16 (s, 5H), 5.32 (s, 2H); LC/MS (ESI, <i>m</i>/<i>z</i>) 395.16 [M + H]<sup>+</sup>.</div></div><div id="sec5-1-40" class="NLM_sec NLM_sec_level_3"><div id="ac_i59" class="anchor-spacer"></div><h4 class="article-section__title" id="_i59"> 1-(4-Aminophenyl)-3-phenyl-1<i>H</i>-pyrazolo[3,4-<i>d</i>]pyrimidin-4-amine (<b>40b</b>)</h4><div class="NLM_p last">Yield = 85.8%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.31 (s, 1H), 7.75–7.70 (m, 5H), 7.58–7.54 (m, 4H), 5.39 (s, 2H); LC/MS (ESI, <i>m</i>/<i>z</i>) 303.13 [M + H]<sup>+</sup>.</div></div><div id="sec5-1-41" class="NLM_sec NLM_sec_level_3"><div id="ac_i60" class="anchor-spacer"></div><h4 class="article-section__title" id="_i60"> 1-(4-Aminophenyl)-3-(4-morpholinophenyl)-1<i>H</i>-pyrazolo[3,4-<i>d</i>]pyrimidin-4-amine (<b>40c</b>)</h4><div class="NLM_p last">Yield = 87.2%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.28 (s, 1H), 7.68 (s, 2H), 7.59 (s, 2H), 7.12 (s, 2H), 6.70 (s, 2H), 5.39 (s, 2H), 3.76 (s, 4H), 3.21 (s, 4H); LC/MS (ESI, <i>m</i>/<i>z</i>) 388.18 [M + H]<sup>+</sup>.</div></div><div id="sec5-1-42" class="NLM_sec NLM_sec_level_3"><div id="ac_i61" class="anchor-spacer"></div><h4 class="article-section__title" id="_i61"> (4-(4-Amino-1-(4-aminophenyl)-1<i>H</i>-pyrazolo[3,4-<i>d</i>]pyrimidin-3-yl)phenyl)(morpholino)methanone (<b>40d</b>)</h4><div class="NLM_p last">Yield = 86.9%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.31 (s, 1H), 7.79 (s, 2H), 7.69 (s, 2H), 7.59 (s, 2H), 6.72 (s, 2H), 5.42 (s, 2H), 3.64 (s, 4H), 3.39 (s, 4H); LC/MS (ESI, <i>m</i>/<i>z</i>) 416.18 [M + H]<sup>+</sup>.</div></div><div id="sec5-1-43" class="NLM_sec NLM_sec_level_3"><div id="ac_i62" class="anchor-spacer"></div><h4 class="article-section__title" id="_i62"> 2-(4-(4-Amino-1-(4-aminophenyl)-1<i>H</i>-pyrazolo[3,4-<i>d</i>]pyrimidin-3-yl)phenyl)-1-morpholinoethanone (<b>40e</b>)</h4><div class="NLM_p last">Yield = 85.1%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.30 (s, 1H), 7.69 (s, 2H), 7.60 (s, 2H), 7.48 (s, 2H), 7.35 (s, 2H), 6.72 (s, 2H), 5.51 (s, 2H), 3.83 (s, 2H), 3.54–3.48 (m, 8H); LC/MS (ESI, <i>m</i>/<i>z</i>) 430.19 [M + H]<sup>+</sup>.</div></div><div id="sec5-1-44" class="NLM_sec NLM_sec_level_3"><div id="ac_i63" class="anchor-spacer"></div><h4 class="article-section__title" id="_i63"> 1-(4-Aminophenyl)-3-(4-(4-methylpiperazin-1-yl)phenyl)-1<i>H</i>-pyrazolo[3,4-<i>d</i>]pyrimidin-4-amine (<b>40f</b>)</h4><div class="NLM_p last">Yield = 87.3%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.28 (s, 1H), 7.61 (s, 2H), 7.50 (s, 2H), 7.19 (s, 2H), 7.04 (s, 2H), 6.70 (s, 2H), 5.40 (s, 2H), 3.84 (s, 2H), 3.26–3.12 (m, 6H), 2.78 (s, 3H); LC/MS (ESI, <i>m</i>/<i>z</i>) 401.22 [M + H]<sup>+</sup>.</div></div><div id="sec5-1-45" class="NLM_sec NLM_sec_level_3"><div id="ac_i64" class="anchor-spacer"></div><h4 class="article-section__title" id="_i64"> 1-(4-Aminophenyl)-3-(4-(4-ethylpiperazin-1-yl)phenyl)-1<i>H</i>-pyrazolo[3,4-<i>d</i>]pyrimidin-4-amine (<b>40g</b>)</h4><div class="NLM_p last">Yield = 84.9%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.28 (s, 1H), 7.68–7.61 (m, 4H), 7.16 (s, 2H), 6.70 (s, 2H), 5.32 (s, 2H), 3.76–3.43 (m, 4H), 2.93 (s, 4H), 2.51 (s, 2H), 1.21 (s, 3H); LC/MS (ESI, <i>m</i>/<i>z</i>) 415.23 [M + H]<sup>+</sup>.</div></div><div id="sec5-1-46" class="NLM_sec NLM_sec_level_3"><div id="ac_i65" class="anchor-spacer"></div><h4 class="article-section__title" id="_i65"> 1-(4-Aminophenyl)-3-(4-(1-ethylpiperidin-4-yl)phenyl)-1<i>H</i>-pyrazolo[3,4-<i>d</i>]pyrimidin-4-amine (<b>40h</b>)</h4><div class="NLM_p">Yield = 88.2%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.31 (s, 1H), 7.69 (s, 4H), 7.44 (s, 2H), 6.70 (s, 2H), 5.31 (s, 2H), 3.17 (s, 4H), 2.66–2.56 (m, 2H), 2.25 (s, 2H), 1.85 (s, 4H), 1.10 (s, 3H); LC/MS (ESI, <i>m</i>/<i>z</i>) 415.23 [M + H]<sup>+</sup>.</div><div class="NLM_p last">Compound <b>41</b> was prepared following previously reported procedures.<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a></div></div><div id="sec5-1-47" class="NLM_sec NLM_sec_level_3"><div id="ac_i66" class="anchor-spacer"></div><h4 class="article-section__title" id="_i66"> <i>tert</i>-Butyl 4-(4-(4-Amino-1-(4-aminophenyl)-1<i>H</i>-pyrazolo[3,4-<i>d</i>]pyrimidin-3-yl)phenyl)piperazine-1-carboxylate (<b>42</b>)</h4><div class="NLM_p last">Yield = 86.7%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.28 (s, 1H), 7.68 (s, 2H), 7.59 (s, 2H), 7.13 (s, 2H), 6.71 (s, 2H), 5.39 (s, 2H), 3.48 (s, 4H), 3.23 (s, 4H), 1.42 (s, 9H); LC/MS (ESI, <i>m</i>/<i>z</i>) 487.25 [M + H]<sup>+</sup>.</div></div></div><div id="sec5-2" class="NLM_sec NLM_sec_level_2"><div id="ac_i67" class="anchor-spacer"></div><h3 class="article-section__title" id="_i67"> Antibodies and Chemicals</h3><div class="NLM_p">The following antibodies were purchased from Cell Signaling Technology (Danvers, MA): FLT3 (8F2) rabbit mAb (no. 3462), Phospho-FLT3 (Tyr589/591) (30D4) rabbit mAb (no. 3464), NF-κB p65 (D14E12) XP rabbit mAb (no. 8242), Phospho-NF-κB p65 (Ser536) (93H1) rabbit mAb (no. 3033), Stat5 antibody (no. 9363), Phospho-Stat5 (Tyr694) (C71E5) rabbit mAb (no. 9314), c-Myc (D84C12) XP rabbit mAb (no. 5605), Akt (pan) (C67E7) rabbit mAb (no. 4691), Phospho-Akt (Ser473) (D9E) XP rabbit mAb (no. 4060), Phospho-Akt (Thr308) (244F9) rabbit mAb (no. 4056), p44/42 MAPK (Erk1/2) (137F5) rabbit mAb (no. 4695), Phospho-p44/42 MAPK (Erk1/2) (Thr202/Tyr204) (D13.14.4E) XP rabbit mAb (no. 4370),GAPDH (D16H11) XP rabbit mAb. Antibodies were used at 1:1000. Cells were lysed for 30 min in lysis buffer supplemented with protease/phosphatase inhibitor cocktail (Cell Signaling Technology). Lysates were cleared by centrifugation at 13 000<i>g</i> at 4 °C for 10 min, and protein concentrations were determined by BCA. Lysates were subjected to electrophoresis through 10% or 15% gel and immobilized on the nitrocellulose membranes.</div><div class="NLM_p last">Compound <b>7</b> (AC220) was purchased from Shanghai Haoyuan Chemexpress Inc. (Shanghai, China).</div></div><div id="sec5-3" class="NLM_sec NLM_sec_level_2"><div id="ac_i68" class="anchor-spacer"></div><h3 class="article-section__title" id="_i68"> TEL-Isogenic Cell Generation</h3><div class="NLM_p last">Retroviral constructs for Ba/F3-TEL-FLT3 and Ba/F3-FLT3 mutants were made based on the pMSCVpuro (Clontech) backbone as described in the literature.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> For TEL-fusion vectors, the first 1 kb of human TEL gene with an artificial myristoylation sequence (MGCGCSSHPEDD) was cloned into pMSCVpuro retroviral vector, followed by a 3xFLAG tag sequence and a stop codon. Then the kinase domain coding sequences of FLT3 variants were inserted in-frame between TEL and 3xFLAG sequences. For full-length expression vectors, the coding sequences of FLT3 variants were directly cloned in pMSCVpuro vector with a 3xFLAG tag at the C-terminal end. All mutations were performed using the QuikChange site-directed mutagenesis kit (Stratagene) following the manufacturer’s instructions. Retrovirus was made using the same method described above and was used to infect Ba/F3 cells. After puromycin selection, the IL-3 concentration in the medium was gradually withdrawn until cells were able to grow in the absence of IL-3.</div></div><div id="sec5-4" class="NLM_sec NLM_sec_level_2"><div id="ac_i69" class="anchor-spacer"></div><h3 class="article-section__title" id="_i69"> FLT3 wt/FLT3-ITD Protein Purification</h3><div class="NLM_p last">The sequences of FLT3 wt and FLT3-ITD were amplified from the cDNA of HL60 and MV4-11 cell lines. The cytoplasmic fragment 564–993 with His tag was cloned into baculovirus expression vector pFASTHT-A. The recombinant bacmid was transfected into SF9 by Cellfectin (Invitrogen). High titer viral stocks were obtained by two rounds of amplification of the virus. The protein was expressed by infecting SF9 cells with high titer viral stocks for 48 h. Cells were harvested and resuspended in lysis buffer (50 mM Tris, pH 7.5, 150 mM NaCl, and 1 mM PMSF). The cells were lysed by ultrasonication, and the cell debris was removed by ultracentrifugation. The supernatant was incubated with Ni-affinity beads (GE). The beads were then washed by lysis buffer containing 50–250 mM imidazole. The elute was loaded to Superdex 75. The protein was concentrated to 1 mg/mL, and aliquots were frozen and stored at −80 °C. The protein was used for ADP-Glo assay.</div></div><div id="sec5-5" class="NLM_sec NLM_sec_level_2"><div id="ac_i70" class="anchor-spacer"></div><h3 class="article-section__title" id="_i70"> Kinase Biochemical Assay</h3><div class="NLM_p last">The ADP-Glo kinase assay (Promega, Madison, WI) was used to screen compound <b>14</b> for its BTK, FLT3-WT, FLT3-ITD, FLT4, CDK11, CDK7, CDK8, CSF1R, DDR1, DDR2, KIT, LCK, MKNK2, PDGFRα, PDGFRβ, and RET inhibition effects, as well as compound <b>7</b> for FLT3-WT and FLT3-ITD inhibition effects. The kinase reaction system contains 4.95 μL of BTK (3 ng/μL), c-KIT (5 ng/μL), FLT3 (10 ng/μL), FLT3-ITD (10 ng/μL), PDGFRα (2.5 ng/μL), PDGFRβ (1.25 ng/μL), and RET (5 ng/μL), 0.55 μL of serially diluted drug, and 5.5 μL of EGFR substrate poly (4:1 Glu, Tyr) peptide (0.2 μg/μL) (Promega, Madison, WI) with 40 μM ATP (Promega, Madison, WI). The reaction in each tube was started immediately by adding ATP and kept going for an hour under 37 °C. After the tube was cooled for 5 min at room temperature, 5 μL solvent reactions were carried out in a 384-well plate. Then 5 μL of ADP-Glo reagent was added into each well to stop the reaction and consume the remaining ADP within 40 min. At the end, 10 μL of kinase detection reagent was added into the well and incubated for 30 min to produce a luminescence signal. Luminescence signal was measured with an automated plate reader (Envision, PE, USA), and the dose–response curve was fitted using Prism 5.0 (GraphPad Software Inc., San Diego, CA). The biochemical tests of FLT4, CDK11, CDK7, CDK8, CSF1R, DDR1, DDR2, KIT, LCK, MKNK2, PDGFRα, PDGFRβ, and RET were provided by Invitrogen (Carlsbad, CA, USA).</div></div><div id="sec5-6" class="NLM_sec NLM_sec_level_2"><div id="ac_i71" class="anchor-spacer"></div><h3 class="article-section__title" id="_i71"> Cell Culture and Proliferation Study</h3><div class="NLM_p">The human AML cell lines MV4-11, U937 were purchased from the American Type Culture Collection (ATCC) (Manassas, VA, USA). The FLT3-wt cell lines SKM-1, OCI-AML-2, HL-60, CMK were purchased from Cobioer Biosciences CO., Ltd. (Nanjing, China). MOLM-13, MOLM14, and NB4 cell lines were provided by Dr. Scott Armstrong, Dana Farber Cancer Institute (DFCI), Boston, MA. MV4-11 was cultured in IMDM media (Corning, USA) with 10% FBS and supplemented with 2% <span class="smallcaps smallerCapital">l</span>-glutamine and 1% pen/strep. MOLM-13, MOLM-14, U937, SKM-1, HL-60, CMK, and FLT3 mutant isogenic BaF3 cells lines were cultured in RPMI 1640 media (Corning, USA) with 10% fetal bovine serum (FBS) and supplemented with 2% <span class="smallcaps smallerCapital">l</span>-glutamine 1% penicillin/streptomycin. OCI-AML-2 was cultured in α-MEM media (Corning, USA) with 20% FBS and supplemented with 2% <span class="smallcaps smallerCapital">l</span>-glutamine and 1% pen/strep. All cell lines were maintained in culture media at 37 °C with 5% CO<sub>2</sub>.</div><div class="NLM_p last">Cells were grown in 96-well culture plates (3000/well). The compounds of various concentrations were added into the plates. Cell proliferation was determined after treatment with compounds for 72 h. Cell viability was measured using the CellTiter-Glo assay (Promega, USA) according to the manufacturer’s instructions, and luminescence was measured in a multilabel reader (Envision, PerkinElmer, USA). Data were normalized to control groups (DMSO) and represented by the mean of three independent measurements with standard error of <20%. GI<sub>50</sub> values were calculated using Prism 5.0 (GraphPad Software, San Diego, CA).</div></div><div id="sec5-7" class="NLM_sec NLM_sec_level_2"><div id="ac_i72" class="anchor-spacer"></div><h3 class="article-section__title" id="_i72"> Signaling Pathway Study</h3><div class="NLM_p last">MOLM13, MOLM14, MV4-11 cells were treated with DMSO, serially diluted compound <b>14</b>, 0.1 μM compound <b>7</b> (AC220) for 4 h. Cells were then washed in PBS and lysed in cell lysis buffer. FLT3, Phospho-FLT3(Tyr589/591), STAT5, Phospho-STAT5(Tyr694), AKT, Phospho-AKT Ser473, Phospho-AKT ERK, Phosphop44/42MAPK(Erk1/2) (Thr202/Tyr204), C-Myc, and GAPDH antibody (Cell Signaling Technology) were used for immunoblotting.</div></div><div id="sec5-8" class="NLM_sec NLM_sec_level_2"><div id="ac_i73" class="anchor-spacer"></div><h3 class="article-section__title" id="_i73"> Apoptosis Effect Examination</h3><div class="NLM_p last">MOLM13, MOLM14, MV4-11 cells were treated with DMSO, serially diluted compound <b>14</b>, 0.1 μM compound <b>7</b> (AC220) for indicated periods. Cells were then washed in PBS and lysed in cell lysis buffer. PARP, caspase-3, GAPDH antibody (Cell Signaling Technology) were used for immunoblotting.</div></div><div id="sec5-9" class="NLM_sec NLM_sec_level_2"><div id="ac_i74" class="anchor-spacer"></div><h3 class="article-section__title" id="_i74"> Cell Cycle Analysis</h3><div class="NLM_p last">MOLM13, MOLM14, MV4-11 cells were treated with DMSO, serially diluted compound <b>14</b>, 0.1 μM compound <b>7</b> (AC220) for indicated periods. The cells were fixed in 70% cold ethanol and incubated at −20 °C overnight and then stained with PI/RNase staining buffer (BD Pharmingen). Flow cytometry was performed using a FACS Calibur (BD), and the results were analyzed by ModFit software.</div></div><div id="sec5-10" class="NLM_sec NLM_sec_level_2"><div id="ac_i75" class="anchor-spacer"></div><h3 class="article-section__title" id="_i75"> Modeling Method</h3><div class="NLM_p last">Currently there is no DFG-in conformation of FLT3 reported in the PDB. It was obtained by homology modeling using the DFG-out conformation of FLT3 (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4RT7">4RT7</a>) for the majority part and DFG-in conformation of CSF1R (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3LCD">3LCD</a>) for the activation loop<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> with Modeller 9.18.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> The homology model was further refined by MacroModel 11.5 (Schrödinger, LLC, New York, NY). For the DGF-out conformation of FLT3, the PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4XUF">4XUF</a> was prepared with Protein Preparation Wizard (Schrödinger, LLC, New York, NY) for docking. The F691L mutation was obtained by site mutation based on the optimized wild type coordinates. The ligands were prepared with LigPrep (Schrödinger, LLC, New York, NY). The geometries were optimized by Jaguar 9.5 (Schrödinger, LLC, New York, NY) at B3LYP/6-31G* level with default settings. The atomic partial charges were calculated based on the optimized ligand coordinates. The binding modes were obtained by Induced Fit (Schrödinger Suite 2017-1 Induced Fit Docking protocol) with extended sampling protocol. The best binding modes were identified by both the IFD score and visual inspection of the binding saturation to further reveal the binding mechanism.</div></div><div id="sec5-11" class="NLM_sec NLM_sec_level_2"><div id="ac_i76" class="anchor-spacer"></div><h3 class="article-section__title" id="_i76"> In Vivo Pharmacokinetics Study</h3><div class="NLM_p last">Compounds <b>14</b> and <b>7</b> were dissolved in 55% saline containing 5% DMSO and 40% PEG400 by vortex. The final concentration of the stock solution was 1 mg/mL for administration. Six 6–8 weeks old male Sprague Dawely rats were fasted overnight before starting drug treatment via intravenous and oral administration. Animal blood collection time points were as follows: for intravenous administration groups (iv), 1 min, 5 min, 15 min, 30 min, 1 h, 2 h, 4 h, 6 h, 8 h, 24 h before and after administration were selected; for oral administration groups (po), 5 min, 15 min, 30 min, 1 h, 2 h, 4 h, 6 h, 8 h, and 24 h before and after dosing were selected. Each time about 0.2 mL of blood was collected through the jugular vein adding heparin for anticoagulation and kept on ice. Then plasma was separated by centrifugation at 8000 rpm for 6 min at 2–8 °C. The obtained plasma was stored at −80 °C before analysis. After the test was finished, all surviving animals were transferred to the repository or euthanaized (CO<sub>2</sub> asphyxiation).</div></div><div id="sec5-12" class="NLM_sec NLM_sec_level_2"><div id="ac_i77" class="anchor-spacer"></div><h3 class="article-section__title" id="_i77"> MV4-11 Xenograft Tumor Model</h3><div class="NLM_p last">Five-week-old female nu/nu mice were purchased from the Shanghai Experimental Center, Chinese Academy of Sciences (Shanghai, China). All animals were housed in a specific pathogen-free facility and used according to the animal care regulations of Hefei Institutes of Physical Science, Chinese Academy of Sciences (Hefei, China). Prior to implantation, cells were harvested during exponential growth. Five million MV4-11 cells in PBS were formulated as a 1:1 mixture with Matrigel (BD Biosciences) and injected into the subcutaneous space on the right flank of nu/nu mice. Daily oral administration was initiated when MV4-11 tumors had reached a size of 200–400 mm<sup>3</sup>. Animals were then randomized into treatment groups of 5–7 mice each for efficacy studies. Compound <b>14</b> was delivered daily in a HKI solution (0.5% methocellulose/0.4% Tween80 in ddH<sub>2</sub>O) by oral gavage. A range of doses of compound <b>14</b> or its vehicle or compound <b>7</b> as control were administered, as indicated in the <a class="ref internalNav" href="#fig7" aria-label="Figure 7">Figure 7</a> caption. Body weight was measured daily, and tumor growth was measured every day after compound <b>14</b> treatment. Tumor volume was calculated as follows: tumor volume (mm<sup>3</sup>) = [(<i>W</i>2 × <i>L</i>)/2] in which width (<i>W</i>) is defined as the smaller of the two measurements and length (<i>L</i>) is defined as the larger of the two measurements.</div></div><div id="sec5-13" class="NLM_sec NLM_sec_level_2"><div id="ac_i78" class="anchor-spacer"></div><h3 class="article-section__title" id="_i78"> HE Staining</h3><div class="NLM_p last">HE staining was carried out according to previous publication.<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> First, the sections were hydrated and then the slide was dipped into a Coplin jar containing Mayer’s hematoxylin and agitated for 30 s. After rinsing the slide in H<sub>2</sub>O for 1 min, it was stained with 1% eosin Y solution for 10–30 s with agitation. Subsequently, the sections were dehydrated with two changes of 95% alcohol and two changes of 100% alcohol for 30 s each, and then the alcohol was extracted with two changes of xylene. Finally, one or two drops of mounting medium was added and covered with a coverslip.</div></div><div id="sec5-14" class="NLM_sec NLM_sec_level_2"><div id="ac_i79" class="anchor-spacer"></div><h3 class="article-section__title" id="_i79"> Ki-67 Staining</h3><div class="NLM_p last">For IHC demonstration of Ki-67, tissue sections were quenched for endogenous peroxides and placed in an antigen retrieval solution (0.01 M citrate buffer, pH 6.0) for 15 min in a microwave oven at 100 °C at 600 W. After incubation in the casein block, mouse mAb anti-Ki-67 (ZSGB-BIO, China) was applied to the sections at dilutions of 1:50. Incubations with primary antibodies lasted overnight at 4 °C. The secondary detection system was used to visualize antibody binding. Staining was developed with DAB, and the slides were counterstained with hematoxylin, dehydrated, and mounted.</div></div><div id="sec5-15" class="NLM_sec NLM_sec_level_2"><div id="ac_i80" class="anchor-spacer"></div><h3 class="article-section__title" id="_i80"> TUNEL Staining</h3><div class="NLM_p last">TUNEL staining was performed using the POD in situ cell death detection kit (Roche, USA). Briefly, sections were deparaffinized in xylene, rehydrated in decreasing concentration of ethanol, and then treated by nuclease free proteinase K for 15 min at room temperature before endogenous peroxidase was blocked in 3% H<sub>2</sub>O<sub>2</sub> in methanol. Terminal deoxynucleotidyl transferase (TdT) in reaction buffer was applied to sections for 1 h at 37 °C. Following washes, the slides were covered by converter-POD solution for 30 min at 37 °C. Apoptotic cells were detected after incubation in 3,3′-diaminobenzidine (DAB) chromogen (Beyotime Biotechnology, China) for approximately 8 min, and the slides were counterstained with hematoxylin.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i81"><a href="/doi/suppl/10.1021/acs.jmedchem.7b00840">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_08333" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_08333" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a href="/" class="ext-link">ACS Publications website</a> at DOI: <a href="/doi/abs/10.1021/acs.jmedchem.7b00840" class="ext-link">10.1021/acs.jmedchem.7b00840</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Table 1 listing the DiscoverX’s KINOMEScan selectivity profiling data of compound <b>14</b> and Figure 1 showing the cellular effects of compound <b>7</b> in FLT3-ITD positive AML cell line (<a href="/doi/suppl/10.1021/acs.jmedchem.7b00840/suppl_file/jm7b00840_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular formula strings and some data (<a href="/doi/suppl/10.1021/acs.jmedchem.7b00840/suppl_file/jm7b00840_si_002.csv" class="ext-link">CSV</a>)</p></li><li><p class="inline">Coordinates information for structure representation (<a href="/doi/suppl/10.1021/acs.jmedchem.7b00840/suppl_file/jm7b00840_si_003.pdb" class="ext-link">PDB</a>)</p></li><li><p class="inline">Coordinates information for structure representation (<a href="/doi/suppl/10.1021/acs.jmedchem.7b00840/suppl_file/jm7b00840_si_004.pdb" class="ext-link">PDB</a>)</p></li><li><p class="inline">Coordinates information for structure representation (<a href="/doi/suppl/10.1021/acs.jmedchem.7b00840/suppl_file/jm7b00840_si_005.pdb" class="ext-link">PDB</a>)</p></li><li><p class="inline">Coordinates information for structure representation (<a href="/doi/suppl/10.1021/acs.jmedchem.7b00840/suppl_file/jm7b00840_si_006.pdb" class="ext-link">PDB</a>)</p></li><li><p class="inline">Coordinates information for structure representation (<a href="/doi/suppl/10.1021/acs.jmedchem.7b00840/suppl_file/jm7b00840_si_007.pdb" class="ext-link">PDB</a>)</p></li><li><p class="inline">Coordinates information for structure representation (<a href="/doi/suppl/10.1021/acs.jmedchem.7b00840/suppl_file/jm7b00840_si_008.pdb" class="ext-link">PDB</a>)</p></li><li><p class="inline">Coordinates information for structure representation (<a href="/doi/suppl/10.1021/acs.jmedchem.7b00840/suppl_file/jm7b00840_si_009.pdb" class="ext-link">PDB</a>)</p></li><li><p class="inline">Coordinates information for structure representation (<a href="/doi/suppl/10.1021/acs.jmedchem.7b00840/suppl_file/jm7b00840_si_010.pdb" class="ext-link">PDB</a>)</p></li><li><p class="inline">Coordinates information for structure representation (<a href="/doi/suppl/10.1021/acs.jmedchem.7b00840/suppl_file/jm7b00840_si_011.pdb" class="ext-link">PDB</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.7b00840/suppl_file/jm7b00840_si_001.pdf">jm7b00840_si_001.pdf (717.68 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.7b00840/suppl_file/jm7b00840_si_002.csv">jm7b00840_si_002.csv (5.66 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.7b00840/suppl_file/jm7b00840_si_003.pdb">jm7b00840_si_003.pdb (365.66 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.7b00840/suppl_file/jm7b00840_si_004.pdb">jm7b00840_si_004.pdb (349.36 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.7b00840/suppl_file/jm7b00840_si_005.pdb">jm7b00840_si_005.pdb (350.26 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.7b00840/suppl_file/jm7b00840_si_006.pdb">jm7b00840_si_006.pdb (365.59 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.7b00840/suppl_file/jm7b00840_si_007.pdb">jm7b00840_si_007.pdb (349.28 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.7b00840/suppl_file/jm7b00840_si_008.pdb">jm7b00840_si_008.pdb (350.19 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.7b00840/suppl_file/jm7b00840_si_009.pdb">jm7b00840_si_009.pdb (504.22 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.7b00840/suppl_file/jm7b00840_si_010.pdb">jm7b00840_si_010.pdb (504.22 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.7b00840/suppl_file/jm7b00840_si_011.pdb">jm7b00840_si_011.pdb (504.22 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.7b00840" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_42046" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_42046" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Xixiang Li</span> - <span class="hlFld-Affiliation affiliation">High Magnetic Field
Laboratory, Chinese Academy of Sciences, Mailbox 1110, 350 Shushanhu Road, Hefei, Anhui 230031, P. R. China</span>; 
    <span class="hlFld-Affiliation affiliation">CHMFL-HCMTC Target Therapy Joint Laboratory, 350 Shushanhu Road, Hefei, Anhui 230031, P. R. China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Qingwang Liu</span> - <span class="hlFld-Affiliation affiliation">Precision Targeted Therapy Discovery Center, Institute of Technology
Innovation, Hefei Institutes of Physical Science, Chinese Academy
of Sciences, Hefei, Anhui 230088, P. R. China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ruixiang Xia</span> - <span class="hlFld-Affiliation affiliation">Department of Hematology, The First Affiliated
Hospital of Anhui Medical University, Hefei, Anhui 230022, P. R. China</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#f1898389c3c1c5c0b1c0c7c2df929e9c"><span class="__cf_email__" data-cfemail="08707a703a383c3948393e3b266b6765">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jing Liu</span> - <span class="hlFld-Affiliation affiliation">High Magnetic Field
Laboratory, Chinese Academy of Sciences, Mailbox 1110, 350 Shushanhu Road, Hefei, Anhui 230031, P. R. China</span>; 
    <span class="hlFld-Affiliation affiliation">CHMFL-HCMTC Target Therapy Joint Laboratory, 350 Shushanhu Road, Hefei, Anhui 230031, P. R. China</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-9513-3591" title="Orcid link">http://orcid.org/0000-0001-9513-3591</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#4f2526212823263a0f27222923612e2c612c21"><span class="__cf_email__" data-cfemail="a4cecdcac3c8cdd1e4ccc9c2c88ac5c78ac7ca">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Qingsong Liu</span> - <span class="hlFld-Affiliation affiliation">High Magnetic Field
Laboratory, Chinese Academy of Sciences, Mailbox 1110, 350 Shushanhu Road, Hefei, Anhui 230031, P. R. China</span>; 
    <span class="hlFld-Affiliation affiliation">CHMFL-HCMTC Target Therapy Joint Laboratory, 350 Shushanhu Road, Hefei, Anhui 230031, P. R. China</span>; 
    <span class="hlFld-Affiliation affiliation">University of Science and Technology of
China, Hefei, Anhui 230036, P.
R. China</span>; 
    <span class="hlFld-Affiliation affiliation">Precision Targeted Therapy Discovery Center, Institute of Technology
Innovation, Hefei Institutes of Physical Science, Chinese Academy
of Sciences, Hefei, Anhui 230088, P. R. China</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-7829-2547" title="Orcid link">http://orcid.org/0000-0002-7829-2547</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#96e7e5faffe3afa1d6fefbf0fab8f7f5b8f5f8"><span class="__cf_email__" data-cfemail="38494b54514d010f7850555e5416595b165b56">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Aoli Wang</span> - <span class="hlFld-Affiliation affiliation">High Magnetic Field
Laboratory, Chinese Academy of Sciences, Mailbox 1110, 350 Shushanhu Road, Hefei, Anhui 230031, P. R. China</span>; 
    <span class="hlFld-Affiliation affiliation">CHMFL-HCMTC Target Therapy Joint Laboratory, 350 Shushanhu Road, Hefei, Anhui 230031, P. R. China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Cheng Chen</span> - <span class="hlFld-Affiliation affiliation">High Magnetic Field
Laboratory, Chinese Academy of Sciences, Mailbox 1110, 350 Shushanhu Road, Hefei, Anhui 230031, P. R. China</span>; 
    <span class="hlFld-Affiliation affiliation">University of Science and Technology of
China, Hefei, Anhui 230036, P.
R. China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Hong Wu</span> - <span class="hlFld-Affiliation affiliation">High Magnetic Field
Laboratory, Chinese Academy of Sciences, Mailbox 1110, 350 Shushanhu Road, Hefei, Anhui 230031, P. R. China</span>; 
    <span class="hlFld-Affiliation affiliation">CHMFL-HCMTC Target Therapy Joint Laboratory, 350 Shushanhu Road, Hefei, Anhui 230031, P. R. China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ziping Qi</span> - <span class="hlFld-Affiliation affiliation">High Magnetic Field
Laboratory, Chinese Academy of Sciences, Mailbox 1110, 350 Shushanhu Road, Hefei, Anhui 230031, P. R. China</span>; 
    <span class="hlFld-Affiliation affiliation">CHMFL-HCMTC Target Therapy Joint Laboratory, 350 Shushanhu Road, Hefei, Anhui 230031, P. R. China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Chen Hu</span> - <span class="hlFld-Affiliation affiliation">High Magnetic Field
Laboratory, Chinese Academy of Sciences, Mailbox 1110, 350 Shushanhu Road, Hefei, Anhui 230031, P. R. China</span>; 
    <span class="hlFld-Affiliation affiliation">University of Science and Technology of
China, Hefei, Anhui 230036, P.
R. China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Kailin Yu</span> - <span class="hlFld-Affiliation affiliation">High Magnetic Field
Laboratory, Chinese Academy of Sciences, Mailbox 1110, 350 Shushanhu Road, Hefei, Anhui 230031, P. R. China</span>; 
    <span class="hlFld-Affiliation affiliation">University of Science and Technology of
China, Hefei, Anhui 230036, P.
R. China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jiaxin Wu</span> - <span class="hlFld-Affiliation affiliation">High Magnetic Field
Laboratory, Chinese Academy of Sciences, Mailbox 1110, 350 Shushanhu Road, Hefei, Anhui 230031, P. R. China</span>; 
    <span class="hlFld-Affiliation affiliation">University of Science and Technology of
China, Hefei, Anhui 230036, P.
R. China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Juan Liu</span> - <span class="hlFld-Affiliation affiliation">Precision Targeted Therapy Discovery Center, Institute of Technology
Innovation, Hefei Institutes of Physical Science, Chinese Academy
of Sciences, Hefei, Anhui 230088, P. R. China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Xiaochuan Liu</span> - <span class="hlFld-Affiliation affiliation">High Magnetic Field
Laboratory, Chinese Academy of Sciences, Mailbox 1110, 350 Shushanhu Road, Hefei, Anhui 230031, P. R. China</span>; 
    <span class="hlFld-Affiliation affiliation">CHMFL-HCMTC Target Therapy Joint Laboratory, 350 Shushanhu Road, Hefei, Anhui 230031, P. R. China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Zhenquan Hu</span> - <span class="hlFld-Affiliation affiliation">High Magnetic Field
Laboratory, Chinese Academy of Sciences, Mailbox 1110, 350 Shushanhu Road, Hefei, Anhui 230031, P. R. China</span>; 
    <span class="hlFld-Affiliation affiliation">CHMFL-HCMTC Target Therapy Joint Laboratory, 350 Shushanhu Road, Hefei, Anhui 230031, P. R. China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Wei Wang</span> - <span class="hlFld-Affiliation affiliation">High Magnetic Field
Laboratory, Chinese Academy of Sciences, Mailbox 1110, 350 Shushanhu Road, Hefei, Anhui 230031, P. R. China</span>; 
    <span class="hlFld-Affiliation affiliation">CHMFL-HCMTC Target Therapy Joint Laboratory, 350 Shushanhu Road, Hefei, Anhui 230031, P. R. China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Wenliang Wang</span> - <span class="hlFld-Affiliation affiliation">High Magnetic Field
Laboratory, Chinese Academy of Sciences, Mailbox 1110, 350 Shushanhu Road, Hefei, Anhui 230031, P. R. China</span>; 
    <span class="hlFld-Affiliation affiliation">University of Science and Technology of
China, Hefei, Anhui 230036, P.
R. China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Wenchao Wang</span> - <span class="hlFld-Affiliation affiliation">High Magnetic Field
Laboratory, Chinese Academy of Sciences, Mailbox 1110, 350 Shushanhu Road, Hefei, Anhui 230031, P. R. China</span>; 
    <span class="hlFld-Affiliation affiliation">CHMFL-HCMTC Target Therapy Joint Laboratory, 350 Shushanhu Road, Hefei, Anhui 230031, P. R. China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Li Wang</span> - <span class="hlFld-Affiliation affiliation">High Magnetic Field
Laboratory, Chinese Academy of Sciences, Mailbox 1110, 350 Shushanhu Road, Hefei, Anhui 230031, P. R. China</span>; 
    <span class="hlFld-Affiliation affiliation">University of Science and Technology of
China, Hefei, Anhui 230036, P.
R. China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Beilei Wang</span> - <span class="hlFld-Affiliation affiliation">High Magnetic Field
Laboratory, Chinese Academy of Sciences, Mailbox 1110, 350 Shushanhu Road, Hefei, Anhui 230031, P. R. China</span>; 
    <span class="hlFld-Affiliation affiliation">University of Science and Technology of
China, Hefei, Anhui 230036, P.
R. China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Lili Li</span> - <span class="hlFld-Affiliation affiliation">Department of Hematology, The First Affiliated
Hospital of Anhui Medical University, Hefei, Anhui 230022, P. R. China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jian Ge</span> - <span class="hlFld-Affiliation affiliation">Department of Hematology, The First Affiliated
Hospital of Anhui Medical University, Hefei, Anhui 230022, P. R. China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Tao Ren</span> - <span class="hlFld-Affiliation affiliation">Precision Targeted Therapy Discovery Center, Institute of Technology
Innovation, Hefei Institutes of Physical Science, Chinese Academy
of Sciences, Hefei, Anhui 230088, P. R. China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Shanchun Zhang</span> - <span class="hlFld-Affiliation affiliation">Hefei Cosource Medicine Technology Co.
Ltd., 358 Ganquan Road, Hefei, Anhui 230031, P. R. China</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>A.W., X.L., C.C., H.W., Z.Q., and C.H. contributed equally.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare the following competing financial interest(s): Dr. Shanchun Zhang is a shareholder of Hefei Cosource Medicine Technology Co. Ltd.<br /></br></div></li></ul></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i83">Acknowledgment</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_00265" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_00265" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">This work was supported by the National Natural Science Foundation of China (Grants. U1432250, U1532154, 81471773, 81473088), the National Key Research and Development Program of China (Grant 2016YFA0400900), the “Personalized Medicines-Molecular Signature-Based Drug Discovery and Development”, Strategic Priority Research Program of the Chinese Academy of Sciences (Grant XDA12020308), the Natural Science Foundation of Anhui Province (Grants 1608085QH180, 1708085MH208), the Postdoctoral Innovative Talents Support Program (Grant BX201600169), and the CAS/SAFEA International Partnership Program for Creative Research Teams. We are also grateful for the National Program for Support of Top-Notch Young Professionals for J.L. and the “Hundred Talents Program” of CAS support for J.L. and W.W.</p></div><div id="" class="NLM_sec NLM_sec_level_1"><div id="ac_i84" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i84"> </h2><div class="NLM_p last"><table summary="" class="NLM_def-list" id="dl1"><div id="ac_i85" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i85"> Abbreviations Used</h2><tr><td class="NLM_term">AML</td><td class="NLM_def"><p class="first last">acute myeloid leukemia</p></td></tr><tr><td class="NLM_term">FLT3</td><td class="NLM_def"><p class="first last">Fms-like tyrosine kinase 3</p></td></tr><tr><td class="NLM_term">RTK</td><td class="NLM_def"><p class="first last">receptor tyrosine kinase</p></td></tr><tr><td class="NLM_term">FLT3-ITD</td><td class="NLM_def"><p class="first last">Fms-like tyrosine kinase internal tandem duplication</p></td></tr><tr><td class="NLM_term">BTK</td><td class="NLM_def"><p class="first last">Bruton’s tyrosine kinase</p></td></tr><tr><td class="NLM_term">CLL</td><td class="NLM_def"><p class="first last">chronic lymphocytic leukemia</p></td></tr><tr><td class="NLM_term">MCL</td><td class="NLM_def"><p class="first last">mantle cell lymphoma</p></td></tr><tr><td class="NLM_term">SAR</td><td class="NLM_def"><p class="first last">structure–activity relationship</p></td></tr><tr><td class="NLM_term">DMSO</td><td class="NLM_def"><p class="first last">dimethyl sulfoxide</p></td></tr><tr><td class="NLM_term">DMF</td><td class="NLM_def"><p class="first last"><i>N</i>,<i>N</i>-dimethylformamide</p></td></tr><tr><td class="NLM_term">DCM</td><td class="NLM_def"><p class="first last">dimethyl chloride</p></td></tr><tr><td class="NLM_term">TUNEL</td><td class="NLM_def"><p class="first last">terminal deoxynucleotidyl transferase dUTP nick end labeling</p></td></tr></table></div></div><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i86">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_80820" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_80820" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 23 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group">Lee, H. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, H. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, I. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jung, D. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, S. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Park, S. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hwang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Choi, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, J. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, S. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, J. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cools, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koh, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Song, H. J.</span><span> </span><span class="NLM_article-title">G-749, a novel FLT3 kinase inhibitor, can overcome drug resistance for the treatment of acute myeloid leukemia</span> <span class="citation_source-journal">Blood</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">123</span><span class="NLM_x">, </span> <span class="NLM_fpage">2209</span><span class="NLM_x">–</span> <span class="NLM_lpage">2219</span><span class="refDoi"> DOI: 10.1182/blood-2013-04-493916</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00840&amp;key=10.1182%2Fblood-2013-04-493916" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00840&amp;key=24532805" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00840&amp;key=1%3ACAS%3A528%3ADC%252BC2cXmtVOju74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=123&publication_year=2014&pages=2209-2219&author=H.+K.+Leeauthor=H.+W.+Kimauthor=I.+Y.+Leeauthor=J.+Leeauthor=J.+Leeauthor=D.+S.+Jungauthor=S.+Y.+Leeauthor=S.+H.+Parkauthor=H.+Hwangauthor=J.+S.+Choiauthor=J.+H.+Kimauthor=S.+W.+Kimauthor=J.+K.+Kimauthor=J.+Coolsauthor=J.+S.+Kohauthor=H.+J.+Song&title=G-749%2C+a+novel+FLT3+kinase+inhibitor%2C+can+overcome+drug+resistance+for+the+treatment+of+acute+myeloid+leukemia&doi=10.1182%2Fblood-2013-04-493916"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">G-749, a novel FLT3 kinase inhibitor, can overcome drug resistance for the treatment of acute myeloid leukemia</span></div><div class="casAuthors">Lee, Hee Kyu; Kim, Hong Woo; Lee, In Yong; Lee, Jungmi; Lee, Jaekyoo; Jung, Dong Sik; Lee, Sang Yeop; Park, Sung Ho; Hwang, Haejun; Choi, Jang-Sik; Kim, Jung-Ho; Kim, Se Won; Kim, Jung Keun; Cools, Jan; Koh, Jong Sung; Song, Ho-Juhn</div><div class="citationInfo"><span class="NLM_cas:title">Blood</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">123</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">2209-2219</span>CODEN:
                <span class="NLM_cas:coden">BLOOAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-4971</span>.
    
            (<span class="NLM_cas:orgname">American Society of Hematology</span>)
        </div><div class="casAbstract">Aberrant activations of Fms-like tyrosine receptor kinase (FLT) 3 are implicated in the pathogenesis of 20% to 30% of patients with acute myeloid leukemia (AML).  G-749 is a novel FLT3 inhibitor that showed potent and sustained inhibition of the FLT3 wild type and mutants including FLT3-ITD, FLT3-D835Y, FLT3-ITD/N676D, and FLT3-ITD/F691L in cellular assays.  G-749 retained its inhibitory potency in various drug-resistance milieus such as patient plasma, FLT3 ligand surge, and stromal protection.  Furthermore, it displayed potent antileukemic activity in bone marrow blasts from AML patients regardless of FLT3 mutation status, including those with little or only minor responses to AC220 or PKC412.  Oral administration of G-749 yielded complete tumor regression and increased life span in animal models.  Thus, G-749 appears to be a promising next-generation drug candidate for the treatment of relapsed and refractory AML patients with various FLT3-ITD/FLT3-TKD mutants and further shows the ability to overcome drug resistance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqS1dwuSRK6JrVg90H21EOLACvtfcHk0lj7t3NzlEORLg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXmtVOju74%253D&md5=07b23569d047a3a16789d673fe904c6b</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1182%2Fblood-2013-04-493916&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1182%252Fblood-2013-04-493916%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DH.%2BK.%26aulast%3DKim%26aufirst%3DH.%2BW.%26aulast%3DLee%26aufirst%3DI.%2BY.%26aulast%3DLee%26aufirst%3DJ.%26aulast%3DLee%26aufirst%3DJ.%26aulast%3DJung%26aufirst%3DD.%2BS.%26aulast%3DLee%26aufirst%3DS.%2BY.%26aulast%3DPark%26aufirst%3DS.%2BH.%26aulast%3DHwang%26aufirst%3DH.%26aulast%3DChoi%26aufirst%3DJ.%2BS.%26aulast%3DKim%26aufirst%3DJ.%2BH.%26aulast%3DKim%26aufirst%3DS.%2BW.%26aulast%3DKim%26aufirst%3DJ.%2BK.%26aulast%3DCools%26aufirst%3DJ.%26aulast%3DKoh%26aufirst%3DJ.%2BS.%26aulast%3DSong%26aufirst%3DH.%2BJ.%26atitle%3DG-749%252C%2520a%2520novel%2520FLT3%2520kinase%2520inhibitor%252C%2520can%2520overcome%2520drug%2520resistance%2520for%2520the%2520treatment%2520of%2520acute%2520myeloid%2520leukemia%26jtitle%3DBlood%26date%3D2014%26volume%3D123%26spage%3D2209%26epage%3D2219%26doi%3D10.1182%2Fblood-2013-04-493916" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group">Kiyoi, H.</span><span> </span><span class="NLM_article-title">FLT3 Inhibitors: recent advances and problems for clinical application</span> <span class="citation_source-journal">Nagoya J. Med. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">77</span><span class="NLM_x">, </span> <span class="NLM_fpage">7</span><span class="NLM_x">–</span> <span class="NLM_lpage">17</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00840&amp;key=25797966" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00840&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhslalt7rL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=77&publication_year=2015&pages=7-17&author=H.+Kiyoi&title=FLT3+Inhibitors%3A+recent+advances+and+problems+for+clinical+application"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">FLT3 inhibitors: recent advances and problems for clinical application</span></div><div class="casAuthors">Kiyoi, Hitoshi</div><div class="citationInfo"><span class="NLM_cas:title">Nagoya Journal of Medical Science</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">77</span>
        (<span class="NLM_cas:issue">1,2</span>),
    <span class="NLM_cas:pages">7-17</span>CODEN:
                <span class="NLM_cas:coden">NJMSAG</span>;
        ISSN:<span class="NLM_cas:issn">0027-7622</span>.
    
            (<span class="NLM_cas:orgname">Nagoya University School of Medicine, Graduate School of Medicine</span>)
        </div><div class="casAbstract">FLT3, a type III receptor tyrosine kinase, expresses on most acute leukemia cells as well as normal hematopoietic stem/progenitor cells.  Mutation in the FLT3 gene is the most frequent genetic alteration in acute myeloid leukemia (AML) and is well known as an important driver mutation for the development of myeloid malignancies.  FLT3 mutation is a strong poor prognostic factor for the long-term survival in AML patients, while neither high-dose chemotherapy nor allogeneic hematopoietic stem cell transplantation can overcome a poor prognosis.  Development of an FLT3 inhibitor is, therefore, much awaited.  To date, several potent FLT3 inhibitors have been developed and some of them were evaluated for efficacy in clin. trials, although no FLT3 inhibitor has been yet approved.  Moreover, several problems for clin. use, such as adverse effects, blood concn. and resistance have been apparent.  Recently developed AC220 is a highly selective and sensitive FLT3 inhibitor.  In Phase I and II trials, AC220 so far showed the best efficacy of AML cells harboring FLT3 mutation among clin. evaluated FLT3 inhibitors, while severe bone marrow suppression and QTc prolongation should be resolved for the clin. use.  In this review, I summarize the characteristics of FLT3 inhibitors in clin. development and discuss important issues to be resolved for clin. use.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoQu7A0XRVxQbVg90H21EOLACvtfcHk0lj7t3NzlEORLg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhslalt7rL&md5=9a44962a3aa888bfdfd3fb49e388999a</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DKiyoi%26aufirst%3DH.%26atitle%3DFLT3%2520Inhibitors%253A%2520recent%2520advances%2520and%2520problems%2520for%2520clinical%2520application%26jtitle%3DNagoya%2520J.%2520Med.%2520Sci.%26date%3D2015%26volume%3D77%26spage%3D7%26epage%3D17" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group">Fathi, A. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, Y. B.</span><span> </span><span class="NLM_article-title">The role of FLT3 inhibitors in the treatment of FLT3-mutated acute myeloid leukemia</span> <span class="citation_source-journal">Eur. J. Haematol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2017</span><span class="NLM_x">, </span> <span class="NLM_volume">98</span><span class="NLM_x">, </span> <span class="NLM_fpage">330</span><span class="NLM_x">–</span> <span class="NLM_lpage">336</span><span class="refDoi"> DOI: 10.1111/ejh.12841</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00840&amp;key=10.1111%2Fejh.12841" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00840&amp;key=28000291" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00840&amp;key=1%3ACAS%3A280%3ADC%252BC1c%252FntVegug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=98&publication_year=2017&pages=330-336&author=A.+T.+Fathiauthor=Y.+B.+Chen&title=The+role+of+FLT3+inhibitors+in+the+treatment+of+FLT3-mutated+acute+myeloid+leukemia&doi=10.1111%2Fejh.12841"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">The role of FLT3 inhibitors in the treatment of FLT3-mutated acute myeloid leukemia</span></div><div class="casAuthors">Fathi Amir T; Chen Yi-Bin</div><div class="citationInfo"><span class="NLM_cas:title">European journal of haematology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">98</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">330-336</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">FLT3 mutations are present in about one-third of patients with acute myeloid leukemia (AML).  Several FLT3 inhibitors have been used in clinical trials, and these include midostaurin, sorafenib, quizartinib, crenolanib, and gilteritinib.  Monotherapy with early tyrosine kinase inhibitors (TKIs) did not have much success; however, later generation agents have shown more promising results.  Combination with conventional chemotherapy may have benefit as evidenced by recently presented results, and data from ongoing trials are eagerly awaited.  Several trials are also evaluating TKI given after HSCT, and a large international randomized trial is planned.  We may be close to an era of targeted therapy where the standard of care for this biologically defined subset will involve incorporation of a FLT3 TKI during induction chemotherapy and after HSCT.  It is important that our community continues to collaborate to conduct well-designed clinical trials to properly define the role of FLT3 TKIs in therapy for FLT3-mutant AML.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRGKE9FArD4cUEksK8CslK8fW6udTcc2ebrsR1pZi440Lntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1c%252FntVegug%253D%253D&md5=6fe9f871da62a2a075e083155be09128</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1111%2Fejh.12841&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fejh.12841%26sid%3Dliteratum%253Aachs%26aulast%3DFathi%26aufirst%3DA.%2BT.%26aulast%3DChen%26aufirst%3DY.%2BB.%26atitle%3DThe%2520role%2520of%2520FLT3%2520inhibitors%2520in%2520the%2520treatment%2520of%2520FLT3-mutated%2520acute%2520myeloid%2520leukemia%26jtitle%3DEur.%2520J.%2520Haematol.%26date%3D2017%26volume%3D98%26spage%3D330%26epage%3D336%26doi%3D10.1111%2Fejh.12841" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group">Smith, C. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stecula, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sali, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shah, N. P.</span><span> </span><span class="NLM_article-title">FLT3 D835 mutations confer differential resistance to type II FLT3 inhibitors</span> <span class="citation_source-journal">Leukemia</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">29</span><span class="NLM_x">, </span> <span class="NLM_fpage">2390</span><span class="NLM_x">–</span> <span class="NLM_lpage">2392</span><span class="refDoi"> DOI: 10.1038/leu.2015.165</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00840&amp;key=10.1038%2Fleu.2015.165" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00840&amp;key=26108694" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00840&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1Wrtb%252FJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2015&pages=2390-2392&author=C.+C.+Smithauthor=K.+Linauthor=A.+Steculaauthor=A.+Saliauthor=N.+P.+Shah&title=FLT3+D835+mutations+confer+differential+resistance+to+type+II+FLT3+inhibitors&doi=10.1038%2Fleu.2015.165"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">FLT3 D835 mutations confer differential resistance to type II FLT3 inhibitors</span></div><div class="casAuthors">Smith, C. C.; Lin, K.; Stecula, A.; Sali, A.; Shah, N. P.</div><div class="citationInfo"><span class="NLM_cas:title">Leukemia</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2390-2392</span>CODEN:
                <span class="NLM_cas:coden">LEUKED</span>;
        ISSN:<span class="NLM_cas:issn">0887-6924</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Activating mutations in FLT3 occur in ∼30% of adult acute myeloid leukemia, primarily consisting of internal tandem duplication (ITD) mutations (∼25%) and point mutations in the tyrosine kinase domain (∼5%), commonly at the activation loop residue D835.  Secondary kinase domain mutations in FLT3-ITD, particularly at the D835 residue are frequently assocd. with acquired clin. resistance to effective FLT3 tyrosine kinase inhibitors (TKIs).  Mol. docking studies have suggested that D835 mutations primarily confer resistance by stabilizing an active Asp-Phe-Gly in ('DFG-in') kinase conformation unfavorable to the binding of type II FLT3 TKIs, which target a 'DFG-out' inactive conformation.  We profiled the activity of active type II FLT3 TKIs against D835 kinase domain mutants that have been clin. detected to date.  We found that type II inhibitors (quizartinib, sorafenib, ponatinib and PLX3397) retain activity against specific D835 substitutions.  Modeling studies suggest that bulky hydrophobic substitutions (D835Y/V/I/F) at this residue are particularly resistant, whereas mutations that preserve interactions between D835 and S838 are relatively sensitive (D835E/N).  All mutants retain sensitivity to the type I inhibitor crenolanib.  These results suggest that patients with relatively sensitive D835 mutations should be included in clin. trials of type II FLT3 TKIs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqQhZ7jQYhBcbVg90H21EOLACvtfcHk0lhaXbQsFcpNCQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht1Wrtb%252FJ&md5=f6de44ca15604b8429a5dab811fe8eeb</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1038%2Fleu.2015.165&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fleu.2015.165%26sid%3Dliteratum%253Aachs%26aulast%3DSmith%26aufirst%3DC.%2BC.%26aulast%3DLin%26aufirst%3DK.%26aulast%3DStecula%26aufirst%3DA.%26aulast%3DSali%26aufirst%3DA.%26aulast%3DShah%26aufirst%3DN.%2BP.%26atitle%3DFLT3%2520D835%2520mutations%2520confer%2520differential%2520resistance%2520to%2520type%2520II%2520FLT3%2520inhibitors%26jtitle%3DLeukemia%26date%3D2015%26volume%3D29%26spage%3D2390%26epage%3D2392%26doi%3D10.1038%2Fleu.2015.165" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group">Weisberg, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roesel, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Furet, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bold, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Imbach, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Florsheimer, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Caravatti, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jiang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Manley, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ray, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Griffin, J. D.</span><span> </span><span class="NLM_article-title">Antileukemic effects of novel first- and second-Generation FLT3 Inhibitors: structure-affinity comparison</span> <span class="citation_source-journal">Genes Cancer</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">1</span><span class="NLM_x">, </span> <span class="NLM_fpage">1021</span><span class="NLM_x">–</span> <span class="NLM_lpage">1032</span><span class="refDoi"> DOI: 10.1177/1947601910396505</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00840&amp;key=10.1177%2F1947601910396505" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00840&amp;key=21779428" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00840&amp;key=1%3ACAS%3A528%3ADC%252BC3MXisF2nsrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2010&pages=1021-1032&author=E.+Weisbergauthor=J.+Roeselauthor=P.+Furetauthor=G.+Boldauthor=P.+Imbachauthor=A.+Florsheimerauthor=G.+Caravattiauthor=J.+Jiangauthor=P.+Manleyauthor=A.+Rayauthor=J.+D.+Griffin&title=Antileukemic+effects+of+novel+first-+and+second-Generation+FLT3+Inhibitors%3A+structure-affinity+comparison&doi=10.1177%2F1947601910396505"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Antileukemic effects of novel first-and second-generation FLT3 inhibitors: structure-affinity comparison</span></div><div class="casAuthors">Weisberg, Ellen; Roesel, Johannes; Furet, Pascal; Bold, Guido; Imbach, Patricia; Florsheimer, Andreas; Caravatti, Georgio; Jiang, Jingrui; Manley, Paul; Ray, Arghya; Griffin, James D.</div><div class="citationInfo"><span class="NLM_cas:title">Genes & Cancer</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1021-1032</span>CODEN:
                <span class="NLM_cas:coden">GCEAAY</span>;
        ISSN:<span class="NLM_cas:issn">1947-6019</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">Constitutively activated mutant FLT3 has emerged as a promising target for therapy for the subpopulation of acute myeloid leukemia (AML) patients who harbor it.  The small mol. inhibitor, PKC412, targets mutant FLT3 and is currently in late-stage clin. trials.  However, the identification of PKC412-resistant leukemic blast cells in the bone marrow of AML patients has propelled the development of novel and structurally distinct FLT3 inhibitors that have the potential to override drug resistance and more efficiently prevent disease progression or recurrence.  Here, we present the novel first-generation "type II" FLT3 inhibitors, AFG206, AFG210, and AHL196, and the second-generation "type II" derivs. and AST487 analogs, AUZ454 and ATH686.  All agents potently and selectively target mutant FLT3 protein kinase activity and inhibit the proliferation of cells harboring FLT3 mutants via induction of apoptosis and cell cycle inhibition.  Cross-resistance between "type I" inhibitors, PKC412 and AAE871, was demonstrated.  While cross-resistance was also obsd. between "type I" and first-generation "type II" FLT3 inhibitors, the high potency of the second-generation "type II" inhibitors was sufficient to potently kill "type I" inhibitor-resistant mutant FLT3-expressing cells.  The increased potency obsd. for the second-generation "type II" inhibitors was obsd. to be due to an improved interaction with the ATP pocket of FLT3, specifically assocd. with introduction of a piperazine moiety and placement of an amino group in position 2 of the pyrimidine ring.  Thus, we present 2 structurally novel classes of FLT3 inhibitors characterized by high selectivity and potency toward mutant FLT3 as a mol. target.  In addn., presentation of the antileukemic effects of "type II" inhibitors, such as AUZ454 and ATH686, highlights a new class of highly potent FLT3 inhibitors able to override drug resistance that less potent "type I" inhibitors and "type II" first-generation FLT3 inhibitors cannot.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoaZyJlPc6jT7Vg90H21EOLACvtfcHk0lhaXbQsFcpNCQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXisF2nsrY%253D&md5=bf25a61a240b5f9a73e11c389cb13870</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1177%2F1947601910396505&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1947601910396505%26sid%3Dliteratum%253Aachs%26aulast%3DWeisberg%26aufirst%3DE.%26aulast%3DRoesel%26aufirst%3DJ.%26aulast%3DFuret%26aufirst%3DP.%26aulast%3DBold%26aufirst%3DG.%26aulast%3DImbach%26aufirst%3DP.%26aulast%3DFlorsheimer%26aufirst%3DA.%26aulast%3DCaravatti%26aufirst%3DG.%26aulast%3DJiang%26aufirst%3DJ.%26aulast%3DManley%26aufirst%3DP.%26aulast%3DRay%26aufirst%3DA.%26aulast%3DGriffin%26aufirst%3DJ.%2BD.%26atitle%3DAntileukemic%2520effects%2520of%2520novel%2520first-%2520and%2520second-Generation%2520FLT3%2520Inhibitors%253A%2520structure-affinity%2520comparison%26jtitle%3DGenes%2520Cancer%26date%3D2010%26volume%3D1%26spage%3D1021%26epage%3D1032%26doi%3D10.1177%2F1947601910396505" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group">Zhao, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knapp, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gray, N. S.</span><span> </span><span class="NLM_article-title">Exploration of type II binding mode: a privileged approach for kinase inhibitor focused drug discovery?</span> <span class="citation_source-journal">ACS Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">1230</span><span class="NLM_x">–</span> <span class="NLM_lpage">1241</span><span class="refDoi"> DOI: 10.1021/cb500129t</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb500129t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00840&amp;key=1%3ACAS%3A528%3ADC%252BC2cXmtV2nt7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2014&pages=1230-1241&author=Z.+Zhaoauthor=H.+Wuauthor=L.+Wangauthor=Y.+Liuauthor=S.+Knappauthor=Q.+Liuauthor=N.+S.+Gray&title=Exploration+of+type+II+binding+mode%3A+a+privileged+approach+for+kinase+inhibitor+focused+drug+discovery%3F&doi=10.1021%2Fcb500129t"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Exploration of Type II Binding Mode: A Privileged Approach for Kinase Inhibitor Focused Drug Discovery?</span></div><div class="casAuthors">Zhao, Zheng; Wu, Hong; Wang, Li; Liu, Yi; Knapp, Stefan; Liu, Qingsong; Gray, Nathanael S.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1230-1241</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The ATP site of kinases displays remarkable conformational flexibility when accommodating chem. diverse small mol. inhibitors.  The so-called activation segment, whose conformation controls catalytic activity and access to the substrate binding pocket, can undergo a large conformational change with the active state assuming a 'DFG-in' and an inactive state assuming a 'DFG-out' conformation.  Compds. that preferentially bind to the DFG-out conformation are typically called 'type II' inhibitors in contrast to 'type I' inhibitors that bind to the DFG-in conformation.  This review surveys the large no. of type II inhibitors that have been developed and provides an anal. of their crystallog. detd. binding modes.  Using a small library of type II inhibitors, we demonstrate that more than 200 kinases can be targeted, suggesting that type II inhibitors may not be intrinsically more selective than type I inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq4jYt6Db3Ap7Vg90H21EOLACvtfcHk0lhaXbQsFcpNCQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXmtV2nt7s%253D&md5=95bf185ff7f973eda5a65f4096bfaa82</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1021%2Fcb500129t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb500129t%26sid%3Dliteratum%253Aachs%26aulast%3DZhao%26aufirst%3DZ.%26aulast%3DWu%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DLiu%26aufirst%3DQ.%26aulast%3DGray%26aufirst%3DN.%2BS.%26atitle%3DExploration%2520of%2520type%2520II%2520binding%2520mode%253A%2520a%2520privileged%2520approach%2520for%2520kinase%2520inhibitor%2520focused%2520drug%2520discovery%253F%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2014%26volume%3D9%26spage%3D1230%26epage%3D1241%26doi%3D10.1021%2Fcb500129t" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group">Yee, K. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schittenhelm, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">O’Farrell, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Town, A. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McGreevey, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bainbridge, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cherrington, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heinrich, M. C.</span><span> </span><span class="NLM_article-title">Synergistic effect of SU11248 with cytarabine or daunorubicin on FLT3 ITD-positive leukemic cells</span> <span class="citation_source-journal">Blood</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">104</span><span class="NLM_x">, </span> <span class="NLM_fpage">4202</span><span class="NLM_x">–</span> <span class="NLM_lpage">4209</span><span class="refDoi"> DOI: 10.1182/blood-2003-10-3381</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00840&amp;key=10.1182%2Fblood-2003-10-3381" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=104&publication_year=2004&pages=4202-4209&author=K.+W.+Yeeauthor=M.+Schittenhelmauthor=A.+M.+O%E2%80%99Farrellauthor=A.+R.+Townauthor=L.+McGreeveyauthor=T.+Bainbridgeauthor=J.+M.+Cherringtonauthor=M.+C.+Heinrich&title=Synergistic+effect+of+SU11248+with+cytarabine+or+daunorubicin+on+FLT3+ITD-positive+leukemic+cells&doi=10.1182%2Fblood-2003-10-3381"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1182%2Fblood-2003-10-3381&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1182%252Fblood-2003-10-3381%26sid%3Dliteratum%253Aachs%26aulast%3DYee%26aufirst%3DK.%2BW.%26aulast%3DSchittenhelm%26aufirst%3DM.%26aulast%3DO%25E2%2580%2599Farrell%26aufirst%3DA.%2BM.%26aulast%3DTown%26aufirst%3DA.%2BR.%26aulast%3DMcGreevey%26aufirst%3DL.%26aulast%3DBainbridge%26aufirst%3DT.%26aulast%3DCherrington%26aufirst%3DJ.%2BM.%26aulast%3DHeinrich%26aufirst%3DM.%2BC.%26atitle%3DSynergistic%2520effect%2520of%2520SU11248%2520with%2520cytarabine%2520or%2520daunorubicin%2520on%2520FLT3%2520ITD-positive%2520leukemic%2520cells%26jtitle%3DBlood%26date%3D2004%26volume%3D104%26spage%3D4202%26epage%3D4209%26doi%3D10.1182%2Fblood-2003-10-3381" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group">Lee, L. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hernandez, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rajkhowa, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, S. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Raman, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nguyen, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Small, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Levis, M.</span><span> </span><span class="NLM_article-title">Preclinical studies of gilteritinib, a next-generation FLT3 inhibitor</span> <span class="citation_source-journal">Blood</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2017</span><span class="NLM_x">, </span> <span class="NLM_volume">129</span><span class="NLM_x">, </span> <span class="NLM_fpage">257</span><span class="NLM_x">–</span> <span class="NLM_lpage">260</span><span class="refDoi"> DOI: 10.1182/blood-2016-10-745133</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00840&amp;key=10.1182%2Fblood-2016-10-745133" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00840&amp;key=27908881" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00840&amp;key=1%3ACAS%3A528%3ADC%252BC2sXosVKitL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=129&publication_year=2017&pages=257-260&author=L.+Y.+Leeauthor=D.+Hernandezauthor=T.+Rajkhowaauthor=S.+C.+Smithauthor=J.+R.+Ramanauthor=B.+Nguyenauthor=D.+Smallauthor=M.+Levis&title=Preclinical+studies+of+gilteritinib%2C+a+next-generation+FLT3+inhibitor&doi=10.1182%2Fblood-2016-10-745133"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Preclinical studies of gilteritinib, a next-generation FLT3 inhibitor</span></div><div class="casAuthors">Lee, Lauren Y.; Hernandez, Daniela; Rajkhowa, Trivikram; Smith, Samuel C.; Raman, Jayant Ranganathan; Nguyen, Bao; Small, Donald; Levis, Mark</div><div class="citationInfo"><span class="NLM_cas:title">Blood</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">129</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">257-260</span>CODEN:
                <span class="NLM_cas:coden">BLOOAW</span>;
        ISSN:<span class="NLM_cas:issn">1528-0020</span>.
    
            (<span class="NLM_cas:orgname">American Society of Hematology</span>)
        </div><div class="casAbstract">This article describes about the activity of gilteritinib and compared it with 4 other FLT3 TKIs in clin. development: midostaurin, sorafenib, quizartinib, and crenolanib.  In this article, inhibitory activity of gilteritinib against different forms of FLT3 in leukemia cells are studied.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqKpTUQzYy5lrVg90H21EOLACvtfcHk0lgg_N9vALojsA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXosVKitL0%253D&md5=75429ea03b906ea92d24033613f0315b</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1182%2Fblood-2016-10-745133&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1182%252Fblood-2016-10-745133%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DL.%2BY.%26aulast%3DHernandez%26aufirst%3DD.%26aulast%3DRajkhowa%26aufirst%3DT.%26aulast%3DSmith%26aufirst%3DS.%2BC.%26aulast%3DRaman%26aufirst%3DJ.%2BR.%26aulast%3DNguyen%26aufirst%3DB.%26aulast%3DSmall%26aufirst%3DD.%26aulast%3DLevis%26aufirst%3DM.%26atitle%3DPreclinical%2520studies%2520of%2520gilteritinib%252C%2520a%2520next-generation%2520FLT3%2520inhibitor%26jtitle%3DBlood%26date%3D2017%26volume%3D129%26spage%3D257%26epage%3D260%26doi%3D10.1182%2Fblood-2016-10-745133" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group">Smith, C. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lasater, E. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, K. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCreery, M. Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stewart, W. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Damon, L. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perl, A. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jeschke, G. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sugita, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carroll, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kogan, S. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuriyan, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shah, N. P.</span><span> </span><span class="NLM_article-title">Crenolanib is a selective type I pan-FLT3 inhibitor</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">111</span><span class="NLM_x">, </span> <span class="NLM_fpage">5319</span><span class="NLM_x">–</span> <span class="NLM_lpage">5124</span><span class="refDoi"> DOI: 10.1073/pnas.1320661111</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00840&amp;key=10.1073%2Fpnas.1320661111" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00840&amp;key=24623852" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00840&amp;key=1%3ACAS%3A528%3ADC%252BC2cXkt1Snsbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=111&publication_year=2014&pages=5319-5124&author=C.+C.+Smithauthor=E.+A.+Lasaterauthor=K.+C.+Linauthor=Q.+Wangauthor=M.+Q.+McCreeryauthor=W.+K.+Stewartauthor=L.+E.+Damonauthor=A.+E.+Perlauthor=G.+R.+Jeschkeauthor=M.+Sugitaauthor=M.+Carrollauthor=S.+C.+Koganauthor=J.+Kuriyanauthor=N.+P.+Shah&title=Crenolanib+is+a+selective+type+I+pan-FLT3+inhibitor&doi=10.1073%2Fpnas.1320661111"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Crenolanib is a selective type I pan-FLT3 inhibitor</span></div><div class="casAuthors">Smith, Catherine Choy; Lasater, Elisabeth A.; Lin, Kimberly C.; Wang, Qi; McCreery, Melissa Quino; Stewart, Whitney K.; Damon, Lauren E.; Perl, Alexander E.; Jeschke, Grace R.; Sugita, Mayumi; Carroll, Martin; Kogan, Scott C.; Kuriyan, John; Shah, Neil P.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">111</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">5319-5324</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Tyrosine kinase inhibitors (TKIs) represent transformative therapies for several malignancies.  Two crit. features necessary for maximizing TKI tolerability and response duration are kinase selectivity and invulnerability to resistance-conferring kinase domain (KD) mutations in the intended target.  No prior TKI has demonstrated both of these properties.  Aiming to maximize selectivity, medicinal chemists have largely sought to create TKIs that bind to an inactive (type II) kinase conformation.  The investigational type I TKI crenolanib is a potent inhibitor of Fms tyrosine kinase-3 (FLT3) internal tandem duplication, a validated therapeutic target in human acute myeloid leukemia (AML), as well as all secondary KD mutants previously shown to confer resistance to the first highly active FLT3 TKI quizartinib.  Moreover, crenolanib is highly selective for FLT3 relative to the closely related protein tyrosine kinase KIT, demonstrating that simultaneous FLT3/KIT inhibition, a prominent feature of other clin. active FLT3 TKIs, is not required for AML cell cytotoxicity in vitro and may contribute to undesirable toxicity in patients.  A satn. mutagenesis screen of FLT3-internal tandem duplication failed to recover any resistant colonies in the presence of a crenolanib concn. well below what has been safely achieved in humans, suggesting that crenolanib has the potential to suppress KD mutation-mediated clin. resistance.  Crenolanib represents the first TKI to exhibit both kinase selectivity and invulnerability to resistance-conferring KD mutations, which is unexpected of a type I inhibitor.  Crenolanib has significant promise for achieving deep and durable responses in FLT3-mutant AML, and may have a profound impact upon future medicinal chem. efforts in oncol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrwxReAanOwB7Vg90H21EOLACvtfcHk0lgg_N9vALojsA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXkt1Snsbw%253D&md5=955f1f242a1690247ae239050e2a8a8f</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1320661111&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1320661111%26sid%3Dliteratum%253Aachs%26aulast%3DSmith%26aufirst%3DC.%2BC.%26aulast%3DLasater%26aufirst%3DE.%2BA.%26aulast%3DLin%26aufirst%3DK.%2BC.%26aulast%3DWang%26aufirst%3DQ.%26aulast%3DMcCreery%26aufirst%3DM.%2BQ.%26aulast%3DStewart%26aufirst%3DW.%2BK.%26aulast%3DDamon%26aufirst%3DL.%2BE.%26aulast%3DPerl%26aufirst%3DA.%2BE.%26aulast%3DJeschke%26aufirst%3DG.%2BR.%26aulast%3DSugita%26aufirst%3DM.%26aulast%3DCarroll%26aufirst%3DM.%26aulast%3DKogan%26aufirst%3DS.%2BC.%26aulast%3DKuriyan%26aufirst%3DJ.%26aulast%3DShah%26aufirst%3DN.%2BP.%26atitle%3DCrenolanib%2520is%2520a%2520selective%2520type%2520I%2520pan-FLT3%2520inhibitor%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2014%26volume%3D111%26spage%3D5319%26epage%3D5124%26doi%3D10.1073%2Fpnas.1320661111" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group">Weisberg, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boulton, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kelly, L. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Manley, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fabbro, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meyer, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gilliland, D. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Griffin, J. D.</span><span> </span><span class="NLM_article-title">Inhibition of mutant FLT3 receptors in leukemia cells by the small molecule tyrosine kinase inhibitor PKC412</span> <span class="citation_source-journal">Cancer Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">1</span><span class="NLM_x">, </span> <span class="NLM_fpage">433</span><span class="NLM_x">–</span> <span class="NLM_lpage">443</span><span class="refDoi"> DOI: 10.1016/S1535-6108(02)00069-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00840&amp;key=10.1016%2FS1535-6108%2802%2900069-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00840&amp;key=12124173" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00840&amp;key=1%3ACAS%3A528%3ADC%252BD38XltVKjsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2002&pages=433-443&author=E.+Weisbergauthor=C.+Boultonauthor=L.+M.+Kellyauthor=P.+Manleyauthor=D.+Fabbroauthor=T.+Meyerauthor=D.+G.+Gillilandauthor=J.+D.+Griffin&title=Inhibition+of+mutant+FLT3+receptors+in+leukemia+cells+by+the+small+molecule+tyrosine+kinase+inhibitor+PKC412&doi=10.1016%2FS1535-6108%2802%2900069-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of mutant FLT3 receptors in leukemia cells by the small molecule tyrosine kinase inhibitor PKC412</span></div><div class="casAuthors">Weisberg, Ellen; Boulton, Christina; Kelly, Louise M.; Manley, Paul; Fabbro, Doriano; Meyer, Thomas; Gilliland, D. Gary; Griffin, James D.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">433-443</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Constitutively activating FLT3 receptor mutations have been found in 35% of patients with acute myeloblastic leukemia (AML).  Here we report the identification of a small mol. FLT3 tyrosine kinase inhibitor PKC412, which selectively induced G1 arrest and apoptosis of Ba/F3 cell lines expressing mutant FLT3 (IC50 < 10 nM) by directly inhibiting the tyrosine kinase.  Ba/F3-FLT3 cell lines made resistant to PKC412 demonstrated overexpression of mutant FLT3, confirming that FLT3 is the target of this drug.  Finally, progressive leukemia was prevented in PKC412-treated Balb/c mice transplanted with marrow transduced with a FLT3-ITD-expressing retrovirus.  PKC412 is a potent inhibitor of mutant FLT3 and is a candidate for testing as an antileukemia agent in AML patients with mutant FLT3 receptors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqq5mVxGyiCu7Vg90H21EOLACvtfcHk0lgg_N9vALojsA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XltVKjsb4%253D&md5=612331d927bd2ca41319b77aaea6cfd1</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1016%2FS1535-6108%2802%2900069-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1535-6108%252802%252900069-7%26sid%3Dliteratum%253Aachs%26aulast%3DWeisberg%26aufirst%3DE.%26aulast%3DBoulton%26aufirst%3DC.%26aulast%3DKelly%26aufirst%3DL.%2BM.%26aulast%3DManley%26aufirst%3DP.%26aulast%3DFabbro%26aufirst%3DD.%26aulast%3DMeyer%26aufirst%3DT.%26aulast%3DGilliland%26aufirst%3DD.%2BG.%26aulast%3DGriffin%26aufirst%3DJ.%2BD.%26atitle%3DInhibition%2520of%2520mutant%2520FLT3%2520receptors%2520in%2520leukemia%2520cells%2520by%2520the%2520small%2520molecule%2520tyrosine%2520kinase%2520inhibitor%2520PKC412%26jtitle%3DCancer%2520Cell%26date%3D2002%26volume%3D1%26spage%3D433%26epage%3D443%26doi%3D10.1016%2FS1535-6108%2802%2900069-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group">Smith, C. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, K. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lasater, E. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Massi, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Damon, L. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pendleton, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bashir, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sebra, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perl, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kasarskis, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shellooe, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tsang, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carias, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Powell, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Burton, E. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Matusow, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spevak, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ibrahim, P. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Le, M. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hsu, H. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Habets, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">West, B. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bollag, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shah, N. P.</span><span> </span><span class="NLM_article-title">Characterizing and overriding the structural mechanism of the quizartinib-resistant FLT3 “gatekeeper” F691L mutation with PLX3397</span> <span class="citation_source-journal">Cancer Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">668</span><span class="NLM_x">–</span> <span class="NLM_lpage">679</span><span class="refDoi"> DOI: 10.1158/2159-8290.CD-15-0060</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00840&amp;key=10.1158%2F2159-8290.CD-15-0060" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00840&amp;key=25847190" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00840&amp;key=1%3ACAS%3A528%3ADC%252BC2MXps1Cgsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2015&pages=668-679&author=C.+C.+Smithauthor=C.+Zhangauthor=K.+C.+Linauthor=E.+A.+Lasaterauthor=Y.+Zhangauthor=E.+Massiauthor=L.+E.+Damonauthor=M.+Pendletonauthor=A.+Bashirauthor=R.+Sebraauthor=A.+Perlauthor=A.+Kasarskisauthor=R.+Shellooeauthor=G.+Tsangauthor=H.+Cariasauthor=B.+Powellauthor=E.+A.+Burtonauthor=B.+Matusowauthor=J.+Zhangauthor=W.+Spevakauthor=P.+N.+Ibrahimauthor=M.+H.+Leauthor=H.+H.+Hsuauthor=G.+Habetsauthor=B.+L.+Westauthor=G.+Bollagauthor=N.+P.+Shah&title=Characterizing+and+overriding+the+structural+mechanism+of+the+quizartinib-resistant+FLT3+%E2%80%9Cgatekeeper%E2%80%9D+F691L+mutation+with+PLX3397&doi=10.1158%2F2159-8290.CD-15-0060"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Characterizing and Overriding the Structural Mechanism of the Quizartinib-Resistant FLT3 "Gatekeeper" F691L Mutation with PLX3397</span></div><div class="casAuthors">Smith, Catherine C.; Zhang, Chao; Lin, Kimberly C.; Lasater, Elisabeth A.; Zhang, Ying; Massi, Evan; Damon, Lauren E.; Pendleton, Matthew; Bashir, Ali; Sebra, Robert; Perl, Alexander; Kasarskis, Andrew; Shellooe, Rafe; Tsang, Garson; Carias, Heidi; Powell, Ben; Burton, Elizabeth A.; Matusow, Bernice; Zhang, Jiazhong; Spevak, Wayne; Ibrahim, Prabha N.; Le, Mai H.; Hsu, Henry H.; Habets, Gaston; West, Brian L.; Bollag, Gideon; Shah, Neil P.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Discovery</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">668-679</span>CODEN:
                <span class="NLM_cas:coden">CDAIB2</span>;
        ISSN:<span class="NLM_cas:issn">2159-8274</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">PLX3397 inhibits the growth of quizartinib-resistant FLT3-ITD+ AML cells expressing the gatekeeper F691L mutation in vitro and in vivo, but not cells expressing kinase domain mutations in the FLT3 activation loop.  Tyrosine kinase domain mutations are a common cause of acquired clin. resistance to tyrosine kinase inhibitors (TKI) used to treat cancer, including the FLT3 inhibitor quizartinib.  Mutation of kinase "gatekeeper" residues, which control access to an allosteric pocket adjacent to the ATP-binding site, has been frequently implicated in TKI resistance.  The mol. underpinnings of gatekeeper mutation-mediated resistance are incompletely understood.  We report the first cocrystal structure of FLT3 with the TKI quizartinib, which demonstrates that quizartinib binding relies on essential edge-to-face arom. interactions with the gatekeeper F691 residue, and F830 within the highly conserved Asp-Phe-Gly motif in the activation loop.  This reliance makes quizartinib critically vulnerable to gatekeeper and activation loop substitutions while minimizing the impact of mutations elsewhere.  Moreover, we identify PLX3397, a novel FLT3 inhibitor that retains activity against the F691L mutant due to a binding mode that depends less vitally on specific interactions with the gatekeeper position.  Significance: We report the first cocrystal structure of FLT3 with a kinase inhibitor, elucidating the structural mechanism of resistance due to the gatekeeper F691L mutation.  PLX3397 is a novel FLT3 inhibitor with in vitro activity against this mutation but is vulnerable to kinase domain mutations in the FLT3 activation loop.  Cancer Discov; 5(6); 668-79. cpr2015 AACR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq9oYwTJ6cznrVg90H21EOLACvtfcHk0lgqYztkq5aI4w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXps1Cgsb4%253D&md5=7e76b2892a49a6e4701f534fce212ca4</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1158%2F2159-8290.CD-15-0060&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F2159-8290.CD-15-0060%26sid%3Dliteratum%253Aachs%26aulast%3DSmith%26aufirst%3DC.%2BC.%26aulast%3DZhang%26aufirst%3DC.%26aulast%3DLin%26aufirst%3DK.%2BC.%26aulast%3DLasater%26aufirst%3DE.%2BA.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DMassi%26aufirst%3DE.%26aulast%3DDamon%26aufirst%3DL.%2BE.%26aulast%3DPendleton%26aufirst%3DM.%26aulast%3DBashir%26aufirst%3DA.%26aulast%3DSebra%26aufirst%3DR.%26aulast%3DPerl%26aufirst%3DA.%26aulast%3DKasarskis%26aufirst%3DA.%26aulast%3DShellooe%26aufirst%3DR.%26aulast%3DTsang%26aufirst%3DG.%26aulast%3DCarias%26aufirst%3DH.%26aulast%3DPowell%26aufirst%3DB.%26aulast%3DBurton%26aufirst%3DE.%2BA.%26aulast%3DMatusow%26aufirst%3DB.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DSpevak%26aufirst%3DW.%26aulast%3DIbrahim%26aufirst%3DP.%2BN.%26aulast%3DLe%26aufirst%3DM.%2BH.%26aulast%3DHsu%26aufirst%3DH.%2BH.%26aulast%3DHabets%26aufirst%3DG.%26aulast%3DWest%26aufirst%3DB.%2BL.%26aulast%3DBollag%26aufirst%3DG.%26aulast%3DShah%26aufirst%3DN.%2BP.%26atitle%3DCharacterizing%2520and%2520overriding%2520the%2520structural%2520mechanism%2520of%2520the%2520quizartinib-resistant%2520FLT3%2520%25E2%2580%259Cgatekeeper%25E2%2580%259D%2520F691L%2520mutation%2520with%2520PLX3397%26jtitle%3DCancer%2520Discovery%26date%3D2015%26volume%3D5%26spage%3D668%26epage%3D679%26doi%3D10.1158%2F2159-8290.CD-15-0060" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group">Shah, N. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Talpaz, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deininger, M. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mauro, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Flinn, I. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bixby, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lustgarten, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gozgit, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clackson, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Turner, C. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Haluska, F. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kantarjian, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cortes, J. E.</span><span> </span><span class="NLM_article-title">Ponatinib in patients with refractory acute myeloid leukaemia: findings from a phase 1 study</span> <span class="citation_source-journal">Br. J. Haematol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">162</span><span class="NLM_x">, </span> <span class="NLM_fpage">548</span><span class="NLM_x">–</span> <span class="NLM_lpage">552</span><span class="refDoi"> DOI: 10.1111/bjh.12382</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00840&amp;key=10.1111%2Fbjh.12382" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00840&amp;key=23691988" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00840&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtFOitrrE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=162&publication_year=2013&pages=548-552&author=N.+P.+Shahauthor=M.+Talpazauthor=M.+W.+Deiningerauthor=M.+J.+Mauroauthor=I.+W.+Flinnauthor=D.+Bixbyauthor=S.+Lustgartenauthor=J.+M.+Gozgitauthor=T.+Clacksonauthor=C.+D.+Turnerauthor=F.+G.+Haluskaauthor=H.+Kantarjianauthor=J.+E.+Cortes&title=Ponatinib+in+patients+with+refractory+acute+myeloid+leukaemia%3A+findings+from+a+phase+1+study&doi=10.1111%2Fbjh.12382"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Ponatinib in patients with refractory acute myeloid leukaemia: findings from a phase 1 study</span></div><div class="casAuthors">Shah, Neil P.; Talpaz, Moshe; Deininger, Michael W. N.; Mauro, Michael J.; Flinn, Ian W.; Bixby, Dale; Lustgarten, Stephanie; Gozgit, Joseph M.; Clackson, Tim; Turner, Christopher D.; Haluska, Frank G.; Kantarjian, Hagop; Cortes, Jorge E.</div><div class="citationInfo"><span class="NLM_cas:title">British Journal of Haematology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">162</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">548-552</span>CODEN:
                <span class="NLM_cas:coden">BJHEAL</span>;
        ISSN:<span class="NLM_cas:issn">0007-1048</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">Activating mutations in the FMS-like tyrosine kinase-3 (FLT3), a tyrosine kinase receptor important in hematopoiesis, are among the most common mol. aberrations in acute myeloid leukemia (AML), occurring in 30% of adult patients.  Ponatinib is a novel, orally administered tyrosine kinase inhibitor (TKI) and a potent pan-BCR-ABL1 inhibitor.  This article reports the first clin. experience with ponatinib in 12 acute myeloid leukemia (AML) patients included in the phase 1 study.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo3dQyPoxgIELVg90H21EOLACvtfcHk0lgqYztkq5aI4w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtFOitrrE&md5=13f70ed786895227e7b47fb79e06662f</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1111%2Fbjh.12382&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fbjh.12382%26sid%3Dliteratum%253Aachs%26aulast%3DShah%26aufirst%3DN.%2BP.%26aulast%3DTalpaz%26aufirst%3DM.%26aulast%3DDeininger%26aufirst%3DM.%2BW.%26aulast%3DMauro%26aufirst%3DM.%2BJ.%26aulast%3DFlinn%26aufirst%3DI.%2BW.%26aulast%3DBixby%26aufirst%3DD.%26aulast%3DLustgarten%26aufirst%3DS.%26aulast%3DGozgit%26aufirst%3DJ.%2BM.%26aulast%3DClackson%26aufirst%3DT.%26aulast%3DTurner%26aufirst%3DC.%2BD.%26aulast%3DHaluska%26aufirst%3DF.%2BG.%26aulast%3DKantarjian%26aufirst%3DH.%26aulast%3DCortes%26aufirst%3DJ.%2BE.%26atitle%3DPonatinib%2520in%2520patients%2520with%2520refractory%2520acute%2520myeloid%2520leukaemia%253A%2520findings%2520from%2520a%2520phase%25201%2520study%26jtitle%3DBr.%2520J.%2520Haematol.%26date%3D2013%26volume%3D162%26spage%3D548%26epage%3D552%26doi%3D10.1111%2Fbjh.12382" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group">Zarrinkar, P. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gunawardane, R. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cramer, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gardner, M. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brigham, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Belli, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Karaman, M. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pratz, K. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pallares, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chao, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sprankle, K. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Patel, H. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Levis, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Armstrong, R. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">James, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bhagwat, S. S.</span><span> </span><span class="NLM_article-title">AC220 is a uniquely potent and selective inhibitor of FLT3 for the treatment of acute myeloid leukemia (AML)</span> <span class="citation_source-journal">Blood</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">114</span><span class="NLM_x">, </span> <span class="NLM_fpage">2984</span><span class="NLM_x">–</span> <span class="NLM_lpage">2992</span><span class="refDoi"> DOI: 10.1182/blood-2009-05-222034</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00840&amp;key=10.1182%2Fblood-2009-05-222034" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00840&amp;key=19654408" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00840&amp;key=1%3ACAS%3A528%3ADC%252BD1MXht12qurzN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=114&publication_year=2009&pages=2984-2992&author=P.+P.+Zarrinkarauthor=R.+N.+Gunawardaneauthor=M.+D.+Cramerauthor=M.+F.+Gardnerauthor=D.+Brighamauthor=B.+Belliauthor=M.+W.+Karamanauthor=K.+W.+Pratzauthor=G.+Pallaresauthor=Q.+Chaoauthor=K.+G.+Sprankleauthor=H.+K.+Patelauthor=M.+Levisauthor=R.+C.+Armstrongauthor=J.+Jamesauthor=S.+S.+Bhagwat&title=AC220+is+a+uniquely+potent+and+selective+inhibitor+of+FLT3+for+the+treatment+of+acute+myeloid+leukemia+%28AML%29&doi=10.1182%2Fblood-2009-05-222034"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">AC220 is a uniquely potent and selective inhibitor of FLT3 for the treatment of acute myeloid leukemia (AML)</span></div><div class="casAuthors">Zarrinkar, Patrick P.; Gunawardane, Ruwanthi N.; Cramer, Merryl D.; Gardner, Michael F.; Brigham, Daniel; Belli, Barbara; Karaman, Mazen W.; Pratz, Keith W.; Pallares, Gabriel; Chao, Qi; Sprankle, Kelly G.; Patel, Hitesh K.; Levis, Mark; Armstrong, Robert C.; James, Joyce; Bhagwat, Shripad S.</div><div class="citationInfo"><span class="NLM_cas:title">Blood</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">114</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">2984-2992</span>CODEN:
                <span class="NLM_cas:coden">BLOOAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-4971</span>.
    
            (<span class="NLM_cas:orgname">American Society of Hematology</span>)
        </div><div class="casAbstract">Activating mutations in the receptor tyrosine kinase FLT3 are present in up to approx. 30% of acute myeloid leukemia (AML) patients, implicating FLT3 as a driver of the disease and therefore as a target for therapy.  We report the characterization of AC220, a second-generation FLT3 inhibitor, and a comparison of AC220 with the first-generation FLT3 inhibitors CEP-701, MLN-518, PKC-412, sorafenib, and sunitinib.  AC220 exhibits low nanomolar potency in biochem. and cellular assays and exceptional kinase selectivity, and in animal models is efficacious at doses as low as 1 mg/kg given orally once daily.  The data reveal that the combination of excellent potency, selectivity, and pharmacokinetic properties is unique to AC220, which therefore is the first drug candidate with a profile that matches the characteristics desirable for a clin. FLT3 inhibitor.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr_RzDr7kzxsLVg90H21EOLACvtfcHk0liutaLmtHUFkA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXht12qurzN&md5=4f8368a7b2eab9d1234ef36e01f5f260</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1182%2Fblood-2009-05-222034&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1182%252Fblood-2009-05-222034%26sid%3Dliteratum%253Aachs%26aulast%3DZarrinkar%26aufirst%3DP.%2BP.%26aulast%3DGunawardane%26aufirst%3DR.%2BN.%26aulast%3DCramer%26aufirst%3DM.%2BD.%26aulast%3DGardner%26aufirst%3DM.%2BF.%26aulast%3DBrigham%26aufirst%3DD.%26aulast%3DBelli%26aufirst%3DB.%26aulast%3DKaraman%26aufirst%3DM.%2BW.%26aulast%3DPratz%26aufirst%3DK.%2BW.%26aulast%3DPallares%26aufirst%3DG.%26aulast%3DChao%26aufirst%3DQ.%26aulast%3DSprankle%26aufirst%3DK.%2BG.%26aulast%3DPatel%26aufirst%3DH.%2BK.%26aulast%3DLevis%26aufirst%3DM.%26aulast%3DArmstrong%26aufirst%3DR.%2BC.%26aulast%3DJames%26aufirst%3DJ.%26aulast%3DBhagwat%26aufirst%3DS.%2BS.%26atitle%3DAC220%2520is%2520a%2520uniquely%2520potent%2520and%2520selective%2520inhibitor%2520of%2520FLT3%2520for%2520the%2520treatment%2520of%2520acute%2520myeloid%2520leukemia%2520%2528AML%2529%26jtitle%3DBlood%26date%3D2009%26volume%3D114%26spage%3D2984%26epage%3D2992%26doi%3D10.1182%2Fblood-2009-05-222034" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group">Man, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fung, T. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ho, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Han, H. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chow, H. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ma, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Choi, W. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lok, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheung, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eaves, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kwong, Y. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leung, A. Y.</span><span> </span><span class="NLM_article-title">Sorafenib treatment of FLT3-ITD(+) acute myeloid leukemia: favorable initial outcome and mechanisms of subsequent nonresponsiveness associated with the emergence of a D835 mutation</span> <span class="citation_source-journal">Blood</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">119</span><span class="NLM_x">, </span> <span class="NLM_fpage">5133</span><span class="NLM_x">–</span> <span class="NLM_lpage">5143</span><span class="refDoi"> DOI: 10.1182/blood-2011-06-363960</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00840&amp;key=10.1182%2Fblood-2011-06-363960" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00840&amp;key=22368270" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00840&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtVWjsrfE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=119&publication_year=2012&pages=5133-5143&author=C.+H.+Manauthor=T.+K.+Fungauthor=C.+Hoauthor=H.+H.+Hanauthor=H.+C.+Chowauthor=A.+C.+Maauthor=W.+W.+Choiauthor=S.+Lokauthor=A.+M.+Cheungauthor=C.+Eavesauthor=Y.+L.+Kwongauthor=A.+Y.+Leung&title=Sorafenib+treatment+of+FLT3-ITD%28%2B%29+acute+myeloid+leukemia%3A+favorable+initial+outcome+and+mechanisms+of+subsequent+nonresponsiveness+associated+with+the+emergence+of+a+D835+mutation&doi=10.1182%2Fblood-2011-06-363960"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Sorafenib treatment of FLT3-ITD+ acute myeloid leukemia: favorable initial outcome and mechanisms of subsequent nonresponsiveness associated with the emergence of a D835 mutation</span></div><div class="casAuthors">Man, Cheuk Him; Fung, Tsz Kan; Ho, Christa; Han, Heron H. C.; Chow, Howard C. H.; Ma, Alvin C. H.; Choi, William W. L.; Lok, Si; Cheung, Alice M. S.; Eaves, Connie; Kwong, Yok Lam; Leung, Anskar Y. H.</div><div class="citationInfo"><span class="NLM_cas:title">Blood</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">119</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">5133-5143</span>CODEN:
                <span class="NLM_cas:coden">BLOOAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-4971</span>.
    
            (<span class="NLM_cas:orgname">American Society of Hematology</span>)
        </div><div class="casAbstract">Internal tandem duplication (ITD) of the fms-related tyrosine kinase-3 (FLT3) gene occurs in 30% of acute myeloid leukemias (AMLs) and confers a poor prognosis.  Thirteen relapsed or chemo-refractory FLT3-ITD+ AML patients were treated with sorafenib (200-400 mg twice daily).  Twelve patients showed clearance or near clearance of bone marrow myeloblasts after 27 (range 21-84) days with evidence of differentiation of leukemia cells.  The sorafenib response was lost in most patients after 72 (range 54-287) days but the FLT3 and downstream effectors remained suppressed.  Gene expression profiling showed that leukemia cells that have become sorafenib resistant expressed several genes including ALDH1A1, JAK3, and MMP15, whose functions were unknown in AML.  Nonobese diabetic/severe combined immunodeficiency mice transplanted with leukemia cells from patients before and during sorafenib resistance recapitulated the clin. results.  Both ITD and tyrosine kinase domain mutations at D835 were identified in leukemia initiating cells (LICs) from samples before sorafenib treatment.  LICs bearing the D835 mutant have expanded during sorafenib treatment and dominated during the subsequent clin. resistance.  These results suggest that sorafenib have selected more aggressive sorafenib-resistant subclones carrying both FLT3-ITD and D835 mutations, and might provide important leads to further improvement of treatment outcome with FLT3 inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGri65wbuOPOW7Vg90H21EOLACvtfcHk0liutaLmtHUFkA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtVWjsrfE&md5=cfcf1b9858a2df595c51f1bfe91d6960</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1182%2Fblood-2011-06-363960&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1182%252Fblood-2011-06-363960%26sid%3Dliteratum%253Aachs%26aulast%3DMan%26aufirst%3DC.%2BH.%26aulast%3DFung%26aufirst%3DT.%2BK.%26aulast%3DHo%26aufirst%3DC.%26aulast%3DHan%26aufirst%3DH.%2BH.%26aulast%3DChow%26aufirst%3DH.%2BC.%26aulast%3DMa%26aufirst%3DA.%2BC.%26aulast%3DChoi%26aufirst%3DW.%2BW.%26aulast%3DLok%26aufirst%3DS.%26aulast%3DCheung%26aufirst%3DA.%2BM.%26aulast%3DEaves%26aufirst%3DC.%26aulast%3DKwong%26aufirst%3DY.%2BL.%26aulast%3DLeung%26aufirst%3DA.%2BY.%26atitle%3DSorafenib%2520treatment%2520of%2520FLT3-ITD%2528%252B%2529%2520acute%2520myeloid%2520leukemia%253A%2520favorable%2520initial%2520outcome%2520and%2520mechanisms%2520of%2520subsequent%2520nonresponsiveness%2520associated%2520with%2520the%2520emergence%2520of%2520a%2520D835%2520mutation%26jtitle%3DBlood%26date%3D2012%26volume%3D119%26spage%3D5133%26epage%3D5143%26doi%3D10.1182%2Fblood-2011-06-363960" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group">Herrera, A. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jacobsen, E. D.</span><span> </span><span class="NLM_article-title">Ibrutinib for the treatment of mantle cell lymphoma</span> <span class="citation_source-journal">Clin. Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">20</span><span class="NLM_x">, </span> <span class="NLM_fpage">5365</span><span class="NLM_x">–</span> <span class="NLM_lpage">5371</span><span class="refDoi"> DOI: 10.1158/1078-0432.CCR-14-0010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00840&amp;key=10.1158%2F1078-0432.CCR-14-0010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00840&amp;key=25361916" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00840&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvVOgu7rN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2014&pages=5365-5371&author=A.+F.+Herreraauthor=E.+D.+Jacobsen&title=Ibrutinib+for+the+treatment+of+mantle+cell+lymphoma&doi=10.1158%2F1078-0432.CCR-14-0010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Ibrutinib for the Treatment of Mantle Cell Lymphoma</span></div><div class="casAuthors">Herrera, Alex F.; Jacobsen, Eric D.</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">5365-5371</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">A review.  Ibrutinib (PCI-32765)-a potent, covalent inhibitor of Bruton tyrosine kinase (BTK), an important kinase in the B-cell receptor signaling pathway-was recently approved by the FDA for the treatment of relapsed or refractory mantle cell lymphoma (MCL).  The drug was granted accelerated approval based on the findings of an international, multicenter, single-arm phase II study that enrolled patients with relapsed or refractory MCL.  In the study, ibrutinib (560 mg daily) was well tolerated as a single agent and resulted in an overall response rate of 68% and an estd. median response duration of 17.5 mo.  Ibrutinib's response rate and duration of response compare favorably with those for other novel agents approved for the treatment of relapsed or refractory MCL, while being less toxic than most chemotherapy or chemoimmunotherapy regimens.  Ibrutinib is currently being studied in combination with chemoimmunotherapy, monoclonal antibody therapy, and novel agents in both the initial and the relapsed/refractory treatment settings.  We review the mechanism of action, preclin. and clin. development, and the role of ibrutinib in the context of other available treatments.  Clin Cancer Res; 20(21); 5365-71. ©2014 AACR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpXYzdm1bO6frVg90H21EOLACvtfcHk0liutaLmtHUFkA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvVOgu7rN&md5=92a7b5c4e954b89a11370eb698642ef3</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1158%2F1078-0432.CCR-14-0010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1078-0432.CCR-14-0010%26sid%3Dliteratum%253Aachs%26aulast%3DHerrera%26aufirst%3DA.%2BF.%26aulast%3DJacobsen%26aufirst%3DE.%2BD.%26atitle%3DIbrutinib%2520for%2520the%2520treatment%2520of%2520mantle%2520cell%2520lymphoma%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2014%26volume%3D20%26spage%3D5365%26epage%3D5371%26doi%3D10.1158%2F1078-0432.CCR-14-0010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group">Brown, J. R.</span><span> </span><span class="NLM_article-title">Ibrutinib (PCI-32765), the first BTK (Bruton’s tyrosine kinase) inhibitor in clinical trials</span> <span class="citation_source-journal">Curr. Hematol. Malig. Rep.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">1</span><span class="NLM_x">–</span> <span class="NLM_lpage">6</span><span class="refDoi"> DOI: 10.1007/s11899-012-0147-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00840&amp;key=10.1007%2Fs11899-012-0147-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00840&amp;key=23296407" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00840&amp;key=1%3ACAS%3A280%3ADC%252BC3s3nsVKjsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&pages=1-6&author=J.+R.+Brown&title=Ibrutinib+%28PCI-32765%29%2C+the+first+BTK+%28Bruton%E2%80%99s+tyrosine+kinase%29+inhibitor+in+clinical+trials&doi=10.1007%2Fs11899-012-0147-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Ibrutinib (PCI-32765), the first BTK (Bruton's tyrosine kinase) inhibitor in clinical trials</span></div><div class="casAuthors">Brown Jennifer R</div><div class="citationInfo"><span class="NLM_cas:title">Current hematologic malignancy reports</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1-6</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Ibrutinib is a potent covalent kinase inhibitor that targets BTK.  BTK, or Bruton's tyrosine kinase, is an obvious target for therapy of B cell diseases because inactivating mutations lead to B cell aplasia in humans and the disease X-linked agammaglobulinemia.  Ibrutinib has modest cytotoxicity against CLL cells in vitro but also blocks trophic stimuli from the microenvironment.  As with other inhibitors of the BCR pathway, ibrutinib causes rapid nodal reduction and response associated with rapid increase in lymphocytosis, which then returns to baseline over time.  The ORR of ibrutinib in relapsed refractory CLL is 67 % with PFS 88 % at 15 months.  In a cohort of untreated patients 65 years and over, the estimated 15 month PFS is 96 %.  Registration trials have been initiated, and the difficult task that remains is to determine where in the course of CLL therapy this drug will have the greatest impact and benefit for patients.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTmwv_ITNl9Dxgd2Wuil24tfW6udTcc2eZYiYAEX8cgoLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3s3nsVKjsw%253D%253D&md5=c64d7d8fa5d40a79ad029c9fb34248c3</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1007%2Fs11899-012-0147-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs11899-012-0147-9%26sid%3Dliteratum%253Aachs%26aulast%3DBrown%26aufirst%3DJ.%2BR.%26atitle%3DIbrutinib%2520%2528PCI-32765%2529%252C%2520the%2520first%2520BTK%2520%2528Bruton%25E2%2580%2599s%2520tyrosine%2520kinase%2529%2520inhibitor%2520in%2520clinical%2520trials%26jtitle%3DCurr.%2520Hematol.%2520Malig.%2520Rep.%26date%3D2013%26volume%3D8%26spage%3D1%26epage%3D6%26doi%3D10.1007%2Fs11899-012-0147-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group">Wu, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hu, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weisberg, E. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yu, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nonami, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stone, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Griffin, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, Q.</span><span> </span><span class="NLM_article-title">Ibrutinib selectively targets FLT3-ITD in mutant FLT3-positive AML</span> <span class="citation_source-journal">Leukemia</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">30</span><span class="NLM_x">, </span> <span class="NLM_fpage">754</span><span class="NLM_x">–</span> <span class="NLM_lpage">757</span><span class="refDoi"> DOI: 10.1038/leu.2015.175</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00840&amp;key=10.1038%2Fleu.2015.175" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00840&amp;key=26139428" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00840&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1emu7vO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2016&pages=754-757&author=H.+Wuauthor=C.+Huauthor=A.+Wangauthor=E.+L.+Weisbergauthor=W.+Wangauthor=C.+Chenauthor=Z.+Zhaoauthor=K.+Yuauthor=J.+Liuauthor=J.+Wuauthor=A.+Nonamiauthor=L.+Wangauthor=B.+Wangauthor=R.+M.+Stoneauthor=S.+Liuauthor=J.+D.+Griffinauthor=J.+Liuauthor=Q.+Liu&title=Ibrutinib+selectively+targets+FLT3-ITD+in+mutant+FLT3-positive+AML&doi=10.1038%2Fleu.2015.175"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Ibrutinib selectively targets FLT3-ITD in mutant FLT3-positive AML</span></div><div class="casAuthors">Wu, H.; Hu, C.; Wang, A.; Weisberg, E. L.; Wang, W.; Chen, C.; Zhao, Z.; Yu, K.; Liu, J.; Wu, J.; Nonami, A.; Wang, L.; Wang, B.; Stone, R. M.; Liu, S.; Griffin, J. D.; Liu, J.; Liu, Q.</div><div class="citationInfo"><span class="NLM_cas:title">Leukemia</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">754-757</span>CODEN:
                <span class="NLM_cas:coden">LEUKED</span>;
        ISSN:<span class="NLM_cas:issn">0887-6924</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Ibrutinib (PCI-32765) is an irreversible BTK (Bruton's tyrosine kinase) kinase inhibitor that has been extensively used as a tool compd. to validate the role of BTK kinase in B cell related malignances.1, 2.  Ibrutinib has been shown in preclin. studies to inhibit the proliferation of diffuse large B-cell lymphoma cells, mantle cell lymphoma cells, chronic lymphocytic leukemia cells and multiple myeloma cells by blocking BTK kinase activity; ibrutinib was recently approved for the clin. application on mantle cell lymphoma and chronic lymphocytic leukemia cells.3, 4, 5, 6, 7.  Ibrutinib has also exhibited anti-inflammatory effects in preclin. models.8, 9.  Recently, it has been reported that ibrutinib is also effective against epidermal growth factor receptor mutant-pos. non-small cell lung cancers through inhibition of epidermal growth factor receptor kinase activities.10.  In addn., there is evidence showing that BTK is also an important target for Acute Myeloid Leukemia (AML).11.  Despite the evidence that BTK knockdown impaired AML cancer cell growth, which suggested that BTK was important for AML cell proliferation, BTK kinase inhibition through use of a small mol. inhibitor like ibrutinib led only to moderate inhibition of proliferation of U937 cells with no apparent activity against other AML cell lines such as HL-60, TF-1 and THP-1.11.  To further investigate the potency and Interestingly, we found that only FLT3-internal tandem duplication (ITD) mutant AML cell lines (MOLM13, MOLM14 and MV4-11) were sensitive to ibrutinib (Figure 1a and Supplementary Table 1).  This is similar to what has been obsd. with the highly potent and selective FLT3 inhibitor, AC220, but differs from that obsd. for the targeted BTK kinase inhibitor, CGI-1746.12, 13 The selective antiproliferative activity was further confirmed in the clonogenic assay (GI50: 170-478 nM) (Supplementary Figure 1).  This suggests that ibrutinib might target FLT3 kinase in addn. to BTK kinase.  We then investigated the effect of ibrutinib on FLT3-ITD mediated signaling in drug-sensitive cell lines.  (Figure 1b) Ibrutinib potently inhibited FLT3-ITD auto-phosphorylation (EC50: 318, 39 and 356 nM resp. for MOLM13, MOLM14 and MV4-11) (Supplementary Figure 2A).  Phosphorylation of STAT5, a well-established downstream target, also was significantly inhibited with an EC50 between 100 and 200 nM (Supplementary Figure 2B).  In addn., c-Myc expression was inhibited with an EC50 between 185 and 315 nM.  (Supplementary Figure 2C) However, in FLT3 wt-expressing cell lines such as NOMO-1, HL-60, OCI-AML-3 and U937 cells, FLT3 signaling was not affected by ibrutinib upon FLT3 ligand stimulation (Supplementary Figure 2D).  Interestingly, despite the expression of the FLT3 wt kinase in all four cell lines, the induction of FLT3 auto-phosphorylation upon FLT3 ligand treatment was not obsd. in NOMO-1 and HL-60 cells like what was obsd. in the OCI-AML-3 and U937 cells, which might be due to the different genetic background and require further detailed mechanistic study.  The FLT3 inhibitor AC220 demonstrated similar inhibitory activity to ibrutinib in FLT3-mediated signaling pathways in the FLT3-ITD pos. cell lines.  However, the BTK inhibitors, CGI-1746 and AVL-292, did not exhibit any effect on the signaling14 (Figure 2b and Supplementary Figure 2D).  As all of these cell lines express BTK kinase, we then studied the effect of ibrutinib on BTK signaling.  Upon anti-IgM (IgM) stimulation of B-cell receptor to active the BTK kinase signaling pathway, we found that only in the FLT3 wild-type (wt)-expressing cell lines (U937 and OCI-AML3), downstream mediators of BTK kinase were affected despite BTK phosphorylation having been inhibited in both FLT3 wt and FLT3-ITD cell lines.9 This suggests that FLT3-ITD mutant cell lines might not rely on BTK signaling for growth (Supplementary Figure 3A).  Interestingly, except NOMO-1 and U937 cells, the phosphorylation of BTKY223 of MOLM14, MOLM13, MV4-11, OCI-AML-3 and HL-60 cells did not respond to anti-IgM stimulation apparently, which might be due to the basal-level expression of the IgM in these AML cells (Supplementary Figure 3B).  AC220 did not affect BTK kinase activity and its direct downstream target PLCγ but instead inhibited phosphorylation of protein kinase B and extracellular-signal-regulated kinase.  The BTK inhibitors, CGI-1746 and AVL-292, were effective against BTK kinase and PLCγ kinase phosphorylation, but had no effect on the phosphorylation of protein kinase B and extracellular-signal-regulated kinase.  These results suggest that ibrutinib may mainly exert its inhibitory activity through FLT3 kinase but not BTK kinase in the FLT3-ITD pos. cells.  To further confirm this, we knocked down BTK kinase in MOLM14 (FLT3-ITD), MOLM13 (FLT3-ITD), human erythroleukemia cell (HEL) (FLT3 wt) and NB4 (FLT3 wt) cells and found that the growth of MOLM14 and MOLM13 was only minimally affected; however NB4 cells were significantly inhibited and HEL cells were moderately inhibited (Figures 2a and b).  The fact that BTK knockdown in HEL cells did not lead to suppressed growth may be due to the fact that mutant JAK2 is the oncogenic driver in this cell line.15 In addn., the finding that ibrutinib still significantly blocked the growth of BTK-knockdown MOLM14 and MOLM13 cells further validated that the growth inhibitory effect of ibrutinib is likely dependent mainly on FLT3-ITD inhibition.  However, BTK kinase inhibition may also exert a combinatorial effect (Figure 2c).  It has recently been reported that BTK kinase is activated by FLT3-ITD in FLT3-ITD pos. cell lines and dual inhibition of BTK and FLT3 kinase demonstrates combinatorial effects; these findings further confirm this observation.16 Previous kinase selectivity profiling revealed that FLT3 wt might be an addnl. target of ibrutinib.9 The biochem. ADP-Glo assay confirmed that ibrutinib has an IC50 of 205.8 nM/65.65 nM against FLT3 wt and FLT3-ITD, resp. (Supplementary Figure 4A).  In the FLT3-ITD-dependent isogenic BaF3 cell line, ibrutinib demonstrated a GI50 of 120 nM; this inhibitory effect could be completely rescued by interleukin-3 treatment (Supplementary Figure 4B).  Interestingly, it also potently inhibited the FLT3-D835H-BaF3 isogenic cell (GI50: 0.063 μM) and moderately inhibited FLT3-D835Y-BaF3 as well as TEL-FLT3-wt-BaF3 isogenic cells (GI50: 0.5 μ/0.46 μ), however was less active against ITD-D835Y, ITD-F691L and FLT3-K663Q mutations.  (Supplementary Figure 5A) In addn., ibrutinib did not affect the growth of other oncogene-expressing Ba/F3 cells, such as c-Kit, PDGFRα, BCR-ABL and NRAS (Supplementary Figure 5B).  Although there are cysteine residues in the ATP-binding pocket, binding mode examn. with mol. modeling demonstrates that unlike the mode of inhibition against BTK kinase, ibrutinib inhibits FLT3 kinase through reversible binding and this is further confirmed by the washing-out expt. (Supplementary Figure 4C and D).  Ibrutinib induced apoptotic cell death in a concn.- and time-dependent manner in FLT3-ITD cell lines but not FLT3 wt-expressing cell lines (Supplementary Figure 6).  Cell cycle anal. revealed that ibrutinib, but not the highly selective BTK kinase inhibitor, CGI-1746, could induce a concn.-dependent manner-arrest cells in G0/G1 phase (Supplementary Figure 7).  In addn., ibrutinib exhibited dose-dependent anti-proliferation activity against FLT3-ITD mutant-expressing primary patient cells but not FLT3 wt-expressing and multiple mutants such as FLT3-ITD/FLT3-D835Y and FLT3-D835Y/E mutants expressing samples, which further confirms that ibrutinib is FLT3-ITD selective (Figure 2d).  AML still remains a serious unmet medical need and the FLT3-ITD mutant has been found in ∼30% of AML patients.  Besides std. chemotherapy, there is no targeted therapy approved in the clinic to date despite several inhibitors that are presently under clin. investigation.17 Currently, ibrutinib is under extensive clin. investigation against a variety of different B-cell malignancies.  Given the fact that the safety profile of ibrutinib is tolerated in the patients, our results might help to expand the application of this drug to AML patients harboring the FLT3-ITD mutant.  Interestingly, despite the fact that the effective concn. of ibrutinib in the in vitro studies in the FLT3-ITD pos. cell lines falls within the range of the peak plasma concn. used in human clin. trials (around 150 ng/mL at 560 mg/day dosage), the concn. required for the induction of apoptosis and cell cycle arrest for ibrutinib is higher, which suggests that an alternative formulation might be needed such as nano technol., to allow the increase of the plasma concns. in the human patients while simultaneously retaining the safety profile for the potential clin. application.18.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq9uujF_Uu1FbVg90H21EOLACvtfcHk0lirDfM9pETz4A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht1emu7vO&md5=cc08960c1d1db9a9f83c9c67bac15f85</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1038%2Fleu.2015.175&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fleu.2015.175%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DH.%26aulast%3DHu%26aufirst%3DC.%26aulast%3DWang%26aufirst%3DA.%26aulast%3DWeisberg%26aufirst%3DE.%2BL.%26aulast%3DWang%26aufirst%3DW.%26aulast%3DChen%26aufirst%3DC.%26aulast%3DZhao%26aufirst%3DZ.%26aulast%3DYu%26aufirst%3DK.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DWu%26aufirst%3DJ.%26aulast%3DNonami%26aufirst%3DA.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DB.%26aulast%3DStone%26aufirst%3DR.%2BM.%26aulast%3DLiu%26aufirst%3DS.%26aulast%3DGriffin%26aufirst%3DJ.%2BD.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DLiu%26aufirst%3DQ.%26atitle%3DIbrutinib%2520selectively%2520targets%2520FLT3-ITD%2520in%2520mutant%2520FLT3-positive%2520AML%26jtitle%3DLeukemia%26date%3D2016%26volume%3D30%26spage%3D754%26epage%3D757%26doi%3D10.1038%2Fleu.2015.175" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group">Li, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yu, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Qi, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hu, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zou, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, Q.</span><span> </span><span class="NLM_article-title">Discovery of (R)-1-(3-(4-Amino-3-(4-phenoxyphenyl)-1H-pyrazolo[3,4-d]pyrimidin-1-yl)piperidin- 1-yl)-2-(dimethylamino)ethanone (CHMFL-FLT3-122) as a potent and orally available FLT3 Kinase inhibitor for FLT3-ITD positive acute myeloid leukemia</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">58</span><span class="NLM_x">, </span> <span class="NLM_fpage">9625</span><span class="NLM_x">–</span> <span class="NLM_lpage">9638</span><span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01611</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01611" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=9625-9638&author=X.+Liauthor=A.+Wangauthor=K.+Yuauthor=Z.+Qiauthor=C.+Chenauthor=W.+Wangauthor=C.+Huauthor=H.+Wuauthor=J.+Wuauthor=Z.+Zhaoauthor=J.+Liuauthor=F.+Zouauthor=L.+Wangauthor=B.+Wangauthor=W.+Wangauthor=S.+Zhangauthor=J.+Liuauthor=Q.+Liu&title=Discovery+of+%28R%29-1-%283-%284-Amino-3-%284-phenoxyphenyl%29-1H-pyrazolo%5B3%2C4-d%5Dpyrimidin-1-yl%29piperidin-+1-yl%29-2-%28dimethylamino%29ethanone+%28CHMFL-FLT3-122%29+as+a+potent+and+orally+available+FLT3+Kinase+inhibitor+for+FLT3-ITD+positive+acute+myeloid+leukemia&doi=10.1021%2Facs.jmedchem.5b01611"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01611&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01611%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DA.%26aulast%3DYu%26aufirst%3DK.%26aulast%3DQi%26aufirst%3DZ.%26aulast%3DChen%26aufirst%3DC.%26aulast%3DWang%26aufirst%3DW.%26aulast%3DHu%26aufirst%3DC.%26aulast%3DWu%26aufirst%3DH.%26aulast%3DWu%26aufirst%3DJ.%26aulast%3DZhao%26aufirst%3DZ.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DZou%26aufirst%3DF.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DB.%26aulast%3DWang%26aufirst%3DW.%26aulast%3DZhang%26aufirst%3DS.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DLiu%26aufirst%3DQ.%26atitle%3DDiscovery%2520of%2520%2528R%2529-1-%25283-%25284-Amino-3-%25284-phenoxyphenyl%2529-1H-pyrazolo%255B3%252C4-d%255Dpyrimidin-1-yl%2529piperidin-%25201-yl%2529-2-%2528dimethylamino%2529ethanone%2520%2528CHMFL-FLT3-122%2529%2520as%2520a%2520potent%2520and%2520orally%2520available%2520FLT3%2520Kinase%2520inhibitor%2520for%2520FLT3-ITD%2520positive%2520acute%2520myeloid%2520leukemia%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D9625%26epage%3D9638%26doi%3D10.1021%2Facs.jmedchem.5b01611" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group">Zorn, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fujimura, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barros, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuriyan, J.</span><span> </span><span class="NLM_article-title">Crystal structure of the FLT3 kinase domain bound to the inhibitor quizartinib (AC220)</span> <span class="citation_source-journal">PLoS One</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">e0121177</span><span class="refDoi"> DOI: 10.1371/journal.pone.0121177</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00840&amp;key=10.1371%2Fjournal.pone.0121177" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00840&amp;key=25837374" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00840&amp;key=1%3ACAS%3A528%3ADC%252BC28Xht1Gmsr%252FM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=e0121177&author=J.+A.+Zornauthor=Q.+Wangauthor=E.+Fujimuraauthor=T.+Barrosauthor=J.+Kuriyan&title=Crystal+structure+of+the+FLT3+kinase+domain+bound+to+the+inhibitor+quizartinib+%28AC220%29&doi=10.1371%2Fjournal.pone.0121177"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal structure of the FLT3 kinase domain bound to the inhibitor quizartinib (AC220)</span></div><div class="casAuthors">Zorn, Julie A.; Wang, Qi; Fujimura, Eric; Barros, Tiago; Kuriyan, John</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">e0121177/1-e0121177/15</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">More than 30% of acute myeloid leukemia (AML) patients possess activating mutations in the receptor tyrosine kinase FMS-like tyrosine kinase 3 or FLT3.  A small-mol. inhibitor of FLT3 (known as quizartinib or AC220) that is currently in clin. trials appears promising for the treatment of AML.  Here, we report the co-crystal structure of the kinase domain of FLT3 in complex with quizartinib.  FLT3 with quizartinib bound adopts an "Abl-like"inactive conformation with the activation loop stabilized in the "DFG-out"orientation and folded back onto the kinase domain.  This conformation is similar to that obsd. for the uncomplexed intracellular domain of FLT3 as well as for related receptor tyrosine kinases, except for a localized induced fit in the activation loop.  The co-crystal structure reveals the interactions between quizartinib and the active site of FLT3 that are key for achieving its high potency against both wild-type FLT3 as well as a FLT3 variant obsd. in many AML patients.  This co-complex further provides a structural rationale for quizartinib-resistance mutations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqOXAklBbZKP7Vg90H21EOLACvtfcHk0lhbNoVw17xNkg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xht1Gmsr%252FM&md5=d3c23bf8a8e242dd35e20e6e9b2de924</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0121177&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0121177%26sid%3Dliteratum%253Aachs%26aulast%3DZorn%26aufirst%3DJ.%2BA.%26aulast%3DWang%26aufirst%3DQ.%26aulast%3DFujimura%26aufirst%3DE.%26aulast%3DBarros%26aufirst%3DT.%26aulast%3DKuriyan%26aufirst%3DJ.%26atitle%3DCrystal%2520structure%2520of%2520the%2520FLT3%2520kinase%2520domain%2520bound%2520to%2520the%2520inhibitor%2520quizartinib%2520%2528AC220%2529%26jtitle%3DPLoS%2520One%26date%3D2015%26volume%3D10%26spage%3De0121177%26doi%3D10.1371%2Fjournal.pone.0121177" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group">Davis, M. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hunt, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Herrgard, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ciceri, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wodicka, L. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pallares, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hocker, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Treiber, D. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zarrinkar, P. P.</span><span> </span><span class="NLM_article-title">Comprehensive analysis of kinase inhibitor selectivity</span> <span class="citation_source-journal">Nat. Biotechnol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">29</span><span class="NLM_x">, </span> <span class="NLM_fpage">1046</span><span class="NLM_x">–</span> <span class="NLM_lpage">1051</span><span class="refDoi"> DOI: 10.1038/nbt.1990</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00840&amp;key=10.1038%2Fnbt.1990" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00840&amp;key=22037378" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00840&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtlyisrrK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2011&pages=1046-1051&author=M.+I.+Davisauthor=J.+P.+Huntauthor=S.+Herrgardauthor=P.+Ciceriauthor=L.+M.+Wodickaauthor=G.+Pallaresauthor=M.+Hockerauthor=D.+K.+Treiberauthor=P.+P.+Zarrinkar&title=Comprehensive+analysis+of+kinase+inhibitor+selectivity&doi=10.1038%2Fnbt.1990"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Comprehensive analysis of kinase inhibitor selectivity</span></div><div class="casAuthors">Davis, Mindy I.; Hunt, Jeremy P.; Herrgard, Sanna; Ciceri, Pietro; Wodicka, Lisa M.; Pallares, Gabriel; Hocker, Michael; Treiber, Daniel K.; Zarrinkar, Patrick P.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1046-1051</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">We tested the interaction of 72 kinase inhibitors with 442 kinases covering >80% of the human catalytic protein kinome.  Our data show that, as a class, type II inhibitors are more selective than type I inhibitors, but that there are important exceptions to this trend.  The data further illustrate that selective inhibitors have been developed against the majority of kinases targeted by the compds. tested.  Anal. of the interaction patterns reveals a class of 'group-selective' inhibitors broadly active against a single subfamily of kinases, but selective outside that subfamily.  The data set suggests compds. to use as tools to study kinases for which no dedicated inhibitors exist.  It also provides a foundation for further exploring kinase inhibitor biol. and toxicity, as well as for studying the structural basis of the obsd. interaction patterns.  Our findings will help to realize the direct enabling potential of genomics for drug development and basic research about cellular signaling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoEZnKVs6Xt5rVg90H21EOLACvtfcHk0lhbNoVw17xNkg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtlyisrrK&md5=67639f702cb48ffb2c60e9a250f0df5e</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1038%2Fnbt.1990&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt.1990%26sid%3Dliteratum%253Aachs%26aulast%3DDavis%26aufirst%3DM.%2BI.%26aulast%3DHunt%26aufirst%3DJ.%2BP.%26aulast%3DHerrgard%26aufirst%3DS.%26aulast%3DCiceri%26aufirst%3DP.%26aulast%3DWodicka%26aufirst%3DL.%2BM.%26aulast%3DPallares%26aufirst%3DG.%26aulast%3DHocker%26aufirst%3DM.%26aulast%3DTreiber%26aufirst%3DD.%2BK.%26aulast%3DZarrinkar%26aufirst%3DP.%2BP.%26atitle%3DComprehensive%2520analysis%2520of%2520kinase%2520inhibitor%2520selectivity%26jtitle%3DNat.%2520Biotechnol.%26date%3D2011%26volume%3D29%26spage%3D1046%26epage%3D1051%26doi%3D10.1038%2Fnbt.1990" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group">Ke, Y. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Singh, V. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Coumar, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hsu, Y. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, W. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Song, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, W. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, S. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hsu, J. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shih, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hsieh, H. P.</span><span> </span><span class="NLM_article-title">Homology modeling of DFG-in FMS-like tyrosine kinase 3 (FLT3) and structure-based virtual screening for inhibitor identification</span> <span class="citation_source-journal">Sci. Rep.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">11702</span><span class="refDoi"> DOI: 10.1038/srep11702</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00840&amp;key=10.1038%2Fsrep11702" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00840&amp;key=26118648" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00840&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFyhsbnK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2015&pages=11702&author=Y.+Y.+Keauthor=V.+K.+Singhauthor=M.+S.+Coumarauthor=Y.+C.+Hsuauthor=W.+C.+Wangauthor=J.+S.+Songauthor=C.+H.+Chenauthor=W.+H.+Linauthor=S.+H.+Wuauthor=J.+T.+Hsuauthor=C.+Shihauthor=H.+P.+Hsieh&title=Homology+modeling+of+DFG-in+FMS-like+tyrosine+kinase+3+%28FLT3%29+and+structure-based+virtual+screening+for+inhibitor+identification&doi=10.1038%2Fsrep11702"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Homology modeling of DFG-in FMS-like tyrosine kinase 3 (FLT3) and structure-based virtual screening for inhibitor identification</span></div><div class="casAuthors">Ke, Yi-Yu; Singh, Vivek Kumar; Coumar, Mohane Selvaraj; Hsu, Yung Chang; Wang, Wen-Chieh; Song, Jen-Shin; Chen, Chun-Hwa; Lin, Wen-Hsing; Wu, Szu-Huei; Hsu, John T. A.; Shih, Chuan; Hsieh, Hsing-Pang</div><div class="citationInfo"><span class="NLM_cas:title">Scientific Reports</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">11702</span>CODEN:
                <span class="NLM_cas:coden">SRCEC3</span>;
        ISSN:<span class="NLM_cas:issn">2045-2322</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The inhibition of FMS-like tyrosine kinase 3 (FLT3) activity using small-mol. inhibitors has emerged as a target-based alternative to traditional chemotherapy for the treatment of acute myeloid leukemia (AML).  In this study, we report the use of structure-based virtual screening (SBVS), a computer-aided drug design technique for the identification of new chemotypes for FLT3 inhibition.  For this purpose, homol. modeling (HM) of the DFG-in FLT3 structure was carried using two template structures, including PDB ID: 1RJB (DFG-out FLT3 kinase domain) and PDB ID: 3LCD (DFG-in CSF-1 kinase domain).  The modeled structure was able to correctly identify known DFG-in (SU11248, CEP-701, and PKC-412) and DFG-out (sorafenib, ABT-869 and AC220) FLT3 inhibitors, in docking studies.  The modeled structure was then used to carry out SBVS of an HTS library of 125,000 compds.  The top scoring 97 compds. were tested for FLT3 kinase inhibition, and two hits (BPR056, IC50 = 2.3 and BPR080, IC50 = 10.7 μM) were identified.  Mol. dynamics simulation and d. functional theory calcn. suggest that BPR056 (MW: 325.32; cLogP: 2.48) interacted with FLT3 in a stable manner and could be chem. optimized to realize a drug-like lead in the future.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqqLlUVxzNzs7Vg90H21EOLACvtfcHk0lhLEt6Uq2_b3Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtFyhsbnK&md5=9b5e05518ac93463fc575134bae6296c</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1038%2Fsrep11702&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsrep11702%26sid%3Dliteratum%253Aachs%26aulast%3DKe%26aufirst%3DY.%2BY.%26aulast%3DSingh%26aufirst%3DV.%2BK.%26aulast%3DCoumar%26aufirst%3DM.%2BS.%26aulast%3DHsu%26aufirst%3DY.%2BC.%26aulast%3DWang%26aufirst%3DW.%2BC.%26aulast%3DSong%26aufirst%3DJ.%2BS.%26aulast%3DChen%26aufirst%3DC.%2BH.%26aulast%3DLin%26aufirst%3DW.%2BH.%26aulast%3DWu%26aufirst%3DS.%2BH.%26aulast%3DHsu%26aufirst%3DJ.%2BT.%26aulast%3DShih%26aufirst%3DC.%26aulast%3DHsieh%26aufirst%3DH.%2BP.%26atitle%3DHomology%2520modeling%2520of%2520DFG-in%2520FMS-like%2520tyrosine%2520kinase%25203%2520%2528FLT3%2529%2520and%2520structure-based%2520virtual%2520screening%2520for%2520inhibitor%2520identification%26jtitle%3DSci.%2520Rep.%26date%3D2015%26volume%3D5%26spage%3D11702%26doi%3D10.1038%2Fsrep11702" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group">Sali, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blundell, T. L.</span><span> </span><span class="NLM_article-title">Comparative protein modelling by satisfaction of spatial restraints</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1993</span><span class="NLM_x">, </span> <span class="NLM_volume">234</span><span class="NLM_x">, </span> <span class="NLM_fpage">779</span><span class="NLM_x">–</span> <span class="NLM_lpage">815</span><span class="refDoi"> DOI: 10.1006/jmbi.1993.1626</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00840&amp;key=10.1006%2Fjmbi.1993.1626" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00840&amp;key=8254673" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00840&amp;key=1%3ACAS%3A528%3ADyaK2cXnt1ylug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=234&publication_year=1993&pages=779-815&author=A.+Saliauthor=T.+L.+Blundell&title=Comparative+protein+modelling+by+satisfaction+of+spatial+restraints&doi=10.1006%2Fjmbi.1993.1626"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Comparative protein modeling by satisfaction of spatial restraints</span></div><div class="casAuthors">Sali, Andrej; Blundell, Tom L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">234</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">779-815</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    </div><div class="casAbstract">The authors describe a comparative protein modeling method designed to find the most probable structure for a sequence given its alignment with related structures.  The three-dimensional (3D) model is obtained by optimally satisfying spatial restraints derived from the alignment and expressed as probability d. functions (pdfs) for the features restrained.  For example, the probabilities for main-chain conformations of a modelled residue may be restrained by its residue type, main-chain conformation of an equiv. residue in a related protein, and the local similarity between the two sequences.  Several such pdfs are obtained from the correlations between structural features in 17 families of homologous proteins which have been aligned on the basis of their 3D structures.  The pdfs restrain Cα-Cα distances, main-chain N-O distances, main-chain and side-chain dihedral angles.  A smoothing procedure is used in the derivation of these relationships to minimize the problem of a sparse database.  The 3D model of a protein is obtained by optimization of the mol. pdf such that the model violates the input restraints as little as possible.  The mol. pdf is derived as a combination of pdfs restraining individual spatial features of the whole mol.  The optimization procedure is a variable target function method that applies the conjugate gradients algorithm to positions of all non-hydrogen atoms.  The method is automated and is illustrated by the modeling of trypsin from two other serine proteinases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpC4wGhwZvib7Vg90H21EOLACvtfcHk0lhLEt6Uq2_b3Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2cXnt1ylug%253D%253D&md5=d4a3c39b2205e36221dc187a3d1a478b</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1006%2Fjmbi.1993.1626&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Fjmbi.1993.1626%26sid%3Dliteratum%253Aachs%26aulast%3DSali%26aufirst%3DA.%26aulast%3DBlundell%26aufirst%3DT.%2BL.%26atitle%3DComparative%2520protein%2520modelling%2520by%2520satisfaction%2520of%2520spatial%2520restraints%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D1993%26volume%3D234%26spage%3D779%26epage%3D815%26doi%3D10.1006%2Fjmbi.1993.1626" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group">Pollard, J. W.</span><span> </span><span class="NLM_article-title">Trophic macrophages in development and disease</span> <span class="citation_source-journal">Nat. Rev. Immunol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">259</span><span class="NLM_x">–</span> <span class="NLM_lpage">270</span><span class="refDoi"> DOI: 10.1038/nri2528</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00840&amp;key=10.1038%2Fnri2528" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00840&amp;key=19282852" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00840&amp;key=1%3ACAS%3A528%3ADC%252BD1MXjtFCisbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2009&pages=259-270&author=J.+W.+Pollard&title=Trophic+macrophages+in+development+and+disease&doi=10.1038%2Fnri2528"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Trophic macrophages in development and disease</span></div><div class="casAuthors">Pollard, Jeffrey W.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Immunology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">259-270</span>CODEN:
                <span class="NLM_cas:coden">NRIABX</span>;
        ISSN:<span class="NLM_cas:issn">1474-1733</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Specialized phagocytes are found in the most primitive multicellular organisms.  Their roles in homeostasis and in distinguishing self from non-self have evolved with the complexity of organisms and their immune systems.  Equally important, but often overlooked, are the roles of macrophages in tissue development.  As discussed in this Review, these include functions in branching morphogenesis, neuronal patterning, angiogenesis, bone morphogenesis and the generation of adipose tissue.  In each case, macrophage depletion impairs the formation of the tissue and compromises its function.  I argue that in several diseases, the unrestrained acquisition of these developmental macrophage functions exacerbates pathol.  For example, macrophages enhance tumor progression and metastasis by affecting tumor-cell migration and invasion, as well as angiogenesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrR03M07hzbU7Vg90H21EOLACvtfcHk0lhLEt6Uq2_b3Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXjtFCisbc%253D&md5=d29685d4cbb8d8bdd296f8b8ff869d36</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1038%2Fnri2528&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnri2528%26sid%3Dliteratum%253Aachs%26aulast%3DPollard%26aufirst%3DJ.%2BW.%26atitle%3DTrophic%2520macrophages%2520in%2520development%2520and%2520disease%26jtitle%3DNat.%2520Rev.%2520Immunol.%26date%3D2009%26volume%3D9%26spage%3D259%26epage%3D270%26doi%3D10.1038%2Fnri2528" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 15 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Junwei Wang, Xiang Pan, Yi Song, Jian Liu, Fei Ma, Ping Wang, Yan Liu, Lin Zhao, Di Kang, <span class="NLM_string-name hlFld-ContribAuthor">Lihong Hu</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of a Potent and Selective FLT3 Inhibitor (Z)-N-(5-((5-Fluoro-2-oxoindolin-3-ylidene)methyl)-4-methyl-1H-pyrrol-3-yl)-3-(pyrrolidin-1-yl)propanamide with Improved Drug-like Properties and Superior Efficacy in FLT3-ITD-Positive Acute Myeloid Leukemia. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (8)
                                     , 4870-4890. <a href="https://doi.org/10.1021/acs.jmedchem.0c02247" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c02247</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c02247&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c02247%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252Ba%252BPotent%252Band%252BSelective%252BFLT3%252BInhibitor%252B%252528Z%252529-N-%2525285-%252528%2525285-Fluoro-2-oxoindolin-3-ylidene%252529methyl%252529-4-methyl-1H-pyrrol-3-yl%252529-3-%252528pyrrolidin-1-yl%252529propanamide%252Bwith%252BImproved%252BDrug-like%252BProperties%252Band%252BSuperior%252BEfficacy%252Bin%252BFLT3-ITD-Positive%252BAcute%252BMyeloid%252BLeukemia%26aulast%3DWang%26aufirst%3DJunwei%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D28122020%26date%3D02042021%26volume%3D64%26issue%3D8%26spage%3D4870%26epage%3D4890" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Zhijie Wang, Jiongheng Cai, Jie Cheng, Wenqianzi Yang, Yifan Zhu, Hongmei Li, Tao Lu, Yadong Chen, <span class="NLM_string-name hlFld-ContribAuthor">Shuai Lu</span>. </span><span class="cited-content_cbyCitation_article-title">FLT3 Inhibitors in Acute Myeloid Leukemia: Challenges and Recent Developments in Overcoming Resistance. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (6)
                                     , 2878-2900. <a href="https://doi.org/10.1021/acs.jmedchem.0c01851" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c01851</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c01851&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c01851%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DFLT3%252BInhibitors%252Bin%252BAcute%252BMyeloid%252BLeukemia%25253A%252BChallenges%252Band%252BRecent%252BDevelopments%252Bin%252BOvercoming%252BResistance%26aulast%3DWang%26aufirst%3DZhijie%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D23102020%26date%3D10032021%26volume%3D64%26issue%3D6%26spage%3D2878%26epage%3D2900" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Yue Zhong, Run-Ze Qiu, Shan-Liang Sun, Chao Zhao, Tian-Yuan Fan, Min Chen, Nian-Guang Li, <span class="NLM_string-name hlFld-ContribAuthor">Zhi-Hao Shi</span>. </span><span class="cited-content_cbyCitation_article-title">Small-Molecule Fms-like Tyrosine Kinase 3 Inhibitors: An Attractive and Efficient Method for the Treatment of Acute Myeloid Leukemia. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (21)
                                     , 12403-12428. <a href="https://doi.org/10.1021/acs.jmedchem.0c00696" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c00696</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c00696&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c00696%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DSmall-Molecule%252BFms-like%252BTyrosine%252BKinase%252B3%252BInhibitors%25253A%252BAn%252BAttractive%252Band%252BEfficient%252BMethod%252Bfor%252Bthe%252BTreatment%252Bof%252BAcute%252BMyeloid%252BLeukemia%26aulast%3DZhong%26aufirst%3DYue%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D26042020%26date%3D21072020%26date%3D13072020%26volume%3D63%26issue%3D21%26spage%3D12403%26epage%3D12428" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Jisheng Liu, Yuting Wang, Chen Chen, Zhengchao Tu, Sihua Zhu, Fengtao Zhou, Hongfei Si, Canhui Zheng, Zhang Zhang, <span class="NLM_string-name hlFld-ContribAuthor">Qian Cai</span>. </span><span class="cited-content_cbyCitation_article-title">Identification and Development of 1,4-Diaryl-1,2,3-triazolo-Based Ureas as Novel FLT3 Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">ACS Medicinal Chemistry Letters</span><span> <strong>2020,</strong> <em>11 </em>
                                    (8)
                                     , 1567-1572. <a href="https://doi.org/10.1021/acsmedchemlett.0c00216" title="DOI URL">https://doi.org/10.1021/acsmedchemlett.0c00216</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsmedchemlett.0c00216&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsmedchemlett.0c00216%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DIdentification%252Band%252BDevelopment%252Bof%252B1%25252C4-Diaryl-1%25252C2%25252C3-triazolo-Based%252BUreas%252Bas%252BNovel%252BFLT3%252BInhibitors%26aulast%3DLiu%26aufirst%3DJisheng%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D27042020%26date%3D27072020%26date%3D30072020%26date%3D27072020%26volume%3D11%26issue%3D8%26spage%3D1567%26epage%3D1572" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Qi Wang, Zhenquan Hu, Xiangchao Luo, Jing Liu, Guoqiang Li, Shugeng Cao, <span class="NLM_string-name hlFld-ContribAuthor">Qingsong Liu</span>. </span><span class="cited-content_cbyCitation_article-title">Clavukoellians A–F, Highly Rearranged Nardosinane Sesquiterpenoids with Antiangiogenic Activity from Clavularia koellikeri. </span><span class="cited-content_cbyCitation_journal-name">Journal of Natural Products</span><span> <strong>2019,</strong> <em>82 </em>
                                    (5)
                                     , 1331-1337. <a href="https://doi.org/10.1021/acs.jnatprod.9b00100" title="DOI URL">https://doi.org/10.1021/acs.jnatprod.9b00100</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jnatprod.9b00100&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jnatprod.9b00100%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Natural%2520Products%26atitle%3DClavukoellians%252BA%2525E2%252580%252593F%25252C%252BHighly%252BRearranged%252BNardosinane%252BSesquiterpenoids%252Bwith%252BAntiangiogenic%252BActivity%252Bfrom%252BClavularia%252Bkoellikeri%26aulast%3DWang%26aufirst%3DQi%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D02022019%26date%3D17042019%26volume%3D82%26issue%3D5%26spage%3D1331%26epage%3D1337" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Xue Yuan, Yong Chen, Wanhua Zhang, Jun He, Lei Lei, Minghai Tang, Jiang Liu, Muzhou Li, Caixia Dou, Tao Yang, Linyu Yang, Shengyong Yang, Yuquan Wei, Aihua Peng, Ting Niu, Mingli Xiang, Haoyu Ye, <span class="NLM_string-name hlFld-ContribAuthor">Lijuan Chen</span>. </span><span class="cited-content_cbyCitation_article-title">Identification of Pyrrolo[2,3-d]pyrimidine-Based Derivatives as Potent and Orally Effective Fms-like Tyrosine Receptor Kinase 3 (FLT3) Inhibitors for Treating Acute Myelogenous Leukemia. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2019,</strong> <em>62 </em>
                                    (8)
                                     , 4158-4173. <a href="https://doi.org/10.1021/acs.jmedchem.9b00223" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b00223</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b00223&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.9b00223%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DIdentification%252Bof%252BPyrrolo%25255B2%25252C3-d%25255Dpyrimidine-Based%252BDerivatives%252Bas%252BPotent%252Band%252BOrally%252BEffective%252BFms-like%252BTyrosine%252BReceptor%252BKinase%252B3%252B%252528FLT3%252529%252BInhibitors%252Bfor%252BTreating%252BAcute%252BMyelogenous%252BLeukemia%26aulast%3DYuan%26aufirst%3DXue%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D02022019%26date%3D15042019%26date%3D02042019%26volume%3D62%26issue%3D8%26spage%3D4158%26epage%3D4173" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Xiaofei Liang, Beilei Wang, Cheng Chen, Aoli Wang, Chen Hu, Fengming Zou, Kailin Yu, Qingwang Liu, Feng Li, Zhenquan Hu, Tingting Lu, Junjie Wang, Li Wang, Ellen L. Weisberg, Lili Li, Ruixiang Xia, Wenchao Wang, Tao Ren, Jian Ge, Jing Liu, <span class="NLM_string-name hlFld-ContribAuthor">Qingsong Liu</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of N-(4-(6-Acetamidopyrimidin-4-yloxy)phenyl)-2-(2-(trifluoromethyl)phenyl)acetamide (CHMFL-FLT3-335) as a Potent FMS-like Tyrosine Kinase 3 Internal Tandem Duplication (FLT3-ITD) Mutant Selective Inhibitor for Acute Myeloid Leukemia. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2019,</strong> <em>62 </em>
                                    (2)
                                     , 875-892. <a href="https://doi.org/10.1021/acs.jmedchem.8b01594" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.8b01594</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.8b01594&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.8b01594%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252BN-%2525284-%2525286-Acetamidopyrimidin-4-yloxy%252529phenyl%252529-2-%2525282-%252528trifluoromethyl%252529phenyl%252529acetamide%252B%252528CHMFL-FLT3-335%252529%252Bas%252Ba%252BPotent%252BFMS-like%252BTyrosine%252BKinase%252B3%252BInternal%252BTandem%252BDuplication%252B%252528FLT3-ITD%252529%252BMutant%252BSelective%252BInhibitor%252Bfor%252BAcute%252BMyeloid%252BLeukemia%26aulast%3DLiang%26aufirst%3DXiaofei%26date%3D2019%26date%3D2018%26date%3D2018%26date%3D13102018%26date%3D08012019%26date%3D19122018%26volume%3D62%26issue%3D2%26spage%3D875%26epage%3D892" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Qi Wang, Zhenquan Hu, Xixiang Li, Aoli Wang, Hong Wu, Jing Liu, Shugeng Cao, <span class="NLM_string-name hlFld-ContribAuthor">Qingsong Liu</span>. </span><span class="cited-content_cbyCitation_article-title">Salviachinensines A–F, Antiproliferative Phenolic Derivatives from the Chinese Medicinal Plant Salvia chinensis. </span><span class="cited-content_cbyCitation_journal-name">Journal of Natural Products</span><span> <strong>2018,</strong> <em>81 </em>
                                    (11)
                                     , 2531-2538. <a href="https://doi.org/10.1021/acs.jnatprod.8b00638" title="DOI URL">https://doi.org/10.1021/acs.jnatprod.8b00638</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jnatprod.8b00638&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jnatprod.8b00638%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Natural%2520Products%26atitle%3DSalviachinensines%252BA%2525E2%252580%252593F%25252C%252BAntiproliferative%252BPhenolic%252BDerivatives%252Bfrom%252Bthe%252BChinese%252BMedicinal%252BPlant%252BSalvia%252Bchinensis%26aulast%3DWang%26aufirst%3DQi%26date%3D2018%26date%3D2018%26date%3D2018%26date%3D31072018%26date%3D29102018%26volume%3D81%26issue%3D11%26spage%3D2531%26epage%3D2538" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Shuai Liu, Yangen Huang, Feng-Ling Qing, <span class="NLM_string-name hlFld-ContribAuthor">Xiu-Hua Xu</span>. </span><span class="cited-content_cbyCitation_article-title">Transition-Metal-Free Decarboxylation of 3,3,3-Trifluoro-2,2-dimethylpropanoic Acid for the Preparation of C(CF3)Me2-Containing Heteroarenes. </span><span class="cited-content_cbyCitation_journal-name">Organic Letters</span><span> <strong>2018,</strong> <em>20 </em>
                                    (17)
                                     , 5497-5501. <a href="https://doi.org/10.1021/acs.orglett.8b02451" title="DOI URL">https://doi.org/10.1021/acs.orglett.8b02451</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.orglett.8b02451&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.orglett.8b02451%26sid%3Dliteratum%253Aachs%26jtitle%3DOrganic%2520Letters%26atitle%3DTransition-Metal-Free%252BDecarboxylation%252Bof%252B3%25252C3%25252C3-Trifluoro-2%25252C2-dimethylpropanoic%252BAcid%252Bfor%252Bthe%252BPreparation%252Bof%252BC%252528CF3%252529Me2-Containing%252BHeteroarenes%26aulast%3DLiu%26aufirst%3DShuai%26date%3D2018%26date%3D2018%26date%3D2018%26date%3D01082018%26date%3D27082018%26volume%3D20%26issue%3D17%26spage%3D5497%26epage%3D5501" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Lexian  Tong</span>, <span class="hlFld-ContribAuthor ">Xuemei  Li</span>, <span class="hlFld-ContribAuthor ">Yongzhou  Hu</span>, <span class="hlFld-ContribAuthor ">Tao  Liu</span>. </span><span class="cited-content_cbyCitation_article-title">Recent advances in FLT3 inhibitors for acute myeloid leukemia. </span><span class="cited-content_cbyCitation_journal-name">Future Medicinal Chemistry</span><span> <strong>2020,</strong> <em>12 </em>
                                    (10)
                                     , 961-981. <a href="https://doi.org/10.4155/fmc-2019-0365" title="DOI URL">https://doi.org/10.4155/fmc-2019-0365</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.4155/fmc-2019-0365&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.4155%2Ffmc-2019-0365%26sid%3Dliteratum%253Aachs%26jtitle%3DFuture%2520Medicinal%2520Chemistry%26atitle%3DRecent%252Badvances%252Bin%252BFLT3%252Binhibitors%252Bfor%252Bacute%252Bmyeloid%252Bleukemia%26aulast%3DTong%26aufirst%3DLexian%26date%3D2020%26volume%3D12%26issue%3D10%26spage%3D961%26epage%3D981" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Cheolhee  Kim</span>, <span class="hlFld-ContribAuthor ">Eunae  Kim</span>. </span><span class="cited-content_cbyCitation_article-title">Rational Drug Design Approach of Receptor Tyrosine Kinase Type III Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Current Medicinal Chemistry</span><span> <strong>2020,</strong> <em>26 </em>
                                    (42)
                                     , 7623-7640. <a href="https://doi.org/10.2174/0929867325666180622143548" title="DOI URL">https://doi.org/10.2174/0929867325666180622143548</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.2174/0929867325666180622143548&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.2174%2F0929867325666180622143548%26sid%3Dliteratum%253Aachs%26jtitle%3DCurrent%2520Medicinal%2520Chemistry%26atitle%3DRational%252BDrug%252BDesign%252BApproach%252Bof%252BReceptor%252BTyrosine%252BKinase%252BType%252BIII%252BInhibitors%26aulast%3DKim%26aufirst%3DCheolhee%26date%3D2020%26volume%3D26%26issue%3D42%26spage%3D7623%26epage%3D7640" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xiangyi  Jiang</span>, <span class="hlFld-ContribAuthor ">Ji  Yu</span>, <span class="hlFld-ContribAuthor ">Zhongxia  Zhou</span>, <span class="hlFld-ContribAuthor ">Jacob  Kongsted</span>, <span class="hlFld-ContribAuthor ">Yuning  Song</span>, <span class="hlFld-ContribAuthor ">Christophe  Pannecouque</span>, <span class="hlFld-ContribAuthor ">Erik  De Clercq</span>, <span class="hlFld-ContribAuthor ">Dongwei  Kang</span>, <span class="hlFld-ContribAuthor ">Vasanthanathan  Poongavanam</span>, <span class="hlFld-ContribAuthor ">Xinyong  Liu</span>, <span class="hlFld-ContribAuthor ">Peng  Zhan</span>. </span><span class="cited-content_cbyCitation_article-title">Molecular design opportunities presented by solvent‐exposed regions of target proteins. </span><span class="cited-content_cbyCitation_journal-name">Medicinal Research Reviews</span><span> <strong>2019,</strong> <em>39 </em>
                                    (6)
                                     , 2194-2238. <a href="https://doi.org/10.1002/med.21581" title="DOI URL">https://doi.org/10.1002/med.21581</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/med.21581&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fmed.21581%26sid%3Dliteratum%253Aachs%26jtitle%3DMedicinal%2520Research%2520Reviews%26atitle%3DMolecular%252Bdesign%252Bopportunities%252Bpresented%252Bby%252Bsolvent%2525E2%252580%252590exposed%252Bregions%252Bof%252Btarget%252Bproteins%26aulast%3DJiang%26aufirst%3DXiangyi%26date%3D2019%26date%3D2019%26volume%3D39%26issue%3D6%26spage%3D2194%26epage%3D2238" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Hao  Heng</span>, <span class="hlFld-ContribAuthor ">Zhijie  Wang</span>, <span class="hlFld-ContribAuthor ">Hongmei  Li</span>, <span class="hlFld-ContribAuthor ">Yatian  Huang</span>, <span class="hlFld-ContribAuthor ">Qingyuan  Lan</span>, <span class="hlFld-ContribAuthor ">Xiaoxing  Guo</span>, <span class="hlFld-ContribAuthor ">Liang  Zhang</span>, <span class="hlFld-ContribAuthor ">Yanle  Zhi</span>, <span class="hlFld-ContribAuthor ">Jiongheng  Cai</span>, <span class="hlFld-ContribAuthor ">Tianren  Qin</span>, <span class="hlFld-ContribAuthor ">Li  Xiang</span>, <span class="hlFld-ContribAuthor ">Shuxian  Wang</span>, <span class="hlFld-ContribAuthor ">Yadong  Chen</span>, <span class="hlFld-ContribAuthor ">Tao  Lu</span>, <span class="hlFld-ContribAuthor ">Shuai  Lu</span>. </span><span class="cited-content_cbyCitation_article-title">Combining structure- and property-based optimization to identify selective FLT3-ITD inhibitors with good antitumor efficacy in AML cell inoculated mouse xenograft model. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2019,</strong> <em>176 </em>, 248-267. <a href="https://doi.org/10.1016/j.ejmech.2019.05.021" title="DOI URL">https://doi.org/10.1016/j.ejmech.2019.05.021</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2019.05.021&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2019.05.021%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DCombining%252Bstructure-%252Band%252Bproperty-based%252Boptimization%252Bto%252Bidentify%252Bselective%252BFLT3-ITD%252Binhibitors%252Bwith%252Bgood%252Bantitumor%252Befficacy%252Bin%252BAML%252Bcell%252Binoculated%252Bmouse%252Bxenograft%252Bmodel%26aulast%3DHeng%26aufirst%3DHao%26date%3D2019%26volume%3D176%26spage%3D248%26epage%3D267" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Baohui  Qi</span>, <span class="hlFld-ContribAuthor ">Xingwei  Xu</span>, <span class="hlFld-ContribAuthor ">Ying  Yang</span>, <span class="hlFld-ContribAuthor ">Yuting  Zhou</span>, <span class="hlFld-ContribAuthor ">Tao  Chen</span>, <span class="hlFld-ContribAuthor ">Guowei  Gong</span>, <span class="hlFld-ContribAuthor ">Xupeng  Yue</span>, <span class="hlFld-ContribAuthor ">Xin  Xu</span>, <span class="hlFld-ContribAuthor ">Liping  Hu</span>, <span class="hlFld-ContribAuthor ">Huan  He</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of thiazolidin-4-one urea analogues as novel multikinase inhibitors that potently inhibit FLT3 and VEGFR2. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry</span><span> <strong>2019,</strong> <em>27 </em>
                                    (10)
                                     , 2127-2139. <a href="https://doi.org/10.1016/j.bmc.2019.03.049" title="DOI URL">https://doi.org/10.1016/j.bmc.2019.03.049</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmc.2019.03.049&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmc.2019.03.049%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252Bthiazolidin-4-one%252Burea%252Banalogues%252Bas%252Bnovel%252Bmultikinase%252Binhibitors%252Bthat%252Bpotently%252Binhibit%252BFLT3%252Band%252BVEGFR2%26aulast%3DQi%26aufirst%3DBaohui%26date%3D2019%26volume%3D27%26issue%3D10%26spage%3D2127%26epage%3D2139" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Nikhil R.  Tasker</span>, <span class="hlFld-ContribAuthor ">Ettore J.  Rastelli</span>, <span class="hlFld-ContribAuthor ">Isabella K.  Blanco</span>, <span class="hlFld-ContribAuthor ">James C.  Burnett</span>, <span class="hlFld-ContribAuthor ">Elizabeth R.  Sharlow</span>, <span class="hlFld-ContribAuthor ">John S.  Lazo</span>, <span class="hlFld-ContribAuthor ">Peter  Wipf</span>. </span><span class="cited-content_cbyCitation_article-title">In-flow photooxygenation of aminothienopyridinones generates iminopyridinedione PTP4A3 phosphatase inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Organic & Biomolecular Chemistry</span><span> <strong>2019,</strong> <em>17 </em>
                                    (9)
                                     , 2448-2466. <a href="https://doi.org/10.1039/C9OB00025A" title="DOI URL">https://doi.org/10.1039/C9OB00025A</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C9OB00025A&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC9OB00025A%26sid%3Dliteratum%253Aachs%26jtitle%3DOrganic%2520%2526%2520Biomolecular%2520Chemistry%26atitle%3DIn-flow%252Bphotooxygenation%252Bof%252Baminothienopyridinones%252Bgenerates%252Biminopyridinedione%252BPTP4A3%252Bphosphatase%252Binhibitors%26aulast%3DTasker%26aufirst%3DNikhil%2BR.%26date%3D2019%26date%3D2019%26volume%3D17%26issue%3D9%26spage%3D2448%26epage%3D2466" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="f_tgr1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-20/acs.jmedchem.7b00840/20171020/images/medium/jm-2017-008406_0014.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-20/acs.jmedchem.7b00840/20171020/images/large/jm-2017-008406_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00840&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-20/acs.jmedchem.7b00840/20171020/images/medium/jm-2017-008406_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-20/acs.jmedchem.7b00840/20171020/images/large/jm-2017-008406_0001.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Chemical structures of representative FLT3 kinase inhibitors.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-20/acs.jmedchem.7b00840/20171020/images/large/jm-2017-008406_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00840&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-20/acs.jmedchem.7b00840/20171020/images/medium/jm-2017-008406_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-20/acs.jmedchem.7b00840/20171020/images/large/jm-2017-008406_0002.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Schematic illustration of discovery of compound <b>14</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-20/acs.jmedchem.7b00840/20171020/images/large/jm-2017-008406_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00840&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-20/acs.jmedchem.7b00840/20171020/images/medium/jm-2017-008406_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-20/acs.jmedchem.7b00840/20171020/images/large/jm-2017-008406_0003.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Design rationale of new FLT3 kinase inhibitor. (A) Docking of <b>9</b> into the active conformation of FLT3 wt kinase (homology model built from PDB codes <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4RT7">4RT7</a> and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3LCD">3LCD</a>). (B) Docking of <b>9</b> into the active conformation of FLT3-F691L kinase (site mutation from FLT3 wt homology model). (C) Docking of <b>12</b> into the DFG-out inactive conformation of FLT3 wt kinase (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4XUF">4XUF</a>). (D) Docking of <b>12</b> into the active conformation of FLT3-F691L kinase (site mutation from PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4XUF">4XUF</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-20/acs.jmedchem.7b00840/20171020/images/large/jm-2017-008406_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00840&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-20/acs.jmedchem.7b00840/20171020/images/medium/jm-2017-008406_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-20/acs.jmedchem.7b00840/20171020/images/large/jm-2017-008406_0004.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Binding mode examination of compound <b>14</b>. (A) Docking of <b>14</b> into the DFG-out conformation of FLT3 kinase (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4XUF">4XUF</a>). (B) Docking of <b>14</b> into the homology model of DFG-out conformation of FLT3-F691L mutant (site mutation from PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4XUF">4XUF</a>). (C) Docking of <b>24</b> into the homology model of DFG-out conformation of FLT3-F691L mutant (green and blue colors represent two possible directions of methyl group respectively). (D) Docking of <b>25</b> into the homology model of DFG-out conformation of FLT3-F691L mutant.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-20/acs.jmedchem.7b00840/20171020/images/large/jm-2017-008406_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00840&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-20/acs.jmedchem.7b00840/20171020/images/medium/jm-2017-008406_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-20/acs.jmedchem.7b00840/20171020/images/large/jm-2017-008406_0005.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Kinome wide selectivity profiling of compound <b>14</b> with DiscoveRx KINOMEScan technology. Measurements were performed at a concentration of 1 μM of the inhibitor. The affinity was defined with respect to a DMSO control. (A) Treespot demonstration of <b>14</b>’s selectivity in 468 kinase targets. (B) Other targets that demonstrated strong binding to <b>14</b> with a percent control number less than 1. (C) ADP-Glo assay and Invitrogen’s biochemical characterization of <b>14</b>. <sup><i>a</i></sup>The biochemical tests used ADP-Glo assay (Promega, Madison, WI). <sup><i>b</i></sup>The biochemical tests were provided by Invitrogen (Carlsbad, CA, USA). All data were obtained by triplet testing (±SD).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-20/acs.jmedchem.7b00840/20171020/images/large/jm-2017-008406_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00840&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-20/acs.jmedchem.7b00840/20171020/images/medium/jm-2017-008406_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-20/acs.jmedchem.7b00840/20171020/images/large/jm-2017-008406_0006.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Cellular effects of compound <b>14</b> in FLT3-ITD positive AML cell lines. (A) Effects of <b>14</b> on FLT3 mediated signaling pathways. (B) Effects of <b>14</b> on cell cycle progression. (C) Effects of <b>14</b> on apoptosis. All data were obtained by duplicate runs, and only one run was shown as representative.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-20/acs.jmedchem.7b00840/20171020/images/large/jm-2017-008406_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00840&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-20/acs.jmedchem.7b00840/20171020/images/medium/jm-2017-008406_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-20/acs.jmedchem.7b00840/20171020/images/large/jm-2017-008406_0007.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Compound <b>14</b>’s antitumor efficacy in MV4-11 cell (FLT3-ITD) inoculated xenograft mouse model. Female nu/nu mice bearing established MV4-11 tumor xenografts were treated with <b>14</b> at 3.75, 7.5, and 15 mg kg<sup>–1</sup> day<sup>–1</sup> dosage or <b>7</b> at 10 mg kg<sup>–1</sup> day<sup>–1</sup> or vehicle. Daily oral administration was initiated when MV4-11 tumors had reached a size of 200–400 mm<sup>3</sup>. Each group contained 5–7 animals. Data, mean ± SEM mpk, mg kg<sup>–1</sup> day<sup>–1</sup>. (A) Body weight and (B) tumor size measurements from MV4-11 xenograft mice after 28 days compound administration. Initial body weight and tumor size were set as 100%. (C) Representative photographs of tumors in each group after 0, 3.75, 7.5, and 15 mg kg<sup>–1</sup> day<sup>–1</sup><b>14</b> or 10 mg kg<sup>–1</sup> day<sup>–1</sup><b>7</b> treatments. (D) Comparison of the final tumor weights in each group after 28-day treatment period. Numbers in columns indicate the mean tumor weight in each group. (∗∗∗∗) <i>p</i> < 0.0001. (E) Representative micrographs of hematoxylin and eosin (HE), Ki-67, and TUNEL staining of tumor tissues with <b>14</b> treatment groups in comparison with the vehicle treatment group. Note the specific nuclear staining of cells with morphology consistent with proliferation and apoptosis (E, blue arrows). (F) Dose-responsive immunoblotting of the signaling pathway with the cells extracted from tumor tissues of the experimental animal after the <b>14</b> and <b>7</b> treatments. (G) Tumor progression examination after stopping compound <b>14</b> treatment.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-20/acs.jmedchem.7b00840/20171020/images/large/jm-2017-008406_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00840&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-20/acs.jmedchem.7b00840/20171020/images/medium/jm-2017-008406_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-20/acs.jmedchem.7b00840/20171020/images/large/jm-2017-008406_0008.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of Compounds <b>14</b>–<b>27</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-20/acs.jmedchem.7b00840/20171020/images/large/jm-2017-008406_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00840&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) methanesulfonyl chloride, DIEPA, THF, 0 °C to rt, overnight, yield 87.3%; (b) K<sub>2</sub>CO<sub>3</sub>, DMF, 70 °C, 8 h, yield 74.3–75.6%; (c) for <b>38a</b>,<b>c</b>,<b>d</b>, K<sub>2</sub>CO<sub>3</sub>, DMF, 100 °C, overnight; for <b>38b</b>, K<sub>2</sub>CO<sub>3</sub>, DMF, 120 °C, overnight, yield 80.1–82.5%; (d) <b>37a</b>–<b>d</b>, Pd(PPh<sub>3</sub>)<sub>4</sub>, K<sub>2</sub>CO<sub>3</sub>, 1,4-dioxane/H<sub>2</sub>O, 90 °C, 12 h; (e) 10% Pd/C, H<sub>2</sub>, MeOH, rt, 4 h, two-step yield 85.4–87.8%; (f) triphosgene, Et<sub>3</sub>N, CH<sub>2</sub>Cl<sub>2</sub>, −10 to 25 °C, 16 h, yield 48.9–55.3%.</p></p></figure><figure data-id="sch2" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-20/acs.jmedchem.7b00840/20171020/images/medium/jm-2017-008406_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-20/acs.jmedchem.7b00840/20171020/images/large/jm-2017-008406_0009.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of Compounds <b>11</b>–<b>13</b>, <b>28</b>–<b>30</b>, <b>32</b>, <b>33</b> ,and <b>35</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-20/acs.jmedchem.7b00840/20171020/images/large/jm-2017-008406_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00840&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) Pd(PPh<sub>3</sub>)<sub>4</sub>, K<sub>2</sub>CO<sub>3</sub>, 1,4-dioxane/H<sub>2</sub>O, 90 °C, 12 h; (b) 10% Pd/C, H<sub>2</sub>, MeOH, rt, 2 h, two-step yield 84.6–88.2%; (c) 5-(<i>tert</i>-butyl)isoxazol-3-amine, triphosgene, Et<sub>3</sub>N, CH<sub>2</sub>Cl<sub>2</sub>, −10 to 25 °C, 16 h, yield 50.4–70.1%.</p></p></figure><figure data-id="sch3" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-20/acs.jmedchem.7b00840/20171020/images/medium/jm-2017-008406_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-20/acs.jmedchem.7b00840/20171020/images/large/jm-2017-008406_0010.jpeg" id="rightTab-gr10" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthesis of Compounds <b>31</b> and <b>34</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-20/acs.jmedchem.7b00840/20171020/images/large/jm-2017-008406_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00840&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) Pd(PPh<sub>3</sub>)<sub>4</sub>, K<sub>2</sub>CO<sub>3</sub>, 1,4-dioxane/H<sub>2</sub>O, 90 °C, 12 h; (b) 10% Pd/C, H<sub>2</sub>, MeOH, rt, 2 h, two-step yield 52.7%; (c) 5-(<i>tert</i>-butyl)isoxazol-3-amine, triphosgene, Et<sub>3</sub>N, CH<sub>2</sub>Cl<sub>2</sub>, −10 to 25 °C, 16 h; (d) 4 N HCl in EtOAc, rt, 2 h, two-step yield 52.7%; (e) 2-iodoethanol, K<sub>2</sub>CO<sub>3</sub>, DMF, rt, 3 h, yield 71.4%.</p></p></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i86">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_33300" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_33300" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 23 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group">Lee, H. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, H. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, I. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jung, D. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, S. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Park, S. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hwang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Choi, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, J. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, S. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, J. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cools, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koh, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Song, H. J.</span><span> </span><span class="NLM_article-title">G-749, a novel FLT3 kinase inhibitor, can overcome drug resistance for the treatment of acute myeloid leukemia</span> <span class="citation_source-journal">Blood</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">123</span><span class="NLM_x">, </span> <span class="NLM_fpage">2209</span><span class="NLM_x">–</span> <span class="NLM_lpage">2219</span><span class="refDoi"> DOI: 10.1182/blood-2013-04-493916</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00840&amp;key=10.1182%2Fblood-2013-04-493916" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00840&amp;key=24532805" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00840&amp;key=1%3ACAS%3A528%3ADC%252BC2cXmtVOju74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=123&publication_year=2014&pages=2209-2219&author=H.+K.+Leeauthor=H.+W.+Kimauthor=I.+Y.+Leeauthor=J.+Leeauthor=J.+Leeauthor=D.+S.+Jungauthor=S.+Y.+Leeauthor=S.+H.+Parkauthor=H.+Hwangauthor=J.+S.+Choiauthor=J.+H.+Kimauthor=S.+W.+Kimauthor=J.+K.+Kimauthor=J.+Coolsauthor=J.+S.+Kohauthor=H.+J.+Song&title=G-749%2C+a+novel+FLT3+kinase+inhibitor%2C+can+overcome+drug+resistance+for+the+treatment+of+acute+myeloid+leukemia&doi=10.1182%2Fblood-2013-04-493916"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">G-749, a novel FLT3 kinase inhibitor, can overcome drug resistance for the treatment of acute myeloid leukemia</span></div><div class="casAuthors">Lee, Hee Kyu; Kim, Hong Woo; Lee, In Yong; Lee, Jungmi; Lee, Jaekyoo; Jung, Dong Sik; Lee, Sang Yeop; Park, Sung Ho; Hwang, Haejun; Choi, Jang-Sik; Kim, Jung-Ho; Kim, Se Won; Kim, Jung Keun; Cools, Jan; Koh, Jong Sung; Song, Ho-Juhn</div><div class="citationInfo"><span class="NLM_cas:title">Blood</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">123</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">2209-2219</span>CODEN:
                <span class="NLM_cas:coden">BLOOAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-4971</span>.
    
            (<span class="NLM_cas:orgname">American Society of Hematology</span>)
        </div><div class="casAbstract">Aberrant activations of Fms-like tyrosine receptor kinase (FLT) 3 are implicated in the pathogenesis of 20% to 30% of patients with acute myeloid leukemia (AML).  G-749 is a novel FLT3 inhibitor that showed potent and sustained inhibition of the FLT3 wild type and mutants including FLT3-ITD, FLT3-D835Y, FLT3-ITD/N676D, and FLT3-ITD/F691L in cellular assays.  G-749 retained its inhibitory potency in various drug-resistance milieus such as patient plasma, FLT3 ligand surge, and stromal protection.  Furthermore, it displayed potent antileukemic activity in bone marrow blasts from AML patients regardless of FLT3 mutation status, including those with little or only minor responses to AC220 or PKC412.  Oral administration of G-749 yielded complete tumor regression and increased life span in animal models.  Thus, G-749 appears to be a promising next-generation drug candidate for the treatment of relapsed and refractory AML patients with various FLT3-ITD/FLT3-TKD mutants and further shows the ability to overcome drug resistance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqS1dwuSRK6JrVg90H21EOLACvtfcHk0lgNLhf8bwUSrA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXmtVOju74%253D&md5=07b23569d047a3a16789d673fe904c6b</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1182%2Fblood-2013-04-493916&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1182%252Fblood-2013-04-493916%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DH.%2BK.%26aulast%3DKim%26aufirst%3DH.%2BW.%26aulast%3DLee%26aufirst%3DI.%2BY.%26aulast%3DLee%26aufirst%3DJ.%26aulast%3DLee%26aufirst%3DJ.%26aulast%3DJung%26aufirst%3DD.%2BS.%26aulast%3DLee%26aufirst%3DS.%2BY.%26aulast%3DPark%26aufirst%3DS.%2BH.%26aulast%3DHwang%26aufirst%3DH.%26aulast%3DChoi%26aufirst%3DJ.%2BS.%26aulast%3DKim%26aufirst%3DJ.%2BH.%26aulast%3DKim%26aufirst%3DS.%2BW.%26aulast%3DKim%26aufirst%3DJ.%2BK.%26aulast%3DCools%26aufirst%3DJ.%26aulast%3DKoh%26aufirst%3DJ.%2BS.%26aulast%3DSong%26aufirst%3DH.%2BJ.%26atitle%3DG-749%252C%2520a%2520novel%2520FLT3%2520kinase%2520inhibitor%252C%2520can%2520overcome%2520drug%2520resistance%2520for%2520the%2520treatment%2520of%2520acute%2520myeloid%2520leukemia%26jtitle%3DBlood%26date%3D2014%26volume%3D123%26spage%3D2209%26epage%3D2219%26doi%3D10.1182%2Fblood-2013-04-493916" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group">Kiyoi, H.</span><span> </span><span class="NLM_article-title">FLT3 Inhibitors: recent advances and problems for clinical application</span> <span class="citation_source-journal">Nagoya J. Med. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">77</span><span class="NLM_x">, </span> <span class="NLM_fpage">7</span><span class="NLM_x">–</span> <span class="NLM_lpage">17</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00840&amp;key=25797966" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00840&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhslalt7rL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=77&publication_year=2015&pages=7-17&author=H.+Kiyoi&title=FLT3+Inhibitors%3A+recent+advances+and+problems+for+clinical+application"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">FLT3 inhibitors: recent advances and problems for clinical application</span></div><div class="casAuthors">Kiyoi, Hitoshi</div><div class="citationInfo"><span class="NLM_cas:title">Nagoya Journal of Medical Science</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">77</span>
        (<span class="NLM_cas:issue">1,2</span>),
    <span class="NLM_cas:pages">7-17</span>CODEN:
                <span class="NLM_cas:coden">NJMSAG</span>;
        ISSN:<span class="NLM_cas:issn">0027-7622</span>.
    
            (<span class="NLM_cas:orgname">Nagoya University School of Medicine, Graduate School of Medicine</span>)
        </div><div class="casAbstract">FLT3, a type III receptor tyrosine kinase, expresses on most acute leukemia cells as well as normal hematopoietic stem/progenitor cells.  Mutation in the FLT3 gene is the most frequent genetic alteration in acute myeloid leukemia (AML) and is well known as an important driver mutation for the development of myeloid malignancies.  FLT3 mutation is a strong poor prognostic factor for the long-term survival in AML patients, while neither high-dose chemotherapy nor allogeneic hematopoietic stem cell transplantation can overcome a poor prognosis.  Development of an FLT3 inhibitor is, therefore, much awaited.  To date, several potent FLT3 inhibitors have been developed and some of them were evaluated for efficacy in clin. trials, although no FLT3 inhibitor has been yet approved.  Moreover, several problems for clin. use, such as adverse effects, blood concn. and resistance have been apparent.  Recently developed AC220 is a highly selective and sensitive FLT3 inhibitor.  In Phase I and II trials, AC220 so far showed the best efficacy of AML cells harboring FLT3 mutation among clin. evaluated FLT3 inhibitors, while severe bone marrow suppression and QTc prolongation should be resolved for the clin. use.  In this review, I summarize the characteristics of FLT3 inhibitors in clin. development and discuss important issues to be resolved for clin. use.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoQu7A0XRVxQbVg90H21EOLACvtfcHk0lgNLhf8bwUSrA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhslalt7rL&md5=9a44962a3aa888bfdfd3fb49e388999a</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DKiyoi%26aufirst%3DH.%26atitle%3DFLT3%2520Inhibitors%253A%2520recent%2520advances%2520and%2520problems%2520for%2520clinical%2520application%26jtitle%3DNagoya%2520J.%2520Med.%2520Sci.%26date%3D2015%26volume%3D77%26spage%3D7%26epage%3D17" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group">Fathi, A. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, Y. B.</span><span> </span><span class="NLM_article-title">The role of FLT3 inhibitors in the treatment of FLT3-mutated acute myeloid leukemia</span> <span class="citation_source-journal">Eur. J. Haematol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2017</span><span class="NLM_x">, </span> <span class="NLM_volume">98</span><span class="NLM_x">, </span> <span class="NLM_fpage">330</span><span class="NLM_x">–</span> <span class="NLM_lpage">336</span><span class="refDoi"> DOI: 10.1111/ejh.12841</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00840&amp;key=10.1111%2Fejh.12841" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00840&amp;key=28000291" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00840&amp;key=1%3ACAS%3A280%3ADC%252BC1c%252FntVegug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=98&publication_year=2017&pages=330-336&author=A.+T.+Fathiauthor=Y.+B.+Chen&title=The+role+of+FLT3+inhibitors+in+the+treatment+of+FLT3-mutated+acute+myeloid+leukemia&doi=10.1111%2Fejh.12841"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">The role of FLT3 inhibitors in the treatment of FLT3-mutated acute myeloid leukemia</span></div><div class="casAuthors">Fathi Amir T; Chen Yi-Bin</div><div class="citationInfo"><span class="NLM_cas:title">European journal of haematology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">98</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">330-336</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">FLT3 mutations are present in about one-third of patients with acute myeloid leukemia (AML).  Several FLT3 inhibitors have been used in clinical trials, and these include midostaurin, sorafenib, quizartinib, crenolanib, and gilteritinib.  Monotherapy with early tyrosine kinase inhibitors (TKIs) did not have much success; however, later generation agents have shown more promising results.  Combination with conventional chemotherapy may have benefit as evidenced by recently presented results, and data from ongoing trials are eagerly awaited.  Several trials are also evaluating TKI given after HSCT, and a large international randomized trial is planned.  We may be close to an era of targeted therapy where the standard of care for this biologically defined subset will involve incorporation of a FLT3 TKI during induction chemotherapy and after HSCT.  It is important that our community continues to collaborate to conduct well-designed clinical trials to properly define the role of FLT3 TKIs in therapy for FLT3-mutant AML.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRGKE9FArD4cUEksK8CslK8fW6udTcc2eaNKkhODF7Eurntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1c%252FntVegug%253D%253D&md5=6fe9f871da62a2a075e083155be09128</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1111%2Fejh.12841&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fejh.12841%26sid%3Dliteratum%253Aachs%26aulast%3DFathi%26aufirst%3DA.%2BT.%26aulast%3DChen%26aufirst%3DY.%2BB.%26atitle%3DThe%2520role%2520of%2520FLT3%2520inhibitors%2520in%2520the%2520treatment%2520of%2520FLT3-mutated%2520acute%2520myeloid%2520leukemia%26jtitle%3DEur.%2520J.%2520Haematol.%26date%3D2017%26volume%3D98%26spage%3D330%26epage%3D336%26doi%3D10.1111%2Fejh.12841" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group">Smith, C. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stecula, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sali, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shah, N. P.</span><span> </span><span class="NLM_article-title">FLT3 D835 mutations confer differential resistance to type II FLT3 inhibitors</span> <span class="citation_source-journal">Leukemia</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">29</span><span class="NLM_x">, </span> <span class="NLM_fpage">2390</span><span class="NLM_x">–</span> <span class="NLM_lpage">2392</span><span class="refDoi"> DOI: 10.1038/leu.2015.165</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00840&amp;key=10.1038%2Fleu.2015.165" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00840&amp;key=26108694" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00840&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1Wrtb%252FJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2015&pages=2390-2392&author=C.+C.+Smithauthor=K.+Linauthor=A.+Steculaauthor=A.+Saliauthor=N.+P.+Shah&title=FLT3+D835+mutations+confer+differential+resistance+to+type+II+FLT3+inhibitors&doi=10.1038%2Fleu.2015.165"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">FLT3 D835 mutations confer differential resistance to type II FLT3 inhibitors</span></div><div class="casAuthors">Smith, C. C.; Lin, K.; Stecula, A.; Sali, A.; Shah, N. P.</div><div class="citationInfo"><span class="NLM_cas:title">Leukemia</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2390-2392</span>CODEN:
                <span class="NLM_cas:coden">LEUKED</span>;
        ISSN:<span class="NLM_cas:issn">0887-6924</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Activating mutations in FLT3 occur in ∼30% of adult acute myeloid leukemia, primarily consisting of internal tandem duplication (ITD) mutations (∼25%) and point mutations in the tyrosine kinase domain (∼5%), commonly at the activation loop residue D835.  Secondary kinase domain mutations in FLT3-ITD, particularly at the D835 residue are frequently assocd. with acquired clin. resistance to effective FLT3 tyrosine kinase inhibitors (TKIs).  Mol. docking studies have suggested that D835 mutations primarily confer resistance by stabilizing an active Asp-Phe-Gly in ('DFG-in') kinase conformation unfavorable to the binding of type II FLT3 TKIs, which target a 'DFG-out' inactive conformation.  We profiled the activity of active type II FLT3 TKIs against D835 kinase domain mutants that have been clin. detected to date.  We found that type II inhibitors (quizartinib, sorafenib, ponatinib and PLX3397) retain activity against specific D835 substitutions.  Modeling studies suggest that bulky hydrophobic substitutions (D835Y/V/I/F) at this residue are particularly resistant, whereas mutations that preserve interactions between D835 and S838 are relatively sensitive (D835E/N).  All mutants retain sensitivity to the type I inhibitor crenolanib.  These results suggest that patients with relatively sensitive D835 mutations should be included in clin. trials of type II FLT3 TKIs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqQhZ7jQYhBcbVg90H21EOLACvtfcHk0liLyRw3GKz7WA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht1Wrtb%252FJ&md5=f6de44ca15604b8429a5dab811fe8eeb</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1038%2Fleu.2015.165&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fleu.2015.165%26sid%3Dliteratum%253Aachs%26aulast%3DSmith%26aufirst%3DC.%2BC.%26aulast%3DLin%26aufirst%3DK.%26aulast%3DStecula%26aufirst%3DA.%26aulast%3DSali%26aufirst%3DA.%26aulast%3DShah%26aufirst%3DN.%2BP.%26atitle%3DFLT3%2520D835%2520mutations%2520confer%2520differential%2520resistance%2520to%2520type%2520II%2520FLT3%2520inhibitors%26jtitle%3DLeukemia%26date%3D2015%26volume%3D29%26spage%3D2390%26epage%3D2392%26doi%3D10.1038%2Fleu.2015.165" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group">Weisberg, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roesel, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Furet, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bold, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Imbach, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Florsheimer, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Caravatti, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jiang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Manley, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ray, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Griffin, J. D.</span><span> </span><span class="NLM_article-title">Antileukemic effects of novel first- and second-Generation FLT3 Inhibitors: structure-affinity comparison</span> <span class="citation_source-journal">Genes Cancer</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">1</span><span class="NLM_x">, </span> <span class="NLM_fpage">1021</span><span class="NLM_x">–</span> <span class="NLM_lpage">1032</span><span class="refDoi"> DOI: 10.1177/1947601910396505</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00840&amp;key=10.1177%2F1947601910396505" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00840&amp;key=21779428" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00840&amp;key=1%3ACAS%3A528%3ADC%252BC3MXisF2nsrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2010&pages=1021-1032&author=E.+Weisbergauthor=J.+Roeselauthor=P.+Furetauthor=G.+Boldauthor=P.+Imbachauthor=A.+Florsheimerauthor=G.+Caravattiauthor=J.+Jiangauthor=P.+Manleyauthor=A.+Rayauthor=J.+D.+Griffin&title=Antileukemic+effects+of+novel+first-+and+second-Generation+FLT3+Inhibitors%3A+structure-affinity+comparison&doi=10.1177%2F1947601910396505"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Antileukemic effects of novel first-and second-generation FLT3 inhibitors: structure-affinity comparison</span></div><div class="casAuthors">Weisberg, Ellen; Roesel, Johannes; Furet, Pascal; Bold, Guido; Imbach, Patricia; Florsheimer, Andreas; Caravatti, Georgio; Jiang, Jingrui; Manley, Paul; Ray, Arghya; Griffin, James D.</div><div class="citationInfo"><span class="NLM_cas:title">Genes & Cancer</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1021-1032</span>CODEN:
                <span class="NLM_cas:coden">GCEAAY</span>;
        ISSN:<span class="NLM_cas:issn">1947-6019</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">Constitutively activated mutant FLT3 has emerged as a promising target for therapy for the subpopulation of acute myeloid leukemia (AML) patients who harbor it.  The small mol. inhibitor, PKC412, targets mutant FLT3 and is currently in late-stage clin. trials.  However, the identification of PKC412-resistant leukemic blast cells in the bone marrow of AML patients has propelled the development of novel and structurally distinct FLT3 inhibitors that have the potential to override drug resistance and more efficiently prevent disease progression or recurrence.  Here, we present the novel first-generation "type II" FLT3 inhibitors, AFG206, AFG210, and AHL196, and the second-generation "type II" derivs. and AST487 analogs, AUZ454 and ATH686.  All agents potently and selectively target mutant FLT3 protein kinase activity and inhibit the proliferation of cells harboring FLT3 mutants via induction of apoptosis and cell cycle inhibition.  Cross-resistance between "type I" inhibitors, PKC412 and AAE871, was demonstrated.  While cross-resistance was also obsd. between "type I" and first-generation "type II" FLT3 inhibitors, the high potency of the second-generation "type II" inhibitors was sufficient to potently kill "type I" inhibitor-resistant mutant FLT3-expressing cells.  The increased potency obsd. for the second-generation "type II" inhibitors was obsd. to be due to an improved interaction with the ATP pocket of FLT3, specifically assocd. with introduction of a piperazine moiety and placement of an amino group in position 2 of the pyrimidine ring.  Thus, we present 2 structurally novel classes of FLT3 inhibitors characterized by high selectivity and potency toward mutant FLT3 as a mol. target.  In addn., presentation of the antileukemic effects of "type II" inhibitors, such as AUZ454 and ATH686, highlights a new class of highly potent FLT3 inhibitors able to override drug resistance that less potent "type I" inhibitors and "type II" first-generation FLT3 inhibitors cannot.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoaZyJlPc6jT7Vg90H21EOLACvtfcHk0liLyRw3GKz7WA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXisF2nsrY%253D&md5=bf25a61a240b5f9a73e11c389cb13870</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1177%2F1947601910396505&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1947601910396505%26sid%3Dliteratum%253Aachs%26aulast%3DWeisberg%26aufirst%3DE.%26aulast%3DRoesel%26aufirst%3DJ.%26aulast%3DFuret%26aufirst%3DP.%26aulast%3DBold%26aufirst%3DG.%26aulast%3DImbach%26aufirst%3DP.%26aulast%3DFlorsheimer%26aufirst%3DA.%26aulast%3DCaravatti%26aufirst%3DG.%26aulast%3DJiang%26aufirst%3DJ.%26aulast%3DManley%26aufirst%3DP.%26aulast%3DRay%26aufirst%3DA.%26aulast%3DGriffin%26aufirst%3DJ.%2BD.%26atitle%3DAntileukemic%2520effects%2520of%2520novel%2520first-%2520and%2520second-Generation%2520FLT3%2520Inhibitors%253A%2520structure-affinity%2520comparison%26jtitle%3DGenes%2520Cancer%26date%3D2010%26volume%3D1%26spage%3D1021%26epage%3D1032%26doi%3D10.1177%2F1947601910396505" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group">Zhao, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knapp, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gray, N. S.</span><span> </span><span class="NLM_article-title">Exploration of type II binding mode: a privileged approach for kinase inhibitor focused drug discovery?</span> <span class="citation_source-journal">ACS Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">1230</span><span class="NLM_x">–</span> <span class="NLM_lpage">1241</span><span class="refDoi"> DOI: 10.1021/cb500129t</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb500129t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00840&amp;key=1%3ACAS%3A528%3ADC%252BC2cXmtV2nt7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2014&pages=1230-1241&author=Z.+Zhaoauthor=H.+Wuauthor=L.+Wangauthor=Y.+Liuauthor=S.+Knappauthor=Q.+Liuauthor=N.+S.+Gray&title=Exploration+of+type+II+binding+mode%3A+a+privileged+approach+for+kinase+inhibitor+focused+drug+discovery%3F&doi=10.1021%2Fcb500129t"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Exploration of Type II Binding Mode: A Privileged Approach for Kinase Inhibitor Focused Drug Discovery?</span></div><div class="casAuthors">Zhao, Zheng; Wu, Hong; Wang, Li; Liu, Yi; Knapp, Stefan; Liu, Qingsong; Gray, Nathanael S.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1230-1241</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The ATP site of kinases displays remarkable conformational flexibility when accommodating chem. diverse small mol. inhibitors.  The so-called activation segment, whose conformation controls catalytic activity and access to the substrate binding pocket, can undergo a large conformational change with the active state assuming a 'DFG-in' and an inactive state assuming a 'DFG-out' conformation.  Compds. that preferentially bind to the DFG-out conformation are typically called 'type II' inhibitors in contrast to 'type I' inhibitors that bind to the DFG-in conformation.  This review surveys the large no. of type II inhibitors that have been developed and provides an anal. of their crystallog. detd. binding modes.  Using a small library of type II inhibitors, we demonstrate that more than 200 kinases can be targeted, suggesting that type II inhibitors may not be intrinsically more selective than type I inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq4jYt6Db3Ap7Vg90H21EOLACvtfcHk0liLyRw3GKz7WA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXmtV2nt7s%253D&md5=95bf185ff7f973eda5a65f4096bfaa82</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1021%2Fcb500129t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb500129t%26sid%3Dliteratum%253Aachs%26aulast%3DZhao%26aufirst%3DZ.%26aulast%3DWu%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DLiu%26aufirst%3DQ.%26aulast%3DGray%26aufirst%3DN.%2BS.%26atitle%3DExploration%2520of%2520type%2520II%2520binding%2520mode%253A%2520a%2520privileged%2520approach%2520for%2520kinase%2520inhibitor%2520focused%2520drug%2520discovery%253F%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2014%26volume%3D9%26spage%3D1230%26epage%3D1241%26doi%3D10.1021%2Fcb500129t" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group">Yee, K. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schittenhelm, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">O’Farrell, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Town, A. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McGreevey, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bainbridge, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cherrington, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heinrich, M. C.</span><span> </span><span class="NLM_article-title">Synergistic effect of SU11248 with cytarabine or daunorubicin on FLT3 ITD-positive leukemic cells</span> <span class="citation_source-journal">Blood</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">104</span><span class="NLM_x">, </span> <span class="NLM_fpage">4202</span><span class="NLM_x">–</span> <span class="NLM_lpage">4209</span><span class="refDoi"> DOI: 10.1182/blood-2003-10-3381</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00840&amp;key=10.1182%2Fblood-2003-10-3381" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=104&publication_year=2004&pages=4202-4209&author=K.+W.+Yeeauthor=M.+Schittenhelmauthor=A.+M.+O%E2%80%99Farrellauthor=A.+R.+Townauthor=L.+McGreeveyauthor=T.+Bainbridgeauthor=J.+M.+Cherringtonauthor=M.+C.+Heinrich&title=Synergistic+effect+of+SU11248+with+cytarabine+or+daunorubicin+on+FLT3+ITD-positive+leukemic+cells&doi=10.1182%2Fblood-2003-10-3381"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1182%2Fblood-2003-10-3381&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1182%252Fblood-2003-10-3381%26sid%3Dliteratum%253Aachs%26aulast%3DYee%26aufirst%3DK.%2BW.%26aulast%3DSchittenhelm%26aufirst%3DM.%26aulast%3DO%25E2%2580%2599Farrell%26aufirst%3DA.%2BM.%26aulast%3DTown%26aufirst%3DA.%2BR.%26aulast%3DMcGreevey%26aufirst%3DL.%26aulast%3DBainbridge%26aufirst%3DT.%26aulast%3DCherrington%26aufirst%3DJ.%2BM.%26aulast%3DHeinrich%26aufirst%3DM.%2BC.%26atitle%3DSynergistic%2520effect%2520of%2520SU11248%2520with%2520cytarabine%2520or%2520daunorubicin%2520on%2520FLT3%2520ITD-positive%2520leukemic%2520cells%26jtitle%3DBlood%26date%3D2004%26volume%3D104%26spage%3D4202%26epage%3D4209%26doi%3D10.1182%2Fblood-2003-10-3381" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group">Lee, L. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hernandez, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rajkhowa, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, S. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Raman, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nguyen, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Small, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Levis, M.</span><span> </span><span class="NLM_article-title">Preclinical studies of gilteritinib, a next-generation FLT3 inhibitor</span> <span class="citation_source-journal">Blood</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2017</span><span class="NLM_x">, </span> <span class="NLM_volume">129</span><span class="NLM_x">, </span> <span class="NLM_fpage">257</span><span class="NLM_x">–</span> <span class="NLM_lpage">260</span><span class="refDoi"> DOI: 10.1182/blood-2016-10-745133</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00840&amp;key=10.1182%2Fblood-2016-10-745133" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00840&amp;key=27908881" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00840&amp;key=1%3ACAS%3A528%3ADC%252BC2sXosVKitL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=129&publication_year=2017&pages=257-260&author=L.+Y.+Leeauthor=D.+Hernandezauthor=T.+Rajkhowaauthor=S.+C.+Smithauthor=J.+R.+Ramanauthor=B.+Nguyenauthor=D.+Smallauthor=M.+Levis&title=Preclinical+studies+of+gilteritinib%2C+a+next-generation+FLT3+inhibitor&doi=10.1182%2Fblood-2016-10-745133"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Preclinical studies of gilteritinib, a next-generation FLT3 inhibitor</span></div><div class="casAuthors">Lee, Lauren Y.; Hernandez, Daniela; Rajkhowa, Trivikram; Smith, Samuel C.; Raman, Jayant Ranganathan; Nguyen, Bao; Small, Donald; Levis, Mark</div><div class="citationInfo"><span class="NLM_cas:title">Blood</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">129</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">257-260</span>CODEN:
                <span class="NLM_cas:coden">BLOOAW</span>;
        ISSN:<span class="NLM_cas:issn">1528-0020</span>.
    
            (<span class="NLM_cas:orgname">American Society of Hematology</span>)
        </div><div class="casAbstract">This article describes about the activity of gilteritinib and compared it with 4 other FLT3 TKIs in clin. development: midostaurin, sorafenib, quizartinib, and crenolanib.  In this article, inhibitory activity of gilteritinib against different forms of FLT3 in leukemia cells are studied.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqKpTUQzYy5lrVg90H21EOLACvtfcHk0lgh9yr5Ogknmw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXosVKitL0%253D&md5=75429ea03b906ea92d24033613f0315b</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1182%2Fblood-2016-10-745133&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1182%252Fblood-2016-10-745133%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DL.%2BY.%26aulast%3DHernandez%26aufirst%3DD.%26aulast%3DRajkhowa%26aufirst%3DT.%26aulast%3DSmith%26aufirst%3DS.%2BC.%26aulast%3DRaman%26aufirst%3DJ.%2BR.%26aulast%3DNguyen%26aufirst%3DB.%26aulast%3DSmall%26aufirst%3DD.%26aulast%3DLevis%26aufirst%3DM.%26atitle%3DPreclinical%2520studies%2520of%2520gilteritinib%252C%2520a%2520next-generation%2520FLT3%2520inhibitor%26jtitle%3DBlood%26date%3D2017%26volume%3D129%26spage%3D257%26epage%3D260%26doi%3D10.1182%2Fblood-2016-10-745133" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group">Smith, C. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lasater, E. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, K. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCreery, M. Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stewart, W. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Damon, L. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perl, A. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jeschke, G. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sugita, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carroll, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kogan, S. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuriyan, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shah, N. P.</span><span> </span><span class="NLM_article-title">Crenolanib is a selective type I pan-FLT3 inhibitor</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">111</span><span class="NLM_x">, </span> <span class="NLM_fpage">5319</span><span class="NLM_x">–</span> <span class="NLM_lpage">5124</span><span class="refDoi"> DOI: 10.1073/pnas.1320661111</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00840&amp;key=10.1073%2Fpnas.1320661111" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00840&amp;key=24623852" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00840&amp;key=1%3ACAS%3A528%3ADC%252BC2cXkt1Snsbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=111&publication_year=2014&pages=5319-5124&author=C.+C.+Smithauthor=E.+A.+Lasaterauthor=K.+C.+Linauthor=Q.+Wangauthor=M.+Q.+McCreeryauthor=W.+K.+Stewartauthor=L.+E.+Damonauthor=A.+E.+Perlauthor=G.+R.+Jeschkeauthor=M.+Sugitaauthor=M.+Carrollauthor=S.+C.+Koganauthor=J.+Kuriyanauthor=N.+P.+Shah&title=Crenolanib+is+a+selective+type+I+pan-FLT3+inhibitor&doi=10.1073%2Fpnas.1320661111"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Crenolanib is a selective type I pan-FLT3 inhibitor</span></div><div class="casAuthors">Smith, Catherine Choy; Lasater, Elisabeth A.; Lin, Kimberly C.; Wang, Qi; McCreery, Melissa Quino; Stewart, Whitney K.; Damon, Lauren E.; Perl, Alexander E.; Jeschke, Grace R.; Sugita, Mayumi; Carroll, Martin; Kogan, Scott C.; Kuriyan, John; Shah, Neil P.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">111</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">5319-5324</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Tyrosine kinase inhibitors (TKIs) represent transformative therapies for several malignancies.  Two crit. features necessary for maximizing TKI tolerability and response duration are kinase selectivity and invulnerability to resistance-conferring kinase domain (KD) mutations in the intended target.  No prior TKI has demonstrated both of these properties.  Aiming to maximize selectivity, medicinal chemists have largely sought to create TKIs that bind to an inactive (type II) kinase conformation.  The investigational type I TKI crenolanib is a potent inhibitor of Fms tyrosine kinase-3 (FLT3) internal tandem duplication, a validated therapeutic target in human acute myeloid leukemia (AML), as well as all secondary KD mutants previously shown to confer resistance to the first highly active FLT3 TKI quizartinib.  Moreover, crenolanib is highly selective for FLT3 relative to the closely related protein tyrosine kinase KIT, demonstrating that simultaneous FLT3/KIT inhibition, a prominent feature of other clin. active FLT3 TKIs, is not required for AML cell cytotoxicity in vitro and may contribute to undesirable toxicity in patients.  A satn. mutagenesis screen of FLT3-internal tandem duplication failed to recover any resistant colonies in the presence of a crenolanib concn. well below what has been safely achieved in humans, suggesting that crenolanib has the potential to suppress KD mutation-mediated clin. resistance.  Crenolanib represents the first TKI to exhibit both kinase selectivity and invulnerability to resistance-conferring KD mutations, which is unexpected of a type I inhibitor.  Crenolanib has significant promise for achieving deep and durable responses in FLT3-mutant AML, and may have a profound impact upon future medicinal chem. efforts in oncol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrwxReAanOwB7Vg90H21EOLACvtfcHk0lgh9yr5Ogknmw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXkt1Snsbw%253D&md5=955f1f242a1690247ae239050e2a8a8f</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1320661111&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1320661111%26sid%3Dliteratum%253Aachs%26aulast%3DSmith%26aufirst%3DC.%2BC.%26aulast%3DLasater%26aufirst%3DE.%2BA.%26aulast%3DLin%26aufirst%3DK.%2BC.%26aulast%3DWang%26aufirst%3DQ.%26aulast%3DMcCreery%26aufirst%3DM.%2BQ.%26aulast%3DStewart%26aufirst%3DW.%2BK.%26aulast%3DDamon%26aufirst%3DL.%2BE.%26aulast%3DPerl%26aufirst%3DA.%2BE.%26aulast%3DJeschke%26aufirst%3DG.%2BR.%26aulast%3DSugita%26aufirst%3DM.%26aulast%3DCarroll%26aufirst%3DM.%26aulast%3DKogan%26aufirst%3DS.%2BC.%26aulast%3DKuriyan%26aufirst%3DJ.%26aulast%3DShah%26aufirst%3DN.%2BP.%26atitle%3DCrenolanib%2520is%2520a%2520selective%2520type%2520I%2520pan-FLT3%2520inhibitor%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2014%26volume%3D111%26spage%3D5319%26epage%3D5124%26doi%3D10.1073%2Fpnas.1320661111" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group">Weisberg, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boulton, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kelly, L. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Manley, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fabbro, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meyer, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gilliland, D. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Griffin, J. D.</span><span> </span><span class="NLM_article-title">Inhibition of mutant FLT3 receptors in leukemia cells by the small molecule tyrosine kinase inhibitor PKC412</span> <span class="citation_source-journal">Cancer Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">1</span><span class="NLM_x">, </span> <span class="NLM_fpage">433</span><span class="NLM_x">–</span> <span class="NLM_lpage">443</span><span class="refDoi"> DOI: 10.1016/S1535-6108(02)00069-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00840&amp;key=10.1016%2FS1535-6108%2802%2900069-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00840&amp;key=12124173" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00840&amp;key=1%3ACAS%3A528%3ADC%252BD38XltVKjsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2002&pages=433-443&author=E.+Weisbergauthor=C.+Boultonauthor=L.+M.+Kellyauthor=P.+Manleyauthor=D.+Fabbroauthor=T.+Meyerauthor=D.+G.+Gillilandauthor=J.+D.+Griffin&title=Inhibition+of+mutant+FLT3+receptors+in+leukemia+cells+by+the+small+molecule+tyrosine+kinase+inhibitor+PKC412&doi=10.1016%2FS1535-6108%2802%2900069-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of mutant FLT3 receptors in leukemia cells by the small molecule tyrosine kinase inhibitor PKC412</span></div><div class="casAuthors">Weisberg, Ellen; Boulton, Christina; Kelly, Louise M.; Manley, Paul; Fabbro, Doriano; Meyer, Thomas; Gilliland, D. Gary; Griffin, James D.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">433-443</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Constitutively activating FLT3 receptor mutations have been found in 35% of patients with acute myeloblastic leukemia (AML).  Here we report the identification of a small mol. FLT3 tyrosine kinase inhibitor PKC412, which selectively induced G1 arrest and apoptosis of Ba/F3 cell lines expressing mutant FLT3 (IC50 < 10 nM) by directly inhibiting the tyrosine kinase.  Ba/F3-FLT3 cell lines made resistant to PKC412 demonstrated overexpression of mutant FLT3, confirming that FLT3 is the target of this drug.  Finally, progressive leukemia was prevented in PKC412-treated Balb/c mice transplanted with marrow transduced with a FLT3-ITD-expressing retrovirus.  PKC412 is a potent inhibitor of mutant FLT3 and is a candidate for testing as an antileukemia agent in AML patients with mutant FLT3 receptors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqq5mVxGyiCu7Vg90H21EOLACvtfcHk0lgh9yr5Ogknmw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XltVKjsb4%253D&md5=612331d927bd2ca41319b77aaea6cfd1</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1016%2FS1535-6108%2802%2900069-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1535-6108%252802%252900069-7%26sid%3Dliteratum%253Aachs%26aulast%3DWeisberg%26aufirst%3DE.%26aulast%3DBoulton%26aufirst%3DC.%26aulast%3DKelly%26aufirst%3DL.%2BM.%26aulast%3DManley%26aufirst%3DP.%26aulast%3DFabbro%26aufirst%3DD.%26aulast%3DMeyer%26aufirst%3DT.%26aulast%3DGilliland%26aufirst%3DD.%2BG.%26aulast%3DGriffin%26aufirst%3DJ.%2BD.%26atitle%3DInhibition%2520of%2520mutant%2520FLT3%2520receptors%2520in%2520leukemia%2520cells%2520by%2520the%2520small%2520molecule%2520tyrosine%2520kinase%2520inhibitor%2520PKC412%26jtitle%3DCancer%2520Cell%26date%3D2002%26volume%3D1%26spage%3D433%26epage%3D443%26doi%3D10.1016%2FS1535-6108%2802%2900069-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group">Smith, C. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, K. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lasater, E. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Massi, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Damon, L. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pendleton, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bashir, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sebra, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perl, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kasarskis, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shellooe, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tsang, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carias, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Powell, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Burton, E. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Matusow, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spevak, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ibrahim, P. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Le, M. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hsu, H. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Habets, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">West, B. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bollag, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shah, N. P.</span><span> </span><span class="NLM_article-title">Characterizing and overriding the structural mechanism of the quizartinib-resistant FLT3 “gatekeeper” F691L mutation with PLX3397</span> <span class="citation_source-journal">Cancer Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">668</span><span class="NLM_x">–</span> <span class="NLM_lpage">679</span><span class="refDoi"> DOI: 10.1158/2159-8290.CD-15-0060</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00840&amp;key=10.1158%2F2159-8290.CD-15-0060" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00840&amp;key=25847190" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00840&amp;key=1%3ACAS%3A528%3ADC%252BC2MXps1Cgsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2015&pages=668-679&author=C.+C.+Smithauthor=C.+Zhangauthor=K.+C.+Linauthor=E.+A.+Lasaterauthor=Y.+Zhangauthor=E.+Massiauthor=L.+E.+Damonauthor=M.+Pendletonauthor=A.+Bashirauthor=R.+Sebraauthor=A.+Perlauthor=A.+Kasarskisauthor=R.+Shellooeauthor=G.+Tsangauthor=H.+Cariasauthor=B.+Powellauthor=E.+A.+Burtonauthor=B.+Matusowauthor=J.+Zhangauthor=W.+Spevakauthor=P.+N.+Ibrahimauthor=M.+H.+Leauthor=H.+H.+Hsuauthor=G.+Habetsauthor=B.+L.+Westauthor=G.+Bollagauthor=N.+P.+Shah&title=Characterizing+and+overriding+the+structural+mechanism+of+the+quizartinib-resistant+FLT3+%E2%80%9Cgatekeeper%E2%80%9D+F691L+mutation+with+PLX3397&doi=10.1158%2F2159-8290.CD-15-0060"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Characterizing and Overriding the Structural Mechanism of the Quizartinib-Resistant FLT3 "Gatekeeper" F691L Mutation with PLX3397</span></div><div class="casAuthors">Smith, Catherine C.; Zhang, Chao; Lin, Kimberly C.; Lasater, Elisabeth A.; Zhang, Ying; Massi, Evan; Damon, Lauren E.; Pendleton, Matthew; Bashir, Ali; Sebra, Robert; Perl, Alexander; Kasarskis, Andrew; Shellooe, Rafe; Tsang, Garson; Carias, Heidi; Powell, Ben; Burton, Elizabeth A.; Matusow, Bernice; Zhang, Jiazhong; Spevak, Wayne; Ibrahim, Prabha N.; Le, Mai H.; Hsu, Henry H.; Habets, Gaston; West, Brian L.; Bollag, Gideon; Shah, Neil P.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Discovery</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">668-679</span>CODEN:
                <span class="NLM_cas:coden">CDAIB2</span>;
        ISSN:<span class="NLM_cas:issn">2159-8274</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">PLX3397 inhibits the growth of quizartinib-resistant FLT3-ITD+ AML cells expressing the gatekeeper F691L mutation in vitro and in vivo, but not cells expressing kinase domain mutations in the FLT3 activation loop.  Tyrosine kinase domain mutations are a common cause of acquired clin. resistance to tyrosine kinase inhibitors (TKI) used to treat cancer, including the FLT3 inhibitor quizartinib.  Mutation of kinase "gatekeeper" residues, which control access to an allosteric pocket adjacent to the ATP-binding site, has been frequently implicated in TKI resistance.  The mol. underpinnings of gatekeeper mutation-mediated resistance are incompletely understood.  We report the first cocrystal structure of FLT3 with the TKI quizartinib, which demonstrates that quizartinib binding relies on essential edge-to-face arom. interactions with the gatekeeper F691 residue, and F830 within the highly conserved Asp-Phe-Gly motif in the activation loop.  This reliance makes quizartinib critically vulnerable to gatekeeper and activation loop substitutions while minimizing the impact of mutations elsewhere.  Moreover, we identify PLX3397, a novel FLT3 inhibitor that retains activity against the F691L mutant due to a binding mode that depends less vitally on specific interactions with the gatekeeper position.  Significance: We report the first cocrystal structure of FLT3 with a kinase inhibitor, elucidating the structural mechanism of resistance due to the gatekeeper F691L mutation.  PLX3397 is a novel FLT3 inhibitor with in vitro activity against this mutation but is vulnerable to kinase domain mutations in the FLT3 activation loop.  Cancer Discov; 5(6); 668-79. cpr2015 AACR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq9oYwTJ6cznrVg90H21EOLACvtfcHk0lhm62fWJltlHw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXps1Cgsb4%253D&md5=7e76b2892a49a6e4701f534fce212ca4</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1158%2F2159-8290.CD-15-0060&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F2159-8290.CD-15-0060%26sid%3Dliteratum%253Aachs%26aulast%3DSmith%26aufirst%3DC.%2BC.%26aulast%3DZhang%26aufirst%3DC.%26aulast%3DLin%26aufirst%3DK.%2BC.%26aulast%3DLasater%26aufirst%3DE.%2BA.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DMassi%26aufirst%3DE.%26aulast%3DDamon%26aufirst%3DL.%2BE.%26aulast%3DPendleton%26aufirst%3DM.%26aulast%3DBashir%26aufirst%3DA.%26aulast%3DSebra%26aufirst%3DR.%26aulast%3DPerl%26aufirst%3DA.%26aulast%3DKasarskis%26aufirst%3DA.%26aulast%3DShellooe%26aufirst%3DR.%26aulast%3DTsang%26aufirst%3DG.%26aulast%3DCarias%26aufirst%3DH.%26aulast%3DPowell%26aufirst%3DB.%26aulast%3DBurton%26aufirst%3DE.%2BA.%26aulast%3DMatusow%26aufirst%3DB.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DSpevak%26aufirst%3DW.%26aulast%3DIbrahim%26aufirst%3DP.%2BN.%26aulast%3DLe%26aufirst%3DM.%2BH.%26aulast%3DHsu%26aufirst%3DH.%2BH.%26aulast%3DHabets%26aufirst%3DG.%26aulast%3DWest%26aufirst%3DB.%2BL.%26aulast%3DBollag%26aufirst%3DG.%26aulast%3DShah%26aufirst%3DN.%2BP.%26atitle%3DCharacterizing%2520and%2520overriding%2520the%2520structural%2520mechanism%2520of%2520the%2520quizartinib-resistant%2520FLT3%2520%25E2%2580%259Cgatekeeper%25E2%2580%259D%2520F691L%2520mutation%2520with%2520PLX3397%26jtitle%3DCancer%2520Discovery%26date%3D2015%26volume%3D5%26spage%3D668%26epage%3D679%26doi%3D10.1158%2F2159-8290.CD-15-0060" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group">Shah, N. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Talpaz, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deininger, M. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mauro, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Flinn, I. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bixby, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lustgarten, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gozgit, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clackson, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Turner, C. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Haluska, F. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kantarjian, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cortes, J. E.</span><span> </span><span class="NLM_article-title">Ponatinib in patients with refractory acute myeloid leukaemia: findings from a phase 1 study</span> <span class="citation_source-journal">Br. J. Haematol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">162</span><span class="NLM_x">, </span> <span class="NLM_fpage">548</span><span class="NLM_x">–</span> <span class="NLM_lpage">552</span><span class="refDoi"> DOI: 10.1111/bjh.12382</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00840&amp;key=10.1111%2Fbjh.12382" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00840&amp;key=23691988" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00840&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtFOitrrE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=162&publication_year=2013&pages=548-552&author=N.+P.+Shahauthor=M.+Talpazauthor=M.+W.+Deiningerauthor=M.+J.+Mauroauthor=I.+W.+Flinnauthor=D.+Bixbyauthor=S.+Lustgartenauthor=J.+M.+Gozgitauthor=T.+Clacksonauthor=C.+D.+Turnerauthor=F.+G.+Haluskaauthor=H.+Kantarjianauthor=J.+E.+Cortes&title=Ponatinib+in+patients+with+refractory+acute+myeloid+leukaemia%3A+findings+from+a+phase+1+study&doi=10.1111%2Fbjh.12382"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Ponatinib in patients with refractory acute myeloid leukaemia: findings from a phase 1 study</span></div><div class="casAuthors">Shah, Neil P.; Talpaz, Moshe; Deininger, Michael W. N.; Mauro, Michael J.; Flinn, Ian W.; Bixby, Dale; Lustgarten, Stephanie; Gozgit, Joseph M.; Clackson, Tim; Turner, Christopher D.; Haluska, Frank G.; Kantarjian, Hagop; Cortes, Jorge E.</div><div class="citationInfo"><span class="NLM_cas:title">British Journal of Haematology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">162</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">548-552</span>CODEN:
                <span class="NLM_cas:coden">BJHEAL</span>;
        ISSN:<span class="NLM_cas:issn">0007-1048</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">Activating mutations in the FMS-like tyrosine kinase-3 (FLT3), a tyrosine kinase receptor important in hematopoiesis, are among the most common mol. aberrations in acute myeloid leukemia (AML), occurring in 30% of adult patients.  Ponatinib is a novel, orally administered tyrosine kinase inhibitor (TKI) and a potent pan-BCR-ABL1 inhibitor.  This article reports the first clin. experience with ponatinib in 12 acute myeloid leukemia (AML) patients included in the phase 1 study.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo3dQyPoxgIELVg90H21EOLACvtfcHk0lhm62fWJltlHw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtFOitrrE&md5=13f70ed786895227e7b47fb79e06662f</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1111%2Fbjh.12382&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fbjh.12382%26sid%3Dliteratum%253Aachs%26aulast%3DShah%26aufirst%3DN.%2BP.%26aulast%3DTalpaz%26aufirst%3DM.%26aulast%3DDeininger%26aufirst%3DM.%2BW.%26aulast%3DMauro%26aufirst%3DM.%2BJ.%26aulast%3DFlinn%26aufirst%3DI.%2BW.%26aulast%3DBixby%26aufirst%3DD.%26aulast%3DLustgarten%26aufirst%3DS.%26aulast%3DGozgit%26aufirst%3DJ.%2BM.%26aulast%3DClackson%26aufirst%3DT.%26aulast%3DTurner%26aufirst%3DC.%2BD.%26aulast%3DHaluska%26aufirst%3DF.%2BG.%26aulast%3DKantarjian%26aufirst%3DH.%26aulast%3DCortes%26aufirst%3DJ.%2BE.%26atitle%3DPonatinib%2520in%2520patients%2520with%2520refractory%2520acute%2520myeloid%2520leukaemia%253A%2520findings%2520from%2520a%2520phase%25201%2520study%26jtitle%3DBr.%2520J.%2520Haematol.%26date%3D2013%26volume%3D162%26spage%3D548%26epage%3D552%26doi%3D10.1111%2Fbjh.12382" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group">Zarrinkar, P. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gunawardane, R. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cramer, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gardner, M. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brigham, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Belli, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Karaman, M. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pratz, K. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pallares, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chao, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sprankle, K. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Patel, H. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Levis, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Armstrong, R. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">James, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bhagwat, S. S.</span><span> </span><span class="NLM_article-title">AC220 is a uniquely potent and selective inhibitor of FLT3 for the treatment of acute myeloid leukemia (AML)</span> <span class="citation_source-journal">Blood</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">114</span><span class="NLM_x">, </span> <span class="NLM_fpage">2984</span><span class="NLM_x">–</span> <span class="NLM_lpage">2992</span><span class="refDoi"> DOI: 10.1182/blood-2009-05-222034</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00840&amp;key=10.1182%2Fblood-2009-05-222034" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00840&amp;key=19654408" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00840&amp;key=1%3ACAS%3A528%3ADC%252BD1MXht12qurzN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=114&publication_year=2009&pages=2984-2992&author=P.+P.+Zarrinkarauthor=R.+N.+Gunawardaneauthor=M.+D.+Cramerauthor=M.+F.+Gardnerauthor=D.+Brighamauthor=B.+Belliauthor=M.+W.+Karamanauthor=K.+W.+Pratzauthor=G.+Pallaresauthor=Q.+Chaoauthor=K.+G.+Sprankleauthor=H.+K.+Patelauthor=M.+Levisauthor=R.+C.+Armstrongauthor=J.+Jamesauthor=S.+S.+Bhagwat&title=AC220+is+a+uniquely+potent+and+selective+inhibitor+of+FLT3+for+the+treatment+of+acute+myeloid+leukemia+%28AML%29&doi=10.1182%2Fblood-2009-05-222034"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">AC220 is a uniquely potent and selective inhibitor of FLT3 for the treatment of acute myeloid leukemia (AML)</span></div><div class="casAuthors">Zarrinkar, Patrick P.; Gunawardane, Ruwanthi N.; Cramer, Merryl D.; Gardner, Michael F.; Brigham, Daniel; Belli, Barbara; Karaman, Mazen W.; Pratz, Keith W.; Pallares, Gabriel; Chao, Qi; Sprankle, Kelly G.; Patel, Hitesh K.; Levis, Mark; Armstrong, Robert C.; James, Joyce; Bhagwat, Shripad S.</div><div class="citationInfo"><span class="NLM_cas:title">Blood</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">114</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">2984-2992</span>CODEN:
                <span class="NLM_cas:coden">BLOOAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-4971</span>.
    
            (<span class="NLM_cas:orgname">American Society of Hematology</span>)
        </div><div class="casAbstract">Activating mutations in the receptor tyrosine kinase FLT3 are present in up to approx. 30% of acute myeloid leukemia (AML) patients, implicating FLT3 as a driver of the disease and therefore as a target for therapy.  We report the characterization of AC220, a second-generation FLT3 inhibitor, and a comparison of AC220 with the first-generation FLT3 inhibitors CEP-701, MLN-518, PKC-412, sorafenib, and sunitinib.  AC220 exhibits low nanomolar potency in biochem. and cellular assays and exceptional kinase selectivity, and in animal models is efficacious at doses as low as 1 mg/kg given orally once daily.  The data reveal that the combination of excellent potency, selectivity, and pharmacokinetic properties is unique to AC220, which therefore is the first drug candidate with a profile that matches the characteristics desirable for a clin. FLT3 inhibitor.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr_RzDr7kzxsLVg90H21EOLACvtfcHk0ljHCczjuwNc0w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXht12qurzN&md5=4f8368a7b2eab9d1234ef36e01f5f260</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1182%2Fblood-2009-05-222034&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1182%252Fblood-2009-05-222034%26sid%3Dliteratum%253Aachs%26aulast%3DZarrinkar%26aufirst%3DP.%2BP.%26aulast%3DGunawardane%26aufirst%3DR.%2BN.%26aulast%3DCramer%26aufirst%3DM.%2BD.%26aulast%3DGardner%26aufirst%3DM.%2BF.%26aulast%3DBrigham%26aufirst%3DD.%26aulast%3DBelli%26aufirst%3DB.%26aulast%3DKaraman%26aufirst%3DM.%2BW.%26aulast%3DPratz%26aufirst%3DK.%2BW.%26aulast%3DPallares%26aufirst%3DG.%26aulast%3DChao%26aufirst%3DQ.%26aulast%3DSprankle%26aufirst%3DK.%2BG.%26aulast%3DPatel%26aufirst%3DH.%2BK.%26aulast%3DLevis%26aufirst%3DM.%26aulast%3DArmstrong%26aufirst%3DR.%2BC.%26aulast%3DJames%26aufirst%3DJ.%26aulast%3DBhagwat%26aufirst%3DS.%2BS.%26atitle%3DAC220%2520is%2520a%2520uniquely%2520potent%2520and%2520selective%2520inhibitor%2520of%2520FLT3%2520for%2520the%2520treatment%2520of%2520acute%2520myeloid%2520leukemia%2520%2528AML%2529%26jtitle%3DBlood%26date%3D2009%26volume%3D114%26spage%3D2984%26epage%3D2992%26doi%3D10.1182%2Fblood-2009-05-222034" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group">Man, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fung, T. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ho, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Han, H. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chow, H. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ma, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Choi, W. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lok, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheung, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eaves, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kwong, Y. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leung, A. Y.</span><span> </span><span class="NLM_article-title">Sorafenib treatment of FLT3-ITD(+) acute myeloid leukemia: favorable initial outcome and mechanisms of subsequent nonresponsiveness associated with the emergence of a D835 mutation</span> <span class="citation_source-journal">Blood</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">119</span><span class="NLM_x">, </span> <span class="NLM_fpage">5133</span><span class="NLM_x">–</span> <span class="NLM_lpage">5143</span><span class="refDoi"> DOI: 10.1182/blood-2011-06-363960</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00840&amp;key=10.1182%2Fblood-2011-06-363960" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00840&amp;key=22368270" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00840&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtVWjsrfE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=119&publication_year=2012&pages=5133-5143&author=C.+H.+Manauthor=T.+K.+Fungauthor=C.+Hoauthor=H.+H.+Hanauthor=H.+C.+Chowauthor=A.+C.+Maauthor=W.+W.+Choiauthor=S.+Lokauthor=A.+M.+Cheungauthor=C.+Eavesauthor=Y.+L.+Kwongauthor=A.+Y.+Leung&title=Sorafenib+treatment+of+FLT3-ITD%28%2B%29+acute+myeloid+leukemia%3A+favorable+initial+outcome+and+mechanisms+of+subsequent+nonresponsiveness+associated+with+the+emergence+of+a+D835+mutation&doi=10.1182%2Fblood-2011-06-363960"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Sorafenib treatment of FLT3-ITD+ acute myeloid leukemia: favorable initial outcome and mechanisms of subsequent nonresponsiveness associated with the emergence of a D835 mutation</span></div><div class="casAuthors">Man, Cheuk Him; Fung, Tsz Kan; Ho, Christa; Han, Heron H. C.; Chow, Howard C. H.; Ma, Alvin C. H.; Choi, William W. L.; Lok, Si; Cheung, Alice M. S.; Eaves, Connie; Kwong, Yok Lam; Leung, Anskar Y. H.</div><div class="citationInfo"><span class="NLM_cas:title">Blood</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">119</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">5133-5143</span>CODEN:
                <span class="NLM_cas:coden">BLOOAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-4971</span>.
    
            (<span class="NLM_cas:orgname">American Society of Hematology</span>)
        </div><div class="casAbstract">Internal tandem duplication (ITD) of the fms-related tyrosine kinase-3 (FLT3) gene occurs in 30% of acute myeloid leukemias (AMLs) and confers a poor prognosis.  Thirteen relapsed or chemo-refractory FLT3-ITD+ AML patients were treated with sorafenib (200-400 mg twice daily).  Twelve patients showed clearance or near clearance of bone marrow myeloblasts after 27 (range 21-84) days with evidence of differentiation of leukemia cells.  The sorafenib response was lost in most patients after 72 (range 54-287) days but the FLT3 and downstream effectors remained suppressed.  Gene expression profiling showed that leukemia cells that have become sorafenib resistant expressed several genes including ALDH1A1, JAK3, and MMP15, whose functions were unknown in AML.  Nonobese diabetic/severe combined immunodeficiency mice transplanted with leukemia cells from patients before and during sorafenib resistance recapitulated the clin. results.  Both ITD and tyrosine kinase domain mutations at D835 were identified in leukemia initiating cells (LICs) from samples before sorafenib treatment.  LICs bearing the D835 mutant have expanded during sorafenib treatment and dominated during the subsequent clin. resistance.  These results suggest that sorafenib have selected more aggressive sorafenib-resistant subclones carrying both FLT3-ITD and D835 mutations, and might provide important leads to further improvement of treatment outcome with FLT3 inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGri65wbuOPOW7Vg90H21EOLACvtfcHk0ljHCczjuwNc0w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtVWjsrfE&md5=cfcf1b9858a2df595c51f1bfe91d6960</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1182%2Fblood-2011-06-363960&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1182%252Fblood-2011-06-363960%26sid%3Dliteratum%253Aachs%26aulast%3DMan%26aufirst%3DC.%2BH.%26aulast%3DFung%26aufirst%3DT.%2BK.%26aulast%3DHo%26aufirst%3DC.%26aulast%3DHan%26aufirst%3DH.%2BH.%26aulast%3DChow%26aufirst%3DH.%2BC.%26aulast%3DMa%26aufirst%3DA.%2BC.%26aulast%3DChoi%26aufirst%3DW.%2BW.%26aulast%3DLok%26aufirst%3DS.%26aulast%3DCheung%26aufirst%3DA.%2BM.%26aulast%3DEaves%26aufirst%3DC.%26aulast%3DKwong%26aufirst%3DY.%2BL.%26aulast%3DLeung%26aufirst%3DA.%2BY.%26atitle%3DSorafenib%2520treatment%2520of%2520FLT3-ITD%2528%252B%2529%2520acute%2520myeloid%2520leukemia%253A%2520favorable%2520initial%2520outcome%2520and%2520mechanisms%2520of%2520subsequent%2520nonresponsiveness%2520associated%2520with%2520the%2520emergence%2520of%2520a%2520D835%2520mutation%26jtitle%3DBlood%26date%3D2012%26volume%3D119%26spage%3D5133%26epage%3D5143%26doi%3D10.1182%2Fblood-2011-06-363960" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group">Herrera, A. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jacobsen, E. D.</span><span> </span><span class="NLM_article-title">Ibrutinib for the treatment of mantle cell lymphoma</span> <span class="citation_source-journal">Clin. Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">20</span><span class="NLM_x">, </span> <span class="NLM_fpage">5365</span><span class="NLM_x">–</span> <span class="NLM_lpage">5371</span><span class="refDoi"> DOI: 10.1158/1078-0432.CCR-14-0010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00840&amp;key=10.1158%2F1078-0432.CCR-14-0010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00840&amp;key=25361916" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00840&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvVOgu7rN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2014&pages=5365-5371&author=A.+F.+Herreraauthor=E.+D.+Jacobsen&title=Ibrutinib+for+the+treatment+of+mantle+cell+lymphoma&doi=10.1158%2F1078-0432.CCR-14-0010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Ibrutinib for the Treatment of Mantle Cell Lymphoma</span></div><div class="casAuthors">Herrera, Alex F.; Jacobsen, Eric D.</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">5365-5371</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">A review.  Ibrutinib (PCI-32765)-a potent, covalent inhibitor of Bruton tyrosine kinase (BTK), an important kinase in the B-cell receptor signaling pathway-was recently approved by the FDA for the treatment of relapsed or refractory mantle cell lymphoma (MCL).  The drug was granted accelerated approval based on the findings of an international, multicenter, single-arm phase II study that enrolled patients with relapsed or refractory MCL.  In the study, ibrutinib (560 mg daily) was well tolerated as a single agent and resulted in an overall response rate of 68% and an estd. median response duration of 17.5 mo.  Ibrutinib's response rate and duration of response compare favorably with those for other novel agents approved for the treatment of relapsed or refractory MCL, while being less toxic than most chemotherapy or chemoimmunotherapy regimens.  Ibrutinib is currently being studied in combination with chemoimmunotherapy, monoclonal antibody therapy, and novel agents in both the initial and the relapsed/refractory treatment settings.  We review the mechanism of action, preclin. and clin. development, and the role of ibrutinib in the context of other available treatments.  Clin Cancer Res; 20(21); 5365-71. ©2014 AACR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpXYzdm1bO6frVg90H21EOLACvtfcHk0ljHCczjuwNc0w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvVOgu7rN&md5=92a7b5c4e954b89a11370eb698642ef3</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1158%2F1078-0432.CCR-14-0010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1078-0432.CCR-14-0010%26sid%3Dliteratum%253Aachs%26aulast%3DHerrera%26aufirst%3DA.%2BF.%26aulast%3DJacobsen%26aufirst%3DE.%2BD.%26atitle%3DIbrutinib%2520for%2520the%2520treatment%2520of%2520mantle%2520cell%2520lymphoma%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2014%26volume%3D20%26spage%3D5365%26epage%3D5371%26doi%3D10.1158%2F1078-0432.CCR-14-0010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group">Brown, J. R.</span><span> </span><span class="NLM_article-title">Ibrutinib (PCI-32765), the first BTK (Bruton’s tyrosine kinase) inhibitor in clinical trials</span> <span class="citation_source-journal">Curr. Hematol. Malig. Rep.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">1</span><span class="NLM_x">–</span> <span class="NLM_lpage">6</span><span class="refDoi"> DOI: 10.1007/s11899-012-0147-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00840&amp;key=10.1007%2Fs11899-012-0147-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00840&amp;key=23296407" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00840&amp;key=1%3ACAS%3A280%3ADC%252BC3s3nsVKjsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&pages=1-6&author=J.+R.+Brown&title=Ibrutinib+%28PCI-32765%29%2C+the+first+BTK+%28Bruton%E2%80%99s+tyrosine+kinase%29+inhibitor+in+clinical+trials&doi=10.1007%2Fs11899-012-0147-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Ibrutinib (PCI-32765), the first BTK (Bruton's tyrosine kinase) inhibitor in clinical trials</span></div><div class="casAuthors">Brown Jennifer R</div><div class="citationInfo"><span class="NLM_cas:title">Current hematologic malignancy reports</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1-6</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Ibrutinib is a potent covalent kinase inhibitor that targets BTK.  BTK, or Bruton's tyrosine kinase, is an obvious target for therapy of B cell diseases because inactivating mutations lead to B cell aplasia in humans and the disease X-linked agammaglobulinemia.  Ibrutinib has modest cytotoxicity against CLL cells in vitro but also blocks trophic stimuli from the microenvironment.  As with other inhibitors of the BCR pathway, ibrutinib causes rapid nodal reduction and response associated with rapid increase in lymphocytosis, which then returns to baseline over time.  The ORR of ibrutinib in relapsed refractory CLL is 67 % with PFS 88 % at 15 months.  In a cohort of untreated patients 65 years and over, the estimated 15 month PFS is 96 %.  Registration trials have been initiated, and the difficult task that remains is to determine where in the course of CLL therapy this drug will have the greatest impact and benefit for patients.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTmwv_ITNl9Dxgd2Wuil24tfW6udTcc2eZ9btMlkFGiJrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3s3nsVKjsw%253D%253D&md5=c64d7d8fa5d40a79ad029c9fb34248c3</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1007%2Fs11899-012-0147-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs11899-012-0147-9%26sid%3Dliteratum%253Aachs%26aulast%3DBrown%26aufirst%3DJ.%2BR.%26atitle%3DIbrutinib%2520%2528PCI-32765%2529%252C%2520the%2520first%2520BTK%2520%2528Bruton%25E2%2580%2599s%2520tyrosine%2520kinase%2529%2520inhibitor%2520in%2520clinical%2520trials%26jtitle%3DCurr.%2520Hematol.%2520Malig.%2520Rep.%26date%3D2013%26volume%3D8%26spage%3D1%26epage%3D6%26doi%3D10.1007%2Fs11899-012-0147-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group">Wu, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hu, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weisberg, E. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yu, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nonami, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stone, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Griffin, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, Q.</span><span> </span><span class="NLM_article-title">Ibrutinib selectively targets FLT3-ITD in mutant FLT3-positive AML</span> <span class="citation_source-journal">Leukemia</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">30</span><span class="NLM_x">, </span> <span class="NLM_fpage">754</span><span class="NLM_x">–</span> <span class="NLM_lpage">757</span><span class="refDoi"> DOI: 10.1038/leu.2015.175</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00840&amp;key=10.1038%2Fleu.2015.175" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00840&amp;key=26139428" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00840&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1emu7vO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2016&pages=754-757&author=H.+Wuauthor=C.+Huauthor=A.+Wangauthor=E.+L.+Weisbergauthor=W.+Wangauthor=C.+Chenauthor=Z.+Zhaoauthor=K.+Yuauthor=J.+Liuauthor=J.+Wuauthor=A.+Nonamiauthor=L.+Wangauthor=B.+Wangauthor=R.+M.+Stoneauthor=S.+Liuauthor=J.+D.+Griffinauthor=J.+Liuauthor=Q.+Liu&title=Ibrutinib+selectively+targets+FLT3-ITD+in+mutant+FLT3-positive+AML&doi=10.1038%2Fleu.2015.175"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Ibrutinib selectively targets FLT3-ITD in mutant FLT3-positive AML</span></div><div class="casAuthors">Wu, H.; Hu, C.; Wang, A.; Weisberg, E. L.; Wang, W.; Chen, C.; Zhao, Z.; Yu, K.; Liu, J.; Wu, J.; Nonami, A.; Wang, L.; Wang, B.; Stone, R. M.; Liu, S.; Griffin, J. D.; Liu, J.; Liu, Q.</div><div class="citationInfo"><span class="NLM_cas:title">Leukemia</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">754-757</span>CODEN:
                <span class="NLM_cas:coden">LEUKED</span>;
        ISSN:<span class="NLM_cas:issn">0887-6924</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Ibrutinib (PCI-32765) is an irreversible BTK (Bruton's tyrosine kinase) kinase inhibitor that has been extensively used as a tool compd. to validate the role of BTK kinase in B cell related malignances.1, 2.  Ibrutinib has been shown in preclin. studies to inhibit the proliferation of diffuse large B-cell lymphoma cells, mantle cell lymphoma cells, chronic lymphocytic leukemia cells and multiple myeloma cells by blocking BTK kinase activity; ibrutinib was recently approved for the clin. application on mantle cell lymphoma and chronic lymphocytic leukemia cells.3, 4, 5, 6, 7.  Ibrutinib has also exhibited anti-inflammatory effects in preclin. models.8, 9.  Recently, it has been reported that ibrutinib is also effective against epidermal growth factor receptor mutant-pos. non-small cell lung cancers through inhibition of epidermal growth factor receptor kinase activities.10.  In addn., there is evidence showing that BTK is also an important target for Acute Myeloid Leukemia (AML).11.  Despite the evidence that BTK knockdown impaired AML cancer cell growth, which suggested that BTK was important for AML cell proliferation, BTK kinase inhibition through use of a small mol. inhibitor like ibrutinib led only to moderate inhibition of proliferation of U937 cells with no apparent activity against other AML cell lines such as HL-60, TF-1 and THP-1.11.  To further investigate the potency and Interestingly, we found that only FLT3-internal tandem duplication (ITD) mutant AML cell lines (MOLM13, MOLM14 and MV4-11) were sensitive to ibrutinib (Figure 1a and Supplementary Table 1).  This is similar to what has been obsd. with the highly potent and selective FLT3 inhibitor, AC220, but differs from that obsd. for the targeted BTK kinase inhibitor, CGI-1746.12, 13 The selective antiproliferative activity was further confirmed in the clonogenic assay (GI50: 170-478 nM) (Supplementary Figure 1).  This suggests that ibrutinib might target FLT3 kinase in addn. to BTK kinase.  We then investigated the effect of ibrutinib on FLT3-ITD mediated signaling in drug-sensitive cell lines.  (Figure 1b) Ibrutinib potently inhibited FLT3-ITD auto-phosphorylation (EC50: 318, 39 and 356 nM resp. for MOLM13, MOLM14 and MV4-11) (Supplementary Figure 2A).  Phosphorylation of STAT5, a well-established downstream target, also was significantly inhibited with an EC50 between 100 and 200 nM (Supplementary Figure 2B).  In addn., c-Myc expression was inhibited with an EC50 between 185 and 315 nM.  (Supplementary Figure 2C) However, in FLT3 wt-expressing cell lines such as NOMO-1, HL-60, OCI-AML-3 and U937 cells, FLT3 signaling was not affected by ibrutinib upon FLT3 ligand stimulation (Supplementary Figure 2D).  Interestingly, despite the expression of the FLT3 wt kinase in all four cell lines, the induction of FLT3 auto-phosphorylation upon FLT3 ligand treatment was not obsd. in NOMO-1 and HL-60 cells like what was obsd. in the OCI-AML-3 and U937 cells, which might be due to the different genetic background and require further detailed mechanistic study.  The FLT3 inhibitor AC220 demonstrated similar inhibitory activity to ibrutinib in FLT3-mediated signaling pathways in the FLT3-ITD pos. cell lines.  However, the BTK inhibitors, CGI-1746 and AVL-292, did not exhibit any effect on the signaling14 (Figure 2b and Supplementary Figure 2D).  As all of these cell lines express BTK kinase, we then studied the effect of ibrutinib on BTK signaling.  Upon anti-IgM (IgM) stimulation of B-cell receptor to active the BTK kinase signaling pathway, we found that only in the FLT3 wild-type (wt)-expressing cell lines (U937 and OCI-AML3), downstream mediators of BTK kinase were affected despite BTK phosphorylation having been inhibited in both FLT3 wt and FLT3-ITD cell lines.9 This suggests that FLT3-ITD mutant cell lines might not rely on BTK signaling for growth (Supplementary Figure 3A).  Interestingly, except NOMO-1 and U937 cells, the phosphorylation of BTKY223 of MOLM14, MOLM13, MV4-11, OCI-AML-3 and HL-60 cells did not respond to anti-IgM stimulation apparently, which might be due to the basal-level expression of the IgM in these AML cells (Supplementary Figure 3B).  AC220 did not affect BTK kinase activity and its direct downstream target PLCγ but instead inhibited phosphorylation of protein kinase B and extracellular-signal-regulated kinase.  The BTK inhibitors, CGI-1746 and AVL-292, were effective against BTK kinase and PLCγ kinase phosphorylation, but had no effect on the phosphorylation of protein kinase B and extracellular-signal-regulated kinase.  These results suggest that ibrutinib may mainly exert its inhibitory activity through FLT3 kinase but not BTK kinase in the FLT3-ITD pos. cells.  To further confirm this, we knocked down BTK kinase in MOLM14 (FLT3-ITD), MOLM13 (FLT3-ITD), human erythroleukemia cell (HEL) (FLT3 wt) and NB4 (FLT3 wt) cells and found that the growth of MOLM14 and MOLM13 was only minimally affected; however NB4 cells were significantly inhibited and HEL cells were moderately inhibited (Figures 2a and b).  The fact that BTK knockdown in HEL cells did not lead to suppressed growth may be due to the fact that mutant JAK2 is the oncogenic driver in this cell line.15 In addn., the finding that ibrutinib still significantly blocked the growth of BTK-knockdown MOLM14 and MOLM13 cells further validated that the growth inhibitory effect of ibrutinib is likely dependent mainly on FLT3-ITD inhibition.  However, BTK kinase inhibition may also exert a combinatorial effect (Figure 2c).  It has recently been reported that BTK kinase is activated by FLT3-ITD in FLT3-ITD pos. cell lines and dual inhibition of BTK and FLT3 kinase demonstrates combinatorial effects; these findings further confirm this observation.16 Previous kinase selectivity profiling revealed that FLT3 wt might be an addnl. target of ibrutinib.9 The biochem. ADP-Glo assay confirmed that ibrutinib has an IC50 of 205.8 nM/65.65 nM against FLT3 wt and FLT3-ITD, resp. (Supplementary Figure 4A).  In the FLT3-ITD-dependent isogenic BaF3 cell line, ibrutinib demonstrated a GI50 of 120 nM; this inhibitory effect could be completely rescued by interleukin-3 treatment (Supplementary Figure 4B).  Interestingly, it also potently inhibited the FLT3-D835H-BaF3 isogenic cell (GI50: 0.063 μM) and moderately inhibited FLT3-D835Y-BaF3 as well as TEL-FLT3-wt-BaF3 isogenic cells (GI50: 0.5 μ/0.46 μ), however was less active against ITD-D835Y, ITD-F691L and FLT3-K663Q mutations.  (Supplementary Figure 5A) In addn., ibrutinib did not affect the growth of other oncogene-expressing Ba/F3 cells, such as c-Kit, PDGFRα, BCR-ABL and NRAS (Supplementary Figure 5B).  Although there are cysteine residues in the ATP-binding pocket, binding mode examn. with mol. modeling demonstrates that unlike the mode of inhibition against BTK kinase, ibrutinib inhibits FLT3 kinase through reversible binding and this is further confirmed by the washing-out expt. (Supplementary Figure 4C and D).  Ibrutinib induced apoptotic cell death in a concn.- and time-dependent manner in FLT3-ITD cell lines but not FLT3 wt-expressing cell lines (Supplementary Figure 6).  Cell cycle anal. revealed that ibrutinib, but not the highly selective BTK kinase inhibitor, CGI-1746, could induce a concn.-dependent manner-arrest cells in G0/G1 phase (Supplementary Figure 7).  In addn., ibrutinib exhibited dose-dependent anti-proliferation activity against FLT3-ITD mutant-expressing primary patient cells but not FLT3 wt-expressing and multiple mutants such as FLT3-ITD/FLT3-D835Y and FLT3-D835Y/E mutants expressing samples, which further confirms that ibrutinib is FLT3-ITD selective (Figure 2d).  AML still remains a serious unmet medical need and the FLT3-ITD mutant has been found in ∼30% of AML patients.  Besides std. chemotherapy, there is no targeted therapy approved in the clinic to date despite several inhibitors that are presently under clin. investigation.17 Currently, ibrutinib is under extensive clin. investigation against a variety of different B-cell malignancies.  Given the fact that the safety profile of ibrutinib is tolerated in the patients, our results might help to expand the application of this drug to AML patients harboring the FLT3-ITD mutant.  Interestingly, despite the fact that the effective concn. of ibrutinib in the in vitro studies in the FLT3-ITD pos. cell lines falls within the range of the peak plasma concn. used in human clin. trials (around 150 ng/mL at 560 mg/day dosage), the concn. required for the induction of apoptosis and cell cycle arrest for ibrutinib is higher, which suggests that an alternative formulation might be needed such as nano technol., to allow the increase of the plasma concns. in the human patients while simultaneously retaining the safety profile for the potential clin. application.18.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq9uujF_Uu1FbVg90H21EOLACvtfcHk0ljVo15eL6Eviw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht1emu7vO&md5=cc08960c1d1db9a9f83c9c67bac15f85</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1038%2Fleu.2015.175&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fleu.2015.175%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DH.%26aulast%3DHu%26aufirst%3DC.%26aulast%3DWang%26aufirst%3DA.%26aulast%3DWeisberg%26aufirst%3DE.%2BL.%26aulast%3DWang%26aufirst%3DW.%26aulast%3DChen%26aufirst%3DC.%26aulast%3DZhao%26aufirst%3DZ.%26aulast%3DYu%26aufirst%3DK.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DWu%26aufirst%3DJ.%26aulast%3DNonami%26aufirst%3DA.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DB.%26aulast%3DStone%26aufirst%3DR.%2BM.%26aulast%3DLiu%26aufirst%3DS.%26aulast%3DGriffin%26aufirst%3DJ.%2BD.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DLiu%26aufirst%3DQ.%26atitle%3DIbrutinib%2520selectively%2520targets%2520FLT3-ITD%2520in%2520mutant%2520FLT3-positive%2520AML%26jtitle%3DLeukemia%26date%3D2016%26volume%3D30%26spage%3D754%26epage%3D757%26doi%3D10.1038%2Fleu.2015.175" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group">Li, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yu, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Qi, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hu, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zou, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, Q.</span><span> </span><span class="NLM_article-title">Discovery of (R)-1-(3-(4-Amino-3-(4-phenoxyphenyl)-1H-pyrazolo[3,4-d]pyrimidin-1-yl)piperidin- 1-yl)-2-(dimethylamino)ethanone (CHMFL-FLT3-122) as a potent and orally available FLT3 Kinase inhibitor for FLT3-ITD positive acute myeloid leukemia</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">58</span><span class="NLM_x">, </span> <span class="NLM_fpage">9625</span><span class="NLM_x">–</span> <span class="NLM_lpage">9638</span><span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01611</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01611" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=9625-9638&author=X.+Liauthor=A.+Wangauthor=K.+Yuauthor=Z.+Qiauthor=C.+Chenauthor=W.+Wangauthor=C.+Huauthor=H.+Wuauthor=J.+Wuauthor=Z.+Zhaoauthor=J.+Liuauthor=F.+Zouauthor=L.+Wangauthor=B.+Wangauthor=W.+Wangauthor=S.+Zhangauthor=J.+Liuauthor=Q.+Liu&title=Discovery+of+%28R%29-1-%283-%284-Amino-3-%284-phenoxyphenyl%29-1H-pyrazolo%5B3%2C4-d%5Dpyrimidin-1-yl%29piperidin-+1-yl%29-2-%28dimethylamino%29ethanone+%28CHMFL-FLT3-122%29+as+a+potent+and+orally+available+FLT3+Kinase+inhibitor+for+FLT3-ITD+positive+acute+myeloid+leukemia&doi=10.1021%2Facs.jmedchem.5b01611"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01611&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01611%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DA.%26aulast%3DYu%26aufirst%3DK.%26aulast%3DQi%26aufirst%3DZ.%26aulast%3DChen%26aufirst%3DC.%26aulast%3DWang%26aufirst%3DW.%26aulast%3DHu%26aufirst%3DC.%26aulast%3DWu%26aufirst%3DH.%26aulast%3DWu%26aufirst%3DJ.%26aulast%3DZhao%26aufirst%3DZ.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DZou%26aufirst%3DF.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DB.%26aulast%3DWang%26aufirst%3DW.%26aulast%3DZhang%26aufirst%3DS.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DLiu%26aufirst%3DQ.%26atitle%3DDiscovery%2520of%2520%2528R%2529-1-%25283-%25284-Amino-3-%25284-phenoxyphenyl%2529-1H-pyrazolo%255B3%252C4-d%255Dpyrimidin-1-yl%2529piperidin-%25201-yl%2529-2-%2528dimethylamino%2529ethanone%2520%2528CHMFL-FLT3-122%2529%2520as%2520a%2520potent%2520and%2520orally%2520available%2520FLT3%2520Kinase%2520inhibitor%2520for%2520FLT3-ITD%2520positive%2520acute%2520myeloid%2520leukemia%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D9625%26epage%3D9638%26doi%3D10.1021%2Facs.jmedchem.5b01611" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group">Zorn, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fujimura, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barros, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuriyan, J.</span><span> </span><span class="NLM_article-title">Crystal structure of the FLT3 kinase domain bound to the inhibitor quizartinib (AC220)</span> <span class="citation_source-journal">PLoS One</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">e0121177</span><span class="refDoi"> DOI: 10.1371/journal.pone.0121177</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00840&amp;key=10.1371%2Fjournal.pone.0121177" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00840&amp;key=25837374" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00840&amp;key=1%3ACAS%3A528%3ADC%252BC28Xht1Gmsr%252FM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=e0121177&author=J.+A.+Zornauthor=Q.+Wangauthor=E.+Fujimuraauthor=T.+Barrosauthor=J.+Kuriyan&title=Crystal+structure+of+the+FLT3+kinase+domain+bound+to+the+inhibitor+quizartinib+%28AC220%29&doi=10.1371%2Fjournal.pone.0121177"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal structure of the FLT3 kinase domain bound to the inhibitor quizartinib (AC220)</span></div><div class="casAuthors">Zorn, Julie A.; Wang, Qi; Fujimura, Eric; Barros, Tiago; Kuriyan, John</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">e0121177/1-e0121177/15</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">More than 30% of acute myeloid leukemia (AML) patients possess activating mutations in the receptor tyrosine kinase FMS-like tyrosine kinase 3 or FLT3.  A small-mol. inhibitor of FLT3 (known as quizartinib or AC220) that is currently in clin. trials appears promising for the treatment of AML.  Here, we report the co-crystal structure of the kinase domain of FLT3 in complex with quizartinib.  FLT3 with quizartinib bound adopts an "Abl-like"inactive conformation with the activation loop stabilized in the "DFG-out"orientation and folded back onto the kinase domain.  This conformation is similar to that obsd. for the uncomplexed intracellular domain of FLT3 as well as for related receptor tyrosine kinases, except for a localized induced fit in the activation loop.  The co-crystal structure reveals the interactions between quizartinib and the active site of FLT3 that are key for achieving its high potency against both wild-type FLT3 as well as a FLT3 variant obsd. in many AML patients.  This co-complex further provides a structural rationale for quizartinib-resistance mutations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqOXAklBbZKP7Vg90H21EOLACvtfcHk0ljVo15eL6Eviw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xht1Gmsr%252FM&md5=d3c23bf8a8e242dd35e20e6e9b2de924</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0121177&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0121177%26sid%3Dliteratum%253Aachs%26aulast%3DZorn%26aufirst%3DJ.%2BA.%26aulast%3DWang%26aufirst%3DQ.%26aulast%3DFujimura%26aufirst%3DE.%26aulast%3DBarros%26aufirst%3DT.%26aulast%3DKuriyan%26aufirst%3DJ.%26atitle%3DCrystal%2520structure%2520of%2520the%2520FLT3%2520kinase%2520domain%2520bound%2520to%2520the%2520inhibitor%2520quizartinib%2520%2528AC220%2529%26jtitle%3DPLoS%2520One%26date%3D2015%26volume%3D10%26spage%3De0121177%26doi%3D10.1371%2Fjournal.pone.0121177" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group">Davis, M. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hunt, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Herrgard, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ciceri, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wodicka, L. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pallares, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hocker, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Treiber, D. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zarrinkar, P. P.</span><span> </span><span class="NLM_article-title">Comprehensive analysis of kinase inhibitor selectivity</span> <span class="citation_source-journal">Nat. Biotechnol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">29</span><span class="NLM_x">, </span> <span class="NLM_fpage">1046</span><span class="NLM_x">–</span> <span class="NLM_lpage">1051</span><span class="refDoi"> DOI: 10.1038/nbt.1990</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00840&amp;key=10.1038%2Fnbt.1990" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00840&amp;key=22037378" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00840&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtlyisrrK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2011&pages=1046-1051&author=M.+I.+Davisauthor=J.+P.+Huntauthor=S.+Herrgardauthor=P.+Ciceriauthor=L.+M.+Wodickaauthor=G.+Pallaresauthor=M.+Hockerauthor=D.+K.+Treiberauthor=P.+P.+Zarrinkar&title=Comprehensive+analysis+of+kinase+inhibitor+selectivity&doi=10.1038%2Fnbt.1990"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Comprehensive analysis of kinase inhibitor selectivity</span></div><div class="casAuthors">Davis, Mindy I.; Hunt, Jeremy P.; Herrgard, Sanna; Ciceri, Pietro; Wodicka, Lisa M.; Pallares, Gabriel; Hocker, Michael; Treiber, Daniel K.; Zarrinkar, Patrick P.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1046-1051</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">We tested the interaction of 72 kinase inhibitors with 442 kinases covering >80% of the human catalytic protein kinome.  Our data show that, as a class, type II inhibitors are more selective than type I inhibitors, but that there are important exceptions to this trend.  The data further illustrate that selective inhibitors have been developed against the majority of kinases targeted by the compds. tested.  Anal. of the interaction patterns reveals a class of 'group-selective' inhibitors broadly active against a single subfamily of kinases, but selective outside that subfamily.  The data set suggests compds. to use as tools to study kinases for which no dedicated inhibitors exist.  It also provides a foundation for further exploring kinase inhibitor biol. and toxicity, as well as for studying the structural basis of the obsd. interaction patterns.  Our findings will help to realize the direct enabling potential of genomics for drug development and basic research about cellular signaling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoEZnKVs6Xt5rVg90H21EOLACvtfcHk0lhV-PaMm5P3zw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtlyisrrK&md5=67639f702cb48ffb2c60e9a250f0df5e</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1038%2Fnbt.1990&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt.1990%26sid%3Dliteratum%253Aachs%26aulast%3DDavis%26aufirst%3DM.%2BI.%26aulast%3DHunt%26aufirst%3DJ.%2BP.%26aulast%3DHerrgard%26aufirst%3DS.%26aulast%3DCiceri%26aufirst%3DP.%26aulast%3DWodicka%26aufirst%3DL.%2BM.%26aulast%3DPallares%26aufirst%3DG.%26aulast%3DHocker%26aufirst%3DM.%26aulast%3DTreiber%26aufirst%3DD.%2BK.%26aulast%3DZarrinkar%26aufirst%3DP.%2BP.%26atitle%3DComprehensive%2520analysis%2520of%2520kinase%2520inhibitor%2520selectivity%26jtitle%3DNat.%2520Biotechnol.%26date%3D2011%26volume%3D29%26spage%3D1046%26epage%3D1051%26doi%3D10.1038%2Fnbt.1990" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group">Ke, Y. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Singh, V. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Coumar, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hsu, Y. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, W. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Song, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, W. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, S. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hsu, J. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shih, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hsieh, H. P.</span><span> </span><span class="NLM_article-title">Homology modeling of DFG-in FMS-like tyrosine kinase 3 (FLT3) and structure-based virtual screening for inhibitor identification</span> <span class="citation_source-journal">Sci. Rep.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">11702</span><span class="refDoi"> DOI: 10.1038/srep11702</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00840&amp;key=10.1038%2Fsrep11702" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00840&amp;key=26118648" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00840&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFyhsbnK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2015&pages=11702&author=Y.+Y.+Keauthor=V.+K.+Singhauthor=M.+S.+Coumarauthor=Y.+C.+Hsuauthor=W.+C.+Wangauthor=J.+S.+Songauthor=C.+H.+Chenauthor=W.+H.+Linauthor=S.+H.+Wuauthor=J.+T.+Hsuauthor=C.+Shihauthor=H.+P.+Hsieh&title=Homology+modeling+of+DFG-in+FMS-like+tyrosine+kinase+3+%28FLT3%29+and+structure-based+virtual+screening+for+inhibitor+identification&doi=10.1038%2Fsrep11702"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Homology modeling of DFG-in FMS-like tyrosine kinase 3 (FLT3) and structure-based virtual screening for inhibitor identification</span></div><div class="casAuthors">Ke, Yi-Yu; Singh, Vivek Kumar; Coumar, Mohane Selvaraj; Hsu, Yung Chang; Wang, Wen-Chieh; Song, Jen-Shin; Chen, Chun-Hwa; Lin, Wen-Hsing; Wu, Szu-Huei; Hsu, John T. A.; Shih, Chuan; Hsieh, Hsing-Pang</div><div class="citationInfo"><span class="NLM_cas:title">Scientific Reports</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">11702</span>CODEN:
                <span class="NLM_cas:coden">SRCEC3</span>;
        ISSN:<span class="NLM_cas:issn">2045-2322</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The inhibition of FMS-like tyrosine kinase 3 (FLT3) activity using small-mol. inhibitors has emerged as a target-based alternative to traditional chemotherapy for the treatment of acute myeloid leukemia (AML).  In this study, we report the use of structure-based virtual screening (SBVS), a computer-aided drug design technique for the identification of new chemotypes for FLT3 inhibition.  For this purpose, homol. modeling (HM) of the DFG-in FLT3 structure was carried using two template structures, including PDB ID: 1RJB (DFG-out FLT3 kinase domain) and PDB ID: 3LCD (DFG-in CSF-1 kinase domain).  The modeled structure was able to correctly identify known DFG-in (SU11248, CEP-701, and PKC-412) and DFG-out (sorafenib, ABT-869 and AC220) FLT3 inhibitors, in docking studies.  The modeled structure was then used to carry out SBVS of an HTS library of 125,000 compds.  The top scoring 97 compds. were tested for FLT3 kinase inhibition, and two hits (BPR056, IC50 = 2.3 and BPR080, IC50 = 10.7 μM) were identified.  Mol. dynamics simulation and d. functional theory calcn. suggest that BPR056 (MW: 325.32; cLogP: 2.48) interacted with FLT3 in a stable manner and could be chem. optimized to realize a drug-like lead in the future.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqqLlUVxzNzs7Vg90H21EOLACvtfcHk0lhV-PaMm5P3zw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtFyhsbnK&md5=9b5e05518ac93463fc575134bae6296c</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1038%2Fsrep11702&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsrep11702%26sid%3Dliteratum%253Aachs%26aulast%3DKe%26aufirst%3DY.%2BY.%26aulast%3DSingh%26aufirst%3DV.%2BK.%26aulast%3DCoumar%26aufirst%3DM.%2BS.%26aulast%3DHsu%26aufirst%3DY.%2BC.%26aulast%3DWang%26aufirst%3DW.%2BC.%26aulast%3DSong%26aufirst%3DJ.%2BS.%26aulast%3DChen%26aufirst%3DC.%2BH.%26aulast%3DLin%26aufirst%3DW.%2BH.%26aulast%3DWu%26aufirst%3DS.%2BH.%26aulast%3DHsu%26aufirst%3DJ.%2BT.%26aulast%3DShih%26aufirst%3DC.%26aulast%3DHsieh%26aufirst%3DH.%2BP.%26atitle%3DHomology%2520modeling%2520of%2520DFG-in%2520FMS-like%2520tyrosine%2520kinase%25203%2520%2528FLT3%2529%2520and%2520structure-based%2520virtual%2520screening%2520for%2520inhibitor%2520identification%26jtitle%3DSci.%2520Rep.%26date%3D2015%26volume%3D5%26spage%3D11702%26doi%3D10.1038%2Fsrep11702" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group">Sali, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blundell, T. L.</span><span> </span><span class="NLM_article-title">Comparative protein modelling by satisfaction of spatial restraints</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1993</span><span class="NLM_x">, </span> <span class="NLM_volume">234</span><span class="NLM_x">, </span> <span class="NLM_fpage">779</span><span class="NLM_x">–</span> <span class="NLM_lpage">815</span><span class="refDoi"> DOI: 10.1006/jmbi.1993.1626</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00840&amp;key=10.1006%2Fjmbi.1993.1626" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00840&amp;key=8254673" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00840&amp;key=1%3ACAS%3A528%3ADyaK2cXnt1ylug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=234&publication_year=1993&pages=779-815&author=A.+Saliauthor=T.+L.+Blundell&title=Comparative+protein+modelling+by+satisfaction+of+spatial+restraints&doi=10.1006%2Fjmbi.1993.1626"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Comparative protein modeling by satisfaction of spatial restraints</span></div><div class="casAuthors">Sali, Andrej; Blundell, Tom L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">234</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">779-815</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    </div><div class="casAbstract">The authors describe a comparative protein modeling method designed to find the most probable structure for a sequence given its alignment with related structures.  The three-dimensional (3D) model is obtained by optimally satisfying spatial restraints derived from the alignment and expressed as probability d. functions (pdfs) for the features restrained.  For example, the probabilities for main-chain conformations of a modelled residue may be restrained by its residue type, main-chain conformation of an equiv. residue in a related protein, and the local similarity between the two sequences.  Several such pdfs are obtained from the correlations between structural features in 17 families of homologous proteins which have been aligned on the basis of their 3D structures.  The pdfs restrain Cα-Cα distances, main-chain N-O distances, main-chain and side-chain dihedral angles.  A smoothing procedure is used in the derivation of these relationships to minimize the problem of a sparse database.  The 3D model of a protein is obtained by optimization of the mol. pdf such that the model violates the input restraints as little as possible.  The mol. pdf is derived as a combination of pdfs restraining individual spatial features of the whole mol.  The optimization procedure is a variable target function method that applies the conjugate gradients algorithm to positions of all non-hydrogen atoms.  The method is automated and is illustrated by the modeling of trypsin from two other serine proteinases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpC4wGhwZvib7Vg90H21EOLACvtfcHk0lhV-PaMm5P3zw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2cXnt1ylug%253D%253D&md5=d4a3c39b2205e36221dc187a3d1a478b</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1006%2Fjmbi.1993.1626&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Fjmbi.1993.1626%26sid%3Dliteratum%253Aachs%26aulast%3DSali%26aufirst%3DA.%26aulast%3DBlundell%26aufirst%3DT.%2BL.%26atitle%3DComparative%2520protein%2520modelling%2520by%2520satisfaction%2520of%2520spatial%2520restraints%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D1993%26volume%3D234%26spage%3D779%26epage%3D815%26doi%3D10.1006%2Fjmbi.1993.1626" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group">Pollard, J. W.</span><span> </span><span class="NLM_article-title">Trophic macrophages in development and disease</span> <span class="citation_source-journal">Nat. Rev. Immunol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">259</span><span class="NLM_x">–</span> <span class="NLM_lpage">270</span><span class="refDoi"> DOI: 10.1038/nri2528</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00840&amp;key=10.1038%2Fnri2528" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00840&amp;key=19282852" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00840&amp;key=1%3ACAS%3A528%3ADC%252BD1MXjtFCisbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2009&pages=259-270&author=J.+W.+Pollard&title=Trophic+macrophages+in+development+and+disease&doi=10.1038%2Fnri2528"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Trophic macrophages in development and disease</span></div><div class="casAuthors">Pollard, Jeffrey W.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Immunology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">259-270</span>CODEN:
                <span class="NLM_cas:coden">NRIABX</span>;
        ISSN:<span class="NLM_cas:issn">1474-1733</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Specialized phagocytes are found in the most primitive multicellular organisms.  Their roles in homeostasis and in distinguishing self from non-self have evolved with the complexity of organisms and their immune systems.  Equally important, but often overlooked, are the roles of macrophages in tissue development.  As discussed in this Review, these include functions in branching morphogenesis, neuronal patterning, angiogenesis, bone morphogenesis and the generation of adipose tissue.  In each case, macrophage depletion impairs the formation of the tissue and compromises its function.  I argue that in several diseases, the unrestrained acquisition of these developmental macrophage functions exacerbates pathol.  For example, macrophages enhance tumor progression and metastasis by affecting tumor-cell migration and invasion, as well as angiogenesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrR03M07hzbU7Vg90H21EOLACvtfcHk0liztB89MxxLxQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXjtFCisbc%253D&md5=d29685d4cbb8d8bdd296f8b8ff869d36</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1038%2Fnri2528&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnri2528%26sid%3Dliteratum%253Aachs%26aulast%3DPollard%26aufirst%3DJ.%2BW.%26atitle%3DTrophic%2520macrophages%2520in%2520development%2520and%2520disease%26jtitle%3DNat.%2520Rev.%2520Immunol.%26date%3D2009%26volume%3D9%26spage%3D259%26epage%3D270%26doi%3D10.1038%2Fnri2528" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4RT7" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4RT7','PDB','4RT7'); return false;">PDB: 4RT7</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3LCD" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3LCD','PDB','3LCD'); return false;">PDB: 3LCD</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4XUF" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4XUF','PDB','4XUF'); return false;">PDB: 4XUF</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i81"><a href="/doi/suppl/10.1021/acs.jmedchem.7b00840">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_40715"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a class="ext-link" href="/">ACS Publications website</a> at DOI: <a class="ext-link" href="/doi/abs/10.1021/acs.jmedchem.7b00840">10.1021/acs.jmedchem.7b00840</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Table 1 listing the DiscoverX’s KINOMEScan selectivity profiling data of compound <b>14</b> and Figure 1 showing the cellular effects of compound <b>7</b> in FLT3-ITD positive AML cell line (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.7b00840/suppl_file/jm7b00840_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Molecular formula strings and some data (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.7b00840/suppl_file/jm7b00840_si_002.csv">CSV</a>)</p></li><li><p class="inline">Coordinates information for structure representation (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.7b00840/suppl_file/jm7b00840_si_003.pdb">PDB</a>)</p></li><li><p class="inline">Coordinates information for structure representation (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.7b00840/suppl_file/jm7b00840_si_004.pdb">PDB</a>)</p></li><li><p class="inline">Coordinates information for structure representation (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.7b00840/suppl_file/jm7b00840_si_005.pdb">PDB</a>)</p></li><li><p class="inline">Coordinates information for structure representation (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.7b00840/suppl_file/jm7b00840_si_006.pdb">PDB</a>)</p></li><li><p class="inline">Coordinates information for structure representation (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.7b00840/suppl_file/jm7b00840_si_007.pdb">PDB</a>)</p></li><li><p class="inline">Coordinates information for structure representation (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.7b00840/suppl_file/jm7b00840_si_008.pdb">PDB</a>)</p></li><li><p class="inline">Coordinates information for structure representation (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.7b00840/suppl_file/jm7b00840_si_009.pdb">PDB</a>)</p></li><li><p class="inline">Coordinates information for structure representation (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.7b00840/suppl_file/jm7b00840_si_010.pdb">PDB</a>)</p></li><li><p class="inline">Coordinates information for structure representation (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.7b00840/suppl_file/jm7b00840_si_011.pdb">PDB</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.7b00840/suppl_file/jm7b00840_si_001.pdf">jm7b00840_si_001.pdf (717.68 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.7b00840/suppl_file/jm7b00840_si_002.csv">jm7b00840_si_002.csv (5.66 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.7b00840/suppl_file/jm7b00840_si_003.pdb">jm7b00840_si_003.pdb (365.66 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.7b00840/suppl_file/jm7b00840_si_004.pdb">jm7b00840_si_004.pdb (349.36 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.7b00840/suppl_file/jm7b00840_si_005.pdb">jm7b00840_si_005.pdb (350.26 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.7b00840/suppl_file/jm7b00840_si_006.pdb">jm7b00840_si_006.pdb (365.59 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.7b00840/suppl_file/jm7b00840_si_007.pdb">jm7b00840_si_007.pdb (349.28 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.7b00840/suppl_file/jm7b00840_si_008.pdb">jm7b00840_si_008.pdb (350.19 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.7b00840/suppl_file/jm7b00840_si_009.pdb">jm7b00840_si_009.pdb (504.22 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.7b00840/suppl_file/jm7b00840_si_010.pdb">jm7b00840_si_010.pdb (504.22 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.7b00840/suppl_file/jm7b00840_si_011.pdb">jm7b00840_si_011.pdb (504.22 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.7b00840&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2017.60.issue-20%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.7b00840%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.7b00840" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                10MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"677b45511e6b3d76","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
