
<html lang="en"     class="pb-page"  data-request-id="baced6f6-f5e4-4657-bc2d-3422ae61dafd"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";issue:issue:10.1021/jmcmar.2018.61.issue-21;requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;article:article:10.1021/acs.jmedchem.8b00598;ctype:string:Journal Content;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Protein Arginine Methyltransferase 5 (PRMT5) as an Anticancer Target and Its Inhibitor Discovery" /></meta><meta name="dc.Creator" content="Yuanxiang  Wang" /></meta><meta name="dc.Creator" content="Wenhao  Hu" /></meta><meta name="dc.Creator" content="Yanqiu  Yuan" /></meta><meta name="dc.Description" content="PRMT5 is a major enzyme responsible for symmetric dimethylation of arginine residues on both histone and non-histone proteins, regulating many biological pathways in mammalian cells. PRMT5 has been..." /></meta><meta name="Description" content="PRMT5 is a major enzyme responsible for symmetric dimethylation of arginine residues on both histone and non-histone proteins, regulating many biological pathways in mammalian cells. PRMT5 has been..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="June 5, 2018" /></meta><meta name="dc.Type" content="review-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.8b00598" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Copyright © 2018 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.8b00598" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.8b00598" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.8b00598" /></link>
        
    
    

<title>Protein Arginine Methyltransferase 5 (PRMT5) as an Anticancer Target and Its Inhibitor Discovery | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.8b00598" /></meta><meta property="og:title" content="Protein Arginine Methyltransferase 5 (PRMT5) as an Anticancer Target and Its Inhibitor Discovery" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-21/acs.jmedchem.8b00598/20181101/images/large/jm-2018-00598z_0014.jpeg" /></meta><meta property="og:description" content="PRMT5 is a major enzyme responsible for symmetric dimethylation of arginine residues on both histone and non-histone proteins, regulating many biological pathways in mammalian cells. PRMT5 has been suggested as a therapeutic target in a variety of diseases including infectious disease, heart disease, and cancer. Many PRMT5 inhibitors have been discovered in the past 5 years, and one entered clinical trial in 2015 for the treatment of solid tumor and mantle cell lymphoma (MCL). The aim of this review is to summarize the current understanding of the roles of PRMT5 in cancer and the discovery of PRMT5 enzymatic inhibitors. By reviewing the structure–activity relationship (SAR) of known inhibitors of PRMT5, we hope to provide guidance for future drug designs and inhibitor optimization. Opportunities and limitations of PRMT5 inhibitors for the treatment of cancer are also discussed." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.8b00598"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.8b00598">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-14/achre4.2021.54.issue-14/20210720/achre4.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-29/aamick.2021.13.issue-29/20210728/aamick.2021.13.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-14/acncdm.2021.12.issue-14/20210721/acncdm.2021.12.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-7/amlcef.2021.3.issue-7/20210705/amlcef.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-29/acsodf.2021.6.issue-29/20210727/acsodf.2021.6.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-29/ascecg.2021.9.issue-29/20210726/ascecg.2021.9.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-29/ancham.2021.93.issue-29/20210727/ancham.2021.93.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-29/bichaw.2021.60.issue-29/20210727/bichaw.2021.60.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-27/cgeabj.2021.99.issue-27/20210726-01/cgeabj.2021.99.issue-27.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-7/cgdefu.2021.21.issue-7/20210707/cgdefu.2021.21.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-14/esthag.2021.55.issue-14/20210720/esthag.2021.55.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-29/iecred.2021.60.issue-29/20210728/iecred.2021.60.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-14/inocaj.2021.60.issue-14/20210719/inocaj.2021.60.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-29/jafcau.2021.69.issue-29/20210728/jafcau.2021.69.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-29/jacsat.2021.143.issue-29/20210728/jacsat.2021.143.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-7/jamsef.2021.32.issue-7/20210707/jamsef.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-29/langd5.2021.37.issue-29/20210727/langd5.2021.37.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-7/mpohbp.2021.18.issue-7/20210705/mpohbp.2021.18.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.8b00598&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.8b00598&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.8b00598&amp;href=/doi/10.1021/acs.jmedchem.8b00598" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2018</span><span class="cit-fg-volume">, 61</span><span class="cit-fg-issue">, 21</span><span class="cit-fg-pageRange">, 9429-9441</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/61/21" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.8b00503" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Perspective</span><a href="/doi/10.1021/acs.jmedchem.8b00675" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Protein Arginine Methyltransferase 5 (PRMT5) as an Anticancer Target and Its Inhibitor Discovery</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Yuanxiang Wang</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Yuanxiang Wang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">School of Pharmaceutical Sciences, Sun Yat-Sen University, Guangzhou 510006, China</div></div><span class="conrtib-corresp"><strong>*</strong>Y.W.: e-mail, <a href="/cdn-cgi/l/email-protection#f087919e978988c9c5b09d91999cde83898385de959485de939e"><span class="__cf_email__" data-cfemail="cbbcaaa5acb2b3f2fe8ba6aaa2a7e5b8b2b8bee5aeafbee5a8a5">[email protected]</span></a>; fax, +86-20-39943004.</span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Yuanxiang++Wang">Yuanxiang Wang</a></span></div><div class="loa-info-bio"><span><a class="internalNav" href="#ath1">View Biography</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-5650-4396" title="Orcid link">http://orcid.org/0000-0001-5650-4396</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Wenhao Hu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Wenhao Hu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">School of Pharmaceutical Sciences, Sun Yat-Sen University, Guangzhou 510006, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Wenhao++Hu">Wenhao Hu</a></span></div><div class="loa-info-bio"><span><a class="internalNav" href="#ath2">View Biography</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-1461-3671" title="Orcid link">http://orcid.org/0000-0002-1461-3671</a></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Yanqiu Yuan</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Yanqiu Yuan</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">School of Pharmaceutical Sciences, Sun Yat-Sen University, Guangzhou 510006, China</div></div><span class="conrtib-corresp"><strong>*</strong>Y.Y.: e-mail, <a href="/cdn-cgi/l/email-protection#c3bab6a2adbab2fb83aea2aaafedb0bab0b6eda6a7b6eda0ad"><span class="__cf_email__" data-cfemail="770e0216190e064f371a161e1b59040e040259121302591419">[email protected]</span></a>; fax, +86-20-39943004.</span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Yanqiu++Yuan">Yanqiu Yuan</a></span></div><div class="loa-info-bio"><span><a class="internalNav" href="#ath3">View Biography</a></span></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.8b00598&amp;href=/doi/10.1021%2Facs.jmedchem.8b00598" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2018</span></span><span class="cit-volume">, 61</span><span class="cit-issue">, 21</span><span class="cit-pageRange">, 9429–9441</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">June 5, 2018</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>13 April 2018</li><li><span class="item_label"><b>Published</b> online</span>5 June 2018</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 8 November 2018</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.8b00598" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.8b00598</a></div><div class="article_header-article-copyright"><strong>Copyright © 2018 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D9429%26pageCount%3D13%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DYuanxiang%2BWang%252C%2BWenhao%2BHu%252C%2BYanqiu%2BYuan%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D61%26issueNum%3D21%26contentID%3Dacs.jmedchem.8b00598%26title%3DProtein%2BArginine%2BMethyltransferase%2B5%2B%2528PRMT5%2529%2Bas%2Ban%2BAnticancer%2BTarget%2Band%2BIts%2BInhibitor%2BDiscovery%26numPages%3D13%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D9441%26publicationDate%3DNovember%2B2018">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.8b00598"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">5289</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">19</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.8b00598" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Protein Arginine Methyltransferase 5 (PRMT5) as an Anticancer Target and Its Inhibitor Discovery&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Yuanxiang&quot;,&quot;last_name&quot;:&quot;Wang&quot;},{&quot;first_name&quot;:&quot;Wenhao&quot;,&quot;last_name&quot;:&quot;Hu&quot;},{&quot;first_name&quot;:&quot;Yanqiu&quot;,&quot;last_name&quot;:&quot;Yuan&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2018&quot;,&quot;month&quot;:&quot;06&quot;,&quot;day&quot;:&quot;05&quot;,&quot;issue&quot;:&quot;21&quot;,&quot;volume&quot;:&quot;61&quot;,&quot;pages&quot;:&quot;9429-9441&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.8b00598&quot;},&quot;abstract&quot;:&quot;PRMT5 is a major enzyme responsible for symmetric dimethylation of arginine residues on both histone and non-histone proteins, regulating many biological pathways in mammalian cells. PRMT5 has been suggested as a therapeutic target in a variety of diseases including infectious disease, heart disease, and cancer. Many PRMT5 inhibitors have been discovered in the past 5 years, and one entered clinical trial in 2015 for the treatment of solid tumor and mantle cell lymphoma (MCL). The aim of this review is to summarize the current understanding of the roles of PRMT5 in cancer and the discovery of PRMT5 enzymatic inhibitors. By reviewing the structure–activity relationship (SAR) of known inhibitors of PRMT5, we hope to provide guidance for future drug designs and inhibitor optimization. Opportunities and limitations of PRMT5 inhibitors for the treatment of cancer are also discussed.&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.8b00598&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.8b00598" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.8b00598&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.8b00598" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.8b00598&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.8b00598" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.8b00598&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.8b00598&amp;href=/doi/10.1021/acs.jmedchem.8b00598" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.8b00598" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.8b00598" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (6 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.8b00598&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.8b00598%26sid%3Dliteratum%253Aachs%26pmid%3D29870258%26genre%3Darticle%26aulast%3DWang%26date%3D2018%26atitle%3DProtein%2BArginine%2BMethyltransferase%2B5%2B%2528PRMT5%2529%2Bas%2Ban%2BAnticancer%2BTarget%2Band%2BIts%2BInhibitor%2BDiscovery%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D61%26issue%3D21%26spage%3D9429%26epage%3D9441%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=290896" title="Cells">Cells</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=291793" title="Inhibition">Inhibition</a>,</li><li><a href="/action/doSearch?ConceptID=292258" title="Cancer">Cancer</a>,</li><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/61/21" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-21/jmcmar.2018.61.issue-21/20181108/jmcmar.2018.61.issue-21.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-21/acs.jmedchem.8b00598/20181101/images/medium/jm-2018-00598z_0014.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-21/acs.jmedchem.8b00598/20181101/images/large/jm-2018-00598z_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00598&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">PRMT5 is a major enzyme responsible for symmetric dimethylation of arginine residues on both histone and non-histone proteins, regulating many biological pathways in mammalian cells. PRMT5 has been suggested as a therapeutic target in a variety of diseases including infectious disease, heart disease, and cancer. Many PRMT5 inhibitors have been discovered in the past 5 years, and one entered clinical trial in 2015 for the treatment of solid tumor and mantle cell lymphoma (MCL). The aim of this review is to summarize the current understanding of the roles of PRMT5 in cancer and the discovery of PRMT5 enzymatic inhibitors. By reviewing the structure–activity relationship (SAR) of known inhibitors of PRMT5, we hope to provide guidance for future drug designs and inhibitor optimization. Opportunities and limitations of PRMT5 inhibitors for the treatment of cancer are also discussed.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">1.  PRMT Family</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_77137" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_77137" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Arginine methylation is a post-translational modification that widely exists in mammalian cells.<a onclick="showRef(event, 'ref1 ref2 ref3'); return false;" href="javascript:void(0);" class="ref ref1 ref2 ref3">(1−3)</a> It is as prevalent as phosphorylation and ubiquitination and regulates a variety of biological processes, including transcription, cell signaling, mRNA translation, DNA damage, receptor trafficking, protein stability, and pre-mRNA splicing.<a onclick="showRef(event, 'ref4 ref5 ref6 ref7 ref8 ref9 ref10 ref11'); return false;" href="javascript:void(0);" class="ref ref4 ref5 ref6 ref7 ref8 ref9 ref10 ref11">(4−11)</a> In human, nine members of the PRMT family carry out this modification by catalyzing the transfer of a methyl group from <i>S</i>-adenosylmethionine (SAM) to the guanidinium nitrogen atoms of arginine residue, releasing 1 equiv of S-adenosyl-<span class="smallcaps smallerCapital">l</span>-homocysteine (SAH). On the basis of the resulting forms of methylated arginines, PRMTs are grouped into three types (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>), type I (PRMT1, PRMT2, PRMT3, PRMT4, PRMT6, and PRMT8) catalyzing the formation of ω-<i>N</i><sup>G</sup>-monomethylarginines (MMA) and ω-<i>N</i><sup>G</sup>,<i>N</i><sup>G</sup>-asymmetric dimethylarginines (aDMA), type II (PRMT5 and PRMT9) catalyzing the formation of ω-N<sup>G</sup>-MMA and ω-N<sup>G</sup>, N’<sup>G</sup>-symmetric dimethylarginines (sDMA), and type III (PRMT7) only known to catalyze the formation of ω-<i>N</i><sup>G</sup>-MMA.<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a></div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-21/acs.jmedchem.8b00598/20181101/images/medium/jm-2018-00598z_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-21/acs.jmedchem.8b00598/20181101/images/large/jm-2018-00598z_0001.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. (A) Arginine methylation reaction catalyzed by three types of PRMTs. (B) Chemical structures of SAM and SAH.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-21/acs.jmedchem.8b00598/20181101/images/large/jm-2018-00598z_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00598&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The nine members of human PRMT family share a highly homologous SAM-dependent methyltransferase (MTase) domain, which is the catalytic domain, however, they have different motif structures outside the MTase domain (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>). PRMT1 and PRMT6 contain an MTase domain only; PRMT2, PRMT3, PRMT4, PRMT5, PRMT8, and PRMT9 all have N terminal motifs preceding the catalytic domain. Both PRMT7 and PRMT9 contain a duplicated MTase domain, although the C terminal one in PRMT9 is suggested to be a pseudodomain.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> It was suggested that the additional N terminal motifs in PRMTs might be involved in substrate recognition or complex formation, contributing to their nonredundant roles in different physiological processes.<a onclick="showRef(event, 'ref13 ref14'); return false;" href="javascript:void(0);" class="ref ref13 ref14">(13,14)</a> In PRMT2, an SH3 domain precedes the MTase domain. SH3 domain is known to interact with other proteins via SH3 binding motif.<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> In PRMT5, a TIM barrel precedes the MTase domain and interacts with MEP50 to form a stable complex.<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> PRMT9 has a triple TPR motif. TPR motif is known to be involved in protein–protein interactions.<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> The nonredundant functions of PRMTs have clearly been shown for the two best-characterized family members, PRMT1 and PRMT4 (also known as CARM1). Genetic knockout of either gene results in accumulation of hypomethylated substrates in vivo, demonstrating that these two PRMTs cannot substitute for each other.<a onclick="showRef(event, 'ref18 ref19'); return false;" href="javascript:void(0);" class="ref ref18 ref19">(18,19)</a> It is possible that the various N terminal motifs in PRMTs provide main or additional recognition site for their respective substrates. However, more studies need to be done to support the claim.</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-21/acs.jmedchem.8b00598/20181101/images/medium/jm-2018-00598z_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-21/acs.jmedchem.8b00598/20181101/images/large/jm-2018-00598z_0002.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Domain structures of human PRMTs.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-21/acs.jmedchem.8b00598/20181101/images/large/jm-2018-00598z_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00598&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Despite a unique N-terminal region of variable length and distinct domain motifs, each member of the PRMT family contains a conserved SAM-dependent MTase domain, consisting of SAM-binding sequences in motif I and post-motif I, substrate-binding sequences including the double E loop (E435 and E444 for PRMT5), and the THW loop (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>A).<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> While the two glutamate residues are absolutely conserved in all PRMTs and are required for activity,<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> conservation in other regions varies among family members. In general, type I PRMTs share higher sequence homology with each other, PRMT9 is closer to Type I in primary sequence, while PRMT5 and PRMT7 are the most distant members of the family, being also quite different from each other (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>B).</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-21/acs.jmedchem.8b00598/20181101/images/medium/jm-2018-00598z_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-21/acs.jmedchem.8b00598/20181101/images/large/jm-2018-00598z_0003.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. (A) Sequence alignment of the MTase domains in human PRMTs. Conserved motifs, double E loop, and THW residues are shown in red. Residues that are separately conserved between type I PRMTs and PRMT5 are highlighted in yellow. (B) Percent identity matrix of MTase domains in human PRMTs. Numbers are colored in green-yellow-red scale with highest homology being green and lowest being red. (C) Phylogenetic tree of human PRMTs based on sequence alignment of their MTase domains.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-21/acs.jmedchem.8b00598/20181101/images/large/jm-2018-00598z_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00598&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">Careful examination around <i>C. elegans</i> PRMT5 SAH binding site reveals an interesting finding.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> There are four residues conserved among PRMT5 orthologs, which are Phe379, Lys385, Ser503, and Ser669. However, the corresponding residues in type I PRMTs are Met, Arg, Tyr, and His, respectively, and these four residues are conserved within type I PRMTs. Studies found that a F379M mutant of <i>C. elegans</i> PRMT5 could carry out both symmetric and asymmetric dimethylation of H4R3. Human PRMT5 contains the same four residues around SAM binding site (corresponding residues in human PRMT5 are marked in yellow in <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>A) as <i>C. elegans</i> PRMT5. The corresponding mutation (F327M) in human PRMT5 also resulted in the gaining of aDMA activity. The finding implies that Phe327 (human PRMT5) occupies a key position for PRMT5’s sDMA activity, which could be an important factor to consider when designing PRMT5 specific inhibitors.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> We will discuss its specific roles in PRMT5 inhibitor binding and design in later sections of this review.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">2.  PRMT5: Biochemistry and Structure</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_00713" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_00713" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">PRMT5 and PRMT9 are the only known mammalian enzymes capable of forming sDMA residues as type II PRMTs.<a onclick="showRef(event, 'ref13 ref21 ref22 ref23'); return false;" href="javascript:void(0);" class="ref ref13 ref21 ref22 ref23">(13,21−23)</a> The loss of PRMT5 activity in mouse embryo fibroblasts results in almost complete loss of sDMA, suggesting that PRMT5 is the primary sDMA-forming enzyme in these cells.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> PRMT5 is highly conserved throughout eukaryotes, with high homology in the SAM-dependent MTases domain.</div><div class="NLM_p">It was shown that human PRMT5 forms a stable complex with MEP50, a WD domain containing cytosolic protein.<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> The PRMT5:MEP50 complex was able to monomethylate and symmetric dimethylate peptide derived from histone 4 N terminal in vitro.<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> The complex consistently had a significantly higher level of MTase activity compared with PRMT5 alone, owing to higher affinities for both the peptide and SAM. A positive allosteric effect of MEP50 on the cofactor and substrate binding of PRMT5 was proposed.<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> In a biochemical assay using purified PRMT5:MEP50 complex, production of the dimethylated species could not be observed until the concentration of the monomethylated peptide exceeded that of the unmethylated substrate and the concentration of monomethylated peptide produced significantly exceeded the enzyme concentration, suggesting a nonprocessive enzymatic mechanism for peptide substrates.<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> This is different from partially processive mechanism described for PRMT1.<a onclick="showRef(event, 'ref24 ref25'); return false;" href="javascript:void(0);" class="ref ref24 ref25">(24,25)</a> The nonprocessive mechanism of PRMT5 may reflect the fact that sDMA happens on two ω-<i>N</i><sup>G</sup>-arginine, and for the correct positioning of the second guanidinium nitrogen atom, a release and rebind mechanism may be the most efficient way for catalysis.</div><div class="NLM_p">From the crystal structure of PRMT5:MEP50 complex, the overall structure of PRMT5 is composed of a TIM-barrel at the N-terminal and a SAM-dependent MTase domain at the C terminal, separated by a short dimerization domain.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> MEP50 adopts a WD40 β-propeller structure with seven blades. PRMT5 makes extensive contacts with top surface of the WD40 domain mainly through loop structures in the TIM-barrel domain (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>A).<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> The determined structure and biochemical experiments suggest that human PRMT5 binds MEP50 to form a hetero-octameric complex with molecular weight around 450 kDa.</div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-21/acs.jmedchem.8b00598/20181101/images/medium/jm-2018-00598z_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-21/acs.jmedchem.8b00598/20181101/images/large/jm-2018-00598z_0004.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. (A) Overall structure of PRMT5:MEP50 complex (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4GQB">4GQB</a>). PRMT5 catalytic domain is colored in green, N terminal TIM barrel is colored in purple, MEP50 is colored in magenta. Highlighted in orange is the linker between the N-terminal TIM barrel and C-terminal MTase catalytic domain of PRMT5. The H4 peptide is shown in stick presentation, colored in magenta, blue, and red. (B) Zoomed-in view of H4R3 substrate peptide binding groove.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-21/acs.jmedchem.8b00598/20181101/images/large/jm-2018-00598z_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00598&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">The catalytic domain of PRMT5 adopts the canonical arginine MTase tertiary structure, which is similar to that of type I PRMTs, with a SAM binding domain containing the nucleotide binding Rossmann fold, followed by a β-sandwich structure involved in substrate binding. In the above crystal structure of PRMT5:MEP50 complex, the histone H4-derived substrate peptide binds in a groove on the surface of the β-barrel domain, inserting the arginine side chain (Arg3 of the peptide) through a narrow tunnel formed by Leu312, Phe327, and Trp579 (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>B). The phenylalanine residue (Phe327) has been shown in biochemical assays to be critical in specifying sDMA activity of PRMT5 π-stacks against the guanidinium group of the arginine residue, orienting the substrate arginine to be methylated. The peptide residues that flank the arginine form a sharp β-turn at the neck of the tunnel and are stabilized by hydrogen bonding network mainly between the main chain of the peptide substrate and protein backbone. Two highly conserved glutamate residues Glu435 and Glu444 in the active site form the so-called double-E loop. Each glutamate residue forms a pair of salt bridges with the guanidinium group of the substrate arginine (H4R3), with the ω-<i>N</i><sup>G</sup> nitrogen atom poised for methyl transfer. These glutamate residues are likely involved in deprotonating and activating the ω-<i>N</i><sup>G</sup> nitrogen atom.</div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i4">3.  PRMT5 Biology and Its Relation to Cancer</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_18802" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_18802" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Being the major sDMA MTase, PRMT5 plays indispensible roles in cell development. It was found to be highly expressed in ES cells, and loss of PRMT5 function is early embryonic-lethal in mouse.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> PRMT5 was also reported to play an essential role in steady-state adult hematopoiesis, with complete loss of PRMT5 leading to BM aplasia and lethal pancytopenia.<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a></div><div class="NLM_p">PRMT5 acts its function by methylation of a myriad of nuclear and cytoplasmic substrates, including histones H3 and H4,<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> Sm proteins,<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> nucleolin,<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> RAF proteins,<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> EGFR,<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> and so on. Histone arginine symmetric dimethylation has been implicated to work as a repression marker,<a onclick="showRef(event, 'ref28 ref33'); return false;" href="javascript:void(0);" class="ref ref28 ref33">(28,33)</a> together with lysine methylation marker, to regulate gene transcription.<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> There is also evidence for PRMT5 working in concert with ATP-dependent chromatin remodelers<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> and transcriptional repressor complexes, including SIN3A/HDAC, MBD2/NURD, and N-CoR/SMRT,<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> to induce epigenetic silencing. Investigation in mouse early development indicated PRMT5 localization in both nucleus and cytoplasm, and its localization was dynamically regulated.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a></div><div class="NLM_p">A growing list of non-histone proteins have also been identified as PRMT5 substrates, whose functions vary from cell signaling to differentiation, mRNA splicing, etc.<a onclick="showRef(event, 'ref29 ref30 ref31 ref32 ref35 ref36 ref37'); return false;" href="javascript:void(0);" class="ref ref29 ref30 ref31 ref32 ref35 ref36 ref37">(29−32,35−37)</a> For example, tumor suppressor p53 is methylated by PRMT5 at three distinct arginines (Arg333, Arg335, Arg337) upon DNA damage. The methylation has an important functional consequence on p53 response.<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> Androgen receptor (AR) is methylated at Arg761 by PRMT5, which was suggested as a mechanism for how the ERG oncogene may coax AR toward inducing proliferation versus differentiation.<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> EGFR Arg1175 is methylated by PRMT5, and Arg1175 methylation positively modulates EGF-induced EGFR trans-autophosphorylation at Tyr1173, which governs ERK activation. So arginine methylation may crosstalk with phosphorylation to comodulate cell signaling.<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a></div><div class="NLM_p">PRMT5 also associates with other cellular proteins to enable methylation of preferred substrates. It is generally accepted that MEP50 is required for protein substrate recruitment to the catalytic domain of PRMT5.<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> Loss of PRMT5 triggered the loss of MEP50 protein, without changing MEP50 mRNA levels.<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> This PRMT5:MEP50 complex is likely to be the core structural unit that interacts with partner proteins to form a plethora of multisubunit complexes with discrete specificities and functions.<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> pICln was reported to recruit the spliceosomal Sm proteins to the PRMT5 complex for methylation, which allows their subsequent loading onto snRNA to form small nuclear ribonucleoproteins.<a onclick="showRef(event, 'ref29 ref40'); return false;" href="javascript:void(0);" class="ref ref29 ref40">(29,40)</a> RioK1 functions in analogy to pICln as an adapter protein by recruiting the RNA-binding protein nucleolin to the PRMT5 complex for its sDMA.<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> The identified substrates of PRMT5 are recently reviewed by A. Richters,<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> and are not the focus of this review.</div><div class="NLM_p">In the past decade, PRMT5 has attracted increasing attention as an anticancer target. It is overexpressed in various cancers, including glioblastoma, leukemia/lymphoma, prostate cancer, and colorectal carcinoma and is associated with poor prognosis.<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> In this section, we will review its suggested roles in cancer and its potential as a therapeutic target.</div><div class="NLM_p">There is substantial preclinical justification for targeting PRMT5-driven oncogenic pathways in glioblastoma.<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> First, PRMT5 is overexpressed in patient-derived primary tumors as well as cell lines, and its overexpression is correlated with cell growth rate and inversely with overall patient survival.<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> Genetic attenuation of PRMT5 led to cell-cycle arrest, apoptosis, and loss of cell migratory activity. Global gene profiling and chromatin immunoprecipitation identified the tumor suppressor ST7 as a key gene silenced by PRMT5.<a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a> PRMT5 inhibition limited PRMT5 recruitment to the ST7 promoter, led to restored expression of ST7 and cell growth inhibition, and enhanced survival in a preclinical glioblastoma xenograft model.<a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a></div><div class="NLM_p">PRMT5 was also found to be highly expressed in human lung cancer tissue samples, whereas its expression was not detectable in benign lung tissues. Silencing PRMT5 expression strongly inhibited proliferation of lung adenocarcinoma A549 cells in tissue culture and abolished growth of lung A549 xenografts in mice.<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> In vitro and in vivo studies showed that the cell growth arrest induced by loss of PRMT5 expression was partially attributable to down-regulation of fibroblast growth factor receptor signaling.<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> Treatment of lung cancer cells with PRMT5 selective inhibitors led to inhibition of the symmetrical arginine methylation of SmD3 and histones, as well as cell proliferation. Oral administration of a PRMT5 inhibitor demonstrated antitumor activity in a lung tumor xenograft model.<a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a> Details about the specific inhibitor used in the study will be discussed in a later section.</div><div class="NLM_p">During a screen for protein targets that regulate leukemia stem cell (LSC), a reciprocal positive regulation between PRMT5 and BCR-ABL was discovered and PRMT5 was found overexpressed in chronic myelogenous leukemia (CML) LSCs as well as primary CD34+ cells from CML patients.<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a> Furthermore, PRMT5 expression was increased by BCR-ABL during malignant transformation of human normal CD34+ cells and attenuation of PRMT5 function by either PRMT5 specific shRNA or PRMT5 selective inhibitor reduced survival, serially replating capacity, and long-term culture-initiating cells in human CML stem cells. Similarly, inhibition of PRMT5 prolonged the survival of CML mice and reduced the growth of CML LSCs in mice. In addition, pharmacological inhibition of PRMT5 reduced long-term engraftment of human CML CD34+ cells in NSI mice. As emergence of resistance becomes a serious threat to targeted therapies, strategies that combat resistance to existing therapies are the focus of many studies. As killing LSC could be the key to overcoming drug tolerance, the association between PRMT5 and BCR-ABL LSC shows an exciting path to novel and better cancer therapy.<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a></div><div class="NLM_p">PRMT5 was reported to promote prostate cancer cell growth by epigenetically activating transcription of the AR in prostate cancer cells.<a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a> Knockdown of PRMT5 or inhibition of PRMT5 by a specific inhibitor reduces the expression of AR and suppresses the growth of multiple AR-positive, but not AR-negative, prostate cancer cells. Significantly, knockdown of PRMT5 in AR-positive LNCaP cells completely suppresses the growth of xenograft tumors in mice. Molecular analysis reveals that PRMT5 binds to the proximal promoter region of the AR gene and contributes mainly to the enriched sDMA of H4R3 in the same region. Mechanistically, PRMT5 is recruited to the AR promoter by its interaction with Sp1, the major transcription factor responsible for AR transcription, and forms a complex with Brg1, an ATP-dependent chromatin remodeler, on the proximal promoter region of the AR gene. Interestingly, in an effort to identify genes that selectively facilitate the growth of TMPRSS2:ERG positive PC cells, a pooled shRNA screen in TMPRSS2:ERG and AR-positive VCaP prostate cancer cells together with coimmunoprecipitation with ERG by mass spectrometry identified PRMT5 as ERG-interacting protein. PRMT5 knockdown with shRNA transduced robust growth inhibition in VCaP cells but had no growth inhibitory effects in ERG-negative 22Rv1 cells and only minor effects in ERG negative LNCaP cells.<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> It is unclear what causes the discrepancy between the above two results regarding the growth effect of PRMT5 knockdown in LNCaP cell; however, these results show that targeting PRMT5 may represent a novel approach for prostate cancer treatment by eliminating AR expression.</div><div class="NLM_p last">Recently PRMT5 has gained special attention as a cancer therapeutic target. Three separate groups concluded from pooled shRNA screens<a onclick="showRef(event, 'ref48 ref49 ref50'); return false;" href="javascript:void(0);" class="ref ref48 ref49 ref50">(48−50)</a> followed by association studies that genetic deficiency in methylthioadenosin phosphorylase (MTAP, <i>P</i> = 1.17 × 10<sup>–13</sup>) and MTAP low expression (<i>P</i> = 5.03 × 10<sup>–35</sup>) are the top features predictive of cancer cell dependence on PRMT5.<a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a> Because of its proximity to the tumor suppressor gene CDKN2A on human chromosome 9p21, MTAP gene is deleted at high frequency in many human tumors, including 53% of glioblastomas, 26% of pancreatic cancers, among the most frequently mutated genes in cancer.<a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a> This new finding significantly expanded the horizon of targeting PRMT5 as anticancer therapies. With careful design, PRMT5 inhibitor could become a highly selective therapy for patients with MTAP deletion or low expression. We will further discuss the approaches and potential pitfalls in this regard in the last section of this review.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i5">4.  PRMT5 Inhibitors</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_88074" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_88074" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Recently, several groups reviewed reported inhibitors of protein methyltransferases (PMTs),<a onclick="showRef(event, 'ref51 ref52 ref53 ref54 ref55'); return false;" href="javascript:void(0);" class="ref ref51 ref52 ref53 ref54 ref55">(51−55)</a> including protein lysine methyltransferases (PKMTs) and PRMTs. In 2017, Richters<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> reviewed the current status and relevance of PRMT5 to the progression of various diseases in detail and also provided a brief introduction of several PRMT5 inhibitors. In this section, we will review in detail the discovery, optimization, characterization, and application of reported PRMT5 inhibitors, with the hope of shedding light on future work toward developing potent PRMT5 specific inhibitors.</div><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i6" class="anchor-spacer"></div><h3 class="article-section__title" id="_i6">4.1.  SAM Uncompetitive Inhibitors</h3><div class="NLM_p">Compound <b>1</b> (EPZ015666, <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>A) is the first reported cell potent and orally bioavailable PRMT5 inhibitor, with an IC<sub>50</sub> value of 22 nM in a biochemical assay.<a onclick="showRef(event, 'ref56'); return false;" href="javascript:void(0);" class="ref ref56">(56)</a> Profiling against 20 other PRMTs and PKMTs showed that it was inactive at concentrations up to 50 μM, with the activity against PRMT9 unevaluated. In a panel of five MCL cell lines (Z-138, Maver-1, Mino, Granta-519, and Jeko-1), compound <b>1</b> decreased the level of SmD3me2s in a concentration-dependent manner. It showed potent antiproliferative effects in vitro with IC<sub>50</sub> values of 96 nM and 450 nM in Z-138 and Maver-1 cells, respectively. Taking advantage of a cellular thermal shift assay (CETSA), compound <b>1</b> was shown to specifically bind to PRMT5 in the cell. The compound also displayed favorable pharmacokinetic (PK) profiles in mice, with plasma clearance of 30 mL min<sup>–1</sup> kg<sup>–1</sup>, volume of distribution of 1.7 L/kg after iv dosing at 2 mg/kg, and oral bioavailability of 69% following oral administration at 10 mg/kg. In SCID mice bearing Z-138 and Maver-1 xenografts, compound <b>1</b> induced tumor stasis at a dose of 200 mg/kg, with tumor growth inhibition (TGI) values of >93%. On the basis of the above potency and specificity data, compound <b>1</b> is the first tool compound to explore biological functions of PRMT5 and test the therapeutic hypotheses of targeting PRMT5 in cancer.<a onclick="showRef(event, 'ref56'); return false;" href="javascript:void(0);" class="ref ref56">(56)</a></div><figure id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-21/acs.jmedchem.8b00598/20181101/images/medium/jm-2018-00598z_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-21/acs.jmedchem.8b00598/20181101/images/large/jm-2018-00598z_0005.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. (A) Structures of compounds <b>1</b>–<b>3</b>. (B) Cocrystal structure of compound <b>1</b> with PRMT5:MEP50 and SAM (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4X61">4X61</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-21/acs.jmedchem.8b00598/20181101/images/large/jm-2018-00598z_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00598&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">In order to understand the mechanism of action and guide further structure optimization, a cocrystal structure of compound <b>1</b> with PRMT5:MEP50 and SAM (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>B) was obtained.<a onclick="showRef(event, 'ref56'); return false;" href="javascript:void(0);" class="ref ref56">(56)</a> It shows that the compound binds in the peptide-binding site, including the pocket occupied by the substrate arginine residue, in the presence of SAM. The finding is consistent with the enzymatic data, which show that compound <b>1</b> is competitive with the peptide substrate (<i>K</i><sub>i</sub> = 5 ± 0.3 nM) and uncompetitive with SAM. Compound <b>1</b> interacts with many residues playing essential roles in peptide binding or in the catalysis of methyl transfer. First, a key cation−π interaction was proposed between the phenyl ring of the tetrahydroisoquinoline (THIQ) and the cofactor SAM, which was believed to contribute to the high binding affinity and selectivity of compound <b>1</b> for PRMT5. The THIQ group also forms a potential π–π stacking interaction with Phe327, a conserved residue that was proposed to decide the sDMA of PRMT5. Removing the phenyl from the THIQ would lead to loss of the above interactions and thus resulted in a closely related but inactive compound <b>2</b> (EPZ019896, <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>A).<a onclick="showRef(event, 'ref56'); return false;" href="javascript:void(0);" class="ref ref56">(56)</a> Furthermore, the THIQ is situated in a small hydrophobic pocket formed by Leu319, Tyr324, Phe327, and Trp579, in which polar functional groups or bulky moieties are not favored. For example, pyridine replacement analogue <b>5</b> and dimethoxyl substituted analogue <b>6</b> lose PRMT5 inhibitory activity (IC<sub>50</sub> > 50 μM), compared to their parent compound <b>4</b> (IC<sub>50</sub> = 0.326 μM, <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>).<a onclick="showRef(event, 'ref57'); return false;" href="javascript:void(0);" class="ref ref57">(57)</a> Glu435, which is the key residue for enzyme catalysis, is possibly involved in a water-mediated interaction with the tertiary nitrogen atom of the THIQ ring system of compound <b>1</b>. Incorrect orientation of this interaction results in reduced potency, such as analogue <b>7</b> (IC<sub>50</sub> = 7.45 μM, <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a> and <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a> (IC<sub>50</sub> = 26.8 μM), compared with compound <b>4</b>. Additionally, there is a possible hydrogen bond interaction between the chiral hydroxyl of compound <b>1</b> and the side chain of Glu444, explaining the potency differences between stereoisomers <b>9</b> (IC<sub>50</sub> = 13 nM, <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>) and <b>10</b> (IC<sub>50</sub> = 95 nM). Moreover, the terminal pyrimidine group of compound <b>1</b> is poised for π–π stacking interactions with Phe327 and Phe580, likely contributing to its high inhibitory potency against PRMT5. All above interactions prompt compound <b>1</b> to adopt a “U” shaped conformation while binding to PRMT5 and ensure its high affinity and selectivity for PRMT5. Notably, the improved compound <b>3</b> (EPZ015938, <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>A), which was licensed to GSK (GSK3326595), has already entered phase 1 clinical trial to assess the safety, PK, and pharmacodynamics (PD) in patients with solid tumors and non-Hodgkin lymphoma (NCT02783300).<a onclick="showRef(event, 'ref58'); return false;" href="javascript:void(0);" class="ref ref58">(58)</a></div><figure id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-21/acs.jmedchem.8b00598/20181101/images/medium/jm-2018-00598z_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-21/acs.jmedchem.8b00598/20181101/images/large/jm-2018-00598z_0006.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Structures of compounds <b>4</b>–<b>8</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-21/acs.jmedchem.8b00598/20181101/images/large/jm-2018-00598z_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00598&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-21/acs.jmedchem.8b00598/20181101/images/medium/jm-2018-00598z_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-21/acs.jmedchem.8b00598/20181101/images/large/jm-2018-00598z_0007.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Structures of compounds <b>9</b> and <b>10</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-21/acs.jmedchem.8b00598/20181101/images/large/jm-2018-00598z_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00598&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">By screening a library of 350 000 compounds using the Transcreener EPIGEN technology, Falk and co-workers<a onclick="showRef(event, 'ref59'); return false;" href="javascript:void(0);" class="ref ref59">(59)</a> also found a series of SAM uncompetitive inhibitors with a similar core structure to compound <b>1</b>. Further optimization led to compound <b>11</b> (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>). It exhibits potent PRMT5 enzyme inhibitory activity with an IC<sub>50</sub> value of 109 nM and is highly selective for PRMT5. Compound <b>11</b> effectively blocks the arginine sDMA of both histone and non-histone substrates in TE-1 cells. Decrease of the H4R3me2s mark in bone marrow cells and peripheral white blood cells of C57BL/6 mice was observed after treatment with compound <b>11</b> in a dose-dependent manner, and no obvious weight loss was observed. In order to develop more potent PRMT5 inhibitors, diversity of the THIQ group was explored. Moving the alcohol scaffolding motif from the N-position of THIQ to the C3 position provided a new series of analogues with one methylene deletion, such as compound <b>12</b> (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>). It is highly potent againt PRMT5, with an IC<sub>50</sub> of 26 nM.<a onclick="showRef(event, 'ref60'); return false;" href="javascript:void(0);" class="ref ref60">(60)</a> Taking the linear amine group out of the THIQ motif produced analogues featuring a dihydroindene ring. The optimized compound <b>13</b> (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>) possesses similar potency (IC<sub>50</sub> = 105 nM) to compound <b>11</b>.<a onclick="showRef(event, 'ref61'); return false;" href="javascript:void(0);" class="ref ref61">(61)</a></div><figure id="fig8" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-21/acs.jmedchem.8b00598/20181101/images/medium/jm-2018-00598z_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-21/acs.jmedchem.8b00598/20181101/images/large/jm-2018-00598z_0008.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Structures of compounds <b>11</b>–<b>13</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-21/acs.jmedchem.8b00598/20181101/images/large/jm-2018-00598z_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00598&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">As depicted in the cocrystal structure of <b>1</b> with PRMT5:MEP50 and SAM, PRMT5 possesses two ligand binding pockets: a SAM-binding site and a substrate peptide binding site. Since the substrate-binding sites vary among PRMTs, a virtual screening against substrate-binding pocket could lead to discovery of inhibitors with good selectivity. On the basis of this theory, Zheng and co-workers<a onclick="showRef(event, 'ref62'); return false;" href="javascript:void(0);" class="ref ref62">(62)</a> discovered a novel scaffold PRMT5 inhibitor <b>14</b> (<a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a></a>) by molecular docking and confirmed by <sup>3</sup>H-labeled methylation assays. Compound <b>14</b> displays 69% inhibitory activity against PRMT5 at a concentration of 50 μM. By analyzing its predictive interactions with PRMT5, it was found that the phenyl group may form π–π stacking interaction with Phe327 and Phe580, similar to the terminal pyrimidine of compound <b>1</b>. Removal of the phenyl group afforded compound <b>15</b> (<a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a></a>), which lost PRMT5 inhibitory activity (2% inhibition at 50 μM), confirming the contribution of the π–π stacking interaction. Compound <b>14</b>, however, does not make a cation−π interaction with SAM nor the π–π stacking interaction with Phe327 and has only moderate potency against PRMT5 (IC<sub>50</sub> = 35.6 μM). Since these two interactions are critical to the outstanding affinity and selectivity of compound <b>1</b> for PRMT5, the result for compound <b>14</b> could be expected. Introducing a phenyl to the second chain of <b>14</b> restored the interactions and produced compound <b>16</b> (<a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a></a>) with 12 times improved activity (IC<sub>50</sub> = 2.8 μM), while it is inactive against PRMT1, EZH2, and DNMT3A. Compound <b>16</b> displays potent antiproliferative activities against Z-138, Maver-1, and Jeko-1 cancer cells with EC<sub>50</sub> values of 12.7 μM, 12.7 μM, and 10.5 μM, respectively.</div><figure id="fig9" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-21/acs.jmedchem.8b00598/20181101/images/medium/jm-2018-00598z_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-21/acs.jmedchem.8b00598/20181101/images/large/jm-2018-00598z_0009.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. Structures of compounds <b>14</b>–<b>16</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-21/acs.jmedchem.8b00598/20181101/images/large/jm-2018-00598z_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00598&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i7" class="anchor-spacer"></div><h3 class="article-section__title" id="_i7">4.2.  SAM Competitive Inhibitors</h3><div class="NLM_p">PRMTs contain highly conserved SAM-binding pockets, and it is not surprising that compound <b>17</b><a onclick="showRef(event, 'ref63'); return false;" href="javascript:void(0);" class="ref ref63">(63)</a> (sinefungin, a natural product, <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">10</a></a>A) and SAH,<a onclick="showRef(event, 'ref64'); return false;" href="javascript:void(0);" class="ref ref64">(64)</a> as close analogs of cofactor SAM, exhibit excellent PRMT5 inhibitory activity but lack of selectivity against other protein methyltransferases. In order to explore the binding mode of these inhibitors with PRMT5 and discover more selective inhibitors, the first crystal structure of human PRMT5:MEP50 in complex with H4 peptide and compound <b>18</b> (dehydrosinefungin, <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">10</a></a>B) was published by Antonysamy<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> in 2012. As shown in <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">10</a></a>B, compound <b>18</b> could interact with several residues around SAM binding pocket. The backbone NH of Met420 and the side chain of Asp419 may interact with adenosine moiety via hydrogen bonds. The side chains of Glu392 and Tyr324 form two potential hydrogen bonds with two hydroxy groups of <b>18</b>. In addition, the NH of allylic amine possibly interacts with guanidinium group of arginine residue located on the H4 peptide substrate (H4R3) via a hydrogen bond. Of particular note, the guanidinium group of H4R3 accepting a methyl group from SAM forms a hydrogen bond network with the side chains of the conserved “PRMT double-E loop” Glu435 and Glu444. At last, the tail acid group of <b>18</b> might be involved in hydrogen bond interactions with residue Tyr334. These interactions anchor compound <b>18</b> to the SAM binding pocket. As described above, the SAM binding site is conserved in PRMTs and compound <b>18</b> maintains the core interactions within the SAM binding site but lacks unique interactions with PRMT5, so it is still a pan-protein methyltransferase inhibitor and is not selective toward PRMT5.</div><figure id="fig10" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 10</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-21/acs.jmedchem.8b00598/20181101/images/medium/jm-2018-00598z_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-21/acs.jmedchem.8b00598/20181101/images/large/jm-2018-00598z_0010.jpeg" id="gr10" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 10. (A) Structure of compounds <b>17</b> and <b>18</b>. (B) Cocrystal structure of compound <b>18</b> with PRMT5:MEP50 in complex with H4 peptide (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4GQB">4GQB</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-21/acs.jmedchem.8b00598/20181101/images/large/jm-2018-00598z_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00598&amp;id=fig10"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">By replacing the amino acid end of SAH with ethyl urea group to recapitulate hydrogen bond interactions between the guanidinium group of H4R3 residue and E444, Schapira and co-workers<a onclick="showRef(event, 'ref65'); return false;" href="javascript:void(0);" class="ref ref65">(65)</a> designed and prepared compound <b>19</b> (DS-437, <a data-tab="pane-pcw-Figures" href="#fig11" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig11" id="" class=" internalNav">11</a></a>). It inhibits PRMT5:MEP50 complex and PRMT7 with an IC<sub>50</sub> value of 6 μM, while it is inactive against 29 other human protein-, DNA-, and RNA-MTases. It could also reduce the symmetrical arginine dimethylation of SmD1/3 and SmB proteins in breast cancer cells, representing a new scaffold to investigate the therapeutic potential for targeting the PRMT5-PRMT7 pathway.</div><figure id="fig11" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 11</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-21/acs.jmedchem.8b00598/20181101/images/medium/jm-2018-00598z_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-21/acs.jmedchem.8b00598/20181101/images/large/jm-2018-00598z_0011.jpeg" id="gr11" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 11. Structures of compounds <b>19</b>–<b>21</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-21/acs.jmedchem.8b00598/20181101/images/large/jm-2018-00598z_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00598&amp;id=fig11"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">A potent and selective PRMT5 inhibitor, compound <b>20</b> (LLY-283, <a data-tab="pane-pcw-Figures" href="#fig11" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig11" id="" class=" internalNav">11</a></a>),<a onclick="showRef(event, 'ref66'); return false;" href="javascript:void(0);" class="ref ref66">(66)</a> was discovered by Eli Lilly. Considering the chemical core structure similarity, compound <b>20</b> is still an analogue of SAM, with the methionine moiety replaced by a phenyl group. Compound <b>20</b> exhibits potent PRMT5 enzyme inhibitory activity with an IC<sub>50</sub> value of <b>20</b> nM. It also displays more than 100-fold selectivity for PRMT5 over other methyltransferases and nonepigenetic targets. Methylation of SmBB′ is inhibited by <b>20</b> with an IC<sub>50</sub> of 25 nM in MCF7 cells, while MDM4 splicing is inhibited with an IC<sub>50</sub> of 40 nM in A375 cells. Recently, the crystal structure of the PRMT5:MEP50 complex with compound <b>20</b> was determined.<a onclick="showRef(event, 'ref66'); return false;" href="javascript:void(0);" class="ref ref66">(66)</a> Comparing with the crystal structure of compound <b>18</b> and PRMT5:MEP50 complex, the adenine and ribose moieties of compound <b>20</b> adopt very similar conformations and make similar interactions with Asp419, Met420, Glu392, and Try324. But the phenyl group of compound <b>20</b> occupies the position of the side chain of Phe327, a residue specific to PRMT5. This change may contribute to the high selectivity of compound <b>20</b> for PRMT5. Aiming at developing SAM analogues as selective PRMT5 inhibitors with improved PK profile and oral bioavailability, researchers in Janssen displaced the tetrahydrofuran ring of SAM with cyclopentane and developed a new series of PRMT5 inhibitors, compound <b>21</b> being one example with IC<sub>50</sub> value for PRMT5 at 6.3 nM.<a onclick="showRef(event, 'ref67'); return false;" href="javascript:void(0);" class="ref ref67">(67)</a></div><div class="NLM_p">A series of <i>N</i>-alkyl-9<i>H</i>-carbazole analogs as PRMT5 inhibitors were discovered by Baiocchi and co-workers<a onclick="showRef(event, 'ref68'); return false;" href="javascript:void(0);" class="ref ref68">(68)</a> by employing a human PRMT5 catalytic site model and virtual screening of a 10 000-compound library. Taking a process of structure-based computational analysis and combinatorial lead optimization, the group obtained compound <b>22</b> (CMP5, <a data-tab="pane-pcw-Figures" href="#fig12" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig12" id="" class=" internalNav">12</a></a>), the lead candidate capable of blocking EBV-driven B-lymphocyte transformation and survival while keeping normal B cells unaffected. Compound <b>22</b> selectively inhibits the formation of sDMA on H4R3, but it is inactive against type I (PRMT1, PRMT4) and type III (PRMT7) enzymes. A model of compound <b>22</b> binding to PRMT5 shows that the pyridine ring possibly forms π–π stacking interactions with Phe327 residue, while the carbazole ring of CMP5 is orientated toward the adenosine binding site. It is important to point out that PRMT5 enzymatic inhibitory activity of compound <b>22</b> was not reported in the original reference. Until recently Zhang and co-workers showed that <b>22</b> displayed 7.8% inhibitory activity against PRMT5 at 50 μM using radioactive methylation assay.<a onclick="showRef(event, 'ref69'); return false;" href="javascript:void(0);" class="ref ref69">(69)</a> After the second-round optimization based on the core structure of <b>22</b>, compound <b>23</b> (HLCL-61, <a data-tab="pane-pcw-Figures" href="#fig12" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig12" id="" class=" internalNav">12</a></a>)<a onclick="showRef(event, 'ref70'); return false;" href="javascript:void(0);" class="ref ref70">(70)</a> was developed as a selective PRMT5 inhibitor by replacing the pyridine group of <b>22</b> with an ortho-methoxyphenyl group. It was proposed that this change not only reserved the π–π stacking interaction with Phe327 but also introduced a new hydrogen bond interaction with the protein residue. Compound <b>23</b> maintains its specificity for PRMT5 and effectively inhibits symmetric arginine dimethylation of histones H3 and H4 in AML samples. Treatment of AML cell lines (MV4-11 and THP-1) and primary blasts with <b>23</b> showed decreased cell viability with an IC<sub>50</sub> of 7.21–21.46 μM for cell lines and 3.98–8.72 μM for patient samples. Additionally, the effect of introducing a halogen atom to the carbazole ring of <b>23</b> was explored. A halogen bond can be formed between a halogen atom and a negative site or the π electrons of an unsaturated system (O, N, S, or π) from the surrounding residues,<a onclick="showRef(event, 'ref71'); return false;" href="javascript:void(0);" class="ref ref71">(71)</a> analogous to hydrogen bond interactions.<a onclick="showRef(event, 'ref72'); return false;" href="javascript:void(0);" class="ref ref72">(72)</a> As expected, introduction of bromine to the carbazole of <b>23</b> produced a more selective and potent PRMT5 inhibitor, compound <b>24</b> (HLCL-65, <a data-tab="pane-pcw-Figures" href="#fig12" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig12" id="" class=" internalNav">12</a></a>).<a onclick="showRef(event, 'ref73'); return false;" href="javascript:void(0);" class="ref ref73">(73)</a> It selectively inhibits the formation of sDMA mediated by PRMT5 and demonstrates potent antiproliferative effect against Th1 cells (IC<sub>50</sub> = 1.1 μM) and Th2 cells (IC<sub>50</sub> = 4.0 μM).</div><figure id="fig12" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Figure 12</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-21/acs.jmedchem.8b00598/20181101/images/medium/jm-2018-00598z_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-21/acs.jmedchem.8b00598/20181101/images/large/jm-2018-00598z_0012.jpeg" id="gr12" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 12. Structures of compounds <b>22</b>–<b>25</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-21/acs.jmedchem.8b00598/20181101/images/large/jm-2018-00598z_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00598&amp;id=fig12"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">To study PRMT5 functions in chronic myelogenous leukemia, Pan and co-workers<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a> screened and discovered a potent small molecule PRMT5 inhibitor, compound <b>25</b> (PJ-68, <a data-tab="pane-pcw-Figures" href="#fig12" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig12" id="" class=" internalNav">12</a></a>), which employed the core structure of <i>N</i>-ethyl-9<i>H</i>-carbazole. Compound <b>25</b> inhibits PRMT5 with an IC<sub>50</sub> of 517 nM while having no effect on the activity of type I PRMT family members (PRMT1, -3, -4, -6, and -8), indicating its specificity for PRMT5. It inhibits the formation of sDMA on histone H2A and H4 in K562 cells in a concentration-dependent manner and significantly decreases the level of BCR-ABL in both CD34<sup>+</sup>CD38<sup>–</sup> cells and CD34<sup>+</sup>CD38<sup>+</sup> cells. PK study in SD rats showed that plasma concentrations were 3.57 ± 0.08 μM (1264.4 ± 28.4 ng/mL) and 7.88 ± 0.31 μM (2788.5 ± 109.7 ng/mL), respectively, after a single oral or intravenous administration, which were supposed to be sufficient to block PRMT5 activity in vitro.</div><div class="NLM_p">Recently, by taking advantage of structure-based virtual screening, Luo and co-workers<a onclick="showRef(event, 'ref74'); return false;" href="javascript:void(0);" class="ref ref74">(74)</a> discovered a new scaffold of PRMT5 inhibitors, as exemplified by compound <b>26</b> (<a data-tab="pane-pcw-Figures" href="#fig13" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig13" id="" class=" internalNav">13</a></a>) with the IC<sub>50</sub> of 0.33 μM. Compound <b>26</b> displays a broad selectivity against a panel of MTases. It binds directly to PRMT5 as validated by SPR, with an equilibrium dissociation constant (<i>K</i><sub>D</sub>) of 0.987 μM. Compound <b>26</b> exhibits competitive inhibition with respect to SAM againt PRMT5 and shows noncompetitive inhibition with respect to peptide substrate, suggesting a SAM competitive PRMT5 inhibitor. Compound <b>26</b> was optimized from hit compound <b>27</b> (IC<sub>50</sub> = 8.1 μM, <a data-tab="pane-pcw-Figures" href="#fig13" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig13" id="" class=" internalNav">13</a></a>) by introducing a methoxy group to the 5-position of benzimidazole. Molecular docking predicted that the methoxy group would form a hydrogen bond with Tyr334<a onclick="showRef(event, 'ref69'); return false;" href="javascript:void(0);" class="ref ref69">(69)</a> and improve inhibitory activity from compound <b>26</b>. It was also predicted that the methoxycarbonyl substituted phenyl was close to a pocket occupied by the adenosine group of SAM. A hydrogen bond interaction was predicted between the carbonyl oxygen of the methoxycarbonyl group and Lys393,<a onclick="showRef(event, 'ref69'); return false;" href="javascript:void(0);" class="ref ref69">(69)</a> a critical interaction determining the high affinity of compound <b>13</b> to PRMT5. Analogues such as compounds <b>28</b>–<b>30</b> without this potential interaction are much weaker binders (<a data-tab="pane-pcw-Figures" href="#fig13" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig13" id="" class=" internalNav">13</a></a>). Unlike the adenosine group of SAM forming two potential hydrogen bonds with Met420 and Asp419, compound <b>26</b> does not engage these interactions. It can be expected that restoring these interactions may further improve its inhibitory activity.</div><figure id="fig13" data-index="13" class="article__inlineFigure"><h2 class="fig-label">Figure 13</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-21/acs.jmedchem.8b00598/20181101/images/medium/jm-2018-00598z_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-21/acs.jmedchem.8b00598/20181101/images/large/jm-2018-00598z_0013.jpeg" id="gr13" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 13. Structures of compounds <b>26–31</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-21/acs.jmedchem.8b00598/20181101/images/large/jm-2018-00598z_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00598&amp;id=fig13"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">To mimic the hydrogen bond interactions between the guanidinium group of substrate arginine residue and “the double E loop” (E435 and E444, <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>), Yang and co-workers<a onclick="showRef(event, 'ref75'); return false;" href="javascript:void(0);" class="ref ref75">(75)</a> developed a series of new PRMT5 inhibitors. They first screened guanidinium analogues, such as 1,2-diamines, benzimidamide, semicarbazone, and amines. Then by performing molecular docking-based virtual screening, a highly selective PRMT5 inhibitor <b>31</b> (P5i-6, <a data-tab="pane-pcw-Figures" href="#fig13" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig13" id="" class=" internalNav">13</a></a>) was discovered. It was based on the fragment of hydrazinecarboxamide. This compound could form multiple hydrogen bonds with residues E435, E444, and L437 in its predicted binding site. Compound <b>31</b> displays potent inhibitory activity against PRMT5 with an IC<sub>50</sub> value of 0.57 μM and is inactive against PRMT1, -3, -4, -6, -7, and -8. Further study showed that the compound was active against two colorectal cancer cell lines (HT-29 and DLD-1) and one hepatic cancer cell line (HepG2) and that it selectively inhibited the formation of sDMA on H4R3 and H3R8 in DLD-1 cells. However, it was uncertain whether it was a SAM cooperative inhibitor or a SAM competitive one without kinetic experiment data.</div></div></div><div id="sec5" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i8">5.  Perspectives</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_73376" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_73376" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">In the past 5 years, a significant number of chemical structures have been discovered to be PRMT5 inhibitors, as we discussed above, and several typical ones were summarized and compared in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>. SAM uncompetitive inhibitors inhibit PRMT5 by occupying the peptide binding pocket in the presence of SAM. This binding mode may offer great advantages. Cellular concentration of SAM is reported to be in several hundred μM range. With PRMT5’s <i>K</i><sub>m</sub> for SAM in low μM and structural information pointing to sequential bi–bi mechanism with SAM binding first, the majority of the enzyme would exist in the SAM complexed form in the cell. Kinetically it is advantageous to directly inhibit the major enzyme form. Compound <b>3</b> with this mode of action has already progressed into clinical trial for the treatment of MCL and solid tumor, based on preclinical activity on several MCL cell lines, good selectivity profile, and animal toxicity results. However, studies reported by Novartis and Agios Inc. raised some concerns regarding this mode of inhibition.<a onclick="showRef(event, 'ref48 ref49'); return false;" href="javascript:void(0);" class="ref ref48 ref49">(48,49)</a> Both groups were investigating the association between cancer cell dependence on PRMT5 function and MTAP genetic status. They indicated that compound <b>1</b> effectively inhibited in vitro growth of many selected cell lines; however, it does so regardless of MTAP status, nor do the results correlate with PRMT5 expression level. Since loss of PRMT5 is known to be embryonic lethal, these findings raised the question if lack of selectivity in cancer cell lines would translate to greater toxicity later in clinical trials. It was shown by Sellers et al. that MTAP deficient cell lines accumulated metabolic molecule methylthioadenosine (MTA), which specifically inhibits PRMT5 in vitro, and it was suggested that PRMT5 in these cells is indeed partially inhibited by MTA, rendering cell lines especially sensitive to further PRMT5 attenuation. If that is true, then in MTAP deficient cells, PRMT5 is partially complexed with MTA, which is different from PRMT5:SAM complex in normal cells. This offers a unique opportunity to target PRMT5 in these diseased cells. One strategy involves designing or screening for small molecules that could synergistically bind to PRMT5 with MTA and drive the whole PRMT5 population to inactive state, thus inhibiting cell growth. For the SAM competitive binding mode, several compounds showed selective inhibition of PRMT5, albeit the highly conserved SAM binding site within the PRMT family and other MTase domain containing proteins. So it is possible to achieve high selectivity among MTases with SAM mimetics. However, PK could be a potential problem with SAM mimetics and thus should be taken into consideration when developing this type of inhibitor into therapies.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Comparison of Typical PRMT5 Inhibitors</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compd</th><th class="colsep0 rowsep0" align="center">binding site</th><th class="colsep0 rowsep0" align="center">selectivity for PRMT5</th><th class="colsep0 rowsep0" align="center">MOA</th><th class="colsep0 rowsep0" align="center">biochemical activity, IC<sub>50</sub> (nM)</th><th class="colsep0 rowsep0" align="center">cellular activity (nM)/cell line</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>1</b></td><td class="colsep0 rowsep0" align="left">substrate-binding site</td><td class="colsep0 rowsep0" align="left">high</td><td class="colsep0 rowsep0" align="left">SAM-uncompetitive inhibition; substrate-competitive inhibition</td><td class="colsep0 rowsep0" align="left">22</td><td class="colsep0 rowsep0" align="left">IC<sub>50</sub> = 96/Z-138</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">IC<sub>50</sub> = 450/Maver-1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>16</b></td><td class="colsep0 rowsep0" align="left">substrate-binding site</td><td class="colsep0 rowsep0" align="left">good</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">2800</td><td class="colsep0 rowsep0" align="left">EC<sub>50</sub> = 12700/Z-138</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">EC<sub>50</sub> = 12700/Maver-1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">EC<sub>50</sub> = 10500/Jeko-1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>18</b></td><td class="colsep0 rowsep0" align="left">SAM-binding site</td><td class="colsep0 rowsep0" align="left">no</td><td class="colsep0 rowsep0" align="left">SAM-competitive inhibition</td><td class="colsep0 rowsep0" align="left">35</td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>20</b></td><td class="colsep0 rowsep0" align="left">SAM-binding site</td><td class="colsep0 rowsep0" align="left">remarkable</td><td class="colsep0 rowsep0" align="left">SAM-noncompetitive inhibition; substrate-noncompetitive inhibition</td><td class="colsep0 rowsep0" align="left">20</td><td class="colsep0 rowsep0" align="left">IC<sub>50</sub> = 25/MCF7</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">IC<sub>50</sub> = 40/A375</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>24</b></td><td class="colsep0 rowsep0" align="left">SAM-binding site</td><td class="colsep0 rowsep0" align="left">good</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">517</td><td class="colsep0 rowsep0" align="left">IC<sub>50</sub> = 1100/Th1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">IC<sub>50</sub> = 4000/Th2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>26</b></td><td class="colsep0 rowsep0" align="left">SAM-binding site</td><td class="colsep0 rowsep0" align="left">broad</td><td class="colsep0 rowsep0" align="left">SAM-competitive inhibition; substrate-noncompetitive inhibition</td><td class="colsep0 rowsep0" align="left">330</td><td class="colsep0 rowsep0" align="left">EC<sub>50</sub> = 6530/MV4-11</td></tr></tbody></table></div></div><div class="NLM_p">In addition, care must be taken to develop any PRMT5 inhibitors into clinical use. Experiments with conditional knockout of PRMT5 in adult mice demonstrated the essential role of PRMT5 in the maintenance of adult hematopoietic cells.<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> PRMT5-deficient hematopoietic stem and progenitor cells exhibited severely impaired cytokine signaling as well as upregulation of p53 and expression of its downstream targets, resulting in a rapid and profound effect on blood cell production. Treating cancer patients with PRMT5 inhibitors could potentially result in significant myelosuppressive side effects. Nonetheless, the clinical activity and side effects of chemical inhibitors of PRMT5 will need to be defined in the clinic.</div><div class="NLM_p last">In conclusion, targeting PRMT5 offers a great opportunity to combat cancer in an efficient and selective way. However, more biology regarding PRMT5 function needs to be revealed for better understanding of its therapeutic potentials and more potent and selective PRMT5 inhibitors need to be developed to validate PRMT5 as an anticancer target.</div></div><div class="NLM_back"><div class="article_supporting-info" id=""></div><div class="testing" data-doi="10.1021/acs.jmedchem.8b00598" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_13282" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_13282" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yuanxiang Wang</span> - <span class="hlFld-Affiliation affiliation">School of Pharmaceutical
Sciences, Sun Yat-Sen University, Guangzhou 510006, China</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-5650-4396" title="Orcid link">http://orcid.org/0000-0001-5650-4396</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#fa8d9b949d8382c3cfba979b9396d48983898fd49f9e8fd49994"><span class="__cf_email__" data-cfemail="e097818e879998d9d5a08d81898cce93999395ce858495ce838e">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yanqiu Yuan</span> - <span class="hlFld-Affiliation affiliation">School of Pharmaceutical
Sciences, Sun Yat-Sen University, Guangzhou 510006, China</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#b0c9c5d1dec9c188f0ddd1d9dc9ec3c9c3c59ed5d4c59ed3de"><span class="__cf_email__" data-cfemail="156c60747b6c642d5578747c793b666c66603b7071603b767b">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Wenhao Hu</span> - <span class="hlFld-Affiliation affiliation">School of Pharmaceutical
Sciences, Sun Yat-Sen University, Guangzhou 510006, China</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-1461-3671" title="Orcid link">http://orcid.org/0000-0002-1461-3671</a></span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>The manuscript was written through contributions of all authors. All authors have given approval to the final version of the manuscript.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="bio" rid="ath1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="biographies">Biographies</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_74812" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_74812" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><figure id="ath1" data-index="14" class="article__inlineFigure"><h2 class="fig-label">Yuanxiang Wang</h2><div class="figure-bottom-links"><div class="download-hi-res-img"></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00598&amp;id=BIO-d7e1535-autogenerated"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><b>Yuanxiang Wang</b> is currently an Associate Professor at School of Pharmaceutical Sciences, Sun Yat-Sen University (SYSU). He received a Ph.D. degree in Medicinal Chemistry from Shanghai Institute of Materia Medica (SIMM), Chinese Academy of Sciences (CAS) in 2012. Dr. Wang completed postdoctoral training at the University of Arizona and the University of Utah from 2012 to 2017. In 2017, Dr. Wang joined SYSU as a faculty member. His research interests include clinical directed anticancer drug design and discovery, and development of new synthetic methodologies for creating peptide therapeutics.</p></figure></div><div class="bio" rid="ath2"><figure id="ath2" data-index="15" class="article__inlineFigure"><h2 class="fig-label">Wenhao Hu</h2><div class="figure-bottom-links"><div class="download-hi-res-img"></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00598&amp;id=BIO-d7e1540-autogenerated"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><b>Wenhao Hu</b> is currently a Full Professor and Dean of School of Pharmaceutical Sciences at SYSU. He received his Ph.D. degree from Hong Kong Polytechnic University with Professor Albert S. C. Chan in 1998. He then became a postdoctoral fellow at the University of Arizona with Professor Michael P. Doyle. In 2002–2006, he worked as a medicinal chemist at Genesoft in San Francisco and a process chemist at Bristol-Myers Squibb in New Jersey. In 2006 he joined East China Normal University, where he was a Professor of Chemistry and Head of Shanghai Engineering Research Center of Molecular Therapeutics and New Drug Development. He moved to SYSU in August 2016. His research interests include synthetic methodology, multicomponent reactions, medicinal chemistry, and process chemistry.</p></figure></div><div class="bio" rid="ath3"><figure id="ath3" data-index="16" class="article__inlineFigure"><h2 class="fig-label">Yanqiu Yuan</h2><div class="figure-bottom-links"><div class="download-hi-res-img"></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00598&amp;id=BIO-d7e1546-autogenerated"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><b>Yanqiu Yuan</b> is currently an Associate Professor at School of Pharmaceutical Sciences, SYSU. She received her Ph.D. degree at Harvard University in 2008 and subsequently worked at Novartis Institute for Biomedical Research at Cambridge as Investigator and at University of Copenhagen as Assistant Professor. Her research interests include the biochemical function of PRMT5 and discovery of small molecule inhibitors as anticancer therapy.</p></figure></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i14">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_44637" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_44637" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">We acknowledge startup funds from SYSU (Grant 36000-18831104 to Y.W.; Grant 36000-18831102 to Y.Y.) and Guangdong Innovative and Entrepreneurial Research Team Program (Grant 2016ZT06Y337 to W.H.) for financial support.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations Used</td></tr><tr><td class="NLM_term">PRMT5</td><td class="NLM_def"><p class="first last">protein arginine methyltransferase 5</p></td></tr><tr><td class="NLM_term">SAM</td><td class="NLM_def"><p class="first last"><i>S</i>-adenosylmethionine</p></td></tr><tr><td class="NLM_term">SAH</td><td class="NLM_def"><p class="first last"><i>S</i>-adenosyl-<span class="smallcaps smallerCapital">l</span>-homocysteine;</p></td></tr><tr><td class="NLM_term">aDMA</td><td class="NLM_def"><p class="first last">asymmetric dimethylarginine</p></td></tr><tr><td class="NLM_term">sDMA</td><td class="NLM_def"><p class="first last">symmetric dimethylarginine</p></td></tr><tr><td class="NLM_term">MMA</td><td class="NLM_def"><p class="first last">monomethylarginine</p></td></tr><tr><td class="NLM_term">MTase</td><td class="NLM_def"><p class="first last">methyltransferase</p></td></tr><tr><td class="NLM_term">AR</td><td class="NLM_def"><p class="first last">androgen receptor</p></td></tr><tr><td class="NLM_term">LSC</td><td class="NLM_def"><p class="first last">leukemia stem cell</p></td></tr><tr><td class="NLM_term">CML</td><td class="NLM_def"><p class="first last">chronic myelogenous leukemia</p></td></tr><tr><td class="NLM_term">MTAP</td><td class="NLM_def"><p class="first last">methylthioadenosin phosphorylase</p></td></tr><tr><td class="NLM_term">PMT</td><td class="NLM_def"><p class="first last">protein lysine methyltransferase</p></td></tr><tr><td class="NLM_term">PKMT</td><td class="NLM_def"><p class="first last">protein lysine methyltransferase</p></td></tr><tr><td class="NLM_term">MCL</td><td class="NLM_def"><p class="first last">mantle cell lymphoma</p></td></tr><tr><td class="NLM_term">CETSA</td><td class="NLM_def"><p class="first last">cellular thermal shift assay</p></td></tr><tr><td class="NLM_term">TGI</td><td class="NLM_def"><p class="first last">tumor growth inhibition</p></td></tr><tr><td class="NLM_term">THIQ</td><td class="NLM_def"><p class="first last">tetrahydroisoquinoline</p></td></tr><tr><td class="NLM_term">PK</td><td class="NLM_def"><p class="first last">pharmacokinetic</p></td></tr><tr><td class="NLM_term">PD</td><td class="NLM_def"><p class="first last">pharmacodynamics</p></td></tr><tr><td class="NLM_term">MTA</td><td class="NLM_def"><p class="first last">methylthioadenosine</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i16">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_08648" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_08648" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 75 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Peng, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, C. C. L.</span></span> <span> </span><span class="NLM_article-title">The story of protein arginine methylation: characterization, regulation, and function</span>. <i>Expert Rev. Proteomics</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">157</span>– <span class="NLM_lpage">170</span>, <span class="refDoi"> DOI: 10.1080/14789450.2017.1275573</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=10.1080%2F14789450.2017.1275573" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=28043171" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=1%3ACAS%3A528%3ADC%252BC2sXltVWluw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2017&pages=157-170&author=C.+Pengauthor=C.+C.+L.+Wong&title=The+story+of+protein+arginine+methylation%3A+characterization%2C+regulation%2C+and+function&doi=10.1080%2F14789450.2017.1275573"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">The story of protein arginine methylation: characterization, regulation, and function</span></div><div class="casAuthors">Peng, Chao; Wong, Catherine C. L.</div><div class="citationInfo"><span class="NLM_cas:title">Expert Review of Proteomics</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">157-170</span>CODEN:
                <span class="NLM_cas:coden">ERPXA3</span>;
        ISSN:<span class="NLM_cas:issn">1478-9450</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">Arginine methylation is an important post-translational modification (PTM) in cells, which is catalyzed by a group of protein arginine methyltransferases (PRMTs).  It plays significant roles in diverse cellular processes and various diseases.  Misregulation and aberrant expression of PRMTs can provide potential biomarkers and therapeutic targets for drug discovery.  Herein, we review the arginine methylation literature and summarize the methodologies for the characterization of this modification, as well as describe the recent insights into arginine methyltransferases and their biol. functions in diseases.  Benefits from the enzyme-based large-scale screening approach, the novel affinity enrichment strategies, arginine methylated protein family is the focus of attention.  Although a no. of arginine methyltransferases and related substrates are identified, the catalytic mechanism of different types of PRMTs remains unclear and few related demethylases are characterized.  Novel functional studies continuously reveal the importance of this modification in the cell cycle and diseases.  A deeper understanding of arginine methylated proteins, modification sites, and their mechanisms of regulation is needed to explore their role in life processes, esp. their relationship with diseases, thus accelerating the generation of potent, selective, cell-penetrant drug candidates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGriWhBKuJixw7Vg90H21EOLACvtfcHk0lhPGE-qq2gkvA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXltVWluw%253D%253D&md5=9f22ba72f797e050f75000112fbe4147</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1080%2F14789450.2017.1275573&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F14789450.2017.1275573%26sid%3Dliteratum%253Aachs%26aulast%3DPeng%26aufirst%3DC.%26aulast%3DWong%26aufirst%3DC.%2BC.%2BL.%26atitle%3DThe%2520story%2520of%2520protein%2520arginine%2520methylation%253A%2520characterization%252C%2520regulation%252C%2520and%2520function%26jtitle%3DExpert%2520Rev.%2520Proteomics%26date%3D2017%26volume%3D14%26spage%3D157%26epage%3D170%26doi%3D10.1080%2F14789450.2017.1275573" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Larsen, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sylvestersen, K. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mund, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lyon, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mullari, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Madsen, M. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daniel, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jensen, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nielsen, M. L.</span></span> <span> </span><span class="NLM_article-title">Proteome-wide analysis of arginine monomethylation reveals widespread occurrence in human cells</span>. <i>Sci. Signaling</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">rs9</span>, <span class="refDoi"> DOI: 10.1126/scisignal.aaf7329</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=10.1126%2Fscisignal.aaf7329" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=27577262" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2016&pages=rs9&author=S.+C.+Larsenauthor=K.+B.+Sylvestersenauthor=A.+Mundauthor=D.+Lyonauthor=M.+Mullariauthor=M.+V.+Madsenauthor=J.+A.+Danielauthor=L.+J.+Jensenauthor=M.+L.+Nielsen&title=Proteome-wide+analysis+of+arginine+monomethylation+reveals+widespread+occurrence+in+human+cells&doi=10.1126%2Fscisignal.aaf7329"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1126%2Fscisignal.aaf7329&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscisignal.aaf7329%26sid%3Dliteratum%253Aachs%26aulast%3DLarsen%26aufirst%3DS.%2BC.%26aulast%3DSylvestersen%26aufirst%3DK.%2BB.%26aulast%3DMund%26aufirst%3DA.%26aulast%3DLyon%26aufirst%3DD.%26aulast%3DMullari%26aufirst%3DM.%26aulast%3DMadsen%26aufirst%3DM.%2BV.%26aulast%3DDaniel%26aufirst%3DJ.%2BA.%26aulast%3DJensen%26aufirst%3DL.%2BJ.%26aulast%3DNielsen%26aufirst%3DM.%2BL.%26atitle%3DProteome-wide%2520analysis%2520of%2520arginine%2520monomethylation%2520reveals%2520widespread%2520occurrence%2520in%2520human%2520cells%26jtitle%3DSci.%2520Signaling%26date%3D2016%26volume%3D9%26spage%3Drs9%26doi%3D10.1126%2Fscisignal.aaf7329" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Guo, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gu, H. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mulhern, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aguiar, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kornhauser, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jia, X. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ren, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beausoleil, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Silva, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vemulapalli, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bedford, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Comb, M. J.</span></span> <span> </span><span class="NLM_article-title">Immunoaffinity enrichment and mass spectrometry analysis of protein methylation</span>. <i>Mol. Cell. Proteomics</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">372</span>– <span class="NLM_lpage">387</span>, <span class="refDoi"> DOI: 10.1074/mcp.O113.027870</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=10.1074%2Fmcp.O113.027870" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=24129315" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=1%3ACAS%3A528%3ADC%252BC2cXit1ynug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2014&pages=372-387&author=A.+L.+Guoauthor=H.+B.+Guauthor=J.+Zhouauthor=D.+Mulhernauthor=Y.+Wangauthor=K.+A.+Leeauthor=V.+Yangauthor=M.+Aguiarauthor=J.+Kornhauserauthor=X.+Y.+Jiaauthor=J.+M.+Renauthor=S.+A.+Beausoleilauthor=J.+C.+Silvaauthor=V.+Vemulapalliauthor=M.+T.+Bedfordauthor=M.+J.+Comb&title=Immunoaffinity+enrichment+and+mass+spectrometry+analysis+of+protein+methylation&doi=10.1074%2Fmcp.O113.027870"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Immunoaffinity Enrichment and Mass Spectrometry Analysis of Protein Methylation</span></div><div class="casAuthors">Guo, Ailan; Gu, Hongbo; Zhou, Jing; Mulhern, Daniel; Wang, Yi; Lee, Kimberly A.; Yang, Vicky; Aguiar, Mike; Kornhauser, Jon; Jia, Xiaoying; Ren, Jianmin; Beausoleil, Sean A.; Silva, Jeffrey C.; Vemulapalli, Vidyasiri; Bedford, Mark T.; Comb, Michael J.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular & Cellular Proteomics</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">372-387</span>CODEN:
                <span class="NLM_cas:coden">MCPOBS</span>;
        ISSN:<span class="NLM_cas:issn">1535-9484</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Protein methylation is a common posttranslational modification that mostly occurs on arginine and lysine residues.  Arginine methylation has been reported to regulate RNA processing, gene transcription, DNA damage repair, protein translocation, and signal transduction.  Lysine methylation is best known to regulate histone function and is involved in epigenetic regulation of gene transcription.  To better study protein methylation, we have developed highly specific antibodies against monomethyl arginine; asym. di-Me arginine; and monomethyl, di-Me, and tri-Me lysine motifs.  These antibodies were used to perform immunoaffinity purifn. of Me peptides followed by LC-MS/MS anal. to identify and quantify arginine and lysine methylation sites in several model studies.  Overall, we identified over 1000 arginine methylation sites in human cell line and mouse tissues, and ∼160 lysine methylation sites in human cell line HCT116.  The no. of methylation sites identified in this study exceeds those found in the literature to date.  Detailed anal. of arginine-methylated proteins obsd. in mouse brain compared with those found in mouse embryo shows a tissue-specific distribution of arginine methylation, and extends the types of proteins that are known to be arginine methylated to include many new protein types.  Many arginine-methylated proteins that we identified from the brain, including receptors, ion channels, transporters, and vesicle proteins, are involved in synaptic transmission, whereas the most abundant methylated proteins identified from mouse embryo are transcriptional regulators and RNA processing proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpBdVRZ74GxBrVg90H21EOLACvtfcHk0lhPGE-qq2gkvA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXit1ynug%253D%253D&md5=244c97ca8bd924f6cb6071908a7d38e4</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1074%2Fmcp.O113.027870&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fmcp.O113.027870%26sid%3Dliteratum%253Aachs%26aulast%3DGuo%26aufirst%3DA.%2BL.%26aulast%3DGu%26aufirst%3DH.%2BB.%26aulast%3DZhou%26aufirst%3DJ.%26aulast%3DMulhern%26aufirst%3DD.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DLee%26aufirst%3DK.%2BA.%26aulast%3DYang%26aufirst%3DV.%26aulast%3DAguiar%26aufirst%3DM.%26aulast%3DKornhauser%26aufirst%3DJ.%26aulast%3DJia%26aufirst%3DX.%2BY.%26aulast%3DRen%26aufirst%3DJ.%2BM.%26aulast%3DBeausoleil%26aufirst%3DS.%2BA.%26aulast%3DSilva%26aufirst%3DJ.%2BC.%26aulast%3DVemulapalli%26aufirst%3DV.%26aulast%3DBedford%26aufirst%3DM.%2BT.%26aulast%3DComb%26aufirst%3DM.%2BJ.%26atitle%3DImmunoaffinity%2520enrichment%2520and%2520mass%2520spectrometry%2520analysis%2520of%2520protein%2520methylation%26jtitle%3DMol.%2520Cell.%2520Proteomics%26date%3D2014%26volume%3D13%26spage%3D372%26epage%3D387%26doi%3D10.1074%2Fmcp.O113.027870" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McBride, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Silver, P. A.</span></span> <span> </span><span class="NLM_article-title">State of the Arg: Protein methylation at arginine comes of age</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>106</i></span>,  <span class="NLM_fpage">5</span>– <span class="NLM_lpage">8</span>, <span class="refDoi"> DOI: 10.1016/S0092-8674(01)00423-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=10.1016%2FS0092-8674%2801%2900423-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=11461695" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=1%3ACAS%3A528%3ADC%252BD3MXlsVCqsbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=106&publication_year=2001&pages=5-8&author=A.+E.+McBrideauthor=P.+A.+Silver&title=State+of+the+Arg%3A+Protein+methylation+at+arginine+comes+of+age&doi=10.1016%2FS0092-8674%2801%2900423-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">State of the arg: protein methylation at arginine comes of age</span></div><div class="casAuthors">McBride, Anne E.; Silver, Pamela A.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">106</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">5-8</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review with 20 refs.  Posttranslational modification of proteins allows the cell to expand its repertoire beyond the constraints imposed by the twenty encoded amino acids.  Methylation at arginines, although discovered over 30 yr ago, has only recently come to the attention of cell biologists through a combination of genetic and mol. biol. expts. that have implicated arginine methylation in processes from signaling and transcription activation to protein sorting.  The panoply of arginine methylated substrates suggests that this specifically eukaryotic modification may parallel phosphorylation in its level of complexity.  We will summarize much of the recent information about the function of methylation and the methyltransferase enzymes that modify arginines.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpL9vHYeFQ-aLVg90H21EOLACvtfcHk0lhkEBXW4HDv6w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXlsVCqsbY%253D&md5=e2191f38b78db27b75b3af07684ea3cb</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1016%2FS0092-8674%2801%2900423-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0092-8674%252801%252900423-8%26sid%3Dliteratum%253Aachs%26aulast%3DMcBride%26aufirst%3DA.%2BE.%26aulast%3DSilver%26aufirst%3DP.%2BA.%26atitle%3DState%2520of%2520the%2520Arg%253A%2520Protein%2520methylation%2520at%2520arginine%2520comes%2520of%2520age%26jtitle%3DCell%26date%3D2001%26volume%3D106%26spage%3D5%26epage%3D8%26doi%3D10.1016%2FS0092-8674%2801%2900423-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Auclair, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richard, S.</span></span> <span> </span><span class="NLM_article-title">The role of arginine methylation in the DNA damage response</span>. <i>DNA Repair</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">459</span>– <span class="NLM_lpage">465</span>, <span class="refDoi"> DOI: 10.1016/j.dnarep.2013.04.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=10.1016%2Fj.dnarep.2013.04.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=23684798" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=1%3ACAS%3A528%3ADC%252BC3sXnvVGksbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2013&pages=459-465&author=Y.+Auclairauthor=S.+Richard&title=The+role+of+arginine+methylation+in+the+DNA+damage+response&doi=10.1016%2Fj.dnarep.2013.04.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">The role of arginine methylation in the DNA damage response</span></div><div class="casAuthors">Auclair, Yannick; Richard, Stephane</div><div class="citationInfo"><span class="NLM_cas:title">DNA Repair</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">459-465</span>CODEN:
                <span class="NLM_cas:coden">DRNEAR</span>;
        ISSN:<span class="NLM_cas:issn">1568-7864</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Post-translational modifications are well-known modulators of DNA damage signaling and epigenetic gene expression.  Protein arginine methylation is a covalent modification that results in the addn. of Me groups to the nitrogen atoms of the arginine side chains and is catalyzed by a family of protein arginine methyltransferases (PRMTs).  In the past, arginine methylation was mainly obsd. on abundant proteins such as RNA-binding proteins and histones, but recent advances have revealed a plethora of arginine methylated proteins implicated in a variety of cellular processes including RNA metab., epigenetic regulation and DNA repair pathways.  Herein, we discuss these recent advances, focusing on the role of PRMTs in DNA damage signaling and its importance for maintaining genomic stability.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGryDaGBPsGY7rVg90H21EOLACvtfcHk0lhfmmIX2uKJEQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXnvVGksbg%253D&md5=9b45a150e051f5687df109c8011e286e</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1016%2Fj.dnarep.2013.04.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.dnarep.2013.04.006%26sid%3Dliteratum%253Aachs%26aulast%3DAuclair%26aufirst%3DY.%26aulast%3DRichard%26aufirst%3DS.%26atitle%3DThe%2520role%2520of%2520arginine%2520methylation%2520in%2520the%2520DNA%2520damage%2520response%26jtitle%3DDNA%2520Repair%26date%3D2013%26volume%3D12%26spage%3D459%26epage%3D465%26doi%3D10.1016%2Fj.dnarep.2013.04.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Blanc, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richard, S.</span></span> <span> </span><span class="NLM_article-title">Arginine Methylation: The coming of age</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>65</i></span>,  <span class="NLM_fpage">8</span>– <span class="NLM_lpage">24</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2016.11.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=10.1016%2Fj.molcel.2016.11.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=28061334" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=1%3ACAS%3A528%3ADC%252BC2sXmsVKmtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=65&publication_year=2017&pages=8-24&author=R.+S.+Blancauthor=S.+Richard&title=Arginine+Methylation%3A+The+coming+of+age&doi=10.1016%2Fj.molcel.2016.11.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Arginine Methylation: The Coming of Age</span></div><div class="casAuthors">Blanc, Romeo S.; Richard, Stephane</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">65</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">8-24</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Arginine methylation is a common post-translational modification functioning as an epigenetic regulator of transcription and playing key roles in pre-mRNA splicing, DNA damage signaling, mRNA translation, cell signaling, and cell fate decision.  Recently, a wealth of studies using transgenic mouse models and selective PRMT inhibitors helped define physiol. roles for protein arginine methyltransferases (PRMTs) linking them to diseases such as cancer and metabolic, neurodegenerative, and muscular disorders.  This review describes the recent mol. advances that have been uncovered in normal and diseased mammalian cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqT2OQsTa2UnbVg90H21EOLACvtfcHk0liL50zR2rIQdQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXmsVKmtA%253D%253D&md5=7ca3a6b7f0914c0034534ab8ca908506</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2016.11.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2016.11.003%26sid%3Dliteratum%253Aachs%26aulast%3DBlanc%26aufirst%3DR.%2BS.%26aulast%3DRichard%26aufirst%3DS.%26atitle%3DArginine%2520Methylation%253A%2520The%2520coming%2520of%2520age%26jtitle%3DMol.%2520Cell%26date%3D2017%26volume%3D65%26spage%3D8%26epage%3D24%26doi%3D10.1016%2Fj.molcel.2016.11.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wesche, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuhn, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kessler, B. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salton, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wolf, A.</span></span> <span> </span><span class="NLM_article-title">Protein arginine methylation: a prominent modification and its demethylation</span>. <i>Cell. Mol. Life Sci.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>74</i></span>,  <span class="NLM_fpage">3305</span>– <span class="NLM_lpage">3315</span>, <span class="refDoi"> DOI: 10.1007/s00018-017-2515-z</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=10.1007%2Fs00018-017-2515-z" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=28364192" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=1%3ACAS%3A528%3ADC%252BC2sXlsV2nu7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=74&publication_year=2017&pages=3305-3315&author=J.+Wescheauthor=S.+Kuhnauthor=B.+M.+Kesslerauthor=M.+Saltonauthor=A.+Wolf&title=Protein+arginine+methylation%3A+a+prominent+modification+and+its+demethylation&doi=10.1007%2Fs00018-017-2515-z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Protein arginine methylation: a prominent modification and its demethylation</span></div><div class="casAuthors">Wesche, Juste; Kuehn, Sarah; Kessler, Benedikt M.; Salton, Maayan; Wolf, Alexander</div><div class="citationInfo"><span class="NLM_cas:title">Cellular and Molecular Life Sciences</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">74</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">3305-3315</span>CODEN:
                <span class="NLM_cas:coden">CMLSFI</span>;
        ISSN:<span class="NLM_cas:issn">1420-682X</span>.
    
            (<span class="NLM_cas:orgname">Birkhaeuser Basel</span>)
        </div><div class="casAbstract">Arginine methylation of histones is one mechanism of epigenetic regulation in eukaryotic cells.  Methylarginines can also be found in non-histone proteins involved in various different processes in a cell.  An enzyme family of nine protein arginine methyltransferases catalyzes the addn. of Me groups on arginines of histone and non-histone proteins, resulting in either mono- or dimethylated-arginine residues.  The reversibility of histone modifications is an essential feature of epigenetic regulation to respond to changes in environmental factors, signalling events, or metabolic alterations.  Prominent histone modifications like lysine acetylation and lysine methylation are reversible.  Enzyme family pairs have been identified, with each pair of lysine acetyltransferases/deacetylases and lysine methyltransferases/demethylases operating complementarily to generate or erase lysine modifications.  Several analyses also indicate a reversible nature of arginine methylation, but the enzymes facilitating direct removal of Me moieties from arginine residues in proteins have been discussed controversially.  Differing reports have been seen for initially characterized putative candidates, like peptidyl arginine deiminase 4 or Jumonji-domain contg. protein 6.  Here, we review the most recent cellular, biochem., and mass spectrometry work on arginine methylation and its reversible nature with a special focus on putative arginine demethylases, including the enzyme superfamily of Fe(II) and 2-oxoglutarate-dependent oxygenases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqRNzVOWOcEu7Vg90H21EOLACvtfcHk0liL50zR2rIQdQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXlsV2nu7s%253D&md5=bf37e8cf8a26c27e86fd48592c084772</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1007%2Fs00018-017-2515-z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00018-017-2515-z%26sid%3Dliteratum%253Aachs%26aulast%3DWesche%26aufirst%3DJ.%26aulast%3DKuhn%26aufirst%3DS.%26aulast%3DKessler%26aufirst%3DB.%2BM.%26aulast%3DSalton%26aufirst%3DM.%26aulast%3DWolf%26aufirst%3DA.%26atitle%3DProtein%2520arginine%2520methylation%253A%2520a%2520prominent%2520modification%2520and%2520its%2520demethylation%26jtitle%3DCell.%2520Mol.%2520Life%2520Sci.%26date%3D2017%26volume%3D74%26spage%3D3305%26epage%3D3315%26doi%3D10.1007%2Fs00018-017-2515-z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jahan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davie, J. R.</span></span> <span> </span><span class="NLM_article-title">Protein arginine methyltransferases (PRMTs): role in chromatin organization</span>. <i>Adv. Biol. Regul.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">173</span>– <span class="NLM_lpage">184</span>, <span class="refDoi"> DOI: 10.1016/j.jbior.2014.09.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=10.1016%2Fj.jbior.2014.09.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=25263650" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsF2lsrrN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2015&pages=173-184&author=S.+Jahanauthor=J.+R.+Davie&title=Protein+arginine+methyltransferases+%28PRMTs%29%3A+role+in+chromatin+organization&doi=10.1016%2Fj.jbior.2014.09.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Protein arginine methyltransferases (PRMTs): Role in chromatin organization</span></div><div class="casAuthors">Jahan, Sanzida; Davie, James R.</div><div class="citationInfo"><span class="NLM_cas:title">Advances in Biological Regulation</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">173-184</span>CODEN:
                <span class="NLM_cas:coden">ABRDE5</span>;
        ISSN:<span class="NLM_cas:issn">2212-4934</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  The mammalian genome encodes eleven protein arginine methyltransferases (PRMTs) that are involved in the transfer of a Me group from S-adenosylmethionine (AdoMet) to the guanidino nitrogen of arginine.  The substrates for these enzymes range from histones to several nuclear and cytoplasmic proteins.  Methylation of histones by PRMTs can block the docking site for other reader/effector mols. and thus this modification can interfere with histone code orchestration.  Several members of the PRMTs have roles in chromatin organization and function.  Although PRMT aberrant expression is correlated with several diseases including cancer, the underlying mechanisms are still obscure in most cases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq3AjcsRV_REbVg90H21EOLACvtfcHk0liP3OmUAlBFZQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsF2lsrrN&md5=4758e1ffecb87e8f4ac6275eb3487971</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1016%2Fj.jbior.2014.09.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jbior.2014.09.003%26sid%3Dliteratum%253Aachs%26aulast%3DJahan%26aufirst%3DS.%26aulast%3DDavie%26aufirst%3DJ.%2BR.%26atitle%3DProtein%2520arginine%2520methyltransferases%2520%2528PRMTs%2529%253A%2520role%2520in%2520chromatin%2520organization%26jtitle%3DAdv.%2520Biol.%2520Regul.%26date%3D2015%26volume%3D57%26spage%3D173%26epage%3D184%26doi%3D10.1016%2Fj.jbior.2014.09.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mowen, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">David, M.</span></span> <span> </span><span class="NLM_article-title">Unconventional post-translational modifications in immunological signaling</span>. <i>Nat. Immunol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">512</span>– <span class="NLM_lpage">520</span>, <span class="refDoi"> DOI: 10.1038/ni.2873</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=10.1038%2Fni.2873" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=24840982" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=1%3ACAS%3A528%3ADC%252BC2cXot1ymsbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2014&pages=512-520&author=K.+A.+Mowenauthor=M.+David&title=Unconventional+post-translational+modifications+in+immunological+signaling&doi=10.1038%2Fni.2873"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Unconventional post-translational modifications in immunological signaling</span></div><div class="casAuthors">Mowen, Kerri A.; David, Michael</div><div class="citationInfo"><span class="NLM_cas:title">Nature Immunology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">512-520</span>CODEN:
                <span class="NLM_cas:coden">NIAMCZ</span>;
        ISSN:<span class="NLM_cas:issn">1529-2908</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The activity of a cell is governed by the signals it receives from the extracellular milieu, which are 'translated' into the appropriate biol. output, such as activation, survival, proliferation, migration or differentiation.  Signaling pathways are responsible for converting environmental cues into discrete intracellular events.  The alteration of existing proteins by post-translational modification (PTM) is a key feature of signal-transduction pathways that allows the modulation of protein function.  Research into PTMs has long been dominated by the investigation of protein phosphorylation; other PTMs, such as methylation of lysine and arginine residues, acetylation, and nitrosylation of thiol groups and tyrosine residues, have received comparatively little attention.  This Review aims to present an overview of these PTMs, with an emphasis on their role in cells of the immune system.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr3jHiQm3Mb-7Vg90H21EOLACvtfcHk0liP3OmUAlBFZQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXot1ymsbs%253D&md5=947c593778f88707f7fb604459def4e4</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1038%2Fni.2873&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fni.2873%26sid%3Dliteratum%253Aachs%26aulast%3DMowen%26aufirst%3DK.%2BA.%26aulast%3DDavid%26aufirst%3DM.%26atitle%3DUnconventional%2520post-translational%2520modifications%2520in%2520immunological%2520signaling%26jtitle%3DNat.%2520Immunol.%26date%3D2014%26volume%3D15%26spage%3D512%26epage%3D520%26doi%3D10.1038%2Fni.2873" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wei, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mundade, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lange, K. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, T.</span></span> <span> </span><span class="NLM_article-title">Protein arginine methylation of non-histone proteins and its role in diseases</span>. <i>Cell Cycle</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">32</span>– <span class="NLM_lpage">41</span>, <span class="refDoi"> DOI: 10.4161/cc.27353</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=10.4161%2Fcc.27353" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=24296620" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=1%3ACAS%3A528%3ADC%252BC2cXltFeruro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2014&pages=32-41&author=H.+Weiauthor=R.+Mundadeauthor=K.+C.+Langeauthor=T.+Lu&title=Protein+arginine+methylation+of+non-histone+proteins+and+its+role+in+diseases&doi=10.4161%2Fcc.27353"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Protein arginine methylation of non-histone proteins and its role in diseases</span></div><div class="casAuthors">Wei, Han; Mundade, Rasika; Lange, Kevin C.; Lu, Tao</div><div class="citationInfo"><span class="NLM_cas:title">Cell Cycle</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">32-41</span>CODEN:
                <span class="NLM_cas:coden">CCEYAS</span>;
        ISSN:<span class="NLM_cas:issn">1538-4101</span>.
    
            (<span class="NLM_cas:orgname">Landes Bioscience</span>)
        </div><div class="casAbstract">A review.  Protein arginine methyltransferases (PRMTs) are a family of enzymes that can methylate arginine residues on histones and other proteins.  PRMTs play a crucial role in influencing various cellular functions, including cellular development and tumorigenesis.  Arginine methylation by PRMTs is found on both nuclear and cytoplasmic proteins.  Recently, there is increasing evidence regarding post-translational modifications of non-histone proteins by PRMTs, illustrating the previously unknown importance of PRMTs in the regulation of various cellular functions by post-translational modifications.  In this review, we present the recent developments in the regulation of non-histone proteins by PRMTs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrIW3UyE1q0IbVg90H21EOLACvtfcHk0liP3OmUAlBFZQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXltFeruro%253D&md5=55f3ea91e0b85ccf07b66eed1c58e471</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.4161%2Fcc.27353&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4161%252Fcc.27353%26sid%3Dliteratum%253Aachs%26aulast%3DWei%26aufirst%3DH.%26aulast%3DMundade%26aufirst%3DR.%26aulast%3DLange%26aufirst%3DK.%2BC.%26aulast%3DLu%26aufirst%3DT.%26atitle%3DProtein%2520arginine%2520methylation%2520of%2520non-histone%2520proteins%2520and%2520its%2520role%2520in%2520diseases%26jtitle%3DCell%2520Cycle%26date%3D2014%26volume%3D13%26spage%3D32%26epage%3D41%26doi%3D10.4161%2Fcc.27353" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Blackwell, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ceman, S.</span></span> <span> </span><span class="NLM_article-title">Arginine methylation of RNA-binding proteins regulates cell function and differentiation</span>. <i>Mol. Reprod. Dev.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>79</i></span>,  <span class="NLM_fpage">163</span>– <span class="NLM_lpage">175</span>, <span class="refDoi"> DOI: 10.1002/mrd.22024</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=10.1002%2Fmrd.22024" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=79&publication_year=2012&pages=163-175&author=E.+Blackwellauthor=S.+Ceman&title=Arginine+methylation+of+RNA-binding+proteins+regulates+cell+function+and+differentiation&doi=10.1002%2Fmrd.22024"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1002%2Fmrd.22024&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fmrd.22024%26sid%3Dliteratum%253Aachs%26aulast%3DBlackwell%26aufirst%3DE.%26aulast%3DCeman%26aufirst%3DS.%26atitle%3DArginine%2520methylation%2520of%2520RNA-binding%2520proteins%2520regulates%2520cell%2520function%2520and%2520differentiation%26jtitle%3DMol.%2520Reprod.%2520Dev.%26date%3D2012%26volume%3D79%26spage%3D163%26epage%3D175%26doi%3D10.1002%2Fmrd.22024" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zurita-Lopez, C. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sandberg, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kelly, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clarke, S. G.</span></span> <span> </span><span class="NLM_article-title">Human protein arginine methyltransferase 7 (PRMT7) is a type III enzyme forming omega-NG-monomethylated arginine residues</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>287</i></span>,  <span class="NLM_fpage">7859</span>– <span class="NLM_lpage">7870</span>, <span class="refDoi"> DOI: 10.1074/jbc.M111.336271</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=10.1074%2Fjbc.M111.336271" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=287&publication_year=2012&pages=7859-7870&author=C.+I.+Zurita-Lopezauthor=T.+Sandbergauthor=R.+Kellyauthor=S.+G.+Clarke&title=Human+protein+arginine+methyltransferase+7+%28PRMT7%29+is+a+type+III+enzyme+forming+omega-NG-monomethylated+arginine+residues&doi=10.1074%2Fjbc.M111.336271"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M111.336271&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M111.336271%26sid%3Dliteratum%253Aachs%26aulast%3DZurita-Lopez%26aufirst%3DC.%2BI.%26aulast%3DSandberg%26aufirst%3DT.%26aulast%3DKelly%26aufirst%3DR.%26aulast%3DClarke%26aufirst%3DS.%2BG.%26atitle%3DHuman%2520protein%2520arginine%2520methyltransferase%25207%2520%2528PRMT7%2529%2520is%2520a%2520type%2520III%2520enzyme%2520forming%2520omega-NG-monomethylated%2520arginine%2520residues%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2012%26volume%3D287%26spage%3D7859%26epage%3D7870%26doi%3D10.1074%2Fjbc.M111.336271" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hadjikyriacou, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Y. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Espejo, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bedford, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clarke, S. G.</span></span> <span> </span><span class="NLM_article-title">Unique features of human protein arginine methyltransferase 9 (PRMT9) and its substrate RNA splicing factor SF3B2</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>290</i></span>,  <span class="NLM_fpage">16723</span>– <span class="NLM_lpage">16743</span>, <span class="refDoi"> DOI: 10.1074/jbc.M115.659433</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=10.1074%2Fjbc.M115.659433" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=290&publication_year=2015&pages=16723-16743&author=A.+Hadjikyriacouauthor=Y.+Z.+Yangauthor=A.+Espejoauthor=M.+T.+Bedfordauthor=S.+G.+Clarke&title=Unique+features+of+human+protein+arginine+methyltransferase+9+%28PRMT9%29+and+its+substrate+RNA+splicing+factor+SF3B2&doi=10.1074%2Fjbc.M115.659433"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M115.659433&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M115.659433%26sid%3Dliteratum%253Aachs%26aulast%3DHadjikyriacou%26aufirst%3DA.%26aulast%3DYang%26aufirst%3DY.%2BZ.%26aulast%3DEspejo%26aufirst%3DA.%26aulast%3DBedford%26aufirst%3DM.%2BT.%26aulast%3DClarke%26aufirst%3DS.%2BG.%26atitle%3DUnique%2520features%2520of%2520human%2520protein%2520arginine%2520methyltransferase%25209%2520%2528PRMT9%2529%2520and%2520its%2520substrate%2520RNA%2520splicing%2520factor%2520SF3B2%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2015%26volume%3D290%26spage%3D16723%26epage%3D16743%26doi%3D10.1074%2Fjbc.M115.659433" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bedford, M. T.</span></span> <span> </span><span class="NLM_article-title">Arginine methylation at a glance</span>. <i>J. Cell Sci.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>120</i></span>,  <span class="NLM_fpage">4243</span>– <span class="NLM_lpage">4246</span>, <span class="refDoi"> DOI: 10.1242/jcs.019885</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=10.1242%2Fjcs.019885" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=18057026" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=1%3ACAS%3A528%3ADC%252BD1cXns1GktA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=120&publication_year=2007&pages=4243-4246&author=M.+T.+Bedford&title=Arginine+methylation+at+a+glance&doi=10.1242%2Fjcs.019885"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Arginine methylation at a glance</span></div><div class="casAuthors">Bedford, Mark T.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Cell Science</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">120</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">4243-4246</span>CODEN:
                <span class="NLM_cas:coden">JNCSAI</span>;
        ISSN:<span class="NLM_cas:issn">0021-9533</span>.
    
            (<span class="NLM_cas:orgname">Company of Biologists Ltd.</span>)
        </div><div class="casAbstract">A review on the enzymes involved in methylation of arginine is presented.  Many different isoenzymes of the PRMT (protein arginine N Me transferase) from PRMT1 to PRMT9 are discussed.  Some of the substrates of the PRMT enzymes are also discussed along with a brief discussion of the role of protein protein interactions in regulation of arginine methylation is also given.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpVWHJj0lIjM7Vg90H21EOLACvtfcHk0lil1H23ZgAQmg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXns1GktA%253D%253D&md5=d6adb4f9193c22342c1d30f16a468590</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1242%2Fjcs.019885&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1242%252Fjcs.019885%26sid%3Dliteratum%253Aachs%26aulast%3DBedford%26aufirst%3DM.%2BT.%26atitle%3DArginine%2520methylation%2520at%2520a%2520glance%26jtitle%3DJ.%2520Cell%2520Sci.%26date%3D2007%26volume%3D120%26spage%3D4243%26epage%3D4246%26doi%3D10.1242%2Fjcs.019885" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kurochkina, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guha, U.</span></span> <span> </span><span class="NLM_article-title">SH3 domains: modules of protein-protein interactions</span>. <i>Biophys. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">29</span>– <span class="NLM_lpage">39</span>, <span class="refDoi"> DOI: 10.1007/s12551-012-0081-z</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=10.1007%2Fs12551-012-0081-z" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=28510178" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=1%3ACAS%3A528%3ADC%252BC3sXitlCktrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2013&pages=29-39&author=N.+Kurochkinaauthor=U.+Guha&title=SH3+domains%3A+modules+of+protein-protein+interactions&doi=10.1007%2Fs12551-012-0081-z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">SH3 domains: modules of protein-protein interactions</span></div><div class="casAuthors">Kurochkina, Natalya; Guha, Udayan</div><div class="citationInfo"><span class="NLM_cas:title">Biophysical Reviews</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">29-39</span>CODEN:
                <span class="NLM_cas:coden">BRIECG</span>;
        ISSN:<span class="NLM_cas:issn">1867-2450</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">A review.  Src homol. 3 (SH3) domains are involved in the regulation of important cellular pathways, such as cell proliferation, migration and cytoskeletal modifications.  Recognition of polyproline and a no. of noncanonical sequences by SH3 domains has been extensively studied by crystallog., NMR and other methods.  High-affinity peptides that bind SH3 domains are used in drug development as candidates for anticancer treatment.  This review summarizes the latest achievements in deciphering structural determinants of SH3 function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpOb5bt5bhZl7Vg90H21EOLACvtfcHk0lil1H23ZgAQmg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXitlCktrY%253D&md5=f9d4c19511a2c08456cdfd1ea8ce7ad1</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1007%2Fs12551-012-0081-z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs12551-012-0081-z%26sid%3Dliteratum%253Aachs%26aulast%3DKurochkina%26aufirst%3DN.%26aulast%3DGuha%26aufirst%3DU.%26atitle%3DSH3%2520domains%253A%2520modules%2520of%2520protein-protein%2520interactions%26jtitle%3DBiophys.%2520Rev.%26date%3D2013%26volume%3D5%26spage%3D29%26epage%3D39%26doi%3D10.1007%2Fs12551-012-0081-z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Antonysamy, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bonday, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campbell, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doyle, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Druzina, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gheyi, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jungheim, L. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qian, Y. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rauch, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Russell, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sauder, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wasserman, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weichert, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Willard, F. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Emtage, S.</span></span> <span> </span><span class="NLM_article-title">Crystal structure of the human PRMT5:MEP50 complex</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>109</i></span>,  <span class="NLM_fpage">17960</span>– <span class="NLM_lpage">17965</span>, <span class="refDoi"> DOI: 10.1073/pnas.1209814109</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=10.1073%2Fpnas.1209814109" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=23071334" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=1%3ACAS%3A528%3ADC%252BC38Xhsl2ktbvL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=109&publication_year=2012&pages=17960-17965&author=S.+Antonysamyauthor=Z.+Bondayauthor=R.+M.+Campbellauthor=B.+Doyleauthor=Z.+Druzinaauthor=T.+Gheyiauthor=B.+Hanauthor=L.+N.+Jungheimauthor=Y.+W.+Qianauthor=C.+Rauchauthor=M.+Russellauthor=J.+M.+Sauderauthor=S.+R.+Wassermanauthor=K.+Weichertauthor=F.+S.+Willardauthor=A.+P.+Zhangauthor=S.+Emtage&title=Crystal+structure+of+the+human+PRMT5%3AMEP50+complex&doi=10.1073%2Fpnas.1209814109"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal structure of the human PRMT5:MEP50 complex</span></div><div class="casAuthors">Antonysamy, Stephen; Bonday, Zahid; Campbell, Robert M.; Doyle, Brandon; Druzina, Zhanna; Gheyi, Tarun; Han, Bomie; Jungheim, Louis N.; Qian, Yuewei; Rauch, Charles; Russell, Marijane; Sauder, J. Michael; Wasserman, Stephen R.; Wichert, Kenneth; Willard, Francis S.; Zhang, Aiping; Emtage, Spencer</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">109</span>
        (<span class="NLM_cas:issue">44</span>),
    <span class="NLM_cas:pages">17960-17965, S17960/1-S17960/9</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Protein arginine methyltransferases (PRMTs) play important roles in several cellular processes, including signaling, gene regulation, and transport of proteins and nucleic acids, to impact growth, differentiation, proliferation, and development.  PRMT5 sym. di-methylates the two-terminal ω-guanidino nitrogens of arginine residues on substrate proteins.  PRMT5 acts as part of a multimeric complex in concert with a variety of partner proteins that regulate its function and specificity.  A core component of these complexes is the WD40 protein MEP50/WDR77/p44, which mediates interactions with binding partners and substrates.  We have detd. the crystal structure of human PRMT5 in complex with MEP50 (methylosome protein 50), bound to an S-adenosylmethionine analog and a peptide substrate derived from histone H4.  The structure of the surprising hetero-octameric complex reveals the close interaction between the seven-bladed β-propeller MEP50 and the N-terminal domain of PRMT5, and delineates the structural elements of substrate recognition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpkcWBh5CVI7LVg90H21EOLACvtfcHk0ljfUm8hANhOjA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xhsl2ktbvL&md5=757af18cfd0787a02c88c840273109aa</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1209814109&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1209814109%26sid%3Dliteratum%253Aachs%26aulast%3DAntonysamy%26aufirst%3DS.%26aulast%3DBonday%26aufirst%3DZ.%26aulast%3DCampbell%26aufirst%3DR.%2BM.%26aulast%3DDoyle%26aufirst%3DB.%26aulast%3DDruzina%26aufirst%3DZ.%26aulast%3DGheyi%26aufirst%3DT.%26aulast%3DHan%26aufirst%3DB.%26aulast%3DJungheim%26aufirst%3DL.%2BN.%26aulast%3DQian%26aufirst%3DY.%2BW.%26aulast%3DRauch%26aufirst%3DC.%26aulast%3DRussell%26aufirst%3DM.%26aulast%3DSauder%26aufirst%3DJ.%2BM.%26aulast%3DWasserman%26aufirst%3DS.%2BR.%26aulast%3DWeichert%26aufirst%3DK.%26aulast%3DWillard%26aufirst%3DF.%2BS.%26aulast%3DZhang%26aufirst%3DA.%2BP.%26aulast%3DEmtage%26aufirst%3DS.%26atitle%3DCrystal%2520structure%2520of%2520the%2520human%2520PRMT5%253AMEP50%2520complex%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2012%26volume%3D109%26spage%3D17960%26epage%3D17965%26doi%3D10.1073%2Fpnas.1209814109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zeytuni, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zarivach, R.</span></span> <span> </span><span class="NLM_article-title">Structural and functional discussion of the tetra-trico-peptide repeat, a protein interaction module</span>. <i>Structure</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">397</span>– <span class="NLM_lpage">405</span>, <span class="refDoi"> DOI: 10.1016/j.str.2012.01.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=10.1016%2Fj.str.2012.01.006" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2012&pages=397-405&author=N.+Zeytuniauthor=R.+Zarivach&title=Structural+and+functional+discussion+of+the+tetra-trico-peptide+repeat%2C+a+protein+interaction+module&doi=10.1016%2Fj.str.2012.01.006"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1016%2Fj.str.2012.01.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.str.2012.01.006%26sid%3Dliteratum%253Aachs%26aulast%3DZeytuni%26aufirst%3DN.%26aulast%3DZarivach%26aufirst%3DR.%26atitle%3DStructural%2520and%2520functional%2520discussion%2520of%2520the%2520tetra-trico-peptide%2520repeat%252C%2520a%2520protein%2520interaction%2520module%26jtitle%3DStructure%26date%3D2012%26volume%3D20%26spage%3D397%26epage%3D405%26doi%3D10.1016%2Fj.str.2012.01.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kim, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yadav, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carter, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richard, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richie, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bedford, M. T.</span></span> <span> </span><span class="NLM_article-title">Loss of CARM1 results in hypomethylation of thymocyte cyclic AMP-regulated phosphoprotein and deregulated early T cell development</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>279</i></span>,  <span class="NLM_fpage">25339</span>– <span class="NLM_lpage">25344</span>, <span class="refDoi"> DOI: 10.1074/jbc.M402544200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=10.1074%2Fjbc.M402544200" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=279&publication_year=2004&pages=25339-25344&author=J.+Kimauthor=J.+Leeauthor=N.+Yadavauthor=Q.+Wuauthor=C.+Carterauthor=S.+Richardauthor=E.+Richieauthor=M.+T.+Bedford&title=Loss+of+CARM1+results+in+hypomethylation+of+thymocyte+cyclic+AMP-regulated+phosphoprotein+and+deregulated+early+T+cell+development&doi=10.1074%2Fjbc.M402544200"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M402544200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M402544200%26sid%3Dliteratum%253Aachs%26aulast%3DKim%26aufirst%3DJ.%26aulast%3DLee%26aufirst%3DJ.%26aulast%3DYadav%26aufirst%3DN.%26aulast%3DWu%26aufirst%3DQ.%26aulast%3DCarter%26aufirst%3DC.%26aulast%3DRichard%26aufirst%3DS.%26aulast%3DRichie%26aufirst%3DE.%26aulast%3DBedford%26aufirst%3DM.%2BT.%26atitle%3DLoss%2520of%2520CARM1%2520results%2520in%2520hypomethylation%2520of%2520thymocyte%2520cyclic%2520AMP-regulated%2520phosphoprotein%2520and%2520deregulated%2520early%2520T%2520cell%2520development%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2004%26volume%3D279%26spage%3D25339%26epage%3D25344%26doi%3D10.1074%2Fjbc.M402544200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pawlak, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banik-Maiti, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pietenpol, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruley, H. E.</span></span> <span> </span><span class="NLM_article-title">Protein arginine methyltransferase I: substrate specificity and role in hnRNP assembly</span>. <i>J. Cell. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>87</i></span>,  <span class="NLM_fpage">394</span>– <span class="NLM_lpage">407</span>, <span class="refDoi"> DOI: 10.1002/jcb.10307</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=10.1002%2Fjcb.10307" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=87&publication_year=2002&pages=394-407&author=M.+R.+Pawlakauthor=S.+Banik-Maitiauthor=J.+A.+Pietenpolauthor=H.+E.+Ruley&title=Protein+arginine+methyltransferase+I%3A+substrate+specificity+and+role+in+hnRNP+assembly&doi=10.1002%2Fjcb.10307"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1002%2Fjcb.10307&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcb.10307%26sid%3Dliteratum%253Aachs%26aulast%3DPawlak%26aufirst%3DM.%2BR.%26aulast%3DBanik-Maiti%26aufirst%3DS.%26aulast%3DPietenpol%26aufirst%3DJ.%2BA.%26aulast%3DRuley%26aufirst%3DH.%2BE.%26atitle%3DProtein%2520arginine%2520methyltransferase%2520I%253A%2520substrate%2520specificity%2520and%2520role%2520in%2520hnRNP%2520assembly%26jtitle%3DJ.%2520Cell.%2520Biochem.%26date%3D2002%26volume%3D87%26spage%3D394%26epage%3D407%26doi%3D10.1002%2Fjcb.10307" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sun, L. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lv, Z. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bao, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gong, W. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, R. M.</span></span> <span> </span><span class="NLM_article-title">Structural insights into protein arginine symmetric dimethylation by PRMT5</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>108</i></span>,  <span class="NLM_fpage">20538</span>– <span class="NLM_lpage">20543</span>, <span class="refDoi"> DOI: 10.1073/pnas.1106946108</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=10.1073%2Fpnas.1106946108" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=22143770" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=1%3ACAS%3A528%3ADC%252BC38Xkt1KltA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=108&publication_year=2011&pages=20538-20543&author=L.+T.+Sunauthor=M.+Z.+Wangauthor=Z.+Y.+Lvauthor=N.+Yangauthor=Y.+F.+Liuauthor=S.+L.+Baoauthor=W.+M.+Gongauthor=R.+M.+Xu&title=Structural+insights+into+protein+arginine+symmetric+dimethylation+by+PRMT5&doi=10.1073%2Fpnas.1106946108"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Structural insights into protein arginine symmetric dimethylation by PRMT5</span></div><div class="casAuthors">Sun, Litao; Wang, Mingzhu; Lv, Zongyang; Yang, Na; Liu, Yingfang; Bao, Shilai; Gong, Weimin; Xu, Rui-Ming</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">108</span>
        (<span class="NLM_cas:issue">51</span>),
    <span class="NLM_cas:pages">20538-20543, S20538/1-S20538/3</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Sym. and asym. dimethylation of arginine are isomeric protein posttranslational modifications with distinct biol. effects, evidenced by the methylation of arginine 3 of histone H4 (H4R3): sym. dimethylation of H4R3 leads to repression of gene expression, while asym. dimethylation of H4R3 is assocd. with gene activation.  The enzymes catalyzing these modifications share identifiable sequence similarities, but the relationship between their catalytic mechanisms is unknown.  Here we analyzed the structure of a prototypic sym. arginine dimethylase, PRMTS, and discovered that a conserved phenylalanine in the active site is crit. for specifying sym. addn. of Me groups.  Changing it to a methionine significantly elevates the overall methylase activity, but also converts PRMT5 to an enzyme that catalyzes both sym. and asym. dimethylation of arginine.  Our results demonstrate a common catalytic mechanism intrinsic to both sym. and asym. arginine dimethylases, and show that steric constrains in the active sites play an essential role in detg. the product specificity of arginine methylases.  This discovery-also implies a potentially regulatable outcome of arginine dimethylation that may provide versatile control of eukaryotic gene expression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqh0uTOIQ5b9LVg90H21EOLACvtfcHk0ljrqNr2qnNSuQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xkt1KltA%253D%253D&md5=3698c8892671220d78fd5491f2daef07</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1106946108&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1106946108%26sid%3Dliteratum%253Aachs%26aulast%3DSun%26aufirst%3DL.%2BT.%26aulast%3DWang%26aufirst%3DM.%2BZ.%26aulast%3DLv%26aufirst%3DZ.%2BY.%26aulast%3DYang%26aufirst%3DN.%26aulast%3DLiu%26aufirst%3DY.%2BF.%26aulast%3DBao%26aufirst%3DS.%2BL.%26aulast%3DGong%26aufirst%3DW.%2BM.%26aulast%3DXu%26aufirst%3DR.%2BM.%26atitle%3DStructural%2520insights%2520into%2520protein%2520arginine%2520symmetric%2520dimethylation%2520by%2520PRMT5%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2011%26volume%3D108%26spage%3D20538%26epage%3D20543%26doi%3D10.1073%2Fpnas.1106946108" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Branscombe, T. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frankel, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cook, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Z. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pestka, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clarke, S.</span></span> <span> </span><span class="NLM_article-title">PRMT5 (Janus kinase-binding protein 1) catalyzes the formation of symmetric dimethylarginine residues in proteins</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>276</i></span>,  <span class="NLM_fpage">32971</span>– <span class="NLM_lpage">32976</span>, <span class="refDoi"> DOI: 10.1074/jbc.M105412200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=10.1074%2Fjbc.M105412200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=11413150" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=1%3ACAS%3A528%3ADC%252BD3MXmvFCmtbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=276&publication_year=2001&pages=32971-32976&author=T.+L.+Branscombeauthor=A.+Frankelauthor=J.+H.+Leeauthor=J.+R.+Cookauthor=Z.+H.+Yangauthor=S.+Pestkaauthor=S.+Clarke&title=PRMT5+%28Janus+kinase-binding+protein+1%29+catalyzes+the+formation+of+symmetric+dimethylarginine+residues+in+proteins&doi=10.1074%2Fjbc.M105412200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">PRMT5 (Janus kinase-binding protein 1) catalyzes the formation of symmetric dimethylarginine residues in proteins</span></div><div class="casAuthors">Branscombe, Tina L.; Frankel, Adam; Lee, Jin-Hyung; Cook, Jeffry R.; Yang, Zhi-Hong; Pestka, Sidney; Clarke, Steven</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">276</span>
        (<span class="NLM_cas:issue">35</span>),
    <span class="NLM_cas:pages">32971-32976</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">We have identified a new mammalian protein arginine N-methyltransferase, PRMT5, formerly designated Janus kinase-binding protein 1, that can catalyze the formation of ω-NG-monomethylarginine and sym. ω-NG,NG'-dimethylarginine in a variety of proteins.  A hemagglutinin peptide-tagged PRMT5 complex purified from human HeLa cells catalyzes the S-adenosyl-L-[methyl-3H]methionine-dependent in vitro methylation of myelin basic protein.  When the radiolabeled myelin basic protein was acid-hydrolyzed to free amino acids, and the products were sepd. by high-resoln. cation exchange chromatog., we were able to detect two tritiated species.  One species co-migrated with a ω-NG-monomethylarginine std., and the other co-chromatographed with a sym. ω-NG,NG'-dimethylarginine std.  Upon base treatment, this second species formed methylamine, a breakdown product characteristic of sym. ω-NG,NG'-dimethylarginine.  Further anal. of these two species by thin layer chromatog. confirmed their identification as ω-NG-monomethylarginine and sym. ω-NG,NG'-dimethylarginine.  The hemagglutinin-PRMT5 complex was also able to monomethylate and sym. dimethylate bovine histone H2A and a glutathione S-transferase-fibrillarin (amino acids 1-148) fusion protein (glutathione S-transferase-GAR).  A mutation introduced into the S-adenosyl-L-methionine-binding motif I of a myc-tagged PRMT5 construct in COS-1 cells led to a near complete loss of obsd. enzymic activity.  PRMT5 is the first example of a catalytic chain for a type II protein arginine N-methyltransferase that can result in the formation of sym. dimethylarginine residues as obsd. previously in myelin basic protein, Sm small nuclear ribonucleoproteins, and other polypeptides.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqZJzrYzZ_ze7Vg90H21EOLACvtfcHk0ljrqNr2qnNSuQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXmvFCmtbw%253D&md5=12e2a0db9d4747c4e7b06668b9fd4d2b</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M105412200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M105412200%26sid%3Dliteratum%253Aachs%26aulast%3DBranscombe%26aufirst%3DT.%2BL.%26aulast%3DFrankel%26aufirst%3DA.%26aulast%3DLee%26aufirst%3DJ.%2BH.%26aulast%3DCook%26aufirst%3DJ.%2BR.%26aulast%3DYang%26aufirst%3DZ.%2BH.%26aulast%3DPestka%26aufirst%3DS.%26aulast%3DClarke%26aufirst%3DS.%26atitle%3DPRMT5%2520%2528Janus%2520kinase-binding%2520protein%25201%2529%2520catalyzes%2520the%2520formation%2520of%2520symmetric%2520dimethylarginine%2520residues%2520in%2520proteins%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2001%26volume%3D276%26spage%3D32971%26epage%3D32976%26doi%3D10.1074%2Fjbc.M105412200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cook, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Z. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krause, C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herth, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoffmann, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pestka, S.</span></span> <span> </span><span class="NLM_article-title">FBXO11/PRMT9, a new protein arginine methyltransferase, symmetrically dimethylates arginine residues</span>. <i>Biochem. Biophys. Res. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>342</i></span>,  <span class="NLM_fpage">472</span>– <span class="NLM_lpage">481</span>, <span class="refDoi"> DOI: 10.1016/j.bbrc.2006.01.167</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=10.1016%2Fj.bbrc.2006.01.167" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=342&publication_year=2006&pages=472-481&author=J.+R.+Cookauthor=J.+H.+Leeauthor=Z.+H.+Yangauthor=C.+D.+Krauseauthor=N.+Herthauthor=R.+Hoffmannauthor=S.+Pestka&title=FBXO11%2FPRMT9%2C+a+new+protein+arginine+methyltransferase%2C+symmetrically+dimethylates+arginine+residues&doi=10.1016%2Fj.bbrc.2006.01.167"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1016%2Fj.bbrc.2006.01.167&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbrc.2006.01.167%26sid%3Dliteratum%253Aachs%26aulast%3DCook%26aufirst%3DJ.%2BR.%26aulast%3DLee%26aufirst%3DJ.%2BH.%26aulast%3DYang%26aufirst%3DZ.%2BH.%26aulast%3DKrause%26aufirst%3DC.%2BD.%26aulast%3DHerth%26aufirst%3DN.%26aulast%3DHoffmann%26aufirst%3DR.%26aulast%3DPestka%26aufirst%3DS.%26atitle%3DFBXO11%252FPRMT9%252C%2520a%2520new%2520protein%2520arginine%2520methyltransferase%252C%2520symmetrically%2520dimethylates%2520arginine%2520residues%26jtitle%3DBiochem.%2520Biophys.%2520Res.%2520Commun.%26date%3D2006%26volume%3D342%26spage%3D472%26epage%3D481%26doi%3D10.1016%2Fj.bbrc.2006.01.167" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Y. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hadjikyriacou, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xia, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gayatri, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zurita-Lopez, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kelly, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clarke, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bedford, M. T.</span></span> <span> </span><span class="NLM_article-title">PRMT9 is a Type II methyltransferase that methylates the splicing factor SAP145</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">6428</span>, <span class="refDoi"> DOI: 10.1038/ncomms7428</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=10.1038%2Fncomms7428" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=25737013" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtF2itrbJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=6428&author=Y.+Z.+Yangauthor=A.+Hadjikyriacouauthor=Z.+Xiaauthor=S.+Gayatriauthor=D.+Kimauthor=C.+Zurita-Lopezauthor=R.+Kellyauthor=A.+L.+Guoauthor=W.+Liauthor=S.+G.+Clarkeauthor=M.+T.+Bedford&title=PRMT9+is+a+Type+II+methyltransferase+that+methylates+the+splicing+factor+SAP145&doi=10.1038%2Fncomms7428"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">PRMT9 is a Type II methyltransferase that methylates the splicing factor SAP145</span></div><div class="casAuthors">Yang, Yanzhong; Hadjikyriacou, Andrea; Xia, Zheng; Gayatri, Sitaram; Kim, Daehoon; Zurita-Lopez, Cecilia; Kelly, Ryan; Guo, Ailan; Li, Wei; Clarke, Steven G.; Bedford, Mark T.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Communications</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">6428</span>CODEN:
                <span class="NLM_cas:coden">NCAOBW</span>;
        ISSN:<span class="NLM_cas:issn">2041-1723</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The human genome encodes a family of nine protein arginine methyltransferases (PRMT1-9), whose members can catalyze three distinct types of methylation on arginine residues.  Here we identify two spliceosome-assocd. proteins-SAP145 and SAP49-as PRMT9-binding partners, linking PRMT9 to U2 snRNP maturation.  We show that SAP145 is methylated by PRMT9 at arginine 508, which takes the form of monomethylated arginine (MMA) and sym. dimethylated arginine (SDMA).  PRMT9 thus joins PRMT5 as the only mammalian enzymes capable of depositing the SDMA mark.  Methylation of SAP145 on Arg 508 generates a binding site for the Tudor domain of the Survival of Motor Neuron (SMN) protein, and RNA-seq anal. reveals gross splicing changes when PRMT9 levels are attenuated.  These results identify PRMT9 as a nonhistone methyltransferase that primes the U2 snRNP for interaction with SMN.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoAQGZ5aTDM27Vg90H21EOLACvtfcHk0li3IvXQqH-TIQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtF2itrbJ&md5=bd2abcf479f24b3f8f4e7bf9908c92e7</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1038%2Fncomms7428&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms7428%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DY.%2BZ.%26aulast%3DHadjikyriacou%26aufirst%3DA.%26aulast%3DXia%26aufirst%3DZ.%26aulast%3DGayatri%26aufirst%3DS.%26aulast%3DKim%26aufirst%3DD.%26aulast%3DZurita-Lopez%26aufirst%3DC.%26aulast%3DKelly%26aufirst%3DR.%26aulast%3DGuo%26aufirst%3DA.%2BL.%26aulast%3DLi%26aufirst%3DW.%26aulast%3DClarke%26aufirst%3DS.%2BG.%26aulast%3DBedford%26aufirst%3DM.%2BT.%26atitle%3DPRMT9%2520is%2520a%2520Type%2520II%2520methyltransferase%2520that%2520methylates%2520the%2520splicing%2520factor%2520SAP145%26jtitle%3DNat.%2520Commun.%26date%3D2015%26volume%3D6%26spage%3D6428%26doi%3D10.1038%2Fncomms7428" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Obianyo, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Osborne, T. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, P. R.</span></span> <span> </span><span class="NLM_article-title">Kinetic mechanism of protein arginine methyltransferase 1</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">10420</span>– <span class="NLM_lpage">10427</span>, <span class="refDoi"> DOI: 10.1021/bi800904m</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi800904m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtVOmtrnO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2008&pages=10420-10427&author=O.+Obianyoauthor=T.+C.+Osborneauthor=P.+R.+Thompson&title=Kinetic+mechanism+of+protein+arginine+methyltransferase+1&doi=10.1021%2Fbi800904m"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Kinetic Mechanism of Protein Arginine Methyltransferase 1</span></div><div class="casAuthors">Obianyo, Obiamaka; Osborne, Tanesha C.; Thompson, Paul R.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">39</span>),
    <span class="NLM_cas:pages">10420-10427</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Protein arginine methyltransferases (PRMTs) are SAM-dependent enzymes that catalyze the mono- and dimethylation of peptidyl arginine residues.  Although all PRMTs produce monomethyl arginine (MMA), type 1 PRMTs go on to form asym. dimethylated arginine (ADMA), while type 2 enzymes form sym. dimethylated arginine (SDMA).  PRMT1 is the major type 1 PRMT in vivo, thus it is the primary producer of the competitive NOS inhibitor, ADMA.  Hence, potent inhibitors, which are highly selective for this particular isoenzyme, could serve as excellent therapeutics for heart disease.  However, the design of such inhibitors is impeded by a lack of information regarding this enzyme's kinetic and catalytic mechanisms.  Herein we report an anal. of the kinetic mechanism of human PRMT1 using both an unmethylated and a monomethylated substrate peptide based on the N-terminus of histone H4.  The results of initial velocity and product and dead-end inhibition expts. indicate that PRMT1 utilizes a rapid equil. random (RER) mechanism with the formation of dead-end EAP and EBQ complexes (where E represents the enzyme, A and B represent substrates, and P and Q represent products).  This mechanism is gratifyingly consistent with previous results demonstrating that PRMT1 catalyzes substrate dimethylation in a partially processive manner.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoptMu_FIU0TrVg90H21EOLACvtfcHk0lh5DNU5EvgTHQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtVOmtrnO&md5=fde187ecc31760e562934a00e2c6a68a</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1021%2Fbi800904m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi800904m%26sid%3Dliteratum%253Aachs%26aulast%3DObianyo%26aufirst%3DO.%26aulast%3DOsborne%26aufirst%3DT.%2BC.%26aulast%3DThompson%26aufirst%3DP.%2BR.%26atitle%3DKinetic%2520mechanism%2520of%2520protein%2520arginine%2520methyltransferase%25201%26jtitle%3DBiochemistry%26date%3D2008%26volume%3D47%26spage%3D10420%26epage%3D10427%26doi%3D10.1021%2Fbi800904m" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Osborne, T. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Obianyo, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, P. R.</span></span> <span> </span><span class="NLM_article-title">Protein arginine methyltransferase 1: Positively charged residues in substrate peptides distal to the site of methylation are important for substrate binding and catalysis</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>46</i></span>,  <span class="NLM_fpage">13370</span>– <span class="NLM_lpage">13381</span>, <span class="refDoi"> DOI: 10.1021/bi701558t</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi701558t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2007&pages=13370-13381&author=T.+C.+Osborneauthor=O.+Obianyoauthor=X.+Zhangauthor=X.+Chengauthor=P.+R.+Thompson&title=Protein+arginine+methyltransferase+1%3A+Positively+charged+residues+in+substrate+peptides+distal+to+the+site+of+methylation+are+important+for+substrate+binding+and+catalysis&doi=10.1021%2Fbi701558t"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Fbi701558t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi701558t%26sid%3Dliteratum%253Aachs%26aulast%3DOsborne%26aufirst%3DT.%2BC.%26aulast%3DObianyo%26aufirst%3DO.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DCheng%26aufirst%3DX.%26aulast%3DThompson%26aufirst%3DP.%2BR.%26atitle%3DProtein%2520arginine%2520methyltransferase%25201%253A%2520Positively%2520charged%2520residues%2520in%2520substrate%2520peptides%2520distal%2520to%2520the%2520site%2520of%2520methylation%2520are%2520important%2520for%2520substrate%2520binding%2520and%2520catalysis%26jtitle%3DBiochemistry%26date%3D2007%26volume%3D46%26spage%3D13370%26epage%3D13381%26doi%3D10.1021%2Fbi701558t" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tee, W. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pardo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Theunissen, T. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choudhary, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hajkova, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Surani, M. A.</span></span> <span> </span><span class="NLM_article-title">Prmt5 is essential for early mouse development and acts in the cytoplasm to maintain ES cell pluripotency</span>. <i>Genes Dev.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">2772</span>– <span class="NLM_lpage">2777</span>, <span class="refDoi"> DOI: 10.1101/gad.606110</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=10.1101%2Fgad.606110" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=21159818" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=1%3ACAS%3A528%3ADC%252BC3MXltVKh" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2010&pages=2772-2777&author=W.+W.+Teeauthor=M.+Pardoauthor=T.+W.+Theunissenauthor=L.+Yuauthor=J.+S.+Choudharyauthor=P.+Hajkovaauthor=M.+A.+Surani&title=Prmt5+is+essential+for+early+mouse+development+and+acts+in+the+cytoplasm+to+maintain+ES+cell+pluripotency&doi=10.1101%2Fgad.606110"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Prmt5 is essential for early mouse development and acts in the cytoplasm to maintain ES cell pluripotency</span></div><div class="casAuthors">Tee, Wee-Wei; Pardo, Mercedes; Theunissen, Thorold W.; Yu, Lu; Choudhary, Jyoti S.; Hajkova, Petra; Surani, M. Azim</div><div class="citationInfo"><span class="NLM_cas:title">Genes & Development</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">2772-2777</span>CODEN:
                <span class="NLM_cas:coden">GEDEEP</span>;
        ISSN:<span class="NLM_cas:issn">0890-9369</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">Prmt5, an arginine methyltransferase, has multiple roles in germ cells, and possibly in pluripotency.  Here we show that loss of Prmt5 function is early embryonic-lethal due to the abrogation of pluripotent cells in blastocysts.  Prmt5 is also up-regulated in the cytoplasm during the derivation of embryonic stem (ES) cells together with Stat3, where they persist to maintain pluripotency.  Prmt5 in assocn. with Mep50 methylates cytosolic histone H2A (H2AR3me2s) to repress differentiation genes in ES cells.  Loss of Prmt5 or Mep50 results in derepression of differentiation genes, indicating the significance of the Prmt5/Mep50 complex for pluripotency, which may occur in conjunction with the leukemia inhibitory factor (LIF)/Stat3 pathway.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrOFBFfgqPpBbVg90H21EOLACvtfcHk0lh_uxBiB7nRbg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXltVKh&md5=f28985d9b27caa385244420809d11e08</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1101%2Fgad.606110&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fgad.606110%26sid%3Dliteratum%253Aachs%26aulast%3DTee%26aufirst%3DW.%2BW.%26aulast%3DPardo%26aufirst%3DM.%26aulast%3DTheunissen%26aufirst%3DT.%2BW.%26aulast%3DYu%26aufirst%3DL.%26aulast%3DChoudhary%26aufirst%3DJ.%2BS.%26aulast%3DHajkova%26aufirst%3DP.%26aulast%3DSurani%26aufirst%3DM.%2BA.%26atitle%3DPrmt5%2520is%2520essential%2520for%2520early%2520mouse%2520development%2520and%2520acts%2520in%2520the%2520cytoplasm%2520to%2520maintain%2520ES%2520cell%2520pluripotency%26jtitle%3DGenes%2520Dev.%26date%3D2010%26volume%3D24%26spage%3D2772%26epage%3D2777%26doi%3D10.1101%2Fgad.606110" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, G. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamard, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greenblatt, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Man, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perna, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tadi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luciani, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nimer, S. D.</span></span> <span> </span><span class="NLM_article-title">Arginine methyltransferase PRMT5 is essential for sustaining normal adult hematopoiesis</span>. <i>J. Clin. Invest.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>125</i></span>,  <span class="NLM_fpage">3532</span>– <span class="NLM_lpage">3544</span>, <span class="refDoi"> DOI: 10.1172/JCI81749</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=10.1172%2FJCI81749" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=26258414" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=1%3ACAS%3A280%3ADC%252BC287ht1eqtg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=125&publication_year=2015&pages=3532-3544&author=F.+Liuauthor=G.+Y.+Chengauthor=P.+J.+Hamardauthor=S.+Greenblattauthor=L.+Wangauthor=N.+Manauthor=F.+Pernaauthor=H.+M.+Xuauthor=M.+Tadiauthor=L.+Lucianiauthor=S.+D.+Nimer&title=Arginine+methyltransferase+PRMT5+is+essential+for+sustaining+normal+adult+hematopoiesis&doi=10.1172%2FJCI81749"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Arginine methyltransferase PRMT5 is essential for sustaining normal adult hematopoiesis</span></div><div class="casAuthors">Liu Fan; Cheng Guoyan; Hamard Pierre-Jacques; Greenblatt Sarah; Wang Lan; Man Na; Perna Fabiana; Xu Haiming; Tadi Madhavi; Luciani Luisa; Nimer Stephen D</div><div class="citationInfo"><span class="NLM_cas:title">The Journal of clinical investigation</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">125</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">3532-44</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Epigenetic regulators play critical roles in normal hematopoiesis, and the activity of these enzymes is frequently altered in hematopoietic cancers.  The major type II protein arginine methyltransferase PRMT5 catalyzes the formation of symmetric dimethyl arginine and has been implicated in various cellular processes, including pluripotency and tumorigenesis.  Here, we generated Prmt5 conditional KO mice to evaluate the contribution of PRMT5 to adult hematopoiesis.  Loss of PRMT5 triggered an initial but transient expansion of hematopoietic stem cells (HSCs); however, Prmt5 deletion resulted in a concurrent loss of hematopoietic progenitor cells (HPCs), leading to fatal BM aplasia.  PRMT5-specific effects on hematopoiesis were cell intrinsic and depended on PRMT5 methyltransferase activity.  We found that PRMT5-deficient hematopoietic stem and progenitor cells exhibited severely impaired cytokine signaling as well as upregulation of p53 and expression of its downstream targets.  Together, our results demonstrate that PRMT5 plays distinct roles in the behavior of HSCs compared with HPCs and is essential for the maintenance of adult hematopoietic cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTN1zt2v1VSmK-9TkgmBqTqfW6udTcc2ebJKhKe-7-3Jrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC287ht1eqtg%253D%253D&md5=1653ed334d91ed4328f98de8c92506a5</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1172%2FJCI81749&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1172%252FJCI81749%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DCheng%26aufirst%3DG.%2BY.%26aulast%3DHamard%26aufirst%3DP.%2BJ.%26aulast%3DGreenblatt%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DMan%26aufirst%3DN.%26aulast%3DPerna%26aufirst%3DF.%26aulast%3DXu%26aufirst%3DH.%2BM.%26aulast%3DTadi%26aufirst%3DM.%26aulast%3DLuciani%26aufirst%3DL.%26aulast%3DNimer%26aufirst%3DS.%2BD.%26atitle%3DArginine%2520methyltransferase%2520PRMT5%2520is%2520essential%2520for%2520sustaining%2520normal%2520adult%2520hematopoiesis%26jtitle%3DJ.%2520Clin.%2520Invest.%26date%3D2015%26volume%3D125%26spage%3D3532%26epage%3D3544%26doi%3D10.1172%2FJCI81749" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pal, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vishwanath, S. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Erdjument-Bromage, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tempst, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sif, S.</span></span> <span> </span><span class="NLM_article-title">Human SWI/SNF-associated PRMT5 methylates histone H3 arginine 8 and negatively regulates expression of ST7 and NM23 tumor suppressor genes</span>. <i>Mol. Cell. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">9630</span>– <span class="NLM_lpage">9645</span>, <span class="refDoi"> DOI: 10.1128/MCB.24.21.9630-9645.2004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=10.1128%2FMCB.24.21.9630-9645.2004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=15485929" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=1%3ACAS%3A528%3ADC%252BD2MXltl2htQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2004&pages=9630-9645&author=S.+Palauthor=S.+N.+Vishwanathauthor=H.+Erdjument-Bromageauthor=P.+Tempstauthor=S.+Sif&title=Human+SWI%2FSNF-associated+PRMT5+methylates+histone+H3+arginine+8+and+negatively+regulates+expression+of+ST7+and+NM23+tumor+suppressor+genes&doi=10.1128%2FMCB.24.21.9630-9645.2004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Human SWI/SNF-associated PRMT5 methylates histone H3 arginine 8 and negatively regulates expression of ST7 and NM23 tumor suppressor genes</span></div><div class="casAuthors">Pal, Sharmistha; Vishwanath, Sheethal N.; Erdjument-Bromage, Hediye; Tempst, Paul; Sif, Said</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">9630-9645</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">0270-7306</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Protein arginine methyltransferases (PRMTs) have been implicated in transcriptional activation and repression, but their role in controlling cell growth and proliferation remains obscure.  We have recently shown that PRMT5 can interact with flag-tagged BRG1- and hBRM-based hSWI/SNF chromatin remodelers and that both complexes can specifically methylate histones H3 and H4.  Here we report that PRMT5 can be found in assocn. with endogenous hSWI/SNF complexes, which can methylate H3 and H4 N-terminal tails, and show that H3 arginine 8 and H4 arginine 3 are preferred sites of methylation by recombinant and hSWI/SNF-assocd. PRMT5.  To elucidate the role played by PRMT5 in gene regulation, we have established a PRMT5 antisense cell line and detd. by microarray anal. that more genes are derepressed when PRMT5 levels are reduced.  Among the affected genes, we show that suppressor of tumorigenicity 7 (ST7) and nonmetastatic 23 (NM23) are direct targets of PRMT5-contg. BRG1 and hBRM complexes.  Furthermore, we demonstrate that expression of ST7 and NM23 is reduced in a cell line that overexpresses PRMT5 and that this decrease in expression correlates with H3R8 methylation, H3K9 deacetylation, and increased transformation of NIH 3T3 cells.  These findings suggest that the BRG1- and hBRM-assocd. PRMT5 regulates cell growth and proliferation by controlling expression of genes involved in tumor suppression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoHC4UpBSGWILVg90H21EOLACvtfcHk0ljmScH6zRJt1w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXltl2htQ%253D%253D&md5=663371d517c0e9e3b709483e320869f7</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1128%2FMCB.24.21.9630-9645.2004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.24.21.9630-9645.2004%26sid%3Dliteratum%253Aachs%26aulast%3DPal%26aufirst%3DS.%26aulast%3DVishwanath%26aufirst%3DS.%2BN.%26aulast%3DErdjument-Bromage%26aufirst%3DH.%26aulast%3DTempst%26aufirst%3DP.%26aulast%3DSif%26aufirst%3DS.%26atitle%3DHuman%2520SWI%252FSNF-associated%2520PRMT5%2520methylates%2520histone%2520H3%2520arginine%25208%2520and%2520negatively%2520regulates%2520expression%2520of%2520ST7%2520and%2520NM23%2520tumor%2520suppressor%2520genes%26jtitle%3DMol.%2520Cell.%2520Biol.%26date%3D2004%26volume%3D24%26spage%3D9630%26epage%3D9645%26doi%3D10.1128%2FMCB.24.21.9630-9645.2004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pesiridis, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diamond, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Duyne, G. D.</span></span> <span> </span><span class="NLM_article-title">Role of pICLn in methylation of Sm proteins by PRMT5</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>284</i></span>,  <span class="NLM_fpage">21347</span>– <span class="NLM_lpage">21359</span>, <span class="refDoi"> DOI: 10.1074/jbc.M109.015578</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=10.1074%2Fjbc.M109.015578" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=19520849" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=1%3ACAS%3A528%3ADC%252BD1MXpt1Wisr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=284&publication_year=2009&pages=21347-21359&author=G.+S.+Pesiridisauthor=E.+Diamondauthor=G.+D.+Van+Duyne&title=Role+of+pICLn+in+methylation+of+Sm+proteins+by+PRMT5&doi=10.1074%2Fjbc.M109.015578"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Role of pICLn in Methylation of Sm Proteins by PRMT5</span></div><div class="casAuthors">Pesiridis, G. Scott; Diamond, Evan; Van Duyne, Gregory D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">284</span>
        (<span class="NLM_cas:issue">32</span>),
    <span class="NLM_cas:pages">21347-21359</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">PICln is an essential, highly conserved 26-kDa protein whose functions include binding to Sm proteins in the cytoplasm of human cells and mediating the ordered and regulated assembly of the cell's RNA-splicing machinery by the survival motor neurons complex. pICln also interacts with PRMT5, the enzyme responsible for generating sym. dimethylarginine modifications on the carboxyl-terminal regions of three of the canonical Sm proteins.  To better understand the role of pICln in these cellular processes, we have investigated the properties of pICln and pICln·Sm complexes and the effects that pICln has on the methyltransferase activity of PRMT5.  We find that pICln is a monomer in soln., binds with high affinity (Kd ∼ 160 nM) to SmD3-SmB, and forms 1:1 complexes with Sm proteins and Sm protein subcomplexes.  The data support an end-capping model of pICln binding that supports current views of how pICln prevents Sm oligomerization on illicit RNA substrates.  We have found that by co-expression with pICln, recombinant PRMT5 can be produced in a sol., active form.  PRMT5 alone has promiscuous activity toward a variety of known substrates.  In the presence of pICln, however, PRMT5 methylation of Sm proteins is stimulated, but methylation of histones is inhibited.  We have also found that mutations in pICln that do not affect Sm protein binding can still have a profound effect on the methyltransferase activity of the PRMT5 complex.  Together, the data provide insights into pICln function and represent an important starting point for biochem. analyses of PRMT5.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoA8wzwfocai7Vg90H21EOLACvtfcHk0lh5VSulcwWc3Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXpt1Wisr4%253D&md5=a9d40f958ab3bbc2b835dcde792bf7b8</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M109.015578&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M109.015578%26sid%3Dliteratum%253Aachs%26aulast%3DPesiridis%26aufirst%3DG.%2BS.%26aulast%3DDiamond%26aufirst%3DE.%26aulast%3DVan%2BDuyne%26aufirst%3DG.%2BD.%26atitle%3DRole%2520of%2520pICLn%2520in%2520methylation%2520of%2520Sm%2520proteins%2520by%2520PRMT5%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2009%26volume%3D284%26spage%3D21347%26epage%3D21359%26doi%3D10.1074%2Fjbc.M109.015578" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Guderian, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peter, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wiesner, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sickmann, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schulze-Osthoff, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grimmler, M.</span></span> <span> </span><span class="NLM_article-title">RioK1, a new interactor of protein arginine methyltransferase 5 (PRMT5), competes with pICln for binding and modulates PRMT5 complex composition and substrate specificity</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>286</i></span>,  <span class="NLM_fpage">1976</span>– <span class="NLM_lpage">1986</span>, <span class="refDoi"> DOI: 10.1074/jbc.M110.148486</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=10.1074%2Fjbc.M110.148486" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=286&publication_year=2011&pages=1976-1986&author=G.+Guderianauthor=C.+Peterauthor=J.+Wiesnerauthor=A.+Sickmannauthor=K.+Schulze-Osthoffauthor=U.+Fischerauthor=M.+Grimmler&title=RioK1%2C+a+new+interactor+of+protein+arginine+methyltransferase+5+%28PRMT5%29%2C+competes+with+pICln+for+binding+and+modulates+PRMT5+complex+composition+and+substrate+specificity&doi=10.1074%2Fjbc.M110.148486"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M110.148486&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M110.148486%26sid%3Dliteratum%253Aachs%26aulast%3DGuderian%26aufirst%3DG.%26aulast%3DPeter%26aufirst%3DC.%26aulast%3DWiesner%26aufirst%3DJ.%26aulast%3DSickmann%26aufirst%3DA.%26aulast%3DSchulze-Osthoff%26aufirst%3DK.%26aulast%3DFischer%26aufirst%3DU.%26aulast%3DGrimmler%26aufirst%3DM.%26atitle%3DRioK1%252C%2520a%2520new%2520interactor%2520of%2520protein%2520arginine%2520methyltransferase%25205%2520%2528PRMT5%2529%252C%2520competes%2520with%2520pICln%2520for%2520binding%2520and%2520modulates%2520PRMT5%2520complex%2520composition%2520and%2520substrate%2520specificity%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2011%26volume%3D286%26spage%3D1976%26epage%3D1986%26doi%3D10.1074%2Fjbc.M110.148486" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Andreu-Perez, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Esteve-Puig, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Torre-Minguela, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lopez-Fauqued, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bech-Serra, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tenbaum, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garcia-Trevijano, E. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Canals, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Merlino, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Avila, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Recio, J. A.</span></span> <span> </span><span class="NLM_article-title">Protein arginine methyltransferase 5 regulates ERK1/2 signal transduction amplitude and cell fate through CRAF</span>. <i>Sci. Signaling</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">ra58</span>, <span class="refDoi"> DOI: 10.1126/scisignal.2001936</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=10.1126%2Fscisignal.2001936" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=21917714" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2011&pages=ra58&author=P.+Andreu-Perezauthor=R.+Esteve-Puigauthor=C.+de+Torre-Minguelaauthor=M.+Lopez-Fauquedauthor=J.+J.+Bech-Serraauthor=S.+Tenbaumauthor=E.+R.+Garcia-Trevijanoauthor=F.+Canalsauthor=G.+Merlinoauthor=M.+A.+Avilaauthor=J.+A.+Recio&title=Protein+arginine+methyltransferase+5+regulates+ERK1%2F2+signal+transduction+amplitude+and+cell+fate+through+CRAF&doi=10.1126%2Fscisignal.2001936"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1126%2Fscisignal.2001936&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscisignal.2001936%26sid%3Dliteratum%253Aachs%26aulast%3DAndreu-Perez%26aufirst%3DP.%26aulast%3DEsteve-Puig%26aufirst%3DR.%26aulast%3Dde%2BTorre-Minguela%26aufirst%3DC.%26aulast%3DLopez-Fauqued%26aufirst%3DM.%26aulast%3DBech-Serra%26aufirst%3DJ.%2BJ.%26aulast%3DTenbaum%26aufirst%3DS.%26aulast%3DGarcia-Trevijano%26aufirst%3DE.%2BR.%26aulast%3DCanals%26aufirst%3DF.%26aulast%3DMerlino%26aufirst%3DG.%26aulast%3DAvila%26aufirst%3DM.%2BA.%26aulast%3DRecio%26aufirst%3DJ.%2BA.%26atitle%3DProtein%2520arginine%2520methyltransferase%25205%2520regulates%2520ERK1%252F2%2520signal%2520transduction%2520amplitude%2520and%2520cell%2520fate%2520through%2520CRAF%26jtitle%3DSci.%2520Signaling%26date%3D2011%26volume%3D4%26spage%3Dra58%26doi%3D10.1126%2Fscisignal.2001936" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hsu, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, C. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chou, C. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuo, H. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, C. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, H. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liao, H. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bedford, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsai, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hung, M. C.</span></span> <span> </span><span class="NLM_article-title">Crosstalk between Arg 1175 methylation and Tyr 1173 phosphorylation negatively modulates EGFR-mediated ERK activation</span>. <i>Nat. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">174</span>– <span class="NLM_lpage">151</span>, <span class="refDoi"> DOI: 10.1038/ncb2158</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=10.1038%2Fncb2158" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=21258366" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtlamtL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2011&pages=174-151&author=J.+M.+Hsuauthor=C.+T.+Chenauthor=C.+K.+Chouauthor=H.+P.+Kuoauthor=L.+Y.+Liauthor=C.+Y.+Linauthor=H.+J.+Leeauthor=Y.+N.+Wangauthor=M.+Liuauthor=H.+W.+Liaoauthor=B.+Shiauthor=C.+C.+Laiauthor=M.+T.+Bedfordauthor=C.+H.+Tsaiauthor=M.+C.+Hung&title=Crosstalk+between+Arg+1175+methylation+and+Tyr+1173+phosphorylation+negatively+modulates+EGFR-mediated+ERK+activation&doi=10.1038%2Fncb2158"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Crosstalk between Arg 1175 methylation and Tyr 1173 phosphorylation negatively modulates EGFR-mediated ERK activation</span></div><div class="casAuthors">Hsu, Jung-Mao; Chen, Chun-Te; Chou, Chao-Kai; Kuo, Hsu-Ping; Li, Long-Yuan; Lin, Chun-Yi; Lee, Hong-Jen; Wang, Ying-Nai; Liu, Mo; Liao, Hsin-Wei; Shi, Bin; Lai, Chien-Chen; Bedford, Mark T.; Tsai, Chang-Hai; Hung, Mien-Chie</div><div class="citationInfo"><span class="NLM_cas:title">Nature Cell Biology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">174-181</span>CODEN:
                <span class="NLM_cas:coden">NCBIFN</span>;
        ISSN:<span class="NLM_cas:issn">1465-7392</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Epidermal growth factor receptor (EGFR) can undergo post-translational modifications, including phosphorylation, glycosylation and ubiquitylation, leading to diverse physiol. consequences and modulation of its biol. activity.  There is increasing evidence that methylation may parallel other post-translational modifications in the regulation of various biol. processes.  It is still not known, however, whether EGFR is regulated by this post-translational event.  EGFR Arg 1175 is methylated by an arginine methyltransferase, PRMT5.  Arg 1175 methylation pos. modulates EGF-induced EGFR trans-autophosphorylation at Tyr 1173, which governs ERK activation.  Abolishment of Arg 1175 methylation enhances EGF-stimulated ERK activation by reducing SHP1 recruitment to EGFR, resulting in augmented cell proliferation, migration and invasion of EGFR-expressing cells.  Therefore, the authors propose a model in which the regulatory crosstalk between PRMT5-mediated Arg 1175 methylation and EGF-induced Tyr 1173 phosphorylation attenuates EGFR-mediated ERK activation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo-rax4WtxKQrVg90H21EOLACvtfcHk0lg2i_SQwbkbRQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtlamtL0%253D&md5=ea4dd8ac92333deee879597c9a657060</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1038%2Fncb2158&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncb2158%26sid%3Dliteratum%253Aachs%26aulast%3DHsu%26aufirst%3DJ.%2BM.%26aulast%3DChen%26aufirst%3DC.%2BT.%26aulast%3DChou%26aufirst%3DC.%2BK.%26aulast%3DKuo%26aufirst%3DH.%2BP.%26aulast%3DLi%26aufirst%3DL.%2BY.%26aulast%3DLin%26aufirst%3DC.%2BY.%26aulast%3DLee%26aufirst%3DH.%2BJ.%26aulast%3DWang%26aufirst%3DY.%2BN.%26aulast%3DLiu%26aufirst%3DM.%26aulast%3DLiao%26aufirst%3DH.%2BW.%26aulast%3DShi%26aufirst%3DB.%26aulast%3DLai%26aufirst%3DC.%2BC.%26aulast%3DBedford%26aufirst%3DM.%2BT.%26aulast%3DTsai%26aufirst%3DC.%2BH.%26aulast%3DHung%26aufirst%3DM.%2BC.%26atitle%3DCrosstalk%2520between%2520Arg%25201175%2520methylation%2520and%2520Tyr%25201173%2520phosphorylation%2520negatively%2520modulates%2520EGFR-mediated%2520ERK%2520activation%26jtitle%3DNat.%2520Cell%2520Biol.%26date%3D2011%26volume%3D13%26spage%3D174%26epage%3D151%26doi%3D10.1038%2Fncb2158" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xu, X. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mayo, M. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bekiranov, S.</span></span> <span> </span><span class="NLM_article-title">Application of machine learning methods to histone methylation ChIP-Seq data reveals H4R3me2 globally represses gene expression</span>. <i>BMC Bioinf</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">396</span>, <span class="refDoi"> DOI: 10.1186/1471-2105-11-396</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=10.1186%2F1471-2105-11-396" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=20653935" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=1%3ACAS%3A280%3ADC%252BC3cjntl2kug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2010&pages=396&author=X.+J.+Xuauthor=S.+Hoangauthor=M.+W.+Mayoauthor=S.+Bekiranov&title=Application+of+machine+learning+methods+to+histone+methylation+ChIP-Seq+data+reveals+H4R3me2+globally+represses+gene+expression&doi=10.1186%2F1471-2105-11-396"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Application of machine learning methods to histone methylation ChIP-Seq data reveals H4R3me2 globally represses gene expression</span></div><div class="casAuthors">Xu Xiaojiang; Hoang Stephen; Mayo Marty W; Bekiranov Stefan</div><div class="citationInfo"><span class="NLM_cas:title">BMC bioinformatics</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">396</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  In the last decade, biochemical studies have revealed that epigenetic modifications including histone modifications, histone variants and DNA methylation form a complex network that regulate the state of chromatin and processes that depend on it including transcription and DNA replication.  Currently, a large number of these epigenetic modifications are being mapped in a variety of cell lines at different stages of development using high throughput sequencing by members of the ENCODE consortium, the NIH Roadmap Epigenomics Program and the Human Epigenome Project.  An extremely promising and underexplored area of research is the application of machine learning methods, which are designed to construct predictive network models, to these large-scale epigenomic data sets.  RESULTS:  Using a ChIP-Seq data set of 20 histone lysine and arginine methylations and histone variant H2A.Z in human CD4+ T-cells, we built predictive models of gene expression as a function of histone modification/variant levels using Multilinear (ML) Regression and Multivariate Adaptive Regression Splines (MARS).  Along with extensive crosstalk among the 20 histone methylations, we found H4R3me2 was the most and second most globally repressive histone methylation among the 20 studied in the ML and MARS models, respectively.  In support of our finding, a number of experimental studies show that PRMT5-catalyzed symmetric dimethylation of H4R3 is associated with repression of gene expression.  This includes a recent study, which demonstrated that H4R3me2 is required for DNMT3A-mediated DNA methylation--a known global repressor of gene expression.  CONCLUSION:  In stark contrast to univariate analysis of the relationship between H4R3me2 and gene expression levels, our study showed that the regulatory role of some modifications like H4R3me2 is masked by confounding variables, but can be elucidated by multivariate/systems-level approaches.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcS1HYQVGZGxJHRxY89Cm6SyfW6udTcc2eYmpkv5lH5xSLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3cjntl2kug%253D%253D&md5=12c49e64af3b0ce2b3a773e7e9f7a21b</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1186%2F1471-2105-11-396&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1471-2105-11-396%26sid%3Dliteratum%253Aachs%26aulast%3DXu%26aufirst%3DX.%2BJ.%26aulast%3DHoang%26aufirst%3DS.%26aulast%3DMayo%26aufirst%3DM.%2BW.%26aulast%3DBekiranov%26aufirst%3DS.%26atitle%3DApplication%2520of%2520machine%2520learning%2520methods%2520to%2520histone%2520methylation%2520ChIP-Seq%2520data%2520reveals%2520H4R3me2%2520globally%2520represses%2520gene%2520expression%26jtitle%3DBMC%2520Bioinf%26date%3D2010%26volume%3D11%26spage%3D396%26doi%3D10.1186%2F1471-2105-11-396" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Karkhanis, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, Y. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baiocchi, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Imbalzano, A. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sif, S.</span></span> <span> </span><span class="NLM_article-title">Versatility of PRMT5-induced methylation in growth control and development</span>. <i>Trends Biochem. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>36</i></span>,  <span class="NLM_fpage">633</span>– <span class="NLM_lpage">641</span>, <span class="refDoi"> DOI: 10.1016/j.tibs.2011.09.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=10.1016%2Fj.tibs.2011.09.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=21975038" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsFejurrJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2011&pages=633-641&author=V.+Karkhanisauthor=Y.+J.+Huauthor=R.+A.+Baiocchiauthor=A.+N.+Imbalzanoauthor=S.+Sif&title=Versatility+of+PRMT5-induced+methylation+in+growth+control+and+development&doi=10.1016%2Fj.tibs.2011.09.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Versatility of PRMT5-induced methylation in growth control and development</span></div><div class="casAuthors">Karkhanis, Vrajesh; Hu, Yu-Jie; Baiocchi, Robert A.; Imbalzano, Anthony N.; Sif, Said</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Biochemical Sciences</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">633-641</span>CODEN:
                <span class="NLM_cas:coden">TBSCDB</span>;
        ISSN:<span class="NLM_cas:issn">0968-0004</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Arginine methylation governs important cellular processes that impact growth and proliferation, as well as differentiation and development.  Through their ability to catalyze sym. or asym. methylation of histone and non-histone proteins, members of the protein arginine methyltransferase (PRMT) family regulate chromatin structure and expression of a wide spectrum of target genes.  Unlike other PRMTs, PRMT5 works in concert with a variety of cellular proteins including ATP-dependent chromatin remodelers and co-repressors to induce epigenetic silencing.  Recent work also implicates PRMT5 in the control of growth-promoting and pro-survival pathways, which demonstrates its versatility as an enzyme involved in both epigenetic regulation of anti-cancer target genes and organelle biogenesis.  These studies not only provide insight into the mol. mechanisms by which PRMT5 contributes to growth control, but also justify therapeutic targeting of PRMT5.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoD1--MutXQ2bVg90H21EOLACvtfcHk0lidBSvDggrRMw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsFejurrJ&md5=e5cb228750165f39636118ddee2a0ee8</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1016%2Fj.tibs.2011.09.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tibs.2011.09.001%26sid%3Dliteratum%253Aachs%26aulast%3DKarkhanis%26aufirst%3DV.%26aulast%3DHu%26aufirst%3DY.%2BJ.%26aulast%3DBaiocchi%26aufirst%3DR.%2BA.%26aulast%3DImbalzano%26aufirst%3DA.%2BN.%26aulast%3DSif%26aufirst%3DS.%26atitle%3DVersatility%2520of%2520PRMT5-induced%2520methylation%2520in%2520growth%2520control%2520and%2520development%26jtitle%3DTrends%2520Biochem.%2520Sci.%26date%3D2011%26volume%3D36%26spage%3D633%26epage%3D641%26doi%3D10.1016%2Fj.tibs.2011.09.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Richters, A.</span></span> <span> </span><span class="NLM_article-title">Targeting protein arginine methyltransferase 5 in disease</span>. <i>Future Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">2081</span>– <span class="NLM_lpage">2098</span>, <span class="refDoi"> DOI: 10.4155/fmc-2017-0089</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=10.4155%2Ffmc-2017-0089" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=29076773" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvVOit7fN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2017&pages=2081-2098&author=A.+Richters&title=Targeting+protein+arginine+methyltransferase+5+in+disease&doi=10.4155%2Ffmc-2017-0089"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting protein arginine methyltransferase 5 in disease</span></div><div class="casAuthors">Richters, Andre</div><div class="citationInfo"><span class="NLM_cas:title">Future Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">2081-2098</span>CODEN:
                <span class="NLM_cas:coden">FMCUA7</span>;
        ISSN:<span class="NLM_cas:issn">1756-8919</span>.
    
            (<span class="NLM_cas:orgname">Future Science Ltd.</span>)
        </div><div class="casAbstract">PRMT5 catalyzes the mono- and sym. dimethylation of the arginine N-guanidine group of a wide variety of target proteins including histones, transcriptional elongation factors, kinases and tumor suppressors by utilizing the essential co-factor S-adenosylmethionine as Me source.  PRMT5 overexpression has been linked to the progression of various diseases, including cancer, and is oftentimes assocd. with a poor prognosis.  Therefore, PRMT5 is promoted as a valuable target for drug discovery approaches and was a subject matter in recent endeavors aiming for the development of specific PRMT5 inhibitors.  This review will embrace the significance of PRMT5 as therapeutic target with respect to its mol. interdependencies in disease states as well as its implication in drug development approaches.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqverMGPRhuJbVg90H21EOLACvtfcHk0lidBSvDggrRMw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvVOit7fN&md5=6f8f3e27b9d7389e5b89ff084e6b691e</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.4155%2Ffmc-2017-0089&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4155%252Ffmc-2017-0089%26sid%3Dliteratum%253Aachs%26aulast%3DRichters%26aufirst%3DA.%26atitle%3DTargeting%2520protein%2520arginine%2520methyltransferase%25205%2520in%2520disease%26jtitle%3DFuture%2520Med.%2520Chem.%26date%3D2017%26volume%3D9%26spage%3D2081%26epage%3D2098%26doi%3D10.4155%2Ffmc-2017-0089" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jansson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Durant, S. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cho, E. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheahan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edelmann, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kessler, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">La Thangue, N. B.</span></span> <span> </span><span class="NLM_article-title">Arginine methylation regulates the p53 response</span>. <i>Nat. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">1431</span>– <span class="NLM_lpage">4139</span>, <span class="refDoi"> DOI: 10.1038/ncb1802</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=10.1038%2Fncb1802" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=19011621" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhsVentbfN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2008&pages=1431-4139&author=M.+Janssonauthor=S.+T.+Durantauthor=E.+C.+Choauthor=S.+Sheahanauthor=M.+Edelmannauthor=B.+Kesslerauthor=N.+B.+La+Thangue&title=Arginine+methylation+regulates+the+p53+response&doi=10.1038%2Fncb1802"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Arginine methylation regulates the p53 response</span></div><div class="casAuthors">Jansson, Martin; Durant, Stephen T.; Cho, Er-Chieh; Sheahan, Sharon; Edelmann, Mariola; Kessler, Benedikt; La Thangue, Nicholas B.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Cell Biology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1431-1439</span>CODEN:
                <span class="NLM_cas:coden">NCBIFN</span>;
        ISSN:<span class="NLM_cas:issn">1465-7392</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Activation of the p53 tumor suppressor protein in response to DNA damage leads to apoptosis or cell-cycle arrest.  Enzymic modifications are widely believed to affect and regulate p53 activity.  We describe here a level of post-translational control that has an important functional consequence on the p53 response.  We show that the protein arginine methyltransferase (PRMT) 5, as a co-factor in a DNA damage responsive Strap (stress-responsive activator of p300)-contg. co-activator complex that interacts with p53, is responsible for methylating p53.  Arginine methylation is regulated during the p53 response and affects the target gene specificity of p53.  Furthermore, PRMT5 depletion triggers p53-dependent apoptosis.  Thus, methylation on arginine residues is an underlying mechanism of control during the p53 response.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqa6VSI-iJwVLVg90H21EOLACvtfcHk0liJkRXSLf5Ibg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhsVentbfN&md5=4e01b57e90fd6ae7622e6cea3be498ce</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1038%2Fncb1802&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncb1802%26sid%3Dliteratum%253Aachs%26aulast%3DJansson%26aufirst%3DM.%26aulast%3DDurant%26aufirst%3DS.%2BT.%26aulast%3DCho%26aufirst%3DE.%2BC.%26aulast%3DSheahan%26aufirst%3DS.%26aulast%3DEdelmann%26aufirst%3DM.%26aulast%3DKessler%26aufirst%3DB.%26aulast%3DLa%2BThangue%26aufirst%3DN.%2BB.%26atitle%3DArginine%2520methylation%2520regulates%2520the%2520p53%2520response%26jtitle%3DNat.%2520Cell%2520Biol.%26date%3D2008%26volume%3D10%26spage%3D1431%26epage%3D4139%26doi%3D10.1038%2Fncb1802" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mounir, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Korn, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Westerling, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirby, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schirle, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McAllister, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoffman, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramadan, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hartung, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kipp, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quinn, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fodor, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baird, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schoumacher, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meyer, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deeds, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buchwalter, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stams, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keen, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sellers, W. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pagliarini, R. A.</span></span> <span> </span><span class="NLM_article-title">ERG signaling in prostate cancer is driven through PRMT5-dependent methylation of the Androgen Receptor</span>. <i>eLife</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">e13964</span>, <span class="refDoi"> DOI: 10.7554/eLife.13964</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=10.7554%2FeLife.13964" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2016&pages=e13964&author=Z.+Mounirauthor=J.+M.+Kornauthor=T.+Westerlingauthor=F.+Linauthor=C.+A.+Kirbyauthor=M.+Schirleauthor=G.+McAllisterauthor=G.+Hoffmanauthor=N.+Ramadanauthor=A.+Hartungauthor=Y.+Fengauthor=D.+R.+Kippauthor=C.+Quinnauthor=M.+Fodorauthor=J.+Bairdauthor=M.+Schoumacherauthor=R.+Meyerauthor=J.+Deedsauthor=G.+Buchwalterauthor=T.+Stamsauthor=N.+Keenauthor=W.+R.+Sellersauthor=M.+Brownauthor=R.+A.+Pagliarini&title=ERG+signaling+in+prostate+cancer+is+driven+through+PRMT5-dependent+methylation+of+the+Androgen+Receptor&doi=10.7554%2FeLife.13964"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.7554%2FeLife.13964&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.7554%252FeLife.13964%26sid%3Dliteratum%253Aachs%26aulast%3DMounir%26aufirst%3DZ.%26aulast%3DKorn%26aufirst%3DJ.%2BM.%26aulast%3DWesterling%26aufirst%3DT.%26aulast%3DLin%26aufirst%3DF.%26aulast%3DKirby%26aufirst%3DC.%2BA.%26aulast%3DSchirle%26aufirst%3DM.%26aulast%3DMcAllister%26aufirst%3DG.%26aulast%3DHoffman%26aufirst%3DG.%26aulast%3DRamadan%26aufirst%3DN.%26aulast%3DHartung%26aufirst%3DA.%26aulast%3DFeng%26aufirst%3DY.%26aulast%3DKipp%26aufirst%3DD.%2BR.%26aulast%3DQuinn%26aufirst%3DC.%26aulast%3DFodor%26aufirst%3DM.%26aulast%3DBaird%26aufirst%3DJ.%26aulast%3DSchoumacher%26aufirst%3DM.%26aulast%3DMeyer%26aufirst%3DR.%26aulast%3DDeeds%26aufirst%3DJ.%26aulast%3DBuchwalter%26aufirst%3DG.%26aulast%3DStams%26aufirst%3DT.%26aulast%3DKeen%26aufirst%3DN.%26aulast%3DSellers%26aufirst%3DW.%2BR.%26aulast%3DBrown%26aufirst%3DM.%26aulast%3DPagliarini%26aufirst%3DR.%2BA.%26atitle%3DERG%2520signaling%2520in%2520prostate%2520cancer%2520is%2520driven%2520through%2520PRMT5-dependent%2520methylation%2520of%2520the%2520Androgen%2520Receptor%26jtitle%3DeLife%26date%3D2016%26volume%3D5%26spage%3De13964%26doi%3D10.7554%2FeLife.13964" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Burgos, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilczek, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Onikubo, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bonanno, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jansong, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reimer, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shechter, D.</span></span> <span> </span><span class="NLM_article-title">Histone H2A and H4 N-terminal tails are positioned by the MEP50 WD repeat protein for efficient methylation by the PRMT5 arginine methyltransferase</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>290</i></span>,  <span class="NLM_fpage">9674</span>– <span class="NLM_lpage">9689</span>, <span class="refDoi"> DOI: 10.1074/jbc.M115.636894</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=10.1074%2Fjbc.M115.636894" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=290&publication_year=2015&pages=9674-9689&author=E.+S.+Burgosauthor=C.+Wilczekauthor=T.+Onikuboauthor=J.+B.+Bonannoauthor=J.+Jansongauthor=U.+Reimerauthor=D.+Shechter&title=Histone+H2A+and+H4+N-terminal+tails+are+positioned+by+the+MEP50+WD+repeat+protein+for+efficient+methylation+by+the+PRMT5+arginine+methyltransferase&doi=10.1074%2Fjbc.M115.636894"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M115.636894&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M115.636894%26sid%3Dliteratum%253Aachs%26aulast%3DBurgos%26aufirst%3DE.%2BS.%26aulast%3DWilczek%26aufirst%3DC.%26aulast%3DOnikubo%26aufirst%3DT.%26aulast%3DBonanno%26aufirst%3DJ.%2BB.%26aulast%3DJansong%26aufirst%3DJ.%26aulast%3DReimer%26aufirst%3DU.%26aulast%3DShechter%26aufirst%3DD.%26atitle%3DHistone%2520H2A%2520and%2520H4%2520N-terminal%2520tails%2520are%2520positioned%2520by%2520the%2520MEP50%2520WD%2520repeat%2520protein%2520for%2520efficient%2520methylation%2520by%2520the%2520PRMT5%2520arginine%2520methyltransferase%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2015%26volume%3D290%26spage%3D9674%26epage%3D9689%26doi%3D10.1074%2Fjbc.M115.636894" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Stopa, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krebs, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shechter, D.</span></span> <span> </span><span class="NLM_article-title">The PRMT5 arginine methyltransferase: many roles in development, cancer and beyond</span>. <i>Cell. Mol. Life Sci.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>72</i></span>,  <span class="NLM_fpage">2041</span>– <span class="NLM_lpage">2059</span>, <span class="refDoi"> DOI: 10.1007/s00018-015-1847-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=10.1007%2Fs00018-015-1847-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=25662273" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=1%3ACAS%3A528%3ADC%252BC2MXitlOrtr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=72&publication_year=2015&pages=2041-2059&author=N.+Stopaauthor=J.+E.+Krebsauthor=D.+Shechter&title=The+PRMT5+arginine+methyltransferase%3A+many+roles+in+development%2C+cancer+and+beyond&doi=10.1007%2Fs00018-015-1847-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">The PRMT5 arginine methyltransferase- many roles in development, cancer and beyond</span></div><div class="casAuthors">Stopa, Nicole; Krebs, Jocelyn E.; Shechter, David</div><div class="citationInfo"><span class="NLM_cas:title">Cellular and Molecular Life Sciences</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">72</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2041-2059</span>CODEN:
                <span class="NLM_cas:coden">CMLSFI</span>;
        ISSN:<span class="NLM_cas:issn">1420-682X</span>.
    
            (<span class="NLM_cas:orgname">Birkhaeuser Basel</span>)
        </div><div class="casAbstract">Post-translational arginine methylation is responsible for regulation of many biol. processes.  The protein arginine methyltransferase 5 (PRMT5, also known as Hsl7, Jbp1, Skb1, Capsuleen, or Dart5) is the major enzyme responsible for mono- and sym. dimethylation of arginine.  An expanding literature demonstrates its crit. biol. function in a wide range of cellular processes.  Histone and other protein methylation by PRMT5 regulate genome organization, transcription, stem cells, primordial germ cells, differentiation, the cell cycle, and spliceosome assembly.  Metazoan PRMT5 is found in complex with the WD-repeat protein MEP50 (also known as Wdr77, androgen receptor coactivator p44, or Valois).  PRMT5 also directly assocs. with a range of other protein factors, including pICln, Menin, CoPR5 and RioK1 that may alter its subcellular localization and protein substrate selection.  Protein substrate and PRMT5-MEP50 post-translation modifications induce crosstalk to regulate PRMT5 activity.  Crystal structures of C. elegans PRMT5 and human and frog PRMT5-MEP50 complexes provide substantial insight into the mechanisms of substrate recognition and procession to dimethylation.  Enzymol. studies of PRMT5 have uncovered compelling insights essential for future development of specific PRMT5 inhibitors.  In addn., newly accumulating evidence implicates PRMT5 and MEP50 expression levels and their methyltransferase activity in cancer tumorigenesis, and, significantly, as markers of poor clin. outcome, marking them as potential oncogenes.  Here, we review the substantial new literature on PRMT5 and its partners to highlight the significance of understanding this essential enzyme in health and disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqGZo-Bwo40_7Vg90H21EOLACvtfcHk0lgq1c-V_1XJ0Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXitlOrtr0%253D&md5=2173fff7825f142cde0ddde7f7b879f5</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1007%2Fs00018-015-1847-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00018-015-1847-9%26sid%3Dliteratum%253Aachs%26aulast%3DStopa%26aufirst%3DN.%26aulast%3DKrebs%26aufirst%3DJ.%2BE.%26aulast%3DShechter%26aufirst%3DD.%26atitle%3DThe%2520PRMT5%2520arginine%2520methyltransferase%253A%2520many%2520roles%2520in%2520development%252C%2520cancer%2520and%2520beyond%26jtitle%3DCell.%2520Mol.%2520Life%2520Sci.%26date%3D2015%26volume%3D72%26spage%3D2041%26epage%3D2059%26doi%3D10.1007%2Fs00018-015-1847-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Meister, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eggert, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buhler, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brahms, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kambach, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, U.</span></span> <span> </span><span class="NLM_article-title">Methylation of Sm proteins by a complex containing PRMT5 and the putative U snRNP assembly factor pICln</span>. <i>Curr. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">1990</span>– <span class="NLM_lpage">1994</span>, <span class="refDoi"> DOI: 10.1016/S0960-9822(01)00592-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=10.1016%2FS0960-9822%2801%2900592-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=11747828" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=1%3ACAS%3A528%3ADC%252BD38XisFKj" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2001&pages=1990-1994&author=G.+Meisterauthor=C.+Eggertauthor=D.+Buhlerauthor=H.+Brahmsauthor=C.+Kambachauthor=U.+Fischer&title=Methylation+of+Sm+proteins+by+a+complex+containing+PRMT5+and+the+putative+U+snRNP+assembly+factor+pICln&doi=10.1016%2FS0960-9822%2801%2900592-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Methylation of Sm proteins by a complex containing PRMT5 and the putative U snRNP assembly factor pICln</span></div><div class="casAuthors">Meister, Gunter; Eggert, Christian; Buhler, Dirk; Brahms, Hero; Kambach, Christian; Fischer, Utz</div><div class="citationInfo"><span class="NLM_cas:title">Current Biology</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">1990-1994</span>CODEN:
                <span class="NLM_cas:coden">CUBLE2</span>;
        ISSN:<span class="NLM_cas:issn">0960-9822</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Seven Sm proteins, termed B/B', D1, D2, D3, E, F, and G, assemble in an ordered manner onto U snRNAs to form the Sm core of the spliceosomal snRNPs U1, U2, U4/U6, and U5.  The survival of motor neuron (SMN) protein binds to Sm proteins and mediates in the context of a macromol. (SMN-) complex the assembly of the Sm core.  Binding of SMN to Sm proteins is enhanced by modification of specific arginine residues in the Sm proteins D1 and D3 to sym. dimethylarginines (sDMAs), suggesting that assembly might be regulated at the posttranslational level.  Here we provide evidence that the previously described pICln-complex, consisting of Sm proteins, the methyltransferase PRMT5, pICln, and two novel factors, catalyzes the sDMA modification of Sm proteins.  In vitro studies further revealed that the pICln complex inhibits the spontaneous assembly of Sm proteins onto a U snRNA.  This effect is mediated by pICln via its binding to the Sm fold of Sm proteins, thereby preventing specific interactions between Sm proteins required for the formation of the Sm core.  Our data suggest that the pICln complex regulates an early step in the assembly of U snRNPs, possibly the transfer of Sm proteins to the SMN-complex.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr1Vh7kSAF-X7Vg90H21EOLACvtfcHk0liKHxLlmvkzhw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XisFKj&md5=7221a40240d94264723c40f5da2e0394</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1016%2FS0960-9822%2801%2900592-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0960-9822%252801%252900592-9%26sid%3Dliteratum%253Aachs%26aulast%3DMeister%26aufirst%3DG.%26aulast%3DEggert%26aufirst%3DC.%26aulast%3DBuhler%26aufirst%3DD.%26aulast%3DBrahms%26aufirst%3DH.%26aulast%3DKambach%26aufirst%3DC.%26aulast%3DFischer%26aufirst%3DU.%26atitle%3DMethylation%2520of%2520Sm%2520proteins%2520by%2520a%2520complex%2520containing%2520PRMT5%2520and%2520the%2520putative%2520U%2520snRNP%2520assembly%2520factor%2520pICln%26jtitle%3DCurr.%2520Biol.%26date%3D2001%26volume%3D11%26spage%3D1990%26epage%3D1994%26doi%3D10.1016%2FS0960-9822%2801%2900592-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Banasavadi-Siddegowda, Y. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Welker, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Imitola, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beattie, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baiocchi, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaur, B.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">Prmt5 as a druggable target for glioblastoma therapy</span>. <i>Neuro. Oncol</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">753</span>– <span class="NLM_lpage">763</span>, <span class="refDoi"> DOI: 10.1093/neuonc/nox206</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=10.1093%2Fneuonc%2Fnox206" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=29106602" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=1%3ACAS%3A280%3ADC%252BC1M7otFWjtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2018&pages=753-763&author=Y.+K.+Banasavadi-Siddegowdaauthor=A.+M.+Welkerauthor=J.+Imitolaauthor=C.+L.+Liauthor=J.+Y.+Zhangauthor=C.+E.+Beattieauthor=R.+Baiocchiauthor=B.+Kaur&title=Prmt5+as+a+druggable+target+for+glioblastoma+therapy&doi=10.1093%2Fneuonc%2Fnox206"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">PRMT5 as a druggable target for glioblastoma therapy</span></div><div class="casAuthors">Banasavadi-Siddegowda Yeshavanth Kumar; Hsu Sigmund; Kaur Balveen; Banasavadi-Siddegowda Yeshavanth Kumar; Kaur Balveen; Welker Alessandra M; An Min; Beattie Christine E; Welker Alessandra M; Welker Alessandra M; Yang Xiaozhi; Li Chenglong; Zhou Wei; Li Chenglong; Shi Guqin; Li Chenglong; Imitola Jaime; Wang Jiang; Phelps Mitch; Zhang Jianying; Baiocchi Robert</div><div class="citationInfo"><span class="NLM_cas:title">Neuro-oncology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">753-763</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Background:  In spite of standard multimodal therapy consisting of surgical resection followed by radiation and concurrent chemotherapy, prognosis for glioblastoma (GBM) patients remains poor.  The identification of both differentiated and undifferentiated "stem cell like" populations in the tumor highlights the significance of finding novel targets that affect the heterogeneous tumor cell population.  Protein arginine methyltransferase 5 (PRMT5) is one such candidate gene whose nuclear expression correlates with poor survival and has been reported to be required for survival of differentiated GBM cells and self-renewal of undifferentiated GBM cells.  In the current study we screened the specificity and efficacy of 4 novel PRMT5 inhibitors in the treatment of GBM.  Methods:  Efficacies of these inhibitors were screened using an in vitro GBM neurosphere model and an in vivo intracranial zebrafish model of glioma.  Standard molecular biology methods were employed to investigate changes in cell cycle, growth, and senescence.  Results:  In vitro and in vivo studies revealed that among the 4 PRMT5 inhibitors, treatment of GBM cells with compound 5 (CMP5) mirrored the effects of PRMT5 knockdown wherein it led to apoptosis of differentiated GBM cells and drove undifferentiated primary patient derived GBM cells into a nonreplicative senescent state.  Conclusion:  In vivo antitumor efficacy combined with the specificity of CMP5 underscores the importance of developing it for translation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSg_a1SxIfsoNj_Vg2vObihfW6udTcc2eYYt725l_GN2rntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1M7otFWjtw%253D%253D&md5=1f89af1ee62765d89fb221b3f68dd0d1</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1093%2Fneuonc%2Fnox206&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fneuonc%252Fnox206%26sid%3Dliteratum%253Aachs%26aulast%3DBanasavadi-Siddegowda%26aufirst%3DY.%2BK.%26aulast%3DWelker%26aufirst%3DA.%2BM.%26aulast%3DImitola%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DC.%2BL.%26aulast%3DZhang%26aufirst%3DJ.%2BY.%26aulast%3DBeattie%26aufirst%3DC.%2BE.%26aulast%3DBaiocchi%26aufirst%3DR.%26aulast%3DKaur%26aufirst%3DB.%26atitle%3DPrmt5%2520as%2520a%2520druggable%2520target%2520for%2520glioblastoma%2520therapy%26jtitle%3DNeuro.%2520Oncol%26date%3D2018%26volume%3D20%26spage%3D753%26epage%3D763%26doi%3D10.1093%2Fneuonc%2Fnox206" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Han, X. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, W. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wheeler, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friedman, G. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Province, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">You, Z. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fathallah-Shaykh, H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gillespie, G. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, X. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">King, P. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nabors, L. B.</span></span> <span> </span><span class="NLM_article-title">Expression of PRMT5 correlates with malignant grade in gliomas and plays a pivotal role in tumor growth in vitro</span>. <i>J. Neuro-Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>118</i></span>,  <span class="NLM_fpage">61</span>– <span class="NLM_lpage">72</span>, <span class="refDoi"> DOI: 10.1007/s11060-014-1419-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=10.1007%2Fs11060-014-1419-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=24664369" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=1%3ACAS%3A528%3ADC%252BC2cXlt1eqtbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=118&publication_year=2014&pages=61-72&author=X.+S.+Hanauthor=R.+Liauthor=W.+B.+Zhangauthor=X.+H.+Yangauthor=C.+G.+Wheelerauthor=G.+K.+Friedmanauthor=P.+Provinceauthor=Q.+Dingauthor=Z.+Y.+Youauthor=H.+M.+Fathallah-Shaykhauthor=G.+Y.+Gillespieauthor=X.+Y.+Zhaoauthor=P.+H.+Kingauthor=L.+B.+Nabors&title=Expression+of+PRMT5+correlates+with+malignant+grade+in+gliomas+and+plays+a+pivotal+role+in+tumor+growth+in+vitro&doi=10.1007%2Fs11060-014-1419-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Expression of PRMT5 correlates with malignant grade in gliomas and plays a pivotal role in tumor growth in vitro</span></div><div class="casAuthors">Han, Xiaosi; Li, Rong; Zhang, Wenbin; Yang, Xiuhua; Wheeler, Crystal G.; Friedman, Gregory K.; Province, Paula; Ding, Qiang; You, Zhiying; Fathallah-Shaykh, Hassan M.; Gillespie, G. Yancey; Zhao, Xinyang; King, Peter H.; Nabors, L. Burt</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Neuro-Oncology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">118</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">61-72</span>CODEN:
                <span class="NLM_cas:coden">JNODD2</span>;
        ISSN:<span class="NLM_cas:issn">0167-594X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Protein arginine methyltransferase 5 (PRMT5) catalyzes the formation of ω-NG,N'G-sym. dimethylarginine residues on histones as well as other proteins.  These modifications play an important role in cell differentiation and tumor cell growth.  However, the role of PRMT5 in human glioma cells has not been characterized.  In this study, we assessed protein expression profiles of PRMT5 in control brain, WHO grade II astrocytomas, anaplastic astrocytomas, and glioblastoma multiforme (GBM) by immunohistochem.  PRMT5 was low in glial cells in control brain tissues and low grade astrocytomas.  Its expression increased in parallel with malignant progression, and was highly expressed in GBM.  Knockdown of PRMT5 by small hairpin RNA caused alterations of p-ERK1/2 and significantly repressed the clonogenic potential and viability of glioma cells.  These findings indicate that PRMT5 is a marker of malignant progression in glioma tumors and plays a pivotal role in tumor growth.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrRIIy_ZOt5bbVg90H21EOLACvtfcHk0liKHxLlmvkzhw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXlt1eqtbc%253D&md5=085dfb02d93a7afddb181c6458b9f5a2</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1007%2Fs11060-014-1419-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs11060-014-1419-0%26sid%3Dliteratum%253Aachs%26aulast%3DHan%26aufirst%3DX.%2BS.%26aulast%3DLi%26aufirst%3DR.%26aulast%3DZhang%26aufirst%3DW.%2BB.%26aulast%3DYang%26aufirst%3DX.%2BH.%26aulast%3DWheeler%26aufirst%3DC.%2BG.%26aulast%3DFriedman%26aufirst%3DG.%2BK.%26aulast%3DProvince%26aufirst%3DP.%26aulast%3DDing%26aufirst%3DQ.%26aulast%3DYou%26aufirst%3DZ.%2BY.%26aulast%3DFathallah-Shaykh%26aufirst%3DH.%2BM.%26aulast%3DGillespie%26aufirst%3DG.%2BY.%26aulast%3DZhao%26aufirst%3DX.%2BY.%26aulast%3DKing%26aufirst%3DP.%2BH.%26aulast%3DNabors%26aufirst%3DL.%2BB.%26atitle%3DExpression%2520of%2520PRMT5%2520correlates%2520with%2520malignant%2520grade%2520in%2520gliomas%2520and%2520plays%2520a%2520pivotal%2520role%2520in%2520tumor%2520growth%2520in%2520vitro%26jtitle%3DJ.%2520Neuro-Oncol.%26date%3D2014%26volume%3D118%26spage%3D61%26epage%3D72%26doi%3D10.1007%2Fs11060-014-1419-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yan, F. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alinari, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lustberg, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, L. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cordero-Nieves, H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banasavadi-Siddegowda, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Virk, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barnholtz-Sloan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bell, E. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wojton, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacob, N. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chakravarti, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nowicki, M. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lapalombella, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Datta, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haseley, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patton, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, P. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ryu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mo, X. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marcucci, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nuovo, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kwon, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Byrd, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiocca, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sif, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacob, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lawler, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaur, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baiocchi, R. A.</span></span> <span> </span><span class="NLM_article-title">Genetic validation of the protein arginine methyltransferase PRMT5 as a candidate therapeutic target in glioblastoma</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>74</i></span>,  <span class="NLM_fpage">1752</span>– <span class="NLM_lpage">1765</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.CAN-13-0884</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=10.1158%2F0008-5472.CAN-13-0884" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=74&publication_year=2014&pages=1752-1765&author=F.+T.+Yanauthor=L.+Alinariauthor=M.+E.+Lustbergauthor=L.+K.+Martinauthor=H.+M.+Cordero-Nievesauthor=Y.+Banasavadi-Siddegowdaauthor=S.+Virkauthor=J.+Barnholtz-Sloanauthor=E.+H.+Bellauthor=J.+Wojtonauthor=N.+K.+Jacobauthor=A.+Chakravartiauthor=M.+O.+Nowickiauthor=X.+Wuauthor=R.+Lapalombellaauthor=J.+Dattaauthor=B.+Yuauthor=K.+Gordonauthor=A.+Haseleyauthor=J.+T.+Pattonauthor=P.+L.+Smithauthor=J.+Ryuauthor=X.+L.+Zhangauthor=X.+K.+Moauthor=G.+Marcucciauthor=G.+Nuovoauthor=C.+H.+Kwonauthor=J.+C.+Byrdauthor=E.+A.+Chioccaauthor=C.+L.+Liauthor=S.+Sifauthor=S.+Jacobauthor=S.+Lawlerauthor=B.+Kaurauthor=R.+A.+Baiocchi&title=Genetic+validation+of+the+protein+arginine+methyltransferase+PRMT5+as+a+candidate+therapeutic+target+in+glioblastoma&doi=10.1158%2F0008-5472.CAN-13-0884"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-13-0884&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-13-0884%26sid%3Dliteratum%253Aachs%26aulast%3DYan%26aufirst%3DF.%2BT.%26aulast%3DAlinari%26aufirst%3DL.%26aulast%3DLustberg%26aufirst%3DM.%2BE.%26aulast%3DMartin%26aufirst%3DL.%2BK.%26aulast%3DCordero-Nieves%26aufirst%3DH.%2BM.%26aulast%3DBanasavadi-Siddegowda%26aufirst%3DY.%26aulast%3DVirk%26aufirst%3DS.%26aulast%3DBarnholtz-Sloan%26aufirst%3DJ.%26aulast%3DBell%26aufirst%3DE.%2BH.%26aulast%3DWojton%26aufirst%3DJ.%26aulast%3DJacob%26aufirst%3DN.%2BK.%26aulast%3DChakravarti%26aufirst%3DA.%26aulast%3DNowicki%26aufirst%3DM.%2BO.%26aulast%3DWu%26aufirst%3DX.%26aulast%3DLapalombella%26aufirst%3DR.%26aulast%3DDatta%26aufirst%3DJ.%26aulast%3DYu%26aufirst%3DB.%26aulast%3DGordon%26aufirst%3DK.%26aulast%3DHaseley%26aufirst%3DA.%26aulast%3DPatton%26aufirst%3DJ.%2BT.%26aulast%3DSmith%26aufirst%3DP.%2BL.%26aulast%3DRyu%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DX.%2BL.%26aulast%3DMo%26aufirst%3DX.%2BK.%26aulast%3DMarcucci%26aufirst%3DG.%26aulast%3DNuovo%26aufirst%3DG.%26aulast%3DKwon%26aufirst%3DC.%2BH.%26aulast%3DByrd%26aufirst%3DJ.%2BC.%26aulast%3DChiocca%26aufirst%3DE.%2BA.%26aulast%3DLi%26aufirst%3DC.%2BL.%26aulast%3DSif%26aufirst%3DS.%26aulast%3DJacob%26aufirst%3DS.%26aulast%3DLawler%26aufirst%3DS.%26aulast%3DKaur%26aufirst%3DB.%26aulast%3DBaiocchi%26aufirst%3DR.%2BA.%26atitle%3DGenetic%2520validation%2520of%2520the%2520protein%2520arginine%2520methyltransferase%2520PRMT5%2520as%2520a%2520candidate%2520therapeutic%2520target%2520in%2520glioblastoma%26jtitle%3DCancer%2520Res.%26date%3D2014%26volume%3D74%26spage%3D1752%26epage%3D1765%26doi%3D10.1158%2F0008-5472.CAN-13-0884" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gu, Z. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, F. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, W. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z. X.</span></span> <span> </span><span class="NLM_article-title">Protein arginine methyltransferase 5 is essential for growth of lung cancer cells</span>. <i>Biochem. J.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>446</i></span>,  <span class="NLM_fpage">235</span>– <span class="NLM_lpage">241</span>, <span class="refDoi"> DOI: 10.1042/BJ20120768</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=10.1042%2FBJ20120768" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=446&publication_year=2012&pages=235-241&author=Z.+P.+Guauthor=S.+Gaoauthor=F.+H.+Zhangauthor=Z.+Q.+Wangauthor=W.+C.+Maauthor=R.+E.+Davisauthor=Z.+X.+Wang&title=Protein+arginine+methyltransferase+5+is+essential+for+growth+of+lung+cancer+cells&doi=10.1042%2FBJ20120768"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1042%2FBJ20120768&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252FBJ20120768%26sid%3Dliteratum%253Aachs%26aulast%3DGu%26aufirst%3DZ.%2BP.%26aulast%3DGao%26aufirst%3DS.%26aulast%3DZhang%26aufirst%3DF.%2BH.%26aulast%3DWang%26aufirst%3DZ.%2BQ.%26aulast%3DMa%26aufirst%3DW.%2BC.%26aulast%3DDavis%26aufirst%3DR.%2BE.%26aulast%3DWang%26aufirst%3DZ.%2BX.%26atitle%3DProtein%2520arginine%2520methyltransferase%25205%2520is%2520essential%2520for%2520growth%2520of%2520lung%2520cancer%2520cells%26jtitle%3DBiochem.%2520J.%26date%3D2012%26volume%3D446%26spage%3D235%26epage%3D241%26doi%3D10.1042%2FBJ20120768" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kong, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gu, Z. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Bryant, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z. X.</span></span> <span> </span><span class="NLM_article-title">Selective small-chemical inhibitors of protein arginine methyltransferase 5 with anti-lung cancer activity</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">e0181601</span>, <span class="refDoi"> DOI: 10.1371/journal.pone.0181601</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=10.1371%2Fjournal.pone.0181601" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=28806746" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtV2rtbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2017&pages=e0181601&author=G.+M.+Kongauthor=M.+Yuauthor=Z.+P.+Guauthor=Z.+Chenauthor=R.+M.+Xuauthor=D.+O%E2%80%99Bryantauthor=Z.+X.+Wang&title=Selective+small-chemical+inhibitors+of+protein+arginine+methyltransferase+5+with+anti-lung+cancer+activity&doi=10.1371%2Fjournal.pone.0181601"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Selective small-chemical inhibitors of protein arginine methyltransferase 5 with anti-lung cancer activity</span></div><div class="casAuthors">Kong, Gui-Mei; Yu, Min; Gu, Zhongping; Chen, Zhi; Xu, Rui-Ming; O'Bryant, Deon; Wang, Zhengxin</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">e0181601/1-e0181601/23</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Protein arginine methyltransferase 5 (PRMT5) plays crit. roles in a wide variety of biol. processes, including tumorigenesis.  By screening a library of small chem. compds., we identified eight compds. that selectively inhibit the PRMT5 enzymic activity, with IC50 values ranging from 0.1 to 6 μM.  Mol. docking simulation and site directed mutagenesis indicated that identified compds. target the substrate-binding site in PRMT5.  Treatment of lung cancer cells with identified inhibitors led to inhibition of the sym. arginine methylation of SmD3 and histones and the cellular proliferation.  Oral administration of the inhibitor demonstrated antitumor activity in a lung tumor xenograft model.  Thus, identified PRMT5-specific small-mol. inhibitors would help elucidate the biol. roles of PRMT5 and serve as lead compds. for future drug development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqHrmbn8I9YMLVg90H21EOLACvtfcHk0lgdPjcIDR2M1g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtV2rtbk%253D&md5=0306ed1351aac8a04d286df255c6e432</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0181601&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0181601%26sid%3Dliteratum%253Aachs%26aulast%3DKong%26aufirst%3DG.%2BM.%26aulast%3DYu%26aufirst%3DM.%26aulast%3DGu%26aufirst%3DZ.%2BP.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DXu%26aufirst%3DR.%2BM.%26aulast%3DO%25E2%2580%2599Bryant%26aufirst%3DD.%26aulast%3DWang%26aufirst%3DZ.%2BX.%26atitle%3DSelective%2520small-chemical%2520inhibitors%2520of%2520protein%2520arginine%2520methyltransferase%25205%2520with%2520anti-lung%2520cancer%2520activity%26jtitle%3DPLoS%2520One%26date%3D2017%26volume%3D12%26spage%3De0181601%26doi%3D10.1371%2Fjournal.pone.0181601" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jin, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cui, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Du, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ren, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pan, J.</span></span> <span> </span><span class="NLM_article-title">Targeting methyltransferase PRMT5 eliminates leukemia stem cells in chronic myelogenous leukemia</span>. <i>J. Clin. Invest.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>126</i></span>,  <span class="NLM_fpage">3961</span>– <span class="NLM_lpage">3980</span>, <span class="refDoi"> DOI: 10.1172/JCI85239</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=10.1172%2FJCI85239" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=126&publication_year=2016&pages=3961-3980&author=Y.+Jinauthor=J.+Zhouauthor=F.+Xuauthor=B.+Jinauthor=L.+Cuiauthor=Y.+Wangauthor=X.+Duauthor=J.+Liauthor=P.+Liauthor=R.+Renauthor=J.+Pan&title=Targeting+methyltransferase+PRMT5+eliminates+leukemia+stem+cells+in+chronic+myelogenous+leukemia&doi=10.1172%2FJCI85239"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1172%2FJCI85239&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1172%252FJCI85239%26sid%3Dliteratum%253Aachs%26aulast%3DJin%26aufirst%3DY.%26aulast%3DZhou%26aufirst%3DJ.%26aulast%3DXu%26aufirst%3DF.%26aulast%3DJin%26aufirst%3DB.%26aulast%3DCui%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DDu%26aufirst%3DX.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DP.%26aulast%3DRen%26aufirst%3DR.%26aulast%3DPan%26aufirst%3DJ.%26atitle%3DTargeting%2520methyltransferase%2520PRMT5%2520eliminates%2520leukemia%2520stem%2520cells%2520in%2520chronic%2520myelogenous%2520leukemia%26jtitle%3DJ.%2520Clin.%2520Invest.%26date%3D2016%26volume%3D126%26spage%3D3961%26epage%3D3980%26doi%3D10.1172%2FJCI85239" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Deng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shao, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, H. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elzey, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pili, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ratliff, T. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, C. D.</span></span> <span> </span><span class="NLM_article-title">Protein arginine methyltransferase 5 functions as an epigenetic activator of the androgen receptor to promote prostate cancer cell growth</span>. <i>Oncogene</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>36</i></span>,  <span class="NLM_fpage">1223</span>– <span class="NLM_lpage">1231</span>, <span class="refDoi"> DOI: 10.1038/onc.2016.287</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=10.1038%2Fonc.2016.287" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=27546619" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtlyisLvK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2017&pages=1223-1231&author=X.+Dengauthor=G.+Shaoauthor=H.+T.+Zhangauthor=C.+Liauthor=D.+Zhangauthor=L.+Chengauthor=B.+D.+Elzeyauthor=R.+Piliauthor=T.+L.+Ratliffauthor=J.+Huangauthor=C.+D.+Hu&title=Protein+arginine+methyltransferase+5+functions+as+an+epigenetic+activator+of+the+androgen+receptor+to+promote+prostate+cancer+cell+growth&doi=10.1038%2Fonc.2016.287"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Protein arginine methyltransferase 5 functions as an epigenetic activator of the androgen receptor to promote prostate cancer cell growth</span></div><div class="casAuthors">Deng, X.; Shao, G.; Zhang, H.-T.; Li, C.; Zhang, D.; Cheng, L.; Elzey, B. D.; Pili, R.; Ratliff, T. L.; Huang, J.; Hu, C.-D.</div><div class="citationInfo"><span class="NLM_cas:title">Oncogene</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1223-1231</span>CODEN:
                <span class="NLM_cas:coden">ONCNES</span>;
        ISSN:<span class="NLM_cas:issn">0950-9232</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Protein arginine methyltransferase 5 (PRMT5) is an emerging epigenetic enzyme that mainly represses transcription of target genes via sym. dimethylation of arginine residues on histones H4R3, H3R8 and H2AR3.  Accumulating evidence suggests that PRMT5 may function as an oncogene to drive cancer cell growth by epigenetic inactivation of several tumor suppressors.  Here, we provide evidence that PRMT5 promotes prostate cancer cell growth by epigenetically activating transcription of the androgen receptor (AR) in prostate cancer cells.  Knockdown of PRMT5 or inhibition of PRMT5 by a specific inhibitor reduces the expression of AR and suppresses the growth of multiple AR-pos., but not AR-neg., prostate cancer cells.  Significantly, knockdown of PRMT5 in AR-pos. LNCaP cells completely suppresses the growth of xenograft tumors in mice.  Mol. anal. reveals that PRMT5 binds to the proximal promoter region of the AR gene and contributes mainly to the enriched sym. dimethylation of H4R3 in the same region.  Mechanistically, PRMT5 is recruited to the AR promoter by its interaction with Sp1, the major transcription factor responsible for AR transcription, and forms a complex with Brg1, an ATP-dependent chromatin remodeler, on the proximal promoter region of the AR gene.  Furthermore, PRMT5 expression in prostate cancer tissues is significantly higher than that in benign prostatic hyperplasia tissues, and PRMT5 expression correlates pos. with AR expression at both the protein and mRNA levels.  Taken together, our results identify PRMT5 as a novel epigenetic activator of AR in prostate cancer.  Given that inhibiting AR transcriptional activity or androgen synthesis remains the major mechanism of action for most existing anti-androgen agents, our findings also raise an interesting possibility that targeting PRMT5 may represent a novel approach for prostate cancer treatment by eliminating AR expression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr6wjniAjJYjbVg90H21EOLACvtfcHk0lgdPjcIDR2M1g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtlyisLvK&md5=75aff420e6c8a819bd41fd45cec7dec9</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1038%2Fonc.2016.287&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fonc.2016.287%26sid%3Dliteratum%253Aachs%26aulast%3DDeng%26aufirst%3DX.%26aulast%3DShao%26aufirst%3DG.%26aulast%3DZhang%26aufirst%3DH.%2BT.%26aulast%3DLi%26aufirst%3DC.%26aulast%3DZhang%26aufirst%3DD.%26aulast%3DCheng%26aufirst%3DL.%26aulast%3DElzey%26aufirst%3DB.%2BD.%26aulast%3DPili%26aufirst%3DR.%26aulast%3DRatliff%26aufirst%3DT.%2BL.%26aulast%3DHuang%26aufirst%3DJ.%26aulast%3DHu%26aufirst%3DC.%2BD.%26atitle%3DProtein%2520arginine%2520methyltransferase%25205%2520functions%2520as%2520an%2520epigenetic%2520activator%2520of%2520the%2520androgen%2520receptor%2520to%2520promote%2520prostate%2520cancer%2520cell%2520growth%26jtitle%3DOncogene%26date%3D2017%26volume%3D36%26spage%3D1223%26epage%3D1231%26doi%3D10.1038%2Fonc.2016.287" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mavrakis, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McDonald, E. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schlabach, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Billy, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoffman, G. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">deWeck, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruddy, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Venkatesan, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McAllister, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">deBeaumont, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ho, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan-Neale, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, G. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yin, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kipp, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McNamara, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sprague, E. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cho, Y. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crawford, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Capka, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hurov, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Porter, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tallarico, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mickanin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lees, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pagliarini, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keen, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmelzle, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hofmann, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stegmeier, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sellers, W. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stump, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yue, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciccone, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vitari, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, A.</span></span> <span> </span><span class="NLM_article-title">Disordered methionine metabolism in MTAP/CDKN2A-deleted cancers leads to marked dependence on PRMT5</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>351</i></span>,  <span class="NLM_fpage">1208</span>– <span class="NLM_lpage">1213</span>, <span class="refDoi"> DOI: 10.1126/science.aad5944</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=10.1126%2Fscience.aad5944" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=26912361" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=1%3ACAS%3A528%3ADC%252BC28Xjs12gtr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=351&publication_year=2016&pages=1208-1213&author=K.+Mavrakisauthor=E.+R.+McDonaldauthor=M.+R.+Schlabachauthor=E.+Billyauthor=G.+R.+Hoffmanauthor=A.+deWeckauthor=D.+A.+Ruddyauthor=K.+Venkatesanauthor=G.+McAllisterauthor=R.+deBeaumontauthor=S.+Hoauthor=Y.+Liuauthor=Y.+Yan-Nealeauthor=G.+Z.+Yangauthor=F.+Linauthor=H.+Yinauthor=H.+Gaoauthor=D.+R.+Kippauthor=S.+P.+Zhaoauthor=J.+T.+McNamaraauthor=E.+R.+Spragueauthor=Y.+S.+Choauthor=J.+Guauthor=K.+Crawfordauthor=V.+Capkaauthor=K.+Hurovauthor=J.+A.+Porterauthor=J.+Tallaricoauthor=C.+Mickaninauthor=E.+Leesauthor=R.+Pagliariniauthor=N.+Keenauthor=T.+Schmelzleauthor=F.+Hofmannauthor=F.+Stegmeierauthor=W.+R.+Sellersauthor=J.+Yuauthor=M.+Stumpauthor=Y.+Yueauthor=B.+Zhengauthor=Y.+Linauthor=A.+C.+Cicconeauthor=A.+C.+Vitariauthor=A.+Lai&title=Disordered+methionine+metabolism+in+MTAP%2FCDKN2A-deleted+cancers+leads+to+marked+dependence+on+PRMT5&doi=10.1126%2Fscience.aad5944"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Disordered methionine metabolism in MTAP/CDKN2A-deleted cancers leads to dependence on PRMT5</span></div><div class="casAuthors">Mavrakis, Konstantinos J.; McDonald, E. Robert, III; Schlabach, Michael R.; Billy, Eric; Hoffman, Gregory R.; de Weck, Antoine; Ruddy, David A.; Venkatesan, Kavitha; Yu, Jianjun; McAllister, Gregg; Stump, Mark; de Beaumont, Rosalie; Ho, Samuel; Yue, Yingzi; Liu, Yue; Yan-Neale, Yan; Yang, Guizhi; Lin, Fallon; Yin, Hong; Gao, Hui; Kipp, D. Randal; Zhao, Songping; McNamara, Joshua T.; Sprague, Elizabeth R.; Zheng, Bing; Lin, Ying; Cho, Young Shin; Gu, Justin; Crawford, Kenneth; Ciccone, David; Vitari, Alberto C.; Lai, Albert; Capka, Vladimir; Hurov, Kristen; Porter, Jeffery A.; Tallarico, John; Mickanin, Craig; Lees, Emma; Pagliarini, Raymond; Keen, Nicholas; Schmelzle, Tobias; Hofmann, Francesco; Stegmeier, Frank; Sellers, William R.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">351</span>
        (<span class="NLM_cas:issue">6278</span>),
    <span class="NLM_cas:pages">1208-1213</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">5-Methylthioadenosine phosphorylase (MTAP) is a key enzyme in the methionine salvage pathway.  The MTAP gene is frequently deleted in human cancers because of its chromosomal proximity to the tumor suppressor gene CDKN2A.  By interrogating data from a large-scale short hairpin RNA-mediated screen across 390 cancer cell line models, we found that the viability of MTAP-deficient cancer cells is impaired by depletion of the protein arginine methyltransferase PRMT5.  MTAP-deleted cells accumulate the metabolite methylthioadenosine (MTA), which we found to inhibit PRMT5 methyltransferase activity.  Deletion of MTAP in MTAP-proficient cells rendered them sensitive to PRMT5 depletion.  Conversely, reconstitution of MTAP in an MTAP-deficient cell line rescued PRMT5 dependence.  Thus, MTA accumulation in MTAP-deleted cancers creates a hypomorphic PRMT5 state that is selectively sensitized toward further PRMT5 inhibition.  Inhibitors of PRMT5 that leverage this dysregulated metabolic state merit further investigation as a potential therapy for MTAP/CDKN2A-deleted tumors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoNsuzYFo1qn7Vg90H21EOLACvtfcHk0lgBs3xhsciCGA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xjs12gtr0%253D&md5=d77531ea765db9762d6815840d39ee4a</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1126%2Fscience.aad5944&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.aad5944%26sid%3Dliteratum%253Aachs%26aulast%3DMavrakis%26aufirst%3DK.%26aulast%3DMcDonald%26aufirst%3DE.%2BR.%26aulast%3DSchlabach%26aufirst%3DM.%2BR.%26aulast%3DBilly%26aufirst%3DE.%26aulast%3DHoffman%26aufirst%3DG.%2BR.%26aulast%3DdeWeck%26aufirst%3DA.%26aulast%3DRuddy%26aufirst%3DD.%2BA.%26aulast%3DVenkatesan%26aufirst%3DK.%26aulast%3DMcAllister%26aufirst%3DG.%26aulast%3DdeBeaumont%26aufirst%3DR.%26aulast%3DHo%26aufirst%3DS.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DYan-Neale%26aufirst%3DY.%26aulast%3DYang%26aufirst%3DG.%2BZ.%26aulast%3DLin%26aufirst%3DF.%26aulast%3DYin%26aufirst%3DH.%26aulast%3DGao%26aufirst%3DH.%26aulast%3DKipp%26aufirst%3DD.%2BR.%26aulast%3DZhao%26aufirst%3DS.%2BP.%26aulast%3DMcNamara%26aufirst%3DJ.%2BT.%26aulast%3DSprague%26aufirst%3DE.%2BR.%26aulast%3DCho%26aufirst%3DY.%2BS.%26aulast%3DGu%26aufirst%3DJ.%26aulast%3DCrawford%26aufirst%3DK.%26aulast%3DCapka%26aufirst%3DV.%26aulast%3DHurov%26aufirst%3DK.%26aulast%3DPorter%26aufirst%3DJ.%2BA.%26aulast%3DTallarico%26aufirst%3DJ.%26aulast%3DMickanin%26aufirst%3DC.%26aulast%3DLees%26aufirst%3DE.%26aulast%3DPagliarini%26aufirst%3DR.%26aulast%3DKeen%26aufirst%3DN.%26aulast%3DSchmelzle%26aufirst%3DT.%26aulast%3DHofmann%26aufirst%3DF.%26aulast%3DStegmeier%26aufirst%3DF.%26aulast%3DSellers%26aufirst%3DW.%2BR.%26aulast%3DYu%26aufirst%3DJ.%26aulast%3DStump%26aufirst%3DM.%26aulast%3DYue%26aufirst%3DY.%26aulast%3DZheng%26aufirst%3DB.%26aulast%3DLin%26aufirst%3DY.%26aulast%3DCiccone%26aufirst%3DA.%2BC.%26aulast%3DVitari%26aufirst%3DA.%2BC.%26aulast%3DLai%26aufirst%3DA.%26atitle%3DDisordered%2520methionine%2520metabolism%2520in%2520MTAP%252FCDKN2A-deleted%2520cancers%2520leads%2520to%2520marked%2520dependence%2520on%2520PRMT5%26jtitle%3DScience%26date%3D2016%26volume%3D351%26spage%3D1208%26epage%3D1213%26doi%3D10.1126%2Fscience.aad5944" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Marjon, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cameron, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clasquin, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mandley, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kunii, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mcvay, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choe, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kernytsky, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gross, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Konteatis, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murtie, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blake, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Travins, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dorsch, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Biller, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marks, K. M.</span></span> <span> </span><span class="NLM_article-title">MTAP deletions in cancer create vulnerability to targeting of the MAT2A/PRMT5/RIOK1 axis</span>. <i>Cell Rep.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">574</span>– <span class="NLM_lpage">587</span>, <span class="refDoi"> DOI: 10.1016/j.celrep.2016.03.043</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=10.1016%2Fj.celrep.2016.03.043" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=27068473" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=1%3ACAS%3A528%3ADC%252BC28Xlsleltrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2016&pages=574-587&author=K.+Marjonauthor=M.+J.+Cameronauthor=P.+Quangauthor=M.+F.+Clasquinauthor=E.+Mandleyauthor=K.+Kuniiauthor=M.+Mcvayauthor=S.+Choeauthor=A.+Kernytskyauthor=S.+Grossauthor=Z.+Konteatisauthor=J.+Murtieauthor=M.+L.+Blakeauthor=J.+Travinsauthor=M.+Dorschauthor=S.+A.+Billerauthor=K.+M.+Marks&title=MTAP+deletions+in+cancer+create+vulnerability+to+targeting+of+the+MAT2A%2FPRMT5%2FRIOK1+axis&doi=10.1016%2Fj.celrep.2016.03.043"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">MTAP Deletions in Cancer Create Vulnerability to Targeting of the MAT2A/PRMT5/RIOK1 Axis</span></div><div class="casAuthors">Marjon, Katya; Cameron, Michael J.; Quang, Phong; Clasquin, Michelle F.; Mandley, Everton; Kunii, Kaiko; McVay, Michael; Choe, Sung; Kernytsky, Andrew; Gross, Stefan; Konteatis, Zenon; Murtie, Joshua; Blake, Michelle L.; Travins, Jeremy; Dorsch, Marion; Biller, Scott A.; Marks, Kevin M.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Reports</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">574-587</span>CODEN:
                <span class="NLM_cas:coden">CREED8</span>;
        ISSN:<span class="NLM_cas:issn">2211-1247</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Homozygous deletions of p16/CDKN2A are prevalent in cancer, and these mutations commonly involve co-deletion of adjacent genes, including methylthioadenosine phosphorylase (MTAP).  Here, we used shRNA screening and identified the metabolic enzyme, methionine adenosyltransferase II alpha (MAT2A), and the arginine methyltransferase, PRMT5, as vulnerable enzymes in cells with MTAP deletion.  Metabolomic and biochem. studies revealed a mechanistic basis for this synthetic lethality.  The MTAP substrate methylthioadenosine (MTA) accumulates upon MTAP loss.  Biochem. profiling of a methyltransferase enzyme panel revealed that MTA is a potent and selective inhibitor of PRMT5.  MTAP-deleted cells have reduced PRMT5 methylation activity and increased sensitivity to PRMT5 depletion.  MAT2A produces the PRMT5 substrate S-adenosylmethionine (SAM), and MAT2A depletion reduces growth and PRMT5 methylation activity selectively in MTAP-deleted cells.  Furthermore, this vulnerability extends to PRMT5 co-complex proteins such as RIOK1.  Thus, the unique biochem. features of PRMT5 create an axis of targets vulnerable in CDKN2A/MTAP-deleted cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGokW739dwptcbVg90H21EOLACvtfcHk0lgBs3xhsciCGA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xlsleltrk%253D&md5=73a6839e63b85a71f3661dc9e0c7b4ea</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1016%2Fj.celrep.2016.03.043&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.celrep.2016.03.043%26sid%3Dliteratum%253Aachs%26aulast%3DMarjon%26aufirst%3DK.%26aulast%3DCameron%26aufirst%3DM.%2BJ.%26aulast%3DQuang%26aufirst%3DP.%26aulast%3DClasquin%26aufirst%3DM.%2BF.%26aulast%3DMandley%26aufirst%3DE.%26aulast%3DKunii%26aufirst%3DK.%26aulast%3DMcvay%26aufirst%3DM.%26aulast%3DChoe%26aufirst%3DS.%26aulast%3DKernytsky%26aufirst%3DA.%26aulast%3DGross%26aufirst%3DS.%26aulast%3DKonteatis%26aufirst%3DZ.%26aulast%3DMurtie%26aufirst%3DJ.%26aulast%3DBlake%26aufirst%3DM.%2BL.%26aulast%3DTravins%26aufirst%3DJ.%26aulast%3DDorsch%26aufirst%3DM.%26aulast%3DBiller%26aufirst%3DS.%2BA.%26aulast%3DMarks%26aufirst%3DK.%2BM.%26atitle%3DMTAP%2520deletions%2520in%2520cancer%2520create%2520vulnerability%2520to%2520targeting%2520of%2520the%2520MAT2A%252FPRMT5%252FRIOK1%2520axis%26jtitle%3DCell%2520Rep.%26date%3D2016%26volume%3D15%26spage%3D574%26epage%3D587%26doi%3D10.1016%2Fj.celrep.2016.03.043" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kryukov, G. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, F. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruth, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paulk, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsherniak, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marlow, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vazquez, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weir, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fitzgerald, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanaka, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bielski, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dennis, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowley, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boehm, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Root, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Golub, T. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clish, C. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hahn, W. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garraway, L. A.</span></span> <span> </span><span class="NLM_article-title">MTAP deletion confers enhanced dependency on the PRMT5 arginine methyltransferase in cancer cells</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>351</i></span>,  <span class="NLM_fpage">1214</span>– <span class="NLM_lpage">1218</span>, <span class="refDoi"> DOI: 10.1126/science.aad5214</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=10.1126%2Fscience.aad5214" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=26912360" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=1%3ACAS%3A528%3ADC%252BC28Xjs12gtro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=351&publication_year=2016&pages=1214-1218&author=G.+V.+Kryukovauthor=F.+H.+Wilsonauthor=J.+R.+Ruthauthor=J.+Paulkauthor=A.+Tsherniakauthor=S.+E.+Marlowauthor=F.+Vazquezauthor=B.+A.+Weirauthor=M.+E.+Fitzgeraldauthor=M.+Tanakaauthor=C.+M.+Bielskiauthor=J.+M.+Scottauthor=C.+Dennisauthor=G.+S.+Cowleyauthor=J.+S.+Boehmauthor=D.+E.+Rootauthor=T.+R.+Golubauthor=C.+B.+Clishauthor=J.+E.+Bradnerauthor=W.+C.+Hahnauthor=L.+A.+Garraway&title=MTAP+deletion+confers+enhanced+dependency+on+the+PRMT5+arginine+methyltransferase+in+cancer+cells&doi=10.1126%2Fscience.aad5214"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">MTAP deletion confers enhanced dependency on the PRMT5 arginine methyltransferase in cancer cells</span></div><div class="casAuthors">Kryukov, Gregory V.; Wilson, Frederick H.; Ruth, Jason R.; Paulk, Joshiawa; Tsherniak, Aviad; Marlow, Sara E.; Vazquez, Francisca; Weir, Barbara A.; Fitzgerald, Mark E.; Tanaka, Minoru; Bielski, Craig M.; Scott, Justin M.; Dennis, Courtney; Cowley, Glenn S.; Boehm, Jesse S.; Root, David E.; Golub, Todd R.; Clish, Clary B.; Bradner, James E.; Hahn, William C.; Garraway, Levi A.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">351</span>
        (<span class="NLM_cas:issue">6278</span>),
    <span class="NLM_cas:pages">1214-1218</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">The discovery of cancer dependencies has the potential to inform therapeutic strategies and to identify putative drug targets.  Integrating data from comprehensive genomic profiling of cancer cell lines and from functional characterization of cancer cell dependencies, we discovered that loss of the enzyme methylthioadenosine phosphorylase (MTAP) confers a selective dependence on protein arginine methyltransferase 5 (PRMT5) and its binding partner WDR77.  MTAP is frequently lost due to its proximity to the commonly deleted tumor suppressor gene, CDKN2A.  We obsd. increased intracellular concns. of methylthioadenosine (MTA, the metabolite cleaved by MTAP) in cells harboring MTAP deletions.  Furthermore, MTA specifically inhibited PRMT5 enzymic activity.  Administration of either MTA or a small-mol. PRMT5 inhibitor showed a modest preferential impairment of cell viability for MTAP-null cancer cell lines compared with isogenic MTAP-expressing counterparts.  Together, our findings reveal PRMT5 as a potential vulnerability across multiple cancer lineages augmented by a common "passenger" genomic alteration.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr94dQPnMgrqbVg90H21EOLACvtfcHk0ljtxFlUH7wYvA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xjs12gtro%253D&md5=2b017a8c54972828c9024a9d85ff2e2d</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1126%2Fscience.aad5214&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.aad5214%26sid%3Dliteratum%253Aachs%26aulast%3DKryukov%26aufirst%3DG.%2BV.%26aulast%3DWilson%26aufirst%3DF.%2BH.%26aulast%3DRuth%26aufirst%3DJ.%2BR.%26aulast%3DPaulk%26aufirst%3DJ.%26aulast%3DTsherniak%26aufirst%3DA.%26aulast%3DMarlow%26aufirst%3DS.%2BE.%26aulast%3DVazquez%26aufirst%3DF.%26aulast%3DWeir%26aufirst%3DB.%2BA.%26aulast%3DFitzgerald%26aufirst%3DM.%2BE.%26aulast%3DTanaka%26aufirst%3DM.%26aulast%3DBielski%26aufirst%3DC.%2BM.%26aulast%3DScott%26aufirst%3DJ.%2BM.%26aulast%3DDennis%26aufirst%3DC.%26aulast%3DCowley%26aufirst%3DG.%2BS.%26aulast%3DBoehm%26aufirst%3DJ.%2BS.%26aulast%3DRoot%26aufirst%3DD.%2BE.%26aulast%3DGolub%26aufirst%3DT.%2BR.%26aulast%3DClish%26aufirst%3DC.%2BB.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DHahn%26aufirst%3DW.%2BC.%26aulast%3DGarraway%26aufirst%3DL.%2BA.%26atitle%3DMTAP%2520deletion%2520confers%2520enhanced%2520dependency%2520on%2520the%2520PRMT5%2520arginine%2520methyltransferase%2520in%2520cancer%2520cells%26jtitle%3DScience%26date%3D2016%26volume%3D351%26spage%3D1214%26epage%3D1218%26doi%3D10.1126%2Fscience.aad5214" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kaniskan, H. Ü.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Konze, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span> <span> </span><span class="NLM_article-title">Selective inhibitors of protein methyltransferases</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">1596</span>– <span class="NLM_lpage">1629</span>, <span class="refDoi"> DOI: 10.1021/jm501234a</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm501234a" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvFKqtbbJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=1596-1629&author=H.+%C3%9C.+Kaniskanauthor=K.+D.+Konzeauthor=J.+Jin&title=Selective+inhibitors+of+protein+methyltransferases&doi=10.1021%2Fjm501234a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Selective inhibitors of protein methyltransferases</span></div><div class="casAuthors">Kaniskan, H. Umit; Konze, Kyle D.; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1596-1629</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Mounting evidence suggests that protein methyltransferases (PMTs), which catalyze the methylation of histone and non-histone proteins, play a crucial role in diverse biol. processes and human diseases.  In particular, PMTs have been recognized as major players in regulating gene expression and chromatin state.  PMTs are divided into 2 categories: protein lysine methyltransferases (PKMTs) and protein arginine methyltransferases (PRMTs).  There has been a steadily growing interest in these enzymes as potential therapeutic targets and therefore discovery of PMT inhibitors has also been pursued increasingly over the past decade.  Here, the authors present a perspective on selective, small-mol. inhibitors of PMTs with an emphasis on their discovery, characterization, and applicability as chem. tools for deciphering the target PMTs' physiol. functions and involvement in human diseases.  The authors highlight the current state of PMT inhibitors and discuss future directions and opportunities for PMT inhibitor discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpEWF_XeLEY1rVg90H21EOLACvtfcHk0ljtxFlUH7wYvA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvFKqtbbJ&md5=8fb2b2d21d306c02a2da60a3fb88c430</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1021%2Fjm501234a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm501234a%26sid%3Dliteratum%253Aachs%26aulast%3DKaniskan%26aufirst%3DH.%2B%25C3%259C.%26aulast%3DKonze%26aufirst%3DK.%2BD.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DSelective%2520inhibitors%2520of%2520protein%2520methyltransferases%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D1596%26epage%3D1629%26doi%3D10.1021%2Fjm501234a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kaniskan, H.
Ü.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martini, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span> <span> </span><span class="NLM_article-title">Inhibitors of protein methyltransferases and demethylases</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>118</i></span>,  <span class="NLM_fpage">989</span>– <span class="NLM_lpage">1068</span>, <span class="refDoi"> DOI: 10.1021/acs.chemrev.6b00801</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.chemrev.6b00801" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=1%3ACAS%3A528%3ADC%252BC2sXkslynsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=118&publication_year=2018&pages=989-1068&author=H.%0A%C3%9C.+Kaniskanauthor=M.+L.+Martiniauthor=J.+Jin&title=Inhibitors+of+protein+methyltransferases+and+demethylases&doi=10.1021%2Facs.chemrev.6b00801"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibitors of Protein Methyltransferases and Demethylases</span></div><div class="casAuthors">Kaniskan, H. Umit; Martini, Michael L.; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">118</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">989-1068</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Post-translational modifications of histones by protein methyltransferases (PMTs) and histone demethylases (KDMs) play an important role in the regulation of gene expression and transcription and are implicated in cancer and many other diseases.  Many of these enzymes also target various nonhistone proteins impacting numerous crucial biol. pathways.  Given their key biol. functions and implications in human diseases, there has been a growing interest in assessing these enzymes as potential therapeutic targets.  Consequently, discovering and developing inhibitors of these enzymes has become a very active and fast-growing research area over the past decade.  In this review, the authors cover the discovery, characterization, and biol. application of inhibitors of PMTs and KDMs with emphasis on key advancements in the field.  The authors also discuss challenges, opportunities, and future directions in this emerging, exciting research field.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpJYjiYtw_n4rVg90H21EOLACvtfcHk0liufZ5Kdh_aIg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXkslynsL8%253D&md5=1ed520c6b0c42d2aa8a7f66b0a2d4e40</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1021%2Facs.chemrev.6b00801&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.chemrev.6b00801%26sid%3Dliteratum%253Aachs%26aulast%3DKaniskan%26aufirst%3DH.%2B%25C3%259C.%26aulast%3DMartini%26aufirst%3DM.%2BL.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DInhibitors%2520of%2520protein%2520methyltransferases%2520and%2520demethylases%26jtitle%3DChem.%2520Rev.%26date%3D2018%26volume%3D118%26spage%3D989%26epage%3D1068%26doi%3D10.1021%2Facs.chemrev.6b00801" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kaniskan, H. Ü.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span> <span> </span><span class="NLM_article-title">Recent progress in developing selective inhibitors of protein methyltransferases</span>. <i>Curr. Opin. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>39</i></span>,  <span class="NLM_fpage">100</span>– <span class="NLM_lpage">108</span>, <span class="refDoi"> DOI: 10.1016/j.cbpa.2017.06.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=10.1016%2Fj.cbpa.2017.06.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=28662389" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtVKnt73P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=2017&pages=100-108&author=H.+%C3%9C.+Kaniskanauthor=J.+Jin&title=Recent+progress+in+developing+selective+inhibitors+of+protein+methyltransferases&doi=10.1016%2Fj.cbpa.2017.06.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Recent progress in developing selective inhibitors of protein methyltransferases</span></div><div class="casAuthors">Kaniskan, H. Umit; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">39</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">100-108</span>CODEN:
                <span class="NLM_cas:coden">COCBF4</span>;
        ISSN:<span class="NLM_cas:issn">1367-5931</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Mounting evidence suggests that protein methyltransferases (PMTs), which catalyze methylation of histones as well as non-histone proteins, play a crucial role in diverse biol. pathways and human diseases.  In particular, PMTs have been recognized as major players in regulating gene expression and chromatin state.  There has been an increasingly growing interest in these enzymes as potential therapeutic targets and over the past two years tremendous progress has been made in the discovery of selective, small mol. inhibitors of protein lysine and arginine methyltransferases.  Inhibitors of PMTs have been used extensively in oncol. studies as tool compds., and inhibitors of EZH2, DOT1L and PRMT5 are currently in clin. trials.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoL3rSZhrkACbVg90H21EOLACvtfcHk0liufZ5Kdh_aIg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtVKnt73P&md5=ce42299d3bf38bc9898892a0364eca36</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1016%2Fj.cbpa.2017.06.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cbpa.2017.06.013%26sid%3Dliteratum%253Aachs%26aulast%3DKaniskan%26aufirst%3DH.%2B%25C3%259C.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DRecent%2520progress%2520in%2520developing%2520selective%2520inhibitors%2520of%2520protein%2520methyltransferases%26jtitle%3DCurr.%2520Opin.%2520Chem.%2520Biol.%26date%3D2017%26volume%3D39%26spage%3D100%26epage%3D108%26doi%3D10.1016%2Fj.cbpa.2017.06.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qian, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ho, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, Y. G.</span></span> <span> </span><span class="NLM_article-title">Small molecule inhibitors of protein arginine methyltransferases</span>. <i>Expert Opin. Invest. Drugs</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">335</span>– <span class="NLM_lpage">358</span>, <span class="refDoi"> DOI: 10.1517/13543784.2016.1144747</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=10.1517%2F13543784.2016.1144747" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2016&pages=335-358&author=H.+Huauthor=K.+Qianauthor=M.+C.+Hoauthor=Y.+G.+Zheng&title=Small+molecule+inhibitors+of+protein+arginine+methyltransferases&doi=10.1517%2F13543784.2016.1144747"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1517%2F13543784.2016.1144747&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1517%252F13543784.2016.1144747%26sid%3Dliteratum%253Aachs%26aulast%3DHu%26aufirst%3DH.%26aulast%3DQian%26aufirst%3DK.%26aulast%3DHo%26aufirst%3DM.%2BC.%26aulast%3DZheng%26aufirst%3DY.%2BG.%26atitle%3DSmall%2520molecule%2520inhibitors%2520of%2520protein%2520arginine%2520methyltransferases%26jtitle%3DExpert%2520Opin.%2520Invest.%2520Drugs%26date%3D2016%26volume%3D25%26spage%3D335%26epage%3D358%26doi%3D10.1517%2F13543784.2016.1144747" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref55"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref55'); return false;" data-citation="" class="refNumLink">55</a></strong><div class="NLM_citation" id="cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Luo, M. K.</span></span> <span> </span><span class="NLM_article-title">Inhibitors of protein methyltransferases as chemical tools</span>. <i>Epigenomics</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">1327</span>– <span class="NLM_lpage">1338</span>, <span class="refDoi"> DOI: 10.2217/epi.15.87</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=10.2217%2Fepi.15.87" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=26646500" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=1%3ACAS%3A528%3ADC%252BC2MXitVejtbvN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2015&pages=1327-1338&author=M.+K.+Luo&title=Inhibitors+of+protein+methyltransferases+as+chemical+tools&doi=10.2217%2Fepi.15.87"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibitors of protein methyltransferases as chemical tools</span></div><div class="casAuthors">Luo, Minkui</div><div class="citationInfo"><span class="NLM_cas:title">Epigenomics</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1327-1338</span>CODEN:
                <span class="NLM_cas:coden">EPIGC7</span>;
        ISSN:<span class="NLM_cas:issn">1750-192X</span>.
    
            (<span class="NLM_cas:orgname">Future Medicine Ltd.</span>)
        </div><div class="casAbstract">Protein methyltransferases (PMT)s play essential roles in many biol. processes through methylation of histones and diverse nonhistone substrates.  Dysregulation of these enzymes has been implicated in many diseases including cancers.  While PMT-assocd. biol. can be probed via genetic perturbation, this approach targets full-length PMTs rather than their methyltransferase activities and often lacks temporal, spatial and dose controls (timing, location and amt. of dosed compds.).  By contrast, small-mol. inhibitors of PMTs can be designed to specifically target the methyltransferase domains in a temporal, spatial and dose-dependent manner.  This utility has motivated the development of hundreds of PMT inhibitors, but meanwhile can make it challenging to select the most suitable PMT inhibitors to interrogate PMT-assocd. biol.  This perspective aims to provide timely guidance to evaluate these PMT inhibitors in their relevant biol. contexts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpNoz6wEWA3UrVg90H21EOLACvtfcHk0lh6DJcOfvDfzQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXitVejtbvN&md5=fb64069b1c54f27a35e604318fee99be</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.2217%2Fepi.15.87&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2217%252Fepi.15.87%26sid%3Dliteratum%253Aachs%26aulast%3DLuo%26aufirst%3DM.%2BK.%26atitle%3DInhibitors%2520of%2520protein%2520methyltransferases%2520as%2520chemical%2520tools%26jtitle%3DEpigenomics%26date%3D2015%26volume%3D7%26spage%3D1327%26epage%3D1338%26doi%3D10.2217%2Fepi.15.87" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref56"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref56'); return false;" data-citation="" class="refNumLink">56</a></strong><div class="NLM_citation" id="cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chan-Penebre, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuplast, K. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Majer, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boriack-Sjodin, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wigle, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnston, L. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rioux, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Munchhof, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacques, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">West, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lingaraj, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stickland, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ribich, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raimondi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waters, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pollock, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barbash, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pappalardi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ho, T. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nurse, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oza, K. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gallagher, K. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kruger, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moyer, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Copeland, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chesworth, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duncan, K. W.</span></span> <span> </span><span class="NLM_article-title">A selective inhibitor of PRMT5 with in vivo and in vitro potency in MCL models</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">432</span>– <span class="NLM_lpage">437</span>, <span class="refDoi"> DOI: 10.1038/nchembio.1810</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=10.1038%2Fnchembio.1810" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=25915199" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=1%3ACAS%3A528%3ADC%252BC2MXotFartbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2015&pages=432-437&author=E.+Chan-Penebreauthor=K.+G.+Kuplastauthor=C.+R.+Majerauthor=P.+A.+Boriack-Sjodinauthor=T.+J.+Wigleauthor=L.+D.+Johnstonauthor=N.+Riouxauthor=M.+J.+Munchhofauthor=L.+Jinauthor=S.+L.+Jacquesauthor=K.+A.+Westauthor=T.+Lingarajauthor=K.+Sticklandauthor=S.+A.+Ribichauthor=A.+Raimondiauthor=M.+P.+Scottauthor=N.+J.+Watersauthor=R.+M.+Pollockauthor=J.+J.+Smithauthor=O.+Barbashauthor=M.+Pappalardiauthor=T.+F.+Hoauthor=K.+Nurseauthor=K.+P.+Ozaauthor=K.+T.+Gallagherauthor=R.+Krugerauthor=M.+P.+Moyerauthor=R.+A.+Copelandauthor=R.+Chesworthauthor=K.+W.+Duncan&title=A+selective+inhibitor+of+PRMT5+with+in+vivo+and+in+vitro+potency+in+MCL+models&doi=10.1038%2Fnchembio.1810"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">A selective inhibitor of PRMT5 with in vivo and in vitro potency in MCL models</span></div><div class="casAuthors">Chan-Penebre, Elayne; Kuplast, Kristy G.; Majer, Christina R.; Boriack-Sjodin, P. Ann; Wigle, Tim J.; Johnston, L. Danielle; Rioux, Nathalie; Munchhof, Michael J.; Jin, Lei; Jacques, Suzanne L.; West, Kip A.; Lingaraj, Trupti; Stickland, Kimberly; Ribich, Scott A.; Raimondi, Alejandra; Scott, Margaret Porter; Waters, Nigel J.; Pollock, Roy M.; Smith, Jesse J.; Barbash, Olena; Pappalardi, Melissa; Ho, Thau F.; Nurse, Kelvin; Oza, Khyati P.; Gallagher, Kathleen T.; Kruger, Ryan; Moyer, Mikel P.; Copeland, Robert A.; Chesworth, Richard; Duncan, Kenneth W.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">432-437</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Protein arginine methyltransferase-5 (PRMT5) is reported to have a role in diverse cellular processes, including tumorigenesis, and its overexpression is obsd. in cell lines and primary patient samples derived from lymphomas, particularly mantle cell lymphoma (MCL).  Here we describe the identification and characterization of a potent and selective inhibitor of PRMT5 with antiproliferative effects in both in vitro and in vivo models of MCL.  EPZ015666 (GSK3235025) is an orally available inhibitor of PRMT5 enzymic activity in biochem. assays with a half-maximal inhibitory concn. (IC50) of 22 nM and broad selectivity against a panel of other histone methyltransferases.  Treatment of MCL cell lines with EPZ015666 led to inhibition of SmD3 methylation and cell death, with IC50 values in the nanomolar range.  Oral dosing with EPZ015666 demonstrated dose-dependent antitumor activity in multiple MCL xenograft models.  EPZ015666 represents a validated chem. probe for further study of PRMT5 biol. and arginine methylation in cancer and other diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoJTtcvcqvOJLVg90H21EOLACvtfcHk0lh6DJcOfvDfzQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXotFartbo%253D&md5=bf0686e1c2d9d96ac5deef22c0bf80b7</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1810&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1810%26sid%3Dliteratum%253Aachs%26aulast%3DChan-Penebre%26aufirst%3DE.%26aulast%3DKuplast%26aufirst%3DK.%2BG.%26aulast%3DMajer%26aufirst%3DC.%2BR.%26aulast%3DBoriack-Sjodin%26aufirst%3DP.%2BA.%26aulast%3DWigle%26aufirst%3DT.%2BJ.%26aulast%3DJohnston%26aufirst%3DL.%2BD.%26aulast%3DRioux%26aufirst%3DN.%26aulast%3DMunchhof%26aufirst%3DM.%2BJ.%26aulast%3DJin%26aufirst%3DL.%26aulast%3DJacques%26aufirst%3DS.%2BL.%26aulast%3DWest%26aufirst%3DK.%2BA.%26aulast%3DLingaraj%26aufirst%3DT.%26aulast%3DStickland%26aufirst%3DK.%26aulast%3DRibich%26aufirst%3DS.%2BA.%26aulast%3DRaimondi%26aufirst%3DA.%26aulast%3DScott%26aufirst%3DM.%2BP.%26aulast%3DWaters%26aufirst%3DN.%2BJ.%26aulast%3DPollock%26aufirst%3DR.%2BM.%26aulast%3DSmith%26aufirst%3DJ.%2BJ.%26aulast%3DBarbash%26aufirst%3DO.%26aulast%3DPappalardi%26aufirst%3DM.%26aulast%3DHo%26aufirst%3DT.%2BF.%26aulast%3DNurse%26aufirst%3DK.%26aulast%3DOza%26aufirst%3DK.%2BP.%26aulast%3DGallagher%26aufirst%3DK.%2BT.%26aulast%3DKruger%26aufirst%3DR.%26aulast%3DMoyer%26aufirst%3DM.%2BP.%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26aulast%3DChesworth%26aufirst%3DR.%26aulast%3DDuncan%26aufirst%3DK.%2BW.%26atitle%3DA%2520selective%2520inhibitor%2520of%2520PRMT5%2520with%2520in%2520vivo%2520and%2520in%2520vitro%2520potency%2520in%2520MCL%2520models%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2015%26volume%3D11%26spage%3D432%26epage%3D437%26doi%3D10.1038%2Fnchembio.1810" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref57"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref57'); return false;" data-citation="" class="refNumLink">57</a></strong><div class="NLM_citation" id="cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Duncan, K. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rioux, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boriack-Sjodin, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Munchhof, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reiter, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Majer, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnston, L. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan-Penebre, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuplast, K. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Porter
Scott, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pollock, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waters, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moyer, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Copeland, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chesworth, R.</span></span> <span> </span><span class="NLM_article-title">Structure and property guided design in the identification of PRMT5 tool compound EPZ015666</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">162</span>– <span class="NLM_lpage">166</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.5b00380</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.5b00380" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvFGnsr3L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=162-166&author=K.+W.+Duncanauthor=N.+Riouxauthor=P.+A.+Boriack-Sjodinauthor=M.+J.+Munchhofauthor=L.+A.+Reiterauthor=C.+R.+Majerauthor=L.+Jinauthor=L.+D.+Johnstonauthor=E.+Chan-Penebreauthor=K.+G.+Kuplastauthor=M.+Porter%0AScottauthor=R.+M.+Pollockauthor=N.+J.+Watersauthor=J.+J.+Smithauthor=M.+P.+Moyerauthor=R.+A.+Copelandauthor=R.+Chesworth&title=Structure+and+property+guided+design+in+the+identification+of+PRMT5+tool+compound+EPZ015666&doi=10.1021%2Facsmedchemlett.5b00380"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">Structure and Property Guided Design in the Identification of PRMT5 Tool Compound EPZ015666</span></div><div class="casAuthors">Duncan, Kenneth W.; Rioux, Nathalie; Boriack-Sjodin, P. Ann; Munchhof, Michael J.; Reiter, Lawrence A.; Majer, Christina R.; Jin, Lei; Johnston, L. Danielle; Chan-Penebre, Elayne; Kuplast, Kristy G.; Porter Scott, Margaret; Pollock, Roy M.; Waters, Nigel J.; Smith, Jesse J.; Moyer, Mikel P.; Copeland, Robert A.; Chesworth, Richard</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">162-166</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The recent publication of a potent and selective inhibitor of protein methyltransferase 5 (PRMT5) provides the scientific community with in vivo-active tool compd. I (EPZ015666, GSK3235025) to probe the underlying pharmacol. of this key enzyme.  Herein, the authors report the design and optimization strategies employed on an initial hit compd. with poor in vitro clearance to yield in vivo tool compd. I and an addnl. potent in vitro tool mol. II (GSK3203591).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq2V7LiQgVZjrVg90H21EOLACvtfcHk0lhXS4gaJnj2_g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvFGnsr3L&md5=e7ac92c2d985da4994e9bb959adededf</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.5b00380&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.5b00380%26sid%3Dliteratum%253Aachs%26aulast%3DDuncan%26aufirst%3DK.%2BW.%26aulast%3DRioux%26aufirst%3DN.%26aulast%3DBoriack-Sjodin%26aufirst%3DP.%2BA.%26aulast%3DMunchhof%26aufirst%3DM.%2BJ.%26aulast%3DReiter%26aufirst%3DL.%2BA.%26aulast%3DMajer%26aufirst%3DC.%2BR.%26aulast%3DJin%26aufirst%3DL.%26aulast%3DJohnston%26aufirst%3DL.%2BD.%26aulast%3DChan-Penebre%26aufirst%3DE.%26aulast%3DKuplast%26aufirst%3DK.%2BG.%26aulast%3DPorter%2BScott%26aufirst%3DM.%26aulast%3DPollock%26aufirst%3DR.%2BM.%26aulast%3DWaters%26aufirst%3DN.%2BJ.%26aulast%3DSmith%26aufirst%3DJ.%2BJ.%26aulast%3DMoyer%26aufirst%3DM.%2BP.%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26aulast%3DChesworth%26aufirst%3DR.%26atitle%3DStructure%2520and%2520property%2520guided%2520design%2520in%2520the%2520identification%2520of%2520PRMT5%2520tool%2520compound%2520EPZ015666%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2016%26volume%3D7%26spage%3D162%26epage%3D166%26doi%3D10.1021%2Facsmedchemlett.5b00380" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref58"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref58'); return false;" data-citation="" class="refNumLink">58</a></strong><div class="NLM_citation" id="cit58"><span>ClinicalTrials.gov. <a href="https://clinicaltrials.gov/ct2/show/NCT02783300" class="extLink">https://clinicaltrials.gov/ct2/show/NCT02783300</a> (accessed March 24, <span class="NLM_year">2018</span>).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=ClinicalTrials.gov.+https%3A%2F%2Fclinicaltrials.gov%2Fct2%2Fshow%2FNCT02783300+%28accessed+March+24%2C+2018%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26date%3D2018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref59"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref59'); return false;" data-citation="" class="refNumLink">59</a></strong><div class="NLM_citation" id="cit59"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Falk, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Foitzik, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allan, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">deSilva, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bozikis, Y. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nikac, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walker, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Camerino, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morrow, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stupple, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lagiakos, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pinson, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lessene, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kersten, W. J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ganame, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holmes, I. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lunniss, G. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hermans, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parker, M. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thistlethwaite, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">White, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Charman, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pilling, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Monahan, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grusovin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peat, T. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sonderegger, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toulmin, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jane, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Curtis, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stupple, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Street, I. P.</span></span> <span> </span><span class="NLM_article-title">PRMT5 inhibitors as novel treatment for cancers</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>75</i></span> (<span class="NLM_issue">15, Suppl.</span>),  <span class="NLM_fpage">5371</span>, (Abstract in Proceedings of the 106th Annual Meeting of the American Association for Cancer Research; Apr 18–22, 2015; Philadelphia, PA) <span class="refDoi"> DOI: 10.1158/1538-7445.AM2015-5371</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=10.1158%2F1538-7445.AM2015-5371" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=75&publication_year=2015&pages=5371&issue=15%2C+Suppl.&author=H.+Falkauthor=R.+C.+Foitzikauthor=E.+Allanauthor=M.+deSilvaauthor=H.+Yangauthor=Y.+E.+Bozikisauthor=M.+Nikacauthor=S.+R.+Walkerauthor=M.+A.+Camerinoauthor=B.+J.+Morrowauthor=A.+E.+Stuppleauthor=R.+Lagiakosauthor=J.+A.+Pinsonauthor=R.+Lesseneauthor=W.+J.+A.+Kerstenauthor=D.+G.+Ganameauthor=I.+P.+Holmesauthor=G.+E.+Lunnissauthor=M.+Chungauthor=S.+J.+Hermansauthor=M.+W.+Parkerauthor=A.+Thistlethwaiteauthor=K.+L.+Whiteauthor=S.+A.+Charmanauthor=P.+Pillingauthor=B.+J.+Monahanauthor=J.+Grusovinauthor=T.+S.+Peatauthor=S.+Sondereggerauthor=E.+Toulminauthor=S.+M.+Janeauthor=D.+J.+Curtisauthor=P.+A.+Stuppleauthor=I.+P.+Street&title=PRMT5+inhibitors+as+novel+treatment+for+cancers&doi=10.1158%2F1538-7445.AM2015-5371"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1158%2F1538-7445.AM2015-5371&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1538-7445.AM2015-5371%26sid%3Dliteratum%253Aachs%26aulast%3DFalk%26aufirst%3DH.%26aulast%3DFoitzik%26aufirst%3DR.%2BC.%26aulast%3DAllan%26aufirst%3DE.%26aulast%3DdeSilva%26aufirst%3DM.%26aulast%3DYang%26aufirst%3DH.%26aulast%3DBozikis%26aufirst%3DY.%2BE.%26aulast%3DNikac%26aufirst%3DM.%26aulast%3DWalker%26aufirst%3DS.%2BR.%26aulast%3DCamerino%26aufirst%3DM.%2BA.%26aulast%3DMorrow%26aufirst%3DB.%2BJ.%26aulast%3DStupple%26aufirst%3DA.%2BE.%26aulast%3DLagiakos%26aufirst%3DR.%26aulast%3DPinson%26aufirst%3DJ.%2BA.%26aulast%3DLessene%26aufirst%3DR.%26aulast%3DKersten%26aufirst%3DW.%2BJ.%2BA.%26aulast%3DGaname%26aufirst%3DD.%2BG.%26aulast%3DHolmes%26aufirst%3DI.%2BP.%26aulast%3DLunniss%26aufirst%3DG.%2BE.%26aulast%3DChung%26aufirst%3DM.%26aulast%3DHermans%26aufirst%3DS.%2BJ.%26aulast%3DParker%26aufirst%3DM.%2BW.%26aulast%3DThistlethwaite%26aufirst%3DA.%26aulast%3DWhite%26aufirst%3DK.%2BL.%26aulast%3DCharman%26aufirst%3DS.%2BA.%26aulast%3DPilling%26aufirst%3DP.%26aulast%3DMonahan%26aufirst%3DB.%2BJ.%26aulast%3DGrusovin%26aufirst%3DJ.%26aulast%3DPeat%26aufirst%3DT.%2BS.%26aulast%3DSonderegger%26aufirst%3DS.%26aulast%3DToulmin%26aufirst%3DE.%26aulast%3DJane%26aufirst%3DS.%2BM.%26aulast%3DCurtis%26aufirst%3DD.%2BJ.%26aulast%3DStupple%26aufirst%3DP.%2BA.%26aulast%3DStreet%26aufirst%3DI.%2BP.%26atitle%3DPRMT5%2520inhibitors%2520as%2520novel%2520treatment%2520for%2520cancers%26jtitle%3DCancer%2520Res.%26date%3D2015%26volume%3D75%26issue%3D15%252C%2520Suppl%26spage%3D5371%26doi%3D10.1158%2F1538-7445.AM2015-5371" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref60"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref60'); return false;" data-citation="" class="refNumLink">60</a></strong><div class="NLM_citation" id="cit60"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bergman, Y. E.</span>; <span class="NLM_string-name">Camerino, M. A.</span>; <span class="NLM_string-name">Stupple, P. A.</span></span> <span> </span><span class="NLM_article-title">Benzopiperidine Derivatives and Their Use in the Treatment of Cancer and Hemoglobinopathies</span>. <span class="NLM_patent">WO2017153520 A1</span>, <span class="NLM_year">2017</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2017&author=Y.+E.+Bergman&author=M.+A.+Camerino&author=P.+A.+Stupple&title=Benzopiperidine+Derivatives+and+Their+Use+in+the+Treatment+of+Cancer+and+Hemoglobinopathies"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DBergman%26aufirst%3DY.%2BE.%26atitle%3DBenzopiperidine%2520Derivatives%2520and%2520Their%2520Use%2520in%2520the%2520Treatment%2520of%2520Cancer%2520and%2520Hemoglobinopathies%26date%3D2017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref61"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref61'); return false;" data-citation="" class="refNumLink">61</a></strong><div class="NLM_citation" id="cit61"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Stupple, P. A.</span></span> <span> </span><span class="NLM_article-title">Pyridine Derivatives and Their Use in the Treatment of Cancer and Hemoglobinopathies</span>. <span class="NLM_patent">WO2017153521 A1</span>, <span class="NLM_year">2017</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2017&author=P.+A.+Stupple&title=Pyridine+Derivatives+and+Their+Use+in+the+Treatment+of+Cancer+and+Hemoglobinopathies"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DStupple%26aufirst%3DP.%2BA.%26atitle%3DPyridine%2520Derivatives%2520and%2520Their%2520Use%2520in%2520the%2520Treatment%2520of%2520Cancer%2520and%2520Hemoglobinopathies%26date%3D2017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref62"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref62'); return false;" data-citation="" class="refNumLink">62</a></strong><div class="NLM_citation" id="cit62"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ye, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mao, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xing, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, M.</span></span> <span> </span><span class="NLM_article-title">Discovery and optimization of selective inhibitors of protein arginine methyltransferase 5 by docking-based virtual screening</span>. <i>Org. Biomol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">3648</span>– <span class="NLM_lpage">3661</span>, <span class="refDoi"> DOI: 10.1039/C7OB00070G</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=10.1039%2FC7OB00070G" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=28397890" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=1%3ACAS%3A528%3ADC%252BC2sXltl2mtr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2017&pages=3648-3661&author=Y.+Yeauthor=B.+Zhangauthor=R.+Maoauthor=C.+Zhangauthor=Y.+Wangauthor=J.+Xingauthor=Y.+C.+Liuauthor=X.+Luoauthor=H.+Dingauthor=Y.+Yangauthor=B.+Zhouauthor=H.+Jiangauthor=K.+Chenauthor=C.+Luoauthor=M.+Zheng&title=Discovery+and+optimization+of+selective+inhibitors+of+protein+arginine+methyltransferase+5+by+docking-based+virtual+screening&doi=10.1039%2FC7OB00070G"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit62R"><div class="casContent"><span class="casTitleNuber">62</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery and optimization of selective inhibitors of protein arginine methyltransferase 5 by docking-based virtual screening</span></div><div class="casAuthors">Ye, Yan; Zhang, Bidong; Mao, Ruifeng; Zhang, Chenhua; Wang, Yulan; Xing, Jing; Liu, Yu-Chih; Luo, Xiaomin; Ding, Hong; Yang, Yaxi; Zhou, Bing; Jiang, Hualiang; Chen, Kaixian; Luo, Cheng; Zheng, Mingyue</div><div class="citationInfo"><span class="NLM_cas:title">Organic & Biomolecular Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">3648-3661</span>CODEN:
                <span class="NLM_cas:coden">OBCRAK</span>;
        ISSN:<span class="NLM_cas:issn">1477-0520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Protein arginine methyltransferase 5 (PRMT5) is a type II PRMT enzyme crit. for diverse cellular processes and different types of cancers.  Many efforts have been made to discover novel scaffold PRMT5 inhibitors.  Herein, the authors report the discovery of DC_P33 as a hit compd. of PRMT5 inhibitor, identified by mol. docking based virtual screening and 3H-labeled radioactive methylation assays.  Structure-activity relationship (SAR) anal. was performed on the analogs of DC_P33 and then structural modifications were done to improve its activity.  Among the derivs., the compd. DC_C01 displayed an IC50 value of 2.8 μM, and good selectivity toward PRMT1, EZH2 and DNMT3A.  Moreover, DC_C01 exhibited anti-proliferation activities against Z-138, Maver-1, and Jeko-1 cancer cells with EC50 values of 12 μM, 12 μM, and 10.5 μM, resp.  Taken together, these results contribute to the development of specific inhibitors against PRMT5 and cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqkWfdAd-COtbVg90H21EOLACvtfcHk0ljYD6zlgBrbSw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXltl2mtr0%253D&md5=ead3b89fa64875e2dbf47c3a99714564</span></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1039%2FC7OB00070G&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC7OB00070G%26sid%3Dliteratum%253Aachs%26aulast%3DYe%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DB.%26aulast%3DMao%26aufirst%3DR.%26aulast%3DZhang%26aufirst%3DC.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DXing%26aufirst%3DJ.%26aulast%3DLiu%26aufirst%3DY.%2BC.%26aulast%3DLuo%26aufirst%3DX.%26aulast%3DDing%26aufirst%3DH.%26aulast%3DYang%26aufirst%3DY.%26aulast%3DZhou%26aufirst%3DB.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DChen%26aufirst%3DK.%26aulast%3DLuo%26aufirst%3DC.%26aulast%3DZheng%26aufirst%3DM.%26atitle%3DDiscovery%2520and%2520optimization%2520of%2520selective%2520inhibitors%2520of%2520protein%2520arginine%2520methyltransferase%25205%2520by%2520docking-based%2520virtual%2520screening%26jtitle%3DOrg.%2520Biomol.%2520Chem.%26date%3D2017%26volume%3D15%26spage%3D3648%26epage%3D3661%26doi%3D10.1039%2FC7OB00070G" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref63"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref63'); return false;" data-citation="" class="refNumLink">63</a></strong><div class="NLM_citation" id="cit63"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Couture, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hauk, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blackburn, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trievel, R. C.</span></span> <span> </span><span class="NLM_article-title">Catalytic roles for carbon–oxygen hydrogen bonding in SET domain lysine methyltransferases</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>281</i></span>,  <span class="NLM_fpage">19280</span>– <span class="NLM_lpage">19287</span>, <span class="refDoi"> DOI: 10.1074/jbc.M602257200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=10.1074%2Fjbc.M602257200" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=281&publication_year=2006&pages=19280-19287&author=J.+F.+Coutureauthor=G.+Haukauthor=M.+J.+Thompsonauthor=G.+M.+Blackburnauthor=R.+C.+Trievel&title=Catalytic+roles+for+carbon%E2%80%93oxygen+hydrogen+bonding+in+SET+domain+lysine+methyltransferases&doi=10.1074%2Fjbc.M602257200"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M602257200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M602257200%26sid%3Dliteratum%253Aachs%26aulast%3DCouture%26aufirst%3DJ.%2BF.%26aulast%3DHauk%26aufirst%3DG.%26aulast%3DThompson%26aufirst%3DM.%2BJ.%26aulast%3DBlackburn%26aufirst%3DG.%2BM.%26aulast%3DTrievel%26aufirst%3DR.%2BC.%26atitle%3DCatalytic%2520roles%2520for%2520carbon%25E2%2580%2593oxygen%2520hydrogen%2520bonding%2520in%2520SET%2520domain%2520lysine%2520methyltransferases%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2006%26volume%3D281%26spage%3D19280%26epage%3D19287%26doi%3D10.1074%2Fjbc.M602257200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref64"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref64'); return false;" data-citation="" class="refNumLink">64</a></strong><div class="NLM_citation" id="cit64"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chin, H. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patnaik, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Esteve, P. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacobsen, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pradhan, S.</span></span> <span> </span><span class="NLM_article-title">Catalytic properties and kinetic mechanism of human recombinant lys-9 histone H3 methyltransferase SUV39H1: participation of the chromodomain in enzymatic catalysis</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>45</i></span>,  <span class="NLM_fpage">3272</span>– <span class="NLM_lpage">3284</span>, <span class="refDoi"> DOI: 10.1021/bi051997r</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi051997r" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2006&pages=3272-3284&author=H.+G.+Chinauthor=D.+Patnaikauthor=P.+O.+Esteveauthor=S.+E.+Jacobsenauthor=S.+Pradhan&title=Catalytic+properties+and+kinetic+mechanism+of+human+recombinant+lys-9+histone+H3+methyltransferase+SUV39H1%3A+participation+of+the+chromodomain+in+enzymatic+catalysis&doi=10.1021%2Fbi051997r"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=10.1021%2Fbi051997r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi051997r%26sid%3Dliteratum%253Aachs%26aulast%3DChin%26aufirst%3DH.%2BG.%26aulast%3DPatnaik%26aufirst%3DD.%26aulast%3DEsteve%26aufirst%3DP.%2BO.%26aulast%3DJacobsen%26aufirst%3DS.%2BE.%26aulast%3DPradhan%26aufirst%3DS.%26atitle%3DCatalytic%2520properties%2520and%2520kinetic%2520mechanism%2520of%2520human%2520recombinant%2520lys-9%2520histone%2520H3%2520methyltransferase%2520SUV39H1%253A%2520participation%2520of%2520the%2520chromodomain%2520in%2520enzymatic%2520catalysis%26jtitle%3DBiochemistry%26date%3D2006%26volume%3D45%26spage%3D3272%26epage%3D3284%26doi%3D10.1021%2Fbi051997r" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref65"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref65'); return false;" data-citation="" class="refNumLink">65</a></strong><div class="NLM_citation" id="cit65"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Smil, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eram, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kennedy, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szewczyk, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barsyte-Lovejoy, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vedadi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schapira, M.</span></span> <span> </span><span class="NLM_article-title">Discovery of a dual PRMT5-PRMT7 inhibitor</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">408</span>– <span class="NLM_lpage">412</span>, <span class="refDoi"> DOI: 10.1021/ml500467h</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ml500467h" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=1%3ACAS%3A528%3ADC%252BC2MXjs1Ggu7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=408-412&author=D.+Smilauthor=M.+S.+Eramauthor=F.+Liauthor=S.+Kennedyauthor=M.+M.+Szewczykauthor=P.+J.+Brownauthor=D.+Barsyte-Lovejoyauthor=C.+H.+Arrowsmithauthor=M.+Vedadiauthor=M.+Schapira&title=Discovery+of+a+dual+PRMT5-PRMT7+inhibitor&doi=10.1021%2Fml500467h"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit65R"><div class="casContent"><span class="casTitleNuber">65</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a Dual PRMT5-PRMT7 Inhibitor</span></div><div class="casAuthors">Smil, David; Eram, Mohammad S.; Li, Fengling; Kennedy, Steven; Szewczyk, Magdalena M.; Brown, Peter J.; Barsyte-Lovejoy, Dalia; Arrowsmith, Cheryl H.; Vedadi, Masoud; Schapira, Matthieu</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">408-412</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The protein arginine methyltransferases PRMT7 and PRMT5, resp., monomethylate and sym. dimethylate arginine side-chains of proteins involved in diverse cellular mechanisms, including chromatin-mediated control of gene transcription, splicing, and the RAS to ERK transduction cascade.  It is believed that PRMT5 and PRMT7 act in conjunction to methylate their substrates, and genetic deletions support the notion that these enzymes derepress cell proliferation and migration in cancer.  Using available structures of PRMT5, we designed DS-437, a PRMT5 inhibitor with an IC50 value of 6 μM against both PRMT5 and PRMT7 that is inactive against 29 other human protein-, DNA-, and RNA-methyltransferases and inhibits sym. dimethylation of PRMT5 substrates in cells.  This compd. behaves as a cofactor competitor and represents a valid scaffold to interrogate the potential of the PRMT5-PRMT7 axis as a target for therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq9J078gEcjRLVg90H21EOLACvtfcHk0lg_O8kenV4SMA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXjs1Ggu7k%253D&md5=8eefe93068418c61881850d3b4d39676</span></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1021%2Fml500467h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fml500467h%26sid%3Dliteratum%253Aachs%26aulast%3DSmil%26aufirst%3DD.%26aulast%3DEram%26aufirst%3DM.%2BS.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DKennedy%26aufirst%3DS.%26aulast%3DSzewczyk%26aufirst%3DM.%2BM.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DSchapira%26aufirst%3DM.%26atitle%3DDiscovery%2520of%2520a%2520dual%2520PRMT5-PRMT7%2520inhibitor%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2015%26volume%3D6%26spage%3D408%26epage%3D412%26doi%3D10.1021%2Fml500467h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref66"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref66'); return false;" data-citation="" class="refNumLink">66</a></strong><div class="NLM_citation" id="cit66"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bonday, Z. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cortez, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grogan, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Antonysamy, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weichert, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bocchinfuso, W. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kennedy, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mader, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eram, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szewczyk, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barsyte-Lovejoy, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vedadi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guccione, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campbell, R. M.</span></span> <span> </span><span class="NLM_article-title">LLY-283, a potent and selective inhibitor of arginine methyltransferase 5, PRMT5, with antitumor activity</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.8b00014</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.8b00014" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2018&author=Z.+Q.+Bondayauthor=G.+S.+Cortezauthor=M.+J.+Groganauthor=S.+Antonysamyauthor=K.+Weichertauthor=W.+P.+Bocchinfusoauthor=F.+Liauthor=S.+Kennedyauthor=B.+Liauthor=M.+M.+Maderauthor=C.+H.+Arrowsmithauthor=P.+J.+Brownauthor=M.+S.+Eramauthor=M.+M.+Szewczykauthor=D.+Barsyte-Lovejoyauthor=M.+Vedadiauthor=E.+Guccioneauthor=R.+M.+Campbell&title=LLY-283%2C+a+potent+and+selective+inhibitor+of+arginine+methyltransferase+5%2C+PRMT5%2C+with+antitumor+activity&doi=10.1021%2Facsmedchemlett.8b00014"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.8b00014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.8b00014%26sid%3Dliteratum%253Aachs%26aulast%3DBonday%26aufirst%3DZ.%2BQ.%26aulast%3DCortez%26aufirst%3DG.%2BS.%26aulast%3DGrogan%26aufirst%3DM.%2BJ.%26aulast%3DAntonysamy%26aufirst%3DS.%26aulast%3DWeichert%26aufirst%3DK.%26aulast%3DBocchinfuso%26aufirst%3DW.%2BP.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DKennedy%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DB.%26aulast%3DMader%26aufirst%3DM.%2BM.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DEram%26aufirst%3DM.%2BS.%26aulast%3DSzewczyk%26aufirst%3DM.%2BM.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DGuccione%26aufirst%3DE.%26aulast%3DCampbell%26aufirst%3DR.%2BM.%26atitle%3DLLY-283%252C%2520a%2520potent%2520and%2520selective%2520inhibitor%2520of%2520arginine%2520methyltransferase%25205%252C%2520PRMT5%252C%2520with%2520antitumor%2520activity%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2018%26doi%3D10.1021%2Facsmedchemlett.8b00014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref67"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref67'); return false;" data-citation="" class="refNumLink">67</a></strong><div class="NLM_citation" id="cit67"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wu, T.</span>; <span class="NLM_string-name">Brehmer, D.</span>; <span class="NLM_string-name">Beke, L.</span>; <span class="NLM_string-name">Boeckx, A.</span>; <span class="NLM_string-name">Diels, G.
S. M.</span>; <span class="NLM_string-name">Gilissen, R. A. H. J.</span>; <span class="NLM_string-name">Lawson, E. C.</span>; <span class="NLM_string-name">Pande, V.</span>; <span class="NLM_string-name">Parade, M. C. B. C.</span>; <span class="NLM_string-name">Schepens, W. B. G.</span></span> <span> </span><span class="NLM_article-title">Novel 6-6 Bicyclic Aromatic Ring Substituted Nucleoside Analogues for Use as PRMT5 Inhibitors</span>. <span class="NLM_patent">WO2017032840 A1</span>, <span class="NLM_year">2017</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2017&author=T.+Wu&author=D.+Brehmer&author=L.+Beke&author=A.+Boeckx&author=G.%0AS.+M.+Diels&author=R.+A.+H.+J.+Gilissen&author=E.+C.+Lawson&author=V.+Pande&author=M.+C.+B.+C.+Parade&author=W.+B.+G.+Schepens&title=Novel+6-6+Bicyclic+Aromatic+Ring+Substituted+Nucleoside+Analogues+for+Use+as+PRMT5+Inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit67&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DT.%26atitle%3DNovel%25206-6%2520Bicyclic%2520Aromatic%2520Ring%2520Substituted%2520Nucleoside%2520Analogues%2520for%2520Use%2520as%2520PRMT5%2520Inhibitors%26date%3D2017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref68"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref68'); return false;" data-citation="" class="refNumLink">68</a></strong><div class="NLM_citation" id="cit68"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Alinari, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mahasenan, K. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karkhanis, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quinion, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, P. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patton, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lapalombella, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roy, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Leo, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pileri, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Agostinelli, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ayers, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen-Kiang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elemento, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Motiwala, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Majumder, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Byrd, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacob, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sif, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baiocchi, R. A.</span></span> <span> </span><span class="NLM_article-title">Selective inhibition of protein arginine methyltransferase 5 blocks initiation and maintenance of B-cell transformation</span>. <i>Blood</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>125</i></span>,  <span class="NLM_fpage">2530</span>– <span class="NLM_lpage">2543</span>, <span class="refDoi"> DOI: 10.1182/blood-2014-12-619783</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=10.1182%2Fblood-2014-12-619783" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=125&publication_year=2015&pages=2530-2543&author=L.+Alinariauthor=K.+V.+Mahasenanauthor=F.+Yanauthor=V.+Karkhanisauthor=J.+H.+Chungauthor=E.+M.+Smithauthor=C.+Quinionauthor=P.+L.+Smithauthor=L.+Kimauthor=J.+T.+Pattonauthor=R.+Lapalombellaauthor=B.+Yuauthor=Y.+Wuauthor=S.+Royauthor=A.+De+Leoauthor=S.+Pileriauthor=C.+Agostinelliauthor=L.+Ayersauthor=J.+E.+Bradnerauthor=S.+Chen-Kiangauthor=O.+Elementoauthor=T.+Motiwalaauthor=S.+Majumderauthor=J.+C.+Byrdauthor=S.+Jacobauthor=S.+Sifauthor=C.+Liauthor=R.+A.+Baiocchi&title=Selective+inhibition+of+protein+arginine+methyltransferase+5+blocks+initiation+and+maintenance+of+B-cell+transformation&doi=10.1182%2Fblood-2014-12-619783"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit68&amp;dbid=16384&amp;doi=10.1182%2Fblood-2014-12-619783&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1182%252Fblood-2014-12-619783%26sid%3Dliteratum%253Aachs%26aulast%3DAlinari%26aufirst%3DL.%26aulast%3DMahasenan%26aufirst%3DK.%2BV.%26aulast%3DYan%26aufirst%3DF.%26aulast%3DKarkhanis%26aufirst%3DV.%26aulast%3DChung%26aufirst%3DJ.%2BH.%26aulast%3DSmith%26aufirst%3DE.%2BM.%26aulast%3DQuinion%26aufirst%3DC.%26aulast%3DSmith%26aufirst%3DP.%2BL.%26aulast%3DKim%26aufirst%3DL.%26aulast%3DPatton%26aufirst%3DJ.%2BT.%26aulast%3DLapalombella%26aufirst%3DR.%26aulast%3DYu%26aufirst%3DB.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DRoy%26aufirst%3DS.%26aulast%3DDe%2BLeo%26aufirst%3DA.%26aulast%3DPileri%26aufirst%3DS.%26aulast%3DAgostinelli%26aufirst%3DC.%26aulast%3DAyers%26aufirst%3DL.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DChen-Kiang%26aufirst%3DS.%26aulast%3DElemento%26aufirst%3DO.%26aulast%3DMotiwala%26aufirst%3DT.%26aulast%3DMajumder%26aufirst%3DS.%26aulast%3DByrd%26aufirst%3DJ.%2BC.%26aulast%3DJacob%26aufirst%3DS.%26aulast%3DSif%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DC.%26aulast%3DBaiocchi%26aufirst%3DR.%2BA.%26atitle%3DSelective%2520inhibition%2520of%2520protein%2520arginine%2520methyltransferase%25205%2520blocks%2520initiation%2520and%2520maintenance%2520of%2520B-cell%2520transformation%26jtitle%3DBlood%26date%3D2015%26volume%3D125%26spage%3D2530%26epage%3D2543%26doi%3D10.1182%2Fblood-2014-12-619783" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref69"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref69'); return false;" data-citation="" class="refNumLink">69</a></strong><div class="NLM_citation" id="cit69"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, H.</span></span> <span> </span><span class="NLM_article-title">Interaction assessments of the first S-adenosylmethionine competitive inhibitor and the essential interacting partner methylosome protein 50 with protein arginine methyltransferase 5 by combined computational methods</span>. <i>Biochem. Biophys. Res. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>495</i></span>,  <span class="NLM_fpage">721</span>– <span class="NLM_lpage">727</span>, <span class="refDoi"> DOI: 10.1016/j.bbrc.2017.11.089</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=10.1016%2Fj.bbrc.2017.11.089" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=495&publication_year=2018&pages=721-727&author=K.+Zhuauthor=C.+S.+Jiangauthor=J.+Huauthor=X.+Liuauthor=X.+Yanauthor=H.+Taoauthor=C.+Luoauthor=H.+Zhang&title=Interaction+assessments+of+the+first+S-adenosylmethionine+competitive+inhibitor+and+the+essential+interacting+partner+methylosome+protein+50+with+protein+arginine+methyltransferase+5+by+combined+computational+methods&doi=10.1016%2Fj.bbrc.2017.11.089"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit69&amp;dbid=16384&amp;doi=10.1016%2Fj.bbrc.2017.11.089&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbrc.2017.11.089%26sid%3Dliteratum%253Aachs%26aulast%3DZhu%26aufirst%3DK.%26aulast%3DJiang%26aufirst%3DC.%2BS.%26aulast%3DHu%26aufirst%3DJ.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DYan%26aufirst%3DX.%26aulast%3DTao%26aufirst%3DH.%26aulast%3DLuo%26aufirst%3DC.%26aulast%3DZhang%26aufirst%3DH.%26atitle%3DInteraction%2520assessments%2520of%2520the%2520first%2520S-adenosylmethionine%2520competitive%2520inhibitor%2520and%2520the%2520essential%2520interacting%2520partner%2520methylosome%2520protein%252050%2520with%2520protein%2520arginine%2520methyltransferase%25205%2520by%2520combined%2520computational%2520methods%26jtitle%3DBiochem.%2520Biophys.%2520Res.%2520Commun.%26date%3D2018%26volume%3D495%26spage%3D721%26epage%3D727%26doi%3D10.1016%2Fj.bbrc.2017.11.089" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref70"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref70'); return false;" data-citation="" class="refNumLink">70</a></strong><div class="NLM_citation" id="cit70"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tarighat, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Santhanam, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frankhouser, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Radomska, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anghelina, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alinari, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walker, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caligiuri, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Croce, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garzon, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baiocchi, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marcucci, G.</span></span> <span> </span><span class="NLM_article-title">The dual epigenetic role of PRMT5 in acute myeloid leukemia: gene activation and repression via histone arginine methylation</span>. <i>Leukemia</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">789</span>– <span class="NLM_lpage">799</span>, <span class="refDoi"> DOI: 10.1038/leu.2015.308</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=10.1038%2Fleu.2015.308" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=26536822" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvFKhtrrN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2016&pages=789-799&author=S.+S.+Tarighatauthor=R.+Santhanamauthor=D.+Frankhouserauthor=H.+S.+Radomskaauthor=H.+Laiauthor=M.+Anghelinaauthor=H.+Wangauthor=X.+Huangauthor=L.+Alinariauthor=A.+Walkerauthor=M.+A.+Caligiuriauthor=C.+M.+Croceauthor=L.+Liauthor=R.+Garzonauthor=C.+Liauthor=R.+A.+Baiocchiauthor=G.+Marcucci&title=The+dual+epigenetic+role+of+PRMT5+in+acute+myeloid+leukemia%3A+gene+activation+and+repression+via+histone+arginine+methylation&doi=10.1038%2Fleu.2015.308"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit70R"><div class="casContent"><span class="casTitleNuber">70</span><div class="casTitle"><span class="NLM_cas:atitle">The dual epigenetic role of PRMT5 in acute myeloid leukemia: gene activation and repression via histone arginine methylation</span></div><div class="casAuthors">Tarighat, S. S.; Santhanam, R.; Frankhouser, D.; Radomska, H. S.; Lai, H.; Anghelina, M.; Wang, H.; Huang, X.; Alinari, L.; Walker, A.; Caligiuri, M. A.; Croce, C. M.; Li, L.; Garzon, R.; Li, C.; Baiocchi, R. A.; Marcucci, G.</div><div class="citationInfo"><span class="NLM_cas:title">Leukemia</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">789-799</span>CODEN:
                <span class="NLM_cas:coden">LEUKED</span>;
        ISSN:<span class="NLM_cas:issn">0887-6924</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Changes in the enzymic activity of protein arginine methyltransferase (PRMT) 5 have been assocd. with cancer; however, the protein's role in acute myeloid leukemia (AML) has not been fully evaluated.  Here, we show that increased PRMT5 activity enhanced AML growth in vitro and in vivo while PRMT5 downregulation reduced it.  In AML cells, PRMT5 interacted with Sp1 in a transcription repressor complex and silenced miR-29b preferentially via dimethylation of histone 4 arginine residue H4R3.  As Sp1 is also a bona fide target of miR-29b, the miR silencing resulted in increased Sp1.  This event in turn led to transcription activation of FLT3, a gene that encodes a receptor tyrosine kinase.  Inhibition of PRMT5 via sh/siRNA or a first-in-class small-mol. inhibitor (HLCL-61) resulted in significantly increased expression of miR-29b and consequent suppression of Sp1 and FLT3 in AML cells.  As a result, significant antileukemic activity was achieved.  Collectively, our data support a novel leukemogenic mechanism in AML where PRMT5 mediates both silencing and transcription of genes that participate in a 'yin-yang' functional network supporting leukemia growth.  As FLT3 is often mutated in AML and pharmacol. inhibition of PRMT5 appears feasible, the PRMT5-miR-29b-FLT3 network should be further explored as a novel therapeutic target for AML.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrGtynJRsb5obVg90H21EOLACvtfcHk0ljZCuZFA6Tl6Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvFKhtrrN&md5=c60b0797c096b171d970ec785d0f6110</span></div><a href="/servlet/linkout?suffix=cit70&amp;dbid=16384&amp;doi=10.1038%2Fleu.2015.308&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fleu.2015.308%26sid%3Dliteratum%253Aachs%26aulast%3DTarighat%26aufirst%3DS.%2BS.%26aulast%3DSanthanam%26aufirst%3DR.%26aulast%3DFrankhouser%26aufirst%3DD.%26aulast%3DRadomska%26aufirst%3DH.%2BS.%26aulast%3DLai%26aufirst%3DH.%26aulast%3DAnghelina%26aufirst%3DM.%26aulast%3DWang%26aufirst%3DH.%26aulast%3DHuang%26aufirst%3DX.%26aulast%3DAlinari%26aufirst%3DL.%26aulast%3DWalker%26aufirst%3DA.%26aulast%3DCaligiuri%26aufirst%3DM.%2BA.%26aulast%3DCroce%26aufirst%3DC.%2BM.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DGarzon%26aufirst%3DR.%26aulast%3DLi%26aufirst%3DC.%26aulast%3DBaiocchi%26aufirst%3DR.%2BA.%26aulast%3DMarcucci%26aufirst%3DG.%26atitle%3DThe%2520dual%2520epigenetic%2520role%2520of%2520PRMT5%2520in%2520acute%2520myeloid%2520leukemia%253A%2520gene%2520activation%2520and%2520repression%2520via%2520histone%2520arginine%2520methylation%26jtitle%3DLeukemia%26date%3D2016%26volume%3D30%26spage%3D789%26epage%3D799%26doi%3D10.1038%2Fleu.2015.308" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref71"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref71'); return false;" data-citation="" class="refNumLink">71</a></strong><div class="NLM_citation" id="cit71"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Politzer, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clark, T.</span></span> <span> </span><span class="NLM_article-title">Halogen bonding: an electrostatically-driven highly directional noncovalent interaction</span>. <i>Phys. Chem. Chem. Phys.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">7748</span>– <span class="NLM_lpage">7757</span>, <span class="refDoi"> DOI: 10.1039/c004189k</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=10.1039%2Fc004189k" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=20571692" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=1%3ACAS%3A528%3ADC%252BC3cXosFaqtrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2010&pages=7748-7757&author=P.+Politzerauthor=J.+S.+Murrayauthor=T.+Clark&title=Halogen+bonding%3A+an+electrostatically-driven+highly+directional+noncovalent+interaction&doi=10.1039%2Fc004189k"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit71R"><div class="casContent"><span class="casTitleNuber">71</span><div class="casTitle"><span class="NLM_cas:atitle">Halogen bonding: an electrostatically-driven highly directional noncovalent interaction</span></div><div class="casAuthors">Politzer, Peter; Murray, Jane S.; Clark, Timothy</div><div class="citationInfo"><span class="NLM_cas:title">Physical Chemistry Chemical Physics</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">28</span>),
    <span class="NLM_cas:pages">7748-7757</span>CODEN:
                <span class="NLM_cas:coden">PPCPFQ</span>;
        ISSN:<span class="NLM_cas:issn">1463-9076</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A review.  A halogen bond is a highly directional, electrostatically-driven noncovalent interaction between a region of pos. electrostatic potential on the outer side of the halogen X in a mol. R-X and a neg. site B, such as a lone pair of a Lewis base or the π-electrons of an unsatd. system.  The pos. region on X corresponds to the electronically-depleted outer lobe of the half-filled p-type orbital of X that is involved in forming the covalent bond to R.  This depletion is labeled a σ-hole.  The resulting pos. electrostatic potential is along the extension of the R-X bond, which accounts for the directionality of halogen bonding.  Pos. σ-holes can also be found on covalently-bonded Group IV-VI atoms, which can similarly interact electrostatically with neg. sites.  Since pos. σ-holes often exist in conjunction with neg. potentials on other portions of the atom's surface, such atoms can interact electrostatically with both nucleophiles and electrophiles, as has been obsd. in surveys of crystallog. structures.  Exptl. as well as computational studies indicate that halogen and other σ-hole interactions can be competitive with hydrogen bonding, which itself can be viewed as a subset of σ-hole bonding.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrnp-PekfjXjrVg90H21EOLACvtfcHk0li2Km_aYjYeYQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXosFaqtrg%253D&md5=552d2699223590f79f8a0954487dc8f6</span></div><a href="/servlet/linkout?suffix=cit71&amp;dbid=16384&amp;doi=10.1039%2Fc004189k&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc004189k%26sid%3Dliteratum%253Aachs%26aulast%3DPolitzer%26aufirst%3DP.%26aulast%3DMurray%26aufirst%3DJ.%2BS.%26aulast%3DClark%26aufirst%3DT.%26atitle%3DHalogen%2520bonding%253A%2520an%2520electrostatically-driven%2520highly%2520directional%2520noncovalent%2520interaction%26jtitle%3DPhys.%2520Chem.%2520Chem.%2520Phys.%26date%3D2010%26volume%3D12%26spage%3D7748%26epage%3D7757%26doi%3D10.1039%2Fc004189k" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref72"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref72'); return false;" data-citation="" class="refNumLink">72</a></strong><div class="NLM_citation" id="cit72"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lian, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ye, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, C.</span></span> <span> </span><span class="NLM_article-title">Development of potent type I protein arginine methyltransferase (PRMT) inhibitors of leukemia cell proliferation</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">8888</span>– <span class="NLM_lpage">8905</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b01134</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b01134" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhs1CisbfN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=8888-8905&author=C.+Wangauthor=H.+Jiangauthor=J.+Jinauthor=Y.+Xieauthor=Z.+Chenauthor=H.+Zhangauthor=F.+Lianauthor=Y.+C.+Liuauthor=C.+Zhangauthor=H.+Dingauthor=S.+Chenauthor=N.+Zhangauthor=Y.+Zhangauthor=H.+Jiangauthor=K.+Chenauthor=F.+Yeauthor=Z.+Yaoauthor=C.+Luo&title=Development+of+potent+type+I+protein+arginine+methyltransferase+%28PRMT%29+inhibitors+of+leukemia+cell+proliferation&doi=10.1021%2Facs.jmedchem.7b01134"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit72R"><div class="casContent"><span class="casTitleNuber">72</span><div class="casTitle"><span class="NLM_cas:atitle">Development of Potent Type I Protein Arginine Methyltransferase (PRMT) Inhibitors of Leukemia Cell Proliferation</span></div><div class="casAuthors">Wang, Chen; Jiang, Hao; Jin, Jia; Xie, Yiqian; Chen, Zhifeng; Zhang, Hao; Lian, Fulin; Liu, Yu-Chih; Zhang, Chenhua; Ding, Hong; Chen, Shijie; Zhang, Naixia; Zhang, Yuanyuan; Jiang, Hualiang; Chen, Kaixian; Ye, Fei; Yao, Zhiyi; Luo, Cheng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">8888-8905</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Protein Arginine Methyltransferases (PRMTs) are crucial players in diverse biol. processes, and dysregulation of PRMTs has been linked to various human diseases, esp. cancer.  Therefore, small mols. targeting PRMTs have profound impact for both academic functional studies and clin. disease treatment.  Here, the authors report the discovery of N1-(2-((2-chlorophenyl)thio)benzyl)-N1-methylethane-1,2-diamine (28d, DCPR049_12), a highly potent inhibitor of type I PRMTs that has good selectivity against a panel of other methyltransferases.  Compd. 28d effectively inhibits cell proliferation in several leukemia cell lines and reduces the cellular asym. arginine dimethylation levels.  Serving as an effective inhibitor, 28d demonstrates the mechanism of cell killing in both cell cycle arrest and apoptotic effect as well as downregulation of the pivotal mixed lineage leukemia (MLL) fusion target genes such as HOXA9 and MEIS1, which reflects the crit. roles of type I PRMTs in MLL leukemia.  These studies present 28d as a valuable inhibitor to investigate the role of type I PRMTs in cancer and other diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrddA_Xz0t9VLVg90H21EOLACvtfcHk0li2Km_aYjYeYQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhs1CisbfN&md5=40953f4e3f8d78075d444451c67ff6e0</span></div><a href="/servlet/linkout?suffix=cit72&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b01134&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b01134%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DC.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DJin%26aufirst%3DJ.%26aulast%3DXie%26aufirst%3DY.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DZhang%26aufirst%3DH.%26aulast%3DLian%26aufirst%3DF.%26aulast%3DLiu%26aufirst%3DY.%2BC.%26aulast%3DZhang%26aufirst%3DC.%26aulast%3DDing%26aufirst%3DH.%26aulast%3DChen%26aufirst%3DS.%26aulast%3DZhang%26aufirst%3DN.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DChen%26aufirst%3DK.%26aulast%3DYe%26aufirst%3DF.%26aulast%3DYao%26aufirst%3DZ.%26aulast%3DLuo%26aufirst%3DC.%26atitle%3DDevelopment%2520of%2520potent%2520type%2520I%2520protein%2520arginine%2520methyltransferase%2520%2528PRMT%2529%2520inhibitors%2520of%2520leukemia%2520cell%2520proliferation%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D8888%26epage%3D8905%26doi%3D10.1021%2Facs.jmedchem.7b01134" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref73"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref73'); return false;" data-citation="" class="refNumLink">73</a></strong><div class="NLM_citation" id="cit73"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Webb, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amici, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jablonski, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Savardekar, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Panfil, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peine, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karkhanis, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bachelder, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ainslie, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Green, P. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baiocchi, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guerau-de-Arellano, M.</span></span> <span> </span><span class="NLM_article-title">PRMT5-selective inhibitors suppress inflammatory T cell responses and experimental autoimmune encephalomyelitis</span>. <i>J. Immunol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>198</i></span>,  <span class="NLM_fpage">1439</span>– <span class="NLM_lpage">1451</span>, <span class="refDoi"> DOI: 10.4049/jimmunol.1601702</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=10.4049%2Fjimmunol.1601702" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=198&publication_year=2017&pages=1439-1451&author=L.+M.+Webbauthor=S.+A.+Amiciauthor=K.+A.+Jablonskiauthor=H.+Savardekarauthor=A.+R.+Panfilauthor=L.+Liauthor=W.+Zhouauthor=K.+Peineauthor=V.+Karkhanisauthor=E.+M.+Bachelderauthor=K.+M.+Ainslieauthor=P.+L.+Greenauthor=C.+Liauthor=R.+A.+Baiocchiauthor=M.+Guerau-de-Arellano&title=PRMT5-selective+inhibitors+suppress+inflammatory+T+cell+responses+and+experimental+autoimmune+encephalomyelitis&doi=10.4049%2Fjimmunol.1601702"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit73&amp;dbid=16384&amp;doi=10.4049%2Fjimmunol.1601702&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4049%252Fjimmunol.1601702%26sid%3Dliteratum%253Aachs%26aulast%3DWebb%26aufirst%3DL.%2BM.%26aulast%3DAmici%26aufirst%3DS.%2BA.%26aulast%3DJablonski%26aufirst%3DK.%2BA.%26aulast%3DSavardekar%26aufirst%3DH.%26aulast%3DPanfil%26aufirst%3DA.%2BR.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DZhou%26aufirst%3DW.%26aulast%3DPeine%26aufirst%3DK.%26aulast%3DKarkhanis%26aufirst%3DV.%26aulast%3DBachelder%26aufirst%3DE.%2BM.%26aulast%3DAinslie%26aufirst%3DK.%2BM.%26aulast%3DGreen%26aufirst%3DP.%2BL.%26aulast%3DLi%26aufirst%3DC.%26aulast%3DBaiocchi%26aufirst%3DR.%2BA.%26aulast%3DGuerau-de-Arellano%26aufirst%3DM.%26atitle%3DPRMT5-selective%2520inhibitors%2520suppress%2520inflammatory%2520T%2520cell%2520responses%2520and%2520experimental%2520autoimmune%2520encephalomyelitis%26jtitle%3DJ.%2520Immunol.%26date%3D2017%26volume%3D198%26spage%3D1439%26epage%3D1451%26doi%3D10.4049%2Fjimmunol.1601702" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref74"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref74'); return false;" data-citation="" class="refNumLink">74</a></strong><div class="NLM_citation" id="cit74"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mao, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duan, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, C.</span></span> <span> </span><span class="NLM_article-title">Potent, selective, and cell active protein arginine methyltransferase 5 (PRMT5) inhibitor developed by structure-based virtual screening and hit optimization</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">6289</span>– <span class="NLM_lpage">6304</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b00587</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b00587" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtVKitr7E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=6289-6304&author=R.+Maoauthor=J.+Shaoauthor=K.+Zhuauthor=Y.+Zhangauthor=H.+Dingauthor=C.+Zhangauthor=Z.+Shiauthor=H.+Jiangauthor=D.+Sunauthor=W.+Duanauthor=C.+Luo&title=Potent%2C+selective%2C+and+cell+active+protein+arginine+methyltransferase+5+%28PRMT5%29+inhibitor+developed+by+structure-based+virtual+screening+and+hit+optimization&doi=10.1021%2Facs.jmedchem.7b00587"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit74R"><div class="casContent"><span class="casTitleNuber">74</span><div class="casTitle"><span class="NLM_cas:atitle">Potent, Selective, and Cell Active Protein Arginine Methyltransferase 5 (PRMT5) Inhibitor Developed by Structure-Based Virtual Screening and Hit Optimization</span></div><div class="casAuthors">Mao, Ruifeng; Shao, Jingwei; Zhu, Kongkai; Zhang, Yuanyuan; Ding, Hong; Zhang, Chenhua; Shi, Zhe; Jiang, Hualiang; Sun, Dequn; Duan, Wenhu; Luo, Cheng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">6289-6304</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Abstr. PRMT5 plays important roles in diverse cellular processes and is upregulated in several human malignancies.  Besides, PRMT5 has been validated as anticancer target in mantle cell lymphoma.  In this study, the authors found a potent and selective PRMT5 inhibitor by performing structure-based virtual screening and hit optimization.  The identified compd. I (IC50 = 0.33 μM) exhibited a broad selectivity against a panel of other methyltransferases.  The direct binding of I to PRMT5 was validated by surface plasmon resonance expts., with a Kd of 0.987 μM.  Kinetic expts. indicated that I was a SAM competitive inhibitor other than substrate.  In addn., I showed selective antiproliferative effects against MV4-11 cells, and further studies indicated that the mechanism of cellular antitumor activity was due to inhibition of PRMT5 mediated SmD3 methylation. I may represent a promising lead compd. to understand more about PRMT5 and potentially assist the development of treatments for leukemia indications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoI1jnM_kEhRrVg90H21EOLACvtfcHk0lglda3E3y21uw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtVKitr7E&md5=5e3687166e2e6b5558409806e3c4b9e5</span></div><a href="/servlet/linkout?suffix=cit74&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b00587&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b00587%26sid%3Dliteratum%253Aachs%26aulast%3DMao%26aufirst%3DR.%26aulast%3DShao%26aufirst%3DJ.%26aulast%3DZhu%26aufirst%3DK.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DDing%26aufirst%3DH.%26aulast%3DZhang%26aufirst%3DC.%26aulast%3DShi%26aufirst%3DZ.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DSun%26aufirst%3DD.%26aulast%3DDuan%26aufirst%3DW.%26aulast%3DLuo%26aufirst%3DC.%26atitle%3DPotent%252C%2520selective%252C%2520and%2520cell%2520active%2520protein%2520arginine%2520methyltransferase%25205%2520%2528PRMT5%2529%2520inhibitor%2520developed%2520by%2520structure-based%2520virtual%2520screening%2520and%2520hit%2520optimization%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D6289%26epage%3D6304%26doi%3D10.1021%2Facs.jmedchem.7b00587" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref75"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref75'); return false;" data-citation="" class="refNumLink">75</a></strong><div class="NLM_citation" id="cit75"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ji, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, W. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, S. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, T. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiang, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, Y. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, S. Y.</span></span> <span> </span><span class="NLM_article-title">Discovery of selective protein arginine methyltransferase 5 inhibitors and biological evaluations</span>. <i>Chem. Biol. Drug Des.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>89</i></span>,  <span class="NLM_fpage">585</span>– <span class="NLM_lpage">598</span>, <span class="refDoi"> DOI: 10.1111/cbdd.12881</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=10.1111%2Fcbdd.12881" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=27714957" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvVWmu73E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=89&publication_year=2017&pages=585-598&author=S.+Jiauthor=S.+Maauthor=W.+J.+Wangauthor=S.+Z.+Huangauthor=T.+Q.+Wangauthor=R.+Xiangauthor=Y.+G.+Huauthor=Q.+Chenauthor=L.+L.+Liauthor=S.+Y.+Yang&title=Discovery+of+selective+protein+arginine+methyltransferase+5+inhibitors+and+biological+evaluations&doi=10.1111%2Fcbdd.12881"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit75R"><div class="casContent"><span class="casTitleNuber">75</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of selective protein arginine methyltransferase 5 inhibitors and biological evaluations</span></div><div class="casAuthors">Ji, Sen; Ma, Shuang; Wang, Wen-Jing; Huang, Shen-Zhen; Wang, Tian-qi; Xiang, Rong; Hu, Yi-Guo; Chen, Qiang; Li, Lin-Li; Yang, Sheng-Yong</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Biology & Drug Design</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">89</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">585-598</span>CODEN:
                <span class="NLM_cas:coden">CBDDAL</span>;
        ISSN:<span class="NLM_cas:issn">1747-0277</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">Protein arginine methyltransferase 5 (PRMT5) is an important protein arginine methyltransferase that catalyzes the sym. dimethylation of arginine resides on histones or nonhistone substrate proteins.  It has been thought as a promising target for many diseases, particularly cancer.  Despite the potential applications of PRMT5 inhibitors in cancer treatment, very few of PRMT5i have been publicly reported.  In this investigation, virtual screening and structure-activity relationship studies were carried out to discover novel PRMT5i, which finally led to the identification of a no. of new PRMT5i.  The most active compd., I, exhibited a considerable inhibitory potency against PRMT5 with an IC50 value of 0.57 μM, and a high selectivity for PRMT5 against other tested PRMTs.  It displayed a very good anti-viability activity against two colorectal cancer cell lines, HT-29 and DLD-1, and one hepatic cancer cell line, HepG2, in a sensitivity assay against 36 different cancer cell lines.  Western blot assays indicated that I selectively inhibited the sym. dimethylations of H4R3 and H3R8 in DLD-1 cells.  Overall, I could be used as a chem. probe to investigate new functions of PRMT5 in biol. and also served as a good lead compd. for the development of new PRMT5-targeting therapeutic agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq_6yDJULZD-LVg90H21EOLACvtfcHk0lglda3E3y21uw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvVWmu73E&md5=1d4f06bef5c95682073c2fcb1e1859d4</span></div><a href="/servlet/linkout?suffix=cit75&amp;dbid=16384&amp;doi=10.1111%2Fcbdd.12881&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fcbdd.12881%26sid%3Dliteratum%253Aachs%26aulast%3DJi%26aufirst%3DS.%26aulast%3DMa%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DW.%2BJ.%26aulast%3DHuang%26aufirst%3DS.%2BZ.%26aulast%3DWang%26aufirst%3DT.%2BQ.%26aulast%3DXiang%26aufirst%3DR.%26aulast%3DHu%26aufirst%3DY.%2BG.%26aulast%3DChen%26aufirst%3DQ.%26aulast%3DLi%26aufirst%3DL.%2BL.%26aulast%3DYang%26aufirst%3DS.%2BY.%26atitle%3DDiscovery%2520of%2520selective%2520protein%2520arginine%2520methyltransferase%25205%2520inhibitors%2520and%2520biological%2520evaluations%26jtitle%3DChem.%2520Biol.%2520Drug%2520Des.%26date%3D2017%26volume%3D89%26spage%3D585%26epage%3D598%26doi%3D10.1111%2Fcbdd.12881" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54'],'ref55':['cit55'],'ref56':['cit56'],'ref57':['cit57'],'ref58':['cit58'],'ref59':['cit59'],'ref60':['cit60'],'ref61':['cit61'],'ref62':['cit62'],'ref63':['cit63'],'ref64':['cit64'],'ref65':['cit65'],'ref66':['cit66'],'ref67':['cit67'],'ref68':['cit68'],'ref69':['cit69'],'ref70':['cit70'],'ref71':['cit71'],'ref72':['cit72'],'ref73':['cit73'],'ref74':['cit74'],'ref75':['cit75']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 19 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Yi Zhang, Deqin Rong, Bingbing Li, <span class="NLM_string-name hlFld-ContribAuthor">Yuanxiang Wang</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting Epigenetic Regulators with Covalent Small-Molecule Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (12)
                                     , 7900-7925. <a href="https://doi.org/10.1021/acs.jmedchem.0c02055" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c02055</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c02055&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c02055%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DTargeting%252BEpigenetic%252BRegulators%252Bwith%252BCovalent%252BSmall-Molecule%252BInhibitors%26aulast%3DZhang%26aufirst%3DYi%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D26112020%26date%3D18022021%26volume%3D64%26issue%3D12%26spage%3D7900%26epage%3D7925" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Ryan V. Quiroz, Michael H. Reutershan, Sebastian E. Schneider, David Sloman, Brian M. Lacey, Brooke M. Swalm, Charles S. Yeung, Craig Gibeau, Daniel S. Spellman, Danica A. Rankic, Dapeng Chen, David Witter, Doug Linn, Erik Munsell, Guo Feng, Haiyan Xu, Jonathan M. E. Hughes, Jongwon Lim, Josep Saurí, Kristin Geddes, Murray Wan, My Sam Mansueto, Nicole E. Follmer, Patrick S. Fier, Phieng Siliphaivanh, Pierre Daublain, Rachel L. Palte, Robert P. Hayes, Sandra Lee, Shuhei Kawamura, Steven Silverman, Sulagna Sanyal, Timothy J. Henderson, Yingchun Ye, Yuanwei Gao, Benjamin Nicholson, <span class="NLM_string-name hlFld-ContribAuthor">Michelle R. Machacek</span>. </span><span class="cited-content_cbyCitation_article-title">The Discovery of Two Novel Classes of 5,5-Bicyclic Nucleoside-Derived PRMT5 Inhibitors for the Treatment of Cancer. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (7)
                                     , 3911-3939. <a href="https://doi.org/10.1021/acs.jmedchem.0c02083" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c02083</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c02083&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c02083%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DThe%252BDiscovery%252Bof%252BTwo%252BNovel%252BClasses%252Bof%252B5%25252C5-Bicyclic%252BNucleoside-Derived%252BPRMT5%252BInhibitors%252Bfor%252Bthe%252BTreatment%252Bof%252BCancer%26aulast%3DQuiroz%26aufirst%3DRyan%2BV.%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D01122020%26date%3D23032021%26volume%3D64%26issue%3D7%26spage%3D3911%26epage%3D3939" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">David A. Candito, Yingchun Ye, Ryan V. Quiroz, Michael H. Reutershan, David Witter, Surendra B. Gadamsetty, Hongming Li, Josep Saurí, Sebastian E. Schneider, Yu-hong Lam, <span class="NLM_string-name hlFld-ContribAuthor">Rachel L. Palte</span>. </span><span class="cited-content_cbyCitation_article-title">Development of a Flexible and Robust Synthesis of Tetrahydrofuro[3,4-b]furan Nucleoside Analogues. </span><span class="cited-content_cbyCitation_journal-name">The Journal of Organic Chemistry</span><span> <strong>2021,</strong> <em>86 </em>
                                    (7)
                                     , 5142-5151. <a href="https://doi.org/10.1021/acs.joc.0c02969" title="DOI URL">https://doi.org/10.1021/acs.joc.0c02969</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.joc.0c02969&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.joc.0c02969%26sid%3Dliteratum%253Aachs%26jtitle%3DThe%2520Journal%2520of%2520Organic%2520Chemistry%26atitle%3DDevelopment%252Bof%252Ba%252BFlexible%252Band%252BRobust%252BSynthesis%252Bof%252BTetrahydrofuro%25255B3%25252C4-b%25255Dfuran%252BNucleoside%252BAnalogues%26aulast%3DCandito%26aufirst%3DDavid%2BA.%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D05012021%26date%3D23032021%26volume%3D86%26issue%3D7%26spage%3D5142%26epage%3D5151" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Hong Lin, Min Wang, Yang W. Zhang, Shuilong Tong, Raul A. Leal, Rupa Shetty, Kris Vaddi, <span class="NLM_string-name hlFld-ContribAuthor">Juan I. Luengo</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of Potent and Selective Covalent Protein Arginine Methyltransferase 5 (PRMT5) Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">ACS Medicinal Chemistry Letters</span><span> <strong>2019,</strong> <em>10 </em>
                                    (7)
                                     , 1033-1038. <a href="https://doi.org/10.1021/acsmedchemlett.9b00074" title="DOI URL">https://doi.org/10.1021/acsmedchemlett.9b00074</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsmedchemlett.9b00074&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsmedchemlett.9b00074%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DDiscovery%252Bof%252BPotent%252Band%252BSelective%252BCovalent%252BProtein%252BArginine%252BMethyltransferase%252B5%252B%252528PRMT5%252529%252BInhibitors%26aulast%3DLin%26aufirst%3DHong%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D26022019%26date%3D22052019%26date%3D04062019%26date%3D22052019%26volume%3D10%26issue%3D7%26spage%3D1033%26epage%3D1038" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Aurélie  Quillien</span>, <span class="hlFld-ContribAuthor ">Guerric  Gilbert</span>, <span class="hlFld-ContribAuthor ">Manon  Boulet</span>, <span class="hlFld-ContribAuthor ">Séverine  Ethuin</span>, <span class="hlFld-ContribAuthor ">Lucas  Waltzer</span>, <span class="hlFld-ContribAuthor ">Laurence  Vandel</span>, . </span><span class="cited-content_cbyCitation_article-title">Prmt5 promotes vascular morphogenesis independently of its methyltransferase activity. </span><span class="cited-content_cbyCitation_journal-name">PLOS Genetics</span><span> <strong>2021,</strong> <em>17 </em>
                                    (6)
                                     , e1009641. <a href="https://doi.org/10.1371/journal.pgen.1009641" title="DOI URL">https://doi.org/10.1371/journal.pgen.1009641</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1371/journal.pgen.1009641&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1371%2Fjournal.pgen.1009641%26sid%3Dliteratum%253Aachs%26jtitle%3DPLOS%2520Genetics%26atitle%3DPrmt5%252Bpromotes%252Bvascular%252Bmorphogenesis%252Bindependently%252Bof%252Bits%252Bmethyltransferase%252Bactivity%26aulast%3DQuillien%26aufirst%3DAur%25C3%25A9lie%26date%3D2021%26date%3D2021%26volume%3D17%26issue%3D6%26spage%3De1009641" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xinye  Wang</span>, <span class="hlFld-ContribAuthor ">Ting  Qiu</span>, <span class="hlFld-ContribAuthor ">Yingying  Wu</span>, <span class="hlFld-ContribAuthor ">Chuanyu  Yang</span>, <span class="hlFld-ContribAuthor ">Yi  Li</span>, <span class="hlFld-ContribAuthor ">Guangshi  Du</span>, <span class="hlFld-ContribAuthor ">Yaohui  He</span>, <span class="hlFld-ContribAuthor ">Wen  Liu</span>, <span class="hlFld-ContribAuthor ">Rong  Liu</span>, <span class="hlFld-ContribAuthor ">Chuan-huizi  Chen</span>, <span class="hlFld-ContribAuthor ">Yujie  Shi</span>, <span class="hlFld-ContribAuthor ">Jingxuan  Pan</span>, <span class="hlFld-ContribAuthor ">Jia  Zhou</span>, <span class="hlFld-ContribAuthor ">Dewei  Jiang</span>, <span class="hlFld-ContribAuthor ">Ceshi  Chen</span>. </span><span class="cited-content_cbyCitation_article-title">Arginine methyltransferase PRMT5 methylates and stabilizes KLF5 via decreasing its phosphorylation and ubiquitination to promote basal-like breast cancer. </span><span class="cited-content_cbyCitation_journal-name">Cell Death & Differentiation</span><span> <strong>2021,</strong> <em>70 </em><a href="https://doi.org/10.1038/s41418-021-00793-0" title="DOI URL">https://doi.org/10.1038/s41418-021-00793-0</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41418-021-00793-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41418-021-00793-0%26sid%3Dliteratum%253Aachs%26jtitle%3DCell%2520Death%2520%2526%2520Differentiation%26atitle%3DArginine%252Bmethyltransferase%252BPRMT5%252Bmethylates%252Band%252Bstabilizes%252BKLF5%252Bvia%252Bdecreasing%252Bits%252Bphosphorylation%252Band%252Bubiquitination%252Bto%252Bpromote%252Bbasal-like%252Bbreast%252Bcancer%26aulast%3DWang%26aufirst%3DXinye%26date%3D2021%26date%3D2021%26volume%3D70" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Katya  Marjon</span>, <span class="hlFld-ContribAuthor ">Peter  Kalev</span>, <span class="hlFld-ContribAuthor ">Kevin  Marks</span>. </span><span class="cited-content_cbyCitation_article-title">Cancer Dependencies: PRMT5 and MAT2A in MTAP/p16-Deleted Cancers. </span><span class="cited-content_cbyCitation_journal-name">Annual Review of Cancer Biology</span><span> <strong>2021,</strong> <em>5 </em>
                                    (1)
                                     , 371-390. <a href="https://doi.org/10.1146/annurev-cancerbio-030419-033444" title="DOI URL">https://doi.org/10.1146/annurev-cancerbio-030419-033444</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1146/annurev-cancerbio-030419-033444&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1146%2Fannurev-cancerbio-030419-033444%26sid%3Dliteratum%253Aachs%26jtitle%3DAnnual%2520Review%2520of%2520Cancer%2520Biology%26atitle%3DCancer%252BDependencies%25253A%252BPRMT5%252Band%252BMAT2A%252Bin%252BMTAP%25252Fp16-Deleted%252BCancers%26aulast%3DMarjon%26aufirst%3DKatya%26date%3D2021%26volume%3D5%26issue%3D1%26spage%3D371%26epage%3D390" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yonghui  Dong</span>, <span class="hlFld-ContribAuthor ">Ping  Wang</span>, <span class="hlFld-ContribAuthor ">Yongguang  Yang</span>, <span class="hlFld-ContribAuthor ">Jincheng  Huang</span>, <span class="hlFld-ContribAuthor ">Zhipeng  Dai</span>, <span class="hlFld-ContribAuthor ">Wendi  Zheng</span>, <span class="hlFld-ContribAuthor ">Zhen  Li</span>, <span class="hlFld-ContribAuthor ">Zheng  Yao</span>, <span class="hlFld-ContribAuthor ">Hongjun  Zhang</span>, <span class="hlFld-ContribAuthor ">Jia  Zheng</span>. </span><span class="cited-content_cbyCitation_article-title">PRMT5 inhibition attenuates cartilage degradation by reducing MAPK and NF-κB signaling. </span><span class="cited-content_cbyCitation_journal-name">Arthritis Research & Therapy</span><span> <strong>2020,</strong> <em>22 </em>
                                    (1)
                                     <a href="https://doi.org/10.1186/s13075-020-02304-x" title="DOI URL">https://doi.org/10.1186/s13075-020-02304-x</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1186/s13075-020-02304-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1186%2Fs13075-020-02304-x%26sid%3Dliteratum%253Aachs%26jtitle%3DArthritis%2520Research%2520%2526%2520Therapy%26atitle%3DPRMT5%252Binhibition%252Battenuates%252Bcartilage%252Bdegradation%252Bby%252Breducing%252BMAPK%252Band%252BNF-%2525CE%2525BAB%252Bsignaling%26aulast%3DDong%26aufirst%3DYonghui%26date%3D2020%26date%3D2020%26volume%3D22%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yongchul  Lim</span>, <span class="hlFld-ContribAuthor ">Yae Eun  Park</span>, <span class="hlFld-ContribAuthor ">Sujin  Ha</span>, <span class="hlFld-ContribAuthor ">Ji Eun  Lee</span>, <span class="hlFld-ContribAuthor ">Hee Cheol  Kim</span>. </span><span class="cited-content_cbyCitation_article-title">A Comprehensive Analysis of Symmetric Arginine Dimethylation in Colorectal Cancer Tissues Using Immunoaffinity Enrichment and Mass Spectrometry. </span><span class="cited-content_cbyCitation_journal-name">PROTEOMICS</span><span> <strong>2020,</strong> <em>20 </em>
                                    (15-16)
                                     , 1900367. <a href="https://doi.org/10.1002/pmic.201900367" title="DOI URL">https://doi.org/10.1002/pmic.201900367</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/pmic.201900367&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fpmic.201900367%26sid%3Dliteratum%253Aachs%26jtitle%3DPROTEOMICS%26atitle%3DA%252BComprehensive%252BAnalysis%252Bof%252BSymmetric%252BArginine%252BDimethylation%252Bin%252BColorectal%252BCancer%252BTissues%252BUsing%252BImmunoaffinity%252BEnrichment%252Band%252BMass%252BSpectrometry%26aulast%3DLim%26aufirst%3DYongchul%26date%3D2020%26date%3D2020%26volume%3D20%26issue%3D15-16%26spage%3D1900367" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Sara  Cherry</span>, <span class="hlFld-ContribAuthor ">Kristen W.  Lynch</span>. </span><span class="cited-content_cbyCitation_article-title">Alternative splicing and cancer: insights, opportunities, and challenges from an expanding view of the transcriptome. </span><span class="cited-content_cbyCitation_journal-name">Genes & Development</span><span> <strong>2020,</strong> <em>34 </em>
                                    (15-16)
                                     , 1005-1016. <a href="https://doi.org/10.1101/gad.338962.120" title="DOI URL">https://doi.org/10.1101/gad.338962.120</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1101/gad.338962.120&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1101%2Fgad.338962.120%26sid%3Dliteratum%253Aachs%26jtitle%3DGenes%2520%2526%2520Development%26atitle%3DAlternative%252Bsplicing%252Band%252Bcancer%25253A%252Binsights%25252C%252Bopportunities%25252C%252Band%252Bchallenges%252Bfrom%252Ban%252Bexpanding%252Bview%252Bof%252Bthe%252Btranscriptome%26aulast%3DCherry%26aufirst%3DSara%26date%3D2020%26date%3D2020%26volume%3D34%26issue%3D15-16%26spage%3D1005%26epage%3D1016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Marie  Hautin</span>, <span class="hlFld-ContribAuthor ">Clélia  Mornet</span>, <span class="hlFld-ContribAuthor ">Aurélie  Chauveau</span>, <span class="hlFld-ContribAuthor ">Delphine  Bernard</span>, <span class="hlFld-ContribAuthor ">Laurent  Corcos</span>, <span class="hlFld-ContribAuthor ">Eric  Lippert</span>. </span><span class="cited-content_cbyCitation_article-title">Splicing Anomalies in Myeloproliferative Neoplasms: Paving the Way for New Therapeutic Venues. </span><span class="cited-content_cbyCitation_journal-name">Cancers</span><span> <strong>2020,</strong> <em>12 </em>
                                    (8)
                                     , 2216. <a href="https://doi.org/10.3390/cancers12082216" title="DOI URL">https://doi.org/10.3390/cancers12082216</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/cancers12082216&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fcancers12082216%26sid%3Dliteratum%253Aachs%26jtitle%3DCancers%26atitle%3DSplicing%252BAnomalies%252Bin%252BMyeloproliferative%252BNeoplasms%25253A%252BPaving%252Bthe%252BWay%252Bfor%252BNew%252BTherapeutic%252BVenues%26aulast%3DHautin%26aufirst%3DMarie%26date%3D2020%26date%3D2020%26volume%3D12%26issue%3D8%26spage%3D2216" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yongbo  Wang</span>, <span class="hlFld-ContribAuthor ">Yufang  Bao</span>, <span class="hlFld-ContribAuthor ">Sirui  Zhang</span>, <span class="hlFld-ContribAuthor ">Zefeng  Wang</span>. </span><span class="cited-content_cbyCitation_article-title">Splicing dysregulation in cancer: from mechanistic understanding to a new class of therapeutic targets. </span><span class="cited-content_cbyCitation_journal-name">Science China Life Sciences</span><span> <strong>2020,</strong> <em>63 </em>
                                    (4)
                                     , 469-484. <a href="https://doi.org/10.1007/s11427-019-1605-0" title="DOI URL">https://doi.org/10.1007/s11427-019-1605-0</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s11427-019-1605-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs11427-019-1605-0%26sid%3Dliteratum%253Aachs%26jtitle%3DScience%2520China%2520Life%2520Sciences%26atitle%3DSplicing%252Bdysregulation%252Bin%252Bcancer%25253A%252Bfrom%252Bmechanistic%252Bunderstanding%252Bto%252Ba%252Bnew%252Bclass%252Bof%252Btherapeutic%252Btargets%26aulast%3DWang%26aufirst%3DYongbo%26date%3D2020%26date%3D2020%26volume%3D63%26issue%3D4%26spage%3D469%26epage%3D484" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yurui  Zhang</span>, <span class="hlFld-ContribAuthor ">Matthijs J.  van Haren</span>, <span class="hlFld-ContribAuthor ">Nathaniel I.  Martin</span>. </span><span class="cited-content_cbyCitation_article-title">Peptidic transition state analogues as PRMT inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Methods</span><span> <strong>2020,</strong> <em>175 </em>, 24-29. <a href="https://doi.org/10.1016/j.ymeth.2019.08.003" title="DOI URL">https://doi.org/10.1016/j.ymeth.2019.08.003</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ymeth.2019.08.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ymeth.2019.08.003%26sid%3Dliteratum%253Aachs%26jtitle%3DMethods%26atitle%3DPeptidic%252Btransition%252Bstate%252Banalogues%252Bas%252BPRMT%252Binhibitors%26aulast%3DZhang%26aufirst%3DYurui%26date%3D2020%26volume%3D175%26spage%3D24%26epage%3D29" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Justin  Taylor</span>, <span class="hlFld-ContribAuthor ">Stanley C.  Lee</span>. </span><span class="cited-content_cbyCitation_article-title">Mutations in spliceosome genes and therapeutic opportunities in myeloid malignancies. </span><span class="cited-content_cbyCitation_journal-name">Genes, Chromosomes and Cancer</span><span> <strong>2019,</strong> <em>58 </em>
                                    (12)
                                     , 889-902. <a href="https://doi.org/10.1002/gcc.22784" title="DOI URL">https://doi.org/10.1002/gcc.22784</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/gcc.22784&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fgcc.22784%26sid%3Dliteratum%253Aachs%26jtitle%3DGenes%252C%2520Chromosomes%2520and%2520Cancer%26atitle%3DMutations%252Bin%252Bspliceosome%252Bgenes%252Band%252Btherapeutic%252Bopportunities%252Bin%252Bmyeloid%252Bmalignancies%26aulast%3DTaylor%26aufirst%3DJustin%26date%3D2019%26date%3D2019%26volume%3D58%26issue%3D12%26spage%3D889%26epage%3D902" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Haopeng  Yang</span>, <span class="hlFld-ContribAuthor ">Michael R.  Green</span>. </span><span class="cited-content_cbyCitation_article-title">Epigenetic Programing of B-Cell Lymphoma by BCL6 and Its Genetic Deregulation. </span><span class="cited-content_cbyCitation_journal-name">Frontiers in Cell and Developmental Biology</span><span> <strong>2019,</strong> <em>7 </em><a href="https://doi.org/10.3389/fcell.2019.00272" title="DOI URL">https://doi.org/10.3389/fcell.2019.00272</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3389/fcell.2019.00272&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3389%2Ffcell.2019.00272%26sid%3Dliteratum%253Aachs%26jtitle%3DFrontiers%2520in%2520Cell%2520and%2520Developmental%2520Biology%26atitle%3DEpigenetic%252BPrograming%252Bof%252BB-Cell%252BLymphoma%252Bby%252BBCL6%252Band%252BIts%252BGenetic%252BDeregulation%26aulast%3DYang%26aufirst%3DHaopeng%26date%3D2019%26date%3D2019%26volume%3D7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Biji  Chatterjee</span>, <span class="hlFld-ContribAuthor ">Krishna  Ghosh</span>, <span class="hlFld-ContribAuthor ">Santosh R.  Kanade</span>. </span><span class="cited-content_cbyCitation_article-title">Resveratrol modulates epigenetic regulators of promoter histone methylation and acetylation that restores BRCA1, p53, p21
              CIP1
              in human breast cancer cell lines. </span><span class="cited-content_cbyCitation_journal-name">BioFactors</span><span> <strong>2019,</strong> <em>45 </em>
                                    (5)
                                     , 818-829. <a href="https://doi.org/10.1002/biof.1544" title="DOI URL">https://doi.org/10.1002/biof.1544</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/biof.1544&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fbiof.1544%26sid%3Dliteratum%253Aachs%26jtitle%3DBioFactors%26atitle%3DResveratrol%252Bmodulates%252Bepigenetic%252Bregulators%252Bof%252Bpromoter%252Bhistone%252Bmethylation%252Band%252Bacetylation%252Bthat%252Brestores%252BBRCA1%25252C%252Bp53%25252C%252Bp21%252BCIP1%252Bin%252Bhuman%252Bbreast%252Bcancer%252Bcell%252Blines%26aulast%3DChatterjee%26aufirst%3DBiji%26date%3D2019%26date%3D2019%26volume%3D45%26issue%3D5%26spage%3D818%26epage%3D829" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Sunil Kumar  Tewary</span>, <span class="hlFld-ContribAuthor ">Y. George  Zheng</span>, <span class="hlFld-ContribAuthor ">Meng-Chiao  Ho</span>. </span><span class="cited-content_cbyCitation_article-title">Protein arginine methyltransferases: insights into the enzyme structure and mechanism at the atomic level. </span><span class="cited-content_cbyCitation_journal-name">Cellular and Molecular Life Sciences</span><span> <strong>2019,</strong> <em>76 </em>
                                    (15)
                                     , 2917-2932. <a href="https://doi.org/10.1007/s00018-019-03145-x" title="DOI URL">https://doi.org/10.1007/s00018-019-03145-x</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s00018-019-03145-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs00018-019-03145-x%26sid%3Dliteratum%253Aachs%26jtitle%3DCellular%2520and%2520Molecular%2520Life%2520Sciences%26atitle%3DProtein%252Barginine%252Bmethyltransferases%25253A%252Binsights%252Binto%252Bthe%252Benzyme%252Bstructure%252Band%252Bmechanism%252Bat%252Bthe%252Batomic%252Blevel%26aulast%3DTewary%26aufirst%3DSunil%2BKumar%26date%3D2019%26date%3D2019%26volume%3D76%26issue%3D15%26spage%3D2917%26epage%3D2932" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Hong  Lin</span>, <span class="hlFld-ContribAuthor ">Juan I.  Luengo</span>. </span><span class="cited-content_cbyCitation_article-title">Nucleoside protein arginine methyltransferase 5 (PRMT5) inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry Letters</span><span> <strong>2019,</strong> <em>29 </em>
                                    (11)
                                     , 1264-1269. <a href="https://doi.org/10.1016/j.bmcl.2019.03.042" title="DOI URL">https://doi.org/10.1016/j.bmcl.2019.03.042</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmcl.2019.03.042&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmcl.2019.03.042%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DNucleoside%252Bprotein%252Barginine%252Bmethyltransferase%252B5%252B%252528PRMT5%252529%252Binhibitors%26aulast%3DLin%26aufirst%3DHong%26date%3D2019%26volume%3D29%26issue%3D11%26spage%3D1264%26epage%3D1269" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Antja-Voy  Hartley</span>, <span class="hlFld-ContribAuthor ">Matthew  Martin</span>, <span class="hlFld-ContribAuthor ">Tao  Lu</span>. </span><span class="cited-content_cbyCitation_article-title">Epigenetic Biomarkers and Their Therapeutic Applications in Colorectal Cancer. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2019,</strong>,<a href="https://doi.org/10.5772/intechopen.82572" title="DOI URL">https://doi.org/10.5772/intechopen.82572</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.5772/intechopen.82572&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.5772%2Fintechopen.82572%26sid%3Dliteratum%253Aachs%26atitle%3DEpigenetic%252BBiomarkers%252Band%252BTheir%252BTherapeutic%252BApplications%252Bin%252BColorectal%252BCancer%26aulast%3DHartley%26aufirst%3DAntja-Voy%26date%3D2019%26date%3D2019%26pub%3DIntechOpen%26atitle%3DAdvances%252Bin%252Bthe%252BMolecular%252BUnderstanding%252Bof%252BColorectal%252BCancer%26aulast%3DSegelov%26aufirst%3DEva%26date%3D2019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-21/acs.jmedchem.8b00598/20181101/images/medium/jm-2018-00598z_0014.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-21/acs.jmedchem.8b00598/20181101/images/large/jm-2018-00598z_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00598&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-21/acs.jmedchem.8b00598/20181101/images/medium/jm-2018-00598z_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-21/acs.jmedchem.8b00598/20181101/images/large/jm-2018-00598z_0001.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. (A) Arginine methylation reaction catalyzed by three types of PRMTs. (B) Chemical structures of SAM and SAH.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-21/acs.jmedchem.8b00598/20181101/images/large/jm-2018-00598z_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00598&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-21/acs.jmedchem.8b00598/20181101/images/medium/jm-2018-00598z_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-21/acs.jmedchem.8b00598/20181101/images/large/jm-2018-00598z_0002.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Domain structures of human PRMTs.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-21/acs.jmedchem.8b00598/20181101/images/large/jm-2018-00598z_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00598&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-21/acs.jmedchem.8b00598/20181101/images/medium/jm-2018-00598z_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-21/acs.jmedchem.8b00598/20181101/images/large/jm-2018-00598z_0003.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. (A) Sequence alignment of the MTase domains in human PRMTs. Conserved motifs, double E loop, and THW residues are shown in red. Residues that are separately conserved between type I PRMTs and PRMT5 are highlighted in yellow. (B) Percent identity matrix of MTase domains in human PRMTs. Numbers are colored in green-yellow-red scale with highest homology being green and lowest being red. (C) Phylogenetic tree of human PRMTs based on sequence alignment of their MTase domains.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-21/acs.jmedchem.8b00598/20181101/images/large/jm-2018-00598z_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00598&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-21/acs.jmedchem.8b00598/20181101/images/medium/jm-2018-00598z_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-21/acs.jmedchem.8b00598/20181101/images/large/jm-2018-00598z_0004.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. (A) Overall structure of PRMT5:MEP50 complex (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4GQB">4GQB</a>). PRMT5 catalytic domain is colored in green, N terminal TIM barrel is colored in purple, MEP50 is colored in magenta. Highlighted in orange is the linker between the N-terminal TIM barrel and C-terminal MTase catalytic domain of PRMT5. The H4 peptide is shown in stick presentation, colored in magenta, blue, and red. (B) Zoomed-in view of H4R3 substrate peptide binding groove.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-21/acs.jmedchem.8b00598/20181101/images/large/jm-2018-00598z_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00598&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-21/acs.jmedchem.8b00598/20181101/images/medium/jm-2018-00598z_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-21/acs.jmedchem.8b00598/20181101/images/large/jm-2018-00598z_0005.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. (A) Structures of compounds <b>1</b>–<b>3</b>. (B) Cocrystal structure of compound <b>1</b> with PRMT5:MEP50 and SAM (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4X61">4X61</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-21/acs.jmedchem.8b00598/20181101/images/large/jm-2018-00598z_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00598&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-21/acs.jmedchem.8b00598/20181101/images/medium/jm-2018-00598z_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-21/acs.jmedchem.8b00598/20181101/images/large/jm-2018-00598z_0006.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Structures of compounds <b>4</b>–<b>8</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-21/acs.jmedchem.8b00598/20181101/images/large/jm-2018-00598z_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00598&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-21/acs.jmedchem.8b00598/20181101/images/medium/jm-2018-00598z_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-21/acs.jmedchem.8b00598/20181101/images/large/jm-2018-00598z_0007.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Structures of compounds <b>9</b> and <b>10</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-21/acs.jmedchem.8b00598/20181101/images/large/jm-2018-00598z_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00598&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig8" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-21/acs.jmedchem.8b00598/20181101/images/medium/jm-2018-00598z_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-21/acs.jmedchem.8b00598/20181101/images/large/jm-2018-00598z_0008.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Structures of compounds <b>11</b>–<b>13</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-21/acs.jmedchem.8b00598/20181101/images/large/jm-2018-00598z_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00598&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig9" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-21/acs.jmedchem.8b00598/20181101/images/medium/jm-2018-00598z_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-21/acs.jmedchem.8b00598/20181101/images/large/jm-2018-00598z_0009.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. Structures of compounds <b>14</b>–<b>16</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-21/acs.jmedchem.8b00598/20181101/images/large/jm-2018-00598z_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00598&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig10" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 10</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-21/acs.jmedchem.8b00598/20181101/images/medium/jm-2018-00598z_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-21/acs.jmedchem.8b00598/20181101/images/large/jm-2018-00598z_0010.jpeg" id="rightTab-gr10" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 10. (A) Structure of compounds <b>17</b> and <b>18</b>. (B) Cocrystal structure of compound <b>18</b> with PRMT5:MEP50 in complex with H4 peptide (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4GQB">4GQB</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-21/acs.jmedchem.8b00598/20181101/images/large/jm-2018-00598z_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00598&amp;id=fig10"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig11" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 11</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-21/acs.jmedchem.8b00598/20181101/images/medium/jm-2018-00598z_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-21/acs.jmedchem.8b00598/20181101/images/large/jm-2018-00598z_0011.jpeg" id="rightTab-gr11" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 11. Structures of compounds <b>19</b>–<b>21</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-21/acs.jmedchem.8b00598/20181101/images/large/jm-2018-00598z_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00598&amp;id=fig11"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig12" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Figure 12</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-21/acs.jmedchem.8b00598/20181101/images/medium/jm-2018-00598z_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-21/acs.jmedchem.8b00598/20181101/images/large/jm-2018-00598z_0012.jpeg" id="rightTab-gr12" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 12. Structures of compounds <b>22</b>–<b>25</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-21/acs.jmedchem.8b00598/20181101/images/large/jm-2018-00598z_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00598&amp;id=fig12"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig13" data-index="13" class="article__inlineFigure"><h2 class="fig-label">Figure 13</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-21/acs.jmedchem.8b00598/20181101/images/medium/jm-2018-00598z_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-21/acs.jmedchem.8b00598/20181101/images/large/jm-2018-00598z_0013.jpeg" id="rightTab-gr13" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 13. Structures of compounds <b>26–31</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-21/acs.jmedchem.8b00598/20181101/images/large/jm-2018-00598z_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00598&amp;id=fig13"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i16">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_07640" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_07640" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 75 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Peng, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, C. C. L.</span></span> <span> </span><span class="NLM_article-title">The story of protein arginine methylation: characterization, regulation, and function</span>. <i>Expert Rev. Proteomics</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">157</span>– <span class="NLM_lpage">170</span>, <span class="refDoi"> DOI: 10.1080/14789450.2017.1275573</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=10.1080%2F14789450.2017.1275573" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=28043171" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=1%3ACAS%3A528%3ADC%252BC2sXltVWluw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2017&pages=157-170&author=C.+Pengauthor=C.+C.+L.+Wong&title=The+story+of+protein+arginine+methylation%3A+characterization%2C+regulation%2C+and+function&doi=10.1080%2F14789450.2017.1275573"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">The story of protein arginine methylation: characterization, regulation, and function</span></div><div class="casAuthors">Peng, Chao; Wong, Catherine C. L.</div><div class="citationInfo"><span class="NLM_cas:title">Expert Review of Proteomics</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">157-170</span>CODEN:
                <span class="NLM_cas:coden">ERPXA3</span>;
        ISSN:<span class="NLM_cas:issn">1478-9450</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">Arginine methylation is an important post-translational modification (PTM) in cells, which is catalyzed by a group of protein arginine methyltransferases (PRMTs).  It plays significant roles in diverse cellular processes and various diseases.  Misregulation and aberrant expression of PRMTs can provide potential biomarkers and therapeutic targets for drug discovery.  Herein, we review the arginine methylation literature and summarize the methodologies for the characterization of this modification, as well as describe the recent insights into arginine methyltransferases and their biol. functions in diseases.  Benefits from the enzyme-based large-scale screening approach, the novel affinity enrichment strategies, arginine methylated protein family is the focus of attention.  Although a no. of arginine methyltransferases and related substrates are identified, the catalytic mechanism of different types of PRMTs remains unclear and few related demethylases are characterized.  Novel functional studies continuously reveal the importance of this modification in the cell cycle and diseases.  A deeper understanding of arginine methylated proteins, modification sites, and their mechanisms of regulation is needed to explore their role in life processes, esp. their relationship with diseases, thus accelerating the generation of potent, selective, cell-penetrant drug candidates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGriWhBKuJixw7Vg90H21EOLACvtfcHk0lg6hgzBt8WcRQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXltVWluw%253D%253D&md5=9f22ba72f797e050f75000112fbe4147</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1080%2F14789450.2017.1275573&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F14789450.2017.1275573%26sid%3Dliteratum%253Aachs%26aulast%3DPeng%26aufirst%3DC.%26aulast%3DWong%26aufirst%3DC.%2BC.%2BL.%26atitle%3DThe%2520story%2520of%2520protein%2520arginine%2520methylation%253A%2520characterization%252C%2520regulation%252C%2520and%2520function%26jtitle%3DExpert%2520Rev.%2520Proteomics%26date%3D2017%26volume%3D14%26spage%3D157%26epage%3D170%26doi%3D10.1080%2F14789450.2017.1275573" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Larsen, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sylvestersen, K. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mund, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lyon, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mullari, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Madsen, M. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daniel, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jensen, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nielsen, M. L.</span></span> <span> </span><span class="NLM_article-title">Proteome-wide analysis of arginine monomethylation reveals widespread occurrence in human cells</span>. <i>Sci. Signaling</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">rs9</span>, <span class="refDoi"> DOI: 10.1126/scisignal.aaf7329</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=10.1126%2Fscisignal.aaf7329" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=27577262" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2016&pages=rs9&author=S.+C.+Larsenauthor=K.+B.+Sylvestersenauthor=A.+Mundauthor=D.+Lyonauthor=M.+Mullariauthor=M.+V.+Madsenauthor=J.+A.+Danielauthor=L.+J.+Jensenauthor=M.+L.+Nielsen&title=Proteome-wide+analysis+of+arginine+monomethylation+reveals+widespread+occurrence+in+human+cells&doi=10.1126%2Fscisignal.aaf7329"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1126%2Fscisignal.aaf7329&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscisignal.aaf7329%26sid%3Dliteratum%253Aachs%26aulast%3DLarsen%26aufirst%3DS.%2BC.%26aulast%3DSylvestersen%26aufirst%3DK.%2BB.%26aulast%3DMund%26aufirst%3DA.%26aulast%3DLyon%26aufirst%3DD.%26aulast%3DMullari%26aufirst%3DM.%26aulast%3DMadsen%26aufirst%3DM.%2BV.%26aulast%3DDaniel%26aufirst%3DJ.%2BA.%26aulast%3DJensen%26aufirst%3DL.%2BJ.%26aulast%3DNielsen%26aufirst%3DM.%2BL.%26atitle%3DProteome-wide%2520analysis%2520of%2520arginine%2520monomethylation%2520reveals%2520widespread%2520occurrence%2520in%2520human%2520cells%26jtitle%3DSci.%2520Signaling%26date%3D2016%26volume%3D9%26spage%3Drs9%26doi%3D10.1126%2Fscisignal.aaf7329" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Guo, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gu, H. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mulhern, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aguiar, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kornhauser, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jia, X. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ren, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beausoleil, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Silva, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vemulapalli, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bedford, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Comb, M. J.</span></span> <span> </span><span class="NLM_article-title">Immunoaffinity enrichment and mass spectrometry analysis of protein methylation</span>. <i>Mol. Cell. Proteomics</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">372</span>– <span class="NLM_lpage">387</span>, <span class="refDoi"> DOI: 10.1074/mcp.O113.027870</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=10.1074%2Fmcp.O113.027870" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=24129315" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=1%3ACAS%3A528%3ADC%252BC2cXit1ynug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2014&pages=372-387&author=A.+L.+Guoauthor=H.+B.+Guauthor=J.+Zhouauthor=D.+Mulhernauthor=Y.+Wangauthor=K.+A.+Leeauthor=V.+Yangauthor=M.+Aguiarauthor=J.+Kornhauserauthor=X.+Y.+Jiaauthor=J.+M.+Renauthor=S.+A.+Beausoleilauthor=J.+C.+Silvaauthor=V.+Vemulapalliauthor=M.+T.+Bedfordauthor=M.+J.+Comb&title=Immunoaffinity+enrichment+and+mass+spectrometry+analysis+of+protein+methylation&doi=10.1074%2Fmcp.O113.027870"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Immunoaffinity Enrichment and Mass Spectrometry Analysis of Protein Methylation</span></div><div class="casAuthors">Guo, Ailan; Gu, Hongbo; Zhou, Jing; Mulhern, Daniel; Wang, Yi; Lee, Kimberly A.; Yang, Vicky; Aguiar, Mike; Kornhauser, Jon; Jia, Xiaoying; Ren, Jianmin; Beausoleil, Sean A.; Silva, Jeffrey C.; Vemulapalli, Vidyasiri; Bedford, Mark T.; Comb, Michael J.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular & Cellular Proteomics</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">372-387</span>CODEN:
                <span class="NLM_cas:coden">MCPOBS</span>;
        ISSN:<span class="NLM_cas:issn">1535-9484</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Protein methylation is a common posttranslational modification that mostly occurs on arginine and lysine residues.  Arginine methylation has been reported to regulate RNA processing, gene transcription, DNA damage repair, protein translocation, and signal transduction.  Lysine methylation is best known to regulate histone function and is involved in epigenetic regulation of gene transcription.  To better study protein methylation, we have developed highly specific antibodies against monomethyl arginine; asym. di-Me arginine; and monomethyl, di-Me, and tri-Me lysine motifs.  These antibodies were used to perform immunoaffinity purifn. of Me peptides followed by LC-MS/MS anal. to identify and quantify arginine and lysine methylation sites in several model studies.  Overall, we identified over 1000 arginine methylation sites in human cell line and mouse tissues, and ∼160 lysine methylation sites in human cell line HCT116.  The no. of methylation sites identified in this study exceeds those found in the literature to date.  Detailed anal. of arginine-methylated proteins obsd. in mouse brain compared with those found in mouse embryo shows a tissue-specific distribution of arginine methylation, and extends the types of proteins that are known to be arginine methylated to include many new protein types.  Many arginine-methylated proteins that we identified from the brain, including receptors, ion channels, transporters, and vesicle proteins, are involved in synaptic transmission, whereas the most abundant methylated proteins identified from mouse embryo are transcriptional regulators and RNA processing proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpBdVRZ74GxBrVg90H21EOLACvtfcHk0liXoCV7X33DnQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXit1ynug%253D%253D&md5=244c97ca8bd924f6cb6071908a7d38e4</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1074%2Fmcp.O113.027870&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fmcp.O113.027870%26sid%3Dliteratum%253Aachs%26aulast%3DGuo%26aufirst%3DA.%2BL.%26aulast%3DGu%26aufirst%3DH.%2BB.%26aulast%3DZhou%26aufirst%3DJ.%26aulast%3DMulhern%26aufirst%3DD.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DLee%26aufirst%3DK.%2BA.%26aulast%3DYang%26aufirst%3DV.%26aulast%3DAguiar%26aufirst%3DM.%26aulast%3DKornhauser%26aufirst%3DJ.%26aulast%3DJia%26aufirst%3DX.%2BY.%26aulast%3DRen%26aufirst%3DJ.%2BM.%26aulast%3DBeausoleil%26aufirst%3DS.%2BA.%26aulast%3DSilva%26aufirst%3DJ.%2BC.%26aulast%3DVemulapalli%26aufirst%3DV.%26aulast%3DBedford%26aufirst%3DM.%2BT.%26aulast%3DComb%26aufirst%3DM.%2BJ.%26atitle%3DImmunoaffinity%2520enrichment%2520and%2520mass%2520spectrometry%2520analysis%2520of%2520protein%2520methylation%26jtitle%3DMol.%2520Cell.%2520Proteomics%26date%3D2014%26volume%3D13%26spage%3D372%26epage%3D387%26doi%3D10.1074%2Fmcp.O113.027870" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McBride, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Silver, P. A.</span></span> <span> </span><span class="NLM_article-title">State of the Arg: Protein methylation at arginine comes of age</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>106</i></span>,  <span class="NLM_fpage">5</span>– <span class="NLM_lpage">8</span>, <span class="refDoi"> DOI: 10.1016/S0092-8674(01)00423-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=10.1016%2FS0092-8674%2801%2900423-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=11461695" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=1%3ACAS%3A528%3ADC%252BD3MXlsVCqsbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=106&publication_year=2001&pages=5-8&author=A.+E.+McBrideauthor=P.+A.+Silver&title=State+of+the+Arg%3A+Protein+methylation+at+arginine+comes+of+age&doi=10.1016%2FS0092-8674%2801%2900423-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">State of the arg: protein methylation at arginine comes of age</span></div><div class="casAuthors">McBride, Anne E.; Silver, Pamela A.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">106</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">5-8</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review with 20 refs.  Posttranslational modification of proteins allows the cell to expand its repertoire beyond the constraints imposed by the twenty encoded amino acids.  Methylation at arginines, although discovered over 30 yr ago, has only recently come to the attention of cell biologists through a combination of genetic and mol. biol. expts. that have implicated arginine methylation in processes from signaling and transcription activation to protein sorting.  The panoply of arginine methylated substrates suggests that this specifically eukaryotic modification may parallel phosphorylation in its level of complexity.  We will summarize much of the recent information about the function of methylation and the methyltransferase enzymes that modify arginines.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpL9vHYeFQ-aLVg90H21EOLACvtfcHk0lhK6g4V78o3yQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXlsVCqsbY%253D&md5=e2191f38b78db27b75b3af07684ea3cb</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1016%2FS0092-8674%2801%2900423-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0092-8674%252801%252900423-8%26sid%3Dliteratum%253Aachs%26aulast%3DMcBride%26aufirst%3DA.%2BE.%26aulast%3DSilver%26aufirst%3DP.%2BA.%26atitle%3DState%2520of%2520the%2520Arg%253A%2520Protein%2520methylation%2520at%2520arginine%2520comes%2520of%2520age%26jtitle%3DCell%26date%3D2001%26volume%3D106%26spage%3D5%26epage%3D8%26doi%3D10.1016%2FS0092-8674%2801%2900423-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Auclair, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richard, S.</span></span> <span> </span><span class="NLM_article-title">The role of arginine methylation in the DNA damage response</span>. <i>DNA Repair</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">459</span>– <span class="NLM_lpage">465</span>, <span class="refDoi"> DOI: 10.1016/j.dnarep.2013.04.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=10.1016%2Fj.dnarep.2013.04.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=23684798" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=1%3ACAS%3A528%3ADC%252BC3sXnvVGksbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2013&pages=459-465&author=Y.+Auclairauthor=S.+Richard&title=The+role+of+arginine+methylation+in+the+DNA+damage+response&doi=10.1016%2Fj.dnarep.2013.04.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">The role of arginine methylation in the DNA damage response</span></div><div class="casAuthors">Auclair, Yannick; Richard, Stephane</div><div class="citationInfo"><span class="NLM_cas:title">DNA Repair</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">459-465</span>CODEN:
                <span class="NLM_cas:coden">DRNEAR</span>;
        ISSN:<span class="NLM_cas:issn">1568-7864</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Post-translational modifications are well-known modulators of DNA damage signaling and epigenetic gene expression.  Protein arginine methylation is a covalent modification that results in the addn. of Me groups to the nitrogen atoms of the arginine side chains and is catalyzed by a family of protein arginine methyltransferases (PRMTs).  In the past, arginine methylation was mainly obsd. on abundant proteins such as RNA-binding proteins and histones, but recent advances have revealed a plethora of arginine methylated proteins implicated in a variety of cellular processes including RNA metab., epigenetic regulation and DNA repair pathways.  Herein, we discuss these recent advances, focusing on the role of PRMTs in DNA damage signaling and its importance for maintaining genomic stability.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGryDaGBPsGY7rVg90H21EOLACvtfcHk0lhK6g4V78o3yQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXnvVGksbg%253D&md5=9b45a150e051f5687df109c8011e286e</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1016%2Fj.dnarep.2013.04.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.dnarep.2013.04.006%26sid%3Dliteratum%253Aachs%26aulast%3DAuclair%26aufirst%3DY.%26aulast%3DRichard%26aufirst%3DS.%26atitle%3DThe%2520role%2520of%2520arginine%2520methylation%2520in%2520the%2520DNA%2520damage%2520response%26jtitle%3DDNA%2520Repair%26date%3D2013%26volume%3D12%26spage%3D459%26epage%3D465%26doi%3D10.1016%2Fj.dnarep.2013.04.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Blanc, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richard, S.</span></span> <span> </span><span class="NLM_article-title">Arginine Methylation: The coming of age</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>65</i></span>,  <span class="NLM_fpage">8</span>– <span class="NLM_lpage">24</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2016.11.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=10.1016%2Fj.molcel.2016.11.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=28061334" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=1%3ACAS%3A528%3ADC%252BC2sXmsVKmtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=65&publication_year=2017&pages=8-24&author=R.+S.+Blancauthor=S.+Richard&title=Arginine+Methylation%3A+The+coming+of+age&doi=10.1016%2Fj.molcel.2016.11.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Arginine Methylation: The Coming of Age</span></div><div class="casAuthors">Blanc, Romeo S.; Richard, Stephane</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">65</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">8-24</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Arginine methylation is a common post-translational modification functioning as an epigenetic regulator of transcription and playing key roles in pre-mRNA splicing, DNA damage signaling, mRNA translation, cell signaling, and cell fate decision.  Recently, a wealth of studies using transgenic mouse models and selective PRMT inhibitors helped define physiol. roles for protein arginine methyltransferases (PRMTs) linking them to diseases such as cancer and metabolic, neurodegenerative, and muscular disorders.  This review describes the recent mol. advances that have been uncovered in normal and diseased mammalian cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqT2OQsTa2UnbVg90H21EOLACvtfcHk0ljNjJnm4Yps5Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXmsVKmtA%253D%253D&md5=7ca3a6b7f0914c0034534ab8ca908506</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2016.11.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2016.11.003%26sid%3Dliteratum%253Aachs%26aulast%3DBlanc%26aufirst%3DR.%2BS.%26aulast%3DRichard%26aufirst%3DS.%26atitle%3DArginine%2520Methylation%253A%2520The%2520coming%2520of%2520age%26jtitle%3DMol.%2520Cell%26date%3D2017%26volume%3D65%26spage%3D8%26epage%3D24%26doi%3D10.1016%2Fj.molcel.2016.11.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wesche, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuhn, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kessler, B. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salton, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wolf, A.</span></span> <span> </span><span class="NLM_article-title">Protein arginine methylation: a prominent modification and its demethylation</span>. <i>Cell. Mol. Life Sci.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>74</i></span>,  <span class="NLM_fpage">3305</span>– <span class="NLM_lpage">3315</span>, <span class="refDoi"> DOI: 10.1007/s00018-017-2515-z</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=10.1007%2Fs00018-017-2515-z" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=28364192" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=1%3ACAS%3A528%3ADC%252BC2sXlsV2nu7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=74&publication_year=2017&pages=3305-3315&author=J.+Wescheauthor=S.+Kuhnauthor=B.+M.+Kesslerauthor=M.+Saltonauthor=A.+Wolf&title=Protein+arginine+methylation%3A+a+prominent+modification+and+its+demethylation&doi=10.1007%2Fs00018-017-2515-z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Protein arginine methylation: a prominent modification and its demethylation</span></div><div class="casAuthors">Wesche, Juste; Kuehn, Sarah; Kessler, Benedikt M.; Salton, Maayan; Wolf, Alexander</div><div class="citationInfo"><span class="NLM_cas:title">Cellular and Molecular Life Sciences</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">74</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">3305-3315</span>CODEN:
                <span class="NLM_cas:coden">CMLSFI</span>;
        ISSN:<span class="NLM_cas:issn">1420-682X</span>.
    
            (<span class="NLM_cas:orgname">Birkhaeuser Basel</span>)
        </div><div class="casAbstract">Arginine methylation of histones is one mechanism of epigenetic regulation in eukaryotic cells.  Methylarginines can also be found in non-histone proteins involved in various different processes in a cell.  An enzyme family of nine protein arginine methyltransferases catalyzes the addn. of Me groups on arginines of histone and non-histone proteins, resulting in either mono- or dimethylated-arginine residues.  The reversibility of histone modifications is an essential feature of epigenetic regulation to respond to changes in environmental factors, signalling events, or metabolic alterations.  Prominent histone modifications like lysine acetylation and lysine methylation are reversible.  Enzyme family pairs have been identified, with each pair of lysine acetyltransferases/deacetylases and lysine methyltransferases/demethylases operating complementarily to generate or erase lysine modifications.  Several analyses also indicate a reversible nature of arginine methylation, but the enzymes facilitating direct removal of Me moieties from arginine residues in proteins have been discussed controversially.  Differing reports have been seen for initially characterized putative candidates, like peptidyl arginine deiminase 4 or Jumonji-domain contg. protein 6.  Here, we review the most recent cellular, biochem., and mass spectrometry work on arginine methylation and its reversible nature with a special focus on putative arginine demethylases, including the enzyme superfamily of Fe(II) and 2-oxoglutarate-dependent oxygenases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqRNzVOWOcEu7Vg90H21EOLACvtfcHk0ljNjJnm4Yps5Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXlsV2nu7s%253D&md5=bf37e8cf8a26c27e86fd48592c084772</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1007%2Fs00018-017-2515-z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00018-017-2515-z%26sid%3Dliteratum%253Aachs%26aulast%3DWesche%26aufirst%3DJ.%26aulast%3DKuhn%26aufirst%3DS.%26aulast%3DKessler%26aufirst%3DB.%2BM.%26aulast%3DSalton%26aufirst%3DM.%26aulast%3DWolf%26aufirst%3DA.%26atitle%3DProtein%2520arginine%2520methylation%253A%2520a%2520prominent%2520modification%2520and%2520its%2520demethylation%26jtitle%3DCell.%2520Mol.%2520Life%2520Sci.%26date%3D2017%26volume%3D74%26spage%3D3305%26epage%3D3315%26doi%3D10.1007%2Fs00018-017-2515-z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jahan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davie, J. R.</span></span> <span> </span><span class="NLM_article-title">Protein arginine methyltransferases (PRMTs): role in chromatin organization</span>. <i>Adv. Biol. Regul.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">173</span>– <span class="NLM_lpage">184</span>, <span class="refDoi"> DOI: 10.1016/j.jbior.2014.09.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=10.1016%2Fj.jbior.2014.09.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=25263650" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsF2lsrrN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2015&pages=173-184&author=S.+Jahanauthor=J.+R.+Davie&title=Protein+arginine+methyltransferases+%28PRMTs%29%3A+role+in+chromatin+organization&doi=10.1016%2Fj.jbior.2014.09.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Protein arginine methyltransferases (PRMTs): Role in chromatin organization</span></div><div class="casAuthors">Jahan, Sanzida; Davie, James R.</div><div class="citationInfo"><span class="NLM_cas:title">Advances in Biological Regulation</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">173-184</span>CODEN:
                <span class="NLM_cas:coden">ABRDE5</span>;
        ISSN:<span class="NLM_cas:issn">2212-4934</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  The mammalian genome encodes eleven protein arginine methyltransferases (PRMTs) that are involved in the transfer of a Me group from S-adenosylmethionine (AdoMet) to the guanidino nitrogen of arginine.  The substrates for these enzymes range from histones to several nuclear and cytoplasmic proteins.  Methylation of histones by PRMTs can block the docking site for other reader/effector mols. and thus this modification can interfere with histone code orchestration.  Several members of the PRMTs have roles in chromatin organization and function.  Although PRMT aberrant expression is correlated with several diseases including cancer, the underlying mechanisms are still obscure in most cases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq3AjcsRV_REbVg90H21EOLACvtfcHk0ljNjJnm4Yps5Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsF2lsrrN&md5=4758e1ffecb87e8f4ac6275eb3487971</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1016%2Fj.jbior.2014.09.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jbior.2014.09.003%26sid%3Dliteratum%253Aachs%26aulast%3DJahan%26aufirst%3DS.%26aulast%3DDavie%26aufirst%3DJ.%2BR.%26atitle%3DProtein%2520arginine%2520methyltransferases%2520%2528PRMTs%2529%253A%2520role%2520in%2520chromatin%2520organization%26jtitle%3DAdv.%2520Biol.%2520Regul.%26date%3D2015%26volume%3D57%26spage%3D173%26epage%3D184%26doi%3D10.1016%2Fj.jbior.2014.09.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mowen, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">David, M.</span></span> <span> </span><span class="NLM_article-title">Unconventional post-translational modifications in immunological signaling</span>. <i>Nat. Immunol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">512</span>– <span class="NLM_lpage">520</span>, <span class="refDoi"> DOI: 10.1038/ni.2873</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=10.1038%2Fni.2873" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=24840982" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=1%3ACAS%3A528%3ADC%252BC2cXot1ymsbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2014&pages=512-520&author=K.+A.+Mowenauthor=M.+David&title=Unconventional+post-translational+modifications+in+immunological+signaling&doi=10.1038%2Fni.2873"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Unconventional post-translational modifications in immunological signaling</span></div><div class="casAuthors">Mowen, Kerri A.; David, Michael</div><div class="citationInfo"><span class="NLM_cas:title">Nature Immunology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">512-520</span>CODEN:
                <span class="NLM_cas:coden">NIAMCZ</span>;
        ISSN:<span class="NLM_cas:issn">1529-2908</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The activity of a cell is governed by the signals it receives from the extracellular milieu, which are 'translated' into the appropriate biol. output, such as activation, survival, proliferation, migration or differentiation.  Signaling pathways are responsible for converting environmental cues into discrete intracellular events.  The alteration of existing proteins by post-translational modification (PTM) is a key feature of signal-transduction pathways that allows the modulation of protein function.  Research into PTMs has long been dominated by the investigation of protein phosphorylation; other PTMs, such as methylation of lysine and arginine residues, acetylation, and nitrosylation of thiol groups and tyrosine residues, have received comparatively little attention.  This Review aims to present an overview of these PTMs, with an emphasis on their role in cells of the immune system.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr3jHiQm3Mb-7Vg90H21EOLACvtfcHk0ljghAScKcEIGA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXot1ymsbs%253D&md5=947c593778f88707f7fb604459def4e4</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1038%2Fni.2873&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fni.2873%26sid%3Dliteratum%253Aachs%26aulast%3DMowen%26aufirst%3DK.%2BA.%26aulast%3DDavid%26aufirst%3DM.%26atitle%3DUnconventional%2520post-translational%2520modifications%2520in%2520immunological%2520signaling%26jtitle%3DNat.%2520Immunol.%26date%3D2014%26volume%3D15%26spage%3D512%26epage%3D520%26doi%3D10.1038%2Fni.2873" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wei, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mundade, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lange, K. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, T.</span></span> <span> </span><span class="NLM_article-title">Protein arginine methylation of non-histone proteins and its role in diseases</span>. <i>Cell Cycle</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">32</span>– <span class="NLM_lpage">41</span>, <span class="refDoi"> DOI: 10.4161/cc.27353</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=10.4161%2Fcc.27353" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=24296620" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=1%3ACAS%3A528%3ADC%252BC2cXltFeruro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2014&pages=32-41&author=H.+Weiauthor=R.+Mundadeauthor=K.+C.+Langeauthor=T.+Lu&title=Protein+arginine+methylation+of+non-histone+proteins+and+its+role+in+diseases&doi=10.4161%2Fcc.27353"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Protein arginine methylation of non-histone proteins and its role in diseases</span></div><div class="casAuthors">Wei, Han; Mundade, Rasika; Lange, Kevin C.; Lu, Tao</div><div class="citationInfo"><span class="NLM_cas:title">Cell Cycle</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">32-41</span>CODEN:
                <span class="NLM_cas:coden">CCEYAS</span>;
        ISSN:<span class="NLM_cas:issn">1538-4101</span>.
    
            (<span class="NLM_cas:orgname">Landes Bioscience</span>)
        </div><div class="casAbstract">A review.  Protein arginine methyltransferases (PRMTs) are a family of enzymes that can methylate arginine residues on histones and other proteins.  PRMTs play a crucial role in influencing various cellular functions, including cellular development and tumorigenesis.  Arginine methylation by PRMTs is found on both nuclear and cytoplasmic proteins.  Recently, there is increasing evidence regarding post-translational modifications of non-histone proteins by PRMTs, illustrating the previously unknown importance of PRMTs in the regulation of various cellular functions by post-translational modifications.  In this review, we present the recent developments in the regulation of non-histone proteins by PRMTs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrIW3UyE1q0IbVg90H21EOLACvtfcHk0ljghAScKcEIGA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXltFeruro%253D&md5=55f3ea91e0b85ccf07b66eed1c58e471</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.4161%2Fcc.27353&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4161%252Fcc.27353%26sid%3Dliteratum%253Aachs%26aulast%3DWei%26aufirst%3DH.%26aulast%3DMundade%26aufirst%3DR.%26aulast%3DLange%26aufirst%3DK.%2BC.%26aulast%3DLu%26aufirst%3DT.%26atitle%3DProtein%2520arginine%2520methylation%2520of%2520non-histone%2520proteins%2520and%2520its%2520role%2520in%2520diseases%26jtitle%3DCell%2520Cycle%26date%3D2014%26volume%3D13%26spage%3D32%26epage%3D41%26doi%3D10.4161%2Fcc.27353" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Blackwell, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ceman, S.</span></span> <span> </span><span class="NLM_article-title">Arginine methylation of RNA-binding proteins regulates cell function and differentiation</span>. <i>Mol. Reprod. Dev.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>79</i></span>,  <span class="NLM_fpage">163</span>– <span class="NLM_lpage">175</span>, <span class="refDoi"> DOI: 10.1002/mrd.22024</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=10.1002%2Fmrd.22024" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=79&publication_year=2012&pages=163-175&author=E.+Blackwellauthor=S.+Ceman&title=Arginine+methylation+of+RNA-binding+proteins+regulates+cell+function+and+differentiation&doi=10.1002%2Fmrd.22024"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1002%2Fmrd.22024&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fmrd.22024%26sid%3Dliteratum%253Aachs%26aulast%3DBlackwell%26aufirst%3DE.%26aulast%3DCeman%26aufirst%3DS.%26atitle%3DArginine%2520methylation%2520of%2520RNA-binding%2520proteins%2520regulates%2520cell%2520function%2520and%2520differentiation%26jtitle%3DMol.%2520Reprod.%2520Dev.%26date%3D2012%26volume%3D79%26spage%3D163%26epage%3D175%26doi%3D10.1002%2Fmrd.22024" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zurita-Lopez, C. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sandberg, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kelly, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clarke, S. G.</span></span> <span> </span><span class="NLM_article-title">Human protein arginine methyltransferase 7 (PRMT7) is a type III enzyme forming omega-NG-monomethylated arginine residues</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>287</i></span>,  <span class="NLM_fpage">7859</span>– <span class="NLM_lpage">7870</span>, <span class="refDoi"> DOI: 10.1074/jbc.M111.336271</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=10.1074%2Fjbc.M111.336271" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=287&publication_year=2012&pages=7859-7870&author=C.+I.+Zurita-Lopezauthor=T.+Sandbergauthor=R.+Kellyauthor=S.+G.+Clarke&title=Human+protein+arginine+methyltransferase+7+%28PRMT7%29+is+a+type+III+enzyme+forming+omega-NG-monomethylated+arginine+residues&doi=10.1074%2Fjbc.M111.336271"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M111.336271&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M111.336271%26sid%3Dliteratum%253Aachs%26aulast%3DZurita-Lopez%26aufirst%3DC.%2BI.%26aulast%3DSandberg%26aufirst%3DT.%26aulast%3DKelly%26aufirst%3DR.%26aulast%3DClarke%26aufirst%3DS.%2BG.%26atitle%3DHuman%2520protein%2520arginine%2520methyltransferase%25207%2520%2528PRMT7%2529%2520is%2520a%2520type%2520III%2520enzyme%2520forming%2520omega-NG-monomethylated%2520arginine%2520residues%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2012%26volume%3D287%26spage%3D7859%26epage%3D7870%26doi%3D10.1074%2Fjbc.M111.336271" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hadjikyriacou, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Y. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Espejo, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bedford, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clarke, S. G.</span></span> <span> </span><span class="NLM_article-title">Unique features of human protein arginine methyltransferase 9 (PRMT9) and its substrate RNA splicing factor SF3B2</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>290</i></span>,  <span class="NLM_fpage">16723</span>– <span class="NLM_lpage">16743</span>, <span class="refDoi"> DOI: 10.1074/jbc.M115.659433</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=10.1074%2Fjbc.M115.659433" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=290&publication_year=2015&pages=16723-16743&author=A.+Hadjikyriacouauthor=Y.+Z.+Yangauthor=A.+Espejoauthor=M.+T.+Bedfordauthor=S.+G.+Clarke&title=Unique+features+of+human+protein+arginine+methyltransferase+9+%28PRMT9%29+and+its+substrate+RNA+splicing+factor+SF3B2&doi=10.1074%2Fjbc.M115.659433"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M115.659433&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M115.659433%26sid%3Dliteratum%253Aachs%26aulast%3DHadjikyriacou%26aufirst%3DA.%26aulast%3DYang%26aufirst%3DY.%2BZ.%26aulast%3DEspejo%26aufirst%3DA.%26aulast%3DBedford%26aufirst%3DM.%2BT.%26aulast%3DClarke%26aufirst%3DS.%2BG.%26atitle%3DUnique%2520features%2520of%2520human%2520protein%2520arginine%2520methyltransferase%25209%2520%2528PRMT9%2529%2520and%2520its%2520substrate%2520RNA%2520splicing%2520factor%2520SF3B2%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2015%26volume%3D290%26spage%3D16723%26epage%3D16743%26doi%3D10.1074%2Fjbc.M115.659433" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bedford, M. T.</span></span> <span> </span><span class="NLM_article-title">Arginine methylation at a glance</span>. <i>J. Cell Sci.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>120</i></span>,  <span class="NLM_fpage">4243</span>– <span class="NLM_lpage">4246</span>, <span class="refDoi"> DOI: 10.1242/jcs.019885</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=10.1242%2Fjcs.019885" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=18057026" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=1%3ACAS%3A528%3ADC%252BD1cXns1GktA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=120&publication_year=2007&pages=4243-4246&author=M.+T.+Bedford&title=Arginine+methylation+at+a+glance&doi=10.1242%2Fjcs.019885"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Arginine methylation at a glance</span></div><div class="casAuthors">Bedford, Mark T.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Cell Science</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">120</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">4243-4246</span>CODEN:
                <span class="NLM_cas:coden">JNCSAI</span>;
        ISSN:<span class="NLM_cas:issn">0021-9533</span>.
    
            (<span class="NLM_cas:orgname">Company of Biologists Ltd.</span>)
        </div><div class="casAbstract">A review on the enzymes involved in methylation of arginine is presented.  Many different isoenzymes of the PRMT (protein arginine N Me transferase) from PRMT1 to PRMT9 are discussed.  Some of the substrates of the PRMT enzymes are also discussed along with a brief discussion of the role of protein protein interactions in regulation of arginine methylation is also given.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpVWHJj0lIjM7Vg90H21EOLACvtfcHk0lhEjoValowr4Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXns1GktA%253D%253D&md5=d6adb4f9193c22342c1d30f16a468590</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1242%2Fjcs.019885&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1242%252Fjcs.019885%26sid%3Dliteratum%253Aachs%26aulast%3DBedford%26aufirst%3DM.%2BT.%26atitle%3DArginine%2520methylation%2520at%2520a%2520glance%26jtitle%3DJ.%2520Cell%2520Sci.%26date%3D2007%26volume%3D120%26spage%3D4243%26epage%3D4246%26doi%3D10.1242%2Fjcs.019885" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kurochkina, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guha, U.</span></span> <span> </span><span class="NLM_article-title">SH3 domains: modules of protein-protein interactions</span>. <i>Biophys. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">29</span>– <span class="NLM_lpage">39</span>, <span class="refDoi"> DOI: 10.1007/s12551-012-0081-z</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=10.1007%2Fs12551-012-0081-z" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=28510178" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=1%3ACAS%3A528%3ADC%252BC3sXitlCktrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2013&pages=29-39&author=N.+Kurochkinaauthor=U.+Guha&title=SH3+domains%3A+modules+of+protein-protein+interactions&doi=10.1007%2Fs12551-012-0081-z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">SH3 domains: modules of protein-protein interactions</span></div><div class="casAuthors">Kurochkina, Natalya; Guha, Udayan</div><div class="citationInfo"><span class="NLM_cas:title">Biophysical Reviews</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">29-39</span>CODEN:
                <span class="NLM_cas:coden">BRIECG</span>;
        ISSN:<span class="NLM_cas:issn">1867-2450</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">A review.  Src homol. 3 (SH3) domains are involved in the regulation of important cellular pathways, such as cell proliferation, migration and cytoskeletal modifications.  Recognition of polyproline and a no. of noncanonical sequences by SH3 domains has been extensively studied by crystallog., NMR and other methods.  High-affinity peptides that bind SH3 domains are used in drug development as candidates for anticancer treatment.  This review summarizes the latest achievements in deciphering structural determinants of SH3 function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpOb5bt5bhZl7Vg90H21EOLACvtfcHk0lhEjoValowr4Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXitlCktrY%253D&md5=f9d4c19511a2c08456cdfd1ea8ce7ad1</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1007%2Fs12551-012-0081-z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs12551-012-0081-z%26sid%3Dliteratum%253Aachs%26aulast%3DKurochkina%26aufirst%3DN.%26aulast%3DGuha%26aufirst%3DU.%26atitle%3DSH3%2520domains%253A%2520modules%2520of%2520protein-protein%2520interactions%26jtitle%3DBiophys.%2520Rev.%26date%3D2013%26volume%3D5%26spage%3D29%26epage%3D39%26doi%3D10.1007%2Fs12551-012-0081-z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Antonysamy, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bonday, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campbell, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doyle, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Druzina, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gheyi, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jungheim, L. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qian, Y. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rauch, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Russell, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sauder, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wasserman, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weichert, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Willard, F. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Emtage, S.</span></span> <span> </span><span class="NLM_article-title">Crystal structure of the human PRMT5:MEP50 complex</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>109</i></span>,  <span class="NLM_fpage">17960</span>– <span class="NLM_lpage">17965</span>, <span class="refDoi"> DOI: 10.1073/pnas.1209814109</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=10.1073%2Fpnas.1209814109" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=23071334" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=1%3ACAS%3A528%3ADC%252BC38Xhsl2ktbvL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=109&publication_year=2012&pages=17960-17965&author=S.+Antonysamyauthor=Z.+Bondayauthor=R.+M.+Campbellauthor=B.+Doyleauthor=Z.+Druzinaauthor=T.+Gheyiauthor=B.+Hanauthor=L.+N.+Jungheimauthor=Y.+W.+Qianauthor=C.+Rauchauthor=M.+Russellauthor=J.+M.+Sauderauthor=S.+R.+Wassermanauthor=K.+Weichertauthor=F.+S.+Willardauthor=A.+P.+Zhangauthor=S.+Emtage&title=Crystal+structure+of+the+human+PRMT5%3AMEP50+complex&doi=10.1073%2Fpnas.1209814109"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal structure of the human PRMT5:MEP50 complex</span></div><div class="casAuthors">Antonysamy, Stephen; Bonday, Zahid; Campbell, Robert M.; Doyle, Brandon; Druzina, Zhanna; Gheyi, Tarun; Han, Bomie; Jungheim, Louis N.; Qian, Yuewei; Rauch, Charles; Russell, Marijane; Sauder, J. Michael; Wasserman, Stephen R.; Wichert, Kenneth; Willard, Francis S.; Zhang, Aiping; Emtage, Spencer</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">109</span>
        (<span class="NLM_cas:issue">44</span>),
    <span class="NLM_cas:pages">17960-17965, S17960/1-S17960/9</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Protein arginine methyltransferases (PRMTs) play important roles in several cellular processes, including signaling, gene regulation, and transport of proteins and nucleic acids, to impact growth, differentiation, proliferation, and development.  PRMT5 sym. di-methylates the two-terminal ω-guanidino nitrogens of arginine residues on substrate proteins.  PRMT5 acts as part of a multimeric complex in concert with a variety of partner proteins that regulate its function and specificity.  A core component of these complexes is the WD40 protein MEP50/WDR77/p44, which mediates interactions with binding partners and substrates.  We have detd. the crystal structure of human PRMT5 in complex with MEP50 (methylosome protein 50), bound to an S-adenosylmethionine analog and a peptide substrate derived from histone H4.  The structure of the surprising hetero-octameric complex reveals the close interaction between the seven-bladed β-propeller MEP50 and the N-terminal domain of PRMT5, and delineates the structural elements of substrate recognition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpkcWBh5CVI7LVg90H21EOLACvtfcHk0lh0_K1UOJvz_w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xhsl2ktbvL&md5=757af18cfd0787a02c88c840273109aa</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1209814109&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1209814109%26sid%3Dliteratum%253Aachs%26aulast%3DAntonysamy%26aufirst%3DS.%26aulast%3DBonday%26aufirst%3DZ.%26aulast%3DCampbell%26aufirst%3DR.%2BM.%26aulast%3DDoyle%26aufirst%3DB.%26aulast%3DDruzina%26aufirst%3DZ.%26aulast%3DGheyi%26aufirst%3DT.%26aulast%3DHan%26aufirst%3DB.%26aulast%3DJungheim%26aufirst%3DL.%2BN.%26aulast%3DQian%26aufirst%3DY.%2BW.%26aulast%3DRauch%26aufirst%3DC.%26aulast%3DRussell%26aufirst%3DM.%26aulast%3DSauder%26aufirst%3DJ.%2BM.%26aulast%3DWasserman%26aufirst%3DS.%2BR.%26aulast%3DWeichert%26aufirst%3DK.%26aulast%3DWillard%26aufirst%3DF.%2BS.%26aulast%3DZhang%26aufirst%3DA.%2BP.%26aulast%3DEmtage%26aufirst%3DS.%26atitle%3DCrystal%2520structure%2520of%2520the%2520human%2520PRMT5%253AMEP50%2520complex%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2012%26volume%3D109%26spage%3D17960%26epage%3D17965%26doi%3D10.1073%2Fpnas.1209814109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zeytuni, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zarivach, R.</span></span> <span> </span><span class="NLM_article-title">Structural and functional discussion of the tetra-trico-peptide repeat, a protein interaction module</span>. <i>Structure</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">397</span>– <span class="NLM_lpage">405</span>, <span class="refDoi"> DOI: 10.1016/j.str.2012.01.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=10.1016%2Fj.str.2012.01.006" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2012&pages=397-405&author=N.+Zeytuniauthor=R.+Zarivach&title=Structural+and+functional+discussion+of+the+tetra-trico-peptide+repeat%2C+a+protein+interaction+module&doi=10.1016%2Fj.str.2012.01.006"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1016%2Fj.str.2012.01.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.str.2012.01.006%26sid%3Dliteratum%253Aachs%26aulast%3DZeytuni%26aufirst%3DN.%26aulast%3DZarivach%26aufirst%3DR.%26atitle%3DStructural%2520and%2520functional%2520discussion%2520of%2520the%2520tetra-trico-peptide%2520repeat%252C%2520a%2520protein%2520interaction%2520module%26jtitle%3DStructure%26date%3D2012%26volume%3D20%26spage%3D397%26epage%3D405%26doi%3D10.1016%2Fj.str.2012.01.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kim, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yadav, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carter, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richard, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richie, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bedford, M. T.</span></span> <span> </span><span class="NLM_article-title">Loss of CARM1 results in hypomethylation of thymocyte cyclic AMP-regulated phosphoprotein and deregulated early T cell development</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>279</i></span>,  <span class="NLM_fpage">25339</span>– <span class="NLM_lpage">25344</span>, <span class="refDoi"> DOI: 10.1074/jbc.M402544200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=10.1074%2Fjbc.M402544200" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=279&publication_year=2004&pages=25339-25344&author=J.+Kimauthor=J.+Leeauthor=N.+Yadavauthor=Q.+Wuauthor=C.+Carterauthor=S.+Richardauthor=E.+Richieauthor=M.+T.+Bedford&title=Loss+of+CARM1+results+in+hypomethylation+of+thymocyte+cyclic+AMP-regulated+phosphoprotein+and+deregulated+early+T+cell+development&doi=10.1074%2Fjbc.M402544200"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M402544200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M402544200%26sid%3Dliteratum%253Aachs%26aulast%3DKim%26aufirst%3DJ.%26aulast%3DLee%26aufirst%3DJ.%26aulast%3DYadav%26aufirst%3DN.%26aulast%3DWu%26aufirst%3DQ.%26aulast%3DCarter%26aufirst%3DC.%26aulast%3DRichard%26aufirst%3DS.%26aulast%3DRichie%26aufirst%3DE.%26aulast%3DBedford%26aufirst%3DM.%2BT.%26atitle%3DLoss%2520of%2520CARM1%2520results%2520in%2520hypomethylation%2520of%2520thymocyte%2520cyclic%2520AMP-regulated%2520phosphoprotein%2520and%2520deregulated%2520early%2520T%2520cell%2520development%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2004%26volume%3D279%26spage%3D25339%26epage%3D25344%26doi%3D10.1074%2Fjbc.M402544200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pawlak, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banik-Maiti, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pietenpol, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruley, H. E.</span></span> <span> </span><span class="NLM_article-title">Protein arginine methyltransferase I: substrate specificity and role in hnRNP assembly</span>. <i>J. Cell. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>87</i></span>,  <span class="NLM_fpage">394</span>– <span class="NLM_lpage">407</span>, <span class="refDoi"> DOI: 10.1002/jcb.10307</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=10.1002%2Fjcb.10307" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=87&publication_year=2002&pages=394-407&author=M.+R.+Pawlakauthor=S.+Banik-Maitiauthor=J.+A.+Pietenpolauthor=H.+E.+Ruley&title=Protein+arginine+methyltransferase+I%3A+substrate+specificity+and+role+in+hnRNP+assembly&doi=10.1002%2Fjcb.10307"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1002%2Fjcb.10307&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcb.10307%26sid%3Dliteratum%253Aachs%26aulast%3DPawlak%26aufirst%3DM.%2BR.%26aulast%3DBanik-Maiti%26aufirst%3DS.%26aulast%3DPietenpol%26aufirst%3DJ.%2BA.%26aulast%3DRuley%26aufirst%3DH.%2BE.%26atitle%3DProtein%2520arginine%2520methyltransferase%2520I%253A%2520substrate%2520specificity%2520and%2520role%2520in%2520hnRNP%2520assembly%26jtitle%3DJ.%2520Cell.%2520Biochem.%26date%3D2002%26volume%3D87%26spage%3D394%26epage%3D407%26doi%3D10.1002%2Fjcb.10307" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sun, L. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lv, Z. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bao, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gong, W. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, R. M.</span></span> <span> </span><span class="NLM_article-title">Structural insights into protein arginine symmetric dimethylation by PRMT5</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>108</i></span>,  <span class="NLM_fpage">20538</span>– <span class="NLM_lpage">20543</span>, <span class="refDoi"> DOI: 10.1073/pnas.1106946108</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=10.1073%2Fpnas.1106946108" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=22143770" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=1%3ACAS%3A528%3ADC%252BC38Xkt1KltA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=108&publication_year=2011&pages=20538-20543&author=L.+T.+Sunauthor=M.+Z.+Wangauthor=Z.+Y.+Lvauthor=N.+Yangauthor=Y.+F.+Liuauthor=S.+L.+Baoauthor=W.+M.+Gongauthor=R.+M.+Xu&title=Structural+insights+into+protein+arginine+symmetric+dimethylation+by+PRMT5&doi=10.1073%2Fpnas.1106946108"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Structural insights into protein arginine symmetric dimethylation by PRMT5</span></div><div class="casAuthors">Sun, Litao; Wang, Mingzhu; Lv, Zongyang; Yang, Na; Liu, Yingfang; Bao, Shilai; Gong, Weimin; Xu, Rui-Ming</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">108</span>
        (<span class="NLM_cas:issue">51</span>),
    <span class="NLM_cas:pages">20538-20543, S20538/1-S20538/3</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Sym. and asym. dimethylation of arginine are isomeric protein posttranslational modifications with distinct biol. effects, evidenced by the methylation of arginine 3 of histone H4 (H4R3): sym. dimethylation of H4R3 leads to repression of gene expression, while asym. dimethylation of H4R3 is assocd. with gene activation.  The enzymes catalyzing these modifications share identifiable sequence similarities, but the relationship between their catalytic mechanisms is unknown.  Here we analyzed the structure of a prototypic sym. arginine dimethylase, PRMTS, and discovered that a conserved phenylalanine in the active site is crit. for specifying sym. addn. of Me groups.  Changing it to a methionine significantly elevates the overall methylase activity, but also converts PRMT5 to an enzyme that catalyzes both sym. and asym. dimethylation of arginine.  Our results demonstrate a common catalytic mechanism intrinsic to both sym. and asym. arginine dimethylases, and show that steric constrains in the active sites play an essential role in detg. the product specificity of arginine methylases.  This discovery-also implies a potentially regulatable outcome of arginine dimethylation that may provide versatile control of eukaryotic gene expression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqh0uTOIQ5b9LVg90H21EOLACvtfcHk0lhXy0bqPELwZw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xkt1KltA%253D%253D&md5=3698c8892671220d78fd5491f2daef07</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1106946108&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1106946108%26sid%3Dliteratum%253Aachs%26aulast%3DSun%26aufirst%3DL.%2BT.%26aulast%3DWang%26aufirst%3DM.%2BZ.%26aulast%3DLv%26aufirst%3DZ.%2BY.%26aulast%3DYang%26aufirst%3DN.%26aulast%3DLiu%26aufirst%3DY.%2BF.%26aulast%3DBao%26aufirst%3DS.%2BL.%26aulast%3DGong%26aufirst%3DW.%2BM.%26aulast%3DXu%26aufirst%3DR.%2BM.%26atitle%3DStructural%2520insights%2520into%2520protein%2520arginine%2520symmetric%2520dimethylation%2520by%2520PRMT5%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2011%26volume%3D108%26spage%3D20538%26epage%3D20543%26doi%3D10.1073%2Fpnas.1106946108" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Branscombe, T. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frankel, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cook, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Z. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pestka, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clarke, S.</span></span> <span> </span><span class="NLM_article-title">PRMT5 (Janus kinase-binding protein 1) catalyzes the formation of symmetric dimethylarginine residues in proteins</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>276</i></span>,  <span class="NLM_fpage">32971</span>– <span class="NLM_lpage">32976</span>, <span class="refDoi"> DOI: 10.1074/jbc.M105412200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=10.1074%2Fjbc.M105412200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=11413150" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=1%3ACAS%3A528%3ADC%252BD3MXmvFCmtbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=276&publication_year=2001&pages=32971-32976&author=T.+L.+Branscombeauthor=A.+Frankelauthor=J.+H.+Leeauthor=J.+R.+Cookauthor=Z.+H.+Yangauthor=S.+Pestkaauthor=S.+Clarke&title=PRMT5+%28Janus+kinase-binding+protein+1%29+catalyzes+the+formation+of+symmetric+dimethylarginine+residues+in+proteins&doi=10.1074%2Fjbc.M105412200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">PRMT5 (Janus kinase-binding protein 1) catalyzes the formation of symmetric dimethylarginine residues in proteins</span></div><div class="casAuthors">Branscombe, Tina L.; Frankel, Adam; Lee, Jin-Hyung; Cook, Jeffry R.; Yang, Zhi-Hong; Pestka, Sidney; Clarke, Steven</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">276</span>
        (<span class="NLM_cas:issue">35</span>),
    <span class="NLM_cas:pages">32971-32976</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">We have identified a new mammalian protein arginine N-methyltransferase, PRMT5, formerly designated Janus kinase-binding protein 1, that can catalyze the formation of ω-NG-monomethylarginine and sym. ω-NG,NG'-dimethylarginine in a variety of proteins.  A hemagglutinin peptide-tagged PRMT5 complex purified from human HeLa cells catalyzes the S-adenosyl-L-[methyl-3H]methionine-dependent in vitro methylation of myelin basic protein.  When the radiolabeled myelin basic protein was acid-hydrolyzed to free amino acids, and the products were sepd. by high-resoln. cation exchange chromatog., we were able to detect two tritiated species.  One species co-migrated with a ω-NG-monomethylarginine std., and the other co-chromatographed with a sym. ω-NG,NG'-dimethylarginine std.  Upon base treatment, this second species formed methylamine, a breakdown product characteristic of sym. ω-NG,NG'-dimethylarginine.  Further anal. of these two species by thin layer chromatog. confirmed their identification as ω-NG-monomethylarginine and sym. ω-NG,NG'-dimethylarginine.  The hemagglutinin-PRMT5 complex was also able to monomethylate and sym. dimethylate bovine histone H2A and a glutathione S-transferase-fibrillarin (amino acids 1-148) fusion protein (glutathione S-transferase-GAR).  A mutation introduced into the S-adenosyl-L-methionine-binding motif I of a myc-tagged PRMT5 construct in COS-1 cells led to a near complete loss of obsd. enzymic activity.  PRMT5 is the first example of a catalytic chain for a type II protein arginine N-methyltransferase that can result in the formation of sym. dimethylarginine residues as obsd. previously in myelin basic protein, Sm small nuclear ribonucleoproteins, and other polypeptides.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqZJzrYzZ_ze7Vg90H21EOLACvtfcHk0lhXy0bqPELwZw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXmvFCmtbw%253D&md5=12e2a0db9d4747c4e7b06668b9fd4d2b</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M105412200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M105412200%26sid%3Dliteratum%253Aachs%26aulast%3DBranscombe%26aufirst%3DT.%2BL.%26aulast%3DFrankel%26aufirst%3DA.%26aulast%3DLee%26aufirst%3DJ.%2BH.%26aulast%3DCook%26aufirst%3DJ.%2BR.%26aulast%3DYang%26aufirst%3DZ.%2BH.%26aulast%3DPestka%26aufirst%3DS.%26aulast%3DClarke%26aufirst%3DS.%26atitle%3DPRMT5%2520%2528Janus%2520kinase-binding%2520protein%25201%2529%2520catalyzes%2520the%2520formation%2520of%2520symmetric%2520dimethylarginine%2520residues%2520in%2520proteins%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2001%26volume%3D276%26spage%3D32971%26epage%3D32976%26doi%3D10.1074%2Fjbc.M105412200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cook, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Z. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krause, C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herth, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoffmann, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pestka, S.</span></span> <span> </span><span class="NLM_article-title">FBXO11/PRMT9, a new protein arginine methyltransferase, symmetrically dimethylates arginine residues</span>. <i>Biochem. Biophys. Res. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>342</i></span>,  <span class="NLM_fpage">472</span>– <span class="NLM_lpage">481</span>, <span class="refDoi"> DOI: 10.1016/j.bbrc.2006.01.167</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=10.1016%2Fj.bbrc.2006.01.167" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=342&publication_year=2006&pages=472-481&author=J.+R.+Cookauthor=J.+H.+Leeauthor=Z.+H.+Yangauthor=C.+D.+Krauseauthor=N.+Herthauthor=R.+Hoffmannauthor=S.+Pestka&title=FBXO11%2FPRMT9%2C+a+new+protein+arginine+methyltransferase%2C+symmetrically+dimethylates+arginine+residues&doi=10.1016%2Fj.bbrc.2006.01.167"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1016%2Fj.bbrc.2006.01.167&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbrc.2006.01.167%26sid%3Dliteratum%253Aachs%26aulast%3DCook%26aufirst%3DJ.%2BR.%26aulast%3DLee%26aufirst%3DJ.%2BH.%26aulast%3DYang%26aufirst%3DZ.%2BH.%26aulast%3DKrause%26aufirst%3DC.%2BD.%26aulast%3DHerth%26aufirst%3DN.%26aulast%3DHoffmann%26aufirst%3DR.%26aulast%3DPestka%26aufirst%3DS.%26atitle%3DFBXO11%252FPRMT9%252C%2520a%2520new%2520protein%2520arginine%2520methyltransferase%252C%2520symmetrically%2520dimethylates%2520arginine%2520residues%26jtitle%3DBiochem.%2520Biophys.%2520Res.%2520Commun.%26date%3D2006%26volume%3D342%26spage%3D472%26epage%3D481%26doi%3D10.1016%2Fj.bbrc.2006.01.167" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Y. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hadjikyriacou, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xia, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gayatri, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zurita-Lopez, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kelly, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clarke, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bedford, M. T.</span></span> <span> </span><span class="NLM_article-title">PRMT9 is a Type II methyltransferase that methylates the splicing factor SAP145</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">6428</span>, <span class="refDoi"> DOI: 10.1038/ncomms7428</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=10.1038%2Fncomms7428" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=25737013" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtF2itrbJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=6428&author=Y.+Z.+Yangauthor=A.+Hadjikyriacouauthor=Z.+Xiaauthor=S.+Gayatriauthor=D.+Kimauthor=C.+Zurita-Lopezauthor=R.+Kellyauthor=A.+L.+Guoauthor=W.+Liauthor=S.+G.+Clarkeauthor=M.+T.+Bedford&title=PRMT9+is+a+Type+II+methyltransferase+that+methylates+the+splicing+factor+SAP145&doi=10.1038%2Fncomms7428"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">PRMT9 is a Type II methyltransferase that methylates the splicing factor SAP145</span></div><div class="casAuthors">Yang, Yanzhong; Hadjikyriacou, Andrea; Xia, Zheng; Gayatri, Sitaram; Kim, Daehoon; Zurita-Lopez, Cecilia; Kelly, Ryan; Guo, Ailan; Li, Wei; Clarke, Steven G.; Bedford, Mark T.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Communications</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">6428</span>CODEN:
                <span class="NLM_cas:coden">NCAOBW</span>;
        ISSN:<span class="NLM_cas:issn">2041-1723</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The human genome encodes a family of nine protein arginine methyltransferases (PRMT1-9), whose members can catalyze three distinct types of methylation on arginine residues.  Here we identify two spliceosome-assocd. proteins-SAP145 and SAP49-as PRMT9-binding partners, linking PRMT9 to U2 snRNP maturation.  We show that SAP145 is methylated by PRMT9 at arginine 508, which takes the form of monomethylated arginine (MMA) and sym. dimethylated arginine (SDMA).  PRMT9 thus joins PRMT5 as the only mammalian enzymes capable of depositing the SDMA mark.  Methylation of SAP145 on Arg 508 generates a binding site for the Tudor domain of the Survival of Motor Neuron (SMN) protein, and RNA-seq anal. reveals gross splicing changes when PRMT9 levels are attenuated.  These results identify PRMT9 as a nonhistone methyltransferase that primes the U2 snRNP for interaction with SMN.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoAQGZ5aTDM27Vg90H21EOLACvtfcHk0lj8YaaQwggcLw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtF2itrbJ&md5=bd2abcf479f24b3f8f4e7bf9908c92e7</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1038%2Fncomms7428&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms7428%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DY.%2BZ.%26aulast%3DHadjikyriacou%26aufirst%3DA.%26aulast%3DXia%26aufirst%3DZ.%26aulast%3DGayatri%26aufirst%3DS.%26aulast%3DKim%26aufirst%3DD.%26aulast%3DZurita-Lopez%26aufirst%3DC.%26aulast%3DKelly%26aufirst%3DR.%26aulast%3DGuo%26aufirst%3DA.%2BL.%26aulast%3DLi%26aufirst%3DW.%26aulast%3DClarke%26aufirst%3DS.%2BG.%26aulast%3DBedford%26aufirst%3DM.%2BT.%26atitle%3DPRMT9%2520is%2520a%2520Type%2520II%2520methyltransferase%2520that%2520methylates%2520the%2520splicing%2520factor%2520SAP145%26jtitle%3DNat.%2520Commun.%26date%3D2015%26volume%3D6%26spage%3D6428%26doi%3D10.1038%2Fncomms7428" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Obianyo, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Osborne, T. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, P. R.</span></span> <span> </span><span class="NLM_article-title">Kinetic mechanism of protein arginine methyltransferase 1</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">10420</span>– <span class="NLM_lpage">10427</span>, <span class="refDoi"> DOI: 10.1021/bi800904m</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi800904m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtVOmtrnO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2008&pages=10420-10427&author=O.+Obianyoauthor=T.+C.+Osborneauthor=P.+R.+Thompson&title=Kinetic+mechanism+of+protein+arginine+methyltransferase+1&doi=10.1021%2Fbi800904m"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Kinetic Mechanism of Protein Arginine Methyltransferase 1</span></div><div class="casAuthors">Obianyo, Obiamaka; Osborne, Tanesha C.; Thompson, Paul R.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">39</span>),
    <span class="NLM_cas:pages">10420-10427</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Protein arginine methyltransferases (PRMTs) are SAM-dependent enzymes that catalyze the mono- and dimethylation of peptidyl arginine residues.  Although all PRMTs produce monomethyl arginine (MMA), type 1 PRMTs go on to form asym. dimethylated arginine (ADMA), while type 2 enzymes form sym. dimethylated arginine (SDMA).  PRMT1 is the major type 1 PRMT in vivo, thus it is the primary producer of the competitive NOS inhibitor, ADMA.  Hence, potent inhibitors, which are highly selective for this particular isoenzyme, could serve as excellent therapeutics for heart disease.  However, the design of such inhibitors is impeded by a lack of information regarding this enzyme's kinetic and catalytic mechanisms.  Herein we report an anal. of the kinetic mechanism of human PRMT1 using both an unmethylated and a monomethylated substrate peptide based on the N-terminus of histone H4.  The results of initial velocity and product and dead-end inhibition expts. indicate that PRMT1 utilizes a rapid equil. random (RER) mechanism with the formation of dead-end EAP and EBQ complexes (where E represents the enzyme, A and B represent substrates, and P and Q represent products).  This mechanism is gratifyingly consistent with previous results demonstrating that PRMT1 catalyzes substrate dimethylation in a partially processive manner.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoptMu_FIU0TrVg90H21EOLACvtfcHk0lj8YaaQwggcLw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtVOmtrnO&md5=fde187ecc31760e562934a00e2c6a68a</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1021%2Fbi800904m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi800904m%26sid%3Dliteratum%253Aachs%26aulast%3DObianyo%26aufirst%3DO.%26aulast%3DOsborne%26aufirst%3DT.%2BC.%26aulast%3DThompson%26aufirst%3DP.%2BR.%26atitle%3DKinetic%2520mechanism%2520of%2520protein%2520arginine%2520methyltransferase%25201%26jtitle%3DBiochemistry%26date%3D2008%26volume%3D47%26spage%3D10420%26epage%3D10427%26doi%3D10.1021%2Fbi800904m" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Osborne, T. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Obianyo, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, P. R.</span></span> <span> </span><span class="NLM_article-title">Protein arginine methyltransferase 1: Positively charged residues in substrate peptides distal to the site of methylation are important for substrate binding and catalysis</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>46</i></span>,  <span class="NLM_fpage">13370</span>– <span class="NLM_lpage">13381</span>, <span class="refDoi"> DOI: 10.1021/bi701558t</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi701558t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2007&pages=13370-13381&author=T.+C.+Osborneauthor=O.+Obianyoauthor=X.+Zhangauthor=X.+Chengauthor=P.+R.+Thompson&title=Protein+arginine+methyltransferase+1%3A+Positively+charged+residues+in+substrate+peptides+distal+to+the+site+of+methylation+are+important+for+substrate+binding+and+catalysis&doi=10.1021%2Fbi701558t"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Fbi701558t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi701558t%26sid%3Dliteratum%253Aachs%26aulast%3DOsborne%26aufirst%3DT.%2BC.%26aulast%3DObianyo%26aufirst%3DO.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DCheng%26aufirst%3DX.%26aulast%3DThompson%26aufirst%3DP.%2BR.%26atitle%3DProtein%2520arginine%2520methyltransferase%25201%253A%2520Positively%2520charged%2520residues%2520in%2520substrate%2520peptides%2520distal%2520to%2520the%2520site%2520of%2520methylation%2520are%2520important%2520for%2520substrate%2520binding%2520and%2520catalysis%26jtitle%3DBiochemistry%26date%3D2007%26volume%3D46%26spage%3D13370%26epage%3D13381%26doi%3D10.1021%2Fbi701558t" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tee, W. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pardo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Theunissen, T. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choudhary, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hajkova, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Surani, M. A.</span></span> <span> </span><span class="NLM_article-title">Prmt5 is essential for early mouse development and acts in the cytoplasm to maintain ES cell pluripotency</span>. <i>Genes Dev.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">2772</span>– <span class="NLM_lpage">2777</span>, <span class="refDoi"> DOI: 10.1101/gad.606110</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=10.1101%2Fgad.606110" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=21159818" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=1%3ACAS%3A528%3ADC%252BC3MXltVKh" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2010&pages=2772-2777&author=W.+W.+Teeauthor=M.+Pardoauthor=T.+W.+Theunissenauthor=L.+Yuauthor=J.+S.+Choudharyauthor=P.+Hajkovaauthor=M.+A.+Surani&title=Prmt5+is+essential+for+early+mouse+development+and+acts+in+the+cytoplasm+to+maintain+ES+cell+pluripotency&doi=10.1101%2Fgad.606110"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Prmt5 is essential for early mouse development and acts in the cytoplasm to maintain ES cell pluripotency</span></div><div class="casAuthors">Tee, Wee-Wei; Pardo, Mercedes; Theunissen, Thorold W.; Yu, Lu; Choudhary, Jyoti S.; Hajkova, Petra; Surani, M. Azim</div><div class="citationInfo"><span class="NLM_cas:title">Genes & Development</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">2772-2777</span>CODEN:
                <span class="NLM_cas:coden">GEDEEP</span>;
        ISSN:<span class="NLM_cas:issn">0890-9369</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">Prmt5, an arginine methyltransferase, has multiple roles in germ cells, and possibly in pluripotency.  Here we show that loss of Prmt5 function is early embryonic-lethal due to the abrogation of pluripotent cells in blastocysts.  Prmt5 is also up-regulated in the cytoplasm during the derivation of embryonic stem (ES) cells together with Stat3, where they persist to maintain pluripotency.  Prmt5 in assocn. with Mep50 methylates cytosolic histone H2A (H2AR3me2s) to repress differentiation genes in ES cells.  Loss of Prmt5 or Mep50 results in derepression of differentiation genes, indicating the significance of the Prmt5/Mep50 complex for pluripotency, which may occur in conjunction with the leukemia inhibitory factor (LIF)/Stat3 pathway.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrOFBFfgqPpBbVg90H21EOLACvtfcHk0lgkYl0O_GCvcw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXltVKh&md5=f28985d9b27caa385244420809d11e08</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1101%2Fgad.606110&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fgad.606110%26sid%3Dliteratum%253Aachs%26aulast%3DTee%26aufirst%3DW.%2BW.%26aulast%3DPardo%26aufirst%3DM.%26aulast%3DTheunissen%26aufirst%3DT.%2BW.%26aulast%3DYu%26aufirst%3DL.%26aulast%3DChoudhary%26aufirst%3DJ.%2BS.%26aulast%3DHajkova%26aufirst%3DP.%26aulast%3DSurani%26aufirst%3DM.%2BA.%26atitle%3DPrmt5%2520is%2520essential%2520for%2520early%2520mouse%2520development%2520and%2520acts%2520in%2520the%2520cytoplasm%2520to%2520maintain%2520ES%2520cell%2520pluripotency%26jtitle%3DGenes%2520Dev.%26date%3D2010%26volume%3D24%26spage%3D2772%26epage%3D2777%26doi%3D10.1101%2Fgad.606110" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, G. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamard, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greenblatt, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Man, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perna, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tadi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luciani, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nimer, S. D.</span></span> <span> </span><span class="NLM_article-title">Arginine methyltransferase PRMT5 is essential for sustaining normal adult hematopoiesis</span>. <i>J. Clin. Invest.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>125</i></span>,  <span class="NLM_fpage">3532</span>– <span class="NLM_lpage">3544</span>, <span class="refDoi"> DOI: 10.1172/JCI81749</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=10.1172%2FJCI81749" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=26258414" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=1%3ACAS%3A280%3ADC%252BC287ht1eqtg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=125&publication_year=2015&pages=3532-3544&author=F.+Liuauthor=G.+Y.+Chengauthor=P.+J.+Hamardauthor=S.+Greenblattauthor=L.+Wangauthor=N.+Manauthor=F.+Pernaauthor=H.+M.+Xuauthor=M.+Tadiauthor=L.+Lucianiauthor=S.+D.+Nimer&title=Arginine+methyltransferase+PRMT5+is+essential+for+sustaining+normal+adult+hematopoiesis&doi=10.1172%2FJCI81749"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Arginine methyltransferase PRMT5 is essential for sustaining normal adult hematopoiesis</span></div><div class="casAuthors">Liu Fan; Cheng Guoyan; Hamard Pierre-Jacques; Greenblatt Sarah; Wang Lan; Man Na; Perna Fabiana; Xu Haiming; Tadi Madhavi; Luciani Luisa; Nimer Stephen D</div><div class="citationInfo"><span class="NLM_cas:title">The Journal of clinical investigation</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">125</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">3532-44</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Epigenetic regulators play critical roles in normal hematopoiesis, and the activity of these enzymes is frequently altered in hematopoietic cancers.  The major type II protein arginine methyltransferase PRMT5 catalyzes the formation of symmetric dimethyl arginine and has been implicated in various cellular processes, including pluripotency and tumorigenesis.  Here, we generated Prmt5 conditional KO mice to evaluate the contribution of PRMT5 to adult hematopoiesis.  Loss of PRMT5 triggered an initial but transient expansion of hematopoietic stem cells (HSCs); however, Prmt5 deletion resulted in a concurrent loss of hematopoietic progenitor cells (HPCs), leading to fatal BM aplasia.  PRMT5-specific effects on hematopoiesis were cell intrinsic and depended on PRMT5 methyltransferase activity.  We found that PRMT5-deficient hematopoietic stem and progenitor cells exhibited severely impaired cytokine signaling as well as upregulation of p53 and expression of its downstream targets.  Together, our results demonstrate that PRMT5 plays distinct roles in the behavior of HSCs compared with HPCs and is essential for the maintenance of adult hematopoietic cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTN1zt2v1VSmK-9TkgmBqTqfW6udTcc2eYClg3WVEPur7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC287ht1eqtg%253D%253D&md5=1653ed334d91ed4328f98de8c92506a5</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1172%2FJCI81749&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1172%252FJCI81749%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DCheng%26aufirst%3DG.%2BY.%26aulast%3DHamard%26aufirst%3DP.%2BJ.%26aulast%3DGreenblatt%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DMan%26aufirst%3DN.%26aulast%3DPerna%26aufirst%3DF.%26aulast%3DXu%26aufirst%3DH.%2BM.%26aulast%3DTadi%26aufirst%3DM.%26aulast%3DLuciani%26aufirst%3DL.%26aulast%3DNimer%26aufirst%3DS.%2BD.%26atitle%3DArginine%2520methyltransferase%2520PRMT5%2520is%2520essential%2520for%2520sustaining%2520normal%2520adult%2520hematopoiesis%26jtitle%3DJ.%2520Clin.%2520Invest.%26date%3D2015%26volume%3D125%26spage%3D3532%26epage%3D3544%26doi%3D10.1172%2FJCI81749" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pal, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vishwanath, S. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Erdjument-Bromage, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tempst, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sif, S.</span></span> <span> </span><span class="NLM_article-title">Human SWI/SNF-associated PRMT5 methylates histone H3 arginine 8 and negatively regulates expression of ST7 and NM23 tumor suppressor genes</span>. <i>Mol. Cell. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">9630</span>– <span class="NLM_lpage">9645</span>, <span class="refDoi"> DOI: 10.1128/MCB.24.21.9630-9645.2004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=10.1128%2FMCB.24.21.9630-9645.2004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=15485929" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=1%3ACAS%3A528%3ADC%252BD2MXltl2htQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2004&pages=9630-9645&author=S.+Palauthor=S.+N.+Vishwanathauthor=H.+Erdjument-Bromageauthor=P.+Tempstauthor=S.+Sif&title=Human+SWI%2FSNF-associated+PRMT5+methylates+histone+H3+arginine+8+and+negatively+regulates+expression+of+ST7+and+NM23+tumor+suppressor+genes&doi=10.1128%2FMCB.24.21.9630-9645.2004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Human SWI/SNF-associated PRMT5 methylates histone H3 arginine 8 and negatively regulates expression of ST7 and NM23 tumor suppressor genes</span></div><div class="casAuthors">Pal, Sharmistha; Vishwanath, Sheethal N.; Erdjument-Bromage, Hediye; Tempst, Paul; Sif, Said</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">9630-9645</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">0270-7306</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Protein arginine methyltransferases (PRMTs) have been implicated in transcriptional activation and repression, but their role in controlling cell growth and proliferation remains obscure.  We have recently shown that PRMT5 can interact with flag-tagged BRG1- and hBRM-based hSWI/SNF chromatin remodelers and that both complexes can specifically methylate histones H3 and H4.  Here we report that PRMT5 can be found in assocn. with endogenous hSWI/SNF complexes, which can methylate H3 and H4 N-terminal tails, and show that H3 arginine 8 and H4 arginine 3 are preferred sites of methylation by recombinant and hSWI/SNF-assocd. PRMT5.  To elucidate the role played by PRMT5 in gene regulation, we have established a PRMT5 antisense cell line and detd. by microarray anal. that more genes are derepressed when PRMT5 levels are reduced.  Among the affected genes, we show that suppressor of tumorigenicity 7 (ST7) and nonmetastatic 23 (NM23) are direct targets of PRMT5-contg. BRG1 and hBRM complexes.  Furthermore, we demonstrate that expression of ST7 and NM23 is reduced in a cell line that overexpresses PRMT5 and that this decrease in expression correlates with H3R8 methylation, H3K9 deacetylation, and increased transformation of NIH 3T3 cells.  These findings suggest that the BRG1- and hBRM-assocd. PRMT5 regulates cell growth and proliferation by controlling expression of genes involved in tumor suppression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoHC4UpBSGWILVg90H21EOLACvtfcHk0lgWZ1fdivPEpA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXltl2htQ%253D%253D&md5=663371d517c0e9e3b709483e320869f7</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1128%2FMCB.24.21.9630-9645.2004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.24.21.9630-9645.2004%26sid%3Dliteratum%253Aachs%26aulast%3DPal%26aufirst%3DS.%26aulast%3DVishwanath%26aufirst%3DS.%2BN.%26aulast%3DErdjument-Bromage%26aufirst%3DH.%26aulast%3DTempst%26aufirst%3DP.%26aulast%3DSif%26aufirst%3DS.%26atitle%3DHuman%2520SWI%252FSNF-associated%2520PRMT5%2520methylates%2520histone%2520H3%2520arginine%25208%2520and%2520negatively%2520regulates%2520expression%2520of%2520ST7%2520and%2520NM23%2520tumor%2520suppressor%2520genes%26jtitle%3DMol.%2520Cell.%2520Biol.%26date%3D2004%26volume%3D24%26spage%3D9630%26epage%3D9645%26doi%3D10.1128%2FMCB.24.21.9630-9645.2004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pesiridis, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diamond, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Duyne, G. D.</span></span> <span> </span><span class="NLM_article-title">Role of pICLn in methylation of Sm proteins by PRMT5</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>284</i></span>,  <span class="NLM_fpage">21347</span>– <span class="NLM_lpage">21359</span>, <span class="refDoi"> DOI: 10.1074/jbc.M109.015578</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=10.1074%2Fjbc.M109.015578" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=19520849" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=1%3ACAS%3A528%3ADC%252BD1MXpt1Wisr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=284&publication_year=2009&pages=21347-21359&author=G.+S.+Pesiridisauthor=E.+Diamondauthor=G.+D.+Van+Duyne&title=Role+of+pICLn+in+methylation+of+Sm+proteins+by+PRMT5&doi=10.1074%2Fjbc.M109.015578"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Role of pICLn in Methylation of Sm Proteins by PRMT5</span></div><div class="casAuthors">Pesiridis, G. Scott; Diamond, Evan; Van Duyne, Gregory D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">284</span>
        (<span class="NLM_cas:issue">32</span>),
    <span class="NLM_cas:pages">21347-21359</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">PICln is an essential, highly conserved 26-kDa protein whose functions include binding to Sm proteins in the cytoplasm of human cells and mediating the ordered and regulated assembly of the cell's RNA-splicing machinery by the survival motor neurons complex. pICln also interacts with PRMT5, the enzyme responsible for generating sym. dimethylarginine modifications on the carboxyl-terminal regions of three of the canonical Sm proteins.  To better understand the role of pICln in these cellular processes, we have investigated the properties of pICln and pICln·Sm complexes and the effects that pICln has on the methyltransferase activity of PRMT5.  We find that pICln is a monomer in soln., binds with high affinity (Kd ∼ 160 nM) to SmD3-SmB, and forms 1:1 complexes with Sm proteins and Sm protein subcomplexes.  The data support an end-capping model of pICln binding that supports current views of how pICln prevents Sm oligomerization on illicit RNA substrates.  We have found that by co-expression with pICln, recombinant PRMT5 can be produced in a sol., active form.  PRMT5 alone has promiscuous activity toward a variety of known substrates.  In the presence of pICln, however, PRMT5 methylation of Sm proteins is stimulated, but methylation of histones is inhibited.  We have also found that mutations in pICln that do not affect Sm protein binding can still have a profound effect on the methyltransferase activity of the PRMT5 complex.  Together, the data provide insights into pICln function and represent an important starting point for biochem. analyses of PRMT5.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoA8wzwfocai7Vg90H21EOLACvtfcHk0lgWZ1fdivPEpA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXpt1Wisr4%253D&md5=a9d40f958ab3bbc2b835dcde792bf7b8</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M109.015578&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M109.015578%26sid%3Dliteratum%253Aachs%26aulast%3DPesiridis%26aufirst%3DG.%2BS.%26aulast%3DDiamond%26aufirst%3DE.%26aulast%3DVan%2BDuyne%26aufirst%3DG.%2BD.%26atitle%3DRole%2520of%2520pICLn%2520in%2520methylation%2520of%2520Sm%2520proteins%2520by%2520PRMT5%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2009%26volume%3D284%26spage%3D21347%26epage%3D21359%26doi%3D10.1074%2Fjbc.M109.015578" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Guderian, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peter, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wiesner, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sickmann, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schulze-Osthoff, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grimmler, M.</span></span> <span> </span><span class="NLM_article-title">RioK1, a new interactor of protein arginine methyltransferase 5 (PRMT5), competes with pICln for binding and modulates PRMT5 complex composition and substrate specificity</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>286</i></span>,  <span class="NLM_fpage">1976</span>– <span class="NLM_lpage">1986</span>, <span class="refDoi"> DOI: 10.1074/jbc.M110.148486</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=10.1074%2Fjbc.M110.148486" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=286&publication_year=2011&pages=1976-1986&author=G.+Guderianauthor=C.+Peterauthor=J.+Wiesnerauthor=A.+Sickmannauthor=K.+Schulze-Osthoffauthor=U.+Fischerauthor=M.+Grimmler&title=RioK1%2C+a+new+interactor+of+protein+arginine+methyltransferase+5+%28PRMT5%29%2C+competes+with+pICln+for+binding+and+modulates+PRMT5+complex+composition+and+substrate+specificity&doi=10.1074%2Fjbc.M110.148486"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M110.148486&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M110.148486%26sid%3Dliteratum%253Aachs%26aulast%3DGuderian%26aufirst%3DG.%26aulast%3DPeter%26aufirst%3DC.%26aulast%3DWiesner%26aufirst%3DJ.%26aulast%3DSickmann%26aufirst%3DA.%26aulast%3DSchulze-Osthoff%26aufirst%3DK.%26aulast%3DFischer%26aufirst%3DU.%26aulast%3DGrimmler%26aufirst%3DM.%26atitle%3DRioK1%252C%2520a%2520new%2520interactor%2520of%2520protein%2520arginine%2520methyltransferase%25205%2520%2528PRMT5%2529%252C%2520competes%2520with%2520pICln%2520for%2520binding%2520and%2520modulates%2520PRMT5%2520complex%2520composition%2520and%2520substrate%2520specificity%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2011%26volume%3D286%26spage%3D1976%26epage%3D1986%26doi%3D10.1074%2Fjbc.M110.148486" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Andreu-Perez, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Esteve-Puig, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Torre-Minguela, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lopez-Fauqued, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bech-Serra, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tenbaum, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garcia-Trevijano, E. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Canals, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Merlino, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Avila, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Recio, J. A.</span></span> <span> </span><span class="NLM_article-title">Protein arginine methyltransferase 5 regulates ERK1/2 signal transduction amplitude and cell fate through CRAF</span>. <i>Sci. Signaling</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">ra58</span>, <span class="refDoi"> DOI: 10.1126/scisignal.2001936</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=10.1126%2Fscisignal.2001936" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=21917714" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2011&pages=ra58&author=P.+Andreu-Perezauthor=R.+Esteve-Puigauthor=C.+de+Torre-Minguelaauthor=M.+Lopez-Fauquedauthor=J.+J.+Bech-Serraauthor=S.+Tenbaumauthor=E.+R.+Garcia-Trevijanoauthor=F.+Canalsauthor=G.+Merlinoauthor=M.+A.+Avilaauthor=J.+A.+Recio&title=Protein+arginine+methyltransferase+5+regulates+ERK1%2F2+signal+transduction+amplitude+and+cell+fate+through+CRAF&doi=10.1126%2Fscisignal.2001936"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1126%2Fscisignal.2001936&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscisignal.2001936%26sid%3Dliteratum%253Aachs%26aulast%3DAndreu-Perez%26aufirst%3DP.%26aulast%3DEsteve-Puig%26aufirst%3DR.%26aulast%3Dde%2BTorre-Minguela%26aufirst%3DC.%26aulast%3DLopez-Fauqued%26aufirst%3DM.%26aulast%3DBech-Serra%26aufirst%3DJ.%2BJ.%26aulast%3DTenbaum%26aufirst%3DS.%26aulast%3DGarcia-Trevijano%26aufirst%3DE.%2BR.%26aulast%3DCanals%26aufirst%3DF.%26aulast%3DMerlino%26aufirst%3DG.%26aulast%3DAvila%26aufirst%3DM.%2BA.%26aulast%3DRecio%26aufirst%3DJ.%2BA.%26atitle%3DProtein%2520arginine%2520methyltransferase%25205%2520regulates%2520ERK1%252F2%2520signal%2520transduction%2520amplitude%2520and%2520cell%2520fate%2520through%2520CRAF%26jtitle%3DSci.%2520Signaling%26date%3D2011%26volume%3D4%26spage%3Dra58%26doi%3D10.1126%2Fscisignal.2001936" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hsu, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, C. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chou, C. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuo, H. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, C. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, H. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liao, H. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bedford, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsai, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hung, M. C.</span></span> <span> </span><span class="NLM_article-title">Crosstalk between Arg 1175 methylation and Tyr 1173 phosphorylation negatively modulates EGFR-mediated ERK activation</span>. <i>Nat. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">174</span>– <span class="NLM_lpage">151</span>, <span class="refDoi"> DOI: 10.1038/ncb2158</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=10.1038%2Fncb2158" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=21258366" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtlamtL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2011&pages=174-151&author=J.+M.+Hsuauthor=C.+T.+Chenauthor=C.+K.+Chouauthor=H.+P.+Kuoauthor=L.+Y.+Liauthor=C.+Y.+Linauthor=H.+J.+Leeauthor=Y.+N.+Wangauthor=M.+Liuauthor=H.+W.+Liaoauthor=B.+Shiauthor=C.+C.+Laiauthor=M.+T.+Bedfordauthor=C.+H.+Tsaiauthor=M.+C.+Hung&title=Crosstalk+between+Arg+1175+methylation+and+Tyr+1173+phosphorylation+negatively+modulates+EGFR-mediated+ERK+activation&doi=10.1038%2Fncb2158"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Crosstalk between Arg 1175 methylation and Tyr 1173 phosphorylation negatively modulates EGFR-mediated ERK activation</span></div><div class="casAuthors">Hsu, Jung-Mao; Chen, Chun-Te; Chou, Chao-Kai; Kuo, Hsu-Ping; Li, Long-Yuan; Lin, Chun-Yi; Lee, Hong-Jen; Wang, Ying-Nai; Liu, Mo; Liao, Hsin-Wei; Shi, Bin; Lai, Chien-Chen; Bedford, Mark T.; Tsai, Chang-Hai; Hung, Mien-Chie</div><div class="citationInfo"><span class="NLM_cas:title">Nature Cell Biology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">174-181</span>CODEN:
                <span class="NLM_cas:coden">NCBIFN</span>;
        ISSN:<span class="NLM_cas:issn">1465-7392</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Epidermal growth factor receptor (EGFR) can undergo post-translational modifications, including phosphorylation, glycosylation and ubiquitylation, leading to diverse physiol. consequences and modulation of its biol. activity.  There is increasing evidence that methylation may parallel other post-translational modifications in the regulation of various biol. processes.  It is still not known, however, whether EGFR is regulated by this post-translational event.  EGFR Arg 1175 is methylated by an arginine methyltransferase, PRMT5.  Arg 1175 methylation pos. modulates EGF-induced EGFR trans-autophosphorylation at Tyr 1173, which governs ERK activation.  Abolishment of Arg 1175 methylation enhances EGF-stimulated ERK activation by reducing SHP1 recruitment to EGFR, resulting in augmented cell proliferation, migration and invasion of EGFR-expressing cells.  Therefore, the authors propose a model in which the regulatory crosstalk between PRMT5-mediated Arg 1175 methylation and EGF-induced Tyr 1173 phosphorylation attenuates EGFR-mediated ERK activation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo-rax4WtxKQrVg90H21EOLACvtfcHk0lil3c5_U8Z4DA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtlamtL0%253D&md5=ea4dd8ac92333deee879597c9a657060</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1038%2Fncb2158&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncb2158%26sid%3Dliteratum%253Aachs%26aulast%3DHsu%26aufirst%3DJ.%2BM.%26aulast%3DChen%26aufirst%3DC.%2BT.%26aulast%3DChou%26aufirst%3DC.%2BK.%26aulast%3DKuo%26aufirst%3DH.%2BP.%26aulast%3DLi%26aufirst%3DL.%2BY.%26aulast%3DLin%26aufirst%3DC.%2BY.%26aulast%3DLee%26aufirst%3DH.%2BJ.%26aulast%3DWang%26aufirst%3DY.%2BN.%26aulast%3DLiu%26aufirst%3DM.%26aulast%3DLiao%26aufirst%3DH.%2BW.%26aulast%3DShi%26aufirst%3DB.%26aulast%3DLai%26aufirst%3DC.%2BC.%26aulast%3DBedford%26aufirst%3DM.%2BT.%26aulast%3DTsai%26aufirst%3DC.%2BH.%26aulast%3DHung%26aufirst%3DM.%2BC.%26atitle%3DCrosstalk%2520between%2520Arg%25201175%2520methylation%2520and%2520Tyr%25201173%2520phosphorylation%2520negatively%2520modulates%2520EGFR-mediated%2520ERK%2520activation%26jtitle%3DNat.%2520Cell%2520Biol.%26date%3D2011%26volume%3D13%26spage%3D174%26epage%3D151%26doi%3D10.1038%2Fncb2158" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xu, X. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mayo, M. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bekiranov, S.</span></span> <span> </span><span class="NLM_article-title">Application of machine learning methods to histone methylation ChIP-Seq data reveals H4R3me2 globally represses gene expression</span>. <i>BMC Bioinf</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">396</span>, <span class="refDoi"> DOI: 10.1186/1471-2105-11-396</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=10.1186%2F1471-2105-11-396" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=20653935" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=1%3ACAS%3A280%3ADC%252BC3cjntl2kug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2010&pages=396&author=X.+J.+Xuauthor=S.+Hoangauthor=M.+W.+Mayoauthor=S.+Bekiranov&title=Application+of+machine+learning+methods+to+histone+methylation+ChIP-Seq+data+reveals+H4R3me2+globally+represses+gene+expression&doi=10.1186%2F1471-2105-11-396"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Application of machine learning methods to histone methylation ChIP-Seq data reveals H4R3me2 globally represses gene expression</span></div><div class="casAuthors">Xu Xiaojiang; Hoang Stephen; Mayo Marty W; Bekiranov Stefan</div><div class="citationInfo"><span class="NLM_cas:title">BMC bioinformatics</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">396</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  In the last decade, biochemical studies have revealed that epigenetic modifications including histone modifications, histone variants and DNA methylation form a complex network that regulate the state of chromatin and processes that depend on it including transcription and DNA replication.  Currently, a large number of these epigenetic modifications are being mapped in a variety of cell lines at different stages of development using high throughput sequencing by members of the ENCODE consortium, the NIH Roadmap Epigenomics Program and the Human Epigenome Project.  An extremely promising and underexplored area of research is the application of machine learning methods, which are designed to construct predictive network models, to these large-scale epigenomic data sets.  RESULTS:  Using a ChIP-Seq data set of 20 histone lysine and arginine methylations and histone variant H2A.Z in human CD4+ T-cells, we built predictive models of gene expression as a function of histone modification/variant levels using Multilinear (ML) Regression and Multivariate Adaptive Regression Splines (MARS).  Along with extensive crosstalk among the 20 histone methylations, we found H4R3me2 was the most and second most globally repressive histone methylation among the 20 studied in the ML and MARS models, respectively.  In support of our finding, a number of experimental studies show that PRMT5-catalyzed symmetric dimethylation of H4R3 is associated with repression of gene expression.  This includes a recent study, which demonstrated that H4R3me2 is required for DNMT3A-mediated DNA methylation--a known global repressor of gene expression.  CONCLUSION:  In stark contrast to univariate analysis of the relationship between H4R3me2 and gene expression levels, our study showed that the regulatory role of some modifications like H4R3me2 is masked by confounding variables, but can be elucidated by multivariate/systems-level approaches.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcS1HYQVGZGxJHRxY89Cm6SyfW6udTcc2eZDHSOr9ik-_rntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3cjntl2kug%253D%253D&md5=12c49e64af3b0ce2b3a773e7e9f7a21b</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1186%2F1471-2105-11-396&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1471-2105-11-396%26sid%3Dliteratum%253Aachs%26aulast%3DXu%26aufirst%3DX.%2BJ.%26aulast%3DHoang%26aufirst%3DS.%26aulast%3DMayo%26aufirst%3DM.%2BW.%26aulast%3DBekiranov%26aufirst%3DS.%26atitle%3DApplication%2520of%2520machine%2520learning%2520methods%2520to%2520histone%2520methylation%2520ChIP-Seq%2520data%2520reveals%2520H4R3me2%2520globally%2520represses%2520gene%2520expression%26jtitle%3DBMC%2520Bioinf%26date%3D2010%26volume%3D11%26spage%3D396%26doi%3D10.1186%2F1471-2105-11-396" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Karkhanis, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, Y. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baiocchi, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Imbalzano, A. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sif, S.</span></span> <span> </span><span class="NLM_article-title">Versatility of PRMT5-induced methylation in growth control and development</span>. <i>Trends Biochem. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>36</i></span>,  <span class="NLM_fpage">633</span>– <span class="NLM_lpage">641</span>, <span class="refDoi"> DOI: 10.1016/j.tibs.2011.09.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=10.1016%2Fj.tibs.2011.09.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=21975038" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsFejurrJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2011&pages=633-641&author=V.+Karkhanisauthor=Y.+J.+Huauthor=R.+A.+Baiocchiauthor=A.+N.+Imbalzanoauthor=S.+Sif&title=Versatility+of+PRMT5-induced+methylation+in+growth+control+and+development&doi=10.1016%2Fj.tibs.2011.09.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Versatility of PRMT5-induced methylation in growth control and development</span></div><div class="casAuthors">Karkhanis, Vrajesh; Hu, Yu-Jie; Baiocchi, Robert A.; Imbalzano, Anthony N.; Sif, Said</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Biochemical Sciences</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">633-641</span>CODEN:
                <span class="NLM_cas:coden">TBSCDB</span>;
        ISSN:<span class="NLM_cas:issn">0968-0004</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Arginine methylation governs important cellular processes that impact growth and proliferation, as well as differentiation and development.  Through their ability to catalyze sym. or asym. methylation of histone and non-histone proteins, members of the protein arginine methyltransferase (PRMT) family regulate chromatin structure and expression of a wide spectrum of target genes.  Unlike other PRMTs, PRMT5 works in concert with a variety of cellular proteins including ATP-dependent chromatin remodelers and co-repressors to induce epigenetic silencing.  Recent work also implicates PRMT5 in the control of growth-promoting and pro-survival pathways, which demonstrates its versatility as an enzyme involved in both epigenetic regulation of anti-cancer target genes and organelle biogenesis.  These studies not only provide insight into the mol. mechanisms by which PRMT5 contributes to growth control, but also justify therapeutic targeting of PRMT5.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoD1--MutXQ2bVg90H21EOLACvtfcHk0lg2crXhyJ1K1g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsFejurrJ&md5=e5cb228750165f39636118ddee2a0ee8</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1016%2Fj.tibs.2011.09.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tibs.2011.09.001%26sid%3Dliteratum%253Aachs%26aulast%3DKarkhanis%26aufirst%3DV.%26aulast%3DHu%26aufirst%3DY.%2BJ.%26aulast%3DBaiocchi%26aufirst%3DR.%2BA.%26aulast%3DImbalzano%26aufirst%3DA.%2BN.%26aulast%3DSif%26aufirst%3DS.%26atitle%3DVersatility%2520of%2520PRMT5-induced%2520methylation%2520in%2520growth%2520control%2520and%2520development%26jtitle%3DTrends%2520Biochem.%2520Sci.%26date%3D2011%26volume%3D36%26spage%3D633%26epage%3D641%26doi%3D10.1016%2Fj.tibs.2011.09.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Richters, A.</span></span> <span> </span><span class="NLM_article-title">Targeting protein arginine methyltransferase 5 in disease</span>. <i>Future Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">2081</span>– <span class="NLM_lpage">2098</span>, <span class="refDoi"> DOI: 10.4155/fmc-2017-0089</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=10.4155%2Ffmc-2017-0089" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=29076773" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvVOit7fN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2017&pages=2081-2098&author=A.+Richters&title=Targeting+protein+arginine+methyltransferase+5+in+disease&doi=10.4155%2Ffmc-2017-0089"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting protein arginine methyltransferase 5 in disease</span></div><div class="casAuthors">Richters, Andre</div><div class="citationInfo"><span class="NLM_cas:title">Future Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">2081-2098</span>CODEN:
                <span class="NLM_cas:coden">FMCUA7</span>;
        ISSN:<span class="NLM_cas:issn">1756-8919</span>.
    
            (<span class="NLM_cas:orgname">Future Science Ltd.</span>)
        </div><div class="casAbstract">PRMT5 catalyzes the mono- and sym. dimethylation of the arginine N-guanidine group of a wide variety of target proteins including histones, transcriptional elongation factors, kinases and tumor suppressors by utilizing the essential co-factor S-adenosylmethionine as Me source.  PRMT5 overexpression has been linked to the progression of various diseases, including cancer, and is oftentimes assocd. with a poor prognosis.  Therefore, PRMT5 is promoted as a valuable target for drug discovery approaches and was a subject matter in recent endeavors aiming for the development of specific PRMT5 inhibitors.  This review will embrace the significance of PRMT5 as therapeutic target with respect to its mol. interdependencies in disease states as well as its implication in drug development approaches.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqverMGPRhuJbVg90H21EOLACvtfcHk0lg2crXhyJ1K1g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvVOit7fN&md5=6f8f3e27b9d7389e5b89ff084e6b691e</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.4155%2Ffmc-2017-0089&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4155%252Ffmc-2017-0089%26sid%3Dliteratum%253Aachs%26aulast%3DRichters%26aufirst%3DA.%26atitle%3DTargeting%2520protein%2520arginine%2520methyltransferase%25205%2520in%2520disease%26jtitle%3DFuture%2520Med.%2520Chem.%26date%3D2017%26volume%3D9%26spage%3D2081%26epage%3D2098%26doi%3D10.4155%2Ffmc-2017-0089" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jansson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Durant, S. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cho, E. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheahan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edelmann, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kessler, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">La Thangue, N. B.</span></span> <span> </span><span class="NLM_article-title">Arginine methylation regulates the p53 response</span>. <i>Nat. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">1431</span>– <span class="NLM_lpage">4139</span>, <span class="refDoi"> DOI: 10.1038/ncb1802</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=10.1038%2Fncb1802" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=19011621" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhsVentbfN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2008&pages=1431-4139&author=M.+Janssonauthor=S.+T.+Durantauthor=E.+C.+Choauthor=S.+Sheahanauthor=M.+Edelmannauthor=B.+Kesslerauthor=N.+B.+La+Thangue&title=Arginine+methylation+regulates+the+p53+response&doi=10.1038%2Fncb1802"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Arginine methylation regulates the p53 response</span></div><div class="casAuthors">Jansson, Martin; Durant, Stephen T.; Cho, Er-Chieh; Sheahan, Sharon; Edelmann, Mariola; Kessler, Benedikt; La Thangue, Nicholas B.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Cell Biology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1431-1439</span>CODEN:
                <span class="NLM_cas:coden">NCBIFN</span>;
        ISSN:<span class="NLM_cas:issn">1465-7392</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Activation of the p53 tumor suppressor protein in response to DNA damage leads to apoptosis or cell-cycle arrest.  Enzymic modifications are widely believed to affect and regulate p53 activity.  We describe here a level of post-translational control that has an important functional consequence on the p53 response.  We show that the protein arginine methyltransferase (PRMT) 5, as a co-factor in a DNA damage responsive Strap (stress-responsive activator of p300)-contg. co-activator complex that interacts with p53, is responsible for methylating p53.  Arginine methylation is regulated during the p53 response and affects the target gene specificity of p53.  Furthermore, PRMT5 depletion triggers p53-dependent apoptosis.  Thus, methylation on arginine residues is an underlying mechanism of control during the p53 response.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqa6VSI-iJwVLVg90H21EOLACvtfcHk0lg2crXhyJ1K1g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhsVentbfN&md5=4e01b57e90fd6ae7622e6cea3be498ce</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1038%2Fncb1802&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncb1802%26sid%3Dliteratum%253Aachs%26aulast%3DJansson%26aufirst%3DM.%26aulast%3DDurant%26aufirst%3DS.%2BT.%26aulast%3DCho%26aufirst%3DE.%2BC.%26aulast%3DSheahan%26aufirst%3DS.%26aulast%3DEdelmann%26aufirst%3DM.%26aulast%3DKessler%26aufirst%3DB.%26aulast%3DLa%2BThangue%26aufirst%3DN.%2BB.%26atitle%3DArginine%2520methylation%2520regulates%2520the%2520p53%2520response%26jtitle%3DNat.%2520Cell%2520Biol.%26date%3D2008%26volume%3D10%26spage%3D1431%26epage%3D4139%26doi%3D10.1038%2Fncb1802" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mounir, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Korn, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Westerling, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirby, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schirle, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McAllister, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoffman, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramadan, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hartung, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kipp, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quinn, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fodor, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baird, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schoumacher, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meyer, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deeds, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buchwalter, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stams, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keen, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sellers, W. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pagliarini, R. A.</span></span> <span> </span><span class="NLM_article-title">ERG signaling in prostate cancer is driven through PRMT5-dependent methylation of the Androgen Receptor</span>. <i>eLife</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">e13964</span>, <span class="refDoi"> DOI: 10.7554/eLife.13964</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=10.7554%2FeLife.13964" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2016&pages=e13964&author=Z.+Mounirauthor=J.+M.+Kornauthor=T.+Westerlingauthor=F.+Linauthor=C.+A.+Kirbyauthor=M.+Schirleauthor=G.+McAllisterauthor=G.+Hoffmanauthor=N.+Ramadanauthor=A.+Hartungauthor=Y.+Fengauthor=D.+R.+Kippauthor=C.+Quinnauthor=M.+Fodorauthor=J.+Bairdauthor=M.+Schoumacherauthor=R.+Meyerauthor=J.+Deedsauthor=G.+Buchwalterauthor=T.+Stamsauthor=N.+Keenauthor=W.+R.+Sellersauthor=M.+Brownauthor=R.+A.+Pagliarini&title=ERG+signaling+in+prostate+cancer+is+driven+through+PRMT5-dependent+methylation+of+the+Androgen+Receptor&doi=10.7554%2FeLife.13964"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.7554%2FeLife.13964&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.7554%252FeLife.13964%26sid%3Dliteratum%253Aachs%26aulast%3DMounir%26aufirst%3DZ.%26aulast%3DKorn%26aufirst%3DJ.%2BM.%26aulast%3DWesterling%26aufirst%3DT.%26aulast%3DLin%26aufirst%3DF.%26aulast%3DKirby%26aufirst%3DC.%2BA.%26aulast%3DSchirle%26aufirst%3DM.%26aulast%3DMcAllister%26aufirst%3DG.%26aulast%3DHoffman%26aufirst%3DG.%26aulast%3DRamadan%26aufirst%3DN.%26aulast%3DHartung%26aufirst%3DA.%26aulast%3DFeng%26aufirst%3DY.%26aulast%3DKipp%26aufirst%3DD.%2BR.%26aulast%3DQuinn%26aufirst%3DC.%26aulast%3DFodor%26aufirst%3DM.%26aulast%3DBaird%26aufirst%3DJ.%26aulast%3DSchoumacher%26aufirst%3DM.%26aulast%3DMeyer%26aufirst%3DR.%26aulast%3DDeeds%26aufirst%3DJ.%26aulast%3DBuchwalter%26aufirst%3DG.%26aulast%3DStams%26aufirst%3DT.%26aulast%3DKeen%26aufirst%3DN.%26aulast%3DSellers%26aufirst%3DW.%2BR.%26aulast%3DBrown%26aufirst%3DM.%26aulast%3DPagliarini%26aufirst%3DR.%2BA.%26atitle%3DERG%2520signaling%2520in%2520prostate%2520cancer%2520is%2520driven%2520through%2520PRMT5-dependent%2520methylation%2520of%2520the%2520Androgen%2520Receptor%26jtitle%3DeLife%26date%3D2016%26volume%3D5%26spage%3De13964%26doi%3D10.7554%2FeLife.13964" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Burgos, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilczek, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Onikubo, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bonanno, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jansong, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reimer, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shechter, D.</span></span> <span> </span><span class="NLM_article-title">Histone H2A and H4 N-terminal tails are positioned by the MEP50 WD repeat protein for efficient methylation by the PRMT5 arginine methyltransferase</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>290</i></span>,  <span class="NLM_fpage">9674</span>– <span class="NLM_lpage">9689</span>, <span class="refDoi"> DOI: 10.1074/jbc.M115.636894</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=10.1074%2Fjbc.M115.636894" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=290&publication_year=2015&pages=9674-9689&author=E.+S.+Burgosauthor=C.+Wilczekauthor=T.+Onikuboauthor=J.+B.+Bonannoauthor=J.+Jansongauthor=U.+Reimerauthor=D.+Shechter&title=Histone+H2A+and+H4+N-terminal+tails+are+positioned+by+the+MEP50+WD+repeat+protein+for+efficient+methylation+by+the+PRMT5+arginine+methyltransferase&doi=10.1074%2Fjbc.M115.636894"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M115.636894&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M115.636894%26sid%3Dliteratum%253Aachs%26aulast%3DBurgos%26aufirst%3DE.%2BS.%26aulast%3DWilczek%26aufirst%3DC.%26aulast%3DOnikubo%26aufirst%3DT.%26aulast%3DBonanno%26aufirst%3DJ.%2BB.%26aulast%3DJansong%26aufirst%3DJ.%26aulast%3DReimer%26aufirst%3DU.%26aulast%3DShechter%26aufirst%3DD.%26atitle%3DHistone%2520H2A%2520and%2520H4%2520N-terminal%2520tails%2520are%2520positioned%2520by%2520the%2520MEP50%2520WD%2520repeat%2520protein%2520for%2520efficient%2520methylation%2520by%2520the%2520PRMT5%2520arginine%2520methyltransferase%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2015%26volume%3D290%26spage%3D9674%26epage%3D9689%26doi%3D10.1074%2Fjbc.M115.636894" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Stopa, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krebs, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shechter, D.</span></span> <span> </span><span class="NLM_article-title">The PRMT5 arginine methyltransferase: many roles in development, cancer and beyond</span>. <i>Cell. Mol. Life Sci.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>72</i></span>,  <span class="NLM_fpage">2041</span>– <span class="NLM_lpage">2059</span>, <span class="refDoi"> DOI: 10.1007/s00018-015-1847-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=10.1007%2Fs00018-015-1847-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=25662273" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=1%3ACAS%3A528%3ADC%252BC2MXitlOrtr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=72&publication_year=2015&pages=2041-2059&author=N.+Stopaauthor=J.+E.+Krebsauthor=D.+Shechter&title=The+PRMT5+arginine+methyltransferase%3A+many+roles+in+development%2C+cancer+and+beyond&doi=10.1007%2Fs00018-015-1847-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">The PRMT5 arginine methyltransferase- many roles in development, cancer and beyond</span></div><div class="casAuthors">Stopa, Nicole; Krebs, Jocelyn E.; Shechter, David</div><div class="citationInfo"><span class="NLM_cas:title">Cellular and Molecular Life Sciences</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">72</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2041-2059</span>CODEN:
                <span class="NLM_cas:coden">CMLSFI</span>;
        ISSN:<span class="NLM_cas:issn">1420-682X</span>.
    
            (<span class="NLM_cas:orgname">Birkhaeuser Basel</span>)
        </div><div class="casAbstract">Post-translational arginine methylation is responsible for regulation of many biol. processes.  The protein arginine methyltransferase 5 (PRMT5, also known as Hsl7, Jbp1, Skb1, Capsuleen, or Dart5) is the major enzyme responsible for mono- and sym. dimethylation of arginine.  An expanding literature demonstrates its crit. biol. function in a wide range of cellular processes.  Histone and other protein methylation by PRMT5 regulate genome organization, transcription, stem cells, primordial germ cells, differentiation, the cell cycle, and spliceosome assembly.  Metazoan PRMT5 is found in complex with the WD-repeat protein MEP50 (also known as Wdr77, androgen receptor coactivator p44, or Valois).  PRMT5 also directly assocs. with a range of other protein factors, including pICln, Menin, CoPR5 and RioK1 that may alter its subcellular localization and protein substrate selection.  Protein substrate and PRMT5-MEP50 post-translation modifications induce crosstalk to regulate PRMT5 activity.  Crystal structures of C. elegans PRMT5 and human and frog PRMT5-MEP50 complexes provide substantial insight into the mechanisms of substrate recognition and procession to dimethylation.  Enzymol. studies of PRMT5 have uncovered compelling insights essential for future development of specific PRMT5 inhibitors.  In addn., newly accumulating evidence implicates PRMT5 and MEP50 expression levels and their methyltransferase activity in cancer tumorigenesis, and, significantly, as markers of poor clin. outcome, marking them as potential oncogenes.  Here, we review the substantial new literature on PRMT5 and its partners to highlight the significance of understanding this essential enzyme in health and disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqGZo-Bwo40_7Vg90H21EOLACvtfcHk0lj02DXqSkjmvQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXitlOrtr0%253D&md5=2173fff7825f142cde0ddde7f7b879f5</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1007%2Fs00018-015-1847-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00018-015-1847-9%26sid%3Dliteratum%253Aachs%26aulast%3DStopa%26aufirst%3DN.%26aulast%3DKrebs%26aufirst%3DJ.%2BE.%26aulast%3DShechter%26aufirst%3DD.%26atitle%3DThe%2520PRMT5%2520arginine%2520methyltransferase%253A%2520many%2520roles%2520in%2520development%252C%2520cancer%2520and%2520beyond%26jtitle%3DCell.%2520Mol.%2520Life%2520Sci.%26date%3D2015%26volume%3D72%26spage%3D2041%26epage%3D2059%26doi%3D10.1007%2Fs00018-015-1847-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Meister, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eggert, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buhler, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brahms, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kambach, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, U.</span></span> <span> </span><span class="NLM_article-title">Methylation of Sm proteins by a complex containing PRMT5 and the putative U snRNP assembly factor pICln</span>. <i>Curr. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">1990</span>– <span class="NLM_lpage">1994</span>, <span class="refDoi"> DOI: 10.1016/S0960-9822(01)00592-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=10.1016%2FS0960-9822%2801%2900592-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=11747828" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=1%3ACAS%3A528%3ADC%252BD38XisFKj" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2001&pages=1990-1994&author=G.+Meisterauthor=C.+Eggertauthor=D.+Buhlerauthor=H.+Brahmsauthor=C.+Kambachauthor=U.+Fischer&title=Methylation+of+Sm+proteins+by+a+complex+containing+PRMT5+and+the+putative+U+snRNP+assembly+factor+pICln&doi=10.1016%2FS0960-9822%2801%2900592-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Methylation of Sm proteins by a complex containing PRMT5 and the putative U snRNP assembly factor pICln</span></div><div class="casAuthors">Meister, Gunter; Eggert, Christian; Buhler, Dirk; Brahms, Hero; Kambach, Christian; Fischer, Utz</div><div class="citationInfo"><span class="NLM_cas:title">Current Biology</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">1990-1994</span>CODEN:
                <span class="NLM_cas:coden">CUBLE2</span>;
        ISSN:<span class="NLM_cas:issn">0960-9822</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Seven Sm proteins, termed B/B', D1, D2, D3, E, F, and G, assemble in an ordered manner onto U snRNAs to form the Sm core of the spliceosomal snRNPs U1, U2, U4/U6, and U5.  The survival of motor neuron (SMN) protein binds to Sm proteins and mediates in the context of a macromol. (SMN-) complex the assembly of the Sm core.  Binding of SMN to Sm proteins is enhanced by modification of specific arginine residues in the Sm proteins D1 and D3 to sym. dimethylarginines (sDMAs), suggesting that assembly might be regulated at the posttranslational level.  Here we provide evidence that the previously described pICln-complex, consisting of Sm proteins, the methyltransferase PRMT5, pICln, and two novel factors, catalyzes the sDMA modification of Sm proteins.  In vitro studies further revealed that the pICln complex inhibits the spontaneous assembly of Sm proteins onto a U snRNA.  This effect is mediated by pICln via its binding to the Sm fold of Sm proteins, thereby preventing specific interactions between Sm proteins required for the formation of the Sm core.  Our data suggest that the pICln complex regulates an early step in the assembly of U snRNPs, possibly the transfer of Sm proteins to the SMN-complex.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr1Vh7kSAF-X7Vg90H21EOLACvtfcHk0ljwGG27m9gjuw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XisFKj&md5=7221a40240d94264723c40f5da2e0394</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1016%2FS0960-9822%2801%2900592-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0960-9822%252801%252900592-9%26sid%3Dliteratum%253Aachs%26aulast%3DMeister%26aufirst%3DG.%26aulast%3DEggert%26aufirst%3DC.%26aulast%3DBuhler%26aufirst%3DD.%26aulast%3DBrahms%26aufirst%3DH.%26aulast%3DKambach%26aufirst%3DC.%26aulast%3DFischer%26aufirst%3DU.%26atitle%3DMethylation%2520of%2520Sm%2520proteins%2520by%2520a%2520complex%2520containing%2520PRMT5%2520and%2520the%2520putative%2520U%2520snRNP%2520assembly%2520factor%2520pICln%26jtitle%3DCurr.%2520Biol.%26date%3D2001%26volume%3D11%26spage%3D1990%26epage%3D1994%26doi%3D10.1016%2FS0960-9822%2801%2900592-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Banasavadi-Siddegowda, Y. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Welker, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Imitola, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beattie, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baiocchi, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaur, B.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">Prmt5 as a druggable target for glioblastoma therapy</span>. <i>Neuro. Oncol</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">753</span>– <span class="NLM_lpage">763</span>, <span class="refDoi"> DOI: 10.1093/neuonc/nox206</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=10.1093%2Fneuonc%2Fnox206" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=29106602" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=1%3ACAS%3A280%3ADC%252BC1M7otFWjtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2018&pages=753-763&author=Y.+K.+Banasavadi-Siddegowdaauthor=A.+M.+Welkerauthor=J.+Imitolaauthor=C.+L.+Liauthor=J.+Y.+Zhangauthor=C.+E.+Beattieauthor=R.+Baiocchiauthor=B.+Kaur&title=Prmt5+as+a+druggable+target+for+glioblastoma+therapy&doi=10.1093%2Fneuonc%2Fnox206"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">PRMT5 as a druggable target for glioblastoma therapy</span></div><div class="casAuthors">Banasavadi-Siddegowda Yeshavanth Kumar; Hsu Sigmund; Kaur Balveen; Banasavadi-Siddegowda Yeshavanth Kumar; Kaur Balveen; Welker Alessandra M; An Min; Beattie Christine E; Welker Alessandra M; Welker Alessandra M; Yang Xiaozhi; Li Chenglong; Zhou Wei; Li Chenglong; Shi Guqin; Li Chenglong; Imitola Jaime; Wang Jiang; Phelps Mitch; Zhang Jianying; Baiocchi Robert</div><div class="citationInfo"><span class="NLM_cas:title">Neuro-oncology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">753-763</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Background:  In spite of standard multimodal therapy consisting of surgical resection followed by radiation and concurrent chemotherapy, prognosis for glioblastoma (GBM) patients remains poor.  The identification of both differentiated and undifferentiated "stem cell like" populations in the tumor highlights the significance of finding novel targets that affect the heterogeneous tumor cell population.  Protein arginine methyltransferase 5 (PRMT5) is one such candidate gene whose nuclear expression correlates with poor survival and has been reported to be required for survival of differentiated GBM cells and self-renewal of undifferentiated GBM cells.  In the current study we screened the specificity and efficacy of 4 novel PRMT5 inhibitors in the treatment of GBM.  Methods:  Efficacies of these inhibitors were screened using an in vitro GBM neurosphere model and an in vivo intracranial zebrafish model of glioma.  Standard molecular biology methods were employed to investigate changes in cell cycle, growth, and senescence.  Results:  In vitro and in vivo studies revealed that among the 4 PRMT5 inhibitors, treatment of GBM cells with compound 5 (CMP5) mirrored the effects of PRMT5 knockdown wherein it led to apoptosis of differentiated GBM cells and drove undifferentiated primary patient derived GBM cells into a nonreplicative senescent state.  Conclusion:  In vivo antitumor efficacy combined with the specificity of CMP5 underscores the importance of developing it for translation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSg_a1SxIfsoNj_Vg2vObihfW6udTcc2eZpMSXXb_mATrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1M7otFWjtw%253D%253D&md5=1f89af1ee62765d89fb221b3f68dd0d1</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1093%2Fneuonc%2Fnox206&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fneuonc%252Fnox206%26sid%3Dliteratum%253Aachs%26aulast%3DBanasavadi-Siddegowda%26aufirst%3DY.%2BK.%26aulast%3DWelker%26aufirst%3DA.%2BM.%26aulast%3DImitola%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DC.%2BL.%26aulast%3DZhang%26aufirst%3DJ.%2BY.%26aulast%3DBeattie%26aufirst%3DC.%2BE.%26aulast%3DBaiocchi%26aufirst%3DR.%26aulast%3DKaur%26aufirst%3DB.%26atitle%3DPrmt5%2520as%2520a%2520druggable%2520target%2520for%2520glioblastoma%2520therapy%26jtitle%3DNeuro.%2520Oncol%26date%3D2018%26volume%3D20%26spage%3D753%26epage%3D763%26doi%3D10.1093%2Fneuonc%2Fnox206" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Han, X. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, W. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wheeler, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friedman, G. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Province, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">You, Z. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fathallah-Shaykh, H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gillespie, G. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, X. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">King, P. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nabors, L. B.</span></span> <span> </span><span class="NLM_article-title">Expression of PRMT5 correlates with malignant grade in gliomas and plays a pivotal role in tumor growth in vitro</span>. <i>J. Neuro-Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>118</i></span>,  <span class="NLM_fpage">61</span>– <span class="NLM_lpage">72</span>, <span class="refDoi"> DOI: 10.1007/s11060-014-1419-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=10.1007%2Fs11060-014-1419-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=24664369" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=1%3ACAS%3A528%3ADC%252BC2cXlt1eqtbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=118&publication_year=2014&pages=61-72&author=X.+S.+Hanauthor=R.+Liauthor=W.+B.+Zhangauthor=X.+H.+Yangauthor=C.+G.+Wheelerauthor=G.+K.+Friedmanauthor=P.+Provinceauthor=Q.+Dingauthor=Z.+Y.+Youauthor=H.+M.+Fathallah-Shaykhauthor=G.+Y.+Gillespieauthor=X.+Y.+Zhaoauthor=P.+H.+Kingauthor=L.+B.+Nabors&title=Expression+of+PRMT5+correlates+with+malignant+grade+in+gliomas+and+plays+a+pivotal+role+in+tumor+growth+in+vitro&doi=10.1007%2Fs11060-014-1419-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Expression of PRMT5 correlates with malignant grade in gliomas and plays a pivotal role in tumor growth in vitro</span></div><div class="casAuthors">Han, Xiaosi; Li, Rong; Zhang, Wenbin; Yang, Xiuhua; Wheeler, Crystal G.; Friedman, Gregory K.; Province, Paula; Ding, Qiang; You, Zhiying; Fathallah-Shaykh, Hassan M.; Gillespie, G. Yancey; Zhao, Xinyang; King, Peter H.; Nabors, L. Burt</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Neuro-Oncology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">118</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">61-72</span>CODEN:
                <span class="NLM_cas:coden">JNODD2</span>;
        ISSN:<span class="NLM_cas:issn">0167-594X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Protein arginine methyltransferase 5 (PRMT5) catalyzes the formation of ω-NG,N'G-sym. dimethylarginine residues on histones as well as other proteins.  These modifications play an important role in cell differentiation and tumor cell growth.  However, the role of PRMT5 in human glioma cells has not been characterized.  In this study, we assessed protein expression profiles of PRMT5 in control brain, WHO grade II astrocytomas, anaplastic astrocytomas, and glioblastoma multiforme (GBM) by immunohistochem.  PRMT5 was low in glial cells in control brain tissues and low grade astrocytomas.  Its expression increased in parallel with malignant progression, and was highly expressed in GBM.  Knockdown of PRMT5 by small hairpin RNA caused alterations of p-ERK1/2 and significantly repressed the clonogenic potential and viability of glioma cells.  These findings indicate that PRMT5 is a marker of malignant progression in glioma tumors and plays a pivotal role in tumor growth.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrRIIy_ZOt5bbVg90H21EOLACvtfcHk0liEm28Fz3UNeQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXlt1eqtbc%253D&md5=085dfb02d93a7afddb181c6458b9f5a2</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1007%2Fs11060-014-1419-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs11060-014-1419-0%26sid%3Dliteratum%253Aachs%26aulast%3DHan%26aufirst%3DX.%2BS.%26aulast%3DLi%26aufirst%3DR.%26aulast%3DZhang%26aufirst%3DW.%2BB.%26aulast%3DYang%26aufirst%3DX.%2BH.%26aulast%3DWheeler%26aufirst%3DC.%2BG.%26aulast%3DFriedman%26aufirst%3DG.%2BK.%26aulast%3DProvince%26aufirst%3DP.%26aulast%3DDing%26aufirst%3DQ.%26aulast%3DYou%26aufirst%3DZ.%2BY.%26aulast%3DFathallah-Shaykh%26aufirst%3DH.%2BM.%26aulast%3DGillespie%26aufirst%3DG.%2BY.%26aulast%3DZhao%26aufirst%3DX.%2BY.%26aulast%3DKing%26aufirst%3DP.%2BH.%26aulast%3DNabors%26aufirst%3DL.%2BB.%26atitle%3DExpression%2520of%2520PRMT5%2520correlates%2520with%2520malignant%2520grade%2520in%2520gliomas%2520and%2520plays%2520a%2520pivotal%2520role%2520in%2520tumor%2520growth%2520in%2520vitro%26jtitle%3DJ.%2520Neuro-Oncol.%26date%3D2014%26volume%3D118%26spage%3D61%26epage%3D72%26doi%3D10.1007%2Fs11060-014-1419-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yan, F. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alinari, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lustberg, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, L. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cordero-Nieves, H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banasavadi-Siddegowda, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Virk, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barnholtz-Sloan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bell, E. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wojton, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacob, N. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chakravarti, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nowicki, M. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lapalombella, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Datta, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haseley, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patton, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, P. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ryu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mo, X. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marcucci, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nuovo, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kwon, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Byrd, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiocca, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sif, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacob, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lawler, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaur, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baiocchi, R. A.</span></span> <span> </span><span class="NLM_article-title">Genetic validation of the protein arginine methyltransferase PRMT5 as a candidate therapeutic target in glioblastoma</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>74</i></span>,  <span class="NLM_fpage">1752</span>– <span class="NLM_lpage">1765</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.CAN-13-0884</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=10.1158%2F0008-5472.CAN-13-0884" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=74&publication_year=2014&pages=1752-1765&author=F.+T.+Yanauthor=L.+Alinariauthor=M.+E.+Lustbergauthor=L.+K.+Martinauthor=H.+M.+Cordero-Nievesauthor=Y.+Banasavadi-Siddegowdaauthor=S.+Virkauthor=J.+Barnholtz-Sloanauthor=E.+H.+Bellauthor=J.+Wojtonauthor=N.+K.+Jacobauthor=A.+Chakravartiauthor=M.+O.+Nowickiauthor=X.+Wuauthor=R.+Lapalombellaauthor=J.+Dattaauthor=B.+Yuauthor=K.+Gordonauthor=A.+Haseleyauthor=J.+T.+Pattonauthor=P.+L.+Smithauthor=J.+Ryuauthor=X.+L.+Zhangauthor=X.+K.+Moauthor=G.+Marcucciauthor=G.+Nuovoauthor=C.+H.+Kwonauthor=J.+C.+Byrdauthor=E.+A.+Chioccaauthor=C.+L.+Liauthor=S.+Sifauthor=S.+Jacobauthor=S.+Lawlerauthor=B.+Kaurauthor=R.+A.+Baiocchi&title=Genetic+validation+of+the+protein+arginine+methyltransferase+PRMT5+as+a+candidate+therapeutic+target+in+glioblastoma&doi=10.1158%2F0008-5472.CAN-13-0884"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-13-0884&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-13-0884%26sid%3Dliteratum%253Aachs%26aulast%3DYan%26aufirst%3DF.%2BT.%26aulast%3DAlinari%26aufirst%3DL.%26aulast%3DLustberg%26aufirst%3DM.%2BE.%26aulast%3DMartin%26aufirst%3DL.%2BK.%26aulast%3DCordero-Nieves%26aufirst%3DH.%2BM.%26aulast%3DBanasavadi-Siddegowda%26aufirst%3DY.%26aulast%3DVirk%26aufirst%3DS.%26aulast%3DBarnholtz-Sloan%26aufirst%3DJ.%26aulast%3DBell%26aufirst%3DE.%2BH.%26aulast%3DWojton%26aufirst%3DJ.%26aulast%3DJacob%26aufirst%3DN.%2BK.%26aulast%3DChakravarti%26aufirst%3DA.%26aulast%3DNowicki%26aufirst%3DM.%2BO.%26aulast%3DWu%26aufirst%3DX.%26aulast%3DLapalombella%26aufirst%3DR.%26aulast%3DDatta%26aufirst%3DJ.%26aulast%3DYu%26aufirst%3DB.%26aulast%3DGordon%26aufirst%3DK.%26aulast%3DHaseley%26aufirst%3DA.%26aulast%3DPatton%26aufirst%3DJ.%2BT.%26aulast%3DSmith%26aufirst%3DP.%2BL.%26aulast%3DRyu%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DX.%2BL.%26aulast%3DMo%26aufirst%3DX.%2BK.%26aulast%3DMarcucci%26aufirst%3DG.%26aulast%3DNuovo%26aufirst%3DG.%26aulast%3DKwon%26aufirst%3DC.%2BH.%26aulast%3DByrd%26aufirst%3DJ.%2BC.%26aulast%3DChiocca%26aufirst%3DE.%2BA.%26aulast%3DLi%26aufirst%3DC.%2BL.%26aulast%3DSif%26aufirst%3DS.%26aulast%3DJacob%26aufirst%3DS.%26aulast%3DLawler%26aufirst%3DS.%26aulast%3DKaur%26aufirst%3DB.%26aulast%3DBaiocchi%26aufirst%3DR.%2BA.%26atitle%3DGenetic%2520validation%2520of%2520the%2520protein%2520arginine%2520methyltransferase%2520PRMT5%2520as%2520a%2520candidate%2520therapeutic%2520target%2520in%2520glioblastoma%26jtitle%3DCancer%2520Res.%26date%3D2014%26volume%3D74%26spage%3D1752%26epage%3D1765%26doi%3D10.1158%2F0008-5472.CAN-13-0884" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gu, Z. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, F. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, W. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z. X.</span></span> <span> </span><span class="NLM_article-title">Protein arginine methyltransferase 5 is essential for growth of lung cancer cells</span>. <i>Biochem. J.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>446</i></span>,  <span class="NLM_fpage">235</span>– <span class="NLM_lpage">241</span>, <span class="refDoi"> DOI: 10.1042/BJ20120768</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=10.1042%2FBJ20120768" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=446&publication_year=2012&pages=235-241&author=Z.+P.+Guauthor=S.+Gaoauthor=F.+H.+Zhangauthor=Z.+Q.+Wangauthor=W.+C.+Maauthor=R.+E.+Davisauthor=Z.+X.+Wang&title=Protein+arginine+methyltransferase+5+is+essential+for+growth+of+lung+cancer+cells&doi=10.1042%2FBJ20120768"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1042%2FBJ20120768&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252FBJ20120768%26sid%3Dliteratum%253Aachs%26aulast%3DGu%26aufirst%3DZ.%2BP.%26aulast%3DGao%26aufirst%3DS.%26aulast%3DZhang%26aufirst%3DF.%2BH.%26aulast%3DWang%26aufirst%3DZ.%2BQ.%26aulast%3DMa%26aufirst%3DW.%2BC.%26aulast%3DDavis%26aufirst%3DR.%2BE.%26aulast%3DWang%26aufirst%3DZ.%2BX.%26atitle%3DProtein%2520arginine%2520methyltransferase%25205%2520is%2520essential%2520for%2520growth%2520of%2520lung%2520cancer%2520cells%26jtitle%3DBiochem.%2520J.%26date%3D2012%26volume%3D446%26spage%3D235%26epage%3D241%26doi%3D10.1042%2FBJ20120768" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kong, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gu, Z. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Bryant, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z. X.</span></span> <span> </span><span class="NLM_article-title">Selective small-chemical inhibitors of protein arginine methyltransferase 5 with anti-lung cancer activity</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">e0181601</span>, <span class="refDoi"> DOI: 10.1371/journal.pone.0181601</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=10.1371%2Fjournal.pone.0181601" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=28806746" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtV2rtbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2017&pages=e0181601&author=G.+M.+Kongauthor=M.+Yuauthor=Z.+P.+Guauthor=Z.+Chenauthor=R.+M.+Xuauthor=D.+O%E2%80%99Bryantauthor=Z.+X.+Wang&title=Selective+small-chemical+inhibitors+of+protein+arginine+methyltransferase+5+with+anti-lung+cancer+activity&doi=10.1371%2Fjournal.pone.0181601"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Selective small-chemical inhibitors of protein arginine methyltransferase 5 with anti-lung cancer activity</span></div><div class="casAuthors">Kong, Gui-Mei; Yu, Min; Gu, Zhongping; Chen, Zhi; Xu, Rui-Ming; O'Bryant, Deon; Wang, Zhengxin</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">e0181601/1-e0181601/23</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Protein arginine methyltransferase 5 (PRMT5) plays crit. roles in a wide variety of biol. processes, including tumorigenesis.  By screening a library of small chem. compds., we identified eight compds. that selectively inhibit the PRMT5 enzymic activity, with IC50 values ranging from 0.1 to 6 μM.  Mol. docking simulation and site directed mutagenesis indicated that identified compds. target the substrate-binding site in PRMT5.  Treatment of lung cancer cells with identified inhibitors led to inhibition of the sym. arginine methylation of SmD3 and histones and the cellular proliferation.  Oral administration of the inhibitor demonstrated antitumor activity in a lung tumor xenograft model.  Thus, identified PRMT5-specific small-mol. inhibitors would help elucidate the biol. roles of PRMT5 and serve as lead compds. for future drug development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqHrmbn8I9YMLVg90H21EOLACvtfcHk0lhpVgDeFnUy8g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtV2rtbk%253D&md5=0306ed1351aac8a04d286df255c6e432</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0181601&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0181601%26sid%3Dliteratum%253Aachs%26aulast%3DKong%26aufirst%3DG.%2BM.%26aulast%3DYu%26aufirst%3DM.%26aulast%3DGu%26aufirst%3DZ.%2BP.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DXu%26aufirst%3DR.%2BM.%26aulast%3DO%25E2%2580%2599Bryant%26aufirst%3DD.%26aulast%3DWang%26aufirst%3DZ.%2BX.%26atitle%3DSelective%2520small-chemical%2520inhibitors%2520of%2520protein%2520arginine%2520methyltransferase%25205%2520with%2520anti-lung%2520cancer%2520activity%26jtitle%3DPLoS%2520One%26date%3D2017%26volume%3D12%26spage%3De0181601%26doi%3D10.1371%2Fjournal.pone.0181601" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jin, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cui, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Du, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ren, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pan, J.</span></span> <span> </span><span class="NLM_article-title">Targeting methyltransferase PRMT5 eliminates leukemia stem cells in chronic myelogenous leukemia</span>. <i>J. Clin. Invest.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>126</i></span>,  <span class="NLM_fpage">3961</span>– <span class="NLM_lpage">3980</span>, <span class="refDoi"> DOI: 10.1172/JCI85239</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=10.1172%2FJCI85239" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=126&publication_year=2016&pages=3961-3980&author=Y.+Jinauthor=J.+Zhouauthor=F.+Xuauthor=B.+Jinauthor=L.+Cuiauthor=Y.+Wangauthor=X.+Duauthor=J.+Liauthor=P.+Liauthor=R.+Renauthor=J.+Pan&title=Targeting+methyltransferase+PRMT5+eliminates+leukemia+stem+cells+in+chronic+myelogenous+leukemia&doi=10.1172%2FJCI85239"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1172%2FJCI85239&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1172%252FJCI85239%26sid%3Dliteratum%253Aachs%26aulast%3DJin%26aufirst%3DY.%26aulast%3DZhou%26aufirst%3DJ.%26aulast%3DXu%26aufirst%3DF.%26aulast%3DJin%26aufirst%3DB.%26aulast%3DCui%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DDu%26aufirst%3DX.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DP.%26aulast%3DRen%26aufirst%3DR.%26aulast%3DPan%26aufirst%3DJ.%26atitle%3DTargeting%2520methyltransferase%2520PRMT5%2520eliminates%2520leukemia%2520stem%2520cells%2520in%2520chronic%2520myelogenous%2520leukemia%26jtitle%3DJ.%2520Clin.%2520Invest.%26date%3D2016%26volume%3D126%26spage%3D3961%26epage%3D3980%26doi%3D10.1172%2FJCI85239" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Deng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shao, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, H. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elzey, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pili, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ratliff, T. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, C. D.</span></span> <span> </span><span class="NLM_article-title">Protein arginine methyltransferase 5 functions as an epigenetic activator of the androgen receptor to promote prostate cancer cell growth</span>. <i>Oncogene</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>36</i></span>,  <span class="NLM_fpage">1223</span>– <span class="NLM_lpage">1231</span>, <span class="refDoi"> DOI: 10.1038/onc.2016.287</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=10.1038%2Fonc.2016.287" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=27546619" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtlyisLvK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2017&pages=1223-1231&author=X.+Dengauthor=G.+Shaoauthor=H.+T.+Zhangauthor=C.+Liauthor=D.+Zhangauthor=L.+Chengauthor=B.+D.+Elzeyauthor=R.+Piliauthor=T.+L.+Ratliffauthor=J.+Huangauthor=C.+D.+Hu&title=Protein+arginine+methyltransferase+5+functions+as+an+epigenetic+activator+of+the+androgen+receptor+to+promote+prostate+cancer+cell+growth&doi=10.1038%2Fonc.2016.287"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Protein arginine methyltransferase 5 functions as an epigenetic activator of the androgen receptor to promote prostate cancer cell growth</span></div><div class="casAuthors">Deng, X.; Shao, G.; Zhang, H.-T.; Li, C.; Zhang, D.; Cheng, L.; Elzey, B. D.; Pili, R.; Ratliff, T. L.; Huang, J.; Hu, C.-D.</div><div class="citationInfo"><span class="NLM_cas:title">Oncogene</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1223-1231</span>CODEN:
                <span class="NLM_cas:coden">ONCNES</span>;
        ISSN:<span class="NLM_cas:issn">0950-9232</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Protein arginine methyltransferase 5 (PRMT5) is an emerging epigenetic enzyme that mainly represses transcription of target genes via sym. dimethylation of arginine residues on histones H4R3, H3R8 and H2AR3.  Accumulating evidence suggests that PRMT5 may function as an oncogene to drive cancer cell growth by epigenetic inactivation of several tumor suppressors.  Here, we provide evidence that PRMT5 promotes prostate cancer cell growth by epigenetically activating transcription of the androgen receptor (AR) in prostate cancer cells.  Knockdown of PRMT5 or inhibition of PRMT5 by a specific inhibitor reduces the expression of AR and suppresses the growth of multiple AR-pos., but not AR-neg., prostate cancer cells.  Significantly, knockdown of PRMT5 in AR-pos. LNCaP cells completely suppresses the growth of xenograft tumors in mice.  Mol. anal. reveals that PRMT5 binds to the proximal promoter region of the AR gene and contributes mainly to the enriched sym. dimethylation of H4R3 in the same region.  Mechanistically, PRMT5 is recruited to the AR promoter by its interaction with Sp1, the major transcription factor responsible for AR transcription, and forms a complex with Brg1, an ATP-dependent chromatin remodeler, on the proximal promoter region of the AR gene.  Furthermore, PRMT5 expression in prostate cancer tissues is significantly higher than that in benign prostatic hyperplasia tissues, and PRMT5 expression correlates pos. with AR expression at both the protein and mRNA levels.  Taken together, our results identify PRMT5 as a novel epigenetic activator of AR in prostate cancer.  Given that inhibiting AR transcriptional activity or androgen synthesis remains the major mechanism of action for most existing anti-androgen agents, our findings also raise an interesting possibility that targeting PRMT5 may represent a novel approach for prostate cancer treatment by eliminating AR expression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr6wjniAjJYjbVg90H21EOLACvtfcHk0lhpVgDeFnUy8g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtlyisLvK&md5=75aff420e6c8a819bd41fd45cec7dec9</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1038%2Fonc.2016.287&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fonc.2016.287%26sid%3Dliteratum%253Aachs%26aulast%3DDeng%26aufirst%3DX.%26aulast%3DShao%26aufirst%3DG.%26aulast%3DZhang%26aufirst%3DH.%2BT.%26aulast%3DLi%26aufirst%3DC.%26aulast%3DZhang%26aufirst%3DD.%26aulast%3DCheng%26aufirst%3DL.%26aulast%3DElzey%26aufirst%3DB.%2BD.%26aulast%3DPili%26aufirst%3DR.%26aulast%3DRatliff%26aufirst%3DT.%2BL.%26aulast%3DHuang%26aufirst%3DJ.%26aulast%3DHu%26aufirst%3DC.%2BD.%26atitle%3DProtein%2520arginine%2520methyltransferase%25205%2520functions%2520as%2520an%2520epigenetic%2520activator%2520of%2520the%2520androgen%2520receptor%2520to%2520promote%2520prostate%2520cancer%2520cell%2520growth%26jtitle%3DOncogene%26date%3D2017%26volume%3D36%26spage%3D1223%26epage%3D1231%26doi%3D10.1038%2Fonc.2016.287" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mavrakis, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McDonald, E. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schlabach, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Billy, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoffman, G. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">deWeck, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruddy, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Venkatesan, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McAllister, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">deBeaumont, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ho, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan-Neale, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, G. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yin, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kipp, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McNamara, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sprague, E. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cho, Y. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crawford, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Capka, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hurov, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Porter, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tallarico, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mickanin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lees, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pagliarini, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keen, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmelzle, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hofmann, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stegmeier, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sellers, W. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stump, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yue, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciccone, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vitari, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, A.</span></span> <span> </span><span class="NLM_article-title">Disordered methionine metabolism in MTAP/CDKN2A-deleted cancers leads to marked dependence on PRMT5</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>351</i></span>,  <span class="NLM_fpage">1208</span>– <span class="NLM_lpage">1213</span>, <span class="refDoi"> DOI: 10.1126/science.aad5944</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=10.1126%2Fscience.aad5944" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=26912361" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=1%3ACAS%3A528%3ADC%252BC28Xjs12gtr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=351&publication_year=2016&pages=1208-1213&author=K.+Mavrakisauthor=E.+R.+McDonaldauthor=M.+R.+Schlabachauthor=E.+Billyauthor=G.+R.+Hoffmanauthor=A.+deWeckauthor=D.+A.+Ruddyauthor=K.+Venkatesanauthor=G.+McAllisterauthor=R.+deBeaumontauthor=S.+Hoauthor=Y.+Liuauthor=Y.+Yan-Nealeauthor=G.+Z.+Yangauthor=F.+Linauthor=H.+Yinauthor=H.+Gaoauthor=D.+R.+Kippauthor=S.+P.+Zhaoauthor=J.+T.+McNamaraauthor=E.+R.+Spragueauthor=Y.+S.+Choauthor=J.+Guauthor=K.+Crawfordauthor=V.+Capkaauthor=K.+Hurovauthor=J.+A.+Porterauthor=J.+Tallaricoauthor=C.+Mickaninauthor=E.+Leesauthor=R.+Pagliariniauthor=N.+Keenauthor=T.+Schmelzleauthor=F.+Hofmannauthor=F.+Stegmeierauthor=W.+R.+Sellersauthor=J.+Yuauthor=M.+Stumpauthor=Y.+Yueauthor=B.+Zhengauthor=Y.+Linauthor=A.+C.+Cicconeauthor=A.+C.+Vitariauthor=A.+Lai&title=Disordered+methionine+metabolism+in+MTAP%2FCDKN2A-deleted+cancers+leads+to+marked+dependence+on+PRMT5&doi=10.1126%2Fscience.aad5944"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Disordered methionine metabolism in MTAP/CDKN2A-deleted cancers leads to dependence on PRMT5</span></div><div class="casAuthors">Mavrakis, Konstantinos J.; McDonald, E. Robert, III; Schlabach, Michael R.; Billy, Eric; Hoffman, Gregory R.; de Weck, Antoine; Ruddy, David A.; Venkatesan, Kavitha; Yu, Jianjun; McAllister, Gregg; Stump, Mark; de Beaumont, Rosalie; Ho, Samuel; Yue, Yingzi; Liu, Yue; Yan-Neale, Yan; Yang, Guizhi; Lin, Fallon; Yin, Hong; Gao, Hui; Kipp, D. Randal; Zhao, Songping; McNamara, Joshua T.; Sprague, Elizabeth R.; Zheng, Bing; Lin, Ying; Cho, Young Shin; Gu, Justin; Crawford, Kenneth; Ciccone, David; Vitari, Alberto C.; Lai, Albert; Capka, Vladimir; Hurov, Kristen; Porter, Jeffery A.; Tallarico, John; Mickanin, Craig; Lees, Emma; Pagliarini, Raymond; Keen, Nicholas; Schmelzle, Tobias; Hofmann, Francesco; Stegmeier, Frank; Sellers, William R.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">351</span>
        (<span class="NLM_cas:issue">6278</span>),
    <span class="NLM_cas:pages">1208-1213</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">5-Methylthioadenosine phosphorylase (MTAP) is a key enzyme in the methionine salvage pathway.  The MTAP gene is frequently deleted in human cancers because of its chromosomal proximity to the tumor suppressor gene CDKN2A.  By interrogating data from a large-scale short hairpin RNA-mediated screen across 390 cancer cell line models, we found that the viability of MTAP-deficient cancer cells is impaired by depletion of the protein arginine methyltransferase PRMT5.  MTAP-deleted cells accumulate the metabolite methylthioadenosine (MTA), which we found to inhibit PRMT5 methyltransferase activity.  Deletion of MTAP in MTAP-proficient cells rendered them sensitive to PRMT5 depletion.  Conversely, reconstitution of MTAP in an MTAP-deficient cell line rescued PRMT5 dependence.  Thus, MTA accumulation in MTAP-deleted cancers creates a hypomorphic PRMT5 state that is selectively sensitized toward further PRMT5 inhibition.  Inhibitors of PRMT5 that leverage this dysregulated metabolic state merit further investigation as a potential therapy for MTAP/CDKN2A-deleted tumors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoNsuzYFo1qn7Vg90H21EOLACvtfcHk0lihcR5WostLEA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xjs12gtr0%253D&md5=d77531ea765db9762d6815840d39ee4a</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1126%2Fscience.aad5944&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.aad5944%26sid%3Dliteratum%253Aachs%26aulast%3DMavrakis%26aufirst%3DK.%26aulast%3DMcDonald%26aufirst%3DE.%2BR.%26aulast%3DSchlabach%26aufirst%3DM.%2BR.%26aulast%3DBilly%26aufirst%3DE.%26aulast%3DHoffman%26aufirst%3DG.%2BR.%26aulast%3DdeWeck%26aufirst%3DA.%26aulast%3DRuddy%26aufirst%3DD.%2BA.%26aulast%3DVenkatesan%26aufirst%3DK.%26aulast%3DMcAllister%26aufirst%3DG.%26aulast%3DdeBeaumont%26aufirst%3DR.%26aulast%3DHo%26aufirst%3DS.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DYan-Neale%26aufirst%3DY.%26aulast%3DYang%26aufirst%3DG.%2BZ.%26aulast%3DLin%26aufirst%3DF.%26aulast%3DYin%26aufirst%3DH.%26aulast%3DGao%26aufirst%3DH.%26aulast%3DKipp%26aufirst%3DD.%2BR.%26aulast%3DZhao%26aufirst%3DS.%2BP.%26aulast%3DMcNamara%26aufirst%3DJ.%2BT.%26aulast%3DSprague%26aufirst%3DE.%2BR.%26aulast%3DCho%26aufirst%3DY.%2BS.%26aulast%3DGu%26aufirst%3DJ.%26aulast%3DCrawford%26aufirst%3DK.%26aulast%3DCapka%26aufirst%3DV.%26aulast%3DHurov%26aufirst%3DK.%26aulast%3DPorter%26aufirst%3DJ.%2BA.%26aulast%3DTallarico%26aufirst%3DJ.%26aulast%3DMickanin%26aufirst%3DC.%26aulast%3DLees%26aufirst%3DE.%26aulast%3DPagliarini%26aufirst%3DR.%26aulast%3DKeen%26aufirst%3DN.%26aulast%3DSchmelzle%26aufirst%3DT.%26aulast%3DHofmann%26aufirst%3DF.%26aulast%3DStegmeier%26aufirst%3DF.%26aulast%3DSellers%26aufirst%3DW.%2BR.%26aulast%3DYu%26aufirst%3DJ.%26aulast%3DStump%26aufirst%3DM.%26aulast%3DYue%26aufirst%3DY.%26aulast%3DZheng%26aufirst%3DB.%26aulast%3DLin%26aufirst%3DY.%26aulast%3DCiccone%26aufirst%3DA.%2BC.%26aulast%3DVitari%26aufirst%3DA.%2BC.%26aulast%3DLai%26aufirst%3DA.%26atitle%3DDisordered%2520methionine%2520metabolism%2520in%2520MTAP%252FCDKN2A-deleted%2520cancers%2520leads%2520to%2520marked%2520dependence%2520on%2520PRMT5%26jtitle%3DScience%26date%3D2016%26volume%3D351%26spage%3D1208%26epage%3D1213%26doi%3D10.1126%2Fscience.aad5944" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Marjon, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cameron, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clasquin, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mandley, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kunii, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mcvay, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choe, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kernytsky, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gross, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Konteatis, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murtie, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blake, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Travins, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dorsch, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Biller, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marks, K. M.</span></span> <span> </span><span class="NLM_article-title">MTAP deletions in cancer create vulnerability to targeting of the MAT2A/PRMT5/RIOK1 axis</span>. <i>Cell Rep.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">574</span>– <span class="NLM_lpage">587</span>, <span class="refDoi"> DOI: 10.1016/j.celrep.2016.03.043</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=10.1016%2Fj.celrep.2016.03.043" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=27068473" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=1%3ACAS%3A528%3ADC%252BC28Xlsleltrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2016&pages=574-587&author=K.+Marjonauthor=M.+J.+Cameronauthor=P.+Quangauthor=M.+F.+Clasquinauthor=E.+Mandleyauthor=K.+Kuniiauthor=M.+Mcvayauthor=S.+Choeauthor=A.+Kernytskyauthor=S.+Grossauthor=Z.+Konteatisauthor=J.+Murtieauthor=M.+L.+Blakeauthor=J.+Travinsauthor=M.+Dorschauthor=S.+A.+Billerauthor=K.+M.+Marks&title=MTAP+deletions+in+cancer+create+vulnerability+to+targeting+of+the+MAT2A%2FPRMT5%2FRIOK1+axis&doi=10.1016%2Fj.celrep.2016.03.043"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">MTAP Deletions in Cancer Create Vulnerability to Targeting of the MAT2A/PRMT5/RIOK1 Axis</span></div><div class="casAuthors">Marjon, Katya; Cameron, Michael J.; Quang, Phong; Clasquin, Michelle F.; Mandley, Everton; Kunii, Kaiko; McVay, Michael; Choe, Sung; Kernytsky, Andrew; Gross, Stefan; Konteatis, Zenon; Murtie, Joshua; Blake, Michelle L.; Travins, Jeremy; Dorsch, Marion; Biller, Scott A.; Marks, Kevin M.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Reports</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">574-587</span>CODEN:
                <span class="NLM_cas:coden">CREED8</span>;
        ISSN:<span class="NLM_cas:issn">2211-1247</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Homozygous deletions of p16/CDKN2A are prevalent in cancer, and these mutations commonly involve co-deletion of adjacent genes, including methylthioadenosine phosphorylase (MTAP).  Here, we used shRNA screening and identified the metabolic enzyme, methionine adenosyltransferase II alpha (MAT2A), and the arginine methyltransferase, PRMT5, as vulnerable enzymes in cells with MTAP deletion.  Metabolomic and biochem. studies revealed a mechanistic basis for this synthetic lethality.  The MTAP substrate methylthioadenosine (MTA) accumulates upon MTAP loss.  Biochem. profiling of a methyltransferase enzyme panel revealed that MTA is a potent and selective inhibitor of PRMT5.  MTAP-deleted cells have reduced PRMT5 methylation activity and increased sensitivity to PRMT5 depletion.  MAT2A produces the PRMT5 substrate S-adenosylmethionine (SAM), and MAT2A depletion reduces growth and PRMT5 methylation activity selectively in MTAP-deleted cells.  Furthermore, this vulnerability extends to PRMT5 co-complex proteins such as RIOK1.  Thus, the unique biochem. features of PRMT5 create an axis of targets vulnerable in CDKN2A/MTAP-deleted cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGokW739dwptcbVg90H21EOLACvtfcHk0lihcR5WostLEA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xlsleltrk%253D&md5=73a6839e63b85a71f3661dc9e0c7b4ea</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1016%2Fj.celrep.2016.03.043&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.celrep.2016.03.043%26sid%3Dliteratum%253Aachs%26aulast%3DMarjon%26aufirst%3DK.%26aulast%3DCameron%26aufirst%3DM.%2BJ.%26aulast%3DQuang%26aufirst%3DP.%26aulast%3DClasquin%26aufirst%3DM.%2BF.%26aulast%3DMandley%26aufirst%3DE.%26aulast%3DKunii%26aufirst%3DK.%26aulast%3DMcvay%26aufirst%3DM.%26aulast%3DChoe%26aufirst%3DS.%26aulast%3DKernytsky%26aufirst%3DA.%26aulast%3DGross%26aufirst%3DS.%26aulast%3DKonteatis%26aufirst%3DZ.%26aulast%3DMurtie%26aufirst%3DJ.%26aulast%3DBlake%26aufirst%3DM.%2BL.%26aulast%3DTravins%26aufirst%3DJ.%26aulast%3DDorsch%26aufirst%3DM.%26aulast%3DBiller%26aufirst%3DS.%2BA.%26aulast%3DMarks%26aufirst%3DK.%2BM.%26atitle%3DMTAP%2520deletions%2520in%2520cancer%2520create%2520vulnerability%2520to%2520targeting%2520of%2520the%2520MAT2A%252FPRMT5%252FRIOK1%2520axis%26jtitle%3DCell%2520Rep.%26date%3D2016%26volume%3D15%26spage%3D574%26epage%3D587%26doi%3D10.1016%2Fj.celrep.2016.03.043" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kryukov, G. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, F. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruth, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paulk, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsherniak, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marlow, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vazquez, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weir, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fitzgerald, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanaka, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bielski, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dennis, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowley, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boehm, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Root, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Golub, T. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clish, C. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hahn, W. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garraway, L. A.</span></span> <span> </span><span class="NLM_article-title">MTAP deletion confers enhanced dependency on the PRMT5 arginine methyltransferase in cancer cells</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>351</i></span>,  <span class="NLM_fpage">1214</span>– <span class="NLM_lpage">1218</span>, <span class="refDoi"> DOI: 10.1126/science.aad5214</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=10.1126%2Fscience.aad5214" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=26912360" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=1%3ACAS%3A528%3ADC%252BC28Xjs12gtro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=351&publication_year=2016&pages=1214-1218&author=G.+V.+Kryukovauthor=F.+H.+Wilsonauthor=J.+R.+Ruthauthor=J.+Paulkauthor=A.+Tsherniakauthor=S.+E.+Marlowauthor=F.+Vazquezauthor=B.+A.+Weirauthor=M.+E.+Fitzgeraldauthor=M.+Tanakaauthor=C.+M.+Bielskiauthor=J.+M.+Scottauthor=C.+Dennisauthor=G.+S.+Cowleyauthor=J.+S.+Boehmauthor=D.+E.+Rootauthor=T.+R.+Golubauthor=C.+B.+Clishauthor=J.+E.+Bradnerauthor=W.+C.+Hahnauthor=L.+A.+Garraway&title=MTAP+deletion+confers+enhanced+dependency+on+the+PRMT5+arginine+methyltransferase+in+cancer+cells&doi=10.1126%2Fscience.aad5214"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">MTAP deletion confers enhanced dependency on the PRMT5 arginine methyltransferase in cancer cells</span></div><div class="casAuthors">Kryukov, Gregory V.; Wilson, Frederick H.; Ruth, Jason R.; Paulk, Joshiawa; Tsherniak, Aviad; Marlow, Sara E.; Vazquez, Francisca; Weir, Barbara A.; Fitzgerald, Mark E.; Tanaka, Minoru; Bielski, Craig M.; Scott, Justin M.; Dennis, Courtney; Cowley, Glenn S.; Boehm, Jesse S.; Root, David E.; Golub, Todd R.; Clish, Clary B.; Bradner, James E.; Hahn, William C.; Garraway, Levi A.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">351</span>
        (<span class="NLM_cas:issue">6278</span>),
    <span class="NLM_cas:pages">1214-1218</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">The discovery of cancer dependencies has the potential to inform therapeutic strategies and to identify putative drug targets.  Integrating data from comprehensive genomic profiling of cancer cell lines and from functional characterization of cancer cell dependencies, we discovered that loss of the enzyme methylthioadenosine phosphorylase (MTAP) confers a selective dependence on protein arginine methyltransferase 5 (PRMT5) and its binding partner WDR77.  MTAP is frequently lost due to its proximity to the commonly deleted tumor suppressor gene, CDKN2A.  We obsd. increased intracellular concns. of methylthioadenosine (MTA, the metabolite cleaved by MTAP) in cells harboring MTAP deletions.  Furthermore, MTA specifically inhibited PRMT5 enzymic activity.  Administration of either MTA or a small-mol. PRMT5 inhibitor showed a modest preferential impairment of cell viability for MTAP-null cancer cell lines compared with isogenic MTAP-expressing counterparts.  Together, our findings reveal PRMT5 as a potential vulnerability across multiple cancer lineages augmented by a common "passenger" genomic alteration.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr94dQPnMgrqbVg90H21EOLACvtfcHk0lhmjfk9R54-Kg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xjs12gtro%253D&md5=2b017a8c54972828c9024a9d85ff2e2d</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1126%2Fscience.aad5214&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.aad5214%26sid%3Dliteratum%253Aachs%26aulast%3DKryukov%26aufirst%3DG.%2BV.%26aulast%3DWilson%26aufirst%3DF.%2BH.%26aulast%3DRuth%26aufirst%3DJ.%2BR.%26aulast%3DPaulk%26aufirst%3DJ.%26aulast%3DTsherniak%26aufirst%3DA.%26aulast%3DMarlow%26aufirst%3DS.%2BE.%26aulast%3DVazquez%26aufirst%3DF.%26aulast%3DWeir%26aufirst%3DB.%2BA.%26aulast%3DFitzgerald%26aufirst%3DM.%2BE.%26aulast%3DTanaka%26aufirst%3DM.%26aulast%3DBielski%26aufirst%3DC.%2BM.%26aulast%3DScott%26aufirst%3DJ.%2BM.%26aulast%3DDennis%26aufirst%3DC.%26aulast%3DCowley%26aufirst%3DG.%2BS.%26aulast%3DBoehm%26aufirst%3DJ.%2BS.%26aulast%3DRoot%26aufirst%3DD.%2BE.%26aulast%3DGolub%26aufirst%3DT.%2BR.%26aulast%3DClish%26aufirst%3DC.%2BB.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DHahn%26aufirst%3DW.%2BC.%26aulast%3DGarraway%26aufirst%3DL.%2BA.%26atitle%3DMTAP%2520deletion%2520confers%2520enhanced%2520dependency%2520on%2520the%2520PRMT5%2520arginine%2520methyltransferase%2520in%2520cancer%2520cells%26jtitle%3DScience%26date%3D2016%26volume%3D351%26spage%3D1214%26epage%3D1218%26doi%3D10.1126%2Fscience.aad5214" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kaniskan, H. Ü.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Konze, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span> <span> </span><span class="NLM_article-title">Selective inhibitors of protein methyltransferases</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">1596</span>– <span class="NLM_lpage">1629</span>, <span class="refDoi"> DOI: 10.1021/jm501234a</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm501234a" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvFKqtbbJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=1596-1629&author=H.+%C3%9C.+Kaniskanauthor=K.+D.+Konzeauthor=J.+Jin&title=Selective+inhibitors+of+protein+methyltransferases&doi=10.1021%2Fjm501234a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Selective inhibitors of protein methyltransferases</span></div><div class="casAuthors">Kaniskan, H. Umit; Konze, Kyle D.; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1596-1629</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Mounting evidence suggests that protein methyltransferases (PMTs), which catalyze the methylation of histone and non-histone proteins, play a crucial role in diverse biol. processes and human diseases.  In particular, PMTs have been recognized as major players in regulating gene expression and chromatin state.  PMTs are divided into 2 categories: protein lysine methyltransferases (PKMTs) and protein arginine methyltransferases (PRMTs).  There has been a steadily growing interest in these enzymes as potential therapeutic targets and therefore discovery of PMT inhibitors has also been pursued increasingly over the past decade.  Here, the authors present a perspective on selective, small-mol. inhibitors of PMTs with an emphasis on their discovery, characterization, and applicability as chem. tools for deciphering the target PMTs' physiol. functions and involvement in human diseases.  The authors highlight the current state of PMT inhibitors and discuss future directions and opportunities for PMT inhibitor discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpEWF_XeLEY1rVg90H21EOLACvtfcHk0lhmjfk9R54-Kg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvFKqtbbJ&md5=8fb2b2d21d306c02a2da60a3fb88c430</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1021%2Fjm501234a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm501234a%26sid%3Dliteratum%253Aachs%26aulast%3DKaniskan%26aufirst%3DH.%2B%25C3%259C.%26aulast%3DKonze%26aufirst%3DK.%2BD.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DSelective%2520inhibitors%2520of%2520protein%2520methyltransferases%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D1596%26epage%3D1629%26doi%3D10.1021%2Fjm501234a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kaniskan, H.
Ü.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martini, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span> <span> </span><span class="NLM_article-title">Inhibitors of protein methyltransferases and demethylases</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>118</i></span>,  <span class="NLM_fpage">989</span>– <span class="NLM_lpage">1068</span>, <span class="refDoi"> DOI: 10.1021/acs.chemrev.6b00801</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.chemrev.6b00801" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=1%3ACAS%3A528%3ADC%252BC2sXkslynsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=118&publication_year=2018&pages=989-1068&author=H.%0A%C3%9C.+Kaniskanauthor=M.+L.+Martiniauthor=J.+Jin&title=Inhibitors+of+protein+methyltransferases+and+demethylases&doi=10.1021%2Facs.chemrev.6b00801"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibitors of Protein Methyltransferases and Demethylases</span></div><div class="casAuthors">Kaniskan, H. Umit; Martini, Michael L.; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">118</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">989-1068</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Post-translational modifications of histones by protein methyltransferases (PMTs) and histone demethylases (KDMs) play an important role in the regulation of gene expression and transcription and are implicated in cancer and many other diseases.  Many of these enzymes also target various nonhistone proteins impacting numerous crucial biol. pathways.  Given their key biol. functions and implications in human diseases, there has been a growing interest in assessing these enzymes as potential therapeutic targets.  Consequently, discovering and developing inhibitors of these enzymes has become a very active and fast-growing research area over the past decade.  In this review, the authors cover the discovery, characterization, and biol. application of inhibitors of PMTs and KDMs with emphasis on key advancements in the field.  The authors also discuss challenges, opportunities, and future directions in this emerging, exciting research field.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpJYjiYtw_n4rVg90H21EOLACvtfcHk0lidMf8qNKvGGQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXkslynsL8%253D&md5=1ed520c6b0c42d2aa8a7f66b0a2d4e40</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1021%2Facs.chemrev.6b00801&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.chemrev.6b00801%26sid%3Dliteratum%253Aachs%26aulast%3DKaniskan%26aufirst%3DH.%2B%25C3%259C.%26aulast%3DMartini%26aufirst%3DM.%2BL.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DInhibitors%2520of%2520protein%2520methyltransferases%2520and%2520demethylases%26jtitle%3DChem.%2520Rev.%26date%3D2018%26volume%3D118%26spage%3D989%26epage%3D1068%26doi%3D10.1021%2Facs.chemrev.6b00801" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kaniskan, H. Ü.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span> <span> </span><span class="NLM_article-title">Recent progress in developing selective inhibitors of protein methyltransferases</span>. <i>Curr. Opin. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>39</i></span>,  <span class="NLM_fpage">100</span>– <span class="NLM_lpage">108</span>, <span class="refDoi"> DOI: 10.1016/j.cbpa.2017.06.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=10.1016%2Fj.cbpa.2017.06.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=28662389" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtVKnt73P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=2017&pages=100-108&author=H.+%C3%9C.+Kaniskanauthor=J.+Jin&title=Recent+progress+in+developing+selective+inhibitors+of+protein+methyltransferases&doi=10.1016%2Fj.cbpa.2017.06.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Recent progress in developing selective inhibitors of protein methyltransferases</span></div><div class="casAuthors">Kaniskan, H. Umit; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">39</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">100-108</span>CODEN:
                <span class="NLM_cas:coden">COCBF4</span>;
        ISSN:<span class="NLM_cas:issn">1367-5931</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Mounting evidence suggests that protein methyltransferases (PMTs), which catalyze methylation of histones as well as non-histone proteins, play a crucial role in diverse biol. pathways and human diseases.  In particular, PMTs have been recognized as major players in regulating gene expression and chromatin state.  There has been an increasingly growing interest in these enzymes as potential therapeutic targets and over the past two years tremendous progress has been made in the discovery of selective, small mol. inhibitors of protein lysine and arginine methyltransferases.  Inhibitors of PMTs have been used extensively in oncol. studies as tool compds., and inhibitors of EZH2, DOT1L and PRMT5 are currently in clin. trials.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoL3rSZhrkACbVg90H21EOLACvtfcHk0lidMf8qNKvGGQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtVKnt73P&md5=ce42299d3bf38bc9898892a0364eca36</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1016%2Fj.cbpa.2017.06.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cbpa.2017.06.013%26sid%3Dliteratum%253Aachs%26aulast%3DKaniskan%26aufirst%3DH.%2B%25C3%259C.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DRecent%2520progress%2520in%2520developing%2520selective%2520inhibitors%2520of%2520protein%2520methyltransferases%26jtitle%3DCurr.%2520Opin.%2520Chem.%2520Biol.%26date%3D2017%26volume%3D39%26spage%3D100%26epage%3D108%26doi%3D10.1016%2Fj.cbpa.2017.06.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qian, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ho, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, Y. G.</span></span> <span> </span><span class="NLM_article-title">Small molecule inhibitors of protein arginine methyltransferases</span>. <i>Expert Opin. Invest. Drugs</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">335</span>– <span class="NLM_lpage">358</span>, <span class="refDoi"> DOI: 10.1517/13543784.2016.1144747</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=10.1517%2F13543784.2016.1144747" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2016&pages=335-358&author=H.+Huauthor=K.+Qianauthor=M.+C.+Hoauthor=Y.+G.+Zheng&title=Small+molecule+inhibitors+of+protein+arginine+methyltransferases&doi=10.1517%2F13543784.2016.1144747"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1517%2F13543784.2016.1144747&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1517%252F13543784.2016.1144747%26sid%3Dliteratum%253Aachs%26aulast%3DHu%26aufirst%3DH.%26aulast%3DQian%26aufirst%3DK.%26aulast%3DHo%26aufirst%3DM.%2BC.%26aulast%3DZheng%26aufirst%3DY.%2BG.%26atitle%3DSmall%2520molecule%2520inhibitors%2520of%2520protein%2520arginine%2520methyltransferases%26jtitle%3DExpert%2520Opin.%2520Invest.%2520Drugs%26date%3D2016%26volume%3D25%26spage%3D335%26epage%3D358%26doi%3D10.1517%2F13543784.2016.1144747" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref55"><div class="reference"><strong class="refLabel"><a href="#ref55" class="rightTabRefNumLink">55</a></strong><div class="NLM_citation" id="rightTab-cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Luo, M. K.</span></span> <span> </span><span class="NLM_article-title">Inhibitors of protein methyltransferases as chemical tools</span>. <i>Epigenomics</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">1327</span>– <span class="NLM_lpage">1338</span>, <span class="refDoi"> DOI: 10.2217/epi.15.87</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=10.2217%2Fepi.15.87" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=26646500" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=1%3ACAS%3A528%3ADC%252BC2MXitVejtbvN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2015&pages=1327-1338&author=M.+K.+Luo&title=Inhibitors+of+protein+methyltransferases+as+chemical+tools&doi=10.2217%2Fepi.15.87"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibitors of protein methyltransferases as chemical tools</span></div><div class="casAuthors">Luo, Minkui</div><div class="citationInfo"><span class="NLM_cas:title">Epigenomics</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1327-1338</span>CODEN:
                <span class="NLM_cas:coden">EPIGC7</span>;
        ISSN:<span class="NLM_cas:issn">1750-192X</span>.
    
            (<span class="NLM_cas:orgname">Future Medicine Ltd.</span>)
        </div><div class="casAbstract">Protein methyltransferases (PMT)s play essential roles in many biol. processes through methylation of histones and diverse nonhistone substrates.  Dysregulation of these enzymes has been implicated in many diseases including cancers.  While PMT-assocd. biol. can be probed via genetic perturbation, this approach targets full-length PMTs rather than their methyltransferase activities and often lacks temporal, spatial and dose controls (timing, location and amt. of dosed compds.).  By contrast, small-mol. inhibitors of PMTs can be designed to specifically target the methyltransferase domains in a temporal, spatial and dose-dependent manner.  This utility has motivated the development of hundreds of PMT inhibitors, but meanwhile can make it challenging to select the most suitable PMT inhibitors to interrogate PMT-assocd. biol.  This perspective aims to provide timely guidance to evaluate these PMT inhibitors in their relevant biol. contexts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpNoz6wEWA3UrVg90H21EOLACvtfcHk0lidMf8qNKvGGQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXitVejtbvN&md5=fb64069b1c54f27a35e604318fee99be</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.2217%2Fepi.15.87&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2217%252Fepi.15.87%26sid%3Dliteratum%253Aachs%26aulast%3DLuo%26aufirst%3DM.%2BK.%26atitle%3DInhibitors%2520of%2520protein%2520methyltransferases%2520as%2520chemical%2520tools%26jtitle%3DEpigenomics%26date%3D2015%26volume%3D7%26spage%3D1327%26epage%3D1338%26doi%3D10.2217%2Fepi.15.87" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref56"><div class="reference"><strong class="refLabel"><a href="#ref56" class="rightTabRefNumLink">56</a></strong><div class="NLM_citation" id="rightTab-cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chan-Penebre, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuplast, K. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Majer, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boriack-Sjodin, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wigle, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnston, L. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rioux, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Munchhof, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacques, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">West, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lingaraj, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stickland, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ribich, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raimondi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waters, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pollock, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barbash, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pappalardi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ho, T. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nurse, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oza, K. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gallagher, K. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kruger, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moyer, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Copeland, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chesworth, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duncan, K. W.</span></span> <span> </span><span class="NLM_article-title">A selective inhibitor of PRMT5 with in vivo and in vitro potency in MCL models</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">432</span>– <span class="NLM_lpage">437</span>, <span class="refDoi"> DOI: 10.1038/nchembio.1810</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=10.1038%2Fnchembio.1810" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=25915199" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=1%3ACAS%3A528%3ADC%252BC2MXotFartbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2015&pages=432-437&author=E.+Chan-Penebreauthor=K.+G.+Kuplastauthor=C.+R.+Majerauthor=P.+A.+Boriack-Sjodinauthor=T.+J.+Wigleauthor=L.+D.+Johnstonauthor=N.+Riouxauthor=M.+J.+Munchhofauthor=L.+Jinauthor=S.+L.+Jacquesauthor=K.+A.+Westauthor=T.+Lingarajauthor=K.+Sticklandauthor=S.+A.+Ribichauthor=A.+Raimondiauthor=M.+P.+Scottauthor=N.+J.+Watersauthor=R.+M.+Pollockauthor=J.+J.+Smithauthor=O.+Barbashauthor=M.+Pappalardiauthor=T.+F.+Hoauthor=K.+Nurseauthor=K.+P.+Ozaauthor=K.+T.+Gallagherauthor=R.+Krugerauthor=M.+P.+Moyerauthor=R.+A.+Copelandauthor=R.+Chesworthauthor=K.+W.+Duncan&title=A+selective+inhibitor+of+PRMT5+with+in+vivo+and+in+vitro+potency+in+MCL+models&doi=10.1038%2Fnchembio.1810"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">A selective inhibitor of PRMT5 with in vivo and in vitro potency in MCL models</span></div><div class="casAuthors">Chan-Penebre, Elayne; Kuplast, Kristy G.; Majer, Christina R.; Boriack-Sjodin, P. Ann; Wigle, Tim J.; Johnston, L. Danielle; Rioux, Nathalie; Munchhof, Michael J.; Jin, Lei; Jacques, Suzanne L.; West, Kip A.; Lingaraj, Trupti; Stickland, Kimberly; Ribich, Scott A.; Raimondi, Alejandra; Scott, Margaret Porter; Waters, Nigel J.; Pollock, Roy M.; Smith, Jesse J.; Barbash, Olena; Pappalardi, Melissa; Ho, Thau F.; Nurse, Kelvin; Oza, Khyati P.; Gallagher, Kathleen T.; Kruger, Ryan; Moyer, Mikel P.; Copeland, Robert A.; Chesworth, Richard; Duncan, Kenneth W.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">432-437</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Protein arginine methyltransferase-5 (PRMT5) is reported to have a role in diverse cellular processes, including tumorigenesis, and its overexpression is obsd. in cell lines and primary patient samples derived from lymphomas, particularly mantle cell lymphoma (MCL).  Here we describe the identification and characterization of a potent and selective inhibitor of PRMT5 with antiproliferative effects in both in vitro and in vivo models of MCL.  EPZ015666 (GSK3235025) is an orally available inhibitor of PRMT5 enzymic activity in biochem. assays with a half-maximal inhibitory concn. (IC50) of 22 nM and broad selectivity against a panel of other histone methyltransferases.  Treatment of MCL cell lines with EPZ015666 led to inhibition of SmD3 methylation and cell death, with IC50 values in the nanomolar range.  Oral dosing with EPZ015666 demonstrated dose-dependent antitumor activity in multiple MCL xenograft models.  EPZ015666 represents a validated chem. probe for further study of PRMT5 biol. and arginine methylation in cancer and other diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoJTtcvcqvOJLVg90H21EOLACvtfcHk0li9eEAeD7_Beg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXotFartbo%253D&md5=bf0686e1c2d9d96ac5deef22c0bf80b7</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1810&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1810%26sid%3Dliteratum%253Aachs%26aulast%3DChan-Penebre%26aufirst%3DE.%26aulast%3DKuplast%26aufirst%3DK.%2BG.%26aulast%3DMajer%26aufirst%3DC.%2BR.%26aulast%3DBoriack-Sjodin%26aufirst%3DP.%2BA.%26aulast%3DWigle%26aufirst%3DT.%2BJ.%26aulast%3DJohnston%26aufirst%3DL.%2BD.%26aulast%3DRioux%26aufirst%3DN.%26aulast%3DMunchhof%26aufirst%3DM.%2BJ.%26aulast%3DJin%26aufirst%3DL.%26aulast%3DJacques%26aufirst%3DS.%2BL.%26aulast%3DWest%26aufirst%3DK.%2BA.%26aulast%3DLingaraj%26aufirst%3DT.%26aulast%3DStickland%26aufirst%3DK.%26aulast%3DRibich%26aufirst%3DS.%2BA.%26aulast%3DRaimondi%26aufirst%3DA.%26aulast%3DScott%26aufirst%3DM.%2BP.%26aulast%3DWaters%26aufirst%3DN.%2BJ.%26aulast%3DPollock%26aufirst%3DR.%2BM.%26aulast%3DSmith%26aufirst%3DJ.%2BJ.%26aulast%3DBarbash%26aufirst%3DO.%26aulast%3DPappalardi%26aufirst%3DM.%26aulast%3DHo%26aufirst%3DT.%2BF.%26aulast%3DNurse%26aufirst%3DK.%26aulast%3DOza%26aufirst%3DK.%2BP.%26aulast%3DGallagher%26aufirst%3DK.%2BT.%26aulast%3DKruger%26aufirst%3DR.%26aulast%3DMoyer%26aufirst%3DM.%2BP.%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26aulast%3DChesworth%26aufirst%3DR.%26aulast%3DDuncan%26aufirst%3DK.%2BW.%26atitle%3DA%2520selective%2520inhibitor%2520of%2520PRMT5%2520with%2520in%2520vivo%2520and%2520in%2520vitro%2520potency%2520in%2520MCL%2520models%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2015%26volume%3D11%26spage%3D432%26epage%3D437%26doi%3D10.1038%2Fnchembio.1810" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref57"><div class="reference"><strong class="refLabel"><a href="#ref57" class="rightTabRefNumLink">57</a></strong><div class="NLM_citation" id="rightTab-cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Duncan, K. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rioux, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boriack-Sjodin, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Munchhof, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reiter, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Majer, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnston, L. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan-Penebre, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuplast, K. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Porter
Scott, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pollock, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waters, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moyer, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Copeland, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chesworth, R.</span></span> <span> </span><span class="NLM_article-title">Structure and property guided design in the identification of PRMT5 tool compound EPZ015666</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">162</span>– <span class="NLM_lpage">166</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.5b00380</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.5b00380" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvFGnsr3L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=162-166&author=K.+W.+Duncanauthor=N.+Riouxauthor=P.+A.+Boriack-Sjodinauthor=M.+J.+Munchhofauthor=L.+A.+Reiterauthor=C.+R.+Majerauthor=L.+Jinauthor=L.+D.+Johnstonauthor=E.+Chan-Penebreauthor=K.+G.+Kuplastauthor=M.+Porter%0AScottauthor=R.+M.+Pollockauthor=N.+J.+Watersauthor=J.+J.+Smithauthor=M.+P.+Moyerauthor=R.+A.+Copelandauthor=R.+Chesworth&title=Structure+and+property+guided+design+in+the+identification+of+PRMT5+tool+compound+EPZ015666&doi=10.1021%2Facsmedchemlett.5b00380"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">Structure and Property Guided Design in the Identification of PRMT5 Tool Compound EPZ015666</span></div><div class="casAuthors">Duncan, Kenneth W.; Rioux, Nathalie; Boriack-Sjodin, P. Ann; Munchhof, Michael J.; Reiter, Lawrence A.; Majer, Christina R.; Jin, Lei; Johnston, L. Danielle; Chan-Penebre, Elayne; Kuplast, Kristy G.; Porter Scott, Margaret; Pollock, Roy M.; Waters, Nigel J.; Smith, Jesse J.; Moyer, Mikel P.; Copeland, Robert A.; Chesworth, Richard</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">162-166</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The recent publication of a potent and selective inhibitor of protein methyltransferase 5 (PRMT5) provides the scientific community with in vivo-active tool compd. I (EPZ015666, GSK3235025) to probe the underlying pharmacol. of this key enzyme.  Herein, the authors report the design and optimization strategies employed on an initial hit compd. with poor in vitro clearance to yield in vivo tool compd. I and an addnl. potent in vitro tool mol. II (GSK3203591).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq2V7LiQgVZjrVg90H21EOLACvtfcHk0li9eEAeD7_Beg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvFGnsr3L&md5=e7ac92c2d985da4994e9bb959adededf</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.5b00380&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.5b00380%26sid%3Dliteratum%253Aachs%26aulast%3DDuncan%26aufirst%3DK.%2BW.%26aulast%3DRioux%26aufirst%3DN.%26aulast%3DBoriack-Sjodin%26aufirst%3DP.%2BA.%26aulast%3DMunchhof%26aufirst%3DM.%2BJ.%26aulast%3DReiter%26aufirst%3DL.%2BA.%26aulast%3DMajer%26aufirst%3DC.%2BR.%26aulast%3DJin%26aufirst%3DL.%26aulast%3DJohnston%26aufirst%3DL.%2BD.%26aulast%3DChan-Penebre%26aufirst%3DE.%26aulast%3DKuplast%26aufirst%3DK.%2BG.%26aulast%3DPorter%2BScott%26aufirst%3DM.%26aulast%3DPollock%26aufirst%3DR.%2BM.%26aulast%3DWaters%26aufirst%3DN.%2BJ.%26aulast%3DSmith%26aufirst%3DJ.%2BJ.%26aulast%3DMoyer%26aufirst%3DM.%2BP.%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26aulast%3DChesworth%26aufirst%3DR.%26atitle%3DStructure%2520and%2520property%2520guided%2520design%2520in%2520the%2520identification%2520of%2520PRMT5%2520tool%2520compound%2520EPZ015666%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2016%26volume%3D7%26spage%3D162%26epage%3D166%26doi%3D10.1021%2Facsmedchemlett.5b00380" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref58"><div class="reference"><strong class="refLabel"><a href="#ref58" class="rightTabRefNumLink">58</a></strong><div class="NLM_citation" id="rightTab-cit58"><span>ClinicalTrials.gov. <a href="https://clinicaltrials.gov/ct2/show/NCT02783300" class="extLink">https://clinicaltrials.gov/ct2/show/NCT02783300</a> (accessed March 24, <span class="NLM_year">2018</span>).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=ClinicalTrials.gov.+https%3A%2F%2Fclinicaltrials.gov%2Fct2%2Fshow%2FNCT02783300+%28accessed+March+24%2C+2018%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26date%3D2018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref59"><div class="reference"><strong class="refLabel"><a href="#ref59" class="rightTabRefNumLink">59</a></strong><div class="NLM_citation" id="rightTab-cit59"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Falk, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Foitzik, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allan, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">deSilva, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bozikis, Y. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nikac, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walker, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Camerino, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morrow, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stupple, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lagiakos, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pinson, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lessene, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kersten, W. J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ganame, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holmes, I. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lunniss, G. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hermans, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parker, M. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thistlethwaite, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">White, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Charman, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pilling, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Monahan, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grusovin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peat, T. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sonderegger, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toulmin, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jane, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Curtis, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stupple, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Street, I. P.</span></span> <span> </span><span class="NLM_article-title">PRMT5 inhibitors as novel treatment for cancers</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>75</i></span> (<span class="NLM_issue">15, Suppl.</span>),  <span class="NLM_fpage">5371</span>, (Abstract in Proceedings of the 106th Annual Meeting of the American Association for Cancer Research; Apr 18–22, 2015; Philadelphia, PA) <span class="refDoi"> DOI: 10.1158/1538-7445.AM2015-5371</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=10.1158%2F1538-7445.AM2015-5371" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=75&publication_year=2015&pages=5371&issue=15%2C+Suppl.&author=H.+Falkauthor=R.+C.+Foitzikauthor=E.+Allanauthor=M.+deSilvaauthor=H.+Yangauthor=Y.+E.+Bozikisauthor=M.+Nikacauthor=S.+R.+Walkerauthor=M.+A.+Camerinoauthor=B.+J.+Morrowauthor=A.+E.+Stuppleauthor=R.+Lagiakosauthor=J.+A.+Pinsonauthor=R.+Lesseneauthor=W.+J.+A.+Kerstenauthor=D.+G.+Ganameauthor=I.+P.+Holmesauthor=G.+E.+Lunnissauthor=M.+Chungauthor=S.+J.+Hermansauthor=M.+W.+Parkerauthor=A.+Thistlethwaiteauthor=K.+L.+Whiteauthor=S.+A.+Charmanauthor=P.+Pillingauthor=B.+J.+Monahanauthor=J.+Grusovinauthor=T.+S.+Peatauthor=S.+Sondereggerauthor=E.+Toulminauthor=S.+M.+Janeauthor=D.+J.+Curtisauthor=P.+A.+Stuppleauthor=I.+P.+Street&title=PRMT5+inhibitors+as+novel+treatment+for+cancers&doi=10.1158%2F1538-7445.AM2015-5371"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1158%2F1538-7445.AM2015-5371&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1538-7445.AM2015-5371%26sid%3Dliteratum%253Aachs%26aulast%3DFalk%26aufirst%3DH.%26aulast%3DFoitzik%26aufirst%3DR.%2BC.%26aulast%3DAllan%26aufirst%3DE.%26aulast%3DdeSilva%26aufirst%3DM.%26aulast%3DYang%26aufirst%3DH.%26aulast%3DBozikis%26aufirst%3DY.%2BE.%26aulast%3DNikac%26aufirst%3DM.%26aulast%3DWalker%26aufirst%3DS.%2BR.%26aulast%3DCamerino%26aufirst%3DM.%2BA.%26aulast%3DMorrow%26aufirst%3DB.%2BJ.%26aulast%3DStupple%26aufirst%3DA.%2BE.%26aulast%3DLagiakos%26aufirst%3DR.%26aulast%3DPinson%26aufirst%3DJ.%2BA.%26aulast%3DLessene%26aufirst%3DR.%26aulast%3DKersten%26aufirst%3DW.%2BJ.%2BA.%26aulast%3DGaname%26aufirst%3DD.%2BG.%26aulast%3DHolmes%26aufirst%3DI.%2BP.%26aulast%3DLunniss%26aufirst%3DG.%2BE.%26aulast%3DChung%26aufirst%3DM.%26aulast%3DHermans%26aufirst%3DS.%2BJ.%26aulast%3DParker%26aufirst%3DM.%2BW.%26aulast%3DThistlethwaite%26aufirst%3DA.%26aulast%3DWhite%26aufirst%3DK.%2BL.%26aulast%3DCharman%26aufirst%3DS.%2BA.%26aulast%3DPilling%26aufirst%3DP.%26aulast%3DMonahan%26aufirst%3DB.%2BJ.%26aulast%3DGrusovin%26aufirst%3DJ.%26aulast%3DPeat%26aufirst%3DT.%2BS.%26aulast%3DSonderegger%26aufirst%3DS.%26aulast%3DToulmin%26aufirst%3DE.%26aulast%3DJane%26aufirst%3DS.%2BM.%26aulast%3DCurtis%26aufirst%3DD.%2BJ.%26aulast%3DStupple%26aufirst%3DP.%2BA.%26aulast%3DStreet%26aufirst%3DI.%2BP.%26atitle%3DPRMT5%2520inhibitors%2520as%2520novel%2520treatment%2520for%2520cancers%26jtitle%3DCancer%2520Res.%26date%3D2015%26volume%3D75%26issue%3D15%252C%2520Suppl%26spage%3D5371%26doi%3D10.1158%2F1538-7445.AM2015-5371" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref60"><div class="reference"><strong class="refLabel"><a href="#ref60" class="rightTabRefNumLink">60</a></strong><div class="NLM_citation" id="rightTab-cit60"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bergman, Y. E.</span>; <span class="NLM_string-name">Camerino, M. A.</span>; <span class="NLM_string-name">Stupple, P. A.</span></span> <span> </span><span class="NLM_article-title">Benzopiperidine Derivatives and Their Use in the Treatment of Cancer and Hemoglobinopathies</span>. <span class="NLM_patent">WO2017153520 A1</span>, <span class="NLM_year">2017</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2017&author=Y.+E.+Bergman&author=M.+A.+Camerino&author=P.+A.+Stupple&title=Benzopiperidine+Derivatives+and+Their+Use+in+the+Treatment+of+Cancer+and+Hemoglobinopathies"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DBergman%26aufirst%3DY.%2BE.%26atitle%3DBenzopiperidine%2520Derivatives%2520and%2520Their%2520Use%2520in%2520the%2520Treatment%2520of%2520Cancer%2520and%2520Hemoglobinopathies%26date%3D2017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref61"><div class="reference"><strong class="refLabel"><a href="#ref61" class="rightTabRefNumLink">61</a></strong><div class="NLM_citation" id="rightTab-cit61"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Stupple, P. A.</span></span> <span> </span><span class="NLM_article-title">Pyridine Derivatives and Their Use in the Treatment of Cancer and Hemoglobinopathies</span>. <span class="NLM_patent">WO2017153521 A1</span>, <span class="NLM_year">2017</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2017&author=P.+A.+Stupple&title=Pyridine+Derivatives+and+Their+Use+in+the+Treatment+of+Cancer+and+Hemoglobinopathies"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DStupple%26aufirst%3DP.%2BA.%26atitle%3DPyridine%2520Derivatives%2520and%2520Their%2520Use%2520in%2520the%2520Treatment%2520of%2520Cancer%2520and%2520Hemoglobinopathies%26date%3D2017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref62"><div class="reference"><strong class="refLabel"><a href="#ref62" class="rightTabRefNumLink">62</a></strong><div class="NLM_citation" id="rightTab-cit62"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ye, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mao, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xing, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, M.</span></span> <span> </span><span class="NLM_article-title">Discovery and optimization of selective inhibitors of protein arginine methyltransferase 5 by docking-based virtual screening</span>. <i>Org. Biomol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">3648</span>– <span class="NLM_lpage">3661</span>, <span class="refDoi"> DOI: 10.1039/C7OB00070G</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=10.1039%2FC7OB00070G" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=28397890" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=1%3ACAS%3A528%3ADC%252BC2sXltl2mtr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2017&pages=3648-3661&author=Y.+Yeauthor=B.+Zhangauthor=R.+Maoauthor=C.+Zhangauthor=Y.+Wangauthor=J.+Xingauthor=Y.+C.+Liuauthor=X.+Luoauthor=H.+Dingauthor=Y.+Yangauthor=B.+Zhouauthor=H.+Jiangauthor=K.+Chenauthor=C.+Luoauthor=M.+Zheng&title=Discovery+and+optimization+of+selective+inhibitors+of+protein+arginine+methyltransferase+5+by+docking-based+virtual+screening&doi=10.1039%2FC7OB00070G"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit62R"><div class="casContent"><span class="casTitleNuber">62</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery and optimization of selective inhibitors of protein arginine methyltransferase 5 by docking-based virtual screening</span></div><div class="casAuthors">Ye, Yan; Zhang, Bidong; Mao, Ruifeng; Zhang, Chenhua; Wang, Yulan; Xing, Jing; Liu, Yu-Chih; Luo, Xiaomin; Ding, Hong; Yang, Yaxi; Zhou, Bing; Jiang, Hualiang; Chen, Kaixian; Luo, Cheng; Zheng, Mingyue</div><div class="citationInfo"><span class="NLM_cas:title">Organic & Biomolecular Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">3648-3661</span>CODEN:
                <span class="NLM_cas:coden">OBCRAK</span>;
        ISSN:<span class="NLM_cas:issn">1477-0520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Protein arginine methyltransferase 5 (PRMT5) is a type II PRMT enzyme crit. for diverse cellular processes and different types of cancers.  Many efforts have been made to discover novel scaffold PRMT5 inhibitors.  Herein, the authors report the discovery of DC_P33 as a hit compd. of PRMT5 inhibitor, identified by mol. docking based virtual screening and 3H-labeled radioactive methylation assays.  Structure-activity relationship (SAR) anal. was performed on the analogs of DC_P33 and then structural modifications were done to improve its activity.  Among the derivs., the compd. DC_C01 displayed an IC50 value of 2.8 μM, and good selectivity toward PRMT1, EZH2 and DNMT3A.  Moreover, DC_C01 exhibited anti-proliferation activities against Z-138, Maver-1, and Jeko-1 cancer cells with EC50 values of 12 μM, 12 μM, and 10.5 μM, resp.  Taken together, these results contribute to the development of specific inhibitors against PRMT5 and cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqkWfdAd-COtbVg90H21EOLACvtfcHk0liIm4lq74IKGw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXltl2mtr0%253D&md5=ead3b89fa64875e2dbf47c3a99714564</span></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1039%2FC7OB00070G&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC7OB00070G%26sid%3Dliteratum%253Aachs%26aulast%3DYe%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DB.%26aulast%3DMao%26aufirst%3DR.%26aulast%3DZhang%26aufirst%3DC.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DXing%26aufirst%3DJ.%26aulast%3DLiu%26aufirst%3DY.%2BC.%26aulast%3DLuo%26aufirst%3DX.%26aulast%3DDing%26aufirst%3DH.%26aulast%3DYang%26aufirst%3DY.%26aulast%3DZhou%26aufirst%3DB.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DChen%26aufirst%3DK.%26aulast%3DLuo%26aufirst%3DC.%26aulast%3DZheng%26aufirst%3DM.%26atitle%3DDiscovery%2520and%2520optimization%2520of%2520selective%2520inhibitors%2520of%2520protein%2520arginine%2520methyltransferase%25205%2520by%2520docking-based%2520virtual%2520screening%26jtitle%3DOrg.%2520Biomol.%2520Chem.%26date%3D2017%26volume%3D15%26spage%3D3648%26epage%3D3661%26doi%3D10.1039%2FC7OB00070G" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref63"><div class="reference"><strong class="refLabel"><a href="#ref63" class="rightTabRefNumLink">63</a></strong><div class="NLM_citation" id="rightTab-cit63"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Couture, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hauk, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blackburn, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trievel, R. C.</span></span> <span> </span><span class="NLM_article-title">Catalytic roles for carbon–oxygen hydrogen bonding in SET domain lysine methyltransferases</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>281</i></span>,  <span class="NLM_fpage">19280</span>– <span class="NLM_lpage">19287</span>, <span class="refDoi"> DOI: 10.1074/jbc.M602257200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=10.1074%2Fjbc.M602257200" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=281&publication_year=2006&pages=19280-19287&author=J.+F.+Coutureauthor=G.+Haukauthor=M.+J.+Thompsonauthor=G.+M.+Blackburnauthor=R.+C.+Trievel&title=Catalytic+roles+for+carbon%E2%80%93oxygen+hydrogen+bonding+in+SET+domain+lysine+methyltransferases&doi=10.1074%2Fjbc.M602257200"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M602257200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M602257200%26sid%3Dliteratum%253Aachs%26aulast%3DCouture%26aufirst%3DJ.%2BF.%26aulast%3DHauk%26aufirst%3DG.%26aulast%3DThompson%26aufirst%3DM.%2BJ.%26aulast%3DBlackburn%26aufirst%3DG.%2BM.%26aulast%3DTrievel%26aufirst%3DR.%2BC.%26atitle%3DCatalytic%2520roles%2520for%2520carbon%25E2%2580%2593oxygen%2520hydrogen%2520bonding%2520in%2520SET%2520domain%2520lysine%2520methyltransferases%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2006%26volume%3D281%26spage%3D19280%26epage%3D19287%26doi%3D10.1074%2Fjbc.M602257200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref64"><div class="reference"><strong class="refLabel"><a href="#ref64" class="rightTabRefNumLink">64</a></strong><div class="NLM_citation" id="rightTab-cit64"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chin, H. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patnaik, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Esteve, P. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacobsen, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pradhan, S.</span></span> <span> </span><span class="NLM_article-title">Catalytic properties and kinetic mechanism of human recombinant lys-9 histone H3 methyltransferase SUV39H1: participation of the chromodomain in enzymatic catalysis</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>45</i></span>,  <span class="NLM_fpage">3272</span>– <span class="NLM_lpage">3284</span>, <span class="refDoi"> DOI: 10.1021/bi051997r</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi051997r" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2006&pages=3272-3284&author=H.+G.+Chinauthor=D.+Patnaikauthor=P.+O.+Esteveauthor=S.+E.+Jacobsenauthor=S.+Pradhan&title=Catalytic+properties+and+kinetic+mechanism+of+human+recombinant+lys-9+histone+H3+methyltransferase+SUV39H1%3A+participation+of+the+chromodomain+in+enzymatic+catalysis&doi=10.1021%2Fbi051997r"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=10.1021%2Fbi051997r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi051997r%26sid%3Dliteratum%253Aachs%26aulast%3DChin%26aufirst%3DH.%2BG.%26aulast%3DPatnaik%26aufirst%3DD.%26aulast%3DEsteve%26aufirst%3DP.%2BO.%26aulast%3DJacobsen%26aufirst%3DS.%2BE.%26aulast%3DPradhan%26aufirst%3DS.%26atitle%3DCatalytic%2520properties%2520and%2520kinetic%2520mechanism%2520of%2520human%2520recombinant%2520lys-9%2520histone%2520H3%2520methyltransferase%2520SUV39H1%253A%2520participation%2520of%2520the%2520chromodomain%2520in%2520enzymatic%2520catalysis%26jtitle%3DBiochemistry%26date%3D2006%26volume%3D45%26spage%3D3272%26epage%3D3284%26doi%3D10.1021%2Fbi051997r" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref65"><div class="reference"><strong class="refLabel"><a href="#ref65" class="rightTabRefNumLink">65</a></strong><div class="NLM_citation" id="rightTab-cit65"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Smil, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eram, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kennedy, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szewczyk, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barsyte-Lovejoy, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vedadi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schapira, M.</span></span> <span> </span><span class="NLM_article-title">Discovery of a dual PRMT5-PRMT7 inhibitor</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">408</span>– <span class="NLM_lpage">412</span>, <span class="refDoi"> DOI: 10.1021/ml500467h</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ml500467h" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=1%3ACAS%3A528%3ADC%252BC2MXjs1Ggu7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=408-412&author=D.+Smilauthor=M.+S.+Eramauthor=F.+Liauthor=S.+Kennedyauthor=M.+M.+Szewczykauthor=P.+J.+Brownauthor=D.+Barsyte-Lovejoyauthor=C.+H.+Arrowsmithauthor=M.+Vedadiauthor=M.+Schapira&title=Discovery+of+a+dual+PRMT5-PRMT7+inhibitor&doi=10.1021%2Fml500467h"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit65R"><div class="casContent"><span class="casTitleNuber">65</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a Dual PRMT5-PRMT7 Inhibitor</span></div><div class="casAuthors">Smil, David; Eram, Mohammad S.; Li, Fengling; Kennedy, Steven; Szewczyk, Magdalena M.; Brown, Peter J.; Barsyte-Lovejoy, Dalia; Arrowsmith, Cheryl H.; Vedadi, Masoud; Schapira, Matthieu</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">408-412</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The protein arginine methyltransferases PRMT7 and PRMT5, resp., monomethylate and sym. dimethylate arginine side-chains of proteins involved in diverse cellular mechanisms, including chromatin-mediated control of gene transcription, splicing, and the RAS to ERK transduction cascade.  It is believed that PRMT5 and PRMT7 act in conjunction to methylate their substrates, and genetic deletions support the notion that these enzymes derepress cell proliferation and migration in cancer.  Using available structures of PRMT5, we designed DS-437, a PRMT5 inhibitor with an IC50 value of 6 μM against both PRMT5 and PRMT7 that is inactive against 29 other human protein-, DNA-, and RNA-methyltransferases and inhibits sym. dimethylation of PRMT5 substrates in cells.  This compd. behaves as a cofactor competitor and represents a valid scaffold to interrogate the potential of the PRMT5-PRMT7 axis as a target for therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq9J078gEcjRLVg90H21EOLACvtfcHk0lifrOkr4TZMPw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXjs1Ggu7k%253D&md5=8eefe93068418c61881850d3b4d39676</span></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1021%2Fml500467h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fml500467h%26sid%3Dliteratum%253Aachs%26aulast%3DSmil%26aufirst%3DD.%26aulast%3DEram%26aufirst%3DM.%2BS.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DKennedy%26aufirst%3DS.%26aulast%3DSzewczyk%26aufirst%3DM.%2BM.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DSchapira%26aufirst%3DM.%26atitle%3DDiscovery%2520of%2520a%2520dual%2520PRMT5-PRMT7%2520inhibitor%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2015%26volume%3D6%26spage%3D408%26epage%3D412%26doi%3D10.1021%2Fml500467h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref66"><div class="reference"><strong class="refLabel"><a href="#ref66" class="rightTabRefNumLink">66</a></strong><div class="NLM_citation" id="rightTab-cit66"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bonday, Z. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cortez, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grogan, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Antonysamy, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weichert, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bocchinfuso, W. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kennedy, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mader, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eram, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szewczyk, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barsyte-Lovejoy, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vedadi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guccione, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campbell, R. M.</span></span> <span> </span><span class="NLM_article-title">LLY-283, a potent and selective inhibitor of arginine methyltransferase 5, PRMT5, with antitumor activity</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.8b00014</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.8b00014" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2018&author=Z.+Q.+Bondayauthor=G.+S.+Cortezauthor=M.+J.+Groganauthor=S.+Antonysamyauthor=K.+Weichertauthor=W.+P.+Bocchinfusoauthor=F.+Liauthor=S.+Kennedyauthor=B.+Liauthor=M.+M.+Maderauthor=C.+H.+Arrowsmithauthor=P.+J.+Brownauthor=M.+S.+Eramauthor=M.+M.+Szewczykauthor=D.+Barsyte-Lovejoyauthor=M.+Vedadiauthor=E.+Guccioneauthor=R.+M.+Campbell&title=LLY-283%2C+a+potent+and+selective+inhibitor+of+arginine+methyltransferase+5%2C+PRMT5%2C+with+antitumor+activity&doi=10.1021%2Facsmedchemlett.8b00014"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.8b00014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.8b00014%26sid%3Dliteratum%253Aachs%26aulast%3DBonday%26aufirst%3DZ.%2BQ.%26aulast%3DCortez%26aufirst%3DG.%2BS.%26aulast%3DGrogan%26aufirst%3DM.%2BJ.%26aulast%3DAntonysamy%26aufirst%3DS.%26aulast%3DWeichert%26aufirst%3DK.%26aulast%3DBocchinfuso%26aufirst%3DW.%2BP.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DKennedy%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DB.%26aulast%3DMader%26aufirst%3DM.%2BM.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DEram%26aufirst%3DM.%2BS.%26aulast%3DSzewczyk%26aufirst%3DM.%2BM.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DGuccione%26aufirst%3DE.%26aulast%3DCampbell%26aufirst%3DR.%2BM.%26atitle%3DLLY-283%252C%2520a%2520potent%2520and%2520selective%2520inhibitor%2520of%2520arginine%2520methyltransferase%25205%252C%2520PRMT5%252C%2520with%2520antitumor%2520activity%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2018%26doi%3D10.1021%2Facsmedchemlett.8b00014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref67"><div class="reference"><strong class="refLabel"><a href="#ref67" class="rightTabRefNumLink">67</a></strong><div class="NLM_citation" id="rightTab-cit67"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wu, T.</span>; <span class="NLM_string-name">Brehmer, D.</span>; <span class="NLM_string-name">Beke, L.</span>; <span class="NLM_string-name">Boeckx, A.</span>; <span class="NLM_string-name">Diels, G.
S. M.</span>; <span class="NLM_string-name">Gilissen, R. A. H. J.</span>; <span class="NLM_string-name">Lawson, E. C.</span>; <span class="NLM_string-name">Pande, V.</span>; <span class="NLM_string-name">Parade, M. C. B. C.</span>; <span class="NLM_string-name">Schepens, W. B. G.</span></span> <span> </span><span class="NLM_article-title">Novel 6-6 Bicyclic Aromatic Ring Substituted Nucleoside Analogues for Use as PRMT5 Inhibitors</span>. <span class="NLM_patent">WO2017032840 A1</span>, <span class="NLM_year">2017</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2017&author=T.+Wu&author=D.+Brehmer&author=L.+Beke&author=A.+Boeckx&author=G.%0AS.+M.+Diels&author=R.+A.+H.+J.+Gilissen&author=E.+C.+Lawson&author=V.+Pande&author=M.+C.+B.+C.+Parade&author=W.+B.+G.+Schepens&title=Novel+6-6+Bicyclic+Aromatic+Ring+Substituted+Nucleoside+Analogues+for+Use+as+PRMT5+Inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit67&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DT.%26atitle%3DNovel%25206-6%2520Bicyclic%2520Aromatic%2520Ring%2520Substituted%2520Nucleoside%2520Analogues%2520for%2520Use%2520as%2520PRMT5%2520Inhibitors%26date%3D2017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref68"><div class="reference"><strong class="refLabel"><a href="#ref68" class="rightTabRefNumLink">68</a></strong><div class="NLM_citation" id="rightTab-cit68"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Alinari, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mahasenan, K. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karkhanis, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quinion, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, P. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patton, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lapalombella, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roy, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Leo, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pileri, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Agostinelli, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ayers, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen-Kiang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elemento, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Motiwala, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Majumder, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Byrd, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacob, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sif, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baiocchi, R. A.</span></span> <span> </span><span class="NLM_article-title">Selective inhibition of protein arginine methyltransferase 5 blocks initiation and maintenance of B-cell transformation</span>. <i>Blood</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>125</i></span>,  <span class="NLM_fpage">2530</span>– <span class="NLM_lpage">2543</span>, <span class="refDoi"> DOI: 10.1182/blood-2014-12-619783</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=10.1182%2Fblood-2014-12-619783" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=125&publication_year=2015&pages=2530-2543&author=L.+Alinariauthor=K.+V.+Mahasenanauthor=F.+Yanauthor=V.+Karkhanisauthor=J.+H.+Chungauthor=E.+M.+Smithauthor=C.+Quinionauthor=P.+L.+Smithauthor=L.+Kimauthor=J.+T.+Pattonauthor=R.+Lapalombellaauthor=B.+Yuauthor=Y.+Wuauthor=S.+Royauthor=A.+De+Leoauthor=S.+Pileriauthor=C.+Agostinelliauthor=L.+Ayersauthor=J.+E.+Bradnerauthor=S.+Chen-Kiangauthor=O.+Elementoauthor=T.+Motiwalaauthor=S.+Majumderauthor=J.+C.+Byrdauthor=S.+Jacobauthor=S.+Sifauthor=C.+Liauthor=R.+A.+Baiocchi&title=Selective+inhibition+of+protein+arginine+methyltransferase+5+blocks+initiation+and+maintenance+of+B-cell+transformation&doi=10.1182%2Fblood-2014-12-619783"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit68&amp;dbid=16384&amp;doi=10.1182%2Fblood-2014-12-619783&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1182%252Fblood-2014-12-619783%26sid%3Dliteratum%253Aachs%26aulast%3DAlinari%26aufirst%3DL.%26aulast%3DMahasenan%26aufirst%3DK.%2BV.%26aulast%3DYan%26aufirst%3DF.%26aulast%3DKarkhanis%26aufirst%3DV.%26aulast%3DChung%26aufirst%3DJ.%2BH.%26aulast%3DSmith%26aufirst%3DE.%2BM.%26aulast%3DQuinion%26aufirst%3DC.%26aulast%3DSmith%26aufirst%3DP.%2BL.%26aulast%3DKim%26aufirst%3DL.%26aulast%3DPatton%26aufirst%3DJ.%2BT.%26aulast%3DLapalombella%26aufirst%3DR.%26aulast%3DYu%26aufirst%3DB.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DRoy%26aufirst%3DS.%26aulast%3DDe%2BLeo%26aufirst%3DA.%26aulast%3DPileri%26aufirst%3DS.%26aulast%3DAgostinelli%26aufirst%3DC.%26aulast%3DAyers%26aufirst%3DL.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DChen-Kiang%26aufirst%3DS.%26aulast%3DElemento%26aufirst%3DO.%26aulast%3DMotiwala%26aufirst%3DT.%26aulast%3DMajumder%26aufirst%3DS.%26aulast%3DByrd%26aufirst%3DJ.%2BC.%26aulast%3DJacob%26aufirst%3DS.%26aulast%3DSif%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DC.%26aulast%3DBaiocchi%26aufirst%3DR.%2BA.%26atitle%3DSelective%2520inhibition%2520of%2520protein%2520arginine%2520methyltransferase%25205%2520blocks%2520initiation%2520and%2520maintenance%2520of%2520B-cell%2520transformation%26jtitle%3DBlood%26date%3D2015%26volume%3D125%26spage%3D2530%26epage%3D2543%26doi%3D10.1182%2Fblood-2014-12-619783" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref69"><div class="reference"><strong class="refLabel"><a href="#ref69" class="rightTabRefNumLink">69</a></strong><div class="NLM_citation" id="rightTab-cit69"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, H.</span></span> <span> </span><span class="NLM_article-title">Interaction assessments of the first S-adenosylmethionine competitive inhibitor and the essential interacting partner methylosome protein 50 with protein arginine methyltransferase 5 by combined computational methods</span>. <i>Biochem. Biophys. Res. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>495</i></span>,  <span class="NLM_fpage">721</span>– <span class="NLM_lpage">727</span>, <span class="refDoi"> DOI: 10.1016/j.bbrc.2017.11.089</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=10.1016%2Fj.bbrc.2017.11.089" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=495&publication_year=2018&pages=721-727&author=K.+Zhuauthor=C.+S.+Jiangauthor=J.+Huauthor=X.+Liuauthor=X.+Yanauthor=H.+Taoauthor=C.+Luoauthor=H.+Zhang&title=Interaction+assessments+of+the+first+S-adenosylmethionine+competitive+inhibitor+and+the+essential+interacting+partner+methylosome+protein+50+with+protein+arginine+methyltransferase+5+by+combined+computational+methods&doi=10.1016%2Fj.bbrc.2017.11.089"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit69&amp;dbid=16384&amp;doi=10.1016%2Fj.bbrc.2017.11.089&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbrc.2017.11.089%26sid%3Dliteratum%253Aachs%26aulast%3DZhu%26aufirst%3DK.%26aulast%3DJiang%26aufirst%3DC.%2BS.%26aulast%3DHu%26aufirst%3DJ.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DYan%26aufirst%3DX.%26aulast%3DTao%26aufirst%3DH.%26aulast%3DLuo%26aufirst%3DC.%26aulast%3DZhang%26aufirst%3DH.%26atitle%3DInteraction%2520assessments%2520of%2520the%2520first%2520S-adenosylmethionine%2520competitive%2520inhibitor%2520and%2520the%2520essential%2520interacting%2520partner%2520methylosome%2520protein%252050%2520with%2520protein%2520arginine%2520methyltransferase%25205%2520by%2520combined%2520computational%2520methods%26jtitle%3DBiochem.%2520Biophys.%2520Res.%2520Commun.%26date%3D2018%26volume%3D495%26spage%3D721%26epage%3D727%26doi%3D10.1016%2Fj.bbrc.2017.11.089" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref70"><div class="reference"><strong class="refLabel"><a href="#ref70" class="rightTabRefNumLink">70</a></strong><div class="NLM_citation" id="rightTab-cit70"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tarighat, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Santhanam, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frankhouser, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Radomska, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anghelina, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alinari, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walker, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caligiuri, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Croce, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garzon, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baiocchi, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marcucci, G.</span></span> <span> </span><span class="NLM_article-title">The dual epigenetic role of PRMT5 in acute myeloid leukemia: gene activation and repression via histone arginine methylation</span>. <i>Leukemia</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">789</span>– <span class="NLM_lpage">799</span>, <span class="refDoi"> DOI: 10.1038/leu.2015.308</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=10.1038%2Fleu.2015.308" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=26536822" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvFKhtrrN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2016&pages=789-799&author=S.+S.+Tarighatauthor=R.+Santhanamauthor=D.+Frankhouserauthor=H.+S.+Radomskaauthor=H.+Laiauthor=M.+Anghelinaauthor=H.+Wangauthor=X.+Huangauthor=L.+Alinariauthor=A.+Walkerauthor=M.+A.+Caligiuriauthor=C.+M.+Croceauthor=L.+Liauthor=R.+Garzonauthor=C.+Liauthor=R.+A.+Baiocchiauthor=G.+Marcucci&title=The+dual+epigenetic+role+of+PRMT5+in+acute+myeloid+leukemia%3A+gene+activation+and+repression+via+histone+arginine+methylation&doi=10.1038%2Fleu.2015.308"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit70R"><div class="casContent"><span class="casTitleNuber">70</span><div class="casTitle"><span class="NLM_cas:atitle">The dual epigenetic role of PRMT5 in acute myeloid leukemia: gene activation and repression via histone arginine methylation</span></div><div class="casAuthors">Tarighat, S. S.; Santhanam, R.; Frankhouser, D.; Radomska, H. S.; Lai, H.; Anghelina, M.; Wang, H.; Huang, X.; Alinari, L.; Walker, A.; Caligiuri, M. A.; Croce, C. M.; Li, L.; Garzon, R.; Li, C.; Baiocchi, R. A.; Marcucci, G.</div><div class="citationInfo"><span class="NLM_cas:title">Leukemia</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">789-799</span>CODEN:
                <span class="NLM_cas:coden">LEUKED</span>;
        ISSN:<span class="NLM_cas:issn">0887-6924</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Changes in the enzymic activity of protein arginine methyltransferase (PRMT) 5 have been assocd. with cancer; however, the protein's role in acute myeloid leukemia (AML) has not been fully evaluated.  Here, we show that increased PRMT5 activity enhanced AML growth in vitro and in vivo while PRMT5 downregulation reduced it.  In AML cells, PRMT5 interacted with Sp1 in a transcription repressor complex and silenced miR-29b preferentially via dimethylation of histone 4 arginine residue H4R3.  As Sp1 is also a bona fide target of miR-29b, the miR silencing resulted in increased Sp1.  This event in turn led to transcription activation of FLT3, a gene that encodes a receptor tyrosine kinase.  Inhibition of PRMT5 via sh/siRNA or a first-in-class small-mol. inhibitor (HLCL-61) resulted in significantly increased expression of miR-29b and consequent suppression of Sp1 and FLT3 in AML cells.  As a result, significant antileukemic activity was achieved.  Collectively, our data support a novel leukemogenic mechanism in AML where PRMT5 mediates both silencing and transcription of genes that participate in a 'yin-yang' functional network supporting leukemia growth.  As FLT3 is often mutated in AML and pharmacol. inhibition of PRMT5 appears feasible, the PRMT5-miR-29b-FLT3 network should be further explored as a novel therapeutic target for AML.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrGtynJRsb5obVg90H21EOLACvtfcHk0lhLoURMcHjqtA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvFKhtrrN&md5=c60b0797c096b171d970ec785d0f6110</span></div><a href="/servlet/linkout?suffix=cit70&amp;dbid=16384&amp;doi=10.1038%2Fleu.2015.308&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fleu.2015.308%26sid%3Dliteratum%253Aachs%26aulast%3DTarighat%26aufirst%3DS.%2BS.%26aulast%3DSanthanam%26aufirst%3DR.%26aulast%3DFrankhouser%26aufirst%3DD.%26aulast%3DRadomska%26aufirst%3DH.%2BS.%26aulast%3DLai%26aufirst%3DH.%26aulast%3DAnghelina%26aufirst%3DM.%26aulast%3DWang%26aufirst%3DH.%26aulast%3DHuang%26aufirst%3DX.%26aulast%3DAlinari%26aufirst%3DL.%26aulast%3DWalker%26aufirst%3DA.%26aulast%3DCaligiuri%26aufirst%3DM.%2BA.%26aulast%3DCroce%26aufirst%3DC.%2BM.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DGarzon%26aufirst%3DR.%26aulast%3DLi%26aufirst%3DC.%26aulast%3DBaiocchi%26aufirst%3DR.%2BA.%26aulast%3DMarcucci%26aufirst%3DG.%26atitle%3DThe%2520dual%2520epigenetic%2520role%2520of%2520PRMT5%2520in%2520acute%2520myeloid%2520leukemia%253A%2520gene%2520activation%2520and%2520repression%2520via%2520histone%2520arginine%2520methylation%26jtitle%3DLeukemia%26date%3D2016%26volume%3D30%26spage%3D789%26epage%3D799%26doi%3D10.1038%2Fleu.2015.308" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref71"><div class="reference"><strong class="refLabel"><a href="#ref71" class="rightTabRefNumLink">71</a></strong><div class="NLM_citation" id="rightTab-cit71"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Politzer, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clark, T.</span></span> <span> </span><span class="NLM_article-title">Halogen bonding: an electrostatically-driven highly directional noncovalent interaction</span>. <i>Phys. Chem. Chem. Phys.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">7748</span>– <span class="NLM_lpage">7757</span>, <span class="refDoi"> DOI: 10.1039/c004189k</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=10.1039%2Fc004189k" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=20571692" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=1%3ACAS%3A528%3ADC%252BC3cXosFaqtrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2010&pages=7748-7757&author=P.+Politzerauthor=J.+S.+Murrayauthor=T.+Clark&title=Halogen+bonding%3A+an+electrostatically-driven+highly+directional+noncovalent+interaction&doi=10.1039%2Fc004189k"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit71R"><div class="casContent"><span class="casTitleNuber">71</span><div class="casTitle"><span class="NLM_cas:atitle">Halogen bonding: an electrostatically-driven highly directional noncovalent interaction</span></div><div class="casAuthors">Politzer, Peter; Murray, Jane S.; Clark, Timothy</div><div class="citationInfo"><span class="NLM_cas:title">Physical Chemistry Chemical Physics</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">28</span>),
    <span class="NLM_cas:pages">7748-7757</span>CODEN:
                <span class="NLM_cas:coden">PPCPFQ</span>;
        ISSN:<span class="NLM_cas:issn">1463-9076</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A review.  A halogen bond is a highly directional, electrostatically-driven noncovalent interaction between a region of pos. electrostatic potential on the outer side of the halogen X in a mol. R-X and a neg. site B, such as a lone pair of a Lewis base or the π-electrons of an unsatd. system.  The pos. region on X corresponds to the electronically-depleted outer lobe of the half-filled p-type orbital of X that is involved in forming the covalent bond to R.  This depletion is labeled a σ-hole.  The resulting pos. electrostatic potential is along the extension of the R-X bond, which accounts for the directionality of halogen bonding.  Pos. σ-holes can also be found on covalently-bonded Group IV-VI atoms, which can similarly interact electrostatically with neg. sites.  Since pos. σ-holes often exist in conjunction with neg. potentials on other portions of the atom's surface, such atoms can interact electrostatically with both nucleophiles and electrophiles, as has been obsd. in surveys of crystallog. structures.  Exptl. as well as computational studies indicate that halogen and other σ-hole interactions can be competitive with hydrogen bonding, which itself can be viewed as a subset of σ-hole bonding.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrnp-PekfjXjrVg90H21EOLACvtfcHk0lhLoURMcHjqtA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXosFaqtrg%253D&md5=552d2699223590f79f8a0954487dc8f6</span></div><a href="/servlet/linkout?suffix=cit71&amp;dbid=16384&amp;doi=10.1039%2Fc004189k&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc004189k%26sid%3Dliteratum%253Aachs%26aulast%3DPolitzer%26aufirst%3DP.%26aulast%3DMurray%26aufirst%3DJ.%2BS.%26aulast%3DClark%26aufirst%3DT.%26atitle%3DHalogen%2520bonding%253A%2520an%2520electrostatically-driven%2520highly%2520directional%2520noncovalent%2520interaction%26jtitle%3DPhys.%2520Chem.%2520Chem.%2520Phys.%26date%3D2010%26volume%3D12%26spage%3D7748%26epage%3D7757%26doi%3D10.1039%2Fc004189k" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref72"><div class="reference"><strong class="refLabel"><a href="#ref72" class="rightTabRefNumLink">72</a></strong><div class="NLM_citation" id="rightTab-cit72"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lian, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ye, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, C.</span></span> <span> </span><span class="NLM_article-title">Development of potent type I protein arginine methyltransferase (PRMT) inhibitors of leukemia cell proliferation</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">8888</span>– <span class="NLM_lpage">8905</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b01134</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b01134" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhs1CisbfN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=8888-8905&author=C.+Wangauthor=H.+Jiangauthor=J.+Jinauthor=Y.+Xieauthor=Z.+Chenauthor=H.+Zhangauthor=F.+Lianauthor=Y.+C.+Liuauthor=C.+Zhangauthor=H.+Dingauthor=S.+Chenauthor=N.+Zhangauthor=Y.+Zhangauthor=H.+Jiangauthor=K.+Chenauthor=F.+Yeauthor=Z.+Yaoauthor=C.+Luo&title=Development+of+potent+type+I+protein+arginine+methyltransferase+%28PRMT%29+inhibitors+of+leukemia+cell+proliferation&doi=10.1021%2Facs.jmedchem.7b01134"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit72R"><div class="casContent"><span class="casTitleNuber">72</span><div class="casTitle"><span class="NLM_cas:atitle">Development of Potent Type I Protein Arginine Methyltransferase (PRMT) Inhibitors of Leukemia Cell Proliferation</span></div><div class="casAuthors">Wang, Chen; Jiang, Hao; Jin, Jia; Xie, Yiqian; Chen, Zhifeng; Zhang, Hao; Lian, Fulin; Liu, Yu-Chih; Zhang, Chenhua; Ding, Hong; Chen, Shijie; Zhang, Naixia; Zhang, Yuanyuan; Jiang, Hualiang; Chen, Kaixian; Ye, Fei; Yao, Zhiyi; Luo, Cheng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">8888-8905</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Protein Arginine Methyltransferases (PRMTs) are crucial players in diverse biol. processes, and dysregulation of PRMTs has been linked to various human diseases, esp. cancer.  Therefore, small mols. targeting PRMTs have profound impact for both academic functional studies and clin. disease treatment.  Here, the authors report the discovery of N1-(2-((2-chlorophenyl)thio)benzyl)-N1-methylethane-1,2-diamine (28d, DCPR049_12), a highly potent inhibitor of type I PRMTs that has good selectivity against a panel of other methyltransferases.  Compd. 28d effectively inhibits cell proliferation in several leukemia cell lines and reduces the cellular asym. arginine dimethylation levels.  Serving as an effective inhibitor, 28d demonstrates the mechanism of cell killing in both cell cycle arrest and apoptotic effect as well as downregulation of the pivotal mixed lineage leukemia (MLL) fusion target genes such as HOXA9 and MEIS1, which reflects the crit. roles of type I PRMTs in MLL leukemia.  These studies present 28d as a valuable inhibitor to investigate the role of type I PRMTs in cancer and other diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrddA_Xz0t9VLVg90H21EOLACvtfcHk0lhLoURMcHjqtA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhs1CisbfN&md5=40953f4e3f8d78075d444451c67ff6e0</span></div><a href="/servlet/linkout?suffix=cit72&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b01134&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b01134%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DC.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DJin%26aufirst%3DJ.%26aulast%3DXie%26aufirst%3DY.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DZhang%26aufirst%3DH.%26aulast%3DLian%26aufirst%3DF.%26aulast%3DLiu%26aufirst%3DY.%2BC.%26aulast%3DZhang%26aufirst%3DC.%26aulast%3DDing%26aufirst%3DH.%26aulast%3DChen%26aufirst%3DS.%26aulast%3DZhang%26aufirst%3DN.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DChen%26aufirst%3DK.%26aulast%3DYe%26aufirst%3DF.%26aulast%3DYao%26aufirst%3DZ.%26aulast%3DLuo%26aufirst%3DC.%26atitle%3DDevelopment%2520of%2520potent%2520type%2520I%2520protein%2520arginine%2520methyltransferase%2520%2528PRMT%2529%2520inhibitors%2520of%2520leukemia%2520cell%2520proliferation%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D8888%26epage%3D8905%26doi%3D10.1021%2Facs.jmedchem.7b01134" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref73"><div class="reference"><strong class="refLabel"><a href="#ref73" class="rightTabRefNumLink">73</a></strong><div class="NLM_citation" id="rightTab-cit73"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Webb, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amici, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jablonski, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Savardekar, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Panfil, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peine, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karkhanis, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bachelder, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ainslie, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Green, P. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baiocchi, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guerau-de-Arellano, M.</span></span> <span> </span><span class="NLM_article-title">PRMT5-selective inhibitors suppress inflammatory T cell responses and experimental autoimmune encephalomyelitis</span>. <i>J. Immunol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>198</i></span>,  <span class="NLM_fpage">1439</span>– <span class="NLM_lpage">1451</span>, <span class="refDoi"> DOI: 10.4049/jimmunol.1601702</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=10.4049%2Fjimmunol.1601702" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=198&publication_year=2017&pages=1439-1451&author=L.+M.+Webbauthor=S.+A.+Amiciauthor=K.+A.+Jablonskiauthor=H.+Savardekarauthor=A.+R.+Panfilauthor=L.+Liauthor=W.+Zhouauthor=K.+Peineauthor=V.+Karkhanisauthor=E.+M.+Bachelderauthor=K.+M.+Ainslieauthor=P.+L.+Greenauthor=C.+Liauthor=R.+A.+Baiocchiauthor=M.+Guerau-de-Arellano&title=PRMT5-selective+inhibitors+suppress+inflammatory+T+cell+responses+and+experimental+autoimmune+encephalomyelitis&doi=10.4049%2Fjimmunol.1601702"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit73&amp;dbid=16384&amp;doi=10.4049%2Fjimmunol.1601702&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4049%252Fjimmunol.1601702%26sid%3Dliteratum%253Aachs%26aulast%3DWebb%26aufirst%3DL.%2BM.%26aulast%3DAmici%26aufirst%3DS.%2BA.%26aulast%3DJablonski%26aufirst%3DK.%2BA.%26aulast%3DSavardekar%26aufirst%3DH.%26aulast%3DPanfil%26aufirst%3DA.%2BR.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DZhou%26aufirst%3DW.%26aulast%3DPeine%26aufirst%3DK.%26aulast%3DKarkhanis%26aufirst%3DV.%26aulast%3DBachelder%26aufirst%3DE.%2BM.%26aulast%3DAinslie%26aufirst%3DK.%2BM.%26aulast%3DGreen%26aufirst%3DP.%2BL.%26aulast%3DLi%26aufirst%3DC.%26aulast%3DBaiocchi%26aufirst%3DR.%2BA.%26aulast%3DGuerau-de-Arellano%26aufirst%3DM.%26atitle%3DPRMT5-selective%2520inhibitors%2520suppress%2520inflammatory%2520T%2520cell%2520responses%2520and%2520experimental%2520autoimmune%2520encephalomyelitis%26jtitle%3DJ.%2520Immunol.%26date%3D2017%26volume%3D198%26spage%3D1439%26epage%3D1451%26doi%3D10.4049%2Fjimmunol.1601702" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref74"><div class="reference"><strong class="refLabel"><a href="#ref74" class="rightTabRefNumLink">74</a></strong><div class="NLM_citation" id="rightTab-cit74"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mao, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duan, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, C.</span></span> <span> </span><span class="NLM_article-title">Potent, selective, and cell active protein arginine methyltransferase 5 (PRMT5) inhibitor developed by structure-based virtual screening and hit optimization</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">6289</span>– <span class="NLM_lpage">6304</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b00587</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b00587" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtVKitr7E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=6289-6304&author=R.+Maoauthor=J.+Shaoauthor=K.+Zhuauthor=Y.+Zhangauthor=H.+Dingauthor=C.+Zhangauthor=Z.+Shiauthor=H.+Jiangauthor=D.+Sunauthor=W.+Duanauthor=C.+Luo&title=Potent%2C+selective%2C+and+cell+active+protein+arginine+methyltransferase+5+%28PRMT5%29+inhibitor+developed+by+structure-based+virtual+screening+and+hit+optimization&doi=10.1021%2Facs.jmedchem.7b00587"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit74R"><div class="casContent"><span class="casTitleNuber">74</span><div class="casTitle"><span class="NLM_cas:atitle">Potent, Selective, and Cell Active Protein Arginine Methyltransferase 5 (PRMT5) Inhibitor Developed by Structure-Based Virtual Screening and Hit Optimization</span></div><div class="casAuthors">Mao, Ruifeng; Shao, Jingwei; Zhu, Kongkai; Zhang, Yuanyuan; Ding, Hong; Zhang, Chenhua; Shi, Zhe; Jiang, Hualiang; Sun, Dequn; Duan, Wenhu; Luo, Cheng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">6289-6304</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Abstr. PRMT5 plays important roles in diverse cellular processes and is upregulated in several human malignancies.  Besides, PRMT5 has been validated as anticancer target in mantle cell lymphoma.  In this study, the authors found a potent and selective PRMT5 inhibitor by performing structure-based virtual screening and hit optimization.  The identified compd. I (IC50 = 0.33 μM) exhibited a broad selectivity against a panel of other methyltransferases.  The direct binding of I to PRMT5 was validated by surface plasmon resonance expts., with a Kd of 0.987 μM.  Kinetic expts. indicated that I was a SAM competitive inhibitor other than substrate.  In addn., I showed selective antiproliferative effects against MV4-11 cells, and further studies indicated that the mechanism of cellular antitumor activity was due to inhibition of PRMT5 mediated SmD3 methylation. I may represent a promising lead compd. to understand more about PRMT5 and potentially assist the development of treatments for leukemia indications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoI1jnM_kEhRrVg90H21EOLACvtfcHk0lgmETRyMYMrVQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtVKitr7E&md5=5e3687166e2e6b5558409806e3c4b9e5</span></div><a href="/servlet/linkout?suffix=cit74&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b00587&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b00587%26sid%3Dliteratum%253Aachs%26aulast%3DMao%26aufirst%3DR.%26aulast%3DShao%26aufirst%3DJ.%26aulast%3DZhu%26aufirst%3DK.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DDing%26aufirst%3DH.%26aulast%3DZhang%26aufirst%3DC.%26aulast%3DShi%26aufirst%3DZ.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DSun%26aufirst%3DD.%26aulast%3DDuan%26aufirst%3DW.%26aulast%3DLuo%26aufirst%3DC.%26atitle%3DPotent%252C%2520selective%252C%2520and%2520cell%2520active%2520protein%2520arginine%2520methyltransferase%25205%2520%2528PRMT5%2529%2520inhibitor%2520developed%2520by%2520structure-based%2520virtual%2520screening%2520and%2520hit%2520optimization%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D6289%26epage%3D6304%26doi%3D10.1021%2Facs.jmedchem.7b00587" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref75"><div class="reference"><strong class="refLabel"><a href="#ref75" class="rightTabRefNumLink">75</a></strong><div class="NLM_citation" id="rightTab-cit75"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ji, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, W. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, S. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, T. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiang, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, Y. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, S. Y.</span></span> <span> </span><span class="NLM_article-title">Discovery of selective protein arginine methyltransferase 5 inhibitors and biological evaluations</span>. <i>Chem. Biol. Drug Des.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>89</i></span>,  <span class="NLM_fpage">585</span>– <span class="NLM_lpage">598</span>, <span class="refDoi"> DOI: 10.1111/cbdd.12881</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=10.1111%2Fcbdd.12881" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=27714957" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvVWmu73E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=89&publication_year=2017&pages=585-598&author=S.+Jiauthor=S.+Maauthor=W.+J.+Wangauthor=S.+Z.+Huangauthor=T.+Q.+Wangauthor=R.+Xiangauthor=Y.+G.+Huauthor=Q.+Chenauthor=L.+L.+Liauthor=S.+Y.+Yang&title=Discovery+of+selective+protein+arginine+methyltransferase+5+inhibitors+and+biological+evaluations&doi=10.1111%2Fcbdd.12881"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit75R"><div class="casContent"><span class="casTitleNuber">75</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of selective protein arginine methyltransferase 5 inhibitors and biological evaluations</span></div><div class="casAuthors">Ji, Sen; Ma, Shuang; Wang, Wen-Jing; Huang, Shen-Zhen; Wang, Tian-qi; Xiang, Rong; Hu, Yi-Guo; Chen, Qiang; Li, Lin-Li; Yang, Sheng-Yong</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Biology & Drug Design</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">89</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">585-598</span>CODEN:
                <span class="NLM_cas:coden">CBDDAL</span>;
        ISSN:<span class="NLM_cas:issn">1747-0277</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">Protein arginine methyltransferase 5 (PRMT5) is an important protein arginine methyltransferase that catalyzes the sym. dimethylation of arginine resides on histones or nonhistone substrate proteins.  It has been thought as a promising target for many diseases, particularly cancer.  Despite the potential applications of PRMT5 inhibitors in cancer treatment, very few of PRMT5i have been publicly reported.  In this investigation, virtual screening and structure-activity relationship studies were carried out to discover novel PRMT5i, which finally led to the identification of a no. of new PRMT5i.  The most active compd., I, exhibited a considerable inhibitory potency against PRMT5 with an IC50 value of 0.57 μM, and a high selectivity for PRMT5 against other tested PRMTs.  It displayed a very good anti-viability activity against two colorectal cancer cell lines, HT-29 and DLD-1, and one hepatic cancer cell line, HepG2, in a sensitivity assay against 36 different cancer cell lines.  Western blot assays indicated that I selectively inhibited the sym. dimethylations of H4R3 and H3R8 in DLD-1 cells.  Overall, I could be used as a chem. probe to investigate new functions of PRMT5 in biol. and also served as a good lead compd. for the development of new PRMT5-targeting therapeutic agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq_6yDJULZD-LVg90H21EOLACvtfcHk0lgmETRyMYMrVQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvVWmu73E&md5=1d4f06bef5c95682073c2fcb1e1859d4</span></div><a href="/servlet/linkout?suffix=cit75&amp;dbid=16384&amp;doi=10.1111%2Fcbdd.12881&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fcbdd.12881%26sid%3Dliteratum%253Aachs%26aulast%3DJi%26aufirst%3DS.%26aulast%3DMa%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DW.%2BJ.%26aulast%3DHuang%26aufirst%3DS.%2BZ.%26aulast%3DWang%26aufirst%3DT.%2BQ.%26aulast%3DXiang%26aufirst%3DR.%26aulast%3DHu%26aufirst%3DY.%2BG.%26aulast%3DChen%26aufirst%3DQ.%26aulast%3DLi%26aufirst%3DL.%2BL.%26aulast%3DYang%26aufirst%3DS.%2BY.%26atitle%3DDiscovery%2520of%2520selective%2520protein%2520arginine%2520methyltransferase%25205%2520inhibitors%2520and%2520biological%2520evaluations%26jtitle%3DChem.%2520Biol.%2520Drug%2520Des.%26date%3D2017%26volume%3D89%26spage%3D585%26epage%3D598%26doi%3D10.1111%2Fcbdd.12881" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4GQB" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4GQB','PDB','4GQB'); return false;">PDB: 4GQB</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4X61" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4X61','PDB','4X61'); return false;">PDB: 4X61</a></li></ul></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;pbContext=%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2018.61.issue-21%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.8b00598%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.8b00598" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                6MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"677ad0dd9d523d94","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
